
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Structures and mechanisms of U6 snRNA m6A modification by METTL16 - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152539AC8AF201630539AC005047AB73.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371086/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Structures and mechanisms of U6 snRNA m6A modification by METTL16">
<meta name="citation_author" content="Jue Ju">
<meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo Kashiwa, Chiba, Japan">
<meta name="citation_author" content="Kozo Tomita">
<meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo Kashiwa, Chiba, Japan">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7708">
<meta name="citation_doi" content="10.1038/s41467-025-63021-0">
<meta name="citation_pmid" content="40841561">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371086/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371086/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371086/pdf/41467_2025_Article_63021.pdf">
<meta name="description" content="The N6-methyladenosine (m6A) modification in U6 snRNA, catalyzed by METTL16 using S-adenosylmethionine (SAM) as the methyl donor, is required for efficient and accurate pre-mRNA splicing. However, the mechanism by which METTL16 modifies U6 snRNA ...">
<meta name="og:title" content="Structures and mechanisms of U6 snRNA m6A modification by METTL16">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The N6-methyladenosine (m6A) modification in U6 snRNA, catalyzed by METTL16 using S-adenosylmethionine (SAM) as the methyl donor, is required for efficient and accurate pre-mRNA splicing. However, the mechanism by which METTL16 modifies U6 snRNA ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371086/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371086">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63021-0"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63021.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371086%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371086/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371086/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371086/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7708. doi: <a href="https://doi.org/10.1038/s41467-025-63021-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63021-0</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Structures and mechanisms of U6 snRNA m<sup>6</sup>A modification by METTL16</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ju%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Jue Ju</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Jue Ju</span></h3>
<div class="p">
<sup>1</sup>Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo Kashiwa, Chiba, Japan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ju%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jue Ju</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tomita%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Kozo Tomita</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Kozo Tomita</span></h3>
<div class="p">
<sup>1</sup>Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo Kashiwa, Chiba, Japan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tomita%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kozo Tomita</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo Kashiwa, Chiba, Japan </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 3; Accepted 2025 Aug 7; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371086  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841561/" class="usa-link">40841561</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The <em>N</em><sup>6</sup>-methyladenosine (m<sup>6</sup>A) modification in U6 snRNA, catalyzed by METTL16 using <em>S</em>-adenosylmethionine (SAM) as the methyl donor, is required for efficient and accurate pre-mRNA splicing. However, the mechanism by which METTL16 modifies U6 snRNA with m<sup>6</sup>A remains elusive. Here, we present cryo-EM structures of METTL16 in complex with U6 snRNA, providing insights into the METTL16-mediated modification of U6 snRNA with m<sup>6</sup>A. The structures reveal that U6 snRNA is recruited to METTL16 through specific interactions between the C-terminal kinase-associated 1 (KA-1) domain of METTL16 and the internal stem-loop (ISL) of U6 snRNA. Upon SAM binding to the catalytic pocket within the N-terminal methyltransferase domain (MTD), U6 snRNA undergoes a structural rearrangement that positions the target adenine-containing motif at the catalytic site. This conformational change is followed by an additional structural adjustment of U6 snRNA into a productive conformation, bringing the target adenosine closer to SAM within the catalytic pocket and thereby ensuring efficient m<sup>6</sup>A modification. The KA-1 domain functions as a scaffold for initial substrate recognition and facilitates the subsequent dynamic methylation process within the MTD, highlighting the cooperative roles of METTL16 domains for U6 snRNA modification.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Cryoelectron microscopy, RNA modification, Enzyme mechanisms</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">U6 snRNA m<sup>6</sup>A modification by METTL16 is essential for accurate pre-mRNA splicing. Here, the authors determined the cryo-EM structure of METTL16 bound to U6 snRNA and SAM, revealing how its two domains cooperatively guide RNA recognition and catalysis via a dynamic twostep mechanism.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3"><em>N</em>
<sup>6</sup>-methyladenosine (m<sup>6</sup>A) is a well-characterized mRNA modification crucial for gene expression in various biological processes<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. It regulates RNA splicing, localization, stability, and translation<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>.</p>
<p id="Par4">Most m<sup>6</sup>A modifications in mRNAs are catalyzed by the METTL3/METTL14 complex, which methylates the specific adenosine in the DRACH (D = A, G or U, R = A or G; H = A, C, or U) motif close to the stop codon or 3’-UTR of mRNAs<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. METTL3 acts as the catalytic subunit, and METTL14 plays a structural role critical for substrate recognition<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>.</p>
<p id="Par5">A distinct m<sup>6</sup>A methyltransferase, METTL16, targets a particular motif within specific RNA structures, including mRNAs and non-coding RNAs<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>–<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. In vertebrates, METTL16 installs m<sup>6</sup>A in RNA hairpins containing the motif UACA*GAR, where A* denotes the methylation site, within the 3′ UTR of methionine adenosyltransferase 2A (MAT2A) mRNA, regulating <em>S</em>-adenosylmethionine (SAM) homeostasis by modulating MAT2A pre-mRNA stability and splicing<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Under high SAM conditions, the YTH domain-containing protein YTHDC1 binds to the m<sup>6</sup>A-modified hairpins, promoting MAT2A mRNA degradation and reducing SAM synthesis<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. In contrast, under SAM-limited conditions, prolonged METTL16 binding to the 3′ UTR hairpin recruits splicing factors, enhancing pre-mRNA splicing and increasing SAM production<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. In invertebrates such as <em>C. elegans</em>, the METTL16 homolog, METT10, methylates the adenosine in the AG dinucleotide within the UACA*GAGA motif at the 3′-splice site (3′-ss) of SAM synthetase (<em>sams</em>) mRNA, affecting spliceosome assembly and splicing efficiency<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. High SAM levels sustain the m<sup>6</sup>A modification at this site, preventing U2AF35 binding and inhibiting splicing<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>, and thereby suppressing SAM production. Conversely, low SAM levels reduce the m<sup>6</sup>A modification, allowing U2AF35 to bind, promoting splicing and enhancing SAM synthetase mRNA expression, and ultimately increasing SAM production.</p>
<p id="Par6">METTL16 and its homologs also methylate a conserved adenosine in the ACA*GAGA motif of U6 snRNA (A43 in human U6 snRNA)<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. The ACA*GAG box in U6 snRNA base pairs with the 5′ splice site during pre-mRNA splicing<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>–<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. The m<sup>6</sup>A modification in U6 snRNA is required for efficient and accurate splicing of specific pre-mRNAs that possess the BBH motif (B: G, C, or U; H: A, C, or U) at positions −3 to −1 and an A at position 4 (A<sub>+4</sub>) of the 5′ splice site (5′-ss)<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>–<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. In spliceosomal B complex, the m<sup>6</sup>A in U6 snRNA and the 5’-ss A<sub>+4</sub> form a trans Hoogsteen sugar edge interaction<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, thereby stabilizes the U6 snRNA-5’-ss interaction<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. The BBH motif at the 5′-ss in pre-mRNAs weakly interact with U5, and thus the absence of m<sup>6</sup>A in U6 snRNA further destabilizes the interaction between the 5′-ss and U6 snRNA, resulting in intron retention within these specific pre-mRNAs<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>.</p>
<p id="Par7">METTL16 consists of an N-terminal methyltransferase domain (MTD) and a C-terminal kinase-associated 1 (KA-1) domain<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. The KA-1 domain was initially named the vertebrate conserved region (VCR) but later renamed<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>, since the KA-1 domain is widely conserved among METTL16 and its homologs across vertebrates, invertebrates, and even fission yeasts<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>, and the structure is homologous to the KA-1 found in MARK/PAR1 kinases<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> and U6-specific terminal uridylyltransferases (TUTase1, TENT1)<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>.</p>
<p id="Par8">The crystal structures of the MTD of human METTL16 (hsMETTL16), its complex with <em>S</em>-adenosyl-L-homocysteine (SAH), and its complex with a short hairpin from the 3’-UTR of MAT2A mRNA (MAT2A-hp) have been reported<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. The MTD of hsMETTL16 (hsMTD) catalyzed adenosine methylation in the MAT2A-hp as efficiently as full-length hsMETTL16<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Both the consensus RNA sequence and intrinsic RNA structural features—the loop and the transition region of MAT2A-hp— are critical determinants of the methylation efficiency by hsMTD<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. The <em>C. elegans</em> METT10 MTD similarly recognizes specific structural features of the RNA surrounding the 3′-ss of <em>sams</em> pre-mRNA, sharing a substrate recognition mechanism with hsMTD<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Regarding the C-terminal KA-1 domains, predictions based on structural studies have suggested that they are well conserved among eukaryotes<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. The KA-1 domains of hsMETTL16 and ceMETT10 facilitate m<sup>6</sup>A methylation of the conserved adenosine in U6 snRNA<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> by increasing the affinity of METTL16 (or ceMETT10) for U6 snRNA and enhancing the catalytic efficiency. A biochemical analysis also suggested that the KA-1 domain would interact with the internal stem-loop (ISL) region of U6 snRNA<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. However, the precise mechanisms of U6 snRNA recognition by METTL16 and the role of its KA-1 domain in enhancing methylation remain elusive.</p>
<p id="Par9">Here, we present the structural and functional analysis of <em>Schizosaccharomyces pombe</em> METTL16, revealing the mechanism of U6 snRNA methylation by METTL16. The KA-1 domain serves as a scaffold for the multi-step methylation process—U6 snRNA ISL recognition, motif recognition in the MTD, and target adenosine positioning in the catalytic site—ensuring the U6 snRNA m<sup>6</sup>A modification, which leads to accurate and efficient pre-mRNA splicing.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">KA-1 of METTL16 facilitates U6 snRNA m<sup>6</sup>A modification</h3>
<p id="Par10">The C-terminal KA-1 domain of human METTL16 (hsMETTL16) reportedly facilitates the m<sup>6</sup>A43 modification of U6 snRNA in vitro<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, and KA-1 domains are conserved among METTL16 homologs in vertebrates and invertebrates<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. The KA-1 domain is also present in METTL16 homologs in fission yeast<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> (Fig. <a href="#Fig1" class="usa-link">1a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">1a, b</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. The KA-1 domain of METTL16 facilitates m<sup>6</sup>A modification of U6 snRNA and splicing.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371086_41467_2025_63021_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f31c/12371086/7a0cb0b59d0f/41467_2025_63021_Fig1_HTML.jpg" loading="lazy" id="d33e554" height="827" width="799" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic diagram of <em>S. pombe</em> METTL16 (spMETTL16) compared with human METTL16 (hsMETTL16) and <em>C. elegans</em> METT10 (ceMETT10). The N-terminal methyltransferase domain (spMTD, magenta) and the C-terminal KA-1 domain (spKA-1, cyan) of <em>S. pombe</em>, used in the experiments, are depicted. <strong>b</strong> In vitro methylation of the <em>S. pombe</em> U6 snRNA (spU6 snRNA) transcript by spMETTL16 and spMTD under standard conditions. spU6 snRNA (0.5 μM) was incubated with 200 nM spMETTL16 or spMTD in the presence of 1 mM SAM for 4 min at 37 °C. Error bars represent the standard deviation (SD) of three independent experiments (<em>N</em> = 3), and the center of the error bands indicates the mean of the measured values. <strong>c</strong> Steady-state kinetics of methylation of the spU6 snRNA transcript by spMETTL16 and spMTD. spU6 snRNA (0–10 μM) was incubated with 200 nM spMETTL16 or spMTD in the presence of 1 mM SAM at 37 °C. All experiments were independently repeated three times with similar results. <strong>d</strong>, <strong>e</strong> Semi-quantitative RT-PCR analyses of two introns: <strong>d</strong>
<em>SPAC18B11.09 C</em> and <strong>e</strong>
<em>ckn1</em>. The nucleotide sequences of the respective 5′ splice sites (−3 to +4) are shown. The upper and lower bands on the gel represent retained and spliced introns, respectively. Intron retention in the Δ<em>mtl16</em> strain was rescued by ectopic expression of plasmid-encoded full-length METTL16 (spMETTL16), but not by the methyltransferase domain (spMTD) alone. All experiments were independently repeated three times with similar results (Supplementary Fig. <a href="#MOESM1" class="usa-link">2a, b</a>). <strong>f</strong> Nucleotide sequence of <em>S. pombe</em> U6 snRNA (spU6 snRNA, left). The nucleotide sequence of the spISL used for crystallization of the spKA-1–spISL complex (spISL, right). <strong>g</strong> Gel retardation assay of spISL by spKA-1. spISL RNAs were incubated with various concentrations of spKA-1 (0–10 μM). The fractions of the shifted RNA in the gel (left) were quantified (right). The experiments were independently repeated two times with similar results. Source data for <strong>b</strong>, <strong>c</strong>, <strong>g</strong> are provided as a <a href="#MOESM4" class="usa-link">Source data</a> file.</p></figcaption></figure><p id="Par11">To investigate the role of the KA-1 domain in <em>Schizosaccharomyces pombe</em> METTL16 (spMETTL16) in the m<sup>6</sup>A modification at A37—corresponding to A43 in human U6 snRNA—of <em>S. pombe</em> U6 snRNA (spU6 snRNA), we examined the methylation activities of full-length spMETTL16 (amino acids 1–398) and its N-terminal methyltransferase domain (MTD; amino acids 1–280) (Fig. <a href="#Fig1" class="usa-link">1a</a>), using an spU6 snRNA transcript as a substrate in vitro. The MTD exhibited ~10% of the methylation activity compared to full-length spMETTL16 (Fig. <a href="#Fig1" class="usa-link">1b</a>). A steady-state kinetic analysis provided estimated <em>K</em><sub>m</sub> values of spU6 snRNA for full-length spMETTL16 and the MTD of 0.182 μM and 0.752 μM, respectively (Fig. <a href="#Fig1" class="usa-link">1c</a>). The <em>k</em><sub>cat</sub> values were estimated to be 0.38 min<sup>−1</sup> for full-length spMETTL16 and 0.037 min<sup>−1</sup> for the MTD. Thus, the catalytic efficiency (<em>k</em><sub>cat</sub>/<em>K</em><sub>m</sub>) of the MTD was ~2% of that of full-length spMETTL16. These results indicate that the KA-1 domain of spMETTL16 facilitates A37 methylation in spU6 snRNA, similar to its role in hsMETTL16<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>.</p></section><section id="Sec4"><h3 class="pmc_sec_title">KA-1 of METTL16 facilitates pre-mRNA splicing</h3>
<p id="Par12">The m<sup>6</sup>A modification at A37 of spU6 snRNA is required for the efficient splicing of specific pre-mRNAs containing the BBH motif (B = G, C, or U; H = A, C, or U) at positions −3 to −1 and an A at position 4 of the 5′ splice site (5′-ss)<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. In <em>S. pombe</em>, the <em>SPAC18B11.09c</em> and <em>ckn1</em> transcripts have the sequences A<sub>-3</sub>C<sub>-2</sub>C<sub>-1</sub>/G<sub>1</sub>U<sub>2</sub>A<sub>3</sub>A<sub>4</sub> at the 5′-ss between exon 1 and exon 2, and U<sub>-3</sub>G<sub>-2</sub>C<sub>-1</sub>/G<sub>1</sub>U<sub>2</sub>A<sub>3</sub>A<sub>4</sub> at the 5′-ss between exon 3 and exon 4, respectively<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> (Fig. <a href="#Fig1" class="usa-link">1d, e</a>). These 5′-ss in pre-mRNAs weakly interact with U5, and the absence of the m<sup>6</sup>A37 modification in spU6 snRNA destabilizes the interaction between the 5′-ss in the pre-mRNA and U6 snRNA, resulting in intron retention<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>.</p>
<p id="Par13">To assess whether the KA-1 domain could facilitate A37 methylation and regulate pre-mRNA splicing in vivo, we transformed a <em>mettl</em>16 knockout strain (Δ<em>mtl</em>16) with a plasmid expressing the MTD of spMETTL16 and analyzed the splicing of specific introns in pre-mRNAs (<em>SPAC18B11.09c</em> and <em>ckn1</em>) by RT-PCR (Fig. <a href="#Fig1" class="usa-link">1d, e</a>). In the Δ<em>mtl</em>16 strain, introns were retained compared to the wild-type strain, whereas ectopic expression of full-length spMETTL16 restored proper splicing and intron removal. In contrast, expression of the spMTD in the Δ<em>mtl</em>16 strain failed to rescue splicing (Fig. <a href="#Fig1" class="usa-link">1d, e</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2a, b</a>).</p>
<p id="Par14">Together with the in vitro analyses (Fig. <a href="#Fig1" class="usa-link">1b, c</a>), these results suggest that the KA-1 domain of spMETTL16 facilitates the m<sup>6</sup>A modification of spU6 snRNA, thereby promoting the efficient splicing of specific pre-mRNAs in vivo.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Crystal structure of the KA-1 domain in complex with the ISL of U6 snRNA</h3>
<p id="Par15">The KA-1 domain of hsMETTL16 has been suggested to interact with the internal stem-loop (ISL) region of U6 snRNA, based on a previous biochemical study<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Our cryo-EM analysis of the spMETTL16–spU6 snRNA complex, as described below, also indicated that the ISL region of spU6 snRNA specifically interacts with the KA-1 domain of spMETTL16. The gel-retardation assay showed that the KA-1 domain of spMETTL16 (spKA-1; residues 270–398) (Fig. <a href="#Fig1" class="usa-link">1a</a>) binds to the isolated ISL RNA fragment of spU6 snRNA (spISL; nucleotides 47–77) (Fig. <a href="#Fig1" class="usa-link">1f</a>) with a <em>K</em><sub>d</sub> of ~1.2 μM (Fig. <a href="#Fig1" class="usa-link">1g</a>).</p>
<p id="Par16">To elucidate the molecular basis of this interaction in detail, we crystallized spKA-1 in complex with spISL and determined its structure at 2.8 Å resolution (Fig. <a href="#Fig2" class="usa-link">2a, b</a>; Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a> and Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). spKA-1 consists of five antiparallel β-sheets (β8 – β12) and three α-helices (α8 – α10) (Fig. <a href="#Fig2" class="usa-link">2a</a>). The conserved arginine-rich region (RRR) is in α9. As expected, its overall structure closely resembles that of the hsMETTL16 KA-1 domain (hsKA-1, PDB ID: 6M1U)<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, with an RMSD of 1.461 Å for 61 structurally equivalent Cα atoms (Fig. <a href="#Fig2" class="usa-link">2c</a>). The arginine-rich region (RRR) in hsKA-1 was disordered in the reported structure<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Crystal structure of spKA-1 in complex with spISL.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371086_41467_2025_63021_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f31c/12371086/1cc2faf17022/41467_2025_63021_Fig2_HTML.jpg" loading="lazy" id="d33e786" height="957" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Crystal structure of spKA−1 in complex with spISL. spKA-1 is shown in blue, and spISL is depicted as an orange stick model. <strong>b</strong> Electrostatic surface potential of spKA-1 in the spKA-1–spISL complex. The electrostatic potential is represented as blue (positive) and red (negative). spISL is shown as an orange stick model. <strong>c</strong> Superimposition of the spKA-1 structure (blue) in (<strong>a</strong>) on the KA-1 domain (hsKA-1, magenta) of human METTL16 (PDB ID: 6M1U)<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. The arginine-rich region (RRR) in spKA-1 is shown in red. The dashed region in hsKA-1 represents a disordered area, where the arginine-rich region (RRR) is located. <strong>d</strong> Structure of spISL bound to spKA-1. Adenosine, guanosine, cytidine, and uridine are colored red, blue, orange, and green, respectively. Nucleotides forming non-canonical base pairs are boxed. <strong>e</strong> Schematic representation of the interaction between spKA-1 and spISL shown in (<strong>d</strong>).</p></figcaption></figure><p id="Par17">In the complex, helices α9 and α10 interact with distinct regions of spISL: α9 engages with the major groove of the loop at the spISL tip, while α10 contacts the minor groove of the double-stranded region of spISL (Fig. <a href="#Fig2" class="usa-link">2a</a>). These interactions are mediated by positively charged residues on α9 and α10, which stabilize the spKA-1–spISL interaction (Fig. <a href="#Fig2" class="usa-link">2b</a>). The spISL RNA structure within the complex contains non-canonical base pairings, including G59-A63, C55-A67, and G53-A70 (Fig. <a href="#Fig2" class="usa-link">2d, e</a>), with the spKA-1 domain specifically recognizing these conserved non-canonical base pairs (Supplementary Fig. <a href="#MOESM1" class="usa-link">1c</a>), as described below.</p>
<p id="Par18">The arginine-rich region (RRR: R339, R342, and R343) on α9 interacts with the tip of the ISL (Figs. <a href="#Fig2" class="usa-link">2</a>a, <a href="#Fig3" class="usa-link">3a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>). The nucleobases of C60-A61-C62 in the loop at the tip of spISL stack continuously, and the C62 base stacks on the A63 base (Fig. <a href="#Fig3" class="usa-link">3a</a>). The A63 base forms a non-canonical base pair with G59, where the 2-NH<sub>2</sub> and the N3 atoms of G59 form hydrogen bonds with the N7 atom and the 6-NH<sub>2</sub> of A63, respectively. The Nε and Nη1 atoms of Arg339 form hydrogen bonds with the O6 and N7 atoms of G59, respectively (Fig. <a href="#Fig3" class="usa-link">3a</a>). The Nη2 atom of Arg339 forms a hydrogen bond with the phosphate backbone of U58. Arg343 forms a hydrogen bond with the phosphate of A61, and the side chain of Arg343 form cation-π interactions with the G59-Arg339 hydrogen bonds. Arg342 forms a hydrogen bond with the phosphate backbone of C56. Thus, the RRR extensively and specifically interacts with the tip of the ISL of spU6 snRNA. The interactions between the RRR and the tip of the ISL in the present structure were quite similar to the arginine forks recognition of RNA as observed in the crystal structures of HIV Tat in complex with TAR RNAs<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup> and Tat or HEXIM in complex with the apical stem-loop of 7SK RNA<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Interactions between the spKA-1 domain and spISL.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371086_41467_2025_63021_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f31c/12371086/3f6ba4e088c0/41467_2025_63021_Fig3_HTML.jpg" loading="lazy" id="d33e879" height="978" width="721" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>–<strong>d</strong> Detailed views of the interactions between spKA-1 and spISL. The colors for nucleosides are the same as in Fig. <a href="#Fig2" class="usa-link">2d</a>. <strong>e</strong> In vitro methylation of the spU6 snRNA transcript by spMETTL16 and its variants with mutations in the KA-1 domain under standard conditions. spU6 snRNA (0.5 μM) was incubated with 200 nM spMETTL16 or its variants in the presence of 1 mM SAM for 4 min at 37 °C. Error bars represent the standard deviation (SD) of three independent experiments (<em>N</em> = 3), and the center of the error bands indicates the mean of the measured values. Source data are provided as a <a href="#MOESM4" class="usa-link">Source data</a> file. Semi-quantitative RT-PCR analyses of two introns: <strong>f</strong> <em>SPAC18B11.09 C</em> and <strong>g</strong>
<em>ckn1</em>. The upper and lower bands on the gel represent retained and spliced introns, respectively. Intron retention in the Δ<em>mtl16</em> strain was rescued by ectopic expression of plasmid-encoded full-length METTL16 (spMETTL16), but not by the spMETTL16 variants with mutations in the KA-1 domain shown in (<strong>e</strong>) (Supplementary Fig. <a href="#MOESM1" class="usa-link">2c, d</a>).</p></figcaption></figure><p id="Par19">α10 contacts the minor groove of the double helix containing the non-canonical C55-A67 and G53-A70 base pairs (Figs. <a href="#Fig2" class="usa-link">2</a>a and <a href="#Fig3" class="usa-link">3b, c</a>). The 4-NH<sub>2</sub> of C55 forms a hydrogen bond with the N1 atom of A67, and the Nη1 atom of Arg391 forms a hydrogen bond with the O2 of C55 (Fig. <a href="#Fig3" class="usa-link">3b</a>). The N1 atom and the O6 of G53 form hydrogen bonds with the N1 atom and the 6-NH<sub>2</sub> of A70, respectively, and Glu383 forms a hydrogen bond with the 2-NH<sub>2</sub> of G53 (Fig. <a href="#Fig3" class="usa-link">3c</a>). The phosphate backbone and ribose 2′-OH form hydrogen bonds with several residues—Asn380, Ser384, Ser387, and Arg394—on α10 (Fig. <a href="#Fig2" class="usa-link">2e</a>). Lys286, Lys291, and His290 in the N-terminal part of spKA-1 form hydrogen bonds with the phosphate backbone of A72, C73 and G75, respectively, and the main-chain carbonyl of Tyr287 forms a hydrogen bond with the ribose 2′-OH of C73 (Fig. <a href="#Fig3" class="usa-link">3d</a>).</p>
<p id="Par20">The R339E, R342E, and R343E mutations in the RRR of spMETTL16 reduced the methylation activity of spMETTL16, and the double mutation (R339E/R343E) and triple mutation (R339E/R342E/R343E) further decreased the activity in vitro (Fig. <a href="#Fig3" class="usa-link">3e</a>). Although the R394E mutation slightly reduced the activity under our conditions, the R391E mutation reduced it to the same extent as a single mutation in the RRR (Fig. <a href="#Fig3" class="usa-link">3e</a>). These results suggest that the specific interactions facilitating the recognition of the ISL of spU6 snRNA are important for A37 methylation by spMETTL16.</p>
<p id="Par21">Consistently, the Δ<em>mtl</em>16 strain expressing spMETTL16 with the R339E, R339E/R343E, or R339E/R342E/R343E mutations failed to rescue the splicing of the <em>SPAC18B11.09c</em> and <em>ckn1</em> pre-mRNAs, compared to the strain expressing wild-type spMETTL16 (Fig. <a href="#Fig3" class="usa-link">3f, g</a>). Expression of spMETTL16 with the R391E or R394E mutations also failed to rescue the splicing of these pre-mRNAs, compared to the strain expressing wild-type spMETTL16, although their effects were less pronounced than those of the R339E, R339E/R343E, or R339E/R342E/R343E mutations (Supplementary Fig. <a href="#MOESM1" class="usa-link">2c, d</a>).</p>
<p id="Par22">These structural features of the interactions between spISL and spKA-1 (Fig. <a href="#Fig3" class="usa-link">3a–c</a>), along with the biochemical (Fig. <a href="#Fig3" class="usa-link">3e</a>) and functional (Fig. <a href="#Fig3" class="usa-link">3f, g</a>) analyses, suggest that spKA-1 specifically recognizes the conserved sequence and structure of the ISL of spU6 snRNA (Supplementary Fig. <a href="#MOESM1" class="usa-link">1c</a>) and is important for U6 snRNA m<sup>6</sup>A methylation and pre-mRNA splicing.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Cryo-EM structure of the METTL16-U6 snRNA complex</h3>
<p id="Par23">To further elucidate the mechanism of spU6 snRNA methylation by spMETTL16, we determined the cryo-EM structure of spMETTL16 in complex with spU6 snRNA lacking the 5′-stem (spU6 snRNA_Δ17), which can be methylated as efficiently as full-length spU6 snRNA, at 3.4 Å resolution (Fig. <a href="#Fig4" class="usa-link">4a</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a> and Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). The spMETTL16–spU6 snRNA_Δ17 complex was reconstituted by mixing purified spMETTL16 with in vitro-transcribed spU6 snRNA_Δ17, followed by size-exclusion chromatography for purification.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Cryo-EM structures of spMETTL16 in complex with spU6 snRNA.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371086_41467_2025_63021_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f31c/12371086/13de3f5bd1e1/41467_2025_63021_Fig4_HTML.jpg" loading="lazy" id="d33e987" height="991" width="646" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Cryo-EM map (left) and model (right) of the spMETTL16–spU6 snRNA complex in the absence of SAM (spMETTL16–spU6). MTD, KA-1 domain, and modeled RNA are colored magenta, blue and orange, respectively. The N-terminal extension (residues 1–61) and the RNA binding loop (residues 172–200) are not modeled, and only nucleotides 41–82 of spU6 snRNA are modeled. <strong>b</strong> Cryo-EM map (left) and model (right) of the spMETTL16–spU6 snRNA complex in the presence of SAM (spMETTL16–SAM–spU6). MTD, KA-1 domain, and modeled RNA are colored as in (<strong>a</strong>). The N-terminal extension and the RNA binding loop (residues 187–194) are not modeled, and only nucleotides 35–83 of spU6 snRNA are modeled. <strong>c</strong> Structural comparison between spMETTL16–spU6 (gray) and spMETTL16–SAM–spU6. In the presence of SAM, α4 in the MTD becomes resolved, and the ACA*GAGA motif in spU6 snRNA shifts toward the catalytic site of the MTD. <strong>d</strong> The secondary structure of spMETTL16. The MTD and KA-1 domain are colored magenta and blue, respectively. The regions enclosed by the dashed lines were not visible in the spMETTL16–spU6 structure. α4 was visible in the spMETTL16–SAM–spU6 complex. The N-terminal extension was predicted by AlphaFold2 (Data base: AF-<a href="https://www.ncbi.nlm.nih.gov/protein/O42662" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">O42662</a>-F1-v4)<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>.</p></figcaption></figure><p id="Par24">The cryo-EM density map of the spMETTL16–spU6 snRNA_Δ17 complex without SAM (hereafter referred to as spMETTL16–spU6) suggests that spU6 snRNA adopts an extended conformation. However, only the ISL region (nucleotides 41–82) could be modeled, and the entire spU6 snRNA structure could not be fully resolved (Fig. <a href="#Fig4" class="usa-link">4a</a>), due to the insufficient quality of the cryo-EM map. In the cryo-EM structure of spMETTL16–spU6, the KA-1 domain of spMETTL16 interacts with the ISL region of spU6 snRNA, resembling the interaction of spKA-1 with spISL in the crystal structure (Fig. <a href="#Fig2" class="usa-link">2a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>). The helices α9 and α10 interact with the major groove of the loop at the spISL tip and the minor groove of the double-stranded region of spISL, respectively. However, the remaining portion of spU6 snRNA does not interact with the protein. Notably, the ACA*GAGA motif containing the target A37– located between the ISL and the telestem—is far from the catalytic site of the MTD domain (Fig. <a href="#Fig4" class="usa-link">4a</a>). The N-terminal extension (residues 1–61) and the segment between β4 and α5 (residues 172–200) within the MTD (Fig. <a href="#Fig4" class="usa-link">4d</a>)—which are both potentially involved in RNA binding<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>—were disordered and could not be modeled due to the lack of a cryo-EM map. Thus, this structure likely represents an initial stage of substrate U6 snRNA recognition by spMETTL16.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Cryo-EM structure of the METTL16-SAM-U6 snRNA complex</h3>
<p id="Par25">We also determined the cryo-EM structure of spMETTL16 in complex with spU6 snRNA in the presence of SAM (hereafter referred to as spMETTL16–SAM–spU6), at 3.0 Å resolution. SAM was added to the spMETTL16–spU6 snRNA complex.</p>
<p id="Par26">In the cryo-EM structure of the spMETTL16–SAM–spU6 complex (Fig. <a href="#Fig4" class="usa-link">4b</a>, Supplementary Figs. <a href="#MOESM1" class="usa-link">6</a> and <a href="#MOESM1" class="usa-link">7</a> and Supplementary Table <a href="#MOESM1" class="usa-link">2</a>), the mode of U6 snRNA binding to spMETTL16 differs from that observed in the spMETTL16–spU6 structure (Fig. <a href="#Fig4" class="usa-link">4b, c</a>). SAM binds to the catalytic site in the MTD, and the ISL of spU6 snRNA retains its interaction with the KA-1 domain, as observed in the SAM-free structure. However, the ACAGAGA motif in spU6 snRNA shifts toward the MTD, and the methylation site, A37, is proximal to SAM in the MTD (Fig. <a href="#Fig4" class="usa-link">4b, c</a>), although the telestem region (nucleotides 1–34 and 84–99) of spU6 snRNA could not be modeled due to the lack of a cryo-EM map. The α4 helix (between β4 and α5) within the MTD, which was not visible in the spMETTL16–spU6 structure, is partially resolved in the spMETTL16–SAM–spU6 structure (Fig. <a href="#Fig4" class="usa-link">4b, c</a>). However, the loop between α4 and α5 (residues 187–194), as well as the N-terminal extension (residues 1–55), remains unresolved (Fig. <a href="#Fig4" class="usa-link">4c, d</a>).</p>
<p id="Par27">SAM is located within the catalytic site of the MTD, and its position is identical to that of SAH observed in human METTL16 (hsMETTL16) in complex with SAH (Fig. <a href="#Fig5" class="usa-link">5a, b</a>, PDB ID: 6B92)<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. The adenine base of SAM stacks with Ile120 and Phe206, while Lys148 in the K-loop<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>, which regulates SAM binding to the catalytic pocket in hsMETTL16, is positioned away from the SAM binding site. A37 in the ACA*GAGA motif of spU6 snRNA resides near the catalytic pocket of the MTD (Fig. <a href="#Fig5" class="usa-link">5b, c</a>). However, the distance between the 6-NH<sub>2</sub> of A37 in spU6 snRNA and the methyl group of SAM is ~6 Å (Fig. <a href="#Fig5" class="usa-link">5b</a>), which is longer than typical distances observed for productive methyl transfer. The methylation of A37 proceeds relatively slowly (<em>k</em><sub>cat</sub> = 0.38 min<sup>−1</sup>, Fig. <a href="#Fig1" class="usa-link">1c</a>), and the cryo-EM map at the position of A37 shows no apparent methylation of 6-NH<sub>2</sub> group of the adenine base (Supplementary Fig. <a href="#MOESM1" class="usa-link">7c</a>). Additionally, we determined the cryo-EM structure of the spMETTL16 in complex with spU6_Δtelestem and SAM at ~2.9 Å (Supplementary Fig. <a href="#MOESM1" class="usa-link">8a, b</a>). The sample was applied to the grid immediately after mixing spMETTL16, spU6_Δtelestem, and SAM on ice. Structural analysis revealed that RNA recognition in this complex is consistent with that observed in the spMETTL16–SAM–spU6 complex (Supplementary Fig. <a href="#MOESM1" class="usa-link">8c</a>). The distance between the 6-NH<sub>2</sub>, of A37 in U6 snRNA and the methyl group of SAM remains ~6 Å, and the cryo-EM map at A37 shows no apparent methylation at the N6 position (Supplementary Fig. <a href="#MOESM1" class="usa-link">8b</a>). These observations suggest that, in the present structure, the 6-NH<sub>2</sub> of A37 is not positioned for nucleophilic attack on the methyl group of SAM<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, and that the present structure represents a pre-productive state of the reaction, as described below.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Interactions between spMETTL16 and spU6 snRNA in the spMETTL16–SAM–spU6 complex.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371086_41467_2025_63021_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f31c/12371086/40fa89faaeee/41467_2025_63021_Fig5_HTML.jpg" loading="lazy" id="d33e1141" height="934" width="797" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Overall structure of spMETTL16–SAM–spU6 (left) and nucleotide sequence of spU6 snRNA. The modeled nucleotides (35–83) in the structure are colored orange. <strong>b</strong> SAM recognition by spMETTL16. The structure of human METTL16 MTD in complex with SAH (hsMETTL16-SAH)<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup> was superimposed onto that of spMETTL16–SAM–spU6. SAH in the hsMETTL16-SAH is shown as yellow sticks and SAM in the spMETTL16–SAM–spU6 is shown as cyan sticks. <strong>c</strong>, <strong>d</strong> Interactions between spMETTL16 and spU6 snRNA. <strong>e</strong> Schematic representation of the interaction between A35C36A37G38A39 and the MTD in (<strong>c</strong>, <strong>d</strong>). <strong>f</strong> In vitro methylation of the spU6 snRNA transcript by spMETTL16 and its variants with mutations in the KA-1 domain under standard conditions. spU6 snRNA (0.5 μM) was incubated with 200 nM spMETTL16 or its variants in the presence of 1 mM SAM for 4 min at 37 °C. Error bars represent the standard deviation (SD) of three independent experiments (<em>N</em> = 3), and the center of the error bands indicates the mean of the measured values. Source data are provided as a <a href="#MOESM4" class="usa-link">Source data</a> file.</p></figcaption></figure><p id="Par28">The C36 base stacks with the side chain of Met197. The N1 atom and the 6-NH<sub>2</sub> group of A37 form hydrogen bonds with the Nδ1 atom and the Oδ1 atom of Asn168, respectively, in the NPPF motif (residues 168–171). The A37 base is sandwiched between Phe171 and the side chain of Arg260, which interacts with the phosphate of G38. The O6 of G38 forms a hydrogen bond with Ser231. The 6-NH<sub>2</sub> of A39 interacts with Lys230, which in turn forms a hydrogen bond with Ser283 (Fig. <a href="#Fig5" class="usa-link">5c–e</a>). Lys257 forms a hydrogen bond with the phosphate of A39. These structural features suggest that part of the ACA*GAGA motif in spU6 snRNA is recognized by the MTD of spMETTL16 (Fig. <a href="#Fig5" class="usa-link">5e</a>), although the 6-NH<sub>2</sub> of A37 remains distant from SAM. Arg278 forms hydrogen bonds with the phosphates of A72 and C73, and Lys286 interacts with the phosphates of U45 and G75. Thr282 interacts with the phosphate of U46 (Fig. <a href="#Fig5" class="usa-link">5d</a>). Consistent with the RNA recognition by the MTD in the present structure, the N168A, F171A, K230E, K257E, R260E, and R278E mutations of spMETTL16 all reduced the methylation activity of spMETTL16 in vitro (Fig. <a href="#Fig5" class="usa-link">5f</a>).</p>
<p id="Par29">Altogether, the present spMETTL16–SAM–spU6 structure represents a stage in which the core of the ACA*GAGA motif of spU6 snRNA is specifically recognized by the MTD, but spMETTL16 is not yet ready for catalysis. spU6 snRNA has not adopted a productive conformation for efficient methylation by spMETTL16, and thus, the present structure would be the pre-productive stage, as described below.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Structural elements in spU6 snRNA for efficient methylation by spMETTL16</h3>
<p id="Par30">The crystal structure of the methyltransferase domain (MTD) of hsMETTL16 in complex with the short hairpin RNA of the 3’-UTR of MAT2A mRNA (MAT2A-hp) has been reported<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. In the structure, MAT2A-hp adopts a distinct structural conformation, including a loop (containing the UACAG recognition motif and linker), a stem and a transition region between the loop and the stem, all of which are essential for efficient catalysis by the MTD<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup> (Fig. <a href="#Fig6" class="usa-link">6a</a>). It has been suggested that the human U6 snRNA could also adopt a similar structural conformation, with the UACAG recognition motif positioned in the same context as in the MAT2A-hp structure<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Likewise, spU6 snRNA is predicted to adopt the same structure as MAT2A-hp and human U6 snRNA (Fig. <a href="#Fig6" class="usa-link">6a</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. spU6 snRNA methylation requires productive conformation of spU6 snRNA.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371086_41467_2025_63021_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f31c/12371086/7c3868701bb3/41467_2025_63021_Fig6_HTML.jpg" loading="lazy" id="d33e1244" height="875" width="799" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Nucleotide sequences of human MAT2A hairpin (MAT2A-hp)<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup> and <em>S. pombe</em> U6 snRNA and its variants used for the assays in (<strong>b</strong>). <strong>b</strong> In vitro methylation of spU6 snRNA and its variants by spMETTL16 under standard conditions. spU6 snRNA or its variants (0.5 μM) was incubated with 200 nM spMETTL16 in the presence of 1 mM SAM for 4 min at 37 °C. Error bars represent the standard deviation (SD) of three independent experiments (<em>N</em> = 3), and the center of the error bands indicates the mean of the measured values. <strong>c</strong> Superimposition of the spMETTL16–SAM–spU6 structure onto the human METTL16 MTD (hsMETTL16_MTD, gray) in complex with MAT2A hairpin RNA (MAT2A-hp: gray, PDB ID: 6DU4)<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. The regions corresponding to the red regions (N-terminal extension and RNA binding loop), enclosed by dashed lines, in hsMETTL16_MTD were not visible in the present spMETTL16–SAM–spU6 structure. <strong>d</strong> Superimposition of the structure of the catalytic pocket of spMETTL16–SAM–spU6 and that of hsMETTL16_MTD–MAT2A-hp (gray). A17 in MAT2A-hp in the hsMETTL16–MAT2A-hp complex is deeply docked in the pocket and closer to SAM (cyan) than A37 in spU6 snRNA in the spMETTL16–SAM–spU6 complex. <strong>e</strong> In vitro methylation of the spU6 snRNA transcript by spMETTL16 and its N-terminal extension deletion mutant (Δ_NTE: amino acids 1–50 were deleted) and RNA binding loop deletion mutant (Δ_RBL: amino acid 186–189 were deleted), under standard conditions as in Fig. <a href="#Fig5" class="usa-link">5e</a>. Error bars represent the standard deviation (SD) of three independent experiments (<em>N</em> = 3), and the center of the error bands indicates the mean of the measured values. Source data for <strong>b</strong>, <strong>e</strong> are provided as a <a href="#MOESM4" class="usa-link">Source data</a> file.</p></figcaption></figure><p id="Par31">To verify the requirement of these structural elements in spU6 snRNA for methylation by spMETTL16, as observed with hsMETTL16, we tested the methylation of mutant spU6 snRNAs by spMETTL16 in vitro (Fig. <a href="#Fig6" class="usa-link">6b</a>). Mutations in the UACAG recognition motif in the loop, such as C36G and G38C, reduce its methylation by spMETTL16 in vitro (Fig. <a href="#Fig6" class="usa-link">6a, b</a>). Mutations in the putative transition region of spU6 snRNA – G40A41A42 to C40U41U42 or G83A84 to C83U84 – also reduce its methylation by spMETTL16 in vitro (Fig. <a href="#Fig6" class="usa-link">6a, b</a>). These results indicate that the structural elements surrounding the methylation site in spU6 snRNA are necessary for efficient A37 methylation by spMETTL16, similar to hsMETTL16 and ceMETT10<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Furthermore, spU6 snRNA lacking the 5′-stem and telestem, spanning nucleotides 26 to 84 (Δtelestem), can be methylated by spMETTL16, although with lower efficiency than that of full-length spU6 snRNA, suggesting that Δtelestem can adopt the same structural conformation as MAT2A-hp. Furthermore, an hsMAT2A-hp-like U6 snRNA variant (Δtelestem_Loop) is also methylated by spMETTL16 to the same extent as Δtelestem (Fig. <a href="#Fig6" class="usa-link">6a, b</a>). These results indicate that spU6 snRNA must adopt a productive conformation for efficient methylation by spMETTL16, as required for MAT2A-hp methylation by hsMETTL16.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Transition of spU6 sRNA to the productive form for efficient methylation by spMETTL16</h3>
<p id="Par32">Superimposition of the structure of the MTD of hsMETTL16 in complex with MAT2A-hp (hsMTD–MAT2A-hp, PDB ID: 6DU4)<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup> onto that of spMETTL16–SAM–spU6 reveals that, while the N-terminal extension and the RNA-binding loop in the MTD of hsMETTL16 interact with MAT2A-hp from various angles, the corresponding regions of spMETTL16 are not visible in the current spMETTL16–SAM–spU6 structure (Figs. <a href="#Fig4" class="usa-link">4</a>d and <a href="#Fig6" class="usa-link">6c</a>). This suggests that the N-terminal extension and the RNA-binding loop of spMETTL16 would facilitate the transition of spU6 snRNA into a productive structure for methylation by spMETTL16.</p>
<p id="Par33">Superimposition of the catalytic core of hsMTD–MAT2A-hp onto that of spMETTL16–SAM–spU6 (Fig. <a href="#Fig6" class="usa-link">6c, d</a>) reveals that A17 in MAT2A-hp is positioned closer to SAM than A37 in spMETTL16–SAM–spU6 (Fig. <a href="#Fig6" class="usa-link">6d</a>). In the hsMTD–MAT2A-hp structure, the 6-NH<sub>2</sub> and N1 atoms of A17 of MAT2A-hp form hydrogen bonds with the main-chain carbonyl and amide of Pro185 and Pro186, respectively<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. These observations suggest that the reported hsMTD–MAT2A-hp structure represents the productive stage, and the current structure of spMETTL16–SAM–spU6 (Fig. <a href="#Fig4" class="usa-link">4b</a>) likely represents a pre-productive stage, in which the ACA*GAGA motif in U6 snRNA is recognized by spMETTL16 (Fig. <a href="#Fig5" class="usa-link">5c</a>), but the spU6 snRNA has not yet transitioned into a catalytically active, productive conformation for methylation by spMETTL16.</p>
<p id="Par34">Subsequently, this pre-productive conformation undergoes further structural adjustment through interactions involving the N-terminal extension, the RNA-binding loop, and U6 snRNA. Consistently, the deletion of the N-terminal extension and the RNA-binding loop in spMETTL16 reduced its methylation activity (Fig. <a href="#Fig6" class="usa-link">6e</a>). The establishment of the interactions between A39G40A41G42 and G83A84, forming the transition base-pair, along with the base pairings in the telestem, enables spU6 snRNA to adopt the productive conformation. In this state, the ACA*GAGA motif protrudes deeper into the catalytic site, positioning the A37 base closer to SAM and thereby facilitating efficient m<sup>6</sup>A37 modification.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par35">The m<sup>6</sup>A methylation of a conserved adenosine in U6 snRNA, catalyzed by METTL16 or its homologs, is essential for the efficient splicing of specific pre-mRNAs<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>–<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. In this study, we elucidated the molecular mechanisms governing the interactions between <em>S. pombe</em> METTL16 (spMETTL16) and U6 snRNA, as well as the m<sup>6</sup>A methylation. The conserved C-terminal KA-1 domain, present across METTL16 homologs in vertebrates, invertebrates, and fission yeast, is a critical element that enhances methylation efficiency and ensures proper pre-mRNA splicing (Fig. <a href="#Fig1" class="usa-link">1</a>). The KA-1 domain, particularly its arginine-rich region (RRR), interacts specifically with the internal stem-loop (ISL) of U6 snRNA, recruiting the substrate U6 snRNA to the enzyme (Figs. <a href="#Fig2" class="usa-link">2</a> and <a href="#Fig3" class="usa-link">3</a>), and serves as an axis in the subsequent methylation process.</p>
<p id="Par36">Cryo-EM structures of the spMETTL16–spU6 snRNA complex, with and without SAM, reveal the dynamic processes of substrate recognition, methylation site recognition, and the methylation reaction (Figs. <a href="#Fig4" class="usa-link">4</a>, <a href="#Fig5" class="usa-link">5</a>, and <a href="#Fig7" class="usa-link">7</a>). Previous biochemical studies showed that hsMETTL16 binds to U6 snRNA with much greater affinity than SAM<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. As for the spMETTL16, the <em>K</em><sub>m</sub> values for U6 snRNA and SAM are 0.182 μM and 482 μM, respectively, differing by three orders of magnitude (Fig. <a href="#Fig1" class="usa-link">1c</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">7d</a>). This substantial difference indicates that spMETTL16 has a significantly higher affinity for U6 snRNA compared to SAM.</p>
<figure class="fig xbox font-sm" id="Fig7"><h3 class="obj_head">Fig. 7. Mechanism of U6 snRNA methylation by METTL16.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371086_41467_2025_63021_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f31c/12371086/9831a1798ef9/41467_2025_63021_Fig7_HTML.jpg" loading="lazy" id="d33e1386" height="313" width="798" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Initial binding stage: U6 snRNA initially binds to METTL16 through interactions between the ISL and KA-1 domain (blue). The ACA*GAGA motif, which contains the methylation site A37 in U6 snRNA, is highlighted in red. <strong>b</strong> Pre-productive stage: SAM (cyan) binding to the MTD (magenta) triggers a conformational change in the MTD, shifting the ACA*GAGA motif of U6 snRNA closer to the catalytic site via the RNA-binding loop. At this stage, the motif is specifically recognized. <strong>c</strong> Productive stage: The interactions of the N-terminal extension (NTE) and the RNA-binding loop with U6 snRNA promote the transition to the productive stage. U6 snRNA undergoes structural rearrangements, adopting productive conformation. This process leads to the formation of the transition region and base-pairing within the telestem in U6 snRNA, shifting A37 deeper into the catalytic site, and thereby facilitating efficient m<sup>6</sup>A modification by MTD.</p></figcaption></figure><p id="Par37">Thus, the m<sup>6</sup>A modification of U6 snRNA occurs by the ordered-sequential mechanism<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. First, U6 RNA is recruited to METTL16 through the specific interactions between KA-1 and the ISL. U6 snRNA adopts an extended structure, and the ACA*GAGA motif, containing the methylation site A37, is apart from the catalytic site in the methyltransferase domain (MTD), representing the initial U6 snRNA binding stage (Fig. <a href="#Fig7" class="usa-link">7a</a>). Next, the SAM binds to the catalytic site in the MTD of the METTL16–U6 snRNA complex. The catalytic pocket and part of the RNA-binding loop in the MTD are resolved, allowing the ACA*GAGA motif in U6 snRNA to shift closer to the catalytic pocket of the MTD. At this stage, the motif is specifically recognized by the MTD, although the target A37 remains ~6 Å from SAM— indicating a pre-productive state insufficient for methyl transfer (Figs. <a href="#Fig4" class="usa-link">4</a>, <a href="#Fig5" class="usa-link">5</a>, and <a href="#Fig5" class="usa-link">7b</a>). A comparison with the hsMETTL16–MAT2A-hp structure, in which MAT2A-hp RNA adopts a productive conformation for m<sup>6</sup>A modification<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>, highlights the need for further adjustment of the U6 snRNA conformation into a productive state for A37 methylation by the MTD (Figs. <a href="#Fig6" class="usa-link">6</a> and <a href="#Fig7" class="usa-link">7c</a>). Finally, additional intramolecular interactions—the formation of the transition region and the stem in the telestem of U6 snRNA—through the interactions of U6 snRNA with the N-terminal extension and RNA-binding loops in the MTD, reorient the ACA*GAGA motif deeper into the catalytic site, aligning the target A37 with SAM for efficient methylation (Figs. <a href="#Fig6" class="usa-link">6</a> and <a href="#Fig7" class="usa-link">7c</a>).</p>
<p id="Par38">The KA-1 domain not only anchors U6 snRNA to METTL16 but also serves as a hub that orchestrates the transition of the METTL16-U6 snRNA complex into a catalytically competent state. This multi-step process, which includes initial ISL recognition, motif recognition, and final positioning of the methylation site A37 within the catalytic site, reflects the coordinated roles of the METTL16 domains (Fig. <a href="#Fig7" class="usa-link">7</a>). The productive conformation, consisting of the transition region and the stem in U6 snRNA (Fig. <a href="#Fig7" class="usa-link">7c</a>), might be transient and inherently unstable. Due to its fleeting nature, we may have been unable to capture it in our cryo-EM analysis.</p>
<p id="Par39">In vertebrates, METTL16 regulates SAM homeostasis by controlling MAT2A mRNA splicing and stability through the m<sup>6</sup>A modification of the 3’-UTR hairpins of MAT2A mRNA<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. The effect of KA-1 on the methylation of MAT2A hairpins was observed in vitro, although its impact is much less significant than its role in the methylation of U6 snRNA<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. MAT2A-hp has a stem-loop structure, and compared to U6 snRNA, its loop can more readily adopt the productive conformation for efficient methylation by METTL16 (Fig. <a href="#Fig6" class="usa-link">6a</a>). Structural adjustment after ACA*GAGA motif recognition would proceed smoothly, with the recognition of MAT2A-hp by METTL16 primarily mediated by the MTD. Therefore, as described below, KA-1 would have been attached to the MTD for the efficient methylation of structured U6 snRNA.</p>
<p id="Par40">The KA-1 domain reportedly promotes splicing when tethered to a reporter<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. The splicing of MAT2A mRNA is collaboratively regulated by METTL16 and the cleavage factor Im (CFIm) complex<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. The KA-1 domain may alter the RNA secondary structure surrounding MAT2A hairpins, particularly by binding to the double-stranded region, thereby facilitating the recruitment of CFIm complexes and promoting MAT2A mRNA splicing. The mechanism and involvement of the KA-1 domain in MAT2A mRNA splicing await further investigation.</p>
<p id="Par41">The KA-1 domain structure is also found in the C-terminal region of the U6 snRNA-specific terminal uridylyltransferase (TUTase, TUT1)<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>–<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>, although the amino acid sequence homology between the KA-1 domains of METTL16 and TUT1 is low<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. The KA-1 domain of human TUT1 (hsTUT1) increases hsTUT1’s affinity for U6 snRNA and facilitates efficient oligo-uridylylation of U6 snRNA<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. A recent structural analysis of hsTUT1 in complex with U6 snRNA showed that the KA-1 domain interacts with the tip of the ISL<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>, where two helices engage distinct regions of the ISL: one containing RRR interacts with the major groove of the loop at the ISL tip, while the other contacts the minor groove of the double-stranded region of the ISL, similar to the interaction of spKA-1 with spISL (Fig. <a href="#Fig2" class="usa-link">2a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>). Notably, these two distinct enzymes, which exert different, yet functionally essential activities on U6 snRNA, share the same structural domains for their activities. Therefore, the KA-1 domain may have become associated with the catalytic domains of these enzymes and evolved to recognize U6 snRNA, particularly the ISL, for the efficient maturation of functional U6 snRNA.</p>
<p id="Par42">The KA-1 domain is widely conserved in METTL16 and its homologs across vertebrates, invertebrates, and even fission yeast. As revealed in this study, the KA-1 domain of spMETTL16 functions as a key regulatory domain for efficient U6 snRNA m<sup>6</sup>A modification and precise pre-mRNA splicing. Therefore, this mechanism is likely to be functionally and evolutionarily conserved among eukaryotes, including humans.</p></section><section id="Sec11"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Plasmid construction</h3>
<p id="Par43">The plasmid encoding the full-length <em>S. pombe</em> Mtl16 gene without untranslated regions (pET28a-SpMettl16) was kindly provided by Dr. Suzuki<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Plasmids encoding spMETTL16 variants were prepared using the pET28a-SpMettl16 plasmid as the template. The DNA fragment encoding the N-terminal methyltransferase domain (MTD: residues 1–281) of SpMettl16 was PCR-amplified and cloned between the NdeI and XhoI sites of pET28a, yielding pET28a_SpMettl16-MTD. The DNA fragment encoding the C-terminal KA-1 domain (residues 270–398) was PCR-amplified and cloned into the pET15SUMO vector<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup> between the NdeI and XhoI sites, yielding pET15SUMO_SpMettl16-KA-1. Mutations were introduced using a KOD-Plus Mutagenesis Kit (Toyobo, Japan), according to the manufacturer’s instructions. The oligonucleotide sequences used for molecular cloning and mutagenesis are listed in Supplementary Table <a href="#MOESM1" class="usa-link">3</a>.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Expression and purification of recombinant proteins</h3>
<p id="Par44">For the overexpression of spMETTL16, its variants, and spMETTL16-MTD (spMTD), as well as that of spMETTL16-KA-1 (spKA-1) and its variants, <em>E. coli</em> Rosetta2(DE3) cells (Novagen, Japan) were transformed with the respective overexpression plasmids. Transformed cells were cultured in LB medium supplemented with 50 μg/mL kanamycin and 40 μg/mL chloramphenicol for the expression of spMETTL16, its variants, and spMTD, or with 50 μg/mL ampicillin and 40 μg/mL chloramphenicol for spKA-1 and its variants. Cultures were grown at 37 °C until the OD<sub>600</sub> reached 0.8, and protein expression was induced by adding isopropyl β-D-thiogalactopyranoside (IPTG) to a final concentration of 0.1 mM, followed by an incubation at 20 °C for 20 h. Cells were harvested and lysed in buffer containing 20 mM Tris-HCl, pH 7.0, 500 mM NaCl, 5 mM β-mercaptoethanol, 20 mM imidazole, and 5% (v/v) glycerol. Lysates were clarified by centrifugation at 30,000 × <em>g</em> for 30 min at 4 °C.</p>
<p id="Par45">SpMETTL16, its variants, and spMTD were first purified by Ni-NTA agarose (Qiagen, Japan) chromatography, with proteins eluted in buffer containing 20 mM Tris-HCl, pH 7.0, 500 mM NaCl, 5 mM β-mercaptoethanol, 300 mM imidazole, and 5% (v/v) glycerol. The eluted proteins were further purified on a HiTrap Heparin column (GE Healthcare, Japan) and separated by size-exclusion chromatography using a HiLoad 16/60 Superdex 200 column (GE Healthcare, Japan), equilibrated with buffer containing 20 mM Tris-HCl, pH 7.0, 200 mM NaCl, and 10 mM β-mercaptoethanol. His6-SUMO-tagged spKA-1 proteins were purified using Ni-NTA agarose chromatography as described above. The His6-SUMO tags were cleaved by ULP1 (ubiquitin-like-specific protease 1) during dialysis against buffer, containing 20 mM Tris-HCl, pH 7.0, 500 mM NaCl, 5 mM β-mercaptoethanol, and 5% (v/v) glycerol, at 4 °C overnight. Following cleavage, the proteins were passed through a Ni-NTA agarose column to remove the uncleaved His6-SUMO-tagged proteins and ULP1. The cleaved proteins were further purified as described above. The final purified proteins were concentrated and stored at −80 °C.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Preparation of synthetic U6 snRNA and its variants</h3>
<p id="Par46">Synthetic <em>S. pombe</em> U6 snRNA internal stem-loop (spISL) RNA was purchased from Fasmac, Japan. The U6 snRNA and its variants were synthesized by T7 RNA polymerase, using plasmids encoding the respective DNA sequences downstream of the T7 promoter as templates. The synthesized RNAs were purified by 10% (w/v) polyacrylamide gel electrophoresis under denaturing conditions. The nucleotide sequences of the templates used for in vitro transcription are listed in Supplementary Table <a href="#MOESM1" class="usa-link">4</a>, and the nucleotide sequences of the RNAs used in this study are listed in Supplementary Table <a href="#MOESM1" class="usa-link">5</a>.</p></section><section id="Sec15"><h3 class="pmc_sec_title">In vitro methylation assay</h3>
<p id="Par47">In vitro methylation assays were conducted using the MTase-Glo system (Promega, Japan), as previously described<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> with slight modifications. The reaction mixture (15 μL), containing 0.2 μM spMETTL16 (or its variants or spMTD), 50 mM HEPES-KOH, pH 8.0, 120 mM NaCl, 2 mM MgCl<sub>2</sub>, 1 mM dithiothreitol (DTT), 1 mM S-(5′-adenosyl)-L-methionine chloride (SAM, Cayman Chemical, USA), 1× MTase-Glo Reagent, and a series of RNA samples at different concentrations (0.05–10.0 μM), was incubated at 37 °C for 4 min. Subsequently, 15 μL of MTase-Glo Detection Solution was added, and the reaction was incubated at room temperature for 30 min. Luminescence was quantified using a GloMax-Multi Detection System (Promega, Japan). The luminescence value of the mixture without the RNA substrate was used as the background and subtracted. One picomole of SAH was estimated to be equivalent to 190,000 LU (light units), according to the standard quantification curve.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Yeast strains and cultivation</h3>
<p id="Par48">The ED668 (<em>h+</em> ade6-M216 ura4-D18 leu1−32) (wild-type) strain and SPAC27D7.08cΔ (Δ<em>mtl</em>16), mettl16 knockout strain<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> were grown at 30 °C in YE5S medium [0.5% yeast extract and 3% glucose, supplemented with adenine, uracil, lysine, histidine, and leucine (225 mg/mL each)] or EMM medium [EMM without dextrose (MP Biochemicals) with 2% glucose, supplemented with adenine and leucine (225 mg/mL each)]. Yeast cells were cultivated at 30 °C to mid-exponential phase for total RNA extraction.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Total RNA preparation</h3>
<p id="Par49">Total RNA from yeast cells was prepared by hot phenol extraction<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. Cell pellets were resuspended in TES solution (10 mM Tris-HCl, pH 7.5, 10 mM EDTA, and 0.5% SDS), mixed with an equal volume of acidic phenol with brief vortexing, incubated at 65 °C for 30 min, and centrifuged at 14,000 × <em>g</em> for 5 min at 4 °C. The aqueous phase was collected, mixed with an equal volume of chloroform, and centrifuged at 14,000 × g for 5 min at 4 °C. The supernatant was subjected to ethanol precipitation to obtain the RNA pellet. The RNA pellet was dissolved in RNase-free water and stored at −80 °C.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Semi-quantification of intron-retention of specific pre-mRNAs</h3>
<p id="Par50">The plasmid encoding the full-length <em>S. pombe mtl</em>16 gene with untranslated regions (pREP42-spMETTL16-FL) and the empty control vector (pREP42-vector) were kindly provided by Dr. Suzuki<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Plasmids expressing mutant spMETTL16 were generated by site-directed mutagenesis. The <em>S. pombe</em> Δ<em>mtl16</em> strain was transformed with these plasmids using a Frozen-EZ Yeast Transformation II Kit (Zymo Research, Funakoshi, Japan) and grown on YE5S or EMM medium plates. After an incubation for 3–4 days, single colonies were selected and cultured in EMM medium without thiamine until the cell density reached an OD<sub>600</sub> of 1.0. Cells were harvested, and total RNA was extracted as described above. One microgram of total RNA was treated with Recombinant RNase-free DNase I (Takara, Japan) and subjected to reverse transcription using random hexamer and oligo(dT) primers with a PrimeScript™ RT Reagent Kit (Takara, Japan). The resulting cDNA was amplified by PCR with specific primers (<em>SPAC18B11.09c</em> and <em>ckn1</em> pre-mRNA)<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> under the following conditions: 30 cycles of 98 °C for 10 s, 53 °C for 10 s, and 72 °C for 20 s. PCR products were resolved on a 10% (w/v) non-denaturing polyacrylamide gel, stained with ethidium bromide, and visualized and quantified using a Gel Doc EZ Imager (Bio-Rad, Japan).</p></section><section id="Sec19"><h3 class="pmc_sec_title">Crystallization and structural determination</h3>
<p id="Par51">The purified spKA-1 was mixed with spISL RNA at a 1:1 molar ratio (final concentration of 50 μM each), and the solution was incubated on ice for 10 min. Crystals were obtained using the sitting-drop vapor diffusion method at 20 °C. A 0.2 μL sample of the spKA-1–spISL complex was mixed with an equal volume of reservoir solution, containing 0.36 M sodium acetate, 0.09 M sodium cacodylate, pH 7.0, 0.01 M Tris-HCl, pH 8.5, and 23% (v/v) PEG 8000. The crystals were cryoprotected with reservoir solution supplemented with 25% (v/v) ethylene glycol and flash-cooled in liquid nitrogen. X-ray diffraction data were collected at Beamline 17 A of the Photon Factory (KEK, Japan) and processed using XDS<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. The initial phase of the spKA-1:spISL complex structure was determined by molecular replacement with Phaser<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>, using the AlphaFold-predicted structure of the spKA-1 domain (AF-<a href="https://www.ncbi.nlm.nih.gov/protein/O42662" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">O42662</a>-F1)<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> and the spISL RNA model predicted with SimRNA<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> as search models. The structure was manually modeled using Coot<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup> and further refined with Phenix.refine<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. The final model was assessed with MolProbity<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. The details of the crystallographic data collection and refinement statistics are provided in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Cryo-EM grid preparation</h3>
<p id="Par52">For the spMETTL16–SAM–spU6 snRNA complex, purified spMETTL16 (0.6 mg/mL) was mixed with the U6 snRNA transcript at a 1:1 molar ratio in buffer C (20 mM Tris-HCl, pH 7.0, 200 mM NaCl, 10 mM β-mercaptoethanol, and 2 mM SAM) and incubated at 37 °C for 10 min. Tween-20 [0.005% (v/v)] was then added, and the mixture was placed on ice prior to grid preparation. A 3 μL aliquot of the sample was applied to a glow-discharged UltrAuFoil R1.2/1.3 Au 300 grid (Quantifoil) at ~4 °C. The grids were blotted for 4 s at 100% humidity and 4 °C, and then flash-frozen in liquid ethane using a Vitrobot Mark IV (Thermo Fisher Scientific). The spMETTL16–SAM–spU6_Δtelestem complex was also prepared as described above, except that the sample was applied to the grid immediately after mixing spMETTL16, SAM, and spU6_Δtelestem on ice. The spMETTL16–spU6 complex was prepared as described above, except that the sample was applied to a Quantifoil R1.2/1.3 mesh Cu300 holey carbon grid (Quantifoil), with a sample concentration of 0.4 mg/mL.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Cryo-EM data collection and processing</h3>
<p id="Par53">All datasets were processed using a similar protocol with minor variations. Data collection parameters are summarized in Supplementary Table <a href="#MOESM1" class="usa-link">2</a>, and data processing flowcharts are shown in Supplementary Figs. <a href="#MOESM1" class="usa-link">4</a> and <a href="#MOESM1" class="usa-link">6</a>. The processing of the spMETTL16–SAM–spU6 snRNA complex is described as follows. Cryo-EM movies were collected using a Krios G4 transmission electron microscope (Thermo Fisher Scientific) equipped with a K3 direct electron detector (Gatan, Inc.) operated at 300 kV. A total of 8497 micrographs were recorded at a pixel size of 0.83 Å, with a total electron dose of 50 e<sup>−</sup>/Å<sup>2</sup> Downstream processing was performed in cryoSPARC (ver. 4.6.0)<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Movies were motion-corrected using Patch Motion Correction, and contrast transfer function (CTF) estimation was performed with Patch CTF. Micrographs were denoised using Micrograph Denoiser, and particles were automatically picked with Blob Picker. Two rounds of two-dimensional (2D) classification were conducted, and the selected high-quality classes were used as templates for template-based particle picking. After three rounds of heterogeneous refinement and one additional 2D classification step to remove junk particles, 486,386 particles were selected and extracted with a 324-pixel box. The final reconstruction from 425,927 particles was obtained using non-uniform refinement, yielding an EM map at an average resolution of 2.99 Å after CTF refinement, particle subtraction, and reference-based motion correction polishing. The final map was sharpened using DeepEMhancer (ver. 0.14)<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>. The data processing for the spMETTL16–spU6 complex was nearly identical to that of the spMETTL16–SAM–spU6 complex, as described above. The final reconstruction was obtained using non-uniform refinement, yielding an EM map at an average resolution of 3.4 Å from 173,139 particles (Supplementary Figs. <a href="#MOESM1" class="usa-link">4</a> and <a href="#MOESM1" class="usa-link">6</a> and Supplementary Table <a href="#MOESM1" class="usa-link">2</a>).</p></section><section id="Sec22"><h3 class="pmc_sec_title">Model building and refinement</h3>
<p id="Par54">To build the model of the spMETTL16–SAM–spU6 structure, the structures of the spMETTL16-MTD domain (residues 56–291) and RNA base pairs (G43-U82, A44-U81, U45-A80, and U46-C79) were predicted using AlphaFold2<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. The spMETTL16-spKA-1 domain (residues 292–398) and U6 RNA nucleotides (47–77) were derived from the spKA-1-spISL structure. These models were docked into the cryo-EM density using ChimeraX (ver. 1.9)<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. Additional RNA nucleotides (A35 to A42 and G83) and SAM were manually fitted, and the model was constructed and adjusted using Coot<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. The structure was refined using phenix.real_space_refine<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. Due to insufficient density for de novo model building, restraints were applied to maintain the correct geometry of the base pairs G43-U82, A44-U81, U45-A80, and U46-C79 during model refinement. The final structures were validated using MolProbity<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>, and figures were prepared using PyMOL (<a href="http://www.pymol.org/pymol" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.pymol.org/pymol</a>) and UCSF ChimeraX (ver. 1.9)<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. The model building process for the spMETTL16–spU6 structure was nearly identical to that for the spMETTL16–SAM–spU6 structure, as described above (Supplementary Table <a href="#MOESM1" class="usa-link">2</a>).</p></section><section id="Sec23"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par55">Further information on research design is available in the <a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec24"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371086/bin/41467_2025_63021_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (3.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12371086/bin/41467_2025_63021_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12371086/bin/41467_2025_63021_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (835.9KB, pdf) </sup>
</div></div></section></section><section id="Sec25"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12371086/bin/41467_2025_63021_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (388.2KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Dr. Suzuki for providing the Δ<em>mtl16</em> strains and vectors expressing spMETTL16. This work was supported by Grants-in-Aid for Scientific Research (A) [grant numbers 23H00368 and 18H03980 to K.T.], and a Grant-in-Aid for Scientific Research on Innovative Areas from the Ministry of Education, Culture, Sports, Science, and Technology of Japan [grant number 26113002 to K.T.]. K.T. was also supported by grants from Takeda Science Foundation, Naito Foundation, Uehara Memorial Foundation, the Terumo Foundation for Life Science and Art, the Yasuda Medical Foundation, and the Princess Takamatsu Cancer Research Fund. The cryo-EM data collection was supported by Research Support Project for Life Science and Drug Discovery (Basis for Supporting Innovative Drug Discovery and Life Science Research (BINDS)) from AMED under Grant Number JP23ama121002.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>K.T. planned and designed the research. J.J. performed structural and biochemical analyses. K.T. and J.J. analyzed the data and wrote the paper.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par56"><em>Nature Communications</em> thanks Jinwei Zhang, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The data that support this study are available from the corresponding author upon request. The data generated in this study are provided as Source Data files with this paper. The coordinates and structure factors for the crystal structure of spKA-1–spISL have been deposited in the PDB, under the accession number <a href="https://doi.org/10.2210/pdb9M86/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9M86</a>. The atomic coordinates and cryo-EM maps of spMETTL16–spU6 and spMETTL16–SAM–spU6 have been deposited in the Protein Data Bank (PDB IDs: <a href="https://doi.org/10.2210/pdb9U47/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9U47</a> and <a href="https://doi.org/10.2210/pdb9U48/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9U48</a>), and the Electron Microscopy Data bank (<a href="https://www.ebi.ac.uk/emdb/EMD-63833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-63833</a> and <a href="https://www.ebi.ac.uk/emdb/EMD-63834" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-63834</a>), respectively. The micrographs of spMETTL16–spU6 and spMETTL16–SAM–spU6 have been deposited in the Electron Microscopy Public Image Archive <a href="https://doi.org/10.6019/EMPIAR-12699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMPIAR-12699</a> and <a href="https://doi.org/10.6019/EMPIAR-12700" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMPIAR-12700</a>, respectively. The structures of hsKA-1 (PDB ID: <a href="https://doi.org/10.2210/pdb6M1U/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6M1U</a>), hsMTD-SAH (PDB ID: <a href="https://doi.org/10.2210/pdb6B92/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6B92</a>), and hsMTD–MAT2A-hp (PDB ID: <a href="https://doi.org/10.2210/pdb6DU4/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6DU4</a>) were used for structural comparisons in this study. <a href="#Sec25" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par57">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec26"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63021-0.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Pan, T. N6-methyl-adenosine modification in messenger and long non-coding RNA. <em>Trends Biochem. Sci.</em><strong>38</strong>, 204–209 (2013).
</cite> [<a href="https://doi.org/10.1016/j.tibs.2012.12.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3608796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23337769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pan,%20T.%20N6-methyl-adenosine%20modification%20in%20messenger%20and%20long%20non-coding%20RNA.%20Trends%20Biochem.%20Sci.38,%20204%E2%80%93209%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Dominissini, D. Topology of the human and mouse m6A RNA methylomes revealed by m6A-seq. <em>Nature</em><strong>485</strong>, 201–206 (2012).
</cite> [<a href="https://doi.org/10.1038/nature11112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22575960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dominissini,%20D.%20Topology%20of%20the%20human%20and%20mouse%20m6A%20RNA%20methylomes%20revealed%20by%20m6A-seq.%20Nature485,%20201%E2%80%93206%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Meyer, K. D. et al. Comprehensive analysis of mRNA methylation reveals enrichment in 3’ UTRs and near stop codons. <em>Cell</em><strong>149</strong>, 1635–1646 (2012).
</cite> [<a href="https://doi.org/10.1016/j.cell.2012.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3383396/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22608085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meyer,%20K.%20D.%20et%20al.%20Comprehensive%20analysis%20of%20mRNA%20methylation%20reveals%20enrichment%20in%203%E2%80%99%20UTRs%20and%20near%20stop%20codons.%20Cell149,%201635%E2%80%931646%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Yue, Y., Liu, J. &amp; He, C. RNA N6-methyladenosine methylation in post-transcriptional gene expression regulation. <em>Genes Dev.</em><strong>29</strong>, 1343–1355 (2015).
</cite> [<a href="https://doi.org/10.1101/gad.262766.115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4511210/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26159994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yue,%20Y.,%20Liu,%20J.%20&amp;%20He,%20C.%20RNA%20N6-methyladenosine%20methylation%20in%20post-transcriptional%20gene%20expression%20regulation.%20Genes%20Dev.29,%201343%E2%80%931355%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Wang, X. et al. N6-methyladenosine-dependent regulation of messenger RNA stability. <em>Nature</em><strong>505</strong>, 117–120 (2014).
</cite> [<a href="https://doi.org/10.1038/nature12730" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3877715/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24284625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20X.%20et%20al.%20N6-methyladenosine-dependent%20regulation%20of%20messenger%20RNA%20stability.%20Nature505,%20117%E2%80%93120%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Xiao, W. et al. Nuclear m6A reader YTHDC1 regulates mRNA splicing. <em>Mol. Cell</em><strong>61</strong>, 507–519 (2016).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2016.01.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26876937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xiao,%20W.%20et%20al.%20Nuclear%20m6A%20reader%20YTHDC1%20regulates%20mRNA%20splicing.%20Mol.%20Cell61,%20507%E2%80%93519%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Shi, H. et al. YTHDF3 facilitates translation and decay of N6-methyladenosine-modified RNA. <em>Cell Res.</em><strong>27</strong>, 315–328 (2017).
</cite> [<a href="https://doi.org/10.1038/cr.2017.15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5339834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28106072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shi,%20H.%20et%20al.%20YTHDF3%20facilitates%20translation%20and%20decay%20of%20N6-methyladenosine-modified%20RNA.%20Cell%20Res.27,%20315%E2%80%93328%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Meyer, K. D. 5′ UTR m6A promotes cap-independent translation. <em>Cell</em><strong>163</strong>, 999–1010 (2015).
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.10.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4695625/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26593424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meyer,%20K.%20D.%205%E2%80%B2%20UTR%20m6A%20promotes%20cap-independent%20translation.%20Cell163,%20999%E2%80%931010%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Wang, X. et al. N6-methyladenosine modulates messenger rna translation efficiency. <em>Cell</em><strong>161</strong>, 1388–1399 (2015).
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.05.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4825696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26046440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20X.%20et%20al.%20N6-methyladenosine%20modulates%20messenger%20rna%20translation%20efficiency.%20Cell161,%201388%E2%80%931399%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Linder, B. et al. Single-nucleotide-resolution mapping of m6A and m6Am throughout the transcriptome. <em>Nat. Methods</em><strong>12</strong>, 767–772 (2015).
</cite> [<a href="https://doi.org/10.1038/nmeth.3453" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4487409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26121403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Linder,%20B.%20et%20al.%20Single-nucleotide-resolution%20mapping%20of%20m6A%20and%20m6Am%20throughout%20the%20transcriptome.%20Nat.%20Methods12,%20767%E2%80%93772%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Liu, J. A METTL3–METTL14 complex mediates mammalian nuclear RNA N6-adenosine methylation. <em>Nat. Chem. Biol.</em><strong>10</strong>, 93–95 (2014).
</cite> [<a href="https://doi.org/10.1038/nchembio.1432" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3911877/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24316715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20J.%20A%20METTL3%E2%80%93METTL14%20complex%20mediates%20mammalian%20nuclear%20RNA%20N6-adenosine%20methylation.%20Nat.%20Chem.%20Biol.10,%2093%E2%80%9395%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Ke, S. et al. A majority of m6A residues are in the last exons, allowing the potential for 3’ UTR regulation. <em>Genes Dev.</em><strong>29</strong>, 2037–2053 (2015).
</cite> [<a href="https://doi.org/10.1101/gad.269415.115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4604345/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26404942/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ke,%20S.%20et%20al.%20A%20majority%20of%20m6A%20residues%20are%20in%20the%20last%20exons,%20allowing%20the%20potential%20for%203%E2%80%99%20UTR%20regulation.%20Genes%20Dev.29,%202037%E2%80%932053%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Oerum, S., Meynier, V., Catala, M. &amp; Tisné, C. A comprehensive review of m6A/m6Am RNA methyltransferase structures. <em>Nucleic Acids Res.</em><strong>49</strong>, 7239–7255 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkab378" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8287941/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34023900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oerum,%20S.,%20Meynier,%20V.,%20Catala,%20M.%20&amp;%20Tisn%C3%A9,%20C.%20A%20comprehensive%20review%20of%20m6A/m6Am%20RNA%20methyltransferase%20structures.%20Nucleic%20Acids%20Res.49,%207239%E2%80%937255%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Wang, P., Doxtader, K. A. &amp; Nam, Y. Structural basis for cooperative function of Mettl3 and Mettl14 methyltransferases. <em>Mol. Cell</em><strong>63</strong>, 306–317 (2016).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2016.05.041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4958592/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27373337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20P.,%20Doxtader,%20K.%20A.%20&amp;%20Nam,%20Y.%20Structural%20basis%20for%20cooperative%20function%20of%20Mettl3%20and%20Mettl14%20methyltransferases.%20Mol.%20Cell63,%20306%E2%80%93317%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Wang, X. et al. Structural basis of N6-adenosine methylation by the METTL3–METTL14 complex. <em>Nature</em><strong>534</strong>, 575–578 (2016).
</cite> [<a href="https://doi.org/10.1038/nature18298" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27281194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20X.%20et%20al.%20Structural%20basis%20of%20N6-adenosine%20methylation%20by%20the%20METTL3%E2%80%93METTL14%20complex.%20Nature534,%20575%E2%80%93578%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Sledz, P. &amp; Jinek, M. Structural insights into the molecular mechanism of the m(6)A writer complex. <em>Elife</em><strong>5</strong>, e18434 (2016).
</cite> [<a href="https://doi.org/10.7554/eLife.18434" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5023411/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27627798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sledz,%20P.%20&amp;%20Jinek,%20M.%20Structural%20insights%20into%20the%20molecular%20mechanism%20of%20the%20m(6)A%20writer%20complex.%20Elife5,%20e18434%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Shima, H. et al. S-adenosylmethionine synthesis is regulated by selective N(6)-adenosine methylation and mRNA degradation involving METTL16 and YTHDC1. <em>Cell Rep.</em><strong>21</strong>, 3354–3363 (2017).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2017.11.092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29262316/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shima,%20H.%20et%20al.%20S-adenosylmethionine%20synthesis%20is%20regulated%20by%20selective%20N(6)-adenosine%20methylation%20and%20mRNA%20degradation%20involving%20METTL16%20and%20YTHDC1.%20Cell%20Rep.21,%203354%E2%80%933363%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Pendleton, K. E. et al. The U6 snRNA m(6)A methyltransferase METTL16 regulates SAM synthetase intron retention. <em>Cell</em><strong>169</strong>, 824–835.e14 (2017).
</cite> [<a href="https://doi.org/10.1016/j.cell.2017.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5502809/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28525753/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pendleton,%20K.%20E.%20et%20al.%20The%20U6%20snRNA%20m(6)A%20methyltransferase%20METTL16%20regulates%20SAM%20synthetase%20intron%20retention.%20Cell169,%20824%E2%80%93835.e14%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Warda, A. S. et al. Human METTL16 is a N6-methyladenosine (m6A) methyltransferase that targets pre-mRNAs and various non-coding RNAs. <em>EMBO Rep.</em><strong>18</strong>, 2004–2014 (2017).
</cite> [<a href="https://doi.org/10.15252/embr.201744940" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5666602/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29051200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Warda,%20A.%20S.%20et%20al.%20Human%20METTL16%20is%20a%20N6-methyladenosine%20(m6A)%20methyltransferase%20that%20targets%20pre-mRNAs%20and%20various%20non-coding%20RNAs.%20EMBO%20Rep.18,%202004%E2%80%932014%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Su, R. et al. METTL16 exerts an m6A-independent function to facilitate translation and tumorigenesis. <em>Nat. Cell Biol.</em><strong>24</strong>, 205–216 (2022).
</cite> [<a href="https://doi.org/10.1038/s41556-021-00835-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9070413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35145225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Su,%20R.%20et%20al.%20METTL16%20exerts%20an%20m6A-independent%20function%20to%20facilitate%20translation%20and%20tumorigenesis.%20Nat.%20Cell%20Biol.24,%20205%E2%80%93216%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Yoshinaga, M. et al. The N6-methyladenosine methyltransferase METTL16 enables erythropoiesis through safeguarding genome integrity. <em>Nat. Commun.</em><strong>13</strong>, 6435 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-34078-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9616860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36307435/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoshinaga,%20M.%20et%20al.%20The%20N6-methyladenosine%20methyltransferase%20METTL16%20enables%20erythropoiesis%20through%20safeguarding%20genome%20integrity.%20Nat.%20Commun.13,%206435%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Mendel, M. et al. Methylation of structured RNA by the m(6)A writer METTL16 is essential for mouse embryonic development. <em>Mol. Cell</em><strong>71</strong>, 986–1000.e11 (2018).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2018.08.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6162343/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30197299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mendel,%20M.%20et%20al.%20Methylation%20of%20structured%20RNA%20by%20the%20m(6)A%20writer%20METTL16%20is%20essential%20for%20mouse%20embryonic%20development.%20Mol.%20Cell71,%20986%E2%80%931000.e11%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Scarborough, A. M. et al. SAM homeostasis is regulated by CFIm-mediated splicing of MAT2A. <em>eLife</em><strong>10</strong>, e64930 (2021).
</cite> [<a href="https://doi.org/10.7554/eLife.64930" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8139829/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33949310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scarborough,%20A.%20M.%20et%20al.%20SAM%20homeostasis%20is%20regulated%20by%20CFIm-mediated%20splicing%20of%20MAT2A.%20eLife10,%20e64930%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Mendel, M. et al. Splice site m(6)A methylation prevents binding of U2AF35 to inhibit RNA splicing. <em>Cell</em><strong>184</strong>, 3125–3142.e25 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.03.062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8208822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33930289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mendel,%20M.%20et%20al.%20Splice%20site%20m(6)A%20methylation%20prevents%20binding%20of%20U2AF35%20to%20inhibit%20RNA%20splicing.%20Cell184,%203125%E2%80%933142.e25%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Watabe, E. et al. m(6) A-mediated alternative splicing coupled with nonsense-mediated mRNA decay regulates SAM synthetase homeostasis. <em>EMBO J.</em><strong>40</strong>, e106434 (2021).
</cite> [<a href="https://doi.org/10.15252/embj.2020106434" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8280822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34152017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Watabe,%20E.%20et%20al.%20m(6)%20A-mediated%20alternative%20splicing%20coupled%20with%20nonsense-mediated%20mRNA%20decay%20regulates%20SAM%20synthetase%20homeostasis.%20EMBO%20J.40,%20e106434%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Yoshida, H. et al. Elucidation of the aberrant 3’ splice site selection by cancer-associated mutations on the U2AF1. <em>Nat. Commun.</em><strong>11</strong>, 4744 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-18559-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7505975/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32958768/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoshida,%20H.%20et%20al.%20Elucidation%20of%20the%20aberrant%203%E2%80%99%20splice%20site%20selection%20by%20cancer-associated%20mutations%20on%20the%20U2AF1.%20Nat.%20Commun.11,%204744%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Gu, J., Patton, J. R., Shimba, S. &amp; Reddy, R. Localization of modified nucleotides in Schizosaccharomyces pombe spliceosomal small nuclear RNAs: modified nucleotides are clustered in functionally important regions. <em>RNA</em><strong>2</strong>, 909–918 (1996).
</cite> [<a href="/articles/PMC1369425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8809017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gu,%20J.,%20Patton,%20J.%20R.,%20Shimba,%20S.%20&amp;%20Reddy,%20R.%20Localization%20of%20modified%20nucleotides%20in%20Schizosaccharomyces%20pombe%20spliceosomal%20small%20nuclear%20RNAs:%20modified%20nucleotides%20are%20clustered%20in%20functionally%20important%20regions.%20RNA2,%20909%E2%80%93918%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Brow, D. A. &amp; Guthrie, C. Spliceosomal RNA U6 is remarkably conserved from yeast to mammals. <em>Nature</em><strong>334</strong>, 213–218 (1988).
</cite> [<a href="https://doi.org/10.1038/334213a0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3041282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brow,%20D.%20A.%20&amp;%20Guthrie,%20C.%20Spliceosomal%20RNA%20U6%20is%20remarkably%20conserved%20from%20yeast%20to%20mammals.%20Nature334,%20213%E2%80%93218%20(1988)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Sawa, H. &amp; Abelson, J. Evidence for a base-pairing interaction between U6 small nuclear RNA and 5’ splice site during the splicing reaction in yeast. <em>Proc. Natl Acad. Sci. USA</em><strong>89</strong>, 11269–11273 (1992).
</cite> [<a href="https://doi.org/10.1073/pnas.89.23.11269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC50531/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1333604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sawa,%20H.%20&amp;%20Abelson,%20J.%20Evidence%20for%20a%20base-pairing%20interaction%20between%20U6%20small%20nuclear%20RNA%20and%205%E2%80%99%20splice%20site%20during%20the%20splicing%20reaction%20in%20yeast.%20Proc.%20Natl%20Acad.%20Sci.%20USA89,%2011269%E2%80%9311273%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Sawa, H. &amp; Shimura, Y. Association of U6 snRNA with the 5’-splice site region of pre-mRNA in the spliceosome. <em>Genes Dev.</em><strong>6</strong>, 244–254 (1992).
</cite> [<a href="https://doi.org/10.1101/gad.6.2.244" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1310665/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sawa,%20H.%20&amp;%20Shimura,%20Y.%20Association%20of%20U6%20snRNA%20with%20the%205%E2%80%99-splice%20site%20region%20of%20pre-mRNA%20in%20the%20spliceosome.%20Genes%20Dev.6,%20244%E2%80%93254%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Wassarman, D. &amp; Steitz, J. Interactions of small nuclear RNA’s with precursor messenger RNA during in vitro splicing. <em>Science</em><strong>257</strong>, 1918–1925 (1992).
</cite> [<a href="https://doi.org/10.1126/science.1411506" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1411506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wassarman,%20D.%20&amp;%20Steitz,%20J.%20Interactions%20of%20small%20nuclear%20RNA%E2%80%99s%20with%20precursor%20messenger%20RNA%20during%20in%20vitro%20splicing.%20Science257,%201918%E2%80%931925%20(1992)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Ishigami, Y., Ohira, T., Isokawa, Y., Suzuki, Y. &amp; Suzuki, T. A single m(6)A modification in U6 snRNA diversifies exon sequence at the 5’ splice site. <em>Nat. Commun.</em><strong>12</strong>, 3244 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-23457-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8163875/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34050143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ishigami,%20Y.,%20Ohira,%20T.,%20Isokawa,%20Y.,%20Suzuki,%20Y.%20&amp;%20Suzuki,%20T.%20A%20single%20m(6)A%20modification%20in%20U6%20snRNA%20diversifies%20exon%20sequence%20at%20the%205%E2%80%99%20splice%20site.%20Nat.%20Commun.12,%203244%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Shen, A. et al. U6 snRNA m6A modification is required for accurate and efficient splicing of C. elegans and human pre-mRNAs. <em>Nucleic Acids Res</em>. <strong>52</strong>, 9139–9160 (2024).</cite> [<a href="https://doi.org/10.1093/nar/gkae447" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11347140/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38808663/" class="usa-link">PubMed</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Parker, M. T., Fica, S. M., Barton, G. J. &amp; Simpson, G. G. Inter-species association mapping links splice site evolution to METTL16 and SNRNP27K. <em>eLife</em><strong>12</strong>, e91997 (2023).
</cite> [<a href="https://doi.org/10.7554/eLife.91997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10581693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37787376/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Parker,%20M.%20T.,%20Fica,%20S.%20M.,%20Barton,%20G.%20J.%20&amp;%20Simpson,%20G.%20G.%20Inter-species%20association%20mapping%20links%20splice%20site%20evolution%20to%20METTL16%20and%20SNRNP27K.%20eLife12,%20e91997%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Roost, C. et al. Structure and thermodynamics of N6-methyladenosine in RNA: a spring-loaded base modification. <em>J. Am. Chem. Soc.</em><strong>137</strong>, 2107–2115 (2015).
</cite> [<a href="https://doi.org/10.1021/ja513080v" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4405242/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25611135/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roost,%20C.%20et%20al.%20Structure%20and%20thermodynamics%20of%20N6-methyladenosine%20in%20RNA:%20a%20spring-loaded%20base%20modification.%20J.%20Am.%20Chem.%20Soc.137,%202107%E2%80%932115%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Moravcevic, K. et al. Kinase associated-1 domains drive MARK/PAR1 kinases to membrane targets by binding acidic phospholipids. <em>Cell</em><strong>143</strong>, 966–977 (2010).
</cite> [<a href="https://doi.org/10.1016/j.cell.2010.11.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3031122/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21145462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moravcevic,%20K.%20et%20al.%20Kinase%20associated-1%20domains%20drive%20MARK/PAR1%20kinases%20to%20membrane%20targets%20by%20binding%20acidic%20phospholipids.%20Cell143,%20966%E2%80%93977%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Aoyama, T., Yamashita, S. &amp; Tomita, K. Mechanistic insights into m6A modification of U6 snRNA by human METTL16. <em>Nucleic Acids Res.</em><strong>48</strong>, 5157–5168 (2020).
</cite> [<a href="https://doi.org/10.1093/nar/gkaa227" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7229813/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32266935/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aoyama,%20T.,%20Yamashita,%20S.%20&amp;%20Tomita,%20K.%20Mechanistic%20insights%20into%20m6A%20modification%20of%20U6%20snRNA%20by%20human%20METTL16.%20Nucleic%20Acids%20Res.48,%205157%E2%80%935168%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Ju, J. et al. Structure of the caenorhabditis elegans m6A methyltransferase METT10 that regulates SAM homeostasis. <em>Nucleic Acids Res.</em><strong>51</strong>, 2434–2446 (2023).
</cite> [<a href="https://doi.org/10.1093/nar/gkad081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10018337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36794723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ju,%20J.%20et%20al.%20Structure%20of%20the%20caenorhabditis%20elegans%20m6A%20methyltransferase%20METT10%20that%20regulates%20SAM%20homeostasis.%20Nucleic%20Acids%20Res.51,%202434%E2%80%932446%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Yamashita, S., Takagi, Y., Nagaike, T. &amp; Tomita, K. Crystal structures of U6 snRNA-specific terminal uridylyltransferase. <em>Nat. Commun.</em><strong>8</strong>, 15788 (2017).
</cite> [<a href="https://doi.org/10.1038/ncomms15788" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5467268/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28589955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yamashita,%20S.,%20Takagi,%20Y.,%20Nagaike,%20T.%20&amp;%20Tomita,%20K.%20Crystal%20structures%20of%20U6%20snRNA-specific%20terminal%20uridylyltransferase.%20Nat.%20Commun.8,%2015788%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Yu, S. &amp; Kim, V. N. A tale of non-canonical tails: gene regulation by post-transcriptional RNA tailing. <em>Nat. Rev. Mol. Cell Biol.</em><strong>21</strong>, 542–556 (2020).
</cite> [<a href="https://doi.org/10.1038/s41580-020-0246-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32483315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20S.%20&amp;%20Kim,%20V.%20N.%20A%20tale%20of%20non-canonical%20tails:%20gene%20regulation%20by%20post-transcriptional%20RNA%20tailing.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.21,%20542%E2%80%93556%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Ruszkowska, A., Ruszkowski, M., Dauter, Z. &amp; Brown, J. A. Structural insights into the RNA methyltransferase domain of METTL16. <em>Sci. Rep.</em><strong>8</strong>, 5311 (2018).
</cite> [<a href="https://doi.org/10.1038/s41598-018-23608-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5871880/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29593291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ruszkowska,%20A.,%20Ruszkowski,%20M.,%20Dauter,%20Z.%20&amp;%20Brown,%20J.%20A.%20Structural%20insights%20into%20the%20RNA%20methyltransferase%20domain%20of%20METTL16.%20Sci.%20Rep.8,%205311%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Doxtader, K. A. et al. Structural basis for regulation of METTL16, an S-adenosylmethionine homeostasis factor. <em>Mol. Cell</em><strong>71</strong>, 1001–1011 e4 (2018).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2018.07.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6367934/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30197297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Doxtader,%20K.%20A.%20et%20al.%20Structural%20basis%20for%20regulation%20of%20METTL16,%20an%20S-adenosylmethionine%20homeostasis%20factor.%20Mol.%20Cell71,%201001%E2%80%931011%20e4%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Chavali, S. S., Cavender, C. E., Mathews, D. H. &amp; Wedekind, J. E. Arginine forks are a widespread motif to recognize phosphate backbones and guanine nucleobases in the RNA major groove. <em>J. Am. Chem. Soc.</em><strong>142</strong>, 19835–19839 (2020).
</cite> [<a href="https://doi.org/10.1021/jacs.0c09689" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7937406/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33170672/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chavali,%20S.%20S.,%20Cavender,%20C.%20E.,%20Mathews,%20D.%20H.%20&amp;%20Wedekind,%20J.%20E.%20Arginine%20forks%20are%20a%20widespread%20motif%20to%20recognize%20phosphate%20backbones%20and%20guanine%20nucleobases%20in%20the%20RNA%20major%20groove.%20J.%20Am.%20Chem.%20Soc.142,%2019835%E2%80%9319839%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Pham, V. V., Gao, M., Meagher, J. L., Smith, J. L. &amp; D’Souza, V. M. A structure-based mechanism for displacement of the HEXIM adapter from 7SK small nuclear RNA. <em>Commun. Biol.</em><strong>5</strong>, 819 (2022).
</cite> [<a href="https://doi.org/10.1038/s42003-022-03734-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9378691/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35970937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pham,%20V.%20V.,%20Gao,%20M.,%20Meagher,%20J.%20L.,%20Smith,%20J.%20L.%20&amp;%20D%E2%80%99Souza,%20V.%20M.%20A%20structure-based%20mechanism%20for%20displacement%20of%20the%20HEXIM%20adapter%20from%207SK%20small%20nuclear%20RNA.%20Commun.%20Biol.5,%20819%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Corbeski, I. et al. The catalytic mechanism of the RNA methyltransferase METTL3. <em>eLife</em><strong>12</strong>, RP92537 (2024).
</cite> [<a href="https://doi.org/10.7554/eLife.92537" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10932547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38470714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Corbeski,%20I.%20et%20al.%20The%20catalytic%20mechanism%20of%20the%20RNA%20methyltransferase%20METTL3.%20eLife12,%20RP92537%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Breger, K. &amp; Brown, J. A. Elucidating the kinetic mechanism of human METTL16. <em>Biochemistry</em><strong>62</strong>, 494–506 (2023).
</cite> [<a href="https://doi.org/10.1021/acs.biochem.2c00601" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9850920/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36584291/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Breger,%20K.%20&amp;%20Brown,%20J.%20A.%20Elucidating%20the%20kinetic%20mechanism%20of%20human%20METTL16.%20Biochemistry62,%20494%E2%80%93506%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Trippe, R., Sandrock, B. &amp; Benecke, B. J. A highly specific terminal uridylyl transferase modifies the 3’-end of U6 small nuclear RNA. <em>Nucleic Acids Res.</em><strong>26</strong>, 3119–3126 (1998).
</cite> [<a href="https://doi.org/10.1093/nar/26.13.3119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC147682/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9628908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trippe,%20R.,%20Sandrock,%20B.%20&amp;%20Benecke,%20B.%20J.%20A%20highly%20specific%20terminal%20uridylyl%20transferase%20modifies%20the%203%E2%80%99-end%20of%20U6%20small%20nuclear%20RNA.%20Nucleic%20Acids%20Res.26,%203119%E2%80%933126%20(1998)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Trippe, R. et al. Identification, cloning, and functional analysis of the human U6 snRNA-specific terminal uridylyl transferase. <em>RNA</em><strong>12</strong>, 1494–1504 (2006).
</cite> [<a href="https://doi.org/10.1261/rna.87706" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1524887/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16790842/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trippe,%20R.%20et%20al.%20Identification,%20cloning,%20and%20functional%20analysis%20of%20the%20human%20U6%20snRNA-specific%20terminal%20uridylyl%20transferase.%20RNA12,%201494%E2%80%931504%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Yashiro, Y. &amp; Tomita, K. Function and regulation of human terminal uridylyltransferases. <em>Front Genet</em><strong>9</strong>, 538 (2018).
</cite> [<a href="https://doi.org/10.3389/fgene.2018.00538" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6240794/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30483311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yashiro,%20Y.%20&amp;%20Tomita,%20K.%20Function%20and%20regulation%20of%20human%20terminal%20uridylyltransferases.%20Front%20Genet9,%20538%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Yamashita, S. &amp; Tomita, K. Mechanism of U6 snRNA oligouridylation by human TUT1. <em>Nat. Commun.</em><strong>14</strong>, 4686 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-40420-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10415362/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37563152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yamashita,%20S.%20&amp;%20Tomita,%20K.%20Mechanism%20of%20U6%20snRNA%20oligouridylation%20by%20human%20TUT1.%20Nat.%20Commun.14,%204686%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Yamashita, S. &amp; Tomita, K. Cryo-EM structure of human TUT1:U6 snRNA complex. <em>Nucleic Acids Res.</em><strong>53</strong>, gkae1314 (2025).
</cite> [<a href="https://doi.org/10.1093/nar/gkae1314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11734702/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39831302/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yamashita,%20S.%20&amp;%20Tomita,%20K.%20Cryo-EM%20structure%20of%20human%20TUT1:U6%20snRNA%20complex.%20Nucleic%20Acids%20Res.53,%20gkae1314%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Wang, J., Yashiro, Y., Sakaguchi, Y., Suzuki, T. &amp; Tomita, K. Mechanistic insights into tRNA cleavage by a contact-dependent growth inhibitor protein and translation factors. <em>Nucleic Acids Res.</em><strong>50</strong>, 4713–4731 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkac228" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9071432/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35411396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20J.,%20Yashiro,%20Y.,%20Sakaguchi,%20Y.,%20Suzuki,%20T.%20&amp;%20Tomita,%20K.%20Mechanistic%20insights%20into%20tRNA%20cleavage%20by%20a%20contact-dependent%20growth%20inhibitor%20protein%20and%20translation%20factors.%20Nucleic%20Acids%20Res.50,%204713%E2%80%934731%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Collart, M. A. &amp; Oliviero, S. Preparation of yeast RNA. <em>Curr. Protoc. Mol. Biol. Chapter</em><strong>13</strong>, Unit13.12 (2001).</cite> [<a href="https://doi.org/10.1002/0471142727.mb1312s23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18265096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Collart,%20M.%20A.%20&amp;%20Oliviero,%20S.%20Preparation%20of%20yeast%20RNA.%20Curr.%20Protoc.%20Mol.%20Biol.%20Chapter13,%20Unit13.12%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Kabsch, W. XDS. <em>Acta Crystallogr. D. Biol. Crystallogr.</em><strong>66</strong>, 125–132 (2010).
</cite> [<a href="https://doi.org/10.1107/S0907444909047337" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2815665/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20124692/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kabsch,%20W.%20XDS.%20Acta%20Crystallogr.%20D.%20Biol.%20Crystallogr.66,%20125%E2%80%93132%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>McCoy, A. J. et al. Phaser crystallographic software. <em>J. Appl. Crystallogr.</em><strong>40</strong>, 658–674 (2007).
</cite> [<a href="https://doi.org/10.1107/S0021889807021206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2483472/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19461840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McCoy,%20A.%20J.%20et%20al.%20Phaser%20crystallographic%20software.%20J.%20Appl.%20Crystallogr.40,%20658%E2%80%93674%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Varadi, M. et al. AlphaFold protein structure database in 2024: providing structure coverage for over 214 million protein sequences. <em>Nucleic Acids Res.</em><strong>52</strong>, D368–D375 (2023).</cite> [<a href="https://doi.org/10.1093/nar/gkad1011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37933859/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Varadi,%20M.%20et%20al.%20AlphaFold%20protein%20structure%20database%20in%202024:%20providing%20structure%20coverage%20for%20over%20214%20million%20protein%20sequences.%20Nucleic%20Acids%20Res.52,%20D368%E2%80%93D375%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Moafinejad, S. N. et al. SimRNAweb v2.0: a web server for RNA folding simulations and 3D structure modeling, with optional restraints and enhanced analysis of folding trajectories. <em>Nucleic Acids Res.</em><strong>52</strong>, W368–W373 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkae356" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11223799/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38738621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moafinejad,%20S.%20N.%20et%20al.%20SimRNAweb%20v2.0:%20a%20web%20server%20for%20RNA%20folding%20simulations%20and%203D%20structure%20modeling,%20with%20optional%20restraints%20and%20enhanced%20analysis%20of%20folding%20trajectories.%20Nucleic%20Acids%20Res.52,%20W368%E2%80%93W373%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Emsley, P., Lohkamp, B., Scott, W. G. &amp; Cowtan, K. Features and development of Coot. <em>Acta Crystallogr. D. Biol. Crystallogr</em><strong>66</strong>, 486–501 (2010).
</cite> [<a href="https://doi.org/10.1107/S0907444910007493" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2852313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20383002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Emsley,%20P.,%20Lohkamp,%20B.,%20Scott,%20W.%20G.%20&amp;%20Cowtan,%20K.%20Features%20and%20development%20of%20Coot.%20Acta%20Crystallogr.%20D.%20Biol.%20Crystallogr66,%20486%E2%80%93501%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Afonine, P. V. et al. Towards automated crystallographic structure refinement with phenix.refine. <em>Acta Crystallogr D. Biol. Crystallogr</em><strong>68</strong>, 352–367 (2012).
</cite> [<a href="https://doi.org/10.1107/S0907444912001308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322595/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22505256/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Afonine,%20P.%20V.%20et%20al.%20Towards%20automated%20crystallographic%20structure%20refinement%20with%20phenix.refine.%20Acta%20Crystallogr%20D.%20Biol.%20Crystallogr68,%20352%E2%80%93367%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Williams, C. J. et al. MolProbity: more and better reference data for improved all-atom structure validation. <em>Protein Sci.</em><strong>27</strong>, 293–315 (2018).
</cite> [<a href="https://doi.org/10.1002/pro.3330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5734394/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29067766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Williams,%20C.%20J.%20et%20al.%20MolProbity:%20more%20and%20better%20reference%20data%20for%20improved%20all-atom%20structure%20validation.%20Protein%20Sci.27,%20293%E2%80%93315%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Punjani, A., Rubinstein, J. L., Fleet, D. J. &amp; Brubaker, M. A. cryoSPARC: algorithms for rapid unsupervised cryo-EM structure determination. <em>Nat. Methods</em><strong>14</strong>, 290–296 (2017).
</cite> [<a href="https://doi.org/10.1038/nmeth.4169" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28165473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Punjani,%20A.,%20Rubinstein,%20J.%20L.,%20Fleet,%20D.%20J.%20&amp;%20Brubaker,%20M.%20A.%20cryoSPARC:%20algorithms%20for%20rapid%20unsupervised%20cryo-EM%20structure%20determination.%20Nat.%20Methods14,%20290%E2%80%93296%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Sanchez-Garcia, R. et al. DeepEMhancer: a deep learning solution for cryo-EM volume post-processing. <em>Commun. Biol.</em><strong>4</strong>, 874 (2021).
</cite> [<a href="https://doi.org/10.1038/s42003-021-02399-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8282847/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34267316/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sanchez-Garcia,%20R.%20et%20al.%20DeepEMhancer:%20a%20deep%20learning%20solution%20for%20cryo-EM%20volume%20post-processing.%20Commun.%20Biol.4,%20874%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Meng, E. C. et al. UCSF ChimeraX: Tools for structure building and analysis. <em>Protein Sci.</em><strong>32</strong>, e4792 (2023).
</cite> [<a href="https://doi.org/10.1002/pro.4792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10588335/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37774136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meng,%20E.%20C.%20et%20al.%20UCSF%20ChimeraX:%20Tools%20for%20structure%20building%20and%20analysis.%20Protein%20Sci.32,%20e4792%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371086/bin/41467_2025_63021_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (3.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371086/bin/41467_2025_63021_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (1.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371086/bin/41467_2025_63021_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (835.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371086/bin/41467_2025_63021_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (388.2KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data that support this study are available from the corresponding author upon request. The data generated in this study are provided as Source Data files with this paper. The coordinates and structure factors for the crystal structure of spKA-1–spISL have been deposited in the PDB, under the accession number <a href="https://doi.org/10.2210/pdb9M86/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9M86</a>. The atomic coordinates and cryo-EM maps of spMETTL16–spU6 and spMETTL16–SAM–spU6 have been deposited in the Protein Data Bank (PDB IDs: <a href="https://doi.org/10.2210/pdb9U47/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9U47</a> and <a href="https://doi.org/10.2210/pdb9U48/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">9U48</a>), and the Electron Microscopy Data bank (<a href="https://www.ebi.ac.uk/emdb/EMD-63833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-63833</a> and <a href="https://www.ebi.ac.uk/emdb/EMD-63834" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMD-63834</a>), respectively. The micrographs of spMETTL16–spU6 and spMETTL16–SAM–spU6 have been deposited in the Electron Microscopy Public Image Archive <a href="https://doi.org/10.6019/EMPIAR-12699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMPIAR-12699</a> and <a href="https://doi.org/10.6019/EMPIAR-12700" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">EMPIAR-12700</a>, respectively. The structures of hsKA-1 (PDB ID: <a href="https://doi.org/10.2210/pdb6M1U/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6M1U</a>), hsMTD-SAH (PDB ID: <a href="https://doi.org/10.2210/pdb6B92/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6B92</a>), and hsMTD–MAT2A-hp (PDB ID: <a href="https://doi.org/10.2210/pdb6DU4/pdb" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">6DU4</a>) were used for structural comparisons in this study. <a href="#Sec25" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63021-0"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63021.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371086/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371086/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371086%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371086/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371086/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371086/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841561/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371086/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841561/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371086/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371086/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="60llenKFkkfQJbHnNE84dryiaH9AG65gTDNUKlHvCKA9gZi5oHzdLg8hqdbEpaaG">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
