
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Molecular characterization and prognostic implications of KRAS mutations in pancreatic cancer patients: insights from multi-cohort analysis - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532FF38AF23383052FF30002B2CE6D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="npjprecisoncol">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373740/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="NPJ Precision Oncology">
<meta name="citation_title" content="Molecular characterization and prognostic implications of KRAS mutations in pancreatic cancer patients: insights from multi-cohort analysis">
<meta name="citation_author" content="Yubo Jiang">
<meta name="citation_author_institution" content="Department of Gastroenterology, Shandong Cancer Hospital and Institute, Shandong First Medical University and Shandong Academy of Medical Science, Huaiyin, Jinan, Shandong 250117 P.R. China">
<meta name="citation_author" content="Gang Mai">
<meta name="citation_author_institution" content="Department of Hepatobiliary Surgery, Deyang People’s Hospital, Deyang, Sichuan 618300 P.R. China">
<meta name="citation_author" content="Xiaokai Zhao">
<meta name="citation_author_institution" content="Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China">
<meta name="citation_author_institution" content="Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China">
<meta name="citation_author" content="Meng Tang">
<meta name="citation_author_institution" content="Thoracic Surgery Department, Qingdao Central Hospital Affiliated with Rehabilitation University, Qiandao, Shandong 266000 P.R. China">
<meta name="citation_author" content="Pengmin Yang">
<meta name="citation_author_institution" content="Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China">
<meta name="citation_author_institution" content="Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China">
<meta name="citation_author" content="Qian Cheng">
<meta name="citation_author_institution" content="Department of Gastroenterology, Shandong Cancer Hospital and Institute, Shandong First Medical University and Shandong Academy of Medical Science, Huaiyin, Jinan, Shandong 250117 P.R. China">
<meta name="citation_author" content="He Tian">
<meta name="citation_author_institution" content="Department of Gastroenterology, Shandong Cancer Hospital and Institute, Shandong First Medical University and Shandong Academy of Medical Science, Huaiyin, Jinan, Shandong 250117 P.R. China">
<meta name="citation_author" content="Zuoxing Niu">
<meta name="citation_author_institution" content="Department of Gastroenterology, Shandong Cancer Hospital and Institute, Shandong First Medical University and Shandong Academy of Medical Science, Huaiyin, Jinan, Shandong 250117 P.R. China">
<meta name="citation_author" content="Xintao Wang">
<meta name="citation_author_institution" content="Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China">
<meta name="citation_author_institution" content="Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China">
<meta name="citation_author" content="Jiao Wang">
<meta name="citation_author_institution" content="Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China">
<meta name="citation_author_institution" content="Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China">
<meta name="citation_author" content="Yudong Zhu">
<meta name="citation_author_institution" content="Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China">
<meta name="citation_author_institution" content="Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China">
<meta name="citation_author" content="Jieyi Li">
<meta name="citation_author_institution" content="Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China">
<meta name="citation_author_institution" content="Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China">
<meta name="citation_author" content="Ziying Gong">
<meta name="citation_author_institution" content="Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China">
<meta name="citation_author_institution" content="Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China">
<meta name="citation_author" content="Daoyun Zhang">
<meta name="citation_author_institution" content="Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China">
<meta name="citation_author_institution" content="Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China">
<meta name="citation_author" content="Huirong Xu">
<meta name="citation_author_institution" content="Department of Radioactive Intervention, Shandong Cancer Hospital and Institute, Shandong First Medical University and Shandong Academy of Medical Science, Huaiyin, Jinan, Shandong 250117 P.R. China">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="9">
<meta name="citation_firstpage" content="299">
<meta name="citation_doi" content="10.1038/s41698-025-01087-1">
<meta name="citation_pmid" content="40846754">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373740/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373740/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373740/pdf/41698_2025_Article_1087.pdf">
<meta name="description" content="KRAS mutations drive pancreatic adenocarcinoma (PDAC) progression. This study investigates molecular heterogeneity among KRAS subtypes and their prognostic implications. This study explores KRAS mutations in PDAC, analyzing molecular heterogeneity ...">
<meta name="og:title" content="Molecular characterization and prognostic implications of KRAS mutations in pancreatic cancer patients: insights from multi-cohort analysis">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="KRAS mutations drive pancreatic adenocarcinoma (PDAC) progression. This study investigates molecular heterogeneity among KRAS subtypes and their prognostic implications. This study explores KRAS mutations in PDAC, analyzing molecular heterogeneity ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373740/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373740">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41698-025-01087-1"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41698_2025_Article_1087.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373740%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373740/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373740/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373740/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-npjprecisoncol.jpg" alt="NPJ Precision Oncology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to NPJ Precision Oncology" title="Link to NPJ Precision Oncology" shape="default" href="https://www.nature.com/npjprecisiononcology/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">NPJ Precis Oncol</button></div>. 2025 Aug 22;9:299. doi: <a href="https://doi.org/10.1038/s41698-025-01087-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41698-025-01087-1</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22NPJ%20Precis%20Oncol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22NPJ%20Precis%20Oncol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22NPJ%20Precis%20Oncol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22NPJ%20Precis%20Oncol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Molecular characterization and prognostic implications of <em>KRAS</em> mutations in pancreatic cancer patients: insights from multi-cohort analysis</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yubo Jiang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yubo Jiang</span></h3>
<div class="p">
<sup>1</sup>Department of Gastroenterology, Shandong Cancer Hospital and Institute, Shandong First Medical University and Shandong Academy of Medical Science, Huaiyin, Jinan, Shandong 250117 P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yubo Jiang</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mai%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Gang Mai</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Gang Mai</span></h3>
<div class="p">
<sup>2</sup>Department of Hepatobiliary Surgery, Deyang People’s Hospital, Deyang, Sichuan 618300 P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mai%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gang Mai</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Xiaokai Zhao</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Xiaokai Zhao</span></h3>
<div class="p">
<sup>3</sup>Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China </div>
<div class="p">
<sup>4</sup>Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaokai Zhao</span></a>
</div>
</div>
<sup>3,</sup><sup>4,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Meng Tang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Meng Tang</span></h3>
<div class="p">
<sup>5</sup>Thoracic Surgery Department, Qingdao Central Hospital Affiliated with Rehabilitation University, Qiandao, Shandong 266000 P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tang%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Meng Tang</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Pengmin Yang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Pengmin Yang</span></h3>
<div class="p">
<sup>3</sup>Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China </div>
<div class="p">
<sup>4</sup>Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pengmin Yang</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Qian Cheng</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Qian Cheng</span></h3>
<div class="p">
<sup>1</sup>Department of Gastroenterology, Shandong Cancer Hospital and Institute, Shandong First Medical University and Shandong Academy of Medical Science, Huaiyin, Jinan, Shandong 250117 P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qian Cheng</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tian%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">He Tian</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">He Tian</span></h3>
<div class="p">
<sup>1</sup>Department of Gastroenterology, Shandong Cancer Hospital and Institute, Shandong First Medical University and Shandong Academy of Medical Science, Huaiyin, Jinan, Shandong 250117 P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tian%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">He Tian</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Niu%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Zuoxing Niu</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Zuoxing Niu</span></h3>
<div class="p">
<sup>1</sup>Department of Gastroenterology, Shandong Cancer Hospital and Institute, Shandong First Medical University and Shandong Academy of Medical Science, Huaiyin, Jinan, Shandong 250117 P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Niu%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zuoxing Niu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Xintao Wang</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Xintao Wang</span></h3>
<div class="p">
<sup>3</sup>Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China </div>
<div class="p">
<sup>4</sup>Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xintao Wang</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Jiao Wang</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Jiao Wang</span></h3>
<div class="p">
<sup>3</sup>Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China </div>
<div class="p">
<sup>4</sup>Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiao Wang</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Yudong Zhu</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Yudong Zhu</span></h3>
<div class="p">
<sup>3</sup>Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China </div>
<div class="p">
<sup>4</sup>Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yudong Zhu</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Jieyi Li</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Jieyi Li</span></h3>
<div class="p">
<sup>3</sup>Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China </div>
<div class="p">
<sup>4</sup>Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jieyi Li</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gong%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Ziying Gong</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Ziying Gong</span></h3>
<div class="p">
<sup>3</sup>Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China </div>
<div class="p">
<sup>4</sup>Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gong%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Ziying Gong</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Daoyun Zhang</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Daoyun Zhang</span></h3>
<div class="p">
<sup>3</sup>Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China </div>
<div class="p">
<sup>4</sup>Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daoyun Zhang</span></a>
</div>
</div>
<sup>3,</sup><sup>4,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Huirong Xu</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Huirong Xu</span></h3>
<div class="p">
<sup>6</sup>Department of Radioactive Intervention, Shandong Cancer Hospital and Institute, Shandong First Medical University and Shandong Academy of Medical Science, Huaiyin, Jinan, Shandong 250117 P.R. China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Huirong Xu</span></a>
</div>
</div>
<sup>6,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Gastroenterology, Shandong Cancer Hospital and Institute, Shandong First Medical University and Shandong Academy of Medical Science, Huaiyin, Jinan, Shandong 250117 P.R. China </div>
<div id="Aff2">
<sup>2</sup>Department of Hepatobiliary Surgery, Deyang People’s Hospital, Deyang, Sichuan 618300 P.R. China </div>
<div id="Aff3">
<sup>3</sup>Jiaxing Key Laboratory of Precision Medicine and Companion Diagnostics, Jiaxing Yunying Medical Inspection Co., Ltd., Jiaxing, Zhejiang 314000 P.R. China </div>
<div id="Aff4">
<sup>4</sup>Department of R&amp;D, Shanghai Yunying Biopharmaceutical Technology Co., Ltd., Shanghai, 201612 P.R. China </div>
<div id="Aff5">
<sup>5</sup>Thoracic Surgery Department, Qingdao Central Hospital Affiliated with Rehabilitation University, Qiandao, Shandong 266000 P.R. China </div>
<div id="Aff6">
<sup>6</sup>Department of Radioactive Intervention, Shandong Cancer Hospital and Institute, Shandong First Medical University and Shandong Academy of Medical Science, Huaiyin, Jinan, Shandong 250117 P.R. China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 10; Accepted 2025 Aug 8; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373740  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846754/" class="usa-link">40846754</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1"><em>KRAS</em> mutations drive pancreatic adenocarcinoma (PDAC) progression. This study investigates molecular heterogeneity among <em>KRAS</em> subtypes and their prognostic implications. This study explores <em>KRAS</em> mutations in PDAC, analyzing molecular heterogeneity and prognosis across our hospital cohort (SDFM, <em>n</em> = 113) with TCGA cohort (<em>n</em> = 183) and QCMG cohort (<em>n</em> = 383). <em>KRAS</em>, <em>TP53</em>, <em>CDKN2A</em>, and <em>SMAD4</em> were the main mutated genes. Co-mutations of <em>KRAS</em> with <em>TP53</em>, and <em>TP53</em> with <em>CDKN2A</em>, correlated with higher tumor mutation burden and poorer outcomes. <em>KRAS</em> subtypes G12D and Q16&amp;others had worse prognosis than G12V and G12R. Combining <em>TP53</em> status with <em>KRAS</em> subtypes improved risk stratification: high-risk patients had shorter survival (<em>P</em> ≤ 0.001), higher PD-L1 expression, P53 pathway alterations, fewer CD4+/CD8 + T cells and macrophages (<em>p</em> &lt; 0.05), but more neutrophils (<em>p</em> &lt; 0.001). These findings underscore the prognostic impact of <em>KRAS</em> and <em>TP53</em> mutations, guiding personalized treatment.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Pancreatic cancer, Pancreatic cancer</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par12">Globally, pancreatic cancer is the 12th most common cancer, and the seventh leading cause of cancer-related deaths<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. In Asia, the incidence of pancreatic cancer is increasing<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Pancreatic cancer is notorious for its high degree of malignancy. Due to the lack of distinctive clinical symptoms and early diagnostic tools, many patients are diagnosed at late stages of the disease, leading to poor prognosis, with a 5-year survival rate of only 10%<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>–<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Pancreatic adenocarcinoma (PDAC) is the most common type of pancreatic cancer, accounting for approximately 85% of all types of pancreatic cancer. The advancement of molecular diagnostics has enabled researchers to gradually pinpoint targets for treating PDAC, including <em>KRAS</em><sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>.</p>
<p id="Par13"><em>KRAS</em> mutations are the most prevalent genetic alterations in PDAC, occurring in approximately 90% of cases<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. However, the specific molecular structure of <em>KRAS</em> results in its low drug ability. Cancer-associated mutations in <em>KRAS</em> cluster in one of three hotspots, with a majority (84%) of mutations causing single amino acid substitutions at G12<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Of the possible single-base missense mutations that can occur at G12, G12D is the most predominant (42%), followed by G12V. Currently, Adagrasib<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup> and Sotorasib<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> can be used for targeted therapy in PDAC patients with the <em>KRAS</em> G12C mutation. However, as G12C mutations are relatively low, this means that the majority of PDAC patients with G12D and G12V mutations face a lack of targeted therapies. In a recent cohort study from MDA, the overall positive rate of <em>KRAS</em> mutations was 82%, with the most common mutation being <em>KRAS</em> G12D (39%), followed by <em>KRAS</em> G12V (31%), <em>KRAS</em> G12R (14%), <em>KRAS</em> Q61 (6%), and other rarer <em>KRAS</em> variants (9%)<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. In terms of prognosis, patients with <em>KRAS</em> G12D or <em>KRAS</em> Q61 mutations have a lower survival rate compared to <em>KRAS</em> wild-type patients, regardless of disease stage. The overall survival of patients with <em>KRAS</em> G12R mutations is similar to that of wild-type patients. Other studies have also indicated that different <em>KRAS</em> mutations have varying impacts on prognosis<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>.</p>
<p id="Par14">Considering the significant roles of <em>KRAS</em> in PDAC, we conducted analyses to examine the influence of <em>KRAS</em> mutations on the molecular characteristics of patients. Our study aimed to provide new insight into <em>KRAS</em> mutations and the development of PDAC. These findings will provide a theoretical foundation for future research on targeted treatments for PDAC</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Mutation overview and analysis</h3>
<p id="Par15">To characterize the mutational landscape of PDAC, we analyzed genomic profiles from three independent cohorts. Consistent with the known molecular architecture of PDAC<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, <em>KRAS</em> and <em>TP53</em> were the most commonly mutated genes across all datasets (SDFM, TCGA, QCMG), followed by <em>CDKN2A</em> and <em>SMAD4</em> (Fig. <a href="#Fig1" class="usa-link">1A–C</a>). Statistically significant co-occurrence was observed between <em>KRAS</em> and <em>TP53</em>, as well as between <em>TP53</em> and <em>CDKN2A</em> (<em>p</em> &lt; 0.05 in all cohorts; Fig. <a href="#Fig1" class="usa-link">1A–C</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Mutation overview and analysis for SDFM, TCGA, and QCMG cohort.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373740_41698_2025_1087_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d63a/12373740/6f6697018e1e/41698_2025_1087_Fig1_HTML.jpg" loading="lazy" id="d33e609" height="536" width="790" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The mutation landscape and co-mutation patterns of the top 20 genes in pancreatic cancer patients were analyzed across three distinct cohorts: 113 patients from the SDFM cohort (<strong>A</strong>), 183 patients from the TCGA cohort (<strong>B</strong>), and 383 patients from the QCMG cohort (<strong>C</strong>); Additionally, in all three cohorts, PDAC patients with mutations in <em>KRAS</em>, <em>TP53</em>, and <em>CDKN2A</em> exhibited significantly higher TMB values compared to those with wild-type alleles (<em>P</em> &lt; 0.01), whereas no significant correlation was observed for <em>SMAD4</em> mutations (<strong>D</strong>–<strong>F</strong>).</p></figcaption></figure><p id="Par16">Mutations in <em>KRAS</em>, <em>TP53</em>, and <em>CDKN2A</em> were consistently associated with significantly elevated TMB compared to their respective wild-type groups across all cohorts (<em>p</em> &lt; 0.05; Fig. <a href="#Fig1" class="usa-link">1D–F</a> and Table <a href="#MOESM1" class="usa-link">S1</a>), whereas <em>SMAD4</em> mutations were not significantly associated with TMB. Despite these associations, overall TMB levels remained low, with mean values ranging from 0.86 to 3.13 mutations per megabase (Table <a href="#MOESM1" class="usa-link">S1</a>).</p></section><section id="Sec4"><h3 class="pmc_sec_title">Survival analysis based on mutation analysis</h3>
<p id="Par17">To investigate the impact of <em>KRAS</em>, <em>TP53</em>, <em>CDKN2A</em>, and <em>SMAD4</em> mutations on prognosis, we conducted a univariate regression analysis of OS using the TCGA cohort data. The results are depicted in Fig. <a href="#Fig2" class="usa-link">2A</a>. Factors such as age, <em>KRAS</em>, and <em>TP53</em> mutations showed significant associations with overall survival (OS) (<em>p</em> &lt; 0.05). In the multivariate regression analysis (Fig. <a href="#Fig2" class="usa-link">2B</a>), only <em>KRAS</em> mutation and age remained significant factors (<em>p</em> &lt; 0.05). Detailed information on regression analysis is provided in Table <a href="#MOESM1" class="usa-link">S2</a>. Kaplan–Meier survival analysis revealed mutations in <em>KRAS</em> and <em>TP53</em> were significantly associated with both poorer OS and progression-free survival (PFS) (<em>p</em> &lt; 0.05) (Fig. <a href="#Fig2" class="usa-link">2C, D</a>). Combined <em>KRAS</em> and <em>TP53</em> mutations result in the worst survival outcomes (<em>p</em> &lt; 0.05) (Fig. <a href="#Fig2" class="usa-link">2E</a>). However, no significant difference in OS or PFS between PDAC patients aged &lt;65 and those aged ≥65 (Fig. <a href="#MOESM1" class="usa-link">S2</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Prognostic analysis of clinical and molecular features.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373740_41698_2025_1087_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d63a/12373740/c2b97780576d/41698_2025_1087_Fig2_HTML.jpg" loading="lazy" id="d33e783" height="545" width="790" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Univariate Cox regression analysis identified Stage II (HR = 2.42, <em>p</em> = 0.026), <em>KRAS</em> mutation (HR = 0.44, <em>p</em> &lt; 0.001), and <em>TP53</em> mutation (HR = 0.59, <em>p</em> = 0.014) as significant factors for overall survival (OS); <strong>B</strong> Multivariate analysis highlighted age (HR = 1.026, <em>p</em> = 0.011) and <em>KRAS</em> mutation (HR = 0.571, <em>p</em> = 0.028) as independent prognostic factors; <strong>C</strong>–<strong>E</strong> Kaplan–Meier survival analyses demonstrated: <em>KRAS</em> mutations are associated with worse outcomes in both OS (<em>p</em> = 0.00032) and PFS (<em>p</em> = 0.01); <em>TP53</em> mutations predict poorer survival compared to wild-type (OS: <em>p</em> = 0.013, PFS: <em>p</em> = 0.0056); combined <em>KRAS</em> and <em>TP53</em> mutations result in the worst survival outcomes (OS: <em>p</em> = 0.0023, PFS: <em>p</em> = 0.011) in TCGA cohort.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Survival analysis of PDAC patients with <em>KRAS</em> mutation</h3>
<p id="Par18">In our genetic mutation prognosis analysis, we further differentiated <em>KRAS</em> mutations. Figure <a href="#Fig3" class="usa-link">3A</a> shows that the <em>KRAS</em> G12D, G12V, and G12R mutations were predominant across all three cohorts, followed by <em>KRAS</em> Q61 and other mutations. Univariate and multivariate regression analyses on the TCGA cohort with <em>KRAS</em> mutations (Table <a href="#Tab1" class="usa-link">1</a>) revealed that the G12D, Q61, and other mutations were associated with a worse prognosis compared to the G12V mutation subgroup. Notably, the Q61 and other mutations were statistically significant (<em>P</em> &lt; 0.05). However, analysis of clinical characteristics and driver gene mutations across the three cohorts (Tables <a href="#MOESM1" class="usa-link">S3</a>–<a href="#MOESM1" class="usa-link">S5</a>) showed no significant differences in age, gender, tumor stage, <em>TP53</em>, <em>CDKN2A</em>, <em>SMAD4</em> mutations, TMB, and MSI (Fig. <a href="#MOESM1" class="usa-link">S1</a>). Kaplan–Meier survival analysis indicated that the <em>KRAS</em> G12D, Q61, and other mutations had a worse prognosis, while G12R and G12V mutations had the highest median OS or PFS (Fig. <a href="#Fig3" class="usa-link">3B</a>). Further analysis of <em>TP53, CDKN2A</em>, and <em>SMAD4</em> mutations within <em>KRAS</em> subgroups showed no inter-group differences in <em>TP53</em> (Fig. <a href="#Fig3" class="usa-link">3C</a>), <em>CDKN2A</em> (Fig. <a href="#MOESM1" class="usa-link">S3A</a>), or <em>SMAD4</em> (Fig. <a href="#MOESM1" class="usa-link">S4A</a>). However, <em>TP53</em> mutations were significantly more prevalent in the <em>KRAS</em> G12D and Q61&amp;other mutation subgroups compared to other <em>KRAS</em> mutations (<em>p</em> &lt; 0.05) across all three cohorts.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. <em>KRAS</em> subtypes and <em>TP53</em> mutation status: distribution and clinical significance.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373740_41698_2025_1087_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d63a/12373740/d61c64e39d8c/41698_2025_1087_Fig3_HTML.jpg" loading="lazy" id="d33e943" height="761" width="789" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Bar charts illustrating the distribution of <em>KRAS</em> mutation subtypes (G12D, G12V, G12R, Q61X, and other <em>KRAS</em> mutations) in the SDFM (<em>n</em> = 113), TCGA (<em>n</em> = 183), and QCMG cohorts (<em>n</em> = 383); <strong>B</strong> Kaplan–Meier survival curves showing OS and PFS stratified by <em>KRAS</em> mutation subtypes in TCGA cohort; <strong>C</strong> Bar charts representing the proportion of <em>TP53</em> mutation status within each <em>KRAS</em> mutation subtype across the three cohorts; <strong>D</strong>, <strong>E</strong> Kaplan–Meier survival curves illustrating OS and PFS for <em>KRAS</em> mutation subtypes (G12D, G12V, G12R) in <em>TP53</em> mutation and <em>TP53</em> wildtype groups.</p></figcaption></figure><section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Univariate and multivariate regression analysis of the association between overall survival and clinical characteristics and key driver mutations in <em>KRAS</em> mutated PDAC patients from the TCGA cohort</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th colspan="1" rowspan="1">Characteristics</th>
<th colspan="1" rowspan="1">Total (<em>N</em>)</th>
<th colspan="1" rowspan="1">HR(95% CI) Univariate analysis</th>
<th colspan="1" rowspan="1">
<em>P</em>-value Univariate analysis</th>
<th colspan="1" rowspan="1">HR(95% CI) Multivariate analysis</th>
<th colspan="1" rowspan="1">
<em>P</em>-value Multivariate analysis</th>
</tr></thead>
<tbody>
<tr>
<td colspan="1" rowspan="1"><strong>Age</strong></td>
<td colspan="1" rowspan="1">116</td>
<td colspan="1" rowspan="1">1.019 (0.997–1.042)</td>
<td colspan="1" rowspan="1">0.087</td>
<td colspan="1" rowspan="1">1.015 (0.992–1.039)</td>
<td colspan="1" rowspan="1">0.195</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>Sex</strong></td>
<td colspan="1" rowspan="1">116</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Male</td>
<td colspan="1" rowspan="1">69</td>
<td colspan="1" rowspan="1">Reference</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Female</td>
<td colspan="1" rowspan="1">47</td>
<td colspan="1" rowspan="1">1.093 (0.685–1.745)</td>
<td colspan="1" rowspan="1">0.708</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>AJCC Stage</strong></td>
<td colspan="1" rowspan="1">116</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Stage I</td>
<td colspan="1" rowspan="1">8</td>
<td colspan="1" rowspan="1">Reference</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Stage II</td>
<td colspan="1" rowspan="1">101</td>
<td colspan="1" rowspan="1">1.114 (0.404–3.073)</td>
<td colspan="1" rowspan="1">0.834</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Stage III</td>
<td colspan="1" rowspan="1">3</td>
<td colspan="1" rowspan="1">1.352 (0.245–7.456)</td>
<td colspan="1" rowspan="1">0.729</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Stage IV</td>
<td colspan="1" rowspan="1">4</td>
<td colspan="1" rowspan="1">1.127 (0.251–5.067)</td>
<td colspan="1" rowspan="1">0.876</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong><em>KRAS</em></strong></td>
<td colspan="1" rowspan="1">116</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1">G12V</td>
<td colspan="1" rowspan="1">33</td>
<td colspan="1" rowspan="1">Reference</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1">Reference</td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1">G12D</td>
<td colspan="1" rowspan="1">48</td>
<td colspan="1" rowspan="1">1.748 (0.984–3.107)</td>
<td colspan="1" rowspan="1">0.057</td>
<td colspan="1" rowspan="1">1.715 (0.965–3.048)</td>
<td colspan="1" rowspan="1">0.066</td>
</tr>
<tr>
<td colspan="1" rowspan="1">G12R</td>
<td colspan="1" rowspan="1">25</td>
<td colspan="1" rowspan="1">1.062 (0.508–2.224)</td>
<td colspan="1" rowspan="1">0.872</td>
<td colspan="1" rowspan="1">1.070 (0.511–2.242)</td>
<td colspan="1" rowspan="1">0.858</td>
</tr>
<tr>
<td colspan="1" rowspan="1">Q61</td>
<td colspan="1" rowspan="1">8</td>
<td colspan="1" rowspan="1">2.954 (1.151–7.585)</td>
<td colspan="1" rowspan="1">0.024</td>
<td colspan="1" rowspan="1">2.712 (1.048–7.020)</td>
<td colspan="1" rowspan="1">0.040</td>
</tr>
<tr>
<td colspan="1" rowspan="1">other</td>
<td colspan="1" rowspan="1">2</td>
<td colspan="1" rowspan="1">5.574 (1.260–24.665)</td>
<td colspan="1" rowspan="1">0.024</td>
<td colspan="1" rowspan="1">4.906 (1.098–21.924)</td>
<td colspan="1" rowspan="1">0.037</td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong><em>TP53</em></strong></td>
<td colspan="1" rowspan="1">116</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Mut</td>
<td colspan="1" rowspan="1">90</td>
<td colspan="1" rowspan="1">Reference</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Wild</td>
<td colspan="1" rowspan="1">26</td>
<td colspan="1" rowspan="1">0.688 (0.375–1.263)</td>
<td colspan="1" rowspan="1">0.227</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong><em>CDKN2A</em></strong></td>
<td colspan="1" rowspan="1">116</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Mut</td>
<td colspan="1" rowspan="1">33</td>
<td colspan="1" rowspan="1">Reference</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Wild</td>
<td colspan="1" rowspan="1">83</td>
<td colspan="1" rowspan="1">0.887 (0.527–1.493)</td>
<td colspan="1" rowspan="1">0.652</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong><em>SMAD4</em></strong></td>
<td colspan="1" rowspan="1">116</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Mut</td>
<td colspan="1" rowspan="1">32</td>
<td colspan="1" rowspan="1">Reference</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1">Wild</td>
<td colspan="1" rowspan="1">84</td>
<td colspan="1" rowspan="1">1.281 (0.750–2.189)</td>
<td colspan="1" rowspan="1">0.365</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>TMB</strong></td>
<td colspan="1" rowspan="1">116</td>
<td colspan="1" rowspan="1">1.111 (0.734–1.682)</td>
<td colspan="1" rowspan="1">0.618</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
<tr>
<td colspan="1" rowspan="1"><strong>MSI</strong></td>
<td colspan="1" rowspan="1">116</td>
<td colspan="1" rowspan="1">1.021 (0.853–1.222)</td>
<td colspan="1" rowspan="1">0.820</td>
<td colspan="1" rowspan="1"></td>
<td colspan="1" rowspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par19">Prognostic analysis (Fig. <a href="#MOESM1" class="usa-link">S5</a>) revealed that <em>TP53</em> mutations do not significantly impact prognosis in cases with <em>KRAS</em> G12D mutations. In contrast, <em>KRAS</em> G12V mutation is associated with poorer prognosis in <em>TP53</em>-mutant patients, while <em>TP53</em> wild-type patients show relatively better outcomes. Conversely, an opposite trend was observed in <em>KRAS</em> G12R mutation cases. Further analysis of <em>KRAS</em> mutation subtypes in <em>TP53</em>-mutant patients demonstrated that <em>KRAS</em> G12R mutation is associated with significantly better prognosis compared to other subtypes (Fig. <a href="#Fig3" class="usa-link">3D</a>, <em>P</em> &lt; 0.01). In <em>TP53</em> wild-type patients, <em>KRAS</em> G12V mutation was linked to a relatively better prognosis (Fig. <a href="#Fig3" class="usa-link">3E</a>). Notably, patients with both <em>KRAS</em> G12R and <em>TP53</em> mutations showed survival outcomes similar to those with wild-type <em>KRAS</em>, while <em>KRAS</em> G12V mutation in <em>TP53</em> wild-type patients (Fig. <a href="#MOESM1" class="usa-link">S6</a>). Conversely, no clear patterns were observed in prognosis analyses based on <em>CDKN2A</em> (Fig. <a href="#MOESM1" class="usa-link">S3B–F</a>) and <em>SMAD4</em> (Fig. <a href="#MOESM1" class="usa-link">S4B–F</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title">Survival value with risk status and relevant differential expressive gene analysis</h3>
<p id="Par20">In light of these observations, we categorized PDAC patients into high- or low-risk cohorts based on their <em>KRAS</em> and <em>TP53</em> mutation status, as delineated below: Those with <em>KRAS</em> G12D or <em>KRAS</em> G12V mutations co-occurring with <em>TP53</em> mutations, as well as those with <em>KRAS</em> G12R mutations co-occurring with <em>TP53</em> wild-type or <em>KRAS</em> Q61 and other mutations, were designated as high-risk. Meanwhile, patients with <em>KRAS</em> wild-type, <em>KRAS</em> G12V mutations co-occurring with <em>TP53</em> wild-type, and <em>KRAS</em> G12R mutations co-occurring with <em>TP53</em> mutations were classified as low-risk individuals. Based on this risk stratification, we conducted a prognostic analysis in TCGA and MSK cohorts. As illustrated in Figs. <a href="#Fig4" class="usa-link">4A</a> and <a href="#MOESM1" class="usa-link">S7</a>, high-risk PDAC patients demonstrated significantly shorter median OS, PFS, DFS (disease free survival), and DSS (disease free survival) compared to low-risk patients in TCGA cohort (<em>p</em> ≤ 0.0001). Multivariate regression analysis further identified high-risk status as an independent predictor of poor prognosis (Fig. <a href="#Fig4" class="usa-link">4B</a>). The prognostic analysis of the MSK cohort further validated that risk stratification based on <em>KRAS</em> mutation status is a reliable predictor of outcomes in PDAC patients (Fig. <a href="#MOESM1" class="usa-link">S8</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Survival and relevant analysis with risk status.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373740_41698_2025_1087_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d63a/12373740/1c3477d130d5/41698_2025_1087_Fig4_HTML.jpg" loading="lazy" id="d33e1460" height="786" width="790" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Kaplan–Meier plot of OS and PFS with risk status and validation with MSK cohort (<em>n</em> = 2270) (<strong>A</strong>); multivariate regression analysis of the association between overall survival and risk status in patients (<strong>B</strong>); the heatmap of gene differential expression profile between risk status (<strong>C</strong>); the volcano map of gene up or downregulation in the high and low-risk group (<strong>D</strong>); Bar graph of the GO enrichment analysis results in high-risk group (<strong>E</strong>); Enrichment analysis plot of differential expression gene (<strong>F</strong>).</p></figcaption></figure><p id="Par21">To understand the potential mechanisms underlying these prognostic differences, we conducted a gene differential analysis. As depicted in the heatmap (Fig. <a href="#Fig4" class="usa-link">4C</a>), the gene expression profiles of high-risk patients show obvious differences compared to low-risk patients. Specifically, high-risk patients exhibit a marked upregulation of genes compared to low-risk patients (232 vs 26, Fig. <a href="#Fig4" class="usa-link">4D</a>). As shown in Fig. <a href="#Fig4" class="usa-link">4E</a>, GO enrichment analysis revealed that, in terms of Biological Processes, genes upregulated in high-risk patients were significantly enriched in pathways related to epidermis development, skin development, cornification, epidermal cell differentiation, and keratinization. Regarding Cellular Components, highly enriched categories included the intermediate filament cytoskeleton, anchored components of the membrane, membrane components, intermediate filaments, and lipid droplets. For Molecular Functions, the upregulated genes were predominantly associated with retinol dehydrogenase activity, signal receptor ligand activity, serine-type endopeptidase activity, serine-type peptidase activity, and extracellular matrix structural constituents. Furthermore, KEGG pathway enrichment analysis (Fig. <a href="#Fig4" class="usa-link">4F</a>) identified significant enrichment in key signaling pathways, including retinol metabolism, estrogen signaling, and PPAR signaling. Collectively, these findings suggest that the gene expression profile of high-risk patients is marked by the activation of pathways related to epidermal development, cytoskeletal organization, and signal transduction.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Analysis of tumor signaling pathways with risk status</h3>
<p id="Par22">To further investigate the underlying mechanisms, we conducted a differential mutation analysis of oncogenic signaling pathways. In the SDFM cohort, high-risk patients exhibited a higher <em>TP53</em> pathway mutation rate (96.2% vs. 67.86%, <em>p</em> &lt; 0.001) (Fig. <a href="#Fig5" class="usa-link">5A</a> and Table <a href="#MOESM1" class="usa-link">S6</a>). In the TCGA cohort, high-risk patients showed higher mutation rates in <em>TP53</em> (96.2% vs. 67.86%, <em>p</em> &lt; 0.001), Cell Cycle (34.07% vs. 5.95%, <em>p</em> &lt; 0.001), TGF-Beta (32.97% vs. 19.05%, <em>p</em> &lt; 0.05), and Hippo (29.67% vs. 16.67%, <em>p</em> &lt; 0.05) signaling pathways (Fig. <a href="#Fig5" class="usa-link">5B</a> and Table <a href="#MOESM1" class="usa-link">S6</a>). In the QCMG cohort, high-risk patients exhibited higher mutation rates in <em>TP53</em> (76.23% vs. 51.28%, <em>p</em> &lt; 0.001) and WNT (21.13% vs. 8.55%, <em>p</em> &lt; 0.01) signaling pathways (Fig. <a href="#Fig5" class="usa-link">5C</a> and Table <a href="#MOESM1" class="usa-link">S6</a>). The corresponding pathway mutation profiles are shown in Fig. <a href="#Fig5" class="usa-link">5D–F</a>. The three cohorts exhibited significant differences in the <em>TP53</em> pathway, consistent with the analysis of clinical characteristics and driver mutation status between high-risk and low-risk groups (Tables <a href="#MOESM1" class="usa-link">S7</a>–<a href="#MOESM1" class="usa-link">S9</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Pathway mutation analysis stratified by risk status.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373740_41698_2025_1087_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d63a/12373740/9dfa77d7bfbf/41698_2025_1087_Fig5_HTML.jpg" loading="lazy" id="d33e1577" height="490" width="790" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong>–<strong>C</strong> Bar plots showing the proportion of tumor signaling pathway mutations in high-risk and low-risk groups across three cohorts: SDFM, TCGA, and QCMG. Statistical significance between the groups was assessed using the Wilcoxon test, where “”, “”, and “” represent <em>P</em> &lt; 0.05, <em>P</em> &lt; 0.011, and <em>P</em> &lt; 0.001, respectively. Exploratory analyses with FDR-adjusted <em>p</em>-values in Supplementary Table <a href="#MOESM1" class="usa-link">S6</a>, see “Methods” for analysis definitions; <strong>D</strong>–<strong>F</strong> Heatmaps showing the mutation profiles of pathway-related genes for high-risk and low-risk groups in the three cohorts.</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">Analysis of PD-L1 expression and immune infiltration with risk status</h3>
<p id="Par23">We further analyzed the association between mutations in <em>KRAS</em>, <em>TP53</em>, <em>CDKN2A</em>, and <em>SMAD</em>4 with PD-L1 expression levels. When setting the threshold of PD-L1 expression using TPS at 1%, a significantly higher proportion of PD-L1 positivity was observed in patients with PDAC harboring <em>KRAS</em> or <em>TP53</em> mutations compared to wild-type cases, with the increase being particularly pronounced in <em>TP53</em>-mutated patients (<em>P</em> &lt; 0.05) (Fig. <a href="#Fig6" class="usa-link">6A, B</a>). Similarly, a higher proportion of PD-L1 positive cases was noted in the high-risk group compared to the low-risk group, although the difference did not reach statistical significance (Fig. <a href="#Fig6" class="usa-link">6C</a>). When the TPS threshold was gradually increased to 5% and 10%, a similar distribution trend in PD-L1 expression was consistently observed (Fig. <a href="#MOESM1" class="usa-link">S9</a>). Notably, the trend appeared more prominent at the 5% threshold, suggesting a degree of stability in the observed association (Table <a href="#MOESM1" class="usa-link">S10</a>). However, when the threshold was further increased to 50%, these differences in distribution became less evident (Table <a href="#MOESM1" class="usa-link">S10</a>). In contrast, mutations in <em>CDKN2A</em> and <em>SMAD4</em> showed no significant impact on PD-L1 expression across all threshold settings (1%, 5%, or 10%) (Fig. <a href="#MOESM1" class="usa-link">S9</a>). Moreover, we found no significant correlation between PD-L1 expression and TMB levels (Fig. <a href="#Fig6" class="usa-link">6D</a> and Table <a href="#MOESM1" class="usa-link">S10</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Immune analysis with risk status.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373740_41698_2025_1087_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/d63a/12373740/1c2183e4b641/41698_2025_1087_Fig6_HTML.jpg" loading="lazy" id="d33e1702" height="619" width="788" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong>–<strong>C</strong> Bar charts displaying the proportion of PD-L1 expression (TPS ≥ 1%) stratified by <em>KRAS</em> mutation status, <em>TP53</em> mutation status, and risk status in SDFM cohort (<em>n</em> = 113). Statistical significance was evaluated using the Chi-square test, and <em>p</em>-values are indicated; <strong>D</strong> Correlation analysis between TMB and PD-L1 expression. Spearman’s correlation coefficient (R) and corresponding <em>p</em>-value are shown; <strong>E</strong>–<strong>H</strong> The expression levels of immune checkpoint genes (<em>CD274</em>, <em>CTLA4</em>, and <em>PDCD1</em>), ESTIMATE scores, TIMER immune infiltration scores, and QUANTISEQ scores between low-risk (blue) and high-risk (red) groups in TCGA cohort, “*”, “**”, and “***” indicate <em>P</em> &lt; 0.05, <em>P</em> &lt; 0.011, and <em>P</em> &lt; 0.001, Wilcoxon rank-sum test. Exploratory analyses with FDR-adjusted <em>p</em>-values in Supplementary Table <a href="#MOESM1" class="usa-link">S11</a>, see “Methods” for analysis definitions.</p></figcaption></figure><p id="Par24">Through the analysis of TCGA transcriptome data, it was found that the mRNA expression level of <em>CD274</em> was higher in high-risk patients compared to the low-risk group, while the mRNA expression levels of <em>CTLA4</em> and <em>PDCD1</em> were lower, although the differences were not statistically significant (Fig. <a href="#Fig6" class="usa-link">6E</a>). However, we found the expression levels of CD3(composed of CD3D, CD3E, and CD3G subunits) and CD8 (composed of CD8A and CD8B subunits) were markedly elevated in the low-risk group relative to the high-risk group (Fig. <a href="#MOESM1" class="usa-link">S10</a>). The ESTIMATE immune scoring analysis showed that the immune scores (including StromalScore, ImmuneScore, and ESTIMATEScore) of high-risk patients were significantly lower than those of the low-risk group (<em>P</em> &lt; 0.05) (Fig. <a href="#Fig6" class="usa-link">6F</a> and Table <a href="#MOESM1" class="usa-link">S11</a>). Further analysis of immune infiltration levels between the two groups using TIMER (Fig. <a href="#Fig6" class="usa-link">6G</a>) and QUANTISEQ (Fig. <a href="#Fig6" class="usa-link">6H</a>) revealed that the infiltration levels of CD4 T cells and CD8 T cells were significantly lower in the high-risk group compared to the low-risk group (<em>P</em> &lt; 0.05) (Table <a href="#MOESM1" class="usa-link">S11</a>). However, QUANTISEQ analysis also found that the infiltration levels of Tregs and M2 macrophages were significantly lower in the high-risk group (<em>P</em> &lt; 0.05), while the infiltration levels of M1 macrophages and neutrophils were significantly higher than those in the low-risk group (<em>P</em> &lt; 0.05) (Table <a href="#MOESM1" class="usa-link">S11</a>).</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par25"><em>KRAS</em> mutations are a key event in pancreatic adenocarcinoma (PDAC). In this study, we conducted a comprehensive analysis of <em>KRAS</em> mutations and their subtypes in the SDFM cohort, and performed parallel validation in the TCGA and QCMG cohorts. Consistent with previous studies<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, we identified <em>KRAS</em>, <em>TP53</em>, <em>SMAD4</em>, and <em>CDKN2A</em> mutations as the most common driver mutations in PDAC across all three cohorts. Additionally, <em>KRAS</em> mutations frequently co-occur with <em>TP53</em> mutations, which in turn often co-occur with <em>CDKN2A</em> mutations. Prognostic analyses have shown that <em>KRAS</em> mutations and <em>TP53</em> mutations are associated with poorer prognosis, which is consistent with earlier research on PDAC<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. A preclinical study suggested that <em>KRAS</em> and <em>TP53</em> mutations synergistically promote PDAC growth and metastasis through interactions with CREB1<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>, potentially explaining the poor prognosis associated with these mutations. Additionally, <em>KRAS</em>, <em>TP53</em>, and <em>CDKN2A</em> mutations correlate with higher TMB, which is considered a key factor in generating immune neoantigens<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. However, despite this correlation, TMB did not show a significant link to PDAC prognosis in this study. Most patients had TMB values below the clinical threshold for TMB-H (10 mutations/Mb), indicating that the role of TMB and these mutations in prognosis and treatment warrants further exploration.</p>
<p id="Par26">Our study found that the proportion of <em>KRAS</em> wild-type mutations is approximately 10%, and the prognosis is better than that of <em>KRAS</em> mutant types, which is consistent with previous research<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. The analysis of <em>KRAS</em> mutation subtypes in PDAC patients aligns with previous studies<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, showing <em>KRAS</em> G12D, G12V, and G12R as the most common, followed by Q61 and other rare mutations. Among them, G12D, Q61, and other mutations are linked to shorter PFS and OS, while G12V and G12R mutations are associated with better prognosis. This finding is consistent with the research conducted by MD Anderson Cancer Center<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. The prognosis differences among G12D, Q61, G12V, and G12R mutations may stem from their distinct effects on KRAS protein function and the activation of downstream signaling pathways. The G12D/Q61 mutations may enhance RAF/MEK/ERK signaling<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup> or create an immunosuppressive tumor microenvironment<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>, promoting tumor proliferation, survival, and invasion. Spatial profiling reveals revealed elevated oncogenic signaling and epithelial-mesenchymal transition (EMT) in G12D tumors, whereas G12R tumors exhibited increased nuclear factor κB (NF-κB) signaling<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>.</p>
<p id="Par27">Given the high prevalence of <em>TP53</em> mutations across all <em>KRAS</em> subtypes, with mutation rates exceeding 60%, particularly in the G12D, Q61, and other mutant subtypes, we further explored the relationship between specific <em>KRAS</em> mutations and <em>TP53</em> alterations. Our analysis reveals, for the first time, that <em>TP53</em> mutation status may impact the prognosis of patients with certain <em>KRAS</em> mutation subtypes. As previously reported<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>, the <em>KRAS</em> G12D mutation is associated with poorer survival outcomes, both independently and in combination with <em>TP53</em> mutations. In contrast, for <em>KRAS</em> G12V subtype, those with the <em>TP53</em> wild-type tend to have a longer survival than those with <em>TP53</em> mutations. <em>TP53</em> mutations are linked to worse OS in various cancers, including pancreatic adenocarcinoma (PDAC)<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, likely due to <em>TP53</em>’s critical role in tumor suppression and DNA repair. Mutations in <em>TP53</em> lead to uncontrolled cell cycle progression and accumulation of DNA damage, exacerbating tumor malignancy. Research by Zou et al. suggests that <em>KRAS</em> G12D and <em>TP53</em> co-mutation patients exhibit a low Th1/Th2 ratio and an immunosuppressive microenvironment (high Treg, high ratio of Treg to tumor-specific CD4 + T cell), while <em>KRAS</em> G12V and <em>TP53</em> wild-type patients show a more inflammatory microenvironment and significant survival improvement following adjuvant chemotherapy<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Based on <em>KRAS</em> mutation subtypes and <em>TP53</em> mutation status, PDAC patients are categorized into high-risk and low-risk groups. The low-risk group, including some patients with specific <em>KRAS</em> G12V and G12R mutations, exhibits significantly better prognosis, as confirmed in the large-scale MSK PDAC cohort<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Pathway enrichment analysis reveals that mutations in high-risk patients are primarily concentrated in TP53 and cell cycle-related pathways, with a particularly high co-occurrence rate of <em>KRAS</em> and <em>TP53</em> mutations. Previous studies have demonstrated a synergistic effect between <em>KRAS</em> and <em>TP53</em> mutations in the development and metastasis of pancreatic cancer<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. <em>TP53</em> mutations cause abnormal exon retention, upregulating hnRNPK, which promotes GAP17 membrane localization and sustains Ras signaling, driving tumorigenesis<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. These findings underscore the importance of considering <em>TP53</em> mutation status when treating KRAS-mutant pancreatic cancer, underscoring the need for risk stratification.</p>
<p id="Par28">IHC analysis of PD-L1 protein expression is a critical biomarker for predicting immunotherapy response. Studies consistently show that high PD-L1 expression in PDAC is associated with a poorer prognosis<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>–<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Notably, patients exhibiting PD-L1 expression levels of ≥10%, as assessed using the anti-PD-L1 monoclonal antibody SP142, demonstrated significantly reduced disease-specific survival, as reported by Tessier-Cloutier et al.<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> Our findings further demonstrate that PD-L1 positivity is more frequently observed in PDAC patients carrying <em>KRAS</em> or <em>TP53</em> mutations, particularly those with <em>TP53</em> mutations, and classified in the high-risk group. Notably, the most pronounced difference in PD-L1 expression distribution was observed when a TPS threshold of 5% was applied. These results suggest that PD-L1 positivity or upregulation may contribute to the poor prognosis observed in these patients; however, further data are required to substantiate this association. In immune infiltration analysis, low-risk patients show higher levels of CD4/CD8 T cells and Treg cells, which correlate with better prognosis across various cancers<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. CD4 + T cells activate CD8+ cytotoxic T cells, playing a vital role in anti-tumor immunity<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. However, in high-risk PDAC patients, reduced CD4 + T cell infiltration weakens CD8 + T cell function, diminishing anti-tumor responses and worsening prognosis. We also observed higher neutrophil infiltration in high-risk PDAC patients, which has been linked to increased malignancy and poorer prognosis<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. <em>KRAS</em> and <em>TP53</em> mutations, common in high-risk PDAC, are associated with a higher density of immunosuppressive myeloid cells like MDSCs and Tregs, which inhibit CD4 + Th1 and CD8 + T cell activity, creating an immune-tolerant microenvironment<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. <em>TP53</em> mutations increase CD8 + T cell density, but this effect is influenced by other mutations<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. <em>KRAS</em> mutations, in particular, enhance Treg conversion and suppress CD8 + T cell anti-tumor activity<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Furthermore, <em>KRAS</em> mutations elevate PD-L1 expression, contributing to immune suppression and tumor progression<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Interestingly, tumors with both <em>KRAS</em> and <em>TP53</em> mutations exhibit higher PD-L1 levels, which have been associated with better immunotherapy outcomes<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, highlighting the complex interplay between genetic mutations and immune responses in PDAC prognosis.</p>
<p id="Par29">This study has several limitations. First, although PDAC patients were included, the sample size was small, and detailed pathological staging data were unavailable. To mitigate this, two external PDAC cohorts were analyzed. However, some cohorts lacked complete clinical and prognostic data, with notable imbalances in pathological stage distribution. Future studies should focus on larger, well-characterized cohorts with balanced clinical profiles for validation. Additionally, the unmeasured treatment modality and performance status may affect the clinical generalizability of the risk stratification. Future prospective studies should incorporate these factors to refine the prognostic model. Second, this study’s SDFM cohort utilized a 639-gene panel to detect mutations and calculate TMB, rather than WES, which might have some impact on the mutation profile and TMB calculation, although the results are quite similar to those of TCGA and QCMG. Also, our study primarily focuses on molecular subtyping based on mutational profiles, without in-depth analysis of transcriptomic or proteomic data. Due to practical constraints, key transcriptional features of pancreatic cancer subtypes, such as the classical and basal-like types, were not included. Third, although we conducted prognostic validation using large-scale cohorts, we did not account for the impact of postoperative adjuvant therapy or systemic treatment, which may significantly influence survival outcomes. The absence of treatment factors could limit the generalizability of our findings. Therefore, future studies should incorporate these treatment variables into the analysis to provide a more comprehensive evaluation of their potential effects on prognosis. Lastly, the results of the mechanistic analysis are based solely on the characteristics of the study cohort and require further validation. Therefore, future studies should validate these observations in larger cohorts and explore the functional significance of the identified molecular changes, with the aim of providing a more robust scientific foundation for the management and prognosis improvement of pancreatic cancer patients.</p>
<p id="Par30">In conclusion, the aforementioned findings provide a comprehensive analysis of the complex molecular landscape of pancreatic cancer, with a particular focus on the critical roles of different <em>KRAS</em> mutation subtypes and <em>TP53</em> mutations in patient prognosis. The risk stratification and immune infiltration analysis based on these findings could offer significant insights for personalized treatment strategies in PDAC, helping to identify patient subgroups that may benefit from targeted or immunotherapy treatments.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Patient and sample characteristics</h3>
<p id="Par31">This study included 113 patients diagnosed with PDAC, confirmed through pathological evaluation, and enrolled between June 2021 and December 2023 at Shandong Cancer Hospital, constituting the SDFM cohort. To minimize potential confounders and ensure data accuracy, patients who had previously received anti-RAS inhibitors or immune checkpoint inhibitors (ICIs) were excluded, as these treatments could alter the tumor microenvironment and mutation profiles. Additionally, inclusion was limited to patients with both tumor tissue and paired blood samples available for genetic analysis. Tumor tissue specimens, obtained via surgical resection or biopsy, were required to contain at least 20% tumor cells to ensure the reliability of genetic testing. Rigorous exclusion criteria were applied to maintain the integrity of the study. Patients without a confirmed PDAC diagnosis, those lacking either tumor tissue or paired blood samples, or those with tumor samples containing less than 20% tumor cells, were excluded to guarantee that the genetic analyses were based on high-quality, representative tumor specimens. Clinical data, including patient age and gender, were retrieved from medical records. All participants provided written informed consent, and the study was approved by the institutional review board of Shandong Cancer Hospital (SDTHEC2023011024) in accordance with the Declaration of Helsinki.</p></section><section id="Sec12"><h3 class="pmc_sec_title">DNA extraction and library construction</h3>
<p id="Par32">Tumor DNA and peripheral blood genomic DNA were extracted using commercial kits from YunYing (Shanghai, China), specifically the Human Tissue DNA Extraction Kit and the Human Blood Genomic DNA Extraction Kit, respectively, following the manufacturer’s protocols. DNA was eluted in the provided elution buffer, and its concentration and purity were assessed using a NanoDrop spectrophotometer. All DNA samples were stored at −20 °C until further processing.</p>
<p id="Par33">Library preparation was carried out using the VAHTS Universal DNA Library Prep Kit for Illumina (Vazyme, Nanjing, China), in accordance with the manufacturer’s instructions. Targeted enrichment was performed using proprietary probes optimized by Shanghai YunYing, which cover the exonic regions and selected intronic sequences of 639 cancer-associated genes (see Table <a href="#MOESM1" class="usa-link">S12</a> for full gene list)<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Sequencing was conducted on an Illumina NextSeq500 platform using standard protocols.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Next-generation sequencing (NGS)-based assay and bioinformatics analysis</h3>
<p id="Par34">FastQC software (version 0.11.2) and customized Python script were used to screen FASTQ files, with the adaptor sequences and sequences with Q below 30 removed. Clean reads were mapped to the reference human genome GRCh37/hg19 using BWA (Burrows Wheeler Aligner version 0.7.7). BAM files were then realigned and recalled using GATK3.5<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, which was also used to detect mutations. Duplicate sequences were removed using Picard MarkDuplicates (version 1.35) to reduce any potential polymerase chain reaction bias. VarScan (version 2.3.2)<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup> was used to select single-nucleotide variations (SNVs) satisfying the following criteria: depth ≥ 100, reads ≥ 10, and allele frequency ≥ 5% (if hotspot, ≥ 1%). Pindel (version 0.2.5b8)<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup> was used for insertion or deletion (indel) detection using default parameters, with at least 5 unique reads.</p>
<p id="Par35">Compared with matched normal samples, somatic SNVs and InDels of tumors were named and functionally annotated using MuTect v. 1.1.4 and Varscan2 v. 2.3.9 software. Mutations with a variant allele frequency of ≥5% were defined as high-confidence mutations (≥1% for hotspots). Tumor mutation burden (TMB) was calculated using the number of all somatic, coding, base substitution, and indel mutations per megabase, including synonymous mutations. The total number of mutations counted was divided by the size of the coding region of the targeted territory (1.36 Mb of the coding genome) to calculate the TMB per megabase. Microsatellite instability (MSI) scores of all samples were calculated using MSIsensor<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> with default parameters, a software tool for quantifying MSI in genome sequencing data using tumor-only or paired tumor-normal samples. We used 29 microsatellite sites as input files for MSI detection of tumor-only patterns. The MSI score was defined as the percentage of unstable microsatellites among all microsatellites used. Each microsatellite site had at least 20 spanning reads and single-nucleotide mutations.</p></section><section id="Sec14"><h3 class="pmc_sec_title">PD-L1 expression test</h3>
<p id="Par36">The PD-L1 expression level for each patient was determined using the Dako 22C3 pharmDx system (Agilent Technologies Inc., Santa Clara, CA, USA) assay, and the results are presented as a tumor proportion score (TPS)<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Data collection and statistical analysis</h3>
<p id="Par37">This study utilized two external validation groups for comparative analysis with the SDFM cohort: 184 PDAC patients from the TCGA cohort and 383 PDAC patients from the QCMG cohort, all of which have comprehensive genetic mutation data. These cohorts served as parallel groups for comparative analysis within the SDFM cohort (detailed clinical information see in Table <a href="#MOESM1" class="usa-link">S13</a>). The data were sourced from the publicly available cBioPortal database (<a href="https://www.cbioportal.org/datasets" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.cbioportal.org/datasets</a>). Among them, 116 <em>KRAS</em>-mutant patients from the TCGA cohort were analyzed for survival outcomes across different groups. Additionally, a large validation cohort (MSK cohort, comprising 2270 patients with survival data) was used to validate subsequent risk stratification<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. We used the survfit function from the R package “survival” to analyze the differences in prognosis between different groups of samples. We then assessed the difference in gene expression between the groups using the limma method in the Sangerbox platform<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>–<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup> (<a href="http://www.sangerbox.com/tool" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.sangerbox.com/tool</a>, a free online platform for comprehensive data analysis).</p>
<p id="Par38">The prevalence and distribution of genomic alterations were visualized using the R package “maftools”<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. The “IOBR” package is used for immune infiltration analysis<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. The R package “ggplot2” was used to draw the boxplots. The nonparametric Wilcox test was subsequently used to test for the significance of the difference in means between the two populations. For exploratory post hoc comparisons, multiple testing adjustments were performed using the FDR to control the false discovery rate.</p></section></section><section id="Sec16"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373740/bin/41698_2025_1087_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary information</a><sup> (1.8MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was supported by the Shandong Province Medical and Health Science and Technology project [202303111291]; the Key Research and Development Program of Zhejiang province [2023C03057]; the Jiaxing Science and Technology Program-Social Development Special Project [2025BS009]; and the Key Technology Innovation Projects of Jiaxing [2024BZ20002]. We thank the patients who agreed to participate in this study. We thank the clinical teams who cared for the patients and the Yunying laboratory teams who realized NGS assay and bioinformatics analysis.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>PDAC</dt>
<dd><p id="Par2">pancreatic adenocarcinoma</p></dd>
<dt>KRAS</dt>
<dd><p id="Par3">Kirsten Rat Sarcoma</p></dd>
<dt>PD-L1</dt>
<dd><p id="Par4">Programmed Death-Ligand 1</p></dd>
<dt>TP53</dt>
<dd><p id="Par5">Tumor Protein 53</p></dd>
<dt>TPS</dt>
<dd><p id="Par6">Tumor Proportion Scores</p></dd>
<dt>TMB</dt>
<dd><p id="Par7">Tumor Mutational Burden</p></dd>
<dt>MSI</dt>
<dd><p id="Par8">MicroSatellite Instability</p></dd>
<dt>OS</dt>
<dd><p id="Par9">Overall Survival</p></dd>
<dt>PFS</dt>
<dd><p id="Par10">Progression-Free Survival</p></dd>
<dt>TGF-Beta</dt>
<dd><p id="Par11">Transforming Growth Factor-Beta</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization: Yubo Jiang, Daoyun Zhang, Huirong Xu; Validation: Yubo Jiang, Gang Mai, Meng, Tang, Qian Cheng, He Tian, Zuoxing Niu; Investigation: Yubo Jiang, Meng Tang, Qian Cheng, He Tian, Zuoxing Niu; Writing–original draft: Yubo Jiang, Gang Mai, Xiaokai Zhao, Pengmin Yang; Writing–review &amp; editing: Meng Tang, Qian Cheng, He Tian, Huirong Xu; Formal analysis: Xiaokai Zhao, Pengmin Yang, Ziying Gong; Visualization: Xiaokai Zhao, Pengmin Yang, Jieyi Li; Methodology: Gang Mai, Xiaokai Zhao, Ziying Gong; Data curation: Xintao Wang, Jiao Wang, Yudong Zhu, Daoyun Zhang, Huirong Xu; Supervision: Ziying Gong, Daoyun Zhang, Huirong Xu; Project administration: Daoyun Zhang, Huirong Xu; Funding acquisition: Ziying Gong, Daoyun Zhang, Huirong Xu. All authors contributed to the manuscript revision, and read, and approved the submitted version.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>The data supporting this study’s findings are available on request from the corresponding author. The data are not publicly available due to privacy or ethical restrictions.</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par39">Xiaokai Zhao, Pengmin Yang, Xintao Wang, Jiao Wang, Yudong Zhu, Jieyi Li, Ziying Gong, and Daoyun Zhang were employed by the Jiaxing Yunying Medical Inspection Co., Ltd., and the Shanghai Yunying Biopharmaceutical Technology Co., Ltd. The remaining authors declare that the research was conducted in the absence of any commercial or financial relationships that could be construed as a potential conflict of interest.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Yubo Jiang, Gang Mai, Xiaokai Zhao.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Daoyun Zhang, Email: zdy@yunyingmedicine.com.</p>
<p>Huirong Xu, Email: Huirong_Xu-SCHI@outlook.com.</p></section><section id="sec17"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41698-025-01087-1.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Bray, F. et al. Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. <em>CA Cancer J. Clin.</em><strong>68</strong>, 394–424 (2018).
</cite> [<a href="https://doi.org/10.3322/caac.21492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30207593/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bray,%20F.%20et%20al.%20Global%20cancer%20statistics%202018:%20GLOBOCAN%20estimates%20of%20incidence%20and%20mortality%20worldwide%20for%2036%20cancers%20in%20185%20countries.%20CA%20Cancer%20J.%20Clin.68,%20394%E2%80%93424%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Collaborators, G. P. C. The global, regional, and national burden of pancreatic cancer and its attributable risk factors in 195 countries and territories, 1990-2017: a systematic analysis for the Global Burden of Disease Study 2017. <em>Lancet Gastroenterol. Hepatol.</em><strong>4</strong>, 934–947 (2019).
</cite> [<a href="https://doi.org/10.1016/S2468-1253(19)30347-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7026711/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31648972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Collaborators,%20G.%20P.%20C.%20The%20global,%20regional,%20and%20national%20burden%20of%20pancreatic%20cancer%20and%20its%20attributable%20risk%20factors%20in%20195%20countries%20and%20territories,%201990-2017:%20a%20systematic%20analysis%20for%20the%20Global%20Burden%20of%20Disease%20Study%202017.%20Lancet%20Gastroenterol.%20Hepatol.4,%20934%E2%80%93947%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Maisonneuve, P. &amp; Lowenfels, A. B. Epidemiology of pancreatic cancer: an update. <em>Dig. Dis.</em><strong>28</strong>, 645–656 (2010).
</cite> [<a href="https://doi.org/10.1159/000320068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21088417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maisonneuve,%20P.%20&amp;%20Lowenfels,%20A.%20B.%20Epidemiology%20of%20pancreatic%20cancer:%20an%20update.%20Dig.%20Dis.28,%20645%E2%80%93656%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Wood, L. D. et al. Pancreatic cancer: pathogenesis, screening, diagnosis, and treatment. <em>Gastroenterology</em><strong>163</strong>, 386–402.e1 (2022).
</cite> [<a href="https://doi.org/10.1053/j.gastro.2022.03.056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9516440/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35398344/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wood,%20L.%20D.%20et%20al.%20Pancreatic%20cancer:%20pathogenesis,%20screening,%20diagnosis,%20and%20treatment.%20Gastroenterology163,%20386%E2%80%93402.e1%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Cai, J. et al. Advances in the epidemiology of pancreatic cancer: trends, risk factors, screening, and prognosis. <em>Cancer Lett.</em><strong>520</strong>, 1–11 (2021).
</cite> [<a href="https://doi.org/10.1016/j.canlet.2021.06.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34216688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cai,%20J.%20et%20al.%20Advances%20in%20the%20epidemiology%20of%20pancreatic%20cancer:%20trends,%20risk%20factors,%20screening,%20and%20prognosis.%20Cancer%20Lett.520,%201%E2%80%9311%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Bekaii-Saab, T. S. et al. Adagrasib in advanced solid tumors harboring a KRAS(G12C) mutation. <em>J. Clin. Oncol.</em><strong>41</strong>, 4097–4106 (2023).
</cite> [<a href="https://doi.org/10.1200/JCO.23.00434" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10852394/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37099736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bekaii-Saab,%20T.%20S.%20et%20al.%20Adagrasib%20in%20advanced%20solid%20tumors%20harboring%20a%20KRAS(G12C)%20mutation.%20J.%20Clin.%20Oncol.41,%204097%E2%80%934106%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Rose, S. Adagrasib moving ahead in GI cancers. <em>Cancer Discov.</em><strong>12</strong>, 590–591 (2022).</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-NB2022-0003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35082124/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rose,%20S.%20Adagrasib%20moving%20ahead%20in%20GI%20cancers.%20Cancer%20Discov.12,%20590%E2%80%93591%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Hong, D. S. et al. KRAS(G12C) inhibition with sotorasib in advanced solid tumors. <em>N. Engl. J. Med.</em><strong>383</strong>, 1207–1217 (2020).
</cite> [<a href="https://doi.org/10.1056/NEJMoa1917239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7571518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32955176/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hong,%20D.%20S.%20et%20al.%20KRAS(G12C)%20inhibition%20with%20sotorasib%20in%20advanced%20solid%20tumors.%20N.%20Engl.%20J.%20Med.383,%201207%E2%80%931217%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>The-Cancer-Genome-Atlas-Research-Network Integrated genomic characterization of pancreatic ductal adenocarcinoma. <em>Cancer Cell</em><strong>32</strong>, 185–203.e13 (2017).
</cite> [<a href="https://doi.org/10.1016/j.ccell.2017.07.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5964983/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28810144/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?The-Cancer-Genome-Atlas-Research-Network%20Integrated%20genomic%20characterization%20of%20pancreatic%20ductal%20adenocarcinoma.%20Cancer%20Cell32,%20185%E2%80%93203.e13%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Varghese, A. M. et al. Clinicogenomic landscape of pancreatic adenocarcinoma identifies KRAS mutant dosage as prognostic of overall survival. <em>Nat. Med.</em><strong>31</strong>, 466–477 (2025).
</cite> [<a href="https://doi.org/10.1038/s41591-024-03362-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11835752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39753968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Varghese,%20A.%20M.%20et%20al.%20Clinicogenomic%20landscape%20of%20pancreatic%20adenocarcinoma%20identifies%20KRAS%20mutant%20dosage%20as%20prognostic%20of%20overall%20survival.%20Nat.%20Med.31,%20466%E2%80%93477%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Cox, A. D. et al. Drugging the undruggable RAS: mission possible? <em>Nat. Rev. Drug Discov.</em><strong>13</strong>, 828–851 (2014).
</cite> [<a href="https://doi.org/10.1038/nrd4389" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4355017/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25323927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cox,%20A.%20D.%20et%20al.%20Drugging%20the%20undruggable%20RAS:%20mission%20possible?%20Nat.%20Rev.%20Drug%20Discov.13,%20828%E2%80%93851%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Yousef, A. et al. Impact of KRAS mutations and co-mutations on clinical outcomes in pancreatic ductal adenocarcinoma. <em>NPJ Precis. Oncol.</em><strong>8</strong>, 27 (2024).
</cite> [<a href="https://doi.org/10.1038/s41698-024-00505-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10838312/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38310130/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yousef,%20A.%20et%20al.%20Impact%20of%20KRAS%20mutations%20and%20co-mutations%20on%20clinical%20outcomes%20in%20pancreatic%20ductal%20adenocarcinoma.%20NPJ%20Precis.%20Oncol.8,%2027%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Diehl, A. C. et al. KRAS mutation variants and co-occurring PI3K pathway alterations impact survival for patients with pancreatic ductal adenocarcinomas. <em>Oncologist</em><strong>27</strong>, 1025–1033 (2022).
</cite> [<a href="https://doi.org/10.1093/oncolo/oyac179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10249424/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36124727/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Diehl,%20A.%20C.%20et%20al.%20KRAS%20mutation%20variants%20and%20co-occurring%20PI3K%20pathway%20alterations%20impact%20survival%20for%20patients%20with%20pancreatic%20ductal%20adenocarcinomas.%20Oncologist27,%201025%E2%80%931033%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Lee, J. K. et al. Comprehensive pan-cancer genomic landscape of KRAS altered cancers and real-world outcomes in solid tumors. <em>NPJ Precis. Oncol.</em><strong>6</strong>, 91 (2022).
</cite> [<a href="https://doi.org/10.1038/s41698-022-00334-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9734185/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36494601/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20J.%20K.%20et%20al.%20Comprehensive%20pan-cancer%20genomic%20landscape%20of%20KRAS%20altered%20cancers%20and%20real-world%20outcomes%20in%20solid%20tumors.%20NPJ%20Precis.%20Oncol.6,%2091%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Kim, S. T. et al. Impact of KRAS mutations on clinical outcomes in pancreatic cancer patients treated with first-line gemcitabine-based chemotherapy. <em>Mol. Cancer Ther</em><strong>10</strong>, 1993–1999 (2011).
</cite> [<a href="https://doi.org/10.1158/1535-7163.MCT-11-0269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21862683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20S.%20T.%20et%20al.%20Impact%20of%20KRAS%20mutations%20on%20clinical%20outcomes%20in%20pancreatic%20cancer%20patients%20treated%20with%20first-line%20gemcitabine-based%20chemotherapy.%20Mol.%20Cancer%20Ther10,%201993%E2%80%931999%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Nakano, Y. et al. KRAS mutations in cell-free DNA from preoperative and postoperative sera as a pancreatic cancer marker: a retrospective study. <em>Br. J. Cancer</em><strong>118</strong>, 662–669 (2018).
</cite> [<a href="https://doi.org/10.1038/bjc.2017.479" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5846073/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29360815/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nakano,%20Y.%20et%20al.%20KRAS%20mutations%20in%20cell-free%20DNA%20from%20preoperative%20and%20postoperative%20sera%20as%20a%20pancreatic%20cancer%20marker:%20a%20retrospective%20study.%20Br.%20J.%20Cancer118,%20662%E2%80%93669%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Kim, M. P. et al. Oncogenic KRAS recruits an expansive transcriptional network through mutant p53 to drive pancreatic cancer metastasis. <em>Cancer Discov.</em><strong>11</strong>, 2094–2111 (2021).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-20-1228" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8338884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33839689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20M.%20P.%20et%20al.%20Oncogenic%20KRAS%20recruits%20an%20expansive%20transcriptional%20network%20through%20mutant%20p53%20to%20drive%20pancreatic%20cancer%20metastasis.%20Cancer%20Discov.11,%202094%E2%80%932111%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Schumacher, T. N. &amp; Schreiber, R. D. Neoantigens in cancer immunotherapy. <em>Science</em><strong>348</strong>, 69–74 (2015).
</cite> [<a href="https://doi.org/10.1126/science.aaa4971" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25838375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schumacher,%20T.%20N.%20&amp;%20Schreiber,%20R.%20D.%20Neoantigens%20in%20cancer%20immunotherapy.%20Science348,%2069%E2%80%9374%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Singh, H. et al. Oncogenic drivers and therapeutic vulnerabilities in KRAS wild-type pancreatic cancer. <em>Clin. Cancer Res.</em><strong>29</strong>, 4627–4643 (2023).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-22-3930" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10795103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37463056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singh,%20H.%20et%20al.%20Oncogenic%20drivers%20and%20therapeutic%20vulnerabilities%20in%20KRAS%20wild-type%20pancreatic%20cancer.%20Clin.%20Cancer%20Res.29,%204627%E2%80%934643%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>McIntyre, C. A. et al. Distinct clinical outcomes and biological features of specific KRAS mutants in human pancreatic cancer. <em>Cancer Cell</em><strong>42</strong>, 1614–1629.e5 (2024).
</cite> [<a href="https://doi.org/10.1016/j.ccell.2024.08.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11419252/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39214094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McIntyre,%20C.%20A.%20et%20al.%20Distinct%20clinical%20outcomes%20and%20biological%20features%20of%20specific%20KRAS%20mutants%20in%20human%20pancreatic%20cancer.%20Cancer%20Cell42,%201614%E2%80%931629.e5%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Hunter, J. C. et al. Biochemical and structural analysis of common cancer-associated KRAS Mutations. <em>Mol. Cancer Res.</em><strong>13</strong>, 1325–1335 (2015).
</cite> [<a href="https://doi.org/10.1158/1541-7786.MCR-15-0203" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26037647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hunter,%20J.%20C.%20et%20al.%20Biochemical%20and%20structural%20analysis%20of%20common%20cancer-associated%20KRAS%20Mutations.%20Mol.%20Cancer%20Res.13,%201325%E2%80%931335%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Hou, P. et al. Tumor microenvironment remodeling enables bypass of oncogenic KRAS Dependency in Pancreatic Cancer. <em>Cancer Discov.</em><strong>10</strong>, 1058–1077 (2020).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-19-0597" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7334087/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32341020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hou,%20P.%20et%20al.%20Tumor%20microenvironment%20remodeling%20enables%20bypass%20of%20oncogenic%20KRAS%20Dependency%20in%20Pancreatic%20Cancer.%20Cancer%20Discov.10,%201058%E2%80%931077%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Dey, P. et al. Oncogenic KRAS-driven metabolic reprogramming in pancreatic cancer cells utilizes cytokines from the tumor microenvironment. <em>Cancer Discov.</em><strong>10</strong>, 608–625 (2020).
</cite> [<a href="https://doi.org/10.1158/2159-8290.CD-19-0297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7125035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32046984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dey,%20P.%20et%20al.%20Oncogenic%20KRAS-driven%20metabolic%20reprogramming%20in%20pancreatic%20cancer%20cells%20utilizes%20cytokines%20from%20the%20tumor%20microenvironment.%20Cancer%20Discov.10,%20608%E2%80%93625%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>McIntyre, C. A. et al. Alterations in driver genes are predictive of survival in patients with resected pancreatic ductal adenocarcinoma. <em>Cancer</em><strong>126</strong>, 3939–3949 (2020).
</cite> [<a href="https://doi.org/10.1002/cncr.33038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7424538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32573775/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McIntyre,%20C.%20A.%20et%20al.%20Alterations%20in%20driver%20genes%20are%20predictive%20of%20survival%20in%20patients%20with%20resected%20pancreatic%20ductal%20adenocarcinoma.%20Cancer126,%203939%E2%80%933949%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Zou, S. et al. Driver mutation subtypes involve with differentiated immunophenotypes influencing pancreatic cancer outcomes. <em>Cancer Lett.</em><strong>599</strong>, 217134 (2024).
</cite> [<a href="https://doi.org/10.1016/j.canlet.2024.217134" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39094824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zou,%20S.%20et%20al.%20Driver%20mutation%20subtypes%20involve%20with%20differentiated%20immunophenotypes%20influencing%20pancreatic%20cancer%20outcomes.%20Cancer%20Lett.599,%20217134%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Bailey, J. M. et al. p53 mutations cooperate with oncogenic Kras to promote adenocarcinoma from pancreatic ductal cells. <em>Oncogene</em><strong>35</strong>, 4282–4288 (2016).
</cite> [<a href="https://doi.org/10.1038/onc.2015.441" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26592447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bailey,%20J.%20M.%20et%20al.%20p53%20mutations%20cooperate%20with%20oncogenic%20Kras%20to%20promote%20adenocarcinoma%20from%20pancreatic%20ductal%20cells.%20Oncogene35,%204282%E2%80%934288%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Escobar-Hoyos, L. F. et al. Altered RNA splicing by mutant p53 activates oncogenic RAS signaling in pancreatic cancer. <em>Cancer Cell</em><strong>38</strong>, 198–211.e8 (2020).
</cite> [<a href="https://doi.org/10.1016/j.ccell.2020.05.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8028848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32559497/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Escobar-Hoyos,%20L.%20F.%20et%20al.%20Altered%20RNA%20splicing%20by%20mutant%20p53%20activates%20oncogenic%20RAS%20signaling%20in%20pancreatic%20cancer.%20Cancer%20Cell38,%20198%E2%80%93211.e8%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Soares, K. C. et al. PD-1/PD-L1 blockade together with vaccine therapy facilitates effector T-cell infiltration into pancreatic tumors. <em>J. Immunother.</em><strong>38</strong>, 1–11 (2015).
</cite> [<a href="https://doi.org/10.1097/CJI.0000000000000062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4258151/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25415283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Soares,%20K.%20C.%20et%20al.%20PD-1/PD-L1%20blockade%20together%20with%20vaccine%20therapy%20facilitates%20effector%20T-cell%20infiltration%20into%20pancreatic%20tumors.%20J.%20Immunother.38,%201%E2%80%9311%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Yamaki, S. et al. PD-L1 expression in pancreatic ductal adenocarcinoma is a poor prognostic factor in patients with high CD8(+) tumor-infiltrating lymphocytes: highly sensitive detection using phosphor-integrated dot staining. <em>Int. J. Clin. Oncol.</em><strong>22</strong>, 726–733 (2017).
</cite> [<a href="https://doi.org/10.1007/s10147-017-1112-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5533855/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28314962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yamaki,%20S.%20et%20al.%20PD-L1%20expression%20in%20pancreatic%20ductal%20adenocarcinoma%20is%20a%20poor%20prognostic%20factor%20in%20patients%20with%20high%20CD8(+)%20tumor-infiltrating%20lymphocytes:%20highly%20sensitive%20detection%20using%20phosphor-integrated%20dot%20staining.%20Int.%20J.%20Clin.%20Oncol.22,%20726%E2%80%93733%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Tessier-Cloutier, B. et al. Programmed cell death ligand 1 cut-point is associated with reduced disease specific survival in resected pancreatic ductal adenocarcinoma. <em>BMC Cancer</em><strong>17</strong>, 618 (2017).
</cite> [<a href="https://doi.org/10.1186/s12885-017-3634-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5584324/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28870260/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tessier-Cloutier,%20B.%20et%20al.%20Programmed%20cell%20death%20ligand%201%20cut-point%20is%20associated%20with%20reduced%20disease%20specific%20survival%20in%20resected%20pancreatic%20ductal%20adenocarcinoma.%20BMC%20Cancer17,%20618%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Öjlert, ÅK. et al. The immune microenvironment in non-small cell lung cancer is predictive of prognosis after surgery. <em>Mol. Oncol.</em><strong>13</strong>, 1166–1179 (2019).
</cite> [<a href="https://doi.org/10.1002/1878-0261.12475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6487716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30854794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?%C3%96jlert,%20%C3%85K.%20et%20al.%20The%20immune%20microenvironment%20in%20non-small%20cell%20lung%20cancer%20is%20predictive%20of%20prognosis%20after%20surgery.%20Mol.%20Oncol.13,%201166%E2%80%931179%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Zeng, D. et al. Tumor microenvironment characterization in gastric cancer identifies prognostic and immunotherapeutically relevant gene signatures. <em>Cancer Immunol. Res.</em><strong>7</strong>, 737–750 (2019).
</cite> [<a href="https://doi.org/10.1158/2326-6066.CIR-18-0436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30842092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng,%20D.%20et%20al.%20Tumor%20microenvironment%20characterization%20in%20gastric%20cancer%20identifies%20prognostic%20and%20immunotherapeutically%20relevant%20gene%20signatures.%20Cancer%20Immunol.%20Res.7,%20737%E2%80%93750%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Borst, J. et al. CD4(+) T cell help in cancer immunology and immunotherapy. <em>Nat. Rev. Immunol.</em><strong>18</strong>, 635–647 (2018).
</cite> [<a href="https://doi.org/10.1038/s41577-018-0044-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30057419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Borst,%20J.%20et%20al.%20CD4(+)%20T%20cell%20help%20in%20cancer%20immunology%20and%20immunotherapy.%20Nat.%20Rev.%20Immunol.18,%20635%E2%80%93647%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Kruse, B. et al. CD4(+) T cell-induced inflammatory cell death controls immune-evasive tumours. <em>Nature</em><strong>618</strong>, 1033–1040 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06199-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10307640/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37316667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kruse,%20B.%20et%20al.%20CD4(+)%20T%20cell-induced%20inflammatory%20cell%20death%20controls%20immune-evasive%20tumours.%20Nature618,%201033%E2%80%931040%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Wang, Y. et al. Neutrophils infiltrating pancreatic ductal adenocarcinoma indicate higher malignancy and worse prognosis. <em>Biochem. Biophys. Res. Commun.</em><strong>501</strong>, 313–319 (2018).
</cite> [<a href="https://doi.org/10.1016/j.bbrc.2018.05.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29738769/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Y.%20et%20al.%20Neutrophils%20infiltrating%20pancreatic%20ductal%20adenocarcinoma%20indicate%20higher%20malignancy%20and%20worse%20prognosis.%20Biochem.%20Biophys.%20Res.%20Commun.501,%20313%E2%80%93319%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Xu, M. et al. Unveiling the role of KRAS in tumor immune microenvironment. <em>Biomed. Pharmacother.</em><strong>171</strong>, 116058 (2024).
</cite> [<a href="https://doi.org/10.1016/j.biopha.2023.116058" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38171240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20M.%20et%20al.%20Unveiling%20the%20role%20of%20KRAS%20in%20tumor%20immune%20microenvironment.%20Biomed.%20Pharmacother.171,%20116058%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Biton, J. et al. TP53, STK11, and EGFR mutations predict tumor immune profile and the response to anti-PD-1 in lung adenocarcinoma. <em>Clin. Cancer Res.</em><strong>24</strong>, 5710–5723 (2018).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-18-0163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29764856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Biton,%20J.%20et%20al.%20TP53,%20STK11,%20and%20EGFR%20mutations%20predict%20tumor%20immune%20profile%20and%20the%20response%20to%20anti-PD-1%20in%20lung%20adenocarcinoma.%20Clin.%20Cancer%20Res.24,%205710%E2%80%935723%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Zhou, Y. et al. Impact of KRAS mutation on the tumor microenvironment in colorectal cancer. <em>Int. J. Biol. Sci.</em><strong>20</strong>, 1947–1964 (2024).
</cite> [<a href="https://doi.org/10.7150/ijbs.88779" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10929194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38481800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20Y.%20et%20al.%20Impact%20of%20KRAS%20mutation%20on%20the%20tumor%20microenvironment%20in%20colorectal%20cancer.%20Int.%20J.%20Biol.%20Sci.20,%201947%E2%80%931964%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Mahadevan, K. K. et al. Elimination of oncogenic KRAS in genetic mouse models eradicates pancreatic cancer by inducing FAS-dependent apoptosis by CD8(+) T cells. <em>Dev. Cell</em><strong>58</strong>, 1562–77.e8 (2023).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2023.07.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10810082/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37625403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mahadevan,%20K.%20K.%20et%20al.%20Elimination%20of%20oncogenic%20KRAS%20in%20genetic%20mouse%20models%20eradicates%20pancreatic%20cancer%20by%20inducing%20FAS-dependent%20apoptosis%20by%20CD8(+)%20T%20cells.%20Dev.%20Cell58,%201562%E2%80%9377.e8%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Coelho, M. A. et al. Oncogenic RAS signaling promotes tumor immunoresistance by stabilizing PD-L1 mRNA. <em>Immunity</em><strong>47</strong>, 1083–1099.e6 (2017).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2017.11.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5746170/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29246442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Coelho,%20M.%20A.%20et%20al.%20Oncogenic%20RAS%20signaling%20promotes%20tumor%20immunoresistance%20by%20stabilizing%20PD-L1%20mRNA.%20Immunity47,%201083%E2%80%931099.e6%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Canon, J. et al. The clinical KRAS(G12C) inhibitor AMG 510 drives anti-tumour immunity. <em>Nature</em><strong>575</strong>, 217–223 (2019).
</cite> [<a href="https://doi.org/10.1038/s41586-019-1694-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31666701/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Canon,%20J.%20et%20al.%20The%20clinical%20KRAS(G12C)%20inhibitor%20AMG%20510%20drives%20anti-tumour%20immunity.%20Nature575,%20217%E2%80%93223%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Dong, Z. Y. et al. Potential predictive value of TP53 and KRAS mutation status for response to PD-1 blockade immunotherapy in lung adenocarcinoma. <em>Clin. Cancer Res.</em><strong>23</strong>, 3012–3024 (2017).
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-16-2554" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28039262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dong,%20Z.%20Y.%20et%20al.%20Potential%20predictive%20value%20of%20TP53%20and%20KRAS%20mutation%20status%20for%20response%20to%20PD-1%20blockade%20immunotherapy%20in%20lung%20adenocarcinoma.%20Clin.%20Cancer%20Res.23,%203012%E2%80%933024%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Budczies, J. et al. KRAS and TP53 co-mutation predicts benefit of immune checkpoint blockade in lung adenocarcinoma. <em>Br. J. Cancer</em><strong>131</strong>, 524–533 (2024).
</cite> [<a href="https://doi.org/10.1038/s41416-024-02746-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11300455/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38866964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Budczies,%20J.%20et%20al.%20KRAS%20and%20TP53%20co-mutation%20predicts%20benefit%20of%20immune%20checkpoint%20blockade%20in%20lung%20adenocarcinoma.%20Br.%20J.%20Cancer131,%20524%E2%80%93533%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Ding, W. et al. Unraveling EGFR-TKI resistance in lung cancer with high PD-L1 or TMB in EGFR-sensitive mutations. <em>Respiratory Res.</em><strong>25</strong>, 40 (2024).</cite> [<a href="https://doi.org/10.1186/s12931-023-02656-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10797755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38238740/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ding,%20W.%20et%20al.%20Unraveling%20EGFR-TKI%20resistance%20in%20lung%20cancer%20with%20high%20PD-L1%20or%20TMB%20in%20EGFR-sensitive%20mutations.%20Respiratory%20Res.25,%2040%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Van der Auwera, G. A. et al. From FastQ data to high confidence variant calls: the Genome Analysis Toolkit best practices pipeline. <em>Curr. Protoc. Bioinforma.</em><strong>43</strong>, 11.0.1–0.33 (2013).</cite> [<a href="https://doi.org/10.1002/0471250953.bi1110s43" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4243306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25431634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20der%20Auwera,%20G.%20A.%20et%20al.%20From%20FastQ%20data%20to%20high%20confidence%20variant%20calls:%20the%20Genome%20Analysis%20Toolkit%20best%20practices%20pipeline.%20Curr.%20Protoc.%20Bioinforma.43,%2011.0.1%E2%80%930.33%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Koboldt, D. C. et al. VarScan: variant detection in massively parallel sequencing of individual and pooled samples. <em>Bioinformatics</em><strong>25</strong>, 2283–2285 (2009).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp373" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2734323/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19542151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koboldt,%20D.%20C.%20et%20al.%20VarScan:%20variant%20detection%20in%20massively%20parallel%20sequencing%20of%20individual%20and%20pooled%20samples.%20Bioinformatics25,%202283%E2%80%932285%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Ye, K. et al. Pindel: a pattern growth approach to detect break points of large deletions and medium sized insertions from paired-end short reads. <em>Bioinformatics</em><strong>25</strong>, 2865–2871 (2009).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp394" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2781750/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19561018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ye,%20K.%20et%20al.%20Pindel:%20a%20pattern%20growth%20approach%20to%20detect%20break%20points%20of%20large%20deletions%20and%20medium%20sized%20insertions%20from%20paired-end%20short%20reads.%20Bioinformatics25,%202865%E2%80%932871%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Boeva, V. et al. Multi-factor data normalization enables the detection of copy number aberrations in amplicon sequencing data. <em>Bioinformatics</em><strong>30</strong>, 3443–3450 (2014).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4253825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25016581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boeva,%20V.%20et%20al.%20Multi-factor%20data%20normalization%20enables%20the%20detection%20of%20copy%20number%20aberrations%20in%20amplicon%20sequencing%20data.%20Bioinformatics30,%203443%E2%80%933450%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Roach, C. et al. Development of a companion diagnostic PD-L1 immunohistochemistry assay for pembrolizumab therapy in non-small-cell lung cancer. <em>Appl. Immunohistochem. Mol. Morphol.</em><strong>24</strong>, 392–397 (2016).
</cite> [<a href="https://doi.org/10.1097/PAI.0000000000000408" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4957959/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27333219/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roach,%20C.%20et%20al.%20Development%20of%20a%20companion%20diagnostic%20PD-L1%20immunohistochemistry%20assay%20for%20pembrolizumab%20therapy%20in%20non-small-cell%20lung%20cancer.%20Appl.%20Immunohistochem.%20Mol.%20Morphol.24,%20392%E2%80%93397%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Shen, W. et al. Sangerbox: a comprehensive, interaction-friendly clinical bioinformatics analysis platform. <em>iMeta</em><strong>1</strong>, e36 (2022).
</cite> [<a href="https://doi.org/10.1002/imt2.36" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10989974/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38868713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen,%20W.%20et%20al.%20Sangerbox:%20a%20comprehensive,%20interaction-friendly%20clinical%20bioinformatics%20analysis%20platform.%20iMeta1,%20e36%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Wang, Y. et al. RAB42 overexpression correlates with poor prognosis, immune cell infiltration and chemoresistance. <em>Front. Pharmacol.</em><strong>15</strong>, 1445170 (2024).
</cite> [<a href="https://doi.org/10.3389/fphar.2024.1445170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11294155/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39101146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Y.%20et%20al.%20RAB42%20overexpression%20correlates%20with%20poor%20prognosis,%20immune%20cell%20infiltration%20and%20chemoresistance.%20Front.%20Pharmacol.15,%201445170%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Bu, Y. et al. MSR1 in lung squamous cell carcinoma: prognostic and immunological values in pan-cancer and single-cell analyses and a cohort study. <em>Int. Immunopharmacol.</em><strong>145</strong>, 113811 (2025).
</cite> [<a href="https://doi.org/10.1016/j.intimp.2024.113811" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39667048/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bu,%20Y.%20et%20al.%20MSR1%20in%20lung%20squamous%20cell%20carcinoma:%20prognostic%20and%20immunological%20values%20in%20pan-cancer%20and%20single-cell%20analyses%20and%20a%20cohort%20study.%20Int.%20Immunopharmacol.145,%20113811%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Mayakonda, A. et al. Maftools: efficient and comprehensive analysis of somatic variants in cancer. <em>Genome Res.</em><strong>28</strong>, 1747–1756 (2018).
</cite> [<a href="https://doi.org/10.1101/gr.239244.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6211645/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30341162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mayakonda,%20A.%20et%20al.%20Maftools:%20efficient%20and%20comprehensive%20analysis%20of%20somatic%20variants%20in%20cancer.%20Genome%20Res.28,%201747%E2%80%931756%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Zeng, D. et al. IOBR: multi-omics immuno-oncology biological research to decode tumor microenvironment and signatures. <em>Front. Immunol.</em><strong>12</strong>, 687975 (2021).
</cite> [<a href="https://doi.org/10.3389/fimmu.2021.687975" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8283787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34276676/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng,%20D.%20et%20al.%20IOBR:%20multi-omics%20immuno-oncology%20biological%20research%20to%20decode%20tumor%20microenvironment%20and%20signatures.%20Front.%20Immunol.12,%20687975%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373740/bin/41698_2025_1087_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary information</a><sup> (1.8MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data supporting this study’s findings are available on request from the corresponding author. The data are not publicly available due to privacy or ethical restrictions.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from NPJ Precision Oncology are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41698-025-01087-1"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41698_2025_Article_1087.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373740/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373740/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373740%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373740/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373740/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373740/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846754/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373740/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846754/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373740/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373740/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="MaXNmcuEMZbDCnB7Tu1bbTxXSdYmc5zT6BJl5j2qQX8OUCJRosWJGbVUUkNRzD7c">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
