
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Seed protein electrophoresis in plant genetics: Commemorating the pioneering contributions of Prof. Chittaranjan Kole and team to the foundation of plant proteomics - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5311F78AEF23630511F7000D3A1630.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12254572/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Seed protein electrophoresis in plant genetics: Commemorating the pioneering contributions of Prof. Chittaranjan Kole and team to the foundation of plant proteomics">
<meta name="citation_author" content="Sarita Pandey">
<meta name="citation_author_institution" content="Department of Genetics and Plant Breeding, School of Agriculture and Allied Sciences, The Neotia University, West Bengal, India">
<meta name="citation_author" content="Shalini Tiwari">
<meta name="citation_author_institution" content="Department of Agricultural and Biosystem Engineering, South Dakota State University, Brookings, South Dakota, USA">
<meta name="citation_author" content="Poulami Bhattacharjee">
<meta name="citation_author_institution" content="Department of Genetics and Plant Breeding, School of Agriculture and Allied Sciences, The Neotia University, West Bengal, India">
<meta name="citation_author" content="Abhishek Anand">
<meta name="citation_author_institution" content="Department of Genetics and Plant Breeding, School of Agriculture and Allied Sciences, The Neotia University, West Bengal, India">
<meta name="citation_author" content="Lekha T Pazhamala">
<meta name="citation_author_institution" content="International Crops Research Institute for Semi‐Arid Tropics, Patancheru, India">
<meta name="citation_author" content="Jeshima K Yasin">
<meta name="citation_author_institution" content="Division of Genomic Resources, ICAR‐National Bureau of Plant Genetic Resources, New Delhi, India">
<meta name="citation_author" content="Roshan Kumar Yadav">
<meta name="citation_author_institution" content="Ministry of Forests and Environment Government of Nepal Singhdurbar, Kathmandu, Nepal">
<meta name="citation_publication_date" content="2025 Jul 11">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70027">
<meta name="citation_doi" content="10.1002/tpg2.70027">
<meta name="citation_pmid" content="40646713">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12254572/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12254572/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12254572/pdf/TPG2-18-e70027.pdf">
<meta name="description" content="Despite the rapid progress of proteomics in human and other model organisms, plant proteomics has advanced at a comparatively slower pace. This review aims to highlight the pioneering work on seed protein markers detected by employing gel ...">
<meta name="og:title" content="Seed protein electrophoresis in plant genetics: Commemorating the pioneering contributions of Prof. Chittaranjan Kole and team to the foundation of plant proteomics">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Despite the rapid progress of proteomics in human and other model organisms, plant proteomics has advanced at a comparatively slower pace. This review aims to highlight the pioneering work on seed protein markers detected by employing gel ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12254572/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12254572">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70027"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70027.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12254572%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12254572/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12254572/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12254572/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70027" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Jul 11;18(3):e70027. doi: <a href="https://doi.org/10.1002/tpg2.70027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70027</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Seed protein electrophoresis in plant genetics: Commemorating the pioneering contributions of Prof. Chittaranjan Kole and team to the foundation of plant proteomics</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pandey%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Sarita Pandey</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Sarita Pandey</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Genetics and Plant Breeding, School of Agriculture and Allied Sciences, The Neotia University, West Bengal, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pandey%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sarita Pandey</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tiwari%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Shalini Tiwari</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Shalini Tiwari</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Agricultural and Biosystem Engineering, South Dakota State University, Brookings, South Dakota, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tiwari%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shalini Tiwari</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bhattacharjee%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Poulami Bhattacharjee</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Poulami Bhattacharjee</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Genetics and Plant Breeding, School of Agriculture and Allied Sciences, The Neotia University, West Bengal, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bhattacharjee%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Poulami Bhattacharjee</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anand%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Abhishek Anand</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Abhishek Anand</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Genetics and Plant Breeding, School of Agriculture and Allied Sciences, The Neotia University, West Bengal, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Anand%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Abhishek Anand</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pazhamala%20LT%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Lekha T Pazhamala</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Lekha T Pazhamala</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>International Crops Research Institute for Semi‐Arid Tropics, Patancheru, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pazhamala%20LT%22%5BAuthor%5D" class="usa-link"><span class="name western">Lekha T Pazhamala</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yasin%20JK%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jeshima K Yasin</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jeshima K Yasin</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Division of Genomic Resources, ICAR‐National Bureau of Plant Genetic Resources, New Delhi, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yasin%20JK%22%5BAuthor%5D" class="usa-link"><span class="name western">Jeshima K Yasin</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yadav%20RK%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Roshan Kumar Yadav</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Roshan Kumar Yadav</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Ministry of Forests and Environment Government of Nepal Singhdurbar, Kathmandu, Nepal</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yadav%20RK%22%5BAuthor%5D" class="usa-link"><span class="name western">Roshan Kumar Yadav</span></a>
</div>
</div>
<sup>5,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270027-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Genetics and Plant Breeding, School of Agriculture and Allied Sciences, The Neotia University, West Bengal, India</div>
<div id="tpg270027-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Agricultural and Biosystem Engineering, South Dakota State University, Brookings, South Dakota, USA</div>
<div id="tpg270027-aff-0003">
<sup>
<sup>3</sup>
</sup>International Crops Research Institute for Semi‐Arid Tropics, Patancheru, India</div>
<div id="tpg270027-aff-0004">
<sup>
<sup>4</sup>
</sup>Division of Genomic Resources, ICAR‐National Bureau of Plant Genetic Resources, New Delhi, India</div>
<div id="tpg270027-aff-0005">
<sup>
<sup>5</sup>
</sup>Ministry of Forests and Environment Government of Nepal Singhdurbar, Kathmandu, Nepal</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Roshan Kumar Yadav, Ministry of Forests and Environment Government of Nepal Singhdurbar, Kathmandu, Nepal. Email: <span>fsroshan@gmail.com</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Feb 27; Received 2024 Nov 28; Accepted 2025 Mar 3; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12254572  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40646713/" class="usa-link">40646713</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Despite the rapid progress of proteomics in human and other model organisms, plant proteomics has advanced at a comparatively slower pace. This review aims to highlight the pioneering work on seed protein markers detected by employing gel electrophoresis primarily by a team of Indian scientists that paved the way for elucidation of intervarietal and interspecific variation, evolution, and phylogenetic relationship of species and their association with resistance to pest and diseases. Far from being replaced, gel electrophoresis remains as an excellent supporting and different approach, offering a pathway to a more profound visualization and understanding of the cell proteome. This review focuses on how, from a historical standpoint, gel electrophoresis has significantly contributed to plant proteomics and other biological research. Acknowledging the pioneering work on seed storage proteins, this review serves as both a congratulatory gesture and a tribute to the eminent scientist Prof. Chittaranjan Kole and his team who pioneered the strategy of seed protein electrophoresis in crop biology research. Their findings, both directly and indirectly, have proven invaluable, particularly for those who ventured into proteomics without easy reach to sophisticated and expensive instruments/equipment to pursue DNA‐based genomics research. This gel electrophoresis‐based plant proteomics review includes the evolution of gel‐based proteomics, their contribution to crop biology research, and future directions. It stands not only as a retrospective analysis but also as a testament to the enduring significance of gel electrophoresis in shaping the landscape of crop proteomics.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270027-list-10001" class="list" style="list-style-type:disc">
<li><p>Gel electrophoresis continues to play an essential role in plant proteomics and broader biological research, offering valuable insights despite the slower progress compared to human and model organism proteomics.</p></li>
<li><p>The pioneering work on seed protein markers, led by Prof. Chittaranjan Kole and his team, paved the way for understanding intervarietal and interspecific variations, species evolution, and resistance to major pests and diseases.</p></li>
<li><p>This review explores the evolution, contributions, and future directions of gel‐based proteomics, emphasizing its enduring significance in the field. It highlights how gel electrophoresis has remained a critical tool, especially for researchers without access to advanced and costly genomic equipment, shaping the landscape of crop proteomics and plant biology.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270027-dl-0001">
<dt>2D‐DIGE</dt>
<dd><p>two‐dimensional differential gel electrophoresis</p></dd>
<dt>2‐DE</dt>
<dd><p>two‐dimensional gel electrophoresis</p></dd>
<dt>CLS</dt>
<dd><p>Cercospora leaf spot</p></dd>
<dt>CNE</dt>
<dd><p>clear native electrophoresis</p></dd>
<dt>DIA</dt>
<dd><p>data‐independent acquisition</p></dd>
<dt>DIGE</dt>
<dd><p>differential gel electrophoresis</p></dd>
<dt>GLH</dt>
<dd><p>green leafhopper</p></dd>
<dt>MYMV</dt>
<dd><p>Mungbean yellow mosaic virus</p></dd>
<dt>SDS‐PAGE</dt>
<dd><p>sodium dodecyl sulfate‐polyacrylamide gel electrophoresis</p></dd>
</dl>
<section id="tpg270027-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>The role of protein markers in crop improvement has gained increasing significance in recent years, offering crucial insights into the genetic, physiological, and biochemical attributes of plants. As global agricultural challenges intensify due to climate change, soil degradation, and population growth, researchers seek innovative strategies to enhance crop yield, nutritional content, and resilience to environmental stressors. Protein markers have emerged as essential tools in this pursuit, enabling scientists to decipher molecular mechanisms governing plant development and stress responses. By leveraging protein markers, researchers can identify desirable traits, optimize breeding strategies, and accelerate the development of improved crop varieties (Eldakak et al., <a href="#tpg270027-bib-0024" class="usa-link" aria-describedby="tpg270027-bib-0024">2013</a>).</p>
<p>Among the diverse applications of protein markers, seed proteins play a crucial role in both plant growth and human nutrition. Seed proteins not only provide essential nutrients for the developing plant embryo but also serve as valuable dietary protein sources. The composition of seed proteins varies among species, with key classes including albumins, globulins, prolamins, and glutelins (Rasheed et al., <a href="#tpg270027-bib-0109" class="usa-link" aria-describedby="tpg270027-bib-0109">2020</a>). Advances in proteomic technologies have significantly enhanced our understanding of these proteins, particularly through gel‐based proteomics. This field has transformed seed protein research, enabling the identification of key proteins linked to stress tolerance, disease resistance, and nutrient efficiency.</p>
<p>Prof. Chittaranjan Kole, a distinguished plant geneticist, has made significant contributions to the field of crop improvement, particularly through his work in genomics and molecular breeding. His research has emphasized the integration of cutting‐edge molecular techniques to enhance agricultural productivity and address food security challenges. Building upon such foundational work, this review aims to provide a comprehensive examination of gel electrophoresis in crop proteomics research, specifically in the context of seed proteins.</p>
<p>By exploring the evolution of proteomic techniques, with a particular emphasis on two‐dimensional gel electrophoresis (2‐DE), this review seeks to highlight how advancements in proteomics have expanded our understanding of seed proteins and their applications in agriculture. Given the growing need for sustainable and high‐yielding crops, a deeper comprehension of seed protein composition and function is essential for breeding resilient varieties. Ultimately, this review underscores the transformative potential of protein markers in modern crop improvement, offering new avenues for enhancing agricultural sustainability and food security.</p>
<section id="tpg270027-sec-0020"><h3 class="pmc_sec_title">1.1. Role of seed proteins in plant growth and development</h3>
<p>Seed proteins are vital for plant growth and development, serving as crucial sources of amino acids and energy for the developing embryo. They are broadly categorized into several types, each with distinct functions and characteristics (Khalid, Hameed, &amp; Tahir, <a href="#tpg270027-bib-0045" class="usa-link" aria-describedby="tpg270027-bib-0045">2023</a>). For example, storage proteins that accumulate in seed tubers, or other plant storage organs, serve as a reserve of amino acids for germination and early seedling growth. Types of storage proteins include albumins, globulins, prolamins, and glutelins (Fujiwara et al., <a href="#tpg270027-bib-0025" class="usa-link" aria-describedby="tpg270027-bib-0025">2002</a>). Notable examples include soybean glycinin and conglycinin (globulins), wheat glutenin and gliadin (prolamins), and rice glutelins. Similarly, structural proteins provide support and rigidity to plant cells and tissues. They are important for maintaining the shape and integrity of plant structures. Cellulose synthases (involved in cellulose formation), tubulins and actins (cytoskeletal proteins), and extensins (involved in cell wall reinforcement) are some known structural proteins. Proteins that function as enzymes also play crucial roles in plants in various metabolic pathways, including photosynthesis, respiration, and biosynthesis of secondary metabolites (Quinn et al., <a href="#tpg270027-bib-0107" class="usa-link" aria-describedby="tpg270027-bib-0107">2024</a>), for example, Rubisco (involved in photosynthesis), amylase (involved in starch breakdown), and catalase (involved in detoxifying reactive oxygen species) (Shewry et al., <a href="#tpg270027-bib-0119" class="usa-link" aria-describedby="tpg270027-bib-0119">1995</a>). Defense proteins such as pathogenesis‐related (PR) proteins, protease inhibitors, and antimicrobial peptides protect plants against pathogens, herbivores, and environmental stresses. They are part of the plant's immune system. Transport proteins facilitate the movement of substances, such as ions and small molecules, within the plant (Oliveira et al., <a href="#tpg270027-bib-0084" class="usa-link" aria-describedby="tpg270027-bib-0084">2022</a>). They are involved in nutrient uptake, translocation, and distribution. Examples include aquaporins (which facilitate water transport), ion channels, and nutrient transporters. Signal transduction proteins relay signals within the plant, regulating various physiological processes in response to external stimuli. Key examples are receptor kinases, G‐proteins, and transcription factors involved in plant hormone signaling. Photosynthetic proteins are involved in the process of photosynthesis, capturing light energy and converting it into chemical energy (Oliveira et al., <a href="#tpg270027-bib-0084" class="usa-link" aria-describedby="tpg270027-bib-0084">2022</a>). Examples include chlorophyll‐binding proteins, photosystem proteins, and enzymes of the Calvin cycle. Regulatory proteins control gene expression and coordinate various cellular processes. They play a role in plant development and responses to environmental stimuli. For example, transcription factors, protein kinases, and other proteins involved in gene regulation. Some plant proteins can trigger allergic reactions in sensitive individuals (Oliveira et al., <a href="#tpg270027-bib-0084" class="usa-link" aria-describedby="tpg270027-bib-0084">2022</a>). Understanding allergenic proteins is crucial for food safety. Proteins in nuts (e.g., peanuts and tree nuts), certain fruits, and grains (e.g., wheat gluten) can be allergenic. Seed maturation proteins are expressed during seed development and are involved in processes related to seed maturation and desiccation tolerance. For instance, late embryogenesis abundant proteins also known as LEA proteins, seed storage proteins, and oleosins (Rasheed et al., <a href="#tpg270027-bib-0109" class="usa-link" aria-describedby="tpg270027-bib-0109">2020</a>).</p></section><section id="tpg270027-sec-0030"><h3 class="pmc_sec_title">1.2. Evolution of proteomics—Biomarker discovery to gel‐based proteomics</h3>
<p>The timeline of developments in gel electrophoresis for protein study began in the 1960s with the introduction of horizontal slab gels, which simplified protein separation based on size. Sodium dodecyl sulfate‐polyacrylamide gel electrophoresis (SDS‐PAGE), the predominant gel electrophoretic technique for protein analysis, relies on the separation of proteins by size and assists in the determination of their relative molecular mass (Lan et al., <a href="#tpg270027-bib-0059" class="usa-link" aria-describedby="tpg270027-bib-0059">2024</a>) (<a href="#tpg270027-supinfo-0001" class="usa-link">Supporting information</a>). This method hinges on the action of SDS, an anionic detergent that binds tightly to proteins, inducing their denaturation and linearization into polypeptide chains. Typically, one SDS molecule associates with every two amino acids, with a constant ratio of approximately 1.4 g of SDS/g of protein. The resulting protein‐SDS complexes carry a net negative charge, propelling them toward the anode during electrophoresis (Otzen et al., <a href="#tpg270027-bib-0088" class="usa-link" aria-describedby="tpg270027-bib-0088">2022</a>).</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270027-sec-0040"><h4 class="pmc_sec_title">Core Ideas</h4>
<ul id="tpg270027-list-0001" class="list" style="list-style-type:disc">
<li><p>Gel electrophoresis continues to play an essential role in plant proteomics and broader biological research, offering valuable insights despite the slower progress compared to human and model organism proteomics.</p></li>
<li><p>The pioneering work on seed protein markers, led by Prof. Chittaranjan Kole and his team, paved the way for understanding intervarietal and interspecific variations, species evolution, and resistance to major pests and diseases.</p></li>
<li><p>This review explores the evolution, contributions, and future directions of gel‐based proteomics, emphasizing its enduring significance in the field. It highlights how gel electrophoresis has remained a critical tool, especially for researchers without access to advanced and costly genomic equipment, shaping the landscape of crop proteomics and plant biology.</p></li>
</ul></section></section><p>The advent of gel‐based proteomics marked a revolutionary shift in the analysis of seed proteins. Techniques named 2‐DE have been instrumental in separating and identifying complex protein mixtures based on their isoelectric point and molecular weight (Oliveira et al., <a href="#tpg270027-bib-0084" class="usa-link" aria-describedby="tpg270027-bib-0084">2022</a>). This groundbreaking technique combined isoelectric focusing with SDS‐PAGE, allowing for the high‐resolution separation of complex protein mixtures. This method, developed in the early 1980s, allowed scientists to visualize and compare protein profiles from various seeds, uncovering critical details about protein composition and function (Dunn &amp; Burghes, <a href="#tpg270027-bib-0021" class="usa-link" aria-describedby="tpg270027-bib-0021">1986</a>; Oliveira et al., <a href="#tpg270027-bib-0084" class="usa-link" aria-describedby="tpg270027-bib-0084">2022</a>).</p>
<p>Pioneers in gel‐based proteomics have made significant contributions by utilizing 2‐DE to characterize seed storage proteins and identify variations related to different growth conditions and developmental stages. This has led to enhanced understanding of protein functions and interactions, which is crucial for developing crops with improved nutritional content, stress tolerance, and overall quality (Oliveira et al., <a href="#tpg270027-bib-0084" class="usa-link" aria-describedby="tpg270027-bib-0084">2022</a>). The 1990s introduced advancements like differential gel electrophoresis (DIGE), which enabled the simultaneous comparison of multiple samples. The 2000s further advanced with the integration of advanced imaging technologies and mass spectrometry (MS) for detailed protein identification (Hirano &amp; Shirakawa, <a href="#tpg270027-bib-0035" class="usa-link" aria-describedby="tpg270027-bib-0035">2022</a>).</p>
<p>In the last 30 years, the molecular biology research landscape has been significantly influenced by omics approaches, with a notable emphasis on proteomics (Wang et al., <a href="#tpg270027-bib-0133" class="usa-link" aria-describedby="tpg270027-bib-0133">2024</a>). This trend aligns with the completion of numerous genome sequences, the widespread adoption of next generation sequencing for transcript analysis, advancements in MS and associated equipment, and the development of bioinformatics tools and pipelines (Tebani et al., <a href="#tpg270027-bib-0127" class="usa-link" aria-describedby="tpg270027-bib-0127">2016</a>; Wang et al., <a href="#tpg270027-bib-0133" class="usa-link" aria-describedby="tpg270027-bib-0133">2024</a>). From a methodological perspective, proteomics has evolved swiftly from the classical 2DE‐MS (first generation) to isobaric or isotopic labeling strategies (Abdallah et al., <a href="#tpg270027-bib-0001" class="usa-link" aria-describedby="tpg270027-bib-0001">2012</a>; Wang et al., <a href="#tpg270027-bib-0133" class="usa-link" aria-describedby="tpg270027-bib-0133">2024</a>) (second generation), then to shotgun or gel‐free/label‐free techniques (third generation), and finally, to targeted, mass‐western, or selective reaction monitoring/multiple reaction monitoring approaches (fourth generation). A noteworthy advancement in proteomics methodology is observed in shotgun proteomics, which is currently undergoing a paradigm shift with the adoption of data‐independent acquisition (DIA). This approach depends on the presence of proteotypic, specific protein species, and the development of spectral libraries (Wang et al., <a href="#tpg270027-bib-0133" class="usa-link" aria-describedby="tpg270027-bib-0133">2024</a>).</p>
<p>More recent developments include the use of DIA techniques, enhancing the quantitative analysis and precision of protein profiling. Each of these milestones has significantly enhanced our ability to study and understand proteins in greater detail. In Figure <a href="#tpg270027-fig-0001" class="usa-link">1</a>, we highlighted the key milestones in the development of gel electrophoresis systems for protein analysis, illustrating the evolution from early techniques to contemporary advancements.</p>
<figure class="fig xbox font-sm" id="tpg270027-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ec11/12254572/a194de888913/TPG2-18-e70027-g001.jpg" loading="lazy" id="jats-graphic-1" height="610" width="603" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270027-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The flowchart visualizes the key milestones in the development of gel electrophoresis systems for protein study, showing the progression from early techniques to modern advancements.</p></figcaption></figure></section></section><section id="tpg270027-sec-0050"><h2 class="pmc_sec_title">2. PLANT PROTEOMIC STUDIES BASED ON GEL ELECTROPHORESIS</h2>
<p>The essential role of molecules in supporting life has been documented since the early stages of biological research. The term “protein,” as coined by Berzelius in 1838, originates from the Greek word “proteios,” meaning “the first rank” (Cristea et al., <a href="#tpg270027-bib-0015" class="usa-link" aria-describedby="tpg270027-bib-0015">2004</a>). In the early 1990s, Prof. Chittaranjan Kole conceptualized the importance of proteins analysis in plant biology research and formulated a research project on “Genetic Characterization of Electrophoretic Banding Patterns of Seed Proteins in Mungbean: Molecular Implications and Seed Protein Improvement” that he operated from June 01, 1994 to May 31, 1997 with funding from the Indian Council of Agricultural Research assisted by his then‐colleague, Mr. B.S. Naik. However, describing the overall protein content of a cell in terms of localization, interactions, post‐translational modifications, and turnover at a specific time, the “proteome” was described by Marc Wilkins in 1996 as the “PROTein complement of a genOME” (Wilkins et al., <a href="#tpg270027-bib-0136" class="usa-link" aria-describedby="tpg270027-bib-0136">1996</a>). Proteomics involves the comprehensive characterization of the proteome, encompassing the expression, structure, functions, interactions, and modifications of proteins at any stage (Mani et al., <a href="#tpg270027-bib-0066" class="usa-link" aria-describedby="tpg270027-bib-0066">2022</a>). The proteome undergoes fluctuations over time, from cell to cell, and in response to external stimuli. Eukaryotic cell proteomics is particularly complex due to post‐translational modifications occurring at various sites through numerous pathways (Krishna &amp; Wold, <a href="#tpg270027-bib-0055" class="usa-link" aria-describedby="tpg270027-bib-0055">2013</a>). Proteomics is a crucial approach for understanding gene function, though it is considerably more complex than genomics (Lander et al., <a href="#tpg270027-bib-0060" class="usa-link" aria-describedby="tpg270027-bib-0060">2006</a>; Mani et al., <a href="#tpg270027-bib-0066" class="usa-link" aria-describedby="tpg270027-bib-0066">2022</a>). Variations in gene expression levels can be identified by analyzing the transcriptome or proteome, allowing comparison between two cellular states. For large‐scale examination of the entire transcriptome, microarray chips have been developed. Proteins serve as executors of biological functions, and their levels are influenced not only by corresponding mRNA but also by host translational regulation and control mechanisms. Consequently, proteomics is often regarded as the most relevant dataset for characterizing a biological system (Cox &amp; Mann, <a href="#tpg270027-bib-0014" class="usa-link" aria-describedby="tpg270027-bib-0014">2007</a>; Mani et al., <a href="#tpg270027-bib-0066" class="usa-link" aria-describedby="tpg270027-bib-0066">2022</a>).</p>
<section id="tpg270027-sec-0060"><h3 class="pmc_sec_title">2.1. Methods for comprehensive analysis of protein</h3>
<p>This section explores the diverse applications of proteomic techniques in enhancing crop improvement (Figure <a href="#tpg270027-fig-0002" class="usa-link">2</a>). By analyzing protein/seed protein expression profiles, modifications, and interactions, researchers can gain insights into stress responses, disease resistance, and nutrient utilization. Traditional protein purification techniques rely on chromatography methods such as ion exchange chromatography, size exclusion chromatography, and affinity chromatography (Hage et al., <a href="#tpg270027-bib-0033" class="usa-link" aria-describedby="tpg270027-bib-0033">2012</a>; Jungbauer &amp; Hahn, <a href="#tpg270027-bib-0042" class="usa-link" aria-describedby="tpg270027-bib-0042">2009</a>; Voedisch &amp; Thie, <a href="#tpg270027-bib-0132" class="usa-link" aria-describedby="tpg270027-bib-0132">2010</a>). To analyze specific proteins, methods like enzyme‐linked immunosorbent assay and western blotting are commonly used (Mani et al., <a href="#tpg270027-bib-0066" class="usa-link" aria-describedby="tpg270027-bib-0066">2022</a>). Although these methods can target a few specific proteins, they are limited in determining overall protein expression levels (Kurien &amp; Scofield, <a href="#tpg270027-bib-0057" class="usa-link" aria-describedby="tpg270027-bib-0057">2006</a>; Lequin, <a href="#tpg270027-bib-0061" class="usa-link" aria-describedby="tpg270027-bib-0061">2005</a>). Techniques such as SDS‐PAGE, 2‐DE, and two‐dimensional differential gel electrophoresis (2D‐DIGE) are employed to separate complex protein samples (Dunn &amp; Burghes, <a href="#tpg270027-bib-0021" class="usa-link" aria-describedby="tpg270027-bib-0021">1986</a>; Issaq &amp; Veenstra, <a href="#tpg270027-bib-0038" class="usa-link" aria-describedby="tpg270027-bib-0038">2008</a>; Marouga et al., <a href="#tpg270027-bib-0068" class="usa-link" aria-describedby="tpg270027-bib-0068">2005</a>). In the early and late 1990s, advanced versions of bi‐dimensional gel electrophoresis emerged, expanding its analytical potential. One such advancement, blue‐native polyacrylamide gel electrophoresis, enabled detailed analysis of protein complexes and their components (Ebhardt et al., <a href="#tpg270027-bib-0023" class="usa-link" aria-describedby="tpg270027-bib-0023">2015</a>; Knezevic et al., <a href="#tpg270027-bib-0048" class="usa-link" aria-describedby="tpg270027-bib-0048">2001</a>; Rosenberg &amp; Utz, <a href="#tpg270027-bib-0113" class="usa-link" aria-describedby="tpg270027-bib-0113">2015</a>). Simultaneously, fluorescence DIGE surfaced, streamlining experimental setups, elevating reproducibility and sensitivity, and reducing variability between replicates (Sanchez‐Carbayo et al., <a href="#tpg270027-bib-0116" class="usa-link" aria-describedby="tpg270027-bib-0116">2006</a>). These methodologies can be effectively combined for comprehensive analyses (Delehanty &amp; Ligler, <a href="#tpg270027-bib-0017" class="usa-link" aria-describedby="tpg270027-bib-0017">2002</a>). The 2D‐DIGE technique was initially embraced in human and model organism research, and this technique was later adopted by the plant community (Popescu et al., <a href="#tpg270027-bib-0101" class="usa-link" aria-describedby="tpg270027-bib-0101">2007</a>; Tibes et al., <a href="#tpg270027-bib-0129" class="usa-link" aria-describedby="tpg270027-bib-0129">2006</a>; Ummanni et al., <a href="#tpg270027-bib-0131" class="usa-link" aria-describedby="tpg270027-bib-0131">2014</a>; Zhu et al., <a href="#tpg270027-bib-0140" class="usa-link" aria-describedby="tpg270027-bib-0140">2001</a>). Additionally, two less commonly employed variants of 2‐DE have been reported, mainly in membrane protein complex studies. Clear native electrophoresis (CNE) and high‐resolution CNE are methods that exclude dye to prevent interference with fluorescence detection or enzymatic assays. Though occasionally applied in plant studies (Jovanovic et al., <a href="#tpg270027-bib-0040" class="usa-link" aria-describedby="tpg270027-bib-0040">2007</a>; Sadia et al., <a href="#tpg270027-bib-0114" class="usa-link" aria-describedby="tpg270027-bib-0114">2009</a>), these techniques are not widely used. Protein microarrays, or protein chips, have been developed for rapid and high‐throughput expression analysis. Advanced proteomics techniques, including MS, allow for the analysis of complex protein mixtures with enhanced sensitivity (Yates, <a href="#tpg270027-bib-0137" class="usa-link" aria-describedby="tpg270027-bib-0137">2011</a>). Additionally, Edman degradation is utilized for sequencing amino acids of specific proteins (Smith, <a href="#tpg270027-bib-0124" class="usa-link" aria-describedby="tpg270027-bib-0124">2001</a>). Quantitative proteomics methods, such as isotope‐coded affinity tag labeling, stable isotope labeling with amino acids in cell culture, and isobaric tags for relative and absolute quantitation, have recently been introduced (Ong et al., <a href="#tpg270027-bib-0085" class="usa-link" aria-describedby="tpg270027-bib-0085">2006</a>; Shiio &amp; Aebersold, <a href="#tpg270027-bib-0120" class="usa-link" aria-describedby="tpg270027-bib-0120">2006</a>; Wiese et al., <a href="#tpg270027-bib-0135" class="usa-link" aria-describedby="tpg270027-bib-0135">2007</a>). X‐ray crystallography and nuclear magnetic resonance spectroscopy are two essential high‐throughput techniques that reveal a protein's three‐dimensional structure, aiding in the understanding of its biological function (Smyth &amp; Martin, <a href="#tpg270027-bib-0125" class="usa-link" aria-describedby="tpg270027-bib-0125">2000</a>; Wiese et al., <a href="#tpg270027-bib-0135" class="usa-link" aria-describedby="tpg270027-bib-0135">2007</a>). Proteome analysis offers a comprehensive view of cellular structural and functional information, as well as insights into cellular responses to various stressors and drugs, through single or multiple proteomics techniques.</p>
<figure class="fig xbox font-sm" id="tpg270027-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ec11/12254572/7a5d9536af9f/TPG2-18-e70027-g003.jpg" loading="lazy" id="jats-graphic-3" height="663" width="703" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270027-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Various applications of proteomic methodologies in crop improvement. ELISA, enzyme‐linked immunosorbent assay; ICAT, isotope‐coded affinity tag; iTRAQ, isobaric tags for relative and absolute quantitation; NMR, nuclear magnetic resonance; SDS‐PAGE, sodium dodecyl sulfate‐polyacrylamide gel electrophoresis; SILAC, stable isotope labeling with amino acids in cell culture.</p></figcaption></figure></section><section id="tpg270027-sec-0070"><h3 class="pmc_sec_title">2.2. Contribution of pioneering work on seed proteins by Prof. Chittaranjan Kole and his co‐workers</h3>
<p>The exploration of seed proteins has been instrumental in advancing agricultural science and crop improvement, largely due to the pioneering efforts of researchers who laid the foundation for understanding and utilizing these crucial biomolecules (Chavhan et al., <a href="#tpg270027-bib-0010" class="usa-link" aria-describedby="tpg270027-bib-0010">2024</a>). The significance of researching seed storage proteins in agriculture and crop enhancement is well‐established. Recognizing the transformative potential of proteomics, Prof. Chittaranjan Kole and his team have been instrumental in unraveling enigmatic aspects at the protein level in crops essential for ensuring food security and promoting sustainable agriculture (Figure <a href="#tpg270027-fig-0003" class="usa-link">3</a>). In this pursuit, they played a key role and made significant contributions to studies related to seed storage proteins in various crops, such as mungbean (<em>Vigna radiata</em>) (Chand &amp; Kole, <a href="#tpg270027-bib-0008" class="usa-link" aria-describedby="tpg270027-bib-0008">2002</a>; Naik, <a href="#tpg270027-bib-0076" class="usa-link" aria-describedby="tpg270027-bib-0076">1998</a>; Naik et al., <a href="#tpg270027-bib-0079" class="usa-link" aria-describedby="tpg270027-bib-0079">2000</a>; Naik &amp; Kole, <a href="#tpg270027-bib-0078" class="usa-link" aria-describedby="tpg270027-bib-0078">2002</a>), urdbean (<em>Vigna mungo</em>) (Kole et al., <a href="#tpg270027-bib-0050" class="usa-link" aria-describedby="tpg270027-bib-0050">2005</a>, <a href="#tpg270027-bib-0051" class="usa-link" aria-describedby="tpg270027-bib-0051">2006</a>), pigeonpea (<em>Cajanus cajan</em>) (Panigrahi et al., <a href="#tpg270027-bib-0096" class="usa-link" aria-describedby="tpg270027-bib-0096">2001a</a>, <a href="#tpg270027-bib-0097" class="usa-link" aria-describedby="tpg270027-bib-0097">2001b</a>, <a href="#tpg270027-bib-0098" class="usa-link" aria-describedby="tpg270027-bib-0098">2002</a>), and rice (<em>Oryza sativa</em>) (Padmavathi et al., <a href="#tpg270027-bib-0089" class="usa-link" aria-describedby="tpg270027-bib-0089">1999</a>, <a href="#tpg270027-bib-0090" class="usa-link" aria-describedby="tpg270027-bib-0090">2001</a>, <a href="#tpg270027-bib-0091" class="usa-link" aria-describedby="tpg270027-bib-0091">2002</a>). Until 2012, gel electrophoresis, specifically the 2‐DE version, has remained the predominant and nearly exclusive platform in plant proteomics research. Today, it remains widely employed on its own or alongside other platforms, ranking among the most commonly used methods for plant proteome analysis. Since its introduction in the early 2000s, this technique has significantly advanced our molecular understanding of plant biology, offering promising applications in agriculture and environmental sciences. Prof. Kole's Laboratory of Molecular Biology and Biotechnology (LMBB) in the Odisha University of Agriculture and Technology has been at the forefront of seed protein‐based proteomics, with a primary focus on mungbean, urdbean, pigeonpea, and rice. Therefore, further, we discussed the role of seed proteins in various aspects of plant biology, as researched in Prof. Kole's lab.</p>
<figure class="fig xbox font-sm" id="tpg270027-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/ec11/12254572/849dc86dd203/TPG2-18-e70027-g002.jpg" loading="lazy" id="jats-graphic-5" height="655" width="683" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270027-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The figure illustrates the early gel‐based research in different crops by Prof. Kole and team. (a) Seed globulin of 24 urdbean derived from sodium dodecyl sulfate‐polyacrylamide gel electrophoresis (SDS‐PAGE). (b) Electropherogram of polypeptide banding pattern of 8 genotypes of <em>Cajanus cajan</em> and <em>C. cajanifolius</em>. (c) Seed protein profile of 12 genotypes studied for resistance to green leafhopper (Padmavathi et al., <a href="#tpg270027-bib-0089" class="usa-link" aria-describedby="tpg270027-bib-0089">1999</a>).</p></figcaption></figure></section></section><section id="tpg270027-sec-0080"><h2 class="pmc_sec_title">3. PROTEIN PROFILING/MARKERS FOR ASSESSING GENETIC DIVERSITY</h2>
<p>The genetic diversity enables breeders to enhance crop resilience to environmental stresses, increase yield potential, and enhance nutritional profiles (Hafeez et al., <a href="#tpg270027-bib-0032" class="usa-link" aria-describedby="tpg270027-bib-0032">2023</a>). Moreover, diverse crop varieties offer adaptability to various agro‐ecological conditions and consumer preferences, thereby bolstering food security and sustainability. Particularly in the case of mungbean, urdbean, cowpea, and <em>Vigna sublobata</em>, earlier studies unveiled extensive diversity both between and within species, leading to profound insights. Sahai and Rana (<a href="#tpg270027-bib-0115" class="usa-link" aria-describedby="tpg270027-bib-0115">1977</a>) observed almost congruous protein profiles of two varieties of mungbean and completely similar banding patterns of two varieties of urdbean. In mungbean, Thakare et al. (<a href="#tpg270027-bib-0141" class="usa-link" aria-describedby="tpg270027-bib-0141">1988</a>) found uniform banding pattern of vicilin protein of four cultivars. They, however, observed two out of 86 accessions of urdbean to differ from the rest. The banding patterns of three accessions of <em>V. sublobata</em> were altogether different. Gomathinayagam and Ramaswamy (<a href="#tpg270027-bib-0029" class="usa-link" aria-describedby="tpg270027-bib-0029">1994</a>) found all the nine bands of the two varieties of cowpea to be uniform. Prasadi et al. (<a href="#tpg270027-bib-0103" class="usa-link" aria-describedby="tpg270027-bib-0103">1996</a>), while studying genetic variation in six varieties of cowpea in relation to insect resistance, observed the major variation in protein bands of molecular weight above 24 kD. Later on, Prof. Kole's research, particularly on mungbean and rice proteomics, has revealed the presence of wide diversity between and within species leading to significant discoveries. In an electrophoretic analysis study of seed albumins and globulins from six <em>Vigna</em> species and four synthetic allopolyploids, 20 albumin and 16 globulin polypeptides were identified. This study supported the hypothesis that <em>Vigna glabrescens</em> had originated through natural allo‐polyploidization involving <em>V. radiata</em> and <em>Vigna umbellata</em> (Kole &amp; Panigrahi, <a href="#tpg270027-bib-0053" class="usa-link" aria-describedby="tpg270027-bib-0053">2001</a>). Variability studies based on gel electrophoresis by Prof. Kole and his team in 37 local landraces of mungbean collected from different parts of Odisha and selections from them revealed a significant association of protein content with early flowering, pod length, pod number, seed number, and yield per plant (Naik et al., <a href="#tpg270027-bib-0079" class="usa-link" aria-describedby="tpg270027-bib-0079">2000</a>). The utilization of gel electrophoresis for basic genetic diversity analysis in cereals and legumes has been instrumental in laying the groundwork for modern research in proteomics. By visualizing DNA or protein fragments based on their size and charge, gel electrophoresis enables researchers to assess the genetic diversity present within different crop populations (Hameed et al., <a href="#tpg270027-bib-0034" class="usa-link" aria-describedby="tpg270027-bib-0034">2012</a>). This foundational knowledge provides a basis for subsequent proteomic investigations, allowing researchers to delve deeper into the functional aspects of genetic diversity at the protein level. Proteomic techniques have revolutionized the study of crop diversity by enabling the comprehensive analysis of protein expression patterns across diverse crop populations (Davidson et al., <a href="#tpg270027-bib-0016" class="usa-link" aria-describedby="tpg270027-bib-0016">2012</a>; Hooper et al., <a href="#tpg270027-bib-0036" class="usa-link" aria-describedby="tpg270027-bib-0036">2020</a>; Lim et al., <a href="#tpg270027-bib-0062" class="usa-link" aria-describedby="tpg270027-bib-0062">2010</a>). Later, several works on seed storage protein were reported in rice, mungbean, pigeonpea, and <em>Cicer</em> across the globe (Dhawale et al., <a href="#tpg270027-bib-0018" class="usa-link" aria-describedby="tpg270027-bib-0018">2015</a>; Jugran et al., <a href="#tpg270027-bib-0041" class="usa-link" aria-describedby="tpg270027-bib-0041">2010</a>; Khan et al., <a href="#tpg270027-bib-0047" class="usa-link" aria-describedby="tpg270027-bib-0047">2013</a>; Khalid et al., <a href="#tpg270027-bib-0046" class="usa-link" aria-describedby="tpg270027-bib-0046">2023</a>; Nayak et al., <a href="#tpg270027-bib-0081" class="usa-link" aria-describedby="tpg270027-bib-0081">2022</a>; Panigrahi et al., <a href="#tpg270027-bib-0095" class="usa-link" aria-describedby="tpg270027-bib-0095">2007</a>). Panigrahi et al. (<a href="#tpg270027-bib-0095" class="usa-link" aria-describedby="tpg270027-bib-0095">2007</a>) elucidated the phylogenetic relationship among 11 species of the genus <em>Cajanus</em> using seed albumin and globulin markers, and their findings were well corroborated on the basing of single nucleotide polymorphism‐based phylogenomic studies (Kassa et al., <a href="#tpg270027-bib-0044" class="usa-link" aria-describedby="tpg270027-bib-0044">2012</a>). Similarly, proteomic studies on other crops such as maize, soybean, and tomato have revealed unique protein profiles linked to traits including disease resistance, nutrient efficiency, and abiotic stress tolerance (Davidson et al., <a href="#tpg270027-bib-0016" class="usa-link" aria-describedby="tpg270027-bib-0016">2012</a>; Hooper et al., <a href="#tpg270027-bib-0036" class="usa-link" aria-describedby="tpg270027-bib-0036">2020</a>; Jaradat &amp; Goldstein, <a href="#tpg270027-bib-0039" class="usa-link" aria-describedby="tpg270027-bib-0039">2018</a>; Lim et al., <a href="#tpg270027-bib-0062" class="usa-link" aria-describedby="tpg270027-bib-0062">2010</a>. These examples underscore the importance of proteomic approaches in elucidating the functional implications of genetic diversity within crops, thereby informing breeding strategies aimed at developing resilient and high‐performing varieties tailored to meet the challenges of modern agriculture.</p>
<section id="tpg270027-sec-0090"><h3 class="pmc_sec_title">3.1. Identification of resistance sources</h3>
<p>The identification of resistance sources in crops, such as in the case of rice against the green leafhopper (GLH), holds significant promise for boosting agricultural yields and enhancing crop resilience. By pinpointing specific protein markers associated with resistance, researchers can develop targeted breeding strategies aimed at selecting resistant traits. For instance, in a study by Padmavathi et al. (<a href="#tpg270027-bib-0089" class="usa-link" aria-describedby="tpg270027-bib-0089">1999</a>), specific protein bands unique to susceptible rice varieties were identified as potential markers for screening GLH resistance. In another study, Prof. Kole and team demonstrated the inheritance of protein markers detecting polymorphism among rice genotypes with contrasting host responses to GLHs (Padmavathi et al., <a href="#tpg270027-bib-0090" class="usa-link" aria-describedby="tpg270027-bib-0090">2001</a>). They demonstrated the monogenic control of two albumin polypeptide bands (46.8, 42.7 kD) through linkage analysis employing an F<sub>2</sub> population derived from the cross TN1/IET15120 (GLH susceptible). These findings not only enable the development of marker‐assisted breeding programs to confer resistance to GLH but also provide insights into the underlying genetics of resistance mechanisms. Electrophoretic analysis by Panigrahi et al. (<a href="#tpg270027-bib-0096" class="usa-link" aria-describedby="tpg270027-bib-0096">2001a</a>) of seed albumin from six varieties of <em>C. cajan</em>, along with two landraces and its putative progenitor <em>Cajanus cajanifolius</em>, revealed 16 distinct polypeptide bands. Among these, four bands specific to <em>C. cajanifolius</em> can serve as reliable markers for verifying hybridity in wide hybridization aimed at the introgression of stress resistance genes. Kole et al. (<a href="#tpg270027-bib-0052" class="usa-link" aria-describedby="tpg270027-bib-0052">2000</a>), based on seed protein type, deciphered that among different local landraces of mungbean, Jhainmung and Kendrapara 2 were resistant to Mungbean yellow mosaic virus (MYMV) and Cercospora leaf spot (CLS), while Bahalmung, Ratila 1, and Karlakhaman were resistant to only MYMV. In 2002, Chand and Kole detected two polypeptide bands in seed protein through SDS‐PAGE that differentiate resistant and susceptible mungbean genotypes. The band with Rm 0.790 was exclusive to resistant genotypes, while the Rm 0.781 band was found only in susceptible ones. These markers aid in screening for CLS resistance in mungbeans. Similarly, Pattnaik and Kole (<a href="#tpg270027-bib-0100" class="usa-link" aria-describedby="tpg270027-bib-0100">2002</a>) detected an albumin polypeptide of Rm 0.202 present only in the susceptible varieties of mungbean that will be useful for negative selection of resistance against MYMV, the most destructive disease in mungbean and other pulse crops. Overall, the identification of resistance sources based on approaches such as seed storage gel electrophoresis offers several advantages in crop improvement efforts. This technique allows for the visualization and characterization of protein profiles associated with resistance traits directly from seeds, offering a convenient and efficient means of screening large populations for resistance. Moreover, seed storage gel electrophoresis facilitates the identification of novel resistance genes and the characterization of genetic diversity within crop germplasm, thereby expanding the genetic resources available for breeding programs. Previously, in French bean (Phaseolus vulgaris L.), presence of arcelin polypeptides was found to be associated with resistance to bruchids (Osborn et al., <a href="#tpg270027-bib-0086" class="usa-link" aria-describedby="tpg270027-bib-0086">1986</a>).</p>
<p>Recent studies have demonstrated the effectiveness of identifying resistance sources through seed storage gel electrophoresis in various crop species. For example, in a study by Dhokane et al. (<a href="#tpg270027-bib-0019" class="usa-link" aria-describedby="tpg270027-bib-0019">2016</a>), integrated metabolon‐transcriptomic was utilized to identify potential resistance sources for Fusarium head blight in wheat, leading to the discovery of novel resistance alleles. Similarly, in a study by Du Plessis (<a href="#tpg270027-bib-0022" class="usa-link" aria-describedby="tpg270027-bib-0022">2013</a>), biochemical characterization was employed to identify resistance sources for Fusarium head blight resistance in wheat, facilitating the development of Fusarium head blight resistant cultivars. These examples highlight the importance of leveraging seed storage gel electrophoresis in identifying resistance sources and accelerating crop improvement efforts to enhance agricultural productivity and sustainability.</p></section><section id="tpg270027-sec-0100"><h3 class="pmc_sec_title">3.2. Assessing genetic purity</h3>
<p>The assessment of genetic purity, denoting the degree of uniformity and absence of genetic contamination within crop cultivars, is pivotal for ensuring the integrity and performance of agricultural germplasm (Hafeez et al., <a href="#tpg270027-bib-0032" class="usa-link" aria-describedby="tpg270027-bib-0032">2023</a>). Panigrahi et al. (<a href="#tpg270027-bib-0097" class="usa-link" aria-describedby="tpg270027-bib-0097">2001b</a>) conducted the first electrophoretic analysis of seed protein fractions to verify hybridity in wide crosses. They analyzed seed globulins to identify markers for hybridity verification in two interspecific crosses involving two pigeon pea varieties as female parents and <em>C. cajanifolius</em> as the male parent. Later on, Mishra et al. (<a href="#tpg270027-bib-0072" class="usa-link" aria-describedby="tpg270027-bib-0072">2012</a>) characterized the interspecific hybrid (<em>C. cajan</em> × <em>C. scarbaeoides</em>) using both seed albumin and seed globulin markers. In a study by Mohanty et al. (<a href="#tpg270027-bib-0073" class="usa-link" aria-describedby="tpg270027-bib-0073">2001</a>), a distinctive electrophoretic pattern based on albumin and globulin profiles was observed across 24 mungbean cultivars. This research underscored the utility of these unique polypeptide patterns as markers for the precise identification and preservation of genetic purity within mungbean cultivars. Similarly, Naik and Kole (<a href="#tpg270027-bib-0077" class="usa-link" aria-describedby="tpg270027-bib-0077">2001</a>) provided the first comprehensive report on the electrophoretic banding patterns of mungbean, encompassing 37 Indian cultivars, further highlighting the significance of protein markers in maintaining genetic purity. Genetic purity in crops refers to the absence of genetic contamination or admixture within a cultivar, ensuring the retention of desired traits and characteristics (Hafeez et al., <a href="#tpg270027-bib-0032" class="usa-link" aria-describedby="tpg270027-bib-0032">2023</a>). It is essential for preserving the genetic integrity of cultivars and preventing unintended changes in agronomic performance, nutritional composition, and other important traits. Maintaining genetic purity is crucial for seed production, breeding programs, and the commercialization of crop varieties, as any deviation from the desired genetic makeup can lead to reduced yield potential, decreased quality, and compromised market value (Hafeez et al., <a href="#tpg270027-bib-0032" class="usa-link" aria-describedby="tpg270027-bib-0032">2023</a>).</p>
<p>Recent research has continued to leverage protein markers for the identification and maintenance of genetic purity in crop cultivars. For instance, in a study by Tripathy et al. (<a href="#tpg270027-bib-0130" class="usa-link" aria-describedby="tpg270027-bib-0130">2015</a>), protein profiling was utilized to assess the genetic purity of upland rice cultivars, revealing distinct protein markers associated with specific genetic backgrounds in upland rice. Similarly, in a study by Manivannan (<a href="#tpg270027-bib-0067" class="usa-link" aria-describedby="tpg270027-bib-0067">2017</a>), seed storage protein markers were employed to evaluate the genetic purity of pearl millet hybrids, enabling the identification of protein markers for hybrid verification and quality assurance. Moreover, Naik et al. (<a href="#tpg270027-bib-0080" class="usa-link" aria-describedby="tpg270027-bib-0080">2019</a>) had characterized the intraspecific hybrids in <em>Clitoria ternatea</em> (a medicinal legume) using seed protein markers. These recent advancements underscore the ongoing importance of protein markers in ensuring genetic purity and enhancing the reliability and effectiveness of crop breeding and seed production programs.</p></section><section id="tpg270027-sec-0110"><h3 class="pmc_sec_title">3.3. Varietal identification and study on genetic inheritance</h3>
<p>Seed protein markers have been utilized in inheritance studies to examine seed protein expression or genetic linkage in various crop plants, including <em>Solanum tuberosum</em> (Rickeman &amp; Desborough, <a href="#tpg270027-bib-0111" class="usa-link" aria-describedby="tpg270027-bib-0111">1978</a>), common bean (Koenig &amp; Gepts, <a href="#tpg270027-bib-0049" class="usa-link" aria-describedby="tpg270027-bib-0049">1989</a>; Osborn et al., <a href="#tpg270027-bib-0086" class="usa-link" aria-describedby="tpg270027-bib-0086">1986</a>; Romero Andreas et al., <a href="#tpg270027-bib-0112" class="usa-link" aria-describedby="tpg270027-bib-0112">1986</a>), soybean (Chen &amp; Shoemaker, <a href="#tpg270027-bib-0011" class="usa-link" aria-describedby="tpg270027-bib-0011">1998</a>), sunflower (Anisimova, <a href="#tpg270027-bib-0003" class="usa-link" aria-describedby="tpg270027-bib-0003">2003</a>; Anisimova &amp; Yang, <a href="#tpg270027-bib-0004" class="usa-link" aria-describedby="tpg270027-bib-0004">2004</a>; Serre et al., <a href="#tpg270027-bib-0117" class="usa-link" aria-describedby="tpg270027-bib-0117">2001</a>), rice (Padmavathi et al., <a href="#tpg270027-bib-0090" class="usa-link" aria-describedby="tpg270027-bib-0090">2001</a>), urdbean (Bashir et al., <a href="#tpg270027-bib-0142" class="usa-link" aria-describedby="tpg270027-bib-0142">2005</a>), <em>Fagopyrum esculentum</em> (Pan &amp; Chen, <a href="#tpg270027-bib-0092" class="usa-link" aria-describedby="tpg270027-bib-0092">2010</a>; Zeller et al., <a href="#tpg270027-bib-0138" class="usa-link" aria-describedby="tpg270027-bib-0138">2004</a>), and sweet chestnut (Martín et al., <a href="#tpg270027-bib-0069" class="usa-link" aria-describedby="tpg270027-bib-0069">2012</a>). In <a href="#tpg270027-bib-0054" class="usa-link" aria-describedby="tpg270027-bib-0054">2002</a>, Kole et al. (<a href="#tpg270027-bib-0054" class="usa-link" aria-describedby="tpg270027-bib-0054">2002</a>) assessed genetic variation in 20 <em>V. mungo</em> genotypes for protein content, protein yield, and seed yield. Protein yield showed a significant positive association with seed yield, protein content, seed weight, and pods/plant suggesting a quantitative inheritance pattern for protein content. A globulin profile was depicted for the first time based on the electrophoretic banding pattern for urdbean leading to the identification of the two genotypes, AKU 7 and LBG 402, with unique protein profiles (Kole et al., <a href="#tpg270027-bib-0051" class="usa-link" aria-describedby="tpg270027-bib-0051">2006</a>). In a similar study by Padmavathi et al. (<a href="#tpg270027-bib-0091" class="usa-link" aria-describedby="tpg270027-bib-0091">2002</a>), variation in water soluble seed albumin banding patterns between and within a set of 12 aromatic and non‐aromatic rice genotypes was observed. They reported the presence of nine polypeptide bands of various molecular weights ranging from 21.4 to 93.3 kD among the aromatic and non‐aromatic varieties elucidating variability in genotypes with respect to albumin fractions. Quantitative variation in total seed protein content was studied in urdbean (Kole et al., <a href="#tpg270027-bib-0050" class="usa-link" aria-describedby="tpg270027-bib-0050">2005</a>). The first report on the use of SDS‐PAGE to delineate the inheritance of protein expression and linkage of seed protein was reported by Prof. Kole and team in the year 2002 in mungbean (Naik &amp; Kole, <a href="#tpg270027-bib-0078" class="usa-link" aria-describedby="tpg270027-bib-0078">2002</a>). They observed a specific polypeptide subunit of 63.1 kD present in non‐aromatic varieties only, which could be used as marker for hybridity confirmation in the crossing program (Padmavathi et al., <a href="#tpg270027-bib-0091" class="usa-link" aria-describedby="tpg270027-bib-0091">2002</a>). In addition, Rath et al. (<a href="#tpg270027-bib-0110" class="usa-link" aria-describedby="tpg270027-bib-0110">2015</a>) also reported the inheritance and genetic linkage of seed protein (albumin and globulin) markers with host resistance to pod borer locus (PPB1) in pigeonpea. These pioneering works on seed protein provided a foundation for modern‐day phylogenetic and diversity studies based on seed protein. Further, they also showed that protein studies can effectively be utilized to understand complex genetic diversity within and between species.</p></section><section id="tpg270027-sec-0120"><h3 class="pmc_sec_title">3.4. Contributions of foundational research toward advancements in crop improvement</h3>
<p>The groundbreaking work led by Prof. Chittaranjan Kole and his team underscores the versatility of seed protein analysis, with far‐reaching applications in plant genetics, taxonomy, and breeding. Seed proteins play a pivotal role in studies related to genetic diversity, variety identification, phylogenetic relationships, evolution, and trait mapping and have become a cornerstone in advancing our understanding of plant biology (Hirano &amp; Shirakawa, <a href="#tpg270027-bib-0035" class="usa-link" aria-describedby="tpg270027-bib-0035">2022</a>). SDS‐PAGE has emerged as a potent tool for discerning genetic diversity through the identification of seed storage proteins, given their resilience to environmental fluctuations. Its application extends to the analysis of variability in seed storage proteins across diverse crops and the differentiation of various varieties and germplasms (Hirano &amp; Shirakawa, <a href="#tpg270027-bib-0035" class="usa-link" aria-describedby="tpg270027-bib-0035">2022</a>; Lan et al., <a href="#tpg270027-bib-0059" class="usa-link" aria-describedby="tpg270027-bib-0059">2024</a>). The first report on the detection of genotype‐specific protein profile in mungbean through electrophoresis of seed albumins and globulins was published by Prof. Kole's lab (Mohanty et al., <a href="#tpg270027-bib-0073" class="usa-link" aria-describedby="tpg270027-bib-0073">2001</a>). Later on, globally, the electrophoretic study of seed storage proteins was extensively conducted, contributing to genotype characterization in crops like wheat, mustard, <em>Solanum, Capsicum</em>, and <em>Vigna</em> (Geetha &amp; Balamurugan, <a href="#tpg270027-bib-0026" class="usa-link" aria-describedby="tpg270027-bib-0026">2011</a>; Ghafoor et al., <a href="#tpg270027-bib-0027" class="usa-link" aria-describedby="tpg270027-bib-0027">2002</a>; Govindaraj et al., <a href="#tpg270027-bib-0030" class="usa-link" aria-describedby="tpg270027-bib-0030">2015</a>; Malik et al., <a href="#tpg270027-bib-0065" class="usa-link" aria-describedby="tpg270027-bib-0065">2013</a>; Mennella et al., <a href="#tpg270027-bib-0071" class="usa-link" aria-describedby="tpg270027-bib-0071">1999</a>; Nagy et al., <a href="#tpg270027-bib-0075" class="usa-link" aria-describedby="tpg270027-bib-0075">2009</a>; Rao et al., <a href="#tpg270027-bib-0108" class="usa-link" aria-describedby="tpg270027-bib-0108">1992</a>; Siddiqui &amp; Naz, <a href="#tpg270027-bib-0122" class="usa-link" aria-describedby="tpg270027-bib-0122">2009</a>).</p>
<p>This electrophoretic approach proves invaluable due to the stability of protein profiles, enabling the study of crop evolution and origins. Beyond that, protein profiling of seed storage proteins facilitates variety identification, determination of phylogenetic relationships between species, germplasm characterization, and biosystematic analysis. Genetic relationship and phylogenetic relationship in mungbean were published in 1998 (Naik, <a href="#tpg270027-bib-0076" class="usa-link" aria-describedby="tpg270027-bib-0076">1998</a>). In the year 2000, Prof. Kole questioned the rationale of assigning separate species to <em>V. mungo</em> and <em>V. radiata</em> through albumin marker analysis in <em>Vigna</em> sps. (Kole et al., <a href="#tpg270027-bib-0052" class="usa-link" aria-describedby="tpg270027-bib-0052">2000</a>). The application of the electrophoretic approach was also done for germplasm evaluation in country bean by Prof. Kole and his team (Pujari, <a href="#tpg270027-bib-0105" class="usa-link" aria-describedby="tpg270027-bib-0105">2000</a>; Shanmugam et al., <a href="#tpg270027-bib-0118" class="usa-link" aria-describedby="tpg270027-bib-0118">2000</a>). Notable studies by other scientists also include the classification of wheat species based on seed storage protein profiling (Malik et al., <a href="#tpg270027-bib-0065" class="usa-link" aria-describedby="tpg270027-bib-0065">2013</a>), identification of relationships among durum wheat genotypes (Pujar et al., <a href="#tpg270027-bib-0104" class="usa-link" aria-describedby="tpg270027-bib-0104">1999</a>), and genetic diversity evaluation of wheat varieties (Shuaib et al., <a href="#tpg270027-bib-0121" class="usa-link" aria-describedby="tpg270027-bib-0121">2007</a>). Noteworthy findings include Malik et al.'s (<a href="#tpg270027-bib-0065" class="usa-link" aria-describedby="tpg270027-bib-0065">2013</a>) classification of wheat species based on seed storage protein profiling, Pujar et al.'s (<a href="#tpg270027-bib-0104" class="usa-link" aria-describedby="tpg270027-bib-0104">1999</a>) identification of relationships among durum wheat genotypes, and genetic diversity evaluation of wheat varieties (Shuaib et al., <a href="#tpg270027-bib-0121" class="usa-link" aria-describedby="tpg270027-bib-0121">2007</a>). In wheat breeding programs, pedigree‐based association mapping, as demonstrated by Ishikawa et al. (<a href="#tpg270027-bib-0037" class="usa-link" aria-describedby="tpg270027-bib-0037">2014</a>), has proven advantageous. Genetic diversity analysis of <em>Brassica napus</em> genotypes emphasizes the efficacy of electrophoretic patterns of seed storage proteins for species differentiation (Choudhary et al., <a href="#tpg270027-bib-0143" class="usa-link" aria-describedby="tpg270027-bib-0143">2015</a>). Khan and Ali recommend wheat endosperm protein for assessing genetic variability and cultivar identification in wheat breeding programs. Additionally, Prof. Kole's team reported that the mungbean (<em>V. radiata</em>) genotype BSN1, a pure line, has potential as a donor for traits such as pod length and seed weight. While it yields well in the spring season, it struggles in summer due to its susceptibility to MYMV (Naik &amp; Kole, <a href="#tpg270027-bib-0078" class="usa-link" aria-describedby="tpg270027-bib-0078">2002</a>). Later that year, Pattnaik and Kole (<a href="#tpg270027-bib-0100" class="usa-link" aria-describedby="tpg270027-bib-0100">2002</a>), identified a seed protein marker through electrophoretic analysis of seed albumins that distinguishes between MYMV‐resistant and susceptible mungbean genotypes, thereby facilitating the incorporation of resistance genes through molecular breeding. In another study, Padmavathi et al. (<a href="#tpg270027-bib-0089" class="usa-link" aria-describedby="tpg270027-bib-0089">1999</a>) reported association of 46.8 and 42.7 kD polypeptides with susceptibility to GLH in rice, thereby differentiating susceptible and resistant varieties and subsequently their introgression into mega varieties through marker‐assisted selection. Another team of Prof. Kole assessed interspecific hybrids of <em>C. cajan</em> and <em>C. cajanifolius</em> concerning various seed protein parameters, noting that a higher albumin‐to‐globulin ratio is a desirable trait for improving protein quality in grain legumes. They successfully developed high‐yielding pigeonpea genotypes with enhanced seed protein content and quality through introgressive wide hybridization with <em>C. cajanifolius</em> (Panigrahi et al., <a href="#tpg270027-bib-0098" class="usa-link" aria-describedby="tpg270027-bib-0098">2002</a>).</p>
<p>Till today, seed protein analysis is widely being used to assess genetic variation within and among plant populations in many crops including cereals, legumes, and oilseeds (Khalid et al., <a href="#tpg270027-bib-0046" class="usa-link" aria-describedby="tpg270027-bib-0046">2023</a>; Pandey &amp; Mann, <a href="#tpg270027-bib-0093" class="usa-link" aria-describedby="tpg270027-bib-0093">2015</a>; Pandey et al., <a href="#tpg270027-bib-0094" class="usa-link" aria-describedby="tpg270027-bib-0094">2018</a>). Such studies are crucial for germplasm characterization, genetic resource conservation, and the identification of diverse genetic pools that can be tapped for crop improvement (Chavhan et al., <a href="#tpg270027-bib-0010" class="usa-link" aria-describedby="tpg270027-bib-0010">2024</a>). Seed protein profiling is being employed for the identification and authentication of different plant varieties, cultivars, and hybrids. Each variety often has a distinct seed protein profile, allowing for accurate identification (Chavhan et al., <a href="#tpg270027-bib-0010" class="usa-link" aria-describedby="tpg270027-bib-0010">2024</a>). This is particularly important in agriculture for maintaining seed purity, preventing mislabeling, and ensuring the integrity of seed stocks. It supports the seed certification process for agricultural varieties. The genetic purity of sunflower hybrids was determined on the basis of isozymes and seed storage proteins and found to be an easy and accurate way of varietal identification (Nikolic et al., <a href="#tpg270027-bib-0083" class="usa-link" aria-describedby="tpg270027-bib-0083">2008</a>). Various studies, such as Maccaferri et al. (<a href="#tpg270027-bib-0063" class="usa-link" aria-describedby="tpg270027-bib-0063">2011</a>) use of simple sequence repeat markers and Zhang et al. (<a href="#tpg270027-bib-0139" class="usa-link" aria-describedby="tpg270027-bib-0139">2011</a>) use of DArT markers for durum wheat genotypes, underscore the importance of differentiating origins, parentage, and distribution. Ishikawa et al. (<a href="#tpg270027-bib-0037" class="usa-link" aria-describedby="tpg270027-bib-0037">2014</a>) findings further emphasize the advantage of pedigree‐based association mapping in breeding programs. Last, Manivannan's (<a href="#tpg270027-bib-0067" class="usa-link" aria-describedby="tpg270027-bib-0067">2017</a>) analysis of pearl millet cultivars highlights the efficiency of SDS‐PAGE in distinguishing varieties based on electrophoretic patterns, showcasing its utility in plant breeding programs and germplasm evaluation for various crops.</p>
<p>Seed protein studies contribute to the elucidation of phylogenetic relationships among plant species. By comparing seed protein profiles, researchers can infer evolutionary relationships and classify plants into taxonomic groups (Hafeez et al., 2024). This aids in refining plant taxonomy, understanding evolutionary patterns, and establishing phylogenetic frameworks for different plant families. SDS‐PAGE, as highlighted by Chavhan et al. (<a href="#tpg270027-bib-0010" class="usa-link" aria-describedby="tpg270027-bib-0010">2024</a>), Hafeez et al. (2024), and Quenum and Yan (<a href="#tpg270027-bib-0106" class="usa-link" aria-describedby="tpg270027-bib-0106">2017</a>), consistently performs well in cluster analysis when cultivars belong to the same geographic location. This is corroborated by Akbar et al. (<a href="#tpg270027-bib-0002" class="usa-link" aria-describedby="tpg270027-bib-0002">2012</a>), and the technique has proven effective in assessing genetic diversity in various research studies (Kakaei &amp; Kahrizi, <a href="#tpg270027-bib-0043" class="usa-link" aria-describedby="tpg270027-bib-0043">2011</a>). Grewal et al. (<a href="#tpg270027-bib-0031" class="usa-link" aria-describedby="tpg270027-bib-0031">2007</a>) reported similar findings using random amplified polymorphic DNA (RAPD) markers for wheat genotypes. Seed protein analysis provides insights into the evolutionary history of plant species. Changes in seed protein composition over time can reveal evolutionary adaptations and relationships between different taxa. Studying the evolution of seed proteins helps trace the divergence of plant lineages, adaptation to ecological niches, and the selection pressures influencing seed protein profiles (Mueller et al., <a href="#tpg270027-bib-0074" class="usa-link" aria-describedby="tpg270027-bib-0074">2023</a>). Despite the availability of numerous DNA markers for wheat, seed storage proteins remain highly valuable, as affirmed by Bean and Lookhart (<a href="#tpg270027-bib-0005" class="usa-link" aria-describedby="tpg270027-bib-0005">2000</a>). In the year 1999, Padmavathi and Kole identified protein markers for the screening of rice genotypes against GLN (Padmavathi et al., <a href="#tpg270027-bib-0089" class="usa-link" aria-describedby="tpg270027-bib-0089">1999</a>). The seed protein markers streamline the breeding process by allowing for the selection of plants with specific protein profiles, expediting the development of new crop varieties with improved traits such as nutritional content or disease resistance.</p></section></section><section id="tpg270027-sec-0130"><h2 class="pmc_sec_title">4. CONCLUSION</h2>
<p>Gel electrophoresis, encompassing both 1D and 2D variations, remains a fundamental technique in plant proteomics, offering cost‐effective and reliable insights into protein composition, genetic diversity, and trait selection. Despite its limitations in detecting low‐abundance proteins and extreme isoelectric point proteins, its integration with gel‐free methods has expanded the scope of proteome analysis, revealing post‐translational modifications and protein interactions. The future of crop proteomics lies in miniaturization, nano‐technological advancements, and high‐throughput MS, which promise deeper insights into seed protein functions. Integration with genomics and metabolomics will provide a holistic understanding of seed development and environmental responses, supporting precision breeding strategies and sustainable agriculture.</p>
<p>A pioneering force in this field, Prof. Chittaranjan Kole has significantly advanced seed protein research through electrophoretic profiling, enabling breakthroughs in genetic diversity assessment, resistance trait identification, and evolutionary studies. His foundational work in plant proteomics has not only provided a deeper understanding of seed storage proteins but has also paved the way for molecular breeding programs that address food security and climate resilience. His contributions continue to inspire innovations in plant molecular biology, reinforcing the relevance of gel electrophoresis as an indispensable tool for researchers worldwide. Looking ahead, the integration of functional genomics, precision agriculture, and bioinformatics‐driven proteomic analyses will redefine crop improvement strategies. Collaborative efforts in data sharing and interdisciplinary research will be crucial in leveraging proteomics for enhanced agricultural productivity, nutritional security, and environmental sustainability.</p></section><section id="tpg270027-sec-0140"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Sarita Pandey</strong>: Conceptualization; data curation; project administration; writing—original draft. <strong>Shalini Tiwari</strong>: Writing—original draft; writing—review and editing. <strong>Poulami Bhattacharjee</strong>: Data curation. <strong>Abhishek Anand</strong>: Visualization. <strong>Lekha T. Pazhamala</strong>: Writing—review and editing. <strong>Yasin Jeshima K</strong>.: Writing—review and editing. <strong>Roshan Kumar Yadav</strong>: Conceptualization; Project administration; Supervision; writing—review and editing.</p></section><section id="tpg270027-sec-0150"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec16"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270027-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270027-supitem-0001"><div class="caption p"><p>
Supplementary Figure 1
</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12254572/bin/TPG2-18-e70027-s001.jpg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70027-s001.jpg</a><sup> (30KB, jpg) </sup>
</div></div></section></section><section id="notes1"><p>


Pandey, S.
, 
Tiwari, S.
, 
Bhattacharjee, P.
, 
Anand, A.
, 
Pazhamala, L. T.
, 
Yasin, J. K.
, &amp; 
Yadav, R. K.
 (2025). Seed protein electrophoresis in plant genetics: Commemorating the pioneering contributions of Prof. Chittaranjan Kole and team to the foundation of plant proteomics. The Plant Genome, 18, e70027. 10.1002/tpg2.70027

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="tpg270027-note-0001"><p>Assigned to Associate Editor Manish Pandey.</p></div>
<div class="fn p" id="tpg270027-note-0002"><p>Sarita Pandey and Shalini Tiwari contributed equally to this work.</p></div>
</div></section></section><section id="tpg270027-sec-0170"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>There are no original data associated with this article. Referenced data are available in the literature.</p></section><section id="tpg270027-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270027-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270027-bib-0001">
<cite>

Abdallah, C.
, 
Dumas‐Gaudot, E.
, 
Renaut, J.
, &amp; 
Sergeant, K.
 (2012). Gel‐based and gel‐free quantitative proteomics approaches at a glance. International Journal of Plant Genomics, 2012(1), 494572.
</cite> [<a href="https://doi.org/10.1155/2012/494572" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3508552/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23213324/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Plant%20Genomics&amp;title=Gel%E2%80%90based%20and%20gel%E2%80%90free%20quantitative%20proteomics%20approaches%20at%20a%20glance&amp;author=C.%20Abdallah&amp;author=E.%20Dumas%E2%80%90Gaudot&amp;author=J.%20Renaut&amp;author=K.%20Sergeant&amp;volume=2012&amp;issue=1&amp;publication_year=2012&amp;pages=494572&amp;pmid=23213324&amp;doi=10.1155/2012/494572&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0002">
<cite>

Akbar, F.
, 
Yousaf, N.
, 
Rabbani, M. A.
, 
Shinwari, Z. K.
, &amp; 
Masood, M. S.
 (2012). Study of total seed proteins pattern of sesame (<em>Sesamum indicum</em> L.) landraces via sodium dodecyl sulfate polyacrylamide gel electrophoresis (SDS‐PAGE). Pakistan Journal of Botany, 44(6), 2009–2014.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Pakistan%20Journal%20of%20Botany&amp;title=Study%20of%20total%20seed%20proteins%20pattern%20of%20sesame%20(Sesamum%20indicum%20L.)%20landraces%20via%20sodium%20dodecyl%20sulfate%20polyacrylamide%20gel%20electrophoresis%20(SDS%E2%80%90PAGE)&amp;author=F.%20Akbar&amp;author=N.%20Yousaf&amp;author=M.%20A.%20Rabbani&amp;author=Z.%20K.%20Shinwari&amp;author=M.%20S.%20Masood&amp;volume=44&amp;issue=6&amp;publication_year=2012&amp;pages=2009-2014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0003">
<cite>

Anisimova, M.
 (2003). 
Detecting positive selection in protein coding genes
. University of London, University College London.</cite> [<a href="https://scholar.google.com/scholar_lookup?Anisimova,%20M.%20(2003).%20Detecting%20positive%20selection%20in%20protein%20coding%20genes%20.%20University%20of%20London,%20University%20College%20London." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0004">
<cite>

Anisimova, M.
, &amp; 
Yang, Z.
 (2004). Molecular evolution of the hepatitis delta virus antigen gene: Recombination or positive selection?
Journal of Molecular Evolution, 59, 815–826. 10.1007/s00239-004-0112-x

</cite> [<a href="https://doi.org/10.1007/s00239-004-0112-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15599513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Molecular%20Evolution&amp;title=Molecular%20evolution%20of%20the%20hepatitis%20delta%20virus%20antigen%20gene:%20Recombination%20or%20positive%20selection?&amp;author=M.%20Anisimova&amp;author=Z.%20Yang&amp;volume=59&amp;publication_year=2004&amp;pages=815-826&amp;pmid=15599513&amp;doi=10.1007/s00239-004-0112-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0005">
<cite>

Bean, S. R.
, &amp; 
Lookhart, G. L.
 (2000). Electrophoresis of cereal storage proteins. Journal of Chromatography A, 881(1–2), 23–36. 10.1016/S0021-9673(99)01270-4

</cite> [<a href="https://doi.org/10.1016/S0021-9673(99)01270-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10905690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Chromatography%20A&amp;title=Electrophoresis%20of%20cereal%20storage%20proteins&amp;author=S.%20R.%20Bean&amp;author=G.%20L.%20Lookhart&amp;volume=881&amp;issue=1%E2%80%932&amp;publication_year=2000&amp;pages=23-36&amp;pmid=10905690&amp;doi=10.1016/S0021-9673(99)01270-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0142">
<cite>

Bashir, M.
, 
Ahmad, Z.
, &amp; 
Ghafoor, A.
 (2005). Sources of genetic resistance in mungbean and blackgram against Urdbean Leaf Crinkle Virus (ULCV). Pakistan Journal of Botany, 37(1), 47–51.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Pakistan%20Journal%20of%20Botany&amp;title=Sources%20of%20genetic%20resistance%20in%20mungbean%20and%20blackgram%20against%20Urdbean%20Leaf%20Crinkle%20Virus%20(ULCV)&amp;author=M.%20Bashir&amp;author=Z.%20Ahmad&amp;author=A.%20Ghafoor&amp;volume=37&amp;issue=1&amp;publication_year=2005&amp;pages=47-51&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0008">
<cite>

Chand, S.
, &amp; 
Kole, C.
 (2002). Protein markers for screening of CLS resistant mungbean (<em>Vigna radiata</em> L. Wilczek.) genotypes. Indian Journal of Genetics and Plant Breeding, 62(04), 340–341.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Indian%20Journal%20of%20Genetics%20and%20Plant%20Breeding&amp;title=Protein%20markers%20for%20screening%20of%20CLS%20resistant%20mungbean%20(Vigna%20radiata%20L.%20Wilczek.)%20genotypes&amp;author=S.%20Chand&amp;author=C.%20Kole&amp;volume=62&amp;issue=04&amp;publication_year=2002&amp;pages=340-341&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0010">
<cite>

Chavhan, R. L.
, 
Hinge, V. R.
, 
Wankhade, D. J.
, 
Deshmukh, A. S.
, 
Mahajan, N.
, &amp; 
Kadam, U. S.
 (2024). Bioinformatics for molecular breeding and enhanced crop performance: Applications and perspectives. Bioinformatics for plant research and crop breeding (pp. 21–74). Wiley. 10.1002/9781394209965.ch2
</cite> [<a href="https://doi.org/10.1002/9781394209965.ch2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Bioinformatics%20for%20plant%20research%20and%20crop%20breeding&amp;author=R.%20L.%20Chavhan&amp;author=V.%20R.%20Hinge&amp;author=D.%20J.%20Wankhade&amp;author=A.%20S.%20Deshmukh&amp;author=N.%20Mahajan&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0011">
<cite>

Chen, A.
, &amp; 
Shoemaker, R. C.
 (1998). Four genes affecting seed traits in soybeans map to linkage group F. Journal of Heredity, 89(3), 211–215. 10.1093/jhered/89.3.211
</cite> [<a href="https://doi.org/10.1093/jhered/89.3.211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Heredity&amp;title=Four%20genes%20affecting%20seed%20traits%20in%20soybeans%20map%20to%20linkage%20group%20F&amp;author=A.%20Chen&amp;author=R.%20C.%20Shoemaker&amp;volume=89&amp;issue=3&amp;publication_year=1998&amp;pages=211-215&amp;doi=10.1093/jhered/89.3.211&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0143">
<cite>

Choudhary, R.
, 
Rai, G. K.
, 
Rai, S. K.
, 
Parveen, A.
, 
Rai, P. K.
, &amp; 
Salgotra, R. K.
 (2015). Genetic diversity of Brassica napus using sodium dodecyl sulphate polyacrylamide gel electrophoresis (SDS‐PAGE). SABRAO Journal of Breeding &amp; Genetics, 47(1), 14–20.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=SABRAO%20Journal%20of%20Breeding%20&amp;%20Genetics&amp;title=Genetic%20diversity%20of%20Brassica%20napus%20using%20sodium%20dodecyl%20sulphate%20polyacrylamide%20gel%20electrophoresis%20(SDS%E2%80%90PAGE)&amp;author=R.%20Choudhary&amp;author=G.%20K.%20Rai&amp;author=S.%20K.%20Rai&amp;author=A.%20Parveen&amp;author=P.%20K.%20Rai&amp;volume=47&amp;issue=1&amp;publication_year=2015&amp;pages=14-20&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0014">
<cite>

Cox, J.
, &amp; 
Mann, M.
 (2007). Is proteomics the new genomics?
Cell, 130(3), 395–398. 10.1016/j.cell.2007.07.032

</cite> [<a href="https://doi.org/10.1016/j.cell.2007.07.032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17693247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Is%20proteomics%20the%20new%20genomics?&amp;author=J.%20Cox&amp;author=M.%20Mann&amp;volume=130&amp;issue=3&amp;publication_year=2007&amp;pages=395-398&amp;pmid=17693247&amp;doi=10.1016/j.cell.2007.07.032&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0015">
<cite>

Cristea, I. M.
, 
Gaskell, S. J.
, &amp; 
Whetton, A. D.
 (2004). Proteomics techniques and their application to hematology. Blood, 103(10), 3624–3634. 10.1182/blood-2003-09-3295

</cite> [<a href="https://doi.org/10.1182/blood-2003-09-3295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14726377/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Proteomics%20techniques%20and%20their%20application%20to%20hematology&amp;author=I.%20M.%20Cristea&amp;author=S.%20J.%20Gaskell&amp;author=A.%20D.%20Whetton&amp;volume=103&amp;issue=10&amp;publication_year=2004&amp;pages=3624-3634&amp;pmid=14726377&amp;doi=10.1182/blood-2003-09-3295&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0016">
<cite>

Davidson, R. M.
, 
Gowda, M.
, 
Moghe, G.
, 
Lin, H.
, 
Vaillancourt, B.
, 
Shiu, S. H.
, &amp; 
Robin Buell, C.
 (2012). Comparative transcriptomics of three Poaceae species reveals patterns of gene expression evolution. The Plant Journal, 71(3), 492–502. 10.1111/j.1365-313X.2012.05005.x

</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2012.05005.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22443345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=Comparative%20transcriptomics%20of%20three%20Poaceae%20species%20reveals%20patterns%20of%20gene%20expression%20evolution&amp;author=R.%20M.%20Davidson&amp;author=M.%20Gowda&amp;author=G.%20Moghe&amp;author=H.%20Lin&amp;author=B.%20Vaillancourt&amp;volume=71&amp;issue=3&amp;publication_year=2012&amp;pages=492-502&amp;pmid=22443345&amp;doi=10.1111/j.1365-313X.2012.05005.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0017">
<cite>

Delehanty, J. B.
, &amp; 
Ligler, F. S.
 (2002). A microarray immunoassay for simultaneous detection of proteins and bacteria. Analytical Chemistry, 74(21), 5681–5687. 10.1021/ac025631l

</cite> [<a href="https://doi.org/10.1021/ac025631l" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12433105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20Chemistry&amp;title=A%20microarray%20immunoassay%20for%20simultaneous%20detection%20of%20proteins%20and%20bacteria&amp;author=J.%20B.%20Delehanty&amp;author=F.%20S.%20Ligler&amp;volume=74&amp;issue=21&amp;publication_year=2002&amp;pages=5681-5687&amp;pmid=12433105&amp;doi=10.1021/ac025631l&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0018">
<cite>

Dhawale, R. N.
, 
Mahalle, M. D.
, &amp; 
Paul, N. S.
 (2015). Biochemical marker (Protein) based characterization of rice accessions bio‐diversity in international rice molecular breeding programme. Asian Journal of Biomedical and Pharmaceutical Science, 5(43), 31–37. 10.15272/ajbps.v5i43.684
</cite> [<a href="https://doi.org/10.15272/ajbps.v5i43.684" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Asian%20Journal%20of%20Biomedical%20and%20Pharmaceutical%20Science&amp;title=Biochemical%20marker%20(Protein)%20based%20characterization%20of%20rice%20accessions%20bio%E2%80%90diversity%20in%20international%20rice%20molecular%20breeding%20programme&amp;author=R.%20N.%20Dhawale&amp;author=M.%20D.%20Mahalle&amp;author=N.%20S.%20Paul&amp;volume=5&amp;issue=43&amp;publication_year=2015&amp;pages=31-37&amp;doi=10.15272/ajbps.v5i43.684&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0019">
<cite>

Dhokane, D.
, 
Karre, S.
, 
Kushalappa, A. C.
, &amp; 
McCartney, C.
 (2016). Integrated metabolo‐transcriptomics reveals Fusarium head blight candidate resistance genes in wheat QTL‐Fhb2. PLoS One, 11(5), e0155851. 10.1371/journal.pone.0155851

</cite> [<a href="https://doi.org/10.1371/journal.pone.0155851" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4883744/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27232496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Integrated%20metabolo%E2%80%90transcriptomics%20reveals%20Fusarium%20head%20blight%20candidate%20resistance%20genes%20in%20wheat%20QTL%E2%80%90Fhb2&amp;author=D.%20Dhokane&amp;author=S.%20Karre&amp;author=A.%20C.%20Kushalappa&amp;author=C.%20McCartney&amp;volume=11&amp;issue=5&amp;publication_year=2016&amp;pages=e0155851&amp;pmid=27232496&amp;doi=10.1371/journal.pone.0155851&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0021">
<cite>

Dunn, M. J.
, &amp; 
Burghes, A. H.
 (1986). High resolution two‐dimensional polyacrylamide‐gel electrophoresis. In Gel electrophoresis of proteins (pp. 203–261). Butterworth‐Heinemann.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Gel%20electrophoresis%20of%20proteins&amp;author=M.%20J.%20Dunn&amp;author=A.%20H.%20Burghes&amp;publication_year=1986&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0022">
<cite>

Du Plessis, A.
 (2013). 
Molecular and biochemical characterisation of rust and Fusarium head blight resistant wheat lines
 [Doctoral dissertation, University of the Free State; ].</cite> [<a href="https://scholar.google.com/scholar_lookup?Du%20Plessis,%20A.%20(2013).%20Molecular%20and%20biochemical%20characterisation%20of%20rust%20and%20Fusarium%20head%20blight%20resistant%20wheat%20lines%20%5BDoctoral%20dissertation,%20University%20of%20the%20Free%20State;%20%5D." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0023">
<cite>

Ebhardt, H. A.
, 
Root, A.
, 
Sander, C.
, &amp; 
Aebersold, R.
 (2015). Applications of targeted proteomics in systems biology and translational medicine. Proteomics, 15(18), 3193–3208. 10.1002/pmic.201500004

</cite> [<a href="https://doi.org/10.1002/pmic.201500004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4758406/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26097198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=Applications%20of%20targeted%20proteomics%20in%20systems%20biology%20and%20translational%20medicine&amp;author=H.%20A.%20Ebhardt&amp;author=A.%20Root&amp;author=C.%20Sander&amp;author=R.%20Aebersold&amp;volume=15&amp;issue=18&amp;publication_year=2015&amp;pages=3193-3208&amp;pmid=26097198&amp;doi=10.1002/pmic.201500004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0024">
<cite>

Eldakak, M.
, 
Milad, S. I.
, 
Nawar, A. I.
, &amp; 
Rohila, J. S.
 (2013). Proteomics: A biotechnology tool for crop improvement. Frontiers in Plant Science, 4, 35. 10.3389/fpls.2013.00035

</cite> [<a href="https://doi.org/10.3389/fpls.2013.00035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3584254/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23450788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Proteomics:%20A%20biotechnology%20tool%20for%20crop%20improvement&amp;author=M.%20Eldakak&amp;author=S.%20I.%20Milad&amp;author=A.%20I.%20Nawar&amp;author=J.%20S.%20Rohila&amp;volume=4&amp;publication_year=2013&amp;pages=35&amp;pmid=23450788&amp;doi=10.3389/fpls.2013.00035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0025">
<cite>

Fujiwara, T.
, 
Nambara, E.
, 
Yamagishi, K.
, 
Goto, D. B.
, &amp; 
Naito, S.
 (2002). Storage proteins. The Arabidopsis Book/American Society of Plant Biologists.</cite> [<a href="https://doi.org/10.1199/tab.0020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3243327/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22303197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Storage%20proteins&amp;author=T.%20Fujiwara&amp;author=E.%20Nambara&amp;author=K.%20Yamagishi&amp;author=D.%20B.%20Goto&amp;author=S.%20Naito&amp;publication_year=2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0026">
<cite>

Geetha, V. V.
, &amp; 
Balamurugan, P.
 (2011). SDS PAGE electrophoresis in mustard cultivars. International Journal of Agriculture Research, 6, 437–443. 10.3923/ijar.2011.437.443
</cite> [<a href="https://doi.org/10.3923/ijar.2011.437.443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Agriculture%20Research&amp;title=SDS%20PAGE%20electrophoresis%20in%20mustard%20cultivars&amp;author=V.%20V.%20Geetha&amp;author=P.%20Balamurugan&amp;volume=6&amp;publication_year=2011&amp;pages=437-443&amp;doi=10.3923/ijar.2011.437.443&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0027">
<cite>

Ghafoor, A.
, 
Ahmad, Z.
, 
Qureshi, A. S.
, &amp; 
Bashir, M.
 (2002). Genetic relationship in <em>Vigna mungo</em> (L.) Hepper and <em>V. radiata</em> (L.) R. Wilczek based on morphological traits and SDS‐PAGE. Euphytica, 123, 367–378. 10.1023/A:1015092502466
</cite> [<a href="https://doi.org/10.1023/A:1015092502466" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Genetic%20relationship%20in%20Vigna%20mungo%20(L.)%20Hepper%20and%20V.%20radiata%20(L.)%20R.%20Wilczek%20based%20on%20morphological%20traits%20and%20SDS%E2%80%90PAGE&amp;author=A.%20Ghafoor&amp;author=Z.%20Ahmad&amp;author=A.%20S.%20Qureshi&amp;author=M.%20Bashir&amp;volume=123&amp;publication_year=2002&amp;pages=367-378&amp;doi=10.1023/A:1015092502466&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0029">
<cite>

Gomathinayagam, P.
, &amp; 
Ramaswamy, N. M.
 (1994). Seed protein pattern of cowpea (<em>Vigna unguiculata</em> L.) and its distant species. Journal of Plant Biochemistry and Biotechnology, 3, 149–151. 10.1007/BF03321968
</cite> [<a href="https://doi.org/10.1007/BF03321968" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Plant%20Biochemistry%20and%20Biotechnology&amp;title=Seed%20protein%20pattern%20of%20cowpea%20(Vigna%20unguiculata%20L.)%20and%20its%20distant%20species&amp;author=P.%20Gomathinayagam&amp;author=N.%20M.%20Ramaswamy&amp;volume=3&amp;publication_year=1994&amp;pages=149-151&amp;doi=10.1007/BF03321968&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0030">
<cite>

Govindaraj, M.
, 
Vetriventhan, M.
, &amp; 
Srinivasan, M.
 (2015). Importance of genetic diversity assessment in crop plants and its recent advances: An overview of its analytical perspectives. Genetics Research International, 2015(1), 431487.
</cite> [<a href="https://doi.org/10.1155/2015/431487" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4383386/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25874132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20Research%20International&amp;title=Importance%20of%20genetic%20diversity%20assessment%20in%20crop%20plants%20and%20its%20recent%20advances:%20An%20overview%20of%20its%20analytical%20perspectives&amp;author=M.%20Govindaraj&amp;author=M.%20Vetriventhan&amp;author=M.%20Srinivasan&amp;volume=2015&amp;issue=1&amp;publication_year=2015&amp;pages=431487&amp;pmid=25874132&amp;doi=10.1155/2015/431487&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0031">
<cite>

Grewal, S.
, 
Kharb, P.
, 
Malik, R.
, 
Jain, S.
, &amp; 
Jain, R. K.
 (2007). Assessment of genetic diversity among some Indian wheat cultivars using random amplified polymorphic DNA (RAPD) markers. Indian Journal of Biotechnology, 6, 18–23.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Indian%20Journal%20of%20Biotechnology&amp;title=Assessment%20of%20genetic%20diversity%20among%20some%20Indian%20wheat%20cultivars%20using%20random%20amplified%20polymorphic%20DNA%20(RAPD)%20markers&amp;author=S.%20Grewal&amp;author=P.%20Kharb&amp;author=R.%20Malik&amp;author=S.%20Jain&amp;author=R.%20K.%20Jain&amp;volume=6&amp;publication_year=2007&amp;pages=18-23&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0032">
<cite>

Hafeez, U.
, 
Ali, M.
, 
Hassan, S. M.
, 
Akram, M. A.
, &amp; 
Zafar, A.
 (2023). Advances in breeding and engineering climate‐resilient crops: A comprehensive review. International Journal of Research and Advances in Agricultural Sciences, 2(2), 85–99.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Research%20and%20Advances%20in%20Agricultural%20Sciences&amp;title=Advances%20in%20breeding%20and%20engineering%20climate%E2%80%90resilient%20crops:%20A%20comprehensive%20review&amp;author=U.%20Hafeez&amp;author=M.%20Ali&amp;author=S.%20M.%20Hassan&amp;author=M.%20A.%20Akram&amp;author=A.%20Zafar&amp;volume=2&amp;issue=2&amp;publication_year=2023&amp;pages=85-99&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0033">
<cite>

Hage, D. S.
, 
Anguizola, J. A.
, 
Bi, C.
, 
Li, R.
, 
Matsuda, R.
, 
Papastavros, E.
, &amp; 
Zheng, X.
 (2012). Pharmaceutical and biomedical applications of affinity chromatography: Recent trends and developments. Journal of Pharmaceutical and Biomedical Analysis, 69, 93–105. 10.1016/j.jpba.2012.01.004

</cite> [<a href="https://doi.org/10.1016/j.jpba.2012.01.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3346860/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22305083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Pharmaceutical%20and%20Biomedical%20Analysis&amp;title=Pharmaceutical%20and%20biomedical%20applications%20of%20affinity%20chromatography:%20Recent%20trends%20and%20developments&amp;author=D.%20S.%20Hage&amp;author=J.%20A.%20Anguizola&amp;author=C.%20Bi&amp;author=R.%20Li&amp;author=R.%20Matsuda&amp;volume=69&amp;publication_year=2012&amp;pages=93-105&amp;pmid=22305083&amp;doi=10.1016/j.jpba.2012.01.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0034">
<cite>

Hameed, A.
, 
Qureshi, M.
, 
Nawaz, M.
, &amp; 
Iqbal, N.
 (2012). Comparative seed storage protein profiling of mung bean genotypes. Pak Journal of Botany, 44(6), 1993–1999.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Pak%20Journal%20of%20Botany&amp;title=Comparative%20seed%20storage%20protein%20profiling%20of%20mung%20bean%20genotypes&amp;author=A.%20Hameed&amp;author=M.%20Qureshi&amp;author=M.%20Nawaz&amp;author=N.%20Iqbal&amp;volume=44&amp;issue=6&amp;publication_year=2012&amp;pages=1993-1999&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0035">
<cite>

Hirano, H.
, &amp; 
Shirakawa, J.
 (2022). Recent developments in Phos‐tag electrophoresis for the analysis of phosphoproteins in proteomics. Expert Review of Proteomics, 19(2), 103–114. 10.1080/14789450.2022.2052850

</cite> [<a href="https://doi.org/10.1080/14789450.2022.2052850" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35285370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Expert%20Review%20of%20Proteomics&amp;title=Recent%20developments%20in%20Phos%E2%80%90tag%20electrophoresis%20for%20the%20analysis%20of%20phosphoproteins%20in%20proteomics&amp;author=H.%20Hirano&amp;author=J.%20Shirakawa&amp;volume=19&amp;issue=2&amp;publication_year=2022&amp;pages=103-114&amp;pmid=35285370&amp;doi=10.1080/14789450.2022.2052850&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0036">
<cite>

Hooper, C. M.
, 
Castleden, I. R.
, 
Aryamanesh, N.
, 
Black, K.
, 
Grasso, S. V.
, &amp; 
Millar, A. H.
 (2020). CropPAL for discovering divergence in protein subcellular location in crops to support strategies for molecular crop breeding. The Plant Journal, 104(3), 812–827. 10.1111/tpj.14961

</cite> [<a href="https://doi.org/10.1111/tpj.14961" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32780488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=CropPAL%20for%20discovering%20divergence%20in%20protein%20subcellular%20location%20in%20crops%20to%20support%20strategies%20for%20molecular%20crop%20breeding&amp;author=C.%20M.%20Hooper&amp;author=I.%20R.%20Castleden&amp;author=N.%20Aryamanesh&amp;author=K.%20Black&amp;author=S.%20V.%20Grasso&amp;volume=104&amp;issue=3&amp;publication_year=2020&amp;pages=812-827&amp;pmid=32780488&amp;doi=10.1111/tpj.14961&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0037">
<cite>

Ishikawa, G.
, 
Nakamura, K.
, 
Ito, H.
, 
Saito, M.
, 
Sato, M.
, 
Jinno, H.
, &amp; 
Nakamura, T.
 (2014). Association mapping and validation of QTLs for flour yield in the soft winter wheat variety Kitahonami. PLoS One, 9(10), 111–337. 10.1371/journal.pone.0111337
</cite> [<a href="https://doi.org/10.1371/journal.pone.0111337" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4215981/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25360619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Association%20mapping%20and%20validation%20of%20QTLs%20for%20flour%20yield%20in%20the%20soft%20winter%20wheat%20variety%20Kitahonami&amp;author=G.%20Ishikawa&amp;author=K.%20Nakamura&amp;author=H.%20Ito&amp;author=M.%20Saito&amp;author=M.%20Sato&amp;volume=9&amp;issue=10&amp;publication_year=2014&amp;pages=111-337&amp;pmid=25360619&amp;doi=10.1371/journal.pone.0111337&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0038">
<cite>

Issaq, H. J.
, &amp; 
Veenstra, T. D.
 (2008). Two‐dimensional polyacrylamide gel electrophoresis (2D‐PAGE): Advances and perspectives. Biotechniques, 44(5), 697–700. 10.2144/000112823

</cite> [<a href="https://doi.org/10.2144/000112823" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18474047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechniques&amp;title=Two%E2%80%90dimensional%20polyacrylamide%20gel%20electrophoresis%20(2D%E2%80%90PAGE):%20Advances%20and%20perspectives&amp;author=H.%20J.%20Issaq&amp;author=T.%20D.%20Veenstra&amp;volume=44&amp;issue=5&amp;publication_year=2008&amp;pages=697-700&amp;pmid=18474047&amp;doi=10.2144/000112823&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0039">
<cite>

Jaradat, A. A.
, &amp; 
Goldstein, W.
 (2018). Diversity of maize kernels from a breeding program for protein quality III: Ionome profiling. Agronomy, 8(2), 9. 10.3390/agronomy8020009
</cite> [<a href="https://doi.org/10.3390/agronomy8020009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agronomy&amp;title=Diversity%20of%20maize%20kernels%20from%20a%20breeding%20program%20for%20protein%20quality%20III:%20Ionome%20profiling&amp;author=A.%20A.%20Jaradat&amp;author=W.%20Goldstein&amp;volume=8&amp;issue=2&amp;publication_year=2018&amp;pages=9&amp;doi=10.3390/agronomy8020009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0040">
<cite>

Jovanovic, S.
, 
Barac, M.
, 
Macej, O.
, 
Vucic, T.
, &amp; 
Lacnjevac, C.
 (2007). SDS‐PAGE analysis of soluble proteins in reconstituted milk exposed to different heat treatments. Sensors, 7(3), 371–383. 10.3390/s7030371
</cite> [<a href="https://doi.org/10.3390/s7030371" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sensors&amp;title=SDS%E2%80%90PAGE%20analysis%20of%20soluble%20proteins%20in%20reconstituted%20milk%20exposed%20to%20different%20heat%20treatments&amp;author=S.%20Jovanovic&amp;author=M.%20Barac&amp;author=O.%20Macej&amp;author=T.%20Vucic&amp;author=C.%20Lacnjevac&amp;volume=7&amp;issue=3&amp;publication_year=2007&amp;pages=371-383&amp;doi=10.3390/s7030371&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0041">
<cite>

Jugran, A.
, 
Bhatt, I. D.
, &amp; 
Rawal, R. S.
 (2010). Characterization of agro‐diversity by seed storage protein electrophoresis: Focus on rice germplasm from Uttarakhand Himalaya, India. Rice Science, 17(2), 122–128. 10.1016/S1672-6308(08)60114-6
</cite> [<a href="https://doi.org/10.1016/S1672-6308(08)60114-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rice%20Science&amp;title=Characterization%20of%20agro%E2%80%90diversity%20by%20seed%20storage%20protein%20electrophoresis:%20Focus%20on%20rice%20germplasm%20from%20Uttarakhand%20Himalaya,%20India&amp;author=A.%20Jugran&amp;author=I.%20D.%20Bhatt&amp;author=R.%20S.%20Rawal&amp;volume=17&amp;issue=2&amp;publication_year=2010&amp;pages=122-128&amp;doi=10.1016/S1672-6308(08)60114-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0042">
<cite>

Jungbauer, A.
, &amp; 
Hahn, R.
 (2009). Ion‐exchange chromatography. Methods in Enzymology, 463, 349–371. 10.1016/S0076-6879(09)63022-6

</cite> [<a href="https://doi.org/10.1016/S0076-6879(09)63022-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19892182/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20in%20Enzymology&amp;title=Ion%E2%80%90exchange%20chromatography&amp;author=A.%20Jungbauer&amp;author=R.%20Hahn&amp;volume=463&amp;publication_year=2009&amp;pages=349-371&amp;pmid=19892182&amp;doi=10.1016/S0076-6879(09)63022-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0043">
<cite>

Kakaei, M.
, &amp; 
Kahrizi, D.
 (2011). Evaluation of seed storage protein patterns of ten wheat varieties using SDS‐PAGE. Biharean Biologist, 5(2), 116–118.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Biharean%20Biologist&amp;title=Evaluation%20of%20seed%20storage%20protein%20patterns%20of%20ten%20wheat%20varieties%20using%20SDS%E2%80%90PAGE&amp;author=M.%20Kakaei&amp;author=D.%20Kahrizi&amp;volume=5&amp;issue=2&amp;publication_year=2011&amp;pages=116-118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0044">
<cite>

Kassa, M. T.
, 
Penmetsa, R. V.
, 
Carrasquilla‐Garcia, N.
, 
Sarma, B. K.
, 
Datta, S.
, 
Upadhyaya, H. D.
, &amp; 
Cook, D. R.
 (2012). Genetic patterns of domestication in pigeon pea (<em>Cajanus cajan</em> (L.) Millsp.) and wild <em>Cajanus</em> relatives. PLoS One, 7(6), 39–563. 10.1371/journal.pone.0039563
</cite> [<a href="https://doi.org/10.1371/journal.pone.0039563" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3382124/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22745789/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Genetic%20patterns%20of%20domestication%20in%20pigeon%20pea%20(Cajanus%20cajan%20(L.)%20Millsp.)%20and%20wild%20Cajanus%20relatives&amp;author=M.%20T.%20Kassa&amp;author=R.%20V.%20Penmetsa&amp;author=N.%20Carrasquilla%E2%80%90Garcia&amp;author=B.%20K.%20Sarma&amp;author=S.%20Datta&amp;volume=7&amp;issue=6&amp;publication_year=2012&amp;pages=39-563&amp;pmid=22745789&amp;doi=10.1371/journal.pone.0039563&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0045">
<cite>

Khalid, A.
, 
Hameed, A.
, &amp; 
Tahir, M. F.
 (2023). Wheat quality: A review on chemical composition, nutritional attributes, grain anatomy, types, classification, and function of seed storage proteins in bread making quality. Frontiers in Nutrition, 10, 1053196. 10.3389/fnut.2023.1053196

</cite> [<a href="https://doi.org/10.3389/fnut.2023.1053196" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9998918/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36908903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Nutrition&amp;title=Wheat%20quality:%20A%20review%20on%20chemical%20composition,%20nutritional%20attributes,%20grain%20anatomy,%20types,%20classification,%20and%20function%20of%20seed%20storage%20proteins%20in%20bread%20making%20quality&amp;author=A.%20Khalid&amp;author=A.%20Hameed&amp;author=M.%20F.%20Tahir&amp;volume=10&amp;publication_year=2023&amp;pages=1053196&amp;pmid=36908903&amp;doi=10.3389/fnut.2023.1053196&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0046">
<cite>

Khalid, U.
, 
Waheed, M. Q.
, 
Parveen, N.
, 
Arif, M. A. R.
, &amp; 
Arif, A.
 (2023). Estimation of genetic diversity using seed storage protein (SSP) profiling in wild and cultivated species of <em>Cicer</em> L. Molecular Biology Reports, 50(5), 4175–4185. 10.1007/s11033-023-08358-9

</cite> [<a href="https://doi.org/10.1007/s11033-023-08358-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36894768/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20Reports&amp;title=Estimation%20of%20genetic%20diversity%20using%20seed%20storage%20protein%20(SSP)%20profiling%20in%20wild%20and%20cultivated%20species%20of%20Cicer%20L&amp;author=U.%20Khalid&amp;author=M.%20Q.%20Waheed&amp;author=N.%20Parveen&amp;author=M.%20A.%20R.%20Arif&amp;author=A.%20Arif&amp;volume=50&amp;issue=5&amp;publication_year=2023&amp;pages=4175-4185&amp;pmid=36894768&amp;doi=10.1007/s11033-023-08358-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0047">
<cite>

Khan, S.
, 
Afzal, M.
, 
Iqbal, S.
, &amp; 
Khan, Q. M.
 (2013). Plant–bacteria partnerships for the remediation of hydrocarbon contaminated soils. Chemosphere, 90(4), 1317–1332. 10.1016/j.chemosphere.2012.09.045

</cite> [<a href="https://doi.org/10.1016/j.chemosphere.2012.09.045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23058201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chemosphere&amp;title=Plant%E2%80%93bacteria%20partnerships%20for%20the%20remediation%20of%20hydrocarbon%20contaminated%20soils&amp;author=S.%20Khan&amp;author=M.%20Afzal&amp;author=S.%20Iqbal&amp;author=Q.%20M.%20Khan&amp;volume=90&amp;issue=4&amp;publication_year=2013&amp;pages=1317-1332&amp;pmid=23058201&amp;doi=10.1016/j.chemosphere.2012.09.045&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0048">
<cite>

Knezevic, V.
, 
Leethanakul, C.
, 
Bichsel, V. E.
, 
Worth, J. M.
, 
Prabhu, V. V.
, 
Gutkind, J. S.
, &amp; 
Krizman, D. B.
 (2001). Proteomic profiling of the cancer microenvironment by antibody arrays. Proteomics: International Edition, 1(10), 1271–1278. 
</cite> [<a href="https://doi.org/10.1002/1615-9861(200110)1:10&amp;#x0003c;1271::AID-PROT1271&amp;#x0003e;3.0.CO;2-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11721638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics:%20International%20Edition&amp;title=Proteomic%20profiling%20of%20the%20cancer%20microenvironment%20by%20antibody%20arrays&amp;author=V.%20Knezevic&amp;author=C.%20Leethanakul&amp;author=V.%20E.%20Bichsel&amp;author=J.%20M.%20Worth&amp;author=V.%20V.%20Prabhu&amp;volume=1&amp;issue=10&amp;publication_year=2001&amp;pages=1271-1278&amp;pmid=11721638&amp;doi=10.1002/1615-9861(200110)1:10&amp;#x0003c;1271::AID-PROT1271&amp;#x0003e;3.0.CO;2-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0049">
<cite>

Koenig, R.
, &amp; 
Gepts, P.
 (1989). Allozyme diversity in wild Phaseolus vulgaris: Further evidence for two major centers of genetic diversity. Theoretical and Applied Genetics, 78, 809–817. 10.1007/BF00266663

</cite> [<a href="https://doi.org/10.1007/BF00266663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24226011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Allozyme%20diversity%20in%20wild%20Phaseolus%20vulgaris:%20Further%20evidence%20for%20two%20major%20centers%20of%20genetic%20diversity&amp;author=R.%20Koenig&amp;author=P.%20Gepts&amp;volume=78&amp;publication_year=1989&amp;pages=809-817&amp;pmid=24226011&amp;doi=10.1007/BF00266663&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0050">
<cite>

Kole, C.
, 
Mohanty, B.
, 
Pal, A.
, 
Bithika Singh, B. S.
, 
Singh, R.
, &amp; 
Agarwal, M.
 (2005). Delineation of quantitative variation in contents of total seed protein and globulins in blackgram (<em>Vigna mungo</em> (L.) Hepper). CABI Digital Library.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kole,%20C.%20,%20Mohanty,%20B.%20,%20Pal,%20A.%20,%20Bithika%20Singh,%20B.%20S.%20,%20Singh,%20R.%20,%20&amp;%20Agarwal,%20M.%20(2005).%20Delineation%20of%20quantitative%20variation%20in%20contents%20of%20total%20seed%20protein%20and%20globulins%20in%20blackgram%20(Vigna%20mungo%20(L.)%20Hepper).%20CABI%20Digital%20Library." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0051">
<cite>

Kole, C.
, 
Mohanty, B.
, 
Pal, A.
, 
Singh, B.
, 
Kole, P. R.
, &amp; 
Shukla, N.
 (2006). Electrophoretic studies on seed protein profile of blackgram, <em>Vigna mungo</em> (L.) Hepper. Legume Research—An International Journal, 29(4), 235–241.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Legume%20Research%E2%80%94An%20International%20Journal&amp;title=Electrophoretic%20studies%20on%20seed%20protein%20profile%20of%20blackgram,%20Vigna%20mungo%20(L.)%20Hepper&amp;author=C.%20Kole&amp;author=B.%20Mohanty&amp;author=A.%20Pal&amp;author=B.%20Singh&amp;author=P.%20R.%20Kole&amp;volume=29&amp;issue=4&amp;publication_year=2006&amp;pages=235-241&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0052">
<cite>

Kole, C.
, 
Naik, B. S.
, &amp; 
Singh, B.
 (2000). Local land races of mungbean from Orissa: A potential resource for some agronomically desirable genes. Crop Research, 20(2), 261–267.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Research&amp;title=Local%20land%20races%20of%20mungbean%20from%20Orissa:%20A%20potential%20resource%20for%20some%20agronomically%20desirable%20genes&amp;author=C.%20Kole&amp;author=B.%20S.%20Naik&amp;author=B.%20Singh&amp;volume=20&amp;issue=2&amp;publication_year=2000&amp;pages=261-267&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0053">
<cite>

Kole, C.
, &amp; 
Panigrahi, J.
 (2001). 
<em>Vigna glabrescens</em>—A natural allopolyploid of <em>V. radiata</em> and <em>V. umbellata</em>: Evidences from seed protein electrophoresis. Crop Research, 22(1), 121–128.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Research&amp;title=Vigna%20glabrescens%E2%80%94A%20natural%20allopolyploid%20of%20V.%20radiata%20and%20V.%20umbellata:%20Evidences%20from%20seed%20protein%20electrophoresis&amp;author=C.%20Kole&amp;author=J.%20Panigrahi&amp;volume=22&amp;issue=1&amp;publication_year=2001&amp;pages=121-128&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0054">
<cite>

Kole, C.
, 
Pattanaik, P. K.
, 
Panigrahi, J.
, &amp; 
Nanda, P. R.
 (2002). 
<em>Vigna radiata</em> and <em>V. mungo</em> are conspecific: Evidences from albumin marker analysis. In National conference on plant biotechnology, biodiversity and ecorestoration for global prosperity. Society for Plant Research (India), Bareilly College.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=National%20conference%20on%20plant%20biotechnology,%20biodiversity%20and%20ecorestoration%20for%20global%20prosperity&amp;author=C.%20Kole&amp;author=P.%20K.%20Pattanaik&amp;author=J.%20Panigrahi&amp;author=P.%20R.%20Nanda&amp;publication_year=2002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0055">
<cite>

Krishna, R. G.
, &amp; 
Wold, F.
 (2013). Post‐translational modification of proteins. Advances in Enzymology and Related Areas of Molecular Biology, 67, 265–298.</cite> [<a href="https://doi.org/10.1002/9780470123133.ch3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8322616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Advances%20in%20Enzymology%20and%20Related%20Areas%20of%20Molecular%20Biology&amp;title=Post%E2%80%90translational%20modification%20of%20proteins&amp;author=R.%20G.%20Krishna&amp;author=F.%20Wold&amp;volume=67&amp;publication_year=2013&amp;pages=265-298&amp;pmid=8322616&amp;doi=10.1002/9780470123133.ch3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0057">
<cite>

Kurien, B. T.
, &amp; 
Scofield, R. H.
 (2006). Western blotting. Methods, 38(4), 283–293. 10.1016/j.ymeth.2005.11.007

</cite> [<a href="https://doi.org/10.1016/j.ymeth.2005.11.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16483794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods&amp;title=Western%20blotting&amp;author=B.%20T.%20Kurien&amp;author=R.%20H.%20Scofield&amp;volume=38&amp;issue=4&amp;publication_year=2006&amp;pages=283-293&amp;pmid=16483794&amp;doi=10.1016/j.ymeth.2005.11.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0059">
<cite>

Lan, T.
, 
Dong, Y.
, 
Jiang, L.
, 
Zhang, Y.
, &amp; 
Sui, X.
 (2024). Analytical approaches for assessing protein structure in protein‐rich food: A comprehensive review. Food Chemistry: X, 22, 101365.
</cite> [<a href="https://doi.org/10.1016/j.fochx.2024.101365" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11016869/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38623506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Chemistry:%20X&amp;title=Analytical%20approaches%20for%20assessing%20protein%20structure%20in%20protein%E2%80%90rich%20food:%20A%20comprehensive%20review&amp;author=T.%20Lan&amp;author=Y.%20Dong&amp;author=L.%20Jiang&amp;author=Y.%20Zhang&amp;author=X.%20Sui&amp;volume=22&amp;publication_year=2024&amp;pages=101365&amp;pmid=38623506&amp;doi=10.1016/j.fochx.2024.101365&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0060">
<cite>

Lander, E. S.
, 
Linton, L. M.
, 
Birren, B.
, 
Nusbaum, C.
, 
Zody, M. C.
, &amp; 
Baldwin, J.
 (2006). Initial sequencing and analysis of the human genome. Nature, 409(6822), 860–921.</cite> [<a href="https://doi.org/10.1038/35057062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11237011/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Initial%20sequencing%20and%20analysis%20of%20the%20human%20genome&amp;author=E.%20S.%20Lander&amp;author=L.%20M.%20Linton&amp;author=B.%20Birren&amp;author=C.%20Nusbaum&amp;author=M.%20C.%20Zody&amp;volume=409&amp;issue=6822&amp;publication_year=2006&amp;pages=860-921&amp;pmid=11237011&amp;doi=10.1038/35057062&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0061">
<cite>

Lequin, R. M.
 (2005). Enzyme immunoassay (EIA)/enzyme‐linked immunosorbent assay (ELISA). Clinical Chemistry, 51(12), 2415–2418. 10.1373/clinchem.2005.051532

</cite> [<a href="https://doi.org/10.1373/clinchem.2005.051532" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16179424/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clinical%20Chemistry&amp;title=Enzyme%20immunoassay%20(EIA)/enzyme%E2%80%90linked%20immunosorbent%20assay%20(ELISA)&amp;author=R.%20M.%20Lequin&amp;volume=51&amp;issue=12&amp;publication_year=2005&amp;pages=2415-2418&amp;pmid=16179424&amp;doi=10.1373/clinchem.2005.051532&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0062">
<cite>

Lim, S. D.
, 
Yim, W. C.
, 
Moon, J. C.
, 
Kim, D. S.
, 
Lee, B. M.
, &amp; 
Jang, C. S.
 (2010). A gene family encoding RING finger proteins in rice: Their expansion, expression diversity, and co‐expressed genes. Plant Molecular Biology, 72, 369–380. 10.1007/s11103-009-9576-9

</cite> [<a href="https://doi.org/10.1007/s11103-009-9576-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19957018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Molecular%20Biology&amp;title=A%20gene%20family%20encoding%20RING%20finger%20proteins%20in%20rice:%20Their%20expansion,%20expression%20diversity,%20and%20co%E2%80%90expressed%20genes&amp;author=S.%20D.%20Lim&amp;author=W.%20C.%20Yim&amp;author=J.%20C.%20Moon&amp;author=D.%20S.%20Kim&amp;author=B.%20M.%20Lee&amp;volume=72&amp;publication_year=2010&amp;pages=369-380&amp;pmid=19957018&amp;doi=10.1007/s11103-009-9576-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0063">
<cite>

Maccaferri, M.
, 
Sanguineti, M. C.
, 
Demontis, A.
, 
El‐Ahmed, A.
, 
Garcia del Moral, L.
, 
Maalouf, F.
, &amp; 
Tuberosa, R.
 (2011). Association mapping in durum wheat grown across a broad range of water regimes. Journal of Experimental Botany, 62(2), 409–438.
</cite> [<a href="https://doi.org/10.1093/jxb/erq287" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21041372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Association%20mapping%20in%20durum%20wheat%20grown%20across%20a%20broad%20range%20of%20water%20regimes&amp;author=M.%20Maccaferri&amp;author=M.%20C.%20Sanguineti&amp;author=A.%20Demontis&amp;author=A.%20El%E2%80%90Ahmed&amp;author=L.%20Garcia%20del%20Moral&amp;volume=62&amp;issue=2&amp;publication_year=2011&amp;pages=409-438&amp;pmid=21041372&amp;doi=10.1093/jxb/erq287&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0065">
<cite>

Malik, R.
, 
Tiwari, R.
, 
Arora, A.
, 
Kumar, P.
, 
Sheoran, S.
, 
Sharma, P.
, &amp; 
Sharma, I.
 (2013). Genotypic characterization of elite Indian wheat genotypes using molecular markers and their pedigree analysis. Australian Journal of Crop Science, 7(5), 561–567.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Australian%20Journal%20of%20Crop%20Science&amp;title=Genotypic%20characterization%20of%20elite%20Indian%20wheat%20genotypes%20using%20molecular%20markers%20and%20their%20pedigree%20analysis&amp;author=R.%20Malik&amp;author=R.%20Tiwari&amp;author=A.%20Arora&amp;author=P.%20Kumar&amp;author=S.%20Sheoran&amp;volume=7&amp;issue=5&amp;publication_year=2013&amp;pages=561-567&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0066">
<cite>

Mani, D. R.
, 
Krug, K.
, 
Zhang, B.
, 
Satpathy, S.
, 
Clauser, K. R.
, 
Ding, L.
, 
Ellis, M.
, 
Gillette, M. A.
, &amp; 
Carr, S. A.
 (2022). Cancer proteogenomics: Current impact and future prospects. Nature Reviews Cancer, 22(5), 298–313. 10.1038/s41568-022-00446-5

</cite> [<a href="https://doi.org/10.1038/s41568-022-00446-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35236940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Reviews%20Cancer&amp;title=Cancer%20proteogenomics:%20Current%20impact%20and%20future%20prospects&amp;author=D.%20R.%20Mani&amp;author=K.%20Krug&amp;author=B.%20Zhang&amp;author=S.%20Satpathy&amp;author=K.%20R.%20Clauser&amp;volume=22&amp;issue=5&amp;publication_year=2022&amp;pages=298-313&amp;pmid=35236940&amp;doi=10.1038/s41568-022-00446-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0067">
<cite>

Manivannan, A.
 (2017). Characterization of pearl millet hybrids and their parental lines by seed storage protein markers. Electronic Journal of Plant Breeding, 8(1), 371–378.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Electronic%20Journal%20of%20Plant%20Breeding&amp;title=Characterization%20of%20pearl%20millet%20hybrids%20and%20their%20parental%20lines%20by%20seed%20storage%20protein%20markers&amp;author=A.%20Manivannan&amp;volume=8&amp;issue=1&amp;publication_year=2017&amp;pages=371-378&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0068">
<cite>

Marouga, R.
, 
David, S.
, &amp; 
Hawkins, E.
 (2005). The development of the DIGE system: 2D fluorescence difference gel analysis technology. Analytical and Bioanalytical Chemistry, 382, 669–678. 10.1007/s00216-005-3126-3

</cite> [<a href="https://doi.org/10.1007/s00216-005-3126-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15900442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Analytical%20and%20Bioanalytical%20Chemistry&amp;title=The%20development%20of%20the%20DIGE%20system:%202D%20fluorescence%20difference%20gel%20analysis%20technology&amp;author=R.%20Marouga&amp;author=S.%20David&amp;author=E.%20Hawkins&amp;volume=382&amp;publication_year=2005&amp;pages=669-678&amp;pmid=15900442&amp;doi=10.1007/s00216-005-3126-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0069">
<cite>

Martín, M. A.
, 
Mattioni, C.
, 
Molina, J. R.
, 
Alvarez, J. B.
, 
Cherubini, M.
, 
Herrera, M. A.
, &amp; 
Martín, L. M.
 (2012). Landscape genetic structure of chestnut (<em>Castanea sativa</em> Mill.) in Spain. Tree Genetics &amp; Genomes, 8, 127–136.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Tree%20Genetics%20&amp;%20Genomes&amp;title=Landscape%20genetic%20structure%20of%20chestnut%20(Castanea%20sativa%20Mill.)%20in%20Spain&amp;author=M.%20A.%20Mart%C3%ADn&amp;author=C.%20Mattioni&amp;author=J.%20R.%20Molina&amp;author=J.%20B.%20Alvarez&amp;author=M.%20Cherubini&amp;volume=8&amp;publication_year=2012&amp;pages=127-136&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0071">
<cite>

Mennella, G.
, 
Sanaja, V. O.
, 
Tonini, A.
, &amp; 
Magnifico, V.
 (1999). Seed storage protein characterization of Solanum species and of cultivars and androgenetic lines of S. melongena L. by SDS‐PAGE and AE‐HPLC. Seed Science Technology, 27, 23–35.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Seed%20Science%20Technology&amp;title=Seed%20storage%20protein%20characterization%20of%20Solanum%20species%20and%20of%20cultivars%20and%20androgenetic%20lines%20of%20S.%20melongena%20L.%20by%20SDS%E2%80%90PAGE%20and%20AE%E2%80%90HPLC&amp;author=G.%20Mennella&amp;author=V.%20O.%20Sanaja&amp;author=A.%20Tonini&amp;author=V.%20Magnifico&amp;volume=27&amp;publication_year=1999&amp;pages=23-35&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0072">
<cite>

Mishra, S.
, 
Sanyal, I.
, &amp; 
Amla, D. V.
 (2012). Changes in protein pattern during different developmental stages of somatic embryos in chickpea. Biologia Plantarum, 56(4), 613–619. 10.1007/s10535-012-0124-0
</cite> [<a href="https://doi.org/10.1007/s10535-012-0124-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biologia%20Plantarum&amp;title=Changes%20in%20protein%20pattern%20during%20different%20developmental%20stages%20of%20somatic%20embryos%20in%20chickpea&amp;author=S.%20Mishra&amp;author=I.%20Sanyal&amp;author=D.%20V.%20Amla&amp;volume=56&amp;issue=4&amp;publication_year=2012&amp;pages=613-619&amp;doi=10.1007/s10535-012-0124-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0073">
<cite>

Mohanty, J. B.
, 
Naik, B. S.
, 
Panigrahi, J.
, &amp; 
Kole, C.
 (2001). Identification of mungbean cultivars based on electrophoretic patterns of seed proteins. Crop Research, 21(2), 134–138.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Research&amp;title=Identification%20of%20mungbean%20cultivars%20based%20on%20electrophoretic%20patterns%20of%20seed%20proteins&amp;author=J.%20B.%20Mohanty&amp;author=B.%20S.%20Naik&amp;author=J.%20Panigrahi&amp;author=C.%20Kole&amp;volume=21&amp;issue=2&amp;publication_year=2001&amp;pages=134-138&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0074">
<cite>

Mueller, K. K.
, 
Pfeifer, L.
, 
Schuldt, L.
, 
Szövényi, P.
, 
de Vries, S.
, 
de Vries, J.
, 
Johnson, K. L.
, &amp; 
Classen, B.
 (2023). Fern cell walls and the evolution of arabinogalactan proteins in streptophytes. The Plant Journal, 114(4), 875–894.
</cite> [<a href="https://doi.org/10.1111/tpj.16178" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36891885/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=Fern%20cell%20walls%20and%20the%20evolution%20of%20arabinogalactan%20proteins%20in%20streptophytes&amp;author=K.%20K.%20Mueller&amp;author=L.%20Pfeifer&amp;author=L.%20Schuldt&amp;author=P.%20Sz%C3%B6v%C3%A9nyi&amp;author=S.%20de%20Vries&amp;volume=114&amp;issue=4&amp;publication_year=2023&amp;pages=875-894&amp;pmid=36891885&amp;doi=10.1111/tpj.16178&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0075">
<cite>

Nagy, N.
, 
Mwizerwa, O.
, 
Yaniv, K.
, 
Carmel, L.
, 
Pieretti‐Vanmarcke, R.
, 
Weinstein, B. M.
, &amp; 
Goldstein, A. M.
 (2009). Endothelial cells promote migration and proliferation of enteric neural crest cells via β1 integrin signaling. Developmental Biology, 330(2), 263–272. 10.1016/j.ydbio.2009.03.025

</cite> [<a href="https://doi.org/10.1016/j.ydbio.2009.03.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2690696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19345201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Developmental%20Biology&amp;title=Endothelial%20cells%20promote%20migration%20and%20proliferation%20of%20enteric%20neural%20crest%20cells%20via%20%CE%B21%20integrin%20signaling&amp;author=N.%20Nagy&amp;author=O.%20Mwizerwa&amp;author=K.%20Yaniv&amp;author=L.%20Carmel&amp;author=R.%20Pieretti%E2%80%90Vanmarcke&amp;volume=330&amp;issue=2&amp;publication_year=2009&amp;pages=263-272&amp;pmid=19345201&amp;doi=10.1016/j.ydbio.2009.03.025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0076">
<cite>

Naik, B.
 (1998). 
Genetic characterization of cultivars and seed protein in mung bean
 [Ph.D. thesis, Utkal University].</cite> [<a href="https://scholar.google.com/scholar_lookup?Naik,%20B.%20(1998).%20Genetic%20characterization%20of%20cultivars%20and%20seed%20protein%20in%20mung%20bean%20%5BPh.D.%20thesis,%20Utkal%20University%5D." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0077">
<cite>

Naik, B. S.
, &amp; 
Kole, C.
 (2001). Electrophoretic studies on seed protein profile of mungbean. Legume Research—An International Journal, 24(3), 169–173.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Legume%20Research%E2%80%94An%20International%20Journal&amp;title=Electrophoretic%20studies%20on%20seed%20protein%20profile%20of%20mungbean&amp;author=B.%20S.%20Naik&amp;author=C.%20Kole&amp;volume=24&amp;issue=3&amp;publication_year=2001&amp;pages=169-173&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0078">
<cite>

Naik, B. S.
, &amp; 
Kole, C.
 (2002). Inheritance of seed protein expression in mungbean. Indian Journal of Genetics and Plant Breeding, 62(01), 79–80.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Indian%20Journal%20of%20Genetics%20and%20Plant%20Breeding&amp;title=Inheritance%20of%20seed%20protein%20expression%20in%20mungbean&amp;author=B.%20S.%20Naik&amp;author=C.%20Kole&amp;volume=62&amp;issue=01&amp;publication_year=2002&amp;pages=79-80&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0079">
<cite>

Naik, B. S.
, 
Patianayak, S. K.
, &amp; 
Kole, C.
 (2000). Selection of protein rich genotypes in mungbean. Indian Journal of Genetics and Plant Breeding, 60(03), 321–326.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Indian%20Journal%20of%20Genetics%20and%20Plant%20Breeding&amp;title=Selection%20of%20protein%20rich%20genotypes%20in%20mungbean&amp;author=B.%20S.%20Naik&amp;author=S.%20K.%20Patianayak&amp;author=C.%20Kole&amp;volume=60&amp;issue=03&amp;publication_year=2000&amp;pages=321-326&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0080">
<cite>

Naik, K.
, 
Mishra, S.
, 
Srichandan, H.
, 
Singh, P. K.
, &amp; 
Sarangi, P. K.
 (2019). Plant growth promoting microbes: Potential link to sustainable agriculture and environment. Biocatalysis and Agricultural Biotechnology, 21, 101326. 10.1016/j.bcab.2019.101326
</cite> [<a href="https://doi.org/10.1016/j.bcab.2019.101326" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biocatalysis%20and%20Agricultural%20Biotechnology&amp;title=Plant%20growth%20promoting%20microbes:%20Potential%20link%20to%20sustainable%20agriculture%20and%20environment&amp;author=K.%20Naik&amp;author=S.%20Mishra&amp;author=H.%20Srichandan&amp;author=P.%20K.%20Singh&amp;author=P.%20K.%20Sarangi&amp;volume=21&amp;publication_year=2019&amp;pages=101326&amp;doi=10.1016/j.bcab.2019.101326&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0081">
<cite>

Nayak, G.
, 
Lenka, D.
, 
Dash, M.
, &amp; 
Tripathy, S. K.
 (2022). Genetic diversity and protein analysis in greengram. Biological Forum, 14(2), 994–999.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Biological%20Forum&amp;title=Genetic%20diversity%20and%20protein%20analysis%20in%20greengram&amp;author=G.%20Nayak&amp;author=D.%20Lenka&amp;author=M.%20Dash&amp;author=S.%20K.%20Tripathy&amp;volume=14&amp;issue=2&amp;publication_year=2022&amp;pages=994-999&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0083">
<cite>

Nikolic, Z.
, 
Vujakovic, M.
, &amp; 
Jevtic, A.
 (2008). Genetic purity of sunflower hybrids determined on the basis of isozymes and seed storage proteins. Helia, 31(48), 47–54.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Helia&amp;title=Genetic%20purity%20of%20sunflower%20hybrids%20determined%20on%20the%20basis%20of%20isozymes%20and%20seed%20storage%20proteins&amp;author=Z.%20Nikolic&amp;author=M.%20Vujakovic&amp;author=A.%20Jevtic&amp;volume=31&amp;issue=48&amp;publication_year=2008&amp;pages=47-54&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0084">
<cite>

Oliveira, C. C.
, 
Jones, A. M.
, 
Fontes, E. P. B.
, &amp; 
Reis, P. A. B. D.
 (2022). G‐Protein phosphorylation: Aspects of binding specificity and function in the plant kingdom. International Journal of Molecular Sciences, 23(12), 6544. 10.3390/ijms23126544

</cite> [<a href="https://doi.org/10.3390/ijms23126544" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9224535/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35742988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=G%E2%80%90Protein%20phosphorylation:%20Aspects%20of%20binding%20specificity%20and%20function%20in%20the%20plant%20kingdom&amp;author=C.%20C.%20Oliveira&amp;author=A.%20M.%20Jones&amp;author=E.%20P.%20B.%20Fontes&amp;author=P.%20A.%20B.%20D.%20Reis&amp;volume=23&amp;issue=12&amp;publication_year=2022&amp;pages=6544&amp;pmid=35742988&amp;doi=10.3390/ijms23126544&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0085">
<cite>

Ong, S. E.
, 
Blagoev, B.
, 
Kratchmarova, I.
, 
Foster, L. J.
, 
Andersen, J. S.
, &amp; 
Mann, M.
 (2006). Stable isotope labeling by amino acids in cell culture for quantitative proteomics. In Cell biology (pp. 427–436). Academic Press.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Cell%20biology&amp;author=S.%20E.%20Ong&amp;author=B.%20Blagoev&amp;author=I.%20Kratchmarova&amp;author=L.%20J.%20Foster&amp;author=J.%20S.%20Andersen&amp;publication_year=2006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0086">
<cite>

Osborn, T. C.
, 
Blake, T.
, 
Gepts, P.
, &amp; 
Bliss, F. A.
 (1986). Bean arcelin: 2. Genetic variation, inheritance and linkage relationships of a novel seed protein of <em>Phaseolus vulgaris</em> L. Theoretical and Applied Genetics, 71, 847–855. 10.1007/BF00276428

</cite> [<a href="https://doi.org/10.1007/BF00276428" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24247713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Bean%20arcelin:%202.%20Genetic%20variation,%20inheritance%20and%20linkage%20relationships%20of%20a%20novel%20seed%20protein%20of%20Phaseolus%20vulgaris%20L&amp;author=T.%20C.%20Osborn&amp;author=T.%20Blake&amp;author=P.%20Gepts&amp;author=F.%20A.%20Bliss&amp;volume=71&amp;publication_year=1986&amp;pages=847-855&amp;pmid=24247713&amp;doi=10.1007/BF00276428&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0088">
<cite>

Otzen, D. E.
, 
Pedersen, J. N.
, 
Rasmussen, H. Ø.
, &amp; 
Pedersen, J. S.
 (2022). How do surfactants unfold and refold proteins?
Advances in Colloid and Interface Science, 308, 102754. 10.1016/j.cis.2022.102754

</cite> [<a href="https://doi.org/10.1016/j.cis.2022.102754" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36027673/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Advances%20in%20Colloid%20and%20Interface%20Science&amp;title=How%20do%20surfactants%20unfold%20and%20refold%20proteins?&amp;author=D.%20E.%20Otzen&amp;author=J.%20N.%20Pedersen&amp;author=H.%20%C3%98.%20Rasmussen&amp;author=J.%20S.%20Pedersen&amp;volume=308&amp;publication_year=2022&amp;pages=102754&amp;pmid=36027673&amp;doi=10.1016/j.cis.2022.102754&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0089">
<cite>

Padmavathi, G.
, 
Kole, C.
, &amp; 
Siddiq, E. A.
 (1999). Detection of protein markers for identification of rice genotypes resistant to green leafhopper. Indian Journal of Genetics and Plant Breeding, 59(4), 417–421.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Indian%20Journal%20of%20Genetics%20and%20Plant%20Breeding&amp;title=Detection%20of%20protein%20markers%20for%20identification%20of%20rice%20genotypes%20resistant%20to%20green%20leafhopper&amp;author=G.%20Padmavathi&amp;author=C.%20Kole&amp;author=E.%20A.%20Siddiq&amp;volume=59&amp;issue=4&amp;publication_year=1999&amp;pages=417-421&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0090">
<cite>

Padmavathi, G.
, 
Siddiq, E. A.
, &amp; 
Kole, C.
 (2001). Inheritance of protein markers detecting polymorphism among rice genotypes with contrasting host response to green leafhopper. Current Science, 80, 1111–1112.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Science&amp;title=Inheritance%20of%20protein%20markers%20detecting%20polymorphism%20among%20rice%20genotypes%20with%20contrasting%20host%20response%20to%20green%20leafhopper&amp;author=G.%20Padmavathi&amp;author=E.%20A.%20Siddiq&amp;author=C.%20Kole&amp;volume=80&amp;publication_year=2001&amp;pages=1111-1112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0091">
<cite>

Padmavathi, V.
, 
Balaiah, A.
, 
Reddy, K. V.
, 
Padmaja, A.
, &amp; 
Reddy, D. B.
 (2002). 3, 3‐Disubstituted 1, 5‐diaryl‐1, 5‐pentanediones as versatile intermediates for spiro heterocycles. ChemInform, 33(47), 119–119.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=ChemInform&amp;title=3,%203%E2%80%90Disubstituted%201,%205%E2%80%90diaryl%E2%80%901,%205%E2%80%90pentanediones%20as%20versatile%20intermediates%20for%20spiro%20heterocycles&amp;author=V.%20Padmavathi&amp;author=A.%20Balaiah&amp;author=K.%20V.%20Reddy&amp;author=A.%20Padmaja&amp;author=D.%20B.%20Reddy&amp;volume=33&amp;issue=47&amp;publication_year=2002&amp;pages=119-119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0092">
<cite>

Pan, S. J.
, &amp; 
Chen, Q. F.
 (2010). Genetic mapping of common buckwheat using DNA, protein and morphological markers. Hereditas, 147(1), 27–33. 10.1111/j.1601-5223.2009.02116.x

</cite> [<a href="https://doi.org/10.1111/j.1601-5223.2009.02116.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20416014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hereditas&amp;title=Genetic%20mapping%20of%20common%20buckwheat%20using%20DNA,%20protein%20and%20morphological%20markers&amp;author=S.%20J.%20Pan&amp;author=Q.%20F.%20Chen&amp;volume=147&amp;issue=1&amp;publication_year=2010&amp;pages=27-33&amp;pmid=20416014&amp;doi=10.1111/j.1601-5223.2009.02116.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0093">
<cite>

Pandey, A.
, &amp; 
Mann, M.
 (2015). Proteomics to study genes and genomes. Nature, 405(6788), 837–846. 10.1038/35015709
</cite> [<a href="https://doi.org/10.1038/35015709" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10866210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Proteomics%20to%20study%20genes%20and%20genomes&amp;author=A.%20Pandey&amp;author=M.%20Mann&amp;volume=405&amp;issue=6788&amp;publication_year=2015&amp;pages=837-846&amp;pmid=10866210&amp;doi=10.1038/35015709&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0094">
<cite>

Pandey, S. K.
, 
Dasgupta, T.
, 
Rathore, A.
, &amp; 
Vemula, A.
 (2018). Relationship of parental genetic distance with heterosis and specific combining ability in sesame <em>(Sesamum indicum</em> L.) based on phenotypic and molecular marker analysis. Biochemical genetics, 56, 188–209. 10.1007/s10528-017-9837-2

</cite> [<a href="https://doi.org/10.1007/s10528-017-9837-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29322371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemical%20genetics&amp;title=Relationship%20of%20parental%20genetic%20distance%20with%20heterosis%20and%20specific%20combining%20ability%20in%20sesame%20(Sesamum%20indicum%20L.)%20based%20on%20phenotypic%20and%20molecular%20marker%20analysis&amp;author=S.%20K.%20Pandey&amp;author=T.%20Dasgupta&amp;author=A.%20Rathore&amp;author=A.%20Vemula&amp;volume=56&amp;publication_year=2018&amp;pages=188-209&amp;pmid=29322371&amp;doi=10.1007/s10528-017-9837-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0095">
<cite>

Panigrahi, J.
, 
Kumar, D. R.
, 
Mishra, M.
, 
Mishra, R. P.
, &amp; 
Jena, P.
 (2007). Genomic relationships among 11 species in the genus Cajanus as revealed by seed protein (albumin and globulin) polymorphisms. Plant Biotechnology Reports, 1, 109–116. 10.1007/s11816-007-0013-8
</cite> [<a href="https://doi.org/10.1007/s11816-007-0013-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Reports&amp;title=Genomic%20relationships%20among%2011%20species%20in%20the%20genus%20Cajanus%20as%20revealed%20by%20seed%20protein%20(albumin%20and%20globulin)%20polymorphisms&amp;author=J.%20Panigrahi&amp;author=D.%20R.%20Kumar&amp;author=M.%20Mishra&amp;author=R.%20P.%20Mishra&amp;author=P.%20Jena&amp;volume=1&amp;publication_year=2007&amp;pages=109-116&amp;doi=10.1007/s11816-007-0013-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0096">
<cite>

Panigrahi, J.
, 
Patnaik, S. N.
, &amp; 
Kole, C.
 (2001a). Detection of species specific protein markers for <em>Cajanus cajan</em> and <em>C. cajanifolius</em>
. Indian Journal of Genetics and Plant Breeding, 61(03), 223–225.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Indian%20Journal%20of%20Genetics%20and%20Plant%20Breeding&amp;title=Detection%20of%20species%20specific%20protein%20markers%20for%20Cajanus%20cajan%20and%20C.%20cajanifolius&amp;author=J.%20Panigrahi&amp;author=S.%20N.%20Patnaik&amp;author=C.%20Kole&amp;volume=61&amp;issue=03&amp;publication_year=2001a&amp;pages=223-225&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0097">
<cite>

Panigrahi, J.
, 
Patnaik, S. N.
, &amp; 
Kole, C.
 (2001b). Identification of putative hybrids between pigeon pea (<em>Cajanus cajan</em>) and <em>C. cajanifolius</em> using·seed protein markers. Indian Journal of Genetics and Plant Breeding, 61(04), 367–368.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Indian%20Journal%20of%20Genetics%20and%20Plant%20Breeding&amp;title=Identification%20of%20putative%20hybrids%20between%20pigeon%20pea%20(Cajanus%20cajan)%20and%20C.%20cajanifolius%20using%C2%B7seed%20protein%20markers&amp;author=J.%20Panigrahi&amp;author=S.%20N.%20Patnaik&amp;author=C.%20Kole&amp;volume=61&amp;issue=04&amp;publication_year=2001b&amp;pages=367-368&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0098">
<cite>

Panigrahi, J.
, 
Patnaik, S. N.
, &amp; 
Kole, C.
 (2002). Evaluation of seed protein content and quality of two <em>Cajanus cajan</em> × <em>C. cajanifolius</em> hybrids. Indian Journal of Genetics and Plant Breeding, 62(04), 338–339.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Indian%20Journal%20of%20Genetics%20and%20Plant%20Breeding&amp;title=Evaluation%20of%20seed%20protein%20content%20and%20quality%20of%20two%20Cajanus%20cajan%20%C3%97%20C.%20cajanifolius%20hybrids&amp;author=J.%20Panigrahi&amp;author=S.%20N.%20Patnaik&amp;author=C.%20Kole&amp;volume=62&amp;issue=04&amp;publication_year=2002&amp;pages=338-339&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0100">
<cite>

Pattnaik, J.
, &amp; 
Kole, C.
 (2002). Detection of a protein marker for screening of MYMV resistant mungbean genotypes. Indian Journal of Genetics and Plant Breeding, 62(01), 77–78.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Indian%20Journal%20of%20Genetics%20and%20Plant%20Breeding&amp;title=Detection%20of%20a%20protein%20marker%20for%20screening%20of%20MYMV%20resistant%20mungbean%20genotypes&amp;author=J.%20Pattnaik&amp;author=C.%20Kole&amp;volume=62&amp;issue=01&amp;publication_year=2002&amp;pages=77-78&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0101">
<cite>

Popescu, S. C.
, 
Snyder, M.
, &amp; 
Dinesh‐Kumar, S. P.
 (2007). Arabidopsis protein microarrays for the high‐throughput identification of protein‐protein interactions. Plant Signaling &amp; Behavior, 2(5), 416–420.
</cite> [<a href="https://doi.org/10.4161/psb.2.5.4416" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2634232/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19704619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Signaling%20&amp;%20Behavior&amp;title=Arabidopsis%20protein%20microarrays%20for%20the%20high%E2%80%90throughput%20identification%20of%20protein%E2%80%90protein%20interactions&amp;author=S.%20C.%20Popescu&amp;author=M.%20Snyder&amp;author=S.%20P.%20Dinesh%E2%80%90Kumar&amp;volume=2&amp;issue=5&amp;publication_year=2007&amp;pages=416-420&amp;pmid=19704619&amp;doi=10.4161/psb.2.5.4416&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0103">
<cite>

Prasadi, D. T.
, 
Umpathy, N. S.
, &amp; 
Veeranna, R.
 (1996). Genotypic variation in cowpea (<em>Vigna unguiculata</em>) cultivars in relation to insect resistance. Journal of Plant Biochemistry and Biotechnology, 5, 47–49. 10.1007/BF03262979
</cite> [<a href="https://doi.org/10.1007/BF03262979" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Plant%20Biochemistry%20and%20Biotechnology&amp;title=Genotypic%20variation%20in%20cowpea%20(Vigna%20unguiculata)%20cultivars%20in%20relation%20to%20insect%20resistance&amp;author=D.%20T.%20Prasadi&amp;author=N.%20S.%20Umpathy&amp;author=R.%20Veeranna&amp;volume=5&amp;publication_year=1996&amp;pages=47-49&amp;doi=10.1007/BF03262979&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0104">
<cite>

Pujar, S.
, 
Tamhankar, S. A.
, 
Rao, V. S.
, 
Gupta, V. S.
, 
Naik, S.
, &amp; 
Ranjekar, P. K.
 (1999). Arbitrarily primed‐PCR based diversity assessment reflects hierarchical groupings of Indian tetraploid wheat genotypes. Theoretical and Applied Genetics, 99, 868–876. 10.1007/s001220051307
</cite> [<a href="https://doi.org/10.1007/s001220051307" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Arbitrarily%20primed%E2%80%90PCR%20based%20diversity%20assessment%20reflects%20hierarchical%20groupings%20of%20Indian%20tetraploid%20wheat%20genotypes&amp;author=S.%20Pujar&amp;author=S.%20A.%20Tamhankar&amp;author=V.%20S.%20Rao&amp;author=V.%20S.%20Gupta&amp;author=S.%20Naik&amp;volume=99&amp;publication_year=1999&amp;pages=868-876&amp;doi=10.1007/s001220051307&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0105">
<cite>

Pujari, I.
 (2000). 
<em>Evaluation o</em>
<em>n genetic variability in</em>
<em>country bean</em>
<em>with conventional and molecular strategies</em>
 [Master's thesis]. OUAT.</cite> [<a href="https://scholar.google.com/scholar_lookup?Pujari,%20I.%20(2000).%20Evaluation%20o%20n%20genetic%20variability%20in%20country%20bean%20with%20conventional%20and%20molecular%20strategies%20%5BMaster's%20thesis%5D.%20OUAT." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0106">
<cite>

Quenum, F. J. B.
, &amp; 
Yan, Q.
 (2017). Assessing genetic variation and relationships among a mini core germplasm of sesame (<em>Sesamum indicum</em> L.) using biochemical and RAPD markers. American Journal of Plant Sciences, 8(3), 311–327. 10.4236/ajps.2017.83022
</cite> [<a href="https://doi.org/10.4236/ajps.2017.83022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Plant%20Sciences&amp;title=Assessing%20genetic%20variation%20and%20relationships%20among%20a%20mini%20core%20germplasm%20of%20sesame%20(Sesamum%20indicum%20L.)%20using%20biochemical%20and%20RAPD%20markers&amp;author=F.%20J.%20B.%20Quenum&amp;author=Q.%20Yan&amp;volume=8&amp;issue=3&amp;publication_year=2017&amp;pages=311-327&amp;doi=10.4236/ajps.2017.83022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0107">
<cite>

Quinn, O.
, 
Kumar, M.
, &amp; 
Turner, S.
 (2024). The role of lipid‐modified proteins in cell wall synthesis and signaling. Plant Physiology, 194(1), 51–66. 10.1093/plphys/kiad491
</cite> [<a href="https://doi.org/10.1093/plphys/kiad491" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10756762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37682865/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=The%20role%20of%20lipid%E2%80%90modified%20proteins%20in%20cell%20wall%20synthesis%20and%20signaling&amp;author=O.%20Quinn&amp;author=M.%20Kumar&amp;author=S.%20Turner&amp;volume=194&amp;issue=1&amp;publication_year=2024&amp;pages=51-66&amp;pmid=37682865&amp;doi=10.1093/plphys/kiad491&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0108">
<cite>

Rao, L.
, 
Debbas, M.
, 
Sabbatini, P.
, 
Hockenbery, D.
, 
Korsmeyer, S.
, &amp; 
White, E.
 (1992). The adenovirus E1A proteins induce apoptosis, which is inhibited by the E1B 19‐kDa and Bcl‐2 proteins. Proceedings of the National Academy of Sciences, 89(16), 7742–7746. 10.1073/pnas.89.16.7742
</cite> [<a href="https://doi.org/10.1073/pnas.89.16.7742" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC49787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1457005/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=The%20adenovirus%20E1A%20proteins%20induce%20apoptosis,%20which%20is%20inhibited%20by%20the%20E1B%2019%E2%80%90kDa%20and%20Bcl%E2%80%902%20proteins&amp;author=L.%20Rao&amp;author=M.%20Debbas&amp;author=P.%20Sabbatini&amp;author=D.%20Hockenbery&amp;author=S.%20Korsmeyer&amp;volume=89&amp;issue=16&amp;publication_year=1992&amp;pages=7742-7746&amp;pmid=1457005&amp;doi=10.1073/pnas.89.16.7742&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0109">
<cite>

Rasheed, F.
, 
Markgren, J.
, 
Hedenqvist, M.
, &amp; 
Johansson, E.
 (2020). Modeling to understand plant protein structure‐function relationships—implications for seed storage proteins. Molecules, 25(4), 873. 10.3390/molecules25040873

</cite> [<a href="https://doi.org/10.3390/molecules25040873" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7071054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32079172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Modeling%20to%20understand%20plant%20protein%20structure%E2%80%90function%20relationships%E2%80%94implications%20for%20seed%20storage%20proteins&amp;author=F.%20Rasheed&amp;author=J.%20Markgren&amp;author=M.%20Hedenqvist&amp;author=E.%20Johansson&amp;volume=25&amp;issue=4&amp;publication_year=2020&amp;pages=873&amp;pmid=32079172&amp;doi=10.3390/molecules25040873&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0110">
<cite>

Rath, D.
, 
Amlinger, L.
, 
Rath, A.
, &amp; 
Lundgren, M.
 (2015). The CRISPR‐Cas immune system: Biology, mechanisms and applications. Biochimie, 117, 119–128. 10.1016/j.biochi.2015.03.025

</cite> [<a href="https://doi.org/10.1016/j.biochi.2015.03.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25868999/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimie&amp;title=The%20CRISPR%E2%80%90Cas%20immune%20system:%20Biology,%20mechanisms%20and%20applications&amp;author=D.%20Rath&amp;author=L.%20Amlinger&amp;author=A.%20Rath&amp;author=M.%20Lundgren&amp;volume=117&amp;publication_year=2015&amp;pages=119-128&amp;pmid=25868999&amp;doi=10.1016/j.biochi.2015.03.025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0111">
<cite>

Rickeman, V. S.
, &amp; 
Desborough, S. L.
 (1978). Inheritance of three electrophoretically determined protein bands in potato (<em>Solanum tuberosum</em> L.). Theoretical and Applied Genetics, 52, 187–190. 10.1007/BF00282576

</cite> [<a href="https://doi.org/10.1007/BF00282576" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24317505/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Inheritance%20of%20three%20electrophoretically%20determined%20protein%20bands%20in%20potato%20(Solanum%20tuberosum%20L.)&amp;author=V.%20S.%20Rickeman&amp;author=S.%20L.%20Desborough&amp;volume=52&amp;publication_year=1978&amp;pages=187-190&amp;pmid=24317505&amp;doi=10.1007/BF00282576&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0112">
<cite>

Romero Andreas, J.
, 
Yandell, B. S.
, &amp; 
Bliss, F. A.
 (1986). Bean arcelin: 1. Inheritance of a novel seed protein of <em>Phaseolus vulgaris</em> L. and its effect on seed composition. Theoretical and Applied Genetics, 72, 123–128. 10.1007/BF00261467

</cite> [<a href="https://doi.org/10.1007/BF00261467" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24247784/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Bean%20arcelin:%201.%20Inheritance%20of%20a%20novel%20seed%20protein%20of%20Phaseolus%20vulgaris%20L.%20and%20its%20effect%20on%20seed%20composition&amp;author=J.%20Romero%20Andreas&amp;author=B.%20S.%20Yandell&amp;author=F.%20A.%20Bliss&amp;volume=72&amp;publication_year=1986&amp;pages=123-128&amp;pmid=24247784&amp;doi=10.1007/BF00261467&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0113">
<cite>

Rosenberg, J. M.
, &amp; 
Utz, P. J.
 (2015). Protein microarrays: A new tool for the study of autoantibodies in immunodeficiency. Frontiers in Immunology, 6, 138. 10.3389/fimmu.2015.00138

</cite> [<a href="https://doi.org/10.3389/fimmu.2015.00138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4387933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25904912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Immunology&amp;title=Protein%20microarrays:%20A%20new%20tool%20for%20the%20study%20of%20autoantibodies%20in%20immunodeficiency&amp;author=J.%20M.%20Rosenberg&amp;author=P.%20J.%20Utz&amp;volume=6&amp;publication_year=2015&amp;pages=138&amp;pmid=25904912&amp;doi=10.3389/fimmu.2015.00138&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0114">
<cite>

Sadia, M.
, 
Malik, S. A.
, 
Rabbani, M. A.
, &amp; 
Pearce, S. R.
 (2009). Electrophoretic characterization and the relationship between some Brassica species. Electronic Journal of Biology, 5(1), 1–4.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Electronic%20Journal%20of%20Biology&amp;title=Electrophoretic%20characterization%20and%20the%20relationship%20between%20some%20Brassica%20species&amp;author=M.%20Sadia&amp;author=S.%20A.%20Malik&amp;author=M.%20A.%20Rabbani&amp;author=S.%20R.%20Pearce&amp;volume=5&amp;issue=1&amp;publication_year=2009&amp;pages=1-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0115">
<cite>

Sahai, S.
, &amp; 
Rana, R. S.
 (1977). Seed protein homology and elucidation of species relationships in Phaseolus and Vigna species. New Phytologist, 79, 527–534. 10.1111/j.1469-8137.1977.tb02237.x
</cite> [<a href="https://doi.org/10.1111/j.1469-8137.1977.tb02237.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Seed%20protein%20homology%20and%20elucidation%20of%20species%20relationships%20in%20Phaseolus%20and%20Vigna%20species&amp;author=S.%20Sahai&amp;author=R.%20S.%20Rana&amp;volume=79&amp;publication_year=1977&amp;pages=527-534&amp;doi=10.1111/j.1469-8137.1977.tb02237.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0116">
<cite>

Sanchez‐Carbayo, M.
, 
Socci, N. D.
, 
Lozano, J. J.
, 
Haab, B. B.
, &amp; 
Cordon‐Cardo, C.
 (2006). Profiling bladder cancer using targeted antibody arrays. The American Journal of Pathology, 168(1), 93–103. 10.2353/ajpath.2006.050601

</cite> [<a href="https://doi.org/10.2353/ajpath.2006.050601" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1592677/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16400012/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20American%20Journal%20of%20Pathology&amp;title=Profiling%20bladder%20cancer%20using%20targeted%20antibody%20arrays&amp;author=M.%20Sanchez%E2%80%90Carbayo&amp;author=N.%20D.%20Socci&amp;author=J.%20J.%20Lozano&amp;author=B.%20B.%20Haab&amp;author=C.%20Cordon%E2%80%90Cardo&amp;volume=168&amp;issue=1&amp;publication_year=2006&amp;pages=93-103&amp;pmid=16400012&amp;doi=10.2353/ajpath.2006.050601&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0117">
<cite>

Serre, E.
, 
Del Arco, E. C.
, &amp; 
Bontoux, P.
 (2001). Annular and spiral patterns in flows between rotating and stationary discs. Journal of Fluid Mechanics, 434, 65–100. 10.1017/S0022112001003494
</cite> [<a href="https://doi.org/10.1017/S0022112001003494" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Fluid%20Mechanics&amp;title=Annular%20and%20spiral%20patterns%20in%20flows%20between%20rotating%20and%20stationary%20discs&amp;author=E.%20Serre&amp;author=E.%20C.%20Del%20Arco&amp;author=P.%20Bontoux&amp;volume=434&amp;publication_year=2001&amp;pages=65-100&amp;doi=10.1017/S0022112001003494&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0118">
<cite>

Shanmugam, A.
, 
Pujari, I.
, 
Panigrahi, J.
, 
Dora, D. K.
, &amp; 
Kole, C.
 (2000). Molecular characterization of country bean germplasm. In National seminar on hi‐tech horticulture. Indian Institute of Horticultural Research.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=National%20seminar%20on%20hi%E2%80%90tech%20horticulture&amp;author=A.%20Shanmugam&amp;author=I.%20Pujari&amp;author=J.%20Panigrahi&amp;author=D.%20K.%20Dora&amp;author=C.%20Kole&amp;publication_year=2000&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0119">
<cite>

Shewry, P. R.
, 
Napier, J. A.
, &amp; 
Tatham, A. S.
 (1995). Seed storage proteins: Structures and biosynthesis. The Plant Cell, 7(7), 945–956.
</cite> [<a href="https://doi.org/10.1105/tpc.7.7.945" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC160892/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7640527/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Cell&amp;title=Seed%20storage%20proteins:%20Structures%20and%20biosynthesis&amp;author=P.%20R.%20Shewry&amp;author=J.%20A.%20Napier&amp;author=A.%20S.%20Tatham&amp;volume=7&amp;issue=7&amp;publication_year=1995&amp;pages=945-956&amp;pmid=7640527&amp;doi=10.1105/tpc.7.7.945&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0120">
<cite>

Shiio, Y.
, &amp; 
Aebersold, R.
 (2006). Quantitative proteome analysis using isotope‐coded affinity tags and mass spectrometry. Nature Protocols, 1(1), 139–145. 10.1038/nprot.2006.22

</cite> [<a href="https://doi.org/10.1038/nprot.2006.22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17406225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Protocols&amp;title=Quantitative%20proteome%20analysis%20using%20isotope%E2%80%90coded%20affinity%20tags%20and%20mass%20spectrometry&amp;author=Y.%20Shiio&amp;author=R.%20Aebersold&amp;volume=1&amp;issue=1&amp;publication_year=2006&amp;pages=139-145&amp;pmid=17406225&amp;doi=10.1038/nprot.2006.22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0121">
<cite>

Shuaib, M.
, 
Zeb, A.
, 
Ali, Z.
, 
Ali, W.
, 
Ahmad, T.
, &amp; 
Khan, I.
 (2007). Characterization of wheat varieties by seed storageprotein electrophoresis. African Journal of Biotechnology, 6(5), 497–500.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=African%20Journal%20of%20Biotechnology&amp;title=Characterization%20of%20wheat%20varieties%20by%20seed%20storageprotein%20electrophoresis&amp;author=M.%20Shuaib&amp;author=A.%20Zeb&amp;author=Z.%20Ali&amp;author=W.%20Ali&amp;author=T.%20Ahmad&amp;volume=6&amp;issue=5&amp;publication_year=2007&amp;pages=497-500&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0122">
<cite>

Siddiqui, M. F.
, &amp; 
Naz, N.
 (2009). Protein landmarks for diversity assessment in wheat genotypes. African Journal of Biotechnology, 8, 1855–1859.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=African%20Journal%20of%20Biotechnology&amp;title=Protein%20landmarks%20for%20diversity%20assessment%20in%20wheat%20genotypes&amp;author=M.%20F.%20Siddiqui&amp;author=N.%20Naz&amp;volume=8&amp;publication_year=2009&amp;pages=1855-1859&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0124">
<cite>

Smith, J. B.
 (2001). Peptide sequencing by Edman degradation. In Encyclopedia of life sciences (pp. 1–3). Nature Pubishing Group, Macmillan Publisher.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Encyclopedia%20of%20life%20sciences&amp;author=J.%20B.%20Smith&amp;publication_year=2001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0125">
<cite>

Smyth, M. S.
, &amp; 
Martin, J. H.
 (2000). x Ray crystallography. Molecular Pathology, 53(1), 8–14. 10.1136/mp.53.1.8

</cite> [<a href="https://doi.org/10.1136/mp.53.1.8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1186895/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10884915/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Pathology&amp;title=x%20Ray%20crystallography&amp;author=M.%20S.%20Smyth&amp;author=J.%20H.%20Martin&amp;volume=53&amp;issue=1&amp;publication_year=2000&amp;pages=8-14&amp;pmid=10884915&amp;doi=10.1136/mp.53.1.8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0127">
<cite>

Tebani, A.
, 
Afonso, C.
, 
Marret, S.
, &amp; 
Bekri, S.
 (2016). Omics‐based strategies in precision medicine: Toward a paradigm shift in inborn errors of metabolism investigations. International Journal of Molecular Sciences, 17(9), 1555. 10.3390/ijms17091555

</cite> [<a href="https://doi.org/10.3390/ijms17091555" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5037827/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27649151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Omics%E2%80%90based%20strategies%20in%20precision%20medicine:%20Toward%20a%20paradigm%20shift%20in%20inborn%20errors%20of%20metabolism%20investigations&amp;author=A.%20Tebani&amp;author=C.%20Afonso&amp;author=S.%20Marret&amp;author=S.%20Bekri&amp;volume=17&amp;issue=9&amp;publication_year=2016&amp;pages=1555&amp;pmid=27649151&amp;doi=10.3390/ijms17091555&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0141">
<cite>

Thakare, R. G.
, 
Gadgil, J. D.
, &amp; 
Mitra, R.
 (1988). Origin and evolution of seed protein genes in <em>Vigna mungo</em> and <em>Vignaradiata</em>
. In Sundaram S. Shanmuga &amp; McLean B. T. (Eds.), Proceedings for the 2nd on Mungbean: Information, program and abstracts (pp. 47–52). Asian Vegetable Research and Development Center.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Proceedings%20for%20the%202nd%20on%20Mungbean:%20Information,%20program%20and%20abstracts&amp;author=R.%20G.%20Thakare&amp;author=J.%20D.%20Gadgil&amp;author=R.%20Mitra&amp;publication_year=1988&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0129">
<cite>

Tibes, R.
, 
Qiu, Y.
, 
Lu, Y.
, 
Hennessy, B.
, 
Andreeff, M.
, 
Mills, G. B.
, &amp; 
Kornblau, S. M.
 (2006). Reverse phase protein array: Validation of a novel proteomic technology and utility for analysis of primary leukemia specimens and hematopoietic stem cells. Molecular Cancer Therapeutics, 5(10), 2512–2521. 10.1158/1535-7163.MCT-06-0334

</cite> [<a href="https://doi.org/10.1158/1535-7163.MCT-06-0334" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17041095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cancer%20Therapeutics&amp;title=Reverse%20phase%20protein%20array:%20Validation%20of%20a%20novel%20proteomic%20technology%20and%20utility%20for%20analysis%20of%20primary%20leukemia%20specimens%20and%20hematopoietic%20stem%20cells&amp;author=R.%20Tibes&amp;author=Y.%20Qiu&amp;author=Y.%20Lu&amp;author=B.%20Hennessy&amp;author=M.%20Andreeff&amp;volume=5&amp;issue=10&amp;publication_year=2006&amp;pages=2512-2521&amp;pmid=17041095&amp;doi=10.1158/1535-7163.MCT-06-0334&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0130">
<cite>

Tripathy, S. K.
, 
Mohapatra, B. R.
, 
Nayak, P. K.
, 
Pal, S.
, 
Senapati, N.
, 
Dash, G. B.
, 
Lenka, D.
, 
Swain, D.
, &amp; 
Ranjan, R.
 (2015). Revealing genetic variation in upland rice using seed storage protein profiling. Research on Crops, 16(2), 320–331. 10.5958/2348-7542.2015.00047.9
</cite> [<a href="https://doi.org/10.5958/2348-7542.2015.00047.9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Research%20on%20Crops&amp;title=Revealing%20genetic%20variation%20in%20upland%20rice%20using%20seed%20storage%20protein%20profiling&amp;author=S.%20K.%20Tripathy&amp;author=B.%20R.%20Mohapatra&amp;author=P.%20K.%20Nayak&amp;author=S.%20Pal&amp;author=N.%20Senapati&amp;volume=16&amp;issue=2&amp;publication_year=2015&amp;pages=320-331&amp;doi=10.5958/2348-7542.2015.00047.9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0131">
<cite>

Ummanni, R.
, 
Mannsperger, H. A.
, 
Sonntag, J.
, 
Oswald, M.
, 
Sharma, A. K.
, 
Koenig, R.
, &amp; 
Korf, U.
 (2014). Evaluation of reverse phase protein array (RPPA)‐based pathway‐activation profiling in 84 non‐small cell lung cancer (NSCLC) cell lines as platform for cancer proteomics and biomarker discovery. Biochimica Biophysica Acta (BBA)‐Proteins and Proteomics, 1844(5), 950–959. 10.1016/j.bbapap.2013.11.017
</cite> [<a href="https://doi.org/10.1016/j.bbapap.2013.11.017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24361481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochimica%20Biophysica%20Acta%20(BBA)%E2%80%90Proteins%20and%20Proteomics&amp;title=Evaluation%20of%20reverse%20phase%20protein%20array%20(RPPA)%E2%80%90based%20pathway%E2%80%90activation%20profiling%20in%2084%20non%E2%80%90small%20cell%20lung%20cancer%20(NSCLC)%20cell%20lines%20as%20platform%20for%20cancer%20proteomics%20and%20biomarker%20discovery&amp;author=R.%20Ummanni&amp;author=H.%20A.%20Mannsperger&amp;author=J.%20Sonntag&amp;author=M.%20Oswald&amp;author=A.%20K.%20Sharma&amp;volume=1844&amp;issue=5&amp;publication_year=2014&amp;pages=950-959&amp;pmid=24361481&amp;doi=10.1016/j.bbapap.2013.11.017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0132">
<cite>

Voedisch, B.
, &amp; 
Thie, H.
 (2010). Size exclusion chromatography. In Kontermann R. &amp; Dübel S. (Eds.), Antibody engineering (pp. 607–612). Springer. 10.1007/978-3-642-01144-3_38
</cite> [<a href="https://doi.org/10.1007/978-3-642-01144-3_38" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Antibody%20engineering&amp;author=B.%20Voedisch&amp;author=H.%20Thie&amp;publication_year=2010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0133">
<cite>

Wang, M.
, 
Lin, H.
, 
Lin, H.
, 
Du, P.
, &amp; 
Zhang, S.
 (2024). From species to varieties: How modern sequencing technologies are shaping medicinal plant identification. Genes, 16(1), 16. 10.3390/genes16010016

</cite> [<a href="https://doi.org/10.3390/genes16010016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11765323/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39858563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes&amp;title=From%20species%20to%20varieties:%20How%20modern%20sequencing%20technologies%20are%20shaping%20medicinal%20plant%20identification&amp;author=M.%20Wang&amp;author=H.%20Lin&amp;author=H.%20Lin&amp;author=P.%20Du&amp;author=S.%20Zhang&amp;volume=16&amp;issue=1&amp;publication_year=2024&amp;pages=16&amp;pmid=39858563&amp;doi=10.3390/genes16010016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0135">
<cite>

Wiese, C.
, 
Dray, E.
, 
Groesser, T.
, 
San Filippo, J.
, 
Shi, I.
, 
Collins, D. W.
, &amp; 
Schild, D.
 (2007). Promotion of homologous recombination and genomic stability by RAD51AP1 via RAD51 recombinase enhancement. Molecular cell, 28(3), 482–490. 10.1016/j.molcel.2007.08.027

</cite> [<a href="https://doi.org/10.1016/j.molcel.2007.08.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2169287/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17996711/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20cell&amp;title=Promotion%20of%20homologous%20recombination%20and%20genomic%20stability%20by%20RAD51AP1%20via%20RAD51%20recombinase%20enhancement&amp;author=C.%20Wiese&amp;author=E.%20Dray&amp;author=T.%20Groesser&amp;author=J.%20San%20Filippo&amp;author=I.%20Shi&amp;volume=28&amp;issue=3&amp;publication_year=2007&amp;pages=482-490&amp;pmid=17996711&amp;doi=10.1016/j.molcel.2007.08.027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0136">
<cite>

Wilkins, M. R.
, 
Sanchez, J. C.
, 
Gooley, A. A.
, 
Appel, R. D.
, 
Humphery‐Smith, I.
, 
Hochstrasser, D. F.
, &amp; 
Williams, K. L.
 (1996). Progress with proteome projects: Why all proteins expressed by a genome should be identified and how to do it. Biotechnology and Genetic Engineering Reviews, 13(1), 19–50. 10.1080/02648725.1996.10647923

</cite> [<a href="https://doi.org/10.1080/02648725.1996.10647923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8948108/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnology%20and%20Genetic%20Engineering%20Reviews&amp;title=Progress%20with%20proteome%20projects:%20Why%20all%20proteins%20expressed%20by%20a%20genome%20should%20be%20identified%20and%20how%20to%20do%20it&amp;author=M.%20R.%20Wilkins&amp;author=J.%20C.%20Sanchez&amp;author=A.%20A.%20Gooley&amp;author=R.%20D.%20Appel&amp;author=I.%20Humphery%E2%80%90Smith&amp;volume=13&amp;issue=1&amp;publication_year=1996&amp;pages=19-50&amp;pmid=8948108&amp;doi=10.1080/02648725.1996.10647923&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0137">
<cite>

Yates, J. R., III.
 (2011). A century of mass spectrometry: From atoms to proteomes. Nature Methods, 8(8), 633–637. 10.1038/nmeth.1659
</cite> [<a href="https://doi.org/10.1038/nmeth.1659" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=A%20century%20of%20mass%20spectrometry:%20From%20atoms%20to%20proteomes&amp;author=J.%20R.%20Yates&amp;volume=8&amp;issue=8&amp;publication_year=2011&amp;pages=633-637&amp;doi=10.1038/nmeth.1659&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0138">
<cite>

Zeller, K. A.
, 
Bowden, R. L.
, &amp; 
Leslie, J. F.
 (2004). Population differentiation and recombination in wheat scab populations of Gibberella zeae from the United States. Molecular Ecology, 13(3), 563–571. 10.1046/j.1365-294X.2004.02098.x

</cite> [<a href="https://doi.org/10.1046/j.1365-294X.2004.02098.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14871361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Population%20differentiation%20and%20recombination%20in%20wheat%20scab%20populations%20of%20Gibberella%20zeae%20from%20the%20United%20States&amp;author=K.%20A.%20Zeller&amp;author=R.%20L.%20Bowden&amp;author=J.%20F.%20Leslie&amp;volume=13&amp;issue=3&amp;publication_year=2004&amp;pages=563-571&amp;pmid=14871361&amp;doi=10.1046/j.1365-294X.2004.02098.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0139">
<cite>

Zhang, L.
, 
Liu, D.
, 
Guo, X.
, 
Yang, W.
, 
Sun, J.
, 
Wang, D.
, &amp; 
Zhang, A.
 (2011). Investigation of genetic diversity and population structure of common wheat cultivars in northern China using DArT markers. BMC Genetics, 12, Article 42. 10.1186/1471-2156-12-42

</cite> [<a href="https://doi.org/10.1186/1471-2156-12-42" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3114777/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21569312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genetics&amp;title=Investigation%20of%20genetic%20diversity%20and%20population%20structure%20of%20common%20wheat%20cultivars%20in%20northern%20China%20using%20DArT%20markers&amp;author=L.%20Zhang&amp;author=D.%20Liu&amp;author=X.%20Guo&amp;author=W.%20Yang&amp;author=J.%20Sun&amp;volume=12&amp;publication_year=2011&amp;pages=42&amp;pmid=21569312&amp;doi=10.1186/1471-2156-12-42&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270027-bib-0140">
<cite>

Zhu, H.
, 
Bilgin, M.
, 
Bangham, R.
, 
Hall, D.
, 
Casamayor, A.
, 
Bertone, P.
, &amp; 
Snyder, M.
 (2001). Global analysis of protein activities using proteome chips. Science, 293(5537), 2101–2105. 10.1126/science.1062191

</cite> [<a href="https://doi.org/10.1126/science.1062191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11474067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Global%20analysis%20of%20protein%20activities%20using%20proteome%20chips&amp;author=H.%20Zhu&amp;author=M.%20Bilgin&amp;author=R.%20Bangham&amp;author=D.%20Hall&amp;author=A.%20Casamayor&amp;volume=293&amp;issue=5537&amp;publication_year=2001&amp;pages=2101-2105&amp;pmid=11474067&amp;doi=10.1126/science.1062191&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>
Supplementary Figure 1
</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12254572/bin/TPG2-18-e70027-s001.jpg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70027-s001.jpg</a><sup> (30KB, jpg) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>There are no original data associated with this article. Referenced data are available in the literature.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70027"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70027.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12254572/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12254572/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12254572%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12254572/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12254572/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12254572/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40646713/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12254572/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40646713/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12254572/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12254572/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="lrFGKLcOBRht7MNXH73bHJLktyAhT9apgHaXMNYwNnEoZZa90JIntfhI10Y5FgRV">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
