
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Unbiased mapping of cereblon neosubstrate landscape by high-throughput proteomics - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AFC18AF3244305AFC1001F03E9CF.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368047/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Unbiased mapping of cereblon neosubstrate landscape by high-throughput proteomics">
<meta name="citation_author" content="Martin Steger">
<meta name="citation_author_institution" content="NEOsphere Biotechnologies GmbH, Martinsried, Germany">
<meta name="citation_author" content="Gisele Nishiguchi">
<meta name="citation_author_institution" content="Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA">
<meta name="citation_author" content="Qiong Wu">
<meta name="citation_author_institution" content="Department of Surgery, St. Jude Children’s Research Hospital, Memphis, TN USA">
<meta name="citation_author" content="Bjoern Schwalb">
<meta name="citation_author_institution" content="NEOsphere Biotechnologies GmbH, Martinsried, Germany">
<meta name="citation_author" content="Bachuki Shashikadze">
<meta name="citation_author_institution" content="NEOsphere Biotechnologies GmbH, Martinsried, Germany">
<meta name="citation_author" content="Kevin McGowan">
<meta name="citation_author_institution" content="Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA">
<meta name="citation_author" content="Marisa Actis">
<meta name="citation_author_institution" content="Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA">
<meta name="citation_author" content="Anup Aggarwal">
<meta name="citation_author_institution" content="Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA">
<meta name="citation_author" content="Zhe Shi">
<meta name="citation_author_institution" content="Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA">
<meta name="citation_author" content="Jeanine Price">
<meta name="citation_author_institution" content="Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA">
<meta name="citation_author" content="Anand Mayasundari">
<meta name="citation_author_institution" content="Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA">
<meta name="citation_author" content="Lei Yang">
<meta name="citation_author_institution" content="Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA">
<meta name="citation_author" content="Anastasia H Bednarz">
<meta name="citation_author_institution" content="NEOsphere Biotechnologies GmbH, Martinsried, Germany">
<meta name="citation_author" content="Sophie Machata">
<meta name="citation_author_institution" content="NEOsphere Biotechnologies GmbH, Martinsried, Germany">
<meta name="citation_author" content="Tobias Graef">
<meta name="citation_author_institution" content="NEOsphere Biotechnologies GmbH, Martinsried, Germany">
<meta name="citation_author" content="Denis Bartoschek">
<meta name="citation_author_institution" content="NEOsphere Biotechnologies GmbH, Martinsried, Germany">
<meta name="citation_author" content="Vadim Demichev">
<meta name="citation_author_institution" content="Department of Biochemistry, Charité – Universitätsmedizin Berlin, Berlin, Germany">
<meta name="citation_author" content="Uli Ohmayer">
<meta name="citation_author_institution" content="NEOsphere Biotechnologies GmbH, Martinsried, Germany">
<meta name="citation_author" content="Jun Yang">
<meta name="citation_author_institution" content="Department of Surgery, St. Jude Children’s Research Hospital, Memphis, TN USA">
<meta name="citation_author_institution" content="Department of Pathology and Laboratory Medicine, College of Medicine, The University of Tennessee Health Science Center, Memphis, TN USA">
<meta name="citation_author_institution" content="College of Graduate Health Sciences, University of Tennessee Health Science Center, Memphis, TN USA">
<meta name="citation_author" content="Henrik Daub">
<meta name="citation_author_institution" content="NEOsphere Biotechnologies GmbH, Martinsried, Germany">
<meta name="citation_author" content="Zoran Rankovic">
<meta name="citation_author_institution" content="Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA">
<meta name="citation_author_institution" content="Present Address: Centre for Protein Degradation, The Institute of Cancer Research, London, UK">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7773">
<meta name="citation_doi" content="10.1038/s41467-025-62829-0">
<meta name="citation_pmid" content="40835825">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368047/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368047/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368047/pdf/41467_2025_Article_62829.pdf">
<meta name="description" content="Molecular glue degraders (MGDs) are small molecules that co-opt the ubiquitin-proteasome system to induce degradation of target proteins, including those considered undruggable. Their discovery remains challenging due to the lack of rational design ...">
<meta name="og:title" content="Unbiased mapping of cereblon neosubstrate landscape by high-throughput proteomics">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Molecular glue degraders (MGDs) are small molecules that co-opt the ubiquitin-proteasome system to induce degradation of target proteins, including those considered undruggable. Their discovery remains challenging due to the lack of rational design ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368047/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368047">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62829-0"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62829.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368047%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368047/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368047/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368047/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 20;16:7773. doi: <a href="https://doi.org/10.1038/s41467-025-62829-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62829-0</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Unbiased mapping of cereblon neosubstrate landscape by high-throughput proteomics</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Steger%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Martin Steger</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Martin Steger</span></h3>
<div class="p">
<sup>1</sup>NEOsphere Biotechnologies GmbH, Martinsried, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Steger%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Steger</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nishiguchi%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Gisele Nishiguchi</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Gisele Nishiguchi</span></h3>
<div class="p">
<sup>2</sup>Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nishiguchi%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Gisele Nishiguchi</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Qiong Wu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Qiong Wu</span></h3>
<div class="p">
<sup>3</sup>Department of Surgery, St. Jude Children’s Research Hospital, Memphis, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qiong Wu</span></a>
</div>
</div>
<sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schwalb%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Bjoern Schwalb</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Bjoern Schwalb</span></h3>
<div class="p">
<sup>1</sup>NEOsphere Biotechnologies GmbH, Martinsried, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schwalb%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bjoern Schwalb</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shashikadze%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Bachuki Shashikadze</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Bachuki Shashikadze</span></h3>
<div class="p">
<sup>1</sup>NEOsphere Biotechnologies GmbH, Martinsried, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shashikadze%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bachuki Shashikadze</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McGowan%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Kevin McGowan</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Kevin McGowan</span></h3>
<div class="p">
<sup>2</sup>Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22McGowan%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kevin McGowan</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Actis%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Marisa Actis</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Marisa Actis</span></h3>
<div class="p">
<sup>2</sup>Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Actis%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marisa Actis</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aggarwal%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Anup Aggarwal</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Anup Aggarwal</span></h3>
<div class="p">
<sup>2</sup>Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aggarwal%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anup Aggarwal</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Zhe Shi</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Zhe Shi</span></h3>
<div class="p">
<sup>2</sup>Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shi%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhe Shi</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Price%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Jeanine Price</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Jeanine Price</span></h3>
<div class="p">
<sup>2</sup>Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Price%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jeanine Price</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mayasundari%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Anand Mayasundari</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Anand Mayasundari</span></h3>
<div class="p">
<sup>2</sup>Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mayasundari%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anand Mayasundari</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Lei Yang</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Lei Yang</span></h3>
<div class="p">
<sup>2</sup>Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lei Yang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bednarz%20AH%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Anastasia H Bednarz</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Anastasia H Bednarz</span></h3>
<div class="p">
<sup>1</sup>NEOsphere Biotechnologies GmbH, Martinsried, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bednarz%20AH%22%5BAuthor%5D" class="usa-link"><span class="name western">Anastasia H Bednarz</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Machata%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Sophie Machata</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Sophie Machata</span></h3>
<div class="p">
<sup>1</sup>NEOsphere Biotechnologies GmbH, Martinsried, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Machata%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sophie Machata</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Graef%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Tobias Graef</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Tobias Graef</span></h3>
<div class="p">
<sup>1</sup>NEOsphere Biotechnologies GmbH, Martinsried, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Graef%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tobias Graef</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bartoschek%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Denis Bartoschek</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Denis Bartoschek</span></h3>
<div class="p">
<sup>1</sup>NEOsphere Biotechnologies GmbH, Martinsried, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bartoschek%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Denis Bartoschek</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Demichev%20V%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Vadim Demichev</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Vadim Demichev</span></h3>
<div class="p">
<sup>4</sup>Department of Biochemistry, Charité – Universitätsmedizin Berlin, Berlin, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Demichev%20V%22%5BAuthor%5D" class="usa-link"><span class="name western">Vadim Demichev</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ohmayer%20U%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Uli Ohmayer</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Uli Ohmayer</span></h3>
<div class="p">
<sup>1</sup>NEOsphere Biotechnologies GmbH, Martinsried, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ohmayer%20U%22%5BAuthor%5D" class="usa-link"><span class="name western">Uli Ohmayer</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Jun Yang</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Jun Yang</span></h3>
<div class="p">
<sup>3</sup>Department of Surgery, St. Jude Children’s Research Hospital, Memphis, TN USA </div>
<div class="p">
<sup>5</sup>Department of Pathology and Laboratory Medicine, College of Medicine, The University of Tennessee Health Science Center, Memphis, TN USA </div>
<div class="p">
<sup>6</sup>College of Graduate Health Sciences, University of Tennessee Health Science Center, Memphis, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jun Yang</span></a>
</div>
</div>
<sup>3,</sup><sup>5,</sup><sup>6,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Daub%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Henrik Daub</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Henrik Daub</span></h3>
<div class="p">
<sup>1</sup>NEOsphere Biotechnologies GmbH, Martinsried, Germany </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Daub%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Henrik Daub</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rankovic%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Zoran Rankovic</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Zoran Rankovic</span></h3>
<div class="p">
<sup>2</sup>Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA </div>
<div class="p">
<sup>7</sup>Present Address: Centre for Protein Degradation, The Institute of Cancer Research, London, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Rankovic%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zoran Rankovic</span></a>
</div>
</div>
<sup>2,</sup><sup>7,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>NEOsphere Biotechnologies GmbH, Martinsried, Germany </div>
<div id="Aff2">
<sup>2</sup>Department of Chemical Biology and Therapeutics, St. Jude Children’s Research Hospital, Memphis, TN USA </div>
<div id="Aff3">
<sup>3</sup>Department of Surgery, St. Jude Children’s Research Hospital, Memphis, TN USA </div>
<div id="Aff4">
<sup>4</sup>Department of Biochemistry, Charité – Universitätsmedizin Berlin, Berlin, Germany </div>
<div id="Aff5">
<sup>5</sup>Department of Pathology and Laboratory Medicine, College of Medicine, The University of Tennessee Health Science Center, Memphis, TN USA </div>
<div id="Aff6">
<sup>6</sup>College of Graduate Health Sciences, University of Tennessee Health Science Center, Memphis, TN USA </div>
<div id="Aff7">
<sup>7</sup>Present Address: Centre for Protein Degradation, The Institute of Cancer Research, London, UK </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 15; Accepted 2025 Jul 31; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368047  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40835825/" class="usa-link">40835825</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Molecular glue degraders (MGDs) are small molecules that co-opt the ubiquitin-proteasome system to induce degradation of target proteins, including those considered undruggable. Their discovery remains challenging due to the lack of rational design strategies and limited throughput of unbiased proteome-wide screening approaches. To address this gap, we develop a high-throughput proteomics platform based on label-free, data-independent acquisition mass spectrometry (DIA-MS), enabling integrated proteomics and ubiquitinomics profiling. Screening a diverse set of 100 cereblon (CRBN)-recruiting ligands on this platform leads to identification of a broad array of novel degraders and neosubstrates. Subsequent hit validation and structure-degradation relationship analyses guided by global proteomics reveal highly selective and potent phenyl glutarimide-based degraders targeting previously uncharacterized neosubstrates such as KDM4B, G3BP2 and VCL; none of which contain the classical CRBN β-hairpin degron. These findings underscore the power of unbiased high-throughput proteomics in MGD drug discovery and reveal a substantially expanded CRBN neosubstrate landscape beyond that defined by classical immunomodulatory imid drugs (IMiDs).</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Phenotypic screening, Proteomic analysis</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">Molecular glue degraders (MGD) offer a way to target undruggable proteins, but their discovery is challenging. Here, the authors develop a high-throughput proteomics platform for MGD drug discovery, revealing a much larger cereblon neosubstrate space than initially thought.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Targeted protein degradation (TPD) is a rapidly emerging drug discovery paradigm that relies on engagement of normal cellular proteolytic processes to degrade disease-associated proteins<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. The two currently most advanced TPD approaches employ proteolysis targeting chimeras (PROTACs)<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup> or molecular glue degraders (MGDs)<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. MGDs are small molecules that can bind and alter the specificity of an E3 ligase, resulting in the recruitment, ubiquitination, and subsequent degradation of new, non-physiological substrates, commonly referred to as neosubstrates. The immunomodulatory imid drugs (IMiDs) thalidomide, lenalidomide, and pomalidomide, clinically used as the first-line therapy for multiple myeloma, were the first drugs found to exert their therapeutic effect through such a mechanism<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. IMiDs bind to cereblon (CRBN), a substrate recognition subunit of CRL4<sup>CRBN</sup> ligase<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>, and induce degradation of the transcription factors IKZF1 and IKZF3 that are required for multiple myeloma cell survival<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Importantly, the neosubstrate recruitment is governed by MGD binding to the E3 ligase, creating a new, neomorphic recognition surface, and does not require a ligandable pocket<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. This mechanism provides an unprecedented and exciting opportunity to tackle currently undruggable therapeutic targets.</p>
<p id="Par4">Interestingly, despite their high structural similarity, lenalidomide and pomalidomide display different protein degradation profiles. While both degrade the transcription factors IKZF1/3, only lenalidomide induces degradation of CSNK1A1 (CK1α), indicating how a small difference in degrader molecular structure can significantly alter the neosubstrate specificity. Indeed, an ever-increasing number of neosubstrates have been discovered in recent years, with each IMiD displaying distinct degradation patterns, supporting the notion that E3 ligase substrate specificity can be modulated by structural alterations of the ligand. However, prospectively designing such molecules and rationalizing the structure-degradation relationship remains highly challenging.</p>
<p id="Par5">We recently reported the synthesis of a chemically diverse library of over 5000 CRBN ligands, which we screened against a large panel of cancer cell lines in viability assays, aiming at discovering novel degraders and cancer vulnerabilities. This proved to be a productive effort that delivered potent and selective degraders of proteins such as G1-to-S-phase transition (GSPT1)<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup> and casein kinase 1A (CK1α)<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup> that exhibited antiproliferative activity against a broad range of human cancer cell lines. However, despite these encouraging results, our phenotypic screening approach relied solely on cell viability as the assay readout. As a result, weak degraders or proteins that are not essential for the survival of cancer cells in our screening panel would be missed by this approach. To investigate the extent of such omissions and explore a potentially wider CRBN neosubstrate landscape, we sought to explore a more direct and proteome-wide screening approach.</p>
<p id="Par6">Mass spectrometry (MS)-based proteomics has undergone major advancements in recent years, driven by deep learning-enhanced software and next-generation MS hardware<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. These developments enable near-complete and highly reproducible proteome quantifications in high-throughput mode. A key advantage of MS-based proteomics is its unbiased, target- and E3-ligase agnostic screening capability, allowing for the simultaneous discovery of novel targets and evaluation of degrader selectivity. Despite its routine application in profiling selected degrader candidates, integrated, high-throughput screening workflows for MGD discovery remain underdeveloped.</p>
<p id="Par7">In this work, we develop a systematic and scalable high-throughput proteomics framework for MGD discovery. To demonstrate the utility of this platform, we perform screening of a diverse set of CRBN ligands, resulting in the discovery of several exquisitely potent, selective, and structurally distinct phenyl glutarimide-based degraders of previously unreported neosubstrates.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Deep proteomic screening of a molecular glue degrader library</h3>
<p id="Par8">Our IMiD-based molecular glue library was designed and synthesized based on the previously described principles and protocols<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. To expand the chemical diversity of our library beyond IMiDs, we incorporated novel CRBN-binding cores, including the recently reported phenyl-glutarimide (PG) and other proprietary scaffolds<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. For each chemical core, we generated a diverse set of chemical scaffolds by scanning vectors with different functionalities, applying an array of chemistries, and using a diverse set of building blocks to construct over 5000 compounds covering a broad range of chemical space and physicochemical properties. As an initial proof of concept to our screening workflow, we selected centroids from each core scaffold, which produced a set of 100 compounds representing the molecular structure coverage of the library (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a</a>). We excluded scaffolds known to degrade GSPT1 or CK1α based on our phenotypic screening campaign but included our previously reported CK1α (SJ10040) and GSPT1 (SJ6986) degraders as controls.</p>
<p id="Par9">We next designed a multilayered proteomics screening workflow aiming to expand the number of targetable proteins through CRBN-based degraders and gain deeper insight into their mechanisms of action. To maximize the detectable proteome, we selected two cancer cell lines, Huh-7 and NB-4, displaying complementary protein expression profiles and high sensitivity to CRBN-based MGDs (Supplementary Fig. <a href="#MOESM1" class="usa-link">1b</a>). Our primary screening workflow consisted of 24-h incubation in 96-well plates of the two cell lines with compound at 10 µM concentration, followed by semi-automated proteomics sample preparation and label-free, data-independent acquisition mass spectrometry (diaPASEF)<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. We profiled the set of one hundred compounds in triplicate across both cell lines, alongside 120 DMSO controls, resulting in 720 LC-MS runs. Following MS data processing and statistical analysis, compounds that induced &gt;25% reduction in protein abundance were designated as primary screening hits and subsequently re-tested at a 6-h time point using global proteomics. For hits that showed significant protein downregulation at both time points, we confirmed cullin-RING E3 ligase (CRL)-dependency by co-treating cells with the respective MGD and the NEDD8-activating enzyme inhibitor MLN4924<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. To further validate putative neosubstrates, we performed global ubiquitinomics. Proteins that exhibited rapid ubiquitination following MGD treatment were classified as bona fide CRBN neosubstrates (Fig. <a href="#Fig1" class="usa-link">1a</a> and Supplementary Data <a href="#MOESM3" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. The molecular glue degrader (MGD) proteomics screening workflow.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368047_41467_2025_62829_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6916/12368047/a571778ada53/41467_2025_62829_Fig1_HTML.jpg" loading="lazy" id="d33e494" height="984" width="723" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic of the proteomics screening workflow. A subset of 100 CRBN-based molecular glue degraders (MGDs) from a 5000-compound library was screened in Huh-7 and NB-4 cells (10 µM, 24 h) in 96-well plates. Each plate included 25 MGDs in triplicate. Proteins were quantified using Bruker timsTOF MS and processed with DIA-NN<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Significantly downregulated proteins (&gt;25%) were selected for 6-h re-testing with or without MLN4924, a neddylation inhibitor. Ubiquitinomics (K-GG peptide profiling) was used to validate MLN4924-dependent neosubstrates. Created in BioRender. BioRender.com/h02z778. <strong>b</strong> Count of significant downregulation events (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 1%) for known neosubstrates in Huh-7 or NB-4 cells after 24-h compound treatment (see Supplementary Data 2) for details. <strong>c</strong> Scatter plot showing suppression of MGD-induced downregulation (6 h) of known CRBN neosubstrates by MLN4924 (5 µM). <strong>d</strong> Volcano plots showing protein-level effects of SJ11322 (10 µM), from left to right: (i) 24 h, (ii) 6 h, and (iii) 6 h with MLN4924 (5 µM). Log<sub>2</sub> fold-change (x-axis) and log<sub>10</sub> standard error (y-axis) are shown. The chemical structure of SJ11322 is shown on the left side. Up- and downregulated proteins or ubiquitination sites (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 1%/5%) are colored in blue and red. <em>P</em>-value adjustments were made for multiple comparisons using the Benjamini-Hochberg procedure. n.s. not significant. The chemical structure of SJ11322 is shown. <strong>e</strong> Left: Volcano plot of SJ11322-induced ubiquitinated peptides (30 min). Upregulated sites (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 5%) are in blue, including four FIZ1 sites. Right: Correlation of proteome (x-axis) and ubiquitinome (y-axis) <em>t</em>-statistics. Significantly regulated proteins/sites are colored; turquoise highlights proteins downregulated at 24 h with increased ubiquitination at 30 min. n.s. not significant. <strong>f</strong> Heatmap of log<sub>2</sub> fold-changes in ubiquitination for CSNK1A1 and ZFP91. Asterisks (*) indicate statistically significant regulations (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 5%). <strong>g</strong> Correlation between site-specific ubiquitination—CSNK1A1-K51 (left; Spearman’s ρ = −1, <em>p</em>-value = 0.02) and ZFP91-K357 (right; Spearman’s ρ = −1, <em>p</em>-value = 0.08)—with corresponding 6-hour protein downregulation. Asterisks (*) indicate significant protein downregulation (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 1%); linear regression line and Spearman coefficient are shown. Source Data are provided as a Source Data file.</p></figcaption></figure><p id="Par10">Our proteomic screening campaign quantified an average of 10,200 protein groups in Huh-7 and 9500 in NB-4 cells, with over 98% of these retained for statistical testing after filtering for missing values. The median coefficient of variation (CV) for protein groups (<em>n</em> = 3) was about 6% in both cell lines (Supplementary Fig. <a href="#MOESM1" class="usa-link">2a, b</a>). This highlights that our single-shot MS-based screening platform detected a large proportion of the proteome expressed by a human cell line, with very high data completeness and excellent quantification precision<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>.</p>
<p id="Par11">We first classified the compounds based on the number of significantly induced protein downregulations. About 25% of them did not modulate proteins in either cell line, while approximately 50% induced specific changes, with 10 or fewer proteins displaying a significant reduction in abundance. The remaining compounds were more promiscuous, inducing numerous protein regulations, some of which may reflect secondary effects resulting from prolonged drug exposure (Supplementary Fig. <a href="#MOESM1" class="usa-link">2c</a>). Our control degraders, SJ6986 (GSPT1) and SJ41478 (promiscuous degrader), were excluded from the neosubstrate analyses.</p>
<p id="Par12">We identified 35 out of 49 previously reported neosubstrates from the literature<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Of these, 33 were retained after filtering for data completeness, and 25 were significantly downregulated after 24 h of treatment with at least one compound (Supplementary Data <a href="#MOESM4" class="usa-link">2</a>). Among these, RAB28, FIZ1, ZBTB16, and ZFP91 emerged as the most frequently downregulated targets across both cell lines. In contrast, Ikaros family zinc finger proteins IKZF1, IKZF2, and IKZF3 were exclusively detected and downregulated in NB-4 cells, consistent with their immune cell-specific expression (Fig. <a href="#Fig1" class="usa-link">1b</a>)<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Strong IKZF1 degraders accordingly induced broader proteome changes in NB-4 cells, suggesting secondary effects due to inhibition of IKZF1-mediated transcriptional programs (Supplementary Fig. <a href="#MOESM1" class="usa-link">2d</a>).</p>
<p id="Par13">Seventeen of the 25 previously reported neosubstrates showed significant downregulation at the 6-h time point, and co-treatment with MLN4924 rescued their levels, consistent with a CRL-dependent mechanism (Fig. <a href="#Fig1" class="usa-link">1c</a>). In addition to confirming previously reported CRBN neosubstrates, our screening uncovered highly selective probes for several of them, including LIMD1, GZF1, and FIZ1 (Fig. <a href="#Fig1" class="usa-link">1d</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>)<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>.</p>
<p id="Par14">Given that intracellular ubiquitination at endogenous levels has not been demonstrated for the majority of previously reported CRBN neosubstrates, we next conducted a series of global ubiquitinomics analyses<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. Since drug-induced protein ubiquitination can occur within minutes<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, we employed a 30-minute treatment time without a proteasome inhibitor to capture ubiquitination dynamics under physiological conditions. These experiments confirmed degrader-induced ubiquitination of several previously reported neosubstrates, including LIMD1, GZF1, PPIL4, and FIZ1, thereby validating our screening strategy (Fig. <a href="#Fig1" class="usa-link">1e</a> and Supplementary Figs. <a href="#MOESM1" class="usa-link">3</a>, <a href="#MOESM1" class="usa-link">4</a>).</p>
<p id="Par15">Our site-specific ubiquitination analysis further enabled a detailed investigation of neosubstrate ubiquitination patterns. A key observation was the frequent occurrence of ubiquitination in the absence of detectable protein degradation, indicating that ubiquitination alone is not always sufficient to trigger proteolysis. Notably, specific lysine residues on individual neosubstrates served as priming sites for degradation, showing the strongest correlation with protein downregulation. For instance, ubiquitination at CSNK1A-K51 emerged as a critical marker, with only a ~4-fold increase sufficient to induce detectable protein degradation. While weaker degraders primarily promoted ubiquitination at K51 and to a lesser extent at K179, the most potent degrader (SJ10040) induced ubiquitination at seven additional sites, suggesting an additive effect on degradation efficiency. Similarly, the extent of ZFP91-K357 ubiquitination showed the strongest correlation with protein loss (Fig. <a href="#Fig1" class="usa-link">1f, g</a>). Echoing prior observations with BRD4-BD2 site-selective ubiquitination by MZ1<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, our findings identified key lysine residues that act as a functional ubiquitination hotspot driving neosubstrate degradation in a native cellular context.</p>
<p id="Par16">Collectively, these results highlight that the combination of our MGD library design with MS-based proteomics and ubiquitinomics can identify and validate high-quality degrader probes for poorly characterized neosubstrates and provide insights into the mechanistic regulation of intracellular neosubstrate ubiquitination.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Systematic discovery of novel CRBN neosubstrates</h3>
<p id="Par17">We next analyzed our dataset for putative novel CRBN neosubstrates, initially focusing on Huh-7 cells. Out of the 100 compounds tested at 24 h, 32 resulted in a decrease in cellular abundance of at least one protein by more than 25%, leading to a preliminary list of 84 novel neosubstrate candidates. The 32 hit compounds were retested at the 6-h time point, confirming the downregulation of 26 target candidates (by 16 compounds). UHRF1, ZNF430, ZNF644, DHX37, and CCNK were just below statistical significance or our 25%-fold change cut-off in the 24-h dataset but showed significant and robust downregulation at 6 h. We therefore included them in the follow-up experiments (Supplementary Data <a href="#MOESM5" class="usa-link">3</a>). For each neosubstrate candidate, we selected at least one representative compound and assessed CRL dependency by treating cells with MLN4924 alongside the degrader. This combination substantially attenuated the downregulation of 23 proteins (to less than 70%), supporting their classification as putative novel CRBN neosubstrates (Fig. <a href="#Fig2" class="usa-link">2a</a>). In contrast, we also observed several CRL-independent downregulation events, such as those involving the cytochrome P450 enzyme CYP2J2 and oxysterol-binding protein-related protein 8 (OSBPL8), likely reflecting CRBN-independent off-target effects (Supplementary Fig. <a href="#MOESM1" class="usa-link">5</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Discovery of novel neosubstrates in Huh-7 cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368047_41467_2025_62829_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6916/12368047/fc5be6a233e6/41467_2025_62829_Fig2_HTML.jpg" loading="lazy" id="d33e667" height="993" width="748" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Scatter plot illustrating newly identified CRBN neosubstrates in Huh-7 cells. The plot displays all proteins that showed significant downregulation following treatment with 10 µM MGD for 6 h (x-axis) alongside their corresponding fold-changes under co-treatment with 5 µM MLN4924 (y-axis). Proteins regulated in a neddylation-dependent manner are highlighted in blue, while those regulated in a neddylation-independent manner are indicated in red. <strong>b</strong> Volcano plots depicting protein (left) and ubiquitinated peptide (right) quantifications for the KDM4B degrader SJ41564 (10 µM), with log<sub>2</sub> fold-change on the x-axis and standard error on the y-axis (log<sub>10</sub> scale). Significant up- and downregulations of proteins (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 1%) or ubiquitination sites (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 5%) are colored in blue and red, respectively. <strong>c</strong> Summary of proteome (x-axis) and ubiquitinome (y-axis) <em>t</em>-statistic comparisons for all newly validated neosubstrates in Huh-7 cells. <em>T</em>-statistics for ubiquitination sites mapped to the same protein were averaged, and only significantly upregulated sites were included in the analysis. Volcano plots of protein quantifications for SJ42101 (<strong>d</strong>) and SJ06986 (<strong>e</strong>) after 24 h of treatment. Significantly up- and downregulated proteins (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 1%) are colored in blue and red, respectively. The proteome (x-axis) and ubiquitinome (y-axis) <em>t</em>-statistic comparison plots are shown (bottom). Statistically significant up- and downregulations in the proteome (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 1%) and the ubiquitinome (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 5%) are colored in blue and red, respectively. <em>P</em>-value adjustments were made for multiple comparisons using the Benjamini-Hochberg procedure. Proteins that are both significantly downregulated after a 24-h compound treatment and harbor upregulated ubiquitination sites at 30 min of compound treatment are colored in turquoise. n.s. not significant. Source Data are provided as a Source Data file.</p></figcaption></figure><p id="Par18">For further validation, we subjected the 23 CRL-dependent neosubstrate candidates to quantitative ubiquitinomics. This revealed a total of 16 novel CRBN neosubstrates, including enzymes (Lysine-specific histone demethylase 4B (KDM4B)<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>), regulators of transcription (CCR4-NOT transcription complex subunit 4 (CNOT4)<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup> and the zinc finger protein ZBTB7B<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>), and tumor suppressors (Folliculin (FLCN)<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>) (Fig. <a href="#Fig2" class="usa-link">2b, c</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">6</a> and Supplementary Data <a href="#MOESM5" class="usa-link">3</a>).</p>
<p id="Par19">Among the newly identified neosubstrates, we noted several members of established protein complexes, including CSNK1A1 and its interaction partners FAM83G and FAM83B, components of the CDK-activating kinase (CAK) complex (CDK7, CCNH, and MNAT1), and the CDK13-CCNK pair (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>)<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>–<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. We hypothesize that the recruitment of a single complex member to MGD-bound CRBN positions associated proteins in close spatial proximity, thereby facilitating their ubiquitination as bystander neosubstrates<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>–<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>.</p>
<p id="Par20">In the case of the CSNK1A1-FAM83B/G complex, CSNK1A1 is the established direct neosubstrate<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>, and consistent with this, we did not identify any β-hairpin G-loop motifs in FAM83G or FAM83B.</p>
<p id="Par21">Similarly, within the CAK complex, MNAT1 appears to be the direct MGD target, as indicated by its stronger downregulation and ubiquitination relative to CDK7 and CCNH. This is further supported by recent reports demonstrating CRBN recruitment of MNAT1 via a β-hairpin G-loop recognition motif, which is absent in the other CAK complex members<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>.</p>
<p id="Par22">We observed a similar pattern for the CDK13-CCNK pair, in which case CDK13 degradation was much more pronounced. The presence of a G-loop motif in CDK13, but not in CCNK, supports its classification as the direct CRBN neosubstrate. Notably, this mechanism diverges from previously reported CCNK degraders, which act through a drug-induced interaction of CDK12 or CDK13 with DDB1<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>–<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>.</p>
<p id="Par23">We identified two compounds (SJ42101 and SJ6986) that downregulated MNAT1. While SJ42101 downregulated only 13 proteins, the potent GSPT1/GSPT2 degrader SJ6986 affected thousands, obscuring direct degradation events due to secondary effects from disrupted protein biosynthesis<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>,<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. To differentiate between direct target degradation and indirect GSPT1-mediated effects, we conducted global ubiquitinomics with both compounds. SJ42101 induced ubiquitination of 7 neosubstrates, whereas SJ6986 triggered the ubiquitination of GSPT1, GSPT2, their complex partner ETF1<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, and the proteins MNAT1, CDK7, RERE, GZF1, WIZ and ZBTB7B (Fig. <a href="#Fig2" class="usa-link">2d, e</a>). This demonstrates that our integrated MS-based screening approach enables the specific extraction of primary degrader targets from complex proteomic profiles, such as those induced by GSPT1 degradation.</p>
<p id="Par24">To assess potential cell line-specific responses of MGDs, we extended our proteomic screening efforts to NB-4 cells. While we could replicate most CRL-dependent protein downregulations identified in Huh-7 cells (e.g., KDM4B or CNOT4), some effects appeared to be cell type-specific. Notably, ZBTB7B was downregulated by SJ42101 only in Huh-7 cells, whereas SJ10040 selectively reduced levels of the ubiquitously expressed kinase CSNK1E in NB-4 cells, despite comparable expression levels in both cell lines (Supplementary Fig. <a href="#MOESM1" class="usa-link">8a–c</a>). Similarly, the KDM4B degrader SJ41564 significantly downregulated the highly expressed cytoskeletal protein vinculin (VCL) only in NB-4, even though it was ubiquitinated in both cell lines (Supplementary Fig. <a href="#MOESM1" class="usa-link">8d</a>). These findings indicate that MGDs can elicit cell-type-specific degradation profiles, even for ubiquitously expressed proteins. While the underlying mechanisms remain to be elucidated, our data highlight the value of performing proteomic MGD screens across diverse cellular contexts to uncover cell-specific neosubstrates.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Overexpression of CRBN enhances neosubstrate identification</h3>
<p id="Par25">Our ubiquitinomics-based validation in Huh-7 cells identified 16 novel neosubstrates (Fig. <a href="#Fig2" class="usa-link">2c</a>), and we further validated VCL as a CRBN neosubstrate in NB-4 cells. However, we did not detect compound-induced ubiquitination for the remaining 11 neosubstrate candidates. These were ZBTB21, CSNK1E, ZNF316, CUL4A, ZNF644, PRXL2B, G3BP2, ZNF430, DHX37, PCYT1A and STK38 (Supplementary Data <a href="#MOESM6" class="usa-link">4</a>). These proteins were either expressed at low levels or exhibited only modest downregulation upon treatment, suggesting potential sensitivity limitations of our assay.</p>
<p id="Par26">We hypothesized that increasing CRBN expression could enhance cellular CRL4<sup>CRBN</sup> activity, thereby amplifying neosubstrate ubiquitination and degradation, as previously shown for substrates regulated by lenalidomide<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. We therefore generated a HEK293 cell line with stable CRBN overexpression (HEK293-CRBNoe), which exhibited about eight-fold higher levels of CRBN than the parental line (Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>). Proteomic profiling of SJ10040-treated HEK293 and HEK293-CRBNoe cells revealed a markedly enhanced neosubstrate degradation in the CRBN-overexpressing background. We confirmed candidates of the primary screen (e.g., ZBTB21 and CSNK1E) and identified numerous additional target candidates, including ZNF324, ZBTB41, STK35, and CHD7 (Fig. <a href="#Fig3" class="usa-link">3a</a>). Follow-up ubiquitinomics confirmed compound-induced ubiquitination for 13 neosubstrates, including four newly identified ones (Fig. <a href="#Fig3" class="usa-link">3b</a>, top panel).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Neosubstrate discovery in HEK293-CRBNoe cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368047_41467_2025_62829_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6916/12368047/1e8ec6660f3b/41467_2025_62829_Fig3_HTML.jpg" loading="lazy" id="d33e857" height="991" width="676" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Volcano plots showing protein quantifications of HEK293 (left) and HEK293-CRBNoe (right) in response to the degrader compound SJ10040. The x-axis represents log<sub>2</sub> fold change, while the y-axis shows standard error on a log<sub>10</sub> scale. Both cell lines were treated with the compound for 6 h. Significantly up- and downregulated proteins, as determined by LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup> (FDR = 1%), are indicated in blue and red, respectively. The chemical structure of SJ10040 is shown on the right side. n.s. not significant. <strong>b</strong> Comparison plot of the <em>t</em>-statistics of the proteome (x-axis) and the ubiquitinome (y-axis) for the two MGDs SJ10040 (top) and SJ42101 (bottom) in HEK293-CRBNoe cells. The <em>t</em>-statistics of significantly upregulated ubiquitination sites mapping to the same gene were averaged. Statistically significant up- and downregulations of the proteome (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 1%) and the ubiquitinome (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 5%) are colored in blue and red, respectively. Proteins that were significantly downregulated after 6 h of compound treatment and exhibited upregulated ubiquitination sites at 30 min of compound treatment are highlighted in turquoise. <em>P</em>-value adjustments were made for multiple comparisons using the Benjamini-Hochberg procedure. Neosubstrates that were identified and validated in HEK293-CRBNoe are indicated in bold. n.s. not significant. <strong>c</strong> Correlation between site-specific ubiquitination—CSNK1A1-K51 (left; Spearman’s ρ = −0.9, <em>p</em>-value = 0.0002), MNAT1-K176 (middle; Spearman’s ρ = −0.9, <em>p</em>-value = 0.0004), and ZFP91-K357 (right; Spearman’s ρ = −0.9, <em>p</em>-value = 0.0003) on the x-axis—and corresponding protein downregulation after 6 h of compound treatment (y-axis). The different colors indicate the cell line, and asterisks (*) statistically significant protein downregulation, and a best-fitting linear regression line, along with the Spearman correlation coefficient, is shown. <strong>d</strong> Sankey diagram summarizing all novel neosubstrates identified and validated through ubiquitinomics in Huh-7, NB-4, and HEK293-CRBNoe cells. Compounds are displayed on the left side of the diagram, and the corresponding cell lines are on the right side. Asterisks (*) denote predicted G-loop motifs, while † and ‡ proteins previously reported as CRBN binders in Refs. <sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup> and <sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>, respectively. Source Data are provided as a Source Data file.</p></figcaption></figure><p id="Par27">Proteomics combined with ubiquitinomics of the remaining seven compounds in HEK293-CRBNoe cells confirmed their enhanced response to MGDs, further validating CUL4A, ZNF316, PRXL2B, G3BP2, ZNF644, ZNF430, DHX37, and PCYT1A as novel neosubstrates (Supplementary Figs. <a href="#MOESM1" class="usa-link">10</a>, <a href="#MOESM1" class="usa-link">11</a>). Interestingly, the promiscuous degrader SJ42101 yielded about 30 neosubstrates, including many novel ones, such as the zinc finger proteins ZNF121, ZNF239, ZNF408, and ZNF507, the peptidase ZUP1, and the mRNA 3’-end processing factor CSFT2T (Fig. <a href="#Fig3" class="usa-link">3b</a>, bottom panel).</p>
<p id="Par28">Similar to previously reported observations, we identified ubiquitination events that are either productive (leading to protein degradation) or non-productive (not resulting in protein degradation)<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. The non-productive events included validated neosubstrates (e.g., ZNF827 or RERE) as well as numerous potential novel CRBN targets (e.g., STK3/4, ZNF136, ZNF358, ZNF419, ZNF773, and others). These findings suggest that non-productive ubiquitination could potentially be converted into productive degradation through broader compound screening campaigns.</p>
<p id="Par29">The systematic profiling of different MGDs by proteomics and ubiquitinomics across different cell lines allowed a cross-comparison of site-specific ubiquitination and protein degradation in different biological contexts. As for HuH-7, we confirmed CSNK1A1-K51 and ZFP91-K375 ubiquitination as priming events for protein degradation in NB-4 and in HEK293 cells. Similarly, ubiquitination of MNAT1 at K176 strongly correlated with protein downregulation across different cell lines (Fig. <a href="#Fig3" class="usa-link">3c</a>).</p>
<p id="Par30">Collectively, our integrated MS-based strategy revealed 50 novel neosubstrates from a set of 100 compounds, including 17 proteins without a predicted GSPT1-like G-loop, demonstrating a much broader target scope for CRBN-based degraders than previously recognized (Fig. <a href="#Fig3" class="usa-link">3d</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>, and Supplementary Data <a href="#MOESM7" class="usa-link">5</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title">Impact of MGD structure on degradation specificity and selectivity: structure-degradation relationship</h3>
<p id="Par31">Almost all screened compounds (98 out of 100) either contained classical IMiD core structures as present in thalidomide and lenalidomide or possessed a phenyl-glutarimide (PG) core recently introduced as an alternative CRBN-binding warhead (Fig. <a href="#Fig4" class="usa-link">4a</a>)<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Our comprehensive data collection allowed us to explore correlations between MGD chemical structure and cellular degradation activities. To do that, we first grouped all compounds that degraded known or newly identified neosubstrates in Huh-7 and NB-4 cells according to whether they contained either an IMiD-like or PG core structure, followed by hierarchical clustering of protein fold-changes (i.e., MGD vs. DMSO) within both subsets. Notably, most classical IMiD targets, including IKZF1, ZFP91, CSNK1A1, GSPT1, GZF1, or RAB28, were selectively degraded by MGDs featuring IMiD-like cores. In contrast, LIMD1 and novel neosubstrates, including KDM4B or FLCN, were exclusively targeted by PG-based compounds, which also exhibited a much higher overall neosubstrate selectivity (Fig. <a href="#Fig4" class="usa-link">4b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>). A third category of neosubstrates included proteins such as FIZ1, ZBTB39, and the novel neosubstrate G3BP2, which were degraded to a similar extent by both PG-based and IMiD-like MGDs. However, while the IMiD-like G3BP2 degrader SJ42229 co-depleted several known IMiD targets, none of these were affected by the selective PG-based G3BP2 degrader SJ41824, further highlighting the higher selectivity of PG-based MGDs (Fig. <a href="#Fig4" class="usa-link">4c</a>). Given the association between IMiD-induced teratogenicity and SALL4 degradation<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>, we extended our analysis to the human testicular embryonal carcinoma cell line SUSA, which expresses high levels of SALL4. Strikingly, all 10 IMiD-like compounds tested induced strong downregulation of SALL4, while only 1 out of 10 PG-based compounds caused a moderate decrease (Fig. <a href="#Fig4" class="usa-link">4d</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Structure-degradation relationship analysis of screened MGDs.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368047_41467_2025_62829_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6916/12368047/dbc0cf9f376e/41467_2025_62829_Fig4_HTML.jpg" loading="lazy" id="d33e996" height="994" width="678" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> t-SNE plot of MGDs clustered by core structure: lenalidomide (Lena), thalidomide (Thal), and phenyl-glutarimide (PG). Each dot represents a compound; black dots indicate the 100 analogs selected for proteomics. Representative core structures are shown. <strong>b</strong> Heat map of log<sub>2</sub> fold-changes (MGD vs. DMSO) for known and novel neosubstrates in Huh-7 cells. Compounds are grouped by IMiD-like or PG-based cores. The top bar chart shows the number of significantly downregulated neosubstrates per compound. Red boxes highlight G3BP2 degradation by SJ42229 and SJ41824. Proteins with predicted G-loop motifs are indicated. <strong>c</strong> Volcano plots depicting protein quantifications for the G3BP2 SJ42229 (IMiD-like), SJ41824 and SJ41813 (PG-based) in NB-4 cells (10 µM, 24 h). The x-axis shows log<sub>2</sub> fold-changes, and the standard errors are plotted on the y-axis (log<sub>10</sub> scale). Significant changes (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 1%) are in red (down) and blue (up). IMiD-derived neosubstrates are highlighted for SJ42229. The compound structures are shown. n.s. not significant. <strong>d</strong> MS quantified SALL4 levels in SUSA cells after treatment with IMiD-like (blue) and PG-based (red) compounds (10 µM, 24 h). Box plots represent log<sub>2</sub>-transformed intensity values for four biological replicates per compound, with 15 DMSO-treated samples included as controls. Horizontal lines indicate the median (bold) and interquartile range (thin). <strong>e</strong> Western blot analysis of the indicated proteins in SK-N-BE(2)-C wild-type and CRBN knockout (CRBN<sup>KO</sup>) cells treated with SJ41813 (10 µM, 24 h). Shown are the results of one experiment. <strong>f</strong> Volcano plot of biotinylated proteins from TurboID-CRBN HEK293 cells treated with SJ41813, showing log<sub>2</sub> fold-changes (x-axis) and log<sub>10</sub> standard error (y-axis). Significantly enriched proteins (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 1%, <em>n</em> = 3) are labeled in blue. n.s. not significant. <strong>g</strong> Immunoblots for G3BP2 and USP10 in SK-N-BE(2)-C cells treated with increasing doses (in µM) of SJ41813 (24 h). Blots are representative of three independent experiments. <strong>h</strong> Quantification of G3BP2 and USP10 blots from (<strong>g</strong>). DC<sub>50</sub> values were calculated from band intensities (mean ± S.D., <em>n</em> = 3) and normalized to DMSO controls. <strong>i</strong> Volcano plot of ubiquitinated peptides in NB-4 cells treated with SJ41813 (30 min). Significantly upregulated peptides (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 5%; <em>n</em> = 4) are in blue; G3BP2 sites are highlighted. n.s. not significant. Source Data are provided as a Source Data file.</p></figcaption></figure><p id="Par32">Given their remarkable selectivity, we next focused on the in-depth characterization of the PG-based degraders of G3BP2 and KDM4B.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Characterization and chemical optimization of MGDs targeting G3BP2 and KDM4B</h3>
<p id="Par33">Given the advantage of high compound selectivity in defining starting points for further chemical optimization, we selected the PG-based degrader SJ41824 targeting G3BP2 for chemical analog testing. G3BP2 is an mRNA-binding protein that physically associates with both G3BP1 and USP10, playing a key role in protecting ribosomes from degradation under cellular stress<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. At present, no selective G3BP2 inhibitors have been reported. To establish a structure-degradation relationship around this chemotype and potentially identify more potent degraders, we evaluated additional SJ41824 analogs in NB-4 cells by global proteomic profiling. Among these, SJ41813 caused stronger and highly selective degradation of both G3BP2 and USP10, as determined by Western blotting and proteomics (Fig. <a href="#Fig4" class="usa-link">4e</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">14a, b</a>). Notably, SJ41813 differs from SJ41824 only by the position of fluorine atoms on the phenyl ring (3,4- vs. 2,4-difluoro), underscoring the impact of subtle structural changes on degradation efficacy<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. CRISPR-Cas9-mediated knockout of CRBN in SK-N-BE(2)-C cells completely abrogated G3BP2 degradation by SJ41813 (Fig. <a href="#Fig4" class="usa-link">4e</a>), confirming a CRBN-dependent mechanism.</p>
<p id="Par34">To demonstrate ternary complex formation, we engineered a HEK293 cell line with doxycycline-inducible expression of TurboID-CRBN<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup> and treated it with SJ41813. Affinity-purification followed by quantitative mass spectrometry revealed a &gt;6-fold enrichment of G3BP2, alongside weaker enrichment of binding partners such as USP10, pointing to G3BP2 as the direct CRBN neosubstrate (Fig. <a href="#Fig4" class="usa-link">4f</a>). Consistently, G3BP2 showed greater degradation potency (DC<sub>50</sub> = 2.2 µM) than USP10 (DC<sub>50</sub> = 10 µM) (Fig. <a href="#Fig4" class="usa-link">4g, h</a>). Ubiquitinomics further revealed two SJ41813-induced ubiquitination sites on G3BP2 but none on USP10 in NB-4 cells (Fig. <a href="#Fig4" class="usa-link">4i</a>), a result corroborated by enhanced G3BP2 ubiquitination in SJ41813-treated HEK293-CRBNoe cells (Supplementary Fig. <a href="#MOESM1" class="usa-link">14c</a>). Together, these findings establish G3BP2 as a bona fide CRBN neosubstrate, aligning with recent reports<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>.</p>
<p id="Par35">The second PG-based degrader selected for analog optimization was SJ41564, which induced potent and selective degradation of KDM4B. KDM4B is a histone demethylase that targets H3K9 and H3K36, which are key histone marks involved in gene regulation and chromatin organization<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. It shares high sequence homology with other KDM4 family members, including KDM4A and KDM4C, which are frequently overexpressed in cancer<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Despite extensive efforts in both industry and academia<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>, selective inhibitors for individual KDM4 isoforms have remained elusive, largely due to the high structural similarity of their JmjC catalytic domains<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. SJ41564 selectively degraded KDM4B without affecting the levels of other KDM4 family members, including KDM4A, KDM4C, and KDM4D, in both NB-4 and Huh-7 cells (Fig. <a href="#Fig5" class="usa-link">5a</a>). Interestingly, in SK-N-BE(2)-C cells, KDM4B degradation was accompanied by a modest increase in KDM4A expression, suggesting a potential compensatory response specific to this neuroblastoma cell line (Supplementary Fig. <a href="#MOESM1" class="usa-link">15a</a>). A similar mechanism has been reported in rhabdosarcoma cells, where KDM4A regulation mitigates the impact of KDM4B silencing on cell growth<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. In-depth characterization of KDM4B and VCL as CRBN neosubstrates.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368047_41467_2025_62829_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6916/12368047/e260f362c8c0/41467_2025_62829_Fig5_HTML.jpg" loading="lazy" id="d33e1161" height="1000" width="745" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Volcano plot of the proteome from SJ41564-treated NB-4 cells (6 h). Log<sub>2</sub> fold-change (x-axis) and log<sub>10</sub> standard error (y-axis) are shown. KDM4 family members are highlighted; KDM4B is significantly downregulated (red, LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 1%). n.s. not significant. <strong>b</strong> Immunoblots of KDM4B and α-tubulin in SK-N-BE(2)-C cells treated with increasing concentrations of SJ41564 (nM, 24 h). Representative of three experiments. <strong>c</strong> Quantification of KDM4B levels from (<strong>b</strong>). The N-methylated negative control SJ48108 was included. DC<sub>50</sub> values were calculated from band intensities (mean ± S.D., <em>n</em> = 3), normalized to DMSO. <strong>d</strong> Top: Schematic of KDM4B protein architecture and truncated fragments (A–F). Bottom: Immunoblots of FLAG-tagged KDM4B fragments in CRBN-overexpressing SK-N-BE(2)-C cells treated with SJ41564 (10 µM, 24 h), shown from one experiment. Fragments E and F are labeled in red and indicate degradation by SJ41564. <strong>e</strong> MS quantification of KDM4B in HEK293-TurboID-CRBN cells treated with bortezomib (0.5 µM) or SJ41564 + bortezomib (120 min). KDM4B was significantly enriched (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 1%, n = 3 per condition). <strong>f</strong> AlphaScreen assay showing dose-dependent ternary complex formation between CRBN–DDB1 and KDM4B (PHD–DTD) upon SJ41564 treatment. EC<sub>50</sub> = 4.7 µM. No signal was observed with the negative control SJ48108. DMSO controls are shown in gray. Data represent mean ± S.D. of three technical replicates. <strong>g</strong> Heat map of log<sub>2</sub> fold-changes (vs. DMSO) for KDM4B and VCL in response to various MGD compounds by global proteomics. <strong>h</strong> Chemical structures of selected KDM4B and VCL degraders. <strong>i</strong> MS quantification of VCL in HEK293-TurboID-CRBN cells treated with bortezomib or SJ41564 + bortezomib (120 min, <em>n</em> = 3). VCL was significantly enriched (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 1%, <em>n</em> = 3 per condition). <strong>j</strong> Volcano plots of proteome (6 h) and ubiquitinome (30 min) in NB-4 cells treated with SJ46479. Significantly regulated proteins (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 1%) and ubiquitinated peptides (LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, FDR = 5%) are shown in red (down) and blue (up). VCL ubiquitination sites are highlighted. n.s. not significant. Source Data are provided as a Source Data file.</p></figcaption></figure><p id="Par36">To confirm the CRBN-dependent mechanism of SJ41564, we compared its activity to a structurally related, inactive analog (SJ48108), featuring an N-methyl modification. As expected, only SJ41564 induced a rapid and potent degradation of KDM4B, with a DC<sub>50</sub> of 36 nM and a D<sub>max</sub> of 92% at 10 µM. (Fig. <a href="#Fig5" class="usa-link">5b, c</a>, and Supplementary Fig. <a href="#MOESM1" class="usa-link">15b, c</a>). Beyond the initial four cell lines (HEK293, Huh-7, NB-4 and SK-N-BE(2)-C), SJ41564 also induced KDM4B degradation across six other cancer cell lines, highlighting its broad cellular activity (Supplementary Fig. <a href="#MOESM1" class="usa-link">15d</a>). Consistent with proteomics findings in NB-4 cells, SJ41564-induced KDM4B degradation in SK-N-BE(2)-C cells was fully blocked with the proteasomal inhibitor MG132, the NEDD8-activating enzyme inhibitor MLN4924, or the CRBN ligand lenalidomide. Moreover, KDM4B degradation was completely abolished in SK-N-BE(2)-C CRBN<sup>KO</sup> cells and occurred without changes in KDM4B mRNA levels, confirming a post-translational mechanism (Supplementary Fig. <a href="#MOESM1" class="usa-link">15e–g</a>). Together, these results establish that SJ41564 mediates KDM4B degradation through a CRBN- and ubiquitin–proteasome-dependent pathway.</p>
<p id="Par37">Having established CRBN dependency, we next sought to identify the region of KDM4B responsible for engaging with CRBN-bound SJ41564. The KDM4 proteins contain several structured domains, including the JmjN and JmjC catalytic domains, as well as tandem TUDOR and PHD domains (Fig. <a href="#Fig5" class="usa-link">5d</a>). While the JmjC domain is highly conserved among KDM4 members, the tandem TUDOR domains exhibit markedly lower sequence homology and are found only in KDM4A and C. To map the degron region, we generated a series of KDM4B protein fragments (A–F) and expressed them in SK-N-BE(2)-C cells stably expressing CRBN. Western blot analysis following incubation with SJ41564 revealed selective degradation of fragments E and F, in contrast to fragments A–D, implicating the tandem TUDOR domains in mediating CRBN-dependent degradation of KDM4B.</p>
<p id="Par38">TurboID-based proximity labeling confirmed SJ41564-induced ternary complex formation in HEK293 cells (Fig. <a href="#Fig5" class="usa-link">5e</a>). To further validate this interaction, we developed an AlphaScreen assay using a GST-KDM4B fragment containing the PHD and tandem TUDOR domains (PHD-DTD) and recombinant His-CRBN-DDB1. SJ41564 induced dose-dependent ternary complex formation with an EC<sub>50</sub> of 4.7 µM, confirming direct interaction between the glue, CRBN, and KDM4B (Fig. <a href="#Fig5" class="usa-link">5f</a>). No signal was observed with N-methylated negative control compound SJ48108, consistent with its inability to bind CRBN.</p>
<p id="Par39">Subsequent quantitative proteomics evaluation of SJ41564 analogs identified additional potent KDM4B degraders, revealing that the <em>meta-</em>position on the distal phenyl moiety is favored for cellular activity (Fig. <a href="#Fig5" class="usa-link">5g</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">16</a>).</p>
<p id="Par40">Strikingly, while para<em>-</em>substituted analogs exhibited little to no KDM4B degradation activity, several of these PG-based CRBN binders unexpectedly triggered a pronounced reduction in the cytoskeletal protein vinculin (VCL)<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. Notably, compounds such as SJ46479 and SJ47878 reduced VCL levels by 90% within 6 h, with high selectivity in both NB-4 and Caki-1 cells (Fig. <a href="#Fig5" class="usa-link">5g, h</a>). Further characterization of SJ46479 in Caki-1 cells revealed time-, dose-, and CRBN-dependent degradation of VCL, with a DC<sub>50</sub> of 255 nM and a D<sub>max</sub> of 96%, establishing it as a potent and selective VCL degrader (Supplementary Fig. <a href="#MOESM1" class="usa-link">17</a>).</p>
<p id="Par41">Finally, proximity labeling coupled to MS analysis confirmed intracellular ternary complex formation between VCL and CRBN. In NB-4 cells, SJ46479 triggered rapid ubiquitination of over 20 lysine residues on VCL following treatment of NB-4 cells with SJ46479 (Fig. <a href="#Fig5" class="usa-link">5i, j</a>). Structural mapping of these ubiquitination sites onto the predicted VCL protein model revealed a concentrated ubiquitination zone, offering insights into the spatial orientation of VCL within the CRBN E3 ligase complex (Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>).</p>
<p id="Par42">To identify the specific region of VCL directly involved in the ternary complex formation, we generated Caki-1 cell lines stably overexpressing FLAG-tagged constructs of each of the five distinct VCL domains (A-E) and assessed their protein levels by immunoblotting (Supplementary Fig. <a href="#MOESM1" class="usa-link">19a–c</a>). Upon treatment with SJ46479, we observed changes only in the level of D1, while the other four constructs remained unaffected (Supplementary Fig. <a href="#MOESM1" class="usa-link">19b</a>), suggesting that the D1 domain is the one involved in the direct interaction of VCL with the neomorphic SJ46479-CRBN surface. Interestingly, while the level of D1 was markedly low in untreated cells, it was found to be significantly increased in cells treated with either SJ46479 or the proteasome inhibitor MG132 (Supplementary Fig. <a href="#MOESM1" class="usa-link">19c</a>). This suggests that, while unstable on its own due to proteasomal degradation, D1 is stabilized by the SJ46479-induced complexation with CRBN. We speculate that the three ubiquitination sites in D1 fragment (K170, K216, and K219), out of over 20 identified in the full-length protein (Fig. <a href="#Fig5" class="usa-link">5i</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">18</a>), are not effectively ubiquitinated when in the ternary complex with SJ46479-CRBN, rendering SJ46479 incapable of inducing degradation and thereby resulting in D1 protein stabilization.</p>
<p id="Par43">In summary, our proteomics-driven screening strategy identified highly selective PG-based degraders for G3BP2, KDM4B, and VCL. This work further highlights the profound impact of subtle chemical modifications on CRBN-based degrader specificity and demonstrates the potential of systematic MGD discovery to uncover potent and selective degraders of previously unreported CRBN neosubstrates.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par44">In this study, we present an unbiased and highly sensitive screening workflow based on MS-based proteomics for the discovery of MGDs and their neosubstrates in intact cells. Our platform integrates compound profiling across multiple time points and cell lines, incorporates neddylation inhibition to distinguish direct and CRBN-dependent protein degradation events, and employs ubiquitinomics for rigorous target validation. By screening a chemically diverse set of 100 CRBN ligands, we identified and validated 50 previously unreported neosubstrates. Notably, many of these had been implicated in ternary complex formation in prior biochemical or cell-based assays, underscoring the robustness and biological relevance of our approach<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>.</p>
<p id="Par45">To enhance the sensitivity for neosubstrate identification, we utilized HEK293 cells overexpressing CRBN, which enabled us to validate weakly degraded neosubstrates such as ZBTB21 and CSNK1E, initially detected in Huh-7 and NB-4 cells. We anticipate that extending MGD screening to a biologically diverse panel of CRBN-overexpressing cell lines will further broaden the neosubstrate landscape of CRBN-based degraders. Interestingly, we detected cell-type-specific degradation of ubiquitously expressed proteins across the profiled cell lines. This differential degradation did not correlate with CRBN or neosubstrate expression levels, suggesting that additional factors, such as post-translational modifications (PTMs), subcellular localization, or cell-type-specific expression of accessory proteins within the ubiquitin-proteasome pathway, may influence neosubstrate engagement and degradation efficiency.</p>
<p id="Par46">Beyond neosubstrate identification and validation, our integrated proteomics screening workflow yields high-quality datasets that can inform subsequent chemical optimization efforts<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. For example, we identified SJ41824 as a selective degrader of G3BP2, a Ras–GAP SH3 domain binding protein implicated in stress granule formation and the pathogenesis of breast and prostate cancers<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>,<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. As no inhibitors targeting G3BP2 have been previously reported, we synthesized and profiled a focused set of SJ41824 analogs using our proteomics platform. This structure-degradation relationship analysis led to the discovery of SJ41813, a closely related regioisomer with enhanced G3BP2 degradation potency (DC<sub>50</sub> = 2.8 µM). USP10 was the only other protein regulated by SJ41813, which, according to our validation experiments, is a bystander rather than a direct neosubstrate.</p>
<p id="Par47">Another promising hit from our screening set was SJ41564, a potent and selective degrader of KDM4B. KDM4B is a member of the histone demethylase gene family and has long been considered an attractive target in cancer drug discovery. However, the development of selective KDM4B inhibitors has proven challenging, largely due to the conserved nature of the catalytic domain and the need to compete with the abundant cofactor 2-oxoglutarate<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Remarkably, SJ41564 robustly induces CRBN-dependent degradation of KDM4B in NB-4 cells, with a DC<sub>50</sub> of 36 nM and a D<sub>max</sub> of 92%, while sparing all other ~9000 quantified proteins. Domain mapping experiments using a series of Flag-tagged KDM4B truncation constructs identified the less conserved double Tudor domains as critical for degradation, potentially explaining the exceptional selectivity of SJ41564.</p>
<p id="Par48">Further proteomic profiling of SJ41564 analogs uncovered not only additional potent KDM4B degraders but highly selective degraders of the cytoskeletal vinculin (VCL), exemplified by SJ46479 (VCL DC<sub>50</sub> = 255 nM; D<sub>max</sub> = 96%). These findings align well with our broader observation that molecular glue degraders often exhibit steep structure-degradation relationships, highlighting that subtle chemical modifications can have a strong impact on neosubstrate engagement.</p>
<p id="Par49">In summary, our unbiased high-throughput proteomics screening approach led to the discovery of several exquisitely potent, selective, and structurally distinct phenyl glutarimide-based degraders of G3BP2, KDM4B, and VCL proteins. Notably, none of these newly identified neosubstrates contain the classical G-loop degron motif, suggesting that the scope of the CRBN-degradable proteome extends well beyond that defined by IMiD substrates. While the precise structural features governing this expanded neosubstrate recognition remain to be elucidated, our findings provide a compelling rationale for proteomics-driven screening of diverse compound libraries built on novel scaffolds. Such efforts hold significant promise for uncovering previously inaccessible targets within the disease-relevant and historically undruggable protein target space.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Reagents</h3>
<p id="Par50">MLN4924, bortezomib, 2-chloroacetamide, tris(2-carboxyethyl)phosphine hydrochloride (TCEP), sodium deoxycholate, Biotin, Na<sub>2</sub>HPO<sub>4</sub>, 3-(N-morpholino)propanesulfonic acid (MOPS), Tris(hydroxymethyl)-aminomethane, trifluoracetic acid, sodium chloride, formic acid, and acetonitrile were from Merck. MG132 (Cat# HY-13259) and Lenalidomide (Cat# HY-A0003) were purchased from MedChem Express (MCE). Protease inhibitor mix from ThermoFisher Scientific (A32955). Trypsin from Promega. PTMScan® HS Ubiquitin/SUMO Remnant Motif (K-ε-GG) Kit (#59322) from Cell Signaling Technology.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Cell culture, drug treatments, and cell lysis (Proteomics)</h3>
<p id="Par51">Huh-7, NB-4, HEK293T, and HEK293 cells were from Cytion. SUSA from DSMZ. Huh-7, HEK293T, and HEK293 cells were cultured in DMEM (VWR) supplemented with 10% FCS (Thermo Fisher Scientific). NB-4 cells were grown in RPMI (VWR) and 10% FCS. RH4 and RH30 cell lines were provided by Dr. Gerald Grosveld at St Jude. The SK-N-BE(2)-C (BE(2)-C) (Cat# CRL-2268), Rh30, Kasumi-1 (Cat# CRL-2724), and Caki-1 (Cat# HTB-46) cell lines were obtained from ATCC. The SIMA (Cat# ACC 164), EW-8 (Cat# ACC 493), and SKNO-1 (Cat# ACC 690) cell lines were obtained from DSMZ. SK-N-BE(2)-C, SIMA, Rh30, Rh4, EW-8, and SKNO-1 cells were cultured in RPMI 1640 (Corning, Cat# 10040CM) supplemented with 10% fetal bovine serum (Gibco, Cat# A5256701) and 1% penicillin/streptomycin (Gibco, Cat# 15140122). The Kasumi-1 cells were cultured in RPMI-1640 Medium supplemented with 20% fetal bovine serum and 1% penicillin/streptomycin. The Caki-1 cells were cultured in McCoy’s 5a Medium (ATCC, Cat# 30-2007) supplemented with 10% fetal bovine serum and 1% penicillin/streptomycin. All compounds were dissolved in DMSO to generate a 1000x stock solution. Compounds were plated in 96-well plates according to a randomized layout using an Opentrons OT-2. 25 compounds were allocated on each plate (in triplicate), along with 15 DMSO controls. Cells were treated for the specified amount of time, washed with PBS, and harvested using iST lysis buffer (PreOmics). The lysates were heated to 95 °C for 10 min while shaking in a Thermomixer (Eppendorf). The rest of the workflow (tryptic digestion and desalting of peptides) was performed according to the manufacturer’s instructions.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Ubiquitinomics sample preparation</h3>
<p id="Par52">Cell treatment and sample preparation were done according to our recently established protocol with a few modifications<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. In brief, cells (four replicates for both DMSO and compound treatments) were cultured in 6-well plates and treated for 30 min with the specified compounds, followed by lysis with SDC buffer. Protein concentrations were determined using the BCA assay (Merck-Millipore), and the proteins were digested overnight at 37 °C using a 100:1 protein:trypsin ratio (Promega). After digestion, immunoprecipitation (IP) buffer (50 mM MOPS pH 7.2, 10 mM Na<sub>2</sub>HPO<sub>4</sub>, 50 mM NaCl) was added to the samples together with K-GG antibody-bead conjugate, followed by a 2 h incubation on a rotor wheel. Beads washing and peptide elution was performed according to the manufacturer’s instructions. The peptide eluate was desalted using in-house prepared, 200 µl two-plug C18 StageTips (3 M EMPORE<sup>TM</sup>)<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Enrichment of biotinylated proteins (TurboID-MS)</h3>
<p id="Par53">After inducing the expression of TurboID-CRBN overnight using 100 ng/ml of doxycycline, the cells were treated with the indicated degrader compounds for 2 h (in the presence of 50 mM biotin and 0.5 µM bortezomib). Cells were lysed with cold RIPA buffer (1% NP-40, 0.5% SDC, 0.1% SDS, 50 mM Tris-HCl pH = 7.5, 150 mM NaCl, freshly supplemented with Protease inhihitor mix (Pierce)) and the lysates were cleared by centrifugation (20,000 g, 10 min). 10 µl of MagReSyn® Streptavidin (Resyn Biosciences) was added to the samples, followed by 1 h incubation on a rotor wheel. Beads were washed 5 times with RIPA buffer and proteins subsequently eluted using UDM (n-Undecyl-Beta-Maltoside, Anatrace) lysis buffer (0.05% UDM, 75 mM Tris-HCl pH 8.5, 40 mM CAA, 10 mM TCEP). The lysates were incubated at 90 °C for 10 min, followed by overnight protein digestion at 37 °C using trypsin. The peptides were desalted using in-house prepared, 200 µl two-plug C18 StageTips (3 M EMPORE<sup>TM</sup>) and resuspended in 0.1% formic acid before MS analysis.</p></section><section id="Sec14"><h3 class="pmc_sec_title">LC-MS/MS measurements</h3>
<p id="Par54">Peptides were loaded on 35 cm reversed-phase columns (75 µm inner diameter, packed in-house with C<sub>18</sub>-AQ 1.9 µm resin [ReproSil-Pur®, Dr. Maisch GmbH]). The column temperature was maintained at 55 °C using a column oven. Either an EASY-nLC 1200 or a Vanquish Neo system (both ThermoFisher) was directly coupled online with the mass spectrometer (timsTOF pro2, HT or Ultra2, Bruker) via a nano-electrospray source. The LC flow rate was 300 nl/min, and the complete gradient was 60 min (proteomics) or 45 min (ubiquitinomics). Data acquisition was done using diaPASEF<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> or slicePASEF<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup> (for ubiquitinomics) methods. The diaPASEF method consisted of a 24 × 2 acquisition scheme (ion mobility (IM) range from 0.65 to 1.35 and m/z range from 300 to 1500) and was generated with py_diAID<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Plasmids and cloning</h3>
<p id="Par55">pCMV-FLAG-KDM4B (1–1096), pCMV-FLAG-KDM4B (1–130), pCMV-FLAG-KDM4B (130–356), pCMV-FLAG-KDM4B (336–916), pCMV-FLAG-KDM4B (336–1096), and pCMV-FLAG-KDM4B (917–1096) were produced by cloning the truncated KDM4B fragments individually into the pCMV vector<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Generation of stable cell lines</h3>
<p id="Par56">The plasmid vectors for stable cell line generation (CRBN and TurboID-CRBN) were from VectorBuilder. The plasmid vectors pGenlenti-FLAG-VCL-A (1–252), pGenlenti-FLAG-VCL-B (253-485), pGenlenti-FLAG-VCL-C (486–717), pGenlenti-FLAG-VCL-D (718–835), and pGenlenti-FLAG-VCL-E (836–1066) were from GenScript. The plasmid vector pLC-Flag-CRBN-P2A-Hygro for the generation of a stable BE2C cell line overexpressing CRBN was a gift from Eva Gottwein (Addgene plasmid #124303). The plasmid vector lentiCRISPR v2 puro - CRBN (#1), used for generating the BE2C cell line with CRBN knockout, was a gift from Günter Schneider (Addgene plasmid # 166240)<sup><a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup>. HEK293 cells with stable gene overexpression were generated by lentiviral-mediated infection of cells according to published protocols<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. In brief, HEK293T cells were co-transfected with psPAX2, pMD2.G, pRSV-Rev, and a lentiviral construct for CRBN expression under the control of an EF1A promoter. The supernatant was harvested after 48 h of cell transfection and passed through a 0.45 µm filter. HEK293 cells were infected with the supernatant for 24 h, followed by selection of transduced cells with puromycin (1 µg/ml). CRBN overexpression was verified by MS-based proteome profiling. HEK293-TurboID-CRBN cells were generated using the PiggyBAC system. Cells were co-transfected with a vector expressing the PiggyBAC transposase and a second vector expressing both TurboID-CRBN (tetracycline-inducible) and rtTA/tTS (Reverse tetracycline-responsive transcriptional activator M2/Tetracycline transcriptional silencer), including a puromycin selection marker. Cells stably transfected with the construct were selected with 1 µg/ml puromycin, and doxycycline-dependent expression of TurboID-CRBN was verified by mass spectrometry.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Transfection of KDM4B plasmids</h3>
<p id="Par57">FLAG-tagged constructs of different KDM4B fragments were transfected into SK-N-BE(2)-C cells using Lipofectamine 3000 Transfection Reagent (Thermo Fischer Scientific, Cat# L3000008). Briefly, on day 1, 1 × 10<sup>5</sup> cells were seeded in a single well of a 6-well plate. On day 2, cells were transfected with 2.5 µg of DNA according to the manufacturer’s instructions. On day 3, the transfection medium was replaced with fresh medium containing 10 µM of SJ41564. On day 4, the cells were harvested for western blot analysis after 24 h of treatment.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Total RNA extraction, cDNA synthesis, and RT-qPCR</h3>
<p id="Par58">Total RNA was extracted from cells and tumor tissues using the RNeasy Mini Kit (Qiagen, Cat # 74106) according to the manufacturer’s instructions. After extraction, 0.5–1 µg of total RNA cDNA was used to synthesize cDNA using SuperScript IV First-Strand Synthesis System (Invitrogen, Cat # 18091050) according to the manufacturer’s instructions. RT-qPCR was performed using SYBR Green PCR Master Mix (Applied Biosystems, Cat # 4367660) and the 7500 Real-Time PCR System (Applied Biosystems). The relative expression of each gene was normalized to GAPDH mRNA using the ΔΔCT method. The sequences of primers used are listed below.</p>
<p id="Par59">KDM4B-F: TCACCAGCCACATCTACCAG</p>
<p id="Par60">KDM4B-R: GATGTCCCCACGCTTCAC</p>
<p id="Par61">GAPDH-F: AACGGGAAGCTTGTCATCAATGGAAA</p>
<p id="Par62">GAPDH-R: GCATCAGCAGAGGGGGCAGAG</p></section><section id="Sec19"><h3 class="pmc_sec_title">Antibodies and Western blot</h3>
<p id="Par63">Cells were lysed on ice using a lysis buffer (0.1 M Tris-HCl, pH 6.8, 200 mM dithiothreitol, 0.01% bromophenol blue, 4% sodium dodecyl sulfate, and 20% glycerol). Samples were sonicated at 4 °C for 10 s, and then boiled at 95 °C for 10 min. Cell lysates were separated on a 4–15% precast polyacrylamide gel (Bio-Rad, Cat # 4568086) and transferred to PVDF membranes (Bio-Rad, Cat # 1704272) pre-activated with methanol. Membranes were blocked in a solution of 5% milk in PBST buffer (0.1% TWEEN in PBS) and incubated for 1 h at room temperature, and incubated overnight with primary antibodies at 4 °C under gentle horizontal shaking. The next day, membranes were washed with PBST buffer and incubated with anti-mouse or anti-rabbit HRP-conjugated secondary antibodies (1:5,000). Membranes were washed with PBST buffer, incubated with SuperSignal West Pico PLUS Chemiluminescent Substrate (ThermoFisher, Cat # 34580), and developed using an Odyssey Fc Imaging System (LI-COR Corp.). Blots were analyzed using Image Studio 6.0 (LI-COR Biotech). Antibodies and working concentrations are provided within the Source Data file.</p></section><section id="Sec20"><h3 class="pmc_sec_title">KDM4B protein construct and AlphaScreen assay</h3>
<section id="Sec21"><h4 class="pmc_sec_title">GST-KDM4B(PHD-DTD)</h4>
<p id="Par64">For AlphaScreen assay development, a KDM4B plant homeodomain-double Tudor domain (PHD-DTD) construct was designed comprising residues P729–P1031, with an N-terminal GST tag connected via a ‘DGGSGSG’ linker. The codon-optimized KDM4B(PHD-DTD) gene fragment was synthesized, cloned into the pET30a(+) vector for E. coli expression, and the protein was subsequently produced by Genscript. The final storage buffer for the purified protein contained 50 mM Tris-HCl, 150 mM NaCl, 10% glycerol, pH 8.0.</p></section><section id="Sec22"><h4 class="pmc_sec_title">Ternary Complex Assay (KDM4B(PHD-DTD)/CRBN-DDB1)</h4>
<p id="Par65">His-tagged CRBN-DDB1 protein was prepared following the procedure reported by Matyskiela et al.<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. All reagents were diluted in assay buffer comprising 1× PBS, 0.1% BSA, and 0.05% Tween 20. An ECHO 650 (Labcyte Inc.) acoustic dispenser was used to generate a 10-point dilution curve from DMSO stocks of the compounds directly into a 384-well OptiPlate (PerkinElmer, cat# 6007290), giving a final DMSO concentration of 0.1%. The assay mixture contained 200 nM His-tagged CRBN-DDB1 and 50 nM GST-tagged KDM4B(PHD-DTD). AlphaScreen glutathione-coated donor and AlphaScreen nickel chelate acceptor beads were purchased from PerkinElmer (cat# 6765300 and 6760141, respectively). Briefly, to a 384-well OptiPlate containing 5× compound in triplicate was added 5 µL of a 5× solution of His-CRBN-DDB1 and GST-KDM4B(PHD-DTD) and then incubated at room temperature (rt) for 1 h. After incubation, 10 µL nickel chelate acceptor (20 µg/mL final concentration) and 10 µL glutathione donor beads (20 µg/mL final concentration) were added under subdued lighting. The plate was sealed and mixed on a MixMate (eppendorf) for 1 h at rt, and then luminescence detection was collected on an Envision plate reader (PerkinElmer). A sigmoidal curve was used to fit the data using GraphPad Prism software (v10.3.1).</p></section></section><section id="Sec23"><h3 class="pmc_sec_title">Raw data processing</h3>
<p id="Par66">Western blot results were analyzed using Licor Image Studio 6.0 (LI-COR Biotech). RT-PCR and DC50 figures were generated using GraphPad Prism version 10 (GraphPad Software). MS raw files were analyzed using DIA-NN<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. Proteomics raw files were processed with v1.8.2beta9 or 1.9.1 (chemical analog testing), and ubiquitinomics raw data was processed with v1.8.2beta27. Reviewed UniProt entries (human, SwissProt 10-2022 [9606]) were used as a protein sequence database for DIA-NN searches. One missed cleavage, a maximum of one variable modification (Oxidation of methionines), and N-terminal excision of methionine were allowed. Carbamidomethylation of cysteines was set as a fixed modification, and K-GG (UniMod: 121) was added in case of ubiquitinomics. All data processings were carried out using library-free analysis mode in DIA-NN. --tims-scan was added as an additional command in case of ubiquitinomics.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Statistical data analysis</h3>
<p id="Par67">DIA-NN outputs were further processed with R. Peptide precursor quantifications with missing values in more than 50% of samples, or &lt;33% of the DMSO-treated samples (for proteomics, &lt;25% of compound-treated samples in case of ubiquitinomics) were discarded. Protein abundances were calculated using both precursor and fragment ion intensities; K-GG peptide abundances were calculated based on precursor ion intensity levels using the MaxLFQ algorithm<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>, as implemented in the DIA-NN R package (<a href="https://github.com/vdemichev/diann-rpackage/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/vdemichev/diann-rpackage/</a>). Complete missing cases in any of the conditions tested were rescued by accepting low-quality precursors (i.e., <em>q</em>-value &gt; 0.01), where possible. K-GG peptide to site mapping was done using reviewed entries of the human UniProt database (SwissProt, release 10-2022). The protein (or peptide) intensities were normalized by median scaling and corrected for variance drift over time (if present) using the principal components (derived from principal component analysis (PCA)) belonging to DMSO samples. Subsequently, protein (or peptide) intensities were subjected to statistical testing with variance and log fold-change moderation using LIMMA<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>. <em>p</em>-values corrected for multiple testing were used to assess significance for proteomics (<em>q</em>-value &lt; 0.01) and ubiquitinomics (<em>q</em>-value &lt; 0.05). For comparing proteome and ubiquitinome data, identifications were mapped at the gene level.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Identification of G-loop motifs</h3>
<p id="Par68">G-loop motifs were identified by structural comparison with the G-loop of GSPT1. Structural data for the human proteome were obtained from models predicted by AlphaFold<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>,<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. From these models, all possible 8-amino acid segments containing glycine at the sixth position were extracted. Each of these candidate sequences was structurally superimposed on the G-loop of GSPT1, and the root mean square deviation (RMSD) was calculated. Sequences with RMSD ≤ 1.0 Å were considered structurally similar to the GSPT1 G-loop and categorized as G-loop motifs. To assess the structural accessibility of the identified G-loop motifs, the surface exposure of individual amino acids within each loop was calculated using the pPSE metric (prediction-aware part-sphere exposure) as previously described<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. The median value of surface exposure was calculated for each loop, and loops with a median pPSE value greater than 5 were excluded from further analysis to prioritize structurally accessible motifs.</p></section><section id="Sec26"><h3 class="pmc_sec_title">General chemistry methods and materials</h3>
<p id="Par69">See Supplementary Information for chemical synthesis of compounds.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par70">Further information on research design is available in the <a href="#MOESM8" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec28"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (7.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62829_MOESM2_ESM.pdf</a><sup> (1.6MB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (1.1MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (28.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (32KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (26.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (55KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (822.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (337.6KB, pdf) </sup>
</div></div></section></section><section id="Sec29"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (5.6MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We are grateful for the support of the American Lebanese Syrian Associated Charities (ALSAC) and St. Jude Children’s Research Hospital. This work was also supported in part by the Alex’s Lemonade Stand Foundation Crazy 8 award. J.Y. was partly supported by American Cancer Society-Research Scholar (130421-RSG-17-071-01-TBG, J.Y.), the National Cancer Institute (1R01CA229739-01, 1R01CA266600-01A1 and 1R01CA289881-01A1, J.Y.), Comprehensive Cancer Center core grant CA021765, and the American Lebanese Syrian Associated Charities (ALSAC). The content is solely the authors’ responsibility and does not necessarily represent the official views of the National Institutes of Health. NEOsphere Biotechnologies GmbH gratefully acknowledges funding for this research from the German Federal Ministry of Education and Research (BMBF, grant number 16LW0372). We thank Dr. Jutta Fritz for her support in project management.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>M.S. designed and performed the experiments, analyzed and interpreted the data, prepared the figures, and wrote the manuscript. B.Sch. developed the data analysis pipeline and interpreted the data. B.Sh. analyzed and interpreted the data, edited the manuscript, and prepared the figures. U.O. analyzed and interpreted the data. H.D. and Z.R. conceived the study, designed the experiments, analyzed and interpreted the data, and wrote the manuscript. A.H.B., S.M., T.G., and D.B. performed the experiments and co-developed the ubiquitinomics and TurboID assays. V.D. provided software tools essential for raw data processing. G.N. designed the molecular glue library, selected compounds for proteomics-directed medicinal chemistry, and edited the manuscript. K.M. designed and synthesized compounds. Z.S., J.P., and L.Y. synthesized and purified compounds. A.A. obtained protein constructs, and M.A. developed ternary complex assays. A.M. performed assay profiling of compounds. Q.W. performed the experiments, analyzed and interpreted the data, edited the manuscript, and prepared the figures. Y.J. designed and supervised the study, interpreted the data, and was involved in manuscript writing and editing. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par71"><em>Nature Communications</em> thanks Danette Daniels, Tatsuya Sawasaki, and the other anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The mass spectrometry proteomics raw data generated in this study have been deposited in the MassIVE repository (<a href="https://massive.ucsd.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://massive.ucsd.edu/</a>) and can be accessed via proteome exchange with accession code <a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD059111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PXD059111</a>. DIA-NN is freely available for download at <a href="https://www.github.com/vdemichev/diann" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.github.com/vdemichev/diann</a>. <a href="#Sec29" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par72">M.S., U.O., B.Sch., B.Sh., D.B., A.H.B., S.M., T.G., and H.D. are employees and shareholders of NEOsphere Biotechnologies GmbH (Martinsried, Germany). Z.R. is an employee of the Institute of Cancer Research, London. V.D. is an employee and shareholder of Aptila Biotech GmbH. The remaining authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Martin Steger, Gisele Nishiguchi, Qiong Wu, Bjoern Schwalb.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Jun Yang, Email: jun.yang2@stjude.org.</p>
<p>Henrik Daub, Email: henrik.daub@neospherebiotech.com.</p>
<p>Zoran Rankovic, Email: zoran.rankovic@icr.ac.uk.</p></section><section id="sec30"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62829-0.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Schapira, M., Calabrese, M. F., Bullock, A. N. &amp; Crews, C. M. Targeted protein degradation: expanding the toolbox. <em>Nat. Rev. Drug Discov.</em><strong>18</strong>, 949–963 (2019).
</cite> [<a href="https://doi.org/10.1038/s41573-019-0047-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31666732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schapira,%20M.,%20Calabrese,%20M.%20F.,%20Bullock,%20A.%20N.%20&amp;%20Crews,%20C.%20M.%20Targeted%20protein%20degradation:%20expanding%20the%20toolbox.%20Nat.%20Rev.%20Drug%20Discov.18,%20949%E2%80%93963%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Nalawansha, D. A. &amp; Crews, C. M. PROTACs: an emerging therapeutic modality in precision medicine. <em>Cell Chem. Biol.</em><strong>27</strong>, 998–1014 (2020).
</cite> [<a href="https://doi.org/10.1016/j.chembiol.2020.07.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9424844/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32795419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nalawansha,%20D.%20A.%20&amp;%20Crews,%20C.%20M.%20PROTACs:%20an%20emerging%20therapeutic%20modality%20in%20precision%20medicine.%20Cell%20Chem.%20Biol.27,%20998%E2%80%931014%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Geiger, T. M., Schäfer, S. C., Dreizler, J. K., Walz, M. &amp; Hausch, F. Clues to molecular glues. <em>Curr. Res. Chem. Biol.</em><strong>2</strong>, 100018 (2022).</cite> [<a href="https://scholar.google.com/scholar_lookup?Geiger,%20T.%20M.,%20Sch%C3%A4fer,%20S.%20C.,%20Dreizler,%20J.%20K.,%20Walz,%20M.%20&amp;%20Hausch,%20F.%20Clues%20to%20molecular%20glues.%20Curr.%20Res.%20Chem.%20Biol.2,%20100018%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Moreau, P., Attal, M. &amp; Facon, T. Frontline therapy of multiple myeloma. <em>Blood</em><strong>125</strong>, 3076–3084 (2015).
</cite> [<a href="https://doi.org/10.1182/blood-2014-09-568915" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25838345/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moreau,%20P.,%20Attal,%20M.%20&amp;%20Facon,%20T.%20Frontline%20therapy%20of%20multiple%20myeloma.%20Blood125,%203076%E2%80%933084%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Ito, T. et al. Identification of a primary target of thalidomide teratogenicity. <em>Science</em><strong>327</strong>, 1345–1350 (2010).
</cite> [<a href="https://doi.org/10.1126/science.1177319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20223979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ito,%20T.%20et%20al.%20Identification%20of%20a%20primary%20target%20of%20thalidomide%20teratogenicity.%20Science327,%201345%E2%80%931350%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Lu, G. et al. The myeloma drug lenalidomide promotes the cereblon-dependent destruction of Ikaros proteins. <em>Science</em><strong>343</strong>, 305 (2014).
</cite> [<a href="https://doi.org/10.1126/science.1244917" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4070318/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24292623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu,%20G.%20et%20al.%20The%20myeloma%20drug%20lenalidomide%20promotes%20the%20cereblon-dependent%20destruction%20of%20Ikaros%20proteins.%20Science343,%20305%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Krönke, J. et al. Lenalidomide causes selective degradation of IKZF1 and IKZF3 in multiple myeloma cells. <em>Science</em><strong>343</strong>, 301–305 (2014).
</cite> [<a href="https://doi.org/10.1126/science.1244851" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4077049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24292625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kr%C3%B6nke,%20J.%20et%20al.%20Lenalidomide%20causes%20selective%20degradation%20of%20IKZF1%20and%20IKZF3%20in%20multiple%20myeloma%20cells.%20Science343,%20301%E2%80%93305%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Chamberlain, P. P. et al. Structure of the human Cereblon–DDB1–lenalidomide complex reveals basis for responsiveness to thalidomide analogs. <em>Nat. Struct. Mol. Biol.</em><strong>21</strong>, 803–809 (2014).
</cite> [<a href="https://doi.org/10.1038/nsmb.2874" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25108355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chamberlain,%20P.%20P.%20et%20al.%20Structure%20of%20the%20human%20Cereblon%E2%80%93DDB1%E2%80%93lenalidomide%20complex%20reveals%20basis%20for%20responsiveness%20to%20thalidomide%20analogs.%20Nat.%20Struct.%20Mol.%20Biol.21,%20803%E2%80%93809%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Sievers, Q. L. et al. Defining the human C2H2 zinc finger degrome targeted by thalidomide analogs through CRBN. <em>Science</em><strong>362</strong>, eaat0572 (2018).
</cite> [<a href="https://doi.org/10.1126/science.aat0572" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6326779/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30385546/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sievers,%20Q.%20L.%20et%20al.%20Defining%20the%20human%20C2H2%20zinc%20finger%20degrome%20targeted%20by%20thalidomide%20analogs%20through%20CRBN.%20Science362,%20eaat0572%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Nishiguchi, G. et al. Identification of potent, selective, and orally bioavailable small-molecule GSPT1/2 degraders from a focused library of cereblon modulators. <em>J. Med. Chem.</em><strong>64</strong>, 7296–7311 (2021).
</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.0c01313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8201443/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34042448/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nishiguchi,%20G.%20et%20al.%20Identification%20of%20potent,%20selective,%20and%20orally%20bioavailable%20small-molecule%20GSPT1/2%20degraders%20from%20a%20focused%20library%20of%20cereblon%20modulators.%20J.%20Med.%20Chem.64,%207296%E2%80%937311%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Chang, Y. et al. The orally bioavailable GSPT1/2 degrader SJ6986 exhibits in vivo efficacy in acute lymphoblastic leukemia. <em>Blood</em><strong>142</strong>, 629–642 (2023).
</cite> [<a href="https://doi.org/10.1182/blood.2022017813" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10447621/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37172201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chang,%20Y.%20et%20al.%20The%20orally%20bioavailable%20GSPT1/2%20degrader%20SJ6986%20exhibits%20in%20vivo%20efficacy%20in%20acute%20lymphoblastic%20leukemia.%20Blood142,%20629%E2%80%93642%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Nishiguchi, G. et al. Structure-activity relationship of potent, selective, and orally bioavailable molecular glue degraders of CK1α. <em>ACS Med. Chem. Lett.</em>10.1021/acsmedchemlett.4c00250 (2024).</cite> [<a href="https://doi.org/10.1021/acsmedchemlett.4c00250" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11472389/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39411539/" class="usa-link">PubMed</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Nishiguchi, G. et al. Selective CK1α degraders exert antiproliferative activity against a broad range of human cancer cell lines. <em>Nat. Commun.</em><strong>15</strong>, 1–17 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-44698-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10791743/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38228616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nishiguchi,%20G.%20et%20al.%20Selective%20CK1%CE%B1%20degraders%20exert%20antiproliferative%20activity%20against%20a%20broad%20range%20of%20human%20cancer%20cell%20lines.%20Nat.%20Commun.15,%201%E2%80%9317%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Demichev, V., Messner, C. B., Vernardis, S. I., Lilley, K. S. &amp; Ralser, M. DIA-NN: neural networks and interference correction enable deep proteome coverage in high throughput. <em>Nat. Methods</em>10.1038/s41592-019-0638-x (2020).</cite> [<a href="https://doi.org/10.1038/s41592-019-0638-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6949130/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31768060/" class="usa-link">PubMed</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Guzman, U. H. et al. Ultra-fast label-free quantification and comprehensive proteome coverage with narrow-window data-independent acquisition. <em>Nat. Biotechnol.</em>10.1038/S41587-023-02099-7 (2024).</cite> [<a href="https://doi.org/10.1038/s41587-023-02099-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11631760/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38302753/" class="usa-link">PubMed</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Wallmann, G. et al. AlphaDIA enables end-to-end transfer learning for feature-free proteomics. Preprint at <em>bioRxiv</em>10.1101/2024.05.28.596182 (2024).</cite>
</li>
<li id="CR17">
<span class="label">17.</span><cite>Min, J. et al. Phenyl-glutarimides: alternative cereblon binders for the design of PROTACs. <em>Angew. Chem. Int Ed. Engl.</em><strong>60</strong>, 26663–26670 (2021).
</cite> [<a href="https://doi.org/10.1002/anie.202108848" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8648984/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34614283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Min,%20J.%20et%20al.%20Phenyl-glutarimides:%20alternative%20cereblon%20binders%20for%20the%20design%20of%20PROTACs.%20Angew.%20Chem.%20Int%20Ed.%20Engl.60,%2026663%E2%80%9326670%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Meier, F. et al. diaPASEF: parallel accumulation–serial fragmentation combined with data-independent acquisition. <em>Nat. Methods</em>. 10.1038/s41592-020-00998-0 (2020).</cite> [<a href="https://doi.org/10.1038/s41592-020-00998-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33257825/" class="usa-link">PubMed</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Soucy, T. A. et al. An inhibitor of NEDD8-activating enzyme as a new approach to treat cancer. <em>Nature</em><strong>458</strong>, 732–736 (2009).
</cite> [<a href="https://doi.org/10.1038/nature07884" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19360080/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Soucy,%20T.%20A.%20et%20al.%20An%20inhibitor%20of%20NEDD8-activating%20enzyme%20as%20a%20new%20approach%20to%20treat%20cancer.%20Nature458,%20732%E2%80%93736%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Sinitcyn, P. et al. Global detection of human variants and isoforms by deep proteome sequencing. <em>Nat. Biotechnol.</em><strong>41</strong>, 1776–1786 (2023).
</cite> [<a href="https://doi.org/10.1038/s41587-023-01714-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10713452/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36959352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sinitcyn,%20P.%20et%20al.%20Global%20detection%20of%20human%20variants%20and%20isoforms%20by%20deep%20proteome%20sequencing.%20Nat.%20Biotechnol.41,%201776%E2%80%931786%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Donovan, K. A. et al. Thalidomide promotes degradation of SALL4, a transcription factor implicated in Duane Radial Ray syndrome. <em>eLife</em><strong>7</strong>, e38430 (2018).
</cite> [<a href="https://doi.org/10.7554/eLife.38430" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6156078/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30067223/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Donovan,%20K.%20A.%20et%20al.%20Thalidomide%20promotes%20degradation%20of%20SALL4,%20a%20transcription%20factor%20implicated%20in%20Duane%20Radial%20Ray%20syndrome.%20eLife7,%20e38430%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Verano, A. L. et al. Redirecting the neo-substrate specificity of cereblon-targeting PROTACs to helios. <em>ACS Chem. Biol.</em><strong>17</strong>, 2404–2410 (2022).
</cite> [<a href="https://doi.org/10.1021/acschembio.2c00439" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36007246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Verano,%20A.%20L.%20et%20al.%20Redirecting%20the%20neo-substrate%20specificity%20of%20cereblon-targeting%20PROTACs%20to%20helios.%20ACS%20Chem.%20Biol.17,%202404%E2%80%932410%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Li, D. et al. Discovery of a dual WDR5 and Ikaros PROTAC degrader as an anti-cancer therapeutic. <em>Oncogene</em><strong>41</strong>, 3328–3340 (2022).
</cite> [<a href="https://doi.org/10.1038/s41388-022-02340-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9189076/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35525905/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20D.%20et%20al.%20Discovery%20of%20a%20dual%20WDR5%20and%20Ikaros%20PROTAC%20degrader%20as%20an%20anti-cancer%20therapeutic.%20Oncogene41,%203328%E2%80%933340%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Oleinikovas, V., Gainza, P., Ryckmans, T., Fasching, B. &amp; Thomä, N. H. From thalidomide to rational molecular glue design for targeted protein degradation. <em>Annu. Rev. Pharmacol. Toxicol.</em><strong>64</strong>, 291–312 (2024).
</cite> [<a href="https://doi.org/10.1146/annurev-pharmtox-022123-104147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37585660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oleinikovas,%20V.,%20Gainza,%20P.,%20Ryckmans,%20T.,%20Fasching,%20B.%20&amp;%20Thom%C3%A4,%20N.%20H.%20From%20thalidomide%20to%20rational%20molecular%20glue%20design%20for%20targeted%20protein%20degradation.%20Annu.%20Rev.%20Pharmacol.%20Toxicol.64,%20291%E2%80%93312%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Jochem, M. et al. Degradome analysis to identify direct protein substrates of small-molecule degraders. <em>Cell Chem. Biol.</em><strong>32</strong>, 192-200.e6 (2024).</cite> [<a href="https://doi.org/10.1016/j.chembiol.2024.10.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39536762/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Xu, G., Paige, J. S. &amp; Jaffrey, S. R. Global analysis of lysine ubiquitination by ubiquitin remnant immunoaffinity profiling. <em>Nat. Biotechnol.</em><strong>28</strong>, 868–873 (2010).
</cite> [<a href="https://doi.org/10.1038/nbt.1654" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2946519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20639865/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu,%20G.,%20Paige,%20J.%20S.%20&amp;%20Jaffrey,%20S.%20R.%20Global%20analysis%20of%20lysine%20ubiquitination%20by%20ubiquitin%20remnant%20immunoaffinity%20profiling.%20Nat.%20Biotechnol.28,%20868%E2%80%93873%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Steger, M. et al. Time-resolved in vivo ubiquitinome profiling by DIA-MS reveals USP7 targets on a proteome-wide scale. <em>Nat. Commun.</em><strong>12</strong>, 1–13 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-25454-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8438043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34518535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Steger,%20M.%20et%20al.%20Time-resolved%20in%20vivo%20ubiquitinome%20profiling%20by%20DIA-MS%20reveals%20USP7%20targets%20on%20a%20proteome-wide%20scale.%20Nat.%20Commun.12,%201%E2%80%9313%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Crowe, C. et al. Mechanism of degrader-targeted protein ubiquitinability. <em>Sci. Adv</em>. <strong>10</strong>, eado6492 (2024).
</cite> [<a href="https://doi.org/10.1126/sciadv.ado6492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11468923/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39392888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crowe,%20C.%20et%20al.%20Mechanism%20of%20degrader-targeted%20protein%20ubiquitinability.%20Sci.%20Adv.%2010,%20eado6492%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Cloos, P. A. C. et al. The putative oncogene GASC1 demethylates tri- and dimethylated lysine 9 on histone H3. <em>Nature</em><strong>442</strong>, 307–311 (2006).
</cite> [<a href="https://doi.org/10.1038/nature04837" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16732293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cloos,%20P.%20A.%20C.%20et%20al.%20The%20putative%20oncogene%20GASC1%20demethylates%20tri-%20and%20dimethylated%20lysine%209%20on%20histone%20H3.%20Nature442,%20307%E2%80%93311%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Absmeier, E. et al. Specific recognition and ubiquitination of translating ribosomes by mammalian CCR4–NOT. <em>Nat. Struct. Mol. Biol.</em><strong>30</strong>, 1314–1322 (2023).
</cite> [<a href="https://doi.org/10.1038/s41594-023-01075-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7615087/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37653243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Absmeier,%20E.%20et%20al.%20Specific%20recognition%20and%20ubiquitination%20of%20translating%20ribosomes%20by%20mammalian%20CCR4%E2%80%93NOT.%20Nat.%20Struct.%20Mol.%20Biol.30,%201314%E2%80%931322%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>He, X. et al. The zinc finger transcription factor Th-POK regulates CD4 versus CD8 T-cell lineage commitment. <em>Nature</em><strong>433</strong>, 826–833 (2005).
</cite> [<a href="https://doi.org/10.1038/nature03338" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15729333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He,%20X.%20et%20al.%20The%20zinc%20finger%20transcription%20factor%20Th-POK%20regulates%20CD4%20versus%20CD8%20T-cell%20lineage%20commitment.%20Nature433,%20826%E2%80%93833%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Nickerson, M. L. et al. Mutations in a novel gene lead to kidney tumors, lung wall defects, and benign tumors of the hair follicle in patients with the Birt-Hogg-Dubé syndrome. <em>Cancer Cell</em><strong>2</strong>, 157–164 (2002).
</cite> [<a href="https://doi.org/10.1016/s1535-6108(02)00104-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12204536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nickerson,%20M.%20L.%20et%20al.%20Mutations%20in%20a%20novel%20gene%20lead%20to%20kidney%20tumors,%20lung%20wall%20defects,%20and%20benign%20tumors%20of%20the%20hair%20follicle%20in%20patients%20with%20the%20Birt-Hogg-Dub%C3%A9%20syndrome.%20Cancer%20Cell2,%20157%E2%80%93164%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Fulcher, L. J. et al. The DUF1669 domain of FAM83 family proteins anchor casein kinase 1 isoforms. <em>Sci. Signal</em>. <strong>11</strong>, eaao2341 (2018).
</cite> [<a href="https://doi.org/10.1126/scisignal.aao2341" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6025793/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29789297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fulcher,%20L.%20J.%20et%20al.%20The%20DUF1669%20domain%20of%20FAM83%20family%20proteins%20anchor%20casein%20kinase%201%20isoforms.%20Sci.%20Signal.%2011,%20eaao2341%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Devault, A. et al. MAT1 (‘menage à trois’) a new RING finger protein subunit stabilizing cyclin H-cdk7 complexes in starfish and Xenopus CAK. <em>EMBO J.</em><strong>14</strong>, 5027–5036 (1995).
</cite> [<a href="https://doi.org/10.1002/j.1460-2075.1995.tb00185.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC394606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7588631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Devault,%20A.%20et%20al.%20MAT1%20(%E2%80%98menage%20%C3%A0%20trois%E2%80%99)%20a%20new%20RING%20finger%20protein%20subunit%20stabilizing%20cyclin%20H-cdk7%20complexes%20in%20starfish%20and%20Xenopus%20CAK.%20EMBO%20J.14,%205027%E2%80%935036%20(1995)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Cheng, S.-W. G. et al. Interaction of cyclin-dependent kinase 12/CrkRS with cyclin K1 is required for the phosphorylation of the C-terminal domain of RNA polymerase II. <em>Mol. Cell. Biol.</em><strong>32</strong>, 4691–4704 (2012).
</cite> [<a href="https://doi.org/10.1128/MCB.06267-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3486194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22988298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng,%20S.-W.%20G.%20et%20al.%20Interaction%20of%20cyclin-dependent%20kinase%2012/CrkRS%20with%20cyclin%20K1%20is%20required%20for%20the%20phosphorylation%20of%20the%20C-terminal%20domain%20of%20RNA%20polymerase%20II.%20Mol.%20Cell.%20Biol.32,%204691%E2%80%934704%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Yamanaka, S. et al. A proximity biotinylation-based approach to identify protein-E3 ligase interactions induced by PROTACs and molecular glues. <em>Nat. Commun.</em><strong>13</strong>, 1–17 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-021-27818-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8748630/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35013300/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yamanaka,%20S.%20et%20al.%20A%20proximity%20biotinylation-based%20approach%20to%20identify%20protein-E3%20ligase%20interactions%20induced%20by%20PROTACs%20and%20molecular%20glues.%20Nat.%20Commun.13,%201%E2%80%9317%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Baek, K. et al. Unveiling the hidden interactome of CRBN molecular glues with chemoproteomics. <em>Nat. Commun.</em><strong>16</strong>, 6831 (2025).</cite> [<a href="https://doi.org/10.1038/s41467-025-62099-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12289921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40707481/" class="usa-link">PubMed</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Petzold, G., Fischer, E. S. &amp; Thomä, N. H. Structural basis of lenalidomide-induced CK1α degradation by the CRL4CRBN ubiquitin ligase. <em>Nature</em><strong>532</strong>, 127–130 (2016).
</cite> [<a href="https://doi.org/10.1038/nature16979" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26909574/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Petzold,%20G.,%20Fischer,%20E.%20S.%20&amp;%20Thom%C3%A4,%20N.%20H.%20Structural%20basis%20of%20lenalidomide-induced%20CK1%CE%B1%20degradation%20by%20the%20CRL4CRBN%20ubiquitin%20ligase.%20Nature532,%20127%E2%80%93130%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Krönke, J. et al. Lenalidomide induces ubiquitination and degradation of CK1α in del(5q) MDS. <em>Nature</em><strong>523</strong>, 183–188 (2015).
</cite> [<a href="https://doi.org/10.1038/nature14610" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4853910/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26131937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kr%C3%B6nke,%20J.%20et%20al.%20Lenalidomide%20induces%20ubiquitination%20and%20degradation%20of%20CK1%CE%B1%20in%20del(5q)%20MDS.%20Nature523,%20183%E2%80%93188%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Petzold, G. et al. Mining the CRBN Target Space Redefines Rules for Molecular Glue-induced Neosubstrate Recognition. <em>Science</em>. <strong>389</strong>, eadt6736 (2025).</cite> [<a href="https://doi.org/10.1126/science.adt6736" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40608931/" class="usa-link">PubMed</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Lv, L. et al. Discovery of a molecular glue promoting cdk12-ddb1 interaction to trigger cyclin k degradation. <em>eLife</em><strong>9</strong>, 1–34 (2020).</cite> [<a href="https://doi.org/10.7554/eLife.59994" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7462607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32804079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lv,%20L.%20et%20al.%20Discovery%20of%20a%20molecular%20glue%20promoting%20cdk12-ddb1%20interaction%20to%20trigger%20cyclin%20k%20degradation.%20eLife9,%201%E2%80%9334%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Słabicki, M. et al. The CDK inhibitor CR8 acts as a molecular glue degrader that depletes cyclin K. <em>Nature</em><strong>585</strong>, 293–297 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2374-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7486275/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32494016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?S%C5%82abicki,%20M.%20et%20al.%20The%20CDK%20inhibitor%20CR8%20acts%20as%20a%20molecular%20glue%20degrader%20that%20depletes%20cyclin%20K.%20Nature585,%20293%E2%80%93297%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Mayor-Ruiz, C. et al. Rational discovery of molecular glue degraders via scalable chemical profiling. <em>Nat. Chem. Biol.</em><strong>16</strong>, 1199–1207 (2020).
</cite> [<a href="https://doi.org/10.1038/s41589-020-0594-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7116640/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32747809/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mayor-Ruiz,%20C.%20et%20al.%20Rational%20discovery%20of%20molecular%20glue%20degraders%20via%20scalable%20chemical%20profiling.%20Nat.%20Chem.%20Biol.16,%201199%E2%80%931207%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Matyskiela, M. E. et al. A novel cereblon modulator recruits GSPT1 to the CRL4CRBN ubiquitin ligase. <em>Nature</em><strong>535</strong>, 252–257 (2016).
</cite> [<a href="https://doi.org/10.1038/nature18611" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27338790/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Matyskiela,%20M.%20E.%20et%20al.%20A%20novel%20cereblon%20modulator%20recruits%20GSPT1%20to%20the%20CRL4CRBN%20ubiquitin%20ligase.%20Nature535,%20252%E2%80%93257%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Alkalaeva, E. Z., Pisarev, A. V., Frolova, L. Y., Kisselev, L. L. &amp; Pestova, T. V. In vitro reconstitution of eukaryotic translation reveals cooperativity between release factors eRF1 and eRF3. <em>Cell</em><strong>125</strong>, 1125–1136 (2006).
</cite> [<a href="https://doi.org/10.1016/j.cell.2006.04.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16777602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alkalaeva,%20E.%20Z.,%20Pisarev,%20A.%20V.,%20Frolova,%20L.%20Y.,%20Kisselev,%20L.%20L.%20&amp;%20Pestova,%20T.%20V.%20In%20vitro%20reconstitution%20of%20eukaryotic%20translation%20reveals%20cooperativity%20between%20release%20factors%20eRF1%20and%20eRF3.%20Cell125,%201125%E2%80%931136%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Sperling, A. S. et al. Patterns of substrate affinity, competition, and degradation kinetics underlie biological activity of thalidomide analogs. <em>Blood</em><strong>134</strong>, 160–170 (2019).
</cite> [<a href="https://doi.org/10.1182/blood.2019000789" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6624968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31043423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sperling,%20A.%20S.%20et%20al.%20Patterns%20of%20substrate%20affinity,%20competition,%20and%20degradation%20kinetics%20underlie%20biological%20activity%20of%20thalidomide%20analogs.%20Blood134,%20160%E2%80%93170%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Huang, H.-T. et al. Ubiquitin-specific proximity labeling for the identification of E3 ligase substrates. <em>Nat. Chem. Biol.</em><strong>20</strong>, 1227–1236 (2024).
</cite> [<a href="https://doi.org/10.1038/s41589-024-01590-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38514884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20H.-T.%20et%20al.%20Ubiquitin-specific%20proximity%20labeling%20for%20the%20identification%20of%20E3%20ligase%20substrates.%20Nat.%20Chem.%20Biol.20,%201227%E2%80%931236%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Matyskiela, M. E. et al. SALL4 mediates teratogenicity as a thalidomide-dependent cereblon substrate. <em>Nat. Chem. Biol.</em><strong>14</strong>, 981–987 (2018).
</cite> [<a href="https://doi.org/10.1038/s41589-018-0129-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30190590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Matyskiela,%20M.%20E.%20et%20al.%20SALL4%20mediates%20teratogenicity%20as%20a%20thalidomide-dependent%20cereblon%20substrate.%20Nat.%20Chem.%20Biol.14,%20981%E2%80%93987%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Meyer, C., Garzia, A., Morozov, P., Molina, H. &amp; Tuschl, T. The G3BP1-family-USP10 deubiquitinase complex rescues ubiquitinated 40S subunits of ribosomes stalled in translation from lysosomal degradation. <em>Mol. Cell</em><strong>77</strong>, 1193–1205.e5 (2020).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2019.12.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31981475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meyer,%20C.,%20Garzia,%20A.,%20Morozov,%20P.,%20Molina,%20H.%20&amp;%20Tuschl,%20T.%20The%20G3BP1-family-USP10%20deubiquitinase%20complex%20rescues%20ubiquitinated%2040S%20subunits%20of%20ribosomes%20stalled%20in%20translation%20from%20lysosomal%20degradation.%20Mol.%20Cell77,%201193%E2%80%931205.e5%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Teng, M. et al. Development of PDE6D and CK1α degraders through chemical derivatization of FPFT-2216. <em>J. Med. Chem.</em><strong>65</strong>, 747–756 (2022).
</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.1c01832" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10297557/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34965125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Teng,%20M.%20et%20al.%20Development%20of%20PDE6D%20and%20CK1%CE%B1%20degraders%20through%20chemical%20derivatization%20of%20FPFT-2216.%20J.%20Med.%20Chem.65,%20747%E2%80%93756%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Branon, T. C. et al. Efficient proximity labeling in living cells and organisms with TurboID. <em>Nat. Biotechnol.</em><strong>36</strong>, 880–887 (2018).
</cite> [<a href="https://doi.org/10.1038/nbt.4201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6126969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30125270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Branon,%20T.%20C.%20et%20al.%20Efficient%20proximity%20labeling%20in%20living%20cells%20and%20organisms%20with%20TurboID.%20Nat.%20Biotechnol.36,%20880%E2%80%93887%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Annunziato, S. et al. Molecular surface mimicry enables CRBN to target G3BP2 for degradation. Preprint at <em>bioRxiv</em>10.1101/2025.04.30.651496 (2025).</cite>
</li>
<li id="CR53">
<span class="label">53.</span><cite>Whetstine, J. R. et al. Reversal of histone lysine trimethylation by the JMJD2 family of histone demethylases. <em>Cell</em><strong>125</strong>, 467–481 (2006).
</cite> [<a href="https://doi.org/10.1016/j.cell.2006.03.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16603238/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Whetstine,%20J.%20R.%20et%20al.%20Reversal%20of%20histone%20lysine%20trimethylation%20by%20the%20JMJD2%20family%20of%20histone%20demethylases.%20Cell125,%20467%E2%80%93481%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Trojer, P. et al. Dynamic histone H1 isotype 4 methylation and demethylation by histone lysine methyltransferase G9a/KMT1C and the Jumonji domain-containing JMJD2/KDM4 proteins. <em>J. Biol. Chem.</em><strong>284</strong>, 8395–8405 (2009).
</cite> [<a href="https://doi.org/10.1074/jbc.M807818200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2659197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19144645/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trojer,%20P.%20et%20al.%20Dynamic%20histone%20H1%20isotype%204%20methylation%20and%20demethylation%20by%20histone%20lysine%20methyltransferase%20G9a/KMT1C%20and%20the%20Jumonji%20domain-containing%20JMJD2/KDM4%20proteins.%20J.%20Biol.%20Chem.284,%208395%E2%80%938405%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Lee, D. H. et al. Advances in histone demethylase KDM4 as cancer therapeutic targets. <em>FASEB J.</em><strong>34</strong>, 3461–3484 (2020).
</cite> [<a href="https://doi.org/10.1096/fj.201902584R" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31961018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20D.%20H.%20et%20al.%20Advances%20in%20histone%20demethylase%20KDM4%20as%20cancer%20therapeutic%20targets.%20FASEB%20J.34,%203461%E2%80%933484%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Wu, Q. et al. Recent advances with KDM4 inhibitors and potential applications. <em>J. Med Chem.</em><strong>65</strong>, 9564–9579 (2022).
</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.2c00680" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9531573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35838529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20Q.%20et%20al.%20Recent%20advances%20with%20KDM4%20inhibitors%20and%20potential%20applications.%20J.%20Med%20Chem.65,%209564%E2%80%939579%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Walters, Z. S. et al. Role for the histone demethylase kdm4b in rhabdomyosarcoma via cdk6 and ccna2: Compensation by kdm4a and apoptotic response of targeting both kdm4b and kdm4a. <em>Cancers</em><strong>13</strong>, 1734 (2021).
</cite> [<a href="https://doi.org/10.3390/cancers13071734" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8038694/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33917420/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Walters,%20Z.%20S.%20et%20al.%20Role%20for%20the%20histone%20demethylase%20kdm4b%20in%20rhabdomyosarcoma%20via%20cdk6%20and%20ccna2:%20Compensation%20by%20kdm4a%20and%20apoptotic%20response%20of%20targeting%20both%20kdm4b%20and%20kdm4a.%20Cancers13,%201734%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Ziegler, W. H., Liddington, R. C. &amp; Critchley, D. R. The structure and regulation of vinculin. <em>Trends Cell Biol.</em><strong>16</strong>, 453–460 (2006).
</cite> [<a href="https://doi.org/10.1016/j.tcb.2006.07.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16893648/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ziegler,%20W.%20H.,%20Liddington,%20R.%20C.%20&amp;%20Critchley,%20D.%20R.%20The%20structure%20and%20regulation%20of%20vinculin.%20Trends%20Cell%20Biol.16,%20453%E2%80%93460%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Wang, E. S. et al. Acute pharmacological degradation of Helios destabilizes regulatory T cells. <em>Nat. Chem. Biol.</em><strong>17</strong>, 711–717 (2021).
</cite> [<a href="https://doi.org/10.1038/s41589-021-00802-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8162940/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34035522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20E.%20S.%20et%20al.%20Acute%20pharmacological%20degradation%20of%20Helios%20destabilizes%20regulatory%20T%20cells.%20Nat.%20Chem.%20Biol.17,%20711%E2%80%93717%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Gupta, N. et al. Stress granule-associated protein G3BP2 regulates breast tumor initiation. <em>Proc. Natl. Acad. Sci. USA</em><strong>114</strong>, 1033–1038 (2017).
</cite> [<a href="https://doi.org/10.1073/pnas.1525387114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5293063/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28096337/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gupta,%20N.%20et%20al.%20Stress%20granule-associated%20protein%20G3BP2%20regulates%20breast%20tumor%20initiation.%20Proc.%20Natl.%20Acad.%20Sci.%20USA114,%201033%E2%80%931038%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Takayama, K. I., Suzuki, T., Fujimura, T., Takahashi, S. &amp; Inoue, S. Association of USP10 with G3BP2 inhibits p53 signaling and contributes to poor outcome in prostate cancer. <em>Mol. Cancer Res.</em><strong>16</strong>, 846–856 (2018).
</cite> [<a href="https://doi.org/10.1158/1541-7786.MCR-17-0471" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29378906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Takayama,%20K.%20I.,%20Suzuki,%20T.,%20Fujimura,%20T.,%20Takahashi,%20S.%20&amp;%20Inoue,%20S.%20Association%20of%20USP10%20with%20G3BP2%20inhibits%20p53%20signaling%20and%20contributes%20to%20poor%20outcome%20in%20prostate%20cancer.%20Mol.%20Cancer%20Res.16,%20846%E2%80%93856%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Rappsilber, J., Ishihama, Y. &amp; Mann, M. Stop and go extraction tips for matrix-assisted laser desorption/ionization, nanoelectrospray, and LC/MS sample pretreatment in proteomics. <em>Anal. Chem</em>. 10.1021/ac026117i (2003).</cite> [<a href="https://doi.org/10.1021/ac026117i" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12585499/" class="usa-link">PubMed</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Szyrwiel, L., Sinn, L., Ralser, M. &amp; Demichev, V. Slice-PASEF: fragmenting all ions for maximum sensitivity in proteomics. Preprint at <em>bioRxiv</em>10.1101/2022.10.31.514544 (2022).</cite>
</li>
<li id="CR64">
<span class="label">64.</span><cite>Skowronek, P., Wallmann, G., Wahle, M., Willems, S. &amp; Mann, M. An accessible workflow for high-sensitivity proteomics using parallel accumulation–serial fragmentation (PASEF). <em>Nat. Protoc</em>. 10.1038/s41596-024-01104-w (2025).</cite> [<a href="https://doi.org/10.1038/s41596-024-01104-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39825144/" class="usa-link">PubMed</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Singh, S. et al. Targeting KDM4 for treating PAX3-FOXO1–driven alveolar rhabdomyosarcoma. <em>Sci. Transl. Med</em>. <strong>14</strong>, eabq2096 (2022).
</cite> [<a href="https://doi.org/10.1126/scitranslmed.abq2096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9548378/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35857643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singh,%20S.%20et%20al.%20Targeting%20KDM4%20for%20treating%20PAX3-FOXO1%E2%80%93driven%20alveolar%20rhabdomyosarcoma.%20Sci.%20Transl.%20Med.%2014,%20eabq2096%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Lier, S. et al. A novel Cereblon E3 ligase modulator with antitumor activity in gastrointestinal cancer. <em>Bioorg. Chem.</em><strong>119</strong>, 105505 (2022).
</cite> [<a href="https://doi.org/10.1016/j.bioorg.2021.105505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34838332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lier,%20S.%20et%20al.%20A%20novel%20Cereblon%20E3%20ligase%20modulator%20with%20antitumor%20activity%20in%20gastrointestinal%20cancer.%20Bioorg.%20Chem.119,%20105505%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Brown, L. Y., Dong, W. &amp; Kantor, B. An improved protocol for the production of lentiviral vectors. <em>STAR Protoc</em>. <strong>1</strong>, 100152 (2020).
</cite> [<a href="https://doi.org/10.1016/j.xpro.2020.100152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7757317/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33377046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brown,%20L.%20Y.,%20Dong,%20W.%20&amp;%20Kantor,%20B.%20An%20improved%20protocol%20for%20the%20production%20of%20lentiviral%20vectors.%20STAR%20Protoc.%201,%20100152%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Cox, J. et al. Accurate proteome-wide label-free quantification by delayed normalization and maximal peptide ratio extraction, termed MaxLFQ. <em>Mol. Cell Proteom.</em><strong>13</strong>, 2513 (2014).</cite> [<a href="https://doi.org/10.1074/mcp.M113.031591" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4159666/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24942700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cox,%20J.%20et%20al.%20Accurate%20proteome-wide%20label-free%20quantification%20by%20delayed%20normalization%20and%20maximal%20peptide%20ratio%20extraction,%20termed%20MaxLFQ.%20Mol.%20Cell%20Proteom.13,%202513%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Ritchie, M. E. et al. Limma powers differential expression analyses for RNA-sequencing and microarray studies. <em>Nucleic Acids Res</em>. 10.1093/nar/gkv007 (2015).</cite> [<a href="https://doi.org/10.1093/nar/gkv007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4402510/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25605792/" class="usa-link">PubMed</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Varadi, M. et al. AlphaFold Protein Structure Database in 2024: providing structure coverage for over 214 million protein sequences. <em>Nucleic Acids Res.</em><strong>52</strong>, D368–D375 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkad1011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10767828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37933859/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Varadi,%20M.%20et%20al.%20AlphaFold%20Protein%20Structure%20Database%20in%202024:%20providing%20structure%20coverage%20for%20over%20214%20million%20protein%20sequences.%20Nucleic%20Acids%20Res.52,%20D368%E2%80%93D375%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. <em>Nature</em><strong>596</strong>, 583–589 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jumper,%20J.%20et%20al.%20Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold.%20Nature596,%20583%E2%80%93589%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (7.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62829_MOESM2_ESM.pdf</a><sup> (1.6MB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (1.1MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (28.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (32KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (26.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (55KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (822.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM9_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (337.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368047/bin/41467_2025_62829_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (5.6MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The mass spectrometry proteomics raw data generated in this study have been deposited in the MassIVE repository (<a href="https://massive.ucsd.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://massive.ucsd.edu/</a>) and can be accessed via proteome exchange with accession code <a href="https://proteomecentral.proteomexchange.org/cgi/GetDataset?ID=PXD059111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PXD059111</a>. DIA-NN is freely available for download at <a href="https://www.github.com/vdemichev/diann" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.github.com/vdemichev/diann</a>. <a href="#Sec29" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62829-0"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62829.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368047/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368047/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368047%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368047/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368047/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368047/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40835825/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368047/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40835825/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368047/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368047/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="cPTrgEKdLb60OIVSHxIrBNQsHeJ6m9iIdbxxOM8hHRpLQYG7Gz2Vqkhvx6o9muxX">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
