
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Uncovering the multi-layer cis-regulatory landscape of rice via integrative nascent RNA analysis - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152553DE8AF44EF30553DE001C53C3D1.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="genbio">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359857/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Genome Biology">
<meta name="citation_title" content="Uncovering the multi-layer cis-regulatory landscape of rice via integrative nascent RNA analysis">
<meta name="citation_author" content="Marina Goliasse">
<meta name="citation_author_institution" content="Département de Chimie, Université du Québec À Montréal, Montréal, QC Canada">
<meta name="citation_author" content="Aurore Johary">
<meta name="citation_author_institution" content="Département de Chimie, Université du Québec À Montréal, Montréal, QC Canada">
<meta name="citation_author" content="Adrian E Platts">
<meta name="citation_author_institution" content="Department of Horticulture, Michigan State University, East Lansing, MI USA">
<meta name="citation_author" content="Fabian Ortner-Krause">
<meta name="citation_author_institution" content="Département de Chimie, Université du Québec À Montréal, Montréal, QC Canada">
<meta name="citation_author" content="Patrick P Edger">
<meta name="citation_author_institution" content="Department of Horticulture, Michigan State University, East Lansing, MI USA">
<meta name="citation_author" content="Jae Young Choi">
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, KS USA">
<meta name="citation_author" content="Michael D Purugganan">
<meta name="citation_author_institution" content="Center for Genomics and Systems Biology, Department of Biology, New York University, New York, NY USA">
<meta name="citation_author_institution" content="Center for Genomics and Systems Biology, NYU Abu Dhabi Research Institute, New York University Abu Dhabi, Saadiyat Island, Abu Dhabi, United Arab Emirates">
<meta name="citation_author" content="Zoé Joly-Lopez">
<meta name="citation_author_institution" content="Département de Chimie, Université du Québec À Montréal, Montréal, QC Canada">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="250">
<meta name="citation_doi" content="10.1186/s13059-025-03715-2">
<meta name="citation_pmid" content="40826469">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359857/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359857/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359857/pdf/13059_2025_Article_3715.pdf">
<meta name="description" content="Efforts to characterize regulatory elements in plant genomes traditionally rely on evolutionary conservation and chromatin accessibility. Recently, intergenic bi-directional nascent transcript has emerged as a putative hallmark of active enhancers. ...">
<meta name="og:title" content="Uncovering the multi-layer cis-regulatory landscape of rice via integrative nascent RNA analysis">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Efforts to characterize regulatory elements in plant genomes traditionally rely on evolutionary conservation and chromatin accessibility. Recently, intergenic bi-directional nascent transcript has emerged as a putative hallmark of active enhancers. ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359857/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12359857">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s13059-025-03715-2"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/13059_2025_Article_3715.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12359857%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12359857/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12359857/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359857/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-genbio.png" alt="Genome Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Genome Biology" title="Link to Genome Biology" shape="default" href="https://genomebiology.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Genome Biol</button></div>. 2025 Aug 18;26:250. doi: <a href="https://doi.org/10.1186/s13059-025-03715-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s13059-025-03715-2</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Genome%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Genome%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Genome%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Genome%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Uncovering the multi-layer <em>cis</em>-regulatory landscape of rice via integrative nascent RNA analysis</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Goliasse%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Marina Goliasse</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Marina Goliasse</span></h3>
<div class="p">
<sup>1</sup>Département de Chimie, Université du Québec À Montréal, Montréal, QC Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Goliasse%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marina Goliasse</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Johary%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Aurore Johary</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Aurore Johary</span></h3>
<div class="p">
<sup>1</sup>Département de Chimie, Université du Québec À Montréal, Montréal, QC Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Johary%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Aurore Johary</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Platts%20AE%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Adrian E Platts</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Adrian E Platts</span></h3>
<div class="p">
<sup>2</sup>Department of Horticulture, Michigan State University, East Lansing, MI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Platts%20AE%22%5BAuthor%5D" class="usa-link"><span class="name western">Adrian E Platts</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ortner-Krause%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Fabian Ortner-Krause</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Fabian Ortner-Krause</span></h3>
<div class="p">
<sup>1</sup>Département de Chimie, Université du Québec À Montréal, Montréal, QC Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ortner-Krause%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fabian Ortner-Krause</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Edger%20PP%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Patrick P Edger</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Patrick P Edger</span></h3>
<div class="p">
<sup>2</sup>Department of Horticulture, Michigan State University, East Lansing, MI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Edger%20PP%22%5BAuthor%5D" class="usa-link"><span class="name western">Patrick P Edger</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Choi%20JY%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jae Young Choi</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jae Young Choi</span></h3>
<div class="p">
<sup>3</sup>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, KS USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Choi%20JY%22%5BAuthor%5D" class="usa-link"><span class="name western">Jae Young Choi</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Purugganan%20MD%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Michael D Purugganan</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Michael D Purugganan</span></h3>
<div class="p">
<sup>4</sup>Center for Genomics and Systems Biology, Department of Biology, New York University, New York, NY USA </div>
<div class="p">
<sup>5</sup>Center for Genomics and Systems Biology, NYU Abu Dhabi Research Institute, New York University Abu Dhabi, Saadiyat Island, Abu Dhabi, United Arab Emirates </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Purugganan%20MD%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael D Purugganan</span></a>
</div>
</div>
<sup>4,</sup><sup>5,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Joly-Lopez%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Zoé Joly-Lopez</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Zoé Joly-Lopez</span></h3>
<div class="p">
<sup>1</sup>Département de Chimie, Université du Québec À Montréal, Montréal, QC Canada </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Joly-Lopez%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zoé Joly-Lopez</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Département de Chimie, Université du Québec À Montréal, Montréal, QC Canada </div>
<div id="Aff2">
<sup>2</sup>Department of Horticulture, Michigan State University, East Lansing, MI USA </div>
<div id="Aff3">
<sup>3</sup>Department of Ecology and Evolutionary Biology, University of Kansas, Lawrence, KS USA </div>
<div id="Aff4">
<sup>4</sup>Center for Genomics and Systems Biology, Department of Biology, New York University, New York, NY USA </div>
<div id="Aff5">
<sup>5</sup>Center for Genomics and Systems Biology, NYU Abu Dhabi Research Institute, New York University Abu Dhabi, Saadiyat Island, Abu Dhabi, United Arab Emirates </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2022 Jul 6; Accepted 2025 Jul 27; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12359857  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40826469/" class="usa-link">40826469</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par11">Efforts to characterize regulatory elements in plant genomes traditionally rely on evolutionary conservation and chromatin accessibility. Recently, intergenic bi-directional nascent transcript has emerged as a putative hallmark of active enhancers. Here, we integrate these approaches to better define the <em>cis</em>-regulatory landscape of the rice genome.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par12">In juvenile leaf tissues of the Azucena rice variety, we analyze conserved noncoding sequences, intergenic bi-directional transcripts, and regions of open chromatin. These three features highlight distinct classes of regulatory targets, each exhibiting complexity and regulatory roles. Conserved noncoding sequences are associated with more complex regulatory interactions, while regions marked by chromatin accessibility or bi-directional nascent transcription tend to promote more stable regulatory activity. Some transcribed regulatory sites harbor elements linked to transposable element silencing, whereas others correlate with increased expression of nearby genes, pointing to candidate transcribed regulatory elements. We further identified molecular interactions between genic regions and intergenic transcribed regulatory elements using 3-dimensional chromatin contact data, we identify physical interactions between transcribed intergenic regions and genic regions. These interactions often co-localize with expression quantitative trait loci and coincide with increased transcription, further supporting a regulatory role.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par13">Our integrative analysis reveals multiple distinct classes of regulatory elements in the rice genome, with overlapping but non-identical targets and signatures. Many candidate elements share features consistent with transcriptional enhancement, though the specific criteria for defining active enhancers in plants require further characterization. These findings underscore the importance of using complementary genomic signals to discover and characterize functionally diverse regulatory elements in plant genomes.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s13059-025-03715-2.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Gene regulation, <em>cis</em>-regulatory elements, Functional genomics, Transposable elements, Transcribed enhancers, Conserved noncoding sequences, PRO-seq, ATAC-seq, <em>Oryza sativa</em>, Methylation, Pore-C, Chromatin architecture</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par26"><em>Cis</em>-regulatory elements (CREs) are noncoding DNA sequences that contribute to the regulation of gene expression, dictating the precise timing, location, and level of gene transcription [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. These elements can be positioned near genes, e.g., promoter elements that are typically a few hundred base pairs upstream of transcription start sites (TSSs), to tens of thousands of base pairs away from the genes they regulate, e.g., enhancers and silencers. Although CREs are estimated to be more numerous than protein-coding genes, their characterization remains challenging due to our incomplete knowledge of the genomic characteristics that define these elements.</p>
<p id="Par27">A common approach to identifying CREs relies on analyzing the presence or absence of various genomic features, often in combination. In plants, these include the presence of known transcription factor (TF) binding motifs, chromatin accessibility, specific histone modifications, levels of DNA methylation, and evidence for transient or stable looping into target genes [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. Integrating these signatures has increased our ability to differentiate between types of CREs, e.g., distal enhancers, silencers and insulators, and proximal regions such as promoter elements, TSSs, and cryptic upstream open reading frames (uORFs) [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Combining these signatures with massively parallel reporter assays has enabled widespread functional characterization of many CREs [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>, <a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]; for example, a significant proportion of active CREs exhibit lower DNA methylation [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>], while the presence of specific histone modifications can signal the state of active CREs (e.g., H3K27ac, H3K18ac) [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>], and CREs with paused polymerases are often enriched for H3K27me3 and low levels of active histone marks like acetylation [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Repressed CREs meanwhile often exhibit both low chromatin accessibility and H3K27me3 levels [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>, <a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>].</p>
<p id="Par28">Several approaches to identifying CREs de novo and subsequently validating these have been developed. In Oryza, for example, weakly conserved enhancer-like CREs often show a correlation with nearby gene expression. In Arabidopsis, enhancer-like examples were also seen in a loss or gain in conserved noncoding sequence (CNS), which was often associated with lower expression and higher expression, respectively [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>], which would be enhancer-like, while a gain in CNS skewed the expression towards higher expression. Similarly in maize and cassava, these sequences show low interspecies conservation but weak negative selection within <em>Oryza sativa</em> populations. This weak interspecies selection in rice was determined using the greenINSIGHT approach, which combines the determination of an ancestral base probability from comparative alignments, with polymorphism data to infer selection [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. The presence of enhancer RNAs (eRNAs) has not yet been widely used to identify plant CREs but shows promise. Enhancer RNAs are short, bidirectionally transcribed, largely unspliced and unpolyadenylated RNAs typically less than 150 nucleotides long (although some may be up to 2 kbase (kb)) [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>], which are transcribed from CREs. In Animalia, eRNAs are predominantly localized in the nucleus and are chromatin-bound [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>] serving to mark active enhancers and may act as a proxy for their spatio-temporal activity. Enhancer transcription is likely important for gene expression, potentially maintaining chromatin accessibility for TF binding, stimulating chromatin remodelers, regulating TF occupancy, and promoting RNA polymerase (Pol) II activity [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>–<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. However, the instability of eRNAs makes them difficult to detect with standard RNA-sequencing, necessitating nascent RNA sequencing technologies [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>–<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>] such as precision nuclear run-on sequencing (PRO-seq) [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>, <a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Many of these elements in <em>O. sativa</em> appear to have emerged recently, as evidenced by recent changes in selection, aligning with their frequently transient and species-specific characteristics [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>, <a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. Understanding this class of regulatory elements in plants remains a challenge, but methods that associate specific genetic variants with quantitative trait loci (QTLs) for agronomic traits can help differentiate functional from dormant noncoding elements [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>].</p>
<p id="Par29">In plants, our understanding of eRNAs is still limited, with reports of their transcription in maize (<em>Zea mays</em>) [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>], cassava (<em>Manihot esculenta</em>) [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>], and rice (<em>Oryza sativa</em>) [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>, <a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>], but they appear less common in other genomes such as Arabidopsis [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. In maize and cassava, intergenic regulatory elements identified through PRO-seq data are enriched for expression quantitative trait loci (eQTLs) identified in kernels and exhibit low levels of conservation, again suggesting many of the elements undergo a rapid turnover [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. Despite the analysis of the various genomic and epigenomic features associated with plant CREs, the presence of eRNAs has, up till now, not generally been considered and was even recently suggested to be rare in plants [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>].</p>
<p id="Par30">Other notable approaches to identify plant CREs include open chromatin assays that have been used in many studies to detect accessible chromatin regions (ACRs) formed by the active displacement of nucleosomes by transcription factor binding [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>–<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. Another approach is phylogenetic footprinting, which identifies CSNs by detecting deviations from neutral levels of constraint. CNSs are generally thought to fine-tune complex transcriptional patterns, particularly in heavily regulated genes particularly [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>].</p>
<p id="Par31">In this study, we explored DNA sequence composition at these different sites, their chromosomal environment, and evolutionary constraint, building on previous work in rice [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] and detailed regulatory element characterization in maize [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. We describe the regulatory landscape of Asian rice (<em>O. sativa japonia ssp</em> Azucena) using comparative data, nascent transcription, chromatin accessibility, and complementary functional genomic sequence features (DNA methylation profiles, histone modification profiles, transcriptomic profiles), and 3D genome architecture. We find that regulatory elements identified using conservation, accessible chromatin, and nascent bi-directional transcription appear to contain sequences with likely different regulatory functions and gene target genes. We also find that while many intergenic eRNAs may be related to transposable element silencing, others share features of enhancers, suggesting they harbor transcribed regulatory elements associated with gene regulation and <em>cis</em>-expression e-QTLs. Overall, this work advances our understanding of regulatory noncoding elements and is motivated by the need to develop improved, informed systematic pipelines for characterizing <em>cis</em>-regulatory elements in different spatio-temporal contexts [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>].</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Profiles of nascent transcription, CNS, and accessible chromatin regions in the rice genome</h3>
<p id="Par32">We considered comparative genomics alongside several functional genomics approaches to determine if integrated approaches could enhance the specificity of searches for different types of <em>cis</em>-regulatory elements. Chromatin accessibility from Assay for Transposes-Accessible Chromatin sequencing (ATAC-seq), nascent transcription from Precision Nuclear Run-On sequencing (PRO-seq), and sequence conservation in genomic regions at conserved noncoding sites (CNSs) were considered jointly using <em>O. sativa</em> japonica cultivar Azucena as a reference genome.</p>
<p id="Par33">A set of 96,570 CNS were identified from a 14-way alignment of Oryzeae species (Additional file 1: Figs. S1-S2, Additional file 2). These 96,570 <em>O. sativa</em> CNSs had a total length of 3.98 megabases (Mb) (~ 1.0% of the assembled genome) and a mean (median) length of 40 nucleotides (nts) (36 nts). The total length is similar to, and the median length almost identical to, that observed in the smaller crucifer genomes <em>Arabidopsis lyrata</em> (<em>N</em> = 92,421 CNSs) and <em>A. thaliana</em> (<em>N</em> = 90,060 CNSs (~ 4.5 Mb or 3.8% of the genome)), where both sets of CNSs had a median length of 37 nts [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>]. This also closely matches the size distribution of a set of CNSs identified across several Solanaceae species using a similar comparative approach (unpublished data) and may suggest that given similar levels of phylogenetic support, CNS length as a proxy for TFBS-cluster length is relatively conserved across plants. Of the 96,570 CNSs identified, 30,587 were intronic CNSs (32%, 0.44/kb), 9635 3'UTR CNSs (10%, 1.4/kb), 3799 5'UTR CNSs (4%, 1.1/kb), 31,192 distal intergenic CNSs (32%, 0.15/kb), 11,634 upstream gene CNSs (12%, 0.35/kb), and 9855 downstream gene CNSs (10%, 0.32/kb) (where upstream and downstream is determined as within 1 kb of a transcript start/end) (Additional file 1: Fig. S3).</p>
<p id="Par34">To examine genome-wide nascent transcription, particularly intergenic transcription including eRNAs, PRO-seq data was generated from Azucena leaf tissues grown under optimal conditions. We combined the newly generated PRO-seq data with previously published PRO-seq data that was also generated from Azucena grown under similar growth conditions [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>], but these two replicates originated from two independent experiments. Therefore, these PRO-seq sequence features were processed individually (Additional file 3: Table S1). Correlation analysis between replicates at genic and promoter regions showed high correlation for genic (<em>r</em> = 0.91, <em>p</em> &lt; 2.2 × 10⁻<sup>1</sup>⁶), while promoter correlations were slightly lower (<em>r</em> = 0.80, <em>p</em> &lt; 2.2 × 10⁻<sup>1</sup>⁶), in line with typical expectations (Additional file 1: Fig. S4). As expected, the number of PRO-seq reads aligned with annotated exonic regions compared to introns (Additional file 1: Fig. S5). Because of the bidirectional nature of the most well studied eRNAs (2-D eRNAs compared to 1-D eRNAs) [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>, <a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>] based on previous nascent transcription studies in animals [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>–<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>], we used the dREG (discriminative Regulatory Element detection) tool [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>, <a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>] to identify bidirectional dual-stranded transcription activity relative to a training set of human regulatory elements as a first criterion for identifying candidate transcribed intergenic regions in the genome [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. While human and plant regulatory signatures can vary [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>], the ability of this tool to discriminate regions with enhancer characteristics in a subset of candidates in rice, maize, and cassava has previously been demonstrated [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>, <a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>].</p>
<p id="Par35">Sites with bidirectional transcription, which we refer to as discriminative regulatory-element detection sites (dREGs) were identified in each replicate and the subset with an intersection +/− 100 bp in both replicates were considered for downstream analysis. As expected, dREG locations showed an enrichment of PRO-seq reads in both DNA strands, with dREG scores increasing with the PRO-seq signal (counts per million) (Fig. <a href="#Fig1" class="usa-link">1</a>a). The bi-directional distribution of PRO-seq signal within and around dREG peaks presented as the typical shifted juxtaposition of the plus and minus strand signal in genic regions compared to intergenic regions (Additional file 1: Fig. S5). In total we detected 32,424 dREG locations across the genome, with dREG scores ranging from 0.33 and 1.46 (Additional file 1: Fig. S6). The genome-wide median size of a dREG site was 420 base pairs (bp). This falls within the expected size range for bidirectional eRNAs in animals (50–2000 bp) (Additional file 1: Fig. S6) and is similar to the average length of 460 nucleotides observed in humans [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. Approximately 60% of dREG sites were in intergenic regions (based on the Azucena annotation; see “<a href="#Sec1" class="usa-link">Methods</a>”). This proportion is similar to that previously reported for the cassava genome [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. We divided the intergenic dREG sites into two classes depending on their distance to genic sequences: (<em>i</em>) proximal dREG locations (dREG<sub>proximal</sub>), which are intergenic dREG locations that are &lt; 1 kb to predicted transcription start or end sites, and (<em>ii</em>) distal dREG locations (dREG<sub>distal</sub>), which are &gt; 1 kb away from predicted transcription start or end sites. Within the intergenic regions, we found approximately equal proportions of dREG<sub>proximal</sub> and dREG<sub>distal</sub> locations (Fig. <a href="#Fig1" class="usa-link">1</a>b). In contrast to genic regions, dREG locations found in intergenic regions, whether proximal or distal, had a similar distribution of dREG scores. Here, the highest ratio of score by number of dREG locations between 0.4 and 0.6, particularly in distal dREG (Additional file 1: Fig. S6).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2a59/12359857/efaee294dd03/13059_2025_3715_Fig1_HTML.jpg" loading="lazy" id="MO1" height="789" width="778" alt="Fig. 1"></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of Azucena genomic classes identified by conservation, chromatin accessibility and bi-directional transcription.<strong> a</strong> Plus (top) and minus (bottom) PRO-seq signal normalized as read counts per million of dREG peaks. Each line corresponds to the signal of dREG peaks grouped by dREG bin scores. <strong>b</strong> Classification of dREG peaks based on annotated genomic regions and features. Genic: within coding sequence regions; proximal: within 1 kb of genic sequences; and distal: more than 1 kb away from genic sequences. Some of the 32,424 dREG peaks identified were found to overlap with more than one annotation. <strong>c</strong> Enrichment relative to IID expectation between either dREG locations, ATAC peaks, or CNSs and one or a combination of the two other elements for all genomic locations (left) and intergenic locations (right). For each plot, the enrichment is based on dREG locations (top), CNSs (middle) and ATAC peaks (bottom)</p></figcaption></figure><p id="Par36">Many <em>cis</em>-regulatory elements (CREs) that are presumed to be actively engaged have been shown to coincide with accessible chromatin regions [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. We inferred open chromatin from high-resolution ATAC-seq data from a similarly matched stage of leaf development as used for PRO-seq (see "<a href="#Sec1" class="usa-link">Methods</a>"). Following MACS2 processing of the ATAC-seq data, as well as a signal cutoff and a conservative overlap of minimum 50% in all three replicates (see "<a href="#Sec1" class="usa-link">Methods</a>"), we obtained a set of 27,772 annotated open chromatin peaks (22.22 Mb, median length 621 nts). Increased ATAC-seq signal was observed for genes with increased expression (based on RNA-seq data) (Additional file 1: Fig. S7). 15,566 of these peaks were found in intergenic regions (Additional file 1: Fig. S7 Additional file 3: Table S2).</p>
<p id="Par37">While these three relatively orthogonal predictors of regulatory sequences could, in principle, fully overlap, in practice, their coverage and location vary significantly. This variation arises from a variety of factors, including the spatio-temporal specificity of regulatory sequences. We first looked at the overlap of combinations of the classes of sequence features and found that many were distinct and did not overlap in the rice genome (Additional file 1: Fig. S8). The difference in overlap may in part be attributed to the ATAC peaks being longer (median of 621 bp vs. 420 bp for dREG and 36 bp for CNS) causing a greater proportion of ATAC peaks to overlap with the two sequence features but not the other way around. To account for biases in the size or distribution of features, we performed a co-occurrence enrichment analysis relative to IID expectation for individual feature and in combination. dREG locations were enriched for ATAC peaks (1.75x) and vice versa for dREG locations enriched in ATAC peaks (Fig. <a href="#Fig1" class="usa-link">1</a>c). CNS locations were either very slightly enriched for dREG (1.05x) or depleted for ATAC (0.88x). CNS enrichment with ATAC and dREG varied little between all regions and specifically intergenic regions, whereas intergenic dREG ATAC locations were less enriched for each other (1.12x) (Fig. <a href="#Fig1" class="usa-link">1</a>c). Altogether, this suggests that these features identified based on distinct metrics, whether it was conservation, chromatin accessibility or nascent transcription, would appear to underlie a level of different categories of regulatory sites in the rice genome. However, this raises the question as to whether these categories share similarities in function if not location or whether they are functionally distinct.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Genomic distribution and sequence motifs reveal different patterns for conserved, transcribed, and open chromatin elements around genes</h3>
<p id="Par38">To assess potential functional differences among regulatory sequences identified through distinct methods, we compared the genomic locations of regions identified by three different sequence features—conserved noncoding sequences (CNSs), PRO-seq, and ATAC-seq—in terms of their distance from genes and the functions of those genes. We began by examining the distribution of these three sequence classes relative to all annotated genes, as well as in intergenic regions relative to an IID model (Fig. <a href="#Fig2" class="usa-link">2</a>a). Within intergenic regions, we detected a strong polarity in enrichment for all three types of elements towards the 5’ start of annotated genes (Fig. <a href="#Fig2" class="usa-link">2</a>a; Additional file 1: Fig. S9). While they display similar distributions relative to TSS (Additional file 1: Fig. S9), the degree to which CNS, ATAC, and dREG sites collocated within the promoter was close to zero, suggesting again that their regulatory roles may be somewhat differentiated. A gene ontology (GO) analysis revealed that genes enriched for large amounts of conserved sequences were enriched for roles in transcription factor activity (FDR-adjusted <em>p</em> = 1.1 × 10<sup>−15</sup>) and cellular differentiation/development processes (e.g., cell wall biogenesis, FDR-adjusted <em>p</em> = 1. × 10<sup>−3</sup>) as previously reported in Arabidopsis (Additional file 1: Fig. S10, Additional file 4) [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>]. In contrast, genes enriched for extended open chromatin regions (&gt; 1274 bp ATAC) had weaker ontology associations and were involved more in translation pathways that are ubiquitously active, such as forming the structural constituents of ribosomes (Benjamini-adjusted <em>p</em> = 3.6 × 10<sup>−5</sup>) (Additional file 1: Fig. S11, Additional file 5). The pathways tend to indicate that these regions involve regulatory stability (mostly open) rather than regulatory dynamism. Of the three sequence features, genes in proximity to a detected dREG location had the weakest ontology associations, with no significant biological process (Additional file 1: Fig. S12, Additional file 6). It is possible that this indicates that dREG sites are more ubiquitously distributed across many types of genes and not biased towards any specific functional categories.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359857_13059_2025_3715_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2a59/12359857/597d30d0c819/13059_2025_3715_Fig2_HTML.jpg" loading="lazy" id="MO2" height="821" width="708" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genomic distribution and sequence motifs for conserved, transcribed and open chromatin elements around genes. <strong>a</strong> Enrichment relative to an IID expectation (horizontal line) of dREG sites, ATAC peaks, and CNSs in regions in and around PCGs (top) and genes with high CNS density (bottom) in the rice genome. The dashed line indicates the IID expectation. <strong>b</strong> Top 5 motifs (length:5-8nt) enriched in proximal-promoter CNSs, ATAC peaks, and dREG sites. <strong>c</strong> Motif frequency rate analysis for CNSs in four different locations. <strong>d</strong> phyloP distribution (conacc mode) for CNS, dREG, and ATAC sites in different genomic contexts</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Sequence characteristics of conserved elements, dREG, and ATAC domains</h3>
<p id="Par39">As distinct genomic locations for CNSs, accessible chromatin and transcribed intergenic regions suggest a potential bias in underlying sequences, we investigated whether there are specific sequence characteristics that distinguish them. Homer [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>] was used to characterize the enriched sequence content of CNSs, ATAC, and dREG sequences in a range of genic and intergenic regions (Fig. <a href="#Fig2" class="usa-link">2</a>; Additional file 1: Figs. S13-14).</p>
<p id="Par40">Motif complexity was notably higher in CNSs (average 3.3 different bases per collapsed-Position weight matrix (PWM)) than ATAC and dREG sequences (average 2.6 different bases per collapsed-PWM) in upstream promoter regions. ATAC regions were frequently enriched in simple tandem repeated sequences (e.g., GAGAGAGA). In contrast, enriched motifs in CNS sequences were observed with a lower associated <em>p</em>-value (Additional file 1: Fig. S13) likely because they consist of a heterogeneous mix of longer more complex sequences, e.g., the G-box motif. ATAC motifs would be more characteristic of housekeeping sequences in which tandem repetitiveness provides redundancy against mutations that could otherwise impact fitness [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>]. For dREG motifs, a feature that stood out was the number and enrichment of TATA-like sequences (Fig. <a href="#Fig2" class="usa-link">2</a>b; Additional file 1: Fig. S14), suggesting that these sites may provide alternative promoter TSS locations.</p>
<p id="Par41">The upstream proximal promoter and first intron have long been recognized as regulatory hot-spots, raising questions around the sequence distinctiveness of CREs located within them relative to other regions. We investigated the complexity of CNSs motif enrichment in distal promoter and intergenic regions to explore potential differences compared with proximal promoter motifs. We noted that intergenic and distal promoter CNSs were highly similar (Pearson correlation in enriched motif frequency <em>r</em> = 0.995) while intergenic and proximal promoter CNSs differed the most (correlation <em>r</em> = 0.981). Therefore, while enriched motifs in CNSs are broadly similar (Fig. <a href="#Fig2" class="usa-link">2</a>c), there is a notable divergence in the sequences of CNSs located in promoter proximal regions relative to more distal CNSs. For example, the motif GCGACG is found at twice the rate in proximal CNS relative to distal and intergenic CNSs. These may suggest different functional roles and regulatory mechanisms of CNSs in these distinct genomic regions.</p>
<p id="Par42">The detection of evolutionary constraint across species [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>–<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>] can contribute to an improved understanding of the evolutionary trajectory of different regulatory elements. For example, while an inter-species conservation approach was used to identify the CNSs, other elements, such as enhancers tend to show more sequence diversification and be more species specific [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>, <a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>], and therefore less amenable to be detected by conservation-based methods. To address the question as to whether the conservation levels of CNS compare between genomic regions (proximal and distal regions) of genes and among different elements, we estimated the level of evolutionary constraint occurring within these peak regions using phyloP [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>]. Selection within dREG and ATAC sequence features were compared to selection in CNS, coding sequences, and neutral intergenic regions of the genome (control) (Fig. <a href="#Fig2" class="usa-link">2</a>d). The results showed that compared to neutral regions (modal value 0; mean − 0.049), we found CNSs displaying the highest evolutionary constrain, followed by CDSs. This is to be expected as CNSs are defined by their constraint (see "<a href="#Sec1" class="usa-link">Methods</a>"), while annotated CDSs are defined by their coding potential. Among annotated CDSs, many are not necessarily in active site and therefore are less constrained while others can be highly constrained. Both ATAC peaks and dREG sites displayed levels of selection near comparable to neutral control whether they would be in promoter regions or distal intergenic regions (Fig. <a href="#Fig2" class="usa-link">2</a>d). The distinct asymmetry between the density of dREG sites around genes may be in part driven by roles in Pol-II initiation of transcription upstream and Pol-IV/Pol-V silencing of elements downstream of the focal gene. However, the epigenetic context here with respect to methylation of nearby TEs may also be associated with a separate noncoding transcription pathway in which a subset of apparently highly expressed genes, particularly in grasses, have notable CHH islands in nearby repetitive elements [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. Although it is beyond the scope of this work, it would likely be valuable to use Pol-IV and Pol-V antibodies to separate dREG elements linked to silencing associated with these CHH elements (Supplementary file 1: Fig. S15) from distal enhancers or proximal activating elements. A similar trend in dREG locations was previously observed in rice [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>] and human populations [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]; in these cases, there was an excess of sites under weak negative selection was observed, suggesting selection since the most recent common ancestor. Taken together, these results suggest dREG sequence features display some level of conservation that is stronger in proximal peaks than distal locations, although this distinction may be due to the increased occurrence of repetitive elements with the latter.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Chromatin landscape show distinct patterns between regulatory sequence features</h3>
<p id="Par43">To further explore the limited intergenic co-occurrence observed among the three regulatory sequence classes, we investigated whether they exhibited distinct patterns of chromatin features. We compared the chromatin features associated with intergenic CNSs, dREGs, and ATAC-seq sites using previously generated genome-wide maps of histone modifications and cytosine methylation [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. This comparison involved correlation analyses and assessments of signal distribution within and 1kb upstream and downstream of each intergenic dataset.</p>
<p id="Par44">DNA methylation is often found in inactive regulatory elements, where their target genes are repressed, and as in other plants different methyltransferases target three types of sites: CpG, CHG, and CHH [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>]. We found that CNSs had at most only a slightly negative correlation with methylation in all contexts and a depletion in DNA methylation compared to nearby locations (Fig. <a href="#Fig3" class="usa-link">3</a>). Histone-tail modifications are also associated with regulatory elements, and we found an enrichment for H3 lysine 18 acetylation (H3K18ac) commonly associated with DNA demethylation. CNSs were also enriched for the common repressive mark H3 lysine 27 trimethylation (H3K27me3); however, when we parsed CNSs by genomic regions, the enrichment for H3K27me3 varied, for example by being depleted in CNSs located downstream 500 bp from TSS (Additional file 1: Fig. S16, Additional file 3: Table S3). This suggests that <em>Oryza</em> CNS are not ubiquitously active in the way that ATAC and dREG sites may be, but experience context-dependent activation in accessible and closed configurations (Fig. <a href="#Fig3" class="usa-link">3</a>b).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2a59/12359857/908e0d9a5a8c/13059_2025_3715_Fig3_HTML.jpg" loading="lazy" id="MO3" height="913" width="778" alt="Fig. 3"></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Evaluation of the chromatin landscape for conserved, open and transcribed chromatin elements.<strong> a</strong> Pearson’s correlation matrices for high-confidence, uniquely mapped reads for histone modification, DNA methylation, and open chromatin sequence features in genomic regions corresponding to intergenic CNSs (left), intergenic ATAC (middle), and intergenic dREG (right). The chromatin signal (reads) for each chromatin mark was summed for each location and standardized for location length. <strong>b</strong> Chip: input ratio of histone modifications at +/− 1 kb around the same intergenic CNS sites (left), ATAC sites (middle), and dREG sites (right) used in <strong>a</strong>; and distribution of methylated cytosine in all contexts (CHH, CHG, CpG) as a percentage of sites</p></figcaption></figure><p id="Par45">In contrast, ATAC sites frequently colocalized with histone H3K4me3, as well as H3K27 acetylation, both marks being suggestive of transcriptional activity (Fig. <a href="#Fig3" class="usa-link">3</a>). DNA cytosine methylation was also enriched in ATAC peaks. Intergenic dREG sequences displayed distinct chromatin patterns, notably a depletion of H3K27me3 histone marks at dREG locations. This was coupled with an enrichment for H3K18ac within and asymmetrically around these regions, despite also being enriched in cytosine methylation and not strongly correlating with accessible chromatin.</p>
<p id="Par46">These observations raise questions on whether these sequence categories, despite differences in their genomic locations as well as their identification methods, do share functional similarities, or are instead functionally distinct. For example, while both CNSs and dREG sequence features contained regions enriched for DNA methylation that are often positively correlated with accessible chromatin, the CNS elements are less targeted by cytosine methylation, whereas dREG elements tend to be more targeted by this repressive mechanism.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Differences in chromatin profiles shape dREG, ATAC, and CNS distributions with annotated transposable elements, or is it the other way around?</h3>
<p id="Par47">A stronger correlation with cytosine methylation, particularly in the CHH context, might suggest an overlap with other types of genomic features whose expressions are more ubiquitously repressed, such as transposable elements (TEs). To address the question of whether dREG locations and ATAC peaks show a different epigenomic landscape from regulatory elements (such as CNSs that are depleted for TEs), we first focused on intergenic regions and looked for enrichment of annotated TEs in all three sequence features. The annotated TE dataset in the Azucena reference was generated for this study (see "<a href="#Sec1" class="usa-link">Methods</a>") and the methylation patterns within all annotated TEs that overlapped with our sequence features agreed with patterns detected in plants—that is, the CpG context being predominant and the CHH context having the lowest occurrence (Additional file 1: Fig. S17 and S18) [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>].</p>
<p id="Par48">We find that ATAC and dREG locations were similarly enriched for TEs, while CNSs were largely depleted of TEs (Fig. <a href="#Fig4" class="usa-link">4</a>a). This depletion in CNS was not necessarily surprising given that the requirement for TEs to be constrained over evolutionary time would be rarer and potentially imply novel function (TE exaptation[<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>, <a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]). This also highlights the potential different roles these regions are bearing.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2a59/12359857/8e412e38b51f/13059_2025_3715_Fig4_HTML.jpg" loading="lazy" id="MO4" height="936" width="778" alt="Fig. 4"></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The presence of transposable elements overlapping genomic classes and its impact on their methylation profile.<strong> a</strong> Enrichment relative to IID expectation for dREG peaks, ATAC peaks, and CNSs with small RNAs, TEs, and CHH methylation. <strong>b</strong> Total methylation percentage and methylation in all three contexts (CpG, CHG, CHH) for dREG sites that are either proximal or distal from annotated genes. Significant differences are calculated using the Mann–Whitney <em>U</em> test with Bonferroni correction. <strong>*</strong> 0.01 &lt; <em>p</em> &lt; 0.05; <strong>****</strong>
<em>p</em> ≤ 0.0001. <strong>c</strong> Spearman correlation for CHH methylation context and dREG scores. <em>P</em> value for dREG proximal is 0.831 and <em>P</em> value for dREG distal is 0.007. <strong>d</strong> Spearman correlation for CHH methylation context and ATAC signal in proximal and distal regions from genes. There is a significant positive selection for proximal ATAC signal (<em>P</em> value 6.7 × 10<sup>−10</sup> compared to distal, which is slightly positive yet significant <em>P</em> value = 2.74 × 10.<sup>−13</sup>)</p></figcaption></figure><p id="Par49">dREG locations were particularly prone to overlap with TEs, with most intergenic dREG locations overlapping partially with repetitive elements (10% or more with an annotated TE), while a small proportion (5.04%) overlapped entirely with a TE (Fig. <a href="#Fig1" class="usa-link">1</a>b; Additional file 3: Table S4). An increase in percentage overlap led, as expected, to higher levels of methylation across all three cytosine contexts. Surprisingly, we noted that the highest methylation level for CHH was found in dREG locations with 30% or more overlap with TEs (and not 100% overlap) (Additional file 1: Fig. S19). When we mapped the dREG locations spanning 30% of TEs onto each respective TE, we found that the overlap coincided with TE termini as well as some domains in the middle. As TE termini are more readily targeted by siRNA [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>], this could potentially explain the increase we detect.</p>
<p id="Par50">To see whether proximity to genes influenced the silencing mechanisms due to the presence of TEs, we compared dREG locations with ATAC peaks based on their proximity to genes – proximal (&lt; 1 kb from genes) versus distal (&gt; 1 kb away). Methylation was significantly higher in dREG<sub>proximal</sub> sites than dREG<sub>distal</sub> locations, particularly in the CHG and CHH context (Fig. <a href="#Fig4" class="usa-link">4</a>b). The CHH context, previously associated with TE silencing [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>–<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>] would suggest higher pressure to keep TEs silenced in regions proximal to genes, but this would have an impact on the activity of nearby <em>cis</em>-regulatory elements. To test this, we considered the dREG score and ATAC signal as proxy to higher bi-directional transcriptional activity and accessible chromatin activity, respectively (Fig. <a href="#Fig1" class="usa-link">1</a>a, Additional file 1: Fig. S7).</p>
<p id="Par51">Since a higher dREG score was associated with higher bi-directional transcriptional activity, we plotted dREG location score and methylation for both dREG<sub>proximal</sub> and dREG<sub>distal</sub> locations. We did the same with ATAC peaks by using an increase in signal as the detection of a more robust location or a region more accessible.</p>
<p id="Par52">We found that while overall methylation levels were negatively correlated with dREG scores (<em>p</em> &lt; 1 × 10<sup>−4</sup>) (Additional file 1: Fig. S20), but CHH methylation, had a slight but significant positive correlation (<em>p</em> &lt; 7 × 10<sup>−3</sup>) with dREG scores in dREG<sub>proximal</sub> locations whereas no correlation for dREG<sub>distal</sub> locations (<em>p</em> &lt; 0.8) was detected (Fig. <a href="#Fig4" class="usa-link">4</a>c). For ATAC peaks and CHH methylation, we detected a stronger positive correction in proximal versus distal ATAC peak (Fig. <a href="#Fig4" class="usa-link">4</a>d). This suggests that promoter regions can be accessible for mediating transcription while maintaining low TE activity.</p>
<p id="Par53">The presence of TEs and their associated epigenetic patterns could impact the genomic location of elements that are depleted for it, such as the CNSs. Indeed, in our Azucena leaf tissue, we see low enrichment of co-occurrence between intergenic CNSs and ATAC and dREG (Fig. <a href="#Fig1" class="usa-link">1</a>c). To test this, we removed all CNSs, ATAC, and dREG sites that overlapped with at least 1 bp of an annotated TE and performed an enrichment analysis for dREG and ATAC in CNSs. We found that dREGs were 3.1 × enriched in CNSs and ATAC sites were 3.7 × enriched in CNSs, compared to the presence of TEs. This could support the idea that TE could physically impact co-occurrence accessibility of CNSs.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Functionality of dREG locations and gene expression</h3>
<p id="Par54">Given the observed overlap of dREG and ATAC sequence features with transposable elements, we wanted to address the question of how might small RNAs, known to be involved in silencing transposable elements, contribute to the regulation of nascent transcripts, and how to parse the signal to retain transcribed regulatory elements potentially involved in enhancing gene expression. The output of nascent transcription assays such as PRO-seq provides a measure of the activity of all RNA polymerases, not only RNA Pol II. We therefore expected to detect both regions that could potentially activate but also be involved in signaling mechanisms. The strong overlap with TEs and the enrichment for cytosine methylation among our dREG locations points to this direction and this is what we observe. In plants, the RNA-directed DNA methylation (RdDM) pathway is crucial for de novo methylation, wherein transcribed noncoding RNA molecules direct the addition of DNA methylation to specific DNA sequences that are largely associated with transcription repression [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>, <a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>]; thus, the PRO-seq data in intergenic regions is likely, in part, be detecting silencing-related transcription.</p>
<p id="Par55">To investigate this, we first complemented a small RNA (sRNA) dataset previously published [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] with two replicates in the same cultivar background to generate an Azucena annotation database of small RNAs (Additional file 1: Fig. S21; Additional file 3: Table S5; Additional file 7). We found that dREG locations were fairly enriched (3.07x) in sRNAs (Fig. <a href="#Fig4" class="usa-link">4</a>a) with a small asymmetry in the sRNA enrichment between regions upstream of TSS (2.97x) versus downstream of stop codons (3.17x). This is a contrast to CNSs and even ATAC peaks that are about three times less enriched (Fig. <a href="#Fig4" class="usa-link">4</a>a). dREG sites with higher scores had less small RNAs but the abundance of TEs remained unchanged whether dREG sites had lower or higher scores (Additional file 1: Figs. S17 and S21). ATAC peaks had the opposite trend on dREG scores, where higher dREG scores had higher ATAC enrichment (Additional file 1: Fig. S21). This finding suggests that we are detecting numerous transcripts likely produced by RNA polymerases IV and V and that these small RNAs target TEs that overlap with dREG locations. It also suggests that dREG score may not be a reliable indicator to solely consider in plants when trying to parse silencing mechanism from an active role in enhancing transcription, as is commonly done with dREG analysis in humans [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>].</p>
<p id="Par56">One of the goals for this study was to identify potential regulatory elements that are transcribed, such as enhancers. To determine whether we may have candidate eRNA enhancer sequences in our dREG dataset, we needed to parse the elements that had likely functions in silencing mechanisms.</p>
<p id="Par57">To do this, we first established a threshold for the level of overlap with TEs that would be permitted. We analyzed dREG locations with varying degrees of overlap with TEs and observed compared based on patterns of levels of methylation in all contexts in dREG locations and 1 kb around (Additional file 1: Figs. S19 and S22). For the methylation profile, particularly in the CHH context, we saw that the profile was distinct for dREG locations with less versus more than 20% overlap with TEs (Fig. <a href="#Fig5" class="usa-link">5</a>a). Based on this, we focused on dREG locations having less than 20% overlap and performed enrichment analysis of these dREG locations for sRNAs, accessible chromatin, and the common histone modification associated with enhancers (H3K27ac). An overlap of 10% or more with TEs led to noticeable changes with a marked increase of sRNAs and a decrease in chromatin accessibility (Fig. <a href="#Fig5" class="usa-link">5</a>b). The dREG locations with more than 10% overlap also lost some of the asymmetrically co-located H3K27ac marks that were previously detected for candidate enhancers determined by dREG locations [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] (Fig. <a href="#Fig5" class="usa-link">5</a>a). dREG sites with a signal of less than 0.4 were the most enriched for sRNAs, and while they appeared to be more accessible than locations with higher scores, many regions may be actively silenced (Additional file 1: Fig. S21).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359857_13059_2025_3715_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2a59/12359857/c17bbced0019/13059_2025_3715_Fig5_HTML.jpg" loading="lazy" id="MO5" height="778" width="708" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Selection of filtered dREG sites based on epigenetic profile and annotated features. <strong>a</strong> Distribution of the percentage of methylated cytosine for the three different cytosine contexts CpG, CHG, and CHH sites and the chip/input ratio of histone modification H3K27ac at and +/− 1 kb around dREG locations that either do not overlap with TEs or overlap with increasing percentage. <strong>b</strong> Enrichment relative to IID expectation for dREG sites overlapping with no transposable element (TE), 10% overlap with TE, and more than 20% overlap with TEs. <strong>c</strong> Violin plot showing the expression levels (log2 Reads per kilo base pairs per million mapped reads, RPKM) for genes in proximity or not from dREG sites. dREG<sub>proximal</sub> locations were divided by whether the application (postfilter dREG<sub>proximal</sub>) or not (prefilter dREG<sub>proximal</sub>) of filters (see Main text) were applied to retain candidate sites more likely to have a role in promoting gene expression. Significant differences after all pairwise comparisons using the Mann–Whitney <em>U</em> test with Bonferroni correction. <strong>****</strong>
<em>p</em> ≤ 0.0001</p></figcaption></figure><p id="Par58">Based on these observations the following filtering steps were applied. We focused on (<em>i</em>) dREG locations with less than 10% overlap with TEs, (<em>ii</em>) removed dREG locations that overlapped with sRNAs, and (<em>iii</em>) removed dREG locations with a score of less than 0.4. Following these filtering steps, we obtained a dataset of 1068 post-filtered dREG locations (herein dREG<sub>postfilter</sub>) (Additional file 8).</p>
<p id="Par59">Next, we investigated whether dREG<sub>postfilter</sub> regions were more likely to function as enhancers. It is known that regulatory elements located in proximal promoter regions typically contain binding sites that influence the expression of downstream nearby genes. We explored whether the predicted dREG regulatory elements could correlate with increased nearby gene expression patterns. This could support a potential role of these elements in transcription regulation. To do so, we took dREG locations detected in the promoter region of all annotated genes (dREG<sub>proximal</sub>) and examined how their activity associated with expression of nearby genes obtained from an Azucena leaf transcriptome. The results showed that genes with a dREG<sub>proximal</sub> element in its promoter region had significantly higher gene expression (Mann Whitney <em>U</em> test with Bonferroni correction <em>p</em> &lt; 0.0001; Fig. <a href="#Fig5" class="usa-link">5</a>c). We also detected a significant increase in nearby gene expression when considering dREG<sub>filtered+</sub><sub>proximal</sub> elements (Mann Whitney <em>U</em> test with Bonferroni correction <em>p</em> &lt; 0.0001). The distribution of transcript expression varied in complexity in the presence of a dREG element and while dREG<sub>proximal</sub> prefilter and post filter were similar, we could see a marked distribution towards higher transcript expression with lower variability (Fig. <a href="#Fig5" class="usa-link">5</a>c; Additional file 1: Fig. S23).</p>
<p id="Par60">In plants, the only dREG threshold that has been explicitly used to characterize these regulatory elements was a dREG score of &gt; 1.0 in the rice genome [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Based on this, we examined the relationship between dREG score and functional genomic activity by comparing the relationship between gene expression levels and dREG<sub>proximal</sub> scores. Results showed no significant difference in expression in dREG<sub>proximal</sub> based on score, whether they would be pre or post filtering scores (Additional file 1: Fig. S24). Altogether, this suggests that predicted dREG elements, at least in proximal regions, may have an activating role in gene expression. Also, dREG score, which relies training on a robust set of eRNAs in human, may not be a primary metric to identify putative active transcribed regulatory elements (Additional file 1: Fig. S21).</p></section><section id="Sec9"><h3 class="pmc_sec_title">Identifying distal active regulatory elements through chromatin architecture</h3>
<p id="Par61">In contrast to proximal regulatory elements, distal elements associated with CNS, ATAC, or dREG<sub>distal</sub> sequence features that may contribute to gene expression regulation are more challenging to link to gene(s). Distal transcribed regulatory elements associated with dREG<sub>distal</sub> sites may signal RNA-producing enhancers, producing unstable transcripts in both directions [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>]. Indeed, in animals, genomic coordinates of distal transcribed regulatory elements have often overlapped with enhancers that actively produce eRNAs, although in plants there is no evidence that these are indeed associated with enhancer activity. In addition, accessible chromatin is also used as a proxy for bearing active transcriptional elements [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. While some level of redundant activity might be expected for conserved elements like CNSs within a given tissue and developmental context, others would likely exhibit distinct patterns linked to their specific spatio-temporal functions.</p>
<p id="Par62">To further characterize rice dREG<sub>postfilter+</sub><sub>distal</sub> locations, distal ATAC peaks and distal CNSs, and identify potential functional roles in the rice genome, we examined long-range chromosomal contact interactions between genes and our candidate <em>cis</em>-regulatory elements using the Pore-C sequencing method [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>]. Pore-C couples chromatin conformation capture with long-read nanopore sequencing to detect genome-wide multi-way chromatin contacts. It has been shown to be highly effective at sequencing through repeat regions of the genome and is able to detect increased contact intensity with less sequencing reads than conventional Hi-C sequencing [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>]. Using Pore-C sequencing on rice leaves, we generated 104 million concatamer reads that correspond to 290 million contacts across the rice genome (Additional file 3: Table S6).</p>
<p id="Par63">Since Pore-C sequencing has not been applied in the rice genome to profile chromatin activity (only in Arabidopsis [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>], we first examined the functionality of the 3D genome architecture detected by this method. Using the Pore-C method, we detected 3261 distinct topologically associated domains (TADs) which are localized chromosomal regions of high physical contact [<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>]. TADs had a median size of 80 kb, and the TAD boundaries were enriched for transcription and active chromatin marks (Fig. <a href="#Fig6" class="usa-link">6</a>a; Additional file 1: Fig. S25). These results were consistent with previous <em>Oryza</em> Hi-C sequencing results [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>] indicating that Pore-C sequencing was able to detect functional 3D contacts across the <em>Oryza</em> chromosome.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2a59/12359857/417d946011bf/13059_2025_3715_Fig6_HTML.jpg" loading="lazy" id="MO6" height="923" width="778" alt="Fig. 6"></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Exploration of regulatory element candidates on target gene expression through chromatin architecture and eQTL overlap.<strong> a</strong> PRO-seq read count (plus, top; minus, bottom) enrichment surrounding TADs. Shown are 10 kbp upstream and downstream of TADs with the TAD scaled to 5 kb. PRO-seq read counts were averaged in 100-bp windows. <strong>b</strong> Proportion of gene–gene, gene-noncoding, and noncoding-noncoding loops that were detected using Pore-C sequencing. <strong>c</strong> Aggregate Peak Analysis (APA) plots showing the aggregated Pore-C contacts around chromatin loops identified in all chromosomes (top) and only between a noncoding-gene loop (bottom). The plot is a pile-up of 25 kb upstream and downstream of loop anchors (centered in each axis) of every identified loop. Color represents log2 fold enrichment of the observed aggregated matrix over a normalization matrix that was aggregated from randomly shifted controls regions across the chromosome. <strong>d</strong> Gene expression level for genes that are contacting a noncoding anchor with either an ATAC peak, dREG<sub>postfilter+distal</sub> location, CNS location, or no annotation. ns = non-significant; * 0.01 &lt; <em>p</em> &lt; 0.05; ** <em>p</em>-value &lt; 0.01; *<strong>***</strong>
<em>p</em> ≤ 0.0001 after a Mann–Whitney <em>U</em> test with Bonferroni correction. <strong>e</strong> Association analysis between eQTL regions and dREG<sub>postfilter+distal</sub> sites (left), ATAC (middle), and CNS (right) present in loops as identified in Fig. 6d. The analysis was performed on 1000 permutation tests. Gray curves represent the number of overlaps of eQTLs found in randomized regions with same length and chromosome location. The green line indicates the number of eQTLs regions found in candidate regulatory sites, the black line represents the mean of the randomized evaluations and red line corresponds to the significance limit of <em>p</em>-value = 0.05</p></figcaption></figure><p id="Par64">To determine whether distal elements in our three sequence features could represent distal regulatory elements of one or multiple target genes in juvenile leaf tissues, we used the Pore-C sequencing data to detect the formation of chromatin loops at 5 kb resolution. Using the FitHiC2 algorithm [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>], we detected 33,779 chromatin loops, where 42% (14,325) of those loops involved a gene (where the coding window had to contain more than 100 bp of coding sequences) and a noncoding region (Fig. <a href="#Fig6" class="usa-link">6</a>b). These represent candidate gene-regulatory element loop interactions. We then visualized this loop interaction by conducting aggregate peak analysis (APA), which measures the enrichment of chromatin contacts at loop positions relative to the local background signal (Fig. <a href="#Fig6" class="usa-link">6</a>c). When we compared the APA plots for all chromatin loops and candidate gene-regulatory element loops, the latter loops had strong enrichment of signals centered at the contact point. Specifically for the candidate gene-regulatory element loops, the central window (i.e., the contact point) had ~ 3.5 fold increased in contacts compared to the lower left windows (i.e., background contact levels) (Fig. <a href="#Fig6" class="usa-link">6</a>c).</p>
<p id="Par65">Next, we examined the potential functionality of candidate gene-regulatory element loops. We analyzed gene-regulatory element loops where the noncoding anchor had only an ATAC peak, only a dREG location, only a CNS, or combinations of elements in a loop (Fig. <a href="#Fig6" class="usa-link">6</a>d). These loops were then compared to loops where the noncoding anchor had no detected element and represented control gene-noncoding sequence interactions (Additional file 1: Fig. S26). We found that for all gene-noncoding sequence loops, and regardless of the annotation within the noncoding anchor, the majority of genes had contact with a single noncoding region (Additional file 1: Fig. S26). Furthermore, the median distance between the gene and the noncoding region was 128 kb to the dREG only peak, 163 kb to the ATAC only peak, and 35 kb the CNS only peak. Focusing on loops that contacted only one genic window (7386 loops contacting one genic window) to ease interpretation of the results, we looked at the gene expression profile of genes that contacted one of our candidate regulatory regions. There were 2123 genes that had contact with a CNS region, 165 genes with dREG<sub>postfilter+distal</sub>, and 2284 with ATAC. We found that the presence of either a CNS, ATAC, or dREG<sub>postfilter+distal</sub> sequence features led to significantly higher gene expression than those that contacted a noncoding anchor with no annotation (Mann Whitney <em>U</em> test CNS <em>p</em> &lt; 0.0001, ATAC <em>p</em> &lt; 0.001, dREG<sub>filtered</sub>
<em>p</em> &lt; 0.01, and Fig. <a href="#Fig6" class="usa-link">6</a>d).</p>
<p id="Par66">Loops with an ATAC site had the lowest median gene expression among the sequence features analyzed. In contrast, the presence of dREG-filtered locations was associated with the highest median gene expression. However, it is important to note that the number of loops analyzed for dREG-filtered regions was more than ten times lower due to the filtering steps applied to retain likely enhancer-like candidates. Gene ontology enrichment analysis highlighted more similarities in pathways or function for genes in contact loops with all three classes of regulatory sites, than what we saw with gene in proximity to these sites, particularly regarding metabolic processes (Additional files 4–6). However, we still detected an enrichment for biological processes related to response to stress and response to stimulus for ATAC-gene loops, while CNS-gene loops were enriched for development and reproduction. Interestingly, genes in loops in contact with dREG<sub>postfilter+distal</sub> sites appear to have functions related to hormone regulation and chromatin organization, although the limited number of genes impacted the significance of some of the terms (Additional files 4–6). As expected, fewer loops contained a combination of sequence features, with 27 loops having all three classes of sequences (Supplementary file 1: Fig. S27).</p></section><section id="Sec10"><h3 class="pmc_sec_title">
<em>dREG</em><sub><em>distal</em></sub><em> locations are targets of transcriptional regulation</em>
</h3>
<p id="Par67">As an orthogonal approach to investigate the gene regulatory functions of dREG peaks, we identified expression quantitative trait loci (eQTLs) associated with gene expression across a panel of rice varieties and intersected those eQTLs with dREG, CNSs, and ATAC sequence features. Using gene expression and SNP data from 84 japonica rice accessions grown in well-watered field conditions [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>] we detected 489,248 eQTLs after an FDR of 5%. We intersected the significant eQTLs with our sequence features and found an overlap of 1169 eQTLs with dREG<sub>filtered</sub>, 11,656 with ATAC, and 886 with intergenic CNS.</p>
<p id="Par68">To test the significance of the observed overlap, we generated a distribution by randomly sampling the potentially non-functional regions of the genome, which were matched for size and total number of CNS, ATAC, and dREG locations. We first examined these three types of genomic classes that were limited to those forming loops with genes and potentially involved in transcription (identified and analyzed in Fig. <a href="#Fig6" class="usa-link">6</a>d). Results showed that the observed number of overlaps between eQTLs and dREG<sub>postfilter+distal</sub> site was higher than the maximum number of overlaps in the distribution across random regions, indicating dREG<sub>postfilter+distal</sub> sites are enriched for eQTLs (Fig. <a href="#Fig6" class="usa-link">6</a>e). eQTLs in ATAC locations were not significantly represented, while the trend being towards an underrepresentation compared to random regions. CNS regions were significantly underrepresented for this given eQTL panel (Fig. <a href="#Fig6" class="usa-link">6</a>e). We note that the dREG<sub>postfilter+distal</sub> sites were close to α=0.05, but the sample size (i.e., number of dREG sites in loops) was much smaller than ATAC and CNS. We expanded our eQTLs analysis by considering overlap with all dREG<sub>postfilter</sub>, CNS intergenic and ATAC intergenic, regardless of loops, and found a similar trend, with the increased sample size increasing the significant. For example, ATAC peaks were now significantly underrepresented for eQTLs, while eQTLs in dREG<sub>postfilter</sub> sites were more significantly overrepresented (Additional file 1: Figs. S28-S30). Taken together, these results suggest that a number of elements, including dREG<sub>postfilter</sub> sites, could be valuable candidate transcribed regulatory elements that could impact gene activity by promoting transcription.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par69">The rice genome encodes a large number of transcripts that require spatio-temporal regulation and for which many of the <em>c-</em>CREs remain to be characterized. In this study, we explored conservation and complementary functional genomics sequence features in the rice (<em>O. sativa</em>) cultivar Azucena to characterize CREs <em>s</em> that have emerged at genomic sites. Azucena is a tropical japonica landrace often used as a parent line in rice breeding programs, and its regulatory diversity has likely not been affected by repeated artificial selection during modern rice breeding [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>].</p>
<p id="Par70">We found that the three sequence features used in this study—a set of CNSs conserved across the <em>Oryza</em> genus, accessible chromatin regions specific to the Azucena genome, and elements detected from bi-directional transcription (dREG sites), likely have distinct regulatory functions, as summarized in Fig. <a href="#Fig7" class="usa-link">7</a>. These sites do not overlap significantly more than may be expected by chance alone and are not enriched for similar motifs. One hypothesis for this would be that they may not share converging evolutionary trajectories with respect to emerging and established constraint. A lack of spatial co-enrichment would not necessarily preclude different types of sites from sharing similar functions, as it would still be possible for these different types of sites to share binding-motif similarities; we did not, however, observe similarly enriched sequences or even similar levels of sequence complexity between these site classes.</p>
<figure class="fig xbox font-sm" id="Fig7"><h3 class="obj_head">Fig. 7.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12359857_13059_2025_3715_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2a59/12359857/847b2f17a9c4/13059_2025_3715_Fig7_HTML.jpg" loading="lazy" id="MO7" height="214" width="708" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Schematic overview of the main findings of this study. It is focusing on three classes of genomic elements: CNS (conserved noncoding sequences), dREG (regulatory regions identified by dREG), and ATAC (ATAC-seq peaks). Genomic features are depicted as rectangles: light gray for <em>cis</em>-regulatory elements (CREs), black for coding sequences (CDSs), and dark gray for transposable elements (TEs). Small RNAs (sRNAs) and messenger RNAs (mRNAs) are indicated, and DNA methylation on TEs is shown as gray circles labeled “Me.” The width of each peak reflects the median length (in nucleotides) of the corresponding feature, and color intensity qualitatively represents signal strength</p></figcaption></figure><p id="Par71">Consequently, the hypothesis that these elements are proto-CNS sequences whose novelty or translocation precludes alignment between species, may be tentatively rejected due to the uniqueness of motif profiles to each regulatory class (Fig. <a href="#Fig2" class="usa-link">2</a>). Similarly, the tandem reiteration of short simple sequences in open chromatin sites that were not CNSs suggests considerable robustness to mutations, and hence low constraint. In contrast, more complex CNS sequences that may be present in one or two copies in a promoter are more likely to result in phenotypic change after disruption, leading in turn to their higher levels of constraint.</p>
<p id="Par72">Studies have frequently focused on the characterization of regulatory elements in regions at or immediately upstream of TSS [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>]. But even given the difficulty of precisely predicting TSS sites for plant genes, we wished to examine potential upstream sites to potential regulatory elements downstream of the TSS (Fig. <a href="#Fig2" class="usa-link">2</a>), as well as much more distal elements. This suggests that CNSs and dREG sites were not only enriched upstream, but also downstream of TSS (500 bp), particularly in genes with high density of CNSs. However, the frequency of motifs enriched, such as GAGC, were subtly shifted between these two locations. This is consistent with recent findings (Voichek et al. [<a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>]) that also show an asymmetric impact of position with respect to TSS for binding sites such as those similar to the GATC motif in populations of <em>A. thaliana</em>. Further analysis of such position-dependent enhancers could help better understand regulatory element movement and activity (from proximal to distal, or from downstream/upstream regions).</p>
<p id="Par73">Accessible chromatin regions (e.g., ATAC peaks) are often used as an indicator of the presence of active distal CREs [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>]. In mammalian genomes, active enhancers are also characterized by an active local transcription termed eRNAs that is increasingly associated with functions such as recruiting transcription-related proteins and stabilizing or promoting transcription loops [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>, <a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>]. Indeed, RNA Pol II has been mapped to enhancers [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>, <a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>] and in humans the histone acetyltransferase p300 and histone-tail modifications such as H3K4me1 are both involved in its recruitment [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>]. Depending on their activity or potential to be activated, enhancers can adopt distinct functional states that have been termed active, primed, latent, poised, or repressed [<a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>–<a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>]. To preserve their functional activity, active enhancers maintain an active chromatin conformation [<a href="#CR97" class="usa-link" aria-describedby="CR97">97</a>, <a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>]. In this study, we used the PRO-seq method to identify eRNA producing regions using the dREG analysis pipeline [<a href="#CR99" class="usa-link" aria-describedby="CR99">99</a>]. In plants, Lozano et al. [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>] used the same approach (dREG/PRO-seq) in maize where they found that 31% of their identified transcribed intergenic elements co-located with a list of distal ATAC peaks characterized by Ricci et al. [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>], as well as an overlap of 17% between their list of intergenic regulatory elements and CREs found to form CRE-gene loops. Our results suggest a low co-location between dREG and ATAC peaks (Figs. <a href="#Fig1" class="usa-link">1</a> and <a href="#Fig6" class="usa-link">6</a>). This could be due in part to differences between large-genome and small-genome genome species with respect to their 3D chromatin architecture, as has been reported in a growing number of chromosomal contact experiments in plants [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>, <a href="#CR100" class="usa-link" aria-describedby="CR100">100</a>–<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a>]. We found that a proportion of dREG<sub>postfilter</sub> sites could be functional distal transcribed regulatory elements, as we detected a significant increase in gene expression in gene-noncoding loops that contain a dREG site, compared to gene-noncoding loops with no CNS, dREG, nor ATAC sites. However, we detected 27 loops where all three sequence features were present and looped into genes having the highest gene expression (Fig. <a href="#Fig6" class="usa-link">6</a>e). We suggest that while they may act primarily independently for many classes of genes, these sequence features may also act to more synergistically to modulate the expression of some classes of genes, for example house-keeping genes. Unfortunately, due to a low number of genes gene ontology enrichment analysis did not highlight specific pathways or function. We also have to point out that the size of each loop is 5kb, which is considerably larger than the median of our sequence features; therefore, future iterations of Pore-C or other 3D architecture methods could help fine-tune our chromatin contact map (i.e., narrower window size). Interestingly, these observations are consistent with the Micro-C maps recently published by Kurbidaeva et al., where TAD boundaries coincide with peaks of nascent transcription and exhibit coordinated gene activity, underscoring the regulatory importance of 3D chromatin compartments [<a href="#CR103" class="usa-link" aria-describedby="CR103">103</a>]</p>
<p id="Par74">One of the observations made in this study that distinguished CNSs from dREG and ATAC regions is the impact of the presence of repetitive elements enriched in the detected dREG locations and ATAC peaks (Fig. <a href="#Fig4" class="usa-link">4</a>). For dREG and ATAC peaks that partially or entirely overlapped with annotated repetitive elements, we detected a higher percentage of methylation across all three cytosine contexts. Interestingly, we observed an increase in CHH methylation, which occurs predominantly at TEs, and has been shown to be involved in the prevention of transposon jumping during development in Arabidopsis [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR104" class="usa-link" aria-describedby="CR104">104</a>]; this methylation occurs through the plant-specific RdDM pathway that operates via noncoding RNA [<a href="#CR105" class="usa-link" aria-describedby="CR105">105</a>]. The fact that DNA methylation can positively or negatively impact transcriptional activity, for example by modulating binding affinity of TFs, makes it a confounding factor when characterizing candidate transcribed regulatory elements. In plants, we argue that methylation profiles should always be considered with PRO-seq sequence features to characterize the intergenic transcription signal.</p>
<p id="Par75">Finally, further characterization eRNA producing regions using alternative methods to detect eRNAs, even at low level, will help us better understand whether assumptions derived from mammalian studies hold true in plants. Recently, McDonald et al. [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] used capped small (cs)RNA sequencing to study the stability of detected bi-directional transcripts in <em>A. thaliana</em>, rice and other plants. They suggest that unstable transcripts, often a feature of 2D-eRNAs in mammals, are rare in plants and found low overlap with published self-transcribing active regulatory region (STARR) sequencing, another method used to identify candidate enhancers [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. Instead, they propose that the bi-directional signal correspond to elements generating stable RNAs (likely promoters).</p></section><section id="Sec12"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par76">Integrating evolutionary constraint, chromatin accessibility, and intergenic bidirectional nascent transcription, we reveal complementary layers of <em>cis</em>‑regulation in rice. TE‑poor conserved noncoding and bidirectionally transcribed accessible regions loop to gene promoters, co‑localize with eQTLs and coincide with elevated expression, which suggests an enhancer‑like activity. Future work focusing on better understand the line between promoters, enhancers, and regulation in <em>cis</em> versus <em>trans</em>, due to the relative instability of eRNAS [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR106" class="usa-link" aria-describedby="CR106">106</a>], in plants, could help us detect plant-specific hallmarks of enhancers, in addition to eRNAs.</p></section><section id="Sec13"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec14"><h3 class="pmc_sec_title">Plant material</h3>
<p id="Par77">Seeds of <em>O. sativa</em> landrace Azucena (PI 439017; tropical Japonica) were provided by the U.S. National Plant Germplasm (USDA) and were used to generate ATAC-seq libraries. Seeds were sown on soil supplemented with Fe2SO4 and Jack’s Fertilizer solution. Plants were grown for 15 days in growth chambers (12-h days; 30 °C/20 °C day/night; 300–500 μmol quanta m − 2 s − 1; relative humidity: 50–70%). Leaf tissue for library construction was collected from 17-day-old, young plants.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Chromatin accessibility</h3>
<p id="Par78">ATAC-seq was performed according to the protocol of Buenrostro et al. and Zhang &amp; Jiang [<a href="#CR107" class="usa-link" aria-describedby="CR107">107</a>, <a href="#CR108" class="usa-link" aria-describedby="CR108">108</a>]. Libraries were prepared from frozen plant tissue. 0.2–0.3 g flash frozen plant material was ground to a fine powder and completely dissolved in ice-cold nuclei isolation buffer (NIB; 10 mM Tris–HCL, 80 mM KCl, 10 mM EDTA, 1 mM spermidine, 1 mM spermine, 0.15% 2-mercaptoethanol, 0.5 M sucrose pH 9.5). The mixture was filtered through Miracloth (Millopore Sigma, Cat. # 475,855) and pelleted by centrifugation at 1100 × <em>g</em> for 10 min at 4 °C. The pellet was washed twice in 10 ml nuclei washing buffer [NWB; NIB with 0.5% Triton X-100 (Sigma, Cat. # T8787)] and again pelleted by centrifugation (1100 × <em>g</em> for 10 min at 4 °C). The nuclei were equilibrated in 10 ml nuclei digestion buffer (NDB; 10 mM Tris–HCl, 10 mM NaCl, 3 mM MgCl<sub>2</sub> pH 7.4) and centrifuged at 1100 × <em>g</em> for 10 min at 4 °C. The nuclei were resuspended in resuspension buffer (RSB; 20 mM Tris–HCl, 10 mM NaCl, 3 mM MgCl<sub>2</sub> pH 7.4), stained using the nucleic acid intercalator propidium iodide, and counted using the BD FACSJazz cell sorter. Afterwards, the nuclei were pelleted via centrifugation at 1100 × <em>g</em> for 10 min at 4 °C. A total of 50,000 nuclei were resuspended in a transposition reaction mix [Tagment DNA (TD) buffer, Tagment DNA Enzyme (TDE1) (Illumina, Cat. # 20,034,210), H<sub>2</sub>O] and incubated at 37 °C for 30 min while being gently shaken. The samples were purified using the QIAgen MinElute PCR Purification Kit (Cat. # 28,004) and eluted in ten µl elution buffer. The tagmented samples were amplified and tagged with custom sequencing primer. The resulting libraries were purified a second time using the QIAgen MinElute PCR Purification Kit and size-selected with AMPure XP beads (Beckman Coulter, Cat. # A63880). Libraries were sequenced using Illumina protocol for 2 × 100 bp on an Illumina NovaSeq 6000 with configuration S4.</p>
<p id="Par79">Sequenced reads were quality checked using the FASTQC tool (<a href="https://github.com/s-andrews/FastQC" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/s-andrews/FastQC</a>). The reads were 3’ trimmed to cut the adapters and parts of the reads with low quality (q &gt; 20) using Trimmomatic (v. 0.39) [<a href="#CR109" class="usa-link" aria-describedby="CR109">109</a>] with the options -headcrop:10 -minlen:16 and -slidingwondow:5:20. The reads were then aligned with the reference genome of <em>Oryza sativa</em> japonica Azucena (<a href="https://ftp.ensemblgenomes.ebi.ac.uk/pub/plants/release-57/fasta/oryza_sativa_azucena/dna/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://ftp.ensemblgenomes.ebi.ac.uk/pub/plants/release-57/fasta/oryza_sativa_azucena/dna/</a>) using the Bowtie2 (v. 2.4.4) [<a href="#CR110" class="usa-link" aria-describedby="CR110">110</a>] tool with the options -very-sensitive -X 1000. The Azucena reference genome included the chloroplast sequence (genbank ID: <a href="https://www.ncbi.nlm.nih.gov/nuccore/GU592207.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GU592207.1</a>) to allow chloroplast originating ATAC-seq reads and were subsequently removed using Samtools (v. 1.17) [<a href="#CR111" class="usa-link" aria-describedby="CR111">111</a>]. The duplicates were removed using Markduplicates (Picard v. 2.26.3 <a href="https://broadinstitute.github.io/picard/command-line-overview.html#MarkDuplicates" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://broadinstitute.github.io/picard/command-line-overview.html#MarkDuplicates</a>) followed by a shift of the reads of 4 bp on the positive strand and 5 bp on the negative strand due to the transposase binding using Deeptools (v. 3.5.4.) [<a href="#CR112" class="usa-link" aria-describedby="CR112">112</a>]. ATAC peaks were called using MACS2 (v. 2.2.8.) [<a href="#CR113" class="usa-link" aria-describedby="CR113">113</a>] peak calling function using genome size of Azucena rice and the options recommended for ATAC peak calling (-q 0.01 -g 379,627,553 –nomodel —shift 37 —extsize 73). FRIP scores for quality check were obtained using FeatureCounts tool (subread <em>v. 2.0.6</em>
<a href="https://github.com/ShiLab-Bioinformatics/subread" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ShiLab-Bioinformatics/subread</a>) An informed cutoff signal of 350 was determined to retain high-confidence ATAC peaks. At 350, we observed a gradual enrichment of ATAC peaks for dREG sites and a decrease in peak coverage for regions 500 bp upstream of TSS (see Additional file 1: Fig. S6). Access to the chromatin accessibility track is available via genome browser screenshots for visualization (see Additional file 1: Figs. S31-S39).</p></section><section id="Sec16"><h3 class="pmc_sec_title">sRNA-seq</h3>
<p id="Par80">Sequencing libraries for small RNA-seq were prepared as described in Joly-Lopez et al. [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. First reads were trimmed for quality (q &lt; 20) and adapter sequence (Trimmomatic v. 0.39) [<a href="#CR109" class="usa-link" aria-describedby="CR109">109</a>] and only reads longer than 10 bp and shorter than 40 bp were retained (-m 10 -M 40) (Cutadapt 1.11) [<a href="#CR114" class="usa-link" aria-describedby="CR114">114</a>]. Azucena and Nipponbare chloroplast genomes were indexed using bowtie-build and reads were aligned using Bowtie 2.2.9 [<a href="#CR110" class="usa-link" aria-describedby="CR110">110</a>] with the options -n 0 and -k 10. Reads were then counted using FeatureCounts (subread 2.0.3) [<a href="#CR115" class="usa-link" aria-describedby="CR115">115</a>]. sRNA localizations were identified in bam files with fasta sequence of small RNA found in Nipponbare genome and stacked against the genome of Azucena using Shortstack (<a href="https://github.com/MikeAxtell/ShortStack" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/MikeAxtell/ShortStack</a>).</p></section><section id="Sec17"><h3 class="pmc_sec_title">ChIP-Seq</h3>
<p id="Par81">ChIP-seq libraries were prepared as described in Joly-Lopez et al. [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Libraries were sequenced as 2 × 50-bp reads on an Illumina HiSeq 2500 instrument. Two biological replicates were generated, and a third replicate, generated under the same conditions and used in a previous study [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>], was used (SRA: SRX7082158 H3K4me3; SRX7082157 H3K18Ac; SRX7082156 H3K27Ac; SRX7082153 H3K27me3; Bioproject: PRJNA586887). Sequencing reads were adapter trimmed and QC controlled using the script bbduk.sh ver. 38.90 (<a href="https://sourceforge.net/projects/bbmap/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://sourceforge.net/projects/bbmap/</a>) with parameters: minlen = 16 qtrim = rl trimq = 20 ktrim = r k = 19 mink = 10 hdist = 1 tpe tbo. Trimmed reads were aligned to the Azucena reference genome [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>] (Bioproject PRJNA424001) using Bowtie 2 version 2.4.2. [<a href="#CR110" class="usa-link" aria-describedby="CR110">110</a>] under option very-sensitive and with the parameter -X 1000. ChIP-Seq peak calling was performed using MACS2, with input DNA used as a control and with the parameters -g 4.0e8 –bw 200 -B -m 3 50. Access to the histone modification track is available via genome browser screenshots for visualization (see Additional file 1: Figs. S31-S39).</p></section><section id="Sec18"><h3 class="pmc_sec_title">DNA methylation</h3>
<p id="Par82">Bisulfite sequencing libraries to measure DNA methylation were prepared as described in Joly-Lopez et al. [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. One biological replicate was generated and a second replicate, generated under the same conditions and used in a previous study [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>], was used (SRA: SRX7082155; Bioproject: PRJNA586887). Libraries were sequenced using Illumina protocols for 2 × 100-bp reads on an Illumina HiSeq 2500. Raw bisulfite sequencing (BS-seq) reads quality controlled using the program trim galore Ver. 0.6.6 (<a href="http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.bioinformatics.babraham.ac.uk/projects/trim_galore/</a>) with default parameters. We used bismark version 0.16.3 [<a href="#CR116" class="usa-link" aria-describedby="CR116">116</a>] for mapping the Bisulfite-seq reads and deduplicating reads and methylation percentage were calculated as done in Choi and Purugganan [<a href="#CR117" class="usa-link" aria-describedby="CR117">117</a>] where methylation was determined using a binomial distribution test. Cytosine sites with <em>p</em>-values less than 0.001 were then considered as sites with significant evidence of methylation.</p></section><section id="Sec19"><h3 class="pmc_sec_title">PRO-seq</h3>
<p id="Par83">PRO-seq libraries were prepared as described in Joly-Lopez et al. [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. The two replicates used in this study were generated as two independent experiments but growth in similar conditions. The raw reads were then used on the Proseq2.0 (<a href="https://github.com/Danko-Lab/proseq2.0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Danko-Lab/proseq2.0</a>) pipeline that automatically pre-processes the reads, aligns to the reference genome, and generates output bigWig files for downstream PRO-seq peak calling analysis. The <em>q</em> value was adjusted to 20. To identify peaks of divergent transcription activity we used the bigwig file generated from the previous step as an input for the cloud computing version of the dREG algorithm (<a href="https://dreg.dnasequence.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://dreg.dnasequence.org/</a>). dREG peaks were then obtained from two replicates. An enrichment analysis of dREG peak locations identified in each replicate shows an enrichment of 13.32 × compared to an equal number of randomly positioned peaks of identical size. To select dREG peaks candidates, regions that overlapped in both replicates as well as candidates that were located within 100 bp from one another in both replicates were selected for the following steps. Access to the PRO-seq tracks is available via genome browser screenshots for visualization (see Additional file 1: Fig. S31-S39).</p></section><section id="Sec20"><h3 class="pmc_sec_title">Pore-C data generation and computational processing</h3>
<p id="Par84">We generated Pore-C libraries following the protocol of Choi et al. [<a href="#CR118" class="usa-link" aria-describedby="CR118">118</a>], and sequencing library was prepared using the Oxford Nanopore Technologies standard ligation sequencing kit SQK-LSK109. Sequencing was conducted on a GridION X5 and PromethION sequencer and the raw data were base-called by Oxford Nanopore Technologies basecaller Guppy version 4.4.0 (available on <a href="https://community.nanoporetech.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://community.nanoporetech.com/</a>) on high-accuracy mode. The Pore-C data analysis was conducted using the Pore-C snakemake workflow developed by Oxford Nanopore Technologies (<a href="https://github.com/nanoporetech/Pore-C-Snakemake" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/nanoporetech/Pore-C-Snakemake</a>). Briefly, the pipeline first aligns the nanopore Pore-C chromosome contact sequence reads to the Azucena genome using bwa-sw version 0.7.17-r1188 [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>] with parameters -b 5 -q 2 -r 1 -T 15 -z 10. The alignment BAM file was processed with Pore-C tools (<a href="https://github.com/nanoporetech/pore-c" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/nanoporetech/pore-c</a>) to filter spurious alignments, detect ligation junctions, and assign fragments that originated from the same chromosomal contacts. The workflow also generates cool and hic files that can be used for downstream analysis.</p></section><section id="Sec21"><h3 class="pmc_sec_title">TAD and chromatin loop calling</h3>
<p id="Par85">The Pore-C contact matrix generated from the previous analysis was normalized using the KR algorithm [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>] with the computational suite HiCExplorer version 3.4 [<a href="#CR120" class="usa-link" aria-describedby="CR120">120</a>]. Using the normalized contact matrix the algorithm topdom [<a href="#CR121" class="usa-link" aria-describedby="CR121">121</a>] was used to call TADs as the method was shown recently to be a highly efficient and accurate method for detecting TADs. The topdom analysis was conducted using a 5 kb resolution contact matrix.</p>
<p id="Par86">Pore-C contact matrix was also used to statistically determine the significant chromatin contacts using the program FitHiC2 [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>]. Using the genomic distance between windows and their contact probability, FitHiC2 applies a spline fit to model an empirical null distribution and detect chromatin contacts as outliers to this null distribution. FitHiC2 was run with default parameters using the 5kbp resolution contact matrix, while setting the lower bound on the intra-chromosomal distance range (parameter -L) as 10 kb and upper bound (parameter -U) as 1 Mb. Candidate chromosome loops were filtered by selecting for window pairs that had a Benjamini–Hochberg procedure based false discovery rate threshold q-value &lt; 0.05. Window pairs that had significant evidence of contact were then classified as whether it was a coding or noncoding window by defining a coding window as those that contained more than 100 bp (i.e., greater than 2% of the window) of coding sequences.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Azucena reference genome repeat sequence and gene annotation</h3>
<p id="Par87">As the released Azucena reference on ENSEMBL [<a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>, <a href="#CR123" class="usa-link" aria-describedby="CR123">123</a>] lacked some gene annotations, we lifted coordinates from the well annotated rice cultivar Nipponbare to supplement the annotation; both Azucena and Nipponbare are members of the rice subspecies japonica. Azucena and Nipponbare syntenic alignments showed that both have similar levels of completeness and contiguity and thus provided confidence in our analyses using annotation in Azucena (Additional file 1: Figs. S1- S2). The small regions that mapped between different chromosomes in Azucena and Nipponbare do not represent evidence of reorganization but rather retained regions from an ancient grass tetraploidy event [<a href="#CR124" class="usa-link" aria-describedby="CR124">124</a>]. Coordinate translation used chain alignments from the comparative genomic analysis and the liftOver program from Kent tools UCSC [<a href="#CR125" class="usa-link" aria-describedby="CR125">125</a>] to complement the Azucena annotations with translated Nipponbare annotations.</p>
<p id="Par88">Repeat sequences were identified using a combination of a de novo consensus library generated by RepeatModeler pipeline [<a href="#CR126" class="usa-link" aria-describedby="CR126">126</a>] and a collection of transposable elements classified within the Poales order from dFAM database [<a href="#CR127" class="usa-link" aria-describedby="CR127">127</a>]. Consensus library was created using three closely related rice genomes: <em>O. sativa cv.</em> Azucena,<em> O. sativa cv.</em> Nipponbare, and <em>O. sativa subsp. Indica</em>. The final annotation of transposable element coordinates in Azucena genome was then performed by running RepeatMasker program [<a href="http://www.repeatmasker.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.repeatmasker.org</a>] with "slow search" parameter and the previously combined library.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Enrichment and randomization analyses</h3>
<p id="Par89">Enrichment between genomic features was achieved using featureBits (Kent tools: UCSC) [<a href="#CR125" class="usa-link" aria-describedby="CR125">125</a>], with genomic data regions as input. The optional enrichment parameter was included to calculate coverage and enrichment relative to an independent and identically distributed assumption.</p>
<p id="Par90">Association analysis of genomic regions was conducted using regioneR package [<a href="#CR128" class="usa-link" aria-describedby="CR128">128</a>] with overlapPermTest function to evaluate associations between feature classes and eQTLs sets with 1000 permutation tests each. Parameters included keeping random regions into the same chromosome, avoiding overlap, and restricting them to intergenic regions.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Plotting functional genomic data</h3>
<p id="Par91">Classification of dREG peak regions according to their genomic annotation was performed using Bedtools [<a href="#CR129" class="usa-link" aria-describedby="CR129">129</a>] with the intersect command, requiring at least 1 bp of overlap. The results were visualized with an Upset plot using UpsetR package [<a href="#CR130" class="usa-link" aria-describedby="CR130">130</a>]. Overlaps between peaks from different annotations are indicated by connected nodes, and the number of overlapping peaks is displayed in the bar chart above.</p>
<p id="Par92">Profile of functional genomic reads around peaks of interest was generated using the program computeMatrix from DeepTools [<a href="#CR112" class="usa-link" aria-describedby="CR112">112</a>] with scale-regions mode selected. Graphs were then plot with plotProfile function.</p>
<p id="Par93">For each genomic annotated region, signals between chromatin marks were summed and normalized by site length. Correlation plot matrix was then calculated using the Pearson correlation method.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Comparative genomics</h3>
<p id="Par94">The Lastz/PHAST approach previously described [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] for identifying conserved noncoding sequences under selection based on a 14-way alignment of species in the grass tribe Oryzeae was used with a minor change—the <em>Leersia perrieri</em> genome that aligned only weakly to the japonica reference was substituted by the recent high-quality genome, <em>Oryza granulata</em> (GG genome type) [<a href="#CR131" class="usa-link" aria-describedby="CR131">131</a>]. This helped to improve multiple alignment completeness in more distal genomes and consequently the updated analysis generated 95,670 CNS sequences, about 4% more than previously detected. Applying the same PhastCons constraint threshold as used to define CNSs (≥ 0.82) identified a total of 24,290,806 constrained bases in the Azucena reference genome. This is highly consistent with what was reported by Haudry et al. [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>] for <em>Arabidopsis lyrata</em>, where approximately 22 million bases were under selective constraint, with ~ 89% located in CDSs and ~ 11% in noncoding regions. Furthermore, Haudry et al. noted that <em>A. lyrata</em> exhibited a lower proportion of constrained noncoding bases compared to <em>A. thaliana</em>, potentially due to challenges associated with aligning TE-rich genomic regions. This trend, where larger, more repetitive genomes show a reduced fraction of identifiable constrained noncoding elements, is also observed in our Azucena dataset.</p></section></section><section id="Sec26"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3715_MOESM1_ESM.pdf</a><sup> (12.8MB, pdf) </sup><p>Additional file 1: Fig S1. Circos plot for synteny links and gene density. Fig S2. Synmap dotplot about syntenic links. Fig S3. Distribution CNS. Fig S4. Correlation between PRO-seq replicates. Fig S5. PRO-seq signal within and around annotated regions. Fig S6. dREG characteristics. Fig S8. Overlap analysis between dREG locations, ATAC peaks and CNSs. Fig S9. Percentage coverage of CNS, ATAC-seq and dREG peaks. Fig S10. Gene ontology analysis for annotated rice genes with top 1,000 CNS content. Fig S11. Gene ontology analysis for annotated rice genes with top 1,000 ATAC content. Fig S12. Gene ontology analysis for annotated rice genes with top 709 dREG content. Fig S13. <em>De novo</em> motif analysis for the top enriched 6-8 bp motifs found in CNSs. Fig S14.<em> De novo</em> motif analysis for the top enriched 6-8 bp motifs found in dREG locations and ATAC peaks. Fig S15. CHH methylation within and around of dREG sites. Fig S16. H3K27me3 signal profile within and around CNS based on their locations. Fig S17. Methylation profile through the selection of dREG candidates. Fig S18. Epigenetic marks for coding and repetitive sequences in the rice genome. Fig S19. Percentage methylation by context within and around intergenic dREG locations and their percentage of overlap with TEs. Fig S20. Correlation of the percentage of cytosine methylation in all three contexts. Fig S21. sRNA and ATAC enrichment analysis in dREG sites by bin score. Fig S22. Enrichment analysis for dREG sites with TE overlap. Fig S23. Chromatin marks within and around dREG post-filtering. Fig S24. Gene expression analysis based on the proximity of dREG sites. Fig S25. Epigenetic marks surrounding TAD boundaries. Fig S26. Distribution of loops detected by Pore-C for dREG<sub>distal</sub> peaks. Fig S27. Expression levels analysis of genes present in genic loops with a contact of a noncoding region. Fig S28. Association analysis between eQTLs regions and dREG<sub>postfilter</sub> sites. Fig S29. Association analysis between eQTLs regions and ATAC intergenic regions. Fig S30. Association analysis between eQTLs regions and CNS intergenic regions. Fig S31. Genome Browser view of an Azucena locus (GAPDH). Fig S32. Genome Browser view of the Azucena locus (WRKY5). Fig S33. Genome Browser view of an Azucena intergenic region. Fig S34. Genome Browser view of an Azucena intergenic region with three types of regulatory elements. Fig S35. Genome Browser view of the Azucena genome #1. Fig S36. Genome Browser view of the Azucena genome #2. Fig S37. Genome Browser view of the Azucena genome #3. Fig S38. Genome Browser view of the Azucena genome #4. Fig S39. Genome Browser view of the Azucena genome #5.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3715_MOESM2_ESM.xlsx</a><sup> (2.9MB, xlsx) </sup><p>Additional file 2. CNS genomic coordinates.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3715_MOESM3_ESM.pdf</a><sup> (98.1KB, pdf) </sup><p>Additional file 3: Table S1. PRO-seq summary statistics. Table S2. ATAC-seq summary statistics. Table S3. ChIP-seq summary statistics. Table S4. Transposable elements percentage overlap by annotation Summary. Table S5. sRNA-seq summary statistics. Table S6: Pore-C summary statistics.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3715_MOESM4_ESM.xlsx</a><sup> (88.2KB, xlsx) </sup><p>Additional file 4: Gene Ontology analysis CNS.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3715_MOESM5_ESM.xlsx</a><sup> (64.6KB, xlsx) </sup><p>Additional file 5: Gene Ontology analysis ATAC.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3715_MOESM6_ESM.xlsx</a><sup> (20.2KB, xlsx) </sup><p>Additional file 6: Gene Ontology analysis dREG.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3715_MOESM7_ESM.xlsx</a><sup> (790.1KB, xlsx) </sup><p>Additional file 7: small RNA coordinates.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3715_MOESM8_ESM.xlsx</a><sup> (1.5MB, xlsx) </sup><p>Additional file 8: Total and postfilter dREG genomic coordinates.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM9_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Additional file 9: Review history.</a><sup> (47.6KB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We would like to thank David Xiaoguang Dai, Priyesh Rughani, Scott Hickey, Eoghan Harrington, and Sissel Juul from Oxford Nanopore Technologies for their assistance on the Pore-C data generation. We thank the New York University Center for Genomics and Systems Biology GenCore Facility for sequencing support. We also thank the CERMO-FC for sequencing support. This research was enabled in part by support provided by Calcul Québec (<a href="https://www.calculquebec.ca" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.calculquebec.ca</a>) and the Digital Research Alliance of Canada (alliancecan.ca).</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>dREG</dt>
<dd><p id="Par14">Detection of Regulatory DNA Sequences using GRO/PRO-seq data</p></dd>
<dt>CNS</dt>
<dd><p id="Par15">Conserved Noncoding Sequences</p></dd>
<dt>CRE</dt>
<dd><p id="Par16"><em>Cis</em>-Regulatory Elements</p></dd>
<dt>ATAC-seq</dt>
<dd><p id="Par17">Assay for Transposase-Accessible Chromatin sequencing</p></dd>
<dt>PRO-seq</dt>
<dd><p id="Par18">Precision Run-On sequencing</p></dd>
<dt>IID</dt>
<dd><p id="Par19">Identically and Independently distributed</p></dd>
<dt>EQTL</dt>
<dd><p id="Par20">Expression Quantitative Trait Loci</p></dd>
<dt>FDR</dt>
<dd><p id="Par21">False Discovery Rate</p></dd>
<dt>TFBS</dt>
<dd><p id="Par22">Transcription Factor Binding Sites</p></dd>
<dt>TSS</dt>
<dd><p id="Par23">Transcription Start Site</p></dd>
<dt>GO</dt>
<dd><p id="Par24">Gene Ontology</p></dd>
<dt>Pol</dt>
<dd><p id="Par25">Polymerase</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>JYC, ZJL, MDP, and AEP designed the idea and participated in discussion; MG, AEP, JYC, AJ, FOK, and ZJL did the data collection; MG, AEP, JYC, and ZJL executed data analysis; AJ and PPE helped in data analysis; ZJL, AEP, and JYC drafted the manuscript with significant participation of PPE and MDP. MG, AJ, AEP, PPE, and ZJL actively participated in the revision process, with participation of JYC and MDP. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This work was supported by a grant from the Zegar Family Foundation (no. A16-0051–004), as well as some support from the National Science Foundation Plant Genome Research Program (no. IOS-1546218) and NYU Abu Dhabi Research Institute to MDP. This work was also supported by a grant from the Natural Sciences and Engineering Research Council of Canada (NSERC) (no. RGPIN-2021–03302) and the Canada Research Chairs (no. CRC-2021–00126) to ZJL. AEP and PPE were supported by National Science Foundation (2029959 &amp; 1929982) and Department of Energy (DE-SC0022987). JYC was supported by a National Institute of General Medical Sciences of the National Institutes of Health (no. R35GM154595).</p>
<p>Zegar Family Foundation, National Science Foundation, 2029959, Patrick P Edger, 1929982, Patrick P Edger, Natural Sciences and Engineering Research Council of Canada, RGPIN-2021-03302, Zoé Joly-Lopez, U.S. Department of Energy, SC0022987, Patrick P Edger, Canada Research Chairs, CRC-2021-00126, Zoé Joly-Lopez, National Institute of General Medical Sciences, R35GM154595, Jae Young Choi.
</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All raw sequencing data used in this study have been deposited in the Sequence Read Archive (SRA) <a href="https://www.ncbi.nlm.nih.gov/sra/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/sra/</a> with BioProject accession numbers PRJNA586887 [<a href="#CR132" class="usa-link" aria-describedby="CR132">132</a>] and PRJNA714935 [<a href="#CR133" class="usa-link" aria-describedby="CR133">133</a>]. This includes RNA-seq, ChIP-seq, BS-seq, and PRO-seq that were generated in this study was uploaded under PRJNA586887 specifically with the identifiers: ChIP-seq (H3K27me3) SRR19666802, SRR19666803, SRR19666804, SRR19666805, SRR19666806; ChIP-seq (H3K4Me3) SRR19666797, SRR19666798, SRR19666799, SR19666800, SRR19666801; ChIP-seq (H3K27ac) SRR19666809, SRR19666810, SRR19666811, SRR19666812, SRR19666813, SRR19666814; ChIP-seq (H3K18ac) SRR19666795, SRR19666796, SRR19666815, SRR19666816, SRR19666817; RNA-seq SRR19666818, SRR19666819; BS-seq SRR19666808; PRO-seq SRR19666807. The ATAC-seq and two replicates of sRNA-seq were uploaded under PRJNA714935 with the identifiers ATAC-seq SRR34572250, SRR34572251, SRR34572249; sRNA-seq SRR34572248, SRR34572247. For Pore-C, the FAST5 files are available under accession numbers SRR13985185, SRR13985186, SRR13985187, SRR13985192, and SRR13985193, and the corresponding FASTQ files are available under accession numbers SRR13985180, SRR13985181, SRR13985182, SRR13985190, and SRR13985191.</p>
<p>All custom scripts and analysis pipelines used in this study are publicly available on GitLab [<a href="#CR134" class="usa-link" aria-describedby="CR134">134</a>] at <a href="https://gitlab.com/Aniram_GS/cns_dreg" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gitlab.com/Aniram_GS/cns_dreg</a>. Processed data files and additional materials are included as supplementary files.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par95">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par96">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par97">The authors declare that they have no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher's Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Michael D. Purugganan, Email: mp132@nyu.edu</p>
<p>Zoé Joly-Lopez, Email: joly-lopez.zoe@uqam.ca.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Marand AP, Eveland AL, Kaufmann K, Springer NM. <em>cis</em>-regulatory elements in plant development, adaptation, and evolution. Annu Rev Plant Biol. 2023. 10.1146/annurev-arplant-070122-030236.
</cite> [<a href="https://doi.org/10.1146/annurev-arplant-070122-030236" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9881396/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36608347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marand%20AP,%20Eveland%20AL,%20Kaufmann%20K,%20Springer%20NM.%20cis-regulatory%20elements%20in%20plant%20development,%20adaptation,%20and%20evolution.%20Annu%20Rev%20Plant%20Biol.%202023.%2010.1146/annurev-arplant-070122-030236." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Wittkopp PJ, Kalay G. Cis-regulatory elements: molecular mechanisms and evolutionary processes underlying divergence. Nat Rev Genet. 2012;13:59–69.</cite> [<a href="https://doi.org/10.1038/nrg3095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22143240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wittkopp%20PJ,%20Kalay%20G.%20Cis-regulatory%20elements:%20molecular%20mechanisms%20and%20evolutionary%20processes%20underlying%20divergence.%20Nat%20Rev%20Genet.%202012;13:59%E2%80%9369." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite> Shlyueva D, Stampfel G, Stark A. Transcriptional enhancers: from properties to genome-wide predictions. Nat Rev Genet [Internet]. 2014;15:272–86. Available from: <a href="http://www.nature.com/articles/nrg3682" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.nature.com/articles/nrg3682</a>.</cite> [<a href="https://doi.org/10.1038/nrg3682" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24614317/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Weber B, Zicola J, Oka R, Stam M. Plant enhancers: a call for discovery. Trends Plant Sci. 2016;21:974–87.
</cite> [<a href="https://doi.org/10.1016/j.tplants.2016.07.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27593567/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weber%20B,%20Zicola%20J,%20Oka%20R,%20Stam%20M.%20Plant%20enhancers:%20a%20call%20for%20discovery.%20Trends%20Plant%20Sci.%202016;21:974%E2%80%9387." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Joly-Lopez Z, Platts AE, Gulko B, Choi JY, Groen SC, Zhong X, et al. An inferred fitness consequence map of the rice genome. Nat Plants. 2020;6:119–30.
</cite> [<a href="https://doi.org/10.1038/s41477-019-0589-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7446671/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32042156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Joly-Lopez%20Z,%20Platts%20AE,%20Gulko%20B,%20Choi%20JY,%20Groen%20SC,%20Zhong%20X,%20et%20al.%20An%20inferred%20fitness%20consequence%20map%20of%20the%20rice%20genome.%20Nat%20Plants.%202020;6:119%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Ricci WA, Lu Z, Ji L, Marand AP, Ethridge CL, Murphy NG, et al. Widespread long-range cis-regulatory elements in the maize genome. Nat Plants. 2019;5:1237–49.
</cite> [<a href="https://doi.org/10.1038/s41477-019-0547-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6904520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31740773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ricci%20WA,%20Lu%20Z,%20Ji%20L,%20Marand%20AP,%20Ethridge%20CL,%20Murphy%20NG,%20et%20al.%20Widespread%20long-range%20cis-regulatory%20elements%20in%20the%20maize%20genome.%20Nat%20Plants.%202019;5:1237%E2%80%9349." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Marand AP, Zhang T, Zhu B, Jiang J. Towards genome-wide prediction and characterization of enhancers in plants. Biochimica et Biophysica Acta (BBA). 2017;1860:131–9.</cite> [<a href="https://doi.org/10.1016/j.bbagrm.2016.06.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27321818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marand%20AP,%20Zhang%20T,%20Zhu%20B,%20Jiang%20J.%20Towards%20genome-wide%20prediction%20and%20characterization%20of%20enhancers%20in%20plants.%20Biochimica%20et%20Biophysica%20Acta%20(BBA).%202017;1860:131%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Oka R, Zicola J, Weber B, Anderson SN, Hodgman C, Gent JI, et al. Genome-wide mapping of transcriptional enhancer candidates using DNA and chromatin features in maize. Genome Biol. 2017;18: 137.
</cite> [<a href="https://doi.org/10.1186/s13059-017-1273-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5522596/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28732548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oka%20R,%20Zicola%20J,%20Weber%20B,%20Anderson%20SN,%20Hodgman%20C,%20Gent%20JI,%20et%20al.%20Genome-wide%20mapping%20of%20transcriptional%20enhancer%20candidates%20using%20DNA%20and%20chromatin%20features%20in%20maize.%20Genome%20Biol.%202017;18:%20137." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Andersson R, Sandelin A. Determinants of enhancer and promoter activities of regulatory elements. Nat Rev Genet. 2020;21:71–87. 10.1038/s41576-019-0173-8.
</cite> [<a href="https://doi.org/10.1038/s41576-019-0173-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31605096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andersson%20R,%20Sandelin%20A.%20Determinants%20of%20enhancer%20and%20promoter%20activities%20of%20regulatory%20elements.%20Nat%20Rev%20Genet.%202020;21:71%E2%80%9387.%2010.1038/s41576-019-0173-8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite> Pajoro A, Madrigal P, Muiño JM, Matus JT, Jin J, Mecchia MA, et al. Dynamics of chromatin accessibility and gene regulation by MADS-domain transcription factors in flower development. 2014;15:R41. Available from: 10.1186/gb-2014-15-3-r41.</cite> [<a href="https://doi.org/10.1186/gb-2014-15-3-r41" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4054849/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24581456/" class="usa-link">PubMed</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Sun J, He N, Niu L, Huang Y, Shen W, Zhang Y, et al. Global quantitative mapping of enhancers in rice by STARR-seq. Genom, Proteom Bioinform. 2019;17:140–53.</cite> [<a href="https://doi.org/10.1016/j.gpb.2018.11.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6624190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31201999/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun%20J,%20He%20N,%20Niu%20L,%20Huang%20Y,%20Shen%20W,%20Zhang%20Y,%20et%20al.%20Global%20quantitative%20mapping%20of%20enhancers%20in%20rice%20by%20STARR-seq.%20Genom,%20Proteom%20Bioinform.%202019;17:140%E2%80%9353." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Jores T, Tonnies J, Dorrity MW, Cuperus JT, Fields S, Queitsch C. Identification of plant enhancers and their constituent elements by STARR-seq in tobacco leaves. Plant Cell. 2020;32:2120–31.
</cite> [<a href="https://doi.org/10.1105/tpc.20.00155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7346570/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32409318/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jores%20T,%20Tonnies%20J,%20Dorrity%20MW,%20Cuperus%20JT,%20Fields%20S,%20Queitsch%20C.%20Identification%20of%20plant%20enhancers%20and%20their%20constituent%20elements%20by%20STARR-seq%20in%20tobacco%20leaves.%20Plant%20Cell.%202020;32:2120%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Oka R, Zicola J, Weber B, Anderson SN, Hodgman C, Gent JI, et al. Genome-wide mapping of transcriptional enhancer candidates using DNA and chromatin features in maize. Genome Biol. 2017;18:137.
</cite> [<a href="https://doi.org/10.1186/s13059-017-1273-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5522596/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28732548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Oka%20R,%20Zicola%20J,%20Weber%20B,%20Anderson%20SN,%20Hodgman%20C,%20Gent%20JI,%20et%20al.%20Genome-wide%20mapping%20of%20transcriptional%20enhancer%20candidates%20using%20DNA%20and%20chromatin%20features%20in%20maize.%20Genome%20Biol.%202017;18:137." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Zilberman D, Coleman-Derr D, Ballinger T, Henikoff S. Histone H2A.Z and DNA methylation are mutually antagonistic chromatin marks. Nature. 2008;456:125–9.
</cite> [<a href="https://doi.org/10.1038/nature07324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2877514/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18815594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zilberman%20D,%20Coleman-Derr%20D,%20Ballinger%20T,%20Henikoff%20S.%20Histone%20H2A.Z%20and%20DNA%20methylation%20are%20mutually%20antagonistic%20chromatin%20marks.%20Nature.%202008;456:125%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Nuthikattu S, McCue AD, Panda K, Fultz D, DeFraia C, Thomas EN, et al. The initiation of epigenetic silencing of active transposable elements is triggered by RDR6 and 21–22 nucleotide small interfering RNAs. Plant Physiol. 2013;162:116–31.
</cite> [<a href="https://doi.org/10.1104/pp.113.216481" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3641197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23542151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nuthikattu%20S,%20McCue%20AD,%20Panda%20K,%20Fultz%20D,%20DeFraia%20C,%20Thomas%20EN,%20et%20al.%20The%20initiation%20of%20epigenetic%20silencing%20of%20active%20transposable%20elements%20is%20triggered%20by%20RDR6%20and%2021%E2%80%9322%20nucleotide%20small%20interfering%20RNAs.%20Plant%20Physiol.%202013;162:116%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Gent JI, Ellis NA, Guo L, Harkess AE, Yao Y, Zhang X, et al. CHH islands: de novo DNA methylation in near-gene chromatin regulation in maize. Genome Res. 2013;23:628–37.
</cite> [<a href="https://doi.org/10.1101/gr.146985.112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3613580/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23269663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gent%20JI,%20Ellis%20NA,%20Guo%20L,%20Harkess%20AE,%20Yao%20Y,%20Zhang%20X,%20et%20al.%20CHH%20islands:%20de%20novo%20DNA%20methylation%20in%20near-gene%20chromatin%20regulation%20in%20maize.%20Genome%20Res.%202013;23:628%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Niederhuth CE, Bewick AJ, Ji L, Alabady MS, Kim KD, Li Q, et al. Widespread natural variation of DNA methylation within angiosperms. Genome Biol. 2016;17: 194.
</cite> [<a href="https://doi.org/10.1186/s13059-016-1059-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5037628/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27671052/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Niederhuth%20CE,%20Bewick%20AJ,%20Ji%20L,%20Alabady%20MS,%20Kim%20KD,%20Li%20Q,%20et%20al.%20Widespread%20natural%20variation%20of%20DNA%20methylation%20within%20angiosperms.%20Genome%20Biol.%202016;17:%20194." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Fenley AT, Anandakrishnan R, Kidane YH, Onufriev AV. Modulation of nucleosomal DNA accessibility via charge-altering post-translational modifications in histone core. Epigenetics Chromatin. 2018;11: 11.
</cite> [<a href="https://doi.org/10.1186/s13072-018-0181-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5856334/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29548294/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fenley%20AT,%20Anandakrishnan%20R,%20Kidane%20YH,%20Onufriev%20AV.%20Modulation%20of%20nucleosomal%20DNA%20accessibility%20via%20charge-altering%20post-translational%20modifications%20in%20histone%20core.%20Epigenetics%20Chromatin.%202018;11:%2011." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Zhang Y, Tang M, Huang M, Xie J, Cheng J, Fu Y, et al. Dynamic enhancer transcription associates with reprogramming of immune genes during pattern triggered immunity in Arabidopsis. BMC Biol. 2022;20: 165.
</cite> [<a href="https://doi.org/10.1186/s12915-022-01362-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9301868/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35864475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Y,%20Tang%20M,%20Huang%20M,%20Xie%20J,%20Cheng%20J,%20Fu%20Y,%20et%20al.%20Dynamic%20enhancer%20transcription%20associates%20with%20reprogramming%20of%20immune%20genes%20during%20pattern%20triggered%20immunity%20in%20Arabidopsis.%20BMC%20Biol.%202022;20:%20165." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Rada-Iglesias A, Bajpai R, Swigut T, Brugmann SA, Flynn RA, Wysocka J. A unique chromatin signature uncovers early developmental enhancers in humans. Nature. 2011;470:279–83.
</cite> [<a href="https://doi.org/10.1038/nature09692" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4445674/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21160473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rada-Iglesias%20A,%20Bajpai%20R,%20Swigut%20T,%20Brugmann%20SA,%20Flynn%20RA,%20Wysocka%20J.%20A%20unique%20chromatin%20signature%20uncovers%20early%20developmental%20enhancers%20in%20humans.%20Nature.%202011;470:279%E2%80%9383." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Ernst J, Kellis M. Chromhmm: automating chromatin-state discovery and characterization. Nat Methods. 2012;9:215–6.
</cite> [<a href="https://doi.org/10.1038/nmeth.1906" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3577932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22373907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ernst%20J,%20Kellis%20M.%20Chromhmm:%20automating%20chromatin-state%20discovery%20and%20characterization.%20Nat%20Methods.%202012;9:215%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Lu Z, Marand AP, Ricci WA, Ethridge CL, Zhang X, Schmitz RJ. The prevalence, evolution and chromatin signatures of plant regulatory elements. Nat Plants. 2019;5:1250–9.
</cite> [<a href="https://doi.org/10.1038/s41477-019-0548-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31740772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu%20Z,%20Marand%20AP,%20Ricci%20WA,%20Ethridge%20CL,%20Zhang%20X,%20Schmitz%20RJ.%20The%20prevalence,%20evolution%20and%20chromatin%20signatures%20of%20plant%20regulatory%20elements.%20Nat%20Plants.%202019;5:1250%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Huang D, Petrykowska HM, Miller BF, Elnitski L, Ovcharenko I. Identification of human silencers by correlating cross-tissue epigenetic profiles and gene expression. Genome Res. 2019;29:657–67.
</cite> [<a href="https://doi.org/10.1101/gr.247007.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6442386/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30886051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20D,%20Petrykowska%20HM,%20Miller%20BF,%20Elnitski%20L,%20Ovcharenko%20I.%20Identification%20of%20human%20silencers%20by%20correlating%20cross-tissue%20epigenetic%20profiles%20and%20gene%20expression.%20Genome%20Res.%202019;29:657%E2%80%9367." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Yocca AE, Lu Z, Schmitz RJ, Freeling M, Edger PP. Evolution of conserved noncoding sequences in <em>Arabidopsis thaliana</em>. Mol Biol Evol. 2021;38:2692–703.
</cite> [<a href="https://doi.org/10.1093/molbev/msab042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8233505/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33565589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yocca%20AE,%20Lu%20Z,%20Schmitz%20RJ,%20Freeling%20M,%20Edger%20PP.%20Evolution%20of%20conserved%20noncoding%20sequences%20in%20Arabidopsis%20thaliana.%20Mol%20Biol%20Evol.%202021;38:2692%E2%80%93703." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Joly-Lopez Z, Platts AE, Gulko B, Choi JY, Groen SC, Zhong X, et al. An inferred fitness consequence map of the rice genome. Nat Plants. 2020;6:119–30.
</cite> [<a href="https://doi.org/10.1038/s41477-019-0589-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7446671/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32042156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Joly-Lopez%20Z,%20Platts%20AE,%20Gulko%20B,%20Choi%20JY,%20Groen%20SC,%20Zhong%20X,%20et%20al.%20An%20inferred%20fitness%20consequence%20map%20of%20the%20rice%20genome.%20Nat%20Plants.%202020;6:119%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Field A, Adelman K. Evaluating enhancer function and transcription. Annu Rev Biochem. 2020. 10.1146/annurev-biochem-011420-095916.
</cite> [<a href="https://doi.org/10.1146/annurev-biochem-011420-095916" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32197056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Field%20A,%20Adelman%20K.%20Evaluating%20enhancer%20function%20and%20transcription.%20Annu%20Rev%20Biochem.%202020.%2010.1146/annurev-biochem-011420-095916." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Sartorelli V, Lauberth SM. Enhancer RNAs are an important regulatory layer of the epigenome. Nat Struct Mol Biol. 2020;27:521–8.
</cite> [<a href="https://doi.org/10.1038/s41594-020-0446-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7343394/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32514177/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sartorelli%20V,%20Lauberth%20SM.%20Enhancer%20RNAs%20are%20an%20important%20regulatory%20layer%20of%20the%20epigenome.%20Nat%20Struct%20Mol%20Biol.%202020;27:521%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Consortium TF, Andersson R, Gebhard C, Miguel-Escalada I, Hoof I, Bornholdt J, et al. An atlas of active enhancers across human cell types and tissues. Nature. 2014;507:455–61.
</cite> [<a href="https://doi.org/10.1038/nature12787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5215096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24670763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Consortium%20TF,%20Andersson%20R,%20Gebhard%20C,%20Miguel-Escalada%20I,%20Hoof%20I,%20Bornholdt%20J,%20et%20al.%20An%20atlas%20of%20active%20enhancers%20across%20human%20cell%20types%20and%20tissues.%20Nature.%202014;507:455%E2%80%9361." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Core LJ, Waterfall JJ, Gilchrist DA, Fargo DC, Kwak H, Adelman K, et al. Defining the status of RNA polymerase at promoters. Cell Rep. 2012;2:1025–35.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2012.08.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3483431/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23062713/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Core%20LJ,%20Waterfall%20JJ,%20Gilchrist%20DA,%20Fargo%20DC,%20Kwak%20H,%20Adelman%20K,%20et%20al.%20Defining%20the%20status%20of%20RNA%20polymerase%20at%20promoters.%20Cell%20Rep.%202012;2:1025%E2%80%9335." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Mousavi K, Zare H, Dell’Orso S, Grontved L, Gutierrez-Cruz G, Derfoul A, et al. eRNAs promote transcription by establishing chromatin accessibility at defined genomic loci. Mol Cell. 2013;51:606–17.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2013.07.022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3786356/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23993744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mousavi%20K,%20Zare%20H,%20Dell%E2%80%99Orso%20S,%20Grontved%20L,%20Gutierrez-Cruz%20G,%20Derfoul%20A,%20et%20al.%20eRNAs%20promote%20transcription%20by%20establishing%20chromatin%20accessibility%20at%20defined%20genomic%20loci.%20Mol%20Cell.%202013;51:606%E2%80%9317." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Kaikkonen MU, Spann NJ, Heinz S, Romanoski CE, Allison KA, Stender JD, et al. Remodeling of the enhancer landscape during macrophage activation is coupled to enhancer transcription. Mol Cell. 2013;51:310–25.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2013.07.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3779836/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23932714/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaikkonen%20MU,%20Spann%20NJ,%20Heinz%20S,%20Romanoski%20CE,%20Allison%20KA,%20Stender%20JD,%20et%20al.%20Remodeling%20of%20the%20enhancer%20landscape%20during%20macrophage%20activation%20is%20coupled%20to%20enhancer%20transcription.%20Mol%20Cell.%202013;51:310%E2%80%9325." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Sigova AA, Abraham BJ, Ji X, Molinie B, Hannett NM, Guo YE, et al. Transcription factor trapping by RNA in gene regulatory elements. Science. 2015;350:978–81.
</cite> [<a href="https://doi.org/10.1126/science.aad3346" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4720525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26516199/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sigova%20AA,%20Abraham%20BJ,%20Ji%20X,%20Molinie%20B,%20Hannett%20NM,%20Guo%20YE,%20et%20al.%20Transcription%20factor%20trapping%20by%20RNA%20in%20gene%20regulatory%20elements.%20Science.%202015;350:978%E2%80%9381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Rahnamoun H, Lee J, Sun Z, Lu H, Ramsey KM, Komives EA, et al. RNAs interact with BRD4 to promote enhanced chromatin engagement and transcription activation. Nat Struct Mol Biol. 2018;25:687–97.
</cite> [<a href="https://doi.org/10.1038/s41594-018-0102-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6859054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30076409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rahnamoun%20H,%20Lee%20J,%20Sun%20Z,%20Lu%20H,%20Ramsey%20KM,%20Komives%20EA,%20et%20al.%20RNAs%20interact%20with%20BRD4%20to%20promote%20enhanced%20chromatin%20engagement%20and%20transcription%20activation.%20Nat%20Struct%20Mol%20Biol.%202018;25:687%E2%80%9397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Zhao ZW, Roy R, Gebhardt JCM, Suter DM, Chapman AR, Xie XS. Spatial organization of RNA polymerase II inside a mammalian cell nucleus revealed by reflected light-sheet superresolution microscopy. Proc Natl Acad Sci. 2014;111:681–6.
</cite> [<a href="https://doi.org/10.1073/pnas.1318496111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3896202/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24379392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20ZW,%20Roy%20R,%20Gebhardt%20JCM,%20Suter%20DM,%20Chapman%20AR,%20Xie%20XS.%20Spatial%20organization%20of%20RNA%20polymerase%20II%20inside%20a%20mammalian%20cell%20nucleus%20revealed%20by%20reflected%20light-sheet%20superresolution%20microscopy.%20Proc%20Natl%20Acad%20Sci.%202014;111:681%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Kwak H, Fuda NJ, Core LJ, Lis JT. Precise maps of RNA polymerase reveal how promoters direct initiation and pausing. Science. 2013;339:950–3.
</cite> [<a href="https://doi.org/10.1126/science.1229386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3974810/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23430654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kwak%20H,%20Fuda%20NJ,%20Core%20LJ,%20Lis%20JT.%20Precise%20maps%20of%20RNA%20polymerase%20reveal%20how%20promoters%20direct%20initiation%20and%20pausing.%20Science.%202013;339:950%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>McDonald BR, Picard CL, Brabb IM, Savenkova MI, Schmitz RJ, Jacobsen SE, et al. Enhancers associated with unstable RNAs are rare in plants. Nat Plants. 2024;10:1246–57.
</cite> [<a href="https://doi.org/10.1038/s41477-024-01741-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11335568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39080503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McDonald%20BR,%20Picard%20CL,%20Brabb%20IM,%20Savenkova%20MI,%20Schmitz%20RJ,%20Jacobsen%20SE,%20et%20al.%20Enhancers%20associated%20with%20unstable%20RNAs%20are%20rare%20in%20plants.%20Nat%20Plants.%202024;10:1246%E2%80%9357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Xie Y, Chen Y, Li Z, Zhu J, Liu M, Zhang Y, et al. Enhancer transcription detected in the nascent transcriptomic landscape of bread wheat. Genome Biol. 2022;23:109.
</cite> [<a href="https://doi.org/10.1186/s13059-022-02675-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9063354/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35501845/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie%20Y,%20Chen%20Y,%20Li%20Z,%20Zhu%20J,%20Liu%20M,%20Zhang%20Y,%20et%20al.%20Enhancer%20transcription%20detected%20in%20the%20nascent%20transcriptomic%20landscape%20of%20bread%20wheat.%20Genome%20Biol.%202022;23:109." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Mahat DB, Kwak H, Booth GT, Jonkers IH, Danko CG, Patel RK, et al. Base-pair-resolution genome-wide mapping of active RNA polymerases using precision nuclear run-on (PRO-seq). Nat Protoc. 2016;11:1455–76.
</cite> [<a href="https://doi.org/10.1038/nprot.2016.086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5502525/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27442863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mahat%20DB,%20Kwak%20H,%20Booth%20GT,%20Jonkers%20IH,%20Danko%20CG,%20Patel%20RK,%20et%20al.%20Base-pair-resolution%20genome-wide%20mapping%20of%20active%20RNA%20polymerases%20using%20precision%20nuclear%20run-on%20(PRO-seq).%20Nat%20Protoc.%202016;11:1455%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Hah N, Danko CG, Core L, Waterfall JJ, Siepel A, Lis JT, et al. A rapid, extensive, and transient transcriptional response to estrogen signaling in breast cancer cells. Cell. 2011;145:622–34.
</cite> [<a href="https://doi.org/10.1016/j.cell.2011.03.042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3099127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21549415/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hah%20N,%20Danko%20CG,%20Core%20L,%20Waterfall%20JJ,%20Siepel%20A,%20Lis%20JT,%20et%20al.%20A%20rapid,%20extensive,%20and%20transient%20transcriptional%20response%20to%20estrogen%20signaling%20in%20breast%20cancer%20cells.%20Cell.%202011;145:622%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Beernink BM, Vogel JP, Lei L. Enhancers in plant development, adaptation and evolution. Plant Cell Physiol. 2024. 10.1093/pcp/pcae121.</cite> [<a href="https://doi.org/10.1093/pcp/pcae121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12085095/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39412125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Beernink%20BM,%20Vogel%20JP,%20Lei%20L.%20Enhancers%20in%20plant%20development,%20adaptation%20and%20evolution.%20Plant%20Cell%20Physiol.%202024.%2010.1093/pcp/pcae121." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Villar D, Berthelot C, Aldridge S, Rayner TF, Lukk M, Pignatelli M, et al. Enhancer evolution across 20 mammalian species. Cell. 2015;160:554–66.
</cite> [<a href="https://doi.org/10.1016/j.cell.2015.01.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4313353/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25635462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Villar%20D,%20Berthelot%20C,%20Aldridge%20S,%20Rayner%20TF,%20Lukk%20M,%20Pignatelli%20M,%20et%20al.%20Enhancer%20evolution%20across%2020%20mammalian%20species.%20Cell.%202015;160:554%E2%80%9366." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Zheng XM, Chen J, Pang HB, Liu S, Gao Q, Wang JR, et al. Genome-wide analyses reveal the role of noncoding variation in complex traits during rice domestication. Sci Adv. 2019;5: eaax3619.
</cite> [<a href="https://doi.org/10.1126/sciadv.aax3619" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6989341/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32064312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng%20XM,%20Chen%20J,%20Pang%20HB,%20Liu%20S,%20Gao%20Q,%20Wang%20JR,%20et%20al.%20Genome-wide%20analyses%20reveal%20the%20role%20of%20noncoding%20variation%20in%20complex%20traits%20during%20rice%20domestication.%20Sci%20Adv.%202019;5:%20eaax3619." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>de Ocampo MP, Ho VT, Thomson MJ, Mitsuya S, Yamauchi A, Ismail AM. QTL mapping under salt stress in rice using a Kalarata-Azucena population. Euphytica. 2022;218:74.
</cite> [<a href="https://doi.org/10.1007/s10681-022-03026-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9427886/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36060537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?de%20Ocampo%20MP,%20Ho%20VT,%20Thomson%20MJ,%20Mitsuya%20S,%20Yamauchi%20A,%20Ismail%20AM.%20QTL%20mapping%20under%20salt%20stress%20in%20rice%20using%20a%20Kalarata-Azucena%20population.%20Euphytica.%202022;218:74." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite> Lozano R, Booth GT, Omar BY, Li B, Buckler ES, Lis JT, et al. RNA polymerase mapping in plants identifies intergenic regulatory elements enriched in causal variants. G3. 2021;11:jkab273.</cite> [<a href="https://doi.org/10.1093/g3journal/jkab273" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8527479/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34499719/" class="usa-link">PubMed</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Hetzel J, Duttke SH, Benner C, Chory J. Nascent RNA sequencing reveals distinct features in plant transcription. Proc Natl Acad Sci. 2016;113:12316–21.
</cite> [<a href="https://doi.org/10.1073/pnas.1603217113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5087027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27729530/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hetzel%20J,%20Duttke%20SH,%20Benner%20C,%20Chory%20J.%20Nascent%20RNA%20sequencing%20reveals%20distinct%20features%20in%20plant%20transcription.%20Proc%20Natl%20Acad%20Sci.%202016;113:12316%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Lu Z, Hofmeister BT, Vollmers C, DuBois RM, Schmitz RJ. Combining ATAC-seq with nuclei sorting for discovery of <em>cis</em>-regulatory regions in plant genomes. Nucleic Acids Res. 2017;45: e41–e41.
</cite> [<a href="https://doi.org/10.1093/nar/gkw1179" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5389718/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27903897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu%20Z,%20Hofmeister%20BT,%20Vollmers%20C,%20DuBois%20RM,%20Schmitz%20RJ.%20Combining%20ATAC-seq%20with%20nuclei%20sorting%20for%20discovery%20of%20cis-regulatory%20regions%20in%20plant%20genomes.%20Nucleic%20Acids%20Res.%202017;45:%20e41%E2%80%93e41." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Rajkumar MS, Tembhare K, Garg R, Jain M. Genome-wide mapping of DNase I hypersensitive sites revealed differential chromatin accessibility and regulatory DNA elements under drought stress in rice cultivars. Plant J. 2024;119:2063–79.
</cite> [<a href="https://doi.org/10.1111/tpj.16864" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38859561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rajkumar%20MS,%20Tembhare%20K,%20Garg%20R,%20Jain%20M.%20Genome-wide%20mapping%20of%20DNase%20I%20hypersensitive%20sites%20revealed%20differential%20chromatin%20accessibility%20and%20regulatory%20DNA%20elements%20under%20drought%20stress%20in%20rice%20cultivars.%20Plant%20J.%202024;119:2063%E2%80%9379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Marand AP, Schmitz RJ. Single-cell analysis of <em>cis</em>-regulatory elements. Curr Opin Plant Biol. 2022;65: 102094.
</cite> [<a href="https://doi.org/10.1016/j.pbi.2021.102094" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34390932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marand%20AP,%20Schmitz%20RJ.%20Single-cell%20analysis%20of%20cis-regulatory%20elements.%20Curr%20Opin%20Plant%20Biol.%202022;65:%20102094." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Haudry A, Platts AE, Vello E, Hoen DR, Leclercq M, Williamson RJ, et al. An atlas of over 90,000 conserved noncoding sequences provides insight into crucifer regulatory regions. Nat Genet. 2013;45:891–8.
</cite> [<a href="https://doi.org/10.1038/ng.2684" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23817568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haudry%20A,%20Platts%20AE,%20Vello%20E,%20Hoen%20DR,%20Leclercq%20M,%20Williamson%20RJ,%20et%20al.%20An%20atlas%20of%20over%2090,000%20conserved%20noncoding%20sequences%20provides%20insight%20into%20crucifer%20regulatory%20regions.%20Nat%20Genet.%202013;45:891%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Wing RA, Purugganan MD, Zhang Q. The rice genome revolution: from an ancient grain to green super rice. Nat Rev Genet. 2018;19:505–17.
</cite> [<a href="https://doi.org/10.1038/s41576-018-0024-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29872215/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wing%20RA,%20Purugganan%20MD,%20Zhang%20Q.%20The%20rice%20genome%20revolution:%20from%20an%20ancient%20grain%20to%20green%20super%20rice.%20Nat%20Rev%20Genet.%202018;19:505%E2%80%9317." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Santa FD, Barozzi I, Mietton F, Ghisletti S, Polletti S, Tusi BK, et al. A large fraction of extragenic RNA Pol II transcription sites overlap enhancers. PLoS Biol. 2010;8: e1000384.
</cite> [<a href="https://doi.org/10.1371/journal.pbio.1000384" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2867938/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20485488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Santa%20FD,%20Barozzi%20I,%20Mietton%20F,%20Ghisletti%20S,%20Polletti%20S,%20Tusi%20BK,%20et%20al.%20A%20large%20fraction%20of%20extragenic%20RNA%20Pol%20II%20transcription%20sites%20overlap%20enhancers.%20PLoS%20Biol.%202010;8:%20e1000384." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Kim TK, Hemberg M, Gray JM, Costa AM, Bear DM, Wu J, et al. Widespread transcription at neuronal activity-regulated enhancers. Nature. 2010;465:182–7.
</cite> [<a href="https://doi.org/10.1038/nature09033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3020079/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20393465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20TK,%20Hemberg%20M,%20Gray%20JM,%20Costa%20AM,%20Bear%20DM,%20Wu%20J,%20et%20al.%20Widespread%20transcription%20at%20neuronal%20activity-regulated%20enhancers.%20Nature.%202010;465:182%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Wang J, Zhao Y, Zhou X, Hiebert SW, Liu Q, Shyr Y. Nascent RNA sequencing analysis provides insights into enhancer-mediated gene regulation. BMC Genomics. 2018;19:1–18.
</cite> [<a href="https://doi.org/10.1186/s12864-018-5016-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6107967/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30139328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20J,%20Zhao%20Y,%20Zhou%20X,%20Hiebert%20SW,%20Liu%20Q,%20Shyr%20Y.%20Nascent%20RNA%20sequencing%20analysis%20provides%20insights%20into%20enhancer-mediated%20gene%20regulation.%20BMC%20Genomics.%202018;19:1%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Arenas-Mena C, Miljovska S, Rice EJ, Gurges J, Shashikant T, Wang Z, et al. Identification and prediction of developmental enhancers in sea urchin embryos. BMC Genomics. 2021;22:751.
</cite> [<a href="https://doi.org/10.1186/s12864-021-07936-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8527612/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34666684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arenas-Mena%20C,%20Miljovska%20S,%20Rice%20EJ,%20Gurges%20J,%20Shashikant%20T,%20Wang%20Z,%20et%20al.%20Identification%20and%20prediction%20of%20developmental%20enhancers%20in%20sea%20urchin%20embryos.%20BMC%20Genomics.%202021;22:751." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite> Chu T, Wang Z, Chou S-P, Danko CG. Discovering Transcriptional Regulatory Elements From Run-On and Sequencing Data Using the Web-Based dREG Gateway. Curr Protoc Bioinform [Internet]. 2019;66:e70. Available from: 10.1002/cpbi.70.</cite> [<a href="https://doi.org/10.1002/cpbi.70" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6584046/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30589513/" class="usa-link">PubMed</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Mikhaylichenko O, Bondarenko V, Harnett D, Schor IE, Males M, Viales RR, et al. The degree of enhancer or promoter activity is reflected by the levels and directionality of eRNA transcription. Gene Dev. 2018;32:42–57.
</cite> [<a href="https://doi.org/10.1101/gad.308619.117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5828394/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29378788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mikhaylichenko%20O,%20Bondarenko%20V,%20Harnett%20D,%20Schor%20IE,%20Males%20M,%20Viales%20RR,%20et%20al.%20The%20degree%20of%20enhancer%20or%20promoter%20activity%20is%20reflected%20by%20the%20levels%20and%20directionality%20of%20eRNA%20transcription.%20Gene%20Dev.%202018;32:42%E2%80%9357." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Vihervaara A, Mahat DB, Guertin MJ, Chu T, Danko CG, Lis JT, et al. Transcriptional response to stress is pre-wired by promoter and enhancer architecture. Nat Commun. 2017;8:255.
</cite> [<a href="https://doi.org/10.1038/s41467-017-00151-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5557961/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28811569/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vihervaara%20A,%20Mahat%20DB,%20Guertin%20MJ,%20Chu%20T,%20Danko%20CG,%20Lis%20JT,%20et%20al.%20Transcriptional%20response%20to%20stress%20is%20pre-wired%20by%20promoter%20and%20enhancer%20architecture.%20Nat%20Commun.%202017;8:255." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Danko CG, Hyland SL, Core LJ, Martins AL, Waters CT, Lee HW, et al. Identification of active transcriptional regulatory elements with GRO-seq. Nat Methods. 2015;12:433–8.
</cite> [<a href="https://doi.org/10.1038/nmeth.3329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4507281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25799441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Danko%20CG,%20Hyland%20SL,%20Core%20LJ,%20Martins%20AL,%20Waters%20CT,%20Lee%20HW,%20et%20al.%20Identification%20of%20active%20transcriptional%20regulatory%20elements%20with%20GRO-seq.%20Nat%20Methods.%202015;12:433%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Melgar MF, Collins FS, Sethupathy P. Discovery of active enhancers through bidirectional expression of short transcripts. Genome Biol. 2011;12:R113.
</cite> [<a href="https://doi.org/10.1186/gb-2011-12-11-r113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3334599/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22082242/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Melgar%20MF,%20Collins%20FS,%20Sethupathy%20P.%20Discovery%20of%20active%20enhancers%20through%20bidirectional%20expression%20of%20short%20transcripts.%20Genome%20Biol.%202011;12:R113." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Schmitz RJ, Grotewold E, Stam M. Cis-regulatory sequences in plants: their importance, discovery, and future challenges. Plant Cell. 2021;34:718–41.</cite> [<a href="https://doi.org/10.1093/plcell/koab281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8824567/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34918159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schmitz%20RJ,%20Grotewold%20E,%20Stam%20M.%20Cis-regulatory%20sequences%20in%20plants:%20their%20importance,%20discovery,%20and%20future%20challenges.%20Plant%20Cell.%202021;34:718%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Wang Z, Chu T, Choate LA, Danko CG. Identification of regulatory elements from nascent transcription using dREG. Genome Res. 2019;29:293–303.
</cite> [<a href="https://doi.org/10.1101/gr.238279.118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6360809/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30573452/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20Z,%20Chu%20T,%20Choate%20LA,%20Danko%20CG.%20Identification%20of%20regulatory%20elements%20from%20nascent%20transcription%20using%20dREG.%20Genome%20Res.%202019;29:293%E2%80%93303." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite> Klemm SL, Shipony Z, Greenleaf WJ. Chromatin accessibility and the regulatory epigenome. Nat Rev Genet [Internet]. 2019;20:207–20. Available from: <a href="http://www.nature.com/articles/s41576-018-0089-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.nature.com/articles/s41576-018-0089-8</a>.</cite> [<a href="https://doi.org/10.1038/s41576-018-0089-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30675018/" class="usa-link">PubMed</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Heinz S, Benner C, Spann N, Bertolino E, Lin YC, Laslo P, et al. Simple combinations of lineage-determining transcription factors prime <em>cis</em>-regulatory elements required for macrophage and B cell identities. Mol Cell. 2010;38:576–89.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2010.05.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2898526/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20513432/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Heinz%20S,%20Benner%20C,%20Spann%20N,%20Bertolino%20E,%20Lin%20YC,%20Laslo%20P,%20et%20al.%20Simple%20combinations%20of%20lineage-determining%20transcription%20factors%20prime%20cis-regulatory%20elements%20required%20for%20macrophage%20and%20B%20cell%20identities.%20Mol%20Cell.%202010;38:576%E2%80%9389." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Cusack SA, Wang P, Lotreck SG, Moore BM, Meng F, Conner JK, et al. Predictive models of genetic redundancy in <em>Arabidopsis thaliana</em>. Mol Biol Evol. 2021;38:3397–414.
</cite> [<a href="https://doi.org/10.1093/molbev/msab111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8321531/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33871641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cusack%20SA,%20Wang%20P,%20Lotreck%20SG,%20Moore%20BM,%20Meng%20F,%20Conner%20JK,%20et%20al.%20Predictive%20models%20of%20genetic%20redundancy%20in%20Arabidopsis%20thaliana.%20Mol%20Biol%20Evol.%202021;38:3397%E2%80%93414." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom K, et al. Evolutionarily conserved elements in vertebrate, insect, worm, and yeast genomes. Genome Res. 2005;15:1034–50.
</cite> [<a href="https://doi.org/10.1101/gr.3715005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1182216/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16024819/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Siepel%20A,%20Bejerano%20G,%20Pedersen%20JS,%20Hinrichs%20AS,%20Hou%20M,%20Rosenbloom%20K,%20et%20al.%20Evolutionarily%20conserved%20elements%20in%20vertebrate,%20insect,%20worm,%20and%20yeast%20genomes.%20Genome%20Res.%202005;15:1034%E2%80%9350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Reynoso MA, Kajala K, Bajic M, West DA, Pauluzzi G, Yao AI, et al. Evolutionary flexibility in flooding response circuitry in angiosperms. Science. 2019;365:1291–5.
</cite> [<a href="https://doi.org/10.1126/science.aax8862" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7710369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31604238/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reynoso%20MA,%20Kajala%20K,%20Bajic%20M,%20West%20DA,%20Pauluzzi%20G,%20Yao%20AI,%20et%20al.%20Evolutionary%20flexibility%20in%20flooding%20response%20circuitry%20in%20angiosperms.%20Science.%202019;365:1291%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Lu Z, Marand AP, Ricci WA, Ethridge CL, Zhang X, Schmitz RJ. The prevalence, evolution and chromatin signatures of plant regulatory elements. Nat Plants. 2019;5:1250–9. 10.1038/s41477-019-0548-z.
</cite> [<a href="https://doi.org/10.1038/s41477-019-0548-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31740772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu%20Z,%20Marand%20AP,%20Ricci%20WA,%20Ethridge%20CL,%20Zhang%20X,%20Schmitz%20RJ.%20The%20prevalence,%20evolution%20and%20chromatin%20signatures%20of%20plant%20regulatory%20elements.%20Nat%20Plants.%202019;5:1250%E2%80%939.%2010.1038/s41477-019-0548-z." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite> Carelli FN, Liechti A, Halbert J, Warnefors M, Kaessmann H. Repurposing of promoters and enhancers during mammalian evolution. 2018;9:4066. Available from: 10.1038/s41467-018-06544-z.</cite> [<a href="https://doi.org/10.1038/s41467-018-06544-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6172195/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30287902/" class="usa-link">PubMed</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Andersson R, Andersen PR, Valen E, Core LJ, Bornholdt J, Boyd M, et al. Nuclear stability and transcriptional directionality separate functionally distinct RNA species. Nat Commun. 2014;5: 5336. 10.1038/ncomms6336.
</cite> [<a href="https://doi.org/10.1038/ncomms6336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25387874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andersson%20R,%20Andersen%20PR,%20Valen%20E,%20Core%20LJ,%20Bornholdt%20J,%20Boyd%20M,%20et%20al.%20Nuclear%20stability%20and%20transcriptional%20directionality%20separate%20functionally%20distinct%20RNA%20species.%20Nat%20Commun.%202014;5:%205336.%2010.1038/ncomms6336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Pollard KS, Hubisz MJ, Rosenbloom KR, Siepel A. Detection of nonneutral substitution rates on mammalian phylogenies. Genome Res. 2010;20:110–21.
</cite> [<a href="https://doi.org/10.1101/gr.097857.109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2798823/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19858363/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pollard%20KS,%20Hubisz%20MJ,%20Rosenbloom%20KR,%20Siepel%20A.%20Detection%20of%20nonneutral%20substitution%20rates%20on%20mammalian%20phylogenies.%20Genome%20Res.%202010;20:110%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Danko CG, Choate LA, Marks BA, Rice EJ, Wang Z, Chu T, et al. Dynamic evolution of regulatory element ensembles in primate CD4+ T cells. Nat Ecol Evol. 2018;2:537–48.
</cite> [<a href="https://doi.org/10.1038/s41559-017-0447-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5957490/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29379187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Danko%20CG,%20Choate%20LA,%20Marks%20BA,%20Rice%20EJ,%20Wang%20Z,%20Chu%20T,%20et%20al.%20Dynamic%20evolution%20of%20regulatory%20element%20ensembles%20in%20primate%20CD4+%20T%20cells.%20Nat%20Ecol%20Evol.%202018;2:537%E2%80%9348." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Lee BH, Rhie SK. Molecular and computational approaches to map regulatory elements in 3d chromatin structure. Epigenetics Chromatin. 2021;14: 14. 10.1186/s13072-021-00390-y.
</cite> [<a href="https://doi.org/10.1186/s13072-021-00390-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7980343/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33741028/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20BH,%20Rhie%20SK.%20Molecular%20and%20computational%20approaches%20to%20map%20regulatory%20elements%20in%203d%20chromatin%20structure.%20Epigenetics%20Chromatin.%202021;14:%2014.%2010.1186/s13072-021-00390-y." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite> Lee T, Zhai J, Meyers BC. Conservation and divergence in eukaryotic DNA methylation. Proc National Acad Sci [Internet]. 2010;107:9027 LP – 9028. Available from: <a href="http://www.pnas.org/content/107/20/9027.abstract" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.pnas.org/content/107/20/9027.abstract</a>.</cite> [<a href="https://doi.org/10.1073/pnas.1005440107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2889049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20457928/" class="usa-link">PubMed</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Chuong EB, Elde NC, Feschotte C. Regulatory activities of transposable elements: from conflicts to benefits. Nat Rev Genet. 2017;18:71–86.
</cite> [<a href="https://doi.org/10.1038/nrg.2016.139" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5498291/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27867194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chuong%20EB,%20Elde%20NC,%20Feschotte%20C.%20Regulatory%20activities%20of%20transposable%20elements:%20from%20conflicts%20to%20benefits.%20Nat%20Rev%20Genet.%202017;18:71%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Joly-Lopez Z, Bureau TE. Exaptation of transposable element coding sequences. Curr Opin Genet Dev. 2018;49:34–42.
</cite> [<a href="https://doi.org/10.1016/j.gde.2018.02.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29525543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Joly-Lopez%20Z,%20Bureau%20TE.%20Exaptation%20of%20transposable%20element%20coding%20sequences.%20Curr%20Opin%20Genet%20Dev.%202018;49:34%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite> Liu B, Zhao M. How transposable elements are recognized and epigenetically silenced in plants? Current Opinion in Plant Biology [Internet]. 2023;75:102428. Available from: <a href="https://www.sciencedirect.com/science/article/pii/S1369526623000936" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.sciencedirect.com/science/article/pii/S1369526623000936</a>.</cite> [<a href="https://doi.org/10.1016/j.pbi.2023.102428" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37481986/" class="usa-link">PubMed</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Li Q, Gent JI, Zynda G, Song J, Makarevitch I, Hirsch CD, et al. RNA-directed DNA methylation enforces boundaries between heterochromatin and euchromatin in the maize genome. Proc National Acad Sci. 2015;112:14728–33.</cite> [<a href="https://doi.org/10.1073/pnas.1514680112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4664327/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26553984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20Q,%20Gent%20JI,%20Zynda%20G,%20Song%20J,%20Makarevitch%20I,%20Hirsch%20CD,%20et%20al.%20RNA-directed%20DNA%20methylation%20enforces%20boundaries%20between%20heterochromatin%20and%20euchromatin%20in%20the%20maize%20genome.%20Proc%20National%20Acad%20Sci.%202015;112:14728%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Guo W, Wang D, Lisch D. RNA-directed DNA methylation prevents rapid and heritable reversal of transposon silencing under heat stress in <em>Zea mays</em>. PLoS Genet. 2021;17: e1009326.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1009326" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8224964/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34125827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20W,%20Wang%20D,%20Lisch%20D.%20RNA-directed%20DNA%20methylation%20prevents%20rapid%20and%20heritable%20reversal%20of%20transposon%20silencing%20under%20heat%20stress%20in%20Zea%20mays.%20PLoS%20Genet.%202021;17:%20e1009326." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite> Martin GT, Seymour DK, Gaut BS. CHH Methylation Islands: A Nonconserved Feature of Grass Genomes That Is Positively Associated with Transposable Elements but Negatively Associated with Gene-Body Methylation. Genome Biol Evol. 2021;13:evab144.</cite> [<a href="https://doi.org/10.1093/gbe/evab144" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8374106/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34146109/" class="usa-link">PubMed</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Zhang H, Lang Z, Zhu J-K. Dynamics and function of DNA methylation in plants. Nat Rev Mol Cell Biol. 2018;19:489–506.
</cite> [<a href="https://doi.org/10.1038/s41580-018-0016-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29784956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20H,%20Lang%20Z,%20Zhu%20J-K.%20Dynamics%20and%20function%20of%20DNA%20methylation%20in%20plants.%20Nat%20Rev%20Mol%20Cell%20Biol.%202018;19:489%E2%80%93506." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Matzke MA, Kanno T, Matzke AJM. RNA-directed DNA methylation: the evolution of a complex epigenetic pathway in flowering plants. Annu Rev Plant Biol. 2014;66:1–25.
</cite> [<a href="https://doi.org/10.1146/annurev-arplant-043014-114633" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25494460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Matzke%20MA,%20Kanno%20T,%20Matzke%20AJM.%20RNA-directed%20DNA%20methylation:%20the%20evolution%20of%20a%20complex%20epigenetic%20pathway%20in%20flowering%20plants.%20Annu%20Rev%20Plant%20Biol.%202014;66:1%E2%80%9325." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Core LJ, Martins AL, Danko CG, Waters C, Siepel A, Lis JT. Analysis of nascent RNA identifies a unified architecture of initiation regions at mammalian promoters and enhancers. Nat Genet. 2014;46:1311–20.
</cite> [<a href="https://doi.org/10.1038/ng.3142" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4254663/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25383968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Core%20LJ,%20Martins%20AL,%20Danko%20CG,%20Waters%20C,%20Siepel%20A,%20Lis%20JT.%20Analysis%20of%20nascent%20RNA%20identifies%20a%20unified%20architecture%20of%20initiation%20regions%20at%20mammalian%20promoters%20and%20enhancers.%20Nat%20Genet.%202014;46:1311%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite> Deshpande AS, Ulahannan N, Pendleton M, Dai X, Ly L, Behr JM, et al. Identifying synergistic high-order 3D chromatin conformations from genome-scale nanopore concatemer sequencing. Nat Biotechnol. 2022;40(10):1488-99.</cite> [<a href="https://doi.org/10.1038/s41587-022-01289-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35637420/" class="usa-link">PubMed</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Li Z, Long Y, Yu Y, Zhang F, Zhang H, Liu Z, et al. Pore-C simultaneously captures genome-wide multi-way chromatin interaction and associated DNA methylation status in <em>Arabidopsis</em>. Plant Biotechnol J. 2022;20:1009–11.
</cite> [<a href="https://doi.org/10.1111/pbi.13811" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9129085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35313066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20Z,%20Long%20Y,%20Yu%20Y,%20Zhang%20F,%20Zhang%20H,%20Liu%20Z,%20et%20al.%20Pore-C%20simultaneously%20captures%20genome-wide%20multi-way%20chromatin%20interaction%20and%20associated%20DNA%20methylation%20status%20in%20Arabidopsis.%20Plant%20Biotechnol%20J.%202022;20:1009%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Szabo Q, Bantignies F, Cavalli G. Principles of genome folding into topologically associating domains. Sci Adv. 2019;5: eaaw1668.
</cite> [<a href="https://doi.org/10.1126/sciadv.aaw1668" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6457944/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30989119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Szabo%20Q,%20Bantignies%20F,%20Cavalli%20G.%20Principles%20of%20genome%20folding%20into%20topologically%20associating%20domains.%20Sci%20Adv.%202019;5:%20eaaw1668." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Liu C, Cheng Y-J, Wang J-W, Weigel D. Prominent topologically associated domains differentiate global chromatin packing in rice from Arabidopsis. Nat Plants. 2017;3:742–8.
</cite> [<a href="https://doi.org/10.1038/s41477-017-0005-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28848243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20C,%20Cheng%20Y-J,%20Wang%20J-W,%20Weigel%20D.%20Prominent%20topologically%20associated%20domains%20differentiate%20global%20chromatin%20packing%20in%20rice%20from%20Arabidopsis.%20Nat%20Plants.%202017;3:742%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Kaul A, Bhattacharyya S, Ay F. Identifying statistically significant chromatin contacts from Hi-C data with FitHiC2. Nat Protoc. 2020;15:991–1012.
</cite> [<a href="https://doi.org/10.1038/s41596-019-0273-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7451401/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31980751/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaul%20A,%20Bhattacharyya%20S,%20Ay%20F.%20Identifying%20statistically%20significant%20chromatin%20contacts%20from%20Hi-C%20data%20with%20FitHiC2.%20Nat%20Protoc.%202020;15:991%E2%80%931012." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Groen SC, Ćalić I, Joly-Lopez Z, Platts AE, Choi JY, Natividad M, et al. The strength and pattern of natural selection on gene expression in rice. Nature. 2020;578:572–6.
</cite> [<a href="https://doi.org/10.1038/s41586-020-1997-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32051590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Groen%20SC,%20%C4%86ali%C4%87%20I,%20Joly-Lopez%20Z,%20Platts%20AE,%20Choi%20JY,%20Natividad%20M,%20et%20al.%20The%20strength%20and%20pattern%20of%20natural%20selection%20on%20gene%20expression%20in%20rice.%20Nature.%202020;578:572%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Voichek Y, Hristova G, Mollá-Morales A, Weigel D, Nordborg M. Widespread position-dependent transcriptional regulatory sequences in plants. Nat Genet. 2024;56:2238–46.
</cite> [<a href="https://doi.org/10.1038/s41588-024-01907-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11525189/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39266765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Voichek%20Y,%20Hristova%20G,%20Moll%C3%A1-Morales%20A,%20Weigel%20D,%20Nordborg%20M.%20Widespread%20position-dependent%20transcriptional%20regulatory%20sequences%20in%20plants.%20Nat%20Genet.%202024;56:2238%E2%80%9346." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Schaukowitch K, Joo J-Y, Liu X, Watts JK, Martinez C, Kim T-K. Enhancer RNA facilitates NELF release from immediate early genes. Mol Cell. 2014;56:29–42.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2014.08.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4186258/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25263592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schaukowitch%20K,%20Joo%20J-Y,%20Liu%20X,%20Watts%20JK,%20Martinez%20C,%20Kim%20T-K.%20Enhancer%20RNA%20facilitates%20NELF%20release%20from%20immediate%20early%20genes.%20Mol%20Cell.%202014;56:29%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Arnold PR, Wells AD, Li XC. Diversity and Emerging Roles of Enhancer RNA in Regulation of Gene Expression and Cell Fate. Frontiers Cell Dev Biology. 2020;7:377.</cite> [<a href="https://doi.org/10.3389/fcell.2019.00377" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6971116/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31993419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arnold%20PR,%20Wells%20AD,%20Li%20XC.%20Diversity%20and%20Emerging%20Roles%20of%20Enhancer%20RNA%20in%20Regulation%20of%20Gene%20Expression%20and%20Cell%20Fate.%20Frontiers%20Cell%20Dev%20Biology.%202020;7:377." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Cinghu S, Yang P, Kosak JP, Conway AE, Kumar D, Oldfield AJ, et al. Intragenic Enhancers Attenuate Host Gene Expression. Mol Cell. 2017;68:104-117.e6.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2017.09.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5683415/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28985501/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cinghu%20S,%20Yang%20P,%20Kosak%20JP,%20Conway%20AE,%20Kumar%20D,%20Oldfield%20AJ,%20et%20al.%20Intragenic%20Enhancers%20Attenuate%20Host%20Gene%20Expression.%20Mol%20Cell.%202017;68:104-117.e6." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Koch F, Fenouil R, Gut M, Cauchy P, Albert TK, Zacarias-Cabeza J, et al. Transcription initiation platforms and GTF recruitment at tissue-specific enhancers and promoters. Nat Struct Mol Biol. 2011;18:956–63.
</cite> [<a href="https://doi.org/10.1038/nsmb.2085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21765417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koch%20F,%20Fenouil%20R,%20Gut%20M,%20Cauchy%20P,%20Albert%20TK,%20Zacarias-Cabeza%20J,%20et%20al.%20Transcription%20initiation%20platforms%20and%20GTF%20recruitment%20at%20tissue-specific%20enhancers%20and%20promoters.%20Nat%20Struct%20Mol%20Biol.%202011;18:956%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Ostuni R, Piccolo V, Barozzi I, Polletti S, Termanini A, Bonifacio S, et al. Latent enhancers activated by stimulation in differentiated cells. Cell. 2013;152:157–71.
</cite> [<a href="https://doi.org/10.1016/j.cell.2012.12.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23332752/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ostuni%20R,%20Piccolo%20V,%20Barozzi%20I,%20Polletti%20S,%20Termanini%20A,%20Bonifacio%20S,%20et%20al.%20Latent%20enhancers%20activated%20by%20stimulation%20in%20differentiated%20cells.%20Cell.%202013;152:157%E2%80%9371." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Neumayr C, Haberle V, Serebreni L, Karner K, Hendy O, Boija A, et al. Differential cofactor dependencies define distinct types of human enhancers. Nature. 2022;606:406–13.
</cite> [<a href="https://doi.org/10.1038/s41586-022-04779-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7613064/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35650434/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Neumayr%20C,%20Haberle%20V,%20Serebreni%20L,%20Karner%20K,%20Hendy%20O,%20Boija%20A,%20et%20al.%20Differential%20cofactor%20dependencies%20define%20distinct%20types%20of%20human%20enhancers.%20Nature.%202022;606:406%E2%80%9313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Cardiello JF, Sanchez GJ, Allen MA, Dowell RD. Lessons from eRNAs: understanding transcriptional regulation through the lens of nascent RNAs. Transcription. 2020;11:3–18.
</cite> [<a href="https://doi.org/10.1080/21541264.2019.1704128" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7053884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31856658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cardiello%20JF,%20Sanchez%20GJ,%20Allen%20MA,%20Dowell%20RD.%20Lessons%20from%20eRNAs:%20understanding%20transcriptional%20regulation%20through%20the%20lens%20of%20nascent%20RNAs.%20Transcription.%202020;11:3%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Wang S-P, Tang Z, Chen C-W, Shimada M, Koche RP, Wang L-H, et al. A UTX-MLL4-p300 Transcriptional Regulatory Network Coordinately Shapes Active Enhancer Landscapes for Eliciting Transcription. Mol Cell. 2017;67:308-321.e6.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2017.06.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5574165/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28732206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20S-P,%20Tang%20Z,%20Chen%20C-W,%20Shimada%20M,%20Koche%20RP,%20Wang%20L-H,%20et%20al.%20A%20UTX-MLL4-p300%20Transcriptional%20Regulatory%20Network%20Coordinately%20Shapes%20Active%20Enhancer%20Landscapes%20for%20Eliciting%20Transcription.%20Mol%20Cell.%202017;67:308-321.e6." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Raisner R, Kharbanda S, Jin L, Jeng E, Chan E, Merchant M, et al. Enhancer activity requires CBP/P300 bromodomain-dependent histone H3K27 acetylation. Cell Rep. 2018;24:1722–9.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2018.07.041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30110629/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raisner%20R,%20Kharbanda%20S,%20Jin%20L,%20Jeng%20E,%20Chan%20E,%20Merchant%20M,%20et%20al.%20Enhancer%20activity%20requires%20CBP/P300%20bromodomain-dependent%20histone%20H3K27%20acetylation.%20Cell%20Rep.%202018;24:1722%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Danko CG, Hyland SL, Core LJ, Martins AL, Waters CT, Lee HW, et al. Identification of active transcriptional regulatory elements from GRO-seq data. Nat Methods. 2015;12:433–8.
</cite> [<a href="https://doi.org/10.1038/nmeth.3329" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4507281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25799441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Danko%20CG,%20Hyland%20SL,%20Core%20LJ,%20Martins%20AL,%20Waters%20CT,%20Lee%20HW,%20et%20al.%20Identification%20of%20active%20transcriptional%20regulatory%20elements%20from%20GRO-seq%20data.%20Nat%20Methods.%202015;12:433%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Ouyang W, Xiong D, Li G, Li X. Unraveling the 3D Genome Architecture in Plants: Present and Future. Mol Plant. 2020;13:1676–93.
</cite> [<a href="https://doi.org/10.1016/j.molp.2020.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33065269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ouyang%20W,%20Xiong%20D,%20Li%20G,%20Li%20X.%20Unraveling%20the%203D%20Genome%20Architecture%20in%20Plants:%20Present%20and%20Future.%20Mol%20Plant.%202020;13:1676%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR101">
<span class="label">101.</span><cite> Dong P, Tu X, Chu P-Y, Lü P, Zhu N, Grierson D, et al. 3D Chromatin Architecture of Large Plant Genomes Determined by Local A/B Compartments. Molecular Plant [Internet]. 2017;10:1497–509. Available from: <a href="https://www.sciencedirect.com/science/article/pii/S1674205217303398" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.sciencedirect.com/science/article/pii/S1674205217303398</a>.</cite> [<a href="https://doi.org/10.1016/j.molp.2017.11.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29175436/" class="usa-link">PubMed</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite> Tourdot E, Grob S. Three-dimensional chromatin architecture in plants – General features and novelties. Eur J Cell Biol [Internet]. 2023;102:151344. Available from: <a href="https://www.sciencedirect.com/science/article/pii/S0171933523000596" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.sciencedirect.com/science/article/pii/S0171933523000596</a>.</cite> [<a href="https://doi.org/10.1016/j.ejcb.2023.151344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37562220/" class="usa-link">PubMed</a>]</li>
<li id="CR103">
<span class="label">103.</span><cite>Kurbidaeva A, Gupta S, Zaidem M, Castanera R, Sato Y, Joly-Lopez Z, et al. Topologically associating domains and the evolution of three-dimensional genome architecture in rice. Plant J. 2025;122: e70139.
</cite> [<a href="https://doi.org/10.1111/tpj.70139" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12086760/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40384625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kurbidaeva%20A,%20Gupta%20S,%20Zaidem%20M,%20Castanera%20R,%20Sato%20Y,%20Joly-Lopez%20Z,%20et%20al.%20Topologically%20associating%20domains%20and%20the%20evolution%20of%20three-dimensional%20genome%20architecture%20in%20rice.%20Plant%20J.%202025;122:%20e70139." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Creasey KM, Zhai J, Borges F, Ex FV, Regulski M, Meyers BC, et al. Mirnas trigger widespread epigenetically activated sirnas from transposons in Arabidopsis. Nature. 2014;508:411–5.
</cite> [<a href="https://doi.org/10.1038/nature13069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4074602/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24670663/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Creasey%20KM,%20Zhai%20J,%20Borges%20F,%20Ex%20FV,%20Regulski%20M,%20Meyers%20BC,%20et%20al.%20Mirnas%20trigger%20widespread%20epigenetically%20activated%20sirnas%20from%20transposons%20in%20Arabidopsis.%20Nature.%202014;508:411%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Erdmann RM, Picard CL. RNA-directed DNA methylation. PLoS Genet. 2020;16: e1009034.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1009034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7544125/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33031395/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Erdmann%20RM,%20Picard%20CL.%20RNA-directed%20DNA%20methylation.%20PLoS%20Genet.%202020;16:%20e1009034." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR106">
<span class="label">106.</span><cite>Andersson R, Andersen PR, Valen E, Core LJ, Bornholdt J, Boyd M, et al. Nuclear stability and transcriptional directionality separate functionally distinct RNA species. Nat Commun. 2014;5: 5336.
</cite> [<a href="https://doi.org/10.1038/ncomms6336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25387874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andersson%20R,%20Andersen%20PR,%20Valen%20E,%20Core%20LJ,%20Bornholdt%20J,%20Boyd%20M,%20et%20al.%20Nuclear%20stability%20and%20transcriptional%20directionality%20separate%20functionally%20distinct%20RNA%20species.%20Nat%20Commun.%202014;5:%205336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR107">
<span class="label">107.</span><cite>Buenrostro JD, Wu B, Chang HY, Greenleaf WJ. ATAC-seq: a method for assaying chromatin accessibility genome-wide. Curr Protoc Mol Biol. 2015;109:21.29.1-21.29.9.
</cite> [<a href="https://doi.org/10.1002/0471142727.mb2129s109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4374986/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25559105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buenrostro%20JD,%20Wu%20B,%20Chang%20HY,%20Greenleaf%20WJ.%20ATAC-seq:%20a%20method%20for%20assaying%20chromatin%20accessibility%20genome-wide.%20Curr%20Protoc%20Mol%20Biol.%202015;109:21.29.1-21.29.9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR108">
<span class="label">108.</span><cite> Zhang W, Jiang J. Genome-Wide Mapping of DNase I Hypersensitive Sites in Plants BT - Plant Functional Genomics: Methods and Protocols. Stepanova"] ["Jose M Alonso and Anna N, editor. Methods Mol Biol [Internet]. 2015;1284:71–89. Available from: 10.1007/978-1-4939-2444-8_4.</cite> [<a href="https://doi.org/10.1007/978-1-4939-2444-8_4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25757768/" class="usa-link">PubMed</a>]</li>
<li id="CR109">
<span class="label">109.</span><cite> Bolger A, Scossa F, Bolger ME, Lanz C, Maumus F, Tohge T, et al. The genome of the stress-tolerant wild tomato species Solanum pennellii. Nat Genet [Internet]. 2014;46:1034–8. Available from: 10.1038/ng.3046.</cite> [<a href="https://doi.org/10.1038/ng.3046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7036041/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25064008/" class="usa-link">PubMed</a>]</li>
<li id="CR110">
<span class="label">110.</span><cite>Langmead B, Salzberg SL. Fast gapped-read alignment with Bowtie 2. Nat Methods. 2012;9:357. 10.1038/nmeth.1923.
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langmead%20B,%20Salzberg%20SL.%20Fast%20gapped-read%20alignment%20with%20Bowtie%202.%20Nat%20Methods.%202012;9:357.%2010.1038/nmeth.1923." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR111">
<span class="label">111.</span><cite> Danecek P, Bonfield JK, Liddle J, Marshall J, Ohan V, Pollard MO, et al. Twelve years of SAMtools and BCFtools. GigaScience. 2021;10:giab008.</cite> [<a href="https://doi.org/10.1093/gigascience/giab008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7931819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33590861/" class="usa-link">PubMed</a>]</li>
<li id="CR112">
<span class="label">112.</span><cite>Ramírez F, Ryan DP, Grüning B, Bhardwaj V, Kilpert F, Richter AS, et al. deepTools2: a next generation web server for deep-sequencing data analysis. Nucleic Acids Res. 2016;44:W160–5.
</cite> [<a href="https://doi.org/10.1093/nar/gkw257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4987876/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27079975/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ram%C3%ADrez%20F,%20Ryan%20DP,%20Gr%C3%BCning%20B,%20Bhardwaj%20V,%20Kilpert%20F,%20Richter%20AS,%20et%20al.%20deepTools2:%20a%20next%20generation%20web%20server%20for%20deep-sequencing%20data%20analysis.%20Nucleic%20Acids%20Res.%202016;44:W160%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR113">
<span class="label">113.</span><cite> Zhang Y, Liu T, Meyer C a, Eeckhoute J, Johnson DS, Bernstein BE, et al. Model-based analysis of ChIP-Seq (MACS). Genome Biol [Internet]. 2008;9:R137. Available from: <a href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2592715&amp;tool=pmcentrez&amp;rendertype=abstract" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=2592715&amp;tool=pmcentrez&amp;rendertype=abstract</a>.</cite> [<a href="https://doi.org/10.1186/gb-2008-9-9-r137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2592715/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18798982/" class="usa-link">PubMed</a>]</li>
<li id="CR114">
<span class="label">114.</span><cite> Martin M. Cutadapt removes adapter sequences from high-throughput sequencing reads. Embnet J [Internet]. 2011;17:10–2. Available from: <a href="http://journal.embnet.org/index.php/embnetjournal/article/view/200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://journal.embnet.org/index.php/embnetjournal/article/view/200</a>.</cite>
</li>
<li id="CR115">
<span class="label">115.</span><cite>Liao Y, Smyth GK, Shi W. Featurecounts: an efficient general purpose program for assigning sequence reads to genomic features. Bioinformatics. 2014;30:923–30.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt656" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liao%20Y,%20Smyth%20GK,%20Shi%20W.%20Featurecounts:%20an%20efficient%20general%20purpose%20program%20for%20assigning%20sequence%20reads%20to%20genomic%20features.%20Bioinformatics.%202014;30:923%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR116">
<span class="label">116.</span><cite>Krueger F, Andrews SR. Bismark: a flexible aligner and methylation caller for Bisulfite-Seq applications. Bioinformatics. 2011;27:1571–2.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3102221/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21493656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krueger%20F,%20Andrews%20SR.%20Bismark:%20a%20flexible%20aligner%20and%20methylation%20caller%20for%20Bisulfite-Seq%20applications.%20Bioinformatics.%202011;27:1571%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR117">
<span class="label">117.</span><cite>Choi JY, Purugganan MD. Evolutionary epigenomics of retrotransposon-mediated methylation spreading in rice. Mol Biol Evol. 2018;35:365–82.
</cite> [<a href="https://doi.org/10.1093/molbev/msx284" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5850837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29126199/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Choi%20JY,%20Purugganan%20MD.%20Evolutionary%20epigenomics%20of%20retrotransposon-mediated%20methylation%20spreading%20in%20rice.%20Mol%20Biol%20Evol.%202018;35:365%E2%80%9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR118">
<span class="label">118.</span><cite>Choi JY, Lye ZN, Groen SC, Dai X, Rughani P, Zaaijer S, et al. Nanopore sequencing-based genome assembly and evolutionary genomics of circum-basmati rice. Genome Biol. 2020;21: 21.
</cite> [<a href="https://doi.org/10.1186/s13059-020-1938-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7001208/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32019604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Choi%20JY,%20Lye%20ZN,%20Groen%20SC,%20Dai%20X,%20Rughani%20P,%20Zaaijer%20S,%20et%20al.%20Nanopore%20sequencing-based%20genome%20assembly%20and%20evolutionary%20genomics%20of%20circum-basmati%20rice.%20Genome%20Biol.%202020;21:%2021." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR119">
<span class="label">119.</span><cite>Knight PA, Ruiz D. A fast algorithm for matrix balancing. IMA J Numer Anal. 2012;33:1029–47. 10.1093/imanum/drs019.</cite> [<a href="https://scholar.google.com/scholar_lookup?Knight%20PA,%20Ruiz%20D.%20A%20fast%20algorithm%20for%20matrix%20balancing.%20IMA%20J%20Numer%20Anal.%202012;33:1029%E2%80%9347.%2010.1093/imanum/drs019." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR120">
<span class="label">120.</span><cite>Ramírez F, Bhardwaj V, Arrigoni L, Lam KC, Grüning BA, Villaveces J, et al. High-resolution TADs reveal DNA sequences underlying genome organization in flies. Nat Commun. 2018;9: 189.
</cite> [<a href="https://doi.org/10.1038/s41467-017-02525-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5768762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29335486/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ram%C3%ADrez%20F,%20Bhardwaj%20V,%20Arrigoni%20L,%20Lam%20KC,%20Gr%C3%BCning%20BA,%20Villaveces%20J,%20et%20al.%20High-resolution%20TADs%20reveal%20DNA%20sequences%20underlying%20genome%20organization%20in%20flies.%20Nat%20Commun.%202018;9:%20189." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR121">
<span class="label">121.</span><cite>Shin H, Shi Y, Dai C, Tjong H, Gong K, Alber F, et al. Topdom: an efficient and deterministic method for identifying topological domains in genomes. Nucleic Acids Res. 2016;44: e70–e70.
</cite> [<a href="https://doi.org/10.1093/nar/gkv1505" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4838359/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26704975/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shin%20H,%20Shi%20Y,%20Dai%20C,%20Tjong%20H,%20Gong%20K,%20Alber%20F,%20et%20al.%20Topdom:%20an%20efficient%20and%20deterministic%20method%20for%20identifying%20topological%20domains%20in%20genomes.%20Nucleic%20Acids%20Res.%202016;44:%20e70%E2%80%93e70." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR122">
<span class="label">122.</span><cite>Zhou Y, Chebotarov D, Kudrna D, Llaca V, Lee S, Rajasekar S, et al. A platinum standard pan-genome resource that represents the population structure of Asian rice. Sci Data. 2020;7: 113.
</cite> [<a href="https://doi.org/10.1038/s41597-020-0438-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7138821/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32265447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou%20Y,%20Chebotarov%20D,%20Kudrna%20D,%20Llaca%20V,%20Lee%20S,%20Rajasekar%20S,%20et%20al.%20A%20platinum%20standard%20pan-genome%20resource%20that%20represents%20the%20population%20structure%20of%20Asian%20rice.%20Sci%20Data.%202020;7:%20113." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR123">
<span class="label">123.</span><cite>Yu Z, Chen Y, Zhou Y, Zhang Y, Li M, Ouyang Y, et al. Rice Gene Index: A comprehensive pan-genome database for comparative and functional genomics of Asian rice. Mol Plant. 2023;16:798–801.
</cite> [<a href="https://doi.org/10.1016/j.molp.2023.03.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36966359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20Z,%20Chen%20Y,%20Zhou%20Y,%20Zhang%20Y,%20Li%20M,%20Ouyang%20Y,%20et%20al.%20Rice%20Gene%20Index:%20A%20comprehensive%20pan-genome%20database%20for%20comparative%20and%20functional%20genomics%20of%20Asian%20rice.%20Mol%20Plant.%202023;16:798%E2%80%93801." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR124">
<span class="label">124.</span><cite>McKain MR, Tang H, McNeal JR, Ayyampalayam S, Davis JI, dePamphilis CW, et al. A Phylogenomic Assessment of Ancient Polyploidy and Genome Evolution across the Poales. Genome Biol Evol. 2016;8:1150–64.
</cite> [<a href="https://doi.org/10.1093/gbe/evw060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4860692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26988252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McKain%20MR,%20Tang%20H,%20McNeal%20JR,%20Ayyampalayam%20S,%20Davis%20JI,%20dePamphilis%20CW,%20et%20al.%20A%20Phylogenomic%20Assessment%20of%20Ancient%20Polyploidy%20and%20Genome%20Evolution%20across%20the%20Poales.%20Genome%20Biol%20Evol.%202016;8:1150%E2%80%9364." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR125">
<span class="label">125.</span><cite>Hinrichs AS, Karolchik D, Baertsch R, Barber GP, Bejerano G, Clawson H, et al. The UCSC Genome Browser Database: update 2006. Nucleic Acids Res. 2006;34:D590–8.
</cite> [<a href="https://doi.org/10.1093/nar/gkj144" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1347506/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16381938/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hinrichs%20AS,%20Karolchik%20D,%20Baertsch%20R,%20Barber%20GP,%20Bejerano%20G,%20Clawson%20H,%20et%20al.%20The%20UCSC%20Genome%20Browser%20Database:%20update%202006.%20Nucleic%20Acids%20Res.%202006;34:D590%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR126">
<span class="label">126.</span><cite>Storer J, Hubley R, Rosen J, Wheeler TJ, Smit AF. The Dfam community resource of transposable element families, sequence models, and genome annotations. Mob DNA. 2021;12: 2.
</cite> [<a href="https://doi.org/10.1186/s13100-020-00230-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7805219/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33436076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Storer%20J,%20Hubley%20R,%20Rosen%20J,%20Wheeler%20TJ,%20Smit%20AF.%20The%20Dfam%20community%20resource%20of%20transposable%20element%20families,%20sequence%20models,%20and%20genome%20annotations.%20Mob%20DNA.%202021;12:%202." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR127">
<span class="label">127.</span><cite>Flynn JM, Hubley R, Goubert C, Rosen J, Clark AG, Feschotte C, et al. RepeatModeler2 for automated genomic discovery of transposable element families. Proc Natl Acad Sci. 2020;117:9451–7.
</cite> [<a href="https://doi.org/10.1073/pnas.1921046117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7196820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32300014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Flynn%20JM,%20Hubley%20R,%20Goubert%20C,%20Rosen%20J,%20Clark%20AG,%20Feschotte%20C,%20et%20al.%20RepeatModeler2%20for%20automated%20genomic%20discovery%20of%20transposable%20element%20families.%20Proc%20Natl%20Acad%20Sci.%202020;117:9451%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR128">
<span class="label">128.</span><cite>Gel B, Díez-Villanueva A, Serra E, Buschbeck M, Peinado MA, Malinverni R. regioneR: an R/Bioconductor package for the association analysis of genomic regions based on permutation tests. Bioinformatics. 2015;32(2):289–91.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv562" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4708104/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26424858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gel%20B,%20D%C3%ADez-Villanueva%20A,%20Serra%20E,%20Buschbeck%20M,%20Peinado%20MA,%20Malinverni%20R.%20regioneR:%20an%20R/Bioconductor%20package%20for%20the%20association%20analysis%20of%20genomic%20regions%20based%20on%20permutation%20tests.%20Bioinformatics.%202015;32(2):289%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR129">
<span class="label">129.</span><cite>Quinlan AR, Hall IM. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics. 2010;26:841–2.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btq033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2832824/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20110278/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quinlan%20AR,%20Hall%20IM.%20BEDTools:%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features.%20Bioinformatics.%202010;26:841%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR130">
<span class="label">130.</span><cite>Conway JR, Lex A, Gehlenborg N. UpSetR: an R package for the visualization of intersecting sets and their properties. Bioinformatics. 2017;33:2938–40.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btx364" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5870712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28645171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Conway%20JR,%20Lex%20A,%20Gehlenborg%20N.%20UpSetR:%20an%20R%20package%20for%20the%20visualization%20of%20intersecting%20sets%20and%20their%20properties.%20Bioinformatics.%202017;33:2938%E2%80%9340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR131">
<span class="label">131.</span><cite>Shi C, Li W, Zhang Q-J, Zhang Y, Tong Y, Li K, et al. The draft genome sequence of an upland wild rice species, <em>Oryza granulata</em>. Sci Data. 2020;7: 131.
</cite> [<a href="https://doi.org/10.1038/s41597-020-0470-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7190833/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32350267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shi%20C,%20Li%20W,%20Zhang%20Q-J,%20Zhang%20Y,%20Tong%20Y,%20Li%20K,%20et%20al.%20The%20draft%20genome%20sequence%20of%20an%20upland%20wild%20rice%20species,%20Oryza%20granulata.%20Sci%20Data.%202020;7:%20131." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR132">
<span class="label">132.</span><cite> Goliasse M, Johary A, Platts AE, Ortner-Krause F, Edger PP, Choi JY, et al. Uncovering the multi-layer cis-regulatory landscape of rice via integrative nascent RNA analysis. Datasets, Sequence Read Archive (SRA) [Internet]. 2025; Available from: <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA586887" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA586887</a>.</cite>
</li>
<li id="CR133">
<span class="label">133.</span><cite> Goliasse M, Johary A, Platts AE, Ortner-Krause F, Edger PP, Choi JY, et al. Uncovering the multi-layer cis-regulatory landscape of rice via integrative nascent RNA analysis. Datasets, Sequence Read Archive (SRA) [Internet]. 2025; Available from: <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA714935" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA714935</a>.
</cite>
</li>
<li id="CR134">
<span class="label">134.</span><cite> Goliasse M. CNS_dREG. Github [Internet]. 2025; Available from: <a href="https://gitlab.com/Aniram_GS/cns_dreg" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gitlab.com/Aniram_GS/cns_dreg</a>.</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3715_MOESM1_ESM.pdf</a><sup> (12.8MB, pdf) </sup><p>Additional file 1: Fig S1. Circos plot for synteny links and gene density. Fig S2. Synmap dotplot about syntenic links. Fig S3. Distribution CNS. Fig S4. Correlation between PRO-seq replicates. Fig S5. PRO-seq signal within and around annotated regions. Fig S6. dREG characteristics. Fig S8. Overlap analysis between dREG locations, ATAC peaks and CNSs. Fig S9. Percentage coverage of CNS, ATAC-seq and dREG peaks. Fig S10. Gene ontology analysis for annotated rice genes with top 1,000 CNS content. Fig S11. Gene ontology analysis for annotated rice genes with top 1,000 ATAC content. Fig S12. Gene ontology analysis for annotated rice genes with top 709 dREG content. Fig S13. <em>De novo</em> motif analysis for the top enriched 6-8 bp motifs found in CNSs. Fig S14.<em> De novo</em> motif analysis for the top enriched 6-8 bp motifs found in dREG locations and ATAC peaks. Fig S15. CHH methylation within and around of dREG sites. Fig S16. H3K27me3 signal profile within and around CNS based on their locations. Fig S17. Methylation profile through the selection of dREG candidates. Fig S18. Epigenetic marks for coding and repetitive sequences in the rice genome. Fig S19. Percentage methylation by context within and around intergenic dREG locations and their percentage of overlap with TEs. Fig S20. Correlation of the percentage of cytosine methylation in all three contexts. Fig S21. sRNA and ATAC enrichment analysis in dREG sites by bin score. Fig S22. Enrichment analysis for dREG sites with TE overlap. Fig S23. Chromatin marks within and around dREG post-filtering. Fig S24. Gene expression analysis based on the proximity of dREG sites. Fig S25. Epigenetic marks surrounding TAD boundaries. Fig S26. Distribution of loops detected by Pore-C for dREG<sub>distal</sub> peaks. Fig S27. Expression levels analysis of genes present in genic loops with a contact of a noncoding region. Fig S28. Association analysis between eQTLs regions and dREG<sub>postfilter</sub> sites. Fig S29. Association analysis between eQTLs regions and ATAC intergenic regions. Fig S30. Association analysis between eQTLs regions and CNS intergenic regions. Fig S31. Genome Browser view of an Azucena locus (GAPDH). Fig S32. Genome Browser view of the Azucena locus (WRKY5). Fig S33. Genome Browser view of an Azucena intergenic region. Fig S34. Genome Browser view of an Azucena intergenic region with three types of regulatory elements. Fig S35. Genome Browser view of the Azucena genome #1. Fig S36. Genome Browser view of the Azucena genome #2. Fig S37. Genome Browser view of the Azucena genome #3. Fig S38. Genome Browser view of the Azucena genome #4. Fig S39. Genome Browser view of the Azucena genome #5.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3715_MOESM2_ESM.xlsx</a><sup> (2.9MB, xlsx) </sup><p>Additional file 2. CNS genomic coordinates.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3715_MOESM3_ESM.pdf</a><sup> (98.1KB, pdf) </sup><p>Additional file 3: Table S1. PRO-seq summary statistics. Table S2. ATAC-seq summary statistics. Table S3. ChIP-seq summary statistics. Table S4. Transposable elements percentage overlap by annotation Summary. Table S5. sRNA-seq summary statistics. Table S6: Pore-C summary statistics.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3715_MOESM4_ESM.xlsx</a><sup> (88.2KB, xlsx) </sup><p>Additional file 4: Gene Ontology analysis CNS.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3715_MOESM5_ESM.xlsx</a><sup> (64.6KB, xlsx) </sup><p>Additional file 5: Gene Ontology analysis ATAC.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3715_MOESM6_ESM.xlsx</a><sup> (20.2KB, xlsx) </sup><p>Additional file 6: Gene Ontology analysis dREG.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3715_MOESM7_ESM.xlsx</a><sup> (790.1KB, xlsx) </sup><p>Additional file 7: small RNA coordinates.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13059_2025_3715_MOESM8_ESM.xlsx</a><sup> (1.5MB, xlsx) </sup><p>Additional file 8: Total and postfilter dREG genomic coordinates.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12359857/bin/13059_2025_3715_MOESM9_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Additional file 9: Review history.</a><sup> (47.6KB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All raw sequencing data used in this study have been deposited in the Sequence Read Archive (SRA) <a href="https://www.ncbi.nlm.nih.gov/sra/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/sra/</a> with BioProject accession numbers PRJNA586887 [<a href="#CR132" class="usa-link" aria-describedby="CR132">132</a>] and PRJNA714935 [<a href="#CR133" class="usa-link" aria-describedby="CR133">133</a>]. This includes RNA-seq, ChIP-seq, BS-seq, and PRO-seq that were generated in this study was uploaded under PRJNA586887 specifically with the identifiers: ChIP-seq (H3K27me3) SRR19666802, SRR19666803, SRR19666804, SRR19666805, SRR19666806; ChIP-seq (H3K4Me3) SRR19666797, SRR19666798, SRR19666799, SR19666800, SRR19666801; ChIP-seq (H3K27ac) SRR19666809, SRR19666810, SRR19666811, SRR19666812, SRR19666813, SRR19666814; ChIP-seq (H3K18ac) SRR19666795, SRR19666796, SRR19666815, SRR19666816, SRR19666817; RNA-seq SRR19666818, SRR19666819; BS-seq SRR19666808; PRO-seq SRR19666807. The ATAC-seq and two replicates of sRNA-seq were uploaded under PRJNA714935 with the identifiers ATAC-seq SRR34572250, SRR34572251, SRR34572249; sRNA-seq SRR34572248, SRR34572247. For Pore-C, the FAST5 files are available under accession numbers SRR13985185, SRR13985186, SRR13985187, SRR13985192, and SRR13985193, and the corresponding FASTQ files are available under accession numbers SRR13985180, SRR13985181, SRR13985182, SRR13985190, and SRR13985191.</p>
<p>All custom scripts and analysis pipelines used in this study are publicly available on GitLab [<a href="#CR134" class="usa-link" aria-describedby="CR134">134</a>] at <a href="https://gitlab.com/Aniram_GS/cns_dreg" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gitlab.com/Aniram_GS/cns_dreg</a>. Processed data files and additional materials are included as supplementary files.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Genome Biology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s13059-025-03715-2"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/13059_2025_Article_3715.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12359857/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12359857/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12359857%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12359857/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12359857/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12359857/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40826469/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12359857/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40826469/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12359857/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12359857/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="J1T57teplnpfQzZD322dUCmNEHP5SNRQVsVdBarI2ktlKwH9sL8nfEHVMhVqzpGA">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
