
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            The PIN2 ortholog in barley modifies root gravitropism and architecture - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E484818AEF0D73058481001D7ACB8E.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328839/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="The PIN2 ortholog in barley modifies root gravitropism and architecture">
<meta name="citation_author" content="Zachary Aldiss">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Yasmine Lam">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Hannah Robinson">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author_institution" content="InterGrain Pty Ltd, Perth, Western Australia, Australia">
<meta name="citation_author" content="Richard Dixon">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Laura Steinhardt">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Peter Crisp">
<meta name="citation_author_institution" content="School of Agriculture and Food Sustainability, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Ian Godwin">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Andrew Borrell">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Warwick, Queensland, Australia">
<meta name="citation_author" content="Lee Hickey">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_author" content="Karen Massel">
<meta name="citation_author_institution" content="Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia">
<meta name="citation_publication_date" content="2025 Aug 6">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70061">
<meta name="citation_doi" content="10.1002/tpg2.70061">
<meta name="citation_pmid" content="40770375">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328839/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328839/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328839/pdf/TPG2-18-e70061.pdf">
<meta name="description" content="Roots provide the critical interface where plants acquire nutrients and water, but our limited understanding of the genetic controls modulating root system architecture (RSA) in crop species constrains opportunities to develop future cultivars with ...">
<meta name="og:title" content="The PIN2 ortholog in barley modifies root gravitropism and architecture">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Roots provide the critical interface where plants acquire nutrients and water, but our limited understanding of the genetic controls modulating root system architecture (RSA) in crop species constrains opportunities to develop future cultivars with ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328839/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12328839">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70061"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70061.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12328839%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12328839/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12328839/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328839/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70061" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Aug 6;18(3):e70061. doi: <a href="https://doi.org/10.1002/tpg2.70061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70061</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>The <em>PIN2</em> ortholog in barley modifies root gravitropism and architecture</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aldiss%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Zachary Aldiss</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zachary Aldiss</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aldiss%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zachary Aldiss</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lam%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yasmine Lam</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yasmine Lam</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lam%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yasmine Lam</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Robinson%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Hannah Robinson</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Hannah Robinson</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>InterGrain Pty Ltd, Perth, Western Australia, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Robinson%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hannah Robinson</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dixon%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Richard Dixon</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Richard Dixon</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dixon%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Richard Dixon</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Steinhardt%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Laura Steinhardt</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Laura Steinhardt</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Steinhardt%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Laura Steinhardt</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Crisp%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Peter Crisp</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Peter Crisp</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>School of Agriculture and Food Sustainability, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Crisp%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Peter Crisp</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Godwin%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Ian Godwin</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Ian Godwin</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Godwin%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ian Godwin</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Borrell%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Andrew Borrell</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Andrew Borrell</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Warwick, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Borrell%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrew Borrell</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hickey%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Lee Hickey</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Lee Hickey</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hickey%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lee Hickey</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Massel%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Karen Massel</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Karen Massel</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Massel%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Karen Massel</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270061-aff-0001">
<sup>
<sup>1</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, Queensland, Australia</div>
<div id="tpg270061-aff-0002">
<sup>
<sup>2</sup>
</sup>InterGrain Pty Ltd, Perth, Western Australia, Australia</div>
<div id="tpg270061-aff-0003">
<sup>
<sup>3</sup>
</sup>School of Agriculture and Food Sustainability, The University of Queensland, Brisbane, Queensland, Australia</div>
<div id="tpg270061-aff-0004">
<sup>
<sup>4</sup>
</sup>Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Warwick, Queensland, Australia</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Karen Massel, Queensland Alliance for Agriculture and Food Innovation, The University of Queensland, Brisbane, QLD, Australia. Email: <span>k.massel@uq.edu.au</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Apr 17; Received 2024 Nov 26; Accepted 2025 May 16; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12328839  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40770375/" class="usa-link">40770375</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Roots provide the critical interface where plants acquire nutrients and water, but our limited understanding of the genetic controls modulating root system architecture (RSA) in crop species constrains opportunities to develop future cultivars with improved root systems. However, there is vast knowledge of root developmental genes in model plant species, which has the potential to accelerate progress in crops with more complex genomes, particularly given that genome editing protocols are now available for most species. <em>PIN‐FORMED2</em> (<em>PIN2</em>) encodes a root‐specific polar auxin transporter, where its absence resulted in roots being unable to orient themselves using gravity, producing a significantly wider root system. To explore the role of <em>PIN2</em> in a cereal crop, we used CRISPR/Cas9 (where CRISPR is clustered regularly interspaced short palindromic repeats) editing to knockout of <em>PIN2</em> in barley (<em>Hordeum vulgare</em>). Like Arabidopsis, the roots of barley <em>pin2</em> loss‐of‐function mutants displayed an agravitropic response at seedling growth stages, resulting in a significantly shallower and wider root system at later growth stages. Notably, despite the significant change in RSA, there was no change in shoot architecture or total shoot biomass, with an insensitivity to the effects of higher planting density. We discuss the future challenges and opportunities to harness the <em>PIN2</em> pathway to optimize RSA in crops for a range of production scenarios without a shoot trade‐off.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270061-list-10001" class="list" style="list-style-type:disc">
<li><p>
<em>PIN‐FORMED</em> (PIN2) regulates root system architecture in barley.</p></li>
<li><p>Gene editing was used to create novel genetic variants.</p></li>
<li><p>Outcomes show wider root systems without impacting canopy development.</p></li>
<li><p>In‐depth root phenotypic platforms explored influence of this transporter holistically throughout the plant.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270061-dl-0001">
<dt>ABC</dt>
<dd><p>ATP‐binding cassette</p></dd>
<dt>AUX/LAX</dt>
<dd><p>AUXIN1/LIKE‐AUX1</p></dd>
<dt>BLUP</dt>
<dd><p>best linear unbiased predictions</p></dd>
<dt>CRISPR</dt>
<dd><p>clustered regularly interspaced short palindromic repeats</p></dd>
<dt>DAS</dt>
<dd><p>days after sowing</p></dd>
<dt>EGT</dt>
<dd><p>
<em>ENHANCED GRAVITROPISM</em>
</p></dd>
<dt>LMM</dt>
<dd><p>linear mixed model</p></dd>
<dt>MDR</dt>
<dd><p>multi‐drug resistance</p></dd>
<dt>PGP</dt>
<dd><p>P‐glycoprotein</p></dd>
<dt>PIN</dt>
<dd><p>
<em>PIN‐FORMED</em>
</p></dd>
<dt>PM</dt>
<dd><p>plasma membrane</p></dd>
<dt>REML</dt>
<dd><p>restricted maximum likelihood</p></dd>
<dt>RGA</dt>
<dd><p>root growth angle</p></dd>
<dt>RSA</dt>
<dd><p>root system architecture</p></dd>
<dt>RSRatio</dt>
<dd><p>root:shoot ratio</p></dd>
<dt>WT</dt>
<dd><p>wild type</p></dd>
</dl>
<section id="tpg270061-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>The uptake of water and nutrients from the soil is critical to support crop growth and productivity. With a rapidly growing global population and the increasing effects of climate change, there is a greater demand for crops that can produce more grain with less resources. Breeding crops with enhanced sustainability relies on access to genetic variation for morphological and physiological traits that influence resource‐use efficiency and adaptation to abiotic stress (Cronin et al., <a href="#tpg270061-bib-0011" class="usa-link" aria-describedby="tpg270061-bib-0011">2007</a>; Vanhala et al., <a href="#tpg270061-bib-0060" class="usa-link" aria-describedby="tpg270061-bib-0060">2004</a>). While there is still limited understanding of the genetics modulating crop development, significant exploration has been performed in model species Arabidopsis, which provides a powerful resource bank of candidate genes to surgically explore the various mechanisms of crop physiology and development. So far, the crop research community has made good progress to investigate the translation of gene targets identified in model species for aboveground architecture (Ahmar et al., <a href="#tpg270061-bib-0001" class="usa-link" aria-describedby="tpg270061-bib-0001">2024</a>; Huang et al., <a href="#tpg270061-bib-0022" class="usa-link" aria-describedby="tpg270061-bib-0022">2021</a>), leaving many promising targets for root system architecture (RSA) as a novel opportunity to improve nutrient‐ and water‐use efficiency. RSA has important functional implications as it is the primary interface for soil nutrient and water acquisition, along with the physical anchoring of the plant and providing resilience to lodging (Bishopp &amp; Lynch, <a href="#tpg270061-bib-0004" class="usa-link" aria-describedby="tpg270061-bib-0004">2015</a>). Among the most important determinants of RSA is the root growth angle (RGA), which is the angle at which the roots grow into the soil.</p>
<p>A key mechanism influencing RGA is how the roots respond to gravity, where a stronger gravitropic response elicits a steeper rooting angle and is often associated with improved drought adaptation through improved access to deep‐soil water (Ho et al., <a href="#tpg270061-bib-0021" class="usa-link" aria-describedby="tpg270061-bib-0021">2004</a>; Kitomi et al., <a href="#tpg270061-bib-0030" class="usa-link" aria-describedby="tpg270061-bib-0030">2020</a>). Root gravitropism is regulated by the phytohormone auxin, a ubiquitous and important chemical responsible for translating external environmental changes into a physiological response by the plant, allowing for rapid adaptation to its environment (Blilou et al., <a href="#tpg270061-bib-0005" class="usa-link" aria-describedby="tpg270061-bib-0005">2005</a>). This process is maintained by polar auxin transporters, establishing asymmetrical organ‐specific and organism‐wide concentration gradients that regulate cell elongation to enable root growth along the gravity vector. Removing the root cap through genetic ablation results in loss of gravity sensing in the roots and increased lateral rooting, a response consistent with auxin insensitivity or inhibition (Mirza et al., <a href="#tpg270061-bib-0041" class="usa-link" aria-describedby="tpg270061-bib-0041">1984</a>; Tsugeki &amp; Fedoroff, <a href="#tpg270061-bib-0058" class="usa-link" aria-describedby="tpg270061-bib-0058">1999</a>). The starch‐statolith hypothesis is the prevailing theory on how the root cells sense gravity, where dense starch granules named amyloplasts sediment on one side of the cell aligned with the gravity vector, triggering a signaling cascade (Stanga et al., <a href="#tpg270061-bib-0052" class="usa-link" aria-describedby="tpg270061-bib-0052">2011</a>). While still poorly understood, it is thought that this cascade involves the rearrangement of auxin transporters to reestablish the auxin maximum at the root tip.</p>
<p>Auxin distribution is facilitated by three major classes of transporters: the PIN‐FORMED (PIN) efflux carrier proteins, the ATP‐binding cassette (ABC)‐B/multi‐drug resistance/P‐glycoprotein (ABCB/MDR/PGP) subfamily of ABC transporters, and the AUXIN1/LIKE‐AUX1 (AUX/LAX) influx carrier proteins (Yue et al., <a href="#tpg270061-bib-0063" class="usa-link" aria-describedby="tpg270061-bib-0063">2015</a>). The ABCD/MDR/PGP transporters are a large family of transporters with diverse roles in the transport of a variety of substrates, where some studies have demonstrated their involvement in both influx and efflux with auxin (Geisler &amp; Murphy, <a href="#tpg270061-bib-0016" class="usa-link" aria-describedby="tpg270061-bib-0016">2006</a>). Members of the AUX/LAX family are polarly localized in expressing cells where they facilitate auxin entry into cells through passive diffusion, resulting in unidirectional distribution of auxin due to their asymmetric positioning on the plasma membrane (PM) (Swarup et al., <a href="#tpg270061-bib-0053" class="usa-link" aria-describedby="tpg270061-bib-0053">2001</a>). The most well‐described polar auxin efflux transporters are the PIN proteins, whose polarity is credited as the key determinant of auxin transport direction (Blilou et al., <a href="#tpg270061-bib-0005" class="usa-link" aria-describedby="tpg270061-bib-0005">2005</a>).</p>
<p>There are two main types of <em>PIN</em> genes: the “short” PINs and “long” PINs based on the size of the intracellular hydrophilic loop domain that separates the flanking transmembrane domains (Luschnig &amp; Vert, <a href="#tpg270061-bib-0036" class="usa-link" aria-describedby="tpg270061-bib-0036">2014</a>). Short PINs play an essential role in maintaining intracellular auxin concentrations by regulating transport between the endoplasmic reticulum and cytosol (Geisler &amp; Murphy, <a href="#tpg270061-bib-0016" class="usa-link" aria-describedby="tpg270061-bib-0016">2006</a>), whereas long PINs regulate the efflux of auxin out of the cell through asymmetrical localization on the PM. This localization changes in response to environmental stimuli by cycling the receptors between the PM and endosomal compartments (Kirschner et al., <a href="#tpg270061-bib-0028" class="usa-link" aria-describedby="tpg270061-bib-0028">2018</a>). This is established through unique and regulated polar localization of these transporters along the PM, allowing directional transport of auxin and forming unique local distributions for organ‐specific and organism‐wide responses (Blilou et al., <a href="#tpg270061-bib-0005" class="usa-link" aria-describedby="tpg270061-bib-0005">2005</a>). Arabidopsis has a total of nine known PIN transporters, whereas higher order species such as cereals often have an increased number of PIN transporters. For example, barley has 13 putative PIN transporters with Hv‐PIN1 and Hv‐PIN2 being homologous to Arabidopsis long PINs At‐PIN1 and At‐PIN2. In contrast, Hv‐PIN5, Hv‐PIN8, and Hv‐PIN9 exhibit short hydrophilic regions. While monocots do not encode At‐PIN3, At‐PIN4, and At‐PIN7 homologues, barley uniquely has four Hv‐PIN10s, which display atypical structure of the hydrophilic loop being noncentral (Hv‐PIN10b2) or missing entirely (Hv‐PIN10a, Hv‐PIN10b1, and Hv‐PIN10b3) (Kirschner et al., <a href="#tpg270061-bib-0028" class="usa-link" aria-describedby="tpg270061-bib-0028">2018</a>).</p>
<p>The developmental plasticity of plants is underpinned by direct and indirect effects. Direct effects controlling organ formation are primarily the consequence of local hormone interactions. Indirect effects can be seen in the dynamic balance of allometry where growth of one organ influences another. For example, the canopy has an allometric relationship with the root system, where increases in the canopy biomass correlate with increased root biomass (Borrell et al., <a href="#tpg270061-bib-0006" class="usa-link" aria-describedby="tpg270061-bib-0006">2022</a>). As <em>PIN2</em> expression is almost exclusively within the roots, the possibility of root‐specific architectural changes, independent of aboveground and belowground allometry, provides an interesting subject for investigation (Müller et al., <a href="#tpg270061-bib-0042" class="usa-link" aria-describedby="tpg270061-bib-0042">1998</a>). PIN2 is a known regulator of root gravitropism; it localizes to the shootward side of epidermal cells and mediates the auxin flow from the root tip toward the elongation zone where the auxin response is activated and inhibits cell elongation (Casimiro et al., <a href="#tpg270061-bib-0008" class="usa-link" aria-describedby="tpg270061-bib-0008">2001</a>; Kleine‐Vehn et al., <a href="#tpg270061-bib-0031" class="usa-link" aria-describedby="tpg270061-bib-0031">2008</a>). In <em>Arabidopsis</em>, <em>pin2</em> mutants show defective root gravitropism because of the mutants’ inability to accumulate auxin in the roots, a loss of curving in RSA formation.</p>
<p>While the PIN2 pathway is well‐studied in model species, knowledge of the extent to which <em>PIN2</em> function is preserved across species, particularly monocots, is lacking. In this study, we explore the function and potential of targeted manipulation of the <em>PIN2</em> orthologue (Hv<em>‐PIN2</em>) in the major cereal crop, barley (<em>Hordeum vulgare</em>). We created a knockout in barley using CRISPR/Cas9 (where CRISPR is clustered regularly interspaced short palindromic repeats) genome editing and adopted a holistic plant phenotyping approach that measured a range of root and shoot traits across growth stages to provide new insights into the function and potential utility of genome editing to support crop improvement.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270061-sec-0020"><h3 class="pmc_sec_title">Core Ideas</h3>
<ul id="tpg270061-list-0001" class="list" style="list-style-type:disc">
<li><p>
<em>PIN‐FORMED</em> (PIN2) regulates root system architecture in barley.</p></li>
<li><p>Gene editing was used to create novel genetic variants.</p></li>
<li><p>Outcomes show wider root systems without impacting canopy development.</p></li>
<li><p>In‐depth root phenotypic platforms explored influence of this transporter holistically throughout the plant.</p></li>
</ul></section></section></section><section id="tpg270061-sec-0030"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="tpg270061-sec-0040"><h3 class="pmc_sec_title">2.1. CRISPR/Cas9 editing of <em>PIN2</em> in barley</h3>
<p>Genome editing of <em>PIN2</em> (<em>HORVU7Hr1G110470</em>) in barley was performed using cv. Golden Promise via particle bombardment following the tissue culture protocol described in Harwood et al. (<a href="#tpg270061-bib-0064" class="usa-link" aria-describedby="tpg270061-bib-0064">2009</a>), with modifications and gene editing strategy explained in Massel et al. (<a href="#tpg270061-bib-0040" class="usa-link" aria-describedby="tpg270061-bib-0040">2022</a>). Wild‐type (WT) embryos that did not go through transformation remained on nonselective regeneration media. Regenerants that produced shoots at least 2 cm long were subsequently transferred to rooting media until development of adequate roots before sowing in glasshouse.</p>
<p>Methods for selection of gRNA are described in Massel et al. (<a href="#tpg270061-bib-0040" class="usa-link" aria-describedby="tpg270061-bib-0040">2022</a>). Using those parameters, two gRNAs (G1: 5′‐GTGGCCAACAAGTTCAAGGG‐3′, G2: 5′‐GCCGGTAGTCCATGGCGTAG‐3′) targeting coding regions of the <em>PIN2</em> orthologue were selected and cloned into the plasmid system. The gRNA target sequences were placed within the NPTII selection vector for co‐bombardment alongside a plasmid containing zCas9 under the maize ubiquitin promoter. Following isolation of barley immature embryos, transformation was performed via microprojectile co‐bombardment using biolistic techniques (Liu &amp; Godwin, <a href="#tpg270061-bib-0034" class="usa-link" aria-describedby="tpg270061-bib-0034">2012</a>) using the particle inflow gun delivery system with modifications to optimize the system for barley where the vacuum pressure remained at −90 kHa with helium being shot at a pressure of 800 kPa (Massel et al., <a href="#tpg270061-bib-0040" class="usa-link" aria-describedby="tpg270061-bib-0040">2022</a>). Immature embryos were bombarded approximately 5–7 days post isolation with approximately 25–30 embryos per shot before being placed on callus induction medium.</p>
<p>Following successful tissue culture, leaf tissue samples were collected from the putatively edited lines for DNA extraction, polymerase chain reaction (PCR), and DNA Sanger sequencing to identify the nature of the edit. PCR was performed using MyTaq (Bioline) DNA Polymerase in reactions containing 30 ng of DNA template, 1X MyTaq Buffer, 0.5uM of the forward and reverse primers (Table <a href="#tpg270061-tbl-0001" class="usa-link">1</a>) and run following the manufacturer's instructions. PCR reactions were run on 1.2% tris‐acetate‐EDTA agarose gel with a standard 1 kb ladder (New England BioLabs) to confirm formation of products. PCR clean‐up was performed with ExoSAP‐IT (Thermo Fisher Scientific) before being sent for Sanger sequencing. CRISPR analysis tool Synthego (Synthego ICE Analysis tool v3; Synthego, <a href="#tpg270061-bib-0055" class="usa-link" aria-describedby="tpg270061-bib-0055">2024</a>), along with sequence alignment tools, were used to determine types of gene edit. Transmembrane prediction software TMHMM was used to estimate resultant impact on protein function (Krogh et al., <a href="#tpg270061-bib-0032" class="usa-link" aria-describedby="tpg270061-bib-0032">2001</a>).</p>
<section class="tw xbox font-sm" id="tpg270061-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>Primers used in PCR to detect presence of gene and potential edits following transformation.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Target gene</th>
<th align="left" rowspan="1" colspan="1">Forward/ reverse</th>
<th align="left" rowspan="1" colspan="1">5′ Top 3′ sequence</th>
<th align="left" rowspan="1" colspan="1">Tm (°C)</th>
<th align="left" rowspan="1" colspan="1">Ta (°C)</th>
<th align="left" rowspan="1" colspan="1">Product size (bp)</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="2" align="left" colspan="1">Hv‐<em>PIN2</em>
</td>
<td align="left" rowspan="1" colspan="1">Forward</td>
<td align="left" rowspan="1" colspan="1">GCGTTGGTGGGGCATCTTCA</td>
<td align="left" rowspan="1" colspan="1">61</td>
<td rowspan="2" align="left" colspan="1">62</td>
<td rowspan="2" align="left" colspan="1">1116</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Reverse</td>
<td align="left" rowspan="1" colspan="1">TGTGCAGCTCCTTGTTCGAGT</td>
<td align="left" rowspan="1" colspan="1">62</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270061-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270061-tbl1-note-0001"><p>Abbreviations: PCR, polymerase chain reaction; PIN, <em>PIN‐FORMED</em>; Ta, temperature of annealing; Tm, temperature of melting.</p></div></div></section></section><section id="tpg270061-sec-0050"><h3 class="pmc_sec_title">2.2. Phenotyping seminal root angle</h3>
<p>The “clear pot” method, routinely applied in wheat and barley root research (Richard et al., <a href="#tpg270061-bib-0045" class="usa-link" aria-describedby="tpg270061-bib-0045">2015</a>; Robinson et al., <a href="#tpg270061-bib-0047" class="usa-link" aria-describedby="tpg270061-bib-0047">2016</a>), was used to evaluate the <em>pin2</em> knockout and WT barley lines for seminal root angle at the seedling stage. Seminal root angle is considered a useful proxy trait that can be assayed in a high‐throughput manner and is often associated with the root angle of mature plants in the field (Alahmad et al., <a href="#tpg270061-bib-0003" class="usa-link" aria-describedby="tpg270061-bib-0003">2019</a>; Voss‐Fels et al., <a href="#tpg270061-bib-0061" class="usa-link" aria-describedby="tpg270061-bib-0061">2018</a>). Plants were watered daily and housed in a temperature‐controlled PC2 glasshouse, with 22°C /17°C (day/night) temperature cycle and an 18h/6h (day/night) photoperiod with 450 nm and 660 nm 5700K white LED growth lights.</p>
<p>All edited lines were assessed and compared to the WT as well as a tissue culture control plant (i.e., unedited line regenerated from tissue culture). The edited lines screened included two mutants with varied edits (<em>pin2‐1</em> and <em>pin2‐2</em>, Figure <a href="#tpg270061-fig-0001" class="usa-link">1</a>). Seeds were sown in 4‐L ANOVApot of 200 mm diameter and 190 mm in height using a randomized design ensuring 12 reps for each line, with a maximum of 24 seeds per pot. Seeds were sown at a depth of 2 cm at a 45° angle, with the embryonic axis of each seed oriented downward, every 2.5 cm along the transparent wall of the pot to result in 24 barley seeds per pot (Robinson et al., <a href="#tpg270061-bib-0047" class="usa-link" aria-describedby="tpg270061-bib-0047">2016</a>). UQ‐23 potting mix was used, containing 70% composted pine bark 0–5 mm and 30% cocoa peat.</p>
<figure class="fig xbox font-sm" id="tpg270061-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/643b/12328839/d0092b661567/TPG2-18-e70061-g003.jpg" loading="lazy" id="jats-graphic-1" height="635" width="700" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270061-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Summary of barley <em>pin2</em> knock‐out lines created using genome editing and resultant protein structure changes. (a) Map of alleles for the long‐looped <em>PIN‐FORMED</em> (<em>PIN</em>) gene showing generalized edits to protein sequence. Scale represents the number of amino acids. Specifically, <em>pin2‐1</em> displayed an early stop codon following G1, <em>pin2‐2</em> displayed a 749 bp drop out along with a frame shift for the remaining coding region, and <em>pin2‐3</em> showed the same editing event‐1 nt deletion at G1 and was homozygous. Notably, <em>pin2‐4</em> displayed no genome editing at <em>PIN2</em> and was treated as the tissue culture (TC) control for future experiments. (b) Outcomes table of transgenics, with the number of alleles edited at each guide location and the resulting PIN2 protein. G1 and G2, guide sites one and two for CRISPR/Cas9 (where CRISPR is clustered regularly interspaced short palindromic repeats) genome editing; HL, hydrophilic loop; TMD, transmembrane domain. NA indicates no change to sequence.</p></figcaption></figure><p>The pots were thoroughly watered the day before sowing, and no additional water was provided, which ensured the orientation of seeds did not shift during germination. After sowing, each clear pot was placed inside a black pot of the same dimension to prevent light from influencing root development. Images of roots were captured at both 3 and 5 days after sowing (DAS) using a camera (iPhone 13 Pro Max). Analysis for seminal root angle and number was determined using ImageJ (Schneider et al., <a href="#tpg270061-bib-0050" class="usa-link" aria-describedby="tpg270061-bib-0050">2012</a>), measuring the average deviation angle of the first pair of seminal roots to the vertical plane (Christopher et al., <a href="#tpg270061-bib-0010" class="usa-link" aria-describedby="tpg270061-bib-0010">2013</a>; Richard et al., <a href="#tpg270061-bib-0045" class="usa-link" aria-describedby="tpg270061-bib-0045">2015</a>; Robinson et al., <a href="#tpg270061-bib-0047" class="usa-link" aria-describedby="tpg270061-bib-0047">2016</a>).</p></section><section id="tpg270061-sec-0060"><h3 class="pmc_sec_title">2.3. Evaluating gravitropic response</h3>
<p>To visualize and monitor the gravitropic response, <em>pin2</em> seeds (<em>pin2‐1</em> and <em>pin2‐2</em>) were germinated in glass chambers (210 mm × 297 mm) lined with black serviettes that were watered and rolled flat to remove air bubbles prior to placing seeds. The developing plants were grown in darkness at 25°C. After 4 days, images of the roots were captured and then chambers were rotated by 90°. Root images were captured at 0, 4, 6, 8, and 24 h after rotation, with the root position marked at each time point. The responsiveness of the root system to gravity was quantified by measuring the change in angle of the root tips for seminal roots and the radicle over time (Kirschner et al., <a href="#tpg270061-bib-0027" class="usa-link" aria-describedby="tpg270061-bib-0027">2021</a>).</p>
<p>For data analysis, the root angle values for the radicle root tip were measured in relation to the horizontal, with the angle following rotation set to 0°. All root angle values were obtained with the software ImageJ (Schneider et al., <a href="#tpg270061-bib-0050" class="usa-link" aria-describedby="tpg270061-bib-0050">2012</a>). The average root angle at each time point and change in root angle over time were calculated and compared using a two‐tailed <em>t</em>‐test.</p></section><section id="tpg270061-sec-0070"><h3 class="pmc_sec_title">2.4. Evaluating root system architecture in rhizoboxes</h3>
<p>The RSA of the <em>pin2</em> mutants and WT lines were assessed in a series of rhizobox experiments. RSA was assessed by determining the root angle, total biomass, shoot biomass, root biomass, and root density at various depths. This set‐up uses wooden boxes lined with a plastic sheet and placed upright on a bench with capillary matting at the base of the bench laid out in a randomized complete block design. The first two experiments were conducted using the rhizobox platform described by Kang et al. (<a href="#tpg270061-bib-0024" class="usa-link" aria-describedby="tpg270061-bib-0024">2024</a>) on <em>pin2‐2</em> and WT. The chambers were 30 cm (width) × 5 cm (thickness) × 80 cm (depth) and were filled with potting mix containing 6 g/L of Osmocote mixed in thoroughly. Plants were watered daily and housed in a temperature‐controlled PC2 glasshouse, with 22°C/17°C (day/night) temperature cycle and standard long‐day conditions using a 18h/6h (light/dark) photoperiod. Two different planting densities were tested, first with three plants per chamber and second with one plant per chamber. In the first experiment, six to nine seeds were evenly dispersed throughout the width of the chamber near the surface of the potting mix (5‐cm depth). After emergence, each chamber was thinned down to three plants per box. In the second rhizobox experiment, a total of three seeds were sown (depth of 5 cm) per rhizobox with one plant retained after emergence. The experiments in these rhizoboxes were harvested after 4 weeks, as the root systems had almost reached the base of the chambers. Two days prior to harvesting, the plants were heavily watered and were not watered the day prior to harvest. Upon harvesting, in both rhizobox experiments, each chamber was divided into three equal segments (26‐cm depth) to represent “upper,” “middle,” and “deep” layers. The potting media‐containing roots sampled from each section were collected and bagged individually. Each root segment was hand washed to remove potting mix, placed in a paper bag, and dried at 80°C before weighing.</p>
<p>While the plants grew well in the 30 cm width boxes, the narrow width could not accommodate the outgrowth of the mutant root system as the roots reached the sides of the chamber early in the plant growth cycle, resulting in either “training” of the roots downward or “bouncing” off the wall and back into the substrate. This reduced the ability to accurately quantify differences in RSA. To accommodate the wider root system, “double width” rhizoboxes (60 cm × 5 cm × 80 cm) were constructed to improve the visualization of the entire RSA.</p>
<p>In the double width rhizboxes, plants were phenotyped after 42 days of growth, which enabled the analysis of root systems that were unimpeded and more extensively developed. The same potting mix, growing, and plant maintenance parameters were used and included assessing both <em>pin2‐1</em> and <em>pin2‐2</em> compared to the WT to validate the trait. Two trials were performed using the double width rhizoboxes. The first included six replicates each for <em>pin2‐</em>2 and WT, and the second trial consisted of four replicates each for <em>pin2‐2, pin2‐1</em>, and WT. For the double width rhizobox experiment, a total of three seeds were sown (depth of 5 cm), which were later thinned to one plant per rhizobox. Thus, it was anticipated that growing barley plants in these wider rhizoboxes would provide a clearer visualization of the root system branching and distribution down to a depth of 80 cm and root:shoot biomass partitioning in a setting that more closely resembled field conditions. The double width rhizobox was analyzed by separating the chamber and image into nine biomass sections (3 × 3 sections of 20 cm × 26 cm). As the root structures of the individual plants in the wide rhizoboxes could not be separated, the root biomass measurements represented the total root biomass produced by the three plants.</p>
<p>To synchronize germination, a cold treatment was performed by placing the seed in a Petri dish lined with a moist paper towel overnight at 4°C, then plates were moved to room temperature (22°C) for 24 h prior to sowing. The rhizoboxes were well‐watered daily by manually adding water to the top until drainage was observed from the bottom. During the experiment, a leaf sample from each plant in the box was taken for DNA extraction to confirm the genotype of the plants.</p>
<p>The root system was photographed for image analysis, first converted into a black and white image to be uploaded into RhizoVision Explorer. This software was used to quantify proportion of roots with shallow, medium, and steep root angles grouping by those &lt;30°, &lt;60°, and &lt;90° from the medial axis (Seethepalli et al., <a href="#tpg270061-bib-0051" class="usa-link" aria-describedby="tpg270061-bib-0051">2020</a>). The suggested default parameters for “whole root” analysis were used, with an image thresholding level of 120, an edge smoothing threshold of 2, and root pruning threshold of 1. To investigate potential changes in aboveground biomass, the entire shoot sample for each plant was removed and placed in individually labeled bags.</p></section><section id="tpg270061-sec-0080"><h3 class="pmc_sec_title">2.5. Evaluating aboveground traits at physiological maturity</h3>
<p>To further explore any shoot trade‐offs associated with <em>pin2</em> in barley, both the <em>pin2‐1</em> and <em>pin2‐2</em> edited lines alongside the WT were grown under controlled conditions and evaluated at physiological maturity. Plants were sown in 1‐L pots at a density of one plant per pot using eight reps per genotype arranged in a randomized complete block design. Plants were grown under stress‐free conditions using a temperature regime of 22°C/17°C (day/night) and a standard long‐day photoperiod of 18h/6h (light/dark). At maturity, each plant was partitioned into spikes, stems with leaves, and grains. Plant height, spike length, and the number of seeds per spike were recorded for the primary tiller of each plant. The total number of fertile tillers (grain‐producing tillers) and days to flowering were recorded for each plant.</p></section><section id="tpg270061-sec-0090"><h3 class="pmc_sec_title">2.6. Analysis of root and shoot phenotype data</h3>
<p>To analyze data from the gravitropism experiment, the root angles obtained by <em>pin2</em> edited line and WT line were compared using parametric <em>t</em>‐tests and one‐way analysis of variances with multiple comparisons where <em>H</em>
<sub>0</sub> &gt; 0.05 using GraphPad Prism (v9.3.1).</p>
<p>All root and shoot data collected in glasshouse phenotyping experiments, including clear pot, rhizobox, and maturity studies, were analyzed using a linear mixed model (LMM) to account for spatial variation and maximize the genetic variance captured in each trait‐specific model. The LMM was fitted with the following equation:
</p>
<table class="disp-formula p" id="tpg270061-disp-0001"><tr><td class="formula"><math id="jats-math-1" display="block"><mrow><mrow><mi>y</mi><mo linebreak="badbreak">=</mo><mi>X</mi><mi mathvariant="normal">τ</mi><mo linebreak="goodbreak">+</mo><mi>Z</mi><mi>u</mi><mo linebreak="goodbreak">+</mo><mi mathvariant="normal">ε</mi><mo>,</mo></mrow></mrow></math></td></tr></table>
<p>where <em>X</em> and <em>Z</em> are design matrices associated with fixed (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline"><mrow><mi mathvariant="normal">τ</mi></mrow></math></span>) and random (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline"><mrow><mi>u</mi></mrow></math></span>) effects and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline"><mrow><mi mathvariant="normal">ε</mi></mrow></math></span> is the residual error. A custom LMM was fit to each trait measured across the glasshouse root and shoot experiments, with the best model fit determined by the Wald test statistics (for fixed effect terms only), the highest restricted maximum likelihood (REML) loglikelihood value, and the smallest Akaike information criterion (Akaike, <a href="#tpg270061-bib-0002" class="usa-link" aria-describedby="tpg270061-bib-0002">1974</a>; Kenward &amp; Roger, <a href="#tpg270061-bib-0025" class="usa-link" aria-describedby="tpg270061-bib-0025">1997</a>). The specific design terms fitted as random and fixed effects in the LMM for each trait are detailed in Table <a href="#tpg270061-supinfo-0001" class="usa-link">S1</a>. The auto‐regressive residual variance structure in the order of 1 was found to fit best across all traits and was thus fitted across all trait LMMs. A generalized measure of broad‐sense heritability was calculated for each trait to quantify the proportion of phenotypic variance that is attributed to genetic effects. This was determined using the following equation:
</p>
<table class="disp-formula p" id="tpg270061-disp-0002"><tr><td class="formula"><math id="jats-math-5" display="block"><mrow><mrow><msubsup><mi>h</mi><mi>g</mi><mn>2</mn></msubsup><mo linebreak="badbreak">=</mo><mn>1</mn><mo linebreak="goodbreak">−</mo><mfrac><msub><mi>A</mi><mrow><mi>t</mi><mi>t</mi></mrow></msub><mrow><mn>2</mn><msub><mi mathvariant="normal">γ</mi><mi>v</mi></msub></mrow></mfrac><mo>,</mo></mrow></mrow></math></td></tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline"><mrow><mrow><msub><mi>A</mi><mrow><mi>t</mi><mi>t</mi></mrow></msub><mspace width="0.28em"></mspace></mrow></mrow></math></span> is the mean predicted error variance and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline"><mrow><mrow><msub><mi mathvariant="normal">γ</mi><mi>v</mi></msub><mspace width="0.28em"></mspace></mrow></mrow></math></span> is the genetic variance (Cullis et al., <a href="#tpg270061-bib-0012" class="usa-link" aria-describedby="tpg270061-bib-0012">2006</a>). The LMMs were fit using ASReml‐R in the R software environment (Posit Team, <a href="#tpg270061-bib-0065" class="usa-link" aria-describedby="tpg270061-bib-0065">2025</a>), and best linear unbiased predictions (BLUPs) were calculated from the variance components of the model (Butler et al., <a href="#tpg270061-bib-0007" class="usa-link" aria-describedby="tpg270061-bib-0007">2017</a>). Figures displaying and comparing trait BLUPs between <em>pin2</em> edited lines and the WT were created using the ggplot package in R (Wickham et al., <a href="#tpg270061-bib-0062" class="usa-link" aria-describedby="tpg270061-bib-0062">2016</a>).</p></section></section><section id="tpg270061-sec-0100"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270061-sec-0110"><h3 class="pmc_sec_title">3.1. Gene edited <em>pin2</em> knockout lines in barley</h3>
<p>A CRISPR/Cas9 strategy was used to generate several mutant alleles in the barley <em>PIN2</em> gene. The <em>PIN2</em> orthologue in barley was identified using Phytozome, showing 92.7% and 92.5% protein homology to <em>Sorghum bicolor</em> (Sb‐<em>PIN11</em>) and <em>Oryza sativa</em> (Os‐<em>PIN2</em>) respectively, with both being orthologues of At‐<em>PIN2</em> (Goodstein et al., <a href="#tpg270061-bib-0017" class="usa-link" aria-describedby="tpg270061-bib-0017">2012</a>). Four stable transgenics were created that contained both the bombarded plasmids showing a range of gene editing outcomes (Figure <a href="#tpg270061-fig-0001" class="usa-link">1</a>). Of these, two lines were shown to contain edits in at least one of the target sites. Sanger sequencing was used to verify the editing outcomes, where the first showed a homozygous‐1 nt deletion, resulting in a frameshift of the coding region. This leads to a truncated peptide fragment that only contains 64/627 of the WT protein sequence before stopping and will be referred to as <em>pin2‐1</em>. The second mutant was biallelic and edited at both sites and was biallelic where one allele was a −749 bp deletion in the sequence between the two guide sites resulting in a frameshift mutation with 65/627 of the original peptides and 377 amino acids before stopping, while the other allele only had the G1 edit causing an early stop codon. The homozygous large deletion will be referred to as <em>pin2‐2</em>, whereas segregation for the homozygous mutant resulted in <em>pin2‐3</em>, an edited line with an early stop codon producing a peptide 65/627 of the original. Lastly, one line showed no evidence of editing at target sites and was described as <em>pin2‐4</em>. Editing efficiency was approximately 75% at G1 and 25% at G2. Transmembrane prediction using TMHMM displayed a loss of the transmembrane domains before the central hydrophilic loop, leading to complete loss of essential domains for the function of PIN proteins as an auxin efflux carrier (Figure <a href="#tpg270061-supinfo-0001" class="usa-link">S1</a>).</p></section><section id="tpg270061-sec-0120"><h3 class="pmc_sec_title">3.2. Loss of PIN2 produced altered root growth angle</h3>
<p>To determine if Hv‐<em>PIN2</em> plays a role in root growth orientation, the seminal root angle was evaluated using the clear pot system. This was performed on two homozygous knockout lines, <em>pin2‐1</em> (<em>n</em> = 9) and <em>pin2‐2</em> (<em>n</em> = 7), alongside a tissue culture null (<em>n</em> = 11) and WT (<em>n</em> = 16) as controls. The seedlings were imaged at days 3, 5, and 12. Both <em>pin2‐1</em> and <em>pin2‐2</em> displayed a significantly wider seminal root angle, from 38° in WT to 85° and 72° between <em>pin2‐1</em> and <em>pin2‐2</em>, respectively, displaying similar patterns of agravitropism as those seen in the previous gravitropism assay, where roots appear “aimless,” growing in the direction that they emerged (Figure <a href="#tpg270061-fig-0002" class="usa-link">2b</a>).</p>
<figure class="fig xbox font-sm" id="tpg270061-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/643b/12328839/80a3b4b5c2d7/TPG2-18-e70061-g001.jpg" loading="lazy" id="jats-graphic-3" height="326" width="560" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270061-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Barley <em>pin2</em> knock‐out lines display significantly wider seminal root angle. (a) Representative image of wild type (WT) seminal root angle illustrating how the measurements for seminal root angle were recorded in this study with representative image of WT compared to <em>pin2‐2</em> (pink). Measurements were recorded in ImageJ for the first pair of seminal roots to the vertical axis and the two angles summed to give a representative measurement of root angle. (b) Comparison of seminal root angle phenotypes for Golden Promise WT carrying <em>PIN‐FORMED2</em> (<em>PIN2</em>), tissue culture (TC), null (<em>pin2‐4</em>), and transgenic knock‐out lines for <em>pin2</em> (<em>pin2‐1</em> and <em>pin2‐2</em>).</p></figcaption></figure></section><section id="tpg270061-sec-0130"><h3 class="pmc_sec_title">3.3. 
<em>PIN2</em> controls root gravitropism in barley</h3>
<p>To evaluate the mechanism behind the increased root angle in <em>pin2</em> mutants, root gravitropism assays were performed. Plates carrying seedlings were rotated 90° following germination to gravistimulate the roots and visualized over the course of 24 h. The rate of change in root angle over time is representative of both gravitropic response and strength of its response. The roots of <em>pin2‐2</em> (<em>n</em> = 7) lacked a gravitropic response, where the roots continued to grow from their initial sprouting (Figure <a href="#tpg270061-fig-0003" class="usa-link">3A</a>). On the other hand, WT (<em>n</em> = 8) roots showed a linear rate of change to root angle toward the gravity vector at an average of 0.83°/h. The shoots appeared to respond to gravity, with both WT and <em>pin2‐2</em> curving upward following rotation (Figure <a href="#tpg270061-fig-0003" class="usa-link">3B</a>, Table <a href="#tpg270061-supinfo-0001" class="usa-link">S3</a>).</p>
<figure class="fig xbox font-sm" id="tpg270061-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/643b/12328839/25051df5114b/TPG2-18-e70061-g002.jpg" loading="lazy" id="jats-graphic-5" height="206" width="400" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270061-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Barley <em>pin2</em> knock‐out lines display root‐specific agravitropism at the seedling stage. (a) Change in root angle for <em>pin2</em> knock‐out lines (<em>pin2‐1</em> and <em>pin2‐2</em>) (red) and wild type (WT) (blue) over 24 h from seedling gravitropic assay conducted in glass panels. Root angle was measured from the radicle in relation to the horizontal axis, and measurements were taken every 2 h for the first 8 h, then again after 24 h (<em>n</em> = 7). The derivative of root angle over time was taken as a standardized measurement for strength of gravitropic response. Bars show standard error of the mean change in seminal root angle over time. (b) Images of WT and <em>pin2</em> seedlings captured 24 h after rotation.</p></figcaption></figure></section><section id="tpg270061-sec-0140"><h3 class="pmc_sec_title">3.4. Barley <em>pin2</em> knockout lines display wider and shallower root architecture</h3>
<p>To explore the role of <em>PIN2</em> on RSA, a series of rhizobox experiments were conducted, which compared <em>pin2‐1</em> and the WT. Initial phenotyping was conducted using narrow rhizoboxes at two planting densities: one plant per box (15 cm row spacing) and three plants per box (5 cm row spacing) at 28 DAS. In the lower density experiment, results revealed that <em>pin2‐1</em> produced approximately 40% more root biomass in the upper section (S1) of the rhizoboxes compared to the WT (<em>p</em> &lt; 0.05, Figure <a href="#tpg270061-supinfo-0001" class="usa-link">S2c</a>). No difference in shoot biomass or total biomass was detected (<em>p</em> &gt; 0.1, Figure <a href="#tpg270061-supinfo-0001" class="usa-link">S2a</a>).</p>
<p>In the high‐density experiment, <em>pin2‐1</em> showed a reduction in total root biomass of approximately 30% (<em>p</em> = 0.07, Figure <a href="#tpg270061-supinfo-0001" class="usa-link">S2a</a>). Again, there was no change to total aboveground biomass between samples (<em>p</em> &gt; 0.1). The major difference in root distribution was observed in the lower section (S3), where <em>pin2‐1</em> produced significantly less root biomass in both low‐ and high‐density experiments (Figure <a href="#tpg270061-supinfo-0001" class="usa-link">S2</a>). As a result of the reduction in root biomass, <em>pin2‐1</em> showed a significantly lower root:shoot ratio (RSRatio) compared to the WT at 1‐planting density and trending reduction at 3‐planting density (<em>p</em> &lt; 0.001, <em>p</em> = 0.057).</p>
<p>It was evident that the narrow width of the rhizoboxes in the first set of experiments constrained our ability to accurately phenotype RSA of the <em>pin2</em> mutants as the roots hit the sides and were trained downward (Figure <a href="#tpg270061-supinfo-0001" class="usa-link">S2b</a>). Thus, rhizoboxes with double the width were used to subsequently phenotype the lines in a scenario where root growth was less impeded. In the wider rhizoboxes, <em>pin2‐1</em> and <em>pin2‐2</em> exhibited a wider and shallower root system, as demonstrated by the increased root biomass produced in the upper wide sections (<a href="#tpg270061-supinfo-0001" class="usa-link">S1</a> and <a href="#tpg270061-supinfo-0001" class="usa-link">S3</a>; <em>p</em> &lt; 0.1 and <em>p</em> &lt; 0.05, respectively, Figure <a href="#tpg270061-fig-0004" class="usa-link">4</a>). The upper middle section (S2) (i.e., directly below the base of the plant) showed no significant change in root biomass, whereas the middle section (S5) showed a significant reduction in root biomass for both lines (<em>p</em> &lt; 0.05). In the wide rhizoboxes there was no significant change to aboveground (<em>p</em> &gt; 0.1) or belowground biomass (<em>p</em> &gt; 0.1) in <em>pin2‐1</em> and <em>pin2‐2</em> compared to the WT. However, image analysis of the whole root system using RhizoVision Explorer showed that <em>pin2‐1 and pin2‐2</em> displayed a higher proportion of roots classified as “shallow” (<em>p</em> &lt; 0.05), and a significant increase in biomass in the upper sections relative to WT, indicative of shallow rooting behavior (Figure <a href="#tpg270061-supinfo-0001" class="usa-link">S3</a>).</p>
<figure class="fig xbox font-sm" id="tpg270061-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/643b/12328839/82f567e3754e/TPG2-18-e70061-g005.jpg" loading="lazy" id="jats-graphic-7" height="465" width="600" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270061-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>PIN‐FORMED2 (PIN2) in barley regulates downward root growth leading to steeper and narrower root architecture. Spatial root system configuration of the wild type (WT) and <em>pin2</em> barley lines (<em>pin2‐1</em> and <em>pin2‐2</em>) in the wide rhizobox chambers. Left images show representative images of the root systems for both the WT (blue) and <em>pin2‐1</em> (red). Solid lines indicate each section (S1–S9). The mean ratio of dry root biomass produced in each section to total root biomass is displayed as bar charts on the right. Significance was calculated by nonparametric <em>t</em>‐test. <em>N</em> = 10 for <em>pin2‐1</em> and WT. <em>N</em> = 4 for <em>pin2‐2</em>. *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01.</p></figcaption></figure></section><section id="tpg270061-sec-0150"><h3 class="pmc_sec_title">3.5. 
<em>PIN2</em> has a limited role in aboveground development</h3>
<p>Canopy traits were assessed during the rhizobox studies to explore the allometric relationships or potential aboveground trade‐offs in these lines. Across both rhizobox trials, no significant difference in canopy biomass was detected despite the significantly reduced root biomass. Early vigor measurements conducted on the three plants per rhizobox trial revealed no significant difference in tiller number (<em>p</em> &gt; 0.1) or leaves per tiller (<em>p</em> &gt; 0.1). Tiller number was approximately 15 across WT and <em>pin2‐1</em> with an average of five leaves per tiller (Figure <a href="#tpg270061-supinfo-0001" class="usa-link">S3</a>). Similarly, leaf size was consistent across mutant and WT (Figure <a href="#tpg270061-supinfo-0001" class="usa-link">S3</a>). The root:shoot biomass ratio was significantly altered in the <em>pin2</em> mutant from 0.7 in WT to 0.5 at higher density (5 cm) row spacing and from 0.5 in WT to 0.4 in <em>pin2</em> at lower density (15 cm) row spacing, representing a general reduction of root biomass of 20% (<em>p</em> &lt; 0.1, <em>p</em> &lt; 0.01, Figure <a href="#tpg270061-supinfo-0001" class="usa-link">S2a,c</a>).</p>
<p>A maturity study was conducted (<em>N</em> = 8) to explore canopy traits at maturity, where no significant changes to aboveground canopy biomass, number of fertile tillers at maturity, and days to flowering were observed (<em>p</em> &gt; 0.05). Following spatial analysis, no genotypic effects were associated with these traits between WT and either of the <em>pin2</em> mutants (Figure <a href="#tpg270061-supinfo-0001" class="usa-link">S4</a>).</p></section></section><section id="tpg270061-sec-0160"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<section id="tpg270061-sec-0170"><h3 class="pmc_sec_title">4.1. 
<em>PIN2</em> is essential for root gravitropism and root angle</h3>
<p>The spatial and temporal distribution of roots via branching, elongation, and curving all affect the ability of the crop to extract water and nutrients from the soil (Manschadi et al., <a href="#tpg270061-bib-0039" class="usa-link" aria-describedby="tpg270061-bib-0039">2006</a>; Rich &amp; Watt, <a href="#tpg270061-bib-0044" class="usa-link" aria-describedby="tpg270061-bib-0044">2013</a>). A substantial body of research suggests that optimized RSA can improve water acquisition with minimal metabolic cost, potentially improving crop yields under both favorable and adverse conditions (Uga, <a href="#tpg270061-bib-0059" class="usa-link" aria-describedby="tpg270061-bib-0059">2021</a>). Specifically, variation in gravitropism can affect RGA, altering root anchoring and exploration of different soil layers to acquire water and nutrients, as shown for <em>ENHANCED GRAVITROPISM 2</em> (<em>EGT2</em>) in barley and wheat (Kirschner et al., <a href="#tpg270061-bib-0027" class="usa-link" aria-describedby="tpg270061-bib-0027">2021</a>). However, our knowledge of all the genes involved in orchestrating root growth and gravitropism is limited, particularly in cereals.</p>
<p>The identification of auxin‐related mutants in model species like Arabidopsis has played a key role in better understanding how auxin orchestrates plant morphogenesis. For example, loss of At‐<em>PIN2</em> causes root gravitropism defects due to the mutant's inability to transport auxin from the roots toward the shoots, and, due to <em>PIN2</em>'s high levels of conservation in cereals, a similar function is predicted in cereals that exhibit more complex root architecture (Lu et al., <a href="#tpg270061-bib-0035" class="usa-link" aria-describedby="tpg270061-bib-0035">2015</a>). Here, we report that <em>PIN2</em> plays an important role in gravitropism and root angle in barley. The loss of <em>PIN2</em> causes root‐specific agravitropism that led to increased seminal root angle resulting in wider and shallower rooting, without impacting the aboveground traits. These findings demonstrate the potential for <em>PIN2</em> as a new target to improve climate resilience while maintaining the agronomic traits of the canopy that have already been optimized through breeding for different crop production regions.</p>
<p>Deep root systems can contribute to drought adaptation by improving access to stored soil moisture in deep soil layers, thereby providing an advantage during terminal or post‐flowering drought stress. The root system of the crop can exhibit morphological, structural, and physiological responses to changes in the growing environment, referred to as root developmental plasticity. This is underpinned by changes in root branching, elongation, and curving, which broadly define the extent to which the root system can explore the soil volume. The “curving” component is associated with angle of seminal roots and the seedling proxy trait of mature root angle (Kang et al., <a href="#tpg270061-bib-0024" class="usa-link" aria-describedby="tpg270061-bib-0024">2024</a>).</p>
<p>Despite the more complex genetic and physiological architecture of barley, our analysis of tropic responses through <em>pin2</em> knockouts revealed homologous function to that of the model species Arabidopsis (Müller et al., <a href="#tpg270061-bib-0042" class="usa-link" aria-describedby="tpg270061-bib-0042">1998</a>). Typically, gravitropism is a guiding force whereby the radicle grows directly downward, followed by a slight weakening of the response to initiate growth at non‐vertical angles in order to facilitate roots spreading for soil exploration (Sachs, <a href="#tpg270061-bib-0049" class="usa-link" aria-describedby="tpg270061-bib-0049">1882</a>). This is termed the gravitropic setpoint angle and is considered a balance between two antagonistic growth signals, gravitropism and an anti‐gravitropic offset (Digby &amp; Firn, <a href="#tpg270061-bib-0013" class="usa-link" aria-describedby="tpg270061-bib-0013">1995</a>). In barley, the EGT1 and EGT2 genes have been identified as key elements in this antigravitropic response (Fusi et al., <a href="#tpg270061-bib-0015" class="usa-link" aria-describedby="tpg270061-bib-0015">2022</a>). Unlike PIN genes that are responsible for auxin transport, they are involved in cell wall stiffness at the elongation zone, further upstream than PIN2, where mutant EGT1 resulted in steeper root growth (Kirschner et al., <a href="#tpg270061-bib-0026" class="usa-link" aria-describedby="tpg270061-bib-0026">2024</a>). In <em>pin2</em> knockouts, the roots appeared aimless, and instead of exhibiting a consistent growth angle in response to gravity as typically seen in seedling assays, the roots continued to grow from their point of initiation out of the soil, against the gravity vector (Figure <a href="#tpg270061-fig-0003" class="usa-link">3</a>). In addition, phenotyping seedlings for seminal root angle, an established proxy for mature root angle, showed significant increases in seminal root angle (Alahmad et al., <a href="#tpg270061-bib-0003" class="usa-link" aria-describedby="tpg270061-bib-0003">2019</a>; Kang et al., <a href="#tpg270061-bib-0024" class="usa-link" aria-describedby="tpg270061-bib-0024">2024</a>). These observations suggest the loss of gravitropism leads to a loss of the primary guiding force for root angling, one of the earliest drivers of root distribution and architecture formation (Digby &amp; Firn, <a href="#tpg270061-bib-0013" class="usa-link" aria-describedby="tpg270061-bib-0013">1995</a>; Nakamoto &amp; Oyanagi, <a href="#tpg270061-bib-0043" class="usa-link" aria-describedby="tpg270061-bib-0043">1996</a>; Roychoudhry et al., <a href="#tpg270061-bib-0048" class="usa-link" aria-describedby="tpg270061-bib-0048">2013</a>).</p>
<p>There is a well‐established link between root distribution in the soil, water access, and water use (Li et al., <a href="#tpg270061-bib-0033" class="usa-link" aria-describedby="tpg270061-bib-0033">2022</a>; Robertson et al., <a href="#tpg270061-bib-0046" class="usa-link" aria-describedby="tpg270061-bib-0046">1993</a>). Further evidence on the role of <em>PIN2</em> and gravitropism in directing RSA formation in barley comes from functional analysis. The increased root angle resulting from inhibited apical auxin transport caused a wider and shallower root system in <em>pin2</em>, resulting in larger root biomass in upper layers, while trending toward a reduction in roots at depth, resembling a topsoil foraging ideotype that is beneficial for capturing phosphorous and intermittent rainfall (Lu et al., <a href="#tpg270061-bib-0035" class="usa-link" aria-describedby="tpg270061-bib-0035">2015</a>; Lynch, <a href="#tpg270061-bib-0037" class="usa-link" aria-describedby="tpg270061-bib-0037">2013</a>, <a href="#tpg270061-bib-0038" class="usa-link" aria-describedby="tpg270061-bib-0038">2019</a>).</p></section><section id="tpg270061-sec-0180"><h3 class="pmc_sec_title">4.2. 
<em>PIN2</em> mutants cause root system‐specific changes</h3>
<p>In barley, the quiescent center is a group of approximately 30 cells located in the root tip (apical meristem) of vascular plants where cell division occurs (Kirschner et al., <a href="#tpg270061-bib-0029" class="usa-link" aria-describedby="tpg270061-bib-0029">2017</a>). In Arabidopsis, PIN1 acts in shoot to root auxin transport, transporting auxin toward the quiescent center of the root meristem (Blilou et al., <a href="#tpg270061-bib-0005" class="usa-link" aria-describedby="tpg270061-bib-0005">2005</a>), whereas the PIN2 functions basally of the quiescent center and meristem to inhibit cell elongation in the root tip. Once auxin is transported to the root tip via PIN1, it accumulates at the root meristem where the high auxin concentrations inhibit elongation (Swarup et al., <a href="#tpg270061-bib-0054" class="usa-link" aria-describedby="tpg270061-bib-0054">2005</a>). Subsequently, PIN2 transports auxin shootward along the apical side of the roots, inhibits the roots elongation, and causes the roots to curve downward, also known as the gravitropic response (Blilou et al., <a href="#tpg270061-bib-0005" class="usa-link" aria-describedby="tpg270061-bib-0005">2005</a>). Throughout plant development, auxin synthesized in leaf primordia is continually transported to the root tip and the process is repeated, allowing for gravitropic root elongation. As such, in the barley <em>pin2</em> mutants, we hypothesize there is a build‐up of auxin in the lateral root cap post‐quiescent center and cause of the root‐specific changes to cell expansion due to an inability to flush auxin shootward, explaining the limited gravitropism and leading to increases in root angle and causing a visible agravitropic phenotype. Meanwhile, without the shootward flushing of auxin, it pools at the root tip, inhibiting elongation apically and basally due to the lack of polar auxin transport (Figure <a href="#tpg270061-supinfo-0001" class="usa-link">S5</a>).</p>
<p>Overall, this downstream effect on root elongation allows for more subtle changes to root architecture without significantly affecting organism‐wide patterning and architecture. For example, in maize, the overexpression of Zm‐<em>PIN1a</em> showed an elevated concentrations of auxin in the roots, increasing lateral root formation and seminal root length, improving root weight, and shortening plant height (Forestan &amp; Varotto, <a href="#tpg270061-bib-0014" class="usa-link" aria-describedby="tpg270061-bib-0014">2010</a>), whereas based on our findings in barley, <em>PIN2</em> has been shown to be a root‐specific auxin efflux transporter required for gravitropic responses in the roots. Loss‐of‐function produced aimless rooting with agravitropism, leading to increased root angle and a shift toward a top‐soil foraging RSA with a potential increase in root biomass (Figure <a href="#tpg270061-supinfo-0001" class="usa-link">S5</a>).</p>
<p>Interestingly, under the well‐watered conditions used in this study, the <em>pin2</em> barley mutant did not show any difference in aboveground biomass or agronomic performance (Figure <a href="#tpg270061-supinfo-0001" class="usa-link">S3</a>). This is contrasting with the overexpression of <em>PIN2</em> homologue in rice, which showed significant changes to plant height, tiller number, and tiller angle, while <em>pin2</em> mutants showed no change to canopy branching (tiller number), leaves per tiller, area per leaf, number of fertile tillers, or days to flower (Chen et al., <a href="#tpg270061-bib-0009" class="usa-link" aria-describedby="tpg270061-bib-0009">2012</a>). The lack of changes to canopy architecture or flowering time in cereal crops suggests this root‐specific target could be directly used in the future to breed cultivars with shallow rooting systems. Summarizing, Hv‐<em>PIN2</em> has been characterized as the functional homologue of At‐<em>PIN2</em>, playing an essential role in root gravitropism and providing new insight into root and shoot auxin transport and its roles in development and gravitropism.</p></section><section id="tpg270061-sec-0190"><h3 class="pmc_sec_title">4.3. Root:shoot ratio of <em>pin2</em> mutants is insensitive to planting density</h3>
<p>Genetic and management solutions are required to develop resilient crops in highly variable environments. Many different management systems are possible to combat drought such as row spacing, population, irrigation, fertilization, and cropping systems. Many different genotypic solutions are also possible, including utilizing the <em>PIN</em> gene family to modify traits affecting water supply and demand. Understanding the interaction of genotypes with different management strategies for specific target environments is critical to improving yield stability. While <em>PIN2</em> clearly plays an important role in orchestrating root curving through gravitropism, it also has novel effects on allometry under different planting densities in glasshouse conditions. Plants with narrow angles and more vertical roots tend to have larger proportions of their roots at depth during later development, which could increase access to water in deep soils. Conversely, genotypes with wider root angle and more horizontal rooting are better able to explore the soil in the inter‐row space, which could increase access to water in skip‐row systems (Hammer et al., <a href="#tpg270061-bib-0019" class="usa-link" aria-describedby="tpg270061-bib-0019">2009</a>).</p>
<p>In fertile environments when water is not limiting, competition among plants in a crop is mainly for light. However, in nutrient‐poor or water‐scarce conditions, competition is mainly for the limiting factor of either nutrients or water, respectively. The effects of competition among root systems are largely determined by the levels of water and nutrient availability, particularly inorganic forms of nutrients such as nitrate and ammonium (Tilman, <a href="#tpg270061-bib-0057" class="usa-link" aria-describedby="tpg270061-bib-0057">1985</a>). The usual row spacing of barley is typically 15–20 cm between rows to maximize the potential of individual plants. In our rhizobox experiments with non‐limiting water and nutrients, planting density (number of plants per rhizobox) was positively correlated with RSRatio in the WT lines. As plants per box increased from 0.5 (30‐cm row spacing) to 1 (15‐cm row spacing) to 3 (5‐cm row spacing), the RSRatio increased from 0.3 to 0.5 to 0.7, respectively (Figure <a href="#tpg270061-fig-0005" class="usa-link">5</a>). This indicates an increased allocation of resources to the roots as planting density increased, likely a cost in response to increased aboveground competition. Whereas in contrast, the <em>PIN2</em> mutant lines maintained an RSRatio of about 0.4 regardless of planting density, highlighting a trending insensitivity of <em>pin2</em> mutants to competition that is a reduced or loss of sensitivity to rooting density on root growth (Figure <a href="#tpg270061-fig-0005" class="usa-link">5</a>). Hence, at higher planting densities there was a significant change in root:shoot biomass ratio, with no significant change to canopy biomass and a reduced root system, consistent with the <em>pin2</em> response in Arabidopsis and likely a reflection of the reduction in root elongation (Figure <a href="#tpg270061-supinfo-0001" class="usa-link">S1</a>; Müller et al., <a href="#tpg270061-bib-0042" class="usa-link" aria-describedby="tpg270061-bib-0042">1998</a>). Field trials will be necessary to fully explore these implications on crop performance across a range of conditions.</p>
<figure class="fig xbox font-sm" id="tpg270061-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/643b/12328839/1af866301398/TPG2-18-e70061-g004.jpg" loading="lazy" id="jats-graphic-9" height="209" width="300" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270061-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>PIN‐FORMED2 (PIN2) increases root biomass under higher planting densities. The root:shoot biomass ratio for wild type (WT) and <em>pin2‐1</em> genotypes grown at three different row spacings indicative of plant density. Error bars represent standard error of the mean. Significance was tested using parametric <em>t</em>‐test (*<em>p</em> &lt; 0.05).</p></figcaption></figure><p>In the controlled environment used in the rhizobox experiments, nutrient availability was not a limiting factor. As such, nutrient uptake or the diffusion rate of the ions in the soil may have been limiting factors. While the WT reflexively altered its biomass allocation depending on planting density, increasing belowground biomass as planting density increased, <em>pin2</em> maintained a consistent RSRatio at all tested densities (Figure <a href="#tpg270061-fig-0005" class="usa-link">5</a>). As such, it is hypothesized that <em>PIN2</em> plays an important role in external sensing alongside gravitropism, where the absence of this PIN2 transporter leads to agravitropic roots that project outwards and appear insensitive to environmental stimuli such as intraspecific competition. In field‐grown barley, when nutrients are not limiting, less biomass is required underground, and capacity for nutrient uptake can be enhanced by concentrating root proliferation in the topsoil (Grime &amp; Hodgson, <a href="#tpg270061-bib-0018" class="usa-link" aria-describedby="tpg270061-bib-0018">1987</a>; Hecht et al., <a href="#tpg270061-bib-0020" class="usa-link" aria-describedby="tpg270061-bib-0020">2016</a>). It is possible the increased root proliferation in upper soil layers by <em>pin2</em> provides improved resource availability, limiting the need for further resource investment into soil exploration. Therefore, modulating the expression of <em>PIN2</em> provides a unique target for improving RSA, which may lead to a range of root angles and influence drought tolerance without any negative impacts on yield or obvious aboveground trade‐offs (Joshi et al., <a href="#tpg270061-bib-0023" class="usa-link" aria-describedby="tpg270061-bib-0023">2016</a>). Currently, the complete loss‐of‐function <em>pin2</em> mutants display a topsoil foraging root system ideotype that may provide benefits for uptake of immobile nutrients in some agricultural settings. However, field studies in different environments are needed to explore the effects of <em>PIN2</em> on yield and its influence under changing planting densities (Liu &amp; Godwin, <a href="#tpg270061-bib-0034" class="usa-link" aria-describedby="tpg270061-bib-0034">2012</a>).</p></section></section><section id="tpg270061-sec-0200"><h2 class="pmc_sec_title">5. CONCLUSION</h2>
<p>Understanding the allometric relationship between canopy and root architecture has important agricultural implications. Here, we demonstrate conserved function of the auxin efflux carrier Hv‐<em>PIN2</em> in directing root angle through the gravitropic response in barley. Alongside altered root distribution that produced a shallower and more expansive RSA, <em>pin2</em> mutants did not display the typical root‐shoot allometric relationship. While further investigation is required regarding the exact mechanism regulating root angle, <em>PIN2</em> shows clear links to agronomic traits such as root angle and root distribution that may offer an advantage under moisture‐limited environments. The novel root phenotypes identified in this study offer useful insight into the potential for genome engineering of root traits for future barley improvement programs. Overall, the investigation of key developmental genes (initially discovered in model species) through gene editing shows potential to support the development of new cultivars with improved nutrient‐use efficiency and climate resilience.</p></section><section id="tpg270061-sec-0210"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Zachary Aldiss</strong>: Conceptualization; data curation; formal analysis; investigation; methodology; validation; writing—original draft. <strong>Yasmine Lam</strong>: Conceptualization; formal analysis; supervision; writing—review and editing. <strong>Hannah Robinson</strong>: Formal analysis; methodology; writing—review and editing. <strong>Richard Dixon</strong>: Formal analysis; investigation; writing—review and editing. <strong>Laura Steinhardt</strong>: Data curation; formal analysis; writing—review and editing. <strong>Peter Crisp</strong>: Methodology; writing—review and editing. <strong>Ian Godwin</strong>: Conceptualization; funding acquisition; writing—review and editing. <strong>Andrew Borrell</strong>: Funding acquisition; project administration; writing—review and editing. <strong>Lee Hickey</strong>: Funding acquisition; project administration; writing—review and editing. <strong>Karen Massel</strong>: Conceptualization; formal analysis; investigation; methodology; project administration; software; supervision; writing—original draft; writing—review and editing.</p></section><section id="tpg270061-sec-0230"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec23"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270061-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270061-supitem-0001"><div class="caption p"><p>
<strong>Figure S1</strong>. Illustration of <em>PIN2</em> genome edited knock out lines with impact on protein structure and transmembrane domains compared to wild type.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328839/bin/TPG2-18-e70061-s001.jpeg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70061-s001.jpeg</a><sup> (52.7KB, jpeg) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270061-supitem-0002"><div class="caption p"><p>
<strong>Figure S2</strong>. Canopy and root trait results from narrow rhizobox experiments conducted using different planting density.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328839/bin/TPG2-18-e70061-s006.jpeg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70061-s006.jpeg</a><sup> (154.8KB, jpeg) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270061-supitem-0003"><div class="caption p"><p>
<strong>Figure S3</strong>. PIN2 in barley regulates downward root growth leading to steeper and narrower root architecture without influencing biomass.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328839/bin/TPG2-18-e70061-s008.jpeg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70061-s008.jpeg</a><sup> (82.2KB, jpeg) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270061-supitem-0004"><div class="caption p"><p>
<strong>Figure S4</strong>. Maturity trial reveal no significant difference to canopy and phenology in <em>pin2</em>.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328839/bin/TPG2-18-e70061-s004.jpeg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70061-s004.jpeg</a><sup> (57.8KB, jpeg) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270061-supitem-0005"><div class="caption p"><p>
<strong>Figure S5</strong>. Summary of hypothesis: disrupted auxin transport in <em>pin2</em> genome edited barley inhibits gravitropic response and alters root development.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328839/bin/TPG2-18-e70061-s002.jpeg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70061-s002.jpeg</a><sup> (111.7KB, jpeg) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270061-supitem-0006"><div class="caption p"><p>
<strong>Table S1</strong>. Fixed and random effects table for the linear mixed model (LMM) fitted to the trait of interest.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328839/bin/TPG2-18-e70061-s007.jpeg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70061-s007.jpeg</a><sup> (69.6KB, jpeg) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270061-supitem-0007"><div class="caption p"><p>
<strong>Table S2</strong>. Synteny of <em>PIN</em> genes across cereals.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328839/bin/TPG2-18-e70061-s005.jpeg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70061-s005.jpeg</a><sup> (49.8KB, jpeg) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270061-supitem-0008"><div class="caption p"><p>
<strong>Table S3</strong>. Table measured shoot growth angles for WT and <em>PIN2 KO</em>.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328839/bin/TPG2-18-e70061-s003.jpeg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70061-s003.jpeg</a><sup> (6.3KB, jpeg) </sup>
</div></div></section></section><section id="tpg270061-sec-0220" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>We are grateful to the Crisp group for stimulating discussions during the work. The authors are also grateful to Dr Millicent Smith and Dr Owen Powell at The University of Queensland for coordinating review and guiding constructing of the early stages of the manuscript. This research was supported by ARC Discovery Project “Cereal blueprints for a water‐limited world” (DP190102185) and ARC Linkage project “Digging deeper to improve yield stability” (LP200200927). Zachary Aldiss received scholarships from the Australian Government Research Training Program and InterGrain. Lee Hickey was supported through an ARC Future Fellowship (FT220100350).</p></section><section id="notes1"><p>


Aldiss, Z.
, 
Lam, Y.
, 
Robinson, H.
, 
Dixon, R.
, 
Steinhardt, L.
, 
Crisp, P.
, 
Godwin, I.
, 
Borrell, A.
, 
Hickey, L.
, &amp; 
Massel, K.
 (2025). The <em>PIN2</em> ortholog in barley modifies root gravitropism and architecture. The Plant Genome, 18, e70061. 10.1002/tpg2.70061

</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270061-note-0001"><p>Assigned to Associate Editor Roberto Tuberosa.</p></div></div></section></section><section id="tpg270061-sec-0250"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>All data generated or analyzed during this study are included in this published article and its Supporting Information files.</p></section><section id="tpg270061-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270061-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270061-bib-0001">
<cite>

Ahmar, S.
, 
Usman, B.
, 
Hensel, G.
, 
Jung, K.‐H.
, &amp; 
Gruszka, D.
 (2024). CRISPR enables sustainable cereal production for a greener future. Trends in Plant Science, 29(2), 179–195. 10.1016/j.tplants.2023.10.016

</cite> [<a href="https://doi.org/10.1016/j.tplants.2023.10.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37981496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Plant%20Science&amp;title=CRISPR%20enables%20sustainable%20cereal%20production%20for%20a%20greener%20future&amp;author=S.%20Ahmar&amp;author=B.%20Usman&amp;author=G.%20Hensel&amp;author=K.%E2%80%90H.%20Jung&amp;author=D.%20Gruszka&amp;volume=29&amp;issue=2&amp;publication_year=2024&amp;pages=179-195&amp;pmid=37981496&amp;doi=10.1016/j.tplants.2023.10.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0002">
<cite>

Akaike, H.
 (1974). A new look at the statistical model identification. IEEE Transactions on Automatic Control, 19(6), 716–723. 10.1109/TAC.1974.1100705
</cite> [<a href="https://doi.org/10.1109/TAC.1974.1100705" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Transactions%20on%20Automatic%20Control&amp;title=A%20new%20look%20at%20the%20statistical%20model%20identification&amp;author=H.%20Akaike&amp;volume=19&amp;issue=6&amp;publication_year=1974&amp;pages=716-723&amp;doi=10.1109/TAC.1974.1100705&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0003">
<cite>

Alahmad, S.
, 
El Hassouni, K.
, 
Bassi, F. M.
, 
Dinglasan, E.
, 
Youssef, C.
, 
Quarry, G.
, 
Aksoy, A.
, 
Mazzucotelli, E.
, 
Juhász, A.
, 
Able, J. A.
, 
Christopher, J.
, 
Voss‐Fels, K. P.
, &amp; 
Hickey, L. T.
 (2019). A major root architecture QTL responding to water limitation in durum wheat. Frontiers in Plant Science, 10, Article 436. 10.3389/fpls.2019.00436

</cite> [<a href="https://doi.org/10.3389/fpls.2019.00436" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6468307/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31024600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=A%20major%20root%20architecture%20QTL%20responding%20to%20water%20limitation%20in%20durum%20wheat&amp;author=S.%20Alahmad&amp;author=K.%20El%20Hassouni&amp;author=F.%20M.%20Bassi&amp;author=E.%20Dinglasan&amp;author=C.%20Youssef&amp;volume=10&amp;publication_year=2019&amp;pages=436&amp;pmid=31024600&amp;doi=10.3389/fpls.2019.00436&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0004">
<cite>

Bishopp, A.
, &amp; 
Lynch, J. P.
 (2015). The hidden half of crop yields. Nature Plants, 1, Article 15117. 10.1038/nplants.2015.117

</cite> [<a href="https://doi.org/10.1038/nplants.2015.117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27250548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Plants&amp;title=The%20hidden%20half%20of%20crop%20yields&amp;author=A.%20Bishopp&amp;author=J.%20P.%20Lynch&amp;volume=1&amp;publication_year=2015&amp;pages=15117&amp;pmid=27250548&amp;doi=10.1038/nplants.2015.117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0005">
<cite>

Blilou, I.
, 
Xu, J.
, 
Wildwater, M.
, 
Willemsen, V.
, 
Paponov, I.
, 
Friml, J.
, 
Heidstra, R.
, 
Aida, M.
, 
Palme, K.
, &amp; 
Scheres, B.
 (2005). The PIN auxin efflux facilitator network controls growth and patterning in Arabidopsis roots. Nature, 433(7021), 39–44. 10.1038/nature03184

</cite> [<a href="https://doi.org/10.1038/nature03184" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15635403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20PIN%20auxin%20efflux%20facilitator%20network%20controls%20growth%20and%20patterning%20in%20Arabidopsis%20roots&amp;author=I.%20Blilou&amp;author=J.%20Xu&amp;author=M.%20Wildwater&amp;author=V.%20Willemsen&amp;author=I.%20Paponov&amp;volume=433&amp;issue=7021&amp;publication_year=2005&amp;pages=39-44&amp;pmid=15635403&amp;doi=10.1038/nature03184&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0006">
<cite>

Borrell, A. K.
, 
Wong, A. C. S.
, 
George‐Jaeggli, B.
, 
van Oosterom, E. J.
, 
Mace, E. S.
, 
Godwin, I. D.
, 
Liu, G.
, 
Mullet, J. E.
, 
Klein, P. E.
, 
Hammer, G. L.
, 
McLean, G.
, 
Hunt, C.
, &amp; 
Jordan, D. R.
 (2022). Genetic modification of PIN genes induces causal mechanisms of stay‐green drought adaptation phenotype. Journal of Experimental Botany, 73(19), 6711–6726. 10.1093/jxb/erac336

</cite> [<a href="https://doi.org/10.1093/jxb/erac336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9629789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35961690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Genetic%20modification%20of%20PIN%20genes%20induces%20causal%20mechanisms%20of%20stay%E2%80%90green%20drought%20adaptation%20phenotype&amp;author=A.%20K.%20Borrell&amp;author=A.%20C.%20S.%20Wong&amp;author=B.%20George%E2%80%90Jaeggli&amp;author=E.%20J.%20van%20Oosterom&amp;author=E.%20S.%20Mace&amp;volume=73&amp;issue=19&amp;publication_year=2022&amp;pages=6711-6726&amp;pmid=35961690&amp;doi=10.1093/jxb/erac336&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0007">
<cite>

Butler, D.
, 
Cullis, B.
, 
Gilmour, A.
, 
Gogel, B.
, &amp; 
Thompson, R.
 (2017). ASReml—R reference manual version 4. VSN International Ltd.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=ASReml%E2%80%94R%20reference%20manual%20version%204&amp;author=D.%20Butler&amp;author=B.%20Cullis&amp;author=A.%20Gilmour&amp;author=B.%20Gogel&amp;author=R.%20Thompson&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0008">
<cite>

Casimiro, I.
, 
Marchant, A.
, 
Bhalerao, R. P.
, 
Beeckman, T.
, 
Dhooge, S.
, 
Swarup, R.
, 
Graham, N.
, 
Inzé, D.
, 
Sandberg, G.
, 
Casero, P. J.
, &amp; 
Bennett, M.
 (2001). Auxin transport promotes Arabidopsis lateral root initiation. Plant Cell, 13(4), 843–852. 10.1105/tpc.13.4.843

</cite> [<a href="https://doi.org/10.1105/tpc.13.4.843" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC135543/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11283340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Auxin%20transport%20promotes%20Arabidopsis%20lateral%20root%20initiation&amp;author=I.%20Casimiro&amp;author=A.%20Marchant&amp;author=R.%20P.%20Bhalerao&amp;author=T.%20Beeckman&amp;author=S.%20Dhooge&amp;volume=13&amp;issue=4&amp;publication_year=2001&amp;pages=843-852&amp;pmid=11283340&amp;doi=10.1105/tpc.13.4.843&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0009">
<cite>

Chen, Y.
, 
Fan, X.
, 
Song, W.
, 
Zhang, Y.
, &amp; 
Xu, G.
 (2012). Over‐expression of OsPIN2 leads to increased tiller numbers, angle and shorter plant height through suppression of OsLAZY1. Plant Biotechnology Journal, 10(2), 139–149. 10.1111/j.1467-7652.2011.00637.x

</cite> [<a href="https://doi.org/10.1111/j.1467-7652.2011.00637.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21777365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=Over%E2%80%90expression%20of%20OsPIN2%20leads%20to%20increased%20tiller%20numbers,%20angle%20and%20shorter%20plant%20height%20through%20suppression%20of%20OsLAZY1&amp;author=Y.%20Chen&amp;author=X.%20Fan&amp;author=W.%20Song&amp;author=Y.%20Zhang&amp;author=G.%20Xu&amp;volume=10&amp;issue=2&amp;publication_year=2012&amp;pages=139-149&amp;pmid=21777365&amp;doi=10.1111/j.1467-7652.2011.00637.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0010">
<cite>

Christopher, J.
, 
Christopher, M.
, 
Jennings, R.
, 
Jones, S.
, 
Fletcher, S.
, 
Borrell, A.
, 
Manschadi, A. M.
, 
Jordan, D.
, 
Mace, E.
, &amp; 
Hammer, G.
 (2013). QTL for root angle and number in a population developed from bread wheats (<em>Triticum aestivum</em>) with contrasting adaptation to water‐limited environments. Theoretical and Applied Genetics [Theoretische Und Angewandte Genetik], 126(6), 1563–1574. 10.1007/s00122-013-2074-0

</cite> [<a href="https://doi.org/10.1007/s00122-013-2074-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23525632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics%20%5BTheoretische%20Und%20Angewandte%20Genetik%5D&amp;title=QTL%20for%20root%20angle%20and%20number%20in%20a%20population%20developed%20from%20bread%20wheats%20(Triticum%20aestivum)%20with%20contrasting%20adaptation%20to%20water%E2%80%90limited%20environments&amp;author=J.%20Christopher&amp;author=M.%20Christopher&amp;author=R.%20Jennings&amp;author=S.%20Jones&amp;author=S.%20Fletcher&amp;volume=126&amp;issue=6&amp;publication_year=2013&amp;pages=1563-1574&amp;pmid=23525632&amp;doi=10.1007/s00122-013-2074-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0011">
<cite>

Cronin, J. K.
, 
Bundock, P. C.
, 
Henry, R. J.
, &amp; 
Nevo, E.
 (2007). Adaptive climatic molecular evolution in wild barley at the <em>Isa</em> defense locus. Proceedings of the National Academy of Sciences, 104(8), 2773–2778. 10.1073/pnas.0611226104
</cite> [<a href="https://doi.org/10.1073/pnas.0611226104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1815257/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17301230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Adaptive%20climatic%20molecular%20evolution%20in%20wild%20barley%20at%20the%20Isa%20defense%20locus&amp;author=J.%20K.%20Cronin&amp;author=P.%20C.%20Bundock&amp;author=R.%20J.%20Henry&amp;author=E.%20Nevo&amp;volume=104&amp;issue=8&amp;publication_year=2007&amp;pages=2773-2778&amp;pmid=17301230&amp;doi=10.1073/pnas.0611226104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0012">
<cite>

Cullis, B. R.
, 
Smith, A. B.
, &amp; 
Coombes, N. E.
 (2006). On the design of early generation variety trials with correlated data. Journal of Agricultural, Biological, and Environmental Statistics, 11(4), 381–393. 10.1198/108571106X154443
</cite> [<a href="https://doi.org/10.1198/108571106X154443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Agricultural,%20Biological,%20and%20Environmental%20Statistics&amp;title=On%20the%20design%20of%20early%20generation%20variety%20trials%20with%20correlated%20data&amp;author=B.%20R.%20Cullis&amp;author=A.%20B.%20Smith&amp;author=N.%20E.%20Coombes&amp;volume=11&amp;issue=4&amp;publication_year=2006&amp;pages=381-393&amp;doi=10.1198/108571106X154443&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0013">
<cite>

Digby, J.
, &amp; 
Firn, R. D.
 (1995). The gravitropic set‐point angle (GSA): The identification of an important developmentally controlled variable governing plant architecture. Plant, Cell and Environment, 18(12), 1434–1440. 10.1111/j.1365-3040.1995.tb00205.x
</cite> [<a href="https://doi.org/10.1111/j.1365-3040.1995.tb00205.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11543210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant,%20Cell%20and%20Environment&amp;title=The%20gravitropic%20set%E2%80%90point%20angle%20(GSA):%20The%20identification%20of%20an%20important%20developmentally%20controlled%20variable%20governing%20plant%20architecture&amp;author=J.%20Digby&amp;author=R.%20D.%20Firn&amp;volume=18&amp;issue=12&amp;publication_year=1995&amp;pages=1434-1440&amp;pmid=11543210&amp;doi=10.1111/j.1365-3040.1995.tb00205.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0014">
<cite>

Forestan, C.
, &amp; 
Varotto, S.
 (2010). PIN1 auxin efflux carriers localization studies in <em>Zea mays</em>
. Plant Signaling &amp; Behavior, 5(4), 436–439. 10.4161/psb.5.4.11339

</cite> [<a href="https://doi.org/10.4161/psb.5.4.11339" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7080414/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20383059/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Signaling%20&amp;%20Behavior&amp;title=PIN1%20auxin%20efflux%20carriers%20localization%20studies%20in%20Zea%20mays&amp;author=C.%20Forestan&amp;author=S.%20Varotto&amp;volume=5&amp;issue=4&amp;publication_year=2010&amp;pages=436-439&amp;pmid=20383059&amp;doi=10.4161/psb.5.4.11339&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0015">
<cite>

Fusi, R.
, 
Rosignoli, S.
, 
Lou, H.
, 
Sangiorgi, G.
, 
Bovina, R.
, 
Pattem, J. K.
, 
Borkar, A. N.
, 
Lombardi, M.
, 
Forestan, C.
, 
Milner, S. G.
, 
Davis, J. L.
, 
Lale, A.
, 
Kirschner, G. K.
, 
Swarup, R.
, 
Tassinari, A.
, 
Pandey, B. K.
, 
York, L. M.
, 
Atkinson, B. S.
, 
Sturrock, C. J.
, … 
Salvi, S.
 (2022). Root angle is controlled by <em>EGT1</em> in cereal crops employing an antigravitropic mechanism. Proceedings of the National Academy of Sciences, 119(31), e2201350119. 10.1073/pnas.2201350119
</cite> [<a href="https://doi.org/10.1073/pnas.2201350119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9351459/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35881796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences&amp;title=Root%20angle%20is%20controlled%20by%20EGT1%20in%20cereal%20crops%20employing%20an%20antigravitropic%20mechanism&amp;author=R.%20Fusi&amp;author=S.%20Rosignoli&amp;author=H.%20Lou&amp;author=G.%20Sangiorgi&amp;author=R.%20Bovina&amp;volume=119&amp;issue=31&amp;publication_year=2022&amp;pages=e2201350119&amp;pmid=35881796&amp;doi=10.1073/pnas.2201350119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0016">
<cite>

Geisler, M.
, &amp; 
Murphy, A. S.
 (2006). The ABC of auxin transport: The role of p‐glycoproteins in plant development. FEBS Letters, 580(4), 1094–1102. 10.1016/j.febslet.2005.11.054

</cite> [<a href="https://doi.org/10.1016/j.febslet.2005.11.054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16359667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Letters&amp;title=The%20ABC%20of%20auxin%20transport:%20The%20role%20of%20p%E2%80%90glycoproteins%20in%20plant%20development&amp;author=M.%20Geisler&amp;author=A.%20S.%20Murphy&amp;volume=580&amp;issue=4&amp;publication_year=2006&amp;pages=1094-1102&amp;pmid=16359667&amp;doi=10.1016/j.febslet.2005.11.054&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0017">
<cite>

Goodstein, D. M.
, 
Shu, S.
, 
Howson, R.
, 
Neupane, R.
, 
Hayes, R. D.
, 
Fazo, J.
, 
Mitros, T.
, 
Dirks, W.
, 
Hellsten, U.
, 
Putnam, N.
, &amp; 
Rokhsar, D. S.
 (2012). Phytozome: A comparative platform for green plant genomics. Nucleic Acids Research, 40, D1178–D1186. 10.1093/nar/gkr944

</cite> [<a href="https://doi.org/10.1093/nar/gkr944" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3245001/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22110026/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Phytozome:%20A%20comparative%20platform%20for%20green%20plant%20genomics&amp;author=D.%20M.%20Goodstein&amp;author=S.%20Shu&amp;author=R.%20Howson&amp;author=R.%20Neupane&amp;author=R.%20D.%20Hayes&amp;volume=40&amp;publication_year=2012&amp;pages=D1178-D1186&amp;pmid=22110026&amp;doi=10.1093/nar/gkr944&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0018">
<cite>

Grime, J. P.
, &amp; 
Hodgson, J. G.
 (1987). Botanical contributions to contemporary ecological theory. New Phytologist, 106(s1), 283–295. 10.1111/j.1469-8137.1987.tb04695.x
</cite> [<a href="https://doi.org/10.1111/j.1469-8137.1987.tb04695.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Botanical%20contributions%20to%20contemporary%20ecological%20theory&amp;author=J.%20P.%20Grime&amp;author=J.%20G.%20Hodgson&amp;volume=106&amp;issue=s1&amp;publication_year=1987&amp;pages=283-295&amp;doi=10.1111/j.1469-8137.1987.tb04695.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0019">
<cite>

Hammer, G. L.
, 
Dong, Z.
, 
McLean, G.
, 
Doherty, A.
, 
Messina, C.
, 
Schussler, J.
, 
Zinselmeier, C.
, 
Paszkiewicz, S.
, &amp; 
Cooper, M.
 (2009). Can changes in canopy and/or root system architecture explain historical maize yield trends in the U.S. Corn Belt?
Crop Science, 49(1), 299–312. 10.2135/cropsci2008.03.0152
</cite> [<a href="https://doi.org/10.2135/cropsci2008.03.0152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20Science&amp;title=Can%20changes%20in%20canopy%20and/or%20root%20system%20architecture%20explain%20historical%20maize%20yield%20trends%20in%20the%20U.S.%20Corn%20Belt?&amp;author=G.%20L.%20Hammer&amp;author=Z.%20Dong&amp;author=G.%20McLean&amp;author=A.%20Doherty&amp;author=C.%20Messina&amp;volume=49&amp;issue=1&amp;publication_year=2009&amp;pages=299-312&amp;doi=10.2135/cropsci2008.03.0152&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0064">
<cite>

Harwood, W. A.
, &amp; 
Smedley, M. A.
 (2009). Barley transformation using biolistic techniques. Methods in Molecular Biology, 478, 125–136. 10.1007/978-1-59745-379-0_8

</cite> [<a href="https://doi.org/10.1007/978-1-59745-379-0_8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19009443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20in%20Molecular%20Biology&amp;title=Barley%20transformation%20using%20biolistic%20techniques&amp;author=W.%20A.%20Harwood&amp;author=M.%20A.%20Smedley&amp;volume=478&amp;publication_year=2009&amp;pages=125-136&amp;pmid=19009443&amp;doi=10.1007/978-1-59745-379-0_8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0020">
<cite>

Hecht, V. L.
, 
Temperton, V. M.
, 
Nagel, K. A.
, 
Rascher, U.
, &amp; 
Postma, J. A.
 (2016). Sowing density: A neglected factor fundamentally affecting root distribution and biomass allocation of field grown spring barley (<em>Hordeum vulgare</em> L.). Frontiers in Plant Science, 7, Article 944. 10.3389/fpls.2016.00944

</cite> [<a href="https://doi.org/10.3389/fpls.2016.00944" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4923255/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27446171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Sowing%20density:%20A%20neglected%20factor%20fundamentally%20affecting%20root%20distribution%20and%20biomass%20allocation%20of%20field%20grown%20spring%20barley%20(Hordeum%20vulgare%20L.)&amp;author=V.%20L.%20Hecht&amp;author=V.%20M.%20Temperton&amp;author=K.%20A.%20Nagel&amp;author=U.%20Rascher&amp;author=J.%20A.%20Postma&amp;volume=7&amp;publication_year=2016&amp;pages=944&amp;pmid=27446171&amp;doi=10.3389/fpls.2016.00944&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0021">
<cite>

Ho, M. D.
, 
McCannon, B. C.
, &amp; 
Lynch, J. P.
 (2004). Optimization modeling of plant root architecture for water and phosphorus acquisition. Journal of Theoretical Biology, 226(3), 331–340. 10.1016/j.jtbi.2003.09.011

</cite> [<a href="https://doi.org/10.1016/j.jtbi.2003.09.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14643647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Theoretical%20Biology&amp;title=Optimization%20modeling%20of%20plant%20root%20architecture%20for%20water%20and%20phosphorus%20acquisition&amp;author=M.%20D.%20Ho&amp;author=B.%20C.%20McCannon&amp;author=J.%20P.%20Lynch&amp;volume=226&amp;issue=3&amp;publication_year=2004&amp;pages=331-340&amp;pmid=14643647&amp;doi=10.1016/j.jtbi.2003.09.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0022">
<cite>

Huang, X.
, 
Hilscher, J.
, 
Stoger, E.
, 
Christou, P.
, &amp; 
Zhu, C.
 (2021). Modification of cereal plant architecture by genome editing to improve yields. Plant Cell Reports, 40(6), 953–978. 10.1007/s00299-021-02668-7

</cite> [<a href="https://doi.org/10.1007/s00299-021-02668-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33559722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Reports&amp;title=Modification%20of%20cereal%20plant%20architecture%20by%20genome%20editing%20to%20improve%20yields&amp;author=X.%20Huang&amp;author=J.%20Hilscher&amp;author=E.%20Stoger&amp;author=P.%20Christou&amp;author=C.%20Zhu&amp;volume=40&amp;issue=6&amp;publication_year=2021&amp;pages=953-978&amp;pmid=33559722&amp;doi=10.1007/s00299-021-02668-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0023">
<cite>

Joshi, D.
, 
Singh, V.
, 
van Oosterom, E.
, 
Mace, E.
, 
Jordan, D.
, &amp; 
Hammer, G.
 (2016). Genetic manipulation of root system architecture to improve drought adaptation in sorghum. In Rakshit S. &amp; Wang Y.‐H. (Eds.), The sorghum genome (pp. 207–226). Springer International Publishing. 10.1007/978-3-319-47789-3_11
</cite> [<a href="https://doi.org/10.1007/978-3-319-47789-3_11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=The%20sorghum%20genome&amp;author=D.%20Joshi&amp;author=V.%20Singh&amp;author=E.%20van%20Oosterom&amp;author=E.%20Mace&amp;author=D.%20Jordan&amp;publication_year=2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0024">
<cite>

Kang, Y.
, 
Rambla, C.
, 
Haeften, S. V.
, 
Fu, B.
, 
Akinlade, O.
, 
Potgieter, A. B.
, 
Borrell, A. K.
, 
Mace, E.
, 
Jordan, D. R.
, 
Alahmad, S.
, &amp; 
Hickey, L. T.
 (2024). Seminal root angle is associated with root system architecture in durum wheat. Food and Energy Security, 13(4), e570. 10.1002/fes3.570
</cite> [<a href="https://doi.org/10.1002/fes3.570" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20and%20Energy%20Security&amp;title=Seminal%20root%20angle%20is%20associated%20with%20root%20system%20architecture%20in%20durum%20wheat&amp;author=Y.%20Kang&amp;author=C.%20Rambla&amp;author=S.%20V.%20Haeften&amp;author=B.%20Fu&amp;author=O.%20Akinlade&amp;volume=13&amp;issue=4&amp;publication_year=2024&amp;pages=e570&amp;doi=10.1002/fes3.570&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0025">
<cite>

Kenward, M. G.
, &amp; 
Roger, J. H.
 (1997). Small Sample inference for fixed effects from restricted maximum likelihood. Biometrics, 53(3), 983–997. 10.2307/2533558

</cite> [<a href="https://doi.org/10.2307/2533558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9333350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biometrics&amp;title=Small%20Sample%20inference%20for%20fixed%20effects%20from%20restricted%20maximum%20likelihood&amp;author=M.%20G.%20Kenward&amp;author=J.%20H.%20Roger&amp;volume=53&amp;issue=3&amp;publication_year=1997&amp;pages=983-997&amp;pmid=9333350&amp;doi=10.2307/2533558&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0026">
<cite>

Kirschner, G. K.
, 
Hochholdinger, F.
, 
Salvi, S.
, 
Bennett, M. J.
, 
Huang, G.
, &amp; 
Bhosale, R. A.
 (2024). Genetic regulation of the root angle in cereals. Trends in Plant Science, 29(7), 814–822. 10.1016/j.tplants.2024.01.008

</cite> [<a href="https://doi.org/10.1016/j.tplants.2024.01.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38402016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Plant%20Science&amp;title=Genetic%20regulation%20of%20the%20root%20angle%20in%20cereals&amp;author=G.%20K.%20Kirschner&amp;author=F.%20Hochholdinger&amp;author=S.%20Salvi&amp;author=M.%20J.%20Bennett&amp;author=G.%20Huang&amp;volume=29&amp;issue=7&amp;publication_year=2024&amp;pages=814-822&amp;pmid=38402016&amp;doi=10.1016/j.tplants.2024.01.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0027">
<cite>

Kirschner, G. K.
, 
Rosignoli, S.
, 
Guo, L.
, 
Vardanega, I.
, 
Imani, J.
, 
Altmüller, J.
, 
Milner, S. G.
, 
Balzano, R.
, 
Nagel, K. A.
, 
Pflugfelder, D.
, 
Forestan, C.
, 
Bovina, R.
, 
Koller, R.
, 
Stöcker, T. G.
, 
Mascher, M.
, 
Simmonds, J.
, 
Uauy, C.
, 
Schoof, H.
, 
Tuberosa, R.
, … 
Hochholdinger, F.
 (2021). ENHANCED GRAVITROPISM 2 encodes a STERILE ALPHA MOTIF‐containing protein that controls root growth angle in barley and wheat. PNAS, 118(35), e2101526118. 10.1073/pnas.2101526118

</cite> [<a href="https://doi.org/10.1073/pnas.2101526118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8536364/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34446550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=ENHANCED%20GRAVITROPISM%202%20encodes%20a%20STERILE%20ALPHA%20MOTIF%E2%80%90containing%20protein%20that%20controls%20root%20growth%20angle%20in%20barley%20and%20wheat&amp;author=G.%20K.%20Kirschner&amp;author=S.%20Rosignoli&amp;author=L.%20Guo&amp;author=I.%20Vardanega&amp;author=J.%20Imani&amp;volume=118&amp;issue=35&amp;publication_year=2021&amp;pages=e2101526118&amp;pmid=34446550&amp;doi=10.1073/pnas.2101526118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0028">
<cite>

Kirschner, G. K.
, 
Stahl, Y.
, 
Imani, J.
, 
von Korff, M.
, &amp; 
Simon, R.
 (2018). Fluorescent reporter lines for auxin and cytokinin signalling in barley (<em>Hordeum vulgare</em>). PLoS ONE, 13(4), e0196086. 10.1371/journal.pone.0196086

</cite> [<a href="https://doi.org/10.1371/journal.pone.0196086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5918912/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29694399/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Fluorescent%20reporter%20lines%20for%20auxin%20and%20cytokinin%20signalling%20in%20barley%20(Hordeum%20vulgare)&amp;author=G.%20K.%20Kirschner&amp;author=Y.%20Stahl&amp;author=J.%20Imani&amp;author=M.%20von%20Korff&amp;author=R.%20Simon&amp;volume=13&amp;issue=4&amp;publication_year=2018&amp;pages=e0196086&amp;pmid=29694399&amp;doi=10.1371/journal.pone.0196086&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0029">
<cite>

Kirschner, G. K.
, 
Stahl, Y.
, 
Von Korff, M.
, &amp; 
Simon, R.
 (2017). Unique and conserved features of the barley root meristem. Frontiers in Plant Science, 8, Article 1240. 10.3389/fpls.2017.01240

</cite> [<a href="https://doi.org/10.3389/fpls.2017.01240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5519606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28785269/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Unique%20and%20conserved%20features%20of%20the%20barley%20root%20meristem&amp;author=G.%20K.%20Kirschner&amp;author=Y.%20Stahl&amp;author=M.%20Von%20Korff&amp;author=R.%20Simon&amp;volume=8&amp;publication_year=2017&amp;pages=1240&amp;pmid=28785269&amp;doi=10.3389/fpls.2017.01240&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0030">
<cite>

Kitomi, Y.
, 
Hanzawa, E.
, 
Kuya, N.
, 
Inoue, H.
, 
Hara, N.
, 
Kawai, S.
, 
Kanno, N.
, 
Endo, M.
, 
Sugimoto, K.
, 
Yamazaki, T.
, 
Sakamoto, S.
, 
Sentoku, N.
, 
Wu, J.
, 
Kanno, H.
, 
Mitsuda, N.
, 
Toriyama, K.
, 
Sato, T.
, &amp; 
Uga, Y.
 (2020). Root angle modifications by the DRO1 homolog improve rice yields in saline paddy fields. PNAS, 117(35), 21242–21250. 10.1073/pnas.2005911117

</cite> [<a href="https://doi.org/10.1073/pnas.2005911117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7474696/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32817523/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Root%20angle%20modifications%20by%20the%20DRO1%20homolog%20improve%20rice%20yields%20in%20saline%20paddy%20fields&amp;author=Y.%20Kitomi&amp;author=E.%20Hanzawa&amp;author=N.%20Kuya&amp;author=H.%20Inoue&amp;author=N.%20Hara&amp;volume=117&amp;issue=35&amp;publication_year=2020&amp;pages=21242-21250&amp;pmid=32817523&amp;doi=10.1073/pnas.2005911117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0031">
<cite>

Kleine‐Vehn, J.
, 
Leitner, J.
, 
Zwiewka, M.
, 
Sauer, M.
, 
Abas, L.
, 
Luschnig, C.
, &amp; 
Friml, J.
 (2008). Differential degradation of PIN2 auxin efflux carrier by retromer‐dependent vacuolar targeting. PNAS, 105(46), 17812–17817. 10.1073/pnas.0808073105

</cite> [<a href="https://doi.org/10.1073/pnas.0808073105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2584678/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19004783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Differential%20degradation%20of%20PIN2%20auxin%20efflux%20carrier%20by%20retromer%E2%80%90dependent%20vacuolar%20targeting&amp;author=J.%20Kleine%E2%80%90Vehn&amp;author=J.%20Leitner&amp;author=M.%20Zwiewka&amp;author=M.%20Sauer&amp;author=L.%20Abas&amp;volume=105&amp;issue=46&amp;publication_year=2008&amp;pages=17812-17817&amp;pmid=19004783&amp;doi=10.1073/pnas.0808073105&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0032">
<cite>

Krogh, A.
, 
Larsson, B.
, 
von Heijne, G.
, &amp; 
Sonnhammer, E. L.
 (2001). Predicting transmembrane protein topology with a hidden Markov model: Application to complete genomes. Journal of Molecular Biology, 305(3), 567–580. 10.1006/jmbi.2000.4315

</cite> [<a href="https://doi.org/10.1006/jmbi.2000.4315" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11152613/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Molecular%20Biology&amp;title=Predicting%20transmembrane%20protein%20topology%20with%20a%20hidden%20Markov%20model:%20Application%20to%20complete%20genomes&amp;author=A.%20Krogh&amp;author=B.%20Larsson&amp;author=G.%20von%20Heijne&amp;author=E.%20L.%20Sonnhammer&amp;volume=305&amp;issue=3&amp;publication_year=2001&amp;pages=567-580&amp;pmid=11152613&amp;doi=10.1006/jmbi.2000.4315&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0033">
<cite>

Li, H.
, 
Li, L.
, 
Liu, N.
, 
Chen, S.
, 
Shao, L.
, 
Sekiya, N.
, &amp; 
Zhang, X.
 (2022). Root efficiency and water use regulation relating to rooting depth of winter wheat. Agricultural Water Management, 269, 107710. 10.1016/j.agwat.2022.107710
</cite> [<a href="https://doi.org/10.1016/j.agwat.2022.107710" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Agricultural%20Water%20Management&amp;title=Root%20efficiency%20and%20water%20use%20regulation%20relating%20to%20rooting%20depth%20of%20winter%20wheat&amp;author=H.%20Li&amp;author=L.%20Li&amp;author=N.%20Liu&amp;author=S.%20Chen&amp;author=L.%20Shao&amp;volume=269&amp;publication_year=2022&amp;pages=107710&amp;doi=10.1016/j.agwat.2022.107710&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0034">
<cite>

Liu, G.
, &amp; 
Godwin, I. D.
 (2012). Highly efficient sorghum transformation. Plant Cell Reports, 31(6), 999–1007. 10.1007/s00299-011-1218-4

</cite> [<a href="https://doi.org/10.1007/s00299-011-1218-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3351618/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22234443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Reports&amp;title=Highly%20efficient%20sorghum%20transformation&amp;author=G.%20Liu&amp;author=I.%20D.%20Godwin&amp;volume=31&amp;issue=6&amp;publication_year=2012&amp;pages=999-1007&amp;pmid=22234443&amp;doi=10.1007/s00299-011-1218-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0035">
<cite>

Lu, G.
, 
Coneva, V.
, 
Casaretto, J. A.
, 
Ying, S.
, 
Mahmood, K.
, 
Liu, F.
, 
Nambara, E.
, 
Bi, Y. M.
, &amp; 
Rothstein, S. J.
 (2015). OsPIN5b modulates rice (<em>Oryza sativa</em>) plant architecture and yield by changing auxin homeostasis, transport and distribution. Plant Journal, 83(5), 913–925. 10.1111/tpj.12939
</cite> [<a href="https://doi.org/10.1111/tpj.12939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26213119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Journal&amp;title=OsPIN5b%20modulates%20rice%20(Oryza%20sativa)%20plant%20architecture%20and%20yield%20by%20changing%20auxin%20homeostasis,%20transport%20and%20distribution&amp;author=G.%20Lu&amp;author=V.%20Coneva&amp;author=J.%20A.%20Casaretto&amp;author=S.%20Ying&amp;author=K.%20Mahmood&amp;volume=83&amp;issue=5&amp;publication_year=2015&amp;pages=913-925&amp;pmid=26213119&amp;doi=10.1111/tpj.12939&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0036">
<cite>

Luschnig, C.
, &amp; 
Vert, G.
 (2014). The dynamics of plant plasma membrane proteins: PINs and beyond. Development, 141(15), 2924–2938. 10.1242/dev.103424

</cite> [<a href="https://doi.org/10.1242/dev.103424" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25053426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Development&amp;title=The%20dynamics%20of%20plant%20plasma%20membrane%20proteins:%20PINs%20and%20beyond&amp;author=C.%20Luschnig&amp;author=G.%20Vert&amp;volume=141&amp;issue=15&amp;publication_year=2014&amp;pages=2924-2938&amp;pmid=25053426&amp;doi=10.1242/dev.103424&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0037">
<cite>

Lynch, J. P.
 (2013). Steep, cheap and deep: An ideotype to optimize water and N acquisition by maize root systems. Annals of Botany, 112(2), 347–357. 10.1093/aob/mcs293

</cite> [<a href="https://doi.org/10.1093/aob/mcs293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3698384/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23328767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Botany&amp;title=Steep,%20cheap%20and%20deep:%20An%20ideotype%20to%20optimize%20water%20and%20N%20acquisition%20by%20maize%20root%20systems&amp;author=J.%20P.%20Lynch&amp;volume=112&amp;issue=2&amp;publication_year=2013&amp;pages=347-357&amp;pmid=23328767&amp;doi=10.1093/aob/mcs293&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0038">
<cite>

Lynch, J. P.
 (2019). Root phenotypes for improved nutrient capture: An underexploited opportunity for global agriculture. New Phytologist, 223(2), 548–564. 10.1111/nph.15738

</cite> [<a href="https://doi.org/10.1111/nph.15738" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30746704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Root%20phenotypes%20for%20improved%20nutrient%20capture:%20An%20underexploited%20opportunity%20for%20global%20agriculture&amp;author=J.%20P.%20Lynch&amp;volume=223&amp;issue=2&amp;publication_year=2019&amp;pages=548-564&amp;pmid=30746704&amp;doi=10.1111/nph.15738&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0039">
<cite>

Manschadi, A. M.
, 
Christopher, J.
, 
deVoil, P.
, &amp; 
Hammer, G. L.
 (2006). The role of root architectural traits in adaptation of wheat to water‐limited environments. Functional Plant Biology, 33(9), 823–837. 10.1071/fp06055

</cite> [<a href="https://doi.org/10.1071/fp06055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32689293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Functional%20Plant%20Biology&amp;title=The%20role%20of%20root%20architectural%20traits%20in%20adaptation%20of%20wheat%20to%20water%E2%80%90limited%20environments&amp;author=A.%20M.%20Manschadi&amp;author=J.%20Christopher&amp;author=P.%20deVoil&amp;author=G.%20L.%20Hammer&amp;volume=33&amp;issue=9&amp;publication_year=2006&amp;pages=823-837&amp;pmid=32689293&amp;doi=10.1071/fp06055&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0040">
<cite>

Massel, K.
, 
Lam, Y.
, 
Hintzsche, J.
, 
Lester, N.
, 
Botella, J. R.
, &amp; 
Godwin, I. D.
 (2022). Endogenous U6 promoters improve CRISPR/Cas9 editing efficiencies in <em>Sorghum bicolor</em> and show potential for applications in other cereals. Plant Cell Reports, 41(2), 489–492. 10.1007/s00299-021-02816-z

</cite> [<a href="https://doi.org/10.1007/s00299-021-02816-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34854968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Reports&amp;title=Endogenous%20U6%20promoters%20improve%20CRISPR/Cas9%20editing%20efficiencies%20in%20Sorghum%20bicolor%20and%20show%20potential%20for%20applications%20in%20other%20cereals&amp;author=K.%20Massel&amp;author=Y.%20Lam&amp;author=J.%20Hintzsche&amp;author=N.%20Lester&amp;author=J.%20R.%20Botella&amp;volume=41&amp;issue=2&amp;publication_year=2022&amp;pages=489-492&amp;pmid=34854968&amp;doi=10.1007/s00299-021-02816-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0041">
<cite>

Mirza, J. I.
, 
Olsen, G. M.
, 
Iversen, T.‐H.
, &amp; 
Maher, E. P.
 (1984). The growth and gravitropic responses of wild‐type and auxinresistant mutants of <em>Arabidopsis thaliana</em>
. Physiologia Plantarum, 60(4), 516–522. 10.1111/j.1399-3054.1984.tb04920.x
</cite> [<a href="https://doi.org/10.1111/j.1399-3054.1984.tb04920.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11541083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Physiologia%20Plantarum&amp;title=The%20growth%20and%20gravitropic%20responses%20of%20wild%E2%80%90type%20and%20auxinresistant%20mutants%20of%20Arabidopsis%20thaliana&amp;author=J.%20I.%20Mirza&amp;author=G.%20M.%20Olsen&amp;author=T.%E2%80%90H.%20Iversen&amp;author=E.%20P.%20Maher&amp;volume=60&amp;issue=4&amp;publication_year=1984&amp;pages=516-522&amp;pmid=11541083&amp;doi=10.1111/j.1399-3054.1984.tb04920.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0042">
<cite>

Müller, A.
, 
Guan, C.
, 
Gälweiler, L.
, 
Tänzler, P.
, 
Huijser, P.
, 
Marchant, A.
, 
Parry, G.
, 
Bennett, M.
, 
Wisman, E.
, &amp; 
Palme, K.
 (1998). 
<em>AtPIN2</em> defines a locus of <em>Arabidopsis</em> for root gravitropism control. EMBO Journal, 17(23), 6903–6911. 10.1093/emboj/17.23.6903

</cite> [<a href="https://doi.org/10.1093/emboj/17.23.6903" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1171038/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9843496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Journal&amp;title=AtPIN2%20defines%20a%20locus%20of%20Arabidopsis%20for%20root%20gravitropism%20control&amp;author=A.%20M%C3%BCller&amp;author=C.%20Guan&amp;author=L.%20G%C3%A4lweiler&amp;author=P.%20T%C3%A4nzler&amp;author=P.%20Huijser&amp;volume=17&amp;issue=23&amp;publication_year=1998&amp;pages=6903-6911&amp;pmid=9843496&amp;doi=10.1093/emboj/17.23.6903&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0043">
<cite>

Nakamoto, T.
, &amp; 
Oyanagi, A.
 (1996). The configuration of the seminal roots of <em>Triticum aestivum</em> L. (Poaceae). Journal of Plant Research, 109(4), 375–380. 10.1007/BF02344552

</cite> [<a href="https://doi.org/10.1007/BF02344552" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11539860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Plant%20Research&amp;title=The%20configuration%20of%20the%20seminal%20roots%20of%20Triticum%20aestivum%20L.%20(Poaceae)&amp;author=T.%20Nakamoto&amp;author=A.%20Oyanagi&amp;volume=109&amp;issue=4&amp;publication_year=1996&amp;pages=375-380&amp;pmid=11539860&amp;doi=10.1007/BF02344552&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0065">
<cite>
Posit Team
. (2025). RStudio: Integrated development environment for R. Posit Software. <a href="http://www.posit.co" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.posit.co</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=RStudio:%20Integrated%20development%20environment%20for%20R&amp;publication_year=2025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0044">
<cite>

Rich, S. M.
, &amp; 
Watt, M.
 (2013). Soil conditions and cereal root system architecture: Review and considerations for linking Darwin and Weaver. Journal of Experimental Botany, 64(5), 1193–1208. 10.1093/jxb/ert043

</cite> [<a href="https://doi.org/10.1093/jxb/ert043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23505309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Soil%20conditions%20and%20cereal%20root%20system%20architecture:%20Review%20and%20considerations%20for%20linking%20Darwin%20and%20Weaver&amp;author=S.%20M.%20Rich&amp;author=M.%20Watt&amp;volume=64&amp;issue=5&amp;publication_year=2013&amp;pages=1193-1208&amp;pmid=23505309&amp;doi=10.1093/jxb/ert043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0045">
<cite>

Richard, C. A.
, 
Hickey, L. T.
, 
Fletcher, S.
, 
Jennings, R.
, 
Chenu, K.
, &amp; 
Christopher, J. T.
 (2015). High‐throughput phenotyping of seminal root traits in wheat. Plant Methods, 11, Article 13. 10.1186/s13007-015-0055-9

</cite> [<a href="https://doi.org/10.1186/s13007-015-0055-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4351910/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25750658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Methods&amp;title=High%E2%80%90throughput%20phenotyping%20of%20seminal%20root%20traits%20in%20wheat&amp;author=C.%20A.%20Richard&amp;author=L.%20T.%20Hickey&amp;author=S.%20Fletcher&amp;author=R.%20Jennings&amp;author=K.%20Chenu&amp;volume=11&amp;publication_year=2015&amp;pages=13&amp;pmid=25750658&amp;doi=10.1186/s13007-015-0055-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0046">
<cite>

Robertson, M. J.
, 
Fukai, S.
, 
Ludlow, M. M.
, &amp; 
Hammer, G. L.
 (1993). Water extraction by grain sorghum in a sub‐humid environment. I. Analysis of the water extraction pattern. Field Crops Research, 33(1), 81–97. 10.1016/0378-4290(93)90095-5
</cite> [<a href="https://doi.org/10.1016/0378-4290(93)90095-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Field%20Crops%20Research&amp;title=Water%20extraction%20by%20grain%20sorghum%20in%20a%20sub%E2%80%90humid%20environment.%20I.%20Analysis%20of%20the%20water%20extraction%20pattern&amp;author=M.%20J.%20Robertson&amp;author=S.%20Fukai&amp;author=M.%20M.%20Ludlow&amp;author=G.%20L.%20Hammer&amp;volume=33&amp;issue=1&amp;publication_year=1993&amp;pages=81-97&amp;doi=10.1016/0378-4290(93)90095-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0047">
<cite>

Robinson, H.
, 
Hickey, L.
, 
Richard, C.
, 
Mace, E.
, 
Kelly, A.
, 
Borrell, A.
, 
Franckowiak, J.
, &amp; 
Fox, G.
 (2016). Genomic regions influencing seminal root traits in barley. The Plant Genome, 9(1), plantgenome2015.03.0012. 10.3835/plantgenome2015.03.0012
</cite> [<a href="https://doi.org/10.3835/plantgenome2015.03.0012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27898766/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Genome&amp;title=Genomic%20regions%20influencing%20seminal%20root%20traits%20in%20barley&amp;author=H.%20Robinson&amp;author=L.%20Hickey&amp;author=C.%20Richard&amp;author=E.%20Mace&amp;author=A.%20Kelly&amp;volume=9&amp;issue=1&amp;publication_year=2016&amp;pmid=27898766&amp;doi=10.3835/plantgenome2015.03.0012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0048">
<cite>

Roychoudhry, S.
, 
Del Bianco, M.
, 
Kieffer, M.
, &amp; 
Kepinski, S.
 (2013). Auxin controls gravitropic setpoint angle in higher plant lateral branches. Current Biology, 23(15), 1497–1504. 10.1016/j.cub.2013.06.034

</cite> [<a href="https://doi.org/10.1016/j.cub.2013.06.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23891109/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Biology&amp;title=Auxin%20controls%20gravitropic%20setpoint%20angle%20in%20higher%20plant%20lateral%20branches&amp;author=S.%20Roychoudhry&amp;author=M.%20Del%20Bianco&amp;author=M.%20Kieffer&amp;author=S.%20Kepinski&amp;volume=23&amp;issue=15&amp;publication_year=2013&amp;pages=1497-1504&amp;pmid=23891109&amp;doi=10.1016/j.cub.2013.06.034&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0049">
<cite>

Sachs, J.
 (1882). Arbeiten Des Botanischen Instituts in Würzburg (Vol. 2). Wilhelm Engelmann.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Arbeiten%20Des%20Botanischen%20Instituts%20in%20W%C3%BCrzburg&amp;author=J.%20Sachs&amp;publication_year=1882&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0050">
<cite>

Schneider, C. A.
, 
Rasband, W. S.
, &amp; 
Eliceiri, K. W.
 (2012). NIH image to ImageJ: 25 Years of image analysis. Nature Methods, 9(7), 671–675. 10.1038/nmeth.2089

</cite> [<a href="https://doi.org/10.1038/nmeth.2089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5554542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22930834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Methods&amp;title=NIH%20image%20to%20ImageJ:%2025%20Years%20of%20image%20analysis&amp;author=C.%20A.%20Schneider&amp;author=W.%20S.%20Rasband&amp;author=K.%20W.%20Eliceiri&amp;volume=9&amp;issue=7&amp;publication_year=2012&amp;pages=671-675&amp;pmid=22930834&amp;doi=10.1038/nmeth.2089&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0051">
<cite>

Seethepalli, A.
, 
Guo, H.
, 
Liu, X.
, 
Griffiths, M.
, 
Almtarfi, H.
, 
Li, Z.
, 
Liu, S.
, 
Zare, A.
, 
Fritschi, F. B.
, 
Blancaflor, E. B.
, 
Ma, X.‐F.
, &amp; 
York, L. M.
 (2020). RhizoVision Crown: An integrated hardware and software platform for root crown phenotyping. Plant Phenomics, 2020, 3074916. 10.34133/2020/3074916

</cite> [<a href="https://doi.org/10.34133/2020/3074916" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7706346/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33313547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Phenomics&amp;title=RhizoVision%20Crown:%20An%20integrated%20hardware%20and%20software%20platform%20for%20root%20crown%20phenotyping&amp;author=A.%20Seethepalli&amp;author=H.%20Guo&amp;author=X.%20Liu&amp;author=M.%20Griffiths&amp;author=H.%20Almtarfi&amp;volume=2020&amp;publication_year=2020&amp;pages=3074916&amp;pmid=33313547&amp;doi=10.34133/2020/3074916&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0052">
<cite>

Stanga, J.
, 
Strohm, A.
, &amp; 
Masson, P. H.
 (2011). Studying starch content and sedimentation of amyloplast statoliths in Arabidopsis roots. In Jarvis R. P. (Ed.), Chloroplast research in Arabidopsis: Methods and protocols (Vol. 1, pp. 103–111). Humana Press. 10.1007/978-1-61779-234-2_7
</cite> [<a href="https://doi.org/10.1007/978-1-61779-234-2_7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21822835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Chloroplast%20research%20in%20Arabidopsis:%20Methods%20and%20protocols&amp;author=J.%20Stanga&amp;author=A.%20Strohm&amp;author=P.%20H.%20Masson&amp;publication_year=2011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0053">
<cite>

Swarup, R.
, 
Friml, J.
, 
Marchant, A.
, 
Ljung, K.
, 
Sandberg, G.
, 
Palme, K.
, &amp; 
Bennett, M.
 (2001). Localization of the auxin permease AUX1 suggests two functionally distinct hormone transport pathways operate in the Arabidopsis root apex. Genes &amp; Development, 15(20), 2648–2653. 10.1101/gad.210501

</cite> [<a href="https://doi.org/10.1101/gad.210501" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC312818/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11641271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20&amp;%20Development&amp;title=Localization%20of%20the%20auxin%20permease%20AUX1%20suggests%20two%20functionally%20distinct%20hormone%20transport%20pathways%20operate%20in%20the%20Arabidopsis%20root%20apex&amp;author=R.%20Swarup&amp;author=J.%20Friml&amp;author=A.%20Marchant&amp;author=K.%20Ljung&amp;author=G.%20Sandberg&amp;volume=15&amp;issue=20&amp;publication_year=2001&amp;pages=2648-2653&amp;pmid=11641271&amp;doi=10.1101/gad.210501&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0054">
<cite>

Swarup, R.
, 
Kramer, E. M.
, 
Perry, P.
, 
Knox, K.
, 
Leyser, H. M. O.
, 
Haseloff, J.
, 
Beemster, G. T. S.
, 
Bhalerao, R.
, &amp; 
Bennett, M. J.
 (2005). Root gravitropism requires lateral root cap and epidermal cells for transport and response to a mobile auxin signal. Nature Cell Biology, 7(11), 1057–1065. 10.1038/ncb1316

</cite> [<a href="https://doi.org/10.1038/ncb1316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16244669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Cell%20Biology&amp;title=Root%20gravitropism%20requires%20lateral%20root%20cap%20and%20epidermal%20cells%20for%20transport%20and%20response%20to%20a%20mobile%20auxin%20signal&amp;author=R.%20Swarup&amp;author=E.%20M.%20Kramer&amp;author=P.%20Perry&amp;author=K.%20Knox&amp;author=H.%20M.%20O.%20Leyser&amp;volume=7&amp;issue=11&amp;publication_year=2005&amp;pages=1057-1065&amp;pmid=16244669&amp;doi=10.1038/ncb1316&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0055">
<cite>
Synthego
. (2024). Synthego performance analysis (ICE Analysis v3.0) [Computer software]. Synthego.</cite> [<a href="https://scholar.google.com/scholar_lookup?Synthego%20.%20(2024).%20Synthego%20performance%20analysis%20(ICE%20Analysis%20v3.0)%20%5BComputer%20software%5D.%20Synthego." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0057">
<cite>

Tilman, D.
 (1985). The resource‐ratio hypothesis of plant succession. The American Naturalist, 125(6), 827–852. 10.1086/284382
</cite> [<a href="https://doi.org/10.1086/284382" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20American%20Naturalist&amp;title=The%20resource%E2%80%90ratio%20hypothesis%20of%20plant%20succession&amp;author=D.%20Tilman&amp;volume=125&amp;issue=6&amp;publication_year=1985&amp;pages=827-852&amp;doi=10.1086/284382&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0058">
<cite>

Tsugeki, R.
, &amp; 
Fedoroff, N. V.
 (1999). Genetic ablation of root cap cells in Arabidopsis. PNAS, 96(22), 12941–12946. 10.1073/pnas.96.22.12941

</cite> [<a href="https://doi.org/10.1073/pnas.96.22.12941" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC23173/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10536027/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PNAS&amp;title=Genetic%20ablation%20of%20root%20cap%20cells%20in%20Arabidopsis&amp;author=R.%20Tsugeki&amp;author=N.%20V.%20Fedoroff&amp;volume=96&amp;issue=22&amp;publication_year=1999&amp;pages=12941-12946&amp;pmid=10536027&amp;doi=10.1073/pnas.96.22.12941&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0059">
<cite>

Uga, Y.
 (2021). Challenges to design‐oriented breeding of root system architecture adapted to climate change. Breed Science, 71(1), 3–12. 10.1270/jsbbs.20118
</cite> [<a href="https://doi.org/10.1270/jsbbs.20118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7973499/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33762871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Breed%20Science&amp;title=Challenges%20to%20design%E2%80%90oriented%20breeding%20of%20root%20system%20architecture%20adapted%20to%20climate%20change&amp;author=Y.%20Uga&amp;volume=71&amp;issue=1&amp;publication_year=2021&amp;pages=3-12&amp;pmid=33762871&amp;doi=10.1270/jsbbs.20118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0060">
<cite>

Vanhala, T. K.
, 
Van Rijn, C. P. E.
, 
Buntjer, J.
, 
Stam, P.
, 
Nevo, E.
, 
Poorter, H.
, &amp; 
Van Eeuwijk, F. A.
 (2004). Environmental, phenotypic and genetic variation of wild barley (<em>Hordeum spontaneum</em>) from Israel. Euphytica, 137, 297–309. 10.1023/B:EUPH.0000040450.91906.c5
</cite> [<a href="https://doi.org/10.1023/B:EUPH.0000040450.91906.c5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Euphytica&amp;title=Environmental,%20phenotypic%20and%20genetic%20variation%20of%20wild%20barley%20(Hordeum%20spontaneum)%20from%20Israel&amp;author=T.%20K.%20Vanhala&amp;author=C.%20P.%20E.%20Van%20Rijn&amp;author=J.%20Buntjer&amp;author=P.%20Stam&amp;author=E.%20Nevo&amp;volume=137&amp;publication_year=2004&amp;pages=297-309&amp;doi=10.1023/B:EUPH.0000040450.91906.c5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0061">
<cite>

Voss‐Fels, K. P.
, 
Robinson, H.
, 
Mudge, S. R.
, 
Richard, C.
, 
Newman, S.
, 
Wittkop, B.
, 
Stahl, A.
, 
Friedt, W.
, 
Frisch, M.
, 
Gabur, I.
, 
Miller‐Cooper, A.
, 
Campbell, B. C.
, 
Kelly, A.
, 
Fox, G.
, 
Christopher, J.
, 
Christopher, M.
, 
Chenu, K.
, 
Franckowiak, J.
, 
Mace, E. S.
, … 
Hickey, L. T.
 (2018). VERNALIZATION1 modulates root system architecture in wheat and barley. Molecular Plant, 11(1), 226–229. 10.1016/j.molp.2017.10.005

</cite> [<a href="https://doi.org/10.1016/j.molp.2017.10.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29056533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant&amp;title=VERNALIZATION1%20modulates%20root%20system%20architecture%20in%20wheat%20and%20barley&amp;author=K.%20P.%20Voss%E2%80%90Fels&amp;author=H.%20Robinson&amp;author=S.%20R.%20Mudge&amp;author=C.%20Richard&amp;author=S.%20Newman&amp;volume=11&amp;issue=1&amp;publication_year=2018&amp;pages=226-229&amp;pmid=29056533&amp;doi=10.1016/j.molp.2017.10.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0062">
<cite>

Wickham, H.
, 
Chang, W.
, &amp; 
Wickham, M. H.
 (2016). Package ‘ggplot2’: Create elegant data visualisations using the grammar of graphics (Version 2) [Computer software]. Springer‐Verlag.</cite> [<a href="https://scholar.google.com/scholar_lookup?Wickham,%20H.%20,%20Chang,%20W.%20,%20&amp;%20Wickham,%20M.%20H.%20(2016).%20Package%20%E2%80%98ggplot2%E2%80%99:%20Create%20elegant%20data%20visualisations%20using%20the%20grammar%20of%20graphics%20(Version%202)%C2%A0%5BComputer%20software%5D.%20Springer%E2%80%90Verlag." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270061-bib-0063">
<cite>

Yue, R.
, 
Tie, S.
, 
Sun, T.
, 
Zhang, L.
, 
Yang, Y.
, 
Qi, J.
, 
Yan, S.
, 
Han, X.
, 
Wang, H.
, &amp; 
Shen, C.
 (2015). Genome‐wide identification and expression profiling analysis of ZmPIN, ZmPILS, ZmLAX and ZmABCB auxin transporter gene families in maize (<em>Zea mays</em> L.) under various abiotic stresses. PLoS ONE, 10(3), e0118751. 10.1371/journal.pone.0118751

</cite> [<a href="https://doi.org/10.1371/journal.pone.0118751" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4351008/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25742625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Genome%E2%80%90wide%20identification%20and%20expression%20profiling%20analysis%20of%20ZmPIN,%20ZmPILS,%20ZmLAX%20and%20ZmABCB%20auxin%20transporter%20gene%20families%20in%20maize%20(Zea%20mays%20L.)%20under%20various%20abiotic%20stresses&amp;author=R.%20Yue&amp;author=S.%20Tie&amp;author=T.%20Sun&amp;author=L.%20Zhang&amp;author=Y.%20Yang&amp;volume=10&amp;issue=3&amp;publication_year=2015&amp;pages=e0118751&amp;pmid=25742625&amp;doi=10.1371/journal.pone.0118751&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>
<strong>Figure S1</strong>. Illustration of <em>PIN2</em> genome edited knock out lines with impact on protein structure and transmembrane domains compared to wild type.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328839/bin/TPG2-18-e70061-s001.jpeg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70061-s001.jpeg</a><sup> (52.7KB, jpeg) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>
<strong>Figure S2</strong>. Canopy and root trait results from narrow rhizobox experiments conducted using different planting density.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328839/bin/TPG2-18-e70061-s006.jpeg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70061-s006.jpeg</a><sup> (154.8KB, jpeg) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><p>
<strong>Figure S3</strong>. PIN2 in barley regulates downward root growth leading to steeper and narrower root architecture without influencing biomass.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328839/bin/TPG2-18-e70061-s008.jpeg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70061-s008.jpeg</a><sup> (82.2KB, jpeg) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p"><p>
<strong>Figure S4</strong>. Maturity trial reveal no significant difference to canopy and phenology in <em>pin2</em>.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328839/bin/TPG2-18-e70061-s004.jpeg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70061-s004.jpeg</a><sup> (57.8KB, jpeg) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p"><p>
<strong>Figure S5</strong>. Summary of hypothesis: disrupted auxin transport in <em>pin2</em> genome edited barley inhibits gravitropic response and alters root development.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328839/bin/TPG2-18-e70061-s002.jpeg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70061-s002.jpeg</a><sup> (111.7KB, jpeg) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p"><p>
<strong>Table S1</strong>. Fixed and random effects table for the linear mixed model (LMM) fitted to the trait of interest.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328839/bin/TPG2-18-e70061-s007.jpeg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70061-s007.jpeg</a><sup> (69.6KB, jpeg) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p"><p>
<strong>Table S2</strong>. Synteny of <em>PIN</em> genes across cereals.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328839/bin/TPG2-18-e70061-s005.jpeg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70061-s005.jpeg</a><sup> (49.8KB, jpeg) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="caption p"><p>
<strong>Table S3</strong>. Table measured shoot growth angles for WT and <em>PIN2 KO</em>.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12328839/bin/TPG2-18-e70061-s003.jpeg" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70061-s003.jpeg</a><sup> (6.3KB, jpeg) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data generated or analyzed during this study are included in this published article and its Supporting Information files.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70061"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70061.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1017.1 KB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12328839/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12328839/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12328839%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12328839/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12328839/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12328839/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40770375/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12328839/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40770375/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12328839/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12328839/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="fPBt5k5BWjMGjeCm9DeEIqC0Vna3BD9OrshxWYzDovwIPb6jTRwA7AMe4dE6hEaq">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
