
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Structural and functional analysis of the accessory gene regulators of Staphylococcus aureus and Staphylococcus epidermidis: an in Silico approach - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53149F8AEF5C5305149F00136F0B87.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcmicr">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374375/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Microbiology">
<meta name="citation_title" content="Structural and functional analysis of the accessory gene regulators of Staphylococcus aureus and Staphylococcus epidermidis: an in Silico approach">
<meta name="citation_author" content="Subhamoy Dey">
<meta name="citation_author_institution" content="Centre for Life Sciences, Vidyasagar University, Midnapore, West Bengal 721102 India">
<meta name="citation_author_institution" content="Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India">
<meta name="citation_author" content="Tuhin Manna">
<meta name="citation_author_institution" content="Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India">
<meta name="citation_author" content="Kartik Chandra Guchhait">
<meta name="citation_author_institution" content="Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India">
<meta name="citation_author" content="Monalisha Karmakar">
<meta name="citation_author_institution" content="Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India">
<meta name="citation_author" content="Debarati Jana">
<meta name="citation_author_institution" content="Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India">
<meta name="citation_author" content="Priyanka Raul">
<meta name="citation_author_institution" content="Centre for Life Sciences, Vidyasagar University, Midnapore, West Bengal 721102 India">
<meta name="citation_author_institution" content="Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India">
<meta name="citation_author" content="Saroj Ballav">
<meta name="citation_author_institution" content="Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India">
<meta name="citation_author" content="Mousumi Manna">
<meta name="citation_author_institution" content="Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India">
<meta name="citation_author" content="Subrata Hazra">
<meta name="citation_author_institution" content="Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India">
<meta name="citation_author" content="Amiya Kumar Panda">
<meta name="citation_author_institution" content="Department of Chemistry, Vidyasagar University, Midnapore, West Bengal 721102 India">
<meta name="citation_author_institution" content="Department of Chemistry, Rani Rashmoni Green University, Singur, West Bengal 712409 India">
<meta name="citation_author" content="Chandradipa Ghosh">
<meta name="citation_author_institution" content="Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India">
<meta name="citation_publication_date" content="2025 Aug 23">
<meta name="citation_volume" content="25">
<meta name="citation_firstpage" content="538">
<meta name="citation_doi" content="10.1186/s12866-025-04257-0">
<meta name="citation_pmid" content="40849622">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374375/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374375/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374375/pdf/12866_2025_Article_4257.pdf">
<meta name="description" content="Staphylococcus aureus and Staphylococcus epidermidis are tenacious pathogens that cause toxic shock syndrome. Accessory gene regulator (Agr) of Staphylococcus sp. controls the expression of multiple genes that encode virulence properties. ...">
<meta name="og:title" content="Structural and functional analysis of the accessory gene regulators of Staphylococcus aureus and Staphylococcus epidermidis: an in Silico approach">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Staphylococcus aureus and Staphylococcus epidermidis are tenacious pathogens that cause toxic shock syndrome. Accessory gene regulator (Agr) of Staphylococcus sp. controls the expression of multiple genes that encode virulence properties. ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374375/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12374375">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12866-025-04257-0"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12866_2025_Article_4257.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374375%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12374375/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12374375/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374375/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcmicr.png" alt="BMC Microbiology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Microbiology" title="Link to BMC Microbiology" shape="default" href="https://bmcmicrobiol.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Microbiol</button></div>. 2025 Aug 23;25:538. doi: <a href="https://doi.org/10.1186/s12866-025-04257-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12866-025-04257-0</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Microbiol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Microbiol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Structural and functional analysis of the accessory gene regulators of <em>Staphylococcus aureus</em> and <em>Staphylococcus epidermidis</em>: an in Silico approach</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dey%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Subhamoy Dey</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Subhamoy Dey</span></h3>
<div class="p">
<sup>1</sup>Centre for Life Sciences, Vidyasagar University, Midnapore, West Bengal 721102 India </div>
<div class="p">
<sup>2</sup>Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dey%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Subhamoy Dey</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Manna%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Tuhin Manna</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Tuhin Manna</span></h3>
<div class="p">
<sup>2</sup>Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Manna%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tuhin Manna</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guchhait%20KC%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Kartik Chandra Guchhait</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Kartik Chandra Guchhait</span></h3>
<div class="p">
<sup>2</sup>Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guchhait%20KC%22%5BAuthor%5D" class="usa-link"><span class="name western">Kartik Chandra Guchhait</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Karmakar%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Monalisha Karmakar</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Monalisha Karmakar</span></h3>
<div class="p">
<sup>2</sup>Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Karmakar%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Monalisha Karmakar</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jana%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Debarati Jana</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Debarati Jana</span></h3>
<div class="p">
<sup>2</sup>Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jana%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Debarati Jana</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Raul%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Priyanka Raul</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Priyanka Raul</span></h3>
<div class="p">
<sup>1</sup>Centre for Life Sciences, Vidyasagar University, Midnapore, West Bengal 721102 India </div>
<div class="p">
<sup>2</sup>Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Raul%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Priyanka Raul</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ballav%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Saroj Ballav</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Saroj Ballav</span></h3>
<div class="p">
<sup>2</sup>Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ballav%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Saroj Ballav</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Manna%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Mousumi Manna</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Mousumi Manna</span></h3>
<div class="p">
<sup>2</sup>Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Manna%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mousumi Manna</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hazra%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Subrata Hazra</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Subrata Hazra</span></h3>
<div class="p">
<sup>2</sup>Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hazra%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Subrata Hazra</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Panda%20AK%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Amiya Kumar Panda</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Amiya Kumar Panda</span></h3>
<div class="p">
<sup>3</sup>Department of Chemistry, Vidyasagar University, Midnapore, West Bengal 721102 India </div>
<div class="p">
<sup>4</sup>Department of Chemistry, Rani Rashmoni Green University, Singur, West Bengal 712409 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Panda%20AK%22%5BAuthor%5D" class="usa-link"><span class="name western">Amiya Kumar Panda</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ghosh%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Chandradipa Ghosh</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Chandradipa Ghosh</span></h3>
<div class="p">
<sup>2</sup>Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ghosh%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chandradipa Ghosh</span></a>
</div>
</div>
<sup>2,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Centre for Life Sciences, Vidyasagar University, Midnapore, West Bengal 721102 India </div>
<div id="Aff2">
<sup>2</sup>Department of Human Physiology, Vidyasagar University, Midnapore, West Bengal 721102 India </div>
<div id="Aff3">
<sup>3</sup>Department of Chemistry, Vidyasagar University, Midnapore, West Bengal 721102 India </div>
<div id="Aff4">
<sup>4</sup>Department of Chemistry, Rani Rashmoni Green University, Singur, West Bengal 712409 India </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 12; Accepted 2025 Jul 24; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the articleâ€™s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12374375Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40849622/" class="usa-link">40849622</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1"><em>Staphylococcus aureus</em> and <em>Staphylococcus epidermidis</em> are tenacious pathogens that cause toxic shock syndrome. Accessory gene regulator (Agr) of <em>Staphylococcus</em> sp. controls the expression of multiple genes that encode virulence properties. Evolutionary covariance of accessory gene regulators of selected strains of two <em>Staphylococcus</em> sp. was entrenched through multiple sequence alignment, relative synonymous codon usage, codon adaptation index and compositional analysis. Artificial intelligence and machine learning based AlphaFold and TrRosetta were used to determine the tertiary structures of the proteins. Structure-based ab initio models could forecast subcellular localization, domain length, molecular docking, and simulation of Agrs in the isolates belonging to <em>Staphylococcus</em> sp.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">AT ending codons are preferred over GC ending codons. Besides, the mutational pressure has been found to be one of the causative factors in shaping the codon usage biasness. Topological investigations reveal the existence of AgrA and AgrD in the cytosol, while AgrB and AgrC to reside in the cellular membrane. All Agrs are acidic and stable, except AgrB. Secondary structural studies showed that Agrs mostly consist of Î±-helix followed by random coils that preferentially remain in the transmembrane region. Protein-protein docking studies using the HDOCK server demonstrated that AgrA has stronger binding affinity with AgrC in <em>S. epidermidis</em> isolates than the same in <em>S. aureus</em>. By analysing the docking potential of AgrB and AgrD, it has been found that <em>S. aureus</em> possesses higher docking score than <em>S. epidermidis</em>.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par3">Such compressive investigations could provide crucial insights into the structural features of Agrs in <em>S. aureus</em> and <em>S. epidermidis</em>, that are actively implicated in quorum sensing signalling-mediated virulence factor regulation and help in the identification of new Agr-dependent quorum sensing inhibitors. AgrA and AgrC are, therefore, appear to be seemingly promising in the management of bacterial infections and are apprehended to be useful therapeutic targets for the discovery of potential antimicrobial drugs.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12866-025-04257-0.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Agr, QS sytem, <em>In Silico</em>, <em>S. aureus</em>, <em>S. epidermidis</em></p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par19">Nosocomial infections caused by <em>Staphylococcus aureus</em> and <em>Staphylococcus epidermidis</em>, two of the most common bacteria found in healthcare settings, are becoming increasingly threatening. <em>S. aureus</em> is one of the most lethal bacterium that infects and colonizes healthy people with weakened immune systems as well as the hospitalized patients with reduced immunity [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. Conventional concept holds that <em>S. epidermidis</em> is a harmless commensal bacterium found on human skin; however, contemporary research works demonstrate that <em>S. epidermidis</em> can also cause bacteraemia and severe sepsis [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. One of the most common causes of illnesses is their inclination for obstinate adherence and long term persistence on medical devices, and that directs the bacterium to continue its infectivity and subsequent pathogenicity [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. At least one billion dollar is spent annually by the public health system in the US on <em>S. epidermidis</em> borne infections. Biofilms, which have mosaic polysaccharide structures consisting microbe agglomerations, are known to be responsible for its ability to remain on medical devices [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. A heterogeneous matrix is formed by the biofilm, that can shield bacteria from the effects of antibiotic treatment, physiologic stress, and possibly even the immune responses from the host [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. These biofilms can noncovalently interact with host tissue or host proteins in course of infection and as a result, they come across the surfaces of the medical devices and get attache adhered onto their surfaces. To differentiate the harmful bacteria from the normal microbiome, researchers have identified a number of genes that can affect biofilm morphologies, leading subsequently to pathogenicity [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. One of such factors of <em>Staphylococcus</em> sp. is the accessory gene regulator (Agr), known to regulate quorum sensing that eventually control different virulent gene expressions of these organisms [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. P2 and P3 promoters govern the two neighbouring transcripts RNAII and RNAIII, respectively, that make up the <em>agr</em> locus. The second RNA subunit consists of the <em>agrB</em>, <em>agrD</em>, <em>agrC</em>, and <em>agrA</em> genes [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. The autoinducing peptide (AIP), a pro-peptide, is encoded by <em>agrD</em>., while the AIP processor is encoded by the <em>agrB</em> gene. The two-component regulatory system comprised of AgrC and AgrA is encoded by the genes <em>agrC</em> and <em>agrA</em> [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. The AIP pro-peptide is converted to octapeptide by AgrB and is released into the extracellular space upon Agr activation. The membrane-bound histidine kinase AgrC undergoes autophosphorylation and activation when AIP approaches a threshold, causing AgrA, its cognate response regulator, to be phosphorylated [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. Phosphorylated AgrA increases AIP production by activating its own RNAII transcript and promoter P3 to enhance RNAIII expression. Pathogenicity is caused by the two primary intracellular effectors, AgrA and RNAIII, which control the expression of virulence factors [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. Role of a functional Agr in inducing disease in infection models in animals has been extensively studied [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. Agr malfunction has been linked to thrombin-induced platelet microbicidal protein, reduced sensitivity to vancomycin, and prolonged bacteraemia. In regard to biofilm capacity, Agr dysfunctional strains typically have a larger capacity [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. The present study focuses on comparing the structural and functional properties of the Agr system between CoNS (<em>S. epidermidis</em>) and CoPS (<em>S. aureus</em>), as they are the most prominent staphylococcal species on the human skin. <em>S. epidermidis</em> accounts for over 90% of the resident aerobic CoNS [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>], while <em>S. aureus</em> makes up to 80% of the CoPS population [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Since quorum sensing (QS) is a crucial mechanism for microbes, several scientific investigations have been carried out to block QS signaling. Developing methods to inhibit Agrs for supressing QS-mediated virulence features is one of the most promising targets for anti-QS therapies. Moreover, the fact that the toxins produced by AgrA activations promote antibiotic resistance and pathogenecity qualifies Agrs to be an ideal therapeutic target for the management of QS-mediated staphylococcal infections [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. Thorough studies on the Agr protein in these two medically significant species of bacterial pathogen are required to address the virulence as well as fatality caused by <em>Staphylococci</em> sp. since Agr is critical for staphylococcal infection. Especially the structural analyses of AgrA and AgrC of <em>Staphylococcus</em> sp. have been carried out [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. However, to solve the entire issue, wider approaches are needed. It is apprehended that computational studies may provide considerably reasonable, reliable as well as prompt solution and most importantly may thrust upon exclusive genomic and proteomic records that happens to be the foundation of every behaviour for living [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. In the rapidly developing field of artificial intelligence, in silico molecular profiling is significant in terms of gathering information and analyses with the objective to finally make molecular correlations [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. Agrs have recently been spotted as a mean that can open-up prospective therapeutic avenues in combating biofilm-mediated microbial infections. It is mostly due to conserved architecture and functional modalities in <em>S. aureus</em> and <em>S. epidermidis.</em> Moreover, both these species are well-known models for studying quorum sensing signalling mechanism [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. Targeting Agr with natural bioactive compounds has been shown in previous studies on <em>S. aureus</em>, including reports from the current research group, to regulate QS quality and biofilm development. Additionally, the current research team is actively investigating a variety of <em>S. aureus</em> issues, such as molecular epidemiology, biofilm signaling, anti-QS development, anti-biofilm tactics, etc [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>â€“<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. It is believed that theoretical techniques can effectively provide distinctive understanding on the fundamental mechanisms prior to laboratory operations, which would ultimately be useful in more effective structuring of the experimental setup. Because some of these Agrs are thought to be transmembrane, it is particularly challenging to do typical X-ray crystallographic research. Since significant intermolecular interactions are required for crystallization, membrane proteins that have been isolated in detergent-bound form provide challenges. Effective protein interactions are hampered by the repulsions of the purification detergents. NMR spectroscopy has the drawback of only being able to study smaller protein (12KDa). It is challenging to study proteins with molecular weights more than 25Â kDa using NMR spectroscopy alone. Here, bioinformatic approaches can offer a way of finding out a trustworthy, efficient, and quick solution to this issue [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>].</p>
<p id="Par20">In this study we made an attempt to substantiate certain specific laboratory experiments. Performing experiments in a wet laboratory is a cumbersome process, which can be minimized by using preconceived ideas obtained through theoretical investigation. Therefore, the propositions derived from such theoretical studies can be further substantiated with a justifiable amount of wet lab work. This study limited our focus on the available and most relevant clinical isolates that could readily satisfy our primary objectives in relation to desirable features that also includes obtaining the preliminary ideas from selected isolated strains that are known to be pathogenic by virtue of the previously reported results. The present work aims to analyse and compare the physico-chemical and structural properties of four Agrs, <em>viz</em>., AgrA, AgrB, AgrC, and AgrD, of S. <em>epidermidis</em> ATCC 35984, <em>S. aureus</em> MRSA 252, <em>S. aureus</em> MSSA 476, and <em>S. aureus</em> NCTC 8325 in addition to protein-protein docking studies. The present research group is engaged to understand the molecular mechanisms of biofilm formation and quorum sensing as underlying mystery for the infection caused by two bacterial pathogens, <em>S. aureus</em> and <em>S. epidermidis</em>, beside other bacteria. In this context, development of comprehensive theoretical knowledge by exploiting in silico approaches, that are providentially available at present time, are considered to be extremely significant and applicable. In silico approaches involving the Agr still remain extremely important as it is capable in generating information on the structure and its interactive modalities [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Additionally, the docking analyses involving Agr appear most significant as this data-sets are apprehended to be most eloquent and contributory towards drug development approaches targeting management of such life-threatening infectious diseases in human [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>].</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods and materials</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Data collection</h3>
<p id="Par21">Amino acid sequence and taxonomic data of four Agr proteins of selected strains of <em>S. aureus</em> (AgrA SA, AgrB SA, AgrC SA, AgrD SA) and <em>S.epidermidis</em> (AgrA SE, AgrB SE, AgrC SE, AgrD SE) were retrieved from UniProt (<a href="http://www.uniprot.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.uniprot.org</a>) and NCBI (<a href="http://www.ncbi.nlm.nih.gov" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.ncbi.nlm.nih.gov</a>) in FASTA configuration [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. The genomic and amino acid sequences of Agrs of <em>S. epidermidis</em> ATCC 35984 (isolated during a catheter-associated sepsis outbreak between 1979 and 1980) [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>] were analyzed. In case of <em>S. aureus</em>, the following strains were taken into account: MRSA 252 (isolated in 1997 from a 64-year-old female with bacteraemia), MSSA 476 (isolated in 1998 from a 9-year-old boy with primary upper tibial osteomyelitis), and NCTC 8325 (isolated from a sepsis patient in 1960). <em>S. aureus</em> NCTC 8325, despite its origin, is predominantly known and utilized as a laboratory reference strain, having undergone extensive passaging [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. All the strains were clinical isolates [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>, <a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>] and were commonly used as positive control strains in a variety of research contexts, especially in investigations pertaining to biofilm formation, antibiotic resistance, and QS-mediated pathogenicity evaluations [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>â€“<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. All these strains were found to develop resistance to several commonly used antibiotics and were isolated from clinical settings. Using an <em>agr</em>-dependent QS mechanism, each of these strains controls the initial cell attachment and cell autolysis during biofilm development that produced RNAIII, which is also responsible for encoding the Î´-toxin. They were also found to be capable of causing a range of infections, including those related to catheters [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>â€“<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>].</p></section><section id="Sec4"><h3 class="pmc_sec_title">Multiple sequence alignment studies (MSA)</h3>
<p id="Par22">MSA of different Agrs of <em>S. aureus</em> and <em>S. epidermidis</em> were carried out using ClustalW module of MEGA6.06 programme with default parameters [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. The MEGA program, which is optimised for repetitive and merged sequence investigations, is designed for comparative analysis of nucleotide and amino acid sequences with the goal of deducing genetic evolutionary trends in different species across time [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. Jalview can visualise MSA using a single letter amino acid logo [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. Patterns of alignment can be imported using a number of widely used FASTA formats or obtained from public databases. It enables the importation of alignments produced by alternative applications. Additionally, Jalviewâ€™s integrated analytic methods assist in calculating the consensus for every alignment column, which is shown as a histogram beneath the alignment [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>].</p></section><section id="Sec5"><h3 class="pmc_sec_title">Physicochemical properties</h3>
<p id="Par23">Different physicochemical properties like aliphatic index (AI), molar extinction coefficient (EC), grand average of hydropathicity (GRAVY), isoelectric point (PI), instability index (II), molecular weight (MW), positive (+â€‰R) and negative (- R) charge of different Agr proteins were analyzed by submitting their FASTA format amino acid sequences, downloaded from the NCBI database, to the ExPASy-ProtParam server (<a href="http://web.expasy.org/protparam/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://web.expasy.org/protparam/</a>) [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. This service appears with great implications as it can provide certain activities related to proteomics research, such as primary, secondary, and tertiary architecture evaluation [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>].</p></section><section id="Sec6"><h3 class="pmc_sec_title">Compositional heterogeneity based on codon usage</h3>
<p id="Par24">Several genomic characteristics, including GC content and relative synonymous codon usage (RCSU), were examined by CAIcal server (<a href="https://genomes.urv.es/CAIcal/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://genomes.urv.es/CAIcal/</a>) [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. This online tool was used to create A3%, T3%, G3%, C3%, GCs (GC1%, GC2%, GC3%, GC%), and number of codons (Nc) values of Agr in <em>S. aureus</em> and <em>S. epidermidis</em>. The initial and last codons were removed from the computation to limit the impact on codon usage. The RSCU value for each codon was calculated using the method shown below:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a805/12374375/67d8183d3fd1/d33e694.gif" loading="lazy" id="d33e694" alt="graphic file with name d33e694.gif"></td>
<td class="label">1</td>
</tr></table>
<p id="Par25">where, X<sub>ij</sub> is the frequency of the j<sup>th</sup> codon for the i<sup>th</sup> amino acid, and n<sub>i</sub> is the number of codons for the i<sup>th</sup> amino acid (i<sup>th</sup> codon family). The effective number of codons (ENc) was used to calculate the overall frequencies of GC content at the first, second, and third codon places for respective gene sequences to ensure entire synonymous codon usage. The ENc value typically has a strong bias in codon usage and vice versa. An ENc and GC3s plot was generated to emphasise the role of a dominant mutation on codon usage patterns [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. The expected ENc values for each GC3 were determined utilising the equation shown below.</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a805/12374375/c711515a5cc8/d33e720.gif" loading="lazy" id="d33e720" alt="graphic file with name d33e720.gif"></td>
<td class="label">2</td>
</tr></table>
<p id="Par27">where, â€˜sâ€™ denotes GC content at the third codon location [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>].</p></section><section id="Sec7"><h3 class="pmc_sec_title">Codon adaptation analysis</h3>
<p id="Par28">Adaptability of the Agr codons to <em>S. aureus</em> and <em>S. epidermidis</em>, were assessed by codon adaptation index (CAI). It was computed by the CAIcal service (<a href="https://genomes.urv.es/CAIcal/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://genomes.urv.es/CAIcal/</a>) [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. <em>S. aureus</em> and <em>S. epidermidis</em> reference datasets were obtained from the Codon Usage Database (<a href="https://www.kazusa.or.jp/codon/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.kazusa.or.jp/codon/</a>) [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. The original approach put out by Sharp and Li is used to compute CAI [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. The CAI value ranges from 0 to 1, a greater value is indicative of better adaptation likelihood and gene compatibility in the genome [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. Detailed procedure has been described in supplementary section.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Subcellular localization and membrane topology</h3>
<p id="Par29">Subcellular localization of different Agrs was analyzed by CELLO v.2.5 (<a href="http://cello.life.nctu.edu.tw/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://cello.life.nctu.edu.tw/</a>) and PSORTb v 3.0.3 (<a href="https://www.psort.org/psortb/results.pl" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.psort.org/psortb/results.pl</a>) [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>, <a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>]. For both servers, the organism type (Gram-negative or Gram-positive) was initially selected, followed by the submission of query sequences in FASTA format. PSORTb predicts subcellular localization using a 5-fold cross-validation approach to ensure robust and accurate results. Prediction of transmembrane parts and topology of four proteins from the two different species were performed by using different online servers like, TOPCONS (<a href="https://topcons.cbr.su.se/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://topcons.cbr.su.se/</a>) [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>], CCTOP (<a href="https://cctop.ttk.hu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cctop.ttk.hu/</a>) [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>], Phobius (<a href="https://phobius.sbc.su.se/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://phobius.sbc.su.se/</a>) [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>], Octopus (<a href="https://github.com/cloudflare/octopus" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/cloudflare/octopus</a>) [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. Besides, 3D structure of transmembrane part of different Agrs were visualized by using Deep TMHMM (<a href="https://dtu.biolib.com/DeepTMHMM" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://dtu.biolib.com/DeepTMHMM</a>) [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>]. Presence of signal peptide examined through Signal P v5.0 web server and Deep TMHMM [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>].</p></section><section id="Sec9"><h3 class="pmc_sec_title">Crystallization probability</h3>
<p id="Par30">The forecast was produced by adding each crystallisation probabilities to get a single crystallisation score. Based on this score, the protein is given to one of five crystallisation classes: ideal, suboptimal, median, challenging, and extremely challenging. The score of different Agrs from both species was generated by using protein sequences in XtalPred online server (<a href="https://xtalpred.godziklab.org/XtalPred-cgi/xtal.pl" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://xtalpred.godziklab.org/XtalPred-cgi/xtal.pl</a>) [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. The analysis involved entering the query sequences of amino acid residues in FASTA format and choosing the option â€œFind close bacterial homologs more likely to crystallizeâ€. XtalPred was evaluated by reexamining the selection of targets from 271 Pfam families that the Joint Center for Structural Genomics had targeted in PSI-2. It was calculated that 30% fewer targets could have been added to the pipeline for protein synthesis and crystallization without reducing the number of families [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>].</p></section><section id="Sec10"><h3 class="pmc_sec_title">Secondary structure analysis</h3>
<p id="Par31">Secondary structures (Î±-helix, Î²-turn, ramdom coil, extended strands) of Agrs were evaluated through GOR IV (<a href="https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page=/NPSA/npsa_gor4.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page=/NPSA/npsa_gor4.html</a>) [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>] and ExPASy SOPMA server (<a href="https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page=/NPSA/npsa_sopma.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://npsa-prabi.ibcp.fr/cgi-bin/npsa_automat.pl?page=/NPSA/npsa_sopma.html</a>) [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. Amino acid sequences were submitted in FASTA format with an output width set to 70 for the analysis. The strategy depends on data science and the idea that all data collected from individual and couple of amino acid residues may be used to estimate the details of the constitutes in a protein. Because of their straightforward physical presumptions The GOR IV and ExPASy SOPMA server have a theoretical edge among various systems like PSIPRED, PHD, and others that rely on algorithms for machine learning [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. Percentage of amino acid participation in secondary structure was also analyzed by ExPASy-ProtParam server (<a href="http://web.expasy.org/protparam/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://web.expasy.org/protparam/</a>) [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Detailed procedure has been described in supplementary section.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Tertiary structure prediction and validation</h3>
<p id="Par32">Tertiary structure of Agrs were derived by using TrRosetta v1.14 (transform-restrained Rosetta; <a href="https://yanglab.qd.sdu.edu.cn/trRosetta/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://yanglab.qd.sdu.edu.cn/trRosetta/</a>) [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>] and AlphaFold2 (<a href="https://alphafold.ebi.ac.uk/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://alphafold.ebi.ac.uk/</a>) [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. The sequences were submitted in FASTA format, excluding the selection of â€˜Do not use templatesâ€™ and â€˜trRosettaX-Singleâ€™ options in TrRosetta. The AlphaFold2 protein structure database was utilized in this investigation with its default parameters [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. TrRosetta v1.14, developed after the 13th Critical Assessment of Protein Structure Prediction (CASP13), uses Rosetta and deep learning to quickly and accurately predict protein structures using default settings. The deep neural network-predicted inter-residue distance and orientation ranges are among the constraints [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. AlphaFold, DeepMindâ€™s deep learning system, has shared 3D representations with the research community. It is a digital architecture influenced by neuronal circuitry that detects patterns in data [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. Results from above mentioned server validated through PROCHECK (<a href="https://saves.mbi.ucla.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://saves.mbi.ucla.edu/</a>) [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>] and ERRAT (<a href="https://www.doe-mbi.ucla.edu/errat/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.doe-mbi.ucla.edu/errat/</a>) [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. It aids in determining the stereochemical properties of a structure of proteins by creating a series of PostScript plots that analyse its overall and residue-by-residue geometry through generating Ramachandran plot [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. The improved models were checked using assessment tool, ProSA (<a href="https://prosa.services.came.sbg.ac.at/prosa.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://prosa.services.came.sbg.ac.at/prosa.php</a>) [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. The Protein Tools server was utilised to identify salt bridges in the predicted structure (<a href="https://proteintools.uni-bayreuth.de/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://proteintools.uni-bayreuth.de/</a>) [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>].</p></section><section id="Sec12"><h3 class="pmc_sec_title">Flexibility modelling of tertiary structure</h3>
<p id="Par33">Agrs structural flexibility was checked by CABS-flex 2.0 (<a href="https://biocomp.chem.uw.edu.pl/CABSflex2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://biocomp.chem.uw.edu.pl/CABSflex2</a>) [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. CABS-flex simulates the dynamics of proteins through a CABS coarse-grained structural framework. The CABS system utilises Monte Carlo dynamics and the asymmetric Metropolis method, which meets the conditions of microscopic reversibility and Boltzmann distribution among produced ensembles. CABS-flex is a complement to existing effective methods for creating structural residue volatility identities, which include sequence-based forecasters that identify proteinâ€™s chaotic areas or other coarse-grained techniques. It included over 50 cycles and 50 trajectory segments in 10 ns each, as well as several additional length constraints, such as a universal mass of 1.0. The flexibility of the structures was presented through root-mean-square fluctuations (RMSF). In the standard setting, the CABS-flex employs a set of range constraints and simulation settings provided in the job [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>].</p></section><section id="Sec13"><h3 class="pmc_sec_title">Domain and motif evaluation</h3>
<p id="Par34">Different functional domains of the studied proteins were evaluated through InterProScan online tool (<a href="https://www.ebi.ac.uk/interpro/search/sequence/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/interpro/search/sequence/</a>) [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. This server helps to examine the amino acid sequences for matches to the InterPro protein signature libraries. In this chosen method by scanning every record of files, entry data, and program outcomes, the Perl cataloguing framework is used in this server to facilitate simple searching and storage of the basic IDs, project accomplishments, and their current condition. Individuals may view the complete or any particular entry data from the webpage [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. Amino acid residues were also evaluated through conserved domain search tool (<a href="https://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/Structure/cdd/cdd.shtml</a>) in expectation of Pfam data server. Motif of the studied proteins were greeted by MEME server (Multiple Em for Motif Elicitation; <a href="https://meme-suite.org/meme/tools/meme" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://meme-suite.org/meme/tools/meme</a>) [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>].</p></section><section id="Sec14"><h3 class="pmc_sec_title">Protein-protein docking</h3>
<p id="Par35">Docking investigations between AgrA-AgrC and AgrB-AgrD of <em>S. aureus</em> and <em>S. epidermidis</em> were carried out by utilising a protein-protein docking server, HDOCK (<a href="http://hdock.phys.hust.edu.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://hdock.phys.hust.edu.cn/</a>) [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>] and LZerD (<a href="https://kiharalab.org/proteindocking/lzerd.php" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://kiharalab.org/proteindocking/lzerd.php</a>) [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>, <a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. Afterwords, interfacing residues were visualized by PyMOL application [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>], PDBsum server (<a href="https://www.ebi.ac.uk/thornton-srv/databases/pdbsum/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/thornton-srv/databases/pdbsum/</a>) [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>], and LigPlot [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. An online service called PDBsum offers structural details about the data contained in the Protein Data Bank (PDB). The studies include structural quality of PROCHECK evaluations and graphics-based protein-protein interactions [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]. Binding affinity Î”G (kcal/mol) and dissociation constant (K<sub>d</sub>) at 37Â Â°C were studied through the PRODIGY website (<a href="https://bianca.science.uu.nl/prodigy/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bianca.science.uu.nl/prodigy/</a>) [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>]. Detailed procedure has been described in supplementary section.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Molecular dynamics (MD) simulation studies</h3>
<p id="Par36">MD simulation was performed on server for better understanding on the behaviour of biomolecules and to determine the steadiness of protein-protein docking. To investigate the interaction reliability and adaptability of the protein-protein complex, normal mode analysis was performed using the iMODS (<a href="https://imods.iqfr.csic.es/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://imods.iqfr.csic.es/</a>) tool, that generates trajectories using Monte Carlo sampling for 1Â atm pressure at 300Â K temperature [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>] and an elastic network model in internal coordinates. PDB structures were submitted to the iMODS online tool, and outcomes were shown with all by default settings. A 50 ns MD simulation was performed on AgrA-AgrC and AgrB-AgrD docked structures of chosen <em>S. aureus</em> and <em>S. epidermidis</em> strains [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>, <a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. Detailed procedure has been described in supplementary section.</p></section></section><section id="Sec16"><h2 class="pmc_sec_title">Results and discussion</h2>
<section id="Sec17"><h3 class="pmc_sec_title">Multiple sequence alignment (MSA) analysis</h3>
<p id="Par37">MSA study helps in classifying the proteins into three regions, <em>viz</em>., fully-conserved, semi-conserved, and non-conserved regions. These locations were investigated in terms of their ancestral evolution including potential protein mutations [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. MSA studies on distinct Agrs were performed using a single letter symbol for each amino acid residue. Figures S1-3 (supplementary section) demonstrate the MSA of non-conserved and conserved amino acid residues in Agrs of <em>S. aureus</em> and <em>S. epidermidis</em>. AgrA of both the <em>Staphylococcus</em> sp. contain semi-conserved (1<sup>st</sup> to 72<sup>nd</sup>, 114<sup>th</sup> to 141<sup>st</sup>, and 190<sup>th</sup> to 230<sup>th</sup> amino acid residues) and fully-conserved regions (73<sup>rd</sup> to 113<sup>th</sup> a<sup>nd</sup> 142<sup>nd</sup> to 189<sup>th</sup> amino acid residues) as shown in Figure S1a-b. Only 19<sup>th</sup> to 40<sup>th</sup> amino acid residues of AgrB were semi-conserved and other parts were non-conserved (Figure S1c-d). Amino acid sequences of AgrC and AgrD from both <em>Staphylococcus</em> sp. were non-conserved (Figure S2a-d and Figure S3). Studied sequences are considered to be variable throughout all Agrs except AgrA. Similar findings have also been reported earlier [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>]. <em>N</em>-terminal receiver domains of AgrA of both the species share similarity with the consensus sequence [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. Two amino acids, His77 and Cys84, were identified to be essential for AgrB proteolytic action [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>]. AgrC of <em>S. epidermidis</em> and <em>S. aureus</em> share a similarity of 50%; the homology discovered for the <em>N</em>-terminal and <em>C</em>-terminal area is 28 and 67%, respectively [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. <em>N</em>-terminal part of AgrB is non-conserved but in case of AgrA it is conserved. AgrA has the most conserved amino acid sequences among all Agrs in both <em>Staphylococcus</em> sp.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Physicochemical properties</h3>
<p id="Par38">Comparative studies on different physiochemical characteristics and amino acids throughout the four Agrs from the selected strains of two distinct species, <em>viz</em>.,<em> S. aureus</em> and <em>S. epidermidis</em> were carried out. TableÂ <a href="#Tab1" class="usa-link">1</a> summarizes the results of the investigation of physiochemical characteristics with unique UniProt ID. Number of amino acid residues is same for AgrA and AgrD, <em>i.e</em>., 238 and 46, respectively, in both the species but number of amino acid residues are higher for AgrC (430 amino acid residues) in <em>S. epidermidis</em> and AgrB (194 amino acid residues) in <em>S. aureus</em>. Irrespective of similar length of amino acid residues, all Agrs have different molecular weights. AgrD shows lowest molecular weight; 5130.92 Da for <em>S. epidermidis</em> and 5243.09 Da for<em>S. aureus</em>. In contrast, AgrC shows highest molecular weight of 50252.59 Da for <em>S. epidermidis</em> and 49677.64 Da for <em>S. aureus</em>. Theoretical isoelectric points (pI) of AgrB from strains of both the species, 9.71 in <em>S. aureus</em> and 9.94 in <em>S. epidermidis</em> were greater than 7.00 but other Agrs were found within the range of 4.25 to 6.93.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Physicochemical properties of accessory gene regulators, AgrA, AgrB, AgrC, and AgrD of <em>S. aureus</em> and <em>S. epidermidis</em></p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Agr</th>
<th align="left" colspan="1" rowspan="1">UniProt ID</th>
<th align="left" colspan="1" rowspan="1">No. of amino acids</th>
<th align="left" colspan="1" rowspan="1">MW/Da</th>
<th align="left" colspan="1" rowspan="1">pI</th>
<th align="left" colspan="1" rowspan="1">(-) <em>R</em>
</th>
<th align="left" colspan="1" rowspan="1">(+) <em>R</em>
</th>
<th align="left" colspan="1" rowspan="1">Îµ/M<sup>âˆ’1</sup>cm<sup>âˆ’1</sup>
</th>
<th align="left" colspan="1" rowspan="1">II</th>
<th align="left" colspan="1" rowspan="1">AI</th>
<th align="left" colspan="1" rowspan="1">GRAVY</th>
</tr></thead>
<tbody>
<tr><td align="left" colspan="11" rowspan="1">Staphylococcus aureus</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAgrA</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/Q6GF33" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q6GF33</a></td>
<td align="left" colspan="1" rowspan="1">238</td>
<td align="left" colspan="1" rowspan="1">27905.90</td>
<td align="left" colspan="1" rowspan="1">5.78</td>
<td align="left" colspan="1" rowspan="1">37</td>
<td align="left" colspan="1" rowspan="1">31</td>
<td align="left" colspan="1" rowspan="1">15,150</td>
<td align="left" colspan="1" rowspan="1">36.25</td>
<td align="left" colspan="1" rowspan="1">91.30</td>
<td align="left" colspan="1" rowspan="1">âˆ’0.379</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAgrB</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/Q6G7S1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q6G7S1</a></td>
<td align="left" colspan="1" rowspan="1">187</td>
<td align="left" colspan="1" rowspan="1">21801.24</td>
<td align="left" colspan="1" rowspan="1">9.71</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="left" colspan="1" rowspan="1">23,380</td>
<td align="left" colspan="1" rowspan="1">45.06</td>
<td align="left" colspan="1" rowspan="1">132.46</td>
<td align="left" colspan="1" rowspan="1">0.621</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAgrC</td>
<td align="left" colspan="1" rowspan="1">B5U0V1</td>
<td align="left" colspan="1" rowspan="1">430</td>
<td align="left" colspan="1" rowspan="1">49677.64</td>
<td align="left" colspan="1" rowspan="1">5.19</td>
<td align="left" colspan="1" rowspan="1">45</td>
<td align="left" colspan="1" rowspan="1">38</td>
<td align="left" colspan="1" rowspan="1">37,040</td>
<td align="left" colspan="1" rowspan="1">39.76</td>
<td align="left" colspan="1" rowspan="1">124.67</td>
<td align="left" colspan="1" rowspan="1">0.501</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAgrD</td>
<td align="left" colspan="1" rowspan="1">B5U0V0</td>
<td align="left" colspan="1" rowspan="1">46</td>
<td align="left" colspan="1" rowspan="1">5243.09</td>
<td align="left" colspan="1" rowspan="1">4.25</td>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">1490</td>
<td align="left" colspan="1" rowspan="1">28.94</td>
<td align="left" colspan="1" rowspan="1">118.70</td>
<td align="left" colspan="1" rowspan="1">0.337</td>
</tr>
<tr><td align="left" colspan="11" rowspan="1">
<strong><em>Staphylococcus epidermidis</em></strong>
</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAgrA</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/Q5HMY2" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q5HMY2</a></td>
<td align="left" colspan="1" rowspan="1">238</td>
<td align="left" colspan="1" rowspan="1">27936.97</td>
<td align="left" colspan="1" rowspan="1">5.54</td>
<td align="left" colspan="1" rowspan="1">39</td>
<td align="left" colspan="1" rowspan="1">31</td>
<td align="left" colspan="1" rowspan="1">13,660</td>
<td align="left" colspan="1" rowspan="1">31.20</td>
<td align="left" colspan="1" rowspan="1">91.72</td>
<td align="left" colspan="1" rowspan="1">âˆ’0.354</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAgrB</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/P0C0N4" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P0C0N4</a></td>
<td align="left" colspan="1" rowspan="1">194</td>
<td align="left" colspan="1" rowspan="1">22573.40</td>
<td align="left" colspan="1" rowspan="1">9.94</td>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">12,045</td>
<td align="left" colspan="1" rowspan="1">46.77</td>
<td align="left" colspan="1" rowspan="1">141.70</td>
<td align="left" colspan="1" rowspan="1">0.626</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAgrC</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/Q5HMY3" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q5HMY3</a></td>
<td align="left" colspan="1" rowspan="1">429</td>
<td align="left" colspan="1" rowspan="1">50252.59</td>
<td align="left" colspan="1" rowspan="1">6.93</td>
<td align="left" colspan="1" rowspan="1">47</td>
<td align="left" colspan="1" rowspan="1">47</td>
<td align="left" colspan="1" rowspan="1">42,290</td>
<td align="left" colspan="1" rowspan="1">35.68</td>
<td align="left" colspan="1" rowspan="1">127.39</td>
<td align="left" colspan="1" rowspan="1">0.354</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAgrD</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/protein/Q8VSX6" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Q8VSX6</a></td>
<td align="left" colspan="1" rowspan="1">46</td>
<td align="left" colspan="1" rowspan="1">5130.92</td>
<td align="left" colspan="1" rowspan="1">4.36</td>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">2980</td>
<td align="left" colspan="1" rowspan="1">24.02</td>
<td align="left" colspan="1" rowspan="1">101.74</td>
<td align="left" colspan="1" rowspan="1">âˆ’0.046</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par40">Theoretical pI values reveal that all Agrs except AgrB are acidic in nature. Numbers of positive charge residues were found higher in AgrB and AgrC, whereas, negative charge residues are higher on AgrA and AgrD for both the species. For values of molar extinction coefficient (Îµ) for <em>S. aureus</em>, a range from 1490 (AgrD) to 37,040 M<sup>âˆ’1</sup>cm<sup>âˆ’1</sup> (AgrC) and for <em>S. epidermidis</em>, from 2980 (AgrD) to 42,290 M<sup>âˆ’1</sup>cm<sup>âˆ’1</sup>(AgrC) could be observed. Î• value of a particular protein is determined by the number of tryptophan, cysteine, and tyrosine residues per protein. These residues have a significant impact with the optical density, that can be observed in the wavelength of 276â€“282Â nm. The instability index (II) values were utilised in accessing the stability of protein in laboratory settings. Proteins having II value less than 40 are stable [<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>]. Except AgrB in <em>S. aureus</em> (45.06) and <em>S. epidermidis</em> (46.77) all Agrs were found as having instability index lesser than 40. Obtained results suggest that except AgrB other Agrs are stable. The aliphatic index (AI) is an additional parameter for evaluating the physicochemical properties of proteins. The AI refers to the fraction of a volume of protein covered by aliphatic amino acid residues [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>]. Proteins with higher values are considered to be thermostable, and vice versa. AI values of <em>S. aureus</em> and <em>S. epidermidis</em> varied in between 91.30 (AgrD) to 132.46 (AgrB), and 91.72 (AgrD) to 141.70 (AgrB), respectively. It may be concluded that all the Agrs tested are thermostable. Previous studies also have suggested that Agrs of <em>S. aureus</em> are thermostable [<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. Studies also reveal that Agrs are high in hydrophobic constituents [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>]. Furthermore, The GRAVY score is calculated by adding the hydropathy values of all amino acids in the protein and dividing by the number of residues. It ranges from âˆ’â€‰2 to +â€‰2, with positive values representing hydrophobic and negative values indicating hydrophilic proteins [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>]. GRAVY value was found negative in AgrA for the selected strains from both species and in AgrD for <em>S. epidermidis</em> (âˆ’0.046), whereas, other Agrs analyzed show positive GRAVY score (0.337 to 0.626). Consequently, it may be inferred that all studied Agrs in present investigation except AgrA in both <em>Staphylococcus</em> sp. strains and AgrD in <em>S. epidermidis</em> are hydrophobic in nature.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Compositional heterogeneity based on codon usage</h3>
<p id="Par41">GC makeup in codons play crucial role in determining the codon usage bias. Moreover, reports reveal that the 3rd position GC, <em>i.e</em>., GC3, has an immediate influence on codon usage patterns [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>]. Occurrence of GC% in the third position was highest and GC% in the first position was lowest among all Agrs of selected both <em>Staphylococcus</em> sp strains. From the analyses it appeared that usage of GC3% for AgrA, AgrB, AgrC, and AgrD are higher in <em>S. epidermidis</em> (43.1, 41.62, 34.64, and 42.55%, respectively) than <em>S. aureus</em> (41.84, 33.17, 33.18, and 34.04%, respectively). Usage of T3% was high in all Agrs except that in AgrD of <em>S. epidermidis</em>. Overall GC% is higher in AgrA (30.96%) and AgrC (27.46%) of <em>S. aureus</em> strains but it was found higher in AgrB (28.09%) and AgrD (31.91%) in <em>S. epidermidis</em> (TableÂ <a href="#Tab2" class="usa-link">2</a>). Numerous standpoints of GC percentages indicate the impact of selection pressure on codon usage pattern.</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Different positional nucleotides (GC1%, GC2%, GC3%, GC%, A3%, T3%, G3%, C3%, and CAI) values of different accessory gene regulators (Agrs) of <em>S. aureus</em> (SA) and <em>S. epidermidis</em> (SE)</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Parameters</th>
<th align="left" colspan="1" rowspan="1">AgrA SA</th>
<th align="left" colspan="1" rowspan="1">AgrB SA</th>
<th align="left" colspan="1" rowspan="1">AgrC SA</th>
<th align="left" colspan="1" rowspan="1">AgrD SA</th>
<th align="left" colspan="1" rowspan="1">AgrA SE</th>
<th align="left" colspan="1" rowspan="1">AgrB SE</th>
<th align="left" colspan="1" rowspan="1">AgrC SE</th>
<th align="left" colspan="1" rowspan="1">AgrD SE</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GC1%</td>
<td align="left" colspan="1" rowspan="1">23.85</td>
<td align="left" colspan="1" rowspan="1">24.04</td>
<td align="left" colspan="1" rowspan="1">23.67</td>
<td align="left" colspan="1" rowspan="1">21.28</td>
<td align="left" colspan="1" rowspan="1">21.76</td>
<td align="left" colspan="1" rowspan="1">17.26</td>
<td align="left" colspan="1" rowspan="1">21.71</td>
<td align="left" colspan="1" rowspan="1">25.53</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GC2%</td>
<td align="left" colspan="1" rowspan="1">27.2</td>
<td align="left" colspan="1" rowspan="1">25.96</td>
<td align="left" colspan="1" rowspan="1">25.52</td>
<td align="left" colspan="1" rowspan="1">23.4</td>
<td align="left" colspan="1" rowspan="1">26.36</td>
<td align="left" colspan="1" rowspan="1">25.38</td>
<td align="left" colspan="1" rowspan="1">25.4</td>
<td align="left" colspan="1" rowspan="1">27.66</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GC3%</td>
<td align="left" colspan="1" rowspan="1">41.84</td>
<td align="left" colspan="1" rowspan="1">33.17</td>
<td align="left" colspan="1" rowspan="1">33.18</td>
<td align="left" colspan="1" rowspan="1">34.04</td>
<td align="left" colspan="1" rowspan="1">43.1</td>
<td align="left" colspan="1" rowspan="1">41.62</td>
<td align="left" colspan="1" rowspan="1">34.64</td>
<td align="left" colspan="1" rowspan="1">42.55</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GC%</td>
<td align="left" colspan="1" rowspan="1">30.96</td>
<td align="left" colspan="1" rowspan="1">27.72</td>
<td align="left" colspan="1" rowspan="1">27.46</td>
<td align="left" colspan="1" rowspan="1">26.24</td>
<td align="left" colspan="1" rowspan="1">30.4</td>
<td align="left" colspan="1" rowspan="1">28.09</td>
<td align="left" colspan="1" rowspan="1">27.25</td>
<td align="left" colspan="1" rowspan="1">31.91</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">A3%</td>
<td align="left" colspan="1" rowspan="1">32.64</td>
<td align="left" colspan="1" rowspan="1">31.73</td>
<td align="left" colspan="1" rowspan="1">32.25</td>
<td align="left" colspan="1" rowspan="1">38.3</td>
<td align="left" colspan="1" rowspan="1">34.73</td>
<td align="left" colspan="1" rowspan="1">40.1</td>
<td align="left" colspan="1" rowspan="1">31.41</td>
<td align="left" colspan="1" rowspan="1">40.43</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">T3%</td>
<td align="left" colspan="1" rowspan="1">43.51</td>
<td align="left" colspan="1" rowspan="1">44.23</td>
<td align="left" colspan="1" rowspan="1">44.08</td>
<td align="left" colspan="1" rowspan="1">40.43</td>
<td align="left" colspan="1" rowspan="1">43.51</td>
<td align="left" colspan="1" rowspan="1">42.64</td>
<td align="left" colspan="1" rowspan="1">46.88</td>
<td align="left" colspan="1" rowspan="1">34.04</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">G3%</td>
<td align="left" colspan="1" rowspan="1">9.62</td>
<td align="left" colspan="1" rowspan="1">12.5</td>
<td align="left" colspan="1" rowspan="1">10.21</td>
<td align="left" colspan="1" rowspan="1">6.38</td>
<td align="left" colspan="1" rowspan="1">11.72</td>
<td align="left" colspan="1" rowspan="1">10.15</td>
<td align="left" colspan="1" rowspan="1">11.09</td>
<td align="left" colspan="1" rowspan="1">12.77</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">C3%</td>
<td align="left" colspan="1" rowspan="1">14.23</td>
<td align="left" colspan="1" rowspan="1">11.54</td>
<td align="left" colspan="1" rowspan="1">13.46</td>
<td align="left" colspan="1" rowspan="1">14.89</td>
<td align="left" colspan="1" rowspan="1">10.04</td>
<td align="left" colspan="1" rowspan="1">7.11</td>
<td align="left" colspan="1" rowspan="1">10.2</td>
<td align="left" colspan="1" rowspan="1">12.77</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CAI</td>
<td align="left" colspan="1" rowspan="1">0.715</td>
<td align="left" colspan="1" rowspan="1">0.650</td>
<td align="left" colspan="1" rowspan="1">0.687</td>
<td align="left" colspan="1" rowspan="1">0.732</td>
<td align="left" colspan="1" rowspan="1">0.746</td>
<td align="left" colspan="1" rowspan="1">0.750</td>
<td align="left" colspan="1" rowspan="1">0.735</td>
<td align="left" colspan="1" rowspan="1">0.769</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec20"><h3 class="pmc_sec_title">Effective number of codon (ENc) plot analyses</h3>
<p id="Par42">The correlation between ENc changes and differences in GC contents can be identified by examining the causal connection between ENc and GC3. ENc chart is an effective approach for evaluating codon use trends, and it was utilised to access the effect of GC3s on codon usage bias [<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>]. ENc was compared against GC3s to determine selection pressure of Agrs (Fig.Â <a href="#Fig1" class="usa-link">1</a>). Traits which appear at or above the predicted curve suggest selection bias, whereas genes that are present below the expected curve but very near to the curve indicate minimal mutational pressure. The ENc values ranging from 44 to 50 were found to be under the predicted curve. Results suggest that minimal mutational pressure is probably acting as the key factor in establishing the effect of genetic incarceration on codon usage bias of Agrs.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374375_12866_2025_4257_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a805/12374375/40d1bbd256da/12866_2025_4257_Fig1_HTML.jpg" loading="lazy" id="MO1" height="530" width="744" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Modulation of the effective number of codons/number of codons (ENc/Nc) with GC content at the third codon position (GC3s) in AgrA, AgrB, AgrC, and AgrD of <em>S. aureus</em> and <em>S. epidermidis</em>, where the solid line reflects the predicted curve</p></figcaption></figure></section><section id="Sec21"><h3 class="pmc_sec_title">Relative synonymous codon usage (RSCU) analyses</h3>
<p id="Par44">RSCU is a computationally derived parameter that determines the incorporation of various codons for a particular amino acid in a protein. It is useful in understanding the codon usage pattern of distinct proteins across various organisms [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. Table S1 (supplementary section) summarizes the values of RSCU for all Agrs. Results showed that all Agrs used more AU ending codons, except for His in AgrD, Ala in AgrB of the <em>S. aureus</em> isolates, and Glu in AgrB of <em>S. epidermidis</em>. Codon inclination was further assessed (Table S1; supplementary section) based on RSCU scores. Synonymous codon use was greater for Leu, Ser, and Thr, average for Ile, Val, Pro, and Ala, and lower for the remaining amino acid residues. Majority of the amino acid residues in all the Agrs preferred AU-ending codons A3 over U3, and vice versa. In accordance with the analysis on chemical composition, characteristics and codon use patterns throughout <em>agr</em> genes, compositional parameters are affected with low mutation pressure. Codon bias variants include biased codon pairs and codon relationship. It is commonly known that codon bias increases translation efficiency by modifying the processâ€™s elongation rate, even though translation initiation is the most crucial step in protein synthesis. Moreover, a variety of biological processes, such as uneven protein production and folding, are impacted by codon bias [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>]. Translational preference is a key element that shapes the variance in codon usage among the genes in <em>Streptococci</em>, even if compositional restrictions are the primary factor controlling this usage. Additionally, it has been shown that U-ending codons are favored in quartets of highly expressed genes, but C-ending codons are always chosen in duets [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. Additionally, there was a bias towards A or U/T in the frequencies of four nucleotides on the third position of codons. An energy-saving tactic by AU-rich codons has been suggested in pathogenic organisms. In a similar vein, the preference for A/U ending codons in <em>Chlamydia trachomatisâ€™</em> codon use bias implies that knowledge of these patterns may help discover potential therapeutic targets and comprehend the development of the infection [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. <em>S. aureus</em> has a regulatory system called the GeaRS two-component system (TCS) that allows it to react to changes in its surroundings. GeaRS is AT-rich and comprises a response regulator and a histidine kinase, just like the AgrAC system. The synthesis of virulence factors is one of the many cellular functions that this system controls [<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>].</p></section><section id="Sec22"><h3 class="pmc_sec_title">Codon adaptation index (CAI) analyses</h3>
<p id="Par45">The expected translation efficiency of four <em>agr</em> genes from <em>S. aureus</em> and <em>S. epidermidis</em> CAI was forecast and evaluated. In the present investigation, the CAI values varied from 0.650 (AgrB of <em>S. aureus</em>) to 0.769 (AgrD of <em>S. epidermidis</em>). Overall CAI values of the four Agrs of selected strains of <em>S. aureus</em> and <em>S. epidermidis</em> were found as 0.696 and 0.750, respectively, indicating considerably greater expression of genes with relatively substantial codon adaptation over an entire sequence (TableÂ <a href="#Tab2" class="usa-link">2</a>). CAI was put forth as a quantitative method of estimating a geneâ€™s degree of expression by analyzing its codon sequence [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. The CAI, the codon bias index, an entropy score pertaining to synonymous codon usage, a TATA-box score, and a pyrimidine bias index were among the various protein features that were used to determine the expression levels. A regression study revealed that the entropy score and the CAI, two metrics pertaining to synonymous codon usage, were the two important model components [<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a>]. Variations in codon usage or codon bias affect gene expression and cellular processes via influencing protein synthesis and folding. Although translation initiation is vital, codon bias adjusts elongation rates, modifies ribosome velocity, and impacts mRNA stability, all of which have an impact on protein folding and abundance. Protein synthesis as well as the structure and functionality of the final proteins are impacted by this bias, which includes preferences for particular codon pairings [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>]. Higher values in <em>S. epidermidis</em> compared to <em>S. aureus</em> suggest that the organismâ€™s genes were moderately expressed and the codons were adaptable. All Agrs exhibit CAI values greater than 0.6, suggesting a high level of adaptation. All Agrs have equivalent expression trend, indicating the simultaneous expression in both <em>Staphylococcus</em> sp. The amino acid identity values and the related protein expression that <em>agr</em> encodes are very close [<a href="#CR87" class="usa-link" aria-describedby="CR87">87</a>]. Agrs play significant functions in controlling the expression of the RNAIII gene are suggested by their sequence conservation, and all staphylococcal species most likely share similar regulation mechanisms. Whereas similarity in the expression profiles of <em>agr</em> did not correlate with the expression of staphylococcal exotoxins [<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a>, <a href="#CR89" class="usa-link" aria-describedby="CR89">89</a>].</p></section><section id="Sec23"><h3 class="pmc_sec_title">Subcellular localization and membrane topology</h3>
<p id="Par47">Two servers, <em>viz</em>., CELLO and PSORTb produced similar results for cellular localization of all the Agrs (TableÂ <a href="#Tab3" class="usa-link">3</a>). AgrA and AgrD of both the species were found to be located in cytosol, but AgrB and AgrC were predicted to be transmembrane proteins. This findings appeared consistent with earlier findings [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>, <a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>]. Cytosolic and transmembrane regions of all Agrs are shown in Fig.Â <a href="#Fig2" class="usa-link">2</a>. Research indicates that among the Agrs, AgrA and AgrD are mostly found in the cytosol, whereas AgrB and AgrC are found in the membrane [<a href="#CR90" class="usa-link" aria-describedby="CR90">90</a>, <a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>]. Length and number of transmembrane regions of AgrB and AgrC are presented in Table S2 (supplementary section). Most interestingly, TOPCON and Deep TMHMM show six, CCTOP and Phobius show five, and Octopus show four transmenbrane regions of AgrB in all <em>S. aureus</em> and <em>S. epidermidis</em> (33 to 184 amino acid residues) strains under present study. AgrB is a transmembrane protein that aids in the digestion of AgrD. AgrB has six transmembrane helices, four hydrophobic and two hydrophilic, with large number of positively charged residues [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>]. Transmembrane-nontransmembrane protein interactions have a direct impact on drug absorption and typically hold the role of hub nodes in related pathways because transmembrane proteins are important drug targets. It is true that amino acids outside of a proteinâ€™s membrane-spanning domain can serve as therapeutic targets. These areas serve a variety of purposes and can be targeted by medications to alter cellular activity [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>, <a href="#CR94" class="usa-link" aria-describedby="CR94">94</a>]. All servers resulted in the same number of transmembrane regions, i.e., seven for AgrC in strains from both species with length between 7 and 205 amino acid residues. SignalP server suggested that there is no signal peptide for any of the Agrs in both species.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Sub cellular localization of different accessory gene regulators (Agrs) of <em>S. aureus</em> (SA) and <em>S. epidermidis</em> (SE) predicted through several servers. Here, C- cytosolic and M- transmenbrane</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Tools</th>
<th align="left" colspan="1" rowspan="1">AgrA<br>SA</th>
<th align="left" colspan="1" rowspan="1">AgrA<br>SE</th>
<th align="left" colspan="1" rowspan="1">AgrB SA</th>
<th align="left" colspan="1" rowspan="1">AgrB<br>SE</th>
<th align="left" colspan="1" rowspan="1">AgrC SA</th>
<th align="left" colspan="1" rowspan="1">AgrC<br>SE</th>
<th align="left" colspan="1" rowspan="1">AgrD<br>SA</th>
<th align="left" colspan="1" rowspan="1">AgrD<br>SE</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>CELLO</strong>
</td>
<td align="left" colspan="1" rowspan="1">C</td>
<td align="left" colspan="1" rowspan="1">C</td>
<td align="left" colspan="1" rowspan="1">M</td>
<td align="left" colspan="1" rowspan="1">M</td>
<td align="left" colspan="1" rowspan="1">M</td>
<td align="left" colspan="1" rowspan="1">M</td>
<td align="left" colspan="1" rowspan="1">C</td>
<td align="left" colspan="1" rowspan="1">C</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>PSORTb</strong>
</td>
<td align="left" colspan="1" rowspan="1">C</td>
<td align="left" colspan="1" rowspan="1">C</td>
<td align="left" colspan="1" rowspan="1">M</td>
<td align="left" colspan="1" rowspan="1">M</td>
<td align="left" colspan="1" rowspan="1">M</td>
<td align="left" colspan="1" rowspan="1">M</td>
<td align="left" colspan="1" rowspan="1">C</td>
<td align="left" colspan="1" rowspan="1">C</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374375_12866_2025_4257_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a805/12374375/6ebedba4eed7/12866_2025_4257_Fig2_HTML.jpg" loading="lazy" id="d33e2029" height="484" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Structure of transmembrane helices of accessory gene regulators (Agrs) from <em>S. aureus</em> (SA) and <em>S. epidermidis</em> (SE) strains: (a) MembraneFold online tool was used to visualise transmembrane helices of Agr, (b) Deep TMHMM was used to visualise membrane-spanning portions of particular proteins based on amino acid residues</p></figcaption></figure></section><section id="Sec24"><h3 class="pmc_sec_title">Crystallisation studies</h3>
<p id="Par50">XtalPred employs a pair of crystallisation categorization methods: expert pool and random forest. The expert pool approach combines multiple protein features like GRAVY, length, insertion value structural disorder, etc., with crystallisation chances into an overall crystallisation value. Scores vary from 1 to 5. Lesser the value, greater the proteinâ€™s crystallisation likelihood [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Except AgrA, all Agrs in both the species show an expert pool crystallizability value of 5 (Figure S4a1-d1 and S4e1-h1; supplementary section). Aside from that, the random forest approach makes use of acid composition and roughness on protein surfaces, as well as hydrophobicity. A recent research reveals that the X-ray crystal structure of the C-terminal DNA-binding domain of the <em>S. aureus</em> response regulator AgrA in contact with a 15Â bp DNA helix comprising unique 9Â bp signature interaction motif [<a href="#CR95" class="usa-link" aria-describedby="CR95">95</a>]. The random forest score for all Agrs was more than eight. XtalPred uses expert pool and random forest scores to determine the likelihood of crystallization. The random forest model tests the predictive potential of protein features like surface ruggedness and hydrophobicity of the protein. It is one of the most precise learning algorithms for assessing crystallization propensity utilizing sequence-related attributes. According to earlier research by Jahangiri et al. [<a href="#CR101" class="usa-link" aria-describedby="CR101">101</a>] and Manna et al. [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>], a protein is unlikely to crystallize if its score is more than six. Consequently, it is likely that the Agrs under study are difficult to crystallize, so in silico structural investigation is a good substitute for analyzing their physical makeup. For <em>S. aureus</em> strains, AgrB was 8, AgrA was 10, AgrC and AgrD were 11 (Figure S4a2-d2; supplementary section). In <em>S. epidermidis</em>, the AgrB score was 8, AgrA and D were 9, and AgrC was 10 (Fig S4e2-h2; supplementary section). It may, therefore, be inferred that the examined Agrs, with the exception of AgrA in both <em>Staphylococcus</em> sp. are extremely difficult to crystallise, so computational structural analyses are considered as a prospective alternate option for their morphological assessment.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Secondary structure prediction</h3>
<p id="Par51">ExPASy SOPMA and the GOR IV server were utilised to forecast the percentage of various secondary structures in Agr proteins found in <em>S. aureus</em> and <em>S. epidermidis</em> strains in this present investigation. SOPMA analysis indicated the highest frequency of Î±-helices and the lowest frequency of Î²-turns in all Agrs (Fig.Â <a href="#Fig3" class="usa-link">3</a>a), whereas GOR IV analysis reveal highest frequency of Î±-helices and the lack of Î²-turns in all the AGRs found in both <em>Staphylococcus</em> sp. (Fig.Â <a href="#Fig3" class="usa-link">3</a>b). AgrA from <em>Staphylococcus pseudintermedius</em> is found to be mostly composed of Î±-helices and is hydrophilic in nature. Previous investigations reveal that AgrA primarily comprises of Î±-helices with twists and some Î²-sheets [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>], where removal of helix Î±âˆ’1 significantly impacted the structure, DNA binding and rigidity, highlighting the significance of the unique Î±-helix in a virulence regulator from <em>S. aureus</em> [<a href="#CR96" class="usa-link" aria-describedby="CR96">96</a>]. Studies also show that AgrA has the most extended strands, while AgrC has the most Î±-helices, and AgrD has the most random coils and Î²-turns among the Agrs.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374375_12866_2025_4257_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a805/12374375/67fc576f3630/12866_2025_4257_Fig3_HTML.jpg" loading="lazy" id="MO3" height="416" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The proportion of secondary structural characteristics in accessory gene regulators (Agrs) in <em>S. aureus</em> (SA) and <em>S. epidermidis</em> (SE) strains: (<strong>a</strong>) ExPASy SOPMA and (<strong>b</strong>) GORIV servers were used to forecast percentage of different secondary structures (Î±-helix, Î²-turn, random coil, and extended strand)</p></figcaption></figure><p id="Par52">Various amino acids in different locations in the structure of a protein have immediate effects on its structural integrity. Furthermore, amino acid variation affects the structure of proteins [<a href="#CR97" class="usa-link" aria-describedby="CR97">97</a>]. In order to comprehend its impact on protein, twenty amino acids that are present in Agrs have been grouped into five categories depending on their charges and polarity: polar uncharged (Cys, His, Asn, Gln, Ser, Thr), negatively charged (Asp, Glu), and positively charged (Arg, Lys), non-polar aliphatic (Ala, Gly, Leu, Ile, Met, Pro, Val), non polar aromatic (Phe, Tyr, Trp) [<a href="#CR98" class="usa-link" aria-describedby="CR98">98</a>]. Percentage distribution patterns of amino acids for all the Agrs of both species are summarised in Figure S5 (supplementary section). Leu and Ile are the most prevalent amino acids in all Agrs. From the Agrs investigated, AgrD of <em>S. epidermidis</em> includes the most Leu (17.4%), whereas AgrC of <em>S. aureus</em> contained the most Ile (16.7%). In AgrC, a putative leucine zipper was discovered, having four leucines separated by six amino acid residues [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. For all Agrs in this study, Arg frequency is around 5% of the twenty amino acids. A prior research found that arginine inhibits glucose catabolic suppression and stimulates <em>S. aureus</em> growth in anaerobic conditions [<a href="#CR99" class="usa-link" aria-describedby="CR99">99</a>]. Among non-polar aromatic amino acids, Trp has been identified to be comparatively small in concentration, but Tyr and Phe are distributed equally throughout all Agrs except AgrA of <em>S. aureus</em> strains. Among the polar uncharged amino acids, Asn, Ser, and Thr were found to be especially numerous, while Cys was the least prevalent, with just 0.2% present in AgrC of <em>S. epidermidis</em>. In case of polar charged amino acids, the negatively charged Asp and Glu outweighed the positively charged residues. Except AgrD, each Agr contains Arg. These charged amino acid residues contribute in developing the salt bridges that help to stabilize the structures of proteins.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Tertiary structure prediction</h3>
<p id="Par53">TrRosetta and AlphaFold were used for modelling the 3D structure of Agrs for further studies. Models from both online tools were evaluated for each Agr with PROCHECK and ERRAT. PROCHECK uses a Ramachandran plot to identify the computationally permissible areas for backbone dihedral angles Ïˆ vs. Ï† of amino acid sequences in a particular protein structure [<a href="#CR100" class="usa-link" aria-describedby="CR100">100</a>]. ERRAT evaluates the general quality level of the model. It was observed that the configurations predicted by both the online servers were adequate and suitable for additional investigation [<a href="#CR101" class="usa-link" aria-describedby="CR101">101</a>]. The <em>agr</em> locus from <em>S. epidermidis</em> and <em>S. aureus</em> strains had a 68% total homology [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. The comprehensive findings from these investigations are shown in Table S3 (supplementary section). This screening approach revealed that the TrRosseta architecture was considerably more trustworthy for AgrA, AgrD of <em>S. aureus</em> strains, and AgrA, AgrB, AgrC of <em>S. epidermidis</em>, whereas Alphafold forecasted the molecular makeup of AgrB, AgrC of <em>S. aureus</em> strains, and AgrD of <em>S. epidermidis</em>. PROCHECK demonstrated an enhanced Ramachandran plot for the Agrs following refining (TableÂ <a href="#Tab4" class="usa-link">4</a>). Amino acid residues in the most favoured locations for AgrA, AgrB, AgrC, and AgrD in <em>S. aureus</em> were found as 92.4, 95.4, 95.6, and 90.2%, respectively (Fig.Â <a href="#Fig4" class="usa-link">4</a>i-l), but in <em>S. epidermidis</em> they are 92.9, 95.7, 94.9, and 94.7%, respectively (Fig.Â <a href="#Fig5" class="usa-link">5</a>i-l). On a Ramachandran plot, a high-quality protein structure with good stereochemistry is shown by more than 90% of residues in the most favored region. This represents the optimal protein structure quality [<a href="#CR102" class="usa-link" aria-describedby="CR102">102</a>]. Previous studies on <em>S. aureus</em> reported that 95.8 and 4.2% of the amino acid residues of AgrC (Protein data bank ID-4BXI), Besides that 96.9 and 3.1% amino acid residues of AgrA (Protein data bank ID- 4XYO) were located in the most favored regions and allowed regions, respectively [<a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>, <a href="#CR103" class="usa-link" aria-describedby="CR103">103</a>].</p>
<section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Percentage of amino acid residue contribution in most favoured regions of Ramachandran plot analyses of different accessory gene regulators (Agrs) in selected strains of <em>S. aureus</em> (SA) and <em>S. epidermidis</em> (SE) through PROCHECK</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Amino acid residues (%)</th>
<th align="left" colspan="1" rowspan="1">AgrA SA</th>
<th align="left" colspan="1" rowspan="1">AgrB SA</th>
<th align="left" colspan="1" rowspan="1">AgrC SA</th>
<th align="left" colspan="1" rowspan="1">AgrD SA</th>
<th align="left" colspan="1" rowspan="1">AgrA<br>SE</th>
<th align="left" colspan="1" rowspan="1">AgrB<br>SE</th>
<th align="left" colspan="1" rowspan="1">AgrC SE</th>
<th align="left" colspan="1" rowspan="1">AgrD SE</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Most favoured regions</strong></td>
<td align="left" colspan="1" rowspan="1">92.4</td>
<td align="left" colspan="1" rowspan="1">95.4</td>
<td align="left" colspan="1" rowspan="1">95.6</td>
<td align="left" colspan="1" rowspan="1">90.2</td>
<td align="left" colspan="1" rowspan="1">92.9</td>
<td align="left" colspan="1" rowspan="1">95.5</td>
<td align="left" colspan="1" rowspan="1">94.6</td>
<td align="left" colspan="1" rowspan="1">94.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Additionally allowed regions</strong></td>
<td align="left" colspan="1" rowspan="1">6.7</td>
<td align="left" colspan="1" rowspan="1">4.6</td>
<td align="left" colspan="1" rowspan="1">3.9</td>
<td align="left" colspan="1" rowspan="1">7.3</td>
<td align="left" colspan="1" rowspan="1">5.3</td>
<td align="left" colspan="1" rowspan="1">4.5</td>
<td align="left" colspan="1" rowspan="1">5.2</td>
<td align="left" colspan="1" rowspan="1">5.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Generously allowed regions</strong></td>
<td align="left" colspan="1" rowspan="1">0.4</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.5</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0.6</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><strong>Disallowed regions</strong></td>
<td align="left" colspan="1" rowspan="1">0.4</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">2.4</td>
<td align="left" colspan="1" rowspan="1">1.2</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0</td>
<td align="left" colspan="1" rowspan="1">0</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374375_12866_2025_4257_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a805/12374375/1a26d7594f7d/12866_2025_4257_Fig4_HTML.jpg" loading="lazy" id="d33e2345" height="502" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Tertiary structure prediction and evaluation of different accessory gene regulators (Agrs) in <em>S. aureus</em>: Structure of AgrA (a), AgrB (b), AgrC (c), AgrD (d) along with the Z-scores regarding AgrA (e), AgrB (f), AgrC (g), AgrD (h). Evaluation of protein presented by Ramachandran plot of AgrA (i), AgrB (j), AgrC (k), AgrD (l) displayed amino acid locations in favoured [red], additionally allowed [dark yellow], generously allowed [light yellow] and disallowed region [white], and additionally protein flexibility evaluation of AgrA (m, RMSF range: 0-3.5 Ã…), AgrB (n, RMSF range: 0-3.5 Ã…), AgrC (o, RMSF range: 0â€“5 Ã…), AgrD (p, RMSF range: 0â€“5 Ã…) are presented</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374375_12866_2025_4257_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a805/12374375/a07f69c696ff/12866_2025_4257_Fig5_HTML.jpg" loading="lazy" id="d33e2356" height="499" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Tertiary structure prediction and evaluation of different accessory gene regulators (Agrs) in <em>S. epidermidis</em>: Structure of AgrA (a), AgrB (b), AgrC (c), AgrD (d) along with the Z-scores regarding AgrA (e), AgrB (f), AgrC (g), AgrD (h). Evaluation of protein presented by Ramachandran plot of AgrA (i), AgrB (j), AgrC (k), AgrD (l) displayed amino acid locations in favoured [red], additionally allowed [dark yellow], generously allowed [light yellow] and disallowed region [white], and additionally protein flexibility evaluation of AgrA (m, RMSF range: 0-3.5 Ã…), AgrB (n, RMSF range: 0-2.5 Ã…), AgrC (o, RMSF range: 0â€“5 Ã…), AgrD (p, RMSF range: 0â€“24 Ã…) are presented</p></figcaption></figure><p id="Par54">The ProSA analysed the post-refinement attributes of improved models. It assesses total protein makeup using a Z-plot [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. The Z-scores for the projected configurations of AgrA, AgrB, AgrC, and AgrD in <em>S. aureus</em> were âˆ’â€‰8.54, âˆ’4.21, âˆ’4.35, and âˆ’â€‰0.05, respectively (Fig.Â <a href="#Fig4" class="usa-link">4</a>e-h), but in <em>S. epidermidis</em> they were âˆ’â€‰7.85, âˆ’3.6, âˆ’4.4, and âˆ’â€‰1.48 (Fig.Â <a href="#Fig5" class="usa-link">5</a>e-h). Except for AgrC in both the species, the Z-score of Agrs fall inside the intersecting area of X-ray crystallography. Salt bridges have a vital role in the stability of protein structures [<a href="#CR104" class="usa-link" aria-describedby="CR104">104</a>]. It was projected that throughout all Agrs, Lys is the most prevalent implicating salt bridge building, whereas His happens to be the least. In AgrA, AgrC, and AgrD, the Lys-Glu salt bridge is the most prevalent one, while the Arg-Asp salt bridge is prominent in AgrB of both species. A salt bridge that connects Asp157 and His208, is positioned in a turn of the 3â€“10 helix, helps to stabilise the interactions within the sheets. Arg195 (which may be found in the loop connecting Î±-helix 1 and Î²-strand 6) and Glu141 (which is found at the starting point of Î²-strand1) form salt bridges with Asp157. Salt bridges connecting Asp176-Lys223 and His174-Glu226 stabilise the interaction in AgrA [<a href="#CR95" class="usa-link" aria-describedby="CR95">95</a>]. Noticeably, AgrA exhibited the same number of Arg-Lys and Glu-His salt bridges in <em>S. aureus</em> and <em>S. epidermidis</em> (Figure S6, Supplementary section). Charged amino acid residues in trans-membrane helices inhibit appropriate membrane penetration. Energy burden of membrane penetration is decreased by the development of salt bridges [<a href="#CR105" class="usa-link" aria-describedby="CR105">105</a>]. It could be concluded that, all the projected Agrs are of high calibre and might be potentially useful in the investigation of microbial cellular functions.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Flexibility modelling of tertiary structure</h3>
<p id="Par55">Protein elasticity modelling provides knowledge about changes in persisting amino acid residues across time. The CABS-flex 2.0 server was utilized to aid in quick modelling of structural flexibility. It acts as an outstanding simulation server that allows the depiction of large orientation changes in protein structure [<a href="#CR106" class="usa-link" aria-describedby="CR106">106</a>]. Similarly, protein elasticity analyses reveal the existence of functioning amino acid residues, which alter and make proteins extremely flexible in key cellular functions. Yet, extrinsic influences may limit or vary the flexibility qualities [<a href="#CR107" class="usa-link" aria-describedby="CR107">107</a>]. Each Agr model was investigated by using the default settings. Flexibility oscillation was found to be larger in AgrA, AgrB, and AgrC throughout a span of RMSF (Ã…) score of 0â€“5. The least oscillation was detected in AgrD with the greatest RMSF (Ã…) score of 0.5â€“11 in <em>S. aureus</em> and <em>S. epidermidis</em> strains used under present study. FigureÂ <a href="#Fig4" class="usa-link">4</a>m-p (<em>S. aureus</em>) and 5Â m-p (<em>S. epidermidis</em>) present the comparison of flexibility modelling plots for each Agr. The RMSF was utilised to assess if the framework is trustworthy throughout simulation or if it departs from original dimensions [<a href="#CR108" class="usa-link" aria-describedby="CR108">108</a>]. Except for AgrD, all other Agrs of both <em>Staphylococcus</em> sp. is steady, in accordance with the RMSF (Figure S15 and S16; Supplementary section).</p></section><section id="Sec28"><h3 class="pmc_sec_title">Domain and motif evaluation</h3>
<p id="Par56">Primary structure analysis of Agr included motif and domain analysis. AgrA of <em>S. aureus</em> and <em>S. epidermidis</em> strains contains two distinct domains. These include the signal transduction response regulator receiver domain, spanning residues 1<sup>st</sup> âˆ’â€‰125<sup>th</sup>, and the LytTR DNA-binding domain, spanning residues 143<sup>rd</sup> âˆ’â€‰238<sup>th</sup>. AgrC of <em>S. aureus</em> contains a histidine kinase domain located between residues 325<sup>th</sup> âˆ’â€‰427<sup>th</sup>. In <em>S. epidermidis</em>, the histidine kinase domain of AgrC spans residues 324<sup>th</sup> âˆ’â€‰427<sup>th</sup>. AgrD of <em>S. aureus</em> includes a quorum-sensing domain ranging 7<sup>th</sup> âˆ’â€‰42<sup>nd</sup> residue, while in <em>S. epidermidis</em>, this domain extends through residues 7<sup>th</sup> âˆ’â€‰43<sup>rd</sup>. Mutations in the histidine kinase domain of <em>S. aureus</em> AgrC alters the synthesis of autoinducing peptide. Different domains in both species are presented in TableÂ <a href="#Tab5" class="usa-link">5</a>. The regulatory domain associated with the LytTR domain has been detected in the clinical isolates of <em>S. epidermidis</em> A086 and <em>S. aureus</em> EMRSA1 (ST239-MRSA-III), which is the oldest pandemic MRSA strain that has been circulating in several nations since the 1970â€‰s (Van Wamel et al. 1998). The <em>C</em>-terminal segment of AgrC has been classified as a member of the autophosphorylating histidine protein kinase family in both species. AgrD is the pro-peptide for an AIP in <em>S. aureus</em> RN6911 and <em>S. epidermidis</em> 12228 that when reaches a specific level, starts the Agr cell QS mechanism and thereafter regulates the transcription of virulence genes [<a href="#CR109" class="usa-link" aria-describedby="CR109">109</a>, <a href="#CR110" class="usa-link" aria-describedby="CR110">110</a>]. Previous studies reported that <em>S. aureus</em> AgrA includes two domains. The first one is the receiver domain (1<sup>st</sup> âˆ’130<sup>th</sup> residues), while the other is the DNA-binding domain (138<sup>th</sup> âˆ’238<sup>th</sup> residues) [<a href="#CR103" class="usa-link" aria-describedby="CR103">103</a>]. There are several drugs currently being found to focus on LytTR DNA-binding domain of AgrA and block its attachment to the P3 promoter [<a href="#CR111" class="usa-link" aria-describedby="CR111">111</a>]. AgrC contains histidine kinase domain from 293 to 430 amino acid residues [<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>].</p>
<section class="tw xbox font-sm" id="Tab5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Different domains with the corresponding amino acid length of all accessory gene regulators (Agrs) in <em>S. aureus</em> and <em>S. epidermidis</em> evaluated through interpro scan</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Organism</th>
<th align="left" colspan="1" rowspan="1">Protein</th>
<th align="left" colspan="1" rowspan="1">Domain</th>
<th align="left" colspan="1" rowspan="1">Amino acid length</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="4" colspan="1"><em>Staphylococcus aureus</em></td>
<td align="left" rowspan="2" colspan="1">AgrA</td>
<td align="left" colspan="1" rowspan="1">Signal transduction response regulator receiver</td>
<td align="left" colspan="1" rowspan="1">1-125</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LytTR DNA-binding domain</td>
<td align="left" colspan="1" rowspan="1">143â€“238</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AgrC</td>
<td align="left" colspan="1" rowspan="1">Histidine kinase</td>
<td align="left" colspan="1" rowspan="1">325â€“427</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AgrD</td>
<td align="left" colspan="1" rowspan="1">Quorum sensing</td>
<td align="left" colspan="1" rowspan="1">7â€“42</td>
</tr>
<tr>
<td align="left" rowspan="4" colspan="1"><em>Staphylococcus epidermidis</em></td>
<td align="left" rowspan="2" colspan="1">AgrA</td>
<td align="left" colspan="1" rowspan="1">Signal transduction response regulator receiver</td>
<td align="left" colspan="1" rowspan="1">1-125</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LytTR DNA-binding domain</td>
<td align="left" colspan="1" rowspan="1">143â€“238</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AgrC</td>
<td align="left" colspan="1" rowspan="1">Histidine kinase</td>
<td align="left" colspan="1" rowspan="1">324â€“427</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">AgrD</td>
<td align="left" colspan="1" rowspan="1">Quorum sensing</td>
<td align="left" colspan="1" rowspan="1">7â€“43</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par57">Agr amino acid sequences obtained from the UniProt library were utilised as initial sequences in the MEME server to envisage the position of motifs within the structures of proteins. Three kinds of motifs comprising consensus sequences such as VYVQYDDIMFFESSTKSHRLIAHLDNRQIEFYGNLKELAQLDERFERCHN, NINKLASYIPKHNELDEIQFVTERSGLQL, and CEDDPKQRENMVSIIKNYIMIEEKPMEJALATDDPYEVLEQAKELNDIGC were prevalent throughout the Agrs (Fig.Â <a href="#Fig6" class="usa-link">6</a>a). In <em>S. aureus</em> and <em>S. epidermidis</em> strains under study, it was found that AgrA possess a total of three kinds of motifs, but AgrB, AgrC, and AgrD possess only one. To recognise <em>S. lugdunensis</em> AgrC-1, the <em>C</em>-terminal hydrophobic motif must have a minimum of one aromatic constituent [<a href="#CR112" class="usa-link" aria-describedby="CR112">112</a>]. Interaction of AgrC suppression between <em>S. lugdunensis</em> group-I and group-II, as well as cross-species inhibitory activity between <em>S. epidermidis</em> group-1 and <em>S. lugdunensis</em> group-I is seen, similar to other <em>Staphylococcal</em> sp [<a href="#CR112" class="usa-link" aria-describedby="CR112">112</a>]. The second motif appears in all Agrs, whereas the first and third motifs were only found in AgrA. Motifs act as hallmark sequences and may be used for recognising any kind of protein. The anticipated <em>e</em>-value of the motif indicates the level of its functional accuracy. Their <em>e</em>-values ranged from 1.4â€‰Ã—â€‰10<sup>âˆ’15</sup> to 1.1â€‰Ã—â€‰10<sup>âˆ’13</sup> (Fig.Â <a href="#Fig6" class="usa-link">6</a>b-d). The <em>N</em>-terminal segment of AgrD peptide has an unique amphipathic alpha helical motif which stabilises it and produces mature autoinducer peptide [<a href="#CR111" class="usa-link" aria-describedby="CR111">111</a>].</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374375_12866_2025_4257_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a805/12374375/1376830b7db4/12866_2025_4257_Fig6_HTML.jpg" loading="lazy" id="d33e2679" height="1016" width="704" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Visualization of motifs in accessory gene regulators (Agrs): (a) prevalence motif conservation; Sequence alignment of different motif sequences of (b) AgrA, (c) all Agrs, (d) AgrA between <em>S. aureus</em> and <em>S. epidermidis</em></p></figcaption></figure></section><section id="Sec29"><h3 class="pmc_sec_title">Protein-protein docking studies</h3>
<p id="Par58">In protein-protein docking investigation, it was observed that AgrA and AgrC of <em>S. epidermidis</em> display the highest docking score among all Agrs included in the study (Fig.Â <a href="#Fig7" class="usa-link">7</a>; Table S4, supplementary section). AgrA docked more strongly with AgrC in <em>S. epidermidis</em> (âˆ’293.08Â kcal mol<sup>âˆ’1</sup>) than that in <em>S. aureus</em> (âˆ’288.23Â kcal mol<sup>âˆ’1</sup>). Docking between AgrB and AgrD revealed that <em>S. aureus</em> has a higher docking score (âˆ’292.82Â kcal mol<sup>âˆ’1</sup>) than <em>S. epidermidis</em> (âˆ’278.69Â kcal mol<sup>âˆ’1</sup>). The confidence level for every protein-protein docking was greater than 0.92. The LGscore between AgrB and AgrD was greater than 1.5, and between AgrA and AgrC it was greater than 4.6, showing that the docking was correct and considerable, respectively in both species (Figure S7, supplementary section). Figure S9 describes the protein-protein docking interface residues for all Agrs. Microbial communities benefit from the ability of the QS machinery to regulate their reactions to external stimuli. Additionally, it controls the synthesis of virulence components, such as biofilm, which aids in the pathophysiology of bacterial infections. Small chemical molecules known as autoinducers are usually responsible for mediating this signaling pathway. Microbes can become nonpathogenic and be easier to eradicate if QS is blocked, for example, by slowing down the manufacture of the signal molecule, breaking it down, or stopping it from attaching to the response regulator. Numerous substances, including furanones and 5-acetyl-4-methyl-2-(3-pyridyl) thiazole, have been suggested as QS inhibitors. In the quest to treat MRSA infections, MD analysis showed encouraging interactions between the QS inhibitors and AgrA in <em>S. aureus</em>, validating their antivirulence and antibiofilm capabilities [<a href="#CR113" class="usa-link" aria-describedby="CR113">113</a>]. A negative score in protein-protein molecular docking indicates a good interaction; higher binding affinities are indicated by more negative values. AgrA and AgrC were shown to have binding affinities of âˆ’9.1 and âˆ’â€‰9.2Â kcal/mol in <em>S. aureus</em> and <em>S. epidermidis</em>, respectively, and to have a higher interaction probability than that in AgrB-AgrD. Consistent with the observed docking score the AgrA-AgrC interaction of <em>S. epidermidis</em> was found to have the most interfacing residues, forty-eight, five hydrogen bonds between Lys57, His68, Glu14, His18, Ser89 of AgrA and Asp169, Ile181, Arg146, and Arg206 of AgrC, respectively (Figure S13 and S14; supplementary section). Glu14 and His18 are associated through hydrogen bond with Arg146. On the other hand, AgrA-AgrC in <em>S. aureus</em> has forty-two interface residues, including two salt bridges between Asp112 and Glu115 of AgrA and Arg134 of AgrC, as well as one hydrogen bond between Glu115 of AgrA and Tyr130 of AgrC. In <em>S. aureus</em>, AgrB-AgrD interaction were through two hydrogen bonds between Tyr51, Tyr143 of AgrB and Met1, Ser26 of AgrD, respectively but in case of <em>S. epidermidis</em> there was no salt bridge or hydrogen bond (TableÂ <a href="#Tab6" class="usa-link">6</a>). All the four complexes lack disulfide linkage (Figure S8; supplementary section). The Ramachandran map of docked complexes reveal that only three amino acid residues of <em>S. epidermidis</em> AgrA-AgrC a located in the disallowed area (Figure S10, supplementary section). All the four complexes have four amino acid residues at the end residues region excluding Gly and Pro (Table S5, supplementary section). The equilibrium constant that results from the dissociation of molecules bound together in a complex is represented by the K<sub>d</sub>. A chemical complex is said to be more strongly linked or to have a higher binding affinity between its molecules if its dissociation constant value is smaller. Based on the dissociation constant and binding affinity values, AgrB-AgrD has a higher K<sub>d</sub> score than AgrA-AgrC in both species. That defines stronger binding affinity between AgrA-AgrC.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374375_12866_2025_4257_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a805/12374375/953afae3f965/12866_2025_4257_Fig7_HTML.jpg" loading="lazy" id="d33e2763" height="481" width="669" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Protein-protein interacting residues between AgrA-AgrC and AgrB-AgrD of <em>S. aureus</em> (SA) and <em>S. epidermidis</em> (SE) selected strains. Different interactions, <em>viz</em>., salt bridge (red), disulphide bond (yellow), hydrogen bond (blue), and nonbond (orange) between AGRs</p></figcaption></figure><section class="tw xbox font-sm" id="Tab6"><h4 class="obj_head">Table 6.</h4>
<div class="caption p"><p>Number of interfacing residues and different interaction of AgrA-AgrC and AgrB-AgrD of <em>S. aureus</em> and <em>S. Epidermidis.</em> Binding affinity (Î”G<sup>0</sup>) and dissociation score (K<sub>d</sub>) at 37Â Â°C analyzed using PRODIGY server</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Protein</th>
<th align="left" colspan="1" rowspan="1">No. of interface residues</th>
<th align="left" colspan="1" rowspan="1">Interface distance (Ã…)</th>
<th align="left" colspan="1" rowspan="1">Î”G<sup>0</sup> (kcal mol<sup>âˆ’1</sup>)</th>
<th align="left" colspan="1" rowspan="1">K<sub>d</sub> at 37 â„ƒ</th>
<th align="left" colspan="1" rowspan="1">Salt bridge</th>
<th align="left" colspan="1" rowspan="1">Disulfide bond</th>
<th align="left" colspan="1" rowspan="1">Hydrogen bond</th>
<th align="left" colspan="1" rowspan="1">Nonbond</th>
</tr></thead>
<tbody>
<tr><td align="left" colspan="9" rowspan="1">AgrA-AgrC (<em>S. aureus</em>)</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAgrA</td>
<td align="left" colspan="1" rowspan="1">22</td>
<td align="left" colspan="1" rowspan="1">1106</td>
<td align="left" rowspan="2" colspan="1">âˆ’9.1</td>
<td align="left" rowspan="2" colspan="1">4.1â€‰Ã—â€‰10<sup>âˆ’7</sup>
</td>
<td align="left" rowspan="2" colspan="1">2</td>
<td align="left" rowspan="2" colspan="1">0</td>
<td align="left" rowspan="2" colspan="1">1</td>
<td align="left" rowspan="2" colspan="1">192</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAgrC</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">1156</td>
</tr>
<tr><td align="left" colspan="9" rowspan="1">AgrA-AgrC (<em>S. epidermidis</em>)</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAgrA</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">1261</td>
<td align="left" rowspan="2" colspan="1">âˆ’9.2</td>
<td align="left" rowspan="2" colspan="1">3.2â€‰Ã—â€‰10<sup>âˆ’7</sup>
</td>
<td align="left" rowspan="2" colspan="1">0</td>
<td align="left" rowspan="2" colspan="1">0</td>
<td align="left" rowspan="2" colspan="1">5</td>
<td align="left" rowspan="2" colspan="1">180</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAgrC</td>
<td align="left" colspan="1" rowspan="1">25</td>
<td align="left" colspan="1" rowspan="1">1202</td>
</tr>
<tr><td align="left" colspan="9" rowspan="1">AgrB-AgrD (<em>S. aureus</em>)</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAgrB</td>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="left" colspan="1" rowspan="1">784</td>
<td align="left" rowspan="2" colspan="1">âˆ’6.6</td>
<td align="left" rowspan="2" colspan="1">2.1â€‰Ã—â€‰10<sup>âˆ’5</sup>
</td>
<td align="left" rowspan="2" colspan="1">0</td>
<td align="left" rowspan="2" colspan="1">0</td>
<td align="left" rowspan="2" colspan="1">2</td>
<td align="left" rowspan="2" colspan="1">154</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAgrD</td>
<td align="left" colspan="1" rowspan="1">14</td>
<td align="left" colspan="1" rowspan="1">937</td>
</tr>
<tr><td align="left" colspan="9" rowspan="1">AgrB-AgrD (<em>S. epidermidis</em>)</td></tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAgrB</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">716</td>
<td align="left" rowspan="2" colspan="1">âˆ’5.5</td>
<td align="left" rowspan="2" colspan="1">1.3â€‰Ã—â€‰10<sup>âˆ’5</sup>
</td>
<td align="left" rowspan="2" colspan="1">0</td>
<td align="left" rowspan="2" colspan="1">0</td>
<td align="left" rowspan="2" colspan="1">0</td>
<td align="left" rowspan="2" colspan="1">149</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">â€ƒAgrD</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">802</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par59">The present findings regarding the interacting interfaces between two Agr regulators might serve as very much promising targets for potential antimicrobials from various sources to exert their anti-QS and anti-biofilm effects to successfully combat the infections. Around 90% of <em>Staphylococcus</em> pathogenecity involves skin and soft tissue infection (SSTI) and the existing reports reveal that infection caused by <em>Staphylococcal </em><em>sp</em><em>.</em> is predominantly biofilm-based that eventually turns to be the underlying cause for emergence of large number of multi-drug resistant (MDR) pathogens [<a href="#CR114" class="usa-link" aria-describedby="CR114">114</a>]. Besides, <em>Staphylococcus</em> sp., being opportunistic human pathogen in case of medical device/implant-related hospital acquired infections have turned to be an emergent issue in clinical settings responsible for a great deal of morbidity and mortality [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>]. Biofilm formation abilities on implant biomaterials play leading roles in persistence of these infectious agents for their adaptive advantages to survive in hostile environments, and overcoming the biocide, and host defense challenges [<a href="#CR115" class="usa-link" aria-describedby="CR115">115</a>]. Although, biofilm forming abilities in <em>S. aureus</em> and <em>S. epidermidis</em> vary distinctively from strains to strains but it has been observed most of the isolates found in clinical settings were high biofilm-formers. Therefore, alternative therapeutic agents to the conventional antibiotics are of high demand in combating these biofilm-based infections [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR116" class="usa-link" aria-describedby="CR116">116</a>]. In this context reports have revealed the applicability of the anti-quorum sensing agents as promising therapeutics that may potentially disintegrate biofilm structures as well as combating antimicrobial resistance [<a href="#CR117" class="usa-link" aria-describedby="CR117">117</a>]. It is worth mentioning that Agr is positively allied with this type of infection. Strongly conserved but heterogeneous QS machinery called the Agr is involved in colonisation, pathogenicity, and biofilm formation in <em>Staphylococci</em> sp. that justifies their role in building-up and persistence of the infection and subsequent clinical significance [<a href="#CR118" class="usa-link" aria-describedby="CR118">118</a>]. In an effort to prevent the negative effects of conventional antibiotics on human health and the surroundings, organic compounds have been thoroughly investigated as significant sources of biocompatible antibiofilm agents [<a href="#CR119" class="usa-link" aria-describedby="CR119">119</a>]. For example, plant-derived essential oils, flavonoids, phenolic acids, and terpenoids have shown biofilm-inhibitory effect by interacting with AgrA and AgrC [<a href="#CR120" class="usa-link" aria-describedby="CR120">120</a>]. Furthermore, a growing number of novel antibiofilm targets of natural compounds have been investigated by computer-based computational screening. The docking score in computational screening represents the possible energy shift that takes place when protein-protein interaction occurs. Accordingly, strong binding is indicated by a greater negative score, while weak or nonexistent binding is indicated by a less negative score [<a href="#CR121" class="usa-link" aria-describedby="CR121">121</a>]. The present observation reveals that a superior AgrA-AgrC interaction could be obtained to AgrB-AgrD in <em>Staphylococcus</em> sp. LytTR domain of AgrA is a gene expression regulator that controls the transcription of several virulence and toxins genes, allowing the bacteria to enter host tissues, and eluding the defence system of the human body resulting in serious illness [<a href="#CR95" class="usa-link" aria-describedby="CR95">95</a>]. Agr serves as a key QS regulator in <em>Staphylococcus</em> sp., enabling the secretion of several toxins by the bacteria and enhancing their pathogenicity. By triggering the production of the <em>agrBDCA</em> operon, which codes for the machinery of QS system, the P2 promoter powers autoregulation network of Agr. On the other hand, the P3 promoter uses RNAIII for modulating different toxins [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR118" class="usa-link" aria-describedby="CR118">118</a>, <a href="#CR122" class="usa-link" aria-describedby="CR122">122</a>]. Additionally, AgrA triggers the transcription of RNAIII from the promoter <em>agr</em>P3. Toxins, <em>viz</em>., Î±-hemolysin (<em>hla</em>), Î²-hemolysin (<em>hlb</em>), Î´-haemolysin (<em>hld</em>) and virulence factors, <em>viz</em>., protein A (<em>spa</em>), toxic shock syndrome toxin-1 (<em>tsst</em>) expression are regulated by RNAIII [<a href="#CR111" class="usa-link" aria-describedby="CR111">111</a>]. PSM (Phenol soluble modulin) - Î±, Î², and Î³ are tiny, amphipathic, Î±-helical peptides that belong to the PSM family. AgrA can directly bind to the promoters of the <em>psm</em> in addition to activating the P2 and P3 promoters in the <em>agr</em> locus. The P3 promoter transcribes the Î´-toxin gene, which is situated in the RNAIII portion of the <em>agr</em> locus. The cytolytic characteristic of PSM peptides is critical to their pathogenicity. Î´-toxin exhibits mild cytolytic activity, PSMÎ² peptides are non-cytolytic, and PSMÎ±, particularly PSMÎ±3, has a strong capacity to lyse erythrocytes, epithelial cells, and leukocytes [<a href="#CR123" class="usa-link" aria-describedby="CR123">123</a>, <a href="#CR124" class="usa-link" aria-describedby="CR124">124</a>]. In recent years, several studies target the Agr system for prospective drug designing and development that would interfere with quorum sensing in <em>Staphylococcus</em> sp. Studies have revealed that inhibitors of AgrA have the potential to be a novel class of anti-virulence and anti-biofilm drugs that can limit the production of virulence factors and hence lessen the pathogenicity of the bacterium [<a href="#CR118" class="usa-link" aria-describedby="CR118">118</a>]. In addition to AgrA, AgrC being a receptor of AIP, and sensor domain (residues 1 to 205) of AgrC that interacts with regulator domain (residues 1 to 130) of AgrA, can also be targeted by drug molecules [<a href="#CR125" class="usa-link" aria-describedby="CR125">125</a>]. In the present study, â‰¥â€‰20 amino acid residues of AgrA and AgrC, interacting with each other to form a protein-protein complex, might be a potential site of drug targeting to destabilize the biofilm formation in these pathogens (TableÂ <a href="#Tab6" class="usa-link">6</a>). The protein-protein interaction investigation contributes to a more thorough comprehension of binding affinity. In AgrA-AgrC complexes, <em>S. epidermidis</em> (âˆ’9.2Â kcal mol<sup>âˆ’1</sup>) has a greater binding affinity than <em>S. aureus</em> (âˆ’9.1Â kcal mol<sup>âˆ’1</sup>). However, the AgrB-AgrD interaction demonstrated that <em>S. aureus</em> (âˆ’6.6Â kcal mol<sup>âˆ’1</sup>) had a greater binding affinity than <em>S. epidermidis</em> (âˆ’5.5Â kcal mol<sup>âˆ’1</sup>).</p>
<p id="Par60">The AgrC protein, especially its cytosolic domain, interacts with the AgrA response regulator domain in <em>Staphylococcus</em> sp. The activation of the Agr two-component system, which is involved in QS, and the control of virulence genes, depends on this association [<a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>].Understanding of this QS machinery may hold the ultimate key to an effective therapeutics development. Within a single species, QS-mediated cell-to-cell communication organizes cooperative activity to improve survival under stress and damage immune systems of human host. One of the most extensively researched communication systems of human bacterial pathogens is the <em>agr</em> operon-encoded QS system in <em>Staphylococcus</em> sp. The effectiveness of the <em>agr</em> quorum-sensing system in <em>S. aureus</em> can be improved by a tighter interaction between AgrA and AgrC, which results in quicker and more reliable reactions to AIP signals. As a result, AgrA and AgrC are thought to be a key target for chemotherapeutic research [<a href="#CR126" class="usa-link" aria-describedby="CR126">126</a>â€“<a href="#CR128" class="usa-link" aria-describedby="CR128">128</a>]. Template-based molecular docking has been used to provide kinase-ligand intricate information for kinase-related drug development. This data, when used to train graph neural networks, produced more accurate binding affinity estimations than models that just employed data on ligand or drug-target interactions [<a href="#CR129" class="usa-link" aria-describedby="CR129">129</a>]. AgrA and AgrC are, therefore, appear to be seemingly promising in the management of bacterial infections and are apprehended to be useful therapeutic targets for the discovery of potential antimicrobial drugs. AgrA may be the ideal therapeutic target because it is found in the cytoplasm. Certain regions outside the spanning area, such as the 130<sup>th</sup> âˆ’146<sup>th</sup> amino acid residues, may function as therapeutic targets since AgrC is present in the transmembrane region.</p></section><section id="Sec30"><h3 class="pmc_sec_title">Molecular dynamics (MD) simulation studies</h3>
<p id="Par61">Even with extremely massive biological macromolecule, iMODS allow for the investigation of the generation of realistic conversion routes between two identical bio molecules. The novel interior construction improves the effectiveness of NMA and widens its applications [<a href="#CR130" class="usa-link" aria-describedby="CR130">130</a>]. Movement of rigidity is represented by the eigen value corresponding to every normal mode analysis. The amount of energy needed to deform the framework defines its value. The eigen value decreases when the distortion becomes simpler [<a href="#CR130" class="usa-link" aria-describedby="CR130">130</a>]. The eigen value of MD simulations indicates how stable the docked complexes are. It is equivalent to the energy needed to cause the structure to deform, and excellent structural stability is linked to low eigenvalues. The relationship between variance and eigenvalues is inverse [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. Computed eigen values of the AgrA-AgrC interaction of <em>S. aureus</em> and <em>S. epidermidis</em> were found as 2.9â€‰Ã—â€‰10<sup>âˆ’7</sup> and 6.1â€‰Ã—â€‰<sup>10âˆ’7</sup>, while that for AgrB-AgrD interaction of <em>S. aureus</em> and <em>S. epidermidis</em> were 6.2â€‰Ã—â€‰10<sup>âˆ’6</sup> and 1.5â€‰Ã—â€‰10<sup>âˆ’5</sup>, respectively (Figs.Â <a href="#Fig8" class="usa-link">8</a>c and f and <a href="#Fig9" class="usa-link">9</a>c, and f). Given the lower estimated eigenvalues of the complexes, it can be inferred from the above data that the complexes under study, namely AgrA-AgrC, were more stable than AgrB-AgrD for both species. The deformability plot displays the energy necessary to deform the amino acid residues (Figs.Â <a href="#Fig8" class="usa-link">8</a>a and d and <a href="#Fig9" class="usa-link">9</a>a, and d). Violet and green bars represent specific and overall (Figure S11a, d, S12a, and d; supplementary section) variances, respectively. The covariance distribution depicts whether correlated, uncorrelated, or anti-correlated through red, white, and blue marks (Figure S11b, e, S12b, and e; supplementary section). Each mark represents an attraction between the corresponding residues, and rigid attraction represented by the darker grey mark and vice versa. Contact map can reveal relationships between amino acid residues and regions by tracking their relative motions (Figure S11c, f, S12c, and f; supplementary section). The B-factor measures the flexibility of a protein and quantifies its ambiguity that is related to the RMSD, as seen in Figs.Â <a href="#Fig8" class="usa-link">8</a>b and e and <a href="#Fig9" class="usa-link">9</a>b, and e. These findings indicate sustained binding between respective amino acid residues with tight structure and little fluctuations in protein-protein interaction. The current work used IMODS to perform MD simulation in order to assess the stability and physical mobility of protein-protein interactions. The durability of docked proteins was also examined using NMA. The deformability and B-factor of AgrA-AgrC show peaks that correlate to the deformable areas of the protein; the higher the peak, the higher the deformability. The NMA analysis demonstrated the structural flexibility of the proteins by revealing the substantial mobility of the AgrA-AgrC aggregates. The interaction has a smaller eigen value, making it simpler to deform and more stable. Because ligands can adopt multiple conformations inside the binding pocket and proteins can alter conformation upon ligand binding. Thus flexibility is essentially required for induced fit in a potential drug-target interaction, and that is crucial for druggability [<a href="#CR131" class="usa-link" aria-describedby="CR131">131</a>]. Reports also suggest that deformability peaks close to 1 represents regions with high flexibility [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>] and that is also revealed from the Fig.Â <a href="#Fig8" class="usa-link">8</a>a and d in the present study. The observed interactions with high flexibilities in studied proteins suggest that they might be viable therapeutic target.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374375_12866_2025_4257_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a805/12374375/707866fdd84a/12866_2025_4257_Fig8_HTML.jpg" loading="lazy" id="d33e3227" height="491" width="669" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>MD simulation of AgrA-AgrC docked complex (a) deformability plot, (b) B-factor, (c) eigen value of <em>S. aureus</em> and (d) deformability plot, (e) B-factor, (f) eigen value of <em>S. epidermidis</em></p></figcaption></figure><figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12374375_12866_2025_4257_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a805/12374375/c6b3e0685338/12866_2025_4257_Fig9_HTML.jpg" loading="lazy" id="d33e3240" height="492" width="669" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>MD simulation of AgrB-AgrD docked complex (a) deformability plot, (b) B-factor, (c) eigen value of <em>S. aureus</em> and (d) deformability plot, (e) B-factor, (f) eigen value of <em>S. epidermidis</em></p></figcaption></figure></section></section><section id="Sec31"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par62">In addition to the virulence mechanism associated with catheters, <em>S. aureus</em> and <em>S. epidermidis</em> primarily cause infections of the skin and soft tissues through QS and biofilm. Because of this, viable long-term solutions for the management of antibiotic-resistant staphylococcal infections are still difficult and ultimately call for alternate strategies. Agr system of <em>Staphylococcus</em> sp. regulates the transcription of multiple factors that encode virulence properties in the current investigation, a variety of computational methods were employed to examine different sorts of structural modifications in four Agrs of <em>S. aureus</em> and <em>S. epidermidis</em>. According to genome study, natural selection is more influential than mutational pressure in determining the habits of codon usage and preferences for AU ending codons, which suggests that these Agrs might be excellent therapeutic targets. Besides that, an energy-saving tactic in pathogenic organisms is suggested by AU-rich codons. All Agrs show codon adaptability index values more than 0.65, indicating a high likelihood of adaptation and expression in a variety of environmental settings. Physicochemical characteristics reveal that AgrB of both the species was predicted to be unstable under in vitro environments. Furthermore, crystallisation study demonstrates that with the exception of AgrA, the others are extremely difficult to be crystallised and thus warrant further structural investigations. Computational investigations are regarded as simple alternative approaches for conducting structural assessments of Agrs. Secondary structural investigations indicate that Î±-helix is among the most common structure in Agrs, followed by random coils, with Leu and Ile being the most numerous amino acid residues. The current work is thought to have the ability to develop comprehensive research to determine the structural characteristics of these Agrs. AgrA and AgrC are, therefore, appear to be seemingly promising in the management of bacterial infections and are apprehended to be useful therapeutic targets for the discovery of potential antimicrobial drugs. AgrA may be the ideal therapeutic target because it is found in the cytoplasm. Certain regions outside the spanning area, such as the 130<sup>th</sup> âˆ’146<sup>th</sup> amino acid residues, may function as therapeutic targets since AgrC is present in the transmembrane region. However, in order to validate the theoretical data collected here, more thorough analytical techniques are required. To understand the physicochemical properties of these proteins and confirm existing discoveries, modern structural biology methods such as NMR spectroscopy and X-ray crystallography may be helpful.</p></section><section id="Sec954"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12374375/bin/12866_2025_4257_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (3.1MB, docx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We sincerely acknowledge Vidyasagar University, West Bengal, India for providing infrastructure support to carry out research work.</p>
<section id="FPar1"><h3 class="pmc_sec_title">Clinical trial number</h3>
<p id="Par63">Not applicable.</p></section></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>Agr</dt>
<dd><p id="Par4">Accessory gene regulator</p></dd>
<dt>AI</dt>
<dd><p id="Par5">Aliphatic index</p></dd>
<dt>CAI</dt>
<dd><p id="Par6">Codon adaptation index</p></dd>
<dt>EC</dt>
<dd><p id="Par7">Molar extinction coefficient</p></dd>
<dt>GRAVY</dt>
<dd><p id="Par8">Grand average of hydropathicity</p></dd>
<dt>II</dt>
<dd><p id="Par9">Instability index</p></dd>
<dt>MD</dt>
<dd><p id="Par10">Molecular dynamics</p></dd>
<dt>MEME</dt>
<dd><p id="Par11">Multiple expectation maximizations for motif elicitation</p></dd>
<dt>MSA</dt>
<dd><p id="Par12">Multiple sequence alignment</p></dd>
<dt>MW</dt>
<dd><p id="Par13">Molecular weight</p></dd>
<dt>NCBI</dt>
<dd><p id="Par14">National Center for Biotechnology Information</p></dd>
<dt>PI</dt>
<dd><p id="Par15">Isoelectric point</p></dd>
<dt>RCSU</dt>
<dd><p id="Par16">Relative synonymous codon usage</p></dd>
<dt>(- R)</dt>
<dd><p id="Par17">Negative charge residues</p></dd>
<dt>(+ R)</dt>
<dd><p id="Par18">Positive charge residues</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors' contributions</h2>
<p>S.D.: Writing â€“ original draft, Visualization, Methodology, Investigation, Formal analysis, Data curation, Conceptualization. T.M.: Writing â€“ review &amp; editing, Formal analysis. K.C.G.: Writing â€“ review &amp; editing, Formal analysis. M.K.: Writing â€“ review &amp; editing, Formal analysis. D.J.: Writing â€“ review &amp; editing. P.R.: Writing â€“ review &amp; editing. S.B.: Formal analysis. M.M.: Writing â€“ review &amp; editing. S.H.: Writing â€“ review &amp; editing. A.K.P.: Writing â€“ review &amp; editing, Supervision, and Conceptualization C.G.: Writing â€“ review &amp; editing, Supervision, and Conceptualization.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Subhamoy Dey acknowledges University Grants Commission (UGC), New Delhi, India for providing fellowships to carry out research work.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Data is provided within the supplementary information files.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar2"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par64">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par65">Not applicable.</p></section><section id="FPar4"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par66">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisherâ€™s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Al-Talib HI, Yean CY, Al-Jashamy K, Hasan H. Methicillin-resistant <em>Staphylococcus aureus</em> nosocomial infection trends in hospital universiti sains Malaysia during 2002â€“2007. Ann Saudi Med. 2010;30:358â€“63.
</cite> [<a href="https://doi.org/10.4103/0256-4947.67077" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2941247/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20697171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Al-Talib%20HI,%20Yean%20CY,%20Al-Jashamy%20K,%20Hasan%20H.%20Methicillin-resistant%20Staphylococcus%20aureus%20nosocomial%20infection%20trends%20in%20hospital%20universiti%20sains%20Malaysia%20during%202002%E2%80%932007.%20Ann%20Saudi%20Med.%202010;30:358%E2%80%9363." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Guchhait KC, Manna T, Barai M, Karmakar M, Nandi SK, Jana D, et al. Antibiofilm and anticancer activities of unripe and ripe <em>Azadirachta indica</em> (neem) seed extracts. BMC Complement Med Ther. 2022;22:42.
</cite> [<a href="https://doi.org/10.1186/s12906-022-03513-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8843028/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35152903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guchhait%20KC,%20Manna%20T,%20Barai%20M,%20Karmakar%20M,%20Nandi%20SK,%20Jana%20D,%20et%20al.%20Antibiofilm%20and%20anticancer%20activities%20of%20unripe%20and%20ripe%20Azadirachta%20indica%20(neem)%20seed%20extracts.%20BMC%20Complement%20Med%20Ther.%202022;22:42." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Dong Y, Speer CP, Glaser K. Beyond sepsis: <em>Staphylococcus epidermidis</em> is an underestimated but significant contributor to neonatal morbidity. Virulence. 2018;9:621â€“33.
</cite> [<a href="https://doi.org/10.1080/21505594.2017.1419117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5955464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29405832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dong%20Y,%20Speer%20CP,%20Glaser%20K.%20Beyond%20sepsis:%20Staphylococcus%20epidermidis%20is%20an%20underestimated%20but%20significant%20contributor%20to%20neonatal%20morbidity.%20Virulence.%202018;9:621%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Tong SYC, Chen LF, Fowler VG. Colonization, pathogenicity, host susceptibility, and therapeutics for <em>Staphylococcus aureus</em>: what is the clinical relevance? Semin Immunopathol. 2012;34:185â€“200.
</cite> [<a href="https://doi.org/10.1007/s00281-011-0300-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3272122/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22160374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tong%20SYC,%20Chen%20LF,%20Fowler%20VG.%20Colonization,%20pathogenicity,%20host%20susceptibility,%20and%20therapeutics%20for%20Staphylococcus%20aureus:%20what%20is%20the%20clinical%20relevance?%20Semin%20Immunopathol.%202012;34:185%E2%80%93200." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Nelson RE, Stevens VW, Khader K, Jones M, Samore MH, Evans ME, et al. Economic analysis of veterans affairs initiative to prevent Methicillin-Resistant <em>Staphylococcus aureus</em> infections. Am J Prev Med. 2016;50:S58â€“65.
</cite> [<a href="https://doi.org/10.1016/j.amepre.2015.10.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7909478/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27102860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nelson%20RE,%20Stevens%20VW,%20Khader%20K,%20Jones%20M,%20Samore%20MH,%20Evans%20ME,%20et%20al.%20Economic%20analysis%20of%20veterans%20affairs%20initiative%20to%20prevent%20Methicillin-Resistant%20Staphylococcus%20aureus%20infections.%20Am%20J%20Prev%20Med.%202016;50:S58%E2%80%9365." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Otto M. <em>Staphylococcus epidermidis</em> â€” the accidental pathogen. Nat Rev Microbiol. 2009;7:555â€“67.
</cite> [<a href="https://doi.org/10.1038/nrmicro2182" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2807625/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19609257/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Otto%20M.%20Staphylococcus%20epidermidis%20%E2%80%94%20the%20accidental%20pathogen.%20Nat%20Rev%20Microbiol.%202009;7:555%E2%80%9367." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Vestby LK, GrÃ¸nseth T, Simm R, Nesse LL. Bacterial biofilm and its role in the pathogenesis of disease. Antibiotics. 2020;9:59.
</cite> [<a href="https://doi.org/10.3390/antibiotics9020059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7167820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32028684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vestby%20LK,%20Gr%C3%B8nseth%20T,%20Simm%20R,%20Nesse%20LL.%20Bacterial%20biofilm%20and%20its%20role%20in%20the%20pathogenesis%20of%20disease.%20Antibiotics.%202020;9:59." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Yang X, Dong F, Qian S, Wang L, Liu Y, Yao K, et al. Accessory gene regulator (agr) dysfunction was unusual in <em>Staphylococcus aureus</em> isolated from Chinese children. BMC Microbiol. 2019;19: 95.
</cite> [<a href="https://doi.org/10.1186/s12866-019-1465-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6518674/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31088356/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20X,%20Dong%20F,%20Qian%20S,%20Wang%20L,%20Liu%20Y,%20Yao%20K,%20et%20al.%20Accessory%20gene%20regulator%20(agr)%20dysfunction%20was%20unusual%20in%20Staphylococcus%20aureus%20isolated%20from%20Chinese%20children.%20BMC%20Microbiol.%202019;19:%2095." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Reynolds J, Wigneshweraraj S. Molecular insights into the control of transcription initiation at the <em>Staphylococcus aureus</em> Agr operon. J Mol Biol. 2011;412:862â€“81.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2011.06.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21741390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reynolds%20J,%20Wigneshweraraj%20S.%20Molecular%20insights%20into%20the%20control%20of%20transcription%20initiation%20at%20the%20Staphylococcus%20aureus%20Agr%20operon.%20J%20Mol%20Biol.%202011;412:862%E2%80%9381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Mirani ZA, Aziz M, Khan MN, Lal I, Hassan N, ul, Khan SI. Biofilm formation and dispersal of <em>Staphylococcus aureus</em> under the influence of Oxacillin. Microb Pathog. 2013;61â€“62:66â€“72.</cite> [<a href="https://doi.org/10.1016/j.micpath.2013.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23711963/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mirani%20ZA,%20Aziz%20M,%20Khan%20MN,%20Lal%20I,%20Hassan%20N,%20ul,%20Khan%20SI.%20Biofilm%20formation%20and%20dispersal%20of%20Staphylococcus%20aureus%20under%20the%20influence%20of%20Oxacillin.%20Microb%20Pathog.%202013;61%E2%80%9362:66%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Rutherford ST, Bassler BL. Bacterial quorum sensing: its role in virulence and possibilities for its control. Cold Spring Harb Perspect Med. 2012;2: a012427.
</cite> [<a href="https://doi.org/10.1101/cshperspect.a012427" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3543102/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23125205/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rutherford%20ST,%20Bassler%20BL.%20Bacterial%20quorum%20sensing:%20its%20role%20in%20virulence%20and%20possibilities%20for%20its%20control.%20Cold%20Spring%20Harb%20Perspect%20Med.%202012;2:%20a012427." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Sittka A, Vogel J. A glimpse at the evolution of virulence control. Cell Host Microbe. 2008;4:310â€“2.
</cite> [<a href="https://doi.org/10.1016/j.chom.2008.09.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18854234/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sittka%20A,%20Vogel%20J.%20A%20glimpse%20at%20the%20evolution%20of%20virulence%20control.%20Cell%20Host%20Microbe.%202008;4:310%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Fey PD, Olson ME. Current concepts in biofilm formation of <em>Staphylococcus epidermidis</em>. Future Microbiol. 2010;5:917â€“33.
</cite> [<a href="https://doi.org/10.2217/fmb.10.56" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2903046/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20521936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fey%20PD,%20Olson%20ME.%20Current%20concepts%20in%20biofilm%20formation%20of%20Staphylococcus%20epidermidis.%20Future%20Microbiol.%202010;5:917%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Lang JC, Brutscher A, EhrstrÃ¶m M, Melican K. Tissue resident cells differentiate <em>S. aureus</em> from <em>S. epidermidis</em> via IL-1Î² following barrier disruption in healthy human skin. PLoS Pathog. 2024;20: e1012056.
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1012056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11389914/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39208402/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lang%20JC,%20Brutscher%20A,%20Ehrstr%C3%B6m%20M,%20Melican%20K.%20Tissue%20resident%20cells%20differentiate%20S.%20aureus%20from%20S.%20epidermidis%20via%20IL-1%CE%B2%20following%20barrier%20disruption%20in%20healthy%20human%20skin.%20PLoS%20Pathog.%202024;20:%20e1012056." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Taylor TA, Unakal CG. <em>Staphylococcus aureus</em> infection. StatPearls. Treasure Island. (FL): StatPearls Publishing; 2025.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/28722898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Taylor%20TA,%20Unakal%20CG.%20Staphylococcus%20aureus%20infection.%20StatPearls.%20Treasure%20Island.%20(FL):%20StatPearls%20Publishing;%202025." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Yamazaki Y, Ito T, Tamai M, Nakagawa S, Nakamura Y. The role of <em>Staphylococcus aureus</em> quorum sensing in cutaneous and systemic infections. Inflamm Regen. 2024;44: 9.
</cite> [<a href="https://doi.org/10.1186/s41232-024-00323-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10905890/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38429810/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yamazaki%20Y,%20Ito%20T,%20Tamai%20M,%20Nakagawa%20S,%20Nakamura%20Y.%20The%20role%20of%20Staphylococcus%20aureus%20quorum%20sensing%20in%20cutaneous%20and%20systemic%20infections.%20Inflamm%20Regen.%202024;44:%209." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Kamer AMA, Abdelaziz AA, Al-Monofy KB, Al-Madboly LA. Antibacterial, antibiofilm, and anti-quorum sensing activities of pyocyanin against methicillin-resistant <em>Staphylococcus aureus</em>: in vitro and in vivo study. BMC Microbiol. 2023;23:116.
</cite> [<a href="https://doi.org/10.1186/s12866-023-02861-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10124065/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37095436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kamer%20AMA,%20Abdelaziz%20AA,%20Al-Monofy%20KB,%20Al-Madboly%20LA.%20Antibacterial,%20antibiofilm,%20and%20anti-quorum%20sensing%20activities%20of%20pyocyanin%20against%20methicillin-resistant%20Staphylococcus%20aureus:%20in%20vitro%20and%20in%20vivo%20study.%20BMC%20Microbiol.%202023;23:116." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Rahmani AM, Gia TN, Negash B, Anzanpour A, Azimi I, Jiang M, et al. Exploiting smart e-Health gateways at the edge of healthcare Internet-of-Things: A fog computing approach. Future Gener Comput Syst. 2018;78:641â€“58.</cite> [<a href="https://scholar.google.com/scholar_lookup?Rahmani%20AM,%20Gia%20TN,%20Negash%20B,%20Anzanpour%20A,%20Azimi%20I,%20Jiang%20M,%20et%20al.%20Exploiting%20smart%20e-Health%20gateways%20at%20the%20edge%20of%20healthcare%20Internet-of-Things:%20A%20fog%20computing%20approach.%20Future%20Gener%20Comput%20Syst.%202018;78:641%E2%80%9358." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Koutsoukas A, Simms B, Kirchmair J, Bond PJ, Whitmore AV, Zimmer S, et al. From <em>in Silico</em> target prediction to multi-target drug design: current databases, methods and applications. J Proteom. 2011;74:2554â€“74.</cite> [<a href="https://doi.org/10.1016/j.jprot.2011.05.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21621023/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koutsoukas%20A,%20Simms%20B,%20Kirchmair%20J,%20Bond%20PJ,%20Whitmore%20AV,%20Zimmer%20S,%20et%20al.%20From%20in%20Silico%20target%20prediction%20to%20multi-target%20drug%20design:%20current%20databases,%20methods%20and%20applications.%20J%20Proteom.%202011;74:2554%E2%80%9374." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Kaur B, Gupta J, Sharma S, Sharma D, Sharma S. Focused review on dual inhibition of quorum sensing and efflux pumps: a potential way to combat multi drug resistant <em>Staphylococcus aureus</em> infections. Int J Biol Macromol. 2021;190:33â€“43.
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2021.08.199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34480904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaur%20B,%20Gupta%20J,%20Sharma%20S,%20Sharma%20D,%20Sharma%20S.%20Focused%20review%20on%20dual%20inhibition%20of%20quorum%20sensing%20and%20efflux%20pumps:%20a%20potential%20way%20to%20combat%20multi%20drug%20resistant%20Staphylococcus%20aureus%20infections.%20Int%20J%20Biol%20Macromol.%202021;190:33%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Dutta K, Karmakar A, Kumar A, et al. Benzyl isocyanate isolated from the leaves of <em>Psidium guajava</em> inhibits <em>Staphylococcus aureus</em> biofilm formation. Biofouling. 2020;36:1000â€“17.
</cite> [<a href="https://doi.org/10.1080/08927014.2020.1842877" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33172298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dutta%20K,%20Karmakar%20A,%20Kumar%20A,%20et%20al.%20Benzyl%20isocyanate%20isolated%20from%20the%20leaves%20of%20Psidium%20guajava%20inhibits%20Staphylococcus%20aureus%20biofilm%20formation.%20Biofouling.%202020;36:1000%E2%80%9317." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Manna T, Guchhait KC, Jana D, Dey S, Karmakar M, Hazra S, et al. Wastewater-based surveillance of <em>vibrio cholerae</em>: molecular insights on biofilm regulatory diguanylate cyclases, virulence factors and antibiotic resistance patterns. Microb Pathog. 2024;196: 106995.
</cite> [<a href="https://doi.org/10.1016/j.micpath.2024.106995" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39368563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manna%20T,%20Guchhait%20KC,%20Jana%20D,%20Dey%20S,%20Karmakar%20M,%20Hazra%20S,%20et%20al.%20Wastewater-based%20surveillance%20of%20vibrio%20cholerae:%20molecular%20insights%20on%20biofilm%20regulatory%20diguanylate%20cyclases,%20virulence%20factors%20and%20antibiotic%20resistance%20patterns.%20Microb%20Pathog.%202024;196:%20106995." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Alattar NS, Tawfeeq HK, Omran AH. Antibacterial and antibiofilm activity of Klebicin crude extract on clinical isolates of <em>salmonella</em>â€‰and <em>â€‰enterobacter</em>. World Acad Sci J. 2024;6:1â€“11.</cite> [<a href="https://scholar.google.com/scholar_lookup?Alattar%20NS,%20Tawfeeq%20HK,%20Omran%20AH.%20Antibacterial%20and%20antibiofilm%20activity%20of%20Klebicin%20crude%20extract%20on%20clinical%20isolates%20of%20salmonella%E2%80%89and%20%E2%80%89enterobacter.%20World%20Acad%20Sci%20J.%202024;6:1%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Manna T, Dey S, Karmakar M, Panda AK, Ghosh C. Investigations on genomic, topological and structural properties of diguanylate cyclases involved in <em>Vibrio cholerae</em> biofilm signalling using <em>in silico</em> techniques: promising drug targets in combating cholera. Curr Res Struct Biol. 2025;9: 100166.
</cite> [<a href="https://doi.org/10.1016/j.crstbi.2025.100166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12051071/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40330072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manna%20T,%20Dey%20S,%20Karmakar%20M,%20Panda%20AK,%20Ghosh%20C.%20Investigations%20on%20genomic,%20topological%20and%20structural%20properties%20of%20diguanylate%20cyclases%20involved%20in%20Vibrio%20cholerae%20biofilm%20signalling%20using%20in%20silico%20techniques:%20promising%20drug%20targets%20in%20combating%20cholera.%20Curr%20Res%20Struct%20Biol.%202025;9:%20100166." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Salam AM, Quave CL. Targeting virulence in <em>Staphylococcus aureus</em> by chemical inhibition of the accessory gene regulator system <em>in vivo</em>. mSphere. 2018. 10.1128/msphere.00500-17.
</cite> [<a href="https://doi.org/10.1128/mSphere.00500-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5770542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29359191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Salam%20AM,%20Quave%20CL.%20Targeting%20virulence%20in%20Staphylococcus%20aureus%20by%20chemical%20inhibition%20of%20the%20accessory%20gene%20regulator%20system%20in%20vivo.%20mSphere.%202018.%2010.1128/msphere.00500-17." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Apweiler R, Bairoch A, Wu CH, Barker WC, Boeckmann B, Ferro S, et al. UniProt: the universal protein knowledgebase. Nucleic Acids Res. 2004;32(Database issue):D115-119.
</cite> [<a href="https://doi.org/10.1093/nar/gkh131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC308865/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14681372/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Apweiler%20R,%20Bairoch%20A,%20Wu%20CH,%20Barker%20WC,%20Boeckmann%20B,%20Ferro%20S,%20et%20al.%20UniProt:%20the%20universal%20protein%20knowledgebase.%20Nucleic%20Acids%20Res.%202004;32(Database%20issue):D115-119." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Raue S, Fan S-H, Rosenstein R, Zabel S, Luqman A, Nieselt K et al. The genome of <em>Staphylococcus epidermidis</em> O47. Front Microbiol. 2020;11. 10.3389/fmicb.2020.02061</cite> [<a href="https://doi.org/10.3389/fmicb.2020.02061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7477909/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32983045/" class="usa-link">PubMed</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Holden MTG, Feil EJ, Lindsay JA, Peacock SJ, Day NPJ, Enright MC, et al. Complete genomes of two clinical <em>Staphylococcus aureus</em> strains: evidence for the rapid evolution of virulence and drug resistance. Proc Natl Acad Sci. 2004;101:9786â€“91.
</cite> [<a href="https://doi.org/10.1073/pnas.0402521101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC470752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15213324/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Holden%20MTG,%20Feil%20EJ,%20Lindsay%20JA,%20Peacock%20SJ,%20Day%20NPJ,%20Enright%20MC,%20et%20al.%20Complete%20genomes%20of%20two%20clinical%20Staphylococcus%20aureus%20strains:%20evidence%20for%20the%20rapid%20evolution%20of%20virulence%20and%20drug%20resistance.%20Proc%20Natl%20Acad%20Sci.%202004;101:9786%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Chen W, Xie T-T, Zeng H, Chen W, Xie T-T, Zeng H, Formation. Antibiotic Resistance, and Control Strategies ofâ€‰<em>Staphylococcus epidermidis</em>â€‰Biofilm. In: Bacterial Biofilms. IntechOpen; 2019. 10.5772/intechopen.89800</cite>
</li>
<li id="CR30">
<span class="label">30.</span><cite>Cao J, Zhang Huimin, He Zhien, Piao Zhongwan, Zong Xianchun, Sun B. Genotypic and phenotypic characterization of some <em>psms</em> hypervirulent clinical isolates of <em>Staphylococcus aureus</em> in a tertiary hospital in Hefei, Anhui. Infect Drug Resist. 2023;16:1471â€“84.
</cite> [<a href="https://doi.org/10.2147/IDR.S399688" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10025015/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36949844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cao%20J,%20Zhang%20Huimin,%20He%20Zhien,%20Piao%20Zhongwan,%20Zong%20Xianchun,%20Sun%20B.%20Genotypic%20and%20phenotypic%20characterization%20of%20some%20psms%20hypervirulent%20clinical%20isolates%20of%20Staphylococcus%20aureus%20in%20a%20tertiary%20hospital%20in%20Hefei,%20Anhui.%20Infect%20Drug%20Resist.%202023;16:1471%E2%80%9384." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Durr-e-Shahwar S, Atia-tul-Wahab, Choudhary MI, Jabeen A. Cloning, purification, structural, and functional characterization of methicillin-resistant <em>Staphylococcus aureus</em> (MRSA252) RsbV protein. Int J Biol Macromol. 2019;134:962â€“6.
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2019.05.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31075336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Durr-e-Shahwar%20S,%20Atia-tul-Wahab,%20Choudhary%20MI,%20Jabeen%20A.%20Cloning,%20purification,%20structural,%20and%20functional%20characterization%20of%20methicillin-resistant%20Staphylococcus%20aureus%20(MRSA252)%20RsbV%20protein.%20Int%20J%20Biol%20Macromol.%202019;134:962%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Herbert S, Ziebandt A-K, Ohlsen K, SchÃ¤fer T, Hecker M, Albrecht D, et al. Repair of global regulators in <em>Staphylococcus aureus</em> 8325 and comparative analysis with other clinical isolates. Infect Immun. 2010;78:2877â€“89.
</cite> [<a href="https://doi.org/10.1128/IAI.00088-10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2876537/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20212089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Herbert%20S,%20Ziebandt%20A-K,%20Ohlsen%20K,%20Sch%C3%A4fer%20T,%20Hecker%20M,%20Albrecht%20D,%20et%20al.%20Repair%20of%20global%20regulators%20in%20Staphylococcus%20aureus%208325%20and%20comparative%20analysis%20with%20other%20clinical%20isolates.%20Infect%20Immun.%202010;78:2877%E2%80%9389." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Reuter S, TÃ¶rÃ¶k ME, Holden MTG, Reynolds R, Raven KE, Blane B, et al. Building a genomic framework for prospective MRSA surveillance in the United Kingdom and the Republic of Ireland. Genome Res. 2016;26:263â€“70.
</cite> [<a href="https://doi.org/10.1101/gr.196709.115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4728378/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26672018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reuter%20S,%20T%C3%B6r%C3%B6k%20ME,%20Holden%20MTG,%20Reynolds%20R,%20Raven%20KE,%20Blane%20B,%20et%20al.%20Building%20a%20genomic%20framework%20for%20prospective%20MRSA%20surveillance%20in%20the%20United%20Kingdom%20and%20the%20Republic%20of%20Ireland.%20Genome%20Res.%202016;26:263%E2%80%9370." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Pozzi C, Waters EM, Rudkin JK, Schaeffer CR, Lohan AJ, Tong P, et al. Methicillin resistance alters the biofilm phenotype and attenuates virulence in <em>Staphylococcus aureus</em> device-associated infections. PLoS Pathog. 2012;8:e1002626.
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1002626" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3320603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22496652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pozzi%20C,%20Waters%20EM,%20Rudkin%20JK,%20Schaeffer%20CR,%20Lohan%20AJ,%20Tong%20P,%20et%20al.%20Methicillin%20resistance%20alters%20the%20biofilm%20phenotype%20and%20attenuates%20virulence%20in%20Staphylococcus%20aureus%20device-associated%20infections.%20PLoS%20Pathog.%202012;8:e1002626." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Dai L, Yang L, Parsons C, Findlay VJ, Molin S, Qin Z. <em>Staphylococcus epidermidis</em> recovered from indwelling catheters exhibit enhanced biofilm dispersal and self-renewal through downregulation of Agr. BMC Microbiol. 2012;12: 102.
</cite> [<a href="https://doi.org/10.1186/1471-2180-12-102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3458918/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22682058/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dai%20L,%20Yang%20L,%20Parsons%20C,%20Findlay%20VJ,%20Molin%20S,%20Qin%20Z.%20Staphylococcus%20epidermidis%20recovered%20from%20indwelling%20catheters%20exhibit%20enhanced%20biofilm%20dispersal%20and%20self-renewal%20through%20downregulation%20of%20Agr.%20BMC%20Microbiol.%202012;12:%20102." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Kumar S, Stecher G, Li M, Knyaz C, Tamura K. MEGA X: molecular evolutionary genetics analysis across computing platforms. Mol Biol Evol. 2018;35:1547â€“9.
</cite> [<a href="https://doi.org/10.1093/molbev/msy096" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5967553/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29722887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar%20S,%20Stecher%20G,%20Li%20M,%20Knyaz%20C,%20Tamura%20K.%20MEGA%20X:%20molecular%20evolutionary%20genetics%20analysis%20across%20computing%20platforms.%20Mol%20Biol%20Evol.%202018;35:1547%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Tamura K, Stecher G, Peterson D, Filipski A, Kumar S. MEGA6: molecular evolutionary genetics analysis version 6.0. Mol Biol Evol. 2013;30:2725â€“9.
</cite> [<a href="https://doi.org/10.1093/molbev/mst197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3840312/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24132122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tamura%20K,%20Stecher%20G,%20Peterson%20D,%20Filipski%20A,%20Kumar%20S.%20MEGA6:%20molecular%20evolutionary%20genetics%20analysis%20version%206.0.%20Mol%20Biol%20Evol.%202013;30:2725%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Waterhouse AM, Procter JB, Martin DMA, Clamp M, Barton GJ. Jalview version 2â€”a multiple sequence alignment editor and analysis workbench. Bioinformatics. 2009;25:1189â€“91.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2672624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19151095/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Waterhouse%20AM,%20Procter%20JB,%20Martin%20DMA,%20Clamp%20M,%20Barton%20GJ.%20Jalview%20version%202%E2%80%94a%20multiple%20sequence%20alignment%20editor%20and%20analysis%20workbench.%20Bioinformatics.%202009;25:1189%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Gasteiger E, Gattiker A, Hoogland C, Ivanyi I, Appel RD, Bairoch A, ExPASy. The proteomics server for in-depth protein knowledge and analysis. Nucleic Acids Res. 2003;31:3784â€“8.
</cite> [<a href="https://doi.org/10.1093/nar/gkg563" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC168970/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12824418/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gasteiger%20E,%20Gattiker%20A,%20Hoogland%20C,%20Ivanyi%20I,%20Appel%20RD,%20Bairoch%20A,%20ExPASy.%20The%20proteomics%20server%20for%20in-depth%20protein%20knowledge%20and%20analysis.%20Nucleic%20Acids%20Res.%202003;31:3784%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>PuigbÃ² P, Bravo IG, Garcia-Vallve S, CAIcal. A combined set of tools to assess codon usage adaptation. Biol Direct. 2008;3:38.
</cite> [<a href="https://doi.org/10.1186/1745-6150-3-38" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2553769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18796141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Puigb%C3%B2%20P,%20Bravo%20IG,%20Garcia-Vallve%20S,%20CAIcal.%20A%20combined%20set%20of%20tools%20to%20assess%20codon%20usage%20adaptation.%20Biol%20Direct.%202008;3:38." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Dey S, Guchhait KC, Manna T, Panda AK, Patra A, Mondal SK, et al. Evolutionary and compositional analysis of streptokinase including its interaction with plasminogen: an <em>in Silico</em> approach. Gene Rep. 2022;29:101689.</cite> [<a href="https://scholar.google.com/scholar_lookup?Dey%20S,%20Guchhait%20KC,%20Manna%20T,%20Panda%20AK,%20Patra%20A,%20Mondal%20SK,%20et%20al.%20Evolutionary%20and%20compositional%20analysis%20of%20streptokinase%20including%20its%20interaction%20with%20plasminogen:%20an%20in%20Silico%20approach.%20Gene%20Rep.%202022;29:101689." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Sharp PM, Li W-H. The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications. Nucleic Acids Res. 1987;15:1281â€“95.
</cite> [<a href="https://doi.org/10.1093/nar/15.3.1281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC340524/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3547335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sharp%20PM,%20Li%20W-H.%20The%20codon%20adaptation%20index-a%20measure%20of%20directional%20synonymous%20codon%20usage%20bias,%20and%20its%20potential%20applications.%20Nucleic%20Acids%20Res.%201987;15:1281%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Slabinski L, Jaroszewski L, Rychlewski L, Wilson IA, Lesley SA, Godzik A. Xtalpred: a web server for prediction of protein crystallizability. Bioinformatics. 2007;23:3403â€“5.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btm477" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17921170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Slabinski%20L,%20Jaroszewski%20L,%20Rychlewski%20L,%20Wilson%20IA,%20Lesley%20SA,%20Godzik%20A.%20Xtalpred:%20a%20web%20server%20for%20prediction%20of%20protein%20crystallizability.%20Bioinformatics.%202007;23:3403%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Yu C-S, Cheng C-W, Su W-C, Chang K-C, Huang S-W, Hwang J-K, et al. CELLO2GO: A web server for protein subcellular localization prediction with functional gene ontology annotation. PLoS One. 2014;9:e99368.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0099368" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4049835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24911789/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20C-S,%20Cheng%20C-W,%20Su%20W-C,%20Chang%20K-C,%20Huang%20S-W,%20Hwang%20J-K,%20et%20al.%20CELLO2GO:%20A%20web%20server%20for%20protein%20subcellular%20localization%20prediction%20with%20functional%20gene%20ontology%20annotation.%20PLoS%20One.%202014;9:e99368." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Bernsel A, Viklund H, Hennerdal A, Elofsson A. TOPCONS: consensus prediction of membrane protein topology. Nucleic Acids Res. 2009;37(Web Server issue):W465-8.
</cite> [<a href="https://doi.org/10.1093/nar/gkp363" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2703981/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19429891/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bernsel%20A,%20Viklund%20H,%20Hennerdal%20A,%20Elofsson%20A.%20TOPCONS:%20consensus%20prediction%20of%20membrane%20protein%20topology.%20Nucleic%20Acids%20Res.%202009;37(Web%20Server%20issue):W465-8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Dobson L, RemÃ©nyi I, TusnÃ¡dy GE. CCTOP: a consensus constrained topology prediction web server. Nucleic Acids Res. 2015;43:W408-12.
</cite> [<a href="https://doi.org/10.1093/nar/gkv451" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4489262/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25943549/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dobson%20L,%20Rem%C3%A9nyi%20I,%20Tusn%C3%A1dy%20GE.%20CCTOP:%20a%20consensus%20constrained%20topology%20prediction%20web%20server.%20Nucleic%20Acids%20Res.%202015;43:W408-12." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>KÃ¤ll L, Krogh A, Sonnhammer ELL. Advantages of combined transmembrane topology and signal peptide predictionâ€”the Phobius web server. Nucleic Acids Res. 2007;35(suppl2):W429-32.
</cite> [<a href="https://doi.org/10.1093/nar/gkm256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1933244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17483518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?K%C3%A4ll%20L,%20Krogh%20A,%20Sonnhammer%20ELL.%20Advantages%20of%20combined%20transmembrane%20topology%20and%20signal%20peptide%20prediction%E2%80%94the%20Phobius%20web%20server.%20Nucleic%20Acids%20Res.%202007;35(suppl2):W429-32." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Tamposis IA, Sarantopoulou D, Theodoropoulou MC, Stasi EA, Kontou PI, Tsirigos KD, et al. Hidden neural networks for transmembrane protein topology prediction. Comput Struct Biotechnol J. 2021;19:6090â€“7.
</cite> [<a href="https://doi.org/10.1016/j.csbj.2021.11.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8606341/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34849210/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tamposis%20IA,%20Sarantopoulou%20D,%20Theodoropoulou%20MC,%20Stasi%20EA,%20Kontou%20PI,%20Tsirigos%20KD,%20et%20al.%20Hidden%20neural%20networks%20for%20transmembrane%20protein%20topology%20prediction.%20Comput%20Struct%20Biotechnol%20J.%202021;19:6090%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Hallgren J, Tsirigos KD, Pedersen MD, Armenteros JJA, Marcatili P, Nielsen H et al. DeepTMHMM predicts alpha and beta transmembrane proteins using deep neural networks. 2022;:2022.04.08.487609. 10.1101/2022.04.08.487609</cite>
</li>
<li id="CR50">
<span class="label">50.</span><cite>Yaghoobizadeh F, Roayaei Ardakani M, Ranjbar MM, Khosravi M, Galehdari H. Development of a potent recombinant ScFv antibody against the SARS-CoV-2 by in-depth bioinformatics study: paving the way for vaccine/diagnostics development. Comput Biol Med. 2024;170: 108091.
</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2024.108091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38295473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yaghoobizadeh%20F,%20Roayaei%20Ardakani%20M,%20Ranjbar%20MM,%20Khosravi%20M,%20Galehdari%20H.%20Development%20of%20a%20potent%20recombinant%20ScFv%20antibody%20against%20the%20SARS-CoV-2%20by%20in-depth%20bioinformatics%20study:%20paving%20the%20way%20for%20vaccine/diagnostics%20development.%20Comput%20Biol%20Med.%202024;170:%20108091." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Edem UL. Evaluation of secondary and tertiary protein structure variation in Rbcl gene of selected legumes using computational approach. J Proteom Bioinform. 2023;16:1â€“7.</cite> [<a href="https://scholar.google.com/scholar_lookup?Edem%20UL.%20Evaluation%20of%20secondary%20and%20tertiary%20protein%20structure%20variation%20in%20Rbcl%20gene%20of%20selected%20legumes%20using%20computational%20approach.%20J%20Proteom%20Bioinform.%202023;16:1%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Geourjon C, DelÃ©age G. SOPMA: significant improvements in protein secondary structure prediction by consensus prediction from multiple alignments. Bioinformatics. 1995;11:681â€“4.</cite> [<a href="https://doi.org/10.1093/bioinformatics/11.6.681" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8808585/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Geourjon%20C,%20Del%C3%A9age%20G.%20SOPMA:%20significant%20improvements%20in%20protein%20secondary%20structure%20prediction%20by%20consensus%20prediction%20from%20multiple%20alignments.%20Bioinformatics.%201995;11:681%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Du Z, Su H, Wang W, Ye L, Wei H, Peng Z, et al. The trrosetta server for fast and accurate protein structure prediction. Nat Protoc. 2021;16:5634â€“51.
</cite> [<a href="https://doi.org/10.1038/s41596-021-00628-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34759384/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Du%20Z,%20Su%20H,%20Wang%20W,%20Ye%20L,%20Wei%20H,%20Peng%20Z,%20et%20al.%20The%20trrosetta%20server%20for%20fast%20and%20accurate%20protein%20structure%20prediction.%20Nat%20Protoc.%202021;16:5634%E2%80%9351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Jumper J, Evans R, Pritzel A, Green T, Figurnov M, Ronneberger O, et al. Highly accurate protein structure prediction with alphafold. Nature. 2021;596:583â€“9.
</cite> [<a href="https://doi.org/10.1038/s41586-021-03819-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8371605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34265844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jumper%20J,%20Evans%20R,%20Pritzel%20A,%20Green%20T,%20Figurnov%20M,%20Ronneberger%20O,%20et%20al.%20Highly%20accurate%20protein%20structure%20prediction%20with%20alphafold.%20Nature.%202021;596:583%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Ruff KM, Pappu RV. AlphaFold and implications for intrinsically disordered proteins. J Mol Biol. 2021;433: 167208.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2021.167208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34418423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ruff%20KM,%20Pappu%20RV.%20AlphaFold%20and%20implications%20for%20intrinsically%20disordered%20proteins.%20J%20Mol%20Biol.%202021;433:%20167208." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Laskowski RA, MacArthur MW, Thornton JM. Chapter 21.4. PROCHECK: validation of protein-structure coordinates. 10.1107/97809553602060000882</cite>
</li>
<li id="CR57">
<span class="label">57.</span><cite>Al-Khayyat MZS, Al-Dabbagh AGA. In silico prediction and docking of tertiary structure of luxi, an inducer synthase of <em>vibrio fischeri</em>. Rep Biochem Mol Biol. 2016;4:66â€“75.
</cite> [<a href="/articles/PMC4986264/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27536699/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Al-Khayyat%20MZS,%20Al-Dabbagh%20AGA.%20In%20silico%20prediction%20and%20docking%20of%20tertiary%20structure%20of%20luxi,%20an%20inducer%20synthase%20of%20vibrio%20fischeri.%20Rep%20Biochem%20Mol%20Biol.%202016;4:66%E2%80%9375." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Wiederstein M, Sippl MJ. ProSA-web: interactive web service for the recognition of errors in three-dimensional structures of proteins. Nucleic Acids Res. 2007;35(suppl2):W407-10.
</cite> [<a href="https://doi.org/10.1093/nar/gkm290" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1933241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17517781/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wiederstein%20M,%20Sippl%20MJ.%20ProSA-web:%20interactive%20web%20service%20for%20the%20recognition%20of%20errors%20in%20three-dimensional%20structures%20of%20proteins.%20Nucleic%20Acids%20Res.%202007;35(suppl2):W407-10." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Ferruz N, Schmidt S, HÃ¶cker B. ProteinTools: a toolkit to analyze protein structures. Nucleic Acids Res. 2021;49:W559-66.
</cite> [<a href="https://doi.org/10.1093/nar/gkab375" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8262690/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34019657/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ferruz%20N,%20Schmidt%20S,%20H%C3%B6cker%20B.%20ProteinTools:%20a%20toolkit%20to%20analyze%20protein%20structures.%20Nucleic%20Acids%20Res.%202021;49:W559-66." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Kuriata A, Gierut AM, Oleniecki T, Ciemny MP, Kolinski A, Kurcinski M, et al. CABS-flex 2.0: a web server for fast simulations of flexibility of protein structures. Nucleic Acids Res. 2018;46:W338â€“43.
</cite> [<a href="https://doi.org/10.1093/nar/gky356" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6031000/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29762700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuriata%20A,%20Gierut%20AM,%20Oleniecki%20T,%20Ciemny%20MP,%20Kolinski%20A,%20Kurcinski%20M,%20et%20al.%20CABS-flex%202.0:%20a%20web%20server%20for%20fast%20simulations%20of%20flexibility%20of%20protein%20structures.%20Nucleic%20Acids%20Res.%202018;46:W338%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Jones P, Binns D, Chang H-Y, Fraser M, Li W, McAnulla C, et al. Interproscan 5: genome-scale protein function classification. Bioinformatics. 2014;30:1236â€“40.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3998142/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24451626/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jones%20P,%20Binns%20D,%20Chang%20H-Y,%20Fraser%20M,%20Li%20W,%20McAnulla%20C,%20et%20al.%20Interproscan%205:%20genome-scale%20protein%20function%20classification.%20Bioinformatics.%202014;30:1236%E2%80%9340." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Quevillon E, Silventoinen V, Pillai S, Harte N, Mulder N, Apweiler R, et al. InterProScan: protein domains identifier. Nucleic Acids Res. 2005;33 suppl2:W116â€“20.
</cite> [<a href="https://doi.org/10.1093/nar/gki442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1160203/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15980438/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quevillon%20E,%20Silventoinen%20V,%20Pillai%20S,%20Harte%20N,%20Mulder%20N,%20Apweiler%20R,%20et%20al.%20InterProScan:%20protein%20domains%20identifier.%20Nucleic%20Acids%20Res.%202005;33%20suppl2:W116%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Bailey TL, Johnson J, Grant CE, Noble WS. The MEME suite. Nucleic Acids Res. 2015;43:W39â€“49.
</cite> [<a href="https://doi.org/10.1093/nar/gkv416" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4489269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25953851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bailey%20TL,%20Johnson%20J,%20Grant%20CE,%20Noble%20WS.%20The%20MEME%20suite.%20Nucleic%20Acids%20Res.%202015;43:W39%E2%80%9349." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Yan Y, Zhang D, Zhou P, Li B, Huang S-Y. HDOCK: a web server for proteinâ€“protein and proteinâ€“DNA/RNA docking based on a hybrid strategy. Nucleic Acids Res. 2017;45:W365-73.
</cite> [<a href="https://doi.org/10.1093/nar/gkx407" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5793843/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28521030/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan%20Y,%20Zhang%20D,%20Zhou%20P,%20Li%20B,%20Huang%20S-Y.%20HDOCK:%20a%20web%20server%20for%20protein%E2%80%93protein%20and%20protein%E2%80%93DNA/RNA%20docking%20based%20on%20a%20hybrid%20strategy.%20Nucleic%20Acids%20Res.%202017;45:W365-73." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Christoffer C, Bharadwaj V, Luu R, Kihara D. LZerD protein-protein docking webserver enhanced with de novo structure prediction. Front Mol Biosci. 2021;8: 724947.
</cite> [<a href="https://doi.org/10.3389/fmolb.2021.724947" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8403062/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34466411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Christoffer%20C,%20Bharadwaj%20V,%20Luu%20R,%20Kihara%20D.%20LZerD%20protein-protein%20docking%20webserver%20enhanced%20with%20de%20novo%20structure%20prediction.%20Front%20Mol%20Biosci.%202021;8:%20724947." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Li H, Huang E, Zhang Y, Huang S, Xiao Y. HDOCK update for modeling protein-RNA/DNA complex structures. Protein Sci Publ Protein Soc. 2022;31:e4441.</cite> [<a href="https://doi.org/10.1002/pro.4441" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9615301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36305764/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20H,%20Huang%20E,%20Zhang%20Y,%20Huang%20S,%20Xiao%20Y.%20HDOCK%20update%20for%20modeling%20protein-RNA/DNA%20complex%20structures.%20Protein%20Sci%20Publ%20Protein%20Soc.%202022;31:e4441." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Seeliger D, de Groot BL. Ligand docking and binding site analysis with PyMOL and autodock/vina. J Comput Aided Mol Des. 2010;24:417â€“22.
</cite> [<a href="https://doi.org/10.1007/s10822-010-9352-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2881210/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20401516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seeliger%20D,%20de%20Groot%20BL.%20Ligand%20docking%20and%20binding%20site%20analysis%20with%20PyMOL%20and%20autodock/vina.%20J%20Comput%20Aided%20Mol%20Des.%202010;24:417%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Laskowski RA, JabÅ‚oÅ„ska J, Pravda L, VaÅ™ekovÃ¡ RS, Thornton JM. PDBsum: structural summaries of PDB entries. Protein Sci. 2018;27:129â€“34.
</cite> [<a href="https://doi.org/10.1002/pro.3289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5734310/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28875543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laskowski%20RA,%20Jab%C5%82o%C5%84ska%20J,%20Pravda%20L,%20Va%C5%99ekov%C3%A1%20RS,%20Thornton%20JM.%20PDBsum:%20structural%20summaries%20of%20PDB%20entries.%20Protein%20Sci.%202018;27:129%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Tallei TE, Fatimawali, Yelnetty A, Idroes R, Kusumawaty D, Emran TB et al. An analysis based on molecular Docking and molecular dynamics simulation study of Bromelain as Anti-SARS-CoV-2 variants. Front Pharmacol. 2021;12. 10.3389/fphar.2021.717757</cite> [<a href="https://doi.org/10.3389/fphar.2021.717757" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8417730/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34489706/" class="usa-link">PubMed</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Hoque AF, Rahman MM, Lamia AS, Islam A, Klena JD, Satter SM, et al. <em>In silico</em> prediction of interaction between Nipah virus attachment glycoprotein and host cell receptors Ephrin-B2 and Ephrin-B3 in domestic and peridomestic mammals. Infect Genet Evol. 2023;116: 105516.
</cite> [<a href="https://doi.org/10.1016/j.meegid.2023.105516" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37924857/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoque%20AF,%20Rahman%20MM,%20Lamia%20AS,%20Islam%20A,%20Klena%20JD,%20Satter%20SM,%20et%20al.%20In%20silico%20prediction%20of%20interaction%20between%20Nipah%20virus%20attachment%20glycoprotein%20and%20host%20cell%20receptors%20Ephrin-B2%20and%20Ephrin-B3%20in%20domestic%20and%20peridomestic%20mammals.%20Infect%20Genet%20Evol.%202023;116:%20105516." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Arumugam S, Varamballi P. In-silico design of envelope based multi-epitope vaccine candidate against Kyasanur forest disease virus. Sci Rep. 2021;11:17118.
</cite> [<a href="https://doi.org/10.1038/s41598-021-94488-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8384868/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34429443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arumugam%20S,%20Varamballi%20P.%20In-silico%20design%20of%20envelope%20based%20multi-epitope%20vaccine%20candidate%20against%20Kyasanur%20forest%20disease%20virus.%20Sci%20Rep.%202021;11:17118." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Manandhar S, Sankhe R, Priya K, Hari G, Kumar BH, Mehta CH, et al. Molecular dynamics and structure-based virtual screening and identification of natural compounds as Wnt signaling modulators: possible therapeutics for alzheimerâ€™s disease. Mol Divers. 2022;26:2793â€“811.
</cite> [<a href="https://doi.org/10.1007/s11030-022-10395-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9532339/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35146638/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manandhar%20S,%20Sankhe%20R,%20Priya%20K,%20Hari%20G,%20Kumar%20BH,%20Mehta%20CH,%20et%20al.%20Molecular%20dynamics%20and%20structure-based%20virtual%20screening%20and%20identification%20of%20natural%20compounds%20as%20Wnt%20signaling%20modulators:%20possible%20therapeutics%20for%20alzheimer%E2%80%99s%20disease.%20Mol%20Divers.%202022;26:2793%E2%80%93811." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Wang X, Chen Y, Zhang S, Deng JN. Molecular dynamics simulations reveal the selectivity mechanism of structurally similar agonists to TLR7 and TLR8. PLoS One. 2022;17:e0260565.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0260565" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9032342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35452465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20X,%20Chen%20Y,%20Zhang%20S,%20Deng%20JN.%20Molecular%20dynamics%20simulations%20reveal%20the%20selectivity%20mechanism%20of%20structurally%20similar%20agonists%20to%20TLR7%20and%20TLR8.%20PLoS%20One.%202022;17:e0260565." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Tan L, Huang Y, Shang W, Yang Y, Peng H, Hu Z, et al. Accessory gene regulator (<em>agr</em>) allelic variants in cognate <em>Staphylococcus aureus</em> strain display similar phenotypes. Front Microbiol. 2022;13:700894.
</cite> [<a href="https://doi.org/10.3389/fmicb.2022.700894" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8919982/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35295312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tan%20L,%20Huang%20Y,%20Shang%20W,%20Yang%20Y,%20Peng%20H,%20Hu%20Z,%20et%20al.%20Accessory%20gene%20regulator%20(agr)%20allelic%20variants%20in%20cognate%20Staphylococcus%20aureus%20strain%20display%20similar%20phenotypes.%20Front%20Microbiol.%202022;13:700894." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Van Wamel WJB, van Rossum G, Verhoef J, Vandenbroucke-Grauls CMJE, Fluit AC. Cloning and characterization of an accessory gene regulator (<em>agr</em>)-like locus from <em>Staphylococcus epidermidis</em>. FEMS Microbiol Lett. 1998;163:1â€“9.
</cite> [<a href="https://doi.org/10.1111/j.1574-6968.1998.tb13018.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9631538/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Van%20Wamel%20WJB,%20van%20Rossum%20G,%20Verhoef%20J,%20Vandenbroucke-Grauls%20CMJE,%20Fluit%20AC.%20Cloning%20and%20characterization%20of%20an%20accessory%20gene%20regulator%20(agr)-like%20locus%20from%20Staphylococcus%20epidermidis.%20FEMS%20Microbiol%20Lett.%201998;163:1%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Chitra MA, Jayanthy C, Nagarajan B. Detection and sequence analysis of accessory gene regulator genes of <em>Staphylococcus pseudintermedius</em> isolates. Vet World. 2015;8:902â€“7.
</cite> [<a href="https://doi.org/10.14202/vetworld.2015.902-907" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4774685/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27047173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chitra%20MA,%20Jayanthy%20C,%20Nagarajan%20B.%20Detection%20and%20sequence%20analysis%20of%20accessory%20gene%20regulator%20genes%20of%20Staphylococcus%20pseudintermedius%20isolates.%20Vet%20World.%202015;8:902%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Gamage DG, Gunaratne A, Periyannan GR, Russell TG. Applicability of Instability Index for In vitro Protein Stability Prediction. <a href="http://www.eurekaselect.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.eurekaselect.com</a></cite> [<a href="https://doi.org/10.2174/0929866526666190228144219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30816075/" class="usa-link">PubMed</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Serra GM, Siqueira AS, de Molfetta FA, Santos AV, Xavier LP. In silico analysis of a GH3 Î²-glucosidase from <em>Microcystis aeruginosa</em> CACIAM 03. Microorganisms. 2023;11:998.
</cite> [<a href="https://doi.org/10.3390/microorganisms11040998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10146135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37110421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Serra%20GM,%20Siqueira%20AS,%20de%20Molfetta%20FA,%20Santos%20AV,%20Xavier%20LP.%20In%20silico%20analysis%20of%20a%20GH3%20%CE%B2-glucosidase%20from%20Microcystis%20aeruginosa%20CACIAM%2003.%20Microorganisms.%202023;11:998." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Novick RP, Ross HF, Figueiredo AMS, Abramochkin G, Muir T. Activation and inhibition of the Staphylococcal AGR system. Science. 2000;287:391â€“391.</cite> [<a href="https://scholar.google.com/scholar_lookup?Novick%20RP,%20Ross%20HF,%20Figueiredo%20AMS,%20Abramochkin%20G,%20Muir%20T.%20Activation%20and%20inhibition%20of%20the%20Staphylococcal%20AGR%20system.%20Science.%202000;287:391%E2%80%93391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Nakamoto M, Ohishi K, Kunimura K, Amano H, Wakamatsu J. Identification and determination of antibacterial constituents in residue discharged from garlic-processing plant. Eur Food Res Technol. 2020;246:1041â€“9.</cite> [<a href="https://scholar.google.com/scholar_lookup?Nakamoto%20M,%20Ohishi%20K,%20Kunimura%20K,%20Amano%20H,%20Wakamatsu%20J.%20Identification%20and%20determination%20of%20antibacterial%20constituents%20in%20residue%20discharged%20from%20garlic-processing%20plant.%20Eur%20Food%20Res%20Technol.%202020;246:1041%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Gouveia M, Schmidt C, Teixeira M, Lopes M, Aveiro SS, Domingues P, et al. Characterization of plasma SDS-protein aggregation profile of patients with heart failure with preserved ejection fraction. J Cardiovasc Transl Res. 2023;16:698â€“714.
</cite> [<a href="https://doi.org/10.1007/s12265-022-10334-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36271180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gouveia%20M,%20Schmidt%20C,%20Teixeira%20M,%20Lopes%20M,%20Aveiro%20SS,%20Domingues%20P,%20et%20al.%20Characterization%20of%20plasma%20SDS-protein%20aggregation%20profile%20of%20patients%20with%20heart%20failure%20with%20preserved%20ejection%20fraction.%20J%20Cardiovasc%20Transl%20Res.%202023;16:698%E2%80%93714." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Wang Z-K, Liu Y, Zheng H-Y, Tang M-Q, Xie S-Q. Comparative analysis of codon usage patterns in nuclear and Chloroplast genome of dalbergia (Fabaceae). Genes. 2023;14:1110.
</cite> [<a href="https://doi.org/10.3390/genes14051110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10217945/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37239470/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20Z-K,%20Liu%20Y,%20Zheng%20H-Y,%20Tang%20M-Q,%20Xie%20S-Q.%20Comparative%20analysis%20of%20codon%20usage%20patterns%20in%20nuclear%20and%20Chloroplast%20genome%20of%20dalbergia%20(Fabaceae).%20Genes.%202023;14:1110." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Liu X. A more accurate relationship between â€˜effective number of codonsâ€™ and GC3s under assumptions of no selection. Comput Biol Chem. 2013;42:35â€“9.
</cite> [<a href="https://doi.org/10.1016/j.compbiolchem.2012.11.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23257412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20X.%20A%20more%20accurate%20relationship%20between%20%E2%80%98effective%20number%20of%20codons%E2%80%99%20and%20GC3s%20under%20assumptions%20of%20no%20selection.%20Comput%20Biol%20Chem.%202013;42:35%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Quax TEF, Claassens NJ, SÃ¶ll D, van der Oost J. Codon bias as a means to Fine-Tune gene expression. Mol Cell. 2015;59:149â€“61.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2015.05.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4794256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26186290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quax%20TEF,%20Claassens%20NJ,%20S%C3%B6ll%20D,%20van%20der%20Oost%20J.%20Codon%20bias%20as%20a%20means%20to%20Fine-Tune%20gene%20expression.%20Mol%20Cell.%202015;59:149%E2%80%9361." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Liu Q, Yeo W-S, Bae T. The SaeRS Two-Component system of <em>Staphylococcus aureus</em>. Genes. 2016;7: 81.27706107
</cite> [<a href="https://scholar.google.com/scholar_lookup?Liu%20Q,%20Yeo%20W-S,%20Bae%20T.%20The%20SaeRS%20Two-Component%20system%20of%20Staphylococcus%20aureus.%20Genes.%202016;7:%2081.27706107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Jansen R, Bussemaker HJ, Gerstein M. Revisiting the codon adaptation index from a whole-genome perspective: analyzing the relationship between gene expression and codon occurrence in yeast using a variety of models. Nucleic Acids Res. 2003;31:2242â€“51.
</cite> [<a href="https://doi.org/10.1093/nar/gkg306" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC153734/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12682375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jansen%20R,%20Bussemaker%20HJ,%20Gerstein%20M.%20Revisiting%20the%20codon%20adaptation%20index%20from%20a%20whole-genome%20perspective:%20analyzing%20the%20relationship%20between%20gene%20expression%20and%20codon%20occurrence%20in%20yeast%20using%20a%20variety%20of%20models.%20Nucleic%20Acids%20Res.%202003;31:2242%E2%80%9351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Otto M. <em>Staphylococcus aureus</em> and <em>Staphylococcus epidermidis</em> peptide pheromones produced by the accessory gene regulator <em>Agr</em> system. Peptides. 2001;22:1603â€“8.
</cite> [<a href="https://doi.org/10.1016/s0196-9781(01)00495-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11587788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Otto%20M.%20Staphylococcus%20aureus%20and%20Staphylococcus%20epidermidis%20peptide%20pheromones%20produced%20by%20the%20accessory%20gene%20regulator%20Agr%20system.%20Peptides.%202001;22:1603%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Tegmark K, Morfeldt E, Arvidson S. Regulation of agr-dependent virulence genes in <em>Staphylococcus aureus</em> by RNAIII from coagulase-negative staphylococci. J Bacteriol. 1998;180:3181â€“6.
</cite> [<a href="https://doi.org/10.1128/jb.180.12.3181-3186.1998" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC107820/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9620969/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tegmark%20K,%20Morfeldt%20E,%20Arvidson%20S.%20Regulation%20of%20agr-dependent%20virulence%20genes%20in%20Staphylococcus%20aureus%20by%20RNAIII%20from%20coagulase-negative%20staphylococci.%20J%20Bacteriol.%201998;180:3181%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Yarwood JM, McCormick JK, Paustian ML, Kapur V, Schlievert PM. Repression of the <em>Staphylococcus aureus</em> accessory gene regulator in serum and in vivo. J Bacteriol. 2002;184:1095â€“101.
</cite> [<a href="https://doi.org/10.1128/jb.184.4.1095-1101.2002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC134826/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11807070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yarwood%20JM,%20McCormick%20JK,%20Paustian%20ML,%20Kapur%20V,%20Schlievert%20PM.%20Repression%20of%20the%20Staphylococcus%20aureus%20accessory%20gene%20regulator%20in%20serum%20and%20in%20vivo.%20J%20Bacteriol.%202002;184:1095%E2%80%93101." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Thoendel M, Horswill AR. Identification of <em>Staphylococcus aureus</em> AgrD residues required for autoinducing peptide biosynthesis. J Biol Chem. 2009;284:21828â€“38.
</cite> [<a href="https://doi.org/10.1074/jbc.M109.031757" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2756194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19520867/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thoendel%20M,%20Horswill%20AR.%20Identification%20of%20Staphylococcus%20aureus%20AgrD%20residues%20required%20for%20autoinducing%20peptide%20biosynthesis.%20J%20Biol%20Chem.%202009;284:21828%E2%80%9338." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Srivastava SK, Rajasree K, Fasim A, Arakere G, Gopal B. Influence of the AgrC-AgrA complex on the response time of <em>Staphylococcus aureus</em> quorum sensing. J Bacteriol. 2014;196:2876â€“88.
</cite> [<a href="https://doi.org/10.1128/JB.01530-14" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4135676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24858185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Srivastava%20SK,%20Rajasree%20K,%20Fasim%20A,%20Arakere%20G,%20Gopal%20B.%20Influence%20of%20the%20AgrC-AgrA%20complex%20on%20the%20response%20time%20of%20Staphylococcus%20aureus%20quorum%20sensing.%20J%20Bacteriol.%202014;196:2876%E2%80%9388." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Zhang L, Ji G. Identification of a Staphylococcal AgrB segment(s) responsible for Group-Specific processing of AgrD by gene swapping. J Bacteriol. 2004;186:6706â€“13.
</cite> [<a href="https://doi.org/10.1128/JB.186.20.6706-6713.2004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC522183/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15466021/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20L,%20Ji%20G.%20Identification%20of%20a%20Staphylococcal%20AgrB%20segment(s)%20responsible%20for%20Group-Specific%20processing%20of%20AgrD%20by%20gene%20swapping.%20J%20Bacteriol.%202004;186:6706%E2%80%9313." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Lu C, Jiang J, Chen Q, Liu H, Ju X, Wang H. Analysis and prediction of interactions between transmembrane and non-transmembrane proteins. BMC Genomics. 2024;25:401.
</cite> [<a href="https://doi.org/10.1186/s12864-024-10251-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11040819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38658824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu%20C,%20Jiang%20J,%20Chen%20Q,%20Liu%20H,%20Ju%20X,%20Wang%20H.%20Analysis%20and%20prediction%20of%20interactions%20between%20transmembrane%20and%20non-transmembrane%20proteins.%20BMC%20Genomics.%202024;25:401." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Bakheet TM, Doig AJ. Properties and identification of antibiotic drug targets. BMC Bioinformatics. 2010;11: 195.
</cite> [<a href="https://doi.org/10.1186/1471-2105-11-195" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2873537/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20406434/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bakheet%20TM,%20Doig%20AJ.%20Properties%20and%20identification%20of%20antibiotic%20drug%20targets.%20BMC%20Bioinformatics.%202010;11:%20195." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Sidote DJ, Barbieri CM, Wu T, Stock AM. Structure of the <em>Staphylococcus aureus</em> AgrA LytTR domain bound to DNA reveals a beta fold with an unusual mode of binding. Structure. 2008;16:727â€“35.
</cite> [<a href="https://doi.org/10.1016/j.str.2008.02.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2430735/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18462677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sidote%20DJ,%20Barbieri%20CM,%20Wu%20T,%20Stock%20AM.%20Structure%20of%20the%20Staphylococcus%20aureus%20AgrA%20LytTR%20domain%20bound%20to%20DNA%20reveals%20a%20beta%20fold%20with%20an%20unusual%20mode%20of%20binding.%20Structure.%202008;16:727%E2%80%9335." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Mahapa A, Mandal S, Sinha D, Sau S, Sau K. Determining the roles of a conserved Î±-helix in a global virulence regulator from <em>Staphylococcus aureus</em>. Protein J. 2018;37:103â€“12.
</cite> [<a href="https://doi.org/10.1007/s10930-018-9762-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29464485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mahapa%20A,%20Mandal%20S,%20Sinha%20D,%20Sau%20S,%20Sau%20K.%20Determining%20the%20roles%20of%20a%20conserved%20%CE%B1-helix%20in%20a%20global%20virulence%20regulator%20from%20Staphylococcus%20aureus.%20Protein%20J.%202018;37:103%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR97">
<span class="label">97.</span><cite>Liao Y-Y, Li J-L, Pan R-L, Chiou T-J. Structureâ€“Function analysis reveals amino acid residues of Arabidopsis phosphate transporter AtPHT1;1 crucial for its activity. Front Plant Sci. 2019;10.</cite> [<a href="https://doi.org/10.3389/fpls.2019.01158" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6761603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31608095/" class="usa-link">PubMed</a>]</li>
<li id="CR98">
<span class="label">98.</span><cite>Shen F, Sergi C, Biochemistry. In: StatPearls, editor. Amino acid synthesis and degradation. Treasure Island (FL): StatPearls Publishing; 2025.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/32644676/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen%20F,%20Sergi%20C,%20Biochemistry.%20In:%20StatPearls,%20editor.%20Amino%20acid%20synthesis%20and%20degradation.%20Treasure%20Island%20(FL):%20StatPearls%20Publishing;%202025." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR99">
<span class="label">99.</span><cite>Makhlin J, Kofman T, Borovok I, Kohler C, Engelmann S, Cohen G, et al. <em>Staphylococcus aureus</em> ArcR controls expression of the arginine deiminase operon. J Bacteriol. 2007;189:5976â€“86.
</cite> [<a href="https://doi.org/10.1128/JB.00592-07" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1952046/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17557828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Makhlin%20J,%20Kofman%20T,%20Borovok%20I,%20Kohler%20C,%20Engelmann%20S,%20Cohen%20G,%20et%20al.%20Staphylococcus%20aureus%20ArcR%20controls%20expression%20of%20the%20arginine%20deiminase%20operon.%20J%20Bacteriol.%202007;189:5976%E2%80%9386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR100">
<span class="label">100.</span><cite>Ding J, Arnold E. Survey of programs for crystal structure determination and analysis of macromolecules. urn:isbn: 2006;F:685â€“94. 10.1107/97809553602060000723</cite>
</li>
<li id="CR101">
<span class="label">101.</span><cite>Dym O, Eisenberg D, Yeates TO. Detection of errors in protein models. Urn:isbn. 2012;F:677â€“83.</cite> [<a href="https://scholar.google.com/scholar_lookup?Dym%20O,%20Eisenberg%20D,%20Yeates%20TO.%20Detection%20of%20errors%20in%20protein%20models.%20Urn:isbn.%202012;F:677%E2%80%9383." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR102">
<span class="label">102.</span><cite>Ramachandran Plot - an overview | ScienceDirect Topics. <a href="https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/ramachandran-plot" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.sciencedirect.com/topics/biochemistry-genetics-and-molecular-biology/ramachandran-plot</a>. Accessed 4 Jun 2025.</cite>
</li>
<li id="CR103">
<span class="label">103.</span><cite>Rajasree K, Fasim A, Gopal B. Conformational features of the <em>Staphylococcus aureus</em> AgrA-promoter interactions rationalize quorum-sensing triggered gene expression. Biochem Biophys Rep. 2016;6:124â€“34.
</cite> [<a href="https://doi.org/10.1016/j.bbrep.2016.03.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5600425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28955870/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rajasree%20K,%20Fasim%20A,%20Gopal%20B.%20Conformational%20features%20of%20the%20Staphylococcus%20aureus%20AgrA-promoter%20interactions%20rationalize%20quorum-sensing%20triggered%20gene%20expression.%20Biochem%20Biophys%20Rep.%202016;6:124%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR104">
<span class="label">104.</span><cite>Jelesarov I, Karshikoff A. Defining the role of salt bridges in protein stability. In: Shriver JW, editor. Protein structure, stability, and interactions. Totowa, NJ: Humana; 2009. pp. 227â€“60.</cite> [<a href="https://doi.org/10.1007/978-1-59745-367-7_10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19157086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jelesarov%20I,%20Karshikoff%20A.%20Defining%20the%20role%20of%20salt%20bridges%20in%20protein%20stability.%20In:%20Shriver%20JW,%20editor.%20Protein%20structure,%20stability,%20and%20interactions.%20Totowa,%20NJ:%20Humana;%202009.%20pp.%20227%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR105">
<span class="label">105.</span><cite>Duart G, Lamb J, Ortiz-Mateu J, Elofsson A, Mingarro I. Intra-helical salt bridge contribution to membrane protein insertion. J Mol Biol. 2022;434: 167467.
</cite> [<a href="https://doi.org/10.1016/j.jmb.2022.167467" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35093395/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duart%20G,%20Lamb%20J,%20Ortiz-Mateu%20J,%20Elofsson%20A,%20Mingarro%20I.%20Intra-helical%20salt%20bridge%20contribution%20to%20membrane%20protein%20insertion.%20J%20Mol%20Biol.%202022;434:%20167467." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR106">
<span class="label">106.</span><cite>Ahmad I, Singh AK, Katari SK. <em>In Silico</em> insight into structural and functional attributes of arsenic resistance proteins from <em>Rhizobium radiobacter</em> strain F4. Journal of Hazardous Materials Advances. 2023;12: 100329.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ahmad%20I,%20Singh%20AK,%20Katari%20SK.%20In%20Silico%20insight%20into%20structural%20and%20functional%20attributes%20of%20arsenic%20resistance%20proteins%20from%20Rhizobium%20radiobacter%20strain%20F4.%20Journal%20of%20Hazardous%20Materials%20Advances.%202023;12:%20100329." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR107">
<span class="label">107.</span><cite>Badaczewska-Dawid A, WrÃ³blewski K, Kurcinski M, Kmiecik S. Structure prediction of linear and cyclic peptides using CABS-flex. Brief Bioinform. 2024;25: bbae003.
</cite> [<a href="https://doi.org/10.1093/bib/bbae003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10836054/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38305457/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Badaczewska-Dawid%20A,%20Wr%C3%B3blewski%20K,%20Kurcinski%20M,%20Kmiecik%20S.%20Structure%20prediction%20of%20linear%20and%20cyclic%20peptides%20using%20CABS-flex.%20Brief%20Bioinform.%202024;25:%20bbae003." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR108">
<span class="label">108.</span><cite>MartÃ­nez L. Automatic identification of mobile and rigid substructures in molecular dynamics simulations and fractional structural fluctuation analysis. PLoS One. 2015;10:e0119264.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0119264" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4376797/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25816325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mart%C3%ADnez%20L.%20Automatic%20identification%20of%20mobile%20and%20rigid%20substructures%20in%20molecular%20dynamics%20simulations%20and%20fractional%20structural%20fluctuation%20analysis.%20PLoS%20One.%202015;10:e0119264." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR109">
<span class="label">109.</span><cite>Blevins JS, Beenken KE, Elasri MO, Hurlburt BK, Smeltzer MS. Strain-dependent differences in the regulatory roles of SarA and Agr in <em>Staphylococcus aureus</em>. Infect Immun. 2002;70:470â€“80.
</cite> [<a href="https://doi.org/10.1128/IAI.70.2.470-480.2002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC127691/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11796572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blevins%20JS,%20Beenken%20KE,%20Elasri%20MO,%20Hurlburt%20BK,%20Smeltzer%20MS.%20Strain-dependent%20differences%20in%20the%20regulatory%20roles%20of%20SarA%20and%20Agr%20in%20Staphylococcus%20aureus.%20Infect%20Immun.%202002;70:470%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR110">
<span class="label">110.</span><cite>Cataneli Pereira V, Pinheiro-Hubinger L, de Oliveira A, Moraes Riboli DF, Benini Martins K, Calixto Romero L, et al. Detection of the Agr system and resistance to antimicrobials in biofilm-producing <em>S. epidermidis</em>. Molecules. 2020;25: 5715.
</cite> [<a href="https://doi.org/10.3390/molecules25235715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7729762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33287389/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cataneli%20Pereira%20V,%20Pinheiro-Hubinger%20L,%20de%20Oliveira%20A,%20Moraes%20Riboli%20DF,%20Benini%20Martins%20K,%20Calixto%20Romero%20L,%20et%20al.%20Detection%20of%20the%20Agr%20system%20and%20resistance%20to%20antimicrobials%20in%20biofilm-producing%20S.%20epidermidis.%20Molecules.%202020;25:%205715." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR111">
<span class="label">111.</span><cite>Tan L, Li SR, Jiang B, Hu XM, Li S. Therapeutic targeting of the <em>Staphylococcus aureus</em> accessory gene regulator (agr) system. Front Microbiol. 2018;9. 10.3389/fmicb.2018.00055</cite> [<a href="https://doi.org/10.3389/fmicb.2018.00055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5789755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29422887/" class="usa-link">PubMed</a>]</li>
<li id="CR112">
<span class="label">112.</span><cite>Horswill AR, Gordon CP. Structureâ€“activity relationship studies of small molecule modulators of the Staphylococcal accessory gene regulator. J Med Chem. 2020;63:2705â€“30.
</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.9b00798" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31658413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Horswill%20AR,%20Gordon%20CP.%20Structure%E2%80%93activity%20relationship%20studies%20of%20small%20molecule%20modulators%20of%20the%20Staphylococcal%20accessory%20gene%20regulator.%20J%20Med%20Chem.%202020;63:2705%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR113">
<span class="label">113.</span><cite>Awuor SO, Omwenga EO, Mariita RM, Daud II. Cholera outbreak: antibiofilm activity, profiling of antibiotic-resistant genes and virulence factors of toxigenic vibrio cholerae isolates reveals concerning traits. Access Microbiol. 2022;4:000324.
</cite> [<a href="https://doi.org/10.1099/acmi.0.000324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9175979/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35693465/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Awuor%20SO,%20Omwenga%20EO,%20Mariita%20RM,%20Daud%20II.%20Cholera%20outbreak:%20antibiofilm%20activity,%20profiling%20of%20antibiotic-resistant%20genes%20and%20virulence%20factors%20of%20toxigenic%20vibrio%20cholerae%20isolates%20reveals%20concerning%20traits.%20Access%20Microbiol.%202022;4:000324." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR114">
<span class="label">114.</span><cite>Dutta K, Karmakar A, Jana D, Ballav S, Shityakov S, Panda AK, et al. Benzyl isocyanate isolated from the leaves of <em>Psidium guajava</em> inhibits <em>Staphylococcus aureus</em> biofilm formation. Biofouling. 2020;36:1000â€“17.
</cite> [<a href="https://doi.org/10.1080/08927014.2020.1842877" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33172298/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dutta%20K,%20Karmakar%20A,%20Jana%20D,%20Ballav%20S,%20Shityakov%20S,%20Panda%20AK,%20et%20al.%20Benzyl%20isocyanate%20isolated%20from%20the%20leaves%20of%20Psidium%20guajava%20inhibits%20Staphylococcus%20aureus%20biofilm%20formation.%20Biofouling.%202020;36:1000%E2%80%9317." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR115">
<span class="label">115.</span><cite>Khatoon Z, McTiernan CD, Suuronen EJ, Mah T-F, Alarcon EI. Bacterial biofilm formation on implantable devices and approaches to its treatment and prevention. Heliyon. 2018;4:e01067.
</cite> [<a href="https://doi.org/10.1016/j.heliyon.2018.e01067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6312881/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30619958/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khatoon%20Z,%20McTiernan%20CD,%20Suuronen%20EJ,%20Mah%20T-F,%20Alarcon%20EI.%20Bacterial%20biofilm%20formation%20on%20implantable%20devices%20and%20approaches%20to%20its%20treatment%20and%20prevention.%20Heliyon.%202018;4:e01067." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR116">
<span class="label">116.</span><cite>Arciola CR, Campoccia D, Ravaioli S, Montanaro L. Polysaccharide intercellular adhesin in biofilm: structural and regulatory aspects. Front Cell Infect Microbiol. 2015;5. 10.3389/fcimb.2015.00007</cite> [<a href="https://doi.org/10.3389/fcimb.2015.00007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4322838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25713785/" class="usa-link">PubMed</a>]</li>
<li id="CR117">
<span class="label">117.</span><cite>Scoffone VC, Trespidi G, Chiarelli LR, Barbieri G, Buroni S. Quorum sensing as antivirulence target in cystic fibrosis pathogens. Int J Mol Sci. 2019;20:1838. 10.3390/ijms20081838
</cite> [<a href="https://doi.org/10.3390/ijms20081838" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6515091/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31013936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scoffone%20VC,%20Trespidi%20G,%20Chiarelli%20LR,%20Barbieri%20G,%20Buroni%20S.%20Quorum%20sensing%20as%20antivirulence%20target%20in%20cystic%20fibrosis%20pathogens.%20Int%20J%20Mol%20Sci.%202019;20:1838.%2010.3390/ijms20081838" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR118">
<span class="label">118.</span><cite>Sully EK, Malachowa N, Elmore BO, Alexander SM, Femling JK, Gray BM, et al. Selective chemical inhibition of Agr quorum sensing in <em>Staphylococcus aureus</em> promotes host defense with minimal impact on resistance. PLoS Pathog. 2014;10:e1004174.
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1004174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4055767/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24945495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sully%20EK,%20Malachowa%20N,%20Elmore%20BO,%20Alexander%20SM,%20Femling%20JK,%20Gray%20BM,%20et%20al.%20Selective%20chemical%20inhibition%20of%20Agr%20quorum%20sensing%20in%20Staphylococcus%20aureus%20promotes%20host%20defense%20with%20minimal%20impact%20on%20resistance.%20PLoS%20Pathog.%202014;10:e1004174." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR119">
<span class="label">119.</span><cite>Shrestha L, Fan H-M, Tao H-R, Huang J-D. Recent strategies to combat biofilms using antimicrobial agents and therapeutic approaches. Pathogens. 2022;11:292.
</cite> [<a href="https://doi.org/10.3390/pathogens11030292" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8955104/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35335616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shrestha%20L,%20Fan%20H-M,%20Tao%20H-R,%20Huang%20J-D.%20Recent%20strategies%20to%20combat%20biofilms%20using%20antimicrobial%20agents%20and%20therapeutic%20approaches.%20Pathogens.%202022;11:292." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR120">
<span class="label">120.</span><cite>Wu X, Wang H, Xiong J, Yang G-X, Hu J-F, Zhu Q, et al. <em>Staphylococcus aureus</em> biofilm: formulation, regulatory, and emerging natural products-derived therapeutics. Biofilm. 2024;7:100175.
</cite> [<a href="https://doi.org/10.1016/j.bioflm.2023.100175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10827693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38298832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20X,%20Wang%20H,%20Xiong%20J,%20Yang%20G-X,%20Hu%20J-F,%20Zhu%20Q,%20et%20al.%20Staphylococcus%20aureus%20biofilm:%20formulation,%20regulatory,%20and%20emerging%20natural%20products-derived%20therapeutics.%20Biofilm.%202024;7:100175." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR121">
<span class="label">121.</span><cite>Chatterjee A, Walters R, Shafi Z, Ahmed OS, Sebek M, Gysi D, et al. Improving the generalizability of protein-ligand binding predictions with AI-Bind. Nat Commun. 2023;14:1989.
</cite> [<a href="https://doi.org/10.1038/s41467-023-37572-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10082765/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37031187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chatterjee%20A,%20Walters%20R,%20Shafi%20Z,%20Ahmed%20OS,%20Sebek%20M,%20Gysi%20D,%20et%20al.%20Improving%20the%20generalizability%20of%20protein-ligand%20binding%20predictions%20with%20AI-Bind.%20Nat%20Commun.%202023;14:1989." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR122">
<span class="label">122.</span><cite>Pomputius WF, Kilgore SH, Schlievert PM. Probable enterotoxin-associated toxic shock syndrome caused by <em>Staphylococcus epidermidis</em>. BMC Pediatr. 2023;23: 108.
</cite> [<a href="https://doi.org/10.1186/s12887-023-03914-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9989563/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36882717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pomputius%20WF,%20Kilgore%20SH,%20Schlievert%20PM.%20Probable%20enterotoxin-associated%20toxic%20shock%20syndrome%20caused%20by%20Staphylococcus%20epidermidis.%20BMC%20Pediatr.%202023;23:%20108." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR123">
<span class="label">123.</span><cite>Maishanu IM, Gbonjubola AO, Mujahid H, Olayinka BO. Molecular detection of phenol-soluble modulin-mec (PSM-mec) in <em>Staphylococcus aureus</em> clinical isolates from federal medical center Birnin kebbi, North-West, Nigeria. GMS Hyg Infect Control. 2025;20: Doc09.
</cite> [<a href="https://doi.org/10.3205/dgkh000538" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12059804/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40352657/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maishanu%20IM,%20Gbonjubola%20AO,%20Mujahid%20H,%20Olayinka%20BO.%20Molecular%20detection%20of%20phenol-soluble%20modulin-mec%20(PSM-mec)%20in%20Staphylococcus%20aureus%20clinical%20isolates%20from%20federal%20medical%20center%20Birnin%20kebbi,%20North-West,%20Nigeria.%20GMS%20Hyg%20Infect%20Control.%202025;20:%20Doc09." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR124">
<span class="label">124.</span><cite>Chen C, Gao Y, Qiao X, Feng Y, Yu X, Cai J, et al. Functional amyloid phenol-soluble modulin Î±1-targeting photothermal nanoplatform for effective elimination of biofilm-associated infections. ACS Nano. 2025. 10.1021/acsnano.5c01761.
</cite> [<a href="https://doi.org/10.1021/acsnano.5c01761" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40407368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20C,%20Gao%20Y,%20Qiao%20X,%20Feng%20Y,%20Yu%20X,%20Cai%20J,%20et%20al.%20Functional%20amyloid%20phenol-soluble%20modulin%20%CE%B11-targeting%20photothermal%20nanoplatform%20for%20effective%20elimination%20of%20biofilm-associated%20infections.%20ACS%20Nano.%202025.%2010.1021/acsnano.5c01761." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR125">
<span class="label">125.</span><cite>Bleul L, Francois P, Wolz C. Two-component systems of <em>S. aureus</em>: signaling and sensing mechanisms. Genes. 2021;13: 34.
</cite> [<a href="https://doi.org/10.3390/genes13010034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8774646/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35052374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bleul%20L,%20Francois%20P,%20Wolz%20C.%20Two-component%20systems%20of%20S.%20aureus:%20signaling%20and%20sensing%20mechanisms.%20Genes.%202021;13:%2034." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR126">
<span class="label">126.</span><cite>Negi A, Kuo C-W, Hazam PK, Yeh J-C, Lin W-C, Lou Y-C, et al. Disruption of MRSA biofilm and virulence by Deep-Sea probiotics: impacts on energy metabolism and host antimicrobial peptides. Probiotics Antimicrob Proteins. 2025. 10.1007/s12602-025-10535-0.
</cite> [<a href="https://doi.org/10.1007/s12602-025-10535-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40229459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Negi%20A,%20Kuo%20C-W,%20Hazam%20PK,%20Yeh%20J-C,%20Lin%20W-C,%20Lou%20Y-C,%20et%20al.%20Disruption%20of%20MRSA%20biofilm%20and%20virulence%20by%20Deep-Sea%20probiotics:%20impacts%20on%20energy%20metabolism%20and%20host%20antimicrobial%20peptides.%20Probiotics%20Antimicrob%20Proteins.%202025.%2010.1007/s12602-025-10535-0." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR127">
<span class="label">127.</span><cite>De A, Sarkar A, Mukherjee S, Das A, Banerjee T, Paul P et al. The Potential of Phytomolecules in Countering Biofilm Formation and Quorum Sensing. 2025:e2210299X355078. 10.2174/012210299X355078241230061234</cite>
</li>
<li id="CR128">
<span class="label">128.</span><cite>Yu H, Wang L, Liu X, Zheng J, Xiang H, Zheng Y, et al. Mechanistic insights into the multitarget synergistic efficacy of Farrerol and Î²-lactam antibiotics in combating methicillin-resistant <em>Staphylococcus aureus</em>. Antimicrob Agents Chemother. 2025;69:e01551â€“24.
</cite> [<a href="https://doi.org/10.1128/aac.01551-24" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11963547/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40019240/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20H,%20Wang%20L,%20Liu%20X,%20Zheng%20J,%20Xiang%20H,%20Zheng%20Y,%20et%20al.%20Mechanistic%20insights%20into%20the%20multitarget%20synergistic%20efficacy%20of%20Farrerol%20and%20%CE%B2-lactam%20antibiotics%20in%20combating%20methicillin-resistant%20Staphylococcus%20aureus.%20Antimicrob%20Agents%20Chemother.%202025;69:e01551%E2%80%9324." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR129">
<span class="label">129.</span><cite>BackenkÃ¶hler M, GroÃŸ J, Wolf V, Volkamer A. Guided docking as a data generation approach facilitates structure-based machine learning on kinases. J Chem Inf Model. 2024;64:4009â€“20.
</cite> [<a href="https://doi.org/10.1021/acs.jcim.4c00055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38751014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Backenk%C3%B6hler%20M,%20Gro%C3%9F%20J,%20Wolf%20V,%20Volkamer%20A.%20Guided%20docking%20as%20a%20data%20generation%20approach%20facilitates%20structure-based%20machine%20learning%20on%20kinases.%20J%20Chem%20Inf%20Model.%202024;64:4009%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR130">
<span class="label">130.</span><cite>Rangisetty PT, Kilaparthi A, Akula S, Bhardwaj M, Singh S. RSAD2: an exclusive target protein for Zika virus comparative modeling, characterization, energy minimization and stabilization. Int J Health Sci. 2023;17:12â€“7.</cite> [<a href="/articles/PMC9832909/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36704497/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rangisetty%20PT,%20Kilaparthi%20A,%20Akula%20S,%20Bhardwaj%20M,%20Singh%20S.%20RSAD2:%20an%20exclusive%20target%20protein%20for%20Zika%20virus%20comparative%20modeling,%20characterization,%20energy%20minimization%20and%20stabilization.%20Int%20J%20Health%20Sci.%202023;17:12%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR131">
<span class="label">131.</span><cite>Singh N, Kumar N, Rathee G, Sood D, Singh A, Tomar V, et al. Privileged scaffold chalcone: synthesis, characterization and its mechanistic interaction studies with BSA employing spectroscopic and chemoinformatics approaches. ACS Omega. 2020;5:2267â€“79.
</cite> [<a href="https://doi.org/10.1021/acsomega.9b03479" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7016911/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32064388/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singh%20N,%20Kumar%20N,%20Rathee%20G,%20Sood%20D,%20Singh%20A,%20Tomar%20V,%20et%20al.%20Privileged%20scaffold%20chalcone:%20synthesis,%20characterization%20and%20its%20mechanistic%20interaction%20studies%20with%20BSA%20employing%20spectroscopic%20and%20chemoinformatics%20approaches.%20ACS%20Omega.%202020;5:2267%E2%80%9379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12374375/bin/12866_2025_4257_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1.</a><sup> (3.1MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data is provided within the supplementary information files.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Microbiology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12866-025-04257-0"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12866_2025_Article_4257.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.6Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12374375/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12374375/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12374375%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12374375/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12374375/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12374375/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40849622/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12374375/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40849622/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12374375/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12374375/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="0QcyzuAcl3pZQAcY56cFc2JRWLKZZeegGHHWfzHCMC81jKRY1C8UbUkLVQ8KmEFG">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
