
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Deep mutational scanning reveals EGFR mutations conferring resistance to the 4th-generation EGFR tyrosine kinase inhibitor BLU-945 - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AAA98AF2B8C305AAA9004E57C2C4.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="npjprecisoncol">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368022/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="NPJ Precision Oncology">
<meta name="citation_title" content="Deep mutational scanning reveals EGFR mutations conferring resistance to the 4th-generation EGFR tyrosine kinase inhibitor BLU-945">
<meta name="citation_author" content="Yueyang Wang">
<meta name="citation_author_institution" content="Vilcek Institute of Graduate Biomedical Sciences, NYU Grossman School of Medicine, New York, NY USA">
<meta name="citation_author" content="Yuan Hao">
<meta name="citation_author_institution" content="Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA">
<meta name="citation_author" content="Michela Ranieri">
<meta name="citation_author_institution" content="Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA">
<meta name="citation_author" content="Tigran M Abramyan">
<meta name="citation_author_institution" content="Department Origin, South San Francisco, CA USA">
<meta name="citation_author" content="Lev Tsidilkovski">
<meta name="citation_author_institution" content="Department Origin, South San Francisco, CA USA">
<meta name="citation_author" content="Michelle Hollenberg">
<meta name="citation_author_institution" content="Vilcek Institute of Graduate Biomedical Sciences, NYU Grossman School of Medicine, New York, NY USA">
<meta name="citation_author" content="Alfonso Lopez">
<meta name="citation_author_institution" content="Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA">
<meta name="citation_author" content="Xavier T R Moore">
<meta name="citation_author_institution" content="Vilcek Institute of Graduate Biomedical Sciences, NYU Grossman School of Medicine, New York, NY USA">
<meta name="citation_author" content="Fiona Sherman">
<meta name="citation_author_institution" content="Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA">
<meta name="citation_author" content="Jiehui Deng">
<meta name="citation_author_institution" content="Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA">
<meta name="citation_author" content="Hayk Saribekyan">
<meta name="citation_author_institution" content="Department Origin, South San Francisco, CA USA">
<meta name="citation_author" content="Garegin Papoian">
<meta name="citation_author_institution" content="Department Origin, South San Francisco, CA USA">
<meta name="citation_author" content="Kwok-Kin Wong">
<meta name="citation_author_institution" content="Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA">
<meta name="citation_author" content="Elaine Shum">
<meta name="citation_author_institution" content="Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA">
<meta name="citation_author" content="John T Poirier">
<meta name="citation_author_institution" content="Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="9">
<meta name="citation_firstpage" content="294">
<meta name="citation_doi" content="10.1038/s41698-025-01086-2">
<meta name="citation_pmid" content="40836025">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368022/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368022/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368022/pdf/41698_2025_Article_1086.pdf">
<meta name="description" content="Fourth-generation EGFR tyrosine kinase are in development to overcome common resistance mutations. We performed deep mutational scanning (DMS) of the EGFR kinase domain in the context of L858R by introducing a saturation library of ~17,000 variants ...">
<meta name="og:title" content="Deep mutational scanning reveals EGFR mutations conferring resistance to the 4th-generation EGFR tyrosine kinase inhibitor BLU-945">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Fourth-generation EGFR tyrosine kinase are in development to overcome common resistance mutations. We performed deep mutational scanning (DMS) of the EGFR kinase domain in the context of L858R by introducing a saturation library of ~17,000 variants ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368022/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12368022">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41698-025-01086-2"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41698_2025_Article_1086.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368022%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12368022/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12368022/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368022/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-npjprecisoncol.jpg" alt="NPJ Precision Oncology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to NPJ Precision Oncology" title="Link to NPJ Precision Oncology" shape="default" href="https://www.nature.com/npjprecisiononcology/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">NPJ Precis Oncol</button></div>. 2025 Aug 20;9:294. doi: <a href="https://doi.org/10.1038/s41698-025-01086-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41698-025-01086-2</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22NPJ%20Precis%20Oncol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22NPJ%20Precis%20Oncol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22NPJ%20Precis%20Oncol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22NPJ%20Precis%20Oncol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Deep mutational scanning reveals EGFR mutations conferring resistance to the 4th-generation EGFR tyrosine kinase inhibitor BLU-945</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yueyang Wang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yueyang Wang</span></h3>
<div class="p">
<sup>1</sup>Vilcek Institute of Graduate Biomedical Sciences, NYU Grossman School of Medicine, New York, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yueyang Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hao%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yuan Hao</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yuan Hao</span></h3>
<div class="p">
<sup>2</sup>Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hao%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuan Hao</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ranieri%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Michela Ranieri</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Michela Ranieri</span></h3>
<div class="p">
<sup>2</sup>Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ranieri%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michela Ranieri</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abramyan%20TM%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Tigran M Abramyan</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Tigran M Abramyan</span></h3>
<div class="p">
<sup>3</sup>Department Origin, South San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abramyan%20TM%22%5BAuthor%5D" class="usa-link"><span class="name western">Tigran M Abramyan</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsidilkovski%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Lev Tsidilkovski</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Lev Tsidilkovski</span></h3>
<div class="p">
<sup>3</sup>Department Origin, South San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsidilkovski%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lev Tsidilkovski</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hollenberg%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Michelle Hollenberg</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Michelle Hollenberg</span></h3>
<div class="p">
<sup>1</sup>Vilcek Institute of Graduate Biomedical Sciences, NYU Grossman School of Medicine, New York, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hollenberg%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michelle Hollenberg</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lopez%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Alfonso Lopez</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Alfonso Lopez</span></h3>
<div class="p">
<sup>2</sup>Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lopez%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alfonso Lopez</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moore%20XTR%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Xavier T R Moore</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Xavier T R Moore</span></h3>
<div class="p">
<sup>1</sup>Vilcek Institute of Graduate Biomedical Sciences, NYU Grossman School of Medicine, New York, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moore%20XTR%22%5BAuthor%5D" class="usa-link"><span class="name western">Xavier T R Moore</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sherman%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Fiona Sherman</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Fiona Sherman</span></h3>
<div class="p">
<sup>2</sup>Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sherman%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fiona Sherman</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deng%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Jiehui Deng</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Jiehui Deng</span></h3>
<div class="p">
<sup>2</sup>Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Deng%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jiehui Deng</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saribekyan%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Hayk Saribekyan</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Hayk Saribekyan</span></h3>
<div class="p">
<sup>3</sup>Department Origin, South San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saribekyan%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hayk Saribekyan</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Papoian%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Garegin Papoian</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Garegin Papoian</span></h3>
<div class="p">
<sup>3</sup>Department Origin, South San Francisco, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Papoian%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Garegin Papoian</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wong%20KK%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Kwok-Kin Wong</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Kwok-Kin Wong</span></h3>
<div class="p">
<sup>2</sup>Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wong%20KK%22%5BAuthor%5D" class="usa-link"><span class="name western">Kwok-Kin Wong</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shum%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Elaine Shum</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Elaine Shum</span></h3>
<div class="p">
<sup>2</sup>Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shum%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Elaine Shum</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Poirier%20JT%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">John T Poirier</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">John T Poirier</span></h3>
<div class="p">
<sup>2</sup>Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Poirier%20JT%22%5BAuthor%5D" class="usa-link"><span class="name western">John T Poirier</span></a>
</div>
</div>
<sup>2,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Vilcek Institute of Graduate Biomedical Sciences, NYU Grossman School of Medicine, New York, NY USA </div>
<div id="Aff2">
<sup>2</sup>Laura and Isaac Perlmutter Cancer Center, NYU Langone Health, New York, NY USA </div>
<div id="Aff3">
<sup>3</sup>Department Origin, South San Francisco, CA USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 13; Accepted 2025 Aug 7; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the articleâ€™s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12368022Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40836025/" class="usa-link">40836025</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Fourth-generation EGFR tyrosine kinase are in development to overcome common resistance mutations. We performed deep mutational scanning (DMS) of the EGFR kinase domain in the context of L858R by introducing a saturation library of ~17,000 variants into Ba/F3 cells. DMS library-expressing cells were exposed to osimertinib or BLU-945 to identify escape mutations. L718X mutations were enriched across all conditions. BLU-945 specific mutations included K714R, K716T, L718V, T725M, K728E, K754E/N, N771S/T, T783I, Q791L/K, G863S, S895N, K929I, and M971L. A secondary DMS screen combining osimertinib and BLU-945, exclusively enriched for L718X mutations. Clinically, L718X mutations emerged in two patients treated with BLU-945. One patient with baseline EGFR L858R and L718Q mutations experienced early progression. Another with baseline <em>EGFR</em> L858R, T790M, and C797S acquired an L718V mutation at progression. This study demonstrate how comprehensive resistance profiling of targeted therapies can predict clinically relevant mutations and guide rational combinations to delay or prevent resistance.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Cancer genetics, Cancer therapeutic resistance, Cancer models</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par2">Lung adenocarcinoma (LUAD) accounts for ~40% of primary lung cancer<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>, with most patients presenting with locally advanced or metastatic disease<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Molecular profiling of LUAD has identified driver mutations in over 80% of cases<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>, with mutations in epidermal growth factor receptor (EGFR) among the most common<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Incidence in patients with Asian ancestry is 40â€“55%, notably higher than the 5â€“15% frequency observed in Caucasians<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. The majority of EGFR mutations occur in the kinase domain and manifest as short 3â€“7 amino acid deletions in exon 19 (ex19del) or as an L858R (p.Leu858Arg) missense mutation encoded in exon 21<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Less common mutations such as G719X (p.Gly719X) and small insertions in exon 20 account for the remainder<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. These mutations cause constitutive activation of EGFR, leading to unchecked cell proliferation, resistance to apoptosis and the promotion of metastasis<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>.</p>
<p id="Par3">Targeting mutant EGFR with small molecule tyrosine kinase inhibitors (TKIs) has proven to be an effective therapeutic strategy in EGFR mutant LUAD, leading to multiple FDA approvals<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>â€“<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. The first- generation TKIs gefitinib and erlotinib target the ATP-binding pocket of the kinase domain, effectively inhibiting downstream signaling<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Both gefitinib and erlotinib demonstrated significant improvement in progression-free survival (PFS) compared to platinum doublet chemotherapy in patients with tumors harboring EGFR L858R and ex19del alterations<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Despite initial clinical responses, acquired resistance frequently developed within one year<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. A dominant mechanism of resistance to first-generation EGFR TKIs is the T790M (p.Thr790Met) Gatekeeper mutation, which abolishes TKI binding while retaining kinase activity<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>.</p>
<p id="Par4">One current standard of care for treatment-naÃ¯ve EGFR mutant LUAD is osimertinib, a third-generation TKI, that was developed to overcome the T790M Gatekeeper mutation<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Osimertinib covalently target C797, inhibiting EGFR ex19del and L858R irrespective of the presence of T790M while sparing wild type EGFR<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Despite initial efficacy, 7â€“26% of patients develop resistance to osimertinib by acquiring additional second-site mutations that are enriched in an allele specific manner<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. EGFR L858R typically escapes osimertinib through L718Q/V (p.Leu718Gln/Val) mutations while ex19del tends to escape <em>via</em> C797S (p.Cys797Ser), which destroys the cystine nucleophile required for covalent binding of osimertinib<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>.</p>
<p id="Par5">A diverse range of resistance mechanisms not attributable to second site mutations have also been described. These include activation of bypass signaling pathways, such as MET amplification-medicated resistance, which occurs in 7â€“15% following first-line osimertinib therapy<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>, as well as histologic transformation to small cell lung cancer (SCLC)<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Other identified mechanisms include EGFR amplification and mutations in downstream effectors such as <em>KRAS, BRAF</em>, and <em>PIK3CA</em><sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Notably, these aberrations can co-occur within the same tumor and co-exist with on target EGFR mutations, underlying the complexity and heterogeneity of tumor evolution and the emergence of resistance to EGFR-TKI therapy<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>.</p>
<p id="Par6">For C797S -trans T790M mutations, where C797S and T790M are encoded on different alleles, combinations of first- and third-generation TKIs have shown some efficacy in preclinical models<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Conversely, in C797S -cis T790M mutations, where C797S and T790M are on the same allele, combination of first-generation TKIs with osimertinib are ineffective<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. These data suggest that suppression of allelic diversity within cells or tumor cell populations may be an important consideration in the development of novel EGFR TKIs.</p>
<p id="Par7">Fourth-generation ATP competitive EGFR TKIs are designed to retain efficacy in the context of C797S<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>â€“<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup> mutations; however, the landscape of potential second-site mutation liabilities associated with these inhibitors remains unknown<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. BLU-945 is an investigational fourth-generation EGFR TKI designed to target EGFR T790M and EGFR T790M/C797S mutations that confer resistance to first-, second- and third-generation TKIs<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Unlike osimertinib which binds covalently to the C797 residue, the scaffold of BLU-945 allows it to adopt a conformation that fits within the ATP binding pocket of EGFR in the presence of the C797S mutation while sparing wild-type EGFR<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. BLU-945 has been evaluated clinically as a monotherapy and in combination with osimertinib in previously treated patients with EGFR driven LUAD<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>.</p>
<p id="Par8">In this study, we aimed to identify potential mutational resistance mechanisms to BLU-945 using deep mutational scanning to assess the functional impact of thousands of EGFR mutations in parallel. We screened a DMS library of mutant cDNAs covering the EGFR L858R kinase domain to saturation in the context of a panel of EGFR TKIs to systematically profile all possible amino acid substitutions and their downstream effects.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Deep mutational scanning of the EGFR kinase domain recovers known osimertinib resistance mutations</h3>
<p id="Par9">We generated a DMS library encompassing the EGFR L858R kinase domain encoded by a lentiviral transfer vector that co-expresses green fluorescent protein (GFP) <em>via</em> internal ribosomal entry site (IRES) (Fig. <a href="#Fig1" class="usa-link">1A</a>). The library was generated using a mutagenic PCR approach, which relies on oligonucleotide primer pools encoding degenerate codons<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>â€“<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Each codon in the 804â€‰bp target region can be mutated to one of 63 possible codons, including stop codons, resulting in a library of 16,884 potential variants (268 amino acidsâ€‰Ã—â€‰63 possible codons). The library consists of over 1â€‰Ã—â€‰10<sup>7</sup> primary clones, which corresponds to &gt;500Ã— representation over the theoretical number of variants. Sanger sequencing of individual clones indicated an average of 1.14 mutations per cDNA (Fig. <a href="#Fig1" class="usa-link">1B</a>). The plasmid library was sequenced using barcoded error correction and showed negligible mutational bias in transitions and transversions (Fig. <a href="#Fig1" class="usa-link">1C</a>). 16,882 unique variants were recovered, corresponding to &gt;99.9% saturation with &gt;99% of variants observed &gt;10 times (Fig. <a href="#Fig1" class="usa-link">1D</a>). The abundance of codon variants was approximately normally distributed with a mean of 168 unique observations (Fig. <a href="#Fig1" class="usa-link">1E</a>). The EGFR L858R DMS library was packaged in VSV-g pseudotyped lentivirus at large scale and concentrated by centrifugation. The titer of the lentiviral library was 2â€‰Ã—â€‰10<sup>7</sup> transduction units (TU) as determined by detection of GFP expression by flow cytometry. The total TU corresponded to ~1185â€‰Ã—â€‰representation.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Deep mutational scanning of the EGFR L858R kinase domain in Ba/F3 cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368022_41698_2025_1086_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dd4/12368022/4c3171a138d0/41698_2025_1086_Fig1_HTML.jpg" loading="lazy" id="d33e456" height="564" width="682" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Map of pHAGE-EGFR-L858R with kinase domain annotated. <strong>B</strong> Frequency of mutations count per cDNA. <strong>C</strong> Per-codon frequency of the indicated mutation types. <strong>D</strong> Cumulative distribution of read counts for all expected variants in the library. <strong>E</strong> Density plot of plasmid library codon counts. <strong>F</strong> Enrichment of library transduced Ba/F3 cells after IL-3 withdrawal. <strong>G</strong> Positive selection screening scheme.</p></figcaption></figure><p id="Par10">We conducted all screens in Ba/F3 cells, a murine pro-B cell line dependent on extracellular interleukin-3 (IL-3) for survival. Upon transduction with potent oncogenes, these cells become dependent on the introduced oncogene for mitogenic signaling<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Ba/F3 cells were transduced at a low multiplicity of infection (MOIâ€‰~â€‰0.1) to bias toward single integrations while achieving &gt;1000-fold representation of the library<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. After allowing 7 days for provirus integration and EGFR L858R mutant protein expression, IL-3 was withdrawn from the culture media. This established a selective pressure under which Ba/F3 cell survival depended on the signaling activity of transduced EGFR mutants<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Cells expressing viable EGFR variants were expected to survive without IL-3, while those harboring non-functional or deleterious mutations, or that were not transduced, were outcompeted and depleted from the population. After 14 days of IL-3 withdrawal, over 95% of the surviving Ba/F3 cells were GFP positive, consistent with positive selection of the transduced pool (Fig. <a href="#Fig1" class="usa-link">1F</a>)<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. The resulting EGFR L858R DMS library-expressing Ba/F3 cells were subsequently challenged with various EGFR inhibitors or DMSO for 14 days, with regular replenishment of compounds to maintain consistent drug pressure. At the end of the treatment period, gDNA was harvested and subjected to deep sequencing with error correction as previously described<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> to identify outgrowing clones. These enriched variants were interpreted as candidate resistance mutations (Fig. <a href="#Fig1" class="usa-link">1G</a>).</p></section><section id="Sec4"><h3 class="pmc_sec_title">Deep mutational scanning reveals EGFR mutations conferring resistance to BLU-945</h3>
<p id="Par11">To address whether known resistance mutations could be recovered from the previously introduced Ba/F3 DMS pool, we exposed the EGFR L858R DMS library-expressing Ba/F3 cells to osimertinib at 100â€‰nM (&gt;EC<sub>90</sub> of osimertinib) over a 14-day period with regular replenishment of drug to maintain consistent pressure Independent biological replicates demonstrated high concordance in the enriched variant profiles, supporting the reproducibility of the results. Among the osimertinib-treated samples, L718X (p. Leu718X) mutations were enriched compared to control (Supplementary <a href="#MOESM1" class="usa-link">1A, B</a>). As L718X is the dominant resistance mutation to osimertinib in the context of EGFR L858R oncogenic driver mutation, these results validate the validity of our screening approach.</p>
<p id="Par12">We next employed the same screening workflow in the context of BLU-945 at 100â€‰nM (&gt;EC<sub>90</sub>) for 14 days with regular drug replenishment. Endpoint samples were harvested, gDNA was extracted, and deep sequencing was performed as above. In contrast to the osimertinib screen, we identified numerous potential resistance mutations for BLU-945 (Fig. <a href="#Fig2" class="usa-link">2A</a>). Candidate resistance mutations clustered in the hinge region amino acids Q791 and M793, which form hydrogen bonds with the adenosine ring of ATP<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Q791L/K and M793X mutations are supported as likely resistance mutations by structural evidence of a previously published co-crystal structure of the BLU-945 analog â€˜compound 24â€™ and EGFR kinase domain (PDB:8D76)<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. The amino-naphthyridine NH group of BLU-945 forms a hydrogen bond to the backbone carbonyl of Q791, while the naphthyridine N2 forms a hydrogen bond with the backbone NH of M793, stabilizing the inhibitor-kinase complex. We conclude that the results of the screen are consistent with the binding mode of BLU-945.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Identification of resistance mutations for BLU-945.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368022_41698_2025_1086_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dd4/12368022/70f3bdbba66c/41698_2025_1086_Fig2_HTML.jpg" loading="lazy" id="d33e544" height="737" width="682" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Scatter plot of enrichment scores after BLU-945 selection. Inset: hinge mutations involved in APT-binding. <strong>B</strong> Scatter plot of enrichment scores after BLU-945 selection, Hamming distance of 1. Inset: hinge mutations present after filtering. <strong>C</strong> Enrichment of BLU-945 resistance mutations relative to the EGFR kinase domain structure. <strong>D</strong> Poseview diagram of BLU-945, illustrating key contact residues. <strong>E</strong> L858R single mutant MD showed no steric clashes with BLU-945 overlayed from PDB: 8D76 (cyan sticks, edited from 8D76 ligand). Transparent gray shows all MD frames; orange highlights the cluster centroid. <strong>F</strong> L858R/L718Q double mutant MD frames sterically clash with BLU-945.</p></figcaption></figure><p id="Par13">DMS screens explore all potential mutations at each amino acid position that are compatible with sustained EGFR L858R signaling, including complex codon changes that are less likely to occur in patients. We focused on amino acid changes that could arise from a single nucleotide substitution (Hamming distance = 1) from the wild-type codon. After filtering, Q791L/K mutations remain while M793X are removed, suggesting that mutations to M793 would be a less likely escape mechanism (Fig. <a href="#Fig2" class="usa-link">2B</a>). We identified the following mutations as resistance mutations that can arise through single base mutations: K714R (p.Lys714Arg), K716T (p.Lys716Thr), L718V (p.Leu718Val), T725M (p.Thr725Met), K728E (p.Lys728Glu), K754E/N (p.Lys754Glu/Asn), N771S/T (p.Asn771Ser/Thr), T783I (p.Thr783Ile), Q791L/K (p.Gln791Leu/Lys), G863S (p.Gly863Ser), S895N (p.Ser895Asn), K929I (p.Lys929Ile), and M971L (p.Met971Leu). Several of these mutations have been observed clinically, such as K714R<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>, L718V<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>, T725M<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, K728E<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>, T751P<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>, K754E/N<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>, and N771S<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>.</p>
<p id="Par14">When viewing BLU-945 mutations in the context of kinase domain structure, several patterns become apparent (Fig. <a href="#Fig2" class="usa-link">2C</a>). First, mutations tend to be clustered in the N lobe and hinge regions of the kinase domain, which play a comparatively larger role in the catalytic activity of EGFR than the C lobe. Second, we observed frequent mutations in lysine residues in the Î²1-3 strands of the N lobe. Third, we see a cluster of mutations in the region between the Î±C-helix and Î²4-5 strands involving N771 and P772, two residues involved in ex20ins variants<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. We suspect that these point mutations in this region might distort P-loop and Î±C-helix conformation in a manner that indirectly affects BLU-945 binding.</p>
<p id="Par15">Many of the mutations highlighted here involve contact residues that are affinity determinants of BLU-945 (Fig. <a href="#Fig2" class="usa-link">2D</a>). Notably, Q791 and M793 hinge contact points were present in compound 7, an early lead compound in the BLU-945 discovery campaign<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>, while other mutations such as K716T and K728E involve residues engaged by the sulfone group characteristic of compound 24 and eventually BLU-945, developed late in the medicinal chemistry program. The large number of potential mutational escape routes for BLU-945 relative to osimertinib may be due to a greater number of required binding affinity determinants. Importantly, residues most critical for BLU945 binding, and therefore most vulnerable to resistance mutations, tend to cluster around the ATP binding pocket occupied by the inhibitor. These include contact residues such as K716, K728, L718, and Q791.</p>
<p id="Par16">Not observed in the screen were mutations to the back pocket residue K745, which forms a hydrogen bond with the piperidinol ring of BLU-945<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Molecular interactions with K745 are a common and distinct feature of reversible, fourth-generation ATP-competitive EGFR kinase inhibitors<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. We suspect that K745 mutations are not observed since this residue is the conserved â€œcatalytic lysineâ€ which is necessary to form a salt bridge with E762 in the kinase active state that is essential for phosphoryl transfer activity<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>.</p>
<p id="Par17">In addition to mutations of key contact point residues, DMS identified L718V as a key resistance mutation to BLU-945, like the osimertinib screen. Structurally, L718 is positioned in the first Î²-strand (Î²1) of the N-lobe near the ATP-binding site. Given the higher clinical prevalence of L718Q over L718V in patients with L858R-mutant EGFR, we sought to further characterize the structural impact of the L718Q variant. To this end, we conducted all-atom molecular dynamics (MD) simulations and absolute binding free energy perturbation (ABFEP) calculations, using structures modeled on PDB: 8D76. We hypothesized that isoform-specific dynamics in the apo state modulate the architecture of the binding pocket and influence BLU-945 affinity.</p>
<p id="Par18">Our 100â€‰ns MD simulations showed that the L718Q mutation induces the greatest conformational perturbation in the binding site, particularly within the P-loop, as quantified by root-mean-square deviation (RMSD) analyses (Supplementary <a href="#MOESM1" class="usa-link">1C, D</a>). Overlay of the most populated MD conformers with the ligand-bound structure revealed a steric clash between the Q718 side chain and BLU-945, suggesting impaired binding compatibility (Fig. <a href="#Fig2" class="usa-link">2E, F</a>).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Functional validation of individual BLU-945 resistance mutations in multiple allelic contexts</h3>
<p id="Par19">To test whether any resistance mutations identified in the primary screen were sufficient to confer resistance to BLU-945, we chose to focus on the Q791L hinge mutation given its importance in the primary screen for BLU-945 and the relative ease of mutation at that site as well as L718Q, since this mutation is most frequently identified as a resistance mutation in patients with L858R mutations<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. We generated individual Ba/F3 cell lines stably expressing candidate resistance mutations in the context of L858R and exon 19 deletion (Del19) mutations (delE746-A750), which represent the most common EGFR-activating mutations in LUAD. Additionally, we introduced co-mutations of T790M and C797S to model clinically relevant resistance scenarios. Although T790M clones may be suppressed when osimertinib is used in the first-line setting, not all fourth-generation EGFR TKIs are designed to against this mutation (e.g., BDTX-1535). Therefore, we included T790M to ensure a more comprehensive evaluation of resistance mechanisms. All mutant constructs conferred growth factor independence in Ba/F3 cells apart from del19/L718Q, consistent with published reports in patients and mouse model systems suggesting that this co-mutation is non-functional<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>,<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>.</p>
<p id="Par20">We observed that Ba/F3 cells in L858R or del19 context, in conjunction with T790M/C797S mutations, displayed increased resistance to BLU-945 when either L718Q or Q791L mutations were present. The EC<sub>50</sub> values for L858R/T790M/C797S and del19/T790M/C797S were 1.71â€‰nM (1.26â€‰nMâ€“2.40â€‰nM 95%CI) and 7.81â€‰nM (6.09â€‰nMâ€“9.96â€‰nM 95%CI) respectively; however, the EC<sub>50</sub> was &gt;100â€‰nM with the addition of L718Q or Q791L resistance mutations. (Fig. <a href="#Fig3" class="usa-link">3Aâ€“D</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Testing of individual resistance mutations in the context of L858R, Del19, T790M, and C797S.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368022_41698_2025_1086_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dd4/12368022/ee72c1baf363/41698_2025_1086_Fig3_HTML.jpg" loading="lazy" id="d33e676" height="1019" width="772" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Viability dose-response curves showing the sensitivity of Ba/F3 cells expressing EGFR L858R or Del19 alone, as well as those harboring candidate resistance mutations (L718Q, Q791L), (D770-N771 InsSVD as a EGFR ex20ins control). <strong>B</strong> Viability dose-response curves of Ba/F3 cells expressing EGFR L858R/T790M or Del19/T790M, with or without co-occurring L718Q or Q791L mutations. <strong>C</strong> Viability dose-response curves of Ba/F3 cells expressing EGFR L858R/C797S or Del19/C797S, with or without co-occurring L718Q or Q791L mutations. <strong>D</strong> Viability dose-response curves of Ba/F3 cells harboring the triplet mutation combinations L858R/T790M/C797S or Del19/T790M/C797S, with or without L718Q or Q791L. <strong>E</strong> Western blot analysis of Ba/F3 cells under the indicated conditions. Both short exposure (S.E.) and long exposure (L.E.) images are shown to capture signal intensity differences.</p></figcaption></figure><p id="Par21">Western blot analysis revealed that the phosphorylation levels of EGFR in L858R/T790M/C797S and del19/T790M/C797S mutant cells were markedly reduced following treatment with 100â€‰nM BLU-945. However, no significant reduction in phosphorylation was observed in cells harboring L718Q or Q791L cDNAs, indicating that these second-site mutations confer resistance to BLU-945 by escaping EGFR inhibition (Fig. <a href="#Fig3" class="usa-link">3E</a>).</p>
<p id="Par22">To further dissect the biophysical mechanisms underlying resistance, we performed ABFEP calculations using representative structures from MD simulations. Starting from the BLU-945 bound crystal structure (PDB: 8D76), we aligned and equilibrated ligandâ€“protein complexes and calculated Î”G values for various EGFR isoforms. The predicted binding affinities closely mirrored experimental trends: L858R, L858R/T790M, and L858R/T790M/C797S all showed strong binding to BLU-945, consistent with low EC<sub>50</sub> values. L858R/C797S also retained favorable binding, though with a modest reduction.</p>
<p id="Par23">In contrast, the L858R/T790M/Q791L mutant exhibited weaker binding affinity, with a Î”G shift consistent with the observed &gt;2.5â€‰kcal/mol drop in binding affinity. This change likely stems from a subtle entropic gain, as the hydrophobic leucine substitution at Q791 exposed to solvent and not directly involved in hydrogen bonding which may increase local flexibility relative to the more constrained wild-type hinge. The L858R/L718Q variant showed the most substantial loss in binding, with a calculated Î”G nearly 10â€‰kcal/mol less favorable than L858R/T790M, reinforcing its role as a potent resistance mutation (Supplementary Fig. <a href="#MOESM1" class="usa-link">1E</a>; Table <a href="#MOESM1" class="usa-link">1</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title">Osimertinib co-treatment suppresses BLU-945 kinase domain resistance mutants apart from L718X</h3>
<p id="Par24">In the first-in-human Phase I/II SYMPHONY trial, patients were treated with BLU-945 in combination with osimertinib<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. To explore the potential of overcoming resistance to BLU-945 through combination therapy, we tested whether the resistance profiles of individual compounds, in this case BLU-945 and osimertinib, could predict the resistance profile for the combination treatment. Pairwise comparison of single agent resistance profiles suggested that osimertinib could effectively inhibit most resistance mutations observed with BLU-945 monotherapy, with the notable exception of mutations at position L718 (Fig. <a href="#Fig4" class="usa-link">4A</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Prediction and experimental validation of EGFRi cross-resistance.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368022_41698_2025_1086_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dd4/12368022/77e5449e113b/41698_2025_1086_Fig4_HTML.jpg" loading="lazy" id="d33e725" height="624" width="682" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Pairwise comparison of osimertinib and BLU-945 resistance profiles. <strong>B</strong> Scatter plot of enrichment scores after combination osimertinib and BLU-945 selection. <strong>C</strong> Heatmap of Log<sub>10</sub> EC<sub>50</sub> values for 9 distinct inhibitors calculated from 7-point dose response curves.</p></figcaption></figure><p id="Par25">To test our hypothesis, we performed a DMS experiment using BLU-945 and osimertinib in combination. The enrichment plot generated from these co-treatment endpoint samples indicated that resistance to both agents exclusively occurs at the L718 position where only proline (L718P) is enriched after filtering for Hamming distance &gt;1 (Fig. <a href="#Fig4" class="usa-link">4B</a>), illustrating a common resistance liability at this position.</p>
<p id="Par26">While osimertinib and BLU-945 share a common resistance liability at position L718, we were encouraged that all other second-site mutations from the BLU-945 screen were effectively suppressed. We tested the resistance profiles of 9 distinct EGFR inhibitors to determine to what extent they shared resistance mechanisms with BLU-945 (Fig. <a href="#Fig4" class="usa-link">4C</a>). Notably, BLU-945 suppressed T790M/C797S in both L858R and Del19 contexts. Mutations conferring resistance to BLU-945 such as Q791L were sensitive to all drugs tested in the context of either L858R or ex19del. However, rescue by osimertinib, sunvozertinib, lazertinib, or zipalertinib was all abolished by the C797S co-mutation. Examination of cross-resistance patterns of these drugs suggests that the combination of BLU-945 with one of the several EGFR TKIs with activity against Q791X or L718X might be an attractive strategy in contexts where the two mutations do not occur <em>in cis</em>.</p></section><section id="Sec7"><h3 class="pmc_sec_title">BLU-945 and osimertinib cross-resistant L718X mutations identified in two patients with primary resistance and on-treatment disease progression</h3>
<p id="Par27">Our institution participated in the SYMPHONY clinical trial evaluating BLU-945, both as monotherapy and in combination with osimertinib, for patients with metastatic EGFR-mutant LUAD. We identified two patients harboring L718X EGFR mutations.</p>
<p id="Par28">Patient A (Fig. <a href="#Fig5" class="usa-link">5A</a>) harbored an EGFR L858R mutation and was previously treated with osimertinib. The patient had a liquid biopsy obtained on day â€“659 which identified the presence of <em>EGFR</em> L858R and L718Q mutations. Baseline imaging on day 0 revealed two nodules in the right lung, measuring 1.2â€‰Ã—â€‰0.7â€‰cm (middle lobe) and 2.7â€‰Ã—â€‰2.2â€‰cm (lower lobe). The patient started BLU-945 on day 12. Although initial follow-up imaging demonstrated stable disease, evaluations on day 37 and day 40 revealed disease progression with new bone metastases and spinal cord compression, necessitating surgical intervention. Post-surgery imaging on day 68 showed stable lung lesions, but further scans on day 121 demonstrated continued disease progression (+20% from nadir) along with new brain metastases. Additional progression in the spine was confirmed on day 177 and day 181.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Clinical courses of two patients with L718X mutation.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12368022_41698_2025_1086_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dd4/12368022/73e20ad099be/41698_2025_1086_Fig5_HTML.jpg" loading="lazy" id="d33e774" height="755" width="683" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Clinical course of patient A. <strong>B</strong> Clinical course of patient B. <strong>C</strong> ctDNA sequencing results for patient A. Left axis is log<sub>10</sub> variant allele frequency, right axis is BLU-945 dose. <strong>D</strong> ctDNA sequencing results for patient B. Left axis is log<sub>10</sub> variant allele frequency, right axis is BLU-945 dose.</p></figcaption></figure><p id="Par29">Patient B (Fig. <a href="#Fig5" class="usa-link">5B</a>) presented with <em>EGFR</em> L858R and developed on-treatment resistance mutations (<em>EGFR</em> T790M and C797S) following multiple prior treatments including osimertinib. A pre-treatment liquid biopsy performed on day-34 revealed <em>EGFR</em> L858R (34.7% VAF), T790M (13.7% VAF), and C797S (14.2% VAF). Baseline imaging on day 0 showed a right middle lobe lung nodule measuring 1.0â€‰Ã—â€‰0.8â€‰cm (TL1) as well as a liver lesion measuring 1.5â€‰Ã—â€‰1.1â€‰cm (TL2). BLU-945 therapy commenced on day 6. Follow-up imaging on day 33 demonstrated stable disease, with the sum of the target lesions decreasing by âˆ’1.7% overall from baseline. However, by day 61, the sum of target lesions increased by 3.3%. On day 117, imaging demonstrated progression due to new liver lesions, prompting a BLU-945 dose escalation to 400â€‰mg.</p>
<p id="Par30">On day 173, CT imaging indicated an overall growth of the target lesions by &gt; 41%. Although the patient initially exhibited stable disease, therapy was ultimately discontinued due to rapid disease progression.</p>
<p id="Par31">Four liquid biopsies were obtained from Patient A at day âˆ’659, âˆ’413, âˆ’51, and 180 relative to day 0 (Fig. <a href="#Fig5" class="usa-link">5C</a>). During this interval, the <em>EGFR</em> L858R mutation rose from 15.8% to 80.0% VAF. Two distinct L718Q variants were also detected: L718Q (2151_2153GCTâ€‰&gt;â€‰ACA) was first measurable at 0.62% on day -51 and increased to 2.29% by day 180. L718Q (2153â€‰Tâ€‰&gt;â€‰A) was detected at 2.4% on day -659, fluctuating thereafter and ultimately reaching 12.11% on day 180.</p>
<p id="Par32">Two on-treatment liquid biopsies were obtained from Patient B on days 119 and 147. The first confirmed <em>EGFR</em> L718V (0.15% VAF) in addition to L858R (39.69% VAF), T790M (17.19% VAF), and C797S (17.01% VAF) (Fig. <a href="#Fig5" class="usa-link">5D</a>). Liquid biopsy on day 147 coincided with an increase in L718V to 0.2% VAF, with reduced levels of L858R (13.9% VAF), T790M (6% VAF), and C797S (6% VAF).</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par33">In this study, we employed deep mutational scanning to systematically map amino acid substitutions in the EGFR L858R kinase domain that confer resistance to BLU-945, a fourth-generation EGFR inhibitor. We validated Q791L and L718Q mutations, which emerged as crucial resistance determinants in L858R and ex19del contexts, irrespective of T790M or C797S co-mutations. L718X mutations are notable as a singular shared liability between BLU-945 and osimertinib, highlighting the challenge of covering all potential mutational escape routes even in the case of structurally distinct inhibitors.</p>
<p id="Par34">DMS has the potential to provide comprehensive foreknowledge of on-target resistance liabilities not previously possible by methods such as long-term drug treatment or chemical mutagenesis. This approach can be integrated into early drug development pipelines to refine chemical scaffolds against known escape mutations. We show that forward genetics can complement structural information by highlighting constrained contact points such as M793 and K745, while minimizing the vulnerability at sites like Q791 and L718. These positions in the EGFR kinase domain may be more â€œmutationally pliableâ€ than others, enabling multiple potential substitutions that preserve kinase function and reduce inhibitor efficacy. Defining these mutationally permissive hotspots could support the development of inhibitors with increased robustness to mutational mechanisms of acquired resistance.</p>
<p id="Par35">Anticipating drug resistance mutations early in drug development could also inform rational design of combination or sequential regimens in the clinic. For example, if preclinical data predict that L718X and Q791L/K will arise under selective pressure from BLU-945, treatment strategies could include pairing BLU-945 with a complementary inhibitor with orthogonal resistance liabilities to diminish the potential for outgrowth of resistant clones. From the perspective of clinical trial design, comprehensive resistance profiles could be used to further stratify patients to enhance response rates, increasing the probability of successful trial. This approach would also respect ethical considerations by providing the necessary information to direct patients away from therapies that have reduced potential for efficacy. It is worth noting that while BLU-945 is no longer in active clinical development, this example highlights the value of such preclinical knowledge for guiding strategy of future drug candidates.</p>
<p id="Par36">Our study is not without limitations. Our screen focuses entirely on the EGFR kinase domain in the context of a single L858R mutant allele. Mutations outside the kinase domain or that require different allelic context would be missed. On target mutations represent only one facet of clinical resistance, whereas real-world tumors can escape <em>via</em> other genetic processes such as gene amplification, bypass signaling, or histological transformations<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. The Ba/F3 system, while well-established for oncogene screening, may not fully capture the complexities of human lung adenocarcinoma, and in some cases, there was variability in total EGFR protein expression level between cell lines, limiting interpretation of pharmacodynamic experiments to comparisons that can be made within cell lines. Screening at a single high-dose condition (â€‰&gt;â€‰EC<sub>90</sub>) could potentially miss subtle resistance phenotypes that might emerge with lower drug concentrations. Finally, while our analysis tested candidate mutations in ex19del and L858R, our primary screen focused on the L858R context, whereas exon ex19del, ex20ins, or comparatively less common point mutations such as G719X or L861Q might yield distinct allele-specific resistance mutations.</p>
<p id="Par37">While the clinical development of BLU-945 has since been discontinued, our results overall illustrate the power of systematic mutational scanning to identify resistance liabilities for emerging EGFR TKIs. Future studies could build upon these findings by exploring combination strategies, building prospective surveillance of likely resistance mutations into early phase clinical trials, and designing novel EGFR TKIs that target common resistance escape routes.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Cell culture</h3>
<p id="Par38">HEK293T cells were cultured in high-glucose DMEM (Gibco #11995073) containing L-glutamine and sodium pyruvate, supplemented with 1% Penicillin-Streptomycin (Gibco #15140-122) and 10% Foundationâ„¢ fetal bovine serum (Gemini Bio-Products). Parental Ba/F3 cells were cultured in RPMI 1640 medium (Gibco #11875085) supplemented with 1% Penicillin-Streptomycin (Gibco #15140-122), 5% Foundationâ„¢ fetal bovine serum (Gemini Bio-Products), and 10â€‰ng/mL recombinant mouse interleukin-3 (BioLegend #575504). Both HEK293T and Ba/F3 parental cell lines were obtained from the American Typed Culture Collection. All cell lines were maintained at 37â€‰Â°C in a humidified incubator with 5% CO<sub>2</sub>.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Plasmids</h3>
<p id="Par39">The plasmid pHAGE-EGFR-L858R was a gift from Gordon Mills &amp; Kenneth Scott (Addgene plasmid #116276; <a href="http://n2t.net/addgene:116276" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://n2t.net/addgene:116276</a>; RRID:Addgene_116276). pHAGE-EGFR-L858R was used as the backbone for subsequent deep mutational scanning (DMS) experiments. Plasmids were transformed and propagated in NEB Stable Competent <em>E.coli (</em>New England Biolabs #C3040H). To prepare the plasmid template for DMS library construction, the plasmid was linearized using the restriction enzyme BSU36I (New England Biolabs #R0524S), which recognizes distinct sequences flanking the region of interest. The digestion reaction was prepared in a final volume of 50â€‰ÂµL, consisting of 1â€‰Âµg of plasmid DNA, 5â€‰ÂµL of 10Ã— rCutSmart Buffer (New England Biolabs #B6004S), 1â€‰ÂµL of BSU36I, and nuclease-free water (New England Biolabs #B1500S) to bring the reaction to final volume. The mixture was incubated at 37â€‰Â°C for 2â€‰h to ensure complete digestion. Following digestion, the reaction products were resolved on a 0.7% agarose gel to confirm successful cleavage. The linear fragment encompassing the EGFR kinase domain was excised and purified using a gel extraction kit.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Construction of EGFR L858R DMS library</h3>
<p id="Par40">To introduce mutations into the EGFR L858R kinase domain, we employed separate forward and reverse mutagenesis PCRs. Forward and reverse mutagenic primer were designed using the CodonTilingPrimers python script with minimum length set to 32â€‰bp and synthesized as two separate oligonucleotide pools (Integrated DNA Technologies)<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. In the forward reaction, 50â€‰ng of linearized EGFR L858R template was combined with Q5 Master Mix (2Ã—, New England Biolabs #M0491S), a 1â€‰ÂµL of a 10â€‰ÂµM forward oligo primer pool, and 1â€‰ÂµL of a 10â€‰ÂµM fixed reverse primer (5â€™-GCTCCAATAAATTCACTGCTTTG-3â€™) in a total volume of 50â€‰ÂµL. The PCR protocol was carried out for 7 cycles using standard Q5 cycling conditions, and the resultant product was purified with a PCR cleanup kit (Qiagen #28506). The reverse mutagenesis PCR was set up similarly using a fixed forward primer (5â€™-GCATACAGTGCCACCCAGAG-3â€™). The PCR protocol was also carried out for 7 cycles using standard Q5 cycling conditions, and the resultant product was purified with the PCR cleanup kit (Qiagen 28506). Each of the purified forward and reverse products was then treated with Exonuclease I (NEB M0293S) to remove any residual mutagenic primers. DNA concentrations were adjusted to approximately 140â€‰ng/ÂµL, and each product was further diluted by adding 60â€‰ÂµL of nuclease-free water to reach a final volume of 90â€‰ÂµL.</p>
<p id="Par41">The forward and reverse products were joined to produce the full-length, mutated insert. Each 30â€‰ÂµL joining PCR reaction consisted of 15â€‰ÂµL of Q5 Master Mix (2Ã—), 4â€‰ÂµL of the diluted forward PCR product, 4â€‰ÂµL of the diluted reverse PCR product, 1â€‰ÂµL of a 10â€‰ÂµM fixed forward primer, 1â€‰ÂµL of 10â€‰ÂµM fixed reverse primer, and 5â€‰ÂµL of nuclease-free water. Six parallel joining PCRs were carried out to ensure sufficient product yield, using the same Q5 cycling conditions for 20 cycles. The products were then purified by PCR cleanup.</p>
<p id="Par42">The mutagenized kinase domain was cloned into pHAGE-EGFR-L858R using restriction enzyme digestion and ligation. 1.6â€‰Âµg of pHAGE-EGFR-L858R and 1â€‰Âµg of the insert were digested with BSU36I and cleaned up by gel extraction. The digestion products were then ligated with T4 DNA ligase (New England Biolabs #M0202S) in a 20â€‰ÂµL reaction containing 166â€‰ng of vector DNA, 100â€‰ng of insert DNA, and 1â€‰ÂµL of T4 DNA ligase. The reaction was incubated at 16â€‰Â°C overnight. A portion of the ligation mixture was tested by chemical transformation to confirm successful ligation prior to proceeding with large-scale transformation.</p>
<p id="Par43">Electrotransformation was performed following the Enduraâ„¢ Competent Cells (COMCEL-002) protocol. Briefly, 25â€‰ÂµL of electrocompetent cells were mixed with up to 5â€‰ÂµL of ligated DNA (final DNA concentration ~55â€‰ng/ÂµL) in each electroporation cuvette. The SOC buffer was warmed to 37â€‰Â°C, and 5â€‰mL culture tubes were prepared in advance. After electroporation, 970â€‰ÂµL of prewarmed SOC buffer was immediately added to each cuvette, and the cells were transferred to 5â€‰mL tubes and recovered at 37â€‰Â°C for 1â€‰h with shaking at 250â€‰rpm.</p>
<p id="Par44">For library expansion, 1â€‰mL of the recovered culture was plated on each Square BioAssay LB agar plate. To determine transformation efficiency by titration, ten-fold serial dilutions were prepared. Plates were incubated overnight at 37â€‰Â°C, and the resulting colony counts were used to calculate transformation efficiencies and ensure sufficient library complexity.</p>
<p id="Par45">After overnight incubation at 37â€‰Â°C, the colonies on the large LB plate were harvested by gently scraping the plate surface with an appropriate volume of LB medium. The collected cells were transferred to a fresh tube, and plasmid DNA was extracted using a standard midi-prep protocol. This pooled plasmid DNA represented the final EGFR L858R DMS library. Individual colonies from the titration plates were selected for Sanger sequencing to determine the presence and frequency of intended mutations.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Lentivirus production</h3>
<p id="Par46">On Day 0, HEK293T cells were seeded in a 15â€‰cm dish to reach ~70% confluence on Day 1. On Day 1, cells were transfected with 6â€‰Âµg total DNA (EGFR L858R DMS library plasmid 3â€‰Âµg, pSPAX 2â€‰Âµg, pMD2.G 1â€‰Âµg) in 200â€‰ÂµL Opti-MEM (Gibco #31985062), using a 2:1 ratio of linear polyethyleneimine (PEI) (Polysciences #23966) to total DNA. The DNAâ€“PEI mixture was incubated at room temperature for 10â€‰min before adding dropwise to the cells. On Day 2, the medium was refreshed. On Day 4, viral supernatants were collected, spun at 300 rcf for 5â€‰min, and filtered through a 0.45â€‰Âµm filter.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Genomic DNA extraction</h3>
<p id="Par47">Genomic DNA (gDNA) was extracted from ~1.5â€‰Ã—â€‰10<sup>7</sup> cells using salt precipitation<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. Cells were resuspended in 200â€‰ÂµL of DPBS in a 15â€‰mL tube. Lysis was achieved by adding 3â€‰mL of lysis buffer (10â€‰mM Tris, 100â€‰mM EDTA, 2% SDS, pH 8.0) containing RNase A (10â€‰mg/mL), followed by vortexing and incubation at 37â€‰Â°C for 5â€‰min, then cooling on ice for 3â€‰min. Protein contaminants were removed by adding 1â€‰mL of protein precipitation solution (7.5â€‰M ammonium acetate), vortexing, and centrifuging at 2000â€‰Ã—â€‰<em>g</em> for 10â€‰min.</p>
<p id="Par48">The supernatant was transferred to a new tube containing 3â€‰mL of isopropanol to precipitate DNA, and the mixture was gently inverted 50 times. DNA was pelleted by centrifugation at 2000â€‰Ã—â€‰<em>g</em> for 3â€‰min, washed with 3â€‰mL of 70% ethanol, and air-dried for 5â€“10â€‰min. The pellet was resuspended in 400â€‰ÂµL of hydration buffer (10â€‰mM Tris, pH 8.0), incubated at 65â€‰Â°C for 1â€‰h, and hydrated overnight at room temperature with shaking at 600â€‰rpm.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Barcoded Subamplicon Sequencing</h3>
<p id="Par49">cDNAs were sequenced using a two-step, barcoded subamplicon PCR strategy based on the method described as previously described<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. In the first step (PCR1), the EGFR kinase domain was divided into two equal subamplicons, and each half was amplified in a separate reaction. Amplicon 1 used primers 5â€™-CTCTTGAGGATCTTGAAGGAAACTGAA-3â€™ and 5â€™-CTTGACATGCTGCGGTGTTTT-3â€™. Amplicon 2 used primers 5â€™-CTTGACATGCTGCGGTGTTTT-3â€™ and 5â€™-ATGCATTCTTTCATCCCCCTGAATGAC-3â€™. Approximately 5â€‰Âµg of the DMS library DNA was split between these two reactions, each containing primers encoding 8 degenerate nucleotides to introduce unique molecular identifiers (UMIs). Following amplification, PCR1 products were purified by isopropanol precipitation to remove residual primers and short fragments, and the concentration of each purified product was determined using the PicoGreen method (Invitrogen #<a href="https://www.ncbi.nlm.nih.gov/protein/P11496" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P11496</a>).</p>
<p id="Par50">For the second PCR step (PCR2), 500 fg of the purified PCR1 product was used as the template for each subamplicon. This PCR step incorporated Illumina sequencing adapters and sample-specific indexes for multiplexed sequencing. The resulting indexed libraries were then purified using SPRI beads.</p>
<p id="Par51">Libraries were pooled at equimolar concentrations and sequenced on an Illumina NovaSeq SP (500 cycles) platform using paired end reads configured to capture the UMI, barcode, and subamplicon regions.</p></section><section id="Sec16"><h3 class="pmc_sec_title">System preparation and molecular dynamics simulations</h3>
<p id="Par52">All systems were prepared using the EGFR T790M_L858R_V948R mutant crystal structure (PDB: 8D76 chain A). Respective reverse or forward mutations were performed to design each isoform for simulations using open source PyMOL version 3.0.0. The bound ligand in the structure was edited to BLU-945. For holo isoform simulations, the ligand was removed. The resulting complex was solvated in a rectangular periodic box with a 1.2â€‰nm buffer of explicit water molecules on all sides.</p>
<p id="Par53">Simulations were carried out using the Deep Origin Simulation Suite. The protein was parameterized with the AMBER FF14SB force field<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>, while the ligand was assigned parameters using the GAFF2 small molecule force field, with atomic charges derived via the AM1-BCC method<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. Solvation employed the TIP3P water model<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> and a physiological ionic strength of 0.15â€‰M was maintained by converting water molecules to Na<sup>+</sup> and Cl<sup>âˆ’</sup> ions to achieve both electroneutrality and the desired salt concentration.</p>
<p id="Par54">To enhance simulation efficiency, hydrogen mass repartitioning (HMR)<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> was applied by increasing the hydrogen mass to 2â€‰Da, enabling the use of extended integration timesteps. All simulations were conducted in the NPT ensemble at 298.15â€‰K and 1â€‰bar, using a BAOAB Langevin integrator and a Monte Carlo barostat<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>,<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup>.</p>
<p id="Par55">To assess structural dynamics and system equilibration, 100â€‰ns molecular dynamics (MD) simulations were performed in duplicate for both the holo (ligand-bound) and apo (ligand-free) systems. The resulting trajectories were subjected to cluster analysis, and the two most populated conformations were selected for subsequent free energy perturbation (FEP) calculations. Prior to FEP, each of these cluster-derived structures was equilibrated with 10â€‰ns of MD using a 4â€‰fs timestep to enable the derivation of Boresch-style restraint parameters<sup><a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>.</p>
<p id="Par56">In-house Python scripts leveraging MDTraj v1.10.3<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup> were used to analyze MD trajectories and compute root-mean-square deviations (RMSD). For each isoform, RMSD was calculated for the binding site backbone atoms, defined as all residues within 4â€‰Ã… of BLU-945 in the reference structure (PDB: 8D76 chain A). The selected residues included: K716, L718, F723, V726, K728, A743, K745, E762, M766, C775, M790, Q791, L792, M793, P794, G796, L844, T854, D855, and D1006.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Free energy calculations</h3>
<p id="Par57">Absolute binding free energy perturbation (ABFEP) simulations were conducted using a standard ligand annihilation approach. For the binding leg of the calculations, 37 intermediate Î» windows were employed, while 31 windows were used for the solvation leg. Each simulation was independently run in triplicate to ensure reproducibility and to estimate statistical uncertainty.</p>
<p id="Par58">For the binding simulations, each window included 3â€‰ns of equilibration followed by 5â€‰ns of production dynamics, using a 4â€‰fs timestep. In the solvation simulations, each window consisted of 0.2â€‰ns of equilibration and 2.5â€‰ns of production, run with a 5â€‰fs timestep.</p>
<p id="Par59">Free energy differences were estimated using the multistate Bennett acceptance ratio (MBAR) method. Convergence was assessed based on time-series stability of the free energy estimates, the standard deviation across the triplicate runs, and overlap matrices to confirm sufficient phase space sampling between adjacent windows.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Western blotting</h3>
<p id="Par60">Cells were lysed in RIPA buffer (Thermo Fisher #89900) supplemented with 1Ã— Halt protease inhibitor cocktail (Thermo Fisher #78437) and phosphatase inhibitor cocktail (Thermo Fisher #78420). Total protein concentrations were determined using the BCA assay (Pierce). For each sample, 25â€‰Âµg of total protein was mixed with 6Ã— Laemmli SDS sample buffer (VWR, AAJ61337-AC) and heated at 95â€‰Â°C for 5â€‰min. Samples were then separated on 4â€“12% gradient Bis-Tris gel and transferred onto PVDF membranes.</p>
<p id="Par61">Membranes were blocked in 5% nonfat milk or 3% BSA (for phospho-specific detection) in TBS-T (20â€‰mM Tris-HCl, 150â€‰mM NaCl, 0.1% Tween-20) for 1â€‰h at room temperature. The blots were then incubated overnight at 4â€‰Â°C with primary antibodies against total EGFR (Cell Signaling Technology #2232S), phospho-EGFR (CST #3777S), total ERK (CST #9102S), phospho-ERK (CST #9101S), and cyclophilin B (CST #43603S), following the manufacturersâ€™ recommended dilutions. After three washes in TBS-T, membranes were incubated with the HRP-conjugated secondary antibodies (Invitrogen #31460) at 1:10,000 for 1â€‰h at room temperature. Following three additional TBS-T washes, protein bands were visualized using the chemiluminescence substrate (Thermo Fisher #34577) and detected by the Invitrogen iBright imaging system.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Cell viability assays</h3>
<p id="Par62">Ba/F3 cells were counted, adjusted to 20,000 cells/mL, and 25â€‰ÂµL (500 cells) was seeded into each well of a half-area 96-well plate (Corning #3688). Drug stocks were prepared at 2Ã— concentrations using a 4-fold serial dilution (100â€‰nM down to 0.0244â€‰nM) in medium containing DMSO. Each drug concentration (including the DMSO-only control) was tested in triplicate by dispensing 25â€‰ÂµL of the 2Ã— dilution into separate wells, yielding a final well volume of 50â€‰ÂµL at the 1Ã— drug concentration. Plates were incubated at 37â€‰Â°C in 5% CO<sub>2</sub> for 72â€‰h, after which 25â€‰ÂµL of CellTiter-Glo 2.0 reagent (Promega #G9242) was added per well. Following a 10-minute incubation at room temperature with gentle shaking, luminescence endpoint was measured on a microplate reader. Viability values were normalized to the DMSO control and used to generate doseâ€“response curves for IC<sub>50</sub> determination using GraphPad Prism (Dotmatics).</p></section><section id="Sec20"><h3 class="pmc_sec_title">Circulating tumor DNA sequencing</h3>
<p id="Par63">Peripheral blood was drawn at the NYU Perlmutter Cancer Center with the approval of the institutional review board (NYU IRB#S18-00980) and in accordance with the Declaration of Helsinki. Blood was collected in Streck Cell-Free DNA blood collection tubes, and samples were mixed gently per manufacturer's instructions, transported at 4â€‰Â°C, processed to aliquot serum, and subsequently stored at âˆ’80â€‰Â°C. Circulating tumor DNA sequencing was performed using the FoundationOneÂ® Monitor (F1M) assay (Foundation Medicine, Inc.), a tissue-naÃ¯ve method designed to monitor therapy response by quantifying ctDNA tumor fraction and variant allele frequency (VAF).</p></section></section><section id="Sec21"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12368022/bin/41698_2025_1086_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary document V3</a><sup> (21.1MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work was partially funded by Blueprint Medicines, who also provided BLU-945.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization Y.W., J.T.P. Methodology Y.W., J.T.P., T.M.A. Software Y.H., M.H., J.T.P., L.T., H.S. Validation Y.W. Formal analysis Y.H. Investigation Y.W., M.R., A.L., F.S., E.S., T.M.A., L.T. Resources K.K.W., J.T.P. Data Curation Y.W., Y.H. Writingâ€”Original Draft Y.W., E.S., J.T.P., T.M.A. Writingâ€”Review and Editing Y.W., M.H., X.T.R.M., J.D., E.S., J.T.P., T.M.A., L.T.V. isualization Y.W., Y.H., J.T.P., X.T.R.M., T.M.A., L.T. Supervision J.D., K.K.W., E.S., J.T.P., H.S., G.P. Project administration J.T.P. Funding acquisition K.K.W., J.T.P.</p></section><section id="notes2"><h2 class="pmc_sec_title">Data availability</h2>
<p>Sequencing data that support the findings of this study is available at the Gene Expression Omnibus (GEO), a public functional genomics data repository (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE305057" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE305057</a>).</p></section><section id="FPar1"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par64">K.K.W. and J.T.P. have received research funding from Blueprint Medicines. The authors declare no non-financial interest.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisherâ€™s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec22"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41698-025-01086-2.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Midha, A., Dearden, S. &amp; McCormack, R. Mutation incidence in non-small-cell lung cancer of adenocarcinoma histology: a systematic review and global map by ethnicity (mutMapII). <em>Am. J. Cancer Res.</em><strong>5</strong>, 2892-+ (2015).
</cite> [<a href="/articles/PMC4633915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26609494/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Midha,%20A.,%20Dearden,%20S.%20&amp;%20McCormack,%20R.%20Mutation%20incidence%20in%20non-small-cell%20lung%20cancer%20of%20adenocarcinoma%20histology:%20a%20systematic%20review%20and%20global%20map%20by%20ethnicity%20(mutMapII).%20Am.%20J.%20Cancer%20Res.5,%202892-+%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Crino, L., Weder, W., van Meerbeeck, J. &amp; Felip, E. Group EGW. Early stage and locally advanced (non-metastatic) non-small-cell lung cancer: ESMO Clinical Practice Guidelines for diagnosis, treatment and follow-up. <em>Ann. Oncol.</em><strong>21</strong>, v103â€“v115 (2010).
</cite> [<a href="https://doi.org/10.1093/annonc/mdq207" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20555058/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Crino,%20L.,%20Weder,%20W.,%20van%20Meerbeeck,%20J.%20&amp;%20Felip,%20E.%20Group%20EGW.%20Early%20stage%20and%20locally%20advanced%20(non-metastatic)%20non-small-cell%20lung%20cancer:%20ESMO%20Clinical%20Practice%20Guidelines%20for%20diagnosis,%20treatment%20and%20follow-up.%20Ann.%20Oncol.21,%20v103%E2%80%93v115%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Dearden, S., Stevens, J., Wu, Y. L. &amp; Blowers, D. Mutation incidence and coincidence in non small-cell lung cancer: meta-analyses by ethnicity and histology (mutMap). <em>Ann. Oncol.</em><strong>24</strong>, 2371â€“2376 (2013).
</cite> [<a href="https://doi.org/10.1093/annonc/mdt205" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3755331/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23723294/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dearden,%20S.,%20Stevens,%20J.,%20Wu,%20Y.%20L.%20&amp;%20Blowers,%20D.%20Mutation%20incidence%20and%20coincidence%20in%20non%20small-cell%20lung%20cancer:%20meta-analyses%20by%20ethnicity%20and%20histology%20(mutMap).%20Ann.%20Oncol.24,%202371%E2%80%932376%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Smits, A. J. J. et al. EGFR and KRAS mutations in lung carcinomas in the Dutch population: increased mutation frequency in malignant pleural effusion of lung adenocarcinoma. <em>Cell Oncol.</em><strong>35</strong>, 189â€“196 (2012).</cite> [<a href="https://doi.org/10.1007/s13402-012-0078-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3396345/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22528563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smits,%20A.%20J.%20J.%20et%20al.%20EGFR%20and%20KRAS%20mutations%20in%20lung%20carcinomas%20in%20the%20Dutch%20population:%20increased%20mutation%20frequency%20in%20malignant%20pleural%20effusion%20of%20lung%20adenocarcinoma.%20Cell%20Oncol.35,%20189%E2%80%93196%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Li, K. D., Yang, M. J., Liang, N. X. &amp; Li, S. Q. Determining EGFR-TKI sensitivity of G719X and other uncommon EGFR mutations in non-small cell lung cancer: perplexity and solution (Review). <em>Oncol. Rep.</em><strong>37</strong>, 1347â€“1358 (2017).
</cite> [<a href="https://doi.org/10.3892/or.2017.5409" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5364853/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28184913/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20K.%20D.,%20Yang,%20M.%20J.,%20Liang,%20N.%20X.%20&amp;%20Li,%20S.%20Q.%20Determining%20EGFR-TKI%20sensitivity%20of%20G719X%20and%20other%20uncommon%20EGFR%20mutations%20in%20non-small%20cell%20lung%20cancer:%20perplexity%20and%20solution%20(Review).%20Oncol.%20Rep.37,%201347%E2%80%931358%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Harrison, P. T., Vyse, S. &amp; Huang, P. H. Rare epidermal growth factor receptor (EGFR) mutations in non-small cell lung cancer. <em>Semin Cancer Biol.</em><strong>61</strong>, 167â€“179 (2020).
</cite> [<a href="https://doi.org/10.1016/j.semcancer.2019.09.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7083237/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31562956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harrison,%20P.%20T.,%20Vyse,%20S.%20&amp;%20Huang,%20P.%20H.%20Rare%20epidermal%20growth%20factor%20receptor%20(EGFR)%20mutations%20in%20non-small%20cell%20lung%20cancer.%20Semin%20Cancer%20Biol.61,%20167%E2%80%93179%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Eck, M. J. &amp; Yun, C. H. Structural and mechanistic underpinnings of the differential drug sensitivity of EGFR mutations in non-small cell lung cancer. <em>Biochim Biophys. Acta</em><strong>1804</strong>, 559â€“566 (2010).
</cite> [<a href="https://doi.org/10.1016/j.bbapap.2009.12.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2859716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20026433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eck,%20M.%20J.%20&amp;%20Yun,%20C.%20H.%20Structural%20and%20mechanistic%20underpinnings%20of%20the%20differential%20drug%20sensitivity%20of%20EGFR%20mutations%20in%20non-small%20cell%20lung%20cancer.%20Biochim%20Biophys.%20Acta1804,%20559%E2%80%93566%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Uribe M. L., Marrocco I., Yarden Y. EGFR in cancer: signaling mechanisms, drugs, and acquired resistance. <em>Cancers</em>. 13. Epub 20210601. 10.3390/cancers13112748 (2021).</cite> [<a href="https://doi.org/10.3390/cancers13112748" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8197917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34206026/" class="usa-link">PubMed</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Lynch, T. J. et al. Activating mutations in the epidermal growth factor receptor underlying responsiveness of non-small-cell lung cancer to gefitinib. <em>N. Engl. J. Med.</em><strong>350</strong>, 2129â€“2139 (2004).
</cite> [<a href="https://doi.org/10.1056/NEJMoa040938" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15118073/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lynch,%20T.%20J.%20et%20al.%20Activating%20mutations%20in%20the%20epidermal%20growth%20factor%20receptor%20underlying%20responsiveness%20of%20non-small-cell%20lung%20cancer%20to%20gefitinib.%20N.%20Engl.%20J.%20Med.350,%202129%E2%80%932139%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Paez, J. G. et al. EGFR mutations in lung cancer: correlation with clinical response to gefitinib therapy. <em>Science</em><strong>304</strong>, 1497â€“1500 (2004).
</cite> [<a href="https://doi.org/10.1126/science.1099314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15118125/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Paez,%20J.%20G.%20et%20al.%20EGFR%20mutations%20in%20lung%20cancer:%20correlation%20with%20clinical%20response%20to%20gefitinib%20therapy.%20Science304,%201497%E2%80%931500%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Mok, T. S. et al. Gefitinib or carboplatin-paclitaxel in pulmonary adenocarcinoma. <em>N. Engl. J. Med.</em><strong>361</strong>, 947â€“957 (2009).
</cite> [<a href="https://doi.org/10.1056/NEJMoa0810699" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19692680/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mok,%20T.%20S.%20et%20al.%20Gefitinib%20or%20carboplatin-paclitaxel%20in%20pulmonary%20adenocarcinoma.%20N.%20Engl.%20J.%20Med.361,%20947%E2%80%93957%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Soria, J. C. et al. Investigators F. Osimertinib in untreated EGFR-mutated advanced non-small-cell lung cancer. <em>N. Engl. J. Med.</em><strong>378</strong>, 113â€“125 (2018).
</cite> [<a href="https://doi.org/10.1056/NEJMoa1713137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29151359/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Soria,%20J.%20C.%20et%20al.%20Investigators%20F.%20Osimertinib%20in%20untreated%20EGFR-mutated%20advanced%20non-small-cell%20lung%20cancer.%20N.%20Engl.%20J.%20Med.378,%20113%E2%80%93125%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>El-Rayes, B. F. &amp; LoRusso, P. M. Targeting the epidermal growth factor receptor. <em>Br. J. Cancer</em><strong>91</strong>, 418â€“424 (2004).
</cite> [<a href="https://doi.org/10.1038/sj.bjc.6601921" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2409851/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15238978/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?El-Rayes,%20B.%20F.%20&amp;%20LoRusso,%20P.%20M.%20Targeting%20the%20epidermal%20growth%20factor%20receptor.%20Br.%20J.%20Cancer91,%20418%E2%80%93424%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Wu S. G., Shih J. Y. Management of acquired resistance to EGFR TKI-targeted therapy in advanced non-small cell lung cancer. <em>Mol. Cancer</em>. <strong>17</strong>. 10.1186/s12943-018-0777-1 (2018).</cite> [<a href="https://doi.org/10.1186/s12943-018-0777-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5817870/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29455650/" class="usa-link">PubMed</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Pao, W. et al. Acquired resistance of lung adenocarcinomas to gefitinib or erlotinib is associated with a second mutation in the EGFR kinase domain. <em>Plos Med</em><strong>2</strong>, 225â€“235 (2005).</cite> [<a href="https://doi.org/10.1371/journal.pmed.0020073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC549606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15737014/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pao,%20W.%20et%20al.%20Acquired%20resistance%20of%20lung%20adenocarcinomas%20to%20gefitinib%20or%20erlotinib%20is%20associated%20with%20a%20second%20mutation%20in%20the%20EGFR%20kinase%20domain.%20Plos%20Med2,%20225%E2%80%93235%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Kobayashi, S. et al. EGFR mutation and resistance of non-small-cell lung cancer to gefitinib. <em>N. Engl. J. Med.</em><strong>352</strong>, 786â€“792 (2005).
</cite> [<a href="https://doi.org/10.1056/NEJMoa044238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15728811/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kobayashi,%20S.%20et%20al.%20EGFR%20mutation%20and%20resistance%20of%20non-small-cell%20lung%20cancer%20to%20gefitinib.%20N.%20Engl.%20J.%20Med.352,%20786%E2%80%93792%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Mok, T. S. et al. Investigators A. Osimertinib or platinum-pemetrexed in EGFR T790M-positive lung cancer. <em>N. Engl. J. Med.</em><strong>376</strong>, 629â€“640 (2017).
</cite> [<a href="https://doi.org/10.1056/NEJMoa1612674" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6762027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27959700/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mok,%20T.%20S.%20et%20al.%20Investigators%20A.%20Osimertinib%20or%20platinum-pemetrexed%20in%20EGFR%20T790M-positive%20lung%20cancer.%20N.%20Engl.%20J.%20Med.376,%20629%E2%80%93640%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Fu, K., Xie, F. C., Wang, F. &amp; Fu, L. W. Therapeutic strategies for EGFR-mutated non-small cell lung cancer patients with osimertinib resistance. <em>J. Hematol. Oncol.</em><strong>15</strong>, 173 (2022).
</cite> [<a href="https://doi.org/10.1186/s13045-022-01391-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9733018/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36482474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fu,%20K.,%20Xie,%20F.%20C.,%20Wang,%20F.%20&amp;%20Fu,%20L.%20W.%20Therapeutic%20strategies%20for%20EGFR-mutated%20non-small%20cell%20lung%20cancer%20patients%20with%20osimertinib%20resistance.%20J.%20Hematol.%20Oncol.15,%20173%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Leonetti, A. et al. Resistance mechanisms to osimertinib in EGFR-mutated non-small cell lung cancer. <em>Br. J. Cancer</em><strong>121</strong>, 725â€“737 (2019).
</cite> [<a href="https://doi.org/10.1038/s41416-019-0573-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6889286/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31564718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leonetti,%20A.%20et%20al.%20Resistance%20mechanisms%20to%20osimertinib%20in%20EGFR-mutated%20non-small%20cell%20lung%20cancer.%20Br.%20J.%20Cancer121,%20725%E2%80%93737%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Enrico, D. et al. Overcoming resistance to osimertinib by T790M loss and C797S acquisition using gefitinib in a patient with EGFR-mutant NSCLC: a case report. <em>Jto Clin. Res Rep.</em><strong>4</strong>, 100456 (2023).
</cite> [<a href="https://doi.org/10.1016/j.jtocrr.2022.100456" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9926104/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36798785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Enrico,%20D.%20et%20al.%20Overcoming%20resistance%20to%20osimertinib%20by%20T790M%20loss%20and%20C797S%20acquisition%20using%20gefitinib%20in%20a%20patient%20with%20EGFR-mutant%20NSCLC:%20a%20case%20report.%20Jto%20Clin.%20Res%20Rep.4,%20100456%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Suzuki, M. et al. A macrocyclic kinase inhibitor overcomes triple resistant mutations in EGFR-positive lung cancer. <em>Npj Precis Oncol.</em><strong>8</strong>, 46 (2024).
</cite> [<a href="https://doi.org/10.1038/s41698-024-00542-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10891166/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38396251/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Suzuki,%20M.%20et%20al.%20A%20macrocyclic%20kinase%20inhibitor%20overcomes%20triple%20resistant%20mutations%20in%20EGFR-positive%20lung%20cancer.%20Npj%20Precis%20Oncol.8,%2046%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Qin K., Hong L., Zhang J., Le X. MET amplification as a resistance driver to TKI therapies in lung cancer: clinical challenges and opportunities. <em>Cancers</em>. 15. Epub 20230118. 10.3390/cancers15030612.</cite> [<a href="https://doi.org/10.3390/cancers15030612" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9913224/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36765572/" class="usa-link">PubMed</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Leonetti, A. et al. Small cell lung cancer transformation as a resistance mechanism to osimertinib in epidermal growth factor receptor-mutated lung adenocarcinoma: case report and literature review. <em>Front. Oncol.</em><strong>11</strong>, 642190 (2021).
</cite> [<a href="https://doi.org/10.3389/fonc.2021.642190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8107466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33981604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leonetti,%20A.%20et%20al.%20Small%20cell%20lung%20cancer%20transformation%20as%20a%20resistance%20mechanism%20to%20osimertinib%20in%20epidermal%20growth%20factor%20receptor-mutated%20lung%20adenocarcinoma:%20case%20report%20and%20literature%20review.%20Front.%20Oncol.11,%20642190%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Ferrer, L. et al. A brief report of transformation from NSCLC to SCLC: molecular and therapeutic characteristics. <em>J. Thorac. Oncol.</em><strong>14</strong>, 130â€“134 (2019).
</cite> [<a href="https://doi.org/10.1016/j.jtho.2018.08.2028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30217489/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ferrer,%20L.%20et%20al.%20A%20brief%20report%20of%20transformation%20from%20NSCLC%20to%20SCLC:%20molecular%20and%20therapeutic%20characteristics.%20J.%20Thorac.%20Oncol.14,%20130%E2%80%93134%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Gunther, M., Juchum, M., Kelter, G., Fiebig, H. &amp; Laufer, S. Lung cancer: EGFR inhibitors with low nanomolar activity against a therapy-resistant L858R/T790M/C797S mutant. <em>Angew. Chem. Int. Ed. Engl.</em><strong>55</strong>, 10890â€“10894 (2016).
</cite> [<a href="https://doi.org/10.1002/anie.201603736" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27466205/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gunther,%20M.,%20Juchum,%20M.,%20Kelter,%20G.,%20Fiebig,%20H.%20&amp;%20Laufer,%20S.%20Lung%20cancer:%20EGFR%20inhibitors%20with%20low%20nanomolar%20activity%20against%20a%20therapy-resistant%20L858R/T790M/C797S%20mutant.%20Angew.%20Chem.%20Int.%20Ed.%20Engl.55,%2010890%E2%80%9310894%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Gunther, M. et al. Trisubstituted pyridinylimidazoles as potent inhibitors of the clinically resistant L858R/T790M/C797S EGFR mutant: targeting of both hydrophobic regions and the phosphate binding site. <em>J. Med. Chem.</em><strong>60</strong>, 5613â€“5637 (2017).
</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.7b00316" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28603991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gunther,%20M.%20et%20al.%20Trisubstituted%20pyridinylimidazoles%20as%20potent%20inhibitors%20of%20the%20clinically%20resistant%20L858R/T790M/C797S%20EGFR%20mutant:%20targeting%20of%20both%20hydrophobic%20regions%20and%20the%20phosphate%20binding%20site.%20J.%20Med.%20Chem.60,%205613%E2%80%935637%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Juchum, M. et al. Trisubstituted imidazoles with a rigidized hinge binding motif act as single digit nM inhibitors of clinically relevant EGFR L858R/T790M and L858R/T790M/C797S mutants: an example of target hopping. <em>J. Med. Chem.</em><strong>60</strong>, 4636â€“4656 (2017).
</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.7b00178" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28482151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Juchum,%20M.%20et%20al.%20Trisubstituted%20imidazoles%20with%20a%20rigidized%20hinge%20binding%20motif%20act%20as%20single%20digit%20nM%20inhibitors%20of%20clinically%20relevant%20EGFR%20L858R/T790M%20and%20L858R/T790M/C797S%20mutants:%20an%20example%20of%20target%20hopping.%20J.%20Med.%20Chem.60,%204636%E2%80%934656%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Patel, H., Pawara, R., Ansari, A. &amp; Surana, S. Recent updates on third generation EGFR inhibitors and emergence of fourth generation EGFR inhibitors to combat C797S resistance. <em>Eur. J. Med. Chem.</em><strong>142</strong>, 32â€“47 (2017).
</cite> [<a href="https://doi.org/10.1016/j.ejmech.2017.05.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28526474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Patel,%20H.,%20Pawara,%20R.,%20Ansari,%20A.%20&amp;%20Surana,%20S.%20Recent%20updates%20on%20third%20generation%20EGFR%20inhibitors%20and%20emergence%20of%20fourth%20generation%20EGFR%20inhibitors%20to%20combat%20C797S%20resistance.%20Eur.%20J.%20Med.%20Chem.142,%2032%E2%80%9347%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Fukuda, S. et al. Potential utility of a 4th-generation EGFR-TKI and exploration of resistance mechanisms-an in vitro study. <em>Biomedicines</em><strong>12</strong>, 1412 (2024).
</cite> [<a href="https://doi.org/10.3390/biomedicines12071412" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11273927/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39061985/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fukuda,%20S.%20et%20al.%20Potential%20utility%20of%20a%204th-generation%20EGFR-TKI%20and%20exploration%20of%20resistance%20mechanisms-an%20in%20vitro%20study.%20Biomedicines12,%201412%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Eno, M. S. et al. Discovery of BLU-945, a reversible, potent, and wild-type-sparing next-generation EGFR mutant inhibitor for treatment-resistant non-small-cell lung cancer. <em>J. Med. Chem.</em><strong>65</strong>, 9662â€“9677 (2022).
</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.2c00704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9340769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35838760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eno,%20M.%20S.%20et%20al.%20Discovery%20of%20BLU-945,%20a%20reversible,%20potent,%20and%20wild-type-sparing%20next-generation%20EGFR%20mutant%20inhibitor%20for%20treatment-resistant%20non-small-cell%20lung%20cancer.%20J.%20Med.%20Chem.65,%209662%E2%80%939677%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Elamin Y. Y. et al. BLU-945 monotherapy and in combination with osimertinib (OSI) in previously treated patients with advanced <em>EGFR</em>-mutant (<em>EGFRm</em>) NSCLC in the phase 1/2 SYMPHONY study. <em>J. Clin. Oncol</em>. <strong>41</strong>, 10.1200/JCO.2023.41.16_suppl.901146 (2023).</cite>
</li>
<li id="CR32">
<span class="label">32.</span><cite>Doud, M. B., Hensley, S. E. &amp; Bloom, J. D. Complete mapping of viral escape from neutralizing antibodies. <em>PLoS Pathog.</em><strong>13</strong>, e1006271 (2017).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1006271" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5363992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28288189/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Doud,%20M.%20B.,%20Hensley,%20S.%20E.%20&amp;%20Bloom,%20J.%20D.%20Complete%20mapping%20of%20viral%20escape%20from%20neutralizing%20antibodies.%20PLoS%20Pathog.13,%20e1006271%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Dingens, A. S., Haddox, H. K., Overbaugh, J. &amp; Bloom, J. D. Comprehensive mapping of HIV-1 escape from a broadly neutralizing antibody. <em>Cell Host Microbe</em><strong>21</strong>, 777-+ (2017).
</cite> [<a href="https://doi.org/10.1016/j.chom.2017.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5512576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28579254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dingens,%20A.%20S.,%20Haddox,%20H.%20K.,%20Overbaugh,%20J.%20&amp;%20Bloom,%20J.%20D.%20Comprehensive%20mapping%20of%20HIV-1%20escape%20from%20a%20broadly%20neutralizing%20antibody.%20Cell%20Host%20Microbe21,%20777-+%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Doud, M. B., Lee, J. M. &amp; Bloom, J. D. How single mutations affect viral escape from broad and narrow antibodies to H1 influenza hemagglutinin. <em>Nat. Commun.</em><strong>9</strong>, 1386 (2018).
</cite> [<a href="https://doi.org/10.1038/s41467-018-03665-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5895760/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29643370/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Doud,%20M.%20B.,%20Lee,%20J.%20M.%20&amp;%20Bloom,%20J.%20D.%20How%20single%20mutations%20affect%20viral%20escape%20from%20broad%20and%20narrow%20antibodies%20to%20H1%20influenza%20hemagglutinin.%20Nat.%20Commun.9,%201386%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Palacios, R. &amp; Steinmetz, M. Il-3-dependent mouse clones that express B-220 surface antigen, contain Ig genes in germ-line configuration, and generate B lymphocytes in vivo. <em>Cell</em><strong>41</strong>, 727â€“734 (1985).
</cite> [<a href="https://doi.org/10.1016/s0092-8674(85)80053-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3924409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palacios,%20R.%20&amp;%20Steinmetz,%20M.%20Il-3-dependent%20mouse%20clones%20that%20express%20B-220%20surface%20antigen,%20contain%20Ig%20genes%20in%20germ-line%20configuration,%20and%20generate%20B%20lymphocytes%20in%20vivo.%20Cell41,%20727%E2%80%93734%20(1985)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Doud, M. B. &amp; Bloom, J. D. Accurate measurement of the effects of all amino-acid mutations on influenza hemagglutinin. <em>Viruses Basel.</em><strong>8</strong>, 155 (2016).</cite> [<a href="https://doi.org/10.3390/v8060155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4926175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27271655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Doud,%20M.%20B.%20&amp;%20Bloom,%20J.%20D.%20Accurate%20measurement%20of%20the%20effects%20of%20all%20amino-acid%20mutations%20on%20influenza%20hemagglutinin.%20Viruses%20Basel.8,%20155%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Koga, T., Suda, K. &amp; Mitsudomi, T. Utility of the Ba/F3 cell system for exploring on-target mechanisms of resistance to targeted therapies for lung cancer. <em>Cancer Sci.</em><strong>113</strong>, 815â€“827 (2022).
</cite> [<a href="https://doi.org/10.1111/cas.15263" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8898722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34997674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koga,%20T.,%20Suda,%20K.%20&amp;%20Mitsudomi,%20T.%20Utility%20of%20the%20Ba/F3%20cell%20system%20for%20exploring%20on-target%20mechanisms%20of%20resistance%20to%20targeted%20therapies%20for%20lung%20cancer.%20Cancer%20Sci.113,%20815%E2%80%93827%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Wu, N. C. et al. High-throughput profiling of influenza A virus hemagglutinin gene at single-nucleotide resolution. <em>Sci. Rep.</em><strong>4</strong>, 4942 (2014).
</cite> [<a href="https://doi.org/10.1038/srep04942" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4018626/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24820965/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20N.%20C.%20et%20al.%20High-throughput%20profiling%20of%20influenza%20A%20virus%20hemagglutinin%20gene%20at%20single-nucleotide%20resolution.%20Sci.%20Rep.4,%204942%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Zhang, T. H., Wu, N. C. &amp; Sun, R. A benchmark study on error-correction by read-pairing and tag-clustering in amplicon-based deep sequencing. <em>BMC Genomics.</em><strong>17</strong>, 108 (2016).
</cite> [<a href="https://doi.org/10.1186/s12864-016-2388-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4751728/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26868371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20T.%20H.,%20Wu,%20N.%20C.%20&amp;%20Sun,%20R.%20A%20benchmark%20study%20on%20error-correction%20by%20read-pairing%20and%20tag-clustering%20in%20amplicon-based%20deep%20sequencing.%20BMC%20Genomics.17,%20108%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Damghani, T. et al. Structural elements that enable specificity for mutant EGFR kinase domains with next-generation small-molecule inhibitors. <em>Methods Enzymol.</em><strong>685</strong>, 171â€“198 (2023).
</cite> [<a href="https://doi.org/10.1016/bs.mie.2023.03.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10445336/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37245901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Damghani,%20T.%20et%20al.%20Structural%20elements%20that%20enable%20specificity%20for%20mutant%20EGFR%20kinase%20domains%20with%20next-generation%20small-molecule%20inhibitors.%20Methods%20Enzymol.685,%20171%E2%80%93198%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Vu, H. A. et al. Spectrum of EGFR gene mutations in Vietnamese patients with non-small cell lung cancer. <em>Asia Pac. J. Clin. Oncol.</em><strong>12</strong>, 86â€“90 (2016).
</cite> [<a href="https://doi.org/10.1111/ajco.12448" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26707566/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vu,%20H.%20A.%20et%20al.%20Spectrum%20of%20EGFR%20gene%20mutations%20in%20Vietnamese%20patients%20with%20non-small%20cell%20lung%20cancer.%20Asia%20Pac.%20J.%20Clin.%20Oncol.12,%2086%E2%80%9390%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Liu, Y. et al. Acquired EGFR L718V mutation mediates resistance to osimertinib in non-small cell lung cancer but retains sensitivity to afatinib. <em>Lung Cancer</em><strong>118</strong>, 1â€“5 (2018).
</cite> [<a href="https://doi.org/10.1016/j.lungcan.2018.01.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29571986/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20Y.%20et%20al.%20Acquired%20EGFR%20L718V%20mutation%20mediates%20resistance%20to%20osimertinib%20in%20non-small%20cell%20lung%20cancer%20but%20retains%20sensitivity%20to%20afatinib.%20Lung%20Cancer118,%201%E2%80%935%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Sharma, P. et al. A Novel EGFR germline mutation in lung adenocarcinoma: case report and literature review. <em>Clin. Lung Cancer</em><strong>25</strong>, 479â€“482 (2024).
</cite> [<a href="https://doi.org/10.1016/j.cllc.2024.04.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38777674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sharma,%20P.%20et%20al.%20A%20Novel%20EGFR%20germline%20mutation%20in%20lung%20adenocarcinoma:%20case%20report%20and%20literature%20review.%20Clin.%20Lung%20Cancer25,%20479%E2%80%93482%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Mao, L. et al. Mutation spectrum of EGFR From 21,324 Chinese patients with non-small cell lung cancer (NSCLC) successfully tested by multiple methods in a CAP-accredited laboratory. <em>Pathol. Oncol. Res.</em><strong>27</strong>, 602726 (2021).
</cite> [<a href="https://doi.org/10.3389/pore.2021.602726" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8262202/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34257561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mao,%20L.%20et%20al.%20Mutation%20spectrum%20of%20EGFR%20From%2021,324%20Chinese%20patients%20with%20non-small%20cell%20lung%20cancer%20(NSCLC)%20successfully%20tested%20by%20multiple%20methods%20in%20a%20CAP-accredited%20laboratory.%20Pathol.%20Oncol.%20Res.27,%20602726%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Si, J. F., Gu, X. D., Wang, W. X., Ying, S. P. &amp; Song, Z. B. Clinical outcomes of lung adenocarcinoma patients harboring uncommon epidermal growth factor receptor () mutations treated with EGFR-tyrosine kinase inhibitors (TKIs). <em>Ann. Palliat. Med.</em><strong>11</strong>, 1624â€“1634 (2022).
</cite> [<a href="https://doi.org/10.21037/apm-21-2828" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35016520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Si,%20J.%20F.,%20Gu,%20X.%20D.,%20Wang,%20W.%20X.,%20Ying,%20S.%20P.%20&amp;%20Song,%20Z.%20B.%20Clinical%20outcomes%20of%20lung%20adenocarcinoma%20patients%20harboring%20uncommon%20epidermal%20growth%20factor%20receptor%20()%20mutations%20treated%20with%20EGFR-tyrosine%20kinase%20inhibitors%20(TKIs).%20Ann.%20Palliat.%20Med.11,%201624%E2%80%931634%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Ruiz R. et al. Mutation pattern of EGFR gene in Peruvian patients with non-small cell lung cancer. <em>J. Clin. Oncol</em>. <strong>42</strong>, 10.1200/JCO.2024.42.16_suppl.e20615 (2024).</cite>
</li>
<li id="CR47">
<span class="label">47.</span><cite>Ma, C. H. et al. Efficacy of afatinib for pulmonary adenocarcinoma with leptomeningeal metastases harboring an epidermal growth factor receptor complex mutation (exon 19delâ€‰+â€‰K754E). A case report. <em>Medicine</em><strong>99</strong>, e22851 (2020).
</cite> [<a href="https://doi.org/10.1097/MD.0000000000022851" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7581162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33120820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ma,%20C.%20H.%20et%20al.%20Efficacy%20of%20afatinib%20for%20pulmonary%20adenocarcinoma%20with%20leptomeningeal%20metastases%20harboring%20an%20epidermal%20growth%20factor%20receptor%20complex%20mutation%20(exon%2019del%E2%80%89+%E2%80%89K754E).%20A%20case%20report.%20Medicine99,%20e22851%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Miyamae, Y. et al. Mutation detection of epidermal growth factor receptor and genes using the smart amplification process version 2 from formalin-fixed, paraffin-embedded lung cancer tissue. <em>J. Mol. Diagn.</em><strong>12</strong>, 257â€“264 (2010).
</cite> [<a href="https://doi.org/10.2353/jmoldx.2010.090105" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2871734/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20093389/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miyamae,%20Y.%20et%20al.%20Mutation%20detection%20of%20epidermal%20growth%20factor%20receptor%20and%20genes%20using%20the%20smart%20amplification%20process%20version%202%20from%20formalin-fixed,%20paraffin-embedded%20lung%20cancer%20tissue.%20J.%20Mol.%20Diagn.12,%20257%E2%80%93264%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Tang, H. R. et al. EGFR mutations in small cell lung cancer (SCLC): genetic heterogeneity and prognostic impact. <em>J. Thorac. Oncol.</em><strong>12</strong>, S710â€“S711 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Tang,%20H.%20R.%20et%20al.%20EGFR%20mutations%20in%20small%20cell%20lung%20cancer%20(SCLC):%20genetic%20heterogeneity%20and%20prognostic%20impact.%20J.%20Thorac.%20Oncol.12,%20S710%E2%80%93S711%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Robichaux, J. P. et al. Structure-based classification predicts drug response in EGFR-mutant NSCLC. <em>Nature</em><strong>597</strong>, 732â€“737 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03898-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8481125/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34526717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Robichaux,%20J.%20P.%20et%20al.%20Structure-based%20classification%20predicts%20drug%20response%20in%20EGFR-mutant%20NSCLC.%20Nature597,%20732%E2%80%93737%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Carrera, A. C., Alexandrov, K. &amp; Roberts, T. M. The conserved lysine of the catalytic domain of protein kinases is actively involved in the phosphotransfer reaction and not required for anchoring ATP. <em>Proc. Natl. Acad. Sci. USA</em><strong>90</strong>, 442â€“446 (1993).
</cite> [<a href="https://doi.org/10.1073/pnas.90.2.442" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC45679/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8421674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carrera,%20A.%20C.,%20Alexandrov,%20K.%20&amp;%20Roberts,%20T.%20M.%20The%20conserved%20lysine%20of%20the%20catalytic%20domain%20of%20protein%20kinases%20is%20actively%20involved%20in%20the%20phosphotransfer%20reaction%20and%20not%20required%20for%20anchoring%20ATP.%20Proc.%20Natl.%20Acad.%20Sci.%20USA90,%20442%E2%80%93446%20(1993)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Rotow, J. K. et al. Real-world genomic profile of EGFR second-site mutations and other osimertinib resistance mechanisms and clinical landscape of NSCLC post-osimertinib. <em>J. Thorac. Oncol.</em><strong>19</strong>, 227â€“239 (2024).
</cite> [<a href="https://doi.org/10.1016/j.jtho.2023.09.1453" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37806383/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rotow,%20J.%20K.%20et%20al.%20Real-world%20genomic%20profile%20of%20EGFR%20second-site%20mutations%20and%20other%20osimertinib%20resistance%20mechanisms%20and%20clinical%20landscape%20of%20NSCLC%20post-osimertinib.%20J.%20Thorac.%20Oncol.19,%20227%E2%80%93239%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Li, M. et al. L718Q/V mutation in exon 18 of EGFR mediates resistance to osimertinib: clinical features and treatment. <em>Discov. Oncol.</em><strong>13</strong>, 72 (2022).
</cite> [<a href="https://doi.org/10.1007/s12672-022-00537-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9363540/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35943592/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20M.%20et%20al.%20L718Q/V%20mutation%20in%20exon%2018%20of%20EGFR%20mediates%20resistance%20to%20osimertinib:%20clinical%20features%20and%20treatment.%20Discov.%20Oncol.13,%2072%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Starrett, J. H. et al. Drug sensitivity and allele specificity of first-line osimertinib resistance mutations. <em>Cancer Res.</em><strong>80</strong>, 2017â€“2030 (2020).
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-19-3819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7392201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32193290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Starrett,%20J.%20H.%20et%20al.%20Drug%20sensitivity%20and%20allele%20specificity%20of%20first-line%20osimertinib%20resistance%20mutations.%20Cancer%20Res.80,%202017%E2%80%932030%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Morgillo, F., Della Corte, C. M., Fasano, M. &amp; Ciardiello, F. Mechanisms of resistance to EGFR-targeted drugs: lung cancer. <em>Esmo Open.</em><strong>1</strong>, e000060 (2016).
</cite> [<a href="https://doi.org/10.1136/esmoopen-2016-000060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5070275/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27843613/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Morgillo,%20F.,%20Della%20Corte,%20C.%20M.,%20Fasano,%20M.%20&amp;%20Ciardiello,%20F.%20Mechanisms%20of%20resistance%20to%20EGFR-targeted%20drugs:%20lung%20cancer.%20Esmo%20Open.1,%20e000060%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Schneider, W. M. et al. Genome-scale identification of SARS-CoV-2 and pan-coronavirus host factor networks. <em>Cell</em><strong>184</strong>, 120â€“32.e14 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.12.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7796900/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33382968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schneider,%20W.%20M.%20et%20al.%20Genome-scale%20identification%20of%20SARS-CoV-2%20and%20pan-coronavirus%20host%20factor%20networks.%20Cell184,%20120%E2%80%9332.e14%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Maier, J. A. et al. ff14SB: improving the accuracy of protein side chain and backbone parameters from ff99SB. <em>J. Chem. Theory Comput.</em><strong>11</strong>, 3696â€“3713 (2015).
</cite> [<a href="https://doi.org/10.1021/acs.jctc.5b00255" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4821407/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26574453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maier,%20J.%20A.%20et%20al.%20ff14SB:%20improving%20the%20accuracy%20of%20protein%20side%20chain%20and%20backbone%20parameters%20from%20ff99SB.%20J.%20Chem.%20Theory%20Comput.11,%203696%E2%80%933713%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Jakalian, A., Jack, D. B. &amp; Bayly, C. I. Fast, efficient generation of high-quality atomic charges. AM1-BCC model: II. Parameterization and validation. <em>J. Comput Chem.</em><strong>23</strong>, 1623â€“1641 (2002).
</cite> [<a href="https://doi.org/10.1002/jcc.10128" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12395429/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jakalian,%20A.,%20Jack,%20D.%20B.%20&amp;%20Bayly,%20C.%20I.%20Fast,%20efficient%20generation%20of%20high-quality%20atomic%20charges.%20AM1-BCC%20model:%20II.%20Parameterization%20and%20validation.%20J.%20Comput%20Chem.23,%201623%E2%80%931641%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Jorgensen, W. L., Chandrasekhar, J., Madura, J. D., Impey, R. W. &amp; Klein, M. L. Comparison of simple potential functions for simulating liquid water. <em>J. Chem. Phys.</em><strong>79</strong>, 926â€“935 (1983).</cite> [<a href="https://scholar.google.com/scholar_lookup?Jorgensen,%20W.%20L.,%20Chandrasekhar,%20J.,%20Madura,%20J.%20D.,%20Impey,%20R.%20W.%20&amp;%20Klein,%20M.%20L.%20Comparison%20of%20simple%20potential%20functions%20for%20simulating%20liquid%20water.%20J.%20Chem.%20Phys.79,%20926%E2%80%93935%20(1983)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Hopkins, C. W., Le Grand, S., Walker, R. C. &amp; Roitberg, A. E. Long-time-step molecular dynamics through hydrogen mass repartitioning. <em>J. Chem. Theory Comput.</em><strong>11</strong>, 1864â€“1874 (2015).
</cite> [<a href="https://doi.org/10.1021/ct5010406" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26574392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hopkins,%20C.%20W.,%20Le%20Grand,%20S.,%20Walker,%20R.%20C.%20&amp;%20Roitberg,%20A.%20E.%20Long-time-step%20molecular%20dynamics%20through%20hydrogen%20mass%20repartitioning.%20J.%20Chem.%20Theory%20Comput.11,%201864%E2%80%931874%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Leimkuhler B., Matthews C. Rational Construction of Stochastic Numerical Methods for Molecular Sampling. <em>Appl. Math. Res. Expres</em>. 34-56. 10.1093/amrx/abs010. (2013).</cite>
</li>
<li id="CR62">
<span class="label">62.</span><cite>Ã…qvist, J., WennerstrÃ¶m, P., Nervall, M., Bjelic, S. &amp; Brandsdal, B. O. Molecular dynamics simulations of water and biomolecules with a Monte Carlo constant pressure algorithm. <em>Chem. Phys. Lett.</em><strong>384</strong>, 288â€“294 (2004).</cite> [<a href="https://scholar.google.com/scholar_lookup?%C3%85qvist,%20J.,%20Wennerstr%C3%B6m,%20P.,%20Nervall,%20M.,%20Bjelic,%20S.%20&amp;%20Brandsdal,%20B.%20O.%20Molecular%20dynamics%20simulations%20of%20water%20and%20biomolecules%20with%20a%20Monte%20Carlo%20constant%20pressure%20algorithm.%20Chem.%20Phys.%20Lett.384,%20288%E2%80%93294%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Boresch, S., Tettinger, F., Leitgeb, M. &amp; Karplus, M. Absolute binding free energies: a quantitative approach for their calculation. <em>J. Phys. Chem. B.</em><strong>107</strong>, 9535â€“9551 (2003).</cite> [<a href="https://scholar.google.com/scholar_lookup?Boresch,%20S.,%20Tettinger,%20F.,%20Leitgeb,%20M.%20&amp;%20Karplus,%20M.%20Absolute%20binding%20free%20energies:%20a%20quantitative%20approach%20for%20their%20calculation.%20J.%20Phys.%20Chem.%20B.107,%209535%E2%80%939551%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>McGibbon, R. T. et al. MDTraj: a modern open library for the analysis of molecular dynamics trajectories. <em>Biophys. J.</em><strong>109</strong>, 1528â€“1532 (2015).
</cite> [<a href="https://doi.org/10.1016/j.bpj.2015.08.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4623899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26488642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McGibbon,%20R.%20T.%20et%20al.%20MDTraj:%20a%20modern%20open%20library%20for%20the%20analysis%20of%20molecular%20dynamics%20trajectories.%20Biophys.%20J.109,%201528%E2%80%931532%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12368022/bin/41698_2025_1086_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary document V3</a><sup> (21.1MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Sequencing data that support the findings of this study is available at the Gene Expression Omnibus (GEO), a public functional genomics data repository (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE305057" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE305057</a>).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from NPJ Precision Oncology are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41698-025-01086-2"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41698_2025_Article_1086.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.9Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12368022/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12368022/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12368022%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12368022/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12368022/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12368022/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40836025/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12368022/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40836025/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12368022/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12368022/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="l02vo1jsD140pp9WY1GwtO2e2oqXcPyyE157Js1Rt4CEX5fjwAy49mkHot19L9lr">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
