
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Structural basis of a human antibody targeting SARS-CoV-2 nucleocapsid protein dimerization domain and interfering with RNA-binding - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152552628AF438330552620000203CDA.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="commbiol">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365033/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Communications Biology">
<meta name="citation_title" content="Structural basis of a human antibody targeting SARS-CoV-2 nucleocapsid protein dimerization domain and interfering with RNA-binding">
<meta name="citation_author" content="Shaojie Xue">
<meta name="citation_author_institution" content="Molecular Imaging Center, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China">
<meta name="citation_author" content="Suhua He">
<meta name="citation_author_institution" content="Guangzhou National Laboratory, Guangzhou, Guangdong China">
<meta name="citation_author" content="Zhaoxia Huang">
<meta name="citation_author_institution" content="Molecular Imaging Center, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China">
<meta name="citation_author" content="Mei Yang">
<meta name="citation_author_institution" content="Molecular Imaging Center, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China">
<meta name="citation_author" content="Guowei Hu">
<meta name="citation_author_institution" content="Molecular Imaging Center, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China">
<meta name="citation_author" content="Xiaoxue Chen">
<meta name="citation_author_institution" content="Department of Pharmacy, Sun-Yat-Sen Memorial Hospital, Sun Yat-sen University, Guangzhou, Guangdong China">
<meta name="citation_author" content="Qiuyue Chen">
<meta name="citation_author_institution" content="Department of Radiology, The Fifth Affiliated Hospital of Sun Yat-sen University, Zhuhai, Guangdong China">
<meta name="citation_author" content="Wenying Zhou">
<meta name="citation_author_institution" content="Central Laboratory, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China">
<meta name="citation_author" content="Sixiao Lin">
<meta name="citation_author_institution" content="Central Laboratory, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China">
<meta name="citation_author" content="Shoudeng Chen">
<meta name="citation_author_institution" content="Molecular Imaging Center, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China">
<meta name="citation_author_institution" content="Central Laboratory, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China">
<meta name="citation_author_institution" content="Guangdong-Hong Kong-Macao University Joint Laboratory of Interventional Medicine, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China">
<meta name="citation_author_institution" content="Guangdong Provincial Engineering Research Center of Molecular Imaging, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="8">
<meta name="citation_firstpage" content="1248">
<meta name="citation_doi" content="10.1038/s42003-025-08648-x">
<meta name="citation_pmid" content="40830575">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365033/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365033/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365033/pdf/42003_2025_Article_8648.pdf">
<meta name="description" content="The transition of SARS-CoV-2 into a recurrent, seasonal pathogen has underscored the need for the induction of durable immune protection. The nucleocapsid (N) protein is regarded as a promising complementary target for therapeutic and vaccine ...">
<meta name="og:title" content="Structural basis of a human antibody targeting SARS-CoV-2 nucleocapsid protein dimerization domain and interfering with RNA-binding">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The transition of SARS-CoV-2 into a recurrent, seasonal pathogen has underscored the need for the induction of durable immune protection. The nucleocapsid (N) protein is regarded as a promising complementary target for therapeutic and vaccine ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365033/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365033">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s42003-025-08648-x"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/42003_2025_Article_8648.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365033%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365033/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365033/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365033/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-commbiol.jpg" alt="Communications Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Communications Biology" title="Link to Communications Biology" shape="default" href="https://www.nature.com/commsbio/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Commun Biol</button></div>. 2025 Aug 19;8:1248. doi: <a href="https://doi.org/10.1038/s42003-025-08648-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s42003-025-08648-x</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Commun%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Commun%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Commun%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Commun%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Structural basis of a human antibody targeting SARS-CoV-2 nucleocapsid protein dimerization domain and interfering with RNA-binding</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xue%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Shaojie Xue</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Shaojie Xue</span></h3>
<div class="p">
<sup>1</sup>Molecular Imaging Center, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xue%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shaojie Xue</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Suhua He</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Suhua He</span></h3>
<div class="p">
<sup>2</sup>Guangzhou National Laboratory, Guangzhou, Guangdong China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Suhua He</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Zhaoxia Huang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Zhaoxia Huang</span></h3>
<div class="p">
<sup>1</sup>Molecular Imaging Center, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhaoxia Huang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Mei Yang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Mei Yang</span></h3>
<div class="p">
<sup>1</sup>Molecular Imaging Center, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mei Yang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Guowei Hu</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Guowei Hu</span></h3>
<div class="p">
<sup>1</sup>Molecular Imaging Center, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Guowei Hu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Xiaoxue Chen</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Xiaoxue Chen</span></h3>
<div class="p">
<sup>3</sup>Department of Pharmacy, Sun-Yat-Sen Memorial Hospital, Sun Yat-sen University, Guangzhou, Guangdong China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaoxue Chen</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Qiuyue Chen</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Qiuyue Chen</span></h3>
<div class="p">
<sup>4</sup>Department of Radiology, The Fifth Affiliated Hospital of Sun Yat-sen University, Zhuhai, Guangdong China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qiuyue Chen</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Wenying Zhou</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Wenying Zhou</span></h3>
<div class="p">
<sup>5</sup>Central Laboratory, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenying Zhou</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Sixiao Lin</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Sixiao Lin</span></h3>
<div class="p">
<sup>5</sup>Central Laboratory, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lin%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sixiao Lin</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Shoudeng Chen</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Shoudeng Chen</span></h3>
<div class="p">
<sup>1</sup>Molecular Imaging Center, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China </div>
<div class="p">
<sup>5</sup>Central Laboratory, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China </div>
<div class="p">
<sup>6</sup>Guangdong-Hong Kong-Macao University Joint Laboratory of Interventional Medicine, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China </div>
<div class="p">
<sup>7</sup>Guangdong Provincial Engineering Research Center of Molecular Imaging, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shoudeng Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>5,</sup><sup>6,</sup><sup>7,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Molecular Imaging Center, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China </div>
<div id="Aff2">
<sup>2</sup>Guangzhou National Laboratory, Guangzhou, Guangdong China </div>
<div id="Aff3">
<sup>3</sup>Department of Pharmacy, Sun-Yat-Sen Memorial Hospital, Sun Yat-sen University, Guangzhou, Guangdong China </div>
<div id="Aff4">
<sup>4</sup>Department of Radiology, The Fifth Affiliated Hospital of Sun Yat-sen University, Zhuhai, Guangdong China </div>
<div id="Aff5">
<sup>5</sup>Central Laboratory, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China </div>
<div id="Aff6">
<sup>6</sup>Guangdong-Hong Kong-Macao University Joint Laboratory of Interventional Medicine, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China </div>
<div id="Aff7">
<sup>7</sup>Guangdong Provincial Engineering Research Center of Molecular Imaging, The Fifth Affiliated Hospital, Sun Yat-sen University, Zhuhai, Guangdong China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 18; Accepted 2025 Aug 1; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365033  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830575/" class="usa-link">40830575</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The transition of SARS-CoV-2 into a recurrent, seasonal pathogen has underscored the need for the induction of durable immune protection. The nucleocapsid (N) protein is regarded as a promising complementary target for therapeutic and vaccine strategies, owing to its structural robustness, clinical relevance, and ability to elicit critical immune response. Within the N protein, the C-terminal domain (N-CTD) plays a pivotal role in assembly of viral RNA (vRNA)-N protein complexes, and in facilitating liquid-liquid phase separation (LLPS) through specific interactions with RNA on its dimerization surface. Despite its functional importance, the molecular mechanisms by which the RNA-binding surface of this domain can be selectively targeted by inhibitors remain poorly defined. Herein, we report a 2.06 Å crystal structure of the SARS-CoV-2 N-CTD in complex with nCoV400Fab, a human monoclonal antibody derived from single B-cell screening. The structure reveals that nCoV400Fab engages multiple basic residues on the RNA-binding surface, forming a steric blockade that hinders vRNA binding. Functional assays demonstrate that nCoV400Fab disrupts both viral ribonucleoprotein (vRNP) assembly and RNA-induced condensate formation. Together, these findings define a structural mechanism by which a human antibody disrupts the RNA-binding surface of N-CTD, laying the foundation for the development of macromolecular inhibitors.</p>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a8dd/12365033/1476819b3b2d/42003_2025_8648_Figa_HTML.jpg" loading="lazy" id="d33e299" height="467" width="633" alt="graphic file with name 42003_2025_8648_Figa_HTML.jpg"></p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Viral proteins, X-ray crystallography, Antibodies</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par3">A human specific monoclonal antibody binds the C-terminal domain of SARS-CoV-2 Nucleocapsid protein, blocking viral RNA interaction and disrupting condensate formation by interfering with the RNA-binding surface.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par4">The majority of current Coronavirus Disease 2019 (COVID-19) therapeutic strategies target the spike (S) protein of SARS-CoV-2, which facilitates viral entry into host cells. However, the emergence of mutations in the S protein across variants has diminished the effectiveness of many spike-specific antibodies<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. The virus’s capacity for immune evasion highlights the urgent need for targeting a broader spectrum of conserved viral components. Among these, the viral nucleocapsid (N) protein has emerged as a promising auxiliary target for therapeutic interventions. As a fundamental and highly conserved component of <em>Sarbecoviruses</em><sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>, the N protein is essential for viral RNA recognizing and packaging into ribonucleoprotein complex (RNP), contributing to multiple stages of the coronavirus life cycle<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Additionally, the N protein has roles in viral genomic RNA transcription, replication and immune regulation<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>–<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Notably, its RNA-binding activity enables the N protein to antagonize host antiviral responses, thereby contributing to virulence and pathogenesis<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>.</p>
<p id="Par5">Beyond its intracellular functions, the N protein is highly immunogenic and has become a promising vaccine target capable of inducing strong post-infection adaptive immune responses<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. For instance, antibodies against the N protein can contribute to protective immunity<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>,<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Moreover, T cell responses targeting conserved viral epitopes within the N protein appear crucial for long-term protection<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. It is reported that a universal SARS-CoV-2 DNA vaccine encoding spike receptor-binding domain alongside M and N proteins elicited broad neutralizing antibodies and T cell responses. Of note, the N protein-specific T cell alone induced by this vaccine was 60% protective<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Additional investigations concur that such cross-reactive T cell immunity could provide broad coverage against emerging variants<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>.</p>
<p id="Par6">In addition to its vaccine potential, the N protein is abundantly present as a circulating antigen in the serum of COVID-19 patients, often detectable within 2 days of symptom onset, preceding the production of anti-N antibodies<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. This high immunogenicity not only makes N protein a highly sensitive early biomarker compared to spike-based assays<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, but also implicates it as an active contributor to COVID-19 pathogenesis. Specifically, circulating N antigen has been shown to activate NF-κB signaling pathways and MASP-2-mediated complement cascades, driving inflammation and dysregulated innate immune responses<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>,<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. These attributes position the N protein as a potentially variant-resilient antiviral target, broadening therapeutic options beyond spike-based antibodies. However, in contrast to the extensively characterized S protein, how the N protein is recognized by the adaptive immune system remains comparatively underexplored<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>.</p>
<p id="Par7">Typically, the N protein forms structurally heterogeneous complexes with viral RNA in vitro or in vivo. Structurally, it consists of a folded N-terminal domain (NTD), a folded C-terminal dimerization domain (CTD), and three intrinsically disordered regions (IDRs), all of which contribute to RNA binding<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Upon RNA engagement, the N protein adopts a more organized conformation that promotes CTD dimer-dimer interactions<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. This structural coordination subsequently facilitates the ordered assembly of the N protein onto the viral RNA scaffold, yielding compact vRNP architectures<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>–<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Among these domains, the N-CTD is especially critical due to its ability to mediate oligomerization, RNA binding and higher-order vRNP assembly. Notably, the N-CTD drives RNA-induced liquid-liquid phase separation (LLPS), enabling the formation of viral replication condensates that protect viral components from innate immune surveillance<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. This functional versatility reflects the N-CTD’s capacity to bridge viral RNA handling and host regulatory engagement through dynamic multivalent protein–RNA and protein–protein interactions<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>–<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. These interactions allow the N-CTD to suppress innate antiviral immunity by inhibiting dsRNA-induced interferon-β (IFN-β) expression<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>–<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>, and to interfere with host cell proliferation via binding to the host translation elongation factor 1α (EF1α)<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Besides, the CTD is the most immunogenic region of the N protein, harboring dominant antigenic sites and exhibiting strong reactivity with convalescent antibodies, supporting its potential to elicit protective immunity against SARS-CoV and SARS-CoV-2<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>–<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. Despite its essential functional roles, the N-CTD remains remarkably conserved and structurally stable, sharing over 90% sequence identity across SARS-CoV-2 variants of concern (VOCs), with no reported mutations throughout viral evolution<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Of note, the SARS-CoV-2 N-CTD shares 96% sequence identity with other <em>Sarbecoviruses</em><sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>, rendering it a promising target for broad-spectrum antiviral therapies aimed at disrupting viral genomic encapsulation, transcription, replication, and modulation of the host immune response<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>,<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. However, most inhibitor development efforts have focused on the N-NTD, leaving the functionally crucial N-CTD pharmacologically underexplored. The α-helical surface of the N-CTD contains a cluster of basic residues, suggesting a functionally essential RNA-binding interface<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Yet, available small-molecule inhibitors engage only limited surface areas and fail to sufficiently block these essential residues, resulting in incomplete inhibition of RNA binding activity<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>,<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Together, these limitations underscore the therapeutic urgency of developing inhibitors that specifically target N-CTD functions.</p>
<p id="Par8">Building on our previous research, we identified a human monoclonal antibody (mAb) named nCoV400, obtained through single B-cell screening, which binds specifically to the N-CTD<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. To elucidate how this antibody interacts with the N-CTD, we determined the crystal structure of the complex formed between the antigen-binding fragment of mAb nCoV400 (nCoV400Fab) and the N-CTD. The structure reveals that nCoV400Fab engages multiple basic residues on the RNA-binding surface, inducing local conformational rearrangements and generating extensive steric hindrance that may underlie its RNA-blocking capability. Functional assays confirmed that nCoV400Fab hinders N-CTD binding to viral genomic RNA segments, thereby interfering with vRNP-complexes assembly and RNA-driven condensate formation. To the best of our knowledge, this represents the first detailed structural characterization of a human monoclonal antibody targeting the RNA-binding interface of the N-CTD. These findings serve as a fundamental framework for comprehending the mechanisms that could potentially promote the development of macromolecular inhibitors.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Specific binding of N-CTD protein with human monoclonal antibody nCoV400Fab</h3>
<p id="Par9">In our previous research, we successfully isolated 32 mAbs that bind the recombinant N protein of SARS-CoV-2<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Among these, one mAb, designated nCoV400, exhibited selective reactivity toward the N-CTD antigen, as determined through single B-cell human antibody screening methodologies (Fig. <a href="#Fig1" class="usa-link">1a</a>). To evaluate the evolutionary conservation of these epitopes, we performed a comparative analysis of N-CTD sequences across circulating SARS-CoV-2 variants, including VOCs, variants of interest (VOIs), and variants under monitoring (VUMs). Our analysis revealed that the N-CTD is remarkably conserved (Fig. <a href="#Fig1" class="usa-link">1b</a>), with no mutations detected across the examined variants (within 6,141,022 viral genomic sequences<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>). Furthermore, the N-CTD of SARS-CoV-2, shares high sequence homology with that of SARS-CoV and other <em>Sarbecoviruses</em> (Supplementary Fig. <a href="#MOESM1" class="usa-link">S1</a>). This exceptional degree of conservation, combined with the domain’s central role in the virus life cycle, highlights the N-CTD as a compelling broad-spectrum therapeutic target.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Isolation of SARS-CoV-2 N-CTD protein specifically targets human monoclonal antibody nCoV400Fab.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365033_42003_2025_8648_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a8dd/12365033/32db5a69f53f/42003_2025_8648_Fig1_HTML.jpg" loading="lazy" id="d33e555" height="542" width="800" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Workflow of isolation of the specific human monoclonal antibody from a COVID-19 convalescent patient, created by Figdraw. <strong>b</strong> SARS-CoV-2 variants mutation prevalence across lineages and the mutated amino acids within the N protein are shown (gray or black). The statistics were generated by outbreak.info. All SARS-CoV-2 sequences are received via direct provision from the GISAID Initiative. The VOCs, VOIs, and VUMs are defined by the World Health Organization (WHO). <strong>c</strong> The binding affinity of SARS-CoV-2 N-FL with nCoV400Fab measured by BLI. The nCoV400Fab at different concentrations was exposed to 5 μg/mL biotinylated N-FL. <strong>d</strong> The binding assay between SARS-CoV-2 N-NTD and N-CTD domains with nCoV400Fab, measured by BLI. nCoV400Fab at three different concentrations was exposed to 5 μg/mL biotinylated N-NTD or N-CTD. <strong>e</strong> The binding affinity of SARS-CoV-2 N-CTD with nCoV400Fab measured by BLI. The nCoV400Fab at different concentrations was exposed to 5 μg/mL biotinylated N-CTD.</p></figcaption></figure><p id="Par10">To further investigate the functional and structural properties of this mAb targeting the conserved N-CTD region, we expressed a recombinant fragment antigen-binding (Fab) antibody, termed nCoV400Fab. Using BLI, we evaluated the binding kinetics of nCoV400Fab with N-FL protein as well as its truncated domains (N-NTD, N-CTD). nCoV400Fab exhibited strong binding to N-FL, with a <em>K</em><sub>D</sub> value of ~56.67 nM (Fig. <a href="#Fig1" class="usa-link">1c</a> and Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>). Competitive BLI assays further demonstrated that nCoV400Fab specifically recognizes the N-CTD, showing negligible affinity for the N-NTD (Fig. <a href="#Fig1" class="usa-link">1d</a>). Moreover, direct binding assays confirmed that nCoV400Fab binds tightly to N-CTD with a <em>K</em><sub>D</sub> of around 96.31 nM (Fig. <a href="#Fig1" class="usa-link">1e</a> and Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>).</p>
<p id="Par11">For comparative evaluation, we next benchmarked nCoV400 against RM3165-00, a commercial diagnostic antibody. In plate-based ELISA, nCoV400 displayed a markedly lower EC<sub>50</sub> than RM3165 (8.66 vs. 130.6 ng/mL; Supplementary Fig. <a href="#MOESM1" class="usa-link">S2a</a>). However, BLI assay revealed that RM3165 exhibited faster association and slower dissociation rates, resulting in a tighter binding profile in solution-phase measurements (Supplementary Fig. <a href="#MOESM1" class="usa-link">S2b, c</a> and Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>). These differences likely reflect distinct epitope orientations in solid-phase versus solution-phase settings.</p>
<p id="Par12">To further assess epitope specificity, we measured binding of nCoV400Fab to the N-CTDs of two non-<em>sarbecovirus betacoronaviruses</em>, MERS-CoV and HCoV-OC43, via BLI. While the full-length antibody nCoV400 showed high affinity for the SARS-CoV-2 N-CTD (<em>K</em><sub>D</sub> = 0.393 nM), its bindings to the MERS-CoV and HCoV-OC43 homologs were markedly weaker (Supplementary Fig. <a href="#MOESM1" class="usa-link">S2d, e</a>), consistent with the limited sequence conservation of the nCoV400 epitope beyond the <em>Sarbecovirus</em> lineage (Supplementary Fig. <a href="#MOESM1" class="usa-link">S2f</a>).</p>
<p id="Par13">Together, these findings establish nCoV400 as a human monoclonal antibody that binds the SARS-CoV-2 N-CTD with high affinity and specificity, recognizing a conserved epitope restricted to <em>Sarbecovirus</em>.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Crystal structure of the nCoV400Fab–N-CTD complex reveals a novel binding mode</h3>
<p id="Par14">To comprehensively investigate the molecular mechanism underlying the interaction between nCoV400Fab and the N-CTD antigen, we determined the crystal structure of the SARS-CoV-2 N-CTD in complex with nCoV400Fab at 2.06 Å resolution using X-ray crystallography. Briefly, the complex structure was solved by molecular replacement using the N-CTD structure (PDB ID: 7DE1) and monoclonal antibody omalizumab Fab (PDB ID: 6TCN) as search models. The final model includes electron density for residues 249–364 (SARS-CoV-2 N-CTD), 1–235 (nCoV400Fab, the heavy chain of the Fab), and 1–215 (nCoV400Fab, the light chain of the Fab, except for the linker residues ranging from 136 to 141). The asymmetric unit contains one dimeric N-CTD complexed with two nCoV400Fab molecules. The comprehensive details regarding data collection, phasing, and refinement can be accessed in Supplementary Table <a href="#MOESM1" class="usa-link">S2</a>. The coordinates have been deposited in the Protein Data Bank under accession code 9KUR.</p>
<p id="Par15">With the aid of the high-resolution structure, we identified the complementarity-determining regions (CDRs) in nCoV400Fab: light chain CDR1, residues 26–31 (L-CDR1), light chain CDR2, residues 49–51 (L-CDR2), light chain CDR3, residues 88–99 (L-CDR3), heavy chain CDR1, residues 26–33 (H-CDR1), heavy chain CDR2, residues 51–58 (H-CDR2), and heavy chain CDR3, residues 97–119 (H-CDR3), as annotated in Supplementary Fig. <a href="#MOESM1" class="usa-link">S3</a>. The nCoV400Fab adopts a unique binding configuration when compared with other reported N-CTD inhibitors<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>,<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>–<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. As shown in Fig. <a href="#Fig2" class="usa-link">2a</a>, the stable dimeric N-CTD provides a scaffold that accommodates dual nCoV400Fab molecules. The N-CTD dimer presents two distinct surfaces: an alpha-helical face enriched with basic residues, and a beta-sheet-rich facade with primarily neutral polar amino acids. Most previously reported inhibitors engage either the beta-sheet face or the cleft on the side of the dimeric N-CTD structure<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>,<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>–<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>, with only limited recognition of the alpha-helix-dominated face. In contrast, our determined structure reveals that two nCoV400Fab molecules simultaneously bind to the positively charged helical surface of the N-CTD dimer in a cooperative manner, likely stabilized by inter-Fab interactions, thereby defining a previously uncharacterized mode of antibody engagement (Fig. <a href="#Fig2" class="usa-link">2b, c</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Structural overview of nCoV400Fab in complex with dimerized SARS-CoV-2 N-CTD.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365033_42003_2025_8648_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a8dd/12365033/98aaf53213be/42003_2025_8648_Fig2_HTML.jpg" loading="lazy" id="d33e673" height="325" width="700" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Overall crystal structure of two nCoV400Fab bound to the α-helix side of the dimerized N-CTD. The color scheme for the different structural elements is consistent throughout the paper. Three boxed regions mark the binding epitopes (Epitope 1–3). <strong>b</strong> The top view of the crystal structure. <strong>c</strong> The bottom view of the crystal structure showed that the nCoV400Fab interacted with the α-helix side of the N-CTD dimer.</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Binding interface and conformational adaptations of the N-CTD dimer upon recognition by a nature-inspired mAb</h3>
<p id="Par16">A key structural feature of the nCoV400Fab–N-CTD complex is the simultaneous occupancy of two nCoV400Fab molecules on the same α-helical facet of the dimeric N-CTD. This results in extensive buried surface areas (BSA) of ~1083 Å<sup>2</sup> for the light chain and 716 Å<sup>2</sup> for the heavy chain (Fig. <a href="#Fig3" class="usa-link">3a</a> and Supplementary Table <a href="#MOESM1" class="usa-link">S3</a>). Based on a BSA threshold of &gt;10 Å<sup>2</sup>, PISA analysis identified 24 residues of the N-CTD involved in nCoV400Fab binding. This interaction creates a defined cavity between the VH1-CDR3 and VL1-CDR3, which forms a 51.98° angle and a 15.93 Å opening that envelops the positively charged N-terminal α-helix of one N-CTD monomer (Fig. <a href="#Fig3" class="usa-link">3b–d</a>, epitope 1, Supplementary Fig. <a href="#MOESM1" class="usa-link">S4a</a>). A similar pocket exists on the opposite monomer between VH2-CDR3 and VL2-CDR3, forming a 49.25° angle and an opening of 16.46 Å (Supplementary Fig. <a href="#MOESM1" class="usa-link">S4b</a>). These positively charged N-terminal α-helices have previously been identified as crucial regions for RNA binding<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, suggesting that nCoV400Fab sterically occludes this functionally essential surface.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Characterization of the nCoV400Fab-N-CTD interface.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365033_42003_2025_8648_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a8dd/12365033/cd59770ff6ca/42003_2025_8648_Fig3_HTML.jpg" loading="lazy" id="d33e732" height="557" width="800" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> The overlook of three main epitopes of N-CTD recognized by nCoV400Fab in the crystal structure. The nCoV400Fab is shown in a ribbon representation while the N-CTD dimer is shown in an electrostatic surface representation. <strong>b</strong>,<strong> c</strong> The first epitope (250–257) of N-CTD (cornflower blue) is recognized by nCoV400Fab VL1-CDR1,3 (medium purple) and VH1-CDR2-3 (dark magenta) via numerous hydrogen bond interactions and π-stacking interactions in the left panel. The interacting residues of N-CTD and nCoV400Fab are highlighted with the stick representation (left panel). The hydrogen bonds are shown as dashed lines. The VL1-CDR1,3 and VH1-CDR2-3 of nCoV400Fab are shown in an electrostatic surface representation in which blue denotes a positive charge potential while red indicates a negative charge potential (right panel). <strong>d</strong> Recognition of K257 in N-CTD by VL1-CDR1-2 of nCoV400Fab with the stick representation (left panel), and the interacting residues of nCoV400Fab are illustrated in an electrostatic surface representation (right panel). <strong>e</strong> The second epitope of N-CTD is recognized by nCoV400Fab VL2-CDR2 (light pink) via hydrogen bond interactions and hydrophilic interactions. <strong>f</strong> The third epitope of N-CTD recognized by nCoV400Fab VH2-CDR3 via hydrogen bonds interactions and π-stacking interactions.</p></figcaption></figure><p id="Par17">Specifically, a network of interactions from residues 255-257 of N-CTD to the light chain L-CDR1 and L-CDR2 highlights the critical contribution of conserved residue K257 of SARS-CoV-2 N-CTD to nCoV400Fab binding at epitope 1 (Fig. <a href="#Fig3" class="usa-link">3c, d</a>). In epitope 2, residues K299, P302, Q303, and Q306 of N-CTD form multiple hydrophilic and hydrophobic interactions with residues Y48, K49, E52, and R53 of L-CDR2. The side chains of residues E52 and R53 of L-CDR2 establish hydrogen bond interactions with N-CTD residues K299 and Q303 (Fig. <a href="#Fig3" class="usa-link">3e</a>). In epitope 3, N-CTD residues D340 and D343 form hydrogen bonds with Y113 and Y115 of H-CDR3, while residues D343 and P344 of N-CTD simultaneously stack with the π-aromatic of Y113 and Y115 of H-CDR3 (Fig. <a href="#Fig3" class="usa-link">3f</a>). Additional stabilizing interactions are mediated by N-CTD residues at positions 251, 261–263, and 268, which engage L-CDR1/CDR3 residues at positions 28, 29, 90, 97, and 98 through hydrophobic and π-stacking contacts (Supplementary Fig. <a href="#MOESM1" class="usa-link">S5a</a>).</p>
<p id="Par18">To assess antibody-induced conformational changes, we aligned the apo N-CTD<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup> (PDB: 7DE1) with our nCoV400Fab-bound complex (Fig. <a href="#Fig4" class="usa-link">4a</a>). The overall Cα root mean square deviation (RMSD) was modest (0.573 Å), but the three epitopes respond differently. Epitope 1 (residues 250–258) underwent the largest rearrangement (local RMSD = 10.12 Å; Fig. <a href="#Fig4" class="usa-link">4b</a>). Specifically, S250 shifted outward by 20.89 Å; E253 and K256 rotated downward by 15.75 and 10.07 Å, respectively; and S255 pivoted upward 9.27 Å to engage VL-CDR1. K257 moved 1.5 Å, consolidating contacts with VL-CDR1/2 (Supplementary Fig. <a href="#MOESM1" class="usa-link">S5b</a>). In contrast, epitope 2 (299–306) and epitope 3 (340–344) displayed moderate backbone shifts of 1.37 and 1.57 Å, respectively (Fig. <a href="#Fig4" class="usa-link">4c, d</a>). In epitope 2, K299 and Q303 each adjusted ~1.2 Å to optimize hydrogen bonds with VL-CDR2, whereas in epitope 3, D340, D343 and P344 moved 2.5–3.0 Å downward to relieve steric strain and reinforce VH-CDR3 contacts. Collectively, these data support an induced-fit model in which nCoV400Fab remodels the highly basic N-terminal helix (epitope 1) while leaving the remaining RNA-binding surface largely intact.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Structural comparison of nCoV400Fab-N-CTD complex and apo N-CTD.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365033_42003_2025_8648_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a8dd/12365033/41e44f13b1de/42003_2025_8648_Fig4_HTML.jpg" loading="lazy" id="d33e785" height="510" width="700" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Overall structure of the nCoV400Fab–N-CTD complex aligned with the apo N-CTD (PDB: 7DE1). nCoV400Fab is shown as ribbons, the N-CTD dimer in the complex is shown in blue (Complex-N-CTD), and the apo N-CTD is shown in yellow (Apo N-CTD). <strong>b</strong> Superposition of the N-CTD structures in the nCoV400Fab–N-CTD complex (blue) and the apo state (yellow, PDB: 7DE1). Red arrows indicate regions of conformational shift in N-CTD upon antibody binding. <strong>c</strong> Close-up view of epitope 2 (residues 299–306), showing local conformational changes in response to nCoV400Fab binding. Key residues of N-CTD in the complex (blue) and apo N-CTD (yellow), as well as nCoV400Fab VL2-CDR2 (light pink), are depicted as sticks. Dashed lines indicate the direction of displacement, with the corresponding distances labeled. <strong>d</strong> Close-up view of epitope 3 (residues 340–344), showing local conformational changes in response to nCoV400Fab binding. Key residues of N-CTD in the complex (blue) and apo N-CTD (yellow), as well as nCoV400Fab VH2-CDR3 (magenta), are depicted as sticks. Dashed lines indicate the direction of displacement, with the corresponding distances labeled.</p></figcaption></figure><p id="Par19">Despite these localized changes, the β-sheet–mediated interface that stabilizes the N-CTD dimer remains intact upon nCoV400Fab binding. In line with its uncomplexed state, the two monomers are stabilized through three key interfacial regions, with a BSA of 2587 Å<sup>2</sup> out of a total N-CTD monomer surface area of 8892 Å<sup>2</sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">S6a–d</a> and Supplementary Table <a href="#MOESM1" class="usa-link">S3</a>). First, β2 strands from both monomers are stabilized by a network of hydrogen bonds in the β-sheet-rich region (Supplementary Fig. <a href="#MOESM1" class="usa-link">S6a</a>). Second, the η1 region forms strong hydrogen bond interactions with the α3 and α4 helices of neighboring monomers, creating a grip that secures them firmly in place (Supplementary Fig. <a href="#MOESM1" class="usa-link">S6b</a>). The third critical stabilization factor for the N-CTD dimer involves residues R277 and N285 interacting with the main chain of G316, S318, and I320 within the α4 helix of the adjacent monomer (Supplementary Fig. <a href="#MOESM1" class="usa-link">S6c</a>). The last feature is that residue R319 further stabilizes this interface by contacting G278, E280, and Q283 via three hydrogen bonds (Supplementary Fig. <a href="#MOESM1" class="usa-link">S6d</a>). The close positioning of the two Fabs bound to the dimerized N-CTD reinforces this configuration, enabling extensive inter-Fab hydrogen bonding, with a BSA of 355 Å<sup>2</sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">S6e, f</a> and Supplementary Table <a href="#MOESM1" class="usa-link">S3</a>).</p></section><section id="Sec6"><h3 class="pmc_sec_title">nCoV400Fab interferes with viral RNA binding and disrupts higher-order vRNP assembly</h3>
<p id="Par20">Given the structural details of the antibody–antigen interaction, we hypothesized that nCoV400Fab may inhibit the binding of N-CTD to viral genomic RNA not only through steric hindrance, but also by engaging key basic residues critical for electrostatic interactions with RNA. To test this, we evaluated the ability of nCoV400Fab to suppress N-CTD binding to viral genomic RNA, using a defined viral packaging signal as a model substrate. In coronaviruses, the initial step of viral RNA packaging involves recognition of a packaging signal—often a stable stem-loop structure—located near the 3′ end of the ORF1b region. This has been documented in mouse hepatitis virus (MHV), MERS-CoV, SARS-CoV, and SARS-CoV-2<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>–<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. Based on secondary structural predictions, we selected a 157-nucleotide segment (vRNA157, encompassing coordinates 19951-20107) from the 3’-end of the ORF1b region of the SARS-CoV-2 genomic RNA, which forms a stem-loop structure (Supplementary Fig. <a href="#MOESM1" class="usa-link">S7a</a>). Sequence alignment confirmed that vRNA157 is relatively conserved across SARS-CoV-2, SARS-CoV, and other <em>Sarbecoviruses</em> (Fig. <a href="#Fig5" class="usa-link">5a</a>). To assess whether the nCoV400Fab affects the interaction between N-CTD and viral RNA, we synthesized vRNA157 for in vitro binding assays. Primarily, upon titration with increasing concentrations of N-CTD, vRNA157 shifts towards the negative electrode, consistent with formation of RNP complexes in native gel shift assay (Fig. <a href="#Fig5" class="usa-link">5b</a>). Moreover, the addition of equimolar nCoV400Fab disrupted this shift, resulting in the release of vRNA157, suggesting disassembly of the RNP complex. We also tested the ability of nCoV400Fab to inhibit the formation of RNP between N-FL and vRNA157. As the concentration of N-FL increased, vRNA157 bound to N-FL, forming higher-order RNP structures with different aggregation states. When N-FL was added at a fourfold molar excess, vRNA157 was completely sequestered into RNP complexes. However, co-incubation with equimolar nCoV400Fab moderately impaired this binding at high concentrations, partially releasing vRNA157 (Supplementary Fig. <a href="#MOESM1" class="usa-link">S7b</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. The nCoV400Fab inhibits the binding of N-CTD to viral RNA by blocking the potential RNA-binding surface of N-CTD.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365033_42003_2025_8648_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a8dd/12365033/c2696cf825a2/42003_2025_8648_Fig5_HTML.jpg" loading="lazy" id="d33e875" height="500" width="800" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Sequence alignment of SARS-CoV-2 genomic RNA (20910-21079) between BatCoV-RaTG13, SARS-CoV, BatCoV-HKU3, HCoV-OC43, and MERS-CoV. <strong>b</strong> Native PAGE of vRNA157 with varying molar ratios of N-CTD, in the absence or presence of nCoV400Fab at an equimolar ratio to N-CTD. Lane 1 contains no protein, and lane 2 contains only nCoV400Fab, while the subsequent lanes contain increasing amounts of N-CTD protein with molar ratios of 1, 2, 4, and 8 to vRNA157. The electrophoretic result was stained with ethidium bromide for nucleic acid detection (left panel), and then the same gel was stained with Coomassie Blue for protein visualization (right panel). <strong>c</strong> Competitive binding of nCoV400Fab to vRNA157 based on SARS-CoV-2 N-CTD. The biotinylated N-CTD was captured onto the streptavidin biosensors. The nCoV400Fab (upper panel) or vRNA157 (lower panel) as the first analyte, followed by nCoV400Fab mixed with vRNA157 as the second analyte. The competitive binding model was created with BioRender.com. <strong>d</strong> BLI analysis of the interaction between nCoV400 and the K257E mutant N-CTD of SARS-CoV-2. nCoV400 (5 μg/mL) was immobilized onto AHC2 biosensors and incubated with serial concentrations (3.125–100 nM) of the N-CTD K257E mutant protein. <strong>e</strong> BLI analysis of the interaction between nCoV400Fab and the N-terminally truncated N-CTD (ΔN-CTD, residues 264–364) under the same conditions as in panel (<strong>a</strong>). <strong>f</strong> Native PAGE analysis of the RNA-binding capacities of three N-CTD variants: N-terminally truncated N-CTD (ΔN-CTD, residues 264–364), K257E mutant, and wild-type N-CTD. In each assay, 2 μg of 55-nt vRNA was used as a fixed amount (defined as 1× for molar ratio calculations). Lane 1 contains vRNA only and lane 6 contains ΔN-CTD (264–364) only, while lanes 2–5 show increasing amounts of ΔN-CTD (264–364) (0.5, 1, 2, 4, and 8× relative to vRNA). In lanes 7–10, a fixed amount of vRNA was incubated with increasing amounts of K257E N-CTD mutant (1, 2, 4, and 8× relative to vRNA), while lane 11 contains only the K257E mutant. Lanes 12–15 show wild-type N-CTD incubated with vRNA at increasing protein-to-RNA molar ratios (1, 2, 4, and 8× relative to vRNA). Gels were stained with ethidium bromide for nucleic acid detection (left panels), followed by Coomassie Blue for protein visualization (right panels). The positions of free vRNA, free N-CTD variants, and bound complexes are indicated.</p></figcaption></figure><p id="Par21">To further validate these findings, we conducted in-tandem BLI binding assays. Pre-loading N-CTD with nCoV400Fab marginally reduced the subsequent binding of vRNA157 (Fig. <a href="#Fig5" class="usa-link">5c</a>, the upper panel). Conversely, pre-binding of N-CTD with vRNA157 did not prevent subsequent binding of nCoV400Fab (Fig. <a href="#Fig5" class="usa-link">5c</a>, the lower panel). Such outcomes denote that nCoV400Fab can partially compete with vRNA157 for binding to the N-CTD, likely by occupying critical RNA-binding residues.</p>
<p id="Par22">To further clarify whether this disruptive effect extends to additional genomic elements, we synthesized two shorter vRNA fragments: a 55-nt segment from the 5′-UTR (positions 1–55) and a 60-nt segment from the Nsp1 coding region (positions 335–394) (Supplementary Fig. <a href="#MOESM1" class="usa-link">S8</a>). Native gel assays confirmed that both vRNAs formed higher-order complexes with N-CTD in a protein concentration-dependent manner. Upon titration of nCoV400Fab into the preformed complexes, the shifted vRNP bands progressively diminished, and were nearly abolished at an equimolar ratio of Fab to N-CTD. These results indicate that nCoV400Fab effectively impedes N-CTD binding to multiple viral RNA segments.</p>
<p id="Par23">To assess the impact of nCoV400Fab on vRNP assembly, we analyzed the vRNA157–N-CTD complex by SEC (Supplementary Fig. <a href="#MOESM1" class="usa-link">S9a</a>). In the absence of nCoV400Fab, a high-molecular-weight peak eluting at 1.60–1.75 mL yielded only a uniform free RNA band without mobility shift, suggesting that the potential complexes were either oversized or insufficiently stable to enter the gel (Supplementary Fig. <a href="#MOESM1" class="usa-link">S9b</a>, the upper panel). Upon addition of nCoV400Fab, these same fractions produced distinct shifted RNA bands that entered the gel matrix, indicating a reduction in complex size and altered assembly state (Supplementary Fig. <a href="#MOESM1" class="usa-link">S9b</a>, the lower panel). Consistently, negative-stain electron microscopy revealed that the large, amorphous aggregates observed in Fab-free samples were replaced by more filamentous and dispersed particles upon nCoV400Fab treatment (Supplementary Fig. <a href="#MOESM1" class="usa-link">S9c</a>).</p>
<p id="Par24">In agreement with SEC and EM results, dynamic light scattering (DLS) analysis showed that N-CTD assembled into large particles with 60-nt vRNA, exhibiting a dominant hydrodynamic diameter of ~800 nm (Supplementary Fig. <a href="#MOESM1" class="usa-link">S9d</a>). Upon the addition of nCoV400Fab, the average particle size was markedly decreased to ~200 nm, indicating disassembly of higher-order vRNPs. A similar but less pronounced reduction was observed for the 55-nt vRNA complex, suggesting moderate disruption of vRNP assembly (Supplementary Fig. <a href="#MOESM1" class="usa-link">S9d</a>).</p>
<p id="Par25">Collectively, these data demonstrate that nCoV400Fab targets a conserved, lysine-rich RNA-binding surface on the N-CTD dimer, competitively blocks RNA binding, and destabilizes the formation of higher-order vRNP structures.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Conserved N-terminal residues within the Fab-binding epitope are essential for RNA interaction</h3>
<p id="Par26">Guided by the crystal structure, we next investigated whether key residues within the antibody-binding epitope also contribute to RNA recognition. Prior studies have shown that the N-terminal region of N-CTD, enriched in basic lysines, forms a prominent electropositive surface essential for RNA binding<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. To test this, we introduced a point mutation (K257E) and generated an N-terminal truncation mutant (ΔN-CTD, residues 264–364). As measured by BLI, both mutants exhibited severely reduced binding to nCoV400Fab (Fig. <a href="#Fig5" class="usa-link">5d, e</a>), and failed to form detectable vRNP complexes with 55-nt RNA in native gel assays, even at elevated protein concentrations (Fig. <a href="#Fig5" class="usa-link">5f</a>). These findings confirm that the basic N-terminal region, particularly K257, is indispensable for both recognition by nCoV400Fab and RNA binding.</p>
<p id="Par27">To further evaluate the mechanistic specificity of this inhibition, we compared nCoV400Fab with L-chicoric acid, a previously reported small-molecule ligand that binds the lateral cleft of N-CTD<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. Structural modeling suggested that L-chicoric acid engages N-CTD at a distant site from the nCoV400Fab epitope, interacting primarily with residues T271, R276, and R277 through electrostatic and hydrogen bond contacts (Supplementary Fig. <a href="#MOESM1" class="usa-link">S10a–c</a>). While both ligands bind the N-CTD, only nCoV400Fab—which directly targets the highly conserved, electropositive α-helical surface—was able to disrupt vRNP formation in native gel assays. In contrast, <span class="font-variant-small-caps">L</span>-chicoric acid showed no inhibitory activity, even at the highest tested concentration (Supplementary Fig. <a href="#MOESM1" class="usa-link">S10d</a>). Together, these results emphasize that effective inhibition of RNA binding requires direct engagement of the structurally conserved, functionally essential epitope on the α-helical face of N-CTD.</p></section><section id="Sec8"><h3 class="pmc_sec_title">The nCoV400Fab attenuates N protein–RNA condensate formation in vitro and in cells</h3>
<p id="Par28">The formation of membraneless condensates by the N protein in coordination with host factors is known to facilitate viral replication machinery assembly while shielding viral components from host immune surveillance. These condensates are stabilized by dynamic and multivalent protein–RNA interactions. Given the potent RNA-binding inhibition exhibited by nCoV400Fab, we sought to determine whether nCoV400Fab could undermine the condensate formation capability of N-CTD.</p>
<p id="Par29">To test this, we performed a fluorescence microscopy experiment using AF488-labeled N-CTD protein (20 μM) mixed with vRNA157 (1 μM), followed by the addition of various concentrations of the crowding polymer PEG8000 (Fig. <a href="#Fig6" class="usa-link">6a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">S11</a>). Condensate formation occurred only when both N-CTD and vRNA157 were present, indicating their interaction was necessary to nucleate these assemblies (Supplementary Fig. <a href="#MOESM1" class="usa-link">S12a</a>). Importantly, equimolar addition of nCoV400Fab significantly reduced both the average fluorescence intensity and the number of observed droplets (Fig. <a href="#Fig6" class="usa-link">6a</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">S12</a>). These results suggest that nCoV400Fab interferes with the multivalent interactions required for condensate formation, which could affect viral genome-associated biological processes. We next assessed the specificity of this inhibition using a 55-nt Cy5-labeled vRNA fragment and AF488-labeled N-CTD, along with various controls (Supplementary Fig. <a href="#MOESM1" class="usa-link">S13</a>). Robust colocalized liquid-like droplets formed in samples containing both components (Fig. <a href="#Fig6" class="usa-link">6b, c</a>), and this condensate formation was abolished by the N-terminally truncated mutant (ΔN-CTD 264–364) (Fig. <a href="#Fig6" class="usa-link">6b, d</a>), reinforcing the essential role of the N-terminal RNA-binding epitope. Similarly, the addition of nCoV400Fab markedly impaired condensate formation, as indicated by decreases in fluorescence intensity, condensate number, and area occupied, whereas nCoV396Fab, which targets the N-NTD, had no detectable effect (Fig. <a href="#Fig6" class="usa-link">6b, d</a>). We further compared the inhibitory potency of nCoV400Fab to small molecules, including <span class="font-variant-small-caps">L</span>-chicoric acid and ampicillin. Both small-molecule inhibitors exhibited only minor effects on condensate formation (Fig. <a href="#Fig6" class="usa-link">6b, d</a>). Structural analysis indicates that, unlike nCoV400Fab, these compounds do not bind the highly basic α-helical surface of N-CTD but instead interact with regions distinct from the primary RNA-binding site (Supplementary Fig. <a href="#MOESM1" class="usa-link">S10a, b</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">S14</a>), which likely accounts for their limited effect.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. nCoV400 antibody suppresses N protein-driven aggregation in vitro and in cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365033_42003_2025_8648_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a8dd/12365033/915bd30e0e67/42003_2025_8648_Fig6_HTML.jpg" loading="lazy" id="d33e1027" height="566" width="800" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Representative fluorescence images of AF488-labeled N-CTD protein (20 μM) mixed with vRNA157 (1 μM) to form condensate liquid droplets in vitro, in the absence or presence of nCoV400Fab at an equimolar ratio to N-CTD. Lanes 1 and 2 with the addition of 10% PEG8000 while lane 3 and 4 contained 20% PEG8000. Scale bars, 50 μm. <strong>b</strong> Fluorescence microscopy images showing condensate droplets formed by incubation of AF488-labeled full-length N-CTD (green, 20 μM) or AF488-labeled truncated N-CTD (Δ264–364) (green, 20 μM) with Cy5-labeled 55-nt vRNA (red, 20 μM) in the presence or absence of different inhibitors. Conditions included full-length N-CTD with vRNA (lane 1), truncated N-CTD (Δ264–364) with vRNA (lane 2), and N-CTD with vRNA incubated separately with nCoV400Fab (lane 3), <span class="font-variant-small-caps">L</span>-chicoric acid (lane 4), ampicillin (lane 5), or the non-N-CTD-specific antibody nCoV396Fab (lane 6). Merged images (yellow) illustrate colocalization of N-CTD and RNA. Scale bar, 10 μm. <strong>c</strong> Representative line profile analysis demonstrating the colocalization pattern between AF488-N-CTD and Cy5-55-nt vRNA signals. <strong>d</strong> Quantitative analysis of condensate formation, including integrated fluorescence intensity, droplet counts, and area occupied by droplets, calculated from four randomly selected fields for each condition. Data represent mean ± SD. Statistical significance was analyzed by one-way ANOVA (Exact <em>p</em> values are indicated; ns, not significant). <strong>e</strong> Confocal fluorescence microscopy images of HEK293T cells under four conditions, corresponding to lanes 1–4 from left to right: Lane 1: transfected with N-FL-mCherry alone; Lane 2: transfected with N-FL-mCherry and treated with poly(I:C); Lane 3: co-transfected with N-FL-mCherry and plasmids encoding nCoV400-GFP, then treated with poly(I:C); Lane 4: transfected with nCoV400-GFP alone. N-FL-mCherry is shown in red, nCoV400-GFP in green, and merged images (yellow) indicate colocalization. White arrows indicate representative N protein condensates. Scale bar, 10 μm. <strong>f</strong> Representative fluorescence intensity profiles showing colocalization of N-FL-mCherry (red) and nCoV400-GFP (green) signals. <strong>g</strong> Quantification of N protein condensates per cell under each condition. Data were collected from three independent experiments, with 17 randomly selected cells analyzed per group. Data represent mean ± SD. Statistical significance was determined by one-way ANOVA (Exact <em>p</em> values are indicated; ns not significant).</p></figcaption></figure><p id="Par30">To evaluate whether this inhibitory effect extends to cellular settings, we co-transfected HEK293T cells with mCherry-tagged N-FL and plasmids encoding nCoV400 heavy chain-GFP fusion and light chain. Following poly(I:C) stimulation, confocal imaging showed cytoplasmic puncta formation in cells expressing N alone (Fig. <a href="#Fig6" class="usa-link">6e</a>), whereas co-expression of nCoV400 markedly reduced their abundance (Fig. <a href="#Fig6" class="usa-link">6e, g</a>). Line-scan analysis further confirmed cytoplasmic colocalization between nCoV400 and N protein (Fig. <a href="#Fig6" class="usa-link">6f</a>), supporting its activity in live cells.</p>
<p id="Par31">Together, these results demonstrate that nCoV400Fab impairs the ability of N-CTD to form RNA-driven condensate both in vitro and in cells, reinforcing its potential as a targeted antiviral agent.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par32">Coronaviruses have long posed a threat to global health, with their pandemic potential recognized well before the emergence of COVID-19. These RNA viruses span a broad spectrum of pathogenicity—from mild respiratory illness to more severe diseases, such as Middle East Respiratory Syndrome (MERS), severe acute respiratory syndrome (SARS), and the ongoing COVID-19 pandemic. Within this family, the <em>Sarbecovirus</em> subgenus of <em>betacoronaviruses</em> has raised particular concern, as it includes some of the most pathogenic viruses known to infect humans, such as SARS-CoV and SARS-CoV-2. Their emergence is frequently attributed to zoonotic spillover events, where viruses circulating in animal reservoirs cross into humans—often through intermediate hosts—setting the stage for outbreaks and global transmission.</p>
<p id="Par33">Given the ongoing threat posed by <em>Sarbecoviruses</em>, there is a pressing need to develop effective functional inhibitors that can disrupt the virus’s ability to replicate or cause disease. A central challenge in this effort lies in identifying conserved viral elements that can serve as broadly effective therapeutic targets—ideally, those preserved across current and emerging <em>Sarbecovirus</em> lineages. One such target is the conserved N-CTD protein, which plays a crucial role in viral pathogenesis and replication via its RNA-binding ability. In particular, a basic α-helical region near the N-terminus of the domain facilitates vRNA capture, initiates vRNP assembly, and drives RNA-mediated condensate formation. Despite its functional importance, only a handful of studies have reported inhibitors that engage this surface<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>,<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>–<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>, and none to date match the breadth or potency achieved by our antibody lead. Our research identifies a unique human monoclonal antibody, nCoV400Fab, that specifically recognizes the RNA-binding surface of SARS-CoV-2 N-CTD and disrupts multiple downstream functions. Sequence alignment reveals that the epitope targeted by nCoV400Fab—a basic α-helical patch—is virtually invariant across all known <em>Sarbecoviruses</em> but diverges sharply in MERS-CoV and HCoV-OC43, consistent with our BLI data showing dramatically reduced cross-clade binding (Supplementary Fig. <a href="#MOESM1" class="usa-link">S2d–f</a>). By selectively recognizing this structurally conserved and functionally indispensable hotspot, nCoV400Fab provides a promising foundation for the development of a pan-<em>Sarbecovirus</em> inhibitor, potentially reducing the impact of future outbreaks from this viral subgenus.</p>
<p id="Par34">As macromolecular therapeutics, human antibodies offer key advantages over small molecules. Firstly, they are derived from naturally paired heavy and light chain genes in single B cells and are thus evolutionarily optimized to recognize exposed, accessible epitopes. In our structure analysis, nCoV400Fab binds the N-CTD dimer in a 2:2 stoichiometry, contacting 24 residues across three conserved epitope clusters. These sites are not only functionally critical but also evolutionarily stable, reducing the likelihood of escape mutations in emerging variants. Secondly, antibodies utilize their CDRs to form highly specific and multivalent contacts, achieving a stronger and more sustained blocking effect than small molecules. These hypervariable loops within the CDRs regions of antibodies directly contact the epitope, forming a shape-complementary pocket that ensures a tight and selective engagement of epitope surfaces. Structural comparison further revealed that nCoV400Fab buries 1799 Å<sup>2</sup> of surface area upon binding and induces subtle conformational changes in the α-helical RNA-binding surface, supporting an induced-fit mechanism. In contrast, small-molecule ligands such as <span class="font-variant-small-caps">L</span>-chicoric acid engage six residues on a peripheral face while ampicillin binds a neutral α-helical flank, and consequently exhibit limited inhibition of RNA binding or condensate formation. Thus, the multivalent and shape-complementary interaction pattern of nCoV400Fab enables high avidity and steric blockade that small molecules are structurally unable to replicate. Our structural and functional findings reveal that the adaptive immune system can generate antibodies that not only recognize but also disrupt the function of the SARS-CoV-2 N protein. This extends the current immunological focus beyond the S protein and highlights the translational potential of N-targeting antibodies as a complementary broad-spectrum antiviral strategy.</p>
<p id="Par35">While our structural data suggest an induced-fit mechanism, our current analysis is limited to a static comparison. We did not perform molecular dynamics (MD) simulations or other computational approaches to directly investigate the dynamic features of antibody–antigen recognition, which remains a limitation of the present study. Recent studies<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>,<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup> have demonstrated that multiscale and machine learning-based approaches can uncover conformational features beyond static structural analysis, and may guide future extensions of our work.</p>
<p id="Par36">Beyond these structural and mechanistic insights, our biochemical and cellular findings now also open clear translational avenues. As a diagnostic reagent, nCoV400 exhibits markedly higher ELISA sensitivity than the commercial antibody RM3165, while retaining sub-nanomolar affinity in solution. Coupled with its strict <em>Sarbecovirus</em> selectivity, nCoV400 is an excellent candidate for a next-generation probe for high-throughput diagnostic platforms or multiplex serological assays. As a therapeutic scaffold, nCoV400 abolishes RNA–induced N-protein condensates in living cells without perturbing basal cellular physiology, confirming that its epitope remains accessible within the crowded cytoplasm. This property, along with the atomic-level definition of its interaction surface, enables structure-guided engineering of optimized Fab or nanobody formats. Ongoing advances in intracellular antibody delivery—including mRNA or lipid nanoparticles (LNPs) formulations<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>–<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>, cell-penetrating peptides (CPPs)-mediated transport<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>,<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>, and viral-vector technologies—are rapidly expanding the feasibility of such strategies<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>–<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup>. Our high-resolution structural data of nCoV400 provide a precise framework for engineering affinity-matured Fab or nanobody derivatives optimized for such platforms. Importantly, because the targeted α-helical patch is essential for RNA binding and functionally indispensable, any potential escape mutation would likely impair viral replication fitness, creating a high genetic barrier to resistance.</p>
<p id="Par37">In sum, nCoV400 combines structural precision with functional potency, targeting a conserved and essential RNA-binding epitope on the N-CTD that underpins multiple stages of the <em>Sarbecovirus</em> life cycle. Its dual utility as both a sensitive diagnostic tool and a structurally guided, variant-resilient antiviral scaffold positions it as a valuable asset in pandemic preparedness and countermeasure development.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Cloning, expression and purification</h3>
<p id="Par38">The SARS-CoV-2 N-FL (UniProtKB: <a href="https://www.ncbi.nlm.nih.gov/protein/P0DTC9" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P0DTC9</a>) plasmid was kindly provided by the Guangdong Medical Laboratory Animal Center. HCoV-OC43-N-FL (UniProtKB: <a href="https://www.ncbi.nlm.nih.gov/protein/P33469" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P33469</a>) and Middle East Respiratory Syndrome coronavirus (MERS-CoV)-N-FL (UniProtKB: <a href="https://www.ncbi.nlm.nih.gov/protein/K9N4V7" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">K9N4V7</a>) plasmids were purchased from RuiBiotech. Based on secondary structure predictions and sequence conservation, we designed a series of SARS-CoV-2 N constructs, including full-length N (residues 1–419), the N-terminal domain (N-NTD, residues 41–174), the C-terminal domain (N-CTD, residues 250–365), a truncation mutant (ΔN-CTD, residues 264–364), and a point mutant N-CTD (K257E). Homologous N-CTD constructs from HCoV-OC43 and MERS-CoV were also cloned for expression. Gene fragments were amplified by polymerase chain reaction (PCR) and subsequently inserted into the pRSF-Duet-1 vector with an N-terminal 6xHis-SUMO tag using BamHI and XhoI restriction sites. DNA primers used in this study are listed in Supplementary Table <a href="#MOESM1" class="usa-link">S4</a>. Protein expression was induced in <em>E. coli Rosetta</em> (DE3) cells grown in terrific broth (TB) at OD<sub>600</sub> = 0.8–1.0 using 0.1 mM isopropyl-β-<span class="font-variant-small-caps">D</span>-1-thiogalactopyranoside (IPTG), followed by overnight incubation at 16 °C.</p>
<p id="Par39">Following nickel affinity chromatography, the fusion tag was removed via Ulp1 protease digestion. The N-CTD and N-NTD proteins were further purified via size-exclusion chromatography (SEC) (Superdex® 75 Increase 16/600, GE Healthcare) in buffer containing 20 mM Tris-HCl (pH 8.0), 150 mM sodium chloride, 1 mM dithiothreitol, and concentrated using a 3 kDa MWCO Amicon Ultra 15 mL centrifugation filter (Milipore) to a final concentration of 36 mg/ mL. The N-FL protein was purified in the form of inclusion bodies. The inclusion bodies were solubilized in 6 M Urea, 50 mM HEPES (pH 8.0), and 150 mM sodium chloride, followed by nickel column chromatography and ion exchange column chromatography (Q and SP-Sepharose, Cytiva). After refolding, SUMO tags were cleaved by Ulp1, and proteins were further purified with a Superdex® 200 Increase 16/600 column (GE Healthcare) with a buffer consisting of 20 mM HEPES (pH 8.0), 150 mM sodium chloride, then concentrated through a 10 kDa Amicon Ultra 15 mL centrifugation filter (Milipore). Purification data are shown in Supplementary Figs. <a href="#MOESM1" class="usa-link">S15</a>–<a href="#MOESM1" class="usa-link">18</a>.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Expression and purification of nCoV400Fab</h3>
<p id="Par40">The variable regions of the nCoV400 heavy and light chains were cloned into the pCDNA3.1<sup>+</sup> (Invitrogen) mammalian expression vector for Fab-format expression. The heavy chain construct comprised the VH and CH1 domains, while the light chain included the full-length human lambda constant region. Expression plasmids were co-transfected into HEK293F cells cultured in serum-free medium using polyethylenimine (PEI). After 7–8 days of culture, supernatants were collected and the Fab fragments were purified by affinity chromatography using Lambda FabSelect resin (GE Healthcare), according to the manufacturer’s instructions.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Crystallization and data collection</h3>
<p id="Par41">Prior to crystallization, the SARS-CoV-2 N-CTD sample and nCoV400Fab were incubated at a 1.5:1 molar ratio for approximately half an hour and then further purified via SEC (Superdex® 200 Increase 10/300 column, GE Healthcare) (Supplementary Fig. <a href="#MOESM1" class="usa-link">S15b</a>). Crystals were grown by the sitting drop method using a mosquito crystallization robot with 0.3 μL of protein (8 mg/mL) mixed with an equal volume of well solution at 16 °C. After 3 days of initial crystallization, optimized crystals were obtained in 0.2 M sodium formate, 16% PEG3350 conditions. Crystals were frozen in liquid nitrogen in a reservoir solution supplemented with 25% glycerol as a cryoprotectant. X-ray diffraction data were collected at Shanghai Synchrotron Radiation Facility BL10U2. The complex structure of N-CTD with the nCoV400Fab was determined by Phenix-1.20.1-4487 molecular replacement using the SARS-CoV-2 N-CTD structure (PDB ID: 7DE1) and the monoclonal antibody omalizumab Fab (PDB ID: 6TCN) as the search models. Structural figures were generated by ChimeraX-1.8. The X-ray diffraction and structure refinement statistics are summarized in Supplementary Table <a href="#MOESM1" class="usa-link">S2</a>. The final Ramachandran statistics are 97.87% favored, 2.13% allowed, and 0% outliers.</p></section><section id="Sec14"><h3 class="pmc_sec_title">RNA synthesis and purification</h3>
<p id="Par42">The 157-nt viral RNA (vRNA157) was transcribed in vitro using a PCR-amplified DNA template derived from the SARS-CoV-2 genomic cDNA. The RNA product was resolved by denaturing PAGE (8 M urea and 10% acrylamide), and the target band was excised and eluted using the Elutrap electroelution system overnight at 4 °C in 0.5× TBE buffer, followed by ethanol precipitation. The 55-nt and 60-nt vRNA fragments were chemically synthesized and HPLC-purified by GENEWIZ (Suzhou, China). Sequences of all DNA primers and RNA products used in this study are listed in Supplementary Table <a href="#MOESM1" class="usa-link">S4</a>.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Affinity and kinetic measurements by BLI</h3>
<p id="Par43">The biolayer interferometry (BLI) assay was performed using an eight-channel OctetRED biolayer interferometry system (FortéBio) at room temperature (25 °C). SARS-CoV-2 N-CTD, N-NTD and N-FL proteins were biotinylated using the Biotinylation Kit (Thermo Scientific™) for 60 min at 4 °C. To characterize nCoV400Fab binding kinetics, biotinylated SARS-CoV-2 N-FL, N-NTD, and N-CTD (5 μg/mL) were loaded onto the Streptavidin-coated (SA) biosensor (Sartorius) tips pre-equilibrated in the kinetics buffer containing PBS, 0.1% BSA, and 0.02% Tween-20 for 300 s. The kinetic buffer without the loading sample was set to correct the non-specific signals. Following 60 s of baseline in the kinetics buffer, the loaded biosensors were dipped into serially diluted nCoV400Fab protein for 180 s for association. The sensors were then dipped into a kinetic buffer for 180 s to record dissociation. The kinetic buffer without analytes was set to correct the background. This configuration was also used for comparative analysis of RM3165, a commercially available diagnostic antibody, with binding to biotin-N-FL assessed in parallel under identical conditions<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>. To assess epitope specificity and cross-reactivity, full-length nCoV400 antibody (10 μg/mL) was immobilized on anti-Human IgG Fc Capture (AHC2) biosensors (Sartorius). Analytes included N-CTD (residues 250–365), a truncated variant ΔN-CTD (residues 264–364), a point mutant N-CTD (K257E), and homologs from MERS-CoV and HCoV-OC43. ForteBio’s data analysis software was used to fit the dissociation constant (<em>K</em><sub>D</sub>) data using the global fitting method, and the binding profile was processed by GraphPad Prism 8 Software.</p>
<p id="Par44">An in-tandem BLI-based binding assay was performed on the OctetRED biolayer interferometry system (FortéBio) at room temperature (25 °C). The biotin-N-CTD (5 μg/mL) was loaded onto the SA biosensor tips for 300 s. The loaded sensors were dipped into the first analyte (2 μM 157-nt-viral RNA (vRNA157) or 10 nM nCoV400Fab) for 600 s. The sensors were then dipped into the second analyte (10 nM nCoV400Fab, 2 μM vRNA157 or a mixture) for 300 s for competitive binding, followed by a 300 s dissociation phase with kinetics buffer. ForteBio’s data analysis software was used to export data, and the binding profile was processed by GraphPad Prism 8 Software.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Enzyme-linked immunosorbent assay (ELISA)</h3>
<p id="Par45">To evaluate the binding affinity of nCoV400 relative to the diagnostic monoclonal antibody RM3165, ELISA assays were performed using recombinant SARS-CoV-2 N-FL protein. High-binding 96-well ELISA plates (Corning, USA) were coated with 100 μL per well of 5 μg/mL N-FL protein diluted in PBS and incubated overnight at 4 °C. The plates were blocked with 4% BSA in 0.5% Tween-20 PBS (PBST) for 1 h at 37 °C. Serial dilutions of nCoV400 or RM3165 (ranging from 0.064 to 5000 ng/mL) were then added and incubated for 1 h at 37 °C. After three washes with PBST, wells were incubated with HRP-conjugated goat anti-human IgG secondary antibody (1:5000, EarthOx) for 1 h at 37 °C. Following the additional three washes, 3,3′,5,5′-tetramethylbenzidine (TMB) substrate solution (JinKangLong, China) was added, and the colorimetric reaction was stopped with 2 M sulfuric acid. Absorbance at 450 nm was measured using an EnVision multimode plate reader (PerkinElmer). All measurements were performed in six independent replicates. Binding curves were fitted using a four-parameter logistic (4PL) regression model in GraphPad Prism 8 Software, and EC<sub>50</sub> values were calculated accordingly.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Native PAGE assays</h3>
<p id="Par46">To investigate the effect of nCoV400Fab on the formation of RNP complexes involving the N protein and viral RNA, native polyacrylamide gel electrophoresis (native PAGE) was performed in 8% separating gel (pH 8.8). The purified proteins without nucleic acids (estimated by OD<sub>260</sub>/OD<sub>280</sub>) were prepared in protein buffer (20 mM HEPES (pH 8.0), 150 mM sodium chloride). Viral RNA substrates included vRNA157, as well as shorter 55-nt (5′-UTR-derived) and 60-nt (Nsp1-derived) segments. In each reaction, 2 μg RNA (defined as 1×) was mixed with increasing molar ratios of N protein, including wild-type N-CTD, truncation mutant ΔN-CTD (264–364), and point mutant N-CTD (K257E). Where indicated, nCoV400Fab was added at an equimolar concentration to the N-CTD protein. These mixtures were incubated at 4 °C for 30 min before analysis by native PAGE. The samples were electrophoresed at 80 V for 3 h at 4 °C. After electrophoresis, the gel was stained with Ethidium Bromide and used a short-wavelength UV transilluminator for nucleic acid detection. Then the same gel was stained with Coomassie Blue R-250 for protein visualization.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Condensate formation assays</h3>
<p id="Par47">In vitro condensate formation assays were performed at room temperature to evaluate the condensate-forming ability of wild-type and truncated N-CTD proteins in the presence of viral RNA. Purified N-CTD protein and its truncated variant (residues 264–364) (in 20 mM HEPES (pH 7.5), 75 mM Potassium chloride) were labeled by incubation with a 10:1 ratio of fluorescent dye Alexa Fluor<sup>®</sup> 488 succinimidyl ester (NHS ester, Thermo Fisher) (termed AF488 in this study) at 4 °C for 1 h. Unreacted dye was removed using a Hitrap Desalting column (5 mL, Cytiva). Condensate formation was induced by mixing 20 μM N-CTD (wild-type or Δ264–364) with 1 μM viral RNA (vRNA157 or Cy5-labeled 55-nt vRNA) in 5 mM HEPES (pH 7.5), 150 mM potassium chloride. Samples were mixed immediately and then added to individual wells in the pretreated 96-well plate. All reactions at room temperature, unless otherwise indicated, before being imaged on an Olympus IX73 inverted microscope with a 40× objective. For quantification, condensates were identified using particle analysis in ImageJ analysis software, and then the mean gray values of fluorescence inside condensates and the total count of condensates were calculated.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Cell culture and transfection</h3>
<p id="Par48">HEK293T cells were obtained from the ATCC (CRL-3216) and maintained in DMEM (Gibco) supplemented with 10% fetal bovine serum (FBS, Gibco) and 100 U/mL penicillin-streptomycin at 37 °C with 5% CO<sub>2</sub>. All cell lines were routinely tested and confirmed to be mycoplasma-free.</p>
<p id="Par49">To visualize expression and intracellular localization, we constructed expression plasmids encoding SARS-CoV-2 N-FL fused to C-terminal mCherry, and full-length nCoV400 antibody comprising heavy and light chain expression vectors. The heavy chain vector included a CMV promoter, a Kozak sequence (GCCACCATGG), and a fusion GFP tag. The light chain gene was cloned into a separate CMV-driven vector. Poly(I:C) (tlrl-pic) was obtained from InvivoGen. Plasmid transfection was performed using Lipofectamine 3000 (Thermo Fisher Scientific) according to the manufacturer’s protocol. For nCoV400 expression, equimolar amounts of paired heavy and light chain plasmids were co-transfected. Poly(I:C) transfection was used to simulate intracellular viral dsRNA signaling.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Confocal microscopy</h3>
<p id="Par50">HEK293T cells were seeded into glass-bottom dishes (NEST) and co-transfected with plasmids encoding N-mCherry and nCoV400-GFP for 6 h. At 24 h post-transfection, cells were further stimulated with poly(I:C). Live-cell imaging was performed 24–72 h after transfection on a Zeiss LSM 880 confocal microscope using a 63× oil immersion objective. Images were acquired using Zen software.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Dynamic light scattering (DLS)</h3>
<p id="Par51">DLS measurements were performed using a Zetasizer Nano ZS instrument (Malvern Instruments) at 25 °C. For each condition, 5 μg of synthetic vRNA (either 55-nt or 60-nt) was mixed with a fourfold molar excess of purified SARS-CoV-2 N-CTD, in the presence or absence of a fivefold molar excess of nCoV400Fab. All samples were prepared in PBS buffer with a final volume of 1 mL and incubated at room temperature for 30 min before measurement. The hydrodynamic diameter corresponding to the predominant peak was used for comparative evaluation. Each condition was independently measured in triplicate, and average values were plotted with standard deviation.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Negative-staining transmission electron microscopy</h3>
<p id="Par52">Samples were collected from SEC fractions and diluted to an appropriate concentration for visualization. For each grid, 3 μL of sample was applied to glow-discharged carbon-coated copper grids (Zhongjingkeyi) and incubated for 30 s at room temperature. Excess liquid was removed by blotting with filter paper. Negative staining was performed by sequential application of 20 μL 0.75% uranyl acetate three times: 10 s, 10 s, and 1 min, respectively, with blotting after each step. Grids were air-dried and imaged on a Tundra cryo-transmission electron microscope (Thermo Fisher Scientific) operated at 100 kV. Images were acquired using EPU software (Thermo Fisher) at a nominal magnification of 87,000×.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Statistics and reproducibility</h3>
<p id="Par53">For comparisons between two groups, the parametric Student’s <em>t</em>-test was used. For multi-group comparisons, one-way ANOVA was applied. The statistical analyses were performed using GraphPad Prism v8.0.2. Data from at least three independent biological replicates were included in the figures. Differences were considered statistically significant when *<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, and ***<em>p</em> &lt; 0.001.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par54">Further information on research design is available in the <a href="#MOESM4" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec25"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365033/bin/42003_2025_8648_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (23.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12365033/bin/42003_2025_8648_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary data 1</a><sup> (1.6MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12365033/bin/42003_2025_8648_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">42003_2025_8648_MOESM3_ESM.docx</a><sup> (20.9KB, docx) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12365033/bin/42003_2025_8648_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting summary</a><sup> (2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12365033/bin/42003_2025_8648_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (29.4MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>This work is supported by the National Key R&amp;D Program of China (2022YFC2304200), National Natural Science Foundation of China (92269106 and 32171192), Sun Yat-sen University supporting grant (2024_89000_B25886). We thank Guangdong Medical Laboratory Animal Center for providing the N-protein encoding gene plasmids. We thank the staff members of BL10U2/BL17B1/BL18U1/BL19U1 beamlines at the National Facility for Protein Science in Shanghai, for technical support in X-ray diffraction data collection and analysis.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>S.C. contributed to the conception of the study, established the construction of the article, designed the experiments, and wrote the manuscript. S.H., Z.H., G.H., and M.Y. contributed to protein purification and crystallization. S.X., S.H., Z.H., and M.Y. performed structural determination and validation. S.X., S.H., X.C., and G.H. contributed to in vitro protein–protein interaction analysis. S.X., G.H., and Q.C. contributed to in vitro condensate formation analysis. S.X. and S.H. drew figures. W.Z. and S.L. contributed to the interpretation of data. S.X. and S.H. contributed equally to this work.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par55"><em>Communications Biology</em> thanks the anonymous reviewers for their contribution to the peer review of this work. Primary Handling Editors: Janesh Kumar and Mengtan Xing. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The structures in this paper are deposited to the Protein Data Bank with 9KUR access code in Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>. Uncropped gel images are presented as Supplementary Fig. <a href="#MOESM1" class="usa-link">S19</a>. Source data for statistical analyses are provided in Supplementary Data <a href="#MOESM2" class="usa-link">1</a>.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par56">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Shaojie Xue, Suhua He.</p></div>
</div></section><section id="sec26"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s42003-025-08648-x.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Harvey, W. T. et al. SARS-CoV-2 variants, spike mutations and immune escape. <em>Nat. Rev. Microbiol.</em><strong>19</strong>, 409–424 (2021).
</cite> [<a href="https://doi.org/10.1038/s41579-021-00573-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8167834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34075212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harvey,%20W.%20T.%20et%20al.%20SARS-CoV-2%20variants,%20spike%20mutations%20and%20immune%20escape.%20Nat.%20Rev.%20Microbiol.19,%20409%E2%80%93424%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Singh, D. D., Sharma, A., Lee, H. J. &amp; Yadav, D. K. SARS-CoV-2: recent variants and clinical efficacy of antibody-based therapy. <em>Front. Cell Infect. Microbiol.</em><strong>12</strong>, 839170 (2022).
</cite> [<a href="https://doi.org/10.3389/fcimb.2022.839170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8883582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35237535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singh,%20D.%20D.,%20Sharma,%20A.,%20Lee,%20H.%20J.%20&amp;%20Yadav,%20D.%20K.%20SARS-CoV-2:%20recent%20variants%20and%20clinical%20efficacy%20of%20antibody-based%20therapy.%20Front.%20Cell%20Infect.%20Microbiol.12,%20839170%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Ye, Q., West, A. M. V., Silletti, S. &amp; Corbett, K. D. Architecture and self-assembly of the SARS-CoV-2 nucleocapsid protein. <em>Protein Sci.</em><strong>29</strong>, 1890–1901 (2020).
</cite> [<a href="https://doi.org/10.1002/pro.3909" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7405475/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32654247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ye,%20Q.,%20West,%20A.%20M.%20V.,%20Silletti,%20S.%20&amp;%20Corbett,%20K.%20D.%20Architecture%20and%20self-assembly%20of%20the%20SARS-CoV-2%20nucleocapsid%20protein.%20Protein%20Sci.29,%201890%E2%80%931901%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Royster, A. et al. SARS-CoV-2 nucleocapsid protein is a potential therapeutic target for anticoronavirus drug discovery. <em>Microbiol. Spectr.</em><strong>11</strong>, e0118623 (2023).
</cite> [<a href="https://doi.org/10.1128/spectrum.01186-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10269701/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37199631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Royster,%20A.%20et%20al.%20SARS-CoV-2%20nucleocapsid%20protein%20is%20a%20potential%20therapeutic%20target%20for%20anticoronavirus%20drug%20discovery.%20Microbiol.%20Spectr.11,%20e0118623%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Chang, C. K., Hou, M. H., Chang, C. F., Hsiao, C. D. &amp; Huang, T. H. The SARS coronavirus nucleocapsid protein-forms and functions. <em>Antivir. Res.</em><strong>103</strong>, 39–50 (2014).
</cite> [<a href="https://doi.org/10.1016/j.antiviral.2013.12.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7113676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24418573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chang,%20C.%20K.,%20Hou,%20M.%20H.,%20Chang,%20C.%20F.,%20Hsiao,%20C.%20D.%20&amp;%20Huang,%20T.%20H.%20The%20SARS%20coronavirus%20nucleocapsid%20protein-forms%20and%20functions.%20Antivir.%20Res.103,%2039%E2%80%9350%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Kang, S. et al. Crystal structure of SARS-CoV-2 nucleocapsid protein RNA binding domain reveals potential unique drug targeting sites. <em>Acta Pharm. Sin. B</em><strong>10</strong>, 1228–1238 (2020).
</cite> [<a href="https://doi.org/10.1016/j.apsb.2020.04.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7194921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32363136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kang,%20S.%20et%20al.%20Crystal%20structure%20of%20SARS-CoV-2%20nucleocapsid%20protein%20RNA%20binding%20domain%20reveals%20potential%20unique%20drug%20targeting%20sites.%20Acta%20Pharm.%20Sin.%20B10,%201228%E2%80%931238%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Peng, Y. et al. Structures of the SARS-CoV-2 nucleocapsid and their perspectives for drug design. <em>EMBO J.</em><strong>39</strong>, e105938 (2020).
</cite> [<a href="https://doi.org/10.15252/embj.2020105938" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7560215/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32914439/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peng,%20Y.%20et%20al.%20Structures%20of%20the%20SARS-CoV-2%20nucleocapsid%20and%20their%20perspectives%20for%20drug%20design.%20EMBO%20J.39,%20e105938%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Zhao, D. et al. Understanding the phase separation characteristics of nucleocapsid protein provides a new therapeutic opportunity against SARS-CoV-2. <em>Protein Cell</em><strong>12</strong>, 734–740 (2021).
</cite> [<a href="https://doi.org/10.1007/s13238-021-00832-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7994959/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33770364/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20D.%20et%20al.%20Understanding%20the%20phase%20separation%20characteristics%20of%20nucleocapsid%20protein%20provides%20a%20new%20therapeutic%20opportunity%20against%20SARS-CoV-2.%20Protein%20Cell12,%20734%E2%80%93740%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Cong, Y. et al. Nucleocapsid protein recruitment to replication-transcription complexes plays a crucial role in coronaviral life cycle. <em>J. Virol.</em>10.1128/jvi.01925-19 (2020).</cite> [<a href="https://doi.org/10.1128/JVI.01925-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6997762/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31776274/" class="usa-link">PubMed</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Lu, X., Pan, J., Tao, J. &amp; Guo, D. SARS-CoV nucleocapsid protein antagonizes IFN-β response by targeting initial step of IFN-β induction pathway, and its C-terminal region is critical for the antagonism. <em>Virus Genes</em><strong>42</strong>, 37–45 (2011).
</cite> [<a href="https://doi.org/10.1007/s11262-010-0544-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7088804/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20976535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu,%20X.,%20Pan,%20J.,%20Tao,%20J.%20&amp;%20Guo,%20D.%20SARS-CoV%20nucleocapsid%20protein%20antagonizes%20IFN-%CE%B2%20response%20by%20targeting%20initial%20step%20of%20IFN-%CE%B2%20induction%20pathway,%20and%20its%20C-terminal%20region%20is%20critical%20for%20the%20antagonism.%20Virus%20Genes42,%2037%E2%80%9345%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Surjit, M., Liu, B., Chow, V. T. &amp; Lal, S. K. The nucleocapsid protein of severe acute respiratory syndrome-coronavirus inhibits the activity of cyclin-cyclin-dependent kinase complex and blocks S phase progression in mammalian cells. <em>J. Biol. Chem.</em><strong>281</strong>, 10669–10681 (2006).
</cite> [<a href="https://doi.org/10.1074/jbc.M509233200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7995956/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16431923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Surjit,%20M.,%20Liu,%20B.,%20Chow,%20V.%20T.%20&amp;%20Lal,%20S.%20K.%20The%20nucleocapsid%20protein%20of%20severe%20acute%20respiratory%20syndrome-coronavirus%20inhibits%20the%20activity%20of%20cyclin-cyclin-dependent%20kinase%20complex%20and%20blocks%20S%20phase%20progression%20in%20mammalian%20cells.%20J.%20Biol.%20Chem.281,%2010669%E2%80%9310681%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Mu, J. et al. SARS-CoV-2-encoded nucleocapsid protein acts as a viral suppressor of RNA interference in cells. <em>Sci. China Life Sci.</em><strong>63</strong>, 1413–1416 (2020).
</cite> [<a href="https://doi.org/10.1007/s11427-020-1692-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7154568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32291557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mu,%20J.%20et%20al.%20SARS-CoV-2-encoded%20nucleocapsid%20protein%20acts%20as%20a%20viral%20suppressor%20of%20RNA%20interference%20in%20cells.%20Sci.%20China%20Life%20Sci.63,%201413%E2%80%931416%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Gao, W. et al. Effects of a SARS-associated coronavirus vaccine in monkeys. <em>Lancet</em><strong>362</strong>, 1895–1896 (2003).
</cite> [<a href="https://doi.org/10.1016/S0140-6736(03)14962-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7112457/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14667748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao,%20W.%20et%20al.%20Effects%20of%20a%20SARS-associated%20coronavirus%20vaccine%20in%20monkeys.%20Lancet362,%201895%E2%80%931896%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Dangi, T., Class, J., Palacio, N., Richner, J. M. &amp; Penaloza MacMaster, P. Combining spike- and nucleocapsid-based vaccines improves distal control of SARS-CoV-2. <em>Cell Rep.</em><strong>36</strong>, 109664 (2021).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2021.109664" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8367759/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34450033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dangi,%20T.,%20Class,%20J.,%20Palacio,%20N.,%20Richner,%20J.%20M.%20&amp;%20Penaloza%20MacMaster,%20P.%20Combining%20spike-%20and%20nucleocapsid-based%20vaccines%20improves%20distal%20control%20of%20SARS-CoV-2.%20Cell%20Rep.36,%20109664%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Hajnik, R. L. et al. Dual spike and nucleocapsid mRNA vaccination confer protection against SARS-CoV-2 Omicron and Delta variants in preclinical models. <em>Sci. Transl. Med.</em><strong>14</strong>, eabq1945 (2022).
</cite> [<a href="https://doi.org/10.1126/scitranslmed.abq1945" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9926941/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36103514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hajnik,%20R.%20L.%20et%20al.%20Dual%20spike%20and%20nucleocapsid%20mRNA%20vaccination%20confer%20protection%20against%20SARS-CoV-2%20Omicron%20and%20Delta%20variants%20in%20preclinical%20models.%20Sci.%20Transl.%20Med.14,%20eabq1945%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Arieta, C. M. et al. The T-cell-directed vaccine BNT162b4 encoding conserved non-spike antigens protects animals from severe SARS-CoV-2 infection. <em>Cell</em><strong>186</strong>, 2392–2409.e2321 (2023).
</cite> [<a href="https://doi.org/10.1016/j.cell.2023.04.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10099181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37164012/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arieta,%20C.%20M.%20et%20al.%20The%20T-cell-directed%20vaccine%20BNT162b4%20encoding%20conserved%20non-spike%20antigens%20protects%20animals%20from%20severe%20SARS-CoV-2%20infection.%20Cell186,%202392%E2%80%932409.e2321%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Nesamari, R. et al. Post-pandemic memory T cell response to SARS-CoV-2 is durable, broadly targeted, and cross-reactive to the hypermutated BA.2.86 variant. <em>Cell Host Microbe</em><strong>32</strong>, 162–169.e163 (2024).
</cite> [<a href="https://doi.org/10.1016/j.chom.2023.12.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10901529/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38211583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nesamari,%20R.%20et%20al.%20Post-pandemic%20memory%20T%20cell%20response%20to%20SARS-CoV-2%20is%20durable,%20broadly%20targeted,%20and%20cross-reactive%20to%20the%20hypermutated%20BA.2.86%20variant.%20Cell%20Host%20Microbe32,%20162%E2%80%93169.e163%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Kang, S. et al. A SARS-CoV-2 antibody curbs viral nucleocapsid protein-induced complement hyperactivation. <em>Nat. Commun.</em><strong>12</strong>, 2697 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-23036-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8113585/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33976229/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kang,%20S.%20et%20al.%20A%20SARS-CoV-2%20antibody%20curbs%20viral%20nucleocapsid%20protein-induced%20complement%20hyperactivation.%20Nat.%20Commun.12,%202697%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Dangi, T. et al. Improved control of SARS-CoV-2 by treatment with a nucleocapsid-specific monoclonal antibody. <em>J. Clin. Invest.</em>10.1172/jci162282 (2022).</cite> [<a href="https://doi.org/10.1172/JCI162282" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9711872/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36219482/" class="usa-link">PubMed</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Braun, A. et al. Mapping the immunopeptidome of seven SARS-CoV-2 antigens across common HLA haplotypes. <em>Nat. Commun.</em><strong>15</strong>, 7547 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-51959-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11364864/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39214998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Braun,%20A.%20et%20al.%20Mapping%20the%20immunopeptidome%20of%20seven%20SARS-CoV-2%20antigens%20across%20common%20HLA%20haplotypes.%20Nat.%20Commun.15,%207547%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Appelberg, S. et al. A universal SARS-CoV DNA vaccine inducing highly cross-reactive neutralizing antibodies and T cells. <em>EMBO Mol. Med.</em><strong>14</strong>, e15821 (2022).
</cite> [<a href="https://doi.org/10.15252/emmm.202215821" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9538582/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35986481/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Appelberg,%20S.%20et%20al.%20A%20universal%20SARS-CoV%20DNA%20vaccine%20inducing%20highly%20cross-reactive%20neutralizing%20antibodies%20and%20T%20cells.%20EMBO%20Mol.%20Med.14,%20e15821%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Low, J. S. et al. Clonal analysis of immunodominance and cross-reactivity of the CD4 T cell response to SARS-CoV-2. <em>Science</em><strong>372</strong>, 1336–1341 (2021).
</cite> [<a href="https://doi.org/10.1126/science.abg8985" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8168615/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34006597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Low,%20J.%20S.%20et%20al.%20Clonal%20analysis%20of%20immunodominance%20and%20cross-reactivity%20of%20the%20CD4%20T%20cell%20response%20to%20SARS-CoV-2.%20Science372,%201336%E2%80%931341%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Mallajosyula, V. et al. CD8(+) T cells specific for conserved coronavirus epitopes correlate with milder disease in COVID-19 patients. <em>Sci. Immunol.</em>10.1126/sciimmunol.abg5669 (2021).</cite> [<a href="https://doi.org/10.1126/sciimmunol.abg5669" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8975171/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34210785/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Augusto, D. G. et al. A common allele of HLA is associated with asymptomatic SARS-CoV-2 infection. <em>Nature</em><strong>620</strong>, 128–136 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-06331-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10396966/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37468623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Augusto,%20D.%20G.%20et%20al.%20A%20common%20allele%20of%20HLA%20is%20associated%20with%20asymptomatic%20SARS-CoV-2%20infection.%20Nature620,%20128%E2%80%93136%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Yokoyama, R. et al. Association of the serum levels of the nucleocapsid antigen of SARS-CoV-2 with the diagnosis, disease severity, and antibody titers in patients with COVID-19: a retrospective cross-sectional study. <em>Front. Microbiol.</em><strong>12</strong>, 791489 (2021).
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.791489" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8696188/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34956158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yokoyama,%20R.%20et%20al.%20Association%20of%20the%20serum%20levels%20of%20the%20nucleocapsid%20antigen%20of%20SARS-CoV-2%20with%20the%20diagnosis,%20disease%20severity,%20and%20antibody%20titers%20in%20patients%20with%20COVID-19:%20a%20retrospective%20cross-sectional%20study.%20Front.%20Microbiol.12,%20791489%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Burbelo, P. D. et al. Sensitivity in detection of antibodies to nucleocapsid and spike proteins of severe acute respiratory syndrome coronavirus 2 in patients with coronavirus disease 2019. <em>J. Infect. Dis.</em><strong>222</strong>, 206–213 (2020).
</cite> [<a href="https://doi.org/10.1093/infdis/jiaa273" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7313936/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32427334/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Burbelo,%20P.%20D.%20et%20al.%20Sensitivity%20in%20detection%20of%20antibodies%20to%20nucleocapsid%20and%20spike%20proteins%20of%20severe%20acute%20respiratory%20syndrome%20coronavirus%202%20in%20patients%20with%20coronavirus%20disease%202019.%20J.%20Infect.%20Dis.222,%20206%E2%80%93213%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Thudium, R. F. et al. Early laboratory diagnosis of COVID-19 by antigen detection in blood samples of the SARS-CoV-2 nucleocapsid protein. <em>J. Clin. Microbiol.</em><strong>59</strong>, e0100121 (2021).
</cite> [<a href="https://doi.org/10.1128/JCM.01001-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8451432/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34260271/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thudium,%20R.%20F.%20et%20al.%20Early%20laboratory%20diagnosis%20of%20COVID-19%20by%20antigen%20detection%20in%20blood%20samples%20of%20the%20SARS-CoV-2%20nucleocapsid%20protein.%20J.%20Clin.%20Microbiol.59,%20e0100121%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Zhang, Y. et al. Diagnostic value of nucleocapsid protein in blood for SARS-CoV-2 infection. <em>Clin. Chem.</em><strong>68</strong>, 240–248 (2021).
</cite> [<a href="https://doi.org/10.1093/clinchem/hvab148" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8436384/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34358289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.%20et%20al.%20Diagnostic%20value%20of%20nucleocapsid%20protein%20in%20blood%20for%20SARS-CoV-2%20infection.%20Clin.%20Chem.68,%20240%E2%80%93248%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Xia, J. et al. SARS-CoV-2 N protein induces acute lung injury in mice via NF-ĸB activation. <em>Front. Immunol.</em><strong>12</strong>, 791753 (2021).
</cite> [<a href="https://doi.org/10.3389/fimmu.2021.791753" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8688532/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34950152/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xia,%20J.%20et%20al.%20SARS-CoV-2%20N%20protein%20induces%20acute%20lung%20injury%20in%20mice%20via%20NF-%C4%B8B%20activation.%20Front.%20Immunol.12,%20791753%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Gao, T. et al. Highly pathogenic coronavirus N protein aggravates inflammation by MASP-2-mediated lectin complement pathway overactivation. <em>Signal Transduct. Target Ther.</em><strong>7</strong>, 318 (2022).
</cite> [<a href="https://doi.org/10.1038/s41392-022-01133-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9470675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36100602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao,%20T.%20et%20al.%20Highly%20pathogenic%20coronavirus%20N%20protein%20aggravates%20inflammation%20by%20MASP-2-mediated%20lectin%20complement%20pathway%20overactivation.%20Signal%20Transduct.%20Target%20Ther.7,%20318%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Yuan, M. &amp; Wilson, I. A. Structural immunology of SARS-CoV-2. <em>Immunol. Rev.</em><strong>329</strong>, e13431 (2025).
</cite> [<a href="https://doi.org/10.1111/imr.13431" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11727448/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39731211/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yuan,%20M.%20&amp;%20Wilson,%20I.%20A.%20Structural%20immunology%20of%20SARS-CoV-2.%20Immunol.%20Rev.329,%20e13431%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Pontoriero, L. et al. NMR reveals specific tracts within the intrinsically disordered regions of the SARS-CoV-2 nucleocapsid protein involved in RNA encountering. <em>Biomolecules</em>10.3390/biom12070929 (2022).</cite> [<a href="https://doi.org/10.3390/biom12070929" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9312987/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35883485/" class="usa-link">PubMed</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Zhao, H. et al. Assembly of SARS-CoV-2 nucleocapsid protein with nucleic acid. <em>Nucleic Acids Res.</em><strong>52</strong>, 6647–6661 (2024).
</cite> [<a href="https://doi.org/10.1093/nar/gkae256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11194069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38587193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20H.%20et%20al.%20Assembly%20of%20SARS-CoV-2%20nucleocapsid%20protein%20with%20nucleic%20acid.%20Nucleic%20Acids%20Res.52,%206647%E2%80%936661%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Zhao, H. et al. Plasticity in structure and assembly of SARS-CoV-2 nucleocapsid protein. <em>PNAS Nexus</em><strong>1</strong>, pgac049 (2022).
</cite> [<a href="https://doi.org/10.1093/pnasnexus/pgac049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9235412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35783502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20H.%20et%20al.%20Plasticity%20in%20structure%20and%20assembly%20of%20SARS-CoV-2%20nucleocapsid%20protein.%20PNAS%20Nexus1,%20pgac049%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Yang, M. et al. Structural insight into the SARS-CoV-2 nucleocapsid protein C-terminal domain reveals a novel recognition mechanism for viral transcriptional regulatory sequences. <em>Front. Chem.</em><strong>8</strong>, 624765 (2020).
</cite> [<a href="https://doi.org/10.3389/fchem.2020.624765" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7835709/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33511102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20M.%20et%20al.%20Structural%20insight%20into%20the%20SARS-CoV-2%20nucleocapsid%20protein%20C-terminal%20domain%20reveals%20a%20novel%20recognition%20mechanism%20for%20viral%20transcriptional%20regulatory%20sequences.%20Front.%20Chem.8,%20624765%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>He, R. et al. Characterization of protein-protein interactions between the nucleocapsid protein and membrane protein of the SARS coronavirus. <em>Virus Res.</em><strong>105</strong>, 121–125 (2004).
</cite> [<a href="https://doi.org/10.1016/j.virusres.2004.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7127797/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15351485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He,%20R.%20et%20al.%20Characterization%20of%20protein-protein%20interactions%20between%20the%20nucleocapsid%20protein%20and%20membrane%20protein%20of%20the%20SARS%20coronavirus.%20Virus%20Res.105,%20121%E2%80%93125%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Yu, I. M., Oldham, M. L., Zhang, J. &amp; Chen, J. Crystal structure of the severe acute respiratory syndrome (SARS) coronavirus nucleocapsid protein dimerization domain reveals evolutionary linkage between corona- and arteriviridae. <em>J. Biol. Chem.</em><strong>281</strong>, 17134–17139 (2006).
</cite> [<a href="https://doi.org/10.1074/jbc.M602107200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7946579/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16627473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20I.%20M.,%20Oldham,%20M.%20L.,%20Zhang,%20J.%20&amp;%20Chen,%20J.%20Crystal%20structure%20of%20the%20severe%20acute%20respiratory%20syndrome%20(SARS)%20coronavirus%20nucleocapsid%20protein%20dimerization%20domain%20reveals%20evolutionary%20linkage%20between%20corona-%20and%20arteriviridae.%20J.%20Biol.%20Chem.281,%2017134%E2%80%9317139%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Lu, S. et al. The SARS-CoV-2 nucleocapsid phosphoprotein forms mutually exclusive condensates with RNA and the membrane-associated M protein. <em>Nat. Commun.</em><strong>12</strong>, 502 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-020-20768-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7820290/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33479198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu,%20S.%20et%20al.%20The%20SARS-CoV-2%20nucleocapsid%20phosphoprotein%20forms%20mutually%20exclusive%20condensates%20with%20RNA%20and%20the%20membrane-associated%20M%20protein.%20Nat.%20Commun.12,%20502%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Savastano, A., Ibáñez de Opakua, A., Rankovic, M. &amp; Zweckstetter, M. Nucleocapsid protein of SARS-CoV-2 phase separates into RNA-rich polymerase-containing condensates. <em>Nat. Commun.</em><strong>11</strong>, 6041 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-19843-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7699647/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33247108/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Savastano,%20A.,%20Ib%C3%A1%C3%B1ez%20de%20Opakua,%20A.,%20Rankovic,%20M.%20&amp;%20Zweckstetter,%20M.%20Nucleocapsid%20protein%20of%20SARS-CoV-2%20phase%20separates%20into%20RNA-rich%20polymerase-containing%20condensates.%20Nat.%20Commun.11,%206041%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Chen, H. et al. Liquid-liquid phase separation by SARS-CoV-2 nucleocapsid protein and RNA. <em>Cell Res.</em><strong>30</strong>, 1143–1145 (2020).
</cite> [<a href="https://doi.org/10.1038/s41422-020-00408-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7477871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32901111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20H.%20et%20al.%20Liquid-liquid%20phase%20separation%20by%20SARS-CoV-2%20nucleocapsid%20protein%20and%20RNA.%20Cell%20Res.30,%201143%E2%80%931145%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Perdikari, T. M. et al. SARS-CoV-2 nucleocapsid protein phase-separates with RNA and with human hnRNPs. <em>EMBO J.</em><strong>39</strong>, e106478 (2020).
</cite> [<a href="https://doi.org/10.15252/embj.2020106478" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7737613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33200826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perdikari,%20T.%20M.%20et%20al.%20SARS-CoV-2%20nucleocapsid%20protein%20phase-separates%20with%20RNA%20and%20with%20human%20hnRNPs.%20EMBO%20J.39,%20e106478%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Masters, P. S. Coronavirus genomic RNA packaging. <em>Virology</em><strong>537</strong>, 198–207 (2019).
</cite> [<a href="https://doi.org/10.1016/j.virol.2019.08.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7112113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31505321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Masters,%20P.%20S.%20Coronavirus%20genomic%20RNA%20packaging.%20Virology537,%20198%E2%80%93207%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Cascarina, S. M. &amp; Ross, E. D. Phase separation by the SARS-CoV-2 nucleocapsid protein: consensus and open questions. <em>J. Biol. Chem.</em><strong>298</strong>, 101677 (2022).
</cite> [<a href="https://doi.org/10.1016/j.jbc.2022.101677" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8813722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35131265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cascarina,%20S.%20M.%20&amp;%20Ross,%20E.%20D.%20Phase%20separation%20by%20the%20SARS-CoV-2%20nucleocapsid%20protein:%20consensus%20and%20open%20questions.%20J.%20Biol.%20Chem.298,%20101677%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Wu, Y. et al. RNA-induced liquid phase separation of SARS-CoV-2 nucleocapsid protein facilitates NF-κB hyper-activation and inflammation. <em>Signal Transduct. Target Ther.</em><strong>6</strong>, 167 (2021).
</cite> [<a href="https://doi.org/10.1038/s41392-021-00575-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8065320/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33895773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20Y.%20et%20al.%20RNA-induced%20liquid%20phase%20separation%20of%20SARS-CoV-2%20nucleocapsid%20protein%20facilitates%20NF-%CE%BAB%20hyper-activation%20and%20inflammation.%20Signal%20Transduct.%20Target%20Ther.6,%20167%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Wang, S. et al. Targeting liquid-liquid phase separation of SARS-CoV-2 nucleocapsid protein promotes innate antiviral immunity by elevating MAVS activity. <em>Nat. Cell Biol.</em><strong>23</strong>, 718–732 (2021).
</cite> [<a href="https://doi.org/10.1038/s41556-021-00710-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34239064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20S.%20et%20al.%20Targeting%20liquid-liquid%20phase%20separation%20of%20SARS-CoV-2%20nucleocapsid%20protein%20promotes%20innate%20antiviral%20immunity%20by%20elevating%20MAVS%20activity.%20Nat.%20Cell%20Biol.23,%20718%E2%80%93732%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Zhou, B. et al. The nucleocapsid protein of severe acute respiratory syndrome coronavirus inhibits cell cytokinesis and proliferation by interacting with translation elongation factor 1alpha. <em>J. Virol.</em><strong>82</strong>, 6962–6971 (2008).
</cite> [<a href="https://doi.org/10.1128/JVI.00133-08" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2446950/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18448518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou,%20B.%20et%20al.%20The%20nucleocapsid%20protein%20of%20severe%20acute%20respiratory%20syndrome%20coronavirus%20inhibits%20cell%20cytokinesis%20and%20proliferation%20by%20interacting%20with%20translation%20elongation%20factor%201alpha.%20J.%20Virol.82,%206962%E2%80%936971%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Shang, B. et al. Characterization and application of monoclonal antibodies against N protein of SARS-coronavirus. <em>Biochem. Biophys. Res. Commun.</em><strong>336</strong>, 110–117 (2005).
</cite> [<a href="https://doi.org/10.1016/j.bbrc.2005.08.032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7092910/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16112641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shang,%20B.%20et%20al.%20Characterization%20and%20application%20of%20monoclonal%20antibodies%20against%20N%20protein%20of%20SARS-coronavirus.%20Biochem.%20Biophys.%20Res.%20Commun.336,%20110%E2%80%93117%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Lutomski, C. A., El-Baba, T. J., Bolla, J. R. &amp; Robinson, C. V. Multiple roles of SARS-CoV-2 N protein facilitated by proteoform-specific interactions with RNA, host proteins, and convalescent antibodies. <em>JACS Au</em><strong>1</strong>, 1147–1157 (2021).
</cite> [<a href="https://doi.org/10.1021/jacsau.1c00139" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8231660/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34462738/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lutomski,%20C.%20A.,%20El-Baba,%20T.%20J.,%20Bolla,%20J.%20R.%20&amp;%20Robinson,%20C.%20V.%20Multiple%20roles%20of%20SARS-CoV-2%20N%20protein%20facilitated%20by%20proteoform-specific%20interactions%20with%20RNA,%20host%20proteins,%20and%20convalescent%20antibodies.%20JACS%20Au1,%201147%E2%80%931157%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Wu, C. et al. Characterization of SARS-CoV-2 nucleocapsid protein reveals multiple functional consequences of the C-terminal domain. <em>iScience</em><strong>24</strong>, 102681 (2021).
</cite> [<a href="https://doi.org/10.1016/j.isci.2021.102681" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8168301/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34095780/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20C.%20et%20al.%20Characterization%20of%20SARS-CoV-2%20nucleocapsid%20protein%20reveals%20multiple%20functional%20consequences%20of%20the%20C-terminal%20domain.%20iScience24,%20102681%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Nguyen, A. et al. Modulation of biophysical properties of nucleocapsid protein in the mutant spectrum of SARS-CoV-2. <em>Elife</em>10.7554/eLife.94836 (2024).</cite> [<a href="https://doi.org/10.7554/eLife.94836" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11213569/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38941236/" class="usa-link">PubMed</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Gupta, R. et al. SARS-CoV-2 (COVID-19) structural and evolutionary dynamicome: insights into functional evolution and human genomics. <em>J. Biol. Chem.</em><strong>295</strong>, 11742–11753 (2020).
</cite> [<a href="https://doi.org/10.1074/jbc.RA120.014873" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7450099/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32587094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gupta,%20R.%20et%20al.%20SARS-CoV-2%20(COVID-19)%20structural%20and%20evolutionary%20dynamicome:%20insights%20into%20functional%20evolution%20and%20human%20genomics.%20J.%20Biol.%20Chem.295,%2011742%E2%80%9311753%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Monod, A. et al. Learning from structure-based drug design and new antivirals targeting the ribonucleoprotein complex for the treatment of influenza. <em>Expert Opin. Drug Discov.</em><strong>10</strong>, 345–371 (2015).
</cite> [<a href="https://doi.org/10.1517/17460441.2015.1019859" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25792362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Monod,%20A.%20et%20al.%20Learning%20from%20structure-based%20drug%20design%20and%20new%20antivirals%20targeting%20the%20ribonucleoprotein%20complex%20for%20the%20treatment%20of%20influenza.%20Expert%20Opin.%20Drug%20Discov.10,%20345%E2%80%93371%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Ahamad, S., Gupta, D. &amp; Kumar, V. Targeting SARS-CoV-2 nucleocapsid oligomerization: insights from molecular docking and molecular dynamics simulations. <em>J. Biomol. Struct. Dyn.</em><strong>40</strong>, 2430–2443 (2022).
</cite> [<a href="https://doi.org/10.1080/07391102.2020.1839563" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7663461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33140703/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ahamad,%20S.,%20Gupta,%20D.%20&amp;%20Kumar,%20V.%20Targeting%20SARS-CoV-2%20nucleocapsid%20oligomerization:%20insights%20from%20molecular%20docking%20and%20molecular%20dynamics%20simulations.%20J.%20Biomol.%20Struct.%20Dyn.40,%202430%E2%80%932443%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Mercaldi, G. F. et al. Discovery and structural characterization of chicoric acid as a SARS-CoV-2 nucleocapsid protein ligand and RNA binding disruptor. <em>Sci. Rep.</em><strong>12</strong>, 18500 (2022).
</cite> [<a href="https://doi.org/10.1038/s41598-022-22576-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9628480/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36323732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mercaldi,%20G.%20F.%20et%20al.%20Discovery%20and%20structural%20characterization%20of%20chicoric%20acid%20as%20a%20SARS-CoV-2%20nucleocapsid%20protein%20ligand%20and%20RNA%20binding%20disruptor.%20Sci.%20Rep.12,%2018500%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Dhaka, P., Mahto, J. K., Singh, A., Kumar, P. &amp; Tomar, S. Structural insights into the RNA binding inhibitors of the C-terminal domain of the SARS-CoV-2 nucleocapsid. <em>J. Struct. Biol.</em><strong>217</strong>, 108197 (2025).
</cite> [<a href="https://doi.org/10.1016/j.jsb.2025.108197" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40113149/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dhaka,%20P.,%20Mahto,%20J.%20K.,%20Singh,%20A.,%20Kumar,%20P.%20&amp;%20Tomar,%20S.%20Structural%20insights%20into%20the%20RNA%20binding%20inhibitors%20of%20the%20C-terminal%20domain%20of%20the%20SARS-CoV-2%20nucleocapsid.%20J.%20Struct.%20Biol.217,%20108197%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Gangavarapu, K. et al. Outbreak.info genomic reports: scalable and dynamic surveillance of SARS-CoV-2 variants and mutations. <em>Nat. Methods</em><strong>20</strong>, 512–522 (2023).
</cite> [<a href="https://doi.org/10.1038/s41592-023-01769-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10399614/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36823332/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gangavarapu,%20K.%20et%20al.%20Outbreak.info%20genomic%20reports:%20scalable%20and%20dynamic%20surveillance%20of%20SARS-CoV-2%20variants%20and%20mutations.%20Nat.%20Methods20,%20512%E2%80%93522%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Tsueng, G. et al. Outbreak.info Research Library: a standardized, searchable platform to discover and explore COVID-19 resources. <em>Nat. Methods</em><strong>20</strong>, 536–540 (2023).
</cite> [<a href="https://doi.org/10.1038/s41592-023-01770-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10393269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36823331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tsueng,%20G.%20et%20al.%20Outbreak.info%20Research%20Library:%20a%20standardized,%20searchable%20platform%20to%20discover%20and%20explore%20COVID-19%20resources.%20Nat.%20Methods20,%20536%E2%80%93540%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Chauhan, A. et al. Structural and conformational analysis of SARS CoV 2 N-CTD revealing monomeric and dimeric active sites during the RNA-binding and stabilization: insights towards potential inhibitors for N-CTD. <em>Comput. Biol. Med.</em><strong>134</strong>, 104495 (2021).
</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2021.104495" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8123409/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34022485/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chauhan,%20A.%20et%20al.%20Structural%20and%20conformational%20analysis%20of%20SARS%20CoV%202%20N-CTD%20revealing%20monomeric%20and%20dimeric%20active%20sites%20during%20the%20RNA-binding%20and%20stabilization:%20insights%20towards%20potential%20inhibitors%20for%20N-CTD.%20Comput.%20Biol.%20Med.134,%20104495%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Hu, X. et al. The study of antiviral drugs targeting SARS-CoV-2 nucleocapsid and spike proteins through large-scale compound repurposing. <em>Heliyon</em><strong>7</strong>, e06387 (2021).
</cite> [<a href="https://doi.org/10.1016/j.heliyon.2021.e06387" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7919521/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33688584/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu,%20X.%20et%20al.%20The%20study%20of%20antiviral%20drugs%20targeting%20SARS-CoV-2%20nucleocapsid%20and%20spike%20proteins%20through%20large-scale%20compound%20repurposing.%20Heliyon7,%20e06387%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Chauhan, A. et al. An insight into the simulation directed understanding of the mechanism in SARS CoV-2 N-CTD, dimer integrity, and RNA-binding: Identifying potential antiviral inhibitors. <em>J. Biomol. Struct. Dyn.</em><strong>40</strong>, 13912–13924 (2022).
</cite> [<a href="https://doi.org/10.1080/07391102.2021.1996463" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34751101/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chauhan,%20A.%20et%20al.%20An%20insight%20into%20the%20simulation%20directed%20understanding%20of%20the%20mechanism%20in%20SARS%20CoV-2%20N-CTD,%20dimer%20integrity,%20and%20RNA-binding:%20Identifying%20potential%20antiviral%20inhibitors.%20J.%20Biomol.%20Struct.%20Dyn.40,%2013912%E2%80%9313924%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Márquez-Moñino, M. et al. The ALS drug riluzole binds to the C-terminal domain of SARS-CoV-2 nucleocapsid protein and has antiviral activity. <em>Structure</em>10.1016/j.str.2024.10.025 (2024).
</cite> [<a href="https://doi.org/10.1016/j.str.2024.10.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39541975/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?M%C3%A1rquez-Mo%C3%B1ino,%20M.%20et%20al.%20The%20ALS%20drug%20riluzole%20binds%20to%20the%20C-terminal%20domain%20of%20SARS-CoV-2%20nucleocapsid%20protein%20and%20has%20antiviral%20activity.%20Structure10.1016/j.str.2024.10.025%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Dang, M. &amp; Song, J. Structural basis of anti-SARS-CoV-2 activity of HCQ: specific binding to N protein to disrupt its interaction with nucleic acids and LLPS. <em>QRB Discov.</em><strong>2</strong>, e13 (2021).
</cite> [<a href="https://doi.org/10.1017/qrd.2021.12" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10392676/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37529681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dang,%20M.%20&amp;%20Song,%20J.%20Structural%20basis%20of%20anti-SARS-CoV-2%20activity%20of%20HCQ:%20specific%20binding%20to%20N%20protein%20to%20disrupt%20its%20interaction%20with%20nucleic%20acids%20and%20LLPS.%20QRB%20Discov.2,%20e13%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Ye, Q., Lu, S. &amp; Corbett, K. D. Structural basis for SARS-CoV-2 nucleocapsid protein recognition by single-domain antibodies. <em>Front. Immunol.</em><strong>12</strong>, 719037 (2021).
</cite> [<a href="https://doi.org/10.3389/fimmu.2021.719037" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8351461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34381460/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ye,%20Q.,%20Lu,%20S.%20&amp;%20Corbett,%20K.%20D.%20Structural%20basis%20for%20SARS-CoV-2%20nucleocapsid%20protein%20recognition%20by%20single-domain%20antibodies.%20Front.%20Immunol.12,%20719037%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Narayanan, K. &amp; Makino, S. Cooperation of an RNA packaging signal and a viral envelope protein in coronavirus RNA packaging. <em>J. Virol.</em><strong>75</strong>, 9059–9067 (2001).
</cite> [<a href="https://doi.org/10.1128/JVI.75.19.9059-9067.2001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC114474/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11533169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Narayanan,%20K.%20&amp;%20Makino,%20S.%20Cooperation%20of%20an%20RNA%20packaging%20signal%20and%20a%20viral%20envelope%20protein%20in%20coronavirus%20RNA%20packaging.%20J.%20Virol.75,%209059%E2%80%939067%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Hsieh, P. K. et al. Assembly of severe acute respiratory syndrome coronavirus RNA packaging signal into virus-like particles is nucleocapsid dependent. <em>J. Virol.</em><strong>79</strong>, 13848–13855 (2005).
</cite> [<a href="https://doi.org/10.1128/JVI.79.22.13848-13855.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1280188/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16254320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hsieh,%20P.%20K.%20et%20al.%20Assembly%20of%20severe%20acute%20respiratory%20syndrome%20coronavirus%20RNA%20packaging%20signal%20into%20virus-like%20particles%20is%20nucleocapsid%20dependent.%20J.%20Virol.79,%2013848%E2%80%9313855%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Hsin, W. C. et al. Nucleocapsid protein-dependent assembly of the RNA packaging signal of Middle East respiratory syndrome coronavirus. <em>J. Biomed. Sci.</em><strong>25</strong>, 47 (2018).
</cite> [<a href="https://doi.org/10.1186/s12929-018-0449-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5966903/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29793506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hsin,%20W.%20C.%20et%20al.%20Nucleocapsid%20protein-dependent%20assembly%20of%20the%20RNA%20packaging%20signal%20of%20Middle%20East%20respiratory%20syndrome%20coronavirus.%20J.%20Biomed.%20Sci.25,%2047%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Skittrall, J. P., Irigoyen, N., Brierley, I. &amp; Gog, J. R. A novel approach to finding conserved features in low-variability gene alignments characterises RNA motifs in SARS-CoV and SARS-CoV-2. <em>Sci. Rep.</em><strong>13</strong>, 12079 (2023).
</cite> [<a href="https://doi.org/10.1038/s41598-023-39207-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10372003/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37495730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Skittrall,%20J.%20P.,%20Irigoyen,%20N.,%20Brierley,%20I.%20&amp;%20Gog,%20J.%20R.%20A%20novel%20approach%20to%20finding%20conserved%20features%20in%20low-variability%20gene%20alignments%20characterises%20RNA%20motifs%20in%20SARS-CoV%20and%20SARS-CoV-2.%20Sci.%20Rep.13,%2012079%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Yadav, A. J., Kumar, S., Maurya, S., Bhagat, K. &amp; Padhi, A. K. Interface design of SARS-CoV-2 symmetrical nsp7 dimer and machine learning-guided nsp7 sequence prediction reveals physicochemical properties and hotspots for nsp7 stability, adaptation, and therapeutic design. <em>Phys. Chem. Chem. Phys.</em><strong>26</strong>, 14046–14061 (2024).
</cite> [<a href="https://doi.org/10.1039/d4cp01014k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38686454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yadav,%20A.%20J.,%20Kumar,%20S.,%20Maurya,%20S.,%20Bhagat,%20K.%20&amp;%20Padhi,%20A.%20K.%20Interface%20design%20of%20SARS-CoV-2%20symmetrical%20nsp7%20dimer%20and%20machine%20learning-guided%20nsp7%20sequence%20prediction%20reveals%20physicochemical%20properties%20and%20hotspots%20for%20nsp7%20stability,%20adaptation,%20and%20therapeutic%20design.%20Phys.%20Chem.%20Chem.%20Phys.26,%2014046%E2%80%9314061%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Sharma, A. et al. An integrated multiscale computational framework deciphers SARS-CoV-2 resistance to sotrovimab. <em>Biophys. J.</em>10.1016/j.bpj.2025.05.015 (2025).
</cite> [<a href="https://doi.org/10.1016/j.bpj.2025.05.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40394898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sharma,%20A.%20et%20al.%20An%20integrated%20multiscale%20computational%20framework%20deciphers%20SARS-CoV-2%20resistance%20to%20sotrovimab.%20Biophys.%20J.10.1016/j.bpj.2025.05.015%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Pardi, N. et al. Nucleoside-modified mRNA immunization elicits influenza virus hemagglutinin stalk-specific antibodies. <em>Nat. Commun.</em><strong>9</strong>, 3361 (2018).
</cite> [<a href="https://doi.org/10.1038/s41467-018-05482-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6105651/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30135514/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pardi,%20N.%20et%20al.%20Nucleoside-modified%20mRNA%20immunization%20elicits%20influenza%20virus%20hemagglutinin%20stalk-specific%20antibodies.%20Nat.%20Commun.9,%203361%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Jackson, L. A. et al. An mRNA vaccine against SARS-CoV-2 - preliminary report. <em>N. Engl. J. Med.</em><strong>383</strong>, 1920–1931 (2020).
</cite> [<a href="https://doi.org/10.1056/NEJMoa2022483" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7377258/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32663912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jackson,%20L.%20A.%20et%20al.%20An%20mRNA%20vaccine%20against%20SARS-CoV-2%20-%20preliminary%20report.%20N.%20Engl.%20J.%20Med.383,%201920%E2%80%931931%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Polack, F. P. et al. Safety and efficacy of the BNT162b2 mRNA Covid-19 vaccine. <em>N. Engl. J. Med.</em><strong>383</strong>, 2603–2615 (2020).
</cite> [<a href="https://doi.org/10.1056/NEJMoa2034577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7745181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33301246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Polack,%20F.%20P.%20et%20al.%20Safety%20and%20efficacy%20of%20the%20BNT162b2%20mRNA%20Covid-19%20vaccine.%20N.%20Engl.%20J.%20Med.383,%202603%E2%80%932615%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Yang, J., Luo, Y., Shibu, M. A., Toth, I. &amp; Skwarczynskia, M. Cell-penetrating peptides: efficient vectors for vaccine delivery. <em>Curr. Drug Deliv.</em><strong>16</strong>, 430–443 (2019).
</cite> [<a href="https://doi.org/10.2174/1567201816666190123120915" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6637094/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30760185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20J.,%20Luo,%20Y.,%20Shibu,%20M.%20A.,%20Toth,%20I.%20&amp;%20Skwarczynskia,%20M.%20Cell-penetrating%20peptides:%20efficient%20vectors%20for%20vaccine%20delivery.%20Curr.%20Drug%20Deliv.16,%20430%E2%80%93443%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Xie, C. et al. A cell-penetrating bispecific antibody suppresses hepatitis B virus replication and secretion. <em>Virus Res.</em><strong>353</strong>, 199531 (2025).
</cite> [<a href="https://doi.org/10.1016/j.virusres.2025.199531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11841211/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39863173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie,%20C.%20et%20al.%20A%20cell-penetrating%20bispecific%20antibody%20suppresses%20hepatitis%20B%20virus%20replication%20and%20secretion.%20Virus%20Res.353,%20199531%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Khanefard, N. et al. Influenza neuraminidase virus-like particle-based nanocarriers as a new platform for the delivery of small-peptide antigens. <em>Mol. Biotechnol.</em>10.1007/s12033-025-01403-x (2025).
</cite> [<a href="https://doi.org/10.1007/s12033-025-01403-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40021575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khanefard,%20N.%20et%20al.%20Influenza%20neuraminidase%20virus-like%20particle-based%20nanocarriers%20as%20a%20new%20platform%20for%20the%20delivery%20of%20small-peptide%20antigens.%20Mol.%20Biotechnol.10.1007/s12033-025-01403-x%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Böldicke, T. Therapeutic potential of intrabodies for cancer immunotherapy: current dtatus and future directions. <em>Antibodies</em>10.3390/antib11030049 (2022).</cite> [<a href="https://doi.org/10.3390/antib11030049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9326752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35892709/" class="usa-link">PubMed</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Hanke, L. et al. Vesicular stomatitis virus N protein-specific single-domain antibody fragments inhibit replication. <em>EMBO Rep.</em><strong>18</strong>, 1027–1037 (2017).
</cite> [<a href="https://doi.org/10.15252/embr.201643764" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5452021/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28396572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hanke,%20L.%20et%20al.%20Vesicular%20stomatitis%20virus%20N%20protein-specific%20single-domain%20antibody%20fragments%20inhibit%20replication.%20EMBO%20Rep.18,%201027%E2%80%931037%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Wongsodirdjo, P. et al. Messenger RNA-encoded antibody approach for targeting extracellular and intracellular tau. <em>Brain Commun.</em><strong>6</strong>, fcae100 (2024).
</cite> [<a href="https://doi.org/10.1093/braincomms/fcae100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10996922/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38585667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wongsodirdjo,%20P.%20et%20al.%20Messenger%20RNA-encoded%20antibody%20approach%20for%20targeting%20extracellular%20and%20intracellular%20tau.%20Brain%20Commun.6,%20fcae100%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Ding, H. et al. A semi-quantitative upconversion nanoparticle-based immunochromatographic assay for SARS-CoV-2 antigen detection. <em>Front. Microbiol.</em><strong>14</strong>, 1289682 (2023).
</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1289682" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10750388/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38149276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ding,%20H.%20et%20al.%20A%20semi-quantitative%20upconversion%20nanoparticle-based%20immunochromatographic%20assay%20for%20SARS-CoV-2%20antigen%20detection.%20Front.%20Microbiol.14,%201289682%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365033/bin/42003_2025_8648_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (23.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365033/bin/42003_2025_8648_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary data 1</a><sup> (1.6MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365033/bin/42003_2025_8648_MOESM3_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">42003_2025_8648_MOESM3_ESM.docx</a><sup> (20.9KB, docx) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365033/bin/42003_2025_8648_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting summary</a><sup> (2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365033/bin/42003_2025_8648_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (29.4MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The structures in this paper are deposited to the Protein Data Bank with 9KUR access code in Supplementary Table <a href="#MOESM1" class="usa-link">S1</a>. Uncropped gel images are presented as Supplementary Fig. <a href="#MOESM1" class="usa-link">S19</a>. Source data for statistical analyses are provided in Supplementary Data <a href="#MOESM2" class="usa-link">1</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Communications Biology are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s42003-025-08648-x"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/42003_2025_Article_8648.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365033/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365033/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365033%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365033/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365033/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365033/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830575/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365033/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830575/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365033/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365033/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="gmTZifzQpD8bXf1aXsEsYYilRh9cHZcTvdidvMvye2vabRehWHh73wrjuOmNzbFB">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
