
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            A computational workflow for assessing drug effects on temporal signaling dynamics reveals robustness in stimulus-specific NFκB signaling - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AD4A8AF2ED8305AD4A001B0A6FA8.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ploscomp">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370059/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS Computational Biology">
<meta name="citation_title" content="A computational workflow for assessing drug effects on temporal signaling dynamics reveals robustness in stimulus-specific NFκB signaling">
<meta name="citation_author" content="Emily R Bozich">
<meta name="citation_author_institution" content="Department of Bioengineering, University of California, Los Angeles, California, United States of America">
<meta name="citation_author_institution" content="Institute for Quantitative and Computational Biosciences, University of California, Los Angeles, California, United States of America">
<meta name="citation_author" content="Xiaolu Guo">
<meta name="citation_author_institution" content="Institute for Quantitative and Computational Biosciences, University of California, Los Angeles, California, United States of America">
<meta name="citation_author_institution" content="Department of Microbiology, Immunology and Molecular Genetics, University of California, Los Angeles, California, United States of America">
<meta name="citation_author" content="Jennifer L Wilson">
<meta name="citation_author_institution" content="Department of Bioengineering, University of California, Los Angeles, California, United States of America">
<meta name="citation_author_institution" content="Institute for Quantitative and Computational Biosciences, University of California, Los Angeles, California, United States of America">
<meta name="citation_author" content="Alexander Hoffmann">
<meta name="citation_author_institution" content="Institute for Quantitative and Computational Biosciences, University of California, Los Angeles, California, United States of America">
<meta name="citation_author_institution" content="Department of Microbiology, Immunology and Molecular Genetics, University of California, Los Angeles, California, United States of America">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="21">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e1013344">
<meta name="citation_doi" content="10.1371/journal.pcbi.1013344">
<meta name="citation_pmid" content="40839679">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370059/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370059/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370059/pdf/pcbi.1013344.pdf">
<meta name="description" content="Single-cell studies of signal transduction have revealed complex temporal dynamics that determine downstream biological function. For example, the stimulus-specific dynamics of the transcription factor NFκB specify stimulus-specific gene expression ...">
<meta name="og:title" content="A computational workflow for assessing drug effects on temporal signaling dynamics reveals robustness in stimulus-specific NFκB signaling">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Single-cell studies of signal transduction have revealed complex temporal dynamics that determine downstream biological function. For example, the stimulus-specific dynamics of the transcription factor NFκB specify stimulus-specific gene expression ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370059/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370059">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pcbi.1013344"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pcbi.1013344.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370059%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370059/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370059/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370059/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ploscomp.png" alt="PLOS Computational Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS Computational Biology" title="Link to PLOS Computational Biology" shape="default" href="https://doi.org/10.1371/journal.pcbi.1013344" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS Comput Biol</button></div>. 2025 Aug 21;21(8):e1013344. doi: <a href="https://doi.org/10.1371/journal.pcbi.1013344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pcbi.1013344</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20Comput%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20Comput%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20Comput%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20Comput%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>A computational workflow for assessing drug effects on temporal signaling dynamics reveals robustness in stimulus-specific NFκB signaling</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bozich%20ER%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Emily R Bozich</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Emily R Bozich</span></h3>
<div class="p">
<sup>1</sup>Department of Bioengineering, University of California, Los Angeles, California, United States of America</div>
<div class="p">
<sup>2</sup>Institute for Quantitative and Computational Biosciences, University of California, Los Angeles, California, United States of America</div>
<div>Conceptualization, Formal analysis, Investigation, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bozich%20ER%22%5BAuthor%5D" class="usa-link"><span class="name western">Emily R Bozich</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Xiaolu Guo</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Xiaolu Guo</span></h3>
<div class="p">
<sup>2</sup>Institute for Quantitative and Computational Biosciences, University of California, Los Angeles, California, United States of America</div>
<div class="p">
<sup>3</sup>Department of Microbiology, Immunology and Molecular Genetics, University of California, Los Angeles, California, United States of America</div>
<div>Conceptualization, Formal analysis, Investigation, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaolu Guo</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilson%20JL%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jennifer L Wilson</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jennifer L Wilson</span></h3>
<div class="p">
<sup>1</sup>Department of Bioengineering, University of California, Los Angeles, California, United States of America</div>
<div class="p">
<sup>2</sup>Institute for Quantitative and Computational Biosciences, University of California, Los Angeles, California, United States of America</div>
<div>Funding acquisition, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wilson%20JL%22%5BAuthor%5D" class="usa-link"><span class="name western">Jennifer L Wilson</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hoffmann%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Alexander Hoffmann</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Alexander Hoffmann</span></h3>
<div class="p">
<sup>2</sup>Institute for Quantitative and Computational Biosciences, University of California, Los Angeles, California, United States of America</div>
<div class="p">
<sup>3</sup>Department of Microbiology, Immunology and Molecular Genetics, University of California, Los Angeles, California, United States of America</div>
<div>Conceptualization, Funding acquisition, Project administration, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hoffmann%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexander Hoffmann</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>*</sup>
</div>
<div class="cg p">Editor: <span class="name western">Anders Wallqvist</span><sup>4</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>Department of Bioengineering, University of California, Los Angeles, California, United States of America</div>
<div id="aff002">
<sup>2</sup>Institute for Quantitative and Computational Biosciences, University of California, Los Angeles, California, United States of America</div>
<div id="aff003">
<sup>3</sup>Department of Microbiology, Immunology and Molecular Genetics, University of California, Los Angeles, California, United States of America</div>
<div id="edit1">
<sup>4</sup>US ArUS Army Medical Research and Development Command, UNITED STATES OF AMERICA</div>
<div class="author-notes p">
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>ahoffmann@ucla.edu</span> (AH); <span>xiaoluguo@g.ucla.edu</span> (XG)</p>
</div>
<div class="fn" id="coi001"><p>The authors have declared that no competing interests exist.</p></div>
<div class="fn" id="fn001">
<sup>#</sup><p class="display-inline">☯ These authors contributed equally to this work.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Emily R Bozich</span></strong>: <span class="role">Conceptualization, Formal analysis, Investigation, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Xiaolu Guo</span></strong>: <span class="role">Conceptualization, Formal analysis, Investigation, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jennifer L Wilson</span></strong>: <span class="role">Funding acquisition, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Alexander Hoffmann</span></strong>: <span class="role">Conceptualization, Funding acquisition, Project administration, Writing – review &amp; editing</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Anders Wallqvist</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 3; Accepted 2025 Jul 17; Collection date 2025 Aug.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Bozich et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370059  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40839679/" class="usa-link">40839679</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article"><div>
<strong>Previous version available:</strong> This article is based on a previously available preprint posted on bioRxiv on April 2, 2025: "<a href="/articles/PMC11996442/" class="usa-link">
A computational workflow for assessing drug effects on temporal signaling dynamics reveals robustness in stimulus-specific NFκB signaling
</a>".</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Single-cell studies of signal transduction have revealed complex temporal dynamics that determine downstream biological function. For example, the stimulus-specific dynamics of the transcription factor NFκB specify stimulus-specific gene expression programs, and loss of specificity leads to disease. Thus, it is intriguing to consider drugs that may restore signaling specificity in disease contexts, or reduce activity but maintain signaling specificity to avoid unwanted side effects. However, while steady-state dose-response relationships have been the focus of pharmacological studies, there are no established methods for quantifying drug impact on stimulus-response signaling dynamics. Here we evaluated how drug treatments affect the stimulus-specificity of NFκB activation dynamics and its ability to accurately code ligand identity and dose. Specifically, we simulated the dynamic NFκB trajectories in response to 15 stimuli representing various immune threats under treatment of 10 representative drugs across 20 dosage levels. To quantify the effects on coding capacity, we introduced a Stimulus Response Specificity (SRS) score and a stimulus confusion score. We constructed stimulus confusion maps by employing epsilon network clustering in the trajectory space and in various dimensionally reduced spaces: canonical polyadic decomposition (CPD), functional principal component analysis (fPCA), and NFκB signaling codons (i.e., established, informative dynamic features). Our results indicated that the SRS score and the stimulus confusion map based on signaling codons are best-suited to quantify stimulus-specific NFκB dynamics confusion under pharmacological perturbations. Using these tools we found that temporal coding capacity of the NFκB signaling network is generally robust to a variety of pharmacological perturbations, thereby enabling the targeting of stimulus-specific dynamics without causing broad side-effects.</p></section><section class="abstract" id="abstract2"><h2>Author summary</h2>
<p>Immune- and stress-response signaling pathways are functionally pleiotropic: they respond to multiple stimuli and mediate distinct biological responses. For innate immune cells distinguishing different immune threats is important, and loss of stimulus-response specificity is linked to autoimmune or inflammatory diseases. This pleiotropy has posed a challenge for the development of safe pharmacologic interventions, and many promising drug compounds have turned out to result in debilitating side effects. Stimulus-response specificity is mediated by the dynamics of cellular signaling, constituting a “temporal code.” This raises the question of whether stimulus responses may be pharmacologically targeted without disrupting stimulus-response specificity, or to restore stimulus-response specificity. However, how to effectively measure and quantify the stimulus-response specificity of signaling dynamics remains unclear. Here, we developed and tested computational metrics on high-throughput simulation data to assess drug effects on the stimulus-response specificity of the NFκB pathway. The results advance both the study of temporal signaling codes and strategies for precision drug targeting.</p></section><section id="sec001"><h2 class="pmc_sec_title">Introduction</h2>
<p>Signaling pathways respond to environmental stimuli and direct downstream gene expression. Environmental information may be encoded in the dynamics of signaling activity, constituting a temporal signaling code [<a href="#pcbi.1013344.ref001" class="usa-link" aria-describedby="pcbi.1013344.ref001">1</a>–<a href="#pcbi.1013344.ref003" class="usa-link" aria-describedby="pcbi.1013344.ref003">3</a>]. In recent years, advances in live-cell imaging techniques and the optimization of analytical software have enabled the single-cell study of the temporal profiles of key signaling pathways, including NFκB [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>–<a href="#pcbi.1013344.ref008" class="usa-link" aria-describedby="pcbi.1013344.ref008">8</a>], MAPKs (ERK, JNK, p38) [<a href="#pcbi.1013344.ref009" class="usa-link" aria-describedby="pcbi.1013344.ref009">9</a>,<a href="#pcbi.1013344.ref010" class="usa-link" aria-describedby="pcbi.1013344.ref010">10</a>], p53 [<a href="#pcbi.1013344.ref011" class="usa-link" aria-describedby="pcbi.1013344.ref011">11</a>], and their combinations [<a href="#pcbi.1013344.ref012" class="usa-link" aria-describedby="pcbi.1013344.ref012">12</a>,<a href="#pcbi.1013344.ref013" class="usa-link" aria-describedby="pcbi.1013344.ref013">13</a>]. These studies have revealed that the activated signaling dynamics are stimulus-specific, and that stimulus-specific dynamics regulate gene expression responses and the resulting biological functions, while dysregulation of this temporal coding is associated with various pathological conditions [<a href="#pcbi.1013344.ref001" class="usa-link" aria-describedby="pcbi.1013344.ref001">1</a>,<a href="#pcbi.1013344.ref014" class="usa-link" aria-describedby="pcbi.1013344.ref014">14</a>–<a href="#pcbi.1013344.ref016" class="usa-link" aria-describedby="pcbi.1013344.ref016">16</a>].</p>
<p>Theoretical studies have explored the dynamics of signaling pathways as new opportunities for therapeutic intervention [<a href="#pcbi.1013344.ref016" class="usa-link" aria-describedby="pcbi.1013344.ref016">16</a>]. Stimulus-specific signaling dynamics have been proposed as drug targets, as demonstrated in studies of NFκB [<a href="#pcbi.1013344.ref016" class="usa-link" aria-describedby="pcbi.1013344.ref016">16</a>], p53 [<a href="#pcbi.1013344.ref011" class="usa-link" aria-describedby="pcbi.1013344.ref011">11</a>], ERK dynamics [<a href="#pcbi.1013344.ref017" class="usa-link" aria-describedby="pcbi.1013344.ref017">17</a>,<a href="#pcbi.1013344.ref018" class="usa-link" aria-describedby="pcbi.1013344.ref018">18</a>], PI3K [<a href="#pcbi.1013344.ref019" class="usa-link" aria-describedby="pcbi.1013344.ref019">19</a>], and the MAPK signaling network [<a href="#pcbi.1013344.ref020" class="usa-link" aria-describedby="pcbi.1013344.ref020">20</a>]. This emerging shift toward designing pharmacological interventions that target the dynamics of signaling offers greater precision than traditional approaches targeting steady-state dose-response relationships.</p>
<p>Despite the importance of stimulus-response specificity (SRS) in mammalian signaling systems, how to quantify how pharmacological interventions affect SRS remains an open question. Drugs may be designed to act on a single molecular target, modulate signaling dynamics, or simultaneously influence multiple nodes within a signaling network. Given the functional pleiotropy of most signaling pathways, such interventions can result in broad and potentially non-specific effects [<a href="#pcbi.1013344.ref021" class="usa-link" aria-describedby="pcbi.1013344.ref021">21</a>–<a href="#pcbi.1013344.ref024" class="usa-link" aria-describedby="pcbi.1013344.ref024">24</a>]. This raises key questions: if a drug affects a specific signaling pathway upstream of the signaling node, how does that impact the SRS of the node? Conversely, if a drug affects multiple pathways [<a href="#pcbi.1013344.ref016" class="usa-link" aria-describedby="pcbi.1013344.ref016">16</a>], does it also disrupt SRS as a side effect?</p>
<p>While the theoretical work established that pharmacological perturbation of signaling dynamics is possible, it remains unclear how stimulus-specific the effects of such perturbations are. Previous studies have primarily focused on specific biochemical reaction rates within abstracted and simplified regulatory networks to alter signal transduction specificity and fidelity [<a href="#pcbi.1013344.ref016" class="usa-link" aria-describedby="pcbi.1013344.ref016">16</a>,<a href="#pcbi.1013344.ref025" class="usa-link" aria-describedby="pcbi.1013344.ref025">25</a>,<a href="#pcbi.1013344.ref026" class="usa-link" aria-describedby="pcbi.1013344.ref026">26</a>]. However, it remains unclear how these perturbations affect stimulus-specific responses by more complex, physiologically relevant signaling networks that are functionally pleiotropic. In fact, we currently lack established methods for evaluating the efficacy and specificity of pharmacological targeting stimulus-response signaling dynamics. To address these questions, we have selected a biological system that exhibits stimulus-specific signaling dynamics in response to a wide range of different stimuli and for which a mathematical model that accounts for the stimulus-response signaling dynamics is available: the NFκB signaling system in macrophages.</p>
<p>NFκB plays critical roles in regulating immune responses to pathogen-associated molecular patterns (PAMPs), damage-associated molecular patterns (DAMPs), and cytokines via regulating transcription, cytokine production, cell division and death, etc [<a href="#pcbi.1013344.ref027" class="usa-link" aria-describedby="pcbi.1013344.ref027">27</a>]. NFκB signaling dynamics encode information about immune threats [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>,<a href="#pcbi.1013344.ref015" class="usa-link" aria-describedby="pcbi.1013344.ref015">15</a>,<a href="#pcbi.1013344.ref028" class="usa-link" aria-describedby="pcbi.1013344.ref028">28</a>,<a href="#pcbi.1013344.ref029" class="usa-link" aria-describedby="pcbi.1013344.ref029">29</a>] and regulate immune genes to achieve appropriate immune responses corresponding to the specific stimuli [<a href="#pcbi.1013344.ref030" class="usa-link" aria-describedby="pcbi.1013344.ref030">30</a>–<a href="#pcbi.1013344.ref037" class="usa-link" aria-describedby="pcbi.1013344.ref037">37</a>]. Loss of specificity in immune cell responses has been associated with autoimmune diseases [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>], and other pathological conditions [<a href="#pcbi.1013344.ref015" class="usa-link" aria-describedby="pcbi.1013344.ref015">15</a>,<a href="#pcbi.1013344.ref038" class="usa-link" aria-describedby="pcbi.1013344.ref038">38</a>]. Recently, we identified six informative dynamic features of NFκB signaling, termed “signaling codons”, that together are sufficient to encode stimulus-specific information [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>]. Specifically, “Speed” (Speed) captures the activation speed, “Peak Amplitude” (Amp) refers to the highest response level, “Duration” (Dur) captures the total time NFκB levels remain above an activation threshold, “Total Activity” (AUC) measures the overall accumulation of NFκB activity, while “Early vs. Late” (EvL) represents the front-loading of NFκB activity and “Oscillatory Power” (Osc) quantifies the oscillation properties of the signaling trajectory.</p>
<p>Prior work also described a mathematical model of the NFκB signaling system that accounts for the temporal trajectories of NFκB that are representative of experimental single-cell stimulus-response datasets [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>]. The model consists of receptor-associated signaling modules (receptor module) that respond to five representative ligands (TNF representing cytokines, LPS, CpG, and Pam3CSK representing bacterial derived molecules, Poly(I:C) representing virus-derived molecules) and a common core module that includes the kinases TAK1 and IKK and the IκBα-NFκB negative feedback loop. The topology of this mathematical model is well supported by decades of experimental studies and the parameterization is based on numerous publications starting in 2002 [<a href="#pcbi.1013344.ref030" class="usa-link" aria-describedby="pcbi.1013344.ref030">30</a>,<a href="#pcbi.1013344.ref039" class="usa-link" aria-describedby="pcbi.1013344.ref039">39</a>].</p>
<p>To assess the impact of pharmacological perturbations on the integrity of the NFκB signaling network’s temporal coding capacity (i.e., its ability to convey stimulus-specific responses), we selected 10 inhibitors targeting biomolecules involved in NFκB signaling pathways, each tested at 20 dosages. Leveraging the well-established mathematical model of the NFκB signaling network allowed us to generate synthetic data representing cellular responses to different stimuli under these drug regimes. With this immense amount of synthetic perturbation data, we developed computational methods—stimulus confusion maps and stimulus confusion scores—to evaluate alterations in stimulus-response specificity resulting from pharmacological perturbations of the NFκB signaling network. We demonstrated the capacity of stimulus confusion maps to (1) identify pharmacological perturbations that may enhance or diminish stimulus-specific NFκB signaling dynamics; (2) select drug regimens that maximize or minimize the stimulus specificity of NFκB pathways; and (3) evaluate the side effects of drugs on altering the stimulus-response specificity of NFκB temporal patterns. Applying these tools shows the robustness of temporal coding capacity of the NFκB signaling network under a variety of pharmacological perturbations.</p></section><section id="sec002"><h2 class="pmc_sec_title">Results</h2>
<section id="sec003"><h3 class="pmc_sec_title">A computational workflow to assess how stimulus-specific NFκB signaling dynamics may be modulated</h3>
<p>Pharmacological perturbations can affect the temporal coding capacity of the innate response network that ultimately controls the dynamical activation of NFκB. To investigate this at scale, we developed a computational workflow. The workflow is anchored by an experimentally validated systems model comprised of interconnected ordinary differential equations (ODEs) [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>]. This model recapitulates the temporal trajectories of nuclear NFκB activity in response to a range of doses of each of five pro-inflammatory ligands, and shows a high degree of specificity in terms of ligand- and dose-specific NFκB dynamics and therefore high temporal coding capacity. The model comprises 94 biochemical reaction that are controlled by 126 kinetic parameters, which may be subject to pharmacologic perturbation. Thus, the model may be used to generate data that addresses how pharmacological perturbation may modulate the coding capacity of the innate immune signaling network (<a href="#pcbi.1013344.g001" class="usa-link">Fig 1A</a>).</p>
<figure class="fig xbox font-sm" id="pcbi.1013344.g001"><h4 class="obj_head">Fig 1. A computational workflow for identifying pharmacological perturbations that alter the temporal coding capacity of the NFκB signaling network.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370059_pcbi.1013344.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f6e6/12370059/6aaff1dfb0fb/pcbi.1013344.g001.jpg" loading="lazy" height="648" width="750" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013344.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Schematic of the NFκB network model. Five ligands bind to their cognate receptors triggering associated signaling modules that activate the kinase IKK which causes degradation of IκBα and hence de-repression of NFκB activity. There are numerous pharmacological agents that interfere with different reactions within this NFκB signaling network. <strong>(B)</strong> Table of pharmacological compounds that inhibit biochemical processes within the NFκB signaling network. Numbers indicate the kinetic parameters that are affected by the indicated compound. <strong>(C)</strong> The computational workflow begins with simulations that yield NFκB trajectories in response to 5 ligands (TNF, LPS, CpG, Poly(I:C), and Pam3CSK) at low/medium/high doses in the presence of pharmacological treatment with one of 10 compounds at one of 20 doses. The resulting 3000 trajectories are defined by 481 timepoints (every 5 min for 8 hrs) which are then reduced through (1) canonical polyadic (CP) decomposition, (2) functional principal component analysis (fPCA), or (3) quantifications of previously identified informative trajectory features, known as signaling codons. Using the outputted features from dimensionality reduction, epsilon network clustering is applied to the 15 stimulus conditions (colored by stimuli) within a drug regime. Cluster-based stimulus confusion, specificity, and inhibition scores are calculated to reveal the pharmacological effects on stimulus-response specificity of NFκB dynamics, also described as the temporal coding capacity of the NFκB signaling pathway.</p></figcaption></figure><p>The stimulating ligands were selected to span a variety of immune threats, including bacteria, virus, and cytokines. The ligand doses were chosen using a log-spaced dosage scheme to capture diverse dynamic behaviors, guided by dose span analyses from experimental data. The simulated trajectories under these doses are directly validated by both representative single-cell NFκB trajectories (from live-cell imaging) and bulk-level IKK activity measurements (via western blots) [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>]. These doses yield simulations that most closely resemble the corresponding biological experimental data. In addition, they were chosen to capture a broad range of dynamic features, including variability in peak values, oscillatory behavior, integral activity, activation speed, timing (early vs. late responses), and duration (<a href="#pcbi.1013344.s002" class="usa-link">S1 Fig</a>). Specifically, LPS and CpG span a spectrum from minimal peak values to high amplitude responses; TNF conditions reflect transitions from minimal to high oscillations at similar peak levels, with low doses producing only early activity and high doses inducing sustained late responses; Pam3CSK demonstrates a shift from low-amplitude oscillations to high, non-oscillatory responses; and PolyIC highlights variation in activation speed.</p>
<p>We identified 10 pharmacological compounds with well-described biochemical mechanisms of action (<a href="#pcbi.1013344.g001" class="usa-link">Fig 1B</a>, <a href="#pcbi.1013344.s001" class="usa-link">S1 Table</a>). As such, we could map each compound to the affected kinetic parameters within the ODE model. This library includes compounds that impact the core module: IKK inhibitor (IKKi) [<a href="#pcbi.1013344.ref040" class="usa-link" aria-describedby="pcbi.1013344.ref040">40</a>,<a href="#pcbi.1013344.ref041" class="usa-link" aria-describedby="pcbi.1013344.ref041">41</a>], TAK1 inhibitor (TAK1i) [<a href="#pcbi.1013344.ref040" class="usa-link" aria-describedby="pcbi.1013344.ref040">40</a>], Beta-TRCP inhibitor (BTRCPi) [<a href="#pcbi.1013344.ref042" class="usa-link" aria-describedby="pcbi.1013344.ref042">42</a>,<a href="#pcbi.1013344.ref043" class="usa-link" aria-describedby="pcbi.1013344.ref043">43</a>], Phosphatase inhibitor (PP2Ai) [<a href="#pcbi.1013344.ref044" class="usa-link" aria-describedby="pcbi.1013344.ref044">44</a>,<a href="#pcbi.1013344.ref045" class="usa-link" aria-describedby="pcbi.1013344.ref045">45</a>], transcription enhancer (TSA) [<a href="#pcbi.1013344.ref046" class="usa-link" aria-describedby="pcbi.1013344.ref046">46</a>], NFκB-induced transcription inhibitor (PDTC) [<a href="#pcbi.1013344.ref016" class="usa-link" aria-describedby="pcbi.1013344.ref016">16</a>], translation inhibitor (CHX) [<a href="#pcbi.1013344.ref016" class="usa-link" aria-describedby="pcbi.1013344.ref016">16</a>], and nuclear export inhibitor (Sel) [<a href="#pcbi.1013344.ref047" class="usa-link" aria-describedby="pcbi.1013344.ref047">47</a>]. A proteasome inhibitor, MG132, inhibits the degradation of IκBα, whether complexed to NFκB or not, in the core module, and several receptors and their associated complexes in receptor modules [<a href="#pcbi.1013344.ref016" class="usa-link" aria-describedby="pcbi.1013344.ref016">16</a>]. Lysosome inhibitor (CHL) inhibits the degradation of CD14 and several toll-like receptors (TLRs) trafficking to the plasma membrane and endosomes [<a href="#pcbi.1013344.ref048" class="usa-link" aria-describedby="pcbi.1013344.ref048">48</a>]. The selected compounds are the inhibitors/enhancers of kinase/enzymes in the NFκB signaling pathway, so which parameters to change is constrained by their biochemical functions in the signaling pathway.</p>
<p>In order to quantify the stimulus-specific temporal coding capacity of NFκB under pharmacological perturbation, we designed a computational workflow. The workflow consists of the following steps: (1) generation of 3000 NFκB simulated trajectories in response to 15 stimulus conditions and under 200 pharmacological alterations (10 drugs at 20 doses resulting in 200 drug doses, DD) to our ODE model; (2) dimensionality reduction of the simulated trajectories using either matrix-based decomposition methods or decomposition into informative dynamic signaling features that were identified using information theory [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>]; (3) in the reduced dimension space, epsilon network clustering of simulated trajectories; (4) quantification of the confusion and specificity effects within each regime through the calculation of corresponding scores (<a href="#pcbi.1013344.g001" class="usa-link">Fig 1C</a>). The goal of this workflow is to characterize the extent to which drug treatments (ranging from attenuating and enhancing effects on NFκB activation) may affect the stimulus-specific dynamics of the NFκB signaling module, either through maintaining or diminishing its temporal coding capacity.</p></section><section id="sec004"><h3 class="pmc_sec_title">Generation of 3000 pharmacologically perturbed NFκB temporal trajectories</h3>
<p>Using the NFκB systems model, we first generated stimulus-responsive nuclear NFκB temporal trajectories in response to 5 ligands (TNF, LPS, CpG, Poly(I:C), and Pam3CSK) at 3 doses (15 total stimulus conditions). These 15 trajectories are stimulus-specific and hence encapsulate NFκB’s temporal coding capacity. Next, we applied the 10 previously described pharmacological compounds to our model by modifying their corresponding kinetic parameters at 20 log-linearly spaced drug doses (DD) ranging from <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e001"><math id="M1" display="inline" overflow="linebreak"><mrow><msup><mn>10</mn><mrow><mo>−</mo><mfrac><mrow><mn>3</mn></mrow><mrow><mn>20</mn></mrow></mfrac></mrow></msup></mrow></math></span>x (DD1) to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e002"><math id="M2" display="inline" overflow="linebreak"><mrow><msup><mn>10</mn><mrow><mo>−</mo><mn>3</mn></mrow></msup></mrow></math></span>x (DD20). This resulted in 3000 simulated NFκB trajectories, representing nuclear NFκB concentration over an 8-hour stimulation time course (<a href="#pcbi.1013344.g002" class="usa-link">Fig 2</a>).</p>
<figure class="fig xbox font-sm" id="pcbi.1013344.g002"><h4 class="obj_head">Fig 2. NFκB signaling trajectories under pharmacological perturbations.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370059_pcbi.1013344.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f6e6/12370059/510f3a834abb/pcbi.1013344.g002.jpg" loading="lazy" height="981" width="776" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013344.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Stimulus-specific NFκB trajectories (0 – 8 hours) following computational compound perturbations. The stimulating ligands are indicated at the top labels. The doses of these stimuli (Low, Medium, and High) are denoted by small labels on the left. A total of 21 trajectories are presented in each graph, from the untreated condition (depicted in black) to a compound dose (CD) of 20 (depicted in bright red). The colorbar on the right corresponds to specific compound dose indices.</p></figcaption></figure><p>Visual inspection of the simulated trajectories suggests that across all 15 stimulus conditions, most compounds cause a reduction in NFκB activation with increasing compound doses. Exceptions include the transcription and translation inhibitors PDTC and CHX and the lysosome inhibitor CHL. By inhibiting the production of the NFκB inhibitor, IκBα, PDTC and CHX treatment enhances nuclear NFκB activity. Similarly, with increasing doses of CHL, lysosome-dependent receptor degradation decreases, resulting in maintained (TNF-stimulated) and enhanced (LPS, CpG, Poly(I:C), and Pam3CSK-stimulated) activation of the NFκB signaling network. By examining dynamic features including “Speed”, “Peak Amplitude”, “Duration”, “Area Under the Curve (AUC)”, “Early vs Late”, and “Oscillatory Power” [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>] (See methods for details), we observed that these simulated pharmacological perturbations significantly alter the NFκB signaling dynamics (<a href="#pcbi.1013344.s003" class="usa-link">S2 Fig</a>).</p>
<p>However, it remains unclear how the stimulus-specific temporal coding capacity of NFκB dynamics is affected, especially in conditions of attenuating drug treatments. For example, while Sel and MG132 both reduce NFκB activation at increasing doses, they do so through altering signaling dynamic features that could thereby affect the stimulus-specificity of the stimulus-response trajectories. Across stimuli, we observe that at intermediate drug doses, Sel generally maintains the same trajectory shape as each stimuli’s untreated trajectory whereas MG132 leads to modulation of oscillatory behavior. Thus, to elucidate compound-specific effects on stimulus-specific signaling dynamics, we applied the next step in our computational workflow, dimensionality reduction, in order to represent the 3000 complex temporal trajectories by key, interpretable features.</p></section><section id="sec005"><h3 class="pmc_sec_title">Defining stimulus-response specificity landscapes for drug regimes</h3>
<p>NFκB signaling is stimulus-response specific, and a loss of this stimulus-response specificity has been linked to autoimmune diseases, such as Sjogren’s Syndrome [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>]. To quantify the stimulus-response specificity, we sought a dimensionality reduction method that reduced data complexity but encoded stimulus-specific information of the signaling dynamics. Given the immense high-dimensional and time-dependent information within our dataset, we set out to test the utility of three different dimensionality reduction methods which prioritize distinct characteristics within our data: canonical polyadic decomposition, functional principal component analysis, and signaling codons. Within each low-dimensional space, the temporal trajectories are represented by a method-specific feature set. While these techniques were applied to all 200 drug regimes, we selected two representative regimes, Sel at DD15 and MG132 at DD6 (for simplicity, referred to as Sel and MG132 for the remainder of this section), to demonstrate the functionality of each dimensionality reduction method.</p>
<p>We first searched for general trends in the pharmacological effects within the original trajectory space. In doing so, we observed that Sel treatment, the nuclear export inhibitor (<a href="#pcbi.1013344.g003" class="usa-link">Fig 3A</a>), reduces NFκB oscillations while generally maintaining each trajectory’s initial peak (untreated vs. treated in <a href="#pcbi.1013344.g003" class="usa-link">Fig 3B</a>). We also noted this treatment’s similar effects on trajectories stimulated by medium doses of LPS and CpG (stimulation indices 5 and 8). In addition, MG132, the proteasome inhibitor (<a href="#pcbi.1013344.g003" class="usa-link">Fig 3C</a>), causes stimulus- and dose-specific alterations to NFκB dynamics (treated vs. untreated in <a href="#pcbi.1013344.g003" class="usa-link">Fig 3D</a>). For instance, all TNF- and Pam3CSK-stimulated trajectories have uniform oscillations (simulation indices 1–3 &amp; 13–15) which can be attributed to their respective receptor’s (TNFR and TLR2) proteasome-mediated degradation. For these 6 stimulus conditions, treatment with MG132 results in competition between reduced receptor degradation and reduced IκBα degradation which yields similar trajectory shapes (simulation indices 1–3 &amp; 13–15 in <a href="#pcbi.1013344.g003" class="usa-link">Fig 3D</a>). These Sel- and MG132-treated trajectories were then compared to their feature vectors in the lower dimensional spaces (described in detail below) to determine the extent to which each technique recapitulates temporal dynamics.</p>
<figure class="fig xbox font-sm" id="pcbi.1013344.g003"><h4 class="obj_head">Fig 3. Feature/Reduced-dimensional representation of the trajectories.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370059_pcbi.1013344.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f6e6/12370059/8227d11a8d8b/pcbi.1013344.g003.jpg" loading="lazy" height="981" width="792" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013344.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Schematic of how example drug treatment Sel DD15 affects the NFκB signaling network. <strong>(B)</strong> Trajectories of NFκB activity over time for drug treatment Sel DD15 across 5 ligands and 3 doses. Solid colored lines represent the trajectories under drug treatment, while dashed lines depict untreated trajectories. Rows indicate different ligands (labeled on the right) and columns specify ligand doses (labeled on the top of the panel). Stimulation indices are labeled in the top right corner. <strong>(C)</strong> Schematic of how example drug treatment MG132 DD6 affects the NFκB signaling network. <strong>(D)</strong> Trajectories of NFκB activity over time for drug treatment MG132 DD6 shown as described in panel B. <strong>(E)</strong> Temporal patterns corresponding to seven CPD components identified by decomposing the entire NFκB trajectory dataset shown in <a href="#pcbi.1013344.g002" class="usa-link">Fig 2</a>. <strong>(F)</strong> Heatmap representations of the weights of the indicated seven CPD components shown in panel E for the Sel DD15 data shown in panel B. <strong>(G)</strong> Heatmap representations of the weights of the indicated seven CPD components shown in panel E for the MG132 DD6 data shown in panel D. <strong>(H)</strong> Temporal patterns corresponding to five fPCA eigenfunctions (components) identified by decomposing the entire NFκB trajectory dataset shown in <a href="#pcbi.1013344.g002" class="usa-link">Fig 2</a>. <strong>(I)</strong> Heatmap representations of the weights of the indicated five fPCA eigenfunctions shown in panel H for the Sel DD15 data shown in panel B. <strong>(J)</strong> Heatmap representations of the weights of the indicated five fPCA eigenfunctions shown in panel H for the MG132 DD6 data shown in panel D. <strong>(K)</strong> Schematic of the 6 dynamical features known as signaling codons, that were identified as informative of the stimulus-response NFκB trajectories. Temporal pattern and signaling codon indices are specified in the top left corner. <strong>(L)</strong> Heatmap representations of the normalized signaling codon values for the Sel DD15 data shown in panel B. <strong>(M)</strong> Heatmap representations of the normalized signaling codon values for the MG132 DD6 data shown in panel D.</p></figcaption></figure><p>The first dimensionality reduction approach is tensor decomposition (canonical polyadic decomposition, CPD), which provides a comprehensive factorized approximation of the data [<a href="#pcbi.1013344.ref049" class="usa-link" aria-describedby="pcbi.1013344.ref049">49</a>,<a href="#pcbi.1013344.ref050" class="usa-link" aria-describedby="pcbi.1013344.ref050">50</a>]. The 3000 simulated trajectories were organized into a five-dimensional tensor (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e003"><math id="M3" display="inline" overflow="linebreak"><mrow><mi>𝒜</mi><mo>∈</mo><msup><mi>𝔽</mi><mrow><mn>481</mn><mi>×</mi><mn>5</mn><mi>×</mi><mn>3</mn><mi>×</mi><mn>10</mn><mi>×</mi><mn>20</mn></mrow></msup><mtext> </mtext></mrow></math></span>), with dimensions representing ligand, ligand dose, drug, drug dose, and time. Employing non-negative CPD, we decomposed this tensor into 7 components, accounting for 87.5% of the variance observed in the original trajectory space (<a href="#pcbi.1013344.s004" class="usa-link">S3A Fig</a>, Methods).</p>
<table class="disp-formula p" id="pcbi.1013344.e004"><tr><td class="formula"><math id="M4" display="block" overflow="linebreak"><mrow><mrow><mi mathvariant="script">A</mi><mo mathvariant="script">≈</mo></mrow><msubsup><mo>∑</mo><mrow><mi>r</mi><mo>=</mo><mn>1</mn></mrow><mrow><mn>7</mn></mrow></msubsup><msub><mi>λ</mi><mrow><mi>r</mi></mrow></msub><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">e</mi><mo>,</mo><mi mathvariant="normal">r</mi></mrow></msub><mo>⊗</mo><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi mathvariant="normal">l</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">g</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">d</mi><mo>,</mo><mi mathvariant="normal">r</mi></mrow></msub><mo>⊗</mo><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi mathvariant="normal">l</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">g</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">d</mi><mtext> </mtext><mi mathvariant="normal">d</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">e</mi><mo>,</mo><mi mathvariant="normal">r</mi></mrow></msub><mo>⊗</mo><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi mathvariant="normal">d</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">g</mi><mo>,</mo><mi mathvariant="normal">r</mi></mrow></msub><mo>⊗</mo><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi mathvariant="normal">d</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">g</mi><mtext> </mtext><mi mathvariant="normal">d</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">e</mi><mo>,</mo><mi mathvariant="normal">r</mi></mrow></msub></mrow></math></td></tr></table>
<p>The factor weights along the time dimension (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e005"><math id="M5" display="inline" overflow="linebreak"><mrow><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi>t</mi><mi>i</mi><mi>m</mi><mi>e</mi><mo>,</mo><mi>r</mi></mrow></msub><mo>,</mo><mtext> </mtext><mi>r</mi><mo>=</mo><mn>1</mn><mo>,</mo><mtext> </mtext><mn>2</mn><mo>,</mo><mtext> </mtext><mi>…</mi><mo>,</mo><mtext> </mtext><mn>7</mn></mrow></math></span>) can be regarded as 7 temporal patterns that encompass key dynamics within the original trajectory space (<a href="#pcbi.1013344.g003" class="usa-link">Fig 3E</a>). Each trajectory is approximated as a weighted sum of these temporal patterns (<a href="#pcbi.1013344.s004" class="usa-link">S3C Fig</a>, Methods for detailed explanation).</p>
<table class="disp-formula p" id="pcbi.1013344.e006"><tr><td class="formula"><math id="M6" display="block" overflow="linebreak"><mrow><msub><mi>X</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi></mrow></msub><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">[</mo><msub><mi>𝒜</mi><mrow><msub><mi>t</mi><mrow><mn>1</mn></mrow></msub><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi></mrow></msub><mo>,</mo><msub><mi>𝒜</mi><mrow><msub><mi>t</mi><mrow><mn>2</mn></mrow></msub><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi></mrow></msub><mo>,</mo><mi>…</mi><mo>,</mo><mtext> </mtext><msub><mi>𝒜</mi><mrow><msub><mi>t</mi><mrow><mn>481</mn></mrow></msub><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">]</mo></mrow><mo>≈</mo><msubsup><mo>∑</mo><mrow><mi>r</mi><mo>=</mo><mn>1</mn></mrow><mrow><mn>7</mn></mrow></msubsup><msub><mi>ω</mi><mrow><mi>r</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi></mrow></msub><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">e</mi><mo>,</mo><mi mathvariant="normal">r</mi></mrow></msub></mrow></math></td></tr></table>
<table class="disp-formula p" id="pcbi.1013344.e007"><tr><td class="formula"><math id="M7" display="block" overflow="linebreak"><mrow><msub><mi>ω</mi><mrow><mi>r</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi></mrow></msub><mo>=</mo><msub><mi>λ</mi><mrow><mi>r</mi></mrow></msub><msub><mi>ρ</mi><mrow><mi>r</mi></mrow></msub><msub><mrow><mo fence="true" form="prefix" stretchy="true">[</mo><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi mathvariant="normal">l</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">g</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">d</mi><mo>,</mo><mi mathvariant="normal">r</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">]</mo></mrow><mrow><mi>i</mi></mrow></msub><msub><mrow><mo fence="true" form="prefix" stretchy="true">[</mo><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi mathvariant="normal">l</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">g</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">d</mi><mtext> </mtext><mi mathvariant="normal">d</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">e</mi><mo>,</mo><mi mathvariant="normal">r</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">]</mo></mrow><mrow><mi>j</mi></mrow></msub><msub><mrow><mo fence="true" form="prefix" stretchy="true">[</mo><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi mathvariant="normal">d</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">g</mi><mo>,</mo><mi mathvariant="normal">r</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">]</mo></mrow><mrow><mi>k</mi></mrow></msub><msub><mrow><mo fence="true" form="prefix" stretchy="true">[</mo><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi mathvariant="normal">d</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">g</mi><mtext> </mtext><mi mathvariant="normal">d</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">e</mi><mo>,</mo><mi mathvariant="normal">r</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">]</mo></mrow><mrow><mi>l</mi></mrow></msub></mrow></math></td></tr></table>
<p>For each trajectory (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e008"><math id="M8" display="inline" overflow="linebreak"><mrow><msub><mi>X</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi></mrow></msub></mrow></math></span>), the 7 weights (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e009"><math id="M9" display="inline" overflow="linebreak"><mrow><msub><mrow><mo fence="true" form="prefix" stretchy="true">{</mo><msub><mi>ω</mi><mrow><mi>r</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">}</mo></mrow><mrow><mi>r</mi><mo>=</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mi>…</mi><mo>,</mo><mn>7</mn></mrow></msub></mrow></math></span>) corresponding to the 7 temporal patterns represent its feature vector within the CPD-defined landscape. The component weights were calculated as the product of the component scalar, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e010"><math id="M10" display="inline" overflow="linebreak"><mrow><msub><mi>λ</mi><mrow><mi>r</mi></mrow></msub></mrow></math></span>, a scaling factor to correct for the inherent scale invariance of CPD, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e011"><math id="M11" display="inline" overflow="linebreak"><mrow><msub><mi>ρ</mi><mrow><mi>r</mi></mrow></msub></mrow></math></span>, and the factor weights in all but the time dimension (<a href="#pcbi.1013344.s004" class="usa-link">S3B</a>-<a href="#pcbi.1013344.s004" class="usa-link">S3D Fig</a>, see methods and supplement for feature vector details). The landscapes (i.e., feature vectors) for the drug regimes highlighted above are visualized in <a href="#pcbi.1013344.g003" class="usa-link">Fig 3F</a> and <a href="#pcbi.1013344.g003" class="usa-link">3G</a>. Focusing first on the Sel-treated landscape, we observe, for instance, that medium doses of LPS and CpG stimulation are mainly weighted in temporal patterns 4 and 7 (stimulation indices 5 and 8 in blue dashed boxes of <a href="#pcbi.1013344.g003" class="usa-link">Fig 3F</a>). However, these stimulus conditions have slightly different weights in temporal pattern 4, while in the original trajectory space medium doses of LPS and CpG stimulation are similar (stimulation indices 5 and 8 in <a href="#pcbi.1013344.g003" class="usa-link">Fig 3B</a>). Next, within the MG132-treated landscape, all doses of Pam3CSK stimulation are weighted in temporal patterns 1 and 6 to varying extents (stimulation indices 13–15 boxed in <a href="#pcbi.1013344.g003" class="usa-link">Fig 3G</a>). This contradicts their nearly indistinguishable dynamics in the original trajectory space (stimulation indices 13–15 in <a href="#pcbi.1013344.g003" class="usa-link">Fig 3D</a>).</p>
<p>Considering the importance of temporal information, we also applied functional principal component analysis (fPCA) to our dataset. A major difference between fPCA and PCA is that the order of functional data carries continuous (i.e., temporal) information meaning that permutations of the multivariate data will lead to different results. We organized the simulated data into a 3000x481 matrix (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e012"><math id="M12" display="inline" overflow="linebreak"><mrow><msub><mrow><mo fence="true" form="prefix" stretchy="true">[</mo><msub><mi>X</mi><mrow><mi>m</mi><mo>,</mo><msub><mi>t</mi><mrow><mi>i</mi></mrow></msub></mrow></msub><mo fence="true" form="postfix" stretchy="true">]</mo></mrow><mrow><mi>m</mi><mo>=</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mi>…</mi><mo>,</mo><mn>3000</mn><mi>;</mi><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mi>…</mi><mo>,</mo><mn>481</mn></mrow></msub><mtext> </mtext></mrow></math></span>) containing the 3000 combinations of stimuli conditions (15) and drug regimes (200) at 481 timepoints. We found that the first 5 components account for 95.6% of the variance (<a href="#pcbi.1013344.s005" class="usa-link">S4A Fig</a>, see methods for details). The resulting eigenfunctions characterize the dynamic temporal patterns (<a href="#pcbi.1013344.g003" class="usa-link">Fig 3H</a>), and each trajectory can be approximated by the linear combination of these temporal patterns (<a href="#pcbi.1013344.s005" class="usa-link">S4B</a>-<a href="#pcbi.1013344.s005" class="usa-link">S4C Fig</a>).</p>
<table class="disp-formula p" id="pcbi.1013344.e013"><tr><td class="formula"><math id="M13" display="block" overflow="linebreak"><mrow><msub><mi>X</mi><mrow><mi mathvariant="normal">m</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>≈</mo><mi>μ</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>+</mo><msubsup><mo>∑</mo><mrow><mi>r</mi><mo>=</mo><mn>1</mn></mrow><mrow><mn>5</mn></mrow></msubsup><mrow><msubsup><mi>ξ</mi><mrow><mi>r</mi></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><msub><mi>ϕ</mi><mrow><mi>r</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></mrow></math></td></tr></table>
<p>Like our CPD-defined feature vectors, the fPCA-defined feature vectors consist of fPCA scores (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e014"><math id="M14" display="inline" overflow="linebreak"><mrow><msub><mrow><mo fence="true" form="prefix" stretchy="true">{</mo><msubsup><mi>ξ</mi><mrow><mi>r</mi></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo fence="true" form="postfix" stretchy="true">}</mo></mrow><mrow><mi>r</mi><mo>=</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mi>…</mi><mo>,</mo><mn>5</mn></mrow></msub></mrow></math></span>) in the 5 components/temporal patterns (<a href="#pcbi.1013344.g003" class="usa-link">Fig 3H</a>). The fPCA-defined Sel-treated landscape shows similar weights in the 5 temporal patterns for medium doses of LPS and CpG stimulation (stimulation indices 5 and 8 in blue dashed boxes of <a href="#pcbi.1013344.g003" class="usa-link">Fig 3I</a>). Regarding the MG132-treated landscape, while fPCA features better recapitulate the similarities between all Pam3CSK-stimulated responses compared to CPD, fPCA still results in subtle variations in temporal patterns 2, 3, and 5 for these stimuli (stimulation indices 13–15 boxed in <a href="#pcbi.1013344.g003" class="usa-link">Fig 3J</a>). Overall, this shows that encoding the timepoint order may lead to a more meaningful representation of temporal trajectories.</p>
<p>Our final decomposition method leveraged the concept of NFκB signaling codons. The approach decomposes the trajectories into 6 informative dynamic features identified by an information maximizing algorithm that identified the dynamic features that best distinguished stimulus- and dose-specific NFκB responses [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>]. These 6 signaling codons include “Speed”, “Peak Amplitude”, “Duration”, “Area Under the Curve (AUC)”, “Early vs Late”, and “Oscillations” (<a href="#pcbi.1013344.g003" class="usa-link">Fig 3K</a>, see methods for calculation details).</p>
<table class="disp-formula p" id="pcbi.1013344.e015"><tr><td class="formula"><math id="M15" display="block" overflow="linebreak"><mrow><mtable columnspacing="0em 2em" displaystyle="true" rowspacing="3pt"><mtr><mtd columnalign="right"><mrow><mi mathvariant="normal">S</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">g</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">g</mi><mtext> </mtext><mi mathvariant="normal">C</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">d</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">n</mi></mrow><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>X</mi><mrow><mi>m</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow></mtd><mtd columnalign="left"><mi></mi><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">[</mo><msub><mi>f</mi><mrow><mi mathvariant="normal">S</mi><mi mathvariant="normal">p</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">d</mi></mrow></msub><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>X</mi><mrow><mi>m</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>,</mo><mtext> </mtext><msub><mi>f</mi><mrow><mi mathvariant="normal">P</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">k</mi></mrow></msub><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>X</mi><mrow><mi>m</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>,</mo><msub><mi>f</mi><mrow><mi>D</mi><mi>u</mi><mi>r</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi></mrow></msub><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>X</mi><mrow><mi>m</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>,</mo><mtext> </mtext><msub><mi>f</mi><mrow><mi>A</mi><mi>U</mi><mi>C</mi></mrow></msub><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>X</mi><mrow><mi>m</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>,</mo><mtext> </mtext><msub><mi>f</mi><mrow><mi>E</mi><mi>v</mi><mi>L</mi></mrow></msub><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>X</mi><mrow><mi>m</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>,</mo><mtext> </mtext><msub><mi>f</mi><mrow><mi>O</mi><mi>s</mi><mi>c</mi></mrow></msub><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>X</mi><mrow><mi>m</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo fence="true" form="postfix" stretchy="true">]</mo></mrow></mtd></mtr><mtr><mtd columnalign="right"></mtd><mtd columnalign="left"><mi></mi><mo>=</mo><mtext> </mtext><mrow><mo fence="true" form="prefix" stretchy="true">[</mo><msub><mi>f</mi><mrow><mn>1</mn></mrow></msub><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>X</mi><mrow><mi>m</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>,</mo><mtext> </mtext><msub><mi>f</mi><mrow><mn>2</mn></mrow></msub><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>X</mi><mrow><mi>m</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>,</mo><msub><mi>f</mi><mrow><mn>3</mn></mrow></msub><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>X</mi><mrow><mi>m</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>,</mo><mtext> </mtext><msub><mi>f</mi><mrow><mn>4</mn></mrow></msub><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>X</mi><mrow><mi>m</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>,</mo><mtext> </mtext><msub><mi>f</mi><mrow><mn>5</mn></mrow></msub><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>X</mi><mrow><mi>m</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>,</mo><mtext> </mtext><msub><mi>f</mi><mrow><mn>6</mn></mrow></msub><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>X</mi><mrow><mi>m</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo fence="true" form="postfix" stretchy="true">]</mo></mrow></mtd></mtr></mtable></mrow></math></td></tr></table>
<p>Normalized signaling codons values (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e016"><math id="M16" display="inline" overflow="linebreak"><mrow><mo stretchy="false">{</mo><msub><mi>f</mi><mrow><mi>s</mi></mrow></msub><msub><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>X</mi><mrow><mi>m</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo fence="true" form="postfix" stretchy="true">}</mo></mrow><mrow><mi>s</mi><mo>=</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mi>…</mi><mo>,</mo><mn>6</mn></mrow></msub></mrow></math></span>) make up the signaling codon-defined feature vectors used to derive the final drug regime landscapes (<a href="#pcbi.1013344.g003" class="usa-link">Fig 3L</a> and <a href="#pcbi.1013344.g003" class="usa-link">3M</a>). In the signaling codon-defined Sel-treated landscape, we observe consistent, near average signaling codon values for medium doses of LPS and CpG stimulation (stimulation indices 5 and 9 in red boxes in <a href="#pcbi.1013344.g003" class="usa-link">Fig 3L</a>). Interestingly, the subtle, yet significant discrepancies in the Oscillations and Early vs. Late codons for these conditions highlights the slightly more apparent three oscillatory peaks for the LPS-stimulated trajectory (stimulation indices 5 and 9 in red boxes in <a href="#pcbi.1013344.g003" class="usa-link">Fig 3B</a>), which was not captured in the matrix-based decomposition landscapes (stimulation indices 5 and 9 in red boxes in <a href="#pcbi.1013344.g003" class="usa-link">Fig 3F</a>). This underscores the importance of the signaling codon method’s capacity to discern both the number and timing of peaks. Further, the signaling codon-defined MG132-treated landscape also more adequately describes trajectory dynamics compared to the matrix-decomposition landscapes. This can be highlighted by the similar signaling codon values across Pam3CSK conditions, with an emphasis on their high oscillation power and relatively late NFκB activity (stimulation indices 13–15 boxed in <a href="#pcbi.1013344.g003" class="usa-link">Fig 3M</a>), whereas matrix methods decomposed these trajectories into weighted damped oscillation patterns.</p>
<p>In summary, based on the visual analysis of the feature spaces for the two drug regimes presented here, we found that matrix-based decompositions of temporal trajectories best recapitulate NFκB dynamics when the method emphasizes timeseries (fPCA) rather than structural information (CPD). Additionally, quantified signaling codons reflect a range of nuanced differences and general similarities between dynamic trajectories. To quantitively assess their performance, we opted to include all feature spaces when evaluating drug effects on NFκB stimulus-response specificity, discussed in detail below.</p></section><section id="sec006"><h3 class="pmc_sec_title">Quantifying stimulus-response specificity of NFκB signaling pathway under drug regimes</h3>
<p>To quantify the effect of drugs on stimulus specificity, we applied a clustering method, epsilon network clustering, across different feature spaces (<a href="#pcbi.1013344.g004" class="usa-link">Fig 4A</a>). In this framework, stimuli grouped into the same cluster are interpreted as exhibiting confused signaling responses, while those in separate clusters are considered distinguishable. This use of unsupervised classification to assess dynamic specificity is similar to prior approaches that employed supervised classifiers to evaluate distinguishability and confusion in signaling dynamics [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>,<a href="#pcbi.1013344.ref007" class="usa-link" aria-describedby="pcbi.1013344.ref007">7</a>]. To define the clustering threshold (epsilon), we first clustered trajectories under the same drug treatment to establish an expert classification based on dynamic pattern similarity, guided by our scientific understanding of the stimulus-response specificity encoded in dynamic features of NFkB signaling [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>,<a href="#pcbi.1013344.ref013" class="usa-link" aria-describedby="pcbi.1013344.ref013">13</a>,<a href="#pcbi.1013344.ref029" class="usa-link" aria-describedby="pcbi.1013344.ref029">29</a>]. We then defined the misclustering rate as the discrepancy between the clustering output and the expert classification, and used it as an objective function to optimize epsilon. The optimized epsilon values in each dimensionality reduction space serve as the clustering criterion that best reflects expert knowledge. Using this workflow, we aimed to determine the dimensionality reduction method that best captures knowledge-based signaling dynamics clusters.</p>
<figure class="fig xbox font-sm" id="pcbi.1013344.g004"><h4 class="obj_head">Fig 4. Construction of stimulus cluster maps for selected drug regimes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370059_pcbi.1013344.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f6e6/12370059/95308676541b/pcbi.1013344.g004.jpg" loading="lazy" height="1121" width="781" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013344.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Illustration of the workflow prior to and including epsilon network clustering for the example drug treatment MG132 DD6. The 15 stimulated trajectories (left panel; indices as ordered in <a href="#pcbi.1013344.g003" class="usa-link">Fig 3D</a>) were represented by a dimensionally reduced feature space (middle panel; e.g., signaling codons with indices as ordered in <a href="#pcbi.1013344.g003" class="usa-link">Fig 3M</a>). In the feature space, the 15 trajectories were grouped using epsilon network clustering (right panel; 2/6 example dimensions shown). Dashed circles and C1-7 annotations represent the distinct clusters. C2 consists of the non-responder trajectories (n = 3). Trajectories with distances &gt; optimized ε threshold are clustered separately (e.g., C4 and C6) while trajectories with distances &lt; optimized ε threshold are clustered together (e.g., C1 whose 6 overlapping trajectories are illustrated). <strong>(B)</strong> Stimulus cluster maps constructed from epsilon network clustering results for drug treatment MG132 DD66. Each row and column within one map correspond to a specific stimulus, as denoted on the left side of the left panel. Within each map, off-diagonal black squares represent responsive clusters and red squares on the diagonal represent inhibited NFκB signaling (non-responder). Left, middle, and right panels display the expert classification, clusters derived from trajectory space, and clusters derived from signaling codon space, respectively. <strong>(C)</strong> Stimulus cluster maps constructed from epsilon network clustering results for drug treatment Sel DD15. For labeling see panel B<strong>. (D)</strong> Misclustering rates (# of misclustered stimuli/15 stimuli) for each feature space across the 5 randomly selected drug regimes.</p></figcaption></figure><p>We implemented epsilon network clustering on the original trajectories and in the previously described low dimensional feature spaces for five selected drug regimes (<a href="#pcbi.1013344.s006" class="usa-link">S5A Fig</a>): CPD (using 7 components,7C CPD, and extended to 40 components, 40C CPD), 5 component fPCA (5C fPCA), and 6 signaling codons (<a href="#pcbi.1013344.g004" class="usa-link">Fig 4A</a>). If signal specificity and coding capacity were maintained, this would yield 15 distinct clusters unique to each stimulus, where fewer, and more intermingled clusters would indicate a loss of signal specificity and loss of coding capacity. All clustering results were then visualized in stimulus confusion maps. Off-diagonal dark points indicate converging responder trajectories in response to two stimuli, that therefore constitutes a case of coding confusion. In contrast, red points on the diagonal indicate successfully inhibited trajectories (<a href="#pcbi.1013344.g004" class="usa-link">Figs 4B</a>, <a href="#pcbi.1013344.g004" class="usa-link">4C</a> and <a href="#pcbi.1013344.s006" class="usa-link">S5</a>). To illustrate, for the drug treatment Sel at DD15, the expert stimulus cluster map shows three confusion clusters: medium doses of LPS and CpG; medium and high doses of Poly(I:C); medium and high doses of Pam3CSK. In addition to the confusion clusters, the response to low dose LPS is characterized as a “non-responder” and the other trajectories exhibit stimulus-specific dynamics (<a href="#pcbi.1013344.g004" class="usa-link">Fig 4B</a>). Visually, the signaling codon-defined cluster maps most similarly resembled the expert classification (<a href="#pcbi.1013344.s006" class="usa-link">S5B Fig</a>) across all 5 validation drug regimes (<a href="#pcbi.1013344.g004" class="usa-link">Figs 4B</a> and <a href="#pcbi.1013344.s006" class="usa-link">S5</a>C).</p>
<p>To quantitively determine which feature space gives the most reliable clusters, we calculated the misclustering rate under the optimal epsilon values in each feature space compared to the corresponding expert classification (see methods for calculation details). Indeed, the signaling codon feature space had an average misclustering rate of 1.6/15 whereas the remaining feature spaces, including the original trajectory space, resulted in misclustering rates of 3.0/15 or greater (<a href="#pcbi.1013344.g004" class="usa-link">Fig 4D</a>). This shows that the low-dimensional signaling codons prioritize key, dynamical features that may be deemphasized in the high-dimensional trajectory space, ultimately resulting in a greater capacity to differentiate between stimulus-specific NFκB responses. Thus, we focused the remainder of our analysis to the signaling codon feature space.</p></section><section id="sec007"><h3 class="pmc_sec_title">Quantifying pharmacological effects on the NFκB temporal coding capacity</h3>
<p>Given the importance of stimulus-response specificity (SRS) of signaling pathways, an efficient pharmacological effect on the NFκB pathway is defined as modulating (enhancing or lowering) NFκB stimulus responses, without affecting SRS, i.e., the specificity of the remaining NFκB signaling. The capacity to generate stimulus-specific responses may be defined as the temporal coding capacity of the NFκB signaling module. To quantify the signaling temporal coding capacity under each drug treatment, we introduced three metrics: the <em>stimulus-response specificity</em> (SRS)<em>, confusion</em> (SRC), and <em>inhibition</em> (INH) <em>scores</em>. We defined these scores based on the reliable clusters obtained from epsilon network clustering in the signaling codon feature space. Let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e017"><math id="M17" display="inline" overflow="linebreak"><mrow><mi>S</mi></mrow></math></span> denote the set of the clusters of the responder trajectories under a specific drug treatment:</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e018"><math id="M18" display="inline" overflow="linebreak"><mrow><mrow><mi mathvariant="normal">S</mi></mrow><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">{</mo><msub><mi>S</mi><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>S</mi><mrow><mn>2</mn></mrow></msub><mo>,</mo><mi>…</mi><mo>,</mo><msub><mi>S</mi><mrow><mi>k</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">}</mo></mrow></mrow></math></span>.</p>
<p>Then, the size of the largest cluster is defined as the <em>stimulus-response confusion score</em> (SRC), as all trajectories within one cluster are considered to be similar and potentially indistinguishable for NFκB target genes.</p>
<table class="disp-formula p" id="pcbi.1013344.e019"><tr><td class="formula"><math id="M19" display="block" overflow="linebreak"><mrow><mi>S</mi><mi>R</mi><mi>C</mi><mo>=</mo><msub><mo>max</mo><mrow><msub><mi>S</mi><mrow><mi>i</mi></mrow></msub><mo>∈</mo><mrow><mrow><mi mathvariant="bold-italic">S</mi></mrow></mrow></mrow></msub><mrow><mo fence="true" form="prefix" stretchy="true">|</mo><msub><mi>S</mi><mrow><mi>i</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">|</mo></mrow></mrow></math></td></tr></table>
<p>The <em>stimulus-response specificity score</em> (SRS) is the total number of clusters, which represents the number of uniquely identifiable NFκB signaling profiles under the same pharmacological perturbation.</p>
<table class="disp-formula p" id="pcbi.1013344.e020"><tr><td class="formula"><math id="M20" display="block" overflow="linebreak"><mrow><mi>S</mi><mi>R</mi><mi>S</mi><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">|</mo><mrow><mrow><mi mathvariant="normal">S</mi></mrow></mrow><mo fence="true" form="postfix" stretchy="true">|</mo></mrow></mrow></math></td></tr></table>
<p>We also defined the <em>inhibition score</em> (INH) as the cluster size of non-responders which quantifies inhibition strength of each drug treatment.</p>
<table class="disp-formula p" id="pcbi.1013344.e021"><tr><td class="formula"><math id="M21" display="block" overflow="linebreak"><mrow><mi>I</mi><mi>N</mi><mi>H</mi><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">|</mo><msub><mi>S</mi><mrow><mi>n</mi><mi>o</mi><mi>n</mi><mo>−</mo><mi>r</mi><mi>e</mi><mi>s</mi><mi>p</mi><mi>o</mi><mi>n</mi><mi>d</mi><mi>e</mi><mi>r</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">|</mo></mrow></mrow></math></td></tr></table>
<p>For the example drug treatment MG132 at DD6 depicted in <a href="#pcbi.1013344.g004" class="usa-link">Fig 4A</a>, epsilon network clustering in the signaling codon space finds seven clusters which represent seven predicted distinct dynamical responses under treatment (SRS = 7). Given the nearly identical TNF and Pam3CSK-stimulated responses across all three doses, these conditions form the largest cluster with an SRC = 6. Low doses of LPS, CpG, and Poly(I:C) yield completely attenuated signals, forming the non-responder cluster with an INH = 3.</p>
<p>For all 200 drug regimes, we applied epsilon network clustering using the optimized ε boundary and calculated the resulting INH, SRC, and SRS scores to gain a comprehensive overview of trends in treatment effects on the temporal coding capacity of the innate NFκB signaling network. First, at high doses of treatment, we found that 5 compounds (e.g., BTRCPi and IKKi) cause complete inhibition while 2 compounds (CHL, CHX) lead to no inhibition of NFκB activity; 3 compounds (PDTC, PP2Ai, Sel) selectively inhibit activity in a stimulus-dependent manner (<a href="#pcbi.1013344.g005" class="usa-link">Fig 5A</a>). Additionally, the maximal stimulus confusion can be reached by compounds at intermediate (e.g., IKKi and MG132), high (e.g., CHX and PP2Ai), or relatively all (e.g., CHL) treatment doses (<a href="#pcbi.1013344.g005" class="usa-link">Fig 5B</a>). With the network’s high innate temporal coding capacity (SRS = 13 for untreated condition), Sel is the only compound that enhanced temporal coding capacity (Sel at DD5–14 SRS = 15) (<a href="#pcbi.1013344.g005" class="usa-link">Fig 5C</a>). Notably, regimes can modulate coding capacity by either partially inhibiting signaling while confusing all other stimuli (e.g., IKKi DD13 INH = 6, SRC = 9 <a href="#pcbi.1013344.g005" class="usa-link">Fig 5A</a> and <a href="#pcbi.1013344.g005" class="usa-link">5B</a>) or by partially inhibiting signaling while still maintaining stimulus-specific responses (e.g., TAK1i DD12 INH = 10, SRS = 5, <a href="#pcbi.1013344.g005" class="usa-link">Fig 5A</a> and <a href="#pcbi.1013344.g005" class="usa-link">5C</a>). Taken together, this suggest that confusion and inhibition can be selective. These distinct patterns of altering temporal coding capacity are summarized in more detail below.</p>
<figure class="fig xbox font-sm" id="pcbi.1013344.g005"><h4 class="obj_head">Fig 5. Signaling codon-derived quantification and visualization of the NFκB temporal coding capacity under each of 200 drug regimes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370059_pcbi.1013344.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f6e6/12370059/d152c0f703b7/pcbi.1013344.g005.jpg" loading="lazy" height="785" width="720" alt="Fig 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013344.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Heatmap of inhibition scores for all 10 compounds across 20 compound doses treatments as indicated. <strong>(B)</strong> Heatmap of stimulus-response confusion (SRC) scores for all 10 compounds across 20 compound doses treatments as indicated. <strong>(C)</strong> Heatmap of stimulus-response specificity (SRS) scores for all 10 compounds across 20 compound doses treatments as indicated. <strong>(D)</strong> Stimulus cluster map for the 15 stimulus conditions under no drug treatment (untreated). Each row and column within the map correspond to a specific stimulus, as denoted on the left and bottom side of the map. Off-diagonal black squares represent responsive clusters and red squares on the diagonal represent inhibited NFκB signaling (non-responders). <strong>(E)</strong> Bar plot of the percentage of drug regimes within a specific range of inhibition scores (specified on x-axis) corresponding to the three categories of drug regime functions. <strong>(F)</strong> Stimulus confusion maps for all 10 compounds (far left labels) across 20 compound doses (bottom labels) treatments.</p></figcaption></figure><p>Using the previously described stimulus confusion maps, we next sought to investigate changes in temporal coding capacity at the level of specific stimulus conditions. For the 15 conditions under no drug treatment (i.e., untreated), 13 out of 15 are specific (SRS = 13), low dose LPS does not elicit a detectable NFκB response (INH = 1) and there are significant similarities between high doses of CpG and Pam3CSK-stimulated responses (SRC = 2) (<a href="#pcbi.1013344.g005" class="usa-link">Fig 5D</a>).</p>
<p>Separately, the compound-treated stimulus confusion maps enabled us to find regimes that alter the temporal coding capacity of NFκB signaling pathway in three distinct ways: (i) complete inhibition of NFκB activity under all 15 stimulus conditions (<a href="#pcbi.1013344.g005" class="usa-link">Fig 5E</a>, INH = 15) (ii-iii) partial stimulus-response specificity and confusion of the remaining stimulus conditions without (ii) (<a href="#pcbi.1013344.g005" class="usa-link">Fig 5E</a>, INH ≤ 1) or with (iii) inhibition (<a href="#pcbi.1013344.g005" class="usa-link">Fig 5E</a> 1&lt; INH &lt; 15). (i) High treatment doses of BTRCPi, IKKi, MG132, TAK1i, and TSA result in complete inhibition of NFκB activity (<a href="#pcbi.1013344.g005" class="usa-link">Fig 5A</a>). As the compound dose increases, inhibition typically begins at low ligand doses and progresses to medium and high doses (<a href="#pcbi.1013344.g005" class="usa-link">Fig 5F</a>). (ii) Drug regimes like CHL DD20 and CHX DD20 exhibit partial stimulus-response specificity and partial confusion without inhibition (<a href="#pcbi.1013344.g005" class="usa-link">Fig 5A</a> - <a href="#pcbi.1013344.g005" class="usa-link">5C</a>). CHL DD20 tends to confuse bacterial PAMPs such as LPS and CpG, while CHX DD20 primarily causes confusion between cytokines and PAMPs (<a href="#pcbi.1013344.g005" class="usa-link">Fig 5F</a>). (iii) Compounds like MG132 at DD6 and PP2Ai at DD20 induce partial stimulus-response specificity and confusion, with the rest of the stimuli inhibited. Under these regimes, the specificity scores are 7/15, with inhibition primarily affecting low-dose ligands and confusion occurring mostly at medium and high doses (<a href="#pcbi.1013344.g005" class="usa-link">Fig 5F</a>).</p>
<p>The compound-treated stimulus confusion maps also allow for the identification of significant drug regimes that influence the NFκB signaling temporal coding capacity. For example, MG132 at DD6 both induces confusion and inhibition for some stimulus conditions while maintaining specificity for the rest (<a href="#pcbi.1013344.g005" class="usa-link">Fig 5F</a>). In the untreated regime, CpG and Pam3CSK are indistinguishable, whereas other stimuli, such as TNF and Pam3CSK, are distinct (<a href="#pcbi.1013344.g005" class="usa-link">Fig 5D</a>). However, under MG132 DD6 treatment, the stimulus confusion map reveals that TNF and Pam3CSK responses are confused, while CPG and Pam3CSK responses are distinct (<a href="#pcbi.1013344.s007" class="usa-link">S6A Fig</a>). By comparing trajectories and the corresponding signaling codons between the untreated and treated regimes, we observe that TNF and Pam3CSK blur together, whereas Pam3CSK and CpG become clearly differentiated (<a href="#pcbi.1013344.s007" class="usa-link">S6B Fig</a>). These results confirm the efficacy of signaling-codon-derived stimulus confusion maps in quantitatively estimating the effects of drug regime on altering NFκB temporal coding capacity.</p></section><section id="sec008"><h3 class="pmc_sec_title">NFκB signaling temporal coding capacity is robust</h3>
<p>While exploring pharmacological perturbations aimed at altering specific aspects of NFκB signaling, a possible side-effect is increased confusion among stimuli and conditions, thereby reducing the signaling temporal coding capacity. Thus, given the diverse pharmacological perturbations explored throughout this study, it is important to evaluate the robustness of the stimulus-specific temporal coding capacity of NFκB signaling pathway. The robustness of the NFκB signaling pathway is defined by the maintenance or preservation of its capacity for stimulus-response specificity across responsive conditions, under various drug treatments. we considered two key categories of robustness evaluation: (1) the degree to which stimulus specificity is maintained under pharmacological perturbation while altering specific NFκB dynamics; (2) the extent to which the drug-treated conditions are confused with the untreated conditions.</p>
<p>To broadly assess how much specificity is preserved within each drug regime, we computed the proportion of those that cause confusion for each pair of stimuli out of the 200 drug regimes (<a href="#pcbi.1013344.g006" class="usa-link">Fig 6A</a>). Stimuli pairs across cytokine and PAMPs are confused under few drug regime treatments. For instance, the TNF vs. Poly(I:C) pair is confused by less than 10% of all drug regimes. Confusion among bacterial PAMPs occurs more frequently among drug regimes. This is exemplified by similar NFκB responses in the condition of high dose CpG and medium or high dose Pam3CSK under 89 and 55 drug regimes which can be explained by the similarity between their NFκB trajectories under the untreated condition.</p>
<figure class="fig xbox font-sm" id="pcbi.1013344.g006"><h4 class="obj_head">Fig 6. Evaluation of NFκB signaling network robustness.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370059_pcbi.1013344.g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f6e6/12370059/8c34a8fa2c18/pcbi.1013344.g006.jpg" loading="lazy" height="606" width="742" alt="Fig 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013344.g006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A-B)</strong> Number of drug regimes where a given stimulus pair is “confused” within a given drug regime for <strong>(A)</strong> single drug (n = 200) and <strong>(B)</strong> drug combination (n = 180) perturbations. Excludes regimes where both stimulated trajectories are completely inhibited by drug treatment. <strong>(C)</strong> Number of drug regimes where a given untreated-treated stimulus pair is “confused.” Columns represent the 15 untreated trajectories, rows represent all treated trajectories. Low dose LPS (column 4) induces an inhibitory response, resulting in no “confusion” across all treated conditions.</p></figcaption></figure><p>We next wanted to determine whether the temporal coding capacity robustness under single drug perturbations extends to combinations of drug perturbations using two representative drug doses. For 180 drug combinations (10 drugs at DD5 and DD10, yielding 45 drug combinations x 4 drug dose combinations), we simulated NFκB’s response to the 15 stimulus conditions and applied the same workflow developed for single perturbations; their stimulus confusion maps are shown in <a href="#pcbi.1013344.s008" class="usa-link">S7 Fig</a>.</p>
<p>In calculating the total pairwise stimulus confusion across all 180 drug combination regimes, we found that select cross-stimulus comparisons tend to follow or exacerbate the same trends described for the 200 single drug regimes while others introduce new confusion patterns (<a href="#pcbi.1013344.g006" class="usa-link">Fig 6B</a>). In comparing TNF vs. PAMPs to the single drug case, for combinations, there is no longer TNF vs. Poly(I:C) confusion (2 single vs. 0 combination drug regimes across all stimulus doses). However, the proportion of drug combination regimes that leads to TNF vs. Pam3SCK confusion increased three-fold. Intriguingly, among bacterial PAMP pairs, confusion between LPS and CpG increased from 10% of all single drug regimes to 21% of all drug combination regimes; that between high dose CpG and medium/high dose Pam3CSK is more consistent (34% of single vs. 31% of combination treatments). Although subtle, drug combinations also resulted in increased confusion between Poly(I:C) and LPS/CpG, which is nearly absent from single drug treatment effects.</p>
<p>To evaluate the potential for confusion between drug-treated conditions and untreated conditions, we compared the pharmacologically perturbed conditions with the untreated NFκB signaling profile (<a href="#pcbi.1013344.s009" class="usa-link">S8A</a>–<a href="#pcbi.1013344.s009" class="usa-link">S8B Fig</a>). Most of the drug regimes show minor confusion between treated and untreated conditions. The average frequency of same-stimulus confusion between treated and untreated conditions is 25/200, while cross-stimulus confusion frequency is less than 10/200 (<a href="#pcbi.1013344.g006" class="usa-link">Fig 6C</a>). Cross-stimulus confusion between treated and untreated conditions occurs primarily among the bacterial PAMPs. For example, Sel-DD13 treated medium-dose Pam3CSK stimulation is confused with untreated medium-dose LPS stimulation (yellow box in <a href="#pcbi.1013344.g006" class="usa-link">Fig 6C</a>), and IKKi-DD3 treated high-dose CpG stimulation resembles untreated medium-dose LPS stimulation (red box in <a href="#pcbi.1013344.g006" class="usa-link">Fig 6C</a>) (<a href="#pcbi.1013344.s009" class="usa-link">S8C</a>–<a href="#pcbi.1013344.s009" class="usa-link">S8D Fig</a>).</p>
<p>In summary, the pharmacological perturbations considered in this study have minor off-target effects of reducing NFκB signaling temporal coding capacity, and the temporal coding capacity of NFκB signaling remains robust against a variety of pharmacological perturbations. However, in extending our workflow to multi-drug perturbations, we found that NFκB’s ability to maintain stimulus-specific responses begins to waver.</p></section></section><section id="sec009"><h2 class="pmc_sec_title">Discussion</h2>
<p>In this work, we introduced the stimulus-response specificity score and stimulus confusion maps to evaluate the efficacy and specificity of pharmacological perturbations in targeting stimulus-specific dynamical responses of the NFκB signaling pathway (<a href="#pcbi.1013344.g004" class="usa-link">Figs 4</a> and <a href="#pcbi.1013344.g005" class="usa-link">5</a>). We applied pharmacological perturbations to a well-established NFκB signaling network model, generating a large, simulated dataset of NFκB temporal trajectories (<a href="#pcbi.1013344.g001" class="usa-link">Figs 1</a> and <a href="#pcbi.1013344.g002" class="usa-link">2</a>). In generating low-dimensional representations of the temporal trajectories, we then found that signaling codons best recapitulated the stimulus- and perturbation-specific signaling dynamics, compared to traditional decomposition methods such as Canonical Polyadic Decomposition (CPD) and functional Principal Component Analysis (fPCA) (<a href="#pcbi.1013344.g003" class="usa-link">Figs 3</a> and <a href="#pcbi.1013344.g004" class="usa-link">4</a>). The application of the newly developed stimulus confusion maps to 200 drug regimens ultimately demonstrated the robustness of the temporal coding capacity of the NFκB signaling network (<a href="#pcbi.1013344.g006" class="usa-link">Fig 6</a>).</p>
<p>One challenge in developing methods to evaluate the extent of pharmacological perturbation effects on stimulus-response specificity is the high dimensionality of time-series or trajectory data. To address this, we first reduced the time dimensions of the trajectories to a lower-dimensional representative space using CPD, fPCA, and signaling codon approaches (<a href="#pcbi.1013344.g003" class="usa-link">Fig 3</a>). To assess the similarity or specificity of NFκB dynamics under drug treatment, we applied epsilon network clustering to the reduced-dimensional spaces, optimizing epsilon for each approach to generate confusion maps for each pharmacological perturbation. By comparing these confusion maps with those derived from expert classification, we found that methods based on signaling codons provided the most accurate calculation of the stimulus confusion maps and the corresponding scores (<a href="#pcbi.1013344.g004" class="usa-link">Fig 4</a>).</p>
<p>We used clustering to quantify the stimulus-response specificity. When confronted with a population of single-cell experimental data, stimulus-response specificity is quantified using mutual information (MI) [<a href="#pcbi.1013344.ref008" class="usa-link" aria-describedby="pcbi.1013344.ref008">8</a>,<a href="#pcbi.1013344.ref051" class="usa-link" aria-describedby="pcbi.1013344.ref051">51</a>] or machine learning classifiers [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>,<a href="#pcbi.1013344.ref007" class="usa-link" aria-describedby="pcbi.1013344.ref007">7</a>]. Both approaches investigate the input and output relationship. Where MI quantifies the correlation between input and output, and machine learning calculates the mislabeling rate of the output, with the label information as the input. The unsupervised classification approach we used in this study does not calculate the mislabeling rate or input-output relationship, but is conceptually similar to supervised classifiers and is appropriate when considering a single representative trajectory. It accounts for the distinguishability of the outputs, which is a key factor of the supervised classifier performance. Intuitively, our methods give an upper bound estimation of the supervised classifier performance.</p>
<p>Why do signaling codons show superior performance over other methods of dimensionality reduction? Signaling codons are precise metrics that quantify informative dynamic features of NFκB signaling trajectories [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>,<a href="#pcbi.1013344.ref028" class="usa-link" aria-describedby="pcbi.1013344.ref028">28</a>]. They capture trajectory features that involve multiple time points and incorporate temporal order information. In contrast, decomposition approaches like canonical polyadic decomposition (CPD) and functional principal component analysis (fPCA) decompose trajectories into structural (CPD) or temporal (fPCA) components with corresponding weights [<a href="#pcbi.1013344.ref049" class="usa-link" aria-describedby="pcbi.1013344.ref049">49</a>,<a href="#pcbi.1013344.ref052" class="usa-link" aria-describedby="pcbi.1013344.ref052">52</a>]. While these methods are effective in approximating or reconstructing the original trajectories through weighted combinations of components [<a href="#pcbi.1013344.ref049" class="usa-link" aria-describedby="pcbi.1013344.ref049">49</a>,<a href="#pcbi.1013344.ref052" class="usa-link" aria-describedby="pcbi.1013344.ref052">52</a>], they are less effective than signaling codons in quantifying the similarity and specificity of NFκB dynamics. CPD does not involve time-point information and is insensitive to time-point order (<a href="#pcbi.1013344.s004" class="usa-link">S3E Fig</a>). Conversely, by emphasizing the dataset’s structural information, CPD calculates a joint decomposition dependent on all dimensions, meaning each temporal pattern is dominated by specific combinations of stimulus, stimulus dose, drug, and drug dose conditions. This may result in misleading comparisons between similar or different trajectories based on the conditions being compared. fPCA involves trajectories but the components are less precise and nimble than signaling codons, which are specific temporal features that were identified as being informative about the biological experimental signaling data.</p>
<p>By applying the newly developed stimulus confusion map to all 200 drug regimes, we found that the majority of the drugs inhibited NFκB signaling without losing the stimulus-specificity of the remaining NFκB stimulus responses, highlighting the robustness of the temporal coding capacity of NFκB signaling pathway (<a href="#pcbi.1013344.g006" class="usa-link">Fig 6</a>). The confusion observed within select drug regimes, however, was primarily attributed to dose-related confusions within the same ligand stimulation rather than cross-ligand confusion (<a href="#pcbi.1013344.g006" class="usa-link">Fig 6</a>). The dynamical system being “sloppy” with respect to their parameters could potentially contribute to the observed robustness of stimulus-response specificity [<a href="#pcbi.1013344.ref053" class="usa-link" aria-describedby="pcbi.1013344.ref053">53</a>]. However, both parameter sensitivity analysis (<a href="#pcbi.1013344.s003" class="usa-link">S2 Fig</a>) and pairwise comparison between treated and untreated conditions (<a href="#pcbi.1013344.g006" class="usa-link">Fig 6C</a>) indeed demonstrate significant dynamic changes under the simulated drug treatments.</p>
<p>The drug treatment changes the macrophage state by altering the NFκB signaling parameters; this is conceptually similar to macrophage polarization, which can be modeled by changing multiple parameters. Singh, Sen et al. (2024) [<a href="#pcbi.1013344.ref029" class="usa-link" aria-describedby="pcbi.1013344.ref029">29</a>] evaluated the stimulus-response specificity for an experimental dataset of polarized macrophages, and showed that the pairwise confusion in the experimental data remains generally similar between the five ligand (TNF, Poly(I:C), Pam3CSK, CpG, LPS) stimulation conditions for different polarization states. This aligns with our model prediction of the temporal coding capacity robustness.</p>
<p>As immune response genes can decode dynamic features of NFκB signaling [<a href="#pcbi.1013344.ref030" class="usa-link" aria-describedby="pcbi.1013344.ref030">30</a>–<a href="#pcbi.1013344.ref035" class="usa-link" aria-describedby="pcbi.1013344.ref035">35</a>], this robustness in ligand distinction—even under drug treatment— might be indicative of minimal to no drug side effects or off-target effects. This could allow cellular NFκB responses in a specific condition to be diminished without causing confusion in immune responses to other stimuli. Such robustness of the temporal coding capacity of the NFκB network to various pharmacological perturbations might be a result of evolutionary processes [<a href="#pcbi.1013344.ref054" class="usa-link" aria-describedby="pcbi.1013344.ref054">54</a>], as losing stimulus specificity could lead to pathological conditions [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>,<a href="#pcbi.1013344.ref015" class="usa-link" aria-describedby="pcbi.1013344.ref015">15</a>].</p>
<p>The workflow presented here may also be generalized to other stimulus-response-specific signaling pathways, such as ERK, JNK, and p53 [<a href="#pcbi.1013344.ref001" class="usa-link" aria-describedby="pcbi.1013344.ref001">1</a>], and the cell types or the biological contexts where they are active. For these pathways, a potential challenge in applying our pipeline is the limited investigation of mechanistic modeling details. However, recent advances in machine learning approaches, such as the hybrid modeling strategy that integrates partially known mechanistic models with artificial neural networks (ANN) [<a href="#pcbi.1013344.ref055" class="usa-link" aria-describedby="pcbi.1013344.ref055">55</a>], have lowered the barrier to implementing our pipeline in other systems. Then, with either a mechanistic or hybrid model, pathway- and cell-specific signaling codons would have to be determined. Using context-specific signaling codons to calculate SRS, SRC, INH scores and stimulus confusion maps would then characterize whether the temporal coding capacity of these pathways may be affected or remain robust under drug perturbations.</p>
<p>There are limitations to our workflow that might also present promising directions for future research. First, since single-cell data show high heterogeneity in signaling [<a href="#pcbi.1013344.ref005" class="usa-link" aria-describedby="pcbi.1013344.ref005">5</a>,<a href="#pcbi.1013344.ref056" class="usa-link" aria-describedby="pcbi.1013344.ref056">56</a>], incorporating cellular heterogeneity might provide new insights and present new challenges in the effective design of drugs. Second, in this study, we tested only idealized pharmacological perturbations, which are modeled to alter specific parameters within the signaling network; the reality may be more complicated and how the drug effects are to be modeled must be refined to move the present study beyond the ‘proof-of -principle’ stage. Further, a broader drug library and whole-cell modeling may yield alternative insights regarding the robustness of the NFκB signaling pathway’s temporal coding capacity. Overall, our work presents an effective tool for evaluating to what extent pharmacological targeting may maintain stimulus-response specificity and reveals principles underlying the robustness of NFκB signaling dynamics.</p></section><section id="sec010"><h2 class="pmc_sec_title">Methods</h2>
<section id="sec011"><h3 class="pmc_sec_title">Simulation of NFκB trajectories under pharmacological perturbation</h3>
<p>The previously developed 52-dimensional ordinary differential equation (ODE) model [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>] was simulated. This model consists of five receptor modules that regulate the NFκB core signaling module. The NFκB signaling experimental data were measured in primary macrophages derived from the bone marrow of mVenus-RelA mice to curate the original mechanistic model for the unperturbed condition.</p>
<p>Drug-treated conditions were modeled by altering specific parameters as indicated in <a href="#pcbi.1013344.g001" class="usa-link">Fig 1B</a> and <a href="#pcbi.1013344.s001" class="usa-link">S1 Table</a>. These changes involve multiplying certain parameters by values ranging from 10<sup>-0.15</sup> to 10<sup>-3</sup> in 20 linearly spaced steps, representing 20 different drug doses for each compound. Specifically, the IKK inhibitor (IKKi) reduces the values of parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e022"><math id="M22" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>3</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e023"><math id="M23" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>25</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e024"><math id="M24" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>26</mn></mrow></msub></mrow></math></span>, while the TAK1 inhibitor (TAK1i) decreases parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e025"><math id="M25" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>67</mn></mrow></msub></mrow></math></span>; the Beta-TRCP inhibitor (BTRCPi) reduces parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e026"><math id="M26" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>25</mn></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e027"><math id="M27" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>26</mn></mrow></msub></mrow></math></span>; the phosphatase inhibitor (PP2Ai) modifies <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e028"><math id="M28" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>4</mn></mrow></msub></mrow></math></span>; the transcription enhancer (TSA) reduces <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e029"><math id="M29" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>5</mn></mrow></msub></mrow></math></span>; the NFκB-induced transcription inhibitor (PDTC) inhibits <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e030"><math id="M30" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>6</mn></mrow></msub></mrow></math></span>; the translation inhibitor (CHX) reduces <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e031"><math id="M31" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>8</mn></mrow></msub></mrow></math></span>; the nuclear export inhibitor (Sel) impacts parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e032"><math id="M32" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>11</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e033"><math id="M33" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>12</mn></mrow></msub></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e034"><math id="M34" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>14</mn></mrow></msub></mrow></math></span>; the proteasome inhibitor (MG132) reduces multiple parameters, including <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e035"><math id="M35" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>15</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e036"><math id="M36" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>16</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e037"><math id="M37" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>26</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e038"><math id="M38" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>26</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e039"><math id="M39" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>53</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e040"><math id="M40" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>58</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e041"><math id="M41" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>61</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e042"><math id="M42" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>64</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e043"><math id="M43" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>69</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e044"><math id="M44" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>73</mn></mrow></msub></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e045"><math id="M45" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>75</mn></mrow></msub></mrow></math></span>; the lysosome inhibitor (CHL) inhibits parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e046"><math id="M46" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>30</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e047"><math id="M47" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>42</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e048"><math id="M48" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>43</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e049"><math id="M49" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>44</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e050"><math id="M50" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>78</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e051"><math id="M51" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>83</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e052"><math id="M52" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>86</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e053"><math id="M53" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>87</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e054"><math id="M54" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>92</mn></mrow></msub></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e055"><math id="M55" display="inline" overflow="linebreak"><mrow><msub><mi>k</mi><mrow><mn>93</mn></mrow></msub></mrow></math></span>.</p>
<p>We chose to alter specific parameters rather than directly modeling the pharmacodynamic reactions because a few targets of the selected 10 compounds aren’t represented in the NFκB model. Thus, for simplicity, they are modeled by changing the corresponding kinetic parameters. Essentially, our approach is similar to traditional PK/PD models which tracks decaying drug concentration over time. Our approach provides snapshots of this decaying drug concentration: higher drug doses are representative of the drug concentration at earlier timepoints and as the drug dose decreases, we also capture later stages of drug treatment (or lower overall dose).</p>
<p>The ODE model was simulated using MATLAB’s ode15s solver. Simulations were conducted in two phases: an initial phase to establish a steady state under each condition (untreated and treated), followed by a second phase where different stimulation were applied. A total of 15 stimulations were applied, involving five ligands with three different doses each. Specifically, the stimulation included 0.3 ng/mL, 3.3ng/mL, and 33ng/mL TNF; 0.33ng/mL, 3.3ng/mL, 33ng/mL LPS; 33nM, 100nM, 330nM CpG; 3.3ug/mL, 33ug/mL, 100ug/mL Poly(I:C); and 10ng/mL, 33ng/mL, 100ng/mL Pam3CSK. All simulation results were visualized using MATLAB.</p></section><section id="sec012"><h3 class="pmc_sec_title">Dimensionality reduction of trajectory data</h3>
<section id="sec013"><h4 class="pmc_sec_title">Canonical polyadic decomposition.</h4>
<p>Our simulated NFκB trajectories can be organized into a fifth-order tensor with dimensions of drug, drug doses, ligand, ligand doses, and time points. We turned to a tensor decomposition method, canonical polyadic decomposition (CPD), as our first dimensionality reduction technique. A key advantage of tensor decomposition over traditional techniques like principal component analysis is that it maintains the structural integrity of the data. We organized our simulated data into a fifth-order tensor, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e056"><math id="M56" display="inline" overflow="linebreak"><mrow><mi>𝒜</mi><mo>∈</mo><msup><mi>𝔽</mi><mrow><mi>I</mi><mi>×</mi><mi>J</mi><mi>×</mi><mi>K</mi><mi>×</mi><mi>L</mi><mi>×</mi><mi>T</mi></mrow></msup></mrow></math></span>, where <em>I</em> represents the 5 ligands, <em>J</em> represents the 3 doses of each ligand, <em>K</em> represents the 10 drugs, <em>L</em> represents the 20 drug doses, and <em>T</em> represents the 481 timepoints. After applying CPD, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e057"><math id="M57" display="inline" overflow="linebreak"><mrow><mi>𝒜</mi></mrow></math></span> was expressed as a linear combination of <em>R</em> rank-1 tensors (i.e., components):</p>
<table class="disp-formula p" id="pcbi.1013344.e058"><tr>
<td class="formula"><math id="M58" display="block" overflow="linebreak"><mrow><mtable displaystyle="true" rowspacing="3pt"><mtr><mtd columnalign="right"><mrow><mi mathvariant="script">A</mi><mo mathvariant="script">≈</mo></mrow><mo>∑</mo><msubsup><mi>\nolimits</mi><mrow><mi>r</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>R</mi></mrow></msubsup><msub><mi>λ</mi><mrow><mi>r</mi></mrow></msub><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi>I</mi><mo>,</mo><mi>r</mi></mrow></msub><mo>⊗</mo><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi>J</mi><mo>,</mo><mi>r</mi></mrow></msub><mo>⊗</mo><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi>K</mi><mo>,</mo><mi>r</mi></mrow></msub><mo>⊗</mo><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi>L</mi><mo>,</mo><mi>r</mi></mrow></msub><mo>⊗</mo><mtext> </mtext><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi>T</mi><mo>,</mo><mi>r</mi></mrow></msub></mtd></mtr></mtable></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e059"><math id="M59" display="inline" overflow="linebreak"><mrow><msub><mi>λ</mi><mrow><mi>r</mi></mrow></msub></mrow></math></span> is the component scalar, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e060"><math id="M60" display="inline" overflow="linebreak"><mrow><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi>I</mi><mo>,</mo><mtext> </mtext><mi>r</mi></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e061"><math id="M61" display="inline" overflow="linebreak"><mrow><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi>J</mi><mo>,</mo><mtext> </mtext><mi>r</mi></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e062"><math id="M62" display="inline" overflow="linebreak"><mrow><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi>K</mi><mo>,</mo><mtext> </mtext><mi>r</mi></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e063"><math id="M63" display="inline" overflow="linebreak"><mrow><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi>L</mi><mo>,</mo><mtext> </mtext><mi>r</mi></mrow></msub></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e064"><math id="M64" display="inline" overflow="linebreak"><mrow><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi>T</mi><mo>,</mo><mtext> </mtext><mi>r</mi></mrow></msub></mrow></math></span> are the factor vectors of the <em>r</em><sup><em>th</em></sup> component. The operator <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e065"><math id="M65" display="inline" overflow="linebreak"><mrow><mo>⊗</mo></mrow></math></span> denotes the outer product. The factor weights in the time dimension, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e066"><math id="M66" display="inline" overflow="linebreak"><mrow><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi>T</mi><mo>,</mo><mi>r</mi></mrow></msub><mo>,</mo><mtext> </mtext><mi>r</mi><mo>=</mo><mn>1</mn><mo>,</mo><mtext> </mtext><mn>2</mn><mo>,</mo><mtext> </mtext><mi>…</mi><mo>,</mo><mtext> </mtext><mi>R</mi></mrow></math></span>, resemble distinct temporal patterns of NFκB trajectories under different stimulus conditions and drug treatment. We therefore referred to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e067"><math id="M67" display="inline" overflow="linebreak"><mrow><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi>T</mi><mo>,</mo><mi>r</mi></mrow></msub></mrow></math></span> as the key <em>R</em> temporal patterns within <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e068"><math id="M68" display="inline" overflow="linebreak"><mrow><mi>𝒜</mi></mrow></math></span>.</p>
<p>For a specific trajectory stimulated by the <em>i</em><sup><em>th</em></sup> ligand at the <em>j</em><sup><em>th</em></sup> ligand dose and perturbed by the <em>k</em><sup><em>th</em></sup> drug at the <em>l</em><sup><em>th</em></sup> drug dose, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e069"><math id="M69" display="inline" overflow="linebreak"><mrow><msub><mi>X</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi></mrow></msub><mo>=</mo><mtext> </mtext><mrow><mo fence="true" form="prefix" stretchy="true">[</mo><msub><mi>𝒜</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi><mo>,</mo><msub><mi>t</mi><mrow><mn>1</mn></mrow></msub></mrow></msub><mo>,</mo><msub><mi>𝒜</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi><mo>,</mo><msub><mi>t</mi><mrow><mn>2</mn></mrow></msub></mrow></msub><mo>,</mo><mi>…</mi><mo>,</mo><mtext> </mtext><msub><mi>𝒜</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi><mo>,</mo><msub><mi>t</mi><mrow><mn>481</mn></mrow></msub></mrow></msub><mo fence="true" form="postfix" stretchy="true">]</mo></mrow></mrow></math></span>, the linear combination of all temporal patterns is regarded as its approximated reconstruction:</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e070"><math id="M70" display="inline" overflow="linebreak"><mrow><msub><mi>X</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi></mrow></msub><mo>≈</mo><msubsup><mo>∑</mo><mrow><mi>r</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>R</mi></mrow></msubsup><msub><mi>ω</mi><mrow><mi>r</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi></mrow></msub><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi>T</mi><mo>,</mo><mi>r</mi></mrow></msub></mrow></math></span>.</p>
<p>Each trajectory’s weight in the <em>r</em><sup><em>th</em></sup> temporal pattern/component (i.e., component weight), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e071"><math id="M71" display="inline" overflow="linebreak"><mrow><msub><mi>ω</mi><mrow><mi>r</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi></mrow></msub></mrow></math></span>, was calculated following <a href="#pcbi.1013344.e072" class="usa-link">Equation (2)</a>:</p>
<table class="disp-formula p" id="pcbi.1013344.e072"><tr>
<td class="formula"><math id="M72" display="block" overflow="linebreak"><mrow><msub><mi>ω</mi><mrow><mi>r</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi></mrow></msub><mo>=</mo><msub><mi>λ</mi><mrow><mi>r</mi></mrow></msub><mi>×</mi><msub><mi>ρ</mi><mrow><mi>r</mi></mrow></msub><mi>×</mi><msub><mi>a</mi><mrow><mi>i</mi><mo>,</mo><mi>r</mi></mrow></msub><mi>×</mi><mtext> </mtext><msub><mi>a</mi><mrow><mi>j</mi><mo>,</mo><mi>r</mi><mtext> </mtext></mrow></msub><mi>×</mi><mtext> </mtext><msub><mi>a</mi><mrow><mi>k</mi><mo>,</mo><mi>r</mi></mrow></msub><mtext> </mtext><mi>×</mi><mtext> </mtext><msub><mi>a</mi><mrow><mi>l</mi><mo>,</mo><mi>r</mi></mrow></msub></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<p>Due to the inherent scale invariance of CPD (e.g., multiplying <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e073"><math id="M73" display="inline" overflow="linebreak"><mrow><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi>I</mi><mo>,</mo><mi>r</mi></mrow></msub></mrow></math></span> by a constant and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e074"><math id="M74" display="inline" overflow="linebreak"><mrow><msub><mrow><mrow><mi mathvariant="bold-italic">a</mi></mrow></mrow><mrow><mi>T</mi><mo>,</mo><mi>r</mi></mrow></msub></mrow></math></span> by its reciprocal yields the same result), the solution to <a href="#pcbi.1013344.e058" class="usa-link">Equation (1)</a> is not unique. To enable meaningful comparisons between components, we rescaled all temporal patterns to have an area of 20 and incorporated each pattern’s scaling factor <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e075"><math id="M75" display="inline" overflow="linebreak"><mrow><mi>ρ</mi></mrow></math></span> into the component weight calculations (<a href="#pcbi.1013344.e072" class="usa-link">Equation (2)</a>).</p>
<p>Prior to decomposition, we normalized each element of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e076"><math id="M76" display="inline" overflow="linebreak"><mrow><mi>𝒜</mi></mrow></math></span> within the drug dimension to ensure that the decomposition captured the effects of NFκB responses across the 10 drugs – each of which with a distinct mechanism of action – in a balanced manner. For drug treatment <em>k</em>, we normalized all timepoints as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e077"><math id="M77" display="inline" overflow="linebreak"><mrow><msub><mrow><mover><mrow><mi>𝒜</mi></mrow><mo stretchy="false">^</mo></mover></mrow><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi><mo>,</mo><mi>t</mi></mrow></msub><mo>=</mo><mfrac><mrow><msub><mi>𝒜</mi><mrow><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi><mo>,</mo><mi>t</mi></mrow></msub></mrow><mrow><mrow><msub><mo>max</mo><mrow><msup><mi>i</mi><mrow><mi>′</mi></mrow></msup><mo>,</mo><msup><mi>j</mi><mrow><mi>′</mi></mrow></msup><mo>,</mo><msup><mi>l</mi><mrow><mi>′</mi></mrow></msup><mo>,</mo><msup><mi>t</mi><mrow><mi>′</mi></mrow></msup></mrow></msub><mtext> </mtext></mrow><mrow><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>𝒜</mi><mrow><msup><mi>i</mi><mrow><mi>′</mi></mrow></msup><mo>,</mo><msup><mi>j</mi><mrow><mi>′</mi></mrow></msup><mo>,</mo><mi>k</mi><mo>,</mo><msup><mi>l</mi><mrow><mi>′</mi></mrow></msup><mo>,</mo><msup><mi>t</mi><mrow><mi>′</mi></mrow></msup></mrow></msub><mo fence="true" form="postfix" stretchy="true">}</mo></mrow></mrow><mi>\</mi><mo stretchy="false">)</mo></mrow></mfrac></mrow></math></span>. We applied CPD to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e078"><math id="M78" display="inline" overflow="linebreak"><mrow><mover><mrow><mi>𝒜</mi></mrow><mo stretchy="false">^</mo></mover></mrow></math></span> using the decomposition.non_negative_parafac function from the TensorLy package in python with ‘int’ = random and n_iter_max = 900 and all other settings left as the default parameters. Additionally, we tested ‘rank’ (i.e., number of components) = 1–8 to determine the optimal number of components. To quantify this, we first reconstructed the decomposition output, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e079"><math id="M79" display="inline" overflow="linebreak"><mrow><msub><mi>𝒜</mi><mrow><mi>r</mi><mi>e</mi><mi>c</mi><mi>o</mi><mi>n</mi><mi>s</mi><mi>t</mi><mi>r</mi><mi>u</mi><mi>c</mi><mi>t</mi></mrow></msub></mrow></math></span>, using the cp_to_tensor function from the TensorLy package and then calculated the reconstruction error as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e080"><math id="M80" display="inline" overflow="linebreak"><mrow><mfrac><mrow><mi>V</mi><mi>a</mi><mi>r</mi><mo stretchy="false">(</mo><msub><mi>𝒜</mi><mrow><mi>r</mi><mi>e</mi><mi>c</mi><mi>o</mi><mi>n</mi><mi>s</mi><mi>t</mi><mi>r</mi><mi>u</mi><mi>c</mi><mi>t</mi></mrow></msub><mo>−</mo><mtext> </mtext><mrow><mi mathvariant="script">A</mi><mo stretchy="false">)</mo></mrow></mrow><mrow><mi>V</mi><mi>a</mi><mi>r</mi><mo stretchy="false">(</mo><mrow><mi mathvariant="script">A</mi><mo stretchy="false">)</mo></mrow></mrow></mfrac></mrow></math></span>. In doing so, we decided on <em><em>r</em> </em>= 7 since higher values started to result in redundant temporal patterns. However, to test the limits of the method to accurately recapitulate and cluster timeseries data (see below), we also implemented it using 40 components.</p>
<p>Finally, for the <em><em>r</em> </em>= 7 and <em>r = </em>40 component decompositions, we organized each trajectory’s component weights into a feature vector, [<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e081"><math id="M81" display="inline" overflow="linebreak"><mrow><msub><mi>ω</mi><mrow><mn>1</mn><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi></mrow></msub><mo>,</mo><mtext> </mtext><msub><mi>ω</mi><mrow><mn>2</mn><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi></mrow></msub><mo>,</mo><mtext> </mtext><mi>…</mi><mo>,</mo><mtext> </mtext><msub><mi>ω</mi><mrow><mi>r</mi><mo>,</mo><mi>i</mi><mo>,</mo><mi>j</mi><mo>,</mo><mi>k</mi><mo>,</mo><mi>l</mi></mrow></msub><mo stretchy="false">]</mo></mrow></math></span>, respectively. These two separate sets of feature vectors represented the 7 component CPD- and 40 component CPD-defined landscapes.</p></section><section id="sec014"><h4 class="pmc_sec_title">Functional principal component analysis.</h4>
<p>Given that the simulated NFκB trajectories are time dependent, we employed functional principal component analysis (fPCA) as our second dimensionality reduction technique. The formalism of fPCA is similar to that of PCA with the key difference being that the data is decomposed into eigenfunctions rather than eigenvectors. Because of this, the variable (i.e., timepoint) order carries information, where random permutations result in different decompositions.</p>
<p>We first organized our data into a matrix, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e082"><math id="M82" display="inline" overflow="linebreak"><mrow><msub><mrow><mo fence="true" form="prefix" stretchy="true">[</mo><msub><mi>X</mi><mrow><mrow><mi mathvariant="normal">m</mi></mrow><mo>,</mo><msub><mi>t</mi><mrow><mi>i</mi></mrow></msub></mrow></msub><mo fence="true" form="postfix" stretchy="true">]</mo></mrow><mrow><mi>m</mi><mo>=</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mi>…</mi><mo>,</mo><mn>3000</mn><mi>;</mi><mi>i</mi><mo>=</mo><mn>1</mn><mo>,</mo><mi>…</mi><mo>,</mo><mn>481</mn></mrow></msub><mtext> </mtext></mrow></math></span> where <em>m</em> is the index representing a particular stimulus condition and a drug regime combination and <em>i</em> representing a given timepoint. Then, in applying fPCA to <em>X</em>, each individual trajectory, denoted by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e083"><math id="M83" display="inline" overflow="linebreak"><mrow><msub><mi>X</mi><mrow><mrow><mi mathvariant="normal">m</mi></mrow></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math></span>, was approximated according to <a href="#pcbi.1013344.e091" class="usa-link">Equation (3)</a> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e084"><math id="M84" display="inline" overflow="linebreak"><mrow><mi>μ</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math></span> represents the mean function used for centering <em>X</em> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e085"><math id="M85" display="inline" overflow="linebreak"><mrow><msubsup><mi>ξ</mi><mrow><mi>r</mi></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow></math></span> represents the m<sup>th</sup> trajectory’s score (i.e., weight) in the r<sup>th</sup> eigenfunction (i.e., functional principal component), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e086"><math id="M86" display="inline" overflow="linebreak"><mrow><msub><mi>ϕ</mi><mrow><mi>r</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math></span>. As with PCA, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e087"><math id="M87" display="inline" overflow="linebreak"><mrow><msub><mi>ϕ</mi><mrow><mn>1</mn></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math></span> captures the dominant pattern of variation within <em>X</em> with each subsequent component representing the dominant pattern of variation orthogonal to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e088"><math id="M88" display="inline" overflow="linebreak"><mrow><msub><mi>ϕ</mi><mrow><mn>1</mn></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>,</mo><mtext> </mtext><msub><mi>ϕ</mi><mrow><mn>2</mn></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>,</mo><mtext> </mtext><msub><mi>ϕ</mi><mrow><mn>3</mn></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>,</mo><mtext> </mtext><msub><mi>ϕ</mi><mrow><mi>r</mi><mo>−</mo><mn>1</mn></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>.</mo><mtext>  </mtext></mrow></math></span> Therefore, each of the 3000 trajectories <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e089"><math id="M89" display="inline" overflow="linebreak"><mrow><msub><mi>X</mi><mrow><mrow><mi mathvariant="normal">m</mi></mrow><mo>=</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mi mathvariant="normal">…</mi><mo>,</mo><mn>3000</mn></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math></span> can be approximated by the linear combination of the eigenfunctions, each of which weighted by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e090"><math id="M90" display="inline" overflow="linebreak"><mrow><msubsup><mi>ξ</mi><mrow><mi>r</mi></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup></mrow></math></span>.</p>
<table class="disp-formula p" id="pcbi.1013344.e091"><tr>
<td class="formula"><math id="M91" display="block" overflow="linebreak"><mrow><msub><mi>X</mi><mrow><mrow><mi mathvariant="normal">m</mi></mrow></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>≈</mo><mi>μ</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>+</mo><msubsup><mo>∑</mo><mrow><mi>r</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>m</mi></mrow></msubsup><mrow><msubsup><mi>ξ</mi><mrow><mi>r</mi></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><msub><mi>ϕ</mi><mrow><mi>r</mi></mrow></msub><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<p>We implemented fPCA on our simulated trajectories using the sckit-fda function in python. We opted to use a grid representation of <em>X</em> which we obtained with skfda.representation.grid.FDataGrid. With the resulting discretized object, we then initialized the fPCA class skfda.preprocessing.dim_reduction.FPCA using the default parameters ‘centering’ = True, ‘regularization’ = None, ‘component_basis’ = None, and ‘_weights’ = None. To determine the optimal number of components (i.e., the minimum number that still captured a sufficient amount of variance in <em>X</em>), we varied the number of components parameter, ‘n_components,’ from 1-10. We then calculated and plotted the cumulative variance explained across all 10 fPCA decompositions. Setting ‘n_components’ = 5, we finally used the.fit_transform method to compute the first 5 components and their scores.</p>
<p>Similar to our interpretation of CPD weightings in the time dimension, we regarded the functional principal components as the key temporal patterns that dominated in our dataset. Therefore, for each trajectory <em>X</em><sub>m</sub>, we organized their 5 weightings/scores in the first 5 functional principal components into a 5-dimensional feature vector. These vectors represented the fPCA-defined landscape.</p></section><section id="sec015"><h4 class="pmc_sec_title">Signaling codons.</h4>
<p>Six signaling codons are dynamic features capturing the stimulus-specific information encoded in macrophage NFκB trajectories [<a href="#pcbi.1013344.ref004" class="usa-link" aria-describedby="pcbi.1013344.ref004">4</a>]. Specifically, “Speed” quantifies the activation response time, and “Peak” is the highest response activity. “Duration” measures the total time of NFκB level above a low threshold, while Area Under the Curve (“AUC”) quantifies the total integral of NFκB activity. Finally, Early Vs Late (“EvL”) represents the front-loading of NFκB activity, while Oscillatory Power (“Osc”) captures the oscillations due to IκBα negative feedback that are present in the trajectory. Below are the detailed definitions of each codon.</p>
<p>All the signaling codons are calculated using MATLAB. For an NFκB signaling trajectory, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e092"><math id="M92" display="inline" overflow="linebreak"><mrow><msup><mi>X</mi><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msup><mo>=</mo><mo stretchy="false">(</mo><msubsup><mi>x</mi><mrow><mn>1</mn></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>,</mo><msubsup><mi>x</mi><mrow><mn>2</mn></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>,</mo><msubsup><mi>x</mi><mrow><mn>3</mn></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>,</mo><mi>…</mi><mtext> </mtext><mo>,</mo><mtext> </mtext><msubsup><mi>x</mi><mrow><mi>N</mi></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo stretchy="false">)</mo></mrow></math></span> represents the simulated trajectory <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e093"><math id="M93" display="inline" overflow="linebreak"><mrow><mi>m</mi></mrow></math></span> across N time points with a time interval <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e094"><math id="M94" display="inline" overflow="linebreak"><mrow><mi>Δ</mi><mi>t</mi></mrow></math></span>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e095"><math id="M95" display="inline" overflow="linebreak"><mrow><msubsup><mi>x</mi><mrow><mn>1</mn></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>,</mo><mtext> </mtext><mi>i</mi><mtext> </mtext><mo>=</mo><mn>1</mn><mo>,</mo><mn>2</mn><mo>,</mo><mn>3</mn><mo>,</mo><mi>…</mi><mo>,</mo><mtext> </mtext><mi>N</mi></mrow></math></span>, is the observation at time <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e096"><math id="M96" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mi>i</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mi>·</mi><mi>Δ</mi><mi>t</mi></mrow></math></span>.</p>
<p>(1) Speed: for the trajectory <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e097"><math id="M97" display="inline" overflow="linebreak"><mrow><msup><mi>X</mi><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msup></mrow></math></span>, the local maxima peak sets <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e098"><math id="M98" display="inline" overflow="linebreak"><mrow><msub><mi>Λ</mi><mrow><mi>p</mi><mi>e</mi><mi>a</mi><mi>k</mi></mrow></msub></mrow></math></span> is defined as</p>
<table class="disp-formula p" id="pcbi.1013344.e099"><tr><td class="formula"><math id="M99" display="block" overflow="linebreak"><mrow><msub><mi>Λ</mi><mrow><mi>p</mi><mi>e</mi><mi>a</mi><mi>k</mi></mrow></msub><mo>=</mo><mo stretchy="false">{</mo><msubsup><mi>x</mi><mrow><mi>i</mi></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mi>:</mi><mtext> </mtext><msubsup><mi>x</mi><mrow><mi>i</mi></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>&gt;</mo><msubsup><mi>x</mi><mrow><mi>i</mi><mo>−</mo><mn>1</mn></mrow><mrow><mi>m</mi></mrow></msubsup><mtext> </mtext><mi>&amp;</mi><mtext> </mtext><msubsup><mi>x</mi><mrow><mi>i</mi></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>≥</mo><msubsup><mi>x</mi><mrow><mi>i</mi><mo>+</mo><mn>1</mn></mrow><mrow><mi>m</mi></mrow></msubsup><mtext> </mtext><mi>&amp;</mi><mtext> </mtext><msubsup><mi>x</mi><mrow><mi>i</mi></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>&gt;</mo><mn>0</mn><mo stretchy="false">}</mo></mrow></math></td></tr></table>
<p>The index of the first peak time point is identified as:</p>
<table class="disp-formula p" id="pcbi.1013344.e100"><tr><td class="formula"><math id="M100" display="block" overflow="linebreak"><mrow><msubsup><mi>i</mi><mrow><mi>p</mi><mi>e</mi><mi>a</mi><mi>k</mi><mn>1</mn></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>=</mo><mo>min</mo><mrow><mo stretchy="false">(</mo><mi>i</mi><mo>∈</mo><msub><mi>Λ</mi><mrow><mi>p</mi><mi>e</mi><mi>a</mi><mi>k</mi></mrow></msub><mtext> </mtext><mo stretchy="false">|</mo><mtext> </mtext><mi>i</mi><mo>&gt;</mo><mn>3</mn><mtext> </mtext><mo stretchy="false">)</mo></mrow></mrow></math></td></tr></table>
<p>Speed is defined by:</p>
<table class="disp-formula p" id="pcbi.1013344.e101"><tr><td class="formula"><math id="M101" display="block" overflow="linebreak"><mrow><mi>S</mi><mi>p</mi><mi>e</mi><mi>e</mi><mi>d</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msup><mi>X</mi><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msup><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msubsup><mi>i</mi><mrow><mi>p</mi><mi>e</mi><mi>a</mi><mi>k</mi><mn>1</mn></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>−</mo><mn>1</mn><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mi>·</mi><mi>Δ</mi><mi>t</mi></mrow></math></td></tr></table>
<p>(2) Peak: the maximal value of each trajectory:</p>
<table class="disp-formula p" id="pcbi.1013344.e102"><tr><td class="formula"><math id="M102" display="block" overflow="linebreak"><mrow><mtext> </mtext><mi>P</mi><mi>e</mi><mi>a</mi><mi>k</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msup><mi>X</mi><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msup><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>=</mo><msub><mo>max</mo><mrow><mi>i</mi></mrow></msub><mrow><mo stretchy="false">(</mo><msubsup><mi>x</mi><mrow><mi>i</mi></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo stretchy="false">)</mo></mrow></mrow></math></td></tr></table>
<p>(3) Duration:</p>
<table class="disp-formula p" id="pcbi.1013344.e103"><tr><td class="formula"><math id="M103" display="block" overflow="linebreak"><mrow><mi>D</mi><mi>u</mi><mi>r</mi><mi>a</mi><mi>t</mi><mi>i</mi><mi>o</mi><mi>n</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msup><mi>X</mi><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msup><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">|</mo><mrow><mo fence="true" form="prefix" stretchy="true">{</mo><mi>i</mi><mtext> </mtext><mi>:</mi><msubsup><mi>x</mi><mrow><mi>i</mi></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>&gt;</mo><mi>t</mi><mi>h</mi><mi>r</mi><mi>e</mi><mi>s</mi><mi>h</mi><mi>o</mi><mi>l</mi><mi>d</mi><mo fence="true" form="postfix" stretchy="true">}</mo></mrow><mo fence="true" form="postfix" stretchy="true">|</mo></mrow><mi>·</mi><mi>Δ</mi><mi>t</mi></mrow></math></td></tr></table>
<p>(4) AUC:</p>
<table class="disp-formula p" id="pcbi.1013344.e104"><tr><td class="formula"><math id="M104" display="block" overflow="linebreak"><mrow><mi>A</mi><mi>U</mi><mi>C</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msup><mi>X</mi><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msup><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo>=</mo><msubsup><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>N</mi></mrow></msubsup><mfrac><mrow><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msubsup><mi>x</mi><mrow><mi>k</mi></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>+</mo><msubsup><mi>x</mi><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mi>Δ</mi><mi>t</mi></mrow><mrow><mn>2</mn></mrow></mfrac></mrow></math></td></tr></table>
<p>(5) EvL:</p>
<table class="disp-formula p" id="pcbi.1013344.e105"><tr><td class="formula"><math id="M105" display="block" overflow="linebreak"><mrow><mi>E</mi><mi>a</mi><mi>r</mi><mi>l</mi><mi>y</mi><mi>V</mi><mi>s</mi><mi>L</mi><mi>a</mi><mi>t</mi><mi>e</mi><mo stretchy="false">(</mo><msup><mi>X</mi><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msup><mo stretchy="false">)</mo><mo>=</mo><munder><mrow><mi mathvariant="normal">a</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">g</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">n</mi></mrow><mrow><mi>i</mi><mtext> </mtext></mrow></munder><mrow><mo fence="true" form="prefix" stretchy="true">|</mo><msubsup><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>i</mi></mrow></msubsup><mfrac><mrow><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msubsup><mi>x</mi><mrow><mi>k</mi></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>+</mo><msubsup><mi>x</mi><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mi>Δ</mi><mi>t</mi></mrow><mrow><mn>2</mn></mrow></mfrac><mo>−</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mn>2</mn></mrow></mfrac><msub><mo>max</mo><mrow><mi>l</mi></mrow></msub><mrow><msubsup><mo>∑</mo><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>l</mi></mrow></msubsup><mfrac><mrow><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msubsup><mi>x</mi><mrow><mi>k</mi></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo>+</mo><msubsup><mi>x</mi><mrow><mi>k</mi><mo>+</mo><mn>1</mn></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mi>Δ</mi><mi>t</mi></mrow><mrow><mn>2</mn></mrow></mfrac></mrow><mtext> </mtext><mo fence="true" form="postfix" stretchy="true">|</mo></mrow><mi>·</mi><mi>Δ</mi><mi>t</mi></mrow></math></td></tr></table>
<p>(6) Osc:</p>
<table class="disp-formula p" id="pcbi.1013344.e106"><tr><td class="formula"><math id="M106" display="block" overflow="linebreak"><mrow><mi>O</mi><mi>s</mi><mi>c</mi><mi>p</mi><mi>o</mi><mi>w</mi><mi>e</mi><mi>r</mi><mo stretchy="false">(</mo><msup><mi>X</mi><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msup><mo stretchy="false">)</mo><mo>=</mo><msub><mo>∑</mo><mrow><mi>k</mi></mrow></msub><mrow><mtext> </mtext><mover><mrow><msup><mi>x</mi><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msup></mrow><mo>^</mo></mover><msup><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>f</mi><mrow><mi>k</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mrow><mn>2</mn></mrow></msup></mrow></mrow></math></td></tr></table>
<p>Where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e107"><math id="M107" display="inline" overflow="linebreak"><mrow><mo stretchy="false">{</mo><mover><mrow><msup><mi>x</mi><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msup></mrow><mo>^</mo></mover><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>f</mi><mrow><mi>k</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">}</mo></mrow></mrow></math></span> is the discrete Fourier transform of the trajectory <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e108"><math id="M108" display="inline" overflow="linebreak"><mrow><mo stretchy="false">{</mo><msubsup><mi>x</mi><mrow><mi>i</mi></mrow><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msubsup><mo stretchy="false">}</mo></mrow></math></span>, calculated using the fft function in MATLAB. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e109"><math id="M109" display="inline" overflow="linebreak"><mrow><mrow><mo fence="true" form="prefix" stretchy="true">{</mo><mover><mrow><msup><mi>x</mi><mrow><mo stretchy="false">(</mo><mi>m</mi><mo stretchy="false">)</mo></mrow></msup></mrow><mo>^</mo></mover><msup><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mi>f</mi><mrow><mi>k</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mrow><mn>2</mn></mrow></msup><mo fence="true" form="postfix" stretchy="true">}</mo></mrow></mrow></math></span> is the power spectral density. In the Osc formula, the summation is performed over the frequency range between 0.33 and 1 hour<sup>-1</sup>.</p>
<p>For each NFκB trajectory, the corresponding six signaling codons were calculated and saved in a 6-dimensional vector. To obtain scores for each drug regime, these vectors were organized into a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e110"><math id="M110" display="inline" overflow="linebreak"><mrow><mn>15</mn><mi>×</mi><mn>6</mn></mrow></math></span> matrix, with each row representing the NFκB trajectory under one of the fifteen stimulation conditions.</p></section></section><section id="sec016"><h3 class="pmc_sec_title">Clustering of trajectories in original and dimensionality-reduced spaces</h3>
<p>We developed an epsilon network clustering framework applied separately to five different feature spaces – 7 component CPD, 40 component CPD, 5 component fPCA, signaling codons, and the original trajectories – to quantify the similarities/differences between NFκB responses under distinct stimuli conditions for a given drug regime. To do this, we calculated the Euclidean distance between all pairwise feature vectors representing the 15 stimulus conditions in a drug regime within a given reduced dimensionality space. Smaller distances are indicative of a more similar response between two stimulus conditions. These distances were used as inputs for epsilon network clustering in which stimulus responses whose distances fell within a radius of ε were clustered together. Stimulus conditions were iteratively clustered according to ε, allowing for responses with a Euclidean distance ≥ ε to be clustered together if there was an intermediate condition with Euclidean distance ≤ ε for the aforementioned conditions.</p>
<p>To evaluate the accuracy of the generated clusters, we randomly selected five drug regimes and manually defined their corresponding expert classification using independent individuals and taking their general consensus as the final set. For each of the original and 4 low dimensional feature spaces, the optimal ε, unique to each space, is defined by optimizing the ε that minimized the Misclustering Rate (MR) between epsilon network clustering and expert classification, i.e.,</p>
<table class="disp-formula p" id="pcbi.1013344.e111"><tr>
<td class="formula"><math id="M111" display="block" overflow="linebreak"><mrow><mi>ε</mi><mo>=</mo><mi>\arg</mi><mrow><msub><mo>min</mo><mrow><mi>ε</mi></mrow></msub><mrow><mi>M</mi><mi>R</mi><mo stretchy="false">(</mo><mrow><mrow><mi mathvariant="bold-italic">S</mi></mrow></mrow><mo stretchy="false">(</mo><mi>ε</mi><mo stretchy="false">)</mo><mo>,</mo><msub><mi>S</mi><mrow><mi>o</mi><mi>b</mi><mi>j</mi><mi>e</mi><mi>c</mi><mi>t</mi><mi>i</mi><mi>v</mi><mi>e</mi></mrow></msub><mo stretchy="false">)</mo></mrow></mrow><mtext> </mtext></mrow></math></td>
<td class="label">(4)</td>
</tr></table>
<p>The misclustering rate between clusters are defined in the next paragraph. The optimized εs were then applied to evaluate clustering performance within the different spaces.</p>
<p>To define MR between clusters A and B, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e112"><math id="M112" display="inline" overflow="linebreak"><mrow><mi>M</mi><mi>R</mi><mo stretchy="false">(</mo><mi>A</mi><mo>,</mo><mi>B</mi><mo stretchy="false">)</mo></mrow></math></span>, we first aligned the clusters from Partition A to those in Partition B in a way that minimizes mismatches. This defines the optimal mapping between the two partitions. We then identified mismatched elements and counted them under the optimal mapping. Third, we calculateed MR as the proportion of elements that were assigned to different clusters after optimal alignment.</p>
<table class="disp-formula p" id="pcbi.1013344.e113"><tr>
<td class="formula"><math id="M113" display="block" overflow="linebreak"><mrow><mi>M</mi><mi>R</mi><mo stretchy="false">(</mo><mi>A</mi><mo>,</mo><mi>B</mi><mo stretchy="false">)</mo><mo>=</mo><mrow><mtext> </mtext><mfrac><mrow><mi mathvariant="normal">N</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">b</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">r</mi><mtext> </mtext><mi mathvariant="normal">o</mi><mi mathvariant="normal">f</mi><mtext> </mtext><mi mathvariant="normal">m</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">d</mi><mtext> </mtext><mi mathvariant="normal">e</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">s</mi></mrow><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">l</mi><mtext> </mtext><mi mathvariant="normal">n</mi><mi mathvariant="normal">u</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">b</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">r</mi><mtext> </mtext><mi mathvariant="normal">o</mi><mi mathvariant="normal">f</mi><mtext> </mtext><mi mathvariant="normal">e</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">m</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">s</mi><mtext> </mtext></mrow></mfrac></mrow><mi mathvariant="normal">\]</mi></mrow></math></td>
<td class="label">(5)</td>
</tr></table>
<p>For example, for epsilon clustered results (Partition A of {1,2,3,…,15}) as</p>
<ul class="list" style="list-style-type:disc">
<li><p>Cluster A1<strong>:</strong> {1, 2, 3, 4, 5}</p></li>
<li><p>Cluster A2: {6, 7, 8, 9, 10}</p></li>
<li><p>Cluster A3: {11, 12, 13, 14, 15}</p></li>
</ul>
<p>and expert classification (Partition B)</p>
<ul class="list" style="list-style-type:disc">
<li><p>Cluster B1: {1, 2, 6, 7, 11}</p></li>
<li><p>Cluster B2: {3, 4, 8, 9, 12}</p></li>
<li><p>Cluster B3: {5, 10, 13, 14, 15}</p></li>
</ul>
<p>The Optimal Mapping assignment is:</p>
<ul class="list" style="list-style-type:disc">
<li><p>A1<strong> ↔ </strong>B1</p></li>
<li><p>A2<strong> ↔ </strong>B2</p></li>
<li><p>A3<strong> ↔ </strong>B3</p></li>
</ul>
<p>Under this mapping, elements 3,4,5,6,7,10,11,12 – totaling eight elements – are mismatched. Thus the MR is</p>
<table class="disp-formula p" id="pcbi.1013344.e114"><tr><td class="formula"><math id="M114" display="block" overflow="linebreak"><mrow><mi>M</mi><mi>R</mi><mo stretchy="false">(</mo><mi>A</mi><mo>,</mo><mi>B</mi><mo stretchy="false">)</mo><mo>=</mo><mtext> </mtext><mfrac><mrow><mi>N</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mtext> </mtext><mi>o</mi><mi>f</mi><mtext> </mtext><mi>m</mi><mi>i</mi><mi>s</mi><mi>m</mi><mi>a</mi><mi>t</mi><mi>c</mi><mi>h</mi><mi>e</mi><mi>d</mi><mtext> </mtext><mi>e</mi><mi>l</mi><mi>e</mi><mi>m</mi><mi>e</mi><mi>n</mi><mi>t</mi><mi>s</mi></mrow><mrow><mi>T</mi><mi>o</mi><mi>t</mi><mi>a</mi><mi>l</mi><mtext> </mtext><mi>n</mi><mi>u</mi><mi>m</mi><mi>b</mi><mi>e</mi><mi>r</mi><mtext> </mtext><mi>o</mi><mi>f</mi><mtext> </mtext><mi>e</mi><mi>l</mi><mi>e</mi><mi>m</mi><mi>e</mi><mi>n</mi><mi>t</mi><mi>s</mi><mtext> </mtext></mrow></mfrac><mo>=</mo><mn>8</mn><mo>/</mo><mn>15</mn><mo>≈</mo><mn>0.533</mn></mrow></math></td></tr></table>
<p>All above algorithms were implemented in MATLAB</p></section><section id="sec017"><h3 class="pmc_sec_title">Calculating stimulus-response specificity and confusion scores</h3>
<p>We expanding our epsilon network clustering framework to all 200 drug regimes, using the predetermined optimal ε within each feature space. Clustering results were saved as confusion matrices and visualized in stimulus cluster maps. Each drug regime’s confusion matrix had a size of 15 × 15 with each column and row representing the different stimuli. Off-diagonal elements were assigned a value of 1 if the corresponding stimuli were in the same cluster and 0 otherwise. Non-responder stimuli for each drug regime were tracked using a separate vector. For each drug regime, since the confusion matrix is symmetric, only the lower triangular portion was visualized in stimulus cluster maps. Diagonal elements within the cluster maps were marked in red for inhibited trajectories and classified as “non-responders” using the separate non-responder vector. For each stimulus pair, the number of drug regimes in which they were confused was tallied. All of the results were implemented and visualized in MATLAB.</p>
<p>Non-responder trajectories are defined by an NFκB peak value below 0.05. Within the signaling codon space, we corrected the signaling codons EvL and Speed for non-responders to avoid non-sensical results. When clustering, the non-responder cluster is defined as the one that overlaps the most with the predefined non-responder trajectories and used in misclustering rate calculations.</p>
<p>Because the signaling codon space achieved the smallest misclustering rate, we defined stimulus-specific and confusion scores solely within the signaling codon space. Given a drug regime’s set of clusters, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e115"><math id="M115" display="inline" overflow="linebreak"><mrow><mrow><mrow><mi mathvariant="bold-italic">S</mi></mrow></mrow><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">{</mo><msub><mi>S</mi><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>S</mi><mrow><mn>2</mn></mrow></msub><mo>,</mo><mi>…</mi><mo>,</mo><msub><mi>S</mi><mrow><mi>k</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">}</mo></mrow></mrow></math></span>, we considered stimuli that were within the same cluster and therefore elicited similar NFκB responses as confused while those within distinct clusters were defined as specific. For all drug regimes, non-responder NFκB trajectories were defined by an NFκB peak value below 0.05. These non-responder trajectories were contained in the non-responder cluster and used to quantify the Inhibition Score (INH): <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e116"><math id="M116" display="inline" overflow="linebreak"><mrow><mi>I</mi><mi>N</mi><mi>H</mi><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">|</mo><msub><mi>S</mi><mrow><mi>n</mi><mi>o</mi><mi>n</mi><mo>−</mo><mi>r</mi><mi>e</mi><mi>s</mi><mi>p</mi><mi>o</mi><mi>n</mi><mi>d</mi><mi>e</mi><mi>r</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">|</mo></mrow></mrow></math></span>. Among other responsive stimuli, the Stimulus Response Specificity (SRS) score was calculated as the total number of clusters: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e117"><math id="M117" display="inline" overflow="linebreak"><mrow><mi>S</mi><mi>R</mi><mi>S</mi><mo>=</mo><mrow><mo fence="true" form="prefix" stretchy="true">|</mo><mrow><mrow><mi mathvariant="bold-italic">S</mi></mrow></mrow><mo fence="true" form="postfix" stretchy="true">|</mo></mrow></mrow></math></span> while the Stimulus-Response Confusion (SRC) score was defined as the size of the largest cluster: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013344.e118"><math id="M118" display="inline" overflow="linebreak"><mrow><mi>S</mi><mi>R</mi><mi>C</mi><mo>=</mo><msub><mo>max</mo><mrow><msub><mi>S</mi><mrow><mi>i</mi></mrow></msub><mo>∈</mo><mrow><mrow><mi mathvariant="bold-italic">S</mi></mrow></mrow></mrow></msub><mrow><mo fence="true" form="prefix" stretchy="true">|</mo><msub><mi>S</mi><mrow><mi>i</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">|</mo></mrow></mrow></math></span>. These calculations were implemented in MATLAB.</p></section><section id="sec018"><h3 class="pmc_sec_title">Comparing drug-treated and untreated conditions</h3>
<p>To compare drug-treated and untreated conditions, we constructed a confusion matrix in the signaling codon space by calculating the Euclidean distance between the untreated 15 stimulus-response vectors and their drug-treated counterparts. For responder conditions, distances less than the optimal ε were classified as confused and assigned a value of 1; otherwise, they were assigned 0. Non-responsive treated conditions were compared with the untreated low-dose LPS condition and marked in red. For each stimulus pair, the total number of drug regimes (out of 200) in which they were confused with untreated conditions was computed and visualized (<a href="#pcbi.1013344.g006" class="usa-link">Fig 6C</a>). All the results were calculated and visualized in MATLAB.</p></section></section><section id="sec019"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pcbi.1013344.s001"><div class="caption p">
<span>S1 Table. Details the NFκB ODE model species and reactions, and which reactions are targeted by each of the study’s drugs.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s001.xlsx</a><sup> (32.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013344.s002"><div class="caption p">
<span>S1 Fig. NFkB dynamic trajectories response to representative ligands and doses.</span><p>Trajectories of nuclear NFκB concentration (y-axis of each plot) over time (x-axis of each plot) for untreated conditions across 5 ligands and 3 doses. Rows indicate different ligands (labeled on the right of the panel), and columns specify ligand doses (labeled on the top of the panel). Stimulation indices are labeled in the top right corner.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s002.pdf</a><sup> (126.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013344.s003"><div class="caption p">
<span>S2 Fig. Signaling dynamic patterns derived from signaling codons.</span><p><strong>(A)</strong> Illustration of six signaling dynamic codons: Duration (Dur), Oscillations (Osc), Peak Amplitude (Amp), Total Activity (AUC), Early vs. Late Activity (EvL), and Activation Speed (Spe). <strong>(B)</strong> Signaling dynamic codons for nuclear NFκB trajectories following pharmacological perturbation. For each drug panel, signaling dynamic codons are stratified by column (labeled on the top) and stimuli are stratified by row (labeled on the left). Each individual plot pertains to one signaling codon (y-axis) for a particular stimulus at 3 doses (low – triangle, medium – square, high – circle, labeled on the bottom left), ranging from untreated to drug dose 20 (log scale x-axis).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s003.pdf</a><sup> (1.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013344.s004"><div class="caption p">
<span>S3 Fig. Construction of drug regime landscape from weights corresponding to seven components from the CP decomposition.</span><p><strong>(A)</strong> R2X plot depicting the percent variance explained after the application of CP decomposition for 1–8 components. <strong>(B)</strong> Heatmap of weights corresponding to seven temporal patterns using the drug treatment TSA at drug dose (DD) 2 as an example in defining the landscape for specific drug regime. Inside the heatmap, 15 rows depict 15 stimuli (5 ligands at 3 different doses). Colors within each row indicate the weights of the seven temporal patterns. These weights are the product of the component weights associated with the respective drug, drug dose, ligand, and ligand dose dimensions. The specific order of the 15 stimuli is outlined on the top of the figure, with L, M, and H denoting low-dose, medium-dose, and high-dose respectively. The weights of the temporal pattern (squares labeled by C1-C7 represent the weights for component 1–7 temporal patterns, respectively) stimulated by high-dose TNF under the TSA DD2 drug regime are specifically highlighted. <strong>(C)</strong> An example of the high-dose TNF stimulated NFκB trajectory perturbed by TSA at DD2 is displayed on the left. This can be approximated by the weighted sum (weights indicated by colors in the squares) of the seven temporal patterns (displayed adjacent to the colored squares). The weights are the product of the component weights corresponding to the specific drug, drug dose, ligand, and ligand dose dimensions obtained from CP decomposition. <strong>(D)</strong> The weights employed in the reconstruction of the simulated trajectory are the products of the weights in the dimensions of ligand, ligand dose, drug, and drug dose, with the yellow bars providing an example of calculating the weights for component 6 of (C). The subpanels and Fig 3E are the outcomes of Canonical Polyadic Decomposition (CPD) to nuclear NFκB time trajectory tensor, resulting in seven distinct components. These components are represented by their respective weights across various dimensions: time (Fig 3E), ligand, ligand doses, drugs, and drug dose index (this figure). <strong>(E)</strong> Temporal patterns of CP decomposition components under time point order alterations. We altered the time point order of the original tensor by splitting it into two parts along the time dimension, and then taking their reversed concatenation (i.e., the altered-time-order tensor had a time point order of (241:481, 1:240). We then applied CP decomposition using 7 components for this altered tensor.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s004.pdf</a><sup> (203.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013344.s005"><div class="caption p">
<span>S4 Fig. Construction of drug regime landscape from weights corresponding to five components from the fPCA.</span><p><strong>(A)</strong> R2X plot depicting the percent variance explained after applying fPCA using 1–10 principal components. <strong>(B)</strong> Heatmap of weights corresponding to five temporal patterns for the drug treatment Sel at drug dose (DD) 15. The weights are the score outputs from the decomposition. The 15 heatmap rows correspond to the 15 stimuli (5 ligands at 3 different doses) with their order annotated at the top of the heatmap (L = low-dose, M = medium-dose, H = high-dose). The squares labeled C1-C5 represent the five temporal pattern (i.e., components 1–5) weights for NFκB activity under high-dose TNF stimulation and perturbed by the Sel DD15 drug regime. <strong>(C)</strong> An example of the high-dose TNF stimulated NFκB trajectory perturbed by Sel at DD15 is displayed on the left. This can be approximated by the weighted sum (weights indicated by colors in the squares) of the five temporal patterns (displayed following each colored square).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s005.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s005.pdf</a><sup> (138.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013344.s006"><div class="caption p">
<span>S5 Fig. Construction of stimulus cluster maps for selected drug regimes.</span><p><strong>(A)</strong> Trajectories of nuclear NFκB concentration (y-axis of each plot) over time (x-axis of each plot) for drug regimes CHL DD8, IKKi DD11, MG132 DD6, Sel DD15, TAK1i DD14 across 5 ligands and 3 doses. Solid colored lines represent the trajectories under drug treatment, while dashed lines depict untreated trajectories. Rows indicate different ligands (labeled on the right of each set of drug regime plots), and columns specify ligand doses (labeled on the top of each set of plots). Stimulation indices are labeled in the top right corner. <strong>(B)</strong> Expert classifications for 5 representative regimes: CHL DD8, IKKi DD11, MG132 DD6, Sel DD15, and TAK1i DD14 used in epsilon network clustering. Clusters for each regime are annotated according to the trajectory indices in (A). <strong>(C)</strong> Stimulus cluster maps constructed from epsilon network clustering results for 5 representative regimes. Each row and column within one map corresponds to a specific stimulus, as denoted in <a href="#pcbi.1013344.s007" class="usa-link">S6A Fig</a>. Within each map, off-diagonal black squares represent responsive clusters and red squares on the diagonal represent inhibited NFkB signaling (non-responder). Panels from left to right display the expert classifications, clusters derived from the trajectory space, signaling codon space, 7 component CPD feature space, and 40 component CPD feature space, and fPCA feature space (Labeled on the top of the panel).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s006.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s006.pdf</a><sup> (450.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013344.s007"><div class="caption p">
<span>S6 Fig. Decoding stimulus-response confusion, specificity, and inhibition scores for an example drug treatment.</span><p><strong>(A)</strong> Stimulus cluster maps for trajectories treated with MG132 at DD6. <strong>(B)</strong> Examples of decreased temporal coding capacity (high doses of TNF and Pam3CSK, top row) and increased temporal coding capacity (high doses of Pam3CSK and CpG, bottom row), represented in the trajectory space (left and middle columns) and signaling codon space (right column).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s007.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s007.pdf</a><sup> (174.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013344.s009"><div class="caption p">
<span>S7 Fig. Signaling codon-derived visualization of temporal coding capacity for 180 drug combination regimes.</span><p>Stimulus cluster map for the 15 stimulus conditions under 180 drug combination regimes. The 180 regimes are comprised of 45 combinations of the 10 drugs and 4 dose combinations using DD5 and DD10 (45 x 4 = 180). Columns represent untreated stimuli and rows represent treated stimuli. Black clusters on and off the diagonal represent “confusion” between stimulus conditions’ NFκB responses. Red squares on the diagonal represent inhibited NFκB signaling (non-responders).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s009.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s009.pdf</a><sup> (215.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013344.s008"><div class="caption p">
<span>S8 Fig. Signaling codon-derived visualization of temporal coding capacity for 200 drug regimes vs. the 15 stimulated, untreated NFκB responses.</span><p>Stimulus cluster map for the 15 stimulus conditions under no drug treatment (untreated) vs. 15 stimulus conditions under <strong>(A)</strong> an example drug regime and (<strong>B)</strong> all 200 drug regimes. Columns represent untreated stimuli and rows represent treated stimuli. Black clusters on and off the diagonal represent “confusion” between treated and untreated NFκB responses under the same or different stimulus condition, respectively. Off-diagonal red squares represent inhibited NFκB signaling (non-responders). <strong>(C-D)</strong> Examples of confusion and distinction of the stimuli before and after drug treatment. Cross-stimulation confusion signaling codon Euclidean distances are annotated which are within the optimal ε boundary = 0.28.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s008.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s008.pdf</a><sup> (215.1KB, pdf) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>All data is deposited in Mendeley (<a href="https://data.mendeley.com/datasets/t3dbj86sjx/1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://data.mendeley.com/datasets/t3dbj86sjx/1</a>). All codes are available at GitHub (<a href="https://github.com/Xiaolu-Guo/Pharmacological_Perturbation_NFkB" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Xiaolu-Guo/Pharmacological_Perturbation_NFkB</a>).</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>The work is supported by National Institutes of Health (R01AI173214) to AH. XG received a salary from National Institutes of Health (R01AI173214). ERB and JLW were supported by National Institutes of Health NIGMS (R35GM147114). The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pcbi.1013344.ref001">
<span class="label">1.</span><cite>Purvis JE, Lahav G. Encoding and decoding cellular information through signaling dynamics. Cell. 2013;152(5):945–56. doi: 10.1016/j.cell.2013.02.005

</cite> [<a href="https://doi.org/10.1016/j.cell.2013.02.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3707615/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23452846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Encoding%20and%20decoding%20cellular%20information%20through%20signaling%20dynamics&amp;author=JE%20Purvis&amp;author=G%20Lahav&amp;volume=152&amp;issue=5&amp;publication_year=2013&amp;pages=945-56&amp;pmid=23452846&amp;doi=10.1016/j.cell.2013.02.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref002">
<span class="label">2.</span><cite>Alexander RP, Kim PM, Emonet T, Gerstein MB. Understanding modularity in molecular networks requires dynamics. Sci Signal. 2009;2(81):pe44. doi: 10.1126/scisignal.281pe44

</cite> [<a href="https://doi.org/10.1126/scisignal.281pe44" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4243459/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19638611/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Signal&amp;title=Understanding%20modularity%20in%20molecular%20networks%20requires%20dynamics&amp;author=RP%20Alexander&amp;author=PM%20Kim&amp;author=T%20Emonet&amp;author=MB%20Gerstein&amp;volume=2&amp;issue=81&amp;publication_year=2009&amp;pmid=19638611&amp;doi=10.1126/scisignal.281pe44&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref003">
<span class="label">3.</span><cite>Behar M, Hoffmann A. Understanding the temporal codes of intra-cellular signals. Curr Opin Genet Dev. 2010;20(6):684–93. doi: 10.1016/j.gde.2010.09.007

</cite> [<a href="https://doi.org/10.1016/j.gde.2010.09.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2982931/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20956081/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Genet%20Dev&amp;title=Understanding%20the%20temporal%20codes%20of%20intra-cellular%20signals&amp;author=M%20Behar&amp;author=A%20Hoffmann&amp;volume=20&amp;issue=6&amp;publication_year=2010&amp;pages=684-93&amp;pmid=20956081&amp;doi=10.1016/j.gde.2010.09.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref004">
<span class="label">4.</span><cite>Adelaja A, Taylor B, Sheu KM, Liu Y, Luecke S, Hoffmann A. Six distinct NFκB signaling codons convey discrete information to distinguish stimuli and enable appropriate macrophage responses. Immunity. 2021;54(5):916-930.e7. doi: 10.1016/j.immuni.2021.04.002</cite> [<a href="https://doi.org/10.1016/j.immuni.2021.04.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8184127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33979588/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Six%20distinct%20NF%CE%BAB%20signaling%20codons%20convey%20discrete%20information%20to%20distinguish%20stimuli%20and%20enable%20appropriate%20macrophage%20responses&amp;author=A%20Adelaja&amp;author=B%20Taylor&amp;author=KM%20Sheu&amp;author=Y%20Liu&amp;author=S%20Luecke&amp;volume=54&amp;issue=5&amp;publication_year=2021&amp;pmid=33979588&amp;doi=10.1016/j.immuni.2021.04.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref005">
<span class="label">5.</span><cite>Tay S, Hughey JJ, Lee TK, Lipniacki T, Quake SR, Covert MW. Single-cell NF-kappaB dynamics reveal digital activation and analogue information processing. Nature. 2010;466(7303):267–71.
</cite> [<a href="https://doi.org/10.1038/nature09145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3105528/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20581820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Single-cell%20NF-kappaB%20dynamics%20reveal%20digital%20activation%20and%20analogue%20information%20processing&amp;author=S%20Tay&amp;author=JJ%20Hughey&amp;author=TK%20Lee&amp;author=T%20Lipniacki&amp;author=SR%20Quake&amp;volume=466&amp;issue=7303&amp;publication_year=2010&amp;pages=267-71&amp;pmid=20581820&amp;doi=10.1038/nature09145&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref006">
<span class="label">6.</span><cite>Zhang Q, Gupta S, Schipper DL, Kowalczyk GJ, Mancini AE, Faeder JR, et al. NF-κB Dynamics Discriminate between TNF Doses in Single Cells. Cell Syst. 2017;5(6):638-645.e5. doi: 10.1016/j.cels.2017.11.007</cite> [<a href="https://doi.org/10.1016/j.cels.2017.11.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5746429/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29128333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Syst&amp;title=NF-%CE%BAB%20Dynamics%20Discriminate%20between%20TNF%20Doses%20in%20Single%20Cells&amp;author=Q%20Zhang&amp;author=S%20Gupta&amp;author=DL%20Schipper&amp;author=GJ%20Kowalczyk&amp;author=AE%20Mancini&amp;volume=5&amp;issue=6&amp;publication_year=2017&amp;pmid=29128333&amp;doi=10.1016/j.cels.2017.11.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref007">
<span class="label">7.</span><cite>Rahman SMT, Singh A, Lowe S, Aqdas M, Jiang K, Vaidehi Narayanan H, et al. Co-imaging of RelA and c-Rel reveals features of NF-κB signaling for ligand discrimination. Cell Rep. 2024;43(3):113940. doi: 10.1016/j.celrep.2024.113940

</cite> [<a href="https://doi.org/10.1016/j.celrep.2024.113940" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11015162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38483906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Co-imaging%20of%20RelA%20and%20c-Rel%20reveals%20features%20of%20NF-%CE%BAB%20signaling%20for%20ligand%20discrimination&amp;author=SMT%20Rahman&amp;author=A%20Singh&amp;author=S%20Lowe&amp;author=M%20Aqdas&amp;author=K%20Jiang&amp;volume=43&amp;issue=3&amp;publication_year=2024&amp;pages=113940&amp;pmid=38483906&amp;doi=10.1016/j.celrep.2024.113940&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref008">
<span class="label">8.</span><cite>Selimkhanov J, Taylor B, Yao J, Pilko A, Albeck J, Hoffmann A, et al. Accurate information transmission through dynamic biochemical signaling networks. Science. 2014;346(6215):1370–3.
</cite> [<a href="https://doi.org/10.1126/science.1254933" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4813785/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25504722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Accurate%20information%20transmission%20through%20dynamic%20biochemical%20signaling%20networks&amp;author=J%20Selimkhanov&amp;author=B%20Taylor&amp;author=J%20Yao&amp;author=A%20Pilko&amp;author=J%20Albeck&amp;volume=346&amp;issue=6215&amp;publication_year=2014&amp;pages=1370-3&amp;pmid=25504722&amp;doi=10.1126/science.1254933&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref009">
<span class="label">9.</span><cite>Davies AE, Pargett M, Siebert S, Gillies TE, Choi Y, Tobin SJ, et al. Systems-Level Properties of EGFR-RAS-ERK Signaling Amplify Local Signals to Generate Dynamic Gene Expression Heterogeneity. Cell Systems. 2020;11(2):161-175.e5. doi: 10.1016/j.cels.2020.07.002</cite> [<a href="https://doi.org/10.1016/j.cels.2020.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7856305/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32726596/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Systems&amp;title=Systems-Level%20Properties%20of%20EGFR-RAS-ERK%20Signaling%20Amplify%20Local%20Signals%20to%20Generate%20Dynamic%20Gene%20Expression%20Heterogeneity&amp;author=AE%20Davies&amp;author=M%20Pargett&amp;author=S%20Siebert&amp;author=TE%20Gillies&amp;author=Y%20Choi&amp;volume=11&amp;issue=2&amp;publication_year=2020&amp;pmid=32726596&amp;doi=10.1016/j.cels.2020.07.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref010">
<span class="label">10.</span><cite>Peterson AF, Ingram K, Huang EJ, Parksong J, McKenney C, Bever GS, et al. Systematic analysis of the MAPK signaling network reveals MAP3K-driven control of cell fate. Cell Systems. 2022;13(11):885-894.e4. doi: 10.1016/j.cels.2022.10.002</cite> [<a href="https://doi.org/10.1016/j.cels.2022.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9923805/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36356576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Systems&amp;title=Systematic%20analysis%20of%20the%20MAPK%20signaling%20network%20reveals%20MAP3K-driven%20control%20of%20cell%20fate&amp;author=AF%20Peterson&amp;author=K%20Ingram&amp;author=EJ%20Huang&amp;author=J%20Parksong&amp;author=C%20McKenney&amp;volume=13&amp;issue=11&amp;publication_year=2022&amp;pmid=36356576&amp;doi=10.1016/j.cels.2022.10.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref011">
<span class="label">11.</span><cite>Paek AL, Liu JC, Loewer A, Forrester WC, Lahav G. Cell-to-Cell Variation in p53 Dynamics Leads to Fractional Killing. Cell. 2016;165(3):631–42. doi: 10.1016/j.cell.2016.03.025

</cite> [<a href="https://doi.org/10.1016/j.cell.2016.03.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5217463/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27062928/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Cell-to-Cell%20Variation%20in%20p53%20Dynamics%20Leads%20to%20Fractional%20Killing&amp;author=AL%20Paek&amp;author=JC%20Liu&amp;author=A%20Loewer&amp;author=WC%20Forrester&amp;author=G%20Lahav&amp;volume=165&amp;issue=3&amp;publication_year=2016&amp;pages=631-42&amp;pmid=27062928&amp;doi=10.1016/j.cell.2016.03.025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref012">
<span class="label">12.</span><cite>Hanson RL, Batchelor E. Coordination of MAPK and p53 dynamics in the cellular responses to DNA damage and oxidative stress. Mol Syst Biol. 2022;18(12):e11401. doi: 10.15252/msb.202211401

</cite> [<a href="https://doi.org/10.15252/msb.202211401" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9724178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36472304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Syst%20Biol&amp;title=Coordination%20of%20MAPK%20and%20p53%20dynamics%20in%20the%20cellular%20responses%20to%20DNA%20damage%20and%20oxidative%20stress&amp;author=RL%20Hanson&amp;author=E%20Batchelor&amp;volume=18&amp;issue=12&amp;publication_year=2022&amp;pmid=36472304&amp;doi=10.15252/msb.202211401&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref013">
<span class="label">13.</span><cite>Luecke S, Guo X, Sheu KM, Singh A, Lowe SC, Han M, et al. Dynamical and combinatorial coding by MAPK p38 and NFκB in the inflammatory response of macrophages. Mol Syst Biol. 2024;20(8):898–932. doi: 10.1038/s44320-024-00047-4

</cite> [<a href="https://doi.org/10.1038/s44320-024-00047-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11297158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38872050/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Syst%20Biol&amp;title=Dynamical%20and%20combinatorial%20coding%20by%20MAPK%20p38%20and%20NF%CE%BAB%20in%20the%20inflammatory%20response%20of%20macrophages&amp;author=S%20Luecke&amp;author=X%20Guo&amp;author=KM%20Sheu&amp;author=A%20Singh&amp;author=SC%20Lowe&amp;volume=20&amp;issue=8&amp;publication_year=2024&amp;pages=898-932&amp;pmid=38872050&amp;doi=10.1038/s44320-024-00047-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref014">
<span class="label">14.</span><cite>Hanahan D, Weinberg RA. Hallmarks of cancer: the next generation. Cell. 2011;144(5):646–74. doi: 10.1016/j.cell.2011.02.013

</cite> [<a href="https://doi.org/10.1016/j.cell.2011.02.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21376230/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Hallmarks%20of%20cancer:%20the%20next%20generation&amp;author=D%20Hanahan&amp;author=RA%20Weinberg&amp;volume=144&amp;issue=5&amp;publication_year=2011&amp;pages=646-74&amp;pmid=21376230&amp;doi=10.1016/j.cell.2011.02.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref015">
<span class="label">15.</span><cite>Luecke S, Sheu KM, Hoffmann A. Stimulus-specific responses in innate immunity: Multilayered regulatory circuits. Immunity. 2021;54(9):1915–32. doi: 10.1016/j.immuni.2021.08.018

</cite> [<a href="https://doi.org/10.1016/j.immuni.2021.08.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8452151/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34525335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Stimulus-specific%20responses%20in%20innate%20immunity:%20Multilayered%20regulatory%20circuits&amp;author=S%20Luecke&amp;author=KM%20Sheu&amp;author=A%20Hoffmann&amp;volume=54&amp;issue=9&amp;publication_year=2021&amp;pages=1915-32&amp;pmid=34525335&amp;doi=10.1016/j.immuni.2021.08.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref016">
<span class="label">16.</span><cite>Behar M, Barken D, Werner SL, Hoffmann A. The dynamics of signaling as a pharmacological target. Cell. 2013;155(2):448–61. doi: 10.1016/j.cell.2013.09.018

</cite> [<a href="https://doi.org/10.1016/j.cell.2013.09.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3856316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24120141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The%20dynamics%20of%20signaling%20as%20a%20pharmacological%20target&amp;author=M%20Behar&amp;author=D%20Barken&amp;author=SL%20Werner&amp;author=A%20Hoffmann&amp;volume=155&amp;issue=2&amp;publication_year=2013&amp;pages=448-61&amp;pmid=24120141&amp;doi=10.1016/j.cell.2013.09.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref017">
<span class="label">17.</span><cite>Bugaj LJ, Sabnis AJ, Mitchell A, Garbarino JE, Toettcher JE, Bivona TG, et al. Cancer mutations and targeted drugs can disrupt dynamic signal encoding by the Ras-Erk pathway. Science. 2018;361(6405):eaao3048. doi: 10.1126/science.aao3048

</cite> [<a href="https://doi.org/10.1126/science.aao3048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6430110/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30166458/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Cancer%20mutations%20and%20targeted%20drugs%20can%20disrupt%20dynamic%20signal%20encoding%20by%20the%20Ras-Erk%20pathway&amp;author=LJ%20Bugaj&amp;author=AJ%20Sabnis&amp;author=A%20Mitchell&amp;author=JE%20Garbarino&amp;author=JE%20Toettcher&amp;volume=361&amp;issue=6405&amp;publication_year=2018&amp;pmid=30166458&amp;doi=10.1126/science.aao3048&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref018">
<span class="label">18.</span><cite>Goglia AG, Wilson MZ, Jena SG, Silbert J, Basta LP, Devenport D, et al. A Live-Cell Screen for Altered Erk Dynamics Reveals Principles of Proliferative Control. Cell Syst. 2020;10(3):240-253.e6. doi: 10.1016/j.cels.2020.01.002</cite> [<a href="https://doi.org/10.1016/j.cels.2020.01.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7540725/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32191874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Syst&amp;title=A%20Live-Cell%20Screen%20for%20Altered%20Erk%20Dynamics%20Reveals%20Principles%20of%20Proliferative%20Control&amp;author=AG%20Goglia&amp;author=MZ%20Wilson&amp;author=SG%20Jena&amp;author=J%20Silbert&amp;author=LP%20Basta&amp;volume=10&amp;issue=3&amp;publication_year=2020&amp;pmid=32191874&amp;doi=10.1016/j.cels.2020.01.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref019">
<span class="label">19.</span><cite>Madsen RR, Vanhaesebroeck B. Cracking the context-specific PI3K signaling code. Sci Signal. 2020;13(613):eaay2940. doi: 10.1126/scisignal.aay2940

</cite> [<a href="https://doi.org/10.1126/scisignal.aay2940" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31911433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Signal&amp;title=Cracking%20the%20context-specific%20PI3K%20signaling%20code&amp;author=RR%20Madsen&amp;author=B%20Vanhaesebroeck&amp;volume=13&amp;issue=613&amp;publication_year=2020&amp;pmid=31911433&amp;doi=10.1126/scisignal.aay2940&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref020">
<span class="label">20.</span><cite>Mitchell A, Wei P, Lim WA. Oscillatory stress stimulation uncovers an Achilles’ heel of the yeast MAPK signaling network. Science. 2015;350(6266):1379–83.
</cite> [<a href="https://doi.org/10.1126/science.aab0892" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4721531/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26586187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Oscillatory%20stress%20stimulation%20uncovers%20an%20Achilles%E2%80%99%20heel%20of%20the%20yeast%20MAPK%20signaling%20network&amp;author=A%20Mitchell&amp;author=P%20Wei&amp;author=WA%20Lim&amp;volume=350&amp;issue=6266&amp;publication_year=2015&amp;pages=1379-83&amp;pmid=26586187&amp;doi=10.1126/science.aab0892&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref021">
<span class="label">21.</span><cite>Zhang X, Crespo A, Fernández A. Turning promiscuous kinase inhibitors into safer drugs. Trends Biotechnol. 2008;26(6):295–301. doi: 10.1016/j.tibtech.2008.02.008

</cite> [<a href="https://doi.org/10.1016/j.tibtech.2008.02.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18403035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Biotechnol&amp;title=Turning%20promiscuous%20kinase%20inhibitors%20into%20safer%20drugs&amp;author=X%20Zhang&amp;author=A%20Crespo&amp;author=A%20Fern%C3%A1ndez&amp;volume=26&amp;issue=6&amp;publication_year=2008&amp;pages=295-301&amp;pmid=18403035&amp;doi=10.1016/j.tibtech.2008.02.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref022">
<span class="label">22.</span><cite>Alidoost M, Wilson JL. Preclinical side effect prediction through pathway engineering of protein interaction network models. CPT Pharmacometrics Syst Pharmacol. 2024;13(7):1180–200. doi: 10.1002/psp4.13150

</cite> [<a href="https://doi.org/10.1002/psp4.13150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11247120/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38736280/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=CPT%20Pharmacometrics%20Syst%20Pharmacol&amp;title=Preclinical%20side%20effect%20prediction%20through%20pathway%20engineering%20of%20protein%20interaction%20network%20models&amp;author=M%20Alidoost&amp;author=JL%20Wilson&amp;volume=13&amp;issue=7&amp;publication_year=2024&amp;pages=1180-200&amp;pmid=38736280&amp;doi=10.1002/psp4.13150&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref023">
<span class="label">23.</span><cite>Berger SI, Iyengar R. Role of systems pharmacology in understanding drug adverse events. Wiley Interdiscip Rev Syst Biol Med. 2011;3(2):129–35. doi: 10.1002/wsbm.114

</cite> [<a href="https://doi.org/10.1002/wsbm.114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3057924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20803507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Wiley%20Interdiscip%20Rev%20Syst%20Biol%20Med&amp;title=Role%20of%20systems%20pharmacology%20in%20understanding%20drug%20adverse%20events&amp;author=SI%20Berger&amp;author=R%20Iyengar&amp;volume=3&amp;issue=2&amp;publication_year=2011&amp;pages=129-35&amp;pmid=20803507&amp;doi=10.1002/wsbm.114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref024">
<span class="label">24.</span><cite>Force T, Krause DS, Van Etten RA. Molecular mechanisms of cardiotoxicity of tyrosine kinase inhibition. Nat Rev Cancer. 2007;7(5):332–44. doi: 10.1038/nrc2106

</cite> [<a href="https://doi.org/10.1038/nrc2106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17457301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Cancer&amp;title=Molecular%20mechanisms%20of%20cardiotoxicity%20of%20tyrosine%20kinase%20inhibition&amp;author=T%20Force&amp;author=DS%20Krause&amp;author=RA%20Van%20Etten&amp;volume=7&amp;issue=5&amp;publication_year=2007&amp;pages=332-44&amp;pmid=17457301&amp;doi=10.1038/nrc2106&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref025">
<span class="label">25.</span><cite>Komarova NL, Zou X, Nie Q, Bardwell L. A theoretical framework for specificity in cell signaling. Molecular Systems Biology. 2005;1:2005.0023. doi: 10.1038/msb.2005.0023</cite> [<a href="https://doi.org/10.1038/msb.2005.0023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1681467/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16729058/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Systems%20Biology&amp;title=A%20theoretical%20framework%20for%20specificity%20in%20cell%20signaling&amp;author=NL%20Komarova&amp;author=X%20Zou&amp;author=Q%20Nie&amp;author=L%20Bardwell&amp;volume=1&amp;publication_year=2005&amp;pmid=16729058&amp;doi=10.1038/msb.2005.0023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref026">
<span class="label">26.</span><cite>Behar M, Dohlman HG, Elston TC. Kinetic insulation as an effective mechanism for achieving pathway specificity in intracellular signaling networks. Proc Natl Acad Sci U S A. 2007;104(41):16146–51.
</cite> [<a href="https://doi.org/10.1073/pnas.0703894104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2042176/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17913886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Kinetic%20insulation%20as%20an%20effective%20mechanism%20for%20achieving%20pathway%20specificity%20in%20intracellular%20signaling%20networks&amp;author=M%20Behar&amp;author=HG%20Dohlman&amp;author=TC%20Elston&amp;volume=104&amp;issue=41&amp;publication_year=2007&amp;pages=16146-51&amp;pmid=17913886&amp;doi=10.1073/pnas.0703894104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref027">
<span class="label">27.</span><cite>Hayden MS, West AP, Ghosh S. NF-kappaB and the immune response. Oncogene. 2006;25(51):6758–80. doi: 10.1038/sj.onc.1209943

</cite> [<a href="https://doi.org/10.1038/sj.onc.1209943" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17072327/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=NF-kappaB%20and%20the%20immune%20response&amp;author=MS%20Hayden&amp;author=AP%20West&amp;author=S%20Ghosh&amp;volume=25&amp;issue=51&amp;publication_year=2006&amp;pages=6758-80&amp;pmid=17072327&amp;doi=10.1038/sj.onc.1209943&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref028">
<span class="label">28.</span><cite>Aqdas M, Sung M-H. NF-κB dynamics in the language of immune cells. Trends Immunol. 2023;44(1):32–43. doi: 10.1016/j.it.2022.11.005

</cite> [<a href="https://doi.org/10.1016/j.it.2022.11.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9811507/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36473794/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Immunol&amp;title=NF-%CE%BAB%20dynamics%20in%20the%20language%20of%20immune%20cells&amp;author=M%20Aqdas&amp;author=M-H%20Sung&amp;volume=44&amp;issue=1&amp;publication_year=2023&amp;pages=32-43&amp;pmid=36473794&amp;doi=10.1016/j.it.2022.11.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref029">
<span class="label">29.</span><cite>Singh A, Sen S, Iter M, Adelaja A, Luecke S, Guo X, et al. Stimulus-response signaling dynamics characterize macrophage polarization states. Cell Syst. 2024;15(6):563-577.e6. doi: 10.1016/j.cels.2024.05.002

</cite> [<a href="https://doi.org/10.1016/j.cels.2024.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11226196/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38843840/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Syst&amp;title=Stimulus-response%20signaling%20dynamics%20characterize%20macrophage%20polarization%20states&amp;author=A%20Singh&amp;author=S%20Sen&amp;author=M%20Iter&amp;author=A%20Adelaja&amp;author=S%20Luecke&amp;volume=15&amp;issue=6&amp;publication_year=2024&amp;pmid=38843840&amp;doi=10.1016/j.cels.2024.05.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref030">
<span class="label">30.</span><cite>Hoffmann A, Levchenko A, Scott ML, Baltimore D. The IkappaB-NF-kappaB signaling module: temporal control and selective gene activation. Science. 2002;298(5596):1241–5. doi: 10.1126/science.1071914

</cite> [<a href="https://doi.org/10.1126/science.1071914" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12424381/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20IkappaB-NF-kappaB%20signaling%20module:%20temporal%20control%20and%20selective%20gene%20activation&amp;author=A%20Hoffmann&amp;author=A%20Levchenko&amp;author=ML%20Scott&amp;author=D%20Baltimore&amp;volume=298&amp;issue=5596&amp;publication_year=2002&amp;pages=1241-5&amp;pmid=12424381&amp;doi=10.1126/science.1071914&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref031">
<span class="label">31.</span><cite>Werner SL, Barken D, Hoffmann A. Stimulus specificity of gene expression programs determined by temporal control of IKK activity. Science. 2005;309(5742).</cite> [<a href="https://doi.org/10.1126/science.1113319" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16166517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Stimulus%20specificity%20of%20gene%20expression%20programs%20determined%20by%20temporal%20control%20of%20IKK%20activity&amp;author=SL%20Werner&amp;author=D%20Barken&amp;author=A%20Hoffmann&amp;volume=309&amp;issue=5742&amp;publication_year=2005&amp;pmid=16166517&amp;doi=10.1126/science.1113319&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref032">
<span class="label">32.</span><cite>Lee REC, Walker SR, Savery K, Frank DA, Gaudet S. Fold-change of nuclear NF-κB determines TNF-induced transcription in single cells. Molecular Cell. 2014;53(6):867–79.
</cite> [<a href="https://doi.org/10.1016/j.molcel.2014.01.026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3977799/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24530305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Cell&amp;title=Fold-change%20of%20nuclear%20NF-%CE%BAB%20determines%20TNF-induced%20transcription%20in%20single%20cells&amp;author=REC%20Lee&amp;author=SR%20Walker&amp;author=K%20Savery&amp;author=DA%20Frank&amp;author=S%20Gaudet&amp;volume=53&amp;issue=6&amp;publication_year=2014&amp;pages=867-79&amp;pmid=24530305&amp;doi=10.1016/j.molcel.2014.01.026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref033">
<span class="label">33.</span><cite>Sen S, Cheng Z, Sheu KM, Chen YH, Hoffmann A. Gene Regulatory Strategies that Decode the Duration of NFκB Dynamics Contribute to LPS- versus TNF-Specific Gene Expression. Cell Syst. 2020.  Feb 26;10(2):169–82.e5.
</cite> [<a href="https://doi.org/10.1016/j.cels.2019.12.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7047529/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31972132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Syst&amp;title=Gene%20Regulatory%20Strategies%20that%20Decode%20the%20Duration%20of%20NF%CE%BAB%20Dynamics%20Contribute%20to%20LPS-%20versus%20TNF-Specific%20Gene%20Expression&amp;author=S%20Sen&amp;author=Z%20Cheng&amp;author=KM%20Sheu&amp;author=YH%20Chen&amp;author=A%20Hoffmann&amp;volume=10&amp;issue=2&amp;publication_year=2020&amp;pages=169-82.e5&amp;pmid=31972132&amp;doi=10.1016/j.cels.2019.12.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref034">
<span class="label">34.</span><cite>Cheng QJ, Ohta S, Sheu KM, Spreafico R, Adelaja A, Taylor B, et al. NF-κB dynamics determine the stimulus specificity of epigenomic reprogramming in macrophages. Science. 2021;372(6548):1349–53.
</cite> [<a href="https://doi.org/10.1126/science.abc0269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8489855/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34140389/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=NF-%CE%BAB%20dynamics%20determine%20the%20stimulus%20specificity%20of%20epigenomic%20reprogramming%20in%20macrophages&amp;author=QJ%20Cheng&amp;author=S%20Ohta&amp;author=KM%20Sheu&amp;author=R%20Spreafico&amp;author=A%20Adelaja&amp;volume=372&amp;issue=6548&amp;publication_year=2021&amp;pages=1349-53&amp;pmid=34140389&amp;doi=10.1126/science.abc0269&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref035">
<span class="label">35.</span><cite>Ando M, Magi S, Seki M, Suzuki Y, Kasukawa T, Lefaudeux D, et al. IκBα is required for full transcriptional induction of some NFκB-regulated genes in response to TNF in MCF-7 cells. NPJ Syst Biol Appl. 2021;7:42.
</cite> [<a href="https://doi.org/10.1038/s41540-021-00204-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8636565/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34853340/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NPJ%20Syst%20Biol%20Appl&amp;title=I%CE%BAB%CE%B1%20is%20required%20for%20full%20transcriptional%20induction%20of%20some%20NF%CE%BAB-regulated%20genes%20in%20response%20to%20TNF%20in%20MCF-7%20cells&amp;author=M%20Ando&amp;author=S%20Magi&amp;author=M%20Seki&amp;author=Y%20Suzuki&amp;author=T%20Kasukawa&amp;volume=7&amp;publication_year=2021&amp;pages=42&amp;pmid=34853340&amp;doi=10.1038/s41540-021-00204-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref036">
<span class="label">36.</span><cite>Sheu KM, Hoffmann A. Functional hallmarks of healthy macrophage responses: their regulatory basis and disease relevance. Annu Rev Immunol. 2022;40:295–321.
</cite> [<a href="https://doi.org/10.1146/annurev-immunol-101320-031555" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10074967/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35471841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu%20Rev%20Immunol&amp;title=Functional%20hallmarks%20of%20healthy%20macrophage%20responses:%20their%20regulatory%20basis%20and%20disease%20relevance&amp;author=KM%20Sheu&amp;author=A%20Hoffmann&amp;volume=40&amp;publication_year=2022&amp;pages=295-321&amp;pmid=35471841&amp;doi=10.1146/annurev-immunol-101320-031555&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref037">
<span class="label">37.</span><cite>Sheu K, Luecke S, Hoffmann A. Stimulus-specificity in the Responses of Immune Sentinel Cells. Curr Opin Syst Biol. 2019;18:53–61. doi: 10.1016/j.coisb.2019.10.011

</cite> [<a href="https://doi.org/10.1016/j.coisb.2019.10.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7450653/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32864512/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opin%20Syst%20Biol&amp;title=Stimulus-specificity%20in%20the%20Responses%20of%20Immune%20Sentinel%20Cells&amp;author=K%20Sheu&amp;author=S%20Luecke&amp;author=A%20Hoffmann&amp;volume=18&amp;publication_year=2019&amp;pages=53-61&amp;pmid=32864512&amp;doi=10.1016/j.coisb.2019.10.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref038">
<span class="label">38.</span><cite>Murray PJ, Wynn TA. Protective and pathogenic functions of macrophage subsets. Nat Rev Immunol. 2011;11(11):723–37.
</cite> [<a href="https://doi.org/10.1038/nri3073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3422549/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21997792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Immunol&amp;title=Protective%20and%20pathogenic%20functions%20of%20macrophage%20subsets&amp;author=PJ%20Murray&amp;author=TA%20Wynn&amp;volume=11&amp;issue=11&amp;publication_year=2011&amp;pages=723-37&amp;pmid=21997792&amp;doi=10.1038/nri3073&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref039">
<span class="label">39.</span><cite>Basak S, Behar M, Hoffmann A. Lessons from mathematically modeling the NF-κB pathway. Immunol Rev. 2012;246(1):221–38. doi: 10.1111/j.1600-065X.2011.01092.x

</cite> [<a href="https://doi.org/10.1111/j.1600-065X.2011.01092.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3343698/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22435558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunol%20Rev&amp;title=Lessons%20from%20mathematically%20modeling%20the%20NF-%CE%BAB%20pathway&amp;author=S%20Basak&amp;author=M%20Behar&amp;author=A%20Hoffmann&amp;volume=246&amp;issue=1&amp;publication_year=2012&amp;pages=221-38&amp;pmid=22435558&amp;doi=10.1111/j.1600-065X.2011.01092.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref040">
<span class="label">40.</span><cite>Gupta SC, Sundaram C, Reuter S, Aggarwal BB. Inhibiting NF-κB activation by small molecules as a therapeutic strategy. Biochim Biophys Acta. 2010;1799(10–12):775–87. doi: 10.1016/j.bbagrm.2010.05.004

</cite> [<a href="https://doi.org/10.1016/j.bbagrm.2010.05.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2955987/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20493977/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&amp;title=Inhibiting%20NF-%CE%BAB%20activation%20by%20small%20molecules%20as%20a%20therapeutic%20strategy&amp;author=SC%20Gupta&amp;author=C%20Sundaram&amp;author=S%20Reuter&amp;author=BB%20Aggarwal&amp;volume=1799&amp;publication_year=2010&amp;pages=775-87&amp;pmid=20493977&amp;doi=10.1016/j.bbagrm.2010.05.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref041">
<span class="label">41.</span><cite>Behar M, Hoffmann A. Tunable signal processing through a kinase control cycle: the IKK signaling node. Biophys J. 2013;105(1):231–41.
</cite> [<a href="https://doi.org/10.1016/j.bpj.2013.05.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3699752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23823243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Tunable%20signal%20processing%20through%20a%20kinase%20control%20cycle:%20the%20IKK%20signaling%20node&amp;author=M%20Behar&amp;author=A%20Hoffmann&amp;volume=105&amp;issue=1&amp;publication_year=2013&amp;pages=231-41&amp;pmid=23823243&amp;doi=10.1016/j.bpj.2013.05.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref042">
<span class="label">42.</span><cite>Winston JT, Strack P, Beer-Romero P, Chu CY, Elledge SJ, Harper JW. The SCFbeta-TRCP-ubiquitin ligase complex associates specifically with phosphorylated destruction motifs in IkappaBalpha and beta-catenin and stimulates IkappaBalpha ubiquitination in vitro. Genes Dev. 1999;13(3):270–83. doi: 10.1101/gad.13.3.270

</cite> [<a href="https://doi.org/10.1101/gad.13.3.270" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC316433/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9990852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev&amp;title=The%20SCFbeta-TRCP-ubiquitin%20ligase%20complex%20associates%20specifically%20with%20phosphorylated%20destruction%20motifs%20in%20IkappaBalpha%20and%20beta-catenin%20and%20stimulates%20IkappaBalpha%20ubiquitination%20in%20vitro&amp;author=JT%20Winston&amp;author=P%20Strack&amp;author=P%20Beer-Romero&amp;author=CY%20Chu&amp;author=SJ%20Elledge&amp;volume=13&amp;issue=3&amp;publication_year=1999&amp;pages=270-83&amp;pmid=9990852&amp;doi=10.1101/gad.13.3.270&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref043">
<span class="label">43.</span><cite>Benary U, Wolf J. Controlling Nuclear NF-κB Dynamics by β-TrCP-Insights from a Computational Model. Biomedicines. 2019;7(2):40. doi: 10.3390/biomedicines7020040

</cite> [<a href="https://doi.org/10.3390/biomedicines7020040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6631534/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31137887/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomedicines&amp;title=Controlling%20Nuclear%20NF-%CE%BAB%20Dynamics%20by%20%CE%B2-TrCP-Insights%20from%20a%20Computational%20Model&amp;author=U%20Benary&amp;author=J%20Wolf&amp;volume=7&amp;issue=2&amp;publication_year=2019&amp;pages=40&amp;pmid=31137887&amp;doi=10.3390/biomedicines7020040&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref044">
<span class="label">44.</span><cite>Witt J, Barisic S, Schumann E, Allgöwer F, Sawodny O, Sauter T, et al. Mechanism of PP2A-mediated IKK beta dephosphorylation: a systems biological approach. BMC Syst Biol. 2009;3:71. doi: 10.1186/1752-0509-3-71

</cite> [<a href="https://doi.org/10.1186/1752-0509-3-71" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2727496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19607706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Syst%20Biol&amp;title=Mechanism%20of%20PP2A-mediated%20IKK%20beta%20dephosphorylation:%20a%20systems%20biological%20approach&amp;author=J%20Witt&amp;author=S%20Barisic&amp;author=E%20Schumann&amp;author=F%20Allg%C3%B6wer&amp;author=O%20Sawodny&amp;volume=3&amp;publication_year=2009&amp;pages=71&amp;pmid=19607706&amp;doi=10.1186/1752-0509-3-71&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref045">
<span class="label">45.</span><cite>Ho WS, Wang H, Maggio D, Kovach JS, Zhang Q, Song Q, et al. Pharmacologic inhibition of protein phosphatase-2A achieves durable immune-mediated antitumor activity when combined with PD-1 blockade. Nat Commun. 2018;9(1):2126. doi: 10.1038/s41467-018-04425-z

</cite> [<a href="https://doi.org/10.1038/s41467-018-04425-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5974350/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29844427/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Pharmacologic%20inhibition%20of%20protein%20phosphatase-2A%20achieves%20durable%20immune-mediated%20antitumor%20activity%20when%20combined%20with%20PD-1%20blockade&amp;author=WS%20Ho&amp;author=H%20Wang&amp;author=D%20Maggio&amp;author=JS%20Kovach&amp;author=Q%20Zhang&amp;volume=9&amp;issue=1&amp;publication_year=2018&amp;pages=2126&amp;pmid=29844427&amp;doi=10.1038/s41467-018-04425-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref046">
<span class="label">46.</span><cite>Saccani S, Pantano S, Natoli G. Two waves of nuclear factor kappaB recruitment to target promoters. J Exp Med. 2001;193(12).</cite> [<a href="https://doi.org/10.1084/jem.193.12.1351" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2193297/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11413190/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Exp%20Med&amp;title=Two%20waves%20of%20nuclear%20factor%20kappaB%20recruitment%20to%20target%20promoters&amp;author=S%20Saccani&amp;author=S%20Pantano&amp;author=G%20Natoli&amp;volume=193&amp;issue=12&amp;publication_year=2001&amp;pmid=11413190&amp;doi=10.1084/jem.193.12.1351&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref047">
<span class="label">47.</span><cite>Kashyap T, Argueta C, Aboukameel A, Unger TJ, Klebanov B, Mohammad RM, et al. Selinexor, a selective inhibitor of nuclear export (SINE) compound, acts through NF-κB deactivation and combines with proteasome inhibitors to synergistically induce tumor cell death. Oncotarget. 2016;7(48):78883–95.
</cite> [<a href="https://doi.org/10.18632/oncotarget.12428" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5346685/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27713151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Selinexor,%20a%20selective%20inhibitor%20of%20nuclear%20export%20(SINE)%20compound,%20acts%20through%20NF-%CE%BAB%20deactivation%20and%20combines%20with%20proteasome%20inhibitors%20to%20synergistically%20induce%20tumor%20cell%20death&amp;author=T%20Kashyap&amp;author=C%20Argueta&amp;author=A%20Aboukameel&amp;author=TJ%20Unger&amp;author=B%20Klebanov&amp;volume=7&amp;issue=48&amp;publication_year=2016&amp;pages=78883-95&amp;pmid=27713151&amp;doi=10.18632/oncotarget.12428&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref048">
<span class="label">48.</span><cite>Newton PT, Vuppalapati KK, Bouderlique T, Chagin AS. Pharmacological inhibition of lysosomes activates the MTORC1 signaling pathway in chondrocytes in an autophagy-independent manner. Autophagy. 2015;11(9):1594–607. doi: 10.1080/15548627.2015.1068489

</cite> [<a href="https://doi.org/10.1080/15548627.2015.1068489" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4590675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26259639/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Autophagy&amp;title=Pharmacological%20inhibition%20of%20lysosomes%20activates%20the%20MTORC1%20signaling%20pathway%20in%20chondrocytes%20in%20an%20autophagy-independent%20manner&amp;author=PT%20Newton&amp;author=KK%20Vuppalapati&amp;author=T%20Bouderlique&amp;author=AS%20Chagin&amp;volume=11&amp;issue=9&amp;publication_year=2015&amp;pages=1594-607&amp;pmid=26259639&amp;doi=10.1080/15548627.2015.1068489&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref049">
<span class="label">49.</span><cite>Kolda TG, Bader BW. Tensor Decompositions and Applications. SIAM Rev. 2009;51(3):455–500. doi: 10.1137/07070111x</cite> [<a href="https://doi.org/10.1137/07070111x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=SIAM%20Rev&amp;title=Tensor%20Decompositions%20and%20Applications&amp;author=TG%20Kolda&amp;author=BW%20Bader&amp;volume=51&amp;issue=3&amp;publication_year=2009&amp;pages=455-500&amp;doi=10.1137/07070111x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref050">
<span class="label">50.</span><cite>Chin JL, Chan LC, Yeaman MR, Meyer AS. Tensor-based insights into systems immunity and infectious disease. Trends Immunol. 2023;44(5):329–32. doi: 10.1016/j.it.2023.03.003

</cite> [<a href="https://doi.org/10.1016/j.it.2023.03.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10411872/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36997459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Immunol&amp;title=Tensor-based%20insights%20into%20systems%20immunity%20and%20infectious%20disease&amp;author=JL%20Chin&amp;author=LC%20Chan&amp;author=MR%20Yeaman&amp;author=AS%20Meyer&amp;volume=44&amp;issue=5&amp;publication_year=2023&amp;pages=329-32&amp;pmid=36997459&amp;doi=10.1016/j.it.2023.03.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref051">
<span class="label">51.</span><cite>Cheong R, Rhee A, Wang CJ, Nemenman I, Levchenko A. Information transduction capacity of noisy biochemical signaling networks. Science. 2011;334(6054):354–8.
</cite> [<a href="https://doi.org/10.1126/science.1204553" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3895446/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21921160/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Information%20transduction%20capacity%20of%20noisy%20biochemical%20signaling%20networks&amp;author=R%20Cheong&amp;author=A%20Rhee&amp;author=CJ%20Wang&amp;author=I%20Nemenman&amp;author=A%20Levchenko&amp;volume=334&amp;issue=6054&amp;publication_year=2011&amp;pages=354-8&amp;pmid=21921160&amp;doi=10.1126/science.1204553&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref052">
<span class="label">52.</span><cite>Ramsay J. Functional Data Analysis. In: Encyclopedia of Statistics in Behavioral Science [Internet]. John Wiley &amp; Sons, Ltd; 2005.  [cited 2024 Sep 27]. Available from: <a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/0470013192.bsa239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://onlinelibrary.wiley.com/doi/abs/10.1002/0470013192.bsa239</a></cite> [<a href="https://scholar.google.com/scholar_lookup?Ramsay%20J.%20Functional%20Data%20Analysis.%20In:%20Encyclopedia%20of%20Statistics%20in%20Behavioral%20Science%20%5BInternet%5D.%20John%20Wiley%20&amp;%20Sons,%20Ltd;%202005.%20%5Bcited%202024%20Sep%2027%5D.%20Available%20from:%20https://onlinelibrary.wiley.com/doi/abs/10.1002/0470013192.bsa239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref053">
<span class="label">53.</span><cite>Gutenkunst RN, Waterfall JJ, Casey FP, Brown KS, Myers CR, Sethna JP. Universally Sloppy Parameter Sensitivities in Systems Biology Models. PLOS Comput Biol. 2007;3(10):e189. doi: 10.1371/journal.pcbi.0030189</cite> [<a href="https://doi.org/10.1371/journal.pcbi.0030189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2000971/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17922568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Comput%20Biol&amp;title=Universally%20Sloppy%20Parameter%20Sensitivities%20in%20Systems%20Biology%20Models&amp;author=RN%20Gutenkunst&amp;author=JJ%20Waterfall&amp;author=FP%20Casey&amp;author=KS%20Brown&amp;author=CR%20Myers&amp;volume=3&amp;issue=10&amp;publication_year=2007&amp;pmid=17922568&amp;doi=10.1371/journal.pcbi.0030189&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref054">
<span class="label">54.</span><cite>Kitano H. Biological robustness. Nat Rev Genet. 2004;5(11):826–37.
</cite> [<a href="https://doi.org/10.1038/nrg1471" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15520792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Genet&amp;title=Biological%20robustness&amp;author=H%20Kitano&amp;volume=5&amp;issue=11&amp;publication_year=2004&amp;pages=826-37&amp;pmid=15520792&amp;doi=10.1038/nrg1471&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref055">
<span class="label">55.</span><cite>Lee D, Jayaraman A, Kwon JS. Development of a hybrid model for a partially known intracellular signaling pathway through correction term estimation and neural network modeling. PLoS Comput Biol. 2020;16(12):e1008472. doi: 10.1371/journal.pcbi.1008472

</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1008472" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7769624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33315899/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=Development%20of%20a%20hybrid%20model%20for%20a%20partially%20known%20intracellular%20signaling%20pathway%20through%20correction%20term%20estimation%20and%20neural%20network%20modeling&amp;author=D%20Lee&amp;author=A%20Jayaraman&amp;author=JS%20Kwon&amp;volume=16&amp;issue=12&amp;publication_year=2020&amp;pmid=33315899&amp;doi=10.1371/journal.pcbi.1008472&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013344.ref056">
<span class="label">56.</span><cite>Altschuler SJ, Wu LF. Cellular heterogeneity: do differences make a difference?
Cell. 2010;141(4):559–63.
</cite> [<a href="https://doi.org/10.1016/j.cell.2010.04.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2918286/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20478246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Cellular%20heterogeneity:%20do%20differences%20make%20a%20difference?&amp;author=SJ%20Altschuler&amp;author=LF%20Wu&amp;volume=141&amp;issue=4&amp;publication_year=2010&amp;pages=559-63&amp;pmid=20478246&amp;doi=10.1016/j.cell.2010.04.033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 Table. Details the NFκB ODE model species and reactions, and which reactions are targeted by each of the study’s drugs.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s001.xlsx</a><sup> (32.7KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>S1 Fig. NFkB dynamic trajectories response to representative ligands and doses.</span><p>Trajectories of nuclear NFκB concentration (y-axis of each plot) over time (x-axis of each plot) for untreated conditions across 5 ligands and 3 doses. Rows indicate different ligands (labeled on the right of the panel), and columns specify ligand doses (labeled on the top of the panel). Stimulation indices are labeled in the top right corner.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s002.pdf</a><sup> (126.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>S2 Fig. Signaling dynamic patterns derived from signaling codons.</span><p><strong>(A)</strong> Illustration of six signaling dynamic codons: Duration (Dur), Oscillations (Osc), Peak Amplitude (Amp), Total Activity (AUC), Early vs. Late Activity (EvL), and Activation Speed (Spe). <strong>(B)</strong> Signaling dynamic codons for nuclear NFκB trajectories following pharmacological perturbation. For each drug panel, signaling dynamic codons are stratified by column (labeled on the top) and stimuli are stratified by row (labeled on the left). Each individual plot pertains to one signaling codon (y-axis) for a particular stimulus at 3 doses (low – triangle, medium – square, high – circle, labeled on the bottom left), ranging from untreated to drug dose 20 (log scale x-axis).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s003.pdf</a><sup> (1.3MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>S3 Fig. Construction of drug regime landscape from weights corresponding to seven components from the CP decomposition.</span><p><strong>(A)</strong> R2X plot depicting the percent variance explained after the application of CP decomposition for 1–8 components. <strong>(B)</strong> Heatmap of weights corresponding to seven temporal patterns using the drug treatment TSA at drug dose (DD) 2 as an example in defining the landscape for specific drug regime. Inside the heatmap, 15 rows depict 15 stimuli (5 ligands at 3 different doses). Colors within each row indicate the weights of the seven temporal patterns. These weights are the product of the component weights associated with the respective drug, drug dose, ligand, and ligand dose dimensions. The specific order of the 15 stimuli is outlined on the top of the figure, with L, M, and H denoting low-dose, medium-dose, and high-dose respectively. The weights of the temporal pattern (squares labeled by C1-C7 represent the weights for component 1–7 temporal patterns, respectively) stimulated by high-dose TNF under the TSA DD2 drug regime are specifically highlighted. <strong>(C)</strong> An example of the high-dose TNF stimulated NFκB trajectory perturbed by TSA at DD2 is displayed on the left. This can be approximated by the weighted sum (weights indicated by colors in the squares) of the seven temporal patterns (displayed adjacent to the colored squares). The weights are the product of the component weights corresponding to the specific drug, drug dose, ligand, and ligand dose dimensions obtained from CP decomposition. <strong>(D)</strong> The weights employed in the reconstruction of the simulated trajectory are the products of the weights in the dimensions of ligand, ligand dose, drug, and drug dose, with the yellow bars providing an example of calculating the weights for component 6 of (C). The subpanels and Fig 3E are the outcomes of Canonical Polyadic Decomposition (CPD) to nuclear NFκB time trajectory tensor, resulting in seven distinct components. These components are represented by their respective weights across various dimensions: time (Fig 3E), ligand, ligand doses, drugs, and drug dose index (this figure). <strong>(E)</strong> Temporal patterns of CP decomposition components under time point order alterations. We altered the time point order of the original tensor by splitting it into two parts along the time dimension, and then taking their reversed concatenation (i.e., the altered-time-order tensor had a time point order of (241:481, 1:240). We then applied CP decomposition using 7 components for this altered tensor.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s004.pdf</a><sup> (203.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>S4 Fig. Construction of drug regime landscape from weights corresponding to five components from the fPCA.</span><p><strong>(A)</strong> R2X plot depicting the percent variance explained after applying fPCA using 1–10 principal components. <strong>(B)</strong> Heatmap of weights corresponding to five temporal patterns for the drug treatment Sel at drug dose (DD) 15. The weights are the score outputs from the decomposition. The 15 heatmap rows correspond to the 15 stimuli (5 ligands at 3 different doses) with their order annotated at the top of the heatmap (L = low-dose, M = medium-dose, H = high-dose). The squares labeled C1-C5 represent the five temporal pattern (i.e., components 1–5) weights for NFκB activity under high-dose TNF stimulation and perturbed by the Sel DD15 drug regime. <strong>(C)</strong> An example of the high-dose TNF stimulated NFκB trajectory perturbed by Sel at DD15 is displayed on the left. This can be approximated by the weighted sum (weights indicated by colors in the squares) of the five temporal patterns (displayed following each colored square).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s005.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s005.pdf</a><sup> (138.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p">
<span>S5 Fig. Construction of stimulus cluster maps for selected drug regimes.</span><p><strong>(A)</strong> Trajectories of nuclear NFκB concentration (y-axis of each plot) over time (x-axis of each plot) for drug regimes CHL DD8, IKKi DD11, MG132 DD6, Sel DD15, TAK1i DD14 across 5 ligands and 3 doses. Solid colored lines represent the trajectories under drug treatment, while dashed lines depict untreated trajectories. Rows indicate different ligands (labeled on the right of each set of drug regime plots), and columns specify ligand doses (labeled on the top of each set of plots). Stimulation indices are labeled in the top right corner. <strong>(B)</strong> Expert classifications for 5 representative regimes: CHL DD8, IKKi DD11, MG132 DD6, Sel DD15, and TAK1i DD14 used in epsilon network clustering. Clusters for each regime are annotated according to the trajectory indices in (A). <strong>(C)</strong> Stimulus cluster maps constructed from epsilon network clustering results for 5 representative regimes. Each row and column within one map corresponds to a specific stimulus, as denoted in <a href="#pcbi.1013344.s007" class="usa-link">S6A Fig</a>. Within each map, off-diagonal black squares represent responsive clusters and red squares on the diagonal represent inhibited NFkB signaling (non-responder). Panels from left to right display the expert classifications, clusters derived from the trajectory space, signaling codon space, 7 component CPD feature space, and 40 component CPD feature space, and fPCA feature space (Labeled on the top of the panel).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s006.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s006.pdf</a><sup> (450.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p">
<span>S6 Fig. Decoding stimulus-response confusion, specificity, and inhibition scores for an example drug treatment.</span><p><strong>(A)</strong> Stimulus cluster maps for trajectories treated with MG132 at DD6. <strong>(B)</strong> Examples of decreased temporal coding capacity (high doses of TNF and Pam3CSK, top row) and increased temporal coding capacity (high doses of Pam3CSK and CpG, bottom row), represented in the trajectory space (left and middle columns) and signaling codon space (right column).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s007.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s007.pdf</a><sup> (174.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p">
<span>S7 Fig. Signaling codon-derived visualization of temporal coding capacity for 180 drug combination regimes.</span><p>Stimulus cluster map for the 15 stimulus conditions under 180 drug combination regimes. The 180 regimes are comprised of 45 combinations of the 10 drugs and 4 dose combinations using DD5 and DD10 (45 x 4 = 180). Columns represent untreated stimuli and rows represent treated stimuli. Black clusters on and off the diagonal represent “confusion” between stimulus conditions’ NFκB responses. Red squares on the diagonal represent inhibited NFκB signaling (non-responders).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s009.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s009.pdf</a><sup> (215.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="caption p">
<span>S8 Fig. Signaling codon-derived visualization of temporal coding capacity for 200 drug regimes vs. the 15 stimulated, untreated NFκB responses.</span><p>Stimulus cluster map for the 15 stimulus conditions under no drug treatment (untreated) vs. 15 stimulus conditions under <strong>(A)</strong> an example drug regime and (<strong>B)</strong> all 200 drug regimes. Columns represent untreated stimuli and rows represent treated stimuli. Black clusters on and off the diagonal represent “confusion” between treated and untreated NFκB responses under the same or different stimulus condition, respectively. Off-diagonal red squares represent inhibited NFκB signaling (non-responders). <strong>(C-D)</strong> Examples of confusion and distinction of the stimuli before and after drug treatment. Cross-stimulation confusion signaling codon Euclidean distances are annotated which are within the optimal ε boundary = 0.28.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370059/bin/pcbi.1013344.s008.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013344.s008.pdf</a><sup> (215.1KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data is deposited in Mendeley (<a href="https://data.mendeley.com/datasets/t3dbj86sjx/1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://data.mendeley.com/datasets/t3dbj86sjx/1</a>). All codes are available at GitHub (<a href="https://github.com/Xiaolu-Guo/Pharmacological_Perturbation_NFkB" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Xiaolu-Guo/Pharmacological_Perturbation_NFkB</a>).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS Computational Biology are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pcbi.1013344"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pcbi.1013344.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370059/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370059/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370059%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370059/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370059/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370059/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40839679/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370059/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40839679/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370059/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370059/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="qe6ie4QyNIXxuo0AaOHT9YkpxcPvaUqYhCIJpicKdiwdN7GesxxvMvKbaW4bEjq1">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
