
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Population Genetic Diversity and Species Distribution Evaluation of Bletilla striata (Orchidaceae) in Southwest China Using SSR Markers - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="1525492C8AF36AA305492C0006AFDFB8.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ece">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362183/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Ecology and Evolution">
<meta name="citation_title" content="Population Genetic Diversity and Species Distribution Evaluation of Bletilla striata (Orchidaceae) in Southwest China Using SSR Markers">
<meta name="citation_author" content="Liangliang Luo">
<meta name="citation_author_institution" content="Microbial Resources and Drug Development Key Laboratory of Guizhou Provincial Department of Education, School of Stomatology, Zunyi Medical University, Zunyi, China">
<meta name="citation_author_institution" content="School of Preclinical Medicine, Zunyi Medical University, Zunyi, China">
<meta name="citation_author" content="Qian Wang">
<meta name="citation_author_institution" content="Microbial Resources and Drug Development Key Laboratory of Guizhou Provincial Department of Education, School of Stomatology, Zunyi Medical University, Zunyi, China">
<meta name="citation_author" content="Xiaolan Li">
<meta name="citation_author_institution" content="Microbial Resources and Drug Development Key Laboratory of Guizhou Provincial Department of Education, School of Stomatology, Zunyi Medical University, Zunyi, China">
<meta name="citation_author" content="Delin Xu">
<meta name="citation_author_institution" content="School of Preclinical Medicine, Zunyi Medical University, Zunyi, China">
<meta name="citation_author" content="Huan Hu">
<meta name="citation_author_institution" content="Microbial Resources and Drug Development Key Laboratory of Guizhou Provincial Department of Education, School of Stomatology, Zunyi Medical University, Zunyi, China">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="15">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e72043">
<meta name="citation_doi" content="10.1002/ece3.72043">
<meta name="citation_pmid" content="40837537">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362183/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362183/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362183/pdf/ECE3-15-e72043.pdf">
<meta name="description" content="We assessed the genetic diversity and population structure of the protected orchid Bletilla striata across 18 wild populations in southwestern China. Eight pairs of simple sequence repeat (SSR) molecular markers were employed for its genetic ...">
<meta name="og:title" content="Population Genetic Diversity and Species Distribution Evaluation of Bletilla striata (Orchidaceae) in Southwest China Using SSR Markers">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="We assessed the genetic diversity and population structure of the protected orchid Bletilla striata across 18 wild populations in southwestern China. Eight pairs of simple sequence repeat (SSR) molecular markers were employed for its genetic ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362183/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12362183">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/ece3.72043"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ECE3-15-e72043.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12362183%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12362183/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12362183/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362183/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ece.gif" alt="Ecology and Evolution logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Ecology and Evolution" title="Link to Ecology and Evolution" shape="default" href="http://onlinelibrary.wiley.com/journal/10.1002/(ISSN)2045-7758" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Ecol Evol</button></div>. 2025 Aug 19;15(8):e72043. doi: <a href="https://doi.org/10.1002/ece3.72043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/ece3.72043</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Ecol%20Evol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ecol%20Evol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Ecol%20Evol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Ecol%20Evol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Population Genetic Diversity and Species Distribution Evaluation of <em>Bletilla striata</em> (Orchidaceae) in Southwest China Using SSR Markers</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luo%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Liangliang Luo</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Liangliang Luo</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Microbial Resources and Drug Development Key Laboratory of Guizhou Provincial Department of Education, School of Stomatology, Zunyi Medical University, Zunyi, China</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>School of Preclinical Medicine, Zunyi Medical University, Zunyi, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luo%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Liangliang Luo</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Qian Wang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Qian Wang</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Microbial Resources and Drug Development Key Laboratory of Guizhou Provincial Department of Education, School of Stomatology, Zunyi Medical University, Zunyi, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qian Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Xiaolan Li</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Xiaolan Li</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Microbial Resources and Drug Development Key Laboratory of Guizhou Provincial Department of Education, School of Stomatology, Zunyi Medical University, Zunyi, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaolan Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Delin Xu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Delin Xu</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>School of Preclinical Medicine, Zunyi Medical University, Zunyi, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xu%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Delin Xu</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Huan Hu</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Huan Hu</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Microbial Resources and Drug Development Key Laboratory of Guizhou Provincial Department of Education, School of Stomatology, Zunyi Medical University, Zunyi, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hu%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Huan Hu</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="ece372043-aff-0001">
<sup>
<sup>1</sup>
</sup>Microbial Resources and Drug Development Key Laboratory of Guizhou Provincial Department of Education, School of Stomatology, Zunyi Medical University, Zunyi, China</div>
<div id="ece372043-aff-0002">
<sup>
<sup>2</sup>
</sup>School of Preclinical Medicine, Zunyi Medical University, Zunyi, China</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong>
 
Qian Wang (<span>qianwang07@126.com</span>), 
Huan Hu (<span>huhuan1990@163.com</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Aug 1; Received 2025 Apr 5; Accepted 2025 Aug 12; Collection date 2025 Aug.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Ecology and Evolution</em> published by British Ecological Society and John Wiley &amp; Sons Ltd.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12362183  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40837537/" class="usa-link">40837537</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>We assessed the genetic diversity and population structure of the protected orchid 
<em>Bletilla striata</em>
 across 18 wild populations in southwestern China. Eight pairs of simple sequence repeat (SSR) molecular markers were employed for its genetic diversity and population structure analyses, while the optimized Maxent model was utilized to predict changes in the habitat distribution under historical conditions and three future climate scenarios (SSP126, SSP245, and SSP585) with 141 natural distribution data and 19 climatic factors. The results revealed an average number of alleles (<em>Na</em>) of 3.549 and an effective number of alleles (<em>Ne</em>) of 2.636, with a mean polymorphic information content (<em>PIC</em>) of 0.748 across the 
<em>B. striata</em>
 populations. Moderate genetic diversity was observed (observed heterozygosity, <em>Ho</em> = 0.402; expected heterozygosity, <em>He</em> = 0.509), with 73% of the total variation found within populations, while the 02 population in Zhijin Guizhou exhibited relatively high genetic differentiation (<em>Ho</em> = 0.675, <em>He</em> = 0.658). UPGMA clustering, population structure analyses, and principal component analysis identified two primary subgroups within 
<em>B. striata</em>
. Among the 19 climate variables analyzed, four temperature‐related factors and two precipitation‐related factors were identified as key drivers influencing the geographical distribution of 
<em>B. striata</em>
. Future projections for the 2050s and 2070s under varying climate scenarios indicate a northward expansion of suitable habitats for 
<em>B. striata</em>
. The proportion of suitable habitat area is expected to increase from 288.3450 × 10<sup>4</sup> km<sup>2</sup> under historical conditions (1970–2000) to 351.9792–405.6077 × 10<sup>4</sup> km<sup>2</sup> (2050s–2070s). The wild 
<em>B. striata</em>
 populations in southwestern China and adjacent regions represent valuable germplasm resources with high genetic diversity, offering significant potential for artificial cultivation initiatives. Moreover, predictions of future distribution dynamics provide critical insights to guide the conservation, development, and sustainable utilization of 
<em>B. striata</em>
.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> 
<em>Bletilla striata</em>
, Maxent model, population genetic structure, potentially suitable habitats, simple sequence repeat (SSR)</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p>These findings provide a scientific basis for conserving and sustainably utilizing 
<em>Bletilla striata</em>
 germplasm. By predicting future distribution shifts under climate change, this research offers critical insights for dynamic monitoring and conservation strategy development, holding significant theoretical and practical value for biodiversity conservation.</p>
<section class="bt xbox font-sm" id="ece372043-blkfxd-0001"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c435/12362183/853d377e515b/ECE3-15-e72043-g001.jpg" loading="lazy" id="jats-graphic-1" height="238" width="331" alt="graphic file with name ECE3-15-e72043-g001.jpg"></p></section></section><section id="ece372043-sec-0001"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>The genus <em>Bletilla</em> Rchb. f. (Orchidaceae) comprises approximately six species, four of which are native to China, predominantly distributed across southern provinces (Chen et al. <a href="#ece372043-bib-0004" class="usa-link" aria-describedby="ece372043-bib-0004">2009</a>). Among these, 
<em>Bletilla striata</em>
 (Thunb. ex A. Murray) Rchb. F. occupies a prominent role as a traditional medical plant with over 1500 years of documented use in Chinese medicine. It is highly regarded for its pharmacological properties, including hemostatic, anti‐inflammatory, analgesic, and wound‐healing effects (Gou et al. <a href="#ece372043-bib-0010" class="usa-link" aria-describedby="ece372043-bib-0010">2022</a>; He et al. <a href="#ece372043-bib-0018" class="usa-link" aria-describedby="ece372043-bib-0018">2024</a>). Despite its significance, the propagation of 
<em>B. striata</em>
 under natural conditions faces considerable challenges due to its tiny seeds, which lack endosperm and hinder direct germination in the wild. This constraint has limited the reproductive success and scale of natural populations. Moreover, the high medicinal value of 
<em>B. striata</em>
 has driven extensive and destructive harvesting of wild resources, resulting in a growing disparity between market demand and supply (Li et al. <a href="#ece372043-bib-0025" class="usa-link" aria-describedby="ece372043-bib-0025">2024</a>) (Figure <a href="#ece372043-fig-0001" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="ece372043-fig-0001"><h3 class="obj_head">FIGURE 1.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c435/12362183/fb7bd8ee7de8/ECE3-15-e72043-g005.jpg" loading="lazy" id="jats-graphic-3" height="475" width="661" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/ece372043-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Represent photo of 
<em>Bletilla striata</em>
. Photographed by Huan Hu.</p></figcaption></figure><p>Large‐scale cultivation can both meet medicinal demand through standardized production and reduce pressure on wild populations. Population genetic studies provide a valuable approach to assessing the genetic diversity of wild 
<em>B. striata</em>
 germplasm and evaluating genetic variation among populations. Such research establishes a scientific basis for resource conservation and germplasm identification and facilitates the development of effective strategies for selecting superior germplasm for agricultural production. These efforts will collectively contribute to the sustainable development of the 
<em>B. striata</em>
 industry.</p>
<p>The application of DNA molecular markers, such as Random Amplified Polymorphic DNA (RAPD), Restriction Fragment Length Polymorphism (RFLP), Amplification Refractory Mutation System (ARMS), Cleaved Amplified Polymorphic Sequence (CAPS), Amplified Fragment Length Polymorphism (AFLP), DNA Amplification Fingerprinting (DAF), Inter‐Simple Sequence Repeat (ISSR), and Simple Sequence Repeat (SSR), offers a cost‐effective and reliable approach for uncovering the mechanisms underlying genetic diversity and evolutionary dynamics of flora and fauna populations. SSR molecular markers, also referred to as microsatellite DNA or SSRs, are based on sequence length polymorphisms (Jin et al. <a href="#ece372043-bib-0023" class="usa-link" aria-describedby="ece372043-bib-0023">2024</a>). These markers consist of DNA fragments formed by one or more tandemly repeated nucleotide sequences and, along with single nucleotide polymorphism (SNP) markers, are widely used to analyze population genetic structure and phylogenetic relationships (Tautz and Renz <a href="#ece372043-bib-0053" class="usa-link" aria-describedby="ece372043-bib-0053">1984</a>). With their high polymorphism, stability, and codominance, SSR markers have been extensively applied in studies of genetic diversity and population structure in resource plants such as 
<em>Passiflora edulis</em>
 Sims (Wu, Cai, et al. <a href="#ece372043-bib-0064" class="usa-link" aria-describedby="ece372043-bib-0064">2024a</a>), <em>Habenaria</em> Willd. (Zhang et al. <a href="#ece372043-bib-0070" class="usa-link" aria-describedby="ece372043-bib-0070">2024</a>), <em>Orchidantha chinensis</em> T. L. Wu (Zhou et al. <a href="#ece372043-bib-0075" class="usa-link" aria-describedby="ece372043-bib-0075">2024</a>), and <em>Meconopsis integrifolia</em> (Maxim.) Franch. (Wu, Yang, et al. <a href="#ece372043-bib-0065" class="usa-link" aria-describedby="ece372043-bib-0065">2024b</a>). However, research on the genetic structure of wild 
<em>B. striata</em>
 populations using SSR markers remains largely unexplored. Our previous work has established a foundation by conducting RNA sequencing of 
<em>B. striata</em>
 tissues and organs at different development stages (Xu et al. <a href="#ece372043-bib-0066" class="usa-link" aria-describedby="ece372043-bib-0066">2018</a>). From this, SSR primers were developed and validated, demonstrating their utility for phylogenetic research and breeding programs.</p>
<p>Moreover, climate change driven by increasing greenhouse gas emissions, coupled with the frequent occurrence of extreme weather events, significantly impacts the potential distribution, spatial patterns, and sustainability of global ecosystems (Zhang et al. <a href="#ece372043-bib-0071" class="usa-link" aria-describedby="ece372043-bib-0071">2021</a>; Habibullah et al. <a href="#ece372043-bib-0016" class="usa-link" aria-describedby="ece372043-bib-0016">2022</a>). The species distribution model (SDM) has become a widely used mathematical approach for predicting species distribution under changing climate conditions (Wan et al. <a href="#ece372043-bib-0057" class="usa-link" aria-describedby="ece372043-bib-0057">2024</a>). Among these, the Maxent model, proposed by Jaynes in 1957 based on entropy theory (Phillips et al. <a href="#ece372043-bib-0038" class="usa-link" aria-describedby="ece372043-bib-0038">2006</a>), is a machine learning algorithm that offers high flexibility in processing various data types. Its predictive accuracy remains stable and reliable even when data are partially missing or sample sizes are small, making it extensively applied in ecological studies (Remya et al. <a href="#ece372043-bib-0045" class="usa-link" aria-describedby="ece372043-bib-0045">2015</a>; Li et al. <a href="#ece372043-bib-0024" class="usa-link" aria-describedby="ece372043-bib-0024">2020</a>).</p>
<p>This study integrates SSR markers and SDMs to systematically reveal the genetic structure and climate adaptability of 
<em>Bletilla striata</em>
 in Southwest China, providing a dual “genetic‐ecological” basis for conservation and sustainable utilization of this endangered orchid. The primary purpose of the present study was to (1) explore the population diversification and structure throughout southwestern China and its adjacent areas, (2) project the potential suitable habitats and centroid migration of 
<em>B. striata</em>
 under different future climate scenarios, (3) uncover the patterns of suitable habitat redistribution in 
<em>B. striata</em>
 populations in response to climate changes.</p></section><section id="ece372043-sec-0002"><h2 class="pmc_sec_title">2. Materials and Methods</h2>
<section id="ece372043-sec-0003"><h3 class="pmc_sec_title">2.1. Wild 
<em>B. striata</em>
 Samples</h3>
<section id="ece372043-sec-0004"><h4 class="pmc_sec_title">2.1.1. Plant Material and DNA Extraction</h4>
<p>In this study, 116 samples from 18 populations of 
<em>B. striata</em>
 were gathered from the southwest mountainous region of China and its adjacent areas, which represent the natural distribution center of 
<em>B. striata</em>
 (Table <a href="#ece372043-tbl-0001" class="usa-link">1</a>). The field sample collection and research procedures were approved by the Ethical Experimentation Committee of Zunyi Medical University (Identification Code: ZMU‐BO‐2003‐021) and adhered to the legal and ethical criteria set by the local government. The original specimens were identified as 
<em>B. striata</em>
 by Professor Delin Xu of Zunyi Medical University, and a voucher specimen was deposited in the Herbarium of Life Science Museum at Zunyi Medical University. Fresh young leaves were collected and dried using silica gel. The entire DNA was extracted using the Tiangen Plant Genome DNA Extraction Kit, electrophoresed on a 2% agarose gel, and stored at −20°C for further use.</p>
<section class="tw xbox font-sm" id="ece372043-tbl-0001"><h5 class="obj_head">TABLE 1.</h5>
<div class="caption p"><p>The sampling sites and genetic diversity information based on eight pairs of simple sequence repeats (SSR) for 
<em>Bletilla striata</em>
 populations collected in this study.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Sample size (<em>N</em>)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Population code</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Collection locations</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Longitude(E)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Latitude(N)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>Na</em>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>Ne</em>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>I</em>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>Ho</em>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>He</em>
</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">8</td>
<td align="center" valign="top" rowspan="1" colspan="1">01</td>
<td align="center" valign="top" rowspan="1" colspan="1">Gouba Village, Zunyi, Guizhou</td>
<td align="center" valign="top" rowspan="1" colspan="1">106.5643</td>
<td align="center" valign="top" rowspan="1" colspan="1">27.6516</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.125</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.209</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.842</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.313</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.479</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">5</td>
<td align="center" valign="top" rowspan="1" colspan="1">02</td>
<td align="center" valign="top" rowspan="1" colspan="1">Zhongzhai Town, Zijin, Guizhou</td>
<td align="center" valign="top" rowspan="1" colspan="1">105.5964</td>
<td align="center" valign="top" rowspan="1" colspan="1">26.6949</td>
<td align="center" valign="top" rowspan="1" colspan="1">4.000</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.378</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.226</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.675</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.658</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">8</td>
<td align="center" valign="top" rowspan="1" colspan="1">03</td>
<td align="center" valign="top" rowspan="1" colspan="1">Xifeng Town, Guizhou Province</td>
<td align="center" valign="top" rowspan="1" colspan="1">106.596</td>
<td align="center" valign="top" rowspan="1" colspan="1">27.246</td>
<td align="center" valign="top" rowspan="1" colspan="1">4.000</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.810</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.072</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.438</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.576</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">5</td>
<td align="center" valign="top" rowspan="1" colspan="1">04</td>
<td align="center" valign="top" rowspan="1" colspan="1">Luokuangyan, Wushan, Chongqing</td>
<td align="center" valign="top" rowspan="1" colspan="1">110.0506</td>
<td align="center" valign="top" rowspan="1" colspan="1">31.1527</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.250</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.676</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.953</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.475</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.543</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">8</td>
<td align="center" valign="top" rowspan="1" colspan="1">05</td>
<td align="center" valign="top" rowspan="1" colspan="1">E'meishan Biological Station, Sichuan</td>
<td align="center" valign="top" rowspan="1" colspan="1">103.3797</td>
<td align="center" valign="top" rowspan="1" colspan="1">29.5922</td>
<td align="center" valign="top" rowspan="1" colspan="1">4.500</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.850</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.075</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.422</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.535</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">4</td>
<td align="center" valign="top" rowspan="1" colspan="1">06</td>
<td align="center" valign="top" rowspan="1" colspan="1">Xicao Village, Wenchuan, Sichuan</td>
<td align="center" valign="top" rowspan="1" colspan="1">103.283</td>
<td align="center" valign="top" rowspan="1" colspan="1">30.8899</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.750</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.256</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.733</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.250</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.406</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2</td>
<td align="center" valign="top" rowspan="1" colspan="1">07</td>
<td align="center" valign="top" rowspan="1" colspan="1">Qingcheng Mountain, Dujiangyan, Sichuan</td>
<td align="center" valign="top" rowspan="1" colspan="1">103.5548</td>
<td align="center" valign="top" rowspan="1" colspan="1">30.9059</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.250</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.008</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.671</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.438</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.422</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">4</td>
<td align="center" valign="top" rowspan="1" colspan="1">08</td>
<td align="center" valign="top" rowspan="1" colspan="1">Longzhu Village, Wenchuan, Sichuan</td>
<td align="center" valign="top" rowspan="1" colspan="1">103.3473</td>
<td align="center" valign="top" rowspan="1" colspan="1">30.8985</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.125</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.506</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.926</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.375</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.520</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">8</td>
<td align="center" valign="top" rowspan="1" colspan="1">09</td>
<td align="center" valign="top" rowspan="1" colspan="1">Huoshiba Village, Zheng'an, Guizhou</td>
<td align="center" valign="top" rowspan="1" colspan="1">107.3578</td>
<td align="center" valign="top" rowspan="1" colspan="1">28.8309</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.500</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.620</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.912</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.172</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.484</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">8</td>
<td align="center" valign="top" rowspan="1" colspan="1">10</td>
<td align="center" valign="top" rowspan="1" colspan="1">Yilong, Nanchong, Sichuan</td>
<td align="center" valign="top" rowspan="1" colspan="1">106.8067</td>
<td align="center" valign="top" rowspan="1" colspan="1">31.4369</td>
<td align="center" valign="top" rowspan="1" colspan="1">4.125</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.506</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.912</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.266</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.461</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">8</td>
<td align="center" valign="top" rowspan="1" colspan="1">11</td>
<td align="center" valign="top" rowspan="1" colspan="1">Wawushan Town, Hongya, Sichuan</td>
<td align="center" valign="top" rowspan="1" colspan="1">102.9986</td>
<td align="center" valign="top" rowspan="1" colspan="1">29.6742</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.500</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.400</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.878</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.156</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.459</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">8</td>
<td align="center" valign="top" rowspan="1" colspan="1">12</td>
<td align="center" valign="top" rowspan="1" colspan="1">Laojun Mountain, Yibin, Sichuan</td>
<td align="center" valign="top" rowspan="1" colspan="1">103.986</td>
<td align="center" valign="top" rowspan="1" colspan="1">28.6916</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.750</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.214</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.965</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.656</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.516</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">8</td>
<td align="center" valign="top" rowspan="1" colspan="1">13</td>
<td align="center" valign="top" rowspan="1" colspan="1">Houhe Tianmen Gorge, Yichang, Hubei</td>
<td align="center" valign="top" rowspan="1" colspan="1">110.6125</td>
<td align="center" valign="top" rowspan="1" colspan="1">30.0827</td>
<td align="center" valign="top" rowspan="1" colspan="1">5.250</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.395</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.298</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.578</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.639</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">6</td>
<td align="center" valign="top" rowspan="1" colspan="1">14</td>
<td align="center" valign="top" rowspan="1" colspan="1">Xintang Town, Enshi, Hubei</td>
<td align="center" valign="top" rowspan="1" colspan="1">109.8738</td>
<td align="center" valign="top" rowspan="1" colspan="1">30.1961</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.875</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.775</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.094</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.479</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.590</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">8</td>
<td align="center" valign="top" rowspan="1" colspan="1">15</td>
<td align="center" valign="top" rowspan="1" colspan="1">Qingliangfeng, Lin'an, Zhejiang</td>
<td align="center" valign="top" rowspan="1" colspan="1">118.959</td>
<td align="center" valign="top" rowspan="1" colspan="1">30.0748</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.750</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.986</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.991</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.359</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.503</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">8</td>
<td align="center" valign="top" rowspan="1" colspan="1">16</td>
<td align="center" valign="top" rowspan="1" colspan="1">Dazhai Village, Zijin, Guizhou</td>
<td align="center" valign="top" rowspan="1" colspan="1">105.6583</td>
<td align="center" valign="top" rowspan="1" colspan="1">26.4541</td>
<td align="center" valign="top" rowspan="1" colspan="1">4.875</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.327</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.292</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.469</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.653</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">4</td>
<td align="center" valign="top" rowspan="1" colspan="1">17</td>
<td align="center" valign="top" rowspan="1" colspan="1">Lütang Town, Dafang, Guizhou</td>
<td align="center" valign="top" rowspan="1" colspan="1">105.3978</td>
<td align="center" valign="top" rowspan="1" colspan="1">27.059</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.875</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.652</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.439</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.375</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.273</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">6</td>
<td align="center" valign="top" rowspan="1" colspan="1">18</td>
<td align="center" valign="top" rowspan="1" colspan="1">Langya Mountain, Chuzhou, Anhui</td>
<td align="center" valign="top" rowspan="1" colspan="1">118.2804</td>
<td align="center" valign="top" rowspan="1" colspan="1">32.2834</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.375</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.874</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.695</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.333</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.438</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="center" colspan="2" valign="top" rowspan="1">Mean</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1">3.549</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.636</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.943</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.402</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.509</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ece372043-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="ece372043-note-0002"><p>
<em>Note: Na</em>: number of alleles, <em>Ne</em>: number of effective alleles, <em>I</em>: Shannon diversity index, <em>Ho</em>: observed heterozygosity, <em>He</em>: expected heterozygosity.</p></div></div></section></section><section id="ece372043-sec-0005"><h4 class="pmc_sec_title">2.1.2. 
SSR‐PCR Amplification</h4>
<p>Based on previous studies, eight pairs of SSR primers were selected and synthesized by Biotech Bioengineering (Shanghai) Co. The PCR reaction system consisted of 1 μL of template DNA, 1 μL each of forward and reverse primers (10 μmol/L), 12.5 μL of 2 × SanTaq PCR Mix (including Mgcl<sub>2</sub>, dNTP, Taq DNA Polymerase, PCR buffer, loading buffer, and PCR enhancer), and 9.5 μL of ddH<sub>2</sub>O. PCR amplification began with 94°C for 3 min, followed by 35 cycles of 94°C for 30 s, annealing at 57°C–58.7°C for 30 s, extension at 72°C for 30 s, and a final extension at 72°C for 7 min (with specific primer information and annealing temperatures provided in Table <a href="#ece372043-supitem-0001" class="usa-link">S1</a>). The amplified products were subsequently sent to Biotech Bioengineering (Shanghai) Co. for capillary electrophoresis gene detection using the 3730XL platform.</p></section><section id="ece372043-sec-0006"><h4 class="pmc_sec_title">2.1.3. Population Genetic Analysis</h4>
<p>After analyzing the SSR results using Excel software, GenAlEx 6.503 was employed for AMOVA analysis, principal component analysis (PCoA), and the calculation of genetic diversity parameters. These parameters included the average number of alleles (<em>Na</em>), effective alleles (<em>Ne</em>), observed heterozygosity (<em>Ho</em>), expected heterozygosity (<em>He</em>), Shannon's information index (<em>I</em>), Nei's genetic identity (<em>GI</em>) and distance (<em>GD</em>), inbreeding coefficient (<em>F</em>is), and genetic differentiation coefficient (<em>F</em>st). Gene flow (Nm) was estimated using Wright's formula (Nm = 0.25 (1‐<em>F</em>st)/<em>F</em>st) (Wright <a href="#ece372043-bib-0063" class="usa-link" aria-describedby="ece372043-bib-0063">1931</a>; Rousset <a href="#ece372043-bib-0046" class="usa-link" aria-describedby="ece372043-bib-0046">1997</a>; Peakall and Smouse <a href="#ece372043-bib-0035" class="usa-link" aria-describedby="ece372043-bib-0035">2012</a>). PowerMarker v3.25 was utilized to calculate polymorphic information content (<em>PIC</em>) and conduct UPGMA cluster analysis (Liu and Muse <a href="#ece372043-bib-0029" class="usa-link" aria-describedby="ece372043-bib-0029">2005</a>). Additionally, Structure v2.3.4 was used to investigate the population structure of 116 
<em>B. striata</em>
 samples from 18 populations (Pritchard et al. <a href="#ece372043-bib-0041" class="usa-link" aria-describedby="ece372043-bib-0041">2000</a>) and genetic similarity weight value (Q). The parameters were set as follows: true number of populations (<em>K</em>) ranged from 1 to 10, with 20 replicates for each <em>K</em>, a length of burn‐in period = 500 000, number of MCMC repeats after burn‐in = 1 500 000. StructureSelector (<a href="https://lmme.ac.cn/StructureSelector/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://lmme.ac.cn/StructureSelector/</a>) was employed to visualize the optimal segmentation <em>K</em> value and the population genetic structure using Evanno's ΔK method (Li and Liu <a href="#ece372043-bib-0026" class="usa-link" aria-describedby="ece372043-bib-0026">2018</a>).</p></section></section><section id="ece372043-sec-0007"><h3 class="pmc_sec_title">2.2. Ecological Niche Modeling</h3>
<section id="ece372043-sec-0008"><h4 class="pmc_sec_title">2.2.1. 

<em>B. striata</em>
 Distribution Data and Climate Variables Selecting</h4>
<p>Based on the coordinate information of 18 wild populations collected during field surveys conducted from 2021 to 2022 (Table <a href="#ece372043-tbl-0001" class="usa-link">1</a>), along with data from the Global Biodiversity Information Facility (GBIF, <a href="http://www.gbif.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.gbif.org</a>), the Chinese Virtual Herbarium (CVH, <a href="http://www.cvh.org.cn/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.cvh.org.cn/</a>), and published literature (Deng et al. <a href="#ece372043-bib-0006" class="usa-link" aria-describedby="ece372043-bib-0006">2019</a>; Ling et al. <a href="#ece372043-bib-0027" class="usa-link" aria-describedby="ece372043-bib-0027">2019</a>; Su et al. <a href="#ece372043-bib-0049" class="usa-link" aria-describedby="ece372043-bib-0049">2014</a>; Sun <a href="#ece372043-bib-0050" class="usa-link" aria-describedby="ece372043-bib-0050">2015</a>; Sun et al. <a href="#ece372043-bib-0051" class="usa-link" aria-describedby="ece372043-bib-0051">2016</a>; Qin et al. <a href="#ece372043-bib-0042" class="usa-link" aria-describedby="ece372043-bib-0042">2020</a>; Zhou et al. <a href="#ece372043-bib-0073" class="usa-link" aria-describedby="ece372043-bib-0073">2010</a>, <a href="#ece372043-bib-0074" class="usa-link" aria-describedby="ece372043-bib-0074">2022</a>; Jian et al. <a href="#ece372043-bib-0021" class="usa-link" aria-describedby="ece372043-bib-0021">2019</a>), a preliminary dataset of 185 geographical distribution records of 
<em>B. striata</em>
 in China was obtained. To reduce bias and prevent overfitting (Bo et al. <a href="#ece372043-bib-0001" class="usa-link" aria-describedby="ece372043-bib-0001">2022</a>), the “Trim duplicate occurrences” function in ENM Tools was employed to remove redundant data, resulting in a final dataset of 141 distribution records of <em>B. straria</em> (Figure <a href="#ece372043-fig-0002" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="ece372043-fig-0002"><h5 class="obj_head">FIGURE 2.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c435/12362183/3fdf03802244/ECE3-15-e72043-g002.jpg" loading="lazy" id="jats-graphic-5" height="506" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/ece372043-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The geographical distribution of 141 recorded occurrence sites of 
<em>Bletilla striata</em>
. The occurrence points are presented in black points.</p></figcaption></figure><p>The map of China was sourced from the National Fundamental Geographic Information System (<a href="https://nfgis.nsdi.gov.cn" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://nfgis.nsdi.gov.cn</a>). Climate data for historical (1970–2000), future 2050s (2041–2060), and future 2070s (2061–2080) were obtained from WorldClim (<a href="https://worldclim.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://worldclim.org/</a>), which includes 19 climate variables. For future scenarios, the BCC‐CSM2‐MR (Beijing Climate Center Climate System Model) climate model from the Coupled Model Intercomparison Project Phase 6 (CMIP6) was selected due to its reliable simulations for China (Liu, Fang, et al. <a href="#ece372043-bib-0030" class="usa-link" aria-describedby="ece372043-bib-0030">2023b</a>). Historical climate data were provided at a spatial resolution of 30′, and future climate data for the 2050s and 2070s were modeled at a spatial resolution of 2.5′ under three Shared Socioeconomic Pathways (SSPs): SSP126 (low greenhouse gas emission scenario with radiative forcing of 2.6 W/m<sup>2</sup> by the year 2100), SSP245 (medium pathway of greenhouse gas emission with 4.5 W/m<sup>2</sup> in 2100), and SSP585 (high greenhouse gas emission scenario with 8.5 W/m<sup>2</sup> in 2100).</p>
<p>SPSS Statistics v27 was employed to conduct a bivariate Pearson correlation analysis on 19 climate variables associated with the distribution records. When the correlation coefficient |<em>r</em>| ≥ 0.8 between two environmental variables, the one with the higher contribution rate was retained for Maxent model construction (Guo, Cao, Zhang, <a href="#ece372043-bib-0013" class="usa-link" aria-describedby="ece372043-bib-0013">2019a</a>) (Figure <a href="#ece372043-supitem-0001" class="usa-link">S1</a>). As a result, six bioclimatic variables (BIO2, BIO4, BIO6, BIO9, BIO12, and BIO17) were selected for further use in Maxent modeling (<a href="http://biodiversityinformatics.amnh.org/open_source/maxent/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://biodiversityinformatics.amnh.org/open_source/maxent/</a>; Table <a href="#ece372043-supitem-0001" class="usa-link">S2</a>).</p></section><section id="ece372043-sec-0009"><h4 class="pmc_sec_title">2.2.2. 
MaxEnt Model and Suitable Habitat Pattern</h4>
<p>Optimizing the feature combination (FC) and regularization multiplier (RM) parameters in the Maxent model can significantly improve the accuracy of predictions (Phillips and Dudík <a href="#ece372043-bib-0039" class="usa-link" aria-describedby="ece372043-bib-0039">2008</a>; Radosavljevic and Anderson <a href="#ece372043-bib-0044" class="usa-link" aria-describedby="ece372043-bib-0044">2014</a>). The R package Kuenm (Cobos et al. <a href="#ece372043-bib-0005" class="usa-link" aria-describedby="ece372043-bib-0005">2019</a>) was employed to explore a total of 248 parameter combinations, consisting of 31 different FC combinations derived from five feature classes (Linear, Quadratic, Product, Threshold, and Hinge), with RM values ranging from 0 to 4 in increments of 0.5. The optimal model was identified when FC = L and RM = 0.5, as this configuration met the model selection criteria, including a data omission rate of less than 5%, statistical significance, and the lowest delta AICc value.</p>
<p>Climate variables for different periods were imported into the Maxent model, with 25% of the distribution data randomly selected for model testing and 75% used for model training. To minimize the effects of random deviations, the bootstrap method with 10 replicates was employed. The “Create response curves” and “Do jackknife” options were activated to assess the relative importance of the climatic variables (Wang et al. <a href="#ece372043-bib-0059" class="usa-link" aria-describedby="ece372043-bib-0059">2024</a>). The predictive accuracy of the model was evaluated using the area under curve (AUC) of the receiver operating curve (ROC) (Peterson et al. <a href="#ece372043-bib-0036" class="usa-link" aria-describedby="ece372043-bib-0036">2007</a>). An AUC value close to 1 indicates high predictive accuracy, while values exceeding 0.7 are considered to reflect reliable results (Wang et al. <a href="#ece372043-bib-0061" class="usa-link" aria-describedby="ece372043-bib-0061">2023</a>).</p>
<p>The predicted distribution map of 
<em>B. striata</em>
 with the highest AUC value was imported into ArcGIS 10.8 alongside a base map and converted to raster data using ArcToolbox. The reclassify tool was then applied to classify the habitat suitability levels of 
<em>B. striata</em>
. Subsequently, the Mask tool in ArcGIS 10.8 was used to extract the potential geographic distribution map of 
<em>B. striata</em>
 within China. Based on the natural discontinuity point method, the potential suitable areas were categorized into four levels: unsuitable (0–0.12), low‐suitability (0.12–0.25), moderate‐suitability (0.25–0.5), and high‐suitability (0.5–1).</p></section></section></section><section id="ece372043-sec-0010"><h2 class="pmc_sec_title">3. Results</h2>
<section id="ece372043-sec-0011"><h3 class="pmc_sec_title">3.1. Genetic Diversity and Structure of 
<em>B. striata</em>
 in Southwest China and Surrounding Regions</h3>
<section id="ece372043-sec-0012"><h4 class="pmc_sec_title">3.1.1. Genetic Diversity</h4>
<p>A total of 511 alleles were identified using eight SSR primer pairs across 18 populations of 
<em>B. striata</em>
 comprising 116 samples from southwest China and surrounding regions. The <em>Na</em> varied from 1 to 12, with a mean of 3.549. The <em>PIC</em> values ranged from 0.535 to 0.910, with an average of 0.748 (Table <a href="#ece372043-supitem-0001" class="usa-link">S3</a>).</p>
<p>At the population level, genetic diversity indices, including <em>Na</em>, <em>Ne</em>, <em>I</em>, <em>Ho</em>, and <em>He</em>, ranged from 1.875 to 5.250, 1.652 to 3.395, 0.439 to 1.298, 0.156 to 0.675, and 0.273 to 0.658, respectively. Among the 18 populations, population 11 exhibited the lowest genetic diversity, whereas population 02 displayed the highest genetic diversity (Table <a href="#ece372043-tbl-0001" class="usa-link">1</a>).</p></section><section id="ece372043-sec-0013"><h4 class="pmc_sec_title">3.1.2. Genetic Differentiation</h4>
<p>The <em>Fst</em> values among 
<em>B. striata</em>
 populations ranged from 0.06 (population 06 vs. population 11) to 0.486 (population 06 vs. population 17), with an average of 0.237. The average <em>Nm</em> was estimated at 1.058, suggesting moderate genetic exchange between populations while maintaining a relatively high degree of genetic independence (Table <a href="#ece372043-supitem-0001" class="usa-link">S4</a>). Additionally, genetic relationships among populations, evaluated based on <em>GD</em> and <em>GI</em>, aligned with the results of the <em>Fst</em> and Nm analyses. Populations 06 and 11 exhibited the closest genetic relationship, with the lowest <em>GD</em> (0.121) and highest <em>GI</em> (0.886). Conversely, populations 07 and 18 displayed the highest <em>GD</em> (2.393) and the lowest <em>GI</em> (0.091), reflecting a higher degree of genetic differentiation. Similarly, populations 06 and 17 exhibited considerable differentiation, with a <em>GD</em> of 2.292 and a <em>GI</em> of 0.101 (Table <a href="#ece372043-supitem-0001" class="usa-link">S5</a>).</p>
<p>AMOVA results revealed that 27% of the genetic variation was attributable to differences among populations, 23% to variation among individuals within populations, and the majority, 50%, to variation within individuals (Table <a href="#ece372043-tbl-0002" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="ece372043-tbl-0002"><h5 class="obj_head">TABLE 2.</h5>
<div class="caption p"><p>Analysis of molecular variance (AMOVA) based on eight pairs of SSR markers for 
<em>Bletilla striata</em>
 populations.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Source of variation</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">df</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">SS</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">MS</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Variance components</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Variation percentage</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Among Pops</td>
<td align="center" valign="top" rowspan="1" colspan="1">17</td>
<td align="center" valign="top" rowspan="1" colspan="1">237.458</td>
<td align="center" valign="top" rowspan="1" colspan="1">13.968</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.852</td>
<td align="center" valign="top" rowspan="1" colspan="1">27</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Among Indiv</td>
<td align="center" valign="top" rowspan="1" colspan="1">98</td>
<td align="center" valign="top" rowspan="1" colspan="1">297.521</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.036</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.723</td>
<td align="center" valign="top" rowspan="1" colspan="1">23</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Within Indiv</td>
<td align="center" valign="top" rowspan="1" colspan="1">116</td>
<td align="center" valign="top" rowspan="1" colspan="1">184.5</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.591</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.591</td>
<td align="center" valign="top" rowspan="1" colspan="1">50</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Total</td>
<td align="center" valign="top" rowspan="1" colspan="1">231</td>
<td align="center" valign="top" rowspan="1" colspan="1">719.478</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1">3.17</td>
<td align="center" valign="top" rowspan="1" colspan="1">100</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ece372043-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="ece372043-note-0003"><p>Abbreviations: df, degrees of freedom; MS, mean square; SS, sum of squares.</p></div></div></section></section><section id="ece372043-sec-0014"><h4 class="pmc_sec_title">3.1.3. Population Structure Analysis</h4>
<p>The population genetic structure of 116 
<em>B. striata</em>
 samples was analyzed using STRUCTURE analysis, UPGMA clustering, and PCoA, all of which consistently identified two distinct subgroups. StructureSelector identified <em>K</em> = 2 as optimal, suggesting two primary genetic clusters: Cluster I (red) and Cluster II (green) (Figure <a href="#ece372043-fig-0003" class="usa-link">3</a>). Cluster I consisted of 63 individuals primarily originating from populations 01, 05, 06, 09, 10, 11, 15, and 18, while Cluster II included 53 individuals predominantly from populations 02, 03, 04, 07, 08, 12, 14, 16, and 17. Four individuals had a <em>Q</em> value of &lt; 0.8, indicating complex or mixed genetic origins. UPGMA clustering analysis supported the division of the 116 samples into two main subgroups (Figure <a href="#ece372043-fig-0004" class="usa-link">4</a>). However, individuals from the same geographic location did not consistently cluster together. For instance, individuals from populations 04, 10, and 13 were distributed across different evolutionary subclades. PCoA analysis showed that PCoA1 contributed 23.17%, and PCoA2 contributed 9.61% of the total variation. The scatter plot exhibited two distinct clusters, with individuals from various populations scattered across the distribution (Figure <a href="#ece372043-fig-0005" class="usa-link">5</a>).</p>
<figure class="fig xbox font-sm" id="ece372043-fig-0003"><h5 class="obj_head">FIGURE 3.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c435/12362183/1b24e6feed33/ECE3-15-e72043-g003.jpg" loading="lazy" id="jats-graphic-7" height="394" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/ece372043-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Determination of subpopulations (A) and population genetic structure (B, C) of 116 
<em>Bletilla striata</em>
 accessions from 18 populations based on SSR data. (A) Δ<em>K</em>. (B) When <em>K</em> = 2, the geographical distribution of 18 populations. (C) Genetic clustering result when <em>K</em> = 2.</p></figcaption></figure><figure class="fig xbox font-sm" id="ece372043-fig-0004"><h5 class="obj_head">FIGURE 4.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c435/12362183/4e2f5c350090/ECE3-15-e72043-g006.jpg" loading="lazy" id="jats-graphic-9" height="536" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/ece372043-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>SSR‐based phylogenetic relationships of 116 
<em>Bletilla striata</em>
 accessions based on UPGMA cluster analysis. Cluster I: red and Cluster II: green.</p></figcaption></figure><figure class="fig xbox font-sm" id="ece372043-fig-0005"><h5 class="obj_head">FIGURE 5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c435/12362183/4b2996b98eed/ECE3-15-e72043-g007.jpg" loading="lazy" id="jats-graphic-11" height="309" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/ece372043-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>SSR‐based principal coordinate analysis (PCoA) of 116 
<em>Bletilla striata</em>
 individuals from 18 populations. Different color points represent 
<em>B. striata</em>
 populations.</p></figcaption></figure></section></section><section id="ece372043-sec-0015"><h3 class="pmc_sec_title">3.2. Distribution Pattern and Change Prediction</h3>
<section id="ece372043-sec-0016"><h4 class="pmc_sec_title">3.2.1. Evaluation of the Maxent Model and Key Bioclimate Variables</h4>
<p>The AUC values for various historical and future climate scenarios consistently exceeded 0.9, highlighting the robust predictive accuracy of the optimized Maxent model in reconstructing the population dynamics of 
<em>B. striata</em>
. The sole exception was the 2070‐SSP585 climate scenario, which achieved an AUC of 0.898 (Table <a href="#ece372043-tbl-0003" class="usa-link">3</a>).</p>
<section class="tw xbox font-sm" id="ece372043-tbl-0003"><h5 class="obj_head">TABLE 3.</h5>
<div class="caption p"><p>AUC mean values of the simulated model of 
<em>Bletilla striata</em>
 under historical and six future climate scenarios.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom">
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="2" valign="bottom" colspan="1"></th>
<th align="center" rowspan="2" valign="bottom" colspan="1">History</th>
<th align="center" colspan="3" valign="bottom" rowspan="1">2050s</th>
<th align="center" colspan="3" valign="bottom" rowspan="1">2070s</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="center" valign="bottom" rowspan="1" colspan="1">SSP126</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">SSP245</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">SSP585</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">SSP126</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">SSP245</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">SSP585</th>
</tr>
</thead>
<tbody valign="top"><tr>
<td align="left" valign="top" rowspan="1" colspan="1">AUC</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.91</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.901</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.919</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.912</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.919</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.915</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.898</td>
</tr></tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ece372043-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>Pearson correlation analysis identified six bioclimate variables that significantly contributed to the Maxent model's predictions, with the three most influential being the mean diurnal range (BIO2, 46.6%), min temperature of the coldest month (BIO6, 26.1%), and temperature seasonality (BIO4, 19%). Collectively, these variables accounted for 91.7% of the Maxent model's predictive power (Table <a href="#ece372043-supitem-0001" class="usa-link">S2</a>). Jackknife tests further corroborated these findings, showing that BIO2 had the highest independent impact on the model gain, followed by BIO6 (Figure <a href="#ece372043-supitem-0001" class="usa-link">S2</a>).</p>
<p>To further elucidate the influence of these bioclimatic factors in shaping the potential distribution of 
<em>B. striata</em>
, response curves were constructed. These curves demonstrate that when an occurrence probability exceeding 0.5 signifies favorable conditions for plant growth, it thus enables the determination of the optimal bioclimatic range (Yan et al. <a href="#ece372043-bib-0067" class="usa-link" aria-describedby="ece372043-bib-0067">2021</a>; Fang et al. <a href="#ece372043-bib-0008" class="usa-link" aria-describedby="ece372043-bib-0008">2024</a>).</p>
<p>The response curves for the key climatic variables exhibited the following trends:
</p>
<ul id="ece372043-list-0001" class="list" style="list-style-type:disc">
<li id="ece372043-li-0001"><p>BIO2: Occurrence probability begins to decline when values exceed 4.9°C, stabilizing around 18.267°C.</p></li>
<li id="ece372043-li-0002"><p>BIO6: The curve rises from −8°C, peaking and stabilizing at 16.8°C.</p></li>
<li id="ece372043-li-0003"><p>BIO4: Occurrence probability decreases beyond 306.716°C, reaching equilibrium at 1762.355°C.</p></li>
</ul>
<p>Based on these results, the optimal bioclimatic ranges for 
<em>B. striata</em>
 in its potential habitat are as follows: BIO2 (3.56°C–9.33°C), BIO6 (0.95°C–22.24°C), and BIO4 (161.15°C–679.24°C) (Figure <a href="#ece372043-supitem-0001" class="usa-link">S3</a>).</p></section><section id="ece372043-sec-0017"><h4 class="pmc_sec_title">3.2.2. Distribution of 
<em>B. striata</em>
 Under Historical Climatic Conditions</h4>
<p>During the 1970–2000 period, the highly suitable habitat for 
<em>B. striata</em>
 within China was estimated to encompass 67.3099 km<sup>2</sup>, with moderately suitable areas covering 104.7786 km<sup>2</sup> and low‐suitability areas spanning 116.2565 km<sup>2</sup>. In contrast, unsuitable areas accounted for 670.6716 km<sup>2</sup>. Overall, the total suitable habitat constituted approximately 30.12% of China's land area (Table <a href="#ece372043-tbl-0004" class="usa-link">4</a>). The highly suitable regions were primarily distributed across southeastern and southwestern China, including Chongqing, Guizhou, western Guangxi, southeastern Yunnan, Hainan, and eastern Taiwan (Figure <a href="#ece372043-fig-0006" class="usa-link">6</a>).</p>
<section class="tw xbox font-sm" id="ece372043-tbl-0004"><h5 class="obj_head">TABLE 4.</h5>
<div class="caption p"><p>Area (×10<sup>4</sup> km<sup>2</sup>) and proportion (%) of suitable habitats for 
<em>Bletilla striata</em>
 under historical and future climate scenarios.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Period</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Unsuitable area (%)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Total suitable area (%)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Low suitable area (%)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Medium suitable area (%)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">High suitable area (%)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">History (1970–2000)</td>
<td align="center" valign="top" rowspan="1" colspan="1">670.6716 (69.88)</td>
<td align="center" valign="top" rowspan="1" colspan="1">288.3450 (30.12)</td>
<td align="center" valign="top" rowspan="1" colspan="1">116.2565 (12.12)</td>
<td align="center" valign="top" rowspan="1" colspan="1">104.7786 (10.93)</td>
<td align="center" valign="top" rowspan="1" colspan="1">67.3099 (7.07)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2050s‐SSP126</td>
<td align="center" valign="top" rowspan="1" colspan="1">607.5052 (63.21)</td>
<td align="center" valign="top" rowspan="1" colspan="1">351.9792 (36.69)</td>
<td align="center" valign="top" rowspan="1" colspan="1">170.8455 (17.81)</td>
<td align="center" valign="top" rowspan="1" colspan="1">107.8542 (11.24)</td>
<td align="center" valign="top" rowspan="1" colspan="1">73.2795 (7.64)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2050s‐SSP245</td>
<td align="center" valign="top" rowspan="1" colspan="1">559.8924 (58.35)</td>
<td align="center" valign="top" rowspan="1" colspan="1">399.5921 (41.65)</td>
<td align="center" valign="top" rowspan="1" colspan="1">169.3056 (17.64)</td>
<td align="center" valign="top" rowspan="1" colspan="1">141.7205 (14.77)</td>
<td align="center" valign="top" rowspan="1" colspan="1">88.56560 (9.24)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2050s‐SSP585</td>
<td align="center" valign="top" rowspan="1" colspan="1">591.1632 (61.52)</td>
<td align="center" valign="top" rowspan="1" colspan="1">368.3211 (38.38)</td>
<td align="center" valign="top" rowspan="1" colspan="1">178.6007 (18.61)</td>
<td align="center" valign="top" rowspan="1" colspan="1">117.2378 (12.22)</td>
<td align="center" valign="top" rowspan="1" colspan="1">72.4826 (7.55)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2070s‐SSP126</td>
<td align="center" valign="top" rowspan="1" colspan="1">594.2413 (61.94)</td>
<td align="center" valign="top" rowspan="1" colspan="1">365.2430 (38.06)</td>
<td align="center" valign="top" rowspan="1" colspan="1">167.2691 (17.44)</td>
<td align="center" valign="top" rowspan="1" colspan="1">125.6128 (13.09)</td>
<td align="center" valign="top" rowspan="1" colspan="1">72.3611 (7.53)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2070s‐SSP245</td>
<td align="center" valign="top" rowspan="1" colspan="1">575.8993 (59.98)</td>
<td align="center" valign="top" rowspan="1" colspan="1">383.5851 (40.01)</td>
<td align="center" valign="top" rowspan="1" colspan="1">173.2743 (18.06)</td>
<td align="center" valign="top" rowspan="1" colspan="1">139.6615 (14.56)</td>
<td align="center" valign="top" rowspan="1" colspan="1">70.6493 (14.56)</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2070s‐SSP585</td>
<td align="center" valign="top" rowspan="1" colspan="1">553.8768 (57.62)</td>
<td align="center" valign="top" rowspan="1" colspan="1">405.6077 (42.25)</td>
<td align="center" valign="top" rowspan="1" colspan="1">201.2240 (20.97)</td>
<td align="center" valign="top" rowspan="1" colspan="1">141.0799 (14.69)</td>
<td align="center" valign="top" rowspan="1" colspan="1">63.3038 (6.59)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ece372043-tbl-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="ece372043-fig-0006"><h5 class="obj_head">FIGURE 6.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c435/12362183/b871c249c9d4/ECE3-15-e72043-g009.jpg" loading="lazy" id="jats-graphic-13" height="408" width="709" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/ece372043-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The potential suitable areas for 
<em>Bletilla striata</em>
 under different climatic scenarios. The prediction was conducted using Maxent modeling in historical (A), 2050s (B), and 2070s (C) periods. Yellow, green, and violet dust areas represent unsuitable, low suitable, medium suitable, and high suitable areas, respectively.</p></figcaption></figure></section><section id="ece372043-sec-0018"><h4 class="pmc_sec_title">3.2.3. Potential Distribution of 
<em>B. striata</em>
 Under Future Climatic Scenarios</h4>
<p>Under future climate scenarios, the potentially suitable growth areas for <em>B. striata</em> are projected to expand as fossil fuel consumption increases. Compared to historical climate conditions, the green development pathway (SSP126) predicts 6.57% and 7.94% increases in suitable habitat area by the 2050s and 2070s, respectively. Similarly, under the SSP245 scenario, the potentially suitable area is anticipated to expand by 11.53% and 9.89%, while the SSP585 scenario forecasts increases of 8.26% and 12.13% for the respective periods (Table <a href="#ece372043-tbl-0004" class="usa-link">4</a>).</p>
<p>In the 2050s, under the SSP126 scenario, the expansion of highly suitable areas is mainly attributed to the transition of low‐suitability regions in northwestern Taiwan and moderate‐suitability regions in eastern Shandong into high‐suitability zones. This indicates a notable northward shift in habitat suitability. Under the SSP245 scenario, highly suitable areas extend into Shaanxi and Hubei provinces, while regions in Shandong, Henan, and Anhui transition from low to moderate suitability. Additionally, suitable areas begin to emerge in northwestern Xinjiang, with a marked increase in moderately suitable regions, reinforcing the northward expansion trend. The SSP585 scenario predicts significant growth in suitability around the Tianshan Mountains in Xinjiang (Figure <a href="#ece372043-fig-0006" class="usa-link">6</a>).</p>
<p>By the 2070s, under the SSP126 and SSP245 scenarios, highly suitable areas are projected to increase by 0.46% and 0.32%, respectively. However, under the SSP585 scenario, a slight decrease of 0.48% is observed. Despite these fluctuations, the primary distribution areas remain concentrated in the southwest, with trends consistent with those observed in the 2050s. These include a continued northward expansion and substantial increases in suitable habitats within Xinjiang (Figure <a href="#ece372043-fig-0006" class="usa-link">6</a>).</p></section><section id="ece372043-sec-0019"><h4 class="pmc_sec_title">3.2.4. Distribution Changes Between Historical and Future Climate Scenarios</h4>
<p>The spatial patterns of suitable habitats for 
<em>B. striata</em>
 were analyzed by comparing historical distributions with projections for the 2050s and 2070s under different climate scenarios (SSP126, SSP245, and SSP585) (Table <a href="#ece372043-tbl-0005" class="usa-link">5</a>, Figure <a href="#ece372043-fig-0007" class="usa-link">7</a>).</p>
<section class="tw xbox font-sm" id="ece372043-tbl-0005"><h5 class="obj_head">TABLE 5.</h5>
<div class="caption p"><p>Changes in the suitable habitats of 
<em>Bletilla striata</em>
 under different future climate scenarios.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom">
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="2" valign="bottom" colspan="1">Period</th>
<th align="center" colspan="3" valign="bottom" rowspan="1">Area (×10<sup>4</sup> km<sup>2</sup>)</th>
<th align="center" colspan="3" valign="bottom" rowspan="1">Change (%)</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="center" valign="bottom" rowspan="1" colspan="1">Gain</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Lost</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Stable</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Gain rate</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Loss rate</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Stable rate</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2050s‐SSP126</td>
<td align="center" valign="top" rowspan="1" colspan="1">81.679</td>
<td align="center" valign="top" rowspan="1" colspan="1">19.387</td>
<td align="center" valign="top" rowspan="1" colspan="1">268.326</td>
<td align="center" valign="top" rowspan="1" colspan="1">22.11</td>
<td align="center" valign="top" rowspan="1" colspan="1">5.25</td>
<td align="center" valign="top" rowspan="1" colspan="1">72.64</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2050s‐SSP245</td>
<td align="center" valign="top" rowspan="1" colspan="1">114.901</td>
<td align="center" valign="top" rowspan="1" colspan="1">5.066</td>
<td align="center" valign="top" rowspan="1" colspan="1">282.648</td>
<td align="center" valign="top" rowspan="1" colspan="1">28.54</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.26</td>
<td align="center" valign="top" rowspan="1" colspan="1">70.2</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2050s‐SSP585</td>
<td align="center" valign="top" rowspan="1" colspan="1">89.152</td>
<td align="center" valign="top" rowspan="1" colspan="1">10.493</td>
<td align="center" valign="top" rowspan="1" colspan="1">277.217</td>
<td align="center" valign="top" rowspan="1" colspan="1">23.66</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.78</td>
<td align="center" valign="top" rowspan="1" colspan="1">73.56</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2070s‐SSP126</td>
<td align="center" valign="top" rowspan="1" colspan="1">80.831</td>
<td align="center" valign="top" rowspan="1" colspan="1">5.179</td>
<td align="center" valign="top" rowspan="1" colspan="1">282.535</td>
<td align="center" valign="top" rowspan="1" colspan="1">21.93</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.41</td>
<td align="center" valign="top" rowspan="1" colspan="1">76.66</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2070s‐SSP245</td>
<td align="center" valign="top" rowspan="1" colspan="1">104.734</td>
<td align="center" valign="top" rowspan="1" colspan="1">10.772</td>
<td align="center" valign="top" rowspan="1" colspan="1">276.943</td>
<td align="center" valign="top" rowspan="1" colspan="1">26.69</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.74</td>
<td align="center" valign="top" rowspan="1" colspan="1">70.57</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2070s‐SSP585</td>
<td align="center" valign="top" rowspan="1" colspan="1">135.611</td>
<td align="center" valign="top" rowspan="1" colspan="1">19.308</td>
<td align="center" valign="top" rowspan="1" colspan="1">268.412</td>
<td align="center" valign="top" rowspan="1" colspan="1">32.03</td>
<td align="center" valign="top" rowspan="1" colspan="1">4.56</td>
<td align="center" valign="top" rowspan="1" colspan="1">63.40</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ece372043-tbl-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="ece372043-fig-0007"><h5 class="obj_head">FIGURE 7.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c435/12362183/c76ed09f64ff/ECE3-15-e72043-g004.jpg" loading="lazy" id="jats-graphic-15" height="390" width="709" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/ece372043-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Potential spatial changes of 
<em>Bletilla striata</em>
 under future climatic scenarios. Blue, red, and green areas represent lost, gain, and stable areas, respectively.</p></figcaption></figure><p>Under the SSP126 scenario, the growth rates of suitable habitats are projected to be 22.11% and 21.93% for the 2050s and 2070s, respectively. The primary regions contributing to habitat expansion include Hebei, Shandong, Tibet, Shanxi, Shaanxi, and Xinjiang. Habitat loss rates are estimated at 5.25% and 1.41% for the respective periods. In the 2050s, habitat loss is concentrated in northwestern Anhui, eastern Hubei, and southern Henan, while in the 2070s, the losses are mainly observed in southwestern Yunnan.</p>
<p>Under the SSP245 scenario, suitable habitat growth rates are expected to reach 28.54% and 26.69% for the 2050s and 2070s, respectively, with growth occurring in regions similar to those under the SSP126 scenario. However, a rapid expansion trend is projected in Inner Mongolia. Habitat loss rates are 1.26% and 2.74% for the 2050s and 2070s, with losses concentrated in southwestern Yunnan during both periods.</p>
<p>Under the SSP585 scenario, the habitat growth rates are projected at 23.66% for the 2050s and 32.03% for the 2070s. The primary regions driving habitat expansion in the 2050s are similar to those under SSP126 and SSP245, while the 2070s are expected to see significant habitat growth across Tibet. Habitat loss rates are estimated at 2.78% and 4.56% for the 2050s and 2070s, respectively. In the 2050s, habitat loss is concentrated in northeastern Anhui, southwestern Yunnan, and parts of eastern Jiangxi. By the 2070s, losses are expected in southeastern Jiangxi, northwestern Fujian, and northwestern Zhejiang.</p>
<p>Across future climate scenarios, the stability rate for suitable habitats is projected to range from 63.40% to 76.66%. These stable regions are predominantly located in southwestern and central China, indicating areas of consistent suitability under changing climatic conditions.</p>
<p>Under historical climate conditions, the distribution center of 
<em>B. striata</em>
 was primarily located in Sichuan Province (101°59′ E, 30°21′ N). In the 2050s, projections under various climate scenarios indicate a northwestern shift to Qinghai Province, with the distribution centers predicted at (99°19′ E, 33°18′ N) for SSP126, (99°13′ E, 33°47′ N) for SSP245, and (99°47′ E, 33°24′ N) for SSP585. By the 2070s, the predicted distribution center under the SSP126 scenario shifts to northwestern Sichuan (98°37′ E, 32°43′ N), while under SSP245, it remains in northwestern Sichuan (99°59′ E, 32°21′ N). For SSP585, the center moves further into southern Qinghai (99°47′ E, 33°24′ N). Overall, these projections suggest a consistent trend of northwestward movement in the distribution center of 
<em>B. striata</em>
 under future climate scenarios (Figure <a href="#ece372043-fig-0008" class="usa-link">8</a>).</p>
<figure class="fig xbox font-sm" id="ece372043-fig-0008"><h5 class="obj_head">FIGURE 8.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/c435/12362183/7c97a603ea31/ECE3-15-e72043-g008.jpg" loading="lazy" id="jats-graphic-17" height="485" width="709" alt="FIGURE 8"></p>
<div class="p text-right font-secondary"><a href="figure/ece372043-fig-0008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Migration of the center of suitable habitats for 
<em>Bletilla striata</em>
 migratory routes in historical and future climate scenarios. Black, green, and red dots represent the suitable habitat distribution centroid in historical, 2050s, and 2070s periods, respectively.</p></figcaption></figure></section></section></section><section id="ece372043-sec-0020"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>SSR molecular markers are widely utilized in genetic breeding, population genetics, and fingerprinting studies in medicinal plants, such as <em>Zanthoxylum nitidum</em> (Roxb.) DC. (Zhu et al. <a href="#ece372043-bib-0076" class="usa-link" aria-describedby="ece372043-bib-0076">2023</a>), <em>Paris polyphylla</em> Sm. (Oliya et al. <a href="#ece372043-bib-0034" class="usa-link" aria-describedby="ece372043-bib-0034">2023</a>) and <em>Polygonatum cyrtonema</em> Hua (Liu, Cheng, et al. <a href="#ece372043-bib-0028" class="usa-link" aria-describedby="ece372043-bib-0028">2023a</a>). Although the genome (Jiang et al. <a href="#ece372043-bib-0022" class="usa-link" aria-describedby="ece372043-bib-0022">2022</a>), transcriptome (Xu et al. <a href="#ece372043-bib-0066" class="usa-link" aria-describedby="ece372043-bib-0066">2018</a>; Niu et al. <a href="#ece372043-bib-0033" class="usa-link" aria-describedby="ece372043-bib-0033">2020</a>), and chloroplast genome (Cai et al. <a href="#ece372043-bib-0003" class="usa-link" aria-describedby="ece372043-bib-0003">2020</a>) of 
<em>B. striata</em>
 have been published, no systematic population genetic studies have been conducted due to challenges in wild sampling. Our research group utilized transcriptome data to identify 25,935 high‐quality SSR primer pairs (Xu et al. <a href="#ece372043-bib-0066" class="usa-link" aria-describedby="ece372043-bib-0066">2018</a>) in earlier work.</p>
<p>In this study, 116 individuals from 18 wild populations of 
<em>B. striata</em>
 across southwestern China and neighboring regions were extensively sampled. Using eight selected SSR primers, the study documented substantial genetic diversity (mean <em>Na</em> = 3.549, <em>Ne</em> = 2.636, and <em>I</em> = 0.943). The pairwise <em>Fst</em> and <em>Nm</em> values—key parameters for assessing genetic partitioning (Yang et al. <a href="#ece372043-bib-0068" class="usa-link" aria-describedby="ece372043-bib-0068">2022</a>)—indicate moderate genetic differentiation among the 18 
<em>B. striata</em>
 populations (mean <em>Fst</em> = 0.237; Wright <a href="#ece372043-bib-0062" class="usa-link" aria-describedby="ece372043-bib-0062">1922</a>) alongside moderate gene flow (mean <em>Nm</em> = 1.058; Slatkin <a href="#ece372043-bib-0048" class="usa-link" aria-describedby="ece372043-bib-0048">1987</a>). This pattern reflects a dynamic equilibrium between isolation‐by‐distance effects and episodic long‐distance dispersal events, potentially mediated by pollinator behavior or anthropogenic seed transport. The mountainous terrain functions as a “semi‐permeable landscape filter”, where ridges promote drift‐induced differentiation while valley corridors facilitate constrained gene flow, collectively amplifying intra‐population genetic variation (Hesselbarth et al. <a href="#ece372043-bib-0019" class="usa-link" aria-describedby="ece372043-bib-0019">2025</a>). These findings are consistent with AMOVA results, which showed that most genetic variation was attributable to within‐population differences. Furthermore, due to the severe depletion of 
<em>B. striata</em>
 resources in the wild and the associated challenges in sampling, earlier studies employed molecular markers such as Sequence‐Related Amplified Polymorphism (SRAP), Start Codon Targeted Polymorphism (SCoT), and Inter‐Retrotransposon Amplified Polymorphism (IRAP) to explore the genetic diversity of 
<em>B. striata</em>
 germplasm across multiple regions, albeit with limited sample sizes (Gui et al. <a href="#ece372043-bib-0012" class="usa-link" aria-describedby="ece372043-bib-0012">2024</a>; Guo et al. <a href="#ece372043-bib-0015" class="usa-link" aria-describedby="ece372043-bib-0015">2018</a>; Sun et al. <a href="#ece372043-bib-0051" class="usa-link" aria-describedby="ece372043-bib-0051">2016</a>). These studies consistently demonstrated high genetic diversity and predominantly within‐population genetic variation, aligning with the results of this study.</p>
<p>Additionally, the UPGMA clustering, STRUCTURE analysis, and PCoA analysis conducted in this study were effective in identifying homogeneous clusters among individuals (Elumalai and Srinivasan <a href="#ece372043-bib-0007" class="usa-link" aria-describedby="ece372043-bib-0007">2023</a>). The results consistently indicated that the 
<em>B. striata</em>
 samples from the 18 populations could be broadly classified into two subgroups. Individuals from the same population exhibited close genetic relationships and tended to cluster together. However, individuals from populations in Wushan (04), Yilong (10), and Houhe Tianmen Gorge (13) were dispersed across different subgroups. This pattern may be attributed to several factors, including intra‐population variation, mixed ancestry, historical gene flow, anthropogenic translocation, incomplete lineage sorting due to recent divergence, or the limitations of the markers and analytical methods used (Wu, Yang, et al. <a href="#ece372043-bib-0065" class="usa-link" aria-describedby="ece372043-bib-0065">2024b</a>). The absence of strict geographical segregation between clusters suggests extensive historical gene flow events across southwestern China, possibly facilitated by epizoochory or anthropogenic movement of tubers prior to recent habitat fragmentation. The dispersed distribution of populations 04, 10, and 13 across subgroups may also reflect their adaptive potential shaped by hybridization. Hybrid populations may integrate adaptive alleles from distinct lineages through dominance or overdominance effects (Harrison and Larson <a href="#ece372043-bib-0017" class="usa-link" aria-describedby="ece372043-bib-0017">2014</a>), thereby increasing genetic diversity and potentially generating more adaptive phenotypes in specific environments. This is particularly relevant for populations in ecotonal zones (e.g., Wushan and Houhe Tianmen Gorge), where fluctuating climates and complex topographies select for phenotypic plasticity (Hoffmann and Sgrò <a href="#ece372043-bib-0020" class="usa-link" aria-describedby="ece372043-bib-0020">2011</a>).</p>
<p>Field sampling poses challenges, and our sampling is limited to mainland China, excluding peripheral populations in Korea, Japan, Taiwan, and Hengduan Mountains. While this may affect inferences about species‐wide diversity patterns, prior studies suggest that peripheral populations often exhibit lower genetic diversity due to founder effects and isolation (Tian et al. <a href="#ece372043-bib-0054" class="usa-link" aria-describedby="ece372043-bib-0054">2018</a>). The core distribution area (southwestern China) likely retains ancestral diversity, as observed in 
<em>B. striata</em>
's climatic optimal habitats (Gong et al. <a href="#ece372043-bib-0009" class="usa-link" aria-describedby="ece372043-bib-0009">2014</a>) and other orchids with glacial refugia in western China (Guo, Cao, Li, et al. <a href="#ece372043-bib-0014" class="usa-link" aria-describedby="ece372043-bib-0014">2019b</a>). Future collaborative efforts should prioritize cross‐border sampling to test biogeographic hypotheses. While our SSR markers exhibited high discriminatory power (mean <em>PIC =</em> 0.748; Botstein et al. <a href="#ece372043-bib-0002" class="usa-link" aria-describedby="ece372043-bib-0002">1980</a>) and association with functional loci (Xu et al. <a href="#ece372043-bib-0066" class="usa-link" aria-describedby="ece372043-bib-0066">2018</a>), we acknowledge methodological limitations. The rarefaction curve (Figure <a href="#ece372043-supitem-0001" class="usa-link">S4</a>) indicates incomplete allele saturation, suggesting potential undetected variation with additional markers. Although this constrained marker set aligns with successful orchid diversity studies using 6–10 SSR loci in orchids (e.g., <em>Zygopetalum mackayi</em>, Moura et al. <a href="#ece372043-bib-0032" class="usa-link" aria-describedby="ece372043-bib-0032">2020</a>; <em>Vanilla odorata</em>, Serna‐Sánchez et al. <a href="#ece372043-bib-0047" class="usa-link" aria-describedby="ece372043-bib-0047">2025</a>), it may obscure fine‐scale population differentiation. Crucially, the convergence of STRUCTURE, UPGMA, and PCoA results in identifying two primary clusters provides robust validation of our macro‐scale genetic structure conclusions. Future investigations should incorporate more SSR markers or genome‐wide approaches to resolve subtle differentiation patterns and adaptive variation.</p>
<p>Importantly, the observed genetic structure and gene flow patterns may be further influenced by future climate‐driven habitat shifts. Climate change is a key driver of species extinction, habitat redistribution, and shifts in vegetation type, with the Orchidaceae family—one of the most species‐rich plant families worldwide—being particularly vulnerable to its effects (Pica et al. <a href="#ece372043-bib-0040" class="usa-link" aria-describedby="ece372043-bib-0040">2024</a>). Our study identified temperature‐related factors as the primary determinants shaping the future distribution patterns of 
<em>B. striata</em>
. Specifically, BIO2 (mean diurnal range; 46.6% contribution), BIO6 (min temperature of coldest month; 26.1%) and BIO4 (temperature seasonality; 19%) contributed the most to the Maxent model, with respective contributions of 46.6%, 26.1%, and 19%. The underlying mechanisms may involve the diurnal temperature range affecting biomass accumulation (Sunoj et al. <a href="#ece372043-bib-0052" class="usa-link" aria-describedby="ece372043-bib-0052">2016</a>), the minimum temperature during the coldest month facilitating vernalization (Griebeler and Kadereit <a href="#ece372043-bib-0011" class="usa-link" aria-describedby="ece372043-bib-0011">2023</a>), and temperature seasonality regulating regular phenological cycles (Zhao et al. <a href="#ece372043-bib-0072" class="usa-link" aria-describedby="ece372043-bib-0072">2023</a>). Optimal climatic conditions for <em>B. striata</em> growth were found to be BIO2 ranging from 3.56°C to 9.33°C, BIO6 from 0.95°C to 22.24°C, and BIO4 from 161.15°C to 679.24°C. While climate variables were prioritized due to their established strong influence on orchid distributions (Ye et al. <a href="#ece372043-bib-0069" class="usa-link" aria-describedby="ece372043-bib-0069">2022</a>; Mafakheri et al. <a href="#ece372043-bib-0031" class="usa-link" aria-describedby="ece372043-bib-0031">2022</a>), we acknowledge that excluding factors such as soil properties and topography represents a limitation for habitat suitability generalization (Urban et al. <a href="#ece372043-bib-0056" class="usa-link" aria-describedby="ece372043-bib-0056">2016</a>).</p>
<p>This study, based on the CMIP6 model, provides valuable insights into the potential distribution patterns of 
<em>B. striata</em>
 under future climate scenarios (2050s and 2070s) across SSP126 (low greenhouse gas emissions), SSP245 (moderate emissions), and SSP585 (high emissions). The results indicate a significant expansion of suitable habitats, with coverage increasing from 30.12% under historical conditions to a range of 36.69% (2050s‐SSP126)–42.25% (2070s‐SSP585). The primary suitable habitats of 
<em>B. striata</em>
 are concentrated in southwestern China, including Guizhou, Chongqing, Sichuan, as well as parts of Hainan, western Guangxi, and southeastern Yunnan. Additionally, its potential distribution extends to areas south of the Qinling Mountains and Huaihe River, with a discernible trend of northward expansion under future climate scenarios. These findings align with those of previous studies (Gong et al. <a href="#ece372043-bib-0009" class="usa-link" aria-describedby="ece372043-bib-0009">2014</a>; Wang et al. <a href="#ece372043-bib-0060" class="usa-link" aria-describedby="ece372043-bib-0060">2011</a>). However, the actual migration may lag behind climatic suitability due to dispersal limitations and soil‐microbiome dependencies, emphasizing the need for assisted migration strategies (Phillips et al. <a href="#ece372043-bib-0037" class="usa-link" aria-describedby="ece372043-bib-0037">2020</a>).</p>
<p>Under projected future scenarios, ecological changes in regions such as Inner Mongolia, Xinjiang, and Tibet may create favorable conditions for the growth of 
<em>B. striata</em>
. The distribution center of the species is also expected to shift northwestward, transitioning from Sichuan (1970–2000) to Qinghai (2050s). By the 2070s, the distribution center migrates from central Sichuan to its northwestern regions under SSP126 and SSP245 scenarios and further into Qinghai under SSP585. The northward range expansion may favor Cluster II populations (predominantly northern Sichuan/Yunnan), which exhibit higher allele richness (Na = 11.625 vs. 11.125 in Cluster I) and could harbor pre‐adaptations to cooler environments; experimental studies are needed to test this hypothesis. These projections suggest that northwestern China may become a viable area for the climate‐resilient cultivation of 
<em>B. striata</em>
 in the future.</p>
<p>These results highlight two key considerations. First, the high genetic diversity observed within natural populations offers critical resources for the conservation of wild 
<em>B. striata</em>
. To mitigate human disturbances, we recommend implementing ex situ conservation and reintroduction measures in suitable habitats, particularly in Sichuan and Guizhou. Large‐scale cultivation of 
<em>B. striata</em>
 in these highly suitable areas offers significant potential for balancing ecological conservation, rational resource utilization, and economic value enhancement. Second, we strongly advocate for establishing long‐term monitoring networks to track population dynamics across the current and projected range, integrating genetic health assessments (e.g., every 5 years) and a threat detection system to enable adaptive management.</p></section><section id="ece372043-sec-0021"><h2 class="pmc_sec_title">5. Conclusion</h2>
<p>This study integrates SSR‐based population genetics and Maxent species distribution modeling to characterize the genetic architecture and habitat dynamics of 
<em>B. striata</em>
 across southwestern China and adjacent regions. Key findings reveal 1. a distinct phylogeographic structure with two genetic clusters exhibiting moderate differentiation (<em>Fst</em> = 0.237) yet sustained gene flow (<em>Nm</em> = 1.058), maintained through a balance of isolation‐by‐distance and episodic long‐distance dispersal across mountainous terrain; 2. significant within‐population diversity (&gt; 80% AMOVA variation), with populations 05 (Cluster I) and 16 (Cluster II) identified as priority genetic reservoirs due to exceptional allelic richness; and 3. climate‐driven distribution shifts dominated by temperature variables (BIO2/4/6), projecting 22%–40% habitat expansion and northwestward centroid migration under future scenarios.</p>
<p>We propose a three‐tiered conservation framework: (1) High‐diversity sources (Pop 05‐Cluster I, Pop 16‐Cluster II) warrant urgent seed banking and serve as primary germplasm for ex‐situ collections, with monitoring for inbreeding depression. (2) Stable founder populations (Pop 01‐Cluster I, Pop 03‐Cluster II) are ideal for cultivation trials in replicable habitats, allowing controlled intraspecific mixing (&lt; 30% admixed propagules) to enhance diversity while preserving local adaptation. (3) Admixed populations (e.g., Pop 04) require spatial isolation in ex‐situ conservation and fitness‐based breeding programs to mitigate outbreeding depression risks.</p>
<p>This work establishes an evolutionarily informed framework for conserving 
<em>B. striata</em>
 resources under climate change, demonstrating how landscape‐genetic patterns can guide sustainable management of threatened medicinal orchids.</p></section><section id="ece372043-sec-0026"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>
<strong>Liangliang Luo:</strong> data curation (equal), software (equal), visualization (equal), writing – original draft (lead). <strong>Qian Wang:</strong> investigation (equal), software (equal), supervision (equal), visualization (equal), writing – review and editing (supporting). <strong>Xiaolan Li:</strong> data curation (equal), formal analysis (equal), resources (equal), software (equal). <strong>Delin Xu:</strong> methodology (equal), project administration (equal), resources (equal), software (equal). <strong>Huan Hu:</strong> funding acquisition (lead), methodology (equal), project administration (equal), supervision (equal), writing – review and editing (lead).</p></section><section id="ece372043-sec-0023"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec24"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="ece372043-supitem-0001"><div class="caption p"><p>
<strong>Appendix S1:</strong> ece372043‐sup‐0001‐AppendixS1.docx.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12362183/bin/ECE3-15-e72043-s003.docx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e72043-s003.docx</a><sup> (1MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="ece372043-supitem-0002"><div class="caption p"><p>
<strong>Data S1:</strong> ece372043‐sup‐0002‐DataS1.xlsx.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12362183/bin/ECE3-15-e72043-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e72043-s001.xlsx</a><sup> (14.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="ece372043-supitem-0003"><div class="caption p"><p>
<strong>Data S2:</strong> ece372043‐sup‐0003‐DataS2.xlsx.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12362183/bin/ECE3-15-e72043-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e72043-s002.xlsx</a><sup> (19.7KB, xlsx) </sup>
</div></div></section></section><section id="ece372043-sec-0022" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>This work was supported by the Science and Technology Program of Guizhou Province (QKHJC‐ZK [2022] 591) (QKHPTRC [2018]069). We are very grateful to the CVH and GBIF resource sharing platforms and the authors who provided the longitude and latitude information of <em>Bletilla striata</em> collection sites; we are also thankful to the WorldClim resource platform for the environmental data used in the species distribution model.</p></section><section id="notes1"><p>


Luo, L.
, Wang Q., Li X., Xu D., and Hu H.. 2025. “Population Genetic Diversity and Species Distribution Evaluation of <em>Bletilla striata</em> (Orchidaceae) in Southwest China Using SSR Markers.” Ecology and Evolution
15, no. 8: e72043. 10.1002/ece3.72043.
</p>
<section id="ece372043-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="ece372043-note-0001"><p>
<strong>Funding:</strong> This work was supported by the Science and Technology Program of Guizhou Province (QKHJC‐ZK [2022] 591) (QKHPTRC [2018]069).</p></div></div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Qian Wang, Email: qianwang07@126.com.</p>
<p>Huan Hu, Email: huhuan1990@163.com.</p></section><section id="ece372043-sec-0025"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>The data and materials supporting this study are included in the article and Appendix <a href="#ece372043-supitem-0001" class="usa-link">S1</a>, Data <a href="#ece372043-supitem-0002" class="usa-link">S1</a>, and Data <a href="#ece372043-supitem-0003" class="usa-link">S2</a>.</p></section><section id="ece372043-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ece372043-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="ece372043-bib-0001">
<cite>

Bo, P. L.
, Wei D., and Jun H. Y.. 2022. “Prediction on the Potential Planting Area of 
<em>Carya illinoinensis</em>
 in China Based on MaxEnt Model.” Journal of Zhejiang A and F University
39: 76–83.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Zhejiang%20A%20and%20F%20University&amp;title=Prediction%20on%20the%20Potential%20Planting%20Area%20of%20Carya%20illinoinensis%20in%20China%20Based%20on%20MaxEnt%20Model&amp;author=P.%20L.%20Bo&amp;author=D.%20Wei&amp;author=H.%20Y.%20Jun&amp;volume=39&amp;publication_year=2022&amp;pages=76-83&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0002">
<cite>

Botstein, D.
, White R. L., Skolnick M., and Davis R. W.. 1980. “Construction of a Genetic Linkage Map in Man Using Restriction Fragment Length Polymorphisms.” American Journal of Human Genetics
32: 314.
</cite> [<a href="/articles/PMC1686077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6247908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Construction%20of%20a%20Genetic%20Linkage%20Map%20in%20Man%20Using%20Restriction%20Fragment%20Length%20Polymorphisms&amp;author=D.%20Botstein&amp;author=R.%20L.%20White&amp;author=M.%20Skolnick&amp;author=R.%20W.%20Davis&amp;volume=32&amp;publication_year=1980&amp;pages=314&amp;pmid=6247908&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0003">
<cite>

Cai, Z.
, Wang H., and Wang G.. 2020. “Complete Chloroplast Genome Sequence of 
<em>Bletilla striata</em>
 (Thunb.) Reichb. F., a Chinese Folk Medicinal Plant.” Mitochondrial DNA Part B Resources
5: 2239–2240.
</cite> [<a href="https://doi.org/10.1080/23802359.2020.1770138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7510607/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33366989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mitochondrial%20DNA%20Part%20B%20Resources&amp;title=Complete%20Chloroplast%20Genome%20Sequence%20of%20Bletilla%20striata%20(Thunb.)%20Reichb.%20F.,%20a%20Chinese%20Folk%20Medicinal%20Plant&amp;author=Z.%20Cai&amp;author=H.%20Wang&amp;author=G.%20Wang&amp;volume=5&amp;publication_year=2020&amp;pages=2239-2240&amp;pmid=33366989&amp;doi=10.1080/23802359.2020.1770138&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0004">
<cite>

Chen, X. Q.
, Gale S. W., and Cribb P. J.. 2009. Flora of China. Vol. 25, 209–210. Missouri Botanical Garden Press.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Flora%20of%20China&amp;author=X.%20Q.%20Chen&amp;author=S.%20W.%20Gale&amp;author=P.%20J.%20Cribb&amp;publication_year=2009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0005">
<cite>

Cobos, M. E.
, Peterson A. T., Barve N., and Osorio‐Olvera L.. 2019. “Kuenm: An R Package for Detailed Development of Ecological Niche Models Using Maxent.” PeerJ
7: e6281.
</cite> [<a href="https://doi.org/10.7717/peerj.6281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6368831/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30755826/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=Kuenm:%20An%20R%20Package%20for%20Detailed%20Development%20of%20Ecological%20Niche%20Models%20Using%20Maxent&amp;author=M.%20E.%20Cobos&amp;author=A.%20T.%20Peterson&amp;author=N.%20Barve&amp;author=L.%20Osorio%E2%80%90Olvera&amp;volume=7&amp;publication_year=2019&amp;pages=e6281&amp;pmid=30755826&amp;doi=10.7717/peerj.6281&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0006">
<cite>

Deng, W. X.
, Zhao M. L., Li Y. M., Wang Z. L., Zhang K., and Zhan F. D.. 2019. “Diversity of Endophytic Fungi Associated With <em>Bletilla Striata</em> Roots.” Mycosystema
38: 1907–1917.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Mycosystema&amp;title=Diversity%20of%20Endophytic%20Fungi%20Associated%20With%20Bletilla%20Striata%20Roots&amp;author=W.%20X.%20Deng&amp;author=M.%20L.%20Zhao&amp;author=Y.%20M.%20Li&amp;author=Z.%20L.%20Wang&amp;author=K.%20Zhang&amp;volume=38&amp;publication_year=2019&amp;pages=1907-1917&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0007">
<cite>

Elumalai, K.
, and Srinivasan A.. 2023. “Genetic Diversity and Population Structure of Sugarcane Introgressed Hybrids by SSR Markers.” 3 Biotech
13: 399.</cite> [<a href="https://doi.org/10.1007/s13205-023-03823-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10645997/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37974927/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=3%20Biotech&amp;title=Genetic%20Diversity%20and%20Population%20Structure%20of%20Sugarcane%20Introgressed%20Hybrids%20by%20SSR%20Markers&amp;author=K.%20Elumalai&amp;author=A.%20Srinivasan&amp;volume=13&amp;publication_year=2023&amp;pages=399&amp;pmid=37974927&amp;doi=10.1007/s13205-023-03823-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0008">
<cite>

Fang, J.
, Shi J., Zhang P., et al. 2024. “Potential Distribution Projections for Senegalia Senegal (L.) Britton Under Climate Change Scenarios.” Forests
15: 379.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Forests&amp;title=Potential%20Distribution%20Projections%20for%20Senegalia%20Senegal%20(L.)%20Britton%20Under%20Climate%20Change%20Scenarios&amp;author=J.%20Fang&amp;author=J.%20Shi&amp;author=P.%20Zhang&amp;volume=15&amp;publication_year=2024&amp;pages=379&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0009">
<cite>

Gong, Y.
, Jing P. F., Wei Y. K., Huang W. C., and Cui L. J.. 2014. “Potential Distribution of 
<em>Bletilla striata</em>
 (Orchidaceae)in China and Its Climate Characteristics.” Plant Diversity
36: 237–244.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Diversity&amp;title=Potential%20Distribution%20of%20Bletilla%20striata%20(Orchidaceae)in%20China%20and%20Its%20Climate%20Characteristics&amp;author=Y.%20Gong&amp;author=P.%20F.%20Jing&amp;author=Y.%20K.%20Wei&amp;author=W.%20C.%20Huang&amp;author=L.%20J.%20Cui&amp;volume=36&amp;publication_year=2014&amp;pages=237-244&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0010">
<cite>

Gou, K.
, Li Y., Qu Y., Li H., and Zeng R.. 2022. “Advances and Prospects of 
<em>Bletilla striata</em>
 Polysaccharide as Promising Multifunctional Biomedical Materials.” Materials and Design
223: 111198.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Materials%20and%20Design&amp;title=Advances%20and%20Prospects%20of%20Bletilla%20striata%20Polysaccharide%20as%20Promising%20Multifunctional%20Biomedical%20Materials&amp;author=K.%20Gou&amp;author=Y.%20Li&amp;author=Y.%20Qu&amp;author=H.%20Li&amp;author=R.%20Zeng&amp;volume=223&amp;publication_year=2022&amp;pages=111198&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0011">
<cite>

Griebeler, E. M.
, and Kadereit J. W.. 2023. “Decline of Tephroseris Helenitis in Hessia (Germany) Over the Last 120 Years: Modeling Implies the Gradual Disappearance of Its Temperature Niche for Flower Induction and Germination.” Ecology and Evolution
13: e10769.
</cite> [<a href="https://doi.org/10.1002/ece3.10769" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10700670/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38077516/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology%20and%20Evolution&amp;title=Decline%20of%20Tephroseris%20Helenitis%20in%20Hessia%20(Germany)%20Over%20the%20Last%20120%E2%80%89Years:%20Modeling%20Implies%20the%20Gradual%20Disappearance%20of%20Its%20Temperature%20Niche%20for%20Flower%20Induction%20and%20Germination&amp;author=E.%20M.%20Griebeler&amp;author=J.%20W.%20Kadereit&amp;volume=13&amp;publication_year=2023&amp;pages=e10769&amp;pmid=38077516&amp;doi=10.1002/ece3.10769&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0012"><cite>

Gui, T. Q.
, Zhang S. P., Wu Z. L., Li C. Y., Wei Y., and Miao G. D.. 2024. “Molecular Genetic Evaluation of Different Geographical Populations of 
<em>Bletilla striata</em>
 Rchb.f. Northern Horticulture.”
</cite></li>
<li id="ece372043-bib-0013">
<cite>

Guo, J.
, Cao W., Zhang Y., Gao Y., and Wang Y. Y.. 2019a. “Prediction of the Potential Distribution Area of 
<em>Solanum rostratum</em>
 in Northeast China.” Pratacultural Science
36: 2476–2484.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Pratacultural%20Science&amp;title=Prediction%20of%20the%20Potential%20Distribution%20Area%20of%20Solanum%20rostratum%20in%20Northeast%20China&amp;author=J.%20Guo&amp;author=W.%20Cao&amp;author=Y.%20Zhang&amp;author=Y.%20Gao&amp;author=Y.%20Y.%20Wang&amp;volume=36&amp;publication_year=2019a&amp;pages=2476-2484&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0014">
<cite>

Guo, J. L.
, Cao W. J., Li Z. M., Zhang Y. H., and Volis S.. 2019b. “Conservation Implications of Population Genetic Structure in a Threatened Orchid Cypripedium Tibeticum.” Plant Diversity
41: 13–18.
</cite> [<a href="https://doi.org/10.1016/j.pld.2018.12.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6412106/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30931413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Diversity&amp;title=Conservation%20Implications%20of%20Population%20Genetic%20Structure%20in%20a%20Threatened%20Orchid%20Cypripedium%20Tibeticum&amp;author=J.%20L.%20Guo&amp;author=W.%20J.%20Cao&amp;author=Z.%20M.%20Li&amp;author=Y.%20H.%20Zhang&amp;author=S.%20Volis&amp;volume=41&amp;publication_year=2019b&amp;pages=13-18&amp;pmid=30931413&amp;doi=10.1016/j.pld.2018.12.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0015">
<cite>

Guo, Y.
, Zhai L., Long H., et al. 2018. “Genetic Diversity of 
<em>Bletilla striata</em>
 Assessed by SCoT and IRAP Markers.” Hereditas
155: 35.
</cite> [<a href="https://doi.org/10.1186/s41065-018-0074-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6233545/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30459533/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hereditas&amp;title=Genetic%20Diversity%20of%20Bletilla%20striata%20Assessed%20by%20SCoT%20and%20IRAP%20Markers&amp;author=Y.%20Guo&amp;author=L.%20Zhai&amp;author=H.%20Long&amp;volume=155&amp;publication_year=2018&amp;pages=35&amp;pmid=30459533&amp;doi=10.1186/s41065-018-0074-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0016">
<cite>

Habibullah, M. S.
, Din B. H., Tan S. H., and Zahid H.. 2022. “Impact of Climate Change on Biodiversity Loss: Global Evidence.” Environmental Science and Pollution Research International
29: 1073–1086.
</cite> [<a href="https://doi.org/10.1007/s11356-021-15702-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34341937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environmental%20Science%20and%20Pollution%20Research%20International&amp;title=Impact%20of%20Climate%20Change%20on%20Biodiversity%20Loss:%20Global%20Evidence&amp;author=M.%20S.%20Habibullah&amp;author=B.%20H.%20Din&amp;author=S.%20H.%20Tan&amp;author=H.%20Zahid&amp;volume=29&amp;publication_year=2022&amp;pages=1073-1086&amp;pmid=34341937&amp;doi=10.1007/s11356-021-15702-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0017">
<cite>

Harrison, R. G.
, and Larson E. L.. 2014. “Hybridization, Introgression, and the Nature of Species Boundaries.” Journal of Heredity
105, no. Suppl 1: 795–809.
</cite> [<a href="https://doi.org/10.1093/jhered/esu033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25149255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Heredity&amp;title=Hybridization,%20Introgression,%20and%20the%20Nature%20of%20Species%20Boundaries&amp;author=R.%20G.%20Harrison&amp;author=E.%20L.%20Larson&amp;volume=105&amp;issue=Suppl%201&amp;publication_year=2014&amp;pages=795-809&amp;pmid=25149255&amp;doi=10.1093/jhered/esu033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0018">
<cite>

He, X.
, Liu L., Gu F., et al. 2024. “Exploration of the Anti‐Inflammatory, Analgesic, and Wound Healing Activities of 
<em>Bletilla Striata</em>
 Polysaccharide.” International Journal of Biological Macromolecules
261: 129874.
</cite> [<a href="https://doi.org/10.1016/j.ijbiomac.2024.129874" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38307430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Biological%20Macromolecules&amp;title=Exploration%20of%20the%20Anti%E2%80%90Inflammatory,%20Analgesic,%20and%20Wound%20Healing%20Activities%20of%20Bletilla%20Striata%20Polysaccharide&amp;author=X.%20He&amp;author=L.%20Liu&amp;author=F.%20Gu&amp;volume=261&amp;publication_year=2024&amp;pages=129874&amp;pmid=38307430&amp;doi=10.1016/j.ijbiomac.2024.129874&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0019">
<cite>

Hesselbarth, M. H.
, Nowosad J., De Flamingh A., et al. 2025. “Computational Methods in Landscape Ecology.” Current Landscape Ecology Reports
10: 1–18.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Landscape%20Ecology%20Reports&amp;title=Computational%20Methods%20in%20Landscape%20Ecology&amp;author=M.%20H.%20Hesselbarth&amp;author=J.%20Nowosad&amp;author=A.%20De%20Flamingh&amp;volume=10&amp;publication_year=2025&amp;pages=1-18&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0020">
<cite>

Hoffmann, A. A.
, and Sgrò C. M.. 2011. “Climate Change and Evolutionary Adaptation.” Nature
470: 479–485.
</cite> [<a href="https://doi.org/10.1038/nature09670" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21350480/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Climate%20Change%20and%20Evolutionary%20Adaptation&amp;author=A.%20A.%20Hoffmann&amp;author=C.%20M.%20Sgr%C3%B2&amp;volume=470&amp;publication_year=2011&amp;pages=479-485&amp;pmid=21350480&amp;doi=10.1038/nature09670&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0021">
<cite>

Jian, Z. X.
, Mei Z., Juan L., Zhu C. T., and Ping W.. 2019. “Investigation on the Resources of Bletilla Medicinal Plants in Sichuan.” Journal of Anhui Agricultural Sciences
47: 178–180.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Anhui%20Agricultural%20Sciences&amp;title=Investigation%20on%20the%20Resources%20of%20Bletilla%20Medicinal%20Plants%20in%20Sichuan&amp;author=Z.%20X.%20Jian&amp;author=Z.%20Mei&amp;author=L.%20Juan&amp;author=C.%20T.%20Zhu&amp;author=W.%20Ping&amp;volume=47&amp;publication_year=2019&amp;pages=178-180&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0022">
<cite>

Jiang, L.
, Lin M., Wang H., et al. 2022. “Haplotype‐Resolved Genome Assembly of 
<em>Bletilla striata</em>
 (Thunb.) Reichb.F. to Elucidate Medicinal Value.” Plant Journal
111: 1340–1353.</cite> [<a href="https://doi.org/10.1111/tpj.15892" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35785503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Journal&amp;title=Haplotype%E2%80%90Resolved%20Genome%20Assembly%20of%20Bletilla%20striata%20(Thunb.)%20Reichb.F.%20to%20Elucidate%20Medicinal%20Value&amp;author=L.%20Jiang&amp;author=M.%20Lin&amp;author=H.%20Wang&amp;volume=111&amp;publication_year=2022&amp;pages=1340-1353&amp;pmid=35785503&amp;doi=10.1111/tpj.15892&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0023">
<cite>

Jin, L.
, Guo F. Q., Fu M. M., Wu C., and Li X. B.. 2024. “Progress on Molecular Authentication of <em>Bletilla striata</em>
.” Molecular Plant Breeding
22: 446–452.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Plant%20Breeding&amp;title=Progress%20on%20Molecular%20Authentication%20of%20Bletilla%20striata&amp;author=L.%20Jin&amp;author=F.%20Q.%20Guo&amp;author=M.%20M.%20Fu&amp;author=C.%20Wu&amp;author=X.%20B.%20Li&amp;volume=22&amp;publication_year=2024&amp;pages=446-452&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0024">
<cite>

Li, J.
, Fan G., and He Y.. 2020. “Predicting the Current and Future Distribution of Three Coptis Herbs in China Under Climate Change Conditions, Using the MaxEnt Model and Chemical Analysis.” Science of the Total Environment
698: 134141.
</cite> [<a href="https://doi.org/10.1016/j.scitotenv.2019.134141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31505366/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20of%20the%20Total%20Environment&amp;title=Predicting%20the%20Current%20and%20Future%20Distribution%20of%20Three%20Coptis%20Herbs%20in%20China%20Under%20Climate%20Change%20Conditions,%20Using%20the%20MaxEnt%20Model%20and%20Chemical%20Analysis&amp;author=J.%20Li&amp;author=G.%20Fan&amp;author=Y.%20He&amp;volume=698&amp;publication_year=2020&amp;pages=134141&amp;pmid=31505366&amp;doi=10.1016/j.scitotenv.2019.134141&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0025">
<cite>

Li, X.
, Sun X., Yang Y., Yang S., Zhang J., and Sun C.. 2024. “The Appearance, Active Components, Antioxidant Activities, and Their Markers in Fibrous Roots of 
<em>Bletilla striata</em>

.” PLoS One
19: e0313318.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0313318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11554209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39527603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=The%20Appearance,%20Active%20Components,%20Antioxidant%20Activities,%20and%20Their%20Markers%20in%20Fibrous%20Roots%20of%20Bletilla%20striata&amp;author=X.%20Li&amp;author=X.%20Sun&amp;author=Y.%20Yang&amp;author=S.%20Yang&amp;author=J.%20Zhang&amp;volume=19&amp;publication_year=2024&amp;pages=e0313318&amp;pmid=39527603&amp;doi=10.1371/journal.pone.0313318&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0026">
<cite>

Li, Y. L.
, and Liu J. X.. 2018. “StructureSelector: A Web‐Based Software to Select and Visualize the Optimal Number of Clusters Using Multiple Methods.” Molecular Ecology Resources
18: 176–177.
</cite> [<a href="https://doi.org/10.1111/1755-0998.12719" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28921901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology%20Resources&amp;title=StructureSelector:%20A%20Web%E2%80%90Based%20Software%20to%20Select%20and%20Visualize%20the%20Optimal%20Number%20of%20Clusters%20Using%20Multiple%20Methods&amp;author=Y.%20L.%20Li&amp;author=J.%20X.%20Liu&amp;volume=18&amp;publication_year=2018&amp;pages=176-177&amp;pmid=28921901&amp;doi=10.1111/1755-0998.12719&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0027">
<cite>

Ling, L.
, Nie X. Y., Lu C. Y., Wen D. C., Huang M. J., and Zhao Z.. 2019. “Study on the Characteristics of <em>Bletilla striata</em> Germplasm Resources Based on the Analysis of Polysaccharide and Bletilla Gum and Mineral Elements.” Journal of Chinese Medicinal Materials
42: 1000–1006.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Chinese%20Medicinal%20Materials&amp;title=Study%20on%20the%20Characteristics%20of%20Bletilla%20striata%20Germplasm%20Resources%20Based%20on%20the%20Analysis%20of%20Polysaccharide%20and%20Bletilla%20Gum%20and%20Mineral%20Elements&amp;author=L.%20Ling&amp;author=X.%20Y.%20Nie&amp;author=C.%20Y.%20Lu&amp;author=D.%20C.%20Wen&amp;author=M.%20J.%20Huang&amp;volume=42&amp;publication_year=2019&amp;pages=1000-1006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0028">
<cite>

Liu, H.
, Cheng H., Xu J., et al. 2023a. “Genetic Diversity and Population Structure of Polygonatum Cyrtonema Hua in China Using SSR Markers.” PLoS One
18: e0290605.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0290605" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10470896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37651363/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Genetic%20Diversity%20and%20Population%20Structure%20of%20Polygonatum%20Cyrtonema%20Hua%20in%20China%20Using%20SSR%20Markers&amp;author=H.%20Liu&amp;author=H.%20Cheng&amp;author=J.%20Xu&amp;volume=18&amp;publication_year=2023a&amp;pages=e0290605&amp;pmid=37651363&amp;doi=10.1371/journal.pone.0290605&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0029">
<cite>

Liu, K.
, and Muse S. V.. 2005. “PowerMarker: An Integrated Analysis Environment for Genetic Marker Analysis.” Bioinformatics
21: 2128–2129.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bti282" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15705655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=PowerMarker:%20An%20Integrated%20Analysis%20Environment%20for%20Genetic%20Marker%20Analysis&amp;author=K.%20Liu&amp;author=S.%20V.%20Muse&amp;volume=21&amp;publication_year=2005&amp;pages=2128-2129&amp;pmid=15705655&amp;doi=10.1093/bioinformatics/bti282&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0030">
<cite>

Liu, Y.
, Fang Y., Zhang Y., et al. 2023b. “Prediction of the Potential Geographical Distribution of 
<em>Solanum sisymbriifolium</em>
 in China.” Plant Quarantine
37: 52–56.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Quarantine&amp;title=Prediction%20of%20the%20Potential%20Geographical%20Distribution%20of%20Solanum%20sisymbriifolium%20in%20China&amp;author=Y.%20Liu&amp;author=Y.%20Fang&amp;author=Y.%20Zhang&amp;volume=37&amp;publication_year=2023b&amp;pages=52-56&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0031">
<cite>

Mafakheri, M.
, Bakhshipour M., Omrani M., et al. 2022. “The Impact of Environmental and Climatic Variables on Genetic Diversity and Plant Functional Traits of the Endangered Tuberous Orchid (Orchis Mascula L.).” Scientific Reports
12: 19765.
</cite> [<a href="https://doi.org/10.1038/s41598-022-19864-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9672365/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36396718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=The%20Impact%20of%20Environmental%20and%20Climatic%20Variables%20on%20Genetic%20Diversity%20and%20Plant%20Functional%20Traits%20of%20the%20Endangered%20Tuberous%20Orchid%20(Orchis%20Mascula%20L.)&amp;author=M.%20Mafakheri&amp;author=M.%20Bakhshipour&amp;author=M.%20Omrani&amp;volume=12&amp;publication_year=2022&amp;pages=19765&amp;pmid=36396718&amp;doi=10.1038/s41598-022-19864-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0032">
<cite>

Moura, Y. A.
, Alves‐Pereira A., Da Silva C. C., Souza L. M., De Souza A. P., and Koehler S.. 2020. “Secondary Origin, Hybridization and Sexual Reproduction in a Diploid‐Tetraploid Contact Zone of the Facultatively Apomictic Orchid Zygopetalum Mackayi.” Plant Biology (Stuttgart, Germany)
22: 939–948.
</cite> [<a href="https://doi.org/10.1111/plb.13148" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32558140/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biology%20(Stuttgart,%20Germany)&amp;title=Secondary%20Origin,%20Hybridization%20and%20Sexual%20Reproduction%20in%20a%20Diploid%E2%80%90Tetraploid%20Contact%20Zone%20of%20the%20Facultatively%20Apomictic%20Orchid%20Zygopetalum%20Mackayi&amp;author=Y.%20A.%20Moura&amp;author=A.%20Alves%E2%80%90Pereira&amp;author=C.%20C.%20Da%20Silva&amp;author=L.%20M.%20Souza&amp;author=A.%20P.%20De%20Souza&amp;volume=22&amp;publication_year=2020&amp;pages=939-948&amp;pmid=32558140&amp;doi=10.1111/plb.13148&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0033">
<cite>

Niu, J.
, Zhao G., Mi Z., et al. 2020. “De Novo Sequencing of <em>Bletilla striata</em> (Orchidaceae) Transcriptome and Identification of Genes Involved in Polysaccharide Biosynthesis.” Genetics and Molecular Biology
43: e20190417.
</cite> [<a href="https://doi.org/10.1590/1678-4685-GMB-2019-0417" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7315133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32609279/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics%20and%20Molecular%20Biology&amp;title=De%20Novo%20Sequencing%20of%20Bletilla%20striata%20(Orchidaceae)%20Transcriptome%20and%20Identification%20of%20Genes%20Involved%20in%20Polysaccharide%20Biosynthesis&amp;author=J.%20Niu&amp;author=G.%20Zhao&amp;author=Z.%20Mi&amp;volume=43&amp;publication_year=2020&amp;pages=e20190417&amp;pmid=32609279&amp;doi=10.1590/1678-4685-GMB-2019-0417&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0034">
<cite>

Oliya, B. K.
, Maharjan L., and Pant B.. 2023. “Genetic Diversity and Population Structure Analysis of Paris Polyphylla Sm. Revealed by SSR Marker.” Heliyon
9: e18230.
</cite> [<a href="https://doi.org/10.1016/j.heliyon.2023.e18230" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10395474/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37539281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heliyon&amp;title=Genetic%20Diversity%20and%20Population%20Structure%20Analysis%20of%20Paris%20Polyphylla%20Sm.%20Revealed%20by%20SSR%20Marker&amp;author=B.%20K.%20Oliya&amp;author=L.%20Maharjan&amp;author=B.%20Pant&amp;volume=9&amp;publication_year=2023&amp;pages=e18230&amp;pmid=37539281&amp;doi=10.1016/j.heliyon.2023.e18230&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0035">
<cite>

Peakall, R.
, and Smouse P. E.. 2012. “GenAlEx 6.5: Genetic Analysis in Excel. Population Genetic Software for Teaching and Research—An Update.” Bioinformatics
28: 2537–2539.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts460" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3463245/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22820204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GenAlEx%206.5:%20Genetic%20Analysis%20in%20Excel.%20Population%20Genetic%20Software%20for%20Teaching%20and%20Research%E2%80%94An%20Update&amp;author=R.%20Peakall&amp;author=P.%20E.%20Smouse&amp;volume=28&amp;publication_year=2012&amp;pages=2537-2539&amp;pmid=22820204&amp;doi=10.1093/bioinformatics/bts460&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0036">
<cite>

Peterson, A.
, Papeş M., and Eaton M.. 2007. “Transferability and Model Evaluation in Ecological Niche Modeling: A Comparison of GARP and Maxent.” Ecography
30: 550–560.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Ecography&amp;title=Transferability%20and%20Model%20Evaluation%20in%20Ecological%20Niche%20Modeling:%20A%20Comparison%20of%20GARP%20and%20Maxent&amp;author=A.%20Peterson&amp;author=M.%20Pape%C5%9F&amp;author=M.%20Eaton&amp;volume=30&amp;publication_year=2007&amp;pages=550-560&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0037">
<cite>

Phillips, R. D.
, Reiter N., and Peakall R.. 2020. “Orchid Conservation: From Theory to Practice.” Annals of Botany
126: 345–362.
</cite> [<a href="https://doi.org/10.1093/aob/mcaa093" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7424752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32407498/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20of%20Botany&amp;title=Orchid%20Conservation:%20From%20Theory%20to%20Practice&amp;author=R.%20D.%20Phillips&amp;author=N.%20Reiter&amp;author=R.%20Peakall&amp;volume=126&amp;publication_year=2020&amp;pages=345-362&amp;pmid=32407498&amp;doi=10.1093/aob/mcaa093&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0038">
<cite>

Phillips, S. J.
, Anderson R. P., and Schapire R. E.. 2006. “Maximum Entropy Modeling of Species Geographic Distributions.” Ecological Modelling
190: 231–259.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Ecological%20Modelling&amp;title=Maximum%20Entropy%20Modeling%20of%20Species%20Geographic%20Distributions&amp;author=S.%20J.%20Phillips&amp;author=R.%20P.%20Anderson&amp;author=R.%20E.%20Schapire&amp;volume=190&amp;publication_year=2006&amp;pages=231-259&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0039">
<cite>

Phillips, S. J.
, and Dudík M.. 2008. “Modeling of Species Distributions With Maxent: New Extensions and a Comprehensive Evaluation.” Ecography
31: 161–175.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Ecography&amp;title=Modeling%20of%20Species%20Distributions%20With%20Maxent:%20New%20Extensions%20and%20a%20Comprehensive%20Evaluation&amp;author=S.%20J.%20Phillips&amp;author=M.%20Dud%C3%ADk&amp;volume=31&amp;publication_year=2008&amp;pages=161-175&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0040">
<cite>

Pica, A.
, Vela D., and Magrini S.. 2024. “Forest Orchids Under Future Climate Scenarios: Habitat Suitability Modelling to Inform Conservation Strategies.” Plants (Basel)
13: 1810.
</cite> [<a href="https://doi.org/10.3390/plants13131810" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11243989/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38999650/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants%20(Basel)&amp;title=Forest%20Orchids%20Under%20Future%20Climate%20Scenarios:%20Habitat%20Suitability%20Modelling%20to%20Inform%20Conservation%20Strategies&amp;author=A.%20Pica&amp;author=D.%20Vela&amp;author=S.%20Magrini&amp;volume=13&amp;publication_year=2024&amp;pages=1810&amp;pmid=38999650&amp;doi=10.3390/plants13131810&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0041">
<cite>

Pritchard, J. K.
, Stephens M., and Donnelly P.. 2000. “Inference of Population Structure Using Multilocus Genotype Data.” Genetics
155: 945–959.
</cite> [<a href="https://doi.org/10.1093/genetics/155.2.945" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1461096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10835412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Inference%20of%20Population%20Structure%20Using%20Multilocus%20Genotype%20Data&amp;author=J.%20K.%20Pritchard&amp;author=M.%20Stephens&amp;author=P.%20Donnelly&amp;volume=155&amp;publication_year=2000&amp;pages=945-959&amp;pmid=10835412&amp;doi=10.1093/genetics/155.2.945&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0042">
<cite>

Qin, Z. X.
, Tao Z., Hong X. C., et al. 2020. “Quality Comprehensive Evaluation of 
<em>Bletilla striata</em>
 With Different Producing Areas From Guizhou.” Journal of Chinese Medicinal Materials
43: 368–373.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Chinese%20Medicinal%20Materials&amp;title=Quality%20Comprehensive%20Evaluation%20of%20Bletilla%20striata%20With%20Different%20Producing%20Areas%20From%20Guizhou&amp;author=Z.%20X.%20Qin&amp;author=Z.%20Tao&amp;author=X.%20C.%20Hong&amp;volume=43&amp;publication_year=2020&amp;pages=368-373&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0044">
<cite>

Radosavljevic, A.
, and Anderson R. P.. 2014. “Making Better Maxent Models of Species Distributions: Complexity, Overfitting and Evaluation.” Journal of Biogeography
41: 629–643.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biogeography&amp;title=Making%20Better%20Maxent%20Models%20of%20Species%20Distributions:%20Complexity,%20Overfitting%20and%20Evaluation&amp;author=A.%20Radosavljevic&amp;author=R.%20P.%20Anderson&amp;volume=41&amp;publication_year=2014&amp;pages=629-643&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0045">
<cite>

Remya, K.
, Ramachandran A., and Jayakumar S.. 2015. “Predicting the Current and Future Suitable Habitat Distribution of Myristica Dactyloides Gaertn. Using MaxEnt Model in the Eastern Ghats, India.” Ecological Engineering
82: 184–188.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Ecological%20Engineering&amp;title=Predicting%20the%20Current%20and%20Future%20Suitable%20Habitat%20Distribution%20of%20Myristica%20Dactyloides%20Gaertn.%20Using%20MaxEnt%20Model%20in%20the%20Eastern%20Ghats,%20India&amp;author=K.%20Remya&amp;author=A.%20Ramachandran&amp;author=S.%20Jayakumar&amp;volume=82&amp;publication_year=2015&amp;pages=184-188&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0046">
<cite>

Rousset, F.

1997. “Genetic Differentiation and Estimation of Gene Flow From F‐Statistics Under Isolation by Distance.” Genetics
145: 1219–1228.
</cite> [<a href="https://doi.org/10.1093/genetics/145.4.1219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1207888/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9093870/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Genetic%20Differentiation%20and%20Estimation%20of%20Gene%20Flow%20From%20F%E2%80%90Statistics%20Under%20Isolation%20by%20Distance&amp;author=F.%20Rousset&amp;volume=145&amp;publication_year=1997&amp;pages=1219-1228&amp;pmid=9093870&amp;doi=10.1093/genetics/145.4.1219&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0047">
<cite>

Serna‐Sánchez, M. A.
, Karremans A. P., Bogarín D., and Fuchs E. J.. 2025. “Contrasting Clonal and Population Genetic Structure in Two Endangered Costa Rican Vanilla Species of Commercial Interest.” Scientific Reports
15: 19295.
</cite> [<a href="https://doi.org/10.1038/s41598-025-04101-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12130201/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40456827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Contrasting%20Clonal%20and%20Population%20Genetic%20Structure%20in%20Two%20Endangered%20Costa%20Rican%20Vanilla%20Species%20of%20Commercial%20Interest&amp;author=M.%20A.%20Serna%E2%80%90S%C3%A1nchez&amp;author=A.%20P.%20Karremans&amp;author=D.%20Bogar%C3%ADn&amp;author=E.%20J.%20Fuchs&amp;volume=15&amp;publication_year=2025&amp;pages=19295&amp;pmid=40456827&amp;doi=10.1038/s41598-025-04101-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0048">
<cite>

Slatkin, M.

1987. “Gene Flow and the Geographic Structure of Natural Populations.” Science
236: 787–792.
</cite> [<a href="https://doi.org/10.1126/science.3576198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3576198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Gene%20Flow%20and%20the%20Geographic%20Structure%20of%20Natural%20Populations&amp;author=M.%20Slatkin&amp;volume=236&amp;publication_year=1987&amp;pages=787-792&amp;pmid=3576198&amp;doi=10.1126/science.3576198&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0049">
<cite>

Su, T.
, Qiu B., and Li Y.. 2014. “Resource Investigation and Market Evaluation of Bletilla Produced in Yunnan.” Chinese Wild Plant Resources
33: 49–52.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Chinese%20Wild%20Plant%20Resources&amp;title=Resource%20Investigation%20and%20Market%20Evaluation%20of%20Bletilla%20Produced%20in%20Yunnan&amp;author=T.%20Su&amp;author=B.%20Qiu&amp;author=Y.%20Li&amp;volume=33&amp;publication_year=2014&amp;pages=49-52&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0050">
<cite>

Sun, Y. L.

2015. Studies on Pseudobulb Induction, Genetic Resource Identification, and Evaluation of 
<em>Bletilla striata</em>

. Nan Jing Normal University.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Studies%20on%20Pseudobulb%20Induction,%20Genetic%20Resource%20Identification,%20and%20Evaluation%20of%20Bletilla%20striata&amp;author=Y.%20L.%20Sun&amp;publication_year=2015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0051">
<cite>

Sun, Y. L.
, Hou B. W., Geng L. X., et al. 2016. “Evaluation of Genetic Diversity and Population Structure of 
<em>Bletilla striata</em>
 Based on SRAP Markers.” Acta Pharmaceutica Sinica
51: 147–152.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/27405177/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Pharmaceutica%20Sinica&amp;title=Evaluation%20of%20Genetic%20Diversity%20and%20Population%20Structure%20of%20Bletilla%20striata%20Based%20on%20SRAP%20Markers&amp;author=Y.%20L.%20Sun&amp;author=B.%20W.%20Hou&amp;author=L.%20X.%20Geng&amp;volume=51&amp;publication_year=2016&amp;pages=147-152&amp;pmid=27405177&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0052">
<cite>

Sunoj, V. S. J.
, Shroyer K. J., Jagadish S. V. K., and Prasad P. V. V.. 2016. “Diurnal Temperature Amplitude Alters Physiological and Growth Response of Maize (
<em>Zea mays</em>
 L.) During the Vegetative Stage.” Environmental and Experimental Botany
130: 113–121.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Environmental%20and%20Experimental%20Botany&amp;title=Diurnal%20Temperature%20Amplitude%20Alters%20Physiological%20and%20Growth%20Response%20of%20Maize%20(%20Zea%20mays%20L.)%20During%20the%20Vegetative%20Stage&amp;author=V.%20S.%20J.%20Sunoj&amp;author=K.%20J.%20Shroyer&amp;author=S.%20V.%20K.%20Jagadish&amp;author=P.%20V.%20V.%20Prasad&amp;volume=130&amp;publication_year=2016&amp;pages=113-121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0053">
<cite>

Tautz, D.
, and Renz M.. 1984. “Simple Sequences Are Ubiquitous Repetitive Components of Eukaryotic Genomes.” Nucleic Acids Research
12: 4127–4138.
</cite> [<a href="https://doi.org/10.1093/nar/12.10.4127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC318821/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6328411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Simple%20Sequences%20Are%20Ubiquitous%20Repetitive%20Components%20of%20Eukaryotic%20Genomes&amp;author=D.%20Tautz&amp;author=M.%20Renz&amp;volume=12&amp;publication_year=1984&amp;pages=4127-4138&amp;pmid=6328411&amp;doi=10.1093/nar/12.10.4127&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0054">
<cite>

Tian, H. Z.
, Han L. X., Zhang J. L., et al. 2018. “Genetic Diversity in the Endangered Terrestrial Orchid Cypripedium Japonicum in East Asia: Insights Into Population History and Implications for Conservation.” Scientific Reports
8: 6467.
</cite> [<a href="https://doi.org/10.1038/s41598-018-24912-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5915404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29691494/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Genetic%20Diversity%20in%20the%20Endangered%20Terrestrial%20Orchid%20Cypripedium%20Japonicum%20in%20East%20Asia:%20Insights%20Into%20Population%20History%20and%20Implications%20for%20Conservation&amp;author=H.%20Z.%20Tian&amp;author=L.%20X.%20Han&amp;author=J.%20L.%20Zhang&amp;volume=8&amp;publication_year=2018&amp;pages=6467&amp;pmid=29691494&amp;doi=10.1038/s41598-018-24912-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0056">
<cite>

Urban, M. C.
, Bocedi G., Hendry A. P., et al. 2016. “Improving the Forecast for Biodiversity Under Climate Change.” Science
353: aad8466.
</cite> [<a href="https://doi.org/10.1126/science.aad8466" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27609898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Improving%20the%20Forecast%20for%20Biodiversity%20Under%20Climate%20Change&amp;author=M.%20C.%20Urban&amp;author=G.%20Bocedi&amp;author=A.%20P.%20Hendry&amp;volume=353&amp;publication_year=2016&amp;pages=aad8466&amp;pmid=27609898&amp;doi=10.1126/science.aad8466&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0057">
<cite>

Wan, G. Z.
, Li Q. Q., Jin L., and Chen J.. 2024. “Integrated Approach to Predicting Habitat Suitability and Evaluating Quality Variations of Notopterygium Franchetii Under Climate Change.” Scientific Reports
14: 26927.
</cite> [<a href="https://doi.org/10.1038/s41598-024-77824-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11541729/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39505945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Integrated%20Approach%20to%20Predicting%20Habitat%20Suitability%20and%20Evaluating%20Quality%20Variations%20of%20Notopterygium%20Franchetii%20Under%20Climate%20Change&amp;author=G.%20Z.%20Wan&amp;author=Q.%20Q.%20Li&amp;author=L.%20Jin&amp;author=J.%20Chen&amp;volume=14&amp;publication_year=2024&amp;pages=26927&amp;pmid=39505945&amp;doi=10.1038/s41598-024-77824-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0059">
<cite>

Wang, W. H.
, Peng S. L., Shu H., Fu X., and Ye X. Y.. 2024. “Potential Adaptive Habitats for the Narrowly Distributed and Rare Bamboo Species Chimonobambusa Tumidissinoda J. R. Xue &amp; T. P. Yi ex Ohrnb. Under Future Climate Change in China.” Ecology and Evolution
14: e70314.
</cite> [<a href="https://doi.org/10.1002/ece3.70314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11402478/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39279795/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology%20and%20Evolution&amp;title=Potential%20Adaptive%20Habitats%20for%20the%20Narrowly%20Distributed%20and%20Rare%20Bamboo%20Species%20Chimonobambusa%20Tumidissinoda%20J.%20R.%20Xue%20&amp;%20T.%20P.%20Yi%20ex%20Ohrnb.%20Under%20Future%20Climate%20Change%20in%20China&amp;author=W.%20H.%20Wang&amp;author=S.%20L.%20Peng&amp;author=H.%20Shu&amp;author=X.%20Fu&amp;author=X.%20Y.%20Ye&amp;volume=14&amp;publication_year=2024&amp;pages=e70314&amp;pmid=39279795&amp;doi=10.1002/ece3.70314&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0060"><cite>

Wang, X. M.
, Wu M. K., and Luo X. Q.. 2011. “Present Situation and Prospect of 
<em>Bletilla striata</em>
.”
</cite></li>
<li id="ece372043-bib-0061">
<cite>

Wang, Z.
, Xu D., Liao W., Xu Y., and Zhuo Z.. 2023. “Predicting the Current and Future Distributions of 
<em>Frankliniella occidentalis</em>
 (Pergande) Based on the MaxEnt Species Distribution Model.” Insects
14: 458.
</cite> [<a href="https://doi.org/10.3390/insects14050458" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10231115/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37233086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Insects&amp;title=Predicting%20the%20Current%20and%20Future%20Distributions%20of%20Frankliniella%20occidentalis%20(Pergande)%20Based%20on%20the%20MaxEnt%20Species%20Distribution%20Model&amp;author=Z.%20Wang&amp;author=D.%20Xu&amp;author=W.%20Liao&amp;author=Y.%20Xu&amp;author=Z.%20Zhuo&amp;volume=14&amp;publication_year=2023&amp;pages=458&amp;pmid=37233086&amp;doi=10.3390/insects14050458&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0062">
<cite>

Wright, S.

1922. “Coefficients of Inbreeding and Relationship.” American Naturalist
56: 330–338.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Naturalist&amp;title=Coefficients%20of%20Inbreeding%20and%20Relationship&amp;author=S.%20Wright&amp;volume=56&amp;publication_year=1922&amp;pages=330-338&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0063">
<cite>

Wright, S.

1931. “Evolution in Mendelian Populations.” Genetics
16: 97–159.
</cite> [<a href="https://doi.org/10.1093/genetics/16.2.97" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1201091/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17246615/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Evolution%20in%20Mendelian%20Populations&amp;author=S.%20Wright&amp;volume=16&amp;publication_year=1931&amp;pages=97-159&amp;pmid=17246615&amp;doi=10.1093/genetics/16.2.97&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0064">
<cite>

Wu, F.
, Cai G., Xi P., Guo Y., Xu M., and Li A.. 2024a. “Genetic Diversity Analysis and Fingerprint Construction for 87 Passionfruit (<em>Passiflora</em> Spp.) Germplasm Accessions on the Basis of SSR Fluorescence Markers.” International Journal of Molecular Sciences
25: 10815.
</cite> [<a href="https://doi.org/10.3390/ijms251910815" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11476748/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39409142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Genetic%20Diversity%20Analysis%20and%20Fingerprint%20Construction%20for%2087%20Passionfruit%20(Passiflora%20Spp.)%20Germplasm%20Accessions%20on%20the%20Basis%20of%20SSR%20Fluorescence%20Markers&amp;author=F.%20Wu&amp;author=G.%20Cai&amp;author=P.%20Xi&amp;author=Y.%20Guo&amp;author=M.%20Xu&amp;volume=25&amp;publication_year=2024a&amp;pages=10815&amp;pmid=39409142&amp;doi=10.3390/ijms251910815&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0065">
<cite>

Wu, J.
, Yang Q., Zhao W., et al. 2024b. “Assessment of Population Genetic Diversity of Medicinal Meconopsis Integrifolia (Maxim.) Franch. Using Newly Developed SSR Markers.” Plants
13: 2561.
</cite> [<a href="https://doi.org/10.3390/plants13182561" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11435270/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39339536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plants&amp;title=Assessment%20of%20Population%20Genetic%20Diversity%20of%20Medicinal%20Meconopsis%20Integrifolia%20(Maxim.)%20Franch.%20Using%20Newly%20Developed%20SSR%20Markers&amp;author=J.%20Wu&amp;author=Q.%20Yang&amp;author=W.%20Zhao&amp;volume=13&amp;publication_year=2024b&amp;pages=2561&amp;pmid=39339536&amp;doi=10.3390/plants13182561&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0066">
<cite>

Xu, D.
, Chen H., Aci M., et al. 2018. “De Novo Assembly, Characterization and Development of EST‐SSRs From 
<em>Bletilla striata</em>
 Transcriptomes Profiled Throughout the Whole Growing Period.” PLoS One
13: e0205954.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0205954" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6203367/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30365506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=De%20Novo%20Assembly,%20Characterization%20and%20Development%20of%20EST%E2%80%90SSRs%20From%20Bletilla%20striata%20Transcriptomes%20Profiled%20Throughout%20the%20Whole%20Growing%20Period&amp;author=D.%20Xu&amp;author=H.%20Chen&amp;author=M.%20Aci&amp;volume=13&amp;publication_year=2018&amp;pages=e0205954&amp;pmid=30365506&amp;doi=10.1371/journal.pone.0205954&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0067">
<cite>

Yan, X.
, Wang S., Duan Y., Han J., Huang D., and Zhou J.. 2021. “Current and Future Distribution of the Deciduous Shrub 
<em>Hydrangea macrophylla</em>
 in China Estimated by MaxEnt.” Ecology and Evolution
11: 16099–16112.
</cite> [<a href="https://doi.org/10.1002/ece3.8288" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8601876/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34824814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology%20and%20Evolution&amp;title=Current%20and%20Future%20Distribution%20of%20the%20Deciduous%20Shrub%20Hydrangea%20macrophylla%20in%20China%20Estimated%20by%20MaxEnt&amp;author=X.%20Yan&amp;author=S.%20Wang&amp;author=Y.%20Duan&amp;author=J.%20Han&amp;author=D.%20Huang&amp;volume=11&amp;publication_year=2021&amp;pages=16099-16112&amp;pmid=34824814&amp;doi=10.1002/ece3.8288&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0068">
<cite>

Yang, W.
, Bai Z., Wang F., et al. 2022. “Analysis of the Genetic Diversity and Population Structure of Monochasma Savatieri Franch. Ex Maxim Using Novel EST‐SSR Markers.” BMC Genomics
23: 597.
</cite> [<a href="https://doi.org/10.1186/s12864-022-08832-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9382759/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35974306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Analysis%20of%20the%20Genetic%20Diversity%20and%20Population%20Structure%20of%20Monochasma%20Savatieri%20Franch.%20Ex%20Maxim%20Using%20Novel%20EST%E2%80%90SSR%20Markers&amp;author=W.%20Yang&amp;author=Z.%20Bai&amp;author=F.%20Wang&amp;volume=23&amp;publication_year=2022&amp;pages=597&amp;pmid=35974306&amp;doi=10.1186/s12864-022-08832-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0069">
<cite>

Ye, C.
, An M., Shi J., Liu F., and Zhang Y.. 2022. “Spatial Distribution and Its Limiting Environmental Factors of Native Orchid Species Diversity in the Beipan River Basin of Guizhou Province, China.” Ecology and Evolution
12: e9470.
</cite> [<a href="https://doi.org/10.1002/ece3.9470" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9663322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36398196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology%20and%20Evolution&amp;title=Spatial%20Distribution%20and%20Its%20Limiting%20Environmental%20Factors%20of%20Native%20Orchid%20Species%20Diversity%20in%20the%20Beipan%20River%20Basin%20of%20Guizhou%20Province,%20China&amp;author=C.%20Ye&amp;author=M.%20An&amp;author=J.%20Shi&amp;author=F.%20Liu&amp;author=Y.%20Zhang&amp;volume=12&amp;publication_year=2022&amp;pages=e9470&amp;pmid=36398196&amp;doi=10.1002/ece3.9470&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0070">
<cite>

Zhang, J.
, Zhou D., Chen W., et al. 2024. “Comparison of the Chloroplast Genomics of Nine Endangered Habenaria Species and Phylogenetic Analysis.” BMC Plant Biology
24: 1046.
</cite> [<a href="https://doi.org/10.1186/s12870-024-05766-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11536600/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39497089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Plant%20Biology&amp;title=Comparison%20of%20the%20Chloroplast%20Genomics%20of%20Nine%20Endangered%20Habenaria%20Species%20and%20Phylogenetic%20Analysis&amp;author=J.%20Zhang&amp;author=D.%20Zhou&amp;author=W.%20Chen&amp;volume=24&amp;publication_year=2024&amp;pages=1046&amp;pmid=39497089&amp;doi=10.1186/s12870-024-05766-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0071">
<cite>

Zhang, M.
, Jiang D., Yang M., et al. 2021. “Influence of the Environment on the Distribution and Quality of Gentiana Dahurica Fisch.” Frontiers in Plant Science
12: 706822.
</cite> [<a href="https://doi.org/10.3389/fpls.2021.706822" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8503573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34646283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Influence%20of%20the%20Environment%20on%20the%20Distribution%20and%20Quality%20of%20Gentiana%20Dahurica%20Fisch&amp;author=M.%20Zhang&amp;author=D.%20Jiang&amp;author=M.%20Yang&amp;volume=12&amp;publication_year=2021&amp;pages=706822&amp;pmid=34646283&amp;doi=10.3389/fpls.2021.706822&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0072">
<cite>

Zhao, Y.
, Wen Y., Zhang W., et al. 2023. “Distribution Pattern and Change Prediction of Phellodendron Habitat in China Under Climate Change.” Ecology and Evolution
13: e10374.
</cite> [<a href="https://doi.org/10.1002/ece3.10374" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10450841/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37636866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology%20and%20Evolution&amp;title=Distribution%20Pattern%20and%20Change%20Prediction%20of%20Phellodendron%20Habitat%20in%20China%20Under%20Climate%20Change&amp;author=Y.%20Zhao&amp;author=Y.%20Wen&amp;author=W.%20Zhang&amp;volume=13&amp;publication_year=2023&amp;pages=e10374&amp;pmid=37636866&amp;doi=10.1002/ece3.10374&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0073">
<cite>

Zhou, T.
, Jiang W. K., Li L., and Wei S. H.. 2010. “Investigation of Wild <em>Bletilla Striata</em> Resources and Evaluation of Market Utilization in Guizhou China.” Journal of Guizhou University of Traditional Chinese Medicine
32: 28–30.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Guizhou%20University%20of%20Traditional%20Chinese%20Medicine&amp;title=Investigation%20of%20Wild%20Bletilla%20Striata%20Resources%20and%20Evaluation%20of%20Market%20Utilization%20in%20Guizhou%20China&amp;author=T.%20Zhou&amp;author=W.%20K.%20Jiang&amp;author=L.%20Li&amp;author=S.%20H.%20Wei&amp;volume=32&amp;publication_year=2010&amp;pages=28-30&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0074">
<cite>

Zhou, Y.
, Li C. Z., Zheng T., et al. 2022. “Physical Properties，Enzyme Activity and Correlation of Soil for Wild 
<em>Bletilla striata</em>
 in Guizhou.” Southwest China Journal of Agricultural Sciences
35: 336–342.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Southwest%20China%20Journal%20of%20Agricultural%20Sciences&amp;title=Physical%20Properties%EF%BC%8CEnzyme%20Activity%20and%20Correlation%20of%20Soil%20for%20Wild%20Bletilla%20striata%20in%20Guizhou&amp;author=Y.%20Zhou&amp;author=C.%20Z.%20Li&amp;author=T.%20Zheng&amp;volume=35&amp;publication_year=2022&amp;pages=336-342&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0075">
<cite>

Zhou, Y.
, Tan J., Huang L., Ye Y., and Xu Y.. 2024. “Assessing Genetic Diversity in Endangered Plant Orchidantha Chinensis: Chloroplast Genome Assembly and Simple Sequence Repeat Marker‐Based Evaluation.” International Journal of Molecular Sciences
25: 11137.
</cite> [<a href="https://doi.org/10.3390/ijms252011137" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11508217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39456923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=International%20Journal%20of%20Molecular%20Sciences&amp;title=Assessing%20Genetic%20Diversity%20in%20Endangered%20Plant%20Orchidantha%20Chinensis:%20Chloroplast%20Genome%20Assembly%20and%20Simple%20Sequence%20Repeat%20Marker%E2%80%90Based%20Evaluation&amp;author=Y.%20Zhou&amp;author=J.%20Tan&amp;author=L.%20Huang&amp;author=Y.%20Ye&amp;author=Y.%20Xu&amp;volume=25&amp;publication_year=2024&amp;pages=11137&amp;pmid=39456923&amp;doi=10.3390/ijms252011137&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece372043-bib-0076">
<cite>

Zhu, Y.
, Ma T., Lin Y., Peng Y., Huang Y., and Jiang J.. 2023. “SSR Molecular Marker Developments and Genetic Diversity Analysis of Zanthoxylum Nitidum (Roxb.) DC.” Scientific Reports
13: 20767.
</cite> [<a href="https://doi.org/10.1038/s41598-023-48022-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10679188/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38008750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=SSR%20Molecular%20Marker%20Developments%20and%20Genetic%20Diversity%20Analysis%20of%20Zanthoxylum%20Nitidum%20(Roxb.)%20DC&amp;author=Y.%20Zhu&amp;author=T.%20Ma&amp;author=Y.%20Lin&amp;author=Y.%20Peng&amp;author=Y.%20Huang&amp;volume=13&amp;publication_year=2023&amp;pages=20767&amp;pmid=38008750&amp;doi=10.1038/s41598-023-48022-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>Appendix S1:</strong> ece372043‐sup‐0001‐AppendixS1.docx.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12362183/bin/ECE3-15-e72043-s003.docx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e72043-s003.docx</a><sup> (1MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>
<strong>Data S1:</strong> ece372043‐sup‐0002‐DataS1.xlsx.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12362183/bin/ECE3-15-e72043-s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e72043-s001.xlsx</a><sup> (14.6KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>
<strong>Data S2:</strong> ece372043‐sup‐0003‐DataS2.xlsx.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12362183/bin/ECE3-15-e72043-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e72043-s002.xlsx</a><sup> (19.7KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data and materials supporting this study are included in the article and Appendix <a href="#ece372043-supitem-0001" class="usa-link">S1</a>, Data <a href="#ece372043-supitem-0002" class="usa-link">S1</a>, and Data <a href="#ece372043-supitem-0003" class="usa-link">S2</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Ecology and Evolution are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/ece3.72043"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ECE3-15-e72043.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12362183/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12362183/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12362183%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12362183/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12362183/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12362183/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40837537/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12362183/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40837537/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12362183/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12362183/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="30HrjI6BGTIXlkpWQ1YX9BryEEOpbOaLmmNbUW1m87PzJLxNfkdGJQU0d9kqJdfl">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
