
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Evaluation of familial phenotype deviation to measure the impact of de novo mutations in autism - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152544D18AF30D330544D10008DF086F.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="genmed">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366145/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Genome Medicine">
<meta name="citation_title" content="Evaluation of familial phenotype deviation to measure the impact of de novo mutations in autism">
<meta name="citation_author" content="Soo-Whee Kim">
<meta name="citation_author_institution" content="Department of Integrated Biomedical and Life Science, Korea University, Seoul, Republic of Korea">
<meta name="citation_author_institution" content="L-HOPE Program for Community-Based Total Learning Health Systems, Korea University, Seoul, Republic of Korea">
<meta name="citation_author" content="Hyeji Lee">
<meta name="citation_author_institution" content="Department of Integrated Biomedical and Life Science, Korea University, Seoul, Republic of Korea">
<meta name="citation_author_institution" content="L-HOPE Program for Community-Based Total Learning Health Systems, Korea University, Seoul, Republic of Korea">
<meta name="citation_author" content="Da Yea Song">
<meta name="citation_author_institution" content="Department of Psychiatry, Seoul National University Bundang Hospital, Seongnam, Republic of Korea">
<meta name="citation_author_institution" content="Department of Psychiatry, Seoul National University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author" content="Gang-Hee Lee">
<meta name="citation_author_institution" content="Department of Integrated Biomedical and Life Science, Korea University, Seoul, Republic of Korea">
<meta name="citation_author_institution" content="L-HOPE Program for Community-Based Total Learning Health Systems, Korea University, Seoul, Republic of Korea">
<meta name="citation_author" content="Jae Hyun Han">
<meta name="citation_author_institution" content="Department of Psychiatry, Seoul National University Bundang Hospital, Seongnam, Republic of Korea">
<meta name="citation_author_institution" content="Department of Psychiatry, College of Medicine, Soonchunhyang University Cheonan Hospital, Cheonan, Republic of Korea">
<meta name="citation_author" content="Jee Won Lee">
<meta name="citation_author_institution" content="Department of Psychiatry, Soonchunhyang University College of Medicine, Cheonan, Republic of Korea">
<meta name="citation_author" content="Hee Jung Byun">
<meta name="citation_author_institution" content="Department of Psychiatry, Seoul Metropolitan Children’s Hospital, Seoul, Republic of Korea">
<meta name="citation_author" content="Ji Hyun Son">
<meta name="citation_author_institution" content="Department of Psychiatry, Seoul Metropolitan Children’s Hospital, Seoul, Republic of Korea">
<meta name="citation_author" content="Ye Rim Kim">
<meta name="citation_author_institution" content="Department of Psychiatry, Seoul National University Bundang Hospital, Seongnam, Republic of Korea">
<meta name="citation_author_institution" content="Department of Psychiatry, Seoul National University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author" content="Yoojeong Lee">
<meta name="citation_author_institution" content="Department of Psychiatry, Seoul National University Bundang Hospital, Seongnam, Republic of Korea">
<meta name="citation_author" content="Eunjoon Kim">
<meta name="citation_author_institution" content="Department of Biological Sciences, Korea Advanced Institute of Science and Technology, Daejeon, Republic of Korea">
<meta name="citation_author_institution" content="Center for Synaptic Brain Dysfunctions, Institute for Basic Science, Daejeon, Republic of Korea">
<meta name="citation_author" content="Donna M Werling">
<meta name="citation_author_institution" content="Laboratory of Genetics, University of Wisconsin-Madison, Madison, WI USA">
<meta name="citation_author" content="So Hyun Kim">
<meta name="citation_author_institution" content="Department of Psychology, Korea University, Seoul, Republic of Korea">
<meta name="citation_author" content="Stephan J Sanders">
<meta name="citation_author_institution" content="Institute of Developmental and Regenerative Medicine, Department of Pediatrics, University of Oxford, Oxford, UK">
<meta name="citation_author_institution" content="Department of Psychiatry and Behavioral Sciences, UCSF Weill Institute for Neurosciences, University of California, San Francisco, USA">
<meta name="citation_author" content="Hee Jeong Yoo">
<meta name="citation_author_institution" content="Department of Psychiatry, Seoul National University Bundang Hospital, Seongnam, Republic of Korea">
<meta name="citation_author_institution" content="Department of Psychiatry, Seoul National University College of Medicine, Seoul, Republic of Korea">
<meta name="citation_author" content="Joon-Yong An">
<meta name="citation_author_institution" content="Department of Integrated Biomedical and Life Science, Korea University, Seoul, Republic of Korea">
<meta name="citation_author_institution" content="L-HOPE Program for Community-Based Total Learning Health Systems, Korea University, Seoul, Republic of Korea">
<meta name="citation_author_institution" content="School of Biosystem and Biomedical Science, College of Health Science, Korea University, Seoul, Republic of Korea">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="17">
<meta name="citation_firstpage" content="93">
<meta name="citation_doi" content="10.1186/s13073-025-01532-7">
<meta name="citation_pmid" content="40836247">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366145/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366145/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366145/pdf/13073_2025_Article_1532.pdf">
<meta name="description" content="The phenotypic outcomes of de novo variants (DNVs) in autism spectrum disorder (ASD) exhibit wide variability. To date, no study has comprehensively estimated DNV effects accounting for familial phenotypic background. To evaluate DNV effects in a ...">
<meta name="og:title" content="Evaluation of familial phenotype deviation to measure the impact of de novo mutations in autism">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The phenotypic outcomes of de novo variants (DNVs) in autism spectrum disorder (ASD) exhibit wide variability. To date, no study has comprehensively estimated DNV effects accounting for familial phenotypic background. To evaluate DNV effects in a ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366145/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366145">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s13073-025-01532-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/13073_2025_Article_1532.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366145%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366145/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366145/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366145/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-genmed.png" alt="Genome Medicine logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Genome Medicine" title="Link to Genome Medicine" shape="default" href="https://genomemedicine.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Genome Med</button></div>. 2025 Aug 20;17:93. doi: <a href="https://doi.org/10.1186/s13073-025-01532-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s13073-025-01532-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Genome%20Med%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Genome%20Med%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Genome%20Med%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Genome%20Med%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Evaluation of familial phenotype deviation to measure the impact of de novo mutations in autism</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20SW%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Soo-Whee Kim</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Soo-Whee Kim</span></h3>
<div class="p">
<sup>1</sup>Department of Integrated Biomedical and Life Science, Korea University, Seoul, Republic of Korea </div>
<div class="p">
<sup>2</sup>L-HOPE Program for Community-Based Total Learning Health Systems, Korea University, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20SW%22%5BAuthor%5D" class="usa-link"><span class="name western">Soo-Whee Kim</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Hyeji Lee</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Hyeji Lee</span></h3>
<div class="p">
<sup>1</sup>Department of Integrated Biomedical and Life Science, Korea University, Seoul, Republic of Korea </div>
<div class="p">
<sup>2</sup>L-HOPE Program for Community-Based Total Learning Health Systems, Korea University, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hyeji Lee</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20DY%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Da Yea Song</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Da Yea Song</span></h3>
<div class="p">
<sup>3</sup>Department of Psychiatry, Seoul National University Bundang Hospital, Seongnam, Republic of Korea </div>
<div class="p">
<sup>4</sup>Department of Psychiatry, Seoul National University College of Medicine, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Song%20DY%22%5BAuthor%5D" class="usa-link"><span class="name western">Da Yea Song</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20GH%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Gang-Hee Lee</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Gang-Hee Lee</span></h3>
<div class="p">
<sup>1</sup>Department of Integrated Biomedical and Life Science, Korea University, Seoul, Republic of Korea </div>
<div class="p">
<sup>2</sup>L-HOPE Program for Community-Based Total Learning Health Systems, Korea University, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20GH%22%5BAuthor%5D" class="usa-link"><span class="name western">Gang-Hee Lee</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20JH%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Jae Hyun Han</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Jae Hyun Han</span></h3>
<div class="p">
<sup>3</sup>Department of Psychiatry, Seoul National University Bundang Hospital, Seongnam, Republic of Korea </div>
<div class="p">
<sup>5</sup>Department of Psychiatry, College of Medicine, Soonchunhyang University Cheonan Hospital, Cheonan, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Han%20JH%22%5BAuthor%5D" class="usa-link"><span class="name western">Jae Hyun Han</span></a>
</div>
</div>
<sup>3,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20JW%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Jee Won Lee</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Jee Won Lee</span></h3>
<div class="p">
<sup>6</sup>Department of Psychiatry, Soonchunhyang University College of Medicine, Cheonan, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20JW%22%5BAuthor%5D" class="usa-link"><span class="name western">Jee Won Lee</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Byun%20HJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Hee Jung Byun</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Hee Jung Byun</span></h3>
<div class="p">
<sup>7</sup>Department of Psychiatry, Seoul Metropolitan Children’s Hospital, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Byun%20HJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Hee Jung Byun</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Son%20JH%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Ji Hyun Son</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Ji Hyun Son</span></h3>
<div class="p">
<sup>7</sup>Department of Psychiatry, Seoul Metropolitan Children’s Hospital, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Son%20JH%22%5BAuthor%5D" class="usa-link"><span class="name western">Ji Hyun Son</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20YR%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Ye Rim Kim</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Ye Rim Kim</span></h3>
<div class="p">
<sup>3</sup>Department of Psychiatry, Seoul National University Bundang Hospital, Seongnam, Republic of Korea </div>
<div class="p">
<sup>4</sup>Department of Psychiatry, Seoul National University College of Medicine, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20YR%22%5BAuthor%5D" class="usa-link"><span class="name western">Ye Rim Kim</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Yoojeong Lee</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Yoojeong Lee</span></h3>
<div class="p">
<sup>3</sup>Department of Psychiatry, Seoul National University Bundang Hospital, Seongnam, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lee%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yoojeong Lee</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Eunjoon Kim</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Eunjoon Kim</span></h3>
<div class="p">
<sup>8</sup>Department of Biological Sciences, Korea Advanced Institute of Science and Technology, Daejeon, Republic of Korea </div>
<div class="p">
<sup>9</sup>Center for Synaptic Brain Dysfunctions, Institute for Basic Science, Daejeon, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Eunjoon Kim</span></a>
</div>
</div>
<sup>8,</sup><sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Werling%20DM%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Donna M Werling</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Donna M Werling</span></h3>
<div class="p">
<sup>10</sup>Laboratory of Genetics, University of Wisconsin-Madison, Madison, WI USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Werling%20DM%22%5BAuthor%5D" class="usa-link"><span class="name western">Donna M Werling</span></a>
</div>
</div>
<sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20SH%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">So Hyun Kim</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">So Hyun Kim</span></h3>
<div class="p">
<sup>11</sup>Department of Psychology, Korea University, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20SH%22%5BAuthor%5D" class="usa-link"><span class="name western">So Hyun Kim</span></a>
</div>
</div>
<sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanders%20SJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Stephan J Sanders</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Stephan J Sanders</span></h3>
<div class="p">
<sup>12</sup>Institute of Developmental and Regenerative Medicine, Department of Pediatrics, University of Oxford, Oxford, UK </div>
<div class="p">
<sup>13</sup>Department of Psychiatry and Behavioral Sciences, UCSF Weill Institute for Neurosciences, University of California, San Francisco, USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sanders%20SJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Stephan J Sanders</span></a>
</div>
</div>
<sup>12,</sup><sup>13</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yoo%20HJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Hee Jeong Yoo</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Hee Jeong Yoo</span></h3>
<div class="p">
<sup>3</sup>Department of Psychiatry, Seoul National University Bundang Hospital, Seongnam, Republic of Korea </div>
<div class="p">
<sup>4</sup>Department of Psychiatry, Seoul National University College of Medicine, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yoo%20HJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Hee Jeong Yoo</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22An%20JY%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Joon-Yong An</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Joon-Yong An</span></h3>
<div class="p">
<sup>1</sup>Department of Integrated Biomedical and Life Science, Korea University, Seoul, Republic of Korea </div>
<div class="p">
<sup>2</sup>L-HOPE Program for Community-Based Total Learning Health Systems, Korea University, Seoul, Republic of Korea </div>
<div class="p">
<sup>14</sup>School of Biosystem and Biomedical Science, College of Health Science, Korea University, Seoul, Republic of Korea </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22An%20JY%22%5BAuthor%5D" class="usa-link"><span class="name western">Joon-Yong An</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>14,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Integrated Biomedical and Life Science, Korea University, Seoul, Republic of Korea </div>
<div id="Aff2">
<sup>2</sup>L-HOPE Program for Community-Based Total Learning Health Systems, Korea University, Seoul, Republic of Korea </div>
<div id="Aff3">
<sup>3</sup>Department of Psychiatry, Seoul National University Bundang Hospital, Seongnam, Republic of Korea </div>
<div id="Aff4">
<sup>4</sup>Department of Psychiatry, Seoul National University College of Medicine, Seoul, Republic of Korea </div>
<div id="Aff5">
<sup>5</sup>Department of Psychiatry, College of Medicine, Soonchunhyang University Cheonan Hospital, Cheonan, Republic of Korea </div>
<div id="Aff6">
<sup>6</sup>Department of Psychiatry, Soonchunhyang University College of Medicine, Cheonan, Republic of Korea </div>
<div id="Aff7">
<sup>7</sup>Department of Psychiatry, Seoul Metropolitan Children’s Hospital, Seoul, Republic of Korea </div>
<div id="Aff8">
<sup>8</sup>Department of Biological Sciences, Korea Advanced Institute of Science and Technology, Daejeon, Republic of Korea </div>
<div id="Aff9">
<sup>9</sup>Center for Synaptic Brain Dysfunctions, Institute for Basic Science, Daejeon, Republic of Korea </div>
<div id="Aff10">
<sup>10</sup>Laboratory of Genetics, University of Wisconsin-Madison, Madison, WI USA </div>
<div id="Aff11">
<sup>11</sup>Department of Psychology, Korea University, Seoul, Republic of Korea </div>
<div id="Aff12">
<sup>12</sup>Institute of Developmental and Regenerative Medicine, Department of Pediatrics, University of Oxford, Oxford, UK </div>
<div id="Aff13">
<sup>13</sup>Department of Psychiatry and Behavioral Sciences, UCSF Weill Institute for Neurosciences, University of California, San Francisco, USA </div>
<div id="Aff14">
<sup>14</sup>School of Biosystem and Biomedical Science, College of Health Science, Korea University, Seoul, Republic of Korea </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 25; Accepted 2025 Aug 14; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366145  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40836247/" class="usa-link">40836247</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">The phenotypic outcomes of de novo variants (DNVs) in autism spectrum disorder (ASD) exhibit wide variability. To date, no study has comprehensively estimated DNV effects accounting for familial phenotypic background.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p id="Par2">To evaluate DNV effects in a family-relative context, we defined within-family standardized deviations (WFSD) by subtracting phenotype scores of unaffected family members and standardizing the result. We applied this approach to 78,685 individuals from 21,735 families from ASD cohorts of diverse ancestries. We compared the distribution, associations with disruptive DNVs, and gene discovery results between WFSD and raw phenotype scores. We further performed outlier analysis based on WFSDs per gene to detect genes with high variability between families.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p id="Par3">We observed that ASD probands with disruptive DNVs exhibited greater behavioral symptoms and lower adaptive functioning relative to their within-family unaffected members. Compared to raw phenotype scores, WFSD provided clearer associations with DNVs and enabled greater yield in DNV-enriched gene discovery, including 18 novel ASD-associated genes. Outlier analysis identified 11 genes with high intrafamilial variability in phenotypic effects, influenced by mutation sites within functional domains or exons.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par4">Familial DNV analysis provides accurate effect estimates, a reliable basis for predicting clinical outcomes, and precise support while minimizing confounding from family background. This approach improves the identification of ASD-associated genes with true phenotypic effects by reducing variability, as well as genes with genuine phenotypic heterogeneity across families driven by mutation site. These findings enhance our understanding of ASD phenotype variability and inform potential targets for intervention.</p></section><section id="sec5"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s13073-025-01532-7.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Familial phenotype deviation, De novo variants, Autism spectrum disorder, Neurodevelopment, Phenotypic variability</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par16">Autism spectrum disorder (ASD) is a complex neurodevelopmental condition with a strong genetic basis [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Germline de novo variants (DNVs), identified in offspring but absent in parents, are significant contributors to ASD [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>–<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. However, the effects of DNVs are highly variable, where the same recurrent mutation can lead to diverse phenotypic outcomes including intellectual disability (ID), heart defects, and facial dysmorphisms [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Recent advances in studies of DNVs in ASD and developmental delay (DD) have identified numerous genes [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>] with substantial overlap between the diagnoses, such that 87% of ASD-associated genes are estimated to contribute to DD and 70% of DD-associated genes contribute to ASD [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>, <a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>].</p>
<p id="Par17">Familial background is a dominant modifier in the variable expressivity of genetic variants. In achondroplasia, where a gain-of-function missense mutation in <em>FGFR3</em> is the primary cause, height is consistently reduced but varies widely, and this variability correlates strongly with parental heights [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>, <a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. Similarly, in ASD, neurodevelopmental profiles of probands with rare copy number variants correlate significantly with those of their parents [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. To address these influences, studies compared phenotype scores of carriers to non-carrier family members to estimate variant effects. By accounting for shared genetic and environmental factors, this approach provides more accurate estimates of variant effects than deviations from population averages [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>–<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. However, most studies to date have not used within-family comparisons, limiting the precision of DNV effect assessments.</p>
<p id="Par18">Even when familial background is considered, DNVs in the same gene can result in variable phenotypic outcomes owing to differences in the functional mechanism or effect size, for example, between protein domains or exons [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>–<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Gain-of-function <em>SCN2A</em> missense DNVs in the voltage-sensing domains are associated with infantile epileptic encephalopathy, while loss-of-function missense variants in the pore-forming domains and protein-truncating variants are associated with ID and/or ASD without infantile seizures [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. Similarly, <em>SHANK3</em> protein-truncating variants in exons encoding synaptic scaffolding regions are associated with severe neurodevelopmental outcomes, whereas variants in other exons result in less severe symptoms [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Incorporating familial background can help clarify the relative phenotypic variation associated with polygenicity or DNV genotype.</p>
<p id="Par19">In this study, we adopted a family-based approach by investigating within-family standard deviation (WFSD), a normalized deviation of the proband’s phenotype score from their unaffected family members. Using data of 78,685 individuals in 21,735 families across diverse ancestries from the Korean Autism [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>], Simons Simplex Collection (SSC) [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>], and Simons Foundation Powering Autism Research for Knowledge (SPARK) [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] cohorts, we assessed WFSD of ASD core symptoms and adaptive abilities. By accounting for the polygenic effects of familial backgrounds, this approach could provide greater insights into the phenotypic impact of the specific DNV. Additionally, per-gene outlier analyses identified genes with high variability in intrafamilial deviations, likely reflecting genotype-specific functional differences. These findings provide a foundation for more accurate clinical interpretation of DNVs and suggest potential targets for precision intervention in ASD.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Cohorts</h3>
<p id="Par20">This study is based on whole-genome sequencing (WGS) and whole-exome sequencing (WES) data of 78,685 individuals from 21,735 full-trio ASD families across three cohorts: the Korean Autism (<em>n</em> = 2,605) [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>], SSC (<em>n</em> = 9,082) [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>], and <em>SPARK (</em>n = 67,098) [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] cohorts (Table <a href="#Tab1" class="usa-link">1</a>). Among these, 24,050 were ASD probands, 11,165 were unaffected siblings, and 43,470 were parents.
</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Sample information</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Cohort</th>
<th align="left" rowspan="2" colspan="1">Data type</th>
<th align="left" colspan="4" rowspan="1">Participants, No</th>
<th align="left" colspan="1" rowspan="1">Families, No</th>
<th align="left" colspan="3" rowspan="1">ASD only, No. (%)<sup>a</sup>
</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">ASD</th>
<th align="left" colspan="1" rowspan="1">Sibling</th>
<th align="left" colspan="1" rowspan="1">Parents</th>
<th align="left" colspan="1" rowspan="1">Total</th>
<th align="left" colspan="1" rowspan="1">Total</th>
<th align="left" colspan="1" rowspan="1">dnPTV carrier<sup>b</sup>
</th>
<th align="left" colspan="1" rowspan="1">dnMIS carrier<sup>c</sup>
</th>
<th align="left" colspan="1" rowspan="1">Non-carrier<sup>d</sup>
</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="3" colspan="1">Korean</td>
<td align="left" colspan="1" rowspan="1">WES</td>
<td align="left" colspan="1" rowspan="1">61</td>
<td align="left" colspan="1" rowspan="1">67</td>
<td align="left" colspan="1" rowspan="1">122</td>
<td align="left" colspan="1" rowspan="1">250</td>
<td align="left" colspan="1" rowspan="1">61</td>
<td align="left" colspan="1" rowspan="1">2 (3.28)</td>
<td align="left" colspan="1" rowspan="1">4 (6.56)</td>
<td align="left" colspan="1" rowspan="1">55 (90.2)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WGS</td>
<td align="left" colspan="1" rowspan="1">693</td>
<td align="left" colspan="1" rowspan="1">216</td>
<td align="left" colspan="1" rowspan="1">1346</td>
<td align="left" colspan="1" rowspan="1">2255</td>
<td align="left" colspan="1" rowspan="1">673</td>
<td align="left" colspan="1" rowspan="1">26 (3.75)</td>
<td align="left" colspan="1" rowspan="1">26 (3.75)</td>
<td align="left" colspan="1" rowspan="1">641 (92.5)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WES + WGS</td>
<td align="left" colspan="1" rowspan="1">754</td>
<td align="left" colspan="1" rowspan="1">283</td>
<td align="left" colspan="1" rowspan="1">1468</td>
<td align="left" colspan="1" rowspan="1">2505</td>
<td align="left" colspan="1" rowspan="1">734</td>
<td align="left" colspan="1" rowspan="1">28 (3.71)</td>
<td align="left" colspan="1" rowspan="1">30 (3.98)</td>
<td align="left" colspan="1" rowspan="1">696 (92.3)</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">SPARK</td>
<td align="left" colspan="1" rowspan="1">WES</td>
<td align="left" colspan="1" rowspan="1">17,420</td>
<td align="left" colspan="1" rowspan="1">6757</td>
<td align="left" colspan="1" rowspan="1">30,534</td>
<td align="left" colspan="1" rowspan="1">54,711</td>
<td align="left" colspan="1" rowspan="1">15,267</td>
<td align="left" colspan="1" rowspan="1">606 (3.48)</td>
<td align="left" colspan="1" rowspan="1">457 (2.62)</td>
<td align="left" colspan="1" rowspan="1">16,357 (93.9)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WGS</td>
<td align="left" colspan="1" rowspan="1">3496</td>
<td align="left" colspan="1" rowspan="1">2187</td>
<td align="left" colspan="1" rowspan="1">6704</td>
<td align="left" colspan="1" rowspan="1">12,387</td>
<td align="left" colspan="1" rowspan="1">3352</td>
<td align="left" colspan="1" rowspan="1">156 (4.46)</td>
<td align="left" colspan="1" rowspan="1">154 (4.41)</td>
<td align="left" colspan="1" rowspan="1">3186 (91.1)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WES + WGS</td>
<td align="left" colspan="1" rowspan="1">20,916</td>
<td align="left" colspan="1" rowspan="1">8944</td>
<td align="left" colspan="1" rowspan="1">37,238</td>
<td align="left" colspan="1" rowspan="1">67,098</td>
<td align="left" colspan="1" rowspan="1">18,619</td>
<td align="left" colspan="1" rowspan="1">762 (3.64)</td>
<td align="left" colspan="1" rowspan="1">611 (2.92)</td>
<td align="left" colspan="1" rowspan="1">19,543 (93.4)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SSC</td>
<td align="left" colspan="1" rowspan="1">WGS</td>
<td align="left" colspan="1" rowspan="1">2380</td>
<td align="left" colspan="1" rowspan="1">1938</td>
<td align="left" colspan="1" rowspan="1">4764</td>
<td align="left" colspan="1" rowspan="1">9802</td>
<td align="left" colspan="1" rowspan="1">2382</td>
<td align="left" colspan="1" rowspan="1">141 (5.92)</td>
<td align="left" colspan="1" rowspan="1">110 (4.62)</td>
<td align="left" colspan="1" rowspan="1">2129 (89.5)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Total</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">24,050</td>
<td align="left" colspan="1" rowspan="1">11,165</td>
<td align="left" colspan="1" rowspan="1">43,470</td>
<td align="left" colspan="1" rowspan="1">78,685</td>
<td align="left" colspan="1" rowspan="1">21,735</td>
<td align="left" colspan="1" rowspan="1">931 (3.87)</td>
<td align="left" colspan="1" rowspan="1">751 (3.12)</td>
<td align="left" colspan="1" rowspan="1">22,368 (93.0)</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p12">
<p>Summary of sample characteristics of ASD probands, siblings, and parents from three sequenced cohorts (Korean, SPARK, SSC) using either whole-exome sequencing (WES), whole-genome sequencing (WGS), or both</p>
<p><sup>a</sup>Number and percentage of ASD probands in each genetic category are presented as number (percentage)</p>
<p><sup>b</sup>De novo protein-truncating variants (dnPTV) with LOEUF &lt; 0.37</p>
<p><sup>c</sup>De novo missense variants (dnMIS) with MPC ≥ 2</p>
<p><sup>d</sup>Non-carriers: ASD probands without dnPTV or dnMIS</p>
</div></div></section></section><section id="Sec4"><h3 class="pmc_sec_title">Samples and genotyping</h3>
<p id="Par21">For the Korean Autism cohort, we collected DNA samples and clinical phenotype data from families with at least one child diagnosed with ASD by clinicians. The data collection took place across three major hospital sites in Korea: Seoul National University Bundang Hospital (SNUBH), which served as the primary center, along with Soon Chun Hyang University Hospital Bucheon (SCHBC) and Seoul Child Hospital (SCH). The ethics committees of SNUBH, SCHBC, and SCH IRB approved the study, with approval numbers as follows: SNUBH (B-1703–388–303 and B-2108–700–107), SCHBC (SCHBC 2018–04–020 and SCHBC 2022–04–016), and SCH (P01-201908-BM-02 and P01-202111–21–003). All phenotype data were cross-validated by clinical specialists. The collected data were anonymized and managed in accordance with the biorepository's standard operating procedures. The Korean cohort included 61 families (250 individuals) with WES data and 673 families (2255 individuals) with WGS data. For the SSC and SPARK cohorts, we downloaded genotype data and clinical data from SFARI Base (<a href="https://sfari.org/sfari-base" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://sfari.org/sfari-base</a>). The SSC cohort included 2382 families (9082 individuals) with WGS data. The SPARK cohort comprised 15,267 families (54,711 individuals) with WES data and 3352 families (12,387 individuals) with WGS data. All procedures adhered to the ethical standards of the Helsinki Declaration, with informed consent obtained from all participants.</p>
<p id="Par22">DNA was extracted from whole blood for the Korean and SSC cohorts and from saliva for the SPARK cohort. Sequencing was performed using Illumina platforms: HiSeq X for Korean WGS, NovaSeq 6000 for SPARK WES, and HiSeq X10 for SSC WGS. WES reads were aligned to the GRCh38 genome using BWA-mem, and variant calling was conducted with the Genome Analysis Toolkit (GATK) following best practices (v4.1.8.1 for Korean WES; v3.5 for SPARK and SSC), including variant quality score recalibration (VQSR). Korean WGS data were processed using the Illumina DRAGEN pipeline (v4.0.3). Joint genotyping was performed with the iterative gVCF genotyper for Korean WGS, GLnexus (v1.4.1) for SPARK, and GATK pipelines for SSC. Variants were filtered to include only those with “PASS” in the FILTER column and with quality metrics (GQ ≥ 20; DP ≥ 10 for SSC) to ensure high-quality calls. Additional filtering steps were applied to exclude low-complexity regions, split multiallelic sites, and remove large INDELs (allele length ≥ 50).</p></section><section id="Sec5"><h3 class="pmc_sec_title">DNV identification</h3>
<p id="Par23">DNVs were identified using the Hail 0.2 (<a href="https://hail.is/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://hail.is/</a>) de_novo() function on variants with an allele frequency (AF) below 0.01% in the gnomAD v3.1 non-neuro population, utilizing default parameters. For the Korean WGS cohort, we refined the DNV probability calculation by assessing variant origin and inheritance probabilities, adjusting the de novo probability threshold from 0.5 to 0.1 to align with GATK-processed DNV counts, referring to our previous work [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Quality filters were applied as follows: heterozygous SNPs required QUAL ≥ 7.5, GQmean ≥ 36, DPmean ≥ 34, and allele balance (AB) between 0.275 and 0.725; heterozygous indels required QUAL ≥ 10.51, gDP ≥ 3, and AB between 0.214 and 0.786. DNVs present in fewer than five individuals were retained. For the Korean WES data, only high/medium confidence DNVs were included with the following cutoffs:; heterozygous SNPs with Qual ≥ 135.58, GQmean ≥ 76, 0.77 ≥ AB ≥ 0.23, gDP ≥ 10; heterozygous indels with Qual ≥ 115.8, QD ≥ 6.64, MQ ≥ 32.15, SOR ≤ 2.35, gDP ≥ 13, 0.83 ≥ AB ≥ 0.17. DNVs present in fewer than two individuals were retained.</p>
<p id="Par24">In the SSC WGS cohort, high/medium confidence DNVs were filtered with cutoffs based on prior studies [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. We additionally filtered DNVs with internal allele count (AC) = 1, excluding outliers with DNV counts exceeding nine median standard deviations. The SPARK WGS and WES data were also filtered to retain high/medium confidence DNVs with AB &lt; 0.8 and internal AC = 1, excluding outliers with DNV counts exceeding nine median standard deviations.</p>
<p id="Par25">High-quality variants were annotated using Hail’s vep() function with Ensembl VEP version 109.3 and classified into protein-truncating variants (PTV), missense variants (MIS), and synonymous variants based on the most severe VEP consequence term. PTV included frameshift, splice acceptor/donor, and stop gain variants verified by the LOFTEE plugin, while MIS included missense and other protein-altering variants. The downstream analysis restricted PTVs in genes with loss-of-function observed over expected upper bound fraction (LOEUF) scores [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>] &lt; 0.37 and missense variants with missense badness, PolyPhen-2, constraint (MPC) [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>] ≥ 2 to ensure functional significance.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Polygenic score calculation</h3>
<p id="Par26">Following our previously described procedures [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>], we derived high-quality common variants from WGS data using the following filters: genotype quality ≥ 20, depth ≥ 10, allele balance 0.2–0.8 for heterozygous and ≥ 0.95 for homozygous calls, call rate ≥ 95%, and Hardy–Weinberg equilibrium <em>P</em> ≥ 1 × 10<sup>−6</sup>. Variants with minor allele frequency &gt; 0.05 among internal unrelated samples were retained. Polygenic scores (PS) were computed using PRScs [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>] with the default parameters (global shrinkage phi = 1.0 × 10<sup>−2</sup>, gamma = (1,0.5)) and the HapMap3 SNP LD reference panel from European-ancestry UK Biobank samples. For ASD PS, we used GWAS summary statistics from Grove et al. [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>], which included SSC and iPSYCH cohorts, and therefore computed ASD PS only in the Korean subset to avoid overlap-related inflation. For the educational attainment PS, we used European-ancestry GWAS summary statistics from Lee et al. [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] and computed scores in the entire cohort. Prior to SNP matching, WGS variants were lifted over to GRCh37 to harmonize genome build versions, and ambiguous SNPs as well as reverse-orientation INDELs were excluded. Polygenic transmission disequilibrium test (pTDT) was performed by comparing each proband’s polygenic score to the mid-parental mean PS. The deviation of proband PS from the parental mean was calculated, and the distribution of these deviations was tested against zero using one-sample t-tests. Differences in pTDT between de novo variant carrier groups were assessed using two-sample t-tests.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Clinical measures</h3>
<p id="Par27">To evaluate phenotype distribution in ASD probands based on the presence of disruptive DNVs, we focused on two primary domains: ASD core symptoms and development-associated abilities (Table <a href="#Tab2" class="usa-link">2</a>). ASD core symptoms were categorized into social communication deficits, restricted/repetitive behaviors, and total symptom severity (summed scores of social communication deficits and restricted/repetitive behaviors). Total symptom severity was assessed using the Autism Diagnostic Observation Schedule-2 [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>, <a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>] (ADOS-2) calibrated severity scores (CSS) total scores, Social Responsiveness Scale [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>] (SRS) T-scores, and Social Communication Questionnaire (SCQ) lifetime and current [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>] scores. Social communication deficits were measured with ADOS CSS Social Affect (SA) and Autism Diagnostic Interview-Revised [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>] (ADI-R) domains A (social interaction) and B (communication), while restricted/repetitive behaviors were evaluated using ADOS CSS RRB, ADI-R RRB (domain C), and the Repetitive Behavior Scale—Revised [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>] (RBSR). Development-associated functions included cognitive ability, assessed via full-scale IQ (FSIQ) [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>] and Leiter International Performance Scale-Revised IQ [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>] (non-verbal IQ), adaptive ability measured by Vineland Adaptive Behavior Scales (VABS)-II [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>] across five domains (total, communication, daily living, socialization, and motor skills), and motor coordination evaluated using the Development Coordination Disorder Questionnaire [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>] (DCDQ).
</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Phenotype distribution in probands</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1"></th>
<th align="left" rowspan="2" colspan="1"></th>
<th align="left" rowspan="2" colspan="1"></th>
<th align="left" colspan="3" rowspan="1">Genetic subgroup, Mean (SD)<sup>a</sup>
</th>
<th align="left" rowspan="2" colspan="1">ANOVA, <em>P</em><sup>b</sup>
</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">dnPTV carrier</th>
<th align="left" colspan="1" rowspan="1">dnMIS carrier</th>
<th align="left" colspan="1" rowspan="1">Non-carrier</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="10" colspan="1">ASD core symptom<sup>c</sup>
</td>
<td align="left" rowspan="4" colspan="1">Total symptom severity</td>
<td align="left" colspan="1" rowspan="1">ADOS total</td>
<td align="left" colspan="1" rowspan="1">7.38 (1.67)</td>
<td align="left" colspan="1" rowspan="1">7.26 (1.65)</td>
<td align="left" colspan="1" rowspan="1">7.43 (1.71)</td>
<td align="left" colspan="1" rowspan="1">0.49</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SCQ current</td>
<td align="left" colspan="1" rowspan="1">14.92 (6.29)</td>
<td align="left" colspan="1" rowspan="1">16.52 (7.69)</td>
<td align="left" colspan="1" rowspan="1">16.26 (7.20)</td>
<td align="left" colspan="1" rowspan="1">0.18</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SCQ lifetime</td>
<td align="left" colspan="1" rowspan="1">20.79 (7.06)</td>
<td align="left" colspan="1" rowspan="1">20.76 (6.90)</td>
<td align="left" colspan="1" rowspan="1">20.84 (7.32)</td>
<td align="left" colspan="1" rowspan="1">0.96</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SRS</td>
<td align="left" colspan="1" rowspan="1">80.64 (10.0)</td>
<td align="left" colspan="1" rowspan="1">78.00 (10.6)</td>
<td align="left" colspan="1" rowspan="1">78.30 (11.1)</td>
<td align="left" colspan="1" rowspan="1">0.03</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">Social communication deficits</td>
<td align="left" colspan="1" rowspan="1">ADOS SA</td>
<td align="left" colspan="1" rowspan="1">7.19 (1.81)</td>
<td align="left" colspan="1" rowspan="1">6.91 (1.78)</td>
<td align="left" colspan="1" rowspan="1">7.32 (1.77)</td>
<td align="left" colspan="1" rowspan="1">0.02</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ADIR A</td>
<td align="left" colspan="1" rowspan="1">19.87 (5.42)</td>
<td align="left" colspan="1" rowspan="1">20.47 (6.04)</td>
<td align="left" colspan="1" rowspan="1">20.24 (5.94)</td>
<td align="left" colspan="1" rowspan="1">0.65</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ADIR B verbal</td>
<td align="left" colspan="1" rowspan="1">16.13 (4.10)</td>
<td align="left" colspan="1" rowspan="1">16.12 (4.26)</td>
<td align="left" colspan="1" rowspan="1">15.98 (4.62)</td>
<td align="left" colspan="1" rowspan="1">0.89</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">Restricted/repetitive behavior</td>
<td align="left" colspan="1" rowspan="1">ADOS RRB</td>
<td align="left" colspan="1" rowspan="1">7.36 (2.42)</td>
<td align="left" colspan="1" rowspan="1">7.81 (1.95)</td>
<td align="left" colspan="1" rowspan="1">7.39 (2.22)</td>
<td align="left" colspan="1" rowspan="1">0.08</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ADIR C</td>
<td align="left" colspan="1" rowspan="1">6.44 (2.68)</td>
<td align="left" colspan="1" rowspan="1">6.26 (2.24)</td>
<td align="left" colspan="1" rowspan="1">6.30 (2.57)</td>
<td align="left" colspan="1" rowspan="1">0.77</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RBSR</td>
<td align="left" colspan="1" rowspan="1">30.20 (18.8)</td>
<td align="left" colspan="1" rowspan="1">29.55 (17.9)</td>
<td align="left" colspan="1" rowspan="1">30.78 (19.0)</td>
<td align="left" colspan="1" rowspan="1">0.25</td>
</tr>
<tr>
<td align="left" rowspan="8" colspan="1">Development-associated<sup>d</sup>
</td>
<td align="left" rowspan="2" colspan="1">Cognitive ability</td>
<td align="left" colspan="1" rowspan="1">FSIQ</td>
<td align="left" colspan="1" rowspan="1">74.11 (23.1)</td>
<td align="left" colspan="1" rowspan="1">74.27 (26.2)</td>
<td align="left" colspan="1" rowspan="1">81.34 (26.5)</td>
<td align="left" colspan="1" rowspan="1">1.41 × 10<sup>−6</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Non-verbal IQ</td>
<td align="left" colspan="1" rowspan="1">77.10 (23.0)</td>
<td align="left" colspan="1" rowspan="1">77.73 (25.7)</td>
<td align="left" colspan="1" rowspan="1">86.16 (25.8)</td>
<td align="left" colspan="1" rowspan="1">5.71 × 10<sup>−9</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="5" colspan="1">Adaptive ability</td>
<td align="left" colspan="1" rowspan="1">VABS total</td>
<td align="left" colspan="1" rowspan="1">69.17 (14.7)</td>
<td align="left" colspan="1" rowspan="1">69.18 (16.1)</td>
<td align="left" colspan="1" rowspan="1">72.36 (14.9)</td>
<td align="left" colspan="1" rowspan="1">5.00 × 10<sup>−8</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">VABS communication</td>
<td align="left" colspan="1" rowspan="1">69.95 (19.8)</td>
<td align="left" colspan="1" rowspan="1">69.72 (21.3)</td>
<td align="left" colspan="1" rowspan="1">73.20 (19.7)</td>
<td align="left" colspan="1" rowspan="1">1.90 × 10<sup>−5</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">VABS daily living</td>
<td align="left" colspan="1" rowspan="1">71.86 (17.0)</td>
<td align="left" colspan="1" rowspan="1">71.32 (18.4)</td>
<td align="left" colspan="1" rowspan="1">75.30 (17.1)</td>
<td align="left" colspan="1" rowspan="1">3.24 × 10<sup>−8</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">VABS socialization</td>
<td align="left" colspan="1" rowspan="1">66.70 (17.4)</td>
<td align="left" colspan="1" rowspan="1">66.89 (17.9)</td>
<td align="left" colspan="1" rowspan="1">69.14 (17.6)</td>
<td align="left" colspan="1" rowspan="1">1.00 × 10<sup>−3</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">VABS motor skills</td>
<td align="left" colspan="1" rowspan="1">76.74 (14.9)</td>
<td align="left" colspan="1" rowspan="1">76.42 (15.5)</td>
<td align="left" colspan="1" rowspan="1">79.94 (15.4)</td>
<td align="left" colspan="1" rowspan="1">1.57 × 10<sup>−4</sup>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Coordination</td>
<td align="left" colspan="1" rowspan="1">DCDQ</td>
<td align="left" colspan="1" rowspan="1">33.65 (10.6)</td>
<td align="left" colspan="1" rowspan="1">34.68 (11.9)</td>
<td align="left" colspan="1" rowspan="1">38.88 (12.5)</td>
<td align="left" colspan="1" rowspan="1">3.83 × 10<sup>−7</sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p25">
<p>ASD probands were grouped into three genetic subgroups based on whether they carry de novo variants: dnPTV carriers (LOEUF &lt; 0.37), dnMIS carriers (MPC ≥ 2), and non-carriers (no dnPTV or dnMIS). Statistical significance was assessed using one-way ANOVA across genetic subgroups</p>
<p><sup>a</sup>Phenotype scores are presented as mean (standard deviation) unless otherwise indicated</p>
<p><sup>b</sup><em>P</em> values are from one-way ANOVA testing across the three genetic subgroups; <em>P</em> values &lt; 0.05 are considered statistically significant</p>
<p><sup>c</sup>ASD core symptoms include total symptom severity (ADOS, SCQ, SRS), social communication deficits (ADOS SA, ADI-R A &amp; B), and restricted/repetitive behavior (ADOS RRB, ADI-R C, RBSR). Higher scores indicate greater symptom severity</p>
<p><sup>d</sup>Development-associated phenotypes include cognitive ability (FSIQ, non-verbal IQ), adaptive ability (VABS), and coordination (DCDQ). Lower scores reflect greater functional impairment</p>
</div></div></section><p id="Par28">We compared those 18 clinical phenotypes between ASD probands with de novo PTV (dnPTV), those with de novo MIS (dnMIS), and those without either dnPTV or dnMIS. We ran analysis of variance (ANOVA) tests and found that dnPTV or dnMIS carriers have significantly higher core symptom severity (SRS T) and lower developmental abilities (IQ, VABS, and DCDQ) (Table <a href="#Tab2" class="usa-link">2</a>).</p></section><section id="Sec8"><h3 class="pmc_sec_title">Variable expressivity of DNVs</h3>
<p id="Par29">To examine expressivity of disruptive de novo variants (dnDIS), including dnPTV and dnMIS, we investigated the phenotypic profiles of ASD probands carrying dnDIS. We defined ID as FSIQ &lt; 70, non-verbal IQ &lt; 70 when FSIQ is not available, and VABS total &lt; 70 when both FSIQ and non-verbal IQ are not available, and individuals with intellectual functioning higher than population averages as FSIQ ≥ 100, non-verbal IQ ≥ 100 when FSIQ is not available, and VABS total ≥ 100 when both FSIQ and non-verbal IQ are not available. We assessed the proportion of comorbid ID and higher-than-population ASD and the proportion of normative, mild to moderate, and severe symptoms using SRS norms (normative: SRS T &lt; 60; mild to moderate: SRS T 60–75; severe: SRS T &gt; 75) [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>] in dnDIS carriers.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Investigation of within-family standard deviation</h3>
<p id="Par30">To calculate WFSD, we selected clinical phenotypes assessed in both ASD probands and their unaffected family members across at least two cohorts. These phenotypes included SRS T-scores (available in the Korean and SSC cohorts), SCQ lifetime scores (available in the Korean, SSC, and SPARK cohorts), VABS scores (available in the Korean and SSC cohorts), ADOS CSS total, SA, and RRB sub-scores (available in the Korean cohort), and FSIQ (available in the Korean cohort). While both SRS T and SCQ lifetime scores measure overall symptom severity, SRS primarily reflects behavioral severity, whereas SCQ scores are more indicative of communicative severity. ADOS total scores provide an independent clinician-rated assessment of observed symptom severity, with the SA and RRB sub-scores capturing specific symptom domains including social impairment and repetitive behavior. The number of families included in each analysis varied: 2699 families for SRS, 7920 families for SCQ lifetime, 1952 families for VABS, 223 families for ADOS total, 215 families for ADOS SA, 215 families for ADOS RRB, and 127 families for FSIQ. Intrafamilial phenotype deviation was determined by comparing the proband’s scores with the mean scores of their unaffected family members and normalizing the differences by the SD of the general population (SD<sub>SRS</sub> = 10, SD<sub>VABS</sub> = 15, SD<sub>FSIQ</sub> = 15). We applied the same normalization strategy across cohorts, given that previous studies have validated the use of K-SRS T-scores [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>] and K-VBAS II scores [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. For SCQ lifetime and ADOS scores, we used the SD derived from our dataset (SD<sub>SCQ</sub> = 10.45, SD<sub>ADOS_Total</sub> = 2.17, SD<sub>ADOS_SA</sub> = 2.13, SD<sub>ADOS_RRB</sub> = 2.61) due to the absence of established population norms.</p>
<p id="Par31">Next, we compared WFSD of phenotype scores in ASD probands with dnPTV, with dnMIS, and those without either dnPTV or dnMIS using a generalized linear model (GLM) regression to evaluate differences in WFSD across carrier groups. When accounting for potential confounding factors, we conducted a GLM controlling for age, sex, and cohort to compare WFSD across genetic subgroups.</p>
<p id="Par32">To comprehensively assess the neurodevelopmental impact of dnDIS, we constructed a two-dimensional (2D) space combining SRS and VABS WFSD scores. This framework enabled the simultaneous evaluation of symptom severity and adaptive abilities, offering a holistic view of neurodevelopmental profiles influenced by DNVs. With this framework, we estimated neurodevelopmental effects of total dnDIS and dnDIS in known gene sets [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>] including ASD-associated genes, DD-associated genes, ASD-dominant genes (predominantly associated with ASD rather than DD), and DD-dominant genes (predominantly associated with DD rather than ASD). For the downstream WFSD outlier analysis, we utilized SRS T and VABS scores, which are normalized for age and sex.</p>
<p id="Par33">To evaluate the distributional properties of raw SRS T-scores and intrafamilial deviations, we conducted normality tests using the Shapiro–Wilk, Anderson–Darling, and Kolmogorov–Smirnov methods. These tests assessed whether phenotype score distributions adhered to normality assumptions, which is essential for subsequent statistical analyses. The Shapiro–Wilk test statistic (W) ranges from 0 to 1, with higher values indicating greater adherence to normality. Conversely, higher Anderson–Darling (AD) and Kolmogorov–Smirnov (KS) test statistics reflect greater deviations from normality. <em>P</em>-values from these tests represent the probability of observing the data under the null hypothesis of normality; lower <em>p</em>-values (&lt; 0.05) indicate non-normal distributions.</p>
<p id="Par34">For association testing, we defined severe ASD phenotypes as those with raw SRS T-scores ≥ 76 (2.6 SD above the population mean) or equivalent intrafamilial deviations (≥ 2.6 WFSD). We compared the odds of carrying dnDIS between individuals with and without severe phenotypes using Fisher’s exact test to calculate odds ratios. Additionally, we identified genes enriched for dnDIS in cases meeting severe phenotype criterion. To further evaluate the functional relevance of these gene sets, we performed Gene Ontology (GO) enrichment analysis using the clusterProfiler R package (v4.12.6) [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. Enrichment was assessed against the Biological Process categories in the GO database (downloaded July 2023), with multiple testing correction by the Benjamini–Hochberg method. Significantly enriched pathways were defined as those with adjusted <em>P</em> &lt; 0.05. Enrichment results were compared between the raw score and WFSD definitions to identify pathways uniquely associated with each gene set. Finally, we conducted enrichment analyses for differentially expressed genes in the developing human brain, encompassing 39 clusters across nine cell types: radial glia, neuroblasts, excitatory neurons, inhibitory neurons, astrocytes, microglia, oligodendrocytes, oligodendrocyte precursor cells, and endothelial cells [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>].</p></section><section id="Sec10"><h3 class="pmc_sec_title">Phenotype outcome outlier analysis</h3>
<p id="Par35">To identify genes with highly variable impacts on phenotypes between unrelated families, we performed outlier analyses for each variant type (dnMIS and dnPTV) across raw SRS T and VABS scores and SRS and VABS WFSD, resulting in eight distinct analyses. For each combination, we calculated the median absolute deviation (MAD) of the phenotype outcomes among all variant carriers within each gene. Genes were classified as outliers if they met the following criteria:</p>
<ol class="list" style="list-style-type:decimal">
<li><p id="Par36">MAD exceeded the overall mean MAD by more than two SDs for the specific variant type and phenotype.</p></li>
<li><p id="Par37">Variants within the gene were observed in more than two samples.</p></li>
<li><p id="Par38">(For WFSD only) WFSD observations were above the 95th percentile for the respective phenotype and variant type.</p></li>
</ol>
<p id="Par39">Identified outlier genes were further analyzed for the location of mutations within specific functional domains or exons using the UCSC Genome Browser (<a href="https://genome.ucsc.edu/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://genome.ucsc.edu/</a>). By conducting analyses separately for each variant type and phenotype, we ensured a comprehensive assessment of gene-specific impacts on neurodevelopmental profiles.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec12"><h3 class="pmc_sec_title">Family-based analyses explain varying phenotype outcomes of a de novo mutation</h3>
<p id="Par40">We analyzed WGS/WES and phenotype data of 78,685 individuals from 21,735 families with ASD probands across diverse ancestries from the Korean, SSC, and SPARK cohorts (Table <a href="#Tab1" class="usa-link">1</a>). The distributions of exonic DNVs per individual were comparable between WES and WGS datasets in all cohorts (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S1). Among the 24,050 ASD probands, approximately 10% carried dnDIS, including damaging dnPTV (LOEUF [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>] &lt; 0.37) and dnMIS (MPC [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>] ≥ 2) (Table <a href="#Tab1" class="usa-link">1</a>). Probands carrying dnDIS demonstrated more severe clinical profiles than non-carriers with ASD, including significantly lower developmental abilities across cognitive, adaptive, and motor coordination domains (Table <a href="#Tab2" class="usa-link">2</a>). Although most ASD core symptom measures, especially social communication deficits, showed a trend toward attenuated severity in probands carrying dnDIS compared to non-carriers, consistent with previous findings [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>], SRS T-scores were significantly higher in dnDIS carriers (Table <a href="#Tab2" class="usa-link">2</a>).</p>
<p id="Par41">However, there was substantial variability in the clinical outcomes of dnDIS carriers. For instance, 45.5% of dnDIS carriers had comorbid ID, while 8.3% had intellectual functioning higher than population averages, and 66.6% exhibited severe symptoms, while 33.4% exhibited normative to moderate symptoms. To investigate the modifying effects of familial background on this variable expressivity, we examined correlations between mid-parental mean and offspring SRS T-scores across all fully phenotyped families (<em>N</em> = 2699). Consistent with previous findings [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>], significant correlations were observed in both ASD cases (<em>R</em> = 0.12, <em>P</em> = 2.8 × 10<sup>−10</sup>) and unaffected siblings (<em>R</em> = 0.37, <em>P</em> &lt; 2.2 × 10<sup>−16</sup>) (Fig. <a href="#Fig1" class="usa-link">1</a>A). Among the 26% of dnPTV and dnMIS carriers who exhibited normative to moderate SRS severity (SRS T 60–75), 87% of their parents scored lower-than-average on SRS (SRS T &lt; 60) [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Without accounting for familial baselines, such cases may lead to underestimation of DNV effects.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366145_13073_2025_1532_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d4e/12366145/5933621c1f17/13073_2025_1532_Fig1_HTML.jpg" loading="lazy" id="MO1" height="502" width="779" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Intrafamilial deviation of social responsive scale from parents. <strong>A</strong> Correlation of social responsiveness scale (SRS) T-scores between offspring and parents within families. The effect of de novo variants (DNVs) may be underestimated when the offspring exhibit milder SRS severity but their parents have lower-than-average SRS. Conversely, effects may be overestimated when the offspring have high SRS severity and their parents also have high SRS T-scores. <strong>B</strong> Calculation of within-family standard deviation (WFSD) for SRS in offspring from parents. WFSD is calculated by subtracting the mean parental SRS T-score from the offspring’s score and dividing by the general population SD. The histogram shows the distribution of SRS deviations in ASD probands (<em>n</em> = 2598) and unaffected siblings (<em>n</em> = 1910). <strong>C</strong> SRS deviation distributions among genetic subgroups in ASD cases, showing greater deviations in carriers of de novo protein-truncating variant (dnPTV) and missense (dnMIS). <strong>D</strong> SRS deviations stratified by gene category: ASD-/DD-associated genes</p></figcaption></figure><p id="Par42">To address this, we calculated WFSD by subtracting the mid-parental mean SRS T-score from the proband’s score and normalizing it by the general population SD (SD = 10) [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>] (Fig. <a href="#Fig1" class="usa-link">1</a>B). ASD probands demonstrated a mean increase in SRS deviation of 3.3 WFSD, a significant shift relative to their parents. In contrast, unaffected siblings showed a small negative deviation of − 0.21 WFSD. We next investigated SRS deviation based on whether probands carry dnPTV, dnMIS, or none. Among the subgroups, dnPTV carriers had the largest deviation from their parents (3.5 WFSD), followed by dnMIS carriers (3.3 WFSD) and non-carriers (3.2 WFSD) (Fig. <a href="#Fig1" class="usa-link">1</a>C; Additional file <a href="#MOESM2" class="usa-link">2</a>: Table S1). We further stratified SRS deviations based on the gene categories harboring dnPTV or dnMIS, using previously characterized ASD-associated genes [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>] (<em>N</em> = 72) and DD genes [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>] (<em>N</em> = 285). Greater deviations were observed for ASD-associated genes compared to DD genes for both dnPTV carriers (3.7 vs. 3.6 WFSD) and dnMIS carriers (3.8 vs. 3.5 WFSD), although the differences were not significant (Fig. <a href="#Fig1" class="usa-link">1</a>D; Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S2A; Additional file <a href="#MOESM2" class="usa-link">2</a>: Table S1). We also examined whether the number of dnPTV or dnMIS variants per individual was associated with increased SRS deviation to test for potential cumulative effects. However, no clear trend in deviation was observed between those with one versus more than one variant, likely due to the extremely small number of such multi-hit carriers (e.g., <em>n</em> = 2 for dnPTV &gt; 1). To further explore the relationship between DNV burden and inherited liability, we calculated pTDT scores based on ASD polygenic scores [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. Non-carrier probands showed significantly elevated pTDT compared to 0 (<em>P</em> &lt; 0.001, one-sample t-test), suggesting over-transmission of common variant risk. In contrast, DNV carriers did not show significant over-transmission. Additionally, we found that dnPTV carriers with mutations in ASD-associated genes exhibited significantly lower pTDT compared to non-carriers (<em>P</em> = 0.02, two-sample <em>t</em>-test), consistent with a liability threshold model (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S3). This result supports a polygenic architecture of ASD, where rare and common variants collectively influence phenotypic severity.</p>
<p id="Par43">Compared to raw SRS T-scores, WFSD distribution was closer to normality with fewer outliers across genetic subgroups in probands (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S4). To further evaluate the utility of WFSD, we assessed the associations between dnDIS and severe SRS profiles using two definitions: raw SRS T-scores ≥ 76 (2.6 SD above the population mean) and intrafamilial deviations ≥ 2.6 WFSD. Stronger associations were observed with WFSD (OR = 1.39, 95% CI = 1.02–1.91) than raw T-scores (OR = 1.13, 95% CI = 0.86–1.50) (Fig. <a href="#Fig2" class="usa-link">2</a>A). We also investigated genes that have dnDIS in ASD probands with severe SRS profiles using both definitions. More dnDIS-enriched genes were identified using the WFSD definition (<em>N</em> = 201) than the raw T-score definition (<em>N</em> = 167), with 38 genes uniquely identified in the WFSD group (Fig. <a href="#Fig2" class="usa-link">2</a>B; Additional file <a href="#MOESM3" class="usa-link">3</a>: Table S2). Of these, 20 were previously associated with ASD in the SFARI gene database (<a href="https://gene.sfari.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://gene.sfari.org</a>) [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>], and 18 were novel.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366145_13073_2025_1532_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d4e/12366145/d3751b6aed89/13073_2025_1532_Fig2_HTML.jpg" loading="lazy" id="MO2" height="580" width="779" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of gene associations using WFSD and raw phenotype scores. <strong>A</strong> Enrichment for disruptive de novo variants (dnDIS) carriers with social responsiveness scale (SRS) within-family standard deviation (WFSD) ≥ 2.6SD and raw SRS T-scores ≥ 76. <strong>B</strong> Number of genes enriched for dnDIS in ASD cases using each definition. Color represents group origin (purple: WFSD, yellow: raw, both: overlapping). Inset shows SFARI gene rankings for genes uniquely identified with WFSD ≥ 2.6SD. <strong>C</strong> Pathway enrichment analysis of SRS-associated genes using GO molecular function terms. Circle size indicates the number of associated genes. <strong>D</strong> Enrichment of SRS-associated genes in cell type–specific differentially expressed gene sets from the developing human brain. Color scale indicates log odds ratio of overlap; asterisks denote FDR &lt; 0.05</p></figcaption></figure><p id="Par44">To assess the functional relevance of SRS-associated genes, we examined pathway enrichment for genes identified under each definition (Fig. <a href="#Fig2" class="usa-link">2</a>C). Many pathways were shared across both groups, including those related to transcriptional regulation, chromatin remodeling, and histone modification. These are well-established ASD-associated pathways [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. Among the pathways uniquely enriched in WFSD-only genes, modification-dependent protein binding involves recognition of post-translationally modified proteins. This pathway included several chromatin-related regulators uniquely found in the WFSD-defined severe group, including <em>CBX4</em>, <em>KMT2E</em>, <em>ZMYND11</em>, and <em>PRPF8</em>. <em>CBX4</em> is a component of the Polycomb Repressive Complex 1, which is associated with NDD and has previously been highlighted as an ASD risk gene in network-based analyses [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>, <a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. <em>ZMYND11</em> and <em>KMT2E</em> have been implicated in ASD and NDD through large-scale mutation studies and clinical case reports [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>, <a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. While <em>PRPF8</em> has not been directly linked to ASD in prior genetic studies, it has been identified as an interactor of the high-confidence ASD gene <em>STXBP1</em> in a protein–protein interaction network [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. Another uniquely enriched pathway, P-type/ATPase-coupled transmembrane transporter activity, included <em>ATP1A1</em> and <em>ATP2B2</em>, both uniquely identified in the WFSD-defined group. This pathway has also been implicated in schizophrenia through enrichment of ATPase-related functions among genes carrying loss-of-function variants [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. Both <em>ATP1A1</em> and <em>ATP2B2</em> have been previously linked to neurodevelopmental disorders, including phenotypes such as intellectual disability, epilepsy, and ASD [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>, <a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>].</p>
<p id="Par45">Given the bimodal distribution of WFSD, we further compared individuals in the better-than-expected subgroup (SRS WFSD &lt; 2.0) and the severe subgroup (SRS WFSD ≥ 2.6). The severe subgroup tended to have a higher burden of disruptive DNVs than the better-than-expected subgroup, and genes carrying such variants showed enrichment for both WFSD-only and well-established ASD-associated pathways (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S5). In contrast, the better-than-expected subgroup had a significantly higher frequency of less deleterious missense variants (MPC &lt; 1) than the severe subgroup. In addition, genes carrying disruptive DNVs in this subset had no clear convergence in functional profiles, indicating more heterogeneous effects (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S5). When comparing dnDIS-enriched genes with cell type-specific differentially expressed genes (DEGs) in the developing human brain [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>], we observed similar enrichment patterns across both raw and WFSD definitions (Fig. <a href="#Fig2" class="usa-link">2</a>D; Additional file <a href="#MOESM3" class="usa-link">3</a>: Table S2). However, significant enrichments (FDR &lt; 0.05) in microglia and oligodendrocyte clusters were exclusively observed in the WFSD-defined severe SRS group.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Estimating the effect size of de novo variants on neurodevelopmental profiles</h3>
<p id="Par46">We further examined SRS deviations in probands from unaffected siblings within the same family (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S6A). Similar to deviations calculated from parents, dnPTV carriers exhibited the largest deviation of 3.9 WFSD, while dnMIS carriers and non-carriers showed deviations of 3.5 WFSD. For dnPTV carriers, the deviation increased to 4.0 WFSD when restricted to ASD/DD-associated genes (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S6B). For dnMIS carriers, deviations reached 3.9 WFSD in ASD genes and 3.7 WFSD in DD genes. Although deviations from siblings were marginally greater than those from parents (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S7; Additional file <a href="#MOESM2" class="usa-link">2</a>: Table S1), the patterns were nearly identical. These results were consistent across the proband’s genetic subgroup within the family, further validating the use of unaffected siblings as reliable proxies when parental phenotype data are unavailable.</p>
<p id="Par47">We evaluated deviations in other clinical phenotypes assessed in unaffected siblings. Specifically, we analyzed VABS total, SCQ lifetime, ADOS CSS total, SA, RRB, and FSIQ scores, calculated following the same procedure as for SRS. Mean VABS scores for ASD probands were negatively shifted to the left relative to their siblings by 2.1 WFSD for dnPTV carriers, 2.2 WFSD for dnMIS carriers, and 2.0 WFSD for non-carriers (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S8A; Additional file <a href="#MOESM2" class="usa-link">2</a>: Table S1). Similarly, SCQ lifetime scores for probands were shifted to the right relative to siblings by 1.8 WFSD for both dnPTV and dnMIS carriers and 1.7 WFSD for non-carriers (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S8C; Additional file <a href="#MOESM2" class="usa-link">2</a>: Table S1). This pattern was more evident for ASD/DD-associated genes (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S8B, D). Although this pattern was more attenuated in ADOS and FSIQ WFSD due to the limited sample size, similar trends were observed, with ADOS scores shifted to the right and FSIQ scores shifted to the left relative to those of siblings (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S8E–L). The smaller number of observations primarily reflects practical challenges in obtaining these assessments, such as limited cooperation among participants and variability in clinical protocols across sites.</p>
<p id="Par48">To comprehensively estimate the effect size of dnDIS on neurodevelopmental profiles, we constructed 2D space combining SRS T and VABS scores. The magnitude and direction of proband deviations were measured relative to unaffected siblings and parents within families. In the 2D space, ASD probands with dnPTV exhibited a diagonal shift of 4.5 WFSD, representing simultaneous increases in behavioral symptom and decreases in adaptive ability (Fig. <a href="#Fig3" class="usa-link">3</a>A). This deviation increased slightly to 4.6 WFSD for dnPTV in ASD/DD-associated genes (Fig. <a href="#Fig3" class="usa-link">3</a>B, C). In contrast, dnMIS carriers showed smaller overall deviations, with shifts of 4.2 WFSD across all genes and 4.5 WFSD for ASD/DD-associated genes (Fig. <a href="#Fig3" class="usa-link">3</a>D–F). The largest neurodevelopmental impacts for both dnPTV and dnMIS carriers were observed when the variants were in DD genes predominantly associated with DD compared to ASD [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>] (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S9).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366145_13073_2025_1532_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d4e/12366145/43be7bfb35c3/13073_2025_1532_Fig3_HTML.jpg" loading="lazy" id="MO3" height="470" width="779" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Effects of de novo variants on neurodevelopmental profiles. <strong>A</strong> Neurodevelopmental profile deviation (behavioral severity vs adaptive ability) for ASD probands with de novo protein-truncating variant (dnPTV) (<em>n</em> = 116 families) in a 2D space defined by behavioral symptom severity and adaptive ability (behavioral symptom severity: social responsiveness scale; adaptive ability: Vineland adaptive behavior scale), showing a shift of 4.5 within-family standard deviation (WFSD) towards higher behavioral severity and lower adaptive ability. <strong>B</strong> Profile deviation in ASD probands with dnPTV located in ASD-associated genes (<em>n</em> = 29 families), showing a shift of 4.6 WFSD. <strong>C</strong> Profile deviation in ASD probands with dnPTV in developmental disorder (DD)-associated genes (<em>n</em> = 37 families), showing a shift of 4.6 WFSD. <strong>D</strong> Profile deviation in ASD probands with de novo missense (dnMIS) (<em>n</em> = 94 families), showing a shift of 4.2 WFSD. <strong>E</strong> Profile deviation in ASD probands with dnMIS located in ASD-associated genes (<em>n</em> = 16 families), showing a shift of 4.5 WFSD. <strong>F</strong> Profile deviation in ASD probands with dnMIS located in DD-associated genes (<em>n</em> = 37 families), showing a shift of 4.5 WFSD</p></figcaption></figure></section><section id="Sec14"><h3 class="pmc_sec_title">Genes with variable intrafamilial impacts</h3>
<p id="Par49">To identify genes with highly variable impacts, we conducted outlier analysis for dnMIS and dnPTV using SRS and VABS. Among 1328 tested genes, 11 genes had variable intrafamilial impacts between unrelated families (Fig. <a href="#Fig4" class="usa-link">4</a>A; Additional file <a href="#MOESM4" class="usa-link">4</a>: Table S3). For dnMIS variants, four genes exhibited high variability in intrafamilial deviations: <em>PTEN</em> for SRS; <em>SCN2A, ELAVL3</em>, and <em>TRRAP</em> for VABS. Among genes with dnPTV, seven genes showed high variability: <em>MED13L, WDFY3,</em> and <em>CHD2</em> for SRS; <em>SHANK3, NCKAP1, WAC, TNRC6B</em>, and <em>WDFY3</em> for VABS.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366145_13073_2025_1532_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/8d4e/12366145/4b74dba051b9/13073_2025_1532_Fig4_HTML.jpg" loading="lazy" id="MO4" height="400" width="779" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Genes with variable expressivity between families for neurodevelopmental profiles. <strong>A</strong> Median absolute deviation (MAD) of phenotype outcomes of de novo missense (dnMIS) and protein-truncating variant (dnPTV) for each gene, highlighting outlier genes with high variability in intrafamilial impacts. <strong>B</strong> Variable social responsiveness scale (SRS) deviation outcomes in unrelated ASD probands carrying dnMIS in different functional domains of <em>PTEN</em>. Functional domains and exons are indicated, demonstrating how mutation position influences phenotypic outcomes. Red points represent amino acid positions of missense variants, and the size of point and length of stem correspond to the effect size of phenotype deviations. <strong>C</strong> Variable Vineland adaptive behavior scale (VABS) deviation outcomes in unrelated ASD cases carrying dnPTV in different functional domains of <em>NCKAP1</em>. The pedigree illustrates how the impact of variants may be underestimated when examining raw VABS scores alone. Red rectangles indicate frameshift variants, and red triangles indicate stop-gain variants. <strong>D</strong> Variable SRS deviation outcomes in unrelated ASD cases carrying dnPTV in different functional domains of <em>CHD2</em></p></figcaption></figure><p id="Par50">Among the genes with dnMIS and variable SRS WFSD between families, <em>PTEN</em> stood out as a notable example (Fig. <a href="#Fig4" class="usa-link">4</a>B). <em>PTEN</em> is associated with a range of clinical outcomes, including ASD and cancers [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>, <a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>]. Missense variants within its phosphatase domain, particularly those affecting the substrate-binding and TI loop regions, exhibited significantly more severe SRS deviations (p.Thr131Ile: 4.1 WFSD; p.Thr167Asn: 4.2 WFSD) than other <em>PTEN</em> dnMIS variants (p.Asp92Asn: 2.4 WFSD). Consistent with these observations, prior multi-model functional assays demonstrated that p.Thr131Ile exhibits near-complete loss-of-function across diverse cellular models, reflecting severe catalytic disruption, while p.Thr167Asn causes substantial impairment of phosphatase activity and partial protein instability [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. In contrast, p.Asp92Asn retained partial catalytic activity in yeast reconstitution assays [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>], consistent with the more modest phenotypic impact we observed. These findings align with previous reports demonstrating that the TI loop and adjacent catalytic motifs in the substrate-binding region are essential for determining the substrate specificity and enzymatic activity of <em>PTEN</em>, which in turn regulate critical downstream signaling pathways [<a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>, <a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. Disruption of these regions likely amplifies the neurodevelopmental impact of <em>PTEN</em> variants, contributing to the observed phenotypic variability in ASD probands. Similarly, the other three dnMIS outlier genes also exhibit clinical variability depending on mutations in different functional domains [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>, <a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>].</p>
<p id="Par51">For dnPTV variants, <em>NCKAP1</em> displayed substantial variability in its impact on VABS intrafamilial deviations (Fig. <a href="#Fig4" class="usa-link">4</a>C). Two PTVs, p.Gly169ValfsTer14 and p.Glu1082Ter, exhibited markedly different VABS deviations, at − 3.9 WFSD and − 2.3 WFSD, respectively. These variants occurred in exon 5 and exon 30 out of total 31 exons. While both exons have similar transcript inclusion, the frameshift in the earlier exon likely caused greater disruption, potentially destabilizing downstream exons, and most transcripts, leading to greater phenotypic severity. This case is particularly noteworthy because, when examining raw VABS scores, the variability between these unrelated cases appears minimal (64 vs. 70), making it easy to overlook. Recent studies link <em>NCKAP1</em> PTV and MIS to varying severities of ASD [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>], but its functional domains remain poorly understood, warranting further investigation into the mechanisms driving this variability.</p>
<p id="Par52">Another example of highly variable dnPTV impacts was observed in <em>CHD2</em>, influencing SRS intrafamilial deviations (Fig. <a href="#Fig4" class="usa-link">4</a>D). Recent studies indicate that while reported mutations in <em>CHD2</em> do not cluster within its known functional domains, the C-terminus, which enhances DNA binding and stimulates ATPase activity, is enriched with mutations [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>, <a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>]. All three dnPTV were in the C-terminus and exhibited SRS deviations ranging from 2.4 WFSD to 4.7 WFSD. Among these, p.Glu966SerfsTer2 frameshift occurring in the earliest exon regulating the P-loop ATPase region had the most severe outcome (4.7 WFSD). In contrast, two mutations in the penultimate exon—a stop-gain (p.Arg1637Ter) and a frameshift (p.Gly1651TrpfsTer16)—resulted in deviations of 4.1 WFSD and 2.4 WFSD, respectively. The milder effect of the latter may reflect partial preservation of protein function, as frameshift mutations affecting less than 25% of the coding sequence can escape nonsense-mediated decay [<a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. The remaining five dnPTV outlier genes have also been reported to show variable expressivity in prior literature, further validating our framework [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>, <a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>–<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>].</p></section></section><section id="Sec15"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par53">We measured the effect sizes of DNVs on behavioral symptom severity and adaptive functioning considering familial background leveraging large and ancestrally diverse ASD cohorts (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S10). This approach improved genotype–phenotype associations and led to the discovery of 18 novel ASD-associated genes. Furthermore, we identified 11 genes with high variability in intrafamilial effects, underscoring the importance of specific mutation sites in shaping the phenotypic heterogeneity in ASD.</p>
<p id="Par54">The use of WFSD provided less variable and more robust associations with ASD than raw phenotype scores, indicating improved biological interpretability (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S4, Fig. <a href="#Fig2" class="usa-link">2</a>). Comparison of gene-level effects of dnPTVs on SRS WFSD with external gene annotations (SFARI Gene) and effect size estimates [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>] demonstrated largely consistent enrichment patterns, supporting the validity of our approach (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S11). Notably, among genes in the 10th decile of WFSD for dnPTV carriers, <em>SPAG9</em>, <em>UNK</em>, and <em>TNFRSF8</em> were neither listed in the SFARI Gene database [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>] nor included in the gene-level effect size estimates by Rolland et al. [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>]. These genes have primarily been implicated in cancer but were also reported in ASD probands or shown to regulate neurogenesis [<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a>–<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a>]. Additionally, our findings support the reliability of using unaffected siblings as proxies for parental data when calculating WFSD, especially when parental phenotypic information is unavailable.</p>
<p id="Par55">Although utilizing WFSD showed a reduced variance of DNV effects on a gene-level as compared to raw phenotype scores (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S12; Additional file <a href="#MOESM4" class="usa-link">4</a>: Table S3), variability between families persisted due to differences in functional domains and exon positions. Outlier analysis of intrafamilial deviations helped identify such genes, suggesting candidates for future studies. When we broadened the analysis to include less- or non-damaging dnMIS (MPC &lt; 2) and dnPTV (LOEUF ≥ 0.37), dnMIS exhibited greater variability than dnPTV as expected, likely due to their regional specificity (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S13; Additional file <a href="#MOESM4" class="usa-link">4</a>: Table S3). Furthermore, for missense variants, this variability often reflected domain-specific effects consistent with pathogenic hotspots or critical functional motifs. For protein-truncating variants, positional effects within the transcript, including the likelihood of nonsense-mediated decay escape, likely contributed to differential outcomes across families. This observation is consistent with previous work demonstrating that the exon-level position of mutations can be a major predictor of phenotypic similarity [<a href="#CR84" class="usa-link" aria-describedby="CR84">84</a>]. These mechanisms, together with isoform-specific expression and splicing, may underlie some of the observed heterogeneity, although larger sample sizes will be needed to systematically assess their impact.</p>
<p id="Par56">However, our study has several limitations. SRS data were unavailable for the SPARK, which may reduce cohort diversity. Moreover, there were negligible contributions of sex, age, and the location of assessment for the variance in WFSD. For example, SRS deviations were larger in females with ASD, school-age probands, and the SSC cohort (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S14) and VABS deviations were larger in females with ASD, school-age probands, and the Korean cohort (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S15). However, the adjustment for these covariates did not change the relative WFSD differences between dnPTV, dnMIS, and non-carriers (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S16). Parental SRS T-scores were lower than the general population mean, and sibling VABS scores were higher than the general population mean, suggesting that comparisons with general population norms might underestimate the DNV effects. However, we cannot exclude the possibility of ascertainment bias being introduced, as participating families could have higher education levels and lower social impairments than the broader population. To evaluate whether such bias could systematically influence the estimated DNV effects, we conducted exploratory analyses testing interactions between parental or sibling phenotypes and DNV carrier status, which did not show any significant interaction effects (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S17A, C). Additionally, mean parental educational attainment polygenic scores were not correlated with within-family deviations (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S17B, D). These results suggest that while ascertainment bias remains a potential limitation, it did not have a detectable impact on the estimation of DNV effects in this study. Additionally, while we focused on two key phenotypic domains—SRS and VABS, which are normalized across age and sex—broadening the scope to a wider range of neurodevelopmental phenotypes could provide a comprehensive understanding of ASD profiles.</p>
<p id="Par57">Our findings have significant clinical and biological implications. By providing accurate estimates of the effects of dnPTV and dnMIS, our approach can inform interventions, predictions, and treatment strategies tailored to individual genetic profiles. The identification of candidate genes associated with diverse clinical outcomes enhances our understanding of ASD etiology. Some genes with significant impacts were missed when using raw scores alone, underscoring the value of incorporating intrafamilial deviations in genetic analyses. Notably, 18 genes not listed in the SFARI Gene Data Base [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>] were recently reported to be associated with neurodevelopmental disorders [<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>–<a href="#CR91" class="usa-link" aria-describedby="CR91">91</a>], emphasizing the need for functional studies.</p>
<p id="Par58">We suggest that future studies integrate intrafamilial deviations to account for familial background effects. WFSD may facilitate personalized variant interpretation or support genetic counseling frameworks. Investigations into the sources of familial influence, such as inherited variants, environmental factors, or their interplay, would deepen our understanding. Additionally, while WFSD provides more specific estimates of DNV effects, the substantially smaller subset of probands with complete two-generation or sibling phenotyping imposes a further limitation. For example, while SRS scores were available in WFSD for a comparable number (Raw <em>N</em> = 2907; WFSD <em>N</em> = 2715), VABS data were not (Raw <em>N</em> = 10,232; WFSD N = 1955). This discrepancy likely contributed to missing signals in deviation-based analyses. Among the 28 genes that showed variable impacts on VABS raw scores but were un-assessable by WFSD (Additional file <a href="#MOESM1" class="usa-link">1</a>: Fig. S12), <em>PTEN</em> represents an informative example. In this case, individuals harboring mutations at residues Arg15 and Ile101 showed markedly low raw VABS scores (23 and 27, respectively). The Arg15 variant disrupts the PIP3-binding domain that mediates membrane localization and substrate interaction, while Ile101 impairs folding of the phosphatase domain and reduces catalytic activity. Both missense mutations have been shown in functional studies to cause severe loss of function [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. These findings support the possibility that some true signals were undetected due to incomplete family phenotyping. Collecting detailed family-based phenotypic assessments, including standardized measures such as SRS and VABS from parents and siblings, alongside functional studies of variant effects, will be essential to validate and extend these observations. Addressing these questions will advance our understanding of the neurobiological mechanisms underlying ASD variability and contribute to the development of precise support and interventions.</p></section><section id="Sec16"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par59">Accounting for familial background enables clinicians to predict the phenotypic outcomes of specific DNVs. This approach provides valuable insights into the biological mechanisms underlying ASD, enabling precise support and effective intervention strategies for individuals and families affected by ASD.</p></section><section id="Sec17"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366145/bin/13073_2025_1532_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">13073_2025_1532_MOESM1_ESM.docx</a><sup> (4.7MB, docx) </sup><p>Additional file 1: 17 supporting Figures S1-S17. Captions for each Fig. S1 to S17 are given within the file.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12366145/bin/13073_2025_1532_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13073_2025_1532_MOESM2_ESM.xlsx</a><sup> (21.6KB, xlsx) </sup><p>Additional file 2: Table S1. This table contains WFSD of each genetic subgroup: SRS deviations from parents (Table S1A), SRS deviations from siblings (Table S1B), VABS deviations from siblings (Table S1C), SCQ lifetime deviations from siblings (Table S1D), ADOS CSS total deviations from siblings (Table S1E), ADOS CSS SA deviations from siblings (Table S1F), ADOS CSS RRB deviations from siblings (Table S1G), and FSIQ deviations from siblings (Table S1H).</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12366145/bin/13073_2025_1532_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13073_2025_1532_MOESM3_ESM.xlsx</a><sup> (48.2KB, xlsx) </sup><p>Additional file 3: Table S2. This table contains enrichment results of dnDIS in ASD probands with severe SRS profiles (Table S2A), lists genes enriched in these probands (Table S2B), and shows enrichment results of each SRS gene set with GO pathways (Table S2C) and with cell-type specific DEGs in the developing human brain (Table S2D).</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12366145/bin/13073_2025_1532_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13073_2025_1532_MOESM4_ESM.xlsx</a><sup> (1.5MB, xlsx) </sup><p>Additional file 4: Table S3. This table contains MADs of phenotype outcomes of DNVs per gene: SRS deviations (Table S3A), and raw SRS scores (Table S3B).</p>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank all individuals and families in Korean Autism, SSC, and SPARK, the clinical sites, and the staff and appreciate obtaining access to genetic and phenotypic data for SSC and SPARK data on SFARI Base.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>DNV</dt>
<dd><p id="Par5">De novo variant</p></dd>
<dt>ASD</dt>
<dd><p id="Par6">Autism spectrum disorder</p></dd>
<dt>ID</dt>
<dd><p id="Par7">Intellectual disability</p></dd>
<dt>DD</dt>
<dd><p id="Par8">Developmental delay</p></dd>
<dt>WFSD</dt>
<dd><p id="Par9">Within-family standard deviation</p></dd>
<dt>dnPTV</dt>
<dd><p id="Par10">De novo protein-truncating variants</p></dd>
<dt>dnMIS</dt>
<dd><p id="Par11">De novo missense</p></dd>
<dt>SRS</dt>
<dd><p id="Par12">Social Responsiveness Scale</p></dd>
<dt>VABS</dt>
<dd><p id="Par13">Vineland Adaptive Behavior Scale</p></dd>
<dt>SCQ</dt>
<dd><p id="Par14">Social Communication Questionnaire</p></dd>
<dt>MAD</dt>
<dd><p id="Par15">Median absolute deviation</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>Study design, S.-W.K., D.M.W., S.H.K., S.J.S., H.J.Y., J.-Y.A. Acquisition, analysis, or interpretation of data: S.-W.K., H.L., D.Y.S., G.-H.L., J.H.H., J.W.L., H.J.B., J.H.S., Y.R.K., Y.L., E.K., D.M.W., S.H.K., S.J.S., H.J.Y., J.-Y.A. Drafting of the manuscript: S.-W.K., D.M.W., S.H.K., S.J.S., H.J.Y., J.-Y.A. Critical review of the manuscript for important intellectual content: S.-W.K., E.K., D.M.W., S.H.K., S.J.S., H.J.Y., J.-Y.A. Statistical analysis: S.-W.K., J.-Y.A. Administrative, technical, or material support: S.-W.K., H.L., D.Y.S., G.-H.L., S.H.K., H.J.Y., J.-Y.A. Supervision: E.K., D.M.W., S.H.K., S.J.S., H.J.Y., J.-Y.A. All authors read and approved the final manuscript.</p>
<section id="FPar1"><h3 class="pmc_sec_title">Authors’ information</h3>
<p id="Par60">These authors jointly supervised this work: Heejeong Yoo, Joon-Yong An.</p></section></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This research was funded by grants from the National Research Foundation (NRF) of Korea (NRF-2021M3E5D9021878 and RS-2024–00439474 to H.J.Y. and J.-Y.A., RS-2025–00553304 to J.-Y.A., RS-2023–00209635 and 2023S1A5C2A0709598711 to S.H.K.) Korea University Grant #K2420391 (to J.-Y.A.); the Simons Foundation Autism Research Initiative (606289 to D.M.W.); and the Institute for Basic Science, Korea (IBS-R002-D1 to E.K.). The funding source had no role in the design and conduct of the study; collection, management, analysis, and interpretation of the data; preparation, review, or approval of the manuscript; and decision to submit the manuscript for publication.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Due to local privacy laws and privileged human information, all requests for the raw genomic data from the Korean Autism cohort are subject to prior approval from the Institutional Review Board at Seoul National University Bundang Hospital. Interested researchers should submit a request to Dr. Hee Jeong Yoo, along with a detailed research plan outlining proposed analyses and data anonymization procedures. This plan will be reviewed by the IRB and data sharing committee at Seoul National University Bundang Hospital, with approval typically granted within two months. Upon approval, the requester will be added to the IRB as a collaborator for secure data sharing. Genetic and phenotypic data for the SSC and SPARK cohorts can be accessed by applying at <a href="https://base.sfari.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://base.sfari.org</a> [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>]. All major analysis scripts and code used to generate key figures will be made publicly available via Zenodo (10.5281/zenodo.15838146) [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>]. Additionally, summary-level statistics from the WFSD analyses and curated gene lists used in this study will be provided as Supplementary Tables to ensure reproducibility and transparency.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar2"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par61">The study was conducted in accordance with the ethical principles of the Declaration of Helsinki. Written informed consent was obtained from all participants. For minors or individuals unable to provide consent themselves, consent was obtained from their legal guardians. The ethics committees of Seoul National University Bundang Hospital (SNUBH; B-1703–388-303, B-2108–700-107), Soon Chun Hyang University Hospital Bucheon (SCHBC; SCHBC 2018–04-08, SCHBC 2022–04-016), and Seoul Child Hospital (SCH; P01-201908-BM-02, P01-202111–21-003) approved the study.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par62">Not applicable.</p></section><section id="FPar4"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par63">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Sandin S, Lichtenstein P, Kuja-Halkola R, Hultman C, Larsson H, Reichenberg A. The heritability of autism spectrum disorder. JAMA. 2017;318(12):1182–4.
</cite> [<a href="https://doi.org/10.1001/jama.2017.12141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5818813/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28973605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sandin%20S,%20Lichtenstein%20P,%20Kuja-Halkola%20R,%20Hultman%20C,%20Larsson%20H,%20Reichenberg%20A.%20The%20heritability%20of%20autism%20spectrum%20disorder.%20JAMA.%202017;318(12):1182%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>De Rubeis S, He X, Goldberg AP, Poultney CS, Samocha K, Cicek AE, et al. Synaptic, transcriptional and chromatin genes disrupted in autism. Nature. 2014;515(7526):209–15.
</cite> [<a href="https://doi.org/10.1038/nature13772" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4402723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25363760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?De%20Rubeis%20S,%20He%20X,%20Goldberg%20AP,%20Poultney%20CS,%20Samocha%20K,%20Cicek%20AE,%20et%20al.%20Synaptic,%20transcriptional%20and%20chromatin%20genes%20disrupted%20in%20autism.%20Nature.%202014;515(7526):209%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Iossifov I, O’Roak BJ, Sanders SJ, Ronemus M, Krumm N, Levy D, et al. The contribution of de novo coding mutations to autism spectrum disorder. Nature. 2014;515(7526):216–21.
</cite> [<a href="https://doi.org/10.1038/nature13908" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4313871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25363768/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Iossifov%20I,%20O%E2%80%99Roak%20BJ,%20Sanders%20SJ,%20Ronemus%20M,%20Krumm%20N,%20Levy%20D,%20et%20al.%20The%20contribution%20of%20de%20novo%20coding%20mutations%20to%20autism%20spectrum%20disorder.%20Nature.%202014;515(7526):216%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Sanders SJ, He X, Willsey AJ, Ercan-Sencicek AG, Samocha KE, Cicek AE, et al. Insights into autism spectrum disorder genomic architecture and biology from 71 risk loci. Neuron. 2015;87(6):1215–33.
</cite> [<a href="https://doi.org/10.1016/j.neuron.2015.09.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4624267/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26402605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sanders%20SJ,%20He%20X,%20Willsey%20AJ,%20Ercan-Sencicek%20AG,%20Samocha%20KE,%20Cicek%20AE,%20et%20al.%20Insights%20into%20autism%20spectrum%20disorder%20genomic%20architecture%20and%20biology%20from%2071%20risk%20loci.%20Neuron.%202015;87(6):1215%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Mefford HC, Sharp AJ, Baker C, Itsara A, Jiang Z, Buysse K, et al. Recurrent rearrangements of chromosome 1q21.1 and variable pediatric phenotypes. N Engl J Med. 2008;359(16):1685–99.
</cite> [<a href="https://doi.org/10.1056/NEJMoa0805384" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2703742/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18784092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mefford%20HC,%20Sharp%20AJ,%20Baker%20C,%20Itsara%20A,%20Jiang%20Z,%20Buysse%20K,%20et%20al.%20Recurrent%20rearrangements%20of%20chromosome%201q21.1%20and%20variable%20pediatric%20phenotypes.%20N%20Engl%20J%20Med.%202008;359(16):1685%E2%80%9399." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Helsmoortel C, Vulto-van Silfhout AT, Coe BP, Vandeweyer G, Rooms L, van den Ende J, et al. A SWI/SNF-related autism syndrome caused by de novo mutations in ADNP. Nat Genet. 2014;46(4):380–4.
</cite> [<a href="https://doi.org/10.1038/ng.2899" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3990853/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24531329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Helsmoortel%20C,%20Vulto-van%20Silfhout%20AT,%20Coe%20BP,%20Vandeweyer%20G,%20Rooms%20L,%20van%20den%20Ende%20J,%20et%20al.%20A%20SWI/SNF-related%20autism%20syndrome%20caused%20by%20de%20novo%20mutations%20in%20ADNP.%20Nat%20Genet.%202014;46(4):380%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Stessman HAF, Willemsen MH, Fenckova M, Penn O, Hoischen A, Xiong B, et al. Disruption of POGZ is associated with intellectual disability and autism spectrum disorders. Am J Hum Genet. 2016;98(3):541–52.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2016.02.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4890241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26942287/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stessman%20HAF,%20Willemsen%20MH,%20Fenckova%20M,%20Penn%20O,%20Hoischen%20A,%20Xiong%20B,%20et%20al.%20Disruption%20of%20POGZ%20is%20associated%20with%20intellectual%20disability%20and%20autism%20spectrum%20disorders.%20Am%20J%20Hum%20Genet.%202016;98(3):541%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Satterstrom FK, Kosmicki JA, Wang J, Breen MS, De Rubeis S, An J-Y, et al. Large-scale exome sequencing study implicates both developmental and functional changes in the neurobiology of autism. Cell. 2020;180(3):568-84. e23.
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.12.036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7250485/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31981491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Satterstrom%20FK,%20Kosmicki%20JA,%20Wang%20J,%20Breen%20MS,%20De%20Rubeis%20S,%20An%20J-Y,%20et%20al.%20Large-scale%20exome%20sequencing%20study%20implicates%20both%20developmental%20and%20functional%20changes%20in%20the%20neurobiology%20of%20autism.%20Cell.%202020;180(3):568-84.%20e23." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Kaplanis J, Samocha KE, Wiel L, Zhang Z, Arvai KJ, Eberhardt RY, et al. Evidence for 28 genetic disorders discovered by combining healthcare and research data. Nature. 2020;586(7831):757–62.
</cite> [<a href="https://doi.org/10.1038/s41586-020-2832-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7116826/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33057194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaplanis%20J,%20Samocha%20KE,%20Wiel%20L,%20Zhang%20Z,%20Arvai%20KJ,%20Eberhardt%20RY,%20et%20al.%20Evidence%20for%2028%20genetic%20disorders%20discovered%20by%20combining%20healthcare%20and%20research%20data.%20Nature.%202020;586(7831):757%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Fu JM, Satterstrom FK, Peng M, Brand H, Collins RL, Dong S, et al. Rare coding variation provides insight into the genetic architecture and phenotypic context of autism. Nat Genet. 2022;54(9):1320–31.
</cite> [<a href="https://doi.org/10.1038/s41588-022-01104-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9653013/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35982160/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fu%20JM,%20Satterstrom%20FK,%20Peng%20M,%20Brand%20H,%20Collins%20RL,%20Dong%20S,%20et%20al.%20Rare%20coding%20variation%20provides%20insight%20into%20the%20genetic%20architecture%20and%20phenotypic%20context%20of%20autism.%20Nat%20Genet.%202022;54(9):1320%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Wilfert AB, Sulovari A, Turner TN, Coe BP, Eichler EE. Recurrent de novo mutations in neurodevelopmental disorders: properties and clinical implications. Genome Med. 2017;9(1):101.
</cite> [<a href="https://doi.org/10.1186/s13073-017-0498-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5704398/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29179772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilfert%20AB,%20Sulovari%20A,%20Turner%20TN,%20Coe%20BP,%20Eichler%20EE.%20Recurrent%20de%20novo%20mutations%20in%20neurodevelopmental%20disorders:%20properties%20and%20clinical%20implications.%20Genome%20Med.%202017;9(1):101." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Pauli RM. Achondroplasia: a comprehensive clinical review. Orphanet J Rare Dis. 2019;14(1):1.
</cite> [<a href="https://doi.org/10.1186/s13023-018-0972-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6318916/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30606190/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pauli%20RM.%20Achondroplasia:%20a%20comprehensive%20clinical%20review.%20Orphanet%20J%20Rare%20Dis.%202019;14(1):1." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Durand C, Rappold GA. Height matters-from monogenic disorders to normal variation. Nat Rev Endocrinol. 2013;9(3):171–7.
</cite> [<a href="https://doi.org/10.1038/nrendo.2012.251" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23337954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Durand%20C,%20Rappold%20GA.%20Height%20matters-from%20monogenic%20disorders%20to%20normal%20variation.%20Nat%20Rev%20Endocrinol.%202013;9(3):171%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Moreno-De-Luca A, Evans DW, Boomer KB, Hanson E, Bernier R, Goin-Kochel RP, et al. The role of parental cognitive, behavioral, and motor profiles in clinical variability in individuals with chromosome 16p11.2 deletions. JAMA Psychiatry. 2015;72(2):119–26.
</cite> [<a href="https://doi.org/10.1001/jamapsychiatry.2014.2147" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25493922/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moreno-De-Luca%20A,%20Evans%20DW,%20Boomer%20KB,%20Hanson%20E,%20Bernier%20R,%20Goin-Kochel%20RP,%20et%20al.%20The%20role%20of%20parental%20cognitive,%20behavioral,%20and%20motor%20profiles%20in%20clinical%20variability%20in%20individuals%20with%20chromosome%2016p11.2%20deletions.%20JAMA%20Psychiatry.%202015;72(2):119%E2%80%9326." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Polyak A, Rosenfeld JA, Girirajan S. An assessment of sex bias in neurodevelopmental disorders. Genome Med. 2015;7(1):94.
</cite> [<a href="https://doi.org/10.1186/s13073-015-0216-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4549901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26307204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Polyak%20A,%20Rosenfeld%20JA,%20Girirajan%20S.%20An%20assessment%20of%20sex%20bias%20in%20neurodevelopmental%20disorders.%20Genome%20Med.%202015;7(1):94." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Davies RW, Fiksinski AM, Breetvelt EJ, Williams NM, Hooper SR, Monfeuga T, et al. Using common genetic variation to examine phenotypic expression and risk prediction in 22q11.2 deletion syndrome. Nat Med. 2020;26(12):1912–8.
</cite> [<a href="https://doi.org/10.1038/s41591-020-1103-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7975627/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33169016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Davies%20RW,%20Fiksinski%20AM,%20Breetvelt%20EJ,%20Williams%20NM,%20Hooper%20SR,%20Monfeuga%20T,%20et%20al.%20Using%20common%20genetic%20variation%20to%20examine%20phenotypic%20expression%20and%20risk%20prediction%20in%2022q11.2%20deletion%20syndrome.%20Nat%20Med.%202020;26(12):1912%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Smolen C, Jensen M, Dyer L, Pizzo L, Tyryshkina A, Banerjee D, et al. Assortative mating and parental genetic relatedness contribute to the pathogenicity of variably expressive variants. Am J Hum Genet. 2023;110(12):2015–28.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2023.10.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10716518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37979581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smolen%20C,%20Jensen%20M,%20Dyer%20L,%20Pizzo%20L,%20Tyryshkina%20A,%20Banerjee%20D,%20et%20al.%20Assortative%20mating%20and%20parental%20genetic%20relatedness%20contribute%20to%20the%20pathogenicity%20of%20variably%20expressive%20variants.%20Am%20J%20Hum%20Genet.%202023;110(12):2015%E2%80%9328." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Kingdom R, Wright CF. Incomplete penetrance and variable expressivity: from clinical studies to population cohorts. Front Genet. 2022;13:920390.
</cite> [<a href="https://doi.org/10.3389/fgene.2022.920390" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9380816/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35983412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kingdom%20R,%20Wright%20CF.%20Incomplete%20penetrance%20and%20variable%20expressivity:%20from%20clinical%20studies%20to%20population%20cohorts.%20Front%20Genet.%202022;13:920390." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Vorstman JAS, Parr JR, Moreno-De-Luca D, Anney RJL, Nurnberger JI Jr, Hallmayer JF. Autism genetics: opportunities and challenges for clinical translation. Nat Rev Genet. 2017;18(6):362–76.
</cite> [<a href="https://doi.org/10.1038/nrg.2017.4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28260791/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vorstman%20JAS,%20Parr%20JR,%20Moreno-De-Luca%20D,%20Anney%20RJL,%20Nurnberger%20JI%20Jr,%20Hallmayer%20JF.%20Autism%20genetics:%20opportunities%20and%20challenges%20for%20clinical%20translation.%20Nat%20Rev%20Genet.%202017;18(6):362%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Hanson E, Bernier R, Porche K, Jackson FI, Goin-Kochel RP, Snyder LG, et al. The cognitive and behavioral phenotype of the 16p11.2 deletion in a clinically ascertained population. Biol Psychiatry. 2015;77(9):785–93.
</cite> [<a href="https://doi.org/10.1016/j.biopsych.2014.04.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5410712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25064419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hanson%20E,%20Bernier%20R,%20Porche%20K,%20Jackson%20FI,%20Goin-Kochel%20RP,%20Snyder%20LG,%20et%20al.%20The%20cognitive%20and%20behavioral%20phenotype%20of%20the%2016p11.2%20deletion%20in%20a%20clinically%20ascertained%20population.%20Biol%20Psychiatry.%202015;77(9):785%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Moreno-De-Luca A, Myers SM, Challman TD, Moreno-De-Luca D, Evans DW, Ledbetter DH. Developmental brain dysfunction: revival and expansion of old concepts based on new genetic evidence. Lancet Neurol. 2013;12(4):406–14.
</cite> [<a href="https://doi.org/10.1016/S1474-4422(13)70011-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4013791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23518333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moreno-De-Luca%20A,%20Myers%20SM,%20Challman%20TD,%20Moreno-De-Luca%20D,%20Evans%20DW,%20Ledbetter%20DH.%20Developmental%20brain%20dysfunction:%20revival%20and%20expansion%20of%20old%20concepts%20based%20on%20new%20genetic%20evidence.%20Lancet%20Neurol.%202013;12(4):406%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Chaste P, Klei L, Sanders SJ, Murtha MT, Hus V, Lowe JK, et al. Adjusting head circumference for covariates in autism: clinical correlates of a highly heritable continuous trait. Biol Psychiatry. 2013;74(8):576–84.
</cite> [<a href="https://doi.org/10.1016/j.biopsych.2013.04.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3772969/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23746936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chaste%20P,%20Klei%20L,%20Sanders%20SJ,%20Murtha%20MT,%20Hus%20V,%20Lowe%20JK,%20et%20al.%20Adjusting%20head%20circumference%20for%20covariates%20in%20autism:%20clinical%20correlates%20of%20a%20highly%20heritable%20continuous%20trait.%20Biol%20Psychiatry.%202013;74(8):576%E2%80%9384." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Ben-Shalom R, Keeshen CM, Berrios KN, An JY, Sanders SJ, Bender KJ. Opposing effects on Na(V)1.2 function underlie differences between SCN2A variants observed in individuals with autism spectrum disorder or infantile seizures. Biol Psychiatry. 2017;82(3):224–32.
</cite> [<a href="https://doi.org/10.1016/j.biopsych.2017.01.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5796785/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28256214/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ben-Shalom%20R,%20Keeshen%20CM,%20Berrios%20KN,%20An%20JY,%20Sanders%20SJ,%20Bender%20KJ.%20Opposing%20effects%20on%20Na(V)1.2%20function%20underlie%20differences%20between%20SCN2A%20variants%20observed%20in%20individuals%20with%20autism%20spectrum%20disorder%20or%20infantile%20seizures.%20Biol%20Psychiatry.%202017;82(3):224%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Geisheker MR, Heymann G, Wang T, Coe BP, Turner TN, Stessman HAF, et al. Hotspots of missense mutation identify neurodevelopmental disorder genes and functional domains. Nat Neurosci. 2017;20(8):1043–51.
</cite> [<a href="https://doi.org/10.1038/nn.4589" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5539915/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28628100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Geisheker%20MR,%20Heymann%20G,%20Wang%20T,%20Coe%20BP,%20Turner%20TN,%20Stessman%20HAF,%20et%20al.%20Hotspots%20of%20missense%20mutation%20identify%20neurodevelopmental%20disorder%20genes%20and%20functional%20domains.%20Nat%20Neurosci.%202017;20(8):1043%E2%80%9351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Chau KK, Zhang P, Urresti J, Amar M, Pramod AB, Chen J, et al. Full-length isoform transcriptome of the developing human brain provides further insights into autism. Cell Rep. 2021;36(9):109631.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2021.109631" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8437376/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34469739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chau%20KK,%20Zhang%20P,%20Urresti%20J,%20Amar%20M,%20Pramod%20AB,%20Chen%20J,%20et%20al.%20Full-length%20isoform%20transcriptome%20of%20the%20developing%20human%20brain%20provides%20further%20insights%20into%20autism.%20Cell%20Rep.%202021;36(9):109631." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Jiang YH, Ehlers MD. Modeling autism by SHANK gene mutations in mice. Neuron. 2013;78(1):8–27.
</cite> [<a href="https://doi.org/10.1016/j.neuron.2013.03.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3659167/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23583105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang%20YH,%20Ehlers%20MD.%20Modeling%20autism%20by%20SHANK%20gene%20mutations%20in%20mice.%20Neuron.%202013;78(1):8%E2%80%9327." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Lu X, Ni P, Suarez-Meade P, Ma Y, Forrest EN, Wang G, et al. Transcriptional determinism and stochasticity contribute to the complexity of autism-associated SHANK family genes. Cell Rep. 2024;43(7):114376.
</cite> [<a href="https://doi.org/10.1016/j.celrep.2024.114376" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11328446/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38900637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu%20X,%20Ni%20P,%20Suarez-Meade%20P,%20Ma%20Y,%20Forrest%20EN,%20Wang%20G,%20et%20al.%20Transcriptional%20determinism%20and%20stochasticity%20contribute%20to%20the%20complexity%20of%20autism-associated%20SHANK%20family%20genes.%20Cell%20Rep.%202024;43(7):114376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Kim SW, Lee H, Song DY, Lee GH, Ji J, Park JW, et al. Whole genome sequencing analysis identifies sex differences of familial pattern contributing to phenotypic diversity in autism. Genome Med. 2024;16(1):114.
</cite> [<a href="https://doi.org/10.1186/s13073-024-01385-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11429951/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39334436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20SW,%20Lee%20H,%20Song%20DY,%20Lee%20GH,%20Ji%20J,%20Park%20JW,%20et%20al.%20Whole%20genome%20sequencing%20analysis%20identifies%20sex%20differences%20of%20familial%20pattern%20contributing%20to%20phenotypic%20diversity%20in%20autism.%20Genome%20Med.%202024;16(1):114." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Fischbach GD, Lord C. The Simons Simplex Collection: a resource for identification of autism genetic risk factors. Neuron. 2010;68(2):192–5.
</cite> [<a href="https://doi.org/10.1016/j.neuron.2010.10.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20955926/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fischbach%20GD,%20Lord%20C.%20The%20Simons%20Simplex%20Collection:%20a%20resource%20for%20identification%20of%20autism%20genetic%20risk%20factors.%20Neuron.%202010;68(2):192%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Spark C. SPARK: a US cohort of 50,000 families to accelerate autism research. Neuron. 2018;97(3):488–93.
</cite> [<a href="https://doi.org/10.1016/j.neuron.2018.01.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7444276/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29420931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Spark%20C.%20SPARK:%20a%20US%20cohort%20of%2050,000%20families%20to%20accelerate%20autism%20research.%20Neuron.%202018;97(3):488%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Werling DM, Brand H, An JY, Stone MR, Zhu L, Glessner JT, et al. An analytical framework for whole-genome sequence association studies and its implications for autism spectrum disorder. Nat Genet. 2018;50(5):727–36.
</cite> [<a href="https://doi.org/10.1038/s41588-018-0107-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5961723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29700473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Werling%20DM,%20Brand%20H,%20An%20JY,%20Stone%20MR,%20Zhu%20L,%20Glessner%20JT,%20et%20al.%20An%20analytical%20framework%20for%20whole-genome%20sequence%20association%20studies%20and%20its%20implications%20for%20autism%20spectrum%20disorder.%20Nat%20Genet.%202018;50(5):727%E2%80%9336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Karczewski KJ, Francioli LC, Tiao G, Cummings BB, Alföldi J, Wang Q, et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature. 2020;581(7809):434–43.
</cite> [<a href="https://doi.org/10.1038/s41586-020-2308-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7334197/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32461654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Karczewski%20KJ,%20Francioli%20LC,%20Tiao%20G,%20Cummings%20BB,%20Alf%C3%B6ldi%20J,%20Wang%20Q,%20et%20al.%20The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141,456%20humans.%20Nature.%202020;581(7809):434%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Samocha KE, Kosmicki JA, Karczewski KJ, O’Donnell-Luria AH, Pierce-Hoffman E, MacArthur DG, et al. Regional missense constraint improves variant deleteriousness prediction. bioRxiv. 2017. Available from. 10.1101/148353.</cite>
</li>
<li id="CR34">
<span class="label">34.</span><cite>Ge T, Chen CY, Ni Y, Feng YA, Smoller JW. Polygenic prediction via Bayesian regression and continuous shrinkage priors. Nat Commun. 2019;10(1):1776.
</cite> [<a href="https://doi.org/10.1038/s41467-019-09718-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6467998/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30992449/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ge%20T,%20Chen%20CY,%20Ni%20Y,%20Feng%20YA,%20Smoller%20JW.%20Polygenic%20prediction%20via%20Bayesian%20regression%20and%20continuous%20shrinkage%20priors.%20Nat%20Commun.%202019;10(1):1776." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Grove J, Ripke S, Als TD, Mattheisen M, Walters RK, Won H, et al. Identification of common genetic risk variants for autism spectrum disorder. Nat Genet. 2019;51(3):431–44.
</cite> [<a href="https://doi.org/10.1038/s41588-019-0344-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6454898/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30804558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grove%20J,%20Ripke%20S,%20Als%20TD,%20Mattheisen%20M,%20Walters%20RK,%20Won%20H,%20et%20al.%20Identification%20of%20common%20genetic%20risk%20variants%20for%20autism%20spectrum%20disorder.%20Nat%20Genet.%202019;51(3):431%E2%80%9344." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Lee JJ, Wedow R, Okbay A, Kong E, Maghzian O, Zacher M, et al. Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals. Nat Genet. 2018;50(8):1112–21.
</cite> [<a href="https://doi.org/10.1038/s41588-018-0147-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6393768/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30038396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee%20JJ,%20Wedow%20R,%20Okbay%20A,%20Kong%20E,%20Maghzian%20O,%20Zacher%20M,%20et%20al.%20Gene%20discovery%20and%20polygenic%20prediction%20from%20a%20genome-wide%20association%20study%20of%20educational%20attainment%20in%201.1%20million%20individuals.%20Nat%20Genet.%202018;50(8):1112%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Lord C, Rutter M, DiLavore P, Risi S, Gotham K, Bishop S. Autism Diagnostic Observation Schedule–(ADOS-2), 2nd Edn Los Angeles: Western Psychological Corporation; 2012.</cite>
</li>
<li id="CR38">
<span class="label">38.</span><cite>Yoo H, Bong G, Kwak Y, Lee M, Cho S, Kim B, et al. Korean autism diagnostic observation schedule-2 (K-ADOS-2). Seoul: Hakjisa; 2018.</cite>
</li>
<li id="CR39">
<span class="label">39.</span><cite>Constantino JN, Gruber CP. Social responsiveness scale: SRS-2. Los Angeles: Western Psychological Services; 2012.</cite>
</li>
<li id="CR40">
<span class="label">40.</span><cite>Rutter M, Bailey A, Lord C. The social communication questionnaire. Los Angeles: Western Psychological Services; 2003.</cite>
</li>
<li id="CR41">
<span class="label">41.</span><cite>Kim J-H, Sunwoo H-J, Park S-B, Noh D-H, Jung YK, Cho I-H, et al. A validation study of the Korean version of social communication questionnaire. J Korean Acad Child Adolesc Psychiatry. 2015;26(3):197–208.</cite> [<a href="https://scholar.google.com/scholar_lookup?Kim%20J-H,%20Sunwoo%20H-J,%20Park%20S-B,%20Noh%20D-H,%20Jung%20YK,%20Cho%20I-H,%20et%20al.%20A%20validation%20study%20of%20the%20Korean%20version%20of%20social%20communication%20questionnaire.%20J%20Korean%20Acad%20Child%20Adolesc%20Psychiatry.%202015;26(3):197%E2%80%93208." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Rutter M, Le Couteur A, Lord C. Autism diagnostic interview-revised. Los Angeles, CA: Western Psychological Services. 2003;29(2003):30.</cite>
</li>
<li id="CR43">
<span class="label">43.</span><cite>Lam KS, Aman MG. The repetitive behavior scale-revised: independent validation in individuals with autism spectrum disorders. J Autism Dev Disord. 2007;37(5):855–66.
</cite> [<a href="https://doi.org/10.1007/s10803-006-0213-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17048092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lam%20KS,%20Aman%20MG.%20The%20repetitive%20behavior%20scale-revised:%20independent%20validation%20in%20individuals%20with%20autism%20spectrum%20disorders.%20J%20Autism%20Dev%20Disord.%202007;37(5):855%E2%80%9366." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Wechsler D. WPPSI-III administration and scoring manual. Psychological Corporation; 2002.</cite>
</li>
<li id="CR45">
<span class="label">45.</span><cite>Wechsler D. Wechsler intelligence scale for children–Fourth Edition (WISC-IV). San Antonio: Harcourt Assessment; 2003.</cite>
</li>
<li id="CR46">
<span class="label">46.</span><cite>Roid GH, Miller LJ. Leiter international performance scale-revised (Leiter-R). Wood Dale, IL: Stoelting. 1997;10.</cite>
</li>
<li id="CR47">
<span class="label">47.</span><cite>Sparrow S, Cicchetti D, Balla D. Vineland Adaptive Behavior Scales–2nd edition (VABS-II). Livonia: Pearson Assessments; 2005.</cite>
</li>
<li id="CR48">
<span class="label">48.</span><cite>Wilson BN, Crawford SG, Green D, Roberts G, Aylott A, Kaplan BJ. Psychometric properties of the revised developmental coordination disorder questionnaire. Phys Occup Ther Pediatr. 2009;29(2):182–202.
</cite> [<a href="https://doi.org/10.1080/01942630902784761" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19401931/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wilson%20BN,%20Crawford%20SG,%20Green%20D,%20Roberts%20G,%20Aylott%20A,%20Kaplan%20BJ.%20Psychometric%20properties%20of%20the%20revised%20developmental%20coordination%20disorder%20questionnaire.%20Phys%20Occup%20Ther%20Pediatr.%202009;29(2):182%E2%80%93202." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Cheon KA, Park JI, Koh YJ, Song J, Hong HJ, Kim YK, et al. The social responsiveness scale in relation to DSM IV and DSM5 ASD in Korean children. Autism Res. 2016;9(9):970–80.
</cite> [<a href="https://doi.org/10.1002/aur.1671" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5370077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27604989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheon%20KA,%20Park%20JI,%20Koh%20YJ,%20Song%20J,%20Hong%20HJ,%20Kim%20YK,%20et%20al.%20The%20social%20responsiveness%20scale%20in%20relation%20to%20DSM%20IV%20and%20DSM5%20ASD%20in%20Korean%20children.%20Autism%20Res.%202016;9(9):970%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Kim JH, Kim YA, Song DY, Cho HB, Lee HB, Park JH, et al. An intervention program targeting daily adaptive skills through executive function training for adults with autism spectrum disorder: a pilot study. Psychiatry Investig. 2021;18(6):513–22.
</cite> [<a href="https://doi.org/10.30773/pi.2020.0423" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8256144/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34218640/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20JH,%20Kim%20YA,%20Song%20DY,%20Cho%20HB,%20Lee%20HB,%20Park%20JH,%20et%20al.%20An%20intervention%20program%20targeting%20daily%20adaptive%20skills%20through%20executive%20function%20training%20for%20adults%20with%20autism%20spectrum%20disorder:%20a%20pilot%20study.%20Psychiatry%20Investig.%202021;18(6):513%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Yu G, Wang LG, Han Y, He QY. clusterProfiler: an R package for comparing biological themes among gene clusters. OMICS. 2012;16(5):284–7.
</cite> [<a href="https://doi.org/10.1089/omi.2011.0118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3339379/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22455463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20G,%20Wang%20LG,%20Han%20Y,%20He%20QY.%20clusterProfiler:%20an%20R%20package%20for%20comparing%20biological%20themes%20among%20gene%20clusters.%20OMICS.%202012;16(5):284%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Kim S, Lee J, Koh IG, Ji J, Kim HJ, Kim E, et al. An integrative single-cell atlas for exploring the cellular and temporal specificity of genes related to neurological disorders during human brain development. Exp Mol Med. 2024;56(10):2271–82.
</cite> [<a href="https://doi.org/10.1038/s12276-024-01328-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11541755/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39363111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20S,%20Lee%20J,%20Koh%20IG,%20Ji%20J,%20Kim%20HJ,%20Kim%20E,%20et%20al.%20An%20integrative%20single-cell%20atlas%20for%20exploring%20the%20cellular%20and%20temporal%20specificity%20of%20genes%20related%20to%20neurological%20disorders%20during%20human%20brain%20development.%20Exp%20Mol%20Med.%202024;56(10):2271%E2%80%9382." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Bishop SL, Farmer C, Bal V, Robinson EB, Willsey AJ, Werling DM, et al. Identification of developmental and behavioral markers associated with genetic abnormalities in autism spectrum disorder. Am J Psychiatry. 2017;174(6):576–85.
</cite> [<a href="https://doi.org/10.1176/appi.ajp.2017.16101115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5578709/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28253736/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bishop%20SL,%20Farmer%20C,%20Bal%20V,%20Robinson%20EB,%20Willsey%20AJ,%20Werling%20DM,%20et%20al.%20Identification%20of%20developmental%20and%20behavioral%20markers%20associated%20with%20genetic%20abnormalities%20in%20autism%20spectrum%20disorder.%20Am%20J%20Psychiatry.%202017;174(6):576%E2%80%9385." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Constantino JN, Todd RD. Intergenerational transmission of subthreshold autistic traits in the general population. Biol Psychiatry. 2005;57(6):655–60.
</cite> [<a href="https://doi.org/10.1016/j.biopsych.2004.12.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15780853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Constantino%20JN,%20Todd%20RD.%20Intergenerational%20transmission%20of%20subthreshold%20autistic%20traits%20in%20the%20general%20population.%20Biol%20Psychiatry.%202005;57(6):655%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Banerjee-Basu S, Packer A. SFARI Gene: an evolving database for the autism research community. Dis Model Mech. 2010;3(3–4):133–5.
</cite> [<a href="https://doi.org/10.1242/dmm.005439" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20212079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Banerjee-Basu%20S,%20Packer%20A.%20SFARI%20Gene:%20an%20evolving%20database%20for%20the%20autism%20research%20community.%20Dis%20Model%20Mech.%202010;3(3%E2%80%934):133%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Liu L, Lei J, Sanders SJ, Willsey AJ, Kou Y, Cicek AE, et al. DAWN: a framework to identify autism genes and subnetworks using gene expression and genetics. Mol Autism. 2014;5(1):22.
</cite> [<a href="https://doi.org/10.1186/2040-2392-5-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4016412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24602502/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20L,%20Lei%20J,%20Sanders%20SJ,%20Willsey%20AJ,%20Kou%20Y,%20Cicek%20AE,%20et%20al.%20DAWN:%20a%20framework%20to%20identify%20autism%20genes%20and%20subnetworks%20using%20gene%20expression%20and%20genetics.%20Mol%20Autism.%202014;5(1):22." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Gabriele M, Lopez Tobon A, D’Agostino G, Testa G. The chromatin basis of neurodevelopmental disorders: rethinking dysfunction along the molecular and temporal axes. Prog Neuropsychopharmacol Biol Psychiatry. 2018;84(Pt B):306–27.
</cite> [<a href="https://doi.org/10.1016/j.pnpbp.2017.12.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29309830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gabriele%20M,%20Lopez%20Tobon%20A,%20D%E2%80%99Agostino%20G,%20Testa%20G.%20The%20chromatin%20basis%20of%20neurodevelopmental%20disorders:%20rethinking%20dysfunction%20along%20the%20molecular%20and%20temporal%20axes.%20Prog%20Neuropsychopharmacol%20Biol%20Psychiatry.%202018;84(Pt%20B):306%E2%80%9327." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Coe BP, Stessman HAF, Sulovari A, Geisheker MR, Bakken TE, Lake AM, et al. Neurodevelopmental disease genes implicated by de novo mutation and copy number variation morbidity. Nat Genet. 2019;51(1):106–16.
</cite> [<a href="https://doi.org/10.1038/s41588-018-0288-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6309590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30559488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Coe%20BP,%20Stessman%20HAF,%20Sulovari%20A,%20Geisheker%20MR,%20Bakken%20TE,%20Lake%20AM,%20et%20al.%20Neurodevelopmental%20disease%20genes%20implicated%20by%20de%20novo%20mutation%20and%20copy%20number%20variation%20morbidity.%20Nat%20Genet.%202019;51(1):106%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>O’Donnell-Luria AH, Pais LS, Faundes V, Wood JC, Sveden A, Luria V, et al. Heterozygous variants in KMT2E cause a spectrum of neurodevelopmental disorders and epilepsy. Am J Hum Genet. 2019;104(6):1210–22.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2019.03.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6556837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31079897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?O%E2%80%99Donnell-Luria%20AH,%20Pais%20LS,%20Faundes%20V,%20Wood%20JC,%20Sveden%20A,%20Luria%20V,%20et%20al.%20Heterozygous%20variants%20in%20KMT2E%20cause%20a%20spectrum%20of%20neurodevelopmental%20disorders%20and%20epilepsy.%20Am%20J%20Hum%20Genet.%202019;104(6):1210%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Wang B, Vartak R, Zaltsman Y, Naing ZZC, Hennick KM, Polacco BJ, et al. A foundational atlas of autism protein interactions reveals molecular convergence. bioRxiv. 2024. Available from. 10.1101/2023.12.03.569805.</cite>
</li>
<li id="CR61">
<span class="label">61.</span><cite>Boudkkazi S, Brechet A, Schwenk J, Fakler B. Cornichon2 dictates the time course of excitatory transmission at individual hippocampal synapses. Neuron. 2014;82(4):848–58.
</cite> [<a href="https://doi.org/10.1016/j.neuron.2014.03.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24853943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boudkkazi%20S,%20Brechet%20A,%20Schwenk%20J,%20Fakler%20B.%20Cornichon2%20dictates%20the%20time%20course%20of%20excitatory%20transmission%20at%20individual%20hippocampal%20synapses.%20Neuron.%202014;82(4):848%E2%80%9358." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Schlingmann KP, Bandulik S, Mammen C, Tarailo-Graovac M, Holm R, Baumann M, et al. Germline de novo mutations in ATP1A1 cause renal hypomagnesemia, refractory seizures, and intellectual disability. Am J Hum Genet. 2018;103(5):808–16.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2018.10.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6218849/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30388404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schlingmann%20KP,%20Bandulik%20S,%20Mammen%20C,%20Tarailo-Graovac%20M,%20Holm%20R,%20Baumann%20M,%20et%20al.%20Germline%20de%20novo%20mutations%20in%20ATP1A1%20cause%20renal%20hypomagnesemia,%20refractory%20seizures,%20and%20intellectual%20disability.%20Am%20J%20Hum%20Genet.%202018;103(5):808%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Poggio E, Barazzuol L, Salmaso A, Milani C, Deligiannopoulou A, Cazorla ÁG, et al. ATP2B2 de novo variants as a cause of variable neurodevelopmental disorders that feature dystonia, ataxia, intellectual disability, behavioral symptoms, and seizures. Genet Med. 2023;25(12):100971.
</cite> [<a href="https://doi.org/10.1016/j.gim.2023.100971" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37675773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Poggio%20E,%20Barazzuol%20L,%20Salmaso%20A,%20Milani%20C,%20Deligiannopoulou%20A,%20Cazorla%20%C3%81G,%20et%20al.%20ATP2B2%20de%20novo%20variants%20as%20a%20cause%20of%20variable%20neurodevelopmental%20disorders%20that%20feature%20dystonia,%20ataxia,%20intellectual%20disability,%20behavioral%20symptoms,%20and%20seizures.%20Genet%20Med.%202023;25(12):100971." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Song MS, Salmena L, Pandolfi PP. The functions and regulation of the PTEN tumour suppressor. Nat Rev Mol Cell Biol. 2012;13(5):283–96.</cite> [<a href="https://doi.org/10.1038/nrm3330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22473468/" class="usa-link">PubMed</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>RK CY, Merico D, Bookman M, J LH, Thiruvahindrapuram B, Patel RV, et al. Whole genome sequencing resource identifies 18 new candidate genes for autism spectrum disorder. Nat Neurosci. 2017;20(4):602–11.</cite> [<a href="https://doi.org/10.1038/nn.4524" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5501701/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28263302/" class="usa-link">PubMed</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Post KL, Belmadani M, Ganguly P, Meili F, Dingwall R, McDiarmid TA, et al. Multi-model functionalization of disease-associated PTEN missense mutations identifies multiple molecular mechanisms underlying protein dysfunction. Nat Commun. 2020;11(1):2073.
</cite> [<a href="https://doi.org/10.1038/s41467-020-15943-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7190743/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32350270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Post%20KL,%20Belmadani%20M,%20Ganguly%20P,%20Meili%20F,%20Dingwall%20R,%20McDiarmid%20TA,%20et%20al.%20Multi-model%20functionalization%20of%20disease-associated%20PTEN%20missense%20mutations%20identifies%20multiple%20molecular%20mechanisms%20underlying%20protein%20dysfunction.%20Nat%20Commun.%202020;11(1):2073." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Rodríguez-Escudero I, Oliver MD, Andrés-Pons A, Molina M, Cid VJ, Pulido R. A comprehensive functional analysis of PTEN mutations: implications in tumor- and autism-related syndromes. Hum Mol Genet. 2011;20(21):4132–42.
</cite> [<a href="https://doi.org/10.1093/hmg/ddr337" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21828076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rodr%C3%ADguez-Escudero%20I,%20Oliver%20MD,%20Andr%C3%A9s-Pons%20A,%20Molina%20M,%20Cid%20VJ,%20Pulido%20R.%20A%20comprehensive%20functional%20analysis%20of%20PTEN%20mutations:%20implications%20in%20tumor-%20and%20autism-related%20syndromes.%20Hum%20Mol%20Genet.%202011;20(21):4132%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Leitner MG, Hobiger K, Mavrantoni A, Feuer A, Oberwinkler J, Oliver D, et al. A126 in the active site and TI167/168 in the TI loop are essential determinants of the substrate specificity of PTEN. Cell Mol Life Sci. 2018;75(22):4235–50.
</cite> [<a href="https://doi.org/10.1007/s00018-018-2867-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6182344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29987362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leitner%20MG,%20Hobiger%20K,%20Mavrantoni%20A,%20Feuer%20A,%20Oberwinkler%20J,%20Oliver%20D,%20et%20al.%20A126%20in%20the%20active%20site%20and%20TI167/168%20in%20the%20TI%20loop%20are%20essential%20determinants%20of%20the%20substrate%20specificity%20of%20PTEN.%20Cell%20Mol%20Life%20Sci.%202018;75(22):4235%E2%80%9350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Mighell TL, Evans-Dutson S, O’Roak BJ. A saturation mutagenesis approach to understanding PTEN lipid phosphatase activity and genotype-phenotype relationships. Am J Hum Genet. 2018;102(5):943–55.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2018.03.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5986715/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29706350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mighell%20TL,%20Evans-Dutson%20S,%20O%E2%80%99Roak%20BJ.%20A%20saturation%20mutagenesis%20approach%20to%20understanding%20PTEN%20lipid%20phosphatase%20activity%20and%20genotype-phenotype%20relationships.%20Am%20J%20Hum%20Genet.%202018;102(5):943%E2%80%9355." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Mulligan MR, Bicknell LS. The molecular genetics of nELAVL in brain development and disease. Eur J Hum Genet. 2023;31(11):1209–17.
</cite> [<a href="https://doi.org/10.1038/s41431-023-01456-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10620143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37697079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mulligan%20MR,%20Bicknell%20LS.%20The%20molecular%20genetics%20of%20nELAVL%20in%20brain%20development%20and%20disease.%20Eur%20J%20Hum%20Genet.%202023;31(11):1209%E2%80%9317." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Cogné B, Ehresmann S, Beauregard-Lacroix E, Rousseau J, Besnard T, Garcia T, et al. Missense variants in the histone acetyltransferase complex component gene TRRAP cause autism and syndromic intellectual disability. Am J Hum Genet. 2019;104(3):530–41.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2019.01.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6407527/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30827496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cogn%C3%A9%20B,%20Ehresmann%20S,%20Beauregard-Lacroix%20E,%20Rousseau%20J,%20Besnard%20T,%20Garcia%20T,%20et%20al.%20Missense%20variants%20in%20the%20histone%20acetyltransferase%20complex%20component%20gene%20TRRAP%20cause%20autism%20and%20syndromic%20intellectual%20disability.%20Am%20J%20Hum%20Genet.%202019;104(3):530%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Guo H, Zhang Q, Dai R, Yu B, Hoekzema K, Tan J, et al. NCKAP1 Disruptive variants lead to a neurodevelopmental disorder with core features of autism. Am J Hum Genet. 2020;107(5):963–76.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2020.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7674997/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33157009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo%20H,%20Zhang%20Q,%20Dai%20R,%20Yu%20B,%20Hoekzema%20K,%20Tan%20J,%20et%20al.%20NCKAP1%20Disruptive%20variants%20lead%20to%20a%20neurodevelopmental%20disorder%20with%20core%20features%20of%20autism.%20Am%20J%20Hum%20Genet.%202020;107(5):963%E2%80%9376." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Galizia EC, Myers CT, Leu C, de Kovel CG, Afrikanova T, Cordero-Maldonado ML, et al. CHD2 variants are a risk factor for photosensitivity in epilepsy. Brain. 2015;138(Pt 5):1198–207.
</cite> [<a href="https://doi.org/10.1093/brain/awv052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4407192/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25783594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Galizia%20EC,%20Myers%20CT,%20Leu%20C,%20de%20Kovel%20CG,%20Afrikanova%20T,%20Cordero-Maldonado%20ML,%20et%20al.%20CHD2%20variants%20are%20a%20risk%20factor%20for%20photosensitivity%20in%20epilepsy.%20Brain.%202015;138(Pt%205):1198%E2%80%93207." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Liu JC, Ferreira CG, Yusufzai T. Human CHD2 is a chromatin assembly ATPase regulated by its chromo- and DNA-binding domains. J Biol Chem. 2015;290(1):25–34.
</cite> [<a href="https://doi.org/10.1074/jbc.M114.609156" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4281729/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25384982/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20JC,%20Ferreira%20CG,%20Yusufzai%20T.%20Human%20CHD2%20is%20a%20chromatin%20assembly%20ATPase%20regulated%20by%20its%20chromo-%20and%20DNA-binding%20domains.%20J%20Biol%20Chem.%202015;290(1):25%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Cummings BB, Karczewski KJ, Kosmicki JA, Seaby EG, Watts NA, Singer-Berk M, et al. Transcript expression-aware annotation improves rare variant interpretation. Nature. 2020;581(7809):452–8.
</cite> [<a href="https://doi.org/10.1038/s41586-020-2329-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7334198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32461655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cummings%20BB,%20Karczewski%20KJ,%20Kosmicki%20JA,%20Seaby%20EG,%20Watts%20NA,%20Singer-Berk%20M,%20et%20al.%20Transcript%20expression-aware%20annotation%20improves%20rare%20variant%20interpretation.%20Nature.%202020;581(7809):452%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Cafiero C, Marangi G, Orteschi D, Ali M, Asaro A, Ponzi E, et al. Novel de novo heterozygous loss-of-function variants in MED13L and further delineation of the MED13L haploinsufficiency syndrome. Eur J Hum Genet. 2015;23(11):1499–504.
</cite> [<a href="https://doi.org/10.1038/ejhg.2015.19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4613466/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25712080/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cafiero%20C,%20Marangi%20G,%20Orteschi%20D,%20Ali%20M,%20Asaro%20A,%20Ponzi%20E,%20et%20al.%20Novel%20de%20novo%20heterozygous%20loss-of-function%20variants%20in%20MED13L%20and%20further%20delineation%20of%20the%20MED13L%20haploinsufficiency%20syndrome.%20Eur%20J%20Hum%20Genet.%202015;23(11):1499%E2%80%93504." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Lugtenberg D, Reijnders MR, Fenckova M, Bijlsma EK, Bernier R, van Bon BW, et al. De novo loss-of-function mutations in WAC cause a recognizable intellectual disability syndrome and learning deficits in Drosophila. Eur J Hum Genet. 2016;24(8):1145–53.
</cite> [<a href="https://doi.org/10.1038/ejhg.2015.282" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4970694/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26757981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lugtenberg%20D,%20Reijnders%20MR,%20Fenckova%20M,%20Bijlsma%20EK,%20Bernier%20R,%20van%20Bon%20BW,%20et%20al.%20De%20novo%20loss-of-function%20mutations%20in%20WAC%20cause%20a%20recognizable%20intellectual%20disability%20syndrome%20and%20learning%20deficits%20in%20Drosophila.%20Eur%20J%20Hum%20Genet.%202016;24(8):1145%E2%80%9353." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Le Duc D, Giulivi C, Hiatt SM, Napoli E, Panoutsopoulos A, Harlan De Crescenzo A, et al. Pathogenic WDFY3 variants cause neurodevelopmental disorders and opposing effects on brain size. Brain. 2019;142(9):2617–30.</cite> [<a href="https://doi.org/10.1093/brain/awz198" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6736092/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31327001/" class="usa-link">PubMed</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Bellido-Cuéllar S, Pérez de la Fuente R, Lezana-Rosales JM, Sánchez-Calvín MT, Saiz-Díaz RA, González de la Aleja J. Epilepsy and autism spectrum disorder caused by a pathogenic variant in TNRC6B. Seizure. 2023;110:117–8.</cite> [<a href="https://doi.org/10.1016/j.seizure.2023.06.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37348364/" class="usa-link">PubMed</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Rolland T, Cliquet F, Anney RJL, Moreau C, Traut N, Mathieu A, et al. Phenotypic effects of genetic variants associated with autism. Nat Med. 2023;29(7):1671–80.
</cite> [<a href="https://doi.org/10.1038/s41591-023-02408-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10353945/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37365347/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rolland%20T,%20Cliquet%20F,%20Anney%20RJL,%20Moreau%20C,%20Traut%20N,%20Mathieu%20A,%20et%20al.%20Phenotypic%20effects%20of%20genetic%20variants%20associated%20with%20autism.%20Nat%20Med.%202023;29(7):1671%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Zhang Y, Li N, Li C, Zhang Z, Teng H, Wang Y, et al. Genetic evidence of gender difference in autism spectrum disorder supports the female-protective effect. Transl Psychiatry. 2020;10(1):4.
</cite> [<a href="https://doi.org/10.1038/s41398-020-0699-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7026157/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32066658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Y,%20Li%20N,%20Li%20C,%20Zhang%20Z,%20Teng%20H,%20Wang%20Y,%20et%20al.%20Genetic%20evidence%20of%20gender%20difference%20in%20autism%20spectrum%20disorder%20supports%20the%20female-protective%20effect.%20Transl%20Psychiatry.%202020;10(1):4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Nishimura Y, Martin CL, Vazquez-Lopez A, Spence SJ, Alvarez-Retuerto AI, Sigman M, et al. Genome-wide expression profiling of lymphoblastoid cell lines distinguishes different forms of autism and reveals shared pathways. Hum Mol Genet. 2007;16(14):1682–98.
</cite> [<a href="https://doi.org/10.1093/hmg/ddm116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17519220/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nishimura%20Y,%20Martin%20CL,%20Vazquez-Lopez%20A,%20Spence%20SJ,%20Alvarez-Retuerto%20AI,%20Sigman%20M,%20et%20al.%20Genome-wide%20expression%20profiling%20of%20lymphoblastoid%20cell%20lines%20distinguishes%20different%20forms%20of%20autism%20and%20reveals%20shared%20pathways.%20Hum%20Mol%20Genet.%202007;16(14):1682%E2%80%9398." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Avet-Rochex A, Carvajal N, Christoforou CP, Yeung K, Maierbrugger KT, Hobbs C, et al. Unkempt is negatively regulated by mTOR and uncouples neuronal differentiation from growth control. PLoS Genet. 2014;10(9):e1004624.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1004624" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4161320/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25210733/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Avet-Rochex%20A,%20Carvajal%20N,%20Christoforou%20CP,%20Yeung%20K,%20Maierbrugger%20KT,%20Hobbs%20C,%20et%20al.%20Unkempt%20is%20negatively%20regulated%20by%20mTOR%20and%20uncouples%20neuronal%20differentiation%20from%20growth%20control.%20PLoS%20Genet.%202014;10(9):e1004624." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Chiang AH, Chang J, Wang J, Vitkup D. Exons as units of phenotypic impact for truncating mutations in autism. Mol Psychiatry. 2021;26(5):1685–95.
</cite> [<a href="https://doi.org/10.1038/s41380-020-00876-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33110259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chiang%20AH,%20Chang%20J,%20Wang%20J,%20Vitkup%20D.%20Exons%20as%20units%20of%20phenotypic%20impact%20for%20truncating%20mutations%20in%20autism.%20Mol%20Psychiatry.%202021;26(5):1685%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Chowdhury TA, Luy DA, Scapellato G, Farache D, Lee ASY, Quinn CC. Ortholog of autism candidate gene RBM27 regulates mitoribosomal assembly factor MALS-1 to protect against mitochondrial dysfunction and axon degeneration during neurodevelopment. PLoS Biol. 2024;22(10):e3002876.</cite> [<a href="https://doi.org/10.1371/journal.pbio.3002876" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11556708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39480871/" class="usa-link">PubMed</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Peng S, Cai X, Chen J, Sun J, Lai B, Chang M, et al. The role of CELF family in neurodevelopment and neurodevelopmental disorders. Neurobiol Dis. 2024;197:106525.
</cite> [<a href="https://doi.org/10.1016/j.nbd.2024.106525" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38729272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peng%20S,%20Cai%20X,%20Chen%20J,%20Sun%20J,%20Lai%20B,%20Chang%20M,%20et%20al.%20The%20role%20of%20CELF%20family%20in%20neurodevelopment%20and%20neurodevelopmental%20disorders.%20Neurobiol%20Dis.%202024;197:106525." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Gazestani VH, Pramparo T, Nalabolu S, Kellman BP, Murray S, Lopez L, et al. A perturbed gene network containing PI3K-AKT, RAS-ERK and WNT-β-catenin pathways in leukocytes is linked to ASD genetics and symptom severity. Nat Neurosci. 2019;22(10):1624–34.
</cite> [<a href="https://doi.org/10.1038/s41593-019-0489-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6764590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31551593/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gazestani%20VH,%20Pramparo%20T,%20Nalabolu%20S,%20Kellman%20BP,%20Murray%20S,%20Lopez%20L,%20et%20al.%20A%20perturbed%20gene%20network%20containing%20PI3K-AKT,%20RAS-ERK%20and%20WNT-%CE%B2-catenin%20pathways%20in%20leukocytes%20is%20linked%20to%20ASD%20genetics%20and%20symptom%20severity.%20Nat%20Neurosci.%202019;22(10):1624%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Wang J, Zhu QW, Mai JH, Zhang S, Wang Y, Liang J, Zhou JY. A multi-omics study of brain tissue transcription and DNA methylation revealing the genetic pathogenesis of ADHD. Brief Bioinform. 2024;25(6):bbae502.</cite> [<a href="https://doi.org/10.1093/bib/bbae502" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11479714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39406522/" class="usa-link">PubMed</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Gillentine MA, Wang T, Hoekzema K, Rosenfeld J, Liu P, Guo H, et al. Rare deleterious mutations of HNRNP genes result in shared neurodevelopmental disorders. Genome Med. 2021;13(1):63.
</cite> [<a href="https://doi.org/10.1186/s13073-021-00870-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8056596/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33874999/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gillentine%20MA,%20Wang%20T,%20Hoekzema%20K,%20Rosenfeld%20J,%20Liu%20P,%20Guo%20H,%20et%20al.%20Rare%20deleterious%20mutations%20of%20HNRNP%20genes%20result%20in%20shared%20neurodevelopmental%20disorders.%20Genome%20Med.%202021;13(1):63." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Wang T, Liu L, Fan T, Xia K, Sun Z. Shared and divergent contribution of vitamin A and oxytocin to the aetiology of autism spectrum disorder. Comput Struct Biotechnol J. 2023;21:3109–23.
</cite> [<a href="https://doi.org/10.1016/j.csbj.2023.05.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10782014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38213898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20T,%20Liu%20L,%20Fan%20T,%20Xia%20K,%20Sun%20Z.%20Shared%20and%20divergent%20contribution%20of%20vitamin%20A%20and%20oxytocin%20to%20the%20aetiology%20of%20autism%20spectrum%20disorder.%20Comput%20Struct%20Biotechnol%20J.%202023;21:3109%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Forés-Martos J, Catalá-López F, Sánchez-Valle J, Ibáñez K, Tejero H, Palma-Gudiel H, et al. Transcriptomic metaanalyses of autistic brains reveals shared gene expression and biological pathway abnormalities with cancer. Mol Autism. 2019;10:17.
</cite> [<a href="https://doi.org/10.1186/s13229-019-0262-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6454734/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31007884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?For%C3%A9s-Martos%20J,%20Catal%C3%A1-L%C3%B3pez%20F,%20S%C3%A1nchez-Valle%20J,%20Ib%C3%A1%C3%B1ez%20K,%20Tejero%20H,%20Palma-Gudiel%20H,%20et%20al.%20Transcriptomic%20metaanalyses%20of%20autistic%20brains%20reveals%20shared%20gene%20expression%20and%20biological%20pathway%20abnormalities%20with%20cancer.%20Mol%20Autism.%202019;10:17." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Simons Foundation Autism Research Initiative (SFARI). <a href="https://www.sfari.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.sfari.org/</a>. Accessed 18 Aug 2025.</cite>
</li>
<li id="CR93">
<span class="label">93.</span><cite>Soo-Whee K. Evaluation of familial phenotype deviation: main analysis and figure code, v1 ed. Zenodo. 2025. Available from: 10.5281/zenodo.15838146.</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366145/bin/13073_2025_1532_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">13073_2025_1532_MOESM1_ESM.docx</a><sup> (4.7MB, docx) </sup><p>Additional file 1: 17 supporting Figures S1-S17. Captions for each Fig. S1 to S17 are given within the file.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366145/bin/13073_2025_1532_MOESM2_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13073_2025_1532_MOESM2_ESM.xlsx</a><sup> (21.6KB, xlsx) </sup><p>Additional file 2: Table S1. This table contains WFSD of each genetic subgroup: SRS deviations from parents (Table S1A), SRS deviations from siblings (Table S1B), VABS deviations from siblings (Table S1C), SCQ lifetime deviations from siblings (Table S1D), ADOS CSS total deviations from siblings (Table S1E), ADOS CSS SA deviations from siblings (Table S1F), ADOS CSS RRB deviations from siblings (Table S1G), and FSIQ deviations from siblings (Table S1H).</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366145/bin/13073_2025_1532_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13073_2025_1532_MOESM3_ESM.xlsx</a><sup> (48.2KB, xlsx) </sup><p>Additional file 3: Table S2. This table contains enrichment results of dnDIS in ASD probands with severe SRS profiles (Table S2A), lists genes enriched in these probands (Table S2B), and shows enrichment results of each SRS gene set with GO pathways (Table S2C) and with cell-type specific DEGs in the developing human brain (Table S2D).</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366145/bin/13073_2025_1532_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">13073_2025_1532_MOESM4_ESM.xlsx</a><sup> (1.5MB, xlsx) </sup><p>Additional file 4: Table S3. This table contains MADs of phenotype outcomes of DNVs per gene: SRS deviations (Table S3A), and raw SRS scores (Table S3B).</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Due to local privacy laws and privileged human information, all requests for the raw genomic data from the Korean Autism cohort are subject to prior approval from the Institutional Review Board at Seoul National University Bundang Hospital. Interested researchers should submit a request to Dr. Hee Jeong Yoo, along with a detailed research plan outlining proposed analyses and data anonymization procedures. This plan will be reviewed by the IRB and data sharing committee at Seoul National University Bundang Hospital, with approval typically granted within two months. Upon approval, the requester will be added to the IRB as a collaborator for secure data sharing. Genetic and phenotypic data for the SSC and SPARK cohorts can be accessed by applying at <a href="https://base.sfari.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://base.sfari.org</a> [<a href="#CR92" class="usa-link" aria-describedby="CR92">92</a>]. All major analysis scripts and code used to generate key figures will be made publicly available via Zenodo (10.5281/zenodo.15838146) [<a href="#CR93" class="usa-link" aria-describedby="CR93">93</a>]. Additionally, summary-level statistics from the WFSD analyses and curated gene lists used in this study will be provided as Supplementary Tables to ensure reproducibility and transparency.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Genome Medicine are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s13073-025-01532-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/13073_2025_Article_1532.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366145/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366145/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366145%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366145/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366145/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366145/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40836247/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366145/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40836247/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366145/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366145/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="hDvKVrRUJirgGIXvUsCGDpB1Fe3G72Evb4NX83i8uetZ1KnGkXrtCnx2Oad9sEUp">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
