
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Tandem repeat–induced sexual silencing: A Rid-dependent RNAi mechanism for fungal genome defense via translational repression - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AF3E8AF3180305AF3E001B6E086C.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="sciadv">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366691/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Science Advances">
<meta name="citation_title" content="Tandem repeat–induced sexual silencing: A Rid-dependent RNAi mechanism for fungal genome defense via translational repression">
<meta name="citation_author" content="Mengde Hou">
<meta name="citation_author_institution" content="State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.">
<meta name="citation_author" content="Yajia Ni">
<meta name="citation_author_institution" content="State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.">
<meta name="citation_author" content="Jinrong Yin">
<meta name="citation_author_institution" content="State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.">
<meta name="citation_author" content="Mengchun Wu">
<meta name="citation_author_institution" content="State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.">
<meta name="citation_author" content="Helan Zhang">
<meta name="citation_author_institution" content="State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.">
<meta name="citation_author" content="Yanfei Du">
<meta name="citation_author_institution" content="State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.">
<meta name="citation_author" content="Shasha Chen">
<meta name="citation_author_institution" content="National Key Laboratory of Agricultural Microbiology, College of Life Science and Technology, Huazhong Agricultural University, Wuhan 430070, China.">
<meta name="citation_author" content="Zhipeng Zhou">
<meta name="citation_author_institution" content="National Key Laboratory of Agricultural Microbiology, College of Life Science and Technology, Huazhong Agricultural University, Wuhan 430070, China.">
<meta name="citation_author" content="Cong Jiang">
<meta name="citation_author_institution" content="State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.">
<meta name="citation_author" content="Qinhu Wang">
<meta name="citation_author_institution" content="State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.">
<meta name="citation_author" content="Huiquan Liu">
<meta name="citation_author_institution" content="State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="11">
<meta name="citation_issue" content="34">
<meta name="citation_firstpage" content="eadu7606">
<meta name="citation_doi" content="10.1126/sciadv.adu7606">
<meta name="citation_pmid" content="40834073">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366691/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366691/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366691/pdf/sciadv.adu7606.pdf">
<meta name="description" content="Eukaryotes have evolved defense systems to protect their genomes from invasive genetic elements. We report a fungal defense mechanism, tandem repeat–induced sexual silencing (TRISS), active during sexual stages. TRISS is a unique RNA interference ...">
<meta name="og:title" content="Tandem repeat–induced sexual silencing: A Rid-dependent RNAi mechanism for fungal genome defense via translational repression">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Eukaryotes have evolved defense systems to protect their genomes from invasive genetic elements. We report a fungal defense mechanism, tandem repeat–induced sexual silencing (TRISS), active during sexual stages. TRISS is a unique RNA interference ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366691/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366691">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1126/sciadv.adu7606"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/sciadv.adu7606.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366691%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366691/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366691/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366691/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-sciadv.gif" alt="Science Advances logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Science Advances" title="Link to Science Advances" shape="default" href="http://advances.sciencemag.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Adv</button></div>. 2025 Aug 20;11(34):eadu7606. doi: <a href="https://doi.org/10.1126/sciadv.adu7606" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1126/sciadv.adu7606</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Adv%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Adv%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Tandem repeat–induced sexual silencing: A Rid-dependent RNAi mechanism for fungal genome defense via translational repression</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hou%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Mengde Hou</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Mengde Hou</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.</div>
<div>Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Software, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hou%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mengde Hou</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ni%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Yajia Ni</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Yajia Ni</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.</div>
<div>Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Resources, Software, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ni%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yajia Ni</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yin%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jinrong Yin</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jinrong Yin</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.</div>
<div>Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Resources, Software, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yin%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jinrong Yin</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Mengchun Wu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Mengchun Wu</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.</div>
<div>Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Resources, Software, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mengchun Wu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Helan Zhang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Helan Zhang</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.</div>
<div>Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Resources, Software, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Helan Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Du%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Yanfei Du</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Yanfei Du</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.</div>
<div>Investigation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Du%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yanfei Du</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Shasha Chen</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Shasha Chen</span></h3>
<div class="p">
<sup><sup>2</sup></sup>National Key Laboratory of Agricultural Microbiology, College of Life Science and Technology, Huazhong Agricultural University, Wuhan 430070, China.</div>
<div>Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Resources, Software, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shasha Chen</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Zhipeng Zhou</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Zhipeng Zhou</span></h3>
<div class="p">
<sup><sup>2</sup></sup>National Key Laboratory of Agricultural Microbiology, College of Life Science and Technology, Huazhong Agricultural University, Wuhan 430070, China.</div>
<div>Methodology, Resources, Supervision, Validation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhou%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhipeng Zhou</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Cong Jiang</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Cong Jiang</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.</div>
<div>Resources, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Cong Jiang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Qinhu Wang</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Qinhu Wang</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.</div>
<div>Methodology, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qinhu Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Huiquan Liu</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Huiquan Liu</span></h3>
<div class="p">
<sup><sup>1</sup></sup>State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.</div>
<div>Conceptualization, Data curation, Funding acquisition, Methodology, Project administration, Resources, Supervision, Validation, Writing - original draft, Writing - review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Huiquan Liu</span></a>
</div>
</div>
<sup>1,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup><sup>1</sup></sup>State Key Laboratory for Crop Stress Resistance and High-Efficiency Production, College of Plant Protection, Northwest A&amp;F University, Yangling, Shaanxi 712100, China.</div>
<div id="aff2">
<sup><sup>2</sup></sup>National Key Laboratory of Agricultural Microbiology, College of Life Science and Technology, Huazhong Agricultural University, Wuhan 430070, China.</div>
<div class="author-notes p"><div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">Corresponding author. Email: <span>liuhuiquan@nwsuaf.edu.cn</span> (H.L.); <span>wangqinhu@nwafu.edu.cn</span> (Q.W.)</p>
</div></div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Mengde Hou</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Software, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Yajia Ni</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Resources, Software, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Jinrong Yin</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Resources, Software, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Mengchun Wu</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Resources, Software, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Helan Zhang</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Resources, Software, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Yanfei Du</span></strong>: <span class="role">Investigation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Shasha Chen</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Funding acquisition, Investigation, Methodology, Project administration, Resources, Software, Supervision, Validation, Visualization, Writing - original draft, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Zhipeng Zhou</span></strong>: <span class="role">Methodology, Resources, Supervision, Validation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Cong Jiang</span></strong>: <span class="role">Resources, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Qinhu Wang</span></strong>: <span class="role">Methodology, Writing - review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Huiquan Liu</span></strong>: <span class="role">Conceptualization, Data curation, Funding acquisition, Methodology, Project administration, Resources, Supervision, Validation, Writing - original draft, Writing - review &amp; editing</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Nov 21; Accepted 2025 Jul 21; Collection date 2025 Aug 22.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright © 2025 The Authors, some rights reserved; exclusive licensee American Association for the Advancement of Science. No claim to original U.S. Government Works. Distributed under a Creative Commons Attribution NonCommercial License 4.0 (CC BY-NC).</div>
<p>This is an open-access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by-nc/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution-NonCommercial license</a>, which permits use, distribution, and reproduction in any medium, so long as the resultant use is <strong>not</strong> for commercial advantage and provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366691  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40834073/" class="usa-link">40834073</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Eukaryotes have evolved defense systems to protect their genomes from invasive genetic elements. We report a fungal defense mechanism, tandem repeat–induced sexual silencing (TRISS), active during sexual stages. TRISS is a unique RNA interference (RNAi) pathway distinct from known mechanisms like quelling and meiotic silencing by unpaired DNA (MSUD). Triggered by tandem repeats, it involves RNAi components similar to quelling but uniquely includes the MSUD-specific Argonaute Sms2. TRISS operates independently of recombination. We identified small interfering RNA sassociated with TRISS (trasiRNAs) in tandem repeats. Sms2, guided by trasiRNAs, mediates translational repression of target mRNAs and is crucial for trasiRNA biogenesis. TRISS requires the DNA methyltransferase Rid but not Rid-mediated repeat-induced point mutation (RIP). Both Sms2 and Rid interact with replication protein A (RPA) to recruit the RPA-Qde1/RdRP complex, dependent on the helicase Qde3. This study reveals a potentially conserved strategy linking RIP and RNAi to silence tandem repeats during fungal sexual stages, offering insights into fungal genome defense.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p>TRISS, an RNAi-mediated defense mechanism, silences tandem repeats in conjunction with RIP during the sexual stages in fungi.</p></section><section id="sec1"><h2 class="pmc_sec_title">INTRODUCTION</h2>
<p>All living organisms face threats from invading genetic elements, such as transgenes, transposons, and viruses. These elements can repeatedly insert themselves into the genome, disrupting normal genes, causing harmful recombination, and leading to uncontrolled gene activity, which ultimately threatens the stability of the genome (<a href="#R1" class="usa-link" aria-describedby="R1"><em>1</em></a>, <a href="#R2" class="usa-link" aria-describedby="R2"><em>2</em></a>). To defend against these threats, most organisms have evolved advanced genome defense systems called homology-dependent gene silencing (HDGS). These systems detect and deactivate repeated sequences introduced by these parasitic genetic elements (<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>, <a href="#R4" class="usa-link" aria-describedby="R4"><em>4</em></a>).</p>
<p>One of the best-studied HDGS mechanisms is RNA interference (RNAi), which is conserved across eukaryotes and protects the genome from both internal elements like transposons and external invaders like viruses (<a href="#R5" class="usa-link" aria-describedby="R5"><em>5</em></a>–<a href="#R8" class="usa-link" aria-describedby="R8"><em>8</em></a>). The RNAi process begins with an enzyme called Dicer, which cuts long double-stranded (ds) RNA molecules into short fragments known as small interfering RNAs (siRNAs). These siRNAs are then loaded into Argonaute (Ago) proteins. One strand of the siRNA is removed, while the other strand guides Ago to identify and destroy matching RNA molecules in the cytoplasm. In addition to silencing RNA posttranscriptionally, RNAi can also act at the transcriptional level by modifying DNA in the nucleus (<a href="#R9" class="usa-link" aria-describedby="R9"><em>9</em></a>–<a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>). Some organisms also use RNA-dependent RNA polymerase (RdRP) to create dsRNA from single-stranded (ss) RNA or to amplify siRNA signals, further strengthening the RNAi response (<a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>–<a href="#R14" class="usa-link" aria-describedby="R14"><em>14</em></a>).</p>
<p>During sexual reproduction, invading genetic elements can spread between the two genomes contributed by the parents. To prevent this, many organisms have evolved specific defense mechanisms that target and destroy these elements before they can be passed on to the next generation (<a href="#R15" class="usa-link" aria-describedby="R15"><em>15</em></a>). For example, in plants, reproductive tissues produce specific siRNAs that guide DNA modifications, silencing transposons at the transcriptional level (<a href="#R16" class="usa-link" aria-describedby="R16"><em>16</em></a>). In animals, a specialized RNAi pathway involving PIWI-interacting RNAs protects reproductive cells by silencing transposons, ensuring the integrity of the germline genome (<a href="#R17" class="usa-link" aria-describedby="R17"><em>17</em></a>). In the yeast <em>Cryptococcus neoformans</em>, a defense mechanism called sex-induced silencing (SIS) uses RNAi to detect and silence repeated genetic sequences during the sexual cycle (<a href="#R18" class="usa-link" aria-describedby="R18"><em>18</em></a>). In the filamentous fungus <em>Neurospora crassa</em>, three distinct genome defense mechanisms have been found: quelling, repeat-induced point mutation (RIP), and meiotic silencing by unpaired DNA (MSUD) (<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>, <a href="#R4" class="usa-link" aria-describedby="R4"><em>4</em></a>, <a href="#R19" class="usa-link" aria-describedby="R19"><em>19</em></a>, <a href="#R20" class="usa-link" aria-describedby="R20"><em>20</em></a>).</p>
<p>Both quelling and MSUD use RNAi to silence harmful genetic elements. Quelling occurs during normal growth, while MSUD is active during meiosis (<a href="#R21" class="usa-link" aria-describedby="R21"><em>21</em></a>). The quelling process involves proteins such as RdRP (Qde-1), Dicer enzymes (Dcl-1 and Dcl-2), Ago (Qde-2), RecQ helicase (Qde-3), and a Qde-2–interacting protein (Qip). The MSUD process consists of overlapping components from quelling (Dcl-1 and Qip), as well as MSUD-specific components such as RdRP (Sad-1), Ago (Sms-2), and helicase (Sad-3). In quelling, siRNA production requires both DNA tandem repeats and a double-strand break (DSB) (<a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>). The putative homologous recombination (HR) intermediates of repetitive sequences are recognized by Qde-3, leading to the recruitment of Qde-1 and ssDNA-binding protein replication protein A (RPA) (<a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>, <a href="#R23" class="usa-link" aria-describedby="R23"><em>23</em></a>). Qde-1 exhibits RdRP and DNA-dependent RNA polymerase (DdRP) activities (<a href="#R24" class="usa-link" aria-describedby="R24"><em>24</em></a>), producing aberrant ssRNAs (aRNAs) and then dsRNAs to activate the downstream RNAi pathway. MSUD, on the other hand, is triggered by unpaired DNA regions during meiosis. These unpaired regions produce siRNAs, which silence both matching and nonmatching copies of genes in the genome (<a href="#R25" class="usa-link" aria-describedby="R25"><em>25</em></a>–<a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>).</p>
<p>RIP, the first genome defense system found in eukaryotes, operates before meiosis during the sexual reproduction of filamentous ascomycetes (<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>, <a href="#R28" class="usa-link" aria-describedby="R28"><em>28</em></a>–<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>). RIP introduces C-to-T mutations into repeated DNA regions, effectively disabling them (<a href="#R31" class="usa-link" aria-describedby="R31"><em>31</em></a>). RIP relies on the putative DNA methyltransferase Rid (<a href="#R32" class="usa-link" aria-describedby="R32"><em>32</em></a>). In <em>N. crassa</em>, RIP-mutated sequences can also direct trimethylation of histone H3 lysine 9 (H3K9me3) by the histone methyltransferase Dim-5, which then leads to methylation of associated cytosines by the DNA methyltransferase Dim-2 (<a href="#R33" class="usa-link" aria-describedby="R33"><em>33</em></a>). Recently, a Rid-independent pathway of RIP was found in <em>N. crassa</em> (<a href="#R34" class="usa-link" aria-describedby="R34"><em>34</em></a>). This pathway tends to mutate the linker region between tandem repeats and requires Dim-2, Dim-5, and heterochromatin protein 1 (HP1). However, there is currently no known cross-talk between the RIP and RNAi mechanisms.</p>
<p>The filamentous fungus <em>Fusarium graminearum</em> is a major pathogen causing Fusarium head blight, a devastating disease on cereal crops worldwide. This fungus is self-fertile and produces abundant sexual fruiting bodies (perithecia) in laboratory conditions, making it an ideal model for studying genome defense mechanisms during sexual reproduction. Both RIP and MSUD have been observed in <em>F. graminearum</em> (<a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>, <a href="#R36" class="usa-link" aria-describedby="R36"><em>36</em></a>). In addition, a regulatory mechanism called A-to-I mRNA editing, which occurs specifically during sexual stages, has been shown to play a critical role in the sexual development of this fungus (<a href="#R37" class="usa-link" aria-describedby="R37"><em>37</em></a>–<a href="#R43" class="usa-link" aria-describedby="R43"><em>43</em></a>).</p>
<p>In this study, we describe a genome defense mechanism in <em>F. graminearum</em> called tandem repeat–induced sexual silencing (TRISS). TRISS is a distinct RNAi pathway specifically activated by tandem repeat sequences. While it shares some components with the quelling pathway, TRISS uniquely involves the MSUD-specific Ago protein FgSms2. We identified TRISS-associated siRNAs (trasiRNAs) localized within tandem repeat regions. Our research demonstrates that FgSms2, guided by trasiRNAs, is essential for both suppressing the translation of target mRNAs and facilitating the production of trasiRNAs. Furthermore, we found that FgRid is indispensable for TRISS, functioning independently of the RIP pathway. We uncovered two key mechanisms that enhance the TRISS response, both of which operate independently of traditional transitive siRNA amplification: (i) FgSms2 interacts with FgRpa1 to recruit the FgRpa1-FgQde1 complex to ssDNA in a trasiRNA-dependent manner, and (ii) FgRid interacts with FgRpa1 to similarly recruit the FgRpa1-FgQde1 complex. These interactions rely on the helicase protein FgQde3, whose recruitment to tandem repeat regions is dependent on FgRid activity. This suggests that FgQde3 may recognize specific intermediates introduced by FgRid, triggering DNA unwinding and activating the TRISS pathway. Our findings provide evidence that TRISS may represent a conserved mechanism linking the RIP and RNAi pathways to silence harmful genetic elements during sexual reproduction. This study not only advances our understanding of fungal genome defense systems but also broadens our knowledge of RNAi mechanisms in eukaryotes.</p></section><section id="sec2"><h2 class="pmc_sec_title">RESULTS</h2>
<section id="sec3"><h3 class="pmc_sec_title">Tandem repeats trigger sexual stage–specific gene silencing in <em>F. graminearum</em>
</h3>
<p>In a previous study (<a href="#R44" class="usa-link" aria-describedby="R44"><em>44</em></a>), we performed complementation assays on the <em>Fgama1</em> deletion mutant, which exhibits defects in ascospore morphology, by introducing the pFL2 plasmid containing the full-length <em>FgAMA1</em> gene. Two distinct types of transformants were observed (<a href="#F1" class="usa-link">Fig. 1A</a>). One group produced asci with eight spindle-shaped ascospores, resembling the wild type and indicating successful complementation. The other group produced asci with eight oval ascospores, similar to the deletion mutant, suggesting unsuccessful complementation. Notably, all transformants contained the <em>FgAMA1</em> gene (<a href="#F1" class="usa-link">Fig. 1B</a>). Subsequent quantitative polymerase chain reaction (qPCR) assays revealed that the complemented transformants had only one copy of <em>FgAMA1</em>, while the uncomplemented transformants had multiple copies (<a href="#F1" class="usa-link">Fig. 1C</a>).</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Fig. 1. Discovery of the TRISS phenomenon.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366691_sciadv.adu7606-f1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a586/12366691/34db033d51fb/sciadv.adu7606-f1.jpg" loading="lazy" height="740" width="787" alt="Fig. 1."></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Spindle-shaped (blue arrowhead) or oval (red arrowhead) ascospores produced by PH-1 (WT), the <em>Fgama1</em> deletion mutant, and complementary strains at 7-dpf on carrot-agar plates. Numbers represent strain identifiers. Scale bar, 20 μm. (<strong>B</strong>) The presence of <em>FgAMA1</em> sequences in specific strains detected by PCR. H<sub>2</sub>O, control; M, marker. (<strong>C</strong>) Copy number of <em>FgAMA1</em> in specific strains assayed by qPCR. (<strong>D</strong>) Nonsynonymous mutations identified in one copy of <em>FgAMA1</em> repeats in 15 progeny from uncomplemented strains <em>Fgama1/FgAMA1</em> #2 and #11. Blue lines indicate mutation sites. (<strong>E</strong>) Diagram illustrating tandem and unlinked repeats generated by HR and NHEJ, respectively, after transformation with a linear segment containing the target gene and the hygromycin-resistance gene (<em>hyg</em>). (<strong>F</strong>) Copy number of <em>FgAMA1</em> and <em>CDC2A</em> in transformants containing repeated (R-) <em>FgAMA1</em> or <em>CDC2A</em> assayed by qPCR, respectively. (<strong>G</strong>) Morphology of asci and ascospores formed at 7-dpf by the <em>cdc2A</em> deletion mutant, R-<em>FgAMA1</em> and R-<em>CDC2A</em> transformants, and the transformant with two unlinked <em>CDC2A</em> repeats (UR-<em>CDC2A</em>). Numbers in parentheses indicate the phenotype frequency. Scale bar, 20 μm. (<strong>H</strong>) Tandem repeats in specific strains detected by PCR. The diagram shows tandem repeat arrays and primer pairs used for upstream (U) and downstream (D) repeats. Dotted red boxes indicate undetermined repeat numbers. (<strong>I</strong>) Nonsynonymous mutations identified in upstream (U) and downstream (D) repeats in eight progeny of TR-<em>FgAMA1</em> transformant #5 containing two tandemly repeated <em>FgAMA1</em> copies. (<strong>J</strong>) Colony morphology on PDA plates (3 days) and asci/ascospore morphology at 7-dpf on carrot-agar plates for <em>tub1</em> and <em>pks12</em> deletion mutants, and transformants with tandemly repeated (TR-) <em>TUB1</em>, <em>PKS12</em>, and <em>FgCLN2</em>. Colony color is shown from the reverse side for <em>pks12</em> and TR-<em>PKS12</em>. Scale bar, 20 μm. For (C) and (F), bar graphs show the mean of three technical replicates.</p></figcaption></figure><p>Although gene silencing can be caused by RIP associated with multicopy transgene repeats, the severity and consistency of the defective phenotype in each ascus suggested that RIP was not the primary cause in this case. To investigate further, we isolated 15 ascospore progeny from the uncomplemented transformants and performed Sanger sequencing to detect RIP mutations in <em>FgAMA1</em>. Six progeny showed no amino acid changes caused by RIP mutations in <em>FgAMA1</em> (<a href="#F1" class="usa-link">Fig. 1D</a>) despite the abnormal phenotypes observed in the isolated ascospores. These observations suggest that RIP is not the primary cause of gene silencing in the uncomplemented transformants. Similar phenomena were observed in complementation assays on the deletion mutants of <em>CDC2A</em> (<a href="#R45" class="usa-link" aria-describedby="R45"><em>45</em></a>) and <em>TUB1</em> (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>) (fig. S1), indicating that the gene silencing phenomenon is not specific to <em>FgAMA1</em> and may be a more general phenomenon.</p>
<p>To explore this silencing further, we transformed a linear fragment containing the full-length <em>FgAMA1</em> gene fused with a hygromycin-resistance gene (<em>hyg</em>) into the wild-type strain PH-1. We expected two types of transformants: one with unlinked repeats of <em>FgAMA1</em> due to random integration via nonhomologous end joining (NHEJ), and the other with tandem repeats of <em>FgAMA1</em> at its native locus via HR (<a href="#F1" class="usa-link">Fig. 1E</a>). We obtained five transformants (R-<em>FgAMA1</em>) containing at least one additional copy of <em>FgAMA1</em> (<a href="#F1" class="usa-link">Fig. 1F</a>), all of which displayed phenotypes similar to the deletion mutant (<a href="#F1" class="usa-link">Fig. 1G</a>). PCR assays confirmed that these transformants had tandemly repeated <em>FgAMA1</em> at the native locus (<a href="#F1" class="usa-link">Fig. 1H</a>). Further analysis of eight ascospore progeny from a transformant with two tandemly repeated <em>FgAMA1</em> copies revealed that five progeny showed no RIP mutations in at least one repeat (<a href="#F1" class="usa-link">Fig. 1I</a>), providing additional evidence that RIP was not the cause of silencing.</p>
<p>A similar experiment with <em>CDC2A</em> yielded four transformants (R-<em>CDC2A</em>) (<a href="#F1" class="usa-link">Fig. 1F</a>), three of which exhibited deletion mutant phenotypes and tandemly repeated <em>CDC2A</em>, while the fourth showed wild-type phenotypes without tandem repeats (<a href="#F1" class="usa-link">Fig. 1, G and H</a>). To confirm this observation, we introduced an ectopic copy of <em>CDC2A</em> at a specific locus (FG1G36140) in PH-1. The resulting transformants with unlinked repeats (UR-<em>CDC2A</em>) exhibited normal phenotypes (<a href="#F1" class="usa-link">Fig. 1G</a>). These results indicate that the silencing phenomenon is triggered by tandem repeats rather than unlinked repeats.</p>
<p>Since <em>CDC2A</em> and <em>FgAMA1</em> deletion mutants exhibited defects only during sexual stages, these genes could only be used to determine whether silencing occurs during sexual development. To investigate whether silencing also occurs during vegetative stages, we examined the <em>TUB1</em> gene, which affects both sexual reproduction and vegetative growth when deleted (<a href="#R46" class="usa-link" aria-describedby="R46"><em>46</em></a>). Transformants with tandemly repeated <em>TUB1</em> at its native locus (TR-<em>TUB1</em>) displayed silencing phenotypes during sexual reproduction but not during vegetative growth (<a href="#F1" class="usa-link">Fig. 1, H and J</a>). To confirm these findings, we conducted a similar experiment with <em>PKS12</em>, a gene involved in pigment biosynthesis (<a href="#R47" class="usa-link" aria-describedby="R47"><em>47</em></a>). While the <em>pks12</em> deletion mutant formed white colonies, the TR-<em>PKS12</em> transformants displayed the normal red-pink phenotype of the wild type (<a href="#F1" class="usa-link">Fig. 1, H and J</a>). These results suggest that tandem repeats trigger gene silencing specifically during the sexual stage.</p>
<p>Since the silencing phenotypes are observed during self-fertilization, we can rule out MSUD as the cause of the silencing. Furthermore, transformants with tandemly repeated <em>FgCLN2</em>, which encodes a B-type cyclin essential for viability, exhibited normal vegetative growth but defects in crozier formation (<a href="#F1" class="usa-link">Fig. 1, H and J</a>). This indicates that silencing occurs premeiotically and can take place throughout sexual development, from the dikaryotic stage to ascospore differentiation. On the basis of these findings, we named this silencing phenomenon TRISS.</p></section><section id="sec4"><h3 class="pmc_sec_title">TRISS is independent of tandem-repeat formation mode and repeat orientation</h3>
<p>To investigate whether the mode of tandem-repeat formation influences TRISS, we used two additional strategies to create tandem repeats of <em>CDC2A</em>. In the first strategy, HR occurred between the native gene and a linear homologous fragment, while in the second strategy, HR occurred in the linker region between the two repeats (<a href="#F2" class="usa-link">Fig. 2A</a>). In both cases, transformants with tandemly repeated <em>CDC2A</em> displayed deletion mutant phenotypes, indicating that TRISS is independent of the repeat formation mode.</p>
<figure class="fig xbox font-sm" id="F2"><h4 class="obj_head">Fig. 2. Characteristics of the TRISS phenomenon.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366691_sciadv.adu7606-f2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a586/12366691/749ce7826228/sciadv.adu7606-f2.jpg" loading="lazy" height="456" width="787" alt="Fig. 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Diagram illustrating two modes of tandem-repeat formation: recombination in the repeated region and in the hygromycin-resistance gene (<em>hyg</em>) region, along with photos of ascus and ascospore morphology of the corresponding transformants. (<strong>B</strong>) Ascus and ascospore morphology of the transformants containing a reverse tandem repeat of <em>CDC2A</em> (a), an extra copy of <em>CDC2A</em> inserted at the FG1G36140 locus in chromosome 1 ectopically expressing antisense transcripts by the RP27 promoter (b), and a tandem repeat of <em>CDC2A</em> inserted at the FG1G36140 locus (c). (<strong>C</strong>) Ascus and ascospore morphology of the transformants containing a tandem repeat of <em>FgAMA1</em> with a <em>hyg</em> (a), a G418-resistance gene (<em>neo</em>) (b), and a nongene sequence (c) as the linker, respectively. (<strong>D</strong>) Ascus and ascospore morphology of the transformants containing different lengths of <em>CDC2A</em> tandem repeats or a 7006-bp linker region. P+5′UTR, promoter and 5′ untranslated region; CDS, coding region; 3′UTR, 3′ untranslated region; R, repeat length; L, linker length. For (A) to (D), numbers in parentheses indicate the phenotype frequency. Scale bar, 20 μm. (<strong>E</strong>) Ascus and ascospore morphology of the outcrosses of WT, the <em>Fgama1</em> and <em>cdc2A</em> deletion mutants, or the transformants containing a tandem repeat of <em>FgAMA1</em> (TR-<em>FgAMA1</em>) or <em>CDC2A</em> (TR-<em>CDC2A</em>) as the male with the <em>mat1–1-1</em> deletion mutant labeled with H1-GFP as the female. Eight ascospores in each ascus showed 1:1 segregation for GFP signals. Diagram represents the allele locus in each outcross. DIC, differential interference contrast. Scale bar, 20 μm.</p></figcaption></figure><p>We also constructed transformants with reverse tandem repeats of <em>CDC2A</em>. These transformants exhibited silencing of <em>CDC2A</em> (<a href="#F2" class="usa-link">Fig. 2B</a>). However, transformants ectopically expressing antisense transcripts of <em>CDC2A</em> from an extra copy inserted at the FG1G36140 locus displayed normal phenotypes (<a href="#F2" class="usa-link">Fig. 2B</a>). This suggests that antisense transcription alone cannot silence <em>CDC2A</em>, and TRISS is independent of the orientation of tandem repeats.</p>
<p>To rule out the possibility that gene silencing is caused by the expression of resistance marker genes in the linker between repeats, we constructed tandemly repeated <em>FgAMA1</em> with a G418-resistance gene (<em>neo</em>) or a nongene sequence as the linker (<a href="#F2" class="usa-link">Fig. 2C</a>). In both cases, <em>FgAMA1</em> was completely silenced, as observed in transformants with the <em>hyg</em> linker. Therefore, TRISS is unrelated to the linker sequence or whether it contains expressed genes.</p></section><section id="sec5"><h3 class="pmc_sec_title">TRISS efficiency depends on repeat length and distance between repeats</h3>
<p>Our results suggest that two tandem repeats are sufficient to trigger TRISS during sexual reproduction. To determine whether silencing efficiency depends on repeat length, we created constructs with direct repeats of varying lengths [1080 to 3187 base pairs (bp)] of <em>CDC2A</em> separated by a <em>neo</em> linker (1433 bp) (<a href="#F2" class="usa-link">Fig. 2D</a>). Longer repeats (3187 and 2643 bp) induced complete silencing of <em>CDC2A</em>, resulting in very few ascospores. Shorter repeats (2117 and 1613 bp) exhibited reduced silencing efficiency, with deformed or normal ascospores observed. Repeats shorter than 1109 bp did not induce silencing. These results indicate that TRISS efficiency depends on repeat length, with longer repeats being more effective.</p>
<p>We also investigated the effect of linker length between tandem repeats. Increasing the linker length from 1433 to 7006 bp reduced silencing efficiency (<a href="#F2" class="usa-link">Fig. 2D</a>). This result aligns with the observation that unlinked repeats cannot induce TRISS. Thus, both repeat length and the distance between repeats are critical factors for TRISS efficiency, with longer repeats and shorter distances enhancing silencing.</p></section><section id="sec6"><h3 class="pmc_sec_title">TRISS can occur in outcrossing and functions in <em>trans</em>
</h3>
<p>Although <em>F. graminearum</em> is homothallic, it can undergo outcrossing. To test whether TRISS occurs during outcrossing, we crossed a TR-<em>FgAMA1</em> transformant (male) with a self-sterile but female-fertile <em>mat1–1-1</em> deletion mutant labeled with H1–green fluorescent protein (GFP) (<a href="#R44" class="usa-link" aria-describedby="R44"><em>44</em></a>). For comparison, we also subjected the <em>Fgama1</em> deletion mutant to a similar outcross to evaluate MSUD efficiency.</p>
<p>As expected, the outcross involving the <em>Fgama1</em> deletion mutant resulted in asci containing ascospores with a 1:1 segregation of eight ascospores with and without GFP signals in each ascus (<a href="#F2" class="usa-link">Fig. 2E</a>). However, some ascospores exhibited abnormal morphologies, such as oval shapes or intermediate forms between oval and spindle-shaped, indicating that MSUD could only partially silence <em>FgAMA1</em>. In contrast, during the outcrossing of the TR-<em>FgAMA1</em> transformant, all eight ascospores in each ascus were oval-shaped (<a href="#F2" class="usa-link">Fig. 2E</a>), resembling the results observed during self-fertilization. This suggests that TRISS, in combination with MSUD, achieves complete silencing of all three copies of <em>FgAMA1</em> during outcrossing. Similar results were obtained when testing the TR-<em>CDC2A</em> transformant, indicating that TRISS occurs in both selfing and outcrossing scenarios.</p>
<p>The outcrossing results also demonstrate that TRISS can function in trans. To further explore the trans silencing of the native <em>CDC2A</em> gene, we introduced a tandem repeat construct of <em>CDC2A</em> at an ectopic locus (FG1G36140) in PH-1 (<a href="#F2" class="usa-link">Fig. 2B</a>). The resulting transformants exhibited defects in ascospore formation, providing evidence that the native <em>CDC2A</em> gene was silenced in trans by the tandem repeat construct.</p></section><section id="sec7"><h3 class="pmc_sec_title">TRISS is mediated by an RNAi mechanism distinct from quelling and MSUD</h3>
<p>The <em>F. graminearum</em> genome encodes two Ago proteins (FgQde2 and FgSms2) and two Dicer proteins (FgDcl1 and FgDcl2) (<a href="#F3" class="usa-link">Fig. 3A</a>) (<a href="#R48" class="usa-link" aria-describedby="R48"><em>48</em></a>, <a href="#R49" class="usa-link" aria-describedby="R49"><em>49</em></a>). To determine whether TRISS depends on the RNAi machinery, we created deletion mutants for each of these four genes and transformed a linear fragment containing the full-length <em>FgAMA1</em> gene fused with the <em>neo</em> marker into these mutants to generate tandemly repeated <em>FgAMA1</em>. We found that deletion of <em>FgQDE2</em>, <em>FgDCL1</em>, or <em>FgDCL2</em> had no effect on <em>FgAMA1</em> silencing. However, in the <em>Fgsms2</em> mutants containing tandemly repeated <em>FgAMA1</em>, oval ascospores were no longer produced (<a href="#F3" class="usa-link">Fig. 3B</a>), indicating that the Ago protein FgSms2, but not FgQde2, is essential for TRISS.</p>
<figure class="fig xbox font-sm" id="F3"><h4 class="obj_head">Fig. 3. The RNAi machinery in TRISS.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366691_sciadv.adu7606-f3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a586/12366691/43ec5ddea5e7/sciadv.adu7606-f3.jpg" loading="lazy" height="650" width="785" alt="Fig. 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Comparison of the components identified in the TRISS machinery with those of two RNAi-related pathways: quelling and MSUD, which are known in <em>N. crassa</em>. (<strong>B</strong>) Morphology of asci and ascospores in marked strains showing the presence or absence of <em>FgAMA1</em> tandem repeats (TR-<em>FgAMA1</em>)–induced TRISS in RNAi-related gene deletion mutants and the slicing-defective <em>FgSMS2<sup>D696A</sup></em> mutant. Scale bar, 20 μm.</p></figcaption></figure><p>Since Dcl-1 and Dcl-2 are functionally redundant in quelling (<a href="#R50" class="usa-link" aria-describedby="R50"><em>50</em></a>, <a href="#R51" class="usa-link" aria-describedby="R51"><em>51</em></a>), we generated double mutants (<em>Fgdcl1 Fgdcl2</em>) and observed that <em>FgAMA1</em> silencing was completely abolished (<a href="#F3" class="usa-link">Fig. 3B</a>). This indicates that FgDcl1 and FgDcl2 are functionally redundant in TRISS.</p>
<p>To identify additional RNAi components involved in TRISS, we created deletion mutants for other core RNAi machinery components in <em>F. graminearum</em>, including Qde2-interacting protein (FgQip), five RdRPs (FgQde1, FgSad1, and FgRdrp3–5), and two helicases (FgQde3 and FgSad3) (<a href="#F3" class="usa-link">Fig. 3A</a>) (<a href="#R48" class="usa-link" aria-describedby="R48"><em>48</em></a>). Silencing of <em>FgAMA1</em> was abolished in the <em>Fgqde1</em>, <em>Fgqde3</em>, and <em>Fgqip</em> mutants (<a href="#F3" class="usa-link">Fig. 3B</a>), suggesting that FgQde1, FgQde3, and FgQip are also required for TRISS. However, deletion of <em>FgSAD1</em>, <em>FgSAD3</em>, and <em>FgRDRP3</em>–<em>5</em> had no effect on <em>FgAMA1</em> silencing. In addition, we generated <em>Fgrdrp3 Fgrdrp4 Fgrdrp5</em> triple mutants and confirmed that these genes are not involved in TRISS.</p>
<p>In conclusion, TRISS is mediated by an RNAi mechanism that uses a distinct combination of components from the RNAi machinery involved in quelling and MSUD pathways. These components include FgQde1, FgQde3, FgSms2, FgDcl1, FgDcl2, and FgQip (<a href="#F3" class="usa-link">Fig. 3A</a>).</p></section><section id="sec8"><h3 class="pmc_sec_title">siRNA biogenesis in tandem repeats is promoted by native transcription</h3>
<p>To investigate whether small RNAs are generated during TRISS, we isolated small RNAs from 7-day postfertilization (dpf) perithecia of the TR-<em>FgAMA1</em> and TR-<em>CDC2A</em> transformants, which contain tandem repeats of full-length <em>FgAMA1</em> and <em>CDC2A</em> at their native loci, respectively. We then performed small RNA sequencing (sRNA-seq).</p>
<p>Abundant small RNAs, ranging from 20 to 27 nucleotides (nt) in length, were detected in the tandem repeat regions of <em>FgAMA1</em> and <em>CDC2A</em>, with 24 nt being the predominant size (<a href="#F4" class="usa-link">Fig. 4, A and B</a>). These small RNAs exhibited a strong preference for uracil (U) at the 5′ end, but no obvious biases at the 3′ end (<a href="#F4" class="usa-link">Fig. 4A</a> and fig. S2). The small RNAs were derived from both sense and antisense strands of the tandem repeats (<a href="#F4" class="usa-link">Fig. 4B</a>), indicating that their biogenesis requires the formation of dsRNA. We refer to these small RNAs as trasiRNAs.</p>
<figure class="fig xbox font-sm" id="F4"><h4 class="obj_head">Fig. 4. trasiRNA biogenesis in TRISS.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366691_sciadv.adu7606-f4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a586/12366691/0c3d87cedc1c/sciadv.adu7606-f4.jpg" loading="lazy" height="667" width="733" alt="Fig. 4."></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Size distribution and normalized abundance of trasiRNAs derived from a tandem repeat of full-length <em>FgAMA1</em> (TR-<em>FgAMA1</em>) and <em>CDC2A</em> (TR-<em>CDC2A</em>) genes, a 2643-bp <em>CDC2A</em> fragment (TR-<em>CDC2A<sup>2643</sup></em>), and an IGR (TR-IGR) in the wild-type background or in the <em>Fgqde1</em>, <em>Fgsms2</em>, and <em>FgSMS2<sup>D696A</sup></em> mutant backgrounds. All trasiRNA abundances were normalized to reads per ten million (RPTM). The 5′ base of reads is represented by different colors. (<strong>B</strong>) Distribution and count of trasiRNAs mapped to the forward (blue) and reverse (red) strands of the marked tandem repeats. The structure of tandem repeats is shown above. (<strong>C</strong>) Distribution and count of trasiRNAs from eight copies of tandemly repeated 3′ regions of <em>PKS12</em> and a tandem repeat of <em>CDC2A</em> with the C terminus of the upstream copy fused to a GFP tag. Arrows indicate transcriptional start sites. Vertical dotted lines mark the transcriptional regions outside the repeat regions.</p></figcaption></figure><p>As a control, only a few small RNAs were detected in the <em>FgAMA1</em> and <em>CDC2A</em> loci in the wild-type strain PH-1, and these lacked the 5′-U preference (fig. S3). Most of the small RNAs detected in PH-1 were sense RNAs likely resulting from mRNA degradation.</p>
<p>To determine whether trasiRNA production is associated with the transcription of repeated genes by native promoters, we generated transformants containing tandem repeats of an intergenic region (IGR; chr3: 5806769–5810188 bp) in situ. The IGR repeat length (3420 bp) was comparable to the lengths of the <em>FgAMA1</em> (3602 bp) and <em>CDC2A</em> (3187 bp) repeats. Abundant trasiRNAs were detected in the IGR repeat region by sRNA-seq (<a href="#F4" class="usa-link">Fig. 4, A and B</a>), suggesting that native transcription of target genes is not strictly necessary for trasiRNA production. However, the amount of trasiRNAs generated from the IGR repeat region was only 50.9 and 27.5% of that produced in the repeat regions of <em>FgAMA1</em> and <em>CDC2A</em>, respectively. This indicates that native transcription of repeated genes can enhance trasiRNA production.</p>
<p>Consistent with our earlier observation that TRISS efficiency depends on repeat length, the amount of trasiRNAs produced from the 2643-bp tandem repeat of the 3′-region of <em>CDC2A</em> was nearly sevenfold lower than that from the 3187-bp tandem repeat of the full-length <em>CDC2A</em> (<a href="#F4" class="usa-link">Fig. 4, A and B</a>). The amount of trasiRNAs from the <em>CDC2A</em> repeat (3187 bp) was higher than that from the <em>FgAMA1</em> repeat (3602 bp) despite its shorter length. This discrepancy may be due to differences in transcription levels during sexual development: <em>CDC2A</em> transcription is higher in earlier stages but lower in later stages compared to <em>FgAMA1</em> (fig. S4). These findings suggest that higher transcription of tandem repeats in early sexual stages is critical for trasiRNA accumulation in later stages.</p></section><section id="sec9"><h3 class="pmc_sec_title">FgQde1 and FgSms2 are essential for the biogenesis of trasiRNAs</h3>
<p>In the quelling pathway, Qde-1 generates aRNA and dsRNA molecules from tandem repetitive transgenes, which are subsequently processed into siRNAs by Dicer proteins (Dcl-1 and Dcl-2) (<a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>). To investigate whether FgQde1 plays a similar role in trasiRNA biogenesis during TRISS, we conducted sRNA-seq analysis of perithecia from the <em>Fgqde1</em> mutant containing the <em>FgAMA1</em> tandem repeat.</p>
<p>As expected, trasiRNAs were completely absent in the repeated <em>FgAMA1</em> region in the <em>Fgqde1</em> mutant (<a href="#F4" class="usa-link">Fig. 4A</a>). This result demonstrates that FgQde1 is essential for the biogenesis of trasiRNAs in TRISS, likely through its role in generating aRNA or dsRNA from tandem repeats. Unexpectedly, our study also revealed that trasiRNAs were absent in the <em>Fgsms2</em> mutant in the repeated <em>FgAMA1</em> region (<a href="#F4" class="usa-link">Fig. 4A</a>). This finding suggests that FgSms2, an Ago protein, plays a critical role in trasiRNA biogenesis or stability.</p>
<p>Previous studies have shown that mutations in the first or second Asp (D) residue of the DDH motif in Ago proteins abolish their slicer activity, without affecting their ability to bind siRNAs (<a href="#R52" class="usa-link" aria-describedby="R52"><em>52</em></a>, <a href="#R53" class="usa-link" aria-describedby="R53"><em>53</em></a>). To investigate the role of FgSms2 in trasiRNA production and stability, we generated slicing-defective <em>FgSMS2<sup>D696A</sup></em> mutants. In these mutants, silencing of <em>FgAMA1</em> was abolished in transformants containing tandemly repeated <em>FgAMA1</em> (<a href="#F3" class="usa-link">Fig. 3B</a>). Furthermore, sRNA-seq analysis revealed that trasiRNAs were absent in the repeated <em>FgAMA1</em> region in the <em>FgSMS2<sup>D696A</sup></em> mutant (<a href="#F4" class="usa-link">Fig. 4A</a>). These results demonstrate that the slicer activity of FgSms2 is essential for trasiRNA production in TRISS. The absence of trasiRNAs in the <em>Fgsms2</em> and <em>FgSMS2<sup>D696A</sup></em> mutants is not merely involved in binding and stabilizing trasiRNAs but is directly required for their biogenesis.</p></section><section id="sec10"><h3 class="pmc_sec_title">No transitive biogenesis of secondary trasiRNAs in TRISS</h3>
<p>In yeast, plants, and nematodes, siRNA-programmed Ago proteins can recruit RdRPs to synthesize dsRNA using the targeted RNA as a template. This dsRNA is then cleaved by Dicers into secondary siRNAs, a process known as transitivity (<a href="#R11" class="usa-link" aria-describedby="R11"><em>11</em></a>–<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>). The critical role of FgSms2 in trasiRNA production prompted us to explore whether transitivity occurs in TRISS.</p>
<p>To investigate this, we generated a TR-<em>PKS12</em> transformant containing tandemly repeated 3′-regions of <em>PKS12</em>. If transitivity were present, secondary trasiRNAs would be produced corresponding to the upstream (5′) nonrepeated regions of native <em>PKS12</em> transcripts, as dsRNA would extend from the siRNA primer in the 3′-repeated regions to the 5′end of the RNA template. However, sRNA-seq analysis detected abundant trasiRNAs corresponding only to the repeated regions of <em>PKS12</em>. No trasiRNAs were observed in the upstream nonrepeated regions of <em>PKS12</em> in the TR-<em>PKS12</em> transformants (<a href="#F4" class="usa-link">Fig. 4C</a> and fig. S5).</p>
<p>To further investigate whether secondary siRNAs could be generated downstream of a primary siRNA trigger, we created a TR-C2AGFP transformant containing a tandem repeat of <em>CDC2A</em> with the C terminus of the upstream copy fused to a GFP tag. sRNA-seq analysis did not detect trasiRNAs corresponding to the GFP region downstream of the repeated regions (<a href="#F4" class="usa-link">Fig. 4C</a>). These results indicate that trasiRNA-programmed FgSms2 does not trigger transitive siRNA amplification in TRISS.</p></section><section id="sec11"><h3 class="pmc_sec_title">TRISS is primarily caused by translational repression by trasiRNAs</h3>
<p>To confirm the suppression of gene expression by trasiRNAs during TRISS, we performed quantitative reverse transcription PCR (qRT-PCR) to analyze the expression levels of <em>FgAMA1</em> in transformants containing an <em>FgAMA1</em> tandem repeat, both with (TR-<em>FgAMA1 Fgqde1</em>) and without (TR-<em>FgAMA1</em>) deletion of <em>FgQDE1</em>. As expected, in the TR-<em>FgAMA1 Fgqde1</em> transformant, where TRISS is disrupted, the transcript abundance of <em>FgAMA1</em> was approximately twofold higher than that in the wild type, consistent with the presence of two copies of <em>FgAMA1</em> (<a href="#F5" class="usa-link">Fig. 5A</a>). Unexpectedly, in the TR-<em>FgAMA1</em> transformant, which exhibited complete silencing phenotypes, the transcript abundance of <em>FgAMA1</em> was comparable to that in the wild type. Similar results were observed for <em>CDC2A</em> and <em>TUB1</em> in transformants containing tandem repeats of these genes (<a href="#F5" class="usa-link">Fig. 5A</a>).</p>
<figure class="fig xbox font-sm" id="F5"><h4 class="obj_head">Fig. 5. trasiRNA-mediated translational repression in TRISS.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366691_sciadv.adu7606-f5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a586/12366691/7eb4fb8b8d0c/sciadv.adu7606-f5.jpg" loading="lazy" height="449" width="787" alt="Fig. 5."></a></p>
<div class="p text-right font-secondary"><a href="figure/F5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Relative expression levels of <em>FgAMA1</em>, <em>CDC2A</em>, and <em>TUB1</em> in 7-dpf perithecia of marked strains, assayed by qRT-PCR. Numbers represent strain identifiers. Data are presented as means ± SD, <em>n</em> = 3 independent replicates. (<strong>B</strong>) Relative expression levels of <em>CDC2A</em> in 24-hour hyphae and 7-dpf perithecia of marked strains, assayed by qRT-PCR. The locations of primer pairs used are shown in the diagram. Data are presented as means ± SD, <em>n</em> = 3 independent replicates. (<strong>C</strong>) Western blots of total proteins isolated from 24-hour hyphae and 7-dpf perithecia of marked strains, showing the protein abundance of Cdc2A-GFP and Tub1. Proteins were probed with anti-GFP and anti-Tub1 antibodies, respectively. Detection with the anti-GAPDH antibody is used as an internal control for protein loading. (<strong>D</strong>) Polysome profiling of global translation efficiency and qRT-PCR assays of polysome-associated <em>CDC2A</em> mRNAs in WT and the TR-C2AGFP transformant, with two independent replicates. Sucrose gradient absorbance (<em>A</em><sub>260</sub>) was monitored in each fraction. Each point represents the mean of two independent replicates.</p></figcaption></figure><p>In a TR-<em>CDC2A</em> transformant (#7) that exhibited complete silencing phenotypes, the <em>CDC2A</em> transcript abundance was even higher than in the wild type. This suggests that transcript degradation is not the primary mechanism of TRISS. To further investigate, we used the TR-C2AGFP transformant, which contains tandem repeats of <em>CDC2A</em> fused with GFP, and measured expression levels using repeat-specific primers. As a control, we generated a C2AGFP transformant containing a single <em>CDC2A</em> copy fused with GFP. We also created a TR-C2AGFP <em>Fgqde1</em> strain by deleting <em>FgQDE1</em> in the TR-C2AGFP transformant (<a href="#F5" class="usa-link">Fig. 5B</a>).</p>
<p>qRT-PCR analysis revealed that during vegetative growth, the transcript abundance of both the upstream and downstream <em>CDC2A</em> repeats in the TR-C2AGFP transformant was comparable to that in the C2AGFP transformant and wild type (<a href="#F5" class="usa-link">Fig. 5B</a>), consistent with the absence of TRISS in hyphae. During sexual reproduction, the transcript abundance of both the upstream and downstream <em>CDC2A</em> repeats was reduced by approximately half in the TR-C2AGFP transformant. However, the total amount of <em>CDC2A</em> transcripts expressed by the TR-C2AGFP transformant remains comparable to that in the C2AGFP transformant and wild type (<a href="#F5" class="usa-link">Fig. 5B</a>). These results suggest that transcript cleavage occurs but is not the primary cause of TRISS-associated silencing phenotypes.</p>
<p>Western blot analysis provided further evidence supporting translational repression as the primary mechanism of TRISS. While the Cdc2A-GFP protein was readily detected in perithecia of the C2AGFP transformant, it was barely detectable in perithecia of the TR-C2AGFP transformant (<a href="#F5" class="usa-link">Fig. 5C</a>). However, the Cdc2A-GFP protein was detected in perithecia of the TR-C2AGFP <em>Fgqde1</em> strain and in the hyphae of the TR-C2AGFP transformant. Similar results were observed for Tub1 protein expression in the TR-<em>TUB1</em> transformant containing tandem repeats of <em>TUB1</em> (<a href="#F5" class="usa-link">Fig. 5C</a>).</p>
<p>Polysome profiling analysis revealed only a modest decrease in the amount of polysome-associated <em>CDC2A</em> mRNAs in the TR-C2AGFP transformant compared to the wild type (<a href="#F5" class="usa-link">Fig. 5D</a>). This suggests that the base pairing of trasiRNAs with target mRNAs may cause ribosome stalling, leading to impaired translational elongation. Together, these findings indicate that TRISS is primarily driven by the translational repression of target mRNAs by trasiRNAs rather than transcript degradation.</p></section><section id="sec12"><h3 class="pmc_sec_title">TRISS is not coupled to DSB repair pathways</h3>
<p>DSB-induced HR is critical for siRNA and quelling-induced siRNA (qiRNA) biogenesis in <em>N. crassa</em>, as it facilitates aRNA production at tandemly inserted transgenes and ribosomal DNA loci (<a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>). To investigate whether TRISS depends on DSB repair pathways, we examined the role of key components in these pathways (<a href="#F6" class="usa-link">Fig. 6A</a>).</p>
<figure class="fig xbox font-sm" id="F6"><h4 class="obj_head">Fig. 6. Independence of TRISS from DSB repair, H3K9me3, and DNA methylation.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366691_sciadv.adu7606-f6.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a586/12366691/c01553ab14b9/sciadv.adu7606-f6.jpg" loading="lazy" height="432" width="733" alt="Fig. 6."></a></p>
<div class="p text-right font-secondary"><a href="figure/F6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Key factors involved in the DSB checkpoint pathway and the two major DSB repair pathways: HR and NHEJ. (<strong>B</strong>) Ascus and ascospore morphology indicate the presence of <em>FgAMA1</em> or <em>FgCLN2</em> tandem repeats (TR-<em>FgAMA1</em> or TR-<em>FgCLN2</em>)–induced TRISS in deletion mutants of DSB repair pathways, with the phenotypes of the deletion mutants as a control. The TR-<em>FgRAD51</em> transformant containing tandemly repeated <em>FgRAD51</em> displays similar defects in ascospore formation to the <em>Fgrad51</em> deletion mutant. Scale bar, 20 μm. (<strong>C</strong>) Agarose gel electrophoresis showing the presence of TR-<em>FgAMA1</em> in random ascospore progeny of TR-<em>FgAMA1</em> and TR-<em>FgAMA1 Fgqde1</em> strains detected by PCR. (<strong>D</strong>) Oval ascospores indicate the presence of TR-<em>FgAMA1</em>–induced TRISS in the <em>Fgdim2</em>, <em>Fghp1</em>, and <em>Fgdim5</em> deletion mutants. Scale bar, 20 μm.</p></figcaption></figure><p>Canonical meiotic recombination requires Spo11 to produce DSB (<a href="#R54" class="usa-link" aria-describedby="R54"><em>54</em></a>). We generated <em>Fgspo11</em> deletion mutants containing tandem repeats of <em>FgAMA1</em> and observed that the mutants produced oval ascospores (<a href="#F6" class="usa-link">Fig. 6B</a>), indicating that TRISS still occurs in the absence of Spo11. This suggests that TRISS does not depend on canonical meiotic recombination.</p>
<p>We next examined the roles of Rad51 and Rad54, which are essential components of the eukaryotic HR pathway (<a href="#R55" class="usa-link" aria-describedby="R55"><em>55</em></a>). Deletion of <em>FgRAD54</em> in transformants containing tandem repeats of <em>FgAMA1</em> did not affect TRISS, as oval ascospores were still produced (<a href="#F6" class="usa-link">Fig. 6B</a>). Since <em>FgRAD51</em> deletion mutants did not form ascospores, we used a “silencing the silencer” approach (<a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>) by generating transformants with tandem repeats of <em>FgRAD51</em>. If <em>FgRAD51</em> was required for TRISS, it would not be efficiently self-silenced by tandem repeats. However, <em>FgRAD51</em> was completely silenced in the TR-<em>FgRAD51</em> transformants (<a href="#F6" class="usa-link">Fig. 6B</a>), suggesting that neither <em>FgRAD51</em> nor <em>FgRAD54</em> is required for TRISS.</p>
<p>We also examined TRISS occurrence in deletion mutants of other genes involved in DSB repair and checkpoint pathways, including two responsible for sensing DSBs (<em>FgATM</em> and <em>FgATR</em>) (<a href="#R56" class="usa-link" aria-describedby="R56"><em>56</em></a>) and two for NHEJ (<em>FgKU70</em> and <em>FgKU80</em>) (<a href="#R55" class="usa-link" aria-describedby="R55"><em>55</em></a>). As with HR mutants, TRISS was still observed in these DSB checkpoint and NHEJ mutants (<a href="#F6" class="usa-link">Fig. 6B</a>).</p>
<p>Together, these results demonstrate that TRISS does not rely on DSB repair pathways. Recombination between repeats can lead to the loss of repeats. To determine whether trasiRNAs play a role in repeat stabilization, we analyzed ascospore progeny from transformants containing tandem repeats of <em>FgAMA1</em> with (TR-<em>FgAMA1 Fgqde1</em>) or without (TR-<em>FgAMA1</em>) <em>FgQDE1</em> deletion. PCR analysis revealed that the <em>FgAMA1</em> tandem repeat was stable in all progeny, regardless of <em>FgQDE1</em> deletion (<a href="#F6" class="usa-link">Fig. 6C</a>). This indicates that tandem repeats are stable during sexual reproduction in <em>F. graminearum</em> and independent of trasiRNAs.</p></section><section id="sec13"><h3 class="pmc_sec_title">TRISS does not depend on H3K9me3 and DNA cytosine methylation</h3>
<p>DNA cytosine methylation and H3K9 methylation are critical for siRNA production in many organisms (<a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>, <a href="#R57" class="usa-link" aria-describedby="R57"><em>57</em></a>). To investigate their role in TRISS, we analyzed mutants lacking key components of these pathways.</p>
<p>Deletion of the DNA methyltransferase gene <em>FgDIM2</em> and the HP1 gene <em>FgHP1</em> in TR-<em>FgAMA1</em> transformants did not affect TRISS. Oval ascospores were still produced in both TR-<em>FgAMA1 Fgdim2</em> and TR-<em>FgAMA1 Fghp1</em> mutants (<a href="#F6" class="usa-link">Fig. 6D</a>), indicating that TRISS is independent of DNA methylation mediated by FgDim2 and FgHP1.</p>
<p>To examine the role of H3K9 methylation, we deleted <em>FgDIM5</em>, which mediates H3K9me3 required for constitutive heterochromatin formation (<a href="#R58" class="usa-link" aria-describedby="R58"><em>58</em></a>), in TR-<em>FgAMA1</em> transformants. The mutants still produced oval ascospores (<a href="#F6" class="usa-link">Fig. 6D</a>), indicating that H3K9me3 is not required for TRISS activation. These findings show that TRISS does not depend on DNA cytosine methylation or H3K9me3-mediated heterochromatin formation.</p></section><section id="sec14"><h3 class="pmc_sec_title">TRISS is promoted by FgRid</h3>
<p>Since tandem repeats induce both TRISS and RIP, we investigated whether there is cross-talk between these two processes. To investigate potential cross-talk between these processes, we analyzed the role of <em>FgRID</em>, the ortholog of the <em>N. crassa rid</em> gene, which is crucial for RIP in <em>F. graminearum</em> (<a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>).</p>
<p>In the TR-<em>FgAMA1 Fgqde1</em> strain, oval ascospores were observed in 20% of asci in 10-dpf perithecia (<a href="#F7" class="usa-link">Fig. 7A</a>). Deletion of <em>FgRID</em> in this strain abolished oval ascospore production (<a href="#F7" class="usa-link">Fig. 7A</a>), confirming that the observed oval ascospores were caused by RIP. These results indicate that RIP operates independently of FgQde1 and TRISS.</p>
<figure class="fig xbox font-sm" id="F7"><h4 class="obj_head">Fig. 7. Roles of FgRid in trasiRNA biogenesis.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366691_sciadv.adu7606-f7.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a586/12366691/cfa2565ca908/sciadv.adu7606-f7.jpg" loading="lazy" height="385" width="787" alt="Fig. 7."></a></p>
<div class="p text-right font-secondary"><a href="figure/F7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) Morphology of asci and ascospores in marked strains at indicated sexual stages (7 or 10 days). Scale bar, 20 μm. (<strong>B</strong>) Distribution and abundance of trasiRNAs in the tandem repeat regions of the TR-<em>FgAMA1 Fgrid</em> strain. (<strong>C</strong>) Multiple sequence alignments showing S366 and E408 sites in the DNA methyltransferase domain of Rid and Dim2 orthologs. (<strong>D</strong>) Morphology of asci and ascospores in marked strains at indicated sexual stages (7 or 10 days). Scale bar, 20 μm.</p></figcaption></figure><p>Unexpectedly, deletion of <em>FgRID</em> in the TR-<em>FgAMA1</em> strain resulted in a substantial reduction in TRISS. Oval ascospores were rarely observed in 7-dpf perithecia of the TR-<em>FgAMA1 Fgrid</em> strain (<a href="#F7" class="usa-link">Fig. 7A</a>). sRNA-seq analysis revealed a limited number of trasiRNAs in the repeated <em>FgAMA1</em> region in 7-dpf perithecia of the TR-<em>FgAMA1 Fgrid</em> strain (<a href="#F7" class="usa-link">Fig. 7B</a>), highlighting the critical role of <em>FgRID</em> in trasiRNA production.</p>
<p>The proportion of asci with oval ascospores increased in 10-dpf perithecia of the TR-<em>FgAMA1 Fgrid</em> strain (<a href="#F7" class="usa-link">Fig. 7A</a>). Sequencing of PCR products from 15 oval ascospore progeny of the TR-<em>FgAMA1 Fgrid</em> strain did not detect any RIP mutants in the repeated <em>FgAMA1</em> regions (fig. S6), confirming that the mutant phenotype is attributed to TRISS rather than RIP. Since later-formed ascospores undergo more mitotic divisions during the dikaryotic stage (<a href="#R59" class="usa-link" aria-describedby="R59"><em>59</em></a>), this suggests that the dikaryotic stage is critical for TRISS.</p></section><section id="sec15"><h3 class="pmc_sec_title">TRISS requires the catalytic activity of FgRid but is independent of RIP</h3>
<p>The putative DNA methyltransferase domain of Rid orthologs contains a conserved glutamate (E) residue in Motif VI, essential for catalytic activity (<a href="#F7" class="usa-link">Fig. 7C</a>) (<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>). To investigate the role of FgRid’s catalytic activity in TRISS, we generated a catalytically inactive mutant (<em>FgRID<sup>E408A</sup></em>) and transformed it with tandem repeats of <em>FgAMA1</em>. Similar to the TR-<em>FgAMA1 Fgrid</em> strain, the TR-<em>FgAMA1 FgRID<sup>E408A</sup></em> strain exhibited a low occurrence of oval ascospores in 7-dpf perithecia (<a href="#F7" class="usa-link">Fig. 7D</a>), indicating that the catalytic activity of FgRid is essential for TRISS.</p>
<p>We also identified a functional divergence site between Rid and Dim2 orthologs, where Rid orthologs have a serine (S) residue while Dim2 orthologs have a glycine (G) residue (<a href="#F7" class="usa-link">Fig. 7C</a>). In TR-F<em>gAMA1 FgRID<sup>S366G</sup></em> strains, TRISS remained intact, as oval ascospores were readily observed (<a href="#F7" class="usa-link">Fig. 7D</a>). However, RIP was abolished in the TR-<em>FgAMA1 Fgqde1 FgRID<sup>S366G</sup></em> strain as no oval ascospores were observed in its 10-dpf perithecia (<a href="#F7" class="usa-link">Fig. 7D</a>), indicating that the S366G mutation disrupts RIP but not TRISS. These findings demonstrate that TRISS relies on the catalytic activity of FgRid but operates independently of RIP.</p></section><section id="sec16"><h3 class="pmc_sec_title">FgRid promotes the recruitment of FgQde1 and FgQde3 to tandem repeat regions</h3>
<p>In <em>N. crassa</em>, Qde-1 preferentially binds ssDNA via its DdRP activity and ssRNA via its RdRP activity, but it is inactivated when dsDNA is used as a template (<a href="#R24" class="usa-link" aria-describedby="R24"><em>24</em></a>, <a href="#R60" class="usa-link" aria-describedby="R60"><em>60</em></a>). To investigate the association of FgRid and FgQde1 with tandem repeats, we used the DamID method (<a href="#R61" class="usa-link" aria-describedby="R61"><em>61</em></a>), which uses the DNA adenine methyltransferase (Dam) from <em>Escherichia coli</em> to probe DNA-protein interactions. We expressed an FgRid-Dam fusion protein in situ or Dam alone at the FG1G36140 locus in the TR-<em>FgAMA1</em> strain and assessed adenine-methylated <em>FgAMA1</em> using digestion with methylation-sensitive restriction enzymes followed by qPCR (DamID-qPCR) (<a href="#R62" class="usa-link" aria-describedby="R62"><em>62</em></a>).</p>
<p>Analysis of genomic DNA from 7-dpf perithecia treated with MboI, which specifically digests unmethylated GATC sites but not methylated ones, revealed that FgRid-Dam was enriched nearly fivefold at the tandem repeat of <em>FgAMA1</em> compared to Dam alone (<a href="#F8" class="usa-link">Fig. 8A</a>). This enrichment was not observed when the genomic DNA was treated with Sau3AI, which digests both methylated and unmethylated GATC sites, serving as a negative control. In addition, FgRid-Dam was not enriched at the IGR in the tested strains (fig. S7), providing further validation as a negative control. Using the same approach, we observed that FgQde1-Dam was also enriched at the tandem repeat region of <em>FgAMA1</em> (<a href="#F8" class="usa-link">Fig. 8B</a>). These findings indicate that both FgRid and FgQde1 are associated with tandem repeats. Notably, deletion of <em>FgRID</em> abolished the association of FgQde1 with the <em>FgAMA1</em> tandem repeat as no enrichment of FgQde1-Dam was detected in the TR-<em>FgAMA1 FgQDE1-Dam Fgrid</em> strain (<a href="#F8" class="usa-link">Fig. 8B</a>). These results suggest that FgQde1 is specifically recruited to tandem repeat regions, and this recruitment is facilitated by FgRid.</p>
<figure class="fig xbox font-sm" id="F8"><h4 class="obj_head">Fig. 8. Mechanisms of FgRid and FgSms2 in trasiRNA biogenesis.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366691_sciadv.adu7606-f8.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a586/12366691/86c2f68a8b5f/sciadv.adu7606-f8.jpg" loading="lazy" height="545" width="787" alt="Fig. 8."></a></p>
<div class="p text-right font-secondary"><a href="figure/F8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(<strong>A</strong>) DAMID-qPCR of FgRid-Dam TR-<em>FgAMA1</em>. Data are presented as means ± SD, <em>n</em> = 3 independent replicates. A flowchart of the DAMID-qPCR assay is shown. (<strong>B</strong> to <strong>D</strong> and <strong>G</strong>) DAMID-qPCR for indicated genotypes. Data are presented as means ± SD, <em>n</em> = 3 independent replicates. Student’s <em>t</em> test; ***<em>P</em> &lt; 0.001; **<em>P</em> &lt; 0.01; *<em>P</em> &lt; 0.05; ns, not significant. (<strong>E</strong>) Co-IP assays showing interactions between FgRpa1, FgRid, and FgSms2. Western blots of total proteins (input) and proteins eluted from anti-GFP beads (GFP-IP) were probed with anti-GFP, anti-FLAG, and anti-HA antibodies. (<strong>F</strong>) Co-IP assays showing interactions between FgRpa1 and FgQde1. Western blots of total proteins (input) and proteins eluted from anti-GFP beads (GFP-IP) were probed with anti-GFP and anti-FLAG antibodies. (<strong>H</strong>) Co-IP assays showing that interactions between FgRid and FgSms2 are independent of FgSms2. Western blots of total proteins (input) and proteins eluted from anti-GFP beads (GFP-IP) were probed with anti-GFP and anti-FLAG antibodies. (<strong>I</strong>) Interaction network among FgRid, FgQde3, FgRpa1, FgQde1, and FgSms2. Arrows indicate the direction of protein recruitment, while dotted lines represent unconfirmed interactions in <em>F. graminearum</em>.</p></figcaption></figure><p>Qde-3 is thought to unwind duplex DNA and DNA/RNA hybrids using its adenosine triphosphate–dependent 3′–5′ DNA helicase activity, which generates ssDNA and ssRNA templates for Qde-1 (<a href="#R24" class="usa-link" aria-describedby="R24"><em>24</em></a>, <a href="#R63" class="usa-link" aria-describedby="R63"><em>63</em></a>). FgQde3-Dam was also enriched at the tandem repeat region of <em>FgAMA1</em>, and this association depended on the catalytic activity of FgRid but not on its RIP function. FgQde3-Dam enrichment was lost in the <em>Fgrid</em> and <em>FgRID<sup>E408A</sup></em> mutants but was still observed in the <em>FgRID<sup>S366G</sup></em> mutants (<a href="#F8" class="usa-link">Fig. 8, C and D</a>). Therefore, FgRid is also critical for recruiting FgQde3 to tandem repeat regions.</p></section><section id="sec17"><h3 class="pmc_sec_title">FgRid interacts with RPA and requires FgQde3 for this interaction</h3>
<p>To identify proteins associated with FgRid, we generated transformants expressing a C-terminal, 3×FLAG–tagged FgRid (FgRid-3×FLAG) under the control of the RP30 promoter. FgRid-3×FLAG and its interacting proteins were copurified from total protein extracts isolated from 7-dpf perithecia using anti-FLAG affinity beads, followed by mass spectrometry analysis. Among the copurified proteins, we identified FgRpa1, a homolog of the RPA subunit 1. To confirm the interaction between FgRid and FgRpa1, we performed coimmunoprecipitation (Co-IP) experiments using strains expressing FgRid-3×FLAG and C-terminal GFP–tagged FgRpa1 (FgRpa1-GFP) under the RP30 promoter. Since <em>FgRID</em> contains a premature stop codon (PSC) editing site and only expresses the full-length functional protein during sexual development after RNA editing (<a href="#R39" class="usa-link" aria-describedby="R39"><em>39</em></a>), the edited <em>FgRID<sup>TGG</sup></em> allele was used in the FgRid-3×FLAG construct. Western blots confirmed that FgRpa1 interacts with FgRid in vivo (<a href="#F8" class="usa-link">Fig. 8E</a>).</p>
<p>In humans, RPA interacts with BLM and WRN, helicase homologs of Qde-3, to enhance their DNA unwinding activity (<a href="#R64" class="usa-link" aria-describedby="R64"><em>64</em></a>, <a href="#R65" class="usa-link" aria-describedby="R65"><em>65</em></a>). In <em>Neurospora</em>, RPA-1 is known to interact with both Qde-1 and Qde-3, with the interaction between Qde-1 and RPA-1 requiring Qde-3 (<a href="#R24" class="usa-link" aria-describedby="R24"><em>24</em></a>). To confirm these interactions in <em>F. graminearum</em>, we performed Co-IP experiments using strains expressing C-terminal 3×FLAG–tagged FgRpa1 (FgRpa1-3×FLAG) along with N-terminal GFP–tagged FgQde1 (GFP-FgQde1) or FgQde3 (GFP-FgQde3). We confirmed that FgRpa1 interacts with FgQde1 (<a href="#F8" class="usa-link">Fig. 8F</a>). However, GFP-FgQde3 could not be detected by Western blot, making Co-IP assays for FgQde3 challenging. Despite this limitation, we demonstrated that the interaction between FgQde1 and FgRpa1 requires FgQde3 (<a href="#F8" class="usa-link">Fig. 8F</a>). Deletion of <em>FgQDE3</em> disrupted the association of FgQde1 with the <em>FgAMA1</em> tandem repeat (<a href="#F8" class="usa-link">Fig. 8G</a>), highlighting the critical role of FgQde3 in recruiting FgQde1 to ssDNA within tandem repeats. It is likely that FgQde3 and FgRpa1 generate the ssDNA template for FgQde1 in TRISS, similar to the quelling process in <em>Neurospora</em>. In addition, the interaction between FgRid and FgRpa1 also requires FgQde3 (<a href="#F8" class="usa-link">Fig. 8E</a>), suggesting that FgQde3 mediates their interaction either through direct protein-protein interaction or by providing an interaction condition.</p></section><section id="sec18"><h3 class="pmc_sec_title">FgSms2 also interacts with FgRpa1 and requires FgQde3 for this interaction</h3>
<p>To investigate the role of FgSms2 in trasiRNA production, we performed mass spectrometry analysis on FgSms2–3×FLAG transformants expressing a C-terminal 3×FLAG–tagged FgSms2 in 7-dpf perithecia. FgRpa1 was identified as a copurified protein. Using strains expressing FgRpa1-GFP and N-terminal hemagglutinin (HA)–tagged FgSms2 (HA-FgSms2) under the RP30 promoter, we confirmed the in vivo interaction between FgSms2 and FgRpa1 through Co-IP experiments (<a href="#F8" class="usa-link">Fig. 8E</a>). Furthermore, this interaction was shown to depend on FgQde3 (<a href="#F8" class="usa-link">Fig. 8E</a>). Since FgRid is only expressed during sexual stages due to PSC editing, the observed interaction between FgSms2 and FgRpa1 in hyphae suggests that this interaction is independent of FgRid. In addition, native <em>FgSMS2</em> expression is minimal in hyphae (fig. S8), and its overexpression did not enhance the interaction between FgRid and FgRpa1 (<a href="#F8" class="usa-link">Fig. 8H</a>), indicating that FgSms2 is not required for this interaction.</p>
<p>In summary, our findings reveal two distinct mechanisms that enhance the TRISS response, independent of traditional transitive siRNA amplification (<a href="#F8" class="usa-link">Figs. 8I</a> and <a href="#F9" class="usa-link">9</a>). The first mechanism involves FgRid, which facilitates the recruitment of the FgRpa1-FgQde1 complex to ssDNA in tandem repeats by interacting with FgRpa1. The second mechanism involves the interaction between FgSms2 and FgRpa1, which promotes the recruitment of the FgRpa1-FgQde1 complex to ssDNA. This process is guided by trasiRNA-programmed FgSms2 to target ssRNA molecules produced in tandem repeat regions.</p>
<figure class="fig xbox font-sm" id="F9"><h4 class="obj_head">Fig. 9. A model showing the proposed mechanism of TRISS.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366691_sciadv.adu7606-f9.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/a586/12366691/97669289a8c4/sciadv.adu7606-f9.jpg" loading="lazy" height="378" width="750" alt="Fig. 9."></a></p>
<div class="p text-right font-secondary"><a href="figure/F9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Tandem repeats are specifically recognized by the TRISS pathway, likely involving FgQde3, which may unwind dsDNA into ssDNA and recruit FgRpa1 and FgQde1. FgQde1 uses its DdRP and RdRP activities to first produce aRNAs and then dsRNA, thereby activating the downstream RNAi pathway. In the cytoplasm, trasiRNA-programmed FgSms2 primarily mediates translational repression of target mRNAs. In the nucleus, trasiRNA-programmed FgSms2 facilitates the recruitment of the FgRpa1-FgQde1 complex to ssDNA by targeting both the aRNA generated by FgQde1 and the native mRNA produced by RNA polymerase II in tandem repeats. In addition, FgRid in tandem repeats promotes the recruitment of the FgRpa1-FgQde1 complex to ssDNA by interacting with FgRpa1. The interactions of FgRpa1 with FgQde1, FgRid, and FgSms2 all depend on FgQde3.</p></figcaption></figure></section></section><section id="sec19"><h2 class="pmc_sec_title">DISCUSSION</h2>
<p>Genome defense systems are critical for restricting the proliferation of selfish genetic elements and ensuring genomic stability, particularly during sexual reproduction when genomes from two distinct parental sources combine. In filamentous ascomycetes, two well-known genome defense mechanisms, RIP and MSUD, are active during the sexual cycle. RIP functions by introducing C-to-T mutations on both DNA strands of repetitive sequences in a paired manner, effectively inactivating both linked and unlinked repetitive elements within a haploid parental genome (<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>, <a href="#R19" class="usa-link" aria-describedby="R19"><em>19</em></a>, <a href="#R28" class="usa-link" aria-describedby="R28"><em>28</em></a>, <a href="#R31" class="usa-link" aria-describedby="R31"><em>31</em></a>). Meanwhile, MSUD acts during meiosis by identifying unpaired DNA segments between homologous chromosomes and silencing all homologous copies posttranscriptionally through RNAi (<a href="#R3" class="usa-link" aria-describedby="R3"><em>3</em></a>, <a href="#R21" class="usa-link" aria-describedby="R21"><em>21</em></a>, <a href="#R27" class="usa-link" aria-describedby="R27"><em>27</em></a>). The genome of <em>F. graminearum</em> is characterized by a low abundance of repetitive sequences, a feature partially attributed to the activity of RIP (<a href="#R30" class="usa-link" aria-describedby="R30"><em>30</em></a>, <a href="#R66" class="usa-link" aria-describedby="R66"><em>66</em></a>, <a href="#R67" class="usa-link" aria-describedby="R67"><em>67</em></a>). However, the efficiency of RIP in <em>F. graminearum</em> is substantially lower than that observed in <em>N. crassa</em> (<a href="#R32" class="usa-link" aria-describedby="R32"><em>32</em></a>, <a href="#R66" class="usa-link" aria-describedby="R66"><em>66</em></a>). Moreover, given that <em>F. graminearum</em> is predominantly self-fertile and rarely outcrosses, the natural activation of MSUD is predicted to be infrequent.</p>
<p>In this study, we identified and characterized TRISS, an RNAi-dependent genome defense system in <em>F. graminearum</em>. TRISS is specifically triggered by tandem repeats and primarily functions to repress the translation of repeated genes during sexual reproduction. This mechanism operates from the dikaryotic stage through ascosporogenesis, overlapping with the stages during which both RIP and MSUD are active. Unlike MSUD, which is predominantly active during outcrossing, TRISS functions during both selfing and outcrossing. Our results indicate that TRISS operates independently of the mode of tandem-repeat formation, suggesting that it could potentially target suitable tandem repeats derived from both endogenous transposons and exogenous nucleic invaders. These findings position TRISS as an additional genome defense mechanism that may contribute to the remarkably low repeat content observed in the <em>F. graminearum</em> genome.</p>
<p>The TRISS mechanism in <em>F. graminearum</em> shares notable similarities with the quelling pathway observed during vegetative growth in <em>N. crassa</em>. Both systems are triggered by tandem transgene repeats and use similar RNAi components, including Qde1, Qde3, Dcl1, Dcl2, and Qip (<a href="#R21" class="usa-link" aria-describedby="R21"><em>21</em></a>, <a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>). However, there are key distinctions. While Qde-2 is essential for quelling, FgSms2, an Ago protein specific to MSUD, is necessary for TRISS. Despite the conservation of these components, quelling has not been observed in <em>F. graminearum</em>. This is evidenced by a TR-<em>PKS12</em> transformant containing eight tandem repeats of <em>PKS12</em>, which displayed a wild-type phenotype (<a href="#F1" class="usa-link">Figs. 1J</a> and <a href="#F4" class="usa-link">4C</a>), indicating minimal or absent quelling activity during vegetative growth.</p>
<p>Another major difference between TRISS and quelling lies in their dependence on HR. While quelling requires DSB-triggered HR of repetitive sequences (<a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>, <a href="#R23" class="usa-link" aria-describedby="R23"><em>23</em></a>), TRISS does not. In addition, MSUD and recombination-independent quelling (RIQ), a recently reported quelling-associated process in <em>N. crassa</em>, do not rely on DSB-triggered HR but depend on Sad-6, a conserved SWItch/Sucrose Non-Fermentable (SWI/SNF) chromatin remodeler (<a href="#R68" class="usa-link" aria-describedby="R68"><em>68</em></a>–<a href="#R70" class="usa-link" aria-describedby="R70"><em>70</em></a>). However, our findings indicate that FgSad6 is not required for TRISS (fig. S1). Furthermore, while Qde-2 is not involved in siRNA production during quelling and RIQ (<a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>, <a href="#R53" class="usa-link" aria-describedby="R53"><em>53</em></a>, <a href="#R68" class="usa-link" aria-describedby="R68"><em>68</em></a>), FgSms2 is indispensable for siRNA production in TRISS. The discovery of TRISS unveils an RNAi pathway in filamentous ascomycetes that is distinct from established RNAi pathways.</p>
<p>SIS is another RNAi mechanism triggered by tandem transgene arrays, identified in both opposite-sex and unisexual reproduction in the basidiomycete yeast <em>C. neoformans</em> (<a href="#R18" class="usa-link" aria-describedby="R18"><em>18</em></a>, <a href="#R71" class="usa-link" aria-describedby="R71"><em>71</em></a>). Unlike TRISS, which is restricted to sexual stages, SIS can propagate in progeny during vegetative growth for many generations once initiated. Notably, TRISS appears to be a more efficient genome defense mechanism than SIS and quelling. A single tandem repeat is sufficient to completely silence the targeted gene in TRISS, whereas this is insufficient to trigger quelling in <em>N. crassa</em> (<a href="#R22" class="usa-link" aria-describedby="R22"><em>22</em></a>) or SIS in <em>C. neoformans</em> (<a href="#R18" class="usa-link" aria-describedby="R18"><em>18</em></a>, <a href="#R71" class="usa-link" aria-describedby="R71"><em>71</em></a>).</p>
<p>While miRNAs typically repress gene expression through translational repression, siRNAs are generally known to induce mRNA cleavage (<a href="#R72" class="usa-link" aria-describedby="R72"><em>72</em></a>). Recent studies in plants and animals have shown that siRNAs can also mediate translational suppression (<a href="#R73" class="usa-link" aria-describedby="R73"><em>73</em></a>–<a href="#R77" class="usa-link" aria-describedby="R77"><em>77</em></a>). However, genome defense mechanisms involving translational repression are rarely documented. In this study, we demonstrate that trasiRNAs in <em>F. graminearum</em> predominantly mediate translational repression, providing a clear example of translational repression being used as a mechanism for genomic defense. Contrary to the traditional inference that imperfect base pairing between small RNAs and target RNAs favors translational repression (<a href="#R72" class="usa-link" aria-describedby="R72"><em>72</em></a>), our results show that trasiRNAs, despite being perfectly complementary to their target RNAs, effectively mediate translational repression but are less efficient at inducing target cleavage. Our findings suggest that the base pairing between trasiRNAs and their target RNAs may cause ribosome stalling, although the precise mechanism remains to be elucidated. It is possible that other factors associated with the FgSms2-trasiRNA complex contribute to facilitating this translational repression.</p>
<p>In various organisms such as fission yeast (<a href="#R14" class="usa-link" aria-describedby="R14"><em>14</em></a>), <em>Mucor</em> (<a href="#R78" class="usa-link" aria-describedby="R78"><em>78</em></a>), nematodes (<a href="#R13" class="usa-link" aria-describedby="R13"><em>13</em></a>), and plants (<a href="#R76" class="usa-link" aria-describedby="R76"><em>76</em></a>), Ago-mediated slicing not only cleaves target RNAs but also initiates transitivity by recruiting RdRP to synthesize dsRNAs, which are then processed into secondary siRNAs, amplifying the RNAi response. However, our study found no evidence of transitive secondary siRNA production in the FgSms2-dependent trasiRNA pathway. This deviation from the classical model is likely attributable to Qde-1, which has both DdRP and RdRP activities (<a href="#R24" class="usa-link" aria-describedby="R24"><em>24</em></a>), enabling it to independently generate aRNAs and dsRNAs without requiring transitivity. Our study demonstrates that the slicer activity of FgSms2 is crucial for trasiRNA production. Ago’s slicer activity is known to play a role in removing the passenger strand from siRNA duplexes (<a href="#R53" class="usa-link" aria-describedby="R53"><em>53</em></a>, <a href="#R79" class="usa-link" aria-describedby="R79"><em>79</em></a>). Our findings show that FgSms2 interacts with FgRpa1, and FgRpa1 interacts with FgQde1. This suggests that siRNAs may guide FgSms2 to complementary target mRNA or aRNA scaffolds, facilitating the recruitment of FgRpa1 and FgQde1 to ssDNA in tandemly repeated regions. This recruitment initiates aRNA and dsRNA production, amplifying the silencing signal without relying on transitive RNAi.</p>
<p>Previous studies have identified self-reinforcing positive feedback loops linking siRNAs to DNA cytosine and H3K9 methylation in organisms like <em>Schizosaccharomyces pombe</em>, <em>N. crassa</em>, Arabidopsis, and mammals (<a href="#R12" class="usa-link" aria-describedby="R12"><em>12</em></a>, <a href="#R57" class="usa-link" aria-describedby="R57"><em>57</em></a>). However, these epigenetic marks are barely detectable in <em>F. graminearum</em> (<a href="#R35" class="usa-link" aria-describedby="R35"><em>35</em></a>), and our results confirm that TRISS functions independently of DNA cytosine and H3K9 methylation. Given that RPA is a conserved eukaryotic ssDNA binding protein (<a href="#R80" class="usa-link" aria-describedby="R80"><em>80</em></a>) and that the essential RNAi components in TRISS are conserved, at least in filamentous ascomycetes (fig. S9), the positive feedback loop involving the interaction between Sms2 and RPA might represent a conserved mechanism in these organisms.</p>
<p>Rid has long been recognized as a key mediator of RIP (<a href="#R32" class="usa-link" aria-describedby="R32"><em>32</em></a>), but its connection to RNAi mechanisms was previously unknown. Our study revealed that FgRid plays a crucial role in TRISS, likely by recruiting the FgQde1-FgRpa1 complex to ssDNA within tandem repeats. In <em>N. crassa</em>, siRNAs derived from endogenous transposon relics that have undergone RIP have been detected in the vegetative stages (<a href="#R81" class="usa-link" aria-describedby="R81"><em>81</em></a>). Similar to our findings for TRISS, the presence of these siRNAs depends on functional Qde-1, Qde-3, and two dicers, but not Qde-2 (<a href="#R81" class="usa-link" aria-describedby="R81"><em>81</em></a>). Although siRNA production during sexual stages and the role of Sms2 were not directly assessed, an increase in siRNA production was observed in the <em>qde-2</em> mutant (<a href="#R81" class="usa-link" aria-describedby="R81"><em>81</em></a>), possibly due to the induction of <em>SMS-2</em> expression following the deletion of <em>QDE-2</em>, as <em>SMS-2</em> is predominantly expressed during sexual stages (<a href="#R82" class="usa-link" aria-describedby="R82"><em>82</em></a>). Therefore, a TRISS-like mechanism may also operate in <em>N. crassa</em>, although its efficiency may be less critical than in <em>F. graminearum</em>. Unlike in <em>F. graminearum</em>, the <em>RID</em> gene in <em>N. crassa</em> does not contain PSC-editing sites in its coding regions and can express the full-length functional protein outside sexual stages (<a href="#R39" class="usa-link" aria-describedby="R39"><em>39</em></a>), which might account for the lower amount of siRNAs detected in vegetative stages. Given the conservation of both TRISS and RIP components among filamentous ascomycetes (fig. S9), it is plausible that the coupling of RIP and RNAi mechanisms to inactivate tandem repeats at both genetic and posttranscriptional levels during sexual stages is a common strategy in these organisms. This dual mechanism could provide a robust means of maintaining genomic stability by effectively silencing potentially harmful genetic elements.</p>
<p>RIP was found nearly four decades ago (<a href="#R29" class="usa-link" aria-describedby="R29"><em>29</em></a>), yet the precise mechanism underlying the C-to-T mutations remains unresolved. Recent studies have shown that RPA can protect ssDNA from uracil DNA glycosylases (UNG), enabling C-to-T mutagenesis by allowing replicative DNA polymerases in yeast to faithfully copy uracil (U) bases formed by C-to-U Apolipoprotein B mRNA-Editing Catalytic Polypeptide-like (APOBEC) deaminases (<a href="#R83" class="usa-link" aria-describedby="R83"><em>83</em></a>). Our discovery of the interaction between FgRid and FgRpa1 provides insights into the Rid-mediated RIP mechanism. This interaction suggests that RPA may play a protective role by shielding T or U bases generated by Rid in ssDNA from the actions of thymine-DNA glycosylase and UNG, thereby facilitating the mutagenic process. The recruitment of FgQde3 to tandem repeat regions relies on the catalytic activity of FgRid but is independent of RIP-induced mutations. This observation suggests that FgQde3 may recognize specific intermediates generated by Rid, thereby initiating DNA unwinding in these regions.</p>
<p>Besides their role in RIP, Rid orthologs have been reported to be essential for sexual development in many filamentous ascomycetes (<a href="#R84" class="usa-link" aria-describedby="R84"><em>84</em></a>–<a href="#R87" class="usa-link" aria-describedby="R87"><em>87</em></a>). Similarly, Sms2 orthologs are critical for sexual development in various filamentous ascomycetes (<a href="#R49" class="usa-link" aria-describedby="R49"><em>49</em></a>, <a href="#R82" class="usa-link" aria-describedby="R82"><em>82</em></a>, <a href="#R88" class="usa-link" aria-describedby="R88"><em>88</em></a>, <a href="#R89" class="usa-link" aria-describedby="R89"><em>89</em></a>). However, the mechanisms by which these proteins regulate sexual development remain unclear. The catalytic activity of PaRid in <em>Podospora anserina</em> is essential for sexual development (<a href="#R87" class="usa-link" aria-describedby="R87"><em>87</em></a>), consistent with our finding that FgRid catalytic activity is crucial for TRISS. Our observation that FgRid functions in TRISS suggests that Rid orthologs may regulate sexual development through RNAi mechanisms. Furthermore, the discovery of RPA interacting with both Sms2 and Rid offers valuable insights into their roles in fungal sexual development. Given that RPA is a multifunctional protein involved in DNA replication, repair, and other DNA-related activities (<a href="#R80" class="usa-link" aria-describedby="R80"><em>80</em></a>), these interactions suggest that Sms2 and Rid may have broader roles in coordinating crucial DNA processes during sexual development, potentially linking RNAi pathways with DNA maintenance and modification.</p></section><section id="sec20"><h2 class="pmc_sec_title">MATERIALS AND METHODS</h2>
<section id="sec21"><h3 class="pmc_sec_title">Strains and culture conditions</h3>
<p>The wild-type strain PH-1 of <em>F. graminearum</em> (<a href="#R66" class="usa-link" aria-describedby="R66"><em>66</em></a>), along with its derived mutants and transformants (data S1), were routinely cultured on potato dextrose agar (PDA) plates at 25°C. For transformant selection, we added hygromycin B (H005, MDbio, China) and geneticin (Sigma-Aldrich, St. Louis, MO) to Top agar medium (0.3% yeast extract, 0.3% casamino acids, 20% sucrose, and 1.5% agar) to final concentrations of 300 and 200 μg/ml, respectively. We assayed growth rate and colony morphology on PDA plates after 3 days. We harvested conidia from 3-day-old liquid carboxymethylcellulose medium cultures inoculated with mycelial plugs at 25°C using a rotary shaker set at 175 rpm. We obtained vegetative hyphae for DNA and RNA isolation and protoplast preparation from 12- to 24-hour liquid yeast extract peptone dextrose cultures. For selfing in sexual reproduction, we pressed 5-day-old aerial hyphae grown on carrot agar plates down with 500 μl of sterile 0.1% Tween-20 and subsequently cultured them under black light at 25°C. For outcrossing, we fertilized the female strains grown on carrot agar plates with 1 ml of conidial suspension (1 × 10<sup>6</sup> conidia/ml) derived from the male strains. We observed ascogenous hyphae, asci, and ascospores with an Olympus BX-51 microscope in squash mounts of perithecia 7 dpf.</p></section><section id="sec22"><h3 class="pmc_sec_title">Generation of gene deletion, complementation, and tandem repeat transformants</h3>
<p>We used polyethylene glycol–mediated protoplast transformation to generate various transformants of <em>F. graminearum</em>. For gene deletion, we used the split-marker approach (<a href="#R90" class="usa-link" aria-describedby="R90"><em>90</em></a>, <a href="#R91" class="usa-link" aria-describedby="R91"><em>91</em></a>). We amplified fragments of approximately 1.0 kb upstream and downstream of the target genes and connected them to the N- and C-terminal regions of either a hygromycin-resistance marker or a recyclable marker module (<a href="#R92" class="usa-link" aria-describedby="R92"><em>92</em></a>) that confers resistance to hygromycin and sensitivity to the nucleoside analog 5-fluoro-2′-deoxyuridine (floxuridine) through overlapping PCR. After transforming PH-1 protoplasts, we screened hygromycin-resistant transformants and confirmed them via PCR assays. For gene complementation, we amplified the full-length genes, along with their native promoter, and cloned them into a pFL2 vector. We confirmed the complementation construct by DNA sequencing and then transformed it into the protoplasts of the corresponding deletion mutants. We screened transformants containing the complementation constructs with geneticin and confirmed them using PCR assays.</p>
<p>To perform site-directed mutagenesis of <em>FgSMS2</em> and <em>FgRID</em>, we first generated gene deletion mutants using the recyclable marker module. We then created allelic fragments with the desired mutations through overlapping PCR. We subsequently transformed the resulting fragments into protoplasts of the corresponding deletion mutants. We screened transformants for resistance to floxuridine (25 μg/ml; HY-B0097, MCE, USA) and confirmed the presence of the desired mutations by DNA sequencing. In addition, we used qPCR to ensure that no unintended integration of allelic fragments occurred in the transformants.</p>
<p>To generate transformants containing tandem transgenic repeats, we connected the target gene fragment (either full-length or partial) to a hygromycin (<em>hyg</em>)– or G418 (<em>neo</em>)–resistance marker through overlapping PCR. We then transformed the resulting linear fragment into protoplasts of PH-1 or specific gene deletion mutants. We screened transformants with hygromycin or G418 resistance and assayed tandem repeats by PCR with specific primer pairs. We used the split-marker approach to generate direct or reverse tandem repeats of <em>CDC2A</em> at either the native locus or the FG1G36140 locus in PH-1. We screened transformants for resistance to hygromycin or geneticin and confirmed them using PCR assays. To generate transformants ectopically expressing antisense transcripts of <em>CDC2A</em> at the FG1G36140 locus, we amplified fragments of approximately 1.0 kb upstream and downstream of the FG1G36140 gene. We then connected these fragments to the reverse complementary fragment of the <em>CDC2A</em> coding region under the control of the PR27 promoter through overlapping PCR and transformed them into protoplasts of the FG1G36140 deletion mutant, where the FG1G36140 gene had been replaced by the recyclable marker module. We screened transformants for resistance to floxuridine. To generate transformants containing tandemly repeated <em>FgAMA1</em> with a nongene linker, we first transformed a linear fragment containing the <em>FgAMA1</em> gene connected with the recyclable marker module into PH-1 protoplasts. This allowed us to obtain transformants containing tandemly repeated <em>FgAMA1</em> with the recyclable marker module as the linker. We then amplified fragments corresponding to the 3′-end of the upstream repeat and the 5′-end of the downstream repeat and connected these fragments to a nontranscriptional region from the pKNT plasmid. We used the resulting linear fragment to replace the recyclable marker module in the previous transformants. Data S1 list all the strains generated in this study, while data S2 provide a comprehensive list of the primers used.</p></section><section id="sec23"><h3 class="pmc_sec_title">Isolation of ascospore progeny and characterization of RIP mutations</h3>
<p>To isolate ascospore progeny from the <em>Fgama1</em> uncomplemented strain and the TR-<em>FgAMA1</em> and TR-<em>FgAMA1 Fgrid</em> transformants, we collected ascospore cirrhi from 10-dpf perithecia under a dissecting microscope. We resuspended the cirrhi in 1 ml of sterile distilled water in a 1.5-ml centrifuge tube. We then spread the ascospore suspensions on 2% water agar and isolated individual oval ascospores, which were subsequently transferred onto PDA plates. To identify RIP mutations in the ascospore progeny, we extracted genomic DNA from each ascospore progeny using cetyltrimethylammonium bromide (CTAB) solution. We amplified the <em>FgAMA1</em> gene from the isolated DNA and sequenced it using Sanger sequencing. For the <em>Fgama1</em> uncomplemented strain, we could not distinguish which repeats of <em>FgAMA1</em> were amplified. For the TR-<em>FgAMA1</em> and TR-<em>FgAMA1 Fgrid</em> transformants, we used specific primer pairs (data S2) to distinguish the upstream and downstream repeats of <em>FgAMA1</em>. We visualized the Sanger sequencing traces using SnapGene Viewer 4.3 (<a href="http://www.snapgene.com/snapgene-viewer/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.snapgene.com/snapgene-viewer/</a>) and manually checked the quality of the sequencing. We aligned the assembled high-quality sequences for each repeat to the wild-type sequence of the <em>FgAMA1</em> gene. We used these aligned sequences to detect RIP mutants in each ascospore progeny and analyzed their consequences on protein sequences using the Biopython package (<a href="https://biopython.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://biopython.org/</a>).</p></section><section id="sec24"><h3 class="pmc_sec_title">Analysis of repeat copy number and gene expression</h3>
<p>We extracted genomic DNA from 24-hour hyphae using a CTAB solution. We isolated total RNA from 24-hour hyphae or 7-dpf perithecia using the TRIzol reagent kit (Invitrogen, USA). Following this, we synthesized cDNA using the First cDNA Synthesis Kit (Thermo Fisher Scientific) according to the manufacturer’s instructions. We then amplified the resulting cDNA products and genomic DNA by real-time PCR using the primers listed in data S2. We calculated repeat copy numbers and relative expression levels using the 2<sup>−ΔΔCT</sup> method, with the <em>GzUBH</em> gene serving as the internal control.</p></section><section id="sec25"><h3 class="pmc_sec_title">Western blot analysis</h3>
<p>We ground samples from 24-hour vegetative hyphae or 7-dpf perithecia into a fine powder in liquid nitrogen and homogenized them in protein extraction buffer. We then centrifuged the samples for 15 min at 12,000 rpm at 4°C, heating the supernatants for 5 min in boiling water. Next, we separated the supernatants on 10% SDS–polyacrylamide gel electrophoresis (SDS-PAGE) gels and transferred the proteins to nitrocellulose membranes. We performed Western blot assays to analyze the samples using specific antibodies, including anti-GFP (Abcam, ab290), anti-Tub1 (<a href="#R93" class="usa-link" aria-describedby="R93"><em>93</em></a>), and anti–glyceraldehyde-3-phosphate dehydrogenase (GAPDH) (Sangon Biotech, D1100160200). We used detection with the anti-GAPDH antibody as the loading control.</p></section><section id="sec26"><h3 class="pmc_sec_title">sRNA-seq analysis</h3>
<p>For sRNA-seq analysis, we isolated total RNA from 7-dpf perithecia using the TRIzol Reagent kit (Invitrogen, USA) and quantified it with the Nanodrop 2000c spectrophotometer (Thermo Fisher Scientific, USA). We assessed RNA quality and integrity using the Experion RNA Standard Sense kit (Bio-Rad, USA), ensuring that all RNA samples had an RNA quality index ≥8. We prepared the sRNA library using the TruSeq Small RNA Sample Prep kit (Illumina, USA) according to the manufacturer’s instructions. Specifically, we ligated 1 μg of total RNA from each sample to specific adapters, reverse transcribed it, and amplified it to obtain a population of double-stranded cDNAs. During PCR amplification, we used barcode-tagged primer pairs to uniquely tag each sample. We then size-selected the resulting cDNAs on a 6% PAGE gel, excised the gel band corresponding to the target fragments, ethanol-precipitated them, and eluted them in ultrapure water. We evaluated the libraries for quality using the Agilent 2100 BioAnalyzer (Agilent, USA). Last, we carried out sequencing reactions using the MiSeq instrument (Illumina, USA) in 50-bp single-end read mode at Novogene Bioinformatics Institute (Tianjin, China), obtaining at least 10 million reads for each library. The information regarding the sRNA-seq data generated in this study has been listed in table S1.</p>
<p>We obtained the reference genome and annotation files of <em>F. graminearum</em> PH-1 from FgBase (<a href="http://fgbase.wheatscab.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://fgbase.wheatscab.com/</a>) (<a href="#R94" class="usa-link" aria-describedby="R94"><em>94</em></a>). We aligned sRNA-seq reads to the tandem repeats and the PH-1 genome using HISAT2 v2.2.1 (<a href="#R95" class="usa-link" aria-describedby="R95"><em>95</em></a>). We analyzed detailed information on aligned reads using bioawk (<a href="https://github.com/lh3/bioawk" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/lh3/bioawk</a>) and samtools (<a href="http://htslib.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://htslib.org/</a>). The length distribution and base preference were visualized using the matplotlib (<a href="https://matplotlib.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://matplotlib.org/</a>) and seaborn (<a href="https://seaborn.pydata.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://seaborn.pydata.org/</a>) packages. Using bioawk, we filtered reads of 20 to 27 nt with a 5′ preferred U. We obtained BAM files of aligned reads using samtools and converted them to BigWig coverage files with deeptools2 (<a href="https://deeptools.readthedocs.io/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://deeptools.readthedocs.io/</a>). We parsed the BigWig coverage files using the pyBigWig Python package (<a href="https://github.com/deeptools/pyBigWig" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/deeptools/pyBigWig</a>) and plotted the sRNA coverage using the matplotlib package.</p></section><section id="sec27"><h3 class="pmc_sec_title">DamID-qPCR</h3>
<p>We isolated genomic DNA from 24-hour hyphae and 7-dpf perithecia using a CTAB solution. For each sample, we digested 1 μg of genomic DNA from the strain expressing a Dam or Dam fusion with 1 μl of MboI (Thermo Fisher Scientific, ER0811) for 16 hours in a total reaction volume of 20 μl. We then analyzed the digested samples by qPCR. To normalize the DNA content, we amplified a DNA fragment without GATC sites. We also amplified DNA fragments containing GATC sites to determine whether they were cut by MboI. We calculated the enrichment of the Dam fusion protein on target DNA using the 2<sup>−ΔΔCT</sup> method by comparing the cycle threshold (Ct) values of each sample with that of the control (Dam only protein). As a negative control, we treated samples with Sau3AI as described above.</p></section><section id="sec28"><h3 class="pmc_sec_title">Polysome profiling analysis</h3>
<p>We froze samples from 7-dpf perithecia in liquid nitrogen immediately after collection. We ground the tissue samples into a fine powder in liquid nitrogen and added an equal volume of the tissue powder from each sample to an equal volume of lysis buffer, which contained 100 mM KCl, 20 mM Hepes-KOH (pH 7.5), 10 mM magnesium acetate (pH 7), 15 mM 2-mercaptoethanol (BME), cycloheximide (100 μg/ml), and a protease inhibitor cocktail including phenylmethylsulfonyl fluoride (1:100), leupeptin (1:1000), pepstatin (1:1000), and dithiothreitol (1:1000). We centrifuged the lysates at 15,000 rpm for 10 min. We measured the absorbance at 260 nm of the supernatant using a NanoDrop Microvolume Spectrophotometer and obtained 500 μl of lysate with the same optical density (20 OD<sub>260nm</sub>) for each sample. From each sample, we used 50 μl of lysate as input, while the remaining lysate was loaded onto a 10 to 50% sucrose gradient buffer containing 20 mM Hepes (pH 7.6), 0.1 M KCl, 5 mM MgCl<sub>2</sub>, cycloheximide (10 μg/ml), 1× protease inhibitor cocktail, and SUPERase•In (10 U/ml).</p>
<p>We centrifuged the sucrose gradients at 35,000 rpm for 2 hours at 4°C using a SW41Ti rotor in a Beckman Coulter Optima L-80 ultracentrifuge. After ultracentrifugation, polysomal RNAs distributed along the sucrose gradient according to their different sedimentation coefficients and were separated using a gradient fractionator system (BioComp Instruments). We collected fractions containing monosomes to the heaviest polysomes and subjected them to RNA isolation using TRIzol followed by qPCR. We also isolated RNA from input lysates using TRIzol followed by qPCR. We normalized the polysome distributions of <em>CDC2A</em> mRNA levels against their own input mRNA levels and calculated <em>CDC2A</em> expression in each polysomal fraction as a percentage of its expression in total RNA.</p></section><section id="sec29"><h3 class="pmc_sec_title">Coimmunoprecipitation</h3>
<p>To assess the interaction of FgRid and FgSms2 with FgRpa1 using a Co-IP assay, we first created the TR-<em>FgAMA1 Fgrid</em> and TR-<em>FgAMA1 Fgsms2</em> strains. This was accomplished by deleting the <em>FgRID</em> and <em>FgSMS2</em> genes in the TR-<em>FgAMA1</em> strain, respectively, using a recyclable marker module. We PCR amplified approximately 1 kb of the upstream and downstream regions of <em>FgRID</em> and <em>FgSMS2</em> and linked them to the <em>FgRID-</em>3×FLAG and HA-<em>FgSMS2</em> fusion constructs, respectively, under the control of the RP30 promoter through overlapping PCR. We then transformed these PCR products into the protoplasts of the TR-<em>FgAMA1 Fgrid</em> and TR-<em>FgAMA1 Fgsms2</em> strains. We screened transformants of <em>FgRID</em>-3×FLAG TR-<em>FgAMA1</em> and HA-<em>FgSMS2</em> TR-<em>FgAMA1</em> using floxuridine (25 μg/ml; HY-B0097, MCE, USA). Using a similar method, we sequentially generated the HA-<em>FgSMS2 FgRID</em>-3×FLAG TR-<em>FgAMA1</em> and HA-<em>FgSMS2 FgRID</em>-3×FLAG TR-<em>FgAMA1 Fgqde3</em> strains in the <em>FgRID</em>-3×FLAG TR-<em>FgAMA1</em> background. In each strain, we created the <em>FgRRA1</em>-GFP construct by inserting the GFP sequence before the stop codon of the <em>FgRPA1</em> gene using a hygromycin resistance marker.</p>
<p>To investigate the interaction between FgQde1 and FgRpa1 in the Co-IP assay, we constructed the <em>FgRPA1</em>–3×FLAG fusion construct under the control of its native promoter and the GFP-<em>FgQDE1</em> fusion construct under the control of the RP27 promoter by inserting their respective sequences into the pKNTF and pDL2 vectors. We then cotransformed these constructs into both the wild type and the <em>Fgqde3</em> mutant strains, selecting transformants with hygromycin and G418 resistance.</p>
<p>For each strain, we extracted total proteins from 24-hour hyphae. We incubated the supernatant containing the target protein with anti-GFP affinity beads (Smart-lifesciences, SA070001) and subsequently eluted the proteins. We performed Western blots on both the total proteins and the proteins eluted from the anti-GFP affinity beads, using anti-FLAG (<a href="https://www.ncbi.nlm.nih.gov/nuccore/F31651" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">F31651</a>, Sigma-Aldrich), anti-GFP (Abcam, ab290), anti-HA (AE008, Abclonal), and anti-GAPDH antibodies for detection.</p></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank G. Wang, M. Xu, P. Xiang, and Z. Tang for invaluable laboratory assistance. We also thank laboratory technicians Q. Zhang and X. Zhou from the State Key Laboratory for Crop Stress Resistance and High-Efficiency Production for assistance with the MS/MS analysis.</p>
<section id="sec30"><p><strong>Funding:</strong> This work was supported by grants from the National Key Research and Development Program of China (2022YFD1400100) and the National Natural Science Foundation of China (no. 32170200) to H.L.</p></section><section id="sec31"><p><strong>Author contributions:</strong> Conceptualization: H.L. and M.H. Methodology: H.L., Q.W., Z.Z., and C.J. Investigation: M.H., Y.N., J.Y., M.W., H.Z., Y.D., and S.C. Resources: H.L., C.J., and Z.Z. Visualization: M.H. and H.L. Supervision: H.L. and Z.Z. Validation: H.L., Z.Z., and M.H. Writing—original draft: H.L. and M.H. Writing—review and editing: H.L., M.H., Q.W., and C.J.</p></section><section id="sec32"><p><strong>Competing interests:</strong> The other authors declare that they have no competing interests.</p></section><section id="sec33"><p><strong>Data and materials availability:</strong> Omics data generated in this study are accessible under the NCBI BioProject accession: PRJNA1168301. All data needed to evaluate the conclusions in the paper are present in the paper and/or the Supplementary Materials.</p></section></section><section id="sec34"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section id="sec35"><h3 class="pmc_sec_title">The PDF file includes:</h3>
<section class="sm xbox font-sm" id="supplementary-material1"><div class="caption p">
<p>Figs. S1 to S9</p>
<p>Table S1</p>
<p>Legends for datasets S1 and S2</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366691/bin/sciadv.adu7606_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adu7606_sm.pdf</a><sup> (1.1MB, pdf) </sup>
</div></div></section></section><section id="sec36"><h3 class="pmc_sec_title">Other Supplementary Material for this manuscript includes the following:</h3>
<section class="sm xbox font-sm" id="supplementary-material2"><div class="caption p"><p>Datasets S1 and S2</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366691/bin/sciadv.adu7606_datasets_s1_and_s2.zip" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adu7606_datasets_s1_and_s2.zip</a><sup> (32.2KB, zip) </sup>
</div></div></section></section></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">REFERENCES AND NOTES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">1.</span><cite>Kazazian H. H. Jr., 
Mobile elements: Drivers of genome evolution. Science
303, 
1626–1632 (2004).
</cite> [<a href="https://doi.org/10.1126/science.1089670" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15016989/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Mobile%20elements:%20Drivers%20of%20genome%20evolution&amp;volume=303&amp;publication_year=2004&amp;pages=1626-1632&amp;pmid=15016989&amp;doi=10.1126/science.1089670&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">2.</span><cite>Kidwell M. G., Lisch D., 
Transposable elements as sources of variation in animals and plants. Proc. Natl. Acad. Sci. U.S.A.
94, 
7704–7711 (1997).
</cite> [<a href="https://doi.org/10.1073/pnas.94.15.7704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC33680/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9223252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Transposable%20elements%20as%20sources%20of%20variation%20in%20animals%20and%20plants&amp;volume=94&amp;publication_year=1997&amp;pages=7704-7711&amp;pmid=9223252&amp;doi=10.1073/pnas.94.15.7704&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">3.</span><cite>Gladyshev E., 
Repeat-induced point mutation and other genome defense mechanisms in fungi. Microbiol. Spectr.
5, 10.1128/microbiolspec.FUNK-0042-2017 (2017).</cite> [<a href="https://doi.org/10.1128/microbiolspec.funk-0042-2017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5607778/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28721856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol.%20Spectr.&amp;title=Repeat-induced%20point%20mutation%20and%20other%20genome%20defense%20mechanisms%20in%20fungi&amp;volume=5&amp;publication_year=2017&amp;pmid=28721856&amp;doi=10.1128/microbiolspec.funk-0042-2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">4.</span><cite>Cogoni C., 
Homology-dependent gene silencing mechanisms in fungi. Annu. Rev. Microbiol.
55, 
381–406 (2001).
</cite> [<a href="https://doi.org/10.1146/annurev.micro.55.1.381" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11544361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Microbiol.&amp;title=Homology-dependent%20gene%20silencing%20mechanisms%20in%20fungi&amp;volume=55&amp;publication_year=2001&amp;pages=381-406&amp;pmid=11544361&amp;doi=10.1146/annurev.micro.55.1.381&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R5">
<span class="label">5.</span><cite>Shabalina S. A., Koonin E. V., 
Origins and evolution of eukaryotic RNA interference. Trends Ecol. Evol.
23, 
578–587 (2008).
</cite> [<a href="https://doi.org/10.1016/j.tree.2008.06.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2695246/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18715673/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Ecol.%20Evol.&amp;title=Origins%20and%20evolution%20of%20eukaryotic%20RNA%20interference&amp;volume=23&amp;publication_year=2008&amp;pages=578-587&amp;pmid=18715673&amp;doi=10.1016/j.tree.2008.06.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">6.</span><cite>Buchon N., Vaury C., 
RNAi: A defensive RNA-silencing against viruses and transposable elements. Heredity
96, 
195–202 (2006).
</cite> [<a href="https://doi.org/10.1038/sj.hdy.6800789" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16369574/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=RNAi:%20A%20defensive%20RNA-silencing%20against%20viruses%20and%20transposable%20elements&amp;volume=96&amp;publication_year=2006&amp;pages=195-202&amp;pmid=16369574&amp;doi=10.1038/sj.hdy.6800789&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R7">
<span class="label">7.</span><cite>Girard A., Hannon G. J., 
Conserved themes in small-RNA-mediated transposon control. Trends Cell Biol.
18, 
136–148 (2008).
</cite> [<a href="https://doi.org/10.1016/j.tcb.2008.01.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2995447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18282709/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Cell%20Biol.&amp;title=Conserved%20themes%20in%20small-RNA-mediated%20transposon%20control&amp;volume=18&amp;publication_year=2008&amp;pages=136-148&amp;pmid=18282709&amp;doi=10.1016/j.tcb.2008.01.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">8.</span><cite>Faugeron G., 
Diversity of homology-dependent gene silencing strategies in fungi. Curr. Opin. Microbiol.
3, 
144–148 (2000).
</cite> [<a href="https://doi.org/10.1016/s1369-5274(00)00066-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10744998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr.%20Opin.%20Microbiol.&amp;title=Diversity%20of%20homology-dependent%20gene%20silencing%20strategies%20in%20fungi&amp;volume=3&amp;publication_year=2000&amp;pages=144-148&amp;pmid=10744998&amp;doi=10.1016/s1369-5274(00)00066-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">9.</span><cite>Matzke M. A., Mosher R. A., 
RNA-directed DNA methylation: An epigenetic pathway of increasing complexity. Nat. Rev. Genet.
15, 
394–408 (2014).
</cite> [<a href="https://doi.org/10.1038/nrg3683" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24805120/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&amp;title=RNA-directed%20DNA%20methylation:%20An%20epigenetic%20pathway%20of%20increasing%20complexity&amp;volume=15&amp;publication_year=2014&amp;pages=394-408&amp;pmid=24805120&amp;doi=10.1038/nrg3683&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">10.</span><cite>Martienssen R., Moazed D., 
RNAi and heterochromatin assembly. Cold Spring Harb. Perspect. Biol.
7, 
a019323 (2015).
</cite> [<a href="https://doi.org/10.1101/cshperspect.a019323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4526745/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26238358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb.%20Perspect.%20Biol.&amp;title=RNAi%20and%20heterochromatin%20assembly&amp;volume=7&amp;publication_year=2015&amp;pages=a019323&amp;pmid=26238358&amp;doi=10.1101/cshperspect.a019323&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R11">
<span class="label">11.</span><cite>Asanuma T., Inagaki S., Kakutani T., Aburatani H., Murakami Y., 
Tandemly repeated genes promote RNAi-mediated heterochromatin formation via an antisilencing factor, Epe1, in fission yeast. Genes Dev.
36, 
1145–1159 (2022).
</cite> [<a href="https://doi.org/10.1101/gad.350129.122" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9851402/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36617881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=Tandemly%20repeated%20genes%20promote%20RNAi-mediated%20heterochromatin%20formation%20via%20an%20antisilencing%20factor,%20Epe1,%20in%20fission%20yeast&amp;volume=36&amp;publication_year=2022&amp;pages=1145-1159&amp;pmid=36617881&amp;doi=10.1101/gad.350129.122&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R12">
<span class="label">12.</span><cite>Holoch D., Moazed D., 
RNA-mediated epigenetic regulation of gene expression. Nat. Rev. Genet.
16, 
71–84 (2015).
</cite> [<a href="https://doi.org/10.1038/nrg3863" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4376354/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25554358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&amp;title=RNA-mediated%20epigenetic%20regulation%20of%20gene%20expression&amp;volume=16&amp;publication_year=2015&amp;pages=71-84&amp;pmid=25554358&amp;doi=10.1038/nrg3863&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">13.</span><cite>Baulcombe D. C., 
Molecular biology. Amplified silencing. Science
315, 
199–200 (2007).
</cite> [<a href="https://doi.org/10.1126/science.1138030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17218517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Molecular%20biology.%20Amplified%20silencing&amp;volume=315&amp;publication_year=2007&amp;pages=199-200&amp;pmid=17218517&amp;doi=10.1126/science.1138030&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R14">
<span class="label">14.</span><cite>Halic M., Moazed D., 
Dicer-independent primal RNAs trigger RNAi and heterochromatin formation. Cell
140, 
504–516 (2010).
</cite> [<a href="https://doi.org/10.1016/j.cell.2010.01.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3020400/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20178743/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Dicer-independent%20primal%20RNAs%20trigger%20RNAi%20and%20heterochromatin%20formation&amp;volume=140&amp;publication_year=2010&amp;pages=504-516&amp;pmid=20178743&amp;doi=10.1016/j.cell.2010.01.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R15">
<span class="label">15.</span><cite>Kelly W. G., 
Standing guard: Perinuclear localization of an RNA-dependent RNA polymerase. Proc. Natl. Acad. Sci. U.S.A.
103, 
2007–2008 (2006).
</cite> [<a href="https://doi.org/10.1073/pnas.0600085103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1413763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16461886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Standing%20guard:%20Perinuclear%20localization%20of%20an%20RNA-dependent%20RNA%20polymerase&amp;volume=103&amp;publication_year=2006&amp;pages=2007-2008&amp;pmid=16461886&amp;doi=10.1073/pnas.0600085103&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">16.</span><cite>Chow H. T., Mosher R. A., 
Small RNA-mediated DNA methylation during plant reproduction. Plant Cell
35, 
1787–1800 (2023).
</cite> [<a href="https://doi.org/10.1093/plcell/koad010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10226566/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36651080/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Small%20RNA-mediated%20DNA%20methylation%20during%20plant%20reproduction&amp;volume=35&amp;publication_year=2023&amp;pages=1787-1800&amp;pmid=36651080&amp;doi=10.1093/plcell/koad010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">17.</span><cite>Onishi R., Yamanaka S., Siomi M. C., 
piRNA- and siRNA-mediated transcriptional repression in Drosophila, mice, and yeast: New insights and biodiversity. EMBO Rep.
22, 
e53062 (2021).
</cite> [<a href="https://doi.org/10.15252/embr.202153062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8490990/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34347367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=EMBO%20Rep.&amp;title=piRNA-%20and%20siRNA-mediated%20transcriptional%20repression%20in%20Drosophila,%20mice,%20and%20yeast:%20New%20insights%20and%20biodiversity&amp;volume=22&amp;publication_year=2021&amp;pages=e53062&amp;pmid=34347367&amp;doi=10.15252/embr.202153062&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">18.</span><cite>Wang X., Hsueh Y.-P., Li W., Floyd A., Skalsky R., Heitman J., 
Sex-induced silencing defends the genome of <em>Cryptococcus neoformans</em> via RNAi. Genes Dev.
24, 
2566–2582 (2010).
</cite> [<a href="https://doi.org/10.1101/gad.1970910" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2975932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21078820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=Sex-induced%20silencing%20defends%20the%20genome%20of%20Cryptococcus%20neoformans%20via%20RNAi&amp;volume=24&amp;publication_year=2010&amp;pages=2566-2582&amp;pmid=21078820&amp;doi=10.1101/gad.1970910&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">19.</span><cite>Aramayo R., Selker E. U., 
Neurospora crassa, a model system for epigenetics research. Cold Spring Harb. Perspect. Biol.
5, 
a017921 (2013).
</cite> [<a href="https://doi.org/10.1101/cshperspect.a017921" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3783048/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24086046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb.%20Perspect.%20Biol.&amp;title=Neurospora%20crassa,%20a%20model%20system%20for%20epigenetics%20research&amp;volume=5&amp;publication_year=2013&amp;pages=a017921&amp;pmid=24086046&amp;doi=10.1101/cshperspect.a017921&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R20">
<span class="label">20.</span><cite>Catalanotto C., Nolan T., Cogoni C., 
Homology effects in <em>Neurospora crassa</em>. FEMS Microbiol. Lett.
254, 
182–189 (2006).
</cite> [<a href="https://doi.org/10.1111/j.1574-6968.2005.00037.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16445744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Lett.&amp;title=Homology%20effects%20in%20Neurospora%20crassa&amp;volume=254&amp;publication_year=2006&amp;pages=182-189&amp;pmid=16445744&amp;doi=10.1111/j.1574-6968.2005.00037.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R21">
<span class="label">21.</span><cite>Torres-Martinez S., Ruiz- Vázquez R. M., 
The RNAi universe in fungi: A varied landscape of small RNAs and biological functions. Annu. Rev. Microbiol.
71, 
371–391 (2017).
</cite> [<a href="https://doi.org/10.1146/annurev-micro-090816-093352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28657888/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Microbiol.&amp;title=The%20RNAi%20universe%20in%20fungi:%20A%20varied%20landscape%20of%20small%20RNAs%20and%20biological%20functions&amp;volume=71&amp;publication_year=2017&amp;pages=371-391&amp;pmid=28657888&amp;doi=10.1146/annurev-micro-090816-093352&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R22">
<span class="label">22.</span><cite>Yang Q., Ye Q. A., Liu Y., 
Mechanism of siRNA production from repetitive DNA. Genes Dev.
29, 
526–537 (2015).
</cite> [<a href="https://doi.org/10.1101/gad.255828.114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4358405/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25691092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=Mechanism%20of%20siRNA%20production%20from%20repetitive%20DNA&amp;volume=29&amp;publication_year=2015&amp;pages=526-537&amp;pmid=25691092&amp;doi=10.1101/gad.255828.114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">23.</span><cite>Zhang Z., Chang S.-S., Zhang Z., Xue Z., Zhang H., Li S., Liu Y., 
Homologous recombination as a mechanism to recognize repetitive DNA sequences in an RNAi pathway. Genes Dev.
27, 
145–150 (2013).
</cite> [<a href="https://doi.org/10.1101/gad.209494.112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3566306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23322299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=Homologous%20recombination%20as%20a%20mechanism%20to%20recognize%20repetitive%20DNA%20sequences%20in%20an%20RNAi%20pathway&amp;volume=27&amp;publication_year=2013&amp;pages=145-150&amp;pmid=23322299&amp;doi=10.1101/gad.209494.112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">24.</span><cite>Lee H.-C., Aalto A. P., Yang Q., Chang S.-S., Huang G., Fisher D., Cha J., Poranen M. M., Bamford D. H., Liu Y., 
The DNA/RNA-dependent RNA polymerase QDE-1 generates aberrant RNA and dsRNA for RNAi in a process requiring replication protein A and a DNA helicase. PLOS Biol.
8, 
e1000496 (2010).
</cite> [<a href="https://doi.org/10.1371/journal.pbio.1000496" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2950127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20957187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Biol.&amp;title=The%20DNA/RNA-dependent%20RNA%20polymerase%20QDE-1%20generates%20aberrant%20RNA%20and%20dsRNA%20for%20RNAi%20in%20a%20process%20requiring%20replication%20protein%20A%20and%20a%20DNA%20helicase&amp;volume=8&amp;publication_year=2010&amp;pages=e1000496&amp;pmid=20957187&amp;doi=10.1371/journal.pbio.1000496&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">25.</span><cite>Hammond T. M., Spollen W. G., Decker L. M., Blake S. M., Springer G. K., Shiu P. K. T., 
Identification of small RNAs associated with meiotic silencing by unpaired DNA. Genetics
194, 
279–284 (2013).
</cite> [<a href="https://doi.org/10.1534/genetics.112.149138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3632476/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23502674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Identification%20of%20small%20RNAs%20associated%20with%20meiotic%20silencing%20by%20unpaired%20DNA&amp;volume=194&amp;publication_year=2013&amp;pages=279-284&amp;pmid=23502674&amp;doi=10.1534/genetics.112.149138&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R26">
<span class="label">26.</span><cite>Wang Y., Smith K. M., Taylor J. W., Freitag M., Stajich J. E., 
Endogenous small RNA mediates meiotic silencing of a novel DNA transposon. G3
5, 
1949–1960 (2015).
</cite> [<a href="https://doi.org/10.1534/g3.115.017921" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4592977/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26109355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3&amp;title=Endogenous%20small%20RNA%20mediates%20meiotic%20silencing%20of%20a%20novel%20DNA%20transposon&amp;volume=5&amp;publication_year=2015&amp;pages=1949-1960&amp;pmid=26109355&amp;doi=10.1534/g3.115.017921&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R27">
<span class="label">27.</span><cite>Hammond T. M., 
Sixteen years of meiotic silencing by unpaired DNA. Adv. Genet.
97, 
1–42 (2017).
</cite> [<a href="https://doi.org/10.1016/bs.adgen.2016.11.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28838355/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv.%20Genet.&amp;title=Sixteen%20years%20of%20meiotic%20silencing%20by%20unpaired%20DNA&amp;volume=97&amp;publication_year=2017&amp;pages=1-42&amp;pmid=28838355&amp;doi=10.1016/bs.adgen.2016.11.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">28.</span><cite>J. K. Hane, A. H. Williams, A. P. Taranto, P. S. Solomon, R. P. Oliver, Repeat-induced point mutation: A fungal-specific, endogenous mutagenesis process in <em>Genetic Transformation Systems in Fungi Volume 2</em>, (2015) pp. 55–68.</cite>
</li>
<li id="R29">
<span class="label">29.</span><cite>Selker E. U., Cambareri E. B., Jensen B. C., Haack K. R., 
Rearrangement of duplicated DNA in specialized cells of <em>Neurospora</em>. Cell
51, 
741–752 (1987).
</cite> [<a href="https://doi.org/10.1016/0092-8674(87)90097-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2960455/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Rearrangement%20of%20duplicated%20DNA%20in%20specialized%20cells%20of%20Neurospora&amp;volume=51&amp;publication_year=1987&amp;pages=741-752&amp;pmid=2960455&amp;doi=10.1016/0092-8674(87)90097-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">30.</span><cite>van Wyk S., Wingfield B. D., De Vos L., van der Merwe N. A., Steenkamp E. T., 
Genome-wide analyses of repeat-induced point mutations in the ascomycota. Front. Microbiol.
11, 
622368 (2021).
</cite> [<a href="https://doi.org/10.3389/fmicb.2020.622368" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7882544/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33597932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Genome-wide%20analyses%20of%20repeat-induced%20point%20mutations%20in%20the%20ascomycota&amp;volume=11&amp;publication_year=2021&amp;pages=622368&amp;pmid=33597932&amp;doi=10.3389/fmicb.2020.622368&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">31.</span><cite>Watters M. K., Randall T. A., Margolin B. S., Selker E. U., Stadler D. R., 
Action of repeat-induced point mutation on both strands of a duplex and on tandem duplications of various sizes in <em>Neurospora</em>. Genetics
153, 
705–714 (1999).
</cite> [<a href="https://doi.org/10.1093/genetics/153.2.705" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1460768/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10511550/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Action%20of%20repeat-induced%20point%20mutation%20on%20both%20strands%20of%20a%20duplex%20and%20on%20tandem%20duplications%20of%20various%20sizes%20in%20Neurospora&amp;volume=153&amp;publication_year=1999&amp;pages=705-714&amp;pmid=10511550&amp;doi=10.1093/genetics/153.2.705&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R32">
<span class="label">32.</span><cite>Freitag M., Williams R. L., Kothe G. O., Selker E. U., 
A cytosine methyltransferase homologue is essential for repeat-induced point mutation in <em>Neurospora crassa</em>. Proc. Natl. Acad. Sci. U.S.A.
99, 
8802–8807 (2002).
</cite> [<a href="https://doi.org/10.1073/pnas.132212899" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC124379/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12072568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=A%20cytosine%20methyltransferase%20homologue%20is%20essential%20for%20repeat-induced%20point%20mutation%20in%20Neurospora%20crassa&amp;volume=99&amp;publication_year=2002&amp;pages=8802-8807&amp;pmid=12072568&amp;doi=10.1073/pnas.132212899&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">33.</span><cite>Lewis Z. A., Honda S., Khlafallah T. K., Jeffress J. K., Freitag M., Mohn F., Schübeler D., Selker E. U., 
Relics of repeat-induced point mutation direct heterochromatin formation in <em>Neurospora crassa</em>. Genome Res.
19, 
427–437 (2009).
</cite> [<a href="https://doi.org/10.1101/gr.086231.108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2661801/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19092133/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Relics%20of%20repeat-induced%20point%20mutation%20direct%20heterochromatin%20formation%20in%20Neurospora%20crassa&amp;volume=19&amp;publication_year=2009&amp;pages=427-437&amp;pmid=19092133&amp;doi=10.1101/gr.086231.108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">34.</span><cite>Gladyshev E., Kleckner N., 
DNA sequence homology induces cytosine-to-thymine mutation by a heterochromatin-related pathway in <em>Neurospora</em>. Nat. Genet.
49, 
887–894 (2017).
</cite> [<a href="https://doi.org/10.1038/ng.3857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5474309/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28459455/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Genet.&amp;title=DNA%20sequence%20homology%20induces%20cytosine-to-thymine%20mutation%20by%20a%20heterochromatin-related%20pathway%20in%20Neurospora&amp;volume=49&amp;publication_year=2017&amp;pages=887-894&amp;pmid=28459455&amp;doi=10.1038/ng.3857&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R35">
<span class="label">35.</span><cite>Pomraning K. R., Connolly L. R., Whalen J. P., Smith K. M., Freitag M., 
Repeat-induced point mutation, DNA methylation and heterochromatin in <em>Gibberella zeae</em> (anamorph: <em>Fusarium graminearum</em>). <em>Fusarium Genomics</em>. Mol. Cell. Biol.
93-109, (2013).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Biol.&amp;title=Repeat-induced%20point%20mutation,%20DNA%20methylation%20and%20heterochromatin%20in%20Gibberella%20zeae%20(anamorph:%20Fusarium%20graminearum).%20Fusarium%20Genomics&amp;volume=93-109&amp;publication_year=2013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">36.</span><cite>Son H., Min K., Lee J., Raju N. B., Lee Y.-W., 
Meiotic silencing in the homothallic fungus <em>Gibberella zeae</em>. Fungal Biol.
115, 
1290–1302 (2011).
</cite> [<a href="https://doi.org/10.1016/j.funbio.2011.09.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22115448/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Biol.&amp;title=Meiotic%20silencing%20in%20the%20homothallic%20fungus%20Gibberella%20zeae&amp;volume=115&amp;publication_year=2011&amp;pages=1290-1302&amp;pmid=22115448&amp;doi=10.1016/j.funbio.2011.09.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R37">
<span class="label">37.</span><cite>Liu H., Wang Q., He Y., Chen L., Hao C., Jiang C., Li Y., Dai Y., Kang Z., Xu J.-R., 
Genome-wide A-to-I RNA editing in fungi independent of ADAR enzymes. Genome Res.
26, 
499–509 (2016).
</cite> [<a href="https://doi.org/10.1101/gr.199877.115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4817773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26934920/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res.&amp;title=Genome-wide%20A-to-I%20RNA%20editing%20in%20fungi%20independent%20of%20ADAR%20enzymes&amp;volume=26&amp;publication_year=2016&amp;pages=499-509&amp;pmid=26934920&amp;doi=10.1101/gr.199877.115&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">38.</span><cite>Liu H., Li Y., Chen D., Qi Z., Wang Q., Wang J., Jiang C., Xu J.-R., 
A-to-I RNA editing is developmentally regulated and generally adaptive for sexual reproduction in <em>Neurospora crassa</em>. Proc. Natl. Acad. Sci. U.S.A.
114, 
E7756–E7765 (2017).
</cite> [<a href="https://doi.org/10.1073/pnas.1702591114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5604002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28847945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=A-to-I%20RNA%20editing%20is%20developmentally%20regulated%20and%20generally%20adaptive%20for%20sexual%20reproduction%20in%20Neurospora%20crassa&amp;volume=114&amp;publication_year=2017&amp;pages=E7756-E7765&amp;pmid=28847945&amp;doi=10.1073/pnas.1702591114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">39.</span><cite>Qi Z., Lu P., Long X., Cao X., Wu M., Xin K., Xue T., Gao X., Huang Y., Wang Q., Jiang C., Xu J.-R., Liu H., 
Adaptive advantages of restorative RNA editing in fungi for resolving survival-reproduction trade-offs. Sci. Adv.
10, 
eadk6130 (2024).
</cite> [<a href="https://doi.org/10.1126/sciadv.adk6130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10776026/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38181075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Adv.&amp;title=Adaptive%20advantages%20of%20restorative%20RNA%20editing%20in%20fungi%20for%20resolving%20survival-reproduction%20trade-offs&amp;volume=10&amp;publication_year=2024&amp;pages=eadk6130&amp;pmid=38181075&amp;doi=10.1126/sciadv.adk6130&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R40">
<span class="label">40.</span><cite>Feng C., Xin K., Du Y., Zou J., Xing X., Xiu Q., Zhang Y., Zhang R., Huang W., Wang Q., Jiang C., Wang X., Kang Z., Xu J.-R., Liu H., 
Unveiling the A-to-I mRNA editing machinery and its regulation and evolution in fungi. Nat. Commun.
15, 
3934 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-48336-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11087585/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38729938/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Commun.&amp;title=Unveiling%20the%20A-to-I%20mRNA%20editing%20machinery%20and%20its%20regulation%20and%20evolution%20in%20fungi&amp;volume=15&amp;publication_year=2024&amp;pages=3934&amp;pmid=38729938&amp;doi=10.1038/s41467-024-48336-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R41">
<span class="label">41.</span><cite>Xin K., Zhang Y., Fan L., Qi Z., Feng C., Wang Q., Jiang C., Xu J.-R., Liu H., 
Experimental evidence for the functional importance and adaptive advantage of A-to-I RNA editing in fungi. Proc. Natl. Acad. Sci. U.S.A.
120, 
e2219029120 (2023).
</cite> [<a href="https://doi.org/10.1073/pnas.2219029120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10041177/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36917661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Experimental%20evidence%20for%20the%20functional%20importance%20and%20adaptive%20advantage%20of%20A-to-I%20RNA%20editing%20in%20fungi&amp;volume=120&amp;publication_year=2023&amp;pages=e2219029120&amp;pmid=36917661&amp;doi=10.1073/pnas.2219029120&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R42">
<span class="label">42.</span><cite>Du Y., Wang C., Zhang Y., Liu H., 
Evolutionary origins and adaptive significance of A-to-I RNA editing in animals and fungi. Bioessays
47, 
e202400220 (2025).
</cite> [<a href="https://doi.org/10.1002/bies.202400220" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39981820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioessays&amp;title=Evolutionary%20origins%20and%20adaptive%20significance%20of%20A-to-I%20RNA%20editing%20in%20animals%20and%20fungi&amp;volume=47&amp;publication_year=2025&amp;pages=e202400220&amp;pmid=39981820&amp;doi=10.1002/bies.202400220&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R43">
<span class="label">43.</span><cite>Xin K., Kang X., Liu M., Fan J., Du Y., Feng C., Liu H., 
Navigating trade-offs: The adaptive significance of A-to-I RNA editing in fungi, bacteria, and animals. Epigenetics Insights
18, (2025).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Epigenetics%20Insights&amp;title=Navigating%20trade-offs:%20The%20adaptive%20significance%20of%20A-to-I%20RNA%20editing%20in%20fungi,%20bacteria,%20and%20animals&amp;volume=18&amp;publication_year=2025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R44">
<span class="label">44.</span><cite>Hao C., Yin J., Sun M., Wang Q., Liang J., Bian Z., Liu H., Xu J.-R., 
The meiosis-specific APC activator <em>FgAMA1</em> is dispensable for meiosis but important for ascosporogenesis in <em>Fusarium graminearum</em>. Mol. Microbiol.
111, 
1245–1262 (2019).
</cite> [<a href="https://doi.org/10.1111/mmi.14219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30746783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Microbiol.&amp;title=The%20meiosis-specific%20APC%20activator%20FgAMA1%20is%20dispensable%20for%20meiosis%20but%20important%20for%20ascosporogenesis%20in%20Fusarium%20graminearum&amp;volume=111&amp;publication_year=2019&amp;pages=1245-1262&amp;pmid=30746783&amp;doi=10.1111/mmi.14219&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R45">
<span class="label">45.</span><cite>Liu H., Zhang S., Ma J., Dai Y., Li C., Lyu X., Wang C., Xu J.-R., 
Two Cdc2 kinase genes with distinct functions in vegetative and infectious hyphae in <em>Fusarium graminearum</em>. PLOS Pathog.
11, 
e1004913 (2015).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1004913" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4470668/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26083253/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Pathog.&amp;title=Two%20Cdc2%20kinase%20genes%20with%20distinct%20functions%20in%20vegetative%20and%20infectious%20hyphae%20in%20Fusarium%20graminearum&amp;volume=11&amp;publication_year=2015&amp;pages=e1004913&amp;pmid=26083253&amp;doi=10.1371/journal.ppat.1004913&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R46">
<span class="label">46.</span><cite>Chen D., Wu C., Hao C., Huang P., Liu H., Bian Z., Xu J.-R., 
Sexual specific functions of Tub1 beta-tubulins require stage-specific RNA processing and expression in <em>Fusarium graminearum</em>. Environ. Microbiol.
20, 
4009–4021 (2018).
</cite> [<a href="https://doi.org/10.1111/1462-2920.14441" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30307105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Environ.%20Microbiol.&amp;title=Sexual%20specific%20functions%20of%20Tub1%20beta-tubulins%20require%20stage-specific%20RNA%20processing%20and%20expression%20in%20Fusarium%20graminearum&amp;volume=20&amp;publication_year=2018&amp;pages=4009-4021&amp;pmid=30307105&amp;doi=10.1111/1462-2920.14441&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R47">
<span class="label">47.</span><cite>Kim J.-E., Han K.-H., Jin J., Kim H., Kim J.-C., Yun S.-H., Lee Y.-W., 
Putative polyketide synthase and laccase genes for biosynthesis of aurofusarin in <em>Gibberella zeae</em>. Appl. Environ. Microbiol.
71, 
1701–1708 (2005).
</cite> [<a href="https://doi.org/10.1128/AEM.71.4.1701-1708.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1082506/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15811992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Environ.%20Microbiol.&amp;title=Putative%20polyketide%20synthase%20and%20laccase%20genes%20for%20biosynthesis%20of%20aurofusarin%20in%20Gibberella%20zeae&amp;volume=71&amp;publication_year=2005&amp;pages=1701-1708&amp;pmid=15811992&amp;doi=10.1128/AEM.71.4.1701-1708.2005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">48.</span><cite>Chen Y., Gao Q., Huang M., Liu Y., Liu Z., Liu X., Ma Z., 
Characterization of RNA silencing components in the plant pathogenic fungus. Sci. Rep.
5, 
12500 (2015).
</cite> [<a href="https://doi.org/10.1038/srep12500" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4515635/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26212591/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Characterization%20of%20RNA%20silencing%20components%20in%20the%20plant%20pathogenic%20fungus&amp;volume=5&amp;publication_year=2015&amp;pages=12500&amp;pmid=26212591&amp;doi=10.1038/srep12500&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R49">
<span class="label">49.</span><cite>Son H., Park A. R., Lim J. Y., Shin C., Lee Y.-W., 
Genome-wide exonic small interference RNA-mediated gene silencing regulates sexual reproduction in the homothallic fungus <em>Fusarium graminearum</em>. PLOS Genet.
13, 
e1006595 (2017).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1006595" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5310905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28146558/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genet.&amp;title=Genome-wide%20exonic%20small%20interference%20RNA-mediated%20gene%20silencing%20regulates%20sexual%20reproduction%20in%20the%20homothallic%20fungus%20Fusarium%20graminearum&amp;volume=13&amp;publication_year=2017&amp;pages=e1006595&amp;pmid=28146558&amp;doi=10.1371/journal.pgen.1006595&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">50.</span><cite>Chang S.-S., Zhang Z., Liu Y., 
RNA interference pathways in fungi: Mechanisms and functions. Annu. Rev. Microbiol.
66, 
305–323 (2012).
</cite> [<a href="https://doi.org/10.1146/annurev-micro-092611-150138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4617789/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22746336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annu.%20Rev.%20Microbiol.&amp;title=RNA%20interference%20pathways%20in%20fungi:%20Mechanisms%20and%20functions&amp;volume=66&amp;publication_year=2012&amp;pages=305-323&amp;pmid=22746336&amp;doi=10.1146/annurev-micro-092611-150138&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R51">
<span class="label">51.</span><cite>Catalanotto C., Pallotta M., ReFalo P., Sachs M. S., Vayssie L., Macino G., Cogoni C., 
Redundancy of the two Dicer genes in transgene-induced posttranscriptional gene silencing in <em>Neurospora crassa</em>. Mol. Cell. Biol.
24, 
2536–2545 (2004).
</cite> [<a href="https://doi.org/10.1128/MCB.24.6.2536-2545.2004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC355837/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14993290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell.%20Biol.&amp;title=Redundancy%20of%20the%20two%20Dicer%20genes%20in%20transgene-induced%20posttranscriptional%20gene%20silencing%20in%20Neurospora%20crassa&amp;volume=24&amp;publication_year=2004&amp;pages=2536-2545&amp;pmid=14993290&amp;doi=10.1128/MCB.24.6.2536-2545.2004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R52">
<span class="label">52.</span><cite>Buker S. M., Iida T., Bühler M., Villén J., Gygi S. P., Nakayama J.-I., Moazed D., 
Two different Argonaute complexes are required for siRNA generation and heterochromatin assembly in fission yeast. Nat. Struct. Mol. Biol.
14, 
200–207 (2007).
</cite> [<a href="https://doi.org/10.1038/nsmb1211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17310250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Struct.%20Mol.%20Biol.&amp;title=Two%20different%20Argonaute%20complexes%20are%20required%20for%20siRNA%20generation%20and%20heterochromatin%20assembly%20in%20fission%20yeast&amp;volume=14&amp;publication_year=2007&amp;pages=200-207&amp;pmid=17310250&amp;doi=10.1038/nsmb1211&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R53">
<span class="label">53.</span><cite>Maiti M., Lee H.-C., Liu Y., 
QIP, a putative exonuclease, interacts with the Neurospora Argonaute protein and facilitates conversion of duplex siRNA into single strands. Genes Dev.
21, 
590–600 (2007).
</cite> [<a href="https://doi.org/10.1101/gad.1497607" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1820900/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17311884/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=QIP,%20a%20putative%20exonuclease,%20interacts%20with%20the%20Neurospora%20Argonaute%20protein%20and%20facilitates%20conversion%20of%20duplex%20siRNA%20into%20single%20strands&amp;volume=21&amp;publication_year=2007&amp;pages=590-600&amp;pmid=17311884&amp;doi=10.1101/gad.1497607&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R54">
<span class="label">54.</span><cite>Claeys Bouuaert C., Tischfield S. E., Pu S., Mimitou E. P., Arias-Palomo E., Berger J. M., Keeney S., 
Structural and functional characterization of the Spo11 core complex. Nat. Struct. Mol. Biol.
28, 
92–102 (2021).
</cite> [<a href="https://doi.org/10.1038/s41594-020-00534-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7855791/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33398171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Struct.%20Mol.%20Biol.&amp;title=Structural%20and%20functional%20characterization%20of%20the%20Spo11%20core%20complex&amp;volume=28&amp;publication_year=2021&amp;pages=92-102&amp;pmid=33398171&amp;doi=10.1038/s41594-020-00534-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R55">
<span class="label">55.</span><cite>Ceccaldi R., Rondinelli B., D’Andrea A. D., 
Repair pathway choices and consequences at the double-strand break. Trends Cell Biol.
26, 
52–64 (2016).
</cite> [<a href="https://doi.org/10.1016/j.tcb.2015.07.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4862604/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26437586/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Cell%20Biol.&amp;title=Repair%20pathway%20choices%20and%20consequences%20at%20the%20double-strand%20break&amp;volume=26&amp;publication_year=2016&amp;pages=52-64&amp;pmid=26437586&amp;doi=10.1016/j.tcb.2015.07.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R56">
<span class="label">56.</span><cite>Blackford A. N., Jackson S. P., 
ATM, ATR, and DNA-PK: The trinity at the heart of the DNA damage response. Mol. Cell
66, 
801–817 (2017).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2017.05.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28622525/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cell&amp;title=ATM,%20ATR,%20and%20DNA-PK:%20The%20trinity%20at%20the%20heart%20of%20the%20DNA%20damage%20response&amp;volume=66&amp;publication_year=2017&amp;pages=801-817&amp;pmid=28622525&amp;doi=10.1016/j.molcel.2017.05.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R57">
<span class="label">57.</span><cite>Dang Y., Li L., Guo W., Xue Z., Liu Y., 
Convergent transcription induces dynamic DNA methylation at disiRNA loci. PLOS Genet.
9, 
e1003761 (2013).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1003761" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3764098/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24039604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genet.&amp;title=Convergent%20transcription%20induces%20dynamic%20DNA%20methylation%20at%20disiRNA%20loci&amp;volume=9&amp;publication_year=2013&amp;pages=e1003761&amp;pmid=24039604&amp;doi=10.1371/journal.pgen.1003761&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R58">
<span class="label">58.</span><cite>Tamaru H., Selker E. U., 
A histone H3 methyltransferase controls DNA methylation in <em>Neurospora crassa</em>. Nature
414, 
277–283 (2001).
</cite> [<a href="https://doi.org/10.1038/35104508" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11713521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=A%20histone%20H3%20methyltransferase%20controls%20DNA%20methylation%20in%20Neurospora%20crassa&amp;volume=414&amp;publication_year=2001&amp;pages=277-283&amp;pmid=11713521&amp;doi=10.1038/35104508&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R59">
<span class="label">59.</span><cite>Singer M. J., Kuzminova E. A., Tharp A., Margolin B. S., Selker E. U., 
Different frequencies of RIP among early vs. late ascospores of <em>Neurospora crassa</em>. Fungal Genet. Rep.
42, 
74–75 (1995).</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Rep.&amp;title=Different%20frequencies%20of%20RIP%20among%20early%20vs.%20late%20ascospores%20of%20Neurospora%20crassa&amp;volume=42&amp;publication_year=1995&amp;pages=74-75&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R60">
<span class="label">60.</span><cite>Cui R., Li H., Zhao J., Li X., Gan J., Ma J., 
Structural insights into the dual activities of the two-barrel RNA polymerase QDE-1. Nucleic Acids Res.
50, 
10169–10186 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkac727" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9508822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36039765/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=Structural%20insights%20into%20the%20dual%20activities%20of%20the%20two-barrel%20RNA%20polymerase%20QDE-1&amp;volume=50&amp;publication_year=2022&amp;pages=10169-10186&amp;pmid=36039765&amp;doi=10.1093/nar/gkac727&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R61">
<span class="label">61.</span><cite>van Steensel B., Henikoff S., 
Identification of in vivo DNA targets of chromatin proteins using tethered dam methyltransferase. Nat. Biotechnol.
18, 
424–428 (2000).
</cite> [<a href="https://doi.org/10.1038/74487" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10748524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Biotechnol.&amp;title=Identification%20of%20in%20vivo%20DNA%20targets%20of%20chromatin%20proteins%20using%20tethered%20dam%20methyltransferase&amp;volume=18&amp;publication_year=2000&amp;pages=424-428&amp;pmid=10748524&amp;doi=10.1038/74487&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R62">
<span class="label">62.</span><cite>Petell C. J., Loiseau G., Gandy R., Pradhan S., Gowher H., 
A refined DNA methylation detection method using MspJI coupled quantitative PCR. Anal. Biochem.
533, 
1–9 (2017).
</cite> [<a href="https://doi.org/10.1016/j.ab.2017.06.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5972016/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28624296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Anal.%20Biochem.&amp;title=A%20refined%20DNA%20methylation%20detection%20method%20using%20MspJI%20coupled%20quantitative%20PCR&amp;volume=533&amp;publication_year=2017&amp;pages=1-9&amp;pmid=28624296&amp;doi=10.1016/j.ab.2017.06.006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R63">
<span class="label">63.</span><cite>Cogoni C., Macino G., 
Posttranscriptional gene silencing in Neurospora by a RecQ DNA helicase. Science
286, 
2342–2344 (1999).
</cite> [<a href="https://doi.org/10.1126/science.286.5448.2342" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10600745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Posttranscriptional%20gene%20silencing%20in%20Neurospora%20by%20a%20RecQ%20DNA%20helicase&amp;volume=286&amp;publication_year=1999&amp;pages=2342-2344&amp;pmid=10600745&amp;doi=10.1126/science.286.5448.2342&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R64">
<span class="label">64.</span><cite>Lu H., Davis A. J., 
Human RecQ helicases in DNA double-strand break repair. Front. Cell Dev. Biol.
9, 
640755 (2021).
</cite> [<a href="https://doi.org/10.3389/fcell.2021.640755" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7947261/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33718381/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Cell%20Dev.%20Biol.&amp;title=Human%20RecQ%20helicases%20in%20DNA%20double-strand%20break%20repair&amp;volume=9&amp;publication_year=2021&amp;pages=640755&amp;pmid=33718381&amp;doi=10.3389/fcell.2021.640755&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R65">
<span class="label">65.</span><cite>Qin Z., Bi L., Hou X.-M., Zhang S., Zhang X., Lu Y., Li M., Modesti M., Xi X.-G., Sun B., 
Human RPA activates BLM’s bidirectional DNA unwinding from a nick. eLife
9, 
e54098 (2020).
</cite> [<a href="https://doi.org/10.7554/eLife.54098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7065910/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32101168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=Human%20RPA%20activates%20BLM%E2%80%99s%20bidirectional%20DNA%20unwinding%20from%20a%20nick&amp;volume=9&amp;publication_year=2020&amp;pages=e54098&amp;pmid=32101168&amp;doi=10.7554/eLife.54098&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R66">
<span class="label">66.</span><cite>Cuomo C. A., Güldener U., Xu J.-R., Trail F., Turgeon B. G., Di Pietro A., Walton J. D., Ma L.-J., Baker S. E., Rep M., Adam G., Antoniw J., Baldwin T., Calvo S., Chang Y.-L., Decaprio D., Gale L. R., Gnerre S., Goswami R. S., Hammond-Kosack K., Harris L. J., Hilburn K., Kennell J. C., Kroken S., Magnuson J. K., Mannhaupt G., Mauceli E., Mewes H. W., Mitterbauer R., Muehlbauer G., Münsterkötter M., Nelson D., O’Donnell K., Ouellet T., Qi W., Quesneville H., Roncero M. I. G., Seong K.-Y., Tetko I. V., Urban M., Waalwijk C., Ward T. J., Yao J., Birren B. W., Kistler H. C., 
The <em>Fusarium graminearum</em> genome reveals a link between localized polymorphism and pathogen specialization. Science
317, 
1400–1402 (2007).
</cite> [<a href="https://doi.org/10.1126/science.1143708" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17823352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20Fusarium%20graminearum%20genome%20reveals%20a%20link%20between%20localized%20polymorphism%20and%20pathogen%20specialization&amp;volume=317&amp;publication_year=2007&amp;pages=1400-1402&amp;pmid=17823352&amp;doi=10.1126/science.1143708&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R67">
<span class="label">67.</span><cite>King R., Urban M., Hammond-Kosack M. C. U., Hassani-Pak K., Hammond-Kosack K. E., 
The completed genome sequence of the pathogenic ascomycete fungus <em>Fusarium graminearum</em>. BMC Genomics
16, 
544 (2015).
</cite> [<a href="https://doi.org/10.1186/s12864-015-1756-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4511438/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26198851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=The%20completed%20genome%20sequence%20of%20the%20pathogenic%20ascomycete%20fungus%20Fusarium%20graminearum&amp;volume=16&amp;publication_year=2015&amp;pages=544&amp;pmid=26198851&amp;doi=10.1186/s12864-015-1756-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R68">
<span class="label">68.</span><cite>Carlier F., Castro Ramirez S., Kilani J., Chehboub S., Loïodice I., Taddei A., Gladyshev E., 
Remodeling of perturbed chromatin can initiate de novo transcriptional and post-transcriptional silencing. Proc. Natl. Acad. Sci. U.S.A.
121, 
e2402944121 (2024).
</cite> [<a href="https://doi.org/10.1073/pnas.2402944121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11295056/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39052837/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Remodeling%20of%20perturbed%20chromatin%20can%20initiate%20de%20novo%20transcriptional%20and%20post-transcriptional%20silencing&amp;volume=121&amp;publication_year=2024&amp;pages=e2402944121&amp;pmid=39052837&amp;doi=10.1073/pnas.2402944121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R69">
<span class="label">69.</span><cite>Rhoades N., Nguyen T.-S., Witz G., Cecere G., Hammond T., Mazur A. K., Gladyshev E., 
Recombination-independent recognition of DNA homology for meiotic silencing in <em>Neurospora crassa</em>. Proc. Natl. Acad. Sci. U.S.A.
118, 
e2108664118 (2021).
</cite> [<a href="https://doi.org/10.1073/pnas.2108664118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8379962/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34385329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Recombination-independent%20recognition%20of%20DNA%20homology%20for%20meiotic%20silencing%20in%20Neurospora%20crassa&amp;volume=118&amp;publication_year=2021&amp;pages=e2108664118&amp;pmid=34385329&amp;doi=10.1073/pnas.2108664118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R70">
<span class="label">70.</span><cite>Samarajeewa D. A., Sauls P. A., Sharp K. J., Smith Z. J., Xiao H., Groskreutz K. M., Malone T. L., Boone E. C., Edwards K. A., Shiu P. K. T., Larson E. D., Hammond T. M., 
Efficient detection of unpaired DNA requires a member of the rad54-like family of homologous recombination proteins. Genetics
198, 
895–904 (2014).
</cite> [<a href="https://doi.org/10.1534/genetics.114.168187" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4224178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25146971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Efficient%20detection%20of%20unpaired%20DNA%20requires%20a%20member%20of%20the%20rad54-like%20family%20of%20homologous%20recombination%20proteins&amp;volume=198&amp;publication_year=2014&amp;pages=895-904&amp;pmid=25146971&amp;doi=10.1534/genetics.114.168187&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R71">
<span class="label">71.</span><cite>Wang X., Darwiche S., Heitman J., 
Sex-induced silencing operates during opposite-sex and unisexual reproduction in Cryptococcus neoformans. Genetics
193, 
1163–1174 (2013).
</cite> [<a href="https://doi.org/10.1534/genetics.113.149443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3606094/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23378067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Sex-induced%20silencing%20operates%20during%20opposite-sex%20and%20unisexual%20reproduction%20in%20Cryptococcus%20neoformans&amp;volume=193&amp;publication_year=2013&amp;pages=1163-1174&amp;pmid=23378067&amp;doi=10.1534/genetics.113.149443&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R72">
<span class="label">72.</span><cite>Carthew R. W., Sontheimer E. J., 
Origins and Mechanisms of miRNAs and siRNAs. Cell
136, 
642–655 (2009).
</cite> [<a href="https://doi.org/10.1016/j.cell.2009.01.035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2675692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19239886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Origins%20and%20Mechanisms%20of%20miRNAs%20and%20siRNAs&amp;volume=136&amp;publication_year=2009&amp;pages=642-655&amp;pmid=19239886&amp;doi=10.1016/j.cell.2009.01.035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R73">
<span class="label">73.</span><cite>Doench J. G., Petersen C. P., Sharp P. A., 
siRNAs can function as miRNAs. Genes Dev.
17, 
438–442 (2003).
</cite> [<a href="https://doi.org/10.1101/gad.1064703" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC195999/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12600936/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes%20Dev.&amp;title=siRNAs%20can%20function%20as%20miRNAs&amp;volume=17&amp;publication_year=2003&amp;pages=438-442&amp;pmid=12600936&amp;doi=10.1101/gad.1064703&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R74">
<span class="label">74.</span><cite>Wu H., Li B., Iwakawa H.-O., Pan Y., Tang X., Ling-Hu Q., Liu Y., Sheng S., Feng L., Zhang H., Zhang X., Tang Z., Xia X., Zhai J., Guo H., 
Plant 22-nt siRNAs mediate translational repression and stress adaptation. Nature
581, 
89–93 (2020).
</cite> [<a href="https://doi.org/10.1038/s41586-020-2231-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32376953/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Plant%2022-nt%20siRNAs%20mediate%20translational%20repression%20and%20stress%20adaptation&amp;volume=581&amp;publication_year=2020&amp;pages=89-93&amp;pmid=32376953&amp;doi=10.1038/s41586-020-2231-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R75">
<span class="label">75.</span><cite>Ma X., Kim E.-J., Kook I., Ma F., Voshall A., Moriyama E., Cerutti H., 
Small interfering RNA-mediated translation repression alters ribosome sensitivity to inhibition by cycloheximide in <em>Chlamydomonas reinhardtii</em>. Plant Cell
25, 
985–998 (2013).
</cite> [<a href="https://doi.org/10.1105/tpc.113.109256" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3634701/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23512853/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell&amp;title=Small%20interfering%20RNA-mediated%20translation%20repression%20alters%20ribosome%20sensitivity%20to%20inhibition%20by%20cycloheximide%20in%20Chlamydomonas%20reinhardtii&amp;volume=25&amp;publication_year=2013&amp;pages=985-998&amp;pmid=23512853&amp;doi=10.1105/tpc.113.109256&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R76">
<span class="label">76.</span><cite>Lee C. H., Carroll B. J., 
Evolution and diversification of small RNA pathways in flowering plants. Plant Cell Physiol.
59, 
2169–2187 (2018).
</cite> [<a href="https://doi.org/10.1093/pcp/pcy167" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30169685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Cell%20Physiol.&amp;title=Evolution%20and%20diversification%20of%20small%20RNA%20pathways%20in%20flowering%20plants&amp;volume=59&amp;publication_year=2018&amp;pages=2169-2187&amp;pmid=30169685&amp;doi=10.1093/pcp/pcy167&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R77">
<span class="label">77.</span><cite>Brodersen P., Sakvarelidze-Achard L., Bruun-Rasmussen M., Dunoyer P., Yamamoto Y. Y., Sieburth L., Voinnet O., 
Widespread translational inhibition by plant miRNAs and siRNAs. Science
320, 
1185–1190 (2008).
</cite> [<a href="https://doi.org/10.1126/science.1159151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18483398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Widespread%20translational%20inhibition%20by%20plant%20miRNAs%20and%20siRNAs&amp;volume=320&amp;publication_year=2008&amp;pages=1185-1190&amp;pmid=18483398&amp;doi=10.1126/science.1159151&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R78">
<span class="label">78.</span><cite>Calo S., Shertz-Wall C., Lee S. C., Bastidas R. J., Nicolás F. E., Granek J. A., Mieczkowski P., Torres-Martínez S., Ruiz-Vázquez R. M., Cardenas M. E., Heitman J., 
Antifungal drug resistance evoked via RNAi-dependent epimutations. Nature
513, 
555–558 (2014).
</cite> [<a href="https://doi.org/10.1038/nature13575" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4177005/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25079329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Antifungal%20drug%20resistance%20evoked%20via%20RNAi-dependent%20epimutations&amp;volume=513&amp;publication_year=2014&amp;pages=555-558&amp;pmid=25079329&amp;doi=10.1038/nature13575&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R79">
<span class="label">79.</span><cite>Meister G., 
Argonaute proteins: Functional insights and emerging roles. Nat. Rev. Genet.
14, 
447–459 (2013).
</cite> [<a href="https://doi.org/10.1038/nrg3462" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23732335/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Genet.&amp;title=Argonaute%20proteins:%20Functional%20insights%20and%20emerging%20roles&amp;volume=14&amp;publication_year=2013&amp;pages=447-459&amp;pmid=23732335&amp;doi=10.1038/nrg3462&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R80">
<span class="label">80.</span><cite>Dueva R., Iliakis G., 
Replication protein A: A multifunctional protein with roles in DNA replication, repair and beyond. NAR Cancer
2, 
zcaa022 (2020).
</cite> [<a href="https://doi.org/10.1093/narcan/zcaa022" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8210275/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34316690/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NAR%20Cancer&amp;title=Replication%20protein%20A:%20A%20multifunctional%20protein%20with%20roles%20in%20DNA%20replication,%20repair%20and%20beyond&amp;volume=2&amp;publication_year=2020&amp;pages=zcaa022&amp;pmid=34316690&amp;doi=10.1093/narcan/zcaa022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R81">
<span class="label">81.</span><cite>Chicas A., Cogoni C., Macino G., 
RNAi-dependent and RNAi-independent mechanisms contribute to the silencing of RIPed sequences in <em>Neurospora crassa</em>. Nucleic Acids Res.
32, 
4237–4243 (2004).
</cite> [<a href="https://doi.org/10.1093/nar/gkh764" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC514385/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15302921/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res.&amp;title=RNAi-dependent%20and%20RNAi-independent%20mechanisms%20contribute%20to%20the%20silencing%20of%20RIPed%20sequences%20in%20Neurospora%20crassa&amp;volume=32&amp;publication_year=2004&amp;pages=4237-4243&amp;pmid=15302921&amp;doi=10.1093/nar/gkh764&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R82">
<span class="label">82.</span><cite>Wang Z., Lopez-Giraldez F., Lehr N., Farré M., Common R., Trail F., Townsend J. P., 
Global gene expression and focused knockout analysis reveals genes associated with fungal fruiting body development in <em>Neurospora crassa</em>. Eukaryot. Cell
13, 
154–169 (2014).
</cite> [<a href="https://doi.org/10.1128/EC.00248-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3910948/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24243796/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eukaryot.%20Cell&amp;title=Global%20gene%20expression%20and%20focused%20knockout%20analysis%20reveals%20genes%20associated%20with%20fungal%20fruiting%20body%20development%20in%20Neurospora%20crassa&amp;volume=13&amp;publication_year=2014&amp;pages=154-169&amp;pmid=24243796&amp;doi=10.1128/EC.00248-13&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R83">
<span class="label">83.</span><cite>Dennen M. S., Kockler Z. W., Roberts S. A., Burkholder A. B., Klimczak L. J., Gordenin D. A., 
Hypomorphic mutation in the large subunit of replication protein A affects mutagenesis by human APOBEC cytidine deaminases in yeast. G3 (Bethesda)
14, 
jkae196 (2024).
</cite> [<a href="https://doi.org/10.1093/g3journal/jkae196" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11457066/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39150943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20(Bethesda)&amp;title=Hypomorphic%20mutation%20in%20the%20large%20subunit%20of%20replication%20protein%20A%20affects%20mutagenesis%20by%20human%20APOBEC%20cytidine%20deaminases%20in%20yeast&amp;volume=14&amp;publication_year=2024&amp;pages=jkae196&amp;pmid=39150943&amp;doi=10.1093/g3journal/jkae196&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R84">
<span class="label">84.</span><cite>Malagnac F., Wendel B., Goyon C., Faugeron G., Zickler D., Rossignol J. L., Noyer-Weidner M., Vollmayr P., Trautner T. A., Walter J., 
A gene essential for de novo methylation and development in <em>Ascobolus</em> reveals a novel type of eukaryotic DNA methyltransferase structure. Cell
91, 
281–290 (1997).
</cite> [<a href="https://doi.org/10.1016/s0092-8674(00)80410-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9346245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A%20gene%20essential%20for%20de%20novo%20methylation%20and%20development%20in%20Ascobolus%20reveals%20a%20novel%20type%20of%20eukaryotic%20DNA%20methyltransferase%20structure&amp;volume=91&amp;publication_year=1997&amp;pages=281-290&amp;pmid=9346245&amp;doi=10.1016/s0092-8674(00)80410-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R85">
<span class="label">85.</span><cite>Lee D. W., Freitag M., Selker E. U., Aramayo R., 
A cytosine methyltransferase homologue is essential for sexual development in <em>Aspergillus nidulans</em>. PLOS ONE
3, 
e2531 (2008).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0002531" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2432034/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18575630/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20ONE&amp;title=A%20cytosine%20methyltransferase%20homologue%20is%20essential%20for%20sexual%20development%20in%20Aspergillus%20nidulans&amp;volume=3&amp;publication_year=2008&amp;pages=e2531&amp;pmid=18575630&amp;doi=10.1371/journal.pone.0002531&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R86">
<span class="label">86.</span><cite>Li W.-C., Chen C.-L., Wang T.-F., 
Repeat-induced point (RIP) mutation in the industrial workhorse fungus <em>Trichoderma reesei</em>. Appl. Microbiol. Biotechnol.
102, 
1567–1574 (2018).
</cite> [<a href="https://doi.org/10.1007/s00253-017-8731-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29308529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Appl.%20Microbiol.%20Biotechnol.&amp;title=Repeat-induced%20point%20(RIP)%20mutation%20in%20the%20industrial%20workhorse%20fungus%20Trichoderma%20reesei&amp;volume=102&amp;publication_year=2018&amp;pages=1567-1574&amp;pmid=29308529&amp;doi=10.1007/s00253-017-8731-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R87">
<span class="label">87.</span><cite>Grognet P., Timpano H., Carlier F., Aït-Benkhali J., Berteaux-Lecellier V., Debuchy R., Bidard F., Malagnac F., 
A RID-like putative cytosine methyltransferase homologue controls sexual development in the fungus <em>Podospora anserina</em>. PLOS Genet.
15, 
e1008086 (2019).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1008086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6709928/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31412020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genet.&amp;title=A%20RID-like%20putative%20cytosine%20methyltransferase%20homologue%20controls%20sexual%20development%20in%20the%20fungus%20Podospora%20anserina&amp;volume=15&amp;publication_year=2019&amp;pages=e1008086&amp;pmid=31412020&amp;doi=10.1371/journal.pgen.1008086&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R88">
<span class="label">88.</span><cite>Kim H.-K., Jo S.-M., Kim G.-Y., Kim D.-W., Kim Y.-K., Yun S.-H., 
A large-scale functional analysis of putative target genes of mating-type loci provides insight into the regulation of sexual development of the cereal pathogen <em>Fusarium graminearum</em>. PLOS Genet.
11, 
e1005486 (2015).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1005486" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4559316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26334536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genet.&amp;title=A%20large-scale%20functional%20analysis%20of%20putative%20target%20genes%20of%20mating-type%20loci%20provides%20insight%20into%20the%20regulation%20of%20sexual%20development%20of%20the%20cereal%20pathogen%20Fusarium%20graminearum&amp;volume=11&amp;publication_year=2015&amp;pages=e1005486&amp;pmid=26334536&amp;doi=10.1371/journal.pgen.1005486&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R89">
<span class="label">89.</span><cite>Girard C., Budin K., Boisnard S., Zhang L., Debuchy R., Zickler D., Espagne E., 
RNAi-related dicer and argonaute proteins play critical roles for meiocyte formation, chromosome-axes lengths and crossover patterning in the Fungus <em>Sordaria macrospora</em>. Front. Cell Dev. Biol.
9, 
684108 (2021).
</cite> [<a href="https://doi.org/10.3389/fcell.2021.684108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8274715/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34262901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Cell%20Dev.%20Biol.&amp;title=RNAi-related%20dicer%20and%20argonaute%20proteins%20play%20critical%20roles%20for%20meiocyte%20formation,%20chromosome-axes%20lengths%20and%20crossover%20patterning%20in%20the%20Fungus%20Sordaria%20macrospora&amp;volume=9&amp;publication_year=2021&amp;pages=684108&amp;pmid=34262901&amp;doi=10.3389/fcell.2021.684108&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R90">
<span class="label">90.</span><cite>Goswami R. S., 
Targeted gene replacement in fungi using a split-marker approach. Methods Mol. Biol.
835, 
255–269 (2012).
</cite> [<a href="https://doi.org/10.1007/978-1-61779-501-5_16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22183659/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol.%20Biol.&amp;title=Targeted%20gene%20replacement%20in%20fungi%20using%20a%20split-marker%20approach&amp;volume=835&amp;publication_year=2012&amp;pages=255-269&amp;pmid=22183659&amp;doi=10.1007/978-1-61779-501-5_16&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R91">
<span class="label">91.</span><cite>Wang C., Zhang S., Hou R., Zhao Z., Zheng Q., Xu Q., Zheng D., Wang G., Liu H., Gao X., Ma J.-W., Kistler H. C., Kang Z., Xu J.-R., 
Functional analysis of the kinome of the wheat scab fungus <em>Fusarium graminearum</em>. PLOS Pathog.
7, 
e1002460 (2011).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1002460" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3245316/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22216007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Pathog.&amp;title=Functional%20analysis%20of%20the%20kinome%20of%20the%20wheat%20scab%20fungus%20Fusarium%20graminearum&amp;volume=7&amp;publication_year=2011&amp;pages=e1002460&amp;pmid=22216007&amp;doi=10.1371/journal.ppat.1002460&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R92">
<span class="label">92.</span><cite>Feng C., Cao X., Du Y., Chen Y., Xin K., Zou J., Jin Q., Xu J.-R., Liu H., 
Uncovering <em>Cis</em>-regulatory elements important for A-to-I RNA Editing in <em>Fusarium graminearum</em>. mBio
13, 
e0187222 (2022).
</cite> [<a href="https://doi.org/10.1128/mbio.01872-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9600606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36102513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=mBio&amp;title=Uncovering%20Cis-regulatory%20elements%20important%20for%20A-to-I%20RNA%20Editing%20in%20Fusarium%20graminearum&amp;volume=13&amp;publication_year=2022&amp;pages=e0187222&amp;pmid=36102513&amp;doi=10.1128/mbio.01872-22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R93">
<span class="label">93.</span><cite>Wang H., Chen D., Li C., Tian N., Zhang J., Xu J.-R., Wang C., 
Stage-specific functional relationships between Tub1 and Tub2 beta-tubulins in the wheat scab fungus <em>Fusarium graminearum</em>. Fungal Genet. Biol.
132, 
103251 (2019).
</cite> [<a href="https://doi.org/10.1016/j.fgb.2019.103251" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31319136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Fungal%20Genet.%20Biol.&amp;title=Stage-specific%20functional%20relationships%20between%20Tub1%20and%20Tub2%20beta-tubulins%20in%20the%20wheat%20scab%20fungus%20Fusarium%20graminearum&amp;volume=132&amp;publication_year=2019&amp;pages=103251&amp;pmid=31319136&amp;doi=10.1016/j.fgb.2019.103251&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R94">
<span class="label">94.</span><cite>Lu P., Chen D., Qi Z., Wang H., Chen Y., Wang Q., Jiang C., Xu J.-R., Liu H., 
Landscape and regulation of alternative splicing and alternative polyadenylation in a plant pathogenic fungus. New Phytol.
235, 
674–689 (2022).
</cite> [<a href="https://doi.org/10.1111/nph.18164" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35451076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytol.&amp;title=Landscape%20and%20regulation%20of%20alternative%20splicing%20and%20alternative%20polyadenylation%20in%20a%20plant%20pathogenic%20fungus&amp;volume=235&amp;publication_year=2022&amp;pages=674-689&amp;pmid=35451076&amp;doi=10.1111/nph.18164&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R95">
<span class="label">95.</span><cite>Kim D., Langmead B., Salzberg S. L., 
HISAT: A fast spliced aligner with low memory requirements. Nat. Methods
12, 
357–360 (2015).
</cite> [<a href="https://doi.org/10.1038/nmeth.3317" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4655817/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25751142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=HISAT:%20A%20fast%20spliced%20aligner%20with%20low%20memory%20requirements&amp;volume=12&amp;publication_year=2015&amp;pages=357-360&amp;pmid=25751142&amp;doi=10.1038/nmeth.3317&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<p>Figs. S1 to S9</p>
<p>Table S1</p>
<p>Legends for datasets S1 and S2</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366691/bin/sciadv.adu7606_sm.pdf" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adu7606_sm.pdf</a><sup> (1.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>Datasets S1 and S2</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12366691/bin/sciadv.adu7606_datasets_s1_and_s2.zip" data-ga-action="click_feat_suppl" class="usa-link">sciadv.adu7606_datasets_s1_and_s2.zip</a><sup> (32.2KB, zip) </sup>
</div></div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Science Advances are provided here courtesy of <strong>American Association for the Advancement of Science</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1126/sciadv.adu7606"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/sciadv.adu7606.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366691/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366691/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366691%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366691/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366691/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366691/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40834073/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366691/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40834073/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366691/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366691/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="AuFwLDN3FFQey87xPDzm7VVburR594qbTZWHRN3SS8Kwp6ink1iZ96qsT1YtdS4z">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
