
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Phylogenomics and structural variations of plastid genomes of the columbine genus (Ranunculaceae) - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5336D28AF2DC230536D2004F2F516A.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcps">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366047/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Plant Biology">
<meta name="citation_title" content="Phylogenomics and structural variations of plastid genomes of the columbine genus (Ranunculaceae)">
<meta name="citation_author" content="Fang-Min Yu">
<meta name="citation_author_institution" content="State Key Laboratory of Plant Diversity and Specialty Crops, Institute of Botany, Chinese Academy of Sciences, Beijing, 100093 China">
<meta name="citation_author_institution" content="China National Botanical Garden, Beijing, 100093 China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, 100049 China">
<meta name="citation_author" content="Wen-Chuang He">
<meta name="citation_author_institution" content="Shenzhen Branch, Guangdong Laboratory of Lingnan Modern Agriculture, Genome Analysis Laboratory of the Ministry of Agriculture and Rural Affairs, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, 518120 China">
<meta name="citation_author" content="Andrey S Erst">
<meta name="citation_author_institution" content="Central Siberian Botanical Garden, Siberian Branch of Russian Academy of Sciences, Zolotodolinskaya str. 101, Novosibirsk, 630090 Russia">
<meta name="citation_author" content="Huan-Wen Peng">
<meta name="citation_author_institution" content="State Key Laboratory of Plant Diversity and Specialty Crops, Institute of Botany, Chinese Academy of Sciences, Beijing, 100093 China">
<meta name="citation_author_institution" content="China National Botanical Garden, Beijing, 100093 China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, 100049 China">
<meta name="citation_author" content="Yuan-Yuan Ling">
<meta name="citation_author_institution" content="State Key Laboratory of Plant Diversity and Specialty Crops, Institute of Botany, Chinese Academy of Sciences, Beijing, 100093 China">
<meta name="citation_author_institution" content="China National Botanical Garden, Beijing, 100093 China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, 100049 China">
<meta name="citation_author" content="Tatyana V Erst">
<meta name="citation_author_institution" content="Institute of Cytology and Genetics, Siberian Branch of Russian Academy of Sciences, Novosibirsk, 630090 Russia">
<meta name="citation_author_institution" content="Siberian Federal Scientific Centre of Agrobiotechnologies of the Russian Academy of Sciences (SFSCA RAS), Krasnoobsk, 630501 Russia">
<meta name="citation_author" content="Michael V Bardukov">
<meta name="citation_author_institution" content="Central Siberian Botanical Garden, Siberian Branch of Russian Academy of Sciences, Zolotodolinskaya str. 101, Novosibirsk, 630090 Russia">
<meta name="citation_author_institution" content="Novosibirsk State University, Novosibirsk, 630090 Russia">
<meta name="citation_author" content="Wei Wang">
<meta name="citation_author_institution" content="State Key Laboratory of Plant Diversity and Specialty Crops, Institute of Botany, Chinese Academy of Sciences, Beijing, 100093 China">
<meta name="citation_author_institution" content="China National Botanical Garden, Beijing, 100093 China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, 100049 China">
<meta name="citation_author" content="Kun-Li Xiang">
<meta name="citation_author_institution" content="State Key Laboratory of Plant Diversity and Specialty Crops, Institute of Botany, Chinese Academy of Sciences, Beijing, 100093 China">
<meta name="citation_author_institution" content="China National Botanical Garden, Beijing, 100093 China">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="25">
<meta name="citation_firstpage" content="1106">
<meta name="citation_doi" content="10.1186/s12870-025-07093-6">
<meta name="citation_pmid" content="40836276">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366047/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366047/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366047/pdf/12870_2025_Article_7093.pdf">
<meta name="description" content="The columbine genus (Aquilegia) contains many species with horticultural and pharmaceutical importance. However, a well-resolved phylogeny for this genus remains lacking owing to recent and rapid radiation. We obtained plastomes of 75 Aquilegia ...">
<meta name="og:title" content="Phylogenomics and structural variations of plastid genomes of the columbine genus (Ranunculaceae)">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The columbine genus (Aquilegia) contains many species with horticultural and pharmaceutical importance. However, a well-resolved phylogeny for this genus remains lacking owing to recent and rapid radiation. We obtained plastomes of 75 Aquilegia ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366047/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366047">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12870-025-07093-6"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12870_2025_Article_7093.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366047%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366047/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366047/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366047/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcps.png" alt="BMC Plant Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Plant Biology" title="Link to BMC Plant Biology" shape="default" href="https://bmcplantbiol.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Plant Biol</button></div>. 2025 Aug 20;25:1106. doi: <a href="https://doi.org/10.1186/s12870-025-07093-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12870-025-07093-6</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Plant%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Plant%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Phylogenomics and structural variations of plastid genomes of the columbine genus (Ranunculaceae)</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20FM%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Fang-Min Yu</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Fang-Min Yu</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Plant Diversity and Specialty Crops, Institute of Botany, Chinese Academy of Sciences, Beijing, 100093 China </div>
<div class="p">
<sup>2</sup>China National Botanical Garden, Beijing, 100093 China </div>
<div class="p">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, 100049 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20FM%22%5BAuthor%5D" class="usa-link"><span class="name western">Fang-Min Yu</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20WC%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Wen-Chuang He</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Wen-Chuang He</span></h3>
<div class="p">
<sup>4</sup>Shenzhen Branch, Guangdong Laboratory of Lingnan Modern Agriculture, Genome Analysis Laboratory of the Ministry of Agriculture and Rural Affairs, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, 518120 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20WC%22%5BAuthor%5D" class="usa-link"><span class="name western">Wen-Chuang He</span></a>
</div>
</div>
<sup>4,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Erst%20AS%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Andrey S Erst</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Andrey S Erst</span></h3>
<div class="p">
<sup>5</sup>Central Siberian Botanical Garden, Siberian Branch of Russian Academy of Sciences, Zolotodolinskaya str. 101, Novosibirsk, 630090 Russia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Erst%20AS%22%5BAuthor%5D" class="usa-link"><span class="name western">Andrey S Erst</span></a>
</div>
</div>
<sup>5,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20HW%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Huan-Wen Peng</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Huan-Wen Peng</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Plant Diversity and Specialty Crops, Institute of Botany, Chinese Academy of Sciences, Beijing, 100093 China </div>
<div class="p">
<sup>2</sup>China National Botanical Garden, Beijing, 100093 China </div>
<div class="p">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, 100049 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Peng%20HW%22%5BAuthor%5D" class="usa-link"><span class="name western">Huan-Wen Peng</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ling%20YY%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Yuan-Yuan Ling</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Yuan-Yuan Ling</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Plant Diversity and Specialty Crops, Institute of Botany, Chinese Academy of Sciences, Beijing, 100093 China </div>
<div class="p">
<sup>2</sup>China National Botanical Garden, Beijing, 100093 China </div>
<div class="p">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, 100049 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ling%20YY%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuan-Yuan Ling</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Erst%20TV%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Tatyana V Erst</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Tatyana V Erst</span></h3>
<div class="p">
<sup>6</sup>Institute of Cytology and Genetics, Siberian Branch of Russian Academy of Sciences, Novosibirsk, 630090 Russia </div>
<div class="p">
<sup>7</sup>Siberian Federal Scientific Centre of Agrobiotechnologies of the Russian Academy of Sciences (SFSCA RAS), Krasnoobsk, 630501 Russia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Erst%20TV%22%5BAuthor%5D" class="usa-link"><span class="name western">Tatyana V Erst</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bardukov%20MV%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Michael V Bardukov</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Michael V Bardukov</span></h3>
<div class="p">
<sup>5</sup>Central Siberian Botanical Garden, Siberian Branch of Russian Academy of Sciences, Zolotodolinskaya str. 101, Novosibirsk, 630090 Russia </div>
<div class="p">
<sup>8</sup>Novosibirsk State University, Novosibirsk, 630090 Russia </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bardukov%20MV%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael V Bardukov</span></a>
</div>
</div>
<sup>5,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Wei Wang</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Wei Wang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Plant Diversity and Specialty Crops, Institute of Botany, Chinese Academy of Sciences, Beijing, 100093 China </div>
<div class="p">
<sup>2</sup>China National Botanical Garden, Beijing, 100093 China </div>
<div class="p">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, 100049 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wei Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>âœ‰</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiang%20KL%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Kun-Li Xiang</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Kun-Li Xiang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Plant Diversity and Specialty Crops, Institute of Botany, Chinese Academy of Sciences, Beijing, 100093 China </div>
<div class="p">
<sup>2</sup>China National Botanical Garden, Beijing, 100093 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xiang%20KL%22%5BAuthor%5D" class="usa-link"><span class="name western">Kun-Li Xiang</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>State Key Laboratory of Plant Diversity and Specialty Crops, Institute of Botany, Chinese Academy of Sciences, Beijing, 100093 China </div>
<div id="Aff2">
<sup>2</sup>China National Botanical Garden, Beijing, 100093 China </div>
<div id="Aff3">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, 100049 China </div>
<div id="Aff4">
<sup>4</sup>Shenzhen Branch, Guangdong Laboratory of Lingnan Modern Agriculture, Genome Analysis Laboratory of the Ministry of Agriculture and Rural Affairs, Agricultural Genomics Institute at Shenzhen, Chinese Academy of Agricultural Sciences, Shenzhen, 518120 China </div>
<div id="Aff5">
<sup>5</sup>Central Siberian Botanical Garden, Siberian Branch of Russian Academy of Sciences, Zolotodolinskaya str. 101, Novosibirsk, 630090 Russia </div>
<div id="Aff6">
<sup>6</sup>Institute of Cytology and Genetics, Siberian Branch of Russian Academy of Sciences, Novosibirsk, 630090 Russia </div>
<div id="Aff7">
<sup>7</sup>Siberian Federal Scientific Centre of Agrobiotechnologies of the Russian Academy of Sciences (SFSCA RAS), Krasnoobsk, 630501 Russia </div>
<div id="Aff8">
<sup>8</sup>Novosibirsk State University, Novosibirsk, 630090 Russia </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 19; Accepted 2025 Jul 18; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the articleâ€™s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366047Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40836276/" class="usa-link">40836276</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">The columbine genus (<em>Aquilegia</em>) contains many species with horticultural and pharmaceutical importance. However, a well-resolved phylogeny for this genus remains lacking owing to recent and rapid radiation. We obtained plastomes of 75 <em>Aquilegia</em> species and six species of other genera in Thalictroideae to reconstruct a robust phylogeny for <em>Aquilegia</em>. Within the phylogenetic framework, we investigated the evolutionary patterns of structural variations in <em>Aquilegia</em> plastomes<em>.</em> We also examined appropriate molecular markers for species identification of this genus.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">Plastid phylogenomic analyses show that <em>Aquilegia</em> is monophyletic and divided into two major clades and eight subclades, largely in agreement with species geographical distributions. <em>Aquilegia</em> plastomes have copy number variations (CNVs) of genes at the boundaries of IR/SC regions. An inversion was identified in the large single-copy region (LSC) plastomes of <em>Aquilegia</em>, <em>Semiaquilegia</em>, and <em>Urophysa</em>. The inversion was adjacent to three distinct structures (type I-III) in LSC region, each characterized by presence or absence of different short inverted repeat (sIR) regions. The most recent common ancestor of <em>Aquilegia</em> species was type III, and other two types were derived by shortening or loss of sIRs. Ten hypervariable regions were identified in <em>Aquilegia</em> plastomes.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par3">We present a well-resolved phylogeny for <em>Aquilegia</em>. The CNVs of genes were observed frequently at the boundaries of IR/SC regions and sIR regions in the genus, and the inversions and sIR in LSC regions have phylogenetic signal and lead to notable variations in gene content and length of plastomes. The new hypervariable regions can be used for phylogenetic analyses and molecular identification of <em>Aquilegia</em>. This study provides new insights into the structural evolution of plastome in angiosperms, and also have important implications for the conservation and utilization of <em>Aquilegia</em> species.</p></section><section id="sec4"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12870-025-07093-6.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> <em>Aquilegia</em>, Plastid genome, Structures variation, Phylogeny, Hypervariable region</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par28">Plastid genomes (plastomes) of angiosperms are generally recognized to be highly conserved in gene content and structure [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>â€“<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Typical angiosperm plastomes display a quadripartite circular structure, including two inverted repeat regions (IRs), a large single-copy region (LSC), and a small single-copy region (SSC) [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>, <a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. However, many studies have identified structural variations (SVs) in the plastomes of heterotrophic plants, i.e., parasites and mycoheterotrophs have more presence/absence variations (PAVs) in their plastomes causing by the loss of gene functions [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>â€“<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. Increasing of reported plastomes in recent years, lots of SVs of plastomes have been widely reported in angiosperms [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. The plastome SVs mainly focus on copy number variations (CNVs) resulting from the expansion/contraction of IR regions [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>â€“<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. For example, large-scale IR expansions exceeding several kb results in increased gene copy numbers through duplication events, including <em>Tetracentron</em> Oliv. and <em>Trochodendron</em> Siebold &amp; Zucc. (Trochodendraceae) [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>] and <em>Annona</em> L. (Annonaceae) [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. Conversely, IR losses have been observed in the boundaries of IR/SC regions of <em>Myosurus</em> L. and <em>Ceratocephala</em> Moench (Ranunculaceae) [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. However, the evolutionary patterns of the variations within LSC region and/or IR region remain poorly understood.</p>
<p id="Par29">In particular, SVs in the LSC region of plastomes have been reported and attracted significant attention [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. Unprecedented SVs have been found in the LSC region of plastomes in Poales, in which one-third of the families have plastomes with inversions exceeding 1 kb in length [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. A 51 kb inversion spanning <em>rbcL</em> to <em>rpsl</em><em>6</em> in the LSC region was reported in the <em>Lotus japonicus</em> (Regel) K. Larsen (Fabaceae) plastome [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. Similar inversions were observed in the LSC region of Ranunculaceae plastomes, including six inversions in Anemoneae [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>] and a single 42 kb inversion in <em>Adonis aestivalis</em> L. [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. Although inversions in the LSC region lead to notable variations in the structure, gene content, and length of plastomes, the phylogenetic significance of SVs in the LSC region still needs to be assessed.</p>
<p id="Par30"><em>Aquilegia</em> L. (Ranunculaceae), commonly known as the columbine genus, comprises <em>ca.</em> 80 perennial herbaceous species and is widely distributed in the Northern Hemiphere [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Because petals base usually prolong into a spur, flowers of <em>Aquilegia</em> are beautiful and unique, making <em>Aquilegia</em> species excellent ornamental plants for gardens. Some <em>Aquilegia</em> species are pharmaceutically important and have become traditional Chinese medicines [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. In addition, <em>Aquilegia</em> has been a new model for studying plant development, ecology, and evolution [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>â€“<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Among the genera with more than 15 species in Ranunculaceae, only <em>Aquilegia</em> lacks a infrageneric classification system [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. This is may be because that <em>Aquilegia</em> experienced a recent and rapid radiation [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. Hodges &amp; Arnold (1994) first used molecular data (plastid <em>atpB-rbcL</em> and nuclear ITS) to investigate phylogenetic relationships within <em>Aquilegia</em> [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>], where 14 species of this genus were sampled. In their phylogenetic tree, the monophyletic <em>Aquilegia</em> was recognized, but intrageneric relationships were poorly resolved [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. Based on plastid (<em>trnK-matK</em> and <em>trnS-G</em>) and nuclear (ITS) sequences, Bastida et al. (2010) sampled 32 <em>Aquilegia</em> species and did not resolved inter-species relationships based on maximum parsimony analysis. By sampling 62 recognized species and subspecific entities of <em>Aquilegia</em> and 21 plastid DNA regions [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>], Fior et al. (2013) divided <em>Aquilegia</em> into six geographic groups, but a few species did not belong to any group and the relationships within Europea group were not resolved at all [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Recently, Zhang et al. (2021) built a well-supported phylogenetic framework for this genus based on complete plastome sequences, but only included 16 species [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]<em>.</em> To reconstruct a robust phylogenetic framework for <em>Aquilegia</em>, more species and plastome sequences of this genus need to be included.</p>
<p id="Par31">Importantly, Zhai et al. [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] identified an inversion between <em>accD</em> and <em>clpP</em>1 in<em> Aquilegia</em> and its allies (<em>Semiaquilegia</em> Makino and <em>Urophysa</em> Ulbr.), as compared to other genera in Ranunculaceae. However, only two <em>Aquilegia</em> species (<em>Aquilegia coerulea</em> E. James and <em>Aquilegia ecalcarata</em> Eastw.), as well as one species each of <em>Semiaquilegia</em> and <em>Urophysa</em>, were sampled by Zhai et al. [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. It is unknown whether this inversion is of phylogenetic significance and widespread in the plastomes of <em>Aquilegia</em> species, especially whether this SV is consistent or variable in <em>Aquilegia</em>.</p>
<p id="Par32">In this study, we sequenced complete plastomes of 75 <em>Aquilegia</em> species using the genome-skimming sequencing technique and performed the first comprehensively phylogenetic analysis of <em>Aquilegia</em> based on plastome data. Our aims were as follows: (1) to reconstruct a robust phylogeny for <em>Aquilegia</em>; (2) to identify SVs of the <em>Aquilegia</em> plastomes; (3) to investigate the evolutionary patterns of SVs in <em>Aquilegia</em> plastomes; (4) to identify hypervariable regions for efficient and accurate species identification in this genus. The study will enhance the understanding of the evolutionary patterns of plastome SVs in angiosperms, and will facilitate exploration and utilization of <em>Aquilegia</em> plants.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Plastome feature</h3>
<p id="Par33">The complete plastomes of 75 <em>Aquilegia</em> species and five species from <em>Dichocarpum</em> W.T. Wang &amp; P.K. Hsiao, <em>Semiaquilegia</em>, <em>Thalictrum</em> L., and <em>Urophysa</em> in Thalictroideae were assembled and annotated (Table S1). Including the plastome of <em>Semiaquilegia danxiashanensis</em> L. Wu, J.J. Zhou, Q. Zhang &amp; W.S. Deng previously reported, the plastomes of the 81 Thalictroideae species that were sampled in this study have the sizes of the plastomes from 153727 bp in <em>Dichocarpum</em> to 164138 bp in <em>Urophysa</em> (Table S2). All plastomes exhibited the typical quadripartite structure (Fig. <a href="#Fig1" class="usa-link">1</a>), which comprises a pair of IRs (25853â€“27544 bp) separated by a LSC region (83381â€“91683 bp) and an SSC region (16702â€“17598 bp). The plastome sizes of <em>Aquilegia</em> (159782â€“162480 bp), <em>Semiaquilegia</em> (160548â€“162227 bp) and <em>Urophysa</em> (164138 bp) were larger than that of other Thalictroideae genera (153727â€“156258 bp). The plastome synteny of the subfamily in terms of the gene arrangement is generally high except for in the LSC region, which shows considerable SVs (Fig. <a href="#Fig1" class="usa-link">1</a>). Based on the structural differences in the LSC region, four types (I-IV) of plastome structures were recognized in Thalictroideae. The type I is present in seven <em>Aquilegia</em> species, having an <em>ca.</em> 12807â€“12825 bp inversion (INV1) between <em>accD</em> and <em>clpP</em>1 (Fig. <a href="#Fig1" class="usa-link">1</a>a; Table S2). The type II is present in 32 <em>Aquilegia</em> species, having a pair of short repeated regions (sIRs1) at both the edges of the 12662â€“12853 bp inversion, compared to type I (Fig. <a href="#Fig1" class="usa-link">1</a>b; Table S2). The type III is present in 36 <em>Aquilegia</em> species and in <em>Urophysa</em> and <em>Semiaquilegia</em>, having longer sIRs (including sIRs1 and sIRs2) at both the edges of the 12592â€“13121 bp inversion, compared to type II (Fig. <a href="#Fig1" class="usa-link">1</a>c; Table S2). The plastome of <em>S. danxiashanensis</em> previously reported have a complete sIR1 and an incomplete sIR2, we also identified it as type III although its sIR2 is very short<em>.</em> The type IV is present in <em>Dichocarpum</em> and <em>Thalictrum</em>, without the inversion and sIR regions (Fig. <a href="#Fig1" class="usa-link">1</a>d; Table S2).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366047_12870_2025_7093_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9b01/12366047/a796700873ce/12870_2025_7093_Fig1_HTML.jpg" loading="lazy" id="MO1" height="816" width="778" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The four types of plastomes in Thalictroideae. <strong>a</strong>Â Type I has an inversion without short inverted repeat (sIR), the plastome of <em>Aquilegia aradanica</em> Shaulo &amp; Erst. <strong>b</strong>Â Type II has an inversion with sIR1, the plastome of <em>Aquilegia alpina</em> L. <strong>c</strong>Â Type III has an inversion with sIR1 and sIR2, the plastome of <em>Aquilegia jonesii</em> Parry. <strong>d</strong>Â Type IV does not have an inversion, the plastome of <em>Dichocarpum dalzielii</em> (J.R.Drumm. &amp; Hutch.) W.T. Wang &amp; P.K. Hsiao. The genes inside and outside of the circle are transcribed in clockwise and counterclockwise directions, respectively. Genes belonging to different functional groups are shown in different colors. The thick lines indicate the extent of inverted repeats (IR<sub>A</sub> and IR<sub>B</sub>) that separate the genomes into small single-copy (SSC) and large single-copy (LSC) regions. The pink thick line indicates an inversion in the LSC region</p></figcaption></figure><p id="Par34">To investigate the contraction and expansion events of plastomes of five genera in Thalictroideae, we compared the borders of the IR/SSC regions (Fig. <a href="#Fig2" class="usa-link">2</a>a). For the IR<sub>A</sub>/SSC region, <em>ycf1</em> spanned the junction region, while a pseudogene fragment (<em>Ïˆycf1</em>) was located at IR<sub>B</sub> side of the IR<sub>B</sub>/SSC region in all sampled five genera of Thalictroideae. In the SSC region, <em>Dichocarpum</em> lacks <em>rpl32</em> gene, whereas other four genera have a pseudogene (<em>Ïˆrpl32</em>). For the border of the IR/LSC regions, <em>Aquilegia</em> has six copies of <em>trnI</em><sup><em>CAU</em></sup> in the two IR regions, while other four genera have two copies of <em>trnI</em><sup><em>CAU</em></sup>. Along with the IRs expansion in <em>Dichocarpum</em> and <em>Thalictrum</em>, the boundary of the IR<sub>B</sub>/LSC moved from intergenic region of <em>rpl2</em>/<em>rpl23</em> to the coding region of <em>rps19</em>, leading to a pseudogene fragment (<em>Ïˆrps19</em>) at the IR<sub>A</sub> region (Fig. <a href="#Fig2" class="usa-link">2</a>a).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366047_12870_2025_7093_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9b01/12366047/30de3cda395a/12870_2025_7093_Fig2_HTML.jpg" loading="lazy" id="MO2" height="559" width="778" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of plastome structure in Thalictroideae. <strong>a</strong>Â The structure variations (SVs) of the large single-copy (LSC), inverted repeats (IRs), and small single-copy (SSC) region boundaries in Thalictroideae. The break in the middle of the bars indicates that other regions and genes are omitted here. The Ïˆ-symbol shows putative pseudogenes. <strong>b</strong>Â The SVs of the <em>clpP</em> gene in Thalictroideae</p></figcaption></figure><p id="Par35">For the LSC region, two copies of <em>clpP</em> (<em>clpP</em>1 and <em>clpP</em>2) and one copy of <em>trnG</em><sup><em>GCC</em></sup> (<em>trnG</em><sup><em>GCC</em></sup>2) were found in <em>Aquilegia</em>, <em>Urophysa</em>, and <em>Semiaquilegia</em>. Notably, both two copies of <em>clpP</em> are intronless, which are different in one copy of <em>clpP</em> with two introns in <em>Dichocarpum</em> and <em>Thalictrum</em> (Fig. <a href="#Fig2" class="usa-link">2</a>b). Additionally, the <em>clpP</em>1 gene of <em>Aquilegia</em> and <em>Urophysa</em> contains an additional tandem repeat region, consisting of 9 and 6 repeats of a 15 bp special motif, respectively.</p>
<p id="Par36">Final, we identified 137 predicted functional genes and three pseudogenes for the 75 plastomes of <em>Aquilegia</em>. These genes were summarized as 112 unique genes including 78 protein-coding genes, 30 transfer RNA (tRNA) genes, and four ribosomal RNA (rRNA) genes (Table S3). One protein-coding gene (<em>ycf3</em>) contains two introns, 15 genes contain a single intron, including nine protein-coding genes (<em>atpF</em>, <em>ndhA</em>, <em>ndhB</em>, <em>petB</em>, <em>petD</em>, <em>rpl2</em>, <em>rpoC1</em>, <em>rps12</em>, and <em>rps16</em>) and six tRNA genes (<em>trnA</em><sup><em>UGC</em></sup>, <em>trnG</em><sup><em>UCC</em></sup>, <em>trnI</em><sup><em>GAU</em></sup>, <em>trnK</em><sup><em>UUU</em></sup>, <em>trnL</em><sup><em>UAA</em></sup>, and <em>trnV</em><sup><em>UAC</em></sup>) in <em>Aquilegia</em> (Table S3). No function genes were found in the sIRs (including sIRs1 and sIRs2) across different species in <em>Aquilegia</em>. The majority of the protein-coding genes (74/78) of <em>Aquilegia</em> contain standard AUG as the initiator codon, three genes (<em>ndhD</em>, <em>rps19</em>, and <em>ycf1</em>) contain GUG instead, and <em>rpl2</em> has ACG.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Phylogenetic analyses</h3>
<p id="Par37">Phylogenetic relationships of <em>Aquilegia</em> were reconstructed using maximum likelihood (ML) and Bayesian inference (BI) analyses, which generated almost identical topologies based on the complete plastome sequences (Fig. <a href="#Fig3" class="usa-link">3</a>). The monophyletic <em>Aquilegia</em> was strongly supported [bootstrap support (BS) = 99%, posterior probability (PP) = 1.0], sister to <em>Urophysa</em> with weak support (BS = 58%, PP = 0.99). <em>Aquilegia</em> includes two major monophyletic groups (I and II) and eight well-supported clades, each defined largely by geography. Clade I-1 consists of one North American species, clade I-2 includes four species from Asia (BS = 100%, PP = 1.0), and clade I-3 comprises most North American species with one nested Italian species (<em>Aquilegia nugorensis</em> Arrigoni &amp; E. Nardi) (BS = 100%, PP = 1.0). Clade II-1 consists of five Asian species (BS = 99%, PP = 1.0), clade II-2 includes 15 Asian species with one nested European species (<em>Aquilegia viscosa</em> Gouan) (BS = 93%, PP = 1.0), clade II-3 comprises six European species (BS = 68%, PP = 1.0), clade II-4 comprises six European species (BS = 96%, PP = 1.0), and clade II-5 comprises 20 European species (BS = 100%, PP = 1.0). The majority of interspecies relationships in <em>Aquilegia</em> are well resolved (Fig. <a href="#Fig3" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366047_12870_2025_7093_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9b01/12366047/a87aee04ee46/12870_2025_7093_Fig3_HTML.jpg" loading="lazy" id="MO3" height="801" width="778" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phylogenetic relationship of <em>Aquilegia</em> obtained from maximum likelihood analysis of the complete plastome dataset. The color of line is referred to the geographical distributions of <em>Aquilegia</em> species. Orange, blue and green color indicate North America, Asia and Europe, respectively. Numbers above branches are bootstrap values (BS â‰¥ 50 %) and posterior probabilities (PP â‰¥ 0.50), â€œ-â€ indicates the nodes not supported</p></figcaption></figure><p id="Par38">Seven plastid data subsets, including the CDS of 78 protein-coding genes, the 15 introns, the 117 intergenic spacer (IGS) regions, a combination of 15 introns and 117 IGS regions, the LSC region, the SSC region, and the IR region, were also used to perform phylogenetic analyses using ML and BI method. Phylogenetic analyses of seven plastid data subsets did not generate highly supported topological conflicts compared to the complete plastome dataset, except for the position of clade I (Figs. S1â€“S7). The 117-IGS and combined IGS and intron subsets recovered clade I-1 as basalmost in clade I with moderate support (BS = 82%, PP = 1.0; BS = 83%, PP = 1.0), the LSC subset recovered clade I-1 as sister to clade I-3 (BS = 79%, PP = 1.0), whereas the SSC subset recovered clade I-1 as sister to clade I-2 (BS = 98%, PP = 1.0).</p></section><section id="Sec5"><h3 class="pmc_sec_title">Evolutionary inference of SVs in LSC region</h3>
<p id="Par39">To infer the evolutionary patterns of SVs of the LSC region, we coded the four types for LSC structures in Thalictroideae: type I with an inversion, type II with an inversion and sIRs1, type III with an inversion, sIRs1, and sIRs2, and type IV without an inversions and sIRs. By comparing the equal-rates (ER), the symmetrical (SYM), and all-rates-different (ARD) likelihood models, we identified ER as best-fit model for our data (Table S4).</p>
<p id="Par40">Ancestral state reconstruction of the LSC structure is shown in Fig. <a href="#Fig4" class="usa-link">4</a>. The most recent common ancestor (MRCA) of <em>Aquilegia</em> is type III, i.e., a 12592â€“13121 bp inversion neighbored with a pair of long sIRs including sIRs1 and sIRs2 (up to 876 bp). Within clade II, the entire sIR structure has lost once in clade II-2 (type III), and the sIR2 structure has lost once for the MRCA of clades II-3, II-4, and II-5 (type II). Interestingly, sIR1 and sIR2 sequences varies greatly across <em>Aquilegia</em>, i.e., the length of all sIRs1 is 49 bp, while the lengths of the sIR2 range from 14 to 876 bp (Fig. <a href="#Fig4" class="usa-link">4</a>; Table S2).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366047_12870_2025_7093_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9b01/12366047/833bdedaaea2/12870_2025_7093_Fig4_HTML.jpg" loading="lazy" id="MO4" height="925" width="709" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Evolutionary inference of structure variation in <em>Aquilegia</em> plastomes. Colored pie charts show relative probabilities of alternative ancestral character states obtained in phytools. The arrow indicates the direction of inversion. The stacking diagram shows the length of short inverted repeat (sIR), as referred to Table S2</p></figcaption></figure><p id="Par41">To further investigate the sequence variation of the sIR regions, we selected 21 representative species of <em>Aquilegia</em>, covering all eight clades, all types of structure and all the mutations in the sIR regions (Fig. S8). Apart from the large insertions or deletions that lead to different structures in the sIR region, there also exist 11 small sequence variations in the variable sIR2 of <em>Aquilegia</em> (Fig. <a href="#Fig5" class="usa-link">5</a>), implying the unusual rich polymorphism of this region.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366047_12870_2025_7093_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9b01/12366047/ed309d629a7b/12870_2025_7093_Fig5_HTML.jpg" loading="lazy" id="MO5" height="490" width="778" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The structure variations (SVs) in the large single-copy (LSC) region of 11 <em>Aquilegia</em> plastomes. The color of branch shows the inferred state of SVs, as referred to Fig. <a href="#Fig4" class="usa-link">4</a>. sIR, short inverted repeat</p></figcaption></figure></section><section id="Sec6"><h3 class="pmc_sec_title">Identification of hypervariable regions</h3>
<p id="Par42">To identify the potential hypervariable regions in plastomes of <em>Aquilegia</em>, we calculated plastome nucleotide diversity for this genus. Nucleotide polymorphism analyses of the genes and intergenic regions consistently show that IR regions are more conserved than LSC and SSC regions (Fig. <a href="#Fig6" class="usa-link">6</a>). For genes, nucleotide diversity ranges from 0 (e.g., <em>trnH</em><sup><em>GUG</em></sup>, <em>trnQ</em><sup><em>UUG</em></sup>, and <em>trnS</em><sup><em>GCU</em></sup>) to 0.07967 (<em>infA</em>) with an average of 0.00118, whereby ten genes (i.e., <em>infA</em>, <em>trnI</em><sup><em>CAU</em></sup>3, <em>Ïˆrpl16</em>, <em>trnS</em><sup><em>GGA</em></sup>, <em>ccsA</em>, <em>atpH</em>, <em>ycf1</em>, <em>trnL</em><sup><em>CAA</em></sup>, <em>psbH</em>, and <em>ndhE</em>) have remarkably high values (Ï€ &gt;0.0017; Fig. <a href="#Fig6" class="usa-link">6</a>a; Table S5). For the exons of genes, nucleotide diversity ranges from 0 (e.g., <em>trnK</em><sup><em>UUU</em></sup>(<em>matK</em>), <em>trnG</em><sup><em>UCC</em></sup>, and <em>trnL</em><sup><em>UAA</em></sup>) to 0.07967 (<em>infA</em>) with an average of 0.00107. For the IGS regions, nucleotide diversity ranges from 0 (e.g., <em>rpoC1</em>â€“<em>rpoB</em>, <em>trnY</em><sup><em>GUA</em></sup>â€“<em>trnE</em><sup><em>UUC</em></sup>, and <em>psbD</em>â€“<em>psbC</em>) to 0.03486 (<em>rpl2</em>â€“<em>rpl23</em>) with an average of 0.00223. Ten of IGS regions have considerably high values (Ï€ &gt;0.00706), including <em>rpl2</em>â€“<em>rpl23</em>, <em>trnL</em><sup><em>CAA</em></sup>â€“<em>trnG</em><sup><em>GCC</em></sup>3, <em>trnN</em><sup><em>GUU</em></sup>1â€“<em>infA</em>, <em>trnT</em><sup><em>GGU</em></sup>â€“<em>psbD</em>, <em>trnH</em><sup><em>GUG</em></sup>â€“<em>Ïˆrpl16</em>, <em>rpl16</em>â€“<em>rps3</em>, <em>psbT</em>â€“<em>psbN</em>, <em>rpl14</em>â€“<em>rpl16</em>, <em>ccsA</em>â€“<em>ndhD</em>, and <em>rps19</em>â€“<em>rpl2</em> (Fig. <a href="#Fig6" class="usa-link">6</a>b; Table S6). After removing the short hypervariable regions (&lt; 100 bp), nine non-coding regions (<em>rpl2</em>â€“<em>rpl23</em>, <em>trnL</em><sup><em>CAA</em></sup>â€“<em>trnG</em><sup><em>GCC</em></sup>3, <em>trnN</em><sup><em>GUU</em></sup>1â€“<em>infA</em>, <em>trnT</em><sup><em>GGU</em></sup>â€“<em>psbD</em>, <em>trnH</em><sup><em>GUG</em></sup>â€“<em>Ïˆrpl16</em>, <em>rpl16</em>â€“<em>rps3</em>, <em>psbT</em>â€“<em>psbN</em>, <em>rpl14</em>â€“<em>rpl16</em>, and <em>ccsA</em>â€“<em>ndhD</em>) and one gene (<em>infA</em>) of plastomes were identified as hypervariable regions in <em>Aquilegia</em>. Phylogenetic analyses based on the ten hypervariable regions resolved the relationships among most species of this genus, and the sequence divergences of the combined ten hypervariable regions are &gt;0 between <em>Aquilegia</em> species (Fig. S9).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366047_12870_2025_7093_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9b01/12366047/b33ec2f7acb5/12870_2025_7093_Fig6_HTML.jpg" loading="lazy" id="MO6" height="588" width="778" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparison of nucleotide variability (Pi) values in <em>Aquilegia</em> plastomes. <strong>a</strong>Â Pi values among genes, <strong>b</strong>Â Pi values among intergenic spacer regions. The dot line denotes the average value. IR, inverted repeat; SSC, small single-copy; LSC, large single-copy</p></figcaption></figure></section></section><section id="Sec7"><h2 class="pmc_sec_title">Discussion</h2>
<section id="Sec8"><h3 class="pmc_sec_title">Phylogenetic relationships</h3>
<p id="Par43">All our phylogenetic analyses strongly supported <em>Aquilegia</em> as monophyletic (Figs. <a href="#Fig3" class="usa-link">3</a> and S1â€“S7), in agreement with previous studies (e.g., Bastida et al. [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]; Fior et al. [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]; Ro &amp; McPheron [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]). <em>Aquilegia</em> has undergone a recent and rapid radiation [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>, <a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>], the interspecies relationships remain ambiguous using a wide variety of molecular data [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>â€“<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Based on the complete plastome data, we discover that <em>Aquilegia</em> contains two major monophyletic groups and eight clades, largely consistent with their geographical distributions (Fig. <a href="#Fig3" class="usa-link">3</a>). We provide a well-resolved phylogenetic framework for <em>Aquilegia</em> with a more extensive sampling than in any previous study, which is a foundation for the future taxonomic revision for this genus.</p>
<p id="Par44">The phylogenetic analyses of <em>Aquilegia</em> were also conducted based on seven plastid subsets. Compared to other subsets as well as complete plastome dataset, the 78-CDS, 15-intron, and SSC subsets have a relatively weak resolution for the relationships within <em>Aquilegia</em> (Figs S1â€“2, S7). Overall, the phylogeny generated by the complete plastome dataset received higher support than those inferred from seven subsets, suggesting that the complete plastome sequences can resolve well phylogenetic relationships of taxa with recent and rapid radiation, such as <em>Aquilegia</em>. The similar cases are also found for <em>Rubus</em> L. (Rosaceae) [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] and <em>Saussurea</em> DC. (Asteraceae) [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>].</p></section><section id="Sec9"><h3 class="pmc_sec_title">Plastome feature</h3>
<p id="Par45">The contraction and expansion in IRs are considered important evolution events, which can change plastome sizes and gene contents [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>â€“<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Our results show that <em>Aquilegia</em>, <em>Urophysa</em>, and <em>Semiaquilegia</em> have one copy for each of <em>rpl2</em> and <em>rps</em><em>19</em> in the LSC regions, while <em>Dichocarpum</em> and <em>Thalictrum</em> have two copies for each of <em>rpl2</em> and <em>rps</em><em>19 </em>in the IRs (Fig. <a href="#Fig2" class="usa-link">2</a>a). These CNVs may result from the contraction of IR regions in <em>Aquilegia</em>, <em>Urophysa</em>, and <em>Semiaquilegia</em>. It is usual that CNVs caused by the expansion/contraction of IR regions. For example, one-copy loss of <em>rpl2</em> and <em>rpl23</em> was found in the IR/LSC region of <em>Myosurus</em> (Ranunculaceae) [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>] and one-copy loss of <em>rbcL</em> was identified in the IR/LSC regions of <em>Pelargonium transvaalense</em> R. Knuth (Geraniaceae) [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>].</p>
<p id="Par46">The length of IR regions of <em>Aquilegia</em> (25853â€“26834 bp) is similar to that of <em>Urophysa</em> (27544 bp), <em>Semiaquilegia</em> (26640â€“26829 bp), <em>Dichocarpum</em> (26448 bp), and <em>Thalictrum</em> (26480â€“26481 bp; Table S2). Especially, we found <em>Aquilegia</em> had six copies of <em>trnI</em><sup><em>CAU</em></sup> in the IR regions near the borders of IR<sub>A</sub>/LSC, while the other four genera in Thalictroideae only have two copies (Fig. <a href="#Fig2" class="usa-link">2</a>a). The CNVs of <em>trnI</em><sup><em>CAU</em></sup> are resulted from the small direct repeats of plastomes, instead of expansion/contraction of IR regions. These SVs might have occurred along with the expansion/contraction of IR regions because they are located near the borders of IR/LSC. The small direct repeats of plastomes in IR regions lead to an increase in the length of IRs in <em>Aquilegia.</em> Thus, the contraction of IRs in <em>Aquilegia</em> do not result in its IR regions being shorter.</p>
<p id="Par47">In fact, the size of plastomes in <em>Aquilegia</em> (159782â€“162480 bp), as well as <em>Urophysa</em> (164138 bp) and <em>Semiaquilegia</em> (160548â€“162227 bp), is bigger than that in <em>Dichocarpum</em> (153727 bp) and <em>Thalictrum</em> (156211â€“156258 bp), as they have longer LSC regions (Table S2; Fig. <a href="#Fig2" class="usa-link">2</a>). We found that <em>Aquilegia</em>, <em>Urophysa</em>, and <em>Semiaquilegia</em> have 12592â€“13121 bp inversions in LSC regions, whereas <em>Dichocarpum</em> and <em>Thalictrum</em> have 12148â€“12743bp homology region. Additionally, <em>Aquilegia</em>, <em>Semiaquilegia</em>, and <em>Urophysa</em> have two copies of sIR near the inversion, while <em>Dichocarpum</em> and <em>Thalictrum</em> have one copy of homology region. <em>Aquilegia</em>, <em>Semiaquilegia</em>, and <em>Urophysa</em> have one copy of <em>trnG</em><sup><em>GCC</em></sup> and two copies of <em>clpP</em> gene separately in the LSC regions near the inversion, while <em>Dichocarpum</em> and <em>Thalictrum</em> have one copy of <em>clpP</em> gene. These CNVs of genes in LSC regions occur along with sIR as it is located near the borders of inversion in <em>Aquilegia</em>, leading to the increase in the length of the LSC in <em>Aquilegia</em>, <em>Semiaquilegia</em>, and <em>Urophysa.</em></p>
<p id="Par48">For the CNV of <em>clpP</em> gene, the intronless <em>clpP</em> gene in <em>Aquilegia</em>, <em>Semiaquilegia</em>, and <em>Urophysa</em> have been duplicated in the LSC region, while <em>Thalictrum</em> and <em>Dichocarpum</em> with have only one copy of <em>clpP</em> with two introns (Fig. <a href="#Fig2" class="usa-link">2</a>b). Zhai et al. [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] inferred that the loss of the second intron likely occurred before the divergence of <em>Aquilegia</em>, <em>Semiaquilegia</em>, and <em>Urophysa</em>, followed by the loss of the first intron before the gene was duplicated. Additionally, our results indicate that the <em>clpP</em>1 gene contained approximately nine and six tandem repeats of a special motif in <em>Aquilegia</em> and <em>Urophysa</em>, while the other copy are without (Fig. <a href="#Fig2" class="usa-link">2</a>b), suggested that CNV play an important role in evolution of plastomes in <em>Aquilegia</em>.</p>
<p id="Par49">In total, <em>Aquilegia</em> plastomes have CNVs of genes at the boundaries of IR/SC regions and sIR regions. The small direct repeats in plastomes of <em>Aquilegia</em> inhibit contraction and shorten of IRs. The length of <em>Aquilegia</em>, <em>Semiaquilegia</em>, and <em>Urophysa</em> plastome is bigger than that in <em>Dichocarpum</em> and <em>Thalictrum</em>, influenced on the inversion event occurring in the LSC region.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Structure evolution of <em>Aquilegia</em> plastomes</h3>
<p id="Par50">All plastomes in <em>Aquilegia</em>, <em>Semiaquilegia</em>, and <em>Urophysa</em> contain an inversion in the LSC region between <em>accD</em>â€“<em>clpP</em>1 (Fig. <a href="#Fig2" class="usa-link">2</a>a). Inversion in the LSC region of the plastome has also been found in other taxa. For example, <em>Adonis sutchuenensis</em> Franch. in Ranunculaceae has an inversion between <em>rps16</em> and <em>trnT</em><sup><em>UGU</em></sup> [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. Plastome inversion has also been reported in the LSC region of <em>Lotus japonicus</em> between <em>rbcL</em>â€“<em>rps16</em> [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>], and multiple inversions have been found in the plastomes of Poales [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>].</p>
<p id="Par51">Here, we found three types of LSC structures in <em>Aquilegia</em> plastomes, which were likely generated by a inversion event of and exhibited phylogenetic signals. The ancestral structure of <em>Aquilegia</em> is type III, having a pair of longer sIRs (including sIRs1 and sIRs2) at the edge of the inversion in <em>Aquilegia</em> (Fig. <a href="#Fig4" class="usa-link">4</a>). The sIRs are shortened and lost at the edge of the inversion in Type II and Type I, respectively (Fig. <a href="#Fig1" class="usa-link">1</a>c), showing 11 small variations in the plastomes of <em>Aquilegia</em> (Fig. <a href="#Fig5" class="usa-link">5</a>). As the inversion could be probably induced by the neighbored sIR region, the large or small variations in the sIR regions (Figs. <a href="#Fig1" class="usa-link">1</a> and <a href="#Fig4" class="usa-link">4</a>) could contribute to the fixation of this inversion in the plastomes of the <em>Aquilegia</em>. Extensive repeats in the plastomes were found to be positively correlated with the observed inversions and rearrangements in Poales [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. We further found that the inversions in LSC regions were induced by repeats and possibly fixed by lose of repeats in <em>Aquilegia</em>.</p>
<p id="Par52">Inversions in the LSC regions also lead to notable variations in gene content and length of plastomes. <em>Aquilegia</em>, <em>Semiaquilegia</em>, and <em>Urophysa</em> had one extra copy of <em>trnG</em><sup><em>GCC</em></sup> and two copies of intronless <em>clpP</em> gene in the LSC regions near the inversion, while <em>Thalictrum</em> and <em>Dichocarpum</em> had only one copy of <em>clpP</em> gene with two introns in similar region. Additionally, although <em>Aquilegia</em>, <em>Semiaquilegia</em>, and <em>Urophysa</em> have longer LSC regions owing to the inversions, their total plastid lengths are thus longer than those of <em>Dichocarpum</em> and <em>Thalictrum</em> (Fig. <a href="#Fig2" class="usa-link">2</a>a). Thus, the inversions may contribute to the increased copy number of genes in near the inversions and have a strong impact on the length of plastome.</p>
<p id="Par53">In all, the LSC structure with both the inversion and the longer sIR was estimated as the ancestral state, which have evolved independently into other structures with loss of different sIR regions. The plastid SVs have phylogenetic signals in <em>Aquilegia</em>. The loss of distinct sIR regions led to notable variations in the structure, gene content, and length of the plastomes.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Hypervariable regions in plastomes of <em>Aquilegia</em>
</h3>
<p id="Par54">Highly polymorphic regions in plastomes are widely utilized as molecular markers for phylogenetic reconstruction [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>â€“<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>] and species identification [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>â€“<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. Phylogenetic analyses of <em>Aquilegia</em> based on nuclear ITS sequences and one plastid region (<em>atpB</em>â€“<em>rbcL</em>; Table S6) [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>] and ITS and two plastid regions (<em>trnK</em>â€“<em>matK</em>, <em>trnS</em>â€“<em>G</em>; Table S6) [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>] have failed to effectively distinguish infrageneric relationships due to low nucleotide variability among different species. More plastid fragments, such as <em>rpl32</em>â€“<em>trnL</em><sup><em>UAG</em></sup>, <em>trnK</em><sup><em>UUU</em></sup>â€“<em>rps16</em>, <em>psbM</em>â€“<em>trnD</em><sup><em>GUC</em></sup>, <em>psaI</em>â€“<em>accD</em>, <em>ndhJ</em>â€“<em>trnF</em>, <em>rps16</em>â€“<em>trnQ</em><sup><em>UUG</em></sup>, <em>trnD</em><sup><em>GUC</em></sup>â€“<em>trnT</em><sup><em>GGU</em></sup>, <em>trnT</em><sup><em>GGU</em></sup>â€“<em>psbD</em>, <em>trnC</em><sup><em>GCA</em></sup>â€“<em>rpoB</em>, <em>trnC</em><sup><em>GCA</em></sup>â€“<em>ycf6</em>, <em>atpH</em>â€“<em>atpI</em>, <em>psbE</em>â€“<em>petL</em>, <em>ycf6</em>â€“<em>psbM</em>, <em>trnS</em><sup><em>GUC</em></sup>â€“<em>trnG</em><sup><em>UUC</em></sup>, <em>ndhC</em>â€“<em>trnV</em><sup><em>UAC</em></sup>, <em>trnS</em><sup><em>UGA</em></sup>â€“<em>trnG</em><sup><em>GCC</em></sup>, <em>psbB</em>â€“<em>psbH</em>, <em>rps4</em>â€“<em>trnF</em><sup><em>GAA</em></sup>, <em>rps12</em>â€“<em>rpl20</em>, <em>matK</em> and intron of <em>ndhA</em>, have also been applied to analyze intrageneric relationships of <em>Aquilegia</em> and result in highly supported phylogenetic trees (Table S5â€“S6) [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Additionally, candidate molecular markers of <em>trnT</em><sup><em>GGU</em></sup>â€“<em>psbD</em> and <em>psbT</em>â€“<em>psbN</em> were identified based on 62 species of the genus by Fior et al. [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Therefore, additional data source beyond the commonly used markers are needed for phylogenetic analysis.</p>
<p id="Par55">In the present study, the whole genome and non-coding regions manifested higher sequence divergence than protein-coding genes of <em>Aquilegia</em> plastomes (Figs. <a href="#Fig3" class="usa-link">3</a> and S1-4), which was consistent with most higher plants [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>, <a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. After removing the short hypervariable regions (&lt; 100 bp), nine non-coding regions (<em>rpl2</em>â€“<em>rpl23</em>, <em>trnL</em><sup><em>CAA</em></sup>â€“<em>trnG</em><sup><em>GCC</em></sup>3, <em>trnN</em><sup><em>GUU</em></sup>1â€“<em>infA</em>, <em>trnT</em><sup><em>GGU</em></sup>â€“<em>psbD</em>, <em>trnH</em><sup><em>GUG</em></sup>â€“<em>Ïˆrpl16</em>, <em>rpl16</em>â€“<em>rps3</em>, <em>psbT</em>â€“<em>psbN</em>, <em>rpl14</em>â€“<em>rpl16</em>, and <em>ccsA</em>â€“<em>ndhD</em>) and one gene (<em>infA</em>) of plastomes were identified as hypervariable regions. Based on the ten identified hypervariable regions, the sisters of most <em>Aquilegia</em> species can be recognized and all sequence divergences between <em>Aquilegia</em> species are &gt;0 (Fig. S9), suggesting that the ten regions can serve as potential molecular markers for <em>Aquilegia</em> and have implications for molecular taxonomy, phylogeography, and genetic diversity assessment of this genus.</p></section></section><section id="Sec12"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par56">In this study, we present a well-supported phylogenetic framework for <em>Aquilegia</em> based on plastome data. We found the CNVs of genes are observed frequently for the contraction of IR and sIR regions, such as <em>rpl2</em>, <em>rps19</em>, <em>trnG</em><sup><em>GCC</em></sup>, <em>trnI</em><sup><em>CAU</em></sup>, and <em>clpP</em>, indicating its important role to the plastome evolution of <em>Aquilegia</em>. In particular, we found a common inversion in plastomes of <em>Aquilegia</em> and its allies. The inversion was divided into three distinct structure types (I-III) in the LSC region, each characterized by different short inverted repeat (sIR) regions. The type III with both the inversion and the longer sIR was inferred as the ancestral structure, which evolved into other structures with loss of different sIR regions. The inversions and sIR have phylogenetic signal and lead to notable variations in gene content and length of plastomes. We also identified potential molecular markers for species identification of this genus. This study provides new insights into the structural evolution pattern of plastome for one of the most diversified plant taxa in angiosperms.</p></section><section id="Sec13"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec14"><h3 class="pmc_sec_title">Sample sequencing, data assembly and annotation</h3>
<p id="Par57">We sampled 75 species of <em>Aquilegia</em>. Based on Ling et al. [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>], we selected six species of <em>Dichocarpum</em>, <em>Semiaquilegia</em>, <em>Thalictrum</em> and <em>Urophysa</em> in Thalictroideae as outgroups. Supplementary Table S1 documents the geographic origins and depository herbaria for all newly acquired samples. Their formal identification was undertaken by Wei Wang and Andrey S. Erst. No specific permissions or licenses were required for our collections and experiments. Total genomic DNA was extracted from about 1.5 g of silica geldried leaves of living plants or herbarium specimens using a modified cetyltrimethylammonium bromide (CTAB) method [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>] and purified using the Tiangen Isolation/Extraction/Purification Kit (Tiangen Biotech (Beijing) Co., Ltd.). Short insert of 300â€“500 bp libraries were prepared for sequencing on the Illumina HiSeq X-Ten platform. Approximately 10 GB of raw sequencing data was generated for each sample. To obtain high-quality genome sequences, raw reads were processed using Fastp v0.19.7 [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>] to remove adapter sequences and low-quality reads.</p>
<p id="Par58">The complete plastomes were assembled from the genome-skimming sequencing data using GetOrganelle v1.7.6.1 [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>] with the default parameters. The obtained plastid contigs were further visualized and scaffolded to complete circular genomes in Bandege v0.8.1 [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. Highly accurate annotation of organelle genomes was performed by using the Organellar Genome GeSeq tool [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>], with subsequent manual correction in Geneious v9.0.2 [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. Four plastomes from <em>Aquilegia barnebyi</em> Munz (GenBank accession No. <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_053820" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NC_053820</a>), <em>Aquilegia kansuensis</em> (BrÃ¼hl) Erst (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_058528" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NC_058528</a>), <em>Aquilegia rockii</em> Munz (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_046738" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NC_046738</a>), and <em>Aquilegia yangii</em> Y. Luo &amp; L. Li (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_058529" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NC_058529</a>) were used as reference sequences. The circular plastomes were visualized by using OGDRAW v1.3.1 [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>], with subsequent manual editing.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Plastome variation analysis</h3>
<p id="Par59">To assess the expansion/contraction of the IR regions, we compared the boundaries between the IR/SC and their adjacent genes for the whole plastomes of five genera in Thalictroideae by IRscope [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>], and then used the results to further manually modify the annotations.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Phylogenetic analyses</h3>
<p id="Par60">For the purpose of reconstructing the phylogenetic relationships, multiple sequence alignments of 81 complete plastomes were implemented using MAFFT v7 [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>] with standard parameters, and further adjusted manually in Geneious v8.0.4 [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. Phylogenetic analyses of <em>Aquilegia</em> were performed using ML and BI methods in RAxML v8.2.11 [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>] and MrBayes 3.2.7 [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>], respectively. The nucleotide substitution model was determined by the Akaike Information Criterion (AIC) via jModeltest 2.1.4 [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. RAxML was conducted with the GTR + Î“ substitution model, and the fast bootstrap option with 1,000 replicates. For BI analyses, two independent runs, each consisting of four Markov Chain Monte Carlo (MCMC) chains, were conducted with one tree sampled for every 1000 generations over 50 million generations. We assessed the convergence of the parameters of the two individual runs in Tracer 1.7.2 [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. A majority rule (&gt;50%) consensus tree was generated after discarding the first 25% of sampled trees as burn-in. Because molecular evolutionary rates of the different regions in plastome are diverse, analyses of phylogenetic relationships also were performed based on seven plastid subsets, including concatenation of 78 protein-coding genes, concatenation of 15 introns, concatenation of 117 IGS regions, concatenation of 15 introns and 117 IGS regions, a LSC region, an SSC region, and an IR region. Following the suggestion by Wang et al. (2014) [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>], the thresholds BS â‰¥ 70% and PP â‰¥ 0.95 were used as an indication of significant incongruence between eight datasets.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Ancestral structural variation reconstruction</h3>
<p id="Par61">We coded the different types of LSC structures as character states. We applied the fit-Discrete function in the R package â€˜geigerâ€™ to decide which likelihood models (ER, SYM, or ARD) should be used. Simulated stochastic character maps were obtained using the <em>make.simmap</em> function in the R package â€˜phytoolsâ€™ under the optimal likelihood model [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>, <a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>].</p>
<p id="Par62">In addition, we randomly selected 21 species covering all clades and all types of LSC region SVs in these clades for comparative genomic analyses. The sequences of inversion and the sIRs in LSC region were further extracted in Bandege v0.8.1 [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>] for three SV types in <em>Aquilegia</em>. To found the conserved synteny, we performed sequence alignment using MAFFT v7 [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>], and adjusted manually in Geneious v9.0.2 [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>].</p></section><section id="Sec18"><h3 class="pmc_sec_title">Identifying hypervariable regions</h3>
<p id="Par63">To identifying hypervariable regions, the sequence alignment of <em>Aquilegia</em> plastomes without outgroups was subjected to a sliding window analysis in DNAsp v6.12.03 [<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. We evaluated the nucleotide diversity (Pi) of all genes, genes without introns, and IGS regions. To evaluate the effectiveness of the ten hypervariable regions identified in our study, we performed phylogenetic analyses for the 75 sampled species of <em>Aquilegia</em> using ML and BI methods, with <em>Urophysa rockii</em> Ulbr. and <em>Semiaquilegia adoxoides</em> (DC.) Makino as outgroups.</p></section></section><section id="Sec19"><h2 class="pmc_sec_title">Supplementary Information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366047/bin/12870_2025_7093_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">12870_2025_7093_MOESM1_ESM.pdf</a><sup> (1.3MB, pdf) </sup><p>Additional file 1:Fig. S1. Phylogenetic relationships in of <em>Aquilegia</em> obtained from an maximum likelihood analysis of the 78-protein-coding gene dataset. Fig. S2. Phylogenetic relationships in of <em>Aquilegia</em> obtained from an maximum likelihood analysis of the 15-introns dataset. Fig. S3. Phylogenetic relationships in of <em>Aquilegia</em> obtained from an maximum likelihood analysis of the concatenated dataset of 117 intergenic spacer regions. Fig. S4. Phylogenetic relationships in of <em>Aquilegia</em> obtained from an maximum likelihood analysis of the concatenated dataset of 15 introns and 117 intergenic spacer regions. Fig. S5. Phylogenetic relationships in of <em>Aquilegia</em> obtained from an maximum likelihood analysis of the large single-copy region dataset. Fig. S6. Phylogenetic relationships in of <em>Aquilegia</em> obtained from an maximum likelihood analysis of the inverted repeat region dataset. Fig. S7. Phylogenetic relationships in of <em>Aquilegia</em> obtained from an maximum likelihood analysis of the small single-copy region dataset. Fig. S8. Comparison of structure variations (SVs) in the large single-copy region of 21 representative species of <em>Aquilegia. </em>Fig. S9. Phylogenetic relationships in <em>Aquilegia </em>obtained an maximum likelihood (ML) analysis of the concatenated dataset of the ten identified hypervariable regions.</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12366047/bin/12870_2025_7093_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">12870_2025_7093_MOESM2_ESM.pdf</a><sup> (380.1KB, pdf) </sup><p>Additional file 2: Table S1. Species, voucher, distribution, and GenBank accession numbers for the samples used in this study. Table S2. Summary of types of structure variations and plastome lengths of Thalictroideae. Table S3. Summary of characteristics of genes in plastomes of Thalictroideae. Table S4. Comparison of three transition models in ancestral structure variation reconstruction. Table S5. Summary of characteristics of 121 genes in plastome sequences in <em>Aquilegia</em>. Table S6. Summary of characteristics of 120 intergenic spacer (IGS) regions in plastome sequences in <em>Aquilegia</em>.</p>
</div></div></section></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>IR</dt>
<dd><p id="Par4">Inverted repeat</p></dd>
<dt>LSC</dt>
<dd><p id="Par5">Large single-copy</p></dd>
<dt>SSC</dt>
<dd><p id="Par6">Small single-copy</p></dd>
<dt>SC</dt>
<dd><p id="Par7">single-copy</p></dd>
<dt>SV</dt>
<dd><p id="Par8">Structure variation</p></dd>
<dt>PAV</dt>
<dd><p id="Par9">Presence/absence variation</p></dd>
<dt>CNV</dt>
<dd><p id="Par10">Copy number variation</p></dd>
<dt>ca.</dt>
<dd><p id="Par11">Circa</p></dd>
<dt>ITS</dt>
<dd><p id="Par12">Internal transcribed spacer</p></dd>
<dt>sIR</dt>
<dd><p id="Par13">Short repeated region</p></dd>
<dt>tRNA</dt>
<dd><p id="Par14">Transfer RNA</p></dd>
<dt>rRNA</dt>
<dd><p id="Par15">Ribosomal RNA</p></dd>
<dt>ML</dt>
<dd><p id="Par16">Maximum likelihood</p></dd>
<dt>BI</dt>
<dd><p id="Par17">Bayesian inference</p></dd>
<dt>BS</dt>
<dd><p id="Par18">Bootstrap support</p></dd>
<dt>PP</dt>
<dd><p id="Par19">Posterior probability</p></dd>
<dt>IGS</dt>
<dd><p id="Par20">Intergenic spacer</p></dd>
<dt>ER</dt>
<dd><p id="Par21">Equal-rates likelihood model</p></dd>
<dt>SYM</dt>
<dd><p id="Par22">Symmetrical likelihood model</p></dd>
<dt>ARD</dt>
<dd><p id="Par23">All-rates-different likelihood model</p></dd>
<dt>MRCA</dt>
<dd><p id="Par24">Most recent common ancestor</p></dd>
<dt>CTAB</dt>
<dd><p id="Par25">Cetyltrimethylammonium bromide</p></dd>
<dt>AIC</dt>
<dd><p id="Par26">Akaike Information Criterion</p></dd>
<dt>MCMC</dt>
<dd><p id="Par27">Markov Chain Monte Carlo</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authorsâ€™ contributions</h2>
<p>WW and KLX planned and designed the research; HWP, ASE, TVE, MVB and WW collected samples; ASE identified the species; FMY and KLX performed the experiments; FMY, WCH, HWP, YYL, ASE and KLX analyzed the data; FMY, WCH, WW, ASE and KLX wrote and revised the manuscript. All authors read and approved the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This research was partially funded by the National Key Research and Development Program of China (2024YFF1701404 and 2023YFF0805800), the National Natural Science Foundation of China (32361133549, 32170210, and 32400182), the Beijing Natural Science Foundation (5244047), Chinese Academy of Sciences Presidentâ€™s International Fellowship Initiative (2025PVA0116), Russian Science Foundation (24-44-00027) and the state assignments performed by the CSBG SB RAS (AAAA-A21-121011290024-05) to ASE and MVB, state assignments performed by the ICG SB RAS, FWNR-2025-0020, state assignments performed by the SFSCA RAS, â„– 075-00563-25-02.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All sequences in this study are available in the National Center for Biotechnology Information (NCBI) (<a href="https://www.ncbi.nlm.nih.gov/nuccore/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/nuccore/</a>), with GenBank accession numbers (<a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PV546610:PV546614%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV546610â€“PV546614</a>; <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PV564728:PV564802%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV564728â€“PV564802</a>) shown in Table S1.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par64">We complied with all relevant institutional, national and international guide lines with permissions from Institute of Botany, Chinese Academy of Sciences. No materials from animal or human were used in this research.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par65">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par66">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher's Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Fang-Min Yu, Wen-Chuang He and Andrey S. Erst contributed equally to this work.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Wei Wang, Email: wangwei1127@ibcas.ac.cn.</p>
<p>Kun-Li Xiang, Email: kunlixiang@ibcas.ac.cn.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Ruhlman TA, Jansen RK. The plastid genomes of flowering plants. In: Maliga P, editor. Chloroplast biotechnology: Methods and protocols. Totowa: Humana Press; 2014.</cite>
</li>
<li id="CR2">
<span class="label">2.</span><cite>Henriquez CL, Abdullah, Ahmed I, Carlsen MM, Zuluaga A, Croat TB, et al. Molecular evolution of chloroplast genomes in Monsteroideae (Araceae). Planta. 2020;251:72.
</cite> [<a href="https://doi.org/10.1007/s00425-020-03365-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32112137/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Henriquez%20CL,%20Abdullah,%20Ahmed%20I,%20Carlsen%20MM,%20Zuluaga%20A,%20Croat%20TB,%20et%20al.%20Molecular%20evolution%20of%20chloroplast%20genomes%20in%20Monsteroideae%20(Araceae).%20Planta.%202020;251:72." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Zhao F, Li B, Drew BT, Chen YP, Wang Q, Yu WB, et al. Leveraging plastomes for comparative analysis and phylogenomic inference within Scutellarioideae (Lamiaceae). PLoS One. 2020;15:e0232602.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0232602" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7205251/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32379799/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao%20F,%20Li%20B,%20Drew%20BT,%20Chen%20YP,%20Wang%20Q,%20Yu%20WB,%20et%20al.%20Leveraging%20plastomes%20for%20comparative%20analysis%20and%20phylogenomic%20inference%20within%20Scutellarioideae%20(Lamiaceae).%20PLoS%20One.%202020;15:e0232602." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Green BR. Chloroplast genomes of photosynthetic eukaryotes. Plant J. 2011;66:34â€“44.
</cite> [<a href="https://doi.org/10.1111/j.1365-313X.2011.04541.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21443621/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Green%20BR.%20Chloroplast%20genomes%20of%20photosynthetic%20eukaryotes.%20Plant%20J.%202011;66:34%E2%80%9344." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Wicke S, Schneeweiss GM, dePamphilis CW, MÃ¼ller KF, Quandt D. The evolution of the plastid chromosome in land plants: Gene content, gene order, gene function. Plant Mol Biol. 2011;76:273â€“97.
</cite> [<a href="https://doi.org/10.1007/s11103-011-9762-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3104136/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21424877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wicke%20S,%20Schneeweiss%20GM,%20dePamphilis%20CW,%20M%C3%BCller%20KF,%20Quandt%20D.%20The%20evolution%20of%20the%20plastid%20chromosome%20in%20land%20plants:%20Gene%20content,%20gene%20order,%20gene%20function.%20Plant%20Mol%20Biol.%202011;76:273%E2%80%9397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Sun YX, Moore MJ, Meng AP, Soltis PS, Soltis DE, Li JQ, et al. Complete plastid genome sequencing of Trochodendraceae reveals a significant expansion of the inverted repeat and suggests a Paleogene divergence between the two extant species. PLoS One. 2013;8: e60429.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0060429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3618518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23577110/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun%20YX,%20Moore%20MJ,%20Meng%20AP,%20Soltis%20PS,%20Soltis%20DE,%20Li%20JQ,%20et%20al.%20Complete%20plastid%20genome%20sequencing%20of%20Trochodendraceae%20reveals%20a%20significant%20expansion%20of%20the%20inverted%20repeat%20and%20suggests%20a%20Paleogene%20divergence%20between%20the%20two%20extant%20species.%20PLoS%20One.%202013;8:%20e60429." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Graham SW, Lam VKY, Merckx VSFT. Plastomes on the edge: The evolutionary breakdown of mycoheterotroph plastid genomes. New Phytol. 2017;214:48â€“55.
</cite> [<a href="https://doi.org/10.1111/nph.14398" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28067952/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Graham%20SW,%20Lam%20VKY,%20Merckx%20VSFT.%20Plastomes%20on%20the%20edge:%20The%20evolutionary%20breakdown%20of%20mycoheterotroph%20plastid%20genomes.%20New%20Phytol.%202017;214:48%E2%80%9355." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Peng HW, Lian L, Zhang J, Erst AS, Wang W. Phylogenomics, plastome degradation and mycoheterotrophy evolution of Neottieae (Orchidaceae), with emphasis on the systematic position and Loess Plateau-Changbai Mountains disjunction of <em>Diplandrorchis</em>. BMC Plant Biol. 2022;22:507.
</cite> [<a href="https://doi.org/10.1186/s12870-022-03906-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9624021/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36316655/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peng%20HW,%20Lian%20L,%20Zhang%20J,%20Erst%20AS,%20Wang%20W.%20Phylogenomics,%20plastome%20degradation%20and%20mycoheterotrophy%20evolution%20of%20Neottieae%20(Orchidaceae),%20with%20emphasis%20on%20the%20systematic%20position%20and%20Loess%20Plateau-Changbai%20Mountains%20disjunction%20of%20Diplandrorchis.%20BMC%20Plant%20Biol.%202022;22:507." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Edlund M, Anderson BM, Su HJ, Robison T, Caraballo-Ortiz MA, Der JP, et al. Plastome evolution in Santalales involves relaxed selection prior to loss of <em>ndh</em> genes and major boundary shifts of the inverted repeat. Ann Bot. 2025;135:515â€“30.
</cite> [<a href="https://doi.org/10.1093/aob/mcae145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11897430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39213003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Edlund%20M,%20Anderson%20BM,%20Su%20HJ,%20Robison%20T,%20Caraballo-Ortiz%20MA,%20Der%20JP,%20et%20al.%20Plastome%20evolution%20in%20Santalales%20involves%20relaxed%20selection%20prior%20to%20loss%20of%20ndh%20genes%20and%20major%20boundary%20shifts%20of%20the%20inverted%20repeat.%20Ann%20Bot.%202025;135:515%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Li CJ, Xie XT, Liu HX, Wang RN, Li DZ. Plastome evolution in the East Asian <em>lobelias</em> (Lobelioideae) using phylogenomic and comparative analyses. Front Plant Sci. 2023;14:1144406.
</cite> [<a href="https://doi.org/10.3389/fpls.2023.1144406" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10102522/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37063184/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20CJ,%20Xie%20XT,%20Liu%20HX,%20Wang%20RN,%20Li%20DZ.%20Plastome%20evolution%20in%20the%20East%20Asian%20lobelias%20(Lobelioideae)%20using%20phylogenomic%20and%20comparative%20analyses.%20Front%20Plant%20Sci.%202023;14:1144406." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Weng M, Ruhlman TA, Jansen RK. Expansion of inverted repeat does not decrease substitution rates in <em>Pelargonium</em> plastid genomes. New Phytol. 2017;214:842â€“51.
</cite> [<a href="https://doi.org/10.1111/nph.14375" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27991660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weng%20M,%20Ruhlman%20TA,%20Jansen%20RK.%20Expansion%20of%20inverted%20repeat%20does%20not%20decrease%20substitution%20rates%20in%20Pelargonium%20plastid%20genomes.%20New%20Phytol.%202017;214:842%E2%80%9351." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Xiang KL, Mao W, Peng HW, Erst AS, Yang YX, He WC, et al. Organization, phylogenetic marker exploitation, and gene evolution in the plastome of <em>Thalictrum</em> (Ranunculaceae). Front Plant Sci. 2022;13: 897843.
</cite> [<a href="https://doi.org/10.3389/fpls.2022.897843" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9166237/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35668810/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xiang%20KL,%20Mao%20W,%20Peng%20HW,%20Erst%20AS,%20Yang%20YX,%20He%20WC,%20et%20al.%20Organization,%20phylogenetic%20marker%20exploitation,%20and%20gene%20evolution%20in%20the%20plastome%20of%20Thalictrum%20(Ranunculaceae).%20Front%20Plant%20Sci.%202022;13:%20897843." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Martin G, Baurens FC, Cardi C, Aury JM, Dâ€™Hont A. The complete chloroplast genome of Banana (<em>Musa acuminata</em>, Zingiberales): Insight into plastid monocotyledon evolution. PLoS One. 2013;8:e67350.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0067350" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3696114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23840670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martin%20G,%20Baurens%20FC,%20Cardi%20C,%20Aury%20JM,%20D%E2%80%99Hont%20A.%20The%20complete%20chloroplast%20genome%20of%20Banana%20(Musa%20acuminata,%20Zingiberales):%20Insight%20into%20plastid%20monocotyledon%20evolution.%20PLoS%20One.%202013;8:e67350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Blazier JC, Ruhlman TA, Weng ML, Rehman SK, Sabir JSM, Jansen RK. Divergence of RNA polymerase Î± subunits in angiosperm plastid genomes is mediated by genomic rearrangement. Sci Rep. 2016;6:24595.
</cite> [<a href="https://doi.org/10.1038/srep24595" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4834550/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27087667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Blazier%20JC,%20Ruhlman%20TA,%20Weng%20ML,%20Rehman%20SK,%20Sabir%20JSM,%20Jansen%20RK.%20Divergence%20of%20RNA%20polymerase%20%CE%B1%20subunits%20in%20angiosperm%20plastid%20genomes%20is%20mediated%20by%20genomic%20rearrangement.%20Sci%20Rep.%202016;6:24595." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Long J, He WC, Peng HW, Erst AS, Wang W, Xiang KL. Comparative plastome analysis of the sister genera <em>Ceratocephala</em> and <em>Myosurus</em> (Ranunculaceae) reveals signals of adaptive evolution to arid and aquatic environments. BMC Plant Biol. 2024;24: 202.
</cite> [<a href="https://doi.org/10.1186/s12870-024-04891-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10953084/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38509479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Long%20J,%20He%20WC,%20Peng%20HW,%20Erst%20AS,%20Wang%20W,%20Xiang%20KL.%20Comparative%20plastome%20analysis%20of%20the%20sister%20genera%20Ceratocephala%20and%20Myosurus%20(Ranunculaceae)%20reveals%20signals%20of%20adaptive%20evolution%20to%20arid%20and%20aquatic%20environments.%20BMC%20Plant%20Biol.%202024;24:%20202." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Zhai W, Duan X, Zhang R, Guo C, Li L, Xu G, et al. Chloroplast genomic data provide new and robust insights into the phylogeny and evolution of the Ranunculaceae. Mol Phylogenet Evol. 2019;135:12â€“21.
</cite> [<a href="https://doi.org/10.1016/j.ympev.2019.02.024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30826488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhai%20W,%20Duan%20X,%20Zhang%20R,%20Guo%20C,%20Li%20L,%20Xu%20G,%20et%20al.%20Chloroplast%20genomic%20data%20provide%20new%20and%20robust%20insights%20into%20the%20phylogeny%20and%20evolution%20of%20the%20Ranunculaceae.%20Mol%20Phylogenet%20Evol.%202019;135:12%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Wu H, Li DZ, Ma PF. Unprecedented variation pattern of plastid genomes and the potential role in adaptive evolution in Poales. BMC Biol. 2024;22:97.
</cite> [<a href="https://doi.org/10.1186/s12915-024-01890-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11057118/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38679718/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20H,%20Li%20DZ,%20Ma%20PF.%20Unprecedented%20variation%20pattern%20of%20plastid%20genomes%20and%20the%20potential%20role%20in%20adaptive%20evolution%20in%20Poales.%20BMC%20Biol.%202024;22:97." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Kato T, Kaneko T, Sato S, Nakamura Y, Tabata S. Complete structure of the chloroplast genome of a legume, <em>Lotus japonicus</em>. DNA Res. 2000;7:323â€“30.
</cite> [<a href="https://doi.org/10.1093/dnares/7.6.323" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11214967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kato%20T,%20Kaneko%20T,%20Sato%20S,%20Nakamura%20Y,%20Tabata%20S.%20Complete%20structure%20of%20the%20chloroplast%20genome%20of%20a%20legume,%20Lotus%20japonicus.%20DNA%20Res.%202000;7:323%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Park S, Park S. Large-scale phylogenomics reveals ancient introgression in Asian <em>Hepatica</em> and new insights into the origin of the insular endemic <em>Hepatica maxima</em>. Sci Rep. 2020;10:16288.
</cite> [<a href="https://doi.org/10.1038/s41598-020-73397-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7529770/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33004955/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Park%20S,%20Park%20S.%20Large-scale%20phylogenomics%20reveals%20ancient%20introgression%20in%20Asian%20Hepatica%20and%20new%20insights%20into%20the%20origin%20of%20the%20insular%20endemic%20Hepatica%20maxima.%20Sci%20Rep.%202020;10:16288." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Johansson JT. There large inversions in the chloroplast genomes and one loss of the chloroplast gene <em>rpsl6</em> suggest an early evolutionary split in the genus <em>Adonis</em> (Ranunculaceae). Plant Syst Evol. 1999;218:133â€“43.</cite> [<a href="https://scholar.google.com/scholar_lookup?Johansson%20JT.%20There%20large%20inversions%20in%20the%20chloroplast%20genomes%20and%20one%20loss%20of%20the%20chloroplast%20gene%20rpsl6%20suggest%20an%20early%20evolutionary%20split%20in%20the%20genus%20Adonis%20(Ranunculaceae).%20Plant%20Syst%20Evol.%201999;218:133%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Munz PA. <em>Aquilegia</em>â€¯: The cultivated and wild columbines. Ithaca: Gentes Herbarium; 1946. p. 7.</cite> [<a href="https://scholar.google.com/scholar_lookup?Munz%20PA.%20Aquilegia%E2%80%AF:%20The%20cultivated%20and%20wild%20columbines.%20Ithaca:%20Gentes%20Herbarium;%201946.%20p.%207." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Tamura M. <em>Aquilegia</em>. In: In Die Naturlichen Pflanzenfamilien, vol. 17a. Berlin: Duncker and Humblot; 1995. p. 223â€“497.</cite> [<a href="https://scholar.google.com/scholar_lookup?Tamura%20M.%20Aquilegia.%20In:%20In%20Die%20Naturlichen%20Pflanzenfamilien,%20vol.%2017a.%20Berlin:%20Duncker%20and%20Humblot;%201995.%20p.%20223%E2%80%93497." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Xiao PG. Aquilegia. In: Flora Reipublicae Popularis Sinicae, vol. 27. Beijing: Science Press; 1979. p. 491â€“501.</cite> [<a href="https://scholar.google.com/scholar_lookup?Xiao%20PG.%20Aquilegia.%20In:%20Flora%20Reipublicae%20Popularis%20Sinicae,%20vol.%2027.%20Beijing:%20Science%20Press;%201979.%20p.%20491%E2%80%93501." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Filiault DL, Ballerini ES, MandÃ¡kovÃ¡ T, AkÃ¶z G, Derieg NJ, Schmutz J, et al. The <em>aquilegia</em> genome provides insight into adaptive radiation and reveals an extraordinarily polymorphic chromosome with a unique history. eLife. 2018;7:e36426.
</cite> [<a href="https://doi.org/10.7554/eLife.36426" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6255393/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30325307/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Filiault%20DL,%20Ballerini%20ES,%20Mand%C3%A1kov%C3%A1%20T,%20Ak%C3%B6z%20G,%20Derieg%20NJ,%20Schmutz%20J,%20et%20al.%20The%20aquilegia%20genome%20provides%20insight%20into%20adaptive%20radiation%20and%20reveals%20an%20extraordinarily%20polymorphic%20chromosome%20with%20a%20unique%20history.%20eLife.%202018;7:e36426." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Kramer EM. <em>Aquilegia</em>â€¯: A new model for plant development, ecology, and evolution. Annu Rev Plant Biol. 2009;60:261â€“77.
</cite> [<a href="https://doi.org/10.1146/annurev.arplant.043008.092051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19575583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kramer%20EM.%20Aquilegia%E2%80%AF:%20A%20new%20model%20for%20plant%20development,%20ecology,%20and%20evolution.%20Annu%20Rev%20Plant%20Biol.%202009;60:261%E2%80%9377." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Shi T, Chen J. A reappraisal of the phylogenetic placement of the <em>Aquilegia</em> whole-genome duplication. Genome Biol. 2020;21:295.
</cite> [<a href="https://doi.org/10.1186/s13059-020-02212-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7722308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33292440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shi%20T,%20Chen%20J.%20A%20reappraisal%20of%20the%20phylogenetic%20placement%20of%20the%20Aquilegia%20whole-genome%20duplication.%20Genome%20Biol.%202020;21:295." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Xie J, Zhao H, Li K, Zhang R, Jiang Y, Wang M, et al. A chromosome-scale reference genome of <em>Aquilegia oxysepala</em> var. <em>kansuensis</em>. Hortic Res. 2020;7: 113.
</cite> [<a href="https://doi.org/10.1038/s41438-020-0328-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7326910/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32637141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie%20J,%20Zhao%20H,%20Li%20K,%20Zhang%20R,%20Jiang%20Y,%20Wang%20M,%20et%20al.%20A%20chromosome-scale%20reference%20genome%20of%20Aquilegia%20oxysepala%20var.%20kansuensis.%20Hortic%20Res.%202020;7:%20113." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Hodges SA, Arnold ML. Spurring plant diversification: Are floral nectar spurs a key innovation? Proc R Soc Lond B Biol Sci. 1995;262:343â€“8.</cite> [<a href="https://scholar.google.com/scholar_lookup?Hodges%20SA,%20Arnold%20ML.%20Spurring%20plant%20diversification:%20Are%20floral%20nectar%20spurs%20a%20key%20innovation?%20Proc%20R%20Soc%20Lond%20B%20Biol%20Sci.%201995;262:343%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Hodges SA, Arnold ML. Columbines: A geographically widespread species flock. Proc Natl Acad Sci U S A. 1994;91:5129â€“32.
</cite> [<a href="https://doi.org/10.1073/pnas.91.11.5129" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC43945/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8197196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hodges%20SA,%20Arnold%20ML.%20Columbines:%20A%20geographically%20widespread%20species%20flock.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%201994;91:5129%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Bastida JM, AlcÃ¡ntara JM, Rey PJ, Vargas P, Herrera CM. Extended phylogeny of <em>Aquilegia</em>: The biogeographical and ecological patterns of two simultaneous but contrasting radiations. Plant Syst Evol. 2010;284:171â€“85.</cite> [<a href="https://scholar.google.com/scholar_lookup?Bastida%20JM,%20Alc%C3%A1ntara%20JM,%20Rey%20PJ,%20Vargas%20P,%20Herrera%20CM.%20Extended%20phylogeny%20of%20Aquilegia:%20The%20biogeographical%20and%20ecological%20patterns%20of%20two%20simultaneous%20but%20contrasting%20radiations.%20Plant%20Syst%20Evol.%202010;284:171%E2%80%9385." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Fior S, Li M, Oxelman B, Viola R, Hodges SA, Ometto L, et al. Spatiotemporal reconstruction of the <em>Aquilegia</em> rapid radiation through next-generation sequencing of rapidly evolving cpDNA regions. New Phytol. 2013;198:579â€“92.
</cite> [<a href="https://doi.org/10.1111/nph.12163" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23379348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fior%20S,%20Li%20M,%20Oxelman%20B,%20Viola%20R,%20Hodges%20SA,%20Ometto%20L,%20et%20al.%20Spatiotemporal%20reconstruction%20of%20the%20Aquilegia%20rapid%20radiation%20through%20next-generation%20sequencing%20of%20rapidly%20evolving%20cpDNA%20regions.%20New%20Phytol.%202013;198:579%E2%80%9392." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Zhang W, Wang H, Dong J, Zhang T, Xiao H. Comparative chloroplast genomes and phylogenetic analysis of <em>Aquilegia</em>. Appl Plant Sci. 2021;9: e11412.
</cite> [<a href="https://doi.org/10.1002/aps3.11412" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8027367/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33854846/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20W,%20Wang%20H,%20Dong%20J,%20Zhang%20T,%20Xiao%20H.%20Comparative%20chloroplast%20genomes%20and%20phylogenetic%20analysis%20of%20Aquilegia.%20Appl%20Plant%20Sci.%202021;9:%20e11412." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Ro KE, McPheron BA. Molecular phylogeny of the <em>Aquilegia</em> group (Ranunculaceae) based on internal transcribed spacers and 5.8S nuclear ribosomal DNA. Biochem Syst Ecol. 1997;25:445â€“61.</cite> [<a href="https://scholar.google.com/scholar_lookup?Ro%20KE,%20McPheron%20BA.%20Molecular%20phylogeny%20of%20the%20Aquilegia%20group%20(Ranunculaceae)%20based%20on%20internal%20transcribed%20spacers%20and%205.8S%20nuclear%20ribosomal%20DNA.%20Biochem%20Syst%20Ecol.%201997;25:445%E2%80%9361." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Whittall JB, Hodges SA. Pollinator shifts drive increasingly long nectar spurs in columbine flowers. Nature. 2007;447:706â€“9.
</cite> [<a href="https://doi.org/10.1038/nature05857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17554306/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Whittall%20JB,%20Hodges%20SA.%20Pollinator%20shifts%20drive%20increasingly%20long%20nectar%20spurs%20in%20columbine%20flowers.%20Nature.%202007;447:706%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Xu Y, Li Y, Chen Y, Wang L, Xue B, Zhang X, et al. Comparative analysis of complete chloroplast genomes of in China: Hypervariable regions and phylogenetic relationships. Genes. 2024;15:716.
</cite> [<a href="https://doi.org/10.3390/genes15060716" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11202638/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38927652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20Y,%20Li%20Y,%20Chen%20Y,%20Wang%20L,%20Xue%20B,%20Zhang%20X,%20et%20al.%20Comparative%20analysis%20of%20complete%20chloroplast%20genomes%20of%20in%20China:%20Hypervariable%20regions%20and%20phylogenetic%20relationships.%20Genes.%202024;15:716." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Xu LS, Herrando-Moraira S, Susanna A, Galbany-Casals M, Chen YS. Phylogeny, origin and dispersal of <em>Saussurea</em> (Asteraceae) based on chloroplast genome data. Mol Phylogenet Evol. 2019;141: 106613.
</cite> [<a href="https://doi.org/10.1016/j.ympev.2019.106613" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31525421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20LS,%20Herrando-Moraira%20S,%20Susanna%20A,%20Galbany-Casals%20M,%20Chen%20YS.%20Phylogeny,%20origin%20and%20dispersal%20of%20Saussurea%20(Asteraceae)%20based%20on%20chloroplast%20genome%20data.%20Mol%20Phylogenet%20Evol.%202019;141:%20106613." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Menezes APA, Resende-Moreira LC, Buzatti RSO, Nazareno AG, Carlsen M, Lobo FP, et al. Chloroplast genomes of species (Malpighiaceae): Comparative analysis and screening of high divergence sequences. Sci Rep. 2018;8:2210.
</cite> [<a href="https://doi.org/10.1038/s41598-018-20189-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5797077/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29396532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Menezes%20APA,%20Resende-Moreira%20LC,%20Buzatti%20RSO,%20Nazareno%20AG,%20Carlsen%20M,%20Lobo%20FP,%20et%20al.%20Chloroplast%20genomes%20of%20species%20(Malpighiaceae):%20Comparative%20analysis%20and%20screening%20of%20high%20divergence%20sequences.%20Sci%20Rep.%202018;8:2210." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Abdullah, Mehmood F, Shahzadi I, Waseem S, Mirza B, Ahmed I, et al. Chloroplast genome of <em>Hibiscus rosa-sinensis</em> (Malvaceae): Comparative analyses and identification of mutational hotspots. Genomics. 2020;112:581â€“91.
</cite> [<a href="https://doi.org/10.1016/j.ygeno.2019.04.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30998967/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abdullah,%20Mehmood%20F,%20Shahzadi%20I,%20Waseem%20S,%20Mirza%20B,%20Ahmed%20I,%20et%20al.%20Chloroplast%20genome%20of%20Hibiscus%20rosa-sinensis%20(Malvaceae):%20Comparative%20analyses%20and%20identification%20of%20mutational%20hotspots.%20Genomics.%202020;112:581%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Kim KJ, Lee HL. Complete chloroplast genome sequences from Korean ginseng (<em>Panax schinseng</em> Nees) and comparative analysis of sequence evolution among 17 vascular plants. DNA Res. 2004;11:247â€“61.
</cite> [<a href="https://doi.org/10.1093/dnares/11.4.247" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15500250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20KJ,%20Lee%20HL.%20Complete%20chloroplast%20genome%20sequences%20from%20Korean%20ginseng%20(Panax%20schinseng%20Nees)%20and%20comparative%20analysis%20of%20sequence%20evolution%20among%2017%20vascular%20plants.%20DNA%20Res.%202004;11:247%E2%80%9361." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Michelangeli FA, Davis JI, Stevenson DWm. Phylogenetic relationships among Poaceae and related families as inferred from morphology, inversions in the plastid genome, and sequence data from the mitochondrial and plastid genomes. Am J Bot. 2003;90:93â€“106.
</cite> [<a href="https://doi.org/10.3732/ajb.90.1.93" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21659084/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Michelangeli%20FA,%20Davis%20JI,%20Stevenson%20DWm.%20Phylogenetic%20relationships%20among%20Poaceae%20and%20related%20families%20as%20inferred%20from%20morphology,%20inversions%20in%20the%20plastid%20genome,%20and%20sequence%20data%20from%20the%20mitochondrial%20and%20plastid%20genomes.%20Am%20J%20Bot.%202003;90:93%E2%80%93106." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Li X, Li Y, Zang M, Li M, Fang Y. Complete chloroplast genome sequence and phylogenetic analysis of. Int J Mol Sci. 2018;19:2443. 10.3390/ijms19082443.
</cite> [<a href="https://doi.org/10.3390/ijms19082443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6121628/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30126202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20X,%20Li%20Y,%20Zang%20M,%20Li%20M,%20Fang%20Y.%20Complete%20chloroplast%20genome%20sequence%20and%20phylogenetic%20analysis%20of.%20Int%20J%20Mol%20Sci.%202018;19:2443.%2010.3390/ijms19082443." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Liu W, Kong H, Zhou J, Fritsch P, Hao G, Gong W. Complete chloroplast genome of <em>Cercis chuniana</em> (Fabaceae) with structural and genetic comparison to six species in Caesalpinioideae. Int J Mol Sci. 2018;19:1286.
</cite> [<a href="https://doi.org/10.3390/ijms19051286" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5983592/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29693617/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20W,%20Kong%20H,%20Zhou%20J,%20Fritsch%20P,%20Hao%20G,%20Gong%20W.%20Complete%20chloroplast%20genome%20of%20Cercis%20chuniana%20(Fabaceae)%20with%20structural%20and%20genetic%20comparison%20to%20six%20species%20in%20Caesalpinioideae.%20Int%20J%20Mol%20Sci.%202018;19:1286." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Sun J, Wang Y, Liu Y, Xu C, Yuan Q, Guo L, et al. Evolutionary and phylogenetic aspects of the chloroplast genome of <em>Chaenomeles</em> species. Sci Rep. 2020;10:11466.
</cite> [<a href="https://doi.org/10.1038/s41598-020-67943-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7351712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32651417/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun%20J,%20Wang%20Y,%20Liu%20Y,%20Xu%20C,%20Yuan%20Q,%20Guo%20L,%20et%20al.%20Evolutionary%20and%20phylogenetic%20aspects%20of%20the%20chloroplast%20genome%20of%20Chaenomeles%20species.%20Sci%20Rep.%202020;10:11466." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Burke SV, Ungerer MC, Duvall MR. Investigation of mitochondrial-derived plastome sequences in the <em>Paspalum</em> lineage (Panicoideae; Poaceae). BMC Plant Biol. 2018;18:152.
</cite> [<a href="https://doi.org/10.1186/s12870-018-1379-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6091044/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30075756/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Burke%20SV,%20Ungerer%20MC,%20Duvall%20MR.%20Investigation%20of%20mitochondrial-derived%20plastome%20sequences%20in%20the%20Paspalum%20lineage%20(Panicoideae;%20Poaceae).%20BMC%20Plant%20Biol.%202018;18:152." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>CBOL Plant Working Group1, Hollingsworth PM, Forrest LL, Spouge JL, Hajibabaei M, Ratnasingham S, et al. A DNA barcode for land plants. Proc Natl Acad Sci. 2009;106:12794â€“7.
</cite> [<a href="https://doi.org/10.1073/pnas.0905845106" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2722355/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19666622/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?CBOL%20Plant%20Working%20Group1,%20Hollingsworth%20PM,%20Forrest%20LL,%20Spouge%20JL,%20Hajibabaei%20M,%20Ratnasingham%20S,%20et%20al.%20A%20DNA%20barcode%20for%20land%20plants.%20Proc%20Natl%20Acad%20Sci.%202009;106:12794%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Li X, Yang Y, Henry RJ, Rossetto M, Wang Y, Chen S. Plant DNA barcoding: From gene to genome. Biol Rev. 2015;90:157â€“66.
</cite> [<a href="https://doi.org/10.1111/brv.12104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24666563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20X,%20Yang%20Y,%20Henry%20RJ,%20Rossetto%20M,%20Wang%20Y,%20Chen%20S.%20Plant%20DNA%20barcoding:%20From%20gene%20to%20genome.%20Biol%20Rev.%202015;90:157%E2%80%9366." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Shaw J, Shafer HL, Leonard OR, Kovach MJ, Schorr M, Morris AB. Chloroplast DNA sequence utility for the lowest phylogenetic and phylogeographic inferences in angiosperms: The tortoise and the hare IV. Am J Bot. 2014;101:1987â€“2004.
</cite> [<a href="https://doi.org/10.3732/ajb.1400398" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25366863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shaw%20J,%20Shafer%20HL,%20Leonard%20OR,%20Kovach%20MJ,%20Schorr%20M,%20Morris%20AB.%20Chloroplast%20DNA%20sequence%20utility%20for%20the%20lowest%20phylogenetic%20and%20phylogeographic%20inferences%20in%20angiosperms:%20The%20tortoise%20and%20the%20hare%20IV.%20Am%20J%20Bot.%202014;101:1987%E2%80%932004." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Zhang Y, Du L, Liu A, Chen J, Wu L, Hu W, et al. The complete chloroplast genome sequences of five species: Lights into phylogenetic and taxonomic analyses. Front Plant Sci. 2016;7:306.
</cite> [<a href="https://doi.org/10.3389/fpls.2016.00306" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4791396/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27014326/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Y,%20Du%20L,%20Liu%20A,%20Chen%20J,%20Wu%20L,%20Hu%20W,%20et%20al.%20The%20complete%20chloroplast%20genome%20sequences%20of%20five%20species:%20Lights%20into%20phylogenetic%20and%20taxonomic%20analyses.%20Front%20Plant%20Sci.%202016;7:306." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Ling YY, Peng HW, Lian L, Erst AS, Xiang KL, Wang W. Out of and in East Asia: Phylogeny, biogeography and diversification of Thalictroideae (Ranunculaceae) in the Northern Hemisphere. Ann Bot. 2024;134:1251â€“62.
</cite> [<a href="https://doi.org/10.1093/aob/mcae148" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11688531/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39196797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ling%20YY,%20Peng%20HW,%20Lian%20L,%20Erst%20AS,%20Xiang%20KL,%20Wang%20W.%20Out%20of%20and%20in%20East%20Asia:%20Phylogeny,%20biogeography%20and%20diversification%20of%20Thalictroideae%20(Ranunculaceae)%20in%20the%20Northern%20Hemisphere.%20Ann%20Bot.%202024;134:1251%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Doyle JJ, Doyle JL. A rapid DNA isolation procedure for small quantities of fresh leaf tissue. Phytochem Bull. 1987;19:11â€“5.</cite> [<a href="https://scholar.google.com/scholar_lookup?Doyle%20JJ,%20Doyle%20JL.%20A%20rapid%20DNA%20isolation%20procedure%20for%20small%20quantities%20of%20fresh%20leaf%20tissue.%20Phytochem%20Bull.%201987;19:11%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Chen S, Zhou Y, Chen Y, Gu J. Fastp: An ultra-fast all-in-one FASTQ preprocessor. Bioinformatics. 2018;34:i884-90.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6129281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30423086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20S,%20Zhou%20Y,%20Chen%20Y,%20Gu%20J.%20Fastp:%20An%20ultra-fast%20all-in-one%20FASTQ%20preprocessor.%20Bioinformatics.%202018;34:i884-90." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Jin JJ, Yu WB, Yang JB, Song Y, dePamphilis CW, Yi TS, et al. Getorganelle: A fast and versatile toolkit for accurate de novo assembly of organelle genomes. Genome Biol. 2020;21:241.
</cite> [<a href="https://doi.org/10.1186/s13059-020-02154-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7488116/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32912315/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jin%20JJ,%20Yu%20WB,%20Yang%20JB,%20Song%20Y,%20dePamphilis%20CW,%20Yi%20TS,%20et%20al.%20Getorganelle:%20A%20fast%20and%20versatile%20toolkit%20for%20accurate%20de%20novo%20assembly%20of%20organelle%20genomes.%20Genome%20Biol.%202020;21:241." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Wick RR, Schultz MB, Zobel J, Holt KE. Bandage: Interactive visualization of <em>de novo</em> genome assemblies. Bioinformatics. 2015;31:3350â€“2.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv383" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4595904/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26099265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wick%20RR,%20Schultz%20MB,%20Zobel%20J,%20Holt%20KE.%20Bandage:%20Interactive%20visualization%20of%20de%20novo%20genome%20assemblies.%20Bioinformatics.%202015;31:3350%E2%80%932." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Tillich M, Lehwark P, Pellizzer T, Ulbricht-Jones ES, Fischer A, Bock R, et al. GeSeq â€“ Versatile and accurate annotation of organelle genomes. Nucleic Acids Res. 2017;45:W6-11.
</cite> [<a href="https://doi.org/10.1093/nar/gkx391" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5570176/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28486635/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tillich%20M,%20Lehwark%20P,%20Pellizzer%20T,%20Ulbricht-Jones%20ES,%20Fischer%20A,%20Bock%20R,%20et%20al.%20GeSeq%20%E2%80%93%20Versatile%20and%20accurate%20annotation%20of%20organelle%20genomes.%20Nucleic%20Acids%20Res.%202017;45:W6-11." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Kearse M, Moir R, Wilson A, Stones-Havas S, Cheung M, Sturrock S, et al. Geneious Basic: An integrated and extendable desktop software platform for the organization and analysis of sequence data. Bioinformatics. 2012;28:1647â€“9.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts199" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3371832/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22543367/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kearse%20M,%20Moir%20R,%20Wilson%20A,%20Stones-Havas%20S,%20Cheung%20M,%20Sturrock%20S,%20et%20al.%20Geneious%20Basic:%20An%20integrated%20and%20extendable%20desktop%20software%20platform%20for%20the%20organization%20and%20analysis%20of%20sequence%20data.%20Bioinformatics.%202012;28:1647%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Greiner S, Lehwark P, Bock R. Organellar Genome DRAW (OGDRAW) version 1.3.1: Expanded toolkit for the graphical visualization of organellar genomes. Nucleic Acids Res. 2019;47:W59-64.
</cite> [<a href="https://doi.org/10.1093/nar/gkz238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6602502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30949694/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Greiner%20S,%20Lehwark%20P,%20Bock%20R.%20Organellar%20Genome%20DRAW%20(OGDRAW)%20version%201.3.1:%20Expanded%20toolkit%20for%20the%20graphical%20visualization%20of%20organellar%20genomes.%20Nucleic%20Acids%20Res.%202019;47:W59-64." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Amiryousefi A, HyvÃ¶nen J, Poczai P. IRscope: An online program to visualize the junction sites of chloroplast genomes. Bioinformatics. 2018;34:3030â€“1.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty220" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29659705/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Amiryousefi%20A,%20Hyv%C3%B6nen%20J,%20Poczai%20P.%20IRscope:%20An%20online%20program%20to%20visualize%20the%20junction%20sites%20of%20chloroplast%20genomes.%20Bioinformatics.%202018;34:3030%E2%80%931." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Katoh K, Toh H. Parallelization of the MAFFT multiple sequence alignment program. Bioinformatics. 2010;26:1899â€“900.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btq224" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2905546/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20427515/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Katoh%20K,%20Toh%20H.%20Parallelization%20of%20the%20MAFFT%20multiple%20sequence%20alignment%20program.%20Bioinformatics.%202010;26:1899%E2%80%93900." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Stamatakis A. RAxML version 8: A tool for phylogenetic analysis and post-analysis of large phylogenies. Bioinformatics. 2014;30:1312â€“3.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3998144/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24451623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stamatakis%20A.%20RAxML%20version%208:%20A%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies.%20Bioinformatics.%202014;30:1312%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Ronquist F, Teslenko M, Mark PVD, Ayres DL, Darling A, Hohna S, et al. MrBayes 3.2: Efficient Bayesian phylogenetic inference and model choice across a large model space. Syst Biol. 2012;61:539â€“42.
</cite> [<a href="https://doi.org/10.1093/sysbio/sys029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3329765/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22357727/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ronquist%20F,%20Teslenko%20M,%20Mark%20PVD,%20Ayres%20DL,%20Darling%20A,%20Hohna%20S,%20et%20al.%20MrBayes%203.2:%20Efficient%20Bayesian%20phylogenetic%20inference%20and%20model%20choice%20across%20a%20large%20model%20space.%20Syst%20Biol.%202012;61:539%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Posada D. jModelTest: Phylogenetic model averaging. Mol Biol Evol. 2008;25:1253â€“6.
</cite> [<a href="https://doi.org/10.1093/molbev/msn083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18397919/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Posada%20D.%20jModelTest:%20Phylogenetic%20model%20averaging.%20Mol%20Biol%20Evol.%202008;25:1253%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Rambaut A, Drummond AJ, Xie D, Baele G, Suchard MA. Posterior summarization in Bayesian phylogenetics using Tracer 1.7. Syst Biol. 2018;67:901â€“4.
</cite> [<a href="https://doi.org/10.1093/sysbio/syy032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6101584/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29718447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rambaut%20A,%20Drummond%20AJ,%20Xie%20D,%20Baele%20G,%20Suchard%20MA.%20Posterior%20summarization%20in%20Bayesian%20phylogenetics%20using%20Tracer%201.7.%20Syst%20Biol.%202018;67:901%E2%80%934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Wang W, Li H, Chen Z. Analysis of plastid and nuclear DNA data in plant phylogenetics-evaluation and improvement. Sci China Life Sci. 2014;57:280â€“6.
</cite> [<a href="https://doi.org/10.1007/s11427-014-4620-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24554473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20W,%20Li%20H,%20Chen%20Z.%20Analysis%20of%20plastid%20and%20nuclear%20DNA%20data%20in%20plant%20phylogenetics-evaluation%20and%20improvement.%20Sci%20China%20Life%20Sci.%202014;57:280%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Pennell MW, Eastman JM, Slater GJ, Brown JW, Uyeda JC, FitzJohn RG, et al. Geiger v2.0: An expanded suite of methods for fitting macroevolutionary models to phylogenetic trees. Bioinformatics. 2014;30:2216â€“8.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu181" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24728855/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pennell%20MW,%20Eastman%20JM,%20Slater%20GJ,%20Brown%20JW,%20Uyeda%20JC,%20FitzJohn%20RG,%20et%20al.%20Geiger%20v2.0:%20An%20expanded%20suite%20of%20methods%20for%20fitting%20macroevolutionary%20models%20to%20phylogenetic%20trees.%20Bioinformatics.%202014;30:2216%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Revell LJ. Phytools: an R package for phylogenetic comparative biology (and other things). Methods Ecol Evol. 2012;3:217â€“23.</cite> [<a href="https://scholar.google.com/scholar_lookup?Revell%20LJ.%20Phytools:%20an%20R%20package%20for%20phylogenetic%20comparative%20biology%20(and%20other%20things).%20Methods%20Ecol%20Evol.%202012;3:217%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Rozas J, Ferrer-Mata A, SÃ¡nchez-DelBarrio JC, Guirao-Rico S, Librado P, Ramos-Onsins SE, et al. DnaSP 6: DNA sequence polymorphism analysis of large data sets. Mol Biol Evol. 2017;34:3299â€“302.
</cite> [<a href="https://doi.org/10.1093/molbev/msx248" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29029172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rozas%20J,%20Ferrer-Mata%20A,%20S%C3%A1nchez-DelBarrio%20JC,%20Guirao-Rico%20S,%20Librado%20P,%20Ramos-Onsins%20SE,%20et%20al.%20DnaSP%206:%20DNA%20sequence%20polymorphism%20analysis%20of%20large%20data%20sets.%20Mol%20Biol%20Evol.%202017;34:3299%E2%80%93302." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366047/bin/12870_2025_7093_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">12870_2025_7093_MOESM1_ESM.pdf</a><sup> (1.3MB, pdf) </sup><p>Additional file 1:Fig. S1. Phylogenetic relationships in of <em>Aquilegia</em> obtained from an maximum likelihood analysis of the 78-protein-coding gene dataset. Fig. S2. Phylogenetic relationships in of <em>Aquilegia</em> obtained from an maximum likelihood analysis of the 15-introns dataset. Fig. S3. Phylogenetic relationships in of <em>Aquilegia</em> obtained from an maximum likelihood analysis of the concatenated dataset of 117 intergenic spacer regions. Fig. S4. Phylogenetic relationships in of <em>Aquilegia</em> obtained from an maximum likelihood analysis of the concatenated dataset of 15 introns and 117 intergenic spacer regions. Fig. S5. Phylogenetic relationships in of <em>Aquilegia</em> obtained from an maximum likelihood analysis of the large single-copy region dataset. Fig. S6. Phylogenetic relationships in of <em>Aquilegia</em> obtained from an maximum likelihood analysis of the inverted repeat region dataset. Fig. S7. Phylogenetic relationships in of <em>Aquilegia</em> obtained from an maximum likelihood analysis of the small single-copy region dataset. Fig. S8. Comparison of structure variations (SVs) in the large single-copy region of 21 representative species of <em>Aquilegia. </em>Fig. S9. Phylogenetic relationships in <em>Aquilegia </em>obtained an maximum likelihood (ML) analysis of the concatenated dataset of the ten identified hypervariable regions.</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366047/bin/12870_2025_7093_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">12870_2025_7093_MOESM2_ESM.pdf</a><sup> (380.1KB, pdf) </sup><p>Additional file 2: Table S1. Species, voucher, distribution, and GenBank accession numbers for the samples used in this study. Table S2. Summary of types of structure variations and plastome lengths of Thalictroideae. Table S3. Summary of characteristics of genes in plastomes of Thalictroideae. Table S4. Comparison of three transition models in ancestral structure variation reconstruction. Table S5. Summary of characteristics of 121 genes in plastome sequences in <em>Aquilegia</em>. Table S6. Summary of characteristics of 120 intergenic spacer (IGS) regions in plastome sequences in <em>Aquilegia</em>.</p>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All sequences in this study are available in the National Center for Biotechnology Information (NCBI) (<a href="https://www.ncbi.nlm.nih.gov/nuccore/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/nuccore/</a>), with GenBank accession numbers (<a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PV546610:PV546614%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV546610â€“PV546614</a>; <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PV564728:PV564802%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV564728â€“PV564802</a>) shown in Table S1.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Plant Biology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12870-025-07093-6"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12870_2025_Article_7093.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.9Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366047/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366047/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366047%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366047/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366047/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366047/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40836276/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366047/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40836276/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366047/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366047/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="zqo6rMSNx73UwqN1tkbYuLispXK1oA8WlNrUzrRoS2ZBnCsSbK21Kow9Gr0W0L4T">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
