
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Coordinated regulation of pH alkalinization by two transcription factors promotes fungal commensalism and pathogenicity - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532B218AF1C293052B21004EBAEE2D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373999/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Coordinated regulation of pH alkalinization by two transcription factors promotes fungal commensalism and pathogenicity">
<meta name="citation_author" content="Xinhua Huang">
<meta name="citation_author_institution" content="Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China">
<meta name="citation_author" content="Guangsheng Chen">
<meta name="citation_author_institution" content="Department of Respiratory and Critical Care Medicine, The First Affiliated Hospital of Guangxi Medical University, Nanning, Guangxi China">
<meta name="citation_author" content="Lei Wu">
<meta name="citation_author_institution" content="Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Yun Zou">
<meta name="citation_author_institution" content="Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Luyao Zhang">
<meta name="citation_author_institution" content="Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Shanshan Li">
<meta name="citation_author_institution" content="Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China">
<meta name="citation_author" content="Kunlin Li">
<meta name="citation_author_institution" content="Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Zaijie Jiang">
<meta name="citation_author_institution" content="Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China">
<meta name="citation_author" content="Yuping Zhang">
<meta name="citation_author_institution" content="Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China">
<meta name="citation_author" content="Xiaoqing Chen">
<meta name="citation_author_institution" content="Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Winnie Shum">
<meta name="citation_author_institution" content="USJ-Kong Hon Academy for Cellular Nutrition and Health, University of Saint Joseph, Macau, China">
<meta name="citation_author" content="Jianbiao Dai">
<meta name="citation_author_institution" content="USJ-Kong Hon Academy for Cellular Nutrition and Health, University of Saint Joseph, Macau, China">
<meta name="citation_author" content="Huichang Huang">
<meta name="citation_author_institution" content="Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China">
<meta name="citation_author" content="Munika Moses">
<meta name="citation_author_institution" content="Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Xianwei Wu">
<meta name="citation_author_institution" content="Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Yuanyuan Wang">
<meta name="citation_author_institution" content="Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China">
<meta name="citation_author" content="Tong Jiang">
<meta name="citation_author_institution" content="Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China">
<meta name="citation_author_institution" content="University of Chinese Academy of Sciences, Beijing, China">
<meta name="citation_author" content="Zhiyi He">
<meta name="citation_author_institution" content="Department of Respiratory and Critical Care Medicine, The First Affiliated Hospital of Guangxi Medical University, Nanning, Guangxi China">
<meta name="citation_author" content="Qing Guo">
<meta name="citation_author_institution" content="Reproductive Medicine Center, The Fourth Hospital of Shijiazhuang (Affiliated Obstetrics and Gynecology Hospital of Hebei Medical University), Shijiazhuang, Hebei China">
<meta name="citation_author" content="Wenwen Xue">
<meta name="citation_author_institution" content="Nanjing Advanced Academy of Life and Health, Nanjing, China">
<meta name="citation_author" content="Hao Li">
<meta name="citation_author_institution" content="Organ Transplantation Clinical Medical Center of Xiamen University, Department of General Surgery, Xiang’an Hospital of Xiamen University, School of Medicine, Xiamen University, Xiamen, China">
<meta name="citation_author" content="Changbin Chen">
<meta name="citation_author_institution" content="Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7855">
<meta name="citation_doi" content="10.1038/s41467-025-62953-x">
<meta name="citation_pmid" content="40846842">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373999/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373999/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373999/pdf/41467_2025_Article_62953.pdf">
<meta name="description" content="As a clinically relevant opportunistic human fungal pathogen, Candida albicans is able to rapidly sense and adapt to changing microenvironments within the host, a process that is essential for its successful invasion and survival. Although studies ...">
<meta name="og:title" content="Coordinated regulation of pH alkalinization by two transcription factors promotes fungal commensalism and pathogenicity">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="As a clinically relevant opportunistic human fungal pathogen, Candida albicans is able to rapidly sense and adapt to changing microenvironments within the host, a process that is essential for its successful invasion and survival. Although studies ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373999/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373999">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62953-x"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62953.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373999%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373999/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373999/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373999/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 22;16:7855. doi: <a href="https://doi.org/10.1038/s41467-025-62953-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62953-x</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Coordinated regulation of pH alkalinization by two transcription factors promotes fungal commensalism and pathogenicity</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Xinhua Huang</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Xinhua Huang</span></h3>
<div class="p">
<sup>1</sup>Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xinhua Huang</span></a>
</div>
</div>
<sup>1,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Guangsheng Chen</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Guangsheng Chen</span></h3>
<div class="p">
<sup>2</sup>Department of Respiratory and Critical Care Medicine, The First Affiliated Hospital of Guangxi Medical University, Nanning, Guangxi China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Guangsheng Chen</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Lei Wu</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Lei Wu</span></h3>
<div class="p">
<sup>1</sup>Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China </div>
<div class="p">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lei Wu</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zou%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Yun Zou</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Yun Zou</span></h3>
<div class="p">
<sup>1</sup>Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China </div>
<div class="p">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zou%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yun Zou</span></a>
</div>
</div>
<sup>1,</sup><sup>3,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Luyao Zhang</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Luyao Zhang</span></h3>
<div class="p">
<sup>1</sup>Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China </div>
<div class="p">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Luyao Zhang</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Shanshan Li</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Shanshan Li</span></h3>
<div class="p">
<sup>1</sup>Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shanshan Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Kunlin Li</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Kunlin Li</span></h3>
<div class="p">
<sup>1</sup>Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China </div>
<div class="p">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Kunlin Li</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Zaijie Jiang</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Zaijie Jiang</span></h3>
<div class="p">
<sup>1</sup>Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zaijie Jiang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Yuping Zhang</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Yuping Zhang</span></h3>
<div class="p">
<sup>1</sup>Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuping Zhang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Xiaoqing Chen</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Xiaoqing Chen</span></h3>
<div class="p">
<sup>1</sup>Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China </div>
<div class="p">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaoqing Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shum%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Winnie Shum</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Winnie Shum</span></h3>
<div class="p">
<sup>4</sup>USJ-Kong Hon Academy for Cellular Nutrition and Health, University of Saint Joseph, Macau, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Shum%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Winnie Shum</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dai%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Jianbiao Dai</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Jianbiao Dai</span></h3>
<div class="p">
<sup>4</sup>USJ-Kong Hon Academy for Cellular Nutrition and Health, University of Saint Joseph, Macau, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dai%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jianbiao Dai</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Huichang Huang</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Huichang Huang</span></h3>
<div class="p">
<sup>1</sup>Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Huichang Huang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moses%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Munika Moses</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Munika Moses</span></h3>
<div class="p">
<sup>1</sup>Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China </div>
<div class="p">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Moses%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Munika Moses</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Xianwei Wu</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Xianwei Wu</span></h3>
<div class="p">
<sup>1</sup>Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China </div>
<div class="p">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xianwei Wu</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Yuanyuan Wang</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Yuanyuan Wang</span></h3>
<div class="p">
<sup>1</sup>Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuanyuan Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Tong Jiang</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Tong Jiang</span></h3>
<div class="p">
<sup>1</sup>Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China </div>
<div class="p">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jiang%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tong Jiang</span></a>
</div>
</div>
<sup>1,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Zhiyi He</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Zhiyi He</span></h3>
<div class="p">
<sup>2</sup>Department of Respiratory and Critical Care Medicine, The First Affiliated Hospital of Guangxi Medical University, Nanning, Guangxi China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22He%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhiyi He</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Qing Guo</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Qing Guo</span></h3>
<div class="p">
<sup>5</sup>Reproductive Medicine Center, The Fourth Hospital of Shijiazhuang (Affiliated Obstetrics and Gynecology Hospital of Hebei Medical University), Shijiazhuang, Hebei China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Guo%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qing Guo</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xue%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Wenwen Xue</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Wenwen Xue</span></h3>
<div class="p">
<sup>6</sup>Nanjing Advanced Academy of Life and Health, Nanjing, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Xue%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenwen Xue</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Hao Li</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Hao Li</span></h3>
<div class="p">
<sup>7</sup>Organ Transplantation Clinical Medical Center of Xiamen University, Department of General Surgery, Xiang’an Hospital of Xiamen University, School of Medicine, Xiamen University, Xiamen, China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hao Li</span></a>
</div>
</div>
<sup>7,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id22"><span class="name western">Changbin Chen</span></a><div hidden="hidden" id="id22">
<h3><span class="name western">Changbin Chen</span></h3>
<div class="p">
<sup>1</sup>Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Changbin Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Joint Laboratory for Biomedical Research and Pharmaceutical Innovation, The Unit of Pathogenic Fungal Infection &amp; Host Immunity, Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, Shanghai, 200031 China </div>
<div id="Aff2">
<sup>2</sup>Department of Respiratory and Critical Care Medicine, The First Affiliated Hospital of Guangxi Medical University, Nanning, Guangxi China </div>
<div id="Aff3">
<sup>3</sup>University of Chinese Academy of Sciences, Beijing, China </div>
<div id="Aff4">
<sup>4</sup>USJ-Kong Hon Academy for Cellular Nutrition and Health, University of Saint Joseph, Macau, China </div>
<div id="Aff5">
<sup>5</sup>Reproductive Medicine Center, The Fourth Hospital of Shijiazhuang (Affiliated Obstetrics and Gynecology Hospital of Hebei Medical University), Shijiazhuang, Hebei China </div>
<div id="Aff6">
<sup>6</sup>Nanjing Advanced Academy of Life and Health, Nanjing, China </div>
<div id="Aff7">
<sup>7</sup>Organ Transplantation Clinical Medical Center of Xiamen University, Department of General Surgery, Xiang’an Hospital of Xiamen University, School of Medicine, Xiamen University, Xiamen, China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Dec 9; Accepted 2025 Aug 4; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373999  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846842/" class="usa-link">40846842</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">As a clinically relevant opportunistic human fungal pathogen, <em>Candida albicans</em> is able to rapidly sense and adapt to changing microenvironments within the host, a process that is essential for its successful invasion and survival. Although studies have shown that the transcription factor Stp2 is the master regulator of environmental alkalinization, accumulating evidence supports a clear involvement of other participants in this adaptation process. Here, following a large-scale genetic screen, we identify the transcription factor Dal81 as an uncharacterized positive regulator of pH alkalinization in <em>C. albicans</em>. Dal81 influences the protein levels of Stp2. A mutant lacking <em>DAL81</em> also fails to alkalinize both in vitro and in the phagolysosome, and this defective phenotype is further enhanced by deleting both factors in most cases. Notably, our results demonstrate that Dal81 physically interacts with Stp2 to co-regulate the expression of a broad set of downstream target genes related to metabolism of organic acids, oxoacids, carboxylic acids and amino acids. This coordinated regulation mode is required for the alkalinization process and plays a role in modulating commensalism and pathogenicity of <em>C. albicans</em>. Taken together, our findings elucidate the cooperative function of Dal81 with Stp2 in the nucleus to orchestrate the expression of downstream genes required for the survival and propagation of <em>C. albicans</em> in the host.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Microbiology, Fungal genes, Fungal genetics</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">In this work, authors perform a large-scale genetic screen to explore the previously unrecognized physical interaction between two transcription factors in <em>Candida albicans</em>. The roles of Stp2 and Dal81 are important in extracellular pH alkalization, fitness, and pathogenicity, key processes requiring coordinated gene regulatory networks governing amino acid metabolism.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Microorganisms are capable of adapting their cells to almost every biological and physicochemical environment. Any deviation from homeostatic conditions can trigger stress reactions in these microbes. Within the spectrum of environmental factors that influence microbial life, pH stands out as one of the most important variables shaping the dynamics of microcosm ecosystems<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Changes in pH can induce diverse stresses on cellular functions, including the availability of essential micronutrients, protein functions, and membrane potentials. Importantly, microbes in the human body must adapt to a variety of pHs in different anatomical sites, ranging from acidic human vagina (pH 4.0) to neutral or slightly alkaline blood and tissue (pH 7.4). In particular, pH along the digestive tract varies considerably, ranging from pH 2.0 to 8.0<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. To survive and grow, microorganisms evolve the ability to rapidly sense and respond to potentially lethal changes in environmental pH, and multiple important signaling pathways are involved. For example, adaptation to extracellular pH is critical for numerous fungal species, primarily mediated by the conserved PacC/Rim101 pathway<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>,<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>. Moreover, both bacterial and fungal pathogens have been shown to actively modify, either lowering or raising, the pH of their surrounding environment to support their survival and enhance virulence. Certain pathogenic fungi, such as <em>Sclerotinia sclerotiorum</em> and <em>Aspergillus spp</em>., secrete acids to acidify the environment and damage host tissues<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>–<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. In contrast, <em>Legionella pneumophila</em> can raise the pH of its phagocytic vacuole, a process that may be critical for the intracellular survival and multiplication of this and other intracellular pathogens<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Similarly, the gastric pathogen <em>Helicobacter pylori</em> employs a urease system to neutralize acidic conditions: it produces ammonia, which enables the bacterium to colonize the highly acidic surface of the stomach<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Some phytopathogenic fungi, including <em>Alternaria alternat, Colletotrichum gleosporioides</em>, and <em>Fusarium oxysporum</em>, also secrete ammonia to alkalinize the acid environment<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. Thus, the capacity to modulate environmental pH appears to be a general feature among most microorganisms during their interactions with hosts. However, the precise mechanisms behind pH acidification or alkalinization remain largely unknown in fungi.</p>
<p id="Par4"><em>Candida albicans</em> is a common opportunistic fungal pathogen in humans, capable of causing superficial infections at mucosal surfaces as well as systemic infections that can disseminate to nearly all major organs. Such disseminated infections, known as candidemia, typically arise in individuals encountering specific risk factors, including prolonged hospitalization in intensive care units, gastrointestinal surgery, the presence of central venous catheters, polytrauma, extremes of age, severe immunosuppression, neutropenia, solid tumors, and hematological malignancies<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Owing to the challenges in eradication and treatment, as well as a substantial risk of morbidity and mortality, this fungus was recently classified in the critical priority fungal pathogen group by the World Health Organization (WHO)<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. A growing body of studies have demonstrated that <em>C. albicans</em> raises extracellular pH, both in vitro and in the macrophage phagosome, when nutrients such as fatty acids, N-acetylglucosamine (GlcNAc), carboxylic acids (<em>e.g</em>., lactate), amino acids, peptides, and proteins are available in distinct microenvironments<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. For example, <em>C. albicans</em> can avidly alkalinize environmental pH by releasing amine groups in the form of ammonia, particularly when amino acids or carboxylic acids serve as its sole carbon source<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Of significance, pH alkalinization has been identified as a key factor linked to infection-associated morphogenetic changes in <em>C. albicans</em>. A rise in environmental pH promotes the neutralization of the macrophage phagosome, triggers the critical yeast-to-hyphal transition, and facilitates the escape from macrophages, all processes that significantly contribute to the fitness and pathogenicity of <em>C. albicans</em> within the host<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Further mechanistic investigations have revealed that <em>C. albicans</em> primarily senses extracellular amino acids via the SPS (Ssy1, Ptr3, and Ssy5) sensor system, followed by the activation of transcription factor Stp2 in a SPS-dependent manner and induction of amino acid influx<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Meanwhile, Stp2 is recognized as the master regulator of environmental alkalinization, playing an essential role in amino acid utilization by predominantly controlling the expression of genes encoding amino acid permeases, Ato transporters, and catabolic enzymes, such as <em>ACH1</em>, <em>CSH3, SSY1</em>, <em>PTR3</em>, <em>SSY5</em>, <em>DUR1,2</em>, <em>DUR31</em>, <em>ATO1,</em> and <em>ATO5</em><sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Interestingly, accumulating evidence suggests that the regulation of pH alkalinization may involve additional players beyond transcription factor Stp2. Mutant cells lacking <em>STP2</em>, when grown on both vaginal simulating fluid (VSF) and artificial saliva (AS), retain the ability to elevate the external environmental pH to levels comparable to those observed in wild-type cells<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. More importantly, <em>C. albicans</em> cells cultured on organic acids do not generate ammonia, and genes required for amino acid-induced pH alkalinization have no impact on growth or pH changes when organic acids or N-acetylglucosamine (GlcNAc) are present<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. These experimental findings strongly indicate that <em>C. albicans</em> regulates environmental alkalinization through multiple distinct mechanisms, and that a range of transcriptional regulators may be involved in this process.</p>
<p id="Par5">Here, we performed a large-scale genetic screen and identified an uncharacterized regulator of pH alkalinization in <em>C. albicans</em>. Our data strongly suggest that the transcription factor Dal81 physically interacts with the known pH-regulator Stp2, and together they co-regulate the expression of a broad set of downstream target genes necessary for the metabolic engineering of organic acids, oxoacids, carboxylic acids, and amino acids. This coordinated regulation mode not only facilitates the alkalinization process but also enhances the commensalism and pathogenicity of <em>C. albicans</em>.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Dal81 is an uncharacterized regulator of pH alkalinization in <em>C. albicans</em>
</h3>
<p id="Par6">A standard set of in vitro alkalinization-inducing condition was used to expose the <em>C. albicans</em> cells on medium 199 (initial pH 4.0) supplemented with phenol red as the pH indicator and incubated for 24 hours at 37 <sup>o</sup>C<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Using this method (Fig. <a href="#Fig1" class="usa-link">1A</a>), we screened the homozygous gene deletion library<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> for mutants defective in pH alkalinization. This screen yielded 23 candidate mutants with reduced alkalinization capacity (Fig. <a href="#MOESM1" class="usa-link">S1</a>). Notably, both the <em>grr1Δ/Δ</em> and <em>dal81Δ/Δ</em> mutants exhibited a complete loss of alkalinization ability, suggesting that Grr1 and Dal81 are potential key regulators of this process. Among these candidates, Dal81 was chosen for further analysis since it encodes a Zn (II)2Cys6 transcription factor and mutant cells lacking <em>GRR1</em> showed significant defects in growth (Fig. <a href="#MOESM1" class="usa-link">S2</a>) and a highly filamentous morphology under non-hypha-inducing conditions<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Clearly, Fig. <a href="#Fig1" class="usa-link">1B</a> showed that deletion of <em>DAL81</em> severely abolished the ability of <em>C. albicans</em> to alkalinize extracellular environment, phenocopying the defects observed in <em>stp2Δ/Δ</em> mutant. Given that both the <em>dal81Δ/Δ</em> mutant strain and its isogenic wild-type control strain (SN250) are auxotrophic for arginine (Arg-), culturing these strains on an amino acid-rich medium (<em>e.g</em>., medium 199)<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> could introduce confounding effects on the alkalinization phenotype. To address this, we generated a new <em>dal81Δ/Δ</em> mutant strain <em>(dal81Δ/Δ</em> M1<em>)</em> in a prototrophic background (SC5314) using the SAT-flipper method<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. We verified through colony PCR and RT-qPCR that the open reading frame (ORF) of the <em>DAL81</em> gene was successfully deleted in the <em>dal81Δ/Δ</em> M1 strain (Fig. <a href="#Fig1" class="usa-link">1C</a>) and that <em>DAL81</em> expression was undetectable in this mutant (Fig. <a href="#Fig1" class="usa-link">1D</a>). Importantly, the alkalinization phenotype of the <em>dal81Δ/Δ</em> M1 strain was consistent with that of the previously constructed <em>dal81Δ/Δ</em> mutant (Fig. <a href="#Fig1" class="usa-link">1E</a>). Given this phenotypic concordance, we proceeded to use the original <em>dal81Δ/Δ</em> mutant for subsequent analysis. A time-course assay further confirmed the observed phenotypes, as we found that when grown in alkalinization-inducing medium (YNB + 1% CAA, pH 4.0), cultures of both wild-type and complemented strains rapidly raised the medium pH from an initial pH of 4.0 to about 5.5 within several hours and reached approximately 6.5 within 24 hours, whereas cultures of the <em>dal81Δ/Δ</em> mutant significantly delayed the alkalinization process, reaching only pH 4.7 at 24 hours (Fig. <a href="#Fig1" class="usa-link">1F</a>). Notably, in comparison to the wild-type strain, both the <em>dal81Δ/Δ</em> and <em>stp2Δ/Δ</em> mutants exhibited only mild growth defects in YPD medium (Fig. <a href="#MOESM1" class="usa-link">S3A, B</a>) but showed significantly impaired growth in YNB medium supplemented with 1% casamino acids (CAA) (Fig. <a href="#MOESM1" class="usa-link">S3C</a>). This growth impairment in YNB + 1% CAA may be due to the inability of the mutants to efficiently catabolize amino acids, particularly CAA, thereby contributing to the observed alkalinization defect. Moreover, the alkalinization defect of the <em>dal81Δ/Δ</em> mutant was also observed on solid GM-BCP or GM-BCG medium (initial pH 4.0; Fig. <a href="#Fig1" class="usa-link">1G</a>). Reintroducing an ectopic copy of the wild-type allele back into the <em>dal81Δ/Δ</em> mutant rescued the alkalinization defect (Fig. <a href="#Fig1" class="usa-link">1F, G</a>), confirming that the phenotype is specifically attributable to the deletion of the <em>DAL81</em> gene. Like the <em>stp2Δ/Δ</em> mutant, the <em>dal81Δ/Δ</em> strain released significantly less ammonia than the wild type (Fig. <a href="#Fig1" class="usa-link">1H</a>). This finding suggests that reduced ammonia release may be a primary factor contributing to the failed alkalinization phenotype of the <em>dal81Δ/Δ</em> mutant. Indeed, consistent with previous findings in the <em>stp2Δ/Δ</em> mutant<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>, we observed that the growth of the <em>dal81Δ/Δ</em> mutant was completely inhibited by the sulfonylurea herbicide metsulfuron methyl (MM), which targets acetohydroxyacid synthase to block branched-chain amino acid biosynthesis (Fig. <a href="#MOESM1" class="usa-link">S4</a>). As expected, the complemented strain exhibited growth nearly indistinguishable from the wild type in the presence of MM (Fig. <a href="#MOESM1" class="usa-link">S4</a>). These results indicate that knocking out either <em>STP2</em> or <em>DAL81</em> yields identical phenotypes, characterized by severe inhibition of amino acid uptake and impaired pH alkalinization. This phenotypic convergence strongly suggests that Stp2 and Dal81 may act in concert or via parallel pathways to regulate this critical process.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. <em>C. albicans</em> Dal81 is required for pH alkalinization, hyphal growth, and macrophage escape.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373999_41467_2025_62953_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/74b6/12373999/1c53c59443f4/41467_2025_62953_Fig1_HTML.jpg" loading="lazy" id="d33e868" height="949" width="798" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic outlining the in vitro pH alkalinization screening workflow. The figure was created in BioRender<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup>. <strong>B</strong> Similar to <em>stp2Δ/Δ</em>, <em>dal81Δ/Δ</em> cells displayed alkalinization defects in liquid assays. Wild-type (WT), <em>dal81Δ/Δ</em>, <em>stp2Δ/Δ</em> and the complemented strain (DAL81 comp.) were grown overnight in YPD, centrifuged, washed with water, and resuspended in medium 199 (initial pH 4.0) at an OD<sub>600</sub> of 1.0. Serial 1:5 dilutions were prepared in 96-well plates and alkalinization was visualized colorimetrically by the development of an orange-red medium color after 24 h incubation at 37 °C. <strong>C</strong> Colony PCR confirmed <em>DAL81</em> deletion. The <em>DAL81</em> ORF was present in WT (SC5314) but absent in the <em>dal81Δ/Δ</em> M1 mutant. <strong>D</strong> RT-qPCR analysis of <em>DAL81</em> expression in SC5314 and <em>DAL81</em> knockout strains following 5 h culture in medium 199 (initial pH 4.0) at 37 °C. <strong>E</strong> Under the same conditions as (B), the <em>dal81Δ/Δ</em> M1 mutant showed comparable alkalinization defects. <strong>F</strong>
<em>DAL81-</em>deficient mutants exhibited delayed pH elevation in YNB + 1% CAA (initial pH 4.0), with medium pH measured at indicated time points. <strong>G</strong> Alkalinization defects in <em>stp2Δ/Δ</em> and <em>dal81Δ/Δ</em> mutants were also observed on solid medium. Cells were spotted onto GM-BCP or GM-BCG solid medium (initial pH 4.0) and incubated at 37 °C for 3 days. <strong>H</strong> Dal81 and Stp2 are required for ammonia release, quantified from GM-BCP agar cultures. <strong>I</strong>
<em>DAL81</em>-deficient cells failed to alkalinize the macrophage phagosome. Intracellular compartment acidification was measured using pHrodo. Left panel shows a representative image of phagosomal pH tracking. Scale bar, 20 μm. The right panel quantified acidified compartments. <strong>J</strong>
<em>DAL81</em> deletion reduced RAW264.7 macrophage killing, measured by LDH release after 5 h infection at MOI = 3. <strong>K</strong><strong>–M </strong><em>DAL81</em>-deficient cells showed decreased survival in RAW264.7 (<strong>K)</strong>, J774A.1 (<strong>L)</strong>, and BMDMs (<strong>M)</strong>. Data in (<strong>D, F, H</strong><strong>–M</strong>) are mean ± SD of three biological replicates. Statistical significance was assessed via one-way ANOVA with Tukey’s test (F, 24 h) and unpaired two-tailed Student’s <em>t</em>-test (<strong>D, H</strong><strong>–M</strong>). ns, not significant. Source data for (<strong>D</strong>, <strong>F</strong>, <strong>H</strong>–<strong>M</strong>) are provided in the Source Data file.</p></figcaption></figure><p id="Par7">The in vitro evidence demonstrating the inability of the <em>dal81Δ/Δ</em> mutant to alkalinize environmental pH was further validated in the phagosomal context. Log phase wild-type cells, <em>stp2Δ/Δ</em>, and <em>dal81Δ/Δ</em> mutant cells were co-cultured with RAW264.7 macrophages cell lines, and the capacity of <em>C. albicans</em> cells to modulate phagosomal pH was assessed using the pH-sensitive dye pHrodo, a probe that fluoresces inside acidic phagosomes<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. In this assay, only 24% of wild-type <em>C. albicans</em> cells were localized to acidified phagolysosomes, whereas over 50% of both <em>stp2Δ/Δ</em> and <em>dal81Δ/Δ</em> mutant cells were found in these acidic compartments (Fig. <a href="#Fig1" class="usa-link">1I</a>). These findings indicate that deletion of either <em>STP2</em> or <em>DAL81</em> impairs the ability of fungal cells to neutralize the phagosomal environment. A similar pattern was observed in J774A.1 macrophages (Fig. <a href="#MOESM1" class="usa-link">S5A, B</a>). Time-course experiments further confirmed this trend: phagosomes containing wild-type cells in J774A.1 macrophages exhibited a marked reduction in acidification over time, while those harboring <em>dal81Δ/Δ</em> or <em>stp2Δ/Δ</em> mutants remained consistently acidic (Fig. <a href="#MOESM1" class="usa-link">S5A, B</a>). Taken together, our data strongly support that transcription factor Dal81 appears to be a new regulator required for pH alkalinization of <em>C. albicans</em>.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Dal81 is required for hyphal growth and escape from macrophages</h3>
<p id="Par8">In <em>C. albicans</em>, neutral to alkaline pH environments drive the yeast-to-hyphal morphogenetic switch, a critical adaptive strategy that enables the fungus to evade macrophage-mediated killing<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. The transcription factor Stp2 has been identified as a key regulator in this process<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Given that Dal81 regulates pH alkalinization, we asked whether Dal81 is also required for hyphal induction and phagosomal evasion. To address this, we investigated the hyphal formation ability of the wild-type, <em>dal81Δ/Δ</em>, and the <em>DAL81</em>-complemented strain in an alkalinization-inducing medium (YNB + 1% CAA, initial pH 4.5), using the <em>stp2Δ/Δ</em> mutant as a control. Similar to the <em>stp2Δ/Δ</em> mutant, the <em>dal81</em>Δ/Δ mutant cells exhibited a defect in the yeast-to-hyphal transition (Fig. <a href="#MOESM1" class="usa-link">S6A</a>). In contrast, both the wild-type and <em>DAL81</em>-complemented strains formed normal hyphae under the same conditions. Consistently, in co-cultures with macrophages, <em>dal81Δ/Δ</em> mutant cells were primarily arrested in the yeast form, whereas wild-type cells robustly developed hyphae in the phagosome (Fig. <a href="#Fig1" class="usa-link">1I</a>). Importantly, the failure of the <em>dal81Δ/Δ</em> mutant to form hyphae in this context is not due to an intrinsic inability to filament, as hyphal growth of the <em>dal81Δ/Δ</em> mutant was indistinguishable from that of the wild type when strains were grown overnight in YPD and then transferred to medium 199 buffered at pH either 4.5 or 7.5. (Fig. <a href="#MOESM1" class="usa-link">S6B</a>). These findings indicate that Dal81 plays a specific role in promoting <em>C. albicans</em> hyphal formation within macrophages.</p>
<p id="Par9">To investigate whether Dal81 acts to enhance the ability of <em>C. albicans</em> to survive within the phagocyte, and ultimately, contributes to host cell death, we examined the cytotoxicity of fungal cells in RAW264.7 macrophages by measuring the lactate dehydrogenase (LDH) release following a 5-hour co-culture. We found that the <em>dal81Δ/Δ</em> mutant cells exhibited significantly lower cytotoxicity compared to both the wild-type and <em>DAL81</em>-complemented strains, and even lower than the <em>stp2Δ/Δ</em> strain (Fig. <a href="#Fig1" class="usa-link">1J</a>). Moreover, we used an endpoint dilution assay to compare the survival of different <em>C. albicans</em> strains across three macrophage types, including RAW264.7, J774 A.1, and bone marrow-derived macrophages (BMDMs). The results revealed that both the <em>dal81Δ/Δ</em> and <em>stp2Δ/Δ</em> strains displayed impaired survival relative to the wild-type when challenged with these macrophages (Fig. <a href="#Fig1" class="usa-link">1K–M</a>).</p>
<p id="Par10">Collectively, these findings affirm the crucial role of Dal81 in modulating environmental alkalinization, hyphal formation, and the escape from innate immune cells such as macrophages.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Dal81 exerts a superposition effect on Stp2-dependent alkalinization</h3>
<p id="Par11">The data presented above strongly suggested that Dal81 behaves similarly to Stp2 in mediating <em>C. albicans</em>-induced alkalinization of the extracellular environment, prompting us to test whether these two transcription factors exhibit functional redundancy in regulating pH alkalinization. We first analyzed the expression patterns of <em>DAL81</em> and <em>STP2</em> in <em>C. albicans</em> during alkalinization in liquid culture (YNB + 1% CAA, initial pH 4.5). Using RT-qPCR, we measured the relative transcription levels of <em>DAL81</em> and <em>STP2</em> following the rise of culture pH. The results revealed that the expression of both genes was significantly upregulated during the alkalinization process (Fig. <a href="#Fig2" class="usa-link">2A</a>), highlighting their shared relevance to extracellular pH elevation.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Fungal cells lacking both <em>DAL81</em> and <em>STP2</em> exhibit more severe pH alkalinization defects than either single mutant.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373999_41467_2025_62953_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/74b6/12373999/eb35ae67f1fd/41467_2025_62953_Fig2_HTML.jpg" loading="lazy" id="d33e1131" height="894" width="798" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> RT-qPCR analysis revealed that both <em>DAL81</em> and <em>STP2</em> were highly induced during pH alkalinization in wild-type cells grown in synthetic dextrose minimal medium (SD) or YNB + 1% CAA medium (initial pH 4.5) for 6 h. <strong>B</strong><strong>–F</strong> The <em>dal81Δ/Δstp2Δ/Δ</em> double mutant showed significant alkalinization defects. Indicated strains were grown in YNB + 1% CAA medium (initial pH 4.3) <strong>B</strong>, artificial saliva medium (initial pH 4.2) <strong>C</strong> or vaginal simulating fluid medium (initial pH 4.5) <strong>D</strong> at 37 °C, with pH measured at indicated time points. <strong>E</strong> Colorimetric assay. Overnight YPD cultures were washed, diluted to OD<sub>600</sub> = 0.05 (top) or 5.0 (bottom) in YNB + 1% CAA (initial pH 4.5) with bromocresol purple, and incubated at 37 °C under aeration. <strong>F</strong> Representative images. Overnight YPD cultures were washed, resuspended in water at OD<sub>600</sub> = 1.0 or 5.0, and 2 μl spotted onto GM-BCP medium (pH 4.0) at 37 °C. <strong>G</strong> Microscopic morphology of colony center from (<strong>F</strong>). Scale bar, 10 μm. Data are mean ± SD of three biological replicates. Statistical significance was assessed by an unpaired two-tailed Student’s <em>t</em>-test (<strong>A</strong>) and one-way ANOVA with Tukey’s test (24 h for <strong>B</strong>, 56 h for <strong>C </strong>and <strong>D</strong>). *<em>P</em> &lt; 0.05, ** <em>P</em> &lt; 0.01, **** <em>P</em> &lt; 0. 0001; ns, not significant. Source data for (<strong>A–D</strong>) are provided in the Source Data file.</p></figcaption></figure><p id="Par12">To fully understand the contributions of Dal81 and Stp2 to environmental alkalinization by <em>C. albicans</em>, we generated a <em>dal81Δ/Δstp</em>2<em>Δ/Δ</em> double mutant strain. As shown in Fig. <a href="#MOESM1" class="usa-link">S7</a>, the double mutant exhibited similar phenotypes comparable to those of the single mutants, including reduced amino acids uptake capacity (Fig. <a href="#MOESM1" class="usa-link">S7A</a>), impaired hyphal formation under alkalinization-inducing conditions (Fig. <a href="#MOESM1" class="usa-link">S7B</a>), diminished macrophage damage (Fig. <a href="#MOESM1" class="usa-link">S7C</a>), and decreased survival in macrophages (Fig. <a href="#MOESM1" class="usa-link">S7D</a>). However, relative to the single mutants, <em>C. albicans</em> cells lacking both <em>DAL81</em> and <em>STP2</em> displayed more severe alkalinization defects when grown in YNB + 1% CAA (initial pH 4.5; Fig. <a href="#Fig2" class="usa-link">2B</a>) and in artificial saliva (AS; Fig. <a href="#Fig2" class="usa-link">2C</a>). Notably, this enhanced defect was independent of inoculum size, as it persisted across varying cell densities (Fig. <a href="#Fig2" class="usa-link">2E, F</a>). Biochemically, the <em>dal81Δ/Δstp2Δ/Δ</em> double mutant showed a more pronounced reduction in ammonia release compared to the <em>stp2Δ/Δ</em> single mutant, though its ammonia levels were indistinguishable from those of the <em>dal81Δ/Δ</em> mutant (Fig. <a href="#Fig1" class="usa-link">1E</a>), suggesting that the severe alkalinization defect in the double mutant is likely driven by impaired ammonia production, and additional Dal81 or Stp2-dependent metabolic endpoints may also contribute to this phenotype. Interestingly, we observed niche-specific differences in their roles. In AS, the <em>dal81Δ/Δ</em> mutant behaved like the wild-type, while in vaginal simulating fluid (VSF), the <em>stp2Δ/Δ</em> mutant resembled the wild-type, and the <em>dal81Δ/Δ</em> mutant phenocopied the double mutant (Fig. <a href="#Fig2" class="usa-link">2D</a>). These results reveal a context-dependent cooperative relationship between Dal81 and Stp2 in regulating pH alkalinization, where their functional importance shifts based on the environmental niche.</p>
<p id="Par13">In addition to their roles in pH regulation, colony morphology varied significantly across strains, reflecting differences in hyphal growth and morphological switching. As shown in Fig. <a href="#Fig2" class="usa-link">2F</a>, the wild-type and Dal81-complemented strains formed wrinkled colonies, a hallmark of robust hyphal development, whereas the <em>stp2Δ/Δ</em> mutant and <em>dal81Δ/Δstp2Δ/Δ</em> double mutant produced smooth colonies, indicative of a yeast-dominant state with no detectable hyphal growth. The <em>dal81Δ/Δ</em> mutant displayed an intermediate phenotype, characterized by limited colony wrinkling. Microscopic examination confirmed these morphological differences: wild-type and complemented strains displayed elongated, branching hyphae, while <em>dal81Δ/Δ</em> cells showed defective filamentation. In contrast, <em>stp2Δ/Δ</em> and the double mutant remained entirely in the yeast form, with no evidence of hyphal differentiation (Fig. <a href="#Fig2" class="usa-link">2G</a>). These observations indicate that impaired hyphal development directly contributes to the altered colony morphology and further highlight the coordinated roles of Dal81 and Stp2 in regulating both alkalinization and morphogenesis in <em>C. albicans</em>.</p></section><section id="Sec6"><h3 class="pmc_sec_title">The process of pH alkalinization requires the co-participation of Dal81 and Stp2</h3>
<p id="Par14">The observation that both Dal81 and Stp2 significantly influence pH alkalinization prompted us to investigate their functional relationship. Knocking out <em>DAL81</em> had no effect on the mRNA levels of <em>STP2</em> but led to a significant reduction in Stp2 protein levels (Fig. <a href="#Fig3" class="usa-link">3A, B</a>). Importantly, the N-terminal proteolytic processing of Stp2, a critical post-translational step for its amino acid-induced transcriptional activation<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>, remained unaffected in the <em>dal81Δ/Δ</em> mutant (Fig. <a href="#Fig3" class="usa-link">3B</a>). These results strongly suggest that Dal81 is necessary for maintaining maximal Stp2 protein levels but is not involved in its proteolytic processing. Interestingly, while <em>DAL81</em> mRNA levels were significantly upregulated in the <em>stp2Δ/Δ</em> mutant compared to the wild-type (Fig. <a href="#Fig3" class="usa-link">3C</a>), Dal81 protein levels remained unchanged (Fig. <a href="#Fig3" class="usa-link">3D</a>). This discrepancy suggests a decoupling between transcriptional and translational regulation of Dal81 in the absence of Stp2.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. pH alkalinization in <em>C. albicans</em> requires the transcriptional activities of both Dal81 and Stp2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373999_41467_2025_62953_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/74b6/12373999/46cd31231ef8/41467_2025_62953_Fig3_HTML.jpg" loading="lazy" id="d33e1339" height="440" width="798" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Dal81 had no effect on the mRNA level of <em>STP2</em>, assayed by RT-qPCR analyses. Indicated strains were grown in YNB + 1% CAA (initial pH 4.5) for 6 h. <strong>B</strong> Dal81 influenced Stp2 protein level but not its proteolysis. Immunoblotting assessed Stp2-HA levels in different strains after 6 h in YNB + 1% CAA (initial pH 4.5). Quantification via ImageJ was shown. <strong>C</strong> Stp2 influenced the mRNA level of <em>DAL8</em>1, assayed by RT-qPCR under the same growth conditions described in <strong>(B)</strong>. <strong>D</strong> Immunoblotting showed Stp2 did not alter Dal81 protein level. Dal81-Myc levels were quantified using ImageJ. Indicated strains were grown as in (<strong>B</strong>). <strong>E, F</strong> Overexpression of <em>STP2</em> in <em>dal81Δ/Δ </em>or <em>DAL81</em> in <em>stp2Δ/Δ</em> failed to rescue alkalinization defects. Overnight YPD cultures were washed, resuspended to OD<sub>600</sub> = 1.0, and 2 μl spotted onto GM-BCP medium (pH 4.0) at 37 °C. Plates were photographed after 3 days. Data are mean ± SD of three biological replicates. Statistical significance for (<strong>A–D</strong>) was determined by unpaired two-tailed Student’s <em>t</em>-test. * <em>P</em> &lt; 0.05, ** <em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001; ns: not significant. Source data for (<strong>A–D</strong>) are provided in the Source Data file.</p></figcaption></figure><p id="Par15">Given the decreased mRNA and protein levels of Stp2 in <em>C. albicans</em> cells lacking <em>DAL81</em>, we asked whether overexpressing <em>STP2</em> in the <em>dal81Δ/Δ</em> mutant could restore the capacity of alkalinizing the extracellular pH. We therefore generated two overexpression strains: one driving expression of full-length, unprocessed Stp2, and another expressing a truncated, processed form (<em>STP2</em><sup><em>⁎</em></sup>) under the control of the strong constitutive <em>TDH3</em> promoter. The <em>STP2</em><sup><em>⁎</em></sup> variant lacks the N-terminal nuclear exclusion domain (amino acids 1–99) and mimics the proteolytically activated form of Stp2<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Notably, the strain (<em>STP2</em><sup>OE</sup>) exhibited a markedly enhanced pH alkalinization phenotype compared to the wild-type (Fig. <a href="#MOESM1" class="usa-link">S8A</a>). To our surprise, overexpression of either the full-length unprocessed or the truncated version of <em>STP2</em> failed to rescue the alkalinization defect caused by <em>DAL81</em> deletion (Fig. <a href="#Fig3" class="usa-link">3E</a> and Fig. <a href="#MOESM1" class="usa-link">S8A, B</a>). Consistently, overexpression of <em>DAL81</em> in the <em>stp2Δ/Δ</em> background also did not restore alkalinization capacity (Fig. <a href="#Fig3" class="usa-link">3F</a>). These results strongly suggest that the relationship between Dal81 and Stp2 does not follow a classic epistatic transcriptional hierarchy. Instead, amino acid-induced pH alkalinization in <em>C. albicans</em> appears to require the transcriptional activities of both Dal81 and Stp2, with neither factor alone sufficient to compensate for the loss of the other.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Dal81 has no effect on nuclear localization of Stp2</h3>
<p id="Par16">Previous studies have demonstrated that under inducing conditions, N-terminally processed Stp2 efficiently enters the nucleus, binds to target gene promoters, and activates their expression<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. The above-mentioned findings have shown that overexpression of <em>STP2</em> in the <em>dal81Δ/Δ</em> mutant failed to rescue the alkalinization defect. Moreover, we performed RT-qPCR analyses and found that the mRNA levels of three major Stp2 target genes (<em>CAN2</em>, <em>HGT2</em>, and <em>ACS1</em>) were significantly decreased in the <em>dal81Δ/Δ</em> mutant, a pattern mimicking the downregulation observed in the <em>stp2Δ/Δ</em> mutant (Fig. <a href="#MOESM1" class="usa-link">S9</a>). Because <em>DAL81</em> deletion had no effect on the proteolytic cleavage of Stp2, we asked whether the loss of Dal81 activity could interfere with Stp2 nuclear localization or promoter transactivation and DNA binding, or both. To explore this, we examined the cellular localization of wild-type and mutant version of Stp2-3xHA by immunofluorescence in amino acids-inducing media. Interestingly, Stp2 retains the ability to translocate to the nucleus independent of Dal81. In wild-type, <em>dal81Δ/Δ,</em> and <em>STP2</em><sup>*</sup><sup>OE</sup> mutant strains, Stp2 localized to both the cytoplasm and nucleus in most cells (Fig. <a href="#Fig4" class="usa-link">4A</a>). While the <em>dal81Δ/Δ</em> mutant exhibited a slight reduction in cells with dual cytoplasmic-nuclear Stp2 localization compared to the wild-type, this was offset by an increased proportion of cells with exclusively nuclear Stp2. As a result, the total percentage of cells with nuclear Stp2 was comparable between <em>dal81Δ/Δ</em> and wild-type strains. Additionally, in YNB + 1% CAA medium, the <em>dal81Δ/Δ</em> mutant expressed both full-length and cleaved (active) forms of Stp2 (Fig. <a href="#Fig3" class="usa-link">3B</a>), confirming that post-translational processing and activation of Stp2 occur normally in the absence of Dal81. Together, these findings suggest that Dal81 is not required for the nuclear localization or functional activation of Stp2.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Dal81 physically interacts with Stp2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373999_41467_2025_62953_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/74b6/12373999/8500453022bf/41467_2025_62953_Fig4_HTML.jpg" loading="lazy" id="d33e1506" height="680" width="792" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Dal81 had no effect on the nuclear localization of Stp2. Indirect immunofluorescence of strains under alkalinization conditions. DIC (phase images), FITC (Stp2-HA staining), DAPI (DNA staining), and Merge (Stp2-HA/DNA overlay). Right panel quantifies 100 cells/experiment. “C” (≥90% cytoplasmic), “N” (≥90% nuclear), and “C + N” (mixed). Scale bar, 10 μm (all images same magnification). Data are mean ± SD (three biological replicates). Statistical significance via two-way ANOVA with Tukey’s test. **** <em>P</em> &lt; 0. 0001; ns: not significant. <strong>B</strong> Yeast two-hybrid analysis of Dal81-Stp2 interaction. Stp2 was fused to the bait vector, Dal81 to the prey vector. Positive control: Interaction of pGADT7-T with pGBKT7-53. Strains were grown on SD medium lacking leucine, tryptophan, adenine, histidine, plus 40 μg/ml X-α-Gal (SD-Leu-Trp-Ade-His+X-α-Gal). <strong>C, D</strong> Co<strong>-</strong>immunoprecipitation of Stp2-HA with Dal81-Myc. Strains expressing Stp2-HA or Dal81-Myc were grown in YNB + 1% CAA medium (initial pH4.5). Whole cell extracts were prepared under nondenaturing conditions and were immunoprecipitated with anti-HA (<strong>C</strong>) or anti<strong>-</strong>c<strong>-</strong>Myc (<strong>D</strong>) beads. Precipitated proteins were separated by SDS-PAGE and probed with anti-Myc or anti-HA monoclonal antibodies. Experiments were independently replicated twice, with consistent results obtained in each repetition. Source data for (<strong>A, C, D</strong>) are provided as a Source Data file.</p></figcaption></figure></section><section id="Sec8"><h3 class="pmc_sec_title">Dal81 physically interacts with Stp2</h3>
<p id="Par17">To investigate whether Dal81 influences the promoter transactivation and DNA binding capacity of Stp2, we first tested for physical interaction between the two transcription factors. A yeast two-hybrid analysis revealed a robust interaction between Stp2 and Dal81. When Stp2 was fused to the Gal4 DNA binding domain (BD-Stp2) and Dal81 to the Gal4 activation domain (AD-Dal81), a strong reporter signal was detected (Fig. <a href="#Fig4" class="usa-link">4B</a>). No interaction was observed when BD-Stp2 or AD-Dal81 was co-expressed with empty vectors, confirming specificity. To validate this interaction in a physiological context, we performed co-immunoprecipitation (Co-IP) experiments. Stp2 was tagged with a C-terminal hemagglutinin (HA) epitope and co-expressed with Dal81-Myc in <em>C. albicans</em>. Importantly, strains expressing HA-tagged Stp2, Myc-tagged Dal81, or both tagged proteins exhibited growth and alkalization phenotypes indistinguishable from the untagged wild-type strain when cultured in YNB + 1% CAA medium (Fig. <a href="#MOESM1" class="usa-link">S10</a>), confirming that the epitope tags do not disrupt Dal81 or Stp2 function. In Co-IP assays, immunoprecipitation of Stp2-HA using an anti-HA antibody co-precipitated Dal81-Myc, as detected by Western blot (Fig. <a href="#Fig4" class="usa-link">4C</a>). Reciprocally, immunoprecipitation of Dal81-Myc with an anti-Myc antibody co-precipitated Stp2-HA (Fig. <a href="#Fig4" class="usa-link">4D</a>). These results provide direct evidence that Dal81 and Stp2 physically interact in vivo.</p></section><section id="Sec9"><h3 class="pmc_sec_title">The physical interaction between Dal81 and Stp2 is required for pH alkalinization of <em>C. albicans</em>
</h3>
<p id="Par18">Building on our earlier findings of a physical interaction between Dal81 and Stp2, we next aimed to identify the specific residues mediating this interaction. A sequence analysis using InterPro (<a href="https://www.ebi.ac.uk/interpro/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/interpro/</a>) revealed that <em>C. albicans</em> Dal81 contains a Zn2/Cys6 DNA-binding domain (amino acids 110-159) and a fungal-specific transcription factor domain (amino acids 325-597) (Fig. <a href="#MOESM1" class="usa-link">S11</a>). The Zn2/Cys6 domain is characterized by alpha-helical structures that coordinate zinc atoms for DNA binding, while the fungal-specific domain comprises several structural motifs implicated in transcriptional regulation.</p>
<p id="Par19">To investigate the interaction interface, we predicted the structure of the Dal81-Stp2 complex using AlphaFold 3 by inputting the full-length amino acid sequences of both proteins. Strikingly, the predicted protein model suggested a lock-and-key configuration, wherein Stp2 fits into a complementary binding groove on Dal81 (Fig. <a href="#Fig5" class="usa-link">5A</a>). Based on the AlphaFold-predicted model and structural analysis using ChimeraX<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>, we identified 21 candidate residues on protein Dal81 that are potentially critical for its interaction with protein Stp2 (Fig. <a href="#Fig5" class="usa-link">5B</a>). To experimentally validate these in silico predictions, we generated site-directed mutants in which the two top-ranked predicted interaction residues, Dal81<sup>L405</sup> and Dal81<sup>S406</sup>, were substituted with alanine (Fig. <a href="#Fig5" class="usa-link">5B</a>). Using the Stp2-HA/<em>dal81Δ/Δ</em> background strain described in Fig. <a href="#Fig3" class="usa-link">3B</a>, we constructed a Myc-tagged Dal81 mutant strains (<em>DAL81m2</em>/<em>dal81Δ</em>) harboring both point mutations, alongside a Myc-tagged wild-type <em>DAL81</em> control strain (<em>DAL81</em>/<em>dal81Δ</em>). Immunoblot analysis of cell lysates from these strains demonstrated that the alanine substitutions had only a minor impact on the protein levels of Dal81 and Stp2 (Fig. <a href="#Fig5" class="usa-link">5C–E</a>), suggesting that the mutations did not impair protein stability.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Identification of critical interaction residues between Dal81 and Stp2 responsible for pH alkalinization.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373999_41467_2025_62953_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/74b6/12373999/7c2646040b6d/41467_2025_62953_Fig5_HTML.jpg" loading="lazy" id="d33e1677" height="908" width="798" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> AlphaFold 3 in silico docking simulations revealed that Dal81 and Stp2 formed two important hydrogen bonds (Dal81<sup>L405</sup> - Stp2<sup>D526</sup> and Dal81<sup>S406</sup> - Stp2<sup>D528</sup>). Dal81 is visualized as yellowish brown cartoons, Stp2 as light blue. Predicted interaction structures are categorized by confidence intervals. <strong>B</strong> Schematics of <em>C. albicans</em> Dal81, with 21 AlphaFold3-predicted Stp2-interacting residues highlighted in red. <strong>C</strong><strong>–E</strong> Immunoblots (<strong>C</strong>) of Dal81-Myc and Stp2-HA expression in indicated strains after 6 h in YNB + 1% CAA (initial pH 4.5). ImageJ quantification of Dal81-Myc (<strong>D</strong>) and Stp2-HA (<strong>E</strong>) band intensities. Data are mean ± SD (three biological replicates). Statistical significance via unpaired two-tailed Student’s <em>t</em>-test. * <em>P</em> &lt; 0.05, ** <em>P</em> &lt; 0.01; ns, not significant. <strong>F</strong> Alkalinization analysis of the specified strains was conducted following the procedures described in Fig. <a href="#Fig1" class="usa-link">1B</a>. <strong>G</strong> Stp2-HA failed to co<strong>-</strong>immunoprecipitate with Dal81m21-Myc. Strains co-expressing Stp2-HA and Dal81m21-Myc were grown in YNB + 1% CAA (initial pH4.5). Whole cell extracts were prepared under nondenaturing conditions, and immunoprecipitated with anti-Myc beads. Precipitates were immunoblotted with anti-HA or anti-Myc antibodies. R1 and R2 denote biological replicates. <strong>H</strong> Growth curves of indicated strains in medium 199 (initial pH 4.0) at 37 °C, with OD<sub>600</sub> measured every 15 min (BioTek Synergy 2 Multi-mode Microplate Reader). Data are mean ± SD (three independent biological replicates). Statistical analysis via one-way ANOVA with Tukey’s multiple comparison test at 24 h. * <em>P</em> &lt; 0.05, ** <em>P</em> &lt; 0.01. <strong>I</strong> Alkalinization assay (as in Fig. <a href="#Fig1" class="usa-link">1B</a>) showing pH modulation defects in <em>DAL81m21</em>/<em>dal81Δ</em>. Source data for (<strong>C–E, G, H</strong>) are provided in the Source Data file.</p></figcaption></figure><p id="Par20">Unexpectedly, we observed that the <em>DAL81m2</em>/<em>dal81Δ</em> mutant displayed normal alkalinization activity on medium 199 (Fig. <a href="#Fig5" class="usa-link">5F</a>), indicating that substitution of Dal81<sup>L405</sup> and Dal81<sup>S406</sup> alone is insufficient to disrupt the Dal81-Stp2 interaction or its downstream functional output. These results suggest that additional residues likely contribute to the interaction interface required for pH alkalinization. To assess the functional importance of the predicted interaction interface between Dal81 and Stp2, we constructed a mutant strain (<em>DAL81m21</em>/<em>dal81Δ</em>) in which all 21 putative binding residues identified from the docking analysis (Fig. <a href="#Fig5" class="usa-link">5B</a>) were substituted with alanine. Co-immunoprecipitation (Co-IP) analysis revealed that the physical interaction between Dal81 and Stp2 was abolished in the <em>DAL81m21</em>/<em>dal81Δ</em> strain (Fig. <a href="#Fig5" class="usa-link">5G</a>), confirming the critical role of these residues in mediating protein–protein interaction. Interestingly, these amino acid substitutions did not affect fungal growth, as the <em>DAL81m21</em>/<em>dal81Δ</em> mutant exhibited growth comparable to its wild-type control strain (<em>DAL81</em>/<em>dal81Δ</em>) (Fig. <a href="#Fig5" class="usa-link">5H</a>). However, <em>DAL81m21</em>/<em>dal81Δ</em> displayed a severe defect in extracellular pH alkalinization when cultured in medium 199 (Fig. <a href="#Fig5" class="usa-link">5I</a>), demonstrating that the physical interaction between Dal81 and Stp2 is functionally required for this phenotype.</p></section><section id="Sec10"><h3 class="pmc_sec_title">The physical interaction between Dal81 and Stp2 promotes target gene expression</h3>
<p id="Par21">To explore the functional relationship between Dal81 and Stp2 in regulating pH alkalinization of <em>C. albicans</em>, we used RNA-seq to determine the transcriptional profiles and pH alkalinization-responsive gene networks that were dependent on the two transcription regulators Dal81 and Stp2. Three independent RNA-seq analyses were performed to profile the <em>dal81Δ/Δ</em> and <em>stp2Δ/Δ</em> mutant strains, alongside two reference strains (SC5314 and SN250), under two culture conditions: SD or YNB + 1% CAA (YC, initial pH 4.5) with a 6-hour incubation period. The inclusion of two reference strains was necessitated by the auxotrophic nature of the SN250 strain for arginine, which prevents its growth in SD medium. Principal component analysis (PCA) (Fig. <a href="#Fig6" class="usa-link">6A</a>) and heat map visualization (Fig. <a href="#Fig6" class="usa-link">6B</a>) revealed a high correlation in gene expression profiles between the <em>dal81Δ/Δ</em> and <em>stp2Δ/Δ</em> mutant strains, suggesting comparable transcriptional responses under the tested conditions. Using a stringent cutoff (at least a 2-fold change between baseline and treatment samples, FDR ≤ 0.1), we identified 2516 differentially expressed genes (DEGs) in wild-type cells upon alkaline pH induction, defined as “alkalinization-responsive genes” (Data <a href="#MOESM3" class="usa-link">S1</a>). Among these, 1280 alkalinization-responsive genes were regulated by Dal81, 1277 genes by Stp2, and 1120 were coregulated by both factors (Fig. <a href="#MOESM1" class="usa-link">S12A, B</a>, Fig. <a href="#Fig6" class="usa-link">6C</a>, Data <a href="#MOESM3" class="usa-link">S1</a>). Gene Ontology (GO) analysis further supported functional convergences, as we observed that the alkalinization-responsive gene sets regulated by the two regulators were enriched in highly similar physiological processes (Fig. <a href="#Fig6" class="usa-link">6D</a>, Data <a href="#MOESM4" class="usa-link">S2</a>). Volcano plots also illustrated that the majority of DEGs (both up- and down-regulated) in the <em>dal81Δ/Δ</em> mutant were similarly misregulated in the <em>stp2Δ/Δ</em> mutant under alkaline pH-inducing conditions (Fig. <a href="#Fig6" class="usa-link">6E</a>). Importantly, among DEGs regulated by either Stp2 or Dal81, 77.9% showed overlapping expression between the two regulators (Fig. <a href="#Fig6" class="usa-link">6C</a>), with the majority involved in the metabolism of carboxylic acids, organic acids, oxoacids and amino acids (Fig. <a href="#Fig6" class="usa-link">6D</a>). Our RNA-seq data thus strongly suggest that Dal81 and Stp2 function similarly in the regulation of transcriptional response to pH alkalinization.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. The transcriptional profiles regulated by Dal81 and Stp2 are highly similar.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373999_41467_2025_62953_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/74b6/12373999/078e37bac1d7/41467_2025_62953_Fig6_HTML.jpg" loading="lazy" id="d33e1843" height="997" width="763" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Principal component analysis (PCA) of <em>C. albicans</em> gene expression at 6 h after transferring to SD or YC medium. SD: synthetic dextrose minimal medium; YC: YNB + 1% CAA, initial pH 4.5. <strong>B</strong> Heat map of gene expression in the indicated strain under different conditions. Each vertical line represents a sample, with gene expression displayed horizontally. Expression values were mean-normalized per gene and shown as z scores relative to the mean. <strong>C</strong> Venn diagram of alkalinization-responsive gene sets controlled by Dal81 and Stp2, based on RNA-Seq analysis of knockout mutants versus SN250. <strong>D</strong> Functional enrichment analysis of alkalinization-responsive genes specifically regulated by Dal81, Stp2, or commonly by both. <em>P</em> values were calculated via one-sided Fisher’s exact test with Benjamini-Hochberg correction; terms with FDR-adjusted <em>p</em> &lt; 0.05 are shown as –log10 (<em>p</em>. adjust). GO terms are clustered by Lin’s semantic similarity. <strong>E</strong> Volcano plots of differentially expressed genes (DEGs). SC5314 in SD vs. YC medium for 6 h, and <em>dal81Δ/Δ </em>or <em>stp2Δ/Δ</em> vs. SN250 in YC medium for 6 h. Vertical dotted lines indicate 2-fold change, and horizontal dotted lines mark adjusted <em>p</em> = 0.05. Statistical significance was determined using DESeq2 (two-sided Wald test with Benjamini-Hochberg correction). Shared DEGs across conditions are highlighted in yellow. Labeled genes were previously reported, with their deletion mutants showing alkalinization defects.</p></figcaption></figure><p id="Par22">To determine whether Dal81 and Stp2 act together and directly participate in the induction of alkalinization-responsive genes, we further performed chromatin immunoprecipitation followed by sequencing (ChIP-seq) to identify the promoters bound by these two regulators. Both Dal81-13xMyc and Stp2-3xHA were expressed under the control of their endogenous promoters. After culturing the respective strains in YNB + 1% CAA (initial pH 4.5) for 6 h, cells were harvested, and cross-linked chromatin from Dal81-13xMyc, Stp2-3xHA, or untagged control strains was precipitated using anti-Myc or anti-HA antibodies. Our analysis identified 2,624 and 2,599 specific binding sites for Dal81 and Stp2, respectively, with 85% of these sites localized to promoter regions (Fig. <a href="#Fig7" class="usa-link">7A</a>). Of significance, HOMER motif analysis revealed a shared core DNA-binding motif recognized by both Dal81 and Stp2 (Fig. <a href="#Fig7" class="usa-link">7B</a>). To infer direct target genes, we integrated ChIP-seq data with differential gene expression profiles<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>, yielding 240 target genes for Dal81 and 211 for Stp2 (Fig. <a href="#MOESM1" class="usa-link">S13A, B</a>, Data <a href="#MOESM5" class="usa-link">S3</a> and Data <a href="#MOESM6" class="usa-link">S4</a>). Among these, 99 genes were identified as common targets of both regulators (Fig. <a href="#Fig7" class="usa-link">7C</a>, Data <a href="#MOESM6" class="usa-link">S4</a>). Functional classification of these shared targets showed that 42 genes were synergistically activated, 56 genes were coordinately repressed, and 1 was antagonistically regulated by both regulators (Fig. <a href="#MOESM1" class="usa-link">S13C</a>, Data <a href="#MOESM6" class="usa-link">S4</a>). Strikingly, Gene ontology (GO) enrichment analysis linked the biological processes of these common targets to the metabolism of organic acids, oxoacids, carboxylic acids, and amino acids (Fig. <a href="#Fig7" class="usa-link">7D</a>, Data <a href="#MOESM7" class="usa-link">S5</a>). Key examples of these target genes include the putative NAD-specific glutamate dehydrogenase-encoding gene, <em>GDH2</em> (Fig. <a href="#MOESM1" class="usa-link">S14A</a>), the putative proline oxidase-encoding gene, <em>PUT1</em> (Fig. <a href="#MOESM1" class="usa-link">S14B</a>), and the phosphoenolpyruvate carboxykinase-encoding gene, <em>PCK1</em> (Fig. <a href="#MOESM1" class="usa-link">S14C</a>), which are involved in multiple biological processes (Fig. <a href="#MOESM1" class="usa-link">S13C</a>, Data <a href="#MOESM7" class="usa-link">S5</a>) and have previously been implicated in pH alkalinization<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. Dal81 and Stp2 coordinate to activate transcription of alkalinization-responsive genes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373999_41467_2025_62953_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/74b6/12373999/50bfd8ba19dd/41467_2025_62953_Fig7_HTML.jpg" loading="lazy" id="d33e2024" height="888" width="784" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Genome-wide distribution of candidate Dal81 or Stp2 binding regions from ChIP-seq assays in untagged wild-type, Stp2-HA, and Dal81-Myc strains grown in alkalinization-inducing medium (YNB + 1% CAA, initial pH 4.5). <strong>B</strong> De novo motif analysis using HOMER identified consensus binding motifs for Dal81 and Stp2 at their binding sites. <strong>C</strong> Venn diagram showing overlap between Dal81 and Stp2 binding sites. <strong>D</strong> GO enrichment analysis of common targets genes shared by Dal81 and Stp2. <strong>E</strong> ChIP-qPCR validation for Dal81 binding to <em>PUT1</em>, <em>GDH2,</em> and <em>PCK1</em> in Dal81-Myc or Dal81-Myc (<em>stp2Δ/Δ</em>) strains, shown as fold enrichment of the signal from immunoprecipitation relative to the input DNA. WT served as the negative control. <strong>F</strong> ChIP-qPCR validation for Stp2 binding to <em>PUT1</em>, <em>GDH2</em> and <em>PCK1</em> in Stp2-HA or Stp2-HA (<em>dal81Δ/Δ</em>) strains. <strong>G</strong> ChIP-qPCR validation for Stp2 binding to <em>PUT1</em>, <em>GDH2</em> and <em>PCK1</em> in Dal81-Myc or Dal81m21-Myc strains. <strong>H</strong><strong>–M</strong> RT- qPCR analysis of <em>PUT1</em>, <em>GDH2</em> and <em>PCK1</em> mRNA abundance in different strains under alkalinization conditions. Values for each gene were normalized against <em>ACT1</em>. Data are means ± SD of three biological replicates. Statistical significance via unpaired two-tailed Student’s <em>t</em>-test (<strong>E, F, H–J</strong>) and one-way ANOVA with Tukey’s multiple comparisons test (<strong>K–M</strong>). * <em>P</em> &lt; 0.05, ** <em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001, **** <em>P</em> &lt; 0. 0001; ns: not significant. Source data for (<strong>E–M</strong>) are provided as a Source Data file.</p></figcaption></figure><p id="Par23">While both regulators shared target genes enriched in the metabolism of organic acids, oxoacids, carboxylic acids, and amino acids, Stp2 uniquely controlled a broader repertoire of genes within these pathways (Fig. <a href="#MOESM1" class="usa-link">S15</a>, Data <a href="#MOESM8" class="usa-link">S6</a>). For example, Stp2 specifically regulated genes involved in organic acid, carboxylic acid, and amino acid transport (Fig. <a href="#MOESM1" class="usa-link">S15</a>, Data <a href="#MOESM8" class="usa-link">S6</a>), underscoring its role in nutrient uptake and metabolic flux. In contrast, Dal81 exhibited specialization in carbohydrate metabolism, with exclusive regulation of genes involved in sugar (<em>e.g</em>., hexoses and glucose) catabolism (Fig. <a href="#MOESM1" class="usa-link">S15</a>, Data <a href="#MOESM8" class="usa-link">S6</a>). These findings suggest that Dal81 and Stp2 coordinate shared metabolic processes (e.g., amino acid and organic acid metabolism) while governing distinct, non-overlapping pathways (Stp2 in transport; Dal81 in sugar metabolism), highlighting their functional complementarity in shaping cellular metabolism and environmental adaptation.</p>
<p id="Par24">The direct binding of Dal81 and Stp2 to downstream target genes such as <em>GDH2</em>, <em>PUT1</em> and <em>PCK1</em> was further validated by ChIP-qPCR (Fig. <a href="#Fig7" class="usa-link">7E, F</a>, Fig. <a href="#MOESM1" class="usa-link">S14D</a>). Interestingly, deletion of <em>DAL81</em> resulted in the loss of Stp2 enrichment at the promoters of these common targets, accompanied by downregulated gene expression (Fig. <a href="#Fig7" class="usa-link">7H–J</a>). In line with this observation, knocking out <em>STP2</em> yielded the same results. Additionally, reduced binding of Stp2 to target genes <em>GDH2</em>, <em>PUT1</em>, and <em>PCK1</em> was observed in the <em>DAL81m21/dal81Δ</em> strains, where the Dal81–Stp2 interaction was disrupted by mutations, compared to the control strain <em>DAL81/dal81Δ</em> (Fig. <a href="#Fig7" class="usa-link">7G</a>). RT-qPCR analysis of these three target genes (Fig. <a href="#Fig7" class="usa-link">7K–M</a>) revealed a downregulated expression trend for <em>GDH2</em> and <em>PCK1</em> in the residue mutant strains (Fig. <a href="#Fig7" class="usa-link">7K and L</a>). These data strongly support that Dal81 forms a complex with Stp2 and co-regulates the expression of most of their common target genes.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Dal81 and Stp2 are essential for regulating the commensalism and pathogenicity of <em>C. albicans</em>
</h3>
<p id="Par25"><em>C. albicans</em> typically survives as a commensal of the mammalian microbiome and also acts as an opportunistic pathogen causing disseminated infection<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Notably, pH levels in mammalian mucosal niches vary widely, ranging from acidic to alkaline, and fungi like <em>C. albicans</em> must adapt to this pH spectrum to survive, indicating that pH alkalinization, co-regulated by Dal81 and Stp2, may contribute to the commensalism and/or virulence of <em>C. albicans</em>. To test this speculation, we first evaluated the intestinal colonization capacity of the <em>dal81Δ/Δ</em> or <em>stp2Δ/Δ</em> knockout strain using the intestinal commensalism mouse model of <em>C. albicans</em><sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. Knockout of either <em>DAL81</em> or <em>STP2</em> led to a significant decrease in intestinal colonization (Fig. <a href="#Fig8" class="usa-link">8A</a>). Most significantly, simultaneous deletion of both <em>DAL81</em> and <em>STP2</em> resulted in the complete absence of <em>C. albicans</em> colonization within the intestine (Fig. <a href="#Fig8" class="usa-link">8B</a>), strongly validating their roles as crucial regulators of <em>C. albicans</em> gastrointestinal colonization. Given that successful gut colonization requires interactions with both the host and the microbes, we further evaluated the specific contributions of Dal81 and Stp2 using competitive infection assays. Immunosuppressed ICR mice were gavaged with 1:1 mixtures of wild-type <em>C. albicans</em> and either the <em>dal81Δ/Δ</em> or <em>stp2Δ/Δ</em> mutant, and the abundance of each strain in fecal pellets was monitored by qPCR over 8 days. Throughout the time course, both the <em>dal81Δ/Δ</em> and <em>stp2Δ/Δ</em> mutants showed a consistent competitive disadvantage relative to the wild-type strain (Fig. <a href="#Fig8" class="usa-link">8C</a>), indicating that Dal81 and Stp2 promote commensalism through mechanisms that are likely independent of the host immune response, as the assay was performed in immunosuppressed mice.</p>
<figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8. Dal81 and Stp2 are key regulators of <em>C. albicans</em> commensalism and pathogenesis.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373999_41467_2025_62953_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/74b6/12373999/1a34bfca845d/41467_2025_62953_Fig8_HTML.jpg" loading="lazy" id="d33e2298" height="602" width="800" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong>
<em>C. albicans dal81Δ/Δ</em> and <em>stp2Δ/Δ </em>mutant strains showed defective gastrointestinal colonization compared to wild-type (WT) or DAL81comp. strains. Female C57BL/6 mice (<em>n</em> = 8) were orally gavaged with 10<sup>8</sup> CFU of indicated strains and colonization was measured in fecal pellets on days 3, 5 and 8 post-inoculations via CFU/g quantification. <strong>B</strong> The <em>dal81Δ/Δstp2Δ/Δ</em> double mutant exhibited complete colonization failure. Mice (<em>n</em> = 8) were gavaged as in (<strong>A</strong>), with colonization assessed on day 3. <strong>C</strong> Dal81 and Stp2 promote gut commensalism in immunosuppressed ICR mice (<em>n</em> = 8/group), inoculated with 1:1 mixtures of WT and <em>dal81Δ/Δ</em>, DAL81comp., or <em>stp2Δ/Δ</em>(1 × 10<sup>8</sup> CFU total). Strain abundance in inoculum (<strong>I</strong>) and after recovery from fecal pellets (R) was quantified by strain-specific qPCR. <strong>D</strong> Deletion of <em>DAL81</em> or <em>STP2</em> attenuated <em>C. albicans</em> virulence, with Wild-type <em>DAL81</em> complementation restoring function. C57BL/6 mice (<em>n</em> = 12) were tail-vein infected with 2 × 10<sup>5</sup> CFU of indicated strains and survival was monitored at the specified timepoints. <strong>E</strong> The <em>dal81Δ/Δstp2Δ/Δ </em>double mutant showed complete virulence loss. Mice (<em>n</em> = 8) were infected with 1 × 10<sup>6</sup> CFU as in (<strong>D</strong>). <strong>F</strong> Survival of mice (<em>n</em> = 8) after intravenous (i.v.) challenge with the <em>DAL81m21</em>/<em>dal81Δ</em> mutant. Experiments were repeated twice with similar results. Statistical significance via two-way ANOVA with Tukey’s test (<strong>A, C</strong>), and one-way ANOVA with Tukey’s test (<strong>B</strong>). Survival data (<strong>D, E</strong>) were evaluated by Kaplan-Meier and log-rank (Mantel–Cox) test. ns, not significant. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par26">On the other hand, the reduced survival of <em>dal81Δ/Δ</em> and <em>stp2Δ/Δ</em> mutants in macrophages raised the question of whether these strains would be compromised during disseminated infection. Therefore, we tested the <em>dal81Δ/Δ</em> and <em>stp2Δ/Δ</em> strains in the standard mouse tail-vein injection model of disseminated hematogenous candidiasis<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Both mutants exhibited significant virulence attenuation compared with the wild-type and <em>DAL81</em> complemented strains (Fig. <a href="#Fig8" class="usa-link">8D</a>), as we found that mice infected with the wild-type strain had a median survival time of 4 days, whereas survival times for those infected with the mutant strains remained undefined due to prolonged survival, indicating that Dal81 and Stp2 are required for virulence in this infection model. Statistical analysis confirmed that the survival curve for the <em>dal81Δ/Δ</em> and <em>stp2Δ/Δ</em> strains were significantly different from those of the wild-type or complemented cells (<em>P</em> &lt; 0.0001). Strikingly, the <em>dal81Δ/Δstp2Δ/Δ</em> double knockout strain displayed complete virulence attenuation relative to the wild-type (Fig. <a href="#Fig8" class="usa-link">8E</a>), underscoring the indispensable and synergistic roles of Dal81 and Stp2 in fungal pathogenesis. To further explore how the Dal81-Stp2 interaction and pH alkalinization contribute to <em>C. albican</em>s virulence, we evaluated the pathogenic potential of the <em>DAL81m21</em>/<em>dal81Δ</em> mutant, defective in Dal81-Stp2 interaction, in the systemic infection model. Notably, this mutant was significantly attenuated in virulence compared to the wild-type control, as we observed that mice infected with the <em>DAL81</em>/<em>dal81Δ</em> control strain had an average survival time of 15 days, whereas survival in the <em>DAL81m21</em>/<em>dal81Δ</em> group remained undefined due to limited mortality over the experimental period (Fig. <a href="#Fig8" class="usa-link">8F</a>). These findings identify specific Dal81-Stp2 interaction sites as critical determinants of both pH alkalinization and <em>C. albicans</em> virulence.</p>
<p id="Par27">In summary, our animal model experiments fully demonstrate that Dal81 and Stp2 coordinately regulate environmental alkalinization and play a pivotal role in modulating both commensalism and pathogenicity of <em>C. albicans</em>, linking their molecular interactions to in vivo fitness and virulence.</p></section></section><section id="Sec12"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par28">Microorganisms encounter various stresses during their inhabitance, and adaptation serves as a crucial strategy for them to enhance the utilization of non-preferred substrates and increase the tolerance to stress. Fungi, in particular, have the ability to thrive in a diverse range of habitats and are considered to be among the most extremotolerance microorganisms due to their morphological plasticity and diverse lifestyles. During its long-term life history in humans, <em>C. albicans</em>, a common fungal commensal and opportunistic pathogen, has evolved sophisticated strategies to rapidly sense, adapt to and even change the fluctuating microenvironments within the host. Meanwhile, the ability to adapt to or modify the environmental pH has been known to be crucial for the survival and growth of this fungus in various host niches<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Literatures have shown that <em>C. albicans</em> is able to actively alkalinize the host environment<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> and the transcription factor Stp2 was identified to be the master regulator controlling the process of pH alkalinization<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Here, we provide compelling evidence that the transcription factor Dal81 also plays a critical role in pH alkalinization in <em>C. albicans</em>. Notably, our findings demonstrate that this important environmental pH-modifying behavior requires the coordinated activity of Dal81 and Stp2. Deletion of either transcription factor significantly impairs the organism’s ability to elevate extracellular pH. To illustrate the proposed mechanism, we developed a schematic model (Fig. <a href="#MOESM1" class="usa-link">S17</a>) depicting the coordination between Dal81 and Stp2 in regulating gene transcription induced by environmental alkalinization. This model highlights the cooperative interaction of the two factors within the nucleus, where they orchestrate the expression of downstream genes required for the survival and propagation of <em>C. albicans</em> in the host.</p>
<p id="Par29">Dal81 is a fungus-specific protein characterized by a Gal4-like zinc finger DNA binding domain<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> and its regulatory role in gene expression has been addressed in multiple fungi. For example, Dal81 in <em>S. cerevisiae</em> was found to be important for nitrogen metabolism by regulating the transcriptional activation of a significantly large number of genes associated with the catabolism of nitrogen sources such as urea, allantoin, arginine, and γ-aminobutyrate (GABA)<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>–<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. The lack of <em>DAL81</em> hinders the maximal induction of these genes<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup> such as the GABA-inducible genes involved in GABA utilization (<em>UGA1</em>, <em>UGA2</em>, and <em>UGA4</em>)<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup> and the allophanate-inducible genes involved in the utilization of urea (<em>DUR1-2</em> and <em>DUR3</em>), allantoate (<em>DAL7</em>), and arginine (<em>CAR2</em>)<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>–<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Interestingly, Dal81 is also required for the activation of <em>AGP1</em> encoding an amino acid permease that mediates the uptake of a wide range of amino acids from the external environment<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Mechanistically, Dal81 is activated by the Ssy1-Ptr3-Ssy5 (SPS) sensor system<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>,<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup> and acts to facilitate the binding of a list of other transcription factors to the promoters of their downstream target genes. For example, the activation of Dal81 was found to significantly promote the binding of Stp1 and Stp2 to the <em>AGP1</em> promoter<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>–<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup> as well as the recruitment of Uga3 and Gal11 to the promoters of <em>UGA1</em> and <em>UGA4</em><sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Interestingly, a divergent functional role for the transcriptional regulator Dal81 has been characterized in <em>Candida parapsilosis</em>. In this organism, the induction of genes involved in GABA metabolism occurs independent of Dal81 and its deletion does not impair the ability of <em>C. parapsilosis</em> to acquire nitrogen from either GABA or allantoin<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>, marking a key departure from its role in <em>C. albicans</em>. Instead, Dal81 serves to repress arginine synthesis under preferred nitrogen conditions<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. A further transcription network analysis suggests that Dal81 either initiates or assists Stp2 during the initiation of nitrogen mobilization from the vacuolar storage via Avt11<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. In <em>C. albicans</em>, we found that Dal81 is required for regulation of gene expression involved in metabolism of organic acids, oxoacids, carboxylic acids, amino acids and carbohydrates. More importantly, the induction of these Dal81 targets enhances its crucial roles in environmental pH alkalinization, yeast-to-hyphal morphogenesis and cellular fitness, linking its role to <em>C. albicans</em> commensalism and pathogenicity.</p>
<p id="Par30">Studies have demonstrated that the utilization of amino acids in <em>S. cerevisiae</em> and <em>C. albicans</em> requires the regulatory activities of two transcript factors, Stp1 and Stp2<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Stp2 regulates the expression of genes encoding amino acid permeases and catabolic machinery of amino acids<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>, whereas Stp1 is considered as an important regulator in controlling the expression of genes associated with peptide and protein utilization<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Intriguingly, a recent study by Miramon et al. found that Stp2 acts as the predominant regulator of genes involved in amino acid and peptide utilization, with Stp1 paying a minor role, supported by the finding that gene expression patterns in <em>stp1Δ/Δ</em> mutant cells were nearly identical to those in wild-type cells<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. In our study, we found that Stp2 is a major regulator of the C. <em>albicans</em> response to amino acids, primarily controlling the metabolism and transport of organic acids, carboxylic acids, and amino acids. This observation aligns with previously published transcriptomic data from <em>C. albicans</em> cells lacking <em>STP2</em><sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. However, we detected approximately four times more differentially expressed genes than reported in that prior study, a discrepancy that may be attributed to differences in strain background. Importantly, our results revealed that Stp2 and Dal81 form a complex that facilitates promoter binding and transcriptional activation of downstream target genes. This cooperative interaction likely explains why deletion of either regulators results in a failure of pH alkalinization. Thus, our work defines an uncharacterized regulatory mode operating in amino acid uptake and metabolism of <em>C. albicans</em>, which requires the coordination of these two transcription factors. However, we have also observed distinct roles of Stp2 and Dal81 in pH alkalinization. For instance, the <em>dal81Δ/Δstp2Δ/Δ</em> double mutant exhibited an additive defect in alkalinization in both YNB + 1% CAA and artificial saliva. Additionally, in GM-BCP medium, the <em>dal81Δ/Δstp2Δ/Δ</em> double mutant showed a more pronounced reduction in ammonia release compared to the <em>dal81Δ/Δ</em> and <em>stp2Δ/Δ</em> single mutants. Moreover, Dal81 is specifically required for effective alkalinization in vaginal simulating fluid. These observations suggest that beyond amino acid metabolism, other Dal81- or Stp2-dependent metabolic endpoints may also contribute to the exacerbated alkalinization defects observed in the <em>dal81Δ/Δstp2Δ/Δ</em> double knockout mutant. Notably, the differences in alkalinization phenotypes among the mutants across various culture media may stem from variations in the metabolic utilization of specific components in each medium. Specifically, YNB + 1% CAA primarily contains casamino acids, while artificial saliva includes both casamino acids and mucin. In contrast, vaginal simulating fluid is mainly composed of glucose, lactic acid, and acetic acid. Finally, GM-BCP medium contains glycerol and yeast extract, providing a rich mix of nucleotides, proteins, amino acids, sugars, and trace elements. Our RNA-seq and ChIP-seq data reveal that, in addition to shared target genes involved in amino acid metabolism, which are coregulated by Dal81 and Stp2, there are other amino acid metabolism-related genes specifically regulated by either Dal81 or Stp2. Moreover, genes associated with amino acid transport are uniquely controlled by Stp2. These findings help explain the enhanced alkalinization defects observed in the <em>dal81Δ/Δstp2Δ/Δ</em> double knockout mutant in YNB + 1% CAA and artificial saliva. Given that Dal81 regulates glucose metabolism (Fig. <a href="#MOESM1" class="usa-link">S15</a>) and is pivotal for the utilization of lactic acid and acetic acid (Fig. <a href="#MOESM1" class="usa-link">S16</a>), together with the previous observation that glucose suppresses environmental alkalinization, we speculate that the impaired utilization of these acidic compounds and glucose in the <em>dal81Δ/Δ</em> mutant likely underlies the notable alkalinization defects observed in vaginal simulating fluid. The combined disruption of sugar metabolism (via Dal81) and amino/organic acid metabolism (via Dal81 and Stp2) in the double mutant probably impairs multiple alkalinization pathways. This could account for the more pronounced reduction in ammonia release observed in the <em>dal81Δ/Δstp2Δ/Δ</em> strain on solid GM-BCP medium (Fig. <a href="#Fig1" class="usa-link">1H</a>). Given that Dal81 regulates glucose metabolism (Fig. <a href="#MOESM1" class="usa-link">S15</a>) and is pivotal for the utilization of lactic acid and acetic acid (Fig. <a href="#MOESM1" class="usa-link">S16</a>), together with the previous observation that glucose suppresses environmental alkalinization, we speculate that the impaired utilization of these acidic compounds and glucose in the <em>dal81Δ/Δ</em> mutant likely underlies the notable alkalinization defects observed in vaginal simulating fluid. The combined disruption of sugar metabolism (via Dal81) and amino/organic acid metabolism (via Dal81 and Stp2) in the double mutant probably impairs multiple alkalinization pathways. This could account for the more pronounced reduction in ammonia release observed in the <em>dal81Δ/Δstp2Δ/Δ</em> strain on solid GM-BCP medium (Fig. <a href="#Fig1" class="usa-link">1H</a>).</p>
<p id="Par31">Notably, a previous study reported that mutant cells lacking <em>STP2</em> exhibited a marked delay in alkalinization when grown in artificial saliva: the pH reached only 5.3 after 5 h but increased to ~8.2 by 24 h<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. However, in our study, using the SN250 genetic background as the wild-type control, we observed that after 56 h, the pH of the <em>stp2Δ/Δ</em> culture reached only 6.3, compared to 7.8 in the wild-type strain (Fig. <a href="#Fig2" class="usa-link">2C</a>). We speculate that the inconsistency in results may be due to differences in strain background. Our <em>stp2Δ/Δ</em> mutant was generated using <em>HIS</em> and <em>LEU</em> as selection markers, whereas the <em>STP2</em>-deficient strain in the previous report was constructed in the SC5314 background using the SAT-flipper method. These findings suggest that <em>C. albicans</em> may employ distinct alkalinzation strategies tailored to specific niches, with Dal81-dependent alkalinization for colonization in the vaginal tract, Stp2-dependent alkalinization to facilitate colonization in the oral cavity, and a combined Dal81- and Stp2-dependent mechanism to support colonization in the gut.</p>
<p id="Par32">In addition to its crucial role in alkalinization, we found that Dal81 is important for regulating hyphae formation both in vitro and in vivo. Notably, our analysis identified an enrichment of hyphae-specific genes (<em>HGC1</em>, <em>UME6</em>, <em>ALS3</em>, <em>HWP1</em>, <em>ECE1</em>, and <em>NRG1</em>) that are abnormally expressed in the <em>dal81Δ/Δ</em> mutant (Fig. <a href="#MOESM1" class="usa-link">S18</a>). Among these genes, two regulators are particularly noteworthy, as Ume6 is recognized as the primary regulator of hyphal extension<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>, especially in response to neutral pH<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>, and Hgc1 controls the expression of <em>UME6</em><sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Although neutral or alkaline pH is one of several cues that induce the yeast-to-hypha transition in <em>C. albicans</em>, and the <em>dal81Δ/Δ</em> mutant does not exhibit an inherent defect in responding to neutral pH, it remains unclear whether the hyphal formation defect observed in <em>dal81Δ/Δ</em> mutant within macrophage phagosomes stems from impaired alkalinization on this environment. The hyphal defect could also be attributed to the stressful conditions within macrophages or other host-derived factors, which warrants further investigation in future studies.</p>
<p id="Par33">In its commensal state, <em>C. albicans</em> is typically characterized by the absence of filamentation, epithelial invasion, and host cell damage<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. Interestingly, recent studies have argued that virulence traits, long recognized primarily for their role in inducing host diseases, such as hyphae formation and production of the hyphal-specific toxin candidalysin, may play an unexpected active role in facilitating the establishment of <em>C. albicans</em> as a gut commensal<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>–<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Specifically, the study from Bennett and colleagues<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> revealed that producing hyphae actually enhances the ability of <em>C. albicans</em> to thrive as a commensal when colonizing mice with an intact, normal gut microbiota. The strain capable of switching to filamentous forms gaines a competitive advantage over those locked in the yeast state and importantly, this competitive benefit relies on hyphal-specific toxin candidalysin, which functions to disrupt the metabolic activity, growth patterns, glucose uptake. Based on these studies, it is plausible that pathogenicity is not an intrinsic characteristic of fungi themselves, instead, it arises as a consequence of disrupted or imbalanced interactions between the microbe and its host. Given that Dal81 plays a significant role in regulating both extracellular pH alkalinization and hyphal formation in <em>C. albicans</em>, these two factors likely contribute to the critical involvement of Dal81 in promoting commensalism and virulence. Supporting this idea, the <em>dal81Δ/Δ</em> mutant displayed notable deficiencies in gut colonization and reduced virulence in a systemic infection model. In similar, the <em>stp2Δ/Δ</em> mutant also showed impairments in gut colonization and diminished virulence. Remarkably, the double knockout strain (<em>dal81Δ/Δstp2Δ/Δ</em>) exhibited a total loss of gut colonization and a complete absence of virulence, pointing to an additive effect of Dal81 and Stp2 in driving both commensalism and pathogenicity. This combined impact may stem not only from the distinct sets of genes regulated by Dal81 and Stp2. Even among the genes they co-regulate, the contribution of each transcription factor may vary based on the biological context. For example, the expression of <em>GDH2</em> was further downregulated in the <em>dal81Δ/Δstp2Δ/Δ</em> mutant compared to the <em>dal81Δ/Δ</em> single mutant, while <em>PUT1</em> displayed significantly different expression patterns across the single and double mutant strains, observations that suggest context-dependent differences in regulatory dominance between the two factors. It is important to note that our current evidence supporting the role of both Dal81- and Stp2-mediated regulation of pH alkalinization and hyphal formation in gut colonization of <em>C. albicans</em> remains incomplete. Further investigations, such as exploring the link between pH alkalinization and candidalysin production, are necessary to strengthen these connections. On the other hand, we cannot exclude the possibility that additional factors, such as the gut-specific morphological variants, may also contribute to Dal81- and Stp2-mediated regulation of gut colonization. A previous study by Pande et al. <sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> highlighted that the “White-to-GUT” switch promotes <em>C. albicans</em> commensalism in the gastrointestinal (GI) tract, with a specialized, gastrointestinally induced transition (GUT) cell type, regulated by the transcription factor Wor1, actively promoting the maintenance of a commensal lifestyle in this fungus. Remarkably, GUT cells exhibit a reprogrammed cellular metabolism that is adapted to optimize nutrient utilization in the GI environment<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. Future studies will focus on investigating whether the regulatory networks controlled by Dal81 and Stp2 influence the White-to-GUT switch and GUT-associated metabolic adaptations.</p>
<p id="Par34">Our RNA-seq and ChIP-seq analyses revealed that Dal81 and Stp2 regulate the metabolism of organic acids, oxoacids, and carboxylic acids. Given that <em>C. albicans</em> cells depend on alternative carbohydrate sources for nutrition within the host environment, the reduced intestinal colonization and survival observed in the mutant strains may reflect an underlying fitness impairment. To investigate this, we evaluated whether deleting <em>DAL81</em> and <em>STP2</em> affects the ability to utilize alternative carbon sources in vitro, by assessing the growth of wild-type, <em>dal81Δ/Δ</em>, <em>stp2Δ/Δ</em>, Dal81 comp., and <em>dal81Δ/Δstp2Δ/Δ</em> strains in YEP medium supplemented with various alternative carbohydrates. The <em>dal81Δ/Δ</em> mutant exhibited significant growth defects when cultured with acetate, lactic acid, and citrate, along with milder impairments when grown on oleic acid, ethanol, and N-acetylglucosamine (GlcNAc). Similarly, the <em>stp2Δ/Δ</em> strain displayed pronounced growth deficiencies on lactic acid and citrate. Notably, the <em>dal81Δ/Δstp2Δ/Δ</em> double mutant showed far more severe growth impairments on acetate, citrate and GlcNAc compared to either single-knockout mutant. These findings suggest that Dal81- and Stp2 -dependent metabolic adaptation to alternative carbon sources may contribute to both commensalism and virulence in <em>C. albicans</em>, potentially explaining why the <em>dal81Δ/Δstp2Δ/Δ</em> strain exhibited a complete loss of gut colonization and full attenuation of virulence. Consistent with this, both the <em>dal81Δ/Δ</em> and <em>stp2Δ/Δ</em> mutants continued to show reduced intestinal colonization compared to the wild-type strain even in immunosuppressed mice (Fig. <a href="#Fig8" class="usa-link">8C</a>), indicating that their colonization defects are likely driven by impaired alkalinization, defective hyphal formation, and compromised metabolic adaptation, rather than altered interactions with the host immune system. Furthermore, in addition to amino acid metabolism known to raise extracellular pH, previous studies have shown that the metabolism of carboxylic acids (<em>e.g</em>., pyruvate, α-ketoglutarate, or lactate) and the alternative carbon source GlcNAc can also raise extracellular pH<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. As such, beyond their roles as nutrients, defects in the metabolism of specific alternative carbohydrates in the <em>dal81Δ/Δ</em> and <em>stp2Δ/Δ</em> mutants may lead to a failure to alkalinize the extracellular environment within the host. Based on these observations, we hypothesize that the colonization and virulence defects in the <em>dal81Δ/Δ</em> and <em>stp2Δ/Δ</em> mutants arise from an overall fitness impairment driven by two key factors: impaired extracellular alkalinization and an inability to effectively utilize host-derived nutrients.</p>
<p id="Par35">Targeting transcription factors essential for fungal survival and proliferation within the host may represent a promising approach for advancing antifungal drug discovery. In our study, Dal81 emerged as a critical regulator in the commensalism and pathogenicity of <em>C. albicans</em>, particularly notable for its specificity to the fungal kingdom, which positions it as an appealing target for the development of selective therapeutics. Importantly, we uncovered a physical interaction between transcription factors Dal81 and Stp2. Further insights from protein modeling and structural analysis, conducted using AlphaFold 3 and ChimeraX, revealed that Dal81 and Stp2 adopt a “key-and-lock” structural configuration, with 21 potential interaction sites identified at their binding interface. Mutagenesis of these sites resulted in reduced binding of Dal81 and Stp2 to their downstream target genes, coupled with diminished expression of select target genes. Functionally, these mutations impaired the ability to alkalinize the extracellular environmental and weakened <em>C. albicans</em> virulence. While these findings support the role of the Dal81–Stp2 interaction in coordinating gene regulation linked to pH alkalinization and fungal virulence, additional studies are required to pinpoint the specific residues critical for this interaction. Given that the coordinated regulation mode between Dal81 and Stp2 is necessary for alkalinization and plays a key role in modulating commensalism and pathogenicity of <em>C. albicans</em>, screening for small molecules that disrupt their interaction holds promise as a potential therapeutic strategy for fungal infections. In summary, our study enhances understanding of how <em>C. albicans</em> adapts to the diverse microenvironments within host, laying groundwork for the development of improved therapeutics to combat this pressing global health challenge.</p></section><section id="Sec13"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec14"><h3 class="pmc_sec_title">Ethics statement</h3>
<p id="Par36">The animal experiments were conducted in strict compliance with the Regulations for the Care and Use of Laboratory Animals issued by the Ministry of Science and Technology of the People’s Republic of China. All efforts were made to minimize animal suffering. The animal experiments performed in this study were approved by Institutional Animal Care and Use Committee (IACUC) at the Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences (Permit Number: A2020025).</p></section><section id="Sec15"><h3 class="pmc_sec_title">Animals</h3>
<p id="Par37">Female C57BL/6 mice (6–8 weeks old, 18–20 g) and female ICR mice (6–8 weeks old, 23–25 g) were purchased from Beijing Vital River Laboratory Animal Technology Company (Beijing, China). All mice were housed in a pathogen-free animal facility under controlled conditions (temperature: 21 °C; relative humidity: 50–70%; 12-hour light/dark cycle) and had ad libitum access to food and water. Mice were given free access to food and water throughout the study. All experimental procedures were conducted in accordance with the protocol approved by the Institutional Animal Care and Use Committee (IACUC) at Shanghai Institute of Immunity and Infection, Chinese Academy of Sciences, China. The study did not include sex-based analysis.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Cell isolation and culture</h3>
<p id="Par38">All cell lines used in this study, including RAW264.7 (ATCC TIB-71) and J774A.1 (ATCC TIB-67), were obtained from the American Type Culture Collection (ATCC). Cells were cultured in Dulbecco’s Modified Eagle’s Medium (DMEM) supplemented with 10% fetal bovine serum (FBS) and 1% penicillin-streptomycin. For experiments, cells were seeded at a density of 2 × 10<sup>5</sup> cells per 35-mm dish or 5 × 10<sup>4</sup> cells per well in 96-well plates, and used 16 h post-seeding. All cell cultures were incubated at 37 °C in a 5% CO<sub>2</sub> atmosphere and maintained under humidified conditions.</p>
<p id="Par39">Bone marrow-derived macrophages (BMDMs) were prepared as described previously<sup><a href="#CR65" class="usa-link" aria-describedby="CR65">65</a></sup>. Briefly, bone marrow cells were isolated from the femurs and tibias of C57BL/6 mice (6–8 weeks old) and subjected to hypotonic lysis to remove erythrocytes. Cells were cultured in RPMI-1640 medium supplemented with 10% fetal bovine serum (FBS), 30% conditioned medium from L929 cells (as a source of macrophage colony-stimulating factor), and 1% penicillin–streptomycin. Nonadherent cells were removed, and adherent cells were passaged every 3 days. After 6 days of incubation, BMDMs were seeded at a density of 2 × 10<sup>5</sup> cells per 35-mm dish or 5 × 10<sup>4</sup> cells per well in 96-well plates, and used for infection assays.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Media and growth conditions</h3>
<p id="Par40"><em>C. albicans</em> strains were routinely grown in YPD (2% peptone, 1% yeast extract, 2% glucose) or YNB + 1% CAA medium (1.7% yeast nitrogen base without amino acids, 0.5% ammonium sulfate, 1% Casamino Acids) at 30 °C. For liquid cultures, the strains were grown in YPD at 30 °C with agitation on a rotary shaker. Transformants were selected on YPD medium supplemented with nourseothricin (200 μg/ml). Alkalinization assays in liquid medium were primarily conducted using YNB + 1% CAA medium or medium 199. In certain experiments, pH alkalinization assays were performed in artificial saliva (initial pH 4.3) and vaginal simulating fluid (initial pH 4.5), using formulations adapted from reference<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. The artificial saliva formulation (per liter) contained: 1.7 g yeast nitrogen base (YNB) without amino acids and ammonium sulfate, 5.0 g casamino acids, 2.5 g mucin, 1.1 g KCl, 0.5 g NaCl, 14 mg choline chloride, 10 mg sodium citrate, and 1.0 mg ascorbate. The vaginal simulating fluid formulation (per liter) included: 2 g glucose, 2 g lactic acid, 1 g acetic acid, 0.4 g urea, 0.25 g mucin, 0.16 g glycerol, 0.018 g bovine serum albumin, 3.51 g NaCl, 1.4 g KOH, and 0.222 g CaCl<sub>2</sub>. For solid-phase alkalinization assays, <em>C. albicans</em> strains were grown on GM-BCP or GM-BCG (1% yeast extract, 30 mM CaCl<sub>2</sub>, 3% glycerol, 0.01% bromocresol purple or bromocresol green, 2% agar) as previously described<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>,<a href="#CR66" class="usa-link" aria-describedby="CR66">66</a></sup> The pH of the medium was adjusted with HCl or NaOH. Artificial saliva medium and vaginal simulating fluid medium were prepared as previously described<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Strain and plasmid construction</h3>
<p id="Par41">The strains, primers, and plasmids used in this study are listed in Supplementary Data <a href="#MOESM9" class="usa-link">7</a>, Supplementary Data <a href="#MOESM10" class="usa-link">8</a>, and Supplementary Table <a href="#MOESM1" class="usa-link">1</a>, respectively. All <em>C. albicans</em> strains used in this study were derived from the reference strain SC5314. Corresponding gene deletion, complementation, and overexpression strains were generated as previously described<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>.The <em>C. albicans</em> mutant library, consisting of 674 knockout mutant strains, was sourced from AD Johnson<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. The <em>C. albicans</em> mutant strains with disrupted Dal81-Stp2 interaction sites were generated via homologous recombination. Point mutations were introduced into <em>dal81Δ/Δ</em> by substituting specific residues with alanine (Ala), either through synthesis of mutated <em>DAL81</em>genes or site-directed mutagenesis using specific primers. PCR-amplified products were treated with DpnI to digest the original plasmid template. The final constructs included a 950 bp region upstream of the <em>DAL81</em> start codon, the full-length <em>DAL81</em> open reading frame (ORF) containing either 21 or 2 alanine substitutions (or wild-type <em>DAL81</em>), and a C-terminal 13xMyc tag. These fragments were assembled into the backbone of the <em>DAL81</em> addback plasmid to generate plasmids bCB557, bCB558, and bCB556, respectively. The resulting plasmids were digested with <em>KpnI</em> and <em>XhoI</em> and transformed into the <em>dal81Δ/Δ</em> strain expressing HA-tagged Stp2. Transformants were selected on SC-Arg medium and verified by colony PCR. To construct the <em>DAL81</em>gene deletion strain in the <em>C. albicans</em> SC5314 background, we used a modified version of the <em>SAT1</em>-flipper method as previously described<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. Briefly, a <em>SAT1–FLP</em> cassette flanked by 88 bp of homology immediately to the 5′ and 3′ untranslated regions of the <em>DAL81</em> open reading frame (ORF) was amplified from the pSFS2A plasmid<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. The PCR product was introduced into the <em>C. albicans</em> SC5314 strain by chemical transformation, and transformants were selected on YPD medium containing nourseothricin. Correct cassette integration was verified by PCR. To remove the <em>SAT1–FLP</em> cassette, cells were cultured in YPM medium (1% yeast extract, 2% peptone, 2% maltose) to induce <em>FLP</em> recombinase gene expression and excise the marker. The same PCR product was then used to delete the second allele of <em>DAL81</em>, resulting in the generation of the homozygous deletion strain YCB1391 (<em>dal81Δ/Δ</em> M1). Primers used for PCR amplification and validation were synthesized by Genewiz (Jinweizhi Biotechnology Co., Ltd.).</p></section><section id="Sec19"><h3 class="pmc_sec_title">In vitro growth assay</h3>
<p id="Par42">For growth curves in liquid medium, cells from overnight cultures were diluted to an initial OD<sub>600</sub> of 0.15 into the indicated medium. At indicated time intervals, the OD<sub>600</sub> was measured. Data are presented from three biological replicates and plotted in Graphpad Prism. For agar plate assays, overnight yeast cultures were washed and adjusted to an OD<sub>600</sub> of 1.0 in sterile water. Serial 10-fold dilutions were prepared, and 5 μl aliquots of each dilution were spotted onto appropriate agar plates. To assess growth on alternative carbon sources, <em>C. albicans</em> cells were cultivated at 37 °C in YEP medium (1% yeast extract, 2% Facto-Peptone) supplemented with the indicated alternative carbohydrates. These included carboxylic acids (acetate, lactic acid, α-ketoglutarate, pyruvic acid), fatty acids (oleic acid), and other substrates such as ethanol, glycerol, citrate, mannitol, sorbitol, and N-acetylglucosamine (GlcNAc). Serial 10-fold dilutions were prepared, and 5 μl aliquots from each dilution were spotted onto agar plates containing the respective carbon sources.</p></section><section id="Sec20"><h3 class="pmc_sec_title">In vitro screen for genes required for pH alkalinization</h3>
<p id="Par43">The <em>C. albicans</em> mutant library was screened to identify mutants defective in pH alkalinization, following a previously described method<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> with minor modifications. Briefly, all mutant strains and the isogenic wild-type reference strain SN250 were grown in YPD at 30 °C for 2 days in 96-deep-well microplates. Subsequently, 5 μl of each culture was transferred into fresh unbuffered medium 199 (initial pH 4.5) in 96-well cell culture microplates. Plates were incubated at 37 °C for 24 h, and alkalinization was assessed by colorimetric change. Candidate mutants displaying reduced or absent alkalinization were further validated using both medium 199 and YNB + 1% CAA medium under the same conditions as described above.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Alkalinization assay</h3>
<p id="Par44">Alkalinization assays were performed as previously described<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> with minor modifications. For liquid culture assays, <em>C. albicans</em> strains were grown overnight in YPD, harvested, washed with water, and diluted to a starting OD<sub>600</sub> of 0.2 in the YNB + 1% CAA medium. Cultures were incubated at 37 °C with shaking, and medium pH was measured at the indicated time intervals. For solid medium assays, 12-well plates were prepared with 2 ml of GM-BCP or GM-BCG per well. Strains were grown overnight in YPD, harvested, washed, and resuspended in water to an OD<sub>600</sub> of 1.0. A 5 μl aliquots of this suspension was spotted into each well (one strain per well). Plates were incubated at 37 °C, and alkalinization was assessed visually by the appearance of a purple (GM-BCP) or blue (GM-BCG) halo surrounding the fungal colonies, indicating an increase in pH.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Hyphal induction assay</h3>
<p id="Par45">Wild-type and mutant <em>C. albicans</em> cells were cultured overnight in YPD medium at 30 °C, harvested, washed with sterile water, and diluted to achieve a final OD<sub>600</sub> of 0.1 in either YNB + 1% CAA medium or medium 199. The pH of the YNB + 1% CAA medium was adjusted to 4.5 with HCl. Medium 199 was either buffered to pH 4.5 using 0.1 M Tris-0.1 M MOPS, or to pH 7.5 using 0.15 M HEPES. Cultures were incubated at 37 °C with shaking. Samples were collected at 0 h, 5 h or 8 h and subjected to microscopic observation to assess hyphal morphology.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Ammonia release assay</h3>
<p id="Par46">The release of ammonia by <em>C. albicans</em> cells during alkalinization was measured based on a previously described method<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, with some modification. Briefly, cells were grown overnight in YPD medium, harvested, and washed with distilled water. A 2 μl aliquot of cell suspension (OD<sub>600</sub> = 1.0) was spotted onto each well of a 96-well microplate containing solid GM-BCP medium adjusted to pH 4.0. A second 96-well plate, serving as an acid trap, was prepared with each reservoir containing 10% citric acid and placed directly beneath the inoculated plate. The plates were incubated at 37 °C for 72 h. Following incubation, samples from the citric acid trap were collected for ammonia quantification. To measure ammonia levels, 20 μl of 10-fold diluted samples was mixed with 80 μl of Nessler’s reagent and incubated at room temperature for 30 minutes. Absorbance at 400 nm (OD<sub>400</sub>) was measured using a Thermo Scientific plate reader. The release of ammonia trapped in the citric acid was quantified using an ammonium chloride standard curve.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Protein extraction and immunoblotting</h3>
<p id="Par47">Total protein was extracted as previously described<sup><a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. Lysates corresponding to 1.5 OD<sub>600</sub> of cells were subjected to SDS-PAGE and analyzed by immunoblotting. Primary antibodies included anti- Myc (9E10, MBL, Cat.#M192-3) for Myc-tagged proteins and anti-HA antibody (3F10, Roche, Cat.#11867423001) for HA-tagged proteins, both used at 1:10000 and 1:1000 dilutions, respectively. Secondary antibody (goat anti-mouse, Protein tech) was used at a 1:8000 dilution. An antibody against α-tubulin (Novus Biologicals, Cat.#NB100-1639) was used as a loading control at a 1:2000 dilution.</p></section><section id="Sec25"><h3 class="pmc_sec_title">Macrophage cytotoxicity assay and <em>C. albicans</em> killing assay</h3>
<p id="Par48">The cytotoxicity of <em>C. albicans</em> toward RAW264.7 macrophages, as well as its survival within RAW264.7, J774A.1, or bone marrow-derived macrophages (BMDMs), was evaluated as previously described<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Briefly, macrophages were seeded at 5 × 10<sup>4</sup> cells per well in 96-well plates and incubated overnight at 37 °C with 5% CO<sub>2</sub>. Cells were then infected with log-phase <em>C. albicans</em> at a multiplicity of infection (MOI) of 3. Macrophage damage was assessed 5 h post-infection by measuring lactate dehydrogenase (LDH) activity in the culture supernatant using the Cytotoxicity Detection Kit (Promega, Cat.#G1780), following the manufacturer’s instructions. LDH release by infected macrophages was calculated relative to the maximum LDH release from lysed macrophages to determine the percentage of macrophage cytotoxicity. For the <em>C. albicans</em> killing assay, 1 × 10<sup>4</sup> log-phase <em>C. albicans</em> cells were resuspended in fresh cell culture medium and added to wells with or without macrophages at an MOI of 0.4, followed by six serial 1:5 dilutions. After incubation for 48 h at 37 °C with 5% CO<sub>2</sub>, microcolonies were counted under an inverted microscope. <em>C. albicans</em> survival was calculated as the ratio of colony numbers in co-cultures to those in macrophage-free controls.</p></section><section id="Sec26"><h3 class="pmc_sec_title">Quantitation of phagosomal acidification</h3>
<p id="Par49">To assess phagosomal acidification, mid-log phase <em>C. albicans</em> cells were labeled with 0.1 mg/ml pHrodo (ThermoFisher, Cat.#<a href="https://www.ncbi.nlm.nih.gov/protein/P36011" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">P36011</a>) as previously described<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>. The labeled fungal cells were then incubated with RAW264.7 or J774 A.1 macrophages at a MOI of 2. To monitor phagosomal pH, images were captured at 10-minute intervals over a period of 1.5 h using a spinning disk confocal microscope system (Olympus SpinSR10) at 40× magnification. One hundred fungal cell containing phagolysosomes were counted and evaluated for acidification. Values represent mean ± SD of three different experiments. Image analysis was performed using ImageJ Fiji (v2.3) and Imaris (v9.5.1) software.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Fungal total RNA isolation and RT-qPCR</h3>
<p id="Par50"><em>C. albicans</em> strains were grown overnight in YPD, harvested, washed with water, and diluted to a starting OD<sub>600</sub> of 0.1 in SD or YNB + 1% CAA medium (initial pH 4.5). Cultures were incubated at 37 °C with shaking for 6 h. Cells were then collected, and total RNA was extracted according to published methods<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. One microgram of total RNA, treated with DNase I was used for cDNA synthesis using a PrimeScript RT reagent Kit (TaKaRa, Cat.#RR047A). Quantitative PCR was performed using TB Green Premix Ex Taq II (TaKaRa, Cat.#RR820A) on an ABI7900HT Fast Real-Time PCR System. The expression level of each gene was quantified and normalized against the level of <em>ACT1</em>. Primers used for RT-qPCR were synthesized by Genewiz (Jinweizhi Biotechnology Co., Ltd.) and are listed in Supplementary Data <a href="#MOESM10" class="usa-link">8</a>.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Subcellular localization assay</h3>
<p id="Par51"><em>C. albicans</em> cells were grown overnight in YPD medium at 30 °C, collected by centrifugation, washed with sterile water, and resuspended in YNB + 1% CAA medium (initial pH 4.5) to an OD<sub>600</sub> of 0.2. Cells were then grown to the log phase. Fixation, cell wall digestion, and antibody hybridization were performed as previously described<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, with the modification that an anti-HA antibody (3F10, Roche, C Cat.#11867423001) was used at a 1:150 dilution, and detected with a Cy2-conjugated secondary antibody at a 1:400 dilution. Images were acquired using a 100x oil objective on an inverted fluorescence microscope (Olympus FV-1200). All images were processed using ImageJ Fiji (v2.3) software.</p></section><section id="Sec29"><h3 class="pmc_sec_title">Co-immunoprecipitation (Co-IP) analysis</h3>
<p id="Par52"><em>C. albicans</em> cells expressing Myc-tagged Dal81 or HA-tagged Stp2 were grown in YNB + 1% CAA medium (initial pH 4.5) until reaching log phase. Cells were harvested by centrifugation at 3,000 rpm for 5 min, washed thrice with ice-cold sterile water, and resuspended in 700 μl of lysis buffer (50 mM Tris-HCl, pH 7.5, 50 mM NaCl, 1 mM EDTA, 0.5% NP40, 5 mM MgCl<sub>2</sub>, 10% glycerol) supplemented with a protease inhibitor cocktail (Roche, Cat.#04693132001). Cells lysis was performed using a FastPrep-24 (MP Biomedicals, USA) in the presence of one-third volume of glass beads. Lysates were centrifuged at 14,000 rpm for 5 min at 4 °C. Protein concentration in the supernatants was determined using the Bradford assay. For immunoprecipitation, 5 mg of total protein was incubated with 50 μl of EZview<sup>TM</sup> Red anti-c-Myc affinity gel (Sigma, Cat.#E6654) or EZviewTM Red anti-HA affinity Gel (Sigma, Cat.#E6779) overnight at 4 °C with gentle rotation. Beads were then washed five times with wash buffer (50 mM Tris-HCl, pH 7.5;150 mM NaCl; 1 mM EDTA; 0.5% NP40; 5 mM MgCl<sub>2</sub>; 10% glycerol; and protease inhibitor cocktail). Bound proteins were eluted by adding 60 μl of 2x SDS-PAGE loading buffer followed by boiling for 5 min. Eluted proteins were subjected to SDS-PAGE and analyzed by western blotting.</p></section><section id="Sec30"><h3 class="pmc_sec_title">Yeast two-hybrid (Y2H) assay</h3>
<p id="Par53">The Y2H assay was performed to investigate the interaction between <em>C. albicans</em> Dal81 and Stp2. The full-length of coding sequence of Dal81 was amplified from <em>C. albicans</em> SC5314 genomic DNA and cloned into the XmaI-BamHI sites of the pGADT7 vector, generating the prey plasmid (bCB335). Similarly, the full-length coding sequence of Stp2 was cloned into the BamHI-PstI sites of the pGBKT7 vector to create the bait plasmid (bCB336). The resulting plasmids were independently transformed into the <em>Saccharomyces cerevisiae</em> strains Y187 (prey) and Y2HGold (bait), respectively. Mating was carried out, and diploids were selected and screened on SD quadruple-dropout (SD/-Leu/-Trp/-His/-Ade) agar supplemented with X-α-Gal to assess protein-protein interactions. Blue colony formation indicated a positive interaction. The interaction between pGBKT7-53 and pGADT7-T served as a positive control.</p></section><section id="Sec31"><h3 class="pmc_sec_title">Protein model prediction</h3>
<p id="Par54">The <em>C. albicans</em> Dal81 and Stp2 protein sequences were obtained from Uniprot. AlphaFold 3<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> was used to model the Dal81-Stp2 protein complex. The most accurate model was selected base on AlphaFold’s predicted Local Distance Difference Test (pLDDT) scores, which assessed the predicted structure’s reliability at the amino acid level. The finalized structures were rendered using UCSF ChimeraX<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>.</p></section><section id="Sec32"><h3 class="pmc_sec_title">RNA-Seq and data analysis</h3>
<p id="Par55">The wild-type (SN250 or SC5314), <em>dal81Δ/Δ</em> and <em>stp2Δ/Δ</em> strains were cultured overnight in YPD at 30 °C. Cells were harvested by centrifugation, washed with water, and resuspended at an OD<sub>600</sub> of 0.2 in either YNB + 1% CAA (initial pH of 4.5) medium or SD medium. Cultures were grown to the log phase, and total RNA was extracted as described above. RNA integrity was determined using an Agilent 2100 Bioanalyzer (Agilent Technologies). Triplicate RNA-seq libraries for each strain and condition were prepared and sequenced on an Illumina HiSeq 2000 genome analyzer (Novogene, Beijing, China). Raw reads were quality-checked using FastQC (<a href="https://github.com/s-andrews/FastQC" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/s-andrews/FastQC</a>) and MultiQC v1.13<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup>. Adapters and low-quality bases were trimmed using Trimmomatic<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. Clean reads were aligned to the <em>C. albicans</em> SC5314 genome (assembly 22) using STAR v2.7.6a<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. The raw count tables were generated by featureCounts v2.0.1<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. Differential expression analysis was performed in DESeq2 v1.20<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a></sup>, with significance defined as <em>P</em> value (adj. <em>p</em> -value) &lt;0.05 and |log2 fold change | &gt;1. Heatmaps were plotted using the R package ComplexHeatmap v2.20.0<sup><a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>, and Gene Ontology (GO) enrichment analysis were conducted with the clusterProfiler package<sup><a href="#CR77" class="usa-link" aria-describedby="CR77">77</a></sup>.</p></section><section id="Sec33"><h3 class="pmc_sec_title">ChIP- Seq and data analysis</h3>
<p id="Par56">Strains were grown overnight in YPD at 30 °C, harvested by centrifugation and washed with water. Cells were then resuspended at an OD<sub>600</sub> of 0.2 in 100 ml of YNB + 1% CAA medium (initial pH 4.5) and grown to log phase. For chromatin immunoprecipitation, cells were cross-linked with 1% formaldehyde for 15 min with gentle agitation, and the reaction was quenched by the addition of glycine. Cells were pelleted, washed three times with ice-cold PBS, and stored at −80 °C in 2 ml screw-cap tubes. For lysis, frozen pellets were resuspended in 700 μl of lysis buffer (50 mM HEPES/KOH pH 7.5; 140 mM NaCl; 1 mM EDTA; 1% Triton X-100; 0.1% SDS; 0.1% Na-deoxycholate; protease inhibitor cocktail [EDTA-free, Roche]). and cells were mechanically disrupted using 500 μl of 0.5 mm ZrO<sub>2</sub> beads in a FastPrep-24 shaker (MP Biomedicals) for three cycles of 60 s at 6.0 m/s, with 5-minute cooling intervals on ice. After lysis, samples were rotated at 4 °C for 30 min, and the ZrO<sub>2</sub> beads were removed by puncturing the tube bottom and centrifuging for 3 min at 1000 rpm to collect the lysates. Chromatin was then sheared via ultrasonication to yield DNA fragments of approximately 100–500 bp. The sheared lysates were clarified by centrifugation at 10,000 g for 15 min at 4 °C, and the supernatants containing the solubilized chromatin were collected. A 50 μl aliquot of each lysate was reserved as Input DNA for downstream analysis.</p>
<p id="Par57">To facilitate immunoprecipitation, 30 μl of magnetic protein A/G beads (Thermo, Cat.#88802) were washed twice with PBST and resuspended in 400 μl of the same buffer. Subsequently, either 5 μl of anti-Myc antibody (Abmart, Cat.#<a href="https://www.ncbi.nlm.nih.gov/nuccore/M20002" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">M20002</a>) or 20 μl of anti-HA antibody (Roche, Cat.#11867423001) was added to the bead suspension and incubated for 6 h at 4 °C with rotation. Following antibody binding, the buffer was replaced with 900 μl of RIPA buffer, and 500 μl of solubilized chromatin was added. The mixture was incubated overnight at 4 °C with gentle agitation. The beads were then sequentially washed with 1 ml of each of the following buffers: buffer 1 (1xTE, 1% Triton X-100, 0.1% SDS, 0.1% Na-deoxycholate), buffer 2 (1xTE, 1% Triton X-100, 0.1% SDS; 0.1% Na-deoxycholate, 300 mM NaCl), buffer 3 (1xTE, 250 mM LiCl, 0.5% NP40, 0.5% Na-deoxycholate), buffer 4 (1xTE, 0.2% Triton X-100), and finally 1xTE buffer. To obtain chromatin-bound DNA, beads and Input DNA samples were incubated in 100 μl TE containing 4 μl of 5 M NaCl and 4 μl of 10% SDS for 4 h at 65 °C. RNA was removed by treatment with RNase A (100 μg/ml) at 37 °C for 1 h, followed by protein digestion using a mixture of 0.01 M EDTA, 0.04 M Tris-HCl (pH 6.5), and 100 μg/ml proteinase K at 55 °C for 3 h. The supernatant was transferred into a new tube, and the beads were washed once with 100 μl of TE. The wash was combined with the previous supernatant, and DNA was purified using the GeneJEET PCR purification kit (Thermo Scientific) according to the manufacturer’s instructions.</p>
<p id="Par58">The eluted DNA was used to construct the sequencing libraries using NEBNext® Ultra™ II DNA Library Prep Kit (Novogene, Beijing, China). Paired-end sequencing (2 ×150 bp) was performed on an Illumina HiSeq 2500 platform (Novogene, Beijing, China). Raw reads were trimmed with Trimmomatic v0.39<sup><a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>, and quality was assessed using FastQC before and after adaptor trimming. Clean reads were aligned to the <em>C. albicans</em> SC5314 genome (assembly 22) using Bowtie2 v1.3.1<sup><a href="#CR78" class="usa-link" aria-describedby="CR78">78</a></sup> with default parameters. Resulting SAM files were converted to BAM format, and non-uniquely mapped reads were filtered out. Duplicates were marked and removed using Sambamba v0.7.1<sup><a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>. Samtools v1.6<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a></sup> was used to merge the BAM files from biological replicates and apply additional filtering. Normalized coverage files (bigWig) were generated using Deeptools 3.5.1<sup><a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup> bamCoverage (RPKM normalization, duplicates ignored). ChIP-seq signal from replicates was combined, and peak calling was performed using Macs2<sup><a href="#CR82" class="usa-link" aria-describedby="CR82">82</a></sup>.</p></section><section id="Sec34"><h3 class="pmc_sec_title">ChIP- qPCR assays</h3>
<p id="Par59">Precipitated and input DNA samples were prepared as described above and used as templates for ChIP-qPCR analysis using TB Green Premix Ex Taq II (TaKaRa, Cat.#RR820A) following the manufacturer’s instructions. Primer sequences are listed in Table <a href="#MOESM1" class="usa-link">S2</a>.</p></section><section id="Sec35"><h3 class="pmc_sec_title">In vivo animal assays</h3>
<p id="Par60">To assess the role of Dal81 and Stp2 in <em>C. albicans</em> commensalism within the gastrointestinal (GI) tract, we employed a previously described mouse colonization model<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>,<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup>. Briefly, female C57BL/6 mice (6–8 weeks old) were administered penicillin (1.5 mg/ml) and streptomycin (2 mg/ml) in their drinking water starting three days prior to fungal inoculation. Mice were orally gavaged with 1 ×10<sup>8</sup> CFUs of wild-type, <em>dal81Δ/Δ</em>, <em>stp2Δ/Δ</em>, <em>dal81Δ/Δstp2Δ/Δ</em>, or Dal81comp. strains of <em>C. albicans</em>. Fecal pellets were collected at various time points post-inoculation, homogenized in PBS, serially diluted, and plated on Sabouraud agar containing 50 μg/ml ampicillin and 15 μg/ml gentamycin. Plates were incubated at 30 °C, and fungal colonization was quantified by enumerating colony-forming units (CFUs). Statistical significance was determined using one-way or two-way ANOVA followed by Tukey’s test. When required, female ICR mice (6–8 weeks old, average weight 23–25 g) were immunosuppressed with intraperitoneal (I.P.) injections of cyclophosphamide at a dose of 150 mg/kg, starting four days prior to fungal infection. Immunosuppression was maintained by administering two additional doses of cyclophosphamide (150 mg/kg, I.P.) on days 1 and 5 post-infection. To assess intestinal colonization by <em>C. albicans</em>, immunosuppressed mice were orally gavaged with a 1:1 mixture of the indicated strains (1 × 10<sup>8</sup> CFUs total). Fecal pellets were collected at defined time points post-infection, and recovery and quantification of <em>C. albicans</em> cells were performed as previously described<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Statistical significance was determined using two-way ANOVA followed by Tukey’s test. The virulence of the WT, <em>dal81Δ/Δ</em>, <em>stp2Δ/Δ dal81Δ/Δstp2Δ/Δ</em>, Dal81comp., <em>DAL81 /dal81Δ</em> or <em>DAL81m21 /dal81Δ</em> was assayed using an established murine systemic infection model as previously described<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>,<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. Briefly, groups of female C57BL/6 mice (6-8 weeks old) were intravenously inoculated with 2 ×10<sup>5</sup> CFUs of the respective <em>C. albicans</em> strains. Mice were monitored daily for clinical indicators including weight loss, disease symptoms, and survival. Survival data were analyzed using the Kaplan-Meier method, and statistical significance was determined by the log-rank (Mantel-Cox) test using GraphPad Prism software.</p></section><section id="Sec36"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par61">All statistical analyses were performed using GraphPad Prism versions 8.2.1 and 10.3.1. Data are presented as mea<em>n</em> ± standard deviation (SD). Specific statistical tests and sample sizes are detailed in the respective figure legends. Statistical significance was defined as follows: *<em>P</em> &lt; <em>0.05</em>; **<em>P</em> &lt; <em>0.01</em>; ***<em>P</em> &lt; <em>0.001</em>; ****<em>P</em> &lt; <em>0.0001</em>; ns, non-significant.</p></section><section id="Sec37"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par62">Further information on research design is available in the <a href="#MOESM11" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec38"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary information</a><sup> (3.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62953_MOESM2_ESM.pdf</a><sup> (129.5KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (330.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (42.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (40.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (65.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (15.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 6</a><sup> (15.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 7</a><sup> (11.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM10"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 8</a><sup> (13.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM11"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM11_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (96.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM12"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM12_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.2MB, pdf) </sup>
</div></div></section></section><section id="Sec39"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM13"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM13_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (3.9MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We are grateful to Suzanne Noble at UCSF for the gift of the <em>C. alblicans</em> homozygous gene deletion library. We also would like to thank members of the Changbin Chen lab for helpful discussions and advice. Special thanks to Miss Grace Morrissey for helping us proofread the manuscript. C.C. is supported by grants from the MOST Key R&amp;D Program (2020YFA0907200; 2022YFC2303200); National Natural Science Foundation of China (32170195; 32200161); The Shanghai Science and Technology Innovation Action Plan 2023 “Basic Research Project” (23JC1404200); The Shanghai “Belt and Road” Joint Laboratory Project (22490750200); The Foundation of State Key Laboratory of Pathogen and Biosecurity (SKLPBS2236). X.H. is supported by grants from the MOST Key R&amp;D Program (2022YFC2303504); National Natural Science Foundation of China (32070146); and Natural Science Foundation of Shanghai (20ZR1463800).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>X.H. and C.C. planned and designed the study; X.H., H.L. and C.C. planned and designed the revision study; X.H. conducted all of the experiments with the help from L.W., Y.Z., L.Z., S.L., K.L., W.X., Z.J., Y.P.Z., J.D., H.H., X.W., Y.W., T.J. and W.X.; G.C. performed all of the bioinformatics analysis. X.C. participated in the RNA-seq and ChIP-seq analysis; C.C. and X.H. wrote the manuscript; W.S. edited the English language. M.M. helped refine the working model. X.H., Z.H., Q.G., H.L. and C.C. discussed the experiments and results.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par63"><em>Nature Communications</em> thanks Christophe d’Enfert, and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data necessary to evaluate the conclusions of this study are provided within the main text and/or the Supplementary Materials. The Dal81 ChIP-seq, Stp2 ChIP -seq, and RNA-seq datasets have been deposited in the National Center for Biotechnology Information Sequence Read Archive (NCBI SRA) under accession number <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1190138/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJNA1190138</a>. Source Data are available with this paper. <a href="#Sec39" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par64">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors contributed equally: Xinhua Huang, Guangsheng Chen, Lei Wu, Yun Zou.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Xinhua Huang, Email: xhhuang@siii.cas.cn.</p>
<p>Hao Li, Email: lihao6656@163.com.</p>
<p>Changbin Chen, Email: cbchen@ips.ac.cn.</p></section><section id="sec40"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62953-x.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Guan, N. &amp; Liu, L. Microbial response to acid stress: mechanisms and applications. <em>Appl. Microbiol. Biotechnol.</em><strong>104</strong>, 51–65 (2020).
</cite> [<a href="https://doi.org/10.1007/s00253-019-10226-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6942593/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31773206/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guan,%20N.%20&amp;%20Liu,%20L.%20Microbial%20response%20to%20acid%20stress:%20mechanisms%20and%20applications.%20Appl.%20Microbiol.%20Biotechnol.104,%2051%E2%80%9365%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Davis, D. A. How human pathogenic fungi sense and adapt to pH: the link to virulence. <em>Curr. Opin. Microbiol.</em><strong>12</strong>, 365–370 (2009).
</cite> [<a href="https://doi.org/10.1016/j.mib.2009.05.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19632143/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Davis,%20D.%20A.%20How%20human%20pathogenic%20fungi%20sense%20and%20adapt%20to%20pH:%20the%20link%20to%20virulence.%20Curr.%20Opin.%20Microbiol.12,%20365%E2%80%93370%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Maeda, T. The signaling mechanism of ambient pH sensing and adaptation in yeast and fungi. <em> FEBS J.</em><strong>279</strong>, 1407–1413 (2012).
</cite> [<a href="https://doi.org/10.1111/j.1742-4658.2012.08548.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22360598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maeda,%20T.%20The%20signaling%20mechanism%20of%20ambient%20pH%20sensing%20and%20adaptation%20in%20yeast%20and%20fungi.%20FEBS%20J.279,%201407%E2%80%931413%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Davis, D. Adaptation to environmental pH in Candida albicans and its relation to pathogenesis. <em>Curr. Genet.</em><strong>44</strong>, 1–7 (2003).
</cite> [<a href="https://doi.org/10.1007/s00294-003-0415-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12819929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Davis,%20D.%20Adaptation%20to%20environmental%20pH%20in%20Candida%20albicans%20and%20its%20relation%20to%20pathogenesis.%20Curr.%20Genet.44,%201%E2%80%937%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Denison, S. H. pH regulation of gene expression in fungi. <em>Fungal Genet. Biol.</em><strong>29</strong>, 61–71 (2000).
</cite> [<a href="https://doi.org/10.1006/fgbi.2000.1188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10919375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Denison,%20S.%20H.%20pH%20regulation%20of%20gene%20expression%20in%20fungi.%20Fungal%20Genet.%20Biol.29,%2061%E2%80%9371%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Manteau, S., Abouna, S., Lambert, B. &amp; Legendre, L. Differential regulation by ambient pH of putative virulence factor secretion by the phytopathogenic fungus botrytis cinerea. <em>FEMS Microbiol. Ecol.</em><strong>43</strong>, 359–366 (2003).
</cite> [<a href="https://doi.org/10.1111/j.1574-6941.2003.tb01076.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19719667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manteau,%20S.,%20Abouna,%20S.,%20Lambert,%20B.%20&amp;%20Legendre,%20L.%20Differential%20regulation%20by%20ambient%20pH%20of%20putative%20virulence%20factor%20secretion%20by%20the%20phytopathogenic%20fungus%20botrytis%20cinerea.%20FEMS%20Microbiol.%20Ecol.43,%20359%E2%80%93366%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Ruijter, G. J. &amp; Visser, J. Characterization of aspergillus niger phosphoglucose isomerase. Use for quantitative determination of erythrose 4-phosphate. <em>Biochimie</em><strong>81</strong>, 267–272 (1999).
</cite> [<a href="https://doi.org/10.1016/s0300-9084(99)80061-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10385009/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ruijter,%20G.%20J.%20&amp;%20Visser,%20J.%20Characterization%20of%20aspergillus%20niger%20phosphoglucose%20isomerase.%20Use%20for%20quantitative%20determination%20of%20erythrose%204-phosphate.%20Biochimie81,%20267%E2%80%93272%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Horwitz, M. A. &amp; Maxfield, F. R. Legionella pneumophila inhibits acidification of its phagosome in human monocytes. <em> J. cell Biol.</em><strong>99</strong>, 1936–1943 (1984).
</cite> [<a href="https://doi.org/10.1083/jcb.99.6.1936" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2113576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6501409/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Horwitz,%20M.%20A.%20&amp;%20Maxfield,%20F.%20R.%20Legionella%20pneumophila%20inhibits%20acidification%20of%20its%20phagosome%20in%20human%20monocytes.%20J.%20cell%20Biol.99,%201936%E2%80%931943%20(1984)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Ansari, S. &amp; Yamaoka, Y. Survival of helicobacter pylori in gastric acidic territory. <em>Helicobacter</em><strong>22</strong>, e12386 (2017).</cite> [<a href="https://doi.org/10.1111/hel.12386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5851894/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28402047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ansari,%20S.%20&amp;%20Yamaoka,%20Y.%20Survival%20of%20helicobacter%20pylori%20in%20gastric%20acidic%20territory.%20Helicobacter22,%20e12386%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Koh, A. Y., Köhler, J. R., Coggshall, K. T., Van Rooijen, N. &amp; Pier, G. B. Mucosal damage and neutropenia are required for candida albicans dissemination. <em>PLoS Pathog.</em><strong>4</strong>, e35 (2008).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.0040035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2242836/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18282097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koh,%20A.%20Y.,%20K%C3%B6hler,%20J.%20R.,%20Coggshall,%20K.%20T.,%20Van%20Rooijen,%20N.%20&amp;%20Pier,%20G.%20B.%20Mucosal%20damage%20and%20neutropenia%20are%20required%20for%20candida%20albicans%20dissemination.%20PLoS%20Pathog.4,%20e35%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Spellberg, B. et al. Risk factors for mortality in patients with mucormycosis. <em>Med. Mycol.</em><strong>50</strong>, 611–618 (2012).
</cite> [<a href="https://doi.org/10.3109/13693786.2012.669502" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3951871/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22435877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Spellberg,%20B.%20et%20al.%20Risk%20factors%20for%20mortality%20in%20patients%20with%20mucormycosis.%20Med.%20Mycol.50,%20611%E2%80%93618%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Brown, G. D. et al. Hidden killers: human fungal infections. <em>Sci. Transl. Med.</em><strong>4</strong>, 165rv113–165rv113 (2012).</cite> [<a href="https://doi.org/10.1126/scitranslmed.3004404" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23253612/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brown,%20G.%20D.%20et%20al.%20Hidden%20killers:%20human%20fungal%20infections.%20Sci.%20Transl.%20Med.4,%20165rv113%E2%80%93165rv113%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Organization WH. <em>WHO fungal priority pathogens list to guide research, development and public health action</em>. World Health Organization (2022).</cite>
</li>
<li id="CR14">
<span class="label">14.</span><cite>Danhof, H. A. &amp; Lorenz, M. C. The candida albicans ATO gene family promotes neutralization of the macrophage phagolysosome. <em>Infect. Immun.</em><strong>83</strong>, 4416–4426 (2015).
</cite> [<a href="https://doi.org/10.1128/IAI.00984-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4598414/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26351284/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Danhof,%20H.%20A.%20&amp;%20Lorenz,%20M.%20C.%20The%20candida%20albicans%20ATO%20gene%20family%20promotes%20neutralization%20of%20the%20macrophage%20phagolysosome.%20Infect.%20Immun.83,%204416%E2%80%934426%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Vylkova, S. &amp; Lorenz, M. C. Modulation of phagosomal ph by candida albicans promotes hyphal morphogenesis and requires Stp2p, a regulator of amino acid transport. <em>PLoS Pathog.</em><strong>10</strong>, e1003995 (2014).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1003995" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3953444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24626429/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vylkova,%20S.%20&amp;%20Lorenz,%20M.%20C.%20Modulation%20of%20phagosomal%20ph%20by%20candida%20albicans%20promotes%20hyphal%20morphogenesis%20and%20requires%20Stp2p,%20a%20regulator%20of%20amino%20acid%20transport.%20PLoS%20Pathog.10,%20e1003995%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Vylkova, S. et al. The fungal pathogen candida albicans autoinduces hyphal morphogenesis by raising extracellular pH. <em>MBio</em><strong>2</strong>, 00055–00011 (2011).</cite> [<a href="https://doi.org/10.1128/mBio.00055-11" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3101780/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21586647/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vylkova,%20S.%20et%20al.%20The%20fungal%20pathogen%20candida%20albicans%20autoinduces%20hyphal%20morphogenesis%20by%20raising%20extracellular%20pH.%20MBio2,%2000055%E2%80%9300011%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Schindler, J. &amp; Sussman, M. Ammonia determines the choice of morphogenetic pathways in Dictyostelium discoideum. <em>J. Mol. Biol.</em><strong>116</strong>, 161–169 (1977).
</cite> [<a href="https://doi.org/10.1016/0022-2836(77)90124-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/563469/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schindler,%20J.%20&amp;%20Sussman,%20M.%20Ammonia%20determines%20the%20choice%20of%20morphogenetic%20pathways%20in%20Dictyostelium%20discoideum.%20J.%20Mol.%20Biol.116,%20161%E2%80%93169%20(1977)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Naseem, S., Araya, E. &amp; Konopka, J. B. Hyphal growth in candida albicans does not require induction of hyphal-specific gene expression. <em>Mol. Biol. cell</em><strong>26</strong>, 1174–1187 (2015).
</cite> [<a href="https://doi.org/10.1091/mbc.E14-08-1312" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4357515/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25609092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Naseem,%20S.,%20Araya,%20E.%20&amp;%20Konopka,%20J.%20B.%20Hyphal%20growth%20in%20candida%20albicans%20does%20not%20require%20induction%20of%20hyphal-specific%20gene%20expression.%20Mol.%20Biol.%20cell26,%201174%E2%80%931187%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Danhof, H. A. et al. Robust extracellular ph modulation by candida albicans during growth in carboxylic acids. <em>MBio</em><strong>7</strong>, 01646–01616 (2016).</cite> [<a href="https://doi.org/10.1128/mBio.01646-16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5111404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27935835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Danhof,%20H.%20A.%20et%20al.%20Robust%20extracellular%20ph%20modulation%20by%20candida%20albicans%20during%20growth%20in%20carboxylic%20acids.%20MBio7,%2001646%E2%80%9301616%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Fernandes, T. R., Segorbe, D., Prusky, D. &amp; Di Pietro, A. How alkalinization drives fungal pathogenicity. <em>PLoS Pathog.</em><strong>13</strong>, e1006621 (2017).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1006621" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5679519/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29121119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fernandes,%20T.%20R.,%20Segorbe,%20D.,%20Prusky,%20D.%20&amp;%20Di%20Pietro,%20A.%20How%20alkalinization%20drives%20fungal%20pathogenicity.%20PLoS%20Pathog.13,%20e1006621%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Miramón, P. &amp; Lorenz, M. C. The SPS amino acid sensor mediates nutrient acquisition and immune evasion in Candida albicans. <em>Cell. Microbiol.</em><strong>18</strong>, 1611–1624 (2016).
</cite> [<a href="https://doi.org/10.1111/cmi.12600" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5501722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27060451/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miram%C3%B3n,%20P.%20&amp;%20Lorenz,%20M.%20C.%20The%20SPS%20amino%20acid%20sensor%20mediates%20nutrient%20acquisition%20and%20immune%20evasion%20in%20Candida%20albicans.%20Cell.%20Microbiol.18,%201611%E2%80%931624%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Martínez, P. &amp; Ljungdahl, P. O. An ER packaging chaperone determines the amino acid uptake capacity and virulence of candida albicans. <em>Mol. Microbiol.</em><strong>51</strong>, 371–384 (2004).
</cite> [<a href="https://doi.org/10.1046/j.1365-2958.2003.03845.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14756779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mart%C3%ADnez,%20P.%20&amp;%20Ljungdahl,%20P.%20O.%20An%20ER%20packaging%20chaperone%20determines%20the%20amino%20acid%20uptake%20capacity%20and%20virulence%20of%20candida%20albicans.%20Mol.%20Microbiol.51,%20371%E2%80%93384%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Martínez, P. &amp; Ljungdahl, P. O. Divergence of Stp1 and Stp2 transcription factors in Candida albicans places virulence factors required for proper nutrient acquisition under amino acid control. <em>Mol. Cell. Biol.</em><strong>25</strong>, 9435–9446 (2005).</cite> [<a href="https://doi.org/10.1128/MCB.25.21.9435-9446.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1265835/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16227594/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Noble, S. M., French, S., Kohn, L. A., Chen, V. &amp; Johnson, A. D. Systematic screens of a Candida albicans homozygous deletion library decouple morphogenetic switching and pathogenicity. <em>Nat. Genet.</em><strong>42</strong>, 590–598 (2010).
</cite> [<a href="https://doi.org/10.1038/ng.605" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2893244/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20543849/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Noble,%20S.%20M.,%20French,%20S.,%20Kohn,%20L.%20A.,%20Chen,%20V.%20&amp;%20Johnson,%20A.%20D.%20Systematic%20screens%20of%20a%20Candida%20albicans%20homozygous%20deletion%20library%20decouple%20morphogenetic%20switching%20and%20pathogenicity.%20Nat.%20Genet.42,%20590%E2%80%93598%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Butler, D. K. et al. The GRR1 gene of candida albicans is involved in the negative control of pseudohyphal morphogenesis. <em>Fungal Genet. Biol.</em><strong>43</strong>, 573–582 (2006).
</cite> [<a href="https://doi.org/10.1016/j.fgb.2006.03.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16730201/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Butler,%20D.%20K.%20et%20al.%20The%20GRR1%20gene%20of%20candida%20albicans%20is%20involved%20in%20the%20negative%20control%20of%20pseudohyphal%20morphogenesis.%20Fungal%20Genet.%20Biol.43,%20573%E2%80%93582%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Reuß, O., Vik, Å, Kolter, R. &amp; Morschhäuser, J. The SAT1 flipper, an optimized tool for gene disruption in Candida albicans. <em>Gene</em><strong>341</strong>, 119–127 (2004).
</cite> [<a href="https://doi.org/10.1016/j.gene.2004.06.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15474295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Reu%C3%9F,%20O.,%20Vik,%20%C3%85,%20Kolter,%20R.%20&amp;%20Morschh%C3%A4user,%20J.%20The%20SAT1%20flipper,%20an%20optimized%20tool%20for%20gene%20disruption%20in%20Candida%20albicans.%20Gene341,%20119%E2%80%93127%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Silao, F. G. S. et al. Glutamate dehydrogenase (Gdh2)-dependent alkalization is dispensable for escape from macrophages and virulence of Candida albicans. <em>PLoS Pathog.</em><strong>16</strong>, e1008328 (2020).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1008328" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7521896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32936835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Silao,%20F.%20G.%20S.%20et%20al.%20Glutamate%20dehydrogenase%20(Gdh2)-dependent%20alkalization%20is%20dispensable%20for%20escape%20from%20macrophages%20and%20virulence%20of%20Candida%20albicans.%20PLoS%20Pathog.16,%20e1008328%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Schmidt, F. et al. Flotillin-dependent membrane microdomains are required for functional phagolysosomes against fungal infections. <em>Cell Rep.</em><strong>32</strong>, 108017 (2020).</cite> [<a href="https://doi.org/10.1016/j.celrep.2020.108017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10054021/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32814035/" class="usa-link">PubMed</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Vylkova, S. Environmental pH modulation by pathogenic fungi as a strategy to conquer the host. <em>PLoS Pathog.</em><strong>13</strong>, e1006149 (2017).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1006149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5322887/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28231317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vylkova,%20S.%20Environmental%20pH%20modulation%20by%20pathogenic%20fungi%20as%20a%20strategy%20to%20conquer%20the%20host.%20PLoS%20Pathog.13,%20e1006149%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Meng, E. C. et al. UCSF ChimeraX: Tools for structure building and analysis. <em>Protein Sci.</em><strong>32</strong>, e4792 (2023).
</cite> [<a href="https://doi.org/10.1002/pro.4792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10588335/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37774136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meng,%20E.%20C.%20et%20al.%20UCSF%20ChimeraX:%20Tools%20for%20structure%20building%20and%20analysis.%20Protein%20Sci.32,%20e4792%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Wang, S. et al. Target analysis by integration of transcriptome and ChIP-seq data with BETA. <em>Nat. Protoc.</em><strong>8</strong>, 2502–2515 (2013).
</cite> [<a href="https://doi.org/10.1038/nprot.2013.150" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4135175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24263090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20S.%20et%20al.%20Target%20analysis%20by%20integration%20of%20transcriptome%20and%20ChIP-seq%20data%20with%20BETA.%20Nat.%20Protoc.8,%202502%E2%80%932515%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Kasper, L. et al. Identification of candida glabrata genes involved in ph modulation and modification of the phagosomal environment in macrophages. <em>PLoS One</em><strong>9</strong>, e96015 (2014).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0096015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4006850/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24789333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kasper,%20L.%20et%20al.%20Identification%20of%20candida%20glabrata%20genes%20involved%20in%20ph%20modulation%20and%20modification%20of%20the%20phagosomal%20environment%20in%20macrophages.%20PLoS%20One9,%20e96015%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Noble, S. M., Gianetti, B. A. &amp; Witchley, J. N. Candida albicans cell-type switching and functional plasticity in the mammalian host. <em>Nat. Rev. Microbiol.</em><strong>15</strong>, 96–108 (2017).
</cite> [<a href="https://doi.org/10.1038/nrmicro.2016.157" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5957277/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27867199/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Noble,%20S.%20M.,%20Gianetti,%20B.%20A.%20&amp;%20Witchley,%20J.%20N.%20Candida%20albicans%20cell-type%20switching%20and%20functional%20plasticity%20in%20the%20mammalian%20host.%20Nat.%20Rev.%20Microbiol.15,%2096%E2%80%93108%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Huang, X. et al. Mitochondrial complex I bridges a connection between regulation of carbon flexibility and gastrointestinal commensalism in the human fungal pathogen Candida albicans. <em>PLoS Pathog.</em><strong>13</strong>, e1006414 (2017).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1006414" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5469625/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28570675/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20X.%20et%20al.%20Mitochondrial%20complex%20I%20bridges%20a%20connection%20between%20regulation%20of%20carbon%20flexibility%20and%20gastrointestinal%20commensalism%20in%20the%20human%20fungal%20pathogen%20Candida%20albicans.%20PLoS%20Pathog.13,%20e1006414%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Wang, Y. et al. Fungal commensalism modulated by a dual-action phosphate transceptor. <em>Cell Rep.</em><strong>38</strong>, 110293 (2022).</cite> [<a href="https://doi.org/10.1016/j.celrep.2021.110293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35081357/" class="usa-link">PubMed</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Alves, R. et al. Adapting to survive: How Candida overcomes host-imposed constraints during human colonization. <em>PLoS Pathog.</em><strong>16</strong>, e1008478 (2020).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1008478" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7241708/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32437438/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alves,%20R.%20et%20al.%20Adapting%20to%20survive:%20How%20Candida%20overcomes%20host-imposed%20constraints%20during%20human%20colonization.%20PLoS%20Pathog.16,%20e1008478%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Todd, R. B. &amp; Andrianopoulos, A. Evolution of a fungal regulatory gene family: the Zn (II) 2Cys6 binuclear cluster DNA binding motif. <em>Fungal Genet. Biol.</em><strong>21</strong>, 388–405 (1997).
</cite> [<a href="https://doi.org/10.1006/fgbi.1997.0993" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9290251/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Todd,%20R.%20B.%20&amp;%20Andrianopoulos,%20A.%20Evolution%20of%20a%20fungal%20regulatory%20gene%20family:%20the%20Zn%20(II)%202Cys6%20binuclear%20cluster%20DNA%20binding%20motif.%20Fungal%20Genet.%20Biol.21,%20388%E2%80%93405%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Vissers, S., André, B., Muyldermans, F. &amp; Grenson, M. Induction of the 4-aminobutyrate and urea-catabolic pathways in saccharomyces cerevisiae: Specific and common transcriptional regulators. <em>Eur. J. Biochem.</em><strong>187</strong>, 611–616 (1990).
</cite> [<a href="https://doi.org/10.1111/j.1432-1033.1990.tb15344.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2406136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vissers,%20S.,%20Andr%C3%A9,%20B.,%20Muyldermans,%20F.%20&amp;%20Grenson,%20M.%20Induction%20of%20the%204-aminobutyrate%20and%20urea-catabolic%20pathways%20in%20saccharomyces%20cerevisiae:%20Specific%20and%20common%20transcriptional%20regulators.%20Eur.%20J.%20Biochem.187,%20611%E2%80%93616%20(1990)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Coornaert, D., Vissers, S. &amp; André, B. The pleiotropic UGA35 (DURL) regulatory gene of Saccharomyces cerevisiae: cloning, sequence and identity with the DAL81 gene. <em>Gene</em><strong>97</strong>, 163–171 (1991).
</cite> [<a href="https://doi.org/10.1016/0378-1119(91)90048-g" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1999281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Coornaert,%20D.,%20Vissers,%20S.%20&amp;%20Andr%C3%A9,%20B.%20The%20pleiotropic%20UGA35%20(DURL)%20regulatory%20gene%20of%20Saccharomyces%20cerevisiae:%20cloning,%20sequence%20and%20identity%20with%20the%20DAL81%20gene.%20Gene97,%20163%E2%80%93171%20(1991)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Palavecino, M. D., Correa-García, S. R. &amp; Bermúdez-Moretti, M. Genes of different catabolic pathways are coordinately regulated by Dal81 in Saccharomyces cerevisiae. <em>J. Amino Acids</em><strong>2015</strong>, 484702 (2015).
</cite> [<a href="https://doi.org/10.1155/2015/484702" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4589623/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26457198/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palavecino,%20M.%20D.,%20Correa-Garc%C3%ADa,%20S.%20R.%20&amp;%20Berm%C3%BAdez-Moretti,%20M.%20Genes%20of%20different%20catabolic%20pathways%20are%20coordinately%20regulated%20by%20Dal81%20in%20Saccharomyces%20cerevisiae.%20J.%20Amino%20Acids2015,%20484702%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Bricmont, P. A., Daugherty, J. R. &amp; Cooper, T. G. The DAL81 gene product is required for induced expression of two differently regulated nitrogen catabolic genes in saccharomyces cerevisiae. <em>Mol. Cell. Biol.</em><strong>11</strong>, 1161–1166 (1991).
</cite> [<a href="https://doi.org/10.1128/mcb.11.2.1161" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC359801/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1990272/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bricmont,%20P.%20A.,%20Daugherty,%20J.%20R.%20&amp;%20Cooper,%20T.%20G.%20The%20DAL81%20gene%20product%20is%20required%20for%20induced%20expression%20of%20two%20differently%20regulated%20nitrogen%20catabolic%20genes%20in%20saccharomyces%20cerevisiae.%20Mol.%20Cell.%20Biol.11,%201161%E2%80%931166%20(1991)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Boban, M. &amp; Ljungdahl, P. O. Dal81 enhances Stp1-and Stp2-dependent transcription necessitating negative modulation by inner nuclear membrane protein Asi1 in Saccharomyces cerevisiae. <em>Genetics</em><strong>176</strong>, 2087–2097 (2007).
</cite> [<a href="https://doi.org/10.1534/genetics.107.075077" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1950616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17603098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Boban,%20M.%20&amp;%20Ljungdahl,%20P.%20O.%20Dal81%20enhances%20Stp1-and%20Stp2-dependent%20transcription%20necessitating%20negative%20modulation%20by%20inner%20nuclear%20membrane%20protein%20Asi1%20in%20Saccharomyces%20cerevisiae.%20Genetics176,%202087%E2%80%932097%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Vissers, S., André, B., Muyldermans, F. &amp; Grenson, M. Positive and negative regulatory elements control the expression of the UGA4 gene coding for the inducible 4-aminobutyric-acid-specific permease in Saccharomyces cerevisiae. <em>Eur. J. Biochem.</em><strong>181</strong>, 357–361 (1989).
</cite> [<a href="https://doi.org/10.1111/j.1432-1033.1989.tb14732.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2653828/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vissers,%20S.,%20Andr%C3%A9,%20B.,%20Muyldermans,%20F.%20&amp;%20Grenson,%20M.%20Positive%20and%20negative%20regulatory%20elements%20control%20the%20expression%20of%20the%20UGA4%20gene%20coding%20for%20the%20inducible%204-aminobutyric-acid-specific%20permease%20in%20Saccharomyces%20cerevisiae.%20Eur.%20J.%20Biochem.181,%20357%E2%80%93361%20(1989)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Hennaut, C. L-ornithine transaminase synthesis in saccharomyces cerevisiae: induction by allophanate, intermediate and inducer of the urea degradative pathway adds to arginine induction. <em>Curr. Genet.</em><strong>4</strong>, 69–72 (1981).
</cite> [<a href="https://doi.org/10.1007/BF00376788" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24185870/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hennaut,%20C.%20L-ornithine%20transaminase%20synthesis%20in%20saccharomyces%20cerevisiae:%20induction%20by%20allophanate,%20intermediate%20and%20inducer%20of%20the%20urea%20degradative%20pathway%20adds%20to%20arginine%20induction.%20Curr.%20Genet.4,%2069%E2%80%9372%20(1981)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Turoscy, V. &amp; Cooper, T. G. Pleiotropic control of five eucaryotic genes by multiple regulatory elements. <em>J. Bacteriol.</em><strong>151</strong>, 1237–1246 (1982).
</cite> [<a href="https://doi.org/10.1128/jb.151.3.1237-1246.1982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC220401/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7050082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Turoscy,%20V.%20&amp;%20Cooper,%20T.%20G.%20Pleiotropic%20control%20of%20five%20eucaryotic%20genes%20by%20multiple%20regulatory%20elements.%20J.%20Bacteriol.151,%201237%E2%80%931246%20(1982)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Jacobs, E., Dubois, E. &amp; Wiame, J.-M. Regulation of ureaamidolyase synthesis in Saccharomyces cerevisiae, RNA analysis, and cloning of the positive regulatory gene DURM. <em>Curr. Genet.</em><strong>9</strong>, 333–339 (1985).
</cite> [<a href="https://doi.org/10.1007/BF00421602" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/3916727/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jacobs,%20E.,%20Dubois,%20E.%20&amp;%20Wiame,%20J.-M.%20Regulation%20of%20ureaamidolyase%20synthesis%20in%20Saccharomyces%20cerevisiae,%20RNA%20analysis,%20and%20cloning%20of%20the%20positive%20regulatory%20gene%20DURM.%20Curr.%20Genet.9,%20333%E2%80%93339%20(1985)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Abdel-Sater, F., Iraqui, I., Urrestarazu, A. &amp; André, B. The external amino acid signaling pathway promotes activation of Stp1 and Uga35/Dal81 transcription factors for induction of the AGP1 gene in Saccharomyces cerevisiae. <em>Genetics</em><strong>166</strong>, 1727–1739 (2004).
</cite> [<a href="https://doi.org/10.1534/genetics.166.4.1727" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1470804/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15126393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abdel-Sater,%20F.,%20Iraqui,%20I.,%20Urrestarazu,%20A.%20&amp;%20Andr%C3%A9,%20B.%20The%20external%20amino%20acid%20signaling%20pathway%20promotes%20activation%20of%20Stp1%20and%20Uga35/Dal81%20transcription%20factors%20for%20induction%20of%20the%20AGP1%20gene%20in%20Saccharomyces%20cerevisiae.%20Genetics166,%201727%E2%80%931739%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Iraqui, I. et al. Amino acid signaling in Saccharomyces cerevisiae: a permease-like sensor of external amino acids and F-Box protein Grr1p are required for transcriptional induction of the AGP1 gene, which encodes a broad-specificity amino acid permease. <em>Mol. Cell. Biol.</em><strong>19</strong>, 989–1001 (1999).
</cite> [<a href="https://doi.org/10.1128/mcb.19.2.989" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC116030/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9891035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Iraqui,%20I.%20et%20al.%20Amino%20acid%20signaling%20in%20Saccharomyces%20cerevisiae:%20a%20permease-like%20sensor%20of%20external%20amino%20acids%20and%20F-Box%20protein%20Grr1p%20are%20required%20for%20transcriptional%20induction%20of%20the%20AGP1%20gene,%20which%20encodes%20a%20broad-specificity%20amino%20acid%20permease.%20Mol.%20Cell.%20Biol.19,%20989%E2%80%931001%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Zhang, W., Du, G., Zhou, J. &amp; Chen, J. Regulation of sensing, transportation, and catabolism of nitrogen sources in Saccharomyces cerevisiae. <em>Microbiol. Mol. Biol. Rev.</em><strong>82</strong>, 00040–00017 (2018).</cite> [<a href="https://doi.org/10.1128/MMBR.00040-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5813884/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29436478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20W.,%20Du,%20G.,%20Zhou,%20J.%20&amp;%20Chen,%20J.%20Regulation%20of%20sensing,%20transportation,%20and%20catabolism%20of%20nitrogen%20sources%20in%20Saccharomyces%20cerevisiae.%20Microbiol.%20Mol.%20Biol.%20Rev.82,%2000040%E2%80%9300017%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Sylvain, M.-A., Liang, X. B., Hellauer, K. &amp; Turcotte, B. Yeast zinc cluster proteins Dal81 and Uga3 cooperate by targeting common coactivators for transcriptional activation of γ-aminobutyrate responsive genes. <em>Genetics</em><strong>188</strong>, 523–534 (2011).
</cite> [<a href="https://doi.org/10.1534/genetics.110.126003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3176551/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21515579/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sylvain,%20M.-A.,%20Liang,%20X.%20B.,%20Hellauer,%20K.%20&amp;%20Turcotte,%20B.%20Yeast%20zinc%20cluster%20proteins%20Dal81%20and%20Uga3%20cooperate%20by%20targeting%20common%20coactivators%20for%20transcriptional%20activation%20of%20%CE%B3-aminobutyrate%20responsive%20genes.%20Genetics188,%20523%E2%80%93534%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Wielemans, K., Jean, C., Vissers, S. &amp; André, B. Amino acid signaling in yeast: post-genome duplication divergence of the Stp1 and Stp2 transcription factors. <em>J. Biol. Chem.</em><strong>285</strong>, 855–865 (2010).
</cite> [<a href="https://doi.org/10.1074/jbc.M109.015263" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2801287/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19906648/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wielemans,%20K.,%20Jean,%20C.,%20Vissers,%20S.%20&amp;%20Andr%C3%A9,%20B.%20Amino%20acid%20signaling%20in%20yeast:%20post-genome%20duplication%20divergence%20of%20the%20Stp1%20and%20Stp2%20transcription%20factors.%20J.%20Biol.%20Chem.285,%20855%E2%80%93865%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Turner, S. A., Ma, Q., Ola, M., Martinez de San Vicente, K. &amp; Butler, G. Dal81 regulates expression of arginine metabolism genes in Candida parapsilosis. <em>Msphere</em><strong>3</strong>, 00028–00018 (2018).</cite> [<a href="https://doi.org/10.1128/mSphere.00028-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5853489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29564399/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Turner,%20S.%20A.,%20Ma,%20Q.,%20Ola,%20M.,%20Martinez%20de%20San%20Vicente,%20K.%20&amp;%20Butler,%20G.%20Dal81%20regulates%20expression%20of%20arginine%20metabolism%20genes%20in%20Candida%20parapsilosis.%20Msphere3,%2000028%E2%80%9300018%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Ramachandra, S. et al. Regulatory networks controlling nitrogen sensing and uptake in Candida albicans. <em>PLoS One</em><strong>9</strong>, e92734 (2014).
</cite> [<a href="https://doi.org/10.1371/journal.pone.0092734" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3961412/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24651113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ramachandra,%20S.%20et%20al.%20Regulatory%20networks%20controlling%20nitrogen%20sensing%20and%20uptake%20in%20Candida%20albicans.%20PLoS%20One9,%20e92734%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Miramón, P., Pountain, A. W., van Hoof, A. &amp; Lorenz, M. C. The paralogous transcription factors Stp1 and Stp2 of Candida albicans have distinct functions in nutrient acquisition and host interaction. <em>Infect. Immun.</em><strong>88</strong>, e00763–19 (2020).
</cite> [<a href="https://doi.org/10.1128/IAI.00763-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7171245/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32094252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miram%C3%B3n,%20P.,%20Pountain,%20A.%20W.,%20van%20Hoof,%20A.%20&amp;%20Lorenz,%20M.%20C.%20The%20paralogous%20transcription%20factors%20Stp1%20and%20Stp2%20of%20Candida%20albicans%20have%20distinct%20functions%20in%20nutrient%20acquisition%20and%20host%20interaction.%20Infect.%20Immun.88,%20e00763%E2%80%9319%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Banerjee, M. et al. UME6, a novel filament-specific regulator of candida albicans hyphal extension and virulence. <em>Mol. Biol. cell</em><strong>19</strong>, 1354–1365 (2008).
</cite> [<a href="https://doi.org/10.1091/mbc.E07-11-1110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2291399/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18216277/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Banerjee,%20M.%20et%20al.%20UME6,%20a%20novel%20filament-specific%20regulator%20of%20candida%20albicans%20hyphal%20extension%20and%20virulence.%20Mol.%20Biol.%20cell19,%201354%E2%80%931365%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Bensen, E. S., Martin, S. J., Li, M., Berman, J. &amp; Davis, D. A. Transcriptional profiling in candida albicans reveals new adaptive responses to extracellular pH and functions for Rim101p. <em>Mol. Microbiol.</em><strong>54</strong>, 1335–1351 (2004).
</cite> [<a href="https://doi.org/10.1111/j.1365-2958.2004.04350.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15554973/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bensen,%20E.%20S.,%20Martin,%20S.%20J.,%20Li,%20M.,%20Berman,%20J.%20&amp;%20Davis,%20D.%20A.%20Transcriptional%20profiling%20in%20candida%20albicans%20reveals%20new%20adaptive%20responses%20to%20extracellular%20pH%20and%20functions%20for%20Rim101p.%20Mol.%20Microbiol.54,%201335%E2%80%931351%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Carlisle, P. L. &amp; Kadosh, D. Candida albicans Ume6, a filament-specific transcriptional regulator, directs hyphal growth via a pathway involving Hgc1 cyclin-related protein. <em>Eukaryot. cell</em><strong>9</strong>, 1320–1328 (2010).
</cite> [<a href="https://doi.org/10.1128/EC.00046-10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2937344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20656912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carlisle,%20P.%20L.%20&amp;%20Kadosh,%20D.%20Candida%20albicans%20Ume6,%20a%20filament-specific%20transcriptional%20regulator,%20directs%20hyphal%20growth%20via%20a%20pathway%20involving%20Hgc1%20cyclin-related%20protein.%20Eukaryot.%20cell9,%201320%E2%80%931328%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Jacobsen, I. D. The role of host and fungal factors in the commensal-to-pathogen transition of Candida albicans. <em>Curr. Clin. Microbiol. Rep.</em><strong>10</strong>, 55–65 (2023).
</cite> [<a href="https://doi.org/10.1007/s40588-023-00190-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10154278/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37151578/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jacobsen,%20I.%20D.%20The%20role%20of%20host%20and%20fungal%20factors%20in%20the%20commensal-to-pathogen%20transition%20of%20Candida%20albicans.%20Curr.%20Clin.%20Microbiol.%20Rep.10,%2055%E2%80%9365%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Fróis-Martins, R., Lagler, J. &amp; LeibundGut-Landmann, S. Candida albicans virulence traits in commensalism and disease. <em>Curr. Clin. Microbiol. Rep.</em><strong>11</strong>, 231–240 (2024).</cite> [<a href="https://scholar.google.com/scholar_lookup?Fr%C3%B3is-Martins,%20R.,%20Lagler,%20J.%20&amp;%20LeibundGut-Landmann,%20S.%20Candida%20albicans%20virulence%20traits%20in%20commensalism%20and%20disease.%20Curr.%20Clin.%20Microbiol.%20Rep.11,%20231%E2%80%93240%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Liang, S.-H. et al. The hyphal-specific toxin candidalysin promotes fungal gut commensalism. <em>Nature</em><strong>627</strong>, 620–627 (2024).
</cite> [<a href="https://doi.org/10.1038/s41586-024-07142-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11230112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38448595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liang,%20S.-H.%20et%20al.%20The%20hyphal-specific%20toxin%20candidalysin%20promotes%20fungal%20gut%20commensalism.%20Nature627,%20620%E2%80%93627%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Du Toit, A. Hyphae promote Candida albicans fitness and commensalism in the gut. <em>Nat. Rev. Microbiol.</em><strong>22</strong>, 258–258 (2024).
</cite> [<a href="https://doi.org/10.1038/s41579-024-01040-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38472546/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Du%20Toit,%20A.%20Hyphae%20promote%20Candida%20albicans%20fitness%20and%20commensalism%20in%20the%20gut.%20Nat.%20Rev.%20Microbiol.22,%20258%E2%80%93258%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Pande, K., Chen, C. &amp; Noble, S. M. Passage through the mammalian gut triggers a phenotypic switch that promotes Candida albicans commensalism. <em>Nat. Genet.</em><strong>45</strong>, 1088–1091 (2013).
</cite> [<a href="https://doi.org/10.1038/ng.2710" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3758371/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23892606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pande,%20K.,%20Chen,%20C.%20&amp;%20Noble,%20S.%20M.%20Passage%20through%20the%20mammalian%20gut%20triggers%20a%20phenotypic%20switch%20that%20promotes%20Candida%20albicans%20commensalism.%20Nat.%20Genet.45,%201088%E2%80%931091%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Witchley, J. N., Basso, P., Brimacombe, C. A., Abon, N. V. &amp; Noble, S. M. Recording of DNA-binding events reveals the importance of a repurposed Candida albicans regulatory network for gut commensalism. <em>Cell host microbe</em><strong>29</strong>, 1002–1013. e1009 (2021).
</cite> [<a href="https://doi.org/10.1016/j.chom.2021.03.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8216204/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33915113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Witchley,%20J.%20N.,%20Basso,%20P.,%20Brimacombe,%20C.%20A.,%20Abon,%20N.%20V.%20&amp;%20Noble,%20S.%20M.%20Recording%20of%20DNA-binding%20events%20reveals%20the%20importance%20of%20a%20repurposed%20Candida%20albicans%20regulatory%20network%20for%20gut%20commensalism.%20Cell%20host%20microbe29,%201002%E2%80%931013.%20e1009%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Vesely, E. M., Williams, R. B., Konopka, J. B. &amp; Lorenz, M. C. N-Acetylglucosamine metabolism promotes survival of Candida albicans in the phagosome. <em>Msphere</em><strong>2</strong>, 00357–00317 (2017).</cite> [<a href="https://doi.org/10.1128/mSphere.00357-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5588037/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28904994/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vesely,%20E.%20M.,%20Williams,%20R.%20B.,%20Konopka,%20J.%20B.%20&amp;%20Lorenz,%20M.%20C.%20N-Acetylglucosamine%20metabolism%20promotes%20survival%20of%20Candida%20albicans%20in%20the%20phagosome.%20Msphere2,%2000357%E2%80%9300317%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Wang, Y. et al. Innate immune responses against the fungal pathogen candida auris. <em>Nat. Commun.</em><strong>13</strong>, 3553 (2022).
</cite> [<a href="https://doi.org/10.1038/s41467-022-31201-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9213489/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35729111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Y.%20et%20al.%20Innate%20immune%20responses%20against%20the%20fungal%20pathogen%20candida%20auris.%20Nat.%20Commun.13,%203553%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Palková, Z. et al. Ammonia pulses and metabolic oscillations guide yeast colony development. <em>Mol. Biol. cell</em><strong>13</strong>, 3901–3914 (2002).
</cite> [<a href="https://doi.org/10.1091/mbc.E01-12-0149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC133602/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12429834/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Palkov%C3%A1,%20Z.%20et%20al.%20Ammonia%20pulses%20and%20metabolic%20oscillations%20guide%20yeast%20colony%20development.%20Mol.%20Biol.%20cell13,%203901%E2%80%933914%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Chen, C., Pande, K., French, S. D., Tuch, B. B. &amp; Noble, S. M. An iron homeostasis regulatory circuit with reciprocal roles in candida albicans commensalism and pathogenesis. <em>Cell host microbe</em><strong>10</strong>, 118–135 (2011).
</cite> [<a href="https://doi.org/10.1016/j.chom.2011.07.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3165008/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21843869/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20C.,%20Pande,%20K.,%20French,%20S.%20D.,%20Tuch,%20B.%20B.%20&amp;%20Noble,%20S.%20M.%20An%20iron%20homeostasis%20regulatory%20circuit%20with%20reciprocal%20roles%20in%20candida%20albicans%20commensalism%20and%20pathogenesis.%20Cell%20host%20microbe10,%20118%E2%80%93135%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Hsu, P.-C., Yang, C.-Y. &amp; Lan, C.-Y. Candida albicans Hap43 is a repressor induced under low-iron conditions and is essential for iron-responsive transcriptional regulation and virulence. <em>Eukaryot. Cell</em><strong>10</strong>, 207–225 (2011).
</cite> [<a href="https://doi.org/10.1128/EC.00158-10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3067405/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21131439/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hsu,%20P.-C.,%20Yang,%20C.-Y.%20&amp;%20Lan,%20C.-Y.%20Candida%20albicans%20Hap43%20is%20a%20repressor%20induced%20under%20low-iron%20conditions%20and%20is%20essential%20for%20iron-responsive%20transcriptional%20regulation%20and%20virulence.%20Eukaryot.%20Cell10,%20207%E2%80%93225%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Inglis, D. O. &amp; Johnson, A. D. Ash1 protein, an asymmetrically localized transcriptional regulator, controls filamentous growth and virulence of candida albicans. <em>Mol. Cell. Biol.</em><strong>22</strong>, 8669–8680 (2002).
</cite> [<a href="https://doi.org/10.1128/MCB.22.24.8669-8680.2002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC139894/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12446785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Inglis,%20D.%20O.%20&amp;%20Johnson,%20A.%20D.%20Ash1%20protein,%20an%20asymmetrically%20localized%20transcriptional%20regulator,%20controls%20filamentous%20growth%20and%20virulence%20of%20candida%20albicans.%20Mol.%20Cell.%20Biol.22,%208669%E2%80%938680%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Abramson, J. et al. Accurate structure prediction of biomolecular interactions with AlphaFold 3. <em>Nature</em><strong>630</strong>, 493–500 (2024).
</cite> [<a href="https://doi.org/10.1038/s41586-024-07487-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11168924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38718835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abramson,%20J.%20et%20al.%20Accurate%20structure%20prediction%20of%20biomolecular%20interactions%20with%20AlphaFold%203.%20Nature630,%20493%E2%80%93500%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Ewels, P., Magnusson, M., Lundin, S. &amp; Käller, M. MultiQC: summarize analysis results for multiple tools and samples in a single report. <em>Bioinformatics</em><strong>32</strong>, 3047–3048 (2016).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5039924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27312411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ewels,%20P.,%20Magnusson,%20M.,%20Lundin,%20S.%20&amp;%20K%C3%A4ller,%20M.%20MultiQC:%20summarize%20analysis%20results%20for%20multiple%20tools%20and%20samples%20in%20a%20single%20report.%20Bioinformatics32,%203047%E2%80%933048%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Bolger, A. M., Lohse, M. &amp; Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. <em>Bioinformatics</em><strong>30</strong>, 2114–2120 (2014).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4103590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bolger,%20A.%20M.,%20Lohse,%20M.%20&amp;%20Usadel,%20B.%20Trimmomatic:%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data.%20Bioinformatics30,%202114%E2%80%932120%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. <em>Bioinformatics</em><strong>29</strong>, 15–21 (2013).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bts635" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3530905/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23104886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dobin,%20A.%20et%20al.%20STAR:%20ultrafast%20universal%20RNA-seq%20aligner.%20Bioinformatics29,%2015%E2%80%9321%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Liao, Y., Smyth, G. K. &amp; Shi, W. featureCounts: an efficient general purpose program for assigning sequence reads to genomic features. <em>Bioinformatics</em><strong>30</strong>, 923–930 (2014).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt656" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24227677/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liao,%20Y.,%20Smyth,%20G.%20K.%20&amp;%20Shi,%20W.%20featureCounts:%20an%20efficient%20general%20purpose%20program%20for%20assigning%20sequence%20reads%20to%20genomic%20features.%20Bioinformatics30,%20923%E2%80%93930%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <em>Genome Biol.</em><strong>15</strong>, 1–21 (2014).</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Love,%20M.%20I.,%20Huber,%20W.%20&amp;%20Anders,%20S.%20Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2.%20Genome%20Biol.15,%201%E2%80%9321%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Gu, Z., Eils, R. &amp; Schlesner, M. Complex heatmaps reveal patterns and correlations in multidimensional genomic data. <em>Bioinformatics</em><strong>32</strong>, 2847–2849 (2016).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw313" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27207943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gu,%20Z.,%20Eils,%20R.%20&amp;%20Schlesner,%20M.%20Complex%20heatmaps%20reveal%20patterns%20and%20correlations%20in%20multidimensional%20genomic%20data.%20Bioinformatics32,%202847%E2%80%932849%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Yu, G., Wang, L.-G., Han, Y. &amp; He, Q.-Y. clusterProfiler: an R package for comparing biological themes among gene clusters. <em>Omics: a J. Integr. Biol.</em><strong>16</strong>, 284–287 (2012).</cite> [<a href="https://doi.org/10.1089/omi.2011.0118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3339379/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22455463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu,%20G.,%20Wang,%20L.-G.,%20Han,%20Y.%20&amp;%20He,%20Q.-Y.%20clusterProfiler:%20an%20R%20package%20for%20comparing%20biological%20themes%20among%20gene%20clusters.%20Omics:%20a%20J.%20Integr.%20Biol.16,%20284%E2%80%93287%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Langmead, B. &amp; Salzberg, S. L. Fast gapped-read alignment with bowtie 2. <em>Nat. methods</em><strong>9</strong>, 357–359 (2012).
</cite> [<a href="https://doi.org/10.1038/nmeth.1923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322381/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Langmead,%20B.%20&amp;%20Salzberg,%20S.%20L.%20Fast%20gapped-read%20alignment%20with%20bowtie%202.%20Nat.%20methods9,%20357%E2%80%93359%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Tarasov, A., Vilella, A. J., Cuppen, E., Nijman, I. J. &amp; Prins, P. Sambamba: fast processing of NGS alignment formats. <em>Bioinformatics</em><strong>31</strong>, 2032–2034 (2015).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4765878/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25697820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tarasov,%20A.,%20Vilella,%20A.%20J.,%20Cuppen,%20E.,%20Nijman,%20I.%20J.%20&amp;%20Prins,%20P.%20Sambamba:%20fast%20processing%20of%20NGS%20alignment%20formats.%20Bioinformatics31,%202032%E2%80%932034%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Li, H. et al. The sequence alignment/map format and SAMtools. <em>bioinformatics</em><strong>25</strong>, 2078–2079 (2009).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2723002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19505943/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20H.%20et%20al.%20The%20sequence%20alignment/map%20format%20and%20SAMtools.%20bioinformatics25,%202078%E2%80%932079%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Ramírez, F., Dündar, F., Diehl, S., Grüning, B. A. &amp; Manke, T. deepTools: a flexible platform for exploring deep-sequencing data. <em>Nucleic acids Res.</em><strong>42</strong>, W187–W191 (2014).
</cite> [<a href="https://doi.org/10.1093/nar/gku365" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4086134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24799436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ram%C3%ADrez,%20F.,%20D%C3%BCndar,%20F.,%20Diehl,%20S.,%20Gr%C3%BCning,%20B.%20A.%20&amp;%20Manke,%20T.%20deepTools:%20a%20flexible%20platform%20for%20exploring%20deep-sequencing%20data.%20Nucleic%20acids%20Res.42,%20W187%E2%80%93W191%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Feng, J., Liu, T., Qin, B., Zhang, Y. &amp; Liu, X. S. Identifying ChIP-seq enrichment using MACS. <em>Nat. Protoc.</em><strong>7</strong>, 1728–1740 (2012).
</cite> [<a href="https://doi.org/10.1038/nprot.2012.101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3868217/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22936215/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Feng,%20J.,%20Liu,%20T.,%20Qin,%20B.,%20Zhang,%20Y.%20&amp;%20Liu,%20X.%20S.%20Identifying%20ChIP-seq%20enrichment%20using%20MACS.%20Nat.%20Protoc.7,%201728%E2%80%931740%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Tebbji, F. et al. A functional portrait of Med7 and the mediator complex in candida albicans. <em>PLoS Genet.</em><strong>10</strong>, e1004770 (2014).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1004770" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4222720/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25375174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tebbji,%20F.%20et%20al.%20A%20functional%20portrait%20of%20Med7%20and%20the%20mediator%20complex%20in%20candida%20albicans.%20PLoS%20Genet.10,%20e1004770%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Hussain, Z. Created in BioRender. <a href="https://BioRender.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/</a> uozl5bb (2025).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary information</a><sup> (3.4MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62953_MOESM2_ESM.pdf</a><sup> (129.5KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (330.3KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (42.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (40.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (65.2KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM7_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 5</a><sup> (15.9KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM8_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 6</a><sup> (15.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM9_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 7</a><sup> (11.8KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM10_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 8</a><sup> (13.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM11_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (96.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM12_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material13_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373999/bin/41467_2025_62953_MOESM13_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source data</a><sup> (3.9MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data necessary to evaluate the conclusions of this study are provided within the main text and/or the Supplementary Materials. The Dal81 ChIP-seq, Stp2 ChIP -seq, and RNA-seq datasets have been deposited in the National Center for Biotechnology Information Sequence Read Archive (NCBI SRA) under accession number <a href="https://www.ncbi.nlm.nih.gov/bioproject/PRJNA1190138/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PRJNA1190138</a>. Source Data are available with this paper. <a href="#Sec39" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62953-x"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62953.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373999/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373999/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373999%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373999/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373999/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373999/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846842/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373999/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846842/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373999/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373999/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="5D7VROWDSBqMnL0b53hr7SfbYSaBGgDqXe9HIdvVInrPu3ugwQwAGufW893hSf8O">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
