
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Investigating the phenotypic alterations associated with hypermucoviscous hypervirulent Klebsiella pneumoniae during phage resistance development - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE532CF78AF1FC13052CF7002F1008CE.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcmicr">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372256/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Microbiology">
<meta name="citation_title" content="Investigating the phenotypic alterations associated with hypermucoviscous hypervirulent Klebsiella pneumoniae during phage resistance development">
<meta name="citation_author" content="Ramya Juliet">
<meta name="citation_author_institution" content="Antibiotic Resistance and Phage Therapy Laboratory, Centre for Advanced Research in Bacteriophage and Infectious Diseases, School of Bio Sciences and Technology, Vellore Institute of Technology (VIT), Vellore, 632014 Tamil Nadu India">
<meta name="citation_author" content="Ramesh Nachimuthu">
<meta name="citation_author_institution" content="Antibiotic Resistance and Phage Therapy Laboratory, Centre for Advanced Research in Bacteriophage and Infectious Diseases, School of Bio Sciences and Technology, Vellore Institute of Technology (VIT), Vellore, 632014 Tamil Nadu India">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="25">
<meta name="citation_firstpage" content="528">
<meta name="citation_doi" content="10.1186/s12866-025-04268-x">
<meta name="citation_pmid" content="40841996">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372256/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372256/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372256/pdf/12866_2025_Article_4268.pdf">
<meta name="description" content="Phage therapy has been explored and used compassionately in the post-antibiotic era, though phage resistance might pose a serious challenge. The advent of hypervirulent and hypermucoviscous traits in Klebsiella pneumoniae limits therapeutic choices. ...">
<meta name="og:title" content="Investigating the phenotypic alterations associated with hypermucoviscous hypervirulent Klebsiella pneumoniae during phage resistance development">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Phage therapy has been explored and used compassionately in the post-antibiotic era, though phage resistance might pose a serious challenge. The advent of hypervirulent and hypermucoviscous traits in Klebsiella pneumoniae limits therapeutic choices. ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372256/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12372256">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12866-025-04268-x"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12866_2025_Article_4268.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372256%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12372256/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12372256/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372256/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcmicr.png" alt="BMC Microbiology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Microbiology" title="Link to BMC Microbiology" shape="default" href="https://bmcmicrobiol.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Microbiol</button></div>. 2025 Aug 22;25:528. doi: <a href="https://doi.org/10.1186/s12866-025-04268-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12866-025-04268-x</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Microbiol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Microbiol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Microbiol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Investigating the phenotypic alterations associated with hypermucoviscous hypervirulent <em>Klebsiella pneumoniae</em> during phage resistance development</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Juliet%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Ramya Juliet</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Ramya Juliet</span></h3>
<div class="p">
<sup>1</sup>Antibiotic Resistance and Phage Therapy Laboratory, Centre for Advanced Research in Bacteriophage and Infectious Diseases, School of Bio Sciences and Technology, Vellore Institute of Technology (VIT), Vellore, 632014 Tamil Nadu India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Juliet%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ramya Juliet</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nachimuthu%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ramesh Nachimuthu</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ramesh Nachimuthu</span></h3>
<div class="p">
<sup>1</sup>Antibiotic Resistance and Phage Therapy Laboratory, Centre for Advanced Research in Bacteriophage and Infectious Diseases, School of Bio Sciences and Technology, Vellore Institute of Technology (VIT), Vellore, 632014 Tamil Nadu India </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nachimuthu%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ramesh Nachimuthu</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Antibiotic Resistance and Phage Therapy Laboratory, Centre for Advanced Research in Bacteriophage and Infectious Diseases, School of Bio Sciences and Technology, Vellore Institute of Technology (VIT), Vellore, 632014 Tamil Nadu India </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Apr 29; Accepted 2025 Jul 25; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12372256  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841996/" class="usa-link">40841996</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Phage therapy has been explored and used compassionately in the post-antibiotic era, though phage resistance might pose a serious challenge. The advent of hypervirulent and hypermucoviscous traits in <em>Klebsiella pneumoniae</em> limits therapeutic choices. This study investigated the phage resistance in hypermucoviscous hypervirulent <em>Klebsiella pneumoniae</em> clinical strain Kleb_53. A Klebsiella phage Disc against the Kleb_53 strain was isolated from sewage. The phage exhibited stability between − 20 °C and 60 °C and within the pH range of 3 to 11. The phage adsorption time was 15 min, with a latent period of 30 min and a burst size of 354 virions. The phage-resistant Kleb_53 variants were screened and examined for their phenotypic variations, antibiotic susceptibility, and biofilm formation. Colony morphotype variants were observed, including smooth, rough, and small colony variants. String, aggregation, and wetness tests confirmed reduced mucoviscosity. The plaque morphology differed between the wild and variants. Additionally, resistance to meropenem and third-generation cephalosporins was reversed, whereas the biofilm-forming ability varied among the recovered variants. This study demonstrates that ongoing phage-host interactions drive phenotypic changes and the emergence of phage-resistant variants with altered antibiotic susceptibility and biofilm-forming capacity. It also underscores the need for further research on phage resistance and strategies to overcome it for the effective application of phage therapy.</p>
<section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Antibiotic resistance, Phage resistance, <em>Klebsiella pneumoniae</em>, Hypermucoviscous, Phage Therapy, Biofilms</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par30">Bacteriophages (phages) and bacteria have existed together and interdependently evolved for eons, shaping each other’s evolutionary trajectories through complex interactions [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Phages are prokaryotic viruses that infect and kill bacteria by binding to the cell wall receptor, such as the proteins, polysaccharides, capsules, pili, or flagella, of the host bacteria through their tail spikes or tail fiber proteins [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Following the initial reversible binding and irreversible attachment, the phage ejects its genome into the host [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>], initiating either the lytic cycle, resulting in bacterial killing, or a lysogenic cycle where the phage genome integrates into the host as a prophage or plasmid [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>].</p>
<p id="Par31">Bacteria have evolved various adaptations to counter each stage of phage infection. One of the primary defense mechanisms is phenotypic variation, which can significantly hinder phage adsorption. Bacteria prevent phage adsorption by modifying/losing the phage receptor proteins on their cell surface, forming biofilms, and producing capsules or decoy receptors. Other mechanisms include abortive infection (phage-infected bacteria self-destruct to protect their bacterial population), restriction-modification system, superinfection exclusion, and CRISPR-Cas [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]. The ongoing co-evolution between phages and bacteria can follow two major phases. Either it occurs through an arms race, marked by repeated cycles of resistance and sensitivity, or through directional evolution, where ongoing mutations in both organisms lead to phages with expanded host ranges and bacteria capable of resisting multiple phages [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Changes in gene expression regulation are often transient and driven by selective pressure; this reversible phenomenon is known as phase variation [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>].</p>
<p id="Par32">Adaptations associated with phage resistance often lead to phenotypic changes, such as the loss of lipopolysaccharides, capsule, mucosity, and alterations in biofilm formation. These changes can restore antibiotic sensitivity and increase bacterial vulnerability to the host immune response [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. This phenomenon, often referred to as a fitness trade-off, highlights how resistance to phages may come at the cost of susceptibility to antibiotics and host immunity [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Antibiotic resistance and phage resistance are bacterial defense mechanisms that help them survive in challenging ecological pressures [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. Although phage therapy has been gaining interest, the area of phage resistance in bacteria, particularly the associated phenotypic and genotypic adaptations, remains largely unexplored.</p>
<p id="Par33"><em>Klebsiella pneumoniae</em> is a major gram-negative pathogen and the second leading cause of nosocomial infections worldwide [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. It exhibits two major pathotypes: classical (cKp) and hypervirulent (HvKp). The classical form is typically linked to multidrug resistance and hospital-acquired infections, while the hypervirulent form is characterized by enhanced virulence and is more commonly associated with community-acquired infections [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. The emergence of hypermucoviscous hypervirulent <em>K. pneumoniae</em> (HmHvKp) poses a growing threat due to its enhanced capsule production, iron acquisition capabilities, and increasing antibiotic resistance, particularly to carbapenems, resulting in high mortality rates ranging from 11.9 to 37.1% [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>–<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>].</p>
<p id="Par34">Given the rise of drug-resistant HmHvKp, Klebsiella-specific phages have gained attention as potential therapeutic agents [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. These phages often encode depolymerases capable of degrading the overproduced capsule and exopolysaccharides, allowing them to effectively infect hypervirulent strains that are otherwise difficult to treat with antibiotics [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. This study investigates the phenotypic alterations that occur in the hypermucoviscous hypervirulent <em>K. pneumoniae</em> (HmHvKp) during the development of phage resistance, especially focusing on changes in colony morphology, capsule production, and virulence traits that may impact phage therapy effectiveness.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methodology</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Bacterial identification and characterization</h3>
<p id="Par35">A clinical strain of <em>K. pneumoniae</em>, Kleb_53, was collected from a diagnostic center in Chennai, Tamil Nadu, India. Initially, the strain was identified using the VITEK identification system and later confirmed by 16 S rRNA sequencing. The universal primers 27 F (5′-AGAGTTTGATCCTGGCTCAG-3′) and 1492R (5′-GGTTACCTTGTTACGACTT-3′) were used, and the amplified product was sequenced and deposited in GenBank (Accession no <a href="https://www.ncbi.nlm.nih.gov/nuccore/PV029729" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV029729</a>) [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. The hypermucoviscosity of the clinical strain was tested using the string test [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]. Biofilm formation of the Kleb_53 clinical strain was quantified using the CV assay and categorized using the protocol described previously [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>].</p>
<p id="Par36">The micro broth dilution method was performed for the three last resort antibiotics, meropenem, colistin, and tigecycline, following Clinical and Laboratory Standards Institute (CLSI, 2023) to determine the minimum inhibitory concentration [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. The resistance genes <em>bla</em><sub>NDM</sub>, <em>bla</em><sub>OXA−48 like</sub>, and <em>bla</em><sub>KPC</sub> in Kleb_53 were screened using the primers F- GCAGCTTGTCGGCCATGCGGGC, R- GGTCGCGAAGCTGAGCACCGCAT; F- GCGTGGTTAAGGATGAACAC, R-CATCAAGTTCAACCCAACCG; F- TGTCACTGTATCGCCGTC, R- CTCAGTGCTCTACAGAAAACC, respectively [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>] and the virulence genes were screened using the primers aerobactin F- 5′-GCATAGGCGGATACGAACAT-3′ and R- 5′-CACAGGGCAATTGCTTACCT-3′, <em>rmpA</em> gene F-5′-ACTGGGCTACCTCTGCTTCA-3′ and R- 5′-CTTGCATGAGCCATCTTTCA-3′ [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>, <a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. All the PCR conditions were described previously [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>].</p></section><section id="Sec4"><h3 class="pmc_sec_title">Bacteriophage isolation</h3>
<p id="Par37">Using the phage enrichment technique, Kleb_53 was used as a host to isolate bacteriophage from sewage water collected from Vellore, Tamil Nadu, India. Briefly, sewage was added to a logarithmic phase bacterial culture and incubated overnight at 37° C, 120 rpm. After incubation, the mixture was centrifuged at 7000 rpm for 15 min [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. The supernatant was filtered by a 0.22-µm pore-sized membrane syringe filter and tested for the presence of phage through a spot test. Briefly, 10 µl of supernatant from enrichment was spotted on a bacterial lawn of Kleb_53 in Luria Bertani (LB) agar medium (Hi Media, India) and incubated at 37 °C. The presence of bacterial clearance indicates a positive spot test. Double agar overlay (DAOL) was performed to confirm the presence of phages and observe plaque morphology. In DAOL, 200 µl of the host, 100 µl of supernatant, and 4 ml of molten soft agar (0.45%) were mixed, poured onto the hard agar plate (2%), and incubated at 37 °C until the appearance of plaques [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. The clear, transparent plaques represent the phages’ lytic nature, whereas the turbid plaques represent their lysogenic nature.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Purification and multiplication of the phage</h3>
<p id="Par38">A clear lytic plaque was selected from the DAOL plate, and purified using the pickate method to obtain a homogenous population of phages. 4 mL of SM buffer [5.8 g NaCl, 50 mL 1 M Tris–HCl (pH 7.5), 2 g MgSO<sub>4</sub>.7H<sub>2</sub>O, and 5 mL of 2% gelatin for 1,000 mL] was flooded onto the plate containing uniform plaque morphology, and incubated for 4 h at 4 °C. The buffer was collected from the plate and centrifuged at 7000 rpm for 10 min. The supernatant was added to the logarithmic phase bacterial culture, allowed to multiply for 24 h, centrifuged, and filtered using a 0.22 μm membrane syringe filter [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]. The phage titer was calculated in plaque-forming units per milliliter (PFU/mL) using the agar overlay method and stored at 4 °C until further use.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Host range and stability</h3>
<p id="Par39">The host range of the phage was identified by performing a spot test against 145 non-repetitive clinical isolates of <em>Klebsiella</em> species. The phage stability at various temperatures and pH was determined. Briefly, 100 µL of the phage lysate (10<sup>10</sup> PFU/mL) was incubated at different temperatures such as − 20°, 4°, 20°, 30°, 37°, 50°, 60°, and 70 °C and with various pH 3, 5, 7, 9, and 11 for 1 h and the titer was determined using DAOL. The experiment was performed in triplicate, and the graph was plotted [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>].</p></section><section id="Sec7"><h3 class="pmc_sec_title">Microscopic examination</h3>
<p id="Par40">Transmission electron microscopy (TEM) was used to examine the phage morphology. Briefly, 10 µl of phage (10<sup>10</sup> PFU/mL) was coated onto a copper grid. The grid was negatively stained using 1% (w/v) uranyl acetate, washed twice with sterile water to remove excess stain, and observed under a transmission electron microscope (FEI-TECNAI G2-20 TWIN, VIT, Vellore) [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>].</p></section><section id="Sec8"><h3 class="pmc_sec_title">Phage adsorption and one-step growth curve</h3>
<p id="Par41">The phage's time to adsorb to the bacterial cell is called adsorption. Briefly, to a logarithmic phase bacterium (OD-0.08), 10<sup>6</sup> PFU/mL phage lysate (Multiplicity of Infection, MOI= 0.01) was added, aliquots were taken at 3 minute time intervals for 15 min, treated with 1 % chloroform to eliminate the bacteria, and DAOL was performed to quantify the number of free phages. In the one-step growth curve, the latency period (time taken by the adsorbed phage to lyse the host) and burst size (number of phages released per bacterial cell) were calculated. Briefly, the phage (MOI 0.01) and host mixture were incubated for 15 min (pre-determined adsorption time) at 37°C, centrifuged at 10,000 rpm for 5 min, the supernatant containing the free phages was discarded, and the pellet was resuspended in broth. DAOL was performed every 10 min up to 1 hour to titrate the number of adsorbed phages. Burst size is the number of phage particles released per infected bacterial cell, calculated by subtracting the initial PFU/mL from the first burst [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. All experiments were conducted in triplicate, and data were analyzed and plotted using GraphPad Prism software.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Acquisition of phage-resistant variants</h3>
<p id="Par42">The phage-resistant variants were obtained using the modified protocol [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Briefly, 10<sup>10</sup>PFU/mL phage was added to the host bacterium Kleb_53 (wild-type, 10<sup>8</sup> CFU/mL) and incubated overnight at 37 °C, 120 rpm. Centrifuged at 6500 rpm for 10 min, the supernatant (containing the phage) was removed, and the pellet (bacteria) was washed twice with SM buffer and resuspended in the LB broth. 100 µl of resuspended bacteria were spread plated on LB and Mac Conkey agar plates using sterile L-rod, the plates were incubated at 37 °C, and observed for colony morphology variations. The 10<sup>10</sup> PFU/mL fresh phage was again added to the resuspended pellet containing the phage-interacting host and incubated overnight, centrifuged, and a spread plate was performed, and the protocol was repeated for 5 days. Three similar colonies of each morphotype were stabbed and stored. For every experiment, the culture was recovered from the stab to avoid repeated subculturing and loss of acquired characteristic phenotype.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Variant growth dynamics</h3>
<p id="Par43">The growth pattern of the Kleb_53 morphotype variants obtained from wild type and phage interaction was determined by measuring optical density readings. The overnight culture was diluted to 0.1 OD and incubated at 37 °C, 120 rpm. Optical density readings were taken at 2 h intervals for 24 h at a wavelength of 600 nm with a spectrophotometer (UV-1280, Shimadzu) [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. The experiments were performed in triplicate for each morphotype, and the graph was plotted based on the mean ± SD values.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Phenotypic methods for assessing the morphological variations in phage-resistant variants</h3>
<section id="Sec12"><h4 class="pmc_sec_title">Plaque morphology variations</h4>
<p id="Par44">All the sub-cultured colonies from the spread plate were inoculated in LB broth, incubated, and adjusted to a 0.5 McFarland turbidity standard. A spot test was performed against each colony variant. The colonies were categorized into two, based on their phage sensitivity. The presence of lysis in the spot test, regardless of observed colony morphology changes, was classified as a phage-sensitive variant (PSV). Variants that did not exhibit lysis in the spot assay were defined as phage-insensitive/resistant variants (PRV). Thus, morphological variation alone was not used to infer phage resistance, as some variants with altered appearance remained susceptible to phage-mediated lysis. Since this study involves only phenotypic observation, the colonies were named as variants, not mutants. The phage-sensitive variants were subjected to double agar overlay to observe for plaque morphology variations. The efficiency of plaquing (EOP) was also examined for the phage-sensitive variants using a double agar overlay.</p></section><section id="Sec13"><h4 class="pmc_sec_title">String test</h4>
<p id="Par45">All the colony variants were examined for the hypermucoviscous phenotype. Briefly, a fresh overnight-grown colony was gently touched with the loop and extended to observe the formation of the viscous string. If the string formed is above 5 mm, it’s considered to be positive for the string test [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>].</p></section><section id="Sec14"><h4 class="pmc_sec_title">Aggregation index</h4>
<p id="Par46">A single colony was inoculated into LB broth from each morphotype variant and incubated overnight at 37°C, 120 rpm. After incubation, the early stationary phase cultures were subjected to slow-speed centrifugation at 2500 rpm for 5 min. The supernatant was transferred to a new tube. The pellet/ sediment was resuspended in broth. The optical density readings of the supernatant and resuspended pellet were measured at 600 nm. The aggregation index was calculated based on the given formula [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. The experiment was performed in triplicate for each of the three colonies belonging to the same colony morphotype, and the graph was plotted based on the mean value. </p>
<p id="Par47"> •Aggregation index= Supernatant OD/ Sediment OD</p></section><section id="Sec15"><h4 class="pmc_sec_title">Wetness assay</h4>
<p id="Par48">A wetness assay was performed for all the variants to determine the wetness of the colony in a swarming plate (0.4% agar). All the variants and wild-type were spotted at a concentration of 10<sup>8</sup> CFU/mL and incubated overnight. A capillary tube was placed on the fresh-grown colony for 10 s. The amount of wetness drawn in the capillary tube is measured in millimeters [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. The experiment was performed in triplicate for each morphotype.</p></section><section id="Sec16"><h4 class="pmc_sec_title">Antibiotic susceptibility testing</h4>
<p id="Par49">Kirby Bauer disk diffusion assay was performed for the variants to observe any changes in their susceptibility pattern compared to the wild type. The following antibiotics were used for screening: meropenem (MRP), imipenem (IPM), cefotaxime/clavulanic acid (CEC), ceftriaxone (CTR), ceftazidime (CAZ), piperacillin-tazobactam (PIT), ciprofloxacin (CIP), and gentamicin (GEN). Briefly, a lawn of culture adjusted to 0.5 McFarland was swabbed on a Mueller Hinton agar plate, disks were placed, and incubated for 16–18 h at 37 °C [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. The zone of inhibition was recorded, the experiments were replicated independently, and the mean values were interpreted based on CLSI guidelines [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>].</p></section><section id="Sec17"><h4 class="pmc_sec_title">Biofilm formation</h4>
<p id="Par50">Biofilm formation of all the colony morphotype variants was quantified using a crystal violet (CV) assay. Briefly, 100 µl of 10: 100 diluted overnight culture was added to the 96-well flat-bottomed microtiter plate and incubated for 24 h at 37 °C. Post-incubation, the planktonic cells were removed, washed twice, and stained with 0.1% crystal violet for 15 min. The stain was removed, and the wells were washed twice. The CV stain was dissolved using 33.3% glacial acetic acid. The optical density reading was taken at 570 nm (BioTek India) [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Wild-type biofilm served as a positive control, and only broth as a negative control. The experiments were conducted in triplicate, and the graph was plotted.</p></section></section><section id="Sec18"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par51">All the experiments were performed in triplicate, and the data were represented as the mean ± standard deviation (SD). The results of the stability, aggregation index, wetness assay, and biofilm formation experiments were analyzed using a t-test and one-way ANOVA. The results were considered significant if p &lt; 0.05. All the data analysis was carried out in GraphPad Prism 8.0 software.</p></section></section><section id="Sec19"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec20"><h3 class="pmc_sec_title">Characteristics of the host strain kleb_53</h3>
<p id="Par52">The clinical strain Kleb_53 used in this study tested positive for the string test (string formed &gt; 5 mm) and was found to be a moderate biofilm producer (OD-0.76; ODC-0.25) based on the criteria established by Loganathan et al., 2022, where moderate producers are defined as 2×ODC &lt; OD ≤ 4×ODC. The strain resisted all three antibiotics tested in MIC: meropenem, colistin, and tigecycline (all ≥ 4 µg/mL). There was no presence of the screened resistance genes. Aerobactin (GenBank accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/PP994840" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP994840</a>) and rmpA (GenBank accession number <a href="https://www.ncbi.nlm.nih.gov/nuccore/PP994850" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP994850</a>) genes were present. Therefore, the strain Kleb_53 was classified as hypervirulent (Hv) and hypermucoviscous (Hm) Klebsiella pneumoniae (HvHmKp).</p></section><section id="Sec21"><h3 class="pmc_sec_title">Isolation and morphological analysis</h3>
<p id="Par53">A lytic bacteriophage, named Klebsiella phage Disc (hereafter referred to as ‘Disc’), was isolated from a sewage water sample using the Kleb_53 strain as the host. The phage was subsequently purified and propagated using the same host strain throughout the study. The plaque was clear, circular, and transparent with a halo zone of 4 mm, indicating depolymerase activity. TEM analysis of the Disc phage showed a T5-like morphology, with a 50 ± 3.0 nm icosahedral head and 170 ± 3.0 nm long non-contractile tail (Fig. <a href="#Fig1" class="usa-link">1</a>C).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372256_12866_2025_4268_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f431/12372256/debab6b1d88b/12866_2025_4268_Fig1_HTML.jpg" loading="lazy" id="d33e588" height="193" width="744" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Morphological characterization of the Disc phage. <strong>A</strong> Spot test plate showing dilution spots of the Disc phage lysate. <strong>B</strong> Double agar overlay plate displaying clear, circular lytic plaques with 4 mm semi-translucent halo zones, indicating depolymerase activity. <strong>C</strong> Transmission electron microscopy (TEM) image of the Disc phage revealing T5-like morphology, characterized by an icosahedral head and a long non-contractile tail</p></figcaption></figure></section><section id="Sec22"><h3 class="pmc_sec_title">Host range and stability</h3>
<p id="Par54">The Disc phage was able to lyse and form plaques in 48 out of 145 <em>Klebsiella</em> clinical strains. The Disc phage remained stable between − 20 and 37 ˚C with no significant difference in the titer (p &gt; 0.05). In contrast, it decreased at 50˚C (p = 0.02) and 60˚C (p = 0.0002) and was inactivated at 70˚C (p &lt; 0.0001) (Fig. 2A). The Disc phage showed high stability at pH 7 and remained stable at pH 5 (p = 0.0002) and 9 (p = 0.01). In contrast, the titer was reduced at extreme acidic and alkaline conditions of pH 3 (p &lt; 0.0001) and pH 11 (p &lt; 0.0001) (Fig. <a href="#Fig2" class="usa-link">2</a>B).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372256_12866_2025_4268_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f431/12372256/6ec7b1611a21/12866_2025_4268_Fig2_HTML.jpg" loading="lazy" id="d33e644" height="526" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The biological characteristics of the Disc phage. <strong>A</strong> Thermal stability, showing no significant difference in titer between − 20 °C and 37 °C (<em>p</em> &gt; 0.05), with reductions observed at 50 °C (<em>p</em> = 0.02), 60 °C (<em>p</em> = 0.0002), and complete inactivation at 70 °C (<em>p</em> &lt; 0.0001). <strong>B</strong> pH stability, showing high stability at pH 7 and moderate stability at pH 5 (<em>p</em> = 0.0002) and pH 9 (<em>p</em> = 0.01), with significant reductions at pH 3 and pH 11 (both <em>p</em> &lt; 0.0001). <strong>C</strong> Phage life cycle focusing on adsorption, and (<strong>D</strong>) one-step growth curve showing a latency period (L) of 30 min and a burst size (<strong>B</strong>) of 354 phage virions per infected cell. All experiments were conducted in triplicate, and the data shown are represented as mean ± SD</p></figcaption></figure></section><section id="Sec23"><h3 class="pmc_sec_title">Life cycle of disc phage</h3>
<p id="Par55">The Disc phage attained 84% adsorption to its host within 6 min and 94% adsorption in 15 min (Fig. <a href="#Fig2" class="usa-link">2</a>C). In a one-step growth curve, the latent period of the phage was found to be 30 min with a rise period of 20 min. The Disc phage had a high burst size of 354 phages per infected cell (Fig. <a href="#Fig2" class="usa-link">2</a>D).</p></section><section id="Sec24"><h3 class="pmc_sec_title">Phage-resistant variants</h3>
<p id="Par56">Phages were allowed to interact with the same host inoculum for five consecutive days. The spread plate technique after each day revealed the variation in the host (wild type) colony morphology. Various colony morphotypes were observed during the study, such as rough dry irregular colony (R), smooth circular mucoid colony (S), and small colony variants (SCV-S, smooth and SCV-R, rough) (Fig. <a href="#Fig3" class="usa-link">3</a>; Table <a href="#Tab1" class="usa-link">1</a>). The bacterial growth dynamics of the variants showed deviation in the growth pattern from the wild type, wherein S and R colonies exhibited almost similar growth patterns. However, small colony variants couldn’t reach more than 0.8 OD, indicating lower cell densities, in which SCV-R had better growth than SCV-S (Fig. <a href="#Fig4" class="usa-link">4</a>). After day 5, the colonies formed on the spread plate became unculturable, yet remained viable. This suggests a potential bacterial survival trade-off, although delayed growth due to phage-induced physiological burden or persistence mechanisms cannot be ruled out.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372256_12866_2025_4268_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f431/12372256/1c96254e28d4/12866_2025_4268_Fig3_HTML.jpg" loading="lazy" id="d33e811" height="182" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Spread plate representation of colony morphotype variants during phage resistance screening. <strong>A</strong>–<strong>E</strong> Spread plates showing the emergence of different colony morphotypes from Day 1 to Day 5 during the screening of phage-resistant variants. The colony diameters ranged from approximately 0.1–4 mm after 24 h of incubation on MacConkey agar. Arrow colors indicate morphotypes: orange – smooth (S), blue – rough (R), black – small colony variant-smooth (SCV-S), green – small colony variant-rough (SCV-R)</p></figcaption></figure><section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Phenotypic variations observed in the wild-type colony during the screening of phage-resistant variants from day 1 to day 5. ✓ indicates the presence of a specific morphotype. Phage activity refers to the ability of the disc phage to lyse each variant in the spot test: ‘+’ indicates Lysis (phage-sensitive), and ‘–’ indicates no Lysis (phage-resistant/insensitive). The colony morphotypes are defined as follows: S – smooth, R – rough, SCV-S – small colony variant-smooth, and SCV-R – small colony variant-rough</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Colony morphotype variants</th>
<th align="left" colspan="2" rowspan="1">Day 1</th>
<th align="left" colspan="2" rowspan="1">Day 2</th>
<th align="left" colspan="2" rowspan="1">Day 3</th>
<th align="left" colspan="2" rowspan="1">Day 4</th>
<th align="left" colspan="2" rowspan="1">Day 5</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">Present</th>
<th align="left" colspan="1" rowspan="1">Phage activity</th>
<th align="left" colspan="1" rowspan="1">Present</th>
<th align="left" colspan="1" rowspan="1">Phage activity</th>
<th align="left" colspan="1" rowspan="1">Present</th>
<th align="left" colspan="1" rowspan="1">Phage activity</th>
<th align="left" colspan="1" rowspan="1">Present</th>
<th align="left" colspan="1" rowspan="1">Phage activity</th>
<th align="left" colspan="1" rowspan="1">Present</th>
<th align="left" colspan="1" rowspan="1">Phage activity</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">S</td>
<td align="left" colspan="1" rowspan="1">⎫</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">⎫</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">⎫</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">R</td>
<td align="left" colspan="1" rowspan="1">⎫</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">⎫</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SCV-S</td>
<td align="left" colspan="1" rowspan="1">⎫</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">⎫</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">⎫</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">⎫</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">⎫</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SCV-R</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1">⎫</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">⎫</td>
<td align="left" colspan="1" rowspan="1">+</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372256_12866_2025_4268_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f431/12372256/f6f4eb450a8f/12866_2025_4268_Fig4_HTML.jpg" loading="lazy" id="d33e818" height="315" width="744" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Bacterial growth curves of colony morphotype variants emerging during bacteria-phage interactions. Growth was monitored over time, and all experiments were performed in triplicate. Data are presented as mean ± standard deviation (SD)</p></figcaption></figure></section><section id="Sec25"><h3 class="pmc_sec_title">Plaque morphology variation</h3>
<p id="Par57">Spot tests against the wild and variants revealed that smooth (S), rough (R), and rough small colony variants (SCV-R) were susceptible to the Disc phage and, therefore, regarded as phage-sensitive. Small colony variants (SCV-S) were non-susceptible and, therefore, considered phage-resistant/insensitive variants (Table <a href="#Tab1" class="usa-link">1</a>). The efficiency of the plaquing analysis revealed that the Disc phage had low EOP (≤ 10<sup>6</sup> PFU/mL) against the phage-sensitive variants compared to the wild type (1012 PFU/mL). DAOL showed differences in the plaque morphology. In Fig. <a href="#Fig5" class="usa-link">5</a>B, the rough (R) morphotype has likely lost its capsule to develop resistance to the phage. This loss explains the absence of halo zones around the plaques, as there is no capsule available for degradation. Whereas with the wild type, the phage produced small lytic zone plaques with a distinct halo zone (Fig. <a href="#Fig5" class="usa-link">5</a>A).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372256_12866_2025_4268_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f431/12372256/bd9a67a1e192/12866_2025_4268_Fig5_HTML.jpg" loading="lazy" id="d33e856" height="151" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Properties of the different morphotypes of Kleb_53. <strong>A</strong> Double agar overlay using the wild-type Kleb_53 as host, showing distinct depolymerase activity. <strong>B</strong> Double agar overlay assay with the rough morphotype (R) as host, showing absence of depolymerase activity and a larger lytic zone. <strong>C</strong> Positive string test result for the wild-type <em>Kleb_53</em>, indicating hypermucoviscosity. <strong>D</strong> Capillary tube-based wetness assay assessing the mucosity of the colonies</p></figcaption></figure></section><section id="Sec26"><h3 class="pmc_sec_title">Hypermucoviscous tests</h3>
<p id="Par58">Only smooth colony variants (S) displayed positive results for the string test, similar to wild type (Fig. <a href="#Fig5" class="usa-link">5</a>C), while rough and small colony variants tested negative. The aggregation index was high in the wild, followed by smooth, rough, and small colony variants (smooth and rough) (Fig. <a href="#Fig6" class="usa-link">6</a>A and E). Wetness assay demonstrated that mucosity in the colonies has reduced significantly in the variants compared to the wild/control group (Fig. <a href="#Fig5" class="usa-link">5</a>D, <a href="#Fig6" class="usa-link">6</a> F–6 J).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372256_12866_2025_4268_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f431/12372256/7cc90d988500/12866_2025_4268_Fig6_HTML.jpg" loading="lazy" id="d33e902" height="338" width="669" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Assessment of phenotypic properties in colony morphotype variants (<strong>A</strong>–<strong>E</strong>) Aggregation index of the wild-type and colony morphotype variants from day 1 to day 5. (<strong>F</strong>–<strong>J</strong>) Wetness assay measurements (in mm) of the wild-type and colony morphotypes from day 1 to day 5. Control – wild-type Kleb_53; S – smooth; R – rough; SCV-S – small colony variant-smooth; SCV-R – small colony variant-rough. All experiments were performed in triplicate, and data are presented as mean ± standard deviation (SD). <em>p</em> ≤ 0.05 compared to the control. (ns = not significant, <em>p</em> ≤ 0.01, *<em>p</em> ≤ 0.001)</p></figcaption></figure></section><section id="Sec27"><h3 class="pmc_sec_title">Antibiotic susceptibility testing</h3>
<p id="Par59">The host strain Kleb_53 was resistant to the following antibiotics in the Kirby-Bauer disk diffusion test: meropenem, imipenem, cefotaxime/clavulanic acid, ceftriaxone, ceftazidime, piperacillin-tazobactam, ciprofloxacin, and gentamicin. The variants were also examined likewise and they displayed alterations in their susceptibility pattern. By the end of day 5, both smooth and rough small colony variants became sensitive to meropenem, ceftazidime, cefotaxime/clavulanic acid, ceftriaxone, and piperacillin/tazobactam. The results were tabulated and analyzed (Table <a href="#Tab2" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Representation of the antibiotic susceptibility pattern of all the colony morphotype variants obtained from day 1 to day 5, assessed using the Kirby-Bauer disk diffusion method. SCV- small colony variants</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="2"></th>
<th align="left" colspan="1" rowspan="1">MRP<br>10mcg</th>
<th align="left" colspan="1" rowspan="1">IPM<br>10mcg</th>
<th align="left" colspan="1" rowspan="1">CAZ<br>30mcg</th>
<th align="left" colspan="1" rowspan="1">CEC<br>30/10mcg</th>
<th align="left" colspan="1" rowspan="1">CTR<br>30mcg</th>
<th align="left" colspan="1" rowspan="1">GEN<br>10mcg</th>
<th align="left" colspan="1" rowspan="1">PIT 100/10mcg</th>
<th align="left" colspan="1" rowspan="1">CIP<br>30mcg</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">≥ 23 20–22 ≤19 mm</th>
<th align="left" colspan="1" rowspan="1">≥ 23 20–22 ≤19</th>
<th align="left" colspan="1" rowspan="1">≥ 21 18–20 ≤ 17</th>
<th align="left" colspan="1" rowspan="1">≥ 26 23–25 ≤22</th>
<th align="left" colspan="1" rowspan="1">≥ 23 20–22 ≤19</th>
<th align="left" colspan="1" rowspan="1">≥ 15 13–14 ≤ 12</th>
<th align="left" colspan="1" rowspan="1">≥ 25 21–24 ≤ 20</th>
<th align="left" colspan="1" rowspan="1">≥ 26 22–25 ≤21</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="4" colspan="1">Day 1</td>
<td align="left" colspan="1" rowspan="1">Control</td>
<td align="left" colspan="1" rowspan="1">14</td>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Smooth</td>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Rough</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">22</td>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">21</td>
<td align="left" colspan="1" rowspan="1">11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SCV (Smooth)</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">21</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">22</td>
<td align="left" colspan="1" rowspan="1">12</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">Day 2</td>
<td align="left" colspan="1" rowspan="1">Smooth</td>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">21</td>
<td align="left" colspan="1" rowspan="1">11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Rough</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">26</td>
<td align="left" colspan="1" rowspan="1">11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SCV (Smooth)</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">22</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">22</td>
<td align="left" colspan="1" rowspan="1">12</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">Day 3</td>
<td align="left" colspan="1" rowspan="1">Smooth</td>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">16</td>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">21</td>
<td align="left" colspan="1" rowspan="1">11</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SCV (Smooth)</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">15</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">12</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">Day 4</td>
<td align="left" colspan="1" rowspan="1">
<p>SCV</p>
<p>(Smooth)</p>
</td>
<td align="left" colspan="1" rowspan="1">22</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">22</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">18</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">12</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>SCV</p>
<p>(Rough)</p>
</td>
<td align="left" colspan="1" rowspan="1">22</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">27</td>
<td align="left" colspan="1" rowspan="1">21</td>
<td align="left" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">28</td>
<td align="left" colspan="1" rowspan="1">12</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">Day 5</td>
<td align="left" colspan="1" rowspan="1">
<p>SCV</p>
<p>(Smooth)</p>
</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">12</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>SCV</p>
<p>(Rough)</p>
</td>
<td align="left" colspan="1" rowspan="1">24</td>
<td align="left" colspan="1" rowspan="1">10</td>
<td align="left" colspan="1" rowspan="1">24</td>
<td align="left" colspan="1" rowspan="1">27</td>
<td align="left" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">11</td>
<td align="left" colspan="1" rowspan="1">28</td>
<td align="left" colspan="1" rowspan="1">12</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec28"><h3 class="pmc_sec_title">Biofilm-forming ability</h3>
<p id="Par60">The biofilm-forming ability of the variants was examined using a crystal violet assay. The wild type is a pre-identified biofilm-forming strain; the smooth morphotype (S) variant showed a slight increase in the OD value compared to the control, but remains lower than the other variants. Rough colony (R) morphotypes from days 1 and 2 exhibited strong biofilm formation. Small colony variants of smooth type (SCV-S) did not form biofilms, whereas the day 4 and 5 rough small colony variants (SCV-R) showed a significant increase in biofilm formation (Fig. <a href="#Fig7" class="usa-link">7</a>).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12372256_12866_2025_4268_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f431/12372256/2a081a7435b6/12866_2025_4268_Fig7_HTML.jpg" loading="lazy" id="d33e1256" height="301" width="744" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Assessment of biofilm-forming ability in colony morphotype variants. Biofilm formation was assessed after 24 h using the crystal violet (CV) assay. The stained biofilms were quantified by measuring absorbance at OD 570 nm using a microtiter plate reader. All experiments were performed in triplicate, and the data shown are represented as mean ± SD. <em>p</em> ≤ 0.05 compared to the control. (ns- not significant, **<em>p</em> ≤ 0.01, ***<em>p</em> ≤ 0.001)</p></figcaption></figure></section></section><section id="Sec29"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par61">In this study, the isolation of the lytic phage Disc against the Kleb_53 clinical strain revealed distinct phenotypic adaptations during phage resistance. The Disc phage demonstrated broad lytic activity and high stability, producing plaques with halo zones indicative of depolymerase activity. The phage had an icosahedral head and non-contractile tail, characteristic of the <em>Siphoviridae</em> morphology (outdated taxon). The adsorption time of the phage was 15 min with a latent period of 30 min and exhibited a notable burst size of 354 virions, larger than other<em> K. pneumoniae</em> phages like KP1801 (300 virions) and vB_KpnS_GH-K3 (291 virions) [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>, <a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. It also showed a wide host range, lysing 48<em> Klebsiella</em> clinical strains, which is comparatively higher than other reported <em>Klebsiella</em> phages, UPM2146 phage lysing 5/22 [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>], and vB_KpnP_Bp5 phage lysing 1/36 [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>], further supporting its therapeutic potential.</p>
<p id="Par62">During coevolution, distinct morphotypes such as rough and small colony variants emerged. The rough colony variant demonstrated larger plaque morphology and reduced halo zones, suggestive of capsular polysaccharide loss, a known receptor for phage adsorption. The emergence of heterogeneous colony types is consistent with earlier observations in phage-bacteria interactions [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>, <a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. The dynamic shift in morphotypes likely reflects adaptive strategies, such as niche partitioning or survival trade-offs, possibly driven by differences in metabolic cost, spatial distribution, or environmental niches [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. This phenotypic diversification may facilitate coexistence and long-term survival under phage pressure [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>].</p>
<p id="Par63">The coexistence of sensitive and resistant variants within the same culture suggests a complex adaptive balance. The unculturable colonies observed after day 5 may reflect a phage-pressure induced adaptation resulting in reduced metabolic activity or delayed growth, though not complete eradication, showing a dormant state [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. This phenomenon is described as a viable but non-culturable (VBNC) state, as reported by Li et al. (2014) [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. Phage resistance has also been associated with functional compensatory effects, such as decreased growth rates and impaired cellular functions. For example, Kortright et al. highlighted how resistance can lead to reduced cell division and function [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>], which matches our findings of slower growth in the SCVs. These changes may be either stable mutations or reversible, transient adaptations based on environmental cues [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>].</p>
<p id="Par64">Further, loss of phage receptors such as capsule or LPS has been previously linked to reduced virulence and altered antibiotic interactions [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. In our study, loss of capsule (evidenced by the absence of halo zones) appears to be a resistance mechanism, consistent with prior reports of LPS or capsule mutations contributing to phage resistance in <em>E. coli</em> and <em>Klebsiella</em> strains [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Similarly, research on bacteria coevolved with the lytic phage OMKO1 identified mutations in genes related to flagella, type IV pilus, and lipopolysaccharide biosynthesis—three of which were associated with phenotypic trade-offs in antibiotic resistance during coevolution [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. Resistance was also linked to altered biofilm phenotypes and changes in antibiotic sensitivity. Our phage-exposed variants showed variable biofilm formation, with some producing stronger biofilms, while others—particularly the smooth, small colony morphotypes—exhibited impaired biofilm development. This is consistent with findings in <em>E. faecalis</em> EFap02 phage-resistant mutants that displayed reduced biofilm production [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>].</p>
<p id="Par65">Interestingly, the emergence of phage-resistant morphotypes coincided with increased susceptibility to several antibiotics, including meropenem, ceftazidime, ceftriaxone, and cefotaxime/clavulanic acid. This “phage-induced antibiotic sensitivity” (PIAS) has been previously documented in <em>P. aeruginosa</em> strains, where phage resistance was accompanied by compromised efflux pump systems [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. Also, the evolved phage-resistant strain R118-2 exhibited reduced virulence, diminished biofilm formation, and decreased aminoglycoside resistance compared to its wild-type counterpart, SMA118 [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. In our case, reduced mucosity and possible loss of surface structures likely contributed to both phage resistance and resensitization to antibiotics [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. This highlights the potential for phage therapy not only to target resistant bacteria but also to reverse certain antimicrobial resistance traits.</p>
<p id="Par66">These results suggest that phage therapy can act as a dual-selective pressure, targeting bacteria directly while simultaneously reshaping bacterial phenotypes and susceptibility profiles. The observed morphological heterogeneity and associated growth-related trade-offs suggest that phage resistance often comes at a biological cost, potentially limiting the competitive advantage of resistant variants. Moreover, the dynamic and sometimes reversible nature of these changes highlights the complexity of bacterial adaptation under phage pressure. Phages may push bacteria to become less virulent and more sensitive to antibiotics, making the treatment more effective.</p>
<p id="Par67">This study primarily focused on phenotypic observations. Although we documented clear changes in colony morphology, virulence, biofilm formation, and antibiotic sensitivity, the genetic and molecular underpinnings of these adaptations remain to be elucidated. Future work should include whole-genome sequencing, transcriptomic profiling, and single-cell analyses to dissect the mechanisms driving resistance and phenotypic heterogeneity. Additionally, prolonged incubation or serial passaging may help clarify whether observed fitness costs are due to temporary dormancy, persistent phage pressure, or stable genetic changes. Understanding these dynamics is critical for predicting the long-term outcomes of phage therapy and its integration into clinical settings.</p></section><section id="Sec30"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par68">With renewed interest in phage therapy, understanding bacteria-phage interactions is essential. Phage resistance, an innate bacterial defense, poses a major challenge. This study highlights phenotypic changes in hypermucoviscous, hypervirulent <em>Klebsiella pneumoniae</em> during phage exposure, including shifts in colony morphology, loss of phage susceptibility and mucosity, antibiotic re-sensitization, reduced cell density, unculturability, and variable biofilm formation. These changes may unmask alternative receptors, aiding antibiotic effectiveness and infection clearance. The study underscores the importance of phenotypic insights in improving phage therapy, though the lack of genotypic correlation remains a key limitation.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>CLSI</dt>
<dd><p id="Par2">Clinical and laboratory standards institute</p></dd>
<dt>LB</dt>
<dd><p id="Par3">Luria bertani</p></dd>
<dt>DAOL</dt>
<dd><p id="Par4">Double agar overlay</p></dd>
<dt>PFU</dt>
<dd><p id="Par5">Plaque-forming units</p></dd>
<dt>TEM</dt>
<dd><p id="Par6">Transmission electron microscopy</p></dd>
<dt>MOI</dt>
<dd><p id="Par7">Multiplicity of infection</p></dd>
<dt>OD</dt>
<dd><p id="Par8">Optical Density</p></dd>
<dt>PSV</dt>
<dd><p id="Par9">Phage-sensitive variant</p></dd>
<dt>PRV</dt>
<dd><p id="Par10">Phage-resistant variants</p></dd>
<dt>EOP</dt>
<dd><p id="Par11">Efficiency of plaquing</p></dd>
<dt>MRP</dt>
<dd><p id="Par12">Meropenem</p></dd>
<dt>IPM</dt>
<dd><p id="Par13">Imipenem</p></dd>
<dt>CEC</dt>
<dd><p id="Par14">Cefotaxime/ clavulanic acid</p></dd>
<dt>CTR</dt>
<dd><p id="Par15">Ceftriaxone</p></dd>
<dt>CAZ</dt>
<dd><p id="Par16">Ceftazidime</p></dd>
<dt>PIT</dt>
<dd><p id="Par17">Piperacillin-tazobactam</p></dd>
<dt>CIP</dt>
<dd><p id="Par18">Ciprofloxacin</p></dd>
<dt>GEN</dt>
<dd><p id="Par19">Gentamicin</p></dd>
<dt>CV</dt>
<dd><p id="Par20">Crystal violet</p></dd>
<dt>SD</dt>
<dd><p id="Par21">Standard deviation</p></dd>
<dt>MIC</dt>
<dd><p id="Par22">Minimum inhibitory concentration</p></dd>
<dt>Hv</dt>
<dd><p id="Par23">Hypervirulent</p></dd>
<dt>Hm</dt>
<dd><p id="Par24">Hypermucoviscous</p></dd>
<dt>Kp</dt>
<dd><p id="Par25">
<em>Klebsiella pneumoniae</em>
</p></dd>
<dt>R</dt>
<dd><p id="Par26">Rough, dry, irregular colony</p></dd>
<dt>S</dt>
<dd><p id="Par27">Smooth circular mucoid colony</p></dd>
<dt>SCV-S </dt>
<dd><p id="Par28">Smooth small colony variants</p></dd>
<dt>SCV-R</dt>
<dd><p id="Par29">Rough small colony variants</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors’ contributions</h2>
<p>Conceptualization: RJ and RN.; methodology: RJ; formal analysis: RJ and RN.; investigation: RJ and RN.; resources: RN; data curation: RJ and RN.; writing—original draft preparation: RJ and RN; writing—review and editing: RJ and RN.; supervision: RN.; project administration: RN.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Open access funding provided by Vellore Institute of Technology. The author(s) are thankful to the ICMR Govt. of India, New Delhi, EM/dev/CAR/ICMRCAREP-2023-0000138/2023.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The sequences are submitted to the NCBI database (https://www.ncbi.nlm.nih.gov/), and the accession numbers are <a href="https://www.ncbi.nlm.nih.gov/nuccore/PV029729" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV029729</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PP994840" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP994840</a>, and <a href="https://www.ncbi.nlm.nih.gov/nuccore/PP994850" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP994850</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par69">Ethical approval was from the Institutional Ethical Committee for Studies on Human Subjects (IECH), ref. no. VIT/IECH/2024/15 IECH/15 June 2024/5. The current study was implemented in coordination with the guidelines of the Declaration of Helsinki. The need for consent to participate was waived by the Institutional Ethical Committee for Studies on Human Subjects (IECH), Vellore Institute of Technology, Vellore, India. ref. no. VIT/IECH/004/Jan28.2017.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par70">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par71">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Koskella B, Brockhurst MA. Bacteria–phage coevolution as a driver of ecological and evolutionary processes in microbial communities. FEMS Microbiol Rev. 2014;38:916.
</cite> [<a href="https://doi.org/10.1111/1574-6976.12072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4257071/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24617569/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koskella%20B,%20Brockhurst%20MA.%20Bacteria%E2%80%93phage%20coevolution%20as%20a%20driver%20of%20ecological%20and%20evolutionary%20processes%20in%20microbial%20communities.%20FEMS%20Microbiol%20Rev.%202014;38:916." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Gordillo Altamirano FL, Barr JJ. Phage therapy in the postantibiotic era. Clin Microbiol Rev. 2019;32:e00066–18.
</cite> [<a href="https://doi.org/10.1128/CMR.00066-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6431132/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30651225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gordillo%20Altamirano%20FL,%20Barr%20JJ.%20Phage%20therapy%20in%20the%20postantibiotic%20era.%20Clin%20Microbiol%20Rev.%202019;32:e00066%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Egido JE, Costa AR, Aparicio-Maldonado C, Haas PJ, Brouns SJJ. Mechanisms and clinical importance of bacteriophage resistance. FEMS Microbiol Rev. 2022. 10.1093/femsre/fuab048.
</cite> [<a href="https://doi.org/10.1093/femsre/fuab048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8829019/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34558600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Egido%20JE,%20Costa%20AR,%20Aparicio-Maldonado%20C,%20Haas%20PJ,%20Brouns%20SJJ.%20Mechanisms%20and%20clinical%20importance%20of%20bacteriophage%20resistance.%20FEMS%20Microbiol%20Rev.%202022.%2010.1093/femsre/fuab048." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Yu Y, Wang M, Ju L, Li M, Zhao M, Deng H, et al. Phage-mediated virulence loss and antimicrobial susceptibility in carbapenem-resistant Klebsiella pneumoniae. mBio. 2024. 10.1128/MBIO.02957-24.
</cite> [<a href="https://doi.org/10.1128/mbio.02957-24" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11796411/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39714187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20Y,%20Wang%20M,%20Ju%20L,%20Li%20M,%20Zhao%20M,%20Deng%20H,%20et%20al.%20Phage-mediated%20virulence%20loss%20and%20antimicrobial%20susceptibility%20in%20carbapenem-resistant%20Klebsiella%20pneumoniae.%20mBio.%202024.%2010.1128/MBIO.02957-24." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Koskella B, Brockhurst MA. Bacteria-phage coevolution as a driver of ecological and evolutionary processes in microbial communities. FEMS Microbiol Rev. 2014;38:916–31.
</cite> [<a href="https://doi.org/10.1111/1574-6976.12072" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4257071/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24617569/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koskella%20B,%20Brockhurst%20MA.%20Bacteria-phage%20coevolution%20as%20a%20driver%20of%20ecological%20and%20evolutionary%20processes%20in%20microbial%20communities.%20FEMS%20Microbiol%20Rev.%202014;38:916%E2%80%9331." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Sumrall ET, Shen Y, Keller AP, Rismondo J, Pavlou M, Eugster MR, et al. Phage resistance at the cost of virulence: <em>Listeria monocytogenes</em> serovar 4b requires galactosylated teichoic acids for InlB-mediated invasion. PLoS Pathog. 2019;15:e1008032.
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1008032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6779246/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31589660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sumrall%20ET,%20Shen%20Y,%20Keller%20AP,%20Rismondo%20J,%20Pavlou%20M,%20Eugster%20MR,%20et%20al.%20Phage%20resistance%20at%20the%20cost%20of%20virulence:%20Listeria%20monocytogenes%20serovar%204b%20requires%20galactosylated%20teichoic%20acids%20for%20InlB-mediated%20invasion.%20PLoS%20Pathog.%202019;15:e1008032." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Han P, Lin W, Fan H, Tong Y. Characterization of phage evolution and phage resistance in drug-resistant <em>Stenotrophomonas maltophilia</em>. J Virol. 2024;98:1–15.</cite> [<a href="https://doi.org/10.1128/jvi.01249-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10878236/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38189285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Han%20P,%20Lin%20W,%20Fan%20H,%20Tong%20Y.%20Characterization%20of%20phage%20evolution%20and%20phage%20resistance%20in%20drug-resistant%20Stenotrophomonas%20maltophilia.%20J%20Virol.%202024;98:1%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Gao Z, Feng Y. Bacteriophage strategies for overcoming host antiviral immunity. Front Microbiol. 2023;14:1211793.
</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1211793" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10286901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37362940/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao%20Z,%20Feng%20Y.%20Bacteriophage%20strategies%20for%20overcoming%20host%20antiviral%20immunity.%20Front%20Microbiol.%202023;14:1211793." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Kou X, Yang X, Zheng R. Challenges and opportunities of phage therapy for <em>Klebsiella pneumoniae</em> infections. Appl Environ Microbiol. 2024. 10.1128/aem.01353-24.
</cite> [<a href="https://doi.org/10.1128/aem.01353-24" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11497816/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39345202/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kou%20X,%20Yang%20X,%20Zheng%20R.%20Challenges%20and%20opportunities%20of%20phage%20therapy%20for%20Klebsiella%20pneumoniae%20infections.%20Appl%20Environ%20Microbiol.%202024.%2010.1128/aem.01353-24." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Tang Y, Du P, Du C, Yang P, Shen N, Russo TA, et al. Genomically defined hypervirulent Klebsiella pneumoniae contributed to early-onset increased mortality. Nat Commun 2025. 2025;16(1):16:1–11.</cite> [<a href="https://doi.org/10.1038/s41467-025-57379-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11873152/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40025046/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang%20Y,%20Du%20P,%20Du%20C,%20Yang%20P,%20Shen%20N,%20Russo%20TA,%20et%20al.%20Genomically%20defined%20hypervirulent%20Klebsiella%20pneumoniae%20contributed%20to%20early-onset%20increased%20mortality.%20Nat%20Commun%202025.%202025;16(1):16:1%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Namikawa H, Oinuma KI, Yamada K, Kaneko Y, Kakeya H, Shuto T. Predictors of hypervirulent Klebsiella pneumoniae infections: a systematic review and meta-analysis. J Hosp Infect. 2023;134:153–60.
</cite> [<a href="https://doi.org/10.1016/j.jhin.2023.02.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36813165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Namikawa%20H,%20Oinuma%20KI,%20Yamada%20K,%20Kaneko%20Y,%20Kakeya%20H,%20Shuto%20T.%20Predictors%20of%20hypervirulent%20Klebsiella%20pneumoniae%20infections:%20a%20systematic%20review%20and%20meta-analysis.%20J%20Hosp%20Infect.%202023;134:153%E2%80%9360." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Sánchez-López J, García-Caballero A, Navarro-San Francisco C, Quereda C, Ruiz-Garbajosa P, Navas E, Dronda F, Morosini MI, Cantón R, Diez-Aguilar M. Hypermucoviscous Klebsiella pneumoniae: a challenge in community acquired infection. IDCases. 2019 Jan 1;17:e00547.</cite> [<a href="https://doi.org/10.1016/j.idcr.2019.e00547" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6514362/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31193033/" class="usa-link">PubMed</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Tian T, Han H, Guan ZH, Zhang K, Huang X, Wang W, et al. A systematic review of hypervirulent Klebsiella pneumoniae research: bibliometric and topic modeling perspectives. Front Med (Lausanne). 2025;12:1545678.
</cite> [<a href="https://doi.org/10.3389/fmed.2025.1545678" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12021593/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40291031/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tian%20T,%20Han%20H,%20Guan%20ZH,%20Zhang%20K,%20Huang%20X,%20Wang%20W,%20et%20al.%20A%20systematic%20review%20of%20hypervirulent%20Klebsiella%20pneumoniae%20research:%20bibliometric%20and%20topic%20modeling%20perspectives.%20Front%20Med%20(Lausanne).%202025;12:1545678." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Townsend EM, Kelly L, Gannon L, Muscatt G, Dunstan R, Michniewski S. Isolation and characterization of <em>Klebsiella</em> phages for phage therapy. PHAGE. 2021;2:26.
</cite> [<a href="https://doi.org/10.1089/phage.2020.0046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8006926/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33796863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Townsend%20EM,%20Kelly%20L,%20Gannon%20L,%20Muscatt%20G,%20Dunstan%20R,%20Michniewski%20S.%20Isolation%20and%20characterization%20of%20Klebsiella%20phages%20for%20phage%20therapy.%20PHAGE.%202021;2:26." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Chen H, Liu H, Gong Y, Dunstan RA, Ma Z, Zhou C, et al. A <em>klebsiella</em>-phage cocktail to broaden the host range and delay bacteriophage resistance both <em>in vitro</em> and <em>in vivo</em>. NPJ Biofilms Microbiomes. 2024;10:1–17.
</cite> [<a href="https://doi.org/10.1038/s41522-024-00603-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11564825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39543151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20H,%20Liu%20H,%20Gong%20Y,%20Dunstan%20RA,%20Ma%20Z,%20Zhou%20C,%20et%20al.%20A%20klebsiella-phage%20cocktail%20to%20broaden%20the%20host%20range%20and%20delay%20bacteriophage%20resistance%20both%20in%20vitro%20and%20in%20vivo.%20NPJ%20Biofilms%20Microbiomes.%202024;10:1%E2%80%9317." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Tang M, Huang Z, Zhang X, Kong J, Zhou B, Han Y, et al. Phage resistance formation and fitness costs of hypervirulent <em>Klebsiella pneumoniae</em> mediated by K2 capsule-specific phage and the corresponding mechanisms. Front Microbiol. 2023;14:1156292.
</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1156292" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10394836/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37538841/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang%20M,%20Huang%20Z,%20Zhang%20X,%20Kong%20J,%20Zhou%20B,%20Han%20Y,%20et%20al.%20Phage%20resistance%20formation%20and%20fitness%20costs%20of%20hypervirulent%20Klebsiella%20pneumoniae%20mediated%20by%20K2%20capsule-specific%20phage%20and%20the%20corresponding%20mechanisms.%20Front%20Microbiol.%202023;14:1156292." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Manohar P, Tamhankar AJ, Lundborg CS, Nachimuthu R. Therapeutic characterization and efficacy of bacteriophage cocktails infecting <em>Escherichia coli</em>, <em>Klebsiella pneumoniae</em>, and <em>Enterobacter</em> species. Front Microbiol. 2019;10(MAR):431926.</cite> [<a href="https://doi.org/10.3389/fmicb.2019.00574" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6437105/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30949158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manohar%20P,%20Tamhankar%20AJ,%20Lundborg%20CS,%20Nachimuthu%20R.%20Therapeutic%20characterization%20and%20efficacy%20of%20bacteriophage%20cocktails%20infecting%20Escherichia%20coli,%20Klebsiella%20pneumoniae,%20and%20Enterobacter%20species.%20Front%20Microbiol.%202019;10(MAR):431926." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Osama DM, Zaki BM, Khalaf WS, Mohamed MYA, Tawfick MM, Amin HM. Occurrence and molecular study of hypermucoviscous/hypervirulence trait in gut commensal <em>K. pneumoniae</em> from healthy subjects. Microorganisms. 2023;11:704.
</cite> [<a href="https://doi.org/10.3390/microorganisms11030704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10059952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36985277/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Osama%20DM,%20Zaki%20BM,%20Khalaf%20WS,%20Mohamed%20MYA,%20Tawfick%20MM,%20Amin%20HM.%20Occurrence%20and%20molecular%20study%20of%20hypermucoviscous/hypervirulence%20trait%20in%20gut%20commensal%20K.%20pneumoniae%20from%20healthy%20subjects.%20Microorganisms.%202023;11:704." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Loganathan A, Nachimuthu R. Antibiotic resistance, biofilm forming ability, and clonal profiling of clinical isolates of <em>Staphylococcus aureus</em> from Southern and Northeastern India. Asian Biomed (Res Rev News). 2022;16:191.
</cite> [<a href="https://doi.org/10.2478/abm-2022-0023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10321179/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37551171/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Loganathan%20A,%20Nachimuthu%20R.%20Antibiotic%20resistance,%20biofilm%20forming%20ability,%20and%20clonal%20profiling%20of%20clinical%20isolates%20of%20Staphylococcus%20aureus%20from%20Southern%20and%20Northeastern%20India.%20Asian%20Biomed%20(Res%20Rev%20News).%202022;16:191." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>CLSI. M100 Performance Standards for Antimicrobial Susceptibility Testing A CLSI supplement for global application. 32nd Edition.</cite>
</li>
<li id="CR21">
<span class="label">21.</span><cite>Manohar P, Leptihn S, Lopes BS, Nachimuthu R. Dissemination of carbapenem resistance and plasmids encoding carbapenemases in Gram-negative bacteria isolated in India. JAC-Antimicrobial Resistance. 2021. 10.1093/jacamr/dlab015.
</cite> [<a href="https://doi.org/10.1093/jacamr/dlab015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8210035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34223092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manohar%20P,%20Leptihn%20S,%20Lopes%20BS,%20Nachimuthu%20R.%20Dissemination%20of%20carbapenem%20resistance%20and%20plasmids%20encoding%20carbapenemases%20in%20Gram-negative%20bacteria%20isolated%20in%20India.%20JAC-Antimicrobial%20Resistance.%202021.%2010.1093/jacamr/dlab015." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Lin JC, Koh TH, Lee N, Fung CP, Chang FY, Tsai YK, et al. Genotypes and virulence in serotype K2 <em>Klebsiella pneumoniae</em> from liver abscess and non-infectious carriers in Hong kong, Singapore and Taiwan. Gut Pathog. 2014;6:1–7.
</cite> [<a href="https://doi.org/10.1186/1757-4749-6-21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4076766/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24987462/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin%20JC,%20Koh%20TH,%20Lee%20N,%20Fung%20CP,%20Chang%20FY,%20Tsai%20YK,%20et%20al.%20Genotypes%20and%20virulence%20in%20serotype%20K2%20Klebsiella%20pneumoniae%20from%20liver%20abscess%20and%20non-infectious%20carriers%20in%20Hong%20kong,%20Singapore%20and%20Taiwan.%20Gut%20Pathog.%202014;6:1%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Brisse S, PV FC, I-JS TR, GP. DL, &amp;. Virulent Clones of Klebsiella pneumoniae: Identification and Evolutionary Scenario Based on Genomic and Phenotypic Characterization| PLOS ONE. 2009. <a href="https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0004982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0004982</a>. Accessed 7 Jan 2025.</cite> [<a href="https://doi.org/10.1371/journal.pone.0004982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2656620/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19319196/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Juliet R, Loganathan A, Neeravi A, Bakthavatchalam YD, Veeraraghavan B, Manohar P, et al. Characterization of <em>Salmonella</em> phage of the genus <em>kayfunavirus</em> isolated from sewage infecting clinical strains of <em>Salmonella enterica</em>. Front Microbiol. 2024;15:1391777.
</cite> [<a href="https://doi.org/10.3389/fmicb.2024.1391777" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11180730/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38887719/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Juliet%20R,%20Loganathan%20A,%20Neeravi%20A,%20Bakthavatchalam%20YD,%20Veeraraghavan%20B,%20Manohar%20P,%20et%20al.%20Characterization%20of%20Salmonella%20phage%20of%20the%20genus%20kayfunavirus%20isolated%20from%20sewage%20infecting%20clinical%20strains%20of%20Salmonella%20enterica.%20Front%20Microbiol.%202024;15:1391777." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Loganathan A, Nachimuthu R. Characterization of vB_Sau_S90, and vB_Sau_S165, a broad host range phages, and multiple phage doses as a potential therapeutic strategy to eliminate transient phage resistant mutants in Staphylococcus aureus. 2021. 10.21203/rs.3.rs-1070956/v1</cite>
</li>
<li id="CR26">
<span class="label">26.</span><cite>Bolivar-Jacobo NA, Reyes-Villagrana RA, Rentería-Monterrubio AL, Sánchez-Vega R, Santellano-Estrada E, Tirado-Gallegos JM, et al. Culture Age, Growth Medium, Ultrasound Amplitude, and Time of Exposure Influence the Kinetic Growth of Lactobacillus acidophilus. Fermentation. 2023;9:63.</cite> [<a href="https://scholar.google.com/scholar_lookup?Bolivar-Jacobo%20NA,%20Reyes-Villagrana%20RA,%20Renter%C3%ADa-Monterrubio%20AL,%20S%C3%A1nchez-Vega%20R,%20Santellano-Estrada%20E,%20Tirado-Gallegos%20JM,%20et%20al.%20Culture%20Age,%20Growth%20Medium,%20Ultrasound%20Amplitude,%20and%20Time%20of%20Exposure%20Influence%20the%20Kinetic%20Growth%20of%20Lactobacillus%20acidophilus.%20Fermentation.%202023;9:63." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Suarez CA, Franceschelli JJ, Tasselli SE, Morbidoni HR. Weirdo19ES is a novel singleton mycobacteriophage that selects for glycolipid deficient phage-resistant <em>M. smegmatis</em> mutants. PLoS One. 2020. 10.1371/journal.pone.0231881.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0231881" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7194413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32357186/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Suarez%20CA,%20Franceschelli%20JJ,%20Tasselli%20SE,%20Morbidoni%20HR.%20Weirdo19ES%20is%20a%20novel%20singleton%20mycobacteriophage%20that%20selects%20for%20glycolipid%20deficient%20phage-resistant%20M.%20smegmatis%20mutants.%20PLoS%20One.%202020.%2010.1371/journal.pone.0231881." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Inoue T, Shingaki R, Fukui K. Inhibition of swarming motility of <em>Pseudomonas aeruginosa</em> by branched-chain fatty acids. FEMS Microbiol Lett. 2008;281:81–6.
</cite> [<a href="https://doi.org/10.1111/j.1574-6968.2008.01089.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18318842/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Inoue%20T,%20Shingaki%20R,%20Fukui%20K.%20Inhibition%20of%20swarming%20motility%20of%20Pseudomonas%20aeruginosa%20by%20branched-chain%20fatty%20acids.%20FEMS%20Microbiol%20Lett.%202008;281:81%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Manohar P, Leptihn S, Lopes BS, Nachimuthu R. Dissemination of carbapenem resistance and plasmids encoding carbapenemases in Gram-negative bacteria isolated in India. JAC-Antimicrobial Resistance. 2021. 10.1093/jacamr/dlab015.
</cite> [<a href="https://doi.org/10.1093/jacamr/dlab015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8210035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34223092/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manohar%20P,%20Leptihn%20S,%20Lopes%20BS,%20Nachimuthu%20R.%20Dissemination%20of%20carbapenem%20resistance%20and%20plasmids%20encoding%20carbapenemases%20in%20Gram-negative%20bacteria%20isolated%20in%20India.%20JAC-Antimicrobial%20Resistance.%202021.%2010.1093/jacamr/dlab015." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Manohar P, Loh B, Nachimuthu R, Leptihn S. Phage-antibiotic combinations to control Pseudomonas aeruginosa–Candida two-species biofilms. Scientific Reports. 2024;14:1–11.
</cite> [<a href="https://doi.org/10.1038/s41598-024-59444-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11039464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38653744/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Manohar%20P,%20Loh%20B,%20Nachimuthu%20R,%20Leptihn%20S.%20Phage-antibiotic%20combinations%20to%20control%20Pseudomonas%20aeruginosa%E2%80%93Candida%20two-species%20biofilms.%20Scientific%20Reports.%202024;14:1%E2%80%9311." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Wintachai P, Naknaen A, Thammaphet J, Pomwised R, Phaonakrop N, Roytrakul S, et al. Characterization of extended-spectrum-β-lactamase producing <em>Klebsiella pneumoniae</em> phage KP1801 and evaluation of therapeutic efficacy <em>in vitro</em> and <em>in vivo</em>. Sci Rep. 2020;10:11803.
</cite> [<a href="https://doi.org/10.1038/s41598-020-68702-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7367294/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32678251/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wintachai%20P,%20Naknaen%20A,%20Thammaphet%20J,%20Pomwised%20R,%20Phaonakrop%20N,%20Roytrakul%20S,%20et%20al.%20Characterization%20of%20extended-spectrum-%CE%B2-lactamase%20producing%20Klebsiella%20pneumoniae%20phage%20KP1801%20and%20evaluation%20of%20therapeutic%20efficacy%20in%20vitro%20and%20in%20vivo.%20Sci%20Rep.%202020;10:11803." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Cai R, Wang Z, Wang G, Zhang H, Cheng M, Guo Z, et al. Biological properties and genomics analysis of vB_KpnS_GH-K3, a <em>Klebsiella</em> phage with a putative depolymerase-like protein. Virus Genes. 2019;55:696–706.
</cite> [<a href="https://doi.org/10.1007/s11262-019-01681-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31254238/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cai%20R,%20Wang%20Z,%20Wang%20G,%20Zhang%20H,%20Cheng%20M,%20Guo%20Z,%20et%20al.%20Biological%20properties%20and%20genomics%20analysis%20of%20vB_KpnS_GH-K3,%20a%20Klebsiella%20phage%20with%20a%20putative%20depolymerase-like%20protein.%20Virus%20Genes.%202019;55:696%E2%80%93706." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Assafiri O, Song AAL, Tan GH, Hanish I, Hashim AM, Yusoff K. <em>Klebsiella</em> virus UPM2146 lyses multiple drug-resistant <em>Klebsiella pneumoniae</em> in vitro and in vivo. PLoS One. 2021;16:e0245354.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0245354" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7794032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33418559/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Assafiri%20O,%20Song%20AAL,%20Tan%20GH,%20Hanish%20I,%20Hashim%20AM,%20Yusoff%20K.%20Klebsiella%20virus%20UPM2146%20lyses%20multiple%20drug-resistant%20Klebsiella%20pneumoniae%20in%20vitro%20and%20in%20vivo.%20PLoS%20One.%202021;16:e0245354." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Zhang C, Yuan J, Guo C, Ge C, Wang X, Wei D, et al. Identification and complete genome of lytic Kp34likevirus phage vB_KpnP_Bp5 and therapeutic potency in the treatment of lethal <em>Klebsiella pneumoniae</em> infections in mice. Virus Res. 2021;297: 198348.
</cite> [<a href="https://doi.org/10.1016/j.virusres.2021.198348" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33631221/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20C,%20Yuan%20J,%20Guo%20C,%20Ge%20C,%20Wang%20X,%20Wei%20D,%20et%20al.%20Identification%20and%20complete%20genome%20of%20lytic%20Kp34likevirus%20phage%20vB_KpnP_Bp5%20and%20therapeutic%20potency%20in%20the%20treatment%20of%20lethal%20Klebsiella%20pneumoniae%20infections%20in%20mice.%20Virus%20Res.%202021;297:%20198348." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Cai R, Wu M, Zhang H, Zhang Y, Cheng M, Guo Z, et al. A smooth-type, phage-resistant <em>Klebsiella pneumoniae</em> mutant strain reveals that OmpC is indispensable for infection by phage GH-K3. Appl Environ Microbiol. 2018. 10.1128/AEM.01585-18.
</cite> [<a href="https://doi.org/10.1128/AEM.01585-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6193389/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30171001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cai%20R,%20Wu%20M,%20Zhang%20H,%20Zhang%20Y,%20Cheng%20M,%20Guo%20Z,%20et%20al.%20A%20smooth-type,%20phage-resistant%20Klebsiella%20pneumoniae%20mutant%20strain%20reveals%20that%20OmpC%20is%20indispensable%20for%20infection%20by%20phage%20GH-K3.%20Appl%20Environ%20Microbiol.%202018.%2010.1128/AEM.01585-18." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Fang Q, Feng Y, McNally A, Zong Z. Characterization of phage resistance and phages capable of intestinal decolonization of carbapenem-resistant Klebsiella pneumoniae in mice. Communications Biology. 2022;5:1–14.
</cite> [<a href="https://doi.org/10.1038/s42003-022-03001-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8758719/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35027665/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fang%20Q,%20Feng%20Y,%20McNally%20A,%20Zong%20Z.%20Characterization%20of%20phage%20resistance%20and%20phages%20capable%20of%20intestinal%20decolonization%20of%20carbapenem-resistant%20Klebsiella%20pneumoniae%20in%20mice.%20Communications%20Biology.%202022;5:1%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Laanto E, Hoikkala V, Ravantti J, Sundberg LR. Long-term genomic coevolution of host-parasite interaction in the natural environment. Nat Commun. 2017;8:1–8.
</cite> [<a href="https://doi.org/10.1038/s41467-017-00158-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5524643/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28740072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laanto%20E,%20Hoikkala%20V,%20Ravantti%20J,%20Sundberg%20LR.%20Long-term%20genomic%20coevolution%20of%20host-parasite%20interaction%20in%20the%20natural%20environment.%20Nat%20Commun.%202017;8:1%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Scanlan PD, Buckling A. Co-evolution with lytic phage selects for the mucoid phenotype of Pseudomonas fluorescens SBW25. ISME J. 2012;6:1148–58.
</cite> [<a href="https://doi.org/10.1038/ismej.2011.174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3358020/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22189495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Scanlan%20PD,%20Buckling%20A.%20Co-evolution%20with%20lytic%20phage%20selects%20for%20the%20mucoid%20phenotype%20of%20Pseudomonas%20fluorescens%20SBW25.%20ISME%20J.%202012;6:1148%E2%80%9358." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Fernández-García L, Kirigo J, Huelgas-Méndez D, Benedik MJ, Tomás M, García-Contreras R, et al. Phages produce persisters. Microb Biotechnol. 2024;17:1–10.</cite> [<a href="https://doi.org/10.1111/1751-7915.14543" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11297538/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39096350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fern%C3%A1ndez-Garc%C3%ADa%20L,%20Kirigo%20J,%20Huelgas-M%C3%A9ndez%20D,%20Benedik%20MJ,%20Tom%C3%A1s%20M,%20Garc%C3%ADa-Contreras%20R,%20et%20al.%20Phages%20produce%20persisters.%20Microb%20Biotechnol.%202024;17:1%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Li L, Mendis N, Trigui H, Oliver JD, Faucher SP. The importance of the viable but non-culturable state in human bacterial pathogens. Front Microbiol. 2014;5 JUN:1–1.
</cite> [<a href="https://doi.org/10.3389/fmicb.2014.00258" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4040921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24917854/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20L,%20Mendis%20N,%20Trigui%20H,%20Oliver%20JD,%20Faucher%20SP.%20The%20importance%20of%20the%20viable%20but%20non-culturable%20state%20in%20human%20bacterial%20pathogens.%20Front%20Microbiol.%202014;5%20JUN:1%E2%80%931." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Kortright KE, Chan BK, Koff JL, Turner PE. Phage therapy: a renewed approach to combat antibiotic-resistant bacteria. Cell Host Microbe. 2019;25:219–32.
</cite> [<a href="https://doi.org/10.1016/j.chom.2019.01.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30763536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kortright%20KE,%20Chan%20BK,%20Koff%20JL,%20Turner%20PE.%20Phage%20therapy:%20a%20renewed%20approach%20to%20combat%20antibiotic-resistant%20bacteria.%20Cell%20Host%20Microbe.%202019;25:219%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Jurado A, Fernández L, Rodríguez A, García P. Understanding the mechanisms that drive phage resistance in Staphylococci to prevent phage therapy failure. Viruses. 2022;14:1–15.</cite> [<a href="https://doi.org/10.3390/v14051061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9146914/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35632803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jurado%20A,%20Fern%C3%A1ndez%20L,%20Rodr%C3%ADguez%20A,%20Garc%C3%ADa%20P.%20Understanding%20the%20mechanisms%20that%20drive%20phage%20resistance%20in%20Staphylococci%20to%20prevent%20phage%20therapy%20failure.%20Viruses.%202022;14:1%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Gaborieau B, Debarbieux L. The role of the animal host in the management of bacteriophage resistance during phage therapy. Curr Opin Virol. 2023;58: 101290.
</cite> [<a href="https://doi.org/10.1016/j.coviro.2022.101290" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36512896/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gaborieau%20B,%20Debarbieux%20L.%20The%20role%20of%20the%20animal%20host%20in%20the%20management%20of%20bacteriophage%20resistance%20during%20phage%20therapy.%20Curr%20Opin%20Virol.%202023;58:%20101290." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Yu J, Zhang H, Ju Z, Huang J, Lin C, Wu J, et al. Increased mutations in lipopolysaccharide biosynthetic genes cause time-dependent development of phage resistance in <em>Salmonella</em>. Antimicrob Agents Chemother. 2024. 10.1128/aac.00594-23.
</cite> [<a href="https://doi.org/10.1128/aac.00594-23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10848759/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38193669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20J,%20Zhang%20H,%20Ju%20Z,%20Huang%20J,%20Lin%20C,%20Wu%20J,%20et%20al.%20Increased%20mutations%20in%20lipopolysaccharide%20biosynthetic%20genes%20cause%20time-dependent%20development%20of%20phage%20resistance%20in%20Salmonella.%20Antimicrob%20Agents%20Chemother.%202024.%2010.1128/aac.00594-23." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Kortright KE, Chan BK, Evans BR, Turner PE. Arms race and fluctuating selection dynamics in <em>Pseudomonas aeruginosa</em> bacteria coevolving with phage OMKO1. J Evol Biol. 2022;35:1475–87.
</cite> [<a href="https://doi.org/10.1111/jeb.14095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36168737/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kortright%20KE,%20Chan%20BK,%20Evans%20BR,%20Turner%20PE.%20Arms%20race%20and%20fluctuating%20selection%20dynamics%20in%20Pseudomonas%20aeruginosa%20bacteria%20coevolving%20with%20phage%20OMKO1.%20J%20Evol%20Biol.%202022;35:1475%E2%80%9387." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Liu J, Zhu Y, Li Y, Lu Y, Xiong K, Zhong Q, Wang J. Bacteriophage-resistant mutant of Enterococcus faecalis is impaired in biofilm formation. Frontiers in Microbiology. 2022 Jun 9;13:913023.</cite> [<a href="https://doi.org/10.3389/fmicb.2022.913023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9218719/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35756031/" class="usa-link">PubMed</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Koderi Valappil S, Shetty P, Deim Z, Terhes G, Urbán E, Váczi S, et al. Survival comes at a cost: a coevolution of phage and its host leads to phage resistance and antibiotic sensitivity of <em>Pseudomonas aeruginosa</em> multidrug resistant strains. Front Microbiol. 2021;12:1–17.</cite> [<a href="https://doi.org/10.3389/fmicb.2021.783722" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8678094/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34925289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koderi%20Valappil%20S,%20Shetty%20P,%20Deim%20Z,%20Terhes%20G,%20Urb%C3%A1n%20E,%20V%C3%A1czi%20S,%20et%20al.%20Survival%20comes%20at%20a%20cost:%20a%20coevolution%20of%20phage%20and%20its%20host%20leads%20to%20phage%20resistance%20and%20antibiotic%20sensitivity%20of%20Pseudomonas%20aeruginosa%20multidrug%20resistant%20strains.%20Front%20Microbiol.%202021;12:1%E2%80%9317." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Yin X, Fang Q, Zong Z. Interruption of capsular polysaccharide biosynthesis gene WbaZ by insertion sequence IS 903B mediates resistance to a lytic phage against ST11 K64 carbapenem-resistant <em>Klebsiella pneumoniae</em>. mSphere. 2022. 10.1128/msphere.00518-22.
</cite> [<a href="https://doi.org/10.1128/msphere.00518-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9769513/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36377881/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yin%20X,%20Fang%20Q,%20Zong%20Z.%20Interruption%20of%20capsular%20polysaccharide%20biosynthesis%20gene%20WbaZ%20by%20insertion%20sequence%20IS%20903B%20mediates%20resistance%20to%20a%20lytic%20phage%20against%20ST11%20K64%20carbapenem-resistant%20Klebsiella%20pneumoniae.%20mSphere.%202022.%2010.1128/msphere.00518-22." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The sequences are submitted to the NCBI database (https://www.ncbi.nlm.nih.gov/), and the accession numbers are <a href="https://www.ncbi.nlm.nih.gov/nuccore/PV029729" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PV029729</a>, <a href="https://www.ncbi.nlm.nih.gov/nuccore/PP994840" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP994840</a>, and <a href="https://www.ncbi.nlm.nih.gov/nuccore/PP994850" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP994850</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Microbiology are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12866-025-04268-x"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12866_2025_Article_4268.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12372256/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12372256/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12372256%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12372256/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12372256/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12372256/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841996/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12372256/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841996/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12372256/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12372256/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="iTKvbL8vPbrBOj9M2wFB1tehxQf1sDJ8ymi9SOLx0aNrWaapdgfbTNoZXOBDDaBt">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
