
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Extremely acidophilic filamentous fungi are more prevalent in diverse ecosystems than previously documented - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4BBD08AF434D305BBD00006DBE2AC.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="scirep">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365195/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Scientific Reports">
<meta name="citation_title" content="Extremely acidophilic filamentous fungi are more prevalent in diverse ecosystems than previously documented">
<meta name="citation_author" content="Nguyen Thanh Thuy">
<meta name="citation_author_institution" content="Food Industries Research Institute, 301 Nguyen Trai, Thanh Xuan, Hanoi, Vietnam">
<meta name="citation_author" content="Tom Coleman">
<meta name="citation_author_institution" content="Division of Industrial Biotechnology, Department of Life Sciences, Chalmers University of Technology, Gothenburg, Sweden">
<meta name="citation_author" content="Meera Christopher">
<meta name="citation_author_institution" content="Division of Industrial Biotechnology, Department of Life Sciences, Chalmers University of Technology, Gothenburg, Sweden">
<meta name="citation_author_institution" content="Wallenberg Wood Science Center, Chalmers University of Technology, Gothenburg, Sweden">
<meta name="citation_author" content="Nguyen Bao Chau">
<meta name="citation_author_institution" content="Food Industries Research Institute, 301 Nguyen Trai, Thanh Xuan, Hanoi, Vietnam">
<meta name="citation_author" content="Cao Xuan Bach">
<meta name="citation_author_institution" content="Food Industries Research Institute, 301 Nguyen Trai, Thanh Xuan, Hanoi, Vietnam">
<meta name="citation_author" content="La Thi My Hanh">
<meta name="citation_author_institution" content="Food Industries Research Institute, 301 Nguyen Trai, Thanh Xuan, Hanoi, Vietnam">
<meta name="citation_author" content="Efstratios Nikolaivits">
<meta name="citation_author_institution" content="Division of Industrial Biotechnology, Department of Life Sciences, Chalmers University of Technology, Gothenburg, Sweden">
<meta name="citation_author_institution" content="Present Address: Industrial Biotechnology &amp; Biocatalysis Group, School of Chemical Engineering, National Technical University of Athens, Athens, Greece">
<meta name="citation_author" content="Johan Larsbrink">
<meta name="citation_author_institution" content="Division of Industrial Biotechnology, Department of Life Sciences, Chalmers University of Technology, Gothenburg, Sweden">
<meta name="citation_author_institution" content="Wallenberg Wood Science Center, Chalmers University of Technology, Gothenburg, Sweden">
<meta name="citation_author" content="Lisbeth Olsson">
<meta name="citation_author_institution" content="Division of Industrial Biotechnology, Department of Life Sciences, Chalmers University of Technology, Gothenburg, Sweden">
<meta name="citation_author_institution" content="Wallenberg Wood Science Center, Chalmers University of Technology, Gothenburg, Sweden">
<meta name="citation_author" content="Vu Nguyen Thanh">
<meta name="citation_author_institution" content="Food Industries Research Institute, 301 Nguyen Trai, Thanh Xuan, Hanoi, Vietnam">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="15">
<meta name="citation_firstpage" content="30445">
<meta name="citation_doi" content="10.1038/s41598-025-06321-1">
<meta name="citation_pmid" content="40830373">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365195/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365195/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365195/pdf/41598_2025_Article_6321.pdf">
<meta name="description" content="The study of extremophiles can lead to the discovery of highly tolerant enzymes of value to biotechnology, and extreme environments are typically sampled to facilitate their discovery. Here, we show that extreme acidotolerant filamentous fungi, able ...">
<meta name="og:title" content="Extremely acidophilic filamentous fungi are more prevalent in diverse ecosystems than previously documented">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The study of extremophiles can lead to the discovery of highly tolerant enzymes of value to biotechnology, and extreme environments are typically sampled to facilitate their discovery. Here, we show that extreme acidotolerant filamentous fungi, able ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365195/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365195">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41598-025-06321-1"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41598_2025_Article_6321.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365195%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365195/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365195/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365195/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-scirep.jpg" alt="Scientific Reports logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Scientific Reports" title="Link to Scientific Reports" shape="default" href="http://www.nature.com/srep" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Sci Rep</button></div>. 2025 Aug 19;15:30445. doi: <a href="https://doi.org/10.1038/s41598-025-06321-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41598-025-06321-1</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sci%20Rep%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Sci%20Rep%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Sci%20Rep%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Extremely acidophilic filamentous fungi are more prevalent in diverse ecosystems than previously documented</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thuy%20NT%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Nguyen Thanh Thuy</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Nguyen Thanh Thuy</span></h3>
<div class="p">
<sup>1</sup>Food Industries Research Institute, 301 Nguyen Trai, Thanh Xuan, Hanoi, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thuy%20NT%22%5BAuthor%5D" class="usa-link"><span class="name western">Nguyen Thanh Thuy</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Coleman%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Tom Coleman</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Tom Coleman</span></h3>
<div class="p">
<sup>2</sup>Division of Industrial Biotechnology, Department of Life Sciences, Chalmers University of Technology, Gothenburg, Sweden </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Coleman%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tom Coleman</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Christopher%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Meera Christopher</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Meera Christopher</span></h3>
<div class="p">
<sup>2</sup>Division of Industrial Biotechnology, Department of Life Sciences, Chalmers University of Technology, Gothenburg, Sweden </div>
<div class="p">
<sup>3</sup>Wallenberg Wood Science Center, Chalmers University of Technology, Gothenburg, Sweden </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Christopher%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Meera Christopher</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chau%20NB%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Nguyen Bao Chau</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Nguyen Bao Chau</span></h3>
<div class="p">
<sup>1</sup>Food Industries Research Institute, 301 Nguyen Trai, Thanh Xuan, Hanoi, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chau%20NB%22%5BAuthor%5D" class="usa-link"><span class="name western">Nguyen Bao Chau</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bach%20CX%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Cao Xuan Bach</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Cao Xuan Bach</span></h3>
<div class="p">
<sup>1</sup>Food Industries Research Institute, 301 Nguyen Trai, Thanh Xuan, Hanoi, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bach%20CX%22%5BAuthor%5D" class="usa-link"><span class="name western">Cao Xuan Bach</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hanh%20LTM%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">La Thi My Hanh</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">La Thi My Hanh</span></h3>
<div class="p">
<sup>1</sup>Food Industries Research Institute, 301 Nguyen Trai, Thanh Xuan, Hanoi, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hanh%20LTM%22%5BAuthor%5D" class="usa-link"><span class="name western">La Thi My Hanh</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nikolaivits%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Efstratios Nikolaivits</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Efstratios Nikolaivits</span></h3>
<div class="p">
<sup>2</sup>Division of Industrial Biotechnology, Department of Life Sciences, Chalmers University of Technology, Gothenburg, Sweden </div>
<div class="p">
<sup>4</sup>Present Address: Industrial Biotechnology &amp; Biocatalysis Group, School of Chemical Engineering, National Technical University of Athens, Athens, Greece </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nikolaivits%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Efstratios Nikolaivits</span></a>
</div>
</div>
<sup>2,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Larsbrink%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Johan Larsbrink</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Johan Larsbrink</span></h3>
<div class="p">
<sup>2</sup>Division of Industrial Biotechnology, Department of Life Sciences, Chalmers University of Technology, Gothenburg, Sweden </div>
<div class="p">
<sup>3</sup>Wallenberg Wood Science Center, Chalmers University of Technology, Gothenburg, Sweden </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Larsbrink%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Johan Larsbrink</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Olsson%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Lisbeth Olsson</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Lisbeth Olsson</span></h3>
<div class="p">
<sup>2</sup>Division of Industrial Biotechnology, Department of Life Sciences, Chalmers University of Technology, Gothenburg, Sweden </div>
<div class="p">
<sup>3</sup>Wallenberg Wood Science Center, Chalmers University of Technology, Gothenburg, Sweden </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Olsson%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lisbeth Olsson</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thanh%20VN%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Vu Nguyen Thanh</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Vu Nguyen Thanh</span></h3>
<div class="p">
<sup>1</sup>Food Industries Research Institute, 301 Nguyen Trai, Thanh Xuan, Hanoi, Vietnam </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thanh%20VN%22%5BAuthor%5D" class="usa-link"><span class="name western">Vu Nguyen Thanh</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Food Industries Research Institute, 301 Nguyen Trai, Thanh Xuan, Hanoi, Vietnam </div>
<div id="Aff2">
<sup>2</sup>Division of Industrial Biotechnology, Department of Life Sciences, Chalmers University of Technology, Gothenburg, Sweden </div>
<div id="Aff3">
<sup>3</sup>Wallenberg Wood Science Center, Chalmers University of Technology, Gothenburg, Sweden </div>
<div id="Aff4">
<sup>4</sup>Present Address: Industrial Biotechnology &amp; Biocatalysis Group, School of Chemical Engineering, National Technical University of Athens, Athens, Greece </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jan 24; Accepted 2025 Jun 9; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365195  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830373/" class="usa-link">40830373</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The study of extremophiles can lead to the discovery of highly tolerant enzymes of value to biotechnology, and extreme environments are typically sampled to facilitate their discovery. Here, we show that extreme acidotolerant filamentous fungi, able to grow at pH &lt; 1, can be found when sampling highly diverse environments, from industrial sites with low pH to typical non-acidic plant and soil samples. Over 100 fungal strains were isolated from over 2,000 samples taken from across Vietnam, and many of the strains were able to grow over wide pH ranges, displaying either acidotolerance or clear acidophilicity. ITS sequencing revealed that 63 isolates represent 12 previously undescribed species, with the majority from the <em>Talaromyces</em> or <em>Penicillium</em> genera. We furthermore report the rediscovery of the previously lost, historically significant acidophile <em>Acontium velatum</em>. Screening of selected fungal secretomes for polysaccharide-cleaving activity revealed that many show broad tolerance to harsh conditions (pH, temperature, organic solvents). Our work greatly expands on the diversity of identified extreme acidotolerant and acidophilic filamentous fungi, which can serve as sources of industrially relevant enzymes. For most species identified, acid tolerance or acidophilicity has not previously been reported, and our results showcase that acidophilicity is more widespread than previously appreciated.</p>
<section id="sec1"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1038/s41598-025-06321-1.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> New fungal taxa, Acidophile, Acidotolerant, Extremophile, Ascomycota, <em>Penicillium</em>, <em>Talaromyces</em>, <em>Acontium</em>, CAZymes, Biodiversity</p></section><section id="kwd-group2" class="kwd-group"><p><strong>Subject terms:</strong> Fungal biology, Fungal ecology, Hydrolases</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">The existence of life in Earth’s extreme habitats, such as polar regions, deserts, volcanic areas, deep-sea trenches, and acidic streams, can give clues about the genesis of life, and inspire studies of evolution, biodiversity, and development of sustainable industrial processes<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>,<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. However, the definitions of “extreme”, “-philic” and “-tolerant” can be unclear<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. The most common proposed definitions for acidophilicity and acidotolerance, which will be used here, are that acidophiles grow optimally below pH 3.0, while acidotolerant strains grow optimally between pH 3.0 and 5.0<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>; furthermore, “extreme” organisms are those that can grow at pH 1. Notably, many filamentous fungi can be regarded as acidotolerant as their growth environments are commonly below neutral pH<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>, while strict acidophilic species are much less common. This can be attributed to the fact that highly acidic biospheres (pH &lt; 3) are rare in nature<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>, and are predominantly a result of human activity.</p>
<p id="Par4">Most research on extreme acidophilic and acidotolerant fungi has been associated with acid mine drainage sites, due to their environmental impact and unique biodiversity<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. These extreme ecological systems are formed by continued exposure of sulfur-containing minerals (mainly pyrite) to atmospheric oxygen and moisture, allowing chemoautotrophic bacteria to convert sulfur to sulfuric acid, which dissolves and oxidizes iron to create hostile acidic ecosystems<sup><a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. Despite these extreme conditions, the ecological diversity can be high and include known genera with acidotolerant and acidophilic fungi such as <em>Acidomyces</em>, <em>Aspergillus</em>, <em>Penicillium</em><sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Acidophilicity among fungi was first reported in 1943 with the identification of two species from acid solutions from industrial plants. These species were able to grow in a pH 1.0 medium<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, and one was named <em>Acontium velatum</em>. The other species, amazingly, grew in a pH 0 medium containing 12% H<sub>2</sub>SO<sub>4</sub> and was named “Fungus D”. Unfortunately, <em>A. velatum</em> appears to have been lost and is today unavailable in strain collections. In 1973, “Fungus D” was found to be conspecific to fungal strains from acidic soil near a natural gas purification plant and the novel species <em>Scytalidium acidophilum</em> (= <em>Acidomyces acidophilus</em>) was proposed<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Many organisms with extremotolerance to a single physical parameter (e.g. low pH) may be polyextremotolerant (tolerant to several parameters)<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>, suggesting that adaptation mechanisms may share common features. As industrial processes often take place under extreme pH and temperature or in the presence of harsh chemicals and solvents, studies on extremophilic species can be a useful way to identify industrially relevant organisms.</p>
<p id="Par5">Past attempts to comprehensively list acidophilic fungi, using growth at pH &lt; 4, predict the total known number of species to be low (81 total including 8 yeasts)<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>; by our count using stricter cutoffs, 15 taxa can be classified as acidophilic (Table <a href="#MOESM1" class="usa-link">S1</a>). It is highly likely that more acidophilic species exist, but knowledge may be limited due to previous studies having focused on highly acidic environments, which are rare. Furthermore, many known species may have acidophilic traits but have yet to be assessed for these because they are from non-acidic environments<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. We hypothesized that acidophilic species might be present in substrates such as soil and plant debris—possibly through the formation of highly localized, transient extreme microenvironments—and thus could be isolated from near-neutral environmental sites by using strongly acidic conditions as the selection criterion. To test this hypothesis, we chose to broadly survey the rich biodiversity of Vietnam<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>–<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup> for acidophilic filamentous fungi, from both acidic and non-acidic environments. From acidic environments, such as acid mine drainage sites (pH &lt; 3), as well as typical soils and vegetation (pH &gt; 3), totaling 2,240 samples, we were able to isolate 130 strains with either extreme acidophilic or acidotolerant phenotypes. Furthermore, the secretomes from a selection of these were screened for lignocellulolytic enzyme activities and tolerance to organic solvents. We report 12 undescribed species across six genera with potential value to biotechnology, and finally, we report the rediscovery of the previously lost and historic first-reported acidophilic filamentous fungus, <em>Acontium velatum</em>. We demonstrate here that the distribution of acidotolerant and acidophilic species is more widespread than previously shown, and that acidophilic species can be isolated from both highly acidic as well as close to pH-neutral environments.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Fungal isolation using low pH as selection criterion</h3>
<p id="Par6">Acidotolerant and acidophilic fungi across various ecosystems were sampled from soil and plant debris from acidic regions (such as copper, pyrite, and coal mines, disrupted bogs, mangrove forests etc.) and unaffected rainforests, forestry plantations, rice fields, and gardens throughout Vietnam (Table <a href="#MOESM1" class="usa-link">S1</a>, Supplemental File 1). The samples were stored in sterile polyethylene bags at ambient temperature before analysis. The samples were pulverized and sprinkled over acidified malt agar (2% malt agar containing 1% sulfuric acid, pH 0.8) plates, and incubated at room temperature for two weeks. Fungal growth was examined using a stereomicroscope (StereoBlue, Euromex). For purification and to confirm acid tolerance, hyphal tips were transferred onto a fresh isolation medium using a sterile tungsten needle. Successfully growing isolates were further cultured on malt agar for two weeks to facilitate sporogenesis and morphological differentiation through analyzing conidia structure, colony colors, textures, and sizes. Isolates with identical morphological appearance on the plates were grouped together, and one representative per group was selected to use for species identification through ITS sequencing. Since all isolates exhibited good growth in non-acidified media, standard malt agar medium was employed for routine cultivation and preservation. For long-term storage, both cryopreservation in liquid nitrogen and lyophilization were utilized.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Determination of optimal growth pH</h3>
<p id="Par7">Optimal growth pH was evaluated using agar medium containing 1% sulfuric acid and 1% citric acid, which provides a strong buffer capacity from pH 0.96 to pH 6.50. To prepare plates with different pH values, a series of buffer solutions was prepared. For each buffer, 20 g of concentrated sulfuric acid and 20 g of citric acid were added to 50 mL of water. After complete dissolution, dry NaOH was added to adjust the pH to the desired values: pH 1.0, pH 1.5, pH 2.0, pH 2.5, pH 3.0, pH 3.5, pH 4.0, pH 4.5, pH 5.0, pH 5.5, pH 6.0, and pH 6.5. Each buffer solution was then brought to a final volume of 100 mL with water and autoclaved at 115 °C for 15 min.</p>
<p id="Par8">For plating, 1 mL of each buffer solution was transferred into a Petri dish, followed by the addition of 19 mL of warm (50 °C) agar medium. The mixture was gently swirled to mix and allowed to solidify. Agar from each pH condition was subsequently collected, homogenized, centrifuged, and the pH of the liquid phase was measured and recorded as the actual pH values: 1.0, 1.6, 2.0, 2.5, 3.0, 3.5, 4.0, 4.5, 5.0, 5.6, 6.0, and 6.5 (final pH measurements are presented in Fig. <a href="#Fig2" class="usa-link">2</a>). The difference between the initial and final pH values was likely caused by interactions between the medium components during autoclavation. The fungal isolates were grown on these plates for 2 weeks, after which they were photographed to evaluate growth.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365195_41598_2025_6321_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/08ed/12365195/0b5ac980b199/41598_2025_6321_Fig2_HTML.jpg" loading="lazy" id="d33e1259" height="712" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Growth of selected strains of acid tolerant/acidophilic fungi on buffered malt agar at different pH values. The media were buffered and contained 1% sulfuric acid and 1% citric acid. Agar plate pH values were checked at the end of the growth period to ensure there was no change. <em>Aspergillus oryzae</em> CNTP 5138 was used as a control strain, as an industrial, non-acidotolerant species. The fungi were grown at 28 °C for one week, except for the slow-growing <em>Acidomyces</em> spp. and <em>Acontium velatum</em> (two weeks).</p></figcaption></figure></section><section id="Sec5"><h3 class="pmc_sec_title">Gene barcoding and phylogenetic analyses</h3>
<p id="Par9">For phylogenetic analysis and species assignment, the Internally Transcribed Spacer (ITS) region was sequenced. DNA was extracted using phenol-chloroform (1:1, v/v) from a loopful of 48-hour old mycelia grown on malt agar. For dark pigmented isolates, extracted DNA was purified further using a Silica Bead DNA Gel Extraction kit (Thermo Scientific) to remove melanoid compounds that might interfere with PCR.</p>
<p id="Par10">PCR amplification of the ITS regions was performed using any one of the previously reported forward primers ITS1 (5’ TCC GTA GGT GAA CCT GCG G 3’), ITS5 (5’ GGA AGT AAA AGT CGT AAC AAG G), ITS1F (CTT GGT CAT TTA GAG GAA GTA A 3’) or SR6R (5’ AAG WAA AAG TCG TAA CAA GG), and the reverse primer ITS4 (TCC TCC GCT TAT TGA TAT GC 3’). The amplicons were sequenced at First BASE Laboratories Sdn Bhd (Selangor, Malaysia).</p>
<p id="Par11">ITS sequences of selected representatives from each genetic group have been deposited in GenBank (Table <a href="#Tab1" class="usa-link">1</a>). The sequences were compared with available GenBank sequences using BLAST. For phylogenetic analysis, sequences were aligned using MAFFT (<a href="https://mafft.cbrc.jp/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://mafft.cbrc.jp/</a>) running a progressive method with an accurate guide tree (G-INS-1)<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. Phylogenetic trees were constructed using MEGA11<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> and visualized using iTOL v8<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Diversity of fungi isolated from plant residues and soil. The horizontal line separates novel species (above the line) with those previously identified.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Species</th>
<th align="left" colspan="1" rowspan="1">
<em>N</em> strains</th>
<th align="left" colspan="1" rowspan="1">Acid affinity *</th>
<th align="left" colspan="1" rowspan="1">Selected strain **</th>
<th align="left" colspan="1" rowspan="1">GenBank number</th>
<th align="left" colspan="1" rowspan="1">Source of isolation</th>
<th align="left" colspan="1" rowspan="1">Location ***</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Acidomyces</em> sp1.</td>
<td align="center" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">Acidophilic</td>
<td align="left" colspan="1" rowspan="1">ASA 2039.1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646903" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646903</a></td>
<td align="left" colspan="1" rowspan="1">Plant debris, mangrove forest</td>
<td align="left" colspan="1" rowspan="1">Phuoc An, Dong Nai</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Acidomyces</em> sp2.</td>
<td align="center" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">Acidophilic</td>
<td align="left" colspan="1" rowspan="1">ASA 2062.4</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646907" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646907</a></td>
<td align="left" colspan="1" rowspan="1">Plant debris, mangrove forest</td>
<td align="left" colspan="1" rowspan="1">Phuoc An, Dong Nai</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Amplistroma</em> sp1. aff. <em>A. erinaceum</em>
</td>
<td align="center" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">Acidophilic</td>
<td align="left" colspan="1" rowspan="1">ASS 88.1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646948" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646948</a></td>
<td align="left" colspan="1" rowspan="1">Forest soil</td>
<td align="left" colspan="1" rowspan="1">Kon Plong, Kon Tum</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Hypocreales incertae sedis</em> sp1.</td>
<td align="center" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">Acidotolerant</td>
<td align="left" colspan="1" rowspan="1">ASS 64.1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646937" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646937</a></td>
<td align="left" colspan="1" rowspan="1">Hill soil</td>
<td align="left" colspan="1" rowspan="1">Luong Son, Hoa Binh</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Hypocreales incertae sedis</em> sp2.</td>
<td align="center" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">Acidophilic</td>
<td align="left" colspan="1" rowspan="1">ASA 2052.1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646923" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646923</a></td>
<td align="left" colspan="1" rowspan="1">Plant debris, mangrove forest</td>
<td align="left" colspan="1" rowspan="1">Phuoc An, Dong Nai</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Hypocreales incertae sedis</em> sp3.</td>
<td align="center" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">Acidophilic</td>
<td align="left" colspan="1" rowspan="1">ASA 1968.1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646924" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646924</a></td>
<td align="left" colspan="1" rowspan="1">Grasses, coal mine</td>
<td align="left" colspan="1" rowspan="1">Cam Pha, Quang Ninh</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Penicillium</em> sp1. aff. <em>P. dravuni</em>
</td>
<td align="center" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">Acidophilic</td>
<td align="left" colspan="1" rowspan="1">ASM 137.2</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646912" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646912</a></td>
<td align="left" colspan="1" rowspan="1">Plant debris, acid waste</td>
<td align="left" colspan="1" rowspan="1">Duong Kinh, Hai Phong</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Phialomyces</em> sp1. aff. <em>P. macrosporus</em>
</td>
<td align="center" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">Tolerant</td>
<td align="left" colspan="1" rowspan="1">ASS 40.3</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646919" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646919</a></td>
<td align="left" colspan="1" rowspan="1">Hill soil</td>
<td align="left" colspan="1" rowspan="1">Uong Bi, Quang Ninh</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Talaromyces</em> sp1. aff. <em>T. clemensii</em>
</td>
<td align="center" colspan="1" rowspan="1">13</td>
<td align="left" colspan="1" rowspan="1">Acidophilic</td>
<td align="left" colspan="1" rowspan="1">AS 616.3</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646927" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646927</a></td>
<td align="left" colspan="1" rowspan="1">
<em>Luffa aegyptiaca</em> leaves</td>
<td align="left" colspan="1" rowspan="1">Dong Hung, Thai Binh</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Talaromyces</em> sp2. aff. <em>T. clemensii</em>
</td>
<td align="center" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">Tolerant</td>
<td align="left" colspan="1" rowspan="1">AS 607.6</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646930" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646930</a></td>
<td align="left" colspan="1" rowspan="1">
<em>Bauhinia variegate</em> flowers</td>
<td align="left" colspan="1" rowspan="1">Thanh Xuan, Hanoi</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Talaromyces</em> sp3. aff. <em>T. resinae</em>
</td>
<td align="center" colspan="1" rowspan="1">12</td>
<td align="left" colspan="1" rowspan="1">Acidophilic</td>
<td align="left" colspan="1" rowspan="1">ASM 153.1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646935" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646935</a></td>
<td align="left" colspan="1" rowspan="1">Plant debris, pyrite mine</td>
<td align="left" colspan="1" rowspan="1">Ba Vi, Hanoi</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Talaromyces</em> sp4. aff. <em>T. iowaense</em>
</td>
<td align="center" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">n.d.</td>
<td align="left" colspan="1" rowspan="1">ASA 1792.1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646936" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646936</a></td>
<td align="left" colspan="1" rowspan="1">
<em>Ficus racemose</em> rotten fruits</td>
<td align="left" colspan="1" rowspan="1">Mong Cai, Quang Ninh</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Acidomyces acidophilus</em> [MB#511856]</td>
<td align="center" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">Acidophilic</td>
<td align="left" colspan="1" rowspan="1">ACA 17.1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646895" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646895</a></td>
<td align="left" colspan="1" rowspan="1">Acacia bark</td>
<td align="left" colspan="1" rowspan="1">Ba Vi, Hanoi</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Acidomyces acidothermus</em> [MB#804969]</td>
<td align="center" colspan="1" rowspan="1">23</td>
<td align="left" colspan="1" rowspan="1">Acidophilic</td>
<td align="left" colspan="1" rowspan="1">AST 152</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646900" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646900</a></td>
<td align="left" colspan="1" rowspan="1">Plant debris, pyrite mine</td>
<td align="left" colspan="1" rowspan="1">Ba Vi, Hanoi</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Acontium velatum</em> [MB#142596]</td>
<td align="center" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1">Acidophilic</td>
<td align="left" colspan="1" rowspan="1">ASS 125.1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646940" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646940</a></td>
<td align="left" colspan="1" rowspan="1">Hilltop soil</td>
<td align="left" colspan="1" rowspan="1">Dong Van, Ha Giang</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Acrodontium griseum</em> [MB#308227]</td>
<td align="center" colspan="1" rowspan="1">20</td>
<td align="left" colspan="1" rowspan="1">Acidophilic</td>
<td align="left" colspan="1" rowspan="1">ASS 104.2</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646945" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646945</a></td>
<td align="left" colspan="1" rowspan="1">Bazal soil</td>
<td align="left" colspan="1" rowspan="1">Pleiku, Gia Lai</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Aspergillus turcosus</em> [MB#506378]</td>
<td align="center" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">Acidotolerant</td>
<td align="left" colspan="1" rowspan="1">ASS 350.1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646908" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646908</a></td>
<td align="left" colspan="1" rowspan="1">Cultivated soil</td>
<td align="left" colspan="1" rowspan="1">Me Linh, Hanoi</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Penicillium citreonigrum</em> [MB#165197]</td>
<td align="center" colspan="1" rowspan="1">9</td>
<td align="left" colspan="1" rowspan="1">Acidotolerant</td>
<td align="left" colspan="1" rowspan="1">AS 173.2</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646909" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646909</a></td>
<td align="left" colspan="1" rowspan="1">Grass debris</td>
<td align="left" colspan="1" rowspan="1">Van Don, Quang Ninh</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Penicillium griseolum</em> [MB#302401]</td>
<td align="center" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1">Acidophilic</td>
<td align="left" colspan="1" rowspan="1">ASS 45.1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646915" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646915</a></td>
<td align="left" colspan="1" rowspan="1">Hilltop soil</td>
<td align="left" colspan="1" rowspan="1">Hoa Vang, Da Nang</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>Talaromyces chlamydosporus</em> [MB#817392]</td>
<td align="center" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1">Acidophilic</td>
<td align="left" colspan="1" rowspan="1">ASS 52.1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646925" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646925</a></td>
<td align="left" colspan="1" rowspan="1">Garden soil</td>
<td align="left" colspan="1" rowspan="1">Loc Vuong, Nam Dinh</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p21">
<p>*Based on the criteria: Acidophile, optimal growth at pH 3.0 or below; acidotolerant, optimal growth at pH 3.0–5.0. “n.d.”, not determined.</p>
<p>**Strain ID in the collection maintained at FIRI, Hanoi, Vietnam.</p>
<p>***Coordinates for each location are provided in Supplemental File 1.</p>
</div></div></section></section><section id="Sec6"><h3 class="pmc_sec_title">Production of secretomes</h3>
<p id="Par12">For evaluation of the potential of the fungi to produce and secrete lignocellulolytic enzymes, fungi were first grown on malt agar for one week at 30 °C, and the cell mass from one plate (prepared by cutting fungal colonies into agar blocks &lt; 2 mm) then used to inoculate a 100 mL Erlenmeyer flask containing 1.5 g of malt residue (Weyermann malt), 1.5 g sugarcane bagasse (Lam Son sugar factory, Thanh Hoa, Vietnam), 1.5 g rice bran (from a local market, Hanoi, Vietnam), and 8 mL of mineral solution (4.0 g L<sup>− 1</sup> KH<sub>2</sub>PO<sub>4</sub>; 13.6 g L<sup>− 1</sup> (NH<sub>4</sub>)<sub>2</sub>SO<sub>4</sub>; 0.8 g L<sup>− 1</sup> CaCl<sub>2</sub>·H<sub>2</sub>O; 0.6 g L<sup>− 1</sup> MgSO<sub>4</sub>·7H<sub>2</sub>O; 1.0 g L<sup>− 1</sup> yeast extract; 1.0 g L<sup>− 1</sup> peptone; 0.2 mL L<sup>− 1</sup> Tween 80; 1.0 mL L<sup>− 1</sup> trace elements (3.2 mg L<sup>− 1</sup> MnSO<sub>4</sub>·H<sub>2</sub>O; 2.8 mg L<sup>− 1</sup> ZnSO<sub>4</sub>·7H<sub>2</sub>O; 10 mg L<sup>− 1</sup> FeSO<sub>4</sub>·7H<sub>2</sub>O; 4.0 mg L<sup>− 1</sup> CoCl<sub>2</sub>·6H<sub>2</sub>O; 3.5 mg L<sup>− 1</sup> CuSO<sub>4</sub>·5H<sub>2</sub>O); adjusted to pH 2.5 with H<sub>2</sub>SO<sub>4</sub>). Solid state fermentation was carried out for 7 days at 30 °C. Secreted enzymes were extracted by addition of 45 mL 20 mM sodium citrate buffer, pH 5, agitation at 150 rpm for 1 h at 30 °C, and recovery of the crude enzyme solution by centrifugation at 4,000 rpm for 10 min at 10 °C. Secretomes were stored at -20 °C until use.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Carbohydrate-active enzyme activity assays</h3>
<p id="Par13">Protein concentrations were determined using the Bradford assay. Xylanase and CMCase activities were measured using the 3,5-dinitrosalicylic acid (DNS) assay<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Briefly. 0.1 mL of appropriately diluted enzyme was mixed with either 0.2 mL of 1% (w/v) xylan from beechwood (Apollo Scientific Ltd.) or 1% (w/v) carboxymethylcellulose (CMC; Sigma-Aldrich), in a buffer at desired pH. The reactions were incubated at 50 °C for 20 min, and released reducing sugars quantified using the DNS assay, against standard curves of glucose or xylose. One unit per mL of enzyme was defined as the amount of enzyme releasing one micromole of reducing sugar per minute under the assay conditions. As a negative control, the enzyme extract from each strain was heated at 100 °C for 10 min, and all assay procedures were performed in the same way as for the test extract. The reducing sugar concentration from the control was then subtracted to account for substrate sugar carryover. To measure enzyme activities at different pH (pH 1, 3, 5, 7), Britton-Robinson buffer was used (40 mM each of boric, phosphoric, and acetic acids), titrated to the desired pH using 20% (w/v) of H<sub>2</sub>SO<sub>4</sub> or NaOH. The same buffer was used to generate monosaccharide standard curves.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Evaluation of thermal and solvent stability of enzymes</h3>
<p id="Par14">For thermal stability assays, the enzymes were diluted with 50 mM sodium citrate buffer, pH 5, incubated at 70–90 °C for 20 min, and then cooled at 4 °C overnight, followed by enzyme activity measurements at 50 °C and pH 5, as described above. Control samples prepared identically but without heating were used as controls. To test tolerance to organic solvents, the enzymes were diluted with 50 mM sodium citrate buffer, pH 5, containing organic solvent (acetone, acetonitrile or dimethylformamide at 10, 20, 30% v/v) or detergent (Sodium Dodecyl Sulfate (SDS), β-mercaptoethanol, TritonX-100, or Tween 80 at 0.1, 1 and 2% v/v), and enzyme activity measured as described previously. Standard curves were generated by diluting glucose and xylose with buffer containing solvent/detergent. PCA analysis was performed using the standard settings on OriginPro (OriginLab).</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Sampling in Vietnam and construction of a diverse library of filamentous fungi</h3>
<p id="Par15">The microbial diversity in Vietnam is understudied, and thus provides an excellent opportunity for deepening our understanding of ecology in Southeast Asia. A selective solid medium containing 1% sulfuric acid (pH = 0.8) was employed for fungal isolation. Sulfuric acid provides strong buffering capacity at pH values below 2.5 (pKa values: -3 and 1.9), and is not likely to be assimilated in significant amounts by fungi. Between 2018 and 2024, 2,240 samples were collected from 131 sampling sites over a wide range of natural and anthropogenic ecosystems (Fig. <a href="#Fig1" class="usa-link">1</a>, Supplemental File 1). Interestingly, extreme acidotolerant fungi were found in most of the sampling locations, although in low numbers. Directly placing samples on the low-pH agar plates typically yielded singular colonies after two weeks of incubation, while sporogenesis was rare (Figure <a href="#MOESM1" class="usa-link">S1</a>). The plates were free of cosmopolitan species, such as <em>Fusarium</em>, <em>Trichoderma</em>, black aspergilli, and mucoraceous fungi. All the isolates grew well in standard non-acidified malt extract media, where sporogenesis was more intense, allowing for detailed morphological differentiation. From more than a thousand isolates, 130 that showed distinctive macromorphological characteristics were selected for species identification by ITS sequencing. Representatives from each species/genetic group were assessed for their growth pH optima, micromorphological characterization, and enzyme production on natural plant substrates.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365195_41598_2025_6321_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/08ed/12365195/88fb0b1a96d4/41598_2025_6321_Fig1_HTML.jpg" loading="lazy" id="d33e592" height="944" width="643" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Origin of the acidotolerant and acidophilic fungal strains identified and screened. <strong>(A)</strong> Sampling locations. Green circles indicate coordinates of sampled locations where strains were subsequently isolated and determined to grow under acidic selection pressure. Yellow squares indicate the sampling sites where strains were selected for ITS sequencing. Red crosses indicate acidic environments where strains were successfully isolated under acidic conditions and ITS sequenced. Hanoi is shown in dark grey. Detailed sampling information, including coordinates, strain information and taxonomy, isolation location and environment are provided in Supplemental File 1. <strong>(B)</strong> Geographical heatmap of number of isolated strains (N) across Vietnam, sorted by province. Vector shapefiles were obtained from the Humanitarian Data Exchange (<a href="https://data.humdata.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://data.humdata.org/</a>). The Python 3 code for this figure is provided in Supplemental File 2.</p></figcaption></figure></section><section id="Sec11"><h3 class="pmc_sec_title">Taxonomy, ecology, and description of selected species</h3>
<p id="Par17">The sequenced ITS regions of the strains indicated that they all belong to Ascomycota, in agreement with previous compilations of known species of acidotolerant/acidophilic fungi<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. Among the 130 strains, 67 could be identified as already known species, namely <em>Acidomyces acidothermus</em> (<em>N</em> = 23 strains), <em>Acrodontium griseum</em> (<em>N</em> = 20), <em>Penicillium citreonigrum</em> (<em>N</em> = 9), <em>Acontium velatum</em> (<em>N</em> = 6), <em>Acidomyces acidophilus</em> (<em>N</em> = 5), <em>Penicillium griseolum</em> (<em>N</em> = 2), <em>Talaromyces chlamydosporus</em> (<em>N</em> = 1), and <em>Aspergillus turcosus</em> (<em>N</em> = 1) (Table <a href="#Tab1" class="usa-link">1</a>). The remaining 63 strains showed low homology in ITS sequences to known species and could be assigned to 12 as-yet-undescribed species, including four species in the genus <em>Talaromyces</em>, one species of <em>Phialomyces</em>, two species of <em>Acidomyces</em>, one species of <em>Penicillium</em>, one species of <em>Amplistroma</em>, and three species of unclear genetic status (Table <a href="#Tab1" class="usa-link">1</a>). The ITS sequences obtained for representatives of all species and genetic groups have been deposited in GenBank with accession numbers <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PP646895:PP646950%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646895-PP646950</a>.</p></section><section id="Sec12"><h3 class="pmc_sec_title"><em>Acidomyces</em></h3>
<p id="Par22">With 36 isolates, <em>Acidomyces</em> was the most frequent fungal genus found in this study, and it was the most clearly dominant genus found in samples collected from extreme acidic environments (mine runoffs, disrupted mangrove forests, and factory acid waste streams). Previously, species of <em>Acidomyces</em> have been isolated from acid mine drainage biofilms, acid streams, sulfur contaminated soils, and acidic soils<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>,<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. Significantly, we found <em>Acidomyces</em> strains also in non-acidic environments and from a vast variety of substrates, including soil, plant debris, bark, and leaves. All isolated <em>Acidomyces</em> strains exhibited optimum growth at pH &lt; 3.0 and can therefore be considered true acidophiles according to our definition (Fig. <a href="#Fig2" class="usa-link">2</a>).</p>
<p id="Par23">To date, two species of <em>Acidomyces</em> have been described – <em>Acidomyces acidophilus</em> and <em>A. acidothermus</em> (= <em>A. richmondensis</em>)<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. In our study, 23 strains were identified as <em>A. acidothermus</em>, and 5 strains as <em>A. acidophilus</em>. Phylogenetic analysis showed that a group of eight strains formed a well-supported cluster with <em>A. acidophilus</em> and <em>A. acidothermus</em> but represent two distinct lineages from those species (Fig. <a href="#Fig3" class="usa-link">3</a>). The two lineages differed from each other by 2.4% in ITS sequence identity and from the known <em>Acidomyces</em> by 5.4% and 7.4%. The differences were significantly higher than the average weighted infraspecific ITS variability value of 1.96% for Ascomycota<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Here, we temporarily termed the groups <em>Acidomyces</em> sp1 (2 strains) and <em>Acidomyces</em> sp2 (6 strains). <em>Acidomyces</em> sp1 and <em>Acidomyces</em> sp2 formed grayish black, compact colonies with swollen chlamydospores and sterile aerial hyphae, typical for <em>Acidomyces</em> spp. The growth of <em>Acidomyces</em> sp2 was slow and more restricted compared to other <em>Acidomyces.</em></p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365195_41598_2025_6321_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/08ed/12365195/eea4cfd9d070/41598_2025_6321_Fig3_HTML.jpg" loading="lazy" id="d33e1306" height="944" width="651" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Maximum likelihood phylogenetic tree depicting the diversity of extreme acid tolerant/acidophilic fungi isolated from Vietnam, and their relationship to closely related/known extreme acid tolerant/acidophilic fungi. The tree was constructed based on ITS sequences using the Maximum Likelihood method and Tamura-Nei model in MEGA11<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. An alignment of 870 positions was used for analysis. All ambiguous positions were removed for each sequence pair. GenBank accession numbers of ITS sequences are given after the taxon names. The scale bar represents 5% sequence divergence; genus groupings colored in alternating blue and orange (for clarity) represent strains obtained in this study; type strains are shown in black with no color bar; type strains are in bold and marked with an asterisk (*) are known extreme acid tolerant/acidophilic taxa.</p></figcaption></figure><p id="Par24">With 23 strains isolated, <em>A. acidothermus</em> was the most frequent species found in this study and its name refers to its ability to grow in acidic conditions and at elevated temperatures (up to 45 °C)<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. <em>A. acidophilus</em>, on the other hand, cannot grow at or above 37 °C<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Accordingly, we found <em>A. acidothermus</em> throughout Vietnam while <em>A. acidophilus</em> was restricted to Northern Vietnam, where the ambient temperature is significantly lower (Figure <a href="#MOESM2" class="usa-link">S2</a>). The genome of <em>A. richmondensis</em> (= <em>A. acidothermus</em>) FRIK2901 has been sequenced, revealing a broad set of putative carbohydrate-active enzymes<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup>. <em>Acidomyces</em> spp. are currently among the most studied acid tolerant/acidophilic fungi due to their unique physiology and technological potential<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>, and here it was evident that they are also widely distributed in diverse environments.</p></section><section id="Sec13"><h3 class="pmc_sec_title"><em>Acontium velatum</em></h3>
<p id="Par26">A group of six acidophilic fungi with similar morphology could not be affiliated to any known genera due to low ITS sequence similarity. These strains were associated with both acidic and non-acidic environments. Extreme acidophilicity of the fungus was suspected as it was found actively growing on wood samples stored in 1% sulfuric acid for almost 2 years. The strains produced white, spreading, yeast-like colonies on malt agar containing 1% sulfuric acid (Fig. <a href="#Fig2" class="usa-link">2</a>). The observed growth optimum at pH &lt; 2, combined with little to no growth at pH &gt; 3.5, suggest <em>A. velatum</em> is a true acidophilic species. In old cultures, a thin, powdery white layer of mycelia with conidiophores was formed from the center of the colonies. The conidiophores were one-celled, elongated, and tipped with aggregated allantoid, bean-like conidia (Fig. <a href="#Fig4" class="usa-link">4</a>). These characteristics fit well with the description of <em>Acontium velatum</em> mentioned in the first report on extreme acid tolerant fungi from 1943<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>, and thus the name was assigned to these strains (Fig. <a href="#Fig3" class="usa-link">3</a>). Although <em>A. velatum</em> is one of the oldest cited examples regarding acidotolerance in fungi, the initial strain appears to have been lost and is not available in major fungal collections, but since the taxon <em>Acontium velatum</em> [MB#142596] is still legitimate, we here proposed the strain ASS 125.1, isolated from a cultivated rocky hilltop soil in Lung Tao, Dong Van District, Ha Giang, Vietnam (coordinates 23°17’35.2"N 105°18’16.3"E) in 2019, as the neotype for the species. The strain is now maintained in a metabolically inactive state at the Vietnam Collection of Industrial Microorganisms, Food Industries Research Institute, Hanoi, Vietnam. The ITS sequence for the strain has been deposited in GenBank with ID: <a href="https://www.ncbi.nlm.nih.gov/nuccore/PP646940" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646940</a>.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365195_41598_2025_6321_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/08ed/12365195/cf61af7b057c/41598_2025_6321_Fig4_HTML.jpg" loading="lazy" id="d33e1398" height="945" width="752" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Plate cultures and conidiogenesis of selected acidophilic and acid tolerant fungi. By row: <em>Av</em> - <em>Acontium velatum</em> ASS 125.1 on malt agar containing 1% sulfuric acid, 3 weeks; <em>As</em> - <em>Amplistroma</em> sp1 ASS 88.1 on potato dextrose agar (PDA), 1 week; <em>Pg</em> - <em>Penicillium griseolum</em> ASS 45.1 on PDA, 1 week; <em>At</em> - <em>Aspergillus turcosus</em> ASS 350.1 on PDA, 1 week. Bars represent 10 μm (center) and 5 μm (right).</p></figcaption></figure></section><section id="Sec14"><h3 class="pmc_sec_title">
<em>Acrodontium</em> and <em>A</em><em>mplistroma</em>
</h3>
<p id="Par28">A group of 28 strains sharing similar morphology and distribution formed a well-supported cluster in the ITS sequence tree with <em>Acrodontium griseum</em> and <em>Amplistroma erinaceum</em>; neither genus has previously been associated with acid tolerance (Fig. <a href="#Fig3" class="usa-link">3</a>). The strains were found in soil and plant debris throughout Vietnam and there was no clear preference for acidic environments. On the isolation plates, the light to dark brown fungi showed a moderate level of conidiogenesis. They formed one-celled apiculate conidia borne on sympodial straight and denticulate conidiogenous cells. Twenty isolates were identified as <em>A. griseum</em> based on ITS sequences. The species has been listed among fungi found in compost<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, endophytes in orchid (<em>Aularthron bilamellatum</em>)<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup> and creeping thistle (<em>Cirsium arvense</em>)<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. The remaining eight isolates were closely related to <em>A. erinaceum</em> but differed from the latter by 15/594 nucleotides. Classification of <em>Amplistroma</em> is mainly based on morphology of fruiting bodies formed on native plant substrates. Among nine validly described species of <em>Amplistroma</em>, four have not been cultured<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>, making it difficult to draw conclusions on the species’ behavior in acidic conditions.</p></section><section id="Sec15"><h3 class="pmc_sec_title"><em>Aspergillus turcosus</em></h3>
<p id="Par30">Despite the cosmopolitan nature and genetic diversity of <em>Aspergillus</em>, in this study, only one strain of acid tolerant <em>Aspergillus</em> was found, and it was identified as <em>Aspergillus turcosus</em> (Figs. <a href="#Fig2" class="usa-link">2</a> and <a href="#Fig4" class="usa-link">4</a>). Strains of <em>A. turcosus</em> have been isolated previously from clinical samples and mistakenly treated as <em>A. fumigatus</em>, and the primary reason for interest in <em>A. turcosus</em> is thus related to its clinical importance and azole drug resistance<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. The genome of <em>A. turcosus</em> encodes a broad carbohydrate-active enzyme repertoire typically found in saprotrophic Aspergilli<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. The lack of diverse <em>Aspergillus</em> strains in our study is noteworthy as several species have been previously linked to acidotolerance<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>.</p></section><section id="Sec16"><h3 class="pmc_sec_title">
<em>Penicillium</em> and <em>Talaromyces</em>
</h3>
<p id="Par31">A group of 46 morphologically heterogenous isolates were identified as species of <em>Penicillium</em> and <em>Talaromyces</em> (Fig. <a href="#Fig3" class="usa-link">3</a>), but, strikingly, no correlation to acidic sampling environments was observed. On primary isolation plates, the fungi showed different adaptation to extreme acidity indicated by the growth patterns and levels of conidiogenesis. <em>Penicillium</em> and <em>Talaromyces</em> represent diverse groups of fungi with worldwide distribution, and furthermore play an important role as decomposers of organic matter<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. The genus <em>Talaromyces</em> was introduced to accommodate a group of <em>Penicillium</em> in the <em>Biverticillium</em> that possess sexual stages<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Currently, the genus <em>Penicillium</em> contains 483 species and the genus <em>Talaromyces</em> 171 species<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. Strains of both <em>Penicillium</em> and <em>Talaromyces</em> have been previously isolated at pH 3, at times showing optimal growth at pH values as low as 2<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>, which clearly indicates acidophilicity.</p>
<p id="Par32">The ITS sequences of nine isolates were closest to <em>Penicillium citreonigrum</em>, which has previously been genome sequenced (<a href="https://mycocosm.jgi.doe.gov/Pencit1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://mycocosm.jgi.doe.gov/Pencit1</a>)<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>, but for which no information on acidophilicity or tolerance has been reported. The assignment of the isolates to <em>P. citreonigrum</em> is however provisional as there are several closely related species in this group, including <em>P. fundyense</em> and <em>P. citreosulfuratum.</em> Two isolates were identified as <em>P. griseolum</em>, which was originally isolated from acidic dune sand in England. The fungus formed monoverticillate, short and smooth conidiophores with globose, grey conidia. The morphology of <em>P. griseolum</em> is unique among <em>Penicillium</em> and the species represent a monophyletic section within the genus<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>. The species is a frequent endophyte in the root of conifers <em>Pinus monticola</em>, <em>Pinus ponderosa</em>, and <em>Pseudotsuga menziesii</em><sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Similar to <em>P. citreonigrum</em>, the tolerance of <em>P. griseolum</em> to extreme acidic conditions has not yet been reported. Four isolates were close to <em>Penicillium dravuni</em> but differed from the later by seven nucleotides. The species was described in 2005, based on a strain isolated from marine alga <em>Dictyosphaeria versluyii</em><sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>.</p>
<p id="Par33">Among 31 strains assigned to the genus <em>Talaromyces</em>, only one strain could be identified on the species level based on its ITS sequence, as <em>Talaromyces chlamydosporus</em>, but since this species was described only recently<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>, predictions or assumptions regarding behavior in acidic conditions cannot be made. The remaining strains represent four lineages, including two closest to <em>Talaromyces clemensii</em>, one to <em>T. resinae</em>, and one to <em>T. iowaense</em>. The differences in ITS sequences to related species were high, excluding the possibility of species-level assignment.</p></section><section id="Sec17"><h3 class="pmc_sec_title">
<em>Phialomyces</em> and <em>Hypocreales incertae sedis</em>
</h3>
<p id="Par34">A group of three grayish-olivaceous pigmented strains formed a cluster with <em>Phialomyces arenicola</em> and <em>Ph. macrosporus</em> but represent a distinct lineage (Fig. <a href="#Fig3" class="usa-link">3</a>). It differed from the closest species <em>Ph. macrosporus</em> by 14/510 nucleotides. The strains grew over a wide pH range, from 2.5 to 6.5, and optimal growth between pH 3.5–4.5 and thus were deemed acidotolerant (Fig. <a href="#Fig2" class="usa-link">2</a>).</p>
<p id="Par35">Ten strains forming two deep-rooted lineages could not be assigned to known genera based on ITS sequences, and hereby are regarded as <em>Hypocreales incertae sedis</em> (Table <a href="#Tab1" class="usa-link">1</a>). These strains were slow-growing and produced conidia in chains on the tip of simple single cell phialides. One slightly pinkish strain represents a previously unidentified species, <em>Hypocreales</em> sp1, which was acidotolerant. Meanwhile, the remaining nine strains were non-pigmented, representing two unidentified species here called <em>Hypocreales</em> sp2 and <em>Hypocreales</em> sp3 which were both acidophilic. Their growth was remarkably restricted to acidic conditions with pH below 3.5 and optimum at pH 2.0 (Fig. <a href="#Fig2" class="usa-link">2</a>) and we thus consider them extreme fungal acidophiles worthy of future deeper investigations.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Enzyme screening and characterization</h3>
<section id="Sec19"><h4 class="pmc_sec_title">Screening of secreted carbohydrate-active enzyme profiles</h4>
<p id="Par36">Filamentous fungi are known for secreting a diverse array of enzymes when growing on complex substrates. In our previous report on acidotolerant thermophilic lignocellulolytic fungi, many strains were found to have higher carboxymethylcellulose hydrolytic activity (CMCase) at pH 3 than at pH 5<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. Given the high acid tolerance during growth observed in the species studied here, we speculated that also their enzymes may have such properties, even when sourced from non-acidic environments. Thus, assessing the pH optimum of secretomes may reveal trends between genera. Several strains of each newly discovered species (where possible) were cultivated on a complex medium containing both sugarcane bagasse and rice bran to potentially induce broad lignocellulolytic activities. The proteins secreted into the media (secretomes) were collected and concentrated, followed by xylanase and CMCase assays (which are representative assays for lignocellulolytic enzyme activities) conducted at pH 1, 3, 5 and 7 (Fig. <a href="#Fig5" class="usa-link">5</a>). Xylanase activity was typically highest at pH 3, with drastic reduction in activity both above and below this value (Fig. <a href="#Fig5" class="usa-link">5</a>A). In contrast, the CMCase activity was broader for most of the secretomes and typically maximal at pH 1 (Fig. <a href="#Fig5" class="usa-link">5</a>B). We here performed controls to rule out the possibility of non-enzymatic acid hydrolysis, as hydrolysis is faster at low pH. This check showed that significant acid hydrolysis occurred only at pH 1, which was subtracted from the appropriate assay results, and thus acid hydrolysis was not responsible for the observed activity. The highest xylanase activity (451 U/mg protein) was observed with the secretome of <em>Amplistroma sp1.</em> (strain ID: ASS 46.1), with a maximum at pH 3. For CMCase activity, the highest activity was seen for the secretome from isolate AST 152 of <em>Acidomyces acidothermus</em> which was most active at pH 1 (0.44 U/mg). While the absolute value of CMCase activity was low, and it being an unnatural substrate for general endo-glucanase activity screening, these results indicate that several secretome enzymes have optimal activity at extremely low pH.</p>
<figure class="fig xbox font-sm" id="Fig5"><h5 class="obj_head">Fig. 5.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365195_41598_2025_6321_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/08ed/12365195/db1cd042ba07/41598_2025_6321_Fig5_HTML.jpg" loading="lazy" id="d33e1780" height="944" width="679" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Measurement of (A) xylanase and (B) CMCase activities of the secretomes of acidotolerant and acidophilic fungi grown on complex substrates (1.5 g/100 mL each of malt residue, sugarcane bagasse, and rice bran). All activities were measured at 50 °C. In each panel is shown the maximum specific activity (U/mg); the relative activity level at pH 1, 3, 5 and 7, scaled to the highest activity observed as 100%; and lastly the residual activity after heating at 70 °C for 20 min followed by overnight cooling at 4 °C before measuring activity at 50 °C. Each value is the average of duplicate measurements. Blanks containing no substrate were subtracted. On the left is shown an unscaled UPGMA (unweighted pair group method with arithmetic mean) tree.</p></figcaption></figure></section><section id="Sec20"><h4 class="pmc_sec_title">Tolerance to organic solvents, surfactants and heat</h4>
<p id="Par37">Our discovery of multiple fungi able to grow in acidic conditions—from both known and hitherto unknown species—and their production of apparently acid-preferring enzymes, inspired us to assess enzyme tolerance to other conditions<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. Assays were performed as previously described, but additionally included common organic solvents and surfactants: acetone, acetonitrile, dimethylformamide (DMF; each at 10, 20, 30% v/v), sodium dodecyl sulfate (SDS), β-mercaptoethanol (BME), Triton X and Tween 80 (each at 0.1, 1, 2% v/v), and compared to control reactions without added organic solvents or surfactants.</p>
<p id="Par38">The xylanase activity was typically reduced to &lt; 50% of initial activity after inclusion of organic solvents or surfactants in the assay, and we therefore consider a particular species’ secreted enzymes “resistant” to an organic solvent or surfactant if the residual activity was ≥ 50% of the control reaction. The results showed that several strains of <em>Aspergillus</em>, <em>Penicillium</em>, <em>Phialomyces</em> and <em>Talaromyces</em> had secretomes with xylanases resistant to certain organic solvents and surfactants (Figure <a href="#MOESM3" class="usa-link">S3</a>). One notable example was the strain <em>Amplistroma</em> sp1. aff. <em>A. erinaceum</em> (ASS 99 − 1), which retained ≥ 50% residual activity in the presence of 20% v/v acetone, 1% BME, 1% Triton X, or 1% Tween 80.</p>
<p id="Par39">Principle component analysis (PCA) was used to investigate similarities and differences on how each solvent influenced the xylanase activity. Two eigenvectors explained 82.7% of the variance (PC1, 66.3%; PC2, 16.5%, Figure <a href="#MOESM4" class="usa-link">S4</a>), and the loading coefficients were &lt; 0.5 for all chemicals tested (including the control condition) on PC1 (Table <a href="#MOESM2" class="usa-link">S2</a>), suggesting that each chemical affected the xylanase activity in a similar fashion. When analyzing CMCase activity, in contrast to the xylanase results, the addition of acetone, acetonitrile, DMF, as well as the anionic SDS increased the observed activity significantly (Supplemental File 3). It cannot be excluded, however, that these observed activity increases stem from an interaction with the reducing sugar assay used to monitor activity.</p>
<p id="Par40">We next assessed the thermotolerance of each species’ secreted enzymes by heat-treating each secretome at 70 °C before analysis of residual enzyme activity. No significant thermotolerance was observed for xylanase activity, where the residual activity after heat treatment was &lt; 30% compared to the untreated samples for the majority of secretomes (Fig. <a href="#Fig5" class="usa-link">5</a>A). However, the secretomes from several <em>Acidomyces</em>,<em> Aspergillus</em>,<em> Penicillium</em>, and <em>Talaromyces</em> strains retained over 50% and up to 100% of CMCase activity after heat treatment (Fig. <a href="#Fig5" class="usa-link">5</a>B). Thermotolerance has been reported for exo- and endo-glucanase activity for some strains previously<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. The <em>Acrodontium</em> and <em>Amplistroma</em> secretomes on the other hand had &lt; 25% residual activity in all cases, which points towards production of much less thermotolerant enzymes. When compared to well-known industrially relevant filamentous fungi, namely <em>Talaromyces cellulolyticus</em> and <em>Aspergillus niger</em>, the enzymes secreted by many members of our collection of fungi displayed similar tolerance to very low pH, as well as elevated temperature. Previously, β-glucosidases from <em>T. cellulolyticus</em> have been shown to have maximum activity at pH 5.5–6, while the CMCase activity of its cellulases was optimal at pH 4<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup>. While <em>A. niger</em> can grow at a remarkable range of pH values (1.4–9.8), the pH optimum of its cellulases have been typically reported as pH 4.0, with optimal stability at pH 5.0 and above<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>–<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. Selected members of our acidophile culture collection can be considered competitive with, or more tolerant to, these desirable properties. In summary, the activity results highlight that enzymes from many fungal genera—including several previously not linked to acidotolerance—possess high tolerance to multiple physical conditions, and this phenomenon may be relatively common.</p></section></section></section><section id="Sec21"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par42">Fungal biodiversity is high in tropical regions, and through application of selection pressures, can be effectively filtered to identify new species with targeted properties. We here used growth at low pH as a moderate selection pressure and were able to expand greatly on the known diversity of acidotolerant and extreme acidophilic fungi. We found a surprisingly high diversity of strains able to grow under this selective pressure even outside of locations considered acidic. We can here report that in previously classified taxa, six additional species can be added to the list of acidotolerant and/or acidophilic filamentous fungi, which adds to the functional diversity of known species. Additionally, we provide genetic evidence that the newly isolated strain ASS 125.1 represents a candidate for the neotype of the previously lost acidophile, <em>Acontium velatum</em>. Furthermore, 12 taxa were identified as being entirely new, the majority of which growing preferentially at low pH, from known genera with acidophilic species, genera previously not linked to acidophilicity, as well as potentially new and previously undescribed genera.</p>
<p id="Par43">Highly acidic biospheres (pH &lt; 3) are rather rare in nature<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>, and our observation of widely spread acidotolerant and acidophilic fungi from acidic to non-acidic environments prompts speculation about the true ecological habitats of these fungi. Except for volcanic springs, most highly acidic environments found today are of anthropogenic origins, and have not exerted selective pressure for millennia. The most prevalent natural acidic environments might instead be associated with acid-containing plant tissues, which especially saprophytic or plant-pathogenic fungi would often encounter. Although most organic acids are weak, some commonly found in plant tissues such as oxalic acid may cause extreme acidification, especially when the tissue is withering up and the acid consequently becomes concentrated<sup><a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. For example, 10 mM and 100 mM solutions of oxalic acid have pH values of 2.09 and 1.31, respectively. Furthermore, there is a wide range of fungi that can produce oxalic acid both when grown on laboratory media and on native substrates<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Oxalic acid plays several biological roles, including initiating the necrosis of plant tissue during pathogenesis, for competition with other species, for scavenging trace minerals, and mobilizing phosphate and sulfate ions<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Thus, extreme acid tolerance might not necessarily be rooted from adaptation to acidic geo-habitats but could be the result of fungal-plant interactions. The biology of these fungi is poorly understood, and these newly isolated species provide a great opportunity for further study.</p>
<p id="Par44">In the current energy industry, there is a growing need for the use of lignocellulose residues as renewable carbon feedstocks to replace fossil resources<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>, particularly wheat straw in Europe<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup> and rice straw in Vietnam and across Southeast Asia<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>,<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup>. The library of species identified here was observed to secrete CAZymes when grown on lignocellulose, and many with optimal activity at very low pH. Potentially these fungi can provide two advantages in lignocellulose deconstruction applications – first, by operating in acidic conditions, where the competition from mesophilic organisms would be reduced; and second, the deconstruction of biomass may be accelerated due to the acidity. Second-generation biofuel production often utilizes steam explosion or other pretreatment methodologies that lead to acidification of the feedstock<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. For continuous bioprocessing, enzymes and microorganisms should retain activity at the pH of the side-stream, as well as tolerate inhibitors present. By discovering and applying enzymes adapted for such conditions, costs connected to both neutralization and microbial contamination—which are common in conditions closer to neutral pH—could potentially be mitigated.</p>
<p id="Par45">Significantly, many of the here studied secretomes were found to have tolerance to organic solvents and surfactants, in addition to low pH, and these may be useful traits in industrial settings. For CMCase activity, there was an added tolerance also to elevated temperatures, suggesting overall robust enzymes secreted by many of the fungal strains, which has also been observed in previous studies on novel <em>Thielavia</em> spp. strains exhibiting both thermal stability and tolerance to acidic conditions<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. In contrast, however, there was no cross-tolerance towards elevated temperatures for xylanase activity in our experimental setup, which may reflect on the sampling strategy not selecting for thermotolerant strains. Future in-depth biochemical characterization of selected enzymes from the here discovered fungi would be needed to conclusively demonstrate enzymatic properties and industrial relevance.</p>
<p id="Par46">In conclusion, multiple new acidotolerant and acidophilic strains were isolated using low pH as the main selection criterion, and it was shown to be a successful strategy to sample functional diversity across the whole country of Vietnam from both acidic and non-acidic environments. We suggest that the design of such selection pressures should be a key part of larger environmental sampling campaigns, to assay different phenotypes or functional range within different biotopes. We speculate that acidophilic strains are much more prevalent than is recognized today, and that such phenotypes have been neglected in previous work. Our results suggest that an expectation that the pH dependence of a species is closely tied to the surrounding environment is not necessarily true, and that organisms exhibiting extreme acidophilic traits can exist in habitats without apparent highly acidic character.</p>
<p id="Par47">The 130 newly isolated strains are now maintained in the Vietnam Collection of Industrial Microorganisms at the Food Industries Research Institute which are available for academic use, and which represent a remarkable level of diversity. Further investigation of these remarkable new species, through genome sequencing and annotation, will be highly relevant for both fundamental biological study of acid-tolerance, as well as to pinpoint which enzymes they employ for successful depolymerization of complex renewable plant biomass at extremely acidic conditions.</p></section><section id="Sec22"><h2 class="pmc_sec_title">Electronic supplementary material</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365195/bin/41598_2025_6321_MOESM1_ESM.py" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (8.4KB, py) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12365195/bin/41598_2025_6321_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (843.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12365195/bin/41598_2025_6321_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (40.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12365195/bin/41598_2025_6321_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4</a><sup> (88.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12365195/bin/41598_2025_6321_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5</a><sup> (48.8KB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors acknowledge the Vietnam Collection of Industrial Microorganisms (VCIM) for maintenance of the fungal isolates.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>CAZyme</dt>
<dd><p id="Par200">Carbohydrate active enzyme</p></dd>
<dt>Extreme acidophilic</dt>
<dd><p id="Par300">Species which grows at pH 1.0 and optimally at pH 3.0 or below</p></dd>
<dt>Extreme acid-tolerant</dt>
<dd><p id="Par400">Species which grows at pH 1.0 and optimally between pH 3.0 and 5.0</p></dd>
<dt>SDS</dt>
<dd><p id="Par500">Sodium dodecyl sulfate</p></dd>
<dt>CMC</dt>
<dd><p id="Par501">Carboxymethyl cellulose</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualization: JL, LO, VNT; Investigation: NTT, BCN, CXB, LTMH, VNT; Formal Analysis: TC, MC, NTT, BCN, CXB, LTMH, VNT; Funding acquisition: VNT, JL, LO; Writing - Original Draft: TC, MC, JL, LO, VNT; Writing - Review &amp; Editing: TC, EN, JL, LO, VNT; Visualization: TC, NTT, BCN, CXB, LTMH, VNT; Supervision: JL, LO, VNT. All authors read and approved the final manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>LO, JL, VNT acknowledge funding from the Swedish Research Council (Dnr 2020–03475); VNT, NTT, CXB, NBC, LTMH acknowledge funding from Vietnam MOST NVQG-2020/ĐT.01 and Vietnam MOIT 038.2022.QuyGen.BO. MC, LO, JL were funded by the Wallenberg Wood Science Center, granted by the Knut and Alice Wallenberg Foundation and the Chalmers Foundation.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data are contained within the manuscript or associated online supplemental information. Raw data used for calculations are available on reasonable request from the corresponding author. The ITS sequences obtained for representatives of all species and genetic groups have been deposited in GenBank with accession numbers from <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PP646895:PP646950%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646895 to PP646950</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar4"><h3 class="pmc_sec_title">Adherence to national and international regulations</h3>
<p id="Par51">All work described in this manuscript has been performed under a memorandum of understanding between FIRI and Chalmers University of Technology.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par50">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Nguyen Thanh Thuy and Tom Coleman contributed equally to this work.</p></div>
</div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Merino, N. et al. Living at the extremes: extremophiles and the limits of life in a planetary context. <em>Front. Microbiol.</em><strong>10</strong>, 780 (2019).
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.00780" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6476344/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31037068/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Merino,%20N.%20et%20al.%20Living%20at%20the%20extremes:%20extremophiles%20and%20the%20limits%20of%20life%20in%20a%20planetary%20context.%20Front.%20Microbiol.10,%20780%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Orellana, R. et al. Living at the frontiers of life: extremophiles in Chile and their potential for bioremediation. <em>Front. Microbiol.</em><strong>9</strong>, 2309 (2018).
</cite> [<a href="https://doi.org/10.3389/fmicb.2018.02309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6218600/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30425685/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Orellana,%20R.%20et%20al.%20Living%20at%20the%20frontiers%20of%20life:%20extremophiles%20in%20Chile%20and%20their%20potential%20for%20bioremediation.%20Front.%20Microbiol.9,%202309%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Seckbach, J. Life on the edge and astrobiology: who is who in the polyextremophiles world? In <em>Polyextremophiles: Life Under Multiple Forms of Stress</em> (eds Seckbach, J. et al.) 63–79 (Springer, 2013).</cite>
</li>
<li id="CR4">
<span class="label">4.</span><cite>Johnson, D. B. Physiology and ecology of acidophilic microorganisms. <em>Physiol. Biochem. Extremophiles</em>. 255–270 (2014).</cite>
</li>
<li id="CR5">
<span class="label">5.</span><cite>Baker-Austin, C. &amp; Dopson, M. Life in acid: pH homeostasis in acidophiles. <em>Trends Microbiol.</em><strong>15</strong>(4), 165–171 (2007).
</cite> [<a href="https://doi.org/10.1016/j.tim.2007.02.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17331729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baker-Austin,%20C.%20&amp;%20Dopson,%20M.%20Life%20in%20acid:%20pH%20homeostasis%20in%20acidophiles.%20Trends%20Microbiol.15(4),%20165%E2%80%93171%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Gross, S. &amp; Robbins, E. I. Acidophilic and acid-tolerant fungi and yeasts. <em>Hydrobiologia</em><strong>433</strong>(1), 91–109 (2000).</cite> [<a href="https://scholar.google.com/scholar_lookup?Gross,%20S.%20&amp;%20Robbins,%20E.%20I.%20Acidophilic%20and%20acid-tolerant%20fungi%20and%20yeasts.%20Hydrobiologia433(1),%2091%E2%80%93109%20(2000)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Quatrini, R. &amp; Johnson, D. B. Microbiomes in extremely acidic environments: functionalities and interactions that allow survival and growth of prokaryotes at low pH. <em>Curr. Opin. Microbiol.</em><strong>43</strong>, 139–147 (2018).
</cite> [<a href="https://doi.org/10.1016/j.mib.2018.01.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29414445/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Quatrini,%20R.%20&amp;%20Johnson,%20D.%20B.%20Microbiomes%20in%20extremely%20acidic%20environments:%20functionalities%20and%20interactions%20that%20allow%20survival%20and%20growth%20of%20prokaryotes%20at%20low%20pH.%20Curr.%20Opin.%20Microbiol.43,%20139%E2%80%93147%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Druschel, G. K., Baker, B. J., Gihring, T. M. &amp; Banfield, J. F. Acid mine drainage biogeochemistry at Iron mountain, California. <em>Geochem. Trans.</em><strong>5</strong>(2), 13 (2004).
</cite> [<a href="https://doi.org/10.1186/1467-4866-5-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1475782/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35412773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Druschel,%20G.%20K.,%20Baker,%20B.%20J.,%20Gihring,%20T.%20M.%20&amp;%20Banfield,%20J.%20F.%20Acid%20mine%20drainage%20biogeochemistry%20at%20Iron%20mountain,%20California.%20Geochem.%20Trans.5(2),%2013%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Ou, S. N. et al. Physiological, genomic and transcriptomic analyses reveal the adaptation mechanisms of <em>Acidiella bohemica</em> to extreme acid mine drainage environments. <em>Front. Microbiol.</em><strong>12</strong>, 705839 (2021).
</cite> [<a href="https://doi.org/10.3389/fmicb.2021.705839" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8298002/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34305876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ou,%20S.%20N.%20et%20al.%20Physiological,%20genomic%20and%20transcriptomic%20analyses%20reveal%20the%20adaptation%20mechanisms%20of%20Acidiella%20bohemica%20to%20extreme%20acid%20mine%20drainage%20environments.%20Front.%20Microbiol.12,%20705839%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Baker, B. J. &amp; Banfield, J. F. Microbial communities in acid mine drainage. <em>FEMS Microbiol. Ecol.</em><strong>44</strong>(2), 139–152 (2003).
</cite> [<a href="https://doi.org/10.1016/S0168-6496(03)00028-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19719632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baker,%20B.%20J.%20&amp;%20Banfield,%20J.%20F.%20Microbial%20communities%20in%20acid%20mine%20drainage.%20FEMS%20Microbiol.%20Ecol.44(2),%20139%E2%80%93152%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Aguilera, A. &amp; González-Toril, E. Eukaryotic life in extreme environments: acidophilic fungi. In <em>Fungi in Extreme Environments: Ecological Role and Biotechnological Significance</em> (eds Tiquia-Arashiro, S. M. &amp; Grube, M.) 21–38 (Springer International Publishing, 2019).</cite> [<a href="https://scholar.google.com/scholar_lookup?Aguilera,%20A.%20&amp;%20Gonz%C3%A1lez-Toril,%20E.%20Eukaryotic%20life%20in%20extreme%20environments:%20acidophilic%20fungi.%20In%20Fungi%20in%20Extreme%20Environments:%20Ecological%20Role%20and%20Biotechnological%20Significance%20(eds%20Tiquia-Arashiro,%20S.%20M.%20&amp;%20Grube,%20M.)%2021%E2%80%9338%20(Springer%20International%20Publishing,%202019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Starkey, R. L. &amp; Waksman, S. A. Fungi tolerant to extreme acidity and high concentrations of copper sulfate. <em>J. Bacteriol.</em><strong>45</strong>(5), 509–519 (1943).
</cite> [<a href="https://doi.org/10.1128/jb.45.5.509-519.1943" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC373771/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16560661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Starkey,%20R.%20L.%20&amp;%20Waksman,%20S.%20A.%20Fungi%20tolerant%20to%20extreme%20acidity%20and%20high%20concentrations%20of%20copper%20sulfate.%20J.%20Bacteriol.45(5),%20509%E2%80%93519%20(1943)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Sigler, L. &amp; Carmichael, J. W. A new acidophilic <em>Scytalidium</em>. <em>Can. J. Microbiol.</em><strong>20</strong>(2), 267–268 (1974).
</cite> [<a href="https://doi.org/10.1139/m74-043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4822789/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sigler,%20L.%20&amp;%20Carmichael,%20J.%20W.%20A%20new%20acidophilic%20Scytalidium.%20Can.%20J.%20Microbiol.20(2),%20267%E2%80%93268%20(1974)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Tedersoo, L. et al. Global diversity and geography of soil fungi. <em>Science</em><strong>346</strong>(6213), 1256688 (2014).
</cite> [<a href="https://doi.org/10.1126/science.1256688" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25430773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tedersoo,%20L.%20et%20al.%20Global%20diversity%20and%20geography%20of%20soil%20fungi.%20Science346(6213),%201256688%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Anasontzis, G. E. et al. Rice straw hydrolysis using secretomes from novel fungal isolates from Vietnam. <em>Biomass Bioenerg.</em><strong>99</strong>, 11–20 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Anasontzis,%20G.%20E.%20et%20al.%20Rice%20straw%20hydrolysis%20using%20secretomes%20from%20novel%20fungal%20isolates%20from%20Vietnam.%20Biomass%20Bioenerg.99,%2011%E2%80%9320%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Thanh, V. N. et al. Surveying of acid-tolerant thermophilic lignocellulolytic fungi in Vietnam reveals surprisingly high genetic diversity. <em>Sci. Rep.</em><strong>9</strong>(1), 3674 (2019).
</cite> [<a href="https://doi.org/10.1038/s41598-019-40213-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6403320/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30842513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thanh,%20V.%20N.%20et%20al.%20Surveying%20of%20acid-tolerant%20thermophilic%20lignocellulolytic%20fungi%20in%20Vietnam%20reveals%20surprisingly%20high%20genetic%20diversity.%20Sci.%20Rep.9(1),%203674%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Nakamura, T., Yamada, K. D., Tomii, K. &amp; Katoh, K. Parallelization of MAFFT for large-scale multiple sequence alignments. <em>Bioinformatics</em><strong>34</strong>(14), 2490–2492 (2018).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6041967/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29506019/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nakamura,%20T.,%20Yamada,%20K.%20D.,%20Tomii,%20K.%20&amp;%20Katoh,%20K.%20Parallelization%20of%20MAFFT%20for%20large-scale%20multiple%20sequence%20alignments.%20Bioinformatics34(14),%202490%E2%80%932492%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Tamura, K., Stecher, G. &amp; Kumar, S. MEGA11: molecular evolutionary genetics analysis version 11. <em>Mol. Biol. Evol.</em><strong>38</strong>(7), 3022–3027 (2021).
</cite> [<a href="https://doi.org/10.1093/molbev/msab120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8233496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33892491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tamura,%20K.,%20Stecher,%20G.%20&amp;%20Kumar,%20S.%20MEGA11:%20molecular%20evolutionary%20genetics%20analysis%20version%2011.%20Mol.%20Biol.%20Evol.38(7),%203022%E2%80%933027%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Letunic, I. &amp; Bork, P. Interactive tree of life (iTOL) v5: an online tool for phylogenetic tree display and annotation. <em>Nucleic Acids Res.</em><strong>49</strong>(W1), W293–W6 (2021).
</cite> [<a href="https://doi.org/10.1093/nar/gkab301" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8265157/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33885785/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Letunic,%20I.%20&amp;%20Bork,%20P.%20Interactive%20tree%20of%20life%20(iTOL)%20v5:%20an%20online%20tool%20for%20phylogenetic%20tree%20display%20and%20annotation.%20Nucleic%20Acids%20Res.49(W1),%20W293%E2%80%93W6%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Ghose, T. K. Measurement of cellulase activities. <em>Pure Appl. Chem.</em><strong>59</strong>(2), 257–268 (1987).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ghose,%20T.%20K.%20Measurement%20of%20cellulase%20activities.%20Pure%20Appl.%20Chem.59(2),%20257%E2%80%93268%20(1987)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Hujslová, M., Kubátová, A., Kostovcík, M. &amp; Kolarík, M. <em>Acidiella bohemica</em> gen. et. sp. nov. and <em>Acidomyces</em> sp. (Teratosphaeriaceae), the Indigenous inhabitants of extremely acidic soils in Europe. <em>Fungal Divers.</em><strong>58</strong>(1), 33–45 (2012).</cite> [<a href="https://scholar.google.com/scholar_lookup?Hujslov%C3%A1,%20M.,%20Kub%C3%A1tov%C3%A1,%20A.,%20Kostovc%C3%ADk,%20M.%20&amp;%20Kolar%C3%ADk,%20M.%20Acidiella%20bohemica%20gen.%20et.%20sp.%20nov.%20and%20Acidomyces%20sp.%20(Teratosphaeriaceae),%20the%20Indigenous%20inhabitants%20of%20extremely%20acidic%20soils%20in%20Europe.%20Fungal%20Divers.58(1),%2033%E2%80%9345%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Yamazaki, A., Toyama, K. &amp; Nakagiri, A. A new acidophilic fungus <em>Teratosphaeria acidotherma</em> (Capnodiales, Ascomycota) from a hot spring. <em>Mycoscience</em><strong>51</strong>(6), 443–455 (2010).</cite> [<a href="https://scholar.google.com/scholar_lookup?Yamazaki,%20A.,%20Toyama,%20K.%20&amp;%20Nakagiri,%20A.%20A%20new%20acidophilic%20fungus%20Teratosphaeria%20acidotherma%20(Capnodiales,%20Ascomycota)%20from%20a%20hot%20spring.%20Mycoscience51(6),%20443%E2%80%93455%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Chan, W. K., Wildeboer, D., Garelick, H. &amp; Purchase, D. <em>Acidomyces acidophilus</em>: Ecology, Biochemical Properties and Application to Bioremediation. Fungi in Extreme Environments: Ecological Role and Biotechnological Significance 505–515 (2019).</cite>
</li>
<li id="CR24">
<span class="label">24.</span><cite>Baker, B. J., Lutz, M. A., Dawson, S. C., Bond, P. L. &amp; Banfield, J. F. Metabolically active eukaryotic communities in extremely acidic mine drainage. <em>Appl. Environ. Microbiol.</em><strong>70</strong>(10), 6264–6271 (2004).
</cite> [<a href="https://doi.org/10.1128/AEM.70.10.6264-6271.2004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC522060/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15466574/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baker,%20B.%20J.,%20Lutz,%20M.%20A.,%20Dawson,%20S.%20C.,%20Bond,%20P.%20L.%20&amp;%20Banfield,%20J.%20F.%20Metabolically%20active%20eukaryotic%20communities%20in%20extremely%20acidic%20mine%20drainage.%20Appl.%20Environ.%20Microbiol.70(10),%206264%E2%80%936271%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Schoch, C. L. et al. Nuclear ribosomal internal transcribed spacer (ITS) region as a universal DNA barcode marker for fungi. <em>Proc. Natl. Acad. Sci.</em><strong>109</strong>(16), 6241-6246. (2012).</cite> [<a href="https://doi.org/10.1073/pnas.1117018109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3341068/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22454494/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Rosienski, M. D., Lee, M. K., Yu, J. H., Kaspar, C. W. &amp; Gibbons, J. G. Genome sequence of the extremely acidophilic fungus <em>Acidomyces richmondensis</em> FRIK2901. <em>Microbiol. Resource Announcements</em>. <strong>7</strong>(16), 01314–01318. 10.1128/mra (2018).</cite> [<a href="https://doi.org/10.1128/MRA.01314-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6256575/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30533739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rosienski,%20M.%20D.,%20Lee,%20M.%20K.,%20Yu,%20J.%20H.,%20Kaspar,%20C.%20W.%20&amp;%20Gibbons,%20J.%20G.%20Genome%20sequence%20of%20the%20extremely%20acidophilic%20fungus%20Acidomyces%20richmondensis%20FRIK2901.%20Microbiol.%20Resource%20Announcements.%207(16),%2001314%E2%80%9301318.%2010.1128/mra%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Anastasi, A., Varese, G. C. &amp; Marchisio, V. F. Isolation and identification of fungal communities in compost and vermicompost. <em>Mycologia</em><strong>97</strong>(1), 33–44 (2005).
</cite> [<a href="https://doi.org/10.3852/mycologia.97.1.33" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16389954/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Anastasi,%20A.,%20Varese,%20G.%20C.%20&amp;%20Marchisio,%20V.%20F.%20Isolation%20and%20identification%20of%20fungal%20communities%20in%20compost%20and%20vermicompost.%20Mycologia97(1),%2033%E2%80%9344%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Gegenbauer, C. et al. Exo- and endophytic fungi enable rapid transfer of nutrients from ant waste to Orchid tissue. <em>New Phytol.</em><strong>238</strong>(5), 2210–2223 (2023).
</cite> [<a href="https://doi.org/10.1111/nph.18761" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10962571/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36683444/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gegenbauer,%20C.%20et%20al.%20Exo-%20and%20endophytic%20fungi%20enable%20rapid%20transfer%20of%20nutrients%20from%20ant%20waste%20to%20Orchid%20tissue.%20New%20Phytol.238(5),%202210%E2%80%932223%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Hartley, S. E., Eschen, R., Horwood, J. M., Gange, A. C. &amp; Hill, E. M. Infection by a foliar endophyte elicits novel arabidopside-based plant defence reactions in its host, <em>Cirsium arvense</em>. <em>New Phytol.</em><strong>205</strong>(2), 816–827 (2015).
</cite> [<a href="https://doi.org/10.1111/nph.13067" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25266631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hartley,%20S.%20E.,%20Eschen,%20R.,%20Horwood,%20J.%20M.,%20Gange,%20A.%20C.%20&amp;%20Hill,%20E.%20M.%20Infection%20by%20a%20foliar%20endophyte%20elicits%20novel%20arabidopside-based%20plant%20defence%20reactions%20in%20its%20host,%20Cirsium%20arvense.%20New%20Phytol.205(2),%20816%E2%80%93827%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Checa, J., Blanco, M. N., Moreno, G., Alvarado, P. &amp; Esquivel, E. Amplistroma erinaceum, a new species and its anamorph from Panama. <em>Mycological Progress</em>. <strong>13</strong>(2), 277–283 (2013).</cite> [<a href="https://scholar.google.com/scholar_lookup?Checa,%20J.,%20Blanco,%20M.%20N.,%20Moreno,%20G.,%20Alvarado,%20P.%20&amp;%20Esquivel,%20E.%20Amplistroma%20erinaceum,%20a%20new%20species%20and%20its%20anamorph%20from%20Panama.%20Mycological%20Progress.%2013(2),%20277%E2%80%93283%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Parent-Michaud, M. et al. Draft genome sequences of azole-resistant and azole-susceptible <em>Aspergillus turcosus</em> clinical isolates recovered from Bronchoalveolar lavage fluid samples. <em>Microbiol. Resour. Announc</em>. <strong>8</strong>(1). 10.1128/mra.01446-18 (2019).</cite> [<a href="https://doi.org/10.1128/MRA.01446-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6318367/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30637396/" class="usa-link">PubMed</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Drula, E. et al. The carbohydrate-active enzyme database: functions and literature. <em>Nucleic Acids Res.</em><strong>50</strong>(D1), D571–D7 (2022).
</cite> [<a href="https://doi.org/10.1093/nar/gkab1045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34850161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Drula,%20E.%20et%20al.%20The%20carbohydrate-active%20enzyme%20database:%20functions%20and%20literature.%20Nucleic%20Acids%20Res.50(D1),%20D571%E2%80%93D7%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Frisvad, J. &amp; Samson, R. Polyphasic taxonomy of <em>Penicillium</em> subgenus <em>Penicillium</em>. A guide to identification of food and airborne terverticillate penicillia and their Mycotoxins. <em>Stud. Mycol.</em><strong>2004</strong>(49), 1–174 (2004).</cite> [<a href="https://scholar.google.com/scholar_lookup?Frisvad,%20J.%20&amp;%20Samson,%20R.%20Polyphasic%20taxonomy%20of%20Penicillium%20subgenus%20Penicillium.%20A%20guide%20to%20identification%20of%20food%20and%20airborne%20terverticillate%20penicillia%20and%20their%20Mycotoxins.%20Stud.%20Mycol.2004(49),%201%E2%80%93174%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Perrone, G. &amp; Susca, A. <em>Penicillium</em> species and their associated Mycotoxins. In <em>Mycotoxigenic Fungi: Methods and Protocols</em> (eds Moretti, A. &amp; Susca, A.) 107–119 (Springer New York, 2017).</cite> [<a href="https://doi.org/10.1007/978-1-4939-6707-0_5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27924532/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Perrone,%20G.%20&amp;%20Susca,%20A.%20Penicillium%20species%20and%20their%20associated%20Mycotoxins.%20In%20Mycotoxigenic%20Fungi:%20Methods%20and%20Protocols%20(eds%20Moretti,%20A.%20&amp;%20Susca,%20A.)%20107%E2%80%93119%20(Springer%20New%20York,%202017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Samuels, G. J. <em>Trichoderma</em>: systematics, the sexual state, and ecology. <em>Phytopathology</em><strong>96</strong>(2), 195–206 (2006).
</cite> [<a href="https://doi.org/10.1094/PHYTO-96-0195" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18943925/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Samuels,%20G.%20J.%20Trichoderma:%20systematics,%20the%20sexual%20state,%20and%20ecology.%20Phytopathology96(2),%20195%E2%80%93206%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Houbraken, J. et al. Classification of aspergillus, penicillium, Talaromyces and related genera (Eurotiales): an overview of families, genera, subgenera, sections, series and species. <em>Stud. Mycol.</em><strong>95</strong>, 5–169 (2020).
</cite> [<a href="https://doi.org/10.1016/j.simyco.2020.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7426331/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32855739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Houbraken,%20J.%20et%20al.%20Classification%20of%20aspergillus,%20penicillium,%20Talaromyces%20and%20related%20genera%20(Eurotiales):%20an%20overview%20of%20families,%20genera,%20subgenera,%20sections,%20series%20and%20species.%20Stud.%20Mycol.95,%205%E2%80%93169%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Gao, H. et al. Bioactive metabolites from acid-tolerant fungi in a Thai Mangrove sediment. <em>Front. Microbiol.</em><strong>11</strong> (2021).</cite> [<a href="https://doi.org/10.3389/fmicb.2020.609952" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7862741/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33552019/" class="usa-link">PubMed</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Punt, M. et al. High sorbic acid resistance of <em>Penicillium roqueforti</em> is mediated by the SORBUS gene cluster. <em>PLoS Genet.</em><strong>18</strong>(6), e1010086 (2022).
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1010086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9200314/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35704633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Punt,%20M.%20et%20al.%20High%20sorbic%20acid%20resistance%20of%20Penicillium%20roqueforti%20is%20mediated%20by%20the%20SORBUS%20gene%20cluster.%20PLoS%20Genet.18(6),%20e1010086%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Okano, T., Kobayashi, N., Izawa, K., Yoshinari, T. &amp; Sugita-Konishi, Y. Whole genome analysis revealed the genes responsible for Citreoviridin biosynthesis in <em>Penicillium citreonigrum</em>. <em>Toxins</em>. <strong>12</strong>(2) (2020).</cite> [<a href="https://doi.org/10.3390/toxins12020125" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7077241/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32075322/" class="usa-link">PubMed</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Ridout, M., Houbraken, J. &amp; Newcombe, G. Xerotolerance of <em>Penicillium</em> and <em>Phialocephala</em> fungi, dominant taxa of fine lateral roots of Woody plants in the intermountain Pacific northwest, USA. <em>Rhizosphere</em><strong>4</strong>, 94–103 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Ridout,%20M.,%20Houbraken,%20J.%20&amp;%20Newcombe,%20G.%20Xerotolerance%20of%20Penicillium%20and%20Phialocephala%20fungi,%20dominant%20taxa%20of%20fine%20lateral%20roots%20of%20Woody%20plants%20in%20the%20intermountain%20Pacific%20northwest,%20USA.%20Rhizosphere4,%2094%E2%80%93103%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Janso, J. E., Bernan, V. S., Greenstein, M., Bugni, T. S. &amp; Ireland, C. M. <em>Penicillium dravuni</em>, a new marine-derived species from an Alga in Fiji. <em>Mycologia</em><strong>97</strong>(2), 444–453 (2005).
</cite> [<a href="https://doi.org/10.3852/mycologia.97.2.444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16396352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Janso,%20J.%20E.,%20Bernan,%20V.%20S.,%20Greenstein,%20M.,%20Bugni,%20T.%20S.%20&amp;%20Ireland,%20C.%20M.%20Penicillium%20dravuni,%20a%20new%20marine-derived%20species%20from%20an%20Alga%20in%20Fiji.%20Mycologia97(2),%20444%E2%80%93453%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Chen, A. J. et al. New Talaromyces species from indoor environments in China. <em>Stud. Mycol.</em><strong>84</strong>(1), 119–144 (2016).
</cite> [<a href="https://doi.org/10.1016/j.simyco.2016.11.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5219591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28070136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20A.%20J.%20et%20al.%20New%20Talaromyces%20species%20from%20indoor%20environments%20in%20China.%20Stud.%20Mycol.84(1),%20119%E2%80%93144%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Arnold, F. H., Wintrode, P. L., Miyazaki, K. &amp; Gershenson, A. How enzymes adapt: lessons from directed evolution. <em>Trends Biochem. Sci.</em><strong>26</strong>(2), 100–106 (2001).
</cite> [<a href="https://doi.org/10.1016/s0968-0004(00)01755-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11166567/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arnold,%20F.%20H.,%20Wintrode,%20P.%20L.,%20Miyazaki,%20K.%20&amp;%20Gershenson,%20A.%20How%20enzymes%20adapt:%20lessons%20from%20directed%20evolution.%20Trends%20Biochem.%20Sci.26(2),%20100%E2%80%93106%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Nigam, P. &amp; Prabhu, K. A. Thermal activation and stability of cellulases derived from two Basidiomycetes. <em>Biotechnol. Lett.</em><strong>10</strong>(12), 919–920 (1988).</cite> [<a href="https://scholar.google.com/scholar_lookup?Nigam,%20P.%20&amp;%20Prabhu,%20K.%20A.%20Thermal%20activation%20and%20stability%20of%20cellulases%20derived%20from%20two%20Basidiomycetes.%20Biotechnol.%20Lett.10(12),%20919%E2%80%93920%20(1988)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Prasetyo, J., Sumita, S., Okuda, N. &amp; Park, E. Y. Response of cellulase activity in pH-controlled cultures of the filamentous fungus acremonium cellulolyticus. <em>Appl. Biochem. Biotechnol.</em><strong>162</strong>(1), 52–61 (2010).
</cite> [<a href="https://doi.org/10.1007/s12010-009-8826-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2848724/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19882113/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Prasetyo,%20J.,%20Sumita,%20S.,%20Okuda,%20N.%20&amp;%20Park,%20E.%20Y.%20Response%20of%20cellulase%20activity%20in%20pH-controlled%20cultures%20of%20the%20filamentous%20fungus%20acremonium%20cellulolyticus.%20Appl.%20Biochem.%20Biotechnol.162(1),%2052%E2%80%9361%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Hurst, P. L., Nielsen, J., Sullivan, P. A. &amp; Shepherd, M. G. Purification and properties of a cellulase from Aspergillus Niger. <em>Biochem. J.</em><strong>165</strong>(1), 33–41 (1977).
</cite> [<a href="https://doi.org/10.1042/bj1650033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1164865/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19015/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hurst,%20P.%20L.,%20Nielsen,%20J.,%20Sullivan,%20P.%20A.%20&amp;%20Shepherd,%20M.%20G.%20Purification%20and%20properties%20of%20a%20cellulase%20from%20Aspergillus%20Niger.%20Biochem.%20J.165(1),%2033%E2%80%9341%20(1977)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Okada, G. Cellulase of Aspergillus niger. In<em>Methods Enzymol.</em> vol. 160, 259–264 (Academic, 1988).</cite>
</li>
<li id="CR48">
<span class="label">48.</span><cite>Schuster, E., Dunn-Coleman, N., Frisvad, J. C. &amp; Van Dijck, P. W. On the safety of <em>Aspergillus niger</em>—a review. <em>Appl. Microbiol. Biotechnol.</em><strong>59</strong>(4–5), 426–435 (2002).
</cite> [<a href="https://doi.org/10.1007/s00253-002-1032-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12172605/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schuster,%20E.,%20Dunn-Coleman,%20N.,%20Frisvad,%20J.%20C.%20&amp;%20Van%20Dijck,%20P.%20W.%20On%20the%20safety%20of%20Aspergillus%20niger%E2%80%94a%20review.%20Appl.%20Microbiol.%20Biotechnol.59(4%E2%80%935),%20426%E2%80%93435%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>García-Fernández, A. J., Espín, S., Gómez-Ramírez, P. &amp; Martínez-López, E. Oxalates. In <em>Encyclopedia of Toxicology</em> (Third Edition) (ed. Wexler, P.) 730–734 (Academic Press, 2014).</cite>
</li>
<li id="CR50">
<span class="label">50.</span><cite>Prasad, R. &amp; Shivay, Y. S. Oxalic acid/oxalates in plants: from self-defence to phytoremediation. <em>Curr. Sci.</em><strong>112</strong>(8), 1665–1667 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Prasad,%20R.%20&amp;%20Shivay,%20Y.%20S.%20Oxalic%20acid/oxalates%20in%20plants:%20from%20self-defence%20to%20phytoremediation.%20Curr.%20Sci.112(8),%201665%E2%80%931667%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Gadd, G. M. Fungal production of citric and oxalic acid: importance in metal speciation, physiology and biogeochemical processes. <em>Adv. Microb. Physiol.</em><strong>41</strong>, 47–92 (1999).
</cite> [<a href="https://doi.org/10.1016/s0065-2911(08)60165-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10500844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gadd,%20G.%20M.%20Fungal%20production%20of%20citric%20and%20oxalic%20acid:%20importance%20in%20metal%20speciation,%20physiology%20and%20biogeochemical%20processes.%20Adv.%20Microb.%20Physiol.41,%2047%E2%80%9392%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Dutton, M. V. &amp; Evans, C. S. Oxalate production by fungi: its role in pathogenicity and ecology in the soil environment. <em>Can. J. Microbiol.</em><strong>42</strong>(9), 881–895 (1996).</cite> [<a href="https://scholar.google.com/scholar_lookup?Dutton,%20M.%20V.%20&amp;%20Evans,%20C.%20S.%20Oxalate%20production%20by%20fungi:%20its%20role%20in%20pathogenicity%20and%20ecology%20in%20the%20soil%20environment.%20Can.%20J.%20Microbiol.42(9),%20881%E2%80%93895%20(1996)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Lynd, L. R., Jin, H., Michels, J. G., Wyman, C. E. &amp; Dale, B. Bioenergy: background, potential, and policy. (2003).</cite>
</li>
<li id="CR54">
<span class="label">54.</span><cite>Kim, S. &amp; Dale, B. E. Global potential bioethanol production from wasted crops and crop residues. <em>Biomass Bioenerg.</em><strong>26</strong>(4), 361–375 (2004).</cite> [<a href="https://scholar.google.com/scholar_lookup?Kim,%20S.%20&amp;%20Dale,%20B.%20E.%20Global%20potential%20bioethanol%20production%20from%20wasted%20crops%20and%20crop%20residues.%20Biomass%20Bioenerg.26(4),%20361%E2%80%93375%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Cuong, T. T. et al. Renewable energy from biomass surplus resource: potential of power generation from rice straw in Vietnam. <em>Sci. Rep.</em><strong>11</strong>(1), 792 (2021).
</cite> [<a href="https://doi.org/10.1038/s41598-020-80678-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7804265/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33436988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cuong,%20T.%20T.%20et%20al.%20Renewable%20energy%20from%20biomass%20surplus%20resource:%20potential%20of%20power%20generation%20from%20rice%20straw%20in%20Vietnam.%20Sci.%20Rep.11(1),%20792%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Le, H., Anh, T. &amp; Hung, N. Air pollutants estimated from rice straw open burning in Hanoi. <em>J. Agric. Sci. Technol.</em><strong>5</strong>, 101–107 (2017).</cite> [<a href="https://scholar.google.com/scholar_lookup?Le,%20H.,%20Anh,%20T.%20&amp;%20Hung,%20N.%20Air%20pollutants%20estimated%20from%20rice%20straw%20open%20burning%20in%20Hanoi.%20J.%20Agric.%20Sci.%20Technol.5,%20101%E2%80%93107%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Caputo, F. et al. Understanding the impact of steam pretreatment severity on cellulose ultrastructure, recalcitrance, and hydrolyzability of Norway spruce. <em>Biomass Convers. Biorefinery</em> (2022).</cite>
</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365195/bin/41598_2025_6321_MOESM1_ESM.py" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (8.4KB, py) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365195/bin/41598_2025_6321_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (843.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365195/bin/41598_2025_6321_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (40.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365195/bin/41598_2025_6321_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4</a><sup> (88.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365195/bin/41598_2025_6321_MOESM5_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5</a><sup> (48.8KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data are contained within the manuscript or associated online supplemental information. Raw data used for calculations are available on reasonable request from the corresponding author. The ITS sequences obtained for representatives of all species and genetic groups have been deposited in GenBank with accession numbers from <a href="https://www.ncbi.nlm.nih.gov/nuccore/?term=PP646895:PP646950%5Bpacc%5D" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PP646895 to PP646950</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Scientific Reports are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41598-025-06321-1"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41598_2025_Article_6321.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365195/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365195/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365195%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365195/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365195/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365195/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830373/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365195/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830373/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365195/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365195/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="8HUGfGosbtRK1N9jpD99nnl8nJrGFy3XdXEgPXBnpCcORFyUdm5UpAEg26Ov3X0T">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
