
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Multi-scale and multi-context interpretable mapping of cell states across heterogeneous spatial samples - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A9518AF2A1B305A9510045CB9474.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370880/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="Multi-scale and multi-context interpretable mapping of cell states across heterogeneous spatial samples">
<meta name="citation_author" content="Patrick C N Martin">
<meta name="citation_author_institution" content="Department of Computational Biomedicine, Cedars-Sinai Medical Center, Hollywood, CA USA">
<meta name="citation_author" content="Wenqi Wang">
<meta name="citation_author_institution" content="Biotech Research and Innovation Centre (BRIC), University of Copenhagen, Copenhagen, Denmark">
<meta name="citation_author" content="Hyobin Kim">
<meta name="citation_author_institution" content="Department of Computational Biomedicine, Cedars-Sinai Medical Center, Hollywood, CA USA">
<meta name="citation_author" content="Henrietta Holze">
<meta name="citation_author_institution" content="Biotech Research and Innovation Centre (BRIC), University of Copenhagen, Copenhagen, Denmark">
<meta name="citation_author" content="Paul B Fisher">
<meta name="citation_author_institution" content="Massey Comprehensive Cancer Center, Virginia Commonwealth University, Richmond, VA USA">
<meta name="citation_author_institution" content="VCU Institute of Molecular Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA">
<meta name="citation_author_institution" content="Department of Cellular, Molecular and Genetic Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA">
<meta name="citation_author" content="Arturo P Saavedra">
<meta name="citation_author_institution" content="Department of Dermatology, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA">
<meta name="citation_author" content="Robert A Winn">
<meta name="citation_author_institution" content="Massey Comprehensive Cancer Center, Virginia Commonwealth University, Richmond, VA USA">
<meta name="citation_author_institution" content="Department of Surgery, Virginia Commonwealth University School of Medicine, Richmond, VA USA">
<meta name="citation_author_institution" content="Division of Pulmonary Disease and Critical Care Medicine, Department of Internal Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA">
<meta name="citation_author" content="Esha Madan">
<meta name="citation_author_institution" content="Massey Comprehensive Cancer Center, Virginia Commonwealth University, Richmond, VA USA">
<meta name="citation_author_institution" content="VCU Institute of Molecular Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA">
<meta name="citation_author_institution" content="Department of Cellular, Molecular and Genetic Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA">
<meta name="citation_author_institution" content="Department of Surgery, Virginia Commonwealth University School of Medicine, Richmond, VA USA">
<meta name="citation_author" content="Rajan Gogna">
<meta name="citation_author_institution" content="Massey Comprehensive Cancer Center, Virginia Commonwealth University, Richmond, VA USA">
<meta name="citation_author_institution" content="VCU Institute of Molecular Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA">
<meta name="citation_author_institution" content="Department of Cellular, Molecular and Genetic Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA">
<meta name="citation_author_institution" content="Department of Surgery, Virginia Commonwealth University School of Medicine, Richmond, VA USA">
<meta name="citation_author" content="Kyoung Jae Won">
<meta name="citation_author_institution" content="Department of Computational Biomedicine, Cedars-Sinai Medical Center, Hollywood, CA USA">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7814">
<meta name="citation_doi" content="10.1038/s41467-025-62782-y">
<meta name="citation_pmid" content="40841370">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370880/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370880/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370880/pdf/41467_2025_Article_62782.pdf">
<meta name="description" content="There is a growing demand for methods that can effectively align and compare spatial data in the absence of obvious visual correspondence. To address this challenge, we developed an interpretable cell mapping strategy based on solving a Linear ...">
<meta name="og:title" content="Multi-scale and multi-context interpretable mapping of cell states across heterogeneous spatial samples">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="There is a growing demand for methods that can effectively align and compare spatial data in the absence of obvious visual correspondence. To address this challenge, we developed an interpretable cell mapping strategy based on solving a Linear ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370880/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370880">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-62782-y"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_62782.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370880%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370880/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370880/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370880/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 21;16:7814. doi: <a href="https://doi.org/10.1038/s41467-025-62782-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-62782-y</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Multi-scale and multi-context interpretable mapping of cell states across heterogeneous spatial samples</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Martin%20PCN%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Patrick C N Martin</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Patrick C N Martin</span></h3>
<div class="p">
<sup>1</sup>Department of Computational Biomedicine, Cedars-Sinai Medical Center, Hollywood, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Martin%20PCN%22%5BAuthor%5D" class="usa-link"><span class="name western">Patrick C N Martin</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Wenqi Wang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Wenqi Wang</span></h3>
<div class="p">
<sup>2</sup>Biotech Research and Innovation Centre (BRIC), University of Copenhagen, Copenhagen, Denmark </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Wenqi Wang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Hyobin Kim</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Hyobin Kim</span></h3>
<div class="p">
<sup>1</sup>Department of Computational Biomedicine, Cedars-Sinai Medical Center, Hollywood, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kim%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hyobin Kim</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Holze%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Henrietta Holze</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Henrietta Holze</span></h3>
<div class="p">
<sup>2</sup>Biotech Research and Innovation Centre (BRIC), University of Copenhagen, Copenhagen, Denmark </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Holze%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Henrietta Holze</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fisher%20PB%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Paul B Fisher</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Paul B Fisher</span></h3>
<div class="p">
<sup>3</sup>Massey Comprehensive Cancer Center, Virginia Commonwealth University, Richmond, VA USA </div>
<div class="p">
<sup>4</sup>VCU Institute of Molecular Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA </div>
<div class="p">
<sup>5</sup>Department of Cellular, Molecular and Genetic Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fisher%20PB%22%5BAuthor%5D" class="usa-link"><span class="name western">Paul B Fisher</span></a>
</div>
</div>
<sup>3,</sup><sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saavedra%20AP%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Arturo P Saavedra</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Arturo P Saavedra</span></h3>
<div class="p">
<sup>6</sup>Department of Dermatology, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Saavedra%20AP%22%5BAuthor%5D" class="usa-link"><span class="name western">Arturo P Saavedra</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Winn%20RA%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Robert A Winn</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Robert A Winn</span></h3>
<div class="p">
<sup>3</sup>Massey Comprehensive Cancer Center, Virginia Commonwealth University, Richmond, VA USA </div>
<div class="p">
<sup>7</sup>Department of Surgery, Virginia Commonwealth University School of Medicine, Richmond, VA USA </div>
<div class="p">
<sup>8</sup>Division of Pulmonary Disease and Critical Care Medicine, Department of Internal Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Winn%20RA%22%5BAuthor%5D" class="usa-link"><span class="name western">Robert A Winn</span></a>
</div>
</div>
<sup>3,</sup><sup>7,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Madan%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Esha Madan</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Esha Madan</span></h3>
<div class="p">
<sup>3</sup>Massey Comprehensive Cancer Center, Virginia Commonwealth University, Richmond, VA USA </div>
<div class="p">
<sup>4</sup>VCU Institute of Molecular Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA </div>
<div class="p">
<sup>5</sup>Department of Cellular, Molecular and Genetic Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA </div>
<div class="p">
<sup>7</sup>Department of Surgery, Virginia Commonwealth University School of Medicine, Richmond, VA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Madan%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Esha Madan</span></a>
</div>
</div>
<sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gogna%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Rajan Gogna</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Rajan Gogna</span></h3>
<div class="p">
<sup>3</sup>Massey Comprehensive Cancer Center, Virginia Commonwealth University, Richmond, VA USA </div>
<div class="p">
<sup>4</sup>VCU Institute of Molecular Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA </div>
<div class="p">
<sup>5</sup>Department of Cellular, Molecular and Genetic Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA </div>
<div class="p">
<sup>7</sup>Department of Surgery, Virginia Commonwealth University School of Medicine, Richmond, VA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gogna%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rajan Gogna</span></a>
</div>
</div>
<sup>3,</sup><sup>4,</sup><sup>5,</sup><sup>7,</sup><sup>âœ‰</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Won%20KJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Kyoung Jae Won</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Kyoung Jae Won</span></h3>
<div class="p">
<sup>1</sup>Department of Computational Biomedicine, Cedars-Sinai Medical Center, Hollywood, CA USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Won%20KJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Kyoung Jae Won</span></a>
</div>
</div>
<sup>1,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Computational Biomedicine, Cedars-Sinai Medical Center, Hollywood, CA USA </div>
<div id="Aff2">
<sup>2</sup>Biotech Research and Innovation Centre (BRIC), University of Copenhagen, Copenhagen, Denmark </div>
<div id="Aff3">
<sup>3</sup>Massey Comprehensive Cancer Center, Virginia Commonwealth University, Richmond, VA USA </div>
<div id="Aff4">
<sup>4</sup>VCU Institute of Molecular Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA </div>
<div id="Aff5">
<sup>5</sup>Department of Cellular, Molecular and Genetic Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA </div>
<div id="Aff6">
<sup>6</sup>Department of Dermatology, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA </div>
<div id="Aff7">
<sup>7</sup>Department of Surgery, Virginia Commonwealth University School of Medicine, Richmond, VA USA </div>
<div id="Aff8">
<sup>8</sup>Division of Pulmonary Disease and Critical Care Medicine, Department of Internal Medicine, VCU School of Medicine, Virginia Commonwealth University, Richmond, VA USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Oct 8; Accepted 2025 Jul 28; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370880Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841370/" class="usa-link">40841370</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">There is a growing demand for methods that can effectively align and compare spatial data in the absence of obvious visual correspondence. To address this challenge, we developed an interpretable cell mapping strategy based on solving a Linear Assignment Problem (LAP) where the total cost is computed by considering cells and their niches. We demonstrate that our approach outperforms other methods at capturing the spatial context of cells in synthetic and real data sets. The flexibility of our implementation enhances the interpretability of mapping and allows for accurate cell mapping across samples, technologies, resolutions, developmental and regenerative time. We show spatiotemporal decoupling of cells during development and patient level sub-populations in In Situ Mass Cytometry (IMC) cancer data sets. Our interpretable mapping approach facilitates systemic comparison and analysis of heterogeneous spatial data. We provide a flexible framework for researchers to tailor their analysis to the specific biological and research context.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Data integration, Computational models, Software, Data mining</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">The alignment of heterogeneous spatial samples has become a growing challenge. Here, authors present a multi-scale, multi-context, and interpretable mapping strategy to map cells across space, time, and disease.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">From the initial discovery of cells through early microscopes in the 17th century, we have come a long way in our understanding of cellular organization<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Today, spatial transcriptomics (ST) has become an increasingly popular means of probing cellular organization in relation to cellular identity<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>â€“<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a></sup>. ST measures the gene expression profiles of cells as well as their location within a tissue. For clarity, we will refer to spatial spots, barcodes, or indices simply as cells. Taken together, ST methods - in their varying flavors â€“ can highlight spatially resolved gene expression patterns<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>â€“<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. They can demonstrate how tumors interact with their microenvironment and how disease can be understood from a spatial perspective<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>â€“<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. They can provide insights into cellular communication through ligand-receptor dynamics as well as cell-to-cell contact-triggered gene expression modulations<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>â€“<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Importantly, ST emphasized how cellular identity should be viewed through the lens of spatial context<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>.</p>
<p id="Par4">With the increasing availability and breadth of ST technologies, it comes as no surprise that they have become part of the biomedical research arsenal. It is still difficult and expensive to obtain high-quality spatial data, especially from human patients. Therefore, there is a need to maximize the use of the spatial data deposited in the public domain. However, datasets that are already available in a biological condition of interest might have been produced using different methodologies or technologies. Moreover, in a clinical setting where a researcher might be interested in following the evolution of cells and their interactions across disease progression or as a response to treatment, it is difficult to expect tissues to share similar structures. And yet, we have seen the emergence of 3D spatial stacks<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> taken from adjacent tissues and, in parallel, a plethora of tools allowing the alignment of these tissue slices, including PASTE<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, GPSA<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, or STalign<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> have been developed. The assumption made by these tools is that adjacent sections share sufficient structural similarity to locally and differentially deform tissue sections to match the neighboring section. Importantly, these methods do not fold, break, or scramble the cells present in the tissue; rather, they displace them by maintaining their relative position to each other. But the irregular nature of tumors across patients and conditions restricts their use to highly structured tissues. As such, the question remains: how do we compare the spatial context of individual cells when tissue structures do not match across samples or where sequential sampling is simply impossible?</p>
<p id="Par5">Mapping single-cell data to a spatial assay based on the transcriptome may suggest an alternative solution to align two or more ST assays based on their transcriptome<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. For instance, Tangram uses a deep learning approach to match sc/snRNA-seq to their estimated spatial location by maximizing the spatial correlation between scRNA and spatial data<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. CytoSpace solves a linear assignment problem (LAP) to match single cells to spatial locations by accounting for cell type proportions in low-resolution Visium spots<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. In conjunction with single-cell labels, CytoSpace can also map to high-resolution spatial data. Along the same lines, methods designed to integrate single-cell data, such as Scanorama<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> can find the closest related cells in their integrated latent space. However, data mapping or integration without spatial and biological context can lose the information related to tissue or cellular microenvironment. Algorithms such as the Spatial-linked alignment tool (SLAT) provide a solution to spatial mapping by adopting a graph adversarial matching strategy<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. Interestingly, SLAT provides an integrated latent space, making it a spatially aware data integration method. SLAT represents cells as neighborhood graphs and applies a graph convolutional neural network to generate its embeddings. This strategy can reduce both context flexibility and interpretability. While SLAT considers the spatial context of cells across samples, it has not been tested systematically for tissues whose structure is highly dissimilar. More importantly, current mapping strategies cannot be applied to a systemic, flexible, and interpretable comparison of hundreds of samples.</p>
<p id="Par6">To address the challenge of mapping cells between structurally heterogeneous tissues, we developed an approach to map cells across 2 or more samples when structural heterogeneity impedes spatial alignment strategies. We map cells using a Linear Assignment Solver where the cost matrix is constructed by explicitly considering spatial or biological context. Specifically, we designed the cost matrix to minimize the pair-wise summation of multi-scale and multi-context spatial similarity matrices, including cell similarity, niche similarity, and spatial territory similarity (see Methods). Explicit use of context-specific matrices increases the interpretability of cell mapping results and provides sufficient flexibility to map samples across technology or collected over time. Importantly, our approach enables customized and interpretable analyses tailored to the specific research needs of individual researchers or datasets. We highlight the performance of our approach using synthetic datasets in a variety of scenarios with increasing complexity. We demonstrate how the performance observed in synthetic data can be translated to real biological datasets, including seqFISH mouse embryo<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>, Stereo-seq mouse embryo<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, and Slide-seq V2 mouse hippocampus<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Vesalius accurately maps cells with their spatial context and outperforms other approaches. We demonstrate how our flexible framework can effectively map cells across datasets, technologies (seqFISH to Stereo-seq), and resolutions (VisiumHD to Visium). We also map cells across developmental and regenerative time in high-resolution Stereo-seq<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup> data. This allowed us to highlight genes involved in brain development and regeneration, which were only present in a limited subset of cells. Additionally, the cost of cellular mapping provides an interpretable and universally comparable metric to cluster in situ mass cytometry (IMC) breast cancer patient samples at the population level. The clustering results demonstrated how cellular context can recapitulate broad clinical metrics. Our cell mapping strategy grants us the ability to investigate biological differences between samples with unprecedented flexibility and interpretability.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Mapping cells and their context across samples</h3>
<p id="Par7">With the increased availability of ST datasets, we aim to provide a flexible framework to compare spatial data by mapping a query dataset onto a reference dataset across conditions, technologies, and sections (Fig.Â <a href="#Fig1" class="usa-link">1a</a>). To achieve this, we formulate the matching of cell pairs as a LAP where the goal is to match â€œa workerâ€ to â€œa tasksâ€ which minimizes the overall cost (Fig.Â <a href="#Fig1" class="usa-link">1b</a>). To solve the assignment problem we used the Jonker-Volgenant algorithm<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. In this instance, a â€œworkerâ€ is a cell from the query data, whereas the â€œtaskâ€ is a cell from the reference data. The overall cost of mapping cells is defined by a pair-wise summation of score matrices across biological features (see â€œMethodsâ€ section - Fig.Â <a href="#Fig1" class="usa-link">1c</a>). We account for transcriptional similarities between cells, their niches, their spatial tissue territory, cell type labels, and the cell type composition of their niche. In addition, we offer a straightforward method to add custom and context-specific matrices. The total cost matrix can be constructed from any combination of these biological features. For instance, while cell type labels can be used, they are not strictly required to map cells between samples. Our algorithm has been included in the Vesalius R package â€“ a tool designed to retrieve spatial territories by applying image processing methodologies to spatial omics data<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup> (Fig.Â <a href="#Fig1" class="usa-link">1d</a>). The territory similarity matrix is computed using Vesaliusâ€™s tissue territory detection capabilities<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. The new and improved functionalities of Vesalius provide a framework for spatial mapping of cells across time, across technologies, and clustering hundreds of spatial samples.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. Overview of Vesaliusâ€™s cell state mapping strategy.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370880_41467_2025_62782_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5561/12370880/e0d348a016ba/41467_2025_62782_Fig1_HTML.jpg" loading="lazy" id="d33e447" height="649" width="799" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Samples across conditions, technologies, and sections are likely to present a heterogeneous structure, and yet, to capitalize on the growing wealth of spatial omics data, it is crucial to accurately map cellular context across conditions. <strong>b</strong> To achieve cell mapping in heterogeneous spatial samples, Vesalius solves an LAP, which aims to minimize the overall cost. To account for spatial context during mapping, Vesalius leverages a multitude of biological features. <strong>c</strong> The total cost matrix can be constructed from a pair-wise summation of reciprocal similarity scores, including cell similarity, label similarity, niche similarity, composition similarity, territory similarity, and even custom similarity scores. <strong>d</strong> Vesalius maps cells across samples, across time, and across technologies. The total cost also provides a useful metric to define patient-level spatial similarities and allow for sample clustering. Source data are provided as a Source Data file.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Benchmarking mapping performance in synthetic spatial data</h3>
<p id="Par8">To demonstrate the effectiveness of our mapping strategy, we elected to use synthetic spatial data since this will provide a robust ground truth (See â€œMethodsâ€ section). Specifically, we aimed to demonstrate our ability to accurately map cells across samples along with their spatial context. First, we generated 5 synthetic spatial regimes (<em>circle</em>, <em>layer, dropped, random one cell, random two cell</em>) that mimic the complexity of biological scenarios (Supplementary Figs.Â <a href="#MOESM1" class="usa-link">S1</a>â€“<a href="#MOESM1" class="usa-link">S5</a>). For each regime, we generated 12 samples where we aimed to map a query dataset onto a reference data for a total of 132 mapping events (excluding 12 self-mapping events) (Supplementary Figs.Â <a href="#MOESM1" class="usa-link">S1</a>â€“<a href="#MOESM1" class="usa-link">S5</a>).</p>
<p id="Par9">Since Vesalius provides a flexible framework to construct the total mapping cost, we compared the performance of Vesalius across 14 different cost matrices in addition to two scRNA-to-spatial mapping tools (CytoSpace<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> and Tangram<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>), spatial alignment tools (PASTE<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup> and GPSA<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>), and scRNA/Spatial data integration tools such as Scanorama<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> and SLAT<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. The cost matrix combinations were built using cell similarity (fâ€‰=â€‰feature), niche similarity (nâ€‰=â€‰niche), territory similarity (tâ€‰=â€‰ territory), niche cell type composition (câ€‰=â€‰composition), cell type labels (yâ€‰=â€‰cell type labels), or a combination of these metrics.</p>
<p id="Par10">To measure the effectiveness of each tool in mapping cells across samples, we computed an Adjusted Rand Index (ARI)<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> between mapped cell labels. In parallel, we measured each toolâ€™s ability to capture spatial context by adding an â€œinteractionâ€ label to our datasets (see â€œMethodsâ€ section). This label concatenates the cell type labels of the k-nearest neighbors (<em>k</em>â€‰=â€‰6) of a center cell. Using these labels, we computed a Jaccard index (JI) to ascertain how similar the neighborhoods are in terms of cell type composition.</p>
<p id="Par11">Vesalius demonstrated performance satisfying both cell type score (ARI) and interaction score (JI) in these tests. When measuring the ability to recover the identity of mapped cells in the <em>circle</em> regime, for instance, Vesalius showed comparable performance with SLAT, Scanorama, and CytoSpace (Fig.Â <a href="#Fig2" class="usa-link">2a</a>) in ARI. The perfect score of CytoSpace was expected, as it only maps cells of the same cell type. With its requirement of shared cell type labels between datasets, CytoSpace was incapable of mapping samples when not all cells were shared (121 were mapped out of 132). This was particularly evident in the <em>dropped</em> regime, where CytoSpace only mapped 7 samples out of 132 (see Supplementary Fig.Â <a href="#MOESM1" class="usa-link">S7</a>). We show the mapping performance for all other regimes in Supplementary Figs.Â <a href="#MOESM1" class="usa-link">S6â€“S9</a>. The performance of CytoSpace dropped dramatically when cell labels were removed (<em>CytoSpace_noLab</em> in Fig.Â <a href="#Fig2" class="usa-link">2</a>, Supplementary Figs.Â <a href="#MOESM1" class="usa-link">S6</a>â€“<a href="#MOESM1" class="usa-link">S9</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Benchmarking mapping performance in synthetic spatial data â€“ circle regime.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370880_41467_2025_62782_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5561/12370880/5fd5979e5b03/41467_2025_62782_Fig2_HTML.jpg" loading="lazy" id="d33e583" height="952" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>We provide a Legend Key on the figure describing the nomenclature of Vesalius cost matrices as well as the meaning of acronyms. <strong>a</strong> Performance comparison in recovering cell types across samples using an ARI in synthetic spatial data. We compared Vesalius using 14 different cost matrices to SLAT, CytoSpace, CytoSpace with no cell type labels (noLab), Tangram, Scanorama, PASTE, and GPSA. The number of samples (<em>n</em>) used in each box plot is included in the axis labels. The box shows the interquartile range (IQRâ€‰=â€‰Q3â€‰âˆ’â€‰Q1), with a line at the median, and whiskers extending to the most extreme values within 1.5Ã— IQR. CytoSpace obtains a perfect score since it only maps cells from the same cell type. <strong>b</strong> JI of cell interactions. To account for the spatial context of cells, we computed the JI between cell type labels in the neighborhood of mapped cells (<em>k</em>â€‰=â€‰6). <strong>c</strong> Heat map representing <em>p</em>-values after performing a two-sided Wilcoxon rank sum test between tools on ARI scores. <strong>d</strong> Heat map representing <em>p</em>-values after performing a two-sided Wilcoxon rank sum test between tools on JI scores. <strong>e</strong> Example mapping event in the circle regime. In the circle regime, there are 5 randomly sized and randomly placed territories in a background territory. Each circle can contain 2 different cell types. We show mapping events for 3 Vesalius cost matrices (<em>fncty</em>, <em>fnct</em>, <em>fnt</em>). Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par12">If we consider each toolâ€™s ability to map spatial context, we see that Vesalius outperforms all other tools when including the cellular context to construct the cost matrix. Unsurprisingly, including neighborhood composition strongly improved Vesaliusâ€™s performance. It is of note that CytoSpace (with true labels) and Scanorama excelled at finding the right cell type but failed at recovering context, falling behind Vesalius even when it did not include cell composition (e.g., <em>n</em>, <em>fn</em>, <em>nt</em> cost matrices â€“ legend key shown in figure legend). We show pair-wise statistical comparison (Wilcoxon Rank Sum test- see methods) between each tool for ARI scores and JI scores in Fig.Â <a href="#Fig2" class="usa-link">2c</a>, <a href="#Fig2" class="usa-link">d</a>, respectively.</p>
<p id="Par13">We exemplify the performance of each tool by plotting a mapping event in the <em>circle</em> regime (Fig.Â <a href="#Fig2" class="usa-link">2e</a>). We aimed to map cells in the query sample onto the reference datasets (panels in the top left of Fig.Â <a href="#Fig2" class="usa-link">2e</a>). For clarity, we selected only 3 cost matrix combinations for Vesalius. Specifically, we selected <em>fncty</em>, <em>fnct,</em> and <em>fnt</em>. Our results indicate that Vesalius was able to effectively map cells from the query sample onto the reference sample while considering spatial context. We noticed that spatial alignment tools such as PASTE and GPSA failed to recover the structure of the reference sample from the query sample. The task of mapping cells across samples is a distinct task from spatial alignment, and the constraints applied to the samples by PASTE and GPSA are incapable of overcoming these differences. We show the results for all other synthetic datasets in Supplementary Figs.Â <a href="#MOESM1" class="usa-link">S6â€“S9</a>.</p>
<p id="Par14">Finally, we highlighted computational performance (run time and memory usage) when increasing the number of cells from 100 to 200,000 (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">S10</a>) in a synthetic circle <em>regime</em> (See â€œMethodsâ€ section). For each tool, we computed the mean run time and peak RAM usage (Resident Set Size) across 3 runs for each number of cells. We also included a comparison of run time and memory usage across cost matrix combinations for 5000 and 10,000 cells.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Benchmarking mapping performance in real spatial data</h3>
<p id="Par15">We further tested Vesalius using the same 14 cost matrices against other tools (excluding GPSA due to excessive memory requirements) in seqFISH mouse embryo<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>, Stereo-seq mouse embryo<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, and Slide-seq V2 mouse hippocampus<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup> (see â€œMethodsâ€ section). We found that the mapping by Vesalius with all available information (<em>fncty</em>) showed ARI larger than 0.75 for all datasets we tested. This was compared with other approaches, whose performance was substantially compromised in real datasets. Again, the perfect ARI score for CytoSpace was expected as it only maps to cells with the same cell type. When the cell type information is lost (CytoSpace_noLab), the ARI becomes less than 0.3 for all datasets. Due to the requirement of matching cell type, CytoSpace cannot run on the Stereo-seq dataset without removing label information (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">S11</a>). SLAT, which showed good performance for synthetic datasets, showed ARIs around 0.5 and even 0.25 for Slide-seq data. Regardless of the datasets, Vesalius showed the best performance when assessed with JI, especially when using cell composition (<em>c</em>). The JI score decreased when adding other configurations to compute the cost in Vesalius, while ARI, in general, is enhanced, indicating that Vesalius tries to find a balance between cells and spatial context depending on the configuration.</p>
<p id="Par16">We show mapping examples using <em>fncty, fnct,</em> and <em>fnt</em>. The mapping examples of Vesalius also demonstrated a strong resemblance to the reference (Fig.Â <a href="#Fig3" class="usa-link">3c, f</a>, and Supplementary Fig.Â <a href="#MOESM1" class="usa-link">S11c</a>). While SLATâ€™s mapping generally aligned with the reference, it exhibited misannotations in highly heterogeneous brain regions (Fig.Â <a href="#Fig3" class="usa-link">3f</a>). SLAT and Scanorama still perform decently in mapping cells across seqFISH samples but fell behind in capturing the cellular context. In fact, Vesalius, using only cell and niche similarity, performs similarly to SLAT at capturing cellular context. Again, PASTE failed to recover the structure of the reference sample from the query sample as observed in the examples using synthetic data.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Benchmarking mapping performance in real spatial data.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370880_41467_2025_62782_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5561/12370880/72406f004ec0/41467_2025_62782_Fig3_HTML.jpg" loading="lazy" id="d33e728" height="863" width="800" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> ARI of mapped cell type labels between a reference seqFISH mouse embryo dataset and a query seqFISH mouse embryo dataset. We compared Vesalius using 14 different cost matrices to SLAT, CytoSpace, CytoSpace with no cell type labels (noLab), Tangram, Scanorama, and PASTE. In seqFISH data, the addition of cell type labels or composition of cellular neighborhoods improves Vesaliusâ€™s ability to accurately map cells across samples, even outperforming SLAT and Scanorama. <strong>b</strong> JI of cell interactions of the neighborhood (<em>k</em>â€‰=â€‰10). <strong>c</strong> Example mapping event in seqFISH mouse embryo. We show mapping events for 3 Vesalius cost matrices (<em>fncty</em>, <em>fnct</em>, <em>fnt</em>). <strong>d</strong> ARI of mapped cell type labels between a reference Slide-seq V2 mouse hippocampus dataset and a query Slide-seq V2 mouse hippocampus dataset. <strong>e</strong> JI of cell interactions of the neighborhood (<em>k</em>â€‰=â€‰10). <strong>f</strong> Example mapping event in Slide-seq V2 mouse hippocampus. We show mapping events for 3 Vesalius cost matrices (<em>fncty</em>, <em>fnct</em>, <em>fnt)</em>. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par17">Finally, we further investigated the role played by each cost matrix in mapping cells across samples. We first computed a coefficient of variation (CV) on each cost matrix across these 3 biological samples (seqFISH, Stereo-seq, and Slide-seq V2). A higher CV would indicate a higher ability for a cost matrix to discriminate cells during the mapping. We also computed a Proportion of Contribution (POC), which measures the proportion of contribution of each cost matrix to the total cost used for mapping (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">S12</a>). A higher POC is associated with higher JI or correlation. Cell type labels and neighborhood composition showed a higher ability to discriminate cells during mapping, but participated much less in the total mapping cost. On the contrary, territory and niche similarity tended to have a low CV but contributed significantly to the total cost of mapping.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Mapping cells across technologies and resolutions</h3>
<p id="Par18">More often than not, datasets of interest are generated using different technologies, and yet we would still like to mine insights from them. We tackled this challenge using our flexible mapping strategy. First, we mapped high-resolution Stereo-seq mouse embryo<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup> (E9.5) onto image-based seqFISH<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> in the same tissue at a slightly earlier developmental stage (E8.75). While both datasets contain cell type labels, these differed greatly between datasets, as is often the case when using data from different sources. As such, we forwent the use of cell type labels, and we constructed the total cost from cell similarity and niche similarity only. Overall, the broad cell type labels matched well when mapping across technologies (Fig.Â <a href="#Fig4" class="usa-link">4a</a>). We show the mapping of cells to different cell labels in an alluvial plot (Fig.Â <a href="#Fig4" class="usa-link">4b</a>). For instance, Brain and Notochord cells in Stereo-seq are mapped to Forebrain/Midbrain/Hindbrain and Spinal Cord in seqFISH (accuracyâ€‰=â€‰0.901). We show the cell mapping results for Stereo-seq to all 3 seqFISH sections (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">S13</a>) and performed the converse mapping - seqFISH mapped to Stereo-seq (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">S14</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Cell Mapping across technologies and resolutions.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370880_41467_2025_62782_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5561/12370880/cca81144c207/41467_2025_62782_Fig4_HTML.jpg" loading="lazy" id="d33e784" height="595" width="800" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Mapping of Stereo-seq mouse embryo cells onto seqFISH mouse embryo cells shows that Vesalius recovers expected tissue structures across technologies. <strong>b</strong> Alluvial plot demonstrating where each mapped cell type in the query (Stereo-seq) maps to in the reference (seqFISH). <strong>c</strong> Mapping VisiumHD mouse brain onto Visium mouse brain with the tissue territories detected by Vesalius in the reference, the query, as well as the mapped cells with a jitter added to the coordinates. We recover the overall structure of the mouse brain, including the Dentate gyrus and the Cornus Amonis fields (CA1 &amp; CA2 â€“ annotated on figure). <strong>d</strong> The mapped p-values after performing a Chi-squared test between cell type frequencies in Visium (after cell type deconvolution) and the mapped cell types taken from VisiumHD. A discretized p-value map shows that only a few spots exhibit a statistically different cell type frequency between spots and mapped cells (<em>p</em>-valueâ€‰&lt;â€‰0.05). <strong>e</strong> JI of mapped cell labels between Visium and VisiumHD. 50% of cells have a JI score above 0.32. The orange vertical line represents the 25th percentile. The light blue line represents the 50th percentile. The green line represents the 75th percentile. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par19">To test mapping across spatial resolutions, we mapped the VisiumHD mouse brain onto Visium of the same tissue (See â€œData availabilityâ€ section). We elected to use the 8â€‰Î¼m bins since this would strike a balance between having high-resolution and reducing noise at each spatial location. In this context, we reasoned that, given the difference in resolution, the most appropriate strategy was to only consider niche similarity and territory similarity. More specifically, for each cell, we defined the niche by using a distance radius around the center cell (see â€œMethodsâ€ section). We defined the radius as equal to the radius of a Visium spot to ensure that a Visium niche would only contain a single spot, while VisiumHD would contain all spots within that radius. We detected tissue territories in the reference Visium sample and the query VisiumHD sample (Fig.Â <a href="#Fig4" class="usa-link">4c</a>) and recovered expected tissue structures such as the CA1 field, CA3 field, and the Dentate Gyrus. Since mapping high-resolution data onto low-resolution data signifies that multiple spots will be mapped to the same low-resolution spots, we added random noise (jitter) to the coordinate values to better visualize the mapping results. Alternatively, we omitted the jitter to create pseudo-Visium data where each mapped cell will be merged into a mini-bulk count data at the spot level (Fig.Â <a href="#Fig4" class="usa-link">4c</a> â€“ left panels). We discovered that we were recovering the expected tissue structures, demonstrating that we can effectively map cells across resolutions. Interestingly, we observed finer tissue territories in the mapped region of the Visium assay compared with the reference.</p>
<p id="Par20">To further investigate the cell types' matching abilities between resolutions, we used RCTD<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> to estimate cell type proportions in both Visium and VisiumHD using the same reference scRNA-seq dataset<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. We scaled cell type frequencies (see â€œMethodsâ€ section) to create a contingency matrix of cell type proportions. We performed a Chi-squared test to determine whether cell type proportions exhibited significant differences between Visium and mapped VisiumHD. The <em>p</em>-value distributions across all mapped cells show that nearly all mapped cells display similar cell type proportions (Fig.Â <a href="#Fig4" class="usa-link">4d</a>). When we discretized the p-value, we observed only a few instances where cell type frequencies statistically differed. In addition, we computed a JI between cell types associated with Visium spots and the mapped cell identities from VisiumHD. Our results showed a reasonable correspondence in mapped cell types between Visium and VisiumHD (Fig.Â <a href="#Fig4" class="usa-link">4e</a>). The distribution of Jaccard scores showed that the median Jaccard score was around 0.32 (Fig.Â <a href="#Fig4" class="usa-link">4e</a> - right panel with 25th, 50th, and 75th percentiles shown by vertical orange, blue, and green lines). Taken together, our results indicated that while Vesalius occasionally missed cell types associated with Visium spots or VisiumHD cells, the cell type frequencies did not statistically differ.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Mapping cells across time in brain regeneration and embryo development highlights spatiotemporal expression patterns among mapped cells</h3>
<p id="Par21">The STOmics database provides a wide variety of high-resolution Stereo-seq data collections. Most notable are the MOSTA<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup> and ARTISTA<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a></sup> data collections, which provide spatial-temporal maps of murine embryo development and Axolotl brain regeneration. Using these collections, we applied our approach to mapping cells across developmental time. First, we focused our attention on brain regeneration with the aim of mapping cells forward in time. We selected 20 Days Post Injury (DPI) as the reference onto which we would map 15 DPI (Fig.Â <a href="#Fig5" class="usa-link">5a</a>) with a cost matrix constructed from cell similarity, niche similarity, and territory similarity. Since cell type labels will differ between time points, we reasoned that these 3 cost matrices represented the most appropriate combination of metrics. We observed that cells are accurately mapped in the 20DPI brain (right side of the spatial assay). Ependymoglial cells (ECG) â€“ equivalent to neural stem cells â€“ formed a mass of reactive ECG cells (reaECG) where regenerative intermediate progenitor cells (rIPC3 and rIPC4 cells) will form at a later stage (Highlighted in Fig.Â <a href="#Fig5" class="usa-link">5a</a> by dark blue circle).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Cell Mapping across time.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370880_41467_2025_62782_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5561/12370880/f2c262f0fce3/41467_2025_62782_Fig5_HTML.jpg" loading="lazy" id="d33e881" height="718" width="667" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Mapping Stereo-seq Axolotl brain regeneration datasets (ARTISTA) at 15 days post injury (DPI) to 20DPI using cell, niche, and territory similarity. Blue circle highlights ependymoglial cells and their transition to regenerative intermediate progenitor cells. <strong>b</strong> Integrated tissue territories of mapped cells and reference cells where tissue territories are recovered using a shared latent space. For visualization purposes, we separate both samples. <strong>c</strong> Two differentially expressed genes (two-sided Wilcoxon rank sum test â€“ <em>p</em>-valueâ€‰&lt;â€‰0.05 â€“ FDR adjusted) at the zone of injury (territory 8 â€“ dark blue): AMEX60DD048805 and AMEX60DD022398 show expression patterns limited to a small subset of cells involved in cell differentiation and proliferation necessary for wound healing. <strong>d</strong> Mapping Stereo-seq Mouse embryo (MOSTA) development at stage E11.5 to stage E12.5 using cell, niche, and territory similarity. <strong>e</strong> Clustering of query brain cells mapped onto the reference brain cells. Brain cells are demarcated into sub-territories, including cells that lie at the interface between brain regions (cluster 2) and the interface between the brain and other tissues (clusters 4). <strong>f</strong> Differential gene expression (two-sided Wilcoxon rank sum test â€“ <em>p</em>-valueâ€‰&lt;â€‰0.05 â€“ FDR adjusted) between clusters exemplified by <em>Hes5</em>, <em>Stnm2</em>, and <em>Cancng4</em> as genes highlighting the different spatial sub-structures present in the mouse brain and the accurate mapping of gene expression patterns across time. <strong>g</strong><em>Crabp1</em> â€“ a gene involved in the regulation of stem cell differentiation â€“ shows a spatial-temporal decoupling behavior where the expression shows a spatial demarcation at the earlier developmental stage (E11.5), which is then extinguished at the following developmental stage. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par22">We created an integrated image stack (see â€œMethodsâ€ section) upon which we applied image processing and isolated common territories between reference and query datasets (Fig.Â <a href="#Fig5" class="usa-link">5b</a>). The left assay represents the mapped cells while the right represents the reference cells. The temporal alignment provided a unique opportunity to identify differentially expressed genes, focusing on associated cell pairs. We computed differentially expressed genes between reference and mapped cells at the injury zone (territory 8 â€“ dark blue) to uncover which genes were involved in the regeneration process (two-sided Wilcoxon rank sum test â€“ <em>p</em>-valueâ€‰&lt;â€‰0.05 â€“ FDR adjusted). Out of the 123 differentially expressed genes, we found 2 notable genes <em>AMEX60DD048805</em> (<em>EDNRB</em>) and <em>AMEX60DD022398</em> (<em>ARPP19</em>). The former â€“ homologous to the EDNRB human gene involved in vasculature and cell proliferation â€“ was expressed in a subset of vascular leptomeningeal cells (VLMC) in the outer layers of the brain (Fig.Â <a href="#Fig5" class="usa-link">5c</a>). The latter â€“ a gene involved in the cell cycle, was expressed in a specialized set of ECG cells, srfpECG, and wtnEGC. The full list of differentially expressed genes is available in Supplementary DataÂ <a href="#MOESM3" class="usa-link">1</a>.</p>
<p id="Par23">We applied a similar process of mapping forward in time in the mouse embryo. As an example, we selected developmental stage E12.5 as the reference data and E11.5 as the query data. The total cost matrix was constructed using cell similarity, niche similarity, and territory similarity using the same reasoning as for the brain regeneration data. Once again, our approach showed a remarkable ability to map cells across space and time (Fig.Â <a href="#Fig5" class="usa-link">5d</a>). We aimed to explore how cell populations preferentially mapped to the same set of cells in the reference data. To achieve this, we performed hierarchical clustering (see â€œMethodsâ€ section) using cell similarity in brain cells taken from the query data and identified 5 clusters (Fig.Â <a href="#Fig5" class="usa-link">5e</a>). We observed spatial distinct structures in the mapped cells: for instance, cluster 2 and 4 highlight cells that were mapped at the interface of these brain regions (Fig.Â <a href="#Fig5" class="usa-link">5e</a>). We computed which genes were enriched in each cluster (two-sided Wilcoxon rank sum test â€“ <em>p</em>-valueâ€‰&lt;â€‰0.05 â€“ FDR adjusted). For instance, <em>Hes5</em>, <em>Stnm2</em>, and <em>Cacng4</em> were differentially expressed in clusters 1, 3, and 4, respectively (Supplementary DataÂ <a href="#MOESM4" class="usa-link">2</a>). Spatial gene expression patterns for each cluster indicated that our mapping strategy recovered expected gene expression patterns across space and time (Fig.Â <a href="#Fig5" class="usa-link">5f</a>). Intriguingly, we discovered examples of spatiotemporal decoupling (Supplementary DataÂ <a href="#MOESM5" class="usa-link">3</a>). For instance, <em>Crabp1</em> (Cellular Retinoic Acid Binding Protein 1), a gene involved in stem cell proliferation and differentiation<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>,<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>, shows high expression in cluster 3 of the query cells but is not expressed in the reference data. To demonstrate that our mapping strategy was able to map cells across highly heterogeneous tissues such as cancer, we mapped cells from prostate cancer tumor samples using Slide-seq V2 technology. Using the metadata provided by the authors, we mapped cells across samples using cell, niche, territory, and cell type labels to construct our cost matrices. We show a mapping example in Supplementary Fig.Â <a href="#MOESM1" class="usa-link">S15</a>. Our results indicate that even in highly heterogeneous cancer samples, Vesalius could map cells across samples.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Mapping and Sample stratification through cellular context mapping in cancer data</h3>
<p id="Par24">We further applied our strategy to align hundreds of spatial samples. We randomly sampled 100 patients from Breast Cancer IMC data containing a total of 37 protein markers<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. We conducted an initial filtering step to ensure that all datasets contained clinical-level metrics such as ER Status (Positive or Negative), ERBB2 positive, cancer grade, death, cancer subtype (PAM50), and years to outcome. We then selected samples that contained at least 1000 cells. From the remaining datasets, we used ER Status to select patient sub-populations since this metric enables balance and interpretability in our patient sampling procedure. We then mapped each sample to all other samples (excluding self-mapping) using a cost matrix generated from cell similarities, niche similarities, territory similarities, cell labels, and niche composition. Here, we aimed to leverage all available information. From each mapping event, we extracted the average cost and mapping scores to define overall sample mapping performance.</p>
<p id="Par25">We selected the best mapping pairs based on the lowest overall cost (Fig.Â <a href="#Fig6" class="usa-link">6a</a>). Across all pairs, cell similarity, niche similarity, and territory similarity exhibited the highest average mapping scores. Niche composition had the lowest overall score. A large fraction of the best-matching pairs also showed high correspondence with clinical metrics, with 80% having at least 3 out of 5 metrics correctly matched (Fig.Â <a href="#Fig6" class="usa-link">6b</a>). ERBB2_pos was the most well-predicted feature for our cost function, followed by ER Status. The least well predicted was cancer subtype (PAM50) with 49 out of 100 correctly predicted. However, this metric contains 5 sub-categories (Basal, HER2, Luminal A, Luminal B, Normal-like), making our predictions well above a random baseline (20). The majority of samples showed a low difference in years to outcome between query and reference samples.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Mapping and clustering of IMC samples.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370880_41467_2025_62782_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/5561/12370880/296645799064/41467_2025_62782_Fig6_HTML.jpg" loading="lazy" id="d33e991" height="999" width="670" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Scores of the best-matching patient pairs across 100 sampled breast cancer datasets. We obtained higher mapping scores in metrics linked to the cell state continuum (cell, niche, territory). For each best-mapped patient pair, we also calculated the correspondence between their respective clinical metrics. 80% of samples share at least 3 of the 5 clinical metrics observed (PAMâˆ’50, ERRBB_2 positive, Grade, ER Status, and deathBreast). <strong>b</strong> Across the 5 clinical metrics, we observed that ERBB2_pos is the metric that is the most accurately recovered (89%) while PAM50 (cancer subtype) is the most poorly predicted (49%). <strong>c</strong> Clustering using ER-negative breast cancer samples. The distinction of these samples is mainly driven by cell label and niche composition, which contrasts with the continuous cell state metrics. Clusters 9 and 10 â€“ for instance â€“ show little to no difference in cell, niche, and territory similarity in contrast to the stark difference in cell label and niche composition scores. <strong>d</strong> We highlight the clinical metrics associated with each cluster. We use the term Boolean to define True (1) or False (0). Cluster 9 is predominantly characterized by the HER2 cancer subtype, and cluster 10 is characterized by the Basal cancer subtype. Source data are provided as a Source Data file.</p></figcaption></figure><p id="Par26">We further performed hierarchical clustering of the ER-negative patients (<em>n</em>â€‰=â€‰50) using our total cost as a measure of distance between samples (Fig.Â <a href="#Fig6" class="usa-link">6c</a>). In addition to cost, we show the average performance associated with each cost metric. In total, we found 14 ER-negative sub-populations. Some clusters were driven by cell composition and cell type labeling (clusters 10, 9, and 4). We did not observe as strong a difference in mapping scores related to cell similarity, niche similarity, and territory similarity in these clusters. We checked which clinical metrics were associated with these clusters (Fig.Â <a href="#Fig6" class="usa-link">6d</a>). Clusters 10 and 9 were characterized by Basal and HER2 cancer subtypes, respectively. Our results suggest that cell type labels of neighboring cells can dictate clinical metrics. Clustering results of ER-positive patients are shown in Supplementary Figs.Â <a href="#MOESM1" class="usa-link">S16</a> and <a href="#MOESM1" class="usa-link">S17</a>. Labels and clustering results are also available in Supplementary DataÂ <a href="#MOESM6" class="usa-link">4</a>.</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par27">With the growing volume of spatial data, the necessity to map and compare cells across samples has become increasingly crucial. To achieve this goal, it is inevitable that we need to reconcile spatial datasets that are dissimilar in their spatial structure, collected using different technologies, different capture resolutions, or might have been collected under different conditions or time points. More importantly, we need to achieve this aim in such a way that flexibility, interpretability, and understanding of the underlying biology are possible. To address the challenge of mapping cell states across heterogeneous samples, we developed a new approach within the framework of Vesalius â€“ a spatial omics toolkit.</p>
<p id="Par28">Mapping cells across heterogeneous samples is a distinct task from the spatial alignment of spatial samples. Tools such as PASTE<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, GPSA<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, or STalign<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup> make an assumption of structural rigidity, which is expected in adjacent tissue samples. While they can locally and differentially deform a query sample to match a target sample, the query sample cannot be broken, folded, or scrambled. However, samples taken from a tumor (even if it is the same tumor) are going to be structurally heterogeneous, and this heterogeneity will be further amplified when sampling across patients. We demonstrated in our benchmarking (synthetic and real data) that spatial alignment tools (PASTE and GPSA) are incapable of mapping cells between samples in a way that will recover the structure of morphologically complex tissues.</p>
<p id="Par29">The many-to-many cell matching framework is similar to scRNA-seq mapping tools such as CytoSpace<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> and Tangram<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Neither tool, however, includes spatial information to map cells across samples. Alternatively, Scanorama<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup> and SLAT<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup> provide single-cell data integration methods, with SLAT utilizing spatial neighborhoods to construct its integrated latent space. However, the inclusion of neighborhood information alone is not always sufficient to accurately map cells across samples. In fact, capitalizing on all available information provides more accurate spatial mapping outcomes (Figs.Â <a href="#Fig2" class="usa-link">2</a> and <a href="#Fig3" class="usa-link">3</a>).</p>
<p id="Par30">In our benchmarking across synthetic and real datasets, we demonstrated how the inclusion multi-scale and multi-context information improves the mapping of cellular spatial context. We observed that tools such as SLAT and Scanorama perform well at accurately mapping cells with the same cell type across synthetic samples (Fig.Â <a href="#Fig2" class="usa-link">2a</a>, Supplementary Figs.Â <a href="#MOESM1" class="usa-link">S6</a>â€“<a href="#MOESM1" class="usa-link">S9</a>). However, they did not capture spatial context with sufficient accuracy to map the cellular and spatial context of cells (Fig.Â <a href="#Fig2" class="usa-link">2b</a>). Vesalius, on the other hand, could exploit all available information (including cell type composition of niches and cell type labels) to accurately map cells and their spatial context. It is of note that Vesalius showed high performance when dealing with more complicated real data where multiple cell types are co-localized, indicating that Vesalius utilizes spatial context effectively for mapping. In contrast, the high performance of SLAT and Scanorama was not reproduced in real biological data.</p>
<p id="Par31">CytoSpace proved to be an interesting case study on the over-reliance on certain modalities. CytoSpace consistently obtained perfect scores at mapping cell identities across samples in both synthetic and real data. However, this is due to the mechanism by which CytoSpace utilizes cell type labels: Only cells with the same cell type label will be mapped to each other. In fact, when cell labels are not shared or removed CytoSpace either fails to run or performs poorly. Even with cell type labels, it does not capture the spatial context of cells (Fig.Â <a href="#Fig2" class="usa-link">2</a>b, Fig.Â <a href="#Fig3" class="usa-link">3b/e</a>). While Vesalius can also capitalize on cell type labels, it does so with much more flexibility. Cell type labels constitute only part of the total cost matrix, and if a cell type label does not match, but the other scores (cell, niche, or territory similarity, for example) are high, then this mapping event might still occur. This mechanism allows Vesalius to utilize cell labels while still accounting for the cell state continuum and compensating for inaccurate cell type labels.</p>
<p id="Par32">The flexibility with which Vesalius constructs its cost matrices encouraged us to explore how we can employ this mechanism across a variety of analysis scenarios. First, we aimed at mapping cells across technologies as distinct as seqFISH and Stereo-seq in mouse embryos (Fig.Â <a href="#Fig4" class="usa-link">4a/b</a>). Despite the technical differences, Vesalius showed himself to be particularly adept at mapping cells across technologies by utilizing the available information (cell and niche similarity). We employed this flexibility with great success when mapping cells across spatial resolutions. By using niches defined by a spatial radius and large-scale territory information, we were able to recover expected cell type composition between Visium and VisiumHD mouse brain data (Fig.Â <a href="#Fig4" class="usa-link">4câ€“e</a>). Second, we mapped cells across time in Axolotl brain regeneration and murine embryonic development, where we were able to recapitulate the expected tissue structures despite only using cell, niche, and territory similarities to construct our total cost (Fig.Â <a href="#Fig5" class="usa-link">5</a>). Our integrated territory approach allowed us to underline genes being activated at the zone of injury. Brain cells in the mouse embryo demonstrated spatial sub-populations, which led to the discovery of spatial-temporal decoupling events related to stem cell differentiation.</p>
<p id="Par33">In addition to its flexible cost construction mechanism, Vesalius uses simple and interpretable scores. A Pearsonâ€™s correlation coefficient or a JI represents universally interpretable measures of how similar or dissimilar 2 datasets are. In contrast, distances within a spatially aware integrated latent space computed by deep learning methodologies (such as SLAT) are difficult to interpret. They do not highlight which aspects of a spatial dataset can be used to discriminate samples. To this effect, we used Vesaliusâ€™s interpretability to our advantage by clustering spatial samples. We applied this sample clustering approach across 100 breast cancer IMC samples and found remarkable concordance with clinical metrics such as ER status and ERBB2-positive patients, even though we used only 37 protein markers (Fig.Â <a href="#Fig6" class="usa-link">6</a>). These results suggest that patient status can be at least partially encoded in a cancerâ€™s multi-scale organization.</p>
<p id="Par34">A key aspect to consider when using our approach is which cost matrix to utilize to obtain optimal cell mapping. In Supplementary Fig.Â <a href="#MOESM1" class="usa-link">S12</a>, we demonstrate how each cost matrix may contribute to discriminating between cells and participate in the total cost. Discrete and categorical cost matrices (cell composition and cell type labels) tend to have a stronger effect on optimal mapping as seen in Figs.Â <a href="#Fig2" class="usa-link">2</a> and <a href="#Fig3" class="usa-link">3</a>, and Supplementary Fig.Â <a href="#MOESM1" class="usa-link">S12</a>. The assumption is that cell type labels (and by extension, niche composition) accurately represent prior biological knowledge. Our algorithm uses this prior biological knowledge to its advantage without requiring explicit weighting of each cost matrix. Across all our benchmarking, we observed that using all available information (cell, niche, territory similarities, niche composition, and cell type) provides the best combination for recovering center cell identity and cellular context. With that said, if cell type labels are not available or are not deemed appropriate (e.g., different nomenclature), using cell and niche similarity can still provide highly accurate mapping of cells across samples. We recommend including territory information since cells can express global context markers as we previously demonstrated<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Nevertheless, it is crucial to tailor any analysis to the question at hand.</p>
<p id="Par35">Vesalius provides a mapping strategy with exceptional flexibility and interpretability, opening the door to tailored analysis utilizing all available information, including marker gene activity or cell fate information. We further investigated running time and memory usage over the number of cells (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">S10 a/b</a>). The running time and the memory usage are also dependent on the configuration used for the cost metric (Supplementary Fig.Â <a href="#MOESM1" class="usa-link">S10 c/d</a>). Vesalius aims at understanding spatial biology across contexts and scales and provides a unique toolkit to investigate spatial samples as exemplified in our analysis related to cell states during brain regeneration, development, and patient classification. Our results demonstrate how crucial it is to consider multiple scales and contexts to understand the spatial relationship between cells.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Mapping cells across samples</h3>
<p id="Par36">To map cells across samples, Vesalius employs the Jonker-Volgenant algorithm (<em>O</em>(<em>n</em><sup>3</sup>))<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, a variant of the LAP. The algorithm is to map â€œworkersâ€ to â€œtaskâ€ while minimizing the overall cost. To allow many-to-many matching of cells, we implemented the solver in a divide-and-conquer framework, which also provides a speedup for large batches through parallelization.</p>
<p id="Par37">Given a requested batch size <em>B</em>, we randomly sample cells from the reference set <em>R</em> and the query set <em>Q</em>. We select a subset <em>R</em><sub><em>B</em></sub> âŠ‚ <em>R</em> and <em>Q</em><sub><em>B</em></sub> âŠ‚ <em>Q</em> such that |<em>R</em><sub><em>B</em></sub>|â€‰=â€‰|<em>Q</em><sub><em>B</em></sub>|â€‰=â€‰<em>B</em>. Once a cell is selected, it is removed from the sampling pool to ensure all cells are eventually selected. If the number of cells in one dataset is smaller than in the other, padding is applied. Without loss of generality, assume |<em>R</em>|â€‰&lt;â€‰|<em>Q</em>|. In this case, we create a duplicated set <em>R</em>â€² and concatenate it with <em>R</em>, forming <em>R</em>*â€‰=â€‰<em>R</em> âˆª <em>R</em>â€², where |<em>R</em>*|â€‰â‰¥â€‰<em>B</em> or matches |<em>Q</em>|, whichever is smaller. The Linear Assignment Problem (LAP) solver is then applied to each batch to optimize the mapping between the sampled reference and query cells. Given a cost matrix <em>C</em> for a batch, the optimal assignment <em>M</em><sup>*</sup> is found by minimizing the total cost:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><math id="d33e1228" display="block"><msup><mrow><mi>M</mi></mrow><mrow><mo>*</mo></mrow></msup><mo>=</mo><mi mathvariant="normal">arg</mi><mi>min</mi><msub><mrow><mo mathsize="big">âˆ‘</mo></mrow><mrow><mrow><mo>(</mo><mrow><mi>i</mi><mo>,</mo><mi>j</mi></mrow><mo>)</mo></mrow><mo>âˆˆ</mo><mi>M</mi></mrow></msub><msub><mrow><mi>C</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></math></td>
<td class="label">1</td>
</tr></table>
<p id="Par38">If a one-to-one matching is required, duplicate matches can be filtered by retaining only the pairs with the lowest overall cost.</p>
<p id="Par39">Since the mapping depends on the cells selected during sampling, the batching and solving steps can be applied across multiple epochs. At each epoch <em>t</em>, new random batches <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e1271"><msubsup><mrow><mi>R</mi></mrow><mrow><mi>B</mi></mrow><mrow><mfenced close=")" open="("><mrow><mi>t</mi></mrow></mfenced></mrow></msubsup><mo>âŠ‚</mo><mi>R</mi></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e1290"><msubsup><mrow><mi>Q</mi></mrow><mrow><mi>B</mi></mrow><mrow><mfenced close=")" open="("><mrow><mi>t</mi></mrow></mfenced></mrow></msubsup><mo>âŠ‚</mo><mi>Q</mi></math></span> are selected and mapped using the LAP solver. Let <em>M</em><sup>(<em>t</em>)</sup> represent the set of matched cell pairs at epoch <em>t</em>. The newly matched cell pairs are compared to those from previous epochs, and a pair (<em>i</em>, <em>j</em>) is retained if it results in a lower matching cost:</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><math id="d33e1328" display="block"><msubsup><mrow><mi>C</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow></mrow></msubsup><mo>&lt;</mo><mspace width="0.25em"></mspace><msubsup><mrow><mi>C</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>t</mi><mo>âˆ’</mo><mn>1</mn></mrow><mo>)</mo></mrow></mrow></msubsup></math></td>
<td class="label">2</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e1364"><msubsup><mrow><mi>C</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>t</mi></mrow><mo>)</mo></mrow></mrow></msubsup></math></span> is the cost associated with matching cell <em>i</em> from <em>R</em> to cell <em>j</em> from <em>Q</em> at epoch <em>t</em>.</p>
<p id="Par40">The final output is a many-to-many mapping of cells across samples, along with the mapping cost, epoch information, and mapping scores for each cell pair.</p></section><section id="Sec12"><h3 class="pmc_sec_title">Computing cost</h3>
<p id="Par41">To solve the LAP, we compute a total cost matrix constructed from comparing cells through a variety of biological features. The default features used by Vesalius are the following:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par42"><em>Cell similarity</em> (<em>f</em>): compares the gene expression of cells between each sample.</p></li>
<li><p id="Par43"><em>Niche similarity</em> (<em>n</em>): compares the gene expression of cell niches between each sample.</p></li>
</ul>
<p id="Par44">Optionally:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par45"><em>Territory similarity</em> (<em>t</em>): compares the gene expression of territories between each sample.</p></li>
<li><p id="Par46"><em>Cell label</em> (<em>y</em>): checks whether both cells share the same cell type label.</p></li>
<li><p id="Par47"><em>Composition similarity</em> (<em>c</em>): compares the cell type composition of cell niches between each sample.</p></li>
<li><p id="Par48"><em>Custom similarity</em>: compares cells between samples using user-defined similarities.</p></li>
</ul>
<p id="Par49">To generalize the formula for an arbitrary number of similarity matrices, we define a set <em>n</em> of similarity matrices <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e1460"><msup><mrow><mi>S</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>k</mi></mrow><mo>)</mo></mrow></mrow></msup><mi>f</mi><mi>o</mi><mi>r</mi><mspace width="0.16em"></mspace><mi>k</mi><mo>=</mo><mn>1</mn><mo>,</mo><mo>â€¦</mo><mo>,</mo><mi>n</mi></math></span>. The final cost matrix <em>C</em> is then computed as the sum of the reciprocals of all provided similarity matrices:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><math id="d33e1493" display="block"><msub><mrow><mi>C</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><msubsup><mrow><mo mathsize="big">âˆ‘</mo></mrow><mrow><mi>k</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><mrow><mo>(</mo><mrow><mn>1</mn><mo>âˆ’</mo><msubsup><mrow><mi>S</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mfenced close=")" open="("><mrow><mi>k</mi></mrow></mfenced></mrow></msubsup></mrow><mo>)</mo></mrow></math></td>
<td class="label">3</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e1534"><msubsup><mrow><mi>S</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow><mrow><mrow><mo>(</mo><mrow><mi>k</mi></mrow><mo>)</mo></mrow></mrow></msubsup></math></span> represents the similarity score between query cell <em>i</em> and reference cell <em>j</em> in the <em>k</em>-th similarity matrix. This formulation allows the incorporation of any number of similarity metrics. Importantly, cost matrices are not weighted. Cost matrices using discrete/categorical inputs (niche composition and cell type labels) leverage prior biological knowledge and will yield a stronger discriminative ability without needing to be explicitly weighted.</p>
<p id="Par50">In the case of continuous features (cell, niche, and territory), we first extract a gene expression signal. The signal represents a gene expression profile (normalized or raw) of all genes, highly variable genes, or a custom gene set. Alternatively, embedding values (PCA, UMAP, LSI, NMF) can also be used to define the gene expression signal. By default, and across all analyses presented here, we used the highly variable genes. Since genes might not perfectly overlap between datasets, we select the genes that lie at the intersection of gene expression signals for each dataset.</p>
<p id="Par51">We calculate a Pearsonâ€™s correlation coefficient between expression signals of each potential cell pair:</p>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><math id="d33e1569" display="block"><mi>r</mi><mo>=</mo><mspace width="0.25em"></mspace><mfrac><mrow><mo mathsize="big">âˆ‘</mo><mrow><mo>(</mo><mrow><msub><mrow><mi>x</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>âˆ’</mo><mover accent="true"><mrow><mi>x</mi></mrow><mo>Â¯</mo></mover></mrow><mo>)</mo></mrow><mrow><mo>(</mo><mrow><msub><mrow><mi>y</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>âˆ’</mo><mi>È³</mi></mrow><mo>)</mo></mrow></mrow><mrow><msqrt><mrow><mo mathsize="big">âˆ‘</mo><msup><mrow><mrow><mo>(</mo><mrow><msub><mrow><mi>x</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>âˆ’</mo><mover accent="true"><mrow><mi>x</mi></mrow><mo>Â¯</mo></mover></mrow><mo>)</mo></mrow></mrow><mrow><mn>2</mn></mrow></msup><mo mathsize="big"> âˆ‘</mo><msup><mrow><mrow><mo>(</mo><mrow><msub><mrow><mi>y</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>âˆ’</mo><mi>È³</mi></mrow><mo>)</mo></mrow></mrow><mrow><mn>2</mn></mrow></msup></mrow></msqrt></mrow></mfrac></math></td>
<td class="label">4</td>
</tr></table>
<p>where <em>x</em><sub><em>i</em></sub> and <em>y</em><sub><em>i</em></sub> are the expression of genes <em>i</em> in sample <em>x</em> and <em>y,</em> respectively. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e1671"><mover accent="true"><mrow><mi>x</mi></mrow><mo>Â¯</mo></mover></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><math id="d33e1683"><mi>È³</mi></math></span> are the mean expression of cell <em>x</em> and cell <em>y</em>.</p>
<p id="Par52">For categorical features such as niche composition, we compute a frequency-aware Jaccard index:</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><math id="d33e1701" display="block"><mi>J</mi><mfenced close=")" open="("><mrow><mi>A</mi><mo>,</mo><mi>B</mi></mrow></mfenced><mo>=</mo><mspace width="0.25em"></mspace><mfrac><mrow><mo>âˆ£</mo><mi>A</mi><mo>âˆ©</mo><mi>B</mi><mo>âˆ£</mo></mrow><mrow><mo>âˆ£</mo><mi>A</mi><mo>âˆª</mo><mi>B</mi><mo>âˆ£</mo></mrow></mfrac></math></td>
<td class="label">5</td>
</tr></table>
<p id="Par53">We assign a unique label to duplicated cell type labels. If cell type A is present more than once, the first instance the label will be A, the second instance A.1, and so on.</p>
<p id="Par54">Similarity matrices, which are computed using Pearsonâ€™s correlation coefficient or a Jaccard Index, can be filtered by a user-defined threshold prior to mapping. For all cells in the query, we check if it has at least one match higher than the threshold.</p>
<p id="Par55">We define an indicator function for the label agreement between a reference cell <em>j</em> and a query cell <em>i</em>. Let <em>L</em><sub><em>i</em></sub> and <em>L</em><sub><em>j</em></sub> be the labels of the query cell <em>i</em> and reference cell <em>j</em>, respectively. Then, we define:</p>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><math id="d33e1761" display="block"><msub><mrow><mi>I</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mfenced open="{"><mrow><mtable><mtr><mtd columnalign="center"><mn>1</mn><mo>,</mo><mi>i</mi><mi>f</mi><msub><mrow><mi>L</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>=</mo><msub><mrow><mi>L</mi></mrow><mrow><mi>j</mi></mrow></msub></mtd></mtr><mtr><mtd columnalign="center"><mn>0</mn><mo>,</mo><mi>i</mi><mi>f</mi><msub><mrow><mi>L</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>â‰ </mo><msub><mrow><mi>L</mi></mrow><mrow><mi>j</mi></mrow></msub></mtd></mtr></mtable></mrow></mfenced></math></td>
<td class="label">6</td>
</tr></table>
<p id="Par56">This function assigns a value of 1 if the labels match and 0 otherwise. A query cell <em>i</em> that satisfies the following condition:</p>
<table class="disp-formula p" id="Equ7"><tr>
<td class="formula"><math id="d33e1818" display="block"><msub><mrow><mo mathsize="big">âˆ‘</mo></mrow><mrow><mi>i</mi></mrow></msub><mspace width="0.25em"></mspace><msub><mrow><mi>I</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo>=</mo><mn>0</mn></math></td>
<td class="label">7</td>
</tr></table>
<p id="Par57">Is assumed not to have any shared cell type labels with the reference data and can be removed upon user request prior to mapping.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Extracting niches</h3>
<p id="Par58">For each cell, we define a niche as the spatially neighboring cells using the following methods:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par59"><em>KNN</em>: K-Nearest Neighbors</p></li>
<li><p id="Par60"><em>Graph</em>: Using Voronoi Tessellation, we define a neighborhood graph and select niches by considering the graph depth from a center cell, e.g., direct neighbors will have a graph depth of 1.</p></li>
<li><p id="Par61"><em>Radius</em>: All cells within a certain radius of a center cell are defined as this cellâ€™s niche.</p></li>
</ul>
<p id="Par62">In most cases, we recommend using <em>Graph</em> since this method best captures the biological realities of cellular organization. We urge any user to carefully consider their question and tailor their analysis (and parameter selection) to the question at hand.</p>
<p id="Par63">In the context of niche similarity, let <em>X</em><sub><em>c</em></sub> be the gene expression profile of cell <em>c</em>, and <em>N</em>(<em>c</em>) be the set of cells in the neighborhood of cell <em>c</em>.</p>
<p id="Par64">Mean gene expression profile of the neighborhood for cell <em>c</em>:</p>
<table class="disp-formula p" id="Equ8"><tr>
<td class="formula"><math id="d33e1894" display="block"><msub><mrow><mover accent="true"><mrow><mi>X</mi></mrow><mo>Â¯</mo></mover></mrow><mrow><mi>N</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></msub><mo>=</mo><mspace width="0.25em"></mspace><mfrac><mrow><mn>1</mn></mrow><mrow><mo>âˆ£</mo><mi>N</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>âˆ£</mo></mrow></mfrac><msub><mrow><mo mathsize="big">âˆ‘</mo></mrow><mrow><mi>c</mi><mo>â€²</mo><mo>âˆˆ</mo><mi>N</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></msub><mspace width="0.25em"></mspace><msub><mrow><mi>X</mi></mrow><mrow><mi>c</mi><mo>â€²</mo></mrow></msub></math></td>
<td class="label">8</td>
</tr></table>
<p>where |<em>N</em>(<em>c</em>)| is the number of cells in the neighborhood of <em>c</em>, and <em>X</em><sub><em>c</em></sub>â€² is the gene expression profile of cell <em>c</em>â€².</p>
<p id="Par65">Gene expression profile of cell <em>c</em> becomes the average expression of its neighborhood:</p>
<table class="disp-formula p" id="Equ9"><tr>
<td class="formula"><math id="d33e1975" display="block"><msub><mrow><mi>X</mi></mrow><mrow><mi>c</mi></mrow></msub><mo>=</mo><mspace width="0.25em"></mspace><msub><mrow><mover accent="true"><mrow><mi>X</mi></mrow><mo>Â¯</mo></mover></mrow><mrow><mi>N</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></msub></math></td>
<td class="label">9</td>
</tr></table></section><section id="Sec14"><h3 class="pmc_sec_title">Extracting territories</h3>
<p id="Par66">Originally, Vesalius was designed to detect tissue territories from spatial transcriptomics data using image processing techniques. We have since improved the territory detection abilities. In addition to a wider range of dimensionality reduction techniques (PCA, UMAP, Latent Semantic Indexing (LSI), Non-Negative Matrix Factorization (NMF), custom) and normalization methods (log-normalization, SCtransform, Term-Frequency Inverse Document Frequency (TF-IDF), custom), we have adapted the image processing steps of Vesalius to function on image tensors instead of RGB formatted images. This upgrade allows us to apply the image processing steps to all latent space dimensions simultaneously and, in turn, improve our ability to recover tissue territories.</p>
<p id="Par67">In summary, spatial transcriptomics data is converted into a grayscale image stack. We normalized counts and extracted highly variable features. With these features, we compute a reduced dimension latent space for each cell. In parallel, we expand punctual coordinates using Voronoi tessellation and rasterize the resulting tiles to produce a grayscale image for each latent space dimension. This image stack can be processed using a variety of image processing techniques such as histogram equalization, image smoothing, and image segmentation. Color segments can be further divided to distinguish spatially distinct color segments. The details of this process are discussed in our previous publication<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>.</p>
<p id="Par68">The territories isolated by Vesalius are used to compare the expression of cells across larger-scale spatial domains. We compare the average expression of the territory in which a cell finds itself across samples.</p>
<p id="Par69">Let <em>T</em> be a territory defined by Vesalius. Let <em>X</em><sub><em>c</em></sub> be the gene expression profile of cell <em>c</em> âˆˆ <em>T</em>, and <em>T</em>(<em>c</em>) be the set of cells in Territory <em>T</em>.</p>
<p id="Par70">Mean gene expression profile of the neighborhood for cell <em>c</em>:</p>
<table class="disp-formula p" id="Equ10"><tr>
<td class="formula"><math id="d33e2047" display="block"><msub><mrow><mover accent="true"><mrow><mi>X</mi></mrow><mo>Â¯</mo></mover></mrow><mrow><mi>N</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></msub><mo>=</mo><mspace width="0.25em"></mspace><mfrac><mrow><mn>1</mn></mrow><mrow><mo>âˆ£</mo><mi>T</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow><mo>âˆ£</mo></mrow></mfrac><msub><mrow><mo mathsize="big">âˆ‘</mo></mrow><mrow><mi>c</mi><mo>â€²</mo><mo>âˆˆ</mo><mi>T</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></msub><mspace width="0.25em"></mspace><msub><mrow><mi>X</mi></mrow><mrow><mi>c</mi><mo>â€²</mo></mrow></msub></math></td>
<td class="label">10</td>
</tr></table>
<p>where |<em>T</em>(<em>c</em>)| is the number of cells territory <em>T</em>, and <em>X</em><sub><em>c</em></sub>â€² is the gene expression profile of cell <em>c</em>â€².</p>
<p id="Par71">Gene expression profile of cell <em>c</em> becomes the average expression of its neighborhood:</p>
<table class="disp-formula p" id="Equ11"><tr>
<td class="formula"><math id="d33e2128" display="block"><msub><mrow><mi>X</mi></mrow><mrow><mi>c</mi></mrow></msub><mo>=</mo><mspace width="0.25em"></mspace><msub><mrow><mover accent="true"><mrow><mi>X</mi></mrow><mo>Â¯</mo></mover></mrow><mrow><mi>T</mi><mrow><mo>(</mo><mrow><mi>c</mi></mrow><mo>)</mo></mrow></mrow></msub></math></td>
<td class="label">11</td>
</tr></table></section><section id="Sec15"><h3 class="pmc_sec_title">Spatial simulations</h3>
<p id="Par72">To benchmark our mapping approach, we elected to use simulated data since they provide a strong ground truth against which we can compare performance. We used 5 regimes: <em>circle</em>, <em>layer, dropped, random one cell</em>, and <em>random two cell</em>. For each regime, we generate 12 samples, each containing 5000 cells and 2000 genes. We used the splatter<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup> package to generate synthetic single-cell-like count data for all samples. All cells for each cell type across each sample are taken from the same synthetic distribution. Reproducible generation of synthetic samples is available in a dedicated R package called oneiric (see â€œCode availabilityâ€ section). The vignette of the package shows how to generate different samples as well as the data used for this analysis. The code is also available in the GitHub repository dedicated to the analysis presented here (see â€œCode availabilityâ€ section).</p>
<p id="Par73">For the <em>circle</em> regime, we created 5 circular territories of random size placed at random within a background territory. We allowed overlaps between territories but ensured that all territories were present across 12 samples. Each territory contains 2 cell types, and there is a probability <em>p</em>â€‰=â€‰0.5 of differential gene expression between cell types.</p>
<p id="Par74">For the <em>layered</em> regime, we created 1 circular territory of random size placed at random within a background territory. The circular territory is divided into 5 layers. The background and each layer will contain 2 cell types. There is a probability <em>p</em>â€‰=â€‰0.5 of differential gene expression between the cells in the background and the circular territory. There is a probability <em>p</em>â€‰=â€‰0.05 of differential gene expression between cells in each layer.</p>
<p id="Par75">For the <em>dropped</em> regime, we created 2 circular territories of random size placed at random within a background territory. Each circular territory can be divided into 1â€“3 layers (the number of layers selected randomly). The background and each layer can contain 1 or 2 cell types. The selection of the number of layers and the number of cells is done on a sample basis, meaning that cells and layers will not necessarily be shared across samples. Cells from the same layers or cell type within a layer will always be sampled from the same statistical distribution. There is a probability <em>p</em>â€‰=â€‰0.5 of differential gene expression between the cells in the background and the circular territories. There is a probability <em>p</em>â€‰=â€‰0.05 of differential gene expression between cells in each layer.</p>
<p id="Par76">For a <em>random one cell</em> and a <em>random two cell</em>, we created a background territory where a random number of territories (1â€“3), of random shape (circular, rod, or chaos map), with a random number of layers (0â€“4) will be placed. <em>Random one cell</em> will only contain at most one cell. <em>Random two cell</em> will contain at most 2 cells (number of cells randomly selected). There is a probability <em>p</em>â€‰=â€‰0.5 of differential gene expression between the cells in the background and territories. There is a probability <em>p</em>â€‰=â€‰0.05 of differential gene expression between cells in each layer.</p>
<p id="Par77">For each synthetic regime, we exported the cell labels of each cell as well as the neighborhood interactions. Simply put, we select the K-nearest neighbors (<em>k</em>â€‰=â€‰6) and concatenate the labels of the neighborhood into an â€œinteractionâ€ label (using @ as separator for discrimination later). Adding interactions is part of the oneiric package with the <em>add</em>_<em>interactions</em> function. Data generation code is available on the GitHub dedicated to the analysis presented in this manuscript (see â€œCode availabilityâ€ section).</p>
<p id="Par78">In addition to these datasets, we also generated 3 samples of the circle regime with varying numbers of cells (100, 500, 1000, 2500, 5000, 10,000, 20,000, 50,000, 100,000, 200,000). These datasets were used for computational performance benchmarking (see below).</p></section><section id="Sec16"><h3 class="pmc_sec_title">Biological data preparation for benchmarking</h3>
<p id="Par79">For format consistency, we prepared and processed each biological dataset used for benchmarking, namely, seqFISH mouse embryo, Slide-seq V2 mouse hippocampus, and Stereo-seq mouse embryo (see â€œData availabilityâ€ section).</p>
<p id="Par80">We split datasets into slices when necessary. seqFISH is available as a single R object, which we split into 3 mouse embryo sections. This step was not necessary for Slide-seq V2 and Stereo-seq since they are provided as separate files. For all datasets, we rescaled spatial coordinates to ensure that the minimum x and y coordinates would be 1. Next, using oneiric, we added neighborhood interactions labels with KNN (<em>k</em>â€‰=â€‰10). All datasets are exported in the same format, consisting of a coordinate data frame with 6 columns (barcodes, x, y, sample, cell_labels, interactions) and a gene-by-cell count matrix.</p>
<p id="Par81">For Slide-seq datasets, we first filtered cells that contained less than 200 total counts. We also filtered genes with less than 100 total counts across all cells. Then, we annotated datasets using RCTD<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup> with the same mouse brain scRNA-seq reference data<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup> in order to quantitatively assess cell and spatial context mapping abilities. For Stereo-seq, we downsampled to 50,000 cells to reduce run time.</p>
<p id="Par82">Data preparation code is available on the GitHub dedicated to the analysis presented in this manuscript (see â€œCode availabilityâ€ section).</p></section><section id="Sec17"><h3 class="pmc_sec_title">Data preparation of cross-technology and cross-resolution mapping</h3>
<p id="Par83">To compare cells across technologies, we processed seqFISH<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup> and Stereo-seq data<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. seqFISH data was filtered by removing low-quality cells as described by the authors. Next, we pre-processed both datasets using Vesalius by log-normalizing the counts and producing a reduced dimension latent space using PCA (PCsâ€‰=â€‰30). Our cross-technology comparison only used cell similarity and niche similarity; as such, no further processing was required.</p>
<p id="Par84">Matching cells across resolutions required extra steps for data preparation. First, we downsampled the VisiumHD mouse brain data to retain 200,000 cells. We then used RCTD in doublet mode to deconvolve and assign cell type labels. As a reference, we used the labeled scRNA data provided by ref. <sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. We remove all cells that contain the â€œrejectâ€ tag. For all remaining cells, we annotated cells with one or two labels depending on how RCTD defined them (singlet or doublet). After this filtering, the VisiumHD data contained 84,287 cells. We also used RCTD to deconvolve the Visium mouse brain data with the same reference dataset<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. In this instance, we only retained cell type labels that were considered â€œhigh confidenceâ€.</p></section><section id="Sec18"><h3 class="pmc_sec_title">Benchmarking mapping performance</h3>
<p id="Par85">We contrasted our mapping approach to single-cell mapping tools, spatial alignment tools, and data integration tools. More specifically, we compared Vesalius to CytoSpace<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, Tangram<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>, PASTE<sup><a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>, GPSA<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>, SLAT<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>, and Scanorama<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. It is of note that CytoSpace requires perfect matching of cell type labels between reference and query and only finds a proposed mapping of cells between the same cell type labels. Importantly, if the cell labels are not shared, CytoSpace will not run to completion. To demonstrate how this strategy differs from our context-specific approach to cell mapping, we provided CytoSpace with true cell labels as well as a single-cell label for all cells (noted with the _<em>noLab</em> label).</p>
<p id="Par86">For Vesalius, we ran mapping with 14 cost matrix combinations:</p>
<ul class="list" style="list-style-type:disc">
<li><p id="Par87">feature</p></li>
<li><p id="Par88">niche</p></li>
<li><p id="Par89">territory</p></li>
<li><p id="Par90">composition</p></li>
<li><p id="Par91">feature-niche</p></li>
<li><p id="Par92">feature-niche-cell_type</p></li>
<li><p id="Par93">feature-composition</p></li>
<li><p id="Par94">feature-territory</p></li>
<li><p id="Par95">niche-territory</p></li>
<li><p id="Par96">niche-composition</p></li>
<li><p id="Par97">feature-niche-territory,</p></li>
<li><p id="Par98">feature-niche-composition</p></li>
<li><p id="Par99">feature-niche-composition-territory</p></li>
<li><p id="Par100">feature-niche-composition-territory-cell_type</p></li>
</ul>
<p id="Par101">Where features represent cell similarity, niche represents niche similarity, territory represents territory similarity, composition represents cell type composition of niche, and cell type represents the cell label score.</p>
<p id="Par102">To quantitatively assess the performance of cell mapping between samples, we computed an Adjusted Rand Index<sup><a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> (ARI) between the cell labels of the query and the reference. We also computed a Jaccard Index between the cell labels of the neighborhood of mapped cells to ascertain if we were capturing the spatial and cellular context of each mapped cell. Since PASTE and GPSA do not match cells but rather modify the coordinates of each cell, we computed the nearest cell neighbor between the reference and query and compared their cell type labels. While this approach disadvantages PASTE and GPSA, our objective is to demonstrate that context mapping is a fundamentally different task from spatial alignment. Scanorama is a tool designed to integrate single-cell datasets; it only provides an integrated latent space, but not best-matching cells. As such, we computed the nearest neighbors between the reference data and query data in the integrated latent space to determine best-matching cell.</p>
<p id="Par103">For Vesalius (==2.0.0), we ran our mapping using 1000 cells (for synthetic data) or 5000 cells (for real data) across 25 epochs. We set a cell filtering threshold of 0.9 and allowed cell type label filtering if they were provided. We used the log-normalized counts to define the gene expression signal. Niches were defined using KNN with <em>k</em>â€‰=â€‰6 for synthetic data and <em>k</em>â€‰=â€‰10 for real data. In both cases, we select the top 2000 variable genes for analysis.</p>
<p id="Par104">For CytoSpace (==1.0.6a0), we reformatted synthetic data and real data to fit the CytoSpace data requirement. We used 5000 cells as a batch size since CytoSpace down-samples based on cell type labels. We ran CytoSpace using their single-cell mode (does not require the deconvolution step to be run).</p>
<p id="Par105">For Tangram (tangram-scâ€‰=â€‰=1.0.4), we loaded data as AnnData objects. We log-normalize the data and extract the top 2000 variable genes. We use these variable genes as seed genes for Tangram mapping. We use â€œrna_count_basedâ€ as the density prior and ran Tangram for 10,000 iterations.</p>
<p id="Par106">For PASTE (paste-bio==1.4.0), we loaded data as AnnData objects. Data objects do not need to be normalized prior to alignment. We ran PASTE for 10,000 iterations.</p>
<p id="Par107">For GPSA (==0.8), we loaded data as AnnData objects. We ran GPSA for 50 iterations as recommended by the authors. The learning rate for the Adam optimizer was set at 0.01 as recommended by the authors.</p>
<p id="Par108">For SLAT (scslat==0.2.1), we loaded data as AnnData objects. SLAT does not require data to be normalized prior to mapping. We selected neighborhoods using KNN (<em>k</em>â€‰=â€‰6 for synthetic data and <em>k</em>â€‰=â€‰10 for real data). We ran SLAT with 5 Graph Convolutional layers across 25 epochs. Parameters were modified in accordance with developer recommendations.</p>
<p id="Par109">For Scanorama (==1.7.4), we loaded data as AnnData objects and ran Scanorama with default parameters.</p>
<p id="Par110">For all benchmarking, all code, parameters, and dependency lists used in this analysis are available in the dedicated GitHub.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Computational performance benchmarking</h3>
<p id="Par111">We included benchmarking for computational performance by varying the number of cells used during mapping (100, 500, 1000, 2500, 5000, 10,000, 20,000, 50,000, 100,000, 200,000) in a circle synthetic regime.</p>
<p id="Par112">For this task, we aimed at making the performance as fair as possible; as such, we assumed that a user would run a full analysis pipeline, including data loading, preprocessing, and mapping on a single CPU. We made this assumption since all tools have different input requirements, different downstream tasks, and different optimization options. Vesalius provides the widest variety of potential downstream tasks and, as such, has the most requirements in terms of preprocessing.</p>
<p id="Par113">We also aimed to avoid any variation in language-specific methods of measuring run time and memory usage. We measured run time and memory usage (Resident Set Size â€“ peak RAM usage) for each tool using custom bash scripts (Available on the dedicated GitHub page) that measured these metrics across 3 submitted jobs. We set hard computational limits: a maximum of 480â€‰GB of RAM and a maximum run time of 12â€‰h. If the job exceeded these limits, the run was cut short.</p>
<p id="Par114">We compared run time and memory usage across cost matrix combinations for 5000 and 10,000 cells. We used the same 14 matrix combinations described above and used the same performance measuring strategy as described above.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Cross-technology and cross-resolution mapping</h3>
<p id="Par115">To map Stereo-seq mouse embryo to seqFISH and vice versa, we mapped cells in batches of 10,000 cells across 10 epochs. Since our approach selects the intersections of genes available between both datasets, we use all genes (135 genes in the seqFISH datasets).</p>
<p id="Par116">For cross-resolution mapping, we constructed the cost matrix using niche and territory. We defined territories by processing each dataset through the Vesalius pipeline, which includes color histogram equalization, image smoothing, and image segmentation. The parameter details are available in the analysis scripts deposited in our dedicated GitHub page (see â€œCode availabilityâ€ section). We defined niches using a radius from the center cell, where the radius is defined as half of the scaled distance between Visium Spots. Cell matching was achieved across 20 epochs with a batch size equal to the number of Visium spots (<em>n</em>â€‰=â€‰2310). We used log-normalized variable features as the expression signal to compare. Once the matching was completed, we ran the mapped dataset through the Vesalius pipeline to retrieve spatial domains. We added a jitter to cell coordinates to enable each mapped cell to have a distinct spatial coordinate. In contrast, no jitter will merge the mapped cells into Visium-like spots. To compare the cell types mapped across resolution, we first scaled the cell type frequencies in the deconvoluted Visium Spots to assume 100 total cells in each spot. We applied the same frequency scaling to the cell type mapped from VisiumHD. We employed this strategy since RCTD returns an estimated cell type proportion and not a number of cells, and the number of cells can vary drastically between spots. Using these scaled cell type frequencies, we constructed a contingency matrix upon which we applied a Chi-squared test to determine if the cell type frequencies were statistically distinct. In addition, we computed a Jaccard Index between cell type labels in the Visium spots and mapped VisiumHD cells to determine how many cell types were being missed despite having potentially low occurrence frequencies.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Count integration</h3>
<p id="Par117">Once cells have been matched across samples, we can integrate the counts and develop an integrated tissue analysis framework. Count integration was only required during the analysis related to Stereo-seq Axolotl brain regeneration and Stereo-seq Mouse embryo across developmental time. To integrate counts, we used the Seurat implementation of Canonical Correlation Analysis<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup> (CCA), which returns scaled and integrated counts and a common latent space between samples. Vesalius can utilize all methods deployed by Seurat (e.g., scVI or Harmony). The integrated latent space is directly used to generate Vesalius images upon which our pipeline can be applied. To avoid duplicated coordinates, we added a jitter to the duplicated coordinates only. The resulting territories are the territories emerging from this joint latent space, which can be used for spatially resolved differentially expressed gene expression analysis.</p>
<p id="Par118">For inter-sample differential gene expression analysis, Vesalius can compare the cells from the integrated count matrix using either territories or cells as a grouping criterion. By default, we used the Wilcoxon ranked sum test with the p-value threshold set at 0.05 after correcting for multiple testing (FDR).</p></section><section id="Sec22"><h3 class="pmc_sec_title">Cell mapping clusters</h3>
<p id="Par119">We developed a cell clustering method that clusters query cells based on which reference cells they tend to co-map to. Our default clustering approach is based on hierarchical clustering with community-based clustering approaches (Leiden and Louvain) also available. First, we define the metric matrix that will be used for clustering. This approach follows the same pair-wise summation of similarity scores used during cost matrix creation described above. By default, we use cell similarity only. We define co-mapping of cells using the following steps:</p>
<p id="Par120">Order of preferential mapping</p>
<p id="Par121">For each cell <em>c</em>, let <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><math id="d33e2435"><mi>M</mi><mfenced close=")" open="("><mrow><mi>c</mi></mrow></mfenced><mo>=</mo><mrow><mo>{</mo><mrow><msub><mrow><mi>m</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>,</mo><mo>â€¦</mo><mo>,</mo><msub><mrow><mi>m</mi></mrow><mrow><mi>k</mi></mrow></msub></mrow><mo>}</mo></mrow></math></span> be the set of mapping costs where <em>m</em><sub><em>i</em></sub> represents the cost of mapping to reference cell <em>i</em>. The preferential mapping order is given by sorting <em>M</em>(<em>c</em>) in descending order:</p>
<table class="disp-formula p" id="Equ12"><tr>
<td class="formula"><math id="d33e2488" display="block"><msup><mrow><mi>M</mi></mrow><mrow><mo>*</mo></mrow></msup><mfenced close=")" open="("><mrow><mi>c</mi></mrow></mfenced><mo>=</mo><mi mathvariant="normal">sort</mi><mrow><mo>(</mo><mrow><mi>M</mi><mfenced close=")" open="("><mrow><mi>c</mi></mrow></mfenced></mrow><mo>)</mo></mrow></math></td>
<td class="label">12</td>
</tr></table>
<p id="Par122">The numerical values are converted into categorical strings.</p>
<p id="Par123">Selection of top <em>n</em> matches</p>
<p id="Par124">We select the top <em>n</em> matches (default <em>n</em>â€‰=â€‰30):</p>
<table class="disp-formula p" id="Equ13"><tr>
<td class="formula"><math id="d33e2530" display="block"><mi>S</mi><mfenced close=")" open="("><mrow><mi>c</mi></mrow></mfenced><mo>=</mo><mrow><mo>{</mo><mrow><msubsup><mrow><mi>m</mi></mrow><mrow><mn>1</mn></mrow><mrow><mo>*</mo></mrow></msubsup><mo>,</mo><msubsup><mrow><mi>m</mi></mrow><mrow><mn>2</mn></mrow><mrow><mo>*</mo></mrow></msubsup><mo>,</mo><mo>â€¦</mo><mo>,</mo><msubsup><mrow><mi>m</mi></mrow><mrow><mi>n</mi></mrow><mrow><mo>*</mo></mrow></msubsup></mrow><mo>}</mo></mrow></math></td>
<td class="label">13</td>
</tr></table>
<p>Where <em>S</em>(<em>c</em>) represents the categorical vector of the top <em>n</em> mappings for cell c.</p>
<p id="Par125">Jaccard Index Computation</p>
<p id="Par126">Given two query cells <em>c</em><sub>1</sub> and <em>c</em><sub>2</sub>, their Jaccard Index is computed as:</p>
<table class="disp-formula p" id="Equ14"><tr>
<td class="formula"><math id="d33e2595" display="block"><mi>J</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></mfenced><mo>=</mo><mfrac><mrow><mo>âˆ£</mo><mi>S</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></mfenced><mo>âˆª</mo><mi>S</mi><mrow><mo>(</mo><mrow><msub><mrow><mi>C</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow><mo>)</mo></mrow><mo>âˆ£</mo></mrow><mrow><mi>S</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></mfenced><mo>âˆ©</mo><mi>S</mi><mrow><mo>(</mo><mrow><msub><mrow><mi>C</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow><mo>)</mo></mrow></mrow></mfrac></math></td>
<td class="label">14</td>
</tr></table>
<p id="Par127">A higher <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><math id="d33e2662"><mi>J</mi><mrow><mo>(</mo><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow><mo>)</mo></mrow></math></span> indicates a greater likelihood that two query cells map to the same set of reference cells.</p>
<p id="Par128">Distance Matrix for Hierarchical Clustering</p>
<p id="Par129">The reciprocal of the Jaccard Index is used as the distance matrix for hierarchical clustering or community clustering algorithms.</p>
<table class="disp-formula p" id="Equ15"><tr>
<td class="formula"><math id="d33e2692" display="block"><mi>d</mi><mfenced close=")" open="("><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></mfenced><mo>=</mo><mspace width="0.25em"></mspace><mn>1</mn><mo>âˆ’</mo><mi>J</mi><mrow><mo>(</mo><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow><mo>)</mo></mrow></math></td>
<td class="label">15</td>
</tr></table>
<p id="Par130">For clarity, we used a fixed number of clusters with <em>k</em>â€‰=â€‰5 in the context of Stereo-seq mouse embryo clustering.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Mapping across spatially heterogeneous prostate cancer samples</h3>
<p id="Par131">We mapped cells from 2 prostate cancer samples using Slide-seq V2 technology<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. In this instance, we downloaded the count matrices, spatial barcodes, and cell type annotations provided by the authors. We processed 2 tumor samples (Tumor01 and Tumor02) using the Vesalius pipeline. In brief, we normalized gene counts, selected variable features (<em>n</em>â€‰=â€‰2000), and reduced dimensionality via PCA (30 PCs). Next, we smoothed the Vesalius image stacks and equalized the color histogram before segmenting the images and isolating territories. We mapped each sample to each other using cell, niche (KNN with <em>k</em>â€‰=â€‰15), territory similarities, and cell type labels. The mapping procedure was carried out across 25 epochs with a batch size set to 5000, consistent with our previous analysis. The details of the analysis are available on the dedicated GitHub.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Spatial proteomics in situ mass cytometry sample preparation and processing</h3>
<p id="Par132">To demonstrate the utility of Vesalius, we mapped spatial in situ mass cytometry (IMC) data taken from human breast cancer patient samples<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Out of the 709 samples, we filtered the samples to ensure that they contained at least 1000 cells. We also filtered out samples that contained NAs in the clinical metrics provided by the authors (ER Status, ERBB2_pos, grade, deathBreast, PAM50, and YearsToStatus). From the remaining samples, we randomly sampled 50 ER-positive and 50 ER-negative samples. Next, we extracted the expression values for the 37 markers measured, which we formatted into a count matrix for Vesalius. We processed every sample using the Vesalius pipeline, including log-normalization, dimensionality reduction (PCA) and image processing, and territory isolation. The details of the parameters used are contained in our analysis scripts deposited in a dedicated GitHub page (see â€œCode availabilityâ€ section).</p>
<p id="Par133">We mapped cells between samples using a cost matrix built from cell similarity, niche similarity, territory similarity, cell label, and niche composition. We optimized the matching pairs across 10 epochs with a batch size equal to the smallest dataset minus one (or 1000 - 1 cells if both datasets were larger than 1000 cells). The neighborhood was defined through our graph method with a depth of 2. For each mapping event, we extracted the mean cost and mean mapping scores.</p>
<p id="Par134">For each sample, we selected its best-matching pair (excluding self-mapping) by taking the pair with the lowest overall cost. From this best-matching pair, we computed the overlap between the clinical metrics to check whether our mapping strategy could recover patient-level information. More specifically, we checked if we obtained the same label between the reference and query. In the case of Grade and PAM50, there are multiple labels: â€œ1â€, â€œ2â€, â€œ3â€ for Grade and â€œBasalâ€, â€œHER2â€, â€œLuminal Aâ€, â€œLuminal Bâ€, â€œNormal-likeâ€ for PAM50. The delta <em>YearsToDeath</em> was calculated by taking the absolute value of the difference in years between query and reference.</p>
<table class="disp-formula p" id="Equ16"><tr>
<td class="formula"><math id="d33e2768" display="block"><mi mathvariant="normal">Î”</mi><mo>=</mo><mo>âˆ£</mo><mi>Q</mi><mo>âˆ’</mo><mi>R</mi><mo>âˆ£</mo></math></td>
<td class="label">16</td>
</tr></table></section><section id="Sec25"><h3 class="pmc_sec_title">Sample mapping clusters</h3>
<p id="Par135">To cluster samples (IMC), we first mapped samples to each other, extracted their average cost, and averaged mapping scores across all cells in those samples. Since we are mapping samples to each other, we can directly use the mapping cost matrix as input to the hierarchical clustering algorithm. In the case of IMC, we used height to define the number of cluster cutoff (<em>h</em>â€‰=â€‰0.5).</p></section><section id="Sec26"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par136">Further information on research design is available in theÂ <a href="#MOESM7" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec27"><h2 class="pmc_sec_title"><sup>Supplementary information</sup></h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (36.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62782_MOESM2_ESM.pdf</a><sup> (41.4KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM3_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (11.5MB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM4_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (583.7KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM5_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (18.3KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (223KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (75.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (10.6MB, pdf) </sup>
</div></div></section></section><section id="Sec28"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM9"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM9_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (51.4MB, zip) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>The authors gratefully acknowledge the support of Cedars-Sinai Medical Center through institutional funding. This study was also supported by VCU Massey Cancer Center and School of Medicine start-up funds, VCU, cMEDA Seed Award, the Adenoid Cystic Carcinoma Research Foundation, the Swiss Cancer League, Seeds of Science, SNSF, FundaÃ§Ã£o para a CiÃªncia e a Tecnologia, Fundamental Mandates (Stichting tegen Kankerâ€”Fondation contre le Cancer), and CSGS, 3P30CA016059âˆ’42S1, NCI, NIH to R.G.; VCU Massey Cancer Center and School of Medicine start-up funds and Tinaâ€™s Wish Foundation Award and CSGS, 3P30CA016059âˆ’42S4, NCI, NIH to E. Madan.;Â R.G and P.B.F acknowledge the support for this project and program from the VCU Institute of Molecular Medicine (VIMM).;Â support from NIH/NCI R01 CA280194 (PBF).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>P.C.N.M. and K.J.W. conceptualized the present study. P.C.N.M., W.W., and H.H. developed the methodology and software. P.C.N.M., W.W., and H.K. performed analysis, including benchmarks, data annotation, and data visualization. P.C.N.M., K.J.W., and R.G. wrote the draft of the manuscript. K.J.W. and R.G. supervised the project. Reviews and editing were performed by P.C.N.M., W.W., H.K., H.H., P.B.F., A.P.S., R.W., E.M., R.G., and K.J.W.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par137"><em>Nature Communications</em> thanks Assya Legrini and the other anonymous reviewer(s) for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All datasets used in this study are publicly available datasets. We provide the URLs below. Simulated datasets can be generated using the dedicated R package. Source Data are provided with this paper. <em>Simulated data</em>: <a href="https://github.com/WonLab-CS/oneiric" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/WonLab-CS/oneiric</a><em>Slide-seq V2</em>: <a href="https://singlecell.broadinstitute.org/single_cell/study/SCP815/sensitive-spatial-genome-wide-expression-profiling-at-cellular-resolution#study-summary" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://singlecell.broadinstitute.org/single_cell/study/SCP815/sensitive-spatial-genome-wide-expression-profiling-at-cellular-resolution#study-summary</a><em>seqFISH</em>: <a href="https://content.cruk.cam.ac.uk/jmlab/SpatialMouseAtlas2020/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://content.cruk.cam.ac.uk/jmlab/SpatialMouseAtlas2020/</a><em>MOSTA</em>: <a href="https://db.cngb.org/stomics/mosta/stereo.seq/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://db.cngb.org/stomics/mosta/stereo.seq/</a><em>Visium</em>: <a href="https://cf.10xgenomics.com/samples/spatial-exp/2.0.0/CytAssist_FFPE_Mouse_Brain_Rep1/CytAssist_FFPE_Mouse_Brain_Rep1_web_summary.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cf.10xgenomics.com/samples/spatial-exp/2.0.0/CytAssist_FFPE_Mouse_Brain_Rep1/CytAssist_FFPE_Mouse_Brain_Rep1_web_summary.html</a><em>VisiumHD</em>: <a href="https://www.10xgenomics.com/datasets/visium-hd-cytassist-gene-expression-libraries-of-mouse-brain-he" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.10xgenomics.com/datasets/visium-hd-cytassist-gene-expression-libraries-of-mouse-brain-he</a> scRNA: Molecular Diversity and Specializations among the Cells of the Adult Mouse Brain<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. <em>ARTISTA</em>: <a href="https://db.cngb.org/stomics/artista/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://db.cngb.org/stomics/artista/</a> Prostate Cancer: Dissecting the immune suppressive human prostate tumor microenvironment via integrated single-cell and spatial transcriptomic analyses<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>
<em>IMC</em>: Breast tumor microenvironment structures are associated with genomic features and clinical outcome<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>Â <a href="#Sec28" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="notes4"><h2 class="pmc_sec_title">Code availability</h2>
<p>The Vesalius package used in this study is publicly available and has been deposited in GitHub at <a href="https://github.com/WonLab-CS/Vesalius" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/WonLab-CS/Vesalius</a> under GPL-3.0. The Oneiric package used to generate the synthetic data is publicly available and has been deposited in GitHub at <a href="https://github.com/WonLab-CS/oneiric" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/WonLab-CS/oneiric</a> under GPL-3.0. The code used to perform analysis and generate results is publicly available and has been deposited in GitHub at <a href="https://github.com/WonLab-CS/Vesalius_analysis" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/WonLab-CS/Vesalius_analysis</a> under GPL-3.0. The specific version of the code (packages and analysis) associated with this publication is archived in Zenodo and is accessible via: 10.5281/zenodo.15733817<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup></p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par138">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1"><p><strong>Publisherâ€™s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div>
<div class="fn p" id="fn2"><p>These authors jointly supervised this work: Rajan Gogna, Kyoung Jae Won.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Rajan Gogna, Email: rajan.gogna@vcuhealth.org.</p>
<p>Kyoung Jae Won, Email: KyoungJae.Won@cshs.org.</p></section><section id="sec29"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-62782-y.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Mukherjee, S. <em>The Song of the Cell: An Exploration of Medicine and the New Human</em> (Scribner, 2022).</cite>
</li>
<li id="CR2">
<span class="label">2.</span><cite>Burgess, D. J. in <em>Nature Reviews Genetics</em> 20 317â€“317 (Nature Publishing Group, 2019).</cite>
</li>
<li id="CR3">
<span class="label">3.</span><cite>Liao, L., Martin, P. C. N., Kim, H., Panahandeh, S. &amp; Won, K. J. in <em>Advances in Cancer Research</em> (Academic Press, 2024).</cite> [<a href="https://doi.org/10.1016/bs.acr.2024.06.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39271267/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Moor, A. E. &amp; Itzkovitz, S. in <em>Current Opinion in Biotechnology</em> 46 126âˆ’133 (Elsevier Ltd, 2017).</cite> [<a href="https://doi.org/10.1016/j.copbio.2017.02.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28346891/" class="usa-link">PubMed</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Moses, L. &amp; Pachter, L. Museum of spatial transcriptomics. <em>Nat. Methods</em><strong>19</strong>, 534â€“546 (2022).
</cite> [<a href="https://doi.org/10.1038/s41592-022-01409-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35273392/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moses,%20L.%20&amp;%20Pachter,%20L.%20Museum%20of%20spatial%20transcriptomics.%20Nat.%20Methods19,%20534%E2%80%93546%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Rao, A., Barkley, D., FranÃ§a, G. S. &amp; Yanai, I. Exploring tissue architecture using spatial transcriptomics. <em>Nature</em><strong>596</strong>, 211â€“220 (2021).</cite> [<a href="https://doi.org/10.1038/s41586-021-03634-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8475179/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34381231/" class="usa-link">PubMed</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Rodriques, S. G. et al. Slide-seq: a scalable technology for measuring genome-wide expression at high spatial resolution. <em>Science</em><strong>363</strong>, 1463â€“1467 (2019).
</cite> [<a href="https://doi.org/10.1126/science.aaw1219" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6927209/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30923225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rodriques,%20S.%20G.%20et%20al.%20Slide-seq:%20a%20scalable%20technology%20for%20measuring%20genome-wide%20expression%20at%20high%20spatial%20resolution.%20Science363,%201463%E2%80%931467%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Stickels, R. R. et al. Highly sensitive spatial transcriptomics at near-cellular resolution with Slide-seqV2. <em>Nat. Biotechnol.</em><strong>39</strong>, 313â€“319 (2021).
</cite> [<a href="https://doi.org/10.1038/s41587-020-0739-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8606189/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33288904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stickels,%20R.%20R.%20et%20al.%20Highly%20sensitive%20spatial%20transcriptomics%20at%20near-cellular%20resolution%20with%20Slide-seqV2.%20Nat.%20Biotechnol.39,%20313%E2%80%93319%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Chen, A. et al. Spatiotemporal transcriptomic atlas of mouse organogenesis using DNA nanoball-patterned arrays. <em>Cell</em><strong>185</strong>, 1777â€“1792.e1721 (2022).</cite> [<a href="https://doi.org/10.1016/j.cell.2022.04.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35512705/" class="usa-link">PubMed</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>StÃ¥hl, P. L. et al. in <em>Science</em> 353 78â€“82 (American Association for the Advancement of Science, 2016).</cite>
</li>
<li id="CR11">
<span class="label">11.</span><cite>Eng, C. H. L. et al. Transcriptome-scale super-resolved imaging in tissues by RNA seqFISH+. <em>Nature</em><strong>568</strong>, 235â€“239 (2019).
</cite> [<a href="https://doi.org/10.1038/s41586-019-1049-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6544023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30911168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eng,%20C.%20H.%20L.%20et%20al.%20Transcriptome-scale%20super-resolved%20imaging%20in%20tissues%20by%20RNA%20seqFISH+.%20Nature568,%20235%E2%80%93239%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Cho, C.-S. S. et al. Microscopic examination of spatial transcriptome using Seq-Scope. <em>Cell</em><strong>184</strong>, 3559â€“3572.e3522 (2021).
</cite> [<a href="https://doi.org/10.1016/j.cell.2021.05.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8238917/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34115981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cho,%20C.-S.%20S.%20et%20al.%20Microscopic%20examination%20of%20spatial%20transcriptome%20using%20Seq-Scope.%20Cell184,%203559%E2%80%933572.e3522%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Dries, R. et al. Giotto: a toolbox for integrative analysis and visualization of spatial expression data. <em>Genome Biol.</em><strong>22</strong>, 1â€“31 (2021).
</cite> [<a href="https://doi.org/10.1186/s13059-021-02286-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7938609/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33685491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dries,%20R.%20et%20al.%20Giotto:%20a%20toolbox%20for%20integrative%20analysis%20and%20visualization%20of%20spatial%20expression%20data.%20Genome%20Biol.22,%201%E2%80%9331%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>He, B. et al. Integrating spatial gene expression and breast tumour morphology via deep learning. <em>Nat. Biomed. Eng.</em><strong>4</strong>, 827â€“834 (2020).
</cite> [<a href="https://doi.org/10.1038/s41551-020-0578-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32572199/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He,%20B.%20et%20al.%20Integrating%20spatial%20gene%20expression%20and%20breast%20tumour%20morphology%20via%20deep%20learning.%20Nat.%20Biomed.%20Eng.4,%20827%E2%80%93834%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Hunter, M. V., Moncada, R., Weiss, J. M., Yanai, I. &amp; White, R. M. Spatially resolved transcriptomics reveals the architecture of the tumor-microenvironment interface. <em>Nat. Commun.</em><strong>12</strong>, 1â€“16 (2021).</cite> [<a href="https://doi.org/10.1038/s41467-021-26614-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8560802/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34725363/" class="usa-link">PubMed</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Janesick, A. et al. High resolution mapping of the tumor microenvironment using integrated single-cell, spatial and in situ analysis. <em>Nat. Commun</em>. 10.1038/s41467-023-43458-x (2023).</cite> [<a href="https://doi.org/10.1038/s41467-023-43458-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10730913/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38114474/" class="usa-link">PubMed</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Kuppe, C. et al. Spatial multi-omic map of human myocardial infarction. <em>Nature</em><strong>608</strong>, 766â€“777 (2022).</cite> [<a href="https://doi.org/10.1038/s41586-022-05060-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9364862/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35948637/" class="usa-link">PubMed</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Madan, E. et al. Flower isoforms promote competitive growth in cancer. <em>Nature</em><strong>572</strong>, 260â€“264 (2019).
</cite> [<a href="https://doi.org/10.1038/s41586-019-1429-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31341286/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Madan,%20E.%20et%20al.%20Flower%20isoforms%20promote%20competitive%20growth%20in%20cancer.%20Nature572,%20260%E2%80%93264%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Madan, E. et al. Ovarian tumor cells gain competitive advantage by actively reducing the cellular fitness of microenvironment cells. <em>Nat. Biotechnol.</em>10.1038/s41587-024-02453-3 (2024).</cite> [<a href="https://doi.org/10.1038/s41587-024-02453-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39653752/" class="usa-link">PubMed</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Browaeys, R., Saelens, W. &amp; Saeys, Y. NicheNet: modeling intercellular communication by linking ligands to target genes. <em>Nat. Methods</em><strong>17</strong>, 159â€“162 (2019).
</cite> [<a href="https://doi.org/10.1038/s41592-019-0667-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31819264/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Browaeys,%20R.,%20Saelens,%20W.%20&amp;%20Saeys,%20Y.%20NicheNet:%20modeling%20intercellular%20communication%20by%20linking%20ligands%20to%20target%20genes.%20Nat.%20Methods17,%20159%E2%80%93162%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Cang, Z. &amp; Nie, Q. Inferring spatial and signaling relationships between cells from single cell transcriptomic data. <em>Nat. Commun.</em><strong>11</strong>, 1â€“13 (2020).
</cite> [<a href="https://doi.org/10.1038/s41467-020-15968-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7190659/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32350282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cang,%20Z.%20&amp;%20Nie,%20Q.%20Inferring%20spatial%20and%20signaling%20relationships%20between%20cells%20from%20single%20cell%20transcriptomic%20data.%20Nat.%20Commun.11,%201%E2%80%9313%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Efremova, M., Vento-Tormo, M., Teichmann, S. A. &amp; Vento-Tormo, R. CellPhoneDB: inferring cellâ€“cell communication from combined expression of multi-subunit ligandâ€“receptor complexes. <em>Nature Protocols</em><strong>15</strong>, 1484â€“1506 (2020).</cite> [<a href="https://doi.org/10.1038/s41596-020-0292-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32103204/" class="usa-link">PubMed</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Kim, H. et al. CellNeighborEX: deciphering neighborâ€dependent gene expression from spatial transcriptomics data. <em>Molecular Systems Biology</em><strong>19</strong>10.15252/msb.202311670 (2023).</cite> [<a href="https://doi.org/10.15252/msb.202311670" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10632736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37815040/" class="usa-link">PubMed</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Martin, P. C. N., Kim, H., LÃ¶vkvist, C., Hong, B. W. &amp; Won, K. J. Vesalius: high-resolution in silico anatomization of spatial transcriptomic data using image analysis. <em>Mol. Syst. Biol</em>. 10.15252/msb.202211080 (2022).</cite> [<a href="https://doi.org/10.15252/msb.202211080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9446088/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36065846/" class="usa-link">PubMed</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Maynard, K. R. et al. Transcriptome-scale spatial gene expression in the human dorsolateral prefrontal cortex. <em>Nat. Neurosci.</em>10.1038/s41593-020-00787-0 (2021).</cite> [<a href="https://doi.org/10.1038/s41593-020-00787-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8095368/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33558695/" class="usa-link">PubMed</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Wei, X. et al. Single-cell Stereo-seq reveals induced progenitor cells involved in axolotl brain regeneration. <em>Science</em><strong>377</strong>, eabp9444 (2022).
</cite> [<a href="https://doi.org/10.1126/science.abp9444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36048929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wei,%20X.%20et%20al.%20Single-cell%20Stereo-seq%20reveals%20induced%20progenitor%20cells%20involved%20in%20axolotl%20brain%20regeneration.%20Science377,%20eabp9444%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Zeira, R., Land, M., Strzalkowski, A. &amp; Raphael, B. J. Alignment and integration of spatial transcriptomics data. <em>Nat. Methods</em><strong>19</strong>, 567â€“575 (2022).
</cite> [<a href="https://doi.org/10.1038/s41592-022-01459-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9334025/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35577957/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeira,%20R.,%20Land,%20M.,%20Strzalkowski,%20A.%20&amp;%20Raphael,%20B.%20J.%20Alignment%20and%20integration%20of%20spatial%20transcriptomics%20data.%20Nat.%20Methods19,%20567%E2%80%93575%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Jones, A., Townes, F. W., Li, D. &amp; Engelhardt, B. E. Alignment of spatial genomics data using deep Gaussian processes. <em>Nat. Methods</em><strong>20</strong>, 1379â€“1387 (2023).
</cite> [<a href="https://doi.org/10.1038/s41592-023-01972-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10482692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37592182/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jones,%20A.,%20Townes,%20F.%20W.,%20Li,%20D.%20&amp;%20Engelhardt,%20B.%20E.%20Alignment%20of%20spatial%20genomics%20data%20using%20deep%20Gaussian%20processes.%20Nat.%20Methods20,%201379%E2%80%931387%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Clifton, K. et al. STalign: alignment of spatial transcriptomics data using diffeomorphic metric mapping. <em>Nat. Commun</em>. 10.1038/s41467-023-43915-7 (2023).</cite> [<a href="https://doi.org/10.1038/s41467-023-43915-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10709594/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38065970/" class="usa-link">PubMed</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Biancalani, T. et al. Deep learning and alignment of spatially resolved single-cell transcriptomes with Tangram. <em>Nature Methods</em><strong>18</strong>, 1352â€“1362 (2021).</cite> [<a href="https://doi.org/10.1038/s41592-021-01264-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8566243/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34711971/" class="usa-link">PubMed</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Vahid, M. R. et al. High-resolution alignment of single-cell and spatial transcriptomes with CytoSPACE. <em>Nat. Biotechnol.</em><strong>41</strong>, 1543â€“1548 (2023).
</cite> [<a href="https://doi.org/10.1038/s41587-023-01697-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10635828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36879008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vahid,%20M.%20R.%20et%20al.%20High-resolution%20alignment%20of%20single-cell%20and%20spatial%20transcriptomes%20with%20CytoSPACE.%20Nat.%20Biotechnol.41,%201543%E2%80%931548%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Hie, B., Bryson, B. &amp; Berger, B. Efficient integration of heterogeneous single-cell transcriptomes using Scanorama. <em>Nat. Biotechnol.</em><strong>37</strong>, 685â€“691 (2019).
</cite> [<a href="https://doi.org/10.1038/s41587-019-0113-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6551256/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31061482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hie,%20B.,%20Bryson,%20B.%20&amp;%20Berger,%20B.%20Efficient%20integration%20of%20heterogeneous%20single-cell%20transcriptomes%20using%20Scanorama.%20Nat.%20Biotechnol.37,%20685%E2%80%93691%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Xia, C.-R., Cao, Z.-J., Tu, X.-M. &amp; Gao, G. Spatial-linked alignment tool (SLAT) for aligning heterogenous slices. <em>Nat. Commun</em>. 10.1038/s41467-023-43105-5 (2023).</cite> [<a href="https://doi.org/10.1038/s41467-023-43105-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10636043/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37945600/" class="usa-link">PubMed</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Lohoff, T. et al. Integration of spatial and single-cell transcriptomic data elucidates mouse organogenesis. <em>Nat. Biotechnol.</em><strong>40</strong>, 74â€“85 (2021).</cite> [<a href="https://doi.org/10.1038/s41587-021-01006-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8763645/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34489600/" class="usa-link">PubMed</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Jonker, R. &amp; Volgenant, A. A shortest augmenting path algorithm for dense and sparse linear assignment problems. <em>Computing</em><strong>38</strong>, 325â€“340 (1987).</cite> [<a href="https://scholar.google.com/scholar_lookup?Jonker,%20R.%20&amp;%20Volgenant,%20A.%20A%20shortest%20augmenting%20path%20algorithm%20for%20dense%20and%20sparse%20linear%20assignment%20problems.%20Computing38,%20325%E2%80%93340%20(1987)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Rand, W. M. Objective Criteria for the Evaluation of Clustering Methods. <em>J. Am. Stat. Assoc.</em><strong>66</strong>, 846â€“850 (1971).</cite> [<a href="https://scholar.google.com/scholar_lookup?Rand,%20W.%20M.%20Objective%20Criteria%20for%20the%20Evaluation%20of%20Clustering%20Methods.%20J.%20Am.%20Stat.%20Assoc.66,%20846%E2%80%93850%20(1971)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Cable, D. M. et al. Robust decomposition of cell type mixtures in spatial transcriptomics. <em>Nat. Biotechnol</em>. 10.1038/s41587-021-00830-w (2021).</cite> [<a href="https://doi.org/10.1038/s41587-021-00830-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8606190/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33603203/" class="usa-link">PubMed</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Saunders, A. et al. Molecular diversity and specializations among the cells of the adult mouse brain. <em>Cell</em><strong>174</strong>, 1015â€“1030.e1016 (2018).
</cite> [<a href="https://doi.org/10.1016/j.cell.2018.07.028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6447408/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30096299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saunders,%20A.%20et%20al.%20Molecular%20diversity%20and%20specializations%20among%20the%20cells%20of%20the%20adult%20mouse%20brain.%20Cell174,%201015%E2%80%931030.e1016%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Nhieu, J. et al. Targeting cellular retinoic acid binding protein 1 with retinoic acid-like compounds to mitigate motor neuron degeneration. <em>Int. J. Mol. Sci</em>. 10.3390/ijms24054980 (2023).</cite> [<a href="https://doi.org/10.3390/ijms24054980" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10002585/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36902410/" class="usa-link">PubMed</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Lin, Y. L., Persaud, S. D., Nhieu, J. &amp; Wei, L. N. Cellular retinoic acid-binding protein 1 modulates stem cell proliferation to affect learning and memory in male mice. <em>Endocrinology</em><strong>158</strong>, 3004â€“3014 (2017).
</cite> [<a href="https://doi.org/10.1210/en.2017-00353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5659671/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28911165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20Y.%20L.,%20Persaud,%20S.%20D.,%20Nhieu,%20J.%20&amp;%20Wei,%20L.%20N.%20Cellular%20retinoic%20acid-binding%20protein%201%20modulates%20stem%20cell%20proliferation%20to%20affect%20learning%20and%20memory%20in%20male%20mice.%20Endocrinology158,%203004%E2%80%933014%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Danenberg, E. et al. Breast tumor microenvironment structures are associated with genomic features and clinical outcome. <em>Nat. Genet.</em><strong>54</strong>, 660â€“669 (2022).
</cite> [<a href="https://doi.org/10.1038/s41588-022-01041-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7612730/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35437329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Danenberg,%20E.%20et%20al.%20Breast%20tumor%20microenvironment%20structures%20are%20associated%20with%20genomic%20features%20and%20clinical%20outcome.%20Nat.%20Genet.54,%20660%E2%80%93669%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Zappia, L., Phipson, B. &amp; Oshlack, A. Splatter: simulation of single-cell RNA sequencing data. <em>Genome Biol</em>. 10.1186/s13059-017-1305-0 (2017).</cite> [<a href="https://doi.org/10.1186/s13059-017-1305-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5596896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28899397/" class="usa-link">PubMed</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Stuart, T. et al. Comprehensive integration of single-cell data resource comprehensive integration of single-cell data. <em>Cell</em><strong>177</strong>, 1888â€“1902.e1821 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.05.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6687398/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31178118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stuart,%20T.%20et%20al.%20Comprehensive%20integration%20of%20single-cell%20data%20resource%20comprehensive%20integration%20of%20single-cell%20data.%20Cell177,%201888%E2%80%931902.e1821%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Hirz, T. et al. Dissecting the immune suppressive human prostate tumor microenvironment via integrated single-cell and spatial transcriptomic analyses. <em>Nat. Commun.</em><strong>14</strong>, 663 (2023).
</cite> [<a href="https://doi.org/10.1038/s41467-023-36325-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9905093/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36750562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hirz,%20T.%20et%20al.%20Dissecting%20the%20immune%20suppressive%20human%20prostate%20tumor%20microenvironment%20via%20integrated%20single-cell%20and%20spatial%20transcriptomic%20analyses.%20Nat.%20Commun.14,%20663%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Martin, P. et al. â€œMulti-scale and multi-context interpretable mapping of cell states across heterogenous spatial samples - archiveâ€. Vesalius. <em>Zenodo</em>10.5281/zenodo.15733817 (2025).</cite> [<a href="https://doi.org/10.1038/s41467-025-62782-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12370880/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40841370/" class="usa-link">PubMed</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (36.1MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_62782_MOESM2_ESM.pdf</a><sup> (41.4KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM3_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (11.5MB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM4_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 2</a><sup> (583.7KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM5_ESM.csv" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 3</a><sup> (18.3KB, csv) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 4</a><sup> (223KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM7_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (75.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM8_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (10.6MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12370880/bin/41467_2025_62782_MOESM9_ESM.zip" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (51.4MB, zip) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All datasets used in this study are publicly available datasets. We provide the URLs below. Simulated datasets can be generated using the dedicated R package. Source Data are provided with this paper. <em>Simulated data</em>: <a href="https://github.com/WonLab-CS/oneiric" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/WonLab-CS/oneiric</a><em>Slide-seq V2</em>: <a href="https://singlecell.broadinstitute.org/single_cell/study/SCP815/sensitive-spatial-genome-wide-expression-profiling-at-cellular-resolution#study-summary" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://singlecell.broadinstitute.org/single_cell/study/SCP815/sensitive-spatial-genome-wide-expression-profiling-at-cellular-resolution#study-summary</a><em>seqFISH</em>: <a href="https://content.cruk.cam.ac.uk/jmlab/SpatialMouseAtlas2020/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://content.cruk.cam.ac.uk/jmlab/SpatialMouseAtlas2020/</a><em>MOSTA</em>: <a href="https://db.cngb.org/stomics/mosta/stereo.seq/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://db.cngb.org/stomics/mosta/stereo.seq/</a><em>Visium</em>: <a href="https://cf.10xgenomics.com/samples/spatial-exp/2.0.0/CytAssist_FFPE_Mouse_Brain_Rep1/CytAssist_FFPE_Mouse_Brain_Rep1_web_summary.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cf.10xgenomics.com/samples/spatial-exp/2.0.0/CytAssist_FFPE_Mouse_Brain_Rep1/CytAssist_FFPE_Mouse_Brain_Rep1_web_summary.html</a><em>VisiumHD</em>: <a href="https://www.10xgenomics.com/datasets/visium-hd-cytassist-gene-expression-libraries-of-mouse-brain-he" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.10xgenomics.com/datasets/visium-hd-cytassist-gene-expression-libraries-of-mouse-brain-he</a> scRNA: Molecular Diversity and Specializations among the Cells of the Adult Mouse Brain<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. <em>ARTISTA</em>: <a href="https://db.cngb.org/stomics/artista/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://db.cngb.org/stomics/artista/</a> Prostate Cancer: Dissecting the immune suppressive human prostate tumor microenvironment via integrated single-cell and spatial transcriptomic analyses<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>
<em>IMC</em>: Breast tumor microenvironment structures are associated with genomic features and clinical outcome<sup><a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>Â <a href="#Sec28" class="usa-link">Source data</a> are provided with this paper.</p>
<p>The Vesalius package used in this study is publicly available and has been deposited in GitHub at <a href="https://github.com/WonLab-CS/Vesalius" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/WonLab-CS/Vesalius</a> under GPL-3.0. The Oneiric package used to generate the synthetic data is publicly available and has been deposited in GitHub at <a href="https://github.com/WonLab-CS/oneiric" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/WonLab-CS/oneiric</a> under GPL-3.0. The code used to perform analysis and generate results is publicly available and has been deposited in GitHub at <a href="https://github.com/WonLab-CS/Vesalius_analysis" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/WonLab-CS/Vesalius_analysis</a> under GPL-3.0. The specific version of the code (packages and analysis) associated with this publication is archived in Zenodo and is accessible via: 10.5281/zenodo.15733817<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup></p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-62782-y"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_62782.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (9.3Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370880/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370880/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370880%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370880/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370880/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370880/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841370/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370880/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841370/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370880/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370880/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="Oeb2PedTUTNA7zprQAmYgi7l0BiIFa5pK5poktmmhTTPoXzoAAj3o46WHETGBiIR">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
