
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Sex Differentiation and Long‐Distance Gene Flow in the Elusive Antarctic Fish Aethotaxis mitopteryx - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152546AB8AF32E130546AB0010FD77CF.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ece">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367274/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Ecology and Evolution">
<meta name="citation_title" content="Sex Differentiation and Long‐Distance Gene Flow in the Elusive Antarctic Fish Aethotaxis mitopteryx">
<meta name="citation_author" content="Luca Schiavon">
<meta name="citation_author_institution" content="Biology Department, University of Padova, Padova, Italy">
<meta name="citation_author" content="Thomas Desvignes">
<meta name="citation_author_institution" content="Institute of Neuroscience, University of Oregon, Eugene, Oregon, USA">
<meta name="citation_author_institution" content="Department of Biology, University of Alabama at Birmingham, Birmingham, Alabama, USA">
<meta name="citation_author" content="Fabrizia Ronco">
<meta name="citation_author_institution" content="Natural History Museum, University of Oslo, Oslo, Norway">
<meta name="citation_author_institution" content="Department of Environmental Sciences, University of Basel, Basel, Switzerland">
<meta name="citation_author" content="Michael Matschiner">
<meta name="citation_author_institution" content="Natural History Museum, University of Oslo, Oslo, Norway">
<meta name="citation_author" content="Martina Gastaldi">
<meta name="citation_author_institution" content="Biology Department, University of Padova, Padova, Italy">
<meta name="citation_author" content="Thore Koppetsch">
<meta name="citation_author_institution" content="Natural History Museum, University of Oslo, Oslo, Norway">
<meta name="citation_author" content="Audun Schrøder‐Nielsen">
<meta name="citation_author_institution" content="Natural History Museum, University of Oslo, Oslo, Norway">
<meta name="citation_author" content="John H Postlethwait">
<meta name="citation_author_institution" content="Institute of Neuroscience, University of Oregon, Eugene, Oregon, USA">
<meta name="citation_author" content="Alessia Prestanti">
<meta name="citation_author_institution" content="Biology Department, University of Padova, Padova, Italy">
<meta name="citation_author" content="Federica Stranci">
<meta name="citation_author_institution" content="Biology Department, University of Padova, Padova, Italy">
<meta name="citation_author" content="Santiago G Ceballos">
<meta name="citation_author_institution" content="Instituto de Ciencias Polares, Ambiente y Recursos Naturales (ICPA), Universidad Nacional de Tierra del Fuego (UNTDF), Ushuaia, Argentina">
<meta name="citation_author" content="Felix C Mark">
<meta name="citation_author_institution" content="Alfred Wegener Institute Helmholtz Centre for Polar and Marine Research, Bremerhaven, Germany">
<meta name="citation_author" content="Magnus Lucassen">
<meta name="citation_author_institution" content="Alfred Wegener Institute Helmholtz Centre for Polar and Marine Research, Bremerhaven, Germany">
<meta name="citation_author" content="Emilio Riginella">
<meta name="citation_author_institution" content="Zoological Station Anton Dohrn, Napoli, Italy">
<meta name="citation_author" content="Mario La Mesa">
<meta name="citation_author_institution" content="CNR‐ISP, Institute of Polar Sciences, Bologna, Italy">
<meta name="citation_author" content="Chiara Papetti">
<meta name="citation_author_institution" content="Biology Department, University of Padova, Padova, Italy">
<meta name="citation_author_institution" content="National Biodiversity Future Centre, Palermo, Italy">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="15">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e71847">
<meta name="citation_doi" content="10.1002/ece3.71847">
<meta name="citation_pmid" content="40843031">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367274/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367274/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367274/pdf/ECE3-15-e71847.pdf">
<meta name="description" content="Understanding population connectivity in the marine realm is crucial for conserving biodiversity, managing fisheries, and predicting species responses to environmental change. This is particularly important in Antarctic waters, where unique ...">
<meta name="og:title" content="Sex Differentiation and Long‐Distance Gene Flow in the Elusive Antarctic Fish Aethotaxis mitopteryx">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Understanding population connectivity in the marine realm is crucial for conserving biodiversity, managing fisheries, and predicting species responses to environmental change. This is particularly important in Antarctic waters, where unique ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367274/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12367274">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/ece3.71847"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ECE3-15-e71847.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367274%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12367274/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12367274/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367274/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ece.gif" alt="Ecology and Evolution logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Ecology and Evolution" title="Link to Ecology and Evolution" shape="default" href="http://onlinelibrary.wiley.com/journal/10.1002/(ISSN)2045-7758" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Ecol Evol</button></div>. 2025 Aug 20;15(8):e71847. doi: <a href="https://doi.org/10.1002/ece3.71847" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/ece3.71847</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Ecol%20Evol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ecol%20Evol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Ecol%20Evol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Ecol%20Evol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Sex Differentiation and Long‐Distance Gene Flow in the Elusive Antarctic Fish 
<em>Aethotaxis mitopteryx</em>

</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schiavon%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Luca Schiavon</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Luca Schiavon</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Biology Department, University of Padova, Padova, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schiavon%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Luca Schiavon</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Desvignes%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Thomas Desvignes</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Thomas Desvignes</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Institute of Neuroscience, University of Oregon, Eugene, Oregon, USA</div>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Biology, University of Alabama at Birmingham, Birmingham, Alabama, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Desvignes%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Thomas Desvignes</span></a>
</div>
</div>
<sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ronco%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Fabrizia Ronco</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Fabrizia Ronco</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Natural History Museum, University of Oslo, Oslo, Norway</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>Department of Environmental Sciences, University of Basel, Basel, Switzerland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ronco%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fabrizia Ronco</span></a>
</div>
</div>
<sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Matschiner%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Michael Matschiner</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Michael Matschiner</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Natural History Museum, University of Oslo, Oslo, Norway</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Matschiner%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael Matschiner</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gastaldi%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Martina Gastaldi</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Martina Gastaldi</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Biology Department, University of Padova, Padova, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gastaldi%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martina Gastaldi</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Koppetsch%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Thore Koppetsch</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Thore Koppetsch</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Natural History Museum, University of Oslo, Oslo, Norway</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Koppetsch%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Thore Koppetsch</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schr%C3%B8der%E2%80%90Nielsen%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Audun Schrøder‐Nielsen</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Audun Schrøder‐Nielsen</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Natural History Museum, University of Oslo, Oslo, Norway</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Schr%C3%B8der%E2%80%90Nielsen%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Audun Schrøder‐Nielsen</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Postlethwait%20JH%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">John H Postlethwait</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">John H Postlethwait</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Institute of Neuroscience, University of Oregon, Eugene, Oregon, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Postlethwait%20JH%22%5BAuthor%5D" class="usa-link"><span class="name western">John H Postlethwait</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prestanti%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Alessia Prestanti</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Alessia Prestanti</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Biology Department, University of Padova, Padova, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Prestanti%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alessia Prestanti</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stranci%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Federica Stranci</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Federica Stranci</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Biology Department, University of Padova, Padova, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Stranci%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Federica Stranci</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ceballos%20SG%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Santiago G Ceballos</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Santiago G Ceballos</span></h3>
<div class="p">
<sup>
<sup>6</sup>
</sup>Instituto de Ciencias Polares, Ambiente y Recursos Naturales (ICPA), Universidad Nacional de Tierra del Fuego (UNTDF), Ushuaia, Argentina</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ceballos%20SG%22%5BAuthor%5D" class="usa-link"><span class="name western">Santiago G Ceballos</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mark%20FC%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Felix C Mark</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Felix C Mark</span></h3>
<div class="p">
<sup>
<sup>7</sup>
</sup>Alfred Wegener Institute Helmholtz Centre for Polar and Marine Research, Bremerhaven, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mark%20FC%22%5BAuthor%5D" class="usa-link"><span class="name western">Felix C Mark</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lucassen%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Magnus Lucassen</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Magnus Lucassen</span></h3>
<div class="p">
<sup>
<sup>7</sup>
</sup>Alfred Wegener Institute Helmholtz Centre for Polar and Marine Research, Bremerhaven, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lucassen%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Magnus Lucassen</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Riginella%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Emilio Riginella</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Emilio Riginella</span></h3>
<div class="p">
<sup>
<sup>8</sup>
</sup>Zoological Station Anton Dohrn, Napoli, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Riginella%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Emilio Riginella</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22La%20Mesa%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Mario La Mesa</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Mario La Mesa</span></h3>
<div class="p">
<sup>
<sup>9</sup>
</sup>CNR‐ISP, Institute of Polar Sciences, Bologna, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22La%20Mesa%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mario La Mesa</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Papetti%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Chiara Papetti</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Chiara Papetti</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Biology Department, University of Padova, Padova, Italy</div>
<div class="p">
<sup>
<sup>10</sup>
</sup>National Biodiversity Future Centre, Palermo, Italy</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Papetti%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chiara Papetti</span></a>
</div>
</div>
<sup>1,</sup><sup>10,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="ece371847-aff-0001">
<sup>
<sup>1</sup>
</sup>Biology Department, University of Padova, Padova, Italy</div>
<div id="ece371847-aff-0002">
<sup>
<sup>2</sup>
</sup>Institute of Neuroscience, University of Oregon, Eugene, Oregon, USA</div>
<div id="ece371847-aff-0003">
<sup>
<sup>3</sup>
</sup>Department of Biology, University of Alabama at Birmingham, Birmingham, Alabama, USA</div>
<div id="ece371847-aff-0004">
<sup>
<sup>4</sup>
</sup>Natural History Museum, University of Oslo, Oslo, Norway</div>
<div id="ece371847-aff-0005">
<sup>
<sup>5</sup>
</sup>Department of Environmental Sciences, University of Basel, Basel, Switzerland</div>
<div id="ece371847-aff-0006">
<sup>
<sup>6</sup>
</sup>Instituto de Ciencias Polares, Ambiente y Recursos Naturales (ICPA), Universidad Nacional de Tierra del Fuego (UNTDF), Ushuaia, Argentina</div>
<div id="ece371847-aff-0007">
<sup>
<sup>7</sup>
</sup>Alfred Wegener Institute Helmholtz Centre for Polar and Marine Research, Bremerhaven, Germany</div>
<div id="ece371847-aff-0008">
<sup>
<sup>8</sup>
</sup>Zoological Station Anton Dohrn, Napoli, Italy</div>
<div id="ece371847-aff-0009">
<sup>
<sup>9</sup>
</sup>CNR‐ISP, Institute of Polar Sciences, Bologna, Italy</div>
<div id="ece371847-aff-0010">
<sup>
<sup>10</sup>
</sup>National Biodiversity Future Centre, Palermo, Italy</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong>
 
Chiara Papetti (<span>chiara.papetti@unipd.it</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jun 26; Accepted 2025 Jul 14; Collection date 2025 Aug.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Ecology and Evolution</em> published by British Ecological Society and John Wiley &amp; Sons Ltd.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12367274  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40843031/" class="usa-link">40843031</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>Understanding population connectivity in the marine realm is crucial for conserving biodiversity, managing fisheries, and predicting species responses to environmental change. This is particularly important in Antarctic waters, where unique evolutionary histories and extreme conditions shape marine biodiversity. The longfin icedevil 
<em>Aethotaxis mitopteryx</em>
 is an elusive notothenioid fish endemic to Antarctic waters. To explore population connectivity in 
<em>A. mitopteryx</em>
, we used RAD‐seq to investigate the genetic differentiation of two populations, one from the Eastern Weddell Sea and the other from the Eastern Antarctic Peninsula, two regions of ecological relevance greatly impacted by climate change. Despite spatial separation, analyses revealed no significant genetic differentiation between the two populations, suggesting extensive gene flow. A pronounced genetic distinction was, however, observed between males and females. This differentiation was largely localized to a specific chromosome, implying a genetic sex determination system with males being the heterogametic sex. These findings contribute novel insights into the genetic structure of 
<em>A. mitopteryx</em>
 populations and expand our understanding of genetic mechanisms in Antarctic fish. This study provides a foundation for further investigations into the evolutionary and ecological implications of sex chromosome differentiation in extreme environments.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> 
<em>Aethotaxis mitopteryx</em>
, 
<em>Dissostichus</em> spp., Notothenioidei, RAD‐sequencing, sex‐chromosomes, Southern Ocean</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p>This genetic study of a rare Antarctic fish, the longfin icedevil (
<em>Aethotaxis mitopteryx</em>
) investigates population connectivity across the Weddel Sea, revealing little to no differentiation even over vast geographic distances. Unexpectedly, the strongest genetic differentiation is instead observed between male and female fishes, which suggests a heterochromosome sex determination system in the longfin icedevil.</p>
<section class="bt xbox font-sm" id="ece371847-blkfxd-0001"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0594/12367274/b441506ace92/ECE3-15-e71847-g002.jpg" loading="lazy" id="jats-graphic-1" height="331" width="332" alt="graphic file with name ECE3-15-e71847-g002.jpg"></p></section></section><section id="ece371847-sec-0001"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Understanding population connectivity (i.e., the exchange of individuals among populations) and monitoring connectivity variability in the marine environment provide indications about a species' potential for population recovery and its capacity for re‐colonization and adaptation in case of demographic collapse (Hilário et al. <a href="#ece371847-bib-0054" class="usa-link" aria-describedby="ece371847-bib-0054">2015</a>). In the marine environment, variation in levels of intraspecific connectivity typically reflects the interacting effect of different biological variables (e.g., natural and sexual selection, population size, pelagic larval duration, adult migratory behavior or dispersal capacity that might be different in males and females in case of sex‐biased dispersal, mating strategies, maximum adult size, egg type and other life history traits) and physical variables (e.g., physical barriers, currents, temperature, etc.) (Ritchie et al. <a href="#ece371847-bib-0105" class="usa-link" aria-describedby="ece371847-bib-0105">2007</a>; Chopelet et al. <a href="#ece371847-bib-0023" class="usa-link" aria-describedby="ece371847-bib-0023">2008</a>). Studies that address the biophysical mechanisms and the spatial and temporal scale of population connectivity are important not only to enhance our understanding of the ecology and evolutionary trajectories of marine organisms, but also to refine conservation and management strategies. In particular, studies of genetic connectivity (i.e., gene flow, population genetic structure) are useful to trace short‐ and long‐term variations in patterns of differentiation and genetic variability in response to environmental parameters, species‐specific life history traits, and human exploitation, which are essential data for the design of conservation measures. This is especially true for spatial approaches to resource management, such as initiatives adopted by the Commission for the Conservation of Antarctic Marine Living Resources (CCAMLR) for a network of marine protected areas (MPAs), including one already established in the Ross Sea (CCAMLR <a href="#ece371847-bib-0019" class="usa-link" aria-describedby="ece371847-bib-0019">2016</a>) and in the South Orkney Islands (Trathan and Grant <a href="#ece371847-bib-0117" class="usa-link" aria-describedby="ece371847-bib-0117">2020</a>) and others under evaluation, such as one proposal in the Weddell Sea (Teschke et al. <a href="#ece371847-bib-0115" class="usa-link" aria-describedby="ece371847-bib-0115">2021</a>) and in the Western Antarctic Peninsula (CCAMLR <a href="#ece371847-bib-0020" class="usa-link" aria-describedby="ece371847-bib-0020">2020</a>). Ideally, decisions governing the choice of size, number, spacing, and location of protected areas reflect species‐specific patterns of population connectivity, dispersal, and use of habitat, gene flow, and genetic structure (Boscari et al. <a href="#ece371847-bib-0014" class="usa-link" aria-describedby="ece371847-bib-0014">2019</a>; Halpern <a href="#ece371847-bib-0051" class="usa-link" aria-describedby="ece371847-bib-0051">2003</a>).</p>
<p>In this context, Antarctic fish of the suborder Notothenioidei can serve as models for population connectivity because their distinct ecologies, life history traits, and the environmental and hydrographic context of the Southern Ocean provide insights into diverse scales of connectivity and imply different impacts of biophysical factors.</p>
<p>Notothenioidei account for over 70% of the fish species present in marine communities in the Southern Ocean over the Antarctic continental shelf, and for over 90% of the fish biomass (Eastman <a href="#ece371847-bib-0038" class="usa-link" aria-describedby="ece371847-bib-0038">2024</a>). They are considered one of the best examples of a marine species flock (Eastman and McCune <a href="#ece371847-bib-0040" class="usa-link" aria-describedby="ece371847-bib-0040">2000</a>; Lecointre et al. <a href="#ece371847-bib-0076" class="usa-link" aria-describedby="ece371847-bib-0076">2013</a>) and have evolved a range of shared and some unique physiological adaptations and life history characteristics. Well‐known adaptations to cold waters include the ability to synthesize antifreeze glycoproteins (AFGP) and antifreeze‐potentiating proteins (AFPP) (DeVries and Wohlschlag <a href="#ece371847-bib-0030" class="usa-link" aria-describedby="ece371847-bib-0030">1969</a>; Duman <a href="#ece371847-bib-0033" class="usa-link" aria-describedby="ece371847-bib-0033">2015</a>).</p>
<p>Notothenioids lack swim bladders, but the majority of species show increased buoyancy that appears to facilitate benthopelagic and benthic lifestyles (Eastman <a href="#ece371847-bib-0034" class="usa-link" aria-describedby="ece371847-bib-0034">1985</a>, <a href="#ece371847-bib-0037" class="usa-link" aria-describedby="ece371847-bib-0037">2017</a>, <a href="#ece371847-bib-0038" class="usa-link" aria-describedby="ece371847-bib-0038">2024</a>; Eastman and Sidell <a href="#ece371847-bib-0041" class="usa-link" aria-describedby="ece371847-bib-0041">2002</a>). In a few notothenioid species, a combination of increased lipid deposition and reduced skeletal development allows further increased buoyancy up to the point of neutrality (Chen et al. <a href="#ece371847-bib-0022" class="usa-link" aria-describedby="ece371847-bib-0022">2019</a>; Eastman <a href="#ece371847-bib-0036" class="usa-link" aria-describedby="ece371847-bib-0036">2005</a>, <a href="#ece371847-bib-0038" class="usa-link" aria-describedby="ece371847-bib-0038">2024</a>). The resulting change in lifestyle, sometimes referred to as pelagization, is shown most strongly by the subfamilies Pleuragramminae and Dissostichinae (family Nototheniidae, classification following Duhamel et al. <a href="#ece371847-bib-0032" class="usa-link" aria-describedby="ece371847-bib-0032">2014</a>). The two subfamilies are sister lineages and diverged early within the clade of Antarctic notothenioids (Near et al. <a href="#ece371847-bib-0089" class="usa-link" aria-describedby="ece371847-bib-0089">2018</a>). The subfamily Pleuragramminae includes the Antarctic silverfish (
<em>Pleuragramma antarcticum</em>
) as its sole extant representative, and Dissostichinae includes the Antarctic and Patagonian toothfishes (
<em>Dissostichus mawsoni</em>
 and 
<em>D. eleginoides</em>
) and the longfin icedevil (
<em>Aethotaxis mitopteryx</em>
)—all of which show adaptations towards neutral buoyancy (Eastman and DeVries <a href="#ece371847-bib-0039" class="usa-link" aria-describedby="ece371847-bib-0039">1989</a> for 
<em>P. antarcticum</em>
; Near et al. <a href="#ece371847-bib-0090" class="usa-link" aria-describedby="ece371847-bib-0090">2003</a> for 
<em>D. mawsoni</em>
; Eastman <a href="#ece371847-bib-0035" class="usa-link" aria-describedby="ece371847-bib-0035">1993</a> for 
<em>D. eleginoides</em>
; Near et al. <a href="#ece371847-bib-0088" class="usa-link" aria-describedby="ece371847-bib-0088">2007</a> for 
<em>A. mitopteryx</em>
). Two other species assigned to Dissostichinae, 
<em>Gvozdarus svetovidovi</em>
 and 
<em>G. balushkini</em>
 (Duhamel et al. <a href="#ece371847-bib-0032" class="usa-link" aria-describedby="ece371847-bib-0032">2014</a>), are presumed to be pelagic, but they are extremely rare, and our knowledge of their biology is based on no more than three specimens (Eastman and Voskoboinikova <a href="#ece371847-bib-0042" class="usa-link" aria-describedby="ece371847-bib-0042">2024</a>).</p>
<p>Many species of the Nototheniidae family are characterized by long pelagic larval phases (Kellermann <a href="#ece371847-bib-0062" class="usa-link" aria-describedby="ece371847-bib-0062">1990</a>). These extended pelagic stages are exposed to the features of physical circulation of the Southern Ocean that connect also very distant sites. The most prominent feature of the Southern Ocean circulation is the Antarctic Circumpolar Current (ACC), which flows clockwise around the Antarctic continent. In proximity to the Western Antarctic Peninsula, the ACC moves over the slope and continues eastward and seaward, offshore of the South Shetland Islands (Ryan et al. <a href="#ece371847-bib-0107" class="usa-link" aria-describedby="ece371847-bib-0107">2016</a>). In contrast to the ACC, the Antarctic Slope Current (ASC) and the Antarctic Coastal Current (AACC) flow westward and closer to the coast (Graham et al. <a href="#ece371847-bib-0048" class="usa-link" aria-describedby="ece371847-bib-0048">2013</a>; Thompson et al. <a href="#ece371847-bib-0116" class="usa-link" aria-describedby="ece371847-bib-0116">2018</a>). Flowing in opposite directions, the ACC and the ASC contribute to the northern and southern boundaries, respectively, of the Weddell Gyre, a mainly wind‐driven, cyclonic ocean gyre (Ryan et al. <a href="#ece371847-bib-0107" class="usa-link" aria-describedby="ece371847-bib-0107">2016</a>).</p>
<p>Since species so far examined have pelagic early life stages that last over several months and often more than a year (Kock <a href="#ece371847-bib-0067" class="usa-link" aria-describedby="ece371847-bib-0067">2005</a> and La Mesa and Ashford <a href="#ece371847-bib-0070" class="usa-link" aria-describedby="ece371847-bib-0070">2007</a>, <a href="#ece371847-bib-0071" class="usa-link" aria-describedby="ece371847-bib-0071">2008</a> for the Channichthyinae subfamily and Loeb et al. <a href="#ece371847-bib-0080" class="usa-link" aria-describedby="ece371847-bib-0080">1993</a> for Nototheniidae in general), notothenioids have traditionally been assumed to have high dispersal potential driven by the main circulation features, like the ACC (see Matschiner et al. <a href="#ece371847-bib-0084" class="usa-link" aria-describedby="ece371847-bib-0084">2015</a>). Dispersal potential over long distances is often associated with expectations of high gene flow and lack of population structure (Hauser and Carvalho <a href="#ece371847-bib-0053" class="usa-link" aria-describedby="ece371847-bib-0053">2008</a>). This conclusion has been confirmed in studies on some benthic species (Damerau et al. <a href="#ece371847-bib-0026" class="usa-link" aria-describedby="ece371847-bib-0026">2012</a>; Jones et al. <a href="#ece371847-bib-0058" class="usa-link" aria-describedby="ece371847-bib-0058">2008</a>; Matschiner et al. <a href="#ece371847-bib-0085" class="usa-link" aria-describedby="ece371847-bib-0085">2009</a>; Papetti et al. <a href="#ece371847-bib-0095" class="usa-link" aria-describedby="ece371847-bib-0095">2012</a>) but for some species, different levels of population structure have been described (Bernal‐Durán et al. <a href="#ece371847-bib-0009" class="usa-link" aria-describedby="ece371847-bib-0009">2024</a>; Damerau et al. <a href="#ece371847-bib-0027" class="usa-link" aria-describedby="ece371847-bib-0027">2014</a>; Papetti et al. <a href="#ece371847-bib-0096" class="usa-link" aria-describedby="ece371847-bib-0096">2009</a>; Schiavon et al. <a href="#ece371847-bib-0111" class="usa-link" aria-describedby="ece371847-bib-0111">2023</a>; Van de Putte et al. <a href="#ece371847-bib-0120" class="usa-link" aria-describedby="ece371847-bib-0120">2012</a>). Among pelagic species, two have been particularly well studied: <em>D. mawsoni</em>, because of its high commercial value (Hanchet et al. <a href="#ece371847-bib-0052" class="usa-link" aria-describedby="ece371847-bib-0052">2015</a>), and 
<em>P. antarcticum</em>
, because of its high abundance and its key role in the Antarctic food web (La Mesa and Eastman <a href="#ece371847-bib-0073" class="usa-link" aria-describedby="ece371847-bib-0073">2012</a>; Vacchi et al. <a href="#ece371847-bib-0119" class="usa-link" aria-describedby="ece371847-bib-0119">2017</a>). For 
<em>D. mawsoni</em>
, no evidence of genetic structure was found at the circum‐Antarctic level (Ceballos et al. <a href="#ece371847-bib-0021" class="usa-link" aria-describedby="ece371847-bib-0021">2021</a>; Maschette et al. <a href="#ece371847-bib-0083" class="usa-link" aria-describedby="ece371847-bib-0083">2023</a>). However, the pelagic and circumpolar 
<em>P. antarcticum</em>
 showed strong gene flow on large spatial scales but reductions in gene flow linked to discontinuities in the major current systems flowing along the continental shelf and slope (Caccavo et al. <a href="#ece371847-bib-0017" class="usa-link" aria-describedby="ece371847-bib-0017">2018</a>).</p>
<p>Given these premises, we therefore hypothesized that 
<em>A. mitopteryx</em>
 also does not show signs of population structure. In fact, its ability for active swimming and its pelagic lifestyle, which expose it to current bidirectional transport with ACC, ASC, and AACC, especially during the larval phase, likely ensures a high dispersal capacity over large distances and thus high connectivity between areas. The absence of evidence for geographical population structure, however, would not exclude other types of population genetic structures based on age‐cohort division, sex (Benestan et al. <a href="#ece371847-bib-0007" class="usa-link" aria-describedby="ece371847-bib-0007">2017</a>), sex‐specific dispersal (Sandoval Laurrabaquio‐A et al. <a href="#ece371847-bib-0108" class="usa-link" aria-describedby="ece371847-bib-0108">2019</a>), mating types (Sonsthagen et al. <a href="#ece371847-bib-0113" class="usa-link" aria-describedby="ece371847-bib-0113">2012</a>) or local adaptation (Benestan et al. <a href="#ece371847-bib-0008" class="usa-link" aria-describedby="ece371847-bib-0008">2016</a>).</p>
<p>So far, though, 
<em>A. mitopteryx</em>
 connectivity patterns are unknown because of the species' elusive nature and the limited number of specimens sampled and studied to date. Indeed, 
<em>A. mitopteryx</em>
 is one of the least studied species among notothenioids. Although its presence has been recorded all around the Antarctic continent (DeWitt et al. <a href="#ece371847-bib-0031" class="usa-link" aria-describedby="ece371847-bib-0031">1990</a>; Zimmermann <a href="#ece371847-bib-0127" class="usa-link" aria-describedby="ece371847-bib-0127">1997</a>), its distribution might be patchy because catches have been scarce and irregular or because the fishing gears used are not specific for this pelagic fish. Kunzmann and Zimmermann (<a href="#ece371847-bib-0069" class="usa-link" aria-describedby="ece371847-bib-0069">1992</a>) reported that the four expeditions of the German research vessel (R/V) <em>Polarstern</em> carried out between 1985 and 1992 in the Weddell and Lazarev Seas yielded a total of 99 
<em>A. mitopteryx</em>
, representing 90% of the total catches of this species reported until then. During two more recent <em>Polarstern</em> expeditions carried out over a large area of the eastern and south‐eastern Weddell Sea in 2013–2014 (PS82) and 2015–2016 (PS96), a few specimens of 
<em>A. mitopteryx</em>
 were caught at two stations north of Cape Vestkapp (in Queen Maud Land) out of a total of 56 stations, confirming the rarity of observations of this species. During these two expeditions, the species was found along the continental slope at depths greater than 700 m (La Mesa et al. <a href="#ece371847-bib-0072" class="usa-link" aria-describedby="ece371847-bib-0072">2018</a>) near the average position of the Antarctic Slope Current (ASC; Thompson et al. <a href="#ece371847-bib-0116" class="usa-link" aria-describedby="ece371847-bib-0116">2018</a>), implying exposure to the along‐shelf circulation. Similarly, to the best of our knowledge, only ten specimens were caught on the Antarctic Peninsula at depths greater than 600 m during the <em>Polarstern</em> expedition in 2018 (PS112).</p>
<p>Despite being rarely captured, 
<em>A. mitopteryx</em>
 is considered to have a higher reproductive potential than most notothenioids because it is assumed to be the longest‐lived notothenioid species (up to 36 years for males and 62 years for females) and because it reproduces for the first time at 30%–50% of its maximum age (around 11 and 32 years for males and females, respectively; La Mesa et al. <a href="#ece371847-bib-0072" class="usa-link" aria-describedby="ece371847-bib-0072">2018</a>). In contrast, all other notothenioids for which we have estimates reach sexual maturity at 50%–80% of their life span (La Mesa and Vacchi <a href="#ece371847-bib-0075" class="usa-link" aria-describedby="ece371847-bib-0075">2001</a>). Unlike many notothenioid species, 
<em>A. mitopteryx</em>
 also has a pronounced sexual dimorphism in size and life expectancy, with females being larger and longer‐lived than males (La Mesa et al. <a href="#ece371847-bib-0072" class="usa-link" aria-describedby="ece371847-bib-0072">2018</a>). Despite the limited amount of samples, the striking sexual dimorphism in size and life span found in 
<em>A. mitopteryx</em>
 can be considered quite reliable, being based on female and male samples of similar size as reported in La Mesa et al. (<a href="#ece371847-bib-0072" class="usa-link" aria-describedby="ece371847-bib-0072">2018</a>). Moreover, a few other notothenioid species also show sexual dimorphism in size and life expectancy (La Mesa et al. <a href="#ece371847-bib-0074" class="usa-link" aria-describedby="ece371847-bib-0074">2021</a>); however, in those cases, the dimorphism is usually associated with differences in body colouration and fin morphology (Kock and Kellermann <a href="#ece371847-bib-0068" class="usa-link" aria-describedby="ece371847-bib-0068">1991</a>). However, we cannot exclude that males and females have different behaviors or occupy different areas or depths and that these are therefore caught with different frequencies, thus biasing the results. Unfortunately, due to the complicated logistics of sampling in the Southern Ocean and of coordinating among and obtaining samples from different institutes and countries doing research in Antarctica, we cannot test this hypothesis yet.</p>
<p>Sexual dimorphism in fish, where males and females differ in size or life span, can significantly impact recruitment and demography by influencing factors like mating success, reproductive output, and growth rates. Indeed, in most gonochoristic species, size is a key factor in mate choice and sexual competition, favoring larger males in the case of sperm competition and larger females that are able to produce more eggs per season. Sexual dimorphism in growth rates directly influences longevity, which in turn determines the timing of sexual maturity and recruitment rates, having the potential to alter the population demographic dynamics. Size differences between sexes can also play a role in how fish populations interact within their environment, leading to different resource utilization and, ultimately, to food niche and spatial segregation, which might negatively affect mating success and the number of recruits determining the strength of connectivity (Prchalová et al. <a href="#ece371847-bib-0099" class="usa-link" aria-describedby="ece371847-bib-0099">2022</a>; Ritchie et al. <a href="#ece371847-bib-0105" class="usa-link" aria-describedby="ece371847-bib-0105">2007</a>).</p>
<p>These observations on life history traits suggest that for 
<em>A. mitopteryx</em>
, the impact of differential recruitment and demographic fluctuations over time could play a possibly stronger role in shaping connectivity than in other Pleuragramminae and Dissostichinae. Hence, an assessment of 
<em>A. mitopteryx</em>
 population genetic structure requires the integration of biological and physical mechanisms to explain connectivity, including aspects of sex differentiation.</p>
<p>Therefore, in this study, we conducted the first assessment of population structure in 
<em>A. mitopteryx</em>
 between the two opposite sides of the Weddell Sea—the East Antarctic Peninsula and the East Weddell Sea (separated by ~1900 km in a straight line). These two locations are of great ecological relevance. On one hand, the Antarctic Peninsula is experiencing one of the fastest warming rates of the planet. On the other hand, the Weddell Sea was considered one of the last regions of the Southern Ocean to experience the consequences of climate change due to its extensive and persistent sea‐ice cover and ocean currents pattern (Teschke et al. <a href="#ece371847-bib-0115" class="usa-link" aria-describedby="ece371847-bib-0115">2021</a>). However, recent studies have reported that the Weddell Sea has also seen dramatic changes, particularly in sea ice cover since 2016 (Raphael et al. <a href="#ece371847-bib-0102" class="usa-link" aria-describedby="ece371847-bib-0102">2025</a>; Turner et al. <a href="#ece371847-bib-0118" class="usa-link" aria-describedby="ece371847-bib-0118">2020</a>). It has also experienced significant long‐term warming, particularly in deep waters (Strass et al. <a href="#ece371847-bib-0114" class="usa-link" aria-describedby="ece371847-bib-0114">2020</a>).</p>
<p>Moreover, since our analysis gave us unexpected access to genetic data differentiating sexes, in this study, we were able for the first time to investigate the genetic bases of sex differentiation in 
<em>A. mitopteryx</em>
.</p>
<p>Given the difficulty of gathering samples from the Southern Ocean and especially for 
<em>A. mitopteryx</em>
, which limits the sample size, this study nonetheless represents a unique opportunity to provide a first assessment of connectivity and genetic sex differentiation in this little‐studied notothenioid, one of the few pelagic species and the longest‐lived notothenioid known so far.</p></section><section id="ece371847-sec-0002"><h2 class="pmc_sec_title">2. Materials and Methods</h2>
<section id="ece371847-sec-0003"><h3 class="pmc_sec_title">2.1. Samples, RAD‐Seq Library and DNA Sequencing</h3>
<p>We analyzed 10 
<em>A. mitopteryx</em>
 specimens (4 males and 6 females) collected from the East Weddell Sea (EWS) during PS96 in January 2016 and eight specimens (2 males and 6 females) collected from the Antarctic Peninsula sector of the Weddell Sea during PS112 in April 2018 (hereafter East Antarctic Peninsula, EAP, Figure <a href="#ece371847-fig-0001" class="usa-link">1</a>). The two sampling sites are separated by approximately 1900 km in a straight line. All samples were collected by bottom trawl, at depths of 878 m in EWS and at 641 m in EAP. The sex of each specimen was first identified by visual inspection of the gonads at dissection and later confirmed by gonad histology for some specimens as previously described for other notothenioid species (e.g., Riginella et al. <a href="#ece371847-bib-0104" class="usa-link" aria-describedby="ece371847-bib-0104">2021</a>). DNA was extracted and restriction site associated DNA sequencing (RAD‐seq; Baird et al. <a href="#ece371847-bib-0006" class="usa-link" aria-describedby="ece371847-bib-0006">2008</a>) libraries were prepared following Schiavon et al. (<a href="#ece371847-bib-0109" class="usa-link" aria-describedby="ece371847-bib-0109">2024</a>). Briefly, DNA was digested with a single restriction enzyme –<em>SbfI</em>– and the first indexed adapter (P1) was ligated to the cut end. All samples were then pooled (multiplexed) and sheared using Covaris. Fragments of the target length (between 400 and 600 bp) were isolated using a size selection step by gel electrophoresis. A second adapter (P2) was ligated to the free ends. Fragments ligated to both adapters, P1 and P2, were selectively enriched by PCR amplification before sequencing. Paired‐end sequencing with a read length of 150 bp was conducted on the Illumina HiSeq4000 platform at the Genomics &amp; Cell Characterization Core Facility (GC3F, University of Oregon, USA).</p>
<figure class="fig xbox font-sm" id="ece371847-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0594/12367274/22a4a20efd63/ECE3-15-e71847-g003.jpg" loading="lazy" id="jats-graphic-3" height="514" width="664" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/ece371847-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Map of the sampling sites of the two populations of 
<em>Aethotaxis mitopteryx</em>
 analyzed in this study. Coordinates of the two sampling sites are 63°42′02.7″S 57°02′01.0″W for EAP and 72°17′47.4″S 16°50′49.8″W for EWS. Map drawn with R package ggOceanMaps (Vihtakari <a href="#ece371847-bib-0122" class="usa-link" aria-describedby="ece371847-bib-0122">2024</a>) and manually modified.</p></figcaption></figure></section><section id="ece371847-sec-0004"><h3 class="pmc_sec_title">2.2. Whole Genome Sequencing and de Novo Assembly</h3>
<p>One 
<em>A. mitopteryx</em>
 female specimen (sampled in the East Weddell Sea) was selected to generate a <em>de novo</em> draft genome assembly following a dual‐sequencing strategy that combined short‐ and long‐read sequencing. Although the goal was to produce a hybrid assembly leveraging both data types, the process was ultimately shaped by limitations in long‐read sequencing coverage.</p>
<p>After DNA extraction (see above), we initially generated 250 bp paired‐end (PE) reads on an Illumina NovaSeq 6000 platform (KAPA Hyper prep PCA free library). Library preparation and sequencing were performed at the D‐BSSE Genomics Facility Basel of ETH Zurich (<a href="https://bsse.ethz.ch/genomicsbasel" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bsse.ethz.ch/genomicsbasel</a>), yielding approximately 100‐fold sequencing depth. Remaining genomic DNA from multiple extracts was pooled and concentrated in a Speed‐Vac centrifuge to a total volume of 60 μL, size‐selected using a Blue Pippin electrophoresis platform (selecting molecules longer than 10 kb on a high pass plus cassette). Library preparation was performed using the SQK‐RAD004 kit followed by sequencing on an Oxford Nanopore Technologies MinION Mk1C device using an R9.4 flow cell. Unfortunately, the long‐read sequencing run resulted in only ~4‐fold genome coverage. Given this constraint, we tested multiple hybrid assembly strategies to make the best use of both data types. The most complete and contiguous draft genome assembly—judged by metrics such as N50, total assembly size, number of contigs, and BUSCO completeness—was obtained with the following assembly pipeline. First, Illumina short reads were trimmed for adapter sequences and low‐quality bases using <span class="font-variant-small-caps">trimmomatic</span> ver. 0.39 (Bolger et al. <a href="#ece371847-bib-0012" class="usa-link" aria-describedby="ece371847-bib-0012">2014</a>). Next, overlapping paired‐end reads were merged into longer reads using <span class="font-variant-small-caps">flash2</span> ver. <span class="font-variant-small-caps">2.2.0</span> (Magoč and Salzberg <a href="#ece371847-bib-0081" class="usa-link" aria-describedby="ece371847-bib-0081">2011</a>). The non‐overlapping paired‐end reads (~50%) were then assembled <em>de novo</em> using <span class="font-variant-small-caps">abyss</span> ver. <span class="font-variant-small-caps">2.3.1</span> (Jackman et al. <a href="#ece371847-bib-0055" class="usa-link" aria-describedby="ece371847-bib-0055">2017</a>) with k‐mer size (k) set to 95 and minimum coverage cutoff (kc) set to 4. The resulting contigs were scaffolded with <span class="font-variant-small-caps">links</span> ver. <span class="font-variant-small-caps">1.8.6</span> (Warren et al. <a href="#ece371847-bib-0123" class="usa-link" aria-describedby="ece371847-bib-0123">2015</a>), followed by patching with low‐coverage MinION long reads, and polished with merged Illumina reads, all using <span class="font-variant-small-caps">racon</span> ver. 1.4.7 (Vaser et al. <a href="#ece371847-bib-0121" class="usa-link" aria-describedby="ece371847-bib-0121">2017</a>). Continuity and completeness of the assembly were assessed using <span class="font-variant-small-caps">quast</span> ver. <span class="font-variant-small-caps">5.0.2</span> (Gurevich et al. <a href="#ece371847-bib-0050" class="usa-link" aria-describedby="ece371847-bib-0050">2013</a>) and <span class="font-variant-small-caps">busco</span> ver. <span class="font-variant-small-caps">5.1.2</span> (Simão et al. <a href="#ece371847-bib-0112" class="usa-link" aria-describedby="ece371847-bib-0112">2015</a>), respectively.</p>
<p>Given that the generated 
<em>A. mitopteryx</em>
 genome assembly remained fragmented, all scaffolds were ordered and oriented into larger scaffolds with the software <span class="font-variant-small-caps">ragtag</span> ver. <span class="font-variant-small-caps">2.1.0</span> (Alonge et al. <a href="#ece371847-bib-0004" class="usa-link" aria-describedby="ece371847-bib-0004">2022</a>) and the option <em>scaffold</em> and the aligner <span class="font-variant-small-caps">unimap</span> (<a href="https://github.com/lh3/unimap" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/lh3/unimap</a>) using as a guide the closely related species 
<em>D. mawsoni</em>
 genome assembly (Lee et al. <a href="#ece371847-bib-0077" class="usa-link" aria-describedby="ece371847-bib-0077">2021</a>; NCBI accession number: GCA_011823955.1). This step does not alter the original sequence of the 
<em>A. mitopteryx</em>
 genome assembly, but allows visualization of genetic variability at a larger scale. Hereafter, we will thus refer to 
<em>A. mitopteryx</em>
 scaffolds following the scaffold numeration in the 
<em>D. mawsoni</em>
 genome assembly.</p>
<p>Before submission to NCBI, the final <em>de novo</em> not‐scaffolded assembly was decontaminated using the <span class="font-variant-small-caps">ncbi</span>
<span class="font-variant-small-caps">foreign contamination screen</span>
<span class="font-variant-small-caps">(fcs)</span> tool ver. <span class="font-variant-small-caps">0.4.0</span> (Astashyn et al. <a href="#ece371847-bib-0005" class="usa-link" aria-describedby="ece371847-bib-0005">2024</a>).</p></section><section id="ece371847-sec-0005"><h3 class="pmc_sec_title">2.3. Bioinformatic Processing of RAD‐Seq Raw Reads, SNP Calling and Filtering</h3>
<p>Raw RAD‐seq data were demultiplexed using the <em>process_radtags</em> module of <span class="font-variant-small-caps">stacks</span> ver. <span class="font-variant-small-caps">2.53</span> (Catchen et al. <a href="#ece371847-bib-0018" class="usa-link" aria-describedby="ece371847-bib-0018">2011</a>; Rochette et al. <a href="#ece371847-bib-0106" class="usa-link" aria-describedby="ece371847-bib-0106">2019</a>) and PCR duplicates were removed with the <span class="font-variant-small-caps">stacks</span> tool <em>clone_filter</em>.</p>
<p>Illumina reads were then mapped to the newly generated and scaffolded 
<em>A. mitopteryx</em>
 genome assembly (see above) using <span class="font-variant-small-caps">bwa</span> ver. <span class="font-variant-small-caps">0.7.17</span> with default parameters (Li and Durbin <a href="#ece371847-bib-0079" class="usa-link" aria-describedby="ece371847-bib-0079">2009</a>). SNP calling was performed with the <span class="font-variant-small-caps">stacks</span> tools <em>gstacks</em> and <em>populations</em> with parameter –r 0.75 (minimum percentage of individuals required to process a locus) and –max‐obs‐het 0.75 (maximum observed heterozygosity allowed to process a SNP). The resulting VCF file was filtered with <span class="font-variant-small-caps">vcf</span>
<span class="font-variant-small-caps">tools</span> ver. <span class="font-variant-small-caps">0.1.17</span> (Danecek et al. <a href="#ece371847-bib-0028" class="usa-link" aria-describedby="ece371847-bib-0028">2011</a>), retaining only genotypes with a minimum Phred‐scaled genotype quality of 30 and a sequencing depth between 5 and 40 (individuals failing these thresholds were marked as having a missing genotype). Only sites with mean sequencing depth between 10 and 40 were retained (sites failing these thresholds were removed from the data set). Singletons were removed. The R package SNPfiltR (DeRaad <a href="#ece371847-bib-0029" class="usa-link" aria-describedby="ece371847-bib-0029">2022</a>; Knaus and Grünwald <a href="#ece371847-bib-0065" class="usa-link" aria-describedby="ece371847-bib-0065">2017</a>) was used to convert heterozygous genotypes to missing data if they had an allele balance (i.e., ratio of reads showing the reference allele among all reads) outside the range 0.25–0.75, since they could derive from misaligned paralogs. SNPs with a missing data proportion above 0.15 were removed, and only SNPs at least 1000 bp apart were retained to obtain a thinned panel of unlinked markers.</p></section><section id="ece371847-sec-0006"><h3 class="pmc_sec_title">2.4. Population Structure</h3>
<p>The presence of population structure was first assessed using both a Principal Component Analysis (PCA) with the R package ‘adegenet’ ver. 2.1.10 (Jombart <a href="#ece371847-bib-0057" class="usa-link" aria-describedby="ece371847-bib-0057">2008</a>) and unsupervised hierarchical clustering with <span class="font-variant-small-caps">admixture</span> ver. 1.3.0 (Alexander et al. <a href="#ece371847-bib-0003" class="usa-link" aria-describedby="ece371847-bib-0003">2009</a>). <span class="font-variant-small-caps">admixture</span> was run to estimate individual ancestry proportions (Q) with cross‐validation, assuming a value of K from 1 to 5 for the estimation of the optimal number of clusters. To quantify differences between groups, the pairwise fixation index (<em>F</em>
<sub>ST</sub>) as defined by Weir and Cockerham (<a href="#ece371847-bib-0124" class="usa-link" aria-describedby="ece371847-bib-0124">1984</a>) between sampling sites and sexes was calculated with the R package ‘StAMPP’ ver. 1.6.3 (Pembleton et al. <a href="#ece371847-bib-0097" class="usa-link" aria-describedby="ece371847-bib-0097">2013</a>).</p>
<p>Haplotype‐based approaches can infer population structure at higher resolution and give a stronger signal of differentiation compared to SNP‐based methods because they exploit information from the full RAD haplotypes, which are generally more variable than individual sites (Bootsma et al. <a href="#ece371847-bib-0013" class="usa-link" aria-describedby="ece371847-bib-0013">2021</a>; Leitwein et al. <a href="#ece371847-bib-0078" class="usa-link" aria-describedby="ece371847-bib-0078">2020</a>). To implement this approach, we first discarded RAD loci with 10 or more SNPs from the <span class="font-variant-small-caps">stacks</span>
<em>populations</em> module output because such high SNP densities are likely to result from misaligned paralogous loci (<a href="https://www.milan-malinsky.org/fineradstructure" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.milan‐malinsky.org/fineradstructure</a>). Then, we ran <em>populations</em> with this subsample of loci to produce an input file for <span class="font-variant-small-caps">fineradstructure</span> (Malinsky et al. <a href="#ece371847-bib-0082" class="usa-link" aria-describedby="ece371847-bib-0082">2018</a>) and to calculate the divergence statistics φ<sub>ST</sub> (a locus‐level version of <em>F</em>
<sub>ST</sub>; Excoffier et al. <a href="#ece371847-bib-0043" class="usa-link" aria-describedby="ece371847-bib-0043">1992</a>) and <em>D</em>
<sub>XY</sub> (absolute divergence; Nei <a href="#ece371847-bib-0091" class="usa-link" aria-describedby="ece371847-bib-0091">1987</a>). <span class="font-variant-small-caps">fineradstructure</span> computes a matrix of haplotype relationships between samples (the “co‐ancestry matrix”). φ<sub>ST</sub> and <em>D</em>
<sub>XY</sub> were calculated comparing geographic regions (EAP vs. EWS) and between sexes. The computation was made per single locus and across a sliding window (specifying the—smooth option in <em>populations</em>, which implies a window size of 900 bp and the average weighted across the window according to a Gaussian distribution centered on the middle of the window).</p></section><section id="ece371847-sec-0007"><h3 class="pmc_sec_title">2.5. Sex Differentiation</h3>
<p>Since our analysis of population structure revealed a clear genetic differentiation between males and females (see Results), we conducted further analyses to investigate this aspect in 
<em>A. mitopteryx</em>
.</p>
<p>Levels of homozygosity were calculated for all samples across the 
<em>A. mitopteryx</em>
 genome assembly to identify sex‐linked regions and the heterogametic sex. Homozygosity in sex‐linked regions in the heterogametic sex is expected to be much lower than that in sex‐linked regions in the homogametic sex and in non‐sex‐linked regions in both sexes. Individual levels of homozygosity per single scaffold were computed with <span class="font-variant-small-caps">vcftools</span>.</p>
<p>If the sex chromosomes in the heterogametic sex are strongly differentiated or if the sex‐linked region is present in only the heterogametic sex, sequencing reads should differentially align in males and females, leading to an imbalance in sequencing depth along sex‐linked regions. To test for such a signal in 
<em>A. mitopteryx</em>
, we followed the procedure described in Feller et al. (<a href="#ece371847-bib-0044" class="usa-link" aria-describedby="ece371847-bib-0044">2021</a>) and quantified sequencing depth per SNP separately for males and females with <span class="font-variant-small-caps">vcftools</span> to then calculate female/male depth ratios. The ratios were corrected for a mean genome‐wide ratio to be centered at 1. Results were then averaged across a sliding window of 5 Mb with the function <em>winScan</em> in the R package ‘windowscanR’ (<a href="https://github.com/tavareshugo/WindowScanR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/tavareshugo/WindowScanR</a>).</p>
<p>Differences in the pattern of linkage disequilibrium (LD) could also indicate that recombination has been reduced or prevented between the differentiated sex chromosomes (McKinney et al. <a href="#ece371847-bib-0086" class="usa-link" aria-describedby="ece371847-bib-0086">2020</a>) or, not exclusively, that strong selection has acted on sex‐linked genomic regions (Koch et al. <a href="#ece371847-bib-0066" class="usa-link" aria-describedby="ece371847-bib-0066">2022</a>). LD was calculated with PLINK ver. 1.90 (Purcell et al. <a href="#ece371847-bib-0101" class="usa-link" aria-describedby="ece371847-bib-0101">2007</a>). Results were averaged across genomic windows of 10 kb with ‘windowscanR’.</p>
<p>To identify sex‐linked markers, we used <span class="font-variant-small-caps">radsex</span> ver. 1.2.0 (Feron et al. <a href="#ece371847-bib-0045" class="usa-link" aria-describedby="ece371847-bib-0045">2021</a>). <span class="font-variant-small-caps">radsex</span> analyzes raw sequencing reads rather than individual SNPs and identifies the presence or absence of each unique read (i.e., marker) in each individual. <span class="font-variant-small-caps">radsex</span> then returns a list of markers that have a statistically different distribution between sexes. Following the software guidelines, we applied the command <em>process</em> to generate a table of marker depths for the whole dataset as well as the distribution of markers between males and females, using four different thresholds for minimum read depth in each specimen (1, 2, 5, 10). Then, markers significantly associated with sex were identified with the command <em>signif</em>. All markers were further aligned to the 
<em>A. mitopteryx</em>
 genome assembly to visualize their distribution and location along different scaffolds with the command <em>map</em>. Results were plotted with the R package ‘sgtr’ ver. 1.1.2 (Feron et al. <a href="#ece371847-bib-0045" class="usa-link" aria-describedby="ece371847-bib-0045">2021</a>).</p>
<p>The scaffolds in the 
<em>D. mawsoni</em>
 genome assembly used to increase the contiguity of the 
<em>A. mitopteryx</em>
 assembly were not numbered following the chromosome naming convention established for notothenioid species (Bista et al. <a href="#ece371847-bib-0010" class="usa-link" aria-describedby="ece371847-bib-0010">2020</a>, <a href="#ece371847-bib-0011" class="usa-link" aria-describedby="ece371847-bib-0011">2023</a>; Kim et al. <a href="#ece371847-bib-0063" class="usa-link" aria-describedby="ece371847-bib-0063">2019</a>), which follows the chromosome numbering in medaka, a species representative of the ancestral percomorph karyotype of 24 chromosomes (Kim et al. <a href="#ece371847-bib-0063" class="usa-link" aria-describedby="ece371847-bib-0063">2019</a>). Thus, to facilitate comparison of our results with other species, we used the <span class="font-variant-small-caps">d‐</span>
<span class="font-variant-small-caps">genies</span> web application (Cabanettes and Klopp <a href="#ece371847-bib-0016" class="usa-link" aria-describedby="ece371847-bib-0016">2018</a>) to align the original 
<em>D. mawsoni</em>
 genome assembly to the genome assembly of the more distantly related non‐Antarctic notothenioid <em>Cottoperca trigloides</em> (formerly known as 
<em>Cottoperca gobio</em>
, NCBI accession number: GCF_900634415.1), in which chromosomes were numbered based on their orthology with medaka (Bista et al. <a href="#ece371847-bib-0010" class="usa-link" aria-describedby="ece371847-bib-0010">2020</a>).</p>
<p>For comparative purposes, we repeated the same <span class="font-variant-small-caps">radsex</span> procedure using 
<em>D. mawsoni</em>
 RAD‐seq data from Ceballos et al. (<a href="#ece371847-bib-0021" class="usa-link" aria-describedby="ece371847-bib-0021">2021</a>). We reanalyzed 12 males and 9 females for which the sex was confirmed by histological examination.</p></section><section id="ece371847-sec-0008"><h3 class="pmc_sec_title">2.6. Search for Candidate Sex Determining Genes</h3>
<p>Considering that many different sex determining genes have independently evolved in various fish lineages but often involved members of the conserved vertebrate sex‐determination and differentiation pathway, especially of the TGF‐β family (Adolfi, Herpin, and Schartl <a href="#ece371847-bib-0002" class="usa-link" aria-describedby="ece371847-bib-0002">2021</a>; Kitano et al. <a href="#ece371847-bib-0064" class="usa-link" aria-describedby="ece371847-bib-0064">2024</a>; Pan, Feron, et al. <a href="#ece371847-bib-0093" class="usa-link" aria-describedby="ece371847-bib-0093">2021</a>), we searched for the presence and location of 35 putative teleost sex‐determining gene candidates (listed in Table <a href="#ece371847-supitem-0001" class="usa-link">S1</a>) in the 
<em>A. mitopteryx</em>
 genome assembly using BLASTN.</p></section></section><section id="ece371847-sec-0009"><h2 class="pmc_sec_title">3. Results</h2>
<section id="ece371847-sec-0010"><h3 class="pmc_sec_title">3.1. De Novo Genome Assembly</h3>
<p>Our newly sequenced and <em>de novo</em> assembled 
<em>A. mitopteryx</em>
 genome was fragmented in 210,180 contigs, with a total length of 596 Mb and an N50 value of 9740 bp. The assembly contained 61.3% of the Actinopterygii BUSCO genes as complete sequences, while an additional 12.4% of the genes were present in fragmented form and 26.3% were missing from the assembly. More detailed statistics on genome continuity and completeness at different assembly stages are reported in Table <a href="#ece371847-supitem-0001" class="usa-link">S2</a>. The scaffolding of the 
<em>A. mitopteryx</em>
 assembly on the 
<em>D. mawsoni</em>
 reference genome combined 10.6% of the contigs (corresponding to 51.6% of the total length) into 310 scaffolds. The unplaced contigs were joined into a single pseudoscaffold. Additional statistics related to assembly and scaffolding are reported in Tables <a href="#ece371847-supitem-0001" class="usa-link">S3</a> and <a href="#ece371847-supitem-0001" class="usa-link">S4</a>.</p></section><section id="ece371847-sec-0011"><h3 class="pmc_sec_title">3.2. Population Structure</h3>
<p>The filtered VCF file included 10,335 SNPs. No geographic genetic structure was observed in the PCA plot, while males and females were separated along the first axis that explains 7.22% of the variance (Figure <a href="#ece371847-fig-0002" class="usa-link">2</a>). Cross‐validation errors of <span class="font-variant-small-caps">admixture</span> results indicated <em>K</em> = 1 as the optimal number of genetic clusters (Figure <a href="#ece371847-supitem-0001" class="usa-link">S1</a>) and when plotting <em>K</em> = 2, individuals were not divided by sampling location or sex (Figure <a href="#ece371847-supitem-0001" class="usa-link">S2</a>, top panel). Weir and Cockerham's (<a href="#ece371847-bib-0124" class="usa-link" aria-describedby="ece371847-bib-0124">1984</a>) <em>F</em>
<sub>ST</sub> between sampling sites was 0.00014 and not significant (<em>p</em>‐value = 0.431) while the <em>F</em>
<sub>ST</sub> between sexes was 0.02258 and significant (<em>p</em>‐value = 0.001).</p>
<figure class="fig xbox font-sm" id="ece371847-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0594/12367274/f26f54d11a51/ECE3-15-e71847-g008.jpg" loading="lazy" id="jats-graphic-5" height="465" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/ece371847-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Scatter plot of the first two Principal Components across the SNP data set. Labels on the axes indicate the contributions of the displayed principal components to the variance.</p></figcaption></figure><p>The co‐ancestry matrix inferred by <span class="font-variant-small-caps">fineradstructure</span> was consistent with the PCA and clustered the specimens according to their sex (Figure <a href="#ece371847-supitem-0001" class="usa-link">S3</a>). When computing haplotype‐based divergence statistics between geographic locations, both φ<sub>ST</sub> and <em>D</em>
<sub>XY</sub> were uniformly distributed along the scaffolds (Figure <a href="#ece371847-supitem-0001" class="usa-link">S4</a>). However, φST was low on most of scaffold 06 compared to the rest of the assembly (Figure <a href="#ece371847-supitem-0001" class="usa-link">S4A</a>). We conclude that evidence for genetic structure is lacking in these data for geographic sites but is readily evident for differences between sexes.</p></section><section id="ece371847-sec-0012"><h3 class="pmc_sec_title">3.3. Sex Determination System</h3>
<p>When comparing sexes, φ<sub>ST</sub> calculated based on the haplotypic frequencies of markers from scaffold 06 (39,811,704 bp long in the 
<em>D. mawsoni</em>
 assembly and 26,171,853 bp long in the scaffolded 
<em>A. mitopteryx</em>
 assembly) was much higher than φ<sub>ST</sub> of all other scaffolds (Figure <a href="#ece371847-fig-0003" class="usa-link">3A</a>). <em>D</em>
<sub>XY</sub>, in contrast, was only slightly higher for scaffold 06 than for the other scaffolds (Figure <a href="#ece371847-fig-0003" class="usa-link">3B</a>). For both statistics, the first ~80% of the scaffold (from 0 to ~21 Mb of the scaffold in the 
<em>A. mitopteryx</em>
 assembly) showed this sex‐linked pattern. When removing from the dataset all markers on scaffold 06 (SNPs and haplotypes), both PCA and <span class="font-variant-small-caps">fineradstructure</span> (Figures <a href="#ece371847-supitem-0001" class="usa-link">S5</a> and <a href="#ece371847-supitem-0001" class="usa-link">S6</a>) did not return any clustering, and the <span class="font-variant-small-caps">admixture</span> result remained almost unchanged (Figure <a href="#ece371847-supitem-0001" class="usa-link">S2</a>, bottom panel).</p>
<figure class="fig xbox font-sm" id="ece371847-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0594/12367274/d3c6a48f6f0e/ECE3-15-e71847-g004.jpg" loading="lazy" id="jats-graphic-7" height="476" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/ece371847-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Manhattan plot of haplotype‐based divergence statistics φ<sub>ST</sub> (A) and <em>D</em>
<sub>XY</sub> (B) calculated between males and females of 
<em>Aethotaxis mitopteryx</em>
. Alternating black and gray dots represent single‐site values in different scaffolds. φ<sub>ST</sub> smoothed average is displayed in green, <em>D</em>
<sub>XY</sub> smoothed average is displayed in purple.</p></figcaption></figure><p>Homozygosity levels were also different between sexes on scaffold 06, but they were equivalent on all other scaffolds (Figure <a href="#ece371847-fig-0004" class="usa-link">4</a>). On scaffolds other than scaffold 06, homozygosity of variable sites was about 90% and was similar between males and females (Figure <a href="#ece371847-fig-0004" class="usa-link">4</a>). In stark contrast, on scaffold 06, males showed values around 60% homozygosity, which is much lower than the genome‐wide average of ~90%. In females, homozygosity values on scaffold 06 were slightly higher than the genome‐wide average (Figure <a href="#ece371847-fig-0004" class="usa-link">4</a>). The higher heterozygosity in males compared to females suggests that males are the heterogametic sex.</p>
<figure class="fig xbox font-sm" id="ece371847-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0594/12367274/c10be306be54/ECE3-15-e71847-g001.jpg" loading="lazy" id="jats-graphic-9" height="265" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/ece371847-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Observed homozygosity. Each dot represents an individual of 
<em>Aethotaxis mitopteryx</em>
 for each scaffold separately.</p></figcaption></figure><p>Sequencing depth was similar between males and females along the entire genome assembly, without noticeable differences on scaffold 06 (Figure <a href="#ece371847-fig-0005" class="usa-link">5</a>).</p>
<figure class="fig xbox font-sm" id="ece371847-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0594/12367274/caf564a4135d/ECE3-15-e71847-g005.jpg" loading="lazy" id="jats-graphic-11" height="329" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/ece371847-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Per SNP sequencing depth ratio between females and males of 
<em>Aethotaxis mitopteryx</em>
. Alternating black and gray dots represent single‐site values in different scaffolds. Smoothed average is displayed in red. All values were corrected for the mean ratio centered at 1.</p></figcaption></figure><p>Linkage disequilibrium (LD) was higher in scaffold 06 than in the rest of the genome assembly (Figure <a href="#ece371847-fig-0006" class="usa-link">6</a>). When analyzing males and females separately, no difference was found among scaffolds, although males showed overall higher LD values than females (Figure <a href="#ece371847-supitem-0001" class="usa-link">S7</a>).</p>
<figure class="fig xbox font-sm" id="ece371847-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0594/12367274/5b33487e3a2a/ECE3-15-e71847-g006.jpg" loading="lazy" id="jats-graphic-13" height="535" width="709" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/ece371847-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Linkage disequilibrium between SNPs along separate scaffolds considering all individuals of 
<em>Aethotaxis mitopteryx</em>
 analyzed in this study.</p></figcaption></figure><p>
<span class="font-variant-small-caps">radsex</span> recovered sex‐biased markers attributed exclusively to one or the other sex at all depth thresholds tested (Figure <a href="#ece371847-supitem-0001" class="usa-link">S8</a>). Despite this result, none of the markers were significantly sex‐linked. However, when markers were mapped to the genome assembly using the <span class="font-variant-small-caps">radsex</span> function <em>map</em>, a clear biased association between males and females was found on the first 20 Mb of scaffold 06 (Figure <a href="#ece371847-fig-0007" class="usa-link">7</a>). When considering only markers on scaffold 06, more markers were male‐specific and almost none were female‐specific (Figure <a href="#ece371847-supitem-0001" class="usa-link">S9</a>). Reducing the depth threshold for the <span class="font-variant-small-caps">radsex</span> analysis consistently recovered more male‐specific markers and fewer female‐specific markers. Further, comparing the original 
<em>D. mawsoni</em>
 genome assembly to the <em>C. trigloides</em> genome assembly revealed that 
<em>D. mawsoni</em>
 scaffold 06 corresponds to <em>C. trigloides</em> chromosome 21 (Figure <a href="#ece371847-supitem-0001" class="usa-link">S10</a>).</p>
<figure class="fig xbox font-sm" id="ece371847-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/0594/12367274/fe85938a528f/ECE3-15-e71847-g007.jpg" loading="lazy" id="jats-graphic-15" height="706" width="709" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/ece371847-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Plot showing the distribution of markers between females and males along the genome assembly, considering a sequencing depth threshold of 10. Each sector represents a different scaffold. The top track indicates the bias of a marker: If present in all males and in none of the females, a marker has a value of 1; if present in all females and in none of the males a marker has a value of −1. The bottom track indicates the probability of association with sex.</p></figcaption></figure></section><section id="ece371847-sec-0013"><h3 class="pmc_sec_title">3.4. Sex Determining Genes</h3>
<p>To identify sex determining gene candidates, we searched 
<em>A. mitopteryx</em>
 genome assembly for the genomic location of 35 genes known to be sex‐determining or involved in the sex differentiation pathway in other teleost species (Adolfi, Herpin, and Schartl <a href="#ece371847-bib-0002" class="usa-link" aria-describedby="ece371847-bib-0002">2021</a>; Kitano et al. <a href="#ece371847-bib-0064" class="usa-link" aria-describedby="ece371847-bib-0064">2024</a>; Pan, Kay, et al. <a href="#ece371847-bib-0094" class="usa-link" aria-describedby="ece371847-bib-0094">2021</a>). None of the searched genes were located within the sex‐linked region on scaffold 06; however, a <em>bmpr2‐like</em> gene paralog was located on scaffold 06 of 
<em>A. mitopteryx</em>
, 1.8 Mb upstream of the last sex‐linked marker (Table <a href="#ece371847-supitem-0001" class="usa-link">S1</a>). A few other genes (<em>bmpr1bb</em>, <em>esr1</em>, <em>gsdf</em>, and <em>gata4</em>) were also located on scaffolds other than 06 that contained few isolated sex‐linked markers; however, in each case they were positioned several Mb away from the sex‐linked SNPs (Table <a href="#ece371847-supitem-0001" class="usa-link">S1</a>).</p></section><section id="ece371847-sec-0014"><h3 class="pmc_sec_title">3.5. Sex Differentiation in 
<em>Dissostichus mawsoni</em>

</h3>
<p>The reanalysis of 
<em>D. mawsoni</em>
 RAD‐seq data (Ceballos et al. <a href="#ece371847-bib-0021" class="usa-link" aria-describedby="ece371847-bib-0021">2021</a>) using <span class="font-variant-small-caps">radsex</span> did not reveal any sex‐linked marker at any studied depth thresholds (Figure <a href="#ece371847-supitem-0001" class="usa-link">S11</a>) nor noticeable marker biases in any specific region of the genome assembly (Figure <a href="#ece371847-supitem-0001" class="usa-link">S12</a>).</p></section></section><section id="ece371847-sec-0015"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>In this study, we found high overall genomic homogeneity between two populations of 
<em>A. mitopteryx</em>
 sampled from the East Antarctic Peninsula (EAP) and the East Weddell Sea (EWS). In contrast, our analyses revealed a clear pattern of sex differentiation, which is based on sex‐biased markers concentrated along a single scaffold. High male heterozygosity suggests that 
<em>A. mitopteryx</em>
 displays a genetic sex determination system with males being the heterogametic sex. Thus, 
<em>A. mitopteryx</em>
 likely has a chromosomal sex‐determination system based on the presence of X and Y chromosomes.</p>
<section id="ece371847-sec-0016"><h3 class="pmc_sec_title">4.1. Population Structure</h3>
<p>None of the analyses we conducted to investigate population differentiation between the two sides of the Weddell Sea (i.e., PCA, unsupervised hierarchical clustering, and haplotype co‐ancestry matrix) detected any geographical population structure, with a low and statistically not significant genetic differentiation (<em>F</em>
<sub>ST</sub> ≈0.0001, <em>p</em>‐value = 0.431). Despite the large geographical distance between the two sampling sites, our result is consistent with the expectation that notothenioids have a high dispersal potential even on a continental scale, especially for pelagic species that may travel long distances as adults like Pleuragramminae and Dissostichinae. The long pelagic larval phase (from 2 months to up to 1 year depending on the species and locality; Kellerman <a href="#ece371847-bib-0061" class="usa-link" aria-describedby="ece371847-bib-0061">1989</a>; North <a href="#ece371847-bib-0092" class="usa-link" aria-describedby="ece371847-bib-0092">2001</a>), the local hydrography of different areas, and the dominant ocean currents in the Southern Ocean are generally considered to be the main drivers of population connectivity, often resulting in genetic homogeneity (Matschiner et al. <a href="#ece371847-bib-0084" class="usa-link" aria-describedby="ece371847-bib-0084">2015</a>). In particular, the eastern and western sectors of the Weddell Sea, where our samples originated, may be connected by several currents, mainly the ACC, ASC, and AACC, that flow in different directions and at different depths (e.g., Figure <a href="#ece371847-fig-0001" class="usa-link">1</a> in Caccavo et al. <a href="#ece371847-bib-0017" class="usa-link" aria-describedby="ece371847-bib-0017">2018</a>). It is also important to note that our two population samples were collected in different years (in 2016 for EWS and in 2018 for EAP). This could mean that gene flow is not only extensive across larger geographic distances but may also be sustained over time. The hypothesis that circulation promotes genetic connectivity between the two sides of the Weddell Sea was previously tested by Schiavon et al. (<a href="#ece371847-bib-0110" class="usa-link" aria-describedby="ece371847-bib-0110">2021</a>) for the icefishes of the genus <em>Chionodraco</em> (Nototheniidae, Channichthyinae), by combining Lagrangian particle tracking simulations with genetic analyses of nuclear (microsatellites) and mitochondrial DNA (D‐loop) markers. While Lagrangian simulations suggested limited bidirectional connectivity between the Western Antarctic Peninsula and the Eastern Weddell Sea, microsatellites and the D‐loop did not discriminate between 
<em>C. rastrospinosus</em>
 population samples collected in the EWS and in several other locations around the Antarctic Peninsula. Instead, the reanalysis of a subset of samples of Schiavon et al. (<a href="#ece371847-bib-0110" class="usa-link" aria-describedby="ece371847-bib-0110">2021</a>) using RAD‐seq revealed weak population differentiation (Schiavon et al. <a href="#ece371847-bib-0109" class="usa-link" aria-describedby="ece371847-bib-0109">2024</a>). Taken together, the genetic results suggest that gene flow occurs, and several hypotheses explaining the discrepancy between Lagrangian simulations and genetic data were put forward. Connectivity may occur in a stepping‐stone fashion through intermediate locations between the two sides of the Weddell Sea, where particle release was not simulated because no specimen captures were ever reported from these areas, although their presence cannot be excluded (Kattner and Koch <a href="#ece371847-bib-0060" class="usa-link" aria-describedby="ece371847-bib-0060">2009</a>). The movements of actively swimming adult individuals or the occasional strengthening of oceanographic currents due to weather conditions could also explain the connectivity between the two populations of 
<em>A. mitopteryx</em>
. Since 
<em>A. mitopteryx</em>
 is a pelagic species, the hypothesis of connectivity driven also by adult movement may be particularly relevant. Indeed, studies on two other pelagic notothenioids, 
<em>P. antarcticum</em>
 and 
<em>D. mawsoni</em>
, did not recover any signal of genetic differentiation at the circum‐Antarctic scale (Caccavo et al. <a href="#ece371847-bib-0017" class="usa-link" aria-describedby="ece371847-bib-0017">2018</a>; Ceballos et al. <a href="#ece371847-bib-0021" class="usa-link" aria-describedby="ece371847-bib-0021">2021</a>; Maschette et al. <a href="#ece371847-bib-0083" class="usa-link" aria-describedby="ece371847-bib-0083">2023</a>). However, areas of 
<em>P. antarcticum</em>
 occurrence that displayed reduced gene flow were separated by discontinuities in the large‐scale ocean circulation, highlighting the importance of currents in along‐shelf population connectivity for pelagic species (Caccavo et al. <a href="#ece371847-bib-0017" class="usa-link" aria-describedby="ece371847-bib-0017">2018</a>). In particular, populations from Charcot Island and Marguerite Bay, both below the Antarctic circle on the Western Antarctic Peninsula, showed genetic differentiation compared to nearby locations more north on the Western Antarctic Peninsula. Evidence that areas on opposite sides of the Antarctic continent are genetically homogeneous while closer locations host differentiated populations illustrates the importance of an extensive sampling design over space and time and suggests that environmental variability plays a key role in shaping population dynamics (Corso et al. <a href="#ece371847-bib-0024" class="usa-link" aria-describedby="ece371847-bib-0024">2022</a>).</p>
<p>Our small sample size and spatial coverage certainly limited the power to detect subtle genetic differences hindering our capacity to provide conclusive evidence of population differentiation. In particular, while we focused mostly on aspects of neutral differentiation, we cannot exclude that the two populations differ at loci experiencing different selective pressures in the two geographic areas. Moreover, although we obtained genetic data that differentiate males and females (see next sections of the Discussion), the small sample size and limited availability of genetic data do not allow testing for sex‐biased dispersal (e.g., Prugnolle and de Meeus <a href="#ece371847-bib-0100" class="usa-link" aria-describedby="ece371847-bib-0100">2002</a>). Therefore, although Antarctic field campaigns are logistically demanding, future studies should pay specific attention in accurately/appropriately sampling and preserving rare species such as 
<em>A. mitopteryx</em>
 and welcome international collaborations to expand sample sizes and spatial scale. New tools for species presence detection could also be used, like analysis of environmental DNA (eDNA, e.g., from water or sponge bycatch as in Jeunen et al. <a href="#ece371847-bib-0056" class="usa-link" aria-describedby="ece371847-bib-0056">2024</a>). As a future perspective, analysis of whole‐genome sequences obtained via e.g., PoolSeq or low‐coverage Whole Genome Sequencing (lcWGS) and of a high‐quality reference genome assembly for 
<em>A. mitopteryx</em>
 should be combined with non‐genetic approaches to overcome the limits of small sample size. Indeed, otolith microchemistry and stable isotope analyses on individuals at different developmental stages and from various areas will also help verify that genetic connectivity is shaped by oceanographic currents.</p></section><section id="ece371847-sec-0017"><h3 class="pmc_sec_title">4.2. Sex Differentiation and the Sex Determination System of 
<em>A. mitopteryx</em>

</h3>
<p>Males and females of 
<em>A. mitopteryx</em>
 are generally externally indistinguishable, although females grow larger than males (La Mesa et al. <a href="#ece371847-bib-0072" class="usa-link" aria-describedby="ece371847-bib-0072">2018</a>). Our data, however, provide the first indication of genetic differences between sexes in 
<em>A. mitopteryx</em>
. Both PCA and the co‐ancestry matrix computed using <span class="font-variant-small-caps">fineradstructure</span> clearly showed the presence of two groups, perfectly separating males and females (<em>F</em>
<sub>ST</sub> ≈0.0226; <em>p</em>‐value = 0.001). In contrast, hierarchical clustering performed by <span class="font-variant-small-caps">admixture</span> failed to identify the two groups. Although we cannot give a conclusive explanation for this difference, we hypothesize that it is due to the different assumptions the programs make about the data. The clustering method implemented in <span class="font-variant-small-caps">admixture</span> assumes that markers are in Hardy–Weinberg (HW) equilibrium, whereas PCA and <span class="font-variant-small-caps">fineradstructure</span> do not. Loci associated with sex may often fail to be in HW equilibrium, which could thus bias the inference made by <span class="font-variant-small-caps">admixture</span>.</p>
<p>The two haplotype‐based divergence statistics (i.e., φ<sub>ST</sub> and <em>D</em>
<sub>XY</sub>) showed that differences between sexes are linked to scaffold 06, suggesting the presence of genetically differentiated sex chromosomes in 
<em>A. mitopteryx</em>
. Homozygosity levels along the genome assembly are similar between the two sexes on all scaffolds except for a major drop in homozygosity in males on scaffold 06, while in females, the homozygosity on scaffold 06 remains within the range of other scaffolds. This observation further supports the presence of differentiated sex chromosomes in males and not in females, and a genetic sex determination system in 
<em>A. mitopteryx</em>
, with males being the heterogametic sex.</p>
<p>It is important to note that a chromosome‐level genome assembly for 
<em>A. mitopteryx</em>
 is not available yet. For this reason, in our study we ordered and oriented scaffolds of the initial 
<em>A. mitopteryx</em>
 draft assembly into longer pseudoscaffolds according to their order mapping onto the 
<em>D. mawsoni</em>
 genome assembly. Although <em>Dissostichus</em> is supposed to be the most closely related genus to 
<em>A. mitopteryx</em>
 (Near et al. <a href="#ece371847-bib-0089" class="usa-link" aria-describedby="ece371847-bib-0089">2018</a>) and colinearity of genes across notothenioids seems to be well conserved (Bista et al. <a href="#ece371847-bib-0010" class="usa-link" aria-describedby="ece371847-bib-0010">2020</a>; Kim et al. <a href="#ece371847-bib-0063" class="usa-link" aria-describedby="ece371847-bib-0063">2019</a>; Rayamajhi et al. <a href="#ece371847-bib-0103" class="usa-link" aria-describedby="ece371847-bib-0103">2024</a>), it is possible that the highly differentiated sex‐linked markers we recovered, seemingly clustered on a single scaffold, may in fact be distributed across multiple chromosomes or, inversely that other 
<em>D. mawsoni</em>
 chromosomes, be fused with this scaffold in 
<em>A. mitopteryx</em>
. Furthermore, the karyotype of 
<em>A. mitopteryx</em>
 is still unknown; hence we cannot conclude with certainty that this species has the same number of chromosomes as <em>Dissostichus</em> spp. and the hypothetical notothenioid ancestor (2n = 48, Ghigliotti et al. <a href="#ece371847-bib-0047" class="usa-link" aria-describedby="ece371847-bib-0047">2007</a>; Pisano et al. <a href="#ece371847-bib-0098" class="usa-link" aria-describedby="ece371847-bib-0098">1998</a>), and that 
<em>A. mitopteryx</em>
 has heteromorphic sex chromosomes. Although XX‐XY is the most common system when males are the heterogametic sex, cytological studies in other notothenioid species described heteromorphic sex chromosomes of either X1X1X2X2‐X1X2Y or XX‐XY1Y2 type in four of the eleven Antarctic notothenioid subfamilies (Nototheniinae, Artedidraconinae, Bathydraconinae, and Channichthyinae) (reviewed in Ghigliotti et al. <a href="#ece371847-bib-0046" class="usa-link" aria-describedby="ece371847-bib-0046">2015</a>). Although 
<em>D. mawsoni</em>
 and 
<em>D. eleginoides</em>
 do not have heteromorphic sex chromosomes (Ghigliotti et al. <a href="#ece371847-bib-0047" class="usa-link" aria-describedby="ece371847-bib-0047">2007</a>), the presence of heteromorphic sex chromosomes in 
<em>A. mitopteryx</em>
 (such as XY, X1X2Y and XY1Y2) cannot be excluded by our results. The comparison between the genome assemblies of 
<em>D. mawsoni</em>
 and <em>C. trigloides</em> revealed that the sex chromosome in 
<em>A. mitopteryx</em>
 corresponds to the notothenioid ancestral chromosome 21 (Bista et al. <a href="#ece371847-bib-0010" class="usa-link" aria-describedby="ece371847-bib-0010">2020</a>; Kim et al. <a href="#ece371847-bib-0063" class="usa-link" aria-describedby="ece371847-bib-0063">2019</a>).</p>
<p>Finally, the absence within the sex‐linked region of known sex‐determining genes or even of a major gene involved in gonad differentiation pathways could be explained if the sex‐determining gene in 
<em>A. mitopteryx</em>
 is not known to be a sex‐determining gene in other fish species, such as <em>irf9</em> in salmonids (Yano et al. <a href="#ece371847-bib-0126" class="usa-link" aria-describedby="ece371847-bib-0126">2012</a>) or <em>idb2</em> in arapaima (
<em>Arapaima gigas</em>
, freshwater bonytongue, order Osteoglossiformes; Adolfi, Du, et al. <a href="#ece371847-bib-0001" class="usa-link" aria-describedby="ece371847-bib-0001">2021</a>). Also, the fragmentation and incompleteness of the genome assembly in this study could preclude the identification of the sex‐determining gene if located in a part of the genome that was not sequenced and assembled or if it is a duplicated and transposed copy of an autosomal gene, such as in panga catfishes, pikes, or midas cichlids (Kitano et al. <a href="#ece371847-bib-0064" class="usa-link" aria-describedby="ece371847-bib-0064">2024</a>; Nacif et al. <a href="#ece371847-bib-0087" class="usa-link" aria-describedby="ece371847-bib-0087">2023</a>; Pan, Kay, et al. <a href="#ece371847-bib-0094" class="usa-link" aria-describedby="ece371847-bib-0094">2021</a>; Wen et al. <a href="#ece371847-bib-0125" class="usa-link" aria-describedby="ece371847-bib-0125">2022</a>), that would have been assembled with its autosomal paralogous gene.</p></section><section id="ece371847-sec-0018"><h3 class="pmc_sec_title">4.3. Origin of 
<em>A. mitopteryx</em>
 Sex Chromosomes</h3>
<p>To understand the evolution of the 
<em>A. mitopteryx</em>
 male heterogametic sex determination system, we reanalyzed a RAD‐seq dataset of about the same number of individuals from 
<em>D. mawsoni</em>
, one of the two <em>Dissotichus</em> spp. sister species of 
<em>A. mitopteryx</em>
. In stark contrast to 
<em>A. mitopteryx</em>
, for which even a small number of specimens were sufficient to reveal the presence of differentiated sex chromosomes, we found no evidence of sex‐linked regions in 
<em>D. mawsoni</em>
. This result suggests that either the 
<em>A. mitopteryx</em>
 sex determination system is not shared with 
<em>D. mawsoni</em>
, that the 
<em>D. mawsoni</em>
 sex chromosome is far less differentiated so that it cannot be detected by a RAD‐seq approach, or that the 
<em>D. mawsoni</em>
 dataset and its sample size were insufficient to detect sex‐linked loci.</p>
<p>The haplotype‐based divergence statistics φ<sub>ST</sub> and <em>D</em>
<sub>XY</sub> vary in the difference of their respective values between scaffold 06 and the rest of the assembly. While the average φ<sub>ST</sub> calculated from scaffold 06 data is clearly higher than the genome‐wide average, <em>D</em>
<sub>XY</sub> values for scaffold 06 are only slightly different from the rest of the genome assembly. Since φ<sub>ST</sub> is based on haplotypic frequencies and <em>D</em>
<sub>XY</sub> relies on the different number of substitutions between groups, it is possible that the differences in the haplotype‐based divergence statistics are due to only a moderate difference between males and females at scaffold 06. Higher values of φ<sub>ST</sub> compared to <em>D</em>
<sub>XY</sub> can also suggest a recent divergence of the putative sex chromosomes because <em>D</em>
<sub>XY</sub> tends to reflect divergence on distant time scales while φ<sub>ST</sub> tends to reflect divergence on a shorter time scale as allele frequencies can change faster than mutations arise (Cruickshank and Hahn <a href="#ece371847-bib-0025" class="usa-link" aria-describedby="ece371847-bib-0025">2014</a>). Further, the failure to detect any major differences in sequencing depth between males and females on scaffold 06 suggests little divergence between the sex‐linked regions and thus supports the hypothesis of young sex chromosomes. A chromosome‐level genome assembly for 
<em>A. mitopteryx</em>
 and a greater depth of sequencing of multiple samples could help test this hypothesis.</p>
<p>Together, both the differentiation of the putative sex chromosomes in 
<em>A. mitopteryx</em>
 and our inability to detect a similar sex determination system in 
<em>D. mawsoni</em>
 suggest that the sex determination system of 
<em>A. mitopteryx</em>
 is species‐specific and that it evolved after the divergence of 
<em>A. mitopteryx</em>
 and the <em>Dissostichus</em> spp. lineages. The alternative hypothesis, that the 
<em>A. mitopteryx</em>
 sex‐determination system is ancestral and was subsequently lost in the <em>Dissostichus</em> lineage, cannot, however, be ruled out because, outside of 
<em>Eleginops maclovinus</em>
 being the only known hermaphrodite notothenioid (Brickle et al. <a href="#ece371847-bib-0015" class="usa-link" aria-describedby="ece371847-bib-0015">2005</a>), the genetic sex determination system in more basally diverging notothenioid species remains to be studied.</p></section></section><section id="ece371847-sec-0019"><h2 class="pmc_sec_title">5. Conclusions</h2>
<p>Our study found a lack of spatial and temporal population structure for the pelagic notothenioid species 
<em>A. mitopteryx</em>
 between the Eastern Antarctic Peninsula and Eastern Weddell Sea. This finding may be explained by oceanic current regimes facilitating passive transport of larvae given the pelagic behavior of adults. Expanding the spatial scale and sample size would help overcome the limitations of this study and enable testing of the hypothesis of large‐scale connectivity in this fish with high potential for dispersal and circum‐Antarctic distribution. The rarity of the species and the difficult logistics of sampling in the Southern Ocean drive the need for a continued monitoring of species connectivity. This implies that more effort is needed to coordinate sampling across different institutes and countries that operate in the Southern Ocean (e.g., Jones et al. <a href="#ece371847-bib-0059" class="usa-link" aria-describedby="ece371847-bib-0059">2024</a>). Continuous monitoring of species connectivity will also be increasingly important to track possible range shifts in the Southern Ocean. In the near future, connectivity between the Antarctic Peninsula and the Weddell Sea, which is facilitated by the circulation pattern in the Southern Ocean, might be of crucial importance for cold‐adapted species living in Antarctic waters. The Antarctic Peninsula is one of the polar regions most affected by climate change and human activities, while the Weddell Sea may respond more slowly than other areas of the Southern Ocean to global warming (Raphael et al. <a href="#ece371847-bib-0102" class="usa-link" aria-describedby="ece371847-bib-0102">2025</a>; Strass et al. <a href="#ece371847-bib-0114" class="usa-link" aria-describedby="ece371847-bib-0114">2020</a>; Teschke et al. <a href="#ece371847-bib-0115" class="usa-link" aria-describedby="ece371847-bib-0115">2021</a>; Turner et al. <a href="#ece371847-bib-0118" class="usa-link" aria-describedby="ece371847-bib-0118">2020</a>) and has been speculated to act as a sink for migrating cold‐adapted species (Griffiths et al. <a href="#ece371847-bib-0049" class="usa-link" aria-describedby="ece371847-bib-0049">2017</a>). For this reason, connectivity between the Antarctic Peninsula and the Weddell Sea is a crucial aspect to monitor, and 
<em>A. mitopteryx</em>
 can serve as a proxy for pelagic species.</p>
<p>Furthermore, we demonstrated the genetic differentiation between male and female 
<em>A. mitopteryx</em>
. We propose that the species has a male heterogametic genetic sex determination system that is located on the notothenioid ancestral chromosome 21 (i.e., scaffold 06 in 
<em>D. mawsoni</em>
 and 
<em>A. mitopteryx</em>
), even though we failed to identify the exact sex‐determining gene. The finding of sex‐linked markers in 
<em>A. mitopteryx</em>
 leads to practical and theoretical considerations. Sex‐linked markers may bias estimates of population genetic diversity and differentiation if not properly recognized. Future efforts might target the expansion of robust genetic resources for 
<em>A. mitopteryx</em>
 to support the identification of sex‐linked markers to develop simple and cheap molecular tests to assign sex to individuals of unknown sex. These resources would valuably aid various types of experimental studies, as well as the description of sex‐specific ecological and evolutionary phenomena without having to sacrifice the individual.</p></section><section id="ece371847-sec-0025"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>
<strong>Luca Schiavon:</strong> conceptualization (equal), data curation (equal), formal analysis (equal), methodology (equal), visualization (equal), writing – original draft (equal), writing – review and editing (equal). <strong>Thomas Desvignes:</strong> conceptualization (supporting), formal analysis (supporting), methodology (supporting), validation (supporting), visualization (supporting), writing – original draft (equal), writing – review and editing (equal). <strong>Fabrizia Ronco:</strong> formal analysis (supporting), methodology (supporting), validation (supporting), writing – original draft (supporting), writing – review and editing (equal). <strong>Michael Matschiner:</strong> conceptualization (supporting), formal analysis (supporting), funding acquisition (supporting), methodology (supporting), validation (supporting), writing – original draft (supporting), writing – review and editing (equal). <strong>Martina Gastaldi:</strong> formal analysis (supporting), writing – original draft (supporting), writing – review and editing (supporting). <strong>Thore Koppetsch:</strong> formal analysis (supporting), methodology (supporting), writing – original draft (supporting), writing – review and editing (supporting). <strong>Audun Schrøder‐Nielsen:</strong> formal analysis (supporting), methodology (supporting), writing – original draft (supporting), writing – review and editing (supporting). <strong>John H. Postlethwait:</strong> formal analysis (supporting), methodology (supporting), writing – original draft (supporting), writing – review and editing (supporting). <strong>Alessia Prestanti:</strong> formal analysis (supporting), methodology (supporting), writing – review and editing (supporting). <strong>Federica Stranci:</strong> formal analysis (supporting), methodology (supporting), writing – review and editing (supporting). <strong>Santiago G. Ceballos:</strong> formal analysis (supporting), methodology (supporting), writing – review and editing (supporting). <strong>Felix C. Mark:</strong> investigation (supporting), resources (equal), writing – review and editing (supporting). <strong>Magnus Lucassen:</strong> funding acquisition (supporting), investigation (supporting), resources (equal), writing – review and editing (supporting). <strong>Emilio Riginella:</strong> investigation (supporting), resources (supporting), writing – review and editing (supporting). <strong>Mario La Mesa:</strong> conceptualization (supporting), funding acquisition (equal), investigation (supporting), project administration (equal), writing – original draft (supporting), writing – review and editing (supporting). <strong>Chiara Papetti:</strong> conceptualization (equal), formal analysis (supporting), funding acquisition (equal), investigation (equal), methodology (supporting), project administration (lead), resources (equal), supervision (lead), writing – original draft (equal), writing – review and editing (equal).</p></section><section id="ece371847-sec-0022"><h2 class="pmc_sec_title">Ethics Statement</h2>
<p>Fish were sampled and processed according to and within the laws, guidelines, and policies of the German and European Animal Welfare legislations. Sample collection during the cruises with R/V <em>Polarstern</em> was approved by the competent German authority for Antarctic research, the UBA (<em>Umweltbundesamt</em>).</p></section><section id="ece371847-sec-0021"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec23"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="ece371847-supitem-0001"><div class="caption p"><p>
Data S1.
</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367274/bin/ECE3-15-e71847-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71847-s001.docx</a><sup> (7.5MB, docx) </sup>
</div></div></section></section><section id="ece371847-sec-0020" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors would like to thank Nils Koschnick (Alfred Wegener Institute Helmholtz Centre for Polar and Marine Research, AWI, Bremerhaven, Germany) and the ship crew for their excellent contribution and support in collecting samples onboard the R/V <em>Polarstern</em>. The authors also thank Krissy Anne Reeve (AWI, Bremerhaven, Germany) for valuable insights into the Weddell Sea oceanography and physics.</p></section><section id="notes1"><p>


Schiavon, L.
, Desvignes T., Ronco F., et al. 2025. “Sex Differentiation and Long‐Distance Gene Flow in the Elusive Antarctic Fish 
<em>Aethotaxis mitopteryx</em>

.” Ecology and Evolution
15, no. 8: e71847. 10.1002/ece3.71847.
</p>
<section id="ece371847-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="ece371847-note-0001"><p>
<strong>Funding:</strong> This research was supported by the European Marie Curie project “Polarexpress” grant n. 622320, the University of Padova BIRD grant n.164793 and SEED2019, to Chiara Papetti. Chiara Papetti also acknowledges the EU funding EU Biodiversa+ WOBEC as UNIPD team leader (Project coordinator Hauke Flores, AWI, Bremerhaven, Germany). Michael Matschiner acknowledges funding from the Research Council of Norway (RCN) project 335549, Mario La Mesa acknowledges support by the Italian National Programme of Antarctic Research (PNRA) project BIOCLEVER PNRA19_00015‐A2, and Fabrizia Ronco was supported by a Post‐Doc Mobility grant of the Swiss National Foundation (SNSF) grant 206869. Felix Mark and Magnus Lucassen acknowledge the support by the Helmholtz research programme “Changing Earth – Sustaining our Future”. Thomas Desvignes and John H. Postlethwait acknowledge support from the National Science Foundation Office of Polar Program (NSF grant OPP‐2232891).</p></div></div></section></section><section id="ece371847-sec-0024"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>All the codes, datasets and accessory files needed to replicate the analyses are available at <a href="https://github.com/Papetti%E2%80%90Lab/Aethotaxis_RAD%E2%80%90Seq" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Papetti‐Lab/Aethotaxis_RAD‐Seq</a>. Raw sequence reads are available under the NCBI BioProject Accession number: PRJNA1214794 (<a href="https://www.ncbi.nlm.nih.gov/sra/PRJNA1214794" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/sra/PRJNA1214794</a>).</p></section><section id="ece371847-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ece371847-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="ece371847-bib-0001">
<cite>

Adolfi, M. C.
, Du K., Kneitz S., et al. 2021. “A Duplicated Copy of <em>id2b</em> Is an Unusual Sex‐Determining Candidate Gene on the Y Chromosome of Arapaima (<em>Arapaima gigas</em>).” Scientific Reports
11: 21544. 10.1038/s41598-021-01066-z.
</cite> [<a href="https://doi.org/10.1038/s41598-021-01066-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8566520/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34732792/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=A%20Duplicated%20Copy%20of%20id2b%20Is%20an%20Unusual%20Sex%E2%80%90Determining%20Candidate%20Gene%20on%20the%20Y%20Chromosome%20of%20Arapaima%20(Arapaima%20gigas)&amp;author=M.%20C.%20Adolfi&amp;author=K.%20Du&amp;author=S.%20Kneitz&amp;volume=11&amp;publication_year=2021&amp;pages=21544&amp;pmid=34732792&amp;doi=10.1038/s41598-021-01066-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0002">
<cite>

Adolfi, M. C.
, Herpin A., and Schartl M.. 2021. “The Replaceable Master of Sex Determination: Bottom‐Up Hypothesis Revisited.” Philosophical Transactions of the Royal Society, B: Biological Sciences
376: 20200090. 10.1098/rstb.2020.0090.</cite> [<a href="https://doi.org/10.1098/rstb.2020.0090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8273498/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34247496/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Philosophical%20Transactions%20of%20the%20Royal%20Society,%20B:%20Biological%20Sciences&amp;title=The%20Replaceable%20Master%20of%20Sex%20Determination:%20Bottom%E2%80%90Up%20Hypothesis%20Revisited&amp;author=M.%20C.%20Adolfi&amp;author=A.%20Herpin&amp;author=M.%20Schartl&amp;volume=376&amp;publication_year=2021&amp;pages=20200090&amp;pmid=34247496&amp;doi=10.1098/rstb.2020.0090&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0003">
<cite>

Alexander, D. H.
, Novembre J., and Lange K.. 2009. “Fast Model‐Based Estimation of Ancestry in Unrelated Individuals.” Genome Research
19: 1655–1664. 10.1101/gr.094052.109.
</cite> [<a href="https://doi.org/10.1101/gr.094052.109" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2752134/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19648217/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=Fast%20Model%E2%80%90Based%20Estimation%20of%20Ancestry%20in%20Unrelated%20Individuals&amp;author=D.%20H.%20Alexander&amp;author=J.%20Novembre&amp;author=K.%20Lange&amp;volume=19&amp;publication_year=2009&amp;pages=1655-1664&amp;pmid=19648217&amp;doi=10.1101/gr.094052.109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0004">
<cite>

Alonge, M.
, Lebeigle L., Kirsche M., et al. 2022. “Automated Assembly Scaffolding Using RagTag Elevates a New Tomato System for High‐Throughput Genome Editing.” Genome Biology
23: 258. 10.1186/s13059-022-02823-7.
</cite> [<a href="https://doi.org/10.1186/s13059-022-02823-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9753292/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36522651/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Automated%20Assembly%20Scaffolding%20Using%20RagTag%20Elevates%20a%20New%20Tomato%20System%20for%20High%E2%80%90Throughput%20Genome%20Editing&amp;author=M.%20Alonge&amp;author=L.%20Lebeigle&amp;author=M.%20Kirsche&amp;volume=23&amp;publication_year=2022&amp;pages=258&amp;pmid=36522651&amp;doi=10.1186/s13059-022-02823-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0005">
<cite>

Astashyn, A.
, Tvedte E. S., Sweeney D., et al. 2024. “Rapid and Sensitive Detection of Genome Contamination at Scale With FCS‐GX.” Genome Biology
25: 60. 10.1186/s13059-024-03198-7.
</cite> [<a href="https://doi.org/10.1186/s13059-024-03198-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10898089/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38409096/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Rapid%20and%20Sensitive%20Detection%20of%20Genome%20Contamination%20at%20Scale%20With%20FCS%E2%80%90GX&amp;author=A.%20Astashyn&amp;author=E.%20S.%20Tvedte&amp;author=D.%20Sweeney&amp;volume=25&amp;publication_year=2024&amp;pages=60&amp;pmid=38409096&amp;doi=10.1186/s13059-024-03198-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0006">
<cite>

Baird, N. A.
, Etter P. D., Atwood T. S., et al. 2008. “Rapid SNP Discovery and Genetic Mapping Using Sequenced RAD Markers.” PLoS One
3: e3376. 10.1371/journal.pone.0003376.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0003376" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2557064/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18852878/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Rapid%20SNP%20Discovery%20and%20Genetic%20Mapping%20Using%20Sequenced%20RAD%20Markers&amp;author=N.%20A.%20Baird&amp;author=P.%20D.%20Etter&amp;author=T.%20S.%20Atwood&amp;volume=3&amp;publication_year=2008&amp;pages=e3376&amp;pmid=18852878&amp;doi=10.1371/journal.pone.0003376&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0007">
<cite>

Benestan, L.
, Moore J., Sutherland B. J. G., et al. 2017. “Sex Matters in Massive Parallel Sequencing: Evidence for Biases in Genetic Parameter Estimation and Investigation of Sex Determination Systems.” Molecular Ecology
26: 6767–6783. 10.1111/mec.14217.
</cite> [<a href="https://doi.org/10.1111/mec.14217" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28658525/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Sex%20Matters%20in%20Massive%20Parallel%20Sequencing:%20Evidence%20for%20Biases%20in%20Genetic%20Parameter%20Estimation%20and%20Investigation%20of%20Sex%20Determination%20Systems&amp;author=L.%20Benestan&amp;author=J.%20Moore&amp;author=B.%20J.%20G.%20Sutherland&amp;volume=26&amp;publication_year=2017&amp;pages=6767-6783&amp;pmid=28658525&amp;doi=10.1111/mec.14217&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0008">
<cite>

Benestan, L.
, Quinn B. K., Maaroufi H., et al. 2016. “Seascape Genomics Provides Evidence for Thermal Adaptation and Current‐Mediated Population Structure in American Lobster (
<em>Homarus americanus</em>
).” Molecular Ecology
25: 5073–5092. 10.1111/mec.13811.
</cite> [<a href="https://doi.org/10.1111/mec.13811" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27543860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Seascape%20Genomics%20Provides%20Evidence%20for%20Thermal%20Adaptation%20and%20Current%E2%80%90Mediated%20Population%20Structure%20in%20American%20Lobster%20(%20Homarus%20americanus%20)&amp;author=L.%20Benestan&amp;author=B.%20K.%20Quinn&amp;author=H.%20Maaroufi&amp;volume=25&amp;publication_year=2016&amp;pages=5073-5092&amp;pmid=27543860&amp;doi=10.1111/mec.13811&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0009">
<cite>

Bernal‐Durán, V.
, Donoso D., Piñones A., et al. 2024. “Combining Population Genomics and Biophysical Modelling to Assess Connectivity Patterns in an Antarctic Fish.” Molecular Ecology
33: e17360. 10.1111/mec.17360.
</cite> [<a href="https://doi.org/10.1111/mec.17360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38656687/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Combining%20Population%20Genomics%20and%20Biophysical%20Modelling%20to%20Assess%20Connectivity%20Patterns%20in%20an%20Antarctic%20Fish&amp;author=V.%20Bernal%E2%80%90Dur%C3%A1n&amp;author=D.%20Donoso&amp;author=A.%20Pi%C3%B1ones&amp;volume=33&amp;publication_year=2024&amp;pages=e17360&amp;pmid=38656687&amp;doi=10.1111/mec.17360&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0010">
<cite>

Bista, I.
, McCarthy S. A., Wood J., et al. 2020. “The Genome Sequence of the Channel Bull Blenny, <em>Cottoperca Gobio</em> (Günther, 1861).” Wellcome Open Research
5: 148. 10.12688/wellcomeopenres.16012.1.
</cite> [<a href="https://doi.org/10.12688/wellcomeopenres.16012.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7649722/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33195818/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Wellcome%20Open%20Research&amp;title=The%20Genome%20Sequence%20of%20the%20Channel%20Bull%20Blenny,%20Cottoperca%20Gobio%20(G%C3%BCnther,%201861)&amp;author=I.%20Bista&amp;author=S.%20A.%20McCarthy&amp;author=J.%20Wood&amp;volume=5&amp;publication_year=2020&amp;pages=148&amp;pmid=33195818&amp;doi=10.12688/wellcomeopenres.16012.1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0011">
<cite>

Bista, I.
, Wood J. M. D., Desvignes T., et al. 2023. “Genomics of Cold Adaptations in the Antarctic Notothenioid Fish Radiation.” Nature Communications
14: 3412. 10.1038/s41467-023-38567-6.</cite> [<a href="https://doi.org/10.1038/s41467-023-38567-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10256766/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37296119/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Genomics%20of%20Cold%20Adaptations%20in%20the%20Antarctic%20Notothenioid%20Fish%20Radiation&amp;author=I.%20Bista&amp;author=J.%20M.%20D.%20Wood&amp;author=T.%20Desvignes&amp;volume=14&amp;publication_year=2023&amp;pages=3412&amp;pmid=37296119&amp;doi=10.1038/s41467-023-38567-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0012">
<cite>

Bolger, A. M.
, Lohse M., and Usadel B.. 2014. “Trimmomatic: A Flexible Trimmer for Illumina Sequence Data.” Bioinformatics
30: 2114–2120. 10.1093/bioinformatics/btu170.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4103590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Trimmomatic:%20A%20Flexible%20Trimmer%20for%20Illumina%20Sequence%20Data&amp;author=A.%20M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;volume=30&amp;publication_year=2014&amp;pages=2114-2120&amp;pmid=24695404&amp;doi=10.1093/bioinformatics/btu170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0013">
<cite>

Bootsma, M. L.
, Miller L., Sass G. G., Euclide P. T., and Larson W. A.. 2021. “The Ghosts of Propagation Past: Haplotype Information Clarifies the Relative Influence of Stocking History and Phylogeographic Processes on Contemporary Population Structure of Walleye (
<em>Sander vitreus</em>
).” Evolutionary Applications
14: 1124–1144. 10.1111/eva.13186.
</cite> [<a href="https://doi.org/10.1111/eva.13186" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8061267/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33897825/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolutionary%20Applications&amp;title=The%20Ghosts%20of%20Propagation%20Past:%20Haplotype%20Information%20Clarifies%20the%20Relative%20Influence%20of%20Stocking%20History%20and%20Phylogeographic%20Processes%20on%20Contemporary%20Population%20Structure%20of%20Walleye%20(%20Sander%20vitreus%20)&amp;author=M.%20L.%20Bootsma&amp;author=L.%20Miller&amp;author=G.%20G.%20Sass&amp;author=P.%20T.%20Euclide&amp;author=W.%20A.%20Larson&amp;volume=14&amp;publication_year=2021&amp;pages=1124-1144&amp;pmid=33897825&amp;doi=10.1111/eva.13186&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0014">
<cite>

Boscari, E.
, Abbiati M., Badalamenti F., et al. 2019. “A Population Genomics Insight by 2b‐RAD Reveals Populations' Uniqueness Along the Italian Coastline in <em>Leptopsammia pruvoti</em> (Scleractinia, Dendrophylliidae).” Diversity and Distributions
25: 1101–1117. 10.1111/ddi.12918.</cite> [<a href="https://doi.org/10.1111/ddi.12918" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diversity%20and%20Distributions&amp;title=A%20Population%20Genomics%20Insight%20by%202b%E2%80%90RAD%20Reveals%20Populations'%20Uniqueness%20Along%20the%20Italian%20Coastline%20in%20Leptopsammia%20pruvoti%20(Scleractinia,%20Dendrophylliidae)&amp;author=E.%20Boscari&amp;author=M.%20Abbiati&amp;author=F.%20Badalamenti&amp;volume=25&amp;publication_year=2019&amp;pages=1101-1117&amp;doi=10.1111/ddi.12918&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0015">
<cite>

Brickle, P.
, Laptikhovsky V., and Arkhipkin A.. 2005. “Reproductive Strategy of a Primitive Temperate Notothenioid <em>Eleginops maclovinus</em>
.” Journal of Fish Biology
66: 1044–1059. 10.1111/j.0022-1112.2005.00663.x.</cite> [<a href="https://doi.org/10.1111/j.0022-1112.2005.00663.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Fish%20Biology&amp;title=Reproductive%20Strategy%20of%20a%20Primitive%20Temperate%20Notothenioid%20Eleginops%20maclovinus&amp;author=P.%20Brickle&amp;author=V.%20Laptikhovsky&amp;author=A.%20Arkhipkin&amp;volume=66&amp;publication_year=2005&amp;pages=1044-1059&amp;doi=10.1111/j.0022-1112.2005.00663.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0016">
<cite>

Cabanettes, F.
, and Klopp C.. 2018. “D‐GENIES: Dot Plot Large Genomes in an Interactive, Efficient and Simple Way.” PeerJ
6: e4958. 10.7717/peerj.4958.
</cite> [<a href="https://doi.org/10.7717/peerj.4958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5991294/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29888139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=D%E2%80%90GENIES:%20Dot%20Plot%20Large%20Genomes%20in%20an%20Interactive,%20Efficient%20and%20Simple%20Way&amp;author=F.%20Cabanettes&amp;author=C.%20Klopp&amp;volume=6&amp;publication_year=2018&amp;pages=e4958&amp;pmid=29888139&amp;doi=10.7717/peerj.4958&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0017">
<cite>

Caccavo, J. A.
, Papetti C., Wetjen M., Knust R., Ashford J. R., and Zane L.. 2018. “Along‐Shelf Connectivity and Circumpolar Gene Flow in Antarctic Silverfish (<em>Pleuragramma antarctica</em>).” Scientific Reports
8: 1–16. 10.1038/s41598-018-36030-x.
</cite> [<a href="https://doi.org/10.1038/s41598-018-36030-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6294782/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30552350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Along%E2%80%90Shelf%20Connectivity%20and%20Circumpolar%20Gene%20Flow%20in%20Antarctic%20Silverfish%20(Pleuragramma%20antarctica)&amp;author=J.%20A.%20Caccavo&amp;author=C.%20Papetti&amp;author=M.%20Wetjen&amp;author=R.%20Knust&amp;author=J.%20R.%20Ashford&amp;volume=8&amp;publication_year=2018&amp;pages=1-16&amp;pmid=30552350&amp;doi=10.1038/s41598-018-36030-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0018">
<cite>

Catchen, J. M.
, Amores A., Hohenlohe P., Cresko W., and Postlethwait J. H.. 2011. “Stacks: Building and Genotyping Loci de Novo From Short‐Read Sequences.” G3: Genes, Genomes, Genetics
1, no. 3: 171–182. 10.1534/g3.111.000240.
</cite> [<a href="https://doi.org/10.1534/g3.111.000240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3276136/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22384329/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes,%20Genomes,%20Genetics&amp;title=Stacks:%20Building%20and%20Genotyping%20Loci%20de%20Novo%20From%20Short%E2%80%90Read%20Sequences&amp;author=J.%20M.%20Catchen&amp;author=A.%20Amores&amp;author=P.%20Hohenlohe&amp;author=W.%20Cresko&amp;author=J.%20H.%20Postlethwait&amp;volume=1&amp;issue=3&amp;publication_year=2011&amp;pages=171-182&amp;pmid=22384329&amp;doi=10.1534/g3.111.000240&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0019"><cite>
CCAMLR
. 2016. Conservation Measure 91–05 (2016), Ross Sea Region Marine Protected Area. <a href="https://cm.ccamlr.org/measure-91-05-2016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://cm.ccamlr.org/measure‐91‐05‐2016</a>.</cite></li>
<li id="ece371847-bib-0020"><cite>
CCAMLR
. 2020. Revised Proposal for a Conservation Measure Establishing a Marine Protected Area in Domain 1 (Western Antarctic Peninsula and South Scotia Arc). <a href="https://meetings.ccamlr.org/en/ccamlr-39/08-rev-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://meetings.ccamlr.org/en/ccamlr‐39/08‐rev‐1</a>.</cite></li>
<li id="ece371847-bib-0021">
<cite>

Ceballos, S. G.
, Papetti C., Babbucci M., Fernández D. A., Schiavon L., and Cheng C.‐H. C.. 2021. “Genome‐Wide Analysis Reveals Striking Lack of Genetic Differentiation Over Long Distances for the Antarctic Toothfish 
<em>Dissostichus mawsoni</em>
: High Genetic Connectivity or Shared Spawning Grounds?” Fisheries Research
243: 106074. 10.1016/j.fishres.2021.106074.</cite> [<a href="https://doi.org/10.1016/j.fishres.2021.106074" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Fisheries%20Research&amp;title=Genome%E2%80%90Wide%20Analysis%20Reveals%20Striking%20Lack%20of%20Genetic%20Differentiation%20Over%20Long%20Distances%20for%20the%20Antarctic%20Toothfish%20Dissostichus%20mawsoni%20:%20High%20Genetic%20Connectivity%20or%20Shared%20Spawning%20Grounds?&amp;author=S.%20G.%20Ceballos&amp;author=C.%20Papetti&amp;author=M.%20Babbucci&amp;author=D.%20A.%20Fern%C3%A1ndez&amp;author=L.%20Schiavon&amp;volume=243&amp;publication_year=2021&amp;pages=106074&amp;doi=10.1016/j.fishres.2021.106074&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0022">
<cite>

Chen, L.
, Lu Y., Li W., et al. 2019. “The Genomic Basis for Colonizing the Freezing Southern Ocean Revealed by Antarctic Toothfish and Patagonian Robalo Genomes.” GigaScience
8: giz016. 10.1093/gigascience/giz016.
</cite> [<a href="https://doi.org/10.1093/gigascience/giz016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6457430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30715292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GigaScience&amp;title=The%20Genomic%20Basis%20for%20Colonizing%20the%20Freezing%20Southern%20Ocean%20Revealed%20by%20Antarctic%20Toothfish%20and%20Patagonian%20Robalo%20Genomes&amp;author=L.%20Chen&amp;author=Y.%20Lu&amp;author=W.%20Li&amp;volume=8&amp;publication_year=2019&amp;pages=giz016&amp;pmid=30715292&amp;doi=10.1093/gigascience/giz016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0023">
<cite>

Chopelet, J.
, Blier P. U., and Dufresne F.. 2008. “Plasticity of Growth Rate and Metabolism in <em>Daphnia magna</em> Populations From Different Thermal Habitats.” Journal of Experimental Zoology Part A: Ecological Genetics and Physiology
309: 553–562. 10.1002/jez.488.
</cite> [<a href="https://doi.org/10.1002/jez.488" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18668644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Zoology%20Part%20A:%20Ecological%20Genetics%20and%20Physiology&amp;title=Plasticity%20of%20Growth%20Rate%20and%20Metabolism%20in%20Daphnia%20magna%20Populations%20From%20Different%20Thermal%20Habitats&amp;author=J.%20Chopelet&amp;author=P.%20U.%20Blier&amp;author=F.%20Dufresne&amp;volume=309&amp;publication_year=2008&amp;pages=553-562&amp;pmid=18668644&amp;doi=10.1002/jez.488&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0024">
<cite>

Corso, A. D.
, Steinberg D. K., Stammerjohn S. E., and Hilton E. J.. 2022. “Climate Drives Long‐Term Change in Antarctic Silverfish Along the Western Antarctic Peninsula.” Communications Biology
5: 104. 10.1038/s42003-022-03042-3.
</cite> [<a href="https://doi.org/10.1038/s42003-022-03042-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8813954/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35115634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Communications%20Biology&amp;title=Climate%20Drives%20Long%E2%80%90Term%20Change%20in%20Antarctic%20Silverfish%20Along%20the%20Western%20Antarctic%20Peninsula&amp;author=A.%20D.%20Corso&amp;author=D.%20K.%20Steinberg&amp;author=S.%20E.%20Stammerjohn&amp;author=E.%20J.%20Hilton&amp;volume=5&amp;publication_year=2022&amp;pages=104&amp;pmid=35115634&amp;doi=10.1038/s42003-022-03042-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0025">
<cite>

Cruickshank, T. E.
, and Hahn M. W.. 2014. “Reanalysis Suggests That Genomic Islands of Speciation Are due to Reduced Diversity, Not Reduced Gene Flow.” Molecular Ecology
23: 3133–3157. 10.1111/mec.12796.
</cite> [<a href="https://doi.org/10.1111/mec.12796" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24845075/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Reanalysis%20Suggests%20That%20Genomic%20Islands%20of%20Speciation%20Are%20due%20to%20Reduced%20Diversity,%20Not%20Reduced%20Gene%20Flow&amp;author=T.%20E.%20Cruickshank&amp;author=M.%20W.%20Hahn&amp;volume=23&amp;publication_year=2014&amp;pages=3133-3157&amp;pmid=24845075&amp;doi=10.1111/mec.12796&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0026">
<cite>

Damerau, M.
, Matschiner M., Salzburger W., and Hanel R.. 2012. “Comparative Population Genetics of Seven Notothenioid Fish Species Reveals High Levels of Gene Flow Along Ocean Currents in the Southern Scotia Arc, Antarctica.” Polar Biology
35: 1073–1086. 10.1007/s00300-012-1155-x.</cite> [<a href="https://doi.org/10.1007/s00300-012-1155-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Polar%20Biology&amp;title=Comparative%20Population%20Genetics%20of%20Seven%20Notothenioid%20Fish%20Species%20Reveals%20High%20Levels%20of%20Gene%20Flow%20Along%20Ocean%20Currents%20in%20the%20Southern%20Scotia%20Arc,%20Antarctica&amp;author=M.%20Damerau&amp;author=M.%20Matschiner&amp;author=W.%20Salzburger&amp;author=R.%20Hanel&amp;volume=35&amp;publication_year=2012&amp;pages=1073-1086&amp;doi=10.1007/s00300-012-1155-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0027">
<cite>

Damerau, M.
, Salzburger W., and Hanel R.. 2014. “Population Genetic Structure of 
<em>Lepidonotothen larseni</em>
 Revisited: Cyb and Microsatellites Suggest Limited Connectivity in the Southern Ocean.” Marine Ecology Progress Series
517: 251–263. 10.3354/meps11061.</cite> [<a href="https://doi.org/10.3354/meps11061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Marine%20Ecology%20Progress%20Series&amp;title=Population%20Genetic%20Structure%20of%20Lepidonotothen%20larseni%20Revisited:%20Cyb%20and%20Microsatellites%20Suggest%20Limited%20Connectivity%20in%20the%20Southern%20Ocean&amp;author=M.%20Damerau&amp;author=W.%20Salzburger&amp;author=R.%20Hanel&amp;volume=517&amp;publication_year=2014&amp;pages=251-263&amp;doi=10.3354/meps11061&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0028">
<cite>

Danecek, P.
, Auton A., Abecasis G., et al. 2011. “The Variant Call Format and VCFtools.” Bioinformatics
27: 2156–2158. 10.1093/bioinformatics/btr330.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3137218/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21653522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=The%20Variant%20Call%20Format%20and%20VCFtools&amp;author=P.%20Danecek&amp;author=A.%20Auton&amp;author=G.%20Abecasis&amp;volume=27&amp;publication_year=2011&amp;pages=2156-2158&amp;pmid=21653522&amp;doi=10.1093/bioinformatics/btr330&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0029">
<cite>

DeRaad, D. A.

2022. “SNPfiltR: An R Package for Interactive and Reproducible SNP Filtering.” Molecular Ecology Resources
22: 2443–2453. 10.1111/1755-0998.13618.
</cite> [<a href="https://doi.org/10.1111/1755-0998.13618" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35398990/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology%20Resources&amp;title=SNPfiltR:%20An%20R%20Package%20for%20Interactive%20and%20Reproducible%20SNP%20Filtering&amp;author=D.%20A.%20DeRaad&amp;volume=22&amp;publication_year=2022&amp;pages=2443-2453&amp;pmid=35398990&amp;doi=10.1111/1755-0998.13618&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0030">
<cite>

DeVries, A. L.
, and Wohlschlag D. E.. 1969. “Freezing Resistance in Some Antarctic Fishes.” Science
163: 1073–1075. 10.1126/science.163.3871.1073.
</cite> [<a href="https://doi.org/10.1126/science.163.3871.1073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5764871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Freezing%20Resistance%20in%20Some%20Antarctic%20Fishes&amp;author=A.%20L.%20DeVries&amp;author=D.%20E.%20Wohlschlag&amp;volume=163&amp;publication_year=1969&amp;pages=1073-1075&amp;pmid=5764871&amp;doi=10.1126/science.163.3871.1073&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0031">
<cite>

DeWitt, H. H.
, Heemstra P. C., and Gon O.. 1990. “Nototheniidae.” In Fishes of the Southern Ocean, edited by Gon O. and Heemstra P. C.. JLB Smith Institute of Ichthyology.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Fishes%20of%20the%20Southern%20Ocean&amp;author=H.%20H.%20DeWitt&amp;author=P.%20C.%20Heemstra&amp;author=O.%20Gon&amp;publication_year=1990&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0032">
<cite>

Duhamel, G.
, Hulley P.‐A., Causse R., et al. 2014. “Biogeographic Patterns of Fish.” In Biogeographic Atlas of the Southern Ocean, edited by Broyer C. D. and Koubbi P., 327–362. Scientific Committee on Antarctic Research.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Biogeographic%20Atlas%20of%20the%20Southern%20Ocean&amp;author=G.%20Duhamel&amp;author=P.%E2%80%90A.%20Hulley&amp;author=R.%20Causse&amp;publication_year=2014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0033">
<cite>

Duman, J. G.

2015. “Animal Ice‐Binding (Antifreeze) Proteins and Glycolipids: An Overview With Emphasis on Physiological Function.” Journal of Experimental Biology
218: 1846–1855. 10.1242/jeb.116905.
</cite> [<a href="https://doi.org/10.1242/jeb.116905" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26085662/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Biology&amp;title=Animal%20Ice%E2%80%90Binding%20(Antifreeze)%20Proteins%20and%20Glycolipids:%20An%20Overview%20With%20Emphasis%20on%20Physiological%20Function&amp;author=J.%20G.%20Duman&amp;volume=218&amp;publication_year=2015&amp;pages=1846-1855&amp;pmid=26085662&amp;doi=10.1242/jeb.116905&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0034">
<cite>

Eastman, J. T.

1985. “The Evolution of Neutrally Buoyant Notothenioid Fishes: Their Specializations and Potential Interactions in the Antarctic Marine Food Web.” In Antarctic Nutrient Cycles and Food Webs, 430–436. Springer Berlin Heidelberg. 10.1007/978-3-642-82275-9_60.</cite> [<a href="https://doi.org/10.1007/978-3-642-82275-9_60" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Antarctic%20Nutrient%20Cycles%20and%20Food%20Webs&amp;author=J.%20T.%20Eastman&amp;publication_year=1985&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0035">
<cite>

Eastman, J. T.

1993. Antarctic Fish Biology. Evolution in a Unique Environment. Academic Press.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Antarctic%20Fish%20Biology.%20Evolution%20in%20a%20Unique%20Environment&amp;author=J.%20T.%20Eastman&amp;publication_year=1993&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0036">
<cite>

Eastman, J. T.

2005. “The Nature of the Diversity of Antarctic Fishes.” Polar Biology
28: 93–107. 10.1007/s00300-004-0667-4.</cite> [<a href="https://doi.org/10.1007/s00300-004-0667-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Polar%20Biology&amp;title=The%20Nature%20of%20the%20Diversity%20of%20Antarctic%20Fishes&amp;author=J.%20T.%20Eastman&amp;volume=28&amp;publication_year=2005&amp;pages=93-107&amp;doi=10.1007/s00300-004-0667-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0037">
<cite>

Eastman, J. T.

2017. “Bathymetric Distributions of Notothenioid Fishes.” Polar Biology
40: 2077–2095. 10.1007/s00300-017-2128-x.</cite> [<a href="https://doi.org/10.1007/s00300-017-2128-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Polar%20Biology&amp;title=Bathymetric%20Distributions%20of%20Notothenioid%20Fishes&amp;author=J.%20T.%20Eastman&amp;volume=40&amp;publication_year=2017&amp;pages=2077-2095&amp;doi=10.1007/s00300-017-2128-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0038">
<cite>

Eastman, J. T.

2024. “The Axes of Divergence for the Evolutionary Radiation of Notothenioid Fishes in Antarctica.” Diversity
16: 214. 10.3390/d16040214.</cite> [<a href="https://doi.org/10.3390/d16040214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diversity&amp;title=The%20Axes%20of%20Divergence%20for%20the%20Evolutionary%20Radiation%20of%20Notothenioid%20Fishes%20in%20Antarctica&amp;author=J.%20T.%20Eastman&amp;volume=16&amp;publication_year=2024&amp;pages=214&amp;doi=10.3390/d16040214&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0039">
<cite>

Eastman, J. T.
, and DeVries A. L.. 1989. “Ultrastructure of the Lipid Sac Wall in the Antarctic Notothenioid Fish 
<em>Pleuragramma antarcticum</em>

.” Polar Biology
9: 333–335. 10.1007/BF00287433.</cite> [<a href="https://doi.org/10.1007/BF00287433" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Polar%20Biology&amp;title=Ultrastructure%20of%20the%20Lipid%20Sac%20Wall%20in%20the%20Antarctic%20Notothenioid%20Fish%20Pleuragramma%20antarcticum&amp;author=J.%20T.%20Eastman&amp;author=A.%20L.%20DeVries&amp;volume=9&amp;publication_year=1989&amp;pages=333-335&amp;doi=10.1007/BF00287433&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0040">
<cite>

Eastman, J. T.
, and McCune A. R.. 2000. “Fishes on the Antarctic Continental Shelf: Evolution of a Marine Species Flock?” Journal of Fish Biology
57: 84–102. 10.1006/jfbi.2000.1604.</cite> [<a href="https://doi.org/10.1006/jfbi.2000.1604" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Fish%20Biology&amp;title=Fishes%20on%20the%20Antarctic%20Continental%20Shelf:%20Evolution%20of%20a%20Marine%20Species%20Flock?&amp;author=J.%20T.%20Eastman&amp;author=A.%20R.%20McCune&amp;volume=57&amp;publication_year=2000&amp;pages=84-102&amp;doi=10.1006/jfbi.2000.1604&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0041">
<cite>

Eastman, J. T.
, and Sidell B. D.. 2002. “Measurements of Buoyancy for Some Antarctic Notothenioid Fishes From the South Shetland Islands.” Polar Biology
25: 753–760. 10.1007/s00300-002-0398-3.</cite> [<a href="https://doi.org/10.1007/s00300-002-0398-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Polar%20Biology&amp;title=Measurements%20of%20Buoyancy%20for%20Some%20Antarctic%20Notothenioid%20Fishes%20From%20the%20South%20Shetland%20Islands&amp;author=J.%20T.%20Eastman&amp;author=B.%20D.%20Sidell&amp;volume=25&amp;publication_year=2002&amp;pages=753-760&amp;doi=10.1007/s00300-002-0398-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0042">
<cite>

Eastman, J. T.
, and Voskoboinikova O. S.. 2024. “Osteology Provides Insight Into the Biology of the Enigmatic Antarctic Notothenioid Fish 
<em>Gvozdarus svetovidovi</em>

.” Polar Biology
47: 1137–1149. 10.1007/s00300-024-03290-5.</cite> [<a href="https://doi.org/10.1007/s00300-024-03290-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Polar%20Biology&amp;title=Osteology%20Provides%20Insight%20Into%20the%20Biology%20of%20the%20Enigmatic%20Antarctic%20Notothenioid%20Fish%20Gvozdarus%20svetovidovi&amp;author=J.%20T.%20Eastman&amp;author=O.%20S.%20Voskoboinikova&amp;volume=47&amp;publication_year=2024&amp;pages=1137-1149&amp;doi=10.1007/s00300-024-03290-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0043">
<cite>

Excoffier, L.
, Smouse P. E., and Quattro J. M.. 1992. “Analysis of Molecular Variance Inferred From Metric Distances Among DNA Haplotypes: Application to Human Mitochondrial DNA Restriction Data.” Genetics
131: 479–491. 10.1093/genetics/131.2.479.
</cite> [<a href="https://doi.org/10.1093/genetics/131.2.479" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1205020/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1644282/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Analysis%20of%20Molecular%20Variance%20Inferred%20From%20Metric%20Distances%20Among%20DNA%20Haplotypes:%20Application%20to%20Human%20Mitochondrial%20DNA%20Restriction%20Data&amp;author=L.%20Excoffier&amp;author=P.%20E.%20Smouse&amp;author=J.%20M.%20Quattro&amp;volume=131&amp;publication_year=1992&amp;pages=479-491&amp;pmid=1644282&amp;doi=10.1093/genetics/131.2.479&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0044">
<cite>

Feller, A. F.
, Ogi V., Seehausen O., and Meier J. I.. 2021. “Identification of a Novel Sex Determining Chromosome in Cichlid Fishes That Acts as XY or ZW in Different Lineages.” Hydrobiologia
848: 3727–3745. 10.1007/s10750-021-04560-7.
</cite> [<a href="https://doi.org/10.1007/s10750-021-04560-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8550731/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34720170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hydrobiologia&amp;title=Identification%20of%20a%20Novel%20Sex%20Determining%20Chromosome%20in%20Cichlid%20Fishes%20That%20Acts%20as%20XY%20or%20ZW%20in%20Different%20Lineages&amp;author=A.%20F.%20Feller&amp;author=V.%20Ogi&amp;author=O.%20Seehausen&amp;author=J.%20I.%20Meier&amp;volume=848&amp;publication_year=2021&amp;pages=3727-3745&amp;pmid=34720170&amp;doi=10.1007/s10750-021-04560-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0045">
<cite>

Feron, R.
, Pan Q., Wen M., et al. 2021. “RADsex: A Computational Workflow to Study Sex Determination Using Restriction Site‐Associated DNA Sequencing Data.” Molecular Ecology Resources
21: 1715–1731. 10.1111/1755-0998.13360.
</cite> [<a href="https://doi.org/10.1111/1755-0998.13360" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8589568/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33590960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology%20Resources&amp;title=RADsex:%20A%20Computational%20Workflow%20to%20Study%20Sex%20Determination%20Using%20Restriction%20Site%E2%80%90Associated%20DNA%20Sequencing%20Data&amp;author=R.%20Feron&amp;author=Q.%20Pan&amp;author=M.%20Wen&amp;volume=21&amp;publication_year=2021&amp;pages=1715-1731&amp;pmid=33590960&amp;doi=10.1111/1755-0998.13360&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0046">
<cite>

Ghigliotti, L.
, Cheng C. C.‐H., Ozouf‐Costaz C., Vacchi M., and Pisano E.. 2015. “Cytogenetic Diversity of Notothenioid Fish From the Ross Sea: Historical Overview and Updates.” Hydrobiologia
761: 373–396. 10.1007/s10750-015-2355-5.</cite> [<a href="https://doi.org/10.1007/s10750-015-2355-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hydrobiologia&amp;title=Cytogenetic%20Diversity%20of%20Notothenioid%20Fish%20From%20the%20Ross%20Sea:%20Historical%20Overview%20and%20Updates&amp;author=L.%20Ghigliotti&amp;author=C.%20C.%E2%80%90H.%20Cheng&amp;author=C.%20Ozouf%E2%80%90Costaz&amp;author=M.%20Vacchi&amp;author=E.%20Pisano&amp;volume=761&amp;publication_year=2015&amp;pages=373-396&amp;doi=10.1007/s10750-015-2355-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0047">
<cite>

Ghigliotti, L.
, Mazzei F., Ozouf‐Costaz C., et al. 2007. “The Two Giant Sister Species of the Southern Ocean, <em>Dissostichus eleginoides and Dissostichus mawsoni
</em>, Differ in Karyotype and Chromosomal Pattern of Ribosomal RNA Genes.” Polar Biology
30: 625–634. 10.1007/s00300-006-0222-6.</cite> [<a href="https://doi.org/10.1007/s00300-006-0222-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Polar%20Biology&amp;title=The%20Two%20Giant%20Sister%20Species%20of%20the%20Southern%20Ocean,%20Dissostichus%20eleginoides%20and%20Dissostichus%20mawsoni%20,%20Differ%20in%20Karyotype%20and%20Chromosomal%20Pattern%20of%20Ribosomal%20RNA%20Genes&amp;author=L.%20Ghigliotti&amp;author=F.%20Mazzei&amp;author=C.%20Ozouf%E2%80%90Costaz&amp;volume=30&amp;publication_year=2007&amp;pages=625-634&amp;doi=10.1007/s00300-006-0222-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0048">
<cite>

Graham, J. A.
, Heywood K. J., Chavanne C. P., and Holland P. R.. 2013. “Seasonal Variability of Water Masses and Transport on the Antarctic Continental Shelf and Slope in the Southeastern Weddell Sea.” Journal of Geophysical Research: Oceans
118: 2201–2214. 10.1002/jgrc.20174.</cite> [<a href="https://doi.org/10.1002/jgrc.20174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Geophysical%20Research:%20Oceans&amp;title=Seasonal%20Variability%20of%20Water%20Masses%20and%20Transport%20on%20the%20Antarctic%20Continental%20Shelf%20and%20Slope%20in%20the%20Southeastern%20Weddell%20Sea&amp;author=J.%20A.%20Graham&amp;author=K.%20J.%20Heywood&amp;author=C.%20P.%20Chavanne&amp;author=P.%20R.%20Holland&amp;volume=118&amp;publication_year=2013&amp;pages=2201-2214&amp;doi=10.1002/jgrc.20174&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0049">
<cite>

Griffiths, H. J.
, Meijers A. J. S., and Bracegirdle T. J.. 2017. “More Losers Than Winners in a Century of Future Southern Ocean Seafloor Warming.” Nature Climate Change
7: 749–754. 10.1038/nclimate3377.</cite> [<a href="https://doi.org/10.1038/nclimate3377" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Climate%20Change&amp;title=More%20Losers%20Than%20Winners%20in%20a%20Century%20of%20Future%20Southern%20Ocean%20Seafloor%20Warming&amp;author=H.%20J.%20Griffiths&amp;author=A.%20J.%20S.%20Meijers&amp;author=T.%20J.%20Bracegirdle&amp;volume=7&amp;publication_year=2017&amp;pages=749-754&amp;doi=10.1038/nclimate3377&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0050">
<cite>

Gurevich, A.
, Saveliev V., Vyahhi N., and Tesler G.. 2013. “QUAST: Quality Assessment Tool for Genome Assemblies.” Bioinformatics
29: 1072–1075. 10.1093/bioinformatics/btt086.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3624806/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23422339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=QUAST:%20Quality%20Assessment%20Tool%20for%20Genome%20Assemblies&amp;author=A.%20Gurevich&amp;author=V.%20Saveliev&amp;author=N.%20Vyahhi&amp;author=G.%20Tesler&amp;volume=29&amp;publication_year=2013&amp;pages=1072-1075&amp;pmid=23422339&amp;doi=10.1093/bioinformatics/btt086&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0051">
<cite>

Halpern, B. S.

2003. “The Impact of Marine Reserves: Do Reserves Work and Does Reserve Size Matter?” Ecological Applications
13: 117–137. 10.1890/1051-0761(2003)013[0117:tiomrd]2.0.co;2.</cite> [<a href="https://doi.org/10.1890/1051-0761(2003)013%5B0117:tiomrd%5D2.0.co;2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecological%20Applications&amp;title=The%20Impact%20of%20Marine%20Reserves:%20Do%20Reserves%20Work%20and%20Does%20Reserve%20Size%20Matter?&amp;author=B.%20S.%20Halpern&amp;volume=13&amp;publication_year=2003&amp;pages=117-137&amp;doi=10.1890/1051-0761(2003)013%5B0117:tiomrd%5D2.0.co;2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0052">
<cite>

Hanchet, S.
, Dunn A., Parker S., Horn P., Stevens D., and Mormede S.. 2015. “The Antarctic Toothfish (
<em>Dissostichus mawsoni</em>
): Biology, Ecology, and Life History in the Ross Sea Region.” Hydrobiologia
761: 397–414. 10.1007/s10750-015-2435-6.</cite> [<a href="https://doi.org/10.1007/s10750-015-2435-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hydrobiologia&amp;title=The%20Antarctic%20Toothfish%20(%20Dissostichus%20mawsoni%20):%20Biology,%20Ecology,%20and%20Life%20History%20in%20the%20Ross%20Sea%20Region&amp;author=S.%20Hanchet&amp;author=A.%20Dunn&amp;author=S.%20Parker&amp;author=P.%20Horn&amp;author=D.%20Stevens&amp;volume=761&amp;publication_year=2015&amp;pages=397-414&amp;doi=10.1007/s10750-015-2435-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0053">
<cite>

Hauser, L.
, and Carvalho G. R.. 2008. “Paradigm Shifts in Marine Fisheries Genetics: Ugly Hypotheses Slain by Beautiful Facts.” Fish and Fisheries
9: 333–362. 10.1111/j.1467-2979.2008.00299.x.</cite> [<a href="https://doi.org/10.1111/j.1467-2979.2008.00299.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Fish%20and%20Fisheries&amp;title=Paradigm%20Shifts%20in%20Marine%20Fisheries%20Genetics:%20Ugly%20Hypotheses%20Slain%20by%20Beautiful%20Facts&amp;author=L.%20Hauser&amp;author=G.%20R.%20Carvalho&amp;volume=9&amp;publication_year=2008&amp;pages=333-362&amp;doi=10.1111/j.1467-2979.2008.00299.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0054">
<cite>

Hilário, A.
, Metaxas A., Gaudron S. M., et al. 2015. “Estimating Dispersal Distance in the Deep Sea: Challenges and Applications to Marine Reserves.” Frontiers in Marine Science
2: 6. 10.3389/fmars.2015.00006.</cite> [<a href="https://doi.org/10.3389/fmars.2015.00006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Marine%20Science&amp;title=Estimating%20Dispersal%20Distance%20in%20the%20Deep%20Sea:%20Challenges%20and%20Applications%20to%20Marine%20Reserves&amp;author=A.%20Hil%C3%A1rio&amp;author=A.%20Metaxas&amp;author=S.%20M.%20Gaudron&amp;volume=2&amp;publication_year=2015&amp;pages=6&amp;doi=10.3389/fmars.2015.00006&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0055">
<cite>

Jackman, S. D.
, Vandervalk B. P., Mohamadi H., et al. 2017. “ABySS 2.0: Resource‐Efficient Assembly of Large Genomes Using a Bloom Filter.” Genome Research
27: 768–777. 10.1101/gr.214346.116.
</cite> [<a href="https://doi.org/10.1101/gr.214346.116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5411771/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28232478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=ABySS%202.0:%20Resource%E2%80%90Efficient%20Assembly%20of%20Large%20Genomes%20Using%20a%20Bloom%20Filter&amp;author=S.%20D.%20Jackman&amp;author=B.%20P.%20Vandervalk&amp;author=H.%20Mohamadi&amp;volume=27&amp;publication_year=2017&amp;pages=768-777&amp;pmid=28232478&amp;doi=10.1101/gr.214346.116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0056">
<cite>

Jeunen, G.‐J.
, Lamare M., Devine J., et al. 2024. “Characterizing Antarctic Fish Assemblages Using eDNA Obtained From Marine Sponge Bycatch Specimens.” Reviews in Fish Biology and Fisheries
34: 221–238. 10.1007/s11160-023-09805-3.</cite> [<a href="https://doi.org/10.1007/s11160-023-09805-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Reviews%20in%20Fish%20Biology%20and%20Fisheries&amp;title=Characterizing%20Antarctic%20Fish%20Assemblages%20Using%20eDNA%20Obtained%20From%20Marine%20Sponge%20Bycatch%20Specimens&amp;author=G.%E2%80%90J.%20Jeunen&amp;author=M.%20Lamare&amp;author=J.%20Devine&amp;volume=34&amp;publication_year=2024&amp;pages=221-238&amp;doi=10.1007/s11160-023-09805-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0057">
<cite>

Jombart, T.

2008. “Adegenet: A R Package for the Multivariate Analysis of Genetic Markers.” Bioinformatics
24: 1403–1405. 10.1093/bioinformatics/btn129.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btn129" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18397895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Adegenet:%20A%20R%20Package%20for%20the%20Multivariate%20Analysis%20of%20Genetic%20Markers&amp;author=T.%20Jombart&amp;volume=24&amp;publication_year=2008&amp;pages=1403-1405&amp;pmid=18397895&amp;doi=10.1093/bioinformatics/btn129&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0058">
<cite>

Jones, C. D.
, Anderson M. E., Balushkin A. V., et al. 2008. “Diversity, Relative Abundance, New Locality Records and Population Structure of Antarctic Demersal Fishes From the Northern Scotia Arc Islands and Bouvetøya.” Polar Biology
31: 1481–1497. 10.1007/s00300-008-0489-x.</cite> [<a href="https://doi.org/10.1007/s00300-008-0489-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Polar%20Biology&amp;title=Diversity,%20Relative%20Abundance,%20New%20Locality%20Records%20and%20Population%20Structure%20of%20Antarctic%20Demersal%20Fishes%20From%20the%20Northern%20Scotia%20Arc%20Islands%20and%20Bouvet%C3%B8ya&amp;author=C.%20D.%20Jones&amp;author=M.%20E.%20Anderson&amp;author=A.%20V.%20Balushkin&amp;volume=31&amp;publication_year=2008&amp;pages=1481-1497&amp;doi=10.1007/s00300-008-0489-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0059"><cite>

Jones, C. D.
, Caccavo J. A., Brooks C., et al. 2024. Introduction to the SCAR Action Group on Fish (SCARFISH). CCAMLR Document WG‐FSA‐IMAF‐2024/74, Hobart, Australia.</cite></li>
<li id="ece371847-bib-0060">
<cite>

Kattner, G.
, and Koch B.. 2009. The Expedition of the Research Vessel “Polarstern” to the Antarctic in 2008 (ANT‐XXV/1). Berichte Zur Polar–Und Meeresforschung.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20Expedition%20of%20the%20Research%20Vessel%20%E2%80%9CPolarstern%E2%80%9D%20to%20the%20Antarctic%20in%202008%20(ANT%E2%80%90XXV/1)&amp;author=G.%20Kattner&amp;author=B.%20Koch&amp;publication_year=2009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0061">
<cite>

Kellerman, A.

1989. “The Larval Fish Community in the Zone of Seasonal Pack‐Ice Cover and Its Seasonal and Interannual Variability.” Archiv für Fischereiwissenschaft
39: 81–109.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Archiv%20f%C3%BCr%20Fischereiwissenschaft&amp;title=The%20Larval%20Fish%20Community%20in%20the%20Zone%20of%20Seasonal%20Pack%E2%80%90Ice%20Cover%20and%20Its%20Seasonal%20and%20Interannual%20Variability&amp;author=A.%20Kellerman&amp;volume=39&amp;publication_year=1989&amp;pages=81-109&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0062">
<cite>

Kellermann, A.

1990. “Catalogue of Early Life Stages of Antarctic Notothenioid Fishes.” Berichte Zur Polarforschung
67: 45–136.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Berichte%20Zur%20Polarforschung&amp;title=Catalogue%20of%20Early%20Life%20Stages%20of%20Antarctic%20Notothenioid%20Fishes&amp;author=A.%20Kellermann&amp;volume=67&amp;publication_year=1990&amp;pages=45-136&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0063">
<cite>

Kim, B. M.
, Amores A., Kang S., et al. 2019. “Antarctic Blackfin Icefish Genome Reveals Adaptations to Extreme Environments.” Nature Ecology &amp; Evolution
3: 469–478. 10.1038/s41559-019-0812-7.
</cite> [<a href="https://doi.org/10.1038/s41559-019-0812-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7307600/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30804520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Ecology%20&amp;%20Evolution&amp;title=Antarctic%20Blackfin%20Icefish%20Genome%20Reveals%20Adaptations%20to%20Extreme%20Environments&amp;author=B.%20M.%20Kim&amp;author=A.%20Amores&amp;author=S.%20Kang&amp;volume=3&amp;publication_year=2019&amp;pages=469-478&amp;pmid=30804520&amp;doi=10.1038/s41559-019-0812-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0064">
<cite>

Kitano, J.
, Ansai S., Takehana Y., and Yamamoto Y.. 2024. “Diversity and Convergence of Sex‐Determination Mechanisms in Teleost Fish.” Annual Review of Animal Biosciences
12: 233–259. 10.1146/annurev-animal-021122-113935.
</cite> [<a href="https://doi.org/10.1146/annurev-animal-021122-113935" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37863090/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Animal%20Biosciences&amp;title=Diversity%20and%20Convergence%20of%20Sex%E2%80%90Determination%20Mechanisms%20in%20Teleost%20Fish&amp;author=J.%20Kitano&amp;author=S.%20Ansai&amp;author=Y.%20Takehana&amp;author=Y.%20Yamamoto&amp;volume=12&amp;publication_year=2024&amp;pages=233-259&amp;pmid=37863090&amp;doi=10.1146/annurev-animal-021122-113935&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0065">
<cite>

Knaus, B. J.
, and Grünwald N. J.. 2017. “VCFR: A Package to Manipulate and Visualize Variant Call Format Data in R.” Molecular Ecology Resources
17: 44–53. 10.1111/1755-0998.12549.
</cite> [<a href="https://doi.org/10.1111/1755-0998.12549" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27401132/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology%20Resources&amp;title=VCFR:%20A%20Package%20to%20Manipulate%20and%20Visualize%20Variant%20Call%20Format%20Data%20in%20R&amp;author=B.%20J.%20Knaus&amp;author=N.%20J.%20Gr%C3%BCnwald&amp;volume=17&amp;publication_year=2017&amp;pages=44-53&amp;pmid=27401132&amp;doi=10.1111/1755-0998.12549&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0066">
<cite>

Koch, E. L.
, Ravinet M., Westram A. M., Johannesson K., and Butlin R. K.. 2022. “Genetic Architecture of Repeated Phenotypic Divergence in <em>Littorina saxatilis</em> Ecotype Evolution.” Evolution
76: 2332–2346. 10.1111/evo.14602.
</cite> [<a href="https://doi.org/10.1111/evo.14602" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9826283/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35994296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=Genetic%20Architecture%20of%20Repeated%20Phenotypic%20Divergence%20in%20Littorina%20saxatilis%20Ecotype%20Evolution&amp;author=E.%20L.%20Koch&amp;author=M.%20Ravinet&amp;author=A.%20M.%20Westram&amp;author=K.%20Johannesson&amp;author=R.%20K.%20Butlin&amp;volume=76&amp;publication_year=2022&amp;pages=2332-2346&amp;pmid=35994296&amp;doi=10.1111/evo.14602&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0067">
<cite>

Kock, K. H.

2005. “Antarctic Icefishes (Channichthyidae): A Unique Family of Fishes. A Review, Part I.” Polar Biology
28: 862–895. 10.1007/s00300-005-0019-z.</cite> [<a href="https://doi.org/10.1007/s00300-005-0019-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Polar%20Biology&amp;title=Antarctic%20Icefishes%20(Channichthyidae):%20A%20Unique%20Family%20of%20Fishes.%20A%20Review,%20Part%20I&amp;author=K.%20H.%20Kock&amp;volume=28&amp;publication_year=2005&amp;pages=862-895&amp;doi=10.1007/s00300-005-0019-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0068">
<cite>

Kock, K. H.
, and Kellermann A.. 1991. “Reproduction in Antarctic Notothenioid Fish.” Antarctic Science
3: 125–150. 10.1017/S0954102091000172.</cite> [<a href="https://doi.org/10.1017/S0954102091000172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antarctic%20Science&amp;title=Reproduction%20in%20Antarctic%20Notothenioid%20Fish&amp;author=K.%20H.%20Kock&amp;author=A.%20Kellermann&amp;volume=3&amp;publication_year=1991&amp;pages=125-150&amp;doi=10.1017/S0954102091000172&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0069">
<cite>

Kunzmann, A.
, and Zimmermann C.. 1992. “
<em>Aethotaxis mitopteryx</em>, a High‐Antarctic Fish With Benthopelagic Mode of Life.” Marine Ecology Progress Series
88: 33–40. <a href="https://api.semanticscholar.org/CorpusID:54178500" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://api.semanticscholar.org/CorpusID:54178500</a>.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Marine%20Ecology%20Progress%20Series&amp;title=Aethotaxis%20mitopteryx,%20a%20High%E2%80%90Antarctic%20Fish%20With%20Benthopelagic%20Mode%20of%20Life&amp;author=A.%20Kunzmann&amp;author=C.%20Zimmermann&amp;volume=88&amp;publication_year=1992&amp;pages=33-40&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0070">
<cite>

La Mesa, M.
, and Ashford J.. 2007. “Age and Early Life History of Juvenile Scotia Sea Icefish, 
<em>Chaenocephalus aceratus</em>
, From Elephant and the South Shetland Islands.” Polar Biology
31: 221–228. 10.1007/s00300-007-0351-6.</cite> [<a href="https://doi.org/10.1007/s00300-007-0351-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Polar%20Biology&amp;title=Age%20and%20Early%20Life%20History%20of%20Juvenile%20Scotia%20Sea%20Icefish,%20Chaenocephalus%20aceratus%20,%20From%20Elephant%20and%20the%20South%20Shetland%20Islands&amp;author=M.%20La%20Mesa&amp;author=J.%20Ashford&amp;volume=31&amp;publication_year=2007&amp;pages=221-228&amp;doi=10.1007/s00300-007-0351-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0071">
<cite>

La Mesa, M.
, and Ashford J.. 2008. “Age and Growth of Ocellated Icefish, 
<em>Chionodraco rastrospinosus</em>
 DeWitt and Hureau, 1979, From the South Shetland Islands.” Polar Biology
31: 1333–1342. 10.1007/s00300-008-0471-7.</cite> [<a href="https://doi.org/10.1007/s00300-008-0471-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Polar%20Biology&amp;title=Age%20and%20Growth%20of%20Ocellated%20Icefish,%20Chionodraco%20rastrospinosus%20DeWitt%20and%20Hureau,%201979,%20From%20the%20South%20Shetland%20Islands&amp;author=M.%20La%20Mesa&amp;author=J.%20Ashford&amp;volume=31&amp;publication_year=2008&amp;pages=1333-1342&amp;doi=10.1007/s00300-008-0471-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0072">
<cite>

La Mesa, M.
, Donato F., Riginella E., and Mazzoldi C.. 2018. “Life History Traits of a Poorly Known Pelagic Fish, 
<em>Aethotaxis mitopteryx</em>
 (Perciformes, Notothenioidei) From the Weddell Sea.” Polar Biology
41: 1777–1788. 10.1007/s00300-018-2318-1.</cite> [<a href="https://doi.org/10.1007/s00300-018-2318-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Polar%20Biology&amp;title=Life%20History%20Traits%20of%20a%20Poorly%20Known%20Pelagic%20Fish,%20Aethotaxis%20mitopteryx%20(Perciformes,%20Notothenioidei)%20From%20the%20Weddell%20Sea&amp;author=M.%20La%20Mesa&amp;author=F.%20Donato&amp;author=E.%20Riginella&amp;author=C.%20Mazzoldi&amp;volume=41&amp;publication_year=2018&amp;pages=1777-1788&amp;doi=10.1007/s00300-018-2318-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0073">
<cite>

La Mesa, M.
, and Eastman J. T.. 2012. “Antarctic Silverfish: Life Strategies of a Key Species in the High‐Antarctic Ecosystem.” Fish and Fisheries
13: 241–266. 10.1111/j.1467-2979.2011.00427.x.</cite> [<a href="https://doi.org/10.1111/j.1467-2979.2011.00427.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Fish%20and%20Fisheries&amp;title=Antarctic%20Silverfish:%20Life%20Strategies%20of%20a%20Key%20Species%20in%20the%20High%E2%80%90Antarctic%20Ecosystem&amp;author=M.%20La%20Mesa&amp;author=J.%20T.%20Eastman&amp;volume=13&amp;publication_year=2012&amp;pages=241-266&amp;doi=10.1111/j.1467-2979.2011.00427.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0074">
<cite>

La Mesa, M.
, Llompart F., Riginella E., and Eastman J. T.. 2021. “Parental Care and Reproductive Strategies in Notothenioid Fishes.” Fish and Fisheries
22: 356–376. 10.1111/faf.12523.</cite> [<a href="https://doi.org/10.1111/faf.12523" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Fish%20and%20Fisheries&amp;title=Parental%20Care%20and%20Reproductive%20Strategies%20in%20Notothenioid%20Fishes&amp;author=M.%20La%20Mesa&amp;author=F.%20Llompart&amp;author=E.%20Riginella&amp;author=J.%20T.%20Eastman&amp;volume=22&amp;publication_year=2021&amp;pages=356-376&amp;doi=10.1111/faf.12523&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0075">
<cite>

La Mesa, M.
, and Vacchi M.. 2001. “Age and Growth of High Antarctic Notothenioid Fish.” Antarctic Science
13: 227–235. 10.1017/S0954102001000335.</cite> [<a href="https://doi.org/10.1017/S0954102001000335" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antarctic%20Science&amp;title=Age%20and%20Growth%20of%20High%20Antarctic%20Notothenioid%20Fish&amp;author=M.%20La%20Mesa&amp;author=M.%20Vacchi&amp;volume=13&amp;publication_year=2001&amp;pages=227-235&amp;doi=10.1017/S0954102001000335&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0076">
<cite>

Lecointre, G.
, Améziane N., Boisselier M.‐C., et al. 2013. “Is the Species Flock Concept Operational? The Antarctic Shelf Case.” PLoS One
8: e68787. 10.1371/journal.pone.0068787.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0068787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3732269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23936311/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Is%20the%20Species%20Flock%20Concept%20Operational?%20The%20Antarctic%20Shelf%20Case&amp;author=G.%20Lecointre&amp;author=N.%20Am%C3%A9ziane&amp;author=M.%E2%80%90C.%20Boisselier&amp;volume=8&amp;publication_year=2013&amp;pages=e68787&amp;pmid=23936311&amp;doi=10.1371/journal.pone.0068787&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0077">
<cite>

Lee, S. J.
, Kim J.‐H., Jo E., et al. 2021. “Chromosomal Assembly of the Antarctic Toothfish (<em>Dissostichus mawsoni</em>) Genome Using Third‐Generation DNA Sequencing and Hi‐C Technology.” Zoological Research
42: 124–129. 10.24272/j.issn.2095-8137.2020.264.
</cite> [<a href="https://doi.org/10.24272/j.issn.2095-8137.2020.264" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7840457/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33258338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Zoological%20Research&amp;title=Chromosomal%20Assembly%20of%20the%20Antarctic%20Toothfish%20(Dissostichus%20mawsoni)%20Genome%20Using%20Third%E2%80%90Generation%20DNA%20Sequencing%20and%20Hi%E2%80%90C%20Technology&amp;author=S.%20J.%20Lee&amp;author=J.%E2%80%90H.%20Kim&amp;author=E.%20Jo&amp;volume=42&amp;publication_year=2021&amp;pages=124-129&amp;pmid=33258338&amp;doi=10.24272/j.issn.2095-8137.2020.264&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0078">
<cite>

Leitwein, M.
, Duranton M., Rougemont Q., Gagnaire P.‐A., and Bernatchez L.. 2020. “Using Haplotype Information for Conservation Genomics.” Trends in Ecology &amp; Evolution
35: 245–258. 10.1016/j.tree.2019.10.012.
</cite> [<a href="https://doi.org/10.1016/j.tree.2019.10.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31810774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Ecology%20&amp;%20Evolution&amp;title=Using%20Haplotype%20Information%20for%20Conservation%20Genomics&amp;author=M.%20Leitwein&amp;author=M.%20Duranton&amp;author=Q.%20Rougemont&amp;author=P.%E2%80%90A.%20Gagnaire&amp;author=L.%20Bernatchez&amp;volume=35&amp;publication_year=2020&amp;pages=245-258&amp;pmid=31810774&amp;doi=10.1016/j.tree.2019.10.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0079">
<cite>

Li, H.
, and Durbin R.. 2009. “Fast and Accurate Short Read Alignment With Burrows‐Wheeler Transform.” Bioinformatics
25: 1754–1760. 10.1093/bioinformatics/btp324.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2705234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Fast%20and%20Accurate%20Short%20Read%20Alignment%20With%20Burrows%E2%80%90Wheeler%20Transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;volume=25&amp;publication_year=2009&amp;pages=1754-1760&amp;pmid=19451168&amp;doi=10.1093/bioinformatics/btp324&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0080">
<cite>

Loeb, V. J.
, Kellermann A. K., Koubbi P., North A. W., and White M. G.. 1993. “Antarctic Larval Fish Assemblages: A Review.” Bulletin of Marine Science
53: 416–449. <a href="https://www.scopus.com/inward/record.uri?eid=2-s2.0-0027837087&amp;partnerID=40&amp;md5=f27a2e971b6a3df760baf042ec5c05d1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.scopus.com/inward/record.uri?eid=2‐s2.0‐0027837087&amp;partnerID=40&amp;md5=f27a2e971b6a3df760baf042ec5c05d1</a>.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Bulletin%20of%20Marine%20Science&amp;title=Antarctic%20Larval%20Fish%20Assemblages:%20A%20Review&amp;author=V.%20J.%20Loeb&amp;author=A.%20K.%20Kellermann&amp;author=P.%20Koubbi&amp;author=A.%20W.%20North&amp;author=M.%20G.%20White&amp;volume=53&amp;publication_year=1993&amp;pages=416-449&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0081">
<cite>

Magoč, T.
, and Salzberg S. L.. 2011. “FLASH: Fast Length Adjustment of Short Reads to Improve Genome Assemblies.” Bioinformatics
27: 2957–2963. 10.1093/bioinformatics/btr507.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr507" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3198573/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21903629/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=FLASH:%20Fast%20Length%20Adjustment%20of%20Short%20Reads%20to%20Improve%20Genome%20Assemblies&amp;author=T.%20Mago%C4%8D&amp;author=S.%20L.%20Salzberg&amp;volume=27&amp;publication_year=2011&amp;pages=2957-2963&amp;pmid=21903629&amp;doi=10.1093/bioinformatics/btr507&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0082">
<cite>

Malinsky, M.
, Trucchi E., Lawson D. J., and Falush D.. 2018. “RADpainter and fineRADstructure: Population Inference From RADseq Data.” Molecular Biology and Evolution
35: 1284–1290. 10.1093/molbev/msy023.
</cite> [<a href="https://doi.org/10.1093/molbev/msy023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5913677/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29474601/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=RADpainter%20and%20fineRADstructure:%20Population%20Inference%20From%20RADseq%20Data&amp;author=M.%20Malinsky&amp;author=E.%20Trucchi&amp;author=D.%20J.%20Lawson&amp;author=D.%20Falush&amp;volume=35&amp;publication_year=2018&amp;pages=1284-1290&amp;pmid=29474601&amp;doi=10.1093/molbev/msy023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0083">
<cite>

Maschette, D.
, Wotherspoon S., Polanowski A., Deagle B., Welsford D., and Ziegler P.. 2023. “Circumpolar Sampling Reveals High Genetic Connectivity of Antarctic Toothfish Across Their Spatial Distribution.” Reviews in Fish Biology and Fisheries
33: 295–310. 10.1007/s11160-023-09756-9.</cite> [<a href="https://doi.org/10.1007/s11160-023-09756-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Reviews%20in%20Fish%20Biology%20and%20Fisheries&amp;title=Circumpolar%20Sampling%20Reveals%20High%20Genetic%20Connectivity%20of%20Antarctic%20Toothfish%20Across%20Their%20Spatial%20Distribution&amp;author=D.%20Maschette&amp;author=S.%20Wotherspoon&amp;author=A.%20Polanowski&amp;author=B.%20Deagle&amp;author=D.%20Welsford&amp;volume=33&amp;publication_year=2023&amp;pages=295-310&amp;doi=10.1007/s11160-023-09756-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0084">
<cite>

Matschiner, M.
, Colombo M., Damerau M., Ceballos S., Hanel R., and Salzburger W.. 2015. “The Adaptive Radiation of Notothenioid Fishes in the Waters of Antarctica.” In Extremophile Fishes: Ecology, Evolution, and Physiology of Teleosts in Extreme Environments, edited by Rüdiger T., Martin M., and Martin P., 35–57. Springer International Publishing. 10.1007/978-3-319-13362-1_3.</cite> [<a href="https://doi.org/10.1007/978-3-319-13362-1_3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Extremophile%20Fishes:%20Ecology,%20Evolution,%20and%20Physiology%20of%20Teleosts%20in%20Extreme%20Environments&amp;author=M.%20Matschiner&amp;author=M.%20Colombo&amp;author=M.%20Damerau&amp;author=S.%20Ceballos&amp;author=R.%20Hanel&amp;publication_year=2015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0085">
<cite>

Matschiner, M.
, Hanel R., and Salzburger W.. 2009. “Gene Flow by Larval Dispersal in the Antarctic Notothenioid Fish 
<em>Gobionotothen gibberifrons</em>

.” Molecular Ecology
18: 2574–2587. 10.1111/j.1365-294X.2009.04220.x.
</cite> [<a href="https://doi.org/10.1111/j.1365-294X.2009.04220.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19457182/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Gene%20Flow%20by%20Larval%20Dispersal%20in%20the%20Antarctic%20Notothenioid%20Fish%20Gobionotothen%20gibberifrons&amp;author=M.%20Matschiner&amp;author=R.%20Hanel&amp;author=W.%20Salzburger&amp;volume=18&amp;publication_year=2009&amp;pages=2574-2587&amp;pmid=19457182&amp;doi=10.1111/j.1365-294X.2009.04220.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0086">
<cite>

McKinney, G.
, McPhee M. V., Pascal C., Seeb J. E., and Seeb L. W.. 2020. “Network Analysis of Linkage Disequilibrium Reveals Genome Architecture in Chum Salmon.” G3: Genes, Genomes, Genetics
10: 1553–1561. 10.1534/g3.119.400972.
</cite> [<a href="https://doi.org/10.1534/g3.119.400972" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7202013/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32165371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes,%20Genomes,%20Genetics&amp;title=Network%20Analysis%20of%20Linkage%20Disequilibrium%20Reveals%20Genome%20Architecture%20in%20Chum%20Salmon&amp;author=G.%20McKinney&amp;author=M.%20V.%20McPhee&amp;author=C.%20Pascal&amp;author=J.%20E.%20Seeb&amp;author=L.%20W.%20Seeb&amp;volume=10&amp;publication_year=2020&amp;pages=1553-1561&amp;pmid=32165371&amp;doi=10.1534/g3.119.400972&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0087">
<cite>

Nacif, C. L.
, Kratochwil C. F., Kautt A. F., et al. 2023. “Molecular Parallelism in the Evolution of a Master Sex‐Determining Role for the Anti‐Mullerian Hormone Receptor 2 Gene (<em>amhr2</em>) in Midas Cichlids.” Molecular Ecology
32: 1398–1410. 10.1111/mec.16466.
</cite> [<a href="https://doi.org/10.1111/mec.16466" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35403749/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Molecular%20Parallelism%20in%20the%20Evolution%20of%20a%20Master%20Sex%E2%80%90Determining%20Role%20for%20the%20Anti%E2%80%90Mullerian%20Hormone%20Receptor%202%20Gene%20(amhr2)%20in%20Midas%20Cichlids&amp;author=C.%20L.%20Nacif&amp;author=C.%20F.%20Kratochwil&amp;author=A.%20F.%20Kautt&amp;volume=32&amp;publication_year=2023&amp;pages=1398-1410&amp;pmid=35403749&amp;doi=10.1111/mec.16466&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0088">
<cite>

Near, T. J.
, Kendrick B. J., Detrich W. H., and Jones C. D.. 2007. “Confirmation of Neutral Buoyancy in <em>Aethotaxis mitopteryx</em> DeWitt (Notothenioidei: Nototheniidae).” Polar Biology
30: 443–447. 10.1007/s00300-006-0201-y.</cite> [<a href="https://doi.org/10.1007/s00300-006-0201-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Polar%20Biology&amp;title=Confirmation%20of%20Neutral%20Buoyancy%20in%20Aethotaxis%20mitopteryx%20DeWitt%20(Notothenioidei:%20Nototheniidae)&amp;author=T.%20J.%20Near&amp;author=B.%20J.%20Kendrick&amp;author=W.%20H.%20Detrich&amp;author=C.%20D.%20Jones&amp;volume=30&amp;publication_year=2007&amp;pages=443-447&amp;doi=10.1007/s00300-006-0201-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0089">
<cite>

Near, T. J.
, MacGuigan D. J., Parker E., Struthers C. D., Jones C. D., and Dornburg A.. 2018. “Phylogenetic Analysis of Antarctic Notothenioids Illuminates the Utility of RADseq for Resolving Cenozoic Adaptive Radiations.” Molecular Phylogenetics and Evolution
129: 268–279. 10.1016/j.ympev.2018.09.001.
</cite> [<a href="https://doi.org/10.1016/j.ympev.2018.09.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30195039/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Phylogenetics%20and%20Evolution&amp;title=Phylogenetic%20Analysis%20of%20Antarctic%20Notothenioids%20Illuminates%20the%20Utility%20of%20RADseq%20for%20Resolving%20Cenozoic%20Adaptive%20Radiations&amp;author=T.%20J.%20Near&amp;author=D.%20J.%20MacGuigan&amp;author=E.%20Parker&amp;author=C.%20D.%20Struthers&amp;author=C.%20D.%20Jones&amp;volume=129&amp;publication_year=2018&amp;pages=268-279&amp;pmid=30195039&amp;doi=10.1016/j.ympev.2018.09.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0090">
<cite>

Near, T. J.
, Russo S. E., Jones C. D., and DeVries A. L.. 2003. “Ontogenetic Shift in Buoyancy and Habitat in the Antarctic Toothfish, 
<em>Dissostichus mawsoni</em>
 (Perciformes: Nototheniidae).” Polar Biology
26: 124–128. 10.1007/s00300-002-0459-7.</cite> [<a href="https://doi.org/10.1007/s00300-002-0459-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Polar%20Biology&amp;title=Ontogenetic%20Shift%20in%20Buoyancy%20and%20Habitat%20in%20the%20Antarctic%20Toothfish,%20Dissostichus%20mawsoni%20(Perciformes:%20Nototheniidae)&amp;author=T.%20J.%20Near&amp;author=S.%20E.%20Russo&amp;author=C.%20D.%20Jones&amp;author=A.%20L.%20DeVries&amp;volume=26&amp;publication_year=2003&amp;pages=124-128&amp;doi=10.1007/s00300-002-0459-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0091">
<cite>

Nei, M.

1987. Molecular Evolutionary Genetics. Columbia University Press. 10.7312/nei-92038.</cite> [<a href="https://doi.org/10.7312/nei-92038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Molecular%20Evolutionary%20Genetics&amp;author=M.%20Nei&amp;publication_year=1987&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0092">
<cite>

North, A. W.

2001. “Early Life History Strategies of Notothenioids at South Georgia.” Journal of Fish Biology
58: 496–505. 10.1111/j.1095-8649.2001.tb02268.x.</cite> [<a href="https://doi.org/10.1111/j.1095-8649.2001.tb02268.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Fish%20Biology&amp;title=Early%20Life%20History%20Strategies%20of%20Notothenioids%20at%20South%20Georgia&amp;author=A.%20W.%20North&amp;volume=58&amp;publication_year=2001&amp;pages=496-505&amp;doi=10.1111/j.1095-8649.2001.tb02268.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0093">
<cite>

Pan, Q.
, Feron R., Jouanno E., et al. 2021. “The Rise and Fall of the Ancient Northern Pike Master Sex‐Determining Gene.” eLife
10: e62858. 10.7554/eLife.62858.
</cite> [<a href="https://doi.org/10.7554/eLife.62858" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7870143/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33506762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=The%20Rise%20and%20Fall%20of%20the%20Ancient%20Northern%20Pike%20Master%20Sex%E2%80%90Determining%20Gene&amp;author=Q.%20Pan&amp;author=R.%20Feron&amp;author=E.%20Jouanno&amp;volume=10&amp;publication_year=2021&amp;pages=e62858&amp;pmid=33506762&amp;doi=10.7554/eLife.62858&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0094">
<cite>

Pan, Q.
, Kay T., Depincé A., et al. 2021. “Evolution of Master Sex Determiners: TGF‐β Signalling Pathways at Regulatory Crossroads.” Philosophical Transactions of the Royal Society, B: Biological Sciences
376: 20200091. 10.1098/rstb.2020.0091.</cite> [<a href="https://doi.org/10.1098/rstb.2020.0091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8273507/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34247498/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Philosophical%20Transactions%20of%20the%20Royal%20Society,%20B:%20Biological%20Sciences&amp;title=Evolution%20of%20Master%20Sex%20Determiners:%20TGF%E2%80%90%CE%B2%20Signalling%20Pathways%20at%20Regulatory%20Crossroads&amp;author=Q.%20Pan&amp;author=T.%20Kay&amp;author=A.%20Depinc%C3%A9&amp;volume=376&amp;publication_year=2021&amp;pages=20200091&amp;pmid=34247498&amp;doi=10.1098/rstb.2020.0091&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0095">
<cite>

Papetti, C.
, Pujolar J. M., Mezzavilla M., et al. 2012. “Population Genetic Structure and Gene Flow Patterns Between Populations of the Antarctic Icefish <em>Chionodraco rastrospinosus</em>
.” Journal of Biogeography
39: 1361–1372. 10.1111/j.1365-2699.2011.02682.x.</cite> [<a href="https://doi.org/10.1111/j.1365-2699.2011.02682.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Biogeography&amp;title=Population%20Genetic%20Structure%20and%20Gene%20Flow%20Patterns%20Between%20Populations%20of%20the%20Antarctic%20Icefish%20Chionodraco%20rastrospinosus&amp;author=C.%20Papetti&amp;author=J.%20M.%20Pujolar&amp;author=M.%20Mezzavilla&amp;volume=39&amp;publication_year=2012&amp;pages=1361-1372&amp;doi=10.1111/j.1365-2699.2011.02682.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0096">
<cite>

Papetti, C.
, Susana E., Patarnello T., and Zane L.. 2009. “Spatial and Temporal Boundaries to Gene Flow Between <em>Chaenocephalus aceratus</em> Populations at South Orkney and South Shetlands.” Marine Ecology Progress Series
376: 269–281. <a href="http://www.jstor.org/stable/24873012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.jstor.org/stable/24873012</a>.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Marine%20Ecology%20Progress%20Series&amp;title=Spatial%20and%20Temporal%20Boundaries%20to%20Gene%20Flow%20Between%20Chaenocephalus%20aceratus%20Populations%20at%20South%20Orkney%20and%20South%20Shetlands&amp;author=C.%20Papetti&amp;author=E.%20Susana&amp;author=T.%20Patarnello&amp;author=L.%20Zane&amp;volume=376&amp;publication_year=2009&amp;pages=269-281&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0097">
<cite>

Pembleton, L. W.
, Cogan N. O. I., and Forster J. W.. 2013. “StAMPP: An R Package for Calculation of Genetic Differentiation and Structure of Mixed‐Ploidy Level Populations.” Molecular Ecology Resources
13: 946–952. 10.1111/1755-0998.12129.
</cite> [<a href="https://doi.org/10.1111/1755-0998.12129" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23738873/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology%20Resources&amp;title=StAMPP:%20An%20R%20Package%20for%20Calculation%20of%20Genetic%20Differentiation%20and%20Structure%20of%20Mixed%E2%80%90Ploidy%20Level%20Populations&amp;author=L.%20W.%20Pembleton&amp;author=N.%20O.%20I.%20Cogan&amp;author=J.%20W.%20Forster&amp;volume=13&amp;publication_year=2013&amp;pages=946-952&amp;pmid=23738873&amp;doi=10.1111/1755-0998.12129&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0098">
<cite>

Pisano, E.
, Ozouf‐Costaz C., and Prirodina V.. 1998. “Chromosome Diversification in Antarctic Fish (Notothenioidei).” In Fishes of Antarctica, 275–285. Springer Milan. 10.1007/978-88-470-2157-0_24.</cite> [<a href="https://doi.org/10.1007/978-88-470-2157-0_24" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Fishes%20of%20Antarctica&amp;author=E.%20Pisano&amp;author=C.%20Ozouf%E2%80%90Costaz&amp;author=V.%20Prirodina&amp;publication_year=1998&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0099">
<cite>

Prchalová, M.
, Žák J., Říha M., et al. 2022. “Sexual Size Dimorphism of Two Common European Percid Fish: Linkage With Spatial Distribution and Diet.” Hydrobiologia
849: 2009–2027. 10.1007/s10750-022-04844-6.</cite> [<a href="https://doi.org/10.1007/s10750-022-04844-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hydrobiologia&amp;title=Sexual%20Size%20Dimorphism%20of%20Two%20Common%20European%20Percid%20Fish:%20Linkage%20With%20Spatial%20Distribution%20and%20Diet&amp;author=M.%20Prchalov%C3%A1&amp;author=J.%20%C5%BD%C3%A1k&amp;author=M.%20%C5%98%C3%ADha&amp;volume=849&amp;publication_year=2022&amp;pages=2009-2027&amp;doi=10.1007/s10750-022-04844-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0100">
<cite>

Prugnolle, F.
, and de Meeus T.. 2002. “Inferring Sex‐Biased Dispersal From Population Genetic Tools: A Review.” Heredity
88: 161–165. 10.1038/sj.hdy.6800060.
</cite> [<a href="https://doi.org/10.1038/sj.hdy.6800060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11920116/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heredity&amp;title=Inferring%20Sex%E2%80%90Biased%20Dispersal%20From%20Population%20Genetic%20Tools:%20A%20Review&amp;author=F.%20Prugnolle&amp;author=T.%20de%20Meeus&amp;volume=88&amp;publication_year=2002&amp;pages=161-165&amp;pmid=11920116&amp;doi=10.1038/sj.hdy.6800060&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0101">
<cite>

Purcell, S.
, Neale B., Todd‐Brown K., et al. 2007. “PLINK: A Tool Set for Whole‐Genome Association and Population‐Based Linkage Analyses.” American Journal of Human Genetics
81: 559–575. 10.1086/519795.
</cite> [<a href="https://doi.org/10.1086/519795" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1950838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17701901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=PLINK:%20A%20Tool%20Set%20for%20Whole%E2%80%90Genome%20Association%20and%20Population%E2%80%90Based%20Linkage%20Analyses&amp;author=S.%20Purcell&amp;author=B.%20Neale&amp;author=K.%20Todd%E2%80%90Brown&amp;volume=81&amp;publication_year=2007&amp;pages=559-575&amp;pmid=17701901&amp;doi=10.1086/519795&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0102">
<cite>

Raphael, M. N.
, Maierhofer T. J., Fogt R. L., Hobbs W. R., and Handcock M. S.. 2025. “A Twenty‐First Century Structural Change in Antarctica's Sea Ice System.” Communications Earth &amp; Environment
6: 131. 10.1038/s43247-025-02107-5.</cite> [<a href="https://doi.org/10.1038/s43247-025-02107-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Communications%20Earth%20&amp;%20Environment&amp;title=A%20Twenty%E2%80%90First%20Century%20Structural%20Change%20in%20Antarctica's%20Sea%20Ice%20System&amp;author=M.%20N.%20Raphael&amp;author=T.%20J.%20Maierhofer&amp;author=R.%20L.%20Fogt&amp;author=W.%20R.%20Hobbs&amp;author=M.%20S.%20Handcock&amp;volume=6&amp;publication_year=2025&amp;pages=131&amp;doi=10.1038/s43247-025-02107-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0103">
<cite>

Rayamajhi, N.
, Rivera‐Colón A. G., Minhas B. F., Cheng C.‐H. C., and Catchen J. M.. 2024. “The Genome of the Cryopelagic Antarctic Bald Notothen, <em>Trematomus borchgrevinki</em>
.” G3 (Bethesda, Md.)
15: jkae267. 10.1093/g3journal/jkae267.</cite> [<a href="https://doi.org/10.1093/g3journal/jkae267" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11708224/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39549265/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3%20(Bethesda,%20Md.)&amp;title=The%20Genome%20of%20the%20Cryopelagic%20Antarctic%20Bald%20Notothen,%20Trematomus%20borchgrevinki&amp;author=N.%20Rayamajhi&amp;author=A.%20G.%20Rivera%E2%80%90Col%C3%B3n&amp;author=B.%20F.%20Minhas&amp;author=C.%E2%80%90H.%20C.%20Cheng&amp;author=J.%20M.%20Catchen&amp;volume=15&amp;publication_year=2024&amp;pages=jkae267&amp;pmid=39549265&amp;doi=10.1093/g3journal/jkae267&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0104">
<cite>

Riginella, E.
, Pineda‐Metz S. E. A., Gerdes D., et al. 2021. “Parental Care and Demography of a Spawning Population of the Channichthyid 
<em>Neopagetopsis ionah</em>
, Nybelin 1947 From the Weddell Sea.” Polar Biology
44: 1725–1735. 10.1007/s00300-021-02913-5.</cite> [<a href="https://doi.org/10.1007/s00300-021-02913-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Polar%20Biology&amp;title=Parental%20Care%20and%20Demography%20of%20a%20Spawning%20Population%20of%20the%20Channichthyid%20Neopagetopsis%20ionah%20,%20Nybelin%201947%20From%20the%20Weddell%20Sea&amp;author=E.%20Riginella&amp;author=S.%20E.%20A.%20Pineda%E2%80%90Metz&amp;author=D.%20Gerdes&amp;volume=44&amp;publication_year=2021&amp;pages=1725-1735&amp;doi=10.1007/s00300-021-02913-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0105">
<cite>

Ritchie, M. G.
, Hamill R. M., Graves J. A., Magurran A. E., Webb S. A., and Macías Garcia C.. 2007. “Sex and Differentiation: Population Genetic Divergence and Sexual Dimorphism in Mexican Goodeid Fish.” Journal of Evolutionary Biology
20: 2048–2055. 10.1111/j.1420-9101.2007.01357.x.
</cite> [<a href="https://doi.org/10.1111/j.1420-9101.2007.01357.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17714321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Evolutionary%20Biology&amp;title=Sex%20and%20Differentiation:%20Population%20Genetic%20Divergence%20and%20Sexual%20Dimorphism%20in%20Mexican%20Goodeid%20Fish&amp;author=M.%20G.%20Ritchie&amp;author=R.%20M.%20Hamill&amp;author=J.%20A.%20Graves&amp;author=A.%20E.%20Magurran&amp;author=S.%20A.%20Webb&amp;volume=20&amp;publication_year=2007&amp;pages=2048-2055&amp;pmid=17714321&amp;doi=10.1111/j.1420-9101.2007.01357.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0106">
<cite>

Rochette, N. C.
, Rivera‐Colón A. G., and Catchen J. M.. 2019. “Stacks 2: Analytical Methods for Paired‐End Sequencing Improve RADseq‐Based Population Genomics.” Molecular Ecology
28: 4737–4754. 10.1111/mec.15253.
</cite> [<a href="https://doi.org/10.1111/mec.15253" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31550391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Stacks%202:%20Analytical%20Methods%20for%20Paired%E2%80%90End%20Sequencing%20Improve%20RADseq%E2%80%90Based%20Population%20Genomics&amp;author=N.%20C.%20Rochette&amp;author=A.%20G.%20Rivera%E2%80%90Col%C3%B3n&amp;author=J.%20M.%20Catchen&amp;volume=28&amp;publication_year=2019&amp;pages=4737-4754&amp;pmid=31550391&amp;doi=10.1111/mec.15253&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0107">
<cite>

Ryan, S.
, Schröder M., Huhn O., and Timmermann R.. 2016. “On the Warm Inflow at the Eastern Boundary of the Weddell Gyre.” Deep‐Sea Research Part I: Oceanographic Research Papers
107: 70–81. 10.1016/j.dsr.2015.11.002.</cite> [<a href="https://doi.org/10.1016/j.dsr.2015.11.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Deep%E2%80%90Sea%20Research%20Part%20I:%20Oceanographic%20Research%20Papers&amp;title=On%20the%20Warm%20Inflow%20at%20the%20Eastern%20Boundary%20of%20the%20Weddell%20Gyre&amp;author=S.%20Ryan&amp;author=M.%20Schr%C3%B6der&amp;author=O.%20Huhn&amp;author=R.%20Timmermann&amp;volume=107&amp;publication_year=2016&amp;pages=70-81&amp;doi=10.1016/j.dsr.2015.11.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0108">
<cite>

Sandoval Laurrabaquio‐A, N.
, Islas‐Villanueva V., Adams D. H., Uribe‐Alcocer M., Alvarado‐Bremer J. R., and Díaz‐Jaimes P.. 2019. “Genetic Evidence for Regional Philopatry of the Bull Shark (
<em>Carcharhinus leucas</em>
), to Nursery Areas in Estuaries of the Gulf of Mexico and Western North Atlantic Ocean.” Fisheries Research
209: 67–74. 10.1016/j.fishres.2018.09.013.</cite> [<a href="https://doi.org/10.1016/j.fishres.2018.09.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Fisheries%20Research&amp;title=Genetic%20Evidence%20for%20Regional%20Philopatry%20of%20the%20Bull%20Shark%20(%20Carcharhinus%20leucas%20),%20to%20Nursery%20Areas%20in%20Estuaries%20of%20the%20Gulf%20of%20Mexico%20and%20Western%20North%20Atlantic%20Ocean&amp;author=N.%20Sandoval%20Laurrabaquio%E2%80%90A&amp;author=V.%20Islas%E2%80%90Villanueva&amp;author=D.%20H.%20Adams&amp;author=M.%20Uribe%E2%80%90Alcocer&amp;author=J.%20R.%20Alvarado%E2%80%90Bremer&amp;volume=209&amp;publication_year=2019&amp;pages=67-74&amp;doi=10.1016/j.fishres.2018.09.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0109">
<cite>

Schiavon, L.
, Ceballos S. G., Matschiner M., et al. 2024. “Limited Interspecific Gene Flow in the Evolutionary History of the Icefish Genus <em>Chionodraco</em>
.” ICES Journal of Marine Science
81: 676–686. 10.1093/icesjms/fsae019.</cite> [<a href="https://doi.org/10.1093/icesjms/fsae019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ICES%20Journal%20of%20Marine%20Science&amp;title=Limited%20Interspecific%20Gene%20Flow%20in%20the%20Evolutionary%20History%20of%20the%20Icefish%20Genus%20Chionodraco&amp;author=L.%20Schiavon&amp;author=S.%20G.%20Ceballos&amp;author=M.%20Matschiner&amp;volume=81&amp;publication_year=2024&amp;pages=676-686&amp;doi=10.1093/icesjms/fsae019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0110">
<cite>

Schiavon, L.
, Dulière V., La Mesa M., et al. 2021. “Species Distribution, Hybridization and Connectivity in the Genus <em>Chionodraco</em>: Unveiling Unknown Icefish Diversity in Antarctica.” Diversity and Distributions
27: ddi.13249. 10.1111/ddi.13249.</cite> [<a href="https://doi.org/10.1111/ddi.13249" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diversity%20and%20Distributions&amp;title=Species%20Distribution,%20Hybridization%20and%20Connectivity%20in%20the%20Genus%20Chionodraco:%20Unveiling%20Unknown%20Icefish%20Diversity%20in%20Antarctica&amp;author=L.%20Schiavon&amp;author=V.%20Duli%C3%A8re&amp;author=M.%20La%20Mesa&amp;volume=27&amp;publication_year=2021&amp;pages=ddi.13249&amp;doi=10.1111/ddi.13249&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0111">
<cite>

Schiavon, L.
, Negrisolo E., Battistotti A., et al. 2023. “Species Identification and Population Genetics of the Antarctic Fish Genera <em>Lepidonotothen</em> and <em>Nototheniops</em> (Perciformes, Notothenioidei).” Zoologica Scripta
52: 136–153. 10.1111/zsc.12580.</cite> [<a href="https://doi.org/10.1111/zsc.12580" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Zoologica%20Scripta&amp;title=Species%20Identification%20and%20Population%20Genetics%20of%20the%20Antarctic%20Fish%20Genera%20Lepidonotothen%20and%20Nototheniops%20(Perciformes,%20Notothenioidei)&amp;author=L.%20Schiavon&amp;author=E.%20Negrisolo&amp;author=A.%20Battistotti&amp;volume=52&amp;publication_year=2023&amp;pages=136-153&amp;doi=10.1111/zsc.12580&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0112">
<cite>

Simão, F. A.
, Waterhouse R. M., Ioannidis P., Kriventseva E. V., and Zdobnov E. M.. 2015. “BUSCO: Assessing Genome Assembly and Annotation Completeness With Single‐Copy Orthologs.” Bioinformatics
31: 3210–3212. 10.1093/bioinformatics/btv351.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btv351" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26059717/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=BUSCO:%20Assessing%20Genome%20Assembly%20and%20Annotation%20Completeness%20With%20Single%E2%80%90Copy%20Orthologs&amp;author=F.%20A.%20Sim%C3%A3o&amp;author=R.%20M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.%20V.%20Kriventseva&amp;author=E.%20M.%20Zdobnov&amp;volume=31&amp;publication_year=2015&amp;pages=3210-3212&amp;pmid=26059717&amp;doi=10.1093/bioinformatics/btv351&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0113">
<cite>

Sonsthagen, S. A.
, Jay C. V., Fischbach A. S., Sage G. K., and Talbot S. L.. 2012. “Spatial Genetic Structure and Asymmetrical Gene Flow Within the Pacific Walrus.” Journal of Mammalogy
93: 1512–1524. 10.1644/11-MAMM-A-344.1.</cite> [<a href="https://doi.org/10.1644/11-MAMM-A-344.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Mammalogy&amp;title=Spatial%20Genetic%20Structure%20and%20Asymmetrical%20Gene%20Flow%20Within%20the%20Pacific%20Walrus&amp;author=S.%20A.%20Sonsthagen&amp;author=C.%20V.%20Jay&amp;author=A.%20S.%20Fischbach&amp;author=G.%20K.%20Sage&amp;author=S.%20L.%20Talbot&amp;volume=93&amp;publication_year=2012&amp;pages=1512-1524&amp;doi=10.1644/11-MAMM-A-344.1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0114">
<cite>

Strass, V. H.
, Rohardt G., Kanzow T., Hoppema M., and Boebel O.. 2020. “Multidecadal Warming and Density Loss in the Deep Weddell Sea, Antarctica.” Journal of Climate
33: 9863–9881. 10.1175/JCLI-D-20-0271.1.</cite> [<a href="https://doi.org/10.1175/JCLI-D-20-0271.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Climate&amp;title=Multidecadal%20Warming%20and%20Density%20Loss%20in%20the%20Deep%20Weddell%20Sea,%20Antarctica&amp;author=V.%20H.%20Strass&amp;author=G.%20Rohardt&amp;author=T.%20Kanzow&amp;author=M.%20Hoppema&amp;author=O.%20Boebel&amp;volume=33&amp;publication_year=2020&amp;pages=9863-9881&amp;doi=10.1175/JCLI-D-20-0271.1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0115">
<cite>

Teschke, K.
, Brtnik P., Hain S., et al. 2021. “Planning Marine Protected Areas Under the CCAMLR Regime – The Case of the Weddell Sea (Antarctica).” Marine Policy
124: 104370. 10.1016/j.marpol.2020.104370.</cite> [<a href="https://doi.org/10.1016/j.marpol.2020.104370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Marine%20Policy&amp;title=Planning%20Marine%20Protected%20Areas%20Under%20the%20CCAMLR%20Regime%20%E2%80%93%20The%20Case%20of%20the%20Weddell%20Sea%20(Antarctica)&amp;author=K.%20Teschke&amp;author=P.%20Brtnik&amp;author=S.%20Hain&amp;volume=124&amp;publication_year=2021&amp;pages=104370&amp;doi=10.1016/j.marpol.2020.104370&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0116">
<cite>

Thompson, A. F.
, Stewart A. L., Spence P., and Heywood K. J.. 2018. “The Antarctic Slope Current in a Changing Climate.” Reviews of Geophysics
56: 741–770. 10.1029/2018RG000624.</cite> [<a href="https://doi.org/10.1029/2018RG000624" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Reviews%20of%20Geophysics&amp;title=The%20Antarctic%20Slope%20Current%20in%20a%20Changing%20Climate&amp;author=A.%20F.%20Thompson&amp;author=A.%20L.%20Stewart&amp;author=P.%20Spence&amp;author=K.%20J.%20Heywood&amp;volume=56&amp;publication_year=2018&amp;pages=741-770&amp;doi=10.1029/2018RG000624&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0117">
<cite>

Trathan, P. N.
, and Grant S. M.. 2020. “The South Orkney Islands Southern Shelf Marine Protected Area.” In Marine Protected Areas, 67–98. Elsevier. 10.1016/B978-0-08-102698-4.00004-6.</cite> [<a href="https://doi.org/10.1016/B978-0-08-102698-4.00004-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Marine%20Protected%20Areas&amp;author=P.%20N.%20Trathan&amp;author=S.%20M.%20Grant&amp;publication_year=2020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0118">
<cite>

Turner, J.
, Guarino M. V., Arnatt J., et al. 2020. “Recent Decrease of Summer Sea Ice in the Weddell Sea, Antarctica.” Geophysical Research Letters
47: e2020GL087127. 10.1029/2020GL087127.</cite> [<a href="https://doi.org/10.1029/2020GL087127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Geophysical%20Research%20Letters&amp;title=Recent%20Decrease%20of%20Summer%20Sea%20Ice%20in%20the%20Weddell%20Sea,%20Antarctica&amp;author=J.%20Turner&amp;author=M.%20V.%20Guarino&amp;author=J.%20Arnatt&amp;volume=47&amp;publication_year=2020&amp;pages=e2020GL087127&amp;doi=10.1029/2020GL087127&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0119">
<cite>

Vacchi, M.
, Pisano E., and Ghigliotti L., eds. 2017. The Antarctic Silverfish: A Keystone Species in a Changing Ecosystem. Vol. 3. Springer International Publishing. 10.1007/978-3-319-55893-6.</cite> [<a href="https://doi.org/10.1007/978-3-319-55893-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=The%20Antarctic%20Silverfish:%20A%20Keystone%20Species%20in%20a%20Changing%20Ecosystem&amp;publication_year=2017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0120">
<cite>

Van de Putte, A. P.
, Van Houdt J. K. J., Maes G. E., Hellemans B., Collins M. A., and Volckaert F. A. M.. 2012. “High Genetic Diversity and Connectivity in a Common Mesopelagic Fish of the Southern Ocean: The Myctophid <em>Electrona antarctica</em>
.” Deep Sea Research Part II: Topical Studies in Oceanography
59: 199–207. 10.1016/j.dsr2.2011.05.011.</cite> [<a href="https://doi.org/10.1016/j.dsr2.2011.05.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Deep%20Sea%20Research%20Part%20II:%20Topical%20Studies%20in%20Oceanography&amp;title=High%20Genetic%20Diversity%20and%20Connectivity%20in%20a%20Common%20Mesopelagic%20Fish%20of%20the%20Southern%20Ocean:%20The%20Myctophid%20Electrona%20antarctica&amp;author=A.%20P.%20Van%20de%20Putte&amp;author=J.%20K.%20J.%20Van%20Houdt&amp;author=G.%20E.%20Maes&amp;author=B.%20Hellemans&amp;author=M.%20A.%20Collins&amp;volume=59&amp;publication_year=2012&amp;pages=199-207&amp;doi=10.1016/j.dsr2.2011.05.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0121">
<cite>

Vaser, R.
, Sović I., Nagarajan N., and Šikić M.. 2017. “Fast and Accurate de Novo Genome Assembly From Long Uncorrected Reads.” Genome Research
27: 737–746. 10.1101/gr.214270.116.
</cite> [<a href="https://doi.org/10.1101/gr.214270.116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5411768/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28100585/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=Fast%20and%20Accurate%20de%20Novo%20Genome%20Assembly%20From%20Long%20Uncorrected%20Reads&amp;author=R.%20Vaser&amp;author=I.%20Sovi%C4%87&amp;author=N.%20Nagarajan&amp;author=M.%20%C5%A0iki%C4%87&amp;volume=27&amp;publication_year=2017&amp;pages=737-746&amp;pmid=28100585&amp;doi=10.1101/gr.214270.116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0122">
<cite>

Vihtakari, M.

2024. ggOceanMaps: Plot Data on Oceanographic Maps Using ggplot2_. R Package Version 2.2.0. <a href="https://cran.r-project.org/package=ggOceanMaps%3e" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://CRAN.R‐project.org/package=ggOceanMaps</a>.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=ggOceanMaps:%20Plot%20Data%20on%20Oceanographic%20Maps%20Using%20ggplot2_&amp;author=M.%20Vihtakari&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0123">
<cite>

Warren, R. L.
, Yang C., Vandervalk B. P., et al. 2015. “LINKS: Scalable, Alignment‐Free Scaffolding of Draft Genomes With Long Reads.” GigaScience
4: 35. 10.1186/s13742-015-0076-3.
</cite> [<a href="https://doi.org/10.1186/s13742-015-0076-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4524009/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26244089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GigaScience&amp;title=LINKS:%20Scalable,%20Alignment%E2%80%90Free%20Scaffolding%20of%20Draft%20Genomes%20With%20Long%20Reads&amp;author=R.%20L.%20Warren&amp;author=C.%20Yang&amp;author=B.%20P.%20Vandervalk&amp;volume=4&amp;publication_year=2015&amp;pages=35&amp;pmid=26244089&amp;doi=10.1186/s13742-015-0076-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0124">
<cite>

Weir, B. S.
, and Cockerham C. C.. 1984. “Estimating F‐Statistics for the Analysis of Population Structure.” Evolution
38: 1358–1370. 10.1111/j.1558-5646.1984.tb05657.x.
</cite> [<a href="https://doi.org/10.1111/j.1558-5646.1984.tb05657.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28563791/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=Estimating%20F%E2%80%90Statistics%20for%20the%20Analysis%20of%20Population%20Structure&amp;author=B.%20S.%20Weir&amp;author=C.%20C.%20Cockerham&amp;volume=38&amp;publication_year=1984&amp;pages=1358-1370&amp;pmid=28563791&amp;doi=10.1111/j.1558-5646.1984.tb05657.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0125">
<cite>

Wen, M.
, Pan Q., Jouanno E., et al. 2022. “An Ancient Truncated Duplication of the Anti‐Müllerian Hormone Receptor Type 2 Gene Is a Potential Conserved Master Sex Determinant in the Pangasiidae Catfish Family.” Molecular Ecology Resources
22: 2411–2428. 10.1111/1755-0998.13620.
</cite> [<a href="https://doi.org/10.1111/1755-0998.13620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9555307/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35429227/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology%20Resources&amp;title=An%20Ancient%20Truncated%20Duplication%20of%20the%20Anti%E2%80%90M%C3%BCllerian%20Hormone%20Receptor%20Type%202%20Gene%20Is%20a%20Potential%20Conserved%20Master%20Sex%20Determinant%20in%20the%20Pangasiidae%20Catfish%20Family&amp;author=M.%20Wen&amp;author=Q.%20Pan&amp;author=E.%20Jouanno&amp;volume=22&amp;publication_year=2022&amp;pages=2411-2428&amp;pmid=35429227&amp;doi=10.1111/1755-0998.13620&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0126">
<cite>

Yano, A.
, Guyomard R., Nicol B., et al. 2012. “An Immune‐Related Gene Evolved Into the Master Sex‐Determining Gene in Rainbow Trout, 
<em>Oncorhynchus mykiss</em>

.” Current Biology
22: 1423–1428. 10.1016/j.cub.2012.05.045.
</cite> [<a href="https://doi.org/10.1016/j.cub.2012.05.045" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22727696/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Biology&amp;title=An%20Immune%E2%80%90Related%20Gene%20Evolved%20Into%20the%20Master%20Sex%E2%80%90Determining%20Gene%20in%20Rainbow%20Trout,%20Oncorhynchus%20mykiss&amp;author=A.%20Yano&amp;author=R.%20Guyomard&amp;author=B.%20Nicol&amp;volume=22&amp;publication_year=2012&amp;pages=1423-1428&amp;pmid=22727696&amp;doi=10.1016/j.cub.2012.05.045&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371847-bib-0127">
<cite>

Zimmermann, C.

1997. “On the Demersal Fish Fauna of the Lazarev Sea (Antarctica): Composition and Community Structure.” In Antarctic Communities: Species, Structure and Survival, edited by Battaglia B., Valencia J., and Walton D. W. H., 26–32. Cambridge University Press.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Antarctic%20Communities:%20Species,%20Structure%20and%20Survival&amp;author=C.%20Zimmermann&amp;publication_year=1997&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
Data S1.
</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367274/bin/ECE3-15-e71847-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71847-s001.docx</a><sup> (7.5MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All the codes, datasets and accessory files needed to replicate the analyses are available at <a href="https://github.com/Papetti%E2%80%90Lab/Aethotaxis_RAD%E2%80%90Seq" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/Papetti‐Lab/Aethotaxis_RAD‐Seq</a>. Raw sequence reads are available under the NCBI BioProject Accession number: PRJNA1214794 (<a href="https://www.ncbi.nlm.nih.gov/sra/PRJNA1214794" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/sra/PRJNA1214794</a>).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Ecology and Evolution are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/ece3.71847"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ECE3-15-e71847.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.9 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12367274/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12367274/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367274%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367274/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12367274/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12367274/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40843031/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12367274/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40843031/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12367274/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12367274/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="fIdZeaBbbR8RivXfGqWycfu3RZcmQgnl5zFSaaRJs1sEKAjCDQTqluGtWVrlea5F">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
