
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Integrative molecular network analysis of genetic risk factors to infer biomarkers and therapeutic targets for rheumatoid arthritis - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A5778AF2305305A5770037D976F1.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="plosone">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370121/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS One">
<meta name="citation_title" content="Integrative molecular network analysis of genetic risk factors to infer biomarkers and therapeutic targets for rheumatoid arthritis">
<meta name="citation_author" content="Sakhaa Alsaedi">
<meta name="citation_author_institution" content="Computer Science, Division of Computer, Electrical and Mathematical Sciences and Engineering (CEMSE), King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia">
<meta name="citation_author_institution" content="Center of Excellence on Smart Health, King Abdullah University of Science and Technology (KAUST), Thuwal, Kingdom of Saudi Arabia">
<meta name="citation_author_institution" content="Center of Excellence for Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Kingdom of Saudi Arabia">
<meta name="citation_author_institution" content="College of Computer Science and Engineering (CCSE), Taibah University, Madinah, Saudi Arabia">
<meta name="citation_author" content="Katsuhiko Mineta">
<meta name="citation_author_institution" content="Research Organization for Nano &amp; Life Innovation, Waseda University, Shinjuku, Tokyo, Japan">
<meta name="citation_author_institution" content="Department of System Engineering, Graduate School of Science and Engineering, Shizuoka Institute of Science and Technology, Aoi, Shizuoka, Japan">
<meta name="citation_author_institution" content="Marine Open Innovation Institute (MaOI), Shimizu, Shizuoka, Japan">
<meta name="citation_author" content="Naoto Tamura">
<meta name="citation_author_institution" content="Department of Internal Medicine and Rheumatology, Juntendo University, Tokyo, Japan">
<meta name="citation_author" content="Xin Gao">
<meta name="citation_author_institution" content="Computer Science, Division of Computer, Electrical and Mathematical Sciences and Engineering (CEMSE), King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia">
<meta name="citation_author_institution" content="Center of Excellence on Smart Health, King Abdullah University of Science and Technology (KAUST), Thuwal, Kingdom of Saudi Arabia">
<meta name="citation_author_institution" content="Center of Excellence for Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Kingdom of Saudi Arabia">
<meta name="citation_author" content="Takashi Gojobori">
<meta name="citation_author_institution" content="Center of Excellence on Smart Health, King Abdullah University of Science and Technology (KAUST), Thuwal, Kingdom of Saudi Arabia">
<meta name="citation_author_institution" content="Center of Excellence for Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Kingdom of Saudi Arabia">
<meta name="citation_author_institution" content="Biological and Environmental Sciences and Engineering, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia">
<meta name="citation_author_institution" content="Marine Open Innovation Institute (MaOI), Shizuoka, Japan">
<meta name="citation_author_institution" content="Department of Life Sciences, National Cheng Kung University, Tainan, Taiwan">
<meta name="citation_author" content="Michihiro Ogasawara">
<meta name="citation_author_institution" content="Biological and Environmental Sciences and Engineering, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="20">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e0329101">
<meta name="citation_doi" content="10.1371/journal.pone.0329101">
<meta name="citation_pmid" content="40839671">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370121/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370121/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370121/pdf/pone.0329101.pdf">
<meta name="description" content="Understanding the interplay between genetic risk factors and molecular pathways in rheumatoid arthritis (RA) is essential for developing effective treatments. This study aims to utilize genetic risk factors of RA and identify their key pathways and ...">
<meta name="og:title" content="Integrative molecular network analysis of genetic risk factors to infer biomarkers and therapeutic targets for rheumatoid arthritis">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Understanding the interplay between genetic risk factors and molecular pathways in rheumatoid arthritis (RA) is essential for developing effective treatments. This study aims to utilize genetic risk factors of RA and identify their key pathways and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370121/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370121">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pone.0329101"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pone.0329101.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370121%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370121/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370121/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370121/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-plosone.png" alt="PLOS One logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS One" title="Link to PLOS One" shape="default" href="https://doi.org/10.1371/journal.pone.0329101" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS One</button></div>. 2025 Aug 21;20(8):e0329101. doi: <a href="https://doi.org/10.1371/journal.pone.0329101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pone.0329101</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20One%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20One%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20One%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Integrative molecular network analysis of genetic risk factors to infer biomarkers and therapeutic targets for rheumatoid arthritis</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alsaedi%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Sakhaa Alsaedi</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Sakhaa Alsaedi</span></h3>
<div class="p">
<sup>1</sup>Computer Science, Division of Computer, Electrical and Mathematical Sciences and Engineering (CEMSE), King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</div>
<div class="p">
<sup>2</sup>Center of Excellence on Smart Health, King Abdullah University of Science and Technology (KAUST), Thuwal, Kingdom of Saudi Arabia</div>
<div class="p">
<sup>3</sup>Center of Excellence for Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Kingdom of Saudi Arabia</div>
<div class="p">
<sup>4</sup>College of Computer Science and Engineering (CCSE), Taibah University, Madinah, Saudi Arabia</div>
<div>Data curation, Formal analysis, Methodology, Validation, Visualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Alsaedi%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sakhaa Alsaedi</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mineta%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Katsuhiko Mineta</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Katsuhiko Mineta</span></h3>
<div class="p">
<sup>5</sup>Research Organization for Nano &amp; Life Innovation, Waseda University, Shinjuku, Tokyo, Japan</div>
<div class="p">
<sup>6</sup>Department of System Engineering, Graduate School of Science and Engineering, Shizuoka Institute of Science and Technology, Aoi, Shizuoka, Japan</div>
<div class="p">
<sup>7</sup>Marine Open Innovation Institute (MaOI), Shimizu, Shizuoka, Japan</div>
<div>Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mineta%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Katsuhiko Mineta</span></a>
</div>
</div>
<sup>5,</sup><sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tamura%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Naoto Tamura</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Naoto Tamura</span></h3>
<div class="p">
<sup>8</sup>Department of Internal Medicine and Rheumatology, Juntendo University, Tokyo, Japan</div>
<div>Investigation, Validation</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tamura%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Naoto Tamura</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Xin Gao</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Xin Gao</span></h3>
<div class="p">
<sup>1</sup>Computer Science, Division of Computer, Electrical and Mathematical Sciences and Engineering (CEMSE), King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</div>
<div class="p">
<sup>2</sup>Center of Excellence on Smart Health, King Abdullah University of Science and Technology (KAUST), Thuwal, Kingdom of Saudi Arabia</div>
<div class="p">
<sup>3</sup>Center of Excellence for Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Kingdom of Saudi Arabia</div>
<div>Supervision</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gao%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xin Gao</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gojobori%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Takashi Gojobori</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Takashi Gojobori</span></h3>
<div class="p">
<sup>2</sup>Center of Excellence on Smart Health, King Abdullah University of Science and Technology (KAUST), Thuwal, Kingdom of Saudi Arabia</div>
<div class="p">
<sup>3</sup>Center of Excellence for Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Kingdom of Saudi Arabia</div>
<div class="p">
<sup>9</sup>Biological and Environmental Sciences and Engineering, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</div>
<div class="p">
<sup>10</sup>Marine Open Innovation Institute (MaOI), Shizuoka, Japan</div>
<div class="p">
<sup>11</sup>Department of Life Sciences, National Cheng Kung University, Tainan, Taiwan</div>
<div>Supervision, Validation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gojobori%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Takashi Gojobori</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>9,</sup><sup>10,</sup><sup>11,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ogasawara%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Michihiro Ogasawara</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Michihiro Ogasawara</span></h3>
<div class="p">
<sup>9</sup>Biological and Environmental Sciences and Engineering, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</div>
<div>Investigation, Validation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ogasawara%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Michihiro Ogasawara</span></a>
</div>
</div>
<sup>9</sup>
</div>
<div class="cg p">Editor: <span class="name western">Cheorl-Ho Kim</span><sup>12</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>Computer Science, Division of Computer, Electrical and Mathematical Sciences and Engineering (CEMSE), King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</div>
<div id="aff002">
<sup>2</sup>Center of Excellence on Smart Health, King Abdullah University of Science and Technology (KAUST), Thuwal, Kingdom of Saudi Arabia</div>
<div id="aff003">
<sup>3</sup>Center of Excellence for Generative AI, King Abdullah University of Science and Technology (KAUST), Thuwal, Kingdom of Saudi Arabia</div>
<div id="aff004">
<sup>4</sup>College of Computer Science and Engineering (CCSE), Taibah University, Madinah, Saudi Arabia</div>
<div id="aff005">
<sup>5</sup>Research Organization for Nano &amp; Life Innovation, Waseda University, Shinjuku, Tokyo, Japan</div>
<div id="aff006">
<sup>6</sup>Department of System Engineering, Graduate School of Science and Engineering, Shizuoka Institute of Science and Technology, Aoi, Shizuoka, Japan</div>
<div id="aff007">
<sup>7</sup>Marine Open Innovation Institute (MaOI), Shimizu, Shizuoka, Japan</div>
<div id="aff008">
<sup>8</sup>Department of Internal Medicine and Rheumatology, Juntendo University, Tokyo, Japan</div>
<div id="aff009">
<sup>9</sup>Biological and Environmental Sciences and Engineering, King Abdullah University of Science and Technology (KAUST), Thuwal, Saudi Arabia</div>
<div id="aff010">
<sup>10</sup>Marine Open Innovation Institute (MaOI), Shizuoka, Japan</div>
<div id="aff011">
<sup>11</sup>Department of Life Sciences, National Cheng Kung University, Tainan, Taiwan</div>
<div id="edit1">
<sup>12</sup>Sungkyunkwan University - Suwon Campus: Sungkyunkwan University - Natural Sciences Campus, KOREA, REPUBLIC OF</div>
<div class="author-notes p">
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>takashi.gojobori@kaust.edu.sa</span></p>
</div>
<div class="fn" id="coi001"><p><strong>Competing Interests: </strong>The authors have declared that no competing interests exist.</p></div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Sakhaa Alsaedi</span></strong>: <span class="role">Data curation, Formal analysis, Methodology, Validation, Visualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Katsuhiko Mineta</span></strong>: <span class="role">Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Naoto Tamura</span></strong>: <span class="role">Investigation, Validation</span>
</div>
<div>
<strong class="contrib"><span class="name western">Xin Gao</span></strong>: <span class="role">Supervision</span>
</div>
<div>
<strong class="contrib"><span class="name western">Takashi Gojobori</span></strong>: <span class="role">Supervision, Validation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Michihiro Ogasawara</span></strong>: <span class="role">Investigation, Validation, Writing – review &amp; editing</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Cheorl-Ho Kim</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Sep 21; Accepted 2025 Jul 11; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Alsaedi et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370121  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40839671/" class="usa-link">40839671</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<section id="sec044"><h3 class="pmc_sec_title">Background</h3>
<p>Understanding the interplay between genetic risk factors and molecular pathways in rheumatoid arthritis (RA) is essential for developing effective treatments. This study aims to utilize genetic risk factors of RA and identify their key pathways and potential therapeutic targets through an integrated multi-omics approach.</p></section><section id="sec045"><h3 class="pmc_sec_title">Methods</h3>
<p>We developed a computational pipeline to construct a knowledge graph that combines genetic risk factor molecular networks with multi-omics enrichment analysis to estimate potential therapeutic target for RA. Genetic risk factors were extracted from the literature, curated, and annotated. Molecular interaction networks were constructed based on these genetic risk factors and their neighboring proteins. Enrichment analysis was performed to identify significantly impacted biological processes and pathways. Multi-omics knowledge graph was used to prioritize candidates potential therapeutic target for RA.</p></section><section id="sec046"><h3 class="pmc_sec_title">Results</h3>
<p>Our analysis identified 35 significant genes associated with RA as potential therapeutic targets and biomarkers, categorized into three pathways: Cytokine Regulation and Production, Hematopoietic or Lymphoid Organ Development, and Myeloid Cell Differentiation. Among these, 25 genes were classified as risk genes, while 10 were neighboring genes. We identified nine novel risk proteins (RELA, ETS1, NFATC1, BATF, LCK, PIK3R1, PRKCB, RASGRP1,and FYN) as potential therapeutic targets for RA and they significantly contribute in the disease pathogenesis.</p></section><section id="sec047"><h3 class="pmc_sec_title">Conclusion</h3>
<p>This study provides a comprehensive integrative molecular network and knowledge graph analysis of genetic risk factors in RA, offering a solid framework for integrating multi-omics data in RA research. These findings may contribute to more accurate clinical decision-making and the development of targeted treatment regimens. Additionally, this study highlights the importance of inferring hidden relationships across networks based on disease associations and functional similarities, further enhancing our understanding of RA pathogenesis.</p></section></section><section id="sec001"><h2 class="pmc_sec_title">1 Introduction</h2>
<p>Rheumatoid arthritis (RA) is a chronic, systemic autoimmune disease characterized by inflammation of the synovial joints, leading to progressive joint destruction, pain, and disability [<a href="#pone.0329101.ref001" class="usa-link" aria-describedby="pone.0329101.ref001">1</a>, <a href="#pone.0329101.ref002" class="usa-link" aria-describedby="pone.0329101.ref002">2</a>]. It is also associated with conditions such as metabolic diseases [<a href="#pone.0329101.ref003" class="usa-link" aria-describedby="pone.0329101.ref003">3</a>, <a href="#pone.0329101.ref004" class="usa-link" aria-describedby="pone.0329101.ref004">4</a>], neurological disorders [<a href="#pone.0329101.ref005" class="usa-link" aria-describedby="pone.0329101.ref005">5</a>–<a href="#pone.0329101.ref007" class="usa-link" aria-describedby="pone.0329101.ref007">7</a>], and infectious diseases [<a href="#pone.0329101.ref008" class="usa-link" aria-describedby="pone.0329101.ref008">8</a>], indicating a complex interplay between the immune system and overall health [<a href="#pone.0329101.ref009" class="usa-link" aria-describedby="pone.0329101.ref009">9</a>]. Despite significant research advancements, the precise etiology and pathogenesis of RA remain incompletely understood. It is widely accepted that genetic predisposition, environmental triggers, and immune dysregulation collectively drive RA pathogenesis [<a href="#pone.0329101.ref010" class="usa-link" aria-describedby="pone.0329101.ref010">10</a>–<a href="#pone.0329101.ref015" class="usa-link" aria-describedby="pone.0329101.ref015">15</a>]. Among these factors, genetic susceptibility plays a critical role in both disease onset and progression [<a href="#pone.0329101.ref016" class="usa-link" aria-describedby="pone.0329101.ref016">16</a>, <a href="#pone.0329101.ref017" class="usa-link" aria-describedby="pone.0329101.ref017">17</a>]. There is a growing body of evidence suggesting that understanding genetic risk factors is essential for gaining deeper insights into the complexity of RA and how these genetic associations contribute to other diseases.</p>
<p>Advances in genomic technologies have facilitated the identification of numerous genetic risk factors associated with RA and the characterization of their molecular mechanisms in disease development, contributing to better disease management [<a href="#pone.0329101.ref017" class="usa-link" aria-describedby="pone.0329101.ref017">17</a>].</p>
<p>In genetic variant association studies, several large-scale meta-analyses of Genome-Wide Association Studies (GWAS) data have expanded our understanding of RA heritability by uncovering novel single nucleotide polymorphisms (SNPs) in immune-related genes that confer an increased risk for RA [<a href="#pone.0329101.ref018" class="usa-link" aria-describedby="pone.0329101.ref018">18</a>]. For instance, a multi-ancestry GWAS meta-analysis identified 34 novel genetic markers influencing RA risk, emphasizing the importance of diverse populations in genetic research [<a href="#pone.0329101.ref019" class="usa-link" aria-describedby="pone.0329101.ref019">19</a>]. Additionally, a cross-trait GWAS analysis revealed shared genetic loci between RA and other autoimmune diseases, suggesting common pathways in autoimmune disease pathogenesis [<a href="#pone.0329101.ref020" class="usa-link" aria-describedby="pone.0329101.ref020">20</a>].</p>
<p>Although GWAS studies have successfully identified genetic variants associated with RA, their functional implications in disease risk and complications remain largely unclear [<a href="#pone.0329101.ref021" class="usa-link" aria-describedby="pone.0329101.ref021">21</a>]. Therefore, understanding how these variants influence biological pathways, immune responses, and disease mechanisms is crucial for the development of targeted therapies and precision medicine strategies [<a href="#pone.0329101.ref022" class="usa-link" aria-describedby="pone.0329101.ref022">22</a>]. Identifying and characterizing the biological pathways and molecular mechanisms influenced by these genetic risk factors are essential for developing targeted therapies and improving disease management [<a href="#pone.0329101.ref017" class="usa-link" aria-describedby="pone.0329101.ref017">17</a>]. This research direction focuses on identifying molecular mechanisms and predictive biomarkers while exploring potential therapeutic drugs through integrative multi-omics analyses. For example, Lu Xiao <em>et al</em>. (2022) applied integrative biomarker identification methods to provide insights into RA [<a href="#pone.0329101.ref023" class="usa-link" aria-describedby="pone.0329101.ref023">23</a>], while Caifang Shen <em>et al</em>. (2024) utilized bioinformatics approaches to identify significant biomarkers for RA diagnosis [<a href="#pone.0329101.ref024" class="usa-link" aria-describedby="pone.0329101.ref024">24</a>].</p>
<p>Despite advances in elucidating RA’s molecular pathways and identifying therapeutic targets [<a href="#pone.0329101.ref017" class="usa-link" aria-describedby="pone.0329101.ref017">17</a>, <a href="#pone.0329101.ref025" class="usa-link" aria-describedby="pone.0329101.ref025">25</a>, <a href="#pone.0329101.ref026" class="usa-link" aria-describedby="pone.0329101.ref026">26</a>], significant limitations remain. Research often isolates systems like the cytokine network, overlooking the interplay between biological pathways [<a href="#pone.0329101.ref027" class="usa-link" aria-describedby="pone.0329101.ref027">27</a>, <a href="#pone.0329101.ref028" class="usa-link" aria-describedby="pone.0329101.ref028">28</a>]. Studies on genetic risk factors frequently lack integration with pathway analyses, limiting understanding of genetic variations’ influence on RA [<a href="#pone.0329101.ref029" class="usa-link" aria-describedby="pone.0329101.ref029">29</a>–<a href="#pone.0329101.ref031" class="usa-link" aria-describedby="pone.0329101.ref031">31</a>]. While gene expression studies identify immune response pathways, the absence of comprehensive genetic factor analysis restricts insights into how risk genes contribute to RA pathogenesis [<a href="#pone.0329101.ref010" class="usa-link" aria-describedby="pone.0329101.ref010">10</a>, <a href="#pone.0329101.ref032" class="usa-link" aria-describedby="pone.0329101.ref032">32</a>]. Comprehensive network analysis is needed to understand these risk variants’ functional implications [<a href="#pone.0329101.ref033" class="usa-link" aria-describedby="pone.0329101.ref033">33</a>], identify the biological pathways they influence, and develop targeted therapies for improved disease management [<a href="#pone.0329101.ref010" class="usa-link" aria-describedby="pone.0329101.ref010">10</a>, <a href="#pone.0329101.ref034" class="usa-link" aria-describedby="pone.0329101.ref034">34</a>].</p>
<p>This study employs an integrated approach combining genetic risk factor analysis with multi-omics enrichment and network analysis to construct a comprehensive knowledge graph of genetic risk factors in RA to identify key pathways and potential therapeutic targets in RA. We construct molecular networks based on identified genetic risk factors and their neighboring proteins to uncover critical molecular interactions. Additionally, we perform a comprehensive enrichment analysis to highlight the most significantly impacted biological processes and pathways. Our objectives are threefold: (1) to identify core risk genes and pathways significantly associated with the complexity of RA; (2) to identify novel potential therapeutic targets for RA; and (3) to provide a foundation for future research aimed at validating these findings through experimental studies. By achieving these objectives, we aim to advance the understanding of RA pathogenesis and contribute to the development of targeted therapeutic strategies for improved patient outcomes [<a href="#pone.0329101.ref033" class="usa-link" aria-describedby="pone.0329101.ref033">33</a>].</p></section><section id="sec002"><h2 class="pmc_sec_title">2 Materials and methods</h2>
<p>We developed a computational pipeline to analyze genetic risk factors across multiple omics levels and construct a multi-omics knowledge graph to estimate potential therapeutic target scores for RA using integrative molecular network analysis. This pipeline integrates various database APIs and bioinformatics packages across five key stages: (1) data curation of RA genetic risk factors, (2) comprehensive enrichment analysis based on molecular function, biological process similarity, tissue and cell-type specificity, molecular system similarity, and disease association, (3) construction of risk factor knowledge graphs, (4) protein prioritization and therapeutic target scoring, and (5) in silico validation of predicted therapeutic targets for RA (<a href="#pone.0329101.g001" class="usa-link">Fig 1</a>).</p>
<figure class="fig xbox font-sm" id="pone.0329101.g001"><h3 class="obj_head">Fig 1. Overview of implemented computational workflow for analysis of genetic risk factors of RA.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370121_pone.0329101.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d00/12370121/e19871552828/pone.0329101.g001.jpg" loading="lazy" height="187" width="730" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329101.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Schema of the computational molecular pipeline for analysis of genetic risk factors associated with RA. The implemented pipeline consists of five stages and phases: Phase 1:retrieving and annotating a list of risk genetic factors, Phase 2: enrichment analysis of the curated dataset of genetic risk factors, Phase 3: constructing molecular network interactions and graphs for the identified genetic risk factors, Phase 4: predicting protein therapeutic targets for RA, and Phase 5: in silico validation of novel RA therapeutic targets.</p></figcaption></figure><section id="sec003"><h3 class="pmc_sec_title">2.1 Data curation of genetic risk factors for RA</h3>
<p>In the data curation, which is the first step in our computational workflow, the phase employed for systematically extracting and annotating genetic risk factors associated with RA from various omics levels.</p>
<section id="sec004"><h4 class="pmc_sec_title">2.1.1 Text-mining-based data sources for RA.</h4>
<p>Instead of direct access to raw GWAS datasets, which require extensive computational resources and ethical considerations, this study employed text mining to systematically extract validated RA-associated genetic variants from peer-reviewed GWAS literature in PubMed. It did not use UK Biobank summary statistics or other GWAS databases directly but compiled genetic risk factors for RA from published studies. This approach enables automatic integration of reported risk variants and genes while ensuring data transparency and reliability.</p></section><section id="sec005"><h4 class="pmc_sec_title">2.1.2 Data retrieval and extraction.</h4>
<p>In the first phase of our data preparation, we utilized PubTator [<a href="#pone.0329101.ref035" class="usa-link" aria-describedby="pone.0329101.ref035">35</a>], a web-based text mining tool, to systematically extract genetic risk factors for RA across different omics levels. PubTator facilitated a comprehensive search of the published literature in English, focusing on relevant research articles and reviews. PubTator was selected for its natural language processing capabilities and PubMed integration, enabling structured extraction of gene-disease associations. Unlike BioBERT or LitVar, it provides broad coverage with minimal redundancy by integrating multiple ontologies. The search strategy leveraged controlled vocabulary and Boolean operators to identify studies investigating genetic susceptibility to RA, including queries such as "rheumatoid arthritis AND genetic risk factors" and "host genetic variant AND rheumatoid arthritis." Our search parameters were restricted to peer-reviewed publications indexed in PubMed and published between January 1, 2018, and December 28, 2023. Preprints were excluded due to their lack of peer review. At the end of this phase, we extracted unfiltered genetic risk factors (SNPs, genes, or proteins) of RA, which were saved in a list for the next phase of data curation and annotation.</p></section><section id="sec006"><h4 class="pmc_sec_title">2.1.3 Data integration and annotation.</h4>
<p>As part of the first phase, we applied quality control and filtering to the genetic risk factors by removing duplicates and selecting GWAS studies with sample sizes of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0329101.e001"><math id="M1" display="inline" overflow="linebreak"><mrow><mo>≥</mo></mrow></math></span> 500 individuals from Asian or European populations, with significant odds ratios above 0.7. Following quality control and manual validation, we implemented a bioinformatics annotation workflow integrating more than four biomedical and genetic databases [<a href="#pone.0329101.ref036" class="usa-link" aria-describedby="pone.0329101.ref036">36</a>–<a href="#pone.0329101.ref040" class="usa-link" aria-describedby="pone.0329101.ref040">40</a>] to annotate 25 genetic features describing genetic risk factors. These features include genetic variant ID, chromosomal loci, associated gene, odds ratio, related phenotypes and diseases, and references. The workflow utilized the myVariants package [<a href="#pone.0329101.ref037" class="usa-link" aria-describedby="pone.0329101.ref037">37</a>] for genetic variant annotation due to its integration of dbSNP [<a href="#pone.0329101.ref036" class="usa-link" aria-describedby="pone.0329101.ref036">36</a>], ClinVar [<a href="#pone.0329101.ref041" class="usa-link" aria-describedby="pone.0329101.ref041">41</a>], and Ensembl [<a href="#pone.0329101.ref039" class="usa-link" aria-describedby="pone.0329101.ref039">39</a>]. Its real-time API queries minimize computational overhead and ensure access to the latest variant information. Each variant associated with RA is termed a risk polymorphism, while its corresponding gene and protein are referred to as a risk gene and risk protein, respectively.</p></section><section id="sec007"><h4 class="pmc_sec_title">2.1.4 Potential data biases and mitigation strategies.</h4>
<p>Despite robust data curation, biases may arise from text-mining limitations in detecting genetic risk factors that are less explicitly described or weakly highlighted in the literature. While PubTator efficiently extracts genetic variants, its reliance on published annotations may overlook less-documented genetic risk factors. Additionally, population-specific biases in GWAS data can limit the generalizability of findings. To address these biases, we cross-referenced extracted genetic risk factors with curated databases such as ClinVar, DisGeNET, and the GWAS Catalog for comprehensive validation. We also included multi-population GWAS studies to reduce population bias; however, most retrieved data were from European cohorts, emphasizing the need for broader representation in future research.</p></section></section><section id="sec008"><h3 class="pmc_sec_title">2.2 Enrichment analysis of genetic risk factors for RA</h3>
<p>In the second stage, we performed a comprehensive enrichment analysis to characterize the biological significance of RA genetic risk factors, including functional annotation, tissue-cell specificity, and disease association.</p>
<section id="sec009"><h4 class="pmc_sec_title">2.2.1 Process of functional enrichment analysis.</h4>
<p>To characterize the functional role of RA risk genes, we performed gene set enrichment analysis using the Gene Ontology (GO) classification system. This approach identifies molecular functions and biological pathways enriched in the risk gene set. GOATools package [<a href="#pone.0329101.ref040" class="usa-link" aria-describedby="pone.0329101.ref040">40</a>] was selected to apply the analysis for its computational efficiency, flexibility, and rigorous statistical framework, implementing Fisher’s exact test with multiple testing correction (Benjamini-Hochberg FDR adjustment) for accurate significance estimation. This step provides biological interpretation and insights into the molecular functions and biological processes contributing to the complexity of RA.</p></section><section id="sec010"><h4 class="pmc_sec_title">2.2.2 Process of tissue-cell type enrichment analysis.</h4>
<p>We used the WebCSEA bioinformatics method to analyze the tissue and cell-type specificity of 196 risk genes [<a href="#pone.0329101.ref042" class="usa-link" aria-describedby="pone.0329101.ref042">42</a>]. A graphical gene-set enrichment package was employed to investigate the ontologies of tissues and compartments associated with these risk genes, using an FDR cutoff of 0.05 [<a href="#pone.0329101.ref040" class="usa-link" aria-describedby="pone.0329101.ref040">40</a>]. The WebCSEA tool conducted tissue and cell-type specificity enrichment analysis on the risk gene list using a curated dataset comprising 111 scRNA-seq panels and 1,355 TCs from 61 human tissues across 11 organ systems [<a href="#pone.0329101.ref042" class="usa-link" aria-describedby="pone.0329101.ref042">42</a>]. The FDR threshold of 0.05 was chosen to maintain a balance between statistical stringency and biological relevance. A lower threshold (e.g., 0.01) would eliminate potentially meaningful tissue-specific associations, whereas a higher threshold would increase the risk of false positives. This threshold is consistent with standard practices in functional enrichment analyses, where a moderate FDR cutoff is necessary to avoid excessive filtering of tissue-specific signals while maintaining statistical rigor [<a href="#pone.0329101.ref043" class="usa-link" aria-describedby="pone.0329101.ref043">43</a>, <a href="#pone.0329101.ref044" class="usa-link" aria-describedby="pone.0329101.ref044">44</a>].</p></section><section id="sec011"><h4 class="pmc_sec_title">2.2.3 Process of diseases enrichment analysis.</h4>
<p>To map genetic risk factors related to RA and other associated diseases, we employed a disease similarity mapping approach to identify shared risk genes and their neighboring genes across related diseases. The associations within disease-gene networks were analyzed to uncover common biological pathways, utilizing biomedical databases such as ClinVar [<a href="#pone.0329101.ref041" class="usa-link" aria-describedby="pone.0329101.ref041">41</a>], Human Phenotype Ontology (HPO) [<a href="#pone.0329101.ref045" class="usa-link" aria-describedby="pone.0329101.ref045">45</a>], DISEASES, the Alliance Disease Database, and OMIM [<a href="#pone.0329101.ref046" class="usa-link" aria-describedby="pone.0329101.ref046">46</a>]. Furthermore, we integrated our curated gene-disease dataset of RA genetic risk factors with biomarkers and therapeutic targets from DisGeNET [<a href="#pone.0329101.ref047" class="usa-link" aria-describedby="pone.0329101.ref047">47</a>] and GeneCard [<a href="#pone.0329101.ref038" class="usa-link" aria-describedby="pone.0329101.ref038">38</a>] to identify potential therapeutic drugs for RA.</p></section></section><section id="sec012"><h3 class="pmc_sec_title">2.3 Construction of molecular networks and knowledge graphs for RA genetic risk factors</h3>
<p>To systematically explore the molecular landscape of RA, we constructed protein-protein interaction (PPI) networks and a multi-layered knowledge graph, integrating diverse biological data sources to uncover functional associations and potential therapeutic targets.</p>
<section id="sec013"><h4 class="pmc_sec_title">2.3.1 Construction of protein-protein interaction networks.</h4>
<p>In the third stage, we constructed PPI networks for genetic risk factors using the STRING 11.5 database [<a href="#pone.0329101.ref048" class="usa-link" aria-describedby="pone.0329101.ref048">48</a>], accessed on February 20, 2023, with a confidence threshold of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0329101.e002"><math id="M2" display="inline" overflow="linebreak"><mrow><mo>≥</mo><mn>0.9</mn></mrow></math></span> . We calculated combined protein interaction confidence scores by summing all STRING confidence values and applying Min-Max normalization. Only interactions with confidence scores <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0329101.e003"><math id="M3" display="inline" overflow="linebreak"><mrow><mo>≥</mo><mn>0.9</mn></mrow></math></span> were retained for further analysis. To enhance network visualization, we limited the number of interactions displayed in the network outputs, prioritizing top interactions for each risk protein based on the highest combined confidence score. We then used Cytoscape [<a href="#pone.0329101.ref049" class="usa-link" aria-describedby="pone.0329101.ref049">49</a>] to integrate the data and generate PPI networks, grouping unconnected networks based on the similarity of their molecular functions. Finally, we clustered the constructed networks based on the results of previous enrichment analyses, considering disease associations, common tissues, shared components, and molecular functions.</p></section><section id="sec014"><h4 class="pmc_sec_title">2.3.2 Construction of knowledge graphs.</h4>
<p>Next, we constructed a multi-layered knowledge graph based on the largest PPI network of genetic risk factors to mitigate graph sparsity. It integrates data from multiple biological sources, including the Proteomic Drug Atlas, GWAS Catalog, GeneAnalytics, Human Gene Atlas, DisGeNET, STRING, KEGG, Reactome Pathways Database [<a href="#pone.0329101.ref047" class="usa-link" aria-describedby="pone.0329101.ref047">47</a>, <a href="#pone.0329101.ref048" class="usa-link" aria-describedby="pone.0329101.ref048">48</a>, <a href="#pone.0329101.ref050" class="usa-link" aria-describedby="pone.0329101.ref050">50</a>, <a href="#pone.0329101.ref051" class="usa-link" aria-describedby="pone.0329101.ref051">51</a>], and the Kinase Library, enabling the identification of proteins functionally linked to RA and related diseases. The knowledge graph comprises nodes representing genes, proteins, diseases, and pathways. Edges define relationships based on co-expression patterns, transcriptional regulation, protein-protein interactions, and shared disease phenotypes. We applied network-based clustering to identify highly interconnected subnetworks that reveal disease mechanisms and therapeutic targets. The graph was further refined using semantic integration techniques to align heterogeneous data sources and infer missing associations, enhancing predictive analyses of RA-related molecular interactions.</p></section></section><section id="sec015"><h3 class="pmc_sec_title">2.4 Calculation of inference score for potential therapeutic targets</h3>
<p>In the fourth stage, we developed a scoring framework to quantify the relevance of each protein as a therapeutic target for RA. This framework integrates molecular network analysis and knowledge graph modeling to enhance prediction accuracy.</p>
<section id="sec016"><h4 class="pmc_sec_title">2.4.1 Calculation of enriched scores.</h4>
<p>Four enriched scores were estimated to calculate the final inference score, determining how likely a protein is to be a therapeutic target based on comparative and integrative analysis from previous steps. These scores are defined as follows: <em>S</em><sub><em>d</em></sub> represents the <em>Disease Mapping Score</em>, reflecting RA-associated protein-disease connections from curated sources. <em>S</em><sub><em>m</em></sub> is the <em>Molecular Interaction Score</em>, derived from PPI and pathway analysis. <em>S</em><sub><em>b</em></sub> corresponds to the <em>Biomedical Knowledge Score</em>, integrating transcriptomic and proteomic annotations. <em>S</em><sub><em>k</em></sub> is the <em>Knowledge Graph (KG) Score</em>, quantifying the strength of inferred relationships within the constructed knowledge graph. Each score component was calculated as follows:</p>
<table class="disp-formula p" id="pone.0329101.e004"><tr>
<td class="formula"><math id="M4" display="block" overflow="linebreak"><mrow><mrow><msub><mi>S</mi><mi>d</mi></msub><mo>=</mo><mfrac><mrow><mo stretchy="false">|</mo><msub><mi>D</mi><mi>p</mi></msub><mo stretchy="false">|</mo></mrow><mrow><mo stretchy="false">|</mo><mi>D</mi><mo stretchy="false">|</mo></mrow></mfrac></mrow></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where |<em>D</em><sub><em>p</em></sub>| represents the number of studies reporting the protein’s association with RA, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0329101.e005"><math id="M5" display="inline" overflow="linebreak"><mrow><mo stretchy="false">|</mo><mi>D</mi><mo stretchy="false">|</mo></mrow></math></span> is the total number of studies in the curated dataset.</p>
<table class="disp-formula p" id="pone.0329101.e006"><tr>
<td class="formula"><math id="M6" display="block" overflow="linebreak"><mrow><mrow><msub><mi>S</mi><mi>m</mi></msub><mo>=</mo><mi>α</mi><msub><mi>C</mi><mi>d</mi></msub><mo>+</mo><mi>β</mi><msub><mi>C</mi><mi>b</mi></msub><mo>+</mo><mi>γ</mi><msub><mi>C</mi><mi>c</mi></msub></mrow></mrow></math></td>
<td class="label">(2)</td>
</tr></table>
<p>where <em>C</em><sub><em>d</em></sub>, <em>C</em><sub><em>b</em></sub>, and <em>C</em><sub><em>c</em></sub> denote degree centrality, betweenness centrality, and closeness centrality, respectively, within the RA-related PPI network, and <em>α</em>, <em>β</em>, and <em>γ</em> are scaling factors.</p>
<table class="disp-formula p" id="pone.0329101.e007"><tr>
<td class="formula"><math id="M7" display="block" overflow="linebreak"><mrow><mrow><msub><mi>S</mi><mi>b</mi></msub><mo>=</mo><mfrac><mrow><mo stretchy="false">|</mo><msub><mi>E</mi><mi>p</mi></msub><mo stretchy="false">|</mo><mo>−</mo><msub><mi>E</mi><mrow><mo>min</mo></mrow></msub></mrow><mrow><msub><mi>E</mi><mrow><mo>max</mo></mrow></msub><mo>−</mo><msub><mi>E</mi><mrow><mo>min</mo></mrow></msub></mrow></mfrac></mrow></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<p>where |<em>E</em><sub><em>p</em></sub>| represents the expression level of the protein in RA patient datasets, normalized using min-max scaling.</p>
<table class="disp-formula p" id="pone.0329101.e008"><tr>
<td class="formula"><math id="M8" display="block" overflow="linebreak"><mrow><mrow><msub><mi>S</mi><mi>k</mi></msub><mo>=</mo><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><msub><mi>w</mi><mi>i</mi></msub><msub><mi>R</mi><mi>i</mi></msub></mrow></mrow></math></td>
<td class="label">(4)</td>
</tr></table>
<p>where <em>R</em><sub><em>i</em></sub> represents the relationship strength between the protein and an RA-related entity in the knowledge graph, and <em>w</em><sub><em>i</em></sub> is the assigned importance weight of the relationship.</p></section><section id="sec017"><h4 class="pmc_sec_title">2.4.2 Inference score calculation.</h4>
<p>The inference score (<em>I</em><sub><em>p</em></sub>) is computed by integrating four independent scores, each representing a different layer of molecular and disease-related evidence, including interactions, associations, and knowledge graphs. This score ranges from 0 to 1, with higher values indicating stronger evidence of RA association.</p>
<table class="disp-formula p" id="pone.0329101.e009"><tr>
<td class="formula"><math id="M9" display="block" overflow="linebreak"><mrow><mrow><msub><mi>I</mi><mi>p</mi></msub><mo>=</mo><msub><mi>w</mi><mi>d</mi></msub><msub><mi>S</mi><mi>d</mi></msub><mo>+</mo><msub><mi>w</mi><mi>m</mi></msub><msub><mi>S</mi><mi>m</mi></msub><mo>+</mo><msub><mi>w</mi><mi>b</mi></msub><msub><mi>S</mi><mi>b</mi></msub><mo>+</mo><msub><mi>w</mi><mi>k</mi></msub><msub><mi>S</mi><mi>k</mi></msub></mrow></mrow></math></td>
<td class="label">(5)</td>
</tr></table>
<p>Weight coefficients were selected based on empirical validation, balancing contributions from disease associations (30%), molecular interactions (30%), biomedical knowledge (20%), and knowledge graphs (20%). This distribution ensures comprehensive integration of multi-source evidence without overemphasizing any single data layer:</p>
<table class="disp-formula p" id="pone.0329101.e010"><tr>
<td class="formula"><math id="M10" display="block" overflow="linebreak"><mrow><mrow><msub><mi>w</mi><mi>d</mi></msub><mo>=</mo><mn>0.3</mn><mo>,</mo><mspace width="1em"></mspace><msub><mi>w</mi><mi>m</mi></msub><mo>=</mo><mn>0.3</mn><mo>,</mo><mspace width="1em"></mspace><msub><mi>w</mi><mi>b</mi></msub><mo>=</mo><mn>0.2</mn><mo>,</mo><mspace width="1em"></mspace><msub><mi>w</mi><mi>k</mi></msub><mo>=</mo><mn>0.2</mn></mrow></mrow></math></td>
<td class="label">(6)</td>
</tr></table>
<p>Each score component was normalized using min-max scaling:</p>
<table class="disp-formula p" id="pone.0329101.e011"><tr>
<td class="formula"><math id="M11" display="block" overflow="linebreak"><mrow><mrow><msup><mi>S</mi><mi>′</mi></msup><mo>=</mo><mfrac><mrow><mi>S</mi><mo>−</mo><msub><mi>S</mi><mrow><mo>min</mo></mrow></msub></mrow><mrow><msub><mi>S</mi><mrow><mo>max</mo></mrow></msub><mo>−</mo><msub><mi>S</mi><mrow><mo>min</mo></mrow></msub></mrow></mfrac></mrow></mrow></math></td>
<td class="label">(7)</td>
</tr></table>
<p>To predict potential therapeutic targets, we ranked proteins based on their (<em>I</em><sub><em>p</em></sub>) and classified them according to biological function, regulatory role, and molecular interactions. Proteins with <em>I</em><sub><em>p</em></sub>&gt;0.85 were identified as high-confidence targets, supported by multiple layers of evidence linking them to RA pathology. Additionally, network-based enrichment analysis was applied to assess their involvement in RA-related pathways.</p></section></section><section id="sec018"><h3 class="pmc_sec_title">2.5 In silico validation of predicted therapeutic therapeutic targets</h3>
<p>To validate the predicted therapeutic targets, we conducted in-silico analysis using data from the Human Protein Atlas (HPA), GTEx, and single-cell RNA sequencing (scRNA-seq) datasets. GTEx provides gene expression data across various tissues, while HPA offers protein-level validation through immunohistochemistry. scRNA-seq datasets add single-cell resolution, allowing us to examine expression patterns in immune cells relevant to RA. We prioritized proteins with high expression in relevant tissues as potential therapeutic candidates. To further confirm their relevance, we consulted RA specialists and cross-referenced findings with biomarkers and therapeutic targets from DrugBank [<a href="#pone.0329101.ref052" class="usa-link" aria-describedby="pone.0329101.ref052">52</a>] and medical literature. This in-silico validation approach enhances confidence in the identified therapeutic targets and their potential therapeutic applications.</p></section></section><section id="sec019"><h2 class="pmc_sec_title">3 Results</h2>
<section id="sec020"><h3 class="pmc_sec_title">3.1 Overview of genetic risk factors in RA</h3>
<p>The retrieved list of genetic risk factors for RA includes 279 SNPs associated with the condition. However, only 60% of the reviewed articles provided odds ratios for the reported significant risk variants. These SNPs were annotated and integrated with public biomedical and genetic databases to create a dataset comprising 196 genes, 158 of which are protein-coding. The genetic profiles in this curated dataset provide descriptive information on RA-associated risk polymorphisms. A shortlist of these genetic risk factors is presented in <a href="#pone.0329101.t001" class="usa-link">Table 1</a>. The full curated dataset is publicly accessible in the supplementary materials (1–3). Summary statistics and processed results can be provided upon reasonable request.</p>
<section class="tw xbox font-sm" id="pone.0329101.t001"><h4 class="obj_head">Table 1. Top 30 genetic risk factors related to rheumatoid arthritis.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Risk variants</th>
<th align="left" rowspan="1" colspan="1">Chromosome</th>
<th align="left" rowspan="1" colspan="1">Risk gene</th>
<th align="left" rowspan="1" colspan="1">OR</th>
<th align="left" rowspan="1" colspan="1">PubMed ID (PMID)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">rs2476601</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">PTPN22</td>
<td align="left" rowspan="1" colspan="1">1.81</td>
<td align="left" rowspan="1" colspan="1">15208781, 20453842</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs9557321</td>
<td align="left" rowspan="1" colspan="1">13</td>
<td align="left" rowspan="1" colspan="1">CLYBL</td>
<td align="left" rowspan="1" colspan="1">1.73</td>
<td align="left" rowspan="1" colspan="1">24532677</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs113066392</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">GTF2IRD1-NCF1</td>
<td align="left" rowspan="1" colspan="1">1.43</td>
<td align="left" rowspan="1" colspan="1">33310728, 28135245, 27272985</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs138193887</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">CUL5</td>
<td align="left" rowspan="1" colspan="1">1.21</td>
<td align="left" rowspan="1" colspan="1">33310728, 35088123, 22446963</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs3753389</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">CD244</td>
<td align="left" rowspan="1" colspan="1">1.30</td>
<td align="left" rowspan="1" colspan="1">18794858</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs909685</td>
<td align="left" rowspan="1" colspan="1">22</td>
<td align="left" rowspan="1" colspan="1">SYNGR1</td>
<td align="left" rowspan="1" colspan="1">1.14</td>
<td align="left" rowspan="1" colspan="1">23143596, 33310728</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs11933540</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">RBPJ</td>
<td align="left" rowspan="1" colspan="1">1.15</td>
<td align="left" rowspan="1" colspan="1">24390342, 23143596, 20453842</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs16903108</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">PVT1</td>
<td align="left" rowspan="1" colspan="1">1.15</td>
<td align="left" rowspan="1" colspan="1">35088123, 23143596</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs911760</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">PLGRKT</td>
<td align="left" rowspan="1" colspan="1">1.15</td>
<td align="left" rowspan="1" colspan="1">35088123</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs940825</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">AGR3-AHR</td>
<td align="left" rowspan="1" colspan="1">1.13</td>
<td align="left" rowspan="1" colspan="1">35088123</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs2867461</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">ANXA3</td>
<td align="left" rowspan="1" colspan="1">1.13</td>
<td align="left" rowspan="1" colspan="1">22446963</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs11574914</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">CCL21</td>
<td align="left" rowspan="1" colspan="1">1.12</td>
<td align="left" rowspan="1" colspan="1">35088123, 23143596, 22446963, 20453842</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs13031237</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">REL</td>
<td align="left" rowspan="1" colspan="1">1.12</td>
<td align="left" rowspan="1" colspan="1">19503088</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs6681482</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">TNFSF4</td>
<td align="left" rowspan="1" colspan="1">1.12</td>
<td align="left" rowspan="1" colspan="1">24532676, 35088123, 24390342</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs705700</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">CDK2</td>
<td align="left" rowspan="1" colspan="1">1.09</td>
<td align="left" rowspan="1" colspan="1">24390342, 33310728</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs34480360</td>
<td align="left" rowspan="1" colspan="1">16</td>
<td align="left" rowspan="1" colspan="1">ZNF689</td>
<td align="left" rowspan="1" colspan="1">1.09</td>
<td align="left" rowspan="1" colspan="1">33310728, 35088123</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs2300373</td>
<td align="left" rowspan="1" colspan="1">21</td>
<td align="left" rowspan="1" colspan="1">IFNGR2</td>
<td align="left" rowspan="1" colspan="1">1.09</td>
<td align="left" rowspan="1" colspan="1">35088123, 33310728, 23143596</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs5754104</td>
<td align="left" rowspan="1" colspan="1">22</td>
<td align="left" rowspan="1" colspan="1">UBE2L3-YDJC</td>
<td align="left" rowspan="1" colspan="1">1.09</td>
<td align="left" rowspan="1" colspan="1">23143596, 35088123, 33310728</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs1427749</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">SCAF11</td>
<td align="left" rowspan="1" colspan="1">1.08</td>
<td align="left" rowspan="1" colspan="1">35088123</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs9979383</td>
<td align="left" rowspan="1" colspan="1">21</td>
<td align="left" rowspan="1" colspan="1">RUNX1</td>
<td align="left" rowspan="1" colspan="1">1.08</td>
<td align="left" rowspan="1" colspan="1">23143596, 35088123, 33310728, 23143596</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs866205108</td>
<td align="left" rowspan="1" colspan="1">18</td>
<td align="left" rowspan="1" colspan="1">TNFRSF1A</td>
<td align="left" rowspan="1" colspan="1">1.10</td>
<td align="left" rowspan="1" colspan="1">35088123</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs1885013</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">RAD51B</td>
<td align="left" rowspan="1" colspan="1">1.11</td>
<td align="left" rowspan="1" colspan="1">24390342, 33310728</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs11375064</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">KSR1</td>
<td align="left" rowspan="1" colspan="1">0.93</td>
<td align="left" rowspan="1" colspan="1">23143596, 35088123, 24390342</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs11777380</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">CCN4</td>
<td align="left" rowspan="1" colspan="1">0.92</td>
<td align="left" rowspan="1" colspan="1">35088123</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs2671692</td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">WDFY4</td>
<td align="left" rowspan="1" colspan="1">0.92</td>
<td align="left" rowspan="1" colspan="1">23143596, 22446963,35088123</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs660442</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">BAD</td>
<td align="left" rowspan="1" colspan="1">0.90</td>
<td align="left" rowspan="1" colspan="1">33310728, 35088123</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs6032662</td>
<td align="left" rowspan="1" colspan="1">20</td>
<td align="left" rowspan="1" colspan="1">CD40</td>
<td align="left" rowspan="1" colspan="1">0.90</td>
<td align="left" rowspan="1" colspan="1">19898481,35088123, 22446963, 23143596</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs6495979</td>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">RasGRP1</td>
<td align="left" rowspan="1" colspan="1">0.88</td>
<td align="left" rowspan="1" colspan="1">23143596, 35088123, 24390342, 33310728</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs10790268</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">CXCR5</td>
<td align="left" rowspan="1" colspan="1">0.87</td>
<td align="left" rowspan="1" colspan="1">24390342</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">rs1571878</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">CCR6</td>
<td align="left" rowspan="1" colspan="1">0.86</td>
<td align="left" rowspan="1" colspan="1">23143596, 22446963, 20453842</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0329101.t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec021"><h3 class="pmc_sec_title">3.2 Enrichment analysis of genetic risk factors in RA</h3>
<p>The results of the multi-omics enrichment analysis of genetic risk factors in RA, including biological processes, molecular functions, cell-type specificity, and disease associations are summarized in the following sections.</p>
<section id="sec022"><h4 class="pmc_sec_title">3.2.1 Biological processes enrichment analysis.</h4>
<p>In the biological processes, three main pathways are significantly impacted by genetic risk factors: Cytokine Regulation and Production, Hematopoietic or Lymphoid Organ Development, and Myeloid Cell Differentiation, as shown in <a href="#pone.0329101.g002" class="usa-link">Fig 2A</a>. Risk genes such as IRAK1, UBASH3A, AIM2, and CD40 are involved in inflammatory responses and cytokine regulation and production. The Hematopoietic or Lymphoid Organ Development pathway is essential for the development and function of immune cells. Notable risk genes in this pathway include SLC8A3, FLT3, TGF-<em>β</em>1, RasGRP1, PLD4, and AIRE, which play important roles in healthy immune responses in RA. A subset of the hematopoietic pathway focuses specifically on the differentiation of myeloid cells, which are crucial for inflammation and tissue damage in RA. Key genes involved include SH2B3, TNFRSF1A, CDK2, MED1, PRKCB, KMT2B, and ZFP361, with some, such as TNFRSF1A and SH2B3, being potential therapeutic targets. Interestingly, seven risk genes (RUNX1, IRF8, BATF, PTPN11, PTPN22, PTPN2, and IFI-16) intersect all three main pathways. These risk genes appear to play a central role in influencing all three pathways, suggesting their potential importance in the complex interplay between inflammation and immune cell function in RA.</p>
<figure class="fig xbox font-sm" id="pone.0329101.g002"><h5 class="obj_head">Fig 2. Biological process and molecular function analysis of genetic risk factors of RA.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370121_pone.0329101.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d00/12370121/74bac084c978/pone.0329101.g002.jpg" loading="lazy" height="415" width="730" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329101.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Enrichment analysis of the molecular function and biological processes related to the genetic risk factors for rheumatoid arthritis. Fold enrichment was calculated by dividing the proportion of genes in the list of risk genes that are involved in a pathway by the equivalent percentage in the background. The FDR indicates how probable enrichment is due to chance alone; higher scores indicate the enrichment is biologically relevant. Fold enrichment reveals the extent to which genes significant to a specific pathway are overrepresented.</p></figcaption></figure></section><section id="sec023"><h4 class="pmc_sec_title">3.2.2 Molecular function enrichment analysis.</h4>
<p>In terms of molecular function, <a href="#pone.0329101.g002" class="usa-link">Fig 2B</a> demonstrates significant enrichment of non-membrane spanning protein tyrosine phosphatase activity with a <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0329101.e012"><math id="M12" display="inline" overflow="linebreak"><mrow><mo>−</mo><mi>log</mi><mo stretchy="false">(</mo><mtext>FDR</mtext><mo stretchy="false">)</mo><mo>=</mo><mn>3</mn></mrow></math></span>, emphasizing its role in immune system responses and the hematopoietic system. Four RA risk genes - BATF, PTPN11, PTPN2, and PTPN22 - play crucial roles in this activity, impacting cytokine production, lymphoid organ development, and myeloid cell differentiation. For example, PTPN2, known for its anti-inflammatory properties, emerges as a potential therapeutic target for both RA and pancreatic adenocarcinoma (PAAD) due to its ability to moderate inflammation and reduce tumor malignancy [<a href="#pone.0329101.ref053" class="usa-link" aria-describedby="pone.0329101.ref053">53</a>, <a href="#pone.0329101.ref054" class="usa-link" aria-describedby="pone.0329101.ref054">54</a>]. Similarly, PTPN22, involved in T cell signaling, serves as both a biomarker for disease predisposition and a potential therapeutic target for RA [<a href="#pone.0329101.ref053" class="usa-link" aria-describedby="pone.0329101.ref053">53</a>, <a href="#pone.0329101.ref055" class="usa-link" aria-describedby="pone.0329101.ref055">55</a>]. Furthermore, BATF regulates immune activation and T-cell memory, underlining its importance in immune responses [<a href="#pone.0329101.ref056" class="usa-link" aria-describedby="pone.0329101.ref056">56</a>, <a href="#pone.0329101.ref057" class="usa-link" aria-describedby="pone.0329101.ref057">57</a>].</p></section><section id="sec024"><h4 class="pmc_sec_title">3.2.3 Cell-type specific enrichment analysis.</h4>
<p><a href="#pone.0329101.g003" class="usa-link">Fig 3</a> shows the cell-type specific enrichment analysis that indicates the genetic risk factors for RA are highly enriched in the cardiovascular, respiratory, and digestive systems, highlighting these organ systems as crucial targets for improving RA treatment efficacy. The digestive system, particularly the intestinal tract, plays a significant role in RA pathogenesis [<a href="#pone.0329101.ref058" class="usa-link" aria-describedby="pone.0329101.ref058">58</a>]. Here, RA risk genes are expressed in fetal chorioamnionitis lung and heart tissue, and in the vasculature, affecting lymphoid cell and T cell memory functions, which are key in RA development. This suggests that targeting these tissues in early interventions may help prevent RA onset. Tissue-specific analysis reveals significant expression of genetic risk factors in immune-related tissues, especially lymphoid cells, underscoring the importance of immune cell regulation in RA treatment. General cell type analysis identifies diverse immune cells, including antigen-presenting cells such as dendritic cells, which are crucial for T cell activation and autoantibody production by B cells. The role of T cells is emphasized, with drugs such as Abatacept and Tacrolimus showing therapeutic potential in RA. Microglia, the central immunocompetent cells of the nervous system, are highly enriched in cytokine production and regulation. Typically associated with the brain and spinal cord, microglia are implicated in synovitis of peripheral joints, contributing to RA-related pain. This suggests that microglial activation, a secondary change due to peripheral joint inflammation, may be important for managing RA symptoms.</p>
<figure class="fig xbox font-sm" id="pone.0329101.g003"><h5 class="obj_head">Fig 3. Cell-type specific enrichment analysis of genetic risk factors of RA.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370121_pone.0329101.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d00/12370121/7c9158808304/pone.0329101.g003.jpg" loading="lazy" height="480" width="730" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329101.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>This figure presents the top enriched tissues and cell types based on cell-type-specific enrichment analysis of genetic risk factors for rheumatoid arthritis. The top left plot shows organ system enrichment, reflecting RA’s autoimmune nature. The top right plot highlights tissue-specific enrichment, emphasizing fetal and adult immune-related tissues. The bottom plot presents cell-type enrichment, identifying key immune cells (T cells, B cells, macrophages, and dendritic cells) as major contributors to RA pathogenesis. The bottom right plot illustrates immune lineage-specific enrichment, showing the relationships of RA-associated immune cells in adaptive and innate immunity</p></figcaption></figure></section><section id="sec025"><h4 class="pmc_sec_title">3.2.4 Disease enrichment analysis.</h4>
<p>We performed a disease enrichment analysis to identify the most enriched diseases associated with the risk genes of RA, as illustrated in <a href="#pone.0329101.g004" class="usa-link">Fig 4</a>. Our disease mapping indicates that Crohn’s disease is highly associated with RA (<a href="#pone.0329101.g004" class="usa-link">Fig 4A</a>). Seven shared risk genes (CD40, PTPN22, CD226, BATF, PRKCB, RasGRP1, and PTPN2) are involved in cytokine pathways linked to Crohn’s disease. Additionally, <a href="#pone.0329101.g004" class="usa-link">Fig 4B</a>–<a href="#pone.0329101.g004" class="usa-link">4C</a> links chronic myeloid leukemia to several RA risk genes (CD40, TGF-<em>β</em>1, SH2B3, and IL1RN). Phenotype analysis revealed significant associations between RA risk genes and lymphatic system abnormalities. Three overlapping genes (RUNX1, FLT3, PTPN11) connect to pathways involved in abnormal cell growth (hematopoiesis). RA risk genes also show strong associations with autoimmune comorbidities, including bronchiolitis, Sjögren’s syndrome, and psoriasis (<a href="#pone.0329101.g004" class="usa-link">Fig 4D</a>). For example, CD40, CRP, and IL1RN are involved in cytokine production pathways, which play a crucial role in regulating inflammatory responses. Dysregulation of these pathways contributes to excessive immune activation in both RA and bronchiolitis, a condition characterized by airway inflammation and immune cell infiltration. Similarly, SH2B3, CD40, and CRP are linked to Sjögren’s syndrome, diabetes mellitus, and celiac disease, reinforcing the genetic connections between RA and other autoimmune diseases. SH2B3 encodes a signaling adaptor that modulates T-cell activation and immune homeostasis, and its variants have been implicated in autoimmune susceptibility across multiple conditions. CD40, a co-stimulatory protein, is critical in B-cell activation and autoantibody production, contributing to glandular inflammation in Sjögren’s syndrome and immune dysregulation in diabetes and celiac disease. CRP, an inflammatory biomarker, is consistently elevated in these disorders, indicating a shared inflammatory cascade underlying their pathogenesis.</p>
<figure class="fig xbox font-sm" id="pone.0329101.g004"><h5 class="obj_head">Fig 4. Disease enrichment analysis of genetic risk factors of RA.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370121_pone.0329101.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d00/12370121/9e3449bbe023/pone.0329101.g004.jpg" loading="lazy" height="800" width="703" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329101.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Disease mapping and relations based on enrichment analysis of the identified genetic risk factors for rheumatoid arthritis. (A) DISEASES database: gene-disease associations from text-mining and experiments. (B) OMIM: inherited disorders and clinically relevant mutations. (C) HPO: phenotype-based links between RA risk genes and patient traits. (D) Alliance Disease: cross-species disease models and functional genomics</p></figcaption></figure></section></section><section id="sec026"><h3 class="pmc_sec_title">3.3 Construction and analysis of rheumatoid arthritis protein networks</h3>
<p>The puzzle plot in <a href="#pone.0329101.g005" class="usa-link">Fig 5</a> shows our 40 constructed PPI networks with 3462 protein interactions based on 158 risk proteins and their molecular function similarities. The green block contains thirteen networks related to the immune system. The yellow block consists of nine networks related to metabolic disorders. Six networks form the repair system, indicated by the blue block, and are involved in DNA and chromosomal processes. Lastly, Networks 13, 29, and 31, associated with the nervous system, relate to Alzheimer’s disease and psychiatric disorders. Most networks are associated with the immune and metabolic systems, but Networks 2 and 5 are related to the hematopoietic system.</p>
<figure class="fig xbox font-sm" id="pone.0329101.g005"><h4 class="obj_head">Fig 5. Molecular network clustering of constructed genetic risk proteins networks in RA.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370121_pone.0329101.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d00/12370121/c8188c185b79/pone.0329101.g005.jpg" loading="lazy" height="538" width="730" alt="Fig 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329101.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>This figure illustrates the molecular system clustering of 40 protein-protein interaction networks. Red nodes represent genetic risk proteins associated with RA, while blue nodes indicate neighboring proteins that interact with them. Each color-coded cluster corresponds to a major molecular system, categorized by shared biological functions and disease relevance. These clusters highlight molecular similarities within functional systems and emphasize the role of each network in RA pathogenesis.</p></figcaption></figure><p>Certain constructed networks have no association with the immune system. For example, Network 3 is related to the cytoskeletal system, which is affected in RA. Network 17 plays an essential role in the regulatory activities and biological processes of vascular metabolism. Moreover, Networks 23, 29, and 31 contribute to the neural system. Detailed summaries of the molecular functions and pathway analysis of the 39 remaining constructed networks are provided in the supplementary material 4. Additionally, Networks 1-5, 13, 14, and 17-16 share common pathways with the cardiovascular system and cardiovascular diseases. Networks 4, 6, 11, and 28 are related to the respiratory system.</p>
<section id="sec027"><h4 class="pmc_sec_title">3.3.1 Significant risk proteins analysis in network 1.</h4>
<p>Network 1, the largest network, Network 1 consists of 893 proteins, including 111 RA risk proteins, 121 direct neighbor proteins, and 661 additional non-risk proteins. It contains 2,883 PPIs, with approximately 71% of interactions involving RA risk proteins and their direct neighbors. It is mainly associated with immune, musculoskeletal, and metabolic systems. Key pathways include cytokine signaling, innate immune processes, myeloid cell differentiation, hematopoietic development, responses to infectious diseases, such as Hepatitis B, measles, and cancer-related pathways, such as leukemia, breast cancer.</p>
<p><a href="#pone.0329101.g006" class="usa-link">Fig 6A</a> shows a subnetwork of PPI in Network 1, with red circles indicating genetic risk factors and blue circles indicating main hub proteins interacting with 80% of genetic risk factors. PADI4, a significant risk protein, is an arginine-to-citrulline converting enzyme, important for RA pathogenesis and diagnosis. Despite its role, PADI4’s interactions in Network 1 are sparse, mainly involving inflammatory cytokines and signaling, more related to pathogenesis than disease risk. In Network 1, VEGF is a hub protein that regulates inflammatory responses and is involved in cancer pathways, supporting tumor growth and chronic inflammation in RA. VEGF also contributes to the TNF signaling pathway, a therapeutic target for diseases associated with immunity and cancer.</p>
<figure class="fig xbox font-sm" id="pone.0329101.g006"><h5 class="obj_head">Fig 6. Integrative analysis of molecular network and disease association of genetic risk factors for RA.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370121_pone.0329101.g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d00/12370121/0eed6edc7ef9/pone.0329101.g006.jpg" loading="lazy" height="429" width="730" alt="Fig 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329101.g006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Key molecular and disease pathways of genetic risk proteins for RA in Network 1. (B) Risk gene-disease mapping, showing shared risk genes between RA and other diseases. (C) Pathway interactions of significant RA risk genes, therapeutic targets, and biomarkers, identified through disease mapping and multi-omics network enrichment. This visualization highlights functional overlaps between biological pathways, demonstrating how key risk genes interact across immune regulation, cytokine production, and hematopoietic differentiation.</p></figcaption></figure><p>The disease mapping analysis of RA risk genes reveals significant associations with various other diseases. Specifically, four RA risk proteins are linked to neurological diseases such as Alzheimer’s, six to psoriasis, five to respiratory diseases, nine to metabolic diseases, five to Sjogren’s syndrome, and six to bone diseases. These findings, illustrated in <a href="#pone.0329101.g006" class="usa-link">Fig 6B</a>, suggest shared molecular pathways and potential therapeutic targets across multiple diseases.</p>
<p>The Venn diagram in <a href="#pone.0329101.g006" class="usa-link">Fig 6C</a> illustrates the intersections among three main pathways involved in RA development: cytokine production regulation, hematopoietic signaling, and myeloid cell differentiation. A total of 28 genetic risk factors overlap between these pathways, including 11 identified therapeutic targets and 7 biomarkers for RA. (See <a href="#pone.0329101.t004" class="usa-link">Table 4</a>)</p>
<section class="tw xbox font-sm" id="pone.0329101.t004"><h5 class="obj_head">Table 4. Therapeutic and pathway profiling of RA risk proteins via omics graphs.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Target</th>
<th align="left" rowspan="1" colspan="1">Drug(s)</th>
<th align="left" rowspan="1" colspan="1">Drug Class</th>
<th align="left" rowspan="1" colspan="1">Mechanism of Action</th>
<th align="left" rowspan="1" colspan="1">Pathway</th>
<th align="left" rowspan="1" colspan="1">Approved Indication(s)</th>
<th align="left" rowspan="1" colspan="1">Status</th>
<th align="left" rowspan="1" colspan="1">DrugBank ID(s)</th>
<th align="left" rowspan="1" colspan="1">Clinical Aspect</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">RELA</td>
<td align="left" rowspan="1" colspan="1">Dexamethasone, Celastrol</td>
<td align="left" rowspan="1" colspan="1">Glucocorticoid, Phytochemical</td>
<td align="left" rowspan="1" colspan="1">Dexamethasone activates NR3C1 to inhibit NF-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0329101.e015"><math id="M15" display="inline" overflow="linebreak"><mrow><mi>κ</mi></mrow></math></span>B; Celastrol directly inhibits NF-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0329101.e016"><math id="M16" display="inline" overflow="linebreak"><mrow><mi>κ</mi></mrow></math></span>B</td>
<td align="left" rowspan="1" colspan="1">NF-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0329101.e017"><math id="M17" display="inline" overflow="linebreak"><mrow><mi>κ</mi></mrow></math></span>B signaling, Cytokine production</td>
<td align="left" rowspan="1" colspan="1">Autoimmune diseases, Inflammation</td>
<td align="left" rowspan="1" colspan="1">Approved (Dexamethasone), Preclinical (Celastrol)</td>
<td align="left" rowspan="1" colspan="1">DB01234</td>
<td align="left" rowspan="1" colspan="1">Exploratory-phase Celastrol has undefined toxicity, precluding RA clinical use</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">NFATC1</td>
<td align="left" rowspan="1" colspan="1">Tacrolimus, Cyclosporine A</td>
<td align="left" rowspan="1" colspan="1">Calcineurin inhibitors</td>
<td align="left" rowspan="1" colspan="1">Inhibit calcineurin, preventing NFATC1 dephosphorylation and nuclear entry</td>
<td align="left" rowspan="1" colspan="1">Calcium–NFAT signaling, T-cell activation, Osteoclastogenesis</td>
<td align="left" rowspan="1" colspan="1">Transplant rejection, Autoimmune diseases</td>
<td align="left" rowspan="1" colspan="1">Approved</td>
<td align="left" rowspan="1" colspan="1">DB00864, DB00091</td>
<td align="left" rowspan="1" colspan="1">Effectively suppresses T-cell activity; requires careful management of immunosuppression in RA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">LCK</td>
<td align="left" rowspan="1" colspan="1">Dasatinib</td>
<td align="left" rowspan="1" colspan="1">Src-family kinase inhibitor</td>
<td align="left" rowspan="1" colspan="1">Inhibits LCK (and FYN), blocking TCR signaling</td>
<td align="left" rowspan="1" colspan="1">T-cell receptor signaling, Immune synapse</td>
<td align="left" rowspan="1" colspan="1">Leukemia (CML, ALL)</td>
<td align="left" rowspan="1" colspan="1">Approved</td>
<td align="left" rowspan="1" colspan="1">DB01254</td>
<td align="left" rowspan="1" colspan="1">Marrow suppression and bleeding limit its use in RA, though it aids pathway analysis</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FYN</td>
<td align="left" rowspan="1" colspan="1">Dasatinib</td>
<td align="left" rowspan="1" colspan="1">Src-family kinase inhibitor</td>
<td align="left" rowspan="1" colspan="1">Inhibits FYN (and LCK), suppressing T-cell receptor signaling</td>
<td align="left" rowspan="1" colspan="1">TCR signaling / Immune cell migration</td>
<td align="left" rowspan="1" colspan="1">Leukemia (CML, ALL)</td>
<td align="left" rowspan="1" colspan="1">Approved</td>
<td align="left" rowspan="1" colspan="1">DB01254</td>
<td align="left" rowspan="1" colspan="1">Safety concerns same as for LCK; severe side effects restrict RA use; relevant in mechanistic studies</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PIK3R1</td>
<td align="left" rowspan="1" colspan="1">Alpelisib</td>
<td align="left" rowspan="1" colspan="1">PI3K<em>α</em> inhibitor</td>
<td align="left" rowspan="1" colspan="1">Inhibits PIK3CA affecting PIK3R1–PIK3CA heterodimer to suppress AKT/mTOR axis</td>
<td align="left" rowspan="1" colspan="1">PI3K–AKT–mTOR pathway, immune regulation</td>
<td align="left" rowspan="1" colspan="1">HR-positive, HER2-negative breast cancer</td>
<td align="left" rowspan="1" colspan="1">Approved</td>
<td align="left" rowspan="1" colspan="1">DB12015</td>
<td align="left" rowspan="1" colspan="1">Offers immune regulatory potential; caution warranted due to immunosuppression and infection risk in RA</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PRKCB</td>
<td align="left" rowspan="1" colspan="1">Enzastaurin</td>
<td align="left" rowspan="1" colspan="1">PKC<em>β</em> inhibitor</td>
<td align="left" rowspan="1" colspan="1">Inhibits PRKCB, disrupting BCR signaling and angiogenesis</td>
<td align="left" rowspan="1" colspan="1">PKC<em>β</em> signaling, B-cell receptor axis, Angiogenesis</td>
<td align="left" rowspan="1" colspan="1">Glioblastoma, Lymphoma</td>
<td align="left" rowspan="1" colspan="1">Investigational</td>
<td align="left" rowspan="1" colspan="1">DB06486</td>
<td align="left" rowspan="1" colspan="1">Investigational status; requires additional validation for use in RA</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0329101.t004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section></section><section id="sec028"><h3 class="pmc_sec_title">3.4 Multi-omics knowledge graph analysis of RA genetic risk factors</h3>
<p><a href="#pone.0329101.t002" class="usa-link">Table 2</a> presents the non-risk proteins that directly interact with RA risk proteins. <a href="#pone.0329101.g007" class="usa-link">Fig 7</a> illustrates the constructed sub-knowledge graphs, highlighting the top 10 enriched pathways or terms from six key databases: Human Gene Atlas, Proteomics Drug Atlas, DiseaseGeneNet [<a href="#pone.0329101.ref047" class="usa-link" aria-describedby="pone.0329101.ref047">47</a>], KEGG, The Kinase Library, and GWAS Catalog [<a href="#pone.0329101.ref059" class="usa-link" aria-describedby="pone.0329101.ref059">59</a>]. These databases were integrated to identify significant proteins that are highly connected with RA risk proteins, aiding in the understanding of potential disease mechanisms and therapeutic targets.</p>
<section class="tw xbox font-sm" id="pone.0329101.t002"><h4 class="obj_head">Table 2. Molecular pathways and disease associations of interacting proteins with risk proteins in network 1.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Category</th>
<th align="left" rowspan="1" colspan="1">Neighboring Genes</th>
<th align="left" rowspan="1" colspan="1">Associated Diseases</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">A) Cytokine Signaling: 39 genes</td>
<td align="left" rowspan="1" colspan="1">IL6ST, SOCS3, TNFSF9, TRAF2, IL12A, IL12B, IL12RB1, IL1B, TNFRSF4, IFNG, IRF9, TNFRSF10A, TNFRSF10B, TNFRSF1A, IL19, IL20, IL20RA, IL22RA1, IL2RB, IL2RG, IL10RA, IL10RB, IFNGR1, IL3RA, IL5RA, TNFRSF25, TNFRSF6B, IL23A, IL1R1, IL1RAP, IL1R2, IRAK3, IFNAR1, IFNAR2, CXCL1, IL1A, IL17RC, TNFSF15, TNFSF13B</td>
<td align="left" rowspan="1" colspan="1">Alzheimer’s Disease, Psoriasis, Crohn’s Disease, Cancer, Diabetes, Inflammatory Diseases, Inflammatory Bowel Disease, Psoriasis, Asthma, Immunodeficiency Disorders, Various Inflammatory Diseases, Immune System Disorders, Autoimmune Diseases, Infectious Diseases, Viral Infections, Autoimmune Diseases, Psoriasis, Inflammatory Bowel Disease, Severe Combined Immunodeficiency, Hematological Malignancies, Eosinophilic Disorders, Autoimmune Diseases, Cancer, Various Inflammatory Diseases, Immune System Disorders, Viral Infections, Autoimmune Diseases, Cancer, Immune System Disorders</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B) Transcription Factors: 30 genes</td>
<td align="left" rowspan="1" colspan="1">SPI1, EGR2, GATA2, CEBPA, BATF, FOXP3, NFATC1, MAFF, PRDM1, JUN, JUNB, FOS, ETS1, TAL1, MYB, RORA, RORC, TBX21, TBXAS1, KLF6, ELF2, RELB, TP63, MAFK, FLI1, STAT5A, STAT5B, STAT6, NF1, RELA</td>
<td align="left" rowspan="1" colspan="1">Leukemia, Multiple Sclerosis, Myelodysplastic Syndromes, Immunodeficiency, Charcot-Marie-Tooth Disease, Cancer, Autoimmune Diseases, Heart Disease, Diabetes, Inflammatory Diseases, Bone Diseases, Blood Disorders, Asthma, Cardiovascular Diseases</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">C) Immune Receptors and Signaling: 37 genes</td>
<td align="left" rowspan="1" colspan="1">CD247, FCER1G, FYN, LYN, NCAM1, SRC, LCK, CD22, CD8A, CD48, CD274, CD276, TNFRSF1B, CSK, PTPN11, PTPN6, PTPRCAP, PTPN1, PTPN12, PTPN18, SH2D1A, SH2D1B, SH2B3, SHC1, CD79A, CD79B, CD40LG, IL2RA, IL2RB, IL2RG, IL10RA, IL10RB, IL5RA, IL3RA, IL17RC, CXCL1, ACKR4</td>
<td align="left" rowspan="1" colspan="1">Immune System Disorders, Allergic Diseases, Cancer, Nervous System Diseases, Bone Diseases, Hematological Malignancies, Eosinophilic Disorders, Psoriasis, Autoimmune Diseases, Inflammatory Bowel Disease, Viral Infections, Various Inflammatory Diseases, Immune System Disorders, Severe Combined Immunodeficiency</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">D) Enzymes and Kinases: 35 genes</td>
<td align="left" rowspan="1" colspan="1">AKT1, MAP2K1, MAP2K2, MAPK1, MAPK8, MAPKAPK2, PRKCB, PRKCD, PRKACA, CDK8, CDKN2A, CDKN2B, CDKN2C, CDKN2D, CCNA1, CCNA2, CCNB1, CCND2, CCND3, CDC25B, SRC, LCK, FYN, LYN, CSK, ITK, TEC, AKT1, AKT2, PIK3CA, PIK3R1, PLCG1, PLCG2, RAF1, PTK2</td>
<td align="left" rowspan="1" colspan="1">Cancer, Diabetes, Neurological Disorders, Various Inflammatory Diseases, Developmental Disorders, Myelodysplastic Syndromes, Neurodevelopmental Disorders, Metabolic Disorders, Bone Diseases, Cardiovascular Diseases, Immune System Disorders</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0329101.t002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="pone.0329101.g007"><h4 class="obj_head">Fig 7. Multi-omics knowledge graph analysis of genetic risk factors in RA.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370121_pone.0329101.g007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d00/12370121/28b30bc747a8/pone.0329101.g007.jpg" loading="lazy" height="627" width="730" alt="Fig 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329101.g007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The constructed knowledge graph displays RA risk proteins and their interacting non-risk proteins, integrating data from multiple databases. This analysis highlights the top 10 enriched pathways and molecular interactions, providing insights into key biological processes, signaling cascades, and potential therapeutic targets involved in RA pathogenesis.</p></figcaption></figure><p>The knowledge graph integrates data from multiple sources to uncover RA-associated molecular interactions, genetic predisposition, and therapeutic targets. Human Gene Atlas identifies tissue-specific expression, while Proteomics Drug Atlas links proteins to therapeutic targets. DiseaseGeneNet and GWAS Catalog highlight shared genetic risk factors, and KEGG maps molecular pathways involved in RA. The Kinase Library focuses on phosphorylation networks essential for immune regulation. Together, these datasets provide a comprehensive view of RA pathogenesis, aiding in drug discovery and precision medicine approaches.</p>
<section id="sec029"><h4 class="pmc_sec_title">3.4.1 Cytokine signaling pathway neighboring proteins.</h4>
<p>The subnetwork analysis of the cytokine signaling pathway, illustrated in <a href="#pone.0329101.g007" class="usa-link">Fig 7A</a>, reveals significant associations with RA and related diseases through the study of neighboring proteins of genetic risk factors. Key proteins such as TNFRSF1A, IL1R1, and IL6ST emerge as potential therapeutic targets and biomarkers for RA. TNFRSF1A is linked to a range of diseases including lupus erythematosus, arthritis, RA, infection, multiple sclerosis, autoimmune diseases, inflammatory bowel diseases, ulcerative colitis, pneumonia, and ankylosing spondylitis [<a href="#pone.0329101.ref047" class="usa-link" aria-describedby="pone.0329101.ref047">47</a>]. Additionally, proteomics data analysis shows that TNFRSF1A is upregulated by drugs such as ascomycin, UNC0631, perhexiline, obatoclax, and berzosertib, and is phosphorylated by kinases including YANK2, MEK1, HPK1, and CDK6 [<a href="#pone.0329101.ref060" class="usa-link" aria-describedby="pone.0329101.ref060">60</a>, <a href="#pone.0329101.ref061" class="usa-link" aria-describedby="pone.0329101.ref061">61</a>]. Similarly, IL1R1 is associated with RA and other inflammatory bowel diseases [<a href="#pone.0329101.ref047" class="usa-link" aria-describedby="pone.0329101.ref047">47</a>, <a href="#pone.0329101.ref059" class="usa-link" aria-describedby="pone.0329101.ref059">59</a>]. IL6ST is linked to RA, multiple sclerosis, autoimmune diseases, inflammatory bowel diseases, pneumonia, and Crohn’s disease, and is regulated by multiple drugs and phosphorylated by CDK6 [<a href="#pone.0329101.ref047" class="usa-link" aria-describedby="pone.0329101.ref047">47</a>, <a href="#pone.0329101.ref059" class="usa-link" aria-describedby="pone.0329101.ref059">59</a>]. From our analysis of 39 neighboring proteins of genetic risk factors in the cytokine signaling pathway, these findings suggest TNFRSF1A, IL1R1, and IL6ST are promising therapeutic targets and biomarkers for developing therapies and diagnostics for RA and related inflammatory diseases. This approach has the potential to improve treatment and disease management.</p></section><section id="sec030"><h4 class="pmc_sec_title">3.4.2 Analysis of transcription factors.</h4>
<p>Subnetwork analysis in <a href="#pone.0329101.g007" class="usa-link">Fig 7B</a> identifies key transcription factors associated with various diseases [<a href="#pone.0329101.ref047" class="usa-link" aria-describedby="pone.0329101.ref047">47</a>, <a href="#pone.0329101.ref059" class="usa-link" aria-describedby="pone.0329101.ref059">59</a>, <a href="#pone.0329101.ref061" class="usa-link" aria-describedby="pone.0329101.ref061">61</a>]. Our results highlight RELA, ETS1, NFATC1, and ITGB2 as central players in the pathogenesis of various diseases, including RA, lymphoma, asthma, acute myelocytic leukemia, and leukemogenesis. The analysis demonstrates phosphorylation of these factors by kinases such as MST3, CHAK2, SLK, and LOK. GWAS data further strengthen the link between these genes and diseases, with BATF, CEBPA, and ETS1 associated with allergic diseases, and PTPN22, ETS1, PTPN11, and BATF linked to RA [<a href="#pone.0329101.ref061" class="usa-link" aria-describedby="pone.0329101.ref061">61</a>]. This convergence of disease association and kinase interaction suggests that RELA, ETS1, and NFATC1 are promising therapeutic targets and biomarkers, particularly for RA and related inflammatory diseases.</p></section><section id="sec031"><h4 class="pmc_sec_title">3.4.3 Analysis of immune receptors and signaling.</h4>
<p>The subnetwork analysis indicates that several kinases phosphorylate essential immune receptors and signaling proteins (<a href="#pone.0329101.g007" class="usa-link">Fig 7C</a>). Specifically, NUAK1 phosphorylates SH2B3, PTPN22, LCK, GPR174, PTPN11, and RUNX1. Type 1 diabetes mellitus shares eight risk genes with RA. SH2B3 regulates JAK-STAT signaling and is a potential therapeutic target for RA [<a href="#pone.0329101.ref062" class="usa-link" aria-describedby="pone.0329101.ref062">62</a>, <a href="#pone.0329101.ref063" class="usa-link" aria-describedby="pone.0329101.ref063">63</a>]. CD40 is a biomarker and therapeutic target for RA [<a href="#pone.0329101.ref047" class="usa-link" aria-describedby="pone.0329101.ref047">47</a>], while CRP is a common biomarker for multiple diseases. IRE2 is shown to phosphorylate FYN, IL2RA, PTPN22, and PTPN2. Additionally, MST3 phosphorylates IL2RA and PTPN22, while MST2 targets FYN and LCK. TNIK phosphorylates IL2RA and CYTIP. These interactions underscore the regulatory complexity and potential points of therapeutic intervention within the immune signaling pathways. Moreover, gene expression analysis reveals that the genes CD247, CYTIP, PTPRC, TRAF3IP3, CD96, IL2RB, PTPN22, PTPN2, FYN, and SLAMF6 are upregulated in CD56+ NK cells. Furthermore, TRAF3IP3, SLAMF6, CD3E, ITK, FYN, CD96, and LCK are upregulated in CD8+ T cells. This differential gene expression highlights the cell-type-specific roles of these genes in immune function and disease. Additionally, key genes such as IL2RA and CD247, linked with risk genes PTPN22, PTPN2, and RUNX1, are consistently implicated across multiple conditions, including RA, lupus erythematosus, B-cell lymphomas, and autoimmune diseases. These findings suggest that IL2RA, RUNX1, and associated kinases are potential targets for developing immune-related disease therapies and diagnostics.</p></section></section><section id="sec032"><h3 class="pmc_sec_title">3.5 Analysis of enzymes and kinases</h3>
<p>Our analysis of neighboring proteins categorized as enzymes and kinases in <a href="#pone.0329101.g007" class="usa-link">Fig 7D</a> highlights the complex interactions and regulatory mechanisms involving key proteins associated with various diseases. Several drugs modulate the expression of these proteins. For instance, betamethasone and formestane upregulate CDC25B, while toceranib upregulates FYN and downregulates CDC25B. GSK 650394 upregulates CDC25B and CCND3, whereas dasatinib downregulates CDC25B and upregulates RASGRP2. This modulation is crucial as gene expression data from the Human Gene Atlas reveals that proteins such as PIK3R1, CDC25B, CD3E, FYN, RASGRP2, and SH2D1A are upregulated in CD8+ T cells, and similar upregulation occurs in CD4+ T cells for CD3E, FYN, CDC25B, PIK3R1, ZAP70, and RASGRP2. Kinase interactions further illustrate that VRK1, CAMKK2, and TAK1 phosphorylate these proteins, highlighting their regulatory roles. These findings indicate that the identified genes CDC25B, FYN, PRKCB, PIK3R1, and CCND3, along with kinases VRK1, CAMKK2, and TAK1, are promising targets for therapeutic strategies and diagnostic tools for enzyme and kinase-related diseases.</p></section><section id="sec033"><h3 class="pmc_sec_title">3.6 Identification of potential therapeutic targets for RA</h3>
<p>From our comprehensive analysis, we identified a total of 35 significant therapeutic targets and biomarkers for RA. These genes were categorized into three primary pathways: Pathway A (16 genes), Pathway B (17 genes), and Pathway C (2 genes). Among these, 25 genes were classified as risk genes, while 10 were neighboring genes, suggesting their functional relevance in RA pathogenesis. Notably, we identified nine novel proteins as potential therapeutic targets for RA based on their strong molecular interactions, involvement in immune signaling pathways, and inferred functional relevance in RA progression.</p>
<p>Based on our inference score, we selected the top genes with a score greater than 0.85 and considered them potential risk genes. We identified nine novel and candidate therapeutic targets for RA: PIK3R1, ETS1, NFATC1, LCK, PRKCB, BATF, RASGRP1, FYN, and RELA. These genes were prioritized based on their involvement in immune regulation, transcriptional control, and inflammatory signaling. (See <a href="#pone.0329101.t003" class="usa-link">Table 3</a>).</p>
<section class="tw xbox font-sm" id="pone.0329101.t003"><h4 class="obj_head">Table 3. Potential therapeutic targets and biomarkers of RA.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">Protein</th>
<th align="left" rowspan="1" colspan="1">Type</th>
<th align="left" rowspan="1" colspan="1">Inference Score</th>
<th align="left" rowspan="1" colspan="1">Situation</th>
<th align="left" rowspan="1" colspan="1">Novelty</th>
<th align="left" rowspan="1" colspan="1">PubMed ID (PMID)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">RASGRP1</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.93</td>
<td align="left" rowspan="1" colspan="1">Potential target</td>
<td align="left" rowspan="1" colspan="1">Novel</td>
<td align="left" rowspan="1" colspan="1">34199962, 34777472</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">NFATC1</td>
<td align="left" rowspan="1" colspan="1">Interaction Partner</td>
<td align="left" rowspan="1" colspan="1">0.93</td>
<td align="left" rowspan="1" colspan="1">Potential target</td>
<td align="left" rowspan="1" colspan="1">Novel</td>
<td align="left" rowspan="1" colspan="1">31972421, 34681582</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">RELA</td>
<td align="left" rowspan="1" colspan="1">Interaction Partner</td>
<td align="left" rowspan="1" colspan="1">0.92</td>
<td align="left" rowspan="1" colspan="1">Potential target</td>
<td align="left" rowspan="1" colspan="1">Novel</td>
<td align="left" rowspan="1" colspan="1">34750358, 37457726</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">ETS1</td>
<td align="left" rowspan="1" colspan="1">Interaction Partner</td>
<td align="left" rowspan="1" colspan="1">0.92</td>
<td align="left" rowspan="1" colspan="1">Potential target</td>
<td align="left" rowspan="1" colspan="1">Novel</td>
<td align="left" rowspan="1" colspan="1">38821936, 34199962</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">LCK</td>
<td align="left" rowspan="1" colspan="1">Interaction Partner</td>
<td align="left" rowspan="1" colspan="1">0.90</td>
<td align="left" rowspan="1" colspan="1">Potential target</td>
<td align="left" rowspan="1" colspan="1">Novel</td>
<td align="left" rowspan="1" colspan="1">36910149, 38473928</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PIK3R1</td>
<td align="left" rowspan="1" colspan="1">Interaction Partner</td>
<td align="left" rowspan="1" colspan="1">0.90</td>
<td align="left" rowspan="1" colspan="1">Potential target</td>
<td align="left" rowspan="1" colspan="1">Novel</td>
<td align="left" rowspan="1" colspan="1">36992829</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">BATF</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.90</td>
<td align="left" rowspan="1" colspan="1">Potential target</td>
<td align="left" rowspan="1" colspan="1">Novel</td>
<td align="left" rowspan="1" colspan="1">38821936</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TNFRSF1A</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.93</td>
<td align="left" rowspan="1" colspan="1">Identified target</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">37396624</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">IL1R1</td>
<td align="left" rowspan="1" colspan="1">Interaction Partner</td>
<td align="left" rowspan="1" colspan="1">0.92</td>
<td align="left" rowspan="1" colspan="1">Identified target</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">DrugBank</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">IL6ST</td>
<td align="left" rowspan="1" colspan="1">Interaction Partner</td>
<td align="left" rowspan="1" colspan="1">0.91</td>
<td align="left" rowspan="1" colspan="1">Identified target</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">35140805</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CD40</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.90</td>
<td align="left" rowspan="1" colspan="1">Identified biomarker</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">Biomarker Database</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TGF-<em>β</em>1</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.90</td>
<td align="left" rowspan="1" colspan="1">Identified target</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">25967237</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PRKCB</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.89</td>
<td align="left" rowspan="1" colspan="1">Potential target</td>
<td align="left" rowspan="1" colspan="1">Novel</td>
<td align="left" rowspan="1" colspan="1">33667569</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">RUNX1</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.89</td>
<td align="left" rowspan="1" colspan="1">Potential target</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">36625319, 34737275</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">IRAK1</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.89</td>
<td align="left" rowspan="1" colspan="1">Identified target</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">Biomarker</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FYN</td>
<td align="left" rowspan="1" colspan="1">Interaction Partner</td>
<td align="left" rowspan="1" colspan="1">0.88</td>
<td align="left" rowspan="1" colspan="1">Potential target</td>
<td align="left" rowspan="1" colspan="1">Novel</td>
<td align="left" rowspan="1" colspan="1">33714238, 36740671, 36693907</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">SH2B3</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.88</td>
<td align="left" rowspan="1" colspan="1">Identified target</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">32879140, 33067605</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CRP</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.88</td>
<td align="left" rowspan="1" colspan="1">Identified biomarker</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">34880861, 35379209</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AIM2</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.88</td>
<td align="left" rowspan="1" colspan="1">Identified target</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">37426634, 33976384</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">FLT3</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.88</td>
<td align="left" rowspan="1" colspan="1">Potential target</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">33003568, 38238321</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">IL2RA</td>
<td align="left" rowspan="1" colspan="1">Interaction Partner</td>
<td align="left" rowspan="1" colspan="1">0.87</td>
<td align="left" rowspan="1" colspan="1">Potential target</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">31253980</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">IRF8</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.87</td>
<td align="left" rowspan="1" colspan="1">Identified target</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">37066336, 33673123</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PTPN2</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.87</td>
<td align="left" rowspan="1" colspan="1">Identified biomarker</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">Biomarker</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PTPN11</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.86</td>
<td align="left" rowspan="1" colspan="1">Identified target</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">35265082</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PTPN22</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.86</td>
<td align="left" rowspan="1" colspan="1">Identified biomarker</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">35637605</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">AIRE</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.85</td>
<td align="left" rowspan="1" colspan="1">Identified biomarker</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">Biomarker Database, 34199962, 32849810</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">MED1</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.85</td>
<td align="left" rowspan="1" colspan="1">Identified target</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">31092410, 34777472</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CDK2</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.85</td>
<td align="left" rowspan="1" colspan="1">Potential target</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">38818077</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">PLD4</td>
<td align="left" rowspan="1" colspan="1">Risk Protein</td>
<td align="left" rowspan="1" colspan="1">0.84</td>
<td align="left" rowspan="1" colspan="1">Identified target</td>
<td align="left" rowspan="1" colspan="1">Previously studied</td>
<td align="left" rowspan="1" colspan="1">34199962, 32849810</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pone.0329101.t003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec034"><h3 class="pmc_sec_title">3.7 Validation of novel RA therapeutic targets</h3>
<p>The in silico validation of identified therapeutic targets was performed by analyzing their tissue-specific and single-cell expression patterns using data from GTEx, the HPA, and publicly available scRNA-seq datasets. As shown in <a href="#pone.0329101.g008" class="usa-link">Fig 8</a>, our analysis of gene expression profiles supports the tissue relevance of the identified therapeutic targets. The result indicates that RELA, PIK3R1, and BATF are highly associated with bone marrow, highlighting their involvement in hematopoietic processes essential for immune function. Furthermore, RELA, RASGRP1, ETS1, PRKCB, BATF, and LCK are predominantly expressed in lymphoid tissues, reinforcing their role in immune cell activation and inflammatory pathways characteristic of RA. Additionally, RASGRP1, PRKCB, and NFATC1 show significant expression in the brain, suggesting potential neuroimmune interactions relevant to RA pathology.</p>
<figure class="fig xbox font-sm" id="pone.0329101.g008"><h4 class="obj_head">Fig 8. RNA tissue expression analysis of novel RA therapeutic targets.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370121_pone.0329101.g008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d00/12370121/9c77c550b8a0/pone.0329101.g008.jpg" loading="lazy" height="599" width="740" alt="Fig 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329101.g008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Gene expression profiles of the nine identified novel RA therapeutic targets across various tissues, highlighting their relevance in immune-related, hematopoietic, and neurological pathways. The analysis integrates GTEx and Human Protein Atlas (HPA) datasets, showing differential expression in bone marrow, lymphoid tissues, and the brain, which are critical for RA pathogenesis. These findings support the potential role of these genes in immune dysregulation and inflammation, reinforcing their viability as therapeutic targets.</p></figcaption></figure><p><a href="#pone.0329101.g009" class="usa-link">Fig 9</a> presents the single-cell RNA expression analysis, which confirmed that all nine genes are highly expressed in immune cells and within the broader immune system, further strengthening their role as RA-related therapeutic targets. Notably, RASGRP1 and PRKCB exhibit elevated expression in brain cells, a finding that aligns with their identification as novel biomarkers for Alzheimer’s disease. This suggests a genetic link between RA and neurodegenerative disorders, potentially indicating shared inflammatory mechanisms. Moreover, PIK3R1 is highly expressed in cardiomyocytes, implicating a possible role in cardiovascular complications associated with RA.</p>
<figure class="fig xbox font-sm" id="pone.0329101.g009"><h4 class="obj_head">Fig 9. Single-cell RNA expression analysis of novel RA therapeutic targets.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370121_pone.0329101.g009.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/3d00/12370121/16ff7a0e374f/pone.0329101.g009.jpg" loading="lazy" height="589" width="740" alt="Fig 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/pone.0329101.g009/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The scRNA-seq analysis of the nine identified novel RA therapeutic targets, highlighting their expression across different immune cell types. The results demonstrate strong enrichment in T cells, B cells, macrophages, and dendritic cells, reinforcing their involvement in immune regulation and RA pathogenesis. Notably, RASGRP1 and PRKCB exhibit elevated expression in brain cells, linking RA-associated genetic risk factors to neuroinflammatory pathways, while PIK3R1 shows significant expression in cardiomyocytes, suggesting potential cardiovascular implications.</p></figcaption></figure></section><section id="sec035"><h3 class="pmc_sec_title">3.8 Therapeutic implications and drug repurposing opportunities</h3>
<p>Among the nine novel RA risk proteins, six are targetable by current or investigational drugs, highlighting opportunities for rapid repurposing with translational potential. For example, RELA is indirectly suppressed by dexamethasone, which binds NR3C1 and induces I<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0329101.e013"><math id="M13" display="inline" overflow="linebreak"><mrow><mi>κ</mi></mrow></math></span>B, thereby blocking NF-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0329101.e014"><math id="M14" display="inline" overflow="linebreak"><mrow><mi>κ</mi></mrow></math></span>B nuclear translocation and inflammatory gene expression [<a href="#pone.0329101.ref064" class="usa-link" aria-describedby="pone.0329101.ref064">64</a>]. Celastrol, an investigational triterpene, directly inhibits RELA by impairing proteasome function and modulating heat shock proteins. However, further validation is required due to unresolved issues related to toxicity and formulation [<a href="#pone.0329101.ref065" class="usa-link" aria-describedby="pone.0329101.ref065">65</a>]. Tacrolimus and cyclosporine A inhibit calcineurin via immunophilin complexes, preventing NFATC1 dephosphorylation and nuclear entry, thus reducing T-cell activation and cytokine release. Dasatinib targets LCK and FYN, two Src kinases involved in T-cell receptor signaling. Although effective at the molecular level, its clinical application in RA is limited due to serious side effects such as bone marrow suppression and bleeding. Nonetheless, it may be useful for studying TCR-associated pathways. Alpelisib, a PI3K<em>α</em>-selective inhibitor, targets the PIK3CA–PIK3R1 complex and suppresses AKT/mTOR signaling, affecting lymphocyte survival and synovial fibroblast function [<a href="#pone.0329101.ref066" class="usa-link" aria-describedby="pone.0329101.ref066">66</a>]. Careful monitoring is necessary due to potential immune suppression. Enzastaurin inhibits PRKCB, thereby disrupting B-cell receptor signaling and angiogenesis processes linked to synovial inflammation [<a href="#pone.0329101.ref067" class="usa-link" aria-describedby="pone.0329101.ref067">67</a>]. RASGRP1 does not have direct inhibitors, but its downstream signaling can be modulated using MEK inhibitors such as trametinib and cobimetinib, which block ERK/MAPK activation. ETS1 and BATF currently lack approved inhibitors, though early-stage small molecules targeting ETS1 show preclinical promise [<a href="#pone.0329101.ref068" class="usa-link" aria-describedby="pone.0329101.ref068">68</a>]. BATF may be indirectly regulated through interventions targeting Th17 differentiation.Several of these drugs function indirectly by modulating upstream regulators or downstream effectors of RA-associated proteins. For instance, dexamethasone regulates RELA via NR3C1, while MEK inhibitors modulate the RASGRP1 pathway. <a href="#pone.0329101.t004" class="usa-link">Table 4</a> summarizes the associated therapeutic agents, molecular mechanisms, and clinical relevance. This list may serve as a library of drug repurposing candidates for RA, with validation in preclinical models anticipated, especially for high-risk or treatment-resistant cases.</p>
<p>These findings provide robust molecular evidence linking RA risk genes to critical immune, neurological, and cardiovascular functions. The convergence of these genes across multiple tissues and disease pathways underscores their therapeutic potential, not only in RA but also in related disorders such as Alzheimer’s disease and cardiovascular conditions. Integrating these insights enhances our understanding of RA pathophysiology and guides precision medicine strategies for identifying biomarkers and therapeutic targets across multifactorial diseases. While these agents show compelling mechanistic rationale, further validation in RA-specific preclinical models and early-phase clinical trials is essential. Future studies should assess efficacy, immunologic safety, and infection risk. Integrating multiomics-based risk profiles with clinical phenotypes may enable more personalized treatment strategies for RA.</p></section></section><section id="sec036"><h2 class="pmc_sec_title">4 Discussion</h2>
<p>This study presents a comprehensive integrative analysis of genetic risk factors in RA using a multi-omics molecular network and knowledge graph approach. We developed a genetic risk factor–based knowledge graph integrating genomics, transcriptomics, and protein–protein interactions to elucidate RA molecular mechanisms. This approach enables the identification of key pathways, therapeutic targets, and drug repurposing opportunities based on functional connectivity, disease associations, and shared pathway signatures.</p>
<section id="sec037"><h3 class="pmc_sec_title">4.1 Key findings from the constructed multi-omics knowledge graph</h3>
<p>Based on our knowledge graph analysis, we identified 35 risk proteins and their neighboring genes that share three main pathways in RA. Our analysis indicates that seven risk proteins were previously identified as biomarkers (CD40, CRP, PTPN2, PTPN22, AIRE, SLC8A3, and CDC25B) and therapeutic targets for RA, while 11 risk proteins (TNFRSF1A, IL1R1, IL6ST, TGF-<em>β</em>1, IRAK1, SH2B3, AIM2, IRF8, PTPN11, MED1, and PLD4) have been recognized for their involvement in disease progression. Among these 35 proteins, we identified nine novel proteins as potential therapeutic targets: RELA, ETS1, NFATC1, BATF, LCK, PIK3R1, PRKCB, RASGRP1, and FYN. Existing RA treatments target 11 previously identified as therapeutic target for RA, including TNFRSF1A, IL1R1, IL6ST, CD40, TGF-<em>β</em>1, IRAK1, SH2B3, AIM2, IL2RA, PTPN2, and FLT3, all of which are involved in cytokine signaling and immune regulation. For instance, RELA, BATF, and NFATC1 influence T helper cell differentiation and pro-inflammatory cytokine expression, including IL-17 [<a href="#pone.0329101.ref069" class="usa-link" aria-describedby="pone.0329101.ref069">69</a>]. ETS1 downregulation weakens Treg cell function, intensifying inflammatory pathways. PIK3R1, a regulatory subunit of phosphoinositide 3-kinase, modulates key immune pathways via AKT/mTOR signaling. RASGRP1, LCK, PRKCB, and FYN are integral to immune receptor signaling, including T-cell receptor activation and downstream kinase cascades. In addition to their role in RA, certain novel targets, such as RASGRP1 and PRKCB exhibit high expression in brain tissues, implicating them in neuroinflammatory processes and suggesting genetic overlap between RA and Alzheimer’s disease [<a href="#pone.0329101.ref070" class="usa-link" aria-describedby="pone.0329101.ref070">70</a>].</p></section><section id="sec038"><h3 class="pmc_sec_title">4.2 Functional characterization and therapeutic potential of RA targets</h3>
<p>We further assessed the therapeutic relevance of these targets by integrating expression patterns with functional network analysis. The results confirmed a strong association between these proteins and RA pathogenesis, reinforcing their potential as therapeutic targets, particularly in drug-resistant cases. Although biologic therapies targeting TNF, IL-6, and IL-17 have significantly improved RA outcomes, a subset of patients remains unresponsive [<a href="#pone.0329101.ref071" class="usa-link" aria-describedby="pone.0329101.ref071">71</a>]. Our findings suggest that these newly identified proteins may contribute to drug resistance and offer alternative therapeutic strategies. RELA, a key regulator of the NF-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0329101.e018"><math id="M18" display="inline" overflow="linebreak"><mrow><mi>κ</mi></mrow></math></span>B pathway, may be persistently activated in TNF-inhibitor-resistant patients, making it a promising alternative target for non-responders [<a href="#pone.0329101.ref072" class="usa-link" aria-describedby="pone.0329101.ref072">72</a>]. The NF-<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pone.0329101.e019"><math id="M19" display="inline" overflow="linebreak"><mrow><mi>κ</mi></mrow></math></span>B pathway mediates pro-inflammatory signaling in both innate and adaptive immunity [<a href="#pone.0329101.ref072" class="usa-link" aria-describedby="pone.0329101.ref072">72</a>]. BATF, a transcription factor critical for Th17 differentiation and IL-17 production, has been implicated in IL-17-mediated inflammation. This suggests potential benefits from IL-17-targeted therapies in TNF-refractory RA cases [<a href="#pone.0329101.ref073" class="usa-link" aria-describedby="pone.0329101.ref073">73</a>, <a href="#pone.0329101.ref074" class="usa-link" aria-describedby="pone.0329101.ref074">74</a>]. NFATC1, implicated in osteoclastogenesis, could serve as a therapeutic target for RA patients with bone erosion who do not respond to disease-modifying antirheumatic drugs [<a href="#pone.0329101.ref069" class="usa-link" aria-describedby="pone.0329101.ref069">69</a>]. Additionally, PIK3R1 is highly expressed in immune-related tissues and exhibits strong network connectivity with known RA-associated proteins, reinforcing its therapeutic potential [<a href="#pone.0329101.ref075" class="usa-link" aria-describedby="pone.0329101.ref075">75</a>]. Furthermore, PIK3R1, which plays a role in kinase signaling, was highly expressed in cardiomyocytes, suggesting a connection between RA and cardiovascular disease. Notably, its expression in cardiomyocytes suggests a mechanistic link between RA and cardiovascular disease, aligning with the observed increase in atherosclerosis among RA patients [<a href="#pone.0329101.ref076" class="usa-link" aria-describedby="pone.0329101.ref076">76</a>]. Additionally, tissue-specific and single-cell analyses showed high expression of the identified genes in synovial tissue, immune cells, and lymphoid organs [<a href="#pone.0329101.ref077" class="usa-link" aria-describedby="pone.0329101.ref077">77</a>]. RASGRP1 and PRKCB displayed elevated expression in brain tissue, suggesting a genetic connection between RA and neuroinflammatory diseases such as Alzheimer’s disease [<a href="#pone.0329101.ref077" class="usa-link" aria-describedby="pone.0329101.ref077">77</a>, <a href="#pone.0329101.ref078" class="usa-link" aria-describedby="pone.0329101.ref078">78</a>].</p></section><section id="sec039"><h3 class="pmc_sec_title">4.3 Implications for drug pepurposing and disease progression</h3>
<p>Our findings highlight the potential of multi-omics knowledge graph analysis in elucidating the genetic risk factors contributing to RA pathogenesis [<a href="#pone.0329101.ref079" class="usa-link" aria-describedby="pone.0329101.ref079">79</a>, <a href="#pone.0329101.ref080" class="usa-link" aria-describedby="pone.0329101.ref080">80</a>]. These graphs integrate biological priors with omics data to uncover functional networks, enabling cross-disease analysis [<a href="#pone.0329101.ref080" class="usa-link" aria-describedby="pone.0329101.ref080">80</a>, <a href="#pone.0329101.ref081" class="usa-link" aria-describedby="pone.0329101.ref081">81</a>]. The overlap in risk pathways among RA, diabetes, AD, and cardiovascular diseases supports the hypothesis of shared inflammatory and immune-related etiologies [<a href="#pone.0329101.ref082" class="usa-link" aria-describedby="pone.0329101.ref082">82</a>].This approach offers a promising framework for understanding the shared pathogenic mechanisms among multiple diseases, including diabetes, Alzheimer’s disease, and cardiovascular disorders. Specifically, RASGRP1, PRKCB, and NFATC1 are implicated in both RA and Alzheimer’s disease, suggesting therapeutic convergence. Dysregulation of these genes contributes to systemic inflammation and immune dysregulation, features common to RA and neurodegenerative disorders [<a href="#pone.0329101.ref078" class="usa-link" aria-describedby="pone.0329101.ref078">78</a>, <a href="#pone.0329101.ref083" class="usa-link" aria-describedby="pone.0329101.ref083">83</a>]. PRKCB has been explored as a therapeutic target in AD via protein kinase C modulation [<a href="#pone.0329101.ref077" class="usa-link" aria-describedby="pone.0329101.ref077">77</a>, <a href="#pone.0329101.ref084" class="usa-link" aria-describedby="pone.0329101.ref084">84</a>]. The observed genetic overlap provides opportunities for drug repurposing. Existing therapies for chronic diseases, such as cancer or AD, may be repositioned for RA based on shared gene expression and protein interaction networks [<a href="#pone.0329101.ref085" class="usa-link" aria-describedby="pone.0329101.ref085">85</a>]. Bioinformatics-based drug repurposing efforts have identified several such candidates [<a href="#pone.0329101.ref085" class="usa-link" aria-describedby="pone.0329101.ref085">85</a>]. These results support the emerging model of precision drug repurposing—leveraging genetic and molecular data to tailor treatments to individual patients [<a href="#pone.0329101.ref085" class="usa-link" aria-describedby="pone.0329101.ref085">85</a>–<a href="#pone.0329101.ref087" class="usa-link" aria-describedby="pone.0329101.ref087">87</a>]. Integrating omics data with clinical profiles allows for the development of predictive models that inform prognosis, treatment response, and long-term risk stratification in RA [<a href="#pone.0329101.ref087" class="usa-link" aria-describedby="pone.0329101.ref087">87</a>].</p></section><section id="sec040"><h3 class="pmc_sec_title">4.4 Potential medical applications and precision medicine approaches</h3>
<p>The application of precision medicine in RA remains limited by the absence of predictive biomarkers [<a href="#pone.0329101.ref008" class="usa-link" aria-describedby="pone.0329101.ref008">8</a>, <a href="#pone.0329101.ref087" class="usa-link" aria-describedby="pone.0329101.ref087">87</a>]. Our comprehensive approach underscores the potential for targeting specific organs, tissues, and cell types to enhance RA treatment efficacy, slow disease progression, and mitigate complications [<a href="#pone.0329101.ref081" class="usa-link" aria-describedby="pone.0329101.ref081">81</a>, <a href="#pone.0329101.ref087" class="usa-link" aria-describedby="pone.0329101.ref087">87</a>]. Additionally, this study establishes a robust framework for utilizing genetic risk network analysis to identify critical pathways, potential therapeutic targets, and repurposing opportunities [<a href="#pone.0329101.ref085" class="usa-link" aria-describedby="pone.0329101.ref085">85</a>, <a href="#pone.0329101.ref086" class="usa-link" aria-describedby="pone.0329101.ref086">86</a>]. By integrating genetic risk factor analyses, this research uncovers previously overlooked therapeutic targets and disease mechanisms, offering new insights for future RA treatment development [<a href="#pone.0329101.ref085" class="usa-link" aria-describedby="pone.0329101.ref085">85</a>, <a href="#pone.0329101.ref088" class="usa-link" aria-describedby="pone.0329101.ref088">88</a>]. Genetic risk scores linked to over 30 RA-associated loci explain approximately half of the disease’s heritability [<a href="#pone.0329101.ref070" class="usa-link" aria-describedby="pone.0329101.ref070">70</a>, <a href="#pone.0329101.ref088" class="usa-link" aria-describedby="pone.0329101.ref088">88</a>]. Integrating these risk loci with clinical, genomic, and environmental data enables risk stratification and early prediction of complications [<a href="#pone.0329101.ref088" class="usa-link" aria-describedby="pone.0329101.ref088">88</a>]. Our findings suggest that existing drugs approved for diseases that share common genetic factors and pathways with RA may be repurposed for RA treatment. Integrating genetic risk factors with genomic and biomedical data is crucial for anticipating future complications in RA. Furthermore, this study sheds new light on the molecular mechanisms underlying RA by integrating multi-omics data, genetic risk factors, and molecular network analyses [<a href="#pone.0329101.ref079" class="usa-link" aria-describedby="pone.0329101.ref079">79</a>, <a href="#pone.0329101.ref084" class="usa-link" aria-describedby="pone.0329101.ref084">84</a>]. Understanding common pathways linking therapeutic targets, biomarkers, and organ dysfunction will enable more precise treatment planning and decision-making [<a href="#pone.0329101.ref024" class="usa-link" aria-describedby="pone.0329101.ref024">24</a>, <a href="#pone.0329101.ref081" class="usa-link" aria-describedby="pone.0329101.ref081">81</a>]. The enrichment network analysis presented here may contribute to the identification of novel biomarkers, prioritization of organ-specific treatments, and discovery of potential therapeutic targets for RA [<a href="#pone.0329101.ref079" class="usa-link" aria-describedby="pone.0329101.ref079">79</a>, <a href="#pone.0329101.ref080" class="usa-link" aria-describedby="pone.0329101.ref080">80</a>]. Graph models that integrate genetic and clinical data will be essential for shaping personalized, proactive, and preventive strategies in RA care. [<a href="#pone.0329101.ref087" class="usa-link" aria-describedby="pone.0329101.ref087">87</a>, <a href="#pone.0329101.ref089" class="usa-link" aria-describedby="pone.0329101.ref089">89</a>]</p></section><section id="sec041"><h3 class="pmc_sec_title">4.5 Limitations and challenges</h3>
<p>This study contributes to the medical analysis of RA by integrating multi-omics knowledge graph analysis of genetic risk factors, which may support personalized treatment strategies and predict potential complications [<a href="#pone.0329101.ref087" class="usa-link" aria-describedby="pone.0329101.ref087">87</a>]. Despite its promising implications, several challenges must be addressed before clinical application. One limitation is that the multi-omics analysis was conducted using a specific dataset, requiring validation across independent cohorts and technological platforms to confirm reproducibility [<a href="#pone.0329101.ref079" class="usa-link" aria-describedby="pone.0329101.ref079">79</a>]. Additionally, a larger sample size and stratified analyses based on disease progression and treatment response are needed to account for RA patient heterogeneity [<a href="#pone.0329101.ref090" class="usa-link" aria-describedby="pone.0329101.ref090">90</a>]. Experimental validation through in vivo and in vitro studies is essential to confirm the roles of these genes in RA pathogenesis and their potential as therapeutic targets. Understanding the interactions between genetic factors and environmental influences may further explain the complex etiology of RA [<a href="#pone.0329101.ref090" class="usa-link" aria-describedby="pone.0329101.ref090">90</a>]. The multifactorial nature of RA, involving the interplay between genetic and environmental factors, complicates treatment strategies [<a href="#pone.0329101.ref078" class="usa-link" aria-describedby="pone.0329101.ref078">78</a>, <a href="#pone.0329101.ref085" class="usa-link" aria-describedby="pone.0329101.ref085">85</a>, <a href="#pone.0329101.ref090" class="usa-link" aria-describedby="pone.0329101.ref090">90</a>]. While these findings provide a foundation for further research, functional studies are needed to determine the mechanistic roles of these genes in RA pathogenesis and treatment response. Future research will incorporate epigenetic and environmental data, diagnostic radiology, and pharmacological treatment information to refine precision medicine approaches, with the goal of developing individualized treatment strategies and preventing chronic RA [<a href="#pone.0329101.ref086" class="usa-link" aria-describedby="pone.0329101.ref086">86</a>, <a href="#pone.0329101.ref089" class="usa-link" aria-describedby="pone.0329101.ref089">89</a>].</p></section></section><section id="sec042"><h2 class="pmc_sec_title">5 Conclusion</h2>
<p>Our study has employed an integrated approach to analyze genetic risk factors and protein interactions in RA, identifying several key pathways and potential therapeutic targets. The identified pathways—Cytokine Regulation and Production, Hematopoietic or Lymphoid Organ Development, and Myeloid Cell Differentiation—are crucial in immune cell development and inflammatory responses. The study highlights seven risk genes intersecting all three pathways, presenting promising targets for therapeutic intervention. Furthermore, we identified 35 core region genes, including both recognized and new potential therapeutic targets, involved in immune responses, cell signaling, and inflammation. This comprehensive analysis underscores the importance of integrating multi-omics enrichment with network analysis to uncover the molecular mechanisms underlying RA. These findings provide a strong foundation for future research and the development of targeted therapies, with the potential to significantly improve RA management and patient outcomes. Further experimental validation and exploration of genetic and environmental interactions are necessary to advance our understanding and treatment of RA.</p></section><section id="sec043"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pone.0329101.s001"><div class="caption p">
<span>S1 Table. Genetic Risk Variants of RA: List of retrieved genetic risk SNP of Rheumatoid Arthritis retrieved from biomedical articles.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370121/bin/pone.0329101.s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0329101.s001.pdf</a><sup> (95.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0329101.s002"><div class="caption p">
<span>S2 Table. Annotated Genetic Risk Variants of RA: List of annotated genetic risk variants of Rheumatoid Arthritis retrieved from biomedical articles.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370121/bin/pone.0329101.s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0329101.s002.pdf</a><sup> (425.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0329101.s003"><div class="caption p">
<span>S3 Table. Annotated Genetic Risk Genes: List of annotated genetic risk genes of Rheumatoid Arthritis retrieved from biomedical articles.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370121/bin/pone.0329101.s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0329101.s003.pdf</a><sup> (121.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0329101.s004"><div class="caption p">
<span>S4 Table. Molecular network: Detailed information on the molecular functions and biological processes associated with the constructed networks of RA genetic risk factors.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370121/bin/pone.0329101.s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0329101.s004.pdf</a><sup> (114.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pone.0329101.s005"><div class="caption p">
<span>S5 Table. Therapeutic targets and biomarker validation dataset of RA: List of approved therapeutic targets and biomarkers of RA from DrugBank and biomarker databases.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370121/bin/pone.0329101.s005.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0329101.s005.pdf</a><sup> (133.9KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We thank Mr. Mohammed Alarawi for the discussion on the molecular functions of protein networks. We would such as to express our special thanks to Dr. Aseel Alsaedi for the medical discussion on the impact of the genetic risk factors for RA on the digestive system and the influence of existing treatments on patients with RA complications, Rt. Bushra Hameed for the discussion on diagnostic radiology in patients with RA , and Dr. Malak Alsaedi for the medical discussion on the impact of risk variants on the development of severe outcomes related to metabolic and immune diseases. This study was previously made available as a preprint on Research Square, licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) License: <a href="https://doi.org/10.21203/rs.3.rs-3607429/v1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.21203/rs.3.rs-3607429/v1</a>.</p></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>The data underlying the results presented in the study are available from in the <a href="#sec043" class="usa-link">Supporting information</a>.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This publication is based upon work supported by the King Abdullah University of Science and Technology (KAUST) Office of Research Administration (ORA) under Award Nos. REI/1/5234-01-01, REI/1/5414-01-01, REI/1/5289-01-01, REI/1/5404-01-01, REI/1/5992-01-01, URF/1/4663-01-01; the Center of Excellence for Smart Health (KCSH) under Award No. 5932; and the Center of Excellence on Generative AI under Award No. 5940 awarded to Professor Xin Gao; and the Office of Sponsored Research (OSR) under Award No. BAS/1/1059-01-01 awarded to Professor Takashi Gojobori. Funder website: <a href="https://www.kaust.edu.sa" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.kaust.edu.sa</a>. The funders had no role in study design, data collection and analysis, decision to publish, or preparation of the manuscript.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pone.0329101.ref001">
<span class="label">1.</span><cite>Dougados M. Comorbidities in rheumatoid arthritis. Curr Opinion Rheumatol. 2016;28(3):282–8.</cite> [<a href="https://doi.org/10.1097/BOR.0000000000000267" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27027814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opinion%20Rheumatol&amp;title=Comorbidities%20in%20rheumatoid%20arthritis&amp;author=M%20Dougados&amp;volume=28&amp;issue=3&amp;publication_year=2016&amp;pages=282-8&amp;pmid=27027814&amp;doi=10.1097/BOR.0000000000000267&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref002">
<span class="label">2.</span><cite>Nielen MMJ, van Schaardenburg D, Reesink HW, van de Stadt RJ, van der Horst-Bruinsma IE, de Koning MHMT, et al. Specific autoantibodies precede the symptoms of rheumatoid arthritis: a study of serial measurements in blood donors. Arthritis Rheumatism. 2004;50(2):380–6. doi: 10.1002/art.20018
</cite> [<a href="https://doi.org/10.1002/art.20018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14872479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheumatism&amp;title=Specific%20autoantibodies%20precede%20the%20symptoms%20of%20rheumatoid%20arthritis:%20a%20study%20of%20serial%20measurements%20in%20blood%20donors&amp;author=MMJ%20Nielen&amp;author=D%20van%20Schaardenburg&amp;author=HW%20Reesink&amp;author=RJ%20van%20de%20Stadt&amp;author=IE%20van%20der%20Horst-Bruinsma&amp;volume=50&amp;issue=2&amp;publication_year=2004&amp;pages=380-6&amp;pmid=14872479&amp;doi=10.1002/art.20018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref003">
<span class="label">3.</span><cite>Kerekes G, Nurmohamed MT, González-Gay MA, Seres I, Paragh G, Kardos Z, et al. Rheumatoid arthritis and metabolic syndrome. Nat Rev Rheumatol. 2014;10(11):691–6. doi: 10.1038/nrrheum.2014.121
</cite> [<a href="https://doi.org/10.1038/nrrheum.2014.121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25090948/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Rheumatol&amp;title=Rheumatoid%20arthritis%20and%20metabolic%20syndrome&amp;author=G%20Kerekes&amp;author=MT%20Nurmohamed&amp;author=MA%20Gonz%C3%A1lez-Gay&amp;author=I%20Seres&amp;author=G%20Paragh&amp;volume=10&amp;issue=11&amp;publication_year=2014&amp;pages=691-6&amp;pmid=25090948&amp;doi=10.1038/nrrheum.2014.121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref004">
<span class="label">4.</span><cite>Zhang Y, Li S, Cao Z, Cheng Y, Xu C, Yang H, et al. A network analysis framework of genetic and nongenetic risks for type 2 diabetes. Rev Endocr Metab Disord. 2020;22(2):461–9. doi: 10.1007/s11154-020-09585-2
</cite> [<a href="https://doi.org/10.1007/s11154-020-09585-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32926312/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rev%20Endocr%20Metab%20Disord&amp;title=A%20network%20analysis%20framework%20of%20genetic%20and%20nongenetic%20risks%20for%20type%202%20diabetes&amp;author=Y%20Zhang&amp;author=S%20Li&amp;author=Z%20Cao&amp;author=Y%20Cheng&amp;author=C%20Xu&amp;volume=22&amp;issue=2&amp;publication_year=2020&amp;pages=461-9&amp;pmid=32926312&amp;doi=10.1007/s11154-020-09585-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref005">
<span class="label">5.</span><cite>Edler MK, Mhatre-Winters I, Richardson JR. Microglia in aging and Alzheimer’s disease: a comparative species review. Cells. 2021;10(5):1138. doi: 10.3390/cells10051138
</cite> [<a href="https://doi.org/10.3390/cells10051138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8150617/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34066847/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=Microglia%20in%20aging%20and%20Alzheimer%E2%80%99s%20disease:%20a%20comparative%20species%20review&amp;author=MK%20Edler&amp;author=I%20Mhatre-Winters&amp;author=JR%20Richardson&amp;volume=10&amp;issue=5&amp;publication_year=2021&amp;pages=1138&amp;pmid=34066847&amp;doi=10.3390/cells10051138&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref006">
<span class="label">6.</span><cite>Montufar S, Calero C, Vinueza R, Correa P, Carrera-Gonzalez A, Villegas F, et al. Association between the APOE<em>ε</em>4 allele and late-onset Alzheimer’s disease in an ecuadorian mestizo population. Int J Alzheimer’s Dis. 2017;2017:1–9. doi: 10.1155/2017/1059678</cite> [<a href="https://doi.org/10.1155/2017/1059678" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5733981/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29348964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int%20J%20Alzheimer%E2%80%99s%20Dis&amp;title=Association%20between%20the%20APOE%CE%B54%20allele%20and%20late-onset%20Alzheimer%E2%80%99s%20disease%20in%20an%20ecuadorian%20mestizo%20population&amp;author=S%20Montufar&amp;author=C%20Calero&amp;author=R%20Vinueza&amp;author=P%20Correa&amp;author=A%20Carrera-Gonzalez&amp;volume=2017&amp;publication_year=2017&amp;pages=1-9&amp;pmid=29348964&amp;doi=10.1155/2017/1059678&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref007">
<span class="label">7.</span><cite>Lu R-C, Yang W, Tan L, Sun F-R, Tan M-S, Zhang W, et al. Association of HLA-DRB1 polymorphism with Alzheimer’s disease: a replication and meta-analysis. Oncotarget. 2017;8(54):93219–26. doi: 10.18632/oncotarget.21479
</cite> [<a href="https://doi.org/10.18632/oncotarget.21479" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5696257/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29190991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Association%20of%20HLA-DRB1%20polymorphism%20with%20Alzheimer%E2%80%99s%20disease:%20a%20replication%20and%20meta-analysis&amp;author=R-C%20Lu&amp;author=W%20Yang&amp;author=L%20Tan&amp;author=F-R%20Sun&amp;author=M-S%20Tan&amp;volume=8&amp;issue=54&amp;publication_year=2017&amp;pages=93219-26&amp;pmid=29190991&amp;doi=10.18632/oncotarget.21479&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref008">
<span class="label">8.</span><cite>Alsaedi SB, Mineta K, Gao X, Gojobori T. Computational network analysis of host genetic risk variants of severe COVID-19. Hum Genom. 2023;17(1):17.</cite> [<a href="https://doi.org/10.1186/s40246-023-00454-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9977643/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36859360/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Hum%20Genom&amp;title=Computational%20network%20analysis%20of%20host%20genetic%20risk%20variants%20of%20severe%20COVID-19&amp;author=SB%20Alsaedi&amp;author=K%20Mineta&amp;author=X%20Gao&amp;author=T%20Gojobori&amp;volume=17&amp;issue=1&amp;publication_year=2023&amp;pages=17&amp;pmid=36859360&amp;doi=10.1186/s40246-023-00454-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref009">
<span class="label">9.</span><cite>van Middendorp H, Evers AWM. The role of psychological factors in inflammatory rheumatic diseases: From burden to tailored treatment. Best Pract Res Clin Rheumatol. 2016;30(5):932–45. doi: 10.1016/j.berh.2016.10.012
</cite> [<a href="https://doi.org/10.1016/j.berh.2016.10.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27964797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Best%20Pract%20Res%20Clin%20Rheumatol&amp;title=The%20role%20of%20psychological%20factors%20in%20inflammatory%20rheumatic%20diseases:%20From%20burden%20to%20tailored%20treatment&amp;author=H%20van%20Middendorp&amp;author=AWM%20Evers&amp;volume=30&amp;issue=5&amp;publication_year=2016&amp;pages=932-45&amp;pmid=27964797&amp;doi=10.1016/j.berh.2016.10.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref010">
<span class="label">10.</span><cite>Romao VC, Fonseca JE. Etiology and risk factors for rheumatoid arthritis: a state-of-the-art review. Front Med. 2021;8:689698.</cite> [<a href="https://doi.org/10.3389/fmed.2021.689698" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8661097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34901047/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Med&amp;title=Etiology%20and%20risk%20factors%20for%20rheumatoid%20arthritis:%20a%20state-of-the-art%20review&amp;author=VC%20Romao&amp;author=JE%20Fonseca&amp;volume=8&amp;publication_year=2021&amp;pages=689698&amp;pmid=34901047&amp;doi=10.3389/fmed.2021.689698&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref011">
<span class="label">11.</span><cite>Guo Q, Wang Y, Xu D, Nossent J, Pavlos NJ, Xu J. Rheumatoid arthritis: pathological mechanisms and modern pharmacologic therapies. Bone Res. 2018;6(1):15. doi: 10.1038/s41413-018-0016-9
</cite> [<a href="https://doi.org/10.1038/s41413-018-0016-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5920070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29736302/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bone%20Res&amp;title=Rheumatoid%20arthritis:%20pathological%20mechanisms%20and%20modern%20pharmacologic%20therapies&amp;author=Q%20Guo&amp;author=Y%20Wang&amp;author=D%20Xu&amp;author=J%20Nossent&amp;author=NJ%20Pavlos&amp;volume=6&amp;issue=1&amp;publication_year=2018&amp;pages=15&amp;pmid=29736302&amp;doi=10.1038/s41413-018-0016-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref012">
<span class="label">12.</span><cite>Ishikawa Y, Terao C. The impact of cigarette smoking on risk of rheumatoid arthritis: a narrative review. Cells. 2020;9(2):475. doi: 10.3390/cells9020475
</cite> [<a href="https://doi.org/10.3390/cells9020475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7072747/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32092988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=The%20impact%20of%20cigarette%20smoking%20on%20risk%20of%20rheumatoid%20arthritis:%20a%20narrative%20review&amp;author=Y%20Ishikawa&amp;author=C%20Terao&amp;volume=9&amp;issue=2&amp;publication_year=2020&amp;pages=475&amp;pmid=32092988&amp;doi=10.3390/cells9020475&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref013">
<span class="label">13.</span><cite>Suzuki A, Yamada R, Chang X, Tokuhiro S, Sawada T, Suzuki M, et al. Functional haplotypes of PADI4, encoding citrullinating enzyme peptidylarginine deiminase 4, are associated with rheumatoid arthritis. Nat Genet. 2003;34(4):395–402. doi: 10.1038/ng1206
</cite> [<a href="https://doi.org/10.1038/ng1206" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12833157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Functional%20haplotypes%20of%20PADI4,%20encoding%20citrullinating%20enzyme%20peptidylarginine%20deiminase%204,%20are%20associated%20with%20rheumatoid%20arthritis&amp;author=A%20Suzuki&amp;author=R%20Yamada&amp;author=X%20Chang&amp;author=S%20Tokuhiro&amp;author=T%20Sawada&amp;volume=34&amp;issue=4&amp;publication_year=2003&amp;pages=395-402&amp;pmid=12833157&amp;doi=10.1038/ng1206&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref014">
<span class="label">14.</span><cite>Bang S, Lee K, Cho S, Lee H, Lee KW, Bae S. Smoking increases rheumatoid arthritis susceptibility in individuals carrying the HLA–DRB1 shared epitope, regardless of rheumatoid factor or anti–cyclic citrullinated peptide antibody status. Arthritis Rheumatism. 2010;62(2):369–77. doi: 10.1002/art.27272
</cite> [<a href="https://doi.org/10.1002/art.27272" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20112396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheumatism&amp;title=Smoking%20increases%20rheumatoid%20arthritis%20susceptibility%20in%20individuals%20carrying%20the%20HLA%E2%80%93DRB1%20shared%20epitope,%20regardless%20of%20rheumatoid%20factor%20or%20anti%E2%80%93cyclic%20citrullinated%20peptide%20antibody%20status&amp;author=S%20Bang&amp;author=K%20Lee&amp;author=S%20Cho&amp;author=H%20Lee&amp;author=KW%20Lee&amp;volume=62&amp;issue=2&amp;publication_year=2010&amp;pages=369-77&amp;pmid=20112396&amp;doi=10.1002/art.27272&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref015">
<span class="label">15.</span><cite>Mankia K, Cheng Z, Do T, Hunt L, Meade J, Kang J, et al. Prevalence of periodontal disease and periodontopathic bacteria in anti–cyclic citrullinated protein antibody–positive at-risk adults without arthritis. JAMA Netw Open. 2019;2(6):e195394. doi: 10.1001/jamanetworkopen.2019.5394</cite> [<a href="https://doi.org/10.1001/jamanetworkopen.2019.5394" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6563551/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31173126/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JAMA%20Netw%20Open&amp;title=Prevalence%20of%20periodontal%20disease%20and%20periodontopathic%20bacteria%20in%20anti%E2%80%93cyclic%20citrullinated%20protein%20antibody%E2%80%93positive%20at-risk%20adults%20without%20arthritis&amp;author=K%20Mankia&amp;author=Z%20Cheng&amp;author=T%20Do&amp;author=L%20Hunt&amp;author=J%20Meade&amp;volume=2&amp;issue=6&amp;publication_year=2019&amp;pmid=31173126&amp;doi=10.1001/jamanetworkopen.2019.5394&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref016">
<span class="label">16.</span><cite>Mikhaylenko DS, Nemtsova MV, Bure IV, Kuznetsova EB, Alekseeva EA, Tarasov VV, et al. Genetic polymorphisms associated with rheumatoid arthritis development and antirheumatic therapy response. IJMS. 2020;21(14):4911. doi: 10.3390/ijms21144911
</cite> [<a href="https://doi.org/10.3390/ijms21144911" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7402327/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32664585/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IJMS&amp;title=Genetic%20polymorphisms%20associated%20with%20rheumatoid%20arthritis%20development%20and%20antirheumatic%20therapy%20response&amp;author=DS%20Mikhaylenko&amp;author=MV%20Nemtsova&amp;author=IV%20Bure&amp;author=EB%20Kuznetsova&amp;author=EA%20Alekseeva&amp;volume=21&amp;issue=14&amp;publication_year=2020&amp;pages=4911&amp;pmid=32664585&amp;doi=10.3390/ijms21144911&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref017">
<span class="label">17.</span><cite>Ding Q, Hu W, Wang R, Yang Q, Zhu M, Li M, et al. Signaling pathways in rheumatoid arthritis: implications for targeted therapy. Sig Transduct Target Ther. 2023;8(1):68.doi: 10.1038/s41392-023-01331-9</cite> [<a href="https://doi.org/10.1038/s41392-023-01331-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9935929/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36797236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sig%20Transduct%20Target%20Ther&amp;title=Signaling%20pathways%20in%20rheumatoid%20arthritis:%20implications%20for%20targeted%20therapy&amp;author=Q%20Ding&amp;author=W%20Hu&amp;author=R%20Wang&amp;author=Q%20Yang&amp;author=M%20Zhu&amp;volume=8&amp;issue=1&amp;publication_year=2023&amp;pages=68&amp;pmid=36797236&amp;doi=10.1038/s41392-023-01331-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref018">
<span class="label">18.</span><cite>Barrie W, Yang Y, Irving-Pease EK, Attfield KE, Scorrano G, Jensen LT, et al. Elevated genetic risk for multiple sclerosis emerged in steppe pastoralist populations. Nature. 2024;625(7994):321–8. doi: 10.1038/s41586-023-06618-z
</cite> [<a href="https://doi.org/10.1038/s41586-023-06618-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10781639/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38200296/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Elevated%20genetic%20risk%20for%20multiple%20sclerosis%20emerged%20in%20steppe%20pastoralist%20populations&amp;author=W%20Barrie&amp;author=Y%20Yang&amp;author=EK%20Irving-Pease&amp;author=KE%20Attfield&amp;author=G%20Scorrano&amp;volume=625&amp;issue=7994&amp;publication_year=2024&amp;pages=321-8&amp;pmid=38200296&amp;doi=10.1038/s41586-023-06618-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref019">
<span class="label">19.</span><cite>Ishigaki K, Sakaue S, Terao C, Luo Y, Sonehara K, Yamaguchi K, et al. Multi-ancestry genome-wide association analyses identify novel genetic mechanisms in rheumatoid arthritis. Nat Genet. 2022;54(11):1640–51. doi: 10.1038/s41588-022-01213-w
</cite> [<a href="https://doi.org/10.1038/s41588-022-01213-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10165422/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36333501/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Genet&amp;title=Multi-ancestry%20genome-wide%20association%20analyses%20identify%20novel%20genetic%20mechanisms%20in%20rheumatoid%20arthritis&amp;author=K%20Ishigaki&amp;author=S%20Sakaue&amp;author=C%20Terao&amp;author=Y%20Luo&amp;author=K%20Sonehara&amp;volume=54&amp;issue=11&amp;publication_year=2022&amp;pages=1640-51&amp;pmid=36333501&amp;doi=10.1038/s41588-022-01213-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref020">
<span class="label">20.</span><cite>Demela P, Pirastu N, Soskic B. Cross-disorder genetic analysis of immune diseases reveals distinct gene associations that converge on common pathways. Nat Commun. 2023;14(1):2743. doi: 10.1038/s41467-023-38389-6
</cite> [<a href="https://doi.org/10.1038/s41467-023-38389-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10182075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37173304/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Cross-disorder%20genetic%20analysis%20of%20immune%20diseases%20reveals%20distinct%20gene%20associations%20that%20converge%20on%20common%20pathways&amp;author=P%20Demela&amp;author=N%20Pirastu&amp;author=B%20Soskic&amp;volume=14&amp;issue=1&amp;publication_year=2023&amp;pages=2743&amp;pmid=37173304&amp;doi=10.1038/s41467-023-38389-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref021">
<span class="label">21.</span><cite>Daghestani M, Othman N, Omair MA, Alenzi F, Omair MA, Alqurtas E, et al. Single nucleotide polymorphisms associated with rheumatoid arthritis in saudi patients. JCM. 2023;12(15):4944. doi: 10.3390/jcm12154944
</cite> [<a href="https://doi.org/10.3390/jcm12154944" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10419658/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37568346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JCM&amp;title=Single%20nucleotide%20polymorphisms%20associated%20with%20rheumatoid%20arthritis%20in%20saudi%20patients&amp;author=M%20Daghestani&amp;author=N%20Othman&amp;author=MA%20Omair&amp;author=F%20Alenzi&amp;author=MA%20Omair&amp;volume=12&amp;issue=15&amp;publication_year=2023&amp;pages=4944&amp;pmid=37568346&amp;doi=10.3390/jcm12154944&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref022">
<span class="label">22.</span><cite>Ciesielski TH, Sirugo G, Iyengar SK, Williams SM. Characterizing the pathogenicity of genetic variants: the consequences of context. NPJ Genom Med. 2024;9(1):3.doi: 10.1038/s41525-023-00386-5
</cite> [<a href="https://doi.org/10.1038/s41525-023-00386-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10776585/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38195641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NPJ%20Genom%20Med&amp;title=Characterizing%20the%20pathogenicity%20of%20genetic%20variants:%20the%20consequences%20of%20context&amp;author=TH%20Ciesielski&amp;author=G%20Sirugo&amp;author=SK%20Iyengar&amp;author=SM%20Williams&amp;volume=9&amp;issue=1&amp;publication_year=2024&amp;pages=3&amp;pmid=38195641&amp;doi=10.1038/s41525-023-00386-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref023">
<span class="label">23.</span><cite>Xiao L, Xiao W, Zhan F. Integrative analyses of biomarkers and potential therapeutic drugs for rheumatoid arthritis. Annals Clin Lab Sci. 2022;52(1):141–53.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/35181628/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20Clin%20Lab%20Sci&amp;title=Integrative%20analyses%20of%20biomarkers%20and%20potential%20therapeutic%20drugs%20for%20rheumatoid%20arthritis&amp;author=L%20Xiao&amp;author=W%20Xiao&amp;author=F%20Zhan&amp;volume=52&amp;issue=1&amp;publication_year=2022&amp;pages=141-53&amp;pmid=35181628&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref024">
<span class="label">24.</span><cite>Shen C, Gu B, Tang M, Ma K, Du K, Wu J, et al. Biomarkers identification of early rheumatoid arthritis via bioinformatics approach and experimental verification. Annals Clin Lab Sci. 2024;54(5):661–70.</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/39537231/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annals%20Clin%20Lab%20Sci&amp;title=Biomarkers%20identification%20of%20early%20rheumatoid%20arthritis%20via%20bioinformatics%20approach%20and%20experimental%20verification&amp;author=C%20Shen&amp;author=B%20Gu&amp;author=M%20Tang&amp;author=K%20Ma&amp;author=K%20Du&amp;volume=54&amp;issue=5&amp;publication_year=2024&amp;pages=661-70&amp;pmid=39537231&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref025">
<span class="label">25.</span><cite>Ainsworth RI, Hammaker D, Nygaard G, Ansalone C, Machado C, Zhang K, et al. Systems-biology analysis of rheumatoid arthritis fibroblast-like synoviocytes implicates cell line-specific transcription factor function. Nat Commun. 2022;13(1):6221. doi: 10.1038/s41467-022-33785-w
</cite> [<a href="https://doi.org/10.1038/s41467-022-33785-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9584907/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36266270/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Systems-biology%20analysis%20of%20rheumatoid%20arthritis%20fibroblast-like%20synoviocytes%20implicates%20cell%20line-specific%20transcription%20factor%20function&amp;author=RI%20Ainsworth&amp;author=D%20Hammaker&amp;author=G%20Nygaard&amp;author=C%20Ansalone&amp;author=C%20Machado&amp;volume=13&amp;issue=1&amp;publication_year=2022&amp;pages=6221&amp;pmid=36266270&amp;doi=10.1038/s41467-022-33785-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref026">
<span class="label">26.</span><cite>Shan F, Ahmad I, Rashid MI. In-silico functional and molecular phenotypic analysis of NFKBIE (V194A), and PADI4 (G55S, V82A, G112A) gene mutations in rheumatoid arthritis. J Proteins Proteomics. 2024; 1–24.</cite>
</li>
<li id="pone.0329101.ref027">
<span class="label">27.</span><cite>Panga V, Raghunathan S. A cytokine protein-protein interaction network for identifying key molecules in rheumatoid arthritis. PLoS ONE. 2018;13(6):e0199530. doi: 10.1371/journal.pone.0199530</cite> [<a href="https://doi.org/10.1371/journal.pone.0199530" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6013252/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29928007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=A%20cytokine%20protein-protein%20interaction%20network%20for%20identifying%20key%20molecules%20in%20rheumatoid%20arthritis&amp;author=V%20Panga&amp;author=S%20Raghunathan&amp;volume=13&amp;issue=6&amp;publication_year=2018&amp;pmid=29928007&amp;doi=10.1371/journal.pone.0199530&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref028">
<span class="label">28.</span><cite>Bolouri H, Young M, Beilke J, Johnson R, Fox B, Huang L, et al. Integrative network modeling reveals mechanisms underlying T cell exhaustion. Sci Rep. 2020;10(1). doi: 10.1038/s41598-020-58600-8</cite> [<a href="https://doi.org/10.1038/s41598-020-58600-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7002445/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32024856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Integrative%20network%20modeling%20reveals%20mechanisms%20underlying%20T%20cell%20exhaustion&amp;author=H%20Bolouri&amp;author=M%20Young&amp;author=J%20Beilke&amp;author=R%20Johnson&amp;author=B%20Fox&amp;volume=10&amp;issue=1&amp;publication_year=2020&amp;pmid=32024856&amp;doi=10.1038/s41598-020-58600-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref029">
<span class="label">29.</span><cite>Zeberg H, Pääbo S. The major genetic risk factor for severe COVID-19 is inherited from Neanderthals. Nature. 2020;587(7835):610–2. doi: 10.1038/s41586-020-2818-3
</cite> [<a href="https://doi.org/10.1038/s41586-020-2818-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32998156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20major%20genetic%20risk%20factor%20for%20severe%20COVID-19%20is%20inherited%20from%20Neanderthals&amp;author=H%20Zeberg&amp;author=S%20P%C3%A4%C3%A4bo&amp;volume=587&amp;issue=7835&amp;publication_year=2020&amp;pages=610-2&amp;pmid=32998156&amp;doi=10.1038/s41586-020-2818-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref030">
<span class="label">30.</span><cite>Powell KL, Stephens SR, Stephens AS. Cardiovascular risk factor mediation of the effects of education and genetic risk score on cardiovascular disease: a prospective observational cohort study of the Framingham Heart Study. BMJ Open. 2021;11(1):e045210. doi: 10.1136/bmjopen-2020-045210</cite> [<a href="https://doi.org/10.1136/bmjopen-2020-045210" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7805364/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33436477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMJ%20Open&amp;title=Cardiovascular%20risk%20factor%20mediation%20of%20the%20effects%20of%20education%20and%20genetic%20risk%20score%20on%20cardiovascular%20disease:%20a%20prospective%20observational%20cohort%20study%20of%20the%20Framingham%20Heart%20Study&amp;author=KL%20Powell&amp;author=SR%20Stephens&amp;author=AS%20Stephens&amp;volume=11&amp;issue=1&amp;publication_year=2021&amp;pmid=33436477&amp;doi=10.1136/bmjopen-2020-045210&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref031">
<span class="label">31.</span><cite>Singh V, Kalliolias GD, Ostaszewski M, Veyssiere M, Pilalis E, Gawron P, et al. RA-map: building a state-of-the-art interactive knowledge base for rheumatoid arthritis. Database. 2020;2020. doi: 10.1093/database/baaa017</cite> [<a href="https://doi.org/10.1093/database/baaa017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7170216/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32311035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Database&amp;title=RA-map:%20building%20a%20state-of-the-art%20interactive%20knowledge%20base%20for%20rheumatoid%20arthritis&amp;author=V%20Singh&amp;author=GD%20Kalliolias&amp;author=M%20Ostaszewski&amp;author=M%20Veyssiere&amp;author=E%20Pilalis&amp;volume=2020&amp;publication_year=2020&amp;pmid=32311035&amp;doi=10.1093/database/baaa017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref032">
<span class="label">32.</span><cite>Deane KD, Demoruelle MK, Kelmenson LB, Kuhn KA, Norris JM, Holers VM. Genetic and environmental risk factors for rheumatoid arthritis. Best Pract Res Clin Rheumatol. 2017;31(1):3–18. doi: 10.1016/j.berh.2017.08.003
</cite> [<a href="https://doi.org/10.1016/j.berh.2017.08.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5726551/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29221595/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Best%20Pract%20Res%20Clin%20Rheumatol&amp;title=Genetic%20and%20environmental%20risk%20factors%20for%20rheumatoid%20arthritis&amp;author=KD%20Deane&amp;author=MK%20Demoruelle&amp;author=LB%20Kelmenson&amp;author=KA%20Kuhn&amp;author=JM%20Norris&amp;volume=31&amp;issue=1&amp;publication_year=2017&amp;pages=3-18&amp;pmid=29221595&amp;doi=10.1016/j.berh.2017.08.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref033">
<span class="label">33.</span><cite>Radu A-F, Bungau SG. Management of rheumatoid arthritis: an overview. Cells. 2021;10(11):2857. doi: 10.3390/cells10112857
</cite> [<a href="https://doi.org/10.3390/cells10112857" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8616326/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34831081/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cells&amp;title=Management%20of%20rheumatoid%20arthritis:%20an%20overview&amp;author=A-F%20Radu&amp;author=SG%20Bungau&amp;volume=10&amp;issue=11&amp;publication_year=2021&amp;pages=2857&amp;pmid=34831081&amp;doi=10.3390/cells10112857&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref034">
<span class="label">34.</span><cite>Arleevskaya M, Takha E, Petrov S, Kazarian G, Novikov A, Larionova R, et al. Causal risk and protective factors in rheumatoid arthritis: a genetic update. J Transl Autoimmun. 2021;4:100119. doi: 10.1016/j.jtauto.2021.100119
</cite> [<a href="https://doi.org/10.1016/j.jtauto.2021.100119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8424591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34522877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Transl%20Autoimmun&amp;title=Causal%20risk%20and%20protective%20factors%20in%20rheumatoid%20arthritis:%20a%20genetic%20update&amp;author=M%20Arleevskaya&amp;author=E%20Takha&amp;author=S%20Petrov&amp;author=G%20Kazarian&amp;author=A%20Novikov&amp;volume=4&amp;publication_year=2021&amp;pages=100119&amp;pmid=34522877&amp;doi=10.1016/j.jtauto.2021.100119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref035">
<span class="label">35.</span><cite>Wei CH, Allot A, Lai PT, Leaman R, Tian S, Luo L. PubTator 3.0: an AI-powered literature resource for unlocking biomedical knowledge. Nucl Acids Res. 2024:gkae235.</cite> [<a href="https://doi.org/10.1093/nar/gkae235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11223843/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38572754/" class="usa-link">PubMed</a>]</li>
<li id="pone.0329101.ref036">
<span class="label">36.</span><cite>Sherry ST, Ward M, Sirotkin K. dbSNP—database for single nucleotide polymorphisms and other classes of minor genetic variation. Genome Res. 1999;9(8):677–9. doi: 10.1101/gr.9.8.677
</cite> [<a href="https://doi.org/10.1101/gr.9.8.677" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10447503/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=dbSNP%E2%80%94database%20for%20single%20nucleotide%20polymorphisms%20and%20other%20classes%20of%20minor%20genetic%20variation&amp;author=ST%20Sherry&amp;author=M%20Ward&amp;author=K%20Sirotkin&amp;volume=9&amp;issue=8&amp;publication_year=1999&amp;pages=677-9&amp;pmid=10447503&amp;doi=10.1101/gr.9.8.677&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref037">
<span class="label">37.</span><cite>Xin J, Mark A, Afrasiabi C, Tsueng G, Juchler M, Gopal N. High-performance web services for querying gene and variant annotation. Genome Biol. 2016;17:1–7.
</cite> [<a href="https://doi.org/10.1186/s13059-016-0953-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4858870/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27154141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=High-performance%20web%20services%20for%20querying%20gene%20and%20variant%20annotation&amp;author=J%20Xin&amp;author=A%20Mark&amp;author=C%20Afrasiabi&amp;author=G%20Tsueng&amp;author=M%20Juchler&amp;volume=17&amp;publication_year=2016&amp;pages=1-7&amp;pmid=27154141&amp;doi=10.1186/s13059-016-0953-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref038">
<span class="label">38.</span><cite>Stelzer G, Rosen N, Plaschkes I, Zimmerman S, Twik M, Fishilevich S, et al. The GeneCards suite: from gene data mining to disease genome sequence analyses. CP Bioinformatics. 2016;54(1):1–30. doi: 10.1002/cpbi.5</cite> [<a href="https://doi.org/10.1002/cpbi.5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27322403/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=CP%20Bioinformatics&amp;title=The%20GeneCards%20suite:%20from%20gene%20data%20mining%20to%20disease%20genome%20sequence%20analyses&amp;author=G%20Stelzer&amp;author=N%20Rosen&amp;author=I%20Plaschkes&amp;author=S%20Zimmerman&amp;author=M%20Twik&amp;volume=54&amp;issue=1&amp;publication_year=2016&amp;pages=1-30&amp;pmid=27322403&amp;doi=10.1002/cpbi.5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref039">
<span class="label">39.</span><cite>Martin FJ, Amode MR, Aneja A, Austine-Orimoloye O, Azov AG, Barnes I, et al. Ensembl 2023 . Nucl Acids Res. 2023;51(D1):D933–41.</cite> [<a href="https://doi.org/10.1093/nar/gkac958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825606/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36318249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucl%20Acids%20Res&amp;title=Ensembl%202023&amp;author=FJ%20Martin&amp;author=MR%20Amode&amp;author=A%20Aneja&amp;author=O%20Austine-Orimoloye&amp;author=AG%20Azov&amp;volume=51&amp;publication_year=2023&amp;pmid=36318249&amp;doi=10.1093/nar/gkac958&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref040">
<span class="label">40.</span><cite>Klopfenstein DV, Zhang L, Pedersen BS, Ramírez F, Warwick Vesztrocy A, Naldi A, et al. GOATOOLS: a python library for gene ontology analyses. Sci Rep. 2018;8(1). doi: 10.1038/s41598-018-28948-z</cite> [<a href="https://doi.org/10.1038/s41598-018-28948-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6052049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30022098/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=GOATOOLS:%20a%20python%20library%20for%20gene%20ontology%20analyses&amp;author=DV%20Klopfenstein&amp;author=L%20Zhang&amp;author=BS%20Pedersen&amp;author=F%20Ram%C3%ADrez&amp;author=A%20Warwick%20Vesztrocy&amp;volume=8&amp;issue=1&amp;publication_year=2018&amp;pmid=30022098&amp;doi=10.1038/s41598-018-28948-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref041">
<span class="label">41.</span><cite>Landrum MJ, Lee JM, Benson M, Brown GR, Chao C, Chitipiralla S, et al. ClinVar: improving access to variant interpretations and supporting evidence. Nucl Acids Res. 2017;46(D1):D1062–7. doi: 10.1093/nar/gkx1153</cite> [<a href="https://doi.org/10.1093/nar/gkx1153" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5753237/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29165669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucl%20Acids%20Res&amp;title=ClinVar:%20improving%20access%20to%20variant%20interpretations%20and%20supporting%20evidence&amp;author=MJ%20Landrum&amp;author=JM%20Lee&amp;author=M%20Benson&amp;author=GR%20Brown&amp;author=C%20Chao&amp;volume=46&amp;publication_year=2017&amp;pmid=29165669&amp;doi=10.1093/nar/gkx1153&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref042">
<span class="label">42.</span><cite>Dai Y, Hu R, Liu A, Cho KS, Manuel AM, Li X, et al. WebCSEA: web-based cell-type-specific enrichment analysis of genes. Nucl Acids Res. 2022;50(W1):W782–90. doi: 10.1093/nar/gkac392</cite> [<a href="https://doi.org/10.1093/nar/gkac392" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10359109/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35610053/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucl%20Acids%20Res&amp;title=WebCSEA:%20web-based%20cell-type-specific%20enrichment%20analysis%20of%20genes&amp;author=Y%20Dai&amp;author=R%20Hu&amp;author=A%20Liu&amp;author=KS%20Cho&amp;author=AM%20Manuel&amp;volume=50&amp;publication_year=2022&amp;pmid=35610053&amp;doi=10.1093/nar/gkac392&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref043">
<span class="label">43.</span><cite>Benjamini Y, Hochberg Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J Roy Statist Soc Ser B: Statist Methodol. 1995;57(1):289–300. doi: 10.1111/j.2517-6161.1995.tb02031.x</cite> [<a href="https://doi.org/10.1111/j.2517-6161.1995.tb02031.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Roy%20Statist%20Soc%20Ser%20B:%20Statist%20Methodol&amp;title=Controlling%20the%20false%20discovery%20rate:%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&amp;author=Y%20Benjamini&amp;author=Y%20Hochberg&amp;volume=57&amp;issue=1&amp;publication_year=1995&amp;pages=289-300&amp;doi=10.1111/j.2517-6161.1995.tb02031.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref044">
<span class="label">44.</span><cite>Subramanian A, Tamayo P, Mootha VK, Mukherjee S, Ebert BL, Gillette MA, et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. Proc Natl Acad Sci USA. 2005;102(43):15545–50. doi: 10.1073/pnas.0506580102
</cite> [<a href="https://doi.org/10.1073/pnas.0506580102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1239896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16199517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20USA&amp;title=Gene%20set%20enrichment%20analysis:%20a%20knowledge-based%20approach%20for%20interpreting%20genome-wide%20expression%20profiles&amp;author=A%20Subramanian&amp;author=P%20Tamayo&amp;author=VK%20Mootha&amp;author=S%20Mukherjee&amp;author=BL%20Ebert&amp;volume=102&amp;issue=43&amp;publication_year=2005&amp;pages=15545-50&amp;pmid=16199517&amp;doi=10.1073/pnas.0506580102&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref045">
<span class="label">45.</span><cite>Kohler S, Gargano M, Matentzoglu N, Carmody LC, Lewis-Smith D, Vasilevsky NA, et al. The human phenotype ontology in 2021 . Nucl Acids Res. 2020;49(D1):D1207–17. doi: 10.1093/nar/gkaa1043</cite> [<a href="https://doi.org/10.1093/nar/gkaa1043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7778952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33264411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucl%20Acids%20Res&amp;title=The%20human%20phenotype%20ontology%20in%202021&amp;author=S%20Kohler&amp;author=M%20Gargano&amp;author=N%20Matentzoglu&amp;author=LC%20Carmody&amp;author=D%20Lewis-Smith&amp;volume=49&amp;publication_year=2020&amp;pmid=33264411&amp;doi=10.1093/nar/gkaa1043&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref046">
<span class="label">46.</span><cite>Amberger JS, Bocchini CA, Scott AF, Hamosh A. OMIM.org: leveraging knowledge across phenotype–gene relationships. Nucl Acids Res. 2018;47(D1):D1038–43. doi: 10.1093/nar/gky1151</cite> [<a href="https://doi.org/10.1093/nar/gky1151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6323937/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30445645/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucl%20Acids%20Res&amp;title=OMIM.org:%20leveraging%20knowledge%20across%20phenotype%E2%80%93gene%20relationships&amp;author=JS%20Amberger&amp;author=CA%20Bocchini&amp;author=AF%20Scott&amp;author=A%20Hamosh&amp;volume=47&amp;publication_year=2018&amp;pmid=30445645&amp;doi=10.1093/nar/gky1151&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref047">
<span class="label">47.</span><cite>Pinero J, Ramırez-Anguita JM, Sauch-Pitarch J, Ronzano F, Centeno E, Sanz F. The DisGeNET knowledge platform for disease genomics: 2019 update. Nucl Acids Res. 2020;48(D1):D845–55.</cite> [<a href="https://doi.org/10.1093/nar/gkz1021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7145631/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31680165/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucl%20Acids%20Res&amp;title=The%20DisGeNET%20knowledge%20platform%20for%20disease%20genomics:%202019%20update&amp;author=J%20Pinero&amp;author=JM%20Ram%C4%B1rez-Anguita&amp;author=J%20Sauch-Pitarch&amp;author=F%20Ronzano&amp;author=E%20Centeno&amp;volume=48&amp;publication_year=2020&amp;pmid=31680165&amp;doi=10.1093/nar/gkz1021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref048">
<span class="label">48.</span><cite>Szklarczyk D, Gable AL, Nastou KC, Lyon D, Kirsch R, Pyysalo S, et al. Correction to ‘The STRING database in 2021 : customizable protein–protein networks, and functional characterization of user-uploaded gene/measurement sets’. Nucl Acids Res. 2021;49(18):10800. doi: 10.1093/nar/gkab835
</cite> [<a href="https://doi.org/10.1093/nar/gkab835" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8501959/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34530444/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucl%20Acids%20Res&amp;title=Correction%20to%20%E2%80%98The%20STRING%20database%20in%202021%20:%20customizable%20protein%E2%80%93protein%20networks,%20and%20functional%20characterization%20of%20user-uploaded%20gene/measurement%20sets%E2%80%99&amp;author=D%20Szklarczyk&amp;author=AL%20Gable&amp;author=KC%20Nastou&amp;author=D%20Lyon&amp;author=R%20Kirsch&amp;volume=49&amp;issue=18&amp;publication_year=2021&amp;pages=10800&amp;pmid=34530444&amp;doi=10.1093/nar/gkab835&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref049">
<span class="label">49.</span><cite>Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, Ramage D, et al. Cytoscape: a software environment for integrated models of biomolecular interaction networks. Genome Res. 2003;13(11):2498–504. doi: 10.1101/gr.1239303
</cite> [<a href="https://doi.org/10.1101/gr.1239303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC403769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14597658/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Res&amp;title=Cytoscape:%20a%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks&amp;author=P%20Shannon&amp;author=A%20Markiel&amp;author=O%20Ozier&amp;author=NS%20Baliga&amp;author=JT%20Wang&amp;volume=13&amp;issue=11&amp;publication_year=2003&amp;pages=2498-504&amp;pmid=14597658&amp;doi=10.1101/gr.1239303&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref050">
<span class="label">50.</span><cite>Croft D, O’Kelly G, Wu G, Haw R, Gillespie M, Matthews L, et al. Reactome: a database of reactions, pathways and biological processes. Nucl Acids Res. 2010;39(Database):D691–7. doi: 10.1093/nar/gkq1018</cite> [<a href="https://doi.org/10.1093/nar/gkq1018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3013646/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21067998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucl%20Acids%20Res&amp;title=Reactome:%20a%20database%20of%20reactions,%20pathways%20and%20biological%20processes&amp;author=D%20Croft&amp;author=G%20O%E2%80%99Kelly&amp;author=G%20Wu&amp;author=R%20Haw&amp;author=M%20Gillespie&amp;volume=39&amp;publication_year=2010&amp;pmid=21067998&amp;doi=10.1093/nar/gkq1018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref051">
<span class="label">51.</span><cite>Kanehisa M. Toward understanding the origin and evolution of cellular organisms. Protein Sci. 2019;28(11):1947–51. doi: 10.1002/pro.3715
</cite> [<a href="https://doi.org/10.1002/pro.3715" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6798127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31441146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=Toward%20understanding%20the%20origin%20and%20evolution%20of%20cellular%20organisms&amp;author=M%20Kanehisa&amp;volume=28&amp;issue=11&amp;publication_year=2019&amp;pages=1947-51&amp;pmid=31441146&amp;doi=10.1002/pro.3715&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref052">
<span class="label">52.</span><cite>Wishart DS, Knox C, Guo AC, Cheng D, Shrivastava S, Tzur D, et al. DrugBank: a knowledgebase for drugs, drug actions and drug targets. Nucl Acids Res. 2007;36(suppl_1):D901–6. doi: 10.1093/nar/gkm958</cite> [<a href="https://doi.org/10.1093/nar/gkm958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2238889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18048412/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucl%20Acids%20Res&amp;title=DrugBank:%20a%20knowledgebase%20for%20drugs,%20drug%20actions%20and%20drug%20targets&amp;author=DS%20Wishart&amp;author=C%20Knox&amp;author=AC%20Guo&amp;author=D%20Cheng&amp;author=S%20Shrivastava&amp;volume=36&amp;publication_year=2007&amp;pmid=18048412&amp;doi=10.1093/nar/gkm958&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref053">
<span class="label">53.</span><cite>Pan J, Zhou L, Zhang C, Xu Q, Sun Y. Targeting protein phosphatases for the treatment of inflammation-related diseases: from signaling to therapy. Signal Transduct Targeted Therapy. 2022;7(1):177.</cite> [<a href="https://doi.org/10.1038/s41392-022-01038-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9166240/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35665742/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduct%20Targeted%20Therapy&amp;title=Targeting%20protein%20phosphatases%20for%20the%20treatment%20of%20inflammation-related%20diseases:%20from%20signaling%20to%20therapy&amp;author=J%20Pan&amp;author=L%20Zhou&amp;author=C%20Zhang&amp;author=Q%20Xu&amp;author=Y%20Sun&amp;volume=7&amp;issue=1&amp;publication_year=2022&amp;pages=177&amp;pmid=35665742&amp;doi=10.1038/s41392-022-01038-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref054">
<span class="label">54.</span><cite>Kuang W, Wang X, Ding J, Li J, Ji M, Chen W. PTPN2, a key predictor of prognosis for pancreatic adenocarcinoma, significantly regulates cell cycles, apoptosis, and metastasis. Front Immunol. 2022;13:805311.
</cite> [<a href="https://doi.org/10.3389/fimmu.2022.805311" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8829144/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35154122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&amp;title=PTPN2,%20a%20key%20predictor%20of%20prognosis%20for%20pancreatic%20adenocarcinoma,%20significantly%20regulates%20cell%20cycles,%20apoptosis,%20and%20metastasis&amp;author=W%20Kuang&amp;author=X%20Wang&amp;author=J%20Ding&amp;author=J%20Li&amp;author=M%20Ji&amp;volume=13&amp;publication_year=2022&amp;pages=805311&amp;pmid=35154122&amp;doi=10.3389/fimmu.2022.805311&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref055">
<span class="label">55.</span><cite>Mustelin T, Bottini N, Stanford SM. The contribution of PTPN22 to rheumatic disease. Arthritis Rheumatol. 2019;71(4):486–95. doi: 10.1002/art.40790
</cite> [<a href="https://doi.org/10.1002/art.40790" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6438733/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30507064/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Arthritis%20Rheumatol&amp;title=The%20contribution%20of%20PTPN22%20to%20rheumatic%20disease&amp;author=T%20Mustelin&amp;author=N%20Bottini&amp;author=SM%20Stanford&amp;volume=71&amp;issue=4&amp;publication_year=2019&amp;pages=486-95&amp;pmid=30507064&amp;doi=10.1002/art.40790&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref056">
<span class="label">56.</span><cite>Sopel N, Graser A, Mousset S, Finotto S. The transcription factor BATF modulates cytokine-mediated responses in T cells. Cytokine Growth Factor Rev. 2016;30:39–45. doi: 10.1016/j.cytogfr.2016.03.004
</cite> [<a href="https://doi.org/10.1016/j.cytogfr.2016.03.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26970726/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cytokine%20Growth%20Factor%20Rev&amp;title=The%20transcription%20factor%20BATF%20modulates%20cytokine-mediated%20responses%20in%20T%20cells&amp;author=N%20Sopel&amp;author=A%20Graser&amp;author=S%20Mousset&amp;author=S%20Finotto&amp;volume=30&amp;publication_year=2016&amp;pages=39-45&amp;pmid=26970726&amp;doi=10.1016/j.cytogfr.2016.03.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref057">
<span class="label">57.</span><cite>Zhang X, Zhang C, Qiao M, Cheng C, Tang N, Lu S, et al. Depletion of BATF in CAR-T cells enhances antitumor activity by inducing resistance against exhaustion and formation of central memory cells. Cancer Cell. 2022;40(11):1407–22.
</cite> [<a href="https://doi.org/10.1016/j.ccell.2022.09.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36240777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=Depletion%20of%20BATF%20in%20CAR-T%20cells%20enhances%20antitumor%20activity%20by%20inducing%20resistance%20against%20exhaustion%20and%20formation%20of%20central%20memory%20cells&amp;author=X%20Zhang&amp;author=C%20Zhang&amp;author=M%20Qiao&amp;author=C%20Cheng&amp;author=N%20Tang&amp;volume=40&amp;issue=11&amp;publication_year=2022&amp;pages=1407-22&amp;pmid=36240777&amp;doi=10.1016/j.ccell.2022.09.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref058">
<span class="label">58.</span><cite>Horta-Baas G, Romero-Figueroa MS, Montiel-Jarquín AJ, Pizano-Zárate ML, García-Mena J, Ramírez-Durán N. Intestinal dysbiosis and rheumatoid arthritis: a link between gut microbiota and the pathogenesis of rheumatoid arthritis. J Immunol Res. 2017;2017(1):4835189. doi: 10.1155/2017/4835189
</cite> [<a href="https://doi.org/10.1155/2017/4835189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5602494/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28948174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Immunol%20Res&amp;title=Intestinal%20dysbiosis%20and%20rheumatoid%20arthritis:%20a%20link%20between%20gut%20microbiota%20and%20the%20pathogenesis%20of%20rheumatoid%20arthritis&amp;author=G%20Horta-Baas&amp;author=MS%20Romero-Figueroa&amp;author=AJ%20Montiel-Jarqu%C3%ADn&amp;author=ML%20Pizano-Z%C3%A1rate&amp;author=J%20Garc%C3%ADa-Mena&amp;volume=2017&amp;issue=1&amp;publication_year=2017&amp;pages=4835189&amp;pmid=28948174&amp;doi=10.1155/2017/4835189&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref059">
<span class="label">59.</span><cite>Sollis E, Mosaku A, Abid A, Buniello A, Cerezo M, Gil L, et al. The NHGRI-EBI GWAS catalog: knowledgebase and deposition resource. Nucleic Acids Res. 2022;51(D1):D977–85. doi: 10.1093/nar/gkac1010</cite> [<a href="https://doi.org/10.1093/nar/gkac1010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9825413/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36350656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20NHGRI-EBI%20GWAS%20catalog:%20knowledgebase%20and%20deposition%20resource&amp;author=E%20Sollis&amp;author=A%20Mosaku&amp;author=A%20Abid&amp;author=A%20Buniello&amp;author=M%20Cerezo&amp;volume=51&amp;publication_year=2022&amp;pmid=36350656&amp;doi=10.1093/nar/gkac1010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref060">
<span class="label">60.</span><cite>Mitchell DC, Kuljanin M, Li J, Van Vranken JG, Bulloch N, Schweppe DK, et al. A proteome-wide atlas of drug mechanism of action. Nat Biotechnol. 2023;41(6):845–57. doi: 10.1038/s41587-022-01539-0
</cite> [<a href="https://doi.org/10.1038/s41587-022-01539-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11069389/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36593396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Biotechnol&amp;title=A%20proteome-wide%20atlas%20of%20drug%20mechanism%20of%20action&amp;author=DC%20Mitchell&amp;author=M%20Kuljanin&amp;author=J%20Li&amp;author=JG%20Van%20Vranken&amp;author=N%20Bulloch&amp;volume=41&amp;issue=6&amp;publication_year=2023&amp;pages=845-57&amp;pmid=36593396&amp;doi=10.1038/s41587-022-01539-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref061">
<span class="label">61.</span><cite>Yaron-Barir TM, Joughin BA, Huntsman EM, Kerelsky A, Cizin DM, Cohen BM, et al. The intrinsic substrate specificity of the human tyrosine kinome. Nature. 2024; p. 1–8.</cite> [<a href="https://doi.org/10.1038/s41586-024-07407-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11136658/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38720073/" class="usa-link">PubMed</a>]</li>
<li id="pone.0329101.ref062">
<span class="label">62.</span><cite>Morris R, Butler L, Perkins A, Kershaw NJ, Babon JJ. The Role of LNK (SH2B3) in the regulation of JAK-STAT signalling in haematopoiesis. Pharmaceuticals. 2021;15(1):24.
</cite> [<a href="https://doi.org/10.3390/ph15010024" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8781068/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35056081/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmaceuticals&amp;title=The%20Role%20of%20LNK%20(SH2B3)%20in%20the%20regulation%20of%20JAK-STAT%20signalling%20in%20haematopoiesis&amp;author=R%20Morris&amp;author=L%20Butler&amp;author=A%20Perkins&amp;author=NJ%20Kershaw&amp;author=JJ%20Babon&amp;volume=15&amp;issue=1&amp;publication_year=2021&amp;pages=24&amp;pmid=35056081&amp;doi=10.3390/ph15010024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref063">
<span class="label">63.</span><cite>Blass G, Mattson DL, Staruschenko A. The function of SH2B3 (LNK) in the kidney. Am J Physiol-Renal Physiol. 2016;311(4):F682–5. doi: 10.1152/ajprenal.00373.2016</cite> [<a href="https://doi.org/10.1152/ajprenal.00373.2016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5142233/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27440780/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am%20J%20Physiol-Renal%20Physiol&amp;title=The%20function%20of%20SH2B3%20(LNK)%20in%20the%20kidney&amp;author=G%20Blass&amp;author=DL%20Mattson&amp;author=A%20Staruschenko&amp;volume=311&amp;issue=4&amp;publication_year=2016&amp;pmid=27440780&amp;doi=10.1152/ajprenal.00373.2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref064">
<span class="label">64.</span><cite>Auphan N, DiDonato JA, Rosette C, Helmberg A, Karin M. Immunosuppression by glucocorticoids: inhibition of NF-κB activity through induction of IκB synthesis. Science. 1995;270(5234):286–90. doi: 10.1126/science.270.5234.286
</cite> [<a href="https://doi.org/10.1126/science.270.5234.286" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7569976/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Immunosuppression%20by%20glucocorticoids:%20inhibition%20of%20NF-%CE%BAB%20activity%20through%20induction%20of%20I%CE%BAB%20synthesis&amp;author=N%20Auphan&amp;author=JA%20DiDonato&amp;author=C%20Rosette&amp;author=A%20Helmberg&amp;author=M%20Karin&amp;volume=270&amp;issue=5234&amp;publication_year=1995&amp;pages=286-90&amp;pmid=7569976&amp;doi=10.1126/science.270.5234.286&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref065">
<span class="label">65.</span><cite>Paimela T, Hyttinen JMT, Viiri J, Ryhanen T, Karjalainen RO, Salminen A, et al. Celastrol regulates innate immunity response via NF-κB and Hsp70 in human retinal pigment epithelial cells. Pharmacol Res. 2011;64(5):501–8. doi: 10.1016/j.phrs.2011.05.027
</cite> [<a href="https://doi.org/10.1016/j.phrs.2011.05.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21683142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Res&amp;title=Celastrol%20regulates%20innate%20immunity%20response%20via%20NF-%CE%BAB%20and%20Hsp70%20in%20human%20retinal%20pigment%20epithelial%20cells&amp;author=T%20Paimela&amp;author=JMT%20Hyttinen&amp;author=J%20Viiri&amp;author=T%20Ryhanen&amp;author=RO%20Karjalainen&amp;volume=64&amp;issue=5&amp;publication_year=2011&amp;pages=501-8&amp;pmid=21683142&amp;doi=10.1016/j.phrs.2011.05.027&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref066">
<span class="label">66.</span><cite>AndrÂ´e F, Ciruelos E, Rubovszky G, Campone M, Loibl S, Rugo HS, et al. Alpelisib for PIK3CA-mutated, hormone receptor–positive advanced breast cancer. New Engl J Med. 2019;380(20):1929–40.
</cite> [<a href="https://doi.org/10.1056/NEJMoa1813904" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31091374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Engl%20J%20Med&amp;title=Alpelisib%20for%20PIK3CA-mutated,%20hormone%20receptor%E2%80%93positive%20advanced%20breast%20cancer&amp;author=F%20Andr%C3%82%C2%B4e&amp;author=E%20Ciruelos&amp;author=G%20Rubovszky&amp;author=M%20Campone&amp;author=S%20Loibl&amp;volume=380&amp;issue=20&amp;publication_year=2019&amp;pages=1929-40&amp;pmid=31091374&amp;doi=10.1056/NEJMoa1813904&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref067">
<span class="label">67.</span><cite>Graff JR, McNulty AM, Hanna KR, Konicek BW, Lynch RL, Bailey SN, et al. The protein kinase C<em>β</em>–selective inhibitor, enzastaurin (LY317615.HCl), suppresses signaling through the AKT pathway, induces apoptosis, and suppresses growth of human colon cancer and glioblastoma xenografts. Cancer Res. 2005;65(16):7462–9. doi: 10.1158/0008-5472.can-05-0071
</cite> [<a href="https://doi.org/10.1158/0008-5472.can-05-0071" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16103100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=The%20protein%20kinase%20C%CE%B2%E2%80%93selective%20inhibitor,%20enzastaurin%20(LY317615.HCl),%20suppresses%20signaling%20through%20the%20AKT%20pathway,%20induces%20apoptosis,%20and%20suppresses%20growth%20of%20human%20colon%20cancer%20and%20glioblastoma%20xenografts&amp;author=JR%20Graff&amp;author=AM%20McNulty&amp;author=KR%20Hanna&amp;author=BW%20Konicek&amp;author=RL%20Lynch&amp;volume=65&amp;issue=16&amp;publication_year=2005&amp;pages=7462-9&amp;pmid=16103100&amp;doi=10.1158/0008-5472.can-05-0071&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref068">
<span class="label">68.</span><cite>Jie Y, Liu G, E M, Li Y, Xu G, Guo J, et al. Novel small molecule inhibitors of the transcription factor ETS-1 and their antitumor activity against hepatocellular carcinoma. Eur J Pharmacol. 2021;906:174214. doi: 10.1016/j.ejphar.2021.174214
</cite> [<a href="https://doi.org/10.1016/j.ejphar.2021.174214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34116044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Pharmacol&amp;title=Novel%20small%20molecule%20inhibitors%20of%20the%20transcription%20factor%20ETS-1%20and%20their%20antitumor%20activity%20against%20hepatocellular%20carcinoma&amp;author=Y%20Jie&amp;author=G%20Liu&amp;author=M%20E&amp;author=Y%20Li&amp;author=G%20Xu&amp;volume=906&amp;publication_year=2021&amp;pages=174214&amp;pmid=34116044&amp;doi=10.1016/j.ejphar.2021.174214&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref069">
<span class="label">69.</span><cite>Zheng H, Liu Y, Deng Y, Li Y, Liu S, Yang Y, et al. Recent advances of NFATc1 in rheumatoid arthritis-related bone destruction: mechanisms and potential therapeutic targets. Mol Med. 2024;30(1):20.doi: 10.1186/s10020-024-00788-w
</cite> [<a href="https://doi.org/10.1186/s10020-024-00788-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10838448/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38310228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Med&amp;title=Recent%20advances%20of%20NFATc1%20in%20rheumatoid%20arthritis-related%20bone%20destruction:%20mechanisms%20and%20potential%20therapeutic%20targets&amp;author=H%20Zheng&amp;author=Y%20Liu&amp;author=Y%20Deng&amp;author=Y%20Li&amp;author=S%20Liu&amp;volume=30&amp;issue=1&amp;publication_year=2024&amp;pages=20&amp;pmid=38310228&amp;doi=10.1186/s10020-024-00788-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref070">
<span class="label">70.</span><cite>Pappas DJ, Gabatto PA, Oksenberg D, Khankhanian P, Baranzini SE, Gan L, et al. Transcriptional expression patterns triggered by chemically distinct neuroprotective molecules. Neuroscience. 2012;226:10–20. doi: 10.1016/j.neuroscience.2012.09.007
</cite> [<a href="https://doi.org/10.1016/j.neuroscience.2012.09.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3489981/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22986168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Neuroscience&amp;title=Transcriptional%20expression%20patterns%20triggered%20by%20chemically%20distinct%20neuroprotective%20molecules&amp;author=DJ%20Pappas&amp;author=PA%20Gabatto&amp;author=D%20Oksenberg&amp;author=P%20Khankhanian&amp;author=SE%20Baranzini&amp;volume=226&amp;publication_year=2012&amp;pages=10-20&amp;pmid=22986168&amp;doi=10.1016/j.neuroscience.2012.09.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref071">
<span class="label">71.</span><cite>Taylor PC, Matucci Cerinic M, Alten R, Avouac J, Westhovens R. Managing inadequate response to initial anti-TNF therapy in rheumatoid arthritis: optimising treatment outcomes. Therap Adv Musculoskeletal. 2022;14. doi: 10.1177/1759720x221114101</cite> [<a href="https://doi.org/10.1177/1759720x221114101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9386864/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35991524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Therap%20Adv%20Musculoskeletal&amp;title=Managing%20inadequate%20response%20to%20initial%20anti-TNF%20therapy%20in%20rheumatoid%20arthritis:%20optimising%20treatment%20outcomes&amp;author=PC%20Taylor&amp;author=M%20Matucci%20Cerinic&amp;author=R%20Alten&amp;author=J%20Avouac&amp;author=R%20Westhovens&amp;volume=14&amp;publication_year=2022&amp;pmid=35991524&amp;doi=10.1177/1759720x221114101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref072">
<span class="label">72.</span><cite>Hu Z, Viswanathan R, Cheng H, Chen J, Yang X, Huynh A, et al. Inhibiting WEE1 and IKK-RELA crosstalk overcomes TNF<em>α</em> resistance in head and neck cancers. Molecul Cancer Res. 2022;20(6):867–82. doi: 10.1158/1541-7786.mcr-21-0624</cite> [<a href="https://doi.org/10.1158/1541-7786.mcr-21-0624" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9177594/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35176168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecul%20Cancer%20Res&amp;title=Inhibiting%20WEE1%20and%20IKK-RELA%20crosstalk%20overcomes%20TNF%CE%B1%20resistance%20in%20head%20and%20neck%20cancers&amp;author=Z%20Hu&amp;author=R%20Viswanathan&amp;author=H%20Cheng&amp;author=J%20Chen&amp;author=X%20Yang&amp;volume=20&amp;issue=6&amp;publication_year=2022&amp;pages=867-82&amp;pmid=35176168&amp;doi=10.1158/1541-7786.mcr-21-0624&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref073">
<span class="label">73.</span><cite>Schraml BU, Hildner K, Ise W, Lee W-L, Smith WA-E, Solomon B, et al. The AP-1 transcription factor Batf controls TH17 differentiation. Nature. 2009;460(7253):405–9. doi: 10.1038/nature08114
</cite> [<a href="https://doi.org/10.1038/nature08114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2716014/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19578362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20AP-1%20transcription%20factor%20Batf%20controls%20TH17%20differentiation&amp;author=BU%20Schraml&amp;author=K%20Hildner&amp;author=W%20Ise&amp;author=W-L%20Lee&amp;author=WA-E%20Smith&amp;volume=460&amp;issue=7253&amp;publication_year=2009&amp;pages=405-9&amp;pmid=19578362&amp;doi=10.1038/nature08114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref074">
<span class="label">74.</span><cite>Murphy TL, Tussiwand R, Murphy KM. Specificity through cooperation: BATF–IRF interactions control immune-regulatory networks. Nat Rev Immunol. 2013;13(7):499–509. doi: 10.1038/nri3470
</cite> [<a href="https://doi.org/10.1038/nri3470" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23787991/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Immunol&amp;title=Specificity%20through%20cooperation:%20BATF%E2%80%93IRF%20interactions%20control%20immune-regulatory%20networks&amp;author=TL%20Murphy&amp;author=R%20Tussiwand&amp;author=KM%20Murphy&amp;volume=13&amp;issue=7&amp;publication_year=2013&amp;pages=499-509&amp;pmid=23787991&amp;doi=10.1038/nri3470&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref075">
<span class="label">75.</span><cite>Liu Y, Wang D, Li Z, Li X, Jin M, Jia N, et al. Pan-cancer analysis on the role of PIK3R1 and PIK3R2 in human tumors. Sci Rep. 2022;12(1):5924.doi: 10.1038/s41598-022-09889-0
</cite> [<a href="https://doi.org/10.1038/s41598-022-09889-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8993854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35395865/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Pan-cancer%20analysis%20on%20the%20role%20of%20PIK3R1%20and%20PIK3R2%20in%20human%20tumors&amp;author=Y%20Liu&amp;author=D%20Wang&amp;author=Z%20Li&amp;author=X%20Li&amp;author=M%20Jin&amp;volume=12&amp;issue=1&amp;publication_year=2022&amp;pages=5924&amp;pmid=35395865&amp;doi=10.1038/s41598-022-09889-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref076">
<span class="label">76.</span><cite>Karpouzas GA, Ormseth SR, Hernandez E, Budoff MJ. The impact of statins on coronary atherosclerosis progression and long-term cardiovascular disease risk in rheumatoid arthritis. Rheumatology. 2021;61(5):1857–66. doi: 10.1093/rheumatology/keab642</cite> [<a href="https://doi.org/10.1093/rheumatology/keab642" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34373923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Rheumatology&amp;title=The%20impact%20of%20statins%20on%20coronary%20atherosclerosis%20progression%20and%20long-term%20cardiovascular%20disease%20risk%20in%20rheumatoid%20arthritis&amp;author=GA%20Karpouzas&amp;author=SR%20Ormseth&amp;author=E%20Hernandez&amp;author=MJ%20Budoff&amp;volume=61&amp;issue=5&amp;publication_year=2021&amp;pages=1857-66&amp;pmid=34373923&amp;doi=10.1093/rheumatology/keab642&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref077">
<span class="label">77.</span><cite>Zheng R, Li Z, Fang W, Lou H, Liu F, Sun Q. A genome-wide association study identified PRKCB as a causal gene and therapeutic target for Mycobacterium avium complex disease. Cell Rep Med. 2025.</cite> [<a href="https://doi.org/10.1016/j.xcrm.2024.101923" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11866518/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39848245/" class="usa-link">PubMed</a>]</li>
<li id="pone.0329101.ref078">
<span class="label">78.</span><cite>Trzeciak P, Herbet M, Dudka J. Common factors of Alzheimer’s disease and rheumatoid arthritis—pathomechanism and treatment. Molecules. 2021;26(19):6038. doi: 10.3390/molecules26196038
</cite> [<a href="https://doi.org/10.3390/molecules26196038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8512006/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34641582/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Common%20factors%20of%20Alzheimer%E2%80%99s%20disease%20and%20rheumatoid%20arthritis%E2%80%94pathomechanism%20and%20treatment&amp;author=P%20Trzeciak&amp;author=M%20Herbet&amp;author=J%20Dudka&amp;volume=26&amp;issue=19&amp;publication_year=2021&amp;pages=6038&amp;pmid=34641582&amp;doi=10.3390/molecules26196038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref079">
<span class="label">79.</span><cite>Valous NA, Popp F, Zörnig I, Jäger D, Charoentong P. Graph machine learning for integrated multi-omics analysis. Br J Cancer. 2024;131(2):205–11. doi: 10.1038/s41416-024-02706-7
</cite> [<a href="https://doi.org/10.1038/s41416-024-02706-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11263675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38729996/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&amp;title=Graph%20machine%20learning%20for%20integrated%20multi-omics%20analysis&amp;author=NA%20Valous&amp;author=F%20Popp&amp;author=I%20Z%C3%B6rnig&amp;author=D%20J%C3%A4ger&amp;author=P%20Charoentong&amp;volume=131&amp;issue=2&amp;publication_year=2024&amp;pages=205-11&amp;pmid=38729996&amp;doi=10.1038/s41416-024-02706-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref080">
<span class="label">80.</span><cite>Hoch M, Gupta S, Wolkenhauer O. Large-scale knowledge graph representations of disease processes. Curr Opinion Syst Biol. 2024;38:100517. doi: 10.1016/j.coisb.2024.100517</cite> [<a href="https://doi.org/10.1016/j.coisb.2024.100517" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Curr%20Opinion%20Syst%20Biol&amp;title=Large-scale%20knowledge%20graph%20representations%20of%20disease%20processes&amp;author=M%20Hoch&amp;author=S%20Gupta&amp;author=O%20Wolkenhauer&amp;volume=38&amp;publication_year=2024&amp;pages=100517&amp;doi=10.1016/j.coisb.2024.100517&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref081">
<span class="label">81.</span><cite>Chen C, Wang J, Pan D, Wang X, Xu Y, Yan J, et al. Applications of multi-omics analysis in human diseases. MedComm. 2023;4(4):e315. doi: 10.1002/mco2.315</cite> [<a href="https://doi.org/10.1002/mco2.315" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10390758/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37533767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=MedComm&amp;title=Applications%20of%20multi-omics%20analysis%20in%20human%20diseases&amp;author=C%20Chen&amp;author=J%20Wang&amp;author=D%20Pan&amp;author=X%20Wang&amp;author=Y%20Xu&amp;volume=4&amp;issue=4&amp;publication_year=2023&amp;pmid=37533767&amp;doi=10.1002/mco2.315&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref082">
<span class="label">82.</span><cite>Tian Z, Mclaughlin J, Verma A, Chinoy H, Heald AH. The relationship between rheumatoid arthritis and diabetes mellitus: a systematic review and meta-analysis. Cardiovasc Endocrinol Metabolism. 2021;10(2):125–31.</cite> [<a href="https://doi.org/10.1097/XCE.0000000000000244" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8189616/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34124603/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Endocrinol%20Metabolism&amp;title=The%20relationship%20between%20rheumatoid%20arthritis%20and%20diabetes%20mellitus:%20a%20systematic%20review%20and%20meta-analysis&amp;author=Z%20Tian&amp;author=J%20Mclaughlin&amp;author=A%20Verma&amp;author=H%20Chinoy&amp;author=AH%20Heald&amp;volume=10&amp;issue=2&amp;publication_year=2021&amp;pages=125-31&amp;pmid=34124603&amp;doi=10.1097/XCE.0000000000000244&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref083">
<span class="label">83.</span><cite>Hsu TC, Rodrigues GOL, Winer H, Hixon JA, Li W, Tarasova NI, et al. A focused review of ras guanine nucleotide-releasing protein 1 in immune cells and cancer. IJMS. 2023;24(2):1652. doi: 10.3390/ijms24021652
</cite> [<a href="https://doi.org/10.3390/ijms24021652" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9864139/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36675167/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IJMS&amp;title=A%20focused%20review%20of%20ras%20guanine%20nucleotide-releasing%20protein%201%20in%20immune%20cells%20and%20cancer&amp;author=TC%20Hsu&amp;author=GOL%20Rodrigues&amp;author=H%20Winer&amp;author=JA%20Hixon&amp;author=W%20Li&amp;volume=24&amp;issue=2&amp;publication_year=2023&amp;pages=1652&amp;pmid=36675167&amp;doi=10.3390/ijms24021652&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref084">
<span class="label">84.</span><cite>Zhou Z, Chen F, Zhong S, Zhou Y, Zhang R, Kang K, et al. Molecular identification of protein kinase C beta in Alzheimer’s disease. Aging. 2020;12(21):21798–808. doi: 10.18632/aging.103994
</cite> [<a href="https://doi.org/10.18632/aging.103994" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7695410/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33186918/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aging&amp;title=Molecular%20identification%20of%20protein%20kinase%20C%20beta%20in%20Alzheimer%E2%80%99s%20disease&amp;author=Z%20Zhou&amp;author=F%20Chen&amp;author=S%20Zhong&amp;author=Y%20Zhou&amp;author=R%20Zhang&amp;volume=12&amp;issue=21&amp;publication_year=2020&amp;pages=21798-808&amp;pmid=33186918&amp;doi=10.18632/aging.103994&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref085">
<span class="label">85.</span><cite>Unal U, Comertpay B, Demirtas TY, Gov E. Drug repurposing for rheumatoid arthritis: identification of new drug candidates via bioinformatics and text mining analysis. Autoimmunity. 2022;55(3):147–56. doi: 10.1080/08916934.2022.2027922
</cite> [<a href="https://doi.org/10.1080/08916934.2022.2027922" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35048767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Autoimmunity&amp;title=Drug%20repurposing%20for%20rheumatoid%20arthritis:%20identification%20of%20new%20drug%20candidates%20via%20bioinformatics%20and%20text%20mining%20analysis&amp;author=U%20Unal&amp;author=B%20Comertpay&amp;author=TY%20Demirtas&amp;author=E%20Gov&amp;volume=55&amp;issue=3&amp;publication_year=2022&amp;pages=147-56&amp;pmid=35048767&amp;doi=10.1080/08916934.2022.2027922&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref086">
<span class="label">86.</span><cite>Meehan RT, Amigues IA, Knight V. Precision medicine for rheumatoid arthritis: the right drug for the right patient—companion diagnostics. Diagnostics. 2021;11(8):1362. doi: 10.3390/diagnostics11081362
</cite> [<a href="https://doi.org/10.3390/diagnostics11081362" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8391624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34441297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Diagnostics&amp;title=Precision%20medicine%20for%20rheumatoid%20arthritis:%20the%20right%20drug%20for%20the%20right%20patient%E2%80%94companion%20diagnostics&amp;author=RT%20Meehan&amp;author=IA%20Amigues&amp;author=V%20Knight&amp;volume=11&amp;issue=8&amp;publication_year=2021&amp;pages=1362&amp;pmid=34441297&amp;doi=10.3390/diagnostics11081362&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref087">
<span class="label">87.</span><cite>Alsaedi S, Ogasawara M, Alarawi M, Gao X, Gojobori T. AI-powered precision medicine: utilizing genetic risk factor optimization to revolutionize healthcare. NAR Genom Bioinform. 2025;7(2):lqaf038.doi: 10.1093/nargab/lqaf038</cite> [<a href="https://doi.org/10.1093/nargab/lqaf038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12051108/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40330081/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NAR%20Genom%20Bioinform&amp;title=AI-powered%20precision%20medicine:%20utilizing%20genetic%20risk%20factor%20optimization%20to%20revolutionize%20healthcare&amp;author=S%20Alsaedi&amp;author=M%20Ogasawara&amp;author=M%20Alarawi&amp;author=X%20Gao&amp;author=T%20Gojobori&amp;volume=7&amp;issue=2&amp;publication_year=2025&amp;pmid=40330081&amp;doi=10.1093/nargab/lqaf038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref088">
<span class="label">88.</span><cite>Okada Y, Wu D, Trynka G, Raj T, Terao C, Ikari K, et al. Genetics of rheumatoid arthritis contributes to biology and drug discovery. Nature. 2014;506(7488):376–81.
</cite> [<a href="https://doi.org/10.1038/nature12873" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3944098/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24390342/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Genetics%20of%20rheumatoid%20arthritis%20contributes%20to%20biology%20and%20drug%20discovery&amp;author=Y%20Okada&amp;author=D%20Wu&amp;author=G%20Trynka&amp;author=T%20Raj&amp;author=C%20Terao&amp;volume=506&amp;issue=7488&amp;publication_year=2014&amp;pages=376-81&amp;pmid=24390342&amp;doi=10.1038/nature12873&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pone.0329101.ref089">
<span class="label">89.</span><cite>Alsaedi S, Gao X, Gojobori T. Beyond digital twins: the role of foundation models in enhancing the interpretability of multiomics modalities in precision medicine. FEBS Open Bio. 2025.</cite> [<a href="https://doi.org/10.1002/2211-5463.70003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12319712/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39995150/" class="usa-link">PubMed</a>]</li>
<li id="pone.0329101.ref090">
<span class="label">90.</span><cite>Plant D, Wilson AG, Barton A. Genetic and epigenetic predictors of responsiveness to treatment in RA. Nat Rev Rheumatol. 2014;10(6):329–37. doi: 10.1038/nrrheum.2014.16
</cite> [<a href="https://doi.org/10.1038/nrrheum.2014.16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24535543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Rheumatol&amp;title=Genetic%20and%20epigenetic%20predictors%20of%20responsiveness%20to%20treatment%20in%20RA&amp;author=D%20Plant&amp;author=AG%20Wilson&amp;author=A%20Barton&amp;volume=10&amp;issue=6&amp;publication_year=2014&amp;pages=329-37&amp;pmid=24535543&amp;doi=10.1038/nrrheum.2014.16&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 Table. Genetic Risk Variants of RA: List of retrieved genetic risk SNP of Rheumatoid Arthritis retrieved from biomedical articles.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370121/bin/pone.0329101.s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0329101.s001.pdf</a><sup> (95.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>S2 Table. Annotated Genetic Risk Variants of RA: List of annotated genetic risk variants of Rheumatoid Arthritis retrieved from biomedical articles.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370121/bin/pone.0329101.s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0329101.s002.pdf</a><sup> (425.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>S3 Table. Annotated Genetic Risk Genes: List of annotated genetic risk genes of Rheumatoid Arthritis retrieved from biomedical articles.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370121/bin/pone.0329101.s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0329101.s003.pdf</a><sup> (121.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>S4 Table. Molecular network: Detailed information on the molecular functions and biological processes associated with the constructed networks of RA genetic risk factors.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370121/bin/pone.0329101.s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0329101.s004.pdf</a><sup> (114.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>S5 Table. Therapeutic targets and biomarker validation dataset of RA: List of approved therapeutic targets and biomarkers of RA from DrugBank and biomarker databases.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370121/bin/pone.0329101.s005.pdf" data-ga-action="click_feat_suppl" class="usa-link">pone.0329101.s005.pdf</a><sup> (133.9KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data underlying the results presented in the study are available from in the <a href="#sec043" class="usa-link">Supporting information</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS One are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pone.0329101"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pone.0329101.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (24.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370121/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370121/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370121%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370121/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370121/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370121/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40839671/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370121/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40839671/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370121/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370121/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="dzAbe2e9s9RRj8Skrcwrudt5PQQE971mWWvqB3ScTU5Eeb0Y8Hwxwzic3RQpdST5">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
