
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Genome‐wide identification and evolutionary analysis of disease resistance genes in Brassica carinata - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE530C5F8AEE7343050C5F0069C5770D.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="blackwellopen">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326289/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="The Plant Genome">
<meta name="citation_title" content="Genome‐wide identification and evolutionary analysis of disease resistance genes in Brassica carinata">
<meta name="citation_author" content="Junrey C Amas">
<meta name="citation_author_institution" content="School of Biological Sciences and the Institute of Agriculture, The University of Western Australia, Crawley, Western Australia, Australia">
<meta name="citation_author" content="Philipp E Bayer">
<meta name="citation_author_institution" content="School of Biological Sciences and the Institute of Agriculture, The University of Western Australia, Crawley, Western Australia, Australia">
<meta name="citation_author" content="Wei Hong Tan">
<meta name="citation_author_institution" content="School of Biological Sciences and the Institute of Agriculture, The University of Western Australia, Crawley, Western Australia, Australia">
<meta name="citation_author" content="Aldrin Cantila">
<meta name="citation_author_institution" content="The Institute of Agriculture, The University of Western Australia, Crawley, Western Australia, Australia">
<meta name="citation_author" content="Aria Dolatabadian">
<meta name="citation_author_institution" content="School of Biological Sciences and the Institute of Agriculture, The University of Western Australia, Crawley, Western Australia, Australia">
<meta name="citation_author" content="William J W Thomas">
<meta name="citation_author_institution" content="School of Biological Sciences and the Institute of Agriculture, The University of Western Australia, Crawley, Western Australia, Australia">
<meta name="citation_author" content="David Edwards">
<meta name="citation_author_institution" content="School of Biological Sciences and the Centre for Applied Bioinformatics, The University of Western Australia, Crawley, Western Australia, Australia">
<meta name="citation_author" content="Jacqueline Batley">
<meta name="citation_author_institution" content="School of Biological Sciences and the Institute of Agriculture, The University of Western Australia, Crawley, Western Australia, Australia">
<meta name="citation_publication_date" content="2025 Aug 6">
<meta name="citation_volume" content="18">
<meta name="citation_issue" content="3">
<meta name="citation_firstpage" content="e70087">
<meta name="citation_doi" content="10.1002/tpg2.70087">
<meta name="citation_pmid" content="40765494">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326289/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326289/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326289/pdf/TPG2-18-e70087.pdf">
<meta name="description" content="Brassica carinata is considered an orphan crop, yet it is vital for understanding the evolution of the triangle of U Brassica species. The availability of a genome reference for this species has allowed for the interrogation of the genomic and ...">
<meta name="og:title" content="Genome‐wide identification and evolutionary analysis of disease resistance genes in Brassica carinata">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Brassica carinata is considered an orphan crop, yet it is vital for understanding the evolution of the triangle of U Brassica species. The availability of a genome reference for this species has allowed for the interrogation of the genomic and ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326289/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12326289">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/tpg2.70087"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/TPG2-18-e70087.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12326289%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12326289/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12326289/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326289/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-blackwellopen.png" alt="Wiley Open Access Collection logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Wiley Open Access Collection" title="Link to Wiley Open Access Collection" shape="default" href="https://doi.org/10.1002/tpg2.70087" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Plant Genome</button></div>. 2025 Aug 6;18(3):e70087. doi: <a href="https://doi.org/10.1002/tpg2.70087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/tpg2.70087</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Plant%20Genome%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Plant%20Genome%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Plant%20Genome%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genome‐wide identification and evolutionary analysis of disease resistance genes in <em>Brassica carinata</em>
</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Amas%20JC%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Junrey C Amas</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Junrey C Amas</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>School of Biological Sciences and the Institute of Agriculture, The University of Western Australia, Crawley, Western Australia, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Amas%20JC%22%5BAuthor%5D" class="usa-link"><span class="name western">Junrey C Amas</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bayer%20PE%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Philipp E Bayer</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Philipp E Bayer</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>School of Biological Sciences and the Institute of Agriculture, The University of Western Australia, Crawley, Western Australia, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bayer%20PE%22%5BAuthor%5D" class="usa-link"><span class="name western">Philipp E Bayer</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tan%20WH%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Wei Hong Tan</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Wei Hong Tan</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>School of Biological Sciences and the Institute of Agriculture, The University of Western Australia, Crawley, Western Australia, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tan%20WH%22%5BAuthor%5D" class="usa-link"><span class="name western">Wei Hong Tan</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cantila%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Aldrin Cantila</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Aldrin Cantila</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>The Institute of Agriculture, The University of Western Australia, Crawley, Western Australia, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cantila%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Aldrin Cantila</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dolatabadian%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Aria Dolatabadian</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Aria Dolatabadian</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>School of Biological Sciences and the Institute of Agriculture, The University of Western Australia, Crawley, Western Australia, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dolatabadian%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Aria Dolatabadian</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thomas%20WJW%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">William J W Thomas</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">William J W Thomas</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>School of Biological Sciences and the Institute of Agriculture, The University of Western Australia, Crawley, Western Australia, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thomas%20WJW%22%5BAuthor%5D" class="usa-link"><span class="name western">William J W Thomas</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Edwards%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">David Edwards</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">David Edwards</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>School of Biological Sciences and the Centre for Applied Bioinformatics, The University of Western Australia, Crawley, Western Australia, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Edwards%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">David Edwards</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Batley%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Jacqueline Batley</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Jacqueline Batley</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>School of Biological Sciences and the Institute of Agriculture, The University of Western Australia, Crawley, Western Australia, Australia</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Batley%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jacqueline Batley</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="tpg270087-aff-0001">
<sup>
<sup>1</sup>
</sup>School of Biological Sciences and the Institute of Agriculture, The University of Western Australia, Crawley, Western Australia, Australia</div>
<div id="tpg270087-aff-0002">
<sup>
<sup>2</sup>
</sup>The Institute of Agriculture, The University of Western Australia, Crawley, Western Australia, Australia</div>
<div id="tpg270087-aff-0003">
<sup>
<sup>3</sup>
</sup>School of Biological Sciences and the Centre for Applied Bioinformatics, The University of Western Australia, Crawley, Western Australia, Australia</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, Jacqueline Batley, School of Biological Sciences and the Institute of Agriculture, The University of Western Australia, Crawley, WA 6009, Australia. Email: <span>jacqueline.batley@uwa.edu.au</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jun 20; Received 2025 Apr 9; Accepted 2025 Jul 1; Issue date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>The Plant Genome</em> published by Wiley Periodicals LLC on behalf of Crop Science Society of America.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12326289  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40765494/" class="usa-link">40765494</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>
<em>Brassica carinata</em> is considered an orphan crop, yet it is vital for understanding the evolution of the triangle of U <em>Brassica</em> species. The availability of a genome reference for this species has allowed for the interrogation of the genomic and genetic underpinnings of important traits, including disease resistance. In this study, we report a comprehensive analysis of resistance gene analogs (RGAs) in the first genome assembly for <em>B. carinata</em> (zd‐1). A total of 2570 RGAs were predicted, of which 2020 were transmembrane leucine‐rich repeats and 550 were nucleotide‐binding site leucine‐rich repeats. Gene duplication events affected 65.2% of the RGAs, which were classified as either intergenomic or intragenomic duplications. The contrasting patterns of these gene duplication events between the two subgenomes (B and C) support previous findings indicating the presence of subgenome dominance in this species, a characteristic that is shared with the other allopolyploid <em>Brassicas</em>. Comparative analysis with its diploid progenitors, <em>B. nigra</em> and <em>B.oleracea</em>, revealed conservation of genomic features among these species, while phylogenetic analysis suggests that <em>B. carinata</em> RGAs have undergone extensive expansion. This study is the first to analyze the complete set of RGAs within the <em>B. carinata</em> genome, presenting a comprehensive view of the disease resistance landscape in this species.</p></section><section class="abstract" id="abstract2"><h2>Core Ideas</h2>
<ul id="tpg270087-list-10001" class="list" style="list-style-type:disc">
<li><p>Significant number of resistance genes analogs (RGAs) were predicted in the <em>Brassica carinata</em> zd‐1 genome assembly.</p></li>
<li><p>The RGAs were highly duplicated and unevenly distributed in the <em>B. carinata</em> genome.</p></li>
<li><p>Subgenome dominance was evident based on the duplication patterns of RGAs.</p></li>
<li><p>Dynamic evolution was observed in <em>B. carinata</em> RGAs relative to the progenitor species.</p></li>
</ul></section><hr class="headless">
<h3>Abbreviations</h3>
<dl class="def-list" id="tpg270087-dl-0001">
<dt>BLAST</dt>
<dd><p>basic local alignment search tool</p></dd>
<dt>CC</dt>
<dd><p>coiled‐coil</p></dd>
<dt>NLR</dt>
<dd><p>nucleotide‐binding site leucine‐rich repeat</p></dd>
<dt>RGA</dt>
<dd><p>resistance gene analog</p></dd>
<dt>RLK</dt>
<dd><p>receptor‐like kinase</p></dd>
<dt>RLP</dt>
<dd><p>receptor‐like protein</p></dd>
<dt>TE</dt>
<dd><p>transposable element</p></dd>
<dt>TIR</dt>
<dd><p>toll/interleukin‐1 receptor</p></dd>
<dt>TM‐LRR</dt>
<dd><p>transmembrane leucine‐rich repeat</p></dd>
</dl>
<section id="tpg270087-sec-0010"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>
<em>Brassica carinata</em> (BBCC, <em>n</em> = 17) is one of the allotetraploid species in the <em>Brassica</em> triangle of U and was formed from the natural hybridization of <em>B. nigra</em> (BB) and <em>B. oleracea</em> (CC) (U, <a href="#tpg270087-bib-0027" class="usa-link" aria-describedby="tpg270087-bib-0027"><sup>1935</sup></a>). This species is thought to have originated in Ethiopia and parts of East Africa extending toward the Mediterranean coast, where it is mainly grown as a vegetable and condiment. It is also used in traditional medicine for treating common tropical diseases (Seepaul et al., <a href="#tpg270087-bib-0037" class="usa-link" aria-describedby="tpg270087-bib-0037">2021</a>). Beyond Africa, <em>B. carinata</em> exhibits a wide range of agroecological adaptation, including in drought‐prone and temperate areas (Kumar et al., <a href="#tpg270087-bib-0019" class="usa-link" aria-describedby="tpg270087-bib-0019">2020</a>). It has also been found well‐suited for crop rotation with major staples including maize and soybean (Karami et al., <a href="#tpg270087-bib-0016" class="usa-link" aria-describedby="tpg270087-bib-0016">2022</a>). Furthermore, when planted as a break crop, it can enhance weed management strategies in major crop production systems due to its ability to compete and eventually reduce the density of noxious weeds (Tiwari et al., <a href="#tpg270087-bib-0044" class="usa-link" aria-describedby="tpg270087-bib-0044">2021</a>).</p>
<p>
<em>Brassica carinata</em> recently gained attention as a potential source of renewable energy that can replace fossil‐based fuels because of its chemical properties (Roslinsky et al., <a href="#tpg270087-bib-0036" class="usa-link" aria-describedby="tpg270087-bib-0036">2021</a>). Its seeds contain high levels of erucic and linoleic acids, which can be processed into a high‐quality biofuel (Redda et al., <a href="#tpg270087-bib-0035" class="usa-link" aria-describedby="tpg270087-bib-0035">2022</a>). There is also potential for doubling the fuel output from its oil as one molecule of erucic acid can be cleaved into two biofuel hydrocarbons (Seepaul et al., <a href="#tpg270087-bib-0037" class="usa-link" aria-describedby="tpg270087-bib-0037">2021</a>). The suitability modeling of Alam and Dwivedi (<a href="#tpg270087-bib-0003" class="usa-link" aria-describedby="tpg270087-bib-0003">2019</a>) projected a production potential between 980 and 2045 million L of jet fuel from <em>B. carinata</em>, which is enough to replace 1.4%–2.33% of petroleum‐based jet fuel usage in the United States. The cultivation of <em>B. carinata</em> for renewable energy production indeed offers a sustainable solution for potentially decreasing carbon emissions without displacing feed and food crops (Iboyi et al., <a href="#tpg270087-bib-0014" class="usa-link" aria-describedby="tpg270087-bib-0014">2023</a>).</p>
<p>Besides its industrial use, <em>B. carinata</em> has been known to be a rich source of useful agronomic traits for crop breeding including resistance to biotic and abiotic stresses (Thakur et al., <a href="#tpg270087-bib-0042" class="usa-link" aria-describedby="tpg270087-bib-0042">2019</a>). Limbalkar et al. (<a href="#tpg270087-bib-0050" class="usa-link" aria-describedby="tpg270087-bib-0050">2021</a>) successfully developed <em>B. juncea</em> lines derived from interspecific hybridization with <em>B. carinata</em>, which showed high drought tolerance and potential for hybrid development. In another study, <em>B. carinata</em> was shown to display higher frost tolerance resulting in greater yield compared with other plants species including rapeseed (<em>B. napus</em>) and pennycress (<em>Thlaspi arvense</em>) (Tiwari et al., <a href="#tpg270087-bib-0044" class="usa-link" aria-describedby="tpg270087-bib-0044">2021</a>). Tonguç and Griffiths (<a href="#tpg270087-bib-0051" class="usa-link" aria-describedby="tpg270087-bib-0051">2004</a>) reported the successful transfer of powdery mildew resistance from <em>B. carinata</em> to <em>B. oleracea</em> with the aid of the embryo rescue technique. Genetic mapping also identified a locus on chromosome B07 underlying resistance to the black rot pathogen, <em>Xanthomonas campestris</em> pv. <em>campestris</em>, enabling molecular breeding for resistance against this disease (Sharma et al., <a href="#tpg270087-bib-0038" class="usa-link" aria-describedby="tpg270087-bib-0038">2016</a>). Furthermore, the <em>B. carinata</em> line, ATC94129P, was identified as resistant to <em>Pseudocercosporella capsellae</em>, the causal pathogen of white leaf spot disease in <em>Brassica</em> species (Gunasinghe et al., <a href="#tpg270087-bib-0011" class="usa-link" aria-describedby="tpg270087-bib-0011">2016</a>). These studies demonstrate the potential use of <em>B. carinata</em> for stress tolerance breeding in <em>Brassica</em> crops.</p>
<p>Compared with other <em>Brassica</em> species, genomic studies in <em>B. carinata</em> have been scarce mainly due to the lack of genome resources available for this species. The first genome reference was only made available in 2021, which was considered a milestone in <em>Brassica</em> genomics as it completed the genome information for all the six species in triangle of U (Song et al., <a href="#tpg270087-bib-0039" class="usa-link" aria-describedby="tpg270087-bib-0039">2021</a>). This genome reference was based on an advanced inbred line, zd‐1, spanning a total genome size of 1.1 Gbp with 97,149 predicted genes, which is more than <em>B. juncea</em> (79,593) but less compared to <em>B. napus</em> (101,040). The availability of this reference enables the estimation of the time of formation of <em>B. carinata</em>, which was deduced to be slightly earlier than <em>B. napus</em> but later than <em>B. juncea</em>. Thus, this resource provides important insights into evolutionary relationship of <em>Brassica</em> species in the triangle of U.</p>
<p>Furthermore, the <em>B. carinata</em> (zd‐1) genome reference shows great potential for advancing genomic studies, including characterizing the disease resistance landscape in this crop, which will assist breeding for improved disease resistance in <em>Brassica</em> crops. This can be facilitated by identifying disease resistance genes or resistance gene analogs (RGAs) in this reference. RGAs can be generally classified as either nucleotide binding‐site leucine‐rich repeats (NBS‐LRRs/NLRs) or transmembrane leucine‐rich repeats (TM‐LRRs) (Li et al., <a href="#tpg270087-bib-0021" class="usa-link" aria-describedby="tpg270087-bib-0021">2016</a>). Within the NLR group, two subclasses are identified: regular NLRs, which carry either the toll/interleukin‐1 receptor (TIR) or coiled‐coil (CC) domains at the amino terminus; thus, they are classified as TIR‐NBS‐LRR (TNLs) and CC‐NBS‐LRR (CNLs), respectively. Some NLRs may lack one or more domains and are categorized as non‐regular NLRs; for example, CN and TN lack the LRR domain, NL does not possess the amino terminus domains (CC or TIR), TX has a TIR domain linked to an unknown domain, and “Other” NLRs contain only TIR and CC domains. On the other hand, TM‐LRRs can be divided into three subclasses, including receptor‐like kinases (RLKs) and receptor‐like proteins (RLPs), which are well‐known types of genes. The last subclass, TM‐CC, is an atypical RGA with a transmembrane domain similar to RLKs and RLPs but with an additional CC domain that is typically observed in NLRs (Li et al., <a href="#tpg270087-bib-0021" class="usa-link" aria-describedby="tpg270087-bib-0021">2016</a>).</p>
<p>While Song et al. (<a href="#tpg270087-bib-0039" class="usa-link" aria-describedby="tpg270087-bib-0039">2021</a>) have initially identified RGAs in zd‐1 assembly, this was only limited to some NLR subclasses, and no information was presented for TM‐LRRs. Thus, there is still an incomplete understanding of the RGA repertoire in this species, which will be crucial for utilizing this species in disease‐resistance breeding. This study aims to bridge the knowledge gap in this research area by characterizing all the RGAs in this species. We predicted all the known RGA families in this reference, including NLRs and TM‐LRRs, to analyze their genome‐wide distribution and organization. Comparative analysis between <em>B. carinata</em>’s diploid progenitor species, <em>B. nigra</em> and <em>B. oleracea</em>, was also performed, providing key insights into the evolution of RGAs in these species. This study is the first to comprehensively perform a genome‐wide assessment of the RGAs in <em>B. carinata</em>, presenting the overall picture of the resistance gene landscape in this species.</p></section><section id="tpg270087-sec-0020"><h2 class="pmc_sec_title">2. METHODS</h2>
<section id="tpg270087-sec-0030"><h3 class="pmc_sec_title">2.1. Identification of RGAs</h3>
<p>The first genome assembly for <em>B. carinata</em> (zd‐1) was downloaded from the <em>Brassica</em> database (<a href="http://brassicadb.bio2db.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://brassicadb.bio2db.com</a>) and used for RGA mining (Table <a href="#tpg270087-supinfo-0001" class="usa-link">S1</a>). The RGAugury pipeline (Li et al., <a href="#tpg270087-bib-0021" class="usa-link" aria-describedby="tpg270087-bib-0021">2016</a>) was implemented to identify the putative RGAs in this genome. The pipeline identifies genes based on their conserved motifs and divides them into two main classes, which can either be a TM‐LRR or an NLR. The TM‐LRRs were further classified into RLPs, RLKs, and TM‐CCs, while the NLRs were subdivided into several groups depending on the presence or absence of specific domains, namely, NBS, CNL, TNL, CN, NL, TX, and Other. RGAs predicted in the progenitor species <em>B. nigra</em> (NI100 <em>v2</em>) (Perumal et al., <a href="#tpg270087-bib-0032" class="usa-link" aria-describedby="tpg270087-bib-0032">2020</a>) and <em>B. oleracea</em> (JZS <em>v1</em>) (Liu et al., <a href="#tpg270087-bib-0022" class="usa-link" aria-describedby="tpg270087-bib-0022">2014</a>) were also retrieved for comparative analysis.</p>
<section class="bt xbox font-sm" id="boxed-text1"><section id="tpg270087-sec-0040"><h4 class="pmc_sec_title">Core Ideas</h4>
<ul id="tpg270087-list-0001" class="list" style="list-style-type:disc">
<li><p>Significant number of resistance genes analogs (RGAs) were predicted in the <em>Brassica carinata</em> zd‐1 genome assembly.</p></li>
<li><p>The RGAs were highly duplicated and unevenly distributed in the <em>B. carinata</em> genome.</p></li>
<li><p>Subgenome dominance was evident based on the duplication patterns of RGAs.</p></li>
<li><p>Dynamic evolution was observed in <em>B. carinata</em> RGAs relative to the progenitor species.</p></li>
</ul></section></section></section><section id="tpg270087-sec-0050"><h3 class="pmc_sec_title">2.2. Gene duplication analysis</h3>
<p>The analysis of gene duplication was performed through basic local alignment search tool (BLAST) comparison of all the RGA protein sequences against each other. A custom database of the protein sequences of all the RGAs was first created and converted to FASTA format, which was then uploaded in Geneious Prime. Genes were considered duplicates when there was at least 70% coverage and identity with an <em>e</em>‐value of 20 in the BLAST hits. Because <em>B. carinata</em> is an allotetraploid species, duplication events located in the same genome were classified as intragenomic, while those detected between the B and C subgenomes were considered as intergenomic. Furthermore, paralogs or duplicated genes located within 5 Mb from each other on the same chromosome were classified as tandemly duplicated, while those located beyond 5 Mb, including other chromosomes in the same genome, were considered as segmentally duplicated (Agarwal et al., <a href="#tpg270087-bib-0002" class="usa-link" aria-describedby="tpg270087-bib-0002">2016</a>). The RGAs predicted in unplaced contigs were not included in the analysis as they do not have genomic coordinates on the 17 chromosomes.</p></section><section id="tpg270087-sec-0060"><h3 class="pmc_sec_title">2.3. Physical clustering</h3>
<p>The method described by Bayer et al. (<a href="#tpg270087-bib-0052" class="usa-link" aria-describedby="tpg270087-bib-0052">2019</a>) was employed for the physical clustering analysis. This utilized a custom Python 3 script (makeRGeneClusterAnalysis), which merges RGAs into RGA‐gene‐rich clusters when there is at least one RGA within 10 upstream or 10 downstream genes. The clusters were then classified as homogeneous (those containing the same type of RGA class i.e., all NLRs or TM‐LRRs) or heterogeneous (those having both NLRs and TM‐LRRs regardless of the number). The same method was applied for <em>B. nigra</em> and <em>B. oleracea</em> RGAs to compare their physical clustering patterns with <em>B. carinata</em>.</p></section><section id="tpg270087-sec-0070"><h3 class="pmc_sec_title">2.4. Comparative analysis with progenitor species</h3>
<p>The best‐matching homologs of <em>B. carinata</em> RGAs in the <em>B. nigra</em> and <em>B. oleracea</em> genomes were identified using BLAST. Similar to the gene duplication analysis mentioned above, a custom database for <em>B. nigra</em> and <em>B. oleracea</em> RGAs was created and converted to FASTA files, which were then uploaded in Geneious Prime. Only significant hits with an <em>e</em>‐value of 40 were considered as conserved genes (Alamery et al., <a href="#tpg270087-bib-0004" class="usa-link" aria-describedby="tpg270087-bib-0004">2018</a>).</p></section><section id="tpg270087-sec-0080"><h3 class="pmc_sec_title">2.5. Phylogenetic analysis</h3>
<p>The protein sequences of the RGAs were aligned using the multiple alignment using fast Fourier transform algorithm. The alignment file was then utilized as input for building maximum likelihood (ML) phylogenetic trees using the FastTree program (Katoh et al., <a href="#tpg270087-bib-0017" class="usa-link" aria-describedby="tpg270087-bib-0017">2009</a>) implemented in Geneious Prime. The trees were then visualized using the FigTree program (Rambaut, <a href="#tpg270087-bib-0034" class="usa-link" aria-describedby="tpg270087-bib-0034">2010</a>). Separate trees were constructed for NLRs and TM‐LRRs. Similarly, phylogenetic trees were constructed for <em>B. carinata</em> RGAs with their homologs in the progenitor species to analyze the evolutionary trajectory of these genes in these species.</p></section><section id="tpg270087-sec-0090"><h3 class="pmc_sec_title">2.6. Statistical analysis</h3>
<p>All the statistical analyses were performed using the Statistical Tool for Agricultural Research (STAR version 2.0.1, <a href="#tpg270087-bib-0040" class="usa-link" aria-describedby="tpg270087-bib-0040"><sup>2014</sup></a>).</p></section><section id="tpg270087-sec-0100"><h3 class="pmc_sec_title">2.7. Plots and graphs</h3>
<p>The chromosome ideograms for plotting the genome‐wide distribution of the RGAs were created using PhenoGram program (Wolfe et al., <a href="#tpg270087-bib-0046" class="usa-link" aria-describedby="tpg270087-bib-0046">2013</a>). The homolog pair plot was created using the RIdeogram package (Hao et al., <a href="#tpg270087-bib-0012" class="usa-link" aria-describedby="tpg270087-bib-0012">2020</a>) and the bar graphs using the ggplot2 package, implemented in R (R Core Team, <a href="#tpg270087-bib-0033" class="usa-link" aria-describedby="tpg270087-bib-0033"><sup>2021</sup></a>).</p></section></section><section id="tpg270087-sec-0110"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="tpg270087-sec-0120"><h3 class="pmc_sec_title">3.1. Identification of RGAs in the <em>B. carinata</em> zd‐1 genome assembly</h3>
<p>A total of 2570 RGAs were identified using RGAugury, accounting for 2.65% of the total genes predicted in the <em>B. carinata</em> zd‐1 genome assembly. The majority of these RGAs were classified as TM‐LRRs (2020), while 550 were predicted as NLRs. Among the TM‐LRRs, RLKs (1352) were the most abundant, followed by TM‐CCs (425) and RLPs (243). Of the predicted NLR genes, TNLs were the most abundant (168), followed by NLs (96), TXs (82), CNLs (76), NBSs (47), TNs (35), and subclass Other (34), while CNs (12) were the least abundant NLR subclass (Table <a href="#tpg270087-tbl-0001" class="usa-link">1</a>).</p>
<section class="tw xbox font-sm" id="tpg270087-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>The total number of RGAs predicted in <em>Brassica carinata</em> in comparison to its progenitor genomes.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead>
<tr>
<th colspan="2" style="border-bottom:solid 1px #000000" rowspan="3" align="left">RGA class</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">
<em>Brassica carinata</em> (zd‐1)</th>
<th style="border-bottom:solid 1px #000000" align="left" rowspan="1" colspan="1">
<em>Brassica nigra</em> (NI100 v2)</th>
<th style="border-bottom:solid 1px #000000" align="left" rowspan="1" colspan="1">
<em>Brassica oleracea</em> (JZS v1)</th>
</tr>
<tr>
<th rowspan="2" align="left" colspan="1">B genome</th>
<th rowspan="2" align="left" colspan="1">C genome</th>
<th rowspan="2" align="left" colspan="1">Unplaced contigs</th>
<th style="border-bottom:solid 1px #000000" align="left" rowspan="1" colspan="1">B genome</th>
<th style="border-bottom:solid 1px #000000" align="left" rowspan="1" colspan="1">C genome</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1"></th>
<th align="left" rowspan="1" colspan="1"></th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="9" align="left" colspan="1">NLR</td>
<td align="left" rowspan="1" colspan="1">NBS</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">29</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">19</td>
<td align="left" rowspan="1" colspan="1">22</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CN</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">3</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CNL</td>
<td align="left" rowspan="1" colspan="1">31</td>
<td align="left" rowspan="1" colspan="1">42</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">38</td>
<td align="left" rowspan="1" colspan="1">8</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">NL</td>
<td align="left" rowspan="1" colspan="1">39</td>
<td align="left" rowspan="1" colspan="1">53</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">61</td>
<td align="left" rowspan="1" colspan="1">22</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TN</td>
<td align="left" rowspan="1" colspan="1">16</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">27</td>
<td align="left" rowspan="1" colspan="1">21</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TNL</td>
<td align="left" rowspan="1" colspan="1">74</td>
<td align="left" rowspan="1" colspan="1">83</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">78</td>
<td align="left" rowspan="1" colspan="1">35</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TX</td>
<td align="left" rowspan="1" colspan="1">27</td>
<td align="left" rowspan="1" colspan="1">47</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">78</td>
<td align="left" rowspan="1" colspan="1">62</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Other</td>
<td align="left" rowspan="1" colspan="1">19</td>
<td align="left" rowspan="1" colspan="1">13</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">32</td>
<td align="left" rowspan="1" colspan="1">10</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Total NLR</td>
<td align="left" rowspan="1" colspan="1">226</td>
<td align="left" rowspan="1" colspan="1">292</td>
<td align="left" rowspan="1" colspan="1">32</td>
<td align="left" rowspan="1" colspan="1">337</td>
<td align="left" rowspan="1" colspan="1">183</td>
</tr>
<tr>
<td rowspan="4" align="left" colspan="1">TM‐LRR</td>
<td align="left" rowspan="1" colspan="1">RLK</td>
<td align="left" rowspan="1" colspan="1">542</td>
<td align="left" rowspan="1" colspan="1">676</td>
<td align="left" rowspan="1" colspan="1">134</td>
<td align="left" rowspan="1" colspan="1">808</td>
<td align="left" rowspan="1" colspan="1">438</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">RLP</td>
<td align="left" rowspan="1" colspan="1">109</td>
<td align="left" rowspan="1" colspan="1">122</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">163</td>
<td align="left" rowspan="1" colspan="1">58</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TM‐CC</td>
<td align="left" rowspan="1" colspan="1">159</td>
<td align="left" rowspan="1" colspan="1">240</td>
<td align="left" rowspan="1" colspan="1">26</td>
<td align="left" rowspan="1" colspan="1">299</td>
<td align="left" rowspan="1" colspan="1">172</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Total TM‐LRR</td>
<td align="left" rowspan="1" colspan="1">810</td>
<td align="left" rowspan="1" colspan="1">1038</td>
<td align="left" rowspan="1" colspan="1">172</td>
<td align="left" rowspan="1" colspan="1">1270</td>
<td align="left" rowspan="1" colspan="1">668</td>
</tr>
<tr>
<td colspan="2" align="left" rowspan="1">Grand total</td>
<td align="left" rowspan="1" colspan="1">1036</td>
<td align="left" rowspan="1" colspan="1">1330</td>
<td align="left" rowspan="1" colspan="1">204</td>
<td align="left" rowspan="1" colspan="1">1607</td>
<td align="left" rowspan="1" colspan="1">851</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270087-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270087-tbl1-note-0001"><p>Abbreviations: CC, coiled‐coil; CNL, CC‐NBS‐LRR; LRR, leucine‐rich repeat; NBS, nucleotide‐binding site; NLR, nucleotide‐binding site leucine‐rich repeat; RGA, resistance gene analog; RLK, receptor‐like kinase; RLP, receptor‐like protein; TIR, toll/interleukin‐1 receptor; TNL, TIR‐NBS‐LRR; TM, transmembrane.</p></div></div></section><p>There were more RGAs predicted in the C subgenome (1330) compared with the B subgenome (1036), and a further 204 RGAs were predicted on unplaced contigs. The RGAs were unevenly distributed among the chromosomes with the B subgenome chromosomes containing an average of 130 RGAs, ranging from 90 to 196 per chromosome, while the C subgenome chromosomes contained an average of 148 RGAs and ranged from 120 to 197 per chromosome. Chromosomes B01 (196) and B08 (90) carried the most and the least number of RGAs within the B subgenome, respectively. In the C subgenome, chromosome C06 harbored the greatest number of RGAs (197), while C09 (120) contained the least number of RGAs (Figures <a href="#tpg270087-fig-0001" class="usa-link">1</a> and <a href="#tpg270087-fig-0002" class="usa-link">2</a>).</p>
<figure class="fig xbox font-sm" id="tpg270087-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7126/12326289/ec18870e1cae/TPG2-18-e70087-g002.jpg" loading="lazy" id="jats-graphic-1" height="807" width="600" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270087-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Chromosome distribution of nucleotide‐binding site leucine‐rich repeat (NLR) genes in the B and C subgenomes of <em>Brassica carinata</em>. CC, coiled‐coil; CNL, CC‐NBS‐LRR; LRR, leucine‐rich repeat; NBS, nucleotide‐binding site; TIR, toll/interleukin‐1 receptor; TNL, TIR‐NBS‐LRR.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270087-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7126/12326289/9d044a6b02c2/TPG2-18-e70087-g003.jpg" loading="lazy" id="jats-graphic-3" height="795" width="600" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270087-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Chromosome distribution of transmembrane (TM)‐leucine‐rich repeat (LRR) genes in B and C subgenomes of <em>Brassica carinata</em>. CC, coiled‐coil; RLK, receptor‐like kinase; RLP, receptor‐like protein; TM, transmembrane.</p></figcaption></figure><p>The two main RGA classes differed in genome distribution. TM‐LRRs were mostly found on chromosome B01 (163) followed by C01 (149), while they were least abundant on B08 (73). The NLR genes were most abundant on B07 (69) followed by chromosomes B03 and C04 (both 47), while B02 harbored the least number of NLR genes. There were more TM‐LRR genes (172) predicted in the unplaced contigs than NLRs (32) (Table <a href="#tpg270087-tbl-0002" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="tpg270087-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>The number of RGAs in the B and C subgenomes of <em>Brassica carinata</em>.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead>
<tr>
<th rowspan="2" align="left" colspan="1">Chromosome</th>
<th colspan="3" style="border-bottom:solid 1px #000000" align="left" rowspan="1">TM‐LRR</th>
<th rowspan="2" align="left" colspan="1">Total TM‐LRR</th>
<th colspan="8" style="border-bottom:solid 1px #000000" align="left" rowspan="1">NLR</th>
<th rowspan="2" align="left" colspan="1">Total NLR</th>
<th rowspan="2" align="left" colspan="1">Grand total</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">RLK</th>
<th align="left" rowspan="1" colspan="1">RLP</th>
<th align="left" rowspan="1" colspan="1">TM‐CC</th>
<th align="left" rowspan="1" colspan="1">NBS</th>
<th align="left" rowspan="1" colspan="1">CN</th>
<th align="left" rowspan="1" colspan="1">CNL</th>
<th align="left" rowspan="1" colspan="1">NL</th>
<th align="left" rowspan="1" colspan="1">TN</th>
<th align="left" rowspan="1" colspan="1">TNL</th>
<th align="left" rowspan="1" colspan="1">TX</th>
<th align="left" rowspan="1" colspan="1">Other</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">B01</td>
<td align="left" rowspan="1" colspan="1">115</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">31</td>
<td align="left" rowspan="1" colspan="1">163</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">33</td>
<td align="left" rowspan="1" colspan="1">196</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B02</td>
<td align="left" rowspan="1" colspan="1">59</td>
<td align="left" rowspan="1" colspan="1">13</td>
<td align="left" rowspan="1" colspan="1">20</td>
<td align="left" rowspan="1" colspan="1">92</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">104</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B03</td>
<td align="left" rowspan="1" colspan="1">93</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">18</td>
<td align="left" rowspan="1" colspan="1">122</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">47</td>
<td align="left" rowspan="1" colspan="1">169</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B04</td>
<td align="left" rowspan="1" colspan="1">53</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">20</td>
<td align="left" rowspan="1" colspan="1">90</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">104</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B05</td>
<td align="left" rowspan="1" colspan="1">63</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">91</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">20</td>
<td align="left" rowspan="1" colspan="1">111</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B06</td>
<td align="left" rowspan="1" colspan="1">53</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">19</td>
<td align="left" rowspan="1" colspan="1">83</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">97</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B07</td>
<td align="left" rowspan="1" colspan="1">55</td>
<td align="left" rowspan="1" colspan="1">18</td>
<td align="left" rowspan="1" colspan="1">23</td>
<td align="left" rowspan="1" colspan="1">96</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">13</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">28</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">69</td>
<td align="left" rowspan="1" colspan="1">165</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">B08</td>
<td align="left" rowspan="1" colspan="1">51</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">73</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">90</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Total B genome</td>
<td align="left" rowspan="1" colspan="1">542</td>
<td align="left" rowspan="1" colspan="1">109</td>
<td align="left" rowspan="1" colspan="1">159</td>
<td align="left" rowspan="1" colspan="1">810</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">31</td>
<td align="left" rowspan="1" colspan="1">39</td>
<td align="left" rowspan="1" colspan="1">16</td>
<td align="left" rowspan="1" colspan="1">74</td>
<td align="left" rowspan="1" colspan="1">27</td>
<td align="left" rowspan="1" colspan="1">19</td>
<td align="left" rowspan="1" colspan="1">226</td>
<td align="left" rowspan="1" colspan="1">1036</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Average</td>
<td align="left" rowspan="1" colspan="1">68</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">20</td>
<td align="left" rowspan="1" colspan="1">101</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">28</td>
<td align="left" rowspan="1" colspan="1">130</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">C01</td>
<td align="left" rowspan="1" colspan="1">103</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">29</td>
<td align="left" rowspan="1" colspan="1">149</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">28</td>
<td align="left" rowspan="1" colspan="1">177</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">C02</td>
<td align="left" rowspan="1" colspan="1">75</td>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">18</td>
<td align="left" rowspan="1" colspan="1">108</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">13</td>
<td align="left" rowspan="1" colspan="1">121</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">C03</td>
<td align="left" rowspan="1" colspan="1">73</td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">31</td>
<td align="left" rowspan="1" colspan="1">114</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">40</td>
<td align="left" rowspan="1" colspan="1">154</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">C04</td>
<td align="left" rowspan="1" colspan="1">76</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">31</td>
<td align="left" rowspan="1" colspan="1">115</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">10</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">47</td>
<td align="left" rowspan="1" colspan="1">162</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">C05</td>
<td align="left" rowspan="1" colspan="1">64</td>
<td align="left" rowspan="1" colspan="1">16</td>
<td align="left" rowspan="1" colspan="1">26</td>
<td align="left" rowspan="1" colspan="1">106</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">20</td>
<td align="left" rowspan="1" colspan="1">126</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">C06</td>
<td align="left" rowspan="1" colspan="1">93</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">47</td>
<td align="left" rowspan="1" colspan="1">152</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">45</td>
<td align="left" rowspan="1" colspan="1">197</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">C07</td>
<td align="left" rowspan="1" colspan="1">56</td>
<td align="left" rowspan="1" colspan="1">20</td>
<td align="left" rowspan="1" colspan="1">23</td>
<td align="left" rowspan="1" colspan="1">99</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">40</td>
<td align="left" rowspan="1" colspan="1">139</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">C08</td>
<td align="left" rowspan="1" colspan="1">67</td>
<td align="left" rowspan="1" colspan="1">15</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">99</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">35</td>
<td align="left" rowspan="1" colspan="1">134</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">C09</td>
<td align="left" rowspan="1" colspan="1">69</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">18</td>
<td align="left" rowspan="1" colspan="1">96</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">7</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">0</td>
<td align="left" rowspan="1" colspan="1">24</td>
<td align="left" rowspan="1" colspan="1">120</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Total C genome</td>
<td align="left" rowspan="1" colspan="1">676</td>
<td align="left" rowspan="1" colspan="1">122</td>
<td align="left" rowspan="1" colspan="1">240</td>
<td align="left" rowspan="1" colspan="1">1038</td>
<td align="left" rowspan="1" colspan="1">29</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">42</td>
<td align="left" rowspan="1" colspan="1">53</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">83</td>
<td align="left" rowspan="1" colspan="1">47</td>
<td align="left" rowspan="1" colspan="1">13</td>
<td align="left" rowspan="1" colspan="1">292</td>
<td align="left" rowspan="1" colspan="1">1330</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Average</td>
<td align="left" rowspan="1" colspan="1">75</td>
<td align="left" rowspan="1" colspan="1">14</td>
<td align="left" rowspan="1" colspan="1">27</td>
<td align="left" rowspan="1" colspan="1">115</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">6</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">9</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">32</td>
<td align="left" rowspan="1" colspan="1">148</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Unplaced contigs</td>
<td align="left" rowspan="1" colspan="1">134</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">26</td>
<td align="left" rowspan="1" colspan="1">172</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">1</td>
<td align="left" rowspan="1" colspan="1">3</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">11</td>
<td align="left" rowspan="1" colspan="1">8</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">32</td>
<td align="left" rowspan="1" colspan="1">204</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Total</td>
<td align="left" rowspan="1" colspan="1">1352</td>
<td align="left" rowspan="1" colspan="1">243</td>
<td align="left" rowspan="1" colspan="1">425</td>
<td align="left" rowspan="1" colspan="1">2020</td>
<td align="left" rowspan="1" colspan="1">47</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">76</td>
<td align="left" rowspan="1" colspan="1">96</td>
<td align="left" rowspan="1" colspan="1">35</td>
<td align="left" rowspan="1" colspan="1">168</td>
<td align="left" rowspan="1" colspan="1">82</td>
<td align="left" rowspan="1" colspan="1">34</td>
<td align="left" rowspan="1" colspan="1">550</td>
<td align="left" rowspan="1" colspan="1">2570</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270087-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270087-tbl2-note-0001"><p>Abbreviations: CC, coiled‐coil; CNL, CC‐NBS‐LRR; LRR, leucine‐rich repeat; NBS, nucleotide‐binding site; NLR, nucleotide‐binding site leucine‐rich repeat; RGA, resistance gene analog; RLK, receptor‐like kinase; RLP, receptor‐like protein; TIR, toll/interleukin‐1 receptor; TNL, TIR‐NBS‐LRR; TM, transmembrane.</p></div></div></section><p>The TM‐LRR subclasses were found in all the chromosomes, with RLKs being most abundant on chromosome B01 (115) and least abundant on B08 (51). RLPs were mostly located on C07 (20), while TM‐CCs were most abundant on C06 (47). In contrast to TM‐LRRs, not all NLR subclasses were present in all 17 chromosomes. For instance, CN genes were only present on seven chromosomes (B02, B07, C03, C04, C07, C08, and C09). Subclass Other was absent on six chromosomes (B04, B06, B07, C04, C05, and C09), and TN genes were not found on five chromosomes (B02, B04, C01, C06, and C09). NBS genes were absent on two chromosomes (B08 and C01), while CNL, NL, and TX genes were absent on chromosomes B05, B03, and B06, respectively (Table <a href="#tpg270087-tbl-0002" class="usa-link">2</a>).</p></section><section id="tpg270087-sec-0130"><h3 class="pmc_sec_title">3.2. Duplication of RGAs</h3>
<p>A total of 1675 RGAs, accounting for 65.2% of the total number, were duplicated in the <em>B. carinata</em> genome (Table <a href="#tpg270087-tbl-0003" class="usa-link">3</a>). Of these, 1133 (67.6%) exhibited intergenomic duplications, while 542 (32.4%) were duplicated within the same genome (intragenomic duplication). In general, there were more TM‐LRRs (1361 or 81.3% of the total duplicated RGAs) than NLRs (314 or 18.7%) that were affected by the two duplication events. Within the TM‐LRR class, RLK genes showed the greatest number of duplicated genes with 1003 (59.9%), followed by TM‐CCs with 237 (14.1%) and RLPs with 121 (7.2%). Among the NLRs, TNLs showed the greatest number of duplicated genes with 79 (4.7%) members, followed by NLs with 64 (3.8%), while CN genes were the least affected with nine (0.5%) duplicated gene members.</p>
<section class="tw xbox font-sm" id="tpg270087-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>The number of <em>Brassica carinata</em> RGAs showing intergenomic and intragenomic duplications.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead>
<tr>
<th rowspan="2" align="left" colspan="1">Type</th>
<th rowspan="2" align="left" colspan="1">Subtype</th>
<th colspan="2" style="border-bottom:solid 1px #000000" align="left" rowspan="1">Duplication type</th>
<th rowspan="2" align="left" colspan="1">Grand total</th>
</tr>
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Intergenomic</th>
<th align="left" rowspan="1" colspan="1">Intragenomic</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="8" align="left" colspan="1">NLR</td>
<td align="left" rowspan="1" colspan="1">CN</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">5</td>
<td align="left" rowspan="1" colspan="1">9</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">CNL</td>
<td align="left" rowspan="1" colspan="1">29</td>
<td align="left" rowspan="1" colspan="1">16</td>
<td align="left" rowspan="1" colspan="1">45</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">NBS</td>
<td align="left" rowspan="1" colspan="1">12</td>
<td align="left" rowspan="1" colspan="1">16</td>
<td align="left" rowspan="1" colspan="1">28</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">NL</td>
<td align="left" rowspan="1" colspan="1">31</td>
<td align="left" rowspan="1" colspan="1">33</td>
<td align="left" rowspan="1" colspan="1">64</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TN</td>
<td align="left" rowspan="1" colspan="1">17</td>
<td align="left" rowspan="1" colspan="1">4</td>
<td align="left" rowspan="1" colspan="1">21</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TNL</td>
<td align="left" rowspan="1" colspan="1">52</td>
<td align="left" rowspan="1" colspan="1">27</td>
<td align="left" rowspan="1" colspan="1">79</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TX</td>
<td align="left" rowspan="1" colspan="1">29</td>
<td align="left" rowspan="1" colspan="1">24</td>
<td align="left" rowspan="1" colspan="1">53</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">OTHER</td>
<td align="left" rowspan="1" colspan="1">13</td>
<td align="left" rowspan="1" colspan="1">2</td>
<td align="left" rowspan="1" colspan="1">15</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Total NLR</td>
<td align="left" rowspan="1" colspan="1">187</td>
<td align="left" rowspan="1" colspan="1">127</td>
<td align="left" rowspan="1" colspan="1">314</td>
</tr>
<tr>
<td rowspan="3" align="left" colspan="1">TM‐LRR</td>
<td align="left" rowspan="1" colspan="1">RLK</td>
<td align="left" rowspan="1" colspan="1">719</td>
<td align="left" rowspan="1" colspan="1">284</td>
<td align="left" rowspan="1" colspan="1">1003</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">RLP</td>
<td align="left" rowspan="1" colspan="1">81</td>
<td align="left" rowspan="1" colspan="1">40</td>
<td align="left" rowspan="1" colspan="1">121</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TM‐CC</td>
<td align="left" rowspan="1" colspan="1">146</td>
<td align="left" rowspan="1" colspan="1">91</td>
<td align="left" rowspan="1" colspan="1">237</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Total TM‐LRR</td>
<td align="left" rowspan="1" colspan="1">946</td>
<td align="left" rowspan="1" colspan="1">415</td>
<td align="left" rowspan="1" colspan="1">1361</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1"></td>
<td align="left" rowspan="1" colspan="1">Grand total</td>
<td align="left" rowspan="1" colspan="1">1133</td>
<td align="left" rowspan="1" colspan="1">542</td>
<td align="left" rowspan="1" colspan="1">1675</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270087-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270087-tbl3-note-0001"><p>Abbreviations: CC, coiled‐coil; CNL, CC‐NBS‐LRR; LRR, leucine‐rich repeat; NBS, nucleotide‐binding site; NLR, nucleotide‐binding site leucine‐rich repeat; RGA, resistance gene analog; RLK, receptor‐like kinase; RLP, receptor‐like protein; TIR, toll/interleukin‐1 receptor; TNL, TIR‐NBS‐LRR; TM, transmembrane.</p></div></div></section><p>The C subgenome contained a higher total and average number of duplicated RGAs (911 total and 101 average per chromosome) compared to the B subgenome (764 total and 96 average per chromosome) (Table <a href="#tpg270087-supinfo-0001" class="usa-link">S2</a>). Similarly, the C subgenome had a higher average number of RGAs (37) per chromosome affected by intragenomic duplication than the B subgenome (26). However, the B subgenome had a higher average number of RGAs (70) per chromosome affected by intergenomic duplication compared to the C subgenome (64). Within the B subgenome, chromosome B03 contained the greatest number of RGAs affected by intergenomic duplication with 99, followed by B01 with 83, while B06 had the least with 51. Chromosome C01 (110) contained the greatest number of intergenomic RGA duplicates in the C subgenome, followed by C04 (105), while C09 had the least number with 21. Thus, there is greater range in the number of intergenomic RGA duplicates along the C subgenome (21‐110 RGAs) than in the B subgenome (51‐99 RGAs).</p>
<p>The RGAs exhibiting intragenomic duplication were further classified as either tandem or segmentally duplicated (Table <a href="#tpg270087-supinfo-0001" class="usa-link">S3</a>). Of the 542 genes showing intragenomic duplications, 283 (or 16.9% of the total duplicated genes) were tandemly duplicated and 259 (15.5%) were segmentally duplicated. There were twice as many segmentally duplicated RGAs on the C subgenome (179 total; 20 on average per chromosome) than the B subgenome (80 total; 10 on average). This difference was less pronounced for tandemly duplicated genes with the C subgenome containing 155 members (17 on average per chromosome) and the B subgenome with 128 (16 on average per chromosome). Among the chromosomes, C06 contained the greatest number of segmentally duplicated RGAs (62), followed by C07 (25) and B01 (23), while B02 harbored the least number (3). Chromosome B01 contained the highest number (43) of tandemly duplicated genes, followed by C05 (34) and C07 (26), while B02 exhibited the smallest number with only one tandemly duplicated RGA. Most of the chromosomes contained more TM‐LRRs compared with NLRs that were affected by intragenomic duplications. However, there were no significant differences in the number of tandem versus segmentally duplicated RGAs for both the NLR (<em>t</em>(28) = −1.5, <em>p </em>= 0.07) and TM‐LRR (<em>t</em>(30) = 0.13, <em>p </em>= 0.44) classes. This suggests that both duplication events affect the RGAs regardless of their classification.</p></section><section id="tpg270087-sec-0140"><h3 class="pmc_sec_title">3.3. Physical clustering of RGAs</h3>
<p>Resistance genes are often arranged in clusters; thus, this genomic organization was also examined in <em>B. carinata</em>. The clustering analysis revealed that 184 RGAs in <em>B. carinata</em>, representing 7.2% of the total RGAs, were distributed in 57 physical clusters (Figure <a href="#tpg270087-fig-0003" class="usa-link">3</a>). Of these, 19 were homogeneous TM‐LRR clusters and 14 were homogeneous NLR clusters. The remaining 24 clusters were heterogeneous, consisting of both TM‐LRRs and NLRs.</p>
<figure class="fig xbox font-sm" id="tpg270087-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7126/12326289/15e7b7f62481/TPG2-18-e70087-g004.jpg" loading="lazy" id="jats-graphic-5" height="820" width="600" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270087-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of the physical clusters of the resistance gene analogs (RGAs) in the B and C subgenomes of <em>Brassica carinata</em>. Abbreviations: CC, coiled‐coil; CNL, CC‐NBS‐LRR; LRR, leucine‐rich repeat; NBS, nucleotide‐binding site; NLR, nucleotide‐binding site leucine‐rich repeat; RLK, receptor‐like kinase; RLP, receptor‐like protein; TIR, toll/interleukin‐1 receptor; TNL, TIR‐NBS‐LRR; TM, transmembrane.</p></figcaption></figure><p>The C subgenome contained a higher number of physical clusters (34) and clustered RGAs (97) compared with the B subgenome, which had 23 clusters harboring 87 RGAs. However, chromosomes B03 and B07 contained the greatest number of physical clusters (six each) and clustered RGAs (22 each). In the C subgenome, chromosome C07 contained the greatest number of clustered RGAs (17), which were found on five clusters. Of the clustered RGAs, 85 were NLRs including 20 NLs, 16 NBS and TNLs, 11 TNs, 10 TXs, five CNLs, four Other, and three CNs. There were 99 clustered TM‐LRRs, of which 83 were RLKs, 11 were RLKs, and five were TM‐CCs. Chromosome B07 (14) harbored the greatest number of clustered NLRs, while B03 and C09 contained the greatest number of clustered TM‐LRRs.</p>
<p>The physical clustering and the gene duplication status of the RGAs were also compared. There were 139 physically clustered RGAs affected by gene duplication events. Of these, 64 were affected by intergenomic duplications, while 75 showed intragenomic duplication. Of the clustered genes affected by intragenomic duplications, 61 were tandemly duplicated and 14 were segmentally duplicated. There were 57 physically clustered NLRs that exhibited duplication, of which 30 experienced intragenomic duplication (26 tandem and four segmental) and 27 were affected by intergenomic duplication. There were 82 physically clustered TM‐LRRs affected by gene duplication events, 37 of which exhibited intergenomic duplication, while 45 showed intragenomic duplication events (35 tandem and 10 segmental).</p></section><section id="tpg270087-sec-0150"><h3 class="pmc_sec_title">3.4. Phylogenetic analysis of <em>B. carinata</em> RGAs</h3>
<p>The evolutionary relationship of the RGAs was examined by constructing maximum likelihood phylogenetic trees presented as cladograms in Figures <a href="#tpg270087-fig-0004" class="usa-link">4</a> and <a href="#tpg270087-fig-0005" class="usa-link">5</a>. Separate trees were constructed for TM‐LRRs (RLKs, RLPs, and TM‐CCs) and NLR genes. The 2020 TM‐LRRs were divided into ten distinct clades, with the largest clade (X) consisting of 771 gene members, followed by clade IX with 450 members and clade II with 262 members, while the smallest clade (III) only contained six members (Figure <a href="#tpg270087-fig-0004" class="usa-link">4</a>). The different TM‐LRR RGA subclasses were generally found in the same clade. The RLKs were the most dominant subclass in clades X, VIII, VII, VI, V, and IV, while the TM‐CCs were prominently found on clade IX. The majority of the RLPs were found on clade II but this clade also included a similar number of RLKs. Clade I contained a mixture of RLPs, RLKs, and TM‐CCs in almost equal number. Other RGA classes can also be found in clades dominated by one type of RGA class. For instance, clade X, which consisted mostly of RLKs, also included several TM‐CCs (15) and RLPs (11) in some of its subclades. Similarly, the TM‐CC clade (IX) included 48 RLKs and 34 RLPs.</p>
<figure class="fig xbox font-sm" id="tpg270087-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7126/12326289/be9b3994613d/TPG2-18-e70087-g005.jpg" loading="lazy" id="jats-graphic-7" height="644" width="600" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270087-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Maximum likelihood phylogenetic tree of transmembrane (TM)‐leucine‐rich repeat (LRR) resistance gene analogs (RGAs) (receptor‐like kinase [RLK], receptor‐like protein [RLP], and TM‐coiled‐coil [CC]). The cladogram was inferred based on alignment of 2020 protein sequences. The branches are colored to distinguish the different clades (I–X). The outer ring colors indicate the RGA type.</p></figcaption></figure><figure class="fig xbox font-sm" id="tpg270087-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7126/12326289/20d5c995d3e8/TPG2-18-e70087-g006.jpg" loading="lazy" id="jats-graphic-9" height="630" width="600" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270087-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Maximum likelihood phylogenetic tree of nucleotide‐binding site leucine‐rich repeat (NLR) resistance gene analogs (RGAs). The cladogram was inferred based on alignment of 550 protein sequences. The branches are colored to distinguish the different clades (I–XIV). The outer ring colors indicate the RGA type. CC, coiled‐coil; CNL, CC‐NBS‐LRR; LRR, leucine‐rich repeat; NBS, nucleotide‐binding site; TIR, toll/interleukin‐1 receptor; TNL, TIR‐NBS‐LRR.</p></figcaption></figure><p>The 550 NLR genes formed 14 distinct clades (Figure <a href="#tpg270087-fig-0005" class="usa-link">5</a>), which was higher compared to TM‐LRRs. Notably, the first nine clades (I–IX) were very divergent and formed small clades containing a maximum of seven members. Most of the gene members in these small clades were RGAs with TIR domains. In one extreme case, a TNL gene (<em>BcaB07g3090</em>) was very distinct and separated from the rest of the clades. Clade XIV was the largest with 190 gene members, of which majority contained the CC domain (CNL and CN). This was followed by clade X with 127 gene members, which were mostly TIR‐containing RGAs (TNL, TN, and TX). Clades XI and XII each had 79 gene members belonging to different NLR subclasses. For example, although more than half of the gene members in clade XI were TIR‐carrying RGAs, the majority of the NBS genes were also located in this clade. Similarly, within clade XII, several NLs (14) and Other (12) subclass were found along with TIR‐containing genes.</p></section><section id="tpg270087-sec-0160"><h3 class="pmc_sec_title">3.5. RGA comparison between <em>B. carinata</em> and its diploid progenitor species</h3>
<p>In general, the <em>B. carinata</em> genome contained more predicted RGAs compared to its diploid progenitor species. However, considering each subgenome, <em>B. nigra</em> contained more RGAs (1607) compared to the B subgenome of <em>B. carinata</em> (1036). In contrast, the <em>B. carinata</em> C subgenome contained more RGAs (1330) compared to <em>B. oleracea</em> (851). There was also an overall increase in the average length of RGAs in <em>B. carinata</em> (5724 bp) versus <em>B. nigra</em> (3650 bp) and <em>B. oleracea</em> (2920 bp). This is equivalent to 1.6x and 2x increase in gene length relative to <em>B. nigra</em> and <em>B. oleracea</em>, respectively. On average, the NLR genes in <em>B. carinata</em> were 1.9x and 2.3x longer (6283 bp) compared with <em>B. nigra</em> (3346 bp) and <em>B. oleracea</em> (2745 bp), respectively. The TM‐LRRs in <em>B. carinata</em> (5827) were 1.5x and 2x longer compared to <em>B. nigra</em> (3800) and <em>B. oleracea</em> (2891), respectively. However, while there was a marked difference in the number of RGAs among species, the proportion of the two main RGA classes was the same for each of the species (approximately 79% TM‐LRR and 21% NLRs).</p>
<p>The distribution of the different RGA classes was compared between the B and C subgenomes of <em>B. carinata</em> relative to its progenitors (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S1</a>). A total of 337 NLRs and 1270 TM‐LRRs were identified in <em>B. nigra</em>, which was higher than in the B subgenome of <em>B. carinata</em> (226 NLRs and 810 TM‐LRRs). Similarly, all the <em>B. nigra</em> chromosomes had higher numbers of NLRs and TM‐LRRs compared with the <em>B. carinata</em> B subgenome, except for B01 and B07, which displayed the opposite pattern. The chromosomes with the greatest number of NLRs and TM‐LRRs also differed between the species. The NLRs were most abundant on B04 (107) in <em>B. nigra</em>, while in <em>B. carinata</em> they were mostly located on B07 (69). In terms of the TM‐LRRs, chromosomes B02 (227) and B01 (163) contained the highest count in <em>B. nigra</em> and <em>B. carinata</em>, respectively.</p>
<p>There were 183 NLRs and 668 TM‐LRR genes identified in <em>B. oleracea</em>, which were less than those predicted in <em>B. carinata</em> C subgenome with 292 NLRs and 1038 TM‐LRRs. Generally, there were more TM‐LRRs in all the C chromosomes of <em>B. carinata</em> than in <em>B. oleracea</em>. A similar pattern was also noted for NLRs, except for chromosome C02 of <em>B. oleracea</em>, which had more NLRs (29) than in <em>B. carinata</em> (13). The two species also differed in the chromosomes that carried the greatest number of NLRs and TM‐LRRs in the C subgenome. The highest number of NLRs in <em>B. carinata</em> C subgenome was found on C04 with 47, while this was on C07 in <em>B. oleracea</em> with 40. The highest number of TM‐LRRs in <em>B. carinata</em> was found on C06 with 152, while in <em>B. oleracea</em> this was exhibited by C03 with 110.</p>
<p>Overall, the number of physically clustered RGAs was the same in <em>B. carinata</em> and <em>B. nigra</em> with both species having 183. However, when considering the B genome only, <em>B. nigra</em> contained a significantly greater number of clustered RGAs compared with the B subgenome of <em>B. carinata</em> (87). In contrast, the <em>B. carinata</em> C subgenome harbored higher number of clustered RGAs (97) compared with <em>B. oleracea</em> (55). The proportion of clustered TM‐LRR and NLRs in each species was the same at 55% and 45%, respectively.</p></section><section id="tpg270087-sec-0170"><h3 class="pmc_sec_title">3.6. Analysis of homologs between <em>B. carinata</em> and its progenitor genomes</h3>
<p>Identification of conserved genes was performed between <em>B. carinata</em> and its progenitor genomes by comparing the protein sequences of the RGAs using BLAST. The analysis revealed that 1515 of the 1607 total RGAs in <em>B. nigra</em> were conserved in the <em>B. carinata</em> genome (Table <a href="#tpg270087-tbl-0004" class="usa-link">4</a>; Figure <a href="#tpg270087-fig-0006" class="usa-link">6</a>). Among the <em>B. nigra</em> TM‐LRRs, the RLKs were the most conserved RGA class with 800 of the total 808 RLKs found with a significant homolog match with <em>B. carinata</em> RGAs. The <em>B. nigra</em> TM‐CCs, on the other hand, showed the lowest conservation percentage among the RGA class with only 236 of the 299 (78.9%) <em>B. nigra</em> TM‐CCs with a homolog match with <em>B. carinata</em> RGAs. The RLPs showed 92.64% (151/163) conservation between <em>B. nigra</em> and <em>B. carinata</em>. Of the 337 <em>B. nigra</em> NLRs, 328 or 97.33% were retained in <em>B. carinata</em>.</p>
<section class="tw xbox font-sm" id="tpg270087-tbl-0004"><h4 class="obj_head">TABLE 4.</h4>
<div class="caption p"><p>Number of conserved RGAs between <em>Brassica</em> carinata and its progenitor genomes, <em>Brassica</em> nigra, and <em>Brassica</em> oleracea.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<thead><tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="1" colspan="1">Class</th>
<th align="left" rowspan="1" colspan="1">Conserved <em>B</em>. <em>nigra</em> RGA in <em>B. carinata</em>
</th>
<th align="left" rowspan="1" colspan="1">Percentage of conserved RGAs</th>
<th align="left" rowspan="1" colspan="1">Conserved <em>B. oleracea</em> RGA in <em>B. carinata</em>
</th>
<th align="left" rowspan="1" colspan="1">Percentage of conserved RGAs</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">NLR</td>
<td align="left" rowspan="1" colspan="1">328</td>
<td align="left" rowspan="1" colspan="1">97.33</td>
<td align="left" rowspan="1" colspan="1">176</td>
<td align="left" rowspan="1" colspan="1">96.17</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">RLK</td>
<td align="left" rowspan="1" colspan="1">800</td>
<td align="left" rowspan="1" colspan="1">99.01</td>
<td align="left" rowspan="1" colspan="1">436</td>
<td align="left" rowspan="1" colspan="1">99.54</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">RLP</td>
<td align="left" rowspan="1" colspan="1">151</td>
<td align="left" rowspan="1" colspan="1">92.64</td>
<td align="left" rowspan="1" colspan="1">53</td>
<td align="left" rowspan="1" colspan="1">91.38</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TM‐CC</td>
<td align="left" rowspan="1" colspan="1">236</td>
<td align="left" rowspan="1" colspan="1">78.93</td>
<td align="left" rowspan="1" colspan="1">143</td>
<td align="left" rowspan="1" colspan="1">83.14</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">Total</td>
<td align="left" rowspan="1" colspan="1">1515</td>
<td align="left" rowspan="1" colspan="1">94.28</td>
<td align="left" rowspan="1" colspan="1">808</td>
<td align="left" rowspan="1" colspan="1">94.95</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tpg270087-tbl-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="tpg270087-tbl4-note-0001"><p>Abbreviations: CC, coiled‐coil; NLR, nucleotide‐binding site leucine‐rich repeat; RGA, resistance gene analog; RLK, receptor‐like kinase; RLP, receptor‐like protein; TM, transmembrane.</p></div></div></section><figure class="fig xbox font-sm" id="tpg270087-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/7126/12326289/4c0cf9613d1a/TPG2-18-e70087-g001.jpg" loading="lazy" id="jats-graphic-11" height="434" width="700" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/tpg270087-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Distribution of conserved transmembrane (TM)‐leucine‐rich repeats (LRRs) (TM‐coiled‐coil [CC], receptor‐like kinase [RLK], and receptor‐like protein [RLP]) and nucleotide‐binding site leucine‐rich repeat (NLR) genes between the B and C subgenomes of <em>Brassica carinata</em> and its progenitor genomes, <em>Brassica nigra</em> and <em>Brassica oleracea</em>.</p></figcaption></figure><p>Significant RGA conservation was also observed between <em>B. oleracea</em> and the C subgenome of <em>B. carinata</em>. There were 808 <em>B. oleracea</em> RGAs out of the total 851 (94.95%) that had a significant homolog match with <em>B. carinata</em> RGAs. Similar to the B subgenome, RLKs showed the greatest number of conserved genes with 436 of the 438 (99.54%) significantly matched to a homolog in <em>B. carinata</em>. Out of the 183 <em>B. oleracea</em> NLR genes, 176 or 96.17% were found to be retained in <em>B. carinata</em>. There were 53 out of the 58 (91.38%) <em>B. oleracea</em> RLPs that were matched to a <em>B. carinata</em> homolog. The TM‐CCs showed 83.14% (143 out of 172 RGAs) gene conservation between <em>B. oleracea</em> and <em>B. carinata</em>.</p></section><section id="tpg270087-sec-0180"><h3 class="pmc_sec_title">3.7. Phylogenetic analysis of RGAs between <em>B. carinata</em> and its diploid progenitors</h3>
<p>Maximum likelihood phylogenetic trees were constructed for the <em>B. carinata</em> RGAs along with <em>B. nigra</em> and <em>B. oleracea</em> RGAs to study their evolutionary trajectories. Separate trees were built for each of the NLR, RLK, RLP, and TM‐CC RGA classes. The analysis revealed at least five patterns of evolution in the RGAs (Figures <a href="#tpg270087-supinfo-0001" class="usa-link">S2–S5</a>).</p>
<p>The first and most common pattern suggests a significant expansion of <em>B. carinata</em> RGAs from <em>B. nigra</em> and <em>B. oleracea</em>, which was consistent to the overall ratio of the RGAs among these species. This was observed in clusters where <em>B. carinata</em> RGAs were overrepresented compared to <em>B. nigra</em> and <em>B. oleracea</em>. There were 95 RLK clusters that displayed this pattern, 24 in NLRs, 25 in TM‐CCs, and 11 in RLPs. In the NLRs, for example, one cluster contained 12 RGAs from <em>B. carinata</em>, six from <em>B. nigra</em> and only one from <em>B. oleracea</em> (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S2B1</a>). Thus, in this cluster, the <em>B. carinata</em> RGA number doubled from its progenitor species. Another example in the TM‐CC class is a cluster that included nine <em>B. carinata</em> RGAs, four from <em>B. nigra</em>, and only one from <em>B. oleracea</em> (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S3B1</a>). In the RLK class, one cluster harbored 10 <em>B. carinata</em> RGAs, along with five from <em>B. nigra</em>, and <em>B. oleracea</em> (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S4B1</a>).</p>
<p>The second pattern indicates a straightforward inheritance of RGAs from <em>B. nigra</em> and <em>B. oleracea</em>. This is depicted in clusters where there is an equal or almost equal number of RGAs from <em>B. nigra</em>, <em>B. oleracea</em>, and <em>B. carinata</em>. There were 33 clusters showing this pattern within the RLK group and 21 in the NLR, while this was significantly lower in the TM‐CC and RLP groups with 10 and three clusters, respectively. One NLR group with this pattern included six RGAs from <em>B. carinata</em> and <em>B. nigra</em> and four from <em>B. oleracea</em> (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S2B2</a>). A cluster within the TM‐CC group contained seven RGAs from <em>B</em>. <em>carinata</em> and <em>B. nigra</em> and five from <em>B. oleracea</em> (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S3B2</a>). An RLK cluster displaying this trajectory included 11 <em>B. carinata</em>, 10 <em>B. nigra</em>, and nine <em>B. oleracea</em> RGAs (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S4B2</a>). In the RLP group, a cluster showing this pattern included five <em>B. carinata</em> and four <em>B. nigra</em> and <em>B. oleracea</em> RGAs (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S5B2</a>).</p>
<p>The third evolutionary pattern is represented in clusters where the members were skewed toward one of the progenitors, either <em>B. nigra</em> or <em>B. oleracea</em>. This suggests that the RGAs were preferentially inherited from one of the progenitor species. Within NLRs, there were 11 groups that were skewed toward <em>B. nigra</em> and six groups toward <em>B. oleracea</em> (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S2B3</a>). Within the TM‐CC class, six groups were skewed for <em>B. nigra</em>, while five were for <em>B. oleracea</em>. Among the RLKs, 21 clusters were skewed for <em>B. nigra</em> and only five clusters for <em>B. oleracea</em>; however, these were generally small groups. Among the RLPs, ten and three groups were skewed toward <em>B. nigra</em> and <em>B. oleracea</em>, respectively. An example of a cluster that is skewed toward <em>B. nigra</em> can be found on Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S4B3</a> where eight RLKs from <em>B. carinata</em> were grouped with seven RLKs from <em>B. nigra</em> but none from <em>B. oleracea</em>. In the RLP group, a cluster that is skewed toward <em>B. oleracea</em> contained five <em>B. oleracea</em>, four <em>B. carinata</em>, and only three from <em>B. nigra</em> (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S5B3</a>).</p>
<p>The fourth pattern suggests loss of <em>B. carinata</em> RGAs from the two progenitor species. This is shown in groups where there were a smaller number of <em>B. carinata</em> RGAs compared to either of the progenitor species. This pattern is not generally common among the different RGA classes and was mainly observed in the TM‐CC group with nine clusters exhibiting this pattern (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S3B4</a>). For example, one TM‐CC cluster included 10 <em>B. nigra</em>, one <em>B. carinata</em>, and no <em>B. oleracea</em> RGAs. In NLRs, there were only three groups that showed this pattern. For instance, one group included five RGAs from <em>B. nigra</em>, three from <em>B. oleracea</em> and only one from <em>B. carinata</em> (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S2B4</a>). Only one RLP group exhibited this pattern, which included five <em>B. carinata</em> along with eight <em>B. nigra</em> RGAs (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S5B4</a>). Similarly, this pattern was not common among the RLKs with only three groups showing this pattern (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S4B4</a>). In one of these clusters, one <em>B. carinata</em> RGA was grouped with five and two RGAs from <em>B. nigra</em> and <em>B. oleracea</em>, respectively.</p>
<p>The last pattern suggests a rapid sequence divergence of RGAs, which is demonstrated in clusters where one or two members show extremely long branches compared to the other RGAs. In the NLRs, eight genes were identified showing this branching pattern (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S2B5</a>). Among the TM‐CCs, two genes exhibited this characteristic. Notably one TM‐CC cluster containing five genes (two <em>B. carinata</em> and three <em>B. oleracea</em>) also showed extremely long branches, separating them from the rest of the clusters (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S3B5</a>). Among the RLKs, there were three groups that contained members showing extremely longer branches compared to the rest of the RGAs within that group (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S4B.5</a>). In RLPs, there were two groups containing three and eight gene members that also showed this branching pattern (Figure <a href="#tpg270087-supinfo-0001" class="usa-link">S5B.5</a>)</p></section></section><section id="tpg270087-sec-0190"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<section id="tpg270087-sec-0200"><h3 class="pmc_sec_title">4.1. 
<em>Brassica carinata</em> RGAs compared with other <em>Brassica</em> species</h3>
<p>
<em>Brassica carinata</em> was the last species to be sequenced in the triangle of U, marking a significant milestone in <em>Brassica</em> genomics as it completed the genome information for all the six species in triangle of U. In this study, we utilized the first reference genome developed for <em>B. carinata</em> (zd‐1) (Song et al., <a href="#tpg270087-bib-0039" class="usa-link" aria-describedby="tpg270087-bib-0039">2021</a>) to identify and characterize the RGAs in this species. A total of 2570 RGAs were predicted, of which 2366 were anchored on the 17 chromosomes of <em>B. carinata</em> and an additional 204 on unplaced contigs. This number of RGAs is the greatest recorded to date among the triangle of U <em>Brassica</em> species, including the other two allotetraploid species, <em>B. napus</em> (1749) (Dolatabadian et al., <a href="#tpg270087-bib-0009" class="usa-link" aria-describedby="tpg270087-bib-0009">2020</a>) and <em>B. juncea</em> (1591) (Tirnaz et al., <a href="#tpg270087-bib-0043" class="usa-link" aria-describedby="tpg270087-bib-0043">2020</a>). Similar to the other <em>Brassica</em> species, the TM‐LRRs constitute the majority of the RGAs in <em>B. carinata</em>, which are four times more abundant than NLRs. We also predicted a higher number of NLR genes (an additional of 101 NLRs) compared to Song et al. (<a href="#tpg270087-bib-0039" class="usa-link" aria-describedby="tpg270087-bib-0039">2021</a>). Moreover, we included the TM‐CC genes and classified them under the TM‐LRR class. This R gene class has been neglected in previous genome‐wide identification studies; thus, this study is the first to comprehensively analyze all the known RGA classes in a <em>Brassica</em> species.</p>
<p>Not only did we identify a greater number of RGAs in <em>B. carinata</em>, but we also observed an overall increase in the gene size across all the RGA classes relative to its progenitor species, <em>B. nigra</em> and <em>B. oleracea</em>. This is different from the findings in <em>B. napus</em>, where the gene length of NLRs was generally decreased when compared with their diploid progenitor (<em>B. rapa</em> and <em>B. oleracea</em>) homologs (Fu et al., <a href="#tpg270087-bib-0010" class="usa-link" aria-describedby="tpg270087-bib-0010">2019</a>). The same observation was made in our previous investigation comparing RGAs in <em>B. napus</em> and <em>B. rapa</em> pangenomes (Amas et al., <a href="#tpg270087-bib-0005" class="usa-link" aria-describedby="tpg270087-bib-0005">2023</a>). The increase in RGA gene size in <em>B. carinata</em> is due to bigger introns and exons as compared to the progenitor species (Song et al., <a href="#tpg270087-bib-0039" class="usa-link" aria-describedby="tpg270087-bib-0039">2021</a>). This might be explained by the significant amount of repetitive sequence, accounting for 58% of <em>B. carinata</em>’s genome (Song et al., <a href="#tpg270087-bib-0039" class="usa-link" aria-describedby="tpg270087-bib-0039">2021</a>). In most plant species, a burst in transposable element (TE) activity has been observed after interspecific hybridization resulting in genome size expansion (Parisod et al., <a href="#tpg270087-bib-0029" class="usa-link" aria-describedby="tpg270087-bib-0029">2010</a>). Consequently, insertion of TEs in genes can introduce new introns and exons resulting in bigger genes (Cai et al., <a href="#tpg270087-bib-0007" class="usa-link" aria-describedby="tpg270087-bib-0007">2022</a>). After expansion, selection pressure is often relaxed, facilitating the retention of the expanded genome (Parisod et al., <a href="#tpg270087-bib-0029" class="usa-link" aria-describedby="tpg270087-bib-0029">2010</a>). Additionally, the retention of these big genes in <em>B. carinata</em> is likely promoted by the constrained selection and breeding activities in this species (Yim et al., <a href="#tpg270087-bib-0049" class="usa-link" aria-describedby="tpg270087-bib-0049">2022</a>). This is consistent to the findings of a recent study (Khedikar et al., <a href="#tpg270087-bib-0018" class="usa-link" aria-describedby="tpg270087-bib-0018">2020</a>) utilizing 620 global collections that revealed the low genetic diversity in <em>B. carinata</em>, which was attributed to the very limited hybridization events in this species (Khedikar et al., <a href="#tpg270087-bib-0018" class="usa-link" aria-describedby="tpg270087-bib-0018">2020</a>). Fixation of alleles in this species is further enhanced by geographical isolation and high inbreeding rates (Khedikar et al., <a href="#tpg270087-bib-0018" class="usa-link" aria-describedby="tpg270087-bib-0018">2020</a>).</p>
<p>In general, <em>B. carinata</em> has undergone dynamic genomic changes following the initial hybridization of the two progenitor species. Yim et al. (<a href="#tpg270087-bib-0049" class="usa-link" aria-describedby="tpg270087-bib-0049">2022</a>) found fewer syntenic blocks between <em>B. carinata</em> subgenomes and its progenitor species than between the other allotetraploids, <em>B. juncea</em> and <em>B. napus</em>, and their respective progenitor genomes. This finding corroborates with the comparative analysis of Song et al. (<a href="#tpg270087-bib-0039" class="usa-link" aria-describedby="tpg270087-bib-0039">2021</a>), which showed that <em>B. carinata</em> subgenomes exhibit lower degrees of collinearity with their progenitor species compared with <em>B. juncea</em> and <em>B. napus</em> and their progenitor species. Song et al. (<a href="#tpg270087-bib-0039" class="usa-link" aria-describedby="tpg270087-bib-0039">2021</a>) attributed this to the heightened chromosomal rearrangements and conversion between the B and C subgenomes of <em>B. carinata</em>. These genomic alterations ultimately resulted in uneven size and compositions of <em>B. carinata's</em> subgenomes in comparison to its progenitor species. This further explains the variation in the number and organization of RGAs in <em>B. carinata</em> compared with its progenitor species. Thus, the current genomic landscape of RGAs in <em>B. carinata</em> has been significantly influenced by the genome‐wide changes in both the B and C subgenomes following polyploidization.</p></section><section id="tpg270087-sec-0210"><h3 class="pmc_sec_title">4.2. Asymmetric genomic organization of RGAs in the <em>B. carinata</em> genome</h3>
<p>The <em>B. carinata</em> RGAs were unevenly distributed along the B and C subgenomes with some of the chromosomes showing contrasting RGA abundance patterns when compared to <em>B. nigra</em> and <em>B. oleracea</em>. However, the results from the homology analysis suggest that the RGAs between these species were significantly conserved. This concurs with the findings of Yim et al. (<a href="#tpg270087-bib-0049" class="usa-link" aria-describedby="tpg270087-bib-0049">2022</a>), which showed that <em>B. carinata</em> retained the greatest number of whole‐genome duplication‐derived genes and experienced the least gene loss among the members of the <em>Brassica</em> triangle of U (<a href="#tpg270087-bib-0027" class="usa-link" aria-describedby="tpg270087-bib-0027">1935</a>). Estimates of the formation of <em>B. carinata</em> differ among studies, but it is generally agreed that this species was formed earlier than <em>B. napus</em> but later than <em>B. juncea</em> (Song et al., <a href="#tpg270087-bib-0039" class="usa-link" aria-describedby="tpg270087-bib-0039">2021</a>; Yim et al., <a href="#tpg270087-bib-0049" class="usa-link" aria-describedby="tpg270087-bib-0049"><sup>2022</sup></a>). However, compared to the other two allotetraploid species, <em>B. carinata</em> is considered an orphan crop and has experienced minimal selection (Thakur et al., <a href="#tpg270087-bib-0042" class="usa-link" aria-describedby="tpg270087-bib-0042">2019</a>). Thus, it is expected that <em>B. carinata</em> has retained most of the genomic features from its diploid progenitors.</p>
<p>Genomic studies have long established that gene duplication is ubiquitous in <em>Brassica</em> genomes due to their redundant evolution. The <em>B. carinata</em> genome is no exception as 65% of the RGAs were found to be duplicated. This is higher than in <em>B. napus</em> (50%) (Alamery et al., <a href="#tpg270087-bib-0004" class="usa-link" aria-describedby="tpg270087-bib-0004">2018</a>) but lower compared to <em>B. juncea</em> (78%) (Yang et al., <a href="#tpg270087-bib-0048" class="usa-link" aria-describedby="tpg270087-bib-0048">2021</a>). We observed a disparity in the patterns of RGA duplication between the two subgenomes. The C subgenome contained more RGAs per chromosome affected by intragenomic duplications compared to the B subgenome. In contrast, the B subgenome contained a higher number of RGAs showing intergenomic duplications despite having fewer RGAs per chromosome compared with the C subgenome. The <em>Brassica</em> C genome has experienced more genomic rearrangements compared with the B genome (Song et al., <a href="#tpg270087-bib-0039" class="usa-link" aria-describedby="tpg270087-bib-0039">2021</a>), which likely contributes to the creation of more intragenomic duplication events that affected the RGAs in this subgenome. Conversely, the B genome is more stable than the C genome, which was attributed to its earlier divergence from their common ancestor (Paritosh et al., <a href="#tpg270087-bib-0030" class="usa-link" aria-describedby="tpg270087-bib-0030">2014</a>). However, gene copies from the B genome are more retained compared to those coming from the C genome after homoeologous transfer following homoeologous exchanges (HEs) (Yim et al., <a href="#tpg270087-bib-0049" class="usa-link" aria-describedby="tpg270087-bib-0049">2022</a>). This preferential retention of homoeologous gene copies has been linked to subgenome dominance and likely explains why there were more RGAs detected as intergenomic duplicates in the B than the C subgenome. Subgenome dominance has also been previously reported in the other two allopolyploid <em>Brassica</em> species. The B subgenome of <em>B. juncea</em> showed dominance over its A subgenome in terms of genome size, gene and TE content, and degree of genetic recombination (Paritosh et al., <a href="#tpg270087-bib-0031" class="usa-link" aria-describedby="tpg270087-bib-0031">2021</a>), while in <em>B. napus</em>, the A subgenome exhibits dominance over its C subgenome in terms of the number of stress tolerance genes (Lu et al., <a href="#tpg270087-bib-0023" class="usa-link" aria-describedby="tpg270087-bib-0023">2019</a>). Hence, subgenome dominance seems to be a hallmark in allopolyploid <em>Brassica</em> species, which can serve as a genomic signature for certain traits including disease resistance.</p>
<p>The <em>B. carinata</em> RGAs formed physical clusters, which were distributed asymmetrically across the genome. Among the chromosomes, B03 and B07 contained the most physical clusters and clustered genes. Interestingly, a quantitative trait locus (QTL) for black rot (<em>Xanthomonas campestris</em> pv. <em>campestris</em>, race 1) has also been identified on chromosome B07 but the causal gene has not been identified (Sharma et al., <a href="#tpg270087-bib-0038" class="usa-link" aria-describedby="tpg270087-bib-0038">2016</a>). Genes conferring resistance against several bacterial pathogens, including the related species <em>Xanthomonas oryzae</em>, affecting rice, are known to be organized in gene clusters; thus, it will be interesting to further examine if this gene clustering pattern on chromosome B07 is associated with black rot resistance in <em>B. carinata</em>. This will potentially facilitate the identification of the causal gene for this QTL. Furthermore, we found that there were more TM‐LRRs found in clusters than NLRs. This could be due to the prevalence of TM‐LRRs among the <em>B. carinata</em> RGAs. However, increasing genomic data suggests that this feature is also common among TM‐LRRs in the <em>Brassica</em> family (Amas et al., <a href="#tpg270087-bib-0005" class="usa-link" aria-describedby="tpg270087-bib-0005">2023</a>; Dolatabadian et al., <a href="#tpg270087-bib-0009" class="usa-link" aria-describedby="tpg270087-bib-0009"><sup>2020</sup></a>; Yang et al., <a href="#tpg270087-bib-0048" class="usa-link" aria-describedby="tpg270087-bib-0048"><sup>2021</sup></a>). As <em>B. carinata</em> has been shown to be an excellent source of genetic resistance, a follow‐up investigation will be needed to understand how this RGA clustering pattern is contributing to this valuable trait in this species.</p></section><section id="tpg270087-sec-0220"><h3 class="pmc_sec_title">4.3. Evolution of RGAs in <em>B. carinata</em>
</h3>
<p>Consistent with the results of the gene conservation analysis, the proportion of the two RGA classes in <em>B. carinata</em> was also maintained from its progenitor diploid species, <em>B. nigra</em> and <em>B. oleracea</em>, at an 8:2 TM‐LRR to NLR ratio. This is also close to the figure reported for the Brassicaceae family wherein 75% of the RGAs were TM‐LRRs and only 25% were NLRs (Tirnaz et al., <a href="#tpg270087-bib-0043" class="usa-link" aria-describedby="tpg270087-bib-0043">2020</a>). Thus, in the Brassicaceae, this proportion of TM‐LRRs to NLRs seems to be preserved among its member species.</p>
<p>The different TM‐LRR RGA subclasses were generally found intact within the same cluster, indicating conservation of gene sequence within the subclasses. Among the TM‐LRRs, the RLKs contained the greatest number of distinct clades, suggesting this subclass has experienced greater gene expansion compared to RLPs and TM‐CCs. RLKs have been shown to exhibit functional plasticity, presumably due to their roles that extend beyond disease resistance, which may also contribute to their diversification (Man et al., <a href="#tpg270087-bib-0025" class="usa-link" aria-describedby="tpg270087-bib-0025">2020</a>). Aside from leucine‐rich repeats, other extracellular domains also exist in this gene family including lectin, lysine motif, or wall‐associated kinases (Yan et al., <a href="#tpg270087-bib-0047" class="usa-link" aria-describedby="tpg270087-bib-0047">2023</a>). Advances in genomics are helping uncover the different RLK domains that play important functions in disease resistance.</p>
<p>The <em>B. carinata</em> RLPs were found in one clade that was shared with RLKs, suggesting a common origin for these RGA classes. Compared to RLKs, RLPs lack a kinase domain and are thought to be more ancient than RLKs. It has been hypothesized that RLKs possibly evolved from RLPs with the integration of the kinase domain as an adaptive strategy shaped by environmental pressure and selection (Ngou et al., <a href="#tpg270087-bib-0028" class="usa-link" aria-describedby="tpg270087-bib-0028">2023</a>). In <em>Brassica</em> species, the number of RLPs is typically lower compared to RLKs (Tirnaz et al., <a href="#tpg270087-bib-0043" class="usa-link" aria-describedby="tpg270087-bib-0043">2020</a>), which is consistent with the results of a comprehensive survey across 350 plants species that found RLPs only constitute 19% of the total cell‐surface receptors (Ngou et al., <a href="#tpg270087-bib-0028" class="usa-link" aria-describedby="tpg270087-bib-0028">2023</a>). This suggests that the evolution of RLPs is more constrained compared to RLKs, yet they remain important genes for breeding disease‐resistant crops (Larkan et al., <a href="#tpg270087-bib-0020" class="usa-link" aria-describedby="tpg270087-bib-0020">2015</a>).</p>
<p>In this study, we classified the TM‐CC genes under the TM‐LRR class due to the presence of a transmembrane domain, a feature shared among RLPs and RLKs. The phylogenetic analysis revealed that this group also formed a distinct clade from the majority of the RLKs and RLPs, although a few of RLKs and RLPs were found in the TM‐CC clade. TM‐CCs are considered atypical resistance genes owing to their structure and understanding of their disease resistance mechanisms is not well established. Cantila et al. (<a href="#tpg270087-bib-0008" class="usa-link" aria-describedby="tpg270087-bib-0008">2023</a>) identified several TM‐CC genes that were co‐localized with disease resistance regions in the Brassicaceae, of which 70 were genetically related to cloned resistance genes in this family. This underscores the possible involvement of this RGA class in disease resistance. This study is the first to include the TM‐CC class in a genome‐wide assessment of RGAs in any <em>Brassica</em> species, providing a key resource for understanding the relationship of this gene type with other known RGAs in the <em>Brassica</em> species.</p>
<p>Compared to TM‐LRRs, the NLRs formed more clades, suggesting higher diversity within this RGA class. This is consistent with previous studies that demonstrated greater tendency of NLRs to diversify, likely due to the different subdomains they can harbor (Barragan &amp; Weigel, <a href="#tpg270087-bib-0006" class="usa-link" aria-describedby="tpg270087-bib-0006">2021</a>; Dolatabadian et al., <a href="#tpg270087-bib-0009" class="usa-link" aria-describedby="tpg270087-bib-0009"><sup>2020</sup></a>; Tirnaz et al., <a href="#tpg270087-bib-0043" class="usa-link" aria-describedby="tpg270087-bib-0043"><sup>2020</sup></a>). Furthermore, the TNL subclass formed a significantly greater number of clades compared with CNLs, suggesting that TNLs are more diverse than CNLs in <em>B. carinata</em>. This result is similar to that in the basal species <em>Arabidopsis thaliana</em>, suggesting that TNLs were more retained compared to CNLs in this lineage despite the CNLs being more ancient compared to TNLs (Jacob et al., <a href="#tpg270087-bib-0015" class="usa-link" aria-describedby="tpg270087-bib-0015">2013</a>). This further supports previous findings indicating that TNLs have recently undergone lineage‐specific expansions in some plant families, including the <em>Brassica</em> species (Tamborski &amp; Krasileva, <a href="#tpg270087-bib-0041" class="usa-link" aria-describedby="tpg270087-bib-0041">2020</a>).</p>
<p>The non‐regular NLRs that lacked the TIR and CC domains were dispersed in the different TNL and CNL clusters. This was also observed in <em>B. napus</em>, suggesting that the origin of these non‐regular NLR genes is diverse rather than monophyletic (Alamery et al., <a href="#tpg270087-bib-0004" class="usa-link" aria-describedby="tpg270087-bib-0004">2018</a>). Non‐regular NLR genes are likely produced from partial domain deletion of duplicated NLR genes. However, while they may be lacking the typical NLR domains, several studies have shown that these truncated NLRs still contribute to pathogen recognition and downstream signaling for mounting an effective disease resistance response in plants (Adachi et al., <a href="#tpg270087-bib-0001" class="usa-link" aria-describedby="tpg270087-bib-0001">2023</a>; Wang et al., <a href="#tpg270087-bib-0045" class="usa-link" aria-describedby="tpg270087-bib-0045"><sup>2021</sup></a>).</p></section><section id="tpg270087-sec-0230"><h3 class="pmc_sec_title">4.4. Dynamic evolutionary trajectories of RGAs in <em>B. carinata</em>
</h3>
<p>The phylogenetic analysis of the different RGA classes among <em>B. carinata</em>, <em>B. nigra</em>, and <em>B. oleracea</em> revealed several trajectories that helped explain the evolution of RGAs among these species. The most common pattern suggests expansion of RGAs, which can be seen in clusters where <em>B. carinata</em> RGAs were overrepresented compared to their diploid progenitor species. Resistance genes often show greater rates of expansion, presumably due to continuous exposure to pathogens and other associated stresses (Hofberger et al., <a href="#tpg270087-bib-0013" class="usa-link" aria-describedby="tpg270087-bib-0013">2014</a>; Macqueen &amp; Bergelson, <a href="#tpg270087-bib-0024" class="usa-link" aria-describedby="tpg270087-bib-0024"><sup>2016</sup></a>). This gene expansion is further promoted by mechanisms such as gene duplication and acquisition of other domains, possibly through recombination, chromosomal rearrangements and conversion, and insertion of TEs as discussed in previous sections. Along with expansion, some genes also exhibited contraction, shown in clusters where there was a loss of <em>B. carinata</em> RGAs relative to <em>B. nigra</em> and <em>B. oleracea</em>. However, the number of genes showing this pattern is relatively lower compared to genes undergoing expansion. Thus, in <em>B. carinata</em>, RGAs are undergoing more gene expansion than contraction, which fits with the overall ratio of <em>B. carinata</em> RGAs relative to its progenitor species and the findings that <em>B. carinata</em> experienced minimal gene loss among the <em>Brassica</em> species (Yim et al., <a href="#tpg270087-bib-0049" class="usa-link" aria-describedby="tpg270087-bib-0049">2022</a>).</p>
<p>Another common pattern described as straightforward inheritance of RGAs is shown in clusters where both progenitors contributed either an equal or unequal (skewed in favor of either of the progenitor species) number of RGAs. This evolutionary trajectory is expected as <em>B. carinata</em> is a relatively newly formed polyploid and has not been subjected to intensive selection (Khedikar et al., <a href="#tpg270087-bib-0018" class="usa-link" aria-describedby="tpg270087-bib-0018">2020</a>; Yim et al., <a href="#tpg270087-bib-0049" class="usa-link" aria-describedby="tpg270087-bib-0049"><sup>2022</sup></a>). Furthermore, this observation is reflective of our homology analysis, which suggests that the RGAs from the diploid parents are highly conserved in <em>B. carinata</em>. The pattern showing some genes with extremely long branches suggests a rapid sequence divergence and represents birth and death of RGAs (Fu et al., <a href="#tpg270087-bib-0010" class="usa-link" aria-describedby="tpg270087-bib-0010">2019</a>). The birth and death process, along with divergent selection, are widely known mechanisms of resistance gene evolution, which facilitates generation of different resistance specificities to recognize virulent pathogens (Michelmore &amp; Meyers, <a href="#tpg270087-bib-0026" class="usa-link" aria-describedby="tpg270087-bib-0026">1998</a>). Overall, this study provided insights into the dynamic evolution that shaped the current RGA repertoire of <em>B. carinata</em>.</p></section></section><section id="tpg270087-sec-0240"><h2 class="pmc_sec_title">5. CONCLUSIONS</h2>
<p>In this study, we identified the complete set of RGAs within the <em>B. carinata</em> genome to describe their distribution, organization, and evolution. We found that the <em>B. carinata</em> genome contained the greatest number of RGAs among the <em>Brassica</em> species in the triangle of U. Comparative analysis with its diploid progenitor species, <em>B. nigra</em> and <em>B. oleracea</em>, showed that the RGAs were generally conserved among these species. The majority of RGAs were duplicated, similar to most <em>Brassica</em> species. However, there was a contrasting pattern of gene duplication between the two subgenomes in that the B genome contained more intergenomic RGA duplicates compared with the C genome, while the opposite was observed in terms of intragenomic duplications. Moreover, the RGAs formed clusters with more TM‐LRRs than NLRs found in these clusters. Lastly, the phylogenetic analysis revealed that there was significant gene expansion in <em>B. carinata</em>. This study provides an important resource for genomics‐based exploration of disease resistance in <em>B. carinata</em>, which can be useful for breeding improved cultivars expressing robust disease resistance.</p></section><section id="tpg270087-sec-0250"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Junrey C. Amas</strong>: Conceptualization; data curation; formal analysis; investigation; methodology; writing—original draft. <strong>Philipp E. Bayer</strong>: Data curation; formal analysis; supervision.<strong>Wei Hong Tan</strong>: Formal analysis. <strong>Aldrin Cantila</strong>: Data curation. <strong>Aria Dolatabadian</strong>: Formal analysis; writing—review and editing. <strong>William J. W. Thomas</strong>: Writing—review and editing. <strong>David Edwards</strong>: Supervision. <strong>Jacqueline Batley</strong>: Conceptualization; supervision; writing—review and editing.</p></section><section id="tpg270087-sec-0270"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec27"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="tpg270087-supinfo-0001"></section><section class="sm xbox font-sm" id="tpg270087-supitem-0001"><div class="caption p">
<p>Table S1 Genome references used for RGA mining in <em>B. carinata</em> and its diploid progenitors, <em>B. nigra</em> and <em>B. oleracea</em>
</p>
<p>Table S2 Number of RGAs exhibiting intergenomic and intragenomic duplications in the <em>B. carinata</em> genome.</p>
<p>Table S3 Number of RGAs exhibiting segmental and tandem duplications in the <em>B. carinata</em> genome.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12326289/bin/TPG2-18-e70087-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70087-s002.xlsx</a><sup> (13.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="tpg270087-supitem-0002"><div class="caption p">
<p>Figure S1. The number of NLR and TM‐LRR RGAs in <em>B. carinata</em> in comparison to <em>B. nigra</em> and <em>B. oleracea</em>
</p>
<p>Figure S2. Phylogenetic analysis of <em>B. carinata</em>, <em>B. nigra</em>, and <em>B. oleracea</em> NLR genes. A. maximum likelihood phylogenetic tree showing collapsed clusters, which were color‐coded according to the evolutionary pattern they exhibit consistent with figure B: pattern 1 (blue), 2 (blue‐green), 3a (brown), 3b (purple), 4 (red), 5 (green). B. Representative clusters showing the specific evolutionary pattern: 1. Expansion of RGAs; 2. Straightforward inheritance of RGAs; 3a. RGAs skewed to <em>B. nigra</em>; 3b. RGAs skewed to <em>B. oleracea</em>; 4. Loss of <em>B. carinata</em> RGAs; 5. Rapid sequence divergence of RGAs. The species were also colour‐coded: <em>B. nigra</em> (red), <em>B. oleracea</em> (blue) and <em>B. carinata</em> (black).</p>
<p>Figure S3. Phylogenetic analysis of <em>B. carinata</em>, <em>B. nigra</em>, and <em>B. oleracea</em> TM‐CC genes. A. Maximum likelihood phylogenetic tree showing collapsed clusters, which were color‐coded according to the evolutionary pattern they exhibit consistent with figure B: pattern 1 (blue), 2 (blue‐green), 3a (brown), 3b (purple), 4 (red), 5 (green). B. Representative clusters showing the specific evolutionary pattern: 1. Expansion of RGAs; 2. Straightforward inheritance of RGAs; 3a. RGAs skewed to <em>B. nigra</em>; 3b. RGAs skewed to <em>B. oleracea</em>; 4. Loss of <em>B. carinata</em> RGAs; 5. Rapid sequence divergence of RGAs. The species were also color‐coded: <em>B. nigra</em> (red), <em>B. oleracea</em> (blue) and <em>B. carinata</em> (black).</p>
<p>Figure S4. Phylogenetic analysis of <em>B. carinata</em>, <em>B. nigra</em> and <em>B. oleracea</em> RLK genes. A. Maximum likelihood phylogenetic tree showing collapsed clusters, which were colour‐coded according to the evolutionary pattern they exhibit consistent with figure B: pattern 1 (blue), 2 (blue‐green), 3a (brown), 3b (purple), 4 (red), 5 (green). B. Representative clusters showing the specific evolutionary pattern: 1. Expansion of RGAs; 2. Straightforward inheritance of RGAs; 3a. RGAs skewed to <em>B. nigra</em>; 3b. RGAs skewed to <em>B. oleracea</em>; 4. Loss of <em>B. carinata</em> RGAs; 5. Rapid sequence divergence of RGAs. The species were also color‐coded: <em>B. nigra</em> (red), <em>B. oleracea</em> (blue) and <em>B. carinata</em> (black).</p>
<p>Figure S5. Phylogenetic analysis of <em>B. carinata</em>, <em>B. nigra</em> and <em>B. oleracea</em> RLP genes. A. Maximum likelihood phylogenetic tree showing collapsed clusters, which were colour‐coded according to the evolutionary pattern they exhibit consistent with figure B: pattern 1 (blue), 2 (blue‐green), 3a (brown), 3b (purple), 4 (red), 5 (green). B. Representative clusters showing the specific evolutionary pattern: 1. Expansion of RGAs; 2. Straightforward inheritance of RGAs; 3a. RGAs skewed to <em>B. nigra</em>; 3b. RGAs skewed to <em>B. oleracea</em>; 4. Loss of <em>B. carinata</em> RGAs; 5. Rapid sequence divergence of RGAs. The species were also colour‐coded: <em>B. nigra</em> (red), <em>B. oleracea</em> (blue) and <em>B. carinata</em> (black).</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12326289/bin/TPG2-18-e70087-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70087-s001.docx</a><sup> (4.1MB, docx) </sup>
</div></div></section></section><section id="tpg270087-sec-0260" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>Junrey C. Amas is supported by University of Western Australia (UWA) through the UWA International Fee Scholarship and University Postgraduate Award. This work was funded by the Australian Research Council project: DP200100762, DP210100296, and FL230100030 (Laureate Fellowship) and the Grains Research and Development Corporation (UWA1905‐006RTX).</p>
<p>Open access publishing facilitated by The University of Western Australia, as part of the Wiley ‐ The University of Western Australia agreement via the Council of Australian University Librarians.</p></section><section id="notes1"><p>


Amas, J. C.
, 
Bayer, P. E.
, 
Tan, W. H.
, 
Cantila, A.
, 
Dolatabadian, A.
, 
Thomas, W. J. W.
, 
Edwards, D.
, &amp; 
Batley, J.
 (2025). Genome‐wide identification and evolutionary analysis of disease resistance genes in <em>Brassica carinata</em>
. The Plant Genome, 18, e70087. 10.1002/tpg2.70087


</p>
<section id="fn-group1" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="tpg270087-note-0001"><p>Assigned to Associate Editor Manuel Spannagl.</p></div></div></section></section><section id="tpg270087-sec-0290"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>All data generated and analysed are included in the main text and Supporting Information figures and tables.</p></section><section id="tpg270087-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="tpg270087-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="tpg270087-bib-0001">
<cite>

Adachi, H.
, 
Sakai, T.
, 
Harant, A.
, 
Pai, H.
, 
Honda, K.
, 
Toghani, A. A.
, 
Claeys, J.
, 
Duggan, C.
, 
Bozkurt, T. O.
, 
Wu, C. H.
, &amp; 
Kamoun, S.
 (2023). An atypical NLR protein modulates the NRC immune receptor network in <em>Nicotiana benthamiana</em>
. PLOS Genetics, 19(1), e1010500. 10.1371/journal.pgen.1010500

</cite> [<a href="https://doi.org/10.1371/journal.pgen.1010500" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9851556/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36656829/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Genetics&amp;title=An%20atypical%20NLR%20protein%20modulates%20the%20NRC%20immune%20receptor%20network%20in%20Nicotiana%20benthamiana&amp;author=H.%20Adachi&amp;author=T.%20Sakai&amp;author=A.%20Harant&amp;author=H.%20Pai&amp;author=K.%20Honda&amp;volume=19&amp;issue=1&amp;publication_year=2023&amp;pages=e1010500&amp;pmid=36656829&amp;doi=10.1371/journal.pgen.1010500&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0002">
<cite>

Agarwal, G.
, 
Garg, V.
, 
Kudapa, H.
, 
Doddamani, D.
, 
Pazhamala, L. T.
, 
Khan, A. W.
, 
Thudi, M.
, 
Lee, S. H.
, &amp; 
Varshney, R. K.
 (2016). Genome‐wide dissection of AP2/ERF and HSP90 gene families in five legumes and expression profiles in chickpea and pigeonpea. Plant Biotechnology Journal, 14(7), 1563–1577. 10.1111/pbi.12520

</cite> [<a href="https://doi.org/10.1111/pbi.12520" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5066796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26800652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=Genome%E2%80%90wide%20dissection%20of%20AP2/ERF%20and%20HSP90%20gene%20families%20in%20five%20legumes%20and%20expression%20profiles%20in%20chickpea%20and%20pigeonpea&amp;author=G.%20Agarwal&amp;author=V.%20Garg&amp;author=H.%20Kudapa&amp;author=D.%20Doddamani&amp;author=L.%20T.%20Pazhamala&amp;volume=14&amp;issue=7&amp;publication_year=2016&amp;pages=1563-1577&amp;pmid=26800652&amp;doi=10.1111/pbi.12520&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0003">
<cite>

Alam, A.
, &amp; 
Dwivedi, P.
 (2019). Modeling site suitability and production potential of carinata‐based sustainable jet fuel in the southeastern United States. Journal of Cleaner Production, 239, 117817. 10.1016/j.jclepro.2019.117817
</cite> [<a href="https://doi.org/10.1016/j.jclepro.2019.117817" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Cleaner%20Production&amp;title=Modeling%20site%20suitability%20and%20production%20potential%20of%20carinata%E2%80%90based%20sustainable%20jet%20fuel%20in%20the%20southeastern%20United%20States&amp;author=A.%20Alam&amp;author=P.%20Dwivedi&amp;volume=239&amp;publication_year=2019&amp;pages=117817&amp;doi=10.1016/j.jclepro.2019.117817&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0004">
<cite>

Alamery, S.
, 
Tirnaz, S.
, 
Bayer, P.
, 
Tollenaere, R.
, 
Chaloub, B.
, 
Edwards, D.
, &amp; 
Batley, J.
 (2018). Genome‐wide identification and comparative analysis of NBS‐LRR resistance genes in <em>Brassica napus</em>
. Crop &amp; Pasture Science, 69(1), 72–93. 10.1071/CP17214
</cite> [<a href="https://doi.org/10.1071/CP17214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crop%20&amp;%20Pasture%20Science&amp;title=Genome%E2%80%90wide%20identification%20and%20comparative%20analysis%20of%20NBS%E2%80%90LRR%20resistance%20genes%20in%20Brassica%20napus&amp;author=S.%20Alamery&amp;author=S.%20Tirnaz&amp;author=P.%20Bayer&amp;author=R.%20Tollenaere&amp;author=B.%20Chaloub&amp;volume=69&amp;issue=1&amp;publication_year=2018&amp;pages=72-93&amp;doi=10.1071/CP17214&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0005">
<cite>

Amas, J. C.
, 
Bayer, P. E.
, 
Hong Tan, W.
, 
Tirnaz, S.
, 
Thomas, W. J. W.
, 
Edwards, D.
, &amp; 
Batley, J.
 (2023). Comparative pangenome analyses provide insights into the evolution of <em>Brassica rapa</em> resistance gene analogues (RGAs). Plant Biotechnology Journal, 21(10), 2100–2112. 10.1111/PBI.14116

</cite> [<a href="https://doi.org/10.1111/PBI.14116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10502758/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37431308/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=Comparative%20pangenome%20analyses%20provide%20insights%20into%20the%20evolution%20of%20Brassica%20rapa%20resistance%20gene%20analogues%20(RGAs)&amp;author=J.%20C.%20Amas&amp;author=P.%20E.%20Bayer&amp;author=W.%20Hong%20Tan&amp;author=S.%20Tirnaz&amp;author=W.%20J.%20W.%20Thomas&amp;volume=21&amp;issue=10&amp;publication_year=2023&amp;pages=2100-2112&amp;pmid=37431308&amp;doi=10.1111/PBI.14116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0006">
<cite>

Barragan, A. C.
, &amp; 
Weigel, D.
 (2021). Plant NLR diversity: The known unknowns of pan‐NLRomes. The Plant Cell, 33(4), 814–831. 10.1093/PLCELL/KOAA002

</cite> [<a href="https://doi.org/10.1093/PLCELL/KOAA002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8226294/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33793812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Cell&amp;title=Plant%20NLR%20diversity:%20The%20known%20unknowns%20of%20pan%E2%80%90NLRomes&amp;author=A.%20C.%20Barragan&amp;author=D.%20Weigel&amp;volume=33&amp;issue=4&amp;publication_year=2021&amp;pages=814-831&amp;pmid=33793812&amp;doi=10.1093/PLCELL/KOAA002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0052">
<cite>

Bayer, P. E.
, 
Golicz, A. A.
, 
Tirnaz, S.
, 
Chan, C. K. K.
, 
Edwards, D.
, &amp; 
Batley, J.
 (2019). Variation in abundance of predicted resistance genes in the <em>Brassica oleracea</em> pangenome. Plant Biotechnology Journal, 17(4), 789–800. 10.1111/PBI.13015

</cite> [<a href="https://doi.org/10.1111/PBI.13015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6419861/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30230187/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=Variation%20in%20abundance%20of%20predicted%20resistance%20genes%20in%20the%20Brassica%20oleracea%20pangenome&amp;author=P.%20E.%20Bayer&amp;author=A.%20A.%20Golicz&amp;author=S.%20Tirnaz&amp;author=C.%20K.%20K.%20Chan&amp;author=D.%20Edwards&amp;volume=17&amp;issue=4&amp;publication_year=2019&amp;pages=789-800&amp;pmid=30230187&amp;doi=10.1111/PBI.13015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0007">
<cite>

Cai, X.
, 
Lin, R.
, 
Liang, J.
, 
King, G. J.
, 
Wu, J.
, &amp; 
Wang, X.
 (2022). Transposable element insertion: A hidden major source of domesticated phenotypic variation in <em>Brassica rapa</em>
. Plant Biotechnology Journal, 20(7), 1298–1310. 10.1111/PBI.13807

</cite> [<a href="https://doi.org/10.1111/PBI.13807" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9241368/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35278263/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=Transposable%20element%20insertion:%20A%20hidden%20major%20source%20of%20domesticated%20phenotypic%20variation%20in%20Brassica%20rapa&amp;author=X.%20Cai&amp;author=R.%20Lin&amp;author=J.%20Liang&amp;author=G.%20J.%20King&amp;author=J.%20Wu&amp;volume=20&amp;issue=7&amp;publication_year=2022&amp;pages=1298-1310&amp;pmid=35278263&amp;doi=10.1111/PBI.13807&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0008">
<cite>

Cantila, A. Y.
, 
Thomas, W. J. W.
, 
Bayer, P. E.
, 
Edwards, D.
, &amp; 
Batley, J.
 (2023). In silico prediction and analysis of transmembrane‐coiled‐coil resistance gene analogues in 27 <em>Brassicaceae</em> species. Plant Pathology, 73, 115–130. 10.1111/ppa.13794
</cite> [<a href="https://doi.org/10.1111/ppa.13794" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Pathology&amp;title=In%20silico%20prediction%20and%20analysis%20of%20transmembrane%E2%80%90coiled%E2%80%90coil%20resistance%20gene%20analogues%20in%2027%20Brassicaceae%20species&amp;author=A.%20Y.%20Cantila&amp;author=W.%20J.%20W.%20Thomas&amp;author=P.%20E.%20Bayer&amp;author=D.%20Edwards&amp;author=J.%20Batley&amp;volume=73&amp;publication_year=2023&amp;pages=115-130&amp;doi=10.1111/ppa.13794&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0009">
<cite>

Dolatabadian, A.
, 
Bayer, P. E.
, 
Tirnaz, S.
, 
Hurgobin, B.
, 
Edwards, D.
, &amp; 
Batley, J.
 (2020). Characterization of disease resistance genes in the <em>Brassica napus</em> pangenome reveals significant structural variation. Plant Biotechnology Journal, 18(4), 969–982. 10.1111/pbi.13262

</cite> [<a href="https://doi.org/10.1111/pbi.13262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7061875/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31553100/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=Characterization%20of%20disease%20resistance%20genes%20in%20the%20Brassica%20napus%20pangenome%20reveals%20significant%20structural%20variation&amp;author=A.%20Dolatabadian&amp;author=P.%20E.%20Bayer&amp;author=S.%20Tirnaz&amp;author=B.%20Hurgobin&amp;author=D.%20Edwards&amp;volume=18&amp;issue=4&amp;publication_year=2020&amp;pages=969-982&amp;pmid=31553100&amp;doi=10.1111/pbi.13262&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0010">
<cite>

Fu, Y.
, 
Zhang, Y.
, 
Mason, A. S.
, 
Lin, B.
, 
Zhang, D.
, 
Yu, H.
, &amp; 
Fu, D.
 (2019). NBS‐encoding genes in <em>Brassica napus</em> evolved rapidly after allopolyploidization and co‐localize with known disease resistance loci. Frontiers in Plant Science, 10, Article 26. 10.3389/FPLS.2019.00026

</cite> [<a href="https://doi.org/10.3389/FPLS.2019.00026" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6363714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30761170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=NBS%E2%80%90encoding%20genes%20in%20Brassica%20napus%20evolved%20rapidly%20after%20allopolyploidization%20and%20co%E2%80%90localize%20with%20known%20disease%20resistance%20loci&amp;author=Y.%20Fu&amp;author=Y.%20Zhang&amp;author=A.%20S.%20Mason&amp;author=B.%20Lin&amp;author=D.%20Zhang&amp;volume=10&amp;publication_year=2019&amp;pages=26&amp;pmid=30761170&amp;doi=10.3389/FPLS.2019.00026&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0011">
<cite>

Gunasinghe, N.
, 
You, M. P.
, 
Clode, P. L.
, &amp; 
Barbetti, M. J.
 (2016). Mechanisms of resistance in <em>Brassica carinata</em>, <em>B. napus</em> and <em>B. juncea</em> to <em>Pseudocercosporella capsellae</em>
. Plant Pathology, 65(6), 888–900. 10.1111/PPA.12484
</cite> [<a href="https://doi.org/10.1111/PPA.12484" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Pathology&amp;title=Mechanisms%20of%20resistance%20in%20Brassica%20carinata,%20B.%20napus%20and%20B.%20juncea%20to%20Pseudocercosporella%20capsellae&amp;author=N.%20Gunasinghe&amp;author=M.%20P.%20You&amp;author=P.%20L.%20Clode&amp;author=M.%20J.%20Barbetti&amp;volume=65&amp;issue=6&amp;publication_year=2016&amp;pages=888-900&amp;doi=10.1111/PPA.12484&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0012">
<cite>

Hao, Z.
, 
Lv, D.
, 
Ge, Y.
, 
Shi, J.
, 
Weijers, D.
, 
Yu, G.
, &amp; 
Chen, J.
 (2020). 
<em>RIdeogram</em>: Drawing SVG graphics to visualize and map genome‐wide data on the idiograms. PeerJ Computer Science, 6, e251. 10.7717/peerj-cs.251
</cite> [<a href="https://doi.org/10.7717/peerj-cs.251" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7924719/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33816903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ%20Computer%20Science&amp;title=RIdeogram:%20Drawing%20SVG%20graphics%20to%20visualize%20and%20map%20genome%E2%80%90wide%20data%20on%20the%20idiograms&amp;author=Z.%20Hao&amp;author=D.%20Lv&amp;author=Y.%20Ge&amp;author=J.%20Shi&amp;author=D.%20Weijers&amp;volume=6&amp;publication_year=2020&amp;pages=e251&amp;pmid=33816903&amp;doi=10.7717/peerj-cs.251&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0013">
<cite>

Hofberger, J. A.
, 
Zhou, B.
, 
Tang, H.
, 
Jones, J. D. G.
, &amp; 
Schranz, M. E.
 (2014). A novel approach for multi‐domain and multi‐gene family identification provides insights into evolutionary dynamics of disease resistance genes in core eudicot plants. BMC Genomics, 15(1), Article 966. 10.1186/1471-2164-15-966

</cite> [<a href="https://doi.org/10.1186/1471-2164-15-966" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4289383/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25380807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=A%20novel%20approach%20for%20multi%E2%80%90domain%20and%20multi%E2%80%90gene%20family%20identification%20provides%20insights%20into%20evolutionary%20dynamics%20of%20disease%20resistance%20genes%20in%20core%20eudicot%20plants&amp;author=J.%20A.%20Hofberger&amp;author=B.%20Zhou&amp;author=H.%20Tang&amp;author=J.%20D.%20G.%20Jones&amp;author=M.%20E.%20Schranz&amp;volume=15&amp;issue=1&amp;publication_year=2014&amp;pages=966&amp;pmid=25380807&amp;doi=10.1186/1471-2164-15-966&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0014">
<cite>

Iboyi, J. E.
, 
Mulvaney, M. J.
, 
Leon, R. G.
, 
Balkcom, K. S.
, 
Bashyal, M.
, 
Devkota, P.
, &amp; 
Small, I. M.
 (2023). Double‐cropping effects of <em>Brassica carinata</em> and summer crops: I. Effects of summer cropping history on carinata production. Industrial Crops and Products, 194, 116364. 10.1016/j.indcrop.2023.116364
</cite> [<a href="https://doi.org/10.1016/j.indcrop.2023.116364" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Industrial%20Crops%20and%20Products&amp;title=Double%E2%80%90cropping%20effects%20of%20Brassica%20carinata%20and%20summer%20crops:%20I.%20Effects%20of%20summer%20cropping%20history%20on%20carinata%20production&amp;author=J.%20E.%20Iboyi&amp;author=M.%20J.%20Mulvaney&amp;author=R.%20G.%20Leon&amp;author=K.%20S.%20Balkcom&amp;author=M.%20Bashyal&amp;volume=194&amp;publication_year=2023&amp;pages=116364&amp;doi=10.1016/j.indcrop.2023.116364&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0015">
<cite>

Jacob, F.
, 
Vernaldi, S.
, &amp; 
Maekawa, T.
 (2013). Evolution and conservation of plant NLR functions. Frontiers in Immunology, 4, Article 297. 10.3389/fimmu.2013.00297

</cite> [<a href="https://doi.org/10.3389/fimmu.2013.00297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3782705/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24093022/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Immunology&amp;title=Evolution%20and%20conservation%20of%20plant%20NLR%20functions&amp;author=F.%20Jacob&amp;author=S.%20Vernaldi&amp;author=T.%20Maekawa&amp;volume=4&amp;publication_year=2013&amp;pages=297&amp;pmid=24093022&amp;doi=10.3389/fimmu.2013.00297&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0016">
<cite>

Karami, O.
, 
Dwivedi, P.
, 
Lamb, M.
, &amp; 
Field, J. L.
 (2022). Economics of crop rotations with and without carinata for sustainable aviation fuel production in the SE United States. Frontiers in Energy Research, 10, Article 830227. 10.3389/FENRG.2022.830227
</cite> [<a href="https://doi.org/10.3389/FENRG.2022.830227" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Energy%20Research&amp;title=Economics%20of%20crop%20rotations%20with%20and%20without%20carinata%20for%20sustainable%20aviation%20fuel%20production%20in%20the%20SE%20United%20States&amp;author=O.%20Karami&amp;author=P.%20Dwivedi&amp;author=M.%20Lamb&amp;author=J.%20L.%20Field&amp;volume=10&amp;publication_year=2022&amp;pages=830227&amp;doi=10.3389/FENRG.2022.830227&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0017">
<cite>

Katoh, K.
, 
Asimenos, G.
, &amp; 
Toh, H.
 (2009). Multiple alignment of DNA sequences with MAFFT. Methods in Molecular Biology, 537, 39–64. 10.1007/978-1-59745-251-9_3

</cite> [<a href="https://doi.org/10.1007/978-1-59745-251-9_3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19378139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20in%20Molecular%20Biology&amp;title=Multiple%20alignment%20of%20DNA%20sequences%20with%20MAFFT&amp;author=K.%20Katoh&amp;author=G.%20Asimenos&amp;author=H.%20Toh&amp;volume=537&amp;publication_year=2009&amp;pages=39-64&amp;pmid=19378139&amp;doi=10.1007/978-1-59745-251-9_3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0018">
<cite>

Khedikar, Y.
, 
Clarke, W. E.
, 
Chen, L.
, 
Higgins, E. E.
, 
Kagale, S.
, 
Koh, C. S.
, 
Bennett, R.
, &amp; 
Parkin, I. A. P.
 (2020). Narrow genetic base shapes population structure and linkage disequilibrium in an industrial oilseed crop, <em>Brassica carinata</em> A. Braun. Scientific Reports, 10(1), Article 12629. 10.1038/s41598-020-69255-w

</cite> [<a href="https://doi.org/10.1038/s41598-020-69255-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7387349/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32724070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Narrow%20genetic%20base%20shapes%20population%20structure%20and%20linkage%20disequilibrium%20in%20an%20industrial%20oilseed%20crop,%20Brassica%20carinata%20A.%20Braun&amp;author=Y.%20Khedikar&amp;author=W.%20E.%20Clarke&amp;author=L.%20Chen&amp;author=E.%20E.%20Higgins&amp;author=S.%20Kagale&amp;volume=10&amp;issue=1&amp;publication_year=2020&amp;pages=12629&amp;pmid=32724070&amp;doi=10.1038/s41598-020-69255-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0019">
<cite>

Kumar, S.
, 
Seepaul, R.
, 
Mulvaney, M. J.
, 
Colvin, B.
, 
George, S.
, 
Marois, J. J.
, 
Bennett, R.
, 
Leon, R.
, 
Wright, D. L.
, &amp; 
Small, I. M.
 (2020). 
<em>Brassica carinata</em> genotypes demonstrate potential as a winter biofuel crop in South East United States. Industrial Crops and Products, 150, 112353. 10.1016/j.indcrop.2020.112353
</cite> [<a href="https://doi.org/10.1016/j.indcrop.2020.112353" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Industrial%20Crops%20and%20Products&amp;title=Brassica%20carinata%20genotypes%20demonstrate%20potential%20as%20a%20winter%20biofuel%20crop%20in%20South%20East%20United%20States&amp;author=S.%20Kumar&amp;author=R.%20Seepaul&amp;author=M.%20J.%20Mulvaney&amp;author=B.%20Colvin&amp;author=S.%20George&amp;volume=150&amp;publication_year=2020&amp;pages=112353&amp;doi=10.1016/j.indcrop.2020.112353&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0020">
<cite>

Larkan, N. J.
, 
Ma, L.
, &amp; 
Borhan, M. H.
 (2015). The <em>Brassica napus</em> receptor‐like protein RLM2 is encoded by a second allele of the <em>LepR3/Rlm2</em> blackleg resistance locus. Plant Biotechnology Journal, 13(7), 983–992. 10.1111/pbi.12341

</cite> [<a href="https://doi.org/10.1111/pbi.12341" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25644479/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=The%20Brassica%20napus%20receptor%E2%80%90like%20protein%20RLM2%20is%20encoded%20by%20a%20second%20allele%20of%20the%20LepR3/Rlm2%20blackleg%20resistance%20locus&amp;author=N.%20J.%20Larkan&amp;author=L.%20Ma&amp;author=M.%20H.%20Borhan&amp;volume=13&amp;issue=7&amp;publication_year=2015&amp;pages=983-992&amp;pmid=25644479&amp;doi=10.1111/pbi.12341&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0021">
<cite>

Li, P.
, 
Quan, X.
, 
Jia, G.
, 
Xiao, J.
, 
Cloutier, S.
, &amp; 
You, F. M.
 (2016). RGAugury: A pipeline for genome‐wide prediction of resistance gene analogs (RGAs) in plants. BMC Genomics, 17(1), Article 852. 10.1186/S12864-016-3197-X

</cite> [<a href="https://doi.org/10.1186/S12864-016-3197-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5093994/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27806688/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=RGAugury:%20A%20pipeline%20for%20genome%E2%80%90wide%20prediction%20of%20resistance%20gene%20analogs%20(RGAs)%20in%20plants&amp;author=P.%20Li&amp;author=X.%20Quan&amp;author=G.%20Jia&amp;author=J.%20Xiao&amp;author=S.%20Cloutier&amp;volume=17&amp;issue=1&amp;publication_year=2016&amp;pages=852&amp;pmid=27806688&amp;doi=10.1186/S12864-016-3197-X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0050">
<cite>

Limbalkar, O. M.
, 
Singh, R.
, 
Kumar, P.
, 
Nanjundan, J.
, 
Parihar, C. M.
, 
Vasisth, P.
, 
Yadava, D. K.
, 
Chinnusamy, V.
, &amp; 
Singh, N.
 (2021). Deployment of <em>Brassica carinata</em> A. Braun derived <em>Brassica juncea</em> (L.) Czern. lines for improving heterosis and water use efficiency under water deficit stress conditions. Frontiers in Plant Science, 12: 765645. 10.3389/FPLS.2021.765645

</cite> [<a href="https://doi.org/10.3389/FPLS.2021.765645" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8655733/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34899786/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Plant%20Science&amp;title=Deployment%20of%20Brassica%20carinata%20A.%20Braun%20derived%20Brassica%20juncea%20(L.)%20Czern.%20lines%20for%20improving%20heterosis%20and%20water%20use%20efficiency%20under%20water%20deficit%20stress%20conditions&amp;author=O.%20M.%20Limbalkar&amp;author=R.%20Singh&amp;author=P.%20Kumar&amp;author=J.%20Nanjundan&amp;author=C.%20M.%20Parihar&amp;volume=12&amp;publication_year=2021&amp;pages=765645&amp;pmid=34899786&amp;doi=10.3389/FPLS.2021.765645&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0022">
<cite>

Liu, S.
, 
Liu, Y.
, 
Yang, X.
, 
Tong, C.
, 
Edwards, D.
, 
Parkin, I. A. P.
, 
Zhao, M.
, 
Ma, J.
, 
Yu, J.
, 
Huang, S.
, 
Wang, X.
, 
Wang, J.
, 
Lu, K.
, 
Fang, Z.
, 
Bancroft, I.
, 
Yang, T. J.
, 
Hu, Q.
, 
Wang, X.
, 
Yue, Z.
, … 
Paterson, A. H.
 (2014). The <em>Brassica oleracea</em> genome reveals the asymmetrical evolution of polyploid genomes. Nature Communications, 5, Article 3930. 10.1038/NCOMMS4930
</cite> [<a href="https://doi.org/10.1038/NCOMMS4930" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4279128/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24852848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=The%20Brassica%20oleracea%20genome%20reveals%20the%20asymmetrical%20evolution%20of%20polyploid%20genomes&amp;author=S.%20Liu&amp;author=Y.%20Liu&amp;author=X.%20Yang&amp;author=C.%20Tong&amp;author=D.%20Edwards&amp;volume=5&amp;publication_year=2014&amp;pages=3930&amp;pmid=24852848&amp;doi=10.1038/NCOMMS4930&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0023">
<cite>

Lu, K.
, 
Wei, L.
, 
Li, X.
, 
Wang, Y.
, 
Wu, J.
, 
Liu, M.
, 
Zhang, C.
, 
Chen, Z.
, 
Xiao, Z.
, 
Jian, H.
, 
Cheng, F.
, 
Zhang, K.
, 
Du, H.
, 
Cheng, X.
, 
Qu, C.
, 
Qian, W.
, 
Liu, L.
, 
Wang, R.
, 
Zou, Q.
, … 
Li, J.
 (2019). Whole‐genome resequencing reveals <em>Brassica napus</em> origin and genetic loci involved in its improvement. Nature Communications, 10(1), Article 1154. 10.1038/s41467-019-09134-9
</cite> [<a href="https://doi.org/10.1038/s41467-019-09134-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6411957/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30858362/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Whole%E2%80%90genome%20resequencing%20reveals%20Brassica%20napus%20origin%20and%20genetic%20loci%20involved%20in%20its%20improvement&amp;author=K.%20Lu&amp;author=L.%20Wei&amp;author=X.%20Li&amp;author=Y.%20Wang&amp;author=J.%20Wu&amp;volume=10&amp;issue=1&amp;publication_year=2019&amp;pages=1154&amp;pmid=30858362&amp;doi=10.1038/s41467-019-09134-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0024">
<cite>

Macqueen, A.
, &amp; 
Bergelson, J.
 (2016). Modulation of <em>R</em>‐gene expression across environments. Journal of Experimental Botany, 67(7), 2093–2105. 10.1093/jxb/erv530

</cite> [<a href="https://doi.org/10.1093/jxb/erv530" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4793800/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26983577/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Botany&amp;title=Modulation%20of%20R%E2%80%90gene%20expression%20across%20environments&amp;author=A.%20Macqueen&amp;author=J.%20Bergelson&amp;volume=67&amp;issue=7&amp;publication_year=2016&amp;pages=2093-2105&amp;pmid=26983577&amp;doi=10.1093/jxb/erv530&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0025">
<cite>

Man, J.
, 
Gallagher, J. P.
, &amp; 
Bartlett, M.
 (2020). Structural evolution drives diversification of the large LRR‐RLK gene family. New Phytologist, 226(5), 1492–1505. 10.1111/NPH.16455

</cite> [<a href="https://doi.org/10.1111/NPH.16455" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7318236/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31990988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Structural%20evolution%20drives%20diversification%20of%20the%20large%20LRR%E2%80%90RLK%20gene%20family&amp;author=J.%20Man&amp;author=J.%20P.%20Gallagher&amp;author=M.%20Bartlett&amp;volume=226&amp;issue=5&amp;publication_year=2020&amp;pages=1492-1505&amp;pmid=31990988&amp;doi=10.1111/NPH.16455&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0026">
<cite>

Michelmore, R. W.
, &amp; 
Meyers, B. C.
 (1998). Clusters of resistance genes in plants evolve by divergent selection and a birth‐and‐death process. Genome Research, 8(11), 1113–1130. 10.1101/gr.8.11.1113

</cite> [<a href="https://doi.org/10.1101/gr.8.11.1113" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9847076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Research&amp;title=Clusters%20of%20resistance%20genes%20in%20plants%20evolve%20by%20divergent%20selection%20and%20a%20birth%E2%80%90and%E2%80%90death%20process&amp;author=R.%20W.%20Michelmore&amp;author=B.%20C.%20Meyers&amp;volume=8&amp;issue=11&amp;publication_year=1998&amp;pages=1113-1130&amp;pmid=9847076&amp;doi=10.1101/gr.8.11.1113&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0028">
<cite>

Ngou, B. P. M.
, 
Wyler, M.
, 
Schmid, M. W.
, 
Kadota, Y.
, &amp; 
Shirasu, K.
 (2023). Evolutionary trajectory of pattern recognition receptors in plants. <em>BioRxiv</em>. 10.1101/2023.07.04.547604
</cite> [<a href="https://doi.org/10.1101/2023.07.04.547604" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10834447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38302456/" class="usa-link">PubMed</a>]</li>
<li id="tpg270087-bib-0029">
<cite>

Parisod, C.
, 
Alix, K.
, 
Just, J.
, 
Petit, M.
, 
Sarilar, V.
, 
Mhiri, C.
, 
Ainouche, M.
, 
Chalhoub, B.
, &amp; 
Grandbastien, M. A.
 (2010). Impact of transposable elements on the organization and function of allopolyploid genomes. New Phytologist, 186(1), 37–45. 10.1111/J.1469-8137.2009.03096.X

</cite> [<a href="https://doi.org/10.1111/J.1469-8137.2009.03096.X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20002321/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=New%20Phytologist&amp;title=Impact%20of%20transposable%20elements%20on%20the%20organization%20and%20function%20of%20allopolyploid%20genomes&amp;author=C.%20Parisod&amp;author=K.%20Alix&amp;author=J.%20Just&amp;author=M.%20Petit&amp;author=V.%20Sarilar&amp;volume=186&amp;issue=1&amp;publication_year=2010&amp;pages=37-45&amp;pmid=20002321&amp;doi=10.1111/J.1469-8137.2009.03096.X&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0030">
<cite>

Paritosh, K.
, 
Gupta, V.
, 
Yadava, S. K.
, 
Singh, P.
, 
Pradhan, A. K.
, &amp; 
Pental, D.
 (2014). RNA‐seq based SNPs for mapping in <em>Brassica juncea</em> (AABB): Synteny analysis between the two constituent genomes A (from <em>B. rapa</em>) and B (from <em>B. nigra</em>) shows highly divergent gene block arrangement and unique block fragmentation patterns. BMC Genomics, 15(1), Article 396. 10.1186/1471-2164-15-396

</cite> [<a href="https://doi.org/10.1186/1471-2164-15-396" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4045973/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24886001/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=RNA%E2%80%90seq%20based%20SNPs%20for%20mapping%20in%20Brassica%20juncea%20(AABB):%20Synteny%20analysis%20between%20the%20two%20constituent%20genomes%20A%20(from%20B.%20rapa)%20and%20B%20(from%20B.%20nigra)%20shows%20highly%20divergent%20gene%20block%20arrangement%20and%20unique%20block%20fragmentation%20patterns&amp;author=K.%20Paritosh&amp;author=V.%20Gupta&amp;author=S.%20K.%20Yadava&amp;author=P.%20Singh&amp;author=A.%20K.%20Pradhan&amp;volume=15&amp;issue=1&amp;publication_year=2014&amp;pages=396&amp;pmid=24886001&amp;doi=10.1186/1471-2164-15-396&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0031">
<cite>

Paritosh, K.
, 
Yadava, S. K.
, 
Singh, P.
, 
Bhayana, L.
, 
Mukhopadhyay, A.
, 
Gupta, V.
, 
Bisht, N. C.
, 
Zhang, J.
, 
Kudrna, D. A.
, 
Copetti, D.
, 
Wing, R. A.
, 
Reddy Lachagari, V. B.
, 
Pradhan, A. K.
, &amp; 
Pental, D.
 (2021). A chromosome‐scale assembly of allotetraploid <em>Brassica juncea</em> (AABB) elucidates comparative architecture of the A and B genomes. Plant Biotechnology Journal, 19(3), 602–614. 10.1111/PBI.13492

</cite> [<a href="https://doi.org/10.1111/PBI.13492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7955877/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33073461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Biotechnology%20Journal&amp;title=A%20chromosome%E2%80%90scale%20assembly%20of%20allotetraploid%20Brassica%20juncea%20(AABB)%20elucidates%20comparative%20architecture%20of%20the%20A%20and%20B%20genomes&amp;author=K.%20Paritosh&amp;author=S.%20K.%20Yadava&amp;author=P.%20Singh&amp;author=L.%20Bhayana&amp;author=A.%20Mukhopadhyay&amp;volume=19&amp;issue=3&amp;publication_year=2021&amp;pages=602-614&amp;pmid=33073461&amp;doi=10.1111/PBI.13492&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0032">
<cite>

Perumal, S.
, 
Koh, C. S.
, 
Jin, L.
, 
Buchwaldt, M.
, 
Higgins, E. E.
, 
Zheng, C.
, 
Sankoff, D.
, 
Robinson, S. J.
, 
Kagale, S.
, 
Navabi, Z. K.
, 
Tang, L.
, 
Horner, K. N.
, 
He, Z.
, 
Bancroft, I.
, 
Chalhoub, B.
, 
Sharpe, A. G.
, &amp; 
Parkin, I. A. P.
 (2020). A high‐contiguity <em>Brassica nigra</em> genome localizes active centromeres and defines the ancestral <em>Brassica</em> genome. Nature Plants, 6(8), 929–941. 10.1038/s41477-020-0735-y

</cite> [<a href="https://doi.org/10.1038/s41477-020-0735-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7419231/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32782408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Plants&amp;title=A%20high%E2%80%90contiguity%20Brassica%20nigra%20genome%20localizes%20active%20centromeres%20and%20defines%20the%20ancestral%20Brassica%20genome&amp;author=S.%20Perumal&amp;author=C.%20S.%20Koh&amp;author=L.%20Jin&amp;author=M.%20Buchwaldt&amp;author=E.%20E.%20Higgins&amp;volume=6&amp;issue=8&amp;publication_year=2020&amp;pages=929-941&amp;pmid=32782408&amp;doi=10.1038/s41477-020-0735-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0033">
<cite>
R Core Team
. (2021). 
R: A language and environment for statistical computing
. The R Foundation. <a href="https://www.r-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.r‐project.org/</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?R%20Core%20Team%20.%20(2021).%20R:%20A%20language%20and%20environment%20for%20statistical%20computing%20.%20The%20R%20Foundation.%20https://www.r%E2%80%90project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0034"><cite>

Rambaut, A.
 (2010). FigTree (v1.4.4) [Computer software]. FigTree. <a href="http://tree.bio.ed.ac.uk/software/figtree/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://tree.bio.ed.ac.uk/software/figtree/</a>
</cite></li>
<li id="tpg270087-bib-0035">
<cite>

Redda, Z. T.
, 
Laß‐Seyoum, A.
, 
Yimam, A.
, 
Barz, M.
, &amp; 
Jabasingh, S. A.
 (2022). Solvent extraction and characterization of <em>Brassica carinata</em> oils as promising alternative feedstock for bio‐jet fuel production. Biomass Conversion and Biorefinery, 14, 12207–12226. 10.1007/s13399-022-03343-x
</cite> [<a href="https://doi.org/10.1007/s13399-022-03343-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9648429/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36406949/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomass%20Conversion%20and%20Biorefinery&amp;title=Solvent%20extraction%20and%20characterization%20of%20Brassica%20carinata%20oils%20as%20promising%20alternative%20feedstock%20for%20bio%E2%80%90jet%20fuel%20production&amp;author=Z.%20T.%20Redda&amp;author=A.%20La%C3%9F%E2%80%90Seyoum&amp;author=A.%20Yimam&amp;author=M.%20Barz&amp;author=S.%20A.%20Jabasingh&amp;volume=14&amp;publication_year=2022&amp;pages=12207-12226&amp;pmid=36406949&amp;doi=10.1007/s13399-022-03343-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0036">
<cite>

Roslinsky, V.
, 
Falk, K. C.
, 
Gaebelein, R.
, 
Mason, A. S.
, &amp; 
Eynck, C.
 (2021). Development of <em>B. carinata</em> with super‐high erucic acid content through interspecific hybridization. Theoretical and Applied Genetics, 134(10), 3167–3181. 10.1007/S00122-021-03883-2

</cite> [<a href="https://doi.org/10.1007/S00122-021-03883-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8440251/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34269830/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Theoretical%20and%20Applied%20Genetics&amp;title=Development%20of%20B.%20carinata%20with%20super%E2%80%90high%20erucic%20acid%20content%20through%20interspecific%20hybridization&amp;author=V.%20Roslinsky&amp;author=K.%20C.%20Falk&amp;author=R.%20Gaebelein&amp;author=A.%20S.%20Mason&amp;author=C.%20Eynck&amp;volume=134&amp;issue=10&amp;publication_year=2021&amp;pages=3167-3181&amp;pmid=34269830&amp;doi=10.1007/S00122-021-03883-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0037">
<cite>

Seepaul, R.
, 
Kumar, S.
, 
Iboyi, J. E.
, 
Bashyal, M.
, 
Stansly, T. L.
, 
Bennett, R.
, 
Boote, K. J.
, 
Mulvaney, M. J.
, 
Small, I. M.
, 
George, S.
, &amp; 
Wright, D. L.
 (2021). 
<em>Brassica carinata</em>: Biology and agronomy as a biofuel crop. GCB Bioenergy, 13(4), 582–599. 10.1111/GCBB.12804
</cite> [<a href="https://doi.org/10.1111/GCBB.12804" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GCB%20Bioenergy&amp;title=Brassica%20carinata:%20Biology%20and%20agronomy%20as%20a%20biofuel%20crop&amp;author=R.%20Seepaul&amp;author=S.%20Kumar&amp;author=J.%20E.%20Iboyi&amp;author=M.%20Bashyal&amp;author=T.%20L.%20Stansly&amp;volume=13&amp;issue=4&amp;publication_year=2021&amp;pages=582-599&amp;doi=10.1111/GCBB.12804&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0038">
<cite>

Sharma, B. B.
, 
Kalia, P.
, 
Yadava, D. K.
, 
Singh, D.
, &amp; 
Sharma, T. R.
 (2016). Genetics and molecular mapping of black rot resistance locus Xca1bc on chromosome B‐7 in Ethiopian mustard (<em>Brassica carinata</em> A. Braun). PLoS ONE, 11(3), e0152290. 10.1371/JOURNAL.PONE.0152290

</cite> [<a href="https://doi.org/10.1371/JOURNAL.PONE.0152290" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4811439/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27023128/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20ONE&amp;title=Genetics%20and%20molecular%20mapping%20of%20black%20rot%20resistance%20locus%20Xca1bc%20on%20chromosome%20B%E2%80%907%20in%20Ethiopian%20mustard%20(Brassica%20carinata%20A.%20Braun)&amp;author=B.%20B.%20Sharma&amp;author=P.%20Kalia&amp;author=D.%20K.%20Yadava&amp;author=D.%20Singh&amp;author=T.%20R.%20Sharma&amp;volume=11&amp;issue=3&amp;publication_year=2016&amp;pages=e0152290&amp;pmid=27023128&amp;doi=10.1371/JOURNAL.PONE.0152290&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0039">
<cite>

Song, X.
, 
Wei, Y.
, 
Xiao, D.
, 
Gong, K.
, 
Sun, P.
, 
Ren, Y.
, 
Yuan, J.
, 
Wu, T.
, 
Yang, Q.
, 
Li, X.
, 
Nie, F.
, 
Li, N.
, 
Feng, S.
, 
Pei, Q.
, 
Yu, T.
, 
Zhang, C.
, 
Liu, T.
, 
Wang, X.
, &amp; 
Yang, J.
 (2021). 
<em>Brassica carinata</em> genome characterization clarifies U's triangle model of evolution and polyploidy in <em>Brassica</em>
. Plant Physiology, 186(1), 388–406. 10.1093/PLPHYS/KIAB048

</cite> [<a href="https://doi.org/10.1093/PLPHYS/KIAB048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8154070/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33599732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Brassica%20carinata%20genome%20characterization%20clarifies%20U's%20triangle%20model%20of%20evolution%20and%20polyploidy%20in%20Brassica&amp;author=X.%20Song&amp;author=Y.%20Wei&amp;author=D.%20Xiao&amp;author=K.%20Gong&amp;author=P.%20Sun&amp;volume=186&amp;issue=1&amp;publication_year=2021&amp;pages=388-406&amp;pmid=33599732&amp;doi=10.1093/PLPHYS/KIAB048&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0040">
<cite>
STAR version 2.0.1
. (2014). Biometrics and breeding informatics. PBGB Division, International Rice Research, Institute. <a href="http://bbi.irri.org/products" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://bbi.irri.org/products</a>
</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Biometrics%20and%20breeding%20informatics&amp;publication_year=2014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0041">
<cite>

Tamborski, J.
, &amp; 
Krasileva, K. V.
 (2020). Evolution of plant NLRs: From natural history to precise modifications. Annual Review of Plant Biology, 71, 355–378. 10.1146/annurev-arplant-081519-035901
</cite> [<a href="https://doi.org/10.1146/annurev-arplant-081519-035901" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32092278/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Plant%20Biology&amp;title=Evolution%20of%20plant%20NLRs:%20From%20natural%20history%20to%20precise%20modifications&amp;author=J.%20Tamborski&amp;author=K.%20V.%20Krasileva&amp;volume=71&amp;publication_year=2020&amp;pages=355-378&amp;pmid=32092278&amp;doi=10.1146/annurev-arplant-081519-035901&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0042">
<cite>

Thakur, A. K.
, 
Singh, K. H.
, 
Sharma, D.
, 
Parmar, N.
, &amp; 
Nanjundan, J.
 (2019). Breeding and genomics interventions in Ethiopian mustard (<em>Brassica carinata</em> A. Braun) improvement—A mini review. South African Journal of Botany, 125, 457–465. 10.1016/J.SAJB.2019.08.002
</cite> [<a href="https://doi.org/10.1016/J.SAJB.2019.08.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=South%20African%20Journal%20of%20Botany&amp;title=Breeding%20and%20genomics%20interventions%20in%20Ethiopian%20mustard%20(Brassica%20carinata%20A.%20Braun)%20improvement%E2%80%94A%20mini%20review&amp;author=A.%20K.%20Thakur&amp;author=K.%20H.%20Singh&amp;author=D.%20Sharma&amp;author=N.%20Parmar&amp;author=J.%20Nanjundan&amp;volume=125&amp;publication_year=2019&amp;pages=457-465&amp;doi=10.1016/J.SAJB.2019.08.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0043">
<cite>

Tirnaz, S.
, 
Bayer, P. E.
, 
Inturrisi, F.
, 
Zhang, F.
, 
Yang, H.
, 
Dolatabadian, A.
, 
Neik, T. X.
, 
Severn‐Ellis, A.
, 
Patel, D. A.
, 
Ibrahim, M. I.
, 
Pradhan, A.
, 
Edwards, D.
, &amp; 
Batley, J.
 (2020). Resistance gene analogs in the Brassicaceae: Identification, characterization, distribution, and evolution. Plant Physiology, 184(2), 909–922. 10.1104/PP.20.00835

</cite> [<a href="https://doi.org/10.1104/PP.20.00835" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7536671/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32796089/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Plant%20Physiology&amp;title=Resistance%20gene%20analogs%20in%20the%20Brassicaceae:%20Identification,%20characterization,%20distribution,%20and%20evolution&amp;author=S.%20Tirnaz&amp;author=P.%20E.%20Bayer&amp;author=F.%20Inturrisi&amp;author=F.%20Zhang&amp;author=H.%20Yang&amp;volume=184&amp;issue=2&amp;publication_year=2020&amp;pages=909-922&amp;pmid=32796089&amp;doi=10.1104/PP.20.00835&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0044">
<cite>

Tiwari, R.
, 
Reinhardt Piskáčková, T. A.
, 
Devkota, P.
, 
Mulvaney, M. J.
, 
Ferrell, J. A.
, &amp; 
Leon, R. G.
 (2021). Growing winter <em>Brassica carinata</em> as part of a diversified crop rotation for integrated weed management. GCB Bioenergy, 13(3), 425–435. 10.1111/GCBB.12799
</cite> [<a href="https://doi.org/10.1111/GCBB.12799" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GCB%20Bioenergy&amp;title=Growing%20winter%20Brassica%20carinata%20as%20part%20of%20a%20diversified%20crop%20rotation%20for%20integrated%20weed%20management&amp;author=R.%20Tiwari&amp;author=T.%20A.%20Reinhardt%20Pisk%C3%A1%C4%8Dkov%C3%A1&amp;author=P.%20Devkota&amp;author=M.%20J.%20Mulvaney&amp;author=J.%20A.%20Ferrell&amp;volume=13&amp;issue=3&amp;publication_year=2021&amp;pages=425-435&amp;doi=10.1111/GCBB.12799&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0051">
<cite>

Tonguç, M.
, &amp; 
Griffiths, P. D.
 (2004). Evaluation of <em>Brassica carinata</em> accessions for resistance to black rot (<em>Xanthomonas campestris</em> pv. <em>campestris</em>). HortScience, 39(5), 952–954. 10.21273/HORTSCI.39.5.952
</cite> [<a href="https://doi.org/10.21273/HORTSCI.39.5.952" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=HortScience&amp;title=Evaluation%20of%20Brassica%20carinata%20accessions%20for%20resistance%20to%20black%20rot%20(Xanthomonas%20campestris%20pv.%20campestris)&amp;author=M.%20Tongu%C3%A7&amp;author=P.%20D.%20Griffiths&amp;volume=39&amp;issue=5&amp;publication_year=2004&amp;pages=952-954&amp;doi=10.21273/HORTSCI.39.5.952&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0027">
<cite>

U, N.
 (1935). Genome‐analysis in <em>Brassica</em> with special reference to the experimental formation of <em>B. napus</em> and peculiar mode of fertilisation. Japanese Journal of Botany, 7(7), 389–452.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Japanese%20Journal%20of%20Botany&amp;title=Genome%E2%80%90analysis%20in%20Brassica%20with%20special%20reference%20to%20the%20experimental%20formation%20of%20B.%20napus%20and%20peculiar%20mode%20of%20fertilisation&amp;author=N.%20U&amp;volume=7&amp;issue=7&amp;publication_year=1935&amp;pages=389-452&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0045">
<cite>

Wang, W.
, 
Liu, N.
, 
Gao, C.
, 
Rui, L.
, 
Jiang, Q.
, 
Chen, S.
, 
Zhang, Q.
, 
Zhong, G.
, &amp; 
Tang, D.
 (2021). The truncated TNL receptor TN2‐mediated immune responses require ADR1 function. The Plant Journal, 108(3), 672–689. 10.1111/TPJ.15463

</cite> [<a href="https://doi.org/10.1111/TPJ.15463" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34396631/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Journal&amp;title=The%20truncated%20TNL%20receptor%20TN2%E2%80%90mediated%20immune%20responses%20require%20ADR1%20function&amp;author=W.%20Wang&amp;author=N.%20Liu&amp;author=C.%20Gao&amp;author=L.%20Rui&amp;author=Q.%20Jiang&amp;volume=108&amp;issue=3&amp;publication_year=2021&amp;pages=672-689&amp;pmid=34396631&amp;doi=10.1111/TPJ.15463&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0046">
<cite>

Wolfe, D.
, 
Dudek, S.
, 
Ritchie, M. D.
, &amp; 
Pendergrass, S. A.
 (2013). Visualizing genomic information across chromosomes with PhenoGram. BioData Mining, 6(1), Article 18. 10.1186/1756-0381-6-18

</cite> [<a href="https://doi.org/10.1186/1756-0381-6-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4015356/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24131735/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BioData%20Mining&amp;title=Visualizing%20genomic%20information%20across%20chromosomes%20with%20PhenoGram&amp;author=D.%20Wolfe&amp;author=S.%20Dudek&amp;author=M.%20D.%20Ritchie&amp;author=S.%20A.%20Pendergrass&amp;volume=6&amp;issue=1&amp;publication_year=2013&amp;pages=18&amp;pmid=24131735&amp;doi=10.1186/1756-0381-6-18&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0047">
<cite>

Yan, J.
, 
Su, P.
, 
Meng, X.
, &amp; 
Liu, P.
 (2023). Phylogeny of the plant receptor‐like kinase (RLK) gene family and expression analysis of wheat RLK genes in response to biotic and abiotic stresses. BMC Genomics, 24(1), Article 224. 10.1186/S12864-023-09303-7

</cite> [<a href="https://doi.org/10.1186/S12864-023-09303-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10152718/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37127571/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Phylogeny%20of%20the%20plant%20receptor%E2%80%90like%20kinase%20(RLK)%20gene%20family%20and%20expression%20analysis%20of%20wheat%20RLK%20genes%20in%20response%20to%20biotic%20and%20abiotic%20stresses&amp;author=J.%20Yan&amp;author=P.%20Su&amp;author=X.%20Meng&amp;author=P.%20Liu&amp;volume=24&amp;issue=1&amp;publication_year=2023&amp;pages=224&amp;pmid=37127571&amp;doi=10.1186/S12864-023-09303-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0048">
<cite>

Yang, H.
, 
Bayer, P. E.
, 
Tirnaz, S.
, 
Edwards, D.
, &amp; 
Batley, J.
 (2021). Genome‐wide identification and evolution of receptor‐like kinases (RLKs) and receptor like proteins (RLPs) in <em>Brassica juncea</em>
. Biology, 10(1), 17. 10.3390/biology10010017
</cite> [<a href="https://doi.org/10.3390/biology10010017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7823396/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33396674/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biology&amp;title=Genome%E2%80%90wide%20identification%20and%20evolution%20of%20receptor%E2%80%90like%20kinases%20(RLKs)%20and%20receptor%20like%20proteins%20(RLPs)%20in%20Brassica%20juncea&amp;author=H.%20Yang&amp;author=P.%20E.%20Bayer&amp;author=S.%20Tirnaz&amp;author=D.%20Edwards&amp;author=J.%20Batley&amp;volume=10&amp;issue=1&amp;publication_year=2021&amp;pages=17&amp;pmid=33396674&amp;doi=10.3390/biology10010017&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="tpg270087-bib-0049">
<cite>

Yim, W. C.
, 
Swain, M. L.
, 
Ma, D.
, 
An, H.
, 
Bird, K. A.
, 
Curdie, D. D.
, 
Wang, S.
, 
Ham, H. D.
, 
Luzuriaga‐Neira, A.
, 
Kirkwood, J. S.
, 
Hur, M.
, 
Solomon, J. K. Q.
, 
Harper, J. F.
, 
Kosma, D. K.
, 
Alvarez‐Ponce, D.
, 
Cushman, J. C.
, 
Edger, P. P.
, 
Mason, A. S.
, 
Pires, J. C.
, … 
Zhang, X.
 (2022). The final piece of the triangle of U: Evolution of the tetraploid <em>Brassica carinata</em> genome. The Plant Cell, 34(11), 4143–4172. 10.1093/plcell/koac249

</cite> [<a href="https://doi.org/10.1093/plcell/koac249" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9614464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35961044/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Plant%20Cell&amp;title=The%20final%20piece%20of%20the%20triangle%20of%20U:%20Evolution%20of%20the%20tetraploid%20Brassica%20carinata%20genome&amp;author=W.%20C.%20Yim&amp;author=M.%20L.%20Swain&amp;author=D.%20Ma&amp;author=H.%20An&amp;author=K.%20A.%20Bird&amp;volume=34&amp;issue=11&amp;publication_year=2022&amp;pages=4143-4172&amp;pmid=35961044&amp;doi=10.1093/plcell/koac249&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<p>Table S1 Genome references used for RGA mining in <em>B. carinata</em> and its diploid progenitors, <em>B. nigra</em> and <em>B. oleracea</em>
</p>
<p>Table S2 Number of RGAs exhibiting intergenomic and intragenomic duplications in the <em>B. carinata</em> genome.</p>
<p>Table S3 Number of RGAs exhibiting segmental and tandem duplications in the <em>B. carinata</em> genome.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12326289/bin/TPG2-18-e70087-s002.xlsx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70087-s002.xlsx</a><sup> (13.4KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<p>Figure S1. The number of NLR and TM‐LRR RGAs in <em>B. carinata</em> in comparison to <em>B. nigra</em> and <em>B. oleracea</em>
</p>
<p>Figure S2. Phylogenetic analysis of <em>B. carinata</em>, <em>B. nigra</em>, and <em>B. oleracea</em> NLR genes. A. maximum likelihood phylogenetic tree showing collapsed clusters, which were color‐coded according to the evolutionary pattern they exhibit consistent with figure B: pattern 1 (blue), 2 (blue‐green), 3a (brown), 3b (purple), 4 (red), 5 (green). B. Representative clusters showing the specific evolutionary pattern: 1. Expansion of RGAs; 2. Straightforward inheritance of RGAs; 3a. RGAs skewed to <em>B. nigra</em>; 3b. RGAs skewed to <em>B. oleracea</em>; 4. Loss of <em>B. carinata</em> RGAs; 5. Rapid sequence divergence of RGAs. The species were also colour‐coded: <em>B. nigra</em> (red), <em>B. oleracea</em> (blue) and <em>B. carinata</em> (black).</p>
<p>Figure S3. Phylogenetic analysis of <em>B. carinata</em>, <em>B. nigra</em>, and <em>B. oleracea</em> TM‐CC genes. A. Maximum likelihood phylogenetic tree showing collapsed clusters, which were color‐coded according to the evolutionary pattern they exhibit consistent with figure B: pattern 1 (blue), 2 (blue‐green), 3a (brown), 3b (purple), 4 (red), 5 (green). B. Representative clusters showing the specific evolutionary pattern: 1. Expansion of RGAs; 2. Straightforward inheritance of RGAs; 3a. RGAs skewed to <em>B. nigra</em>; 3b. RGAs skewed to <em>B. oleracea</em>; 4. Loss of <em>B. carinata</em> RGAs; 5. Rapid sequence divergence of RGAs. The species were also color‐coded: <em>B. nigra</em> (red), <em>B. oleracea</em> (blue) and <em>B. carinata</em> (black).</p>
<p>Figure S4. Phylogenetic analysis of <em>B. carinata</em>, <em>B. nigra</em> and <em>B. oleracea</em> RLK genes. A. Maximum likelihood phylogenetic tree showing collapsed clusters, which were colour‐coded according to the evolutionary pattern they exhibit consistent with figure B: pattern 1 (blue), 2 (blue‐green), 3a (brown), 3b (purple), 4 (red), 5 (green). B. Representative clusters showing the specific evolutionary pattern: 1. Expansion of RGAs; 2. Straightforward inheritance of RGAs; 3a. RGAs skewed to <em>B. nigra</em>; 3b. RGAs skewed to <em>B. oleracea</em>; 4. Loss of <em>B. carinata</em> RGAs; 5. Rapid sequence divergence of RGAs. The species were also color‐coded: <em>B. nigra</em> (red), <em>B. oleracea</em> (blue) and <em>B. carinata</em> (black).</p>
<p>Figure S5. Phylogenetic analysis of <em>B. carinata</em>, <em>B. nigra</em> and <em>B. oleracea</em> RLP genes. A. Maximum likelihood phylogenetic tree showing collapsed clusters, which were colour‐coded according to the evolutionary pattern they exhibit consistent with figure B: pattern 1 (blue), 2 (blue‐green), 3a (brown), 3b (purple), 4 (red), 5 (green). B. Representative clusters showing the specific evolutionary pattern: 1. Expansion of RGAs; 2. Straightforward inheritance of RGAs; 3a. RGAs skewed to <em>B. nigra</em>; 3b. RGAs skewed to <em>B. oleracea</em>; 4. Loss of <em>B. carinata</em> RGAs; 5. Rapid sequence divergence of RGAs. The species were also colour‐coded: <em>B. nigra</em> (red), <em>B. oleracea</em> (blue) and <em>B. carinata</em> (black).</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12326289/bin/TPG2-18-e70087-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">TPG2-18-e70087-s001.docx</a><sup> (4.1MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data generated and analysed are included in the main text and Supporting Information figures and tables.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from The Plant Genome are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/tpg2.70087"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/TPG2-18-e70087.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12326289/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12326289/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12326289%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12326289/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12326289/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12326289/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40765494/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12326289/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40765494/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12326289/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12326289/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="DTrfvtcSfsnrJLRAFWVHt0NbWnOVUdrZIn9sXiXcesQnFoxSnJudQmBuTiPiTH4a">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
