
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Phenylalkyl Acetophenones and Anacardic Acids from Knema oblongifolia with Synthetic Analogues as Anti-infectives and Antibacterial Agents - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE530F798AEEE723050F79002878CD2E.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="acsbmcau">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371487/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="ACS Bio &amp; Med Chem Au">
<meta name="citation_title" content="Phenylalkyl Acetophenones and Anacardic Acids from Knema oblongifolia with Synthetic Analogues as Anti-infectives and Antibacterial Agents">
<meta name="citation_author" content="Olivier Auguste Kirchhoffer">
<meta name="citation_author_institution" content="Institute of Pharmaceutical Sciences of Western Switzerland, University of Geneva, CMU, Geneva 1211, Switzerland">
<meta name="citation_author_institution" content="School of Pharmaceutical Sciences, University of Geneva, CMU, Geneva 1211, Switzerland">
<meta name="citation_author" content="Jahn Nitschke">
<meta name="citation_author_institution" content="Department of Biochemistry, Faculty of Sciences, University of Geneva, Quai Ernest-Ansermet 30, Geneva 1205, Switzerland">
<meta name="citation_author" content="Alexandre Luscher">
<meta name="citation_author_institution" content="Department of Microbiology and Molecular Medicine, University of Geneva, CMU, Geneva 1211, Switzerland">
<meta name="citation_author" content="Louis-Félix Nothias">
<meta name="citation_author_institution" content="Institute of Pharmaceutical Sciences of Western Switzerland, University of Geneva, CMU, Geneva 1211, Switzerland">
<meta name="citation_author_institution" content="School of Pharmaceutical Sciences, University of Geneva, CMU, Geneva 1211, Switzerland">
<meta name="citation_author_institution" content="Université Côte d’Azur, CNRS, ICN, Nice 06103, France">
<meta name="citation_author" content="Laurence Marcourt">
<meta name="citation_author_institution" content="Institute of Pharmaceutical Sciences of Western Switzerland, University of Geneva, CMU, Geneva 1211, Switzerland">
<meta name="citation_author_institution" content="School of Pharmaceutical Sciences, University of Geneva, CMU, Geneva 1211, Switzerland">
<meta name="citation_author" content="Nabil Hanna">
<meta name="citation_author_institution" content="Department of Biochemistry, Faculty of Sciences, University of Geneva, Quai Ernest-Ansermet 30, Geneva 1205, Switzerland">
<meta name="citation_author" content="Antonio Grondin">
<meta name="citation_author_institution" content="Green Mission Department, Herbal Products Laboratory, Pierre Fabre Research Institute, Toulouse 31100, France">
<meta name="citation_author" content="Thilo Köhler">
<meta name="citation_author_institution" content="Department of Microbiology and Molecular Medicine, University of Geneva, CMU, Geneva 1211, Switzerland">
<meta name="citation_author" content="Emerson Ferreira Queiroz">
<meta name="citation_author_institution" content="Institute of Pharmaceutical Sciences of Western Switzerland, University of Geneva, CMU, Geneva 1211, Switzerland">
<meta name="citation_author_institution" content="School of Pharmaceutical Sciences, University of Geneva, CMU, Geneva 1211, Switzerland">
<meta name="citation_author" content="Thierry Soldati">
<meta name="citation_author_institution" content="Department of Biochemistry, Faculty of Sciences, University of Geneva, Quai Ernest-Ansermet 30, Geneva 1205, Switzerland">
<meta name="citation_author" content="Jean-Luc Wolfender">
<meta name="citation_author_institution" content="Institute of Pharmaceutical Sciences of Western Switzerland, University of Geneva, CMU, Geneva 1211, Switzerland">
<meta name="citation_author_institution" content="School of Pharmaceutical Sciences, University of Geneva, CMU, Geneva 1211, Switzerland">
<meta name="citation_publication_date" content="2025 Jun 9">
<meta name="citation_volume" content="5">
<meta name="citation_issue" content="4">
<meta name="citation_firstpage" content="650">
<meta name="citation_doi" content="10.1021/acsbiomedchemau.5c00052">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371487/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371487/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371487/pdf/bg5c00052.pdf">
<meta name="description" content="The present study investigates the potential anti-infective and antibacterial properties of phenylalkyl acetophenones and anacardic acids isolated from the ethyl acetate extract of the leaves of Knema oblongifolia, along with synthetic derivatives ...">
<meta name="og:title" content="Phenylalkyl Acetophenones and Anacardic Acids from Knema oblongifolia with Synthetic Analogues as Anti-infectives and Antibacterial Agents">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The present study investigates the potential anti-infective and antibacterial properties of phenylalkyl acetophenones and anacardic acids isolated from the ethyl acetate extract of the leaves of Knema oblongifolia, along with synthetic derivatives ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371487/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12371487">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1021/acsbiomedchemau.5c00052"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/bg5c00052.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371487%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12371487/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12371487/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371487/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-acsbmcau.png" alt="ACS Bio &amp; Med Chem Au logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to ACS Bio &amp; Med Chem Au" title="Link to ACS Bio &amp; Med Chem Au" shape="default" href="https://pubs.acs.org/journal/abmcb8" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">ACS Bio Med Chem Au</button></div>. 2025 Jun 9;5(4):650–664. doi: <a href="https://doi.org/10.1021/acsbiomedchemau.5c00052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1021/acsbiomedchemau.5c00052</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22ACS%20Bio%20Med%20Chem%20Au%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Phenylalkyl
Acetophenones and Anacardic Acids from
Knema oblongifolia with Synthetic Analogues as Anti-infectives and
Antibacterial Agents</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kirchhoffer%20OA%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Olivier Auguste Kirchhoffer</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Olivier Auguste Kirchhoffer</span></h3>
<div class="p">
<sup>†</sup>
Institute
of Pharmaceutical Sciences of Western Switzerland, University of Geneva, CMU, Geneva
1211, Switzerland
</div>
<div class="p">
<sup>‡</sup>
School
of Pharmaceutical Sciences, University of
Geneva, CMU, Geneva
1211, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kirchhoffer%20OA%22%5BAuthor%5D" class="usa-link"><span class="name western">Olivier Auguste Kirchhoffer</span></a>
</div>
</div>
<sup>†,</sup><sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nitschke%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Jahn Nitschke</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Jahn Nitschke</span></h3>
<div class="p">
<sup>§</sup>
Department
of Biochemistry, Faculty of Sciences, University
of Geneva, Quai Ernest-Ansermet
30, Geneva
1205, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nitschke%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jahn Nitschke</span></a>
</div>
</div>
<sup>§</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luscher%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Alexandre Luscher</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Alexandre Luscher</span></h3>
<div class="p">
<sup>∥</sup>
Department
of Microbiology and Molecular Medicine, University of Geneva, CMU, Geneva
1211, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Luscher%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Alexandre Luscher</span></a>
</div>
</div>
<sup>∥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nothias%20LF%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Louis-Félix Nothias</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Louis-Félix Nothias</span></h3>
<div class="p">
<sup>†</sup>
Institute
of Pharmaceutical Sciences of Western Switzerland, University of Geneva, CMU, Geneva
1211, Switzerland
</div>
<div class="p">
<sup>‡</sup>
School
of Pharmaceutical Sciences, University of
Geneva, CMU, Geneva
1211, Switzerland
</div>
<div class="p">
<sup>⊥</sup>
Université
Côte d’Azur, CNRS, ICN, Nice
06103, France
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nothias%20LF%22%5BAuthor%5D" class="usa-link"><span class="name western">Louis-Félix Nothias</span></a>
</div>
</div>
<sup>†,</sup><sup>‡,</sup><sup>⊥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marcourt%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Laurence Marcourt</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Laurence Marcourt</span></h3>
<div class="p">
<sup>†</sup>
Institute
of Pharmaceutical Sciences of Western Switzerland, University of Geneva, CMU, Geneva
1211, Switzerland
</div>
<div class="p">
<sup>‡</sup>
School
of Pharmaceutical Sciences, University of
Geneva, CMU, Geneva
1211, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Marcourt%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Laurence Marcourt</span></a>
</div>
</div>
<sup>†,</sup><sup>‡</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hanna%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Nabil Hanna</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Nabil Hanna</span></h3>
<div class="p">
<sup>§</sup>
Department
of Biochemistry, Faculty of Sciences, University
of Geneva, Quai Ernest-Ansermet
30, Geneva
1205, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hanna%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nabil Hanna</span></a>
</div>
</div>
<sup>§</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grondin%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Antonio Grondin</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Antonio Grondin</span></h3>
<div class="p">
<sup>#</sup>
Green Mission
Department, Herbal Products Laboratory,
Pierre Fabre Research Institute, Toulouse
31100, France
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Grondin%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Antonio Grondin</span></a>
</div>
</div>
<sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22K%C3%B6hler%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Thilo Köhler</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Thilo Köhler</span></h3>
<div class="p">
<sup>∥</sup>
Department
of Microbiology and Molecular Medicine, University of Geneva, CMU, Geneva
1211, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22K%C3%B6hler%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Thilo Köhler</span></a>
</div>
</div>
<sup>∥</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferreira%20Queiroz%20E%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Emerson Ferreira Queiroz</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Emerson Ferreira Queiroz</span></h3>
<div class="p">
<sup>†</sup>
Institute
of Pharmaceutical Sciences of Western Switzerland, University of Geneva, CMU, Geneva
1211, Switzerland
</div>
<div class="p">
<sup>‡</sup>
School
of Pharmaceutical Sciences, University of
Geneva, CMU, Geneva
1211, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ferreira%20Queiroz%20E%22%5BAuthor%5D" class="usa-link"><span class="name western">Emerson Ferreira Queiroz</span></a>
</div>
</div>
<sup>†,</sup><sup>‡,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Soldati%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Thierry Soldati</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Thierry Soldati</span></h3>
<div class="p">
<sup>§</sup>
Department
of Biochemistry, Faculty of Sciences, University
of Geneva, Quai Ernest-Ansermet
30, Geneva
1205, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Soldati%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Thierry Soldati</span></a>
</div>
</div>
<sup>§</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wolfender%20JL%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Jean-Luc Wolfender</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Jean-Luc Wolfender</span></h3>
<div class="p">
<sup>†</sup>
Institute
of Pharmaceutical Sciences of Western Switzerland, University of Geneva, CMU, Geneva
1211, Switzerland
</div>
<div class="p">
<sup>‡</sup>
School
of Pharmaceutical Sciences, University of
Geneva, CMU, Geneva
1211, Switzerland
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wolfender%20JL%22%5BAuthor%5D" class="usa-link"><span class="name western">Jean-Luc Wolfender</span></a>
</div>
</div>
<sup>†,</sup><sup>‡,</sup><sup>*</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>†</sup>
Institute
of Pharmaceutical Sciences of Western Switzerland, University of Geneva, CMU, Geneva
1211, Switzerland
</div>
<div id="aff2">
<sup>‡</sup>
School
of Pharmaceutical Sciences, University of
Geneva, CMU, Geneva
1211, Switzerland
</div>
<div id="aff3">
<sup>§</sup>
Department
of Biochemistry, Faculty of Sciences, University
of Geneva, Quai Ernest-Ansermet
30, Geneva
1205, Switzerland
</div>
<div id="aff4">
<sup>∥</sup>
Department
of Microbiology and Molecular Medicine, University of Geneva, CMU, Geneva
1211, Switzerland
</div>
<div id="aff5">
<sup>⊥</sup>
Université
Côte d’Azur, CNRS, ICN, Nice
06103, France
</div>
<div id="aff6">
<sup>#</sup>
Green Mission
Department, Herbal Products Laboratory,
Pierre Fabre Research Institute, Toulouse
31100, France
</div>
<div class="author-notes p">
<div class="fn" id="cor1">
<sup>*</sup><p class="display-inline">
Email: <span>emerson.ferreira@unige.ch</span>.</p>
</div>
<div class="fn" id="cor2">
<sup>*</sup><p class="display-inline">
Email: <span>jean-luc.wolfender@unige.ch</span>.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 17; Accepted 2025 May 16; Revised 2025 May 16; Collection date 2025 Aug 20.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Authors. Published by American Chemical Society</div>
<p>This article is licensed under CC-BY 4.0</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12371487</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The present study investigates the potential anti-infective
and
antibacterial properties of phenylalkyl acetophenones and anacardic
acids isolated from the ethyl acetate extract of the leaves of Knema
oblongifolia, along with synthetic derivatives that were generated.
As antibiotic resistance grows, the discovery of new anti-infective
agents becomes crucial. The study utilizes a phenotypic screening
approach, employing a 3R infection model with (Mm) and (Dd) as proxies for and human macrophages. This model helps to distinguish between general
antibiotics and specific anti-infectives that inhibit bacterial growth
inside host cells. A previous screening carried out on a collection
of 1600 plant natural extracts revealed as a significant source of anti-infective compounds. The ethyl acetate
extract of this plant exhibited a strong inhibition of Mm intracellular
growth in the infection model while minimally affecting bacterial
growth in broth. HPLC bioactivity profiling of this extract based
on a high-resolution microfractionation strategy uncovered that the
activity was associated with different LC-peaks spread over the chromatogram.
LC–MS-based metabolite profiling of the extract revealed that
they shared common substructural elements. Based on such information,
fractionation of the extract at a larger scale led to the isolation
of 12 bioactive natural products (NPs): four newly described acetophenone
NPs and eight salicylic acid derivatives (three of which were new).
These NPs were further tested for their activities against Mm (antibacterial
and anti-infective), , and . Additionally,
the study involved de novo synthesis of derivatives based on the backbones
of the isolated acetophenones to enhance their bioactivity. Hemisynthesis
on one of the isolated natural acetophenone was also carried out and
resulted in an increase in potency but no increase in selectivity
toward the inhibition of Mm intracellular growth. Overall, biological
activity assessments revealed that some of the synthetic analogues
generated were better candidates in terms of both selectivity and
potency, with an improved activity profile compared to natural analogues.
The best synthetic candidate reached an IC<sub>50</sub> of 0.59 μM
for the inhibition of intracellular bacterial growth during infection
(anti-infective activity).</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> antimicrobial resistance, natural products, plant extract library, anti-infective, 3R-infection
model, HPLC bioactivity profiling, hemisynthesis</p></section></section><section class="abstract" id="abstract2"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/96b1/12371487/7bcd88612324/bg5c00052_0009.jpg" loading="lazy" height="316" width="665" alt="graphic file with name bg5c00052_0009.jpg"></p></section><section class="abstract" id="abstract3"><hr class="headless">
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/96b1/12371487/7bcd88612324/bg5c00052_0007.jpg" loading="lazy" id="tgr1" height="316" width="665" alt="graphic file with name bg5c00052_0007.jpg"></p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Antibiotics have had a paradigm-changing
impact on medicine since
their discovery in the middle of the 20th century, saving innumerable
human lives since their introduction into commercial use. Yet, the efficacy of those “magic bullets”
has been gradually eroded due to overconsumption and injudicious use
in clinical contexts. While their effects have largely been taken
for granted by the public, this hides a looming threat in the emergence
of multidrug-resistant (MDR) and extremely drug-resistant (XDR) bacterial
pathogens. It was estimated that by 2050, about 10 million people
each year would be dying from antibiotic-resistant infections, which
would make it the largest cause of death globally.<em>
,
</em> One
bacterial pathogen that is particularly worrying for the future of
antibiotic treatments is (Mtb), with an estimated death toll of 1.3 million people as of
2022.
</p>
<p>Historically natural products
(NPs) have been a rich source of
chemical diversity, with about 50% of all drugs approved in the past
four decades having been derived from nature. Several methods, mostly phenotypic-based, have been established
to assess antimicrobial activities of NPs, notably disk well diffusion,
agar dilution, or broth dilution, to name a few. The era of target-based drug discovery, which started in
the 1990s, has generally failed to deliver significant progress in
the discovery of antimicrobials. Phenotypic screening approaches have
therefore returned to fashion recently, especially as they increasingly
enable the discovery of new targets.<em>
−

</em> This renewed attention
toward phenotypic screening was enabled by the development of more
complex and specific phenotypic models, notably models which capture
the intracellular nature of Mtb’s pathogenicity.<em>
,
</em> In the current study, a 3R-infection model that mimics the behavior
of Mtb when it infects human macrophages was established, using (Mm, ATCC BAA-535) and (Dd, ATCC MYA-4120), respectively,
as stand-ins for Mtb and macrophages.<em>
−

</em> The goal of this assay
is to find “anti-infective” samples that would affect
only the growth of the bacteria in the infection model and would therefore
be less prone to resistance mechanisms. To distinguish between anti-infectives
and antibiotics, a second assay where Mm was cultured in broth was
carried out.
</p>
<p>This approach also
provided a double readout on the growth of both
the bacteria and the amoeba, making it possible to distinguish between
“strict anti-infectives” (only inhibiting bacterial
growth in infection) and “anti-infective Dd inhibitors”
(inhibiting the growth of both bacteria and amoeba).
</p>
<p>In addition to stringent biological assays, it is
also crucial
to have access to a large source of chemical diversity to tackle the
challenges of antibiotics resistance. For this, an internal collection
of 1600 registered Natural plant Extracts (NEs) previously described,
which contains species that encompass about 30% of all known botanical
families, was screened for anti-infective
activities with the Dd-Mm anti-infective assay.<em>
,
</em> Based on that screening, this work focused on the phytochemical
investigation of the ethyl acetate extract of <em>Knema oblongifolia</em> (King) Warb., Myristicaceae (leaves), which was highlighted as a
strong anti-infective. The aim was then to identify its active principles,
obtain a series of structurally related natural and synthetic derivatives,
characterize their activities, and determine structural features possibly
responsible for the observed activity.</p>
<p>The investigation included
high-resolution bioactivity profiling
to target bioactive compounds, large-scale isolation of anti-infective
entities, and synthetic derivatization of the backbone of some of
the identified NPs. In addition to the anti-infective potency assessment
of the isolated NPs, antibacterial assays on one strain of (Sa, Gram-positive) and one strain of (Pa, Gram-negative)<em>
,
</em> were also carried out to assess general antibacterial activities
and contextualize them with the antimycobacterial activities.</p></section><section id="sec2"><h2 class="pmc_sec_title">Results and Discussion</h2>
<section id="sec2.1"><h3 class="pmc_sec_title">Screening of 1600 Plant Extracts for Anti-infective Activities</h3>
<p>The biological screening<em>
,
</em> of a 1600 NE collection
previously described initiated this study.
For this, a model based on the infection of the amoeba host Dd by
Mm was used.<em>
,
</em> Two readouts were obtained from
this anti-infective assay: “anti-infectives” were the
samples that reduced the growth of Mm by at least 50% during infection,
while “Dd inhibitors” were those that reduced the growth
of Dd by at least 50%. A second assay to assess the antibiotic activity
of samples was also carried out with Mm grown in broth (without any
host). This approach allowed for the discrimination of antibiotics
from anti-infectives, with the latter acting predominantly during
infection. NEs were therefore considered hits if they were anti-infectives
but not antibiotic. Extracts classified as “anti-infective
Dd inhibitors” were also considered hits, despite the ideal
scenario being that of “strict anti-infectives”. Indeed,
components responsible for the anti-infective effect in an extract
might not necessarily be the same as those responsible for inhibiting
the growth of the amoeba.</p>
<p>Among the hit extracts that this screening
yielded, the ethyl acetate extract of leaves (King) Warb., Myristicaceae (Q5438126) was one of the anti-infective
Dd inhibitors with a striking anti-infective activity but also affecting
the growth of the amoeba. To our knowledge, no phytochemical investigations
were reported for this species at the time of writing (see WikiData
query), it was therefore chosen as the focus of the current study.</p></section><section id="sec2.2"><h3 class="pmc_sec_title">Biological Activities of the Extract of (Leaves)</h3>
<p>The ethyl acetate extract of leaves strikingly inhibited (by 155%)
the intracellular growth of Mm during infection compared to the DMSO
control (<a href="#fig1" class="usa-link">Figure 
</a>C,D)
while leaving the bacteria almost unaffected in broth (<a href="#fig1" class="usa-link">Figure 
</a>A,B). Note that a growth inhibition
of over 100% effectively corresponds to a reduction of the initial
Mm load present inside Dd at the start of the infection, which means
that the extract is bactericidal. The extract also decreased the cell
mass of the amoeba compared to the initial value, which indicates
cytotoxicity (<a href="#fig1" class="usa-link">Figure 
</a>E,F). Despite the latter “red flag”, the extract was
nevertheless considered for further study because it exerted selective
anti-infective activity without activity against the bacteria in broth.
The hope was also for the anti-infective and the cytotoxic activity
to be disentangled during fractionation.</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371487_bg5c00052_0001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/96b1/12371487/527c74a24fec/bg5c00052_0001.jpg" loading="lazy" id="gr1" height="346" width="693" alt="1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Biological activities
of the ethyl acetate extract of leaves (Ko extract) (A) <em>in
vitro</em> Mm growth curves; (B) normalized <em>in vitro</em> Mm growth, inhibited by 4% with the extract; (C) <em>in cellulo</em> Mm growth curves; (D) normalized <em>in cellulo</em> Mm
growth, inhibited by 155% with the extract; (E) <em>in cellulo</em> Dd growth curves; (F) normalized <em>in cellulo</em> Dd
growth, inhibited by 133% with the extract; vehicle control (VC):
DMSO 0.3%; positive control (PC): rifabutin (10 μM); and extract
tested at 25 μg/mL. 50% growth inhibition is the threshold (represented
by a dotted line in B, D, and F) set for a NE to be considered bioactive
in each respective category. The dotted lines in panels A, C, and
E represent the median of luminescence/fluorescence of all wells at
time 0. Each experiment was carried out in 3 biological replicates,
1 technical replicate.</p></figcaption></figure></section><section id="sec2.3"><h3 class="pmc_sec_title">Phytochemical Investigations on (Leaves)</h3>
<p>In order to disentangle the activities of the
various NPs in the extract, an HPLC bioactivity profiling method was used to monitor the activity on the Dd-Mm
model. For this, the extract was subjected to generic metabolite profiling
by UHPLC-PDA-CAD-HRMS/MS. The UV-PDA trace revealed that the extract
was rich in UV-active NPs. The Charged Aerosol Detection (CAD) trace,
which is known to provide a semiquantitative readout, revealed that
it fitted well with the UV trace at 329 nm. Indeed, for almost all
CAD-detected peaks, a proportional UV peak was detected, UV detection
was therefore well adapted to study this plant extract (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00052/suppl_file/bg5c00052_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Information</a> Figure 1).</p>
<p>Dereplication
based on tandem MS spectra (MS<sup>2</sup>) recorded by data-dependent
analysis using the computational MS framework software SIRIUS was largely inconclusive.</p>
<p>The HPLC bioactivity
profiling was done by a microfractionation
into a 96-well plate in a single chromatographic run on an aliquot
of 13 mg. Individual microfractions were dried, all diluted in a conserved
volume, and tested for anti-infective activities. Following the color
patterns established in <a href="#fig1" class="usa-link">Figure 
</a>, individual values for bacterial growth in infection for
each microfraction are displayed in red on the chromatogram, the growth
of the amoeba in green, and the bacterial growth in broth in purple
(<a href="#fig2" class="usa-link">Figure 
</a>).</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371487_bg5c00052_0002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/96b1/12371487/80f367e9c3cb/bg5c00052_0002.jpg" loading="lazy" id="gr2" height="435" width="644" alt="2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Semipreparative
HPLC-PDA microfractionation of the ethyl acetate
extract of leaves (30
min chromatographic). Vertical lines correspond to the 92 microfractions
that were tested for their anti-infective activities in infection
(in red), their effect on the host in infection (in green) as well
as antibiotic activities <em>in broth</em> (in purple) with
3 biological replicates, and 1 technical replicate for each experiment.
Ten zones (hit microfractions) with distinct <em>m</em>/<em>z</em> values detected and with inhibitory activities above 50%
can be observed. Two more microfractions with distinct <em>m</em>/<em>z</em> values and inhibitory activities close to 50%
(but not above) were also considered hits (M08 and M31). The 30 min
chromatogram was truncated between 0 and 21 min to focus on bioactive
regions.</p></figcaption></figure><p>A total of 16 microfractions (<strong>M</strong>) among
the 92 collected
from the EtOAc extract of displayed over 50% inhibition of intracellular bacterial growth
and were flagged as anti-infective. Subsequent UHPLC-PDA-CAD-HRMS/MS
analyses of these microfractions and adjacent ones revealed 12 distinct
target [M-H]<sup>−</sup> MS features (<em>m</em>/<em>z</em> @ RT), two of which came from adjacent microfractions
that were close to passing the 50% bioassay cutoff (<strong>M08</strong> and <strong>M31</strong> with 57% and 65% inhibition of intracellular
bacterial growth, respectively). These two microfractions were considered
due to their low toxicity toward the amoeba (in red, <a href="#fig3" class="usa-link">Figure 
</a>), since it appeared that many
of the active microfractions were also affecting the growth of the
amoeba.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371487_bg5c00052_0003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/96b1/12371487/4aa6b0082a4b/bg5c00052_0003.jpg" loading="lazy" id="gr3" height="546" width="765" alt="3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Compounds isolated from (leaves).</p></figcaption></figure><p>Dereplication based on tandem MS spectra (MS<sup>2</sup>) recorded
by data-dependent analysis of the 12 bioactive microfractions using
SIRIUS (in both PI and NI modes) was largely inconclusive. The annotations were generally of low confidence
according to their associated quality scores (by CSI/FingerID), and the proposed structures did not correspond
to those expected based on phytochemical reports on the genus (<em>Knema</em>) level. Nevertheless, molecular formulas (MF) could
be established with confidence at the MS1 level based on HRMS data
in both positive- (PI) and negative-ion (NI) mode (see <a href="#tbl1" class="usa-link">Table 
</a>). Three of them could be matched
with NPs reported in the literature of the genus <em>Knema</em>. <strong>M25</strong>-<strong>26</strong> corresponded to kneglobularic
acid B, previously isolated from <em>Knema globularia</em>, along with kneglobularic acid A which corresponded to <strong>M30</strong>-<strong>31</strong>. Khookerianic acid
A was matching the molecular formula found in <strong>M29</strong>-<strong>30</strong> and was reported in . All of them were derived from a common
salicylic acid scaffold and belong to the family of anacardic acids.</p>
<section class="tw xbox font-sm" id="tbl1"><h4 class="obj_head">1. Bioactive Microfractions with Their
Major Ion <em>m/z</em> and Characteristic Fragment Flagged
in MS/MS Analyses<sup></sup>
.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">microfraction number</th>
<th align="center" colspan="1" rowspan="1">
<em>m</em>/<em>z</em> NI</th>
<th align="center" colspan="1" rowspan="1">
<em>m</em>/<em>z</em> PI</th>
<th align="center" colspan="1" rowspan="1">molecular formula</th>
<th align="center" colspan="1" rowspan="1">MS<sup>2</sup> fragment</th>
<th align="center" colspan="1" rowspan="1">compound <em>n</em>°</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>M08</strong>
</td>
<td align="left" colspan="1" rowspan="1">327.1237 [M – H]<sup>−</sup>
</td>
<td align="left" colspan="1" rowspan="1">329.1382 [M + H]<sup>+</sup>
</td>
<td align="left" colspan="1" rowspan="1">C<sub>19</sub>H<sub>20</sub>O<sub>5</sub>
</td>
<td align="left" colspan="1" rowspan="1">149.0598 [C<sub>9</sub>H<sub>9</sub>O<sub>2</sub>
<sup>+</sup>]</td>
<td align="center" colspan="1" rowspan="1">
<strong>1</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>M09</strong>
</td>
<td align="left" colspan="1" rowspan="1">283.1336 [M – H]<sup>−</sup>
</td>
<td align="left" colspan="1" rowspan="1">285.1483 [M + H]<sup>+</sup>
</td>
<td align="left" colspan="1" rowspan="1">C<sub>18</sub>H<sub>20</sub>O<sub>3</sub>
</td>
<td align="left" colspan="1" rowspan="1">149.0598 [C<sub>9</sub>H<sub>9</sub>O<sub>2</sub>
<sup>+</sup>]</td>
<td align="center" colspan="1" rowspan="1">
<strong>2</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>M11</strong>
</td>
<td align="left" colspan="1" rowspan="1">313.1079 [M – H]<sup>−</sup>
</td>
<td align="left" colspan="1" rowspan="1">297.1482 [M-H<sub>2</sub>O + H]<sup>+</sup>
</td>
<td align="left" colspan="1" rowspan="1">C<sub>18</sub>H<sub>18</sub>O<sub>5</sub>
</td>
<td align="left" colspan="1" rowspan="1">107.0492 [C<sub>7</sub>H<sub>7</sub>O<sup>+</sup>]</td>
<td align="center" colspan="1" rowspan="1">
<strong>3</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>M15</strong>
</td>
<td align="left" colspan="1" rowspan="1">311.1651 [M – H]<sup>−</sup>
</td>
<td align="left" colspan="1" rowspan="1">313.1796 [M + H]<sup>+</sup>
</td>
<td align="left" colspan="1" rowspan="1">C<sub>20</sub>H<sub>24</sub>O<sub>3</sub>
</td>
<td align="left" colspan="1" rowspan="1">149.0598 [C<sub>9</sub>H<sub>9</sub>O<sub>2</sub>
<sup>+</sup>]</td>
<td align="center" colspan="1" rowspan="1">
<strong>4</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>M16–18</strong>
</td>
<td align="left" colspan="1" rowspan="1">341.1390 [M – H]<sup>−</sup>
</td>
<td align="left" colspan="1" rowspan="1">325.1430 [M-H<sub>2</sub>O + H]<sup>+</sup>
</td>
<td align="left" colspan="1" rowspan="1">C<sub>20</sub>H<sub>22</sub>O<sub>5</sub>
</td>
<td align="left" colspan="1" rowspan="1">107.0491
[C<sub>7</sub>H<sub>7</sub>O<sup>+</sup>]</td>
<td align="center" colspan="1" rowspan="1">
<strong>6</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>M20–21</strong>
</td>
<td align="left" colspan="1" rowspan="1">297.1495 [M – H]<sup>−</sup>
</td>
<td align="left" colspan="1" rowspan="1">281.1530 [M-H<sub>2</sub>O + H]<sup>+</sup>
</td>
<td align="left" colspan="1" rowspan="1">C<sub>19</sub>H<sub>22</sub>O<sub>3</sub>
</td>
<td align="left" colspan="1" rowspan="1">107.0491 [C<sub>7</sub>H<sub>7</sub>O<sup>+</sup>]</td>
<td align="center" colspan="1" rowspan="1">
<strong>7</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>M22</strong>
</td>
<td align="left" colspan="1" rowspan="1">277.1809 [M – H]<sup>−</sup>
</td>
<td align="left" colspan="1" rowspan="1">279.1949 [M + H]<sup>+</sup>
</td>
<td align="left" colspan="1" rowspan="1">C<sub>17</sub>H<sub>26</sub>O<sub>3</sub>
</td>
<td align="left" colspan="1" rowspan="1">149.0598 [C<sub>9</sub>H<sub>9</sub>O<sub>2</sub>
<sup>+</sup>]</td>
<td align="center" colspan="1" rowspan="1">
<strong>5</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>M25–26</strong>
</td>
<td align="left" colspan="1" rowspan="1">369.1703 [M – H]<sup>−</sup>
</td>
<td align="left" colspan="1" rowspan="1">353.1744 [M-H<sub>2</sub>O + H]<sup>+</sup>
</td>
<td align="left" colspan="1" rowspan="1">C<sub>22</sub>H<sub>26</sub>O<sub>5</sub>
</td>
<td align="left" colspan="1" rowspan="1">107.0490
[C<sub>7</sub>H<sub>7</sub>O<sup>+</sup>]</td>
<td align="center" colspan="1" rowspan="1">
<strong>9</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>M29–30</strong>
</td>
<td align="left" colspan="1" rowspan="1">263.1651 [M – H]<sup>−</sup>
</td>
<td align="left" colspan="1" rowspan="1">265.1794 [M + H]<sup>+</sup>
</td>
<td align="left" colspan="1" rowspan="1">C<sub>16</sub>H<sub>24</sub>O<sub>3</sub>
</td>
<td align="left" colspan="1" rowspan="1">107.0491 [C<sub>7</sub>H<sub>7</sub>O<sup>+</sup>]</td>
<td align="center" colspan="1" rowspan="1">
<strong>8</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>M30–31</strong>
</td>
<td align="left" colspan="1" rowspan="1">325.1806 [M – H]<sup>−</sup>
</td>
<td align="left" colspan="1" rowspan="1">309.1845 [M-H<sub>2</sub>O + H]<sup>+</sup>
</td>
<td align="left" colspan="1" rowspan="1">C<sub>21</sub>H<sub>26</sub>O<sub>3</sub>
</td>
<td align="left" colspan="1" rowspan="1">107.0491 [C<sub>7</sub>H<sub>7</sub>O<sup>+</sup>]</td>
<td align="center" colspan="1" rowspan="1">
<strong>10</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>M43–44</strong>
</td>
<td align="left" colspan="1" rowspan="1">291.1964 [M – H]<sup>−</sup>
</td>
<td align="left" colspan="1" rowspan="1">293.2106 [M + H]<sup>+</sup>
</td>
<td align="left" colspan="1" rowspan="1">C<sub>18</sub>H<sub>28</sub>O<sub>3</sub>
</td>
<td align="left" colspan="1" rowspan="1">107.0491 [C<sub>7</sub>H<sub>7</sub>O<sup>+</sup>]</td>
<td align="center" colspan="1" rowspan="1">
<strong>11</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>M58</strong>
</td>
<td align="left" colspan="1" rowspan="1">319.2276 [M – H]<sup>−</sup>
</td>
<td align="left" colspan="1" rowspan="1">303.2319 [M-H<sub>2</sub>O + H]<sup>+</sup>
</td>
<td align="left" colspan="1" rowspan="1">C<sub>20</sub>H<sub>32</sub>O<sub>3</sub>
</td>
<td align="left" colspan="1" rowspan="1">107.0491
[C<sub>7</sub>H<sub>7</sub>O<sup>+</sup>]</td>
<td align="center" colspan="1" rowspan="1">
<strong>12</strong>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t1fn1">
<sup>a</sup><p class="display-inline">For each microfraction of interest: <em>m/z</em> in positive (PI) and negative ion mode (NI), molecular
formulae, and characteristic fragment found in the MS<sup>2</sup> spectrum
and the associated compound number corresponding to eventually purified
compound.</p>
</div></div></section><p>Manual inspection of MS<sup>2</sup> spectra then revealed
some
fragmentation patterns that appeared across the different fractions.
Two fragment ions were found repeatedly and in a mutually exclusive
manner in the different fractions targeted (<a href="#tbl1" class="usa-link">Table 
</a>), one with an <em>m</em>/<em>z</em> of 149.0597 (C<sub>9</sub>H<sub>9</sub>O<sup>+</sup>)
and another with an <em>m</em>/<em>z</em> of 107.0491
(C<sub>7</sub>H<sub>7</sub>O<sup>+</sup>). This observation suggested
that the main compounds highlighted as being bioactive exhibited repeated
structural patterns. The C<sub>7</sub>H<sub>7</sub>O<sup>+</sup> fragment
is in fact a plausible fragment of anacardic acids and was found in
all three sets of microfractions (<strong>M25</strong>-<strong>26</strong>, <strong>M29</strong>-<strong>30</strong>, and <strong>M30</strong>-<strong>31</strong>) annotated as anacardic acid derivatives (<a href="#tbl1" class="usa-link">Table 
</a>).</p>
<p>Several bioactive HPLC peaks could
not be fully dereplicated at
this level, and several MF were not previously reported in the <em>Knema</em> genus. To establish biological activity for individual
compounds and unambiguously determine their structure, targeted isolations
were carried out to purify all of the NPs of interest listed in <a href="#tbl1" class="usa-link">Table 
</a>.</p></section><section id="sec2.4"><h3 class="pmc_sec_title">Isolation of Targeted Bioactive NPs</h3>
<p>For complete de
novo structural identification and bioactivity characterization, isolation
of compounds corresponding to MS<sup>2</sup> spectra highlighted by
HPLC bioactivity profiling was necessary. This was done at a larger
scale using flash-UV chromatography (0.5 g of extract used against
13.05 mg used for the HPLC bioactivity profiling process). First,
the standard conditions were optimized at the analytical scale using
an HPLC before being geometrically transferred to the flash-UV chromatography scale. Using this approach,
12 NPs were isolated in amounts ranging from 2.4 to 38 mg (<a href="#fig3" class="usa-link">Figure 
</a>).</p>
<p>Full de
novo structure identifications of the isolated compounds revealed
that they were acetophenone and anacardic acid derivatives. A total
of five structures among those in <a href="#fig3" class="usa-link">Figure 
</a> were known (<strong>8</strong>–<strong>12</strong>). As expected from previous dereplications, khookerianic
acid A (<strong>8</strong>, <strong>M29</strong>-<strong>30</strong>) and khookerianic
acid C (<strong>10</strong>, <strong>M30</strong>-<strong>31</strong>) were previously
isolated from (Q5457252). Kneglobularic acid B (<strong>9</strong>, <strong>M25</strong>-<strong>26</strong>) and kneglobularic acid A (other name
for <strong>10</strong>) were previously isolated from (Q5399660). Anagigantic acid (<strong>11</strong>) was first isolated from the pericarp
of <em>Anacardium giganteum</em> (Q9673175) and 6-tridecylsalicylic acid (<strong>12</strong>) was isolated
from the brown algae <em>Caulocystis cephalornithos</em> (Q29146806).
</p>
<p>Additionally, seven new structures were
identified and were named
knemolone A–D (<strong>1–2</strong>, <strong>4</strong>–<strong>5</strong>) and knemolic acid A–C (<strong>3</strong>, <strong>6</strong>–<strong>7</strong>) based on characteristic 2D-NMR correlations
(<a href="#fig4" class="usa-link">Figure 
</a>).</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371487_bg5c00052_0004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/96b1/12371487/277d8928022d/bg5c00052_0004.jpg" loading="lazy" id="gr4" height="178" width="800" alt="4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Representative
2D NMR interactions for acetophenone (1) and salicylic
acid scaffolds (3). Blue: HMBC correlations, green: COSY correlations,
and red: ROESY correlations.</p></figcaption></figure><p>Knemolone A–D (<strong>1–2</strong>, <strong>4–5</strong>) were isolated as yellow oils with [M-H]<sup>−</sup> of <em>m</em>/<em>z</em> 327.1237/283.1337/311.1651/277.1808
corresponding to MF of C<sub>19</sub>H<sub>20</sub>O<sub>5</sub> (error
−0.31 ppm)/C<sub>18</sub>H<sub>20</sub>O<sub>3</sub> (error
−1.06 ppm)/C<sub>20</sub>H<sub>24</sub>O<sub>3</sub> (error
−0.64 ppm)/C<sub>17</sub>H<sub>26</sub>O<sub>3</sub> (error
−0.36 ppm), respectively. The <sup>1</sup>H and <sup>13</sup>C NMR data (<a href="#tbl2" class="usa-link">Table 
</a> and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00052/suppl_file/bg5c00052_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Information</a> S1.3./S2.3./S4.3./S5.3.
and S1.4./S2.4./S4.4./S5.4.) displayed evidence of the resacetophenone
moiety in these four compounds being reminiscent of that described
in khookerianone A. Similarly, the alkyl
chain connected to the aromatic ring at C-6 with correlations described
in that report was also observed in the data of all the new knemolones.</p>
<section class="tw xbox font-sm" id="tbl2"><h4 class="obj_head">2. 
<sup>1</sup>H and <sup>13</sup>C NMR
Data of Knemolones A–D 1–2 and 4–5.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
</colgroup>
<thead>
<tr>
<th rowspan="2" align="center" colspan="1">position</th>
<th colspan="2" align="center" rowspan="1">1<hr>
</th>
<th colspan="2" align="center" rowspan="1">2<hr>
</th>
<th colspan="2" align="center" rowspan="1">4<hr>
</th>
<th colspan="2" align="center" rowspan="1">5<hr>
</th>
</tr>
<tr>
<th align="center" colspan="1" rowspan="1">
<sup>1</sup>
<em>H</em>
</th>
<th align="center" colspan="1" rowspan="1">
<sup>13</sup>
<em>C</em>
</th>
<th align="center" colspan="1" rowspan="1">
<sup>1</sup>
<em>H</em>
</th>
<th align="center" colspan="1" rowspan="1">
<sup>13</sup>
<em>C</em>
</th>
<th align="center" colspan="1" rowspan="1">
<sup>1</sup>
<em>H</em>
</th>
<th align="center" colspan="1" rowspan="1">
<sup>13</sup>
<em>C</em>
</th>
<th align="center" colspan="1" rowspan="1">
<sup>1</sup>
<em>H</em>
</th>
<th align="center" colspan="1" rowspan="1">
<sup>13</sup>
<em>C</em>
</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">115.5</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">115.6</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">115.5</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">115.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">166.1</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">166.1</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">166.1</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">166.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">6.24 d (2.6)</td>
<td align="center" colspan="1" rowspan="1">101.9</td>
<td align="left" colspan="1" rowspan="1">6.24 d (2.4)</td>
<td align="center" colspan="1" rowspan="1">100.9</td>
<td align="left" colspan="1" rowspan="1">6.24 q (2.6)</td>
<td align="center" colspan="1" rowspan="1">101.8</td>
<td align="left" colspan="1" rowspan="1">6.24 d (2.6)</td>
<td align="center" colspan="1" rowspan="1">101.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">160.9</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">160.8</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">161.0</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">160.9</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">6.22
d (2.5)</td>
<td align="center" colspan="1" rowspan="1">110.6</td>
<td align="left" colspan="1" rowspan="1">6.21 d (2.5)</td>
<td align="center" colspan="1" rowspan="1">110.6</td>
<td align="left" colspan="1" rowspan="1">6.24 q (2.6)</td>
<td align="center" colspan="1" rowspan="1">110.7</td>
<td align="left" colspan="1" rowspan="1">6.26 d (2.6)</td>
<td align="center" colspan="1" rowspan="1">110.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">147.5</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">147.5</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">147.9</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">148.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">204.2</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">204.3</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">204.4</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">204.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8</td>
<td align="left" colspan="1" rowspan="1">2.60 s</td>
<td align="center" colspan="1" rowspan="1">32.3</td>
<td align="left" colspan="1" rowspan="1">2.59 s</td>
<td align="center" colspan="1" rowspan="1">32.3</td>
<td align="left" colspan="1" rowspan="1">2.62 s</td>
<td align="center" colspan="1" rowspan="1">32.3</td>
<td align="left" colspan="1" rowspan="1">2.64 s</td>
<td align="center" colspan="1" rowspan="1">32.3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1′</td>
<td align="left" colspan="1" rowspan="1">2.85 t (7.5)</td>
<td align="center" colspan="1" rowspan="1">36.3</td>
<td align="left" colspan="1" rowspan="1">2.86 t (7.5)</td>
<td align="center" colspan="1" rowspan="1">36.3</td>
<td align="left" colspan="1" rowspan="1">2.82 t (8.0)</td>
<td align="center" colspan="1" rowspan="1">36.4</td>
<td align="left" colspan="1" rowspan="1">2.83 t (8.1)</td>
<td align="center" colspan="1" rowspan="1">36.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2′</td>
<td align="left" colspan="1" rowspan="1">1.64 m</td>
<td align="center" colspan="1" rowspan="1">31.7</td>
<td align="left" colspan="1" rowspan="1">1.64 m</td>
<td align="center" colspan="1" rowspan="1">31.7</td>
<td align="left" colspan="1" rowspan="1">1.55–1.67 m</td>
<td align="center" colspan="1" rowspan="1">32.3</td>
<td align="left" colspan="1" rowspan="1">1.59 m</td>
<td align="center" colspan="1" rowspan="1">32.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3′</td>
<td align="left" colspan="1" rowspan="1">1.64 m</td>
<td align="center" colspan="1" rowspan="1">31.6</td>
<td align="left" colspan="1" rowspan="1">1.71 m</td>
<td align="center" colspan="1" rowspan="1">31.5</td>
<td align="left" colspan="1" rowspan="1">1.38 m</td>
<td align="center" colspan="1" rowspan="1">29.7</td>
<td align="left" colspan="1" rowspan="1">1.36 m</td>
<td align="center" colspan="1" rowspan="1">29.9</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4′</td>
<td align="left" colspan="1" rowspan="1">2.56
t (7.1)</td>
<td align="center" colspan="1" rowspan="1">35.5</td>
<td align="left" colspan="1" rowspan="1">2.64 t (7.5)</td>
<td align="center" colspan="1" rowspan="1">35.8</td>
<td align="left" colspan="1" rowspan="1">1.38 m</td>
<td align="center" colspan="1" rowspan="1">29.1</td>
<td align="left" colspan="1" rowspan="1">1.22–1.30 m</td>
<td align="center" colspan="1" rowspan="1">29.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5′</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">1.55–1.67 m</td>
<td align="center" colspan="1" rowspan="1">31.5</td>
<td align="left" colspan="1" rowspan="1">1.22–1.30 m</td>
<td align="center" colspan="1" rowspan="1">29.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6′</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">2.60 t (7.7)</td>
<td align="center" colspan="1" rowspan="1">36.0</td>
<td align="left" colspan="1" rowspan="1">1.22–1.30 m</td>
<td align="center" colspan="1" rowspan="1">29.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7′</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">1.22–1.30 m</td>
<td align="center" colspan="1" rowspan="1">32.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">8′</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">1.22–1.30 m</td>
<td align="center" colspan="1" rowspan="1">22.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">9′</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">0.88 t (6.9)</td>
<td align="center" colspan="1" rowspan="1">14.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1″</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">136.0</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">142.1</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">142.7</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2″</td>
<td align="left" colspan="1" rowspan="1">6.65 s</td>
<td align="center" colspan="1" rowspan="1">108.3</td>
<td align="left" colspan="1" rowspan="1">7.28 t (7.5)</td>
<td align="center" colspan="1" rowspan="1">128.5</td>
<td align="left" colspan="1" rowspan="1">7.28 t (7.5)</td>
<td align="center" colspan="1" rowspan="1">128.4</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3′’</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">147.7</td>
<td align="left" colspan="1" rowspan="1">7.14–7.22 m</td>
<td align="center" colspan="1" rowspan="1">128.5</td>
<td align="left" colspan="1" rowspan="1">7.14–7.21 m</td>
<td align="center" colspan="1" rowspan="1">128.5</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4″</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">145.8</td>
<td align="left" colspan="1" rowspan="1">7.14–7.22 m</td>
<td align="center" colspan="1" rowspan="1">126.0</td>
<td align="left" colspan="1" rowspan="1">7.14–7.21 m</td>
<td align="center" colspan="1" rowspan="1">125.8</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5′’</td>
<td align="left" colspan="1" rowspan="1">6.72 d
(7.8)</td>
<td align="center" colspan="1" rowspan="1">108.9</td>
<td align="left" colspan="1" rowspan="1">7.14–7.22 m</td>
<td align="center" colspan="1" rowspan="1">128.5</td>
<td align="left" colspan="1" rowspan="1">7.14–7.21 m</td>
<td align="center" colspan="1" rowspan="1">128.5</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6″</td>
<td align="left" colspan="1" rowspan="1">6.60 d (7.8)</td>
<td align="center" colspan="1" rowspan="1">121.2</td>
<td align="left" colspan="1" rowspan="1">7.28 t (7.5)</td>
<td align="center" colspan="1" rowspan="1">128.5</td>
<td align="left" colspan="1" rowspan="1">7.28 t (7.5)</td>
<td align="center" colspan="1" rowspan="1">128.4</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1‴</td>
<td align="left" colspan="1" rowspan="1">5.92 s</td>
<td align="center" colspan="1" rowspan="1">100.9</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>Differences appeared in the lengths of the alkyl chains
and its
termination. Knemolone A (<strong>1</strong>), B (<strong>2</strong>), and
C (<strong>4</strong>) all showed evidence of the other end of the alkyl
chain being connected to another aromatic ring at the C-1a position
(δ<sub>
<em>C</em>
</sub> 136.0/142.1/142.7), with HMBC
correlations between H-4′ (δ<sub>
<em>H</em>
</sub> 2.56/2.64) and H-6′ (δ<sub>
<em>H</em>
</sub> 2.60) and C-2″ (δ<sub>
<em>C</em>
</sub> 108.3/128.5/128.4)
and C-6″ (δ<sub>
<em>C</em>
</sub> 121.2/128.5/128.4).
This aromatic ring appeared to be trisubstituted in <strong>1</strong> as opposed to monosubstituted and terminal in <strong>2</strong> and <strong>4</strong> (see δ<sub>
<em>H</em>
</sub> in <a href="#tbl2" class="usa-link">Table 
</a>). Compound <strong>1</strong> was
substituted at C-3′’ and C-4″ with a dioxolane
cycle, evidenced by the HMBC correlations between the dioxolane OCH<sub>2</sub>O (C-1‴, δ<sub>
<em>H</em>
</sub> 5.92)
and C-3″ (δ<sub>
<em>C</em>
</sub> 147.7) and C-4″
(δ<sub>
<em>C</em>
</sub> 145.8). Knemolone D (<strong>5</strong>), on the other hand, had no second aromatic ring, instead having
a terminal methyl group at the end of a longer chain of 9 carbons
at C-9′ (δ<sub>
<em>H</em>
</sub> 0.88) correlating
with C-8′ (δ<sub>
<em>C</em>
</sub> 22.8) and C-7′
(δ<sub>
<em>C</em>
</sub> 32.0).</p>
<p>Knemolic acids
A–C (<strong>3</strong>, <strong>6–7</strong>) were isolated
as light green amorphous powders with [M-H]<sup>−</sup> of <em>m</em>/<em>z</em> 313.1079/341.1391/297.1495
corresponding to MF of C<sub>18</sub>H<sub>18</sub>O<sub>5</sub> (error
−0.64 ppm)/C<sub>20</sub>H<sub>22</sub>O<sub>5</sub> (error
−0.88 ppm)/C<sub>19</sub>H<sub>22</sub>O<sub>3</sub> (error
−0.34 ppm), respectively. The <sup>1</sup>H and <sup>13</sup>C NMR data (<a href="#tbl3" class="usa-link">Table 
</a> and <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00052/suppl_file/bg5c00052_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Information</a> S3.2./S6.3./S7.3.
and S3.3./S6.4./S7.4.) for knemolic acid A was in fact similar to
that of knemolone A (<strong>1</strong>), except for positions 1–7.
Instead of a tetrasubstituted resacetophenone scaffold, compound <strong>3</strong> showed evidence of a trisubstituted anacardic acid scaffold.
C-7 (δ<sub>
<em>C</em>
</sub> 174.0) has a chemical shift
in line with that of a carboxylic acid, which is much lower than that
of the previously observed ketone. This carboxylic acid group was
positioned on C-1 (δ<sub>
<em>C</em>
</sub> 110.5) as
it was the last available point of substitution of the aromatic ring
not yet assigned. The chemical shift of C-1 was also in agreement
with the presence of an acid group and a neighboring hydroxyl group
on C-2 (δ<sub>
<em>C</em>
</sub> 163.8). Chemical shifts
on that aromatic ring were also in line with those reported for khookerianic
acid C.
</p>
<section class="tw xbox font-sm" id="tbl3"><h4 class="obj_head">3. 
<sup>1</sup>H and <sup>13</sup>C NMR
Data of Knemolic Acids A–C 3 and 6–7.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
</colgroup>
<thead>
<tr>
<th rowspan="2" align="center" colspan="1">position</th>
<th colspan="2" align="center" rowspan="1">3<hr>
</th>
<th colspan="2" align="center" rowspan="1">6<hr>
</th>
<th colspan="2" align="center" rowspan="1">7<hr>
</th>
</tr>
<tr>
<th align="center" colspan="1" rowspan="1">
<sup>1</sup>
<em>H</em>
</th>
<th align="center" colspan="1" rowspan="1">
<sup>13</sup>
<em>C</em>
</th>
<th align="center" colspan="1" rowspan="1">
<sup>1</sup>
<em>H</em>
</th>
<th align="center" colspan="1" rowspan="1">
<sup>13</sup>
<em>C</em>
</th>
<th align="center" colspan="1" rowspan="1">
<sup>1</sup>
<em>H</em>
</th>
<th align="center" colspan="1" rowspan="1">
<sup>13</sup>
<em>C</em>
</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">1</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">110.5</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">110.5</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">110.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">163.8</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">163.8</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">163.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3</td>
<td align="left" colspan="1" rowspan="1">6.86 dd (8.4, 1.3)</td>
<td align="center" colspan="1" rowspan="1">116.1</td>
<td align="left" colspan="1" rowspan="1">6.87 dd (8.3, 1.2)</td>
<td align="center" colspan="1" rowspan="1">116.0</td>
<td align="left" colspan="1" rowspan="1">6.87 dd
(8.3, 1.2)</td>
<td align="center" colspan="1" rowspan="1">116.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4</td>
<td align="left" colspan="1" rowspan="1">7.34 t (7.9)</td>
<td align="center" colspan="1" rowspan="1">135.3</td>
<td align="left" colspan="1" rowspan="1">7.36 t (7.9)</td>
<td align="center" colspan="1" rowspan="1">135.6</td>
<td align="left" colspan="1" rowspan="1">7.36 t (7.5)</td>
<td align="center" colspan="1" rowspan="1">135.5</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5</td>
<td align="left" colspan="1" rowspan="1">6.74 dd (7.5, 1.3)</td>
<td align="center" colspan="1" rowspan="1">122.8</td>
<td align="left" colspan="1" rowspan="1">6.76 dd (7.5, 1.2)</td>
<td align="center" colspan="1" rowspan="1">122.9</td>
<td align="left" colspan="1" rowspan="1">6.76 dd (7.5, 1.3)</td>
<td align="center" colspan="1" rowspan="1">122.9</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">147.1</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">147.8</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">147.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">7</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">174.0</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">175.9</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">175.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1′</td>
<td align="left" colspan="1" rowspan="1">2.98 t (7.3)</td>
<td align="center" colspan="1" rowspan="1">36.5</td>
<td align="left" colspan="1" rowspan="1">2.97 t (8.0)</td>
<td align="center" colspan="1" rowspan="1">36.6</td>
<td align="left" colspan="1" rowspan="1">2.97 t (8.0)</td>
<td align="center" colspan="1" rowspan="1">36.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2′</td>
<td align="left" colspan="1" rowspan="1">1.63 m</td>
<td align="center" colspan="1" rowspan="1">31.6</td>
<td align="left" colspan="1" rowspan="1">1.59 m</td>
<td align="center" colspan="1" rowspan="1">32.0</td>
<td align="left" colspan="1" rowspan="1">1.62 m</td>
<td align="center" colspan="1" rowspan="1">32.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3′</td>
<td align="left" colspan="1" rowspan="1">1.63 m</td>
<td align="center" colspan="1" rowspan="1">31.9</td>
<td align="left" colspan="1" rowspan="1">1.37 m</td>
<td align="center" colspan="1" rowspan="1">29.1</td>
<td align="left" colspan="1" rowspan="1">1.40 m</td>
<td align="center" colspan="1" rowspan="1">29.8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4′</td>
<td align="left" colspan="1" rowspan="1">2.55
t (7.0)</td>
<td align="center" colspan="1" rowspan="1">35.6</td>
<td align="left" colspan="1" rowspan="1">1.37 m</td>
<td align="center" colspan="1" rowspan="1">29.7</td>
<td align="left" colspan="1" rowspan="1">1.40 m</td>
<td align="center" colspan="1" rowspan="1">29.2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5′</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">1.59 m</td>
<td align="center" colspan="1" rowspan="1">31.8</td>
<td align="left" colspan="1" rowspan="1">1.62 m</td>
<td align="center" colspan="1" rowspan="1">31.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6′</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
<td align="left" colspan="1" rowspan="1">2.52 t (7.7)</td>
<td align="center" colspan="1" rowspan="1">35.8</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">36.1</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1″</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">136.5</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">136.8</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">143.0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">2″</td>
<td align="left" colspan="1" rowspan="1">6.67 d (1.8)</td>
<td align="center" colspan="1" rowspan="1">108.2</td>
<td align="left" colspan="1" rowspan="1">6.67 d (1.8)</td>
<td align="center" colspan="1" rowspan="1">109.0</td>
<td align="left" colspan="1" rowspan="1">7.27 dd (8.3, 7.0)</td>
<td align="center" colspan="1" rowspan="1">128.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">3′’</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">147.6</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">147.6</td>
<td align="left" colspan="1" rowspan="1">7.15–7.20 m</td>
<td align="center" colspan="1" rowspan="1">128.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">4″</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">145.6</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1">145.5</td>
<td align="left" colspan="1" rowspan="1">7.15–7.20 m</td>
<td align="center" colspan="1" rowspan="1">125.7</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">5″</td>
<td align="left" colspan="1" rowspan="1">6.71 d (7.8)</td>
<td align="center" colspan="1" rowspan="1">109.0</td>
<td align="left" colspan="1" rowspan="1">6.71 d (6.71)</td>
<td align="center" colspan="1" rowspan="1">108.2</td>
<td align="left" colspan="1" rowspan="1">7.15–7.20 m</td>
<td align="center" colspan="1" rowspan="1">128.6</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">6″</td>
<td align="left" colspan="1" rowspan="1">6.61 dd (7.9,
1.8)</td>
<td align="center" colspan="1" rowspan="1">121.2</td>
<td align="left" colspan="1" rowspan="1">6.61 dd (7.9, 1.7)</td>
<td align="center" colspan="1" rowspan="1">121.2</td>
<td align="left" colspan="1" rowspan="1">7.27 dd (8.3, 7.0)</td>
<td align="center" colspan="1" rowspan="1">128.4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">1‴</td>
<td align="left" colspan="1" rowspan="1">5.91 s</td>
<td align="center" colspan="1" rowspan="1">100.9</td>
<td align="left" colspan="1" rowspan="1">5.90 s</td>
<td align="center" colspan="1" rowspan="1">100.8</td>
<td align="left" colspan="1" rowspan="1"> </td>
<td align="center" colspan="1" rowspan="1"> </td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>In the end, 4 new acetophenone derivatives (<strong>1</strong>–<strong>2</strong>, <strong>4</strong>–<strong>5</strong>) and 8 salicylic acid
derivatives (<strong>3</strong>, <strong>6</strong>–<strong>12</strong>), of which 3 were newly described NPs (<strong>3</strong>, <strong>6</strong>, <strong>7</strong>), were isolated. Once the structures were elucidated,
it became clear that the C<sub>9</sub>H<sub>9</sub>O<sup>+</sup> fragment
(<a href="#tbl1" class="usa-link">Table 
</a>) was characteristic
to all the acetophenone derivatives, while the C<sub>7</sub>H<sub>7</sub>O<sup>+</sup> fragment was characteristic to all salicylic
acid derivatives.</p>
<p>The purity of each identified NP was assessed
using an NMR-based
process (described in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00052/suppl_file/bg5c00052_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Information</a> Figure 2), which revealed varying levels of impurities for the 12
isolated NPs (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00052/suppl_file/bg5c00052_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Information</a> Table
1). The following part on biological results was refined to focus
on compounds that were considered pure (at least 90% purity according
to the described metric, leaving compounds <strong>7</strong> and <strong>8</strong>, <a href="#tbl4" class="usa-link">Table 
</a>),
but all 12 samples containing the NPs have been tested, and the full
data with associated purity scores can be found in <a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00052/suppl_file/bg5c00052_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Supporting Information</a> Table 2. Additionally, a commercial
standard of similar chemical structure (anacardic acid) was also tested
to confirm that the anti-infective activities observed were indeed
associated with the studied chemical structures. For all compounds,
the level of impurities did not preclude their unambiguous structural
identification (by NMR and HRMS).</p>
<section class="tw xbox font-sm" id="tbl4"><h4 class="obj_head">4. Biological Activities of NPs Isolated
from Ethyl Acetate
Leaf Extract<sup></sup>
.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">compound</th>
<th align="center" colspan="1" rowspan="1">IC<sub>50</sub> on Mm in infection (μM)</th>
<th align="center" colspan="1" rowspan="1">IC<sub>50</sub> on Dd in infection (μM)</th>
<th align="center" colspan="1" rowspan="1">IC<sub>50</sub> on Mm <em>in broth</em> (μM)</th>
<th align="center" colspan="1" rowspan="1">MIC on Sa (mg·L–1)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>7</strong>
</td>
<td align="center" colspan="1" rowspan="1">7.41</td>
<td align="left" colspan="1" rowspan="1">7.41</td>
<td align="left" colspan="1" rowspan="1">not active</td>
<td align="center" colspan="1" rowspan="1">8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>8</strong>
</td>
<td align="center" colspan="1" rowspan="1">2.47</td>
<td align="left" colspan="1" rowspan="1">2.47</td>
<td align="left" colspan="1" rowspan="1">not active</td>
<td align="center" colspan="1" rowspan="1">8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>anacardic acid</strong>
</td>
<td align="center" colspan="1" rowspan="1">22.22</td>
<td align="left" colspan="1" rowspan="1">not
active</td>
<td align="left" colspan="1" rowspan="1">not active</td>
<td align="center" colspan="1" rowspan="1">16</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>rifabutin</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.2</td>
<td align="left" colspan="1" rowspan="1">not
active</td>
<td align="left" colspan="1" rowspan="1">0.03</td>
<td align="center" colspan="1" rowspan="1">not tested</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>vancomycin</strong>
</td>
<td align="center" colspan="1" rowspan="1">not tested</td>
<td align="left" colspan="1" rowspan="1">not tested</td>
<td align="left" colspan="1" rowspan="1">not tested</td>
<td align="center" colspan="1" rowspan="1">1</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="t4fn1">
<sup>a</sup><p class="display-inline">Anacardic was added as a standard
of the same chemical class as some isolated NPs, rifabutin as the
positive control for antimycobacterial activity against Mm, and vancomycin
for antibacterial activities against (Sa).</p>
</div></div></section></section><section id="sec2.5"><h3 class="pmc_sec_title">Bioactivities of Isolated NPs</h3>
<p>Most of the isolated
compounds followed a similar pattern of activity when it came to their
effects on Mm. They were broadly inactive against the bacterium alone
(“in broth”), while in the infection model, most of
them affected both Mm and the host Dd in a similar manner (<a href="#tbl4" class="usa-link">Table 
</a>). This followed the
initial pattern of activity observed at the extract level, with no
separation of the anti-infective effect from the unwanted inhibition
of the amoeba growth. Salicylic acid derivatives were generally more
active than the acetophenones on bacterial growth in infection.</p>
<p>All 12 compounds were also tested against Gram-negative (Pa) and Gram-positive (Sa). While none of them displayed specific
activity against Pa, most of them showed some activity when tested
against Sa with minimum inhibitory concentrations (MICs) reaching
as low as 8 μg.mL<sup>–1</sup> (<a href="#tbl4" class="usa-link">Table 
</a>). Readouts on those two additional bacteria
seemed to suggest that Mm in broth behaved more closely toward Pa
than toward Sa in the case of these isolated NPs.</p>
<p>Anacardic
acid was already reported as active against Sa at a dose of 25 μg mL<sup>–1</sup>, which
is broadly in line with the experimental MIC value observed
at 16 μg mL<sup>–1</sup> (<a href="#tbl4" class="usa-link">Table 
</a>).</p>
<p>In the end, it was decided that
the acetophenone scaffold was worth
investigating as there was more chemical novelty in NPs isolated in
this category, while more extensive reports exist on the biological
activities of salicylic acid derivatives/anacardic acids.<em>
,,
</em>
</p></section><section id="sec2.6"><h3 class="pmc_sec_title">Synthetic Derivatization of a Bioactive NP Scaffold</h3>
<p>Synthetic derivatization was pursued using the acetophenone scaffold,
which offered a balance between moderate anti-infective properties
and good antibacterial activity against Sa for certain derivatives.
We opted to introduce halogen atoms (chlorine and bromine in particular)
into this scaffold, as such substitutions can boost biological activities.<em>
−

</em> Halogen bonding is the typical type of interactions halogens can
build with a target, where they behave as Lewis acids (driven by a
positively charged region known as a “σ-hole”
on the halogen) and can interact with Lewis bases. For these derivatizations, cheaply available and abundant
2,4-dihydroxyacetophenone (<strong>13</strong>, <a href="#fig5" class="usa-link">Figure 
</a>) was used as the starting material.</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12371487_bg5c00052_0005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/96b1/12371487/8f68bb647069/bg5c00052_0005.jpg" loading="lazy" id="gr5" height="1043" width="765" alt="5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Compounds synthesized
from derivatization of the 2,4-dihydroxyacetophenone
scaffold.</p></figcaption></figure><p>The first step of halogenation reaction was chosen to alter the electronegativity of
the 2,4-dihydroxyacetophenone
cycle. Both chlorine and bromine could be added in a nonselective
manner through an Electrophilic Aromatic Substitution (EAS) mechanism
on positions 3- and/or 5- of the starting material <strong>13</strong>. Bioactivity measurements on the halogenated acetophenone derivatives
suggested that chlorination (<strong>15</strong>–<strong>17</strong>) increased the potency of compounds on both anti-infective and Dd-inhibition
(<a href="#tbl5" class="usa-link">Table 
</a>). Bromination
(<strong>18</strong>–<strong>20</strong>) did not seem to affect activity
readouts in a significant manner.</p>
<section class="tw xbox font-sm" id="tbl5"><h4 class="obj_head">5. Biological Activities of Acetophenone-Based
Synthetic Derivatives, with Rifabutin and Vancomycin as Positive Controls
for Antimycobacterial and Antibacterial Activities, Respectively.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="char" char="." span="1">
<col align="left" span="1">
<col align="left" span="1">
<col align="left" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">compound</th>
<th align="center" colspan="1" rowspan="1">IC<sub>50</sub> on Mm in infection (μM)</th>
<th align="center" colspan="1" rowspan="1">IC<sub>50</sub> on Dd in infection (μM)</th>
<th align="center" colspan="1" rowspan="1">IC<sub>50</sub> on Mm <em>in broth</em> (μM)</th>
<th align="center" colspan="1" rowspan="1">MIC on Sa (mg·L<sup>–1</sup>)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>13</strong>
</td>
<td align="center" colspan="1" rowspan="1">66.67</td>
<td align="left" colspan="1" rowspan="1">not active</td>
<td align="left" colspan="1" rowspan="1">not active</td>
<td align="left" colspan="1" rowspan="1">not active</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>14</strong>
</td>
<td align="center" colspan="1" rowspan="1">2.47</td>
<td align="left" colspan="1" rowspan="1">7.41</td>
<td align="left" colspan="1" rowspan="1">66.67</td>
<td align="left" colspan="1" rowspan="1">32</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>15</strong>
</td>
<td align="center" colspan="1" rowspan="1">22.22</td>
<td align="left" colspan="1" rowspan="1">22.22</td>
<td align="left" colspan="1" rowspan="1">not active</td>
<td align="left" colspan="1" rowspan="1">not active</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>16</strong>
</td>
<td align="center" colspan="1" rowspan="1">22.22</td>
<td align="left" colspan="1" rowspan="1">22.22</td>
<td align="left" colspan="1" rowspan="1">not active</td>
<td align="left" colspan="1" rowspan="1">not active</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>17</strong>
</td>
<td align="center" colspan="1" rowspan="1">66.67</td>
<td align="left" colspan="1" rowspan="1">66.67</td>
<td align="left" colspan="1" rowspan="1">not active</td>
<td align="left" colspan="1" rowspan="1">not active</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>18</strong>
</td>
<td align="center" colspan="1" rowspan="1">66.67</td>
<td align="left" colspan="1" rowspan="1">66.67</td>
<td align="left" colspan="1" rowspan="1">not active</td>
<td align="left" colspan="1" rowspan="1">not active</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>19</strong>
</td>
<td align="center" colspan="1" rowspan="1">66.67</td>
<td align="left" colspan="1" rowspan="1">22.22</td>
<td align="left" colspan="1" rowspan="1">not
active</td>
<td align="left" colspan="1" rowspan="1">not active</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>20</strong>
</td>
<td align="center" colspan="1" rowspan="1">66.67</td>
<td align="left" colspan="1" rowspan="1">66.67</td>
<td align="left" colspan="1" rowspan="1">not active</td>
<td align="left" colspan="1" rowspan="1">not active</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>21</strong>
</td>
<td align="center" colspan="1" rowspan="1">1.76</td>
<td align="left" colspan="1" rowspan="1">0.59</td>
<td align="left" colspan="1" rowspan="1">47.60</td>
<td align="left" colspan="1" rowspan="1">32–64</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>22</strong>
</td>
<td align="center" colspan="1" rowspan="1">1.76</td>
<td align="left" colspan="1" rowspan="1">1.76</td>
<td align="left" colspan="1" rowspan="1">47.60</td>
<td align="left" colspan="1" rowspan="1">32</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>23</strong>
</td>
<td align="center" colspan="1" rowspan="1">1.76</td>
<td align="left" colspan="1" rowspan="1">1.76</td>
<td align="left" colspan="1" rowspan="1">47.60</td>
<td align="left" colspan="1" rowspan="1">4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>24</strong>
</td>
<td align="center" colspan="1" rowspan="1">2.47</td>
<td align="left" colspan="1" rowspan="1">2.47</td>
<td align="left" colspan="1" rowspan="1">not active</td>
<td align="left" colspan="1" rowspan="1">64</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>25</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.82</td>
<td align="left" colspan="1" rowspan="1">0.82</td>
<td align="left" colspan="1" rowspan="1">66.67</td>
<td align="left" colspan="1" rowspan="1">not active</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>26</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.59</td>
<td align="left" colspan="1" rowspan="1">1.47</td>
<td align="left" colspan="1" rowspan="1">47.60</td>
<td align="left" colspan="1" rowspan="1">not
active</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>27</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.59</td>
<td align="left" colspan="1" rowspan="1">15.87</td>
<td align="left" colspan="1" rowspan="1">not active</td>
<td align="left" colspan="1" rowspan="1">2</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>rifabutin</strong>
</td>
<td align="center" colspan="1" rowspan="1">0.2</td>
<td align="left" colspan="1" rowspan="1">not
active</td>
<td align="left" colspan="1" rowspan="1">0.03</td>
<td align="left" colspan="1" rowspan="1">not tested</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>vancomycin</strong>
</td>
<td align="center" colspan="1" rowspan="1">not tested</td>
<td align="left" colspan="1" rowspan="1">not tested</td>
<td align="left" colspan="1" rowspan="1">not tested</td>
<td align="left" colspan="1" rowspan="1">1</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>In the second step, an aldolization reaction was carried
out on
the ketone moiety to introduce a phenyl-containing side chain in analogy
to compounds <strong>2</strong> and <strong>4</strong>. It seemed at first
that this addition even on nonhalogenated starting material <strong>13</strong> generated toxicity toward the amoeba, with compound <strong>14</strong> having a similar effect on both the amoeba and the bacteria.
This was also reflected in all halogenated derivatives that have undergone
aldolization (<strong>15</strong>–<strong>20</strong> vs <strong>21</strong>–<strong>27</strong>), where the absolute anti-infective activity
had increased but so did the activity on the growth of the amoeba.
Eventually, the 5-chloro-chalcone scaffold (<strong>22</strong>) was chosen
for further derivatization based mainly on higher yields during the
chlorination reaction (39.3%) compared to the dichlorinated analogue
(<strong>23</strong>, 3.54%).</p>
<p>Derivatizing the aldehyde moiety used
in the aldolization reaction was another
method used to reduce the toxicity
observed with compounds derived from benzaldehyde. Both naphthaldehyde
and anthracene-carboxaldehyde were chosen as more bulky alternatives
to benzaldehyde that would retain the same types of π–π
stacking interactions. In practice, the naphthalene group did not
reduce the toxicity toward the amoeba for compound <strong>26</strong> compared to its benzaldehyde-derived analogue <strong>22</strong>. The
bulkiness of the anthracene-carboxaldehyde moiety, however, appeared
to have a positive effect as it reduced toxicity toward the amoeba
to an IC<sub>50</sub> of 15.87 μM for compound <strong>27</strong> (<a href="#tbl5" class="usa-link">Table 
</a>). In the
end, compound <strong>27</strong> was the best candidate with improved
activity compared to its natural analogues, with an IC<sub>50</sub> of 0.59 μM on intracellular Mm and 15.87 μM against
Dd growth, representing a 25-fold specificity window, making it a
promising strict anti-infective candidate for further investigations.</p>
<p>As all NPs previously introduced were inactive against Pa, synthetic
derivatives were not tested against that strain. However, they were
all tested against Sa and some showed improved activity, with the
overall best candidate also being compound <strong>27</strong>. Interestingly,
in the case of , there was
a difference in activity between the monochlorinated chalcone derivatives
(<strong>21</strong> and <strong>22</strong> with MICs at 32 μg mL<sup>–1</sup>) and the dichlorinated chalcone (<strong>23</strong>),
which had significantly lower MICs at 4 μg mL<sup>–1</sup>. This distinction was not observable with Mm in infection, where
all three patterns of chlorination yielded identical IC<sub>50</sub> values (1.76 μM).</p></section><section id="sec2.7"><h3 class="pmc_sec_title">Hemisynthesis to Derivatize a Bioactive NP</h3>
<p>Additional
experiments to correlate biological readouts between NPs and synthetic
analogues were made through hemisynthesis. The same chlorination reaction
was carried out on an isolated aliquot of compound <strong>4</strong>,
resulting in both its 3-chloro- (<strong>28</strong>) and 3,5-dichloro-derivatives
(<strong>29</strong>) being successfully isolated (<a href="#fig6" class="usa-link">Figure 
</a>, <a href="#tbl6" class="usa-link">Table 
</a>). No monochlorinated 5-chloro-derivative was observed
in sufficient quantity to be isolated, which can be explained by strong
para-directing effects in the EAS mechanism exerted by the hydroxyl
groups present in the molecule. In that
respect, the hydroxyl group at position 6 had the most notable effect
in directing the chlorination at position 3 as it was unsubstituted.
The formation of the dichlorinated analogue (<strong>29</strong>) was
likely due to some excess chlorinating reagent being involved in the
second EAS step on position 5 of monochlorinated product <strong>28</strong>.</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/96b1/12371487/0447cd617447/bg5c00052_0006.jpg" loading="lazy" id="gr6" height="489" width="3502" alt="6"></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Hemisynthetic chlorinated derivatives of compound <strong>4</strong>.</p></figcaption></figure><section class="tw xbox font-sm" id="tbl6"><h4 class="obj_head">6. Biological Activities of Hemisynthetic
Chlorinated Derivatives of Compound <strong>4</strong>
.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
</colgroup>
<thead><tr>
<th align="center" colspan="1" rowspan="1">compound</th>
<th align="center" colspan="1" rowspan="1">IC<sub>50</sub> on Mm in infection
(μM)</th>
<th align="center" colspan="1" rowspan="1">IC<sub>50</sub> on Dd in infection
(μM)</th>
<th align="center" colspan="1" rowspan="1">IC<sub>50</sub> on Mm <em>in broth</em> (μM)</th>
<th align="center" colspan="1" rowspan="1">MIC on
Sa (μg·mL<sup>–1</sup>)</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>4</strong>
</td>
<td align="center" colspan="1" rowspan="1">22.22</td>
<td align="center" colspan="1" rowspan="1">22.22</td>
<td align="center" colspan="1" rowspan="1">not active</td>
<td align="center" colspan="1" rowspan="1">4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>28</strong>
</td>
<td align="center" colspan="1" rowspan="1">2.47</td>
<td align="center" colspan="1" rowspan="1">0.82</td>
<td align="center" colspan="1" rowspan="1">22.22</td>
<td align="center" colspan="1" rowspan="1">4</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>29</strong>
</td>
<td align="center" colspan="1" rowspan="1">2.47</td>
<td align="center" colspan="1" rowspan="1">7.41</td>
<td align="center" colspan="1" rowspan="1">not active</td>
<td align="center" colspan="1" rowspan="1">4–8</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tbl6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>Chlorination of <strong>4</strong> resulted in an increase
of potency
for both anti-infective activity and Dd inhibition, while the activity
on Sa remained largely unchanged. This difference in bioactivity between
both experiments (anti-infective on Mm vs antibacterial on Sa) suggested
that the compounds may be acting through different mechanisms of action.</p></section></section><section id="sec3"><h2 class="pmc_sec_title">Conclusion</h2>
<p>This bioguided study eventually led to the
identification of 12
NPs as a first phytochemical report on the species , among which 7 structures were newly
described. The two prominent structural backbones found in the plant
(2,4-dihydroxyacetophenone and salicylic acid) were associated with
growth inhibition of both Mm and Dd, but limited to the infection
model, with no activity observed on the naked mycobacteria. Both scaffolds
were also active on Gram-positive Sa, which is in line with extensive
reports of such antibacterial activities especially for anacardic
acid derivatives.<em>
,
</em> In terms of anti-infective activities,
however, the unmodified NPs presented limited selectivity toward Mm
as they were equally active to reduce the growth of Dd.</p>
<p>Subsequent
synthetic derivatization of the 2,4-dihydroxyacetophenone
scaffold resulted in a series of derivatives with improved inhibitory
activity on both Mm in infection and. The chlorination of the acetophenone had a beneficial effect on
the anti-infective activity, as did the reduction of size of the aliphatic
chain compared to natural analogues and the increase in size of the
aromatic unsubstituted parts of the NPs. Compound <strong>27</strong> was
the best candidate for both models, having an IC<sub>50</sub> of 0.59
μM on Mm (with an improvement in its selectivity against Mm
as the IC<sub>50</sub> on the amoeba stood at 15.87 μM) as well
as an MIC of 2 μg mL<sup>–1</sup> on Sa. Such activities
recorded on a low MW molecule (374 g·mol<sup>–1</sup>)
held promise in terms of room for further functional improvements
in structure–activity relationship investigations. This compound
is also a promising candidate for the relatively simple chemistry
and a cheap two-step synthetic route it relies on.</p>
<p>Chlorination
on the NP <strong>4</strong> also showed a beneficial
effect in improving both its absolute value of IC<sub>50</sub> against
Mm and its therapeutic window compared to the host growth inhibition
(dichlorination mostly). It seemed though that fully synthetic analogues
inspired from the isolated NPs showed more potential in terms of anti-infective
activities.</p></section><section id="sec4"><h2 class="pmc_sec_title">Experimental Section</h2>
<section id="sec4.1"><h3 class="pmc_sec_title">General Experimental Procedures</h3>
<p>NMR data were collected
on a Bruker Avance III HD 600 MHz NMR spectrometer equipped with a
QCI 5 mm cryoprobe and a SampleJet automated sample changer (Bruker
BioSpin, Rheinstetten, Germany). Chemical shifts are presented in
parts per million (δ), referencing the residual CDCl<sub>3</sub> (for NPs, δH 7.26; δC 77.0) or DMSO-<em>d</em>
<sub>6</sub> (for synthetic molecules, δH 2.50; δC 39.5)
signals as internal standards for 1H and 13C NMR, respectively, with
coupling constants (J) reported in Hz. Full assignments were determined
through 2D NMR experiments (COSY, NOESY, HSQC, and HMBC). HRMS data
were acquired using an Orbitrap Exploris 120 mass spectrometer (Thermo
Scientific, Germany) with a heated electrospray (H-ESI) source. Fraction
contents were monitored using a multidetection UHPLC-PDA-ELSD-MS platform
(Waters) equipped with a single quadrupole detector and heated electrospray
ionization. Analytical HPLC utilized an Agilent Model 1260 system
with a photodiode array detector (Agilent Technologies, Santa Clara,
CA, USA). Semipreparative HPLC was conducted on a modular system (puriFlash-MS
4250, INTERCHIM, Montluçon, France) equipped with a quaternary
pump, a UV detector module, and a fraction collector.</p></section><section id="sec4.2"><h3 class="pmc_sec_title">Plant Material</h3>
<p>The plant containing the compounds of
interest was (King)
Warb. This plant belongs to the Pierre Fabre Laboratories (PFL) collection
with over 17,000 unique samples collected worldwide. The PFL collection
was registered with the European Commission under accession number
03-FR-2020. This registration certifies that the collection meets
the criteria set out in the EU ABS Regulation, which implements at
the EU level the requirements of the Nagoya Protocol regarding access
to genetic resources and the fair and equitable sharing of benefits
arising from their utilization (Sharing nature, 2022). PFL supplied
all of the vegetal material (ground dry material). The plant material
was dried for 3 days at 55 °C in an oven; then the material was
ground and stored in plastic pots at controlled temperature and humidity
in PFL facilities. The leaves were the chosen plant part used for
the plant studied, with their following unique IDs within the PFL
collection: V113295 ( leaves).</p></section><section id="sec4.3"><h3 class="pmc_sec_title">Plant Extraction</h3>
<p>The plant material was extracted in
a Thermo Scientific Dionex ASE 350 Accelerated Solvent Extractor.
The roots of (35.78
g) were extracted in a 100 mL pressure-resistant stainless steel extraction
cell using the ASE system. At the bottom and top of the cell, a cellulose
filter (Dionex) was added to prevent solid particles from reaching
the system. The cell was loaded in the tray then pressurized and extracted
with hexane, ethyl acetate, and methanol, respectively. The rinse
volume was set at 60% and the temperature at 40 °C, with 3 cycles
for each solvent (6 cycles for the ethyl acetate extracts) and a static
time set at 5 min. The resulting extracts were collected in round-bottom
flasks, combined and evaporated to dryness on a rotary evaporator
(Büchi Rotavapor R114 Labortechnik AG, Switzerland) for each
solvent to constitute the final extracts. The following extraction
yields were obtained: hexane 659.4 mg (1.84%), ethyl acetate 628.1 mg (1.76%), and methanol
1.1617 g (3.25%).</p></section><section id="sec4.4"><h3 class="pmc_sec_title">UHPLC-PDA-ELSD-MS Analyses of Fractions</h3>
<p>Aliquots (50
μL) of fractions were analyzed by UHPLC-PDA-ELSD-MS. The conditions
for the ESI (Waters Acquity QDA Detector) were set as follows: capillary
voltage 0.8 kV (negative ion mode) or 1.2 kV (positive ion mode),
cone voltage 15 V, probe temperature 600 °C, source temperature
120 °C. Detection was performed in negative (NI) then positive
ion mode (PI) with an <em>m</em>/<em>z</em> range
of 150–1250 Da. The separation was done on an Acquity UPLC
BEH C18 column (50 × 2.1 mm inner diameter, 1.7 μm; Waters)
at 0.6 mL/min, 40 °C with H<sub>2</sub>O (A) and MeCN (B) both
containing 0.1% formic acid. The following gradient was applied for
the separation: from 5 to 100% of B from 0 to 7 min, 1 min at 100%
B, and a re-equilibration step of 2 min. The ELSD was set at 45 °C,
with a gain of 3. The PDA detector (Waters Acquity) was set in the
range from 190 to 500 nm, with a resolution of 1.2 nm. Sampling rate
was set at 20 points/s.</p></section><section id="sec4.5"><h3 class="pmc_sec_title">UHPLC-HRMS/MS of Extracts, Fractions, and Pure Compounds</h3>
<p>Analyses were performed with a Waters Acquity UHPLC system coupled
to a Corona Veo RS Charged Aerosol Detector (CAD, Thermo Scientific,
Germany) and an Orbitrap Exploris 120 mass spectrometer (Thermo Scientific,
Germany). The Orbitrap employed a heated electrospray ionization source
(H-ESI) with the following parameters: spray voltage: +3.5 kV; ion
transfer tube temperature: 320.00 °C; vaporizer temperature:
320.00 °C; S-lens RF: 45 (arb units); sheath gas flow rate: 35.00
(arb units); sweep gas (arb): 1; and auxiliary gas flow rate: 10.00
(arb. units). Control of the instruments was done using Thermo Scientific
Xcalibur software v. 4.6.67.17. Full scans were acquired at a resolution
of 30,000 fwhm (at <em>m</em>/<em>z</em> 200) and
MS2 scans at 15,000 fwhm in the range of 100–1000 <em>m</em>/<em>z</em>, with 1 microscan, time (ms): 200, an RF lens
(%): 70; AGC target custom (Normalized AGC target (%): 300); maximum
injection time (ms): 130; microscans: 1; data type: profile; Use EASY-IC­(TM):
ON. The settings for dynamic exclusion mode were customized; exclude
after n times: 1; exclusion duration (s): 5; mass tolerance: ppm;
low: 10, high: 10, exclude isotopes: true. Apex detention: Desired
Apex Window (%): 50. Isotope exclusion: assigned and unassigned with
an exclusion window (<em>m</em>/<em>z</em>) for unassigned
isotopes: 8. The intensity threshold was set to 2.5 × 10<sup>5</sup>, and a targeted mass exclusion list was used.</p>
<p>The centroid
data-dependent MS2 scan acquisition events were performed in discovery
mode, triggered by Apex detection with a trigger detection (%) of
300 with a maximum injection time of 120 ms, performing 1 microscan.
The top 3 abundant precursors (charge states 1 and 2) within an isolation
window of 1.2 <em>m</em>/<em>z</em> were considered
for MS/MS analysis. For precursor fragmentation in the HCD mode, a
normalized collision energy of 15, 30, and 45% was used. Data was
recorded in profile mode (Use EASY-IC­(TM): ON).</p>
<p>The chromatographic
separation was done on a Waters BEH C18 column
(50 × 2.1 mm i.d., 1.7 μm, Waters, Milford, MA) using the
following gradient (time (min), %B): 5%B from 0 to 0.5 min; from 5%B
to 100%B between 0.5 and 7 min; 100%B from 7 to 8 min, from 100%B
to 5%B from 8 to 8.10 min; and 5%B from 8.10 to 10 min. The mobile
phases were (A) H<sub>2</sub>O and (B) acetonitrile, both containing
0.1% formic acid. The flow rate was set to 600 μL/min, the injection
volume was 2 μL, and the column was kept at 40 °C. The
PDA detector was used from 210 to 400 nm with a resolution of 1.2
nm. The CAD detector was kept at 40 °C, with 5 bar of N<sub>2</sub> and power function 1 for a data collection rate of 20 Hz.</p></section><section id="sec4.6"><h3 class="pmc_sec_title">HPLC-PDA Gradient Optimizations on the Crude Extract</h3>
<p>The analysis was carried out using an HP 1260 system equipped with
a diode-array detection unit from Agilent Technologies in Santa Clara,
CA, United States. An INTERCHIM puriFlash HQ C18 column (250 ×
4.6 mm inner diameter, 15 μm, Montluçon, France) was
employed. Detection was performed using a PDA, with UV wavelengths
set at 220, 254, 280, and 329 nm. UV spectra between 190 and 500 nm
were recorded with a threshold of 10 mAU and setting increments of
2 nm. HPLC conditions involved a mobile phase of H<sub>2</sub>O (A)
and MeOH (B), both containing 0.1% FA. The flow rate was set at 1
mL/min, with an injection volume of 10 μL. The separation temperature
was maintained at 25 °C, and the sample concentration was 10
mg/mL dissolved in MeOH. The gradient of the mobile phases was programmed
as follows: an initial hold of 3 min at 70% B, followed by a gradient
flow of 70%–87.5% of B over 39 min, followed by 3 min during
which the gradient was kept at 87.5%, then another gradient flow of
87.5%–100% of B over 19 min, ending with a 6 min washing step
with 100% B. These optimized HPLC analytical conditions were geometrically
transferred by gradient transfer to the flash-LC scale.
</p></section><section id="sec4.7"><h3 class="pmc_sec_title">UV-Flash Chromatography on the Crude Extract</h3>
<p>The ethyl
acetate extract of (leaves)
was purified with a Büchi Flash chromatography system (Büchi
Pump Module C-605, UV Photometer C-640, Control Unit C-620, Fraction
Collector C-660), using an INTERCHIM puriFlash HQ C18 column (120
g, 210 × 30 mm i.d., 15 μm, Moluçon, France). 0.5
g of extract was mixed in the stationary phase (C18 Zeoprep 40–63
μm) and sand (50–70 mesh particle size) in a proportion
of 1:1:1 and then introduced in a dry load cell. The detection was
performed by a UV Photometer with the parameters set as follows: UV
wavelengths were 220, 254, 280, and 329 nm. The mobile phase was composed
of H<sub>2</sub>O (A) and technical grade methanol (B), both containing
0.1% F.A (flow rate of 30 mL/min). The gradient slope was set as follows:
an initial hold of 4 min at 70% B, a gradient flow of 70%–87.5%
of B in 56 min, followed by a hold of 4 min at 87.5%, then another
gradient flow of 87.5%–100% in 27 min, followed by 9 min of
washing at 100% B. The separation yielded 70 fractions of 50 mL each
(F01–F70) that were dried using a multiunit evaporator (Multivapor,
Büchi Labortechnik AG, Switzerland). The following compounds
were identified after HRMS and NMR analyses for confirmations: <strong>1</strong> (F08, 2.9 mg, RT 12 min), <strong>2</strong> (F09, 2.8 mg, RT
13 min), <strong>3</strong> (F14, 2.4 mg, RT 22 min), <strong>4</strong> (F15,
6.7 mg, RT 24 min), <strong>5</strong> (F20, 5.3 mg, RT 32 min), <strong>6</strong> (F22–23, 20.8 mg, RT 35 min), <strong>7</strong> (F24–25,
11.8 mg, RT 39 min), <strong>8</strong> (F30–31, 8.6 mg, RT 48
min), <strong>9</strong> (F33, 2.5 mg, RT 53 min), <strong>10</strong> (F34–35,
5 mg, RT 56 min), <strong>11</strong> (F39–40, 6.7 mg, RT 64 min),
and <strong>12</strong> (F48, 3.1 mg, RT 79 min).</p>
<p>Knemolone A (<strong>1</strong>) Yellow amorphous solid; UV (MeOH) λ<sub>max</sub> (log ε) 201 (3.83), 222 (4.09), 285 (3.35) nm; NMR see in <a href="#tbl2" class="usa-link">Table 
</a>, NP-MRD ID: NP0333015;
HRESIMS <em>m</em>/<em>z</em> 327.1237 [M-H]<sup>−</sup> (calcd for C<sub>19</sub>H<sub>19</sub>O<sub>5</sub>
<sup>–</sup> 327.1238, Δ = −0.31 ppm), <em>m</em>/<em>z</em> 329.1382 [M + H]<sup>+</sup> (calcd for C<sub>19</sub>H<sub>21</sub>O<sub>5</sub>
<sup>+</sup> 329.1384, Δ = −0.61
ppm), MS/MS spectrum: CCMSLIB00012475055.</p>
<p>Knemolone B (<strong>2</strong>) Yellow amorphous solid; UV (MeOH)
λ<sub>max</sub> (log ε) 222 (3.95), 282 (3.03) nm; NMR
see in <a href="#tbl2" class="usa-link">Table 
</a>, NP-MRD
ID: NP0333016; HRESIMS <em>m</em>/<em>z</em> 283.1337
[M-H]<sup>−</sup> (calcd for C<sub>18</sub>H<sub>19</sub>O<sub>3</sub>
<sup>–</sup> 283.1340, Δ = −1.06 ppm), <em>m</em>/<em>z</em> 285.1483 [M + H]<sup>+</sup> (calcd
for C<sub>18</sub>H<sub>21</sub>O<sub>3</sub>
<sup>+</sup> 285.1485,
Δ = −0.70 ppm), MS/MS spectrum: CCMSLIB00012475054.</p>
<p>Knemolic acid A (<strong>3</strong>) Yellow amorphous solid; UV (MeOH)
λ<sub>max</sub> (log ε) 219 (4.04), 288 (3.06) nm; NMR
see in <a href="#tbl3" class="usa-link">Table 
</a>, NP-MRD
ID: NP0333017; HRESIMS <em>m</em>/<em>z</em> 313.1079
[M-H]<sup>−</sup> (calcd for C<sub>18</sub>H<sub>17</sub>O<sub>5</sub>
<sup>–</sup> 313.1081, Δ = −0.64 ppm), <em>m</em>/<em>z</em> 297.1119 [M-H<sub>2</sub>O + H]<sup>+</sup> (calcd for C<sub>18</sub>H<sub>17</sub>O<sub>4</sub>
<sup>+</sup> 297.1121, Δ = −0.67 ppm), MS/MS spectrum: CCMSLIB00012475067.</p>
<p>Knemolone C (<strong>4</strong>) Yellow amorphous solid; UV (MeOH)
λ<sub>max</sub> (log ε) 193 (5.10), 217 (4.79), 282 (4.33)
nm; NMR see in <a href="#tbl2" class="usa-link">Table 
</a>, NP-MRD ID: NP0333018; HRESIMS <em>m</em>/<em>z</em> 311.1651 [M-H]<sup>−</sup> (calcd for C<sub>20</sub>H<sub>23</sub>O<sub>3</sub>
<sup>–</sup> 311.1653, Δ = −0.64
ppm), <em>m</em>/<em>z</em> 313.1796 [M + H]<sup>+</sup> (calcd for C<sub>20</sub>H<sub>25</sub>O<sub>3</sub>
<sup>+</sup> 313.1789, Δ = 2.23 ppm), MS/MS spectrum: CCMSLIB00012475059.</p>
<p>Knemolone D (<strong>5</strong>) Yellow amorphous solid; UV (MeOH)
λ<sub>max</sub> (log ε) 196 (4.17), 222 (4.44), 282 (3.96)
nm; NMR see in <a href="#tbl2" class="usa-link">Table 
</a>, NP-MRD ID: NP0333019; HRESIMS <em>m</em>/<em>z</em> 277.1808 [M-H]<sup>−</sup> (calcd for C<sub>17</sub>H<sub>25</sub>O<sub>3</sub>
<sup>–</sup> 277.1809, Δ = −0.36
ppm), <em>m</em>/<em>z</em> 279.1949 [M + H]<sup>+</sup> (calcd for C<sub>17</sub>H<sub>27</sub>O<sub>3</sub>
<sup>+</sup> 279.1955, Δ = −2.15 ppm), MS/MS spectrum: CCMSLIB00012475056.</p>
<p>Knemolic acid B (<strong>6</strong>) Yellow-green amorphous solid;
UV (MeOH) λ<sub>max</sub> (log ε) 196 (5.11), 214 (5.08),
233 (4.82), 288 (4.57), 310 (4.33) nm; NMR see in <a href="#tbl3" class="usa-link">Table 
</a>, NP-MRD ID: NP0333020; HRESIMS <em>m</em>/<em>z</em> 341.1391 [M-H]<sup>−</sup> (calcd
for C<sub>20</sub>H<sub>21</sub>O<sub>5</sub>
<sup>–</sup> 341.1394,
Δ = −0.88 ppm), <em>m</em>/<em>z</em> 325.1430 [M-H<sub>2</sub>O + H]<sup>+</sup> (calcd for C<sub>20</sub>H<sub>21</sub>O<sub>4</sub>
<sup>+</sup> 325.1434, Δ = −1.23
ppm), MS/MS spectrum: CCMSLIB00012475058.</p>
<p>Knemolic acid C (<strong>7</strong>) Yellow-green amorphous solid;
UV (MeOH) λ<sub>max</sub> (log ε) 193 (5.01), 210 (4.94),
240 (4.33), 310 (3.98) nm; NMR see in <a href="#tbl3" class="usa-link">Table 
</a>, NP-MRD ID: NP0333021; HRESIMS <em>m</em>/<em>z</em> 297.1495 [M-H]<sup>−</sup> (calcd for
C<sub>19</sub>H<sub>21</sub>O<sub>3</sub>
<sup>–</sup> 297.1496,
Δ = −0.34 ppm), <em>m</em>/<em>z</em> 281.1530 [M-H<sub>2</sub>O + H]<sup>+</sup> (calcd for C<sub>19</sub>H<sub>21</sub>O<sub>2</sub>
<sup>+</sup> 281.1536, Δ = −2.13
ppm), MS/MS spectrum: CCMSLIB00012475057.</p>
<p>Khookerianic acid
A (<strong>8</strong>) Green amorphous solid; UV
(MeOH) λ<sub>max</sub> (log ε) 211 (4.51), 310 (3.62)
nm; <sup>1</sup>H NMR (CDCl<sub>3</sub>, 600 MHz): δ 7.36 (1H,
t, <em>J</em> = 7.9 Hz), 6.87 (1H, dd, <em>J</em> =
8.3, 1.1 Hz), 6.80–6.73 (1H, m), 3.00–2.95 (2H, m),
1.64–1.55 (2H, m), 1.41–1.33 (2H, m), 1.34–1.22
(10H, m), 0.87 (3H, t, <em>J</em> = 7.0 Hz); <sup>13</sup>C NMR (CDCl<sub>3</sub>, 151 MHz): δ 175.7, 163.8, 147.9, 135.5,
122.9, 116.0, 110.6, 36.6, 32.2, 32.0, 30.0, 29.7, 29.6, 29.5, 22.8,
14.3 (NP-MRD ID: NP0333022); HRESIMS <em>m</em>/<em>z</em> 263.1651 [M-H]<sup>−</sup> (calcd for C<sub>16</sub>H<sub>23</sub>O<sub>3</sub>
<sup>–</sup> 263.1653, Δ = −0.76
ppm), <em>m</em>/<em>z</em> 265.1794 [M + H]<sup>+</sup> (calcd for C<sub>16</sub>H<sub>25</sub>O<sub>3</sub>
<sup>+</sup> 265.1798, Δ = −1.51 ppm), MS/MS spectrum: CCMSLIB00012475061.</p>
<p>Kneglobularic acid B (<strong>9</strong>) Green amorphous solid; UV
(MeOH) λ<sub>max</sub> (log ε) 224 (3.93), 288 (3.66)
nm; <sup>1</sup>H NMR (CDCl<sub>3</sub>, 600 MHz): δ 7.34 (1H,
t, <em>J</em> = 7.9 Hz), 6.85 (1H, dd, <em>J</em> =
8.3, 1.2 Hz), 6.75 (1H, dd, <em>J</em> = 7.5, 1.1 Hz), 6.71
(1H, d, <em>J</em> = 7.8 Hz), 6.67 (1H, d, <em>J</em> = 1.7 Hz), 6.61 (1H, dd, <em>J</em> = 7.8, 1.7 Hz), 5.91
(2H, s), 2.94 (2H, t, <em>J</em> = 7.9 Hz), 2.51 (2H, t, <em>J</em> = 7.8 Hz), 1.60–1.53 (4H, m), 1.32–1.28
(8H, m); <sup>13</sup>C NMR (CDCl<sub>3</sub>, 151 MHz): δ 174.2,
163.7, 147.5, 145.5, 137.0, 135.3, 122.7, 121.2, 115.9, 110.5, 109.0,
108.2, 100.8, 36.6, 35.8, 32.2, 31.8, 29.9, 29.5, 29.5, 29.2 (NP-MRD
ID: NP0333023); HRESIMS <em>m</em>/<em>z</em> 369.1705
[M-H]<sup>−</sup> (calcd for C<sub>22</sub>H<sub>25</sub>O<sub>5</sub>
<sup>–</sup> 369.1707, Δ = −0.54 ppm), <em>m</em>/<em>z</em> 353.1744 [M-H<sub>2</sub>O + H]<sup>+</sup> (calcd for C<sub>22</sub>H<sub>25</sub>O<sub>4</sub>
<sup>+</sup> 353.1747, Δ = −0.85 ppm), MS/MS spectrum: CCMSLIB00012475062.</p>
<p>Khookerianic acid C/Kneglobularic acid A (<strong>10</strong>) Green
amorphous solid; UV (MeOH) λ<sub>max</sub> (log ε) 191
(4.58), 212 (4.58), 310 (3.55) nm; <sup>1</sup>H NMR (CDCl<sub>3</sub>, 600 MHz): δ 7.35 (1H, t, <em>J</em> = 7.9 Hz), 7.28–7.24
(2H, m), 7.19–7.15 (3H, m), 6.86 (1H, dd, <em>J</em> = 8.4, 1.2 Hz), 6.76 (1H, dd, <em>J</em> = 7.5, 1.2 Hz),
2.96 (2H, t, <em>J</em> = 7.9 Hz), 2.60 (2H, t, <em>J</em> = 7.8 Hz), 1.62–1.57 (4H, m), 1.36–1.29 (8H, m); <sup>13</sup>C NMR (CDCl<sub>3</sub>, 151 MHz) δ 175.1, 163.8, 147.7,
143.1, 135.5, 128.5, 128.4, 125.7, 122.8, 116.0, 110.5, 36.6, 36.1,
32.2, 31.6, 29.9, 29.6, 29.5, 29.4 (NP-MRD ID: NP0333024); HRESIMS <em>m</em>/<em>z</em> 325.1808 [M-H]<sup>−</sup> (calcd
for C<sub>21</sub>H<sub>25</sub>O<sub>3</sub>
<sup>–</sup> 325.1809,
Δ = −0.31 ppm), <em>m</em>/<em>z</em> 309.1845 [M-H<sub>2</sub>O + H]<sup>+</sup> (calcd for C<sub>21</sub>H<sub>25</sub>O<sub>2</sub>
<sup>+</sup> 309.1849, Δ = −1.29
ppm), MS/MS spectrum: CCMSLIB00012475063.</p>
<p>Anagigantic acid (<strong>11</strong>) Dark-green amorphous solid;
UV (MeOH) λ<sub>max</sub> (log ε) 213 (4.46), 311 (3.58)
nm; <sup>1</sup>H NMR (CDCl<sub>3</sub>, 600 MHz): δ 7.35 (1H,
t, <em>J</em> = 7.9 Hz), 6.86 (1H, d, <em>J</em> =
8.3 Hz), 6.77 (1H, d, <em>J</em> = 7.4 Hz), 2.96 (2H, t, <em>J</em> = 7.8 Hz), 1.59 (2H, p, <em>J</em> = 7.6 Hz),
1.36 (2H, dd, <em>J</em> = 10.3, 5.2 Hz), 1.27 (14H, d, <em>J</em> = 13.0 Hz), 0.87 (3H, t, <em>J</em> = 7.0 Hz); <sup>13</sup>C NMR (CDCl<sub>3</sub>, 151 MHz) δ 175.45, 163.76,
147.85, 135.48, 122.86, 115.98, 110.55, 36.64, 32.19, 32.07, 29.97,
29.83, 29.80, 29.78, 29.65, 29.50, 22.84, 14.27 (NP-MRD ID: NP0333025);
HRESIMS <em>m</em>/<em>z</em> 291.1964 [M-H]<sup>−</sup> (calcd for C<sub>18</sub>H<sub>27</sub>O<sub>3</sub>
<sup>–</sup> 291.1966, Δ = −0.69 ppm), <em>m</em>/<em>z</em> 293.2108 [M + H]<sup>+</sup> (calcd for C<sub>18</sub>H<sub>29</sub>O<sub>3</sub>
<sup>+</sup> 293.2111, Δ = −1.02
ppm), MS/MS spectrum: CCMSLIB00012475070.</p>
<p>6-Tridecylsalicylic
acid (<strong>12</strong>) Dark-green amorphous
solid; UV (MeOH) λ<sub>max</sub> (log ε) 225 (4.00), 311
(2.80) nm; <sup>1</sup>H NMR (CDCl<sub>3</sub>, 600 MHz): δ
7.34 (1H, dd, <em>J</em> = 8.3, 7.5 Hz), 6.85 (1H, dd, <em>J</em> = 8.4, 1.2 Hz), 6.76 (1H, dd, <em>J</em> = 7.5,
1.2 Hz), 2.98–2.92 (2H, m), 1.58 (2H, p, <em>J</em> = 7.6 Hz), 1.35 (2H, s), 1.27–1.24 (18H, m), 0.88 (3H, t, <em>J</em> = 7.0 Hz); <sup>13</sup>C NMR (CDCl<sub>3</sub>, 151 MHz):
δ 174.5, 163.8, 147.7, 135.3, 122.8, 115.9, 110.5, 36.7, 32.2,
32.1, 30.0, 29.8, 29.8, 29.8, 29.8, 29.7, 29.5, 22.8, 14.3 (NP-MRD
ID: NP0333026); HR–ESI–MS <em>m</em>/<em>z</em> 319.2277 [M-H]<sup>−</sup> (calcd for C<sub>20</sub>H<sub>31</sub>O<sub>3</sub>
<sup>–</sup> 319.2279, Δ
= −0.63 ppm), <em>m</em>/<em>z</em> 303.2318
[M-H<sub>2</sub>O + H]<sup>+</sup> (calcd for C<sub>20</sub>H<sub>31</sub>O<sub>2</sub>
<sup>+</sup> 303.2319, Δ = −0.33
ppm), MS/MS spectrum: CCMSLIB00012475069.</p></section><section id="sec4.8"><h3 class="pmc_sec_title">Generic Procedure Used for Halogenation Reactions</h3>
<p>This
procedure was inspired by that described in Wu et al. (2020). To a solution of 2,4-dihydroxyacetophenone (5
g, 1 equiv) in EtOH (50 mL) was added con. H<sub>2</sub>SO<sub>4</sub> (1 equiv) at room temperature; the mixture was stirred for 5 min.
Then, N-chlorosuccinimide (NCS) or N-bromosuccinimide (NBS) (1 equiv)
was added to the mixture. The reaction was monitored by using an UHPLC-PDA-ELSD-MS
instrument. With NBS, the reaction was generally instantaneous, while
with NCS, the reaction was left to stir for 24 h. The mixture was
evaporated to dryness and redissolved in EtOAc (500 mL). The organic
layer was washed with H<sub>2</sub>O (2 × 500 mL) and then brine
(500 mL), before being dried with MgSO4 and evaporated to dryness
to obtain 6.3314 g (with NCS) and 4.8777 g (with NBS) of the product
as a white solid (NCS)/yellow solid (NBS) mixture of 3 compounds (2
monohalogenated isomers +1 dihalogenated compound).</p>
<p>For the
chlorination reaction, a mass of 2.92 g of the mixture was then subjected
to flash chromatography using a BGB Scorpius C18e-HP column (155 g,
220 × 48 mm i.d., 30 μm, Bockten, Switzerland). The mobile
phases were (A) H<sub>2</sub>O and (B) technical grade methanol, both
containing 0.1% F.A (flow rate of 50 mL/min). The gradient slope was
set as follows: an initial hold of 2 min at 20% B, a gradient flow
of 20%–40% of B in 59 min, followed by a gradient flow of 47
min from 40%–65%, then another gradient flow of 65%–100%
in 16 min. The separation yielded 112 fractions of 50 mL each (F01–F112)
that were combined according to their respective peaks and evaporated
to dryness. The following compounds were identified after HRMS and
NMR analyses for confirmations: <strong>15</strong> (F46–59, 990.2
mg, 35.0%, RT 43–57 min), <strong>16</strong> (F61–78, 1.1116
g, 39.3%, RT 57–73 min), and <strong>17</strong> (F81–87,
100 mg, 3.54%, RT 75–81 min).</p>
<p>For the bromination reaction,
a mass of 4.6 g of the mixture was
then subjected to flash chromatography using a BGB Scorpius C18e-HP
column (155 g, 220 × 48 mm i.d., 30 μm, Bockten, Switzerland).
The mobile phases were (A) H<sub>2</sub>O and (B) technical grade
methanol, both containing 0.1% F.A (flow rate of 50 mL/min). The gradient
slope was set as follows: an isocratic gradient of 65% B during 74
min, followed by a gradient flow of 12 min from 65%–100% followed
by an isocratic washing step at 100% B for 17 min. The separation
yielded 103 fractions of 50 mL each (F01–F103) that were combined
according to their respective peaks and evaporated to dryness. The
following compounds were identified after HRMS and NMR analyses for
confirmations: <strong>18</strong> (F28–47, 1.7603 g, 24.7%, RT
30–49 min), <strong>19</strong> (F49–88, 1.2801 g, 18.0%,
RT 50–89 min), and <strong>17</strong> (F90–92, 622.4 mg,
6.52%, RT 90–92 min).</p></section><section id="sec4.9"><h3 class="pmc_sec_title">Generic Procedure Used for All Aldolization Reactions</h3>
<p>This procedure was inspired by that described in Devakaram et al.
(2011). To a solution of ketone (200
mg, 1 equiv) in MeOH (10 mL), an aldehyde (1 equiv) was added, then
10 mL of a solution of 60% KOH in water was added, and the reaction
was left to stir for 24 h. The reaction was monitored using an UHPLC-PDA-ELSD-MS
instrument. Once over, the reaction was quenched with a cold 1 M HCl
solution (100 mL), and the mixture was extracted with DCM (2 ×
20 mL). The organic layer was dried with MgSO<sub>4</sub> before being
evaporated under reduced pressure to result in 248.9 mg of the aldolized
product as a yellow powder (93%).</p>
<p>2,4-Dihydroxyacetophenone
(<strong>13</strong>), white solid; <sup>1</sup>H NMR (DMSO, 600 MHz):
δ 12.60 (1H, s), 10.61 (1H, s), 7.75 (1H, d, <em>J</em> = 8.8 Hz), 6.37 (1H, dd, <em>J</em> = 8.8, 2.4 Hz), 6.24
(1H, d, <em>J</em> = 2.3 Hz), 2.52 (3H, s); <sup>13</sup>C
NMR (DMSO, 151 MHz): δ 202.7, 164.9, 164.2, 133.7, 112.9, 108.1,
102.3, 26.4; HR–ESI–MS <em>m</em>/<em>z</em> 153.0545 [M + H]<sup>+</sup> (calcd for C<sub>8</sub>H<sub>9</sub>O<sub>3</sub>
<sup>+</sup> 153.0546, Δ = −0.65 ppm), <em>m</em>/<em>z</em> 151.0400 [M-H]<sup>−</sup> (calcd
for C<sub>8</sub>H<sub>7</sub>O<sub>3</sub>
<sup>–</sup> 151.0401,
Δ = −0.66 ppm).</p>
<p>2,4-Dihydroxychalcone (<strong>14</strong>), yellow solid; <sup>1</sup>H NMR (DMSO, 600 MHz): δ 13.38
(1H, d, <em>J</em> =
1.6 Hz), 10.75 (1H, s), 8.21 (1H, dd, <em>J</em> = 9.0, 3.3
Hz), 7.98 (1H, dd, <em>J</em> = 15.5, 3.4 Hz), 7.90 (2H, td, <em>J</em> = 4.9, 2.8 Hz), 7.80 (1H, dd, <em>J</em> = 15.4,
2.8 Hz), 7.47 (3H, td, <em>J</em> = 4.6, 2.0 Hz), 6.43 (1H,
dt, <em>J</em> = 8.9, 2.2 Hz), 6.30 (1H, t, <em>J</em> = 2.0 Hz); <sup>13</sup>C NMR (DMSO, 151 MHz): δ 191.5, 165.8,
165.3, 143.7, 134.6, 133.2, 130.7, 129.0, 128.9, 121.3, 113.0, 108.3,
102.6; HR–ESI–MS <em>m</em>/<em>z</em> 241.0856 [M + H]<sup>+</sup> (calcd for C<sub>15</sub>H<sub>13</sub>O<sub>3</sub>
<sup>+</sup> 241.0859, Δ = −1.24 ppm), <em>m</em>/<em>z</em> 239.0711 [M-H]<sup>−</sup> (calcd
for C<sub>15</sub>H<sub>11</sub>O<sub>3</sub>
<sup>–</sup> 239.0714,
Δ = −1.25 ppm).</p>
<p>3-Chloro-2,4-dihydroxyacetophenone
(<strong>15</strong>), white solid; <sup>1</sup>H NMR (DMSO, 600 MHz):
δ 13.36 (1H, s), 7.77 (1H, dd, <em>J</em> = 9.0, 0.9
Hz), 6.59 (1H, d, <em>J</em> = 8.9
Hz), 2.57 (3H, s); <sup>13</sup>C NMR (DMSO, 151 MHz): δ 203.4,
160.5, 159.9, 131.2, 113.0, 107.7, 106.7, 26.3; HR–ESI–MS <em>m</em>/<em>z</em> 187.0154 [M + H]<sup>+</sup> (calcd
for C<sub>8</sub>H<sub>8</sub>ClO<sub>3</sub>
<sup>+</sup> 187.0156,
Δ = −1.07 ppm), <em>m</em>/<em>z</em> 185.0010 [M-H]<sup>−</sup> (calcd for C<sub>8</sub>H<sub>6</sub>ClO<sub>3</sub>
<sup>–</sup> 185.0011, Δ = −0.54
ppm).</p>
<p>5-Chloro-2,4-dihydroxyacetophenone (<strong>16</strong>),
white solid; <sup>1</sup>H NMR (DMSO, 600 MHz): δ 12.34 (1H,
s), 11.43 (1H, s),
7.89 (1H, s), 6.47 (1H, s), 2.55 (3H, s); <sup>13</sup>C NMR (DMSO,
151 MHz): δ 202.1, 162.1, 159.9, 132.5, 113.7, 111.3, 103.5,
27.0; HR–ESI–MS <em>m</em>/<em>z</em> 187.0155 [M + H]<sup>+</sup> (calcd for C<sub>8</sub>H<sub>8</sub>ClO<sub>3</sub>
<sup>+</sup> 187.0156, Δ = −0.53 ppm), <em>m</em>/<em>z</em> 185.0009 [M-H]<sup>−</sup> (calcd
for C<sub>8</sub>H<sub>6</sub>ClO<sub>3</sub>
<sup>–</sup> 185.0011,
Δ = −1.08 ppm).</p>
<p>3,5-Dichloro-2,4-dihydroxyacetophenone
(<strong>17</strong>), white
solid; <sup>1</sup>H NMR (DMSO, 600 MHz): δ 13.18 (1H, s), 7.99
(1H, s), 2.61 (3H, s); <sup>13</sup>C NMR (DMSO, 151 MHz): δ
203.3, 158.2, 155.9, 130.4, 113.2, 112.2, 108.9, 26.6; HR–ESI–MS <em>m</em>/<em>z</em> 220.9767 [M + H]<sup>+</sup> (calcd
for C<sub>8</sub>H<sub>7</sub>Cl<sub>2</sub>O<sub>3</sub>
<sup>+</sup> 220.9767, Δ = 0 ppm), <em>m</em>/<em>z</em> 218.9619 [M-H]<sup>−</sup> (calcd for C<sub>8</sub>H<sub>5</sub>Cl<sub>2</sub>O<sub>3</sub>
<sup>–</sup> 218.9621, Δ
= −0.91 ppm).</p>
<p>3-Bromo-2,4-dihydroxyacetophenone (<strong>18</strong>), light brown
solid; <sup>1</sup>H NMR (DMSO, 600 MHz): δ 13.50 (1H, s), 11.50
(1H, s), 7.81 (1H, d, <em>J</em> = 8.8 Hz), 6.59 (1H, d, <em>J</em> = 8.9 Hz), 2.57 (3H, s); <sup>13</sup>C NMR (DMSO, 151
MHz): δ 203.3, 161.6, 160.9, 132.1, 113.1, 107.6, 97.2, 26.2;
HR–ESI–MS <em>m</em>/<em>z</em> 230.9650
[M + H]<sup>+</sup> (calcd for C<sub>8</sub>H<sub>8</sub>BrO<sub>3</sub>
<sup>+</sup> 230.9651, Δ = −0.43 ppm), <em>m</em>/<em>z</em> 228.9505 [M-H]<sup>−</sup> (calcd for
C<sub>8</sub>H<sub>6</sub>BrO<sub>3</sub>
<sup>–</sup> 228.9506,
Δ = −0.44 ppm).</p>
<p>5-Bromo-2,4-dihydroxyacetophenone
(<strong>19</strong>), light yellow
solid; <sup>1</sup>H NMR (DMSO, 600 MHz): δ 12.34 (1H, s), 11.48
(1H, s), 8.01 (1H, s), 6.47 (1H, s), 2.55 (3H, s); <sup>13</sup>C
NMR (DMSO, 151 MHz): δ 202.0, 162.6, 160.8, 135.6, 114.4, 103.3,
100.0, 27.0; HR–ESI–MS <em>m</em>/<em>z</em> 230.9650 [M + H]<sup>+</sup> (calcd for C<sub>8</sub>H<sub>8</sub>BrO<sub>3</sub>
<sup>+</sup> 230.9651, Δ = −0.43 ppm), <em>m</em>/<em>z</em> 228.9505 [M-H]<sup>−</sup> (calcd
for C<sub>8</sub>H<sub>6</sub>BrO<sub>3</sub>
<sup>–</sup> 228.9506,
Δ = −0.44 ppm).</p>
<p>3,5-Dibromo-2,4-dihydroxyacetophenone
(<strong>20</strong>), light
yellow solid; <sup>1</sup>H NMR (DMSO, 600 MHz): δ 13.34 (1H,
s), 12.34 (0H, s), 8.14 (1H, s), 2.62 (3H, s); <sup>13</sup>C NMR
(DMSO, 151 MHz): δ 203.2, 159.7, 157.5, 134.3, 114.3, 100.8,
99.5, 26.5; HR–ESI–MS <em>m</em>/<em>z</em> 308.8756 [M + H]<sup>+</sup> (calcd for C<sub>8</sub>H<sub>7</sub>Br<sub>2</sub>O<sub>3</sub>
<sup>+</sup> 308.8756, Δ = 0 ppm), <em>m</em>/<em>z</em> 306.8615 [M-H]<sup>−</sup> (calcd
for C<sub>8</sub>H<sub>5</sub>Br<sub>2</sub>O<sub>3</sub>
<sup>–</sup> 306.8611, Δ = 1.30 ppm).</p>
<p>3-Chloro-2,4-dihydroxychalcone
(<strong>21</strong>), yellow solid; <sup>1</sup>H NMR (DMSO, 600 MHz):
δ 14.21 (1H, s), 11.56 (1H, s),
8.23 (1H, d, <em>J</em> = 9.0 Hz), 8.00 (1H, d, <em>J</em> = 15.5 Hz), 7.92 (2H, dd, <em>J</em> = 6.6, 2.9 Hz), 7.86
(1H, d, <em>J</em> = 15.4 Hz), 7.50–7.46 (3H, m), 6.63
(1H, d, <em>J</em> = 8.9 Hz); <sup>13</sup>C NMR (DMSO, 151
MHz): δ 193.0, 161.5, 144.6, 134.5, 130.9, 130.6, 129.2, 129.0,
120.8, 113.1, 107.8, 106.9; HR–ESI–MS <em>m</em>/<em>z</em> 275.0467 [M + H]<sup>+</sup> (calcd for C<sub>15</sub>H<sub>12</sub>ClO<sub>3</sub>
<sup>+</sup> 275.0469, Δ
= −0.72 ppm), <em>m</em>/<em>z</em> 273.0322
[M-H]<sup>−</sup> (calcd for C<sub>15</sub>H<sub>10</sub>ClO<sub>3</sub>
<sup>–</sup> 273.0324, Δ = −0.73 ppm).</p>
<p>5-Chloro-2,4-dihydroxychalcone (<strong>22</strong>), yellow solid; <sup>1</sup>H NMR (DMSO, 600 MHz): δ 13.23 (1H, s), 8.42 (1H, s),
8.04 (1H, d, <em>J</em> = 15.4 Hz), 7.95 (2H, dd, <em>J</em> = 6.6, 3.0 Hz), 7.81 (1H, d, <em>J</em> = 15.4
Hz), 7.47 (3H, ddt, <em>J</em> = 5.7, 3.9, 2.2 Hz), 6.50 (1H,
s); <sup>13</sup>C NMR (DMSO, 151 MHz): δ 191.09, 163.8, 144.4,
134.6, 132.0, 130.8, 129.3, 128.9, 121.3, 113.6, 111.9, 103.7; HR–ESI–MS <em>m</em>/<em>z</em> 275.0467 [M + H]<sup>+</sup> (calcd
for C<sub>15</sub>H<sub>12</sub>ClO<sub>3</sub>
<sup>+</sup> 275.0469,
Δ = −0.73 ppm), <em>m</em>/<em>z</em> 273.0323 [M-H]<sup>−</sup> (calcd for C<sub>15</sub>H<sub>10</sub>ClO<sub>3</sub>
<sup>–</sup> 273.0324, Δ = −0.37
ppm).</p>
<p>3,5-Dichloro-2,4-dihydroxychalcone (<strong>23</strong>),
yellow solid; <sup>1</sup>H NMR (DMSO, 600 MHz): δ 14.15 (1H,
s), 8.53 (1H, s),
8.08 (1H, d, <em>J</em> = 15.4 Hz), 8.00–7.96 (2H,
m), 7.88 (1H, d, <em>J</em> = 15.4 Hz), 7.49 (3H, dd, <em>J</em> = 4.8, 1.9 Hz); <sup>13</sup>C NMR (DMSO, 151 MHz): δ
190.1, 159.9, 144.3, 134.4, 131.1, 129.6, 129.5, 128.9, 120.7; HR–ESI–MS <em>m</em>/<em>z</em> 309.0078 [M + H]<sup>+</sup> (calcd
for C<sub>15</sub>H<sub>11</sub>Cl<sub>2</sub>O<sub>3</sub>
<sup>+</sup> 309.0080, Δ = −0.65 ppm), <em>m</em>/<em>z</em> 306.9933 [M-H]<sup>−</sup> (calcd for C<sub>15</sub>H<sub>9</sub>Cl<sub>2</sub>O<sub>3</sub>
<sup>–</sup> 306.9934,
Δ = −0.33 ppm).</p>
<p>3-Bromo-2,4-dihydroxychalcone (<strong>24</strong>), light brown solid; <sup>1</sup>H NMR (DMSO, 600 MHz):
δ 14.34 (1H, s), 13.50 (1H, s),
8.27 (1H, d, <em>J</em> = 9.0 Hz), 8.01 (1H, d, <em>J</em> = 15.4 Hz), 7.92 (2H, dd, <em>J</em> = 6.6, 2.9 Hz), 7.87
(1H, d, <em>J</em> = 15.4 Hz), 7.50–7.46 (3H, m), 6.63
(2H, dd, <em>J</em> = 28.9, 8.8 Hz); <sup>13</sup>C NMR (DMSO,
151 MHz): δ 191.7, 162.5, 161.6, 144.7, 134.5, 131.5, 131.0,
129.2, 129.0, 120.7, 113.3, 107.7, 97.6; HR–ESI–MS <em>m</em>/<em>z</em> 318.9964 [M + H]<sup>+</sup> (calcd
for C<sub>15</sub>H<sub>12</sub>BrO<sub>3</sub>
<sup>+</sup> 318.9964,
Δ = 0 ppm), <em>m</em>/<em>z</em> 316.9818 [M-H]<sup>−</sup> (calcd for C<sub>15</sub>H<sub>10</sub>BrO<sub>3</sub>
<sup>–</sup> 316.9819, Δ = −0.32 ppm).</p>
<p>5-Bromo-2,4-dihydroxychalcone (<strong>25</strong>), light brown solid; <sup>1</sup>H NMR (DMSO, 600 MHz): δ 13.23 (1H, s), 11.62 (1H, s),
8.53 (1H, s), 8.04 (1H, d, <em>J</em> = 15.4 Hz), 7.96–7.92
(2H, m), 7.81 (1H, d, <em>J</em> = 15.4 Hz), 7.46 (3H, t, <em>J</em> = 3.1 Hz), 6.52 (1H, d, <em>J</em> = 1.1 Hz); <sup>13</sup>C NMR (DMSO, 151 MHz): δ 191.1, 164.3, 161.2, 144.5,
135.0, 134.6, 130.8, 129.3, 128.9, 121.3, 114.5, 103.5, 100.6; HR–ESI–MS <em>m</em>/<em>z</em> 318.9963 [M + H]<sup>+</sup> (calcd
for C<sub>15</sub>H<sub>12</sub>BrO<sub>3</sub>
<sup>+</sup> 318.9964,
Δ = −0.31 ppm), <em>m</em>/<em>z</em> 316.9819 [M-H]<sup>−</sup> (calcd for C<sub>15</sub>H<sub>10</sub>BrO<sub>3</sub>
<sup>–</sup> 316.9819, Δ = 0
ppm).</p>
<p>(E)-1-(5-Chloro-2,4-dihydroxyphenyl)-3-(naphthalen-1-yl)­prop-2-en-1-one
(<strong>26</strong>), yellow solid; <sup>1</sup>H NMR (DMSO, 600 MHz):
δ 13.19 (1H, s), 8.64 (1H, dd, <em>J</em> = 15.2, 3.5
Hz), 8.43 (1H, d, <em>J</em> = 3.9 Hz), 8.35 (1H, d, <em>J</em> = 7.1 Hz), 8.30 (1H, d, <em>J</em> = 8.4 Hz),
8.13 (1H, dd, <em>J</em> = 15.2, 4.3 Hz), 8.08 (1H, d, <em>J</em> = 8.0 Hz), 8.01 (1H, d, <em>J</em> = 8.0 Hz),
7.69–7.56 (3H, m), 6.54 (1H, d, <em>J</em> = 2.4 Hz); <sup>13</sup>C NMR (DMSO, 151 MHz): δ 190.9, 163.8, 160.4, 140.0,
133.4, 132.1, 131.3, 131.2, 131.0, 128.9, 127.4, 126.3, 126.2, 125.7,
123.7, 122.9, 113.8, 112.0, 103.8; HR–ESI–MS <em>m</em>/<em>z</em> 325.0626 [M + H]<sup>+</sup> (calcd
for C<sub>19</sub>H<sub>14</sub>ClO<sub>3</sub>
<sup>+</sup> 325.0626,
Δ = 0 ppm), <em>m</em>/<em>z</em> 323.0477 [M-H]<sup>−</sup> (calcd for C<sub>29</sub>H<sub>12</sub>ClO<sub>3</sub>
<sup>–</sup> 323.0480, Δ = −0.93 ppm).</p>
<p>(E)-3-(Anthracen-9-yl)-1-(5-chloro-2,4-dihydroxyphenyl)­prop-2-en-1-one
(<strong>27</strong>), yellow solid; <sup>1</sup>H NMR (DMSO, 600 MHz):
δ 13.28 (1H, s), 8.64 (1H, s), 8.35 (2H, dd, <em>J</em> = 8.7, 1.3 Hz), 8.26 (1H, d, <em>J</em> = 15.8 Hz), 8.16–8.11
(3H, m), 7.94 (1H, dd, <em>J</em> = 14.6, 11.1 Hz), 7.66 (1H,
d, <em>J</em> = 14.7 Hz), 7.58 (4H, dddd, <em>J</em> = 15.6, 7.8, 6.5, 1.4 Hz), 7.08 (1H, dd, <em>J</em> = 15.7,
11.1 Hz), 6.46 (1H, s); <sup>13</sup>C NMR (DMSO, 151 MHz): δ
189.4, 163.5, 139.7, 131.9, 131.4, 130.9, 129.9, 128.9, 128.2, 126.8,
125.6, 125.2, 118.1, 113.6, 103.8; HR–ESI–MS <em>m</em>/<em>z</em> 375.0780 [M + H]<sup>+</sup> (calcd
for C<sub>23</sub>H<sub>16</sub>ClO<sub>3</sub>
<sup>+</sup> 375.0782,
Δ = −0.80 ppm), <em>m</em>/<em>z</em> 373.0635 [M-H]<sup>−</sup> (calcd for C<sub>23</sub>H<sub>14</sub>ClO<sub>3</sub>
<sup>–</sup> 373.0637, Δ = −0.54
ppm).</p></section><section id="sec4.10"><h3 class="pmc_sec_title">Procedure for the Halogenation Reaction on Knemolone C (<strong>4</strong>)</h3>
<p>This procedure was inspired by that described
in Wu et al. (2020). To a solution of
Knemolone C (<strong>4</strong>, 3.05 mg 1 equiv) in EtOH (0.5 mL) was
added con. H<sub>2</sub>SO<sub>4</sub> (0.549 μL, 1.05 equiv)
at room temperature, and the mixture was stirred for 5 min. Then,
N-chlorosuccinimide (NCS) (1.304 mg, 1 equiv) was added to the mixture.
The reaction was monitored using an UHPLC-PDA-ELSD-MS instrument (reaction
was left to stir for 24 h). The mixture was evaporated to dryness
and redissolved in EtOAc (10 mL). The organic layer was washed with
H<sub>2</sub>O (2 × 10 mL) and then brine (10 mL), before being
dried with MgSO<sub>4</sub> and evaporated to dryness to obtain 2.07
mg of the product as a white solid mixture, which was then dissolved
in 200 μL of EtOAc and subjected to HPLC microfractionation
using a XBridge BEH C18 OBD Prep column (130 Å, 250 × 10
mm, 5 μm, Waters Corporation, Milford, MA). The mobile phases
were (A) H<sub>2</sub>O and (B) HPLC grade acetonitrile, both containing
0.1% F.A (flow rate of 4.7 mL/min). The gradient slope was set as
follows: an initial hold at 50% of B during 1 min, followed by a gradient
flow of 19 min from 50%–80% followed by a gradient flow from
80%–100% of B for 5 min, and finally a washing step of 5 min
at 100% B. The separation yielded 96 fractions (A01–H12) that
were combined according to their respective peaks and evaporated to
dryness. The following compounds were identified after HRMS and NMR
analyses for confirmations: <strong>28</strong> (E3–E4, 0.81 mg,
23.9%, RT 16 min) and <strong>29</strong> (E11–E12, 0.25 mg, 6.74%,
RT 19 min).</p>
<p>1-(3-Chloro-4,6-dihydroxy-2-(6-phenylhexyl)­phenyl)­ethan-1-one
(<strong>28</strong>); <sup>1</sup>H NMR (DMSO, 600 MHz): δ 10.23
(1H, s), 9.98 (1H, s), 7.29–7.23 (2H, m), 7.20–7.12
(3H, m), 6.44 (1H, s), 2.55 (2H, t, <em>J</em> = 7.8 Hz),
2.49–2.46 (2H, m), 2.39 (3H, s), 1.55 (2H, p, <em>J</em> = 7.4 Hz), 1.42 (2H, p, <em>J</em> = 6.9 Hz), 1.30 (4H,
dq, <em>J</em> = 9.4, 4.1 Hz); <sup>13</sup>C NMR (DMSO, 151
MHz): δ 203.4, 154.4, 154.0, 142.3, 138.6, 128.2, 128.2, 125.6,
122.0, 111.2, 101.3, 35.1, 32.4, 30.9, 30.2, 29.4, 28.9, 28.2; HR–ESI–MS <em>m</em>/<em>z</em> 347.1408 [M + H]<sup>+</sup> (calcd
for C<sub>20</sub>H<sub>24</sub>ClO<sub>3</sub>
<sup>+</sup> 347.1408,
Δ = 0 ppm), <em>m</em>/<em>z</em> 345.1269 [M-H]<sup>−</sup> (calcd for C<sub>20</sub>H<sub>22</sub>ClO<sub>3</sub>
<sup>–</sup> 345.1263, Δ = 1.74 ppm).</p>
<p>1-(3,5-Dichloro-2,4-dihydroxy-6-(6-phenylhexyl)­phenyl)­ethan-1-one
(<strong>29</strong>); <sup>1</sup>H NMR (DMSO, 600 MHz): δ 10.12
(1H, s), 8.13 (1H, s), 7.26 (2H, t, <em>J</em> = 7.6 Hz),
7.20–7.13 (3H, m), 2.56 (2H, t, <em>J</em> = 7.5 Hz),
2.52 (2H, t, <em>J</em> = 1.9 Hz), 2.43 (3H, s), 1.56 (2H,
p, <em>J</em> = 7.5 Hz), 1.44 (2H, p, <em>J</em> =
7.8 Hz), 1.31 (4H, dq, <em>J</em> = 11.5, 5.6 Hz); <sup>13</sup>C NMR (151 MHz, DMSO): δ 201.9, 142.2, 142.2, 128.2, 128.0,
125.6, 35.0, 30.8, 28.5, 28.1, 28.1 (partial data, sample quantity
too low for adequate <sup>13</sup>C NMR resolution); HR–ESI–MS
± no corresponding ion observed (calcd for C<sub>20</sub>H<sub>23</sub>Cl<sub>2</sub>O<sub>3</sub>
<sup>+</sup> 381.1019), (calcd
for C<sub>20</sub>H<sub>25</sub>Cl<sub>2</sub>O<sub>3</sub>
<sup>–</sup> 379.0873).</p></section><section id="sec4.11"><h3 class="pmc_sec_title">Bioactivity Screening (Mm)</h3>
<p>Plant extracts, fractions,
or isolated compounds were stored at −20 °C and wrapped
in aluminum foil if necessary. All manipulations with extracts, fractions,
or compounds were performed under a sterile hood. Extracts, fractions,
and compounds were resuspended throughout in DMSO to best solubilize
extracts with diverse constituents. Extracts, fractions, or compounds
were added to the assay plate in a 1:100 dilution. Assay solutions
of extracts, fractions, and compounds were prepared in 96-well plates,
stored at −20 °C, and thawed before the experiment at
room temperature or warmer with or without shaking/vortexing to obtain
a clear assay solution. Extracts were tested at 25 μg/mL and
purified compounds in a dose–response curve of 6 concentrations
with 100 μM being the top concentration and a 1:3 dilution step
between each testing concentration. Since fractions were not weighed
individually, the injection mass was used to calculate an average
mass per fraction and thus a nominal mass concentration. Fractions
were tested at a nominal concentration of 10 μg/mL.</p>
<p>As
described in Nitschke et al. and Mottet
et al., Dd Ax2­(ka) expressing mCherry
from the act5 chromosomal locus was infected
with Mm M strain expressing the lux operon (luxCDABE)<em>
,
</em> by spinoculation.</p>
<p>Briefly, the day before the experiment,
Mm was cultivated in 7H9
broth (Becton Dickinson, Difco Middlebrook 7H9) supplemented with
10% OADC (Becton Dickinson) and 0.05% tyloxapol (Sigma-Aldrich) and
50 μg/mL kanamycin at 32 °C overnight with continuous shaking.
Additionally, the day before the experiment, 10<sup>7</sup> amoebae
were plated in HL5-C in a 10 cm Petri dish (Falcon). On the day of
the experiment, a volume of the Mm culture corresponding to a multiplicity
of infection of 25 with respect to the number of amoebae in the Petri
dish was added to the semiconfluent amoebae monolayer. Subsequently,
the Petri dishes were centrifuged twice at 500<em>g</em>,
as described in Mottet et al. To remove
extracellular bacteria, dishes were rinsed with fresh HL5-C, and the
infected cell population was resuspended in HL5-C with 5 U/mL penicillin
and 5 μg/mL streptomycin (Gibco) to inhibit the extracellular
growth of bacteria during the course of the experiment.</p>
<p>For
testing fractions and purified compounds, 20 μL of infected
cell suspension was plated into each well of a 384-well plate (Interchim
FP-BA8240) to an effective cell number of 1 × 10<sup>4</sup> cells
per well. Fractions or compounds including a vehicle control (0.3%
DMSO final concentration) and a positive control (rifabutin, 10 μM
final concentration) were added using an electronic multipipette (Sartorius).
Subsequently, the well plates were sealed with a gas impermeable membrane
(H769.1, Carl Roth), briefly centrifuged, and intracellular bacterial
growth was monitored using an Agilent BioTek H1 plate reader and an
Agilent BioTek BioStack plate stacker by recording luminescence over
72 h at 25 °C with readings taken every hour. Fluorescence was
also recorded to monitor amoeba growth.</p>
<p>For testing bacteria
in broth, the preculture was diluted to a
bacterial density of 3.75 × 10<sup>5</sup> bacteria per mL in
7H9 medium. Plating bacteria and compounds or fractions was performed
analogously to the infection assay described above. Bacteria growth
was monitored with the Agilent BioTek H1 plate reader by recording
luminescence at 32 °C.</p>
<p>For both assays, growth curves were
obtained by measuring the luminescence
and fluorescence as a proxy for bacterial growth and host growth,
respectively, for 72 h with time-points taken every hour. The “normalized
residual growth” was computed by calculating the area under
the curve (AUC, trapezoid method) and normalizing it to the vehicle
control (0.3% DMSO, set at 1) and a baseline curve (set at 0). The
baseline curve was calculated by taking the median of the first measurement
of all wells in a plate and extrapolating it over the complete time
course. The threshold for hit detection was arbitrarily fixed at a
cutoff of normalized residual growth ≤0.5. Normalized values
were averaged over technical and biological replicates (all experiments
on isolated compounds have at least <em>n</em> = 3 and <em>N</em> = 3, whereas the primary extract screening and microfractions
testing had values of <em>n</em> = 1 and <em>N</em> = 3).</p>
<p>This procedure was also applied to screening extracts
with slight
modifications. The day before the experiment we preplated 10 μL
of HL5-C using a dispenser (Thermo Multidrop), subsequently we preplated
2.2 μL of dissolved extracts from 96-well plates into quadrants
1, 2, and 3 of 384-well plates, whereas quadrant 4 was used for positive
and vehicle controls. Preplating of extracts was performed using a
liquid handler (Agilent Bravo). In total, 24 extract plates were distributed
in triplicate into eight 384-well plates, amounting to 24 assay plates.
The prepared plates were sealed and stored at 4 °C overnight.
On the day of the experiment, nine 10 cm Petri dishes were infected
(as described before) and grouped into three pools. The cell suspension
was adjusted to 10<sup>6</sup> cells/mL, and 10 μL was plated
into the 24 assay plates, resulting in 10<sup>4</sup> cells per well,
as used for conventional infection experiments. Plates were sealed
and placed in the plate stacker that supplied the plate reader. The
same procedure was used to screen the same extracts on Mm in a broth.
For both assays, the first time points and a time point after 72 h
were recorded. Subsequently, we normalized the end point, first with
the median of the full assay plate at the first time point and second
with the end point of the vehicle controls in the respective assay
plate.</p>
<p>For IC<sub>50</sub> estimation, we used a rudimentary
approach
of interpolating the sample concentration between the two normalized
residual growth values, which were closest to a value of 0.5.</p></section><section id="sec4.12"><h3 class="pmc_sec_title">Bioactivity Screening (Pa and Sa)</h3>
<p>MICs were determined
in Mueller–Hinton (MH) broth according to CLSI guidelines and were repeated at least on three different
occasions.</p>
<p>The Newman strain of and the UCBPP-PA14 strain of were used
in this study.</p></section></section><section id="sec24"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="sifile1"><div class="media p"><div class="caption">
<a href="/articles/instance/12371487/bin/bg5c00052_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">bg5c00052_si_001.pdf</a><sup> (10.2MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>The authors
are grateful to Green Mission Pierre
Fabre and the Pierre Fabre Research Institute, Toulouse, France, for
establishing and sharing their unique libraries of plants and NEs.
BioRender was used for the conception of most figures with a publication
license provided by author T.S.. Graphs were made using GraphPad Prism
(version 9.2.0), and chemical structures were drawn with ChemDraw
(version 20.1.0.110), both licensed to O.A.K. by the University of
Geneva.</p></section><section id="notes3"><p>All data relative
to the above-mentioned collection of 1600 NEs was described in Allard
et al. (2023) at <a href="https://doi.org/10.1093/gigascience/giac124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1093/gigascience/giac124</a>. The raw NMR data of all isolated compounds presented in this study
is available through the NP-MRD platform via the NP-MRD IDs provided
for each compound in the experimental section. The raw HRMS/MS data
of all isolated compounds presented in this study is accessible through
the GNPS platform via the spectrum IDs provided for each compound
in the experimental section.</p></section><section id="notes1"><p>The Supporting
Information is
available free of charge at <a href="https://pubs.acs.org/doi/10.1021/acsbiomedchemau.5c00052?goto=supporting-info" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubs.acs.org/doi/10.1021/acsbiomedchemau.5c00052</a>.</p>
<ul id="silist" class="list" style="list-style-type:none"><li><p>Raw data associated with the isolated compounds, chromatographic
separations, process for the purity estimation, chromatograms of the extract, description of the NMR-based
purity assessment method, purity and bioactivity tables for isolated
NPs, and all the raw HRMS and NMR data for all isolated compounds
(NPs and synthetic) (<a href="https://pubs.acs.org/doi/suppl/10.1021/acsbiomedchemau.5c00052/suppl_file/bg5c00052_si_001.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PDF</a>)</p></li></ul></section><section id="notes4"><p>O.A.K: conceptualization,
data curation, investigation, methodology, software, validation, visualization,
writingoriginal draft, writingreview and editing.
J.N.: conceptualization, data curation, investigation, validation,
visualization, writingoriginal draft. A.L.: data curation,
investigation, validation, visualization, writingreview and
editing. L.F.N.: conceptualization, data curation, methodology, software,
supervision, writingreview and editing. L.M.: data curation,
investigation, writingreview and editing. N.H.: data curation,
methodology, supervision, writingreview and editing. F.M.:
data curation, investigation, methodology, software. A.G.: resources,
writingreview and editing. T.K.: supervision, writingreview
and editing. E.F.Q.: investigation, methodology, supervision, writingreview
and editing. T.S.: conceptualization, funding acquisition, project
administration, supervision, writingreview and editing. J.-L.W:
conceptualization, funding acquisition, project administration, supervision,
writingoriginal draft, writingreview and editing.
CRediT: <strong>Olivier Auguste Kirchhoffer</strong> conceptualization,
data curation, investigation, methodology, software, validation, visualization,
writing - original draft, writing - review &amp; editing; <strong>Jahn
Nitschke</strong> conceptualization, data curation, investigation, validation,
visualization, writing - original draft; <strong>Alexandre Lüscher</strong> data curation, investigation, validation, visualization, writing
- review &amp; editing; <strong>Louis-Félix Nothias</strong> conceptualization,
data curation, methodology, software, supervision, writing - review
&amp; editing; <strong>Laurence Marcourt</strong> data curation, investigation,
writing - review &amp; editing; <strong>Nabil Hanna</strong> data curation,
methodology, supervision, writing - review &amp; editing; <strong>Antonio
Grondin</strong> resources, writing - review &amp; editing; <strong>Thilo
Koehler</strong> supervision, writing - review &amp; editing; <strong>Emerson Ferreira Queiroz</strong> conceptualization, investigation,
methodology, supervision, writing - review &amp; editing; <strong>Thierry
Soldati</strong> conceptualization, funding acquisition, project administration,
supervision, writing - review &amp; editing; <strong>Jean-Luc Wolfender</strong> conceptualization, funding acquisition, project administration,
supervision, writing - original draft, writing - review &amp; editing.</p></section><section id="notes-1"><p>The authors
declare financial support was received for the research, authorship,
and/or publication of this article. The authors are thankful to the
Swiss National Science Foundation (SNSF) for the financial support
of this project (no. CRSII5_189921/1). J.-L.W. is also thankful to
the SNSF for the support in the acquisition of the NMR 600 MHz (SNF
Research Equipment grant 316030_164095).</p></section><section id="notes2"><p>The authors
declare no competing financial interest.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="ref1">
<cite>Aminov R. I.. A Brief
History of the Antibiotic Era: Lessons Learned and Challenges for
the Future. Front. Microbiol. 2010;1:134. doi: 10.3389/fmicb.2010.00134.</cite> [<a href="https://doi.org/10.3389/fmicb.2010.00134" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3109405/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21687759/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=A%20Brief%20History%20of%20the%20Antibiotic%20Era:%20Lessons%20Learned%20and%20Challenges%20for%20the%20Future&amp;author=R.%20I.%20Aminov&amp;volume=1&amp;publication_year=2010&amp;pages=134&amp;pmid=21687759&amp;doi=10.3389/fmicb.2010.00134&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref2">
<cite>Banin E., Hughes D., Kuipers O. P.. Editorial: Bacterial
Pathogens, Antibiotics
and Antibiotic Resistance. FEMS Microbiol. Rev. 2017;41(3):450–452. doi: 10.1093/femsre/fux016.</cite> [<a href="https://doi.org/10.1093/femsre/fux016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28486583/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEMS%20Microbiol.%20Rev.&amp;title=Editorial:%20Bacterial%20Pathogens,%20Antibiotics%20and%20Antibiotic%20Resistance&amp;author=E.%20Banin&amp;author=D.%20Hughes&amp;author=O.%20P.%20Kuipers&amp;volume=41&amp;issue=3&amp;publication_year=2017&amp;pages=450-452&amp;pmid=28486583&amp;doi=10.1093/femsre/fux016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref3"><cite>

O’Neill, J.


<em>Tackling Drug-Resistant Infections Globally:
Final Report and Recommendations</em>; Review on Antimicrobial
Resistance: United Kingdom. 2016. <a href="https://amr-review.org/sites/default/files/160525_Finalpaper_withcover.pdf" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://amr-review.org/sites/default/files/160525_Finalpaper_withcover.pdf</a> (accessed Apr 25, 2024).</cite></li>
<li id="ref4">
<cite>
Geneva: World Health Organisation . Global Tuberculosis Report 2022; Geneva: World Health Organisation, 2022.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Global%20Tuberculosis%20Report%202022&amp;publication_year=2022&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref5">
<cite>Newman D. J., Cragg G. M.. Natural Products
as Sources of New Drugs over the Nearly
Four Decades from 01/1981 to 09/2019. J. Nat.
Prod. 2020;83(3):770–803. doi: 10.1021/acs.jnatprod.9b01285.</cite> [<a href="https://doi.org/10.1021/acs.jnatprod.9b01285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32162523/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Nat.%20Prod.&amp;title=Natural%20Products%20as%20Sources%20of%20New%20Drugs%20over%20the%20Nearly%20Four%20Decades%20from%2001/1981%20to%2009/2019&amp;author=D.%20J.%20Newman&amp;author=G.%20M.%20Cragg&amp;volume=83&amp;issue=3&amp;publication_year=2020&amp;pages=770-803&amp;pmid=32162523&amp;doi=10.1021/acs.jnatprod.9b01285&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref6">
<cite>Gonzalez-Pastor R., Carrera-Pacheco S. E., Zúñiga-Miranda J., Rodríguez-Pólit C., Mayorga-Ramos A., Guamán L. P., Barba-Ostria C.. Current Landscape
of Methods to Evaluate
Antimicrobial Activity of Natural Extracts. Molecules. 2023;28(3):1068. doi: 10.3390/molecules28031068.</cite> [<a href="https://doi.org/10.3390/molecules28031068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9920787/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36770734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Current%20Landscape%20of%20Methods%20to%20Evaluate%20Antimicrobial%20Activity%20of%20Natural%20Extracts&amp;author=R.%20Gonzalez-Pastor&amp;author=S.%20E.%20Carrera-Pacheco&amp;author=J.%20Z%C3%BA%C3%B1iga-Miranda&amp;author=C.%20Rodr%C3%ADguez-P%C3%B3lit&amp;author=A.%20Mayorga-Ramos&amp;volume=28&amp;issue=3&amp;publication_year=2023&amp;pages=1068&amp;pmid=36770734&amp;doi=10.3390/molecules28031068&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref7">
<cite>Xie Y., Feng Y., Di Capua A., Mak T., Buchko G. W., Myler P. J., Liu M., Quinn R. J.. A Phenotarget
Approach
for Identifying an Alkaloid Interacting with the Tuberculosis Protein
Rv1466. Mar. Drugs. 2020;18(3):149. doi: 10.3390/md18030149.</cite> [<a href="https://doi.org/10.3390/md18030149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7143284/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32150903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mar.%20Drugs&amp;title=A%20Phenotarget%20Approach%20for%20Identifying%20an%20Alkaloid%20Interacting%20with%20the%20Tuberculosis%20Protein%20Rv1466&amp;author=Y.%20Xie&amp;author=Y.%20Feng&amp;author=A.%20Di%20Capua&amp;author=T.%20Mak&amp;author=G.%20W.%20Buchko&amp;volume=18&amp;issue=3&amp;publication_year=2020&amp;pages=149&amp;pmid=32150903&amp;doi=10.3390/md18030149&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref8">
<cite>Zheng W., Thorne N., McKew J. C.. Phenotypic Screens as a Renewed Approach
for Drug Discovery. Drug Discovery Today. 2013;18(21):1067–1073. doi: 10.1016/j.drudis.2013.07.001.</cite> [<a href="https://doi.org/10.1016/j.drudis.2013.07.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4531371/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23850704/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Discovery%20Today&amp;title=Phenotypic%20Screens%20as%20a%20Renewed%20Approach%20for%20Drug%20Discovery&amp;author=W.%20Zheng&amp;author=N.%20Thorne&amp;author=J.%20C.%20McKew&amp;volume=18&amp;issue=21&amp;publication_year=2013&amp;pages=1067-1073&amp;pmid=23850704&amp;doi=10.1016/j.drudis.2013.07.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref9">
<cite>Payne D. J., Gwynn M. N., Holmes D. J., Pompliano D. L.. Drugs for
Bad Bugs: Confronting the Challenges of Antibacterial Discovery. Nat. Rev. Drug Discovery. 2007;6(1):29–40. doi: 10.1038/nrd2201.</cite> [<a href="https://doi.org/10.1038/nrd2201" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17159923/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Drug%20Discovery&amp;title=Drugs%20for%20Bad%20Bugs:%20Confronting%20the%20Challenges%20of%20Antibacterial%20Discovery&amp;author=D.%20J.%20Payne&amp;author=M.%20N.%20Gwynn&amp;author=D.%20J.%20Holmes&amp;author=D.%20L.%20Pompliano&amp;volume=6&amp;issue=1&amp;publication_year=2007&amp;pages=29-40&amp;pmid=17159923&amp;doi=10.1038/nrd2201&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref10">
<cite>Brodin P., Poquet Y., Levillain F., Peguillet I., Larrouy-Maumus G., Gilleron M., Ewann F., Christophe T., Fenistein D., Jang J., Jang M.-S., Park S.-J., Rauzier J., Carralot J.-P., Shrimpton R., Genovesio A., Gonzalo-Asensio J. A., Puzo G., Martin C., Brosch R., Stewart G. R., Gicquel B., Neyrolles O.. High Content
Phenotypic Cell-Based Visual Screen Identifies Mycobacterium Tuberculosis
Acyltrehalose-Containing Glycolipids Involved in Phagosome Remodeling. Public Libr. Sci. Pathog. 2010;6(9):e1001100.  doi: 10.1371/journal.ppat.1001100.</cite> [<a href="https://doi.org/10.1371/journal.ppat.1001100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2936551/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20844580/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Public%20Libr.%20Sci.%20Pathog.&amp;title=High%20Content%20Phenotypic%20Cell-Based%20Visual%20Screen%20Identifies%20Mycobacterium%20Tuberculosis%20Acyltrehalose-Containing%20Glycolipids%20Involved%20in%20Phagosome%20Remodeling&amp;author=P.%20Brodin&amp;author=Y.%20Poquet&amp;author=F.%20Levillain&amp;author=I.%20Peguillet&amp;author=G.%20Larrouy-Maumus&amp;volume=6&amp;issue=9&amp;publication_year=2010&amp;pages=e1001100&amp;pmid=20844580&amp;doi=10.1371/journal.ppat.1001100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref11">
<cite>Kalsum S., Otrocka M., Andersson B., Welin A., Schön T., Jenmalm-Jensen A., Lundbäck T., Lerm M.. A High Content Screening
Assay for Discovery of Antimycobacterial Compounds Based on Primary
Human Macrophages Infected with Virulent Mycobacterium Tuberculosis. Tuberculosis. 2022;135:102222. doi: 10.1016/j.tube.2022.102222.</cite> [<a href="https://doi.org/10.1016/j.tube.2022.102222" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35738191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Tuberculosis&amp;title=A%20High%20Content%20Screening%20Assay%20for%20Discovery%20of%20Antimycobacterial%20Compounds%20Based%20on%20Primary%20Human%20Macrophages%20Infected%20with%20Virulent%20Mycobacterium%20Tuberculosis&amp;author=S.%20Kalsum&amp;author=M.%20Otrocka&amp;author=B.%20Andersson&amp;author=A.%20Welin&amp;author=T.%20Sch%C3%B6n&amp;volume=135&amp;publication_year=2022&amp;pages=102222&amp;pmid=35738191&amp;doi=10.1016/j.tube.2022.102222&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref12">
<cite>Tobin D. M., Ramakrishnan L.. Comparative
Pathogenesis of Mycobacterium Marinum and Mycobacterium Tuberculosis
. Cell.
Microbiol. 2008;10(5):1027–1039. doi: 10.1111/j.1462-5822.2008.01133.x.</cite> [<a href="https://doi.org/10.1111/j.1462-5822.2008.01133.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18298637/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell.%20Microbiol.&amp;title=Comparative%20Pathogenesis%20of%20Mycobacterium%20Marinum%20and%20Mycobacterium%20Tuberculosis&amp;author=D.%20M.%20Tobin&amp;author=L.%20Ramakrishnan&amp;volume=10&amp;issue=5&amp;publication_year=2008&amp;pages=1027-1039&amp;pmid=18298637&amp;doi=10.1111/j.1462-5822.2008.01133.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref13">
<cite>Habjan E., Ho V. Q. T., Gallant J., van Stempvoort G., Jim K. K., Kuijl C., Geerke D. P., Bitter W., Speer A.. An Anti-Tuberculosis Compound Screen Using a Zebrafish Infection
Model Identifies an Aspartyl-tRNA Synthetase Inhibitor. Dis. Model. Mech. 2021;14(12):dmm049145. doi: 10.1242/dmm.049145.</cite> [<a href="https://doi.org/10.1242/dmm.049145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8713996/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34643222/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dis.%20Model.%20Mech.&amp;title=An%20Anti-Tuberculosis%20Compound%20Screen%20Using%20a%20Zebrafish%20Infection%20Model%20Identifies%20an%20Aspartyl-tRNA%20Synthetase%20Inhibitor&amp;author=E.%20Habjan&amp;author=V.%20Q.%20T.%20Ho&amp;author=J.%20Gallant&amp;author=G.%20van%20Stempvoort&amp;author=K.%20K.%20Jim&amp;volume=14&amp;issue=12&amp;publication_year=2021&amp;pages=dmm049145&amp;pmid=34643222&amp;doi=10.1242/dmm.049145&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref14">
<cite>Nitschke J., Huber R., Vossio S., Moreau D., Marcourt L., Gindro K., Queiroz E. F., Soldati T., Hanna N.. Discovery
of Anti-Infective Compounds against Mycobacterium Marinum after Biotransformation
of Simple Natural Stilbene Scaffolds by a Fungal Secretome. bioRxiv. 2024 doi: 10.1101/2024.04.18.590143.</cite> [<a href="https://doi.org/10.1101/2024.04.18.590143" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11443511/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39355425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Discovery%20of%20Anti-Infective%20Compounds%20against%20Mycobacterium%20Marinum%20after%20Biotransformation%20of%20Simple%20Natural%20Stilbene%20Scaffolds%20by%20a%20Fungal%20Secretome&amp;author=J.%20Nitschke&amp;author=R.%20Huber&amp;author=S.%20Vossio&amp;author=D.%20Moreau&amp;author=L.%20Marcourt&amp;publication_year=2024&amp;pmid=39355425&amp;doi=10.1101/2024.04.18.590143&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref15">
<cite>Allard P.-M., Gaudry A., Quirós-Guerrero L.-M., Rutz A., Dounoue-Kubo M., Walker T. W. N., Defossez E., Long C., Grondin A., David B., Wolfender J.-L.. Open and
Reusable Annotated Mass Spectrometry Dataset of a Chemodiverse Collection
of 1,600 Plant Extracts. GigaScience. 2022;12:giac124. doi: 10.1093/gigascience/giac124.</cite> [<a href="https://doi.org/10.1093/gigascience/giac124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9845059/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36649739/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GigaScience&amp;title=Open%20and%20Reusable%20Annotated%20Mass%20Spectrometry%20Dataset%20of%20a%20Chemodiverse%20Collection%20of%201,600%20Plant%20Extracts&amp;author=P.-M.%20Allard&amp;author=A.%20Gaudry&amp;author=L.-M.%20Quir%C3%B3s-Guerrero&amp;author=A.%20Rutz&amp;author=M.%20Dounoue-Kubo&amp;volume=12&amp;publication_year=2022&amp;pages=giac124&amp;pmid=36649739&amp;doi=10.1093/gigascience/giac124&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref16">
<cite>

Nitschke, J.

The Dictyostelium Discoideum
- Mycobacterium Marinum Platform in Natural Product Drug Discovery; Université de Genève, 2024; . 10.13097/archive-ouverte/unige:179581.</cite> [<a href="https://doi.org/10.13097/archive-ouverte/unige:179581" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=The%20Dictyostelium%20Discoideum%20-%20Mycobacterium%20Marinum%20Platform%20in%20Natural%20Product%20Drug%20Discovery&amp;publication_year=2024&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref17">
<cite>Duthie E. S., Lorenz L. L.. Staphylococcal Coagulase:
Mode of Action and Antigenicity. Microbiology. 1952;6:95–107. doi: 10.1099/00221287-6-1-2-95.</cite> [<a href="https://doi.org/10.1099/00221287-6-1-2-95" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14927856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiology&amp;title=Staphylococcal%20Coagulase:%20Mode%20of%20Action%20and%20Antigenicity&amp;author=E.%20S.%20Duthie&amp;author=L.%20L.%20Lorenz&amp;volume=6&amp;publication_year=1952&amp;pages=95-107&amp;pmid=14927856&amp;doi=10.1099/00221287-6-1-2-95&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref18">
<cite>Rahme L. G., Stevens E. J., Wolfort S. F., Shao J., Tompkins R. G., Ausubel F. M.. Common Virulence
Factors for Bacterial Pathogenicity
in Plants and Animals. Science. 1995;268(5219):1899–1902. doi: 10.1126/science.7604262.</cite> [<a href="https://doi.org/10.1126/science.7604262" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7604262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Common%20Virulence%20Factors%20for%20Bacterial%20Pathogenicity%20in%20Plants%20and%20Animals&amp;author=L.%20G.%20Rahme&amp;author=E.%20J.%20Stevens&amp;author=S.%20F.%20Wolfort&amp;author=J.%20Shao&amp;author=R.%20G.%20Tompkins&amp;volume=268&amp;issue=5219&amp;publication_year=1995&amp;pages=1899-1902&amp;pmid=7604262&amp;doi=10.1126/science.7604262&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref19">
<cite>Grace A., Sahu R., Owen D. R., Dennis V. A.. 
Pseudomonas
Aeruginosa Reference Strains PAO1 and PA14: A Genomic, Phenotypic,
and Therapeutic Review. Front. Microbiol. 2022;13:1023523. doi: 10.3389/fmicb.2022.1023523.</cite> [<a href="https://doi.org/10.3389/fmicb.2022.1023523" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9607943/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36312971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Pseudomonas%20Aeruginosa%20Reference%20Strains%20PAO1%20and%20PA14:%20A%20Genomic,%20Phenotypic,%20and%20Therapeutic%20Review&amp;author=A.%20Grace&amp;author=R.%20Sahu&amp;author=D.%20R.%20Owen&amp;author=V.%20A.%20Dennis&amp;volume=13&amp;publication_year=2022&amp;pages=1023523&amp;pmid=36312971&amp;doi=10.3389/fmicb.2022.1023523&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref20">
<cite>Kirchhoffer O. A., Nitschke J., Allard P.-M., Marcourt L., David B., Grondin A., Hanna N., Queiroz E. F., Soldati T., Wolfender J.-L.. Targeted Isolation of Natural Analogs
of Anti-Mycobacterial
Hit Compounds Based on the Metabolite Profiling of a Large Collection
of Plant Extracts. Front. Nat. Prod. 2023;2:1279761. doi: 10.3389/fntpr.2023.1279761.</cite> [<a href="https://doi.org/10.3389/fntpr.2023.1279761" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Nat.%20Prod&amp;title=Targeted%20Isolation%20of%20Natural%20Analogs%20of%20Anti-Mycobacterial%20Hit%20Compounds%20Based%20on%20the%20Metabolite%20Profiling%20of%20a%20Large%20Collection%20of%20Plant%20Extracts&amp;author=O.%20A.%20Kirchhoffer&amp;author=J.%20Nitschke&amp;author=P.-M.%20Allard&amp;author=L.%20Marcourt&amp;author=B.%20David&amp;volume=2&amp;publication_year=2023&amp;pages=1279761&amp;doi=10.3389/fntpr.2023.1279761&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref21">
<cite>Hamburger M.. HPLC-Based
Activity Profiling for Pharmacologically and Toxicologically Relevant
Natural Products – Principles and Recent Examples. Pharm. Biol. 2019;57(1):328–334. doi: 10.1080/13880209.2019.1606261.</cite> [<a href="https://doi.org/10.1080/13880209.2019.1606261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6507960/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31057026/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharm.%20Biol.&amp;title=HPLC-Based%20Activity%20Profiling%20for%20Pharmacologically%20and%20Toxicologically%20Relevant%20Natural%20Products%20%E2%80%93%20Principles%20and%20Recent%20Examples&amp;author=M.%20Hamburger&amp;volume=57&amp;issue=1&amp;publication_year=2019&amp;pages=328-334&amp;pmid=31057026&amp;doi=10.1080/13880209.2019.1606261&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref22">
<cite>Dührkop K., Fleischauer M., Ludwig M., Aksenov A. A., Melnik A. V., Meusel M., Dorrestein P. C., Rousu J., Böcker S.. SIRIUS 4:
A Rapid Tool for Turning Tandem Mass Spectra into Metabolite Structure
Information. Nat. Methods. 2019;16(4):299–302. doi: 10.1038/s41592-019-0344-8.</cite> [<a href="https://doi.org/10.1038/s41592-019-0344-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30886413/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Methods&amp;title=SIRIUS%204:%20A%20Rapid%20Tool%20for%20Turning%20Tandem%20Mass%20Spectra%20into%20Metabolite%20Structure%20Information&amp;author=K.%20D%C3%BChrkop&amp;author=M.%20Fleischauer&amp;author=M.%20Ludwig&amp;author=A.%20A.%20Aksenov&amp;author=A.%20V.%20Melnik&amp;volume=16&amp;issue=4&amp;publication_year=2019&amp;pages=299-302&amp;pmid=30886413&amp;doi=10.1038/s41592-019-0344-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref23">
<cite>Dührkop K., Shen H., Meusel M., Rousu J., Böcker S.. Searching
Molecular Structure Databases with Tandem Mass Spectra Using CSI:FingerID. Proc. Natl. Acad. Sci. U.S.A. 2015;112(41):12580–12585. doi: 10.1073/pnas.1509788112.</cite> [<a href="https://doi.org/10.1073/pnas.1509788112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4611636/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26392543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A.&amp;title=Searching%20Molecular%20Structure%20Databases%20with%20Tandem%20Mass%20Spectra%20Using%20CSI:FingerID&amp;author=K.%20D%C3%BChrkop&amp;author=H.%20Shen&amp;author=M.%20Meusel&amp;author=J.%20Rousu&amp;author=S.%20B%C3%B6cker&amp;volume=112&amp;issue=41&amp;publication_year=2015&amp;pages=12580-12585&amp;pmid=26392543&amp;doi=10.1073/pnas.1509788112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref24">
<cite>Sriphana U., Yenjai C., Koatthada M.. Cytotoxicity
of Chemical Constituents
from the Roots of Knema Globularia
. Phytochem. Lett. 2016;16:129–133. doi: 10.1016/j.phytol.2016.03.010.</cite> [<a href="https://doi.org/10.1016/j.phytol.2016.03.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phytochem.%20Lett.&amp;title=Cytotoxicity%20of%20Chemical%20Constituents%20from%20the%20Roots%20of%20Knema%20Globularia&amp;author=U.%20Sriphana&amp;author=C.%20Yenjai&amp;author=M.%20Koatthada&amp;volume=16&amp;publication_year=2016&amp;pages=129-133&amp;doi=10.1016/j.phytol.2016.03.010&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref25">
<cite>Gény C., Rivière G., Bignon J., Birlirakis N., Guittet E., Awang K., Litaudon M., Roussi F., Dumontet V.. Anacardic Acids from Knema Hookeriana as Modulators of Bcl-xL/Bak and Mcl-1/Bid
Interactions. J. Nat. Prod. 2016;79(4):838–844. doi: 10.1021/acs.jnatprod.5b00915.</cite> [<a href="https://doi.org/10.1021/acs.jnatprod.5b00915" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27008174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Nat.%20Prod.&amp;title=Anacardic%20Acids%20from%20Knema%20Hookeriana%20as%20Modulators%20of%20Bcl-xL/Bak%20and%20Mcl-1/Bid%20Interactions&amp;author=C.%20G%C3%A9ny&amp;author=G.%20Rivi%C3%A8re&amp;author=J.%20Bignon&amp;author=N.%20Birlirakis&amp;author=E.%20Guittet&amp;volume=79&amp;issue=4&amp;publication_year=2016&amp;pages=838-844&amp;pmid=27008174&amp;doi=10.1021/acs.jnatprod.5b00915&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref26">
<cite>Guillarme D., Nguyen D. T. T., Rudaz S., Veuthey J.-L.. Method Transfer
for Fast Liquid Chromatography in Pharmaceutical Analysis: Application
to Short Columns Packed with Small Particle. Part II: Gradient Experiments. Eur. J. Pharm. Biopharm. 2008;68(2):430–440. doi: 10.1016/j.ejpb.2007.06.018.</cite> [<a href="https://doi.org/10.1016/j.ejpb.2007.06.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17703929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Pharm.%20Biopharm.&amp;title=Method%20Transfer%20for%20Fast%20Liquid%20Chromatography%20in%20Pharmaceutical%20Analysis:%20Application%20to%20Short%20Columns%20Packed%20with%20Small%20Particle.%20Part%20II:%20Gradient%20Experiments&amp;author=D.%20Guillarme&amp;author=D.%20T.%20T.%20Nguyen&amp;author=S.%20Rudaz&amp;author=J.-L.%20Veuthey&amp;volume=68&amp;issue=2&amp;publication_year=2008&amp;pages=430-440&amp;pmid=17703929&amp;doi=10.1016/j.ejpb.2007.06.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref27">
<cite>

Sharma, N. K.
; 
Sharma, V. N.
. 
Structure of
Anagigantic Acid Isolated from <em>Anacardium Giganteum</em>
. Indian J. Chem.
1966, 4, 504.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Indian%20J.%20Chem.&amp;title=Structure%20of%20Anagigantic%20Acid%20Isolated%20from%20Anacardium%20Giganteum&amp;volume=4&amp;publication_year=1966&amp;pages=504&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref28">
<cite>Kazlauskas R., Mulder J., Murphy P., Wells R.. New Metabolites from
the Brown Alga Caulocystis Cephalornithos. Aust.
J. Chem. 1980;33(9):2097–2101. doi: 10.1071/CH9802097.</cite> [<a href="https://doi.org/10.1071/CH9802097" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aust.%20J.%20Chem.&amp;title=New%20Metabolites%20from%20the%20Brown%20Alga%20Caulocystis%20Cephalornithos&amp;author=R.%20Kazlauskas&amp;author=J.%20Mulder&amp;author=P.%20Murphy&amp;author=R.%20Wells&amp;volume=33&amp;issue=9&amp;publication_year=1980&amp;pages=2097-2101&amp;doi=10.1071/CH9802097&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref29">
<cite>Hemshekhar M., Sebastin
Santhosh M., Kemparaju K., Girish K. S.. Emerging Roles of
Anacardic Acid and Its Derivatives: A Pharmacological Overview. Basic Clin. Pharmacol. Toxicol. 2012;110(2):122–132. doi: 10.1111/j.1742-7843.2011.00833.x.</cite> [<a href="https://doi.org/10.1111/j.1742-7843.2011.00833.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22103711/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Basic%20Clin.%20Pharmacol.%20Toxicol.&amp;title=Emerging%20Roles%20of%20Anacardic%20Acid%20and%20Its%20Derivatives:%20A%20Pharmacological%20Overview&amp;author=M.%20Hemshekhar&amp;author=M.%20Sebastin%20Santhosh&amp;author=K.%20Kemparaju&amp;author=K.%20S.%20Girish&amp;volume=110&amp;issue=2&amp;publication_year=2012&amp;pages=122-132&amp;pmid=22103711&amp;doi=10.1111/j.1742-7843.2011.00833.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref30">
<cite>Muroi H., Kubo I.. Antibacterial Activity
of Anacardic Acid and Totarol, Alone and in
Combination with Methicillin, against Methicillinresistant Staphylococcus Aureus
. J. Appl. Bacteriol. 1996;80(4):387–394. doi: 10.1111/j.1365-2672.1996.tb03233.x.</cite> [<a href="https://doi.org/10.1111/j.1365-2672.1996.tb03233.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8849640/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Appl.%20Bacteriol.&amp;title=Antibacterial%20Activity%20of%20Anacardic%20Acid%20and%20Totarol,%20Alone%20and%20in%20Combination%20with%20Methicillin,%20against%20Methicillinresistant%20Staphylococcus%20Aureus&amp;author=H.%20Muroi&amp;author=I.%20Kubo&amp;volume=80&amp;issue=4&amp;publication_year=1996&amp;pages=387-394&amp;pmid=8849640&amp;doi=10.1111/j.1365-2672.1996.tb03233.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref31">
<cite>Faleye O. S., Boya B. R., Lee J.-H., Choi I., Lee J.. Halogenated
Antimicrobial Agents to Combat Drug-Resistant Pathogens. Pharmacol. Rev. 2024;76(1):90–141. doi: 10.1124/pharmrev.123.000863.</cite> [<a href="https://doi.org/10.1124/pharmrev.123.000863" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37845080/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Rev.&amp;title=Halogenated%20Antimicrobial%20Agents%20to%20Combat%20Drug-Resistant%20Pathogens&amp;author=O.%20S.%20Faleye&amp;author=B.%20R.%20Boya&amp;author=J.-H.%20Lee&amp;author=I.%20Choi&amp;author=J.%20Lee&amp;volume=76&amp;issue=1&amp;publication_year=2024&amp;pages=90-141&amp;pmid=37845080&amp;doi=10.1124/pharmrev.123.000863&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref32">
<cite>Neuenschwander A., Rocha V. P. C., Bastos T. M., Marcourt L., Morin H., da Rocha C. Q., Grimaldi G. B., de Sousa K. A. F., Borges J. N., Rivara-Minten E., Wolfender J.-L., Soares M. B. P., Queiroz E. F.. Production
of Highly Active Antiparasitic Compounds from the Controlled Halogenation
of the Crude
Plant Extract. J. Nat. Prod. 2020;83(9):2631–2640. doi: 10.1021/acs.jnatprod.0c00433.</cite> [<a href="https://doi.org/10.1021/acs.jnatprod.0c00433" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32902988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Nat.%20Prod.&amp;title=Production%20of%20Highly%20Active%20Antiparasitic%20Compounds%20from%20the%20Controlled%20Halogenation%20of%20the%20Crude%20Plant%20Extract&amp;author=A.%20Neuenschwander&amp;author=V.%20P.%20C.%20Rocha&amp;author=T.%20M.%20Bastos&amp;author=L.%20Marcourt&amp;author=H.%20Morin&amp;volume=83&amp;issue=9&amp;publication_year=2020&amp;pages=2631-2640&amp;pmid=32902988&amp;doi=10.1021/acs.jnatprod.0c00433&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref33">
<cite>Huber R., Marcourt L., Héritier M., Luscher A., Guebey L., Schnee S., Michellod E., Guerrier S., Wolfender J.-L., Scapozza L., Köhler T., Gindro K., Queiroz E. F.. Generation
of Potent Antibacterial Compounds through Enzymatic and Chemical Modifications
of the Trans-δ-Viniferin Scaffold. Sci.
Rep. 2023;13(1):15986. doi: 10.1038/s41598-023-43000-5.</cite> [<a href="https://doi.org/10.1038/s41598-023-43000-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10520035/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37749179/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=Generation%20of%20Potent%20Antibacterial%20Compounds%20through%20Enzymatic%20and%20Chemical%20Modifications%20of%20the%20Trans-%CE%B4-Viniferin%20Scaffold&amp;author=R.%20Huber&amp;author=L.%20Marcourt&amp;author=M.%20H%C3%A9ritier&amp;author=A.%20Luscher&amp;author=L.%20Guebey&amp;volume=13&amp;issue=1&amp;publication_year=2023&amp;pages=15986&amp;pmid=37749179&amp;doi=10.1038/s41598-023-43000-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref34">
<cite>Wilcken R., Zimmermann M. O., Lange A., Joerger A. C., Boeckler F. M.. Principles
and Applications of Halogen Bonding in Medicinal Chemistry and Chemical
Biology. J. Med. Chem. 2013;56(4):1363–1388. doi: 10.1021/jm3012068.</cite> [<a href="https://doi.org/10.1021/jm3012068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23145854/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&amp;title=Principles%20and%20Applications%20of%20Halogen%20Bonding%20in%20Medicinal%20Chemistry%20and%20Chemical%20Biology&amp;author=R.%20Wilcken&amp;author=M.%20O.%20Zimmermann&amp;author=A.%20Lange&amp;author=A.%20C.%20Joerger&amp;author=F.%20M.%20Boeckler&amp;volume=56&amp;issue=4&amp;publication_year=2013&amp;pages=1363-1388&amp;pmid=23145854&amp;doi=10.1021/jm3012068&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref35">
<cite>Wu Y.-Q., Lu H.-J., Zhao W.-T., Zhao H.-Y., Lin Z.-Y., Zhang D.-F., Huang H.-H.. A Convenient and Efficient H2SO4-Promoted
Regioselective Monobromination of Phenol Derivatives Using N-Bromosuccinimide. Synth. Commun. 2020;50(6):813–822. doi: 10.1080/00397911.2019.1711415.</cite> [<a href="https://doi.org/10.1080/00397911.2019.1711415" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Synth.%20Commun.&amp;title=A%20Convenient%20and%20Efficient%20H2SO4-Promoted%20Regioselective%20Monobromination%20of%20Phenol%20Derivatives%20Using%20N-Bromosuccinimide&amp;author=Y.-Q.%20Wu&amp;author=H.-J.%20Lu&amp;author=W.-T.%20Zhao&amp;author=H.-Y.%20Zhao&amp;author=Z.-Y.%20Lin&amp;volume=50&amp;issue=6&amp;publication_year=2020&amp;pages=813-822&amp;doi=10.1080/00397911.2019.1711415&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref36">
<cite>Devakaram R., Black D. StC., Andrews K. T., Fisher G. M., Davis R. A., Kumar N.. Synthesis and Antimalarial Evaluation
of Novel Benzopyrano­[4,3-b]­Benzopyran
Derivatives. Bioorg. Med. Chem. 2011;19(17):5199–5206. doi: 10.1016/j.bmc.2011.07.009.</cite> [<a href="https://doi.org/10.1016/j.bmc.2011.07.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21824783/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Med.%20Chem.&amp;title=Synthesis%20and%20Antimalarial%20Evaluation%20of%20Novel%20Benzopyrano%C2%AD%5B4,3-b%5D%C2%ADBenzopyran%20Derivatives&amp;author=R.%20Devakaram&amp;author=D.%20StC.%20Black&amp;author=K.%20T.%20Andrews&amp;author=G.%20M.%20Fisher&amp;author=R.%20A.%20Davis&amp;volume=19&amp;issue=17&amp;publication_year=2011&amp;pages=5199-5206&amp;pmid=21824783&amp;doi=10.1016/j.bmc.2011.07.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref37">
<cite>Pérez P., Domingo L. R., Duque-Noreña M., Chamorro E.. A Condensed-to-Atom
Nucleophilicity Index. An Application to the Director Effects on the
Electrophilic Aromatic Substitutions. J. Mol.
Struct. THEOCHEM. 2009;895(1):86–91. doi: 10.1016/j.theochem.2008.10.014.</cite> [<a href="https://doi.org/10.1016/j.theochem.2008.10.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Struct.%20THEOCHEM&amp;title=A%20Condensed-to-Atom%20Nucleophilicity%20Index.%20An%20Application%20to%20the%20Director%20Effects%20on%20the%20Electrophilic%20Aromatic%20Substitutions&amp;author=P.%20P%C3%A9rez&amp;author=L.%20R.%20Domingo&amp;author=M.%20Duque-Nore%C3%B1a&amp;author=E.%20Chamorro&amp;volume=895&amp;issue=1&amp;publication_year=2009&amp;pages=86-91&amp;doi=10.1016/j.theochem.2008.10.014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref38">
<cite>Saedtler M., Förtig N., Ohlsen K., Faber F., Masota N., Kowalick K., Holzgrabe U., Meinel L.. Antibacterial Anacardic
Acid Derivatives. ACS Infect. Dis. 2020;6(7):1674–1685. doi: 10.1021/acsinfecdis.9b00378.</cite> [<a href="https://doi.org/10.1021/acsinfecdis.9b00378" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32519844/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Infect.%20Dis.&amp;title=Antibacterial%20Anacardic%20Acid%20Derivatives&amp;author=M.%20Saedtler&amp;author=N.%20F%C3%B6rtig&amp;author=K.%20Ohlsen&amp;author=F.%20Faber&amp;author=N.%20Masota&amp;volume=6&amp;issue=7&amp;publication_year=2020&amp;pages=1674-1685&amp;pmid=32519844&amp;doi=10.1021/acsinfecdis.9b00378&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref39">
<cite>Hamad F. B., Mubofu E. B.. Potential Biological
Applications of Bio-Based Anacardic
Acids and Their Derivatives. Int. J. Mol. Sci. 2015;16(4):8569–8590. doi: 10.3390/ijms16048569.</cite> [<a href="https://doi.org/10.3390/ijms16048569" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4425097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25894225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Potential%20Biological%20Applications%20of%20Bio-Based%20Anacardic%20Acids%20and%20Their%20Derivatives&amp;author=F.%20B.%20Hamad&amp;author=E.%20B.%20Mubofu&amp;volume=16&amp;issue=4&amp;publication_year=2015&amp;pages=8569-8590&amp;pmid=25894225&amp;doi=10.3390/ijms16048569&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref40">
<cite>Nitschke J., Huber R., Vossio S., Moreau D., Marcourt L., Gindro K., Queiroz E. F., Soldati T., Hanna N.. Discovery
of Anti-Infective Compounds against Mycobacterium Marinum after Biotransformation of Simple Natural Stilbenes by a Fungal
Secretome. Front. Microbiol. 2024;15:1439814. doi: 10.3389/fmicb.2024.1439814.</cite> [<a href="https://doi.org/10.3389/fmicb.2024.1439814" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11443511/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39355425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front.%20Microbiol.&amp;title=Discovery%20of%20Anti-Infective%20Compounds%20against%20Mycobacterium%20Marinum%20after%20Biotransformation%20of%20Simple%20Natural%20Stilbenes%20by%20a%20Fungal%20Secretome&amp;author=J.%20Nitschke&amp;author=R.%20Huber&amp;author=S.%20Vossio&amp;author=D.%20Moreau&amp;author=L.%20Marcourt&amp;volume=15&amp;publication_year=2024&amp;pages=1439814&amp;pmid=39355425&amp;doi=10.3389/fmicb.2024.1439814&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref41">
<cite>

Mottet, M.
; 
Bosmani, C.
; 
Hanna, N.
; 
Nitschke, J.
; 
Lefrançois, L.
H.
; 
Soldati, T.
. 
Novel Single-Cell
and High-Throughput Microscopy Techniques to Monitor <em>Dictyostelium
Discoideum</em>–<em>Mycobacterium Marinum</em> (<em>M. Marinum</em>) Infection Dynamics. In Mycobacteria, Protocols. Methods in Molecular Biology; 
Parish, T.
, 
Kumar, A.
, Eds.; Springer US: New York, NY, 2021;
pp 183–203. 10.1007/978-1-0716-1460-0_7.</cite> [<a href="https://doi.org/10.1007/978-1-0716-1460-0_7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34235653/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Mycobacteria,%20Protocols.%20Methods%20in%20Molecular%20Biology&amp;publication_year=2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref42">
<cite>Paschke P., Knecht D. A., Williams T. D., Thomason P. A., Insall R. H., Chubb J. R., Kay R. R., Veltman D. M.. Genetic
Engineering
of Dictyostelium Discoideum Cells Based on Selection
and Growth on Bacteria. J. Vis. Exp. 2019;143:e58981.  doi: 10.3791/58981.</cite> [<a href="https://doi.org/10.3791/58981" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7039707/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30735174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Vis.%20Exp.&amp;title=Genetic%20Engineering%20of%20Dictyostelium%20Discoideum%20Cells%20Based%20on%20Selection%20and%20Growth%20on%20Bacteria&amp;author=P.%20Paschke&amp;author=D.%20A.%20Knecht&amp;author=T.%20D.%20Williams&amp;author=P.%20A.%20Thomason&amp;author=R.%20H.%20Insall&amp;volume=143&amp;publication_year=2019&amp;pages=e58981&amp;pmid=30735174&amp;doi=10.3791/58981&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref43">
<cite>Andreu N., Zelmer A., Fletcher T., Elkington P. T., Ward T. H., Ripoll J., Parish T., Bancroft G. J., Schaible U., Robertson B. D., Wiles S.. Optimisation of Bioluminescent
Reporters for Use with Mycobacteria. PLoS One. 2010;5(5):e10777.  doi: 10.1371/journal.pone.0010777.</cite> [<a href="https://doi.org/10.1371/journal.pone.0010777" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2875389/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20520722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Optimisation%20of%20Bioluminescent%20Reporters%20for%20Use%20with%20Mycobacteria&amp;author=N.%20Andreu&amp;author=A.%20Zelmer&amp;author=T.%20Fletcher&amp;author=P.%20T.%20Elkington&amp;author=T.%20H.%20Ward&amp;volume=5&amp;issue=5&amp;publication_year=2010&amp;pages=e10777&amp;pmid=20520722&amp;doi=10.1371/journal.pone.0010777&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref44">
<cite>

Arafah, S.
; 
Kicka, S.
; 
Trofimov, V.
; 
Hagedorn, M.
; 
Andreu, N.
; 
Wiles, S.
; 
Robertson, B.
; 
Soldati, T.
. 
Setting Up and
Monitoring an Infection
of <em>Dictyostelium Discoideum</em> with Mycobacteria. In Dictyostelium discoideum Protocols; 
Eichinger, L.
, 
Rivero, F.
, Eds.; Humana
Press: Totowa, NJ, 2013; pp 403–417. 10.1007/978-1-62703-302-2_22.</cite> [<a href="https://doi.org/10.1007/978-1-62703-302-2_22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23494320/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Dictyostelium%20discoideum%20Protocols&amp;publication_year=2013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ref45">
<cite>

Weinstein, M. P.
; 
Patel, J. B.
. 
Methods for Dilution
Antimicrobial Susceptibility Tests for Bacteria that Grow Aerobically:
M07-A11; Documents/Clinical and Laboratory
Standards Institute; Committee for Clinical Laboratory Standards: Wayne, PA, 2018.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=Methods%20for%20Dilution%20Antimicrobial%20Susceptibility%20Tests%20for%20Bacteria%20that%20Grow%20Aerobically:%20M07-A11&amp;publication_year=2018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12371487/bin/bg5c00052_si_001.pdf" data-ga-action="click_feat_suppl" class="usa-link">bg5c00052_si_001.pdf</a><sup> (10.2MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data relative
to the above-mentioned collection of 1600 NEs was described in Allard
et al. (2023) at <a href="https://doi.org/10.1093/gigascience/giac124" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1093/gigascience/giac124</a>. The raw NMR data of all isolated compounds presented in this study
is available through the NP-MRD platform via the NP-MRD IDs provided
for each compound in the experimental section. The raw HRMS/MS data
of all isolated compounds presented in this study is accessible through
the GNPS platform via the spectrum IDs provided for each compound
in the experimental section.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from ACS Bio &amp; Med Chem Au are provided here courtesy of <strong>American Chemical Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1021/acsbiomedchemau.5c00052"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/bg5c00052.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12371487/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12371487/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12371487%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12371487/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12371487/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12371487/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12371487/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12371487/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12371487/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="tgdR92Hr7WxJ7bcjorwyCx5e3eo2oWq0WUUmVk8htqVuwKDZQW09VmnEecTYVjK0">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
