
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            RNA binding and coacervation promote preservation of peptide form and function across the heterochiral–homochiral divide - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4824D8AEED0D305824D0058158421.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="prosci">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375978/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Protein Science : A Publication of the Protein Society">
<meta name="citation_title" content="RNA binding and coacervation promote preservation of peptide form and function across the heterochiral–homochiral divide">
<meta name="citation_author" content="Manas Seal">
<meta name="citation_author_institution" content="Department of Chemical and Biological Physics, Weizmann Institute of Science, Rehovot, Israel">
<meta name="citation_author_institution" content="Department of Education, Indian Institute of Technology Kharagpur, Kharagpur, India">
<meta name="citation_author" content="Ilan Edelstein">
<meta name="citation_author_institution" content="Department of Chemical and Structural Biology, Weizmann Institute of Science, Rehovot, Israel">
<meta name="citation_author" content="Yosef Scolnik">
<meta name="citation_author_institution" content="Israel Institute for Advanced Research, Jerusalem, Israel">
<meta name="citation_author" content="Orit Weil‐Ktorza">
<meta name="citation_author_institution" content="Institute of Chemistry, The Hebrew University of Jerusalem, Jerusalem, Israel">
<meta name="citation_author" content="Norman Metanis">
<meta name="citation_author_institution" content="Institute of Chemistry, The Hebrew University of Jerusalem, Jerusalem, Israel">
<meta name="citation_author_institution" content="Casali Center for Applied Chemistry, The Hebrew University of Jerusalem, Jerusalem, Israel">
<meta name="citation_author_institution" content="The Center for Nanoscience and Nanotechnology, The Hebrew University of Jerusalem, Jerusalem, Israel">
<meta name="citation_author" content="Yaakov Levy">
<meta name="citation_author_institution" content="Department of Chemical and Structural Biology, Weizmann Institute of Science, Rehovot, Israel">
<meta name="citation_author" content="Liam M Longo">
<meta name="citation_author_institution" content="Earth‐Life Science Institute, Institute of Science Tokyo, Tokyo, Japan">
<meta name="citation_author_institution" content="Blue Marble Space Institute of Science, Seattle, Washington, USA">
<meta name="citation_author" content="Daniella Goldfarb">
<meta name="citation_author_institution" content="Department of Chemical and Biological Physics, Weizmann Institute of Science, Rehovot, Israel">
<meta name="citation_publication_date" content="2025 Aug 25">
<meta name="citation_volume" content="34">
<meta name="citation_issue" content="9">
<meta name="citation_firstpage" content="e70273">
<meta name="citation_doi" content="10.1002/pro.70273">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375978/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375978/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375978/pdf/PRO-34-e70273.pdf">
<meta name="description" content="Recent evidence suggests that peptide‐RNA coacervates may have buffered the emergence of folded domains from flexible peptides. As primitive peptides were likely composed of both L‐ and D‐amino acids, we hypothesized that coacervates may have also ...">
<meta name="og:title" content="RNA binding and coacervation promote preservation of peptide form and function across the heterochiral–homochiral divide">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Recent evidence suggests that peptide‐RNA coacervates may have buffered the emergence of folded domains from flexible peptides. As primitive peptides were likely composed of both L‐ and D‐amino acids, we hypothesized that coacervates may have also ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375978/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12375978">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/pro.70273"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/PRO-34-e70273.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375978%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12375978/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12375978/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375978/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-prosci.gif" alt="Protein Science : A Publication of the Protein Society logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Protein Science : A Publication of the Protein Society" title="Link to Protein Science : A Publication of the Protein Society" shape="default" href="http://www.proteinscience.org/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Protein Sci</button></div>. 2025 Aug 25;34(9):e70273. doi: <a href="https://doi.org/10.1002/pro.70273" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/pro.70273</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Protein%20Sci%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Protein%20Sci%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Protein%20Sci%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>
RNA binding and coacervation promote preservation of peptide form and function across the heterochiral–homochiral divide</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Seal%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Manas Seal</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Manas Seal</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemical and Biological Physics, Weizmann Institute of Science, Rehovot, Israel</div>
<div class="p">
<sup>
<sup>2</sup>
</sup>Department of Education, Indian Institute of Technology Kharagpur, Kharagpur, India</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Seal%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Manas Seal</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Edelstein%20I%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Ilan Edelstein</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Ilan Edelstein</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Chemical and Structural Biology, Weizmann Institute of Science, Rehovot, Israel</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Edelstein%20I%22%5BAuthor%5D" class="usa-link"><span class="name western">Ilan Edelstein</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Scolnik%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yosef Scolnik</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yosef Scolnik</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Israel Institute for Advanced Research, Jerusalem, Israel</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Scolnik%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yosef Scolnik</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weil%E2%80%90Ktorza%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Orit Weil‐Ktorza</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Orit Weil‐Ktorza</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Institute of Chemistry, The Hebrew University of Jerusalem, Jerusalem, Israel</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Weil%E2%80%90Ktorza%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Orit Weil‐Ktorza</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Metanis%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Norman Metanis</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Norman Metanis</span></h3>
<div class="p">
<sup>
<sup>5</sup>
</sup>Institute of Chemistry, The Hebrew University of Jerusalem, Jerusalem, Israel</div>
<div class="p">
<sup>
<sup>6</sup>
</sup>Casali Center for Applied Chemistry, The Hebrew University of Jerusalem, Jerusalem, Israel</div>
<div class="p">
<sup>
<sup>7</sup>
</sup>The Center for Nanoscience and Nanotechnology, The Hebrew University of Jerusalem, Jerusalem, Israel</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Metanis%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Norman Metanis</span></a>
</div>
</div>
<sup>5,</sup><sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Levy%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Yaakov Levy</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Yaakov Levy</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>Department of Chemical and Structural Biology, Weizmann Institute of Science, Rehovot, Israel</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Levy%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yaakov Levy</span></a>
</div>
</div>
<sup>3,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Longo%20LM%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Liam M Longo</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Liam M Longo</span></h3>
<div class="p">
<sup>
<sup>8</sup>
</sup>Earth‐Life Science Institute, Institute of Science Tokyo, Tokyo, Japan</div>
<div class="p">
<sup>
<sup>9</sup>
</sup>Blue Marble Space Institute of Science, Seattle, Washington, USA</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Longo%20LM%22%5BAuthor%5D" class="usa-link"><span class="name western">Liam M Longo</span></a>
</div>
</div>
<sup>8,</sup><sup>9,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Goldfarb%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Daniella Goldfarb</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Daniella Goldfarb</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Chemical and Biological Physics, Weizmann Institute of Science, Rehovot, Israel</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Goldfarb%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Daniella Goldfarb</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="pro70273-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Chemical and Biological Physics, Weizmann Institute of Science, Rehovot, Israel</div>
<div id="pro70273-aff-0002">
<sup>
<sup>2</sup>
</sup>Department of Education, Indian Institute of Technology Kharagpur, Kharagpur, India</div>
<div id="pro70273-aff-0003">
<sup>
<sup>3</sup>
</sup>Department of Chemical and Structural Biology, Weizmann Institute of Science, Rehovot, Israel</div>
<div id="pro70273-aff-0004">
<sup>
<sup>4</sup>
</sup>Israel Institute for Advanced Research, Jerusalem, Israel</div>
<div id="pro70273-aff-0005">
<sup>
<sup>5</sup>
</sup>Institute of Chemistry, The Hebrew University of Jerusalem, Jerusalem, Israel</div>
<div id="pro70273-aff-0006">
<sup>
<sup>6</sup>
</sup>Casali Center for Applied Chemistry, The Hebrew University of Jerusalem, Jerusalem, Israel</div>
<div id="pro70273-aff-0007">
<sup>
<sup>7</sup>
</sup>The Center for Nanoscience and Nanotechnology, The Hebrew University of Jerusalem, Jerusalem, Israel</div>
<div id="pro70273-aff-0008">
<sup>
<sup>8</sup>
</sup>Earth‐Life Science Institute, Institute of Science Tokyo, Tokyo, Japan</div>
<div id="pro70273-aff-0009">
<sup>
<sup>9</sup>
</sup>Blue Marble Space Institute of Science, Seattle, Washington, USA</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence</strong>
, 
Yaakov Levy, Department of Chemical and Structural Biology, Weizmann Institute of Science, Rehovot, Israel. 
Email: <span>koby.levy@weizmann.ac.il</span>
, 
Liam M. Longo, Earth‐Life Science Institute, Institute of Science Tokyo, Tokyo, Japan. 
Email: <span>llongo@elsi.jp</span>
, 
Daniella Goldfarb, Department of Chemical and Biological Physics, Weizmann Institute of Science, Rehovot, Israel. 
Email: <span>daniella.goldfarb@weizmann.ac.il</span>

</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Revised 2025 Jul 11; Received 2025 Mar 13; Accepted 2025 Aug 5; Collection date 2025 Sep.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Protein Science</em> published by Wiley Periodicals LLC on behalf of The Protein Society.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a> License, which permits use and distribution in any medium, provided the original work is properly cited, the use is non‐commercial and no modifications or adaptations are made.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12375978</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Recent evidence suggests that peptide‐RNA coacervates may have buffered the emergence of folded domains from flexible peptides. As primitive peptides were likely composed of both <em>L</em>‐ and <em>D</em>‐amino acids, we hypothesized that coacervates may have also supported the emergence of chiral control. To test this hypothesis, we compared the coacervation propensities of an isotactic (homochiral) peptide and a syndiotactic (alternating chirality) peptide, both with an identical sequence derived from the ancient helix–hairpin–helix (HhH) motif. Using electron paramagnetic resonance (EPR) spectroscopy and atomistic molecular dynamics (MD) simulations, we found that the syndiotactic peptide does not form stable dimers with high α‐helicity <em>in solution</em>, unlike the isotactic peptide. However, both peptides do coacervate with RNA, albeit with distinct reentrant phase behaviors. Coacervation in each case is facilitated by oligomer formation, likely dimerization, upon RNA binding that promotes RNA cross‐linking. Additionally, RNA cross‐linking and coacervation of the syndiotactic peptide may involve α‐helical conformations, according to atomistic MD simulations. Coarse‐grained MD simulations indicate that the differences in reentrant phase behavior of isotactic and syndiotactic peptides are associated with differences in dimer flexibility and stability, which modulate the strength of peptide–peptide and peptide–RNA interactions and, consequently, the effectiveness of RNA cross‐linking. These results illustrate how RNA binding and/or coacervation by early proteins could have promoted the transition of flexible, heterochiral peptides into folded, homochiral domains.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> EPR, heterochiral and homochiral peptides, molecular dynamics, peptide‐RNA coacervates, protein evolution</p></section></section><section id="pro70273-sec-0001"><h2 class="pmc_sec_title">1. INTRODUCTION</h2>
<p>Chiral control over translation is a prerequisite for reliably encoding well‐defined protein conformations. Although both homochiral and heterochiral peptides can adopt complex, stable structures (Urry et al. <a href="#pro70273-bib-0033" class="usa-link" aria-describedby="pro70273-bib-0033">1971</a>; Veatch et al. <a href="#pro70273-bib-0034" class="usa-link" aria-describedby="pro70273-bib-0034"><sup>1974</sup></a>), this is almost always achieved in the context of a defined pattern of amino acid chirality. The incorporation of even a single mirror‐image amino acid into a contemporary protein can be significantly destabilizing (Krause et al. <a href="#pro70273-bib-0018" class="usa-link" aria-describedby="pro70273-bib-0018">2000</a>)—and statistical patterns of amino acid chirality would seem to radically limit any <em>specific</em> structural and functional property due to differences in foldability and folding between chiral patterns. And yet, the earliest protein forms were likely composed of a combination of <em>L</em>‐ and <em>D</em>‐amino acids (Kim et al. <a href="#pro70273-bib-0016" class="usa-link" aria-describedby="pro70273-bib-0016">2018</a>), which were present in the environment, and constructed from processes vastly more permissive than the exquisite system of contemporary translation, which is itself able to incorporate <em>D</em>‐amino acids (Fujino et al. <a href="#pro70273-bib-0009" class="usa-link" aria-describedby="pro70273-bib-0009">2013</a>). The emergence of homochirality, then, does not regard just the intrinsic structure‐forming potential of homochiral versus heterochiral peptides but also the extent to which biochemical and biophysical properties of a given amino acid sequence can persist against the backdrop of radically different patterns of Cα chirality. If persistence of form or function across the heterochiral–homochiral divide is somehow possible, it would suggest a smoother evolutionary landscape, one more amenable to exploration and discovery than previously thought possible.</p>
<p>We have argued that coacervates represent an evolutionarily significant context for exploring novel protein conformations, particularly for oligomers (Longo et al. <a href="#pro70273-bib-0020" class="usa-link" aria-describedby="pro70273-bib-0020">2020</a>; Seal et al. <a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030"><sup>2022</sup></a>). Previously, we reported a simplified peptide derived from the ubiquitous nucleic acid‐binding helix–hairpin–helix (HhH) motif (Alva et al. <a href="#pro70273-bib-0002" class="usa-link" aria-describedby="pro70273-bib-0002">2015</a>; Doherty et al. <a href="#pro70273-bib-0006" class="usa-link" aria-describedby="pro70273-bib-0006"><sup>1996</sup></a>), the precursor to the symmetric (HhH)<sub>2</sub>‐fold that is produced upon duplication and fusion of this structural element (Alva et al. <a href="#pro70273-bib-0002" class="usa-link" aria-describedby="pro70273-bib-0002">2015</a>). We found that dimerization of the simplified HhH peptide, which occurs weakly in solution, is promoted upon binding to RNA and forming coacervates (Seal et al. <a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>). Based on these data, we hypothesized that these dimers represent a step in the evolution of the (HhH)<sub>2</sub>‐fold from a flexible peptide into a folded domain. More recently, we have demonstrated that the HhH motif and (HhH)<sub>2</sub>‐fold are functionally ambidextrous (Weil‐Ktorza et al. <a href="#pro70273-bib-0027" class="usa-link" aria-describedby="pro70273-bib-0027"><sup>2025</sup></a>)—able to retain their biochemical functions (coacervation or dsDNA binding, respectively) irrespective of the chirality of the binding partner (RNA or DNA derived from <em>D</em>‐ or <em>L</em>‐ribose). To determine whether dimerizing into the (HhH)<sub>2</sub>‐fold was essential for HhH motif coacervation in the presence of RNA, a syndiotactic (alternating chirality) variant of the HhH peptide with alternating <em>L</em>‐ and <em>D</em>‐amino acids was synthesized. The resulting peptide was observed to coacervate with RNA (Weil‐Ktorza et al. <a href="#pro70273-bib-0027" class="usa-link" aria-describedby="pro70273-bib-0027"><sup>2025</sup></a>). However, a molecular description of the syndiotactic HhH coacervation process—and thus, the potential significance of dimerization, α‐helix formation, and binding to the phosphodiester backbone—was unknown, limiting our understanding of this result (Weil‐Ktorza et al. <a href="#pro70273-bib-0027" class="usa-link" aria-describedby="pro70273-bib-0027"><sup>2025</sup></a>).</p>
<p>Here, we return to the syndiotactic variant of the HhH motif for a detailed characterization. The choice of comparing isotactic (single backbone chirality) and syndiotactic (alternating backbone chirality) variants of HhH was motivated by lattice models and molecular dynamics simulations indicating that their structural spaces are extremes of intrinsic compaction and foldability (Nanda et al. <a href="#pro70273-bib-0025" class="usa-link" aria-describedby="pro70273-bib-0025">2007</a>). In other words, the syndiotactic variant of HhH is likely to be among the most disruptive chiral patterns possible and, thus, an ideal foil for comparison to the isotactic form of HhH. To uncover the details of syndiotactic HhH coacervation at the molecular level, we analyzed spin‐labeled peptides by three electron paramagnetic resonance (EPR) approaches—double electron‐electron resonance (DEER) spectroscopy to detect oligomerization and continuous wave (CW) EPR along with electron‐nuclear double resonance (ENDOR) to probe binding to RNA. These experimental approaches were then complemented with atomistic and coarse‐grained molecular dynamics (MD) simulations to characterize the structural and energetic properties of the isotactic and syndiotactic peptides when they interact with RNA molecules in coacervates.</p>
<p>As expected, oligomerization of the syndiotactic HhH motif was not detected in solution. Remarkably, oligomerization of the syndiotactic peptide was detected upon RNA binding and coacervation, though with a higher degree of disorder compared to the isotactic analog. Atomistic MD simulations further supported dimer formation and revealed the potential formation of an α‐helical element upon RNA binding. Phase diagrams showed that the range of peptide/RNA ratios that support coacervation is significantly narrower for the syndiotactic peptide, which is explained by coarse‐grained MD simulations showing how dimer flexibility can affect the strength of the interactions between peptides and RNA, thereby impacting phase behavior. We ascribe the lower coacervation propensity of the syndiotactic peptide to differences in monomer–dimer equilibria and structural content that affect RNA cross‐linking efficiency. Foremost, these results suggest that RNA binding and coacervation may have helped buffer complex molecular transitions, such as the emergence of folded domains and the establishment of homochirality.</p></section><section id="pro70273-sec-0002"><h2 class="pmc_sec_title">2. MATERIALS AND METHODS</h2>
<section id="pro70273-sec-0003"><h3 class="pmc_sec_title">2.1. Materials</h3>
<p>Syndiotactic peptides (PA(<em>D</em>/<em>L</em>); Figure <a href="#pro70273-fig-0001" class="usa-link">1</a>) were purchased from Synpeptide Co., Limited (Shanghai, China, <a href="http://www.synpeptide.com" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.synpeptide.com</a>) with &gt;90% purity and were used for labeling without further purification. The synthesis and purification of isotactic peptides (PA(<em>L</em>)) were reported earlier (Seal et al. <a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>). 4‐Malemido‐TEMPO, deuterium oxide (D<sub>2</sub>O), and trifluoroacetic acid (TFA) were purchased from Sigma‐Aldrich. Tris(2‐carboxyethyl)‐phosphine (TCEP) was purchased from Merck. Deuterated glycerol was purchased from Cambridge Isotopes. HPLC‐grade acetonitrile and water were purchased from Bio‐Lab and J.T. Baker, respectively.</p>
<figure class="fig xbox font-sm" id="pro70273-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bcfb/12375978/f39a1173f535/PRO-34-e70273-g006.jpg" loading="lazy" id="jats-graphic-1" height="193" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/pro70273-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) Sequences, charges, and chiral patterns of the peptides used in this study. PA is the parent peptide and comprises only <em>L</em>‐amino acids. PA(<em>D</em>/<em>L</em>) has the same sequence as PA but with alternating <em>D</em>‐ and <em>L</em>‐amino acids. Achiral glycine is indicated with an A in the chiral patterns. The spin‐labeled positions are colored red. (b) The structure of the spin label, 4‐maleimido‐TEMPO.</p></figcaption></figure></section><section id="pro70273-sec-0004"><h3 class="pmc_sec_title">2.2. Peptide labeling and EPR sample preparation</h3>
<p>The syndiotactic peptides were labeled following the previously described protocol (Seal et al. <a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>) with slight modifications. Briefly, lyophilized peptide was dissolved in Milli‐Q water to prepare a stock solution of 2 mM peptide, which was then diluted to 1 mM peptide with 100 mM phosphate buffer, pH ~7.2. Ten equivalents of TCEP were added to reduce any disulfide bonds. A 50 mM stock solution of 4‐maleimido‐TEMPO (M‐TEMPO) was prepared in DMSO, and 10 equivalents of the M‐TEMPO solution were added to the peptide and placed in a rotatory shaker at room temperature for approximately 5 h. Labeled peptides were purified by RP‐HPLC (using a Vydac 214TP C4 5 μm column) and a gradient of 10%–70% acetonitrile over 40 min with a 1 mL/min flow rate. The injected sample volume was 30 μL, and the labeled peptide elutes around 26.5 min. The presence of spin‐labeled peptide in the collected fractions was confirmed by CW‐EPR spectroscopy. Finally, the purified, labeled peptides were lyophilized and dissolved in D<sub>2</sub>O. The final peptide concentration, assuming complete labeling, was determined from the area of the CW‐EPR spectrum.</p></section><section id="pro70273-sec-0005"><h3 class="pmc_sec_title">2.3. Optical microscopy</h3>
<p>Optical microscope images of phase‐separated samples were taken at the de Picciotto Cancer Cell Observatory in the Life Sciences Core Facilities (Weizmann Institute of Science) using a 100× objective (oil immersion) on a Leica DMI8 microscope with differential interface contrast (DIC). One to two microliters of thedroplet solution was placed on an imaging chamber prepared by attaching a coverslip to a clean glass slide using a thin strip of double‐sided tape (AJ Sign World). Images were processed using the software package Fiji (Schindelin et al. <a href="#pro70273-bib-0029" class="usa-link" aria-describedby="pro70273-bib-0029">2012</a>).</p></section><section id="pro70273-sec-0006"><h3 class="pmc_sec_title">2.4. 
EPR spectroscopy</h3>
<p>CW‐EPR spectra were recorded on an Elexsys E500 X‐Band (9.5 GHz) Bruker spectrometer using a high‐sensitivity resonator at room temperature. The parameters used to record the EPR spectra were 0.1 mT field modulation amplitude and 20 mW microwave power. Each scan was 42 s, and at least 4 scans were accumulated for each spectrum. At least 25–36 scans were collected for samples with coacervates to improve the signal‐to‐noise (S/N) ratio. For CW‐EPR measurements, a quartz capillary with a 0.6‐mm inner diameter (i.d.) and a 0.84‐mm outer diameter (o.d.) was filled with 7–8 μL of sample and sealed with Critoseal (Fisherbrand, Fisher Scientific) at one end. A single capillary was placed in an X‐band EPR tube with a 2‐mm i.d. and a 2.4‐mm o.d. for measurement.</p>
<p>The EPR spectra were simulated using the Chili routine of EasySpin (<a href="http://www.easyspin.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.easyspin.org</a>; Stoll and Schweiger <a href="#pro70273-bib-0032" class="usa-link" aria-describedby="pro70273-bib-0032"><sup>2006</sup></a>). As before, we simulated the EPR spectra using two components, employing the isotropic rotational diffusion model (Seal et al. <a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>). The use of the isotropic rotational model may be an oversimplification of the system, considering local order and ordering potential; however, it is sufficient to understand the rise of a slow‐motion component while avoiding a large number of parameters. The parameters for simulations are given in Tables <a href="#pro70273-supitem-0001" class="usa-link">S1</a> and <a href="#pro70273-supitem-0001" class="usa-link">S2</a>, Supporting Information (more details are given in Data <a href="#pro70273-supitem-0001" class="usa-link">S1</a>).</p>
<p>Pulse EPR measurements were carried out at the W‐band (94.9 GHz) on a home‐built spectrometer at 25 K (Bahrenberg et al. <a href="#pro70273-bib-0003" class="usa-link" aria-describedby="pro70273-bib-0003">2017</a>; Goldfarb et al. <a href="#pro70273-bib-0011" class="usa-link" aria-describedby="pro70273-bib-0011"><sup>2008</sup></a>; Mentink‐Vigier et al. <a href="#pro70273-bib-0024" class="usa-link" aria-describedby="pro70273-bib-0024"><sup>2013</sup></a>). Echo intensities and echo decays were measured with a Hahn echo sequence (π/2–τ–π–τ–echo) using π/2 and π pulses of 15–20 ns and 30–40 ns, respectively, and τ = 500 ns. For pulse EPR measurements, 3–4 μL of peptide in D<sub>2</sub>O with 20% glycerol‐d<sub>8</sub> were introduced into a quartz capillary with a 0.6‐mm i.d. and a 0.84‐mm o.d. and sealed with Critoseal at one end. We confirmed the formation of coacervates in the presence of 20% glycerol with the PA(<em>L</em>) peptide. The samples with droplets (e.g., 100 μM PA(<em>D</em>/<em>L</em>)‐12 with 0.2/0.35/0.5 mg/mL polyU) were flash frozen in liquid nitrogen before inserting them into the spectrometer, whereas other samples were directly inserted into the spectrometer at 25 K.</p>
<p>The general setup was as follows: The maximum echo intensity of the nitroxide spectrum was set to 94.925 GHz, the observer pulses were set to 94.83 GHz, and the π/2 and π pulse durations were 15–20 and 30–40 ns, respectively. The chirp pump pulse frequency was 94.88–94.98 GHz with a duration of 128 ns. The repetition time was 10 ms. The data for samples without polyU were acquired for 1‐5 h and for samples with polyU for 5–15 h. Echo decay traces were fitted using the stretched exponential function <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-1" display="inline" overflow="linebreak"><mrow><mrow><mi>A</mi><mo>·</mo><mi>exp</mi><msup><mfenced open="(" close=")"><mrow><mo>−</mo><mn>2</mn><mfrac><mi>τ</mi><msub><mi>T</mi><mi>m</mi></msub></mfrac></mrow></mfenced><mi>d</mi></msup></mrow></mrow></math></span>, where <em>A</em> ~ 1, <em>T</em>
<sub>m</sub> is the phase memory time, and <em>d</em> is the stretched exponent.</p>
<p>DEER measurements were recorded with the four‐pulse DEER sequence (π/2<sub>νobs</sub>–τ<sub>1</sub>–π<sub>νobs</sub>–(τ<sub>1</sub> + <em>t</em>)–π<sub>νpump</sub>–(τ<sub>2</sub> − <em>t</em>)–π<sub>νobs</sub>–τ<sub>2</sub>–echo) with chirp pump pulse(s) using 4 step phase cycling and monitoring the echo intensity as a function of <em>t</em>. DEER data were analyzed assuming a Gaussian distribution of distances using the program DD (Hustedt et al. <a href="#pro70273-bib-0013" class="usa-link" aria-describedby="pro70273-bib-0013">2018</a>).</p>
<p>
<sup>31</sup>P Mims ENDOR spectra were recorded at 15 K at a magnetic field corresponding to the maximum echo intensity using the sequence (π/2–τ–π/2–π<sub>RF</sub>–π/2–τ–echo), complemented with CPMG detection (Mentink‐Vigier et al. <a href="#pro70273-bib-0024" class="usa-link" aria-describedby="pro70273-bib-0024"><sup>2013</sup></a>) with π pulses of 30 ns and a pulse spacing of 1.1 μs. Additional delays of 200 ns before and 5 μs after the RF pulse were added in order to avoid artifacts from ring‐down effects. Data were recorded using random acquisition mode (Epel et al. <a href="#pro70273-bib-0007" class="usa-link" aria-describedby="pro70273-bib-0007">2003</a>). The repetition time was 15–50 ms. The typical acquisition time for the solution samples was 6–18 h. All the ENDOR spectra were recorded with RF pulses amplified by a 2‐kW RF amplifier (TOMCO Technologies, 2 kW PEP, 5–310 MHz). The random acquisition mode was used with steps of 4 kHz for PA(<em>D</em>/<em>L</em>)‐12 and 4 and 10 kHz for PA(<em>D</em>/<em>L</em>)‐29 and the spectral width was ±200 kHz and ±400 kHz <span class="text-strike">scans</span>, respectively, around the <sup>31</sup>P Larmor frequency. Ten shots per point were used. The RF π pulse length was 50 μs and was optimized for <sup>7</sup>Li using B<sub>2</sub> nutation measurements on a standard gamma‐irradiated LiF sample. <sup>7</sup>Li has a Larmor frequency of 56.26 MHz at 3.4 T, whereas <sup>31</sup>P has a Larmor frequency of 58.60 MHz; therefore, it is convenient to adjust the RF on a reference sample that has a good EPR signal and very strong Li ENDOR lines (Gromov et al. <a href="#pro70273-bib-0012" class="usa-link" aria-describedby="pro70273-bib-0012">1999</a>).</p></section><section id="pro70273-sec-0007"><h3 class="pmc_sec_title">2.5. Circular dichroism</h3>
<p>Circular dichroism (CD) spectra were collected on a Chirascan CD spectrometer (Applied Photophysics) with a 0.2‐mm pathlength quartz cuvette. All spectra were collected from 180 to 360 nm with a data pitch of 1 nm at room temperature, adaptive sampling (variable signal averaging at each wavelength), and a slit width of 1 nm. The photomultiplier tube voltage during measurement was kept below 700 V; data points exceeding this value were discarded. The buffer used was the same as in the samples used for the EPR and microscopy measurements. All reported spectra have had the spectrum of the buffer subtracted.</p></section><section id="pro70273-sec-0008"><h3 class="pmc_sec_title">2.6. 
MD simulations</h3>
<p>Atomistic MD simulations of PA(<em>L</em>) and PA(<em>D</em>/<em>L</em>) were performed using GROMACS (v. 2022.1) (Pronk et al. <a href="#pro70273-bib-0028" class="usa-link" aria-describedby="pro70273-bib-0028">2013</a>) with the CHARMM36m force field parameters for protein, RNA, TIP3P water, and ions (Abraham et al. <a href="#pro70273-bib-0001" class="usa-link" aria-describedby="pro70273-bib-0001">2015</a>). Three‐dimensional periodic boundary conditions were used for all systems. The box dimensions were chosen to be 1 nm from the solute, resulting in approximately 9 nm side length for systems with RNA and 6 nm side length for systems without RNA. Peptide dimers and polyU molecules were placed at the center of a cubic box and solvated. Sodium and chloride ions were added to neutralize the overall charge of the system. All systems underwent minimization, NVT equilibration, and NPT equilibration. Peptides were capped with an N‐terminal acetyl and a C‐terminal N‐methylamide in order to avoid artificial charge–charge interactions between the termini. Initial atomic coordinates for the PA(<em>L</em>) dimer were as before (Seal et al. <a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>). The structure of the PA(<em>D</em>/<em>L</em>) peptide was initialized using the invert function in PyMOL (<a href="http://www.pymol.org" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.pymol.org</a>) on the initial coordinates of PA(<em>L</em>). The inverted amino acids were specified as <em>D</em>‐amino acids DAAA in the initial structure file, where AAA is the conventional amino acid name. For all heterochiral systems, we manually checked that the chirality of the residues is preserved throughout the simulations. The structure of polyU was generated using the fnab PyMOL command. Unless stated otherwise, two 20‐nucleotide polyU molecules were added to each system. Interaction energies were calculated using the GROMACS gmx energy and α‐helicity was estimated by the define secondary structure of proteins (DSSP) algorithm (Kabsch and Sander <a href="#pro70273-bib-0015" class="usa-link" aria-describedby="pro70273-bib-0015">1983</a>). Radius of gyration (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-2" display="inline" overflow="linebreak"><mrow><mrow><msub><mi>R</mi><mi>g</mi></msub></mrow></mrow></math></span>), α‐helical propensity, and root mean square deviation (RMSD) were calculated using MDTraj (McGibbon et al. <a href="#pro70273-bib-0023" class="usa-link" aria-describedby="pro70273-bib-0023"><sup>2015</sup></a>). Simulations of the isotactic systems were 500 ns in duration and the first 100 ns of each simulation was excluded from analysis. For the syndiotactic systems, simulations were extended to 1 μs to ensure full equilibration, and only the final 400 ns were used for analysis. Three simulations, a total of 1.2 μs, were performed for all systems. Long‐range electrostatic interactions were modeled using Particle Mesh Ewald with a cutoff of 1.0 nm. Non‐bonded neighboring pairs were searched using the Verlet algorithm and hydrogen bonds were constrained using the LINCS algorithm. The temperature was controlled using a Berendsen coupling thermostat and maintained at a temperature of 300 K. Pressure was controlled using a Parrinello‐Rahman coupling barostat and maintained at 1 bar.</p></section><section id="pro70273-sec-0009"><h3 class="pmc_sec_title">2.7. Coarse‐grained modeling</h3>
<p>Coarse‐grained (CG) simulations were performed on three versions of the PA peptide. PA(<em>L</em>) was modeled as a folded peptide, PA(<em>D</em>/<em>L</em>) as a transiently folded peptide, and finally, PA(<em>Dis</em>) as a predominantly disordered peptide. Each residue was represented by a single bead centered on the Cα, while RNA nucleotides were modeled with three beads for the phosphate, sugar, and base. Simulations were conducted using OpenMM (Fried et al. <a href="#pro70273-bib-0008" class="usa-link" aria-describedby="pro70273-bib-0008">2022</a>), with bonded interactions defined via standard harmonic bond and angle potentials. Hydrophobicity was incorporated using the Mpipi model (Joseph et al. <a href="#pro70273-bib-0037" class="usa-link" aria-describedby="pro70273-bib-0037">2021</a>), where short‐range interactions are modeled as pairwise interactions via the Wang‐Frenkel (WF) potential. Electrostatic interactions were modeled using the Debye‐Hückel potential. Peptide secondary structure was introduced via torsion angles and contact pairs. The torsion angles were applied to 1–4 successive Cα beads to maintain the backbone conformation. Specific contact pairs, identified via the shadow algorithm (Noel et al. <a href="#pro70273-bib-0038" class="usa-link" aria-describedby="pro70273-bib-0038">2012</a>), were modeled using a 12‐10 Lennard‐Jones (LJ) potential. The degree of folding of the three variants of the PA peptide was modulated by varying the LJ well depths. Inter‐monomer contact pairs in all systems were extended to any other monomer present in the simulation, allowing initially associated monomers that became separated to fully or partially re‐dimerize with any other available monomer.</p>
<p>All simulations were run in a 30‐nm cubic box with periodic boundaries, containing 120 peptides and four 100 nt polyU molecules (or peptides alone for systems without RNA). Following energy minimization and equilibration, a 1 μs production run was performed with 10 fs time steps. Three independent simulations were performed for each system, yielding a total aggregate simulation time of 3 μs. Phase diagrams were constructed by simulating each system at increasing temperatures. The critical temperature, <em>T</em>
<sub>c</sub>, for each system were determined by fitting the densities distributions through a scaling law. All systems were subsequently compared at <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-3" display="inline" overflow="linebreak"><mrow><mrow><mn>0.9</mn><msub><mi>T</mi><mi mathvariant="normal">c</mi></msub></mrow></mrow></math></span>. Diffusion coefficients of the peptides in the dense phase were extracted from the slope of the mean squared displacements. Interaction energies were obtained directly from OpenMM methods, summing electrostatic and hydrophobic contributions in the dense phase. More details regarding the model and the analysis are given in Data <a href="#pro70273-supitem-0001" class="usa-link">S1</a> and Figures <a href="#pro70273-supitem-0001" class="usa-link">S1</a> and <a href="#pro70273-supitem-0001" class="usa-link">S2</a>.</p></section></section><section id="pro70273-sec-0010"><h2 class="pmc_sec_title">3. RESULTS</h2>
<section id="pro70273-sec-0011"><h3 class="pmc_sec_title">3.1. Syndiotactic and isotactic HhH peptides have distinct phase behavior</h3>
<p>The peptide sequence used in this study, Precursor‐Arg (PA), is the result of an ancestral sequence reconstruction and simplification of the (HhH)<sub>2</sub>‐fold reported earlier by Tawfik and coworkers (Longo et al. <a href="#pro70273-bib-0020" class="usa-link" aria-describedby="pro70273-bib-0020">2020</a>). For clarity, the isotactic variant of PA with natural chirality will be referred to as PA(<em>L</em>) and the syndiotactic variant of PA will be referred to as PA(<em>D</em>/<em>L</em>). Peptide sequences and chiral patterns are provided in Figure <a href="#pro70273-fig-0001" class="usa-link">1a</a>. As before (Seal et al. <a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>), positions 12 and 29, which are at the end of the first and second α‐helices of PA(<em>L</em>), respectively, were labeled with M‐TEMPO (Figure <a href="#pro70273-fig-0001" class="usa-link">1b</a>). The naming convention used for spin‐labeled peptides is the parent peptide followed by the chirality in parenthesis and the site of spin labeling. For example, PA(<em>D</em>/<em>L</em>)‐12 refers to the Precursor‐Arg peptide with alternating chirality (<em>D</em>/<em>L</em>) in which position 12 is a cysteine residue attached to an M‐TEMPO spin label.</p>
<p>We compared the phase separation behavior of PA(<em>L</em>)‐12 and PA(<em>D</em>/<em>L</em>)‐12 at a fixed concentration of 100 μM peptide and varying amounts of polyU in the range of 0–2 mg/mL. Coacervates were detected for both labeled PA(<em>L</em>)‐12 and unlabeled PA(<em>L</em>) (Figures <a href="#pro70273-supitem-0001" class="usa-link">S3</a> and <a href="#pro70273-supitem-0001" class="usa-link">S4</a>). Comparison of the coacervation behavior of PA(<em>D</em>/<em>L</em>)‐12 and PA(<em>L</em>)‐12 across a range of polyU concentrations (Figures <a href="#pro70273-fig-0002" class="usa-link">2</a>, <a href="#pro70273-supitem-0001" class="usa-link">S3</a>, and <a href="#pro70273-supitem-0001" class="usa-link">S5</a>) revealed a notable difference: Both the appearance and disappearance of coacervates occur at lower concentrations of polyU for PA(<em>D</em>/<em>L</em>)‐12 than for PA(<em>L</em>)‐12. More specifically, PA(<em>D</em>/<em>L</em>)‐12 phase separated only in the relatively narrow range of 0.1–0.35 mg/mL polyU, whereas for PA(<em>L</em>)‐12 coacervates emerge somewhat later at 0.35 mg/mL polyU and are stable until at least 2.0 mg/mL polyU (higher concentrations were not tested), as previously observed (Seal et al. <a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>). To rationalize the differences in phase behavior of the syndiotactic and isotactic HhH peptides on a molecular level, we proceeded to study PA(<em>D</em>/<em>L</em>)‐12 using EPR spectroscopy.</p>
<figure class="fig xbox font-sm" id="pro70273-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bcfb/12375978/85a1d36a290f/PRO-34-e70273-g004.jpg" loading="lazy" id="jats-graphic-3" height="207" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/pro70273-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) Micrograph of 100 μM PA(<em>D</em>/<em>L</em>)‐12 in the presence of 0.2 mg/mL polyU. Inset shows a magnified view of the indicated area. (b) Micrographs of 100 μM PA(<em>D</em>/<em>L</em>)‐12 in the presence of 0–1 mg/mL polyU (indicated above each image) with a summary table highlighting the differences between PA(<em>L</em>) and PA(<em>D</em>/<em>L</em>). Full field of view for PA(<em>D</em>/<em>L</em>)‐12 micrographs are shown in Figure <a href="#pro70273-supitem-0001" class="usa-link">S5</a>. Phase separation behavior of PA(<em>L</em>)‐12 is presented in Figure <a href="#pro70273-supitem-0001" class="usa-link">S3</a> and in Seal et al. (<a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>).</p></figcaption></figure></section><section id="pro70273-sec-0012"><h3 class="pmc_sec_title">3.2. 
PA(<em>D</em>/<em>L</em>) does not oligomerize in solution</h3>
<p>We have previously demonstrated that PA(<em>L</em>)‐12 forms dimers in solution in the absence of polyU with a dissociation constant, <em>K</em>
<sub>D</sub>, of 233 ± 134 μM (Seal et al. <a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>)—can PA(<em>D</em>/<em>L</em>) also form dimers in solution? To answer this question, PA(<em>D</em>/<em>L</em>)‐12 and PA(<em>D</em>/<em>L</em>)‐29 were subjected to EPR characterization (Figure <a href="#pro70273-fig-0003" class="usa-link">3</a>). The CW‐EPR spectra of the two peptides are characteristic of a spin label undergoing fast motion. The comparatively greater intensity of the high field line of PA(<em>D</em>/<em>L</em>)‐29 indicates somewhat faster rotational dynamics than PA(<em>D</em>/<em>L</em>)‐12 (Figure <a href="#pro70273-fig-0003" class="usa-link">3a</a>). This is an expected result, given that position 29 is at the C‐terminus of the peptide whereas position 12 is in the middle of the peptide. Both spectra fall in the typical range of fast motion for nitroxide‐labeled small peptides (Bordignon <a href="#pro70273-bib-0004" class="usa-link" aria-describedby="pro70273-bib-0004">2017</a>). We then carried out DEER measurements on the solution and, unlike PA(<em>L</em>)‐12, evidence of oligomerization was observed for neither PA(<em>D</em>/<em>L</em>)‐12 nor ‐29 (Figure <a href="#pro70273-fig-0003" class="usa-link">3b</a>). Even at a concentration as high as 330 μM peptide, the DEER trace showed no modulation. Under similar conditions, 150 μM PA(<em>L</em>)‐12 forms dimers with a mean distance of 3.4 nm between the spin labels (Figure <a href="#pro70273-supitem-0001" class="usa-link">S6</a>), consistent with the (HhH)<sub>2</sub>‐Fold (Seal et al. <a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>). The absence of PA(<em>D</em>/<em>L</em>) dimers in solution suggests that either monomeric PA(<em>D</em>/<em>L</em>) mediates coacervation or, as observed for PA(<em>L</em>), RNA binding and/or coacervation stabilize dimeric or oligomeric states.</p>
<figure class="fig xbox font-sm" id="pro70273-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bcfb/12375978/6f429246b1f0/PRO-34-e70273-g003.jpg" loading="lazy" id="jats-graphic-5" height="305" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/pro70273-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) Room temperature X‐band CW‐EPR spectra of PA(<em>D</em>/<em>L</em>)‐12 and PA(<em>D</em>/<em>L</em>)‐29. (b) 25 K W‐band DEER of PA(<em>D</em>/<em>L</em>)‐12 and ‐29 compared with that of PA(<em>L</em>)‐12.</p></figcaption></figure></section><section id="pro70273-sec-0013"><h3 class="pmc_sec_title">3.3. Coacervation of PA(<em>D</em>/<em>L</em>)‐12 reduces peptide mobility</h3>
<p>The CW‐EPR spectra of PA(<em>D</em>/<em>L</em>)‐12 in the presence of various polyU concentrations exhibited changes that correlate with coacervate formation observed by optical microscopy. Up to 0.35 mg/mL polyU, a slow‐motion component gradually emerged (Figure <a href="#pro70273-fig-0004" class="usa-link">4a</a>, black arrow), tracking binding to RNA, coacervation, or some combination of these two processes. Above 0.35 mg/mL polyU, the contribution of the slow‐motion component started to decrease; at 1 mg/mL, it was practically undetectable (Figure <a href="#pro70273-fig-0004" class="usa-link">4a</a>). The relative contribution of the slow‐motion component was estimated by simulating the CW‐EPR spectra (Figure <a href="#pro70273-supitem-0001" class="usa-link">S7</a> and Table <a href="#pro70273-supitem-0001" class="usa-link">S1</a>). We found a correlation between the relative intensity of the slow‐motion component (Figure <a href="#pro70273-fig-0004" class="usa-link">4b</a>) and the appearance of coacervates (Figures <a href="#pro70273-fig-0002" class="usa-link">2b</a> and <a href="#pro70273-supitem-0001" class="usa-link">S5</a>), indicating that the slow‐motion component is specifically due to peptide binding to polyU within the coacervates. Accordingly, it reports the percentage of the RNA‐bound peptide within coacervates as a function of polyU concentration (Figure <a href="#pro70273-fig-0004" class="usa-link">4c</a>). Intriguingly, PA(<em>D</em>/<em>L</em>) exhibits a notably higher proportion of the slow‐motion component than PA(<em>L</em>) in the range of 0.1–0.35 mg/mL of polyU and forms coacervates at lower polyU concentrations (Figures <a href="#pro70273-fig-0004" class="usa-link">4c</a> and <a href="#pro70273-supitem-0001" class="usa-link">S8</a> and Table <a href="#pro70273-supitem-0001" class="usa-link">S2</a>).</p>
<figure class="fig xbox font-sm" id="pro70273-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bcfb/12375978/38ce850278d3/PRO-34-e70273-g009.jpg" loading="lazy" id="jats-graphic-7" height="559" width="709" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/pro70273-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) Normalized CW‐EPR spectra of 100 μM PA(<em>D</em>/<em>L</em>)‐12 in the presence of different polyU concentrations. The arrow indicates the position of the slow‐motion component and the asterisks (*) marks the background signal of the cavity. (b) EPR signal intensity at 333.7 mT of the slow‐motion component in panel (a) and the number of observed droplets in the same field of view as presented in Figures <a href="#pro70273-fig-0002" class="usa-link">2</a> and <a href="#pro70273-supitem-0001" class="usa-link">S5</a>. (c) Relative abundance of the slow‐motion component of the CW‐EPR spectra for 100 μM PA(<em>D</em>/<em>L</em>)‐12 and PA(<em>L</em>)‐12 at different concentrations of polyU. The simulated spectra and the simulation parameters are shown in Figures <a href="#pro70273-supitem-0001" class="usa-link">S7</a> and <a href="#pro70273-supitem-0001" class="usa-link">S8</a> and Tables <a href="#pro70273-supitem-0001" class="usa-link">S1</a> and <a href="#pro70273-supitem-0001" class="usa-link">S2</a>. The values for PA(<em>L</em>)‐12 with 1 and 2 mg/mL polyU were taken from Seal et al. (<a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>). (d) 15 K W‐band <sup>31</sup>P Mims ENDOR spectra of 100 μM PA(<em>D</em>/<em>L</em>)‐12 and PA(<em>D</em>/<em>L</em>)‐29 in the presence of 1 mg/mL polyU, where coacervation is not observed. Two measurements were done for each sample, differing in the spectral width chosen (dark color for large width and light color for small width). The arrow indicates the position of the <sup>31</sup>P Larmor frequency at the W‐band. The ENDOR efficiency for PA(<em>D</em>/<em>L</em>)‐29 and PA(<em>D</em>/<em>L</em>)‐12 were 1 ± 0.2% and 1.5 ± 0.3%, respectively.</p></figcaption></figure><p>At 1 mg/mL polyU, the CW‐EPR spectrum of PA(<em>D</em>/<em>L</em>)‐12 consists of only a fast‐motion component. Still, it is significantly broadened and exhibits slower dynamics than PA(<em>D</em>/<em>L</em>)‐12 in the absence of polyU (Figure <a href="#pro70273-fig-0004" class="usa-link">4a</a> and <a href="#pro70273-supitem-0001" class="usa-link">S9</a>). This observation indicates that at high polyU concentrations, PA(<em>D</em>/<em>L</em>)‐12 does interact with polyU and is not unbound, despite the absence of a typical slow‐motion component. To further confirm peptide binding to polyU at high concentrations of polyU, we probed the proximity of the spin label in PA(<em>D</em>/<em>L</em>) to the polyU phosphate groups by <sup>31</sup>P Mims ENDOR, which can detect <sup>31</sup>P nuclei at a distance of up to 1 nm from the spin label (Zänker et al. <a href="#pro70273-bib-0036" class="usa-link" aria-describedby="pro70273-bib-0036">2004</a>). The ENDOR spectra of both PA(<em>D</em>/<em>L</em>)‐12 and PA(<em>D</em>/<em>L</em>)‐29 show a peak at the <sup>31</sup>P Larmor frequency (Figure <a href="#pro70273-fig-0004" class="usa-link">4d</a>), confirming that the peptides are interacting with polyU. The peak is rather broad, indicating a wide distribution of distances between the nitroxide spin label and the phosphodiester backbone of the bound polyU. In short, EPR measurements can distinguish between three types of PA(<em>D</em>/<em>L</em>)‐12 molecules: (i) free in solution, (ii) bound to polyU in the coacervate at intermediate PA(<em>D</em>/<em>L</em>)‐12/polyU ratios, and (iii) bound to polyU at high PA(<em>D</em>/<em>L</em>)‐12/polyU ratios in the absence of coacervates.</p></section><section id="pro70273-sec-0014"><h3 class="pmc_sec_title">3.4. 
PA(<em>D</em>/<em>L</em>)‐12 oligomerizes upon binding RNA
</h3>
<p>To detect PA(<em>D</em>/<em>L</em>)‐12 oligomerization upon polyU binding, we carried out pulse EPR measurements. A significant reduction in echo intensity was observed in samples where the polyU concentration was sufficient for coacervate formation (Figures <a href="#pro70273-fig-0005" class="usa-link">5a</a> and <a href="#pro70273-supitem-0001" class="usa-link">S10a</a>). This reduction resulted from a fast echo decay (short phase memory time, <em>T</em>
<sub>m</sub>) due to strong spin–spin interactions arising from the high local concentrations of PA(<em>D</em>/<em>L</em>)‐12 within the coacervates, which turns them into “dark” spins in pulse EPR experiments. The echo decay curves for different polyU concentrations (Figure <a href="#pro70273-supitem-0001" class="usa-link">S10b</a>) were fitted with a stretched exponential decay, yielding the phase memory time <em>T</em>
<sub>m</sub> and the stretched exponent <em>d</em> (Figure <a href="#pro70273-supitem-0001" class="usa-link">S10c</a>). In Figure <a href="#pro70273-fig-0005" class="usa-link">5a</a>, we present the mean <em>T</em>
<sub>m</sub> value, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-4" display="inline" overflow="linebreak"><mrow><mrow><mfenced open="⟨" close="⟩"><msub><mi>T</mi><mi mathvariant="normal">m</mi></msub></mfenced></mrow></mrow></math></span>, as defined in section <a href="#pro70273-sec-0002" class="usa-link">2</a>; it shows an increase up to 0.5 mg/mL polyU, which contrasts with the trend in echo intensity and indicates that the observed echo decay is from peptides not bound to polyU. The concentration of these free peptides decreases with increasing polyU concentration. The echo decay of the bound peptides is not detected due to strong spin–spin interactions. We prepared a spin‐diluted sample with 20% PA(<em>D</em>/<em>L</em>)‐12 and 80% PA(<em>D</em>/<em>L</em>) to further support this interpretation. The spin‐diluted sample contained 100 μM total peptide and 0.35 mg/mL polyU, the condition with the highest amount of coacervates (the shaded region of Figure <a href="#pro70273-fig-0005" class="usa-link">5a</a>). In this sample, the density of spin labels was significantly reduced without changing its chemical composition or phase behavior. This spin dilution is expected to increase the <em>T</em>
<sub>m</sub> of the dark spins so they become observable. Indeed, we observed the appearance of a fast‐relaxing component in the echo decay profile (Figure <a href="#pro70273-fig-0005" class="usa-link">5b</a>), confirming the high local concentration of bound peptide within the coacervate. Note that, even with 80% dilution, the relaxation is still fast, yet detectable within the experimental time window. The decay curves were fitted to a sum of two stretched exponents, slow and fast (see Figure <a href="#pro70273-fig-0005" class="usa-link">5b</a>). The decay of the slow component was the same for both the undiluted and spin‐diluted samples, within experimental error. A clear increase of the relative contribution of the fast component is observed upon spin‐dilution.</p>
<figure class="fig xbox font-sm" id="pro70273-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bcfb/12375978/57cede416cbc/PRO-34-e70273-g008.jpg" loading="lazy" id="jats-graphic-9" height="309" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/pro70273-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(a) Mean phase memory time (<em>T</em>
<sub>m</sub>; upper red symbols, derived from <em>T</em>
<sub>m</sub> and <em>d</em> according to <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-5" display="inline" overflow="linebreak"><mrow><mrow><mfenced open="⟨" close="⟩"><msub><mi>T</mi><mi mathvariant="normal">m</mi></msub></mfenced><mo>=</mo><mfrac><msub><mi>T</mi><mi mathvariant="normal">m</mi></msub><mi>d</mi></mfrac><mi mathvariant="normal">Γ</mi><mfenced open="(" close=")"><mfrac><mn>1</mn><mi>d</mi></mfrac></mfenced></mrow></mrow></math></span>, where Γ is the gamma function) and echo intensity (lower black symbols) for 100 μM PA(<em>D</em>/<em>L</em>)‐12 in the presence of different polyU concentrations. The concentration of the sample without polyU was 220 μM and its echo intensity was scaled by a factor of 2.2. <em>T</em>
<sub>m</sub> and <em>d</em> values were determined from fitting the echo decay curves in Figure <a href="#pro70273-supitem-0001" class="usa-link">S10b</a>. The echo intensity was taken from Figure <a href="#pro70273-supitem-0001" class="usa-link">S10a</a>. The shaded area shows the region with a high local spin concentration. (b) Echo decay of 100% labeled (black) and 20% labeled (red) PA(<em>D</em>/<em>L</em>)‐12 in 0.35 mg/mL polyU. The total concentration of PA(<em>D</em>/<em>L</em>) in both samples was 100 μM. The decays were fitted to a sum of two stretched exponents (thin lines). For the undiluted sample (black decay curve), the slow relaxing component had a phase memory time <em>T</em>
<sub>s</sub> = 9.1 ± 0.035 μs and a stretched exponent <em>d</em>
<sub>s</sub> = 1.64 ± 0.016 and the fast‐relaxing component had <em>T</em>
<sub>f</sub> = 1.38 ± 0.34 μs and <em>d</em>
<sub>f</sub> = 2.05 ± 0.9. The relative contribution of the fast component was 0.25 ± 0.13. For the spin‐diluted sample (red decay curve), the slow‐relaxing component had <em>T</em>
<sub>s</sub> = 8.0 ± 0.18 μs, <em>d</em>
<sub>s</sub> = 1.24 ± 0.034, and the fast‐relaxing component had <em>T</em>
<sub>f</sub> = 0.96 ± 0.014 μs and <em>d</em>
<sub>f</sub> = 2.24 ± 0.07. The relative contribution of the fast component was 0.61 ± 0.01 for the spin‐diluted sample.</p></figcaption></figure><p>The echo detection experiments show that at 0.35 mg/mL polyU, 90% of PA(<em>D</em>/<em>L</em>)‐12 peptides are invisible. We assign these invisible peptides to peptides bound to polyU within the coacervate. Interestingly, the CW‐EPR spectra show that, under the same conditions, 80% of PA(<em>D</em>/<em>L</em>)‐12 molecules exhibit slow motion, in agreement with the loss in echo intensity due to a high local concentration. We assign molecules with slow dynamics to PA(<em>D</em>/<em>L</em>)‐12 molecules involved in cross‐linking polyU, which is needed for coacervate formation. The fast motion components below 0.35 mg/mL can be assigned to unbound peptides, whereas above 0.35 mg/mL they are from polyU‐bound peptides in the dilute solution, as reflected by different rotational correlation times (Table <a href="#pro70273-supitem-0001" class="usa-link">S1</a>).</p>
<p>Next, we performed DEER measurements to detectdimerization/oligomerization in the presence of polyU. The dependence of PA(<em>D</em>/<em>L</em>)‐12 DEER traces on polyU concentration is presented in Figure <a href="#pro70273-fig-0006" class="usa-link">6</a>. Up to 0.5 mg/mL polyU, the DEER data display only a decay without modulation (Figure <a href="#pro70273-fig-0006" class="usa-link">6a,b</a>), indicating the absence of detectable oligomerization. This decay, often referred to as background decay, is given by <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-6" display="inline" overflow="linebreak"><mrow><mrow><mi>V</mi><mo>=</mo><msup><mi>e</mi><mrow><mo>−</mo><mi mathvariant="italic">kC</mi><mo>/</mo><mi>t</mi></mrow></msup></mrow></mrow></math></span> for a homogenous three‐dimensional distribution of spins where <em>k</em> is a constant and <em>C</em> is the concentration of the visible spins (Jeschke and Polyhach <a href="#pro70273-bib-0014" class="usa-link" aria-describedby="pro70273-bib-0014">2007</a>). For small <em>kC</em> values, the background decay is linear with a slope of −<em>kC</em>. Noticeably, a decline in the <em>kC</em> values was observed up to 0.5 mg/mL (Figure <a href="#pro70273-supitem-0001" class="usa-link">S11a</a>). This reduction with increasing polyU concentration indicates a decrease in the local spin concentration of the observable spins. This observation agrees with the behavior of <em>T</em>
<sub>m</sub> and echo intensity (Figure <a href="#pro70273-fig-0005" class="usa-link">5a</a>), as only the unbound peptides contributed to the signal. We find a clear correlation between the relative population of the slow‐motion component, the loss of echo intensity, and the reduction in the DEER background decay in the range of 0–0.35 mg/mL polyU, during which the amount of coacervation increases (Figure <a href="#pro70273-supitem-0001" class="usa-link">S11b</a>). This concordance between measurements supports the consistency of the different types of data used to characterize the system, similar to earlier observations for PA(<em>L</em>) (Seal et al. <a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>).</p>
<figure class="fig xbox font-sm" id="pro70273-fig-0006"><h4 class="obj_head">FIGURE 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bcfb/12375978/c7b3e4412e0d/PRO-34-e70273-g002.jpg" loading="lazy" id="jats-graphic-11" height="586" width="709" alt="FIGURE 6"></p>
<div class="p text-right font-secondary"><a href="figure/pro70273-fig-0006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>DEER traces of PA(<em>D</em>/<em>L</em>)‐12 in the presence of different polyU concentrations (a) 0–0.1 mg/mL and (b) 0.28–1 mg/mL. All DEER trace were collected on 100 μM PA(<em>D</em>/<em>L</em>)‐12 except that at 0 mg/mL polyU, where the concentration was 220 μM. (c) Distance distribution extracted from the DEER trace with 1 mg/mL polyU using a Gaussian distance distribution. The solid black line indicates best fit, and the shaded gray region is the 2σ confidence band. (d) Comparison of the DEER traces of the sample with 1 mg/mL polyU before and after 5‐fold spin dilution. The black trace is the background decay.</p></figcaption></figure><p>Interestingly, when the polyU concentration reached 0.75 mg/mL, where coacervates are not present, a broad oscillation with an increased background slope emerged in the DEER data, which became more distinct at 1 mg/mL polyU (Figures <a href="#pro70273-fig-0006" class="usa-link">6b</a> and <a href="#pro70273-supitem-0001" class="usa-link">S12a</a>). This result indicates a general dilution of spins at high polyU/peptide ratios due to the dissolution of the coacervates, yet some PA(<em>D</em>/<em>L</em>) molecules remained close (emergence of oscillation) due to dimerization/oligomerization. The fact that PA(<em>D</em>/<em>L</em>) does not form dimers in solution and oligomer states are observed in the presence of polyU indicates that polyU promotes dimerization/oligomerization of PA(<em>D</em>/<em>L</em>), as we reported earlier for PA(<em>L</em>) (Seal et al. <a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>). Analyzing the broad oscillation using a Gaussian distance distribution yields a mean spin–spin distance of about 4.1 nm (Figure <a href="#pro70273-fig-0006" class="usa-link">6c</a>). For comparison, PA(<em>L</em>)‐12‐bound polyU exhibited a narrower distance distribution with a peak at 3.4 nm (Figure <a href="#pro70273-supitem-0001" class="usa-link">S12b</a>) (Seal et al. <a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>).</p>
<p>The broad distribution of PA(<em>D</em>/<em>L</em>)‐12 could result from multiple conformations of PA(<em>D</em>/<em>L</em>)‐12 dimers or oligomers bound to polyU. To distinguish between dimers and higher‐order oligomers, we measured DEER on samples with 1 mg/mL polyU and only 20% labeled peptides while maintaining the total peptide concentration at 100 μM. This dilution resulted in the disappearance of the modulation, as shown in Figure <a href="#pro70273-fig-0006" class="usa-link">6d</a>. If PA(<em>D</em>/<em>L</em>) forms dimers, the observed modulation depth (9.6%) should decrease to 9.6 × 0.04 = 0.38%. In contrast, trimers would be expected to reduce the modulation depth to 9.6 × (0.008 + 0.096) = 1.0%. As the difference between these modulation depths is small, we cannot unambiguously exclude the possibility that oligomers other than dimers are present. Nevertheless, a large population of higher‐order oligomers is less likely based on the modulation depth (Figure <a href="#pro70273-fig-0006" class="usa-link">6c</a>) and the long <em>T</em>
<sub>m</sub> of the sample (Figure <a href="#pro70273-supitem-0001" class="usa-link">S10b</a>).</p></section><section id="pro70273-sec-0015"><h3 class="pmc_sec_title">3.5. Syndiotactic peptide retains partial helicity in the presence of RNA: Atomistic MD simulations</h3>
<p>To further explore the capacity of syndiotactic peptides to form dimers and facilitate RNA cross‐linking, we turned to atomistic MD simulations. We constructed models for PA(<em>L</em>) and PA(<em>D</em>/<em>L</em>) dimers, and each dimer was simulated in isolation and in the presence of two molecules of 20‐nucleotide polyU. In the absence of polyU, PA(<em>D</em>/<em>L</em>) is predominately disordered, though with some residual helicity in the C‐terminus, and adopts more extended conformations than PA(<em>L</em>), as illustrated by its larger radius of gyration (<em>R</em>
<sub>
<em>g</em>
</sub>) (Figure <a href="#pro70273-fig-0007" class="usa-link">7a,b</a>). The dimerization potential of PA(<em>D</em>/<em>L</em>) in the absence of polyU is significantly lower than PA(<em>L</em>), as indicated by higher RMSD values for conformational stability (Figure <a href="#pro70273-supitem-0001" class="usa-link">S13</a>). Moreover, the tendency to dimerize increases for both PA(<em>D</em>/<em>L</em>) and PA(<em>L</em>) in the presence of polyU, consistent with the experimental observation that RNA promotes dimerization. Additionally, the presence of RNA molecules results in compaction of PA(<em>D</em>/<em>L</em>) and an increase in its right‐handed α‐helical content.</p>
<figure class="fig xbox font-sm" id="pro70273-fig-0007"><h4 class="obj_head">FIGURE 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bcfb/12375978/554de9bd939d/PRO-34-e70273-g001.jpg" loading="lazy" id="jats-graphic-13" height="672" width="709" alt="FIGURE 7"></p>
<div class="p text-right font-secondary"><a href="figure/pro70273-fig-0007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Computational characterization of the effect of chirality on peptides conformations and interactions with RNA. (a) The conformational ensemble of the dimeric PA(<em>D</em>/<em>L</em>) in the absence (light blue) and presence (dark blue) of RNA molecules is analyzed by its radius of gyration (<em>R</em>
<sub>
<em>g</em>
</sub>, left) and degree of α‐helicity (middle). While PA(<em>D</em>/<em>L</em>) is mostly disordered in solution, the presence of RNA promotes the formation of more compact conformations and an increase in α‐helical content despite its alternating amino acid chirality. (b) Like panels in (a) but for the PA(<em>L</em>) in the absence (light green) and presence (dark green) of RNA. Selected conformations of the dimers in the presence of two polyU molecules (shown in red) illustrate the interactions between RNA and the dimeric peptides (right panel). (c) Analysis of the probability of adopting an α‐helix conformation per residue (in the presence or absence of RNA) for dimeric PA (upper panel) and monomeric PA (lower panel). (d) The interaction energy (sum of non‐bonded Lennard‐Jones and electrostatic energies) between the peptides and RNA shows that PA(<em>L</em>) interacts more strongly with the RNA backbone (orange) than with uracil (pink).</p></figcaption></figure><p>The enhanced α‐helicity of PA(<em>D</em>/<em>L</em>) in the presence of RNA is localized primarily to the C‐terminal subdomain of PA(<em>D</em>/<em>L</em>) (Figure <a href="#pro70273-fig-0007" class="usa-link">7a,c</a>). To investigate the origin of this partial helicity in PA(<em>D</em>/<em>L</em>), we compared its monomeric and dimeric variants in the absence of RNA. While dimeric PA(<em>D</em>/<em>L</em>) showed clear α‐helicity at the C‐terminus, this signal was largely lost in the monomeric form (Figure <a href="#pro70273-fig-0007" class="usa-link">7c</a>), suggesting that dimerization helps stabilize partial folding in this region. In contrast, PA(<em>L</em>) retained helicity in both termini regardless of its oligomeric state (Figure <a href="#pro70273-fig-0007" class="usa-link">7c</a>). Solvent‐accessible surface area (SASA) analysis revealed that specific residues in the C‐terminal region (e.g., Leu20, Ala21, Ile24, Leu25) are buried in the dimer but tend to be more exposed in the monomer (Figure <a href="#pro70273-supitem-0001" class="usa-link">S14</a>), highlighting their role in stabilizing the dimeric interface. These residues are also part of the hydrophobic interface involved in PA(<em>L</em>) dimerization, suggesting that specific intermolecular interactions help stabilize partial helicity even in the syndiotactic peptide.</p>
<p>We then asked whether specific interactions contribute to the sequence‐specific stabilization of helicity in the C‐terminal region, potentially explaining why partial helicity is observed there, but not in the N‐terminal region, despite its alternating backbone chirality, which is expected to destabilize helical secondary structure. Pairwise energy decomposition revealed strong interactions between <em>L</em>‐Arg22 or <em>D</em>‐Arg23 and Glu26 (Figure <a href="#pro70273-supitem-0001" class="usa-link">S15</a>), which are ideally positioned to interact across one helical turn. The C‐terminal region also carries a net positive charge of (+3), compared to the N‐terminal region, which is overall charge‐neutral due to balanced positive and negative residues. To examine the functional role of these interactions, we studied the effect of some point mutations on the helicity of the PA(<em>D</em>/<em>L</em>). Substituting Arg23 with Glu effectively abolished helicity in monomeric PA(<em>D</em>/<em>L</em>), while mutation of Arg22 had a more modest effect (Figures <a href="#pro70273-fig-0007" class="usa-link">7c</a> and <a href="#pro70273-supitem-0001" class="usa-link">S16</a>). This result indicates that the Arg23‐Glu26 pair is a critical intramolecular stabilizing interaction that enables partial helicity to persist in a sequence otherwise biased toward disorder.</p>
<p>These results suggest that structures reminiscent of right‐handed α‐helices are accessible to some syndiotactic sequences under the right conditions. Accordingly, some sequences may be more robust to drastic changes in chiral patterns than others, as shown by the difference in helicity observed between the N‐ and C‐termini of the syndiotactic PA(<em>D</em>/<em>L</em>) peptide. Moreover, this phenomenon causes the structure of the syndiotactic peptide to more closely resemble that of the isotactic one (Figure <a href="#pro70273-fig-0007" class="usa-link">7b</a>). Note, however, that the interface between the two monomers within the dimer are not identical, with PA(<em>D</em>/<em>L</em>) dimers having somewhat more pairwise intermolecular interactions (Figure <a href="#pro70273-supitem-0001" class="usa-link">S15</a>). Furthermore, PA(<em>L</em>) dimers interact more strongly with RNA, as evidenced by the lower interaction energy. In both cases, the protein–RNA interactions are stronger with the RNA backbone than with the uracil bases (Figure <a href="#pro70273-fig-0007" class="usa-link">7d</a>). SASA analysis further revealed that solvent‐exposed residues are likely involved in RNA binding. Notably, several arginine residues, particularly Arg1, Arg4, and Arg19, showed consistently high solvent accessibility, suggesting accessibility for RNA interactions. These findings suggest that electrostatic interactions between positively charged peptide residues and RNA are partially conserved in the syndiotactic peptide. However, differences in the spatial distribution of negative charges, particularly increased exposure of acidic residues in PA(<em>D</em>/<em>L</em>), may contribute to distinct RNA binding specificity. This interpretation supports the idea that folding facilitates interaction with RNA by shielding repulsive negative–negative contacts through stabilized intra‐ and inter‐molecular interactions.</p></section><section id="pro70273-sec-0016"><h3 class="pmc_sec_title">3.6. Distinct molecular forces govern the condensates of the syndiotactic and isotactic HhH peptides: Coarse‐grained MD simulations</h3>
<p>To investigate how the molecular interactions, which were identified in the all‐atom molecular simulations for the PA(<em>L</em>) and the PA(<em>D</em>/<em>L</em>) peptides, shape the properties of their coacervates, we performed coarse‐grained (CG) molecular simulations. The CG modeling allowed us to significantly increase the number of peptides and polyU molecules present in the simulations and directly observe the phase separation process while maintaining computationally accessible time scales. We studied three types of peptides that differ in their degree of structural order: PA(<em>L</em>), PA(<em>D</em>/<em>L</em>), and PA(<em>Dis</em>). PA(<em>L</em>) is modeled in the CG simulations as a dimeric peptide that is fully folded, and PA(<em>D</em>/<em>L</em>) is modeled as a dimeric peptide that is partially folded with a greater <em>R</em>
<sub>
<em>g</em>
</sub> than that of PA(<em>L</em>), consistent with the atomistic MD simulations. As a control system, PA(<em>Dis</em>) is a disordered monomeric peptide. In this model, each peptide residue is represented by a single bead centered on its Cα, and each RNA nucleotide is modeled using three beads per nucleotide (see section <a href="#pro70273-sec-0002" class="usa-link">2</a> for details). Stability and energetic analyses were conducted on coacervates formed between 120 peptides and four 100‐nucleotide polyU molecules.</p>
<p>In the presence of RNA, PA(<em>L</em>) showed the highest coacervate stability, while PA(<em>Dis</em>) was the least stable, as indicated by its lower critical temperature (<em>T</em>
<sub>c</sub>) (Figure <a href="#pro70273-fig-0008" class="usa-link">8a</a>). The lower <em>T</em>
<sub>c</sub> of PA(<em>Dis</em>), despite its high conformational heterogeneity, suggests that structured domains can promote phase separation, and that evolutionary pressure to stabilize a coacervate may, in turn, promote the emergence of structured domains. Removing RNA had an overall destabilizing effect on the coacervate, with a more pronounced reduction in <em>T</em>
<sub>c</sub> for the partially folded PA(<em>D</em>/<em>L</em>) peptides than for fully folded PA(<em>L</em>).</p>
<figure class="fig xbox font-sm" id="pro70273-fig-0008"><h4 class="obj_head">FIGURE 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bcfb/12375978/b161cfb71229/PRO-34-e70273-g005.jpg" loading="lazy" id="jats-graphic-15" height="702" width="709" alt="FIGURE 8"></p>
<div class="p text-right font-secondary"><a href="figure/pro70273-fig-0008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Coarse‐grained modeling of collective phase behavior. (a) Phase diagrams for the PA(<em>L</em>), PA(<em>D</em>/<em>L</em>), and PA(<em>Dis</em>) coacervates both in the presence (dark colors) and absence (lighter colors) of RNA. Critical temperatures are shown in white for each system. Below this temperature, phase separation exists, and there are two coexisting phases: a peptide‐rich phase with high density (right arm) and a peptide‐poor phase with low density (left arm). All subsequent comparisons are performed at <em>T = 0.9T<sub>c</sub>
</em>. (b) Changes in <em>R<sub>g</sub>
</em> and helicity of the peptides upon the transition from the dilute phase (bulk) to the dense phase (coacervate). Each peptide is designated with the same color as in panel (a). (c) Mean diffusion coefficient for the peptides inside the coacervates in the presence or absence of RNA. Overall peptide–peptide (d) and peptide–RNA (e) interaction energy in the coacervate phase. (f) selected snapshots of the coacervates in the presence of RNA (red). PA(<em>L</em>) coacervates are the more open (top, dark green), whereas PA(<em>D</em>/<em>L</em>) and PA(<em>Dis</em>) coacervates are increasingly more compact (middle and bottom, dark blue and gray, respectively).</p></figcaption></figure><p>In the coacervate, PA(<em>L</em>) peptides largely retained their folded dimeric structure, with only a slight decrease in helicity and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-7" display="inline" overflow="linebreak"><mrow><mrow><msub><mi>R</mi><mi>g</mi></msub></mrow></mrow></math></span>, whereas PA(<em>Dis</em>) showed minimal helicity changes but a notable increase in <em>R<sub>g</sub>
</em>. PA(<em>D</em>/<em>L</em>), in contrast, exhibited both an increase in helicity and a decrease in <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="jats-math-8" display="inline" overflow="linebreak"><mrow><mrow><msub><mi>R</mi><mi>g</mi></msub></mrow></mrow></math></span> (Figure <a href="#pro70273-fig-0008" class="usa-link">8b</a>). These results are consistent with the atomistic simulations of the PA(<em>D</em>/<em>L</em>) (Figure <a href="#pro70273-fig-0007" class="usa-link">7a</a>), which demonstrated the increase in helicity in PA(<em>D</em>/<em>L</em>) in the presence of RNA, whereas PA(<em>L</em>) maintains its helicity independently (Figure <a href="#pro70273-fig-0007" class="usa-link">7b</a>). It is also consistent with the EPR results showing that for PA(<em>D</em>/<em>L</em>), dimerization takes place only in the presence of RNA. The consequences of the lower helicity of PA(<em>D</em>/<em>L</em>) in the presence of RNA, as compared to PA(<em>L</em>) are demonstrated in the distance distribution between the C<sub>β</sub> atom of Glu12 in one monomer to the corresponding C<sub>β</sub> atom in the second monomer, computed from the atomistic MD calculations (Figure <a href="#pro70273-supitem-0001" class="usa-link">S12c</a>). The peak of the distance distribution for PA(<em>D</em>/<em>L</em>) is at 2.5 nm, and it is significantly broader than that of PA(<em>L</em>), which peaks at 2.2 nm. This trend reproduces the DEER distance distributions obtained for PA(<em>L</em>)‐12 and PA(<em>D</em>/<em>L</em>)‐12 shown in Figure <a href="#pro70273-supitem-0001" class="usa-link">S12b</a>. In this case, the peak for PA(<em>D</em>/<em>L</em>)‐12 is at 4.2 nm and that of PA(<em>L</em>)‐12 at 3.2 nm, having a significantly narrower width. The longer distances in the experimental DEER arise from the additional length of the spin label, estimated to be 0.79 nm, and the larger width most probably arises from the additional flexibility introduced by the spin label tether.</p>
<p>We also carried out CD (circular dichroism) spectroscopic measurements to obtain additional experimental evidence for the formation of some helical structure in PA(<em>D</em>/<em>L</em>) and PA(<em>L</em>) upon binding to RNA (Figure <a href="#pro70273-supitem-0001" class="usa-link">S17</a>). For PA(<em>L</em>), the presence of some right‐handed α‐helix is apparent in solution and promoted by coacervation with RNA, as predicted by the atomistic MD simulations and consistent with our previous EPR analysis. For PA(<em>D</em>/<em>L</em>) in the presence of either 0.35 or 1 mg/mL polyU, a modest degree of order seems to have been acquired, but the CD difference data do not yield unambiguous conclusions regarding helix formation.</p>
<p>Chirality cannot be explicitly encoded in a CG model. To account for the structural differences observed between the isotactic and syndiotactic variants, we therefore built the CG model based on insights from the atomistic simulations of PA, which showed that these variants differ in both folding and oligomerization propensities. We asked how condensates composed of folded dimers compare in their biophysical properties to those formed by disordered monomers. The degree of disorder of the modeled peptides in the CG model affects their diffusivity in the dense phase. In condensates in the presence of RNA, the PA(<em>L</em>) peptide has a lower diffusion coefficient in the dense phase, whereas PA(<em>D</em>/<em>L</em>) and PA(<em>Dis</em>) peptides have higher diffusion coefficients compared to condensates with RNA (Figure <a href="#pro70273-fig-0008" class="usa-link">8c</a>). These findings are consistent with atomistic simulations, indicating that RNA interacts with PA(<em>L</em>) and PA(<em>D</em>/<em>L</em>) differently, resulting in a more dynamic and diffusive PA(<em>D</em>/<em>L</em>)/RNA coacervate, consistent with a lower degree of inter‐RNA cross linking.</p>
<p>To understand the differences between the coacervates formed by the three peptides, we computed the energies of the network peptide–peptide and all peptide–RNA interactions (PPI and PRI) (Figure <a href="#pro70273-fig-0008" class="usa-link">8d,e</a>) in the coacervates. PPI and PRI energies are found to be inversely correlated. In coacervates formed with RNA, the PRI interactions follow the trend PA(<em>Dis</em>) &gt; PA(<em>D</em>/<em>L</em>) &gt; PA(<em>L</em>), which is inversely proportional to the coacervates stability as reflected by the <em>T</em>
<sub>c</sub> values (Figure <a href="#pro70273-fig-0008" class="usa-link">8e</a>). In the coacervates, the PPI interaction trend is different and follows PA(<em>L</em>) ≥ PA(<em>D</em>/<em>L</em>) &gt; PA(<em>Dis</em>) (Figure <a href="#pro70273-fig-0008" class="usa-link">8d</a>). This indicates that it is not the strength of the interaction with the RNA that dictates the coacervates stability but rather the PPI that allows the peptide to act as an efficient cross‐linker, which is related to its dimer's structural properties. Interestingly, the addition of RNA to coacervates of only peptides reduced the PPI for PA(<em>D</em>/<em>L</em>) but increased it for PA(<em>L</em>). This suggests divergent RNA‐mediated interaction mechanisms in PA(<em>L</em>) and PA(<em>D</em>/<em>L</em>) coacervates, where a reduction in the number of PPI‐mediated cross‐links in PA(<em>D</em>/<em>L</em>) takes place. Consequently, peptides form more diffusive, less stable coacervates, whereas PA(<em>L</em>) coacervates are less dynamic and more stable.</p>
<p>The different strengths of PPI and PRI for PA(<em>L</em>) and PA(<em>D</em>/<em>L</em>) based on the CG‐MD simulations can explain the experimentally observed reentrant phase behavior of PA(<em>D</em>/<em>L</em>) (Figure <a href="#pro70273-fig-0002" class="usa-link">2b</a>). For PA(<em>D</em>/<em>L</em>), increasing RNA concentrations likely encourage competition between peptide–peptide binding and RNA binding, disrupting crucial PPIs that are essential to support cross‐linking with RNA chains and thus phase separation. Conversely, in PA(<em>L</em>) coacervates, RNA appears to facilitate peptide networks that yield a less compact, more open coacervate (Figure <a href="#pro70273-fig-0008" class="usa-link">8d,e</a>) and that may be better able to accommodate additional RNA molecules without out‐competing the PPIs that are essential for cross‐linking.</p></section></section><section id="pro70273-sec-0017"><h2 class="pmc_sec_title">4. DISCUSSION</h2>
<p>PA(<em>L</em>) and PA(<em>D</em>/<em>L</em>) exhibit similarities and differences. Concerning similarities, both peptides bind to RNA and form coacervates—processes that promote peptide dimerization. In addition, the large majority of PA(<em>L</em>) and PA(<em>D</em>/<em>L</em>) molecules are in the coacervates, with 95% and 80% of molecules, respectively. As for differences, while PA(<em>L</em>) exhibits significant α‐helix propensity and dimer formation in solution, PA(<em>D</em>/<em>L</em>) is highly disordered, and dimerization is not detected in the absence of RNA. Although both peptides adopt compact, hairpin‐like conformations when bound to RNA, only for PA(<em>L</em>) does this occur with the involvement of both N‐ and C‐terminal α‐helices. For PA(<em>D</em>/<em>L</em>), all‐atom simulations suggest that α‐helix formation is restricted to the C‐terminus. Differences are found in the phase diagram too: the polyU concentration range for which coacervates are formed is smaller for PA(<em>D</em>/<em>L</em>) than for PA(<em>L</em>). At 1 mg/mL polyU, PA(<em>D</em>/<em>L</em>) coacervates are fully dissolved, whereas PA(<em>L</em>) coacervates persist even at 2 mg/mL polyU. Finally, PA(<em>L</em>) and PA(<em>D</em>/<em>L</em>) differ in the strength of their peptide–peptide interactions and their peptide–RNA interactions. The similarities provide surprising insights into the emergence of chiral control, while the differences are clues to the rules governing PA coacervation.</p>
<section id="pro70273-sec-0018"><h3 class="pmc_sec_title">4.1. Dimer stability is a handle for tuning coacervate properties</h3>
<p>We propose that differences in the flexibility, stability, and cross‐linking efficiency of PA(<em>L</em>) and PA(<em>D</em>/<em>L</em>) dimers and monomers can account for the differences in phase behavior of these two peptides. For a consistent account of these differences, we postulate that (i) RNA binding promotes dimerization of both peptides; (ii) coacervation requires peptide‐mediated cross‐links between different RNA chains, which are experimentally observable as the relative population of the slow‐motion EPR component; (iii) both intra‐ and inter‐polyU cross‐linking occur due to the binding of PA to polyU, with the relative populations depending on the PA/polyU ratio; (iv) PA(<em>D</em>/<em>L</em>) dimers are less stable than PA(<em>L</em>) dimers in solution and are, therefore, less efficient polyU cross‐linkers because of competition with monomer binding. Moreover, the stronger peptide‐RNA interaction of PA(<em>D</em>/<em>L</em>), along with its somewhat weaker peptide–peptide interactions found in the coacervates by CG‐MD, reduce its cross‐linking efficiency; (v) while monomers and dimers can bind polyU, dimers are better cross‐linkers than monomers because of the patches of positive charge. Points (i), (ii), and (iv) are based on experimental evidence and MD calculations, and (iii) and (v) are reasonable consequences of these experimental data.</p>
<p>The phase behavior of PA(<em>L</em>) and PA(<em>D</em>/<em>L</em>) can be divided into three regions based on the PA/polyU ratio (Figure <a href="#pro70273-fig-0009" class="usa-link">9</a>). Region I is characterized by a high PA/polyU ratio, where no coacervates are formed (Figure <a href="#pro70273-fig-0009" class="usa-link">9b,e</a>). In this region, a large excess of peptide over polyU is present, and multiple PA molecules bind to individual RNA molecules. Consequently, over‐coated RNA molecules repel each other (charge inversion), and inter‐polyU cross‐links are rare. Intra‐polyU cross‐linking, however, cannot be excluded. The high local concentration of bound PA leads to fast spin–spin relaxation and makes the labeled PA molecules effectively “invisible” and reduces the DEER background slope (e.g., Figure <a href="#pro70273-fig-0006" class="usa-link">6a</a>). In this region, the DEER data report on only the “visible” free peptides in the solution. Since RNA binding promotes dimerization, RNA‐bound PA molecules are expected to be enriched in dimers relative to the unbound PA molecules. For PA(<em>L</em>) (Figure <a href="#pro70273-fig-0009" class="usa-link">9b</a>), this region constitutes a mixture of dimers and monomers bound to RNA and existing freely in solution.</p>
<figure class="fig xbox font-sm" id="pro70273-fig-0009"><h4 class="obj_head">FIGURE 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/bcfb/12375978/9e75d31aea6e/PRO-34-e70273-g007.jpg" loading="lazy" id="jats-graphic-17" height="543" width="709" alt="FIGURE 9"></p>
<div class="p text-right font-secondary"><a href="figure/pro70273-fig-0009/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Schematic representations of phase separation by PA(<em>L</em>) and PA(<em>D</em>/<em>L</em>). (a) Representation of the PA(<em>L</em>) and PL(<em>D</em>/<em>L</em>) monomers, the PA(<em>L</em>) and PA(<em>D</em>/<em>L</em>) dimers, and polyU. (b, e) Region I, characterized by a high PA/polyU ratio with monomer binding and intra‐polyU cross‐linking, resulting in a charge inversion and no coacervates. (c, f) Region II is characterized by an optimal PA/polyU ratio resulting in inter‐polyU cross‐linking and coacervate formation. (d, g) Region III is characterized by a low PA/polyU ratio with no coacervate formation due to insufficient binding affinity to overcome charge repulsion between polyU molecules. Middle panels (b–d) depict PA(<em>L</em>), while the lower panels (e, g) depict PA(<em>D</em>/<em>L</em>). The bracket in (d) indicates proposed reentrant behavior (not observed).</p></figcaption></figure><p>With increasing polyU concentration, the PA/polyU ratio reaches region II, where coacervates appear as inter‐RNA cross‐linking becomes optimal (Figure <a href="#pro70273-fig-0009" class="usa-link">9c,f</a>), featuring the required balance between peptide–peptide and peptide–RNA interaction strengths as shown by the MD calculations. For PA(<em>D</em>/<em>L</em>), region II spans 0.2–0.5 mg/mL polyU, whereas for PA(<em>L</em>) it ranges from 0.35 mg/mL to at least 2 mg/mL polyU. The higher peptide/polyU ratio for PA(<em>D</em>/<em>L</em>) coacervation appearance is likely because of its higher flexibility (lower structure content), leading to a lower propensity to form a dimer and stronger peptide–RNA interactions compared to peptide–peptide interactions, determined from the MD coarse‐grained simulations. The less efficient charge inversion of monomer‐bound polyU versus dimer‐bound polyU contributes as well. As there are no PA(<em>D</em>/<em>L</em>) dimers in solution (without RNA), monomers bind RNA first, and only upon binding of a second monomer can dimers form in large numbers and potentially mediate cross‐links. This contrasts with PA(<em>L</em>), where pre‐formed dimers can bind RNA, resulting in more efficient charge balance and onset of coacervation at a lower peptide/polyU ratio for PA(<em>D</em>/<em>L</em>). This difference in the ability to form dimers accounts for the somewhat lower maximum percentage of peptide involved in cross‐linking for PA(<em>D</em>/<em>L</em>) relative to PA(<em>L</em>) (80% vs. 95% slow‐motion, respectively). For PA(<em>L</em>), we did observe dimerization within coacervates at high polyU concentrations (Seal et al. <a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>), which reduced the peptide local concentration. However, for PA(<em>D</em>/<em>L</em>), dimerization was also observed at a high polyU concentration, but when the coacervates have already dissolved. Nevertheless, this implies that dimers were also present in the coacervates, where the peptide/polyU ratio is higher.</p>
<p>Region III is characterized by a low PA/polyU ratio and the disappearance of coacervates (Figure <a href="#pro70273-fig-0009" class="usa-link">9d,g</a>). Such reentrant behavior was observed for PA(<em>D</em>/<em>L</em>) coacervates but was not detected for PA(<em>L</em>) (Figure <a href="#pro70273-fig-0009" class="usa-link">9d</a>) (Seal et al. <a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>). This result can also be related to the cross‐linking efficiency of the two peptides and their different peptide–peptide and peptide–RNA interaction strengths. When the RNA concentration increases relative to the optimal peptide concentration, the lower cross‐linking efficiency of PA(<em>D</em>/<em>L</em>) comes into play, as the peptide–peptide interaction strength is too weak relative to the peptide–RNA interaction strength to maintain the necessary degree of cross‐linking for coacervation, resulting in reentrant phase behavior.</p>
<p>The observation that coacervation of PA(<em>D</em>/<em>L</em>) starts from a solution containing monomers and that RNA promotes dimerization suggests that monomers are less competent to cross‐link RNA molecules. This interpretation is supported by the low <em>T</em>
<sub>c</sub> found in the CG‐MD for the monomeric disordered peptides, PA(<em>Dis</em>). Moreover, atomistic molecular dynamics simulations of PA(<em>L</em>) with RNA indicated that the interactions involved are largely non‐specific and mediated by side chains. This is roughly consistent with the <sup>31</sup>P Mims ENDOR results for PA(<em>D</em>/<em>L</em>), showing a broad distribution of distances. This result is also consistent with the DEER data on RNA‐bound peptides that show a significantly broader distance distribution, indicating higher disorder, making PA(<em>D</em>/<em>L</em>) an even less efficient cross‐linker. We conclude that the interplay between dimer formation, dimer stabilization upon RNA binding, and the dimer's efficiency as an RNA cross‐linker shape the phase diagram of HhH coacervation. This interplay is illustrated in the CG‐MD simulations showing that dimerization and degree of helicity affect the stability of the coacervates via modulation of the interaction strength between peptides and RNA. As such, the HhH peptide has a clear handle—the <em>K</em>
<sub>D</sub> of dimerization—through which evolution can tune the phase properties of the coacervate, with higher affinities yielding potentially significant gains in coacervate stability but lower affinities allowing access to the coacervate state at lower concentrations of RNA.</p></section><section id="pro70273-sec-0019"><h3 class="pmc_sec_title">4.2. 
RNA binding, coacervation, and the emergence of chiral control</h3>
<p>The isotactic and syndiotactic variants of PA reveal an unexpected preservation of form (dimerization, potential right‐handed α‐helical folding, bent conformations) and function (RNA binding, primarily via interaction with the phosphodiester backbone, and coacervation). For comparison, function is not preserved upon scrambling the PA(<em>L</em>) sequence, which results in a dramatic shift to precipitation with polyU (Longo et al. <a href="#pro70273-bib-0020" class="usa-link" aria-describedby="pro70273-bib-0020">2020</a>). Chiral inversions may be less disruptive than sequence shuffles because the hydrophobic/polar/charged patterning—a determinant of protein folding and coacervation—can be retained, even if secondary structure propensities are disrupted. As only a few sequences derived from natural protein families have been studied in both their isotactic and syndiotactic forms, the extent to which these data reflect a property unique to the HhH motif or a property of primitive sequences more broadly is unknown. However, just as early efforts of prebiotic protein design (Longo et al. <a href="#pro70273-bib-0021" class="usa-link" aria-describedby="pro70273-bib-0021">2013</a>) were followed up by several studies demonstrating the robustness of alphabet simplification in other systems (Despotović et al. <a href="#pro70273-bib-0005" class="usa-link" aria-describedby="pro70273-bib-0005">2020</a>; Giacobelli et al. <a href="#pro70273-bib-0010" class="usa-link" aria-describedby="pro70273-bib-0010"><sup>2022</sup></a>; Kimura and Akanuma <a href="#pro70273-bib-0017" class="usa-link" aria-describedby="pro70273-bib-0017"><sup>2020</sup></a>; Longo et al. <a href="#pro70273-bib-0020" class="usa-link" aria-describedby="pro70273-bib-0020"><sup>2020</sup></a>; Yagi et al. <a href="#pro70273-bib-0035" class="usa-link" aria-describedby="pro70273-bib-0035"><sup>2021</sup></a>), we anticipate the discovery of similar continuities across chiral patterns, particularly for peptides with simple functions such as coacervation. Taken together, the evidence presented here suggests that a transition from heterochiral to homochiral peptides (if such a transition occurred) could have adapted sequences already in play rather than displacing them entirely.</p>
<p>Coacervation as a form of liquid–liquid phase separation (LLPS) has long been considered an important site of (pre‐) biological evolution, dating back to proposals from Oparin himself (Oparin and Morgulis <a href="#pro70273-bib-0026" class="usa-link" aria-describedby="pro70273-bib-0026">1938</a>). Besides conferring an aspect of individuality—a separation between the system and the surrounding environment—coacervates can organize and tune chemical reactions (Smokers et al. <a href="#pro70273-bib-0031" class="usa-link" aria-describedby="pro70273-bib-0031">2024</a>). We have recently argued that the physicochemical environment within a coacervate promotes the emergence of folded domains from flexible peptides (Seal et al. <a href="#pro70273-bib-0030" class="usa-link" aria-describedby="pro70273-bib-0030">2022</a>). By allowing access to conformational and oligomeric states with marginal stability, these structures can be more readily recruited by evolution. Similar arguments have been made for other environmental contexts, such as at high salt (halophile) conditions (Longo et al. <a href="#pro70273-bib-0021" class="usa-link" aria-describedby="pro70273-bib-0021">2013</a>; Longo and Blaber <a href="#pro70273-bib-0019" class="usa-link" aria-describedby="pro70273-bib-0019"><sup>2014</sup></a>) or through the action of chemical chaperones like polyamines and di‐cations (Despotović et al. <a href="#pro70273-bib-0005" class="usa-link" aria-describedby="pro70273-bib-0005">2020</a>). But only in the case of phase‐separating peptides does the peptide contribute to the construction of its own environmental context. The result is a kind of “molecular self‐domestication” where phase‐separating peptides help create the environmental contexts that favor their own elaboration into folded domains that, in turn, may further stabilize the coacervate.</p>
<p>While in line with the coacervate‐as‐chaperone framing above, the PA(<em>D</em>/<em>L</em>) results presented here expand the potential relevance of coacervation and the interaction between peptides and nucleic acids more generally (Fried et al. <a href="#pro70273-bib-0008" class="usa-link" aria-describedby="pro70273-bib-0008">2022</a>) in early protein evolution. First, PA(<em>D</em>/<em>L</em>) should have a much less funneled energy landscape than PA(<em>L</em>), given that we do not observe PA(<em>D</em>/<em>L</em>) oligomers in aqueous solution. Second, by demonstrating that the coacervate context and RNA binding can buffer evolutionary transitions even when the rules of folding are altered, as in the case for isotactic and syndiotactic variants of the same sequence, for which the accessible secondary structure types are different. We refer to this as a “buffered transition” because various aspects of continuity in form are achieved due to the influence of the environmental context and without the need for compensatory mutations to re‐establish these properties. We propose that buffering of this type can confer a smoother evolutionary landscape, one more amenable to exploration and thus less dependent on contingency.</p></section></section><section id="pro70273-sec-0020"><h2 class="pmc_sec_title">5. CONCLUSIONS</h2>
<p>We compared the structure and phase behavior of an isotactic primordial peptide, PA(<em>L</em>), with its syndiotactic analog, PA(<em>D</em>/<em>L</em>), in the presence and absence of RNA to better understand whether coacervation and RNA binding can support the evolution of complex structures. We found that both chiral forms of PA form coacervates, with PA(<em>D</em>/<em>L</em>) exhibiting a lower propensity for coacervation. We correlated differences in phase behavior to dimer stability, the propensity for α‐helical folding, and interplay between PA–PA and PA–RNA interaction strength. Our results indicate that peptide dimers promote the RNA cross‐linking that drives coacervation and clarify the role of dimer stability in tuning coacervate properties. We propose that the emergence of chiral control could have been buffered by coacervation and RNA binding, as demonstrated by the preservation of PA form and function between radically different patterns of amino acid chirality. Consequently, an aspect of evolutionary continuity between primitive heterochiral peptides and modern homochiral domains may be possible. Realizing unexpected continuities—such as the persistence of form (dimerization, α‐helicity) and function (RNA binding, coacervation) between a syndiotactic peptide and its isotactic counterpart—may be a matter of environmental context. In the case of phase‐separating peptides, this environmental context may be one in which the peptide itself participated in creating. This work contributes to a growing body of evidence that simple and complex states in protein evolution are readily traversable.</p></section><section id="pro70273-sec-0025"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTIONS</h2>
<p>
<strong>Manas Seal:</strong> Conceptualization; investigation; formal analysis; visualization; writing – review and editing. <strong>Ilan Edelstein:</strong> Investigation; writing – review and editing; formal analysis; visualization. <strong>Yosef Scolnik:</strong> Methodology; investigation. <strong>Orit Weil‐Ktorza:</strong> Resources. <strong>Norman Metanis:</strong> Resources; writing – review and editing. <strong>Yaakov Levy:</strong> Conceptualization; project administration; supervision; writing – review and editing; funding acquisition. <strong>Liam M. Longo:</strong> Project administration; writing – original draft; conceptualization. <strong>Daniella Goldfarb:</strong> Conceptualization; writing – original draft; project administration; supervision; funding acquisition.</p></section><section id="pro70273-sec-0022"><h2 class="pmc_sec_title">CONFLICT OF INTEREST STATEMENT</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec23"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pro70273-supitem-0001"><div class="caption p"><p>
<strong>Data S1.</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375978/bin/PRO-34-e70273-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70273-s001.docx</a><sup> (4MB, docx) </sup>
</div></div></section></section><section id="pro70273-sec-0021" class="ack"><h2 class="pmc_sec_title">ACKNOWLEDGMENTS</h2>
<p>This work was funded by the Israel Science Foundation (grant numbers 2253/18, 2072/22 and 1388/22). L.M.L. acknowledges Dragana Despotović for insightful discussions during the preparation of this manuscript. We thank Dr. Arina Dalaloyan and Dr. Alexey Bogdanov for their help with data collection. We also acknowledge the contributions of Mr. Raphael Samson, who assisted with some of the EPR simulations. This research was made possible in part by the historic generosity of the Harold Perlman Family and by support from the Helen and Martin Kimmel Institute for Magnetic Resonance Research (D.G.). Y.L. acknowledges research grants from the Estate of Gerald Alexander and the Donald Gordon Foundation. Y.L. holds the Morton and Gladys Pickman professional chair in Structural Biology.</p></section><section id="notes1"><p>

Seal M, Edelstein I, Scolnik Y, Weil‐Ktorza O, Metanis N, Levy Y, et al. 
RNA binding and coacervation promote preservation of peptide form and function across the heterochiral–homochiral divide. Protein Science. 2025;34(9):e70273. 10.1002/pro.70273

</p>
<section id="pro70273-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="pro70273-note-0001"><p>Manas Seal and Ilan Edelstein contributed equally to this study.</p></div>
<div class="fn p" id="pro70273-note-0002"><p>
<strong>Review Editor:</strong> Nir Ben‐Tal</p></div>
</div></section></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Yaakov Levy, Email: koby.levy@weizmann.ac.il.</p>
<p>Liam M. Longo, Email: llongo@elsi.jp.</p>
<p>Daniella Goldfarb, Email: daniella.goldfarb@weizmann.ac.il.</p></section><section id="pro70273-sec-0024"><h2 class="pmc_sec_title">DATA AVAILABILITY STATEMENT</h2>
<p>All data discussed in the paper are available in the main text and in Supporting Information.</p></section><section id="pro70273-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="pro70273-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="pro70273-bib-0001">
<cite>
Abraham MJ, Murtola T, Schulz R, Páll S, Smith JC, Hess B, et al. GROMACS: high performance molecular simulations through multi‐level parallelism from laptops to supercomputers. SoftwX. 2015;1–2:19–25.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=SoftwX&amp;title=GROMACS:%20high%20performance%20molecular%20simulations%20through%20multi%E2%80%90level%20parallelism%20from%20laptops%20to%20supercomputers&amp;author=MJ%20Abraham&amp;author=T%20Murtola&amp;author=R%20Schulz&amp;author=S%20P%C3%A1ll&amp;author=JC%20Smith&amp;volume=1%E2%80%932&amp;publication_year=2015&amp;pages=19-25&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0002">
<cite>
Alva V, Söding J, Lupas AN. A vocabulary of ancient peptides at the origin of folded proteins. eLife. 2015;4:e09410.
</cite> [<a href="https://doi.org/10.7554/eLife.09410" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4739770/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26653858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=eLife&amp;title=A%20vocabulary%20of%20ancient%20peptides%20at%20the%20origin%20of%20folded%20proteins&amp;author=V%20Alva&amp;author=J%20S%C3%B6ding&amp;author=AN%20Lupas&amp;volume=4&amp;publication_year=2015&amp;pages=e09410&amp;pmid=26653858&amp;doi=10.7554/eLife.09410&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0003">
<cite>
Bahrenberg T, Rosenski Y, Carmieli R, Zibzener K, Qi M, Frydman V, et al. Improved sensitivity for W‐band Gd(III)‐Gd(III) and nitroxide–nitroxide DEER measurements with shaped pulses. J Magn Reson. 2017;283:1–13.
</cite> [<a href="https://doi.org/10.1016/j.jmr.2017.08.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28834777/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Magn%20Reson&amp;title=Improved%20sensitivity%20for%20W%E2%80%90band%20Gd(III)%E2%80%90Gd(III)%20and%20nitroxide%E2%80%93nitroxide%20DEER%20measurements%20with%20shaped%20pulses&amp;author=T%20Bahrenberg&amp;author=Y%20Rosenski&amp;author=R%20Carmieli&amp;author=K%20Zibzener&amp;author=M%20Qi&amp;volume=283&amp;publication_year=2017&amp;pages=1-13&amp;pmid=28834777&amp;doi=10.1016/j.jmr.2017.08.003&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0004">
<cite>
Bordignon E. EPR spectroscopy of nitroxide spin probes. eMagRes. 2017;6:235–254.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=eMagRes&amp;title=EPR%20spectroscopy%20of%20nitroxide%20spin%20probes&amp;author=E%20Bordignon&amp;volume=6&amp;publication_year=2017&amp;pages=235-254&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0005">
<cite>
Despotović D, Longo LM, Aharon E, Kahana A, Scherf T, Gruic‐Sovulj I, et al. Polyamines mediate folding of primordial hyperacidic helical proteins. Biochemistry. 2020;59:4456–4462.
</cite> [<a href="https://doi.org/10.1021/acs.biochem.0c00800" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7735664/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33175508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=Polyamines%20mediate%20folding%20of%20primordial%20hyperacidic%20helical%20proteins&amp;author=D%20Despotovi%C4%87&amp;author=LM%20Longo&amp;author=E%20Aharon&amp;author=A%20Kahana&amp;author=T%20Scherf&amp;volume=59&amp;publication_year=2020&amp;pages=4456-4462&amp;pmid=33175508&amp;doi=10.1021/acs.biochem.0c00800&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0006">
<cite>
Doherty AJ, Serpell LC, Ponting CP. The helix‐hairpin‐helix DNA‐binding motif: a structural basis for non‐sequence‐specific recognition of DNA. Nucleic Acids Res. 1996;24:2488–2497.
</cite> [<a href="https://doi.org/10.1093/nar/24.13.2488" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC145986/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8692686/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Res&amp;title=The%20helix%E2%80%90hairpin%E2%80%90helix%20DNA%E2%80%90binding%20motif:%20a%20structural%20basis%20for%20non%E2%80%90sequence%E2%80%90specific%20recognition%20of%20DNA&amp;author=AJ%20Doherty&amp;author=LC%20Serpell&amp;author=CP%20Ponting&amp;volume=24&amp;publication_year=1996&amp;pages=2488-2497&amp;pmid=8692686&amp;doi=10.1093/nar/24.13.2488&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0007">
<cite>
Epel B, Arieli D, Baute D, Goldfarb D. Improving W‐band pulsed ENDOR sensitivity‐random acquisition and pulsed special TRIPLE. J Magn Reson. 2003;164:78–83.
</cite> [<a href="https://doi.org/10.1016/s1090-7807(03)00191-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12932459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Magn%20Reson&amp;title=Improving%20W%E2%80%90band%20pulsed%20ENDOR%20sensitivity%E2%80%90random%20acquisition%20and%20pulsed%20special%20TRIPLE&amp;author=B%20Epel&amp;author=D%20Arieli&amp;author=D%20Baute&amp;author=D%20Goldfarb&amp;volume=164&amp;publication_year=2003&amp;pages=78-83&amp;pmid=12932459&amp;doi=10.1016/s1090-7807(03)00191-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0008">
<cite>
Fried SD, Fujishima K, Makarov M, Cherepashuk I, Hlouchova K. Peptides before and during the nucleotide world: an origins story emphasizing cooperation between proteins and nucleic acids. J R Soc Interface. 2022;19:20210641.
</cite> [<a href="https://doi.org/10.1098/rsif.2021.0641" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8833103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35135297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20R%20Soc%20Interface&amp;title=Peptides%20before%20and%20during%20the%20nucleotide%20world:%20an%20origins%20story%20emphasizing%20cooperation%20between%20proteins%20and%20nucleic%20acids&amp;author=SD%20Fried&amp;author=K%20Fujishima&amp;author=M%20Makarov&amp;author=I%20Cherepashuk&amp;author=K%20Hlouchova&amp;volume=19&amp;publication_year=2022&amp;pages=20210641&amp;pmid=35135297&amp;doi=10.1098/rsif.2021.0641&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0009">
<cite>
Fujino T, Goto Y, Suga H, Murakami H. Reevaluation of the D‐amino acid compatibility with the elongation event in translation. J Am Chem Soc. 2013;135:1830–1837.
</cite> [<a href="https://doi.org/10.1021/ja309570x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23301668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=Reevaluation%20of%20the%20D%E2%80%90amino%20acid%20compatibility%20with%20the%20elongation%20event%20in%20translation&amp;author=T%20Fujino&amp;author=Y%20Goto&amp;author=H%20Suga&amp;author=H%20Murakami&amp;volume=135&amp;publication_year=2013&amp;pages=1830-1837&amp;pmid=23301668&amp;doi=10.1021/ja309570x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0010">
<cite>
Giacobelli VG, Fujishima K, Lepšík M, Tretyachenko V, Kadavá T, Makarov M, et al. In vitro evolution reveals noncationic protein–RNA interaction mediated by metal ions. Mol Biol Evol. 2022;39:msac032.
</cite> [<a href="https://doi.org/10.1093/molbev/msac032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8892947/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35137196/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Evol&amp;title=In%20vitro%20evolution%20reveals%20noncationic%20protein%E2%80%93RNA%20interaction%20mediated%20by%20metal%20ions&amp;author=VG%20Giacobelli&amp;author=K%20Fujishima&amp;author=M%20Lep%C5%A1%C3%ADk&amp;author=V%20Tretyachenko&amp;author=T%20Kadav%C3%A1&amp;volume=39&amp;publication_year=2022&amp;pages=msac032&amp;pmid=35137196&amp;doi=10.1093/molbev/msac032&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0011">
<cite>
Goldfarb D, Lipkin Y, Potapov A, Gorodetsky Y, Epel B, Raitsimring AM, et al. HYSCORE and DEER with an upgraded 95 GHz pulse EPR spectrometer. J Magn Reson. 2008;194:8–15.
</cite> [<a href="https://doi.org/10.1016/j.jmr.2008.05.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18571956/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Magn%20Reson&amp;title=HYSCORE%20and%20DEER%20with%20an%20upgraded%2095%20GHz%20pulse%20EPR%20spectrometer&amp;author=D%20Goldfarb&amp;author=Y%20Lipkin&amp;author=A%20Potapov&amp;author=Y%20Gorodetsky&amp;author=B%20Epel&amp;volume=194&amp;publication_year=2008&amp;pages=8-15&amp;pmid=18571956&amp;doi=10.1016/j.jmr.2008.05.019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0012">
<cite>
Gromov I, Krymov V, Manikandan P, Arieli D, Goldfarb D. A W‐band pulsed ENDOR spectrometer: setup and application to transition metal centers. J Magn Reson. 1999;139:8–17.
</cite> [<a href="https://doi.org/10.1006/jmre.1999.1762" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10388579/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Magn%20Reson&amp;title=A%20W%E2%80%90band%20pulsed%20ENDOR%20spectrometer:%20setup%20and%20application%20to%20transition%20metal%20centers&amp;author=I%20Gromov&amp;author=V%20Krymov&amp;author=P%20Manikandan&amp;author=D%20Arieli&amp;author=D%20Goldfarb&amp;volume=139&amp;publication_year=1999&amp;pages=8-17&amp;pmid=10388579&amp;doi=10.1006/jmre.1999.1762&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0013">
<cite>
Hustedt EJ, Marinelli F, Stein RA, Faraldo‐Gómez JD, McHaourab HS. Confidence analysis of DEER data and its structural interpretation with ensemble‐biased metadynamics. Biophys J. 2018;115:1200–1216.
</cite> [<a href="https://doi.org/10.1016/j.bpj.2018.08.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6170522/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30197182/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=Confidence%20analysis%20of%20DEER%20data%20and%20its%20structural%20interpretation%20with%20ensemble%E2%80%90biased%20metadynamics&amp;author=EJ%20Hustedt&amp;author=F%20Marinelli&amp;author=RA%20Stein&amp;author=JD%20Faraldo%E2%80%90G%C3%B3mez&amp;author=HS%20McHaourab&amp;volume=115&amp;publication_year=2018&amp;pages=1200-1216&amp;pmid=30197182&amp;doi=10.1016/j.bpj.2018.08.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0014">
<cite>
Jeschke G, Polyhach Y. Distance measurements on spin‐labelled biomacromolecules by pulsed electron paramagnetic resonance. Phys Chem Chem Phys. 2007;9:1895–1910.
</cite> [<a href="https://doi.org/10.1039/b614920k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17431518/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Phys%20Chem%20Chem%20Phys&amp;title=Distance%20measurements%20on%20spin%E2%80%90labelled%20biomacromolecules%20by%20pulsed%20electron%20paramagnetic%20resonance&amp;author=G%20Jeschke&amp;author=Y%20Polyhach&amp;volume=9&amp;publication_year=2007&amp;pages=1895-1910&amp;pmid=17431518&amp;doi=10.1039/b614920k&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0037">
<cite>
Joseph JA, Reinhardt A, Aguirre A, Chew PY, Russell KO, Espinosa JR, Garaizar A, &amp; Collepardo‐Guevara R.
Physics‐driven coarse‐grained model for biomolecular phase separation with near‐quantitative accuracy. Nature Computational Science. 2021;1:732–743.
</cite> [<a href="https://doi.org/10.1038/s43588-021-00155-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7612994/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35795820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Computational%20Science&amp;title=Physics%E2%80%90driven%20coarse%E2%80%90grained%20model%20for%20biomolecular%20phase%20separation%20with%20near%E2%80%90quantitative%20accuracy&amp;volume=1&amp;publication_year=2021&amp;pages=732-743&amp;pmid=35795820&amp;doi=10.1038/s43588-021-00155-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0015">
<cite>
Kabsch W, Sander C. Dictionary of protein secondary structure: pattern recognition of hydrogen‐bonded and geometrical features. Biopolymers. 1983;22:2577–2637.
</cite> [<a href="https://doi.org/10.1002/bip.360221211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/6667333/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biopolymers&amp;title=Dictionary%20of%20protein%20secondary%20structure:%20pattern%20recognition%20of%20hydrogen%E2%80%90bonded%20and%20geometrical%20features&amp;author=W%20Kabsch&amp;author=C%20Sander&amp;volume=22&amp;publication_year=1983&amp;pages=2577-2637&amp;pmid=6667333&amp;doi=10.1002/bip.360221211&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0016">
<cite>
Kim JD, Pike DH, Tyryshkin AM, Swapna GVT, Raanan H, Montelione GT, et al. Minimal heterochiral de novo designed 4Fe–4S binding peptide capable of robust electron transfer. J Am Chem Soc. 2018;140:11210–11213.
</cite> [<a href="https://doi.org/10.1021/jacs.8b07553" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7050467/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30141918/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=Minimal%20heterochiral%20de%20novo%20designed%204Fe%E2%80%934S%20binding%20peptide%20capable%20of%20robust%20electron%20transfer&amp;author=JD%20Kim&amp;author=DH%20Pike&amp;author=AM%20Tyryshkin&amp;author=GVT%20Swapna&amp;author=H%20Raanan&amp;volume=140&amp;publication_year=2018&amp;pages=11210-11213&amp;pmid=30141918&amp;doi=10.1021/jacs.8b07553&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0017">
<cite>
Kimura M, Akanuma S. Reconstruction and characterization of thermally stable and catalytically active proteins comprising an alphabet of ~13 amino acids. J Mol Evol. 2020;88:372–381.
</cite> [<a href="https://doi.org/10.1007/s00239-020-09938-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32201904/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Mol%20Evol&amp;title=Reconstruction%20and%20characterization%20of%20thermally%20stable%20and%20catalytically%20active%20proteins%20comprising%20an%20alphabet%20of%20~13%20amino%20acids&amp;author=M%20Kimura&amp;author=S%20Akanuma&amp;volume=88&amp;publication_year=2020&amp;pages=372-381&amp;pmid=32201904&amp;doi=10.1007/s00239-020-09938-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0018">
<cite>
Krause E, Bienert M, Schmieder P, Wenschuh H. The helix‐destabilizing propensity scale of d‐amino acids: the influence of side chain steric effects. J Am Chem Soc. 2000;122:4865–4870.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=The%20helix%E2%80%90destabilizing%20propensity%20scale%20of%20d%E2%80%90amino%20acids:%20the%20influence%20of%20side%20chain%20steric%20effects&amp;author=E%20Krause&amp;author=M%20Bienert&amp;author=P%20Schmieder&amp;author=H%20Wenschuh&amp;volume=122&amp;publication_year=2000&amp;pages=4865-4870&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0019">
<cite>
Longo LM, Blaber M. Prebiotic protein design supports a halophile origin of foldable proteins. Front Microbiol. 2014;4:418.
</cite> [<a href="https://doi.org/10.3389/fmicb.2013.00418" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3880840/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24432016/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Prebiotic%20protein%20design%20supports%20a%20halophile%20origin%20of%20foldable%20proteins&amp;author=LM%20Longo&amp;author=M%20Blaber&amp;volume=4&amp;publication_year=2014&amp;pages=418&amp;pmid=24432016&amp;doi=10.3389/fmicb.2013.00418&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0020">
<cite>
Longo LM, Despotović D, Weil‐Ktorza O, Walker MJ, Jabłońska J, Fridmann‐Sirkis Y, et al. Primordial emergence of a nucleic acid‐binding protein via phase separation and statistical ornithine‐to‐arginine conversion. Proc Natl Acad Sci U S A. 2020;117:15731–15739.
</cite> [<a href="https://doi.org/10.1073/pnas.2001989117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7355028/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32561643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Primordial%20emergence%20of%20a%20nucleic%20acid%E2%80%90binding%20protein%20via%20phase%20separation%20and%20statistical%20ornithine%E2%80%90to%E2%80%90arginine%20conversion&amp;author=LM%20Longo&amp;author=D%20Despotovi%C4%87&amp;author=O%20Weil%E2%80%90Ktorza&amp;author=MJ%20Walker&amp;author=J%20Jab%C5%82o%C5%84ska&amp;volume=117&amp;publication_year=2020&amp;pages=15731-15739&amp;pmid=32561643&amp;doi=10.1073/pnas.2001989117&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0021">
<cite>
Longo LM, Lee J, Blaber M. Simplified protein design biased for prebiotic amino acids yields a foldable, halophilic protein. Proc Natl Acad Sci U S A. 2013;110:2135–2139.
</cite> [<a href="https://doi.org/10.1073/pnas.1219530110" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3568330/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23341608/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Simplified%20protein%20design%20biased%20for%20prebiotic%20amino%20acids%20yields%20a%20foldable,%20halophilic%20protein&amp;author=LM%20Longo&amp;author=J%20Lee&amp;author=M%20Blaber&amp;volume=110&amp;publication_year=2013&amp;pages=2135-2139&amp;pmid=23341608&amp;doi=10.1073/pnas.1219530110&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0023">
<cite>
McGibbon RT, Beauchamp KA, Harrigan MP, Klein C, Swails JM, Hernández CX, et al. MDTraj: a modern open library for the analysis of molecular dynamics trajectories. Biophys J. 2015;109:1528–1532.
</cite> [<a href="https://doi.org/10.1016/j.bpj.2015.08.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4623899/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26488642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=MDTraj:%20a%20modern%20open%20library%20for%20the%20analysis%20of%20molecular%20dynamics%20trajectories&amp;author=RT%20McGibbon&amp;author=KA%20Beauchamp&amp;author=MP%20Harrigan&amp;author=C%20Klein&amp;author=JM%20Swails&amp;volume=109&amp;publication_year=2015&amp;pages=1528-1532&amp;pmid=26488642&amp;doi=10.1016/j.bpj.2015.08.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0024">
<cite>
Mentink‐Vigier F, Collauto A, Feintuch A, Kaminker I, Le VT, Goldfarb D. Increasing sensitivity of pulse EPR experiments using echo train detection schemes. J Magn Reson. 2013;236:117–125.
</cite> [<a href="https://doi.org/10.1016/j.jmr.2013.08.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24121563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Magn%20Reson&amp;title=Increasing%20sensitivity%20of%20pulse%20EPR%20experiments%20using%20echo%20train%20detection%20schemes&amp;author=F%20Mentink%E2%80%90Vigier&amp;author=A%20Collauto&amp;author=A%20Feintuch&amp;author=I%20Kaminker&amp;author=VT%20Le&amp;volume=236&amp;publication_year=2013&amp;pages=117-125&amp;pmid=24121563&amp;doi=10.1016/j.jmr.2013.08.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0025">
<cite>
Nanda V, Andrianarijaona A, Narayanan C. The role of protein homochirality in shaping the energy landscape of folding. Protein Sci. 2007;16:1667–1675.
</cite> [<a href="https://doi.org/10.1110/ps.072867007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2203351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17600146/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Protein%20Sci&amp;title=The%20role%20of%20protein%20homochirality%20in%20shaping%20the%20energy%20landscape%20of%20folding&amp;author=V%20Nanda&amp;author=A%20Andrianarijaona&amp;author=C%20Narayanan&amp;volume=16&amp;publication_year=2007&amp;pages=1667-1675&amp;pmid=17600146&amp;doi=10.1110/ps.072867007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0038">
<cite>
Noel JK, Whitford PC, &amp; Onuchic JN. The Shadow Map: A general contact definition for capturing the dynamics of biomolecular folding and function. The Journal of Physical Chemistry B. 2012; 116, 8692–8702.
</cite> [<a href="https://doi.org/10.1021/jp300852d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3406251/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22536820/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Journal%20of%20Physical%20Chemistry%20B&amp;title=The%20Shadow%20Map:%20A%20general%20contact%20definition%20for%20capturing%20the%20dynamics%20of%20biomolecular%20folding%20and%20function&amp;volume=116&amp;publication_year=2012&amp;pages=8692-8702&amp;pmid=22536820&amp;doi=10.1021/jp300852d&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0026">
<cite>
Oparin AI, Morgulis S. The origin of life. New York, NY: The Macmillan Company; 1938.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=The%20origin%20of%20life&amp;author=AI%20Oparin&amp;author=S%20Morgulis&amp;publication_year=1938&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0027">
<cite>
Weil‐Ktorza O, Naveh‐Tassa S, Fridmann‐Sirkis Y, Despotovic D, Cherukuri KP, Corlett T, et al. Functional ambidexterity of an acient nucleic acid‐binding domain. Angewandte Chemie. 2025;64:e202505188

</cite> [<a href="https://doi.org/10.1002/anie.202505188" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12171331/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40151087/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angewandte%20Chemie&amp;title=Functional%20ambidexterity%20of%20an%20acient%20nucleic%20acid%E2%80%90binding%20domain&amp;volume=64&amp;publication_year=2025&amp;pages=e202505188&amp;pmid=40151087&amp;doi=10.1002/anie.202505188&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0028">
<cite>
Pronk S, Páll S, Schulz R, Larsson P, Bjelkmar P, Apostolov R, et al. GROMACS 4.5: a high‐throughput and highly parallel open source molecular simulation toolkit. Bioinformatics. 2013;29:845–854.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3605599/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23407358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=GROMACS%204.5:%20a%20high%E2%80%90throughput%20and%20highly%20parallel%20open%20source%20molecular%20simulation%20toolkit&amp;author=S%20Pronk&amp;author=S%20P%C3%A1ll&amp;author=R%20Schulz&amp;author=P%20Larsson&amp;author=P%20Bjelkmar&amp;volume=29&amp;publication_year=2013&amp;pages=845-854&amp;pmid=23407358&amp;doi=10.1093/bioinformatics/btt055&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0029">
<cite>
Schindelin J, Arganda‐Carreras I, Frise E, Kaynig V, Longair M, Pietzsch T, et al. Fiji: an open‐source platform for biological‐image analysis. Nat Methods. 2012;9:676–682.
</cite> [<a href="https://doi.org/10.1038/nmeth.2019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3855844/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Fiji:%20an%20open%E2%80%90source%20platform%20for%20biological%E2%80%90image%20analysis&amp;author=J%20Schindelin&amp;author=I%20Arganda%E2%80%90Carreras&amp;author=E%20Frise&amp;author=V%20Kaynig&amp;author=M%20Longair&amp;volume=9&amp;publication_year=2012&amp;pages=676-682&amp;pmid=22743772&amp;doi=10.1038/nmeth.2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0030">
<cite>
Seal M, Weil‐Ktorza O, Despotović D, Tawfik DS, Levy Y, Metanis N, et al. Peptide‐RNA coacervates as a cradle for the evolution of folded domains. J Am Chem Soc. 2022;144:14150–14160.
</cite> [<a href="https://doi.org/10.1021/jacs.2c03819" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9376946/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35904499/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=Peptide%E2%80%90RNA%20coacervates%20as%20a%20cradle%20for%20the%20evolution%20of%20folded%20domains&amp;author=M%20Seal&amp;author=O%20Weil%E2%80%90Ktorza&amp;author=D%20Despotovi%C4%87&amp;author=DS%20Tawfik&amp;author=Y%20Levy&amp;volume=144&amp;publication_year=2022&amp;pages=14150-14160&amp;pmid=35904499&amp;doi=10.1021/jacs.2c03819&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0031">
<cite>
Smokers IBA, Visser BS, Slootbeek AD, Huck WTS, Spruijt E. How droplets can accelerate reactions—coacervate protocells as catalytic microcompartments. Acc Chem Res. 2024;57:1885–1895.
</cite> [<a href="https://doi.org/10.1021/acs.accounts.4c00114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11256357/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38968602/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acc%20Chem%20Res&amp;title=How%20droplets%20can%20accelerate%20reactions%E2%80%94coacervate%20protocells%20as%20catalytic%20microcompartments&amp;author=IBA%20Smokers&amp;author=BS%20Visser&amp;author=AD%20Slootbeek&amp;author=WTS%20Huck&amp;author=E%20Spruijt&amp;volume=57&amp;publication_year=2024&amp;pages=1885-1895&amp;pmid=38968602&amp;doi=10.1021/acs.accounts.4c00114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0032">
<cite>
Stoll S, Schweiger A. EasySpin, a comprehensive software package for spectral simulation and analysis in EPR. J Magn Reson. 2006;178:42–55.
</cite> [<a href="https://doi.org/10.1016/j.jmr.2005.08.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16188474/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Magn%20Reson&amp;title=EasySpin,%20a%20comprehensive%20software%20package%20for%20spectral%20simulation%20and%20analysis%20in%20EPR&amp;author=S%20Stoll&amp;author=A%20Schweiger&amp;volume=178&amp;publication_year=2006&amp;pages=42-55&amp;pmid=16188474&amp;doi=10.1016/j.jmr.2005.08.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0033">
<cite>
Urry DW, Goodall MC, Glickson JD, Mayers DF. The gramicidin a transmembrane channel: characteristics of head‐to‐head dimerized pi(L,D) helices. Proc Natl Acad Sci U S A. 1971;68:1907–1911.
</cite> [<a href="https://doi.org/10.1073/pnas.68.8.1907" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC389319/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/5288776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=The%20gramicidin%20a%20transmembrane%20channel:%20characteristics%20of%20head%E2%80%90to%E2%80%90head%20dimerized%20pi(L,D)%20helices&amp;author=DW%20Urry&amp;author=MC%20Goodall&amp;author=JD%20Glickson&amp;author=DF%20Mayers&amp;volume=68&amp;publication_year=1971&amp;pages=1907-1911&amp;pmid=5288776&amp;doi=10.1073/pnas.68.8.1907&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0034">
<cite>
Veatch WR, Fossel ET, Blout ER. The conformation of gramicidin A. Biochemistry. 1974;13:5249–5256.
</cite> [<a href="https://doi.org/10.1021/bi00723a001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4139971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemistry&amp;title=The%20conformation%20of%20gramicidin%20A&amp;author=WR%20Veatch&amp;author=ET%20Fossel&amp;author=ER%20Blout&amp;volume=13&amp;publication_year=1974&amp;pages=5249-5256&amp;pmid=4139971&amp;doi=10.1021/bi00723a001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0035">
<cite>
Yagi S, Padhi AK, Vucinic J, Barbe S, Schiex T, Nakagawa R, et al. Seven amino acid types suffice to create the core fold of RNA polymerase. J Am Chem Soc. 2021;143:15998–16006.
</cite> [<a href="https://doi.org/10.1021/jacs.1c05367" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34559526/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Chem%20Soc&amp;title=Seven%20amino%20acid%20types%20suffice%20to%20create%20the%20core%20fold%20of%20RNA%20polymerase&amp;author=S%20Yagi&amp;author=AK%20Padhi&amp;author=J%20Vucinic&amp;author=S%20Barbe&amp;author=T%20Schiex&amp;volume=143&amp;publication_year=2021&amp;pages=15998-16006&amp;pmid=34559526&amp;doi=10.1021/jacs.1c05367&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pro70273-bib-0036">
<cite>
Zänker P‐P, Jeschke G, Goldfarb D. Distance measurements between paramagnetic centers and a planar object by matrix Mims electron nuclear double resonance. J Chem Phys. 2004;122:024515.</cite> [<a href="https://doi.org/10.1063/1.1828435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15638606/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Chem%20Phys&amp;title=Distance%20measurements%20between%20paramagnetic%20centers%20and%20a%20planar%20object%20by%20matrix%20Mims%20electron%20nuclear%20double%20resonance&amp;author=P%E2%80%90P%20Z%C3%A4nker&amp;author=G%20Jeschke&amp;author=D%20Goldfarb&amp;volume=122&amp;publication_year=2004&amp;pages=024515&amp;pmid=15638606&amp;doi=10.1063/1.1828435&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>Data S1.</strong> Supporting Information.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12375978/bin/PRO-34-e70273-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">PRO-34-e70273-s001.docx</a><sup> (4MB, docx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data discussed in the paper are available in the main text and in Supporting Information.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Protein Science : A Publication of the Protein Society are provided here courtesy of <strong>The Protein Society</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/pro.70273"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/PRO-34-e70273.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.4 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12375978/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12375978/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12375978%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12375978/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12375978/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12375978/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12375978/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12375978/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12375978/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="l4mso9e93Ci2kwhxBgGzTQyXdUEYw7cfX5VtOmLSCyPMYGdJAbVFXS7xQ1OqSETt">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
