
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Tumor microenvironment signatures enhances lung adenocarcinoma prognosis prediction: Implication of intratumoral microbiota - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E484428AEF08F30584420008980BBB.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="microbcell">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373402/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Microbial Cell">
<meta name="citation_title" content="Tumor microenvironment signatures enhances lung adenocarcinoma prognosis prediction: Implication of intratumoral microbiota">
<meta name="citation_author" content="Fei Zhao">
<meta name="citation_author_institution" content="School of Mathematical Sciences, Ocean University of China, Qingdao, 266100, China.">
<meta name="citation_author_institution" content="The authors contributed equally to this work.">
<meta name="citation_author" content="Lei Wang">
<meta name="citation_author_institution" content="Hebei Key Laboratory of Molecular Oncology, Hebei, 063000, China.">
<meta name="citation_author_institution" content="Tangshan Key Laboratory of Cancer Prevention and Therapy, Hebei, 063000, China.">
<meta name="citation_author_institution" content="Department of Pathology, Tangshan People&#x27;s Hospital, Hebei, 063000, China.">
<meta name="citation_author_institution" content="The authors contributed equally to this work.">
<meta name="citation_author" content="Dongjie Du">
<meta name="citation_author_institution" content="Department of Vascular Surgery, Hebei General Hospital, Shijiazhuang, 050051, China.">
<meta name="citation_author" content="Heaven Zhao">
<meta name="citation_author_institution" content="Geneis Beijing Co., Ltd., Beijing, 100102, China.">
<meta name="citation_author_institution" content="Qingdao Geneis Institute of Big Data Mining and Precision Medicine, Qingdao 266000, China.">
<meta name="citation_author" content="Geng Tian">
<meta name="citation_author_institution" content="Geneis Beijing Co., Ltd., Beijing, 100102, China.">
<meta name="citation_author_institution" content="Qingdao Geneis Institute of Big Data Mining and Precision Medicine, Qingdao 266000, China.">
<meta name="citation_author" content="Yufeng Li">
<meta name="citation_author_institution" content="Hebei Key Laboratory of Molecular Oncology, Hebei, 063000, China.">
<meta name="citation_author_institution" content="Tangshan Key Laboratory of Cancer Prevention and Therapy, Hebei, 063000, China.">
<meta name="citation_author_institution" content="The Cancer Institute, Tangshan People&#x27;s Hospital, Hebei, 063000, China.">
<meta name="citation_author" content="Yankun Liu">
<meta name="citation_author_institution" content="Hebei Key Laboratory of Molecular Oncology, Hebei, 063000, China.">
<meta name="citation_author_institution" content="The Cancer Institute, Tangshan People&#x27;s Hospital, Hebei, 063000, China.">
<meta name="citation_author_institution" content="Tangshan Key Laboratory of Precision Medicine Testing, Hebei, 063000, China.">
<meta name="citation_author" content="Zhiwu Wang">
<meta name="citation_author_institution" content="Hebei Key Laboratory of Molecular Oncology, Hebei, 063000, China.">
<meta name="citation_author_institution" content="Tangshan Key Laboratory of Cancer Prevention and Therapy, Hebei, 063000, China.">
<meta name="citation_author_institution" content="Department of Chemoradiotherapy, Tangshan People&#x27;s Hospital, Hebei, 063000, China.">
<meta name="citation_author" content="Dasheng Liu">
<meta name="citation_author_institution" content="Department of Thoracic Surgery, Jiamusi Central Hospital, Jiamusi, 154000, China.">
<meta name="citation_author" content="Jingwu Li">
<meta name="citation_author_institution" content="Hebei Key Laboratory of Molecular Oncology, Hebei, 063000, China.">
<meta name="citation_author_institution" content="Tangshan Key Laboratory of Cancer Prevention and Therapy, Hebei, 063000, China.">
<meta name="citation_author_institution" content="Department of Gastrointestinal Surgery, Tangshan People&#x27;s Hospital, Hebei, 063000, China.">
<meta name="citation_author" content="Lei Ji">
<meta name="citation_author_institution" content="Geneis Beijing Co., Ltd., Beijing, 100102, China.">
<meta name="citation_author_institution" content="Qingdao Geneis Institute of Big Data Mining and Precision Medicine, Qingdao 266000, China.">
<meta name="citation_author" content="Hong Zhao">
<meta name="citation_author_institution" content="School of Mathematical Sciences, Ocean University of China, Qingdao, 266100, China.">
<meta name="citation_publication_date" content="2025 Aug 11">
<meta name="citation_volume" content="12">
<meta name="citation_firstpage" content="182">
<meta name="citation_doi" content="10.15698/mic2025.08.855">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373402/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373402/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373402/pdf/mic-12-182.pdf">
<meta name="description" content="The interaction between intratumoral microbiome and the tumor microenvironment (TME) has furthered our understanding of tumor ecology. Yet, the implications of their interaction for lung cancer management remain unclear. In the current work, we ...">
<meta name="og:title" content="Tumor microenvironment signatures enhances lung adenocarcinoma prognosis prediction: Implication of intratumoral microbiota">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The interaction between intratumoral microbiome and the tumor microenvironment (TME) has furthered our understanding of tumor ecology. Yet, the implications of their interaction for lung cancer management remain unclear. In the current work, we ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373402/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373402">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.15698/mic2025.08.855"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/mic-12-182.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373402%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373402/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373402/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373402/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-microbcell.png" alt="Microbial Cell logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Microbial Cell" title="Link to Microbial Cell" shape="default" href="http://microbialcell.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Microb Cell</button></div>. 2025 Aug 11;12:182–194. doi: <a href="https://doi.org/10.15698/mic2025.08.855" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.15698/mic2025.08.855</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Microb%20Cell%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Microb%20Cell%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Microb%20Cell%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Microb%20Cell%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Tumor microenvironment signatures enhances lung adenocarcinoma prognosis prediction: Implication of intratumoral microbiota</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Fei Zhao</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Fei Zhao</span></h3>
<div class="p">
<sup>1</sup>School of Mathematical Sciences, Ocean University of China, Qingdao, 266100, China.
</div>
<div class="p">
<sup>a</sup>The authors contributed equally to this work.
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fei Zhao</span></a>
</div>
</div>
<sup>1,</sup><sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Lei Wang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Lei Wang</span></h3>
<div class="p">
<sup>2</sup>Hebei Key Laboratory of Molecular Oncology, Hebei, 063000, China.
</div>
<div class="p">
<sup>3</sup>Tangshan Key Laboratory of Cancer Prevention and Therapy, Hebei, 063000, China.
</div>
<div class="p">
<sup>4</sup>Department of Pathology, Tangshan People's Hospital, Hebei, 063000, China.
</div>
<div class="p">
<sup>a</sup>The authors contributed equally to this work.
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lei Wang</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Du%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Dongjie Du</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Dongjie Du</span></h3>
<div class="p">
<sup>5</sup>Department of Vascular Surgery, Hebei General Hospital, Shijiazhuang, 050051, China.
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Du%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dongjie Du</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Heaven Zhao</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Heaven Zhao</span></h3>
<div class="p">
<sup>6</sup>Geneis Beijing Co., Ltd., Beijing, 100102, China.
</div>
<div class="p">
<sup>7</sup>Qingdao Geneis Institute of Big Data Mining and Precision Medicine, Qingdao 266000, China.
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Heaven Zhao</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tian%20G%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Geng Tian</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Geng Tian</span></h3>
<div class="p">
<sup>6</sup>Geneis Beijing Co., Ltd., Beijing, 100102, China.
</div>
<div class="p">
<sup>7</sup>Qingdao Geneis Institute of Big Data Mining and Precision Medicine, Qingdao 266000, China.
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tian%20G%22%5BAuthor%5D" class="usa-link"><span class="name western">Geng Tian</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Yufeng Li</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Yufeng Li</span></h3>
<div class="p">
<sup>2</sup>Hebei Key Laboratory of Molecular Oncology, Hebei, 063000, China.
</div>
<div class="p">
<sup>3</sup>Tangshan Key Laboratory of Cancer Prevention and Therapy, Hebei, 063000, China.
</div>
<div class="p">
<sup>8</sup>The Cancer Institute, Tangshan People's Hospital, Hebei, 063000, China.
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yufeng Li</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Yankun Liu</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Yankun Liu</span></h3>
<div class="p">
<sup>2</sup>Hebei Key Laboratory of Molecular Oncology, Hebei, 063000, China.
</div>
<div class="p">
<sup>8</sup>The Cancer Institute, Tangshan People's Hospital, Hebei, 063000, China.
</div>
<div class="p">
<sup>9</sup>Tangshan Key Laboratory of Precision Medicine Testing, Hebei, 063000, China.
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yankun Liu</span></a>
</div>
</div>
<sup>2,</sup><sup>8,</sup><sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Zhiwu Wang</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Zhiwu Wang</span></h3>
<div class="p">
<sup>2</sup>Hebei Key Laboratory of Molecular Oncology, Hebei, 063000, China.
</div>
<div class="p">
<sup>3</sup>Tangshan Key Laboratory of Cancer Prevention and Therapy, Hebei, 063000, China.
</div>
<div class="p">
<sup>10</sup>Department of Chemoradiotherapy, Tangshan People's Hospital, Hebei, 063000, China.
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhiwu Wang</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Dasheng Liu</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Dasheng Liu</span></h3>
<div class="p">
<sup>11</sup>Department of Thoracic Surgery, Jiamusi Central Hospital, Jiamusi, 154000, China.
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dasheng Liu</span></a>
</div>
</div>
<sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Jingwu Li</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Jingwu Li</span></h3>
<div class="p">
<sup>2</sup>Hebei Key Laboratory of Molecular Oncology, Hebei, 063000, China.
</div>
<div class="p">
<sup>3</sup>Tangshan Key Laboratory of Cancer Prevention and Therapy, Hebei, 063000, China.
</div>
<div class="p">
<sup>12</sup>Department of Gastrointestinal Surgery, Tangshan People's Hospital, Hebei, 063000, China.
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jingwu Li</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>12,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ji%20L%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Lei Ji</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Lei Ji</span></h3>
<div class="p">
<sup>6</sup>Geneis Beijing Co., Ltd., Beijing, 100102, China.
</div>
<div class="p">
<sup>7</sup>Qingdao Geneis Institute of Big Data Mining and Precision Medicine, Qingdao 266000, China.
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ji%20L%22%5BAuthor%5D" class="usa-link"><span class="name western">Lei Ji</span></a>
</div>
</div>
<sup>6,</sup><sup>7,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Hong Zhao</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Hong Zhao</span></h3>
<div class="p">
<sup>1</sup>School of Mathematical Sciences, Ocean University of China, Qingdao, 266100, China.
</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hong Zhao</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>School of Mathematical Sciences, Ocean University of China, Qingdao, 266100, China.
</div>
<div id="aff2">
<sup>2</sup>Hebei Key Laboratory of Molecular Oncology, Hebei, 063000, China.
</div>
<div id="aff3">
<sup>3</sup>Tangshan Key Laboratory of Cancer Prevention and Therapy, Hebei, 063000, China.
</div>
<div id="aff4">
<sup>4</sup>Department of Pathology, Tangshan People's Hospital, Hebei, 063000, China.
</div>
<div id="aff5">
<sup>5</sup>Department of Vascular Surgery, Hebei General Hospital, Shijiazhuang, 050051, China.
</div>
<div id="aff6">
<sup>6</sup>Geneis Beijing Co., Ltd., Beijing, 100102, China.
</div>
<div id="aff7">
<sup>7</sup>Qingdao Geneis Institute of Big Data Mining and Precision Medicine, Qingdao 266000, China.
</div>
<div id="aff8">
<sup>8</sup>The Cancer Institute, Tangshan People's Hospital, Hebei, 063000, China.
</div>
<div id="aff9">
<sup>9</sup>Tangshan Key Laboratory of Precision Medicine Testing, Hebei, 063000, China.
</div>
<div id="aff10">
<sup>10</sup>Department of Chemoradiotherapy, Tangshan People's Hospital, Hebei, 063000, China.
</div>
<div id="aff11">
<sup>11</sup>Department of Thoracic Surgery, Jiamusi Central Hospital, Jiamusi, 154000, China.
</div>
<div id="aff12">
<sup>12</sup>Department of Gastrointestinal Surgery, Tangshan People's Hospital, Hebei, 063000, China.
</div>
<div id="affa">
<sup>a</sup>The authors contributed equally to this work.
</div>
<div class="author-notes p">
<div class="fn" id="fn1"><p>Please cite this article as: Fei Zhao, Lei Wang, Dongjie Du, Heaven Zhao, Geng Tian, Yufeng Li, Yankun Liu, Zhiwu Wang, Dasheng Liu, Jingwu Li, Lei Ji and Hong Zhao (<strong>2025</strong>). Tumor microenvironment signatures enhances lung adenocarcinoma prognosis prediction: Implication of intratumoral microbiota <strong>Microbial Cell</strong> 12(1): 182-194. doi: 10.15698/mic2025.08.855</p></div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 14; Accepted 2025 Jul 30; Revised 2025 Jul 9; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright: © 2025 Zhao <em>et al</em>.</div>
<p>This is an open-access article released under the terms of the Creative Commons Attribution (CC BY) license, which allows the unrestricted use, distribution, and reproduction in any medium, provided the original author and source are acknowledged.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373402</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>The interaction between intratumoral microbiome and the tumor microenvironment (TME) has furthered our understanding of tumor ecology. Yet, the implications of their interaction for lung cancer management remain unclear. In the current work, we collected host transcriptome samples and matched intratumoral microbiome samples, as well as detailed clinical metadata from The Cancer Genome Atlas (TCGA) of 478 patients with lung adenocarcinoma (LUAD). Utilizing the multiomics integration approach, we comprehensively investigated the crosstalk between the TME and intratumoral microbiome in patients with LUAD. First, we developed a prognostic model based on the TME signatures (TMEindex) that clearly distinguished clinical, survival, and response to immunotherapy of patients with LUAD. Additionally, we found profound differences in intratumoral microbiota signatures, including alpha- and beta-diversity, among patients with different survival risks based on the TME signatures. In depth, we detected that genera <em>Luteibacter </em>and <em>Chryseobacterium </em>were strongly negatively and positively associated with patients’ survival risk, respectively, suggesting their opposing roles in cancer progression. Moreover, we developed a model that fused intratumoral microbial abundance information with TME signatures, called intratumoral microbiome-modified TMEindex (IMTMEindex), leading in predicting patient overall survival at 1-, 3-, and 5-years. Future clinical profiling of the specific intratumoral microbes in the TME could improve prognosis, inform immunotherapy, and facilitate the development of novel therapeutics for LUAD.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Keywords:</strong> intratumoral microbiome, lung adenocarcinoma, machine learning, prognosis, tumor microenvironment</p></section></section><section id="sec1"><h2 class="pmc_sec_title">Abbreviations</h2>
<p>DEG - differentially expressed gene,</p>
<p>ICGs - immune checkpoint-related genes,</p>
<p>ICIs - immune checkpoint inhibitors,</p>
<p>LUAD - lung adenocarcinoma,</p>
<p>NK - natural killer,</p>
<p>NSCLC - non-small cell lung cancer,</p>
<p>OS - overall survival,</p>
<p>TCGA - The Cancer Genome Atlas,</p>
<p>TIDE - Tumor immun dysfunction and exclusion,</p>
<p>TME - tumor microenvironment,</p>
<p>WGCNA - Weighted Gene Co-expression Network Analysis.</p></section><section id="sec2"><h2 class="pmc_sec_title">INTRODUCTION</h2>
<p>Lung cancer remains the leading cause of cancer-related death worldwide <a href="#B1" class="usa-link" aria-describedby="B1">1</a>,<a href="#B2" class="usa-link" aria-describedby="B2">2</a>. Lung adenocarcinoma (LUAD) is the most common type of lung cancer, belonging to the non-small cell cancer, accounting for 30% to 35% of all lung cancers <a href="#B3" class="usa-link" aria-describedby="B3">3</a>. Despite advances in treatment strategies, five-year survival for patients with LUAD remains limited <a href="#B4" class="usa-link" aria-describedby="B4">4</a>. At present, the tumor-related mechanism of LUAD remains unclear <a href="#B5" class="usa-link" aria-describedby="B5">5</a>, and the heterogeneity of tumors makes it difficult to accurately assess the prognosis of each patient <a href="#B6" class="usa-link" aria-describedby="B6">6</a>. Therefore, accurate and individualized assessment and improvement of survival in patients with LUAD remains a huge challenge.</p>
<p>The tumor microenvironment (TME) is the environment in which the tumor resides and is composed of various immune cells, stromal cells (including mesenchymal cells, endothelial cells), extracellular matrix molecules, and a variety of cytokines. Factors such as immune cells, angiogenesis, and cytokines in the TME are closely related to the biological characteristics of the tumor, and the components in the TME can define the immunophenotype of cancer, thus affecting the development of the tumor and the prognosis of patients <a href="#B7" class="usa-link" aria-describedby="B7">7</a>,<a href="#B8" class="usa-link" aria-describedby="B8">8</a>. Various cells in TME can be tumor suppressor or tumor supportive <a href="#B9" class="usa-link" aria-describedby="B9">9</a>. For example, low levels of infiltration of cytotoxic immune cells can help tumor cells evade immune attack, thereby reducing a patient's probability of survival. In addition to immune cells, stromal cells also regulate tumor immunophenotypes, such as cancer-associated fibroblasts, which exert a direct immunosuppressive mechanism of action <a href="#B10" class="usa-link" aria-describedby="B10">10</a>. In addition, abnormal changes in the TME not only affect patient prognosis, but can also be used as a biomarker for treatment, such as immunotherapy. Finding characteristic markers from immune cells and stromal cells may be an effective way to predict the prognosis and benefit of treatment.</p>
<p>Tumor tissues once thought to be sterile are actually home to a variety of microorganisms, which have been shown to be closely related to tumorigenesis <a href="#B11" class="usa-link" aria-describedby="B11">11</a>,<a href="#B12" class="usa-link" aria-describedby="B12">12</a>,<a href="#B13" class="usa-link" aria-describedby="B13">13</a>,<a href="#B14" class="usa-link" aria-describedby="B14">14</a>. Microorganisms in tumors can significantly change the biological characteristics of different cell compartments and affect the occurrence, development and metastasis of tumors as well as anti-tumor immunity <a href="#B15" class="usa-link" aria-describedby="B15">15</a>,<a href="#B16" class="usa-link" aria-describedby="B16">16</a>,<a href="#B17" class="usa-link" aria-describedby="B17">17</a>,<a href="#B18" class="usa-link" aria-describedby="B18">18</a>. Anti-tumor immune activity can be stimulated or inhibited by signaling pathways, which in turn can consist of polysaccharides of microbial origin. This association underscores the importance that changes in the microbiome may lead to a more or less favorable tumor response <a href="#B19" class="usa-link" aria-describedby="B19">19</a>,<a href="#B20" class="usa-link" aria-describedby="B20">20</a>. The analysis of the intratumoral microbiome associated with lung cancer has only just begun and some revelatory studies have been conducted. For example, one study suggested that the number of <em>Veillonella</em> and <em>Megasphaera </em>in patients with lung cancer was significantly higher than that in patients with benign disease <a href="#B21" class="usa-link" aria-describedby="B21">21</a>. Yu <em>et al</em>. reported that the microbiota of lung tumor tissues showed significantly lower alpha diversity compared to non-malignant lung tissue samples, and that bacterial composition was correlated with cancer stage <a href="#B22" class="usa-link" aria-describedby="B22">22</a>.</p>
<p>Studies have shown that intratumoral microbiota affect tumor progression mainly by changing the tumor immune microenvironment and regulating local immune response <a href="#B23" class="usa-link" aria-describedby="B23">23</a>,<a href="#B24" class="usa-link" aria-describedby="B24">24</a>,<a href="#B25" class="usa-link" aria-describedby="B25">25</a>,<a href="#B26" class="usa-link" aria-describedby="B26">26</a>,<a href="#B27" class="usa-link" aria-describedby="B27">27</a>. Reducing the number of bacteria in the lungs of mice by antibiotic aerosol inhalation can activate tumor-infiltrating T cells and natural killer (NK) cells, reduce the number of immunosuppressive regulatory T cells, and enhance local anti-tumor immune response <a href="#B28" class="usa-link" aria-describedby="B28">28</a>,<a href="#B29" class="usa-link" aria-describedby="B29">29</a>. Increased local bacterial load and changes in microbial composition can stimulate myeloid immune cells to produce IL-1β and IL-23 through the myD88-dependent pathway, and these cytokines stimulate the activation and proliferation of Vγ6<sup>+</sup> Vδ1<sup>+</sup> γδ T cells, and then produce IL-17 and other effector molecules to promote inflammation and tumourgenesis <a href="#B30" class="usa-link" aria-describedby="B30">30</a>. The intratumoral microbiota can modulate anti-tumor immunity through its interaction with the TME and influence the efficacy of cancer immunotherapy represented by immune checkpoint inhibitors (ICIs).</p>
<p>It has been widely demonstrated that TME signatures and intratumoral microbial profiles are independently associated with lung tumors. However, previous studies have neither systematically established a connection between the two nor considered their combined prognostic effects. To bridge this gap, this study incorporated a total of 478 patients with LUAD from The Cancer Genome Atlas (TCGA). Through a systematic and integrated analysis of multiple cell populations and microbiota within the TME, as well as clinical phenotypes and prognostic outcomes, we discovered that the abundance of specific intratumoral microbes was strongly correlated with the survival risk predicted by a model constructed based on TME characteristics. This finding suggests the existence of crosstalk between intratumoral microbes and the TME, and highlights its impact on patient prognosis. Our study offers a valuable opportunity to identify microbial-based biomarkers for cancer prevention or prognosis. This, in turn, can aid in the design and testing of therapeutic strategies aimed at modulating the microbiota to improve cancer treatment and, ultimately, even prevent or intercept malignant transformation.</p></section><section id="sec3"><h2 class="pmc_sec_title">RESULTS</h2>
<section id="sec4"><h3 class="pmc_sec_title">Identification of TME-related genes</h3>
<p>First, the Estimation of STromal and Immune cells in Malignant Tumor tissues using Expression data (ESTIMATE) algorithm was used to obtain the stromal and immune scores of all patients. In order to define TME-related genes, differentially expressed gene (DEG) analysis was carried out on the two groups with different stromal and immune scores (<strong>Fig. 1A</strong> and <strong>1B</strong>). Weighted Gene Co-expression Network Analysis (WGCNA) delimited distinguishable modules by hierarchical clustering (<strong>Fig. 1C</strong>), and the results showed that the black, green, brown, and yellow modules were strongly positively correlated with stromal score, while the honeydew1, navajowhite2, brown, darkorange2, and yellow modules were strongly positively correlated with immune score (<strong>Fig. 1D</strong>).</p>
<figure class="fig xbox font-sm" id="fig1"><h4 class="obj_head">Figure 1. FIGURE 1: Identification of the TME-related genes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373402_mic-12-182-g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e95/12373402/9d4d49cce067/mic-12-182-g001.jpg" loading="lazy" height="1067" width="793" alt="Figure 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Differentially expressed genes between patients with high and low stromal scores. <strong>(B)</strong> Differentially expressed genes between patients with high and low immune scores. <strong>(C)</strong> Gene modules recognized by WGCNA. <strong>(D)</strong> Correlation between gene modules and stromal score, immune score and ESTIMATE score. <strong>(E-F)</strong> LASSO algorithm was used to further screen genes extracted from WGCNA. ***<em>P</em> &lt; 0.001.</p></figcaption></figure><p>Next, the intersection of the identified DEGs with genes in the WGCNA modules was used as input to least absolute shrinkage and selection operator (LASSO)-Cox regression from the perspective of stromal and immune scores, respectively. From the stromal perspective, <em>PIK3CG</em>, <em>CCR2</em>, <em>RGS13</em>, <em>MS4A2</em>, <em>GCSAML</em>, <em>APOC4-APOC2</em>, <em>CHRDL1</em>, <em>GPR1</em>, and <em>CPS1</em> genes were identified; meanwhile, from the immune perspective, <em>GAB3</em>, <em>CCR2</em>, <em>PIK3CG</em>, <em>CIITA</em>, <em>APOC4-APOC2</em>, <em>IKZF3</em>, <em>TREML1</em>, <em>P2RX1</em>, <em>CD70</em>, <em>CHRNA6</em>, <em>KLRC2</em>, and <em>CPS1</em> genes were identified (<strong>Fig. 1E</strong> and <strong>1F</strong>). Then, these genes were subjected to univariate Cox regression and stepwise regression to determine the final TME-related genes using minimum AIC as the criterion. The model consisting of <em>GCSAML</em>, <em>APOC4-APOC2</em>, <em>CHRDL1</em>, <em>GPR1</em>, <em>CPS1</em>, <em>IKZF3</em>, <em>P2RX1</em>, <em>CD70</em>, and<em> KLRC2</em> had a minimal AIC value (AIC = 1768.2, Table S1). Therefore, we developed the final risk model, called the tumor microenvironment-related index (TMEindex):</p>
<p>TEMindex = 0,93117 x GCSAML - 1.75862 x APOC4 - APOC2 + 0.116545 x CHRDL1 + 0.41756 x GPR1 + 0.06990 x CPS1 - 0.36391 x IKZF3 - 0.36001 x P2RX1 + 0.47444 x CD70 + 0.54178 x KLRC2</p></section><section id="sec5"><h3 class="pmc_sec_title">The TMEindex exhibits clinical and prognostic implications</h3>
<p>Next, we correlated the genes included in the TMEindex with tumorigenesis of LUAD. In TCGA-LUAD cohort, the expressions of <em>CHRDL1</em>, <em>P2RX1</em>, <em>GCSAML</em> and <em>APOC4-APOC2</em> in tumor tissues were significantly lower than those in normal tissues, indicating the tumor-suppressing role of these genes in LUAD. Meanwhile, the expressions of <em>IKZF3</em> and <em>CD70</em> were significantly higher in tumor tissues than those in normal tissues (<strong>Fig. 2A</strong>). Additionally, similar results were detected in <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE101929" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE101929</a> dataset (<strong>Fig. 2B</strong>). Interestingly, we found that most of these gene signatures were positively related to each other (<strong>Fig. 2C</strong>), indicating their joint role in regulating tumor pathology. Together, these results highlight the potential association of genes included in our risk model with LUAD tumorigenesis.</p>
<figure class="fig xbox font-sm" id="fig2"><h4 class="obj_head">Figure 2. FIGURE 2: TME-related genes and the TMEindex exhibit prognostic and clinical implications.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373402_mic-12-182-g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e95/12373402/6ef2f8b10bd7/mic-12-182-g002.jpg" loading="lazy" height="719" width="778" alt="Figure 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Boxplots showing the differences in the expression of TME-related genes between tumor and tumor adjacent tissues in TCGA-LUAD. <strong>(B)</strong> Boxplots showing the differences in the expression of TME-related genes between tumor and tumor adjacent tissues in <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE101929" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE101929</a>. <strong>(C)</strong> Correlation network of the TME-related genes. The size of node indicates the statistical significance by univariate Cox regression analysis. <strong>(D)</strong> Barplot showing the association of the TME-related genes with clinical information. Kruskal-Wallis test was used to conduct the statistical test. <strong>(E)</strong> Survival curves of the high- and low-TMEindex group. Log-rank test was used to conducted the statistical test. <strong>(F)</strong> Differences in the clinical information between the high- and low-TMEindex group. Chi-square test was used to perform the statistical test. *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01, ***<em>P</em> &lt; 0.001 and ****<em>P</em> &lt; 0.0001. ns, not significant.</p></figcaption></figure><p>Next, we sought to explore the association of the TMEindex with clinical phenotypes and patient outcome in LUAD. First, we found that the expression of TME-related genes was significantly correlated with multiple clinical parameters, such as age, tumor stage, and tumor size (<strong>Fig. 2D</strong>, adjusted <em>P</em> &lt; 0.05). Furthermore, according to the median score, patients were divided into two groups, the high-TMEindex group and the low-TMEindex group. Survival analysis showed that patients in the high-TMEindex group exhibited a significantly shorter overall survival (OS) (<strong>Fig. 2E</strong>, <em>P</em> &lt; 0.0001). Moreover, we compared multiple clinical parameters between the two groups. The results showed a significantly higher proportion of patients with advanced LUAD in the high-TMEindex group than in the low-TMEindex group (<strong>Fig. 2F</strong>, <em>P</em> &lt; 0.05). Therefore, the TMEindex has profound clinical and prognostic implications.</p></section><section id="sec6"><h3 class="pmc_sec_title">Patients in the high-TMEindex group present immunosuppressive TME</h3>
<p>Immune cells can be directly or indirectly involved in the immune response, affecting tumor growth and therapeutic response <a href="#B31" class="usa-link" aria-describedby="B31">31</a>. Chemokines have the ability to induce directed chemotaxis of nearby reactive cells, which can recruit immune cells into tumors <a href="#B32" class="usa-link" aria-describedby="B32">32</a>. We detected that the expression of chemokines in the high-TMEindex group was significantly lower than that in the low-TMEindex group (<strong>Fig. 3A</strong> and <strong>3B</strong>). Next, we analyzed the profile of infiltrating immune cells inferred by multiple approaches, including xCELL, CIBERSORT, EPIC, MCPcounter, QuanTIseq and TIMER. Intriguingly, compared with the low-TMEindex group, the high-TMEindex group had a significantly lower degree of immune cell infiltration (<strong>Fig. 3C</strong>). In addition, we found a clear divergence of the expression of immune checkpoint-related genes (ICGs) (<strong>Fig. 3D</strong>, <em>P</em> = 0.001). Specifically, we identified 26 ICGs with significantly higher expression in the low-TMEindex group, such as <em>CD86 </em>and <em>HLA-E</em> (<strong>Fig. 3E</strong>, adjusted <em>P</em> &lt; 0.05). In conclusion, our prognostic model accurately delineates the degree of immune infiltration in patients with LUAD, and the difference in immune infiltration between the two groups explains the significant difference in prognosis.</p>
<figure class="fig xbox font-sm" id="fig3"><h4 class="obj_head">Figure 3. FIGURE 3: High-TMEindex group showed immunosuppressive characteristics compared with low-TMEindex.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373402_mic-12-182-g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e95/12373402/fe96da77063a/mic-12-182-g003.jpg" loading="lazy" height="810" width="776" alt="Figure 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> PCoA showing the differences in the expression of chemokines between the high- and low-TMEindex. <strong>(B)</strong> Heamap showing the expression of chemokines in the high- and low-TMEindex. <strong>(C)</strong> Heatmap showing the normalized abundance of various immune cells in the high- and low-TMEindex. <strong>(D)</strong> PCoA showing the differences in the expression of immune checkpoint genes between the high- and low-TMEindex. <strong>(E)</strong> Barbell plot showing the difference in the expression of immune checkpoint genes between the two groups. Statistical tests were performed using Wilcoxon test. **<em>P</em> &lt; 0.01 and ***<em>P</em> &lt; 0.001.</p></figcaption></figure></section><section id="sec7"><h3 class="pmc_sec_title">Intratumoral microbiome is closely related to the TMEindex</h3>
<p>The microbiota can regulate anti-tumor immunity through its interaction with the TME and influence the efficacy of cancer immunotherapy <a href="#B33" class="usa-link" aria-describedby="B33">33</a>. We comprehensively characterized the intratumoral microbial community profile in patients with LUAD. At the genus level, in both two groups, genus <em>Pseudomonas</em> had the highest relative abundance, followed by <em>Neisseria </em>and <em>Mycobacterium </em>(<strong>Fig. 4A</strong>). The Shannon and Simpson indices of the high-TMEindex group were significantly higher than those of the low-TMEindex group, and there was no significant difference in the ACE and Chao1 indices between the two groups (<strong>Fig. 4B</strong>, <em>P</em> &lt; 0.05). In addition, we also observed a divergence of the microbial community at the genus level between the two groups (<strong>Fig. 4C</strong>, <em>P</em> = 0.038). Next, Linear discriminant analysis (LDA) effect size (LEfSe) analysis was performed to identify the microbial biomarkers. A total of 29 genera were identified, of which eight were enriched in the low-TMEindex group and 21 were more abundant in the high-TMEindex group (<strong>Fig. 4D</strong>).</p>
<figure class="fig xbox font-sm" id="fig4"><h4 class="obj_head">Figure 4. FIGURE 4: Association of the TMEindex with intratumoral microbiome.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373402_mic-12-182-g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e95/12373402/1d77f1e9f15a/mic-12-182-g004.jpg" loading="lazy" height="779" width="701" alt="Figure 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Barplot showing the relative abundance of top ten genera in the high- and low-TMEindex groups. <strong>(B)</strong> Boxplot showing the differences in alpha diversity of microbial community between the high- and low-TMEindex groups. <strong>(C)</strong> PCoA showing the difference in microbiota composition between the high- and low-TMEindex groups. <strong>(D)</strong> Microbial biomarkers at the genus level identified by LEfSe. <strong>(E) </strong>Heatmap showing the Spearman correlation between the microbial biomarkers and TME-related genes and the TMEindex. *<em>P</em> &lt; 0.05, **<em>P</em> &lt; 0.01 and ***<em>P</em> &lt; 0.001.</p></figcaption></figure><p>Next, we explored the relationship between the 29 microbial biomarkers with our prognostic model. Interestingly, we observed that the microbes enriched in the low-TMEindex group were negatively correlated with the TMEindex score and positively correlated with the TME-related genes. On the contrary, the microbes enriched in the high-TMEindex group were positively correlated with the TMEindex score and negatively correlated with the TME-related genes (<strong>Fig. 4E</strong>). Specifically, the genus <em>Luteibacter</em>, enriched in the low-TMEindex group, was significantly negatively correlated with the TMEindex score. In addition, the genus <em>Chryseobacterium</em>, enriched in the high-TMEindex group, was significantly positively correlated with the TMEindex scores. In summary, our results suggest that intratumoral microbes closely interact with the TME and have the potential to be integrated into the prognostic model.</p></section><section id="sec8"><h3 class="pmc_sec_title">The TMEindex can inform immunotherapy and drug sensitivity</h3>
<p>The application of immunotherapy to cancer has high clinical value, but only a subset of patients show response to immunotherapy <a href="#B34" class="usa-link" aria-describedby="B34">34</a>. The degree of immune invasion can usually reflect the prognosis and immune response of the tumor <a href="#B35" class="usa-link" aria-describedby="B35">35</a>. First, we found that the abundance of immune cell populations was significantly correlated with the expression of the TME-related genes (<strong>Fig. 5A</strong>). Next, we observed that the Tumor immun dysfunction and exclusion (TIDE) score of the high-TMEindex group was significantly higher than that in the low-TMEindex group (<strong>Fig. 5B</strong>, <em>P </em> &lt; 0.05, Table S2), potentially indicating that patients in the high-TMEindex group showed adverse response to immunotherapy. Moreover, we detected that the expression of <em>GCSAML</em>, <em>APOC4−APOC2</em>, <em>CHRDL1</em> was significantly positively correlated with the TIDE score, while the expression of <em>GPR1</em> and <em>CPS1</em> was significantly negatively correlated with the TIDE score (<strong>Fig. 5C</strong>).</p>
<figure class="fig xbox font-sm" id="fig5"><h4 class="obj_head">Figure 5. FIGURE 5: Potential role of the TMEindex in immunotherapy and conventional therapy.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373402_mic-12-182-g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e95/12373402/bc8a6fc20bba/mic-12-182-g005.jpg" loading="lazy" height="781" width="718" alt="Figure 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Heatmap showing the association of the expression of TME-related genes with immune cells. <strong>(B)</strong> Boxplot showing the difference in the TIDE value between the high- and low-TMEindex groups. <strong>(C)</strong> Heatmap showing the association of the expression of the TME-related genes with TIDE value. <strong>(D)</strong> PCoA showing the differences in IC50 values of drugs between the two groups. <strong>(E)</strong> Heatmap showing the association of the expression of the TME-related genes with drug sensitivity. <strong>(F-H)</strong> Boxplots showing the differences in sensitivity of three drug candidates between the two groups. <strong>(I)</strong> Heatmap showing the association of the drug sensitivity with the TME characteristics, microbial biomarkers, and the TMEindex.</p></figcaption></figure><p>Next, we sought to explore the difference in biological function between the two groups. The DEGs between the high- and low-TMEindex were mainly enriched in hormone activity regulation, hormone metabolism, extracellular matrix organization, antibacterial humoral immune response, potential involvement in the interaction between neuroactive substances and their receptors, as well as steroid hormone biosynthesis (Fig. S1A-S1C).</p>
<p>Considering the importance of combination therapy in the treatment of LUAD, drug sensitivity analysis was conducted. The IC50 value of 198 drugs for each patient was predicted using R package "oncoPredict". We observed a clear divergence in drug sensitivity between the high- and low-TMEindex group (<strong>Fig. 5D</strong>, <em>P</em> = 0.001). We further explored the relationship between drug sensitivity and expression of the TME-related genes, and found that <em>IKZF3</em>, <em>P2RX1</em>, <em>CD70</em>, <em>GPR1</em>, <em>KLRC2</em>, and <em>CPS1</em> were potently associated with most of drugs (<strong>Fig. 5E</strong>). Specifically, we observed that patients in the high-TMEindex group could benefit from GDC0810, while Doramapimod and SB216763 were fit for patients in the low-TMEindex group (<strong>Fig. 5F-5G</strong>). Additionally, we comprehensively investigated the association of the TME characteristics, microbial biomarkers, and TMEindex with drug sensitivity, respectively (<strong>Fig. 5I</strong>). By Spearman correlation analysis, we found that drugs enriched in the high-TMEindex group was mainly positively correlated with the TMEindex and the microbial markers enriched in the high-TMEindex group (<strong>Fig. 5I</strong>). In addition, stromal score, immune score and microbial biomarkers enriched in the low-TMEindex showed the opposite relationship with drug sensitivity. Consequently, these results demonstrate that the TMEindex can identify patients fit for immunotherapy and can predict patients' sensitivity to conventional drugs.</p></section><section id="sec9"><h3 class="pmc_sec_title">Integrating microbial biomarkers and the TME characteristics for prognosis prediction</h3>
<p>Although the interaction between intratumoral microbiota and the TME has been revealed, the fusion of the two to establish a prognostic model has not yet been implemented. Samples of TCGA-LUAD were assigned to the training set at a ratio of 70% and to the testing set at a ratio of 30%, and the partitioning and training process was repeated 20 times. First, Cox risk proportional regression model was constructed based on the TME-related genes, microbial biomarkers, and combination of the two on the training set, respectively (<strong>Fig. 6A-6C</strong>). The mean The Area Under the ROC curve (AUC) of the model for predicting 1-, 3-, and 5-year survival on the testing set was 0.7, 0.67, and 0.66, respectively (<strong>Fig. 6D</strong>). Secondly, Cox risk proportional regression model was established based on microbial biomarkers, and the average AUC of 1-, 3-, and 5-year survival rates predicted on the testing set were 0.53, 0.53, and 0.56, respectively (<strong>Fig. 6E</strong>). Finally, the combination of the TME-related genes and microbial biomarkers were input into the Cox model, and the model predicted the mean AUC of 1-, 3-, and 5-year survival on the testing set to be 0.7, 0.67, and 0.65, respectively (<strong>Fig. 6F</strong>). We aggregated the C index and AUC values of all models and found that the model established by mixed features had a better effect (<strong>Fig. 6G-6J</strong>, Table S3-S5). At this point, we finally determined the final model, called intratumoral microbiome-modified TMEindex (IMTMEindex) (Table S6)：</p>
<p>IMTMEindex = 0.43657 x GCSAML - 0.24064 x APOC4 - APOC2 + 0.16043 x CHRDL1 + 0.15889 x GPR1 + 0.14000 x CPS1 - 0.30345 x IKZF3 + 0.26408 x CD70 + 0.29906 x KLRC2 - 0.20135 x g_Terrabacter + 0.14646 x g_Effusibacillus + 0.19592 x Marinithermus + 0.27673 x g_Renibacterium - 0.62063 x g_Achromobacter + 0.24935 x g_Herbaspirillum + 0.19307 x g_Chlamydia - 4.33628 x g_Lactobacillus - 0.20560 x g_Aquimarina + 0.61934 x g_Klebsiella + 0.22734 x g_Streptococcus</p>
<figure class="fig xbox font-sm" id="fig6"><h4 class="obj_head">Figure 6. FIGURE 6: Assessment of the survival prediction ability of the TME-related genes and intratumoral microbiome.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373402_mic-12-182-g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1e95/12373402/567da8617fe7/mic-12-182-g006.jpg" loading="lazy" height="627" width="716" alt="Figure 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A-C) </strong>Average ROC curves for predicting 1-, 3-, and 5-year survival on training set, based on TME-related genes, microbial biomarkers, and their combination, respectively. <strong>(D-E)</strong> Average ROC curves for predicting 1-, 3-, and 5-year survival on testing set. (G-J) Boxplots showing the differences in mean C index and the mean AUC among the three models. Statistical tests were performed using Wilcoxon test.</p></figcaption></figure></section></section><section id="sec10"><h2 class="pmc_sec_title">DISCUSSION</h2>
<p>The intratumoral microbiome is related to the occurrence, development and metastasis of tumors, and may even affect the prognosis of patients <a href="#B36" class="usa-link" aria-describedby="B36">36</a>. Here, we constructed a 9-gene signature to characterize the TME. Besides, we built a scoring system TMEindex based on these nine genes. The results showed that clinical information, response to immunotherapy, drug sensitivity, and intratumoral microbiota of patients with LUAD were significantly different between the high- and low-TMEindex groups. We identified specific microbial biomarkers that could affect the prognosis of patients with LUAD, and based on this, we established the IMTMEindex. It was found that combining the characteristics of intratumoral microbiota and the TME signatures was more effective in predicting the prognosis of patients.</p>
<p>LUAD-related genes are being studied. Deng <em>et al</em>. reported that <em>CHRDL1</em> expression showed a good ability to distinguish LUAD tissue from normal samples. Low <em>CHRDL1</em> is associated with adverse clinicopathological features and poor OS <a href="#B37" class="usa-link" aria-describedby="B37">37</a>. Yang <em>et al</em>. found that low expression of <em>P2RX1 </em>associated with poor OS and decreased immune infiltration <a href="#B38" class="usa-link" aria-describedby="B38">38</a>. Wen <em>et al</em>. reported that P2RX1 is a risk factor that has an inverse relationship with risk score <a href="#B39" class="usa-link" aria-describedby="B39">39</a>. In the process of extracting TME characteristics, nine TME-related genes were screened by LASSO-Cox regression. Among them, the expression of <em>CHRDL1</em>, <em>P2RX1</em>, <em>GCSAML</em>, and <em>APOC4-APOC2</em> in the tumor region was significantly lower than that in the normal region. This suggested that these genes may be protective genes. These nine genes were also associated with clinical characteristics such as age and TNM stage. These results are consistent with previous studies.</p>
<p>Some studies have shown that the infiltration level of immune cells in TME is directly related to immune efficacy <a href="#B40" class="usa-link" aria-describedby="B40">40</a>,<a href="#B41" class="usa-link" aria-describedby="B41">41</a>,<a href="#B42" class="usa-link" aria-describedby="B42">42</a>,<a href="#B43" class="usa-link" aria-describedby="B43">43</a>. Our results showed that the infiltration degree of T lymphocytes (including CD4+ and CD8+T cells), B lymphocytes, NK cells and dendritic cells in the high-TMEindex group was significantly lower than that in the low-TMEindex group. ICGs play pivotal roles in tumor immune microenvironment (TIME) <a href="#B44" class="usa-link" aria-describedby="B44">44</a>. In the comparison of immune checkpoint gene expression between the two groups, most ICGs, especially HLA families, also showed lower expression in the high-TMEindex group. We found that patients in the high-TMEindex group had a suppressed immune system compared to those in the low-TMEindex group. Lower TIDE score indicated that patients in the low-TMEindex group might respond better to immunotherapy.</p>
<p>We found that there may be some interaction between microorganisms and the TME. Surprisingly, the microbial biomarkers enriched in the low-TMEindex group were negatively correlated with the TMEindex score, while the microbial biomarkers significantly enriched in the high-TMEindex group were positively correlated with the TMEindex score. In particular, relevant conclusions have been found not only in our study, but also in previous studies on <em>Luteibacter</em> and <em>Chryseobacterium</em>. Deng <em>et al</em>. found that in patients with LUAD, the abundance of <em>Luteibacter</em> was lower in the stage III-IV group <a href="#B45" class="usa-link" aria-describedby="B45">45</a>. Zheng <em>et al</em>. found that among tumor-resident microorganisms in patients with non-small cell lung cancer (NSCLC), the abundance of <em>Chryseobacterium</em> is different in tumor and adjacent tissue. <em>Chryseobacterium</em> was significantly increased in a group of tumor-bearing mice that inhaled bronchoalveolar fluid (BAL) collected from patients with NSCLC <a href="#B46" class="usa-link" aria-describedby="B46">46</a>. Ye <em>et al</em>. found that Pachymaran (PCP), a drug ingredient, can intervene to significantly reduce the abundance of <em>Chrysobacterium</em>, thereby reducing immunosuppressive lung injury <a href="#B47" class="usa-link" aria-describedby="B47">47</a>. Based on these studies, our study uncovered the prognostic implications of tumor-resident microbes in patients with LUAD and demonstrated the close crosstalk between the TME and tumor-resident microbes.</p>
<p>At present, in order to accurately predict the prognosis of patients with LUAD, many risk models have been established. Yang <em>et al</em>. used the TCGA database to screen long non-coding RNA-associated immune genes (lrRIGs) associated with LUAD prognosis, and then constructed a tag consisting of nine prognostic genes. This risk score showed excellent performance in predicting the OS rate <a href="#B48" class="usa-link" aria-describedby="B48">48</a>. Wu <em>et al</em>. constructed a TMEindex model to predict OS in patients with LUAD. Functionally, TMEindex score is related to immune response in patients with LUAD <a href="#B49" class="usa-link" aria-describedby="B49">49</a>. Deng <em>et al</em>. established a prognosis score derived from the abundance of microorganisms in tumors of patients with LUAD, namely lung resident microbial score (LMS), and evaluated its relationship with LUAD prognosis and immune invasion <a href="#B45" class="usa-link" aria-describedby="B45">45</a>. In our study, a risk model scoring system IMTMEindex, was established with mixed TME-related genes and microbial biomarkers. Moreover, the C index of IMTMEindex was significantly higher than models based solely on the TME characteristics. IMTMEindex may be used as a prognostic predictor of LUAD and provide valuable information for precise treatment.</p>
<p>Current studies are mostly based on bulk sequencing data. In the future, single-cell sequencing, spatial transcriptomics and other technologies can be combined to analyze the dynamic interaction network of microorganisms with immune cells and stromal cells within tumors, reveal how microbes directly affect TME function through metabolites (such as short-chain fatty acid, tryptophan derivatives) or immunomodulatory molecules (such as PD-L1). For example, specific bacterial species may contribute to immunosuppression by inducing polarization of M2 macrophages, a process that can be precisely targeted by single-cell techniques. In addition, microbial characteristics may become a new dimension of stratified treatment. For example, in patients with microbiota-driven immunosuppressive TME, combinations of antibiotics (such as narrow-spectrum antibiotics against specific strains), probiotic supplementation, or phage therapy, may be indicated, synergism with ICIs. Prospective clinical trials are needed to verify the effect of microbial regulation on the improvement of ICIs response rate.</p>
<p>There are several limitations in our study. First, the intratumoral microbiome data in our study have limited resolution and only genus-level annotations. In the future, metagenomic sequencing technology will be applied to lung tumor samples to obtain intratumoral microbial information at the species and even strain level. Second, independent data on the intratumoral microbiome abundance are not available. However, it is important to note that host transcriptome samples and matched intratumoral microbiome samples from 478 patients were included in our study, already a relatively large sample size to support our findings. Future accumulations of larger lung tumor microbiome data will refine our results. Finally, our results are largely based on correlations and hypotheses, and more <em>in vitro</em> and <em>in vivo</em> trials are needed in the future to explore the mechanisms of interaction between the tumor microbiota and the TME, as well as causality.</p>
<p>Overall, our study investigated the interaction between the TME signatures and intratumoral microbiota in LUAD. We found that tumor-resident <em>Luteibacter </em>and <em>Chryseobacterium </em>were substantially associated with survival risk of patients with LUAD. Besides, the combination of TME characteristics and intratumoral microbiome introduced additionally prognostic implications. IMTMEindex was a promising biomarker for predciting prognosis and therapeutic opportunities in LUAD. Further studies are needed to clarify the regulatory mechanisms of specific intratumoral microbes on LUAD microecology.</p></section><section id="sec11"><h2 class="pmc_sec_title">MATERIALS AND METHODS</h2>
<section id="sec12"><h3 class="pmc_sec_title">Data acquisition</h3>
<p>A total of 478 patients with LUAD were included in our study. Each patient had a host transcriptome sample and a matched intratumoral microbiome sample. The gene expression data was obtained from TCGA (https://portal.gdc.cancer.gov/). Clinical metadata of patients was downloaded from UCSC Xena (<a href="http://xena.ucsc.edu" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://xena.ucsc.edu</a> . The majority of the samples were primary solid tumors, with a small number of recurrent or metastatic tumors. The sample covers stage I-IV patients, with the highest proportion of stage I (about 40%) and the lowest proportion of stage IV (about 10%). Intratumoral microbiome abundance data was obtained from the work by Narunsky-Haziza <em>et al</em>. and was downloaded from https://github.com/knightlab-analyses/mycobiome. The quality control and taxonomic annotation were detailed in the original paper <a href="#B50" class="usa-link" aria-describedby="B50">50</a>. Independent validation dataset <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE101929" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE101929</a> was downloaded from The Gene Expression Omnibus (GEO) dataset (https://www.ncbi.nlm.nih.gov/geo/).</p></section><section id="sec13"><h3 class="pmc_sec_title">Construction of prognostic model</h3>
<p>The DEGs were identified with R package "DESeq2" <a href="#B51" class="usa-link" aria-describedby="B51">51</a>. Then, Weighted Gene Co-expression Network Analysis (WGCNA) was performed using R package "WGCNA" to identify genes associated with the stromal and immune scores <a href="#B52" class="usa-link" aria-describedby="B52">52</a>. Subsequently, least absolute shrinkage and selection operator (LASSO) <a href="#B53" class="usa-link" aria-describedby="B53">53</a> regression algorithm was conducted to further screen. Then, the multivariate Cox proportional hazards regression analysis was performed to calculate the coefficient of each gene. Therefore, the final prognostic model, called the tumor microenvironment-related index (TMEindex), was as follows:</p>
<p>TMErisk = 0.93117 x GCSAML - 1.75862 x APOC4 - APOC2 + 0.11645 x CHRDL1 + 0.41756 x GPR1 + 0.06990 x CPS1 - 0.36391 x IKZF3 - 0.36001 x P2RX1 + 0.47444 x CD70 + 0.54178 x KLRC2</p></section><section id="sec14"><h3 class="pmc_sec_title">Immune analyses</h3>
<p>Tumor immun dysfunction and exclusion (TIDE) algorithm simulates the mechanism of tumor immune escape by inducing T cell dysfunction in tumors with high infiltrating cytotoxic T lymphocytes (CTL) and preventing T cell infiltration in tumors with low infiltrating cytotoxic T lymphocytes, thereby predicting patients' immunotherapy response <a href="#B54" class="usa-link" aria-describedby="B54">54</a>. TIDE algorithm was used to evaluate the possibility of tumor immune escape in gene expression profiles of tumor samples. Patients with LUAD were divided into two groups according to the expression levels of CTL markers (<em>CD8A</em>, <em>CD8B</em>, <em>GZMA</em>, <em>GZMB</em>, and <em>PRF1</em>). In patients with high CTL marker levels, the correlation between the expression profile of each patient and the signature of T cell dysfunction was used as the TIDE score. In patients with low CTL marker levels, the correlation between the expression profile of each patient and T cell rejection signature was taken as the TIDE score.</p>
<p>Estimation of STromal and Immune cells in Malignant Tumor tissues using Expression data (ESTIMATE) algorithm <a href="#B55" class="usa-link" aria-describedby="B55">55</a> was used to obtain the immune score and stromal score of patients with LUAD. The CIBERSORT <a href="#B56" class="usa-link" aria-describedby="B56">56</a>, TIMER <a href="#B57" class="usa-link" aria-describedby="B57">57</a>, xCell <a href="#B58" class="usa-link" aria-describedby="B58">58</a>, EPIC <a href="#B59" class="usa-link" aria-describedby="B59">59</a>, quanTIseq <a href="#B60" class="usa-link" aria-describedby="B60">60</a> and MCPcounter <a href="#B61" class="usa-link" aria-describedby="B61">61</a> tools were used to assess the abundance of immune cells in the TME.</p></section><section id="sec15"><h3 class="pmc_sec_title">Microbial analyses</h3>
<p>The alpha diversity of microbial communities was measured by the Shannon, Simpson, ACE and Chao1 indices. The beta diversity analysis was carried out by Principal Co-ordinates Analysis (PCoA) based on Bray-Curtis distance matrix. Linear discriminant analysis (LDA) effect size (LEfSe) was used to identify the genera with significant differences in abundance <a href="#B62" class="usa-link" aria-describedby="B62">62</a>.</p></section><section id="sec16"><h3 class="pmc_sec_title">Other statistical analysis</h3>
<p>All statistical analyses were performed using RStudio (version 4.3.2). K-M curves and log-rank tests were used to plot survival curves and compare survival differences. Wilcoxon test was used to explore the differences between the high- and low-TMEindex groups in feature genes, TIDE scores, immune checkpoint genes and other continuous variables. Chi-square test was used to compare the differences in clinical traits between the high- and low-TMEindex. Kruskal-Wallis test was used to explore the relationship between feature genes and clinical traits. The gene expression profile, microbial community, and drug sensitivity between the high- and low-TMEindex groups were delineated using Principal Coordinate Analysis (PCoA) and Permutational Multivariate Analysis of Variance (PERMANOVA). Spearman correlation analysis was used to calculate correlations between feature genes, between feature genes and immune cells, between feature genes and drug IC50 values, and between immune cells and microbial abundance. All comparisons reached statistical significance when the two-tailed adjusted p-value was less than 0.05.</p></section><section id="sec17"><h3 class="pmc_sec_title">Availability of data and material</h3>
<p>The data supporting the findings of this study are available in Materials and methods. Further inquiries can be directed to the corresponding author.</p></section></section><section id="sec18"><h2 class="pmc_sec_title">AUTHOR CONTRIBUTION</h2>
<p>F Z (Fei Zhao): Formal analysis, Writing—original draft, Writing—review &amp; editing, Validation. L W (Lei Wang): Software, Methodology, Formal analysis. D D (Dongjie Du) and H Z (Heaven Zhao): Data curation, Writing—review &amp; editing, Software. G T (Geng Tian): Software, Writing—review &amp; editing. J L (Jingwu Li): Data curation, Writing—review &amp; editing. Y L (Yufeng Li): Software, Writing—review &amp; editing. Y L (Yankun Liu): Software, Writing—review &amp; editing. Z W (Zhiwu Wang): Conceptualization, Writing-review &amp; editing. D L (Dasheng Liu): Software, Writing—review &amp; editing. L J (Lei Ji): Conceptualization, Writing-review &amp; editing. H Z (Hong Zhao): Supervision, Project administration. All authors have read and approved the final manuscript.</p></section><section id="sec19"><h2 class="pmc_sec_title">CONFLICT OF INTEREST</h2>
<p>The authors declare no competing interests.</p></section><section id="sec20"><h2 class="pmc_sec_title">SUPPLEMENTAL MATERIAL</h2>
<section class="sm xbox font-sm" id="mic-12-182-s01"><div class="media p" id="d67e1051"><div class="caption">
<a href="/articles/instance/12373402/bin/mic-12-182-s01.pdf" data-ga-action="click_feat_suppl" class="usa-link">Click here for supplemental data file.</a><sup> (985.5KB, pdf) </sup>
</div></div></section><p>All supplemental data for this article are available online at <a href="https://www.microbialcell.com/researcharticles/2025a-zhao-microbial-cell/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.microbialcell.com/researcharticles/2025a-zhao-microbial-cell/</a>.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This study was supported by Hebei Natural Science Foundation NO.H2023105017, Hebei Natural Science Foundation NO.H2024105019, 2024 Science and Technology Planning Project Fund of Tangshan NO.24150220C, Shandong Natural Science Foundation NO.ZR2021MD035 and Shandong Higher Education Undergraduate Teaching Reform Research Project NO.Z2023168. We thank professor Kwok Pui Choi for his support in polishing the manuscript.</p></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Jingwu Li, Email: tslijingwu@163.com.</p>
<p>Lei Ji, Email: jil@geneis.cn.</p>
<p>Hong Zhao, Email: zhaohong@ouc.edu.cn.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">REFERENCES</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="B1">
<span class="label">1.</span><cite>Bray F, Ferlay J, Soerjomataram I, Siegel RL, Torre LA, Jemal A. Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries. CA Cancer J Clin. 2018;68(6):394–424. doi: 10.3322/caac.21492.</cite> [<a href="https://doi.org/10.3322/caac.21492" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30207593/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=CA%20Cancer%20J%20Clin&amp;title=Global%20cancer%20statistics%202018:%20GLOBOCAN%20estimates%20of%20incidence%20and%20mortality%20worldwide%20for%2036%20cancers%20in%20185%20countries.&amp;author=F%20Bray&amp;author=J%20Ferlay&amp;author=I%20Soerjomataram&amp;author=RL%20Siegel&amp;author=LA%20Torre&amp;volume=68&amp;issue=6&amp;publication_year=2018&amp;pages=394-424&amp;pmid=30207593&amp;doi=10.3322/caac.21492&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B2">
<span class="label">2.</span><cite>Barta JA, Powell CA, Wisnivesky JP. Global Epidemiology of Lung Cancer. Ann Glob Health. 2019;85(1):8. doi: 10.5334/aogh.2419.</cite> [<a href="https://doi.org/10.5334/aogh.2419" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6724220/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30741509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ann%20Glob%20Health&amp;title=Global%20Epidemiology%20of%20Lung%20Cancer.&amp;author=JA%20Barta&amp;author=CA%20Powell&amp;author=JP%20Wisnivesky&amp;volume=85&amp;issue=1&amp;publication_year=2019&amp;pages=8&amp;pmid=30741509&amp;doi=10.5334/aogh.2419&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B3">
<span class="label">3.</span><cite>Sivakumar S, Lucas FAS, McDowell TL, Lang W, Xu L, Fujimoto J, Zhang J, Futreal PA, Fukuoka J, Yatabe Y, Dubinett SM, Spira AE, Fowler J, Hawk ET, Wistuba II, Scheet P, Kadara H. Genomic Landscape of Atypical Adenomatous Hyperplasia Reveals Divergent Modes to Lung Adenocarcinoma. Cancer Res. 2017;77(22):6119–6130. doi: 10.1158/0008-5472.Can-17-1605.</cite> [<a href="https://doi.org/10.1158/0008-5472.Can-17-1605" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5774855/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28951454/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Genomic%20Landscape%20of%20Atypical%20Adenomatous%20Hyperplasia%20Reveals%20Divergent%20Modes%20to%20Lung%20Adenocarcinoma.&amp;author=S%20Sivakumar&amp;author=FAS%20Lucas&amp;author=TL%20McDowell&amp;author=W%20Lang&amp;author=L%20Xu&amp;volume=77&amp;issue=22&amp;publication_year=2017&amp;pages=6119-6130&amp;pmid=28951454&amp;doi=10.1158/0008-5472.Can-17-1605&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B4">
<span class="label">4.</span><cite>Zhou X, Ji L, Ma Y, Tian G, Lv K, Yang J. Intratumoral Microbiota-Host Interactions Shape the Variability of Lung Adenocarcinoma and Lung Squamous Cell Carcinoma in Recurrence and Metastasis. Microbiol Spectr. 2023;11(3):e0373822. doi: 10.1128/spectrum.03738-22.</cite> [<a href="https://doi.org/10.1128/spectrum.03738-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10269859/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37074188/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol%20Spectr&amp;title=Intratumoral%20Microbiota-Host%20Interactions%20Shape%20the%20Variability%20of%20Lung%20Adenocarcinoma%20and%20Lung%20Squamous%20Cell%20Carcinoma%20in%20Recurrence%20and%20Metastasis.&amp;author=X%20Zhou&amp;author=L%20Ji&amp;author=Y%20Ma&amp;author=G%20Tian&amp;author=K%20Lv&amp;volume=11&amp;issue=3&amp;publication_year=2023&amp;pages=e0373822&amp;pmid=37074188&amp;doi=10.1128/spectrum.03738-22&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B5">
<span class="label">5.</span><cite>Bender E. Epidemiology: The dominant malignancy. Nature. 2014;513(7517):S2–S3. doi: 10.1038/513S2a.</cite> [<a href="https://doi.org/10.1038/513S2a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25208070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Epidemiology:%20The%20dominant%20malignancy.&amp;author=E%20Bender&amp;volume=513&amp;issue=7517&amp;publication_year=2014&amp;pages=S2-S3&amp;pmid=25208070&amp;doi=10.1038/513S2a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B6">
<span class="label">6.</span><cite>Jamal-Hanjani M, Quezada SA, Larkin J, Swanton C. Translational implications of tumor heterogeneity. Clin Cancer Res. 2015;21(6):1258–1266. doi: 10.1158/1078-0432.Ccr-14-1429.</cite> [<a href="https://doi.org/10.1158/1078-0432.Ccr-14-1429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4374162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25770293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Clin%20Cancer%20Res&amp;title=Translational%20implications%20of%20tumor%20heterogeneity.&amp;author=M%20Jamal-Hanjani&amp;author=SA%20Quezada&amp;author=J%20Larkin&amp;author=C%20Swanton&amp;volume=21&amp;issue=6&amp;publication_year=2015&amp;pages=1258-1266&amp;pmid=25770293&amp;doi=10.1158/1078-0432.Ccr-14-1429&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B7">
<span class="label">7.</span><cite>Binnewies M, Roberts EW, Kersten K, Chan V, Fearon DF, Merad M, Coussens LM, Gabrilovich DI, Ostrand-Rosenberg S, Hedrick CC, Vonderheide RH, Pittet MJ, Jain RK, Zou W, Howcroft TK, Woodhouse EC, Weinberg RA, Krummel MF. Understanding the tumor immune microenvironment (TIME) for effective therapy. Nature Med. 2018;24(5):541–550. doi: 10.1038/s41591-018-0014-x.</cite> [<a href="https://doi.org/10.1038/s41591-018-0014-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5998822/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29686425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Med&amp;title=Understanding%20the%20tumor%20immune%20microenvironment%20(TIME)%20for%20effective%20therapy.&amp;author=M%20Binnewies&amp;author=EW%20Roberts&amp;author=K%20Kersten&amp;author=V%20Chan&amp;author=DF%20Fearon&amp;volume=24&amp;issue=5&amp;publication_year=2018&amp;pages=541-550&amp;pmid=29686425&amp;doi=10.1038/s41591-018-0014-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B8">
<span class="label">8.</span><cite>Bilotta MT, Antignani A, Fitzgerald DJ. Managing the TME to improve the efficacy of cancer therapy. Front Immunol. 2022;13:954992. doi: 10.3389/fimmu.2022.954992.</cite> [<a href="https://doi.org/10.3389/fimmu.2022.954992" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9630343/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36341428/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&amp;title=Managing%20the%20TME%20to%20improve%20the%20efficacy%20of%20cancer%20therapy.&amp;author=MT%20Bilotta&amp;author=A%20Antignani&amp;author=DJ%20Fitzgerald&amp;volume=13&amp;publication_year=2022&amp;pages=954992&amp;pmid=36341428&amp;doi=10.3389/fimmu.2022.954992&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B9">
<span class="label">9.</span><cite>de Visser KE, Joyce JA. The evolving tumor microenvironment: From cancer initiation to metastatic outgrowth. Cancer Cell. 2023;41(3):374–403. doi: 10.1016/j.ccell.2023.02.016.</cite> [<a href="https://doi.org/10.1016/j.ccell.2023.02.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36917948/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Cell&amp;title=The%20evolving%20tumor%20microenvironment:%20From%20cancer%20initiation%20to%20metastatic%20outgrowth.&amp;author=KE%20de%20Visser&amp;author=JA%20Joyce&amp;volume=41&amp;issue=3&amp;publication_year=2023&amp;pages=374-403&amp;pmid=36917948&amp;doi=10.1016/j.ccell.2023.02.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B10">
<span class="label">10.</span><cite>Denton AE, Roberts EW, Fearon DT. Stromal Cells in the Tumor Microenvironment. Adv Exp Med Biol. 2018;1060:99–114. doi: 10.1007/978-3-319-78127-3_6.</cite> [<a href="https://doi.org/10.1007/978-3-319-78127-3_6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30155624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Adv%20Exp%20Med%20Biol&amp;title=Stromal%20Cells%20in%20the%20Tumor%20Microenvironment.&amp;author=AE%20Denton&amp;author=EW%20Roberts&amp;author=DT%20Fearon&amp;volume=1060&amp;publication_year=2018&amp;pages=99-114&amp;pmid=30155624&amp;doi=10.1007/978-3-319-78127-3_6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B11">
<span class="label">11.</span><cite>Cao Y, Xia H, Tan X, Shi C, Ma Y, Meng D, Zhou M, Lv Z, Wang S, Jin Y. Intratumoural microbiota: a new frontier in cancer development and therapy. Signal Transduct Target Ther. 2024;9(1):15. doi: 10.1038/s41392-023-01693-0.</cite> [<a href="https://doi.org/10.1038/s41392-023-01693-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10776793/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38195689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Signal%20Transduct%20Target%20Ther&amp;title=Intratumoural%20microbiota:%20a%20new%20frontier%20in%20cancer%20development%20and%20therapy.&amp;author=Y%20Cao&amp;author=H%20Xia&amp;author=X%20Tan&amp;author=C%20Shi&amp;author=Y%20Ma&amp;volume=9&amp;issue=1&amp;publication_year=2024&amp;pages=15&amp;pmid=38195689&amp;doi=10.1038/s41392-023-01693-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B12">
<span class="label">12.</span><cite>Mouradov D, Greenfield P, Li S, In EJ, Storey C, Sakthianandeswaren A, Georgeson P, Buchanan DD, Ward RL, Hawkins NJ, Skinner I, Jones IT, Gibbs P, Ma C, Liew YJ, Fung KYC, Sieber OM. Oncomicrobial Community Profiling Identifies Clinicomolecular and Prognostic Subtypes of Colorectal Cancer. Gastroenterology. 2023;165(1):104–120. doi: 10.1053/j.gastro.2023.03.205.</cite> [<a href="https://doi.org/10.1053/j.gastro.2023.03.205" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36933623/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Oncomicrobial%20Community%20Profiling%20Identifies%20Clinicomolecular%20and%20Prognostic%20Subtypes%20of%20Colorectal%20Cancer.&amp;author=D%20Mouradov&amp;author=P%20Greenfield&amp;author=S%20Li&amp;author=EJ%20In&amp;author=C%20Storey&amp;volume=165&amp;issue=1&amp;publication_year=2023&amp;pages=104-120&amp;pmid=36933623&amp;doi=10.1053/j.gastro.2023.03.205&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B13">
<span class="label">13.</span><cite>Yuan X, Wang Z, Li C, Lv K, Tian G, Tang M, Ji L, Yang J. Bacterial biomarkers capable of identifying recurrence or metastasis carry disease severity information for lung cancer. Front Microbiol. 2022;13:1007831. doi: 10.3389/fmicb.2022.1007831.</cite> [<a href="https://doi.org/10.3389/fmicb.2022.1007831" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9523266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36187983/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Bacterial%20biomarkers%20capable%20of%20identifying%20recurrence%20or%20metastasis%20carry%20disease%20severity%20information%20for%20lung%20cancer.&amp;author=X%20Yuan&amp;author=Z%20Wang&amp;author=C%20Li&amp;author=K%20Lv&amp;author=G%20Tian&amp;volume=13&amp;publication_year=2022&amp;pages=1007831&amp;pmid=36187983&amp;doi=10.3389/fmicb.2022.1007831&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B14">
<span class="label">14.</span><cite>Ji L, Zhang H, Tian G, Xi S, Chu Y, Zhang Y, Liu J, Ning K, Yang J. Tumor microenvironment interplay amid microbial community, host gene expression and pathological features elucidates cancer heterogeneity and prognosis risk. The Innovation Life. 2023;1(2):100028. doi: 10.59717/j.xinn-life.2023.100028.</cite> [<a href="https://doi.org/10.59717/j.xinn-life.2023.100028" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=The%20Innovation%20Life&amp;title=Tumor%20microenvironment%20interplay%20amid%20microbial%20community,%20host%20gene%20expression%20and%20pathological%20features%20elucidates%20cancer%20heterogeneity%20and%20prognosis%20risk.&amp;author=L%20Ji&amp;author=H%20Zhang&amp;author=G%20Tian&amp;author=S%20Xi&amp;author=Y%20Chu&amp;volume=1&amp;issue=2&amp;publication_year=2023&amp;pages=100028&amp;doi=10.59717/j.xinn-life.2023.100028&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B15">
<span class="label">15.</span><cite>Nejman D, Livyatan I, Fuks G, Gavert N, Zwang Y, Geller LT, Rotter-Maskowitz A, Weiser R, Mallel G, Gigi E, Meltser A, Douglas GM, Kamer I, Gopalakrishnan V, Dadosh T, Levin-Zaidman S, Avnet S, Atlan T, Cooper ZA, Arora R, Cogdill AP, Khan MAW, Ologun G, Bussi Y, Weinberger A, Lotan-Pompan M, Golani O, Perry G, Rokah M, Bahar-Shany K. The human tumor microbiome is composed of tumor type-specific intracellular bacteria. Science. 2020;368(6494):973–980. doi: 10.1126/science.aay9189.</cite> [<a href="https://doi.org/10.1126/science.aay9189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7757858/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32467386/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20human%20tumor%20microbiome%20is%20composed%20of%20tumor%20type-specific%20intracellular%20bacteria.&amp;author=D%20Nejman&amp;author=I%20Livyatan&amp;author=G%20Fuks&amp;author=N%20Gavert&amp;author=Y%20Zwang&amp;volume=368&amp;issue=6494&amp;publication_year=2020&amp;pages=973-980&amp;pmid=32467386&amp;doi=10.1126/science.aay9189&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B16">
<span class="label">16.</span><cite>Elinav E, Garrett WS, Trinchieri G, Wargo J. The cancer microbiome. Nature Rev Cancer. 2019;19(7):371–376. doi: 10.1038/s41568-019-0155-3.</cite> [<a href="https://doi.org/10.1038/s41568-019-0155-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6700740/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31186547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Rev%20Cancer&amp;title=The%20cancer%20microbiome.&amp;author=E%20Elinav&amp;author=WS%20Garrett&amp;author=G%20Trinchieri&amp;author=J%20Wargo&amp;volume=19&amp;issue=7&amp;publication_year=2019&amp;pages=371-376&amp;pmid=31186547&amp;doi=10.1038/s41568-019-0155-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B17">
<span class="label">17.</span><cite>Zhang Y, Ji L, Wen H, Chu Y, Xing W, Tian G, Yao Y, Yang J. Pan-cancer analyses reveal the stratification of patient prognosis by viral composition in tumor tissues. Comput Biol Med. 2023;167:107586. doi: 10.1016/j.compbiomed.2023.107586.</cite> [<a href="https://doi.org/10.1016/j.compbiomed.2023.107586" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37907029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput%20Biol%20Med&amp;title=Pan-cancer%20analyses%20reveal%20the%20stratification%20of%20patient%20prognosis%20by%20viral%20composition%20in%20tumor%20tissues.&amp;author=Y%20Zhang&amp;author=L%20Ji&amp;author=H%20Wen&amp;author=Y%20Chu&amp;author=W%20Xing&amp;volume=167&amp;publication_year=2023&amp;pages=107586&amp;pmid=37907029&amp;doi=10.1016/j.compbiomed.2023.107586&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B18">
<span class="label">18.</span><cite>Li X, Yuan X, Zhu X, Li C, Ji L, Lv K, Tian G, Ning K, Yang J. A meta-analysis of tissue microbial biomarkers for recurrence and metastasis in multiple cancer types. J Med Microbiol. 2023;72(8) doi: 10.1099/jmm.0.001744.</cite> [<a href="https://doi.org/10.1099/jmm.0.001744" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37624368/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Med%20Microbiol&amp;title=A%20meta-analysis%20of%20tissue%20microbial%20biomarkers%20for%20recurrence%20and%20metastasis%20in%20multiple%20cancer%20types.&amp;author=X%20Li&amp;author=X%20Yuan&amp;author=X%20Zhu&amp;author=C%20Li&amp;author=L%20Ji&amp;volume=72&amp;issue=8&amp;publication_year=2023&amp;pmid=37624368&amp;doi=10.1099/jmm.0.001744&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B19">
<span class="label">19.</span><cite>Yang X, Guo Y, Chen C, Shao B, Zhao L, Zhou Q, Liu J, Wang G, Yuan W, Sun Z. Interaction between intestinal microbiota and tumour immunity in the tumour microenvironment. Immunology. 2021;164(3):476–493. doi: 10.1111/imm.13397.</cite> [<a href="https://doi.org/10.1111/imm.13397" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8517597/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34322877/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunology&amp;title=Interaction%20between%20intestinal%20microbiota%20and%20tumour%20immunity%20in%20the%20tumour%20microenvironment.&amp;author=X%20Yang&amp;author=Y%20Guo&amp;author=C%20Chen&amp;author=B%20Shao&amp;author=L%20Zhao&amp;volume=164&amp;issue=3&amp;publication_year=2021&amp;pages=476-493&amp;pmid=34322877&amp;doi=10.1111/imm.13397&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B20">
<span class="label">20.</span><cite>Bagheri Z, Moeinzadeh L, Razmkhah M. Roles of Microbiota in Cancer: From Tumor Development to Treatment. J Oncol. 2022;2022:3845104. doi: 10.1155/2022/3845104.</cite> [<a href="https://doi.org/10.1155/2022/3845104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8941494/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35342407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Oncol&amp;title=Roles%20of%20Microbiota%20in%20Cancer:%20From%20Tumor%20Development%20to%20Treatment.&amp;author=Z%20Bagheri&amp;author=L%20Moeinzadeh&amp;author=M%20Razmkhah&amp;volume=2022&amp;publication_year=2022&amp;pages=3845104&amp;pmid=35342407&amp;doi=10.1155/2022/3845104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B21">
<span class="label">21.</span><cite>Lee SH, Sung JY, Yong D, Chun J, Kim SY, Song JH, Chung KS, Kim EY, Jung JY, Kang YA, Kim YS, Kim SK, Chang J, Park MS. Characterization of microbiome in bronchoalveolar lavage fluid of patients with lung cancer comparing with benign mass like lesions. Lung Cancer. 2016;102:89–95. doi: 10.1016/j.lungcan.2016.10.016.</cite> [<a href="https://doi.org/10.1016/j.lungcan.2016.10.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27987594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lung%20Cancer&amp;title=Characterization%20of%20microbiome%20in%20bronchoalveolar%20lavage%20fluid%20of%20patients%20with%20lung%20cancer%20comparing%20with%20benign%20mass%20like%20lesions.&amp;author=SH%20Lee&amp;author=JY%20Sung&amp;author=D%20Yong&amp;author=J%20Chun&amp;author=SY%20Kim&amp;volume=102&amp;publication_year=2016&amp;pages=89-95&amp;pmid=27987594&amp;doi=10.1016/j.lungcan.2016.10.016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B22">
<span class="label">22.</span><cite>Yu G, Gail MH, Consonni D, Carugno M, Humphrys M, Pesatori AC, Caporaso NE, Goedert JJ, Ravel J, Landi MT. Characterizing human lung tissue microbiota and its relationship to epidemiological and clinical features. Genome Biol. 2016;17(1):163. doi: 10.1186/s13059-016-1021-1.</cite> [<a href="https://doi.org/10.1186/s13059-016-1021-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4964003/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27468850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Characterizing%20human%20lung%20tissue%20microbiota%20and%20its%20relationship%20to%20epidemiological%20and%20clinical%20features.&amp;author=G%20Yu&amp;author=MH%20Gail&amp;author=D%20Consonni&amp;author=M%20Carugno&amp;author=M%20Humphrys&amp;volume=17&amp;issue=1&amp;publication_year=2016&amp;pages=163&amp;pmid=27468850&amp;doi=10.1186/s13059-016-1021-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B23">
<span class="label">23.</span><cite>Jain T, Sharma P, Are AC, Vickers SM, Dudeja V. New Insights Into the Cancer-Microbiome-Immune Axis: Decrypting a Decade of Discoveries. Front Immunol. 2021;12:622064. doi: 10.3389/fimmu.2021.622064.</cite> [<a href="https://doi.org/10.3389/fimmu.2021.622064" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7940198/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33708214/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&amp;title=New%20Insights%20Into%20the%20Cancer-Microbiome-Immune%20Axis:%20Decrypting%20a%20Decade%20of%20Discoveries.&amp;author=T%20Jain&amp;author=P%20Sharma&amp;author=AC%20Are&amp;author=SM%20Vickers&amp;author=V%20Dudeja&amp;volume=12&amp;publication_year=2021&amp;pages=622064&amp;pmid=33708214&amp;doi=10.3389/fimmu.2021.622064&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B24">
<span class="label">24.</span><cite>Zhou X, You L, Xin Z, Su H, Zhou J, Ma Y. Leveraging circulating microbiome signatures to predict tumor immune microenvironment and prognosis of patients with non-small cell lung cancer. J Translat Med. 2023;21(1):800. doi: 10.1186/s12967-023-04582-w.</cite> [<a href="https://doi.org/10.1186/s12967-023-04582-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10636862/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37950236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Translat%20Med&amp;title=Leveraging%20circulating%20microbiome%20signatures%20to%20predict%20tumor%20immune%20microenvironment%20and%20prognosis%20of%20patients%20with%20non-small%20cell%20lung%20cancer.&amp;author=X%20Zhou&amp;author=L%20You&amp;author=Z%20Xin&amp;author=H%20Su&amp;author=J%20Zhou&amp;volume=21&amp;issue=1&amp;publication_year=2023&amp;pages=800&amp;pmid=37950236&amp;doi=10.1186/s12967-023-04582-w&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B25">
<span class="label">25.</span><cite>Khan S, Zakariah M, Rolfo C, Robrecht L, Palaniappan S. Prediction of mycoplasma hominis proteins targeting in mitochondria and cytoplasm of host cells and their implication in prostate cancer etiology. Oncotarget. 2017;8(19):30830–30843. doi: 10.18632/oncotarget.8306.</cite> [<a href="https://doi.org/10.18632/oncotarget.8306" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5458171/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27027344/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=Prediction%20of%20mycoplasma%20hominis%20proteins%20targeting%20in%20mitochondria%20and%20cytoplasm%20of%20host%20cells%20and%20their%20implication%20in%20prostate%20cancer%20etiology.&amp;author=S%20Khan&amp;author=M%20Zakariah&amp;author=C%20Rolfo&amp;author=L%20Robrecht&amp;author=S%20Palaniappan&amp;volume=8&amp;issue=19&amp;publication_year=2017&amp;pages=30830-30843&amp;pmid=27027344&amp;doi=10.18632/oncotarget.8306&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B26">
<span class="label">26.</span><cite>Khan S, Zaidi S, Alouffi AS, Hassan I, Imran A, Khan RA. Computational Proteome-Wide Study for the Prediction of Escherichia coli Protein Targeting in Host Cell Organelles and Their Implication in Development of Colon Cancer. ACS Omega. 2020;5(13):7254–7261. doi: 10.1021/acsomega.9b04042.</cite> [<a href="https://doi.org/10.1021/acsomega.9b04042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7143404/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32280866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Omega&amp;title=Computational%20Proteome-Wide%20Study%20for%20the%20Prediction%20of%20Escherichia%20coli%20Protein%20Targeting%20in%20Host%20Cell%20Organelles%20and%20Their%20Implication%20in%20Development%20of%20Colon%20Cancer.&amp;author=S%20Khan&amp;author=S%20Zaidi&amp;author=AS%20Alouffi&amp;author=I%20Hassan&amp;author=A%20Imran&amp;volume=5&amp;issue=13&amp;publication_year=2020&amp;pages=7254-7261&amp;pmid=32280866&amp;doi=10.1021/acsomega.9b04042&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B27">
<span class="label">27.</span><cite>Wang Y, Imran A, Shami A, Chaudhary AA, Khan S. Decipher the Helicobacter pylori Protein Targeting in the Nucleus of Host Cell and their Implications in Gallbladder Cancer: An insilico approach. J Cancer. 2021;12(23):7214–7222. doi: 10.7150/jca.63517.</cite> [<a href="https://doi.org/10.7150/jca.63517" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8558644/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34729122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cancer&amp;title=Decipher%20the%20Helicobacter%20pylori%20Protein%20Targeting%20in%20the%20Nucleus%20of%20Host%20Cell%20and%20their%20Implications%20in%20Gallbladder%20Cancer:%20An%20insilico%20approach.&amp;author=Y%20Wang&amp;author=A%20Imran&amp;author=A%20Shami&amp;author=AA%20Chaudhary&amp;author=S%20Khan&amp;volume=12&amp;issue=23&amp;publication_year=2021&amp;pages=7214-7222&amp;pmid=34729122&amp;doi=10.7150/jca.63517&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B28">
<span class="label">28.</span><cite>Zhang H, García Rodríguez LA, Hernández-Díaz S. Antibiotic use and the risk of lung cancer. Cancer Epidemiol Biomarkers Prev. 2008;17(6):1308–1315. doi: 10.1158/1055-9965.Epi-07-2817.</cite> [<a href="https://doi.org/10.1158/1055-9965.Epi-07-2817" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18544646/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Epidemiol%20Biomarkers%20Prev&amp;title=Antibiotic%20use%20and%20the%20risk%20of%20lung%20cancer.&amp;author=H%20Zhang&amp;author=LA%20Garc%C3%ADa%20Rodr%C3%ADguez&amp;author=S%20Hern%C3%A1ndez-D%C3%ADaz&amp;volume=17&amp;issue=6&amp;publication_year=2008&amp;pages=1308-1315&amp;pmid=18544646&amp;doi=10.1158/1055-9965.Epi-07-2817&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B29">
<span class="label">29.</span><cite>Le Noci V, Guglielmetti S, Arioli S, Camisaschi C, Bianchi F, Sommariva M, Storti C, Triulzi T, Castelli C, Balsari A, Tagliabue E, Sfondrini L. Modulation of Pulmonary Microbiota by Antibiotic or Probiotic Aerosol Therapy: A Strategy to Promote Immunosurveillance against Lung Metastases. Cell Rep. 2018;24(13):3528–3538. doi: 10.1016/j.celrep.2018.08.090.</cite> [<a href="https://doi.org/10.1016/j.celrep.2018.08.090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30257213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Rep&amp;title=Modulation%20of%20Pulmonary%20Microbiota%20by%20Antibiotic%20or%20Probiotic%20Aerosol%20Therapy:%20A%20Strategy%20to%20Promote%20Immunosurveillance%20against%20Lung%20Metastases.&amp;author=V%20Le%20Noci&amp;author=S%20Guglielmetti&amp;author=S%20Arioli&amp;author=C%20Camisaschi&amp;author=F%20Bianchi&amp;volume=24&amp;issue=13&amp;publication_year=2018&amp;pages=3528-3538&amp;pmid=30257213&amp;doi=10.1016/j.celrep.2018.08.090&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B30">
<span class="label">30.</span><cite>Jin C, Lagoudas GK, Zhao C, Bullman S, Bhutkar A, Hu B, Ameh S, Sandel D, Liang XS, Mazzilli S, Whary MT, Meyerson M, Germain R, Blainey PC, Fox JG, Jacks T. Commensal Microbiota Promote Lung Cancer Development via γδ T Cells. Cell. 2019;176(5):998–1013.e1016. doi: 10.1016/j.cell.2018.12.040.</cite> [<a href="https://doi.org/10.1016/j.cell.2018.12.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6691977/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30712876/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Commensal%20Microbiota%20Promote%20Lung%20Cancer%20Development%20via%20%CE%B3%CE%B4%20T%20Cells.&amp;author=C%20Jin&amp;author=GK%20Lagoudas&amp;author=C%20Zhao&amp;author=S%20Bullman&amp;author=A%20Bhutkar&amp;volume=176&amp;issue=5&amp;publication_year=2019&amp;pages=998-1013.e1016&amp;pmid=30712876&amp;doi=10.1016/j.cell.2018.12.040&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B31">
<span class="label">31.</span><cite>Mao X, Xu J, Wang W, Liang C, Hua J, Liu J, Zhang B, Meng Q, Yu X, Shi S. Crosstalk between cancer-associated fibroblasts and immune cells in the tumor microenvironment: new findings and future perspectives. Mol Cancer. 2021;20(1):131. doi: 10.1186/s12943-021-01428-1.</cite> [<a href="https://doi.org/10.1186/s12943-021-01428-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8504100/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34635121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Cancer&amp;title=Crosstalk%20between%20cancer-associated%20fibroblasts%20and%20immune%20cells%20in%20the%20tumor%20microenvironment:%20new%20findings%20and%20future%20perspectives.&amp;author=X%20Mao&amp;author=J%20Xu&amp;author=W%20Wang&amp;author=C%20Liang&amp;author=J%20Hua&amp;volume=20&amp;issue=1&amp;publication_year=2021&amp;pages=131&amp;pmid=34635121&amp;doi=10.1186/s12943-021-01428-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B32">
<span class="label">32.</span><cite>Ozga AJ, Chow MT, Luster AD. Chemokines and the immune response to cancer. Immunity. 2021;54(5):859–874. doi: 10.1016/j.immuni.2021.01.012.</cite> [<a href="https://doi.org/10.1016/j.immuni.2021.01.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8434759/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33838745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Immunity&amp;title=Chemokines%20and%20the%20immune%20response%20to%20cancer.&amp;author=AJ%20Ozga&amp;author=MT%20Chow&amp;author=AD%20Luster&amp;volume=54&amp;issue=5&amp;publication_year=2021&amp;pages=859-874&amp;pmid=33838745&amp;doi=10.1016/j.immuni.2021.01.012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B33">
<span class="label">33.</span><cite>Sepich-Poore GD, Zitvogel L, Straussman R, Hasty J, Wargo JA, Knight R. The microbiome and human cancer. Science. 2021;371(6536) doi: 10.1126/science.abc4552.</cite> [<a href="https://doi.org/10.1126/science.abc4552" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8767999/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33766858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=The%20microbiome%20and%20human%20cancer.&amp;author=GD%20Sepich-Poore&amp;author=L%20Zitvogel&amp;author=R%20Straussman&amp;author=J%20Hasty&amp;author=JA%20Wargo&amp;volume=371&amp;issue=6536&amp;publication_year=2021&amp;pmid=33766858&amp;doi=10.1126/science.abc4552&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B34">
<span class="label">34.</span><cite>Murciano-Goroff YR, Warner AB, Wolchok JD. The future of cancer immunotherapy: microenvironment-targeting combinations. Cell Res. 2020;30(6):507–519. doi: 10.1038/s41422-020-0337-2.</cite> [<a href="https://doi.org/10.1038/s41422-020-0337-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7264181/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32467593/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell%20Res&amp;title=The%20future%20of%20cancer%20immunotherapy:%20microenvironment-targeting%20combinations.&amp;author=YR%20Murciano-Goroff&amp;author=AB%20Warner&amp;author=JD%20Wolchok&amp;volume=30&amp;issue=6&amp;publication_year=2020&amp;pages=507-519&amp;pmid=32467593&amp;doi=10.1038/s41422-020-0337-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B35">
<span class="label">35.</span><cite>Petitprez F, Meylan M, de Reyniès A, Sautès-Fridman C, Fridman WH. The Tumor Microenvironment in the Response to Immune Checkpoint Blockade Therapies. Front Immunol. 2020;11:784. doi: 10.3389/fimmu.2020.00784.</cite> [<a href="https://doi.org/10.3389/fimmu.2020.00784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7221158/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32457745/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&amp;title=The%20Tumor%20Microenvironment%20in%20the%20Response%20to%20Immune%20Checkpoint%20Blockade%20Therapies.&amp;author=F%20Petitprez&amp;author=M%20Meylan&amp;author=A%20de%20Reyni%C3%A8s&amp;author=C%20Saut%C3%A8s-Fridman&amp;author=WH%20Fridman&amp;volume=11&amp;publication_year=2020&amp;pages=784&amp;pmid=32457745&amp;doi=10.3389/fimmu.2020.00784&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B36">
<span class="label">36.</span><cite>Khan S, Imran A, Khan AA, Abul Kalam M, Alshamsan A. Systems Biology Approaches for the Prediction of Possible Role of Chlamydia pneumoniae Proteins in the Etiology of Lung Cancer. PLoS One. 2016;11(2):e0148530. doi: 10.1371/journal.pone.0148530.</cite> [<a href="https://doi.org/10.1371/journal.pone.0148530" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4752481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26871581/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Systems%20Biology%20Approaches%20for%20the%20Prediction%20of%20Possible%20Role%20of%20Chlamydia%20pneumoniae%20Proteins%20in%20the%20Etiology%20of%20Lung%20Cancer.&amp;author=S%20Khan&amp;author=A%20Imran&amp;author=AA%20Khan&amp;author=M%20Abul%20Kalam&amp;author=A%20Alshamsan&amp;volume=11&amp;issue=2&amp;publication_year=2016&amp;pages=e0148530&amp;pmid=26871581&amp;doi=10.1371/journal.pone.0148530&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B37">
<span class="label">37.</span><cite>Deng B, Chen X, Xu L, Zheng L, Zhu X, Shi J, Yang L, Wang D, Jiang D. Chordin-like 1 is a novel prognostic biomarker and correlative with immune cell infiltration in lung adenocarcinoma. Aging. 2022;14(1):389–409. doi: 10.18632/aging.203814.</cite> [<a href="https://doi.org/10.18632/aging.203814" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8791215/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35021154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Aging&amp;title=Chordin-like%201%20is%20a%20novel%20prognostic%20biomarker%20and%20correlative%20with%20immune%20cell%20infiltration%20in%20lung%20adenocarcinoma.&amp;author=B%20Deng&amp;author=X%20Chen&amp;author=L%20Xu&amp;author=L%20Zheng&amp;author=X%20Zhu&amp;volume=14&amp;issue=1&amp;publication_year=2022&amp;pages=389-409&amp;pmid=35021154&amp;doi=10.18632/aging.203814&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B38">
<span class="label">38.</span><cite>Yang S, Chen S, Zhao Y, Wu T, Wang Y, Li T, Fu L, Ye T, Hu YQ, Chen H. Identification of a coagulation-related signature correlated with immune infiltration and their prognostic implications in lung adenocarcinoma. Thorac cancer. 2023;14(33):3295–3308. doi: 10.1111/1759-7714.15121.</cite> [<a href="https://doi.org/10.1111/1759-7714.15121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10665780/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37795779/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Thorac%20cancer&amp;title=Identification%20of%20a%20coagulation-related%20signature%20correlated%20with%20immune%20infiltration%20and%20their%20prognostic%20implications%20in%20lung%20adenocarcinoma.&amp;author=S%20Yang&amp;author=S%20Chen&amp;author=Y%20Zhao&amp;author=T%20Wu&amp;author=Y%20Wang&amp;volume=14&amp;issue=33&amp;publication_year=2023&amp;pages=3295-3308&amp;pmid=37795779&amp;doi=10.1111/1759-7714.15121&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B39">
<span class="label">39.</span><cite>Wen Z, Pei B, Dai L, Lu P, Li X, Zhang C, Ge S. Risk factors analysis and survival prediction model establishment of patients with lung adenocarcinoma based on different pyroptosis-related gene subtypes. Eur J Med Res. 2023;28(1):601. doi: 10.1186/s40001-023-01581-x.</cite> [<a href="https://doi.org/10.1186/s40001-023-01581-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10726488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38111060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Med%20Res&amp;title=Risk%20factors%20analysis%20and%20survival%20prediction%20model%20establishment%20of%20patients%20with%20lung%20adenocarcinoma%20based%20on%20different%20pyroptosis-related%20gene%20subtypes.&amp;author=Z%20Wen&amp;author=B%20Pei&amp;author=L%20Dai&amp;author=P%20Lu&amp;author=X%20Li&amp;volume=28&amp;issue=1&amp;publication_year=2023&amp;pages=601&amp;pmid=38111060&amp;doi=10.1186/s40001-023-01581-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B40">
<span class="label">40.</span><cite>Borst J, Ahrends T, Bąbała N, Melief CJM, Kastenmüller W. CD4(+) T cell help in cancer immunology and immunotherapy. Nat Rev Immunol. 2018;18(10):635–647. doi: 10.1038/s41577-018-0044-0.</cite> [<a href="https://doi.org/10.1038/s41577-018-0044-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30057419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Immunol&amp;title=CD4(+)%20T%20cell%20help%20in%20cancer%20immunology%20and%20immunotherapy.&amp;author=J%20Borst&amp;author=T%20Ahrends&amp;author=N%20B%C4%85ba%C5%82a&amp;author=CJM%20Melief&amp;author=W%20Kastenm%C3%BCller&amp;volume=18&amp;issue=10&amp;publication_year=2018&amp;pages=635-647&amp;pmid=30057419&amp;doi=10.1038/s41577-018-0044-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B41">
<span class="label">41.</span><cite>Farhood B, Najafi M, Mortezaee K. CD8(+) cytotoxic T lymphocytes in cancer immunotherapy: A review. J Cell Physiol. 2019;234(6):8509–8521. doi: 10.1002/jcp.27782.</cite> [<a href="https://doi.org/10.1002/jcp.27782" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30520029/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Physiol&amp;title=CD8(+)%20cytotoxic%20T%20lymphocytes%20in%20cancer%20immunotherapy:%20A%20review.&amp;author=B%20Farhood&amp;author=M%20Najafi&amp;author=K%20Mortezaee&amp;volume=234&amp;issue=6&amp;publication_year=2019&amp;pages=8509-8521&amp;pmid=30520029&amp;doi=10.1002/jcp.27782&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B42">
<span class="label">42.</span><cite>Feng Z, Chen Y, Cai C, Tan J, Liu P, Chen Y, Shen H, Zeng S, Han Y. Pan-Cancer and Single-Cell Analysis Reveals CENPL as a Cancer Prognosis and Immune Infiltration-Related Biomarker. Front Immunol. 2022;13:916594. doi: 10.3389/fimmu.2022.916594.</cite> [<a href="https://doi.org/10.3389/fimmu.2022.916594" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9279617/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35844598/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Immunol&amp;title=Pan-Cancer%20and%20Single-Cell%20Analysis%20Reveals%20CENPL%20as%20a%20Cancer%20Prognosis%20and%20Immune%20Infiltration-Related%20Biomarker.&amp;author=Z%20Feng&amp;author=Y%20Chen&amp;author=C%20Cai&amp;author=J%20Tan&amp;author=P%20Liu&amp;volume=13&amp;publication_year=2022&amp;pages=916594&amp;pmid=35844598&amp;doi=10.3389/fimmu.2022.916594&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B43">
<span class="label">43.</span><cite>Raskov H, Orhan A, Christensen JP, Gögenur I. Cytotoxic CD8(+) T cells in cancer and cancer immunotherapy. Br J Cancer. 2021;124(2):359–367. doi: 10.1038/s41416-020-01048-4.</cite> [<a href="https://doi.org/10.1038/s41416-020-01048-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7853123/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32929195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&amp;title=Cytotoxic%20CD8(+)%20T%20cells%20in%20cancer%20and%20cancer%20immunotherapy.&amp;author=H%20Raskov&amp;author=A%20Orhan&amp;author=JP%20Christensen&amp;author=I%20G%C3%B6genur&amp;volume=124&amp;issue=2&amp;publication_year=2021&amp;pages=359-367&amp;pmid=32929195&amp;doi=10.1038/s41416-020-01048-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B44">
<span class="label">44.</span><cite>Zhang J, Han X, Lin L, Chen J, Wang F, Ding Q, Hao L, Wang L, Wei J, Wang Y, Pan Y. Unraveling the Expression Patterns of Immune Checkpoints Identifies New Subtypes and Emerging Therapeutic Indicators in Lung Adenocarcinoma. Oxid Med Cell Longev. 2022;2022:3583985. doi: 10.1155/2022/3583985.</cite> [<a href="https://doi.org/10.1155/2022/3583985" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8843963/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35178154/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oxid%20Med%20Cell%20Longev&amp;title=Unraveling%20the%20Expression%20Patterns%20of%20Immune%20Checkpoints%20Identifies%20New%20Subtypes%20and%20Emerging%20Therapeutic%20Indicators%20in%20Lung%20Adenocarcinoma.&amp;author=J%20Zhang&amp;author=X%20Han&amp;author=L%20Lin&amp;author=J%20Chen&amp;author=F%20Wang&amp;volume=2022&amp;publication_year=2022&amp;pages=3583985&amp;pmid=35178154&amp;doi=10.1155/2022/3583985&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B45">
<span class="label">45.</span><cite>Deng X, Chen X, Luo Y, Que J, Chen L. Intratumor microbiome derived glycolysis-lactate signatures depicts immune heterogeneity in lung adenocarcinoma by integration of microbiomic, transcriptomic, proteomic and single-cell data. Front Microbiol. 2023;14:1202454. doi: 10.3389/fmicb.2023.1202454.</cite> [<a href="https://doi.org/10.3389/fmicb.2023.1202454" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10469687/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37664112/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Microbiol&amp;title=Intratumor%20microbiome%20derived%20glycolysis-lactate%20signatures%20depicts%20immune%20heterogeneity%20in%20lung%20adenocarcinoma%20by%20integration%20of%20microbiomic,%20transcriptomic,%20proteomic%20and%20single-cell%20data.&amp;author=X%20Deng&amp;author=X%20Chen&amp;author=Y%20Luo&amp;author=J%20Que&amp;author=L%20Chen&amp;volume=14&amp;publication_year=2023&amp;pages=1202454&amp;pmid=37664112&amp;doi=10.3389/fmicb.2023.1202454&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B46">
<span class="label">46.</span><cite>Zheng L, Xu J, Sai B, Zhu Y, Wang L, Yin N, Yu F, Zhou W, Wu M, Tang J, Xiang J. Microbiome Related Cytotoxically Active CD8+ TIL Are Inversely Associated With Lung Cancer Development. Front Oncol. 2020;10:531131. doi: 10.3389/fonc.2020.531131.</cite> [<a href="https://doi.org/10.3389/fonc.2020.531131" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7756114/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33363002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Oncol&amp;title=Microbiome%20Related%20Cytotoxically%20Active%20CD8+%20TIL%20Are%20Inversely%20Associated%20With%20Lung%20Cancer%20Development.&amp;author=L%20Zheng&amp;author=J%20Xu&amp;author=B%20Sai&amp;author=Y%20Zhu&amp;author=L%20Wang&amp;volume=10&amp;publication_year=2020&amp;pages=531131&amp;pmid=33363002&amp;doi=10.3389/fonc.2020.531131&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B47">
<span class="label">47.</span><cite>Ye C, Gao ZH, Chen KQ, Lu FG, Wei K. Research on Pachymaran to Ameliorate CsA-Induced Immunosuppressive Lung Injury by Regulating Microflora Metabolism. Microorganisms. 2023;11(9):2249. doi: 10.3390/microorganisms11092249.</cite> [<a href="https://doi.org/10.3390/microorganisms11092249" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10537689/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37764093/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microorganisms&amp;title=Research%20on%20Pachymaran%20to%20Ameliorate%20CsA-Induced%20Immunosuppressive%20Lung%20Injury%20by%20Regulating%20Microflora%20Metabolism.&amp;author=C%20Ye&amp;author=ZH%20Gao&amp;author=KQ%20Chen&amp;author=FG%20Lu&amp;author=K%20Wei&amp;volume=11&amp;issue=9&amp;publication_year=2023&amp;pages=2249&amp;pmid=37764093&amp;doi=10.3390/microorganisms11092249&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B48">
<span class="label">48.</span><cite>Yang Z, Zhu J, Yang T, Tang W, Zheng X, Ji S, Ren Z, Lu F. Comprehensive analysis of the lncRNAs-related immune gene signatures and their correlation with immunotherapy in lung adenocarcinoma. Br J Cancer. 2023;129(9):1397–1408. doi: 10.1038/s41416-023-02379-8.</cite> [<a href="https://doi.org/10.1038/s41416-023-02379-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10628174/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37543671/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Br%20J%20Cancer&amp;title=Comprehensive%20analysis%20of%20the%20lncRNAs-related%20immune%20gene%20signatures%20and%20their%20correlation%20with%20immunotherapy%20in%20lung%20adenocarcinoma.&amp;author=Z%20Yang&amp;author=J%20Zhu&amp;author=T%20Yang&amp;author=W%20Tang&amp;author=X%20Zheng&amp;volume=129&amp;issue=9&amp;publication_year=2023&amp;pages=1397-1408&amp;pmid=37543671&amp;doi=10.1038/s41416-023-02379-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B49">
<span class="label">49.</span><cite>Wu J, Li L, Zhang H, Zhao Y, Zhang H, Wu S, Xu B. A risk model developed based on tumor microenvironment predicts overall survival and associates with tumor immunity of patients with lung adenocarcinoma. Oncogene. 2021;40(26):4413–4424. doi: 10.1038/s41388-021-01853-y.</cite> [<a href="https://doi.org/10.1038/s41388-021-01853-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34108619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=A%20risk%20model%20developed%20based%20on%20tumor%20microenvironment%20predicts%20overall%20survival%20and%20associates%20with%20tumor%20immunity%20of%20patients%20with%20lung%20adenocarcinoma.&amp;author=J%20Wu&amp;author=L%20Li&amp;author=H%20Zhang&amp;author=Y%20Zhao&amp;author=H%20Zhang&amp;volume=40&amp;issue=26&amp;publication_year=2021&amp;pages=4413-4424&amp;pmid=34108619&amp;doi=10.1038/s41388-021-01853-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B50">
<span class="label">50.</span><cite>Narunsky-Haziza L, Sepich-Poore GD, Livyatan I, Asraf O, Martino C, Nejman D, Gavert N, Stajich JE, Amit G, González A, Wandro S, Perry G, Ariel R, Meltser A, Shaffer JP, Zhu Q, Balint-Lahat N, Barshack I, Dadiani M, Gal-Yam EN, Patel SP, Bashan A, Swafford AD, Pilpel Y, Knight R, Straussman R. Pan-cancer analyses reveal cancer-type-specific fungal ecologies and bacteriome interactions. Cell. 2022;185(20):3789–3806.e3717. doi: 10.1016/j.cell.2022.09.005.</cite> [<a href="https://doi.org/10.1016/j.cell.2022.09.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9567272/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36179670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Pan-cancer%20analyses%20reveal%20cancer-type-specific%20fungal%20ecologies%20and%20bacteriome%20interactions.&amp;author=L%20Narunsky-Haziza&amp;author=GD%20Sepich-Poore&amp;author=I%20Livyatan&amp;author=O%20Asraf&amp;author=C%20Martino&amp;volume=185&amp;issue=20&amp;publication_year=2022&amp;pages=3789-3806.e3717&amp;pmid=36179670&amp;doi=10.1016/j.cell.2022.09.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B51">
<span class="label">51.</span><cite>Liu S, Wang Z, Zhu R, Wang F, Cheng Y, Liu Y. Three Differential Expression Analysis Methods for RNA Sequencing: limma, EdgeR, DESeq2. J Vis Exp. 2021;18:175. doi: 10.3791/62528.</cite> [<a href="https://doi.org/10.3791/62528" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34605806/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Vis%20Exp&amp;title=Three%20Differential%20Expression%20Analysis%20Methods%20for%20RNA%20Sequencing:%20limma,%20EdgeR,%20DESeq2.&amp;author=S%20Liu&amp;author=Z%20Wang&amp;author=R%20Zhu&amp;author=F%20Wang&amp;author=Y%20Cheng&amp;volume=18&amp;publication_year=2021&amp;pages=175&amp;pmid=34605806&amp;doi=10.3791/62528&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B52">
<span class="label">52.</span><cite>Langfelder P, Horvath S. WGCNA: an R package for weighted correlation network analysis. BMC Bioinformatics. 2008;9:559. doi: 10.1186/1471-2105-9-559.</cite> [<a href="https://doi.org/10.1186/1471-2105-9-559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2631488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19114008/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Bioinformatics&amp;title=WGCNA:%20an%20R%20package%20for%20weighted%20correlation%20network%20analysis.&amp;author=P%20Langfelder&amp;author=S%20Horvath&amp;volume=9&amp;publication_year=2008&amp;pages=559&amp;pmid=19114008&amp;doi=10.1186/1471-2105-9-559&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B53">
<span class="label">53.</span><cite>Tibshirani R. The lasso method for variable selection in the Cox model. Stat Med. 1997;16(4):385–395. doi: 10.1002/(sici)1097-0258(19970228)16:4&amp;#x0003c;385::aid-sim380&amp;#x0003e;3.0.co;2-3.</cite> [<a href="https://doi.org/10.1002/(sici)1097-0258(19970228)16:4&amp;#x0003c;385::aid-sim380&amp;#x0003e;3.0.co;2-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9044528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Stat%20Med&amp;title=The%20lasso%20method%20for%20variable%20selection%20in%20the%20Cox%20model.&amp;author=R%20Tibshirani&amp;volume=16&amp;issue=4&amp;publication_year=1997&amp;pages=385-395&amp;pmid=9044528&amp;doi=10.1002/(sici)1097-0258(19970228)16:4&amp;#x0003c;385::aid-sim380&amp;#x0003e;3.0.co;2-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B54">
<span class="label">54.</span><cite>Jiang P, Gu S, Pan D, Fu J, Sahu A, Hu X, Li Z, Traugh N, Bu X, Li B, Liu J, Freeman GJ, Brown MA, Wucherpfennig KW, Liu XS. Signatures of T cell dysfunction and exclusion predict cancer immunotherapy response. Nat Med. 2018;24(10):1550–1558. doi: 10.1038/s41591-018-0136-1.</cite> [<a href="https://doi.org/10.1038/s41591-018-0136-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6487502/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30127393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Med&amp;title=Signatures%20of%20T%20cell%20dysfunction%20and%20exclusion%20predict%20cancer%20immunotherapy%20response.&amp;author=P%20Jiang&amp;author=S%20Gu&amp;author=D%20Pan&amp;author=J%20Fu&amp;author=A%20Sahu&amp;volume=24&amp;issue=10&amp;publication_year=2018&amp;pages=1550-1558&amp;pmid=30127393&amp;doi=10.1038/s41591-018-0136-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B55">
<span class="label">55.</span><cite>Yoshihara K, Shahmoradgoli M, Martínez E, Vegesna R, Kim H, Torres-Garcia W, Treviño V, Shen H, Laird PW, Levine DA, Carter SL, Getz G, Stemke-Hale K, Mills GB, Verhaak RG. Inferring tumour purity and stromal and immune cell admixture from expression data. Nat Commun. 2013;4:2612. doi: 10.1038/ncomms3612.</cite> [<a href="https://doi.org/10.1038/ncomms3612" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3826632/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24113773/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Inferring%20tumour%20purity%20and%20stromal%20and%20immune%20cell%20admixture%20from%20expression%20data.&amp;author=K%20Yoshihara&amp;author=M%20Shahmoradgoli&amp;author=E%20Mart%C3%ADnez&amp;author=R%20Vegesna&amp;author=H%20Kim&amp;volume=4&amp;publication_year=2013&amp;pages=2612&amp;pmid=24113773&amp;doi=10.1038/ncomms3612&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B56">
<span class="label">56.</span><cite>Newman AM, Liu CL, Green MR, Gentles AJ, Feng W, Xu Y, Hoang CD, Diehn M, Alizadeh AA. Robust enumeration of cell subsets from tissue expression profiles. Nat Methods. 2015;12(5):453–457. doi: 10.1038/nmeth.3337.</cite> [<a href="https://doi.org/10.1038/nmeth.3337" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4739640/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25822800/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Robust%20enumeration%20of%20cell%20subsets%20from%20tissue%20expression%20profiles.&amp;author=AM%20Newman&amp;author=CL%20Liu&amp;author=MR%20Green&amp;author=AJ%20Gentles&amp;author=W%20Feng&amp;volume=12&amp;issue=5&amp;publication_year=2015&amp;pages=453-457&amp;pmid=25822800&amp;doi=10.1038/nmeth.3337&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B57">
<span class="label">57.</span><cite>Li B, Severson E, Pignon JC, Zhao H, Li T, Novak J, Jiang P, Shen H, Aster JC, Rodig S, Signoretti S, Liu JS, Liu XS. Comprehensive analyses of tumor immunity: implications for cancer immunotherapy. Genome Biol. 2016;17(1):174. doi: 10.1186/s13059-016-1028-7.</cite> [<a href="https://doi.org/10.1186/s13059-016-1028-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4993001/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27549193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Comprehensive%20analyses%20of%20tumor%20immunity:%20implications%20for%20cancer%20immunotherapy.&amp;author=B%20Li&amp;author=E%20Severson&amp;author=JC%20Pignon&amp;author=H%20Zhao&amp;author=T%20Li&amp;volume=17&amp;issue=1&amp;publication_year=2016&amp;pages=174&amp;pmid=27549193&amp;doi=10.1186/s13059-016-1028-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B58">
<span class="label">58.</span><cite>Aran D, Hu Z, Butte AJ. xCell: digitally portraying the tissue cellular heterogeneity landscape. Genome Biol. 2017;18(1):220. doi: 10.1186/s13059-017-1349-1.</cite> [<a href="https://doi.org/10.1186/s13059-017-1349-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5688663/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29141660/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=xCell:%20digitally%20portraying%20the%20tissue%20cellular%20heterogeneity%20landscape.&amp;author=D%20Aran&amp;author=Z%20Hu&amp;author=AJ%20Butte&amp;volume=18&amp;issue=1&amp;publication_year=2017&amp;pages=220&amp;pmid=29141660&amp;doi=10.1186/s13059-017-1349-1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B59">
<span class="label">59.</span><cite>Racle J, Gfeller D. EPIC: A Tool to Estimate the Proportions of Different Cell Types from Bulk Gene Expression Data. Methods Mol Biol. 2020;2120:233–248. doi: 10.1007/978-1-0716-0327-7_17.</cite> [<a href="https://doi.org/10.1007/978-1-0716-0327-7_17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32124324/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Mol%20Biol&amp;title=EPIC:%20A%20Tool%20to%20Estimate%20the%20Proportions%20of%20Different%20Cell%20Types%20from%20Bulk%20Gene%20Expression%20Data.&amp;author=J%20Racle&amp;author=D%20Gfeller&amp;volume=2120&amp;publication_year=2020&amp;pages=233-248&amp;pmid=32124324&amp;doi=10.1007/978-1-0716-0327-7_17&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B60">
<span class="label">60.</span><cite>Plattner C, Finotello F, Rieder D. Deconvoluting tumor-infiltrating immune cells from RNA-seq data using quanTIseq. Methods Enzymol. 2020;636:261–285. doi: 10.1016/bs.mie.2019.05.056.</cite> [<a href="https://doi.org/10.1016/bs.mie.2019.05.056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32178821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Methods%20Enzymol&amp;title=Deconvoluting%20tumor-infiltrating%20immune%20cells%20from%20RNA-seq%20data%20using%20quanTIseq.&amp;author=C%20Plattner&amp;author=F%20Finotello&amp;author=D%20Rieder&amp;volume=636&amp;publication_year=2020&amp;pages=261-285&amp;pmid=32178821&amp;doi=10.1016/bs.mie.2019.05.056&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B61">
<span class="label">61.</span><cite>Becht E, Giraldo NA, Lacroix L, Buttard B, Elarouci N, Petitprez F, Selves J, Laurent-Puig P, Sautès-Fridman C, Fridman WH, de Reyniès A. Estimating the population abundance of tissue-infiltrating immune and stromal cell populations using gene expression. Genome Biol. 2016;17(1):218. doi: 10.1186/s13059-016-1070-5.</cite> [<a href="https://doi.org/10.1186/s13059-016-1070-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5073889/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27765066/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biol&amp;title=Estimating%20the%20population%20abundance%20of%20tissue-infiltrating%20immune%20and%20stromal%20cell%20populations%20using%20gene%20expression.&amp;author=E%20Becht&amp;author=NA%20Giraldo&amp;author=L%20Lacroix&amp;author=B%20Buttard&amp;author=N%20Elarouci&amp;volume=17&amp;issue=1&amp;publication_year=2016&amp;pages=218&amp;pmid=27765066&amp;doi=10.1186/s13059-016-1070-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="B62">
<span class="label">62.</span><cite>Segata N, Izard J, Waldron L, Gevers D, Miropolsky L, Garrett WS, Huttenhower C. Metagenomic biomarker discovery and explanation. Genome Biology. 2011;12(6):R60. doi: 10.1186/gb-2011-12-6-r60.</cite> [<a href="https://doi.org/10.1186/gb-2011-12-6-r60" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3218848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21702898/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Metagenomic%20biomarker%20discovery%20and%20explanation.&amp;author=N%20Segata&amp;author=J%20Izard&amp;author=L%20Waldron&amp;author=D%20Gevers&amp;author=L%20Miropolsky&amp;volume=12&amp;issue=6&amp;publication_year=2011&amp;pages=R60&amp;pmid=21702898&amp;doi=10.1186/gb-2011-12-6-r60&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373402/bin/mic-12-182-s01.pdf" data-ga-action="click_feat_suppl" class="usa-link">Click here for supplemental data file.</a><sup> (985.5KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The data supporting the findings of this study are available in Materials and methods. Further inquiries can be directed to the corresponding author.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Microbial Cell are provided here courtesy of <strong>Shared Science Publishers</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.15698/mic2025.08.855"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/mic-12-182.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.6 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373402/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373402/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373402%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373402/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373402/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373402/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373402/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373402/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373402/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="HZfKQUqRGEccah69zwn8oFmipukVOSfbfuHxpwmgC2DguU79U2Vl9iIH3XTXjX3Y">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
