
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Biochemical characterisation supports a central role for Oep80 from Arabidopsis thaliana in chloroplastic β-barrel protein assembly - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E486998AEF4D3305869900312EE4AF.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="commbiol">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373785/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Communications Biology">
<meta name="citation_title" content="Biochemical characterisation supports a central role for Oep80 from Arabidopsis thaliana in chloroplastic β-barrel protein assembly">
<meta name="citation_author" content="Rhiannon J Durant">
<meta name="citation_author_institution" content="Molecular Plant Biology, Department of Biology, University of Oxford, Oxford, UK">
<meta name="citation_author" content="R Paul Jarvis">
<meta name="citation_author_institution" content="Molecular Plant Biology, Department of Biology, University of Oxford, Oxford, UK">
<meta name="citation_author" content="Jani R Bolla">
<meta name="citation_author_institution" content="Molecular Plant Biology, Department of Biology, University of Oxford, Oxford, UK">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="8">
<meta name="citation_firstpage" content="1265">
<meta name="citation_doi" content="10.1038/s42003-025-08689-2">
<meta name="citation_pmid" content="40846801">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373785/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373785/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373785/pdf/42003_2025_Article_8689.pdf">
<meta name="description" content="The outer membranes of Gram-negative bacteria and endosymbiotic organelles mitochondria and chloroplasts contain β-barrel proteins essential for transporting metabolites, ions and proteins, as well as regulatory functions. While the assembly ...">
<meta name="og:title" content="Biochemical characterisation supports a central role for Oep80 from Arabidopsis thaliana in chloroplastic β-barrel protein assembly">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="The outer membranes of Gram-negative bacteria and endosymbiotic organelles mitochondria and chloroplasts contain β-barrel proteins essential for transporting metabolites, ions and proteins, as well as regulatory functions. While the assembly ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373785/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373785">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s42003-025-08689-2"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/42003_2025_Article_8689.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373785%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373785/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373785/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373785/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-commbiol.jpg" alt="Communications Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Communications Biology" title="Link to Communications Biology" shape="default" href="https://www.nature.com/commsbio/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Commun Biol</button></div>. 2025 Aug 22;8:1265. doi: <a href="https://doi.org/10.1038/s42003-025-08689-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s42003-025-08689-2</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Commun%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Commun%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Commun%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Commun%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Biochemical characterisation supports a central role for Oep80 from <em>Arabidopsis thaliana</em> in chloroplastic β-barrel protein assembly</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Durant%20RJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Rhiannon J Durant</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Rhiannon J Durant</span></h3>
<div class="p">
<sup>1</sup>Molecular Plant Biology, Department of Biology, University of Oxford, Oxford, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Durant%20RJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Rhiannon J Durant</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jarvis%20RP%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">R Paul Jarvis</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">R Paul Jarvis</span></h3>
<div class="p">
<sup>1</sup>Molecular Plant Biology, Department of Biology, University of Oxford, Oxford, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jarvis%20RP%22%5BAuthor%5D" class="usa-link"><span class="name western">R Paul Jarvis</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bolla%20JR%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Jani R Bolla</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Jani R Bolla</span></h3>
<div class="p">
<sup>1</sup>Molecular Plant Biology, Department of Biology, University of Oxford, Oxford, UK </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bolla%20JR%22%5BAuthor%5D" class="usa-link"><span class="name western">Jani R Bolla</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Molecular Plant Biology, Department of Biology, University of Oxford, Oxford, UK </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 31; Accepted 2025 Aug 7; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article's Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article's Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373785  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40846801/" class="usa-link">40846801</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">The outer membranes of Gram-negative bacteria and endosymbiotic organelles mitochondria and chloroplasts contain β-barrel proteins essential for transporting metabolites, ions and proteins, as well as regulatory functions. While the assembly mechanisms of these proteins are well-studied in bacteria and mitochondria, they remain poorly understood in chloroplasts due to challenges in producing sufficient quantities of relevant plant proteins for structural and biochemical analyses. Here, we show that Oep80, the presumed core component of the chloroplastic β-barrel assembly machinery, can be overexpressed in bacteria and refolded in large quantities. While the soluble POTRA domain caused the heterologous protein to aggregate, we found that the isolated β-barrel membrane domain (Oep80<sup>β</sup>) is very stable and refolds well. Using native mass spectrometry, we further show that purified Oep80<sup>β</sup> binds predicted β-signals found in several substrate β-barrel proteins in an anti-parallel manner. Overall, our findings support a central role for Oep80 in chloroplast β-barrel biogenesis. They provide evidence that the chloroplastic β-barrel assembly machinery uses a β-signal to recognise its substrates, suggesting mechanistic parallels with the BAM and SAM complexes of Gram-negative bacteria and mitochondria, respectively.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Membrane proteins, Membrane structure and assembly, Plant molecular biology</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">A study supports Oep80’s role in chloroplast β-barrel assembly, showing that refolded Oep80<sup>β</sup> binds to the β-signal (β<sub>-1</sub>) sequences of several substrate OEPs and that conserved C-terminal residues in these β<sub>-1</sub> sequences are important for interaction.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Mitochondria and chloroplasts both arose via endosymbiosis from Gram-negative bacteria<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Consequently, all three share common features, including the presence of an outer membrane (OM) containing β-barrel proteins. These β-barrel proteins perform essential roles such as the transport of small metabolites, ions and proteins across the membrane as well as regulatory and signalling functions<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. Mutation, misfolding or misassembly of these β-barrel proteins is often associated with cell death, disease or developmental defects<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Any attempts to harness or control the biogenesis of the β-barrel proteins will require a comprehensive understanding of the associated molecular mechanisms. In the past 10–20 years, considerable insight has been gained into the pathways of intracellular transport and insertion of β-barrel membrane proteins in Gram-negative bacteria and mitochondria. Indeed, β-barrel assembly has even been identified as a novel antibiotic target<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. However, several questions regarding the complete mechanism for folding and insertion of β-barrels into the OM still remain (Fig. <a href="#Fig1" class="usa-link">1</a>)<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Of even greater importance, the mechanism of β-barrel assembly in chloroplasts remains mostly uncharacterised.</p>
<figure class="fig xbox font-sm" id="Fig1"><h3 class="obj_head">Fig. 1. The outer membranes of Gram-negative bacteria, mitochondria and chloroplasts contain β-barrel proteins, which require specialised machinery for assembly.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373785_42003_2025_8689_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1876/12373785/c922ca8ea75f/42003_2025_8689_Fig1_HTML.jpg" loading="lazy" id="d33e261" height="606" width="800" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic diagrams of the β-barrel assembly complexes in each system. Question marks indicate assumed binding partners of Oep80 that have not yet been identified. <strong>b</strong> Two prevailing models for β-barrel assembly by Omp85 family proteins. N and C indicate N and C termini, respectively. Dotted black lines represent antiparallel β-strands at the lateral gate. Green arrow represents the β-signal sequence of a substrate protein.</p></figcaption></figure><p id="Par4">Chloroplasts are responsible for the bulk of terrestrial photosynthetic primary production, and their proper function relies on importing thousands of nucleus-encoded proteins via the TOC-TIC machinery (Translocons of the Outer and Inner Chloroplast envelope membranes)<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. This machinery is further regulated by the chloroplast-associated protein degradation complex (CHLORAD)<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. The core components of TOC and CHLORAD are β-barrel proteins, namely Toc75 (which co-assembles with Toc159) and SP2 (p39), respectively<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Additionally, the chloroplast outer envelope membrane (OEM) contains several other β-barrel proteins (e.g., Oep21, Oep24, Oep37, Tgd4) required for metabolite and solute transport<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>–<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup> adding to around 16 β-barrel proteins identified in <em>A.thaliana</em>. How these β-barrel proteins reach the chloroplast and assemble in the OEM is largely unknown. Recent studies indicated that the translocation of β-barrel proteins across the OEM is TOC-dependent while membrane insertion requires the essential Omp85 family protein, Oep80 (also known as Toc75-V)<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Homozygous knockout of Oep80 is embryonic lethal, while RNAi-mediated knockdown reduces levels of β-barrel outer envelope proteins (OEPs)<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>,<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>. Although Oep80 appears to be a central component of the chloroplast β-barrel protein assembly machinery, other factors that assist in this β-barrel biogenesis process are assumed to exist and remain to be discovered. The membrane embedded protein Crumpled Leaf (CRL) has been shown to interact with Oep80, but whether it is directly involved with substrate folding is unclear<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>.</p>
<p id="Par5">In Gram-negative bacteria and mitochondria, β-barrel folding is predominantly performed by the BAM and SAM complexes, respectively (Fig. <a href="#Fig1" class="usa-link">1a</a>). Both complexes contain a core Omp85 family protein, BamA or Sam50, composed of POTRA (polypeptide transport–associated) domains followed by a membrane-embedded β-barrel<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Omp85 barrels have a dynamic lateral gate, which enables recognition and binding of β-signal sequences to their first β-strand (β<sub>1</sub>) (Fig. <a href="#Fig1" class="usa-link">1b</a>)<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a></sup>. These sequences are found in the C-terminal β-strand of substrate barrels (β<sub>-1</sub>). Sam50 recognises the consensus sequence PoXGXXφXφ, while BAM recognises XφXφXYXF (φ indicates hydrophobic, Po indicates polar)<sup><a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. A recent study further demonstrated that internal sequences, termed internal β-signals, also contribute to substrate recognition by the BAM complex via interactions with BamD, suggesting a more complex, distributed recognition mechanism than previously envisaged<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Two major models exist for the mechanism of β-barrel assembly by Omp85 family proteins. In the assisted model, the shorter β-strands around the lateral gate causes thinning and instability in the adjacent membrane, which lowers the energy barrier for spontaneous folding and insertion of substrate barrel proteins (Fig. <a href="#Fig1" class="usa-link">1b</a>)<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. On the other hand, recent structural evidence supports a templated folding model, where barrel proteins fold strand by strand onto the β<sub>1</sub> strand of Sam50/BamA<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>, forming an extended hybrid β-sheet. This overcomes the energy barrier presented by the C-terminal strand’s loss of hydrogen bonding with water. Once all strands of the new β-barrel have been incorporated, the first and last strands zipper together allowing the protein to bud off into the membrane (Fig. <a href="#Fig1" class="usa-link">1b</a>)<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a></sup>.</p>
<p id="Par6">Import assays using chloroplastic β-barrel proteins Oep37 and SP2 showed their presence within Oep80-containing complexes, suggesting that Oep80 folds them into the OEM<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Additionally, mutating the antepenultimate hydrophobic residue and penultimate acidic residue in the β<sub>-1</sub> strand of Oep37 reduced the amount of Oep37 present in these Oep80-containing complexes, indicating these residues may be important for substrate recognition by Oep80. However, such a signal in other β-barrel proteins has not been identified. Identification of such a signal will help clarify whether all the chloroplastic β-barrel proteins biogenesis is mediated by Oep80; currently, there is no known alternative β-barrel assembly machinery in plastids. Furthermore, more evidence is needed to definitively show that Oep80 recognises and binds β-signals in substrate barrel proteins, and further characterisation of the β-signal consensus sequence is also required. Ultimately, mechanistic and structural analysis is needed to demonstrate that Oep80 forms the core of the chloroplastic β-barrel assembly machinery, and to determine whether it uses similar mechanisms to BAM and SAM to fold its substrates. In vitro structural and biophysical approaches are best suited for these analyses. Therefore, Oep80 must be overexpressed and purified from a heterologous expression system. So far, few chloroplast β-barrel proteins have been purified heterologously at large scale. Most notably, the small channel protein Oep21 has been refolded from <em>E.coli</em> inclusion bodies for structural analysis by NMR<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>; and the barrel domains of Toc159, Toc75 and Toc132 have also been refolded and analysed by circular dichroism (CD)<sup><a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>.</p>
<p id="Par7">Here, we have screened various constructs and optimised the purification and refolding of milligram quantities of Oep80 from bacterial inclusion bodies. We observed that the soluble POTRA domain of Oep80 caused the full-length protein to aggregate. However, the refolded β-barrel domain (Oep80<sup>β</sup>) proved to be stable. Further analysis by native mass spectrometry (MS) and circular dichroism indicated that the protein is folded correctly, enabling us to test for β-signal interactions. The data show that the chloroplastic β-barrel assembly machinery very likely uses a β-signal sequence mechanism to recognise its substrates, revealing mechanistic similarity with its mitochondrial and bacterial counterparts, SAM and BAM.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Heterologous expression of Oep80 in <em>E. coli</em> results in inclusion bodies</h3>
<p id="Par8">To produce Oep80 for in vitro studies, we generated a range of Oep80 constructs for heterologous expression in <em>E. coli</em> (Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). These include affinity tags and fusion tags both at the N-terminus and C-terminus of the mature Oep80 (β-domain and POTRA domains) without the N-terminal transit peptide and the disordered domain (Fig. <a href="#Fig2" class="usa-link">2a</a>). Additionally, we also made a construct with the OprM signal sequence at the N-terminus to target the Oep80 to <em>E. coli</em> outer membrane<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Next, we expressed all these constructs in several different cell lines including Arctic Express<sup>TM</sup>, BL21(DE3) and BL21(DE3)PLysS. To our surprise, all these constructs led to expression in inclusion bodies only, even of the soluble POTRA domain (Supplementary Table <a href="#MOESM1" class="usa-link">1</a>). We then decided to establish a refolding process. For this, we selected the conditions and constructs that produced the greatest yield of inclusion bodies (Fig. <a href="#Fig2" class="usa-link">2b</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">1</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Screening of various constructs indicated that the POTRA domain of Oep80 causes aggregation whereas the β-barrel domain alone can produce a monodisperse SEC peak.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373785_42003_2025_8689_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1876/12373785/38db50441ea7/42003_2025_8689_Fig2_HTML.jpg" loading="lazy" id="d33e456" height="357" width="800" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> AlphaFold predicted structure of mature Oep80<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. The cleavable transit peptide has been removed. <strong>b</strong> Domains of full-length AtOep80 (1.), plus constructs of Oep80 used to screen for refolding from <em>E. coli</em> inclusion bodies: 2 = 6×His<strong>-</strong>Oep80, 3 = 6×His<strong>-</strong>Oep80<sup>POTRA</sup>, 4 = Oep80<sup>β</sup>-6×His. <strong>c</strong> Overlaid SEC traces of different Oep80 constructs. TP= chloroplast transit peptide, Dis.=disordered region, P = POTRA domain, β = β-barrel domain. N and C refer to the N- and C-termini, respectively.</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Oep80 POTRA domain is a barrier to refolding</h3>
<p id="Par9">Prior to optimising refolding screens, we first solubilised the inclusion bodies in urea for all constructs and purified via affinity chromatography in urea. We then tested various refolding processes, including drop-wise dilution, dialysis and on-column refolding with different detergents, salt concentrations, pH values, and additives including L-arginine. We noticed that drop-wise dilution worked well for most cases. Most of these tests resulted in no clear precipitation, indicating that milligram quantities of proteins can be refolded. However, when passed through size exclusion chromatography (SEC) the constructs including the β-domain and POTRA domains eluted at the void volume, suggesting misfolding and aggregation (Fig. <a href="#Fig2" class="usa-link">2c</a>). We then sought to determine whether the POTRA or β-barrel domain was causing the protein to aggregate. For this, we attempted to refold a construct comprising only the soluble POTRA domains of Oep80 (Oep80<sup>POTRA</sup>). To our surprise, this protein eluted at the void volume in SEC (Fig. <a href="#Fig2" class="usa-link">2c</a>), which contrasted with previous observations on the homologous Toc75 POTRA domains<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Varying refolding conditions and methods did not improve the resulting SEC trace for Oep80<sup>POTRA</sup>.</p>
<p id="Par10">In contrast, the constructs containing the β-barrel domain only (Oep80<sup>β</sup>) refolded well and showed a monodisperse peak beyond the void volume after SEC (Fig. <a href="#Fig2" class="usa-link">2c</a>), indicating the POTRA domain was responsible for protein aggregation in full-length constructs. We next focused on further optimising the purification of Oep80<sup>β</sup> to enable in vitro biochemical characterisation.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Oep80<sup>β</sup> refolds well in LDAO detergent</h3>
<p id="Par11">In order to produce Oep80<sup>β</sup> in milligram quantities, we used our optimised drop-wise dilution method from above and performed a refolding screen of Oep80<sup>β</sup> with various detergents including Lauryldimethylamine oxide (LDAO), n-dodecyl β-D-maltoside (DDM) and n-octyl-β-D-glucoside (OG) and subjected the refolded protein to SEC (Fig. <a href="#Fig3" class="usa-link">3a</a>). The monodisperse peak is also observed in other detergents; however, the best conditions that yielded good amounts of refolded protein were 200 mM L-arginine, 300 mM NaCl and pH 8 buffer with LDAO as the optimal detergent. The presence of L-arginine considerably increased the maximum concentration of the urea-unfolded sample that could be refolded by drop-wise dilution without visible precipitation.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Refolded Oep80<sup>β</sup> behaves as a folded β-barrel protein and exists as a non-hybrid barrel dimer in solution.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373785_42003_2025_8689_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1876/12373785/4249f7996e00/42003_2025_8689_Fig3_HTML.jpg" loading="lazy" id="d33e547" height="540" width="800" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Overlaid traces from SEC performed on Oep80<sup>β</sup> refolded using three different detergents. <strong>b</strong> CD spectrum performed on Oep80<sup>β</sup> in LDAO. <strong>c</strong> Native mass spectrum of Oep80<sup>β</sup>-6×His. For predicted and observed masses, see Supplementary Table <a href="#MOESM1" class="usa-link">2</a>. <strong>d</strong> Schematic to show a cross-gate disulfide should prevent hybrid barrel formation. Red circles represent introduced cysteine residues and yellow represents disulfide crosslinking. <strong>e</strong> AlphaFold 3 model of Oep80<sup>β</sup> C587S/S408C/G727C, showing a lateral gate-spanning disulfide bond in yellow. Introduced β<sub>1</sub> and β<sub>-1</sub> cysteines coloured in red. pTM = 0.84. <strong>f</strong> SDS-PAGE on Oep80<sup>β</sup> C587S/S408C/G727C post SEC purification and after incubation at 4 °C with 0.1 mM CuSO<sub>4</sub> for 3 h. +DTT indicates the sample was boiled with 50 mM DTT in loading dye.</p></figcaption></figure><p id="Par12">We also observed that Oep80<sup>β</sup> was sensitive to concentration using centrifugal ultrafiltration concentrators. Consistent with this observation, more extensive concentration by ultrafiltration prior to SEC led to a much larger proportion of the protein eluting at the void volume (Supplementary Fig. <a href="#MOESM1" class="usa-link">2</a>). To eliminate the need for ultrafiltration, cation exchange columns were used to concentrate the protein to a smaller volume prior to SEC. We also noted that the Oep80<sup>β</sup> protein when concentrated in DDM, began to visibly come out of solution, even though DDM is a mild detergent and has been used for structural studies extensively.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Purified Oep80<sup>β</sup> shows correct folding and exists as a dimer</h3>
<p id="Par13">To confirm that the purified Oep80<sup>β</sup> protein is folded properly, we subjected the sample from the SEC peak fraction to circular dichroism (CD) spectroscopy. The spectrum showed a minimum at 218 nm, which is as expected for β-barrel proteins (Fig. <a href="#Fig3" class="usa-link">3b</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">3</a>)<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Moreover, thermal scanning CD at 218 nm showed a sharp transition, implying that the protein was initially folded, with a midpoint indicating a melting temperature of around 48 °C (Supplementary Fig. <a href="#MOESM1" class="usa-link">4</a>). We also used native MS to further confirm the extent of folding. After buffer-exchange of the sample into 200 mM ammonium acetate pH 8.0 containing 0.5% C8E4, the resultant mass spectrum showed the presence of two distinct charge state series with measured masses corresponding to monomeric and dimeric Oep80<sup>β</sup> (Fig. <a href="#Fig3" class="usa-link">3c</a>, Supplementary Table <a href="#MOESM1" class="usa-link">2</a>). The tight distributions of these two charge state series is indicative of the protein being in a properly folded state. Our data also suggest that the protein is mostly dimeric in solution with &lt;5% of monomers.</p>
<p id="Par14">To confirm that there is no influence of the 6×His tag on dimerisation, we similarly analysed an N-terminally-tagged 6×His-Oep80<sup>β</sup> construct. This protein also showed dimerisation, both before and after thrombin cleavage to remove the 6×His tag (Supplementary Figs. <a href="#MOESM1" class="usa-link">5</a> and <a href="#MOESM1" class="usa-link">6</a>). We then incubated tagged and untagged (post thrombin cleavage) Oep80<sup>β</sup> together and applied the mixture to Ni-NTA beads; after washing, the untagged protein co-eluted with the 6×His-tagged version under high imidazole but eluted in the flow-through only in a negative control experiment lacking the tagged protein (Supplementary Fig. <a href="#MOESM1" class="usa-link">7</a>). This supported the view that dimerisation is an inherent property of the Oep80 β-barrel domain.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Oep80<sup>β</sup> dimer is not a hybrid-barrel</h3>
<p id="Par15">To confirm the barrel closure and to determine whether the observed dimer forms a hybrid barrel or two independent β-barrels, we generated a mutant of Oep80<sup>β</sup> that has native cysteine C587 removed and 2 new cysteines introduced in the β<sub>1</sub> and β<sub>−1</sub> strands (C587S/S408C/G727C) (Fig. <a href="#Fig3" class="usa-link">3d</a>). Mutation of BamA residues G433 and N805 to cysteine had previously been shown to form a gate-bridging disulfide<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a></sup>. Comparison of Oep80’s AlphaFold model with BamA’s known structure suggested S408 and G727, respectively, as the corresponding residues in Oep80. Strikingly, AlphaFold 3 spontaneously predicted a gate-bridging disulfide bond for the C587S/S408C/G727C mutant (Fig. <a href="#Fig3" class="usa-link">3e</a>). After refolding and purification of this mutant, we then monitored disulfide bond formation on SDS-PAGE under reducing and non-reducing conditions<sup><a href="#CR33" class="usa-link" aria-describedby="CR33">33</a></sup>. A faster-migrating band was observed in the absence of DTT, consistent with the formation of an intramolecular disulfide bond between the added cysteines (Fig. <a href="#Fig3" class="usa-link">3f</a>). This band disappeared upon DTT treatment. This confirmed that the cysteines are in close proximity in the folded state and form intra-strand disulfide bonds, which likely span the lateral gate. We also attempted to track folding kinetics using disulfide crosslinking but found that the need for reducing conditions during refolding complicated time-resolved quantification. Nevertheless, crosslinking becomes detectable within 10 min, suggesting rapid folding similar to that observed with dilution refolding of bacterial barrel proteins<sup><a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>.</p>
<p id="Par16">In a hybrid-barrel configuration, crosslinking across the gate would block β<sub>1</sub> strand availability and thus be expected to interfere with inter-subunit strand exchange. If the observed dimer resulted from a hybrid barrel, we would expect Cu<sup>2+</sup>-mediated disulfide formation to disrupt this interaction and shift the equilibrium toward the monomeric species (Fig. <a href="#Fig3" class="usa-link">3d</a>). However, native MS analysis of Oep80<sup>β</sup> C587S/S408C/G727C both before (Supplementary Fig. <a href="#MOESM1" class="usa-link">8a</a>) and after (Supplementary Fig. <a href="#MOESM1" class="usa-link">8b</a>) Cu<sup>2+</sup> incubation showed no significant presence of a monomeric form of the protein. This suggests that the observed dimer is not a large hybrid-barrel but instead two independent barrels that are associated in solution.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Identified OEP β-signal sequences show conservation and are predicted to interact with the β<sub>1</sub> strand of Oep80</h3>
<p id="Par17">Both BamA and Sam50, the core components of the BAM and SAM complexes, respectively, are known to use β-signal sequences to recognise their substrates. We asked if Oep80 does the same in chloroplasts. For this, we turned our attention to AlphaFold 3<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a></sup>. Firstly, we performed positive control experiments using BamA and Sam50 to test if AlphaFold 3 could predict binding of known β-signal peptides at the expected location. As expected, the models showed antiparallel β-sheet formation between the β-signal peptides and the β<sub>1</sub> strand of the barrel domain in both cases (Supplementary Fig. <a href="#MOESM1" class="usa-link">9</a>), matching what has been observed by structural determination and crosslinking studies<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup>.</p>
<p id="Par18">Next, we predicted the structures of all known <em>A. thaliana</em> chloroplast outer envelope β-barrel proteins using AlphaFold 3 and split them into four groups by function (Groups 1–4). The nine residues making up the final β-strand of each barrel were then defined as the β<sub>-1</sub> strand and taken to be the presumed β-signal in each case (Supplementary Fig. <a href="#MOESM1" class="usa-link">10</a> and Supplementary Table <a href="#MOESM1" class="usa-link">3</a>). In addition, ConSurf analysis was performed on these putative signals to provide an estimation of conservation and functional importance (Fig. <a href="#Fig4" class="usa-link">4a</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">11</a>)<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Group 1 proteins are solute transporters, and previous work has shown they all share antepenultimate hydrophobic and penultimate acidic residues in the putative signal<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Group 2 proteins are lipid transporters of the LptD family, and their putative β-signals possess a similar hydrophobic/acidic motif<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>,<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Group 3 proteins are the Omp85 family homologues in Arabidopsis<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a></sup>. Finally, Group 4 proteins are the Toc159 family proteins which have N-terminal cytosolic domains and co-assemble with Toc75 within the TOC complex<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Identification and bioinformatic analysis of putative β-signals in chloroplast β-barrel proteins.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373785_42003_2025_8689_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1876/12373785/84eb63566f17/42003_2025_8689_Fig4_HTML.jpg" loading="lazy" id="d33e745" height="362" width="800" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Outer envelope β-barrel proteins were assigned to four functional groups. Conservation analysis using ConSurf was performed on the predicted β-signal sequences, and a representative example is shown for each group. <strong>b</strong> AlphaFold 3 model of Oep80 interacting with the Oep21a β<sub>-1</sub> peptide (residues 159–167). IPTM = 0.6. Conservation analysis of the β<sub>1</sub> strand of Oep80 is also shown. Black dotted lines in the inset indicate backbone N-to-O distances of 3 Å or less, i.e., likely hydrogen bonding interactions. *Although conservation score is low for this residue, all 150 sampled orthologs have an aromatic side chain at this position (either Y or F).</p></figcaption></figure><p id="Par19">Next, we modelled the β<sub>-1</sub> strand sequences of <em>At</em>Oep21a and <em>At</em>Oep37 with <em>At</em>Oep80. These were chosen as previous work has suggested their <em>P.sativum</em> homologues interact with Oep80 and their β<sub>-1</sub> sequences were almost identical in <em>P.sativum</em> and <em>A.thaliana</em><sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. In both cases, the peptide interacted via antiparallel β-sheet formation with the β<sub>1</sub> strand of the Oep80 barrel, paralleling the Sam50 and BamA control results (Fig. <a href="#Fig4" class="usa-link">4</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">12</a>). Of note, the iPTM (interface predicted template modelling) scores for Omp85-family protein interactions with cognate β-signals are relatively low and are in the order of 0.60–0.80 even for the controls.</p>
<p id="Par20">Lastly, we selected a representative sequence from each of the four groups for further in vitro analysis (Fig. <a href="#Fig4" class="usa-link">4a</a>).</p></section><section id="Sec9"><h3 class="pmc_sec_title">Purified Oep80<sup>β</sup> binds predicted β-signals of substrate OEPs</h3>
<p id="Par21">Our analysis above revealed variations in the putative β-signal sequences in different substrates. To test whether Oep80<sup>β</sup> can recognise these sequences, we used native MS. We selected representative members of each group—namely Oep21a and Oep37 for Group 1, LptD1 for Group 2, Toc75 for Group 3, and Toc159 for Group 4—and synthesised the respective β<sub>-1</sub> peptides. We combined 100 µM of each synthesised peptide with 7 µM of the purified Oep80<sup>β</sup> protein in C8E4 buffer and acquired native MS data (Fig. <a href="#Fig5" class="usa-link">5a</a>, Supplementary Fig. <a href="#MOESM1" class="usa-link">13</a>, Supplementary Table <a href="#MOESM1" class="usa-link">4</a>). In all cases, in addition to the main charge state series, we observed adduct peaks corresponding to a mass consistent with the peptide binding to Oep80<sup>β</sup>. Interestingly, the peptides of Oep37 and Oep21a showed much higher binding than those of Lptd1, Toc75-3 and Toc159 (Fig. <a href="#Fig5" class="usa-link">5a</a>). Because the peptides differ in net charge and hydrophobicity, it is likely that they dissolve and ionise differently which could affect their apparent binding affinity in native MS. However, in all cases, peaks corresponding to the β-signal peptide alone were observed in the data, indicating that the peptides were present in solution and could be ionised. Additionally, analysis of the hydrophobicity and charge of each peptide at pH 8 did not reveal any pattern correlating with the data (Supplementary Table <a href="#MOESM1" class="usa-link">5</a>); for example, the hydropathy and charge of Oep21a β<sub>-1</sub> and Lptd1 β<sub>-1</sub> are almost identical, and yet Oep21a interacts more strongly with Oep80<sup>β</sup>.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Oep80 interacts with β<sub>-1</sub> peptides of known substrate proteins with a penultimate acidic residue being important for the interaction.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373785_42003_2025_8689_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1876/12373785/ff339d950542/42003_2025_8689_Fig5_HTML.jpg" loading="lazy" id="d33e880" height="522" width="800" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Average intensity of peptide-bound Oep80<sup>β</sup> relative to unbound Oep80<sup>β</sup> species for representative β-signal peptides from all four OEP groups, as measured by native MS. <em>n</em> = 4. <strong>b</strong> Native mass spectra of Oep80<sup>β</sup> plus Oep21a β<sub>-1</sub> wild-type (w.t.) peptide (GKWNVRYDL), Oep21a β<sub>-1</sub> D-to-A peptide (GKWNVRYAL), or Oep21a β<sub>-1</sub> D-to-E peptide (GKWNVRYEL). See Supplementary Table <a href="#MOESM1" class="usa-link">4</a> for predicted and observed masses. <strong>c</strong> Intensity of peptide-bound Oep80<sup>β</sup> relative to unbound Oep80<sup>β</sup> for Oep21a β<sub>-1</sub> wild-type peptide and the two mutants. Significance was assessed using a two-tailed unpaired Student’s <em>t</em>-test; n.s. = mean differences not significant, ** = means significantly different with <em>p</em> &lt; 0.01 (<em>n</em> = 4). Error bars represent standard deviation.</p></figcaption></figure><p id="Par22">Having found that Oep21a β <sub>–1</sub> interacts strongly with Oep80<sup>β</sup>, we used this peptide to further validate our conclusion that the Oep80<sup>β</sup> dimer does not form a hybrid barrel. We generated an Oep80<sup>β</sup> mutant with the Oep21a β<sub>–1</sub> peptide sequence covalently attached via a flexible GS linker. This mutant also had the Oep80<sup>β</sup>’s one native cysteine residue mutated to serine. In this construct, the covalently attached β<sub>-1</sub> peptide is expected to form an anti-parallel β-sheet with the β<sub>1</sub> strand, mimicking the interaction observed in our peptide-binding native MS experiments. If the Oep80 dimer were a hybrid barrel formed by inter-subunit strand exchange, this steric occlusion should reduce dimerisation. Native MS revealed that this fusion protein also exists as a dimer (Supplementary Fig. <a href="#MOESM1" class="usa-link">14</a>). This reinforces the conclusion that the Oep80 dimer comprises two independently folded β-barrels that associate non-covalently, rather than a single hybrid structure formed by strand-swapping.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Mutation of the predicted β-signal peptide sequence reduces interaction with Oep80<sup>β</sup>
</h3>
<p id="Par23">Previous work has suggested that the penultimate acidic and antepenultimate hydrophobic residues of Group 1 and 2 outer envelope proteins are important for interaction with Oep80<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. In particular, mutation of these residues in Oep37 was shown to reduce incorporation of the mutant protein into the outer envelope membrane. To probe whether these differences are related to the binding of these residues to Oep80, we generated mutants of the Oep37 β-signal sequence, one with a D-to-A mutation and one with a W-to-E mutation. We mixed these two mutants with Oep80<sup>β</sup> in separate experiments and acquired native MS data. In both cases, the resultant spectra indicated reduced peptide interaction, though the reduction observed was stronger when the acidic residue was mutated (Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>).</p>
<p id="Par24">To confirm that this specificity was not unique to Oep37, we also mutated the penultimate residue of the Oep21a β<sub>-1</sub> peptide, which is also D. Subsequent native MS analysis of this D-to-A mutant revealed a significant reduction in its interaction with Oep80<sup>β</sup>. We then sought to confirm that the properties of the residue were important, rather than its exact identity, by analysing a D-to-E mutation. This mutation showed no significant difference from the wild-type sequence (Fig. <a href="#Fig5" class="usa-link">5b, c</a>), indicating that negative charge at this position is important. These results collectively confirm the importance of charged and hydrophobic residues (at −2 and −3 positions, respectively) in the β-signal sequences.</p></section><section id="Sec11"><h3 class="pmc_sec_title">Oep21a β-signal peptide appears to bind in an anti-parallel fashion to Oep80’s β<sub>1</sub> strand</h3>
<p id="Par25">To directly test the location and orientation of β-signal peptide binding to Oep80<sup>β</sup>, we generated a cysteine mutant of Oep80<sup>β</sup> (C587S/S408C), placing the cysteine at the C-terminal end of the β<sub>1</sub> strand. We then synthesised two versions of the Oep21a β<sub>-1</sub> signal peptide: one with a C-terminal cysteine (CterC; GKWNVRYLC) and one with K160 toward the N-terminus mutated to cysteine (KtoC; GCWNVRYL). If the peptide binds to the β<sub>1</sub> strand in an anti-parallel orientation, as predicted by AlphaFold 3, only the KtoC peptide should be able to form a disulfide bond with the S408C site (Fig. <a href="#Fig6" class="usa-link">6a</a>). This prediction was further supported by AlphaFold 3 structural models of the mutant Oep80<sup>β</sup> in complex with KtoC (Fig. <a href="#Fig6" class="usa-link">6b</a>), which show a plausible lateral gate interaction and a predicted disulfide bridge between the introduced cysteines.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. Oep80<sup>β</sup> C587S/S408C crosslinks efficiently with Oep21a β<sub>-1</sub> KtoC but not Oep21a β<sub>-1</sub> CterC, indicating anti-parallel binding to Oep80’s β<sub>1</sub> strand.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373785_42003_2025_8689_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/1876/12373785/7d7459126230/42003_2025_8689_Fig6_HTML.jpg" loading="lazy" id="d33e991" height="471" width="800" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong> Schematic showing the experimental strategy. Oep80<sup>β</sup> C587S/S408C combined with mutant Oep21a β<sub>-1</sub> peptides containing cysteines on either end enables differentiation between parallel and antiparallel signal peptide binding. Green arrow represents peptide; red circles represent introduced cysteine residues and the yellow lines represent disulfide bonds. <strong>b</strong> AlphaFold 3 model of Oep80<sup>β</sup> C587S/S408C plus Oep21a β<sub>-1</sub> KtoC. The inset shows the interaction interface in detail. Peptide backbone in green, Oep80 barrel backbone in cyan. Mutant residues coloured red. Dotted lines in the inset indicate likely hydrogen bonding interactions. <strong>c</strong> Oep80<sup>β</sup> S048C/C587S alone or plus either peptide was incubated at 4 °C with 0.1 mM CuSO<sub>4</sub>. Samples were taken after 3 h (3 h) or overnight (O/N). DTT (+) indicates the sample was boiled with 50 mM DTT in loading dye.</p></figcaption></figure><p id="Par26">Consistent with this model, incubation of Oep80<sup>β</sup> C587S/S408C with each peptide in the presence of CuSO<sub>4</sub> to promote oxidation led to disulfide crosslink formation with the KtoC peptide but not with CterC (Fig. <a href="#Fig6" class="usa-link">6c</a>). A higher molecular weight band was observed by SDS-PAGE only in the KtoC condition, and this band was reduced by DTT treatment, confirming disulfide bond formation.</p>
<p id="Par27">To ensure the absence of crosslinking with CterC was not due to reduced binding affinity, we performed native MS analysis using both peptides. We observed the charge state series corresponding to both KtoC and CterC binding with similar intensity, indicating comparable non-covalent binding (Supplementary Fig. <a href="#MOESM1" class="usa-link">16</a>, Supplementary Table <a href="#MOESM1" class="usa-link">6</a>). This supports the interpretation that both peptides bind to the same site, but only KtoC positions its cysteine close enough to form a disulfide with Oep80 β<sub>1</sub>-S408C.</p></section></section><section id="Sec12"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par28">In this study, we optimised the purification of full-length and various truncated forms of Oep80, a protein that has been implicated in the chloroplast β-barrel assembly machinery. Despite several attempts, we were unable to express Oep80 in its natively folded form. Previous work has shown that the OprM signal sequence should be most effective for directing barrel proteins to the bacterial outer membrane, where they may fold natively<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>, but this did not result in folded Oep80 protein. It is possible that the protein is unable to fold into the OM upon arrival due to the latter’s dissimilarity with the chloroplast OEM. The protein is also very likely to require assistance with folding, and the bacterial β-barrel assembly machinery may be unable to recognise its β-signal sequence. Therefore, expression into inclusion bodies and refolding were optimised. The POTRA domain of Oep80 appears to be prone to misfolding or aggregation, as we were unable to find refolding methods or buffer conditions that would result in non-void SEC peaks for this domain alone or the full-length protein. It is possible that the unique environment of the chloroplast inter-membrane space, or assistance from specific chaperones, is required for correct folding of this domain.</p>
<p id="Par29">In any case, it was possible to refold the isolated β-barrel domain of Oep80, Oep80<sup>β</sup>. Analysis of refolded Oep80<sup>β</sup> by native MS, SEC and CD spectroscopy indicated that it is correctly folded, and that it is most stable in LDAO. Therefore, this construct was deemed suitable for use in biochemical assays. The apparent preference of Oep80 for LDAO is interesting given that this detergent is relatively stringent. Crystal structures of BamA in LDAO showed the protein in the inwards-open conformation where the periplasmic face of the barrel is widest, and the C-terminal β-strand is tucked into the barrel. Conversely, cryo-EM structures of BamA in DDM showed the barrel in the inwards-closed conformation where the extracellular ends of the N-terminal β-strands point away from the barrel and the periplasmic face of the barrel is constricted<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>,<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. As a homologue of BamA, the conformation of Oep80 may be affected similarly by detergent identity; it could be that the inwards-closed conformation of Oep80 is less stable than the inwards-open conformation.</p>
<p id="Par30">Our native MS analysis revealed that Oep80<sup>β</sup> undergoes dimerisation. Through a series of control experiments, we demonstrated that dimer formation is an inherent property of the Oep80 barrel, as opposed to an artefact related to the purification tags used. We further used crosslinking over the lateral gate or covalent attachment of a predicted β-signal peptide to occlude the β<sub>1</sub> strand and demonstrate that the dimer is unlikely to be a hybrid barrel. We also show there is an equilibrium between dimeric and monomeric forms, as we observed that untagged barrels can replace tagged barrels in interaction assays. The biological relevance of this monomer-dimer equilibrium will need to be investigated in future studies. Nonetheless, previous studies show some other Omp85 family members dimerise. For example, Sam50 uses a β-barrel switching mechanism to fold substrates. When not engaged in assembly, Sam50 exists as a dimer. However, the β-signal of a substrate binds to the β<sub>1</sub> strand of one of the two subunits and begins to fold, and eventually this nascent barrel displaces the second Sam50<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. If Oep80 operates in a similar manner, this would explain why the β-signal peptides we identified are able to interact with the barrel in dimeric form. Similarly, the bacterial Omp85 family member TamA of the TAM complex<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a></sup> has been shown to exist in equilibrium between monomeric and dimeric states by native PAGE, and is found as a dimer of dimers when bound to its partner TamB<sup><a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Some evolutionarily more distant BamA homologues, e.g., from <em>D. radiodurans</em><sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>, have also shown dimerisation in structural studies. Additionally, native PAGE analysis of solubilised Arabidopsis OEMs using anti-Oep80 antibodies revealed that the protein is mostly present in a complex of around 200 kDa<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. This size would allow for a dimer and additional accessory proteins. Loss of Crumpled Leaf (CRL) protein reduces the size to 160 kDa but this could still contain an Oep80 dimer<sup><a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Collectively, these observations strongly support our view that the dimerisation of Oep80 observed by native MS reflects its native state.</p>
<p id="Par31">Robust interaction between Oep80<sup>β</sup> and the presumed β-signal sequences of the OEM β-barrel proteins Oep21a and Oep37 in native MS assays strongly supported the hypothesis that Oep80<sup>β</sup> has a role in β-barrel assembly. Importantly, mutation of the β-signals at their antepenultimate hydrophobic and penultimate residues acidic residues reduced the interactions. This is consistent with a previous study in which mutation of the C-terminal –2 and –3 residues of <em>Ps</em>Oep37 resulted in reduced formation of an Oep80/Oep37-containing complex in the OEM, and increased aggregation of Oep37 in the IMS<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. However, direct evidence of binding between the C-terminal β-strand of a presumed substrate and Oep80, with dependency on specific residues, was previously lacking and is provided by our data. The comparatively weaker β-barrel interactions shown by representative peptides from the other three groups (i.e., TGD4, Toc75-3 and Toc159) does not necessarily indicate that these are not substrates of Oep80. They may require a chaperone or accessory protein to be brought to Oep80, or the presence of the POTRA domains to support their recognition. Alternatively, it is possible that these proteins have a β-signal that is different from those selected in our analysis of the AlphaFold structures. Moreover, our analysis focused on canonical C-terminal β-strands, but some chloroplast β-barrel proteins may use non-canonical or internal β-signals for recognition. This is analogous to the mechanism recently described in bacteria, where internal β-strands promote binding to BAM accessory proteins such as BamD<sup><a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Similar principles may apply in chloroplasts, though the identity of any chloroplast accessory factors remains to be elucidated. Nevertheless, the fact that the predicted β-signal peptides interacted to different extents does indicate that Oep80<sup>β</sup> is able to show sequence selectivity. This indicates that there is likely a consensus β-signal sequence that Oep80 uses to recognise its substrates, in the same manner as BamA and Sam50. If the first recognition step of β-barrel assembly is the same for all three Omp85 family members, it would be reasonable to assume that the downstream steps in each pathway are similar too, although this will require further investigation.</p>
<p id="Par32">Here, we also provide experimental evidence that these β-signal peptides bind anti-parallel to the Oep80 β<sub>1</sub> strand, as is predicted by AlphaFold, using Oep21a β<sub>-1</sub> as a model. Of course, it is possible that this orientation is unique to Oep21a and that other peptides would not have shown the same crosslinking pattern. However, it seems likely the binding site and orientation is shared at least by other Group 1 and 2 OEPs, given their common consensus sequence. Additionally, this binding location would match that of β-signal binding to BamA and Sam50 in bacteria and mitochondria<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a></sup>. In these systems, cryo-EM and crystal structures provide conclusive evidence that β-signals hybridise in an anti-parallel orientation with the Omp85’s proteins β<sub>1</sub> strand<sup><a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>,<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>.</p>
<p id="Par33">In summary, this work has used in vitro, biophysical approaches to explore the role of Oep80 as the presumed core of the chloroplastic β-barrel assembly machinery. We have presented evidence that Oep80 recognises its substrates via a consensus C-terminal β-signal sequence, revealing a marked parallel with the BAM and SAM systems. In the future, a method to express the POTRA domain of Oep80 in soluble form or to refold it without aggregation from inclusion bodies, will be highly beneficial. This will enable biochemical analysis of full-length Oep80 and a more complete assessment of the protein’s function. Similarly, identification of binding partners of Oep80 will offer further insights, for example, via the reconstitution of the full complex in liposomes to probe its mechanism in detail. Previous in vivo work<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup> and our in vitro characterisation here collectively highlight the importance of conserved residues of putative β-signal sequences in Oep37, Oep21 and Oep24 in recognition by Oep80 and membrane insertion. Additionally, future work must demonstrate that Oep80 binds directly to β-signals in vivo as well. Uncovering the mechanism of chloroplastic β-barrel assembly is an important prerequisite for the genetic manipulation of chloroplast β-barrel proteins, helping to ensure that attempts to improve chloroplast function and productivity do not have unintended consequences for β-barrel assembly. It also will help generate a comprehensive picture of β-barrel assembly across all domains of life, which will have far reaching implications, including in the design of novel antibiotics and understanding mitochondrial disease.</p></section><section id="Sec13"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec14"><h3 class="pmc_sec_title">Constructs design</h3>
<p id="Par34">Codon-optimised sequences of AtOep80 corresponding to the full-length protein without its disordered N-terminus (residues 151–732), the β barrel domain (398–732), or the POTRA domain (156–395), were synthesised (IDT, Supplementary Table <a href="#MOESM1" class="usa-link">7</a>). These were inserted into NdeI and XhoI digested Pet28a to add an N-terminal 6×His or modified Pet15b digested with NheI and NdeI for C-terminal 6×His or GFP-6×His using In-Fusion (TaKaRa). Oep80<sup>β</sup> C587S/S408C/G727C and C587S/S408C were similarly synthesised and cloned into Pet28a, whereas the Oep21a β<sub>-1</sub>-fused mutant was cloned into Pet15b. For the OprM signal sequence containing constructs, the OprM, His and TEV site were also synthesised and inserted with the AtOep80 sequences into Pet28a digested with XhoI and NcoI.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Protein expression and inclusion body unfolding</h3>
<p id="Par35">Plasmids were transformed into BL21(DE3) <em>E. coli</em>. One litre LB flasks were inoculated with starter cultures, grown at 220 rpm and 37 °C until OD<sub>600</sub> = 0.6, then induced with 0.5 M IPTG for a further 3 h at 37 °C. Overnight expression at 18 °C was also tested but all protein still formed inclusion bodies. Cultures were then pelleted at 5000 <em>g</em> for 10 min, resuspended in 20 mM Tris pH 8 and 150 mM NaCl (buffer A) and frozen. Cell pellets were resuspended with C0mplete mini protease inhibitor (Merck), sonicated and then centrifuged at 25,000 g for 30 min. Inclusion bodies were washed 3 times by resuspension in buffer A plus 0.1% Triton X-100 and re-pelleting at 25,000 g for 20 min. They were unfolded in buffer A plus 8 M urea, then centrifuged at 25,000 g for 20 min and the pellet was discarded. Urea unfolded protein was batch-bound to Ni-NTA agarose beads (Qiagen) before washing on a gravity column using buffer A plus 8 M urea and 40 mM imidazole. For proteins containing 2 or more cysteine residues, concentrated urea samples were pre-incubated with 5 mM DTT and then diluted 10 times in urea prior to purification. Elution into 2 mL fractions with buffer B and 300 mM imidazole was used to concentrate the unfolded sample.</p></section><section id="Sec16"><h3 class="pmc_sec_title">Protein refolding from urea</h3>
<p id="Par36">Protein-containing 8 M urea fractions were incubated with 10 mM DTT prior to refolding to reduce non-specific disulfides. Proteins containing the β-barrel domain were refolded dropwise into vigorously stirring buffer to achieve a final 8-fold dilution. For refolding conditions screens, presence or absence of 200 mM L-arginine, 150 mM or 300 mM NaCl, 1% DDM or 0.5% LDAO were all tested. Final dilution refolding buffers for Oep80<sup>β</sup> used 0.5% LDAO, 200 mM L-Arginine, 300 mM NaCl, 20 mM TRIS pH 8 and 2 mM DTT; using this buffer, urea sample concentrations up to 9 mg/ml could be refolded. Oep80<sup>POTRA</sup> was refolded by overnight dialysis using 3.5 kD cut-off dialysis tubing against 20 mM Tris, 200 mM L-Arginine, 300 mM NaCl and 2 mM DTT. Dropwise refolding was also tested.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Concentration and purification of refolded protein</h3>
<p id="Par37">After refolding, proteins were concentrated by ultrafiltration or by ion exchange chromatography (for Oep80<sup>β</sup> constructs). For the latter, refolded protein was dialysed overnight to reduce NaCl concentration to 40 mM with 20 mM Tris pH 7. This was loaded onto a HiTrap Q FF cation exchange column pre-equilibrated in 20 mM Tris pH 7, 40 mM NaCl, 2 mM DTT and 0.05% LDAO. The protein was washed and then eluted in a sharp peak using 1 M NaCl and 20 mM Tris pH 8, 1 mM DTT and 0.05% LDAO. Aliquots of 500 µL were taken for SEC. For the POTRA domain, buffer A + 2 mM DTT was used, and for the β-barrel containing constructs 0.05% LDAO was also added unless other detergents are specified. SEC was conducted on a Superdex 200 Increase 10/300 GL column using AKTA-Pure<sup>TM</sup>. Peak fractions were analysed by SDS-PAGE and flash-frozen before storage at −80 °C.</p>
<p id="Par38">To generate tag-free Oep80<sup>β</sup>, 500 µL of 6×His-Oep80<sup>β</sup> post cation exchange was incubated overnight with thrombin protease in a 1:25 total mass ratio. Thrombin and untagged Oep80<sup>β</sup> were separated using size exclusion on a Superdex 200 Increase 10/300 GL column in buffer A + 1 mM DTT and 0.05% LDAO. Peak fractions were flash frozen before storage at −80 °C.</p></section><section id="Sec18"><h3 class="pmc_sec_title">CD spectroscopy</h3>
<p id="Par39">Purified Oep80<sup>β</sup>-6×His was buffer exchanged into 20 mM NaF plus either 0.05% LDAO or 0.5% C8E4 using a Biospin column. The sample was then diluted to 0.2 mg/ml. CD was conducted on a Jasco J-815 Spectropolarimeter, using the buffer alone as a blank. A 1 mm pathlength quartz cuvette was used with 1 s digital integration time. 10 accumulations over 250–190 nm were averaged. HT voltage signal was also collected in parallel. For thermal denaturation analysis, 200–250 nm scans were collected over 20–90 °C with 2 °C intervals.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Native mass spectrometry</h3>
<p id="Par40">Before MS analysis, the proteins were buffer exchanged into 200 mM ammonium acetate pH 8.0 and 0.5% C8E4 using a Biospin-6 (BioRad) column. For the oxidised Oep80<sup>β</sup> C587S/S408C/G727C mutant, 0.1 mM CuSO<sub>4</sub> was added to one aliquot and pre-incubated for 4 h prior to buffer exchange. All peptides used were purchased from Proteogenix at 99% purity in powdered form. They were first dissolved in DMSO to 5 mM concentration, then diluted 10-fold into 200 mM NH<sub>4</sub>CH<sub>3</sub>COO plus 0.5% C8E4.</p>
<p id="Par41">To analyse the binding of all wild type OEP β-signal peptides and the penultimate/antepenultimate residue mutants, the peptides were combined with Oep80<sup>β</sup>-6×His to a final peptide concentration of 100 µM and protein concentration of 7 µM. To test for interaction of the cysteine mutants of Oep21a β<sub>-1</sub>, 5 µM peptide was sprayed with approximately 3 µM protein.</p>
<p id="Par42">Samples were introduced directly into the mass spectrometer using gold-coated capillary needles (prepared in-house). Data were collected on a Q-Exactive UHMR mass spectrometer (Thermo Fisher Scientific). The instrument parameters were as follows: capillary voltage 1.1 kV, S-lens RF 100%, quadrupole selection from 1000 to 20,000 m/z range, collisional activation in the HCD cell 100–200 V, trapping gas pressure setting 7.5, temperature 200 °C, and resolution of the instrument 12,500. The noise level was set at 3 rather than the default value of 4.64. No in-source dissociation was applied. Data were analysed using Xcalibur 4.2 (Thermo Scientific) and UniDec software packages. All experiments were repeated with similar outcomes. Statistical analysis was performed using OriginPro. Comparisons for two groups were calculated using unpaired two-tailed Student’s <em>t</em>-tests. Exact <em>n</em> values are indicated in figure legends.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Co-affinity purification</h3>
<p id="Par43">A sample of untagged Oep80<sup>β</sup> was combined with tagged 6×His-Oep80<sup>β</sup> in an approximately 1:1 molar ratio, and another was combined with buffer A + 0.05% LDAO as a control. The samples were diluted to 4 mL in the same buffer and then incubated for 4 h before loading onto separate Ni-NTA columns. Flow-through was collected and then the column was washed with 6 column volumes of 40 mM imidazole + buffer B, and then 300 mM imidazole + buffer B was used for elution. Samples were analysed by SDS-PAGE.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Disulfide crosslinking assay</h3>
<p id="Par44">To encourage disulfide bond formation, protein aliquots post SEC purification at approximately 3 µM were incubated with 0.1 mM CuSO<sub>4</sub> at 4 °C. For peptide crosslinking, peptide was added to a 10 µM final concentration. After specified time periods, samples were boiled with 4x Laemmli loading dye with or without 50 mM DTT prior to SDS-PAGE analysis.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Conservation analysis</h3>
<p id="Par45">ConSurf was used for conservation analysis<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. The AlphaFold database structure of each OEP of interest from <em>A. thaliana</em> was used as the input. For this analysis, HMMER was used to search for homologues from UniRef90 with an <em>E</em>-value of 0.0001. The maximal % identity between sequences was 95%, and minimum was 35%. MAFFT was used to build MSAs. A sample of 150 sequences were selected for conservation score calculation using the Bayesian method.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Statistics and reproducibility</h3>
<p id="Par46">Traces for size exclusion chromatography are representative of at least 2 repeats of the same sample and buffer conditions. CD data was averaged over 10 accumulations and is representative of 2 repeats which showed similar spectra. Native MS spectra are representative of 4 repeats. For statistical analysis of native MS results, comparisons for two groups were calculated using unpaired two-tailed Student’s <em>t</em>-tests, with <em>n</em> = 4. Threshold <em>p</em> values are indicated in figure legends. Means, standard deviations, effect sizes, confidence intervals, exact <em>p</em> values and t statistics are available via Figshare 10.6084/m9.figshare.29733101, under Source Data for Charts and Graphs (Main Fig. <a href="#Fig5" class="usa-link">5</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">15</a>).</p></section><section id="Sec24"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par47">Further information on research design is available in the <a href="#MOESM2" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec25"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373785/bin/42003_2025_8689_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary information</a><sup> (10.2MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12373785/bin/42003_2025_8689_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting summary</a><sup> (1.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12373785/bin/42003_2025_8689_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.7MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank D. Staunton from the Molecular Biophysics Suite of the Oxford Biochemistry department for assistance with CD, as well as Aron Czako and Dušan Živković for scientific support and discussions. This research was supported by the Royal Society (URF\R1\211567 awarded to J.R.B.); UK Research and Innovation (UKRI) Frontier Research Guarantee Grant (EP/Y036158/1 awarded to J.R.B.); and UKRI Biotechnology and Biological Sciences Research Council (BB/V007300/1, BB/W015021/1, BB/X000192/1 and APP23489 awarded to R.P.J.). For the purpose of Open Access, the authors have applied a CC BY public copyright licence to any Author Accepted Manuscript (AAM) version arising from this submission.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Conceptualisation of project: R.J.D. and J.R.B.; Methodology: R.J.D. and J.R.B.; Investigation: R.J.D. and J.R.B.; Funding acquisition: R.J.D., R.P.J., and J.R.B.; Project administration: J.R.B.; Supervision: J.R.B.; Writing – original draft: R.J.D. and J.R.B.; Writing – review and editing: R.J.D., R.P.J., and J.R.B. All authors commented on the final version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par48"><em>Communications Biology</em> thanks Yih-Lin Chen and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling Editors: Xiaoling Xu and David Favero. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The authors declare that data displayed in all graphs and charts in the main manuscript and Supplementary Information as well as all native MS. raw files are available via Figshare under 10.6084/m9.figshare.29733101. Uncropped unedited gel images are available as Supplementary Figs. <a href="#MOESM1" class="usa-link">17</a>–<a href="#MOESM1" class="usa-link">20</a> in the Supplementary Information file. All other data are available from the corresponding author on reasonable request.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par49">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec26"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s42003-025-08689-2.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Keeling, P. J. The endosymbiotic origin, diversification and fate of plastids. <em>Philos. Trans. R. Soc. B Biol. Sci.</em><strong>365</strong>, 729–748 (2010).</cite> [<a href="https://doi.org/10.1098/rstb.2009.0103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2817223/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20124341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Keeling,%20P.%20J.%20The%20endosymbiotic%20origin,%20diversification%20and%20fate%20of%20plastids.%20Philos.%20Trans.%20R.%20Soc.%20B%20Biol.%20Sci.365,%20729%E2%80%93748%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Hayashi, S., Buchanan, S. K. &amp; Botos, I. The name is barrel, β-barrel. <em>Methods Mol. Biol.</em><strong>2778</strong>, 1–30 (2024).
</cite> [<a href="https://doi.org/10.1007/978-1-0716-3734-0_1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38478268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hayashi,%20S.,%20Buchanan,%20S.%20K.%20&amp;%20Botos,%20I.%20The%20name%20is%20barrel,%20%CE%B2-barrel.%20Methods%20Mol.%20Biol.2778,%201%E2%80%9330%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Diederichs, K. A., Buchanan, S. K. &amp; Botos, I. Building better barrels – β-barrel biogenesis and insertion in bacteria and mitochondria. <em>J. Mol. Biol.</em><strong>433</strong>, 166894 (2021).
</cite> [<a href="https://doi.org/10.1016/j.jmb.2021.166894" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8292188/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33639212/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Diederichs,%20K.%20A.,%20Buchanan,%20S.%20K.%20&amp;%20Botos,%20I.%20Building%20better%20barrels%20%E2%80%93%20%CE%B2-barrel%20biogenesis%20and%20insertion%20in%20bacteria%20and%20mitochondria.%20J.%20Mol.%20Biol.433,%20166894%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Kaur, H. et al. The antibiotic darobactin mimics a β-strand to inhibit outer membrane insertase. <em>Nature</em><strong>593</strong>, 125–129 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-021-03455-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33854236/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kaur,%20H.%20et%20al.%20The%20antibiotic%20darobactin%20mimics%20a%20%CE%B2-strand%20to%20inhibit%20outer%20membrane%20insertase.%20Nature593,%20125%E2%80%93129%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Steenhuis, M., van Ulsen, P., Martin, N. I. &amp; Luirink, J. A ban on BAM: an update on inhibitors of the β-barrel assembly machinery. <em>FEMS Microbiol. Lett.</em><strong>368</strong>10.1093/femsle/fnab059 (2021).</cite> [<a href="https://doi.org/10.1093/femsle/fnab059" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34048543/" class="usa-link">PubMed</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Tomasek, D. &amp; Kahne, D. The assembly of β-barrel outer membrane proteins. <em>Curr. Opin. Microbiol.</em><strong>60</strong>, 16–23 (2021).
</cite> [<a href="https://doi.org/10.1016/j.mib.2021.01.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7988294/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33561734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tomasek,%20D.%20&amp;%20Kahne,%20D.%20The%20assembly%20of%20%CE%B2-barrel%20outer%20membrane%20proteins.%20Curr.%20Opin.%20Microbiol.60,%2016%E2%80%9323%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Lundquist, K., Billings, E., Bi, M., Wellnitz, J. &amp; Noinaj, N. The assembly of β-barrel membrane proteins by BAM and SAM. <em>Mol. Microbiol.</em><strong>115</strong>, 425–435 (2021).
</cite> [<a href="https://doi.org/10.1111/mmi.14666" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8608283/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33314350/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lundquist,%20K.,%20Billings,%20E.,%20Bi,%20M.,%20Wellnitz,%20J.%20&amp;%20Noinaj,%20N.%20The%20assembly%20of%20%CE%B2-barrel%20membrane%20proteins%20by%20BAM%20and%20SAM.%20Mol.%20Microbiol.115,%20425%E2%80%93435%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Li, H. M. &amp; Chiu, C. C. Protein transport into chloroplasts. <em>Annu. Rev. Plant Biol.</em><strong>61</strong>, 157–180 (2010).
</cite> [<a href="https://doi.org/10.1146/annurev-arplant-042809-112222" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20192748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20H.%20M.%20&amp;%20Chiu,%20C.%20C.%20Protein%20transport%20into%20chloroplasts.%20Annu.%20Rev.%20Plant%20Biol.61,%20157%E2%80%93180%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Ballabani, G., Forough, M., Kessler, F. &amp; Shanmugabalaji, V. The journey of preproteins across the chloroplast membrane systems. <em>Front Physiol.</em><strong>14</strong>, 1213866 (2023).
</cite> [<a href="https://doi.org/10.3389/fphys.2023.1213866" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10267391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37324391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ballabani,%20G.,%20Forough,%20M.,%20Kessler,%20F.%20&amp;%20Shanmugabalaji,%20V.%20The%20journey%20of%20preproteins%20across%20the%20chloroplast%20membrane%20systems.%20Front%20Physiol.14,%201213866%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Shi, L. X. &amp; Theg, S. M. The chloroplast protein import system: from algae to trees. <em>Biochim. Biophys. Acta</em><strong>1833</strong>, 314–331 (2013).
</cite> [<a href="https://doi.org/10.1016/j.bbamcr.2012.10.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23063942/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shi,%20L.%20X.%20&amp;%20Theg,%20S.%20M.%20The%20chloroplast%20protein%20import%20system:%20from%20algae%20to%20trees.%20Biochim.%20Biophys.%20Acta1833,%20314%E2%80%93331%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Ling, Q. et al. Ubiquitin-dependent chloroplast-associated protein degradation in plants. <em>Science</em><strong>363</strong>10.1126/science.aav4467 (2019).</cite> [<a href="https://doi.org/10.1126/science.aav4467" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30792274/" class="usa-link">PubMed</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Sun, Y. &amp; Jarvis, R. P. Chloroplast proteostasis: import, sorting, ubiquitination, and proteolysis. <em>Annu. Rev. Plant Biol.</em><strong>74</strong>, 259–283 (2023).
</cite> [<a href="https://doi.org/10.1146/annurev-arplant-070122-032532" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36854475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun,%20Y.%20&amp;%20Jarvis,%20R.%20P.%20Chloroplast%20proteostasis:%20import,%20sorting,%20ubiquitination,%20and%20proteolysis.%20Annu.%20Rev.%20Plant%20Biol.74,%20259%E2%80%93283%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Günsel, U. et al. Structural basis of metabolite transport by the chloroplast outer envelope channel OEP21. <em>Nat. Struct. Mol. Biol.</em><strong>30</strong>, 761–769 (2023).
</cite> [<a href="https://doi.org/10.1038/s41594-023-00984-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10279527/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37156968/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?G%C3%BCnsel,%20U.%20et%20al.%20Structural%20basis%20of%20metabolite%20transport%20by%20the%20chloroplast%20outer%20envelope%20channel%20OEP21.%20Nat.%20Struct.%20Mol.%20Biol.30,%20761%E2%80%93769%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Harsman, A. et al. OEP40, a regulated glucose-permeable β-barrel solute channel in the chloroplast outer envelope membrane *. <em>J. Biol. Chem.</em><strong>291</strong>, 17848–17860 (2016).
</cite> [<a href="https://doi.org/10.1074/jbc.M115.712398" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5016175/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27339897/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harsman,%20A.%20et%20al.%20OEP40,%20a%20regulated%20glucose-permeable%20%CE%B2-barrel%20solute%20channel%20in%20the%20chloroplast%20outer%20envelope%20membrane%20*.%20J.%20Biol.%20Chem.291,%2017848%E2%80%9317860%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Röhl, T., Motzkus, M. &amp; Soll, J. The outer envelope protein OEP24 from pea chloroplasts can functionally replace the mitochondrial VDAC in yeast. <em>FEBS Lett.</em><strong>460</strong>, 491–494 (1999).
</cite> [<a href="https://doi.org/10.1016/s0014-5793(99)01399-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10556523/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?R%C3%B6hl,%20T.,%20Motzkus,%20M.%20&amp;%20Soll,%20J.%20The%20outer%20envelope%20protein%20OEP24%20from%20pea%20chloroplasts%20can%20functionally%20replace%20the%20mitochondrial%20VDAC%20in%20yeast.%20FEBS%20Lett.460,%20491%E2%80%93494%20(1999)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Goetze, T. A., Philippar, K., Ilkavets, I., Soll, J. &amp; Wagner, R. OEP37 is a new member of the chloroplast outer membrane ion channels *. <em>J. Biol. Chem.</em><strong>281</strong>, 17989–17998 (2006).
</cite> [<a href="https://doi.org/10.1074/jbc.M600700200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16624824/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goetze,%20T.%20A.,%20Philippar,%20K.,%20Ilkavets,%20I.,%20Soll,%20J.%20&amp;%20Wagner,%20R.%20OEP37%20is%20a%20new%20member%20of%20the%20chloroplast%20outer%20membrane%20ion%20channels%20*.%20J.%20Biol.%20Chem.281,%2017989%E2%80%9317998%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Schwenkert, S., Leister, D. &amp; Lo, W. T. Hidden keyholders – exploring metabolite transport across the outer chloroplast membrane. <em>J. Mitochondria Plastids Endosymbiosis</em><strong>1</strong>, 2247168 (2023).</cite> [<a href="https://scholar.google.com/scholar_lookup?Schwenkert,%20S.,%20Leister,%20D.%20&amp;%20Lo,%20W.%20T.%20Hidden%20keyholders%20%E2%80%93%20exploring%20metabolite%20transport%20across%20the%20outer%20chloroplast%20membrane.%20J.%20Mitochondria%20Plastids%20Endosymbiosis1,%202247168%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Gross, L. E. et al. Insertion of plastidic β-barrel proteins into the outer envelopes of plastids involves an intermembrane space intermediate formed with Toc75-V/OEP80. <em>Plant Cell</em><strong>33</strong>, 1657–1681 (2021).
</cite> [<a href="https://doi.org/10.1093/plcell/koab052" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8254496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33624803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gross,%20L.%20E.%20et%20al.%20Insertion%20of%20plastidic%20%CE%B2-barrel%20proteins%20into%20the%20outer%20envelopes%20of%20plastids%20involves%20an%20intermembrane%20space%20intermediate%20formed%20with%20Toc75-V/OEP80.%20Plant%20Cell33,%201657%E2%80%931681%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Patel, R., Hsu, S.-C., Bédard, J., Inoue, K. &amp; Jarvis, P. The Omp85-related chloroplast outer envelope protein OEP80 is essential for viability in arabidopsis. <em>Plant Physiol.</em><strong>148</strong>, 235–245 (2008).
</cite> [<a href="https://doi.org/10.1104/pp.108.122754" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2528115/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18621981/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Patel,%20R.,%20Hsu,%20S.-C.,%20B%C3%A9dard,%20J.,%20Inoue,%20K.%20&amp;%20Jarvis,%20P.%20The%20Omp85-related%20chloroplast%20outer%20envelope%20protein%20OEP80%20is%20essential%20for%20viability%20in%20arabidopsis.%20Plant%20Physiol.148,%20235%E2%80%93245%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Huang, W., Ling, Q., Bédard, J., Lilley, K. &amp; Jarvis, P. In vivo analyses of the roles of essential Omp85-related proteins in the chloroplast outer envelope membrane. <em>Plant Physiol.</em><strong>157</strong>, 147–159 (2011).
</cite> [<a href="https://doi.org/10.1104/pp.111.181891" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3165866/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21757633/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang,%20W.,%20Ling,%20Q.,%20B%C3%A9dard,%20J.,%20Lilley,%20K.%20&amp;%20Jarvis,%20P.%20In%20vivo%20analyses%20of%20the%20roles%20of%20essential%20Omp85-related%20proteins%20in%20the%20chloroplast%20outer%20envelope%20membrane.%20Plant%20Physiol.157,%20147%E2%80%93159%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Yoshimura, R. et al. CRUMPLED LEAF supports plastid OUTER ENVELOPE PROTEIN OF 80 KDA complex formation in Arabidopsis. <em>Plant Physiol.</em><strong>194</strong>, 2422–2433 (2024).
</cite> [<a href="https://doi.org/10.1093/plphys/kiae005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38235762/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yoshimura,%20R.%20et%20al.%20CRUMPLED%20LEAF%20supports%20plastid%20OUTER%20ENVELOPE%20PROTEIN%20OF%2080%20KDA%20complex%20formation%20in%20Arabidopsis.%20Plant%20Physiol.194,%202422%E2%80%932433%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Höhr, A. I. C. et al. Membrane protein insertion through a mitochondrial β-barrel gate. <em>Science</em><strong>359</strong>, eaah6834 (2018).
</cite> [<a href="https://doi.org/10.1126/science.aah6834" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5959003/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29348211/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?H%C3%B6hr,%20A.%20I.%20C.%20et%20al.%20Membrane%20protein%20insertion%20through%20a%20mitochondrial%20%CE%B2-barrel%20gate.%20Science359,%20eaah6834%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Höhr, A. I., Straub, S. P., Warscheid, B., Becker, T. &amp; Wiedemann, N. Assembly of β-barrel proteins in the mitochondrial outer membrane. <em>Biochim Biophys. Acta</em><strong>1853</strong>, 74–88 (2015).
</cite> [<a href="https://doi.org/10.1016/j.bbamcr.2014.10.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25305573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?H%C3%B6hr,%20A.%20I.,%20Straub,%20S.%20P.,%20Warscheid,%20B.,%20Becker,%20T.%20&amp;%20Wiedemann,%20N.%20Assembly%20of%20%CE%B2-barrel%20proteins%20in%20the%20mitochondrial%20outer%20membrane.%20Biochim%20Biophys.%20Acta1853,%2074%E2%80%9388%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Germany, E. M. et al. Dual recognition of multiple signals in bacterial outer membrane proteins enhances assembly and maintains membrane integrity. <em>eLife</em><strong>12</strong>, RP90274 (2024).
</cite> [<a href="https://doi.org/10.7554/eLife.90274" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10945584/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38226797/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Germany,%20E.%20M.%20et%20al.%20Dual%20recognition%20of%20multiple%20signals%20in%20bacterial%20outer%20membrane%20proteins%20enhances%20assembly%20and%20maintains%20membrane%20integrity.%20eLife12,%20RP90274%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Doyle, M. T. &amp; Bernstein, H. D. Function of the Omp85 superfamily of outer membrane protein assembly factors and polypeptide transporters. <em>Annu. Rev. Microbiol.</em><strong>76</strong>, 259–279 (2022).
</cite> [<a href="https://doi.org/10.1146/annurev-micro-033021-023719" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35650668/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Doyle,%20M.%20T.%20&amp;%20Bernstein,%20H.%20D.%20Function%20of%20the%20Omp85%20superfamily%20of%20outer%20membrane%20protein%20assembly%20factors%20and%20polypeptide%20transporters.%20Annu.%20Rev.%20Microbiol.76,%20259%E2%80%93279%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Takeda, H. et al. A multipoint guidance mechanism for β-barrel folding on the SAM complex. <em>Nat. Struct. Mol. Biol.</em><strong>30</strong>, 176–187 (2023).
</cite> [<a href="https://doi.org/10.1038/s41594-022-00897-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36604501/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Takeda,%20H.%20et%20al.%20A%20multipoint%20guidance%20mechanism%20for%20%CE%B2-barrel%20folding%20on%20the%20SAM%20complex.%20Nat.%20Struct.%20Mol.%20Biol.30,%20176%E2%80%93187%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Shen, C. et al. Structural basis of BAM-mediated outer membrane β-barrel protein assembly. <em>Nature</em><strong>617</strong>, 185–193 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-05988-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37100902/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen,%20C.%20et%20al.%20Structural%20basis%20of%20BAM-mediated%20outer%20membrane%20%CE%B2-barrel%20protein%20assembly.%20Nature617,%20185%E2%80%93193%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Fish, M., Saudan, G., Chuong, S. D. X., Jelokhani-Niaraki, M. &amp; Smith, M. D. AtTOC159 receptors are targeted to the chloroplast outer membrane by a β-signal and galactolipid-specific transit peptide-like sequence at the C-terminus. <em>bioRxiv</em>, 2024.2010.2025.620236 10.1101/2024.10.25.620236 (2024).</cite>
</li>
<li id="CR29">
<span class="label">29.</span><cite>Delmar, J. A., Bolla, J. R., Su, C.-C. &amp; Yu, E. W. in <em>Methods in Enzymology</em> Vol. 557 (ed Shukla, A. K.) 363–392 (Academic Press, 2015).</cite> [<a href="https://doi.org/10.1016/bs.mie.2014.12.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4755347/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25950974/" class="usa-link">PubMed</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>O’Neil, P. K. et al. The POTRA domains of Toc75 exhibit chaperone-like function to facilitate import into chloroplasts. <em>Proc. Natl. Acad. Sci. USA</em><strong>114</strong>, E4868–E4876 (2017).
</cite> [<a href="https://doi.org/10.1073/pnas.1621179114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5474763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28559331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?O%E2%80%99Neil,%20P.%20K.%20et%20al.%20The%20POTRA%20domains%20of%20Toc75%20exhibit%20chaperone-like%20function%20to%20facilitate%20import%20into%20chloroplasts.%20Proc.%20Natl.%20Acad.%20Sci.%20USA114,%20E4868%E2%80%93E4876%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Miles, A. J. &amp; Wallace, B. A. Circular dichroism spectroscopy of membrane proteins. <em>Chem. Soc. Rev.</em><strong>45</strong>, 4859–4872 (2016).
</cite> [<a href="https://doi.org/10.1039/c5cs00084j" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27347568/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miles,%20A.%20J.%20&amp;%20Wallace,%20B.%20A.%20Circular%20dichroism%20spectroscopy%20of%20membrane%20proteins.%20Chem.%20Soc.%20Rev.45,%204859%E2%80%934872%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Noinaj, N., Kuszak, A. damJ., Balusek, C., Gumbart, James, C. &amp; Buchanan, Susan, K. Lateral opening and exit pore formation are required for BamA function. <em>Structure</em><strong>22</strong>, 1055–1062 (2014).
</cite> [<a href="https://doi.org/10.1016/j.str.2014.05.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4100585/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24980798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Noinaj,%20N.,%20Kuszak,%20A.%20damJ.,%20Balusek,%20C.,%20Gumbart,%20James,%20C.%20&amp;%20Buchanan,%20Susan,%20K.%20Lateral%20opening%20and%20exit%20pore%20formation%20are%20required%20for%20BamA%20function.%20Structure22,%201055%E2%80%931062%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Kuszak, A. J., Noinaj, N. &amp; Buchanan, S. K. Methods to characterize folding and function of BamA cross-link mutants. <em>Methods Mol. Biol.</em><strong>1329</strong>, 137–147 (2015).
</cite> [<a href="https://doi.org/10.1007/978-1-4939-2871-2_10" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4880027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26427681/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuszak,%20A.%20J.,%20Noinaj,%20N.%20&amp;%20Buchanan,%20S.%20K.%20Methods%20to%20characterize%20folding%20and%20function%20of%20BamA%20cross-link%20mutants.%20Methods%20Mol.%20Biol.1329,%20137%E2%80%93147%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Burgess, N. K., Dao, T. P., Stanley, A. M. &amp; Fleming, K. G. β-Barrel proteins that reside in the Escherichia coli outer membrane in vivo demonstrate varied folding behavior in vitro*. <em>J. Biol. Chem.</em><strong>283</strong>, 26748–26758 (2008).
</cite> [<a href="https://doi.org/10.1074/jbc.M802754200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3258919/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18641391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Burgess,%20N.%20K.,%20Dao,%20T.%20P.,%20Stanley,%20A.%20M.%20&amp;%20Fleming,%20K.%20G.%20%CE%B2-Barrel%20proteins%20that%20reside%20in%20the%20Escherichia%20coli%20outer%20membrane%20in%20vivo%20demonstrate%20varied%20folding%20behavior%20in%20vitro*.%20J.%20Biol.%20Chem.283,%2026748%E2%80%9326758%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Abramson, J. et al. Accurate structure prediction of biomolecular interactions with AlphaFold 3. <em>Nature</em><strong>630</strong>, 493–500 (2024).
</cite> [<a href="https://doi.org/10.1038/s41586-024-07487-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11168924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38718835/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abramson,%20J.%20et%20al.%20Accurate%20structure%20prediction%20of%20biomolecular%20interactions%20with%20AlphaFold%203.%20Nature630,%20493%E2%80%93500%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Doyle, M. T. et al. Cryo-EM structures reveal multiple stages of bacterial outer membrane protein folding. <em>Cell</em><strong>185</strong>, 1143–1156.e1113 (2022).
</cite> [<a href="https://doi.org/10.1016/j.cell.2022.02.016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8985213/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35294859/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Doyle,%20M.%20T.%20et%20al.%20Cryo-EM%20structures%20reveal%20multiple%20stages%20of%20bacterial%20outer%20membrane%20protein%20folding.%20Cell185,%201143%E2%80%931156.e1113%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Yariv, B. et al. Using evolutionary data to make sense of macromolecules with a “face-lifted” ConSurf. <em>Protein Sci.</em><strong>32</strong>, e4582 (2023).
</cite> [<a href="https://doi.org/10.1002/pro.4582" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9942591/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36718848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yariv,%20B.%20et%20al.%20Using%20evolutionary%20data%20to%20make%20sense%20of%20macromolecules%20with%20a%20%E2%80%9Cface-lifted%E2%80%9D%20ConSurf.%20Protein%20Sci.32,%20e4582%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Wang, Z., Anderson, N. S. &amp; Benning, C. The phosphatidic acid binding site of the &lt;em&gt;Arabidopsis&lt;/em&gt; trigalactosyldiacylglycerol 4 (TGD4) protein required for lipid import into chloroplasts *. <em>J. Biol. Chem.</em><strong>288</strong>, 4763–4771 (2013).
</cite> [<a href="https://doi.org/10.1074/jbc.M112.438986" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3576081/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23297418/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Z.,%20Anderson,%20N.%20S.%20&amp;%20Benning,%20C.%20The%20phosphatidic%20acid%20binding%20site%20of%20the%20&lt;em&gt;Arabidopsis&lt;/em&gt;%20trigalactosyldiacylglycerol%204%20(TGD4)%20protein%20required%20for%20lipid%20import%20into%20chloroplasts%20*.%20J.%20Biol.%20Chem.288,%204763%E2%80%934771%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Hsueh, Y.-C., Ehmann, C., Flinner, N., Ladig, R. &amp; Schleiff, E. The plastid outer membrane localized LPTD1 is important for glycerolipid remodelling under phosphate starvation. <em>Plant Cell Environ.</em><strong>40</strong>, 1643–1657 (2017).
</cite> [<a href="https://doi.org/10.1111/pce.12973" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28433003/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hsueh,%20Y.-C.,%20Ehmann,%20C.,%20Flinner,%20N.,%20Ladig,%20R.%20&amp;%20Schleiff,%20E.%20The%20plastid%20outer%20membrane%20localized%20LPTD1%20is%20important%20for%20glycerolipid%20remodelling%20under%20phosphate%20starvation.%20Plant%20Cell%20Environ.40,%201643%E2%80%931657%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Day, P. M., Potter, D. &amp; Inoue, K. Evolution and targeting of Omp85 homologs in the chloroplast outer envelope membrane. <em>Front. Plant Sci.</em><strong>5</strong>10.3389/fpls.2014.00535 (2014).</cite> [<a href="https://doi.org/10.3389/fpls.2014.00535" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4195282/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25352854/" class="usa-link">PubMed</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Liu, H., Li, A., Rochaix, J.-D. &amp; Liu, Z. Architecture of chloroplast TOC–TIC translocon supercomplex. <em>Nature</em><strong>615</strong>, 349–357 (2023).
</cite> [<a href="https://doi.org/10.1038/s41586-023-05744-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36702157/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20H.,%20Li,%20A.,%20Rochaix,%20J.-D.%20&amp;%20Liu,%20Z.%20Architecture%20of%20chloroplast%20TOC%E2%80%93TIC%20translocon%20supercomplex.%20Nature615,%20349%E2%80%93357%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Xiao, L. et al. Structures of the β-barrel assembly machine recognizing outer membrane protein substrates. <em> FASEB J.</em><strong>35</strong>, e21207 (2021).
</cite> [<a href="https://doi.org/10.1096/fj.202001443RR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33368572/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xiao,%20L.%20et%20al.%20Structures%20of%20the%20%CE%B2-barrel%20assembly%20machine%20recognizing%20outer%20membrane%20protein%20substrates.%20FASEB%20J.35,%20e21207%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Gopinath, A. &amp; Joseph, B. Conformational flexibility of the protein insertase BamA in the native asymmetric bilayer elucidated by ESR spectroscopy. <em>Angew. Chem. Int. Ed.</em><strong>61</strong>, e202113448 (2022).</cite> [<a href="https://doi.org/10.1002/anie.202113448" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9299766/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34761852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gopinath,%20A.%20&amp;%20Joseph,%20B.%20Conformational%20flexibility%20of%20the%20protein%20insertase%20BamA%20in%20the%20native%20asymmetric%20bilayer%20elucidated%20by%20ESR%20spectroscopy.%20Angew.%20Chem.%20Int.%20Ed.61,%20e202113448%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Takeda, H. et al. Mitochondrial sorting and assembly machinery operates by β-barrel switching. <em>Nature</em><strong>590</strong>, 163–169 (2021).
</cite> [<a href="https://doi.org/10.1038/s41586-020-03113-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33408415/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Takeda,%20H.%20et%20al.%20Mitochondrial%20sorting%20and%20assembly%20machinery%20operates%20by%20%CE%B2-barrel%20switching.%20Nature590,%20163%E2%80%93169%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Wang, X., Nyenhuis, S. B. &amp; Bernstein, H. D. The translocation assembly module (TAM) catalyzes the assembly of bacterial outer membrane proteins in vitro. <em>Nat. Commun.</em><strong>15</strong>, 7246 (2024).
</cite> [<a href="https://doi.org/10.1038/s41467-024-51628-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11341756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39174534/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20X.,%20Nyenhuis,%20S.%20B.%20&amp;%20Bernstein,%20H.%20D.%20The%20translocation%20assembly%20module%20(TAM)%20catalyzes%20the%20assembly%20of%20bacterial%20outer%20membrane%20proteins%20in%20vitro.%20Nat.%20Commun.15,%207246%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Stubenrauch Christopher, J. &amp; Lithgow, T. The TAM: a translocation and assembly module of the β-barrel assembly machinery in bacterial outer membranes. <em>EcoSal</em><strong>8</strong>, 10.1128/ecosalplus.ESP-0036-2018 (2019).</cite> [<a href="https://doi.org/10.1128/ecosalplus.esp-0036-2018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11573294/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30816086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stubenrauch%20Christopher,%20J.%20&amp;%20Lithgow,%20T.%20The%20TAM:%20a%20translocation%20and%20assembly%20module%20of%20the%20%CE%B2-barrel%20assembly%20machinery%20in%20bacterial%20outer%20membranes.%20EcoSal8,%2010.1128/ecosalplus.ESP-0036-2018%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Wang, Z. Z. et al. Structural insights into Deinococcus radiodurans BamA: extracellular loop diversity and its evolutionary implications. <em>JUSTC</em>10.52396/JUSTC-2024-0012 (2024).</cite>
</li>
<li id="CR48">
<span class="label">48.</span><cite>Höhr, A. I. C. et al. Membrane protein insertion through a mitochondrial β-barrel gate. <em>Science</em><strong>359</strong>10.1126/science.aah6834 (2018).</cite> [<a href="https://doi.org/10.1126/science.aah6834" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5959003/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29348211/" class="usa-link">PubMed</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Landau, M. et al. ConSurf 2005: the projection of evolutionary conservation scores of residues on protein structures. <em>Nucleic Acids Res.</em><strong>33</strong>, W299–W302 (2005).
</cite> [<a href="https://doi.org/10.1093/nar/gki370" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1160131/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15980475/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Landau,%20M.%20et%20al.%20ConSurf%202005:%20the%20projection%20of%20evolutionary%20conservation%20scores%20of%20residues%20on%20protein%20structures.%20Nucleic%20Acids%20Res.33,%20W299%E2%80%93W302%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373785/bin/42003_2025_8689_MOESM1_ESM.docx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary information</a><sup> (10.2MB, docx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373785/bin/42003_2025_8689_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting summary</a><sup> (1.5MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373785/bin/42003_2025_8689_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (1.7MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The authors declare that data displayed in all graphs and charts in the main manuscript and Supplementary Information as well as all native MS. raw files are available via Figshare under 10.6084/m9.figshare.29733101. Uncropped unedited gel images are available as Supplementary Figs. <a href="#MOESM1" class="usa-link">17</a>–<a href="#MOESM1" class="usa-link">20</a> in the Supplementary Information file. All other data are available from the corresponding author on reasonable request.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Communications Biology are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s42003-025-08689-2"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/42003_2025_Article_8689.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (1.8 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373785/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373785/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373785%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373785/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373785/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373785/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40846801/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373785/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40846801/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373785/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373785/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="wLdvsDNcuQuGibaQ35ZxoMNjyYjfbhztNfMAb37s1OKHtbZrpphMWB03E2eYUUi2">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
