
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Leveraging Retrieval-Augmented Large Language Models for Dietary Recommendations With Traditional Chinese Medicine’s Medicine Food Homology: Algorithm Development and Validation - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152539AC8AF201630539AC005403BF7F.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="medinform">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370266/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="JMIR Medical Informatics">
<meta name="citation_title" content="Leveraging Retrieval-Augmented Large Language Models for Dietary Recommendations With Traditional Chinese Medicine’s Medicine Food Homology: Algorithm Development and Validation">
<meta name="citation_author" content="Hangyu Sha">
<meta name="citation_author_institution" content="School of Computer Science and Engineering, Southeast University, 2 Southeast University Road, Jiangning District, Nanjing, 210096, China, 86 15077889931">
<meta name="citation_author_institution" content="Key Laboratory of New Generation Artificial Intelligence Technology and Its Interdisciplinary Applications (Southeast University), Ministry of Education, China">
<meta name="citation_author" content="Fan Gong">
<meta name="citation_author_institution" content="Department of Endocrinology, Shuguang Hospital Affiliated to Shanghai University of Traditional Chinese Medicine, Shanghai, China">
<meta name="citation_author" content="Bo Liu">
<meta name="citation_author_institution" content="Informatization Office, Shanghai University of Traditional Chinese Medicine, Shanghai, China">
<meta name="citation_author" content="Runfeng Liu">
<meta name="citation_author_institution" content="School of Computer Science and Engineering, Southeast University, 2 Southeast University Road, Jiangning District, Nanjing, 210096, China, 86 15077889931">
<meta name="citation_author_institution" content="Key Laboratory of New Generation Artificial Intelligence Technology and Its Interdisciplinary Applications (Southeast University), Ministry of Education, China">
<meta name="citation_author" content="Haofen Wang">
<meta name="citation_author_institution" content="College of Design and Innovation, Tongji University, Shanghai, China">
<meta name="citation_author" content="Tianxing Wu">
<meta name="citation_author_institution" content="School of Computer Science and Engineering, Southeast University, 2 Southeast University Road, Jiangning District, Nanjing, 210096, China, 86 15077889931">
<meta name="citation_author_institution" content="Key Laboratory of New Generation Artificial Intelligence Technology and Its Interdisciplinary Applications (Southeast University), Ministry of Education, China">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="13">
<meta name="citation_firstpage" content="e75279">
<meta name="citation_doi" content="10.2196/75279">
<meta name="citation_pmid" content="40840437">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370266/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370266/">
<meta name="description" content="Traditional Chinese Medicine (TCM) emphasizes the concept of medicine food homology (MFH), which integrates dietary therapy into health care. However, the practical application of MFH principles relies heavily on expert knowledge and manual ...">
<meta name="og:title" content="Leveraging Retrieval-Augmented Large Language Models for Dietary Recommendations With Traditional Chinese Medicine’s Medicine Food Homology: Algorithm Development and Validation">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Traditional Chinese Medicine (TCM) emphasizes the concept of medicine food homology (MFH), which integrates dietary therapy into health care. However, the practical application of MFH principles relies heavily on expert knowledge and manual ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370266/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370266">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.2196/75279"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370266%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370266/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370266/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370266/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-medinform.png" alt="JMIR Medical Informatics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to JMIR Medical Informatics" title="Link to JMIR Medical Informatics" shape="default" href="https://doi.org/10.2196/75279" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">JMIR Med Inform</button></div>. 2025 Aug 21;13:e75279. doi: <a href="https://doi.org/10.2196/75279" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.2196/75279</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22JMIR%20Med%20Inform%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22JMIR%20Med%20Inform%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22JMIR%20Med%20Inform%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22JMIR%20Med%20Inform%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Leveraging Retrieval-Augmented Large Language Models for Dietary Recommendations With Traditional Chinese Medicine’s Medicine Food Homology: Algorithm Development and Validation</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sha%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Hangyu Sha</span></a><div hidden="hidden" id="id1">
<h3>
<span class="name western">Hangyu Sha</span>, <span class="degrees">BEng</span>
</h3>
<div class="p">
<sup>1</sup>School of Computer Science and Engineering, Southeast University, 2 Southeast University Road, Jiangning District, Nanjing, 210096, China, 86 15077889931</div>
<div class="p">
<sup>2</sup>Key Laboratory of New Generation Artificial Intelligence Technology and Its Interdisciplinary Applications (Southeast University), Ministry of Education, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sha%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hangyu Sha</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gong%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Fan Gong</span></a><div hidden="hidden" id="id2">
<h3>
<span class="name western">Fan Gong</span>, <span class="degrees">PhD</span>
</h3>
<div class="p">
<sup>3</sup>Department of Endocrinology, Shuguang Hospital Affiliated to Shanghai University of Traditional Chinese Medicine, Shanghai, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gong%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Fan Gong</span></a>
</div>
</div>
<sup>3,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Bo Liu</span></a><div hidden="hidden" id="id3">
<h3>
<span class="name western">Bo Liu</span>, <span class="degrees">PhD</span>
</h3>
<div class="p">
<sup>4</sup>Informatization Office, Shanghai University of Traditional Chinese Medicine, Shanghai, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bo Liu</span></a>
</div>
</div>
<sup>4,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Runfeng Liu</span></a><div hidden="hidden" id="id4">
<h3>
<span class="name western">Runfeng Liu</span>, <span class="degrees">BEng</span>
</h3>
<div class="p">
<sup>1</sup>School of Computer Science and Engineering, Southeast University, 2 Southeast University Road, Jiangning District, Nanjing, 210096, China, 86 15077889931</div>
<div class="p">
<sup>2</sup>Key Laboratory of New Generation Artificial Intelligence Technology and Its Interdisciplinary Applications (Southeast University), Ministry of Education, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Runfeng Liu</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Haofen Wang</span></a><div hidden="hidden" id="id5">
<h3>
<span class="name western">Haofen Wang</span>, <span class="degrees">PhD</span>
</h3>
<div class="p">
<sup>5</sup>College of Design and Innovation, Tongji University, Shanghai, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Haofen Wang</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Tianxing Wu</span></a><div hidden="hidden" id="id6">
<h3>
<span class="name western">Tianxing Wu</span>, <span class="degrees">PhD</span>
</h3>
<div class="p">
<sup>1</sup>School of Computer Science and Engineering, Southeast University, 2 Southeast University Road, Jiangning District, Nanjing, 210096, China, 86 15077889931</div>
<div class="p">
<sup>2</sup>Key Laboratory of New Generation Artificial Intelligence Technology and Its Interdisciplinary Applications (Southeast University), Ministry of Education, China</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tianxing Wu</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<div class="cg p">Editors: <span class="name western">Ling Luo</span>, <span class="name western">Qiao Jin</span>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff1">
<sup>1</sup>School of Computer Science and Engineering, Southeast University, 2 Southeast University Road, Jiangning District, Nanjing, 210096, China, 86 15077889931</div>
<div id="aff2">
<sup>2</sup>Key Laboratory of New Generation Artificial Intelligence Technology and Its Interdisciplinary Applications (Southeast University), Ministry of Education, China</div>
<div id="aff3">
<sup>3</sup>Department of Endocrinology, Shuguang Hospital Affiliated to Shanghai University of Traditional Chinese Medicine, Shanghai, China</div>
<div id="aff4">
<sup>4</sup>Informatization Office, Shanghai University of Traditional Chinese Medicine, Shanghai, China</div>
<div id="aff5">
<sup>5</sup>College of Design and Innovation, Tongji University, Shanghai, China</div>
<div class="author-notes p">
<div class="fn" id="equal-contrib1">
<sup>*</sup><p class="display-inline">these authors contributed equally</p>
</div>
<div class="fn" id="cor1">
<sup>✉</sup><p class="display-inline">Tianxing Wu, PhD, School of Computer Science and Engineering, Southeast University, 2 Southeast University Road, Jiangning District, Nanjing, 210096, China, 86 15077889931; <span>tianxingwu@seu.edu.cn</span></p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 31; Revised 2025 Jun 24; Accepted 2025 Jul 1; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Copyright © Hangyu Sha, Fan Gong, Bo Liu, Runfeng Liu, Haofen Wang, Tianxing Wu. Originally published in JMIR Medical Informatics (https://medinform.jmir.org)</div>
<p>This is an open-access article distributed under the terms of the Creative Commons Attribution License (<a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work, first published in JMIR Medical Informatics, is properly cited. The complete bibliographic information, a link to the original publication on <a href="https://medinform.jmir.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://medinform.jmir.org/</a>, as well as this copyright and license information must be included.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370266  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40840437/" class="usa-link">40840437</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p>Traditional Chinese Medicine (TCM) emphasizes the concept of medicine food homology (MFH), which integrates dietary therapy into health care. However, the practical application of MFH principles relies heavily on expert knowledge and manual interpretation, posing challenges for automating MFH-based dietary recommendations. Although large language models (LLMs) have shown potential in health care decision support, their performance in specialized domains such as TCM is often hindered by hallucinations and a lack of domain knowledge. The integration of uncertain knowledge graphs (UKGs) with LLMs via retrieval-augmented generation (RAG) offers a promising solution to overcome these limitations by enabling a structured and faithful representation of MFH principles while enhancing LLMs’ ability to understand the inherent uncertainty and heterogeneity of TCM knowledge. Consequently, it holds potential to improve the reliability and accuracy of MFH-based dietary recommendations generated by LLMs.</p></section><section id="sec2"><h3 class="pmc_sec_title">Objective</h3>
<p>This study aimed to introduce Yaoshi-RAG, a framework that leverages UKGs to enhance LLMs' capabilities in generating accurate and personalized MFH-based dietary recommendations.</p></section><section id="sec3"><h3 class="pmc_sec_title">Methods</h3>
<p>The proposed framework began by constructing a comprehensive MFH knowledge graph (KG) through LLM-driven open information extraction, which extracted structured knowledge from multiple sources. To address the incompleteness and uncertainty within the MFH KG, UKG reasoning was used to measure the confidence of existing triples and to complete missing triples. When processing user queries, query entities were identified and linked to the MFH KG, enabling retrieval of relevant reasoning paths. These reasoning paths were then ranked based on triple confidence scores and entity importance. Finally, the most informative reasoning paths were encoded into prompts using prompt engineering, enabling the LLM to generate personalized dietary recommendations that aligned with both individual health needs and MFH principles. The effectiveness of Yaoshi-RAG was evaluated through both automated metrics and human evaluation.</p></section><section id="sec4"><h3 class="pmc_sec_title">Results</h3>
<p>The constructed MFH KG comprised 24,984 entities, 22 relations, and 29,292 triples. Extensive experiments demonstrate the superiority of Yaoshi-RAG in different evaluation metrics. Integrating the MFH KG significantly improved the performance of LLMs, yielding an average increase of 14.5% in Hits@1 and 8.7% in <em>F</em><sub>1</sub>-score, respectively. Among the evaluated LLMs, DeepSeek-R1 achieved the best performance, with 84.2% in Hits@1 and 71.5% in <em>F</em><sub>1</sub>-score, respectively. Human evaluation further validated these results, confirming that Yaoshi-RAG consistently outperformed baseline models across all assessed quality dimensions.</p></section><section id="sec5"><h3 class="pmc_sec_title">Conclusions</h3>
<p>This study shows Yaoshi-RAG, a new framework that enhances LLMs’ capabilities in generating MFH-based dietary recommendations through the knowledge retrieved from a UKG. By constructing a comprehensive TCM knowledge representation, our framework effectively extracts and uses MFH principles. Experimental results demonstrate the effectiveness of our framework in synthesizing traditional wisdom with advanced language models, facilitating personalized dietary recommendations that address individual health conditions while providing evidence-based explanations.</p></section></section><section id="s1"><h2 class="pmc_sec_title">Introduction</h2>
<section id="s1-1"><h3 class="pmc_sec_title">Background</h3>
<p>The concept of “medicine food homology” (MFH) in Traditional Chinese Medicine (TCM) means that certain substances can function as both nutritional food and therapeutic medicines depending on their application and dosage [<a href="#R1" class="usa-link" aria-describedby="R1">1</a>]. This ancient philosophy has gained considerable traction in modern health care systems [<a href="#R2" class="usa-link" aria-describedby="R2">2</a>], offering approaches to disease prevention and management through dietary interventions [<a href="#R3" class="usa-link" aria-describedby="R3">3</a>]. However, the practical implementation of MFH principles remains largely dependent on the expert knowledge of TCM practitioners, leading to inconsistent application and limited accessibility. The absence of automated and systematic approaches to using MFH principles for dietary recommendation generation has hindered its broader integration into contemporary health care frameworks and constrained its potential benefits for public health.</p>
<p>Recent advances in large language models (LLMs) [<a href="#R4" class="usa-link" aria-describedby="R4">4</a>] have demonstrated their remarkable performance across a variety of medical tasks [<a href="#R5" class="usa-link" aria-describedby="R5">5</a>], including automated clinical decision support [<a href="#R6" class="usa-link" aria-describedby="R6">6</a>], medical education [<a href="#R7" class="usa-link" aria-describedby="R7">7</a>], and health care information retrieval [<a href="#R8" class="usa-link" aria-describedby="R8">8</a>]. These capabilities are primarily derived from extensive pretraining on large-scale corpora, which encode rich knowledge from diverse textual sources. The application of MFH principles heavily depends on the domain knowledge documented in TCM literature, including foods, herbs, and formulations. Given that many LLMs have been pretrained on the corpora, which incorporate such TCM literature, combined with their strong natural language generation abilities, they present promising potential for automating MFH-based dietary recommendations. Nonetheless, LLMs exhibit substantial limitations as they often produce hallucinations that refer to factually incorrect information [<a href="#R9" class="usa-link" aria-describedby="R9">9</a>]. This problem is especially evident when generating information related to the medicinal properties of food in the context of TCM. For example, when querying with “I have stomach discomfort recently, can you recommend some summer drinks?” the LLMs respond: “Mung bean soup clears heat and is suitable for heat-related discomfort.” However, according to MFH principles, mung beans are cold (yin), and their consumption may aggravate stomach discomfort. Such inaccuracies significantly affect the safety and reliability of LLMs in practical health care applications [<a href="#R10" class="usa-link" aria-describedby="R10">10</a>].</p>
<p>Retrieval-augmented generation (RAG) has emerged as a promising strategy to mitigate these limitations by incorporating external knowledge into LLMs, thereby enhancing factual accuracy and reducing hallucinations [<a href="#R11" class="usa-link" aria-describedby="R11">11</a>]. However, using TCM knowledge as the retrieval source of RAG presents unique challenges. TCM knowledge is sourced from diverse and heterogeneous materials, including ancient and classical literature, modern TCM literature, food and herbs information, regulatory documents, research papers, and TCM formulations. This multisource nature results in inconsistent knowledge representation, terminological ambiguities, and varying levels of evidence quality. Furthermore, MFH principles inherently involve uncertainties arising from dosage effects, individual variability, and context-dependent applications [<a href="#R12" class="usa-link" aria-describedby="R12">12</a>]. Currently, there is no comprehensive and structured knowledge base specifically tailored for supporting RAG grounded in MFH principles, creating a significant gap between traditional wisdom and modern technologies.</p></section><section id="s1-2"><h3 class="pmc_sec_title">Related Work</h3>
<section id="s1-2-1"><h4 class="pmc_sec_title">Dietary Recommendation Systems</h4>
<p>Dietary recommendation systems provide personalized food and nutrition suggestions based on individual health status and preferences. Traditional approaches have primarily relied on rule-based methods using expert knowledge and predefined guidelines, often lacking personalization and accessibility. Recent advancements in artificial intelligence have transformed this field through deep learning and knowledge graphs (KGs). Chen et al [<a href="#R13" class="usa-link" aria-describedby="R13">13</a>] defined food recommendation as constrained question answering over a food KG, integrating user preferences with health guidelines. PPKG [<a href="#R14" class="usa-link" aria-describedby="R14">14</a>] is a cancer-specific dietary recommendation system that combines a KG with time-aware long short-term memory networks to dynamically adjust recommendations for cancer prevention and rehabilitation. Ma et al [<a href="#R15" class="usa-link" aria-describedby="R15">15</a>] proposed a nutrition-related KG method based on graph convolutional networks to enhance food recommendation diversity while promoting more healthy eating habits. LLMs have further advanced dietary recommendation capabilities. ChatDiet [<a href="#R16" class="usa-link" aria-describedby="R16">16</a>] integrates personal and population models with causal inference to enhance personalization and explainability. Jin et al [<a href="#R17" class="usa-link" aria-describedby="R17">17</a>] demonstrated the clinical effectiveness of a GPT-based dietary recommendation system for patients with hemodialysis, successfully reducing serum potassium levels. Kopitar et al [<a href="#R18" class="usa-link" aria-describedby="R18">18</a>] created a generative artificial intelligence system for personalized inpatient meal planning that incorporates electronic health records and clinical guidelines.</p>
<p>However, we still do not have methods to integrate MFH principles with LLMs for dietary recommendation. Our framework bridges this gap by combining MFH KG with LLMs to develop more comprehensive and culturally informed dietary recommendations.</p></section><section id="s1-2-2"><h4 class="pmc_sec_title">The Interplay Between LLMs and KGs</h4>
<p>KGs serve as structured representations of factual knowledge, providing an excellent complement to parameterized language models [<a href="#R19" class="usa-link" aria-describedby="R19">19</a>,<a href="#R20" class="usa-link" aria-describedby="R20">20</a>]. With the emergence of LLMs, research has evolved along 2 complementary trajectories: leveraging LLMs to construct KGs and enhancing LLMs with KG integration. Several approaches have emerged for LLM-assisted KG construction. Extract-define-canonicalize [<a href="#R21" class="usa-link" aria-describedby="R21">21</a>] addresses scalability through open information extraction (OpenIE) and post hoc canonicalization. BEAR [<a href="#R22" class="usa-link" aria-describedby="R22">22</a>] leverages LLMs for zero-shot knowledge extraction in service-oriented domains. LLM-TIKG [<a href="#R23" class="usa-link" aria-describedby="R23">23</a>] applies few-shot learning for constructing threat intelligence KGs, while Yang et al [<a href="#R24" class="usa-link" aria-describedby="R24">24</a>] presented an approach for medical ontology expansion using LLMs to generate competency questions.</p>
<p>Researchers have also explored integrating KGs to enhance LLMs’ capabilities. GraphGPT [<a href="#R25" class="usa-link" aria-describedby="R25">25</a>] combines LLMs with graph structural knowledge through instruction tuning. GaLM [<a href="#R26" class="usa-link" aria-describedby="R26">26</a>] transforms KGs into text representations to improve reasoning while reducing hallucinations. Think-on-Graph [<a href="#R27" class="usa-link" aria-describedby="R27">27</a>] is a training-free framework that uses iterative beam search to enable interactive reasoning, while StructGPT [<a href="#R28" class="usa-link" aria-describedby="R28">28</a>] enhances zero-shot reasoning through an iterative Reading-then-Reasoning approach. Our work builds upon these dual research trajectories by proposing a framework that leverages LLMs for KG construction and uses KGs to enhance the capabilities of LLMs specifically within the domain of TCM, thereby providing reliable dietary recommendations grounded in MFH principles.</p></section></section><section id="s1-3"><h3 class="pmc_sec_title">Objectives</h3>
<p>To overcome the limitations of existing methods, we propose a new framework that enhances LLMs’ capability to generate MFH dietary recommendations by using an uncertain knowledge graph (UKG) as an external resource. This framework uses multistep LLM calls for OpenIE [<a href="#R29" class="usa-link" aria-describedby="R29">29</a>] to automatically construct an MFH KG from unstructured and heterogeneous literature sources. It then implements UKG reasoning to measure the confidence of existing triples and complete missing triples. To improve knowledge retrieval, the framework adopts a reasoning path–based knowledge retrieval strategy, extracting relevant knowledge from the MFH KG and refining the results through ranking and filtering based on triple confidence scores and entity importance. By encoding domain-specific knowledge and addressing uncertainties related to MFH, the MFH KG empowers LLMs to deliver more personalized, evidence-based, and reliable dietary recommendations.</p>
<p>The key contributions of this study are as follows: (1) an integrated framework that leverages KG-augmented LLMs to generate personalized and evidence-based MFH dietary recommendations, (2) an LLM-driven OpenIE methodology for automatically constructing a UKG specifically tailored to TCM’s MFH from heterogeneous multisource data, (3) a systematic evaluation of several mainstream LLMs for the generation of MFH dietary recommendations, and (4) experimental results and case studies that demonstrate the capability of the proposed framework to deliver personalized and professional MFH-based dietary recommendations.</p></section></section><section id="s2"><h2 class="pmc_sec_title">Methods</h2>
<section id="s2-1"><h3 class="pmc_sec_title">Overview</h3>
<p>This study proposed Yaoshi-RAG, an RAG framework designed to enhance LLMs’ capabilities in generating evidence-based dietary recommendations based on MFH principles. As illustrated in <a href="#F1" class="usa-link">Figure 1</a>, the framework consists of 2 modules:</p>
<figure class="fig xbox font-sm" id="F1"><h4 class="obj_head">Figure 1.  The architecture of the proposed framework. KG: knowledge graph; LLM: large language model; MFH: medicine food homology; OpenIE: open information extraction; TCM: Traditional Chinese Medicine; UKG: uncertain knowledge graph.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370266_medinform-v13-e75279-g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9ee/12370266/1f969ed2d0ca/medinform-v13-e75279-g001.jpg" loading="lazy" height="474" width="755" alt="Figure 1."></a></p>
<div class="p text-right font-secondary"><a href="figure/F1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><ol class="list" style="list-style-type:decimal">
<li><p>Knowledge graph construction: A comprehensive corpus was compiled from multiple sources, and an MFH KG was constructed using LLM-driven OpenIE. Then, UKG reasoning was applied to measure the confidence of extracted triples and complete missing triples.</p></li>
<li><p>Retrieval-augmented generation: Given a user query, the framework first identified and linked query entities to the MFH KG. The LLM then constructed relation paths to facilitate the retrieval of reasoning paths. Subsequently, these candidate reasoning paths were refined through postprocessing, including ranking and filtering, to extract the top-<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M1" alttext="k" overflow="linebreak"><mi>k</mi></math></span> most relevant reasoning paths, thereby improving retrieval accuracy. The retrieved knowledge was integrated with the user query through prompt engineering, guiding the LLM to generate personalized dietary recommendations that aligned with both the user’s needs and MFH principles.</p></li>
</ol>
<p>A comprehensive corpus was compiled from multiple sources, and an MFH KG was constructed using LLM-driven OpenIE. Then, UKG reasoning was applied to measure the confidence of extracted triples and complete missing triples. In the following sections, we provide a detailed explanation of both modules, outlining their design, implementation, and contributions to the overall framework.</p></section><section id="s2-2"><h3 class="pmc_sec_title">KG Construction</h3>
<section id="s2-2-1"><h4 class="pmc_sec_title">Corpus Collection</h4>
<p>To construct an MFH KG, we systematically constructed a corpus from diverse sources, including ancient and classical literature, modern TCM literature, food and herbs information, regulatory documents, research papers, and TCM formulations. This process results in a corpus of 1359 relevant documents. <a href="#T1" class="usa-link">Table 1</a> shows the statistical distribution of documents across different source categories.</p>
<section class="tw xbox font-sm" id="T1"><h5 class="obj_head">Table 1.  The statistical distribution of 1359 documents.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th valign="bottom" rowspan="1" colspan="1">Source category</th>
<th valign="bottom" rowspan="1" colspan="1">Number of documents</th>
</tr></thead>
<tbody>
<tr>
<td valign="top" rowspan="1" colspan="1">Ancient and classical literature</td>
<td align="left" valign="top" rowspan="1" colspan="1">75</td>
</tr>
<tr>
<td valign="top" rowspan="1" colspan="1">Modern TCM<a href="#T1_FN1" class="usa-link"><sup>a</sup></a> literature</td>
<td align="left" valign="top" rowspan="1" colspan="1">228</td>
</tr>
<tr>
<td valign="top" rowspan="1" colspan="1">Food and herbs information</td>
<td align="left" valign="top" rowspan="1" colspan="1">400</td>
</tr>
<tr>
<td valign="top" rowspan="1" colspan="1">Regulatory documents</td>
<td align="left" valign="top" rowspan="1" colspan="1">42</td>
</tr>
<tr>
<td valign="top" rowspan="1" colspan="1">Research papers</td>
<td align="left" valign="top" rowspan="1" colspan="1">432</td>
</tr>
<tr>
<td valign="top" rowspan="1" colspan="1">TCM formulations</td>
<td align="left" valign="top" rowspan="1" colspan="1">182</td>
</tr>
<tr>
<td valign="top" rowspan="1" colspan="1">Total</td>
<td align="left" valign="top" rowspan="1" colspan="1">1359</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="T1_FN1">
<sup>a</sup><p class="display-inline">TCM: Traditional Chinese Medicine.</p>
</div></div></section><p>To improve the efficiency and accuracy of entity-relation triple extraction, each document was segmented into passages of 300 tokens. These passages were then input into LLMs with a carefully designed prompt (Textbox S1 in <a href="#SAP1" class="usa-link">Multimedia Appendix 1</a>) to resolve pronominal references by incorporating information from document titles and neighboring passages. Additionally, irrelevant content within passages was filtered out to ensure domain-specific focus.</p></section><section id="s2-2-2"><h4 class="pmc_sec_title">MFH Knowledge Graph</h4>
<p>Based on the collected corpus, we constructed an MFH KG <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M2" alttext="G" overflow="linebreak"><mi>G</mi></math></span>, as shown in <a href="#F2" class="usa-link">Figure 2</a>, which illustrates the detailed process. Formally, the graph is defined as a triple <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M3" alttext="G=\left (E,R,F\right)" overflow="linebreak"><mi>G</mi><mo>=</mo><mo>(</mo><mi>E</mi><mo>,</mo><mi>R</mi><mo>,</mo><mi>F</mi><mo>)</mo></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M4" alttext="E" overflow="linebreak"><mi>E</mi></math></span> represents the set of entities, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M5" alttext="R" overflow="linebreak"><mi>R</mi></math></span> represents the set of relations, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M6" alttext="F\subseteq E\times R\times E" overflow="linebreak"><mi>F</mi><mo>⊆</mo><mi>E</mi><mo>×</mo><mi>R</mi><mo>×</mo><mi>E</mi></math></span> represents the fact set. Each fact in <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M7" alttext="F" overflow="linebreak"><mi>F</mi></math></span> is represented as a quadruple <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M8" alttext="\left (h,r,t,s\right)" overflow="linebreak"><mo>(</mo><mi>h</mi><mo>,</mo><mi>r</mi><mo>,</mo><mi>t</mi><mo>,</mo><mi>s</mi><mo>)</mo></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M9" alttext="h" overflow="linebreak"><mi>h</mi></math></span> is the head entity, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M10" alttext="r" overflow="linebreak"><mi>r</mi></math></span> is the relation, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M11" alttext="t" overflow="linebreak"><mi>t</mi></math></span> is the tail entity, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M12" alttext="s" overflow="linebreak"><mi>s</mi></math></span> is the confidence score reflecting the likelihood that the triple <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M13" alttext="\left (h,r,t\right)" overflow="linebreak"><mo>(</mo><mi>h</mi><mo>,</mo><mi>r</mi><mo>,</mo><mi>t</mi><mo>)</mo></math></span> holds true. For instance, consider the quadruple (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M14" alttext="h" overflow="linebreak"><mi>h</mi></math></span>: Insomnia and Vivid Dreams, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M15" alttext="r" overflow="linebreak"><mi>r</mi></math></span>: Related Constitution, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M16" alttext="t" overflow="linebreak"><mi>t</mi></math></span>: Yin-Deficient Constitution, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M17" alttext="s" overflow="linebreak"><mi>s</mi></math></span>: 0.87). This quadruple indicates that Insomnia and Vivid Dreams are associated with the Yin-Deficient Constitution, with a confidence score of 0.87.</p>
<figure class="fig xbox font-sm" id="F2"><h5 class="obj_head">Figure 2. The detailed process of medicine food homology knowledge graph construction. KG: knowledge graph; MFH: medicine food homology.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370266_medinform-v13-e75279-g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9ee/12370266/9236d6f3bad5/medinform-v13-e75279-g002.jpg" loading="lazy" height="255" width="759" alt="Figure 2."></a></p>
<div class="p text-right font-secondary"><a href="figure/F2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>We used LLM-driven OpenIE to extract triples <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M18" alttext="\ \left (h,\ r,\ t\right)" overflow="linebreak"><mi></mi><mfenced separators="|"><mrow><mi>h</mi><mo>,</mo><mi></mi><mi>r</mi><mo>,</mo><mi></mi><mi>t</mi></mrow></mfenced></math></span> from each passage within the corpus. Specifically, we first integrated predefined relation definitions into a prompt (Textbox S2 in <a href="#SAP1" class="usa-link">Multimedia Appendix 1</a>), guiding the LLM to identify and extract potential relations. These candidate relations, along with existing ones, were subsequently processed using another prompt (Textbox S3 in <a href="#SAP1" class="usa-link">Multimedia Appendix 1</a>), where the LLM determined whether to retain new relations or filter out redundant ones. This process resulted in the construction of a relation set <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M19" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><msub><mi>r</mi><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">l</mi></mrow></msub></mrow></mstyle></math></span>, which was incorporated into OpenIE prompts, instructing the LLM to extract entity-relation triples. The relations of extracted triples were strictly constrained to the relation set <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M20" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><msub><mi>r</mi><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">l</mi></mrow></msub></mrow></mstyle></math></span>.</p>
<p>Specifically, during the OpenIE, we instructed the LLM to extract attributes of food entities and dish entities (Textboxes S4 and S5 in <a href="#SAP1" class="usa-link">Multimedia Appendix 1</a>). For food entities, LLMs referenced their source passages to update attributes, including name, primary effects, consumption methods, indications, and contraindications. Similarly, for dish entities, attributes such as name, ingredient composition, cooking method, and dietary benefits were updated.</p>
<p>The automated construction of the MFH KG inevitably introduced noise, incompleteness, and factual inaccuracies, making absolute factual correctness unattainable. To mitigate these issues, we adopted UKG reasoning to measure the confidence of extracted triples and complete missing triples. Specifically, we first used unKR [<a href="#R30" class="usa-link" aria-describedby="R30">30</a>], an open-source Python library for UKG reasoning, to learn vector representations of entities and relations. Based on these representations, we applied the triple confidence measurement method [<a href="#R31" class="usa-link" aria-describedby="R31">31</a>], integrating heterogeneous evidences to measure confidence of each triple. For each given head entity <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M21" alttext="h" overflow="linebreak"><mi>h</mi></math></span> and relation <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M22" alttext="r" overflow="linebreak"><mi>r</mi></math></span>, candidate tail entities were enumerated to construct new triples absent from the current graph. The triples with confidence scores larger than 0.85 were added to the MFH KG.</p></section></section><section id="s2-3"><h3 class="pmc_sec_title">Retrieval-Augmented Generation</h3>
<section id="s2-3-1"><h4 class="pmc_sec_title">The Retrieval Strategy</h4>
<p>Given a user query <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M23" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mrow><mi mathvariant="normal">Q</mi></mrow></mrow></mstyle></math></span>, we first constructed a prompt (<a href="#B1" class="usa-link">Textbox 1</a>) to guide the LLM to extract a set of relevant keywords <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M24" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mrow><mi mathvariant="normal">E</mi></mrow></mrow></mstyle></math></span> from <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M25" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mrow><mi mathvariant="normal">Q</mi></mrow></mrow></mstyle></math></span>. We then encoded both the keywords and all entities from the MFH KG into dense vector representations, denoted as <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M26" alttext="H_{R}" overflow="linebreak"><msub><mrow><mi>H</mi></mrow><mrow><mi>R</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M27" alttext="H_{G}" overflow="linebreak"><msub><mrow><mi>H</mi></mrow><mrow><mi>G</mi></mrow></msub></math></span>, respectively. Considering that SBERT [<a href="#R32" class="usa-link" aria-describedby="R32">32</a>] has demonstrated superior performance over BERT [<a href="#R33" class="usa-link" aria-describedby="R33">33</a>] and RoBERTa [<a href="#R34" class="usa-link" aria-describedby="R34">34</a>] on semantic textual similarity tasks while maintaining lower computational overhead [<a href="#R35" class="usa-link" aria-describedby="R35">35</a>], we adopted SBERT for embedding both the extracted keywords and entities. Subsequently, we computed the cosine similarities between the embeddings of each keyword and all entities. For each keyword, we retrieved the top-10 most similar entities from the MFH KG based on cosine similarity scores. These candidate entities were then presented to the LLM, which was instructed to identify the matched entity (<a href="#B2" class="usa-link">Textbox 2</a>). The selected entity corresponding to each keyword constitutes the query entity set <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M28" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><msub><mi>E</mi><mrow><mi mathvariant="normal">Q</mi></mrow></msub><mo>=</mo><mrow><mo>{</mo><mrow><msub><mi>e</mi><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>e</mi><mrow><mn>2</mn></mrow></msub><mo>,</mo><mo>…</mo><mo> </mo><mo>,</mo><msub><mi>e</mi><mrow><mi>n</mi></mrow></msub></mrow><mo>}</mo></mrow></mrow></mstyle></math></span>, which serves as the starting point for subsequent reasoning path retrieval.</p>
<section class="bt xbox font-sm" id="B1"><h5 class="obj_head">Textbox 1. Prompt for keyword extraction.</h5>
<p>You are a medical knowledge assistant specialized in Traditional Chinese Medicine. Given a user query, extract a concise set of keywords that represent the user's core health-related intent and information needs. These keywords will be used to retrieve relevant knowledge from a medicine-food homology knowledge graph to answer the user query.</p>
<p>User query: {user_query}</p>
<p>Please return the output as a Python-style set of keywords:</p></section><section class="bt xbox font-sm" id="B2"><h5 class="obj_head">Textbox 2. Prompt for entity linking.</h5>
<p>Given a keyword and a list of candidate entities retrieved from the medicine food homology knowledge graph, select the entity that most accurately matches the keyword based on contextual meaning and traditional usage. If none of the entities are appropriate matches, return "None."</p>
<p>Keyword: {keyword}</p>
<p>Candidate entities: {candidate_entities}</p>
<p>Please return the output as a JSON list:</p></section><p>This study used path-based retrieval from the MFH KG rather than retrieving contextual subgraphs for linked entities. This is motivated by the advantages reasoning paths offer in generating MFH-based dietary recommendations. Specifically, reasoning paths provide structured knowledge by capturing direct relations between query entities and answer entities. For example, the reasoning path “(Insomnia and Vivid Dreams, Related Constitution, Qi-deficient Constitution); (Qi-deficient Constitution, Suitable Food, Codonopsis)<em>”</em> enables LLMs to clearly infer the relations between a user’s symptoms, their constitution, and the recommended food, thereby enhancing the interpretability. In contrast, subgraph-based retrieval in long-hop reasoning tasks often introduces irrelevant information [<a href="#R36" class="usa-link" aria-describedby="R36">36</a>]. This not only leads to significant computational overhead but also hinders LLMs from effectively extracting relevant information from the retrieval results.</p>
<p>Built upon the previous work [<a href="#R37" class="usa-link" aria-describedby="R37">37</a>], we first provided the LLM with a structured prompt (<a href="#B3" class="usa-link">Textbox 3</a>) listing all relations in <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M29" alttext="G" overflow="linebreak"><mi>G</mi></math></span>. The LLM then generated multiple relation paths <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M30" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mtext> </mtext><mrow><mo>{</mo><mrow><msubsup><mi>p</mi><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">l</mi></mrow><mrow><mn>1</mn></mrow></msubsup><mo>,</mo><mtext> </mtext><msubsup><mi>p</mi><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">l</mi></mrow><mrow><mn>2</mn></mrow></msubsup><mo>,</mo><mtext> </mtext><mo>…</mo><mo>,</mo><mtext> </mtext><msubsup><mi>p</mi><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">l</mi></mrow><mrow><mi>k</mi></mrow></msubsup></mrow><mo>}</mo></mrow></mrow></mstyle></math></span>, where each <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M31" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><msubsup><mi>p</mi><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">l</mi></mrow><mrow><mi>i</mi></mrow></msubsup></mrow></mstyle></math></span> is an ordered sequence of relations<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M32" alttext="\ \left \{r_{1},r_{2},\ldots ,r_{l}\right \}" overflow="linebreak"><mi></mi><mfenced open="{" close="}" separators="|"><mrow><msub><mrow><mi>r</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mrow><mi>r</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>,</mo><mo>…</mo><mo>,</mo><msub><mrow><mi>r</mi></mrow><mrow><mi>l</mi></mrow></msub></mrow></mfenced></math></span>. These relation paths serve as guides for retrieving knowledge essential to answering user queries. Given that the MFH KG contained considerably fewer relations compared with large-scale KGs such as FreeBase [<a href="#R38" class="usa-link" aria-describedby="R38">38</a>], we did not fine-tune the LLM for relation path generation. Instead, we used few-shot prompting to enhance the LLM’s ability to generate meaningful relation paths.</p>
<section class="bt xbox font-sm" id="B3"><h5 class="obj_head">Textbox 3. Prompt for relation paths generation.</h5>
<p>Task instructions:</p>
<ol class="list" style="list-style-type:decimal">
<li><p>Please generate multiple relation paths (sequences of relations) based on the existing relation types. Each relation path should contain relations that are connected through the same entity.</p></li>
<li><p>The purpose of the relation paths is to assist in retrieving relevant entities that can answer the user query.</p></li>
</ol>
<p>Relation types: {existing_relations}</p>
<p>User query: {user_query}</p>
<p>Query entities: {query_entities}</p>
<p>Few-shot examples: {examples}</p>
<p>Please strictly follow the task instructions and return only the required JSON list:</p></section><p>Subsequently, starting from each entity in the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M33" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><msub><mrow><mi mathvariant="normal">E</mi></mrow><mrow><mi mathvariant="normal">Q</mi></mrow></msub></mrow></mstyle></math></span>, we followed each generated relation path <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M34" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><msubsup><mi>p</mi><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">l</mi></mrow><mrow><mi>i</mi></mrow></msubsup></mrow></mstyle></math></span> to retrieve a set of reasoning paths <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M35" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><msub><mi>P</mi><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">a</mi><mo>​</mo></mrow></msub><mo>=</mo><mrow><mo>{</mo><mrow><msubsup><mi>p</mi><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">a</mi></mrow><mrow><mn>1</mn></mrow></msubsup><mo>,</mo><mtext> </mtext><msubsup><mi>p</mi><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">a</mi></mrow><mrow><mn>2</mn></mrow></msubsup><mo>,</mo><mtext> </mtext><mo>…</mo><mo>,</mo><mtext> </mtext><msubsup><mi>p</mi><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">a</mi></mrow><mrow><mi>m</mi></mrow></msubsup></mrow><mo>}</mo></mrow></mrow></mstyle></math></span> from <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M36" alttext="G" overflow="linebreak"><mi>G</mi></math></span>, where each reasoning path <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M37" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><msubsup><mi>p</mi><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">a</mi></mrow><mrow><mi>i</mi></mrow></msubsup><mo>=</mo><mrow><mo>{</mo><mrow><mrow><mo>(</mo><mrow><msub><mi>e</mi><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>r</mi><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mi>e</mi><mrow><mn>2</mn></mrow></msub></mrow><mo>)</mo></mrow><mo>,</mo><mrow><mo>(</mo><mrow><msub><mi>e</mi><mrow><mn>2</mn></mrow></msub><mo>,</mo><msub><mi>r</mi><mrow><mn>2</mn></mrow></msub><mo>,</mo><msub><mi>e</mi><mrow><mn>3</mn></mrow></msub></mrow><mo>)</mo></mrow><mo>,</mo><mo>…</mo><mo>,</mo><mrow><mo>(</mo><mrow><msub><mi>e</mi><mrow><mi>l</mi></mrow></msub><mo>,</mo><msub><mi>r</mi><mrow><mi>l</mi></mrow></msub><mo>,</mo><msub><mi>e</mi><mrow><mi>l</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow><mo>)</mo></mrow></mrow><mo>}</mo></mrow></mrow></mstyle></math></span> consists of multiple triples <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M38" alttext="\left (e_{j},r_{j},e_{j+1}\right)" overflow="linebreak"><mfenced separators="|"><mrow><msub><mrow><mi>e</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>,</mo><msub><mrow><mi>r</mi></mrow><mrow><mi>j</mi></mrow></msub><mo>,</mo><msub><mrow><mi>e</mi></mrow><mrow><mi>j</mi><mo>+</mo><mn>1</mn></mrow></msub></mrow></mfenced></math></span> and represents an instance of one of the relation paths.</p></section><section id="s2-3-2"><h4 class="pmc_sec_title">Retrieval Result Postprocessing</h4>
<p>In our initial experiment, we noticed that the reasoning paths in <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M39" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><msub><mi>P</mi><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">a</mi><mo>​</mo></mrow></msub></mrow></mstyle></math></span> contained redundant and irrelevant information. To enhance the quality of these reasoning paths, we computed a score for each reasoning path in <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M40" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><msub><mi>P</mi><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">a</mi><mo>​</mo></mrow></msub></mrow></mstyle></math></span> by considering both the entity importance and the triple confidence within the reasoning path. These scores enabled us to filter out less relevant reasoning paths.</p>
<p>The importance of each entity in the reasoning path was calculated using a weighted PageRank [<a href="#R39" class="usa-link" aria-describedby="R39">39</a>] algorithm. Traditional PageRank ignored the confidence of triples, which could lead to equivalent treatment of false and true relations, thus causing inaccuracies in entity importance. To overcome this limitation, we constructed a subgraph using entities from the <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M41" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><msub><mi>P</mi><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">a</mi><mo>​</mo></mrow></msub></mrow></mstyle></math></span> along with their one-hop neighbors and incorporated confidence scores of triples as edge weights in the PageRank computation. The weighted PageRank formula is defined as:</p>
<table class="disp-formula p" id="eqn1"><tr>
<td class="formula"><math id="M42" display="block" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mstyle displaystyle="true" scriptlevel="0"><mtable rowspacing="4pt" columnspacing="1em"><mtr><mtd><mrow><mi mathvariant="normal">P</mi><mi mathvariant="normal">R</mi></mrow><mrow><mo>(</mo><msub><mi>e</mi><mrow><mi>i</mi></mrow></msub><mo>)</mo></mrow><mo>=</mo><mrow><mo>(</mo><mrow><mn>1</mn><mo>−</mo><mi>d</mi></mrow><mo>)</mo></mrow><mo>⋅</mo><mfrac><mn>1</mn><mi>N</mi></mfrac><mo>+</mo><mi>d</mi><mo>⋅</mo><munder><mo>∑</mo><mrow><msub><mi>e</mi><mrow><mi>j</mi></mrow></msub><mo>⊂</mo><mi>I</mi><mi>n</mi><mrow><mo>(</mo><msub><mi>e</mi><mrow><mi>i</mi></mrow></msub><mo>)</mo></mrow></mrow></munder><mfrac><msub><mi>w</mi><mrow><mi>j</mi><mo stretchy="false">→</mo><mi>i</mi></mrow></msub><mrow><munder><mo>∑</mo><mi>k</mi></munder><msub><mi>w</mi><mrow><mi>j</mi><mo stretchy="false">→</mo><mi>k</mi></mrow></msub></mrow></mfrac><mo>⋅</mo><mrow><mi mathvariant="normal">P</mi><mi mathvariant="normal">R</mi></mrow><mrow><mo>(</mo><msub><mi>e</mi><mrow><mi>j</mi></mrow></msub><mo>)</mo></mrow></mtd></mtr></mtable></mstyle></mrow></mstyle></math></td>
<td class="label">(1)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M43" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mrow><mi mathvariant="normal">P</mi><mi mathvariant="normal">R</mi></mrow><mrow><mo>(</mo><msub><mi>e</mi><mrow><mi>i</mi></mrow></msub><mo>)</mo></mrow></mrow></mstyle></math></span> represents the weighted PageRank score of entity <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M44" alttext="e_{i}" overflow="linebreak"><msub><mrow><mi>e</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M45" alttext="d" overflow="linebreak"><mi>d</mi></math></span> is the damping factor, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M46" alttext="N" overflow="linebreak"><mi>N</mi></math></span> is the total number of entities in the graph. The term <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M47" alttext="w_{j\rightarrow i}" overflow="linebreak"><msub><mrow><mi>w</mi></mrow><mrow><mi>j</mi><mo>→</mo><mi>i</mi></mrow></msub></math></span> denotes the edge weight from entity <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M48" alttext="e_{i}" overflow="linebreak"><msub><mrow><mi>e</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> to <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M49" alttext="e_{j}" overflow="linebreak"><msub><mrow><mi>e</mi></mrow><mrow><mi>j</mi></mrow></msub></math></span>, derived from the confidence score of the corresponding triples. When multiple relations exist between <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M50" alttext="e_{i}" overflow="linebreak"><msub><mrow><mi>e</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M51" alttext="e_{j}" overflow="linebreak"><msub><mrow><mi>e</mi></mrow><mrow><mi>j</mi></mrow></msub></math></span> with different confidence scores, their influence was aggregated by averaging these scores. For scoring the reasoning paths, we integrated entity importance and triple confidence scores to formulate the following path-scoring function:</p>
<table class="disp-formula p" id="eqn2"><tr>
<td class="formula"><math id="M52" display="block" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mstyle displaystyle="true" scriptlevel="0"><mtable rowspacing="4pt" columnspacing="1em"><mtr><mtd><mrow><mi mathvariant="normal">P</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">h</mi><mi mathvariant="normal">S</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi></mrow><mrow><mo>(</mo><mi>P</mi><mo>)</mo></mrow><mo>=</mo><munderover><mo movablelimits="false">∏</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></munderover><msub><mi>w</mi><mrow><mi>i</mi><mo>−</mo><mn>1</mn><mo stretchy="false">→</mo><mi>i</mi></mrow></msub><mo>×</mo><mrow><mo>(</mo><mrow><mfrac><mn>1</mn><mi>n</mi></mfrac><munder><mo>∑</mo><mrow><msub><mi>e</mi><mrow><mi>j</mi></mrow></msub><mo>∈</mo><mi>P</mi></mrow></munder><mrow><mi mathvariant="normal">P</mi><mi mathvariant="normal">R</mi></mrow><mrow><mo>(</mo><msub><mi>e</mi><mrow><mi>j</mi></mrow></msub><mo>)</mo></mrow></mrow><mo>)</mo></mrow></mtd></mtr></mtable></mstyle></mrow></mstyle></math></td>
<td class="label">(2)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M53" alttext="w_{i-1\rightarrow i}" overflow="linebreak"><msub><mrow><mi>w</mi></mrow><mrow><mi>i</mi><mo>-</mo><mn>1</mn><mo>→</mo><mi>i</mi></mrow></msub></math></span> represents the confidence score of the relation from entity <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M54" alttext="e_{i}" overflow="linebreak"><msub><mrow><mi>e</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> to <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M55" alttext="e_{j}" overflow="linebreak"><msub><mrow><mi>e</mi></mrow><mrow><mi>j</mi></mrow></msub></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M56" alttext="n" overflow="linebreak"><mi>n</mi></math></span> denotes the total number of entities in the path, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M57" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mrow><mi mathvariant="normal">P</mi><mi mathvariant="normal">R</mi></mrow><mrow><mo>(</mo><msub><mi>e</mi><mrow><mi>j</mi></mrow></msub><mo>)</mo></mrow></mrow></mstyle></math></span> is the weighted PageRank score of entity <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M58" alttext="e_{j}" overflow="linebreak"><msub><mrow><mi>e</mi></mrow><mrow><mi>j</mi></mrow></msub></math></span> within the reasoning path. This scoring methodology captured both the confidence of triples within the reasoning path and the importance of the entities involved, striking a balance between path reliability and informational importance. Finally, we ranked the reasoning paths based on their scores and retained only the top-<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M59" alttext="k" overflow="linebreak"><mi>k</mi></math></span> reasoning paths for subsequent inference.</p></section><section id="s2-3-3"><h4 class="pmc_sec_title">Answer Generation</h4>
<p>We used prompt engineering to improve the accuracy and relevance of generated answers of LLM. Prompt engineering involves constructing carefully designed prompts to guide LLMs toward generating high-quality and task-specific responses. Specifically, we constructed a prompt template (<a href="#B4" class="usa-link">Textbox 4</a>) comprising four key components: (1) task specification, (2) user query, (3) retrieved reasoning paths, and (4) attributes of food and dish entities. This structured prompt facilitates the integration of external knowledge, improves the LLMs’ comprehension of MFH principles, and supports the generation of more accurate responses.</p>
<section class="bt xbox font-sm" id="B4"><h5 class="obj_head">Textbox 4. Prompt for answer generation.</h5>
<p>You are a professional Traditional Chinese Medicine doctor, providing dietary recommendations using the concept of “medicine food homology.” Given a user query and the relevant reasoning paths and entity attributes, generate an answer.</p>
<p>Task instructions:</p>
<ol class="list" style="list-style-type:decimal">
<li><p>Please provide a detailed, accurate, and concise response based on the information provided.</p></li>
<li><p>The response should be well-structured and ensure logical consistency with the given reasoning paths and entity attributes.</p></li>
<li><p>Incorporate medicine food homology principles in your recommendations.</p></li>
<li><p>Output format: {“answer”: “(Your answer here.)”}. The answer must be written in Chinese.</p></li>
</ol>
<p>User query: {user_query}</p>
<p>Reasoning paths: {reasoning_paths}</p>
<p>Entity attributes: {entity_attributes}</p>
<p>Please strictly follow the task instructions and return only the required JSON format:</p></section></section></section><section id="s2-4"><h3 class="pmc_sec_title">LLM Selection</h3>
<p>To identify the most suitable LLM for MFH dietary recommendation tasks, we evaluated several mainstream LLMs, including GPT-4 [<a href="#R40" class="usa-link" aria-describedby="R40">40</a>], LLaMA2-Chat-7B [<a href="#R41" class="usa-link" aria-describedby="R41">41</a>], Qwen2.5-7B [<a href="#R42" class="usa-link" aria-describedby="R42">42</a>], and DeepSeek-R1 [<a href="#R43" class="usa-link" aria-describedby="R43">43</a>]. Additionally, we considered the influence of model size, incorporating both large-scale models (GPT-4 and DeepSeek-R1) and 7B-parameter models (LLaMA2-Chat-7B and Qwen2.5-7B). During the experiment, we implemented the same prompt set across all models to ensure comparative evaluation under identical input conditions. All models were integrated with our proposed framework for dietary recommendation generation based on the MFH principles.</p></section><section id="s2-5"><h3 class="pmc_sec_title">Evaluation Metrics</h3>
<section id="s2-5-1"><h4 class="pmc_sec_title">Automated Evaluation</h4>
<p>To quantitatively assess the effectiveness and accuracy of the model’s dietary recommendations, we used 2 evaluation metrics: <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M60" alttext="\mathrm{H}\mathrm{i}\mathrm{t}\mathrm{s}@1" overflow="linebreak"><mi mathvariant="normal">H</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">s</mi><mo>@</mo><mn>1</mn></math></span> and <em>F</em><sub>1</sub>-score. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M61" alttext="\mathrm{H}\mathrm{i}\mathrm{t}\mathrm{s}@1" overflow="linebreak"><mi mathvariant="normal">H</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">s</mi><mo>@</mo><mn>1</mn></math></span> measures the proportion of queries where the top-ranked predicted recommendation matches a ground truth answer. This metric evaluates the model’s ability to prioritize the most relevant dietary recommendation. <em>F</em><sub>1</sub>-score captures the harmonic mean of precision (P) and recall (R), offering a balanced evaluation of both aspects. This metric is particularly appropriate in scenarios where multiple valid recommendations may exist for a single query. It is defined as follows:</p>
<table class="disp-formula p" id="eqn3"><tr>
<td class="formula"><math id="M62" display="block" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mtable rowspacing="4pt" columnspacing="1em"><mtr><mtd><msub><mi>F</mi><mn>1</mn></msub><mo>=</mo><mn>2</mn><mo>×</mo><mfrac><mrow><mi>P</mi><mo>×</mo><mi>R</mi></mrow><mrow><mi>P</mi><mo>+</mo><mi>R</mi></mrow></mfrac></mtd></mtr></mtable></mrow></mstyle></math></td>
<td class="label">(3)</td>
</tr></table>
<p>where precision and recall are defined as:</p>
<table class="disp-formula p" id="eqn4"><tr>
<td class="formula"><math id="M63" display="block" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mtable rowspacing="4pt" columnspacing="1em"><mtr><mtd><mi>P</mi><mo>=</mo><mfrac><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">P</mi></mrow><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">P</mi><mo>+</mo><mi mathvariant="normal">F</mi><mi mathvariant="normal">P</mi></mrow></mfrac><mo>,</mo><mtext> </mtext><mi>R</mi><mo>=</mo><mfrac><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">P</mi></mrow><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">P</mi><mo>+</mo><mi mathvariant="normal">F</mi><mi mathvariant="normal">N</mi></mrow></mfrac></mtd></mtr></mtable></mrow></mstyle></math></td>
<td class="label">(4)</td>
</tr></table>
<p>In this context, <em><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M64" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mrow><mi mathvariant="normal">T</mi><mi mathvariant="normal">P</mi></mrow></mrow></mstyle></math></span></em> (true positives) represents the number of dietary recommendations correctly suggested, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M65" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mrow><mi mathvariant="normal">F</mi><mi mathvariant="normal">P</mi></mrow></mrow></mstyle></math></span> (false positives) indicates inappropriate dietary recommendations incorrectly suggested, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M66" overflow="linebreak"><mstyle displaystyle="true" scriptlevel="0"><mrow><mrow><mi mathvariant="normal">F</mi><mi mathvariant="normal">N</mi></mrow></mrow></mstyle></math></span> (false negatives) refers to appropriate dietary recommendations that the model failed to suggest.</p></section><section id="s2-5-2"><h4 class="pmc_sec_title">Human Evaluation</h4>
<p>While automated evaluation methods are widely applied in natural language processing tasks, they have certain limitations in the context of dietary recommendations. Automated metrics struggle to assess the effectiveness of recommendations, specifically whether the suggested food and their combinations align with MFH principles. Moreover, user acceptability and explainability of the generated dietary recommendations are difficult to accurately assess using automated methods. Therefore, we incorporated human evaluation, integrating subjective ratings from both experts and general users to provide a more comprehensive assessment of the generated dietary recommendations.</p>
<p>Specifically, we assessed the generated dietary plans based on the following five evaluation criteria: (1) rationality, (2) explainability, (3) user acceptability, (4) personalization, and (5) consistency. Rationality assesses whether the recommended dietary plan adhered to both TCM principles and modern nutritional standards. Explainability evaluates whether the recommendations included clear TCM-based justifications. A higher score indicates that the system provided reasonable and understandable explanations rather than simply outputting ingredient combinations. User acceptability measures how well the recommendations aligned with users’ subjective preferences, including taste, cultural adaptability, and feasibility. Personalization assesses whether the system provided tailored recommendations based on an individual’s constitution and health conditions. Consistency examines whether the model produced stable recommendations under similar inputs.</p>
<p>Among these evaluation criteria, rationality, explainability, user acceptability, and personalization were assessed using a 5-point Likert scale (1‐5, where 1 represented the lowest score and 5 represented the highest). Consistency was evaluated using a binary judgment (<em>Yes</em><em>=</em><em>5</em><em> points and </em><em>No</em><em>=</em><em>0</em><em> points</em>).</p>
<p>To conduct the evaluation, we recruited 5 experts with backgrounds in TCM or nutrition, along with 20 general users. Participants followed a structured human evaluation guideline to rate the generated dietary recommendations. The evaluation was conducted using a survey system, where assessors were provided with the user queries along with the corresponding dietary recommendations generated by LLMs. Finally, we computed the mean scores for each evaluation criterion.</p></section></section><section id="s2-6"><h3 class="pmc_sec_title">Ethical Considerations</h3>
<p>Institutional review board approval was not required for this study, as it did not involve direct research with human participants. All corpora used, such as research papers and TCM formulations, were publicly available [<a href="#R44" class="usa-link" aria-describedby="R44">44-46</a>], and no identifiable private or sensitive information was accessed or recorded. Annotators involved in the human evaluation participated voluntarily, provided informed consent prior to participation, and were informed of their right to withdraw at any time. No compensation was provided, and no private information was collected.</p></section></section><section id="s3"><h2 class="pmc_sec_title">Results</h2>
<p>We constructed an MFH KG comprising 24,984 entities, 22 relations, and 29,292 triples. To evaluate the LLMs’ performance in dietary recommendation tasks, we developed a specialized question answering dataset consisting of 2000 queries focused on MFH principles. This dataset was designed to ensure balanced coverage across diverse health conditions and TCM constitutional types.</p>
<p><a href="#T2" class="usa-link">Table 2</a> shows a comparative analysis of LLMs with or with no augmentation by the MFH KG on dietary recommendation tasks. Overall, the integration of the MFH KG substantially improved model performance across all evaluated metrics, with an average increase of 14.5% in Hits@1 and 8.7% in <em>F</em><sub>1</sub>-score, respectively. Among the baseline LLMs, DeepSeek-R1 achieved the best performance (Hits@1: 70.4%, <em>F</em><sub>1</sub>-score: 64.1%), followed by GPT-4 (Hits@1: 68.6%, <em>F</em><sub>1</sub>-score: 55.8%), Qwen2.5-7B (Hits@1: 58.7%, <em>F</em><sub>1</sub>-score: 53.2%), and LLaMA2-Chat-7B (Hits@1: 54.1%, <em>F</em><sub>1</sub>-score: 49.3%). The performance ranking remained consistent after MFH KG augmentation, with DeepSeek-R1 outperforming the others (Hits@1: 84.2%, <em>F</em><sub>1</sub>-score: 71.5%). Due to its superior performance in the above automated evaluation, DeepSeek-R1 was selected as the LLM for all subsequent experiments.</p>
<section class="tw xbox font-sm" id="T2"><h3 class="obj_head">Table 2. The performance comparison between large language models and medicine food homology knowledge graph–augmented large language models on dietary recommendation tasks.</h3>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th valign="bottom" rowspan="1" colspan="1">Model</th>
<th valign="bottom" rowspan="1" colspan="1">Hits@1</th>
<th valign="bottom" rowspan="1" colspan="1">
<em>F</em><sub>1</sub>-score</th>
</tr></thead>
<tbody>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">LLMs<a href="#T2_FN1" class="usa-link"><sup>a</sup></a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1"> GPT-4</td>
<td align="left" valign="top" rowspan="1" colspan="1">68.6</td>
<td align="left" valign="top" rowspan="1" colspan="1">55.8</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1"> LLaMA2-Chat-7B</td>
<td align="left" valign="top" rowspan="1" colspan="1">54.1</td>
<td align="left" valign="top" rowspan="1" colspan="1">49.3</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1"> Qwen2.5-7B</td>
<td align="left" valign="top" rowspan="1" colspan="1">58.7</td>
<td align="left" valign="top" rowspan="1" colspan="1">53.2</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1"> DeepSeek-R1</td>
<td align="left" valign="top" rowspan="1" colspan="1">70.4</td>
<td align="left" valign="top" rowspan="1" colspan="1">64.1</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">MFH<a href="#T2_FN2" class="usa-link"><sup>b</sup></a> KG<a href="#T2_FN3" class="usa-link"><sup>c</sup></a>+ LLMs</td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
<td align="left" valign="top" rowspan="1" colspan="1"></td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1"> GPT-4</td>
<td align="left" valign="top" rowspan="1" colspan="1">82.6</td>
<td align="left" valign="top" rowspan="1" colspan="1">69.4</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1"> LLaMA2-Chat-7B</td>
<td align="left" valign="top" rowspan="1" colspan="1">70.3</td>
<td align="left" valign="top" rowspan="1" colspan="1">55.6</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1"> Qwen2.5-7B</td>
<td align="left" valign="top" rowspan="1" colspan="1">72.8</td>
<td align="left" valign="top" rowspan="1" colspan="1">60.6</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1"> DeepSeek-R1</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>84.2</em>
<a href="#T2_FN4" class="usa-link">
<sup>
<em>d</em>
</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>71.5</em>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="T2_FN1">
<sup>a</sup><p class="display-inline">LLM: large language model.</p>
</div>
<div class="fn" id="T2_FN2">
<sup>b</sup><p class="display-inline">MFH: medicine food homology.</p>
</div>
<div class="fn" id="T2_FN3">
<sup>c</sup><p class="display-inline">KG: knowledge graph.</p>
</div>
<div class="fn" id="T2_FN4">
<sup>d</sup><p class="display-inline">Values in italics indicate the best performance for each metric.</p>
</div>
</div></section><p>In human evaluation, we conducted a comparative assessment between LLMs and MFH KG–augmented LLMs across 5 key metrics. As shown in <a href="#T3" class="usa-link">Table 3</a>, the MFH KG–augmented DeepSeek-R1 consistently outperformed the baseline DeepSeek-R1 model across all evaluation criteria. The MFH KG–augmented model achieved substantially higher scores in rationality (4.1 vs 3.2), explainability (4.3 vs 2.9), user acceptability (4.4 vs 3.6), personalization (4.2 vs 3.3), and consistency (4.6 vs 3.4). These results provide compelling evidence that augmenting DeepSeek-R1 with the specialized MFH KG significantly improves the quality of dietary recommendations based on MFH principles.</p>
<section class="tw xbox font-sm" id="T3"><h3 class="obj_head">Table 3. The performance comparison between DeepSeek-R1 and medicine food homology knowledge graph–augmented DeepSeek-R1 by human evaluation.</h3>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th valign="top" rowspan="1" colspan="1">Models</th>
<th valign="top" rowspan="1" colspan="1">Rationality</th>
<th valign="top" rowspan="1" colspan="1">Explainability</th>
<th valign="top" rowspan="1" colspan="1">User acceptability</th>
<th valign="top" rowspan="1" colspan="1">Personalization</th>
<th valign="top" rowspan="1" colspan="1">Consistency</th>
</tr></thead>
<tbody>
<tr>
<td valign="top" rowspan="1" colspan="1">DeepSeek-R1</td>
<td valign="top" rowspan="1" colspan="1">3.2</td>
<td valign="top" rowspan="1" colspan="1">2.9</td>
<td valign="top" rowspan="1" colspan="1">3.6</td>
<td valign="top" rowspan="1" colspan="1">3.3</td>
<td valign="top" rowspan="1" colspan="1">3.4</td>
</tr>
<tr>
<td valign="top" rowspan="1" colspan="1">MFH<a href="#T3_FN1" class="usa-link"><sup>a</sup></a> KG<a href="#T3_FN2" class="usa-link"><sup>b</sup></a> + DeepSeek−R1</td>
<td valign="top" rowspan="1" colspan="1">4.1</td>
<td valign="top" rowspan="1" colspan="1">4.3</td>
<td valign="top" rowspan="1" colspan="1">4.4</td>
<td valign="top" rowspan="1" colspan="1">4.2</td>
<td valign="top" rowspan="1" colspan="1">4.6</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="T3_FN1">
<sup>a</sup><p class="display-inline">MFH: medicine food homology.</p>
</div>
<div class="fn" id="T3_FN2">
<sup>b</sup><p class="display-inline">KG: knowledge graph.</p>
</div>
</div></section><p>We analyzed the impact of parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M67" alttext="k" overflow="linebreak"><mi>k</mi></math></span>, which controls the number of retrieved reasoning paths in the RAG module. The evaluation across <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M68" alttext="k" overflow="linebreak"><mi>k</mi></math></span> values [1, 3, 5, 10, 20, and 30] is shown in <a href="#F3" class="usa-link">Figure 3</a>. The <em>F</em><sub>1</sub>-scores indicate that small increases in <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M69" alttext="k" overflow="linebreak"><mi>k</mi></math></span> initially improved performance, but after reaching an optimal point (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M70" alttext="k" overflow="linebreak"><mi>k</mi></math></span>=10), further increases led to gradual accuracy decline. This pattern demonstrates the effectiveness of our reasoning path postprocessing, as it successfully prioritizes the most relevant reasoning paths while filtering out potentially misleading or irrelevant information.</p>
<figure class="fig xbox font-sm" id="F3"><h3 class="obj_head">Figure 3. Different knowledge retrieval settings and the corresponding <em>F</em><sub>1</sub>-scores.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370266_medinform-v13-e75279-g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9ee/12370266/cab9a30ccbd5/medinform-v13-e75279-g003.jpg" loading="lazy" height="444" width="765" alt="Figure 3."></a></p>
<div class="p text-right font-secondary"><a href="figure/F3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>To study the impact of the damping factor <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M71" alttext="d" overflow="linebreak"><mi>d</mi></math></span>, where 1-<em>d</em> is the probability of randomly jumping to other nodes, we varied <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M72" alttext="d" overflow="linebreak"><mi>d</mi></math></span> within the range of [0.1, 0.9]. As shown in <a href="#F4" class="usa-link">Figure 4</a>, the performance of Yaoshi-RAG initially improved with increasing <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M73" alttext="d" overflow="linebreak"><mi>d</mi></math></span>, reached the peak (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M74" alttext="d" overflow="linebreak"><mi>d</mi></math></span>=0.8), and subsequently declined. This trend aligns with the commonly adopted value (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M75" alttext="d" overflow="linebreak"><mi>d</mi></math></span>=0.85) proposed by Page et al [<a href="#R47" class="usa-link" aria-describedby="R47">47</a>] and is consistent with previous findings [<a href="#R48" class="usa-link" aria-describedby="R48">48</a>,<a href="#R49" class="usa-link" aria-describedby="R49">49</a>]. Specifically, when <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M76" alttext="d" overflow="linebreak"><mi>d</mi></math></span> is too small, the resulting PageRank distribution is dominated by random jumps, which weakens the influence of the graph structure and yields less meaningful entity rankings. Conversely, when <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M77" alttext="d" overflow="linebreak"><mi>d</mi></math></span> approaches 1, the distribution becomes nearly uniform, and the convergence of the power iteration method slows significantly, posing computational challenges.</p>
<figure class="fig xbox font-sm" id="F4"><h3 class="obj_head">Figure 4. Different damping factor settings and the corresponding <em>F</em><sub>1</sub>-scores.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370266_medinform-v13-e75279-g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f9ee/12370266/170fd783f555/medinform-v13-e75279-g004.jpg" loading="lazy" height="495" width="798" alt="Figure 4."></a></p>
<div class="p text-right font-secondary"><a href="figure/F4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>We evaluated the effectiveness of Yaoshi-RAG’s retrieval strategy against multiple baselines. For sparse retrieval, we used BM25 [<a href="#R50" class="usa-link" aria-describedby="R50">50</a>] and followed DecAF [<a href="#R51" class="usa-link" aria-describedby="R51">51</a>] by transforming 1-hop subgraphs of topic entities into text. In KG embedding–based methods (TransE [<a href="#R52" class="usa-link" aria-describedby="R52">52</a>] RAG and RotatE [<a href="#R53" class="usa-link" aria-describedby="R53">53</a>] RAG), we ranked triples by embedding similarity. We also extracted the 2-hop subgraph to prioritize semantically relevant triples from query entities’ neighborhood. All baseline methods retained the top 30 triples as retrieval results. <a href="#T4" class="usa-link">Table 4</a> shows the performance comparison of different retrieval strategies. Yaoshi-RAG achieved superior performance with the highest Hits@1 (84.2%) and <em>F</em><sub>1</sub>-score (71.5%).</p>
<section class="tw xbox font-sm" id="T4"><h3 class="obj_head">Table 4. The performance comparison of retrieval strategies.</h3>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th valign="bottom" rowspan="1" colspan="1">Retrieval strategies</th>
<th valign="bottom" rowspan="1" colspan="1">Hits@1</th>
<th valign="bottom" rowspan="1" colspan="1">
<em>F</em><sub>1</sub>-score</th>
</tr></thead>
<tbody>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">BM25 (2009)</td>
<td align="left" valign="top" rowspan="1" colspan="1">72.8</td>
<td align="left" valign="top" rowspan="1" colspan="1">65.3</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">TransE RAG (2013)</td>
<td align="left" valign="top" rowspan="1" colspan="1">82.6</td>
<td align="left" valign="top" rowspan="1" colspan="1">69.9</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">RotatE RAG (2019)</td>
<td align="left" valign="top" rowspan="1" colspan="1">81.9</td>
<td align="left" valign="top" rowspan="1" colspan="1">68.5</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">2-Hop Subgraph</td>
<td align="left" valign="top" rowspan="1" colspan="1">75.4</td>
<td align="left" valign="top" rowspan="1" colspan="1">67.2</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Yaoshi-RAG (Ours)</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>84.2</em>
<a href="#T4_FN1" class="usa-link">
<sup>
<em>a</em>
</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>71.5</em>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="T4_FN1">
<sup>a</sup><p class="display-inline">Values in italics indicate the best performance for each metric.</p>
</div></div></section><p>To evaluate the impact of key modules in Yaoshi-RAG, we compared three variants: (1) a variant replacing the UKG with a deterministic KG, in which all triples were assumed to be fully reliable (ie, confidence scores were set to 1); (2) a variant removing the postprocessing of retrieval results, in which all retrieved reasoning paths from the UKG were directly input into the LLM without ranking or filtering; and (3) a variant randomly generating relation paths to guide the retrieval of reasoning paths.</p>
<p>As shown in <a href="#T5" class="usa-link">Table 5</a>, using a deterministic KG leads to a performance decline, highlighting the importance of incorporating uncertainty information derived from the UKG. Performance degrades further when postprocessing is removed, due to the inclusion of noisy and irrelevant reasoning paths. This confirms the effectiveness of leveraging weighted PageRank and triple confidence scores in identifying relevant reasoning paths. The variant using randomly generated relation paths for retrieval performs the worst among all settings, emphasizing the necessity of a retrieval strategy that generates faithful relation paths.</p>
<section class="tw xbox font-sm" id="T5"><h3 class="obj_head">Table 5. Ablation studies of Yaoshi-RAG.</h3>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th valign="bottom" rowspan="1" colspan="1">Model</th>
<th valign="bottom" rowspan="1" colspan="1">Hits@1</th>
<th valign="bottom" rowspan="1" colspan="1">
<em>F</em><sub>1</sub>-score</th>
</tr></thead>
<tbody>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">Yaoshi-RAG (Ours)</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>84.2</em>
<a href="#T5_FN2" class="usa-link">
<sup>
<em>a</em>
</sup>
</a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">
<em>71.5</em>
</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">w/o UKG<a href="#T5_FN1" class="usa-link"><sup>b</sup></a>
</td>
<td align="left" valign="top" rowspan="1" colspan="1">80.7</td>
<td align="left" valign="top" rowspan="1" colspan="1">67.9</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">w/o post-processing</td>
<td align="left" valign="top" rowspan="1" colspan="1">75.3</td>
<td align="left" valign="top" rowspan="1" colspan="1">66.2</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">w/ random relation paths</td>
<td align="left" valign="top" rowspan="1" colspan="1">65.4</td>
<td align="left" valign="top" rowspan="1" colspan="1">54.2</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/T5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="T5_FN2">
<sup>a</sup><p class="display-inline">Values in italics indicate the best performance for each metric.</p>
</div>
<div class="fn" id="T5_FN1">
<sup>b</sup><p class="display-inline">UKG: uncertain knowledge graph.</p>
</div>
</div></section><p><a href="#B5" class="usa-link">Textbox 5</a> demonstrates an actual case of a dietary question and the answer generated by our proposed framework. The framework delivers a comprehensive response to a patient presenting peripheral heat sensations and sleep disturbances. By leveraging reasoning paths derived from the MFH KG, the framework establishes clear relations between reported symptoms, underlying constitutional patterns, and appropriate dietary recommendations. The framework also enriches its recommendations by incorporating detailed food entities’ attributes from the KG into the LLM, thereby enhancing the specificity of guidance regarding consumption methods and other relevant details. This case demonstrates how Yaoshi-RAG effectively translates MFH principles into clear and practical guidance, helping patients better understand the recommendations and adhere to the suggested treatments.</p>
<section class="bt xbox font-sm" id="B5"><h3 class="obj_head">Textbox 5. An actual case of a dietary question and the answer generated by our proposed framework.</h3>
<p>User query:</p>
<p>Lately my hands and feet keep feeling hot, and I’m having trouble sleeping with lots of dreams. Are there any food you’d recommend for this?</p>
<p>我最近手足心热、失眠多梦，请给我推荐一些食材。 Top-3 reasoning paths:</p>
<ol class="list" style="list-style-type:decimal">
<li><p>(Heat in Hands and Feet, Related Constitution, Yin-deficient Constitution) → (Yin-deficient Constitution, Suitable Food, Lily Bulb)</p></li>
<li><p>(Heat in Hands and Feet, Related Constitution, Yin-deficient Constitution) → (Yin-deficient Constitution, Suitable Food, White Fungus)</p></li>
<li><p>(Insomnia and Vivid Dreams, Influencing Factor, Disharmony between Heart and Kidney) → (Disharmony between Heart and Kidney, Suitable Food, Mulberry)</p></li>
</ol>
<ol class="list" style="list-style-type:decimal">
<li><p>(手足心热, 关联体质, 阴虚质) → (阴虚质, 适宜食物, 百合)</p></li>
<li><p>(手足心热, 关联体质, 阴虚质) → (阴虚质, 适宜食物, 银耳)</p></li>
<li><p>(失眠多梦, 影响因素, 心肾不交) → (心肾不交, 适宜食物, 桑葚)</p></li>
</ol>
<p>Attributes of related food entities:</p>
<p>[Name], [Primary Effects], [Consumption Methods], [Indications], [Contraindications]</p>
<p>Generated answer:</p>
<p>Based on the symptoms you described—Heat in Hands and Feet, along with Insomnia and Vivid Dreams—the following analysis and dietary recommendations are provided according to Traditional Chinese Medicine (TCM) principles:</p>
<p>TCM syndrome differentiation analysisHeat in Hands and Feet is a typical manifestation of “Five-Center Heat” syndrome. When accompanied by Insomnia and Vivid Dreams, the condition may be attributed to three primary constitution types: (1) Yin Deficiency with Excessive Fire: Insufficient Yin fluids in the body lead to internal heat, disturbing the Shen (spirit) and causing insomnia. (2) Disharmony between Heart and Kidney: The inability of Kidney Water to balance Heart Fire results in excessive Heart Fire, affecting sleep quality. (3) Liver Blood Deficiency: A deficiency in Yin and Blood leads to an unsettled spirit, causing frequent dreams and light sleep. These syndromes often intertwine, requiring dietary adjustments to nourish Yin, reduce internal heat, and harmonize the Heart and Kidney.Recommended food and recipes</p>
<ul class="list" style="list-style-type:disc">
<li><p>Lily, lotus seed, and white fungus soup: Efficacy: Lily calms the mind, Tremella nourishes Yin and moistens dryness, and lotus seed strengthens the kidney and astringes essence. Together, they create a “Metal-Water Mutual Generation” effect. Preparation: Soak 15 grams of Tremella in cold water until fully expanded, then tear it into small pieces. Add 30 grams of fresh lily and 20 grams of decored lotus seeds to 800 mL of water and simmer for 1 hour. Add rock sugar before serving. Best time to consume: Between 3 and 5 PM (Shen Hour), when the bladder meridian is active, enhancing Yin nourishment.</p></li>
<li><p>Mulberry and black sesame paste: Efficacy: Mulberries nourish the liver and kidneys, while black sesame replenishes blood and moisturizes dryness, making them an ideal pairing for dual liver-kidney nourishment. Preparation: Lightly toast 50 grams of black sesame over low heat and then blend it with 20 grams of dried mulberries and 30 grams of glutinous rice with 600 mL of water into a smooth paste. Best time to consume: About 2 hours before bedtime for a calming effect.</p></li>
</ul>
<p>Precautions and lifestyle adjustments</p>
<ul class="list" style="list-style-type:disc">
<li><p>Dietary restrictions: Avoid spicy and heat-inducing food (eg, chili and Sichuan pepper), warming tonics (eg, lamb and longan), and stimulants such as strong tea and coffee.</p></li>
<li><p>Lifestyle recommendations: Prepare for sleep before 9 and 11 PM (Hai Hour). Practice the “Shaking Head and Wagging Tail to Reduce Heart Fire” movement from the Eight-Section Brocade (Ba Duan Jin). Engage in calligraphy or meditation to calm the mind.</p></li>
</ul>
<p>The above plan should be followed continuously for at least 2 weeks to see effects. If diarrhea occurs, discontinue the use of Poria. For patients with diabetes, it is recommended to substitute rock sugar with xylitol. If symptoms persist, seek medical attention promptly for tongue and pulse diagnosis.</p>
<p>根据您描述的手足心热、失眠多梦症状，结合中医辨证及食材特性分析如下:</p>
<p>中医辨证分析</p>
<p>手足心热属于“五心烦热”的典型表现，伴随失眠多梦可综合辨证为三类体质问题：其一为阴虚火旺，因体内阴液不足导致虚火内生，上扰心神引发失眠；其二属心肾不交，肾水无法上济心火，致使心火亢盛影响睡眠质量；其三与肝血不足相关，阴血亏虚致使魂不守舍，出现多梦易醒症状。这三种证型常相互交织，需通过滋阴降火、交通心肾的食疗调理。</p>
<p>推荐食材与烹饪方案</p>
<ul class="list" style="list-style-type:disc">
<li><p>百合莲子银耳羹：功效配伍：百合清心安神，银耳滋阴润燥，莲子固肾涩精，三者协同形成“金水相生”之效。制作方法：取银耳15克冷水泡发后撕小朵，配鲜百合30克、去芯莲子20克，加水800毫升炖煮1小时，起锅前加冰糖调味。建议申时（下午3-5点）食用，此时膀胱经当令，可增强滋阴效果。</p></li>
<li><p>桑葚黑芝麻糊：功效配伍：桑葚滋补肝肾之阴，黑芝麻养血润燥，两者搭配实现肝肾同补。制作方法：将文火炒香的50克黑芝麻与20克桑葚干、30克糯米混合，加600毫升水用破壁机打成米糊。适宜睡前2小时温服，有助宁神助眠。</p></li>
</ul>
<p>注意事项与生活调摄</p>
<ul class="list" style="list-style-type:disc">
<li><p>饮食禁忌：避免辛辣助火之物（如辣椒、花椒）、温热补品（如羊肉、桂圆）及咖啡浓茶等兴奋性饮品。</p></li>
<li><p>生活建议：保证亥时（21-23点）前准备入睡，日常练习八段锦“摇头摆尾去心火”招式，并通过书法、冥想安神定志。</p></li>
</ul>
<p>以上方案需连续食用2周以上见效，若出现腹泻可暂停茯苓，糖尿病患者建议以木糖醇替代冰糖。症状持续者应及时就医进行舌脉诊察。</p></section></section><section id="s4"><h2 class="pmc_sec_title">Discussion</h2>
<section id="s4-1"><h3 class="pmc_sec_title">Principal Findings</h3>
<p>Evaluation results indicate that our framework improved the performance of all evaluated LLMs, showcasing its robust adaptability. High-performing models, such as DeepSeek-R1, exhibited marked improvements when enhanced by the MFH KG. This suggests that the MFH KG offers valuable context about ingredient relations, therapeutic properties, and dietary principles, which might not be fully captured during the models’ pretraining. Remarkably, even LLMs with fewer parameters showed performance improvement when integrated with the MFH KG. These results highlight the effectiveness of the proposed framework, which combines the contextual reasoning capabilities of LLMs with the rich and structured knowledge of KGs, making it a powerful tool for MFH-based dietary recommendations.</p>
<p>The human evaluation results revealed that the KG-augmented LLM demonstrated comprehensive improvements. The augmented LLM exhibited improved rationality through the relations between symptoms and dietary recommendations provided by the MFH KG. It also achieved superior explainability by integrating theoretical reasoning paths. Furthermore, the framework demonstrated increased user acceptability with more clinically applicable suggestions. The augmented LLM also maintained greater consistency across varying inputs. These findings collectively indicate that RAG can effectively enhance LLMs’ capabilities with domain-specific knowledge.</p>
<p>The parameter analysis and ablation studies validate the effectiveness of our postprocessing strategy applied to the retrieved reasoning paths. Retrieving more information does not necessarily lead to performance improvement and may instead introduce irrelevant or noisy information [<a href="#R54" class="usa-link" aria-describedby="R54">54</a>], thereby degrading the quality of recommendations. By incorporating triple confidence scores from UKG reasoning, the framework prioritizes informative reasoning paths for dietary recommendations. The performance decline beyond <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math id="M78" alttext="k" overflow="linebreak"><mi>k</mi></math></span>=10 indicates that confidence-based ranking effectively distinguishes informative reasoning paths from less relevant ones. This underscores the importance of uncertainty modeling in TCM knowledge representation, particularly given the heterogeneous quality of evidence. By measuring confidence scores, the inherent uncertainty in MFH principles can be better managed, prioritizing reliable relations while mitigating the influence of less trustworthy ones.</p></section><section id="s4-2"><h3 class="pmc_sec_title">Limitations</h3>
<p>Several limitations of this study should be acknowledged. The MFH KG was constructed using OpenIE, which may introduce inaccuracies and cannot cover all knowledge on MFH principles. Besides, we did not fine-tune the LLMs specifically for relation path generation, which may have constrained the retrieval effectiveness of our framework, particularly for complex cases involving multiple symptoms or conditions. Addressing these limitations in the future would likely further enhance the LLMs’ clinical uses and recommendation quality.</p></section><section id="s4-3"><h3 class="pmc_sec_title">Conclusions</h3>
<p>This study shows Yaoshi-RAG, a new framework that enhances LLMs’ capabilities in generating MFH dietary recommendations through the integration of a UKG. This framework uses multistep LLM calls for OpenIE to automatically construct an MFH KG, incorporating UKG reasoning to measure the confidence of existing triples and complete missing triples. Upon receiving user queries, the framework implements a reasoning path–based knowledge retrieval strategy, extracting relevant reasoning paths from the MFH KG and optimizing the retrieval results through a postprocessing mechanism. Experimental evaluations demonstrate that DeepSeek-R1 is the best-performing base model for MFH-based dietary recommendation generation. This framework facilitates dietary recommendations that adapt to individual health conditions and symptom requirements while considering diverse ingredients and dishes, accompanied by comprehensive explanations grounded in MFH principles. In the future, we plan to focus on implementing more effective knowledge extraction methodologies, expanding our KG through additional MFH literature, exploring optimized retrieval strategies, and fine-tuning open-source LLMs to further improve the accuracy and reliability of the generated dietary recommendations.</p></section></section><section id="s5"><h2 class="pmc_sec_title">Supplementary material</h2>
<section class="sm xbox font-sm" id="SAP1"><div class="caption p"><span>Multimedia Appendix 1. Prompts for medicine food homology knowledge graph construction.</span></div>
<div class="media p" id="d67e2569">
<div class="caption">
<a href="/articles/instance/12370266/bin/medinform-v13-e75279-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">medinform-v13-e75279-s001.docx</a><sup> (21.8KB, docx) </sup>
</div>
<div class="object-id font-xs text-base-dark">DOI: 10.2196/75279</div>
</div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>This paper is supported by the National Natural Science Foundation of China (grants 82104786, 62376058, 52378009, and 62276063), the Southeast University Interdisciplinary Research Program for Young Scholars, and the Big Data Computing Center of Southeast University.</p></section><section id="notes1"><h3>Abbreviations</h3>
<dl class="def-list">
<dt>KG</dt>
<dd><p>knowledge graph</p></dd>
<dt>LLM</dt>
<dd><p>large language model</p></dd>
<dt>MFH</dt>
<dd><p>medicine food homology</p></dd>
<dt>OpenIE</dt>
<dd><p>open information extraction</p></dd>
<dt>RAG</dt>
<dd><p>retrieval-augmented generation</p></dd>
<dt>TCM</dt>
<dd><p>Traditional Chinese Medicine</p></dd>
<dt>UKG</dt>
<dd><p>uncertain knowledge graph</p></dd>
</dl></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn12"><p><strong>Authors’ Contributions:</strong> HS and TW contributed to the algorithm design; HS performed experiments and wrote the paper; RL performed experiments; and FG, TW, BL, and HW provided theoretical guidance and the revision of this paper.</p></div>
<div class="fn p" id="fn13"><p><strong>Conflicts of Interest:</strong> None declared.</p></div>
</div></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="R1">
<span class="label">1.</span><cite>Zhong H, Tang ZQ, Li YF, Wang M, Sun WY, He RR. The evolution and significance of medicine and food homology. Acupunct Herbal Med. 2024;4(1):19–35. doi: 10.1097/HM9.0000000000000104. doi.</cite> [<a href="https://doi.org/10.1097/HM9.0000000000000104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acupunct%20Herbal%20Med&amp;title=The%20evolution%20and%20significance%20of%20medicine%20and%20food%20homology&amp;author=H%20Zhong&amp;author=ZQ%20Tang&amp;author=YF%20Li&amp;author=M%20Wang&amp;author=WY%20Sun&amp;volume=4&amp;issue=1&amp;publication_year=2024&amp;pages=19-35&amp;doi=10.1097/HM9.0000000000000104&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R2">
<span class="label">2.</span><cite>Chen J. Essential role of medicine and food homology in health and wellness. Chin Herb Med. 2023 Jul;15(3):347–348. doi: 10.1016/j.chmed.2023.05.001. doi. Medline.</cite> [<a href="https://doi.org/10.1016/j.chmed.2023.05.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10394323/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37538868/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chin%20Herb%20Med&amp;title=Essential%20role%20of%20medicine%20and%20food%20homology%20in%20health%20and%20wellness&amp;author=J%20Chen&amp;volume=15&amp;issue=3&amp;publication_year=2023&amp;pages=347-348&amp;pmid=37538868&amp;doi=10.1016/j.chmed.2023.05.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R3">
<span class="label">3.</span><cite>Gong X, Ji M, Xu J, Zhang C, Li M. Hypoglycemic effects of bioactive ingredients from medicine food homology and medicinal health food species used in China. Crit Rev Food Sci Nutr. 2020;60(14):2303–2326. doi: 10.1080/10408398.2019.1634517. doi. Medline.</cite> [<a href="https://doi.org/10.1080/10408398.2019.1634517" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31309854/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crit%20Rev%20Food%20Sci%20Nutr&amp;title=Hypoglycemic%20effects%20of%20bioactive%20ingredients%20from%20medicine%20food%20homology%20and%20medicinal%20health%20food%20species%20used%20in%20China&amp;author=X%20Gong&amp;author=M%20Ji&amp;author=J%20Xu&amp;author=C%20Zhang&amp;author=M%20Li&amp;volume=60&amp;issue=14&amp;publication_year=2020&amp;pages=2303-2326&amp;pmid=31309854&amp;doi=10.1080/10408398.2019.1634517&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R4">
<span class="label">4.</span><cite>Zhao W, Zhou K, Li J, et al.  A survey of large language models. arXiv. 2023 doi: 10.48550/arXiv.2303.18223. Preprint posted online on. doi.</cite> [<a href="https://doi.org/10.48550/arXiv.2303.18223" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R5">
<span class="label">5.</span><cite>Thirunavukarasu AJ, Ting DSJ, Elangovan K, Gutierrez L, Tan TF, Ting DSW. Large language models in medicine. Nat Med. 2023 Aug;29(8):1930–1940. doi: 10.1038/s41591-023-02448-8. doi. Medline.</cite> [<a href="https://doi.org/10.1038/s41591-023-02448-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37460753/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Med&amp;title=Large%20language%20models%20in%20medicine&amp;author=AJ%20Thirunavukarasu&amp;author=DSJ%20Ting&amp;author=K%20Elangovan&amp;author=L%20Gutierrez&amp;author=TF%20Tan&amp;volume=29&amp;issue=8&amp;publication_year=2023&amp;pages=1930-1940&amp;pmid=37460753&amp;doi=10.1038/s41591-023-02448-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R6">
<span class="label">6.</span><cite>Liu S, McCoy AB, Wright AP, et al.  Why do users override alerts? Utilizing large language model to summarize comments and optimize clinical decision support. J Am Med Inform Assoc. 2024 May 20;31(6):1388–1396. doi: 10.1093/jamia/ocae041. doi. Medline.</cite> [<a href="https://doi.org/10.1093/jamia/ocae041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11105133/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38452289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Am%20Med%20Inform%20Assoc&amp;title=Why%20do%20users%20override%20alerts?%20Utilizing%20large%20language%20model%20to%20summarize%20comments%20and%20optimize%20clinical%20decision%20support&amp;author=S%20Liu&amp;author=AB%20McCoy&amp;author=AP%20Wright&amp;volume=31&amp;issue=6&amp;publication_year=2024&amp;pages=1388-1396&amp;pmid=38452289&amp;doi=10.1093/jamia/ocae041&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R7">
<span class="label">7.</span><cite>Abd-Alrazaq A, AlSaad R, Alhuwail D, et al.  Large language models in medical education: opportunities, challenges, and future directions. JMIR Med Educ. 2023 Jun 1;9:e48291.  doi: 10.2196/48291. doi. Medline.</cite> [<a href="https://doi.org/10.2196/48291" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10273039/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37261894/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JMIR%20Med%20Educ&amp;title=Large%20language%20models%20in%20medical%20education:%20opportunities,%20challenges,%20and%20future%20directions&amp;author=A%20Abd-Alrazaq&amp;author=R%20AlSaad&amp;author=D%20Alhuwail&amp;volume=9&amp;publication_year=2023&amp;pages=e48291&amp;pmid=37261894&amp;doi=10.2196/48291&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R8">
<span class="label">8.</span><cite>Wiest IC, Ferber D, Zhu J, et al.  Privacy-preserving large language models for structured medical information retrieval. NPJ Digit Med. 2024 Sep 20;7(1):257.  doi: 10.1038/s41746-024-01233-2. doi. Medline.</cite> [<a href="https://doi.org/10.1038/s41746-024-01233-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11415382/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39304709/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=NPJ%20Digit%20Med&amp;title=Privacy-preserving%20large%20language%20models%20for%20structured%20medical%20information%20retrieval&amp;author=IC%20Wiest&amp;author=D%20Ferber&amp;author=J%20Zhu&amp;volume=7&amp;issue=1&amp;publication_year=2024&amp;pages=257&amp;pmid=39304709&amp;doi=10.1038/s41746-024-01233-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R9">
<span class="label">9.</span><cite>Azamfirei R, Kudchadkar SR, Fackler J. Large language models and the perils of their hallucinations. Crit Care. 2023 Mar 21;27(1):120.  doi: 10.1186/s13054-023-04393-x. doi. Medline.</cite> [<a href="https://doi.org/10.1186/s13054-023-04393-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10032023/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36945051/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Crit%20Care&amp;title=Large%20language%20models%20and%20the%20perils%20of%20their%20hallucinations&amp;author=R%20Azamfirei&amp;author=SR%20Kudchadkar&amp;author=J%20Fackler&amp;volume=27&amp;issue=1&amp;publication_year=2023&amp;pages=120&amp;pmid=36945051&amp;doi=10.1186/s13054-023-04393-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R10">
<span class="label">10.</span><cite>Kim Y, Jeong H, Chen S, et al.  Medical hallucination in foundation models and their impact on healthcare. medRxiv. 2025 doi: 10.1101/2025.02.28.25323115. Preprint posted online on. doi.</cite> [<a href="https://doi.org/10.1101/2025.02.28.25323115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R11">
<span class="label">11.</span><cite>Gao Y, Xiong Y, Gao X, et al.  Retrieval-augmented generation for large language models: a survey. arXiv. 2023 doi: 10.48550/arXiv.2312.10997. Preprint posted online on. doi.</cite> [<a href="https://doi.org/10.48550/arXiv.2312.10997" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R12">
<span class="label">12.</span><cite>Sun-Waterhouse DX, Chen XY, Liu ZH, I.N. Waterhouse G, Kang WY. Transformation from traditional medicine-food homology to modern food-medicine homology. Food Med Homology. 2024 Sep;1(1):9420014. doi: 10.26599/FMH.2024.9420014. doi.</cite> [<a href="https://doi.org/10.26599/FMH.2024.9420014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Med%20Homology&amp;title=Transformation%20from%20traditional%20medicine-food%20homology%20to%20modern%20food-medicine%20homology&amp;author=DX%20Sun-Waterhouse&amp;author=XY%20Chen&amp;author=ZH%20Liu&amp;author=G%20I.N.%20Waterhouse&amp;author=WY%20Kang&amp;volume=1&amp;issue=1&amp;publication_year=2024&amp;pages=9420014&amp;doi=10.26599/FMH.2024.9420014&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R13">
<span class="label">13.</span><cite>Chen Y, Subburathinam A, Chen CH, Zaki MJ. Personalized food recommendation as constrained question answering over a large-scale food knowledge graph. Proceedings of the 14th ACM International Conference on Web Search and Data Mining; Mar 8-12, 2021; Virtual Event, Israel. pp. 544–552. Presented at. doi.</cite> [<a href="https://doi.org/10.1145/3437963.3441816" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20Y,%20Subburathinam%20A,%20Chen%20CH,%20Zaki%20MJ.%20Personalized%20food%20recommendation%20as%20constrained%20question%20answering%20over%20a%20large-scale%20food%20knowledge%20graph.%20Proceedings%20of%20the%2014th%20ACM%20International%20Conference%20on%20Web%20Search%20and%20Data%20Mining;%20Mar%208-12,%202021;%20Virtual%20Event,%20Israel.%20pp.%20544%E2%80%93552.%20Presented%20at.%20doi." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R14">
<span class="label">14.</span><cite>Tang J, Huang B, Xie M. Anticancer recipe recommendation based on cancer dietary knowledge graph. Eur J Cancer Care (Engl) 2023 Oct 18;2023(1):1–13. doi: 10.1155/2023/8816960. doi.</cite> [<a href="https://doi.org/10.1155/2023/8816960" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur%20J%20Cancer%20Care%20(Engl)&amp;title=Anticancer%20recipe%20recommendation%20based%20on%20cancer%20dietary%20knowledge%20graph&amp;author=J%20Tang&amp;author=B%20Huang&amp;author=M%20Xie&amp;volume=2023&amp;issue=1&amp;publication_year=2023&amp;pages=1-13&amp;doi=10.1155/2023/8816960&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R15">
<span class="label">15.</span><cite>Ma W, Li M, Dai J, Ding J, Chu Z, Chen H. Nutrition-related knowledge graph neural network for food recommendation. Foods. 2024 Jul 5;13(13):38998649. doi: 10.3390/foods13132144. doi. Medline.</cite> [<a href="https://doi.org/10.3390/foods13132144" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11241430/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38998649/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Foods&amp;title=Nutrition-related%20knowledge%20graph%20neural%20network%20for%20food%20recommendation&amp;author=W%20Ma&amp;author=M%20Li&amp;author=J%20Dai&amp;author=J%20Ding&amp;author=Z%20Chu&amp;volume=13&amp;issue=13&amp;publication_year=2024&amp;pages=38998649&amp;pmid=38998649&amp;doi=10.3390/foods13132144&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R16">
<span class="label">16.</span><cite>Yang Z, Khatibi E, Nagesh N, et al.  ChatDiet: empowering personalized nutrition-oriented food recommender chatbots through an LLM-augmented framework. Smart Health (2014) 2024 Jun;32:100465. doi: 10.1016/j.smhl.2024.100465. doi.</cite> [<a href="https://doi.org/10.1016/j.smhl.2024.100465" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Smart%20Health%20(2014)&amp;title=ChatDiet:%20empowering%20personalized%20nutrition-oriented%20food%20recommender%20chatbots%20through%20an%20LLM-augmented%20framework&amp;author=Z%20Yang&amp;author=E%20Khatibi&amp;author=N%20Nagesh&amp;volume=32&amp;publication_year=2024&amp;pages=100465&amp;doi=10.1016/j.smhl.2024.100465&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R17">
<span class="label">17.</span><cite>Jin H, Lin Q, Lu J, et al.  Evaluating the effectiveness of a generative pretrained transformer-based dietary recommendation system in managing potassium intake for hemodialysis patients. J Ren Nutr. 2024 Nov;34(6):539–545. doi: 10.1053/j.jrn.2024.04.001. doi. Medline.</cite> [<a href="https://doi.org/10.1053/j.jrn.2024.04.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38615701/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Ren%20Nutr&amp;title=Evaluating%20the%20effectiveness%20of%20a%20generative%20pretrained%20transformer-based%20dietary%20recommendation%20system%20in%20managing%20potassium%20intake%20for%20hemodialysis%20patients&amp;author=H%20Jin&amp;author=Q%20Lin&amp;author=J%20Lu&amp;volume=34&amp;issue=6&amp;publication_year=2024&amp;pages=539-545&amp;pmid=38615701&amp;doi=10.1053/j.jrn.2024.04.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R18">
<span class="label">18.</span><cite>Kopitar L, Stiglic G, Bedrac L, Bian J. Personalized meal planning in inpatient clinical dietetics using generative artificial intelligence: system description. 2024 IEEE 12th International Conference on Healthcare Informatics (ICHI); Jun 3-6, 2024; Orlando, FL, USA. pp. 2024–2331. Presented at. doi.</cite> [<a href="https://doi.org/10.1109/ICHI61247.2024.00049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Kopitar%20L,%20Stiglic%20G,%20Bedrac%20L,%20Bian%20J.%20Personalized%20meal%20planning%20in%20inpatient%20clinical%20dietetics%20using%20generative%20artificial%20intelligence:%20system%20description.%202024%20IEEE%2012th%20International%20Conference%20on%20Healthcare%20Informatics%20(ICHI);%20Jun%203-6,%202024;%20Orlando,%20FL,%20USA.%20pp.%202024%E2%80%932331.%20Presented%20at.%20doi." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R19">
<span class="label">19.</span><cite>Pan S, Luo L, Wang Y, Chen C, Wang J, Wu X. Unifying large language models and knowledge graphs: a roadmap. IEEE Trans Knowl Data Eng. 2024 Jul;36(7):3580–3599. doi: 10.1109/TKDE.2024.3352100. doi.</cite> [<a href="https://doi.org/10.1109/TKDE.2024.3352100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IEEE%20Trans%20Knowl%20Data%20Eng&amp;title=Unifying%20large%20language%20models%20and%20knowledge%20graphs:%20a%20roadmap&amp;author=S%20Pan&amp;author=L%20Luo&amp;author=Y%20Wang&amp;author=C%20Chen&amp;author=J%20Wang&amp;volume=36&amp;issue=7&amp;publication_year=2024&amp;pages=3580-3599&amp;doi=10.1109/TKDE.2024.3352100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R20">
<span class="label">20.</span><cite>Pan J, Razniewski S, Kalo JC, et al.  Large language models and knowledge graphs: opportunities and challenges. arXiv. 2023 doi: 10.48550/arXiv.2308.06374. Preprint posted online on. doi.</cite> [<a href="https://doi.org/10.48550/arXiv.2308.06374" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R21">
<span class="label">21.</span><cite>Zhang B, Soh H. Extract, define, canonicalize: an LLM-based framework for knowledge graph construction. Proceedings of the 2024 Conference on Empirical Methods in Natural Language Processing; 2024; Miami, Florida, USA. 2024. pp. 9820–9836. Presented at. doi.</cite> [<a href="https://doi.org/10.18653/v1/2024.emnlp-main.548" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20B,%20Soh%20H.%20Extract,%20define,%20canonicalize:%20an%20LLM-based%20framework%20for%20knowledge%20graph%20construction.%20Proceedings%20of%20the%202024%20Conference%20on%20Empirical%20Methods%20in%20Natural%20Language%20Processing;%202024;%20Miami,%20Florida,%20USA.%202024.%20pp.%209820%E2%80%939836.%20Presented%20at.%20doi." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R22">
<span class="label">22.</span><cite>Yu S, Huang T, Liu M, Wang Z. B.  Service-Oriented Computing ICSOC 2023 Lecture Notes in Computer Science. Vol. 14419. Springer; 2023. BEAR: revolutionizing service domain knowledge graph construction with LLM. Vol. doi.</cite> [<a href="https://doi.org/10.1007/978-3-031-48421-6_23" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Service-Oriented%20Computing%20ICSOC%202023%20Lecture%20Notes%20in%20Computer%20Science&amp;author=S%20Yu&amp;author=T%20Huang&amp;author=M%20Liu&amp;author=B%20Wang%20Z.&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R23">
<span class="label">23.</span><cite>Hu Y, Zou F, Han J, Sun X, Wang Y. LLM-TIKG: Threat intelligence knowledge graph construction utilizing large language model. Comput Secur. 2024 Oct;145:103999. doi: 10.1016/j.cose.2024.103999. doi.</cite> [<a href="https://doi.org/10.1016/j.cose.2024.103999" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Comput%20Secur&amp;title=LLM-TIKG:%20Threat%20intelligence%20knowledge%20graph%20construction%20utilizing%20large%20language%20model&amp;author=Y%20Hu&amp;author=F%20Zou&amp;author=J%20Han&amp;author=X%20Sun&amp;author=Y%20Wang&amp;volume=145&amp;publication_year=2024&amp;pages=103999&amp;doi=10.1016/j.cose.2024.103999&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R24">
<span class="label">24.</span><cite>Yang H, Xiao L, Zhu R, Liu Z, Chen J. An LLM supported approach to ontology and knowledge graph construction. 2024 IEEE International Conference on Bioinformatics and Biomedicine (BIBM); Dec 3-6, 2024; Lisbon, Portugal. pp. 5240–5246. Presented at. doi.</cite> [<a href="https://doi.org/10.1109/BIBM62325.2024.10822222" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20H,%20Xiao%20L,%20Zhu%20R,%20Liu%20Z,%20Chen%20J.%20An%20LLM%20supported%20approach%20to%20ontology%20and%20knowledge%20graph%20construction.%202024%20IEEE%20International%20Conference%20on%20Bioinformatics%20and%20Biomedicine%20(BIBM);%20Dec%203-6,%202024;%20Lisbon,%20Portugal.%20pp.%205240%E2%80%935246.%20Presented%20at.%20doi." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R25">
<span class="label">25.</span><cite>Tang J, Yang Y, Wei W, et al.  GraphGPT: graph instruction tuning for large language models. Proceedings of the 47th International ACM SIGIR Conference on Research and Development in Information Retrieval (SIGIR ’24); Jul 14-18, 2024; Washington, DC, USA. Jul 10, 2024. pp. 491–500. Presented at. doi.</cite> [<a href="https://doi.org/10.1145/3626772.3657775" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Tang%20J,%20Yang%20Y,%20Wei%20W,%20et%20al.%20GraphGPT:%20graph%20instruction%20tuning%20for%20large%20language%20models.%20Proceedings%20of%20the%2047th%20International%20ACM%20SIGIR%20Conference%20on%20Research%20and%20Development%20in%20Information%20Retrieval%20(SIGIR%20%E2%80%9924);%20Jul%2014-18,%202024;%20Washington,%20DC,%20USA.%20Jul%2010,%202024.%20pp.%20491%E2%80%93500.%20Presented%20at.%20doi." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R26">
<span class="label">26.</span><cite>Dernbach S, Agarwal K, Zuniga A, Henry M, Choudhury S. GLaM: fine-tuning large language models for domain knowledge graph alignment via neighborhood partitioning and generative subgraph encoding. AAAI-SS. 2024;3(1):82–89. doi: 10.1609/aaaiss.v3i1.31186. doi.</cite> [<a href="https://doi.org/10.1609/aaaiss.v3i1.31186" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=AAAI-SS&amp;title=GLaM:%20fine-tuning%20large%20language%20models%20for%20domain%20knowledge%20graph%20alignment%20via%20neighborhood%20partitioning%20and%20generative%20subgraph%20encoding&amp;author=S%20Dernbach&amp;author=K%20Agarwal&amp;author=A%20Zuniga&amp;author=M%20Henry&amp;author=S%20Choudhury&amp;volume=3&amp;issue=1&amp;publication_year=2024&amp;pages=82-89&amp;doi=10.1609/aaaiss.v3i1.31186&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R27">
<span class="label">27.</span><cite>Sun J, Xu C, Tang L, et al.  Think-on-graph: deep and responsible reasoning of large language model on knowledge graph. The Twelfth International Conference on Learning Representations (ICLR; May, 2024.  [08-08-2025].  Presented at. URL. Accessed.</cite> [<a href="https://scholar.google.com/scholar_lookup?Sun%20J,%20Xu%20C,%20Tang%20L,%20et%20al.%20Think-on-graph:%20deep%20and%20responsible%20reasoning%20of%20large%20language%20model%20on%20knowledge%20graph.%20The%20Twelfth%20International%20Conference%20on%20Learning%20Representations%20(ICLR;%20May,%202024.%20%5B08-08-2025%5D.%20Presented%20at.%20URL.%20Accessed." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R28">
<span class="label">28.</span><cite>Jiang J, Zhou K, Dong Z, Ye K, Zhao X, Wen JR. StructGPT: a general framework for large language model to reason over structured data. Proceedings of the 2023 Conference on Empirical Methods in Natural Language Processing; 2023; Singapore. pp. 9237–9251. Presented at. doi.</cite> [<a href="https://doi.org/10.18653/v1/2023.emnlp-main.574" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang%20J,%20Zhou%20K,%20Dong%20Z,%20Ye%20K,%20Zhao%20X,%20Wen%20JR.%20StructGPT:%20a%20general%20framework%20for%20large%20language%20model%20to%20reason%20over%20structured%20data.%20Proceedings%20of%20the%202023%20Conference%20on%20Empirical%20Methods%20in%20Natural%20Language%20Processing;%202023;%20Singapore.%20pp.%209237%E2%80%939251.%20Presented%20at.%20doi." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R29">
<span class="label">29.</span><cite>Niklaus C, Cetto M, Freitas A, et al.  A survey on open information extraction. The 27th International Conference on Computational Linguistics (COLING); 2018; Santa Fe, New Mexico, USA.  [08-08-2025]. pp. 3866–3878. Presented at. URL. Accessed.</cite> [<a href="https://scholar.google.com/scholar_lookup?Niklaus%20C,%20Cetto%20M,%20Freitas%20A,%20et%20al.%20A%20survey%20on%20open%20information%20extraction.%20The%2027th%20International%20Conference%20on%20Computational%20Linguistics%20(COLING);%202018;%20Santa%20Fe,%20New%20Mexico,%20USA.%20%5B08-08-2025%5D.%20pp.%203866%E2%80%933878.%20Presented%20at.%20URL.%20Accessed." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R30">
<span class="label">30.</span><cite>Wang J, Wu T, Chen S, et al.  UnKR: a Python library for uncertain knowledge graph reasoning by representation learning. Proceedings of the 47th International ACM SIGIR Conference on Research and Development in Information Retrieval (SIGIR ’24); 2024; United States. pp. 2822–2826. Presented at. doi.</cite> [<a href="https://doi.org/10.1145/3626772.3657661" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20J,%20Wu%20T,%20Chen%20S,%20et%20al.%20UnKR:%20a%20Python%20library%20for%20uncertain%20knowledge%20graph%20reasoning%20by%20representation%20learning.%20Proceedings%20of%20the%2047th%20International%20ACM%20SIGIR%20Conference%20on%20Research%20and%20Development%20in%20Information%20Retrieval%20(SIGIR%20%E2%80%9924);%202024;%20United%20States.%20pp.%202822%E2%80%932826.%20Presented%20at.%20doi." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R31">
<span class="label">31.</span><cite>Wu T, Yao K, Li W, et al.  Triple confidence measurement in knowledge graph with multiple heterogeneous evidences. World Wide Web. 2024 Nov;27(6):70. doi: 10.1007/s11280-024-01307-x. doi.</cite> [<a href="https://doi.org/10.1007/s11280-024-01307-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=World%20Wide%20Web&amp;title=Triple%20confidence%20measurement%20in%20knowledge%20graph%20with%20multiple%20heterogeneous%20evidences&amp;author=T%20Wu&amp;author=K%20Yao&amp;author=W%20Li&amp;volume=27&amp;issue=6&amp;publication_year=2024&amp;pages=70&amp;doi=10.1007/s11280-024-01307-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R32">
<span class="label">32.</span><cite>Reimers N, Gurevych I. Sentence-BERT: sentence embeddings using Siamese BERT-networks. Proceedings of the 2019 Conference on Empirical Methods in Natural Language Processing and the 9th International Joint Conference on Natural Language Processing (EMNLP-IJCNLP); 2019; Hong Kong, China. pp. 3982–3992. Presented at. doi.</cite> [<a href="https://doi.org/10.18653/v1/D19-1410" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Reimers%20N,%20Gurevych%20I.%20Sentence-BERT:%20sentence%20embeddings%20using%20Siamese%20BERT-networks.%20Proceedings%20of%20the%202019%20Conference%20on%20Empirical%20Methods%20in%20Natural%20Language%20Processing%20and%20the%209th%20International%20Joint%20Conference%20on%20Natural%20Language%20Processing%20(EMNLP-IJCNLP);%202019;%20Hong%20Kong,%20China.%20pp.%203982%E2%80%933992.%20Presented%20at.%20doi." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R33">
<span class="label">33.</span><cite>Devlin J, Chang MW, Lee K, Toutanova K. BERT: pre-training of deep bidirectional transformers for language understanding. Proceedings of the 2019 Conference of the North American Chapter of the Association for Computational Linguistics: Human Language Technologies; 2019; Minneapolis, MA, USA. pp. 4171–4186. Presented at. doi.</cite> [<a href="https://doi.org/10.18653/v1/N19-1423" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Devlin%20J,%20Chang%20MW,%20Lee%20K,%20Toutanova%20K.%20BERT:%20pre-training%20of%20deep%20bidirectional%20transformers%20for%20language%20understanding.%20Proceedings%20of%20the%202019%20Conference%20of%20the%20North%20American%20Chapter%20of%20the%20Association%20for%20Computational%20Linguistics:%20Human%20Language%20Technologies;%202019;%20Minneapolis,%20MA,%20USA.%20pp.%204171%E2%80%934186.%20Presented%20at.%20doi." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R34">
<span class="label">34.</span><cite>Liu Y, Ott M, Goyal N, et al.  RoBERTa: a robustly optimized BERT pretraining approach. arXiv. 2019 doi: 10.48550/arXiv.1907.11692. Preprint posted online on. doi.</cite> [<a href="https://doi.org/10.48550/arXiv.1907.11692" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R35">
<span class="label">35.</span><cite>Bekamiri H, Hain DS, Jurowetzki R. PatentSBERTa: a deep NLP based hybrid model for patent distance and classification using augmented SBERT. Technol Forecast Soc Change. 2024 Sep;206:123536. doi: 10.1016/j.techfore.2024.123536. doi.</cite> [<a href="https://doi.org/10.1016/j.techfore.2024.123536" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Technol%20Forecast%20Soc%20Change&amp;title=PatentSBERTa:%20a%20deep%20NLP%20based%20hybrid%20model%20for%20patent%20distance%20and%20classification%20using%20augmented%20SBERT&amp;author=H%20Bekamiri&amp;author=DS%20Hain&amp;author=R%20Jurowetzki&amp;volume=206&amp;publication_year=2024&amp;pages=123536&amp;doi=10.1016/j.techfore.2024.123536&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R36">
<span class="label">36.</span><cite>Li M, Miao S, Li P. Simple is effective: the roles of graphs and large language models in knowledge-graph-based retrieval-augmented generation. arXiv. 2024 doi: 10.48550/arXiv.2410.20724. Preprint posted online on. doi.</cite> [<a href="https://doi.org/10.48550/arXiv.2410.20724" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R37">
<span class="label">37.</span><cite>Luo L, Li YF, Haffari G, et al.  Reasoning on graphs: faithful and interpretable large language model reasoning. The Twelfth International Conference on Learning Representations; 2024.  [08-08-2025].  Presented at. URL. Accessed.</cite> [<a href="https://scholar.google.com/scholar_lookup?Luo%20L,%20Li%20YF,%20Haffari%20G,%20et%20al.%20Reasoning%20on%20graphs:%20faithful%20and%20interpretable%20large%20language%20model%20reasoning.%20The%20Twelfth%20International%20Conference%20on%20Learning%20Representations;%202024.%20%5B08-08-2025%5D.%20Presented%20at.%20URL.%20Accessed." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R38">
<span class="label">38.</span><cite>Bollacker K, Evans C, Paritosh P, et al.  Freebase: a collaboratively created graph database for structuring human knowledge. Proceedings of the 2008 ACM SIGMOD International Conference on Management of Data (SIGMOD ’08); Jun 9-12, 2008; New York, NY. 2008. pp. 1247–1250. Presented at. doi.</cite> [<a href="https://doi.org/10.1145/1376616.1376746" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Bollacker%20K,%20Evans%20C,%20Paritosh%20P,%20et%20al.%20Freebase:%20a%20collaboratively%20created%20graph%20database%20for%20structuring%20human%20knowledge.%20Proceedings%20of%20the%202008%20ACM%20SIGMOD%20International%20Conference%20on%20Management%20of%20Data%20(SIGMOD%20%E2%80%9908);%20Jun%209-12,%202008;%20New%20York,%20NY.%202008.%20pp.%201247%E2%80%931250.%20Presented%20at.%20doi." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R39">
<span class="label">39.</span><cite>Bianchini M, Gori M, Scarselli F. Inside PageRank. ACM Trans Internet Technol. 2005 Feb;5(1):92–128. doi: 10.1145/1052934.1052938. doi.</cite> [<a href="https://doi.org/10.1145/1052934.1052938" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACM%20Trans%20Internet%20Technol&amp;title=Inside%20PageRank&amp;author=M%20Bianchini&amp;author=M%20Gori&amp;author=F%20Scarselli&amp;volume=5&amp;issue=1&amp;publication_year=2005&amp;pages=92-128&amp;doi=10.1145/1052934.1052938&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R40">
<span class="label">40.</span><cite>OpenAI AJ, Adler S, et al.  GPT-4 technical report. arXiv. 2024 doi: 10.48550/arXiv.2303.08774. Preprint posted online on. doi.</cite> [<a href="https://doi.org/10.48550/arXiv.2303.08774" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R41">
<span class="label">41.</span><cite>Touvron H, Martin L, Stone K, et al.  LLaMA 2: open foundation and fine-tuned chat models. 2023 doi: 10.48550/arXiv.2307.09288. Preprint posted online on. doi.</cite> [<a href="https://doi.org/10.48550/arXiv.2307.09288" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R42">
<span class="label">42.</span><cite>Yang A, Yang B, Zhang B, et al.  Qwen2.5 technical report. arXiv. 2025 doi: 10.48550/arXiv.2412.15115. Preprint posted online on. doi.</cite> [<a href="https://doi.org/10.48550/arXiv.2412.15115" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R43">
<span class="label">43.</span><cite>DeepSeek-AI. Guo D, Yang D, et al.  DeepSeek-R1: incentivizing reasoning capability in LLMs via reinforcement learning. 2025 doi: 10.48550/arXiv.2501.12948. Preprint posted online on. doi.</cite> [<a href="https://doi.org/10.48550/arXiv.2501.12948" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>]</li>
<li id="R44">
<span class="label">44.</span><cite>TCM-ID: Traditional Chinese Medicine information database. TCM-ID.  [10-12-2024]. <a href="https://bidd.group/TCMID/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bidd.group/TCMID/</a> URL. Accessed.</cite>
</li>
<li id="R45">
<span class="label">45.</span><cite>PubMed.  [10-12-2024]. <a href="https://pubmed.ncbi.nlm.nih.gov/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pubmed.ncbi.nlm.nih.gov/</a> URL. Accessed.</cite>
</li>
<li id="R46">
<span class="label">46.</span><cite>Global academic insights from China and beyond. Cnki.  [10-12-2024]. <a href="https://oversea.cnki.net/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://oversea.cnki.net/</a> URL. Accessed.</cite>
</li>
<li id="R47">
<span class="label">47.</span><cite>Page L, Brin S, Motwani R, Winograd T.  Stanford InfoLab; 1999.  [08-08-2025]. The PageRank citation ranking: bringing order to the web.<a href="http://ilpubs.stanford.edu:8090/422" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://ilpubs.stanford.edu:8090/422</a> URL. Accessed.</cite> [<a href="https://scholar.google.com/scholar_lookup?Page%20L,%20Brin%20S,%20Motwani%20R,%20Winograd%20T.%20Stanford%20InfoLab;%201999.%20%5B08-08-2025%5D.%20The%20PageRank%20citation%20ranking:%20bringing%20order%20to%20the%20web.http://ilpubs.stanford.edu:8090/422%20URL.%20Accessed." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R48">
<span class="label">48.</span><cite>Srivastava AK, Garg R, Mishra PK. Discussion on damping factor value in PageRank computation. IJISA. 2017 Sep 8;9(9):19–28. doi: 10.5815/ijisa.2017.09.03. doi.</cite> [<a href="https://doi.org/10.5815/ijisa.2017.09.03" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=IJISA&amp;title=Discussion%20on%20damping%20factor%20value%20in%20PageRank%20computation&amp;author=AK%20Srivastava&amp;author=R%20Garg&amp;author=PK%20Mishra&amp;volume=9&amp;issue=9&amp;publication_year=2017&amp;pages=19-28&amp;doi=10.5815/ijisa.2017.09.03&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R49">
<span class="label">49.</span><cite>Boldi P, Santini M, Vigna S. PageRank as a function of the damping factor. Proceedings of the 14th International Conference on World Wide Web; Chiba, Japan. 2005. pp. 557–566. Presented at. doi.</cite> [<a href="https://doi.org/10.1145/1060745.1060827" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Boldi%20P,%20Santini%20M,%20Vigna%20S.%20PageRank%20as%20a%20function%20of%20the%20damping%20factor.%20Proceedings%20of%20the%2014th%20International%20Conference%20on%20World%20Wide%20Web;%20Chiba,%20Japan.%202005.%20pp.%20557%E2%80%93566.%20Presented%20at.%20doi." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R50">
<span class="label">50.</span><cite>Robertson S, Zaragoza H. The probabilistic relevance framework: BM25 and beyond. FNT Inf Retrieval. 2009;3(4):333–389. doi: 10.1561/1500000019. doi.</cite> [<a href="https://doi.org/10.1561/1500000019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FNT%20Inf%20Retrieval&amp;title=The%20probabilistic%20relevance%20framework:%20BM25%20and%20beyond&amp;author=S%20Robertson&amp;author=H%20Zaragoza&amp;volume=3&amp;issue=4&amp;publication_year=2009&amp;pages=333-389&amp;doi=10.1561/1500000019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R51">
<span class="label">51.</span><cite>Yu D, Zhang S, Ng P, et al.  DecAF: joint decoding of answers and logical forms for question answering over knowledge bases. The Eleventh International Conference on Learning Representations; 2023.  [08-08-2025].  Presented at. URL. Accessed.</cite> [<a href="https://scholar.google.com/scholar_lookup?Yu%20D,%20Zhang%20S,%20Ng%20P,%20et%20al.%20DecAF:%20joint%20decoding%20of%20answers%20and%20logical%20forms%20for%20question%20answering%20over%20knowledge%20bases.%20The%20Eleventh%20International%20Conference%20on%20Learning%20Representations;%202023.%20%5B08-08-2025%5D.%20Presented%20at.%20URL.%20Accessed." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R52">
<span class="label">52.</span><cite>Bordes A, Usunier N, Garcia-Duran A, et al.  Translating embeddings for modeling multi-relational data. In: Burges CJ, Bottou L, Welling M, Ghahramani Z, Weinberger KQ, editors. NIPS’13: Neural Information Processing Systems; Dec 5-10, 2013; Lake Tahoe, NV.  [08-08-2025].  In. Presented at. URL. Accessed.</cite> [<a href="https://scholar.google.com/scholar_lookup?Bordes%20A,%20Usunier%20N,%20Garcia-Duran%20A,%20et%20al.%20Translating%20embeddings%20for%20modeling%20multi-relational%20data.%20In:%20Burges%20CJ,%20Bottou%20L,%20Welling%20M,%20Ghahramani%20Z,%20Weinberger%20KQ,%20editors.%20NIPS%E2%80%9913:%20Neural%20Information%20Processing%20Systems;%20Dec%205-10,%202013;%20Lake%20Tahoe,%20NV.%20%5B08-08-2025%5D.%20In.%20Presented%20at.%20URL.%20Accessed." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R53">
<span class="label">53.</span><cite>Sun Z, Deng ZH, Nie JY, et al.  RotatE: knowledge graph embedding by relational rotation in complex space. International Conference on Learning Representations; May 6-9, 2019; New Orleans, LA.  [08-08-2025].  Presented at. URL. Accessed.</cite> [<a href="https://scholar.google.com/scholar_lookup?Sun%20Z,%20Deng%20ZH,%20Nie%20JY,%20et%20al.%20RotatE:%20knowledge%20graph%20embedding%20by%20relational%20rotation%20in%20complex%20space.%20International%20Conference%20on%20Learning%20Representations;%20May%206-9,%202019;%20New%20Orleans,%20LA.%20%5B08-08-2025%5D.%20Presented%20at.%20URL.%20Accessed." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="R54">
<span class="label">54.</span><cite>Cuconasu F, Trappolini G, Siciliano F, et al.  The power of noise: redefining retrieval for RAG systems. Proceedings of the 47th International ACM SIGIR Conference on Research and Development in Information Retrieval (SIGIR ’24); Jul 14-18, 2024; Washington, DC, USA. pp. 719–729. Presented at. doi.</cite> [<a href="https://doi.org/10.1145/3626772.3657834" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?Cuconasu%20F,%20Trappolini%20G,%20Siciliano%20F,%20et%20al.%20The%20power%20of%20noise:%20redefining%20retrieval%20for%20RAG%20systems.%20Proceedings%20of%20the%2047th%20International%20ACM%20SIGIR%20Conference%20on%20Research%20and%20Development%20in%20Information%20Retrieval%20(SIGIR%20%E2%80%9924);%20Jul%2014-18,%202024;%20Washington,%20DC,%20USA.%20pp.%20719%E2%80%93729.%20Presented%20at.%20doi." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>Multimedia Appendix 1. Prompts for medicine food homology knowledge graph construction.</span></div>
<div class="media p">
<div class="caption">
<a href="/articles/instance/12370266/bin/medinform-v13-e75279-s001.docx" data-ga-action="click_feat_suppl" class="usa-link">medinform-v13-e75279-s001.docx</a><sup> (21.8KB, docx) </sup>
</div>
<div class="object-id font-xs text-base-dark">DOI: 10.2196/75279</div>
</div></section></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from JMIR Medical Informatics are provided here courtesy of <strong>JMIR Publications Inc.</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.2196/75279"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370266/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370266/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370266%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370266/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370266/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370266/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40840437/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370266/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40840437/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370266/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370266/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="OtT4CLGzj7kt3degp53SnediMNqImtjKK1m0ITtJrD86WJ2enEf7u1XIa1Z0dzLy">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
