
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            GNODEVAE: a graph-based ODE-VAE enhances clustering for single-cell data - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4A9ED8AF2A81305A9ED004749C0A7.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="bmcgeno">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369121/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="BMC Genomics">
<meta name="citation_title" content="GNODEVAE: a graph-based ODE-VAE enhances clustering for single-cell data">
<meta name="citation_author" content="Zeyu Fu">
<meta name="citation_author_institution" content="State Key Laboratory of Trauma and Chemical Poisoning, Institute of Combined Injury, Chongqing Engineering Research Center for Nanomedicine, College of Preventive Medicine, Army Medical University, Chongqing, 400038 China">
<meta name="citation_author" content="Chunlin Chen">
<meta name="citation_author_institution" content="Department of Rehabilitation Medicine, The First Affiliated Hospital, Sun Yat-sen University, Guangzhou, 510080 China">
<meta name="citation_author" content="Song Wang">
<meta name="citation_author_institution" content="State Key Laboratory of Trauma and Chemical Poisoning, Institute of Combined Injury, Chongqing Engineering Research Center for Nanomedicine, College of Preventive Medicine, Army Medical University, Chongqing, 400038 China">
<meta name="citation_author" content="Junping Wang">
<meta name="citation_author_institution" content="State Key Laboratory of Trauma and Chemical Poisoning, Institute of Combined Injury, Chongqing Engineering Research Center for Nanomedicine, College of Preventive Medicine, Army Medical University, Chongqing, 400038 China">
<meta name="citation_author" content="Shilei Chen">
<meta name="citation_author_institution" content="State Key Laboratory of Trauma and Chemical Poisoning, Institute of Combined Injury, Chongqing Engineering Research Center for Nanomedicine, College of Preventive Medicine, Army Medical University, Chongqing, 400038 China">
<meta name="citation_publication_date" content="2025 Aug 21">
<meta name="citation_volume" content="26">
<meta name="citation_firstpage" content="767">
<meta name="citation_doi" content="10.1186/s12864-025-11946-7">
<meta name="citation_pmid" content="40841610">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369121/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369121/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369121/pdf/12864_2025_Article_11946.pdf">
<meta name="description" content="Single-cell RNA sequencing analysis faces critical challenges including high dimensionality, sparsity, and complex topological relationships between cells. Current methods struggle to simultaneously preserve global structure, model cellular ...">
<meta name="og:title" content="GNODEVAE: a graph-based ODE-VAE enhances clustering for single-cell data">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Single-cell RNA sequencing analysis faces critical challenges including high dimensionality, sparsity, and complex topological relationships between cells. Current methods struggle to simultaneously preserve global structure, model cellular ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369121/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12369121">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12864-025-11946-7"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12864_2025_Article_11946.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369121%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12369121/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12369121/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369121/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-bmcgeno.png" alt="BMC Genomics logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to BMC Genomics" title="Link to BMC Genomics" shape="default" href="https://bmcgenomics.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">BMC Genomics</button></div>. 2025 Aug 21;26:767. doi: <a href="https://doi.org/10.1186/s12864-025-11946-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12864-025-11946-7</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22BMC%20Genomics%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22BMC%20Genomics%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22BMC%20Genomics%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>GNODEVAE: a graph-based ODE-VAE enhances clustering for single-cell data</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fu%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Zeyu Fu</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Zeyu Fu</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Trauma and Chemical Poisoning, Institute of Combined Injury, Chongqing Engineering Research Center for Nanomedicine, College of Preventive Medicine, Army Medical University, Chongqing, 400038 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Fu%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zeyu Fu</span></a>
</div>
</div>
<sup>1,</sup><sup>✉,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Chunlin Chen</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Chunlin Chen</span></h3>
<div class="p">
<sup>2</sup>Department of Rehabilitation Medicine, The First Affiliated Hospital, Sun Yat-sen University, Guangzhou, 510080 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chunlin Chen</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Song Wang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Song Wang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Trauma and Chemical Poisoning, Institute of Combined Injury, Chongqing Engineering Research Center for Nanomedicine, College of Preventive Medicine, Army Medical University, Chongqing, 400038 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Song Wang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Junping Wang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Junping Wang</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Trauma and Chemical Poisoning, Institute of Combined Injury, Chongqing Engineering Research Center for Nanomedicine, College of Preventive Medicine, Army Medical University, Chongqing, 400038 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Junping Wang</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Shilei Chen</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Shilei Chen</span></h3>
<div class="p">
<sup>1</sup>State Key Laboratory of Trauma and Chemical Poisoning, Institute of Combined Injury, Chongqing Engineering Research Center for Nanomedicine, College of Preventive Medicine, Army Medical University, Chongqing, 400038 China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chen%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Shilei Chen</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>State Key Laboratory of Trauma and Chemical Poisoning, Institute of Combined Injury, Chongqing Engineering Research Center for Nanomedicine, College of Preventive Medicine, Army Medical University, Chongqing, 400038 China </div>
<div id="Aff2">
<sup>2</sup>Department of Rehabilitation Medicine, The First Affiliated Hospital, Sun Yat-sen University, Guangzhou, 510080 China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 24; Accepted 2025 Jul 25; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12369121  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40841610/" class="usa-link">40841610</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Single-cell RNA sequencing analysis faces critical challenges including high dimensionality, sparsity, and complex topological relationships between cells. Current methods struggle to simultaneously preserve global structure, model cellular dynamics, and handle technical noise effectively.</p></section><section id="sec2"><h3 class="pmc_sec_title">Results</h3>
<p id="Par2">We present GNODEVAE, a novel architecture integrating Graph Attention Networks (GAT), Neural Ordinary Differential Equations (NODE), and Variational Autoencoders (VAE) for comprehensive single-cell analysis. Through systematic evaluation across 10 graph convolutional layers, GAT demonstrated optimal performance, achieving average ARI advantages of 0.108 and 0.112 over alternative graph convolutional layers in VGAE and GNODEVAE architectures respectively, along with ASW advantages of 0.047 and 0.098. Extensive comparison across 50 diverse single cell datasets against 18 existing methods demonstrates that GNODEVAE consistently outperforms three major categories of benchmark methods: 8 machine learning dimensionality reduction techniques, 7 deep generative VAE variants, and 3 graph-based and contrastive learning deep predictive models. GNODEVAE achieved average advantages of 0.112 in reconstruction clustering quality (ARI) and 0.113 in clustering geometry quality (ASW) over standard VGAE, with an average ASW advantage of 0.286 over all benchmark methods in clustering geometry quality. In gene dynamics clustering evaluation, GNODEVAE outperformed Diffusion map and Palantir methods across all geometric metrics.</p></section><section id="sec3"><h3 class="pmc_sec_title">Conclusions</h3>
<p id="Par3">GNODEVAE establishes a robust computational framework that synergistically combines neighborhood-awareness, dynamic modeling, and probabilistic expressiveness for single-cell multi-omics analysis. The consistent superior performance across diverse datasets demonstrates its effectiveness as a versatile tool for cell clustering, dimensionality reduction, and pseudotime trajectory analysis in both scRNA-seq and scATAC-seq data mining.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Graph attention networks, Neural ordinary differential equation, Variational autoencoders, Clustering, ScRNA-seq, ScATAC-seq</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Background</h2>
<p id="Par34">Single-cell RNA sequencing (scRNA-seq) technology offers unprecedented resolution for investigating cellular heterogeneity and state transitions by measuring gene expression at the individual cell level [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>]. Unlike bulk RNA sequencing, scRNA-seq distinguishes transcriptomic profiles among different cell types within complex tissues, enabling identification of rare cell populations, elucidation of regulatory factors, and reconstruction of cell fate determination trajectories. The applications of scRNA-seq have expanded rapidly across developmental biology, cancer research, immunology, and neuroscience, establishing it as a foundational technology for precision medicine and biological discovery.</p>
<p id="Par35">Single-cell RNA sequencing data presents significant analytical challenges due to its high dimensionality, sparsity, and technical noise [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. Core computational tasks include dimensionality reduction for visualization, clustering for subpopulation identification, and trajectory analysis for developmental pathway reconstruction. Current methods face three critical challenges: simultaneously preserving global structure and local relationships [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]; accurately modeling complex topological relationships between cells [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>]; and effectively mitigating technical noise and biological variation [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>].</p>
<p id="Par36">Traditional clustering approaches face significant limitations when applied to scRNA-seq data analysis. Classical methods such as K-means assume spherical cluster shapes and require pre-specification of cluster numbers, while hierarchical clustering suffers from computational complexity and noise sensitivity in high-dimensional sparse data [<a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>, <a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Despite its consensus-based approach, SC3 (Single-Cell Consensus Clustering) encounters scalability issues with datasets containing large numbers of cells and struggles with highly sparse data due to unreliable distance-based similarity measures [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>, <a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Seurat’s clustering workflow and spectral clustering methods face additional challenges with batch effects, parameter sensitivity, and computational bottlenecks in large datasets [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. Fundamentally, these conventional approaches treat cells as independent entities in Euclidean space, failing to model the continuous nature of cellular differentiation processes and complex topological structures that characterize developmental trajectories [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. The inability to effectively integrate structural relationships, handle uncertainty, and model dynamic processes necessitates more sophisticated approaches that can simultaneously address these interconnected challenges.</p>
<p id="Par37">Pseudotime analysis has evolved through several generations to address these challenges. First-generation methods focused on structure preservation through minimum spanning trees (Monocle [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]), diffusion maps (Diffusion Pseudotime [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]), and enhanced branch point detection (Wishbone [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]). Second-generation algorithms incorporated advanced graph theory: Monocle 3 [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>] implemented uniform manifold approximation and projection (UMAP) and principal graph analysis, Palantir [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>] applied Markov processes for fate decisions, and PAGA [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>] developed partition-based graph abstraction. Third-generation methods addressed noise and heterogeneity through transcript splicing analysis (RNA Velocity [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>]), optimal transport theory (Waddington-OT [<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>]), and combined velocity with transfer learning (CellRank [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]).</p>
<p id="Par38">Recent machine learning approaches have provided increasingly effective solutions. Deep autoencoder architectures demonstrate superior performance in capturing multi-scale structures. DCA [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>] simultaneously preserves macroscopic cell type distributions and microscopic state transitions, while scVI [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>] employs a variational framework that learns global structure and local variation through probabilistic latent representations. Graph learning methods explicitly model cell relationships: GraphSAINT [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>] balances computational efficiency with structure preservation, while scGNN [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>] iteratively optimizes cell graphs and feature representations.</p>
<p id="Par39">Graph Attention Networks (GAT) [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>] excel at capturing complex topologies by dynamically determining neighbor importance based on cell features, particularly at differentiation branch points. NODE [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] provide a continuous mathematical framework for modeling developmental processes. TrajectoryNet [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>] and scNODE [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>] have successfully applied NODEs to trajectory inference, while PRESCIENT [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>] integrates stochastic simulation to model developmental landscapes. Topological approaches further enhance structure preservation through TopoAE [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>], TGCN [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>], and CellPath [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>].</p>
<p id="Par40">Deep generative models effectively handle uncertainty through probabilistic frameworks. scVI [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>] models gene expression with specialized count distributions while learning batch covariates. Harmony [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>] employs iterative clustering and linear correction for batch effect removal, while scArches [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>] enables reference-based integration across datasets. Multi-task learning approaches enhance robustness: SAUCIE [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>] simultaneously performs dimensionality reduction, batch correction, and clustering; Totalvi [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>] jointly models RNA and protein expression; and DESC [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>] combines deep embedding with soft clustering for superior performance in noisy environments.</p>
<p id="Par41">Graph Neural Networks (GNNs) have emerged as powerful tools for processing non-Euclidean data by learning node representations through neighborhood information aggregation [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>]. Their key advantage lies in processing irregular structural data and explicitly modeling entity relationships [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. In single-cell analysis, cells are represented as graph nodes with similarity-based edges, enabling GNNs to leverage intercellular relationships for representation learning [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. This approach excels at capturing complex topological structures such as differentiation branch points, transitional states, and rare subpopulations, while mitigating noise and batch effects.</p>
<p id="Par42">Variational Autoencoders (VAE) are probabilistic generative models that learn data’s latent probability distribution [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. VAEs offer distinct advantages for single-cell analysis: their probabilistic framework naturally handles technical and biological noise; continuous latent space effectively models gradual cell state transitions; generative capabilities enable simulation of hypothetical cell states; and they can integrate prior biological knowledge [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Several specialized implementations demonstrate VAE’s utility: scVI [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>] models gene expression with zero-inflated negative binomial distributions while handling batch effects; trVAE predicts cellular responses to perturbations [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]; and MM-VAE integrates multi-modal data for comprehensive cell state understanding [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>].</p>
<p id="Par43">The integration of GNNs and VAEs represents a significant advancement for analyzing complex structured data [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>], creating a powerful framework that leverages both structural and probabilistic data characteristics. Variational Graph Autoencoders (VGAE) exemplify this fusion, using graph convolutional networks as encoders while reconstructing graph structures through decoders. This integration offers complementary advantages: GNNs capture topological structure while VAEs provide probabilistic frameworks and generative capabilities; GNN message passing mitigates data noise and sparsity while VAE latent space constraints generate meaningful representations [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. Recent advances demonstrate VGAE’s superior performance: scGNN [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>] iteratively optimizes cell graphs and feature representations; GraphSCI integrates multiple cell interaction types for multi-omics analysis [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]; and scVGAE significantly improves subpopulation identification accuracy [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>].</p>
<p id="Par44">NODE [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>] combine deep learning with differential equation theory to model continuous dynamic processes, parameterizing neural networks as continuous dynamical systems. Unlike traditional networks with fixed layer numbers, NODEs calculate state changes through numerical solvers. The NODE-VAE architecture combines these approaches to learn continuous trajectories in latent space [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>], offering key advantages for single-cell analysis: handling irregularly sampled temporal data [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]; modeling continuous developmental processes [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]; capturing uncertainties through probabilistic frameworks [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]; and generating hypothetical cell states at different timepoints [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Applications in pseudotime analysis have shown promising results: TrajectoryNet [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>] models developmental trajectories and predicts intermediate cell states, while scNODE incorporates cell type prior knowledge to guide trajectory learning [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>].</p>
<p id="Par45">However, traditional NODE-VAE approaches have limitations when processing complex topological structures, as they typically view cells as independent entities and struggle with highly heterogeneous populations. The integration of graph neural networks, neural ordinary differential equations, and variational autoencoders represents a promising direction for simultaneously addressing all three challenges through complementary strengths in topology representation, dynamics modeling, and uncertainty handling. Our proposed GNODEVAE framework implements this integrated approach, combining GAT’s relationship modeling, NODE’s continuous dynamics representation, and VAE’s probabilistic inference.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">GNODEVAE model overview</h3>
<p id="Par46">We propose GNODEVAE, a graph-based NODE variational autoencoder that integrates graph neural networks, NODEs, and probabilistic inference to model the temporal evolution of graph-structured single-cell data. Unlike traditional variational autoencoders, GNODEVAE infers a continuous-time representation by estimating a time variable <em>t</em> for each node, enabling the modeling of dynamic processes on graphs without explicit temporal annotations.</p>
<p id="Par47">The model consists of three main components: 1) an encoder that maps input features to a latent space and estimates time variables, 2) a NODE that models the continuous dynamics in the latent space, and 3) decoders that reconstruct both node features and graph structure.</p>
<section id="Sec4"><h4 class="pmc_sec_title">Encoder with time inference</h4>
<p id="Par48">Our framework supports two encoder variants: a graph-based encoder and a linear encoder, both extended to estimate a time variable for each node.</p>
<p id="Par49">The graph encoder utilizes a sequence of graph convolutional layers to process node features while accounting for graph topology. Given an input graph with <em>n</em> nodes, <em>d</em> node features, and <em>m</em> edges, we define node features <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/fd6b67878d31/d33e601.gif" loading="lazy" id="d33e601" alt="Inline graphic"></span>, edge indices <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/6bbe8fa244c2/d33e607.gif" loading="lazy" id="d33e607" alt="Inline graphic"></span>, and optional edge weights <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/74ddd4bac83d/d33e614.gif" loading="lazy" id="d33e614" alt="Inline graphic"></span>.</p>
<p id="Par50">The encoder processes the input through <em>L</em> graph convolutional layers:</p>
<table class="disp-formula p" id="Equ1"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/127e91f7588a/d33e625.gif" loading="lazy" id="d33e625" alt="graphic file with name d33e625.gif"></td>
<td class="label">1</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/1908f66bd2f2/d33e632.gif" loading="lazy" id="d33e632" alt="Inline graphic"></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/2aabd379ce4a/d33e638.gif" loading="lazy" id="d33e638" alt="Inline graphic"></span> is a ReLU activation function, BN denotes batch normalization, and GNN represents a graph neural network layer. Our implementation supports various graph convolutional architectures, including GAT, Graph Convolutional Networks (GCN), Chebyshev Graph Convolutional Networks (ChebNet), Graph Sample and Aggregate (GraphSAGE), Topology Adaptive Graph Convolutional Networks (TAGConv), AutoRegressive Moving Average (ARMA), Graph Transformers, Simple Graph Convolution (SGConv), and Simple Spectral Graph Convolution (SSGConv).</p>
<p id="Par51">To enhance gradient flow, we implement a residual connection from the first layer’s output:</p>
<table class="disp-formula p" id="Equ2"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/b5f4f2639b3d/d33e646.gif" loading="lazy" id="d33e646" alt="graphic file with name d33e646.gif"></td>
<td class="label">2</td>
</tr></table>
<p id="Par52">From the final hidden representation <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/7695c7fa69ae/d33e654.gif" loading="lazy" id="d33e654" alt="Inline graphic"></span>, we compute the mean and log-variance of the latent distribution:</p>
<table class="disp-formula p" id="Equ3"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/44ad7e14161d/d33e660.gif" loading="lazy" id="d33e660" alt="graphic file with name d33e660.gif"></td>
<td class="label">3</td>
</tr></table>
<table class="disp-formula p" id="Equ4"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01b46ba12bc9/d33e666.gif" loading="lazy" id="d33e666" alt="graphic file with name d33e666.gif"></td>
<td class="label">4</td>
</tr></table>
<p>and the time variable for each node:</p>
<table class="disp-formula p" id="Equ5"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/fca14da0373b/d33e673.gif" loading="lazy" id="d33e673" alt="graphic file with name d33e673.gif"></td>
<td class="label">5</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq7"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/8834842c2581/d33e680.gif" loading="lazy" id="d33e680" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq8"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/a17349517377/d33e686.gif" loading="lazy" id="d33e686" alt="Inline graphic"></span> are learnable parameters, <em>h</em> is the hidden dimension, and the sigmoid activation ensures <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq9"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/8c3bc163f74a/d33e696.gif" loading="lazy" id="d33e696" alt="Inline graphic"></span> for each node <em>i</em>.</p>
<p id="Par53">For datasets without graph structure, we provide a linear encoder variant that processes features through linear layers with the same output structure.</p>
<p id="Par54">The latent representation <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq10"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/afeeb888ade3/d33e709.gif" loading="lazy" id="d33e709" alt="Inline graphic"></span> is sampled using the reparameterization trick:</p>
<table class="disp-formula p" id="Equ6"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/33e1846c62d1/d33e715.gif" loading="lazy" id="d33e715" alt="graphic file with name d33e715.gif"></td>
<td class="label">6</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq11"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/17d61a6ef591/d33e722.gif" loading="lazy" id="d33e722" alt="Inline graphic"></span> ensures numerical stability.</p></section><section id="Sec5"><h4 class="pmc_sec_title">NODE component</h4>
<p id="Par55">The NODE models the continuous-time dynamics in the latent space. Given the inferred latent representations <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq12"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/601a807055bf/d33e732.gif" loading="lazy" id="d33e732" alt="Inline graphic"></span> and time variables <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq13"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/67f29668e433/d33e738.gif" loading="lazy" id="d33e738" alt="Inline graphic"></span>, we sort and filter the data points by time to obtain a sequence of <em>K</em> unique time points <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq14"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/6d4ae5aa5201/d33e747.gif" loading="lazy" id="d33e747" alt="Inline graphic"></span> and corresponding latent states <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq15"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/47535dc50d72/d33e753.gif" loading="lazy" id="d33e753" alt="Inline graphic"></span>.</p>
<p id="Par56">The NODE is defined by:</p>
<table class="disp-formula p" id="Equ7"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/cfb3af9f3918/d33e761.gif" loading="lazy" id="d33e761" alt="graphic file with name d33e761.gif"></td>
<td class="label">7</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq16"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/6c0ff336c5eb/d33e768.gif" loading="lazy" id="d33e768" alt="Inline graphic"></span> is parameterized as a multi-layer perceptron:</p>
<table class="disp-formula p" id="Equ8"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/15908f91b594/d33e774.gif" loading="lazy" id="d33e774" alt="graphic file with name d33e774.gif"></td>
<td class="label">8</td>
</tr></table>
<p id="Par57">Given the initial state <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq17"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/4728a9b07817/d33e782.gif" loading="lazy" id="d33e782" alt="Inline graphic"></span>, we solve this ODE system using the Euler method with adaptive step sizes to obtain the trajectory:</p>
<table class="disp-formula p" id="Equ9"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/b400219f91f1/d33e788.gif" loading="lazy" id="d33e788" alt="graphic file with name d33e788.gif"></td>
<td class="label">9</td>
</tr></table></section><section id="Sec6"><h4 class="pmc_sec_title">Decoder architecture</h4>
<p id="Par58">GNODEVAE employs two decoders: a structure decoder for reconstructing graph connectivity and a feature decoder for reconstructing node features.</p>
<p id="Par59">The structure decoder reconstructs the adjacency matrix by computing pairwise interactions between nodes:</p>
<table class="disp-formula p" id="Equ10"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/d7f33d6da45e/d33e800.gif" loading="lazy" id="d33e800" alt="graphic file with name d33e800.gif"></td>
<td class="label">10</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq18"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/d807e01f4c7e/d33e807.gif" loading="lazy" id="d33e807" alt="Inline graphic"></span> is implemented as an multi-layer perceptron (MLP)-based function:</p>
<table class="disp-formula p" id="Equ11"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/927e0425537b/d33e813.gif" loading="lazy" id="d33e813" alt="graphic file with name d33e813.gif"></td>
<td class="label">11</td>
</tr></table>
<p id="Par60">The output <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq19"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/b989952677ee/d33e821.gif" loading="lazy" id="d33e821" alt="Inline graphic"></span> represents logits that are passed through a sigmoid function during loss computation. For sparse graphs, we apply a threshold <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq20"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/61c6e424cbc5/d33e827.gif" loading="lazy" id="d33e827" alt="Inline graphic"></span> to retain only significant connections.</p>
<p id="Par61">The feature decoder reconstructs node features from latent representations:</p>
<table class="disp-formula p" id="Equ12"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/a1f4d3888424/d33e835.gif" loading="lazy" id="d33e835" alt="graphic file with name d33e835.gif"></td>
<td class="label">12</td>
</tr></table>
<p id="Par62">This reconstruction is applied to both the VAE latent codes <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq21"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/afeeb888ade3/d33e843.gif" loading="lazy" id="d33e843" alt="Inline graphic"></span> and the ODE-generated trajectories <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq22"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/42759f6f6b04/d33e849.gif" loading="lazy" id="d33e849" alt="Inline graphic"></span>.</p></section><section id="Sec7"><h4 class="pmc_sec_title">Training objective</h4>
<p id="Par63">We train GNODEVAE using a weighted combination of five loss terms:</p>
<table class="disp-formula p" id="Equ13"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/07269ff32e68/d33e859.gif" loading="lazy" id="d33e859" alt="graphic file with name d33e859.gif"></td>
<td class="label">13</td>
</tr></table>
<p id="Par64">The reconstruction loss for single-cell RNA-seq data uses a negative binomial distribution with library size normalization:</p>
<table class="disp-formula p" id="Equ14"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/4ca1f59e76b7/d33e867.gif" loading="lazy" id="d33e867" alt="graphic file with name d33e867.gif"></td>
<td class="label">14</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq23"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/73bc52e64d5d/d33e874.gif" loading="lazy" id="d33e874" alt="Inline graphic"></span> with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq24"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/fd0ee58a0949/d33e880.gif" loading="lazy" id="d33e880" alt="Inline graphic"></span> being the library size, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq25"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/ac623e20d0f8/d33e886.gif" loading="lazy" id="d33e886" alt="Inline graphic"></span> is the learned dispersion parameter.</p>
<p id="Par65">The Kullback-Leibler (KL) divergence term regularizes the latent space:</p>
<table class="disp-formula p" id="Equ15"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/bf08aa5a3b96/d33e894.gif" loading="lazy" id="d33e894" alt="graphic file with name d33e894.gif"></td>
<td class="label">15</td>
</tr></table>
<p id="Par66">The graph reconstruction loss uses binary cross-entropy with logits:</p>
<table class="disp-formula p" id="Equ16"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/62d51eff5f92/d33e902.gif" loading="lazy" id="d33e902" alt="graphic file with name d33e902.gif"></td>
<td class="label">16</td>
</tr></table>
<p id="Par67">The ODE reconstruction loss applies the same negative binomial formulation to the trajectory points:</p>
<table class="disp-formula p" id="Equ17"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/9a69d781c0ff/d33e910.gif" loading="lazy" id="d33e910" alt="graphic file with name d33e910.gif"></td>
<td class="label">17</td>
</tr></table>
<p>where <em>K</em> is the number of unique time points after filtering, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq26"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/405ba0388189/d33e920.gif" loading="lazy" id="d33e920" alt="Inline graphic"></span> represents the corresponding input features, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq27"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/c7e6cc02f361/d33e926.gif" loading="lazy" id="d33e926" alt="Inline graphic"></span>.</p>
<p id="Par68">The latent space divergence loss measures consistency between the VAE latent space and ODE trajectory:</p>
<table class="disp-formula p" id="Equ18"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/50c464bc285b/d33e934.gif" loading="lazy" id="d33e934" alt="graphic file with name d33e934.gif"></td>
<td class="label">18</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq28"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/1db49dc3aea0/d33e941.gif" loading="lazy" id="d33e941" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq29"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/cb1db5ea1915/d33e947.gif" loading="lazy" id="d33e947" alt="Inline graphic"></span> represent the mean latent representations and ODE trajectory points corresponding to the filtered unique time points, respectively.</p>
<p id="Par69">The model is optimized using the Adam optimizer. During training, time variables are sorted and filtered to obtain unique time points for ODE solving, and all loss components are weighted according to the specified hyperparameters.</p>
<p id="Par70">This study benchmarks computational methods for learning a low-dimensional embedding <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq30"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/05c9a236cc8c/d33e957.gif" loading="lazy" id="d33e957" alt="Inline graphic"></span> from a single-cell gene expression matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq31"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/531a273ab4df/d33e963.gif" loading="lazy" id="d33e963" alt="Inline graphic"></span>, where <em>N</em> is the number of cells, <em>D</em> is the number of genes, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq32"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/d83058a601a2/d33e975.gif" loading="lazy" id="d33e975" alt="Inline graphic"></span> is the latent dimension. The methods are grouped into three categories.</p></section></section><section id="Sec8"><h3 class="pmc_sec_title">Dimensionality reduction machine learning</h3>
<section id="Sec9"><h4 class="pmc_sec_title">Principal component analysis (PCA)</h4>
<p id="Par71">PCA identifies a linear projection that maximizes the variance in the projected data. It finds the eigenvectors of the data’s covariance matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq33"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/a4554c8a13b3/d33e987.gif" loading="lazy" id="d33e987" alt="Inline graphic"></span> (assuming mean-centered data). The objective is:</p>
<table class="disp-formula p" id="Equ19"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/4a374e9f3763/d33e993.gif" loading="lazy" id="d33e993" alt="graphic file with name d33e993.gif"></td>
<td class="label">19</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq34"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/d017740f3f71/d33e1000.gif" loading="lazy" id="d33e1000" alt="Inline graphic"></span> is the projection matrix containing the top <em>d</em> eigenvectors, and the embedding is <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq35"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/a8ec9ece3f57/d33e1009.gif" loading="lazy" id="d33e1009" alt="Inline graphic"></span>.</p></section><section id="Sec10"><h4 class="pmc_sec_title">Independent component analysis (ICA)</h4>
<p id="Par72">ICA aims to linearly separate the mixed observed signals <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq36"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/0a12696f95c2/d33e1019.gif" loading="lazy" id="d33e1019" alt="Inline graphic"></span> into a set of statistically independent source signals <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq37"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/ed16e193612f/d33e1025.gif" loading="lazy" id="d33e1025" alt="Inline graphic"></span> by finding an unmixing matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq38"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/d017740f3f71/d33e1031.gif" loading="lazy" id="d33e1031" alt="Inline graphic"></span>. The model assumes <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq39"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/1448827dd5b3/d33e1037.gif" loading="lazy" id="d33e1037" alt="Inline graphic"></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq40"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/b5df4d6bf12f/d33e1043.gif" loading="lazy" id="d33e1043" alt="Inline graphic"></span> is the mixing matrix. The goal is to find <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq41"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/8a4d07dabda6/d33e1050.gif" loading="lazy" id="d33e1050" alt="Inline graphic"></span> such that the components of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq42"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/a53d13187d59/d33e1056.gif" loading="lazy" id="d33e1056" alt="Inline graphic"></span> are as independent as possible, typically by maximizing their non-Gaussianity.</p></section><section id="Sec11"><h4 class="pmc_sec_title">Non-negative matrix factorization (NMF)</h4>
<p id="Par73">NMF decomposes the non-negative expression matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq43"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/0a12696f95c2/d33e1066.gif" loading="lazy" id="d33e1066" alt="Inline graphic"></span> into two non-negative matrices: a matrix of basis vectors or"metagenes"<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq44"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/a94a6b8698d8/d33e1072.gif" loading="lazy" id="d33e1072" alt="Inline graphic"></span> and a matrix of coefficients or cell loadings <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq45"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/4726a6df5eda/d33e1078.gif" loading="lazy" id="d33e1078" alt="Inline graphic"></span>.</p>
<table class="disp-formula p" id="Equ20"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/361e06a1e8f2/d33e1084.gif" loading="lazy" id="d33e1084" alt="graphic file with name d33e1084.gif"></td>
<td class="label">20</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq46"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/46cc3d830fcb/d33e1091.gif" loading="lazy" id="d33e1091" alt="Inline graphic"></span> is the Frobenius norm.</p></section><section id="Sec12"><h4 class="pmc_sec_title">Factor analysis (FA)</h4>
<p id="Par74">This model assumes that the observed data <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq47"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/f824da3435d0/d33e1101.gif" loading="lazy" id="d33e1101" alt="Inline graphic"></span> (for cell <em>i</em>) is a linear combination of a small number of unobserved latent factors <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq48"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/78e8abd9b4ab/d33e1110.gif" loading="lazy" id="d33e1110" alt="Inline graphic"></span>, plus Gaussian noise <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq49"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/ca0db37b795e/d33e1116.gif" loading="lazy" id="d33e1116" alt="Inline graphic"></span>.</p>
<table class="disp-formula p" id="Equ21"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/6ab78c4fa8a5/d33e1122.gif" loading="lazy" id="d33e1122" alt="graphic file with name d33e1122.gif"></td>
<td class="label">21</td>
</tr></table>
<p id="Par75">Here, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq50"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/522dcb2e5063/d33e1130.gif" loading="lazy" id="d33e1130" alt="Inline graphic"></span> is the data mean, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq51"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/aa56b919a624/d33e1136.gif" loading="lazy" id="d33e1136" alt="Inline graphic"></span> is the factor loading matrix, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq52"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/ddc5a38f758b/d33e1142.gif" loading="lazy" id="d33e1142" alt="Inline graphic"></span> is a diagonal matrix of noise covariances.</p></section><section id="Sec13"><h4 class="pmc_sec_title">Truncated singular value decomposition (SVD)</h4>
<p id="Par76">Truncated SVD provides a low-rank approximation of the data matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq53"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/0a12696f95c2/d33e1152.gif" loading="lazy" id="d33e1152" alt="Inline graphic"></span> by retaining the top <em>d</em> singular values and their corresponding singular vectors.</p>
<table class="disp-formula p" id="Equ22"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/59f4388586ec/d33e1161.gif" loading="lazy" id="d33e1161" alt="graphic file with name d33e1161.gif"></td>
<td class="label">22</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq54"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/7e18745a0060/d33e1168.gif" loading="lazy" id="d33e1168" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq55"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/023ac5b9c84b/d33e1174.gif" loading="lazy" id="d33e1174" alt="Inline graphic"></span> are orthogonal matrices of the top <em>d</em> singular vectors and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq56"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/fd58db8b2cc4/d33e1184.gif" loading="lazy" id="d33e1184" alt="Inline graphic"></span> is a diagonal matrix of the top <em>d</em> singular values. The embedding is typically <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq57"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/626e95c890af/d33e1193.gif" loading="lazy" id="d33e1193" alt="Inline graphic"></span>.</p></section><section id="Sec14"><h4 class="pmc_sec_title">Dictionary learning (DICL)</h4>
<p id="Par77">This method learns a dictionary of basis vectors <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq58"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/d7044cd39d99/d33e1203.gif" loading="lazy" id="d33e1203" alt="Inline graphic"></span> (atoms) and a sparse code matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq59"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/3cbd37253727/d33e1209.gif" loading="lazy" id="d33e1209" alt="Inline graphic"></span> to represent the data.</p>
<table class="disp-formula p" id="Equ23"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/67f34187727e/d33e1215.gif" loading="lazy" id="d33e1215" alt="graphic file with name d33e1215.gif"></td>
<td class="label">23</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq60"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/48c78f82213c/d33e1222.gif" loading="lazy" id="d33e1222" alt="Inline graphic"></span> is the sparse code for cell <em>i</em> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq61"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/95c7c0c413ff/d33e1232.gif" loading="lazy" id="d33e1232" alt="Inline graphic"></span> is a regularization parameter controlling sparsity.</p></section><section id="Sec15"><h4 class="pmc_sec_title">Diffusion map</h4>
<p id="Par78">A non-linear manifold learning technique that constructs a diffusion operator on a data graph. Given a kernel matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq62"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/11c666e08a22/d33e1242.gif" loading="lazy" id="d33e1242" alt="Inline graphic"></span>, a diagonal degree matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq63"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/0927ad1b6f15/d33e1248.gif" loading="lazy" id="d33e1248" alt="Inline graphic"></span> is formed. The transition matrix is <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq64"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/35e3f0a57570/d33e1254.gif" loading="lazy" id="d33e1254" alt="Inline graphic"></span>. The embedding for cell <em>i</em> at diffusion time <em>t</em> is:</p>
<table class="disp-formula p" id="Equ24"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/ee8b701f2099/d33e1267.gif" loading="lazy" id="d33e1267" alt="graphic file with name d33e1267.gif"></td>
<td class="label">24</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq65"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/68b04e206d0e/d33e1274.gif" loading="lazy" id="d33e1274" alt="Inline graphic"></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq66"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/12f478557c2e/d33e1280.gif" loading="lazy" id="d33e1280" alt="Inline graphic"></span> are the <em>j</em>-th eigenvalue and eigenvector of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq67"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/75fe7d1091fa/d33e1289.gif" loading="lazy" id="d33e1289" alt="Inline graphic"></span>, respectively. PALANTIR builds upon this framework to model cell trajectories [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>].</p></section></section><section id="Sec16"><h3 class="pmc_sec_title">Generative models (VAE variants)</h3>
<p id="Par79">Variational Autoencoders are generative models that learn a mapping from the data <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq68"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/0a12696f95c2/d33e1302.gif" loading="lazy" id="d33e1302" alt="Inline graphic"></span> to a latent space <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq69"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/4780b67a973e/d33e1308.gif" loading="lazy" id="d33e1308" alt="Inline graphic"></span>. They consist of an encoder <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq70"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/f00d209907de/d33e1314.gif" loading="lazy" id="d33e1314" alt="Inline graphic"></span> and a decoder <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq71"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/0c70d59dad97/d33e1320.gif" loading="lazy" id="d33e1320" alt="Inline graphic"></span>, and are trained by maximizing the Evidence Lower Bound (ELBO):</p>
<table class="disp-formula p" id="Equ25"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/708964d5d2ad/d33e1326.gif" loading="lazy" id="d33e1326" alt="graphic file with name d33e1326.gif"></td>
<td class="label">25</td>
</tr></table>
<p>where the first term is the reconstruction log-likelihood and the second is the KL divergence between the approximate posterior and a prior <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq72"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/0638bbb5e8cd/d33e1334.gif" loading="lazy" id="d33e1334" alt="Inline graphic"></span> (usually <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq73"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/52efcd3a91ae/d33e1340.gif" loading="lazy" id="d33e1340" alt="Inline graphic"></span>).</p>
<section id="Sec17"><h4 class="pmc_sec_title">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq74"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/f5e0e8aa5fab/d33e1348.gif" loading="lazy" id="d33e1348" alt="Inline graphic"></span>-VAE</h4>
<p id="Par80">This variant introduces a hyperparameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq75"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/f5e0e8aa5fab/d33e1356.gif" loading="lazy" id="d33e1356" alt="Inline graphic"></span> to control the capacity of the latent information bottleneck. A larger <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq76"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/f5e0e8aa5fab/d33e1362.gif" loading="lazy" id="d33e1362" alt="Inline graphic"></span> encourages a more disentangled latent space.</p>
<table class="disp-formula p" id="Equ26"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/b9c896a443cf/d33e1368.gif" loading="lazy" id="d33e1368" alt="graphic file with name d33e1368.gif"></td>
<td class="label">26</td>
</tr></table></section><section id="Sec18"><h4 class="pmc_sec_title">Total correlation VAE (TC-VAE)</h4>
<p id="Par81">TC-VAE penalizes statistical dependence among the dimensions of the latent space by isolating the Total Correlation (TC) term from the KL divergence. The objective includes a penalty on TC, defined as:</p>
<table class="disp-formula p" id="Equ27"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/4a0b3c52175f/d33e1378.gif" loading="lazy" id="d33e1378" alt="graphic file with name d33e1378.gif"></td>
<td class="label">27</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq77"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/1ca7158f0e31/d33e1385.gif" loading="lazy" id="d33e1385" alt="Inline graphic"></span> is the aggregated posterior.</p></section><section id="Sec19"><h4 class="pmc_sec_title">INFO-VAE</h4>
<p id="Par82">INFO-VAE improves training stability and representation quality by adding a Maximum Mean Discrepancy (MMD) term to the objective, which measures the distance between the aggregated posterior <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq78"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/ef355f7c1e9e/d33e1395.gif" loading="lazy" id="d33e1395" alt="Inline graphic"></span> and the prior <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq79"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/0638bbb5e8cd/d33e1401.gif" loading="lazy" id="d33e1401" alt="Inline graphic"></span>.</p>
<table class="disp-formula p" id="Equ28"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/764de580ed25/d33e1407.gif" loading="lazy" id="d33e1407" alt="graphic file with name d33e1407.gif"></td>
<td class="label">28</td>
</tr></table>
<p>where <em>k</em> is a positive definite kernel.</p></section><section id="Sec20"><h4 class="pmc_sec_title">DIP-VAE</h4>
<p id="Par83">Disentangled Inferred Prior VAE regularizes the aggregated posterior <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq80"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/ef355f7c1e9e/d33e1421.gif" loading="lazy" id="d33e1421" alt="Inline graphic"></span> by explicitly penalizing its covariance matrix to encourage it to be diagonal. The penalty term is:</p>
<table class="disp-formula p" id="Equ29"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/b4b7add78356/d33e1427.gif" loading="lazy" id="d33e1427" alt="graphic file with name d33e1427.gif"></td>
<td class="label">29</td>
</tr></table>
<p id="Par84">This forces the latent factors to be uncorrelated.</p></section><section id="Sec21"><h4 class="pmc_sec_title">scVI</h4>
<p id="Par85">scVI adapts the VAE framework to scRNA-seq data by using a Zero-Inflated Negative Binomial (ZINB) distribution for the decoder’s likelihood, accounting for high dropout rates and overdispersion. The ZINB probability mass function is:</p>
<table class="disp-formula p" id="Equ30"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/de51daec7f50/d33e1439.gif" loading="lazy" id="d33e1439" alt="graphic file with name d33e1439.gif"></td>
<td class="label">30</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq81"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/f21e7a500209/d33e1446.gif" loading="lazy" id="d33e1446" alt="Inline graphic"></span> is the zero-inflation probability, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq82"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/522dcb2e5063/d33e1452.gif" loading="lazy" id="d33e1452" alt="Inline graphic"></span> is the mean, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq83"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/edd700580a15/d33e1458.gif" loading="lazy" id="d33e1458" alt="Inline graphic"></span> is the dispersion of the Negative Binomial (NB) component, all learned from the latent variable <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq84"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/afeeb888ade3/d33e1464.gif" loading="lazy" id="d33e1464" alt="Inline graphic"></span> [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>].</p></section><section id="Sec22"><h4 class="pmc_sec_title">scTour</h4>
<p id="Par86">This model uses a NODE to learn a continuous vector field in the latent space, modeling cellular dynamics and trajectories.</p>
<table class="disp-formula p" id="Equ31"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/0f4b84b9db3e/d33e1478.gif" loading="lazy" id="d33e1478" alt="graphic file with name d33e1478.gif"></td>
<td class="label">31</td>
</tr></table>
<p>where <em>f</em> is a neural network parameterized by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq85"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/edd700580a15/d33e1488.gif" loading="lazy" id="d33e1488" alt="Inline graphic"></span> that defines the latent dynamics [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>].</p></section></section><section id="Sec23"><h3 class="pmc_sec_title">Predictive models (Contrastive and graph learning)</h3>
<p id="Par87">These methods learn representations by discriminating between similar and dissimilar cells in a self-supervised manner. Most use the InfoNCE loss:</p>
<table class="disp-formula p" id="Equ32"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/2782b636b6e9/d33e1501.gif" loading="lazy" id="d33e1501" alt="graphic file with name d33e1501.gif"></td>
<td class="label">32</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq86"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/1c269efe12f7/d33e1508.gif" loading="lazy" id="d33e1508" alt="Inline graphic"></span> is the embedding of an anchor cell (query), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq87"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/41949cce6ef8/d33e1514.gif" loading="lazy" id="d33e1514" alt="Inline graphic"></span> is the embedding of a similar cell (positive key), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq88"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/69a4ae95d189/d33e1520.gif" loading="lazy" id="d33e1520" alt="Inline graphic"></span> are embeddings of dissimilar cells (negative keys), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq89"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/65f2b1ffbe41/d33e1526.gif" loading="lazy" id="d33e1526" alt="Inline graphic"></span> is the similarity function, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq90"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/61c6e424cbc5/d33e1533.gif" loading="lazy" id="d33e1533" alt="Inline graphic"></span> is the temperature parameter.</p>
<section id="Sec24"><h4 class="pmc_sec_title">scGCC</h4>
<p id="Par88">scGCC is a graph contrastive learning method. It first builds a k-NN graph from the expression data. Positive pairs are formed by a cell and its augmented version (via feature masking and noise). The model uses a GAT Network encoder within a Momentum Contrast (MoCo) framework to optimize the InfoNCE loss, using other cells in the batch and a dynamic queue as negative keys [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>].</p></section><section id="Sec25"><h4 class="pmc_sec_title">CLEAR</h4>
<p id="Par89">CLEAR learns representations by maximizing agreement between two differently augmented views of the same cell, without an explicit graph. Each cell profile is stochastically augmented to create a positive pair <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq91"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/496925cdc85c/d33e1550.gif" loading="lazy" id="d33e1550" alt="Inline graphic"></span>. It uses a standard MLP encoder and the MoCo framework to optimize the InfoNCE loss against negative keys from other cells [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>].</p></section><section id="Sec26"><h4 class="pmc_sec_title">scGNN</h4>
<p id="Par90">scGNN is a VAE that incorporates graph-based regularization. Its loss function includes the standard VAE terms plus a graph Laplacian regularizer that encourages connected cells in a k-NN graph to have similar reconstructions.</p>
<table class="disp-formula p" id="Equ33"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/5c88bcdfc389/d33e1563.gif" loading="lazy" id="d33e1563" alt="graphic file with name d33e1563.gif"></td>
<td class="label">33</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq92"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/ce41b6aba1c6/d33e1570.gif" loading="lazy" id="d33e1570" alt="Inline graphic"></span> is the reconstruction of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq93"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/f824da3435d0/d33e1576.gif" loading="lazy" id="d33e1576" alt="Inline graphic"></span>, <em>E</em> is the set of graph edges, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq94"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/0aa8ae52fa3b/d33e1585.gif" loading="lazy" id="d33e1585" alt="Inline graphic"></span> is the edge weight. It uses an EM-like procedure, alternating between updating the latent embeddings and rebuilding the graph based on the new embeddings [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>].</p></section></section><section id="Sec27"><h3 class="pmc_sec_title">Evaluation metrics</h3>
<p id="Par91">The performance of each embedding was assessed using a suite of metrics evaluating both the preservation of biological cell identity and the effective removal of non-biological batch effects.</p>
<section id="Sec28"><h4 class="pmc_sec_title">Adjusted rand index (ARI)</h4>
<p id="Par92">The ARI measures the similarity between the true cell labels (<em>C</em>) and the predicted cluster assignments (<em>K</em>), correcting for agreements that may occur by chance. An ARI of 1 indicates perfect agreement, while 0 corresponds to random labeling. It is calculated as:</p>
<table class="disp-formula p" id="Equ34"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/5287b790bde7/d33e1609.gif" loading="lazy" id="d33e1609" alt="graphic file with name d33e1609.gif"></td>
<td class="label">34</td>
</tr></table>
<p>where the Index is the standard Rand Index, which counts the fraction of cell pairs that are either in the same group in both <em>C</em> and <em>K</em> or in different groups in both clusterings.</p></section><section id="Sec29"><h4 class="pmc_sec_title">Normalized mutual information (NMI)</h4>
<p id="Par93">NMI is an information-theoretic metric that quantifies the mutual dependence between the true labels and the predicted clusters. It is normalized by the entropies of <em>C</em> and <em>K</em> to scale the result between 0 (no mutual information) and 1 (perfect correlation).</p>
<table class="disp-formula p" id="Equ35"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/078eccba7e98/d33e1632.gif" loading="lazy" id="d33e1632" alt="graphic file with name d33e1632.gif"></td>
<td class="label">35</td>
</tr></table>
<p id="Par94">Here, <em>I</em>(<em>C</em>, <em>K</em>) is the mutual information, and <em>H</em>(<em>C</em>) and <em>H</em>(<em>K</em>) are the entropies of the respective label sets.</p></section><section id="Sec30"><h4 class="pmc_sec_title">Average silhouette width (ASW)</h4>
<p id="Par95">ASW quantifies the quality of clustering by measuring how similar a cell is to its own cluster compared to other clusters. For a single cell <em>i</em>, the silhouette value <em>s</em>(<em>i</em>) is defined as:</p>
<table class="disp-formula p" id="Equ36"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/a30a76774f56/d33e1675.gif" loading="lazy" id="d33e1675" alt="graphic file with name d33e1675.gif"></td>
<td class="label">36</td>
</tr></table>
<p>where <em>a</em>(<em>i</em>) is the mean intra-cluster distance for cell <em>i</em>, and <em>b</em>(<em>i</em>) is its mean nearest-cluster distance. The ASW is the average of <em>s</em>(<em>i</em>) over all cells, with higher values indicating better-defined clusters.</p></section><section id="Sec31"><h4 class="pmc_sec_title">Calinski-Harabasz (CAL) score</h4>
<p id="Par96">Also known as the variance ratio criterion, the CAL score is the ratio of the sum of between-cluster dispersion to the sum of within-cluster dispersion. A higher CAL score indicates that clusters are dense and well separated.</p></section><section id="Sec32"><h4 class="pmc_sec_title">Davies-Bouldin (DAV) score</h4>
<p id="Par97">The DAV score measures the average similarity of each cluster with its most similar one. Similarity is defined as a ratio of within-cluster distances to between-cluster distances. Lower DAV scores correspond to better clustering, as it signifies that clusters are farther apart from each other.</p></section><section id="Sec33"><h4 class="pmc_sec_title">Latent space correlation (COR)</h4>
<p id="Par98">This metric quantifies the degree of redundancy or entanglement in the learned latent representation <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq95"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/05c9a236cc8c/d33e1716.gif" loading="lazy" id="d33e1716" alt="Inline graphic"></span>. It is calculated as the mean of the sum of absolute Pearson correlations for each latent dimension, excluding self-correlation. A lower score signifies a more disentangled representation.</p>
<table class="disp-formula p" id="Equ37"><tr>
<td class="formula"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/96febedb75bb/d33e1722.gif" loading="lazy" id="d33e1722" alt="graphic file with name d33e1722.gif"></td>
<td class="label">37</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq96"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/78e8abd9b4ab/d33e1729.gif" loading="lazy" id="d33e1729" alt="Inline graphic"></span> is the <em>i</em>-th column (dimension) of the latent matrix <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq97"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/4780b67a973e/d33e1738.gif" loading="lazy" id="d33e1738" alt="Inline graphic"></span>.</p></section><section id="Sec34"><h4 class="pmc_sec_title">Integration LISI (ILISI)</h4>
<p id="Par99">The Local Inverse Simpson’s Index (LISI) measures the diversity of labels within the local neighborhood of each cell. The ILISI variant assesses batch correction by measuring the diversity of batch labels in these neighborhoods. A high raw score indicates that cells from different batches are well-mixed. We report a scaled score where 1 represents perfect mixing and 0 represents no mixing [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>].</p></section><section id="Sec35"><h4 class="pmc_sec_title">Cell-type LISI (CLISI)</h4>
<p id="Par100">The CLISI variant uses the same principle to assess the preservation of biological identity. It measures the diversity of cell-type labels in local neighborhoods. A low raw score indicates that neighborhoods are homogenous with respect to cell type, suggesting good separation of biological structure. We report a scaled score where 1 is optimal (perfect separation of cell types) and 0 is the worst (uniform mixing) [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>].</p></section></section><section id="Sec36"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par101">To rigorously evaluate the performance of the benchmarked methods, we conducted a series of statistical tests on the resulting evaluation metrics. The comparisons were performed under a paired experimental design, where each method was applied to the same collection of datasets, allowing for direct and controlled comparisons. A significance level of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq98"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/fd9fa2378a83/d33e1762.gif" loading="lazy" id="d33e1762" alt="Inline graphic"></span> was used for all tests. The initial step for each metric was to assess the normality of its distribution for each method using the Shapiro-Wilk test. The choice of the subsequent statistical test was dependent on the number of methods being compared and whether the normality assumption was met. For comparing the performance of two methods, a paired Student’s t-test was employed if the distributions of their metric scores were both normal. If the normality assumption was violated for either method, the non-parametric Wilcoxon signed-rank test was used as an alternative. For comparisons involving more than two methods, an omnibus test was first performed to detect any overall significant difference among the group. For normally distributed data, a one-way Repeated Measures Analysis of Variance (RM ANOVA) was conducted. In cases where the data was not normally distributed, the non-parametric Friedman test was used. If the RM ANOVA or Friedman test yielded a significant result (p &lt; 0.05), post-hoc tests were conducted to identify which specific pairs of methods differed significantly. Following a significant RM ANOVA, Tukey’s Honest Significant Difference (HSD) test was used for pairwise comparisons. In the case of a significant Friedman test, pairwise comparisons were performed using the Wilcoxon signed-rank test, with p-values adjusted for multiple comparisons using the Bonferroni correction.</p></section></section><section id="Sec37"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec38"><h3 class="pmc_sec_title">Data selection and research design</h3>
<p id="Par102">To minimize experimental bias related to data selection, we obtained 45 single-cell datasets from public databases, along with an additional 5 datasets specifically included for enhanced multi-batch analysis, as experimental samples for this study (Fig. <a href="#Fig1" class="usa-link">1</a>A). To emphasize the characteristic of single-cell sequencing as continuous biological snapshots, we deliberately selected 15 normal developmental datasets with prominent continuous states during differentiation processes and 15 cancer datasets with stronger cellular state continuity and clinical significance. Additionally, to validate the practical application value of our proposed method, we expanded our analysis to include 15 scATAC-seq datasets, demonstrating the multi-modal application capabilities of GNODEVAE’s improved VAE architecture.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369121_12864_2025_11946_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/af9422b30720/12864_2025_11946_Fig1_HTML.jpg" loading="lazy" id="MO1" height="687" width="778" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Dataset utilization and experimental design of this study. <strong>A</strong> Source accession numbers and tissue types of normal developmental (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq99"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e1796.gif" loading="lazy" id="d33e1796" alt="Inline graphic"></span>) and cancer-associated (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq100"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e1802.gif" loading="lazy" id="d33e1802" alt="Inline graphic"></span>) scRNA-seq datasets, as well as scATAC-seq datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq101"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e1808.gif" loading="lazy" id="d33e1808" alt="Inline graphic"></span>). <strong>B</strong> Workflow and design of the study, including data preprocessing after curation, model architecture development progressing from VAE-based approaches through VGAE refinement to the final GNODEVAE architecture, and comprehensive benchmarking and comparisons between graph convolution layers and other dimensionality reduction methods</p></figcaption></figure><p id="Par103">Based on the fundamental principles of graph learning, we conducted data preprocessing to extract graph structural features, followed by stepwise model improvements guided by experimental evidence. Our approach progressed from graph convolutional layer selection to VGAE implementation, culminating in the construction of the final GNODEVAE model. To validate the generalizability and practical applicability of our improved model, we performed comprehensive benchmarking and performance comparison against 8 machine learning-based dimensionality reduction techniques, 7 deep generative VAE variant models, and 3 contrastive and graph learning models across all 45 datasets. Finally, we conducted hyperparameter tuning of loss terms and ablation studies to further investigate the interpretability and origins of model performance improvements (Fig. <a href="#Fig1" class="usa-link">1</a>B).</p>
<p id="Par104">Given the characteristics of single-cell unsupervised tasks and the representation learning properties of VAE, we selected unsupervised clustering metrics to evaluate the quality of latent embeddings obtained by our model (Fig. <a href="#Fig1" class="usa-link">1</a>B). Among these metrics, ARI and NMI are based on predetermined labels and reflect reconstruction quality, while ASW, CAL, and DAV require latent embeddings as input and reflect the intrinsic clustering geometry quality of the latent space. COR serves as an interpretability metric for disentanglement, elucidating the biological significance of latent space performance improvements achieved through each refinement step.</p></section><section id="Sec39"><h3 class="pmc_sec_title">GAT performance superiority across graph convolution layer alternatives</h3>
<p id="Par105">Systematic evaluation across 45 datasets demonstrates GAT’s consistent superiority over nine alternative graph convolution layers in both VGAE and GNODEVAE architectures, revealing optimal balancing characteristics that establish GAT as the comprehensive performance choice for single-cell analysis. In the VGAE architecture, GAT exhibits significant reconstruction quality advantages in scRNA-seq normal developmental datasets, achieving substantial improvements over traditional methods like ARMA (ARI: +0.341, NMI: +0.338) while maintaining slight advantages over advanced methods like Transformer (ARI: +0.017, NMI: +0.017), with clustering ensemble metrics revealing favorable geometric performance including ASW improvements of +0.096 compared to Graph, CAL substantial enhancement of +121.772 compared to GCN, and DAV improvement of +0.074 compared to TAG, demonstrating that traditional spectral methods consistently underperform across all metrics while intermediate graph neural networks show progressive improvements (Fig. <a href="#Fig2" class="usa-link">2</a>A, Table <a href="#Tab1" class="usa-link">1</a>). In scRNA-seq cancer datasets, GAT advantages become more pronounced with reconstruction quality metrics reaching ARI: +0.195, NMI: +0.193 compared to traditional method Cheb while maintaining slight superiority over Transformer (ARI: +0.001, NMI: +0.001), and clustering geometric performance showing ASW improvement of +0.100 compared to Cheb, CAL significant enhancement of +135.011 compared to GCN, and DAV improvement of +0.083 compared to SG, confirming advanced attention methods approach GAT levels but exhibit complex trade-off relationships (Fig. <a href="#Fig2" class="usa-link">2</a>B, Table <a href="#Tab2" class="usa-link">2</a>). scATAC-seq datasets validate GAT’s cross-modal stability with reconstruction quality improvements of ARI: +0.089, NMI: +0.088 compared to Graph and slight advantages over Transformer (ARI: +0.013, NMI: +0.013), while clustering geometric metrics demonstrate ASW improvement of +0.088 compared to Graph, CAL enhancement of +124.075 compared to GCN, and DAV improvement of +0.162 compared to TAG (Fig. <a href="#Fig2" class="usa-link">2</a>C, Table <a href="#Tab3" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369121_12864_2025_11946_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/0e338dc6c0b1/12864_2025_11946_Fig2_HTML.jpg" loading="lazy" id="MO2" height="992" width="773" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Performance comparison of ten graph convolution layers across VGAE and GNODE (GNODEVAE) architectures. Bar charts represent the average performance across 15 datasets for each convolution layer (from left to right: ARMA, Cheb, Graph, GCN, SSG, SG, SAGE, TAG, Transformer, GAT). Six evaluation metrics are employed: ARI, NMI, ASW, CAL, DAV, and COR. Note that lower DAV values indicate better performance. <strong>A</strong> VGAE architecture performance on scRNA-seq normal developmental datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq102"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e1872.gif" loading="lazy" id="d33e1872" alt="Inline graphic"></span>). <strong>B</strong> VGAE architecture performance on scRNA-seq cancer datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq103"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e1881.gif" loading="lazy" id="d33e1881" alt="Inline graphic"></span>). <strong>C</strong> VGAE architecture performance on scATAC-seq datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq104"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e1891.gif" loading="lazy" id="d33e1891" alt="Inline graphic"></span>). <strong>D</strong> GNODE architecture performance on scRNA-seq normal developmental datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq105"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e1900.gif" loading="lazy" id="d33e1900" alt="Inline graphic"></span>). <strong>E</strong> GNODE architecture performance on scRNA-seq cancer datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq106"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e1909.gif" loading="lazy" id="d33e1909" alt="Inline graphic"></span>). <strong>F</strong> GNODE architecture performance on scATAC-seq datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq107"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e1919.gif" loading="lazy" id="d33e1919" alt="Inline graphic"></span>)</p></figcaption></figure><section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Performance advantage of GAT over alternative convolution layers in VGAE architecture: Mean differences across scRNA-seq normal developmental datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq108"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e1935.gif" loading="lazy" id="d33e1935" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance, with DAV improvements shown as positive values in the comparison results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ARI</th>
<th align="left" colspan="1" rowspan="1">NMI</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
<th align="left" colspan="1" rowspan="1">COR</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. ARMA</td>
<td align="left" colspan="1" rowspan="1">0.341</td>
<td align="left" colspan="1" rowspan="1">0.338</td>
<td align="left" colspan="1" rowspan="1">0.105</td>
<td align="left" colspan="1" rowspan="1">254.135</td>
<td align="left" colspan="1" rowspan="1">0.505</td>
<td align="left" colspan="1" rowspan="1">1.622</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Cheb</td>
<td align="left" colspan="1" rowspan="1">0.261</td>
<td align="left" colspan="1" rowspan="1">0.259</td>
<td align="left" colspan="1" rowspan="1">0.083</td>
<td align="left" colspan="1" rowspan="1">233.810</td>
<td align="left" colspan="1" rowspan="1">0.363</td>
<td align="left" colspan="1" rowspan="1">0.766</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Graph</td>
<td align="left" colspan="1" rowspan="1">0.317</td>
<td align="left" colspan="1" rowspan="1">0.314</td>
<td align="left" colspan="1" rowspan="1">0.096</td>
<td align="left" colspan="1" rowspan="1">219.924</td>
<td align="left" colspan="1" rowspan="1">0.299</td>
<td align="left" colspan="1" rowspan="1">1.084</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. GCN</td>
<td align="left" colspan="1" rowspan="1">0.091</td>
<td align="left" colspan="1" rowspan="1">0.090</td>
<td align="left" colspan="1" rowspan="1">0.027</td>
<td align="left" colspan="1" rowspan="1">121.772</td>
<td align="left" colspan="1" rowspan="1">0.074</td>
<td align="left" colspan="1" rowspan="1">0.303</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SSG</td>
<td align="left" colspan="1" rowspan="1">0.128</td>
<td align="left" colspan="1" rowspan="1">0.126</td>
<td align="left" colspan="1" rowspan="1">0.020</td>
<td align="left" colspan="1" rowspan="1">77.642</td>
<td align="left" colspan="1" rowspan="1">0.023</td>
<td align="left" colspan="1" rowspan="1">−0.071</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SG</td>
<td align="left" colspan="1" rowspan="1">0.114</td>
<td align="left" colspan="1" rowspan="1">0.113</td>
<td align="left" colspan="1" rowspan="1">0.020</td>
<td align="left" colspan="1" rowspan="1">94.395</td>
<td align="left" colspan="1" rowspan="1">0.034</td>
<td align="left" colspan="1" rowspan="1">0.042</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SAGE</td>
<td align="left" colspan="1" rowspan="1">0.064</td>
<td align="left" colspan="1" rowspan="1">0.064</td>
<td align="left" colspan="1" rowspan="1">0.026</td>
<td align="left" colspan="1" rowspan="1">74.191</td>
<td align="left" colspan="1" rowspan="1">0.062</td>
<td align="left" colspan="1" rowspan="1">−0.182</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. TAG</td>
<td align="left" colspan="1" rowspan="1">0.054</td>
<td align="left" colspan="1" rowspan="1">0.054</td>
<td align="left" colspan="1" rowspan="1">0.028</td>
<td align="left" colspan="1" rowspan="1">44.351</td>
<td align="left" colspan="1" rowspan="1">0.074</td>
<td align="left" colspan="1" rowspan="1">−0.208</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Transformer</td>
<td align="left" colspan="1" rowspan="1">0.017</td>
<td align="left" colspan="1" rowspan="1">0.017</td>
<td align="left" colspan="1" rowspan="1">0.012</td>
<td align="left" colspan="1" rowspan="1">−6.424</td>
<td align="left" colspan="1" rowspan="1">0.012</td>
<td align="left" colspan="1" rowspan="1">−0.084</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Performance advantage of GAT over alternative convolution layers in VGAE architecture: Mean differences across scRNA-seq cancer datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq109"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e2110.gif" loading="lazy" id="d33e2110" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance, with DAV improvements shown as positive values in the comparison results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ARI</th>
<th align="left" colspan="1" rowspan="1">NMI</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
<th align="left" colspan="1" rowspan="1">COR</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. ARMA</td>
<td align="left" colspan="1" rowspan="1">0.384</td>
<td align="left" colspan="1" rowspan="1">0.380</td>
<td align="left" colspan="1" rowspan="1">0.119</td>
<td align="left" colspan="1" rowspan="1">254.077</td>
<td align="left" colspan="1" rowspan="1">0.591</td>
<td align="left" colspan="1" rowspan="1">1.760</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Cheb</td>
<td align="left" colspan="1" rowspan="1">0.195</td>
<td align="left" colspan="1" rowspan="1">0.193</td>
<td align="left" colspan="1" rowspan="1">0.100</td>
<td align="left" colspan="1" rowspan="1">247.167</td>
<td align="left" colspan="1" rowspan="1">0.434</td>
<td align="left" colspan="1" rowspan="1">0.700</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Graph</td>
<td align="left" colspan="1" rowspan="1">0.364</td>
<td align="left" colspan="1" rowspan="1">0.360</td>
<td align="left" colspan="1" rowspan="1">0.104</td>
<td align="left" colspan="1" rowspan="1">172.112</td>
<td align="left" colspan="1" rowspan="1">0.298</td>
<td align="left" colspan="1" rowspan="1">1.167</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. GCN</td>
<td align="left" colspan="1" rowspan="1">0.066</td>
<td align="left" colspan="1" rowspan="1">0.066</td>
<td align="left" colspan="1" rowspan="1">0.047</td>
<td align="left" colspan="1" rowspan="1">135.011</td>
<td align="left" colspan="1" rowspan="1">0.133</td>
<td align="left" colspan="1" rowspan="1">0.371</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SSG</td>
<td align="left" colspan="1" rowspan="1">0.086</td>
<td align="left" colspan="1" rowspan="1">0.085</td>
<td align="left" colspan="1" rowspan="1">0.030</td>
<td align="left" colspan="1" rowspan="1">90.541</td>
<td align="left" colspan="1" rowspan="1">0.054</td>
<td align="left" colspan="1" rowspan="1">−0.033</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SG</td>
<td align="left" colspan="1" rowspan="1">0.058</td>
<td align="left" colspan="1" rowspan="1">0.057</td>
<td align="left" colspan="1" rowspan="1">0.023</td>
<td align="left" colspan="1" rowspan="1">70.154</td>
<td align="left" colspan="1" rowspan="1">−0.030</td>
<td align="left" colspan="1" rowspan="1">−0.066</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SAGE</td>
<td align="left" colspan="1" rowspan="1">0.017</td>
<td align="left" colspan="1" rowspan="1">0.017</td>
<td align="left" colspan="1" rowspan="1">0.034</td>
<td align="left" colspan="1" rowspan="1">72.168</td>
<td align="left" colspan="1" rowspan="1">0.083</td>
<td align="left" colspan="1" rowspan="1">0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. TAG</td>
<td align="left" colspan="1" rowspan="1">0.028</td>
<td align="left" colspan="1" rowspan="1">0.027</td>
<td align="left" colspan="1" rowspan="1">0.026</td>
<td align="left" colspan="1" rowspan="1">38.273</td>
<td align="left" colspan="1" rowspan="1">0.077</td>
<td align="left" colspan="1" rowspan="1">−0.016</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Transformer</td>
<td align="left" colspan="1" rowspan="1">0.001</td>
<td align="left" colspan="1" rowspan="1">0.001</td>
<td align="left" colspan="1" rowspan="1">0.011</td>
<td align="left" colspan="1" rowspan="1">7.359</td>
<td align="left" colspan="1" rowspan="1">0.054</td>
<td align="left" colspan="1" rowspan="1">−0.028</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Performance advantage of GAT over alternative convolution layers in VGAE architecture: Mean differences across scATAC-seq datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq110"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e2285.gif" loading="lazy" id="d33e2285" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance, with DAV improvements shown as positive values in the comparison results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ARI</th>
<th align="left" colspan="1" rowspan="1">NMI</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
<th align="left" colspan="1" rowspan="1">COR</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. ARMA</td>
<td align="left" colspan="1" rowspan="1">0.041</td>
<td align="left" colspan="1" rowspan="1">0.040</td>
<td align="left" colspan="1" rowspan="1">0.076</td>
<td align="left" colspan="1" rowspan="1">230.486</td>
<td align="left" colspan="1" rowspan="1">0.383</td>
<td align="left" colspan="1" rowspan="1">1.544</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Cheb</td>
<td align="left" colspan="1" rowspan="1">0.051</td>
<td align="left" colspan="1" rowspan="1">0.050</td>
<td align="left" colspan="1" rowspan="1">0.071</td>
<td align="left" colspan="1" rowspan="1">235.923</td>
<td align="left" colspan="1" rowspan="1">0.376</td>
<td align="left" colspan="1" rowspan="1">0.956</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Graph</td>
<td align="left" colspan="1" rowspan="1">0.089</td>
<td align="left" colspan="1" rowspan="1">0.088</td>
<td align="left" colspan="1" rowspan="1">0.088</td>
<td align="left" colspan="1" rowspan="1">233.436</td>
<td align="left" colspan="1" rowspan="1">0.212</td>
<td align="left" colspan="1" rowspan="1">1.027</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. GCN</td>
<td align="left" colspan="1" rowspan="1">0.040</td>
<td align="left" colspan="1" rowspan="1">0.039</td>
<td align="left" colspan="1" rowspan="1">0.022</td>
<td align="left" colspan="1" rowspan="1">124.075</td>
<td align="left" colspan="1" rowspan="1">0.086</td>
<td align="left" colspan="1" rowspan="1">0.493</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SSG</td>
<td align="left" colspan="1" rowspan="1">0.044</td>
<td align="left" colspan="1" rowspan="1">0.043</td>
<td align="left" colspan="1" rowspan="1">0.015</td>
<td align="left" colspan="1" rowspan="1">60.590</td>
<td align="left" colspan="1" rowspan="1">0.089</td>
<td align="left" colspan="1" rowspan="1">0.052</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SG</td>
<td align="left" colspan="1" rowspan="1">0.029</td>
<td align="left" colspan="1" rowspan="1">0.029</td>
<td align="left" colspan="1" rowspan="1">−0.001</td>
<td align="left" colspan="1" rowspan="1">70.312</td>
<td align="left" colspan="1" rowspan="1">−0.032</td>
<td align="left" colspan="1" rowspan="1">0.038</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SAGE</td>
<td align="left" colspan="1" rowspan="1">0.020</td>
<td align="left" colspan="1" rowspan="1">0.020</td>
<td align="left" colspan="1" rowspan="1">0.025</td>
<td align="left" colspan="1" rowspan="1">90.508</td>
<td align="left" colspan="1" rowspan="1">0.142</td>
<td align="left" colspan="1" rowspan="1">0.254</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. TAG</td>
<td align="left" colspan="1" rowspan="1">0.011</td>
<td align="left" colspan="1" rowspan="1">0.011</td>
<td align="left" colspan="1" rowspan="1">0.036</td>
<td align="left" colspan="1" rowspan="1">95.693</td>
<td align="left" colspan="1" rowspan="1">0.162</td>
<td align="left" colspan="1" rowspan="1">0.211</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Transformer</td>
<td align="left" colspan="1" rowspan="1">0.013</td>
<td align="left" colspan="1" rowspan="1">0.013</td>
<td align="left" colspan="1" rowspan="1">0.026</td>
<td align="left" colspan="1" rowspan="1">50.984</td>
<td align="left" colspan="1" rowspan="1">0.116</td>
<td align="left" colspan="1" rowspan="1">0.170</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p id="Par106">The GNODEVAE architecture significantly amplifies GAT’s performance advantages through architectural synergistic effects, with scRNA-seq normal developmental datasets showing GAT achieving steady reconstruction quality improvements compared to Graph (ARI: +0.186, NMI: +0.184) while performing nearly equivalently to Transformer (ARI: +0.001, NMI: +0.001), and clustering geometric performance realizing qualitative leaps with ASW reaching +0.191 compared to Cheb, CAL achieving massive improvement of +794.152 compared to GCN, and DAV enhancement of +0.804 compared to Cheb, while COR latent space coupling shows significant strengthening of +2.259 compared to traditional method ARMA but exhibits a decrease of −0.297 compared to TAG, reflecting GAT’s achievement of reasonable balance in latent space component disentanglement (Fig. <a href="#Fig2" class="usa-link">2</a>D, Table <a href="#Tab4" class="usa-link">4</a>). scRNA-seq cancer datasets demonstrate the strongest performance with reconstruction quality metrics reaching peak levels compared to Cheb (ARI: +0.347, NMI: +0.344) while maintaining clear advantages over Transformer (ARI: +0.047, NMI: +0.047), and clustering geometric metrics showing comprehensive optimization with ASW of +0.198 compared to Cheb, CAL significant improvement of +365.633 compared to GCN, and DAV reaching +0.916 compared to Cheb, with COR exhibiting higher coupling compared to traditional methods (+2.744) but decreases compared to advanced method TAG (−0.337), further confirming the latent space coupling trade-off patterns where GAT exhibits higher latent component correlations compared to traditional methods but shows lower coupling when compared to both TAG and Transformer (Fig. <a href="#Fig2" class="usa-link">2</a>E, Table <a href="#Tab5" class="usa-link">5</a>). scATAC-seq datasets further confirm these patterns with reconstruction quality significantly improving compared to Cheb (ARI: +0.137, NMI: +0.136) while maintaining clear advantages over Transformer (ARI: +0.091, NMI: +0.090), clustering cohesion ASW improving +0.104 compared to SSG, clustering separation CAL achieving substantial optimization of +491.822 compared to GCN, and overall clustering quality DAV improving +0.672 compared to Cheb, with COR demonstrating the highest coupling compared to traditional methods (+3.207) but showing significant decrease compared to TAG (−0.756), establishing that although advanced methods like TAG and Transformer perform better in certain clustering geometric metrics, GAT maintains stable advantages in reconstruction quality while maintaining reconstruction quality and clustering geometric performance (Fig. <a href="#Fig2" class="usa-link">2</a>F, Table <a href="#Tab6" class="usa-link">6</a>).</p>
<section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Performance advantage of GAT over alternative convolution layers in GNODEVAE architecture: Mean differences across scRNA-seq normal developmental datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq111"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e2460.gif" loading="lazy" id="d33e2460" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance, with DAV improvements shown as positive values in the comparison results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ARI</th>
<th align="left" colspan="1" rowspan="1">NMI</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
<th align="left" colspan="1" rowspan="1">COR</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. ARMA</td>
<td align="left" colspan="1" rowspan="1">0.163</td>
<td align="left" colspan="1" rowspan="1">0.162</td>
<td align="left" colspan="1" rowspan="1">0.158</td>
<td align="left" colspan="1" rowspan="1">1211.298</td>
<td align="left" colspan="1" rowspan="1">0.552</td>
<td align="left" colspan="1" rowspan="1">2.259</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Cheb</td>
<td align="left" colspan="1" rowspan="1">0.157</td>
<td align="left" colspan="1" rowspan="1">0.156</td>
<td align="left" colspan="1" rowspan="1">0.191</td>
<td align="left" colspan="1" rowspan="1">1261.785</td>
<td align="left" colspan="1" rowspan="1">0.804</td>
<td align="left" colspan="1" rowspan="1">1.626</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Graph</td>
<td align="left" colspan="1" rowspan="1">0.186</td>
<td align="left" colspan="1" rowspan="1">0.184</td>
<td align="left" colspan="1" rowspan="1">0.153</td>
<td align="left" colspan="1" rowspan="1">1144.690</td>
<td align="left" colspan="1" rowspan="1">0.483</td>
<td align="left" colspan="1" rowspan="1">1.539</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. GCN</td>
<td align="left" colspan="1" rowspan="1">0.051</td>
<td align="left" colspan="1" rowspan="1">0.050</td>
<td align="left" colspan="1" rowspan="1">0.103</td>
<td align="left" colspan="1" rowspan="1">794.152</td>
<td align="left" colspan="1" rowspan="1">0.333</td>
<td align="left" colspan="1" rowspan="1">0.422</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SSG</td>
<td align="left" colspan="1" rowspan="1">0.067</td>
<td align="left" colspan="1" rowspan="1">0.067</td>
<td align="left" colspan="1" rowspan="1">0.112</td>
<td align="left" colspan="1" rowspan="1">730.911</td>
<td align="left" colspan="1" rowspan="1">0.345</td>
<td align="left" colspan="1" rowspan="1">0.052</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SG</td>
<td align="left" colspan="1" rowspan="1">0.049</td>
<td align="left" colspan="1" rowspan="1">0.048</td>
<td align="left" colspan="1" rowspan="1">0.095</td>
<td align="left" colspan="1" rowspan="1">627.984</td>
<td align="left" colspan="1" rowspan="1">0.252</td>
<td align="left" colspan="1" rowspan="1">−0.184</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SAGE</td>
<td align="left" colspan="1" rowspan="1">0.043</td>
<td align="left" colspan="1" rowspan="1">0.043</td>
<td align="left" colspan="1" rowspan="1">0.113</td>
<td align="left" colspan="1" rowspan="1">691.872</td>
<td align="left" colspan="1" rowspan="1">0.375</td>
<td align="left" colspan="1" rowspan="1">0.086</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. TAG</td>
<td align="left" colspan="1" rowspan="1">0.028</td>
<td align="left" colspan="1" rowspan="1">0.027</td>
<td align="left" colspan="1" rowspan="1">0.083</td>
<td align="left" colspan="1" rowspan="1">454.817</td>
<td align="left" colspan="1" rowspan="1">0.242</td>
<td align="left" colspan="1" rowspan="1">−0.297</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Transformer</td>
<td align="left" colspan="1" rowspan="1">0.001</td>
<td align="left" colspan="1" rowspan="1">0.001</td>
<td align="left" colspan="1" rowspan="1">0.032</td>
<td align="left" colspan="1" rowspan="1">−70.162</td>
<td align="left" colspan="1" rowspan="1">0.091</td>
<td align="left" colspan="1" rowspan="1">−0.245</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Performance advantage of GAT over alternative convolution layers in GNODEVAE architecture: Mean differences across scRNA-seq cancer datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq112"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e2635.gif" loading="lazy" id="d33e2635" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance, with DAV improvements shown as positive values in the comparison results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ARI</th>
<th align="left" colspan="1" rowspan="1">NMI</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
<th align="left" colspan="1" rowspan="1">COR</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. ARMA</td>
<td align="left" colspan="1" rowspan="1">0.465</td>
<td align="left" colspan="1" rowspan="1">0.461</td>
<td align="left" colspan="1" rowspan="1">0.200</td>
<td align="left" colspan="1" rowspan="1">634.634</td>
<td align="left" colspan="1" rowspan="1">0.913</td>
<td align="left" colspan="1" rowspan="1">2.744</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Cheb</td>
<td align="left" colspan="1" rowspan="1">0.347</td>
<td align="left" colspan="1" rowspan="1">0.344</td>
<td align="left" colspan="1" rowspan="1">0.198</td>
<td align="left" colspan="1" rowspan="1">680.743</td>
<td align="left" colspan="1" rowspan="1">0.916</td>
<td align="left" colspan="1" rowspan="1">1.584</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Graph</td>
<td align="left" colspan="1" rowspan="1">0.244</td>
<td align="left" colspan="1" rowspan="1">0.241</td>
<td align="left" colspan="1" rowspan="1">0.121</td>
<td align="left" colspan="1" rowspan="1">558.404</td>
<td align="left" colspan="1" rowspan="1">0.215</td>
<td align="left" colspan="1" rowspan="1">1.222</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. GCN</td>
<td align="left" colspan="1" rowspan="1">0.053</td>
<td align="left" colspan="1" rowspan="1">0.052</td>
<td align="left" colspan="1" rowspan="1">0.080</td>
<td align="left" colspan="1" rowspan="1">365.633</td>
<td align="left" colspan="1" rowspan="1">0.197</td>
<td align="left" colspan="1" rowspan="1">0.655</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SSG</td>
<td align="left" colspan="1" rowspan="1">0.130</td>
<td align="left" colspan="1" rowspan="1">0.129</td>
<td align="left" colspan="1" rowspan="1">0.107</td>
<td align="left" colspan="1" rowspan="1">280.016</td>
<td align="left" colspan="1" rowspan="1">0.346</td>
<td align="left" colspan="1" rowspan="1">−0.052</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SG</td>
<td align="left" colspan="1" rowspan="1">0.049</td>
<td align="left" colspan="1" rowspan="1">0.048</td>
<td align="left" colspan="1" rowspan="1">0.067</td>
<td align="left" colspan="1" rowspan="1">102.957</td>
<td align="left" colspan="1" rowspan="1">0.126</td>
<td align="left" colspan="1" rowspan="1">−0.159</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SAGE</td>
<td align="left" colspan="1" rowspan="1">0.086</td>
<td align="left" colspan="1" rowspan="1">0.085</td>
<td align="left" colspan="1" rowspan="1">0.099</td>
<td align="left" colspan="1" rowspan="1">125.671</td>
<td align="left" colspan="1" rowspan="1">0.332</td>
<td align="left" colspan="1" rowspan="1">−0.045</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. TAG</td>
<td align="left" colspan="1" rowspan="1">0.015</td>
<td align="left" colspan="1" rowspan="1">0.015</td>
<td align="left" colspan="1" rowspan="1">0.025</td>
<td align="left" colspan="1" rowspan="1">−293.997</td>
<td align="left" colspan="1" rowspan="1">−0.002</td>
<td align="left" colspan="1" rowspan="1">−0.337</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Transformer</td>
<td align="left" colspan="1" rowspan="1">0.047</td>
<td align="left" colspan="1" rowspan="1">0.047</td>
<td align="left" colspan="1" rowspan="1">0.053</td>
<td align="left" colspan="1" rowspan="1">−244.628</td>
<td align="left" colspan="1" rowspan="1">0.221</td>
<td align="left" colspan="1" rowspan="1">−0.475</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab6"><h4 class="obj_head">Table 6.</h4>
<div class="caption p"><p>Performance advantage of GAT over alternative convolution layers in GNODEVAE architecture: Mean differences across scATAC-seq datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq113"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e2810.gif" loading="lazy" id="d33e2810" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance, with DAV improvements shown as positive values in the comparison results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ARI</th>
<th align="left" colspan="1" rowspan="1">NMI</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
<th align="left" colspan="1" rowspan="1">COR</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. ARMA</td>
<td align="left" colspan="1" rowspan="1">0.188</td>
<td align="left" colspan="1" rowspan="1">0.186</td>
<td align="left" colspan="1" rowspan="1">0.116</td>
<td align="left" colspan="1" rowspan="1">811.320</td>
<td align="left" colspan="1" rowspan="1">0.592</td>
<td align="left" colspan="1" rowspan="1">3.207</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Cheb</td>
<td align="left" colspan="1" rowspan="1">0.137</td>
<td align="left" colspan="1" rowspan="1">0.136</td>
<td align="left" colspan="1" rowspan="1">0.098</td>
<td align="left" colspan="1" rowspan="1">860.785</td>
<td align="left" colspan="1" rowspan="1">0.672</td>
<td align="left" colspan="1" rowspan="1">1.722</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Graph</td>
<td align="left" colspan="1" rowspan="1">0.142</td>
<td align="left" colspan="1" rowspan="1">0.141</td>
<td align="left" colspan="1" rowspan="1">−0.003</td>
<td align="left" colspan="1" rowspan="1">420.559</td>
<td align="left" colspan="1" rowspan="1">−0.094</td>
<td align="left" colspan="1" rowspan="1">0.748</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. GCN</td>
<td align="left" colspan="1" rowspan="1">0.033</td>
<td align="left" colspan="1" rowspan="1">0.033</td>
<td align="left" colspan="1" rowspan="1">0.077</td>
<td align="left" colspan="1" rowspan="1">491.822</td>
<td align="left" colspan="1" rowspan="1">0.278</td>
<td align="left" colspan="1" rowspan="1">0.306</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SSG</td>
<td align="left" colspan="1" rowspan="1">0.150</td>
<td align="left" colspan="1" rowspan="1">0.149</td>
<td align="left" colspan="1" rowspan="1">0.104</td>
<td align="left" colspan="1" rowspan="1">612.310</td>
<td align="left" colspan="1" rowspan="1">0.484</td>
<td align="left" colspan="1" rowspan="1">0.762</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SG</td>
<td align="left" colspan="1" rowspan="1">0.100</td>
<td align="left" colspan="1" rowspan="1">0.100</td>
<td align="left" colspan="1" rowspan="1">0.071</td>
<td align="left" colspan="1" rowspan="1">268.020</td>
<td align="left" colspan="1" rowspan="1">0.253</td>
<td align="left" colspan="1" rowspan="1">−0.157</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. SAGE</td>
<td align="left" colspan="1" rowspan="1">0.105</td>
<td align="left" colspan="1" rowspan="1">0.104</td>
<td align="left" colspan="1" rowspan="1">0.093</td>
<td align="left" colspan="1" rowspan="1">74.528</td>
<td align="left" colspan="1" rowspan="1">0.402</td>
<td align="left" colspan="1" rowspan="1">0.097</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. TAG</td>
<td align="left" colspan="1" rowspan="1">0.011</td>
<td align="left" colspan="1" rowspan="1">0.011</td>
<td align="left" colspan="1" rowspan="1">0.024</td>
<td align="left" colspan="1" rowspan="1">−308.388</td>
<td align="left" colspan="1" rowspan="1">0.070</td>
<td align="left" colspan="1" rowspan="1">−0.756</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GAT vs. Transformer</td>
<td align="left" colspan="1" rowspan="1">0.091</td>
<td align="left" colspan="1" rowspan="1">0.090</td>
<td align="left" colspan="1" rowspan="1">0.074</td>
<td align="left" colspan="1" rowspan="1">−186.995</td>
<td align="left" colspan="1" rowspan="1">0.290</td>
<td align="left" colspan="1" rowspan="1">−0.381</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec40"><h3 class="pmc_sec_title">Architectural superiority of GNODEVAE over VGAE: continuous-time modeling for single-cell state transitions</h3>
<p id="Par113">Systematic comparison across 45 datasets demonstrates GNODEVAE architecture’s significant performance advantages over VGAE architecture across all dataset categories and evaluation metrics, with the GNODEVAE architecture integrating Graph Attention Networks, Neural Ordinary Differential Equations, and Variational Autoencoders to directly model the continuous characteristics of cellular state transitions, addressing the fundamental limitation of traditional discrete modeling approaches that fail to capture the gradual, continuous changes inherent in cellular differentiation, activation, and transformation processes (Fig. <a href="#Fig3" class="usa-link">3</a>A, Table <a href="#Tab7" class="usa-link">7</a>). Performance patterns across different data types reveal distinct biological complexity-related characteristics where scRNA-seq normal developmental datasets exhibit balanced performance improvements with reconstruction quality metrics ARI and NMI increasing by +0.100 and +0.099 respectively, clustering geometric metrics ASW, CAL, and DAV improving by +0.113, +1094.718, and +0.318 respectively, while COR latent space coupling strengthens by +1.321, reflecting how well-defined continuous trajectories from pluripotent to specialized states naturally align with neural ordinary differential equation modeling (Fig. <a href="#Fig3" class="usa-link">3</a>B, Table <a href="#Tab7" class="usa-link">7</a>). scRNA-seq cancer datasets demonstrate the greatest reconstruction quality advantages with ARI and NMI achieving +0.167 and +0.165 improvements respectively, accompanied by significant clustering geometric enhancements (ASW: +0.140, CAL: +738.352, DAV: +0.437) and COR coupling improvement of +1.222, establishing that complex non-linear state transitions involved in malignant transformation enable continuous-time modeling to demonstrate maximal reconstruction quality advantages (Fig. <a href="#Fig3" class="usa-link">3</a>C, Table <a href="#Tab7" class="usa-link">7</a>). scATAC-seq datasets present characteristic performance trade-offs of GNODEVAE architecture where clustering geometric performance achieves substantial improvements, particularly CAL reaching a dramatic enhancement of +1326.966, with ASW and DAV improving by +0.086 and +0.244 respectively, while reconstruction quality metrics ARI and NMI only achieve modest improvements of +0.069 and +0.068, and COR latent space coupling exhibits the highest increase of +1.633, reflecting the distinctive properties of GNODEVAE architecture when processing sparse, high-dimensional chromatin accessibility data where continuous-time modeling effectively transforms sparse accessibility patterns into smooth latent trajectories, dramatically improving clustering separation and latent space structurization, but this enhancement in latent space coupling does not fully equate to reconstruction quality improvement, instead representing a trade-off relationship between continuous-time dynamics modeling and reconstruction accuracy (Fig. <a href="#Fig3" class="usa-link">3</a>D, Table <a href="#Tab7" class="usa-link">7</a>). These systematic performance differences establish GNODEVAE architecture as a fundamental advancement in continuous-time dynamics modeling for single-cell genomics applications, while simultaneously indicating that different architectural choices require careful consideration based on specific analytical objectives and data characteristics.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369121_12864_2025_11946_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/0710a07bfe07/12864_2025_11946_Fig3_HTML.jpg" loading="lazy" id="MO3" height="603" width="778" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>GNODEVAE architecture and clustering performance comparison with VGAE. <strong>A</strong> Schematic overview of the GNODEVAE architecture. The model takes raw expression matrices and processed graph information as inputs, employs GAT as the graph encoder to obtain latent representations for ODE solving, and performs reconstruction within the VAE framework. The architecture is optimized using three primary loss functions: reconstruction loss, ODE loss, and KL regularization term. <strong>B</strong> Performance comparison on scRNA-seq normal developmental datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq114"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3018.gif" loading="lazy" id="d33e3018" alt="Inline graphic"></span>). Statistical analysis employed paired t-tests for CAL, DAV, and COR (data met normality assumptions) or Wilcoxon signed-rank tests for ARI, NMI, and ASW (normality assumptions not met). <strong>C</strong> Performance comparison on scRNA-seq cancer datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq115"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3027.gif" loading="lazy" id="d33e3027" alt="Inline graphic"></span>). Statistical analysis employed paired t-tests for all metrics except CAL (data met normality assumptions) or Wilcoxon signed-rank test for CAL (normality assumptions not met). <strong>D</strong> Performance comparison on scATAC-seq datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq116"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3037.gif" loading="lazy" id="d33e3037" alt="Inline graphic"></span>). Statistical analysis employed paired t-tests for ARI, NMI, and COR (data met normality assumptions) or Wilcoxon signed-rank tests for ASW, CAL, and DAV (normality assumptions not met). Significance levels are indicated as: *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001; ns indicates not significant</p></figcaption></figure><section class="tw xbox font-sm" id="Tab7"><h4 class="obj_head">Table 7.</h4>
<div class="caption p"><p>Performance advantage of GNODEVAE over VGAE architecture: Mean differences across dataset categories (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq117"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3053.gif" loading="lazy" id="d33e3053" alt="Inline graphic"></span> each). Note that lower DAV values indicate better performance, with DAV improvements shown as positive values in the comparison results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Dataset Category</th>
<th align="left" colspan="1" rowspan="1">ARI</th>
<th align="left" colspan="1" rowspan="1">NMI</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
<th align="left" colspan="1" rowspan="1">COR</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">scATAC-seq</td>
<td align="left" colspan="1" rowspan="1">0.069</td>
<td align="left" colspan="1" rowspan="1">0.068</td>
<td align="left" colspan="1" rowspan="1">0.086</td>
<td align="left" colspan="1" rowspan="1">1326.966</td>
<td align="left" colspan="1" rowspan="1">0.244</td>
<td align="left" colspan="1" rowspan="1">1.633</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">scRNA-seq Developmental</td>
<td align="left" colspan="1" rowspan="1">0.100</td>
<td align="left" colspan="1" rowspan="1">0.099</td>
<td align="left" colspan="1" rowspan="1">0.113</td>
<td align="left" colspan="1" rowspan="1">1094.718</td>
<td align="left" colspan="1" rowspan="1">0.318</td>
<td align="left" colspan="1" rowspan="1">1.321</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">scRNA-seq Cancer</td>
<td align="left" colspan="1" rowspan="1">0.167</td>
<td align="left" colspan="1" rowspan="1">0.165</td>
<td align="left" colspan="1" rowspan="1">0.140</td>
<td align="left" colspan="1" rowspan="1">738.352</td>
<td align="left" colspan="1" rowspan="1">0.437</td>
<td align="left" colspan="1" rowspan="1">1.222</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec41"><h3 class="pmc_sec_title">GNODEVAE’s clustering geometry superiority over traditional dimensionality reduction methods</h3>
<p id="Par115">Systematic evaluation across 45 datasets demonstrates GNODEVAE’s significant advantages over traditional machine learning dimensionality reduction methods in clustering geometry metrics while reconstruction quality metrics do not exhibit similar improvement patterns, with ARI and NMI reconstruction quality metrics showing no statistically significant differences when comparing GNODEVAE with traditional methods, primarily due to the inherent advantages of machine learning techniques in feature extraction and data representation where PCA preserves maximum variance information through principal component analysis, ICA captures intrinsic data structure through independent component decomposition, and NMF extracts local feature patterns through non-negative matrix factorization, as these linear transformation methods effectively maintain the feature structure of original data, thus demonstrating performance comparable to deep learning methods in reconstruction tasks (Fig. <a href="#Fig4" class="usa-link">4</a>, Tables <a href="#Tab8" class="usa-link">8</a>, <a href="#Tab9" class="usa-link">9</a> and <a href="#Tab10" class="usa-link">10</a>). Despite equivalent reconstruction quality, GNODEVAE exhibits systematic advantages in latent space clustering geometry organization, with scRNA-seq normal developmental datasets showing significant geometric improvements compared to traditional linear methods (PCA, ICA, NMF, TSVD) with ASW improvements ranging from +0.273 to +0.327 and CAL enhancements from +824.599 to +1040.542, while compared to advanced trajectory inference methods (DIFF, PALANTIR), GNODEVAE maintains modest but consistent advantages (ASW: +0.006 to +0.012, CAL: +769.142 to +773.901) though with slight DAV disadvantages (−0.079 to −0.033), suggesting that specialized trajectory methods may produce more compact clusters for well-characterized developmental trajectories (Fig. <a href="#Fig4" class="usa-link">4</a>A, Table <a href="#Tab8" class="usa-link">8</a>). scRNA-seq cancer datasets demonstrate larger performance gaps with GNODEVAE achieving ASW improvements of +0.133 to +0.496 and CAL enhancements of +2736.732 to +3098.999 compared to traditional methods while maintaining substantial advantages even against advanced trajectory methods, reflecting GNODEVAE’s superior performance in complex dysregulated cellular state spaces through general-purpose continuous-time modeling (Fig. <a href="#Fig4" class="usa-link">4</a>B, Table <a href="#Tab9" class="usa-link">9</a>). scATAC-seq datasets show the most pronounced clustering geometry improvements with CAL enhancements ranging from +4462.961 to +5009.348 and ASW improvements from +0.064 to +0.498, demonstrating the particular suitability of continuous-time modeling for processing sparse binary chromatin accessibility data (Fig. <a href="#Fig4" class="usa-link">4</a>C, Table <a href="#Tab10" class="usa-link">10</a>). These results establish GNODEVAE’s unique value proposition in single-cell genomics applications where while traditional machine learning methods possess advantages in computational efficiency and reconstruction accuracy, GNODEVAE achieves systematic improvements in latent space clustering geometry organization through neural ordinary differential equation frameworks, particularly demonstrating clear advantages when processing complex cellular state spaces and sparse high-dimensional data, indicating that when analytical objectives emphasize high-quality unsupervised clustering and biological interpretation, the increased computational complexity of GNODEVAE is justified, while traditional methods retain application value when pursuing computational efficiency and fundamental feature preservation.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369121_12864_2025_11946_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/93100720cc4f/12864_2025_11946_Fig4_HTML.jpg" loading="lazy" id="MO4" height="832" width="778" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Clustering performance comparison between GNODEVAE and machine learning-based single-cell analysis methods. Box plots display performance comparisons across dimensionality reduction approaches (from left to right: PCA, ICA, NMF, FA, TSVD, DICL, DIFF, PALANTIR) versus GNODE (GNODEVAE) architecture. <strong>A</strong> Performance comparison on scRNA-seq normal developmental datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq118"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3174.gif" loading="lazy" id="d33e3174" alt="Inline graphic"></span>). Statistical analysis employed repeated measures ANOVA with Tukey’s HSD post-hoc tests for metrics meeting normality assumptions (ARI, NMI, DAV), or Friedman tests with Wilcoxon signed-rank post-hoc tests for non-normally distributed metrics (ASW, CAL). <strong>B</strong> Performance comparison on scRNA-seq cancer associated datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq119"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3183.gif" loading="lazy" id="d33e3183" alt="Inline graphic"></span>). Statistical analysis employed repeated measures ANOVA with Tukey’s HSD post-hoc tests for metrics meeting normality assumptions (ARI, NMI, ASW, DAV), or Friedman test with Wilcoxon signed-rank post-hoc test for non-normally distributed metrics (CAL). <strong>C</strong> Performance comparison on scATAC-seq associated datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq120"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3193.gif" loading="lazy" id="d33e3193" alt="Inline graphic"></span>). Statistical analysis employed Friedman tests with Wilcoxon signed-rank post-hoc tests for all metrics due to violation of normality assumptions. Significance levels are indicated as: *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001 with Bonferroni correction</p></figcaption></figure><section class="tw xbox font-sm" id="Tab8"><h4 class="obj_head">Table 8.</h4>
<div class="caption p"><p>Performance advantage of GNODEVAE over alternative dimensionality reduction methods: Clustering geometry metrics in scRNA-seq normal developmental datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq121"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3209.gif" loading="lazy" id="d33e3209" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance, with DAV improvements shown as positive values in the comparison results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. PCA</td>
<td align="left" colspan="1" rowspan="1">0.280</td>
<td align="left" colspan="1" rowspan="1">851.703</td>
<td align="left" colspan="1" rowspan="1">0.878</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. ICA</td>
<td align="left" colspan="1" rowspan="1">0.327</td>
<td align="left" colspan="1" rowspan="1">1040.542</td>
<td align="left" colspan="1" rowspan="1">1.115</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. NMF</td>
<td align="left" colspan="1" rowspan="1">0.273</td>
<td align="left" colspan="1" rowspan="1">824.599</td>
<td align="left" colspan="1" rowspan="1">0.636</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. FA</td>
<td align="left" colspan="1" rowspan="1">0.281</td>
<td align="left" colspan="1" rowspan="1">980.704</td>
<td align="left" colspan="1" rowspan="1">0.459</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. TSVD</td>
<td align="left" colspan="1" rowspan="1">0.282</td>
<td align="left" colspan="1" rowspan="1">866.688</td>
<td align="left" colspan="1" rowspan="1">0.892</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. DICL</td>
<td align="left" colspan="1" rowspan="1">0.142</td>
<td align="left" colspan="1" rowspan="1">828.204</td>
<td align="left" colspan="1" rowspan="1">0.298</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. DIFF</td>
<td align="left" colspan="1" rowspan="1">0.006</td>
<td align="left" colspan="1" rowspan="1">769.142</td>
<td align="left" colspan="1" rowspan="1">−0.079</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. PALANTIR</td>
<td align="left" colspan="1" rowspan="1">0.012</td>
<td align="left" colspan="1" rowspan="1">773.901</td>
<td align="left" colspan="1" rowspan="1">−0.033</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab9"><h4 class="obj_head">Table 9.</h4>
<div class="caption p"><p>Performance advantage of GNODEVAE over alternative dimensionality reduction methods: Clustering geometry metrics in scRNA-seq cancer datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq122"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3312.gif" loading="lazy" id="d33e3312" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance, with DAV improvements shown as positive values in the comparison results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. PCA</td>
<td align="left" colspan="1" rowspan="1">0.450</td>
<td align="left" colspan="1" rowspan="1">2937.293</td>
<td align="left" colspan="1" rowspan="1">1.215</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. ICA</td>
<td align="left" colspan="1" rowspan="1">0.496</td>
<td align="left" colspan="1" rowspan="1">3098.999</td>
<td align="left" colspan="1" rowspan="1">1.342</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. NMF</td>
<td align="left" colspan="1" rowspan="1">0.441</td>
<td align="left" colspan="1" rowspan="1">2948.179</td>
<td align="left" colspan="1" rowspan="1">0.994</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. FA</td>
<td align="left" colspan="1" rowspan="1">0.423</td>
<td align="left" colspan="1" rowspan="1">3037.520</td>
<td align="left" colspan="1" rowspan="1">0.747</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. TSVD</td>
<td align="left" colspan="1" rowspan="1">0.463</td>
<td align="left" colspan="1" rowspan="1">2954.963</td>
<td align="left" colspan="1" rowspan="1">1.262</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. DICL</td>
<td align="left" colspan="1" rowspan="1">0.284</td>
<td align="left" colspan="1" rowspan="1">2859.419</td>
<td align="left" colspan="1" rowspan="1">0.612</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. DIFF</td>
<td align="left" colspan="1" rowspan="1">0.133</td>
<td align="left" colspan="1" rowspan="1">2736.732</td>
<td align="left" colspan="1" rowspan="1">0.184</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. PALANTIR</td>
<td align="left" colspan="1" rowspan="1">0.142</td>
<td align="left" colspan="1" rowspan="1">2763.099</td>
<td align="left" colspan="1" rowspan="1">0.245</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab10"><h4 class="obj_head">Table 10.</h4>
<div class="caption p"><p>Performance advantage of GNODEVAE over alternative dimensionality reduction methods: Clustering geometry metrics in scATAC-seq datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq123"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3415.gif" loading="lazy" id="d33e3415" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance, with DAV improvements shown as positive values in the comparison results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. PCA</td>
<td align="left" colspan="1" rowspan="1">0.498</td>
<td align="left" colspan="1" rowspan="1">4999.375</td>
<td align="left" colspan="1" rowspan="1">1.257</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. ICA</td>
<td align="left" colspan="1" rowspan="1">0.484</td>
<td align="left" colspan="1" rowspan="1">5009.348</td>
<td align="left" colspan="1" rowspan="1">0.549</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. NMF</td>
<td align="left" colspan="1" rowspan="1">0.373</td>
<td align="left" colspan="1" rowspan="1">4565.803</td>
<td align="left" colspan="1" rowspan="1">0.155</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. FA</td>
<td align="left" colspan="1" rowspan="1">0.064</td>
<td align="left" colspan="1" rowspan="1">4462.961</td>
<td align="left" colspan="1" rowspan="1">−0.289</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. TSVD</td>
<td align="left" colspan="1" rowspan="1">0.469</td>
<td align="left" colspan="1" rowspan="1">4977.946</td>
<td align="left" colspan="1" rowspan="1">0.761</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. DICL</td>
<td align="left" colspan="1" rowspan="1">0.071</td>
<td align="left" colspan="1" rowspan="1">4503.363</td>
<td align="left" colspan="1" rowspan="1">−0.245</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. DIFF</td>
<td align="left" colspan="1" rowspan="1">0.320</td>
<td align="left" colspan="1" rowspan="1">4955.734</td>
<td align="left" colspan="1" rowspan="1">0.450</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. PALANTIR</td>
<td align="left" colspan="1" rowspan="1">0.325</td>
<td align="left" colspan="1" rowspan="1">4967.382</td>
<td align="left" colspan="1" rowspan="1">0.541</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec42"><h3 class="pmc_sec_title">GNODEVAE’s systematic advantages over VAE-based generative models in single-cell clustering</h3>
<p id="Par119">Systematic comparison across 45 datasets demonstrates GNODEVAE’s consistent and significant performance advantages over VAE-based generative models across all dataset types, with particularly pronounced benefits in high-dimensional sparse data contexts that confirm the integration of neural ordinary differential equation dynamics represents a critical architectural evolution addressing fundamental limitations of static latent variable models (Fig. <a href="#Fig5" class="usa-link">5</a>, Tables <a href="#Tab11" class="usa-link">11</a>, <a href="#Tab12" class="usa-link">12</a> and <a href="#Tab13" class="usa-link">13</a>). In scRNA-seq normal developmental datasets, GNODEVAE achieved systematic improvements compared to all VAE variants with ASW enhancements ranging from +0.218 to +0.318 and CAL improvements from +482.294 to +934.374, while VAEs with strong <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq124"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/f5e0e8aa5fab/d33e3526.gif" loading="lazy" id="d33e3526" alt="Inline graphic"></span> regualrization (highBeta-VAE, TC-VAE, DIP-VAE) demonstrated the poorest performance with DAV improvements of +1.779 to +2.552, indicating that excessive regularization severely compromises clustering geometric performance, and among single-cell specialized methods, scTour showed the best performance within VAE-based approaches but still exhibited significant gaps (ASW: +0.237, CAL: +482.294), demonstrating that despite scTour’s integration of ODE modules for trajectory inference, GNODEVAE maintains significant advantages potentially attributable to the unique role of GAT attention mechanisms that dynamically aggregate neighborhood information through attention weights, incorporating graph structural prior knowledge to more accurately capture intercellular interaction patterns and state transition relationships during continuous-time evolution, while traditional VAE methods including scTour lack this explicit modeling capability for graph structural information (Fig. <a href="#Fig5" class="usa-link">5</a>A, Table <a href="#Tab11" class="usa-link">11</a>). scRNA-seq cancer datasets revealed larger performance gaps with GNODEVAE achieving ASW improvements ranging from +0.290 to +0.403 and CAL enhancements from +966.434 to +1409.069, with the systematic amplification of performance advantages compared to normal developmental datasets reflecting the exponentially greater benefits provided by continuous-time modeling frameworks as cellular state complexity increases (Fig. <a href="#Fig5" class="usa-link">5</a>B, Table <a href="#Tab12" class="usa-link">12</a>). scATAC-seq datasets demonstrated the most pronounced performance advantages with CAL improvements ranging from +1991.597 to +2423.563 and ASW enhancements from +0.312 to +0.393, with even the best-performing VAE variant scTour remaining significantly inferior (CAL: +1991.597), demonstrating the particular suitability of continuous-time modeling for processing sparse binary chromatin accessibility data, while performance advantages exhibit a clear hierarchical pattern scaling with dataset complexity: scATAC-seq datasets (CAL: +1991.597 to +2423.563) &gt; scRNA-seq cancer datasets (CAL: +966.434 to +1409.069) &gt; scRNA-seq normal developmental datasets (CAL: +482.294 to +934.374), indicating that GNODEVAE’s architectural innovations become increasingly important as single-cell genomics advances toward more complex multi-modal and spatially resolved datasets, where static latent variable models face fundamental limitations in capturing underlying biological complexity (Fig. <a href="#Fig5" class="usa-link">5</a>C, Table <a href="#Tab13" class="usa-link">13</a>). These systematic results confirm that GNODEVAE achieves systematic advantages across all evaluation criteria and dataset types through continuous-time modeling without requiring complex regularization strategies or domain-specific modifications, with particularly pronounced performance in high-dimensional sparse data contexts suggesting that future single-cell generative modeling development should prioritize continuous-time dynamics over static latent representations, while explicit modeling of graph structural information proves indispensable for accurately capturing cellular state transitions, establishing the combined framework of attention mechanisms and continuous-time dynamics as the optimal approach for single-cell analysis applications.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369121_12864_2025_11946_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/b2782d1dc4ec/12864_2025_11946_Fig5_HTML.jpg" loading="lazy" id="MO5" height="843" width="778" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Clustering performance comparison between GNODEVAE and deep generative VAE-based variants. Box plots display performance comparisons across VAE variant approaches (from left to right: lowBeta-VAE, highBeta-VAE, DIP-VAE, TC-VAE, INFO-VAE, scVI, scTour) versus GNODE (GNODEVAE) architecture. <strong>A</strong> Performance comparison on scRNA-seq normal developmental datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq125"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3561.gif" loading="lazy" id="d33e3561" alt="Inline graphic"></span>). Statistical analysis employed repeated measures ANOVA with Tukey’s HSD post-hoc tests for normally distributed metrics (ARI, NMI), or Friedman tests with Wilcoxon signed-rank post-hoc tests for non-normally distributed metrics (ASW, CAL, DAV). <strong>B</strong> Performance comparison on scRNA-seq cancer datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq126"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3570.gif" loading="lazy" id="d33e3570" alt="Inline graphic"></span>). Statistical analysis employed repeated measures ANOVA with Tukey’s HSD post-hoc tests for normally distributed metrics (ARI, NMI), or Friedman tests with Wilcoxon signed-rank post-hoc tests for non-normally distributed metrics (ASW, CAL, DAV). <strong>C</strong> Performance comparison on scATAC-seq datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq127"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3580.gif" loading="lazy" id="d33e3580" alt="Inline graphic"></span>). Statistical analysis employed repeated measures ANOVA with Tukey’s HSD post-hoc tests for normally distributed metrics (DAV), or Friedman tests with Wilcoxon signed-rank post-hoc tests for non-normally distributed metrics (ARI, NMI, ASW, CAL). Significance levels are indicated as: *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001 with Bonferroni correction</p></figcaption></figure><section class="tw xbox font-sm" id="Tab11"><h4 class="obj_head">Table 11.</h4>
<div class="caption p"><p>Performance advantage of GNODEVAE over VAE-based and single-cell specific methods: Clustering geometry metrics in scRNA-seq normal developmental datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq128"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3596.gif" loading="lazy" id="d33e3596" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance, with DAV improvements shown as positive values in the comparison results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. lowBeta-VAE</td>
<td align="left" colspan="1" rowspan="1">0.218</td>
<td align="left" colspan="1" rowspan="1">776.157</td>
<td align="left" colspan="1" rowspan="1">0.844</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. highBeta-VAE</td>
<td align="left" colspan="1" rowspan="1">0.318</td>
<td align="left" colspan="1" rowspan="1">934.374</td>
<td align="left" colspan="1" rowspan="1">2.552</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. DIP-VAE</td>
<td align="left" colspan="1" rowspan="1">0.300</td>
<td align="left" colspan="1" rowspan="1">927.476</td>
<td align="left" colspan="1" rowspan="1">1.779</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. TC-VAE</td>
<td align="left" colspan="1" rowspan="1">0.293</td>
<td align="left" colspan="1" rowspan="1">898.463</td>
<td align="left" colspan="1" rowspan="1">1.958</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. INFO-VAE</td>
<td align="left" colspan="1" rowspan="1">0.261</td>
<td align="left" colspan="1" rowspan="1">843.788</td>
<td align="left" colspan="1" rowspan="1">1.443</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. scVI</td>
<td align="left" colspan="1" rowspan="1">0.246</td>
<td align="left" colspan="1" rowspan="1">855.164</td>
<td align="left" colspan="1" rowspan="1">1.175</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. scTour</td>
<td align="left" colspan="1" rowspan="1">0.237</td>
<td align="left" colspan="1" rowspan="1">482.294</td>
<td align="left" colspan="1" rowspan="1">1.406</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab12"><h4 class="obj_head">Table 12.</h4>
<div class="caption p"><p>Performance advantage of GNODEVAE over VAE-based and single-cell specific methods: Clustering geometry metrics in scRNA-seq cancer datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq129"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3690.gif" loading="lazy" id="d33e3690" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance, with DAV improvements shown as positive values in the comparison results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. lowBeta-VAE</td>
<td align="left" colspan="1" rowspan="1">0.290</td>
<td align="left" colspan="1" rowspan="1">1271.003</td>
<td align="left" colspan="1" rowspan="1">1.033</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. highBeta-VAE</td>
<td align="left" colspan="1" rowspan="1">0.403</td>
<td align="left" colspan="1" rowspan="1">1409.069</td>
<td align="left" colspan="1" rowspan="1">2.732</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. DIP-VAE</td>
<td align="left" colspan="1" rowspan="1">0.377</td>
<td align="left" colspan="1" rowspan="1">1398.378</td>
<td align="left" colspan="1" rowspan="1">1.751</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. TC-VAE</td>
<td align="left" colspan="1" rowspan="1">0.376</td>
<td align="left" colspan="1" rowspan="1">1377.754</td>
<td align="left" colspan="1" rowspan="1">2.038</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. INFO-VAE</td>
<td align="left" colspan="1" rowspan="1">0.337</td>
<td align="left" colspan="1" rowspan="1">1323.624</td>
<td align="left" colspan="1" rowspan="1">1.440</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. scVI</td>
<td align="left" colspan="1" rowspan="1">0.339</td>
<td align="left" colspan="1" rowspan="1">1348.412</td>
<td align="left" colspan="1" rowspan="1">1.364</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. scTour</td>
<td align="left" colspan="1" rowspan="1">0.316</td>
<td align="left" colspan="1" rowspan="1">966.434</td>
<td align="left" colspan="1" rowspan="1">1.494</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab13"><h4 class="obj_head">Table 13.</h4>
<div class="caption p"><p>Performance advantage of GNODEVAE over VAE-based and single-cell specific methods: Clustering geometry metrics in scATAC-seq datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq130"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3784.gif" loading="lazy" id="d33e3784" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance, with DAV improvements shown as positive values in the comparison results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. lowBeta-VAE</td>
<td align="left" colspan="1" rowspan="1">0.338</td>
<td align="left" colspan="1" rowspan="1">2336.509</td>
<td align="left" colspan="1" rowspan="1">1.467</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. highBeta-VAE</td>
<td align="left" colspan="1" rowspan="1">0.393</td>
<td align="left" colspan="1" rowspan="1">2421.822</td>
<td align="left" colspan="1" rowspan="1">2.863</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. DIP-VAE</td>
<td align="left" colspan="1" rowspan="1">0.349</td>
<td align="left" colspan="1" rowspan="1">2423.563</td>
<td align="left" colspan="1" rowspan="1">2.281</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. TC-VAE</td>
<td align="left" colspan="1" rowspan="1">0.363</td>
<td align="left" colspan="1" rowspan="1">2369.312</td>
<td align="left" colspan="1" rowspan="1">2.088</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. INFO-VAE</td>
<td align="left" colspan="1" rowspan="1">0.334</td>
<td align="left" colspan="1" rowspan="1">2319.370</td>
<td align="left" colspan="1" rowspan="1">1.636</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. scVI</td>
<td align="left" colspan="1" rowspan="1">0.361</td>
<td align="left" colspan="1" rowspan="1">2385.277</td>
<td align="left" colspan="1" rowspan="1">1.857</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. scTour</td>
<td align="left" colspan="1" rowspan="1">0.312</td>
<td align="left" colspan="1" rowspan="1">1991.597</td>
<td align="left" colspan="1" rowspan="1">1.408</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab13/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec43"><h3 class="pmc_sec_title">Comparative analysis of GNODEVAE against graph-based and contrastive learning approaches</h3>
<p id="Par123">Benchmarking against graph learning and contrastive learning predictive models reveals complex performance patterns that illuminate GNODEVAE’s multi-layered characteristics as a generative continuous-time modeling method (Fig. <a href="#Fig6" class="usa-link">6</a>, Tables <a href="#Tab14" class="usa-link">14</a>, <a href="#Tab15" class="usa-link">15</a> and <a href="#Tab16" class="usa-link">16</a>). In scRNA-seq normal developmental datasets, GNODEVAE achieved moderate advantages in clustering geometry over the pure graph learning method scGNN (ASW: +0.184, CAL: +707.409) and larger performance gaps relative to scGCC, which integrates graph learning with contrastive learning (ASW: +0.202, CAL: +794.492), while demonstrating the most significant advantages over the pure contrastive learning method CLEAR (ASW: +0.386, CAL: +1147.797, DAV: +2.428), reflecting the fundamental importance of generative modeling in maintaining continuity of cellular state transitions through reconstruction regularization, and for reconstruction quality metrics, GNODEVAE achieved statistically significant improvements in ARI and NMI compared to both scGCC and scGNN, demonstrating the advantages of generative modeling in maintaining data reconstruction fidelity, while differences with CLEAR in ARI and NMI did not reach statistical significance, reflecting the competitiveness of contrastive learning methods in certain reconstruction-related tasks (Fig. <a href="#Fig6" class="usa-link">6</a>A, Table <a href="#Tab14" class="usa-link">14</a>). In scRNA-seq cancer datasets, GNODEVAE maintained strong clustering geometry advantages over the integrated method scGCC (ASW: +0.147, CAL: +1258.799), showed larger performance gaps relative to the pure graph learning method scGNN (ASW: +0.215, CAL: +1300.083), and continued to maintain maximum geometric advantages over the pure contrastive learning method CLEAR (ASW: +0.375, CAL: +1537.944, DAV: +2.585), with reconstruction quality metrics similarly showing significant improvements for GNODEVAE relative to scGCC and scGNN in ARI and NMI while showing no significant differences with CLEAR, further confirming the consistency of this pattern and the superiority of continuous-time generative modeling in maintaining data representation fidelity (Fig. <a href="#Fig6" class="usa-link">6</a>B, Table <a href="#Tab15" class="usa-link">15</a>). scATAC-seq datasets presented a striking reversal pattern that demonstrates the unique value of specialized method integration in specific data modalities, with scGCC, which integrates graph learning and contrastive learning, achieving superior clustering geometry performance over GNODEVAE (ASW: −0.074, CAL: −8863.300), while GNODEVAE still maintained significant advantages over the pure graph learning method scGNN (ASW: +0.256, CAL: +3990.052) and maximum advantages over the pure contrastive learning method CLEAR (ASW: +0.498, CAL: +4224.173, DAV: +3.025), yet for reconstruction quality metrics, scATAC-seq data exhibited patterns distinctly different from RNA data where GNODEVAE showed significant differences in ARI and NMI only when compared to scGCC while showing no significant differences with either scGNN or CLEAR, reflecting the unique characteristics of sparse binary chromatin accessibility data where the complexity of reconstruction tasks causes different modeling paradigms to converge in reconstruction quality while clustering geometry differences remain significant (Fig. <a href="#Fig6" class="usa-link">6</a>C, Table <a href="#Tab16" class="usa-link">16</a>).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369121_12864_2025_11946_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/419bc3fad242/12864_2025_11946_Fig6_HTML.jpg" loading="lazy" id="MO6" height="812" width="778" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Clustering performance comparison between GNODEVAE and deep graph learning with contrastive learning-based methods. Box plots display performance comparisons across deep graph and contrastive learning approaches (from left to right: scGCC, scGNN, CLEAR) versus GNODE (GNODEVAE) architecture. <strong>A</strong> Performance comparison on scRNA-seq normal developmental datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq131"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3914.gif" loading="lazy" id="d33e3914" alt="Inline graphic"></span>). Statistical analysis employed repeated measures ANOVA with Tukey’s HSD post-hoc tests for normally distributed metrics (ASW, DAV), or Friedman tests with Wilcoxon signed-rank post-hoc tests for non-normally distributed metrics (ARI, NMI, CAL). <strong>B</strong> Performance comparison on scRNA-seq cancer datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq132"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3923.gif" loading="lazy" id="d33e3923" alt="Inline graphic"></span>). Statistical analysis employed repeated measures ANOVA with Tukey’s HSD post-hoc tests for normally distributed metrics (ARI, NMI), or Friedman tests with Wilcoxon signed-rank post-hoc tests for non-normally distributed metrics (ASW, CAL, DAV). <strong>C</strong> Performance comparison on scATAC-seq datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq133"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3933.gif" loading="lazy" id="d33e3933" alt="Inline graphic"></span>). Statistical analysis employed repeated measures ANOVA with Tukey’s HSD post-hoc tests for normally distributed metrics (DAV), or Friedman tests with Wilcoxon signed-rank post-hoc tests for non-normally distributed metrics (ARI, NMI, ASW, CAL). Significance levels are indicated as: *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001 with Bonferroni correction</p></figcaption></figure><section class="tw xbox font-sm" id="Tab14"><h4 class="obj_head">Table 14.</h4>
<div class="caption p"><p>Performance advantage of GNODEVAE over graph-based and contrastive learning methods: Clustering geometry metrics in scRNA-seq normal developmental datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq134"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e3949.gif" loading="lazy" id="d33e3949" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance, with DAV improvements shown as positive values in the comparison results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. scGCC</td>
<td align="left" colspan="1" rowspan="1">0.202</td>
<td align="left" colspan="1" rowspan="1">794.492</td>
<td align="left" colspan="1" rowspan="1">0.541</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. scGNN</td>
<td align="left" colspan="1" rowspan="1">0.184</td>
<td align="left" colspan="1" rowspan="1">707.409</td>
<td align="left" colspan="1" rowspan="1">0.344</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. CLEAR</td>
<td align="left" colspan="1" rowspan="1">0.386</td>
<td align="left" colspan="1" rowspan="1">1147.797</td>
<td align="left" colspan="1" rowspan="1">2.428</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab14/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab15"><h4 class="obj_head">Table 15.</h4>
<div class="caption p"><p>Performance advantage of GNODEVAE over graph-based and contrastive learning methods: Clustering geometry metrics in scRNA-seq cancer datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq135"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e4007.gif" loading="lazy" id="d33e4007" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance, with DAV improvements shown as positive values in the comparison results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. scGCC</td>
<td align="left" colspan="1" rowspan="1">0.147</td>
<td align="left" colspan="1" rowspan="1">1258.799</td>
<td align="left" colspan="1" rowspan="1">0.541</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. scGNN</td>
<td align="left" colspan="1" rowspan="1">0.215</td>
<td align="left" colspan="1" rowspan="1">1300.083</td>
<td align="left" colspan="1" rowspan="1">0.534</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. CLEAR</td>
<td align="left" colspan="1" rowspan="1">0.375</td>
<td align="left" colspan="1" rowspan="1">1537.944</td>
<td align="left" colspan="1" rowspan="1">2.585</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab15/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab16"><h4 class="obj_head">Table 16.</h4>
<div class="caption p"><p>Performance advantage of GNODEVAE over graph-based and contrastive learning methods: Clustering geometry metrics in scATAC-seq datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq136"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e4065.gif" loading="lazy" id="d33e4065" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance, with DAV improvements shown as positive values in the comparison results</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. scGCC</td>
<td align="left" colspan="1" rowspan="1">−0.074</td>
<td align="left" colspan="1" rowspan="1">−8863.300</td>
<td align="left" colspan="1" rowspan="1">−0.041</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. scGNN</td>
<td align="left" colspan="1" rowspan="1">0.256</td>
<td align="left" colspan="1" rowspan="1">3990.052</td>
<td align="left" colspan="1" rowspan="1">0.845</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. CLEAR</td>
<td align="left" colspan="1" rowspan="1">0.498</td>
<td align="left" colspan="1" rowspan="1">4224.173</td>
<td align="left" colspan="1" rowspan="1">3.025</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab16/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec44"><h3 class="pmc_sec_title">Batch integration performance and computational efficiency evaluation</h3>
<p id="Par127">GNODEVAE’s batch integration evaluation across multiple scales reveals a consistent performance-efficiency trade-off pattern (Fig. <a href="#Fig7" class="usa-link">7</a>). At 2,000 cells, GNODEVAE maintained equivalent cell-type neighborhood purity compared to advanced methods (scVI, scTour, scGNN, scGCC, CLEAR) on the CLISI metric but significantly outperformed scVI, scTour, scGNN, and scGCC on the ILISI batch mixing metric while achieving comparable performance to CLEAR, demonstrating that the neural ODE framework’s continuous-time dynamics modeling and graph attention mechanisms effectively remove batch-specific technical artifacts while preserving underlying biological variation (Fig. <a href="#Fig7" class="usa-link">7</a>A). However, this superior integration performance comes at the cost of highest computational runtime overhead, establishing the fundamental trade-off relationship between architectural sophistication and computational demands. As cell populations expanded to 4,000, 6,000, and 8,000 cells, GNODEVAE sustained its equivalent CLISI performance and enhanced ILISI batch mixing effectiveness, with this performance pattern becoming further consolidated across scales, indicating the scalable nature of its integration advantages (Fig. <a href="#Fig7" class="usa-link">7</a>B-D). Simultaneously, the systematic growth pattern of computational overhead became increasingly pronounced with dataset complexity, reflecting the characteristic challenge where neural ODE architectural complexity directly correlates with data scale and computational resource requirements. These findings demonstrate that GNODEVAE achieves superior and scalable batch integration performance through its continuous-time modeling framework, but the significant computational costs present challenges for routine analysis workflows and provide important considerations for practical deployment in large-scale single-cell genomics applications.</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369121_12864_2025_11946_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/5db4851f13ad/12864_2025_11946_Fig7_HTML.jpg" loading="lazy" id="MO7" height="886" width="778" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Batch integration performance and computational efficiency comparison across different cell population sizes. (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq137"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/4967b3ebbeba/d33e4134.gif" loading="lazy" id="d33e4134" alt="Inline graphic"></span>) Each panel displays batch integration metrics (CLISI and ILISI) as box plots, indicating statistical significance relative to GNODE (GNODEVAE), alongside computational runtime represented by bar plots showing mean values across 13 plus 5 additional multi-batch datasets. Methods evaluated include scVI, scTour, scGNN, scGCC, and CLEAR (from left to right). <strong>A</strong> Performance at 2,000 cells. Statistical analysis employed repeated measures ANOVA with Tukey’s HSD post-hoc tests for normally distributed CLISI metric and Friedman tests with Wilcoxon signed-rank post-hoc tests for non-normally distributed ILISI metric. Right panel displays computational runtime for this dataset size. <strong>B</strong> Performance at 4,000 cells. Statistical analysis employed repeated measures ANOVA with Tukey’s HSD post-hoc tests for normally distributed CLISI metric and Friedman tests with Wilcoxon signed-rank post-hoc tests for non-normally distributed ILISI metric. Right panel displays computational runtime for this dataset size. <strong>C</strong> Performance at 6,000 cells. Statistical analysis employed Friedman tests with Wilcoxon signed-rank post-hoc tests for all non-normally distributed metrics. Right panel displays computational runtime for this dataset size. <strong>D</strong> Performance at 8,000 cells. Statistical analysis employed Friedman tests with Wilcoxon signed-rank post-hoc tests for all non-normally distributed metrics. Right panel displays computational runtime for this dataset size. Significance levels are indicated as: *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001 with Bonferroni correction</p></figcaption></figure></section><section id="Sec45"><h3 class="pmc_sec_title">Hyperparameter sensitivity and component ablation analysis</h3>
<p id="Par128">Systematic ablation studies based on scRNA-seq normal developmental datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq138"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e4158.gif" loading="lazy" id="d33e4158" alt="Inline graphic"></span>) reveal critical architectural insights into GNODEVAE’s design, demonstrating that reconstruction weight analysis with W-recon=10 achieves optimal performance across all metrics with significant improvements in cluster separation (CAL: +1101.365 to +1260.340) and geometric quality (ASW: +0.151 to +0.172) compared to lower weights, establishing reconstruction fidelity as the fundamental driver of GNODEVAE’s clustering capabilities and reflecting the inherent characteristics of VAE theoretical architectures where maximum impact stems from relative weight changes in KL regularization terms (Table <a href="#Tab17" class="usa-link">17</a>, Fig. <a href="#Fig8" class="usa-link">8</a>A). ODE regularization exhibits more complex optimization dynamics with different weights achieving optimal performance across different metrics-ODE-recon=1 achieves optimal clustering accuracy while ODE-recon=10 excels in geometric metrics (ASW: +0.036, CAL: +334.560)—where moderate performance differences (±0.025 for ARI/NMI) primarily reflect that KL terms respond less sensitively to ODE changes compared to main reconstruction terms, but this does not diminish the fundamental importance of ODE terms as evidenced by GNODEVAE’s significant advantages over VGAE across 45 datasets in previous architectural improvement experiments (Table <a href="#Tab18" class="usa-link">18</a>, Fig. <a href="#Fig8" class="usa-link">8</a>B). KL weight analysis confirms the complex role of distributional regularization, with W-kl=0.01 achieving optimal results across most metrics while high KL weights (W-kl=10) severely degrade performance (ARI: −0.088, ASW: −0.131), reflecting the fundamental trade-off between latent component independence and biological relevance where lower KL regularization leads to entangled latent components better suited to the biological nature of single-cell data, and changes in KL weights lead to significant variations in coupling between latent space components, which theoretically resembles GNODEVAE’s improvement principles by enhancing biologically meaningful coupling (Table <a href="#Tab19" class="usa-link">19</a>, Fig. <a href="#Fig8" class="usa-link">8</a>C). Component ablation studies validate each GNODEVAE element’s importance, with reconstruction knockout (RECON-KO) producing severe performance degradation across all metrics (ARI: −0.061, CAL: −261.542) while KL knockout (KL-KO) shows improved geometric metrics (CAL: +323.498) but degraded clustering accuracy (ARI: −0.034), confirming that KL regularization must be carefully calibrated to balance statistical constraints with biological meaningfulness, and suggesting that future neural ODE clustering architectures should prioritize reconstruction fidelity over distributional regularization, potentially requiring integration of specialized correlation regulation modules rather than relying on traditional KL constraints (Table <a href="#Tab20" class="usa-link">20</a>, Fig. <a href="#Fig8" class="usa-link">8</a>D).</p>
<section class="tw xbox font-sm" id="Tab17"><h4 class="obj_head">Table 17.</h4>
<div class="caption p"><p>Ablation study of reconstruction weight impact: W-recon=10 vs. other weight configurations in scRNA-seq normal developmental datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq140"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e4223.gif" loading="lazy" id="d33e4223" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ARI</th>
<th align="left" colspan="1" rowspan="1">NMI</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
<th align="left" colspan="1" rowspan="1">COR</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">W-recon=10 vs. W-recon=0.01</td>
<td align="left" colspan="1" rowspan="1">0.080</td>
<td align="left" colspan="1" rowspan="1">0.079</td>
<td align="left" colspan="1" rowspan="1">0.156</td>
<td align="left" colspan="1" rowspan="1">1213.360</td>
<td align="left" colspan="1" rowspan="1">−0.482</td>
<td align="left" colspan="1" rowspan="1">1.558</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">W-recon=10 vs. W-recon=0.1</td>
<td align="left" colspan="1" rowspan="1">0.137</td>
<td align="left" colspan="1" rowspan="1">0.136</td>
<td align="left" colspan="1" rowspan="1">0.172</td>
<td align="left" colspan="1" rowspan="1">1260.340</td>
<td align="left" colspan="1" rowspan="1">−0.527</td>
<td align="left" colspan="1" rowspan="1">1.687</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">W-recon=10 vs. W-recon=1</td>
<td align="left" colspan="1" rowspan="1">0.116</td>
<td align="left" colspan="1" rowspan="1">0.115</td>
<td align="left" colspan="1" rowspan="1">0.151</td>
<td align="left" colspan="1" rowspan="1">1101.365</td>
<td align="left" colspan="1" rowspan="1">−0.457</td>
<td align="left" colspan="1" rowspan="1">0.924</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab17/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369121_12864_2025_11946_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/b4534310c59e/12864_2025_11946_Fig8_HTML.jpg" loading="lazy" id="MO8" height="733" width="778" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Hyperparameter sensitivity analysis and ablation study of GNODEVAE architecture. Bar charts represent average performance scores across <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq139"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e4195.gif" loading="lazy" id="d33e4195" alt="Inline graphic"></span> datasets for six evaluation metrics (from left to right: ARI, NMI, ASW, CAL, DAV, COR). Note that lower DAV values indicate better performance <strong>A</strong> Reconstruction loss weight sensitivity analysis with weights ranging from 0.01, 0.1, 1, to 10. <strong>B</strong> ODE loss weight sensitivity analysis with weights ranging from 0.01, 0.1, 1, to 10. <strong>C</strong> KL regularization weight sensitivity analysis with weights ranging from 0.01, 0.1, 1, to 10. <strong>D</strong> Ablation study comparing the complete model architecture with variants excluding reconstruction loss and KL regularization components</p></figcaption></figure><section class="tw xbox font-sm" id="Tab18"><h4 class="obj_head">Table 18.</h4>
<div class="caption p"><p>Ablation study of ODE regularization weight impact: ODE-recon=10 vs. other weight configurations in scRNA-seq normal developmental datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq141"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e4308.gif" loading="lazy" id="d33e4308" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ARI</th>
<th align="left" colspan="1" rowspan="1">NMI</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
<th align="left" colspan="1" rowspan="1">COR</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">ODE-recon=10 vs. ODE-recon=0.01</td>
<td align="left" colspan="1" rowspan="1">0.009</td>
<td align="left" colspan="1" rowspan="1">0.009</td>
<td align="left" colspan="1" rowspan="1">0.036</td>
<td align="left" colspan="1" rowspan="1">334.560</td>
<td align="left" colspan="1" rowspan="1">−0.166</td>
<td align="left" colspan="1" rowspan="1">0.602</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ODE-recon=10 vs. ODE-recon=0.1</td>
<td align="left" colspan="1" rowspan="1">0.013</td>
<td align="left" colspan="1" rowspan="1">0.013</td>
<td align="left" colspan="1" rowspan="1">0.032</td>
<td align="left" colspan="1" rowspan="1">366.199</td>
<td align="left" colspan="1" rowspan="1">−0.163</td>
<td align="left" colspan="1" rowspan="1">0.670</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ODE-recon=10 vs. ODE-recon=1</td>
<td align="left" colspan="1" rowspan="1">−0.025</td>
<td align="left" colspan="1" rowspan="1">−0.025</td>
<td align="left" colspan="1" rowspan="1">0.011</td>
<td align="left" colspan="1" rowspan="1">175.830</td>
<td align="left" colspan="1" rowspan="1">−0.101</td>
<td align="left" colspan="1" rowspan="1">0.175</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab18/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab19"><h4 class="obj_head">Table 19.</h4>
<div class="caption p"><p>Ablation study of KL divergence weight impact: W-kl=0.01 vs. other weight configurations in scRNA-seq normal developmental datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq142"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e4393.gif" loading="lazy" id="d33e4393" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ARI</th>
<th align="left" colspan="1" rowspan="1">NMI</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
<th align="left" colspan="1" rowspan="1">COR</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">W-kl=0.01 vs. W-kl=10</td>
<td align="left" colspan="1" rowspan="1">0.088</td>
<td align="left" colspan="1" rowspan="1">0.087</td>
<td align="left" colspan="1" rowspan="1">0.131</td>
<td align="left" colspan="1" rowspan="1">600.933</td>
<td align="left" colspan="1" rowspan="1">−0.443</td>
<td align="left" colspan="1" rowspan="1">1.387</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">W-kl=0.01 vs. W-kl=1</td>
<td align="left" colspan="1" rowspan="1">0.033</td>
<td align="left" colspan="1" rowspan="1">0.032</td>
<td align="left" colspan="1" rowspan="1">0.068</td>
<td align="left" colspan="1" rowspan="1">399.718</td>
<td align="left" colspan="1" rowspan="1">−0.146</td>
<td align="left" colspan="1" rowspan="1">0.972</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">W-kl=0.01 vs. W-kl=0.1</td>
<td align="left" colspan="1" rowspan="1">0.005</td>
<td align="left" colspan="1" rowspan="1">0.005</td>
<td align="left" colspan="1" rowspan="1">0.014</td>
<td align="left" colspan="1" rowspan="1">−26.119</td>
<td align="left" colspan="1" rowspan="1">−0.021</td>
<td align="left" colspan="1" rowspan="1">0.177</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab19/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="Tab20"><h4 class="obj_head">Table 20.</h4>
<div class="caption p"><p>Ablation study of GNODEVAE components: Performance comparison against reconstruction knockout (RECON-KO) and KL divergence knockout (KL-KO) in scRNA-seq normal developmental datasets (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq143"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/01d870d9b55a/d33e4478.gif" loading="lazy" id="d33e4478" alt="Inline graphic"></span>). Note that lower DAV values indicate better performance</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Comparison</th>
<th align="left" colspan="1" rowspan="1">ARI</th>
<th align="left" colspan="1" rowspan="1">NMI</th>
<th align="left" colspan="1" rowspan="1">ASW</th>
<th align="left" colspan="1" rowspan="1">CAL</th>
<th align="left" colspan="1" rowspan="1">DAV</th>
<th align="left" colspan="1" rowspan="1">COR</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. RECON-KO</td>
<td align="left" colspan="1" rowspan="1">0.061</td>
<td align="left" colspan="1" rowspan="1">0.061</td>
<td align="left" colspan="1" rowspan="1">0.066</td>
<td align="left" colspan="1" rowspan="1">261.542</td>
<td align="left" colspan="1" rowspan="1">−0.203</td>
<td align="left" colspan="1" rowspan="1">0.531</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">GNODE vs. KL-KO</td>
<td align="left" colspan="1" rowspan="1">0.034</td>
<td align="left" colspan="1" rowspan="1">0.034</td>
<td align="left" colspan="1" rowspan="1">−0.012</td>
<td align="left" colspan="1" rowspan="1">−323.498</td>
<td align="left" colspan="1" rowspan="1">0.080</td>
<td align="left" colspan="1" rowspan="1">−0.604</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab20/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="Sec46"><h3 class="pmc_sec_title">Biological validation and application using hematopoietic differentiation data</h3>
<p id="Par133">Comprehensive biological validation using hematopoietic differentiation data demonstrates GNODEVAE’s capacity to capture differentiation dynamics through gene-level interpretation, utilizing a validation framework designed to assess continuous differentiation processes through gene-level dynamics analysis and subpopulation-level pseudotime inference (Fig. <a href="#Fig9" class="usa-link">9</a>A). Systematic comparison of pseudotime inference revealed distinct differences between GNODEVAE and established trajectory inference methods (Diffusion map and Palantir) in temporal ordering quality, with GNODEVAE providing both overall pseudotime sampling and subpopulation-level pseudotime resolution that better explains subpopulation-level differentiation processes, as confirmed by visual inspection of t-SNE embeddings showing that the continuous-time modeling approach more effectively captures temporal relationships between gene expression changes during differentiation (Fig. <a href="#Fig9" class="usa-link">9</a>B). Comprehensive evaluation using ASW, CAL, and DAV metrics demonstrated GNODEVAE’s superior capability in organizing genes according to dynamic expression patterns, achieving higher composite scores across all clustering geometry metrics compared to Diffusion map and Palantir methods, indicating that the neural ODE framework more effectively captures the temporal dynamics inherent in biological differentiation processes (Fig. <a href="#Fig9" class="usa-link">9</a>C). Dynamic gene expression analysis revealed that genes with high dynamic scores were not consistently identified as significant markers through conventional approaches, with subpopulation visualization and analysis of the top 15 genes ranked by dynamic expression scores demonstrating that GNODEVAE identifies genes undergoing significant temporal changes during differentiation but exhibiting non-static differential expression patterns, as confirmed by volcano plot analysis showing that many dynamically important genes occupy non-significant positions in conventional marker gene rankings, highlighting the complementary nature of dynamic versus static gene expression analysis (Fig. <a href="#Fig9" class="usa-link">9</a>D). Gene Ontology Biological Process (GOBP) enrichment analysis provided strong validation for the biological relevance of GNODEVAE-identified dynamic genes, with dynamically identified genes showing enrichment for diverse biological processes directly related to cellular differentiation—including cytokine production, cell death, cell division, and hematopoietic differentiation—while conventional marker genes enriched for more specialized functional categories such as translation, biosynthesis, and protein folding, confirming that GNODEVAE’s continuous-time modeling approach enables more precise analysis of biological continuous processes captured in single-cell snapshots by identifying genes that participate in active differentiation processes rather than simply distinguishing between static cell states (Fig. <a href="#Fig9" class="usa-link">9</a>E).</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12369121_12864_2025_11946_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/17d73ac811a7/12864_2025_11946_Fig9_HTML.jpg" loading="lazy" id="MO9" height="867" width="778" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Biological validation and interpretation of GNODEVAE using hematopoietic differentiation data. <strong>A</strong> Workflow of the validation process for identifying dynamically changing genes during differentiation. <strong>B</strong> Pseudotime comparison across methods. From left to right: GNODE (GNODEVAE) overall pseudotime sampling, GNODE subpopulation-level pseudotime, diffusion map pseudotime, and Palantir pseudotime projected onto t-SNE two-dimensional embeddings. <strong>C</strong> Gene clustering performance comparison using ASW, CAL, and DAV metrics. <strong>D</strong> Application of GNODE for dynamic gene expression analysis compared with conventional gene analysis methods. From left to right: subpopulation visualization on t-SNE embedding, top 15 genes ranked by dynamic expression scores, and volcano plot showing the significance and positioning of these 15 genes under conventional marker gene discovery approaches. <strong>E</strong> Gene Ontology Biological Process (GOBP) enrichment validation comparing dynamically identified genes (left) with conventional marker genes (right)</p></figcaption></figure></section></section><section id="Sec47"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par134">The intrinsic biological snapshot nature of single-cell data fundamentally motivates our exploration of continuous-time modeling architectures. Single-cell sequencing technologies capture molecular snapshots of cells that exist along continuous biological trajectories, whether during normal development, disease progression, or chromatin accessibility changes [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. This biological reality necessitates computational methods capable of modeling the continuous nature of cellular state transitions, rather than treating cells as discrete, static entities. Our systematic progression from traditional machine learning to deep learning methods, from predictive to generative models, and from contrastive to continuous learning reflects a deeper understanding of how to authentically model biological processes. The evolution toward continuous-time frameworks represents not merely technical advancement, but a fundamental alignment between computational modeling and biological reality, where cellular differentiation, activation, and transformation processes occur through gradual, continuous changes rather than discrete jumps.</p>
<p id="Par135">GNODEVAE’s architecture addresses this biological imperative through the strategic integration of GAT, NODE, and VAE components. The GAT component captures local structural relationships through attention mechanisms, ensuring optimal modeling of cellular neighborhoods [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>], while the NODE component directly models continuous cellular states through differential equation-based dynamics. This combination creates a framework that simultaneously maintains local cellular relationships and global trajectory dynamics. Systematic evaluation of graph convolutional layers confirmed GAT’s superiority in single-cell applications, with attention mechanisms effectively capturing graph topology in the data and providing interpretable weights corresponding to biological relationships between cell types. The NODE component’s ability to transform static representations into continuous-time dynamical systems enables faithful modeling of gradual cellular state changes, revealing patterns of differentiation acceleration and deceleration consistent with known biological phenomena [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>].</p>
<p id="Par136">Our extensive comparative analysis across multiple method categories provides compelling evidence for the superiority of continuous-time modeling approaches. GNODEVAE’s systematic advantages over VAE-based generative models demonstrate that neural ODE integration addresses fundamental limitations of static latent variable models. Consistent performance improvements across all dataset types, particularly the significant advantages in high-dimensional sparse data environments, establish continuous-time dynamics as a key architectural innovation. Comparisons with traditional dimensionality reduction methods reveal that while classical approaches offer computational efficiency, they systematically sacrifice clustering quality for speed. The statistical significance of GNODEVAE’s improvements across all evaluation metrics indicates that the computational overhead of neural ODE-based modeling yields substantial analytical benefits, particularly for exploratory analysis and cell type discovery tasks. Evaluation against graph-based and contrastive learning methods highlights the fundamental importance of reconstruction regularization for single-cell applications, as contrastive learning methods, despite their theoretical advantages in discriminative tasks, systematically fail due to the lack of reconstruction objectives that preserve the continuous nature of cellular state transitions.</p>
<p id="Par137">Systematic ablation studies reveal fundamental design principles extending beyond GNODEVAE, providing guidance for the broader field of single-cell method development. The complex interactions between reconstruction weights, ODE weights, and KL weights reveal inherent properties of VAE architectures, where adjustment of any weight fundamentally alters the balance between reconstruction fidelity, dynamic regularization, and distributional constraints. The complex effects of KL regularization warrant particular attention, with lower KL weights (W-kl=0.01) achieving optimal performance while high KL weights severely degrade clustering quality, reflecting a fundamental trade-off between latent component independence and biological relevance. This trade-off mechanism suggests that KL regularization in traditional VAE architectures may be inappropriate for the biologically relevant nature of single-cell data, requiring specialized correlation adjustment modules to balance statistical constraints with biological meaningfulness. GNODEVAE’s continuous-time dynamics partially mitigate this issue through implicit regularization, but future architectural designs should more explicitly address the tension between biological relevance and statistical independence.</p>
<p id="Par138">
Despite GNODEVAE’s significant performance advantages, its computational complexity presents practical challenges for real-world applications. The integration and solution processes of neural ODEs require substantially more computational resources compared to traditional methods, particularly on large-scale datasets. Increased training time and memory requirements limit the method’s scalability, which becomes particularly pronounced when processing modern single-cell datasets containing hundreds of thousands of cells. Additionally, hyperparameter sensitivity requires more careful tuning processes, adding complexity to method application. However, these computational costs must be considered within the context of modern artificial intelligence (AI) infrastructure, where initial training complexity can be mitigated through strategies such as model reuse, transfer learning, and knowledge distillation. For research applications pursuing high-quality analytical results, such computational investment is justified, particularly considering the fundamental advantages of continuous-time modeling in capturing biological processes.</p>
<p id="Par139">
These architectural insights have profound implications for the development of biological foundation models. As the field moves toward large generative models capable of predicting biological outcomes, the fundamental architectural exploration demonstrated here directly benefits foundational choices for large-scale single-cell models. The consistent superiority of continuous-time modeling across diverse data types and analytical tasks suggests that future foundation models should prioritize neural ODE-based architectures over traditional static methods. The particularly significant advantages observed in scATAC-seq data, where sparse, high-dimensional chromatin accessibility measurements amplify the limitations of traditional methods, indicate that continuous-time modeling will become increasingly important as single-cell technologies expand to new modalities.</p></section><section id="Sec48"><h2 class="pmc_sec_title">Conclusions</h2>
<p id="Par140">This study presents GNODEVAE, a novel single-cell analysis architecture integrating Graph Attention Networks, Neural Ordinary Differential Equations, and Variational Autoencoders. Through systematic evaluation across 45 diverse datasets, including 5 additional datasets specifically utilized for multi-batch data analysis, GNODEVAE consistently outperforms 18 existing benchmark methods across all clustering metrics, including 8 machine learning dimensionality reduction techniques, 7 deep generative VAE variants, and 3 graph-based and contrastive learning deep predictive models, demonstrating the fundamental advantages of continuous-time modeling in single-cell genomics. Architectural ablation studies reveal the critical importance of reconstruction fidelity and the complex effects of KL regularization in single-cell data, while biological validation confirms the biological relevance of dynamic gene identification. These findings establish continuous-time neural ODE modeling as a key advancement in single-cell analysis, providing a solid foundation for next-generation analytical methods capable of capturing the full complexity of biological systems.</p></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Not applicable.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>GNODEVAE</dt>
<dd><p id="Par4">Graph Neural Ordinary Differential Equation Variational Autoencoder</p></dd>
<dt>GAT</dt>
<dd><p id="Par5">Graph Attention Network</p></dd>
<dt>NODE</dt>
<dd><p id="Par6">Neural Ordinary Differential Equation</p></dd>
<dt>VAE</dt>
<dd><p id="Par7">Variational Autoencoder</p></dd>
<dt>VGAE</dt>
<dd><p id="Par8">Variational Graph Autoencoder</p></dd>
<dt>GNN</dt>
<dd><p id="Par9">Graph Neural Network</p></dd>
<dt>ELBO</dt>
<dd><p id="Par10">Evidence Lower Bound</p></dd>
<dt>GCN</dt>
<dd><p id="Par11">Graph Convolutional Network</p></dd>
<dt>ChebNet</dt>
<dd><p id="Par12">Chebyshev Spectral Graph Convolutional Network</p></dd>
<dt>GraphSAGE</dt>
<dd><p id="Par13">Graph Sample and Aggregate</p></dd>
<dt>TAGConv</dt>
<dd><p id="Par14">Topology Adaptive Graph Convolutional Network</p></dd>
<dt>ARMA</dt>
<dd><p id="Par15">Autoregressive Moving Average Graph Neural Network</p></dd>
<dt>SGConv</dt>
<dd><p id="Par16">Simple Graph Convolution</p></dd>
<dt>SSGConv</dt>
<dd><p id="Par17">Simple Spectral Graph Convolution</p></dd>
<dt>MLP</dt>
<dd><p id="Par18">Multi-Layer Perceptron</p></dd>
<dt>PCA</dt>
<dd><p id="Par19">Principal Component Analysis</p></dd>
<dt>ICA</dt>
<dd><p id="Par20">Independent Component Analysis</p></dd>
<dt>NMF</dt>
<dd><p id="Par21">Non-negative Matrix Factorization</p></dd>
<dt>FA</dt>
<dd><p id="Par22">Factor Analysis</p></dd>
<dt>SVD</dt>
<dd><p id="Par23">Singular Value Decomposition</p></dd>
<dt>DICL</dt>
<dd><p id="Par24">Dictionary Learning</p></dd>
<dt>ARI</dt>
<dd><p id="Par25">Adjusted Rand Index</p></dd>
<dt>NMI</dt>
<dd><p id="Par26">Normalized Mutual Information</p></dd>
<dt>ASW</dt>
<dd><p id="Par27">Average Silhouette Width</p></dd>
<dt>CAL</dt>
<dd><p id="Par28">Calinski-Harabasz Index</p></dd>
<dt>DAV</dt>
<dd><p id="Par29">Davies-Bouldin Index</p></dd>
<dt>COR</dt>
<dd><p id="Par30">Latent Space Correlation</p></dd>
<dt>ILISI</dt>
<dd><p id="Par31">Integration Local Inverse Simpson’s Index</p></dd>
<dt>CLISI</dt>
<dd><p id="Par32">Cell-type Local Inverse Simpson’s Index</p></dd>
<dt>GOBP</dt>
<dd><p id="Par33">Gene Oncological Biological Process</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Authors'contributions</h2>
<p>Z.F.: Conceptualization, methodology, software, validation, formal analysis, investigation, data curation, writing-original draft, writing-review and editing, visualization. C.C.: Visualization, formal analysis, data curation, writing-original draft, writing-review and editing. S.W.: Resources, supervision, project administration. J.W.: Resources, supervision, project administration, funding acquisition. S.C.: Writing-review and editing, resources, supervision, project administration, funding acquisition.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This research was funded by National Natural Science Foundation of China of funder grant number 82222060, 82430103, 82473572, 81930090, 81725019, 82073487, 81602790.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>All data generated or analysed during this study are included in this published article, its supplementary information files and publicly available repositories. The source code for this study is publicly available in the GitHub repository at <a href="https://github.com/PeterPonyu/GNODEVAE" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/PeterPonyu/GNODEVAE</a> and Zenodo at <a href="https://doi.org/10.5281/zenodo.15826043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.15826043</a>.</p>
<p> Datasets utilized for developmental and normal tissue analyses:</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq144"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4642.gif" loading="lazy" id="d33e4642" alt="Inline graphic"></span> Bastidas-Ponce, A., Tritschler, S., Leander, D., Scheibner, K., Tarquis-Medina, M., Salinno, C., Schirge, S., Burtscher, I., Böttcher, A., Theis, F. J., Lickert, H. and Bakhti, M. Comprehensive single-cell mRNA profiling reveals a detailed roadmap for pancreatic endocrinogenesis. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE132188" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE132188</a> (2025)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq145"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4653.gif" loading="lazy" id="d33e4653" alt="Inline graphic"></span> Bandyopadhyay, S., Ahn, K. J., Duffy, M., Qin, L. and Tan, K. Mapping the Cellular Biogeography of Human Bone Marrow Niches Using Single-Cell Transcriptomics. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?</a>acc=<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE253355" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE253355</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq146"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4664.gif" loading="lazy" id="d33e4664" alt="Inline graphic"></span> Alkaslasi, M. R., Piccus, Z. E., Hareendran, S., Silberberg, H., Chen, L., Zhang, Y., Petros, T. J. and Le Pichon, C. E. Single nucleus RNA-sequencing defines unexpected diversity of cholinergic neuron types in the adult mouse spinal cord. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE167597" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE167597</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq147"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4675.gif" loading="lazy" id="d33e4675" alt="Inline graphic"></span> Wang, H., He, J., Xu, C., Chen, X., Yang, H., Shi, L., Liu, B. and Zhou, J. Decoding Human Megakaryocyte Development. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE144024" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE144024</a> (2020)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq148"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4686.gif" loading="lazy" id="d33e4686" alt="Inline graphic"></span> Spildrejorde, M., Samara, A., Sharma, A., Falck, M., Leithaug, M., Modafferi, S., Bjornstad, P. M., Gervin, K., Lyle, R. and Eskeland, R. Gene expression analysis of hESCs undergoing neuronal differentiation [scRNA-seq]. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192857" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192857</a> (2023)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq149"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4697.gif" loading="lazy" id="d33e4697" alt="Inline graphic"></span> Poscablo, D., Forsberg, C. and Medina, P. An age specific platelet differentiation path from hematopoietic stem cells causes thrombocytosis and exacerbated thrombosis. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE255019" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE255019</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq150"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4708.gif" loading="lazy" id="d33e4708" alt="Inline graphic"></span> Cosgrove, J. and Perie, L. Single-cell RNA sequencing of hematopoietic stem and progenitor cells from young and aged mice. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE226131" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE226131</a> (2023)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq151"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4719.gif" loading="lazy" id="d33e4719" alt="Inline graphic"></span> Fast, E. External signals regulate continuous transcriptional states in hematopoietic stem cells. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165844" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165844</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq152"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4731.gif" loading="lazy" id="d33e4731" alt="Inline graphic"></span> Simon, L. M. and Schiller, H. B. Longitudinal single-cell transcriptomics analysis of mouse lung upon bleomycin-induced injury. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE141259" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE141259</a> (2022)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq153"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4742.gif" loading="lazy" id="d33e4742" alt="Inline graphic"></span> Linnarsson, S. Transcriptome analysis of single cells from the mouse dentate gyrus. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95753" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95753</a> (2020)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq154"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4753.gif" loading="lazy" id="d33e4753" alt="Inline graphic"></span> Demerdash, Y., Bouman, B. J., Haghverdi, L. and Essers, M. A. Time series single-cell RNA sequencing of murine hematopoetic stem and progenitors (HSPCs) following in vivo IFNα treatment. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE226824" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE226824</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq155"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4764.gif" loading="lazy" id="d33e4764" alt="Inline graphic"></span> Setty, M. Profiling of CD34+ cells from human bone marrow to understand hematopoiesis. ENA <a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB37166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/ena/browser/view/PRJEB37166</a> (2019)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq156"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4775.gif" loading="lazy" id="d33e4775" alt="Inline graphic"></span> Zhu, Y., Wang, T., Gu, J. and Pan, G. Characterization and generation of human definitive multipotent hematopoietic stem/progenitor cells. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148215" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148215</a> (2020)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq157"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4786.gif" loading="lazy" id="d33e4786" alt="Inline graphic"></span> Dillon, L. Human Bone Marrow Assessment by Single Cell RNA Sequencing, Mass Cytometry and Flow Cytometry. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE120446" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE120446</a> (2018)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq158"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4797.gif" loading="lazy" id="d33e4797" alt="Inline graphic"></span> Teo, Y. V., Hinthorn, S. J., Webb, A. E. and Nicola, N. Single-cell RNA seq of aging peripheral blood. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE120505" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE120505</a> (2023)</p>
<p> Datasets utilized for cancer-related analyses:</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq159"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4809.gif" loading="lazy" id="d33e4809" alt="Inline graphic"></span> Mehtonen, J., Teppo, S., Lahnalampi, M., Kokko, A., Kaukonen, R., Oksa, L. et al. Single cell characterization of arrested B-lymphoid differentiation and leukemic cell states in ETV6-RUNX1-positive pediatric leukemia. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148218" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148218</a> (2020)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq160"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4820.gif" loading="lazy" id="d33e4820" alt="Inline graphic"></span> Li, Z. and Luo, L. Single-Cell RNA Sequencing Identifies Molecular Biomarkers Predicting Response to CDK4/6 Inhibition in Metastatic HR+/HER2- Breast Cancer. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE262288" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE262288</a> (2025)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq161"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4831.gif" loading="lazy" id="d33e4831" alt="Inline graphic"></span> Geldhof, V., de Rooij, L. P., Sokol, L., Amersfoort, J., De Schepper, M., Rohlenova, K. et al. Single cell atlas identifies lipid-processing and immunomodulatory endothelial cells in healthy and malignant breast. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE155109" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE155109</a> (2022)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq162"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4843.gif" loading="lazy" id="d33e4843" alt="Inline graphic"></span> Yost, K. E., Satpathy, A. T., Wells, D. K., Qi, Y., Kageyama, R., Wang, C. et al. Clonal replacement of tumor-specific T cells following PD-1 blockade [single cells]. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123813" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123813</a> (2019)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq163"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4854.gif" loading="lazy" id="d33e4854" alt="Inline graphic"></span> Wang, L., Dai, J., Han, R. and Jin, W. Single-Cell Map of Diverse Immune Phenotypes in the Metastatic Brain Tumor Microenvironment of Non Small Cell Lung Cancer and triple negative breast cancer. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE143423" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE143423</a> (2025)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq164"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4865.gif" loading="lazy" id="d33e4865" alt="Inline graphic"></span> Laughney, A. M., Hu, J., Campbell, N. R., Bakhoum, S. F., Setty, M., Lavallee et al. The single cell transcriptional landscape of human lung adenocarcinoma (primary tumours and metastases). GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123902" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123902</a> (2020)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq165"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4876.gif" loading="lazy" id="d33e4876" alt="Inline graphic"></span> Liu, Y., Ge, J., Chen, Y. and Yu, K. Single-cell profiling and spatial transcriptome of primary tumors and paired metastatic lymph nodes in breast cancer patients. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE225600" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE225600</a> (2023)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq166"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4887.gif" loading="lazy" id="d33e4887" alt="Inline graphic"></span> Yang, J., Yu, J., Huang, X. and Li, Y. Single-cell network pharmacology predicts total therapies targeting multiple developmental clones in B-cell acute lymphoblastic leukemia. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235787" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235787</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq167"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4898.gif" loading="lazy" id="d33e4898" alt="Inline graphic"></span> Chen, X. and Li, Z. Targeting tumor cells toward the antigenic specificity of bystander T cells in tumor microenvironment potentiates cancer immunotherapy. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE222002" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE222002</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq168"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4909.gif" loading="lazy" id="d33e4909" alt="Inline graphic"></span> Zheng, C., Zheng, L., Yoo, J., Guo, H., Zhang, Y., Guo, X. et al. Landscape of infiltrating T cells in liver cancer revealed by single-cell sequencing. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98638" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98638</a> (2019)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq169"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4920.gif" loading="lazy" id="d33e4920" alt="Inline graphic"></span> Kumar, V., Ramnarayanan, K. and Tan, P. Single-cell atlas of lineage states, tumor microenvironment and subtype specific expression programs in gastric cancer. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE183904" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE183904</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq170"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4931.gif" loading="lazy" id="d33e4931" alt="Inline graphic"></span> Caron, M., St-Onge, P., Sontag, T., Wang, Y. C., Richer, C., Ragoussis, I. et al. Single-cell analysis of childhood leukemia reveals a link between developmental states and ribosomal protein expression as a source of intra-individual heterogeneity. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE132509" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE132509</a> (2021)</p>
<p> Datasets utilized for scATAC-related analyses:</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq171"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4943.gif" loading="lazy" id="d33e4943" alt="Inline graphic"></span> Turner A, Shengen H, Zang C, Miller C. Cell-specific chromatin landscape in human coronary artery resolves regulatory mechanisms of disease risk. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175621" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE175621</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175621" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175621</a> (2023)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq172"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4955.gif" loading="lazy" id="d33e4955" alt="Inline graphic"></span> Guilliams M, Bonnardel J, Haest B, Vanderborght B et al. Spatial proteogenomics reveals distinct and evolutionarily-conserved hepatic macrophage niches. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192742" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE192742</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192742" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192742</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq173"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4966.gif" loading="lazy" id="d33e4966" alt="Inline graphic"></span> Wu J, Castro LN, Suva ML, Bernstein BE. Single-cell transcription and chromatin accessibility profiles of IDH mutant gliomas. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE241745" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE241745</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE241745" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE241745</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq174"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4977.gif" loading="lazy" id="d33e4977" alt="Inline graphic"></span> Spildrejorde M, Samara A, Sharma A, Falck M, Leithaug M, Modafferi S, Bjornstad PM et al. Single-cell ATAC-seq analysis of hESCs undergoing neuronal differentiation [scATAC-seq]. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192856" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE192856</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192856" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192856</a> (2023)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq175"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4988.gif" loading="lazy" id="d33e4988" alt="Inline graphic"></span> Itahashi K, Irie T, Nishikawa H. Integrated multiomics profiling identifies the differentiation program of regulatory T cells in human tumors [scATAC-seq]. GEO accessions: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE211087" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE211087</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE211155" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE211155</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE211155" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE211155</a> (2022)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq176"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4999.gif" loading="lazy" id="d33e4999" alt="Inline graphic"></span> Chen R, DeAngelis MM, Li Y, Wang J, Cheng X, Liang Q, Li J. Single-cell multiomics of the human retina reveals hierarchical transcription factor collaboration in mediating cell type-specific effects of genetic variants on gene regulation. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE247157" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE247157</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE247157" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE247157</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq177"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5010.gif" loading="lazy" id="d33e5010" alt="Inline graphic"></span> Lin YH, Duong HG, Limary AE, Kim ES, Hsu P, Patel SA, Wong WH et al. Small intestine and colon tissue-resident memory CD8+ T cells exhibit transcriptional, epigenetic, and functional heterogeneity in concert with differential dependence on Eomesodermin [scATAC-seq]. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205549" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE205549</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205549" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205549</a> (2025)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq178"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5021.gif" loading="lazy" id="d33e5021" alt="Inline graphic"></span> Roudier MP, Gulati R, Sayar E, Patel RA, Tratt M, Richards HM, Cejas P et al. Patterns of intra- and inter-tumor phenotypic heterogeneity in men with lethal prostate cancer [scATAC-seq]. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE292194" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE292194</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE292194" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE292194</a> (2025)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq179"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5032.gif" loading="lazy" id="d33e5032" alt="Inline graphic"></span> Li F, Xu T, Li J, Hao X et al. Single cell ATAC-seq of mouse hair follicle morphogenesis. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201213" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE201213</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201213" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201213</a> (2023)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq180"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5043.gif" loading="lazy" id="d33e5043" alt="Inline graphic"></span> Wang G, Zhang D, Qin L, Huang B. Chemical-based external stimulation reprograms BJs into FiNs (scATAC-seq). GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE206767" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE206767</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE206767" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE206767</a> (2025)</p>
<p>Datasets utilized for multi-batch specific analyses:</p>
<p> As previously included: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123902" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123902</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE132509" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE132509</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148218" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE148218</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE183904" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE183904</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE222002" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE222002</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE262288" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE262288</a> in cancer-associated datasets; <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148215" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE148215</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165844" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE165844</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192857" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE192857</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE226131" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE226131</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE226824" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE226824</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE141259" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE141259</a> in developmental datasets.</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq181"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5057.gif" loading="lazy" id="d33e5057" alt="Inline graphic"></span> Sousa C, Poovathingal SK, Kaoma T, Azuaje F, Skupin A, Michelucci A. Single-cell transcriptomics reveals distinct microglia signatures under inflammation. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115571" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE115571</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115571" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115571</a> (2018)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq182"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5068.gif" loading="lazy" id="d33e5068" alt="Inline graphic"></span> Henry GH. Single-cell RNA-sequencing of adult mouse lower urinary tracts. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE145929" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE145929</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE145929" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE145929</a> (2020)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq183"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5079.gif" loading="lazy" id="d33e5079" alt="Inline graphic"></span> Yuan S, Liu Q, Hu Z, Mao X. Single-cell RNA-sequencing reveals the heterogeneity of microglia in fibrous membrane derived from proliferative diabetic retinopathy and proliferative vitreoretinopathy. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165784" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE165784</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165784" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165784</a> (2022)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq184"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5090.gif" loading="lazy" id="d33e5090" alt="Inline graphic"></span> Hou J, Bi H, Jiang Q, Gu X. Heterogeneity analysis of astrocytes following spinal cord injury at single-cell resolution. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE189070" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE189070</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE189070" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE189070</a> (2022)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq185"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5101.gif" loading="lazy" id="d33e5101" alt="Inline graphic"></span> Zhang B, Zeng K, Guan R, Yang Y. Single-Cell RNA-Seq analysis reveals macrophage involved in pathogenesis of human sporadic type A aortic dissection. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE213740" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE213740</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE213740" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE213740</a> (2023).</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par141">Not applicable.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par142">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par143">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher's Note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Zeyu Fu and Chunlin Chen contributed equally to this work.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Zeyu Fu, Email: fuzeyu99@126.com.</p>
<p>Junping Wang, Email: wangjunping@tmmu.edu.cn.</p>
<p>Shilei Chen, Email: chen.shilei@foxmail.com.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Stuart T, Satija R. Integrative single-cell analysis. Nat Rev Genet. 2019;20(5):257–72. 10.1038/s41576-019-0093-7.
</cite> [<a href="https://doi.org/10.1038/s41576-019-0093-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30696980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stuart%20T,%20Satija%20R.%20Integrative%20single-cell%20analysis.%20Nat%20Rev%20Genet.%202019;20(5):257%E2%80%9372.%2010.1038/s41576-019-0093-7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Kiselev VY, Andrews TS, Hemberg M. Challenges in unsupervised clustering of single-cell RNA-seq data. Nat Rev Genet. 2019;20(5):273–82. 10.1038/s41576-018-0088-9.
</cite> [<a href="https://doi.org/10.1038/s41576-018-0088-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30617341/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kiselev%20VY,%20Andrews%20TS,%20Hemberg%20M.%20Challenges%20in%20unsupervised%20clustering%20of%20single-cell%20RNA-seq%20data.%20Nat%20Rev%20Genet.%202019;20(5):273%E2%80%9382.%2010.1038/s41576-018-0088-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Luecken MD, Theis FJ. Current best practices in single-cell RNA-seq analysis: a tutorial. Mol Syst Biol. 2019;15(6): e8746. 10.15252/msb.20188746.
</cite> [<a href="https://doi.org/10.15252/msb.20188746" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6582955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31217225/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luecken%20MD,%20Theis%20FJ.%20Current%20best%20practices%20in%20single-cell%20RNA-seq%20analysis:%20a%20tutorial.%20Mol%20Syst%20Biol.%202019;15(6):%20e8746.%2010.15252/msb.20188746." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Saelens W, Cannoodt R, Todorov H, Saeys Y. A comparison of single-cell trajectory inference methods. Nat Biotechnol. 2019;37(5):547–54. 10.1038/s41587-019-0071-9.
</cite> [<a href="https://doi.org/10.1038/s41587-019-0071-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30936559/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saelens%20W,%20Cannoodt%20R,%20Todorov%20H,%20Saeys%20Y.%20A%20comparison%20of%20single-cell%20trajectory%20inference%20methods.%20Nat%20Biotechnol.%202019;37(5):547%E2%80%9354.%2010.1038/s41587-019-0071-9." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Andrews TS, Hemberg M. Identifying cell populations with scRNASeq. Mol Aspects Med. 2018;59:114–22.
</cite> [<a href="https://doi.org/10.1016/j.mam.2017.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28712804/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Andrews%20TS,%20Hemberg%20M.%20Identifying%20cell%20populations%20with%20scRNASeq.%20Mol%20Aspects%20Med.%202018;59:114%E2%80%9322." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Kiselev VY, Kirschner K, Schaub MT, Andrews T, Yiu A, Chandra T, et al. SC3: consensus clustering of single-cell RNA-seq data. Nat Methods. 2017;14(5):483–6.
</cite> [<a href="https://doi.org/10.1038/nmeth.4236" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5410170/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28346451/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kiselev%20VY,%20Kirschner%20K,%20Schaub%20MT,%20Andrews%20T,%20Yiu%20A,%20Chandra%20T,%20et%20al.%20SC3:%20consensus%20clustering%20of%20single-cell%20RNA-seq%20data.%20Nat%20Methods.%202017;14(5):483%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Zhang S, Li X, Lin J, Lin Q, Wong KC. Review of single-cell RNA-seq data clustering for cell-type identification and characterization. RNA. 2023;29(5):517–30.
</cite> [<a href="https://doi.org/10.1261/rna.078965.121" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10158997/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36737104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20S,%20Li%20X,%20Lin%20J,%20Lin%20Q,%20Wong%20KC.%20Review%20of%20single-cell%20RNA-seq%20data%20clustering%20for%20cell-type%20identification%20and%20characterization.%20RNA.%202023;29(5):517%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Peyvandipour A, Saberian N, Shafi A, Donato M, Draghici S. Identification of cell types from single cell data using stable clustering. Sci Rep. 2020;10(1):12349.
</cite> [<a href="https://doi.org/10.1038/s41598-020-66848-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7378075/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32703984/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peyvandipour%20A,%20Saberian%20N,%20Shafi%20A,%20Donato%20M,%20Draghici%20S.%20Identification%20of%20cell%20types%20from%20single%20cell%20data%20using%20stable%20clustering.%20Sci%20Rep.%202020;10(1):12349." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Chen Y, et al. Improving the performance of single-cell RNA-seq data mining based on relative expression orderings. Brief Bioinform. 2023;24(1): bbac556.
</cite> [<a href="https://doi.org/10.1093/bib/bbac556" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9851298/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36528803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20Y,%20et%20al.%20Improving%20the%20performance%20of%20single-cell%20RNA-seq%20data%20mining%20based%20on%20relative%20expression%20orderings.%20Brief%20Bioinform.%202023;24(1):%20bbac556." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Lähnemann D, Köster J, Szczurek E, McCarthy DJ, Hicks SC, Robinson MD, et al. Eleven grand challenges in single-cell data science. Genome Biol. 2020;21(1):31.
</cite> [<a href="https://doi.org/10.1186/s13059-020-1926-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7007675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32033589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?L%C3%A4hnemann%20D,%20K%C3%B6ster%20J,%20Szczurek%20E,%20McCarthy%20DJ,%20Hicks%20SC,%20Robinson%20MD,%20et%20al.%20Eleven%20grand%20challenges%20in%20single-cell%20data%20science.%20Genome%20Biol.%202020;21(1):31." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Trapnell C, Cacchiarelli D, Grimsby J, Pokharel P, Li S, Morse M, et al. The dynamics and regulators of cell fate decisions are revealed by pseudotemporal ordering of single cells. Nat Biotechnol. 2014;32(4):381–6. 10.1038/nbt.2859.
</cite> [<a href="https://doi.org/10.1038/nbt.2859" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4122333/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24658644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Trapnell%20C,%20Cacchiarelli%20D,%20Grimsby%20J,%20Pokharel%20P,%20Li%20S,%20Morse%20M,%20et%20al.%20The%20dynamics%20and%20regulators%20of%20cell%20fate%20decisions%20are%20revealed%20by%20pseudotemporal%20ordering%20of%20single%20cells.%20Nat%20Biotechnol.%202014;32(4):381%E2%80%936.%2010.1038/nbt.2859." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Haghverdi L, Büttner M, Wolf FA, Buettner F, Theis FJ. Diffusion pseudotime robustly reconstructs lineage branching. Nat Methods. 2016;13(10):845–8. 10.1038/nmeth.3971.
</cite> [<a href="https://doi.org/10.1038/nmeth.3971" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27571553/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haghverdi%20L,%20B%C3%BCttner%20M,%20Wolf%20FA,%20Buettner%20F,%20Theis%20FJ.%20Diffusion%20pseudotime%20robustly%20reconstructs%20lineage%20branching.%20Nat%20Methods.%202016;13(10):845%E2%80%938.%2010.1038/nmeth.3971." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Setty M, Tadmor MD, Reich-Zeliger S, Angel O, Salame TM, Kathail P, et al. Wishbone identifies bifurcating developmental trajectories from single-cell data. Nat Biotechnol. 2016;34(6):637–45. 10.1038/nbt.3569.
</cite> [<a href="https://doi.org/10.1038/nbt.3569" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4900897/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27136076/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Setty%20M,%20Tadmor%20MD,%20Reich-Zeliger%20S,%20Angel%20O,%20Salame%20TM,%20Kathail%20P,%20et%20al.%20Wishbone%20identifies%20bifurcating%20developmental%20trajectories%20from%20single-cell%20data.%20Nat%20Biotechnol.%202016;34(6):637%E2%80%9345.%2010.1038/nbt.3569." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Cao J, Spielmann M, Qiu X, Huang X, Ibrahim DM, Hill AJ, et al. The single-cell transcriptional landscape of mammalian organogenesis. Nature. 2019;566(7745):496–502. 10.1038/s41586-019-0969-x.
</cite> [<a href="https://doi.org/10.1038/s41586-019-0969-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6434952/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30787437/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cao%20J,%20Spielmann%20M,%20Qiu%20X,%20Huang%20X,%20Ibrahim%20DM,%20Hill%20AJ,%20et%20al.%20The%20single-cell%20transcriptional%20landscape%20of%20mammalian%20organogenesis.%20Nature.%202019;566(7745):496%E2%80%93502.%2010.1038/s41586-019-0969-x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Setty M, Kiseliovas V, Levine J, Gayoso A, Mazutis L, Pe’er D. Characterization of cell fate probabilities in single-cell data with Palantir. Nat Biotechnol. 2019;37(4):451–60. 10.1038/s41587-019-0068-4.
</cite> [<a href="https://doi.org/10.1038/s41587-019-0068-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7549125/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30899105/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Setty%20M,%20Kiseliovas%20V,%20Levine%20J,%20Gayoso%20A,%20Mazutis%20L,%20Pe%E2%80%99er%20D.%20Characterization%20of%20cell%20fate%20probabilities%20in%20single-cell%20data%20with%20Palantir.%20Nat%20Biotechnol.%202019;37(4):451%E2%80%9360.%2010.1038/s41587-019-0068-4." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Wolf FA, Hamey FK, Plass M, Solana J, Dahlin JS, Göttgens B, et al. PAGA: graph abstraction reconciles clustering with trajectory inference through a topology preserving map of single cells. Genome Biol. 2019;20(1):59. 10.1186/s13059-019-1663-x.
</cite> [<a href="https://doi.org/10.1186/s13059-019-1663-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6425583/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30890159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wolf%20FA,%20Hamey%20FK,%20Plass%20M,%20Solana%20J,%20Dahlin%20JS,%20G%C3%B6ttgens%20B,%20et%20al.%20PAGA:%20graph%20abstraction%20reconciles%20clustering%20with%20trajectory%20inference%20through%20a%20topology%20preserving%20map%20of%20single%20cells.%20Genome%20Biol.%202019;20(1):59.%2010.1186/s13059-019-1663-x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>La Manno G, Soldatov R, Zeisel A, Braun E, Hochgerner H, Petukhov V, et al. RNA velocity of single cells. Nature. 2018;560(7719):494–8. 10.1038/s41586-018-0414-6.
</cite> [<a href="https://doi.org/10.1038/s41586-018-0414-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6130801/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30089906/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?La%20Manno%20G,%20Soldatov%20R,%20Zeisel%20A,%20Braun%20E,%20Hochgerner%20H,%20Petukhov%20V,%20et%20al.%20RNA%20velocity%20of%20single%20cells.%20Nature.%202018;560(7719):494%E2%80%938.%2010.1038/s41586-018-0414-6." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Schiebinger G, Shu J, Tabaka M, Cleary B, Subramanian V, Solomon A, et al. Optimal-transport analysis of single-cell gene expression identifies developmental trajectories in reprogramming. Cell. 2019;176(4):928-943.e22. 10.1016/j.cell.2019.01.006.
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.01.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6402800/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30712874/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schiebinger%20G,%20Shu%20J,%20Tabaka%20M,%20Cleary%20B,%20Subramanian%20V,%20Solomon%20A,%20et%20al.%20Optimal-transport%20analysis%20of%20single-cell%20gene%20expression%20identifies%20developmental%20trajectories%20in%20reprogramming.%20Cell.%202019;176(4):928-943.e22.%2010.1016/j.cell.2019.01.006." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Lange M, Bergen V, Klein M, Setty M, Reuter B, Bakhti M, et al. Cell rank for directed single-cell fate mapping. Nat Methods. 2022;19(2):159–70. 10.1038/s41592-021-01346-6.
</cite> [<a href="https://doi.org/10.1038/s41592-021-01346-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8828480/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35027767/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lange%20M,%20Bergen%20V,%20Klein%20M,%20Setty%20M,%20Reuter%20B,%20Bakhti%20M,%20et%20al.%20Cell%20rank%20for%20directed%20single-cell%20fate%20mapping.%20Nat%20Methods.%202022;19(2):159%E2%80%9370.%2010.1038/s41592-021-01346-6." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Eraslan G, Simon LM, Mircea M, Mueller NS, Theis FJ. Single-cell RNA-seq denoising using a deep count autoencoder. Nat Commun. 2019;10(1):390. 10.1038/s41467-018-07931-2.
</cite> [<a href="https://doi.org/10.1038/s41467-018-07931-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6344535/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30674886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eraslan%20G,%20Simon%20LM,%20Mircea%20M,%20Mueller%20NS,%20Theis%20FJ.%20Single-cell%20RNA-seq%20denoising%20using%20a%20deep%20count%20autoencoder.%20Nat%20Commun.%202019;10(1):390.%2010.1038/s41467-018-07931-2." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Lopez R, Regier J, Cole MB, Jordan MI, Yosef N. Deep generative modeling for single-cell transcriptomics. Nat Methods. 2018;15(12):1053–8. 10.1038/s41592-018-0229-2.
</cite> [<a href="https://doi.org/10.1038/s41592-018-0229-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6289068/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30504886/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lopez%20R,%20Regier%20J,%20Cole%20MB,%20Jordan%20MI,%20Yosef%20N.%20Deep%20generative%20modeling%20for%20single-cell%20transcriptomics.%20Nat%20Methods.%202018;15(12):1053%E2%80%938.%2010.1038/s41592-018-0229-2." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Zeng H, Zhou H, Srivastava A, Kannan R, Prasanna V. GraphSAINT: graph sampling based inductive learning method. In: International Conference on Learning Representations (ICLR), 2020. Red Hook: Curran Associates Inc.; 2020.</cite>
</li>
<li id="CR23">
<span class="label">23.</span><cite>Wang J, Ma A, Chang Y, Gong J, Jiang Y, Qi R, et al. Scgnn is a novel graph neural network framework for single-cell RNA-seq analyses. Nat Commun. 2021;12(1):1302. 10.1038/s41467-021-22197-x.
</cite> [<a href="https://doi.org/10.1038/s41467-021-22197-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7994447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33767197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20J,%20Ma%20A,%20Chang%20Y,%20Gong%20J,%20Jiang%20Y,%20Qi%20R,%20et%20al.%20Scgnn%20is%20a%20novel%20graph%20neural%20network%20framework%20for%20single-cell%20RNA-seq%20analyses.%20Nat%20Commun.%202021;12(1):1302.%2010.1038/s41467-021-22197-x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Veličković P, Cucurull G, Casanova A, Romero A, Liò P, Bengio Y. Graph attention networks. In: International Conference on Learning Representations (ICLR), 2018. Red Hook: Curran Associates Inc.; 2018.</cite>
</li>
<li id="CR25">
<span class="label">25.</span><cite>Chen RTQ, Rubanova Y, Bettencourt J, Duvenaud DK. Neural ordinary differential equations. In: Bengio S, Wallach H, Larochelle H, Grauman K, Cesa-Bianchi N, Garnett R, editors. Advances in Neural Information Processing Systems 31 (NeurIPS 2018). Red Hook: Curran Associates Inc.; 2018. pp. 6571–83.</cite>
</li>
<li id="CR26">
<span class="label">26.</span><cite>Tong A, Huang J, Wolf G, van Dijk D, Krishnaswamy S. TrajectoryNet: a dynamic optimal transport network for modeling cellular dynamics. In: Daumé III H, Singh A, editors. Proceedings of the 37th International Conference on Machine Learning, vol. 119. Cambridge: PMLR; 2020. pp. 9526–36.</cite> [<a href="/articles/PMC8320749/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34337419/" class="usa-link">PubMed</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Zhang J, Larschan E, Bigness J, Singh R. scNODE: generative model for temporal single cell transcriptomic data prediction. Bioinformatics. 2024;40(Suppl 2):ii146–54. 10.1093/bioinformatics/btae393.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btae393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11373355/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39230694/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20J,%20Larschan%20E,%20Bigness%20J,%20Singh%20R.%20scNODE:%20generative%20model%20for%20temporal%20single%20cell%20transcriptomic%20data%20prediction.%20Bioinformatics.%202024;40(Suppl%202):ii146%E2%80%9354.%2010.1093/bioinformatics/btae393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Yeo GHT, Saksena SD, Gifford DK. Generative modeling of single-cell time series with PRESCIENT enables prediction of cell trajectories with interventions. Nat Commun. 2021;12(1):3222. 10.1038/s41467-021-23518-w.
</cite> [<a href="https://doi.org/10.1038/s41467-021-23518-w" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8163769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34050150/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yeo%20GHT,%20Saksena%20SD,%20Gifford%20DK.%20Generative%20modeling%20of%20single-cell%20time%20series%20with%20PRESCIENT%20enables%20prediction%20of%20cell%20trajectories%20with%20interventions.%20Nat%20Commun.%202021;12(1):3222.%2010.1038/s41467-021-23518-w." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Moor M, Horn M, Rieck B, Borgwardt K. Topological autoencoders. In: Daumé III H, Singh A, editors. Proceedings of the 37th International Conference on Machine Learning, vol. 119. Cambridge: PMLR; 2020. pp. 7045–54.</cite>
</li>
<li id="CR30">
<span class="label">30.</span><cite>Hofer C, Graf F, Rieck B, Niethammer M, Kwitt R. Graph filtration learning. In: Daumé III H, Singh A, editors. Proceedings of the 37th International Conference on Machine Learning, vol. 119. Cambridge: PMLR; 2020. pp. 4314–23.</cite>
</li>
<li id="CR31">
<span class="label">31.</span><cite>Zhang Z, Zhang X. Inference of high-resolution trajectories in single-cell RNA-seq data by using RNA velocity. Cell Rep Methods. 2021;1(6):100095. 10.1016/j.crmeth.2021.100095.
</cite> [<a href="https://doi.org/10.1016/j.crmeth.2021.100095" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9017235/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35474895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Z,%20Zhang%20X.%20Inference%20of%20high-resolution%20trajectories%20in%20single-cell%20RNA-seq%20data%20by%20using%20RNA%20velocity.%20Cell%20Rep%20Methods.%202021;1(6):100095.%2010.1016/j.crmeth.2021.100095." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Korsunsky I, Millard N, Fan J, Slowikowski K, Zhang F, Wei K, et al. Fast, sensitive and accurate integration of single-cell data with harmony. Nat Methods. 2019;16(12):1289–96. 10.1038/s41592-019-0619-0.
</cite> [<a href="https://doi.org/10.1038/s41592-019-0619-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6884693/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31740819/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Korsunsky%20I,%20Millard%20N,%20Fan%20J,%20Slowikowski%20K,%20Zhang%20F,%20Wei%20K,%20et%20al.%20Fast,%20sensitive%20and%20accurate%20integration%20of%20single-cell%20data%20with%20harmony.%20Nat%20Methods.%202019;16(12):1289%E2%80%9396.%2010.1038/s41592-019-0619-0." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Lotfollahi M, Naghipourfar M, Luecken MD, Khajavi M, Büttner M, Wagenstetter M, et al. Mapping single-cell data to reference atlases by transfer learning. Bioinformatics. 2022;38(8):2089–96. 10.1093/bioinformatics/btac100.</cite> [<a href="https://doi.org/10.1038/s41587-021-01001-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8763644/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34462589/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lotfollahi%20M,%20Naghipourfar%20M,%20Luecken%20MD,%20Khajavi%20M,%20B%C3%BCttner%20M,%20Wagenstetter%20M,%20et%20al.%20Mapping%20single-cell%20data%20to%20reference%20atlases%20by%20transfer%20learning.%20Bioinformatics.%202022;38(8):2089%E2%80%9396.%2010.1093/bioinformatics/btac100." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Amodio M, Van Dijk D, Srinivasan K, Chen WS, Mohsen H, Moon KR, et al. Exploring single-cell data with deep multitasking neural networks. Nat Methods. 2019;16(11):1139–45. 10.1038/s41592-019-0576-7.
</cite> [<a href="https://doi.org/10.1038/s41592-019-0576-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10164410/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31591579/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Amodio%20M,%20Van%20Dijk%20D,%20Srinivasan%20K,%20Chen%20WS,%20Mohsen%20H,%20Moon%20KR,%20et%20al.%20Exploring%20single-cell%20data%20with%20deep%20multitasking%20neural%20networks.%20Nat%20Methods.%202019;16(11):1139%E2%80%9345.%2010.1038/s41592-019-0576-7." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Gayoso A, Steier Z, Lopez R, Regier J, Nazor KL, Streets A, et al. Joint probabilistic modeling of single-cell multi-omic data with totalVI. Nat Methods. 2021;18(3):272–82. 10.1038/s41592-020-01050-x.
</cite> [<a href="https://doi.org/10.1038/s41592-020-01050-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7954949/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33589839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gayoso%20A,%20Steier%20Z,%20Lopez%20R,%20Regier%20J,%20Nazor%20KL,%20Streets%20A,%20et%20al.%20Joint%20probabilistic%20modeling%20of%20single-cell%20multi-omic%20data%20with%20totalVI.%20Nat%20Methods.%202021;18(3):272%E2%80%9382.%2010.1038/s41592-020-01050-x." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Li X, Wang K, Lyu Y, Pan H, Zhang J, Stambolian D, et al. Deep learning enables accurate clustering with batch effect removal in single-cell RNA-seq analysis. Nat Commun. 2020;11(1): 2338. 10.1038/s41467-020-15851-3.
</cite> [<a href="https://doi.org/10.1038/s41467-020-15851-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7214470/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32393754/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20X,%20Wang%20K,%20Lyu%20Y,%20Pan%20H,%20Zhang%20J,%20Stambolian%20D,%20et%20al.%20Deep%20learning%20enables%20accurate%20clustering%20with%20batch%20effect%20removal%20in%20single-cell%20RNA-seq%20analysis.%20Nat%20Commun.%202020;11(1):%202338.%2010.1038/s41467-020-15851-3." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Wu Z, Pan S, Chen F, Long G, Zhang C, Yu PS. A comprehensive survey on graph neural networks. IEEE Trans Neural Netw Learn Syst. 2021;32(1):4–24. 10.1109/TNNLS.2020.2978386.
</cite> [<a href="https://doi.org/10.1109/TNNLS.2020.2978386" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32217482/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20Z,%20Pan%20S,%20Chen%20F,%20Long%20G,%20Zhang%20C,%20Yu%20PS.%20A%20comprehensive%20survey%20on%20graph%20neural%20networks.%20IEEE%20Trans%20Neural%20Netw%20Learn%20Syst.%202021;32(1):4%E2%80%9324.%2010.1109/TNNLS.2020.2978386." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Zhou J, Cui G, Hu S, Zhang Z, Yang C, Liu Z, et al. Graph neural networks: a review of methods and applications. AI Open. 2020;1:57–81. 10.1016/j.aiopen.2021.01.001.</cite> [<a href="https://scholar.google.com/scholar_lookup?Zhou%20J,%20Cui%20G,%20Hu%20S,%20Zhang%20Z,%20Yang%20C,%20Liu%20Z,%20et%20al.%20Graph%20neural%20networks:%20a%20review%20of%20methods%20and%20applications.%20AI%20Open.%202020;1:57%E2%80%9381.%2010.1016/j.aiopen.2021.01.001." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Xu C, Lopez R, Mehlman E, Regier J, Jordan MI, Yosef N. Probabilistic harmonization and annotation of single-cell transcriptomics data with deep generative models. Mol Syst Biol. 2021;17(1): e9620. 10.15252/msb.20209620.
</cite> [<a href="https://doi.org/10.15252/msb.20209620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7829634/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33491336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20C,%20Lopez%20R,%20Mehlman%20E,%20Regier%20J,%20Jordan%20MI,%20Yosef%20N.%20Probabilistic%20harmonization%20and%20annotation%20of%20single-cell%20transcriptomics%20data%20with%20deep%20generative%20models.%20Mol%20Syst%20Biol.%202021;17(1):%20e9620.%2010.15252/msb.20209620." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Lotfollahi M, Naghipourfar M, Theis FJ, Wolf FA. Conditional out-of-distribution generation for unpaired data using transfer VAE. Bioinformatics. 2020;36(Supplement 2):i610–7. 10.1093/bioinformatics/btaa800.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btaa800" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33381839/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lotfollahi%20M,%20Naghipourfar%20M,%20Theis%20FJ,%20Wolf%20FA.%20Conditional%20out-of-distribution%20generation%20for%20unpaired%20data%20using%20transfer%20VAE.%20Bioinformatics.%202020;36(Supplement%202):i610%E2%80%937.%2010.1093/bioinformatics/btaa800." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Minoura K, Abe K, Nam H, Nishi H, Miyoshi H, Shimamura T. A mixture-of-experts deep generative model for integrated analysis of single-cell multiomics data. Cell Rep Methods. 2021;1(5):100071. 10.1016/j.crmeth.2021.100071.
</cite> [<a href="https://doi.org/10.1016/j.crmeth.2021.100071" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9017195/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35474667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Minoura%20K,%20Abe%20K,%20Nam%20H,%20Nishi%20H,%20Miyoshi%20H,%20Shimamura%20T.%20A%20mixture-of-experts%20deep%20generative%20model%20for%20integrated%20analysis%20of%20single-cell%20multiomics%20data.%20Cell%20Rep%20Methods.%202021;1(5):100071.%2010.1016/j.crmeth.2021.100071." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Pan S, Hu R, Long G, Jiang J, Yao L, Zhang C. Adversarially regularized graph autoencoder for graph embedding. In: Lang J, editor. Proceedings of the 27th International Joint Conference on Artificial Intelligence (IJCAI-18). Washington, DC: AAAI Press; 2018. pp. 2609–15.</cite>
</li>
<li id="CR43">
<span class="label">43.</span><cite>Rao J, Zhou X, Lu Y, Zhao H, Yang Y. Imputing single-cell RNA-seq data by combining graph convolution and autoencoder neural networks. iScience. 2021;24(5): 102393. 10.1016/j.isci.2021.102393.
</cite> [<a href="https://doi.org/10.1016/j.isci.2021.102393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8091052/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33997678/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rao%20J,%20Zhou%20X,%20Lu%20Y,%20Zhao%20H,%20Yang%20Y.%20Imputing%20single-cell%20RNA-seq%20data%20by%20combining%20graph%20convolution%20and%20autoencoder%20neural%20networks.%20iScience.%202021;24(5):%20102393.%2010.1016/j.isci.2021.102393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Inoue Y. scVGAE: A Novel Approach using ZINB-Based Variational Graph Autoencoder for Single-Cell RNA-Seq Imputation. 2024. arXiv preprint <a href="http://arxiv.org/abs/2403.08959" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">arXiv:2403.08959</a>.</cite>
</li>
<li id="CR45">
<span class="label">45.</span><cite>Rubanova Y, Chen RTQ, Duvenaud D. Latent ODEs for irregularly-sampled time series. In: Wallach H, Larochelle H, Beygelzimer A, d'Alché-Buc F, Fox E, Garnett R, editors. Advances in Neural Information Processing Systems 32 (NeurIPS 2019). Red Hook: Curran Associates Inc.; 2019. pp. 5320–30.</cite>
</li>
<li id="CR46">
<span class="label">46.</span><cite>Li X, Wong TKL, Chen RTQ, Duvenaud D. Scalable gradients for stochastic differential equations. In: Chiappa S, Calandra R, editors. Proceedings of the 23rd International Conference on Artificial Intelligence and Statistics, vol. 108. Cambridge: PMLR; 2020. pp. 3870–82.</cite>
</li>
<li id="CR47">
<span class="label">47.</span><cite>Li Q. Sctour: a deep learning architecture for robust inference and accurate prediction of cellular dynamics. Genome Biol. 2023;24(1):149.
</cite> [<a href="https://doi.org/10.1186/s13059-023-02988-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10290357/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37353848/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20Q.%20Sctour:%20a%20deep%20learning%20architecture%20for%20robust%20inference%20and%20accurate%20prediction%20of%20cellular%20dynamics.%20Genome%20Biol.%202023;24(1):149." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Tian SW, Ni JC, Wang YT, Zheng CH, Ji CM. scGCC: graph contrastive clustering with neighborhood augmentations for scRNA-seq data analysis. IEEE J Biomed Health Inform. 2023;27(12):6133–43.
</cite> [<a href="https://doi.org/10.1109/JBHI.2023.3319551" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37751336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tian%20SW,%20Ni%20JC,%20Wang%20YT,%20Zheng%20CH,%20Ji%20CM.%20scGCC:%20graph%20contrastive%20clustering%20with%20neighborhood%20augmentations%20for%20scRNA-seq%20data%20analysis.%20IEEE%20J%20Biomed%20Health%20Inform.%202023;27(12):6133%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Han W, Cheng Y, Chen J, Zhong H, Hu Z, Chen S, et al. Self-supervised contrastive learning for integrative single cell RNA-seq data analysis. Brief Bioinform. 2022;23(5): bbac377.
</cite> [<a href="https://doi.org/10.1093/bib/bbac377" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9487595/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36089561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Han%20W,%20Cheng%20Y,%20Chen%20J,%20Zhong%20H,%20Hu%20Z,%20Chen%20S,%20et%20al.%20Self-supervised%20contrastive%20learning%20for%20integrative%20single%20cell%20RNA-seq%20data%20analysis.%20Brief%20Bioinform.%202022;23(5):%20bbac377." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Wang J, Ma A, Chang Y, Ma A, Huang Y, Xu J, et al. scGNN is a novel graph neural network framework for single-cell RNA-seq analyses. Nat Commun. 2021;12:1882.
</cite> [<a href="https://doi.org/10.1038/s41467-021-22197-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7994447/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33767197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20J,%20Ma%20A,%20Chang%20Y,%20Ma%20A,%20Huang%20Y,%20Xu%20J,%20et%20al.%20scGNN%20is%20a%20novel%20graph%20neural%20network%20framework%20for%20single-cell%20RNA-seq%20analyses.%20Nat%20Commun.%202021;12:1882." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Luecken MD, Büttner M, Chaichoompu K, Collier A, Czarnewski P, Malzl D, et al. Benchmarking atlas-level data integration in single-cell genomics. Nat Methods. 2022;19:41–50.
</cite> [<a href="https://doi.org/10.1038/s41592-021-01336-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8748196/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34949812/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Luecken%20MD,%20B%C3%BCttner%20M,%20Chaichoompu%20K,%20Collier%20A,%20Czarnewski%20P,%20Malzl%20D,%20et%20al.%20Benchmarking%20atlas-level%20data%20integration%20in%20single-cell%20genomics.%20Nat%20Methods.%202022;19:41%E2%80%9350." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Weinreb C, Rodriguez-Fraticelli A, Camargo FD, Klein AM. Lineage tracing on transcriptional landscapes links state to fate during differentiation. Science. 2020;367(6479): eaaw3381. 10.1126/science.aaw3381.
</cite> [<a href="https://doi.org/10.1126/science.aaw3381" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7608074/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31974159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Weinreb%20C,%20Rodriguez-Fraticelli%20A,%20Camargo%20FD,%20Klein%20AM.%20Lineage%20tracing%20on%20transcriptional%20landscapes%20links%20state%20to%20fate%20during%20differentiation.%20Science.%202020;367(6479):%20eaaw3381.%2010.1126/science.aaw3381." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Kipf TN, Welling M. Semi-supervised classification with graph convolutional networks. 2016. arXiv preprint <a href="http://arxiv.org/abs/1609.02907" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">arXiv:1609.02907</a>.</cite>
</li>
<li id="CR54">
<span class="label">54.</span><cite>Tritschler S, Büttner M, Fischer DS, Lange M, Bergen V, Lickert H, Theis FJ. Concepts and limitations for learning developmental trajectories from single cell genomics. Development. 2019;146(12): dev170506. 10.1242/dev.170506.
</cite> [<a href="https://doi.org/10.1242/dev.170506" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31249007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tritschler%20S,%20B%C3%BCttner%20M,%20Fischer%20DS,%20Lange%20M,%20Bergen%20V,%20Lickert%20H,%20Theis%20FJ.%20Concepts%20and%20limitations%20for%20learning%20developmental%20trajectories%20from%20single%20cell%20genomics.%20Development.%202019;146(12):%20dev170506.%2010.1242/dev.170506." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>All data generated or analysed during this study are included in this published article, its supplementary information files and publicly available repositories. The source code for this study is publicly available in the GitHub repository at <a href="https://github.com/PeterPonyu/GNODEVAE" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/PeterPonyu/GNODEVAE</a> and Zenodo at <a href="https://doi.org/10.5281/zenodo.15826043" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.5281/zenodo.15826043</a>.</p>
<p> Datasets utilized for developmental and normal tissue analyses:</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4642.gif" loading="lazy" alt="Inline graphic"></span> Bastidas-Ponce, A., Tritschler, S., Leander, D., Scheibner, K., Tarquis-Medina, M., Salinno, C., Schirge, S., Burtscher, I., Böttcher, A., Theis, F. J., Lickert, H. and Bakhti, M. Comprehensive single-cell mRNA profiling reveals a detailed roadmap for pancreatic endocrinogenesis. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE132188" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE132188</a> (2025)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4653.gif" loading="lazy" alt="Inline graphic"></span> Bandyopadhyay, S., Ahn, K. J., Duffy, M., Qin, L. and Tan, K. Mapping the Cellular Biogeography of Human Bone Marrow Niches Using Single-Cell Transcriptomics. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?</a>acc=<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE253355" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE253355</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4664.gif" loading="lazy" alt="Inline graphic"></span> Alkaslasi, M. R., Piccus, Z. E., Hareendran, S., Silberberg, H., Chen, L., Zhang, Y., Petros, T. J. and Le Pichon, C. E. Single nucleus RNA-sequencing defines unexpected diversity of cholinergic neuron types in the adult mouse spinal cord. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE167597" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE167597</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4675.gif" loading="lazy" alt="Inline graphic"></span> Wang, H., He, J., Xu, C., Chen, X., Yang, H., Shi, L., Liu, B. and Zhou, J. Decoding Human Megakaryocyte Development. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE144024" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE144024</a> (2020)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4686.gif" loading="lazy" alt="Inline graphic"></span> Spildrejorde, M., Samara, A., Sharma, A., Falck, M., Leithaug, M., Modafferi, S., Bjornstad, P. M., Gervin, K., Lyle, R. and Eskeland, R. Gene expression analysis of hESCs undergoing neuronal differentiation [scRNA-seq]. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192857" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192857</a> (2023)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4697.gif" loading="lazy" alt="Inline graphic"></span> Poscablo, D., Forsberg, C. and Medina, P. An age specific platelet differentiation path from hematopoietic stem cells causes thrombocytosis and exacerbated thrombosis. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE255019" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE255019</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4708.gif" loading="lazy" alt="Inline graphic"></span> Cosgrove, J. and Perie, L. Single-cell RNA sequencing of hematopoietic stem and progenitor cells from young and aged mice. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE226131" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE226131</a> (2023)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4719.gif" loading="lazy" alt="Inline graphic"></span> Fast, E. External signals regulate continuous transcriptional states in hematopoietic stem cells. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165844" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165844</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4731.gif" loading="lazy" alt="Inline graphic"></span> Simon, L. M. and Schiller, H. B. Longitudinal single-cell transcriptomics analysis of mouse lung upon bleomycin-induced injury. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE141259" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE141259</a> (2022)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4742.gif" loading="lazy" alt="Inline graphic"></span> Linnarsson, S. Transcriptome analysis of single cells from the mouse dentate gyrus. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95753" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE95753</a> (2020)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4753.gif" loading="lazy" alt="Inline graphic"></span> Demerdash, Y., Bouman, B. J., Haghverdi, L. and Essers, M. A. Time series single-cell RNA sequencing of murine hematopoetic stem and progenitors (HSPCs) following in vivo IFNα treatment. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE226824" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE226824</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4764.gif" loading="lazy" alt="Inline graphic"></span> Setty, M. Profiling of CD34+ cells from human bone marrow to understand hematopoiesis. ENA <a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB37166" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ebi.ac.uk/ena/browser/view/PRJEB37166</a> (2019)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4775.gif" loading="lazy" alt="Inline graphic"></span> Zhu, Y., Wang, T., Gu, J. and Pan, G. Characterization and generation of human definitive multipotent hematopoietic stem/progenitor cells. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148215" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148215</a> (2020)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4786.gif" loading="lazy" alt="Inline graphic"></span> Dillon, L. Human Bone Marrow Assessment by Single Cell RNA Sequencing, Mass Cytometry and Flow Cytometry. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE120446" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE120446</a> (2018)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4797.gif" loading="lazy" alt="Inline graphic"></span> Teo, Y. V., Hinthorn, S. J., Webb, A. E. and Nicola, N. Single-cell RNA seq of aging peripheral blood. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE120505" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE120505</a> (2023)</p>
<p> Datasets utilized for cancer-related analyses:</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4809.gif" loading="lazy" alt="Inline graphic"></span> Mehtonen, J., Teppo, S., Lahnalampi, M., Kokko, A., Kaukonen, R., Oksa, L. et al. Single cell characterization of arrested B-lymphoid differentiation and leukemic cell states in ETV6-RUNX1-positive pediatric leukemia. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148218" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148218</a> (2020)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4820.gif" loading="lazy" alt="Inline graphic"></span> Li, Z. and Luo, L. Single-Cell RNA Sequencing Identifies Molecular Biomarkers Predicting Response to CDK4/6 Inhibition in Metastatic HR+/HER2- Breast Cancer. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE262288" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE262288</a> (2025)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4831.gif" loading="lazy" alt="Inline graphic"></span> Geldhof, V., de Rooij, L. P., Sokol, L., Amersfoort, J., De Schepper, M., Rohlenova, K. et al. Single cell atlas identifies lipid-processing and immunomodulatory endothelial cells in healthy and malignant breast. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE155109" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE155109</a> (2022)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4843.gif" loading="lazy" alt="Inline graphic"></span> Yost, K. E., Satpathy, A. T., Wells, D. K., Qi, Y., Kageyama, R., Wang, C. et al. Clonal replacement of tumor-specific T cells following PD-1 blockade [single cells]. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123813" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123813</a> (2019)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4854.gif" loading="lazy" alt="Inline graphic"></span> Wang, L., Dai, J., Han, R. and Jin, W. Single-Cell Map of Diverse Immune Phenotypes in the Metastatic Brain Tumor Microenvironment of Non Small Cell Lung Cancer and triple negative breast cancer. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE143423" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE143423</a> (2025)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4865.gif" loading="lazy" alt="Inline graphic"></span> Laughney, A. M., Hu, J., Campbell, N. R., Bakhoum, S. F., Setty, M., Lavallee et al. The single cell transcriptional landscape of human lung adenocarcinoma (primary tumours and metastases). GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123902" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123902</a> (2020)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4876.gif" loading="lazy" alt="Inline graphic"></span> Liu, Y., Ge, J., Chen, Y. and Yu, K. Single-cell profiling and spatial transcriptome of primary tumors and paired metastatic lymph nodes in breast cancer patients. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE225600" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE225600</a> (2023)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4887.gif" loading="lazy" alt="Inline graphic"></span> Yang, J., Yu, J., Huang, X. and Li, Y. Single-cell network pharmacology predicts total therapies targeting multiple developmental clones in B-cell acute lymphoblastic leukemia. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235787" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE235787</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4898.gif" loading="lazy" alt="Inline graphic"></span> Chen, X. and Li, Z. Targeting tumor cells toward the antigenic specificity of bystander T cells in tumor microenvironment potentiates cancer immunotherapy. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE222002" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE222002</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4909.gif" loading="lazy" alt="Inline graphic"></span> Zheng, C., Zheng, L., Yoo, J., Guo, H., Zhang, Y., Guo, X. et al. Landscape of infiltrating T cells in liver cancer revealed by single-cell sequencing. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98638" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE98638</a> (2019)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4920.gif" loading="lazy" alt="Inline graphic"></span> Kumar, V., Ramnarayanan, K. and Tan, P. Single-cell atlas of lineage states, tumor microenvironment and subtype specific expression programs in gastric cancer. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE183904" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE183904</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4931.gif" loading="lazy" alt="Inline graphic"></span> Caron, M., St-Onge, P., Sontag, T., Wang, Y. C., Richer, C., Ragoussis, I. et al. Single-cell analysis of childhood leukemia reveals a link between developmental states and ribosomal protein expression as a source of intra-individual heterogeneity. GEO <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE132509" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE132509</a> (2021)</p>
<p> Datasets utilized for scATAC-related analyses:</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4943.gif" loading="lazy" alt="Inline graphic"></span> Turner A, Shengen H, Zang C, Miller C. Cell-specific chromatin landscape in human coronary artery resolves regulatory mechanisms of disease risk. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175621" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE175621</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175621" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE175621</a> (2023)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4955.gif" loading="lazy" alt="Inline graphic"></span> Guilliams M, Bonnardel J, Haest B, Vanderborght B et al. Spatial proteogenomics reveals distinct and evolutionarily-conserved hepatic macrophage niches. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192742" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE192742</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192742" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192742</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4966.gif" loading="lazy" alt="Inline graphic"></span> Wu J, Castro LN, Suva ML, Bernstein BE. Single-cell transcription and chromatin accessibility profiles of IDH mutant gliomas. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE241745" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE241745</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE241745" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE241745</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4977.gif" loading="lazy" alt="Inline graphic"></span> Spildrejorde M, Samara A, Sharma A, Falck M, Leithaug M, Modafferi S, Bjornstad PM et al. Single-cell ATAC-seq analysis of hESCs undergoing neuronal differentiation [scATAC-seq]. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192856" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE192856</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192856" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192856</a> (2023)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4988.gif" loading="lazy" alt="Inline graphic"></span> Itahashi K, Irie T, Nishikawa H. Integrated multiomics profiling identifies the differentiation program of regulatory T cells in human tumors [scATAC-seq]. GEO accessions: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE211087" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE211087</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE211155" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE211155</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE211155" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE211155</a> (2022)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e4999.gif" loading="lazy" alt="Inline graphic"></span> Chen R, DeAngelis MM, Li Y, Wang J, Cheng X, Liang Q, Li J. Single-cell multiomics of the human retina reveals hierarchical transcription factor collaboration in mediating cell type-specific effects of genetic variants on gene regulation. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE247157" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE247157</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE247157" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE247157</a> (2024)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5010.gif" loading="lazy" alt="Inline graphic"></span> Lin YH, Duong HG, Limary AE, Kim ES, Hsu P, Patel SA, Wong WH et al. Small intestine and colon tissue-resident memory CD8+ T cells exhibit transcriptional, epigenetic, and functional heterogeneity in concert with differential dependence on Eomesodermin [scATAC-seq]. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205549" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE205549</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205549" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE205549</a> (2025)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5021.gif" loading="lazy" alt="Inline graphic"></span> Roudier MP, Gulati R, Sayar E, Patel RA, Tratt M, Richards HM, Cejas P et al. Patterns of intra- and inter-tumor phenotypic heterogeneity in men with lethal prostate cancer [scATAC-seq]. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE292194" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE292194</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE292194" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE292194</a> (2025)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5032.gif" loading="lazy" alt="Inline graphic"></span> Li F, Xu T, Li J, Hao X et al. Single cell ATAC-seq of mouse hair follicle morphogenesis. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201213" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE201213</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201213" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE201213</a> (2023)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5043.gif" loading="lazy" alt="Inline graphic"></span> Wang G, Zhang D, Qin L, Huang B. Chemical-based external stimulation reprograms BJs into FiNs (scATAC-seq). GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE206767" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE206767</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE206767" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE206767</a> (2025)</p>
<p>Datasets utilized for multi-batch specific analyses:</p>
<p> As previously included: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE123902" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE123902</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE132509" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE132509</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148218" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE148218</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE183904" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE183904</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE222002" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE222002</a> and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE262288" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE262288</a> in cancer-associated datasets; <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148215" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE148215</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165844" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE165844</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE192857" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE192857</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE226131" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE226131</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE226824" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE226824</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE141259" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE141259</a> in developmental datasets.</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5057.gif" loading="lazy" alt="Inline graphic"></span> Sousa C, Poovathingal SK, Kaoma T, Azuaje F, Skupin A, Michelucci A. Single-cell transcriptomics reveals distinct microglia signatures under inflammation. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115571" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE115571</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115571" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE115571</a> (2018)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5068.gif" loading="lazy" alt="Inline graphic"></span> Henry GH. Single-cell RNA-sequencing of adult mouse lower urinary tracts. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE145929" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE145929</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE145929" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE145929</a> (2020)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5079.gif" loading="lazy" alt="Inline graphic"></span> Yuan S, Liu Q, Hu Z, Mao X. Single-cell RNA-sequencing reveals the heterogeneity of microglia in fibrous membrane derived from proliferative diabetic retinopathy and proliferative vitreoretinopathy. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165784" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE165784</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165784" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165784</a> (2022)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5090.gif" loading="lazy" alt="Inline graphic"></span> Hou J, Bi H, Jiang Q, Gu X. Heterogeneity analysis of astrocytes following spinal cord injury at single-cell resolution. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE189070" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE189070</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE189070" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE189070</a> (2022)</p>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><img class="inline" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dfa2/12369121/dcd0eadbe23f/d33e5101.gif" loading="lazy" alt="Inline graphic"></span> Zhang B, Zeng K, Guan R, Yang Y. Single-Cell RNA-Seq analysis reveals macrophage involved in pathogenesis of human sporadic type A aortic dissection. GEO accession: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE213740" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE213740</a>. Available at: <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE213740" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE213740</a> (2023).</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from BMC Genomics are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12864-025-11946-7"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12864_2025_Article_11946.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (18.3 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12369121/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12369121/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12369121%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12369121/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12369121/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12369121/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40841610/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12369121/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40841610/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12369121/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12369121/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="vB4pEhezCrdLlqAD9jPaU8h62rOAkjBsFxAQwAhqLpDk3gpm2qlOmhaEJlOT6umJ">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
