
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Trans-omics analyses identify the biochemical network of LPCAT1 associated with coronary artery disease - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE533B4B8AF34C43053B4B000912ED32.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="biomarkerres">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366056/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Biomarker Research">
<meta name="citation_title" content="Trans-omics analyses identify the biochemical network of LPCAT1 associated with coronary artery disease">
<meta name="citation_author" content="Paul Wei-Che Hsu">
<meta name="citation_author_institution" content="Institute of Molecular and Genomic Medicine, National Health Research Institutes, Miaoli, Taiwan">
<meta name="citation_author" content="Chi-Hsiao Yeh">
<meta name="citation_author_institution" content="Department of Thoracic and Cardiovascular Surgery, Chang Gung Memorial Hospital, Linkou, Taoyuan, Taiwan">
<meta name="citation_author_institution" content="Community Medicine Research Center, Chang Gung Memorial Hospital, Keelung, Taiwan">
<meta name="citation_author_institution" content="College of Medicine, Chang Gung University, Taoyuan, Taiwan">
<meta name="citation_author" content="Chi-Jen Lo">
<meta name="citation_author_institution" content="Metabolomics Core Laboratory, Healthy Aging Research Center, Chang Gung University, Taoyuan, Taiwan">
<meta name="citation_author" content="Tsung-Hsien Tsai">
<meta name="citation_author_institution" content="Advanced Tech BU, Acer Inc, New Taipei City, Taiwan">
<meta name="citation_author" content="Yun-Hsuan Chan">
<meta name="citation_author_institution" content="Advanced Tech BU, Acer Inc, New Taipei City, Taiwan">
<meta name="citation_author" content="Yi-Ju Chou">
<meta name="citation_author_institution" content="Institute of Molecular and Genomic Medicine, National Health Research Institutes, Miaoli, Taiwan">
<meta name="citation_author" content="Ning-I Yang">
<meta name="citation_author_institution" content="Community Medicine Research Center, Chang Gung Memorial Hospital, Keelung, Taiwan">
<meta name="citation_author_institution" content="College of Medicine, Chang Gung University, Taoyuan, Taiwan">
<meta name="citation_author_institution" content="Division of Cardiology, Department of Internal Medicine, Chang Gung Memorial Hospital, Keelung, Taiwan">
<meta name="citation_author" content="Mei-Ling Cheng">
<meta name="citation_author_institution" content="Metabolomics Core Laboratory, Healthy Aging Research Center, Chang Gung University, Taoyuan, Taiwan">
<meta name="citation_author_institution" content="Clinical Metabolomics Core Laboratory, Chang Gung Memorial Hospital, Taoyuan, Taiwan">
<meta name="citation_author_institution" content="Department of Biomedical Sciences, College of Medicine, Chang Gung University, Taoyuan, Taiwan">
<meta name="citation_author" content="Wayne Huey-Herng Sheu">
<meta name="citation_author_institution" content="Institute of Molecular and Genomic Medicine, National Health Research Institutes, Miaoli, Taiwan">
<meta name="citation_author_institution" content="Division of Endocrinology and Metabolism, Department of Internal Medicine, Taipei Veterans General Hospital, Taipei, Taiwan">
<meta name="citation_author_institution" content="School of Medicine, National Yang Ming Chiao Tung University, Taipei, Taiwan">
<meta name="citation_author" content="Chi-Chun Lai">
<meta name="citation_author_institution" content="Community Medicine Research Center, Chang Gung Memorial Hospital, Keelung, Taiwan">
<meta name="citation_author_institution" content="College of Medicine, Chang Gung University, Taoyuan, Taiwan">
<meta name="citation_author_institution" content="Department of Ophthalmology, Chang Gung Memorial Hospital, Keelung, Taiwan">
<meta name="citation_author" content="Huey-Kang Sytwu">
<meta name="citation_author_institution" content="National Institute of Infectious Diseases and Vaccinology, National Health Research Institutes, Miaoli, Taiwan">
<meta name="citation_author_institution" content="Department &amp; Graduate Institute of Microbiology and Immunology, National Defense Medical Center, Taipei, Taiwan">
<meta name="citation_author" content="Ting-Fen Tsai">
<meta name="citation_author_institution" content="Institute of Molecular and Genomic Medicine, National Health Research Institutes, Miaoli, Taiwan">
<meta name="citation_author_institution" content="Department of Life Sciences and Institute of Genome Sciences, National Yang Ming Chiao Tung University, Taipei, Taiwan">
<meta name="citation_author_institution" content="Aging and Health Research Center, National Yang Ming Chiao Tung University, Taipei, Taiwan">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="13">
<meta name="citation_firstpage" content="107">
<meta name="citation_doi" content="10.1186/s40364-025-00821-y">
<meta name="citation_pmid" content="40830908">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366056/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366056/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366056/pdf/40364_2025_Article_821.pdf">
<meta name="description" content="Coronary artery disease (CAD) remains a leading cause of mortality in developed nations. While previous genome-wide association studies have identified single-nucleotide polymorphisms (SNPs) linked to CAD, their impact on disease progression ...">
<meta name="og:title" content="Trans-omics analyses identify the biochemical network of LPCAT1 associated with coronary artery disease">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Coronary artery disease (CAD) remains a leading cause of mortality in developed nations. While previous genome-wide association studies have identified single-nucleotide polymorphisms (SNPs) linked to CAD, their impact on disease progression ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366056/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366056">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s40364-025-00821-y"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/40364_2025_Article_821.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366056%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366056/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366056/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366056/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-biomarkerres.png" alt="Biomarker Research logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Biomarker Research" title="Link to Biomarker Research" shape="default" href="https://biomarkerres.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Biomark Res</button></div>. 2025 Aug 20;13:107. doi: <a href="https://doi.org/10.1186/s40364-025-00821-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s40364-025-00821-y</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Biomark%20Res%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Biomark%20Res%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Biomark%20Res%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Biomark%20Res%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Trans-omics analyses identify the biochemical network of LPCAT1 associated with coronary artery disease</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hsu%20PWC%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Paul Wei-Che Hsu</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Paul Wei-Che Hsu</span></h3>
<div class="p">
<sup>1</sup>Institute of Molecular and Genomic Medicine, National Health Research Institutes, Miaoli, Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hsu%20PWC%22%5BAuthor%5D" class="usa-link"><span class="name western">Paul Wei-Che Hsu</span></a>
</div>
</div>
<sup>1,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yeh%20CH%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Chi-Hsiao Yeh</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Chi-Hsiao Yeh</span></h3>
<div class="p">
<sup>2</sup>Department of Thoracic and Cardiovascular Surgery, Chang Gung Memorial Hospital, Linkou, Taoyuan, Taiwan </div>
<div class="p">
<sup>3</sup>Community Medicine Research Center, Chang Gung Memorial Hospital, Keelung, Taiwan </div>
<div class="p">
<sup>4</sup>College of Medicine, Chang Gung University, Taoyuan, Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yeh%20CH%22%5BAuthor%5D" class="usa-link"><span class="name western">Chi-Hsiao Yeh</span></a>
</div>
</div>
<sup>2,</sup><sup>3,</sup><sup>4,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lo%20CJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Chi-Jen Lo</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Chi-Jen Lo</span></h3>
<div class="p">
<sup>5</sup>Metabolomics Core Laboratory, Healthy Aging Research Center, Chang Gung University, Taoyuan, Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lo%20CJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Chi-Jen Lo</span></a>
</div>
</div>
<sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsai%20TH%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Tsung-Hsien Tsai</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Tsung-Hsien Tsai</span></h3>
<div class="p">
<sup>6</sup>Advanced Tech BU, Acer Inc, New Taipei City, Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsai%20TH%22%5BAuthor%5D" class="usa-link"><span class="name western">Tsung-Hsien Tsai</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chan%20YH%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Yun-Hsuan Chan</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Yun-Hsuan Chan</span></h3>
<div class="p">
<sup>6</sup>Advanced Tech BU, Acer Inc, New Taipei City, Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chan%20YH%22%5BAuthor%5D" class="usa-link"><span class="name western">Yun-Hsuan Chan</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chou%20YJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Yi-Ju Chou</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Yi-Ju Chou</span></h3>
<div class="p">
<sup>1</sup>Institute of Molecular and Genomic Medicine, National Health Research Institutes, Miaoli, Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chou%20YJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Yi-Ju Chou</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20NI%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Ning-I Yang</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Ning-I Yang</span></h3>
<div class="p">
<sup>3</sup>Community Medicine Research Center, Chang Gung Memorial Hospital, Keelung, Taiwan </div>
<div class="p">
<sup>4</sup>College of Medicine, Chang Gung University, Taoyuan, Taiwan </div>
<div class="p">
<sup>7</sup>Division of Cardiology, Department of Internal Medicine, Chang Gung Memorial Hospital, Keelung, Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yang%20NI%22%5BAuthor%5D" class="usa-link"><span class="name western">Ning-I Yang</span></a>
</div>
</div>
<sup>3,</sup><sup>4,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20ML%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Mei-Ling Cheng</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Mei-Ling Cheng</span></h3>
<div class="p">
<sup>5</sup>Metabolomics Core Laboratory, Healthy Aging Research Center, Chang Gung University, Taoyuan, Taiwan </div>
<div class="p">
<sup>8</sup>Clinical Metabolomics Core Laboratory, Chang Gung Memorial Hospital, Taoyuan, Taiwan </div>
<div class="p">
<sup>9</sup>Department of Biomedical Sciences, College of Medicine, Chang Gung University, Taoyuan, Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20ML%22%5BAuthor%5D" class="usa-link"><span class="name western">Mei-Ling Cheng</span></a>
</div>
</div>
<sup>5,</sup><sup>8,</sup><sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sheu%20WHH%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Wayne Huey-Herng Sheu</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Wayne Huey-Herng Sheu</span></h3>
<div class="p">
<sup>1</sup>Institute of Molecular and Genomic Medicine, National Health Research Institutes, Miaoli, Taiwan </div>
<div class="p">
<sup>10</sup>Division of Endocrinology and Metabolism, Department of Internal Medicine, Taipei Veterans General Hospital, Taipei, Taiwan </div>
<div class="p">
<sup>11</sup>School of Medicine, National Yang Ming Chiao Tung University, Taipei, Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sheu%20WHH%22%5BAuthor%5D" class="usa-link"><span class="name western">Wayne Huey-Herng Sheu</span></a>
</div>
</div>
<sup>1,</sup><sup>10,</sup><sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lai%20CC%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Chi-Chun Lai</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Chi-Chun Lai</span></h3>
<div class="p">
<sup>3</sup>Community Medicine Research Center, Chang Gung Memorial Hospital, Keelung, Taiwan </div>
<div class="p">
<sup>4</sup>College of Medicine, Chang Gung University, Taoyuan, Taiwan </div>
<div class="p">
<sup>12</sup>Department of Ophthalmology, Chang Gung Memorial Hospital, Keelung, Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lai%20CC%22%5BAuthor%5D" class="usa-link"><span class="name western">Chi-Chun Lai</span></a>
</div>
</div>
<sup>3,</sup><sup>4,</sup><sup>12,</sup><sup>âœ‰</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sytwu%20HK%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Huey-Kang Sytwu</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Huey-Kang Sytwu</span></h3>
<div class="p">
<sup>13</sup>National Institute of Infectious Diseases and Vaccinology, National Health Research Institutes, Miaoli, Taiwan </div>
<div class="p">
<sup>14</sup>Department &amp; Graduate Institute of Microbiology and Immunology, National Defense Medical Center, Taipei, Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sytwu%20HK%22%5BAuthor%5D" class="usa-link"><span class="name western">Huey-Kang Sytwu</span></a>
</div>
</div>
<sup>13,</sup><sup>14,</sup><sup>âœ‰</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsai%20TF%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Ting-Fen Tsai</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Ting-Fen Tsai</span></h3>
<div class="p">
<sup>1</sup>Institute of Molecular and Genomic Medicine, National Health Research Institutes, Miaoli, Taiwan </div>
<div class="p">
<sup>15</sup>Department of Life Sciences and Institute of Genome Sciences, National Yang Ming Chiao Tung University, Taipei, Taiwan </div>
<div class="p">
<sup>16</sup>Aging and Health Research Center, National Yang Ming Chiao Tung University, Taipei, Taiwan </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Tsai%20TF%22%5BAuthor%5D" class="usa-link"><span class="name western">Ting-Fen Tsai</span></a>
</div>
</div>
<sup>1,</sup><sup>15,</sup><sup>16,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Institute of Molecular and Genomic Medicine, National Health Research Institutes, Miaoli, Taiwan </div>
<div id="Aff2">
<sup>2</sup>Department of Thoracic and Cardiovascular Surgery, Chang Gung Memorial Hospital, Linkou, Taoyuan, Taiwan </div>
<div id="Aff3">
<sup>3</sup>Community Medicine Research Center, Chang Gung Memorial Hospital, Keelung, Taiwan </div>
<div id="Aff4">
<sup>4</sup>College of Medicine, Chang Gung University, Taoyuan, Taiwan </div>
<div id="Aff5">
<sup>5</sup>Metabolomics Core Laboratory, Healthy Aging Research Center, Chang Gung University, Taoyuan, Taiwan </div>
<div id="Aff6">
<sup>6</sup>Advanced Tech BU, Acer Inc, New Taipei City, Taiwan </div>
<div id="Aff7">
<sup>7</sup>Division of Cardiology, Department of Internal Medicine, Chang Gung Memorial Hospital, Keelung, Taiwan </div>
<div id="Aff8">
<sup>8</sup>Clinical Metabolomics Core Laboratory, Chang Gung Memorial Hospital, Taoyuan, Taiwan </div>
<div id="Aff9">
<sup>9</sup>Department of Biomedical Sciences, College of Medicine, Chang Gung University, Taoyuan, Taiwan </div>
<div id="Aff10">
<sup>10</sup>Division of Endocrinology and Metabolism, Department of Internal Medicine, Taipei Veterans General Hospital, Taipei, Taiwan </div>
<div id="Aff11">
<sup>11</sup>School of Medicine, National Yang Ming Chiao Tung University, Taipei, Taiwan </div>
<div id="Aff12">
<sup>12</sup>Department of Ophthalmology, Chang Gung Memorial Hospital, Keelung, Taiwan </div>
<div id="Aff13">
<sup>13</sup>National Institute of Infectious Diseases and Vaccinology, National Health Research Institutes, Miaoli, Taiwan </div>
<div id="Aff14">
<sup>14</sup>Department &amp; Graduate Institute of Microbiology and Immunology, National Defense Medical Center, Taipei, Taiwan </div>
<div id="Aff15">
<sup>15</sup>Department of Life Sciences and Institute of Genome Sciences, National Yang Ming Chiao Tung University, Taipei, Taiwan </div>
<div id="Aff16">
<sup>16</sup>Aging and Health Research Center, National Yang Ming Chiao Tung University, Taipei, Taiwan </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 6; Accepted 2025 Aug 7; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the articleâ€™s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>. The Creative Commons Public Domain Dedication waiver (<a href="https://creativecommons.org/publicdomain/zero/1.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/publicdomain/zero/1.0/</a>) applies to the data made available in this article, unless otherwise stated in a credit line to the data.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366056Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830908/" class="usa-link">40830908</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Coronary artery disease (CAD) remains a leading cause of mortality in developed nations. While previous genome-wide association studies have identified single-nucleotide polymorphisms (SNPs) linked to CAD, their impact on disease progression requires trans-omics validation.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p id="Par2">This study merges whole genome SNP analysis and metabolomic profiling to distinguish CAD patients from high-risk and healthy individuals. A cross-sectional study was conducted, enrolling participants from the Northeastern Taiwan Community Medicine Research Cohort, which spans the period between August 2013 and November 2020. A total of 781 participants were included in the study and categorized into three groups: control (<em>n</em>â€‰=â€‰271), high-risk (<em>n</em>â€‰=â€‰363), and CAD (<em>n</em>â€‰=â€‰147) groups, following a stratification protocol. The study integrated K-clustering of metabolomics and SNP datasets. Subsequently, a machine-learning (ML)-assisted prediction model was developed specifically for CAD identification.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p id="Par3">Four significant findings emerged. Firstly, plasma levels of phospholipids decline from healthy controls to high-risk individuals and then decline further among CAD patients. This indicates that plasma phospholipids have potential as biomarkers and implies that they have a role in CAD progression. Secondly, five genes are linked to lipidomic alterations via their top-ranking among CAD-associated SNPs. Thirdly, a specific LPCAT1 haplotype is associated with CAD using a trans-omics approach. Lastly, an ML-assisted trans-omics prediction model for CAD was developed, which achieves an area under the curve of 0.917, with LPCAT1 among the 16 top-ranked predictive features.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par4">This study highlights the usefulness of a multi-omics signature when discriminating CAD patients and suggests that abnormalities in phospholipid metabolism are influenced by LPCAT1 genetic variants. Our findings underscore the potential of multi-omics approaches to our understanding and identification of critical factors in CAD development.</p></section><section id="sec5"><h3 class="pmc_sec_title">Trial registration number and date of registration</h3>
<p id="Par5">ClinicalTrials.gov Identifier: <a href="https://clinicaltrials.gov/ct2/show/NCT04839796" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT04839796</a>; Aug 2013.</p></section><section id="sec6"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s40364-025-00821-y.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Coronary artery disease (CAD), LPCAT1, Single-nucleotide polymorphism (SNP), Metabolomics, Machine learning, Biomarker</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par6">Coronary artery disease (CAD) remains the primary global cause of mortality and thus there is a need for a deeper understanding of its origins. This should lead to the development of more effective treatments. Genome-wide association studies (GWAS) focusing on the CAD phenotypes over the past decade have identified over 300 chromosomal loci significantly associated with the disease [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>â€“<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>]. Remarkably, more than 90% of these common risk variants are situated outside of protein-coding regions [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>], and these variants are only associated with about 25% of the diseaseâ€™s heritability [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Additionally, trans-omics analysis, which amalgamates metabolic profiles with genomics, transcriptomics, proteomics, and clinical data, can offer more comprehensive datasets. Leveraging these approaches allows for the exploration of the underlying biological networks that influence susceptibility to a given disease [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>]. By combining genetic association studies with the impact of various metabolites on biological pathways, the trans-omics analysis approach provides a more holistic perspective [<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. Notably, the integration of multiple layers of data in multi-omics studies has been shown to result in more dependable and more robust outcomes [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>].</p>
<p id="Par7">Metabolites encapsulate the outcomes of pathological processes and reflect the influences of genomic, epigenomic, and environmental factors [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>, <a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. Within biomedical research, numerous plasma metabolites have emerged as pivotal predictors and risk factors for complex diseases, notably Alzheimerâ€™s disease [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>], diabetes [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>], and CAD [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>, <a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>]. In the realm of CAD research specifically, this identification of metabolites across various pathways has unveiled a number of potential novel clinical biomarkers [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>].</p>
<p id="Par8">Two distinct approaches have been employed to merge multiple omics datasets in order to unveil CAD risk factors. The first method, a multi-dimensional approach, such as network-based integration, uses high-dimensional data to impartially discern pathways causally linked to CAD from a comprehensive multi-omics viewpoint. Alternatively, the second approach targets specific genetic pathways associated with downstream omics data that are directly implicated in CAD development [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>, <a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. In this context, our hypothesis proposes that integrating metabolomics with genomics should be able to identify novel mechanisms or potential risk factors associated with CAD pathogenesis. To explore this, we conducted a genome-wide SNP analysis coupled with a targeted metabolomic assessment using three cohorts, namely a CAD cohort comprising individuals from cardiac catheterization (the CAD group), a CAD high-risk group, and a healthy control group. Our findings revealed that genetic loci associated with lipidome levels have an augmented CAD risk. Thus, by employing this multi-platform omics approach, we were able to identify a lipid metabolism pathway that is intricately linked with CAD progression.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Methods</h2>
<p id="Par9">The authors declare that all supporting data, methods, and materials are available within the main body of the manuscript and in the online supplementary data.</p>
<section id="Sec3"><h3 class="pmc_sec_title">Study subjects</h3>
<p id="Par10">A total of 781 subjects from the initially recruited 6,176 participants in the Northeastern Taiwan Community Medicine Research Cohort (ClinicalTrials.gov Identifier: <a href="https://clinicaltrials.gov/ct2/show/NCT04839796" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NCT04839796</a>, from July 2019 to November 2020; Fig.Â <a href="#Fig1" class="usa-link">1</a>) were included in this study. All participants underwent a comprehensive clinical examination, blood tests, and a detailed personal and medical history assessment at the Department of Cardiology, Chang Gung Memorial Hospital, Keelung. By incorporating both a high-risk group and a CAD group, the potential confounding factors inherent to the study were mitigated. The disparities observed between the high-risk and CAD groups compared to the control group were thus able to attenuate the effects of any underlying coexisting conditions, allowing the primary focus to be on the CAD. The CAD group comprised individuals demonstrating angiographic evidence showing significant (&gt;â€‰75%) coronary artery stenosis post-administration of intracoronary nitroglycerine (50â€“200Â Âµg), individuals with a history of percutaneous coronary revascularization or individuals who had undergone coronary artery bypass grafting. The high-risk group consisted of individuals whose calculated 10-year and lifetime risk of atherosclerotic cardiovascular disease (ASCVD) equaled or exceeded 20% based on established criteria [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>, <a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>], but without any history of angina pectoris or myocardial infarction. All participants provided informed and signed consent for their participation in this study. The criteria for inclusion in the control group encompassed a number of factors, namely the absence of major systemic diseases, various specific fasting measurements including high-density lipoprotein (HDL) levels (&gt;â€‰40Â mg/dL for males, &gt;â€‰50Â mg/dL for females), fasting glucose levels (&lt;â€‰100Â mg/dL), fasting triglyceride (TG) levels (&lt;â€‰150Â mg/dL), systolic blood pressure (SBP) below 130 mmHg, diastolic blood pressure (DBP) below 85 mmHg, and a waist circumference below 80Â cm for females or 90Â cm for males. The study protocol adheres to the ethical guidelines outlined in the 1975 Declaration of Helsinki and received approval from the Institutional Review Board of Chang Gung Medical Foundation. (IRB No: 201800686B0, 202000077B0A3, and 202101810B0D001). Informed consent was obtained from all participants in the study.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366056_40364_2025_821_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f7b9/12366056/f9f6959e1dd0/40364_2025_821_Fig1_HTML.jpg" loading="lazy" id="d33e525" height="487" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Flowchart of the study protocol. Three groups of subjects were included in this study. Cluster analyses of the genomic and metabolomic data separated the study subjects into five genetic and five metabolomic clusters. Trans-omics comparison identified two distinct patterns of subjects. CAD, coronary artery disease; HDL, high-density lipoprotein; TG, triglyceride; SBP, systolic blood pressure; DBP, diastolic blood pressure; ASCVD, atherosclerotic cardiovascular disease; HF, heart failure; SNP, single nucleotide polymorphism</p></figcaption></figure></section><section id="Sec4"><h3 class="pmc_sec_title">Clinical assessment</h3>
<p id="Par12">Upon recruitment, all participants underwent an extensive assessment involving a comprehensive personal history review, a thorough physical examination, the completion of a detailed questionnaire, and a range of biochemical tests, as previously described [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Blood pressure was assessed by taking the average of two seated measurements, while heart rate was determined via a resting 12-lead electrocardiogram. Body mass index (BMI) was calculated as weight divided by height squared (kg/mÂ²). Regarding the blood pressure measurements, hypertension was deemed present if blood pressure exceeded 140/90 mmHg or if the participant was receiving antihypertensive medications for this condition. The aim of this comprehensive evaluation was to ensure a meticulous examination of the cardiovascular health and associated risk factors within the study population [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Diabetes mellitus was characterized by a fasting glucose level of â‰¥â€‰126Â mg/dL, a glycohemoglobin level of â‰¥â€‰6.5%, a random glucose level of â‰¥â€‰200Â mg/dL, or the use of hypoglycemic medications. Chronic kidney disease (CKD) was defined according to the National Kidney Foundation: Kidney Disease Outcomes Quality Initiative classification for CKD. This determination involved the presence of persistent proteinuria or a reduced estimated glomerular filtration rate (&lt;â€‰60 mL/min/1.73Â mÂ²), which was calculated using the abbreviated Modification of Diet in Renal Disease equation [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>]. Proteinuria was identified if the urine protein to creatinine ratio equaled or exceeded 150Â mg/g. A history of CAD was determined based on various indicators, namely the presence of angina pectoris during a positive exercise test, a history of myocardial infarction, angiographic evidence indicating significant (&gt;â€‰75%) coronary artery stenosis after intracoronary nitroglycerine administration (50â€“200Â Âµg), or a history of undergoing percutaneous coronary revascularization or coronary artery bypass grafting. The current smoking status was classified as individuals who had smoked more than 100 cigarettes during their lifetime and had smoked within one month before their enrollment in the study.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Targeted metabolite analysis using the p180 kit</h3>
<p id="Par13">Blood samples were collected using EDTA-treated tubes when the participants were recruited. Following centrifugation for a minimum of 15 minutes, the plasma component was isolated, frozen, and then shipped on dry ice to our hospitalâ€™s core laboratory center. Subsequently, these samples were stored at -80Â Â°C to maintain stability before undergoing analysis using the AbsoluteIDQ p180 kit (BIOCRATES Life Sciences AG, Austria). For analysis, a 10 ÂµL plasma sample was prepared according to the manufacturerâ€™s instructions. The metabolites of amino acids and biogenic amines were quantified using liquid chromatography coupled with tandem mass spectrometry, while the lipid profile was assessed using flow injection analysis coupled with tandem mass spectrometry. The latter analysis was conducted in the positive electrospray ionization mode and employed multiple reaction monitoring on a TQS mass spectrometer (Waters Corp., Milford, USA). The MetIQ software package (Biocrates Life Science AG, Innsbruck, Austria) was used to facilitate the automatic calculation of metabolite concentrations, and the results were expressed in micromolar (ÂµM) units [<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. The MetIQ software confirmed that the measured values of internal standards (D<sub>8</sub>-Val, D<sub>7</sub>-Pro, D<sub>7</sub>-ADMA, D<sub>6</sub>-Orn, D<sub>5</sub>-Phe, D<sub>5</sub>-Gln, D<sub>4</sub>-Tyr, D<sub>4</sub>-Serotonin, D<sub>4</sub>-Putresine, D<sub>4</sub>-Ala, D<sub>3</sub>-Ser, D<sub>3</sub>-Sarcosine, D<sub>3</sub>-Met, D<sub>3</sub>-Glu, D<sub>3</sub>-Asp, Creatinine, <sup>15</sup>N<sub>2</sub>-Trp, <sup>15</sup>N<sub>2</sub>-Asn, <sup>15</sup>N<sub>2</sub>-Arg, <sup>13</sup>C<sub>6</sub>-Ile, <sup>13</sup>C<sub>6</sub>-His, <sup>13</sup>C<sub>4</sub>-<sup>15</sup>N-Thr, <sup>13</sup>C<sub>2</sub>-Taurine, <sup>13</sup>C<sub>2</sub>-<sup>15</sup>N-Gly, and <sup>13</sup>C-D<sub>4</sub>-Cit) and quality control samples (low QC, medium QC, and high QC) adhere to the predefined tolerance ranges. Altogether 143 targeted metabolites (21 amino acids, 9 biogenic amines, 15 acylcarnitines, 84 phospholipids, and 14 sphingolipids; Table <a href="#MOESM1" class="usa-link">S1</a>) were measured using the unit variance method, which scales each variable by dividing by its standard deviation [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. The data were processed by orthogonal partial least squares discriminant analysis (OPLS-DA) using the SIMCA-P 13.0 software (Umetrics, UmeÃ¥, Sweden). The 95% critical limit of the Hotelling <em>T</em><sup>2</sup> was defined by the ellipse in the score plot. The OPLS-DA model was constructed based on the internal 7-fold cross-validation.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Whole-genome SNP analysis</h3>
<p id="Par14">The genotypes of the subjects were analyzed using the Axiomâ„¢ Genome-Wide TWB 2.0 array using genomic DNA isolated from white blood cells. Following this analysis, the SNPs were filtered sequentially based on a minor allele frequencyâ€‰&lt;â€‰0.01, deviation from Hardy-Weinberg equilibrium in the controls (<em>p</em>â€‰&lt;â€‰1â€‰Ã—â€‰10â»â¶), and a missing rateâ€‰&gt;â€‰10%. For the remaining missing SNPs, imputation was performed by assigning the median allele observed across the dataset. This filtering process resulted in the removal of SNPs from an initial pool of 686,478 to leave a total of 370,394 SNPs that were eligible for subsequent analysis (Figure <a href="#MOESM1" class="usa-link">S1</a>). Sample-level quality control confirmed that all individuals met Thermo Fisher Scientificâ€™s genotyping standards (average call rateâ€‰â‰¥â€‰98.5%), showed no heterozygosity outliers, and were unrelated (PI_HAT meanâ€‰=â€‰0.175, PLINK).</p></section><section id="Sec7"><h3 class="pmc_sec_title">Data normalization and K-means clustering of the various metabolites and SNPs</h3>
<p id="Par15">To ensure comparability across metabolites, missing values were imputed using the median, and zeros were replaced with 0.001 to facilitate log transformation. Each metaboliteâ€™s concentration across the 781 subjects was then normalized by dividing by its median and converted to a log scale. To standardize the varying concentrations of different metabolites, each metaboliteâ€™s concentration across the 781 subjects was normalized by dividing it with the respective metaboliteâ€™s median value and then converting the resulting values to log<sub>2</sub>. For the SNP data, genotypes were directly translated into numerical values, where the homozygous reference corresponded to 0, the heterozygous state corresponded to 0.5, and the homozygous alternate corresponded to 1. For the clustering analysis, K-means clustering was executed utilizing the open-source software Multi-experiment Viewer (MeV 4.9.0, <a href="https://sourceforge.net/projects/mev-tm4/files/mev-tm4/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://sourceforge.net/projects/mev-tm4/files/mev-tm4/</a>) [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>]. The Elbow Method was used to determine the optimal number of clusters, with kâ€‰=â€‰5 being selected as the most appropriate value for both metabolomic and SNP data (Figure <a href="#MOESM1" class="usa-link">S2</a>A). This analysis categorized the metabolites (Figure <a href="#MOESM1" class="usa-link">S2</a>B) and the SNPs (Figure <a href="#MOESM1" class="usa-link">S3</a>) into five groups based on their concentrations or genotypes and employed Euclidean distance metrics. Upon completion of the k-means clustering analysis, a heatmap was generated using MeV 4.9.0 to visually represent the clustered data.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Machine learning (ML)-assisted prediction of CAD</h3>
<p id="Par16">When conducting the machine learning analysis, we utilized R Version 3.6.3 together with various packages, including data.Table (1.14.2), dplyr (1.1.4), stringr (1.5.0), doParallel (1.0.15), MatchIt (4.0.1), randomForest (4.6.14), e1071 (1.7.3), glmnet (4.1.4), rpart (4.1.15), xgboost (0.90.0.2), caret (6.0.85), pROC (1.16.1), and cvAUC (1.1.0). The dataset was randomly divided into training and validation sets using an 80:20 split ratio. Four ML algorithms, namely Random Forest (RF), Support Vector Machine (SVM), Least Absolute Shrinkage and Selection Operator (LASSO), and Extreme Gradient Boosting (XGBoost), were employed for the model training, feature selection, and validation. Among these algorithms, the RF model was identified as able to extract the most pertinent features while achieving the highest performance in terms of Area Under Curve (AUC) and accuracy rate. We also performed hyperparameter optimization for XGBoost, testing eta values at [0.1, 0.3, 0.5, 0.7, 0.9], max_depth at [6, 12, 29], and min_child_weight at [1, 2, 4, 8, 10], which resulted in a total of 75 parameter combinations (Table <a href="#MOESM1" class="usa-link">S2</a>). However, the results of the RF and XGBoost models showed no significant difference (Table <a href="#MOESM1" class="usa-link">S3</a>). Additionally, to assess the robustness of the different sampling schemes using the same selected features, we first performed feature selection and model building with 100 bootstrapped replications. For each of the 100 bootstrap iterations, the data were randomly split into training and validation sets. Three ML methods, namely LASSO, RF, and SVM, were independently applied for feature selection. In each iteration, LASSO-selected features were defined as those with non-zero coefficients; RF and SVM selected the top k features based on feature importance and weighted support vectors, respectively; where k was determined by the number of features selected by LASSO in that iteration. The selected features from all three methods were recorded, and at the end of 100 iterations, we summarized the total selection counts for each feature. Final feature ranking was determined based on the cumulative frequency of selection across all iterations and methods. The 16 top-ranked features were then used in a 10-fold cross-validation to evaluate the consistency of prediction performance, with both sampling schemes yielding similar results (Table <a href="#MOESM1" class="usa-link">S4</a>). Additionally, in order to verify the stability of feature selection across the bootstrapped and cross-validation methods, we conducted an independent 10-fold cross-validation using the LASSO, SVM, and RF methods for feature selection. As shown in Table <a href="#MOESM1" class="usa-link">S5</a>, 60â€“75% of the selected features were consistent with the bootstrapped method in each fold, and the average prediction performance was comparable to that of the bootstrapped method. In this study, we applied the RF model for the determination of model construction. The RF model was trained using an ensemble of decision trees, leveraging bootstrap sampling and random feature selection at each split to enhance robustness and reduce overfitting. Compared to other ML models, namely SVM, Decision Tree, and XGBoost, the RF model demonstrated superior performance, as shown in Figure <a href="#MOESM1" class="usa-link">S4</a>.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Statistics</h3>
<p id="Par17">The statistical analyses were conducted using SPSS software for Windows, version 25.0 (IBM Corporation, Armonk, NY). To assess the normality of the various variable distributions, the Shapiro-Wilk test was employed. For comparisons between groups based on continuous variables, either a one-way analysis of variance (ANOVA) or a Kruskal-Wallis test was utilized, depending on the nature of the data distribution. Results for continuous variables are presented as meansâ€‰Â±â€‰standard deviations (SD) or median with an interquartile range (IQR). For the categorical variables, the distribution across groups were examined using the Freeman-Halton test, with the results expressed as frequencies and percentages. This methodology allowed for comprehensive comparisons across the various variable types in order to better understand the differences between the three groups. For the statistical analysis of SNPs, we used both Ï‡Â² tests and the SAIGE [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>] method. The Ï‡Â² test offers a simple and computationally efficient approach for initial exploration. To account for population structure and in order to handle small <em>p</em>-values in GWAS, we employed the SAIGE method, which improves accuracy by addressing potential imbalances when there is case-control data.</p></section></section><section id="Sec10"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Clinical characteristics of the studyâ€™s subjects</h3>
<p id="Par18">The study encompassed 781 subjects, which were divided into a CAD group (<em>n</em>â€‰=â€‰147), a high-risk group (<em>n</em>â€‰=â€‰363), and a control group (<em>n</em>â€‰=â€‰271) (TableÂ <a href="#Tab1" class="usa-link">1</a>). The CAD group, with a median SYNTAX score of 27.0 (IQR: 22.0â€“43.0), a quantitative angiographic measure of lesion complexity and disease severity [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>], had a higher proportion of male participants. The CAD group and the high-risk group both exhibited a number of elevated clinical markers (age, body weight, BMI, waist circumference, SBP, DBP) compared to the control group. Comorbidities, such as diabetes and CKD, were more prevalent in the CAD group and the high-risk group. Lipid profiles (total cholesterol, HDL, low-density lipoprotein (LDL) of the CAD group was significantly lower than the other two groups, possibly due to increased statin use. The CAD group and the high-risk group showed elevated levels of TG, creatinine, adiponectin, and leptin compared to the control group. Notably, the CAD group had a higher WBC and high-sensitive C-reactive protein values than the other two cohorts.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Basic characteristics of subjects in different groups</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Group</th>
<th align="left" colspan="1" rowspan="1">Total</th>
<th align="left" colspan="1" rowspan="1">Control</th>
<th align="left" colspan="1" rowspan="1">High-risk</th>
<th align="left" colspan="1" rowspan="1">CAD</th>
<th align="left" colspan="1" rowspan="1"></th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">n</td>
<td align="left" colspan="1" rowspan="1">781</td>
<td align="left" colspan="1" rowspan="1">271</td>
<td align="left" colspan="1" rowspan="1">363</td>
<td align="left" colspan="1" rowspan="1">147</td>
<td align="left" colspan="1" rowspan="1">
<em>p</em>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Sex (male, %)</td>
<td align="left" colspan="1" rowspan="1">356 (45.6)</td>
<td align="left" colspan="1" rowspan="1">96 (35.4)</td>
<td align="left" colspan="1" rowspan="1">151 (41.6)</td>
<td align="left" colspan="1" rowspan="1">109 (74.1)</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Age (years, mean)</td>
<td align="left" colspan="1" rowspan="1">62.1â€‰Â±â€‰12.7</td>
<td align="left" colspan="1" rowspan="1">56.3â€‰Â±â€‰14.4</td>
<td align="left" colspan="1" rowspan="1">65.7â€‰Â±â€‰10.6</td>
<td align="left" colspan="1" rowspan="1">63.8â€‰Â±â€‰10.1</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Height (cm)</td>
<td align="left" colspan="1" rowspan="1">159.4â€‰Â±â€‰9.0</td>
<td align="left" colspan="1" rowspan="1">158.3â€‰Â±â€‰8.6</td>
<td align="left" colspan="1" rowspan="1">158.1â€‰Â±â€‰8.8</td>
<td align="left" colspan="1" rowspan="1">164.3â€‰Â±â€‰8.5</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Weight (Kg)</td>
<td align="left" colspan="1" rowspan="1">66.5â€‰Â±â€‰13.7</td>
<td align="left" colspan="1" rowspan="1">59.2â€‰Â±â€‰12.0</td>
<td align="left" colspan="1" rowspan="1">68.7â€‰Â±â€‰12.2</td>
<td align="left" colspan="1" rowspan="1">74.6â€‰Â±â€‰13.6</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">BMI</td>
<td align="left" colspan="1" rowspan="1">26.09â€‰Â±â€‰4.3</td>
<td align="left" colspan="1" rowspan="1">23.55â€‰Â±â€‰4.06</td>
<td align="left" colspan="1" rowspan="1">27.37â€‰Â±â€‰3.65</td>
<td align="left" colspan="1" rowspan="1">27.62â€‰Â±â€‰4.13</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Waist (cm)</td>
<td align="left" colspan="1" rowspan="1">85.8â€‰Â±â€‰11.8</td>
<td align="left" colspan="1" rowspan="1">77.2â€‰Â±â€‰9.7</td>
<td align="left" colspan="1" rowspan="1">89.7â€‰Â±â€‰9.4</td>
<td align="left" colspan="1" rowspan="1">94.9â€‰Â±â€‰9.8</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Systolic BP (mmHg)</td>
<td align="left" colspan="1" rowspan="1">132.6â€‰Â±â€‰45.7</td>
<td align="left" colspan="1" rowspan="1">117.6â€‰Â±â€‰13.6</td>
<td align="left" colspan="1" rowspan="1">142.3â€‰Â±â€‰62.5</td>
<td align="left" colspan="1" rowspan="1">136.3â€‰Â±â€‰21.2</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Diastolic BP (mmHg)</td>
<td align="left" colspan="1" rowspan="1">76.5â€‰Â±â€‰11</td>
<td align="left" colspan="1" rowspan="1">70.8â€‰Â±â€‰9.1</td>
<td align="left" colspan="1" rowspan="1">80.2â€‰Â±â€‰10.8</td>
<td align="left" colspan="1" rowspan="1">78.1â€‰Â±â€‰10.7</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Heart rate (BMP)</td>
<td align="left" colspan="1" rowspan="1">75.4â€‰Â±â€‰12.7</td>
<td align="left" colspan="1" rowspan="1">75.7â€‰Â±â€‰11.2</td>
<td align="left" colspan="1" rowspan="1">76.2â€‰Â±â€‰13</td>
<td align="left" colspan="1" rowspan="1">72.9â€‰Â±â€‰14.5</td>
<td align="left" colspan="1" rowspan="1">0.028</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">DM (%)</td>
<td align="left" colspan="1" rowspan="1">249 (31.9)</td>
<td align="left" colspan="1" rowspan="1">2 (0.7)</td>
<td align="left" colspan="1" rowspan="1">156 (43.0)</td>
<td align="left" colspan="1" rowspan="1">91 (62.3)</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CKD (%)</td>
<td align="left" colspan="1" rowspan="1">190 (24.4)</td>
<td align="left" colspan="1" rowspan="1">13 (4.8)</td>
<td align="left" colspan="1" rowspan="1">120 (33.1)</td>
<td align="left" colspan="1" rowspan="1">57 (39.0)</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Fasting glucose (mg/dL)</td>
<td align="left" colspan="1" rowspan="1">111.0â€‰Â±â€‰41.1</td>
<td align="left" colspan="1" rowspan="1">91.5â€‰Â±â€‰6.6</td>
<td align="left" colspan="1" rowspan="1">119.8â€‰Â±â€‰41.9</td>
<td align="left" colspan="1" rowspan="1">127.2â€‰Â±â€‰60.7</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Glycohemoglobin (%)</td>
<td align="left" colspan="1" rowspan="1">6.33â€‰Â±â€‰0.78</td>
<td align="left" colspan="1" rowspan="1">5.6â€‰Â±â€‰0.31</td>
<td align="left" colspan="1" rowspan="1">6.5â€‰Â±â€‰1.27</td>
<td align="left" colspan="1" rowspan="1">6.20â€‰Â±â€‰1.11</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Total cholesterol (mg/dL)</td>
<td align="left" colspan="1" rowspan="1">185.8â€‰Â±â€‰38.7</td>
<td align="left" colspan="1" rowspan="1">199.5â€‰Â±â€‰31.3</td>
<td align="left" colspan="1" rowspan="1">186.3â€‰Â±â€‰38.5</td>
<td align="left" colspan="1" rowspan="1">157.5â€‰Â±â€‰37.2</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HDL (mg/dL)</td>
<td align="left" colspan="1" rowspan="1">54.6â€‰Â±â€‰16.2</td>
<td align="left" colspan="1" rowspan="1">65.7â€‰Â±â€‰14.5</td>
<td align="left" colspan="1" rowspan="1">49.6â€‰Â±â€‰13.1</td>
<td align="left" colspan="1" rowspan="1">45.7â€‰Â±â€‰14.7</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">LDL (mg/dL)</td>
<td align="left" colspan="1" rowspan="1">119.6â€‰Â±â€‰36</td>
<td align="left" colspan="1" rowspan="1">128.7â€‰Â±â€‰31.1</td>
<td align="left" colspan="1" rowspan="1">121.4â€‰Â±â€‰37.9</td>
<td align="left" colspan="1" rowspan="1">97â€‰Â±â€‰30.1</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Triglyceride (mg/dL)</td>
<td align="left" colspan="1" rowspan="1">134.2â€‰Â±â€‰87.3</td>
<td align="left" colspan="1" rowspan="1">84.8â€‰Â±â€‰32.4</td>
<td align="left" colspan="1" rowspan="1">165.2â€‰Â±â€‰100</td>
<td align="left" colspan="1" rowspan="1">150.4â€‰Â±â€‰83.1</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">BUN (mg/dL)</td>
<td align="left" colspan="1" rowspan="1">16.6â€‰Â±â€‰8.5</td>
<td align="left" colspan="1" rowspan="1">14.2â€‰Â±â€‰3.9</td>
<td align="left" colspan="1" rowspan="1">17.4â€‰Â±â€‰7.6</td>
<td align="left" colspan="1" rowspan="1">19.6â€‰Â±â€‰14.2</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Creatinine (mg/dL)</td>
<td align="left" colspan="1" rowspan="1">0.92â€‰Â±â€‰0.63</td>
<td align="left" colspan="1" rowspan="1">0.74â€‰Â±â€‰0.17</td>
<td align="left" colspan="1" rowspan="1">0.94â€‰Â±â€‰0.39</td>
<td align="left" colspan="1" rowspan="1">1.23â€‰Â±â€‰1.26</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Albumin (g/dL)</td>
<td align="left" colspan="1" rowspan="1">4.86â€‰Â±â€‰8.78</td>
<td align="left" colspan="1" rowspan="1">4.64â€‰Â±â€‰0.24</td>
<td align="left" colspan="1" rowspan="1">4.57â€‰Â±â€‰0.26</td>
<td align="left" colspan="1" rowspan="1">4.53â€‰Â±â€‰0.32</td>
<td align="left" colspan="1" rowspan="1">0.198</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ALT (U/L)</td>
<td align="left" colspan="1" rowspan="1">27.55â€‰Â±â€‰18.66</td>
<td align="left" colspan="1" rowspan="1">22.82â€‰Â±â€‰11.81</td>
<td align="left" colspan="1" rowspan="1">29.96â€‰Â±â€‰19.88</td>
<td align="left" colspan="1" rowspan="1">30.44â€‰Â±â€‰23.84</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Total bilirubin (mg/dL)</td>
<td align="left" colspan="1" rowspan="1">0.63â€‰Â±â€‰0.33</td>
<td align="left" colspan="1" rowspan="1">0.67â€‰Â±â€‰0.29</td>
<td align="left" colspan="1" rowspan="1">0.6â€‰Â±â€‰0.27</td>
<td align="left" colspan="1" rowspan="1">0.64â€‰Â±â€‰0.52</td>
<td align="left" colspan="1" rowspan="1">0.046</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Alkaline phosphatase (U/L)</td>
<td align="left" colspan="1" rowspan="1">71.29â€‰Â±â€‰21.12</td>
<td align="left" colspan="1" rowspan="1">67.18â€‰Â±â€‰19.29</td>
<td align="left" colspan="1" rowspan="1">74.95â€‰Â±â€‰20.32</td>
<td align="left" colspan="1" rowspan="1">69.65â€‰Â±â€‰25.14</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">RBC (10<sup>6</sup>/ÂµL)</td>
<td align="left" colspan="1" rowspan="1">4.7â€‰Â±â€‰0.6</td>
<td align="left" colspan="1" rowspan="1">4.7â€‰Â±â€‰0.55</td>
<td align="left" colspan="1" rowspan="1">4.78â€‰Â±â€‰0.59</td>
<td align="left" colspan="1" rowspan="1">4.52â€‰Â±â€‰0.68</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Hemoglobin (g/dL)</td>
<td align="left" colspan="1" rowspan="1">13.88â€‰Â±â€‰1.55</td>
<td align="left" colspan="1" rowspan="1">13.69â€‰Â±â€‰1.32</td>
<td align="left" colspan="1" rowspan="1">14.14â€‰Â±â€‰1.52</td>
<td align="left" colspan="1" rowspan="1">13.56â€‰Â±â€‰1.93</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">WBC (10<sup>3</sup>/ÂµL)</td>
<td align="left" colspan="1" rowspan="1">6.4â€‰Â±â€‰2.12</td>
<td align="left" colspan="1" rowspan="1">5.68â€‰Â±â€‰2.03</td>
<td align="left" colspan="1" rowspan="1">6.65â€‰Â±â€‰2.01</td>
<td align="left" colspan="1" rowspan="1">7.14â€‰Â±â€‰2.18</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.0001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Platelets (10<sup>3</sup>/ÂµL)</td>
<td align="left" colspan="1" rowspan="1">260.7â€‰Â±â€‰65.7</td>
<td align="left" colspan="1" rowspan="1">260.7â€‰Â±â€‰59.5</td>
<td align="left" colspan="1" rowspan="1">266.8â€‰Â±â€‰68.6</td>
<td align="left" colspan="1" rowspan="1">244.9â€‰Â±â€‰67.3</td>
<td align="left" colspan="1" rowspan="1">0.004</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Adiponectin (Âµg/mL)</td>
<td align="left" colspan="1" rowspan="1">8â€‰Â±â€‰7.4</td>
<td align="left" colspan="1" rowspan="1">6.9â€‰Â±â€‰5.1</td>
<td align="left" colspan="1" rowspan="1">7.9â€‰Â±â€‰7.1</td>
<td align="left" colspan="1" rowspan="1">9.2â€‰Â±â€‰9.5</td>
<td align="left" colspan="1" rowspan="1">0.105</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Leptin (ng/mL)</td>
<td align="left" colspan="1" rowspan="1">10â€‰Â±â€‰9.5</td>
<td align="left" colspan="1" rowspan="1">7.9â€‰Â±â€‰8.4</td>
<td align="left" colspan="1" rowspan="1">11.5â€‰Â±â€‰10.1</td>
<td align="left" colspan="1" rowspan="1">9.2â€‰Â±â€‰8.8</td>
<td align="left" colspan="1" rowspan="1">0.002</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">HS-CRP (mg/L)</td>
<td align="left" colspan="1" rowspan="1">3.4â€‰Â±â€‰13.6</td>
<td align="left" colspan="1" rowspan="1">2.1â€‰Â±â€‰5.4</td>
<td align="left" colspan="1" rowspan="1">2.8â€‰Â±â€‰5.8</td>
<td align="left" colspan="1" rowspan="1">7.5â€‰Â±â€‰29.2</td>
<td align="left" colspan="1" rowspan="1">0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Medications</strong>
</td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
<td align="left" colspan="1" rowspan="1"></td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Statin (%)</td>
<td align="left" colspan="1" rowspan="1">209 (26.8)</td>
<td align="left" colspan="1" rowspan="1">18 (6.6)</td>
<td align="left" colspan="1" rowspan="1">86 (23.7)</td>
<td align="left" colspan="1" rowspan="1">105 (71.4)</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Fibrate (%)</td>
<td align="left" colspan="1" rowspan="1">14 (1.8)</td>
<td align="left" colspan="1" rowspan="1">0 (0)</td>
<td align="left" colspan="1" rowspan="1">10 (2.8)</td>
<td align="left" colspan="1" rowspan="1">4 (2.7)</td>
<td align="left" colspan="1" rowspan="1">0.023</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">CCB (%)</td>
<td align="left" colspan="1" rowspan="1">103 (13.2)</td>
<td align="left" colspan="1" rowspan="1">1 (0.4)</td>
<td align="left" colspan="1" rowspan="1">71 (19.6)</td>
<td align="left" colspan="1" rowspan="1">31 (21.1)</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Î²-blocker (%)</td>
<td align="left" colspan="1" rowspan="1">113 (14.5)</td>
<td align="left" colspan="1" rowspan="1">20 (7.4)</td>
<td align="left" colspan="1" rowspan="1">46 (12.7)</td>
<td align="left" colspan="1" rowspan="1">47 (32.0)</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">ARB (%)</td>
<td align="left" colspan="1" rowspan="1">138 (17.7)</td>
<td align="left" colspan="1" rowspan="1">4 (1.5)</td>
<td align="left" colspan="1" rowspan="1">91 (25.1)</td>
<td align="left" colspan="1" rowspan="1">43 (29.3)</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.001</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Diuretics (%)</td>
<td align="left" colspan="1" rowspan="1">43 (5.5)</td>
<td align="left" colspan="1" rowspan="1">5 (1.8)</td>
<td align="left" colspan="1" rowspan="1">22(6.1)</td>
<td align="left" colspan="1" rowspan="1">16 (10.9)</td>
<td align="left" colspan="1" rowspan="1">&lt;â€‰0.001</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p23"><p>CAD, coronary artery disease; BP, Blood pressure; BMI, body mass index; DM, diabetes mellitus; CKD, chronic kidney disease; HDL, high-density lipoprotein; LDL, low-density lipoprotein; BUN, blood urea nitrogen; ALT, alanine aminotransferase; RBC, red blood cell; WBC, white blood cell; HS-CRP, high-sensitive C-Reactive protein; CCB, calcium channel blocker; ARB, angiotensin receptor blocker. Data are presented as the meanâ€‰Â±â€‰SD and analyzed by one-way ANOVA with Dunnettâ€™s correction</p></div></div></section></section><section id="Sec12"><h3 class="pmc_sec_title">Cluster analysis of subjects according to their metabolome data</h3>
<p id="Par20">Using extensive datasets, machine learning algorithms have been able to demonstrate efficacy when identifying patterns by utilizing distance metrics to quantify dissimilarities between distinct data points [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>]. For this investigation, the Euclidean distance metric was deliberately chosen to mitigate distortions within the K-means algorithm [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>]. Using the Elbow Method, which indicated an optimal range of 3â€“5 clusters for the metabolomic data, we selected kâ€‰=â€‰5 as the most suitable value for both datasets (Figure <a href="#MOESM1" class="usa-link">S2</a>A). The results revealed that, by employing the Euclidean distance metric with the designation of kâ€‰=â€‰5 subtypes, it was possible to consistently obtain higher silhouette scores and connectedness metrics, which in turn gave a normalized Jaccard similarity exceeding 0.80 for each cluster identified by the two respective methodologies.</p>
<p id="Par21">All study subjects could be classified into five metabolic clusters, which are denoted as cluster 1 through cluster 5 (Fig.Â <a href="#Fig2" class="usa-link">2</a>A and Figure <a href="#MOESM1" class="usa-link">S2</a>B). Based on key characteristics, such as BMI, age, and gender distribution (Table <a href="#MOESM1" class="usa-link">S6</a>), cluster 4 emerged as the most at-risk group, with the highest BMI (28.1), a male-dominant gender distribution (male 64.6%), and a high incidence of CAD (40%). In contrast, cluster 2, which represented the healthiest group, had a normal BMI (22.9), an average age of 59.7, and showed female dominance (female 63.3%). A boxplot depicting the distribution of these characteristics across clusters is shown in Figure <a href="#MOESM1" class="usa-link">S5</a>E. Notably, cluster 4 subjects displayed the highest incidence rates of diabetes, of CKD, and various abnormal clinical indices, such as body weight and fasting glucose, among the five clusters. Conversely, cluster 2 presented with the lowest prevalence of these comorbidities and the lowest levels of TG. Interestingly, cluster 2 demonstrated the highest levels of total cholesterol and HDL, whereas cluster 4 showcased the lowest levels among the five clusters.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366056_40364_2025_821_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f7b9/12366056/696aee89522c/40364_2025_821_Fig2_HTML.jpg" loading="lazy" id="d33e1313" height="619" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Characteristics of the subjects in different metabolomic clusters and their genetics. <strong>A</strong>, Clustering analysis of the dataset of 143 metabolites. Red color indicates the abundance of metabolite was up-regulated compared to the control and blue color indicates down-regulation. Each raw in the score plots represents each individual subject. The alignment of subjects, from top to bottom, was arranged from control group to high-risk group to CAD group. In each group, the female subjects were arranged on top of male subjects. In each gender, the younger subjects were arranged on top of the older ones. The colors in the heatmap reflect the plasma metabolite abundance (mean centered and divided by the range of each variable). <strong>B</strong>, Comparison of the 143 targeted metabolites using an orthogonal partial least squares-discriminant analysis (OPLS-DA) plot representing the separation of the plasma metabolites from cluster 2 and 4. The 143 variables and 780 observations were selected to build this OPLS-DA score plot by cross-validation rules. The sum of squares captured by this model (R2) was 0.777 and the cross-validated (Q2) was 0.705. <strong>C</strong>, Manhattan Plot of whole genome SNP analysis for the different groups, namely control, high-risk and CAD. Plot of negative log<sub>10</sub> of the <em>p</em>-values for the single SNP association analysis of 271 control, 363 high-risk, and 147 CAD subjects; these are ordered along the x-axis for each chromosome by chromosomal position. <strong>D</strong>, The significant 195 SNPs (<em>p</em>-valuesâ€‰&lt;â€‰0.05) are divided into homozygous reference (colored in blue), homozygous SNP (colored in red) and heterozygous SNPs (colored in white) according to their diploid genotypes. The SNPs are grouped into five clusters on the basis of Euclidean distance by K-means clustering. Each raw value in the score plots represent each individual subject. The alignment of subjects, from top to bottom, is arranged from control to high-risk to CAD group. In each group, the female subjects were arranged on top of male subjects. for each gender, the younger subjects were placed on top of older ones</p></figcaption></figure><p id="Par23">Table <a href="#MOESM1" class="usa-link">S7</a> outlines the 10 top-ranked metabolites that distinguish the CAD, high-risk, and control groups obtained via ML prediction. Nine of these metabolites are glycerophospholipids, while one is a sphingomyelin. The score plot, generated from an Orthogonal Partial Least Squares-Discriminant Analysis (OPLS-DA, Fig.Â <a href="#Fig2" class="usa-link">2</a>B), vividly displays the distinct patterns found across the 143 plasma metabolites derived from clusters 2 and 4. These clusters exhibit marked differences in their metabolic profiles. Employing K-means cluster analysis, two distinct metabolomic patterns emerged, one of these representing the healthiest group of subjects (metabolic cluster 4), while the other represents the least healthy group of subjects (metabolic cluster 2). This analysis highlights the extreme variations present in the metabolic profiles of the subjects in this study.</p></section><section id="Sec13"><h3 class="pmc_sec_title">Cluster analysis of subjects according to their SNP data</h3>
<p id="Par24">To investigate potential genetic associations within the various distinct patterns of plasma metabolome, we conducted GWAS using the Axiom Genome-Wide TWB 2.0 Array. Only 195 SNPs (Figure <a href="#MOESM1" class="usa-link">S1</a>A) met our stringent criteria, exhibiting adjusted <em>p</em>-values below 1â€‰Ã—â€‰10<sup>âˆ’â€‰7</sup> and an Odds ratioâ€‰&gt;â€‰1 or &lt;â€‰0.5 among the initial pool of 370,394 SNPs. These values were calculated using both Ï‡Â² tests and the SAIGE method, and further adjusted using the Bonferroni correction. Subsequently, these selected SNPs were employed for correlation analysis focusing on the CAD group compared to the control group (Figure <a href="#MOESM1" class="usa-link">S1</a>B and <a href="#MOESM1" class="usa-link">S1</a>C). As shown in a Manhattan plot of the GWAS (Fig.Â <a href="#Fig2" class="usa-link">2</a>C), none of the SNPs reached the genome-wide significance threshold of 2.6â€‰Ã—â€‰10<sup>âˆ’â€‰4</sup> (0.05/195). Table <a href="#MOESM1" class="usa-link">S8</a> provides the genotype frequencies of these SNPs for the control, high-risk, and CAD groups.</p>
<p id="Par25">Our examination concentrated on two distinct chromosomal regions where <em>p</em>-values were lower than 1.0â€‰Ã—â€‰10<sup>âˆ’â€‰2</sup>. Table <a href="#MOESM1" class="usa-link">S9</a> lists the 10 top-ranked SNPs that most effectively differentiate between the CAD group and the control group, as predicted by ML. Key SNPs, such as those in the Lysophosphatidylcholine Acyltransferase 1 (LPCAT1) gene, showed higher statistical significance using SAIGE compared to the Ï‡Â² test. A comparison of <em>p</em>-values from both methods in Table <a href="#MOESM1" class="usa-link">S9</a> highlights the effectiveness of each approach in detecting associations with CAD. Additionally, the genotype frequencies of these SNPs were compared with data obtained from the Taiwan Biobank (<a href="https://taiwanview.twbiobank.org.tw/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://taiwanview.twbiobank.org.tw/</a>) in order to provide additional context and validation. This examination identified five genes, these being LPCAT1, SH3 Domain Containing GRB2 Like 2, Endophilin A1 (SH3GL2), Vesicle Associated Membrane Protein 8 (VAMP8), Tubby-related protein 4 (TULP4), and ATPase Cation Transporting 13A2 (ATP13A2); these genes were then extracted from the list of top-ranked SNPs. In order to pursuit further the genetic factors significantly influencing differences in metabolomic patterns across the control, high-risk and CAD groups, we unbiasedly clustered SNPs into five distinct groups (clusters A-E, Fig.Â <a href="#Fig2" class="usa-link">2</a>D and Figure <a href="#MOESM1" class="usa-link">S3</a>) using K-means clustering. The Elbow Method was used to determine the appropriate number of clusters, and kâ€‰=â€‰5 was selected based on its consistency with both metabolomic and SNP data clustering (Figure <a href="#MOESM1" class="usa-link">S2</a>A). Comparative analysis of SNP patterns across these clusters revealed significant differences, particularly in clinical characteristics such as BMI, age, and gender distribution (Table <a href="#MOESM1" class="usa-link">S10</a>). A boxplot depicting these characteristics across SNP clusters is provided in Figure <a href="#MOESM1" class="usa-link">S5</a>F. For example, cluster B exhibited the highest CAD risk (47.4%), with a BMI of 26.7, and an older average age (63.6 years). On the other hand, cluster E had a low CAD risk (10.4%), with a BMI of 25.2, and an average age of 61.9 years. The comparative analysis of SNP patterns among these clusters revealed statistically significant differences. However, when we evaluated clinical characteristics such as SBP, DBP, and TG among these genetic clusters (Table <a href="#MOESM1" class="usa-link">S10</a>), the observed distinctions were not as pronounced as those evident in the metabolomic clusters. Moreover, the disparity in lipid-lowering and antihypertensive medication prescriptions among the SNP clusters was not as conspicuous as observed within the metabolite clusters. This observation suggests that SNP clustering might mitigate the impact of medication on clinical manifestations. Furthermore, we analyzed the genotype frequencies of the CAD-associated SNPs that have been reported previously and that are able to be identified by the Axiomâ„¢ Genome-Wide TWB 2.0 array (Table <a href="#MOESM1" class="usa-link">S11</a>). However, statistical analysis by the proportion test revealed that none of the risk alleles of previously reported SNPs pass the threshold (<em>p</em>â€‰&lt;â€‰0.05) for statistical significance between the control and CAD groups.</p></section><section id="Sec14"><h3 class="pmc_sec_title">Filtered metabolomic and genetic cluster analyses</h3>
<p id="Par26">Subsequent analysis revealed a prominent association between metabolomic cluster 2 and genetic clusters D and E, while metabolomic cluster 4 was correlated predominantly with genetic clusters B and C (Figure <a href="#MOESM1" class="usa-link">S5</a>). Based on this association, participants within metabolomic cluster 2, which was linked to genetic clusters D/E, were grouped into â€œSubtype-Iâ€, while those in metabolomic cluster 4, which was associated with genetic clusters B/C, were grouped into â€œSubtype-IIâ€ (Fig.Â <a href="#Fig3" class="usa-link">3</a>A). It should be noted that these two subtypes exhibit significant disparities in clinical characteristics and comorbidities (Table <a href="#MOESM1" class="usa-link">S12</a>). Subtype-I has notably higher total cholesterol, HDL, and LDL levels compared to Subtype-II. Moreover, the latter subtype exhibited a higher representation of individuals from the high-risk and CAD groups. Pronounced variations in specific metabolomic markers (particularly glycerophospholipids) and genetic markers (specifically LPCAT1) were observed when the two subtypes were compared (Figs.Â <a href="#Fig2" class="usa-link">2</a>D and <a href="#Fig3" class="usa-link">3</a>A, and Figure <a href="#MOESM1" class="usa-link">S6</a>). To validate these associations, we performed the Chi-square test and compared the observed SNP distributions in each of the metabolomic cluster. SNP clusters D and E were significantly enriched in metabolomic cluster 2 (<em>p</em>â€‰=â€‰2.5â€‰Ã—â€‰10â»Â¹â¹), while SNP clusters B and C were overrepresented in metabolomic cluster 4 (<em>p</em>â€‰=â€‰1.13â€‰Ã—â€‰10â»Â¹Â³). These results confirm that the clustering patterns are statistically significant and supporting the hypothesis that there is a genetic-metabolic association based on our finding that there is a connection between the genomic and metabolomic datasets (Figure <a href="#MOESM1" class="usa-link">S5</a>B). Additionally, analysis of the genotype frequencies of the five LPCAT1 SNPs (rs1032752, rs1032751, rs10475026, rs9799949 and rs36993) across the control, high-risk, and CAD groups revealed a significant difference (<em>p</em>â€‰=â€‰0.0365) by ANOVA. Notably, the proportion test showed a significant difference between the control and CAD groups for these LPCAT1 SNPs (<em>p</em>â€‰=â€‰0.0295), which suggests that there is potential linkage disequilibrium as well as a possible haplotype linkage; this might be associated with a genetic predisposition towards CAD (Figure <a href="#MOESM1" class="usa-link">S7</a>).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366056_40364_2025_821_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f7b9/12366056/94c8a2c8738f/40364_2025_821_Fig3_HTML.jpg" loading="lazy" id="d33e1485" height="997" width="717" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Comparisons of the subjects filtered by metabolomics and SNP cluster analyses. <strong>A</strong>, Bi-allelic SNPs of LPCAT1 are presented as homozygous allele 2 (SNP, red), homozygous allele 1 (WT, blue), and heterozygous SNPs (white). The pattern of glycerophospholipids detected based on targeted metabolomic abundance (mean centered and divided by the range of each variable) in each pair of subtypes of subjects aligned by SNPs are presented. <strong>B</strong>, Two specific haplotypes of LPCAT1 SNPs were detected as following: rs36993 (Aâ€‰&gt;â€‰G), rs9799949 (Câ€‰&gt;â€‰T), rs10475026 (Aâ€‰&gt;â€‰C), rs1032751 (Tâ€‰&gt;â€‰C) and rs1032752 (Gâ€‰&gt;â€‰A). <strong>C</strong>, Orthogonal partial least squares discriminant analysis (OPLSâ€“DA) score plot of plasma metabolites of the subjects in cluster 2 and 4. Subjects with haplotype 2 or haplotype 3 have differential metabolomic patterns. In total, 143 variables and 151 observations were selected to build this OPLS-DA score plot by cross-validation rules. The sum of squares captured by this model (R2) was 0.814 and the cross-validated (Q2) was 0.755. <strong>D</strong>, Pie charts indicating the frequencies of the subjectsâ€™ group, filtered by metabolomic and SNP cluster analyses for the different haplotypes of LPCAT1. The haplotype 3 of LPCAT1 had a significantly increased percentage of subjects in the high-risk and CAD groups. The ratio of PCae/LysoPC <strong>(E)</strong> and the level of PCae <strong>(F)</strong> were significantly reduced in both male and female subjects with haplotype 3. In addition, the total cholesterol level <strong>(G)</strong>, high-density lipoprotein level <strong>(H)</strong> and low-density lipoprotein level <strong>(I)</strong> of the subjects with haplotype 3 were lower than those of haplotype 2. Furthermore, the plasma triglyceride level <strong>(J)</strong> of the subjects with haplotype 3 was significantly higher than the other haplotypes. Quantification data are presented as meanâ€‰Â±â€‰SD and analyzed by one-way ANOVA with Bonferroni multiple comparison test. *<em>p</em>â€‰&lt;â€‰0.05; **<em>p</em>â€‰&lt;â€‰0.005</p></figcaption></figure><p id="Par28">Further investigation revealed two distinct haplotypes of LPCAT1 associated with different metabolomic clusters. Haplotype 2 was defined by a specific combination of homozygous references, namely rs1032752, rs1032751, rs10475026, rs9799949, along with a homozygous SNP, rs36993. Conversely, haplotype 3 was characterized by homozygous SNPs, rs1032752, rs1032751, rs10475026, rs9799949, paired with a homozygous SNP, rs36993. All other combinations were classified as haplotype 1 (wild type, WT; Fig.Â <a href="#Fig3" class="usa-link">3</a>B). Examining the impact of the LPCAT1 haplotypes on plasma metabolomic patterns using OPLS-DA (Fig.Â <a href="#Fig3" class="usa-link">3</a>C) indicated that subjects with haplotype 2 or haplotype 3 have differential metabolomic patterns. These findings suggest a significant influence of specific LPCAT1 haplotypes on a subjectâ€™s plasma metabolomic expression.</p></section><section id="Sec15"><h3 class="pmc_sec_title">The LPCAT1 haplotypes affect the phosphatidylcholine profile</h3>
<p id="Par29">LPCAT1 influences disease progression by enhancing the level of saturated phosphatidylcholine (PC) [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Notably, a higher percentage of individuals with the LPCAT1 haplotype 3 were found in the high-risk (42.86%) and CAD (52.38%) groups, while LPCAT1 haplotype 2 individuals showed a lower association with the high-risk group (20.45%) and the CAD groups (4.55%) (Fig.Â <a href="#Fig3" class="usa-link">3</a>D). LPCAT1 haplotype 1 (WT) was found to be evenly distributed across all three groups. When a gender-specific analysis was carried out, male participants with the LPCAT1 haplotype 2 displayed an increased PCae/LysoPC ratio (Fig.Â <a href="#Fig3" class="usa-link">3</a>E), a higher PCae level (Fig.Â <a href="#Fig3" class="usa-link">3</a>F), a higher total cholesterol level (Fig.Â <a href="#Fig3" class="usa-link">3</a>G), a higher HDL level (Fig.Â <a href="#Fig3" class="usa-link">3</a>H), and a lower TG level (Fig.Â <a href="#Fig3" class="usa-link">3</a>J) compared to haplotype 3. Conversely, females with the LPCAT1 haplotype 3 exhibited a lower PCae level (Fig.Â <a href="#Fig3" class="usa-link">3</a>F), a lower total cholesterol level (Fig.Â <a href="#Fig3" class="usa-link">3</a>G), a lower HDL level (Fig.Â <a href="#Fig3" class="usa-link">3</a>H), and a lower LDL level (Fig.Â <a href="#Fig3" class="usa-link">3</a>I) levels compared to haplotype 2. These findings indicate there are gender-specific differences in various lipid profiles and these are associated with LPCAT1 haplotypes within the groups.</p></section><section id="Sec16"><h3 class="pmc_sec_title">The ML-assisted identification of CAD risk factors across the whole cohort</h3>
<p id="Par30">We extended our investigation to explore the potential role of the LPCAT1 haplotypes in influencing phenotypic and metabolomic patterns across the entire cohort. This used an integrated ML-assisted analysis to identify specific risk factors among the control, high-risk, and CAD groups; thus leveraging the previously mentioned data [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. This comprehensive study comprised three main segments: (1) conducting GWAS using the Axiom Genome-Wide TWB 2.0 Array; (2) selecting metabolomic and clinical characteristic features; and (3) deriving and validating predictive models. Among various machine-learning models, the Random Forest model was found to show superior performance in terms of AUC and accuracy rate (Figure <a href="#MOESM1" class="usa-link">S4</a>).</p>
<p id="Par31">Figure <a href="#Fig4" class="usa-link">4</a>A illustrates the ML-assisted CAD prediction performance across the entire cohort by utilizing the 16 top-ranked interpretable features outlined in Fig.Â <a href="#Fig4" class="usa-link">4</a>B; this achieved an accuracy rate of 0.805 and an AUC of 0.917. Interestingly, the distribution of individuals carrying the LPCAT1 haplotype 1 (H-1) across the control, high-risk, and CAD groups was similar (Fig.Â <a href="#Fig4" class="usa-link">4</a>C). However, a notable increase in the percentage of participants carrying the LPCAT1 haplotype 3 (H-3) could be observed across the three groups, this rose from 13.33% in controls, to 19.28% in the high-risk group, and then to 27.21% in the CAD group (Ï‡<sup>2</sup> of <em>p</em>â€‰=â€‰0.02). In addition to the LPCAT1 haplotype, 15 other risk factors were identified. Elevated levels of sarcosine, creatinine, aspartate, isoleucine, proline, valerylcarnitine, LysoPCa C26:1, and having a male gender were all positively associated with the CAD group. Conversely, lower levels of serotonin, PCaa C40:2, PCae C34:3, PCae C40:2, PCae C42:3, and SM(OH) C22:1 were linked to the CAD group (Fig.Â <a href="#Fig4" class="usa-link">4</a>D and P). When distinguishing between the high-risk and CAD groups, the ML-assisted prediction model achieved an accuracy rate of 0.740 and an AUC of 0.880 on differentiating between these two groups (Table <a href="#MOESM1" class="usa-link">S13</a>). Although the performance was slightly decreased compared with the prediction of CAD using full datasets that include control, high-risk and CAD (Fig.Â <a href="#Fig4" class="usa-link">4</a>A), the AUC, a measurement of sensitivity and specificity, remained at a high score of 0.88. These findings underscored that there are distinct associations between CAD and various metabolites as well as various clinical factors within the cohort.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366056_40364_2025_821_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f7b9/12366056/821ee6ee23df/40364_2025_821_Fig4_HTML.jpg" loading="lazy" id="d33e1641" height="986" width="743" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>ML-assisted prediction of coronary artery disease (CAD). A ML-assisted trans-omics prediction model, based on clinical features, metabolome, and genome-wide SNPs, was used to identify patients with CAD across the whole cohort. <strong>A</strong>, The receiver operating characteristic (ROC) curve for predicting CAD. <strong>B</strong>, Adjusted odds ratios of features associated with CAD patients. The risks were estimated by the backward stepwise selection method. <strong>C</strong>, The frequency of LPCAT1 haplotype-3 is increased in the high-risk group and then is further raised in the CAD group. <strong>D-I</strong>, The deregulated levels of various clinical risk factors, amino acids and biogenic amines associated with CAD subjects, including sarcosine <strong>(D)</strong>, creatinine <strong>(E)</strong>, aspartate <strong>(F)</strong>, isoleucine <strong>(G)</strong>, proline <strong>(H)</strong> and serotonin <strong>(I)</strong>. <strong>J-P</strong>, The lipidomic pattern, with an elevation of valerylcarnitine (<strong>J</strong>) and lysoPC a26:1(<strong>K</strong>), as well as a decrease of other lipids, including PCaa C40:2 (<strong>L</strong>), PCae C34:3 (<strong>M</strong>), PCae C40:2 (<strong>N</strong>), PCae C42:3 (<strong>O</strong>), and SM(OH) C22:1 (<strong>P</strong>); these were associated with progression from the control group to the high-risk group to the CAD group. Quantification data are presented as meanâ€‰Â±â€‰SD and analyzed by one-way ANOVA with Bonferroni multiple comparison test. AUC, area under the curve. *<em>p</em>â€‰&lt;â€‰0.05; **<em>p</em>â€‰&lt;â€‰0.005</p></figcaption></figure></section></section><section id="Sec17"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par33">This study identifies several pivotal findings regarding CAD. <strong>Firstly</strong>, it shows that decreased plasma phospholipids are associated with high-risk patients and that there are further declines observed in CAD patients. This progressive decrease in phospholipids across control, high-risk, and CAD groups suggests a potential role for these lipids in the mechanism of CAD progression and as biomarker for CAD progression. <strong>Secondly</strong>, among the 10 top-ranked SNPs linked to CAD, five genes were identified as contributors to phosphatidylcholine pattern alterations. <strong>Thirdly</strong>, on employing a trans-omics approach, we identified a specific LPCAT1 haplotype to be associated with CAD patients. <strong>Lastly</strong>, using ML-assisted techniques, we established a trans-omics predictive model for CAD and confirmed that LPCAT1 haplotype is one of the 16 top-ranked risk factors in relation to CAD prediction. Overall, our findings highlight a multi-omics signature for CAD, and this, in turn, suggests that disturbances in glycerophospholipid metabolism due to LPCAT1 genetic variants seem to be associated with the occurrence of CAD.</p>
<section id="Sec18"><h3 class="pmc_sec_title">The significance of the trans-omics approach</h3>
<p id="Par34">Despite numerous genetic loci having been linked to CAD via genome-wide association studies, a mere 5% of predicted CAD-related SNPs are situated within gene boundaries plus the 30 kilobase pairs at either end of a gene [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Most of these loci demonstrate modest effects on a range of common diseases, making them challenging to use experimentally. Consequently, these associations offer limited mechanistic insights and have little in the way of translational applications. In this context, understanding allele interactions remains critical. Utilizing a trans-omics approach with parallel measurements across multiple omics is able to offer a more comprehensive comparison, better validation, and significant enhancement compared to existing integration methods [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. However, the application of the trans-omics approach to CAD studies has been relatively limited up to the present [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Delineated metabolomic patterns in CAD offers clues that allow physiological and pathological exploration of the disease [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>, <a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Nonetheless, further exploration is needed in order to elucidate the relationship between plasma metabolic profiling and the detailed characterization and quantification of metabolomic profiles in high-risk and CAD patients. Compared to the large-scale GWAS reported by Aragam et al. [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>], which focused on European populations, our study takes a multi-omics approach by integrating SNP and metabolomic data from a Taiwanese cohort. While Aragam et al. found no significant associations for LPCAT1, our metabolomics data suggest a possible role for this gene in CAD within the Taiwanese population, which points to there being potential population-specific genetic variation.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Metabolic pattern transition from high-risk patient to CAD patient</h3>
<p id="Par35">Plasma metabolites are postulated to serve as markers and effectors in cardiometabolic diseases [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Inter-individual metabolite variation may result from heritable factors, such as genome-wide significant loci, and clinical factors, such as sex, age, medication, comorbidities, and environmental factors. Approximately 20% of the variation in 66% of plasma metabolites is estimated to stem from heritability, while less than 20% of the variation in 93% of plasma metabolites is due to clinical factors [<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. Our study design involved three participant groups instead of two. This structure was aimed at eliminating confounding clinical factors and focused on potential heritable factors that are associated with CAD. Notably, the elevated levels of three amino acids (aspartate, isoleucine, proline) in the high-risk and CAD group participants suggest that these are associated with CAD risk factors such as type II diabetes mellitus and chronic inflammation. Additionally, an increased sarcosine level, a metabolite derived from dietary choline, was found to be associated with CAD, adding to the links between choline-derived metabolites and adverse cardiovascular events.</p>
<p id="Par36">In plasma, amino acids and lysophosphatidylcholine (LysoPC) show considerable heritability [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>]. Literature reports have highlighted altered plasma metabolomic profiles in individuals with CAD, including disruptions to aspartate and glutamate homeostasis and increased levels of isoleucine [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>, <a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. Our study also detected heightened plasma levels of aspartate and isoleucine in our high-risk participants, and, notably, these were even more elevated in individuals who had been diagnosed with CAD. Furthermore, elevated proline, another amino acid identified in our findings, was found to be associated with type II diabetes mellitus and chronic inflammation, which are established risk factors for CAD. Chronic sterile inflammation serves as a connection between type II diabetes mellitus and CAD [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>], as evidenced by significantly elevated levels of high-sensitive C-reactive protein (HS-CRP) in the CAD group compared to the control and high-risk groups (TableÂ <a href="#Tab1" class="usa-link">1</a>). This suggests a potential link between inflammation and CAD. Although elevated serotonin has been associated with coronary microvascular disease and coronary artery disease, the wide range in plasma cutoff levels used for diagnosis produces significant variation [<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>, <a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Interestingly, our results revealed decreased plasma serotonin levels in the high-risk and CAD group participants.</p>
<p id="Par37">Our analysis identified key genetic and metabolic differences between the high-risk and CAD groups. As shown in Fig.Â <a href="#Fig4" class="usa-link">4</a>, the frequency of LPCAT1 haplotype 3 increased progressively from 13.33% (control) to 19.28% (high-risk) and 27.21% (CAD) (Ï‡Â², <em>p</em>â€‰=â€‰0.02), which suggests that it has a potential role in disease progression. Additionally, distinct metabolomic shifts were observed when CAD was present, such as elevated levels of valerylcarnitine and LysoPCa C26:1, as well as decrease in the PCaa and PCae subclasses; these changes serve to further differentiate these two groups. Moreover, elevated plasma sarcosine, a downstream metabolite of dietary choline, showed an association with CAD. Metabolites derived from choline produced by gut microbiota, such as trimethylamine and trimethylamine-N-oxide, have been linked to increased cardiovascular risk [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. The metabolic pathway of dietary choline, which progresses via glycine, betaine and dimethylglycine and culminates in sarcosine, in parallel with a decrease in glycine betaine and an increase in dimethylglycine, has been linked to cardiometabolic risks and atherosclerosis [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>, <a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>]. Our results provide additional evidence supporting the association between sarcosine level and CAD.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Phosphatidylcholine and lysophosphatidylcholine in CAD</h3>
<p id="Par38">Phospholipids, primarily phosphatidylcholine and sphingomyelin, serve as crucial constituents of the plasma membrane and are the binding sites for proteins within the plasma membrane [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]; these are synthesized in the liver and subsequently secreted into the bloodstream. They have been implicated in glucose metabolism, with low levels of these metabolites correlating with insulin resistance [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>] and with an elevated risk of Type 2 diabetes mellitus [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. Minute alterations in phospholipid levels across various tissues can have a significant impact on the parameters associated with metabolic syndrome, such as lipid profile, obesity, and insulin resistance.</p>
<p id="Par39">Phosphatidylcholines encompassing a single fatty acid are termed LysoPC. These lipid compounds, which act as secondary messengers, are linked to various metabolic disorders, insulin resistance, and obesity [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. Their signaling cascade operates through the G protein-coupled receptor G2A and various Toll-like receptors [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>]. PC, when metabolized by gut microbiota, has been shown to promote cardiovascular disease [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. PCs and sphingomyelin within HDL are presumed to play a protective role against atherosclerosis. Perturbations in these lipids can impair HDL function, potentially leading to arteriosclerosis [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>].</p>
<p id="Par40">Among plasma metabolites, specific phosphatidylcholines, notably PC(32:1), PC(33:0), PC(35:1), PC(36:1), and PC(36:3), exhibit an inverse relationship with the coronary artery calcium score [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]; furthermore, studies have associated lower levels of PCs with cardiovascular disease [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>]. Previous study revealed that PC (a substrate) can be hydrolyzed by the enzyme phospholipase to release fatty acid and LysoPC (a product) [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. Interestingly, a trend towards higher levels of certain LysoPCs was found by other group [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>] and this study (Fig.Â <a href="#Fig4" class="usa-link">4</a>K). This is consistent with the metabolism of PC, namely there is a decrease in PC (Fig.Â <a href="#Fig4" class="usa-link">4</a>L-O) that is accompanied by an increase in LysoPC (Fig.Â <a href="#Fig4" class="usa-link">4</a>K). In addition, elevated fasting plasma valerylcarnitine has been linked to increased risk of cardiovascular death and of myocardial infarction among individuals with stable angina [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>]. Valerylcarnitine, derived from branched-chain amino acids or odd carbon-chain fatty acid metabolism, may serve as a signature for insulin resistance in CAD participants [<a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>, <a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. Our results demonstrated significantly higher plasma levels of valerylcarnitine and LysoPC 26:1 in CAD participants compared to the high-risk and control groups. Intriguingly, lower plasma levels of PCae were found to be associated with the CAD group of subjects compared to the other two groups.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Lipid profiles are associated with SNPs</h3>
<p id="Par41">LPCAT1 plays a crucial role in the Landsâ€™ circulation by transforming lysoPCs into PCs using phosphatidyl coenzyme A (Fig.Â <a href="#Fig5" class="usa-link">5</a>) [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. LPCAT1 has been found to be localized on the plasma membrane, in endoplasmic reticulum, and in lipid droplet membranes [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>, <a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. Its action as an enzyme brings about PC re-acylation in alveolar type II cells and red blood cells, and thus it regulates inflammatory lipid levels, and mediates O-palmitoylation of histone H4 in the nuclei [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>]. LPCAT1, which is under cellular redox status control and Ca<sup>2+</sup> concentration control, is able to re-acylate lysoPCs into PCs, thus removing oxidized and damaged unsaturated fatty acids from glycerophospholipids [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. Dysregulation of LPCAT1 has been linked to cancer cell genetic alterations, abnormal metabolism, plasma membrane restructuring, and malignant transformations with a poor prognosis [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Loss of LPCAT1 in cells exposed to excessive polyunsaturated fatty acids exacerbates cytotoxicity and induces various cellular responses via pathways including the unfolded protein response [<a href="#CR60" class="usa-link" aria-describedby="CR60">60</a>]. Moreover, recent studies have further implicated LPCAT1 in myocardial infarction, as evidenced by the findings from GWAS and other genetic datasets available in the dbGAP Gene-Trait Associations database. These investigations have suggested a potential association between LPCAT1 and increased susceptibility to myocardial infarction, underscoring its significance beyond its known roles in lipid metabolism and inflammation [<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>].</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366056_40364_2025_821_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f7b9/12366056/ef0480696b64/40364_2025_821_Fig5_HTML.jpg" loading="lazy" id="d33e1845" height="456" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Graphic summary illustrating the trans-omics signatures of CAD patients. <strong>A</strong>, In the Landâ€™s cycle, lysophosphatidylcholine acyltransferase (LPCAT) reacts with phosphatidyl coenzyme A and transforms LysoPCs into PCs. LysoPC is increasingly recognized as a key factor that is positively associated with atherosclerosis. <strong>B</strong>, LPCAT1 has also been found to be localized in the plasma membrane, endoplasmic reticulum, and the membrane of lipid droplets. Intracellular LPCAT rapidly converts LysoPCs into PCs, which are then assembled and secreted as VLDL. However, elevated intracellular LysoPCs will increase the amount of LysoPCs in the LDL and oxidated LDL and this results in atherosclerosis. <strong>C</strong>, The disturbed interaction of the Landâ€™s cycle and the Kennedy pathway might further affect the metabolism of choline resulting in increased sarcosine in the CAD patients. Elevated sarcosine might affect the downstream TCA and urea cycles. Abbreviations: 3PG, 3-phosphoglycerate; Î±KG, Î±-ketoglutaric acid; CDP-choline, cytidinediphosphocholine; DAG, diacylglycerol; GAA, guanidinoacetic acid; G6P, glucose 6-phosphate; HDL, high-density lipoprotein; LysoPC, Lysophosphatidylcholine; LPCAT, lysophosphatidylcholine acyltransferase; Lp-PLA2, lipoprotein-associated phospholipase A2; PC, phosphatidylcholine; TCA, tricarboxylic acid; VLDL, very low density lipoprotein</p></figcaption></figure><p id="Par43">In our findings, we observed an association between LPCAT1 haplotype 3 and CAD, which points to there being a significant impact of an individualâ€™s LPCAT1 haplotype on their plasma lipidomic pattern. Additionally, other identified SNPs, such as ATP13A2 and VAMP8, are also involved in lipid metabolism. ATP13A2 dysfunction results in lysosomal dysfunction, oxidative stress susceptibility, impaired mitochondrial function, and cell toxicity [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. Furthermore, ATP13A2 overexpression leads to increased intracellular phosphatidylethanolamine [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. VAMP8 is involved in membrane vesicular trafficking and shares a molecular pathway that has been associated with both CAD and periodontitis. These findings collectively highlight the interplay between various genetic variants associated with lipid metabolism and pinpoints their potential role in cardiovascular disease mechanisms [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>].</p></section><section id="Sec22"><h3 class="pmc_sec_title">Limitations and perspectives</h3>
<p id="Par44">Despite the careful utilization of clustering methodologies to explore extensive multi-omics datasets, which encompass clinical profiles, targeted metabolomics (p180-metabolites), and SNP information, in order to obtain insights into their associations with CAD, several areas warrant attention as limitations to our study. <strong>Firstly</strong>, our focus is primarily on targeted circulating metabolites when carrying out the CAD association, and this leaves open the question as to whether other metabolites might also influence our CAD prediction models. This question remains unanswered. <strong>Secondly</strong>, the unbalanced demographics between groups may have complicated our data interpretation. Key limitations include demographic differences, such as age, gender and comorbid conditions, all of which can introduce bias. The studyâ€™s observational nature limits causality, identifying associations rather than definitive relationships. Additionally, uncontrolled confounding factors, such as lifestyle and environmental influences, might have also affected our findings. The lack of imaging-based examinations in the high-risk group means that underlying subclinical atherosclerosis and/or undiagnosed CAD cannot be ruled out. A case-control study design that matches key variables would simplify interpretation and would provide clearer insights. <strong>Thirdly</strong>, design of our study, which is a cross-sectional one and derived from a single ethnic group, may restrict the generalizability of our findings to other populations. Thus, validation using a larger and more diverse cohort that includes ethnic diversity is essential. <strong>Fourthly</strong>, while we have identified a LPCAT1 haplotype as having a potential functional influence on LPCAT1 activity, its precise functional implications remain unknown. This will necessitate further investigation involving in vitro and in vivo models. <strong>Lastly</strong>, in order for the multi-omics signature to evolve into a biomarker profile for personalized care of CAD patients, a longitudinal study validating the utility of our prediction model over time is needed.</p>
<p id="Par45">Regarding the imbalanced issue wherein most of the participants in this study do not have CAD, we carried out a balance test using different prediction models by subsampling the training datasets with the ratio of CAD to non-CAD set at 1:1 (Table <a href="#MOESM1" class="usa-link">S14</a>). Our results revealed that the sensitivity, specificity, accuracy and AUC were similar to the original results (Table <a href="#MOESM1" class="usa-link">S3</a>). However, when we match age and gender during the sampling process, the accuracy and AUC drop by about 5% (Table <a href="#MOESM1" class="usa-link">S14</a>). Other performance matrices, including AUC-precision recall curve (AUC-PR), Brier score and calibration curve [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>], are presented in Figure <a href="#MOESM1" class="usa-link">S8</a>. We achieved an AUC of 0.92 and an AUC-PR of 0.74. AUC-PR may be sensitive to precision (PPV, Positive Predictive Value); however, precision is dependent on disease rate and it will change when disease rate is different. Regarding the Brier score and the calibration curve, these are more suitable when targeting a risk assessment application (more like a regression issue). The calibration curve is used to compare the modelâ€™s predicted probabilities and verify whether it is consistent with the observed probabilities. However, our modelâ€™s predicted probabilities tended to be higher than observed probabilities. Therefore, we adapted a probability calibration method. The slope of the calibration curve becomes closer to 1 after adaption, but this caused a lower AUC. Since our model is dealing with a classification issue and trying to identify important features related to potential pathogenetic factors associated with disease progression, we have focused in this study on a model that provides a higher AUC.</p>
<p id="Par46">To understand the potential influence of medication on our study, we conducted the following experiments. Firstly, six medications (statin, fibrate, Î²-blocker, calcium channel blocker, angiotensin receptor blocker, and diuretics) were included in model analysis; to do this we used matched sampling to ensure that the same usage proportion in the CAD and non-CAD groups. Compared to the matched sampling without medication adjustment, we found that matching by age, gender, and statin slightly decreased the model performance (Table <a href="#MOESM1" class="usa-link">S15</a>). Secondly, we created new features based on the assumption of an interaction between statin and metabolites, and carried out the model analyses again. Notably, when matching the statin condition, there was a big drop in the specificity (it drops from 0.849 to 0.526), indicating that adding interaction features into the model seemed to result in a misclassification of non-CAD subjects as CAD patients (Table <a href="#MOESM1" class="usa-link">S16</a>).</p></section></section><section id="Sec23"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par47">Our findings underscore the potential of integrating machine learning models with traditional approaches in order to uncover new biological mechanisms. When we compared our approach to various existing risk assessment tools (Table <a href="#MOESM1" class="usa-link">S17</a>) in order to identify individuals at high risk of CAD, our prediction model showed promise. Our approach appears to outperform many demographic-based, blood biochemistry-based, and image-based assessment methods; nevertheless, further validation is necessary. An understanding of the key SNPs associated with CAD and their interactions with metabolites linked to CAD will shed light on the potential genetic predispositions and metabolic changes that form the complex disease CAD. Ultimately, the creation of a multi-omics predictive model that is capable of distinguishing CAD patients from high-risk individuals will present an avenue for developing personalized medical strategies for CAD prevention.</p></section><section id="Sec24"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366056/bin/40364_2025_821_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (4.1MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We acknowledge the participant recruitment and sample preservation carried out as part of Northeastern Taiwan Community Medicine Research Cohort Study.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>ANOVA</dt>
<dd><p id="Par51">Analysis of variance.</p></dd>
<dt>ASCVD</dt>
<dd><p id="Par52">Atherosclerotic cardiovascular disease</p></dd>
<dt>AUC</dt>
<dd><p id="Par53">Area under the curve</p></dd>
<dt>AUC-PR</dt>
<dd><p id="Par54">Area under the curve-precision recall curve</p></dd>
<dt>BMI</dt>
<dd><p id="Par55">Body mass index</p></dd>
<dt>CAD</dt>
<dd><p id="Par56">Coronary artery disease</p></dd>
<dt>CKD</dt>
<dd><p id="Par57">Chronic kidney disease</p></dd>
<dt>DBP</dt>
<dd><p id="Par58">Diastolic blood pressure</p></dd>
<dt>GWAS</dt>
<dd><p id="Par59">Genome-wide association studies</p></dd>
<dt>HDL</dt>
<dd><p id="Par60">High-density lipoprotein</p></dd>
<dt>LASSO</dt>
<dd><p id="Par61">Least absolute shrinkage and selection operator</p></dd>
<dt>LDL</dt>
<dd><p id="Par62">Low-density lipoprotein</p></dd>
<dt>LPCAT1</dt>
<dd><p id="Par63">Lysophosphatidylcholine acyltransferase 1</p></dd>
<dt>ML</dt>
<dd><p id="Par64">Machine learning</p></dd>
<dt>OPLS-DA</dt>
<dd><p id="Par65">Orthogonal partial least squares-discriminant analysis</p></dd>
<dt>SBP</dt>
<dd><p id="Par66">Systolic blood pressure</p></dd>
<dt>SNPs</dt>
<dd><p id="Par67">Single-nucleotide polymorphisms</p></dd>
<dt>SD</dt>
<dd><p id="Par68">Standard deviations</p></dd>
<dt>SVM</dt>
<dd><p id="Par69">Support vector machine</p></dd>
<dt>TG</dt>
<dd><p id="Par70">Triglyceride</p></dd>
<dt>XGBoost</dt>
<dd><p id="Par71">Extreme gradient boosting</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>All the authors contributed to the manuscript preparation. Chi-Chun Lai, Huey-Kang Sytwu and Ting-Fen Tsai co-designed the research framework. Paul Wei-Che Hsu and Chi-Hsiao Yeh co-designed the manuscript topics and structure, prepared the figures, and drafted the manuscript. Chi-Hsiao Yeh and Ning-I Yang recruited subjects and defined the clinical stages. Tsung-Hsien Tsai designed and wrote the AI-assisted method and supervised the ML analyses. Yun-Hsuan Chan contributed to ML analysis. Chi-Jen Lo, Yi-Ju Chou and Mei-Ling Cheng participated in metabolic analyses. Wayne Huey-Herng Sheu provided critical and inspired comments. Ting-Fen Tsai wrote the final version of manuscript. All authors have read and agreed to the published version of the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>This research was funded by grants from the Ministry of Health and Welfare (Smart Healthcare for Obesity Therapeutics, PD-109-GP-02, MG-110-GP-03 and MG-111-GP-03 to HKS and MG-112-GP-03 to WHHS; Development of Precision Prevention and Treatment Strategies for Metabolic and Related Chronic Diseases: Prediction and Implementation of an Intelligent Prediction System, MG-113-GP-03 and MG-114-GP-01 to WHHS), and from Chang Gung Memorial Hospital (CMRPG2K0141-4, CORPG2P0181 to CCL; CORPG2H0041-0043, CMRPG2H000091-0093 to CHY).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The SNP datasets generated and analyzed in this study are available in the Gene Expression Omnibus (GEO) repository; accession code <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE303414" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE303414</a>. In addition, both the SNP and metabolomics datasets can be downloaded directly from NHRI website: <a href="http://bioinfolab.nhri.edu.tw/pub/bio2025.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://bioinfolab.nhri.edu.tw/pub/bio2025.html</a>.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par48">The study protocol adheres to the ethical guidelines outlined in the 1975 Declaration of Helsinki and received approval from the Institutional Review Board of Chang Gung Medical Foundation. (IRB No: 201800686B0 and 202000077B0A3). Informed consent was obtained from all participants in the study.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par49">Not applicable.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par50">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisherâ€™s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Paul Wei-Che Hsu and Chi-Hsiao Yeh contributed equally to this article as co-first authors.</p></div>
</div></section><section id="_ci93_" lang="en" class="contrib-info"><h2 class="pmc_sec_title">Contributor Information</h2>
<p>Chi-Chun Lai, Email: chichun.lai@gmail.com.</p>
<p>Huey-Kang Sytwu, Email: sytwu@nhri.edu.tw.</p>
<p>Ting-Fen Tsai, Email: tftsai@nycu.edu.tw.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Pulit SL, Stoneman C, Morris AP, Wood AR, Glastonbury CA, Tyrrell J, et al. Meta-analysis of genome-wide association studies for body fat distribution in 694 649 individuals of European ancestry. Hum Mol Genet. 2019;28:166â€“74.
</cite> [<a href="https://doi.org/10.1093/hmg/ddy327" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6298238/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30239722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pulit%20SL,%20Stoneman%20C,%20Morris%20AP,%20Wood%20AR,%20Glastonbury%20CA,%20Tyrrell%20J,%20et%20al.%20Meta-analysis%20of%20genome-wide%20association%20studies%20for%20body%20fat%20distribution%20in%20694%20649%20individuals%20of%20European%20ancestry.%20Hum%20Mol%20Genet.%202019;28:166%E2%80%9374." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Nelson CP, Goel A, Butterworth AS, Kanoni S, Webb TR, Marouli E, et al. Association analyses based on false discovery rate implicate new loci for coronary artery disease. Nat Genet. 2017;49:1385â€“91.
</cite> [<a href="https://doi.org/10.1038/ng.3913" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28714975/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nelson%20CP,%20Goel%20A,%20Butterworth%20AS,%20Kanoni%20S,%20Webb%20TR,%20Marouli%20E,%20et%20al.%20Association%20analyses%20based%20on%20false%20discovery%20rate%20implicate%20new%20loci%20for%20coronary%20artery%20disease.%20Nat%20Genet.%202017;49:1385%E2%80%9391." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Myocardial Infarction Genetics and CARDIoGRAM Exome Consortia Investigators, Stitziel NO, Stirrups KE, Masca NG, Erdmann J, Ferrario PG, KÃ¶nig IR, et al. Coding variation in ANGPTL4, LPL, and SVEP1 and the risk of coronary disease. New Engl J Med. 2016;374:1134â€“44.
</cite> [<a href="https://doi.org/10.1056/NEJMoa1507652" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4850838/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26934567/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Myocardial%20Infarction%20Genetics%20and%20CARDIoGRAM%20Exome%20Consortia%20Investigators,%20Stitziel%20NO,%20Stirrups%20KE,%20Masca%20NG,%20Erdmann%20J,%20Ferrario%20PG,%20K%C3%B6nig%20IR,%20et%20al.%20Coding%20variation%20in%20ANGPTL4,%20LPL,%20and%20SVEP1%20and%20the%20risk%20of%20coronary%20disease.%20New%20Engl%20J%20Med.%202016;374:1134%E2%80%9344." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Nikpay M, Goel A, Won H-H, Hall LM, Willenborg C, Kanoni S, et al. A comprehensive 1000 Genomesâ€“based genome-wide association meta-analysis of coronary artery disease. Nat Genet. 2015;47:1121â€“30.
</cite> [<a href="https://doi.org/10.1038/ng.3396" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4589895/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26343387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nikpay%20M,%20Goel%20A,%20Won%20H-H,%20Hall%20LM,%20Willenborg%20C,%20Kanoni%20S,%20et%20al.%20A%20comprehensive%201000%20Genomes%E2%80%93based%20genome-wide%20association%20meta-analysis%20of%20coronary%20artery%20disease.%20Nat%20Genet.%202015;47:1121%E2%80%9330." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Aragam KG, Jiang T, Goel A, Kanoni S, Wolford BN, Atri DS, et al. Discovery and systematic characterization of risk variants and genes for coronary artery disease in over a million participants. Nat Genet. 2022;54:1803â€“15.
</cite> [<a href="https://doi.org/10.1038/s41588-022-01233-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9729111/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36474045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aragam%20KG,%20Jiang%20T,%20Goel%20A,%20Kanoni%20S,%20Wolford%20BN,%20Atri%20DS,%20et%20al.%20Discovery%20and%20systematic%20characterization%20of%20risk%20variants%20and%20genes%20for%20coronary%20artery%20disease%20in%20over%20a%20million%20participants.%20Nat%20Genet.%202022;54:1803%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Kessler T, Vilne B, Schunkert H. The impact of genome-wide association studies on the pathophysiology and therapy of cardiovascular disease. EMBO Mol Med. 2016;8:688â€“701.
</cite> [<a href="https://doi.org/10.15252/emmm.201506174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4931285/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27189168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kessler%20T,%20Vilne%20B,%20Schunkert%20H.%20The%20impact%20of%20genome-wide%20association%20studies%20on%20the%20pathophysiology%20and%20therapy%20of%20cardiovascular%20disease.%20EMBO%20Mol%20Med.%202016;8:688%E2%80%93701." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Vilne B, Schunkert H. Integrating genes affecting coronary artery disease in functional networks by multi-OMICs approach. Front Cardiovasc Med. 2018;5:89.
</cite> [<a href="https://doi.org/10.3389/fcvm.2018.00089" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6056735/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30065929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vilne%20B,%20Schunkert%20H.%20Integrating%20genes%20affecting%20coronary%20artery%20disease%20in%20functional%20networks%20by%20multi-OMICs%20approach.%20Front%20Cardiovasc%20Med.%202018;5:89." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Rau CD, Lusis AJ, Wang Y. Systems genetics for mechanistic discovery in heart diseases. Circ Res. 2020;126:1795â€“815.
</cite> [<a href="https://doi.org/10.1161/CIRCRESAHA.119.315863" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7286104/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32496909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rau%20CD,%20Lusis%20AJ,%20Wang%20Y.%20Systems%20genetics%20for%20mechanistic%20discovery%20in%20heart%20diseases.%20Circ%20Res.%202020;126:1795%E2%80%93815." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Buniello A, MacArthur JAL, Cerezo M, Harris LW, Hayhurst J, Malangone C, et al. The NHGRI-EBI GWAS catalog of published genome-wide association studies, targeted arrays and summary statistics 2019. Nucleic Acids Res. 2019;47:D1005â€“12.
</cite> [<a href="https://doi.org/10.1093/nar/gky1120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6323933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30445434/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buniello%20A,%20MacArthur%20JAL,%20Cerezo%20M,%20Harris%20LW,%20Hayhurst%20J,%20Malangone%20C,%20et%20al.%20The%20NHGRI-EBI%20GWAS%20catalog%20of%20published%20genome-wide%20association%20studies,%20targeted%20arrays%20and%20summary%20statistics%202019.%20Nucleic%20Acids%20Res.%202019;47:D1005%E2%80%9312." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Ritchie MD, Holzinger ER, Li R, Pendergrass SA, Kim D. Methods of integrating data to uncover genotypeâ€“phenotype interactions. Nat Rev Genet. 2015;16:85â€“97.
</cite> [<a href="https://doi.org/10.1038/nrg3868" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25582081/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ritchie%20MD,%20Holzinger%20ER,%20Li%20R,%20Pendergrass%20SA,%20Kim%20D.%20Methods%20of%20integrating%20data%20to%20uncover%20genotype%E2%80%93phenotype%20interactions.%20Nat%20Rev%20Genet.%202015;16:85%E2%80%9397." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>WÃ¶rheide MA, Krumsiek J, KastenmÃ¼ller G, Arnold M. Multi-omics integration in biomedical research - A metabolomics-centric review. Anal Chim Acta. 2021;1141:144â€“62.
</cite> [<a href="https://doi.org/10.1016/j.aca.2020.10.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7701361/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33248648/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?W%C3%B6rheide%20MA,%20Krumsiek%20J,%20Kastenm%C3%BCller%20G,%20Arnold%20M.%20Multi-omics%20integration%20in%20biomedical%20research%20-%20A%20metabolomics-centric%20review.%20Anal%20Chim%20Acta.%202021;1141:144%E2%80%9362." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Krumsiek J, Bartel J, Theis FJ. Computational approaches for systems metabolomics. Curr Opin Biotechnol. 2016;39:198â€“206.
</cite> [<a href="https://doi.org/10.1016/j.copbio.2016.04.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27135552/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krumsiek%20J,%20Bartel%20J,%20Theis%20FJ.%20Computational%20approaches%20for%20systems%20metabolomics.%20Curr%20Opin%20Biotechnol.%202016;39:198%E2%80%93206." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Toledo JB, Arnold M, KastenmÃ¼ller G, Chang R, Baillie RA, Han X, et al. Metabolic network failures in alzheimerâ€™s disease: A biochemical road map. Alzheimers Dement. 2017;13:965â€“84.
</cite> [<a href="https://doi.org/10.1016/j.jalz.2017.01.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5866045/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28341160/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Toledo%20JB,%20Arnold%20M,%20Kastenm%C3%BCller%20G,%20Chang%20R,%20Baillie%20RA,%20Han%20X,%20et%20al.%20Metabolic%20network%20failures%20in%20alzheimer%E2%80%99s%20disease:%20A%20biochemical%20road%20map.%20Alzheimers%20Dement.%202017;13:965%E2%80%9384." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Fall T, Salihovic S, Brandmaier S, Nowak C, Ganna A, Gustafsson S, et al. Non-targeted metabolomics combined with genetic analyses identifies bile acid synthesis and phospholipid metabolism as being associated with incident type 2 diabetes. Diabetologia. 2016;59:2114â€“24.
</cite> [<a href="https://doi.org/10.1007/s00125-016-4041-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5451119/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27406814/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fall%20T,%20Salihovic%20S,%20Brandmaier%20S,%20Nowak%20C,%20Ganna%20A,%20Gustafsson%20S,%20et%20al.%20Non-targeted%20metabolomics%20combined%20with%20genetic%20analyses%20identifies%20bile%20acid%20synthesis%20and%20phospholipid%20metabolism%20as%20being%20associated%20with%20incident%20type%202%20diabetes.%20Diabetologia.%202016;59:2114%E2%80%9324." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Krishnan S, Huang J, Lee H, Guerrero A, Berglund L, Anuurad E, et al. Combined high-density lipoprotein proteomic and glycomic profiles in patients at risk for coronary artery disease. J Proteome Res. 2015;14:5109â€“18.
</cite> [<a href="https://doi.org/10.1021/acs.jproteome.5b00730" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26535788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Krishnan%20S,%20Huang%20J,%20Lee%20H,%20Guerrero%20A,%20Berglund%20L,%20Anuurad%20E,%20et%20al.%20Combined%20high-density%20lipoprotein%20proteomic%20and%20glycomic%20profiles%20in%20patients%20at%20risk%20for%20coronary%20artery%20disease.%20J%20Proteome%20Res.%202015;14:5109%E2%80%9318." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Kraus WE, Muoio DM, Stevens R, Craig D, Bain JR, Grass E, et al. Metabolomic quantitative trait loci (mQTL) mapping implicates the ubiquitin proteasome system in cardiovascular disease pathogenesis. PLoS Genet. 2015;11:e1005553.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1005553" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4634848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26540294/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kraus%20WE,%20Muoio%20DM,%20Stevens%20R,%20Craig%20D,%20Bain%20JR,%20Grass%20E,%20et%20al.%20Metabolomic%20quantitative%20trait%20loci%20(mQTL)%20mapping%20implicates%20the%20ubiquitin%20proteasome%20system%20in%20cardiovascular%20disease%20pathogenesis.%20PLoS%20Genet.%202015;11:e1005553." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Cheng S, Shah SH, Corwin EJ, Fiehn O, Fitzgerald RL, Gerszten RE, American Heart Association Council on Functional Genomics and Translational Biology; Council on Cardiovascular and Stroke Nursing; Council on Clinical Cardiology; and Stroke Council, et al. Potential impact and study considerations of metabolomics in cardiovascular health and disease: A scientific statement from the American heart association. Circ Cardiovasc Genet. 2017;10:e000032.
</cite> [<a href="https://doi.org/10.1161/HCG.0000000000000032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6001363/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28360086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng%20S,%20Shah%20SH,%20Corwin%20EJ,%20Fiehn%20O,%20Fitzgerald%20RL,%20Gerszten%20RE,%20American%20Heart%20Association%20Council%20on%20Functional%20Genomics%20and%20Translational%20Biology;%20Council%20on%20Cardiovascular%20and%20Stroke%20Nursing;%20Council%20on%20Clinical%20Cardiology;%20and%20Stroke%20Council,%20et%20al.%20Potential%20impact%20and%20study%20considerations%20of%20metabolomics%20in%20cardiovascular%20health%20and%20disease:%20A%20scientific%20statement%20from%20the%20American%20heart%20association.%20Circ%20Cardiovasc%20Genet.%202017;10:e000032." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Hartiala JA, Tang WH, Wang Z, Crow AL, Stewart AF, Roberts R, et al. Genome-wide association study and targeted metabolomics identifies sex-specific association of CPS1 with coronary artery disease. Nat Commun. 2016;7:10558.
</cite> [<a href="https://doi.org/10.1038/ncomms10558" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4740183/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26822151/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hartiala%20JA,%20Tang%20WH,%20Wang%20Z,%20Crow%20AL,%20Stewart%20AF,%20Roberts%20R,%20et%20al.%20Genome-wide%20association%20study%20and%20targeted%20metabolomics%20identifies%20sex-specific%20association%20of%20CPS1%20with%20coronary%20artery%20disease.%20Nat%20Commun.%202016;7:10558." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Ruiz-Canela M, Hruby A, Clish CB, Liang L, MartÃ­nez-GonzÃ¡lez MA, Hu FB. Comprehensive metabolomic profiling and incident cardiovascular disease: A systematic review. J Am Heart Assoc. 2017;6:e005705.
</cite> [<a href="https://doi.org/10.1161/JAHA.117.005705" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5721826/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28963102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ruiz-Canela%20M,%20Hruby%20A,%20Clish%20CB,%20Liang%20L,%20Mart%C3%ADnez-Gonz%C3%A1lez%20MA,%20Hu%20FB.%20Comprehensive%20metabolomic%20profiling%20and%20incident%20cardiovascular%20disease:%20A%20systematic%20review.%20J%20Am%20Heart%20Assoc.%202017;6:e005705." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Yang NI, Yeh CH, Tsai TH, Chou YJ, Hsu PW, Li CH, et al. Artificial intelligence-assisted identification of genetic factors predisposing high-risk individuals to asymptomatic heart failure. Cells. 2021;10:2430.
</cite> [<a href="https://doi.org/10.3390/cells10092430" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8470162/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34572079/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang%20NI,%20Yeh%20CH,%20Tsai%20TH,%20Chou%20YJ,%20Hsu%20PW,%20Li%20CH,%20et%20al.%20Artificial%20intelligence-assisted%20identification%20of%20genetic%20factors%20predisposing%20high-risk%20individuals%20to%20asymptomatic%20heart%20failure.%20Cells.%202021;10:2430." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Goff DC Jr., Lloyd-Jones DM, Bennett G, Coady S, Dâ€™Agostino RB, Gibbons R, et al. 2013 ACC/AHA guideline on the assessment of cardiovascular risk: a report of the American college of cardiology/american heart association task force on practice guidelines. Circulation. 2014;129(25 Suppl 2):S49â€“73.
</cite> [<a href="https://doi.org/10.1161/01.cir.0000437741.48606.98" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24222018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Goff%20DC%20Jr.,%20Lloyd-Jones%20DM,%20Bennett%20G,%20Coady%20S,%20D%E2%80%99Agostino%20RB,%20Gibbons%20R,%20et%20al.%202013%20ACC/AHA%20guideline%20on%20the%20assessment%20of%20cardiovascular%20risk:%20a%20report%20of%20the%20American%20college%20of%20cardiology/american%20heart%20association%20task%20force%20on%20practice%20guidelines.%20Circulation.%202014;129(25%20Suppl%202):S49%E2%80%9373." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Whelton PK, Carey RM, Aronow WS, Jr. Casey DE, Collins KJ, Dennison Himmelfarb C, et al. 2017 acc/aha/aapa/abc/acpm/ags/apha/ash/aspc/nma/pcna guideline for the prevention, detection, evaluation, and management of high blood pressure in adults: executive summary: A report of the American college of cardiology/american heart association task force on clinical practice guidelines. Hypertension. 2018;71:1269â€“324.
</cite> [<a href="https://doi.org/10.1161/HYP.0000000000000066" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29133354/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Whelton%20PK,%20Carey%20RM,%20Aronow%20WS,%20Jr.%20Casey%20DE,%20Collins%20KJ,%20Dennison%20Himmelfarb%20C,%20et%20al.%202017%20acc/aha/aapa/abc/acpm/ags/apha/ash/aspc/nma/pcna%20guideline%20for%20the%20prevention,%20detection,%20evaluation,%20and%20management%20of%20high%20blood%20pressure%20in%20adults:%20executive%20summary:%20A%20report%20of%20the%20American%20college%20of%20cardiology/american%20heart%20association%20task%20force%20on%20clinical%20practice%20guidelines.%20Hypertension.%202018;71:1269%E2%80%93324." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Kidney Disease: Improving Global Outcomes (KDIGO) CKD Work Group. KDIGO 2024 clinical practice guideline for the evaluation and management of chronic kidney disease. Kidney Int. 2024;105(4S):S117â€“314.
</cite> [<a href="https://doi.org/10.1016/j.kint.2023.10.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38490803/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kidney%20Disease:%20Improving%20Global%20Outcomes%20(KDIGO)%20CKD%20Work%20Group.%20KDIGO%202024%20clinical%20practice%20guideline%20for%20the%20evaluation%20and%20management%20of%20chronic%20kidney%20disease.%20Kidney%20Int.%202024;105(4S):S117%E2%80%93314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Cheng ML, Wang CH, Shiao MS, Liu MH, Huang YY, Huang CY, et al. Metabolic disturbances identified in plasma are associated with outcomes in patients with heart failure: diagnostic and prognostic value of metabolomics. J Am Coll Cardiol. 2015;65:1509â€“20.
</cite> [<a href="https://doi.org/10.1016/j.jacc.2015.02.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25881932/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cheng%20ML,%20Wang%20CH,%20Shiao%20MS,%20Liu%20MH,%20Huang%20YY,%20Huang%20CY,%20et%20al.%20Metabolic%20disturbances%20identified%20in%20plasma%20are%20associated%20with%20outcomes%20in%20patients%20with%20heart%20failure:%20diagnostic%20and%20prognostic%20value%20of%20metabolomics.%20J%20Am%20Coll%20Cardiol.%202015;65:1509%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Worley B, Powers R. Multivariate analysis in metabolomics. Curr Metabolomics. 2013;1:92â€“107.
</cite> [<a href="https://doi.org/10.2174/2213235X11301010092" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4465187/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26078916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Worley%20B,%20Powers%20R.%20Multivariate%20analysis%20in%20metabolomics.%20Curr%20Metabolomics.%202013;1:92%E2%80%93107." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Zhou W, Nielsen JB, Fritsche LG, Dey R, Gabrielsen ME, Wolford BN, et al. Efficiently controlling for case-control imbalance and sample relatedness in large-scale genetic association studies. Nat Genet. 2018;50:1335â€“41.
</cite> [<a href="https://doi.org/10.1038/s41588-018-0184-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6119127/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30104761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhou%20W,%20Nielsen%20JB,%20Fritsche%20LG,%20Dey%20R,%20Gabrielsen%20ME,%20Wolford%20BN,%20et%20al.%20Efficiently%20controlling%20for%20case-control%20imbalance%20and%20sample%20relatedness%20in%20large-scale%20genetic%20association%20studies.%20Nat%20Genet.%202018;50:1335%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Thuijs DJFM, Kappetein AP, Serruys PW, Mohr FW, Morice MC, Mack MJ, et al. SYNTAX extended survival investigators. Percutaneous coronary intervention versus coronary artery bypass grafting in patients with three-vessel or left main coronary artery disease: 10-year follow-up of the multicentre randomised controlled SYNTAX trial. Lancet. 2019;394:1325â€“34.
</cite> [<a href="https://doi.org/10.1016/S0140-6736(19)31997-X" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31488373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thuijs%20DJFM,%20Kappetein%20AP,%20Serruys%20PW,%20Mohr%20FW,%20Morice%20MC,%20Mack%20MJ,%20et%20al.%20SYNTAX%20extended%20survival%20investigators.%20Percutaneous%20coronary%20intervention%20versus%20coronary%20artery%20bypass%20grafting%20in%20patients%20with%20three-vessel%20or%20left%20main%20coronary%20artery%20disease:%2010-year%20follow-up%20of%20the%20multicentre%20randomised%20controlled%20SYNTAX%20trial.%20Lancet.%202019;394:1325%E2%80%9334." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Cabbia A, Hilbers PA, van Riel NA. A distance-based framework for the characterization of metabolic heterogeneity in large sets of genome-scale metabolic models. Patterns. 2020;1:100080.
</cite> [<a href="https://doi.org/10.1016/j.patter.2020.100080" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7660451/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33205127/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cabbia%20A,%20Hilbers%20PA,%20van%20Riel%20NA.%20A%20distance-based%20framework%20for%20the%20characterization%20of%20metabolic%20heterogeneity%20in%20large%20sets%20of%20genome-scale%20metabolic%20models.%20Patterns.%202020;1:100080." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Peng X, Zhu H, Feng J, Shen C, Zhang H, Zhou JT. Deep clustering with sample-assignment invariance prior. IEEE Trans Neural Netw Learn Syst. 2020;31:4857â€“68.
</cite> [<a href="https://doi.org/10.1109/TNNLS.2019.2958324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31902782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Peng%20X,%20Zhu%20H,%20Feng%20J,%20Shen%20C,%20Zhang%20H,%20Zhou%20JT.%20Deep%20clustering%20with%20sample-assignment%20invariance%20prior.%20IEEE%20Trans%20Neural%20Netw%20Learn%20Syst.%202020;31:4857%E2%80%9368." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Bi J, Ichu TA, Zanca C, Yang H, Zhang W, Gu Y, et al. Oncogene amplification in growth factor signaling pathways renders cancers dependent on membrane lipid remodeling. Cell Metab. 2019;30:525â€“38.e8. 10.1016/j.cmet.2019.06.014</cite> [<a href="https://doi.org/10.1016/j.cmet.2019.06.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6742496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31303424/" class="usa-link">PubMed</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Arneson D, Shu L, Tsai B, Barrere-Cain R, Sun C, Yang X. Multidimensional integrative genomics approaches to dissecting cardiovascular disease. Front Cardiovasc Med. 2017;4:8.
</cite> [<a href="https://doi.org/10.3389/fcvm.2017.00008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5327355/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28289683/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arneson%20D,%20Shu%20L,%20Tsai%20B,%20Barrere-Cain%20R,%20Sun%20C,%20Yang%20X.%20Multidimensional%20integrative%20genomics%20approaches%20to%20dissecting%20cardiovascular%20disease.%20Front%20Cardiovasc%20Med.%202017;4:8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Tzoulaki I, CastagnÃ© R, BoulangÃ© CL, Karaman I, Chekmeneva E, Evangelou E, et al. Serum metabolic signatures of coronary and carotid atherosclerosis and subsequent cardiovascular disease. Eur Heart J. 2019;40:2883â€“96.
</cite> [<a href="https://doi.org/10.1093/eurheartj/ehz235" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7963131/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31102408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tzoulaki%20I,%20Castagn%C3%A9%20R,%20Boulang%C3%A9%20CL,%20Karaman%20I,%20Chekmeneva%20E,%20Evangelou%20E,%20et%20al.%20Serum%20metabolic%20signatures%20of%20coronary%20and%20carotid%20atherosclerosis%20and%20subsequent%20cardiovascular%20disease.%20Eur%20Heart%20J.%202019;40:2883%E2%80%9396." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Chen H, Wang Z, Qin M, Zhang B, Lin L, Ma Q, et al. Comprehensive metabolomics identified the prominent role of glycerophospholipid metabolism in coronary artery disease progression. Front Mol Biosci. 2021;8:632950.
</cite> [<a href="https://doi.org/10.3389/fmolb.2021.632950" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8080796/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33937325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20H,%20Wang%20Z,%20Qin%20M,%20Zhang%20B,%20Lin%20L,%20Ma%20Q,%20et%20al.%20Comprehensive%20metabolomics%20identified%20the%20prominent%20role%20of%20glycerophospholipid%20metabolism%20in%20coronary%20artery%20disease%20progression.%20Front%20Mol%20Biosci.%202021;8:632950." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Ottensmann L, Tabassum R, Ruotsalainen SE, Gerl MJ, Klose C, WidÃ©n E, et al. Genome-wide association analysis of plasma lipidome identifies 495 genetic associations. Nat Commun. 2023;14:6934.
</cite> [<a href="https://doi.org/10.1038/s41467-023-42532-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10618167/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37907536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ottensmann%20L,%20Tabassum%20R,%20Ruotsalainen%20SE,%20Gerl%20MJ,%20Klose%20C,%20Wid%C3%A9n%20E,%20et%20al.%20Genome-wide%20association%20analysis%20of%20plasma%20lipidome%20identifies%20495%20genetic%20associations.%20Nat%20Commun.%202023;14:6934." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Ottosson F, Smith E, Melander O, Fernandez C. Altered asparagine and glutamate homeostasis precede coronary artery disease and type 2 diabetes. J Clin Endocrinol Metab. 2018;103:3060â€“69.
</cite> [<a href="https://doi.org/10.1210/jc.2018-00546" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29788285/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ottosson%20F,%20Smith%20E,%20Melander%20O,%20Fernandez%20C.%20Altered%20asparagine%20and%20glutamate%20homeostasis%20precede%20coronary%20artery%20disease%20and%20type%202%20diabetes.%20J%20Clin%20Endocrinol%20Metab.%202018;103:3060%E2%80%9369." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>McGarrah RW, Crown SB, Zhang GF, Shah SH, Newgard CB. Cardiovascular metabolomics. Circ Res. 2018;122:1238â€“58.
</cite> [<a href="https://doi.org/10.1161/CIRCRESAHA.117.311002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6029726/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29700070/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McGarrah%20RW,%20Crown%20SB,%20Zhang%20GF,%20Shah%20SH,%20Newgard%20CB.%20Cardiovascular%20metabolomics.%20Circ%20Res.%202018;122:1238%E2%80%9358." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Aravindhan V, Madhumitha H. Metainflammation in diabetic coronary artery disease: emerging role of innate and adaptive immune responses. J Diabetes Res. 2016;2016:6264149.
</cite> [<a href="https://doi.org/10.1155/2016/6264149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5004008/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27610390/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Aravindhan%20V,%20Madhumitha%20H.%20Metainflammation%20in%20diabetic%20coronary%20artery%20disease:%20emerging%20role%20of%20innate%20and%20adaptive%20immune%20responses.%20J%20Diabetes%20Res.%202016;2016:6264149." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Odaka Y, Takahashi J, Tsuburaya R, Nishimiya K, Hao K, Matsumoto Y, et al. Plasma concentration of serotonin is a novel biomarker for coronary microvascular dysfunction in patients with suspected angina and unobstructive coronary arteries. Eur Heart J. 2017;38:489â€“96.
</cite> [<a href="https://doi.org/10.1093/eurheartj/ehw448" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27694191/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Odaka%20Y,%20Takahashi%20J,%20Tsuburaya%20R,%20Nishimiya%20K,%20Hao%20K,%20Matsumoto%20Y,%20et%20al.%20Plasma%20concentration%20of%20serotonin%20is%20a%20novel%20biomarker%20for%20coronary%20microvascular%20dysfunction%20in%20patients%20with%20suspected%20angina%20and%20unobstructive%20coronary%20arteries.%20Eur%20Heart%20J.%202017;38:489%E2%80%9396." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Selim AM, Sarswat N, Kelesidis I, Iqbal M, Chandra R, Zolty R. Plasma serotonin in heart failure: possible marker and potential treatment target. Heart Lung Circ. 2017;26:442â€“9.
</cite> [<a href="https://doi.org/10.1016/j.hlc.2016.08.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27746058/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Selim%20AM,%20Sarswat%20N,%20Kelesidis%20I,%20Iqbal%20M,%20Chandra%20R,%20Zolty%20R.%20Plasma%20serotonin%20in%20heart%20failure:%20possible%20marker%20and%20potential%20treatment%20target.%20Heart%20Lung%20Circ.%202017;26:442%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Xu J, Yang Y. Gut Microbiome and its meta-omics perspectives: profound implications for cardiovascular diseases. Gut Microbes. 2021;13:1936379.
</cite> [<a href="https://doi.org/10.1080/19490976.2021.1936379" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8237965/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34170211/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xu%20J,%20Yang%20Y.%20Gut%20Microbiome%20and%20its%20meta-omics%20perspectives:%20profound%20implications%20for%20cardiovascular%20diseases.%20Gut%20Microbes.%202021;13:1936379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Pan XF, Yang JJ, Shu XO, Moore SC, Palmer ND, Guasch-FerrÃ© M, et al. Associations of Circulating choline and its related metabolites with cardiometabolic biomarkers: an international pooled analysis. Am J Clin Nutr. 2021;114:893â€“906.
</cite> [<a href="https://doi.org/10.1093/ajcn/nqab152" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8408854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34020444/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pan%20XF,%20Yang%20JJ,%20Shu%20XO,%20Moore%20SC,%20Palmer%20ND,%20Guasch-Ferr%C3%A9%20M,%20et%20al.%20Associations%20of%20Circulating%20choline%20and%20its%20related%20metabolites%20with%20cardiometabolic%20biomarkers:%20an%20international%20pooled%20analysis.%20Am%20J%20Clin%20Nutr.%202021;114:893%E2%80%93906." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Jia Q, Han Y, Huang P, Woodward NC, Gukasyan J, Kettunen J, et al. Genetic determinants of Circulating Glycine levels and risk of coronary artery disease. J Am Heart Assoc. 2019;8:e011922.
</cite> [<a href="https://doi.org/10.1161/JAHA.119.011922" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6585317/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31070104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jia%20Q,%20Han%20Y,%20Huang%20P,%20Woodward%20NC,%20Gukasyan%20J,%20Kettunen%20J,%20et%20al.%20Genetic%20determinants%20of%20Circulating%20Glycine%20levels%20and%20risk%20of%20coronary%20artery%20disease.%20J%20Am%20Heart%20Assoc.%202019;8:e011922." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Donato P, Dugo P, Mondello L. Chapter 8 - Separation of lipids. In: Fanali S, Haddad PR, Poole CF, Riekkola M-L, editors. Liquid chromatography. Second Edition): Elsevier; 2017. pp. 201â€“43.</cite> [<a href="https://scholar.google.com/scholar_lookup?Donato%20P,%20Dugo%20P,%20Mondello%20L.%20Chapter%208%20-%20Separation%20of%20lipids.%20In:%20Fanali%20S,%20Haddad%20PR,%20Poole%20CF,%20Riekkola%20M-L,%20editors.%20Liquid%20chromatography.%20Second%20Edition):%20Elsevier;%202017.%20pp.%20201%E2%80%9343." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Tonks KT, Coster AC, Christopher MJ, Chaudhuri R, Xu A, Gagnon-Bartsch J, et al. Skeletal muscle and plasma lipidomic signatures of insulin resistance and overweight/obesity in humans. Obesity. 2016;24:908â€“16.
</cite> [<a href="https://doi.org/10.1002/oby.21448" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6585800/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26916476/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tonks%20KT,%20Coster%20AC,%20Christopher%20MJ,%20Chaudhuri%20R,%20Xu%20A,%20Gagnon-Bartsch%20J,%20et%20al.%20Skeletal%20muscle%20and%20plasma%20lipidomic%20signatures%20of%20insulin%20resistance%20and%20overweight/obesity%20in%20humans.%20Obesity.%202016;24:908%E2%80%9316." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Jia C, Xu H, Xu Y, Xu Y, Shi Q. Serum metabolomics analysis of patients with polycystic ovary syndrome by mass spectrometry. Mol Reprod Dev. 2019;86:292â€“7.
</cite> [<a href="https://doi.org/10.1002/mrd.23104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30624822/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jia%20C,%20Xu%20H,%20Xu%20Y,%20Xu%20Y,%20Shi%20Q.%20Serum%20metabolomics%20analysis%20of%20patients%20with%20polycystic%20ovary%20syndrome%20by%20mass%20spectrometry.%20Mol%20Reprod%20Dev.%202019;86:292%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>JovÃ© M, Pradas I, NaudÃ­ A, Rovira-Llopis S, BaÃ±uls C, Rocha M, et al. Lipidomics reveals altered biosynthetic pathways of glycerophospholipids and cell signaling as biomarkers of the polycystic ovary syndrome. Oncotarget. 2017;9:4522â€“36.
</cite> [<a href="https://doi.org/10.18632/oncotarget.23393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5796992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29435121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jov%C3%A9%20M,%20Pradas%20I,%20Naud%C3%AD%20A,%20Rovira-Llopis%20S,%20Ba%C3%B1uls%20C,%20Rocha%20M,%20et%20al.%20Lipidomics%20reveals%20altered%20biosynthetic%20pathways%20of%20glycerophospholipids%20and%20cell%20signaling%20as%20biomarkers%20of%20the%20polycystic%20ovary%20syndrome.%20Oncotarget.%202017;9:4522%E2%80%9336." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Law S-H, Chan M-L, Marathe GK, Parveen F, Chen C-H, Ke L-Y. An updated review of lysophosphatidylcholine metabolism in human diseases. Int J Mol Sci. 2019;20:1149.
</cite> [<a href="https://doi.org/10.3390/ijms20051149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6429061/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30845751/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Law%20S-H,%20Chan%20M-L,%20Marathe%20GK,%20Parveen%20F,%20Chen%20C-H,%20Ke%20L-Y.%20An%20updated%20review%20of%20lysophosphatidylcholine%20metabolism%20in%20human%20diseases.%20Int%20J%20Mol%20Sci.%202019;20:1149." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Chakaroun RM, Olsson LM, BÃ¤ckhed F. The potential of tailoring the gut Microbiome to prevent and treat cardiometabolic disease. Nat Rev Cardiol. 2023;20:217â€“35.
</cite> [<a href="https://doi.org/10.1038/s41569-022-00771-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36241728/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chakaroun%20RM,%20Olsson%20LM,%20B%C3%A4ckhed%20F.%20The%20potential%20of%20tailoring%20the%20gut%20Microbiome%20to%20prevent%20and%20treat%20cardiometabolic%20disease.%20Nat%20Rev%20Cardiol.%202023;20:217%E2%80%9335." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Denimal D, Nguyen A, Pais de Barros J-P, Bouillet B, Petit J-M, VergÃ¨s B, et al. Major changes in the sphingophospholipidome of HDL in non-diabetic patients with metabolic syndrome. Atherosclerosis. 2016;246:106â€“14.
</cite> [<a href="https://doi.org/10.1016/j.atherosclerosis.2015.12.042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26773402/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Denimal%20D,%20Nguyen%20A,%20Pais%20de%20Barros%20J-P,%20Bouillet%20B,%20Petit%20J-M,%20Verg%C3%A8s%20B,%20et%20al.%20Major%20changes%20in%20the%20sphingophospholipidome%20of%20HDL%20in%20non-diabetic%20patients%20with%20metabolic%20syndrome.%20Atherosclerosis.%202016;246:106%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Paapstel K, Kals J, Eha J, Tootsi K, Ottas A, Piir A, et al. Inverse relations of serum phosphatidylcholines and lysophosphatidylcholines with vascular damage and heart rate in patients with atherosclerosis. Nutr Metab Cardiovasc Dis. 2018;28:44â€“52.
</cite> [<a href="https://doi.org/10.1016/j.numecd.2017.07.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28986077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Paapstel%20K,%20Kals%20J,%20Eha%20J,%20Tootsi%20K,%20Ottas%20A,%20Piir%20A,%20et%20al.%20Inverse%20relations%20of%20serum%20phosphatidylcholines%20and%20lysophosphatidylcholines%20with%20vascular%20damage%20and%20heart%20rate%20in%20patients%20with%20atherosclerosis.%20Nutr%20Metab%20Cardiovasc%20Dis.%202018;28:44%E2%80%9352." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Djekic D, Pinto R, Repsilber D, Hyotylainen T, Henein M. Serum untargeted lipidomic profiling reveals dysfunction of phospholipid metabolism in subclinical coronary artery disease. Vasc Health Risk Manag. 2019;15:123â€“35.
</cite> [<a href="https://doi.org/10.2147/VHRM.S202344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6526169/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31190850/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Djekic%20D,%20Pinto%20R,%20Repsilber%20D,%20Hyotylainen%20T,%20Henein%20M.%20Serum%20untargeted%20lipidomic%20profiling%20reveals%20dysfunction%20of%20phospholipid%20metabolism%20in%20subclinical%20coronary%20artery%20disease.%20Vasc%20Health%20Risk%20Manag.%202019;15:123%E2%80%9335." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Wang YN, Zhang ZH, Liu HJ, Guo ZY, Zou L, Zhang YM, et al. Integrative phosphatidylcholine metabolism through phospholipase A<sub>2</sub> in rats with chronic kidney disease. Acta Pharmacol Sin. 2023;44:393â€“405.
</cite> [<a href="https://doi.org/10.1038/s41401-022-00947-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9889763/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35922553/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20YN,%20Zhang%20ZH,%20Liu%20HJ,%20Guo%20ZY,%20Zou%20L,%20Zhang%20YM,%20et%20al.%20Integrative%20phosphatidylcholine%20metabolism%20through%20phospholipase%20A2%20in%20rats%20with%20chronic%20kidney%20disease.%20Acta%20Pharmacol%20Sin.%202023;44:393%E2%80%93405." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Strand E, Pedersen ER, Svingen GFT, Olsen T, BjÃ¸rndal B, Karlsson T, et al. Serum acylcarnitines and risk of cardiovascular death and acute myocardial infarction in patients with stable angina pectoris. J Am Heart Assoc. 2017;6:e003620.
</cite> [<a href="https://doi.org/10.1161/JAHA.116.003620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5523736/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28159823/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Strand%20E,%20Pedersen%20ER,%20Svingen%20GFT,%20Olsen%20T,%20Bj%C3%B8rndal%20B,%20Karlsson%20T,%20et%20al.%20Serum%20acylcarnitines%20and%20risk%20of%20cardiovascular%20death%20and%20acute%20myocardial%20infarction%20in%20patients%20with%20stable%20angina%20pectoris.%20J%20Am%20Heart%20Assoc.%202017;6:e003620." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>McCann MR, De la Rosa G, Rosania MV, Stringer GR. L-Carnitine and acylcarnitines: mitochondrial biomarkers for precision medicine. Metabolites. 2021;11:51.
</cite> [<a href="https://doi.org/10.3390/metabo11010051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7829830/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33466750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?McCann%20MR,%20De%20la%20Rosa%20G,%20Rosania%20MV,%20Stringer%20GR.%20L-Carnitine%20and%20acylcarnitines:%20mitochondrial%20biomarkers%20for%20precision%20medicine.%20Metabolites.%202021;11:51." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Sun Y, Gao HY, Fan ZY, He Y, Yan YX. Metabolomics signatures in type 2 diabetes: A systematic review and integrative analysis. J Clin Endocrinol Metab. 2020;105:dgz240.
</cite> [<a href="https://doi.org/10.1210/clinem/dgz240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31782507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sun%20Y,%20Gao%20HY,%20Fan%20ZY,%20He%20Y,%20Yan%20YX.%20Metabolomics%20signatures%20in%20type%202%20diabetes:%20A%20systematic%20review%20and%20integrative%20analysis.%20J%20Clin%20Endocrinol%20Metab.%202020;105:dgz240." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Mao Z, Li T, Zhao H, Wang X, Kang Y, Kang Y. Methylome and transcriptome profiling revealed epigenetic Silencing of LPCAT1 and PCYT1A associated with lipidome alterations in polycystic ovary syndrome. J Cell Physiol. 2021;236:6362â€“75.
</cite> [<a href="https://doi.org/10.1002/jcp.30309" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33521992/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mao%20Z,%20Li%20T,%20Zhao%20H,%20Wang%20X,%20Kang%20Y,%20Kang%20Y.%20Methylome%20and%20transcriptome%20profiling%20revealed%20epigenetic%20Silencing%20of%20LPCAT1%20and%20PCYT1A%20associated%20with%20lipidome%20alterations%20in%20polycystic%20ovary%20syndrome.%20J%20Cell%20Physiol.%202021;236:6362%E2%80%9375." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Tanosaki T, Mikami Y, Shindou H, Suzuki T, Hashidate-Yoshida T, Hosoki K, et al. Lysophosphatidylcholine acyltransferase 1 deficiency promotes pulmonary emphysema via apoptosis of alveolar epithelial cells. Inflammation. 2022;45:1765â€“79.
</cite> [<a href="https://doi.org/10.1007/s10753-022-01659-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35338433/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tanosaki%20T,%20Mikami%20Y,%20Shindou%20H,%20Suzuki%20T,%20Hashidate-Yoshida%20T,%20Hosoki%20K,%20et%20al.%20Lysophosphatidylcholine%20acyltransferase%201%20deficiency%20promotes%20pulmonary%20emphysema%20via%20apoptosis%20of%20alveolar%20epithelial%20cells.%20Inflammation.%202022;45:1765%E2%80%9379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Wang B, Tontonoz P. Phospholipid remodeling in physiology and disease. Annu Rev Physiol. 2019;81:165â€“88.
</cite> [<a href="https://doi.org/10.1146/annurev-physiol-020518-114444" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7008953/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30379616/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20B,%20Tontonoz%20P.%20Phospholipid%20remodeling%20in%20physiology%20and%20disease.%20Annu%20Rev%20Physiol.%202019;81:165%E2%80%9388." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Soupene E, Kuypers FA. Phosphatidylcholine formation by LPCAT1 is regulated by Ca(2+) and the redox status of the cell. BMC Biochem. 2012;13:8.
</cite> [<a href="https://doi.org/10.1186/1471-2091-13-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3439698/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22676268/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Soupene%20E,%20Kuypers%20FA.%20Phosphatidylcholine%20formation%20by%20LPCAT1%20is%20regulated%20by%20Ca(2+)%20and%20the%20redox%20status%20of%20the%20cell.%20BMC%20Biochem.%202012;13:8." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Akagi S, Kono N, Ariyama H, Shindou H, Shimizu T, Arai H. Lysophosphatidylcholine acyltransferase 1 protects against cytotoxicity induced by polyunsaturated fatty acids. FASEB J. 2016;30:2027â€“39.
</cite> [<a href="https://doi.org/10.1096/fj.201500149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26887439/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Akagi%20S,%20Kono%20N,%20Ariyama%20H,%20Shindou%20H,%20Shimizu%20T,%20Arai%20H.%20Lysophosphatidylcholine%20acyltransferase%201%20protects%20against%20cytotoxicity%20induced%20by%20polyunsaturated%20fatty%20acids.%20FASEB%20J.%202016;30:2027%E2%80%9339." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Rouillard AD, Gundersen GW, Fernandez NF, Wang Z, Monteiro CD, McDermott MG et al. The harmonizome: a collection of processed datasets gathered to serve and mine knowledge about genes and proteins. Database (Oxford). 2016;2016:baw100.</cite> [<a href="https://doi.org/10.1093/database/baw100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4930834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27374120/" class="usa-link">PubMed</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Vrijsen S, Besora-Casals L, van Veen S, Zielich J, Van den Haute C, Hamouda NN, et al. ATP13A2-mediated endo-lysosomal polyamine export counters mitochondrial oxidative stress. Proc Natl Acad Sci U S A. 2020;117:31198â€“207.
</cite> [<a href="https://doi.org/10.1073/pnas.1922342117" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7733819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33229544/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Vrijsen%20S,%20Besora-Casals%20L,%20van%20Veen%20S,%20Zielich%20J,%20Van%20den%20Haute%20C,%20Hamouda%20NN,%20et%20al.%20ATP13A2-mediated%20endo-lysosomal%20polyamine%20export%20counters%20mitochondrial%20oxidative%20stress.%20Proc%20Natl%20Acad%20Sci%20U%20S%20A.%202020;117:31198%E2%80%93207." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Marcos AL, Corradi GR, Mazzitelli LR, Casali CI, FernÃ¡ndez Tome MC, Adamo HP, et al. The Parkinson-associated human P5B-ATPase ATP13A2 modifies lipid homeostasis. Biochim Biophys Acta Biomembr. 2019;1861:182993.
</cite> [<a href="https://doi.org/10.1016/j.bbamem.2019.05.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31132336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Marcos%20AL,%20Corradi%20GR,%20Mazzitelli%20LR,%20Casali%20CI,%20Fern%C3%A1ndez%20Tome%20MC,%20Adamo%20HP,%20et%20al.%20The%20Parkinson-associated%20human%20P5B-ATPase%20ATP13A2%20modifies%20lipid%20homeostasis.%20Biochim%20Biophys%20Acta%20Biomembr.%202019;1861:182993." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Munz M, Richter GM, Loos BG, Jepsen S, Divaris K, Offenbacher S, et al. Genome-wide association meta-analysis of coronary artery disease and periodontitis reveals a novel shared risk locus. Sci Rep. 2018;8:13678.
</cite> [<a href="https://doi.org/10.1038/s41598-018-31980-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6135769/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30209331/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Munz%20M,%20Richter%20GM,%20Loos%20BG,%20Jepsen%20S,%20Divaris%20K,%20Offenbacher%20S,%20et%20al.%20Genome-wide%20association%20meta-analysis%20of%20coronary%20artery%20disease%20and%20periodontitis%20reveals%20a%20novel%20shared%20risk%20locus.%20Sci%20Rep.%202018;8:13678." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Huang C, Li SX, Caraballo C, Masoudi FA, Rumsfeld JS, Spertus JA, et al. Performance metrics for the comparative analysis of clinical risk prediction models employing machine learning. Circ Cardiovasc Qual Outcomes. 2021;14:e007526.
</cite> [<a href="https://doi.org/10.1161/CIRCOUTCOMES.120.007526" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34601947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20C,%20Li%20SX,%20Caraballo%20C,%20Masoudi%20FA,%20Rumsfeld%20JS,%20Spertus%20JA,%20et%20al.%20Performance%20metrics%20for%20the%20comparative%20analysis%20of%20clinical%20risk%20prediction%20models%20employing%20machine%20learning.%20Circ%20Cardiovasc%20Qual%20Outcomes.%202021;14:e007526." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366056/bin/40364_2025_821_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (4.1MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The SNP datasets generated and analyzed in this study are available in the Gene Expression Omnibus (GEO) repository; accession code <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE303414" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE303414</a>. In addition, both the SNP and metabolomics datasets can be downloaded directly from NHRI website: <a href="http://bioinfolab.nhri.edu.tw/pub/bio2025.html" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://bioinfolab.nhri.edu.tw/pub/bio2025.html</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Biomarker Research are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s40364-025-00821-y"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/40364_2025_Article_821.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (7.9Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366056/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366056/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366056%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366056/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366056/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366056/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830908/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366056/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830908/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366056/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366056/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="IG6FNdPHzjDPAhrWHkBX5EW2RJqooLE6L7s7LjvHYCG4aDTTVLEEE2FEuLaN1HY0">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
