
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Phenotypic characterization and complete genome of a tumorigenic pathobiont Escherichia coli LI60C3 - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4B0918AF33AB305B09100049581C3.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="gutpath">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366417/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Gut Pathogens">
<meta name="citation_title" content="Phenotypic characterization and complete genome of a tumorigenic pathobiont Escherichia coli LI60C3">
<meta name="citation_author" content="Linda Chia-Hui Yu">
<meta name="citation_author_institution" content="Graduate Institute of Physiology, National Taiwan University College of Medicine, Taipei, Taiwan R.O.C.">
<meta name="citation_author" content="Shu-Chen Wei">
<meta name="citation_author_institution" content="Department of Internal Medicine, National Taiwan University Hospital, Taipei, Taiwan R.O.C.">
<meta name="citation_author" content="Yi-Hsuan Li">
<meta name="citation_author_institution" content="Graduate Institute of Physiology, National Taiwan University College of Medicine, Taipei, Taiwan R.O.C.">
<meta name="citation_author" content="Chung-Yen Huang">
<meta name="citation_author_institution" content="Department of Pathology, National Taiwan University Hospital, Taipei, Taiwan R.O.C.">
<meta name="citation_author" content="Yu-Chen Pai">
<meta name="citation_author_institution" content="Graduate Institute of Physiology, National Taiwan University College of Medicine, Taipei, Taiwan R.O.C.">
<meta name="citation_author" content="Yuan-Mao Hung">
<meta name="citation_author_institution" content="Bioinformatics and Biostatistics Core, Center of Genomic and Precision Medicine, National Taiwan University, Taipei, Taiwan R.O.C.">
<meta name="citation_author" content="Liang-Chuan Lai">
<meta name="citation_author_institution" content="Graduate Institute of Physiology, National Taiwan University College of Medicine, Taipei, Taiwan R.O.C.">
<meta name="citation_author_institution" content="Bioinformatics and Biostatistics Core, Center of Genomic and Precision Medicine, National Taiwan University, Taipei, Taiwan R.O.C.">
<meta name="citation_author_institution" content="Graduate Institute of Biomedical Electronics and Bioinformatics, National Taiwan University, Taipei, Taiwan R.O.C.">
<meta name="citation_author" content="Yen-Hsuan Ni">
<meta name="citation_author_institution" content="Department of Pediatrics, National Taiwan University Hospital, Taipei, Taiwan R.O.C.">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="17">
<meta name="citation_firstpage" content="63">
<meta name="citation_doi" content="10.1186/s13099-025-00732-1">
<meta name="citation_pmid" content="40836244">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366417/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366417/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366417/pdf/13099_2025_Article_732.pdf">
<meta name="description" content="Symbiotic microbes benefit the host, but the emergence of pathobionts leads to disease. An invasive Escherichia coli LI60C3, isolated from mouse colonocytes, shows colitogenic and tumorigenic properties. Despite extensive research on the role of ...">
<meta name="og:title" content="Phenotypic characterization and complete genome of a tumorigenic pathobiont Escherichia coli LI60C3">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Symbiotic microbes benefit the host, but the emergence of pathobionts leads to disease. An invasive Escherichia coli LI60C3, isolated from mouse colonocytes, shows colitogenic and tumorigenic properties. Despite extensive research on the role of ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366417/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366417">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s13099-025-00732-1"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/13099_2025_Article_732.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366417%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366417/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366417/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366417/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-gutpath.png" alt="Gut Pathogens logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Gut Pathogens" title="Link to Gut Pathogens" shape="default" href="https://gutpathogens.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Gut Pathog</button></div>. 2025 Aug 20;17:63. doi: <a href="https://doi.org/10.1186/s13099-025-00732-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s13099-025-00732-1</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Gut%20Pathog%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gut%20Pathog%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Gut%20Pathog%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Gut%20Pathog%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Phenotypic characterization and complete genome of a tumorigenic pathobiont <em>Escherichia coli</em> LI60C3</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20LCH%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Linda Chia-Hui Yu</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Linda Chia-Hui Yu</span></h3>
<div class="p">
<sup>1</sup>Graduate Institute of Physiology, National Taiwan University College of Medicine, Taipei, Taiwan R.O.C. </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20LCH%22%5BAuthor%5D" class="usa-link"><span class="name western">Linda Chia-Hui Yu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wei%20SC%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Shu-Chen Wei</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Shu-Chen Wei</span></h3>
<div class="p">
<sup>2</sup>Department of Internal Medicine, National Taiwan University Hospital, Taipei, Taiwan R.O.C. </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wei%20SC%22%5BAuthor%5D" class="usa-link"><span class="name western">Shu-Chen Wei</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20YH%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Yi-Hsuan Li</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Yi-Hsuan Li</span></h3>
<div class="p">
<sup>1</sup>Graduate Institute of Physiology, National Taiwan University College of Medicine, Taipei, Taiwan R.O.C. </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20YH%22%5BAuthor%5D" class="usa-link"><span class="name western">Yi-Hsuan Li</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20CY%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Chung-Yen Huang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Chung-Yen Huang</span></h3>
<div class="p">
<sup>3</sup>Department of Pathology, National Taiwan University Hospital, Taipei, Taiwan R.O.C. </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Huang%20CY%22%5BAuthor%5D" class="usa-link"><span class="name western">Chung-Yen Huang</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pai%20YC%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Yu-Chen Pai</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Yu-Chen Pai</span></h3>
<div class="p">
<sup>1</sup>Graduate Institute of Physiology, National Taiwan University College of Medicine, Taipei, Taiwan R.O.C. </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Pai%20YC%22%5BAuthor%5D" class="usa-link"><span class="name western">Yu-Chen Pai</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hung%20YM%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Yuan-Mao Hung</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Yuan-Mao Hung</span></h3>
<div class="p">
<sup>4</sup>Bioinformatics and Biostatistics Core, Center of Genomic and Precision Medicine, National Taiwan University, Taipei, Taiwan R.O.C. </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hung%20YM%22%5BAuthor%5D" class="usa-link"><span class="name western">Yuan-Mao Hung</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lai%20LC%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Liang-Chuan Lai</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Liang-Chuan Lai</span></h3>
<div class="p">
<sup>1</sup>Graduate Institute of Physiology, National Taiwan University College of Medicine, Taipei, Taiwan R.O.C. </div>
<div class="p">
<sup>4</sup>Bioinformatics and Biostatistics Core, Center of Genomic and Precision Medicine, National Taiwan University, Taipei, Taiwan R.O.C. </div>
<div class="p">
<sup>5</sup>Graduate Institute of Biomedical Electronics and Bioinformatics, National Taiwan University, Taipei, Taiwan R.O.C. </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lai%20LC%22%5BAuthor%5D" class="usa-link"><span class="name western">Liang-Chuan Lai</span></a>
</div>
</div>
<sup>1,</sup><sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ni%20YH%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Yen-Hsuan Ni</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Yen-Hsuan Ni</span></h3>
<div class="p">
<sup>6</sup>Department of Pediatrics, National Taiwan University Hospital, Taipei, Taiwan R.O.C. </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ni%20YH%22%5BAuthor%5D" class="usa-link"><span class="name western">Yen-Hsuan Ni</span></a>
</div>
</div>
<sup>6,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Graduate Institute of Physiology, National Taiwan University College of Medicine, Taipei, Taiwan R.O.C. </div>
<div id="Aff2">
<sup>2</sup>Department of Internal Medicine, National Taiwan University Hospital, Taipei, Taiwan R.O.C. </div>
<div id="Aff3">
<sup>3</sup>Department of Pathology, National Taiwan University Hospital, Taipei, Taiwan R.O.C. </div>
<div id="Aff4">
<sup>4</sup>Bioinformatics and Biostatistics Core, Center of Genomic and Precision Medicine, National Taiwan University, Taipei, Taiwan R.O.C. </div>
<div id="Aff5">
<sup>5</sup>Graduate Institute of Biomedical Electronics and Bioinformatics, National Taiwan University, Taipei, Taiwan R.O.C. </div>
<div id="Aff6">
<sup>6</sup>Department of Pediatrics, National Taiwan University Hospital, Taipei, Taiwan R.O.C. </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 May 26; Accepted 2025 Jul 22; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366417  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40836244/" class="usa-link">40836244</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Symbiotic microbes benefit the host, but the emergence of pathobionts leads to disease. An invasive <em>Escherichia coli</em> LI60C3, isolated from mouse colonocytes, shows colitogenic and tumorigenic properties. Despite extensive research on the role of microbiota in colorectal cancer (CRC) development, the genetic markers associated with this pathobiont remain elusive. The objective is to characterize the tumorigenic <em>E. coli</em> through whole-genome sequencing (WGS) and phenotypic assays, and validate their presence in human CRC.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p id="Par2">The intracellular bacterial counts and proliferation rates of human intestinal epithelial cells were evaluated after exposure to various <em>E. coli</em> strains. Tumor burden was assessed in mice orally administered LI60C3. WGS of LI60C3 was performed on a PacBio Sequel II platform, and the long reads were assembled <em>de novo</em> for gene annotation and detection of virulence factors and antibiotic resistance. Bacteria-specific genes were assessed in CRC specimens by qPCR analysis.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p id="Par3">A 100-fold increase in intracellular bacterial count was observed in epithelial cells exposed to LI60C3 compared to commensal <em>E. coli</em> strains. LI60C3 resulted in a threefold increase in epithelial cell cycle rate and a fourfold rise in mouse tumor numbers. WGS revealed a circular chromosome of 4,863,930 bases for LI60C3, demonstrating a high sequence homology to adherent-invasive <em>E. coli</em> LF82 (91%) and NC101 (87%) and to uropathogenic <em>E. coli</em> 536 (88%). Two extrachromosomal plasmids, pTra and pCoMb, were identified. While pTra exhibits sequence homology with other commensal <em>E. coli</em> plasmids, pCoMb has partial matches with those found in pathogenic bacteria. LI60C3 is classified as phylogroup B2 and expresses virulence factors, including Type 1 and P fimbriae, contact-dependent growth inhibition system, iron acquisition system, and hemolysin. Unique gene clusters, named Epm and Phz islands, were identified in the LI60C3 genome. The emergence of LI60C3-specific genes was observed in mouse tumors induced by chemicals and gene mutation, and higher levels of LI60C3 markers were validated in human CRC specimens compared with healthy mucosal samples.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par4">Genetic signatures of LI60C3 were detected in mouse and human CRC. The comparative genome analysis for LI60C3 helps identify pathobionts and may be used as cancer predictors.</p></section><section id="sec5"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s13099-025-00732-1.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Colorectal cancer microbiome, Tumorigenic <em>E. coli</em>, Invasive pathobionts, Comparative genomics, Complete genome sequencing, Intestinal epithelial cells, Phylogenetic tree, Experimental pathobiont models</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par5">Microbiota dysbiosis is associated with developing inflammatory bowel diseases (IBDs) and colorectal cancers (CRC) [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. While symbiotic commensals inhabit the gut lumen in healthy humans, elevated levels of mucosal bacteria were detected in clinical specimens from patients [<a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>–<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. The presence of intraepithelial bacteria in gut tissues was identified in mouse models of colitis and cancers, suggesting microbial etiology of the diseases [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>–<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>]. Meta-analysis and machine-learning algorithms revealed specific microbial markers and distinct microbiome and metabolome profiles associated with colon polyposis and CRC development [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. Growing evidence indicates that pathobionts (commensal-derived opportunistic pathogens) are involved in the onset or perpetuation of diseases, and mucosal bacteria may play a more significant role than fecal bacteria in the pathogenesis associated with microbiota.</p>
<p id="Par6"><em>Escherichia coli</em> is a facultative anaerobe that colonizes the intestine of humans and animals, and constitutes a minor population of the fecal microbiota in a healthy state of gut anaerobiosis. Nevertheless, a high abundance of <em>E. coli</em> was found in the gut tissues of patients with IBDs, colorectal adenoma and carcinoma, and familial adenomatous polyposis (FAP) [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>], suggesting that mucosa-associated bacteria were involved in inflammation and tumorigenesis. We previously demonstrated that an invasive <em>E. coli</em> strain isolated from intestinal epithelial cells, designated LI60C3, increased colitis susceptibility and promoted cancer initiation in mouse models [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Direct actions of LI60C3 on epithelial cell functions were shown in primary colonoid cultures and human Caco-2 cell lines [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. These studies revealed that LI60C3 accelerated cell cycle progression, enhanced spheroid growth, increased cytokine synthesis, and disturbed the circadian rhythm of epithelial cells in vitro [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. These findings underscore the tumorigenic potential of LI60C3, highlighting its capacity to influence epithelial cell behavior independently of immune cell activation. These unique features of LI60C3 prompted us to conduct further genomic characterization of this tumorigenic pathobiont. Other laboratories found adherent-invasive <em>E. coli</em> (AIEC) and diffusely adherent <em>E. coli</em> (DAEC) linked to inflammatory and diarrheal diseases, with colitogenic effects shown in experimental models through orogavage administration [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>–<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>]. Additionally, colibactin-producing <em>E. coli</em> have been identified for their roles in causing DNA damage and cell senescence in epithelial cells in vitro, as well as promoting severe colitis and invasive tumor growth in chemically induced and genetically modified mice [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>–<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>]. Although AIECs and DAECs were initially isolated from patients, they also exist in a quarter of healthy individuals [<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>–<a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. Currently, the virulence characteristics of intestinal pathobionts are mainly assessed through phenotypic assays. The genetic differences between commensals and pathobionts are still not well understood.</p>
<p id="Par7">In contrast to commensal and pathobiont <em>E. coli</em> that persistently colonize the gut, pathogenic <em>E. coli</em> strains are accountable for acute infections in the enteric, urinary, and respiratory tracts, and the nervous system. The <em>E</em>. <em>coli</em> pathotypes causing foodborne diarrheal diseases consist of enteropathogenic <em>E. coli</em> (EPEC), enterohaemorrhagic <em>E. coli</em> (EHEC) (a subset of Shiga toxin-producing <em>E. coli</em>), heat-stable and -labile toxin-producing enterotoxigenic <em>E. coli</em> (ETEC), enteroinvasive <em>E. coli</em> (EIEC), and enteroaggregative <em>E. coli</em> (EAEC) [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>]. Other <em>E. coli</em> pathotypes can cause extra-intestinal diseases, including uropathogenic <em>E. coli</em> (UPEC), which is responsible for cystitis, pyelonephritis, septicemia, and meningitis [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. Pathogenic strains of <em>E. coli</em> possess a variety of virulence factors, which include adherence and invasion determinants, enterotoxins, and multiple secretion systems. These systems facilitate the injection of bacterial components into eukaryotic cells, leading to the manifestation of symptoms in the host.</p>
<p id="Par8">Despite the pathogenic traits of LI60C3, the genetic markers and virulence factors to distinguish the bacteria from commensal <em>E. coli</em> strains remain unresolved. Virulence factors are encoded by a cluster of genes in a pathogenicity island localized in the chromosome or extrachromosomal plasmids. Bacteria use plasmids for horizontal gene transfer to enhance virulence and acquire toxin production and invasive capabilities. Increased expression of fimbrial adhesion (fimbriae/pili) and genes related to stress survival responses (high temperature requirement (htr) A) were detected in LI60C3 following repeated internalization in epithelial cells, which correlated with their higher levels in mouse fecal samples during the tumor transition phase from dysplasia to carcinoma [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>]. Nevertheless, fimbriae and htrA proteases are not specific to the LI60C3 strain but exist in many <em>Enterobacteriaceae</em> genera. In this regard, virulence factors and genetic signatures of the tumorigenic LI60C3 await revelation. Investigating specific markers of the tumorigenic LI60C3 through the integration of genomic, phenotypic, and in vivo approaches may aid in clinical treatment and facilitate non-invasive CRC screening.</p>
<p id="Par9">This study examines the phenotypic characteristics of LI60C3 using bacteria-epithelial cocultures and mouse tumor models. Whole genome sequencing (WGS) was performed to decipher the genetic signatures of LI60C3. Here, we report the <em>de novo</em> assembly of a circular chromosome and two extrachromosomal plasmids for the LI60C3 strain. The pathobiont-specific biomarkers identified through genome comparison between LI60C3 and reference <em>E. coli</em> strains were evaluated in mouse tumor models and human CRC specimens.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Animal study</h3>
<p id="Par10">Experimental procedures in mouse models were approved by the IACUC Committee of NTUCM (IACUC#20210023, IACUC#20230363). Specific pathogen-free BALB/c mice and Apc(Min/+) mice were purchased from the National Institutes of Applied Research and kept in temperature-controlled rooms (23 ± 2 °C) with 12 h light-dark cycles. Mice were fed standard chow and water <em>ad libitum</em>. At the end of the experiments, mice were injected with pentobarbital (200 mg/kg) overdose for laparotomy and inspection of intestinal tumors. Regular monitoring of animals for signs of distress and significant weight loss with welfare-based criteria (e.g., lethargy, inability to reach food and water), and timely intervention are applied as humane endpoints.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Bacterial strains</h3>
<p id="Par11">The bacterial colonies were stored in 50% glycerol at -80 °C until use, and only the first three generations after Luria-Bertani (LB) broth culturing were used as stocks. Glycerol stocks of <em>E. coli</em> strains were inoculated into LB broth for culturing at 37 °C with constant shaking overnight. The bacterial culture was plated on LB agar to grow overnight, and a single colony was then inoculated into LB broth for culturing at 37 °C to mid-exponential phase (OD<sub>600</sub> at 0.6–0.8). The <em>E. coli</em> LI60C3 strain was isolated from colonic epithelial cells (colonocytes) of a mouse colitis-associated cancer model [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. Other bacteria included the laboratory K-12 strains of <em>E. coli</em> (i.e., MG1655, DH5α, and JM109) or strains isolated from mouse colonocytes in colitis and cancer models (e.g., HM784C1, HM926C2, and HM936C4) [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. The Crohn’s-associated AIEC strain LF82 [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>] was from Dr. Shu-Chen Wei, Department of Internal Medicine of NTUCM. The mouse AIEC strain NC101 [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>, <a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>] was a kind gift from Dr. Christian Jobin, Department of Medicine, University of Florida College of Medicine.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Chemically induced mouse tumor model</h3>
<p id="Par12">Mouse models of colitis-associated CRC were prepared according to previously established methods [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. Briefly, BALB/c mice (6–8 weeks old) were randomly divided into two experimental groups (<em>N</em> = 8–10/group). One group was injected intraperitoneally with azoxymethane (AOM) (10 mg/kg body weight) and after 7 days, given 2% dextran sodium sulfate (DSS) in drinking water for 4 days, followed by 3 days of regular water (AOM/DSS). The other group was age-matched mice that were not administered agents and used as untreated controls. Three cycles of AOM/DSS were administered for tumor induction. Our previous studies have shown high numbers of colon tumors in the AOM/DSS-induced mice 56–90 days after the first AOM injection [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>].</p>
<p id="Par13">In another setting, BALB/c mice (6–8 weeks old) were randomly divided into three experimental groups (<em>N</em> = 8–10/group). The mice subjected to AOM/DSS cycles were administered ABX mixtures (vancomycin 500 mg/L, neomycin 1 g/L, metronidazole 1 g/L, and ampicillin 1 g/L) in drinking water from day 56–63 to disturb gut microbiota, followed by orogastric administration of either phosphate-buffered saline (PBS), LI60C3, or LF82 strain at 10<sup>9</sup> CFU per mouse on day 64. At the end of the experiments, the entire intestine was examined macroscopically under a dissecting microscope to quantify the number and size of tumors on day 90. Tumor area was assessed using imaging software (AxioVision Imaging System, Zeiss). Paraffin-embedded tissues were used for histological assessment and tumor grading by a pathologist blinded to the treatment groups (see below).</p></section><section id="Sec6"><h3 class="pmc_sec_title">Genetic mutant Apc(Min/+) mouse model</h3>
<p id="Par14">The Apc(Min/+) mice is a well-established animal model bearing multiple intestinal neoplasia (Min), that simulates human FAP and CRC [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. The adenomatous polyposis coli (APC) gene is a key tumor suppressor gene frequently mutated in the early stages of human CRC. Mice with a single allele of the <em>Apc</em> gene mutation were generated by in vitro fertilization (IVF) of C57BL/6 oocytes with Apc(Min/+) sperm in the National Institutes of Applied Research in Academia Sinica, Taipei. Our pilot studies by intercrossing of Apc(Min/+) males with C57BL/6 females showed high mortality of the offspring, and the breeding strategy of heterozygotic pairs was also unfeasible due to infertility of the female Apc(Min/+) mice. The IVF embryos were transferred to surrogate CByB6F1 dams (a cross-strain of C57BL/6 and BALB/c mice), and the offspring of Apc(Min/+) and wild-type (WT(+/+)) littermates were born in a 1:1 ratio. The WT(+/+) and Apc(Min/+) mice were confirmed by genotyping (<em>N</em> = 10/group), and tumor development was observed by macroscopic inspection at 20 weeks of age. Paraffin-embedded tissues were used for histological assessment and tumor grading by a pathologist blinded to the treatment groups (see below).</p></section><section id="Sec7"><h3 class="pmc_sec_title">Tumor assessment criteria</h3>
<p id="Par15">The intestinal tissues were fixed in 4% paraformaldehyde and processed for paraffin embedding. The sections stained with hematoxylin and eosin dyes (H&amp;E) were used for histopathological examination and tumor grading by a blinded pathologist [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. Tumor morphology was classified as low-grade dysplasia, high-grade dysplasia, and carcinoma. Low-grade dysplasia is characterized by disorganized crypt structures and monolayered, hyperchromatic epithelial cells. High-grade dysplasia is characterized by disoriented crypts lined with stratified epithelial cells, where the glands appear back-to-back with limited seperation, and the cells exhibiting high nuclear to cytoplasmic ratio and loss of cellular polarity. Carcinoma is defined as the loss of crypt architecture and the penetration of the dysplastic cells through the basement membrane.</p>
<section id="Sec8"><h4 class="pmc_sec_title">Human cell lines</h4>
<p id="Par16">Human colorectal adenocarcinoma Caco-2 cell lines, which exhibit monolayer features, were employed for the bacteria-epithelial cocultures. The cells were purchased from the ATCC/Bioresource Collection and Research Center (BCRC), and an STR-PCR profile was performed by BCRC. The Caco-2 cells were grown in Dulbecco’s Modified Eagle’s Medium (DMEM) (Cat. #11885-076, Gibco brand, ThermoFisher Scientific, Waltham, MA, USA). The medium was supplemented with 10% fetal bovine serum (FBS) (Cat. #26140-079, Gibco), 100 U/ml penicillin and 0.1 mg/ml streptomycin (Cat. #P4333, Sigma-Aldrich, St. Louis, MO, USA), and 15 mM HEPES (Cat. #26140-079, Gibco). The Caco-2 (C2BBe clone) was also supplemented with human holo-transferrin (11 mg/l) (Cat. #T0665, Sigma). The cells were grown to confluency at 37 °C with 5% CO<sub>2</sub> in a humidified incubator for experiments [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>, <a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>].</p></section><section id="Sec9"><h4 class="pmc_sec_title">Bacterial internalization in epithelial cell cultures</h4>
<p id="Par17">The quantity of bacteria within epithelial cells was measured as described [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>, <a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. Briefly, Caco-2 cells were seeded in 12-well plates at a concentration of 3 × 10<sup>4</sup> cells/ml and cultured for 7 days to reach confluency (4 × 10<sup>5</sup> cells per well). After removal of the antibiotic-containing culture media, the cells were rinsed with sterile PBS twice and incubated in antibiotic-free culture media for 1 h. The Caco-2 cells were then exposed to bacteria in antibiotic-free culture media at multiplicity-of-infection (MOI) = 10 and incubated for 4 h at 37 °C. For the groups without bacteria, PBS was added as a vehicle control. The bacterial numbers in the apical solution and inside the cells were determined by plating apical culture media or cell lysates on LB agar plates for CFU counting. To obtain intracellular bacterial counts, the monolayer was rinsed with sterile PBS twice and incubated with PBS containing 0.25% Trypsin/EDTA (Cat. #25200, ThermoFisher Scientific) for 10 min at 37 °C. The epithelial cells were centrifuged at 300 × <em>g</em> for 3 min, washed with PBS, and centrifuged again. The cells were then resuspended in a PBS solution containing 300 µg/ml gentamicin (Cat. #VGE1AC04, Taiwan Biotech Co., Taoyuan, Taiwan) for incubation at 37 °C for 1 h. After cell lysis with 200 µl of 1% Triton X-100 (Cat. #X100, Sigma-Aldrich) in PBS for 10 min on ice, the lysate was plated on LB agar plates at 37 °C overnight. The number of viable intracellular bacteria was expressed as log<sub>10</sub>CFU/10<sup>6</sup> cells.</p></section><section id="Sec10"><h4 class="pmc_sec_title">Flow cytometric analysis of cell cycle rates</h4>
<p id="Par18">Human cell lines exposed to bacteria for 4 h were rinsed with PBS twice and incubated in culture medium containing 300 µg/ml of gentamicin for 1 h. After this initial incubation, the medium was replaced with fresh culture medium containing 300 µg/ml of gentamicin, and the cells were allowed to incubate for up to 24 h to evaluate cell proliferation, following established protocols [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. The monolayered cells was rinsed with sterile PBS twice, and incubated with PBS containing 0.25% Trypsin/EDTA for 10 min at 37 °C to obtain single cells. After centrifugation at 300 × <em>g</em> for 3 min and supernatant removed, the cells were washed with PBS and centrifuged again. The cells were then fixed with ice-cold 70% ethanol overnight, followed by washing with PBS containing 1% bovine serum albumin (BSA) (Cat. #A2153, Sigma-Aldrich) and centrifuged at 1500 × <em>g</em> for 5 min. After decanting the supernatant, the cells were stained with a primary antibody rabbit anti-human Ki67 (Cat. #9129) (1:400, Cell Signaling, Beverly, MA, USA) in PBS containing 0.5% TWEEN-20 (Cat. #TWN508, BioShop, Burlington, ON, Canada) and 1% BSA for 1 h at room temperature. The cells were rinsed with 1% BSA in PBS, and then incubated with a secondary antibody, goat anti-rabbit IgG conjugated to Alexa-Fluor 488 (Cat. #A11034) (1:500, Life Technologies Inc., Gaithersburg, MD, USA), in PBS containing 0.5% TWEEN-20 and 1% BSA for 30 min at room temperature. After washing with 1% BSA in PBS, the cells were stained with a propidium iodide (PI) solution in PBS containing 20 µg/ml PI (Cat. #P4170, Sigma-Aldrich) and 200 µg/ml RNase A (Cat. #RNA888, BioShop) for 30 min at room temperature in a light-protected container. The cells were washed again with 1% BSA in PBS, and collected after centrifugation. A minimum of 10,000 PI-stained nuclei were analyzed by flow cytometry (FACS Calibur, BD Life Sciences, Franklin Lakes, NJ, USA), and the percentages of cells in the G0–G1, S, and G2–M phases of the cell cycle and the ratio of cells with high to low intensity of Ki67 were determined using Mod Fit LT cell cycle analysis software (Verity Software, Topsham, ME, USA) and FlowJo software (BD Life Sciences).</p></section><section id="Sec11"><h4 class="pmc_sec_title">Whole genome sequencing, data post-processing, and de Novo assembly</h4>
<p id="Par19">The complete genome of <em>E. coli</em> strain LI60C3 was obtained by third-generation sequencing using a Pacific Biosciences (PacBio) Sequel II platform (Taiwan Genomic Industry Alliances Inc., Taipei, Taiwan). Bacterial DNA was extracted using a Qiagen Genomic-tip 20/G kit, and the purity of nucleic acid preparation was determined by Qubit<sup>®</sup> fluorometer (ThermoFisher Scientific) with High Sensitivity dsDNA assay kits. Each sample was quantified in quadruplicate. Pure DNA was designated with an A260/280 absorbance ratio of 1.8, and an A260/230 ratio of 2.0 or higher to pass the quality control.</p>
<p id="Par20">Continuous long-read (CLR) datasets generated by the PacBio platform were used for <em>de novo</em> assembly of the whole genome with SMRT Link Software (v. 10.1.0.119588). The parameters of predicted accuracy (Phred Scale) are set to 20, genome length to 5 M, maximum plasmid size to 300,000, and downsampled coverage to 100. The depth of coverage plots and the GC coverage plots were produced using the built-in software (corrected error rate = 0.045, cor out coverage = 40). The obtained draft assembly was polished by removing redundant sequences and correcting raw reads using overlaps with other reads. The core polishing uses the arrow algorithm, a left-right Hidden Markov-Model (HMM) that models the enzymatic and photophysical events in SMRT Sequencing. Polished contigs with the origin of replication (ori) for rotation and header adjustment were applied before the sequence submission to a public database. The contigs generated by the long-read sequences, including a circular chromosome and two plasmids, in the LI60C3 genome were deposited in GenBank under the accession numbers CP136900, CP136901, and CP136902.</p></section><section id="Sec12"><h4 class="pmc_sec_title">Genome annotation and comparative genomic analysis</h4>
<p id="Par21">The bacterial sequences were annotated using a gene prediction tool Prokka (v1.14.6). The genome sequences were compared to other <em>E. coli</em> strains published in the context of the ColiScope project [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>] by using the basic local alignment search tool (BLAST) in NCBI (<a href="http://www.ncbi.nlm.nih.gov/BLAST/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.ncbi.nlm.nih.gov/BLAST/</a>). The mapping rates were calculated by using the quality assessment tool for genome assemblies (QUAST) (<a href="https://quast.sourceforge.net/quast" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://quast.sourceforge.net/quast</a>) [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. Moreover, the conserved gene clusters, i.e., synteny groups, were computed and plotted using XMatchView (<a href="https://github.com/bcgsc/xmatchview" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/bcgsc/xmatchview</a>). The circular map of the bacterial genome was drawn using pyCircos (<a href="https://github.com/ponnhide/pyCircos" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/ponnhide/pyCircos</a>).</p></section><section id="Sec13"><h4 class="pmc_sec_title">Virulence and resistance genes prediction</h4>
<p id="Par22">Virulence factor-encoding genes were predicted based on the curated virulence factor database (VFDB) v. 2022 (<a href="http://www.mgc.ac.cn/VFs/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://www.mgc.ac.cn/VFs/</a>) [<a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>, <a href="#CR54" class="usa-link" aria-describedby="CR54">54</a>]. Based on the complete genome FASTA sequence, the coding region was predicted using a gene prediction tool GLIMMER3 applied in the VFanalyzer. The official tool provided by the VFDB developers, VFanalyzer, predicts virulence genes through whole-genome ortholog grouping, hierarchical and iterative homologous searches, and context-based data refinement. Analyses were performed using default settings, including internally defined thresholds for gene identity and coverage.</p>
<p id="Par23">Resistance genes were predicted by the Resistance Gene Identifier (RGI) (v.6.0.3) which performed homolog detection using DIAMOND in the Comprehensive Antibiotic Resistance Database (CARD) (v.3.3.0) (<a href="https://arpcard.mcmaster.ca" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://arpcard.mcmaster.ca</a>) [<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>]. We set the select criteria in RGI to be perfect and strict hits only, corresponding to a sequence identity of ≥ 95% and coverage of ≥ 80%.</p></section><section id="Sec14"><h4 class="pmc_sec_title">Phylogenetic analysis of E. coli strains</h4>
<p id="Par24">The phylogenetic analysis was based on the sequences of seven housekeeping genes (<em>arcA</em>,<em> aroE</em>,<em> icd</em>,<em> mdh</em>,<em> mtlD</em>,<em> pgi</em>, and <em>rpoS</em>) of <em>E. coli</em> reference strains to compare with the new isolates. The sequences of the seven housekeeping genes of each <em>E. coli</em> strain were subjected to MUltiple Sequence Comparison by Log- Expectation (MUSCLE) analysis. The multiple sequence alignment (MSA) methods refer to a series of algorithmic solutions for the alignment of evolutionarily related sequences, while taking into account gene mutations, insertions, deletions, and rearrangements under certain conditions. Gblocks (<a href="https://bio.tools/gblocks" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bio.tools/gblocks</a>) for trimming the MSA was performed, and the genetic tree was drawn using Molecular Evolutionary Genetics Analysis version 11 (MEGA11) software (<a href="https://www.megasoftware.net/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.megasoftware.net/</a>) [<a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>]. A neighbor-joining method for metric clustering was used to determine the length of the branches between two taxa. The evolutionary distances were computed in the units of the number of amino acid substitutions per site, and are defined as the fraction of mismatches at aligned positions, with gaps either ignored or counted as mismatches.</p></section><section id="Sec15"><h4 class="pmc_sec_title">Sanger sequencing</h4>
<p id="Par25">The bacterial DNA was extracted from a single colony on LB agar plates. The plasmid DNA was purified by using a Presto<sup>®</sup> Mini Plasmid kit, whereas the bacterial chromosome DNA, which may also contain plasmids, was extracted by using a VIOGENE Geno Plus Mini kit. PCR amplification was performed by the First Core Laboratory of NTUCM for Sanger sequencing. PCR conditions were as follows: 95◦C for 3 min, followed by 35 cycles of 95 °C for 30 s, 58 °C for 30 s, and 72 °C for 2 min, and a final elongation step at 72 °C for 5 min. The annealing temperatures were different depending on the target genes.</p></section><section id="Sec16"><h4 class="pmc_sec_title">Semi-quantitative, quantitative, and triplex polymerase chain reaction (PCR) methods</h4>
<p id="Par26">Genomic DNA extracted from <em>E. coli</em> was subjected to PCR amplification. The <em>E. coli</em> phylotyping was performed using Clermont’s triplex PCR analysis by semi-quantitative methods [<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a>]. The semi-quantitative PCR primers for bacterial genes are listed (Suppl Table <a href="#MOESM1" class="usa-link">1</a>). The PCR products containing Nucleic Acid Stains (Thermo Scientific) were separated on a 1% agarose gel, and visualized by ultraviolet transillumination.</p>
<p id="Par27">Quantitative real-time PCR (qPCR) analysis was performed using an Applied Biosystems StepOnePlus Real-Time PCR Systems (Applied Biosystems, MA, USA). The qPCR primer pairs for analysis of bacterial genes are listed (Suppl Table <a href="#MOESM1" class="usa-link">2</a>), including the universal bacterial marker <em>16S rRNA</em> gene and the <em>E. coli-</em>specific <em>uidA</em> gene [<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a>]. Moreover, the forward and reverse primers of the eukaryotic <em>18S rRNA</em> gene (<em>F</em>: 5’-TCAACTTTCGATGGTAGTCGCCGT-3’, and <em>R</em>: 5’-TCCTTGGATGTGGTAGCCGTTTCT-3’) were also used to generate a 108-bp fragment for normalization in some cases. The PCR reaction mixture consisted of 50 ng of RT product, 10 µL of Power SYBR Green PCR Master Mix and 125 nM specific primer pairs in a final reaction volume of 20 µL. The PCR conditions were 95 °C for 10 min, followed by 40 cycles of denaturation at 95 °C for 15 s, and annealing and extension at 60 °C for 1 min. Amplification plots were obtained by using a StepOnePlus™ Software v2.3 (Applied Biosystems). Each sample was run in duplicate, and the mean threshold cycle (CT) was determined from the two runs. The data analysis was based on the comparative threshold method. The CT value of the target genes was subtracted from that of <em>16 S rRNA</em> gene as the housekeeping gene (ΔCT). The difference of the ΔCT values between the treatment and the control conditions was determined as the double delta CT value (ΔΔCT). The relative gene expression was calculated as the logarithm of the value of -ΔΔCT to base 2 (2<sup>− ΔΔCT</sup>).</p></section><section id="Sec17"><h4 class="pmc_sec_title">Human sample collection for bacterial gene analysis</h4>
<p id="Par28">Colonic specimens were collected from 49 healthy individuals (21 males and 28 females, age interval: 33–78 years) and 114 patients with CRC (65 males and 49 females, age interval: 26–88 years) for analysis in NTUH. The numbers of patients at tumor stages I, II, III, and IV were 20, 39, 33, and 22, respectively. Written informed consent was obtained from all study subjects, and approval for the research ethics of this study was granted by the Institutional Review Board (IRB) for NTUH (#202104001RINC). The tissue RNA was reverse-transcribed and analyzed by qPCR primers of bacterial genes.</p></section></section><section id="Sec18"><h3 class="pmc_sec_title">Statistical analysis</h3>
<p id="Par29">Data are presented as mean ± SEM (standard error of mean), unless otherwise stated. The normality of the data was assessed using the Shapiro–Wilk test. When the normality assumption was appropriate, data comparisons were made by unpaired Student’s <em>t</em>-test for two groups and one-way analysis of variance (ANOVA) followed by Tukey’s multiple comparison post-hoc tests for three or more groups (GraphPad Prism v. 8.0.1). For data that failed the normality test, non-parametric analysis was employed. Mann–Whitney U test was used for two-group comparisons, and the Kruskal–Wallis test was used for comparisons among three or more groups, followed by Dunn’s multiple comparisons test as a post hoc analysis. A <em>P</em> value less than 0.05 is considered significant.</p></section></section><section id="Sec19"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec20"><h3 class="pmc_sec_title">Phenotypic characteristics of <em>E. coli</em> LI60C3 with invasive and tumorigenic ability</h3>
<p id="Par30">Our previous studies have demonstrated that the <em>E. coli</em> strain LI60C3, isolated from mouse colonocytes, exacerbates colitis severity and increases tumor burden in mouse models [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. A schematic diagram of the isolation of intraepithelial bacteria for phenotypic characterization is illustrated in Fig. <a href="#Fig1" class="usa-link">1</a>A. The invasive characteristics of LI60C3 were compared with those of other <em>E. coli</em>, including the adherent-invasive and laboratory strains, using in vitro bacteria-epithelial cocultures. The bacteria were exposed to human Caco-2 cell monolayers at an MOI of 10 for 4 h, and the intracellular bacteria were quantified using a gentamicin resistance assay. The intracellular bacterial counts of LI60C3 were 100 times higher than the nonpathogenic laboratory K-12 strains of <em>E. coli</em>, such as MG1655, DH5α, and JM109 (Fig. <a href="#Fig1" class="usa-link">1</a>B). Moreover, the apical bacterial counts of LI60C3 also exceeded those of the laboratory strains, indicating a greater extracellular multiplicity or viability of LI60C3 (Fig. <a href="#Fig1" class="usa-link">1</a>B). The invasive ability of LI60C3 was comparable to that of the prototypic AIEC strains LF82 and NC101 (Fig. <a href="#Fig1" class="usa-link">1</a>B). The LF82 strain was isolated from the ileal lesion of a patient with Crohn’s disease and had strong colitogenic ability in transgenic mice overexpressing human carcinoembryonic antigen-related cell adhesion molecule (CEACAM) 6 <sup>23, 41</sup>. The NC101 strain carries a pathogenicity island encoding genotoxins with colitogenic and tumorigenic ability in mouse models [<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>, <a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Although several <em>E. coli</em> strains displayed similar levels of invasiveness, exposure to LI60C3 exerted faster epithelial cell cycle rates compared to the other bacteria (Fig. <a href="#Fig1" class="usa-link">1</a>C), suggesting that LI60C3 has a stronger ability to induce epithelial proliferation. Orogastric administration of LI60C3 resulted in an increase in the number and area of colon tumors in mouse models when compared to the LF82 and PBS vehicle groups (Fig. <a href="#Fig1" class="usa-link">1</a>D-E).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366417_13099_2025_732_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f57a/12366417/c255b832734c/13099_2025_732_Fig1_HTML.jpg" loading="lazy" id="d33e950" height="1015" width="737" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phenotypic characterization of the invasive feature and tumorigenic ability of <em>E. coli</em> strains. <strong>(A)</strong> Schematic diagram of microbial isolation from intestinal epithelial cells for in vitro and in vivo phenotypic assays. The invasive features, proliferation-inducing ability, and tumorigenic characteristics of the intraepithelial bacteria obtained from mouse colonocytes were evaluated in human cell lines and experimental murine models. <strong>(B)</strong> The intraepithelial <em>E. coli</em> strains (LI60C3, HM784C1, HM926C2, and HM936C4), adherent-invasive <em>E. coli</em> strains (LF82 and NC101), and nonpathogenic laboratory strains (MG1655, DH5α and JM109) were exposed to Caco-2 cells at MOI = 10 for 4 h. (a) Intracellular bacterial counts and (b) apical bacterial counts in the epithelial cultures are shown. <strong>(C)</strong> Cell cycle progression of Caco-2 cells after bacterial exposure for 24 h. The cell ratios in (a) S/G2/M to G1/G0 phase and (b) high to low Ki67 staining intensity are shown. Data are expressed as mean ± SEM (<em>N</em> = 5–8/group). Each dot represents the data of one sample. *<em>P</em> &lt; 0.05 <em>vs.</em> w/o, and <sup>#</sup><em>P</em> &lt; 0.05 <em>vs.</em> LI60C3. <strong>(D)</strong> A murine model with chemically induced colon cancer was inoculated with PBS vehicle, LI60C3, or LF82 at 10<sup>9</sup> CFU per mouse via the orogastric route. Representative images of intestinal tumors in each mouse group are displayed. Bar: 2 mm (macroscopic images) and 200 μm (histological images, H&amp;E staining). <strong>(E)</strong> Increased tumor load was observed in mice inoculated with LI60C3, but not LF82. The tumor (a) number, (b) area, and (c) grade were determined for each mouse. The percentages of tumors with low- and high-grade dysplasia and carcinoma in each group are presented. Data are expressed as mean ± SEM (<em>N</em> = 8–10/group). Each dot represents the data of one mouse. *<em>P</em> &lt; 0.05 <em>vs.</em> PBS, and <sup>#</sup><em>P</em> &lt; 0.05 <em>vs.</em> LI60C3. The statistical significance of all panels is calculated by ANOVA followed by Tukey’s multiple comparison test, except those of (B-a), (C-a) and (E-a) are determined by the Kruskal–Wallis test. Experiments were repeated at least twice.</p></figcaption></figure></section><section id="Sec21"><h3 class="pmc_sec_title">Genome comparison of LI60C3 to reference <em>E. coli</em> strains</h3>
<p id="Par32">To understand the virulent strategy of LI60C3, the bacterial genome was analyzed using third-generation sequencing. Table <a href="#Tab1" class="usa-link">1</a> summarizes the key features of the <em>E. coli</em> LI60C3 strain and the genome sequencing methods. The experimental design for nucleic acid extraction from intraepithelial bacteria and the analysis workflow are presented (Fig. <a href="#Fig2" class="usa-link">2</a>A and B). The bacterial genome was assembled <em>de novo</em> through CLR datasets generated by the PacBio sequencing into one contig, revealing a circular chromosome of 4,863,930 bases (Fig. <a href="#Fig2" class="usa-link">2</a>C). The bacterial chromosome replication, which occurs at the origin of replication (<em>oriC</em>), is marked on the circular gene map. Moreover, two extrachromosomal circular plasmids with the lengths of 81,809 bases and 8,556 bases were identified by long-read assembly and were named pTra and pCoMb, respectively (Fig. <a href="#Fig2" class="usa-link">2</a>C). Based on the final polished assembly data, there are 4464 coding sequences (CDSs), 88 transfer RNA (tRNA), 22 ribosomal RNA (rRNA), 1 transfer-messenger RNA (tmRNA), and 1 repeat region in the LI60C3 chromosome.</p>
<section class="tw xbox font-sm" id="Tab1"><h4 class="obj_head">Table 1.</h4>
<div class="caption p"><p>Analysis of <em>Escherichia coli</em> LI60C3 strain</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Items</th>
<th align="left" colspan="1" rowspan="1">Descriptors</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Investigation type</strong>
</td>
<td align="left" colspan="1" rowspan="1">Bacteria</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Project name</strong>
</td>
<td align="left" colspan="1" rowspan="1">Whole genome sequencing of <em>Escherichia coli</em> LI60C3</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Isolation sources</strong>
</td>
<td align="left" colspan="1" rowspan="1">Mouse colonic epithelial cells</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Isolation date</strong>
</td>
<td align="left" colspan="1" rowspan="1">2017.06.26</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>
<strong>Estimated length</strong>
</p>
<p>
<strong>in bases</strong>
</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p>4,863,930 (chromosome),</p>
<p>81,809 and 8,556 (plasmid)</p>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Sequencing method</strong>
</td>
<td align="left" colspan="1" rowspan="1">PacBio Single Molecule Real-Time (SMRT) sequencing</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Sequencing format</strong>
</td>
<td align="left" colspan="1" rowspan="1">continuous long reads (CLR)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Assembly</strong>
</td>
<td align="left" colspan="1" rowspan="1">SMRT link</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Assembly level</strong>
</td>
<td align="left" colspan="1" rowspan="1">1 contig (chromosome), 2 contigs (2 plasmids)</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Circular</strong>
</td>
<td align="left" colspan="1" rowspan="1">yes</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Polymerase Read bases</strong>
</td>
<td align="left" colspan="1" rowspan="1">3,182,269,527</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Polymerase Read counts</strong>
</td>
<td align="left" colspan="1" rowspan="1">45,050</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Polymerase Read length (mean)</strong>
</td>
<td align="left" colspan="1" rowspan="1">70,638</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Subread counts</strong>
</td>
<td align="left" colspan="1" rowspan="1">346,106</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Subread length (mean)</strong>
</td>
<td align="left" colspan="1" rowspan="1">9,129</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Subread N50</strong>
</td>
<td align="left" colspan="1" rowspan="1">10,398</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Trimmed coverage</strong>
</td>
<td align="left" colspan="1" rowspan="1">576x, 1046x, 766x</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>No. of coding sequences</strong>
</td>
<td align="left" colspan="1" rowspan="1">4464, 89, 8</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>No. of tRNA</strong>
</td>
<td align="left" colspan="1" rowspan="1">88, 0, 0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>No. of rRNA</strong>
</td>
<td align="left" colspan="1" rowspan="1">22, 0, 0</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>GC content</strong>
</td>
<td align="left" colspan="1" rowspan="1">50.7%, 51.3%, 47.5%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<strong>Finishing strategy</strong>
</td>
<td align="left" colspan="1" rowspan="1">Sequencing &amp; <em>de novo</em> Assembly</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366417_13099_2025_732_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f57a/12366417/cf3896f6a15c/13099_2025_732_Fig2_HTML.jpg" loading="lazy" id="d33e1254" height="729" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Whole genome sequencing of the tumorigenic <em>E. coli</em> strain LI60C3. <strong>(A)</strong> Schema of nucleic acid extraction from LI60C3 for genome sequencing using a PacBio Sequel II platform. The continuous long-read datasets generated by PacBio platform were used for <em>de novo</em> assembly of the whole genome. <strong>(B)</strong> Analysis flow chart for quality assessment, trimmed coverage, and contig assembly. <strong>(C)</strong> The bacterial genome was assembled <em>de novo</em> through long reads and revealed a circular chromosome (4,863,930 bases) and two extrachromosomal plasmids pTra (81,809 bases) and pCoMb (8,556 bases). Circular maps of <em>E. coli</em> LI60C3 chromosome are drawn starting at the origin of replication (<em>oriC</em>). From the outside in, the first circle shows the locations of phage integrases (green) and prophage-like genes (purple). Circle 2 shows the nucleotide sequence positions (in Mbp). Circles 3 and 4 show protein coding sequences (CDSs) transcribed clockwise and anticlockwise, respectively (genes conserved among LI60C3 and the other twelve <em>E. coli</em> reference strains (grey); unique cluster genes of LI60C3 transcribed clockwise (red); unique cluster genes of LI60C3 transcribed anticlockwise (blue)). Circle 5 shows the tRNA genes (brown) and rRNA genes (light blue). Circle 6 shows a plot of GC content in a 10-kb window. (mean GC content in a 10-kb window– overall mean GC). Red areas indicate that the deviation is higher than 2 Standard deviations. Circle 7 shows a plot of GC skew ([G-C]/[G + C], in a 10-kb window). Orange areas indicate that G &gt; C, and blue areas indicate that G &lt; C. Moreover, two plasmids were found in the LI60C3 genome, including pTra and pCoMb. The boxes in the outer and inner circles represent CDSs transcribed clockwise and anticlockwise, respectively</p></figcaption></figure><p id="Par35">The complete genome of LI60C3 was compared with those of reference <em>E. coli</em> in public databases, which included two commensal strains, four pathobiont strains, and six pathogenic strains (Table <a href="#Tab2" class="usa-link">2</a>). The LI60C3 gene sequence was aligned with those of nonpathogenic laboratory strains of <em>E. coli</em>, such as MG1655 and W3110. The genome of K-12 substrains (i.e., MG1655 and W3110) derived from human stool with minimal genetic manipulation was used for comparison, whereas those engineered for plasmid transfection efficiency were not (e.g., DH5α and JM109). The chromosome sequence of LI60C3 exhibited mapping rates of 83.36% and 83.31%, in comparison to MG1655 and W3110 (Table <a href="#Tab2" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="Tab2"><h4 class="obj_head">Table 2.</h4>
<div class="caption p"><p>Genome comparison of <em>Escherichia coli</em> LI60C3 to reference strains</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Bacteria<br>(Pathotype)</th>
<th align="left" colspan="1" rowspan="1">Serotype</th>
<th align="left" colspan="1" rowspan="1">Phylo-<br>group</th>
<th align="left" colspan="1" rowspan="1">Source,<br>Human or mouse</th>
<th align="left" colspan="1" rowspan="1">GenBank<br>Accession no.</th>
<th align="left" colspan="1" rowspan="1">Length in bases</th>
<th align="left" colspan="1" rowspan="1">Mapping rate (%) vs. LI60C3 or pTra and pCoMb</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>
<strong>K12 MG1655</strong>
</p>
<p>(nonpathogenic)</p>
</td>
<td align="left" colspan="1" rowspan="1">OR: H48:K-</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">Feces, a healthy subject</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/U00096.3" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">U00096.3</a></td>
<td align="left" colspan="1" rowspan="1">
<strong>4</strong>,<strong>641</strong>,<strong>652 (chromosome</strong>,<strong> circular)</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>83.36%</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>
<strong>K-12 W3110</strong>
</p>
<p>(nonpathogenic)</p>
</td>
<td align="left" colspan="1" rowspan="1">Not defined</td>
<td align="left" colspan="1" rowspan="1">A</td>
<td align="left" colspan="1" rowspan="1">Feces, a healthy subject</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/AP009048.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AP009048.1</a></td>
<td align="left" colspan="1" rowspan="1">
<strong>4</strong>,<strong>646</strong>,<strong>332 (chromosome</strong>,<strong> circular)</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>83.31%</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<p>
<strong>LF82</strong>
</p>
<p>(AIEC)</p>
</td>
<td align="left" rowspan="2" colspan="1">O83:H1</td>
<td align="left" rowspan="2" colspan="1">B2</td>
<td align="left" rowspan="2" colspan="1">Ileum, a patient with Crohn’s disease</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/CU651637" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CU651637</a></td>
<td align="left" colspan="1" rowspan="1">
<strong>4</strong>,<strong>773</strong>,<strong>108 (chromosome</strong>,<strong> circular)</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>91.00%</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/CU638872" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CU638872</a></td>
<td align="left" colspan="1" rowspan="1">108,379 (plasmid, pLF82)</td>
<td align="left" colspan="1" rowspan="1">0 and 0%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>
<strong>NC101</strong>
</p>
<p>(AIEC)</p>
</td>
<td align="left" colspan="1" rowspan="1">O2:H6/41</td>
<td align="left" colspan="1" rowspan="1">B2</td>
<td align="left" colspan="1" rowspan="1">Cecum, wild-type 129S6/SvEv mice</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/CP072787" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP072787</a></td>
<td align="left" colspan="1" rowspan="1">
<strong>5</strong>,<strong>030</strong>,<strong>087 (chromosome</strong>,<strong> circular)</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>86.97%</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<p>
<strong>SK1144</strong>
</p>
<p>(DAEC)</p>
</td>
<td align="left" rowspan="2" colspan="1">Not defined</td>
<td align="left" rowspan="2" colspan="1">B2</td>
<td align="left" rowspan="2" colspan="1">Feces, a healthy subject</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NZ_AP018784.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NZ_AP018784.1</a></td>
<td align="left" colspan="1" rowspan="1">
<strong>5</strong>,<strong>029</strong>,<strong>244 (chromosome</strong>,<strong> circular)</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>83.93%</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NZ_AP018785.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NZ_AP018785.1</a></td>
<td align="left" colspan="1" rowspan="1">169,561 (plasmid, pSK1144)</td>
<td align="left" colspan="1" rowspan="1">22.63 and 0%</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">
<p>
<strong>2017 C-4173W12</strong>
</p>
<p>(DAEC)</p>
</td>
<td align="left" rowspan="3" colspan="1">O<sup>−</sup>:H2</td>
<td align="left" rowspan="3" colspan="1">D</td>
<td align="left" rowspan="3" colspan="1">Feces, a patient with diarrheal disease</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP030768.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NZ_CP030768.1</a></td>
<td align="left" colspan="1" rowspan="1">
<strong>5</strong>,<strong>300</strong>,<strong>308 (chromosome</strong>,<strong> circular)</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>75.53%</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP030770.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NZ_CP030770.1</a></td>
<td align="left" colspan="1" rowspan="1">149,680 (plasmid, p2017C-4173W12)</td>
<td align="left" colspan="1" rowspan="1">0 and 0%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP030769.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NZ_CP030769.1</a></td>
<td align="left" colspan="1" rowspan="1">27,128 (plasmid, pMCR-1_2017C-4173W12)</td>
<td align="left" colspan="1" rowspan="1">24.32 and 0%</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">
<p>
<strong>E2348/69</strong>
</p>
<p>(EPEC)</p>
</td>
<td align="left" rowspan="3" colspan="1">O127:H6</td>
<td align="left" rowspan="3" colspan="1">B2</td>
<td align="left" rowspan="3" colspan="1">Feces, a patient with diarrheal disease</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_011601" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NC_011601</a></td>
<td align="left" colspan="1" rowspan="1">
<strong>4</strong>,<strong>965</strong>,<strong>553(chromosome</strong>,<strong> circular)</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>82.95%</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_011603" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NC_011603</a></td>
<td align="left" colspan="1" rowspan="1">97,978 (plasmid, pMAR2)</td>
<td align="left" colspan="1" rowspan="1">37.18 and 0%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/DQ388534.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DQ388534.1</a></td>
<td align="left" colspan="1" rowspan="1">101,558 (plasmid, pMAR7)</td>
<td align="left" colspan="1" rowspan="1">35.76 and 0%</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">
<p>
<strong>CFSAN029787</strong>
</p>
<p>(EIEC)</p>
</td>
<td align="left" rowspan="3" colspan="1">O96: H19</td>
<td align="left" rowspan="3" colspan="1">B1</td>
<td align="left" rowspan="3" colspan="1">Feces, a patient with diarrheal disease</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP011416.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NZ_CP011416.1</a></td>
<td align="left" colspan="1" rowspan="1">
<strong>4</strong>,<strong>947</strong>,<strong>515 (chromosome</strong>,<strong> circular)</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>78.66%</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP011417.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NZ_CP011417.1</a></td>
<td align="left" colspan="1" rowspan="1">293,826 (plasmid, pCFSAN029787_01)</td>
<td align="left" colspan="1" rowspan="1">11.93 and 0%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP011418.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NZ_CP011418.1</a></td>
<td align="left" colspan="1" rowspan="1">47,606 (plasmid, pCFSAN029787_02)</td>
<td align="left" colspan="1" rowspan="1">23.39 and 0%</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<p>
<strong>EDL933</strong>
</p>
<p>(EHEC/STEC)</p>
</td>
<td align="left" rowspan="2" colspan="1">O157:H7</td>
<td align="left" rowspan="2" colspan="1">E</td>
<td align="left" rowspan="2" colspan="1">Feces, a patient with diarrheal disease</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP008957.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NZ_CP008957.1</a></td>
<td align="left" colspan="1" rowspan="1">
<strong>5</strong>,<strong>547</strong>,<strong>323(chromosome</strong>,<strong> circular)</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>69.67%</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP008958.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NZ_CP008958.1</a></td>
<td align="left" colspan="1" rowspan="1">92,076 (plasmid, pEDL933)</td>
<td align="left" colspan="1" rowspan="1">19.59 and 0%</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">
<p>
<strong><a href="https://www.ncbi.nlm.nih.gov/nuccore/H10407" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">H10407</a></strong>
</p>
<p>(ETEC)</p>
</td>
<td align="left" rowspan="3" colspan="1">O78:H11</td>
<td align="left" rowspan="3" colspan="1">A</td>
<td align="left" rowspan="3" colspan="1">Feces, a patient with diarrheal disease</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_017633.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NC_017633.1</a></td>
<td align="left" colspan="1" rowspan="1">
<strong>5</strong>,<strong>153</strong>,<strong>435 (chromosome</strong>,<strong> circular)</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>76.32%</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_017724.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NC_017724.1</a></td>
<td align="left" colspan="1" rowspan="1">94,797 (plasmid, p948)</td>
<td align="left" colspan="1" rowspan="1">41.63 and 0%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_017722.1" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">NC_017722.1</a></td>
<td align="left" colspan="1" rowspan="1">66,681 (plasmid, p666)</td>
<td align="left" colspan="1" rowspan="1">4.09 and 0%</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>
<strong>536</strong>
</p>
<p>(UPEC)</p>
</td>
<td align="left" colspan="1" rowspan="1">O6:K15:H31</td>
<td align="left" colspan="1" rowspan="1">B2</td>
<td align="left" colspan="1" rowspan="1">Urine, a patient with pyelonephritis</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/CP000247" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP000247</a></td>
<td align="left" colspan="1" rowspan="1">
<strong>4</strong>,<strong>938</strong>,<strong>920 (chromosome</strong>,<strong> circular)</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>87.80%</strong>
</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<p>
<strong>UT189</strong>
</p>
<p>(UPEC)</p>
</td>
<td align="left" rowspan="2" colspan="1">O18:K1:H7</td>
<td align="left" rowspan="2" colspan="1">B2</td>
<td align="left" rowspan="2" colspan="1">Urine, a patient with cystitis</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/CP000243" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP000243</a></td>
<td align="left" colspan="1" rowspan="1">
<strong>5</strong>,<strong>076</strong>,<strong>615 (chromosome</strong>,<strong> circular)</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong>85.66%</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/CP000244" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP000244</a></td>
<td align="left" colspan="1" rowspan="1">114,230 (plasmid, pUT189)</td>
<td align="left" colspan="1" rowspan="1">32.27 and 0%</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p70"><p>Footnote: The mapping rates were calculated by using the quality assessment tool for genome assemblies (QUAST). AIEC, adherent-invasive <em>E. coli</em>; DAEC, diffusely adherent <em>E. coli</em>; EPEC, enteropathogenic <em>E. coli</em>; EIEC, enteroinvasive <em>E. coli</em>; EHEC/STEC, enterohemorrhagic <em>E. coli</em>/Shiga toxin-producing <em>E. coli</em>; ETEC; enterotoxigenic <em>E. coli</em>; UPEC, uropathogenic <em>E. coli</em>.</p></div></div></section><p id="Par37">The sequences of pathobiont strains with adherent-invasive characteristics (AIEC strains of LF82 and NC101) or with diffusely adherent properties (DAEC strains of SK1144 and 2017 C-4173W12) were compared with the genome of LI60C3. The DAEC strains are <em>E. coli</em> that harbor afimbrial adhesins encoded by <em>afa</em> genes, which were isolated from healthy or diarrheal patients [<a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>–<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>]. In a sequence alignment with LI60C3, the chromosome of LF82 exhibited the highest mapping rate of 91.00%, while the NC101 strain showed a sequence homology of 86.97% (Table <a href="#Tab2" class="usa-link">2</a>). On the other hand, SK1144 and 2017C-4173W12 were matched with 83.93% and 75.53% sequence homology, respectively (Table <a href="#Tab2" class="usa-link">2</a>).</p>
<p id="Par38">The <em>E. coli</em> strains 536 and UT189, classified as UPEC, showed sequence homology to LI60C3 with mapping rates of 87.80% and 85.66%, respectively. Other pathogenic <em>E. coli</em>, such as E2348/69 (an EPEC strain), CFSAN029787 (an EIEC strain), EDL933 (an EHEC/STEC strain), and <a href="https://www.ncbi.nlm.nih.gov/nuccore/H10407" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">H10407</a> (an ETEC strain), had less similarity with LI603 that ranges from 69.67 to 82.95%. Among the chromosome sequences of pathogenic <em>E. coli</em>, the UPEC strains had the highest and the EHEC strains had the lowest genome homology to LI60C3 (Table <a href="#Tab2" class="usa-link">2</a>). By comparing with plasmids found in the reference <em>E. coli</em> strains, the sequence of pTra in LI60C3 is partially homologous to that of p948 in <a href="https://www.ncbi.nlm.nih.gov/nuccore/H10407" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">H10407</a> (an ETEC strain) with a 41.63% similarity and to that of pUT189 in UT189 (a UPEC strain) with a 32.27% similarity (Table <a href="#Tab2" class="usa-link">2</a>). No sequence homology was observed between pTra and the plasmid pLF82 in the AIEC strain LF82 (Table <a href="#Tab2" class="usa-link">2</a>).</p>
<p id="Par39">The conserved gene clusters identified within the LI60C3 sequence were aligned with the three <em>E. coli strains</em> exhibiting the highest mapping accuracy, as depicted in the colinear regions in the synteny plots (Fig. <a href="#Fig3" class="usa-link">3</a>). The genome rearrangements are demonstrated in homologous synteny blocks in a pairwise manner between LI60C3 and other <em>E. coli</em> strains, such as LF82 (Fig. <a href="#Fig3" class="usa-link">3</a>A), NC101 (Fig. <a href="#Fig3" class="usa-link">3</a>B), and 536 (Fig. <a href="#Fig3" class="usa-link">3</a>C). In summary, the chromosome sequence of LI60C3 displays a high degree of genome homology with AIEC strains (i.e., LF82 and NC101) and UPEC strains (i.e., 536 and UT189).</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366417_13099_2025_732_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f57a/12366417/e625cf4a56a5/13099_2025_732_Fig3_HTML.jpg" loading="lazy" id="d33e1935" height="530" width="669" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Synteny plots of the LI60C3 genome with other reference strains. High sequence similarity and conserved gene clusters were identified between LI60C3 and other <em>E. coli</em> strains, such as the AIEC strains <strong>(A)</strong> LF82 and <strong>(B)</strong> NC101, and <strong>(C)</strong> a UPEC strain 536</p></figcaption></figure></section><section id="Sec22"><h3 class="pmc_sec_title">Phylogeny and virulence genes of the <em>E. coli</em> LI60C3</h3>
<p id="Par41">The evolutionary phylogenetic tree was constructed using sequences of seven house-keeping genes <em>(arcA</em>,<em> aroE</em>,<em> icd</em>,<em> mdh</em>,<em> mtlD</em>,<em> pgi</em>, and <em>rpoS)</em> from 25 reference strains of <em>E. coli</em>, including LI60C3. The phylogeny of LI60C3 is classified as B2, with the closest ancestral root linked to a UPEC strain 536 (Fig. <a href="#Fig4" class="usa-link">4</a>). Other <em>E. coli</em> strains within the B2 phylogenetic group closely related to LI60C3 included LF82, NC101, E2348/69, UT189, and CFT073 (Fig. <a href="#Fig4" class="usa-link">4</a>). The B2 phylogroup of LI60C3 was further validated using Clermont’s triplex PCR assay (Fig. <a href="#Fig5" class="usa-link">5</a>A and B). While LF82 and NC101 strains belong to the B2 phylogroup, other intraepithelial <em>E. coli</em> isolated from mouse colonocytes, such as HM784C1 and HM926C2, fall within phylogroup B1. In contrast, laboratory K-12 substrains (i.e., MG1655 and DH5α) are classified under phylogroup A, and the strain JM109 belongs to phylogroup B1 (Fig. <a href="#Fig5" class="usa-link">5</a>A and B).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366417_13099_2025_732_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f57a/12366417/2c8e5f2db13e/13099_2025_732_Fig4_HTML.jpg" loading="lazy" id="d33e2026" height="565" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phylogenetic analysis of <em>E. coli</em> strains. The analysis was based on the sequence of seven house-keeping genes (<em>arcA</em>,<em> aroE</em>,<em> icd</em>,<em> mdh</em>,<em> mtlD</em>,<em> pgi</em>, and <em>rpoS</em>) of 25 reference strains of <em>E. coli</em>, including LI60C3. The major branches are labeled according to the phylogroups (i.e., A, B1, B2, D, E, F), which are written in bold letters. Distance is defined as the fraction of mismatches at aligned positions, with gaps either ignored or counted as mismatches</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366417_13099_2025_732_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f57a/12366417/bb52286d0644/13099_2025_732_Fig5_HTML.jpg" loading="lazy" id="d33e2078" height="423" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Phylotyping of LI60C3 based on the Clermont method of triplex PCR assay. The phylogroups of <em>E. coli</em> were determined using primer pairs targeting three genes (<em>chuA</em>, <em>yjaA</em>,<em> and tspE4.C2)</em>, in accordance with Clermont’s triplex PCR assay. <strong>(A)</strong> Representative gel image showing PCR bands generated from genomic DNA samples of LI60C3 and laboratory <em>E. coli</em> strains. The phylogroup of each strain is labeled at the bottom. <strong>(B)</strong> Representative gel image showing the phylogroup of mouse-isolated <em>E. coli</em> strains and adherent-invasive LF82 and NC101 strains. <strong>(C)</strong> The plasmid <em>mbeA</em> and <em>colY</em> genes were found in the genomic DNA samples extracted from LI60C3 but not from other <em>E. coli</em> strains. <strong>(D)</strong> Electrophoretic gel images of the extracted plasmid and genomic DNA samples obtained from bacteria. The presence of extrachromosomal plasmids is confirmed in the LI60C3 and LF82 strains but not found in other bacteria. Left: samples eluted from the plasmid extraction kit. Right: samples eluted from the genomic extraction kit</p></figcaption></figure><p id="Par44">Virulence elements and antibiotic-resistance genes in LI60C3 were predicted by sequence alignment using the CARD and VFDB databases. Consistent with our previous antibiogram profile from the antimicrobial susceptibility test, the LI60C3 genome contains genes that confer antibiotic resistance through efflux pumps (Table <a href="#Tab3" class="usa-link">3</a>). The LI60C3 genome harbors virulence-encoding genes related to adherence (pilus and fimbriae), invasion, iron uptake, type IV secretion system, and hemolysins (Table <a href="#Tab4" class="usa-link">4</a>). Additionally, it was confirmed that the LI60C3 genome lacks both genotoxin and cyclomodulin genes.</p>
<section class="tw xbox font-sm" id="Tab3"><h4 class="obj_head">Table 3.</h4>
<div class="caption p"><p>Antibiotic-resistant genes in LI60C3 genome</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Resistance mechanism</th>
<th align="left" colspan="1" rowspan="1">Gene</th>
<th align="left" colspan="1" rowspan="1">Gene Family</th>
<th align="left" colspan="1" rowspan="1">Antibiotic Class</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="17" colspan="1">antibiotic efflux</td>
<td align="left" colspan="1" rowspan="1">
<em>acrA/B</em>
</td>
<td align="left" colspan="1" rowspan="1">RND antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">fluoroquinolone antibiotic, cephalosporin, glycylcycline, penam, tetracycline antibiotic, rifamycin antibiotic, phenicol antibiotic</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>cpxA</em>
</td>
<td align="left" colspan="1" rowspan="1">RND antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">aminoglycoside antibiotic, aminocoumarin antibiotic</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>emrY</em>
</td>
<td align="left" colspan="1" rowspan="1">MFS antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">tetracycline antibiotic</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>evgA</em>
</td>
<td align="left" colspan="1" rowspan="1">MFS and RND antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">macrolide antibiotic, fluoroquinolone antibiotic, penam, tetracycline</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>emrR</em>
</td>
<td align="left" colspan="1" rowspan="1">MFS antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">fluoroquinolone antibiotic</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>gadW</em>
</td>
<td align="left" colspan="1" rowspan="1">RND antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">macrolide antibiotic, fluoroquinolone antibiotic, penam</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>hns</em>
</td>
<td align="left" colspan="1" rowspan="1">MFS and RND antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">macrolide antibiotic, fluoroquinolone antibiotic, cephalosporin, cephamycin, penam, tetracycline antibiotic</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>kdpE</em>
</td>
<td align="left" colspan="1" rowspan="1">kdpDE</td>
<td align="left" colspan="1" rowspan="1">aminoglycoside antibiotic</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>leuO</em>
</td>
<td align="left" colspan="1" rowspan="1">MFS antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">nucleoside antibiotic, disinfecting agents and antiseptics</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>mdfA</em>
</td>
<td align="left" colspan="1" rowspan="1">MFS antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">tetracycline antibiotic, phenicol antibiotic, disinfecting agents, antiseptics</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>mdtG</em>
</td>
<td align="left" colspan="1" rowspan="1">MFS antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">phosphonic acid antibiotic</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>mdtH</em>
</td>
<td align="left" colspan="1" rowspan="1">MFS antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">fluoroquinolone antibiotic</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>mdtP</em>
</td>
<td align="left" colspan="1" rowspan="1">MFS antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">nucleoside antibiotic, disinfecting agents, and antiseptics</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>mdtO</em>
</td>
<td align="left" colspan="1" rowspan="1">MFS antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">nucleoside antibiotic, disinfecting agents, and antiseptics</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>mdtN</em>
</td>
<td align="left" colspan="1" rowspan="1">MFS antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">nucleoside antibiotic, disinfecting agents, and antiseptics</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>msbA</em>
</td>
<td align="left" colspan="1" rowspan="1">ABC antibiotic efflux pump</td>
<td align="left" colspan="1" rowspan="1">nitroimidazole antibiotic</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>marA</em>
</td>
<td align="left" colspan="1" rowspan="1">RND antibiotic efflux pump, General Bacterial Porin with reduced permeability to beta-lactams</td>
<td align="left" colspan="1" rowspan="1">fluoroquinolone antibiotic, monobactam, carbapenem, cephalosporin, glycylcycline, cephamycin, penam, tetracycline antibiotic, rifamycin antibiotic, phenicol antibiotic, penem, disinfecting agents, antiseptics</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">antibiotic target alteration</td>
<td align="left" colspan="1" rowspan="1">
<em>eptA</em>
</td>
<td align="left" colspan="1" rowspan="1">pmr phosphoethanolamine transferase</td>
<td align="left" colspan="1" rowspan="1">peptide antibiotic</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>vanG</em>
</td>
<td align="left" colspan="1" rowspan="1">glycopeptide resistance gene cluster, Van ligase</td>
<td align="left" colspan="1" rowspan="1">glycopeptide antibiotic</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">antibiotic inactivation</td>
<td align="left" colspan="1" rowspan="1">
<em>EC-5</em>
</td>
<td align="left" colspan="1" rowspan="1">EC beta-lactamase</td>
<td align="left" colspan="1" rowspan="1">cephalosporin</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p84"><p>Footnote: Resistance genes were predicted by the Resistance Gene Identifier (RGI) (v.6.0.3) which performed homolog detection using DIAMOND in Comprehensive Antibiotic Resistance Database (CARD) (v.3.3.0). ABC, ATP-binding cassette; MFS, major facilitator superfamily; RND, resistance-nodulation-cell division</p></div></div></section><section class="tw xbox font-sm" id="Tab4"><h4 class="obj_head">Table 4.</h4>
<div class="caption p"><p>Virulence-encoding genes in LI60C3 genome</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Virulence Class</th>
<th align="left" rowspan="2" colspan="1">Virulence Factors</th>
<th align="left" rowspan="2" colspan="1">Gene</th>
<th align="left" colspan="8" rowspan="1">Phylogroup B2</th>
<th align="left" colspan="2" rowspan="1">Phylogroup A</th>
</tr>
<tr>
<th align="left" colspan="1" rowspan="1">LI603</th>
<th align="left" colspan="1" rowspan="1">LF82</th>
<th align="left" colspan="1" rowspan="1">N101</th>
<th align="left" colspan="1" rowspan="1">536</th>
<th align="left" colspan="1" rowspan="1">UT189</th>
<th align="left" colspan="1" rowspan="1">CFT073</th>
<th align="left" colspan="1" rowspan="1">SK1144</th>
<th align="left" colspan="1" rowspan="1">E2348/69</th>
<th align="left" colspan="1" rowspan="1">MG1655</th>
<th align="left" colspan="1" rowspan="1">W3110</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="4" colspan="1">Adherence</td>
<td align="left" colspan="1" rowspan="1">E. coli common pilus</td>
<td align="left" colspan="1" rowspan="1">
<p>
<em>ecp</em>
</p>
<p>
<em>A/B/C/D/E</em>
</p>
</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Hemorrhagic E. coli pilus</td>
<td align="left" colspan="1" rowspan="1">
<p>
<em>hcp</em>
</p>
<p>
<em>A/B/C</em>
</p>
</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">P fimbriae</td>
<td align="left" colspan="1" rowspan="1">
<p>
<em>pap</em>
</p>
<p>
<em>C/D/H</em>
</p>
</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Type 1 fimbriae</td>
<td align="left" colspan="1" rowspan="1">
<p>
<em>fim</em>
</p>
<p>
<em>A/C/D/E/F/G/H/I</em>
</p>
</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">Autotransporter</td>
<td align="left" colspan="1" rowspan="1">Contact-dependent inhibition CDI system</td>
<td align="left" colspan="1" rowspan="1">
<em>cdiA</em>
</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Vacuolating autotransporter gene</td>
<td align="left" colspan="1" rowspan="1">
<em>vat</em>
</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Invasion</td>
<td align="left" colspan="1" rowspan="1">Invasion of brain endothelial cells</td>
<td align="left" colspan="1" rowspan="1">
<p>
<em>ibe</em>
</p>
<p>
<em>B/C</em>
</p>
</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
</tr>
<tr>
<td align="left" rowspan="3" colspan="1">Iron uptake</td>
<td align="left" colspan="1" rowspan="1">Heme uptake</td>
<td align="left" colspan="1" rowspan="1">
<p>
<em>chu</em>
</p>
<p>
<em>A/X/T/U/W/X/Y</em>
</p>
</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Iron/manganese transport</td>
<td align="left" colspan="1" rowspan="1">
<p>
<em>sit</em>
</p>
<p>
<em>A/B/C/D</em>
</p>
</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Yersiniabactin siderophore</td>
<td align="left" colspan="1" rowspan="1">
<p>
<em>ybt</em>
</p>
<p>
<em>A/E/P/Q/S/T/U/X</em>
</p>
</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">Secretion system</td>
<td align="left" colspan="1" rowspan="1">Non-LEE encoded TTSS effectors</td>
<td align="left" colspan="1" rowspan="1">
<p>
<em>espL1</em>
</p>
<p>
<em>espR1</em>
</p>
</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">- <sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">- <sup>a</sup>
</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">SCI-I T6SS</td>
<td align="left" colspan="1" rowspan="1">
<em>Undetermined</em>
</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Toxin</td>
<td align="left" colspan="1" rowspan="1">Hemolysin/cytolysin A</td>
<td align="left" colspan="1" rowspan="1">
<em>hlyE/clyA</em>
</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">+</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Antimicrobial activity</td>
<td align="left" colspan="1" rowspan="1">Phenazine biosynthesis</td>
<td align="left" colspan="1" rowspan="1">
<p>
<em>phzB2</em>
</p>
<p>
<em>phzF2</em>
</p>
</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Serum resistance</td>
<td align="left" colspan="1" rowspan="1">LPS rfb locus</td>
<td align="left" colspan="1" rowspan="1">Undetermined</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="left" colspan="1" rowspan="1">-</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p107">
<p>Footnote: Virulence factor-encoding genes were predicted based on sequence alignment using the virulence factor database (VFDB). LEE, locus of enterocyte effacement; TTSS, Type III secretion systems; T6SS, type VI secretion system; CDI, contact-dependent growth inhibition</p>
<p><sup>a</sup> Contains other <em>esp</em> or <em>nle</em> genes encoding for the non-LEE encoded TTSS effectors</p>
</div></div></section></section><section id="Sec23"><h3 class="pmc_sec_title">Two extrachromosomal plasmids were detected in the LI60C3 genome</h3>
<p id="Par47">The presence of two extrachromosomal plasmids in LI60C3 was revealed by WGS analysis, which were derived from contigs assembled from PacBio sequencing. The larger plasmid named pTra (81,809 bases) contains <em>tra</em> genes that encode for transfer proteins involved in plasmid DNA transfer during bacterial conjugation, such as pili synthesis and assembly, and helicase activity. Several <em>tra</em> genes are annotated in the 82-kb plasmid sequence, including <em>traI</em>,<em> traD</em>,<em> traS</em>,<em> traQ</em>,<em> traC</em>,<em> traV</em>,<em> traA</em>,<em> traY</em>,<em> traJ</em>,<em> and traM</em>. In the <em>tra</em> operons, <em>traA</em> encodes the precursor of the pilus subunit propilin; <em>traQ</em> gene products facilitate TraA maturation; <em>traI</em> gene encodes for helicase with DNA unwinding activity, and <em>traY</em>, <em>traJ</em>, and <em>traM</em> are involved in the autoregulatory pathway to induce DNA bending and stimulate nicking [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. We used the basic local alignment search tool (BLAST) in NCBI to search for plasmids in public databases with homology to pTra. The NCBI blasting indicated that several plasmids in commensal <em>E. coli</em> strains exhibited a 99.75% genome identity with a query coverage of 57% when compared to the pTra sequence (Table <a href="#Tab5" class="usa-link">5</a>).</p>
<section class="tw xbox font-sm" id="Tab5"><h4 class="obj_head">Table 5.</h4>
<div class="caption p"><p>Genome comparison of LI60C3 pTra to other plasmids</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Plasmid Name</th>
<th align="left" colspan="1" rowspan="1">GenBank<br>Accession No.</th>
<th align="left" colspan="1" rowspan="1">Genome Size<br>(bp)</th>
<th align="left" colspan="1" rowspan="1">Query <br>Coverage (%)</th>
<th align="left" colspan="1" rowspan="1">Percent <br>Identity (%)</th>
<th align="left" colspan="1" rowspan="1">Total Score</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Escherichia coli strain MG1655 K12 plasmid F-Tn10</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/MK492260" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MK492260</a></td>
<td align="center" colspan="1" rowspan="1">108,314</td>
<td align="center" colspan="1" rowspan="1">57</td>
<td align="center" colspan="1" rowspan="1">99.75</td>
<td align="center" colspan="1" rowspan="1">85,004</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Escherichia coli strain Stbl4 plasmid pF</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/CP076044" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP076044</a></td>
<td align="center" colspan="1" rowspan="1">98,268</td>
<td align="center" colspan="1" rowspan="1">57</td>
<td align="center" colspan="1" rowspan="1">99.75</td>
<td align="center" colspan="1" rowspan="1">84,952</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Escherichia coli K-12 plasmid F DNA</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/AP001918" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">AP001918</a></td>
<td align="center" colspan="1" rowspan="1">99,159</td>
<td align="center" colspan="1" rowspan="1">57</td>
<td align="center" colspan="1" rowspan="1">99.75</td>
<td align="center" colspan="1" rowspan="1">84,952</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Escherichia coli strain NEB5-alpha_F’Iq plasmid F’Iq</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/CP053608" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP053608</a></td>
<td align="center" colspan="1" rowspan="1">242,042</td>
<td align="center" colspan="1" rowspan="1">57</td>
<td align="center" colspan="1" rowspan="1">99.75</td>
<td align="center" colspan="1" rowspan="1">84,991</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Escherichia coli strain NEB_Turbo plasmid F’</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/CP053606" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP053606</a></td>
<td align="center" colspan="1" rowspan="1">231,547</td>
<td align="center" colspan="1" rowspan="1">57</td>
<td align="center" colspan="1" rowspan="1">99.75</td>
<td align="center" colspan="1" rowspan="1">84,996</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p112"><p>Footnote: The complete sequence of plasmids was compared using NCBI blasting. Plasmids with a query coverage of less than 56% are not shown, as this is set as the main threshold</p></div></div></section><p id="Par49">The smaller plasmid named pCoMb (8,556 bases) encompasses several genes encoding for colicin Y (colY), colicin lysis, colicin immunity protein, DNA relaxase/mobilization protein (mbe) A and mbeC, regulatory protein Rop, and entry exclusion protein. The annotated genes in pCoMb were also confirmed by Sanger sequencing on the plasmid DNA samples extracted from LI60C3 using a commercially available Mini Plasmid kit. The NCBI blasting showed that pCoMb closely matched the plasmid sequences found in <em>E. coli</em> strains and other pathogenic bacteria. These include the plasmids reported in <em>E. coli</em> strain NGF1 (Accession no. <a href="https://www.ncbi.nlm.nih.gov/nuccore/CP016009" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP016009</a>) and O25b: H4-ST131 strain 2017 (Accession no. <a href="https://www.ncbi.nlm.nih.gov/nuccore/CP051617" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP051617</a>) with a query coverage of 100% and a percent identity over 99%, and the plasmid pHS08-175-2-52445 bp-repAIncN in <em>Enterobacter hormaechei</em> (Accession no. <a href="https://www.ncbi.nlm.nih.gov/nuccore/MT010563" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MT010563</a>) with a query coverage of 98% and a percent identity of 96%, and unnamed plasmids in <em>Klebsiella pneumoniae</em> (Accession no. <a href="https://www.ncbi.nlm.nih.gov/nuccore/CP054989" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP054989</a>) and <em>Shigella flexneri</em> (Accession no. <a href="https://www.ncbi.nlm.nih.gov/nuccore/CP055169" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP055169</a>) with a query coverage of 51% and a percent identity of 97% (Table <a href="#Tab6" class="usa-link">6</a>).</p>
<section class="tw xbox font-sm" id="Tab6"><h4 class="obj_head">Table 6.</h4>
<div class="caption p"><p>Genome comparison of pCoMb to other plasmids</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Plasmid Name</th>
<th align="left" colspan="1" rowspan="1">GenBank Accession No.</th>
<th align="left" colspan="1" rowspan="1">Genome Size<br>(bp)</th>
<th align="left" colspan="1" rowspan="1">Query Coverage (%)</th>
<th align="left" colspan="1" rowspan="1">Percent Identity (%)</th>
<th align="left" colspan="1" rowspan="1">Total Score</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">Escherichia coli strain NGF1 plasmid pNGF1_pCol_let_like</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/CP016009" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP016009</a></td>
<td align="center" colspan="1" rowspan="1">8,556</td>
<td align="center" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">100</td>
<td align="center" colspan="1" rowspan="1">15,796</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Escherichia coli O25b: H4-ST131 strain 2017_APHA plasmid pAPHA_2017_2</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/CP051617" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP051617</a></td>
<td align="center" colspan="1" rowspan="1">8,554</td>
<td align="center" colspan="1" rowspan="1">100</td>
<td align="left" colspan="1" rowspan="1">99.69</td>
<td align="center" colspan="1" rowspan="1">15,667</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Enterobacter hormaechei subsp. steigerwaltii strain 08-175 plasmid pHS08-175-2-52445 bp-repAIncN</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/MT010563" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">MT010563</a></td>
<td align="center" colspan="1" rowspan="1">52,445</td>
<td align="center" colspan="1" rowspan="1">98</td>
<td align="left" colspan="1" rowspan="1">95.88</td>
<td align="center" colspan="1" rowspan="1">11,882</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Escherichia coli strain SCU-101 plasmid pSCU-101-1</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/CP051850" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP051850</a></td>
<td align="center" colspan="1" rowspan="1">7,472</td>
<td align="center" colspan="1" rowspan="1">79</td>
<td align="left" colspan="1" rowspan="1">98.51</td>
<td align="center" colspan="1" rowspan="1">12,211</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Escherichia coli pCol-let plasmid</td>
<td align="left" colspan="1" rowspan="1">AF1967335</td>
<td align="center" colspan="1" rowspan="1">5,847</td>
<td align="center" colspan="1" rowspan="1">56</td>
<td align="left" colspan="1" rowspan="1">99.35</td>
<td align="center" colspan="1" rowspan="1">8675</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Klebsiella pneumoniae strain STIN_90 plasmid unnamed5</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/CP054989" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP054989</a></td>
<td align="center" colspan="1" rowspan="1">6,817</td>
<td align="center" colspan="1" rowspan="1">51</td>
<td align="left" colspan="1" rowspan="1">97.04</td>
<td align="center" colspan="1" rowspan="1">6751</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">Shigella flexneri strain STEFF_24 plasmid unnamed2</td>
<td align="left" colspan="1" rowspan="1"><a href="https://www.ncbi.nlm.nih.gov/nuccore/CP055169" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CP055169</a></td>
<td align="center" colspan="1" rowspan="1">6,817</td>
<td align="center" colspan="1" rowspan="1">51</td>
<td align="left" colspan="1" rowspan="1">97.04</td>
<td align="center" colspan="1" rowspan="1">6741</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p116"><p>Footnote: The complete sequence of plasmids was compared using NCBI blasting. Plasmids with a query coverage of less than 50% are not shown, as this is set as the main threshold</p></div></div></section><p id="Par51">Furthermore, the plasmid DNA was extracted from different bacterial strains for PCR analysis. Among the isolated <em>E. coli</em> strains from mice, only LI60C3 contained a plasmid with the <em>mbeA</em> and <em>colY</em> genes; the other strains isolated from mouse colonocytes (HM784C1, HM926C2, and HM936C4) and fecal bacteria (IX2F1, IX2F2, and IX3F1) did not (Fig. <a href="#Fig5" class="usa-link">5</a>C). Moreover, nonpathogenic laboratory strains of DH5α and JM109 also exhibited the absence of plasmids containing <em>mbeA</em> and <em>colY</em> genes (Fig. <a href="#Fig5" class="usa-link">5</a>C). Since the presence of a plasmid pLF82 (Accession no. <a href="https://www.ncbi.nlm.nih.gov/nuccore/CU638872" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CU638872</a>) was reported in the LF82 genome, we stained the plasmid DNA with a bioluminescent dye on the electrophoretic gel. Multiple bands were shown on the gel, including a sharp band at the 10-kb site, which could be pLF82 (Fig. <a href="#Fig5" class="usa-link">5</a>D). According to the NCBI blasting results, the plasmids in LF82 had no sequence homology with pTra and pCoMb in LI60C3.</p></section><section id="Sec24"><h3 class="pmc_sec_title">Unique gene clusters named Epm and Phz Islands in the LI60C3 chromosome</h3>
<p id="Par52">Unique gene clusters were identified in the LI60C3 genome by comparing the whole genome with reference <em>E. coli</em> strains (Table <a href="#Tab7" class="usa-link">7</a>). One of the two unique gene clusters in the LI60C3 genome was designated as the Epm (epimerase) island, which included one copy of genes encoding for capsule biosynthesis protein (<em>capA</em>), UDP-N-acetylglucosamine 4-epimerase <em>(wbgU)</em>, and dTDP-4-dehydrorhamnose 3,5-epimerase <em>(rmlC)</em> (Table <a href="#Tab7" class="usa-link">7</a>). The structures of capsules (K-antigen), lipopolysaccharides (LPS) (O-antigen), flagella (H-antigen), and fimbriae (F-antigen) are essential virulence determinants for serotyping <em>E. coli</em> in relation to clinical infections [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. The <em>capA</em> gene encodes for subtypes of capsular polysaccharides that enable the bacteria to evade or counteract the innate host defense during the early phase of infection. The <em>wbgU</em> and <em>rmlC</em> genes encode for epimerases which are enzymes acting on carbohydrates and are involved in LPS synthesis. In summary, the Epm genes account for synthesizing the particular capsular and LPS subtypes of the bacteria.</p>
<section class="tw xbox font-sm" id="Tab7"><h4 class="obj_head">Table 7.</h4>
<div class="caption p"><p>Specific gene clusters in LI60C3 genome unmatched to reference strains</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Gene</th>
<th align="left" colspan="1" rowspan="1">Protein</th>
<th align="left" colspan="1" rowspan="1">start</th>
<th align="left" colspan="1" rowspan="1">stop</th>
<th align="left" colspan="1" rowspan="1">Strand</th>
<th align="left" colspan="1" rowspan="1">bp</th>
</tr></thead>
<tbody>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>wbgU</em>
</td>
<td align="left" colspan="1" rowspan="1">UDP-N-acetylglucosamine 4-epimerase</td>
<td align="center" colspan="1" rowspan="1">2,932,700</td>
<td align="center" colspan="1" rowspan="1">2,933,704</td>
<td align="left" colspan="1" rowspan="1">+</td>
<td align="center" colspan="1" rowspan="1">1004</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>rmlC</em>
</td>
<td align="left" colspan="1" rowspan="1">dTDP-4-dehydrorhamnose 3,5-epimerase</td>
<td align="center" colspan="1" rowspan="1">2,936,733</td>
<td align="center" colspan="1" rowspan="1">2,937,275</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="center" colspan="1" rowspan="1">542</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>capA</em>
</td>
<td align="left" colspan="1" rowspan="1">Capsule biosynthesis protein CapA</td>
<td align="center" colspan="1" rowspan="1">2,938,144</td>
<td align="center" colspan="1" rowspan="1">2,939,148</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="center" colspan="1" rowspan="1">1004</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>dhbF</em>
</td>
<td align="left" colspan="1" rowspan="1">Dimodular nonribosomal peptide synthase</td>
<td align="center" colspan="1" rowspan="1">3,955,229</td>
<td align="center" colspan="1" rowspan="1">3,956,998</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="center" colspan="1" rowspan="1">1769</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>phzG</em>
</td>
<td align="left" colspan="1" rowspan="1">Phenazine biosynthesis protein PhzG</td>
<td align="center" colspan="1" rowspan="1">3,960,417</td>
<td align="center" colspan="1" rowspan="1">3,961,052</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="center" colspan="1" rowspan="1">635</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>phzF</em>
</td>
<td align="left" colspan="1" rowspan="1">Trans-2,3-dihydro-3-hydroxyanthranilate isomerase</td>
<td align="center" colspan="1" rowspan="1">3,961,066</td>
<td align="center" colspan="1" rowspan="1">3,961,911</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="center" colspan="1" rowspan="1">845</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>phzD</em>
</td>
<td align="left" colspan="1" rowspan="1">Phenazine biosynthesis protein PhzD</td>
<td align="center" colspan="1" rowspan="1">3,964,102</td>
<td align="center" colspan="1" rowspan="1">3,964,722</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="center" colspan="1" rowspan="1">620</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>phzB1</em>
</td>
<td align="left" colspan="1" rowspan="1">Phenazine biosynthesis protein PhzB1</td>
<td align="center" colspan="1" rowspan="1">3,964,798</td>
<td align="center" colspan="1" rowspan="1">3,965,256</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="center" colspan="1" rowspan="1">458</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>ehpR</em>
</td>
<td align="left" colspan="1" rowspan="1">Phenazine antibiotic resistance protein EhpR</td>
<td align="center" colspan="1" rowspan="1">3,966,190</td>
<td align="center" colspan="1" rowspan="1">3,966,576</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="center" colspan="1" rowspan="1">386</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<em>yadA</em>
</td>
<td align="left" colspan="1" rowspan="1">Adhesin YadA</td>
<td align="center" colspan="1" rowspan="1">3,995,109</td>
<td align="center" colspan="1" rowspan="1">3,997,334</td>
<td align="left" colspan="1" rowspan="1">-</td>
<td align="center" colspan="1" rowspan="1">2225</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p121"><p>Footnote: Sequence alignment was determined by NCBI blasting. The nucleotide locations and lengths of the gene in LI60C3 that did not match any reference <em>E. coli</em> were summarized</p></div></div></section><p id="Par54">The second unique gene cluster, named the Phz (phenazine) island, consists of single copies of the <em>dhbF gene</em> and a series of genes that encode proteins involved in phenazine biosynthesis (Table <a href="#Tab7" class="usa-link">7</a>). The <em>dhbF</em> gene encodes for a dimodular nonribosomal peptide synthase. The <em>phzG</em>,<em> phzF</em>,<em> phzD</em>,<em> phzB1</em>, and <em> ehpR</em> genes respectively encode for phenazine biosynthesis protein G, F, D, and B1, and the phenazine antibiotic resistance protein. Phenazines are bacterial secondary metabolites that are oxidases, yielding superoxide and peroxide ions and hydroxyl radicals. These reactions may confer competitive growth advantages for microbes [<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>].</p></section><section id="Sec25"><h3 class="pmc_sec_title">Mouse and human CRC specimens exhibit higher levels of LI60C3 genetic signatures compared to healthy mucosal samples</h3>
<p id="Par55">The presence of LI60C3 genetic signatures was examined in mouse tumor tissues from two established models, including chemically induced AOM/DSS mice and genetic mutant Apc(Min/+) mice, and compared to the intestinal samples from their corresponding controls without tumors (Fig. <a href="#Fig6" class="usa-link">6</a>A-E). The genomic DNA was extracted from mouse intestinal tissues and analyzed by qPCR. The relative expression levels of the <em>E. coli</em>-specific <em>uidA</em> gene, normalized to the bacterial <em>16S rRNA</em>, were higher in the AOM/DSS mice than in the untreated mice, indicating an abundance of <em>E. coli</em> in tumor tissues. The AOM/DSS tumor samples exhibited significantly increased levels of LI60C3-specific genes (i.e., <em>wbgU</em>,<em> capA</em>, <em>dhbF</em>, and <em>ephR</em>) compared to colonic tissues of untreated mice (Fig. <a href="#Fig6" class="usa-link">6</a>F). Similarly, elevated levels of the <em>E. coli uidA</em> gene and LI60C3-specific genes were detected in tumor samples of Apc(Min/+) mice compared with normal tissues of WT(+/+) littermates (Fig. 6GF).</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366417_13099_2025_732_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f57a/12366417/3e498ae32ee5/13099_2025_732_Fig6_HTML.jpg" loading="lazy" id="d33e3665" height="960" width="689" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>LI60C3-specific genes were detected in mouse tumors but are absent in the intestinal tissues of healthy control mice. Unique gene clusters of LI60C3 are designated as the Epm island (<em>wbgU</em> and <em>capA</em> genes) and the Phz island (<em>dhbF</em> and <em>ephR</em> genes). The levels of LI60C3 biomarkers were examined in mouse tumors induced by chemicals and genetic mutation. <strong>(A</strong> and <strong>B)</strong> Representative images of intestinal tumors in the AOM/DSS and Apc(Min/+) mice. Bar: 2 mm (macroscopic images) and 200 μm (histological images with H&amp;E staining). <strong>(C)</strong> Tumor area in the untreated and AOM/DSS mice. <strong>(D)</strong> Tumor area in the WT(+/+) and Apc(Min/+) mice. Data are expressed as mean ± SEM (<em>N</em> = 8–10/group). Each dot represents the data of one mouse. *<em>P</em> &lt; 0.05 <em>vs.</em> untreated or WT(+/+) mice, as calculated by Student’s <em>t</em> test. <strong>(E)</strong> The percentages of tumors with low- and high-grade dysplasia, and carcinoma in the AOM/DSS and Apc(Min/+) mice. <strong>(F</strong> and <strong>G)</strong> Quantitative PCR analysis of <em>uidA</em>,<em> wbgU</em>,<em> capA</em>,<em> dhbF</em>, and <em>ephR</em> levels in the tumor tissues of AOM/DSS and Apc(Min/+) mice. Data are expressed as mean ± SEM (<em>N</em> = 8–10/group). Each dot represents the data of one mouse. *<em>P</em> &lt; 0.05 <em>vs.</em> untreated or WT(+/+) mice, as calculated by Mann-Whitney U test</p></figcaption></figure><p id="Par57">Lastly, the expression of LI60C3-specific genes was validated in the colonic specimens of healthy individuals and patients with CRC. The human tissue RNA was reverse-transcribed to cDNA samples for qPCR analysis. A 3.4-fold higher bacterial amount was observed in the CRC specimens than in the healthy mucosal samples, as identified by a higher ratio of <em>16S</em> to <em>18S rRNA</em> genes in tumors (Fig. <a href="#Fig7" class="usa-link">7</a>A). The <em>E. coli</em>-specific <em>uidA</em> gene levels were slightly increased in the CRC specimens compared with the healthy mucosal samples, albeit without statistical significance. Moreover, the abundance of LI60C3-specific genes (i.e., <em>wbgU</em>,<em> capA</em>,<em> dhbF</em>, and <em>ephR)</em> was 27, 9, 9, and 14 times higher in the CRC specimens than the healthy mucosal samples, respectively (Fig. <a href="#Fig7" class="usa-link">7</a>B-F). When the data were stratified into the four stages of CRC, elevated amounts of <em>capA</em> were noted at tumor stage II and III, and an increase in the <em>ephR</em> level was found at tumor stage III relative to healthy individuals (Table <a href="#Tab8" class="usa-link">8</a>).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366417_13099_2025_732_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/f57a/12366417/9c25db9adc7f/13099_2025_732_Fig7_HTML.jpg" loading="lazy" id="d33e3769" height="651" width="669" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Human CRC specimens exhibit higher levels of LI60C3 genetic signatures compared to healthy mucosal samples. Colonic specimens from healthy individuals (<em>N</em> = 49) and CRC patients (<em>N</em> = 114) were analyzed for bacterial genes. <strong>(A)</strong> The levels of the universal bacterial marker (<em>16S rRNA</em> gene) in CRC specimens were higher than those in healthy mucosal samples. The expression of the bacterial <em>16S rRNA</em> gene was normalized to the levels of the human <em>18S rRNA</em> gene. <strong>(B)</strong> The amount of the <em>E. coli</em>-specific <em>uidA</em> gene in CRC specimens showed an increasing trend compared to healthy mucosal samples. <strong>(C-F)</strong> The levels of LI60C3-specific genes (<em>wbgU</em>, c<em>apA</em>,<em> dhbF</em>, and <em>ephR)</em> were significantly elevated in CRC specimens when compared to healthy mucosal samples. The expression of LI60C3-specific genes was normalized to the levels of <em>16S rRNA</em>. Each dot represents the data of one sample, and the line indicates the mean value of the group. *<em>P</em> &lt; 0.05 vs. Healthy, as calculated by Mann-Whitney U test</p></figcaption></figure><section class="tw xbox font-sm" id="Tab8"><h4 class="obj_head">Table 8.</h4>
<div class="caption p"><p>The LI60C3-specific gene levels in healthy mucosal samples and CRC specimens</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<thead><tr>
<th align="left" colspan="1" rowspan="1">Gene<br>/16S (mean ± SEM)</th>
<th align="left" colspan="1" rowspan="1">Healthy<br>(<em>N</em> = 49)</th>
<th align="left" colspan="2" rowspan="1">CRC<br>(<em>N</em> = 114)</th>
<th align="left" colspan="1" rowspan="1">
<em>P</em> value</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="2" colspan="1">
<strong><em>16 S</em></strong>
</td>
<td align="center" rowspan="2" colspan="1">1.0 ± 0.2</td>
<td align="left" colspan="2" rowspan="1">
<strong>3.4 ± 0.5*</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong><em>P</em></strong> <strong>&lt; 0.0001</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>I</p>
<p>II</p>
<p>III</p>
<p>IV</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p>3.2 ± 1.3</p>
<p>
<strong>2.6 ± 0.6*</strong>
</p>
<p>
<strong>4.2 ± 1.1*</strong>
</p>
<p>4.0 ± 1.6</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p><em>P</em> = 0.097</p>
<p><strong><em>P</em></strong> <strong>= 0.009</strong></p>
<p><strong><em>P</em></strong> <strong>= 0.007</strong></p>
<p><em>P</em> = 0.069</p>
</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<strong><em>uidA</em></strong>
</td>
<td align="center" rowspan="2" colspan="1">1.0 ± 0.3</td>
<td align="left" colspan="2" rowspan="1">
<strong>11.9 ± 4.8*</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong><em>P</em></strong> <strong>= 0.028</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>I</p>
<p>II</p>
<p>III</p>
<p>IV</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p>1.4 ± 0.5</p>
<p>23.2 ± 11.9</p>
<p>11.6 ± 8.3</p>
<p>0.6 ± 10.2</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p><em>P</em> = 0.467</p>
<p><em>P</em> = 0.070</p>
<p><em>P</em> = 0.212</p>
<p><em>P</em> = 0.332</p>
</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<strong><em>wbgU</em></strong>
</td>
<td align="center" rowspan="2" colspan="1">1.0 ± 0.4</td>
<td align="left" colspan="2" rowspan="1">
<strong>27.4 ± 10.1 *</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong><em>P</em></strong> <strong>= 0.010</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>I</p>
<p>II</p>
<p>III</p>
<p>IV</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p>29.3 ± 25.8</p>
<p>36.4 ± 23.3</p>
<p>22.0 ± 13.1</p>
<p>17.7 ± 12.0</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p><em>P</em> = 0.287</p>
<p><em>P</em> = 0.137</p>
<p><em>P</em> = 0.120</p>
<p><em>P</em> = 0.181</p>
</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<strong><em>capA</em></strong>
</td>
<td align="center" rowspan="2" colspan="1">1.0 ± 0.4</td>
<td align="left" colspan="2" rowspan="1">
<strong>9.0 ± 2.3 *</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong><em>P</em></strong> <strong>= 0.001</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>I</p>
<p>II</p>
<p>III</p>
<p>IV</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p>4.0 ± 2.6</p>
<p>
<strong>16.0 ± 6.1 *</strong>
</p>
<p>
<strong>7.8 ± 2.9 *</strong>
</p>
<p>2.8 ± 1.4</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p><em>P</em> = 0.260</p>
<p><em>P</em> = <strong>0.018</strong></p>
<p><em>P</em> = <strong>0.028</strong></p>
<p><em>P</em> = 0.214</p>
</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<strong><em>dhbF</em></strong>
</td>
<td align="center" rowspan="2" colspan="1">1.0 ± 0.5</td>
<td align="left" colspan="2" rowspan="1">
<strong>9.0 ± 3.9 *</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong><em>P</em></strong> <strong>= 0.046</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>I</p>
<p>II</p>
<p>III</p>
<p>IV</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p>1.8 ± 0.5</p>
<p>16.9 ± 11.2</p>
<p>9.9 ± 4.5</p>
<p>1.2 ± 0.4</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p><em>P</em> = 0.275</p>
<p><em>P</em> = 0.166</p>
<p><em>P</em> = 0.059</p>
<p><em>P</em> = 0.796</p>
</td>
</tr>
<tr>
<td align="left" rowspan="2" colspan="1">
<strong><em>ehpR</em></strong>
</td>
<td align="center" rowspan="2" colspan="1">1.0 ± 0.3</td>
<td align="left" colspan="2" rowspan="1">
<strong>14.0 ± 5.4 *</strong>
</td>
<td align="left" colspan="1" rowspan="1">
<strong><em>P</em></strong> <strong>= 0.017</strong>
</td>
</tr>
<tr>
<td align="left" colspan="1" rowspan="1">
<p>I</p>
<p>II</p>
<p>III</p>
<p>IV</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p>3.6 ± 1.9</p>
<p>26.5 ± 14.9</p>
<p>
<strong>7.8 ± 2.8 *</strong>
</p>
<p>11.8 ± 9.8</p>
</td>
<td align="left" colspan="1" rowspan="1">
<p><em>P</em> = 0.200</p>
<p><em>P</em> = 0.096</p>
<p><em>P</em> = <strong>0.020</strong></p>
<p><em>P</em> = 0.286</p>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/Tab8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="_fn_p203"><p>Footnote: Colonic specimens were collected from 49 healthy individuals (age interval: 33–78 years) and 114 patients with colorectal carcinoma (CRC) (age interval: 26–88 years). The number of patients at tumor stages I, II, III, and IV were 20, 39, 33, and 22, respectively. Tissue RNA was reverse-transcribed and analyzed by qPCR. Data presented as fold change of relative expression (mean ± SEM). * <em>P</em> &lt; 0.05 vs. healthy by Student’s <em>t</em> test</p></div></div></section></section></section><section id="Sec26"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par60">Our previous work demonstrated microbial virulence emergence during a key phase of colon cancer initiation, and an invasive <em>E. coli</em> LI60C3 isolated from mouse colonocytes was identified with tumorigenic and colitogenic ability [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. To clarify the genetic characteristics of this pathobiont, the bacterial genomic DNA was sent for whole-genome sequencing. When compared to other <em>E. coli</em> strains with similar invasive capabilities, LI60C3 exhibited a greater ability to promote epithelial proliferation and increase tumor burden in mice. The assembly of long reads revealed a circular chromosome and two extrachromosomal plasmids in LI60C3. Unique gene clusters were identified in the chromosome of LI60C3, with higher levels observed in mouse tumors and human colorectal cancer specimens compared to healthy mucosal samples. Having reported the first complete genome of this bacterial species, the information will serve as a foundation for comparative genome analysis aimed at identifying oncogenic signatures.</p>
<p id="Par61">We conducted a genomic comparison of the protumoral <em>E. coli</em> LI60C3 strain to commensal K-12 strains, AIEC pathobiont strains, and pathogenic strains responsible for intestinal and extraintestinal diseases. The highest mapping rates were observed when comparing the LI60C3 genome to those of AIEC strains, such as LF82 and NC101 (&gt; 90%) followed by UPEC strain<em>s</em> (&gt; 88%). The LI60C3 strain expressed the <em>fim</em> operon encoding Type 1 fimbriae, <em>pap</em> genes for P fimbriae, and <em>lpfA</em> gene encoding long polar fimbriae for adherence, but did not contain the <em>afa</em> gene. The LF82 strain, with Type 1 fimbriae and colitogenic ability, displays the highest genome homology to LI60C3 but has no effect on mouse tumor growth. The NC101 strain contains the <em>fim</em> operon encoding Type 1 fimbriae and <em>afa</em> genes encoding afimbrial adhesins; it also produces genotoxic colibactin to cause DNA damage and promote tumor growth in murine models. Although there is a high degree of sequence homology between the tumorigenic <em>E. coli</em> strains LI60C3 and NC101, it is noteworthy that the polyketide synthase <em>(pks)</em> pathogenicity island is absent in the LI60C3 genome. These findings suggest that LI60C3 may promote tumorigenesis through different mechanisms compared to the colibactin-producing NC101. The UPEC strains, such as 536 and UT189, expressed the <em>fim</em> operon and <em>pap</em> genes for adherence, similar to LI60C3. The sequence homology between LI60C3 and commensal <em>E. coli</em> is approximately 80%, which is higher than its genome similarity to pathogenic EIEC, EHEC, or ETEC strains, ranging from 70 to 78%. The commensal laboratory <em>E. coli</em> strains, MG1655 and W3110, are sub-strains of K-12 isolated from human feces [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>, <a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. The MG1655 strain was chosen as a reference nonpathogenic <em>E. coli</em> since it has been maintained in the laboratory with minimal genetic manipulation and has only been cured of the temperate bacteriophages [<a href="#CR65" class="usa-link" aria-describedby="CR65">65</a>, <a href="#CR66" class="usa-link" aria-describedby="CR66">66</a>]. Other laboratory bacteria derived from the ancestral K-12 strain, such as DH5α and JM109, are not suitable for genome comparison. The DH5α and JM109 strains, used as competent cells with maximized plasmid transformation efficiency, were engineered for <em>recA1</em> and <em>endA1</em> gene mutations to inactivate homologous recombination and intracellular endonuclease [<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a>, <a href="#CR68" class="usa-link" aria-describedby="CR68">68</a>]. In summary, LI60C3 exhibited high sequence homology with AIEC strains, and its genome showed greater ancestral similarity to commensal <em>E. coli</em> than to pathogenic strains.</p>
<p id="Par62">Unique gene clusters were identified in the LI60C3 chromosome sequence, referred to as Epm and Phz islands, which could serve as specific markers of the pathobiont. One of the gene clusters, named Phz (phenazine) island, contains multiple genes <em>phzG</em>,<em> phzF</em>,<em> phzD</em>,<em> phzB1</em>, and <em>ehpR</em> encoding for phenazine families. Phenazines are bacterial secondary metabolites that serve as electron shuttles to alternate terminal acceptors, modify cellular redox states, act as cell signals that regulate gene expression patterns, contribute to biofilm formation and architecture, and enhance bacterial survival. The Epm (epimerase) island, encompassing the <em>capA</em>,<em> wbgU</em>, and <em>rmlC</em> genes, is involved in synthesizing bacterial cell surface structures. Microbial capsules (K-antigen), lipopolysaccharides (O-antigen), flagella (H-antigen), and fimbriae (F-antigen) are essential virulence determinants for serotyping <em>E. coli</em> in clinical infections [<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>]. The <em>capA</em> gene encodes for capsular polysaccharides that enable the bacteria to evade or counteract the innate host defense during the early phase of infection. The capsules could also act as receptors for some bacteriophages containing capsule depolymerase while preventing infection of other phages [<a href="#CR69" class="usa-link" aria-describedby="CR69">69</a>]. Moreover, the <em>wbgU</em> and <em>rmlC</em> genes encoding for epimerases (UDP-N-acetylglucosamine 4-epimerase and dTDP-4-dehydrorhamnose 3,5-epimerase) are involved in the synthesis of the O side-chain of LPS. A few reports have classified UPEC strains in urinary tract and bloodstream infections with O: K:H serotypes [<a href="#CR70" class="usa-link" aria-describedby="CR70">70</a>, <a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. Here, we identified a high sequence similarity of LI60C3 with UPEC strains, i.e., 536 (88%) and UTI89 (86%), which possess the K1 and K2 antigen types of capsules [<a href="#CR71" class="usa-link" aria-describedby="CR71">71</a>]. In summary, the Epm island on the LI60C3 genome is responsible for synthesizing particular capsular and LPS subtypes. These genes could be useful biomarkers to differentiate the existence of pathobionts from other commensal <em>E. coli</em> in fecal samples.</p>
<p id="Par63">As we demonstrated that LI60C3 promoted neoplasia formation in animal models, the validation of pathobiont emergence during the course of tumorigenesis would support their commensal roots in the intestinal microbiota. High levels of LI60C3-specific genes were exhibited in two mouse tumor models and human CRC specimens compared with their healthy counterparts. Previous reports showed a high abundance of <em>E. coli</em> in intestinal polyps and tumor tissues of patients with FAP and CRC [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]; however, microbial genetic signatures to distinguish between commensals and pathobionts are lacking. The presence of pathobiont markers in biopsy samples may confer prognostic value for predicting tumor advancement. Moreover, detection of serum immunoglobulins targeting bacterial outer membrane antigens, such as those encoded by the <em>capA</em> gene, may signal microbiota-associated tumor progression. Such markers could indicate malignant transformation or metastasis, thereby aiding in more accurate assessments of patient outcomes and guiding treatment decisions. Overall, genetic signatures of the tumorigenic LI60C3 strain may be used as cancer predictors.</p>
<p id="Par64">LI60C3 expresses virulence elements that are also present in known AIEC strains (e.g., LF82 and NC101), including Type 1 and P fimbriae, siderophores, non-LEE encoded TTSS effectors, and hemolysin/cytolysin A. The virulent roles of FimH variants, FadA, cyclomodulins, and colibactin were implicated in colitis and tumorigenesis. Point mutations in the <em>fimH</em> gene confer AIEC bacteria a significantly higher ability to adhere to CEACAM-expressing T84 intestinal epithelial cells and to worsen gut inflammation in the infected CEABAC10 transgenic mice expressing human CEACAM receptors [<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a>]. Our previous study has demonstrated that deletion of the <em>fimH</em> or <em>htrA</em> gene partially inhibited the tumorigenic and proliferation-inducing abilities of LI60C3 <sup>21</sup>. However, these cannot be considered pathogenic factors for oncogenicity since all <em>E. coli</em> harbor Type 1 fimbriae for adhesion and htrA proteases to enhance bacterial survival and replication under stress conditions [<a href="#CR73" class="usa-link" aria-describedby="CR73">73</a>]. Other reports indicate that the FadA adhesin/invasin conserved in <em>Fusobacterium nucleatum</em> (an oral anaerobic bacterium with tumorigenic capability) binds to E-cadherins on epithelial cells and activates Wnt/beta-catenin pathways [<a href="#CR74" class="usa-link" aria-describedby="CR74">74</a>, <a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>]. The presence of operons encoding Type 1 and P fimbriae and long polar fimbriae, and the <em>fadA</em> gene (nt. 11436–115699) was observed in the LI60C3 genome. Bacteria-derived cyclomodulins and genotoxins cause DNA damage and cell cycle arrest, which were implicated in oncogenesis [<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a>, <a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>]. Colibactins, encoded by the <em>Clb</em> pathogenicity island, promoted cell senescence and tumor growth in mice [<a href="#CR26" class="usa-link" aria-describedby="CR26">26</a>, <a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>, <a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>]. Consistent with our previous qPCR data, genes encoding for cyclomodulin-related toxin (<em>crt</em>), cytolethal distending toxin (<em>cdt</em>), cytotoxic necrotizing factor (<em>cnf</em>), and colibactin <em>(clb)</em> or polyketide synthase <em>(pks)</em> were absent in the LI60C3 genome. Since none of the aforementioned genotoxins were found, whether the Epm and Phz islands are oncogenic warrants further investigation. Moreover, intestinal inflammation caused by microbial invasion could also be considered as one of the indirect mechanisms of LI60C3-associated tumorigenesis.</p>
<p id="Par65">Other possible virulence factors in the LI60C3 genome that fit in the pathogenicity frameworks may be encoded by genes on the extrachromosomal plasmids. Bacteria utilize plasmids for horizontal gene transfer through structures known as sex pili, which can significantly enhance their survival by enabling the sharing of beneficial traits, such as toxin production and invasive capabilities [<a href="#CR78" class="usa-link" aria-describedby="CR78">78</a>]. This process is particularly prevalent among bacteria residing in various ecosystems, including the intestinal microflora. The evolutionary importance of virulence acquisition through plasmid mobility is essential for microbial adaptation and resilience in diverse environments, and may contribute to the oncogenic ability of pathobionts. Based on the WGS results, the larger plasmid pTra encompassing <em>tra</em> operons resembles many fertility-like plasmids in commensal <em>E. coli</em> [<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>]. The smaller plasmid pCoMb harbors two key proteins, MbeA and ColY, and shares sequence homology with plasmids of pathogenic bacteria such as <em>Enterobacter hormaechei</em>, <em>Klebsiella pneumoniae</em>, and <em>Shigella flexneri.</em> Colicin is one type of bacteriocin produced by <em>E. coli</em> for intra-species competition, by binding to outer membrane proteins (OMPs) and being transported into other bacteria where it exerts cytotoxic effects. These effects include depolarizing the cytoplasmic membrane and executing DNase and RNase activities, which can damage the genetic material of the target bacteria [<a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR79" class="usa-link" aria-describedby="CR79">79</a>]. The <em>mbeA</em> gene encodes for DNA relaxase, a single-strand DNA transesterase enzyme produced by bacteria. This enzyme is capable of site-specific nicks in double-stranded DNA. The relaxase subclass Mbe protein nicks at the origin of transfer sequence to initiate the transfer of a piece of DNA during bacterial conjugation [<a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>]. Notably, colicin and mobilization genes identified in the pCoMb are also found in the chromosome of LI60C3, suggesting plasmid insertion into the bacterial genome. The oncogenic potential of pathobionts as a consequence of plasmid mobility through bacterial conjugation needs further investigation.</p>
<p id="Par66">By keeping the multifactorial nature of oncogenesis in mind, there are several limitations of the observational microbiome studies and experimental pathobiont models. First, small sample sizes in microbiome studies can lead to unreliable results due to increased variability and reduced statistical power. The microbiota composition only represents the microbial factors for disease pathogenesis in a subset of patients. Second, fecal microbiota does not reflect the complete population of intestinal microorganisms, and comparison of fecal composition to the microbiota profiles in mucosal compartments and tumor regions may provide a more comprehensive view of gut ecosystem alteration. Third, longitudinal studies on the transition of resident microbes from beneficial commensals to pathogenic bacteria are scarce in the literature. This gap in research hinders our understanding of how microbial communities evolve in response to stressors or upon genetic material transfer through plasmids. Fourth, bacterial strain heterogeneity and variation of virulence elements may account for differences in tumor outcomes. As live microbes divide rapidly and undergo significant changes in their gene makeup compared to mammalian cells, this characteristic could either amplify or obscure the role of microbial factors in the development of host diseases at later stages. Fifth, there is insufficient mechanistic studies of the oncogenic properties of virulence factors. Functional knockouts to validate bacterial gene function for tumorigenesis are still lacking at this stage. It should be cautioned that deletion of specific virulent elements in <em>E. coli</em> can result in a reduction of the proliferative response and tumor load; however, this does not mean that the virulence factors per se are oncogenic. For example, the fimbriae system is important for bacteria to adhere to the host, and therefore, the functional impairment of these mutants is likely due to the defect in adhesion, rather than their direct involvement in epithelial proliferation and tumorigenesis.</p>
<p id="Par67">In conclusion, the genomic characterization of the tumorigenic pathobiont LI60C3 is pivotal in identifying microbial biomarkers for CRC screening. The genome sequence of LI60C3 presents new opportunities for research aimed at differentiating commensal-derived pathobionts from harmless symbionts within the microbiota. This phenotypic and genetic assessment of commensal-derived pathobionts will enhance our understanding and contribute to the advancement of precision interventions targeting the microbiome.</p></section><section id="Sec27"><h2 class="pmc_sec_title">Supplementary Information</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366417/bin/13099_2025_732_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (688.2KB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank the staff of the imaging and sequencing facility at the First Research Core, Animal Center, Medical Microbiota Core, and Genomic Core in NTUCM for their technical assistance. We thank the staff of the Second Core Lab, Department of Medical Research, National Taiwan University Hospital, for technical support.</p></section><section id="glossary1" class="glossary"><h2 class="pmc_sec_title">Abbreviations</h2>
<dl class="def-list">
<dt>AIEC</dt>
<dd><p id="Par71">Adherent-invasive E. coli</p></dd>
<dt>AOM</dt>
<dd><p id="Par72">Azoxymethane</p></dd>
<dt>Apc</dt>
<dd><p id="Par73">Adenomatous polyposis coli</p></dd>
<dt>BLAST</dt>
<dd><p id="Par74">Basic local alignment search tool</p></dd>
<dt>CARD</dt>
<dd><p id="Par75">Comprehensive Antibiotic Resistance Database</p></dd>
<dt>CDS</dt>
<dd><p id="Par76">Coding sequences</p></dd>
<dt>cdt</dt>
<dd><p id="Par77">Cytolethal distending toxin</p></dd>
<dt>Clb</dt>
<dd><p id="Par78">Colibactin</p></dd>
<dt>CLR</dt>
<dd><p id="Par79">Continuous long-read</p></dd>
<dt>colY</dt>
<dd><p id="Par80">colicin Y</p></dd>
<dt>cnf</dt>
<dd><p id="Par81">Cytotoxic necrotizing factor</p></dd>
<dt>crt</dt>
<dd><p id="Par82">Cyclomodulin-related toxin</p></dd>
<dt>CRC</dt>
<dd><p id="Par83">Colorectal cancer</p></dd>
<dt>DSS</dt>
<dd><p id="Par84">Dextran sodium sulfate</p></dd>
<dt>DAEC</dt>
<dd><p id="Par85">Diffusely adherent E. coli</p></dd>
<dt>EAEC</dt>
<dd><p id="Par86">Enteroaggregative E. coli</p></dd>
<dt>EHEC</dt>
<dd><p id="Par87">Enterohaemorrhagic E. coli</p></dd>
<dt>EIEC</dt>
<dd><p id="Par88">Enteroinvasive E. coli</p></dd>
<dt>EPEC</dt>
<dd><p id="Par89">Enteropathogenic E. coli</p></dd>
<dt>ETEC</dt>
<dd><p id="Par90">Heat-stable and–labile toxin-producing enterotoxigenic E. coli</p></dd>
<dt>Epm</dt>
<dd><p id="Par91">Epimerase</p></dd>
<dt>FAP</dt>
<dd><p id="Par92">Familial adenomatous polyposis</p></dd>
<dt>Fim</dt>
<dd><p id="Par93">Fimbrial adhesion</p></dd>
<dt>HtrA</dt>
<dd><p id="Par94">High temperature requirement A</p></dd>
<dt>IBD</dt>
<dd><p id="Par95">Inflammatory bowel diseases</p></dd>
<dt>LB</dt>
<dd><p id="Par96">Luria-Bertani</p></dd>
<dt>mbeA</dt>
<dd><p id="Par97">DNA mobilization protein A</p></dd>
<dt>MOI</dt>
<dd><p id="Par98">Multiplicity-of-infection</p></dd>
<dt>MSA</dt>
<dd><p id="Par99">Multiple sequence alignment</p></dd>
<dt>oriC</dt>
<dd><p id="Par100">Origin of replication</p></dd>
<dt>PCR</dt>
<dd><p id="Par101">Polymerase chain reaction</p></dd>
<dt>Phz</dt>
<dd><p id="Par102">Phenazine</p></dd>
<dt>pks</dt>
<dd><p id="Par103">Polyketide synthase</p></dd>
<dt>UPEC</dt>
<dd><p id="Par104">Uropathogenic E. coli</p></dd>
<dt>VFDB</dt>
<dd><p id="Par105">Virulence factor database</p></dd>
</dl></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Guarantor of integrity of the entire study, LCY and YHN; study concepts and design: LCY, SCW, LCL, and YHN; data acquisition: YHL, CYH, YCP, and YMH; data analysis/interpretation: YHL, CYH, YCP, and YMH; statistical analysis: YHL, CYH, YCP, and YMH; obtained funding: LCY, SCW, LCL, and YHN; manuscript drafting or revision for important intellectual content, literature research, manuscript editing, and manuscript final version approval: all authors.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>Supported by the National Health Research Institute, Taiwan (NHRI-EX111/112/113-11108BI), National Science and Technology Council, Taiwan (NSTC 113-2320-B002-062-MY3, MoST 110-2320-B-002-011-MY3), Excellent Translational Medicine Research Projects of National Taiwan University College of Medicine (NSCCMOH-216-81), National Taiwan University (NTU-CC-112L895002, 113L892702, 114L890902), and the Higher Education Sprout Project of the Ministry of Education, Taiwan, with funding administered through National Taiwan University under Grant No. 114L91020.</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The whole genome of the bacteria was deposited in GenBank under the accession numbers CP136900, CP136901, and CP136902.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par68">Approval for the human study was granted by the Institutional Review Board (IRB) for NTUH (#202104001RINC). Approval for the animal study was granted by the institutional animal care and use committee (IACUC) of NTUCM (IACUC#20210023, IACUC#20230363).</p></section><section id="FPar2"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par69">Written informed consent was obtained from all study subjects.</p></section><section id="FPar3"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par70">The authors declare no competing interests.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div></div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Abdullah M, Sukartini N, Nursyirwan SA, et al. Gut microbiota profiles in Early- and Late-Onset colorectal cancer: A potential diagnostic biomarker in the future. Digestion. 2021;102:823–32.
</cite> [<a href="https://doi.org/10.1159/000516689" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34433172/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abdullah%20M,%20Sukartini%20N,%20Nursyirwan%20SA,%20et%20al.%20Gut%20microbiota%20profiles%20in%20Early-%20and%20Late-Onset%20colorectal%20cancer:%20A%20potential%20diagnostic%20biomarker%20in%20the%20future.%20Digestion.%202021;102:823%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Yu LC. Microbiota dysbiosis and barrier dysfunction in inflammatory bowel disease and colorectal cancers: exploring a common ground hypothesis. J Biomed Sci. 2018;25:79.
</cite> [<a href="https://doi.org/10.1186/s12929-018-0483-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6234774/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30413188/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20LC.%20Microbiota%20dysbiosis%20and%20barrier%20dysfunction%20in%20inflammatory%20bowel%20disease%20and%20colorectal%20cancers:%20exploring%20a%20common%20ground%20hypothesis.%20J%20Biomed%20Sci.%202018;25:79." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Jandl B, Dighe S, Baumgartner M, et al. Gastrointestinal biofilms: endoscopic detection, disease relevance, and therapeutic strategies. Gastroenterology. 2024;167:1098–e11125.
</cite> [<a href="https://doi.org/10.1053/j.gastro.2024.04.032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38876174/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jandl%20B,%20Dighe%20S,%20Baumgartner%20M,%20et%20al.%20Gastrointestinal%20biofilms:%20endoscopic%20detection,%20disease%20relevance,%20and%20therapeutic%20strategies.%20Gastroenterology.%202024;167:1098%E2%80%93e11125." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Swidsinski A, Weber J, Loening-Baucke V, et al. Spatial organization and composition of the mucosal flora in patients with inflammatory bowel disease. J Clin Microbiol. 2005;43:3380–9.
</cite> [<a href="https://doi.org/10.1128/JCM.43.7.3380-3389.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1169142/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16000463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Swidsinski%20A,%20Weber%20J,%20Loening-Baucke%20V,%20et%20al.%20Spatial%20organization%20and%20composition%20of%20the%20mucosal%20flora%20in%20patients%20with%20inflammatory%20bowel%20disease.%20J%20Clin%20Microbiol.%202005;43:3380%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Kleessen B, Kroesen AJ, Buhr HJ, et al. Mucosal and invading bacteria in patients with inflammatory bowel disease compared with controls. Scand J Gastroenterol. 2002;37:1034–41.
</cite> [<a href="https://doi.org/10.1080/003655202320378220" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12374228/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kleessen%20B,%20Kroesen%20AJ,%20Buhr%20HJ,%20et%20al.%20Mucosal%20and%20invading%20bacteria%20in%20patients%20with%20inflammatory%20bowel%20disease%20compared%20with%20controls.%20Scand%20J%20Gastroenterol.%202002;37:1034%E2%80%9341." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Martin HM, Campbell BJ, Hart CA, et al. Enhanced Escherichia coli adherence and invasion in crohn’s disease and colon cancer. Gastroenterology. 2004;127:80–93.
</cite> [<a href="https://doi.org/10.1053/j.gastro.2004.03.054" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15236175/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martin%20HM,%20Campbell%20BJ,%20Hart%20CA,%20et%20al.%20Enhanced%20Escherichia%20coli%20adherence%20and%20invasion%20in%20crohn%E2%80%99s%20disease%20and%20colon%20cancer.%20Gastroenterology.%202004;127:80%E2%80%9393." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Tuganbaev T, Mor U, Bashiardes S, et al. Diet diurnally regulates small intestinal Microbiome-Epithelial-Immune homeostasis and enteritis. Cell. 2020;182:1441–59. e21.
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.08.027" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32888430/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tuganbaev%20T,%20Mor%20U,%20Bashiardes%20S,%20et%20al.%20Diet%20diurnally%20regulates%20small%20intestinal%20Microbiome-Epithelial-Immune%20homeostasis%20and%20enteritis.%20Cell.%202020;182:1441%E2%80%9359.%20e21." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Noble A, Durant L, Hoyles L, et al. Deficient resident memory T cell and CD8 T cell response to commensals in inflammatory bowel disease. J Crohns Colitis. 2020;14:525–37.
</cite> [<a href="https://doi.org/10.1093/ecco-jcc/jjz175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7242004/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31665283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Noble%20A,%20Durant%20L,%20Hoyles%20L,%20et%20al.%20Deficient%20resident%20memory%20T%20cell%20and%20CD8%20T%20cell%20response%20to%20commensals%20in%20inflammatory%20bowel%20disease.%20J%20Crohns%20Colitis.%202020;14:525%E2%80%9337." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Pai YC, Weng LT, Wei SC, et al. Gut microbial transcytosis induced by tumor necrosis factor-like 1A-dependent activation of a myosin light chain kinase splice variant contributes to IBD. J Crohns Colitis. 2021;15:258–72.
</cite> [<a href="https://doi.org/10.1093/ecco-jcc/jjaa165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7904084/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32770194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pai%20YC,%20Weng%20LT,%20Wei%20SC,%20et%20al.%20Gut%20microbial%20transcytosis%20induced%20by%20tumor%20necrosis%20factor-like%201A-dependent%20activation%20of%20a%20myosin%20light%20chain%20kinase%20splice%20variant%20contributes%20to%20IBD.%20J%20Crohns%20Colitis.%202021;15:258%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Katona BW, Shukla A, Hu W, et al. Microbiota and metabolite-based prediction tool for colonic polyposis with and without a known genetic driver. Gut Microbes. 2025;17:2474141.
</cite> [<a href="https://doi.org/10.1080/19490976.2025.2474141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11913376/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40069167/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Katona%20BW,%20Shukla%20A,%20Hu%20W,%20et%20al.%20Microbiota%20and%20metabolite-based%20prediction%20tool%20for%20colonic%20polyposis%20with%20and%20without%20a%20known%20genetic%20driver.%20Gut%20Microbes.%202025;17:2474141." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Wu YJ, Xiong JF, Zhan CN, et al. Gut microbiota alterations in colorectal adenoma-carcinoma sequence based on 16S rRNA gene sequencing: A systematic review and meta-analysis. Microb Pathog. 2024;195:106889.
</cite> [<a href="https://doi.org/10.1016/j.micpath.2024.106889" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39197689/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20YJ,%20Xiong%20JF,%20Zhan%20CN,%20et%20al.%20Gut%20microbiota%20alterations%20in%20colorectal%20adenoma-carcinoma%20sequence%20based%20on%2016S%20rRNA%20gene%20sequencing:%20A%20systematic%20review%20and%20meta-analysis.%20Microb%20Pathog.%202024;195:106889." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Guven Gulhan U, Nikerel E, Cakir T, et al. Species-level identification of enterotype-specific microbial markers for colorectal cancer and adenoma. Mol Omics. 2024;20:397–416.
</cite> [<a href="https://doi.org/10.1039/d4mo00016a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38780313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guven%20Gulhan%20U,%20Nikerel%20E,%20Cakir%20T,%20et%20al.%20Species-level%20identification%20of%20enterotype-specific%20microbial%20markers%20for%20colorectal%20cancer%20and%20adenoma.%20Mol%20Omics.%202024;20:397%E2%80%93416." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Wang S, Wu M. Decoding the link between microbial secondary metabolites and colorectal cancer. Comput Biol Chem. 2025;115:108372.
</cite> [<a href="https://doi.org/10.1016/j.compbiolchem.2025.108372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39923290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20S,%20Wu%20M.%20Decoding%20the%20link%20between%20microbial%20secondary%20metabolites%20and%20colorectal%20cancer.%20Comput%20Biol%20Chem.%202025;115:108372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Obon-Santacana M, Mas-Lloret J, Bars-Cortina D et al. Meta-Analysis and validation of a colorectal cancer risk prediction model using deep sequenced fecal metagenomes. Cancers (Basel) 2022;14.</cite> [<a href="https://doi.org/10.3390/cancers14174214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9454621/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36077748/" class="usa-link">PubMed</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Swidsinski A, Khilkin M, Kerjaschki D, et al. Association between intraepithelial Escherichia coli and colorectal cancer. Gastroenterology. 1998;115:281–6.
</cite> [<a href="https://doi.org/10.1016/s0016-5085(98)70194-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9679033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Swidsinski%20A,%20Khilkin%20M,%20Kerjaschki%20D,%20et%20al.%20Association%20between%20intraepithelial%20Escherichia%20coli%20and%20colorectal%20cancer.%20Gastroenterology.%201998;115:281%E2%80%936." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Darfeuille-Michaud A, Boudeau J, Bulois P, et al. High prevalence of adherent-invasive Escherichia coli associated with ileal mucosa in crohn’s disease. Gastroenterology. 2004;127:412–21.
</cite> [<a href="https://doi.org/10.1053/j.gastro.2004.04.061" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15300573/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Darfeuille-Michaud%20A,%20Boudeau%20J,%20Bulois%20P,%20et%20al.%20High%20prevalence%20of%20adherent-invasive%20Escherichia%20coli%20associated%20with%20ileal%20mucosa%20in%20crohn%E2%80%99s%20disease.%20Gastroenterology.%202004;127:412%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Kohoutova D, Smajs D, Moravkova P, et al. Escherichia coli strains of phylogenetic group B2 and D and bacteriocin production are associated with advanced colorectal neoplasia. BMC Infect Dis. 2014;14:733.
</cite> [<a href="https://doi.org/10.1186/s12879-014-0733-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4300055/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25540872/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kohoutova%20D,%20Smajs%20D,%20Moravkova%20P,%20et%20al.%20Escherichia%20coli%20strains%20of%20phylogenetic%20group%20B2%20and%20D%20and%20bacteriocin%20production%20are%20associated%20with%20advanced%20colorectal%20neoplasia.%20BMC%20Infect%20Dis.%202014;14:733." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Dejea CM, Fathi P, Craig JM, et al. Patients with Familial adenomatous polyposis harbor colonic biofilms containing tumorigenic bacteria. Science. 2018;359:592–7.
</cite> [<a href="https://doi.org/10.1126/science.aah3648" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5881113/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29420293/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dejea%20CM,%20Fathi%20P,%20Craig%20JM,%20et%20al.%20Patients%20with%20Familial%20adenomatous%20polyposis%20harbor%20colonic%20biofilms%20containing%20tumorigenic%20bacteria.%20Science.%202018;359:592%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Bonnet M, Buc E, Sauvanet P, et al. Colonization of the human gut by E. coli and colorectal cancer risk. Clin Cancer Res. 2014;20:859–67.
</cite> [<a href="https://doi.org/10.1158/1078-0432.CCR-13-1343" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24334760/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bonnet%20M,%20Buc%20E,%20Sauvanet%20P,%20et%20al.%20Colonization%20of%20the%20human%20gut%20by%20E.%20coli%20and%20colorectal%20cancer%20risk.%20Clin%20Cancer%20Res.%202014;20:859%E2%80%9367." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Buc E, Dubois D, Sauvanet P, et al. High prevalence of mucosa-associated E. coli producing cyclomodulin and genotoxin in colon cancer. PLoS ONE. 2013;8:e56964.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0056964" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3572998/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23457644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buc%20E,%20Dubois%20D,%20Sauvanet%20P,%20et%20al.%20High%20prevalence%20of%20mucosa-associated%20E.%20coli%20producing%20cyclomodulin%20and%20genotoxin%20in%20colon%20cancer.%20PLoS%20ONE.%202013;8:e56964." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Yu LC, Wei SC, Li YH, et al. Invasive pathobionts contribute to colon cancer initiation by counterbalancing epithelial antimicrobial responses. Cell Mol Gastroenterol Hepatol. 2022;13:57–79.
</cite> [<a href="https://doi.org/10.1016/j.jcmgh.2021.08.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8600093/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34418587/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20LC,%20Wei%20SC,%20Li%20YH,%20et%20al.%20Invasive%20pathobionts%20contribute%20to%20colon%20cancer%20initiation%20by%20counterbalancing%20epithelial%20antimicrobial%20responses.%20Cell%20Mol%20Gastroenterol%20Hepatol.%202022;13:57%E2%80%9379." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Pai YC, Li YH, Turner JR, et al. Transepithelial barrier dysfunction drives microbiota dysbiosis to initiate epithelial Clock-driven inflammation. J Crohns Colitis. 2023;17:1471–88.
</cite> [<a href="https://doi.org/10.1093/ecco-jcc/jjad064" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10588795/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37004200/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pai%20YC,%20Li%20YH,%20Turner%20JR,%20et%20al.%20Transepithelial%20barrier%20dysfunction%20drives%20microbiota%20dysbiosis%20to%20initiate%20epithelial%20Clock-driven%20inflammation.%20J%20Crohns%20Colitis.%202023;17:1471%E2%80%9388." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Carvalho FA, Barnich N, Sivignon A, et al. Crohn’s disease adherent-invasive Escherichia coli colonize and induce strong gut inflammation in Transgenic mice expressing human CEACAM. J Exp Med. 2009;206:2179–89.
</cite> [<a href="https://doi.org/10.1084/jem.20090741" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2757893/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19737864/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Carvalho%20FA,%20Barnich%20N,%20Sivignon%20A,%20et%20al.%20Crohn%E2%80%99s%20disease%20adherent-invasive%20Escherichia%20coli%20colonize%20and%20induce%20strong%20gut%20inflammation%20in%20Transgenic%20mice%20expressing%20human%20CEACAM.%20J%20Exp%20Med.%202009;206:2179%E2%80%9389." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Chevarin C, Xu Z, Martin L, et al. Comparison of crohn’s disease-associated adherent-invasive Escherichia coli (AIEC) from France and Hong kong: results from the Pacific study. Gut Microbes. 2024;16:2431645.
</cite> [<a href="https://doi.org/10.1080/19490976.2024.2431645" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11601055/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39587720/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chevarin%20C,%20Xu%20Z,%20Martin%20L,%20et%20al.%20Comparison%20of%20crohn%E2%80%99s%20disease-associated%20adherent-invasive%20Escherichia%20coli%20(AIEC)%20from%20France%20and%20Hong%20kong:%20results%20from%20the%20Pacific%20study.%20Gut%20Microbes.%202024;16:2431645." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Kim J, Zhang J, Kinch L, et al. Genetic and microbial analysis of invasiveness for Escherichia coli strains associated with inflammatory bowel disease. Cell Mol Gastroenterol Hepatol. 2025;19:101451.
</cite> [<a href="https://doi.org/10.1016/j.jcmgh.2024.101451" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11879602/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40437706/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim%20J,%20Zhang%20J,%20Kinch%20L,%20et%20al.%20Genetic%20and%20microbial%20analysis%20of%20invasiveness%20for%20Escherichia%20coli%20strains%20associated%20with%20inflammatory%20bowel%20disease.%20Cell%20Mol%20Gastroenterol%20Hepatol.%202025;19:101451." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Arthur JC, Perez-Chanona E, Muhlbauer M, et al. Intestinal inflammation targets cancer-inducing activity of the microbiota. Science. 2012;338:120–3.
</cite> [<a href="https://doi.org/10.1126/science.1224820" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3645302/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22903521/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arthur%20JC,%20Perez-Chanona%20E,%20Muhlbauer%20M,%20et%20al.%20Intestinal%20inflammation%20targets%20cancer-inducing%20activity%20of%20the%20microbiota.%20Science.%202012;338:120%E2%80%933." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Arthur JC, Gharaibeh RZ, Muhlbauer M, et al. Microbial genomic analysis reveals the essential role of inflammation in bacteria-induced colorectal cancer. Nat Commun. 2014;5:4724.
</cite> [<a href="https://doi.org/10.1038/ncomms5724" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4155410/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25182170/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Arthur%20JC,%20Gharaibeh%20RZ,%20Muhlbauer%20M,%20et%20al.%20Microbial%20genomic%20analysis%20reveals%20the%20essential%20role%20of%20inflammation%20in%20bacteria-induced%20colorectal%20cancer.%20Nat%20Commun.%202014;5:4724." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Thiele Orberg E, Fan H, Tam AJ, et al. The myeloid immune signature of enterotoxigenic bacteroides fragilis-induced murine colon tumorigenesis. Mucosal Immunol. 2017;10:421–33.
</cite> [<a href="https://doi.org/10.1038/mi.2016.53" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5159334/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27301879/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thiele%20Orberg%20E,%20Fan%20H,%20Tam%20AJ,%20et%20al.%20The%20myeloid%20immune%20signature%20of%20enterotoxigenic%20bacteroides%20fragilis-induced%20murine%20colon%20tumorigenesis.%20Mucosal%20Immunol.%202017;10:421%E2%80%9333." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Chen Y, Peng Y, Yu J, et al. Invasive Fusobacterium nucleatum activates beta-catenin signaling in colorectal cancer via a TLR4/P-PAK1 cascade. Oncotarget. 2017;8:31802–14.
</cite> [<a href="https://doi.org/10.18632/oncotarget.15992" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5458249/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28423670/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20Y,%20Peng%20Y,%20Yu%20J,%20et%20al.%20Invasive%20Fusobacterium%20nucleatum%20activates%20beta-catenin%20signaling%20in%20colorectal%20cancer%20via%20a%20TLR4/P-PAK1%20cascade.%20Oncotarget.%202017;8:31802%E2%80%9314." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Cougnoux A, Dalmasso G, Martinez R, et al. Bacterial genotoxin colibactin promotes colon tumour growth by inducing a senescence-associated secretory phenotype. Gut. 2014;63:1932–42.
</cite> [<a href="https://doi.org/10.1136/gutjnl-2013-305257" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24658599/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cougnoux%20A,%20Dalmasso%20G,%20Martinez%20R,%20et%20al.%20Bacterial%20genotoxin%20colibactin%20promotes%20colon%20tumour%20growth%20by%20inducing%20a%20senescence-associated%20secretory%20phenotype.%20Gut.%202014;63:1932%E2%80%9342." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Dalmasso G, Cougnoux A, Delmas J, et al. The bacterial genotoxin colibactin promotes colon tumor growth by modifying the tumor microenvironment. Gut Microbes. 2014;5:675–80.
</cite> [<a href="https://doi.org/10.4161/19490976.2014.969989" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4615906/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25483338/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dalmasso%20G,%20Cougnoux%20A,%20Delmas%20J,%20et%20al.%20The%20bacterial%20genotoxin%20colibactin%20promotes%20colon%20tumor%20growth%20by%20modifying%20the%20tumor%20microenvironment.%20Gut%20Microbes.%202014;5:675%E2%80%9380." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Raisch J, Buc E, Bonnet M, et al. Colon cancer-associated B2 Escherichia coli colonize gut mucosa and promote cell proliferation. World J Gastroenterol. 2014;20:6560–72.
</cite> [<a href="https://doi.org/10.3748/wjg.v20.i21.6560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4047342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24914378/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raisch%20J,%20Buc%20E,%20Bonnet%20M,%20et%20al.%20Colon%20cancer-associated%20B2%20Escherichia%20coli%20colonize%20gut%20mucosa%20and%20promote%20cell%20proliferation.%20World%20J%20Gastroenterol.%202014;20:6560%E2%80%9372." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Raisch J, Rolhion N, Dubois A, et al. Intracellular colon cancer-associated Escherichia coli promote protumoral activities of human macrophages by inducing sustained COX-2 expression. Lab Invest. 2015;95:296–307.
</cite> [<a href="https://doi.org/10.1038/labinvest.2014.161" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25545478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Raisch%20J,%20Rolhion%20N,%20Dubois%20A,%20et%20al.%20Intracellular%20colon%20cancer-associated%20Escherichia%20coli%20promote%20protumoral%20activities%20of%20human%20macrophages%20by%20inducing%20sustained%20COX-2%20expression.%20Lab%20Invest.%202015;95:296%E2%80%93307." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Rahmouni O, Vignal C, Titecat M, et al. High carriage of adherent invasive E. coli in wildlife and healthy individuals. Gut Pathog. 2018;10:23.
</cite> [<a href="https://doi.org/10.1186/s13099-018-0248-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6001069/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29946365/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rahmouni%20O,%20Vignal%20C,%20Titecat%20M,%20et%20al.%20High%20carriage%20of%20adherent%20invasive%20E.%20coli%20in%20wildlife%20and%20healthy%20individuals.%20Gut%20Pathog.%202018;10:23." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Mansan-Almeida R, Pereira AL, Giugliano LG. Diffusely adherent Escherichia coli strains isolated from children and adults constitute two different populations. BMC Microbiol. 2013;13:22.
</cite> [<a href="https://doi.org/10.1186/1471-2180-13-22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3577467/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23374248/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mansan-Almeida%20R,%20Pereira%20AL,%20Giugliano%20LG.%20Diffusely%20adherent%20Escherichia%20coli%20strains%20isolated%20from%20children%20and%20adults%20constitute%20two%20different%20populations.%20BMC%20Microbiol.%202013;13:22." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Singh T, Das S, Ramachandran VG, et al. Spectrum of diarrhoeagenic Escherichia coli in paediatric population suffering from diarrhoea and as commensals in healthy children. Indian J Med Microbiol. 2017;35:204–10.
</cite> [<a href="https://doi.org/10.4103/ijmm.IJMM_16_21" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28681807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Singh%20T,%20Das%20S,%20Ramachandran%20VG,%20et%20al.%20Spectrum%20of%20diarrhoeagenic%20Escherichia%20coli%20in%20paediatric%20population%20suffering%20from%20diarrhoea%20and%20as%20commensals%20in%20healthy%20children.%20Indian%20J%20Med%20Microbiol.%202017;35:204%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Barrios-Villa E, de la Martinez CF, Lozano-Zarain P, et al. Comparative genomics of a subset of adherent/invasive Escherichia coli strains isolated from individuals without inflammatory bowel disease. Genomics. 2020;112:1813–20.
</cite> [<a href="https://doi.org/10.1016/j.ygeno.2019.10.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31689478/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Barrios-Villa%20E,%20de%20la%20Martinez%20CF,%20Lozano-Zarain%20P,%20et%20al.%20Comparative%20genomics%20of%20a%20subset%20of%20adherent/invasive%20Escherichia%20coli%20strains%20isolated%20from%20individuals%20without%20inflammatory%20bowel%20disease.%20Genomics.%202020;112:1813%E2%80%9320." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Camprubi-Font C, Ewers C, Lopez-Siles M, et al. Genetic and phenotypic features to screen for putative Adherent-Invasive Escherichia coli. Front Microbiol. 2019;10:108.
</cite> [<a href="https://doi.org/10.3389/fmicb.2019.00108" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6393329/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30846972/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Camprubi-Font%20C,%20Ewers%20C,%20Lopez-Siles%20M,%20et%20al.%20Genetic%20and%20phenotypic%20features%20to%20screen%20for%20putative%20Adherent-Invasive%20Escherichia%20coli.%20Front%20Microbiol.%202019;10:108." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Meyer J, Roos E, Combescure C, et al. Mapping of aetiologies of gastroenteritis: a systematic review and meta-analysis of pathogens identified using a multiplex screening array. Scand J Gastroenterol. 2020;55:1405–10.
</cite> [<a href="https://doi.org/10.1080/00365521.2020.1839128" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33147077/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Meyer%20J,%20Roos%20E,%20Combescure%20C,%20et%20al.%20Mapping%20of%20aetiologies%20of%20gastroenteritis:%20a%20systematic%20review%20and%20meta-analysis%20of%20pathogens%20identified%20using%20a%20multiplex%20screening%20array.%20Scand%20J%20Gastroenterol.%202020;55:1405%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Muenzner P, Kengmo Tchoupa A, Klauser B, et al. Uropathogenic E. coli exploit CEA to promote colonization of the urogenital tract mucosa. PLoS Pathog. 2016;12:e1005608.
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1005608" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4865239/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27171273/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Muenzner%20P,%20Kengmo%20Tchoupa%20A,%20Klauser%20B,%20et%20al.%20Uropathogenic%20E.%20coli%20exploit%20CEA%20to%20promote%20colonization%20of%20the%20urogenital%20tract%20mucosa.%20PLoS%20Pathog.%202016;12:e1005608." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Denizot J, Sivignon A, Barreau F, et al. Adherent-invasive Escherichia coli induce claudin-2 expression and barrier defect in CEABAC10 mice and crohn’s disease patients. Inflamm Bowel Dis. 2012;18:294–304.
</cite> [<a href="https://doi.org/10.1002/ibd.21787" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21688348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Denizot%20J,%20Sivignon%20A,%20Barreau%20F,%20et%20al.%20Adherent-invasive%20Escherichia%20coli%20induce%20claudin-2%20expression%20and%20barrier%20defect%20in%20CEABAC10%20mice%20and%20crohn%E2%80%99s%20disease%20patients.%20Inflamm%20Bowel%20Dis.%202012;18:294%E2%80%93304." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Tomkovich S, Yang Y, Winglee K, et al. Locoregional effects of microbiota in a preclinical model of colon carcinogenesis. Cancer Res. 2017;77:2620–32.
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-16-3472" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5468752/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28416491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tomkovich%20S,%20Yang%20Y,%20Winglee%20K,%20et%20al.%20Locoregional%20effects%20of%20microbiota%20in%20a%20preclinical%20model%20of%20colon%20carcinogenesis.%20Cancer%20Res.%202017;77:2620%E2%80%9332." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Wang J, Bleich RM, Zarmer S, et al. Long-read sequencing to interrogate strain-level variation among adherent-invasive Escherichia coli isolated from human intestinal tissue. PLoS ONE. 2021;16:e0259141.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0259141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8553045/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34710159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang%20J,%20Bleich%20RM,%20Zarmer%20S,%20et%20al.%20Long-read%20sequencing%20to%20interrogate%20strain-level%20variation%20among%20adherent-invasive%20Escherichia%20coli%20isolated%20from%20human%20intestinal%20tissue.%20PLoS%20ONE.%202021;16:e0259141." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Kuo WT, Lee TC, Yang HY, et al. LPS receptor subunits have antagonistic roles in epithelial apoptosis and colonic carcinogenesis. Cell Death Differ. 2015;22:1590–604.
</cite> [<a href="https://doi.org/10.1038/cdd.2014.240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4563778/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25633197/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuo%20WT,%20Lee%20TC,%20Yang%20HY,%20et%20al.%20LPS%20receptor%20subunits%20have%20antagonistic%20roles%20in%20epithelial%20apoptosis%20and%20colonic%20carcinogenesis.%20Cell%20Death%20Differ.%202015;22:1590%E2%80%93604." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Kuo WT, Lee TC, Yu LC. Eritoran suppresses colon cancer by altering a functional balance in Toll-like receptors that bind lipopolysaccharide. Cancer Res. 2016;76:4684–95.
</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-16-0172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27328732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kuo%20WT,%20Lee%20TC,%20Yu%20LC.%20Eritoran%20suppresses%20colon%20cancer%20by%20altering%20a%20functional%20balance%20in%20Toll-like%20receptors%20that%20bind%20lipopolysaccharide.%20Cancer%20Res.%202016;76:4684%E2%80%9395." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Li L, Li X, Zhong W, et al. Gut microbiota from colorectal cancer patients enhances the progression of intestinal adenoma in Apc(min/+) mice. EBioMedicine. 2019;48:301–15.
</cite> [<a href="https://doi.org/10.1016/j.ebiom.2019.09.021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6838415/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31594750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20L,%20Li%20X,%20Zhong%20W,%20et%20al.%20Gut%20microbiota%20from%20colorectal%20cancer%20patients%20enhances%20the%20progression%20of%20intestinal%20adenoma%20in%20Apc(min/+)%20mice.%20EBioMedicine.%202019;48:301%E2%80%9315." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>She MP, Hsieh YT, Lin LY, et al. Differential roles of serotonin receptor subtypes in regulation of neurotrophin receptor expression and intestinal hypernociception. Histol Histopathol. 2024;39:903–19.
</cite> [<a href="https://doi.org/10.14670/HH-18-687" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38108436/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?She%20MP,%20Hsieh%20YT,%20Lin%20LY,%20et%20al.%20Differential%20roles%20of%20serotonin%20receptor%20subtypes%20in%20regulation%20of%20neurotrophin%20receptor%20expression%20and%20intestinal%20hypernociception.%20Histol%20Histopathol.%202024;39:903%E2%80%9319." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Huang CY, Pai YC, Yu LC. Glucose conferred Irinotecan chemoresistance through divergent actions of pyruvate and ATP in cell death and proliferation of colorectal cancer. Oncology. 2022;100:555–68.
</cite> [<a href="https://doi.org/10.1159/000525977" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35850102/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20CY,%20Pai%20YC,%20Yu%20LC.%20Glucose%20conferred%20Irinotecan%20chemoresistance%20through%20divergent%20actions%20of%20pyruvate%20and%20ATP%20in%20cell%20death%20and%20proliferation%20of%20colorectal%20cancer.%20Oncology.%202022;100:555%E2%80%9368." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Wu LL, Peng WH, Kuo WT, et al. Commensal bacterial endocytosis in epithelial cells is dependent on myosin light chain Kinase-Activated brush border fanning by Interferon-gamma. Am J Pathol. 2014;184:2260–74.
</cite> [<a href="https://doi.org/10.1016/j.ajpath.2014.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4188866/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24911373/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20LL,%20Peng%20WH,%20Kuo%20WT,%20et%20al.%20Commensal%20bacterial%20endocytosis%20in%20epithelial%20cells%20is%20dependent%20on%20myosin%20light%20chain%20Kinase-Activated%20brush%20border%20fanning%20by%20Interferon-gamma.%20Am%20J%20Pathol.%202014;184:2260%E2%80%9374." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Yu LC, Shih YA, Wu LL, et al. Enteric dysbiosis promotes antibiotic-resistant bacterial infection: systemic dissemination of resistant and commensal bacteria through epithelial transcytosis. Am J Physiol Gastrointest Liver Physiol. 2014;307:G824–35.
</cite> [<a href="https://doi.org/10.1152/ajpgi.00070.2014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4214854/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25059827/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yu%20LC,%20Shih%20YA,%20Wu%20LL,%20et%20al.%20Enteric%20dysbiosis%20promotes%20antibiotic-resistant%20bacterial%20infection:%20systemic%20dissemination%20of%20resistant%20and%20commensal%20bacteria%20through%20epithelial%20transcytosis.%20Am%20J%20Physiol%20Gastrointest%20Liver%20Physiol.%202014;307:G824%E2%80%9335." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Touchon M, Hoede C, Tenaillon O, et al. Organised genome dynamics in the Escherichia coli species results in highly diverse adaptive paths. PLoS Genet. 2009;5:e1000344.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1000344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2617782/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19165319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Touchon%20M,%20Hoede%20C,%20Tenaillon%20O,%20et%20al.%20Organised%20genome%20dynamics%20in%20the%20Escherichia%20coli%20species%20results%20in%20highly%20diverse%20adaptive%20paths.%20PLoS%20Genet.%202009;5:e1000344." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Gurevich A, Saveliev V, Vyahhi N, et al. QUAST: quality assessment tool for genome assemblies. Bioinformatics. 2013;29:1072–5.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btt086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3624806/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23422339/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gurevich%20A,%20Saveliev%20V,%20Vyahhi%20N,%20et%20al.%20QUAST:%20quality%20assessment%20tool%20for%20genome%20assemblies.%20Bioinformatics.%202013;29:1072%E2%80%935." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Chen L, Yang J, Yu J, et al. VFDB: a reference database for bacterial virulence factors. Nucleic Acids Res. 2005;33:D325–8.
</cite> [<a href="https://doi.org/10.1093/nar/gki008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC539962/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15608208/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen%20L,%20Yang%20J,%20Yu%20J,%20et%20al.%20VFDB:%20a%20reference%20database%20for%20bacterial%20virulence%20factors.%20Nucleic%20Acids%20Res.%202005;33:D325%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Liu B, Zheng D, Zhou S, et al. VFDB 2022: a general classification scheme for bacterial virulence factors. Nucleic Acids Res. 2022;50:D912–7.
</cite> [<a href="https://doi.org/10.1093/nar/gkab1107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8728188/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34850947/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu%20B,%20Zheng%20D,%20Zhou%20S,%20et%20al.%20VFDB%202022:%20a%20general%20classification%20scheme%20for%20bacterial%20virulence%20factors.%20Nucleic%20Acids%20Res.%202022;50:D912%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Alcock BP, Raphenya AR, Lau TTY, et al. CARD 2020: antibiotic resistome surveillance with the comprehensive antibiotic resistance database. Nucleic Acids Res. 2020;48:D517–25.
</cite> [<a href="https://doi.org/10.1093/nar/gkz935" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7145624/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31665441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Alcock%20BP,%20Raphenya%20AR,%20Lau%20TTY,%20et%20al.%20CARD%202020:%20antibiotic%20resistome%20surveillance%20with%20the%20comprehensive%20antibiotic%20resistance%20database.%20Nucleic%20Acids%20Res.%202020;48:D517%E2%80%9325." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Tamura K, Stecher G, Kumar S. MEGA11: molecular evolutionary genetics analysis version 11. Mol Biol Evol. 2021;38:3022–7.
</cite> [<a href="https://doi.org/10.1093/molbev/msab120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8233496/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33892491/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tamura%20K,%20Stecher%20G,%20Kumar%20S.%20MEGA11:%20molecular%20evolutionary%20genetics%20analysis%20version%2011.%20Mol%20Biol%20Evol.%202021;38:3022%E2%80%937." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Clermont O, Bonacorsi S, Bingen E. Rapid and simple determination of the Escherichia coli phylogenetic group. Appl Environ Microbiol. 2000;66:4555–8.
</cite> [<a href="https://doi.org/10.1128/aem.66.10.4555-4558.2000" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC92342/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11010916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clermont%20O,%20Bonacorsi%20S,%20Bingen%20E.%20Rapid%20and%20simple%20determination%20of%20the%20Escherichia%20coli%20phylogenetic%20group.%20Appl%20Environ%20Microbiol.%202000;66:4555%E2%80%938." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Walker DI, McQuillan J, Taiwo M, et al. A highly specific Escherichia coli qPCR and its comparison with existing methods for environmental waters. Water Res. 2017;126:101–10.
</cite> [<a href="https://doi.org/10.1016/j.watres.2017.08.032" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28930669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Walker%20DI,%20McQuillan%20J,%20Taiwo%20M,%20et%20al.%20A%20highly%20specific%20Escherichia%20coli%20qPCR%20and%20its%20comparison%20with%20existing%20methods%20for%20environmental%20waters.%20Water%20Res.%202017;126:101%E2%80%9310." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Tanimoto Y, Tamai S, Matsuzaki T et al. Diffusely adherent Escherichia coli strains isolated from healthy carriers suppress cytokine secretions of epithelial cells stimulated by inflammatory substances. Infect Immun 2019;87.</cite> [<a href="https://doi.org/10.1128/IAI.00683-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6300626/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30323026/" class="usa-link">PubMed</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Fujihara S, Arikawa K, Aota T, et al. Prevalence and properties of diarrheagenic Escherichia coli among healthy individuals in Osaka city, Japan. Jpn J Infect Dis. 2009;62:318–23.
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/19628916/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fujihara%20S,%20Arikawa%20K,%20Aota%20T,%20et%20al.%20Prevalence%20and%20properties%20of%20diarrheagenic%20Escherichia%20coli%20among%20healthy%20individuals%20in%20Osaka%20city,%20Japan.%20Jpn%20J%20Infect%20Dis.%202009;62:318%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Lindsey RL, Batra D, Smith P et al. PacBio genome sequences of Escherichia coli serotype O157:H7, diffusely adherent E. coli, and Salmonella enterica strains, all carrying plasmids with an mcr-1 resistance gene. Microbiol Resour Announc 2018;7.</cite> [<a href="https://doi.org/10.1128/MRA.01025-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6214480/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30393782/" class="usa-link">PubMed</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Virolle C, Goldlust K, Djermoun S et al. Plasmid transfer by conjugation in Gram-Negative bacteria: from the cellular to the community level. Genes (Basel) 2020;11.</cite> [<a href="https://doi.org/10.3390/genes11111239" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7690428/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33105635/" class="usa-link">PubMed</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Whitfield C. Structure and assembly of Escherichia coli capsules. EcoSal Plus 2009;3.</cite> [<a href="https://doi.org/10.1128/ecosalplus.4.7.3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26443756/" class="usa-link">PubMed</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Ciemniecki JA, Newman DK. The potential for Redox-Active metabolites to enhance or unlock anaerobic survival metabolisms in aerobes. J Bacteriol 2020;202.</cite> [<a href="https://doi.org/10.1128/JB.00797-19" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7221258/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32071098/" class="usa-link">PubMed</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Blattner FR, Plunkett G 3rd, Bloch CA, et al. The complete genome sequence of Escherichia coli K-12. Science. 1997;277:1453–62.</cite> [<a href="https://doi.org/10.1126/science.277.5331.1453" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9278503/" class="usa-link">PubMed</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Hayashi K, Morooka N, Yamamoto Y, et al. Highly accurate genome sequences of Escherichia coli K-12 strains MG1655 and W3110. Mol Syst Biol. 2006;2:20060007.</cite> [<a href="https://doi.org/10.1038/msb4100049" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1681481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16738553/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hayashi%20K,%20Morooka%20N,%20Yamamoto%20Y,%20et%20al.%20Highly%20accurate%20genome%20sequences%20of%20Escherichia%20coli%20K-12%20strains%20MG1655%20and%20W3110.%20Mol%20Syst%20Biol.%202006;2:20060007." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Chen J, Li Y, Zhang K et al. Whole-Genome sequence of Phage-Resistant strain Escherichia coli DH5alpha. Genome Announc 2018;6.</cite> [<a href="https://doi.org/10.1128/genomeA.00097-18" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5843723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29519826/" class="usa-link">PubMed</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Anton BP, Raleigh EA. Complete genome sequence of NEB 5-alpha, a derivative of Escherichia coli K-12 DH5alpha. Genome Announc 2016;4.</cite> [<a href="https://doi.org/10.1128/genomeA.01245-16" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5105096/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27834703/" class="usa-link">PubMed</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Whitfield C. Biosynthesis and assembly of capsular polysaccharides in Escherichia coli. Annu Rev Biochem. 2006;75:39–68.
</cite> [<a href="https://doi.org/10.1146/annurev.biochem.75.103004.142545" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16756484/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Whitfield%20C.%20Biosynthesis%20and%20assembly%20of%20capsular%20polysaccharides%20in%20Escherichia%20coli.%20Annu%20Rev%20Biochem.%202006;75:39%E2%80%9368." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Johnson JR, Stell AL, O’Bryan TT, et al. Global molecular epidemiology of the O15:K52:H1 extraintestinal pathogenic Escherichia coli clonal group: evidence of distribution beyond Europe. J Clin Microbiol. 2002;40:1913–23.
</cite> [<a href="https://doi.org/10.1128/JCM.40.6.1913-1923.2002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC130829/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12037043/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Johnson%20JR,%20Stell%20AL,%20O%E2%80%99Bryan%20TT,%20et%20al.%20Global%20molecular%20epidemiology%20of%20the%20O15:K52:H1%20extraintestinal%20pathogenic%20Escherichia%20coli%20clonal%20group:%20evidence%20of%20distribution%20beyond%20Europe.%20J%20Clin%20Microbiol.%202002;40:1913%E2%80%9323." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Goh KGK, Phan MD, Forde BM et al. Genome-Wide discovery of genes required for capsule production by uropathogenic Escherichia coli. mBio 2017;8.</cite> [<a href="https://doi.org/10.1128/mBio.01558-17" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5654933/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29066548/" class="usa-link">PubMed</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Dreux N, Denizot J, Martinez-Medina M, et al. Point mutations in FimH adhesin of crohn’s disease-associated adherent-invasive Escherichia coli enhance intestinal inflammatory response. PLoS Pathog. 2013;9:e1003141.
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1003141" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3554634/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23358328/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dreux%20N,%20Denizot%20J,%20Martinez-Medina%20M,%20et%20al.%20Point%20mutations%20in%20FimH%20adhesin%20of%20crohn%E2%80%99s%20disease-associated%20adherent-invasive%20Escherichia%20coli%20enhance%20intestinal%20inflammatory%20response.%20PLoS%20Pathog.%202013;9:e1003141." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Bringer MA, Barnich N, Glasser AL, et al. HtrA stress protein is involved in intramacrophagic replication of adherent and invasive Escherichia coli strain LF82 isolated from a patient with crohn’s disease. Infect Immun. 2005;73:712–21.
</cite> [<a href="https://doi.org/10.1128/IAI.73.2.712-721.2005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC546957/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15664909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bringer%20MA,%20Barnich%20N,%20Glasser%20AL,%20et%20al.%20HtrA%20stress%20protein%20is%20involved%20in%20intramacrophagic%20replication%20of%20adherent%20and%20invasive%20Escherichia%20coli%20strain%20LF82%20isolated%20from%20a%20patient%20with%20crohn%E2%80%99s%20disease.%20Infect%20Immun.%202005;73:712%E2%80%9321." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Rubinstein MR, Wang X, Liu W, et al. Fusobacterium nucleatum promotes colorectal carcinogenesis by modulating E-cadherin/beta-catenin signaling via its FadA adhesin. Cell Host Microbe. 2013;14:195–206.
</cite> [<a href="https://doi.org/10.1016/j.chom.2013.07.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3770529/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23954158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rubinstein%20MR,%20Wang%20X,%20Liu%20W,%20et%20al.%20Fusobacterium%20nucleatum%20promotes%20colorectal%20carcinogenesis%20by%20modulating%20E-cadherin/beta-catenin%20signaling%20via%20its%20FadA%20adhesin.%20Cell%20Host%20Microbe.%202013;14:195%E2%80%93206." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Britton TA, Lee JH, Chang C, et al. Inactivation of the Fusobacterium nucleatum Rnf complex reduces FadA-mediated amyloid formation and tumor development. mBio. 2025;16:e0103225.
</cite> [<a href="https://doi.org/10.1128/mbio.01032-25" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC12153266/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40401912/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Britton%20TA,%20Lee%20JH,%20Chang%20C,%20et%20al.%20Inactivation%20of%20the%20Fusobacterium%20nucleatum%20Rnf%20complex%20reduces%20FadA-mediated%20amyloid%20formation%20and%20tumor%20development.%20mBio.%202025;16:e0103225." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Zhang Z, Aung KM, Uhlin BE, et al. Reversible senescence of human colon cancer cells after blockage of mitosis/cytokinesis caused by the CNF1 cyclomodulin from Escherichia coli. Sci Rep. 2018;8:17780.
</cite> [<a href="https://doi.org/10.1038/s41598-018-36036-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6290797/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30542142/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang%20Z,%20Aung%20KM,%20Uhlin%20BE,%20et%20al.%20Reversible%20senescence%20of%20human%20colon%20cancer%20cells%20after%20blockage%20of%20mitosis/cytokinesis%20caused%20by%20the%20CNF1%20cyclomodulin%20from%20Escherichia%20coli.%20Sci%20Rep.%202018;8:17780." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Fabbri A, Travaglione S, Rosadi F et al. The Escherichia coli protein toxin cytotoxic necrotizing factor 1 induces epithelial mesenchymal transition. Cell Microbiol 2019:e13138.</cite> [<a href="https://doi.org/10.1111/cmi.13138" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31698538/" class="usa-link">PubMed</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Belotserkovsky I, Sansonetti PJ. Shigella and enteroinvasive Escherichia coli. Curr Top Microbiol Immunol. 2018;416:1–26.
</cite> [<a href="https://doi.org/10.1007/82_2018_104" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30218158/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Belotserkovsky%20I,%20Sansonetti%20PJ.%20Shigella%20and%20enteroinvasive%20Escherichia%20coli.%20Curr%20Top%20Microbiol%20Immunol.%202018;416:1%E2%80%9326." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Bosak J, Micenkova L, Dolezalova M, et al. Colicins U and Y inhibit growth of Escherichia coli strains via recognition of conserved OmpA extracellular loop 1. Int J Med Microbiol. 2016;306:486–94.
</cite> [<a href="https://doi.org/10.1016/j.ijmm.2016.07.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27510856/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bosak%20J,%20Micenkova%20L,%20Dolezalova%20M,%20et%20al.%20Colicins%20U%20and%20Y%20inhibit%20growth%20of%20Escherichia%20coli%20strains%20via%20recognition%20of%20conserved%20OmpA%20extracellular%20loop%201.%20Int%20J%20Med%20Microbiol.%202016;306:486%E2%80%9394." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Varsaki A, Lamb HK, Eleftheriadou O, et al. Interaction between relaxase MbeA and accessory protein MbeC of the conjugally mobilizable plasmid ColE1. FEBS Lett. 2012;586:675–9.
</cite> [<a href="https://doi.org/10.1016/j.febslet.2012.01.060" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22449962/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Varsaki%20A,%20Lamb%20HK,%20Eleftheriadou%20O,%20et%20al.%20Interaction%20between%20relaxase%20MbeA%20and%20accessory%20protein%20MbeC%20of%20the%20conjugally%20mobilizable%20plasmid%20ColE1.%20FEBS%20Lett.%202012;586:675%E2%80%939." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366417/bin/13099_2025_732_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (688.2KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The whole genome of the bacteria was deposited in GenBank under the accession numbers CP136900, CP136901, and CP136902.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Gut Pathogens are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s13099-025-00732-1"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/13099_2025_Article_732.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (5.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366417/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366417/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366417%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366417/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366417/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366417/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40836244/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366417/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40836244/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366417/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366417/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="1F9Z4BpwmO3E7BRTe1CwaVl9CdvGDjG8BBwQmY9rHM733ZY3QtW3oSakh6FgJoSq">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
