
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            O-GlcNAc transferase plays dual antiviral roles by integrating innate immunity and lipid metabolism - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE533DDC8AF372F3053DDC000D9E7541.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ncomms">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365020/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Nature Communications">
<meta name="citation_title" content="O-GlcNAc transferase plays dual antiviral roles by integrating innate immunity and lipid metabolism">
<meta name="citation_author" content="Hong Dong">
<meta name="citation_author_institution" content="Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Chenxi Liang">
<meta name="citation_author_institution" content="Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Junjie Zhang">
<meta name="citation_author_institution" content="Division of Medicinal Chemistry and Pharmacognosy, College of Pharmacy, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Weidong Wu">
<meta name="citation_author_institution" content="Department of Biomedical Informatics, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Nitesh Kumar">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Zihao Liu">
<meta name="citation_author_institution" content="Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Yajun Sun">
<meta name="citation_author_institution" content="Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Zhiwei Liao">
<meta name="citation_author_institution" content="Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Xiaolin Cheng">
<meta name="citation_author_institution" content="Division of Medicinal Chemistry and Pharmacognosy, College of Pharmacy, The Ohio State University, Columbus, OH USA">
<meta name="citation_author_institution" content="Translational Data Analytics Institute (TDAI), The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Yanbao Yu">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of Delaware, Newark, DE USA">
<meta name="citation_author" content="Yong Zhang">
<meta name="citation_author_institution" content="Pulmonary and Critical Care Medicine, Department of Medicine, Washington University School of Medicine, St. Louis, MO USA">
<meta name="citation_author" content="Michael J Holtzman">
<meta name="citation_author_institution" content="Pulmonary and Critical Care Medicine, Department of Medicine, Washington University School of Medicine, St. Louis, MO USA">
<meta name="citation_author" content="Jianrong Li">
<meta name="citation_author_institution" content="Department Veterinary Biosciences, College of Veterinary Medicine, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Kymberly M Gowdy">
<meta name="citation_author_institution" content="Division of Pulmonary, Critical Care, and Sleep Medicine, Department of Internal Medicine, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Paul G Thomas">
<meta name="citation_author_institution" content="Department of Host-Microbe Interactions and Department of Immunology, St. Jude Children’s Research Hospital, Memphis, TN USA">
<meta name="citation_author" content="Jovica D Badjic">
<meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Anjun Ma">
<meta name="citation_author_institution" content="Department of Biomedical Informatics, The Ohio State University, Columbus, OH USA">
<meta name="citation_author_institution" content="Pelotonia Institute for Immuno-Oncology, The Ohio State University Comprehensive Cancer Center, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Qin Ma">
<meta name="citation_author_institution" content="Department of Biomedical Informatics, The Ohio State University, Columbus, OH USA">
<meta name="citation_author_institution" content="Pelotonia Institute for Immuno-Oncology, The Ohio State University Comprehensive Cancer Center, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Jacob S Yount">
<meta name="citation_author_institution" content="Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Shan-Lu Liu">
<meta name="citation_author_institution" content="Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA">
<meta name="citation_author_institution" content="Department Veterinary Biosciences, College of Veterinary Medicine, The Ohio State University, Columbus, OH USA">
<meta name="citation_author" content="Haitao Wen">
<meta name="citation_author_institution" content="Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA">
<meta name="citation_author_institution" content="Pelotonia Institute for Immuno-Oncology, The Ohio State University Comprehensive Cancer Center, The Ohio State University, Columbus, OH USA">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="16">
<meta name="citation_firstpage" content="7721">
<meta name="citation_doi" content="10.1038/s41467-025-63085-y">
<meta name="citation_pmid" content="40830102">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365020/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365020/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365020/pdf/41467_2025_Article_63085.pdf">
<meta name="description" content="Viral infection induces robust reprogramming of metabolic pathways in host cells. However, whether host metabolic enzymes detect viral components remains unknown. Our group and others previously identified O-GlcNAc transferase (OGT), an important ...">
<meta name="og:title" content="O-GlcNAc transferase plays dual antiviral roles by integrating innate immunity and lipid metabolism">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Viral infection induces robust reprogramming of metabolic pathways in host cells. However, whether host metabolic enzymes detect viral components remains unknown. Our group and others previously identified O-GlcNAc transferase (OGT), an important ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365020/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12365020">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s41467-025-63085-y"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/41467_2025_Article_63085.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365020%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12365020/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12365020/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365020/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ncomms.gif" alt="Nature Communications logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Nature Communications" title="Link to Nature Communications" shape="default" href="http://www.nature.com/ncomms/index.html" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Nat Commun</button></div>. 2025 Aug 19;16:7721. doi: <a href="https://doi.org/10.1038/s41467-025-63085-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s41467-025-63085-y</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Nat%20Commun%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Nat%20Commun%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Nat%20Commun%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>
<em>O</em>-GlcNAc transferase plays dual antiviral roles by integrating innate immunity and lipid metabolism</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Hong Dong</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Hong Dong</span></h3>
<div class="p">
<sup>1</sup>Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Dong%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Hong Dong</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liang%20C%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Chenxi Liang</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Chenxi Liang</span></h3>
<div class="p">
<sup>1</sup>Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liang%20C%22%5BAuthor%5D" class="usa-link"><span class="name western">Chenxi Liang</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Junjie Zhang</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Junjie Zhang</span></h3>
<div class="p">
<sup>2</sup>Division of Medicinal Chemistry and Pharmacognosy, College of Pharmacy, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Junjie Zhang</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20W%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Weidong Wu</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Weidong Wu</span></h3>
<div class="p">
<sup>3</sup>Department of Biomedical Informatics, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wu%20W%22%5BAuthor%5D" class="usa-link"><span class="name western">Weidong Wu</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20N%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Nitesh Kumar</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Nitesh Kumar</span></h3>
<div class="p">
<sup>4</sup>Department of Chemistry and Biochemistry, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kumar%20N%22%5BAuthor%5D" class="usa-link"><span class="name western">Nitesh Kumar</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Zihao Liu</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Zihao Liu</span></h3>
<div class="p">
<sup>1</sup>Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zihao Liu</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Yajun Sun</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Yajun Sun</span></h3>
<div class="p">
<sup>1</sup>Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sun%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yajun Sun</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Zhiwei Liao</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Zhiwei Liao</span></h3>
<div class="p">
<sup>1</sup>Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liao%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhiwei Liao</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Xiaolin Cheng</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Xiaolin Cheng</span></h3>
<div class="p">
<sup>2</sup>Division of Medicinal Chemistry and Pharmacognosy, College of Pharmacy, The Ohio State University, Columbus, OH USA </div>
<div class="p">
<sup>5</sup>Translational Data Analytics Institute (TDAI), The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Cheng%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xiaolin Cheng</span></a>
</div>
</div>
<sup>2,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Yanbao Yu</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Yanbao Yu</span></h3>
<div class="p">
<sup>6</sup>Department of Chemistry and Biochemistry, University of Delaware, Newark, DE USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yu%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yanbao Yu</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Yong Zhang</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Yong Zhang</span></h3>
<div class="p">
<sup>7</sup>Pulmonary and Critical Care Medicine, Department of Medicine, Washington University School of Medicine, St. Louis, MO USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhang%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yong Zhang</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Holtzman%20MJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Michael J Holtzman</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Michael J Holtzman</span></h3>
<div class="p">
<sup>7</sup>Pulmonary and Critical Care Medicine, Department of Medicine, Washington University School of Medicine, St. Louis, MO USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Holtzman%20MJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Michael J Holtzman</span></a>
</div>
</div>
<sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Jianrong Li</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Jianrong Li</span></h3>
<div class="p">
<sup>8</sup>Department Veterinary Biosciences, College of Veterinary Medicine, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Li%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Jianrong Li</span></a>
</div>
</div>
<sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gowdy%20KM%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Kymberly M Gowdy</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Kymberly M Gowdy</span></h3>
<div class="p">
<sup>9</sup>Division of Pulmonary, Critical Care, and Sleep Medicine, Department of Internal Medicine, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gowdy%20KM%22%5BAuthor%5D" class="usa-link"><span class="name western">Kymberly M Gowdy</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thomas%20PG%22%5BAuthor%5D" class="usa-link" aria-describedby="id15"><span class="name western">Paul G Thomas</span></a><div hidden="hidden" id="id15">
<h3><span class="name western">Paul G Thomas</span></h3>
<div class="p">
<sup>10</sup>Department of Host-Microbe Interactions and Department of Immunology, St. Jude Children’s Research Hospital, Memphis, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Thomas%20PG%22%5BAuthor%5D" class="usa-link"><span class="name western">Paul G Thomas</span></a>
</div>
</div>
<sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Badjic%20JD%22%5BAuthor%5D" class="usa-link" aria-describedby="id16"><span class="name western">Jovica D Badjic</span></a><div hidden="hidden" id="id16">
<h3><span class="name western">Jovica D Badjic</span></h3>
<div class="p">
<sup>4</sup>Department of Chemistry and Biochemistry, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Badjic%20JD%22%5BAuthor%5D" class="usa-link"><span class="name western">Jovica D Badjic</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id17"><span class="name western">Anjun Ma</span></a><div hidden="hidden" id="id17">
<h3><span class="name western">Anjun Ma</span></h3>
<div class="p">
<sup>3</sup>Department of Biomedical Informatics, The Ohio State University, Columbus, OH USA </div>
<div class="p">
<sup>11</sup>Pelotonia Institute for Immuno-Oncology, The Ohio State University Comprehensive Cancer Center, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Anjun Ma</span></a>
</div>
</div>
<sup>3,</sup><sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id18"><span class="name western">Qin Ma</span></a><div hidden="hidden" id="id18">
<h3><span class="name western">Qin Ma</span></h3>
<div class="p">
<sup>3</sup>Department of Biomedical Informatics, The Ohio State University, Columbus, OH USA </div>
<div class="p">
<sup>11</sup>Pelotonia Institute for Immuno-Oncology, The Ohio State University Comprehensive Cancer Center, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ma%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qin Ma</span></a>
</div>
</div>
<sup>3,</sup><sup>11</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yount%20JS%22%5BAuthor%5D" class="usa-link" aria-describedby="id19"><span class="name western">Jacob S Yount</span></a><div hidden="hidden" id="id19">
<h3><span class="name western">Jacob S Yount</span></h3>
<div class="p">
<sup>1</sup>Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yount%20JS%22%5BAuthor%5D" class="usa-link"><span class="name western">Jacob S Yount</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20SL%22%5BAuthor%5D" class="usa-link" aria-describedby="id20"><span class="name western">Shan-Lu Liu</span></a><div hidden="hidden" id="id20">
<h3><span class="name western">Shan-Lu Liu</span></h3>
<div class="p">
<sup>1</sup>Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA </div>
<div class="p">
<sup>8</sup>Department Veterinary Biosciences, College of Veterinary Medicine, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20SL%22%5BAuthor%5D" class="usa-link"><span class="name western">Shan-Lu Liu</span></a>
</div>
</div>
<sup>1,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20H%22%5BAuthor%5D" class="usa-link" aria-describedby="id21"><span class="name western">Haitao Wen</span></a><div hidden="hidden" id="id21">
<h3><span class="name western">Haitao Wen</span></h3>
<div class="p">
<sup>1</sup>Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA </div>
<div class="p">
<sup>11</sup>Pelotonia Institute for Immuno-Oncology, The Ohio State University Comprehensive Cancer Center, The Ohio State University, Columbus, OH USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wen%20H%22%5BAuthor%5D" class="usa-link"><span class="name western">Haitao Wen</span></a>
</div>
</div>
<sup>1,</sup><sup>11,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Microbial Infection and Immunity, Infectious Disease Institute, The Ohio State University, Columbus, OH USA </div>
<div id="Aff2">
<sup>2</sup>Division of Medicinal Chemistry and Pharmacognosy, College of Pharmacy, The Ohio State University, Columbus, OH USA </div>
<div id="Aff3">
<sup>3</sup>Department of Biomedical Informatics, The Ohio State University, Columbus, OH USA </div>
<div id="Aff4">
<sup>4</sup>Department of Chemistry and Biochemistry, The Ohio State University, Columbus, OH USA </div>
<div id="Aff5">
<sup>5</sup>Translational Data Analytics Institute (TDAI), The Ohio State University, Columbus, OH USA </div>
<div id="Aff6">
<sup>6</sup>Department of Chemistry and Biochemistry, University of Delaware, Newark, DE USA </div>
<div id="Aff7">
<sup>7</sup>Pulmonary and Critical Care Medicine, Department of Medicine, Washington University School of Medicine, St. Louis, MO USA </div>
<div id="Aff8">
<sup>8</sup>Department Veterinary Biosciences, College of Veterinary Medicine, The Ohio State University, Columbus, OH USA </div>
<div id="Aff9">
<sup>9</sup>Division of Pulmonary, Critical Care, and Sleep Medicine, Department of Internal Medicine, The Ohio State University, Columbus, OH USA </div>
<div id="Aff10">
<sup>10</sup>Department of Host-Microbe Interactions and Department of Immunology, St. Jude Children’s Research Hospital, Memphis, TN USA </div>
<div id="Aff11">
<sup>11</sup>Pelotonia Institute for Immuno-Oncology, The Ohio State University Comprehensive Cancer Center, The Ohio State University, Columbus, OH USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2024 Sep 24; Accepted 2025 Aug 8; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution-NonCommercial-NoDerivatives 4.0 International License, which permits any non-commercial use, sharing, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if you modified the licensed material. You do not have permission under this licence to share adapted material derived from this article or parts of it. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by-nc-nd/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by-nc-nd/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12365020  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830102/" class="usa-link">40830102</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Viral infection induces robust reprogramming of metabolic pathways in host cells. However, whether host metabolic enzymes detect viral components remains unknown. Our group and others previously identified <em>O</em>-GlcNAc transferase (OGT), an important glucose metabolic enzyme, as a crucial mediator of the antiviral immune responses. Here, by studying a mouse model with a catalytically impaired OGT, we discover a catalytic activity-independent function of OGT in restraining influenza A virus (IAV) infection in addition to its catalytic activity-dependent effect on MAVS-mediated antiviral immunity. Biochemical studies reveal a critical antiviral effect based on OGT interacting with IAV genomic RNA that requires its N-terminal tetracopeptide repeat-4 motif. This interaction causes the translocation of nuclear OGT to cytosolic lipid droplets (LDs) to destabilize LDs-coating perilipin 2, thereby limiting LDs accumulation and in turn virus replication. In sum, our findings reveal OGT as a multifaceted metabolic sensor that integrates MAVS signaling and lipid metabolism to combat viral infection.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Pattern recognition receptors, Influenza virus, Fatty acids, Glycosylation, Ubiquitylation</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">O-GlcNAc transferase (OGT) mediates antiviral host immune response. Here, the authors identify a catalytic activity-independent function of OGT in restraining influenza A virus replication by translocating to lipid droplets and limiting their accumulation following interaction with viral RNA.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">During RNA virus infection, the innate immune system uses retinoic-acid inducible gene I (RIG-I)-like receptors (RLRs), including RIG-I and melanoma differentiation associated gene 5 (MDA5), to detect cytosolic viral RNA species and initiate a signaling cascade leading to immune activation<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>–<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Activation of RIG-I and MDA5 promotes the recruitment of the signaling adaptor protein MAVS and activation of downstream NF-κB and IRF3 antiviral signaling, causing the production of type I interferon (IFN-I) and IFN-stimulated genes (ISGs)<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>,<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a></sup>. In addition, viral genomic RNA can be detected by Z-DNA binding protein 1 (ZBP1), which initiates receptor-interacting serine/threonine-protein kinase 3 (RIPK3)-mediated necroptosis<sup><a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>–<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a></sup>. These two evolutionarily conserved sensing mechanisms constitute the front line of host defense responses against viral infection<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>.</p>
<p id="Par4">Reprogramming of cellular metabolic activities has recently been demonstrated to play a critical role in the activation of host defense responses following viral infection<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>–<a href="#CR18" class="usa-link" aria-describedby="CR18">18</a></sup>. Elevated catabolic activity in activated immune cells is required to provide biomolecules and energy for effective immune functions and virus clearance. Meanwhile, a variety of metabolic enzymes, metabolites or proteins involved in cell metabolism have been shown to play essential roles in orchestrating immune activation, which is required for host response against viral infection<sup><a href="#CR19" class="usa-link" aria-describedby="CR19">19</a></sup>. Therefore, metabolic system regulates immune cell function and inflammation through combined strategies. Indeed, though metabolic reprogramming is a hallmark feature of host antiviral defense responses, it remains unclear whether metabolism-coupled sensing mechanisms exist to alarm metabolic networks for reprogramming upon viral infection.</p>
<p id="Par5"><em>O</em>-GlcNAc transferase (OGT) is an essential glycolytic enzyme that mediates the transfer of UDP-GlcNAc to serine or threonine residues of target proteins<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>–<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. This unique posttranslational modification (PTM), known as protein <em>O</em>-GlcNAcylation, targets many nuclear and cytosolic proteins<sup><a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>,<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a></sup>. Previous studies have discovered important functions of OGT and <em>O</em>-GlcNAc signaling in many fundamental biological activities, including transcription, translation and signal transduction<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>–<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. Recent studies from our group<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> and others<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>–<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a></sup> have shown OGT-dependent protein <em>O</em>-GlcNAcylation as an important cellular mechanism to promote antiviral innate immune response. OGT contains a C-terminal glycosyltransferase catalytic domain and an N-terminal tetratricopeptide repeats (TPR) motif involved in protein-protein interactions. Previously, most OGT studies have focused on its catalytic function (i.e., <em>O</em>-GlcNAcylation of target proteins affecting their functions such as translocation, kinase activity, and protein stability, etc.). Meanwhile, a noncatalytic function of OGT likely exists because the TPR motif has been well known to have a scaffolding function involving many proteins, such as antiviral IFIT proteins (interferon-induced protein with tetratricopeptide repeats)<sup><a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. Indeed, one study reported a critical function of noncatalytic activity of OGT in regulating mammalian cell proliferation and survival<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. Using a whole-body catalytically impaired mouse strain with missense variant of OGT that is associated with intellectual disability, one recent study reported that the catalytic activity of OGT is essential for normal skull morphology and brain size<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. However, the relative contribution of catalytic versus noncatalytic effects of OGT in modulating antiviral responses remains unexplored.</p>
<p id="Par6">In this study, we generated a conditional OGT catalytically impaired knock-in mouse model to investigate its catalytic versus noncatalytic effects on host antiviral responses. Surprisingly, our findings demonstrate an OGT catalytic activity-dependent effect on MAVS antiviral signaling and an OGT catalytic activity-independent antiviral effect via modulating lipid droplets (LDs) during influenza A virus (IAV) infection. Biochemical studies detected OGT interacting with IAV genomic RNA that requires its N-terminal TPR4 motif, which promotes a translocation of nuclear OGT to cytosolic LDs compartment and thereby limits viral replication. Therefore, our studies identified OGT as an important metabolic component of host antiviral defense through sensing viral infection and integrating innate immune signaling and cellular lipid metabolism.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">OGT inhibits IAV infection via both catalytic activity-dependent and -independent manner</h3>
<p id="Par7">We and others have recently identified OGT as an essential antiviral protein that modifies MAVS to enhance its signaling capacity<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. To determine if OGT-mediated antiviral effects are solely dependent on protein <em>O</em>-GlcNAcylation, we generated a conditional OGT catalytically impaired knock-in mouse model by replacing lysine 908, which is a residue critical for capturing substrate UDP-GlcNAc and for its enzymatic activity<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>–<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>, with alanine (<em>Ogt-K908A</em><sup><em>fl/fl</em></sup>) (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a, b</a>). <em>Ogt-K908A</em><sup><em>fl/fl</em></sup> and <em>Ogt</em><sup><em>fl/fl</em></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">1c, d</a>)<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> mice were then crossed with <em>R26-Cre-ER</em><sup><em>T2</em></sup> mice to achieve tamoxifen-inducible <em>Ogt</em> catalytically impaired (referred to as <em>Ogt</em><sup><em>K908A</em></sup>) and <em>Ogt</em>-knockout (referred to as <em>Ogt</em><sup><em>KO</em></sup>) models, respectively. Treatment with tamoxifen for 72 h caused efficient reduction of total <em>O</em>-GlcNAc signal in <em>Ogt</em><sup><em>K908A</em></sup> and <em>Ogt</em><sup><em>KO</em></sup> mouse embryonic fibroblasts (MEFs) (Supplementary Fig. <a href="#MOESM1" class="usa-link">1e, f</a>). Next, we infected <em>Ogt</em><sup><em>K908A</em></sup> and <em>Ogt</em><sup><em>KO</em></sup> MEFs with IAV (PR8) at a multiplicity of infection (MOI) of 1<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>. <em>Ogt</em><sup><em>KO</em></sup> MEFs exhibited a significantly increased IAV infection, evidenced by elevated IAV transcripts <em>NS1</em> and <em>NP</em> (encoding non-structural protein 1 and nucleoprotein of IAV, respectively) (Fig. <a href="#Fig1" class="usa-link">1a</a>) and viral titers (Fig. <a href="#Fig1" class="usa-link">1b</a>), compared to <em>Ogt</em><sup><em>fl/fl</em></sup> control cells. Moreover, <em>Ogt</em><sup><em>KO</em></sup> MEFs contained a markedly increased amount of IAV protein hemagglutinin (HA) and neuraminidase (NA) following viral infection compared to <em>Ogt</em><sup><em>fl/fl</em></sup> control cells (Fig. <a href="#Fig1" class="usa-link">1c</a>). Despite being more permissive for viral replication than WT MEFs, <em>Ogt</em><sup><em>K908A</em></sup> MEFs showed an attenuated viral replication compared to <em>Ogt</em><sup><em>KO</em></sup> MEFs (Fig. <a href="#Fig1" class="usa-link">1a–c</a>). When cells were treated with GFP-expressing IAV, we observed a similarly attenuated viral replication phenotype in <em>Ogt</em><sup><em>K908A</em></sup> MEFs compared to <em>Ogt</em><sup><em>KO</em></sup> MEFs (Fig. <a href="#Fig1" class="usa-link">1d</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">1g</a>). These results suggest a possibility that OGT may regulate antiviral response through both catalytic activity-dependent and -independent mechanism.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. OGT inhibits IAV infection via both catalytic activity-dependent and -independent manner.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365020_41467_2025_63085_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2bb8/12365020/44e1c7d52ce1/41467_2025_63085_Fig1_HTML.jpg" loading="lazy" id="d33e857" height="989" width="727" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>–<strong>g</strong> Mouse embryonic fibroblasts (MEFs) generated from <em>Ogt</em><sup><em>fl/fl</em></sup>, <em>Ogt-K908A</em><sup><em>fl/fl</em></sup>, <em>Ogt</em>
<sup><em>KO</em></sup> and <em>Ogt</em><sup><em>K908A</em></sup> mice were infected with PR8 or PR8-GFP virus at a multiplicity of infection (MOI) = 1 for 12 h. Viral mRNAs (<strong>a</strong>), viral titers (<strong>b</strong>), expression of IAV HA and NA proteins (<strong>c</strong>), and virus infected cells (<strong>d</strong>) were measured by RT-PCR, TCID<sub>50</sub> assays, immunoblotting with corresponding antibodies and flow cytometry respectively. <strong>e</strong> Gene transcripts including <em>Ifna4</em>, <em>Ifnb1</em>, <em>Mx1</em>, <em>Isg15</em>, <em>Cxcl10</em>, <em>Il6</em>, and <em>Tnfa</em> in the cells were measured with RT-PCR. <strong>f</strong> Immunoblotting of phosphorylated TBK1, IRF3, NF-κB signaling molecules and protein <em>O</em>-GlcNAcylation was performed with indicated antibodies. <strong>g</strong> Immunoprecipitated MAVS from the MEFs was assessed for <em>O</em>-GlcNAcylation with anti-<em>O</em>-GlcNAc antibody. Data are representative of three independent experiments (<strong>a</strong>, <strong>b</strong> and <strong>d</strong>, <strong>e</strong>). Statistical significance was determined by two-way ANOVA followed by Tukey’s test (<strong>a</strong>, <strong>b</strong> and <strong>d</strong>, <strong>e</strong>), <em>P</em> values of statistical comparisons are shown in each graph. The error bars represent SEM from <em>n</em> = 3 independent biological replicates (<strong>a</strong>, <strong>b</strong> and <strong>d</strong>, <strong>e</strong>). Source data are provided as a <a href="#MOESM6" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par8">We next sought to determine if impaired catalytic activity of OGT abolished MAVS-mediated innate immune activation and ISG production<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Following IAV infection, we found a significantly decreased production of multiple ISGs such as <em>Ifna4</em>, <em>Ifnb1</em>, <em>Mx1</em>, <em>Isg15</em>, <em>Cxcl10</em> and genes encoding inflammatory cytokine <em>Il6</em> and <em>Tnfa</em> in <em>Ogt</em><sup><em>K908A</em></sup> MEFs compared to <em>Ogt-K908A</em><sup><em>fl/fl</em></sup> cells (Fig. <a href="#Fig1" class="usa-link">1e</a>). Moreover, IAV-induced activation of MAVS antiviral signaling, evidenced by phosphorylation of IRF3, TBK1 and NF-κB p65, was markedly blunted in <em>Ogt</em><sup><em>K908A</em></sup> MEFs (Fig. <a href="#Fig1" class="usa-link">1f</a>). These results were consistent with previous studies showing a critical function of OGT in the activation of innate immune signaling upon viral infection<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>,<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. <em>Ogt</em><sup><em>K908A</em></sup> and <em>Ogt</em><sup><em>KO</em></sup> MEFs showed a comparable level of defect in ISG production and antiviral signaling activation (Fig. <a href="#Fig1" class="usa-link">1e, f</a>), suggesting that impaired catalytical activity of OGT is sufficient to cause defective MAVS-mediated antiviral immune signaling in <em>Ogt</em><sup><em>K908A</em></sup> MEFs. <em>O</em>-GlcNAcylation of MAVS was enhanced in WT cells following IAV infection but was completely abolished in both <em>Ogt</em><sup><em>K908A</em></sup> and <em>Ogt</em><sup><em>KO</em></sup> MEFs (Fig. <a href="#Fig1" class="usa-link">1g</a>). These results support the concept that OGT promotes MAVS innate immune signaling via its catalytic activity, as we recently reported<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>.</p>
<p id="Par9">We further examined whether catalytic activity-dependent versus catalytic activity-independent antiviral effects of OGT also exist in human cells. <em>OGT</em>-KO HT29 cells were reconstituted with stable expression of OGT WT or K908A mutant construct, as reported in our recent study<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Reconstitution of <em>OGT</em>-KO cells with WT OGT caused a significant decrease in IAV transcripts <em>NS1</em> and <em>NP</em> (Fig. <a href="#Fig2" class="usa-link">2a</a>) and viral titers (Fig. <a href="#Fig2" class="usa-link">2b</a>), compared to <em>OGT</em>-KO cells reconstituted with empty vector. In contrast, reconstitution of <em>OGT</em>-KO cells with OGT K908A mutant only partially inhibited viral replication compared to <em>OGT</em>-KO cells reconstituted with WT OGT (Fig. <a href="#Fig2" class="usa-link">2a, b</a>). As expected, reconstitution of <em>OGT</em>-KO cells with WT OGT markedly elevated the production of multiple ISGs (Fig. <a href="#Fig2" class="usa-link">2c</a>) and activation of antiviral innate immune signaling (Fig. <a href="#Fig2" class="usa-link">2d</a>). However, the inhibitory effect of OGT K908A mutant on viral replication was unlikely related to MAVS signaling due to the lack of change in ISG production and no phosphorylation of IRF3, TBK1, or NF-κB p65 in <em>OGT</em>-KO cells reconstituted with OGT K908A mutant compared to empty vector (Fig. <a href="#Fig2" class="usa-link">2c, d</a>).</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. OGT inhibits IAV infection via both catalytic activity-dependent and -independent manner in human cells.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365020_41467_2025_63085_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2bb8/12365020/c4cd16512bed/41467_2025_63085_Fig2_HTML.jpg" loading="lazy" id="d33e1163" height="937" width="784" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>–<strong>d</strong>
<em>OGT</em>-KO HT29 cells were reconstituted with empty vector (EV), GFP-tagged OGT (OGT-WT) or its K908A mutants, followed by the infection with PR8 virus at a MOI = 1 for 12 h. Viral mRNAs (<strong>a</strong>), and viral titers (<strong>b</strong>) were measured by RT-PCR, and TCID<sub>50</sub> assays respectively. <strong>c</strong> Gene transcripts including <em>IFNA4</em>, <em>IFNB</em>, <em>MX1</em>, <em>ISG15</em>, <em>IL6</em>, and <em>TNFA</em> in the cells were measured with RT-PCR. <strong>d</strong> Immunoblotting of phosphorylated TBK1, IRF3, IKKα/β, IκBα, and NF-κB signaling molecules and protein <em>O</em>-GlcNAcylation was performed with indicated antibodies. <strong>e</strong>–<strong>h,</strong>
<em>MAVS-KO</em> or <em>IRF3-KO</em> and WT control of HT29 EV, OGT-WT and OGT-K908A cells were infected with PR8 virus at a MOI = 1 for 12 h. <strong>e</strong> The expression of MAVS and IRF3 proteins in cells were assessed by immunoblotting. Viral mRNAs (<strong>f</strong>), and viral titers (<strong>g</strong>) were measured by RT-PCR and TCID<sub>50</sub> assays respectively. <strong>h</strong> Gene transcripts including <em>IFNB</em>, <em>MX1</em>, <em>ISG15</em>, <em>IL6</em>, and <em>TNFA</em> in the cells were measured with RT-PCR. Data are representative of three independent experiments (<strong>a</strong>–<strong>c</strong> and <strong>f</strong>–<strong>h</strong>). Statistical significance was determined by two-way ANOVA followed by Tukey’s test (<strong>a</strong>–<strong>c</strong> and <strong>f</strong>–<strong>h</strong>), <em>P</em> values of statistical comparisons are shown in each graph. The error bars represent SEM from <em>n</em> = 3 independent biological replicates (<strong>a</strong>–<strong>c</strong> and <strong>f</strong>–<strong>h</strong>). Source data are provided as a <a href="#MOESM6" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par10">To determine if additional ISGs induced by MAVS signaling other than those examined (Fig. <a href="#Fig2" class="usa-link">2c</a>) may contribute to antiviral effect of OGT K908A mutant, we compared antiviral effect of OGT WT and K908A mutant in cells with the depletion of MAVS (<em>MAVS</em>-KO) or IRF3 (<em>IRF</em>-KO). CRISPR/Cas9-based gene targeting strategy with the use of single guide RNA (gRNA) specific to human <em>MAVS</em> or <em>IRF3</em> gene was employed in cells with various <em>OGT</em> genetic backgrounds to generate cells with single or double gene mutations (Fig. <a href="#Fig2" class="usa-link">2e</a>). When MAVS or IRF3 was genetically depleted, we detected a comparable viral-restriction effect between OGT WT and K908A mutant, evidenced by similar levels of IAV transcripts (Fig. <a href="#Fig2" class="usa-link">2f</a>) and viral titers (Fig. <a href="#Fig2" class="usa-link">2g</a>). As expected, the deletion of MAVS or IRF3 abolished ISG production upon IAV infection despite <em>OGT</em> genetic background (Fig. <a href="#Fig2" class="usa-link">2h</a>). In sum, these results from genetic studies indicate that OGT-mediated antiviral response relies on both catalytic activity-dependent and -independent mechanisms and that OGT catalytic activity-dependent antiviral effect requires MAVS-IRF3 innate immune signaling pathway.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Catalytic activity-dependent and -independent antiviral functions of OGT in vivo</h3>
<p id="Par11">To examine catalytic activity-dependent and -independent antiviral effects of OGT in vivo, we generated mice with lung epithelial cell (LEC)-specific OGT deletion (referred to as <em>Ogt</em><sup><em>ΔLEC</em></sup>) or catalytically impaired OGT (referred to as <em>Ogt-K908A</em><sup><em>ΔLEC</em></sup>) by crossing <em>Ogt</em><sup><em>fl/fl</em></sup> or <em>Ogt-K908A</em><sup><em>fl/fl</em></sup> with <em>Foxj1-Scgb1a1-Cre</em> mice<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. Previous study has demonstrated that <em>Foxj1-Scgb1a1-Cre</em> system could efficiently induce gene deletion specifically in ciliated cells and columnar cells in the mouse respiratory tract, the two major cell compartments targeted by respiratory viral infection<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup>. We infected mice intranasally with IAV at a sublethal dose of 10 TCID<sub>50</sub>. All <em>Ogt</em><sup><em>fl/fl</em></sup> WT mice survived over the entire course of 20 days after infection, as we previously reported<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>, whereas all <em>Ogt</em><sup><em>ΔLEC</em></sup> mice succumbed to infection by day 12 after infection (Fig. <a href="#Fig3" class="usa-link">3a</a>). <em>Ogt</em><sup><em>ΔLEC</em></sup> mice also experienced markedly exacerbated weight loss compared to <em>Ogt</em><sup><em>fl/fl</em></sup> WT mice (Fig. <a href="#Fig3" class="usa-link">3b</a>). In contrast, 50% of <em>Ogt-K908A</em><sup><em>ΔLEC</em></sup> mice survived over 20 days after infection and experienced moderately increased weight loss compared to <em>Ogt</em><sup><em>fl/fl</em></sup> WT mice (Fig. <a href="#Fig3" class="usa-link">3a, b</a>). These results indicate that impaired catalytic activity of OGT resulted in less severe disease outcomes compared to OGT deletion in lung epithelial cells.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Catalytic activity-dependent and -independent antiviral function of OGT in vivo<em>.</em>
</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365020_41467_2025_63085_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2bb8/12365020/18662a1a25e8/41467_2025_63085_Fig3_HTML.jpg" loading="lazy" id="d33e1440" height="814" width="798" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>–<strong>g</strong>
<em>Ogt</em><sup><em>fl/fl</em></sup>, <em>Ogt</em><sup><em>ΔLEC</em></sup> and <em>Ogt-K908A</em><sup><em>ΔLEC</em></sup> mice were infected with 10 TCID<sub>50</sub> of PR8 virus. Survival (<strong>a</strong>) and changes in body weight (<strong>b</strong>) of mice were monitored every 2 days. <em>Ogt</em><sup><em>fl/fl</em></sup> mice (<em>n</em> = 18), <em>Ogt</em><sup><em>ΔLEC</em></sup> mice (<em>n</em> = 17) and <em>Ogt-K908A</em><sup><em>ΔLEC</em></sup> mice (<em>n</em> = 18). Viral mRNAs (<strong>c</strong>), and viral titers (<strong>d</strong>) in the lung were measured by RT-PCR and TCID<sub>50</sub> assays respectively. <strong>e</strong> Gene transcripts including <em>Ifna4</em>, <em>Ifnb1</em>, <em>Mx1</em>, <em>Isg15</em> and <em>Cxcl10</em> in the lung were measured with RT-PCR. <strong>f</strong> IFN-β protein in the lung was measured with ELISA. <strong>g</strong> Histological analysis of the lung tissue collected from animals at day 2 after viral infection. Scale bars, 20 μm. Data are representative of three independent experiments (<strong>a</strong>–<strong>f</strong>). Statistical significance was determined by log-rank (Mantel-Cox) test survival analysis (<strong>a</strong>) or two-way ANOVA followed by Tukey’s test (<strong>b</strong>–<strong>f</strong>), <em>P</em> values of statistical comparisons are shown in each graph. The error bars represent SEM from <em>n</em> = 3 independent experiments (<strong>a</strong>–<strong>f</strong>), each dot represents the means of each of the three experiments (<strong>c</strong>–<strong>f</strong>). Source data are provided as a <a href="#MOESM6" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par12">We further examined viral replication and antiviral immune responses in lung tissue following IAV infection. Consistent with elevated viral replication in <em>Ogt</em><sup><em>KO</em></sup> cells, <em>Ogt</em><sup><em>ΔLEC</em></sup> mice contained significantly higher amounts of virus transcripts (Fig. <a href="#Fig3" class="usa-link">3c</a>) and viral titers (Fig. <a href="#Fig3" class="usa-link">3d</a>) in lung tissues at day 1 and 2 after infection. <em>Ogt-K908A</em><sup><em>ΔLEC</em></sup> mice exhibited attenuated viral replication in lung tissue compared to <em>Ogt</em><sup><em>ΔLEC</em></sup> mice, indicating both catalytic activity-dependent and -independent functions of OGT in limiting IAV replication in vivo. Furthermore, we detected robust induction of ISG gene transcription (Fig. <a href="#Fig3" class="usa-link">3e</a>) and production of IFN-β protein (Fig. <a href="#Fig3" class="usa-link">3f</a>) in the lung from WT mice at day 1 and 2 after infection, indicating efficient induction of antiviral immunity. In contrast, ISG induction and IFN-β production were significantly suppressed in both <em>Ogt</em><sup><em>ΔLEC</em></sup> mice and <em>Ogt-K908A</em><sup><em>ΔLEC</em></sup> mice to the same extent compared to <em>Ogt</em><sup><em>fl/fl</em></sup> WT mice (Fig. <a href="#Fig3" class="usa-link">3e, f</a>). Histological analysis of the lung tissue from animals 2 days after viral infection showed greater infiltration of immune cells and injury in <em>Ogt</em><sup><em>ΔLEC</em></sup> mice compared with WT mice (Fig. <a href="#Fig3" class="usa-link">3g</a>). Less immune cell infiltration was observed in the lung from IAV- infected <em>Ogt-K908A</em><sup><em>ΔLEC</em></sup> mice compared to similarly treated <em>Ogt</em><sup><em>ΔLEC</em></sup> mice, likely due to less viral replication. These findings indicate that OGT is crucial for limiting IAV viral replication in vivo through both catalytic activity-dependent and -independent mechanisms.</p></section><section id="Sec5"><h3 class="pmc_sec_title">OGT interacts with IAV genomic RNA</h3>
<p id="Par13">The N-terminal region of OGT is composed of 13.5 TPR repeats that are involved in protein-protein interactions and substrate recognition<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. It has been documented that critical antiviral effector molecules IFIT1 and IFIT5 recognize influenza and vesicular stomatitis virus RNA via their TPR motif<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. Thus, we sought to determine whether OGT could recognize viral genomic RNA that requires its TPR motif and, if so, whether OGT interaction with viral RNA is important for its antiviral effect. Several steps of examination of OGT-viral genomic RNA interaction were performed. First, we overexpressed a full-length (FL) OGT, or its N-terminal or C-terminal truncations in 293 T cells (Fig. <a href="#Fig4" class="usa-link">4a, b</a>), followed by IAV infection. Using IAV genomic RNA specific Uni-12 primer for RT-PCR assay<sup><a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>,<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>, we found that all eight viral genomic RNA (<em>PA</em>, <em>PB2</em>, <em>NP</em>, <em>NA</em>, <em>HA</em>, <em>NS1</em>, <em>PB1</em> and <em>M</em>) were coimmunoprecipitated with OGT FL (Fig. <a href="#Fig4" class="usa-link">4c</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2a</a>). Coimmunoprecipitation of viral genomic RNA with OGT depends on its N-terminal, but not C-terminal (Fig. <a href="#Fig4" class="usa-link">4c</a>). These results indicate that OGT interacts with viral genomic RNA that requires its N-terminal TPR motif. Second, we generated mutant OGT construct with the deletion of either TPR1-7 or TPR8-13.5 (Fig. <a href="#Fig4" class="usa-link">4d</a>) and found that the lack of TPR1-7 abolished OGT coimmunoprecipitation with viral <em>PA</em> and <em>PB2</em> genomic RNA (Fig. <a href="#Fig4" class="usa-link">4e</a>). Third, we examined the role of TPR1-7 by deleting each repeat (Fig. <a href="#Fig4" class="usa-link">4f, g</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2b</a>) and found that the lack of TPR4 (ΔTPR4) abolished OGT-viral genomic RNA interaction (Fig. <a href="#Fig4" class="usa-link">4h</a>). As controls, OGT failed to immuneprecipitate host-derived housekeeping transcripts <em>ACTB</em> and <em>GAPDH</em> (Fig. <a href="#Fig4" class="usa-link">4h</a>). These results suggest that OGT interacts with IAV genomic RNA that requires its TPR4 motif.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. OGT interacts with IAV genomic RNA.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365020_41467_2025_63085_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2bb8/12365020/b7c87a069f99/41467_2025_63085_Fig4_HTML.jpg" loading="lazy" id="d33e1717" height="992" width="730" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>–<strong>h</strong>, 293T cells were transfected with indicated vectors. At 12 h after transfection, cells were infected with PR8 virus at MOI = 1 and cell lysates were harvested at 48 h after transfection and immunoprecipitated with anti-Myc Agarose. <strong>a</strong> Schematic diagram of full-length OGT and its truncated mutants. N-ter, N-terminal; C-ter, C-terminal; TPR, tetratricopeptide repeats motif; <strong>b</strong>, <strong>d</strong> Immunoblotting of full-length OGT and its truncated mutants in immunoprecipitation assay. <strong>c</strong>, <strong>e</strong> Viral PA and PB2 genomic RNAs in immunoprecipitation assay were quantified by RT-PCR. <strong>f</strong> Schematic diagram of full-length OGT and its TPR motif deletion mutants. <strong>g</strong> Immunoblotting of full-length OGT and its TPR motif deletion mutants in immunoprecipitation assay. <strong>h</strong> Viral PA, PB2 genomic RNAs and human housekeeping gene transcripts in immunoprecipitation assay were quantified by RT-PCR. <strong>i</strong>, <strong>j</strong> 293T cells were transfected with indicated vectors for 36 h, cell lysates were incubated with biotin-labeled viral PB2 genomic RNA and immunoprecipitated with streptavidin beads. Immunoblotting of Flag-RIG-I (Positive control) and OGT (<strong>i</strong>) or OGT and its mutants (<strong>j</strong>) in immunoprecipitation assay. <strong>k</strong>, <strong>l</strong> Electrophoretic mobility shift assay (EMSA) of in vitro transcribed viral PB2 genomic RNA in the presence of purified Flag-RIG-I (Positive control), Flag-OGT or its mutants. Mobility of viral PB2 genomic RNA (600 ng) in the presence of purified Flag-RIG-I (1, 2, and 4 μg) and Flag-OGT (1, 3, and 6 μg) (<strong>k</strong>) or in the presence of purified Flag-OGT and its mutants (4 μg for each sample) (<strong>l</strong>). Data are representative of three independent experiments (<strong>c</strong>, <strong>e</strong>, <strong>h</strong>). Statistical significance was determined by one-way ANOVA followed by Tukey’s test (<strong>c</strong>, <strong>e</strong>, <strong>h</strong>), <em>P</em> values of statistical comparisons are shown in each graph. The error bars represent SEM from <em>n</em> = 3 independent biological replicates (<strong>c</strong>, <strong>e</strong>, <strong>h</strong>). Source data are provided as a <a href="#MOESM6" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par14">A reciprocal coimmunoprecipitation assay was performed to further examine the interaction between OGT and viral genomic RNA. The <em>PB2</em> genomic RNA of IAV generated via in vitro transcription was biotinylated and mixed with Streptavidin beads (Supplementary Fig. <a href="#MOESM1" class="usa-link">2c</a>). After the incubation with Flag-tagged OGT protein overexpressed in 293T cells, biotinylated RNA/beads mixture, but not unlabeled control sample, coimmunoprecipitated FL OGT (Fig. <a href="#Fig4" class="usa-link">4i</a>). RIG-I, a well-established innate immune sensor for viral RNA, was also immunoprecipitated by biotinylated IAV <em>PB2</em> genomic RNA. We further found that pulldown of OGT by IAV genomic RNA required the presence of TPR4 (Fig. <a href="#Fig4" class="usa-link">4j</a>), suggest a critical role of TPR4 in the interaction between OGT and IAV genomic RNA.</p>
<p id="Par15">To assess the viral genomic RNA-interacting properties of OGT, we measured the mobility of synthetic IAV <em>PB2</em> genomic RNA in the absence or presence of OGT by electrophoretic mobility shift assay (EMSA)<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a></sup>. Purified FL OGT protein (Supplementary Fig. <a href="#MOESM1" class="usa-link">2d</a>) caused mobility retardations of IAV <em>PB2</em> genomic RNA in a dose-dependent manner (Fig. <a href="#Fig4" class="usa-link">4k</a>). RIG-I protein exhibited a similar inhibitory effect on <em>PB2</em> genomic RNA mobility. Deletion of TPR4, but not TPR5, of OGT rescued the migration of <em>PB2</em> genomic RNA compared to FL OGT (Fig. <a href="#Fig4" class="usa-link">4l</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">2e</a>). Therefore, these results strongly support that OGT interacts with IAV genomic RNA that requires its TPR4 motif.</p>
<p id="Par16">To probe the noncovalent interaction between OGT and viral genomic RNA, we performed isothermal titration calorimetry (ITC) assay, a widely used method to measure the binding of two molecules including protein-RNA binding<sup><a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. It is worth noting that for ITC assay, production of OGT protein was carried out in prokaryotic system (BL21 cells) due to a greater expression yield compared to 293 T cells (Supplementary Fig. <a href="#MOESM1" class="usa-link">3a</a>). When adding OGT WT protein to the solution containing PB2 RNA, an enthalpy-driven (Δ<em>H</em>° = 36.9 ± 0.91 kcal/mol) binding with an apparent <em>K</em><sub>d</sub> of 3.95 μM was observed, which fit well to noncooperative binding model. With the entropic driving force (Δ<em>S</em>° = 148.4 cal/mol), the favorable complexation is likely driven by hydrophobic effect (Supplementary Fig. <a href="#MOESM1" class="usa-link">3b</a>). In contrast, an incremental addition of OGT ΔTPR4 protein to PB2 produced negligible exothermic heat, which indicated the absence of noncovalent complexation (Supplementary Fig. <a href="#MOESM1" class="usa-link">3c</a>). These findings indicate that OGT directly interacts with viral RNA and their interaction requires TPR4.</p>
<p id="Par17">To determine whether OGT specifically interacts with viral RNA, we performed sequencing assay of OGT-immunoprecipitated RNA under eight experimental conditions, including both IP and input samples with and without IAV infection. We followed a similar method reported recently, which calculated the transcript enrichment of OGT versus empty vector control with and without IAV infection in IP and input samples<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup> (Supplementary Fig. <a href="#MOESM1" class="usa-link">4a</a>). All eight IAV genomic RNA species were detected in OGT precipitate after viral infection (Supplementary Fig. <a href="#MOESM1" class="usa-link">4b, e</a>). Meanwhile, we also observed the enrichment for OGT-bound human RNA species, such as ribosomal RNA (rRNA) pseudogene transcripts in nontreated cells (Supplementary Fig. <a href="#MOESM1" class="usa-link">4c, g</a>). OGT-human RNA interaction was attenuated after viral infection (Supplementary Fig. <a href="#MOESM1" class="usa-link">4d, f</a>). Importantly, we found that OGT dominantly immunoprecipitated viral RNA compared to human RNA in viral-infected cells (Supplementary Fig. <a href="#MOESM1" class="usa-link">4d</a>). Therefore, our results support that OGT interacts with viral RNA, which is more dominant compared to interaction with human RNA in viral-infected cells. The functional consequence of OGT interaction with host cell RNA species warrants further investigations.</p></section><section id="Sec6"><h3 class="pmc_sec_title">OGT interaction with IAV genomic RNA inhibits viral replication independently of MAVS</h3>
<p id="Par18">Based on the identification of OGT ΔTPR4 as a mutant lacking the interaction with IAV genomic RNA, we sought to explore the functional importance of OGT-IAV genomic RNA interaction. Compared to <em>OGT</em>-KO cells reconstituted with either FL OGT or OGT ΔTPR5, reconstitution of <em>OGT</em>-KO cells with OGT ΔTPR4 caused a partial reduction of IAV transcripts <em>NS1</em> and <em>NP</em> (Fig. <a href="#Fig5" class="usa-link">5a</a>) and viral titers (Fig. <a href="#Fig5" class="usa-link">5b</a>). These results indicate that TPR4-dependent OGT-IAV genomic RNA interaction contributes to antiviral function of OGT. Meanwhile, reconstitution with OGT ΔTPR4 caused a similar induction of ISGs (Fig. <a href="#Fig5" class="usa-link">5c</a>) and activation of antiviral immune signaling (Fig. <a href="#Fig5" class="usa-link">5d</a>) upon IAV infection compared to WT OGT or OGT ΔTPR5. In addition, no defect in total protein <em>O</em>-GlcNAcylation or MAVS <em>O</em>-GlcNAcylation was observed in <em>OGT</em>-KO cells reconstituted with OGT ΔTPR4, indicating an intact catalytic activity of OGT lacking TPR4 (Fig. <a href="#Fig5" class="usa-link">5e</a>). We further genetically deleted MAVS in cells with FL OGT, OGT ΔTPR4, OGT ΔTPR5, or OGT-K908A to determine if TPR4-dependent OGT antiviral function requires MAVS signaling (Fig. <a href="#Fig5" class="usa-link">5f</a>). In cells lacking MAVS, reconstitution of <em>OGT</em>-KO cells with FL OGT or OGT ΔTPR5 caused a significant reduction of IAV <em>NS1</em> and <em>NP</em> transcripts (Fig. <a href="#Fig5" class="usa-link">5g</a>) compared to empty vector, highlighting a critical virus-restriction capacity of OGT in the absence of MAVS signaling. In sharp contrast, despite efficient ablation of ISG induction (Fig. <a href="#Fig5" class="usa-link">5h</a>), reconstitution with OGT ΔTPR4 caused no reduction in IAV transcripts compared to empty vector in cells lacking MAVS. In sum, these genetic results demonstrate TPR4-dependent OGT-IAV genomic RNA interaction as an essential antiviral response that is independent of OGT catalytic activity and MAVS signaling.</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. OGT interaction with IAV genomic RNA inhibits viral replication independently of MAVS.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365020_41467_2025_63085_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2bb8/12365020/4582f5180f1b/41467_2025_63085_Fig5_HTML.jpg" loading="lazy" id="d33e2009" height="980" width="794" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>–<strong>d</strong>
<em>OGT</em>-KO HT29 cells were reconstituted with empty vector (EV), GFP-tagged OGT (OGT FL) or its indicated mutants, followed by the infection with PR8 virus at a MOI = 1 for 12 h. Viral mRNAs (<strong>a</strong>) and viral titers (<strong>b</strong>) were measured by RT-PCR and TCID<sub>50</sub> assays, respectively. <strong>c</strong> Gene transcripts including <em>IFNB</em>, <em>MX1</em>, <em>ISG15</em>, <em>IL6</em>, and <em>TNFA</em> in the cells were measured with RT-PCR. <strong>d</strong> Immunoblotting of phosphorylated TBK1, IRF3, NF-κB signaling molecules was performed with indicated antibodies. <strong>e</strong> 293T cells were transfected with indicated vectors. After 48 h of transfection, cell lysates were harvested and immunoprecipitated with anti-Flag Agarose. Immunoprecipitated Flag-MAVS was assessed for <em>O</em>-GlcNAcylation with anti-<em>O</em>-GlcNAc antibody by immunoblotting. <strong>f</strong>–<strong>h</strong><em>MAVS-KO</em> and WT control of HT29 EV, OGT-FL and its indicated mutant cells were infected with PR8 virus at a MOI = 1 for 12 h. <strong>f</strong> MAVS protein expression in cells was assessed by immunoblotting. Viral mRNAs (<strong>g</strong>), and gene transcripts including <em>IFNB</em>, <em>MX1</em>, and <em>ISG15</em> in the cells (<strong>h</strong>) were measured with RT-PCR. Data are representative of three independent experiments (<strong>a</strong>–<strong>c</strong> and <strong>g</strong>, <strong>h</strong>). Statistical significance was determined by two-way ANOVA followed by Tukey’s test (<strong>a</strong>–<strong>c</strong> and <strong>g</strong>, <strong>h</strong>), <em>P</em> values of statistical comparisons are shown in each graph. The error bars represent SEM from <em>n</em> = 3 independent biological replicates (<strong>a</strong>–<strong>c</strong> and <strong>g</strong>, <strong>h</strong>). Source data are provided as a <a href="#MOESM6" class="usa-link">Source Data</a> file.</p></figcaption></figure></section><section id="Sec7"><h3 class="pmc_sec_title">OGT counteracts lipid droplets accumulation upon IAV infection</h3>
<p id="Par19">We next sought to determine the signaling mechanism by which TPR4-dependent OGT-IAV genomic RNA interaction suppresses IAV replication. Previous studies have reported that OGT can translocate between nuclear and cytoplasmic compartments in response to extracellular nutrient cues<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a>–<a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Following IAV infection, we observed a robust cytoplasmic translocation of nuclear OGT, which is independent of its catalytic activity (Fig. <a href="#Fig6" class="usa-link">6a</a>). Surprisingly, loss of TPR4, but not TPR5, abolished IAV-induced OGT cytoplasmic translocation, highlighting a possibility that TPR4-dependent virus-restriction capacity of OGT may involve signaling occurring in the cytosol (Fig. <a href="#Fig6" class="usa-link">6b</a>). Moreover, we isolated nuclear versus cytosolic compartment in noninfected and IAV-infected cells to monitor OGT translocation. IAV infection caused an increased accumulation of WT OGT protein in the cytosol, but decreased OGT in nuclear compartments, indicating the translocation of WT OGT from nuclear to cytosolic compartment (Supplementary Fig. <a href="#MOESM1" class="usa-link">5a</a>). This translocation depends on OGT TPR4 (ΔTPR4), but not TRP5 (ΔTPR5) or its catalytic activity (K908A). In sum, these results indicate a cytoplasmic translocation of nuclear OGT upon IAV infection.</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6. OGT counteracts lipid droplets accumulation upon IAV infection.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365020_41467_2025_63085_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2bb8/12365020/08e40e75deb6/41467_2025_63085_Fig6_HTML.jpg" loading="lazy" id="d33e2238" height="990" width="783" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, <strong>b</strong>
<em>OGT</em>-KO HT29 cells were reconstituted with empty vector (EV), GFP-tagged OGT (OGT-WT) or its indicated mutants, followed by the infection with PR8 virus at a MOI = 1 for 12 h. Representative confocal microscopy images and quantification of subcellular localizations of OGT-WT and its K908A mutant in cells (<strong>a</strong>), <em>n</em> = 50 cells in EV group, <em>n</em> = 49 cells in OGT-WT group, <em>n</em> = 46 cells in OGT-K908A group. Scale bar: 15 μm. Representative confocal microscopy images and quantification of subcellular localizations of OGT-WT and its indicated mutants in cells (<strong>b</strong>), <em>n</em> = 50 cells in EV group, <em>n</em> = 43 cells in OGT-WT group, <em>n</em> = 47 cells in OGT ΔTPR4 group, <em>n</em> = 46 cells in OGT ΔTPR5 group, <em>n</em> = 50 cells in OGT-K908A group. Scale bar, 15 μm. <strong>c</strong> Volcano Plot of differential protein level analysis and subcellular localization of the OGT interactome from raw MS assay count data comparing the OGT and OGT PR8 infection groups. Red: upregulated protein (log<sub>2</sub> Fold Change &gt; 1 and adjust <em>p</em> &lt; 0.01; Gray: protein with no significant change (−1 &lt; log<sub>2</sub> Fold Change &lt; 1 or adjust <em>p</em> &gt; 0.01); Blue: downregulated protein (log<sub>2</sub> Fold Change &lt; -1 and adjust <em>p</em> &lt; 0.01). Circle: nuclear protein; Triangle: non- nuclear protein. <strong>d</strong>–<strong>f</strong> OGT-WT or its indicated mutants HT29 cells were infected with PR8 virus at a MOI = 1 for 12 h. OGT-WT and its K908A mutant (<strong>d</strong>) or OGT-WT and its indicated mutants (<strong>e</strong>)-interacting proteins were assessed by immunoprecipitation assay and immunoblotting with indicated antibodies. <strong>f</strong> Lipid droplets levels in OGT-WT and its mutant cells were stained with LipidTOX Deep Red and assessed by flow cytometry. <strong>g</strong>–<strong>j</strong>
<em>PLIN2-KO</em> and WT control of HT29 EV, OGT-FL and its indicated mutant cells were infected with PR8 virus at a MOI = 1 for 12 h. <strong>g</strong> PLIN2 protein expression in cells was assessed by immunoblotting. Viral mRNAs (<strong>h</strong>), viral titers (<strong>i</strong>), and gene transcripts including <em>IFNB</em>, <em>MX1</em>, and <em>ISG15</em> in t<strong>h</strong>e cells (<strong>j</strong>) were measured with RT-PCR and TCID<sub>50</sub> assays, respectively. Data are representative of three independent experiments (<strong>a</strong>, <strong>b</strong>, <strong>f</strong> and <strong>h</strong>–<strong>j</strong>). Statistical significance was determined by two-way ANOVA followed by Tukey’s test (<strong>a</strong>, <strong>b</strong>, <strong>f</strong> and <strong>h</strong>–<strong>j</strong>), <em>P</em> values of statistical comparisons are shown in each graph. The error bars represent SEM from <em>n</em> = 50 cells in EV group, <em>n</em> = 49 cells in OGT-WT group, <em>n</em> = 46 cells in OGT-K908A group in (<strong>a</strong>), <em>n</em> = 50 cells in EV group, <em>n</em> = 43 cells in OGT-WT group, <em>n</em> = 47 cells in OGT ΔTPR4 group, <em>n</em> = 46 cells in OGT ΔTPR5 group, <em>n</em> = 50 cells in OGT-K908A group in (<strong>b</strong>), and <em>n</em> = 3 independent biological replicates (<strong>f</strong>, <strong>h</strong>–<strong>j</strong>). Statistical significance was calculated using two-sided Wald test and ad<strong>j</strong>usted with Benjamini and Hochberg method (<strong>c</strong>). Source data are provided as a <a href="#MOESM6" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par20">We next tested the hypothesis that OGT antagonizes viral replication through its interaction with cytoplasmic components upon IAV infection. To compare the interactome of OGT in the absence or presence of IAV infection, we immunoprecipitated OGT-containing protein complexes from cells reconstituted with GFP-tagged OGT or empty vector, followed by liquid chromatography coupled to tandem MS (LC-MS/MS) assay (Supplementary Data <a href="#MOESM3" class="usa-link">1</a>). A total of 459 proteins were detected that specifically coimmunoprecipitated with OGT-GFP, but not with empty vector. Importantly, we discovered perilipin 2 (PLIN2) and PLIN3 as enriched proteins in the precipitates of OGT-GFP from IAV- infected cells, showed at volcano plot analysis of OGT interactome (Fig. <a href="#Fig6" class="usa-link">6c</a>). We cross-examined OGT-interacting proteins between this study and two other studies<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a>,<a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup>. Twenty-two proteins were identified that were shared by three studies, including several well-known OGT-interacting proteins such as USP10 and TET3 (Supplementary Fig. <a href="#MOESM1" class="usa-link">5b</a> and Supplementary Table <a href="#MOESM1" class="usa-link">3</a>). The limited number of OGT-interacting proteins shared by three studies is presumably due to different cell types and stimulation approaches utilized in three studies. PLIN2 and 3 were identified as OGT-interacting proteins only in our study, indicating that OGT-PLIN2/3 interaction was specific in response to IAV challenge. Moreover, coimmunoprecipitation assay confirmed that the interaction between OGT and PLIN2 and PLIN3 was enhanced following IAV stimulation (Fig. <a href="#Fig6" class="usa-link">6d</a>). OGT interaction with PLIN2 and PLIN3 was independent of its enzyme activity (Fig. <a href="#Fig6" class="usa-link">6d</a>), but instead required TPR4 (Fig. <a href="#Fig6" class="usa-link">6e</a>), which was consistent with TPR4-dependent OGT cytoplasmic translocation.</p>
<p id="Par21">PLIN2 and PLIN3 are essential proteins that maintain the stability of cellular LDs<sup><a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. Previous studies have well documented that LDs are important organelles downstream of fatty acid synthesis and promote viral replication<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>–<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Consistent with IAV-induced OGT interaction with PLIN2 and PLIN3, we observed remarkedly elevated colocalization between OGT and LDs following IAV infection, indicating that OGT was translocated to LDs (Supplementary Fig. <a href="#MOESM1" class="usa-link">5c</a>). Reconstitution of <em>OGT</em>-KO cells with WT OGT significantly reduced the number of LDs following IAV infection (Fig. <a href="#Fig6" class="usa-link">6f</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">5e</a>). Therefore, OGT relocation to LDs correlates with an inhibitory role in LDs accumulation upon IAV infection. OGT-K908A mutant showed a comparable level of OGT-LDs colocalization and LDs inhibition as WT OGT, indicating no involvement of OGT catalytic activity in these processes. In contrast, deletion of OGT TPR4, but not TPR5, decreased OGT-LDs colocalization and showed no inhibitory effect on LDs accumulation compared to WT OGT. In sum, these results support that TPR4-dependent OGT-viral genomic RNA interaction promotes OGT relocation to LDs and negatively affects LDs accumulation.</p>
<p id="Par22">We next employed both pharmacological and genetic approaches to examine if reduced LDs accumulation was responsible for TPR4-mediated antiviral function of OGT. Acetyl-CoA carboxylase (ACC) inhibitor (TOFA)<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a></sup> or diacylglycerol <em>O</em>-acyltransferase (DGAT1) inhibitor (A922500) were used to inhibit fatty acid synthesis and LDs formation, respectively. Pretreatment of cells with either TOFA (50 μg/ml) or A922500 (20 μg/ml) reduced IAV <em>NS1</em> and <em>NP</em> transcript levels and viral titers in cells with OGT ΔTPR4 to those in cells with WT OGT or OGT ΔTPR5 (Supplementary Fig. <a href="#MOESM1" class="usa-link">6a, b</a>). In addition, TOFA or A922500 treatment did not significantly affect ISG production compared to vehicle control (Supplementary Fig. <a href="#MOESM1" class="usa-link">6c</a>). We further genetically deleted PLIN2 in cells with various OGT genetic background (Fig. <a href="#Fig6" class="usa-link">6g</a>) and found that PLIN2 deletion reduced IAV transcript levels (Fig. <a href="#Fig6" class="usa-link">6h</a>) and viral titers (Fig. <a href="#Fig6" class="usa-link">6i</a>) in cells with OGT ΔTPR4 to those in cells with WT OGT or OGT ΔTPR5 without affecting ISG production (Fig. <a href="#Fig6" class="usa-link">6j</a>). Furthermore, the virus-restriction role of OGT-K908A compared to empty vector was completely abolished upon PLIN2 deletion (Fig. <a href="#Fig6" class="usa-link">6h, i</a>). In sum, these results suggest TPR4-mediated OGT antiviral function depends on LDs inhibition, which supplements catalytic activity-dependent effect of OGT on MAVS signaling to antagonize viral replication.</p></section><section id="Sec8"><h3 class="pmc_sec_title">OGT promotes PLIN2 degradation via K48-linked ubiquitination</h3>
<p id="Par23">It has been well documented that viral infection usually leads to enhanced fatty acid synthesis and LDs accumulation, which eventually benefits viral replication<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>–<a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. We next sought to determine the mechanism by which TPR4-dependent OGT LDs translocation suppresses LDs accumulation. Based on IAV-induced OGT interaction with PLIN2 and PLIN3 (Fig. <a href="#Fig6" class="usa-link">6d, e</a>), we examined proteins levels of multiple perilipins by immunoblotting and found that only PLIN2 protein was remarkedly reduced by IAV infection in cells with WT OGT (Fig. <a href="#Fig7" class="usa-link">7a</a>). Decreased PLIN2 protein was not due to lower gene transcription (Fig. <a href="#Fig7" class="usa-link">7b</a>), indicating a viral-induced PLIN2 protein degradation. We further found that PLIN2 protein degradation was independent of catalytic activity of OGT, but instead required OGT TPR4, suggesting that TPR4-dependent OGT-PLIN2 interaction causes PLIN2 degradation (Fig. <a href="#Fig7" class="usa-link">7a</a>).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7. OGT promotes PLIN2 degradation via K48-linked ubiquitination.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12365020_41467_2025_63085_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/2bb8/12365020/aa7ebcb5e1ce/41467_2025_63085_Fig7_HTML.jpg" loading="lazy" id="d33e2436" height="952" width="790" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>a</strong>, <strong>b</strong>
<em>OGT</em>-KO HT29 cells were reconstituted with empty vector (EV), GFP-tagged OGT (OGT-WT) or its indicated mutants, followed by the infection with PR8 virus at a MOI = 1 for 12 h. PLIN2-5 proteins expression (<strong>a</strong>) and PLIN2 mRNA (<strong>b</strong>) in cells were assessed by immunoblotting and RT-PCR respectively. <strong>c</strong>–<strong>e</strong> 293T cells were transfected with GFP-tagged PLIN2 and Myc-tagged OGT, in the presence of HA-tagged ubiquitin WT, K48-only, or K63-only mutants. After 48 h of transfection, cell lysates were immunoprecipitated with anti-GFP Agarose. Total (<strong>c</strong>), K48-only (<strong>d</strong>) or K63-only (<strong>e</strong>) ubiquitinated PLIN2 was assessed by immunoblotting with indicated antibodies. <strong>f</strong>, <strong>g</strong> 293T cells were transfected with GFP-tagged PLIN2 and Myc-tagged OGT, in the presence of HA-tagged ubiquitin WT and K48-only. After 30 h of transfection, cells were treated with MG132 (10 μΜ) for 6 h, and then the cell lysates were immunoprecipitated with anti-GFP Agarose. Total (<strong>f</strong>) and K48-only (<strong>g</strong>) ubiquitinated PLIN2 was assessed by immunoblotting with indicated antibodies. OGT-WT and its K908A mutant (<strong>h</strong>) or OGT-WT and its indicated mutants (<strong>i</strong>) HT29 cells were infected with PR8 virus at a MOI = 1 for 24 h, total PLIN2 was immunoprecipitated and followed by immunoblotting with anti-ubiquitin antibody. Data are representative of three independent experiments (<strong>b</strong>). Statistical significance was determined by two-way ANOVA followed by Tukey’s test (<strong>b</strong>), <em>P</em> values of statistical comparisons are shown in each graph. The error bars represent SEM from <em>n</em> = 3 independent biological replicates (<strong>b</strong>). Source data are provided as a <a href="#MOESM6" class="usa-link">Source Data</a> file.</p></figcaption></figure><p id="Par24">Previous studies have reported that OGT affects protein stability via a ubiquitin-proteasome system (UPS)-dependent mechanism<sup><a href="#CR64" class="usa-link" aria-describedby="CR64">64</a>–<a href="#CR67" class="usa-link" aria-describedby="CR67">67</a></sup>. We therefore examined whether OGT promotes PLIN2 protein degradation via ubiquitination. When 293T cells were transfected with plasmids expressing GFP-tagged PLIN2 and HA-tagged ubiquitin, total ubiquitination, K48-linked (K48-Ub) or K63-linked (K63-Ub) ubiquitination of PLIN2 were readily detected. Co-expression of OGT induced further enhancement in total and K48-Ub of PLIN2 but showed no effect on its K63-Ub (Fig. <a href="#Fig7" class="usa-link">7c</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">7a</a>). We further assessed the effect of OGT on PLIN2 ubiquitination using either K48-only or K63-only ubiquitin mutants, which contain replacement of all lysine residues by alanine except lysine at position 48 and 63, respectively<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. OGT increased PLIN2 ubiquitination in the presence of K48-only ubiquitin (Fig. <a href="#Fig7" class="usa-link">7d</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">7b</a>), but not K63-only ubiquitin (Fig. <a href="#Fig7" class="usa-link">7e</a> and Supplementary Fig. <a href="#MOESM1" class="usa-link">7c</a>), suggesting that OGT promotes K48-Ub of PLIN2. Treatment with proteosome inhibitor MG132 further elevated total and K48-Ub of PLIN2 (Fig. <a href="#Fig7" class="usa-link">7f, g</a>). Moreover, IAV infection for 24 h induced a robust PLIN2 ubiquitination, which correlated with PLIN2 protein degradation, in the presence of WT OGT (OGT-WT, Fig. <a href="#Fig7" class="usa-link">7h</a>). Deletion of OGT completely abolished PLIN2 ubiquitination and protein degradation, indicating a critical role of OGT in promoting PLIN2 protein degradation through K48-Ub. We further found that OGT-dependent PLIN2 ubiquitination and protein degradation did not involve its enzyme activity (OGT-K908A, Fig. <a href="#Fig7" class="usa-link">7i</a>), but instead required TPR4 of OGT (OGT ΔTPR4, Fig. <a href="#Fig7" class="usa-link">7i</a>). Together, these findings suggest that TPR4-dependent OGT-PLIN2 interaction correlates with PLIN2 ubiquitination and protein degradation, which is accompanied with LDs inhibition and viral replication (Supplementary Fig. <a href="#MOESM1" class="usa-link">8</a>).</p></section></section><section id="Sec9"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par25">Since its initial discovery, OGT has been most recognized for its catalytic activity-dependent role in modifying proteins with <em>O</em>-GlcNAcylation<sup><a href="#CR68" class="usa-link" aria-describedby="CR68">68</a></sup>. Studying the catalytic activity-independent functions of OGT is crucial for enhancing our overall understanding of this key metabolic enzyme. To specifically focus on the catalytic activity-independent functions of OGT, we generated a conditional OGT catalytically impaired knock-in mouse and examined its phenotypes in an IAV infection model. Most surprisingly, OGT-K908A mutation decreased viral replication both in vitro and in vivo compared to OGT KO, although it was more permissive for viral replication than WT OGT. IAV infection activates RIG-I and subsequently induces IFN-I and ISGs in virus-infected cells<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. Our previous study demonstrated that OGT catalytic activity-dependent antiviral effect requires MAVS- mediated IFN-I signaling<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. In this study, we found that the inhibition of IAV replication by the catalytically impaired OGT K908A mutant involves a mechanism unrelated to MAVS signaling. These findings significantly expand the current knowledge of OGT by demonstrating that OGT has catalytic activity-independent antiviral function.</p>
<p id="Par26">There are two evolutionarily conserved mechanisms for sensing viral pathogen-associated molecular patterns (PAMPs) that contribute to the innate immune response against viral infection<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>–<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a></sup>. One mechanism is mediated by RLRs and involves the induction of IFN-I and ISGs<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>,<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. The other mechanism involves programmed cell death mediated by nucleotide-binding domain leucine-rich repeat (NLR) proteins<sup><a href="#CR69" class="usa-link" aria-describedby="CR69">69</a></sup>, such as NLRP1<sup><a href="#CR70" class="usa-link" aria-describedby="CR70">70</a></sup> and NLRP3<sup><a href="#CR71" class="usa-link" aria-describedby="CR71">71</a></sup> or ZBP1<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. A widely accepted concept is that activation of immune signaling in response to virus infection reprograms cellular metabolic activities, which not only provides the energy and substrates for efficient immune responses but also further conveys danger signals to other restorative stress responses<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a>,<a href="#CR72" class="usa-link" aria-describedby="CR72">72</a></sup>. Interestingly, one study reported that the glycolytic enzyme hexokinase unexpectedly acts as a primary pattern recognition receptor (PRR) that recognizes bacterial peptidoglycan and triggers activation of inflammasomes<sup><a href="#CR73" class="usa-link" aria-describedby="CR73">73</a></sup>. However, whether viral components can be detected directly by metabolic sensors as danger signals remains completely unknown. We sought to examine the mechanism by which OGT-K908A inhibits IAV replication through biochemical studies. Intriguingly, we discovered that OGT interacts with IAV genomic RNA in a manner independent of its catalytic activity. Thus, we propose that OGT, as a metabolic sensor, represents a branch in initiating the antiviral immune response by sensing IAV genomic RNA. It is worth noting that in some human cells, the catalytic activity of OGT may promote replication of viruses other than IAV such as herpes simplex virus and human cytomegalovirus based on the assays with the use of OGT inhibitor OSMI−1<sup><a href="#CR74" class="usa-link" aria-describedby="CR74">74</a></sup>. Therefore, it seems that how OGT and <em>O</em>-GlcNAc levels impact viral infection are dependent on the virus and cell model used.</p>
<p id="Par27">In this study, <em>Ogt</em><sup><em>K908A</em></sup> conditional knock-in model was achieved by crossing <em>Ogt-K908A</em><sup><em>fl/fl</em></sup> with individual Cre-expressing deleter mouse strain. For Cre-loxP-mediated recombinant system, it is well known that individual Cre systems driven by different promoters function with different efficiencies. In our <em>Ogt</em><sup><em>K908A</em></sup> conditional knock-in models (cells and animals), it is unlikely that Cre-loxP-mediated recombinant will lead to 100% replacement of WT OGT by K908A. Similarly, <em>Ogt</em><sup><em>KO</em></sup> conditional knockout (by using <em>Ogt</em><sup><em>fl/fl</em></sup>) will unlikely achieve an 100% <em>Ogt</em> gene deletion, as observed in most studies using Cre-loxP system. Regarding the effect of K908A mutation on the loss of catalytic activity, two studies reported a less than 10% activity of K908A mutant<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. Therefore, incomplete knock-in efficacy and residual activity of K908A mutant are two major contributors to some residual OGT catalytic activity in K908A knock-in models in vitro and in vivo. Despite an incomplete loss-of-catalytic-activity of OGT in K908A knock-in models, we observed comparable levels of activation of antiviral immune signaling and ISG induction between <em>Ogt</em><sup><em>K908A</em></sup> and <em>Ogt</em><sup><em>KO</em></sup> cell and animals. These results support that impaired catalytical activity of OGT in <em>Ogt</em><sup><em>K908A</em></sup> cells and animal is sufficient to cause a similar defect phenotype in MAVS antiviral signaling compared to <em>Ogt</em><sup><em>KO</em></sup> models. Therefore, attenuated viral replication in <em>Ogt</em><sup><em>K908A</em></sup> models compared to <em>Ogt</em><sup><em>KO</em></sup> models may not be explained by residual activation of MAVS antiviral signaling. Indeed, when MAVS or IRF3 was removed from cells containing empty vector, OGT-WT or K908A mutant, different viral replication was abolished between cells containing OGT-WT or K908A, suggesting that OGT catalytic activity-mediated antiviral effect fully requires MAVS signaling. Meanwhile, OGT K908A still inhibits viral replication compared to cells without OGT, therefore highlighting OGT catalytic activity-independent antiviral effect.</p>
<p id="Par28">OGT is composed of two separate domains: the N-terminal TPR repeats, which mediate the recognition of a broad range of target proteins and the C-terminal domain, which has glycosyltransferase activity<sup><a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>,<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a></sup>. Previous work has shown that IFIT1 and IFIT5 directly bind 5′ triphosphate RNA (5′ ppp-RNA) in a non-sequence-specific manner through their TPR motifs, representing a model of TPR-containing proteins that sense viral RNA<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. The structural similarity between OGT and IFIT proteins suggests that viral RNA interaction may account for the antiviral ability of OGT-K908A mutant against IAV infection. Our current study showed that the N-terminal TPR4 motif of OGT is indeed required for the interaction between OGT and IAV genomic RNA. Deletion of TPR4 motif impaired the anti-IAV function of OGT, genetically verifying the hypothesis that the interaction with viral RNA is critical for OGT’s antiviral activity. This is similar to the antiviral mechanism of IFIT1/5 protein<sup><a href="#CR47" class="usa-link" aria-describedby="CR47">47</a></sup>. IAV PA and PB2 genomic RNAs, which are ligands for RIG-I<sup><a href="#CR75" class="usa-link" aria-describedby="CR75">75</a>,<a href="#CR76" class="usa-link" aria-describedby="CR76">76</a></sup>, were also sensed by OGT in this study. Whether OGT interacts with shorter subviral IAV gene segments, defective interfering (DI) RNA or dsRNA, or whether special RNA modifications are acquired for this interaction, remain to be elucidated. Moreover, our current results do not support that TPR4 directly interact with viral RNA. Instead, we reason that TPR4 may serve as a structural bridge between TPR3 and TPR5, facilitating the formation of a continuous TPR motif. This configuration may mediate RNA interacting through adjacent structural elements enriched in basic residues (e.g., R102, R113, R117, R172, and K182), which collectively contribute to the RNA-interacting interface of the OGT. Detailed interacting features between OGT and viral RNA requires structural biology techniques such as cryogenic electron microscopy in future studies.</p>
<p id="Par29">Here, we found that LDs accumulated significantly after IAV infection but were inhibited in the presence of OGT, indicating that viral infection reprograms lipid metabolism in cells and OGT negatively regulates this process. This finding is consistent with recent studies showing that viral infection enhances the activity of the fatty acid biosynthesis pathway<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a>–<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>,<a href="#CR77" class="usa-link" aria-describedby="CR77">77</a>–<a href="#CR79" class="usa-link" aria-describedby="CR79">79</a></sup>. Enhanced fatty acid and LDs biosynthesis is required for virus replication<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup>. The perilipin family members, including PLINs 1-5, are major LD-associated proteins<sup><a href="#CR80" class="usa-link" aria-describedby="CR80">80</a>,<a href="#CR81" class="usa-link" aria-describedby="CR81">81</a></sup>. We identified the interaction between OGT and PLIN2 through unbiased LC-MS/MS analysis and found OGT induced PLIN2 degradation through its TPR4 motif. Importantly, PLIN2 deficiency leads to significantly decreased viral replication in OGT TPR4 deletion mutant cells, suggesting that PLIN2 is a key target of OGT on LDs. This finding aligns with recent studies showing that inhibition of the fatty acid biosynthesis pathway and LDs formation effectively restrict virus replication across a wide range of virus types<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a>,<a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>,<a href="#CR82" class="usa-link" aria-describedby="CR82">82</a>,<a href="#CR83" class="usa-link" aria-describedby="CR83">83</a></sup>, including IAV<sup><a href="#CR84" class="usa-link" aria-describedby="CR84">84</a></sup>, thus supporting the promising therapeutic potential of targeting OGT in viral infectious diseases. It has been known for a long time that <em>O</em>-GlcNAc signaling significantly impacts lipid metabolism<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>,<a href="#CR85" class="usa-link" aria-describedby="CR85">85</a>,<a href="#CR86" class="usa-link" aria-describedby="CR86">86</a></sup>. Here, our study showed that directly targeting PLIN2 by OGT resulted in a significant lipid metabolism reprogramming towards decreased LDs formation, demonstrating that the OGT-LDs axis is an antiviral route based on the crosstalk networks of sugar and lipid metabolism. In summary, our findings reveal OGT as a multifaceted metabolic sensor that integrates MAVS-dependent antiviral innate immune signaling and lipid metabolism. This expands our current understanding of the crosstalk between immune signaling pathways and glucose and lipid metabolism, highlighting the essential role of immunometabolism in viral infection-associated diseases.</p></section><section id="Sec10"><h2 class="pmc_sec_title">Methods</h2>
<section id="Sec11"><h3 class="pmc_sec_title">Mice</h3>
<p id="Par30">For the generation of an inducible <em>Ogt</em> allele with the replacement of lysine 908 by alanine (<em>Ogt-K908A</em>), a targeting vector was generated in which the mutant exons 20–22 was cloned downstream of the wild type exons 20–22. The K908A (AAA to GCA) mutation was introduced to the mutant exon 20. This mutant module containing exons 20–22 was inserted downstream of a cassette containing an FRT-flanked neomycin-resistance gene and LoxP-flanked wild type exons 20–22 and transcriptional stop element (Supplementary Fig. <a href="#MOESM1" class="usa-link">1a, b</a>). The targeting vector was generated by Biocytogen and used to establish <em>Ogt-K908A</em><sup><em>fl/fl</em></sup> mice by homologous recombination in BCGEN B61-6 (C57BL/6) embryonic stem cells. <em>Ogt-K908A</em><sup><em>fl/fl</em></sup> mice were generated after successful germ-line transmission and backcrossed to C57BL/6J. C57BL/6J mice (000664) and <em>R26-Cre-ER</em><sup><em>T2</em></sup> (referred to as <em>ER-Cre</em> mice, 008463) mice were obtained from The Jackson Laboratory. <em>Ogt</em><sup><em>fl/fl</em></sup> mice<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup> and <em>Foxj1-Scgb1a1-Cre</em> mice<sup><a href="#CR44" class="usa-link" aria-describedby="CR44">44</a></sup> have been previously described. <em>Ogt</em><sup><em>fl/fl</em></sup> and <em>Ogt-K908A</em><sup><em>fl/fl</em></sup> mice were crossed with <em>ER-Cre</em> mice to generate tamoxifen-inducible <em>Ogt</em> gene-deletion (<em>Ogt</em><sup><em>KO</em></sup>) and <em>Ogt</em> catalytically impaired (<em>Ogt</em><sup><em>K908A</em></sup>) strains, respectively. <em>Ogt</em><sup><em>ΔLEC</em></sup> and Ogt-<em>K908A</em><sup><em>ΔLEC</em></sup> mutant mice were generated by crossing <em>Ogt</em><sup><em>fl/fl</em></sup> and <em>Ogt-K908A</em><sup><em>fl/fl</em></sup> mice with <em>Foxj1-Scgb1a1-Cre</em> mice. Mice of both sexes were used in the study, gender-matched mice were randomly assigned to mice study. All mice were housed in standard rodent micro-isolator cages and acclimated to study conditions for at least 7 days before manipulation. Mice were kept in animal rooms maintained on 12-h light–dark cycle, temperature and humidity-controlled, between 68 and 74 ℉ and 30–70%, respectively. The animal work was approved by The Ohio State University and National Institute of Health Guide for the Care and Use of Laboratory Animals and the Institutional Animal Care and Use Committee (IACUC). All in vivo experiments were performed according to the guidelines established by The Ohio State University and National Institute of Health Guide for the Care and Use of Laboratory Animals and the Institutional Animal Care and Use Committee (protocol no. 2018A00000022-R1).</p></section><section id="Sec12"><h3 class="pmc_sec_title">Reagents and antibodies</h3>
<p id="Par31">Tamoxifen (B5965) was from APExBIO. Diethyl pyrocarbonate (DEPC) (AC170250050) was from Acros Organics. X-tremeGENE HP DNA transfection reagent (06366546001) was from Roche. 10% neutral buffered formalin (5701) and AF488 goat anti-mouse secondary antibody (A−11001) were from Thermo Fisher Scientific. Bio-Safe Coomassie G-250 Stain (1610786) was from Bio-Rad Laboratories. TOFA (10005263) and DGAT−1 inhibitor A922500 (10012708) was from Cayman Chemical. Polybrene (H9268) was from Sigma. Flag peptide (RP21087) was from GenScript. Antibodies for immunoblotting included anti-OGT (5368), anti-MAVS (4983), anti-p-TBK1 (S172) (5483), anti-TBK1 (3504), anti-p-IRF3 (S396) (4947), anti-IRF3 (4962), anti-p-IKKα/β (S176/180) (2697), anti-p-IκBα (S32) (2859), anti-IκBα (9242S), anti-p-p65 (S536) (3033), anti-p65 (8242), and anti-ubiquitin (3933) from Cell Signaling Technology; anti-IAV (B65141G) from Meridian Life Science, Inc.; anti-IAV NA (11058-T62) from Sino biological; anti-MAVS (sc-365334), anti-GFP (sc-9996) and HRP-conjugated anti-β-actin (sc−1615) from Santa Cruz Biotechnology; HRP-conjugated anti-HA (11667475001), HRP-conjugated anti-Myc (11814150001) and HRP-conjugated anti-Flag (A8592) from Sigma; anti-IKKα (07−1107) from EMD Millipore; anti-PLIN3 (10694−1-AP), anti-PLIN4 (55404−1-AP) and anti-PLIN5 (26951−1-AP) from Proteintech; anti-<em>O</em>-GlcNAc (ab2739), anti-PLIN2 (ab78920) and anti-influenza A virus NP (ab20343) from Abcam. Antibody-conjugated agarose beads for immunoprecipitation included Anti-Flag M2 Affinity Gel (A2220, Sigma), Pierce Anti-Myc Agarose (20168, Thermo Fisher Scientific), GFP-Trap resin (gta-20, ChromoTek), and Protein A/G UltraLink Resin (53132, Thermo Fisher Scientific).</p></section><section id="Sec13"><h3 class="pmc_sec_title">Cell culture</h3>
<p id="Par32">293T (CVCL_0063) and HT29 (CVCL_7297) cells were purchased from the American Type Culture Collection (ATCC). Madin-Darby Canine Kidney (MDCK) cells (NR-2628) were from BEI Resources. All these cells were cultured in Dulbecco’s Modified Eagle Medium (DMEM; Gibco) supplemented with 10% fetal bovine serum (FBS). Mouse embryonic fibroblasts (MEFs) were generated from day−13.5 embryos of <em>Ogt</em><sup><em>fl/fl</em></sup>, <em>Ogt-K908A</em><sup><em>fl/fl</em></sup><em>, Ogt</em><sup><em>KO</em></sup> and <em>Ogt</em><sup><em>K908A</em></sup> mice and cultured in DMEM supplemented with 10% FBS, as previously described<sup><a href="#CR87" class="usa-link" aria-describedby="CR87">87</a></sup> (Supplementary Table <a href="#MOESM1" class="usa-link">6</a>).</p></section><section id="Sec14"><h3 class="pmc_sec_title">Plasmids and molecular cloning</h3>
<p id="Par33">pCMV vector expressing human Myc-tagged full-length <em>OGT</em>, pWPXLd lentivector expressing OGT <em>WT</em> and K908A mutant have been previously described<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR63" class="usa-link" aria-describedby="CR63">63</a></sup>. pCMV-myc-<em>OGT</em> and pWPXLd-<em>OGT</em> were used as templates to generate a series of <em>OGT</em> mutants (Supplementary Table <a href="#MOESM1" class="usa-link">5</a>). pcDNA3-HA-ubiquitin and Flag-tagged <em>RIG-I</em> has been previously described<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>,<a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup>. pBJG1 T7-8xHis-OGT (154271) was purchased from the Addgene and used as template to generate pBJG1 T7-8xHis-OGT ΔTPR4. All primers used for cloning and mutants constructing are listed in the Supplementary Table <a href="#MOESM1" class="usa-link">2</a>.</p></section><section id="Sec15"><h3 class="pmc_sec_title">Cell transfection</h3>
<p id="Par34">293T cells were transfected with different combinations of expression plasmids as indicated in the Figure Legends for 48 h with X-tremeGENE HP DNA Transfection Reagent (Roche). For lentivector-based transduction, 293T cells were used to package the pWPXLd lentivirus expressing <em>OGT</em> WT or K908A mutant, harvesting supernatant from which and was further used for HT29 Cell transduction.</p></section><section id="Sec16"><h3 class="pmc_sec_title">RT-PCR</h3>
<p id="Par35">Total RNA was extracted from cultured cells or lung homogenates using TRIzol (15596018, Thermo Fisher Scientific). cDNA was generated from total RNA by using Moloney murine leukemia virus reverse transcriptase (28025013, Thermo Fisher Scientific) at 38 °C for 60 min. RT-PCR was performed using SYBR Green PCR Master Mix in an Applied Biosystems StepOnePlus detection system. The fold difference of mRNA between different groups was determined by a standard delta-delta Ct method. Mouse <em>β-actin</em> and human <em>GADPH</em> were analyzed as an internal control. The primer sequences of individual genes are listed in Supplementary Table <a href="#MOESM1" class="usa-link">1</a>.</p></section><section id="Sec17"><h3 class="pmc_sec_title">Immunoblotting and immunoprecipitation</h3>
<p id="Par36">For immunoblotting assay, electrophoresis of proteins was performed by using the NuPAGE system (Thermo Fisher Scientific) according to the manufacturer’s protocol. Briefly, cultured cells were collected and lysed with RIPA buffer. Proteins were separated on a NuPAGE gel and were transferred onto nitrocellulose membranes (Bio-Rad). Appropriate primary antibodies and HRP-conjugated secondary antibodies were used for immunoblotting and proteins were detected using the Enhanced Chemiluminescent (ECL) reagent (Thermo Fisher Scientific). The images were acquired with ChemiDoc MP System (Bio-Rad). For immunoprecipitation assay, cells were lysed with RIPA buffer supplemented with cOmplete<sup>TM</sup> Protease Inhibitor Cocktail (Sigma). Cell lysates were incubated with Anti-Flag M2 Affinity Gel, Pierce Anti-Myc Agarose, or GFP-Trap resin overnight at 4 °C. The mixture was washed for 5 times with cold RIPA buffer and boiled at 95 °C for 5 min, followed by immunoblotting.</p></section><section id="Sec18"><h3 class="pmc_sec_title">ELISA</h3>
<p id="Par37">IFN-β in lung tissue homogenates from mice study was quantified using the ELISA, which was performed using rat monoclonal anti-mouse IFN-β (7891, Cosmo Bio USA) as capture antibody and rabbit polyclonal anti-mouse IFN-β (32400−1, R&amp;D Systems) as detection antibody, as previously described<sup><a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>.</p></section><section id="Sec19"><h3 class="pmc_sec_title">Influenza A virus infection in vivo</h3>
<p id="Par38">Sex- and age-matched <em>Ogt</em><sup><em>fl/fl</em></sup>, <em>Ogt</em><sup><em>ΔLEC</em></sup> and Ogt-<em>K908A</em><sup><em>ΔLEC</em></sup> mice (7-week-old) were intranasally inoculated with influenza A/Puerto Rico/8/1934 (PR8; H1N1) strain at a 50% tissue culture infectious dose (TCID<sub>50</sub>) of 10 in 50 μl of PBS per mouse. Animal survival and body weight were monitored for 20 days after PR8 infection. Lung tissues were collected at indicated timepoints after infection for virus titration, immunological and histological analyses. Lung tissues were removed from mice at 2 days after IAV infection and fixed with 10% neutral-buffered formaldehyde. Fixed tissues were embedded in paraffin wax, sectioned and stained with Hematoxylin-Eosin (H&amp;E) for histopathology assessment.</p></section><section id="Sec20"><h3 class="pmc_sec_title">Virus titration</h3>
<p id="Par39">MDCK cells were seeded in 96-well plates, infected with 10-fold serially diluted viruses, and incubated at 37 °C for 72 h. Cells were fixed with 4% neutral buffered formaldehyde at room temperature for 30 min and permeabilized in PBS containing 0.1% Triton X−100 for 10 min and blocked at 37 °C for 1 h using 1% bovine serum albumin in PBS. Cells were incubated with anti-IAV NP protein monoclonal antibody at a dilution of 1:3000 at room temperature for 2 h, followed by incubation with AF488 goat anti-mouse secondary antibody at a dilution of 1:5000. Virus-positive cells were visualized under a microscope. 50% tissue culture infectious dose (TCID<sub>50</sub>) values were calculated according to the Reed-Muench method<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a></sup>.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Flow cytometry</h3>
<p id="Par40"><em>Ogt</em><sup><em>fl/fl</em></sup>, <em>Ogt-K908A</em><sup><em>fl/fl</em></sup><em>, Ogt</em><sup><em>KO</em></sup> and <em>Ogt</em><sup><em>K908A</em></sup> MEFs were infected with PR8-GFP at an MOI of 1 for 12 h and then were fixed with 4% paraformaldehyde for 10 min at room temperature. The percentage of GFP-positive cells was quantified by flow cytometry. <em>OGT</em>-KO HT29 cells reconstituted with empty vector, GFP-tagged WT <em>OGT</em>, or a series of <em>OGT</em> mutants were infected with IAV (MOI of 1) for 12 h. Cells were stained with HCS LipidTOX™ Deep Red Lipid Stain at room temperature for 30 mins for detecting lipid droplets. Flow cytometry analysis was performed by a BD FACS-Canto flow cytometer and FlowJo software was used to analyze the data.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Confocal microscopy</h3>
<p id="Par41"><em>OGT</em>-KO HT29 cells reconstituted with empty vector, GFP-tagged WT OGT, or a series of OGT mutants were seeded in 8-well chamber slides (154526, Thermo Fisher Scientific). 12 h after PR8 infection (MOI = 1), cells were fixed with 4% paraformaldehyde in PBS for 8 min at room temperature, and were permeabilized with 0.01% Triton X−100 for 10 min, followed by blocking with 2% BSA at room temperature for 30 min. The samples were incubated with HCS LipidTOX™ Deep Red Lipid Stain (<a href="https://www.ncbi.nlm.nih.gov/nuccore/H34477" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">H34477</a>, Thermo Fisher Scientific) for 30 min, followed by the incubation with DAPI (H−1200, Vector Laboratories) to stain nuclei. Images were acquired using a point scanning confocal unit (FV3000 Spectral, Olympus Scientific) on an Olympus IX83 inverted microscope stand (Olympus Scientific) equipped with a 60x/1.42 NA Plan ApoN objective lens (Olympus Scientific) and immoil-F30CC type F immersion oil (Olympus Scientific). DAPI fluorescence was excited with 405 laser line and collected using GaAsp multi-alkali photomultiplier tubes (PMT) with the following detection wavelengths 420-470 nm. Alexa Fluor 488 fluorescence was excited with 488 laser line and collected using GaAsp multi-alkali PMT with the following detection wavelengths 500-550 nm. Alexa Fluor 647 fluorescence was excited with 647 laser line and collected using GaAsp PMT with the following detection wavelengths 660–740 nm. Images were acquired with a Galvano scanner sequentially by line scanning at 4.0 μs pixel dwell time and acquired at 1.5× scan zoom, 1024 × 1024 image size, 3× line averaging controlled with FV315-SW software (Olympus Scientific). The intracellular distribution of GFP-OGT was quantified using the following ratio: (green fluorescence intensity of the cytosol/cytosolic area)/(green fluorescence intensity of the nucleus/nuclear area). The green fluorescence intensity of the cytosol was determined by subtracting the nuclear green fluorescence intensity from the total cellular green fluorescence intensity. Fluorescence intensity and area measurements for each cellular compartment were automatically obtained using the “Measure” function in ImageJ. To analyze OGT-lipid droplet colocalization, Pearson’s correlation coefficients were automatically calculated for each cell using the Coloc 2 algorithm in Fiji ImageJ 1.52p. For each biological replicate, ~10 cells per field were quantified across five randomly selected fields of view per group. A total of three biological replicates were performed.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Preparation of nuclear and cytoplasmic extracts from Cells</h3>
<p id="Par42">Nuclear and cytoplasmic extracts were prepared from HT29 cells using the Nuclear Extract Kit (40010, Active Motif) according to the manufacturer’s instructions. Briefly, HT29 cells grown in a 100 mm dish (~9 × 10<sup>6</sup> cells) were washed with ice-cold PBS containing phosphatase inhibitors. Next, 3 mL of ice-cold PBS with phosphatase inhibitors was added to each dish. Cells were scraped and collected by centrifugation at 400 × <em>g</em> for 5 min at 4 °C. The cell pellet was gently resuspended in 500 µL of 1X hypotonic buffer and incubated on ice for 15 min. Subsequently, 25 µL of detergent was added, and the suspension was vortexed at the highest setting for 10 s. The lysate was centrifuged at 14,000 × <em>g</em> for 30 s at 4 °C. The supernatant (cytoplasmic fraction) was collected and stored for immunoblotting analysis. The remaining pellet was resuspended in 50 µL of complete lysis buffer and incubated on ice for 30 min on a rocking platform set at 150 rpm. After incubation, the suspension was vortexed for 30 s at the highest setting and centrifuged at 14,000 × <em>g</em> for 10 min at 4 °C. The supernatant (nuclear fraction) was collected for immunoblotting analysis.</p></section><section id="Sec24"><h3 class="pmc_sec_title">In vitro RNA transcription</h3>
<p id="Par43">Templates for T7 RNA transcription reaction were generated by PCR reaction from plasmid encoding PB2 segment of IAV PR8 virus using forward primer AGCRAAAGCAGGTCAATTATATTCA and reverse primer GCTAATACGACTCACTATAGGGAGTAGAAACAAGG. Plasmids encoding individual RNA segments of IAV PR8 virus have been previously described<sup><a href="#CR89" class="usa-link" aria-describedby="CR89">89</a></sup>. T7 transcription reactions were carried out with MEGAscript™ T7 Transcription Kit according to the manufacturer’s instructions (AM1333, Thermo Fisher Scientific). DNA was digested with 0.05 U/ml of DNase I (EN0525, Thermo Fisher Scientific) and incubated at 37 °C for 1 h followed by lithium chloride precipitation. Final concentrations of IAV PB2 genomic RNA products were measured by NanoDrop Spectrophotometer (Thermo Fisher Scientific) and further analyzed on denaturing agarose gels for correct size.</p></section><section id="Sec25"><h3 class="pmc_sec_title">OGT protein generation and purification</h3>
<p id="Par44">For eukaryotic expression of human OGT protein, 293T cells were transfected with Flag-tagged <em>OGT</em> WT or mutants. Flag-tagged RIG-I was included as a positive control. After 48 h of transfection, cells were lysed with RIPA buffer supplemented with complete protease inhibitor cocktail (Sigma). Cell lysates were incubated with anti-Flag M2 affinity gel (Sigma) overnight at 4 °C. The bead mixture was washed 5 times with cold RIPA buffer and then eluted with Flag peptide (500 ng/ml, RP21087, GenScript) at 4 °C for 6 h. The supernatant was analyzed on SDS-PAGE gel with Coomassie blue staining and subjected to electrophoretic mobility shift assay. For prokaryotic expression of human OGT, pBJG1 T7-8xHis-OGT (154271, Addgene) and pBJG1 T7-8xHis-OGT ΔTPR4 were transformed in BL21 (DE3) cells and grown at 37 °C to an OD = 0.9−1.0. The culture was cooled to 16 °C and induced with 0.2 mM IPTG. After 20 h, cells were collected and resuspended in the binding buffer (250 mM NaCl, 20 mM Tris, 40 mM imidazole, pH = 7.7) and lysed with ultrasonication. Cell lysates were clarified by centrifugation at 17,000 × <em>g</em>, then loaded onto Ni<sup>2+</sup>-charged His Bind RESIN (69670, Sigma). The column was washed with three column volumes of 50 mM imidazole, pH = 7.5, 250 mM NaCl, and the protein was eluted with 250 mM imidazole, pH = 7.5, 250 mM NaCl. Elute fraction was dialyzed against 150 mM NaCl, 20 mM Tris, pH = 7.5 and stored at −80 °C.</p></section><section id="Sec26"><h3 class="pmc_sec_title">OGT-IAV genomic RNA interaction</h3>
<p id="Par45">Two reciprocal immunoprecipitation assays were performed to investigate OGT interaction with IAV genomic RNA. First, 293 T cells were transfected with Flag-tagged <em>OGT</em> WT or a series of indicated mutants. Flag-tagged RIG-I was included as a positive control, as previously described<sup><a href="#CR88" class="usa-link" aria-describedby="CR88">88</a></sup>. At 12 h after transfection, cells were infected with IAV (MOI of 1). Cell lysates were harvested at 48 h after transfection and was incubated with Anti-Flag M2 Affinity Gel (Sigma) at 4 °C for 6 h. After washes of 3 times with lysis buffer, total RNA was extracted from beads mixture using TRIzol reagent. cDNA of viral <em>PA</em> and <em>PB2</em> genomic RNA was generated using Uni−12 primer, which were further quantified by RT-PCR using the primers listed in the Supplementary Table <a href="#MOESM1" class="usa-link">1</a> and the RT-PCR products were sequenced (Supplementary Table <a href="#MOESM1" class="usa-link">4</a>). Second, <em>PB2</em> genomic RNA of IAV generated via in vitro transcription was biotinylated with Pierce RNA 3′ End Biotinylation Kit (20160, Thermo Fisher Scientific) according to the manufacturer’s instructions. Biotinylated RNA was incubated with MyOne Streptavidin C1 beads (Thermo Fisher Scientific) at 4 °C for 4 h with rotation in the presence of RNase inhibitor. RNA and beads mixture were washed three times and incubated with 1 ml of cell lysate containing overexpressed OGT at 4 °C for 6 h with rotation. The mixture was then washed three times and analyzed by immunoblotting. The mixture of cell lysate containing overexpressed OGT and MyOne Streptavidin C1 beads in the absence of PB2 genomic RNA was included as a negative control.</p></section><section id="Sec27"><h3 class="pmc_sec_title">Electrophoretic mobility shift assay</h3>
<p id="Par46">PB2 genomic RNA of IAV generated via in vitro transcription (600 ng) was mixed with Flag-tagged <em>OGT</em> WT or mutants or Flag-tagged RIG-I protein at indicated concentrations in PBS buffer with additional 5 mM MgCl<sub>2</sub> for 30 min at room temperature. Samples were then separated by non-denaturing 1% agarose gel electrophoresis in 0.5 × TB buffer (44.5 mM Tris, 44.5 mM boric acid) at 130 V for 1 h. For RNA signal, the gel was prestained with ethidium bromide and was imaged on ChemiDoc Touch Imaging System. The gel was stained with Bio-Safe Coomassie G-250 Stain and imaged on ChemiDoc Touch Imaging System for protein signal.</p></section><section id="Sec28"><h3 class="pmc_sec_title">Isothermal titration calorimetry assay</h3>
<p id="Par47">Isothermal titration calorimetry (ITC) assay was carried out using a Nano ITC (TA waters), as previously described<sup><a href="#CR90" class="usa-link" aria-describedby="CR90">90</a></sup>. Briefly, OGT-WT or OGT ∆TPR4 protein (200 μM) in the ITC buffer (20 mM Tris buffer, pH = 7.5 ± 0.1, 150 mM NaCl) was loaded to the injection syringe. The reference cell was loaded with PB2 RNA (2 μM) in 300 µL of degassed ITC buffer. Titrations were performed at 298 K as a series of 2 µL injections with a 300 s delay between succeeding injections at a stirring rate of 300 rpm. The heat of dilution measured by titrating the buffer solution with protein solution of corresponding concentration was considered during fitting of the binding isotherm as the background heat. The data was analyzed and fitted with Nano Analyze software 2.5.0.</p></section><section id="Sec29"><h3 class="pmc_sec_title">Co-immunoprecipitation of RNA-interacting OGT from IAV-infected cells</h3>
<p id="Par48">HEK 293 T cells (3 × 10<sup>6</sup> cells per 100 mm dish, two dishes per sample) were transfected with constructs encoding Flag-OGT or Flag-empty vector (10 μg DNA per dish) using polyethylenimine. 24 h after transfection, cells were infected with either PR8 (MOI of 2) or were left uninfected (mock). Cells were harvested 16 h after infection and lysed in RIPA buffer (50 mM Tris, pH 7.4, 150 mM NaCl, 2 mM EDTA, 1% (v/v) NP-40 and 0.1% (v/v) SDS) supplemented with protease inhibitor for 30 min at 4 °C. Cell lysates were clarified by centrifugation at 17,000 × <em>g</em> for 20 min at 4 °C and supernatants were harvested. Aliquots of lysates were saved for total RNA input control. Lysates were then incubated with anti-Flag M2 Affinity Gel (A2220, Sigma) overnight with rotation at 4 °C. Precipitates were washed five times with RIPA buffer followed by treatment with 4 units proteinase K (EO0491, Thermo Fisher Scientific) at 55 °C for 30 min. The input and immunoprecipitated RNAs were extracted using TRIzol (Thermo Fisher Scientific) according to the manufacturer’s instructions.</p></section><section id="Sec30"><h3 class="pmc_sec_title">RNA-seq analysis</h3>
<p id="Par49">RNA purified from Flag-OGT or Flag-empty vector precipitates from mock-or PR8-infected HEK 293T cells, as well as total RNA input samples, were subjected to RNA quality control test using Qubit Flex Fluorometer and Agilent 2100 Bioanalyzer. 100 ng total RNA was used for RNA-Seq Library preparation with QIAseq® FastSelect™ RNA Library Kit. Briefly, ribosomal RNA (rRNA) depletion was carried out using the QIAseq FastSelect rRNA HMR Kit (334387, Qiagen), following the manufacturer’s instructions. cDNA synthesis was performed using the QIAseq Low Input RNA Library Kit (334205, Qiagen), employing both the N6-T RT Primer (random hexamer) and the ODT-T RT Primer (oligo-dT primer). Library amplification/indexing were carried out with QIAseq UX Index kit UDI-A (331815, Qiagen), which incorporates unique 10-base dual indices (UDIs) into each library. The final library quality control test was again conducted using the Qubit Flex Fluorometer and the Agilent 2100 Bioanalyzer. Libraries were sequenced on the Illumina NovaSeq X Plus platform. Raw sequencing reads were quality-controlled, trimmed, and filtered using fastp (v0.23.2)<sup><a href="#CR91" class="usa-link" aria-describedby="CR91">91</a></sup>, followed by alignment to IAV PR8 strain reference genome (NCBI accession number: GCF_000865725.1) and human genome (GRCh38.99) using Hisat2<sup><a href="#CR92" class="usa-link" aria-describedby="CR92">92</a></sup>, respectively. Human and IAV counts were normalized using Counts Per Million (CPM), adjusting for library size differences across samples (eq#1): <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e3145"><mi mathvariant="normal">CP</mi><msub><mrow><mi mathvariant="normal">M</mi></mrow><mrow><mi mathvariant="normal">ij</mi></mrow></msub><mo>=</mo><mfrac><mrow><msub><mrow><mi>x</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow><mrow><msub><mrow><mo>∑</mo></mrow><mrow><mi>k</mi></mrow></msub><msub><mrow><mi>x</mi></mrow><mrow><mi>k</mi><mi>j</mi></mrow></msub></mrow></mfrac><mo>×</mo><msup><mrow><mn>10</mn></mrow><mrow><mn>6</mn></mrow></msup></math></span>, where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq2"><math id="d33e3189"><msub><mrow><mi>x</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></math></span> is the count for gene <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq3"><math id="d33e3203"><mi>i</mi></math></span> in sample <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq4"><math id="d33e3213"><mi>j</mi></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq5"><math id="d33e3222"><msub><mrow><mo>∑</mo></mrow><mrow><mi>k</mi></mrow></msub><msub><mrow><mi>x</mi></mrow><mrow><mi>k</mi><mi>j</mi></mrow></msub></math></span> is the total number of counts in sample <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq6"><math id="d33e3241"><mi>j</mi></math></span>. Next, Flag-OGT and Flag empty vector precipitate counts were normalized to their respective input RNAs. OGT-specific transcript enrichment was calculated by dividing the normalized Flag-OGT precipitate counts through the normalized Flag-precipitate counts using R scripts. The data were multiplied and visualized in R using the ggplot2 package.</p></section><section id="Sec31"><h3 class="pmc_sec_title">MS assay of OGT interactome</h3>
<p id="Par50">High resolution/accurate mass (HR/AM)-based quantitative proteomics strategy was employed to identify OGT-interacting partners. Briefly, <em>OGT</em>-KO HT29 cells reconstituted with empty vector or GFP-tagged WT <em>OGT</em> were infected with IAV (MOI of 1) for 24 h, followed by immunoprecipitation with GFP-Trap agarose. The protein complexes were boiled with SDS buffer and processed for proteomics analysis following the E3filter protocol described previously<sup><a href="#CR93" class="usa-link" aria-describedby="CR93">93</a></sup>. Three biological replicates of the IP assay were performed. The digests were desalted using C18 StageTips (CDS Analytical), dried in a SpeedVac and stored in −80 °C until further analysis.</p>
<p id="Par51">For LC-MS/MS analysis, an Orbitrap Eclipse MS (Thermo Fisher Scientific) coupled with an Ultimate 3000 nanoLC system and a nanospray Flex ion source and a FAIMS Pro Interface (Thermo Fisher Scientific) was used. Peptides were first loaded onto a trap column (PepMap C18; 2 cm, length, 100 mm I.D.) and then separated by an analytical column (PepMap C18, 3.0 mm; 20 cm × 75 mm I.D.) using a binary buffer system (buffer A, 0.1% formic acid in water; buffer B, 0.1% formic acid in acetonitrile) with a 165-min gradient (1–25% buffer B over 95 min; 25–32% buffer B in 10 min, 32–95% buffer B over 5 min; back to 1% B in 5 min for equilibration after staying on 80% B for 15 min). MS data were acquired in a data-dependent top−12 method with a maximum injection time of 20 ms, a scan range of 350–1500 Da, and an automatic gain control target of 4 × 10<sup>5</sup>. MS/MS was performed via higher energy collisional dissociation fragmentation with a target value of 5 × 10<sup>4</sup> and maximum injection time of 22 ms. Full MS and MS/MS scans were acquired by Orbitrap at resolutions of 60,000 and 17,500, respectively. Dynamic exclusion was set to 20 s. For FAIMS, a 3-CV combination (−40|−55|−75) was applied. MS data were searched against UniProt human protein database (20,387 sequences; Reviewed only; August 2021 release) using Sequest HT algorithm and Proteome Discoverer software (version 2.5, Thermo Scientific). Detailed parameters include: trypsin as an enzyme with maximally two missed cleavage sites; protein N-terminal acetylation and methionine oxidation as variable modifications; cysteine carbamidomethylation as a fixed modification; peptide length must be at least seven amino acids. False discovery rate was set at 1% for both proteins. We have also reorganized the raw data and provided the updated login details (MSV000097062).</p></section><section id="Sec32"><h3 class="pmc_sec_title">Volcano plot visualization</h3>
<p id="Par52">Differential protein level analysis was performed using the DESeq2 package (Version 1.42.1) in RStudio and DESeqDataSet was constructed from raw MS assay count data<sup><a href="#CR94" class="usa-link" aria-describedby="CR94">94</a></sup>. A volcano plot was generated to visualize the relationship between fold change and statistical significance. The plot was created using the ggplot2 package, with log<sub>2</sub> fold change on the x-axis and -log<sub>10</sub> of the adjusted <em>p</em>-value on the y-axis. Proteins that met predefined thresholds for fold change (|log<sub>2</sub> fold change| &gt; 1) and adjusted <em>p</em> &lt; 0.01 were highlighted to indicate statistically significant upregulation or downregulation (up or down). The subcellular localization of proteins was determined using data from the Human Protein Atlas database<sup><a href="#CR95" class="usa-link" aria-describedby="CR95">95</a></sup> (Human Protein Atlas proteinatlas.org).</p></section><section id="Sec33"><h3 class="pmc_sec_title">Venn diagram</h3>
<p id="Par53">The Venn diagram was generated using the online version of Jvenn<sup><a href="#CR96" class="usa-link" aria-describedby="CR96">96</a></sup>. The OGT interactome 1<sup><a href="#CR57" class="usa-link" aria-describedby="CR57">57</a></sup> and OGT interactome 2<sup><a href="#CR56" class="usa-link" aria-describedby="CR56">56</a></sup> were obtained from previously published studies. The OGT interactome presented in this study was derived from raw MS assay count data comparing IAV-treated and nontreated OGT WT.</p></section><section id="Sec34"><h3 class="pmc_sec_title">CRISPR/Cas9 gene knockout</h3>
<p id="Par54">For human <em>MAVS, IRF3</em> and <em>PLIN2</em> knockout, Alt-R CRISPR-Cas9 crRNAs targeting <em>MAVS</em> (5′- GTAGATACAACTGACCCTGTGGG), <em>IRF3</em> (5′- GAGGTGACAGCCTTCTACCGGGG), or <em>PLIN2</em> (5′- TCATACGTGGAGCTCACCAAGGG) were used for ribonucleoprotein packaging in the presence of Cas9 protein (A36498, Thermo Fisher Scientific), Alt-R CRISPR-Cas9 tracrRNA ATTO 550 (1075927, IDT) and Alt-R Cas9 Electroporation Enhancer (1075915, IDT). Ribonucleoprotein complexes were electroporated into cells with Neon Transfection System (MPK5000, Thermo Fisher Scientific) according to the manufacturer’s instructions. Alt-R CRISPR-Cas9 negative control crRNA (1072544, IDT) was included as non-targeting control. After 24 h, cells with fluorescence were sorted with FACS for further use.</p></section><section id="Sec35"><h3 class="pmc_sec_title">Statistics</h3>
<p id="Par55">Statistical analysis was carried out with GraphPad Prism 10.2.0 for macOS. All data are shown as SEM. The mean values for biochemical data from each group were compared by two-tailed Tukey’s test. Comparisons between multiple time points were analyzed by repeated-measurements analysis of variance (ANOVA) with Bonferroni post-tests. In all tests, <em>P</em> values of less than 0.05 were considered statistically significant.</p></section><section id="Sec36"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par56">Further information on research design is available in the <a href="#MOESM4" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec37"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12365020/bin/41467_2025_63085_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (4.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12365020/bin/41467_2025_63085_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63085_MOESM2_ESM.pdf</a><sup> (84.9KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12365020/bin/41467_2025_63085_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (353.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12365020/bin/41467_2025_63085_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (148.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12365020/bin/41467_2025_63085_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (421KB, pdf) </sup>
</div></div></section></section><section id="Sec38"><h2 class="pmc_sec_title">Source data</h2>
<section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12365020/bin/41467_2025_63085_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (4MB, xlsx) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Dr. Xiaoyong Yang for <em>Ogt</em><sup><em>fl/fl</em></sup> mice, Dr. Jing Wang for CRISPR/Cas9 gene deletion. This work was supported by NIH grants to H.W. (R01GM152205 and R01AI135234), X.C. (R01GM141394), J.L. (R01AI090060), J.S.Y. (R01AI130110), and S.L.L. (R01AI150473). Z. Liu was supported by Cure Cystic Fibrosis Columbus Research &amp; Development Program (C3RDP) Trainee Grant. Y. Sun was supported by Pelotonia Postdoc Fellowship. The Ohio State University Comprehensive Cancer Center Genomics Shared Resource was supported by National Cancer Institute grant (P30-CA016058).</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>H.D., C.L., and H.W. designed the experiments; H.D., C.L., Z.H.L., Y.S., and Z.W.L. performed experiments and provided intellectual input; Y.Y. performed key mass spectrometry experiment; J.Z., W.W., N.K., J.D.B., A.M., Q.M., X.C., and J.L. contributed to key protein-RNA interaction experiments and provided intellectual input; Y.Z., M.J.H., K.M.G., P.G.T., J.S.Y., and S.L.L. provided critical reagents and intellectual input. H.W. supervised the study, interpreted the data and wrote the manuscript. All authors read and approved the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par57"><em>Nature Communications</em> thanks the anonymous reviewers for their contribution to the peer review of this work. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The RNA-seq data generated in this study have been deposited in the Gene Expression Omnibus (GEO) under accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE297354" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE297354</a>. The MS data generated in this study have been deposited in the MassIVE server under accession number MSV000097062 (<a href="https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=dffa2d6a49a04a69a79cf1df8218c2c4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=dffa2d6a49a04a69a79cf1df8218c2c4</a>). <a href="#Sec38" class="usa-link">Source data</a> are provided with this paper.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par58">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisher’s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec39"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s41467-025-63085-y.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Ablasser, A. &amp; Hur, S. Regulation of cGAS- and RLR-mediated immunity to nucleic acids. <em>Nat. Immunol.</em><strong>21</strong>, 17–29 (2020).
</cite> [<a href="https://doi.org/10.1038/s41590-019-0556-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31819255/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ablasser,%20A.%20&amp;%20Hur,%20S.%20Regulation%20of%20cGAS-%20and%20RLR-mediated%20immunity%20to%20nucleic%20acids.%20Nat.%20Immunol.21,%2017%E2%80%9329%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Brubaker, S. W., Bonham, K. S., Zanoni, I. &amp; Kagan, J. C. Innate immune pattern recognition: a cell biological perspective. <em>Annu. Rev. Immunol.</em><strong>33</strong>, 257–290 (2015).
</cite> [<a href="https://doi.org/10.1146/annurev-immunol-032414-112240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5146691/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25581309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brubaker,%20S.%20W.,%20Bonham,%20K.%20S.,%20Zanoni,%20I.%20&amp;%20Kagan,%20J.%20C.%20Innate%20immune%20pattern%20recognition:%20a%20cell%20biological%20perspective.%20Annu.%20Rev.%20Immunol.33,%20257%E2%80%93290%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Rehwinkel, J. &amp; Gack, M. U. RIG-I-like receptors: their regulation and roles in RNA sensing. <em>Nat. Rev. Immunol.</em><strong>20</strong>, 537–551 (2020).
</cite> [<a href="https://doi.org/10.1038/s41577-020-0288-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7094958/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32203325/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Rehwinkel,%20J.%20&amp;%20Gack,%20M.%20U.%20RIG-I-like%20receptors:%20their%20regulation%20and%20roles%20in%20RNA%20sensing.%20Nat.%20Rev.%20Immunol.20,%20537%E2%80%93551%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Seth, R. B., Sun, L., Ea, C. K. &amp; Chen, Z. J. Identification and characterization of MAVS, a mitochondrial antiviral signaling protein that activates NF-kappaB and IRF 3. <em>Cell</em><strong>122</strong>, 669–682 (2005).
</cite> [<a href="https://doi.org/10.1016/j.cell.2005.08.012" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16125763/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seth,%20R.%20B.,%20Sun,%20L.,%20Ea,%20C.%20K.%20&amp;%20Chen,%20Z.%20J.%20Identification%20and%20characterization%20of%20MAVS,%20a%20mitochondrial%20antiviral%20signaling%20protein%20that%20activates%20NF-kappaB%20and%20IRF%203.%20Cell122,%20669%E2%80%93682%20(2005)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Kenney, A. D. et al. Human genetic determinants of viral diseases. <em>Annu. Rev. Genet.</em><strong>51</strong>, 241–263 (2017).
</cite> [<a href="https://doi.org/10.1146/annurev-genet-120116-023425" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6038703/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28853921/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kenney,%20A.%20D.%20et%20al.%20Human%20genetic%20determinants%20of%20viral%20diseases.%20Annu.%20Rev.%20Genet.51,%20241%E2%80%93263%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>Koehler, H. et al. Vaccinia virus E3 prevents sensing of Z-RNA to block ZBP1-dependent necroptosis. <em>Cell Host Microbe</em><strong>29</strong>, 1266–1276.e1265 (2021).
</cite> [<a href="https://doi.org/10.1016/j.chom.2021.05.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9333947/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34192517/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Koehler,%20H.%20et%20al.%20Vaccinia%20virus%20E3%20prevents%20sensing%20of%20Z-RNA%20to%20block%20ZBP1-dependent%20necroptosis.%20Cell%20Host%20Microbe29,%201266%E2%80%931276.e1265%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Thapa, R. J. et al. DAI senses influenza A virus genomic RNA and activates RIPK3-dependent cell death. <em>Cell Host Microbe</em><strong>20</strong>, 674–681 (2016).
</cite> [<a href="https://doi.org/10.1016/j.chom.2016.09.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5687825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27746097/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Thapa,%20R.%20J.%20et%20al.%20DAI%20senses%20influenza%20A%20virus%20genomic%20RNA%20and%20activates%20RIPK3-dependent%20cell%20death.%20Cell%20Host%20Microbe20,%20674%E2%80%93681%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Maelfait, J. et al. Sensing of viral and endogenous RNA by ZBP1/DAI induces necroptosis. <em>EMBO J.</em><strong>36</strong>, 2529–2543 (2017).
</cite> [<a href="https://doi.org/10.15252/embj.201796476" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5579359/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28716805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Maelfait,%20J.%20et%20al.%20Sensing%20of%20viral%20and%20endogenous%20RNA%20by%20ZBP1/DAI%20induces%20necroptosis.%20EMBO%20J.36,%202529%E2%80%932543%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Zhang, T. et al. Influenza virus Z-RNAs induce ZBP1-mediated necroptosis. <em>Cell</em><strong>180</strong>, 1115–1129.e1113 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.02.050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7153753/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32200799/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20T.%20et%20al.%20Influenza%20virus%20Z-RNAs%20induce%20ZBP1-mediated%20necroptosis.%20Cell180,%201115%E2%80%931129.e1113%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Zheng, M., Karki, R., Vogel, P. &amp; Kanneganti, T. D. Caspase-6 is a key regulator of innate immunity, inflammasome activation, and host defense. <em>Cell</em><strong>181</strong>, 674–687.e613 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.03.040" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7425208/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32298652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zheng,%20M.,%20Karki,%20R.,%20Vogel,%20P.%20&amp;%20Kanneganti,%20T.%20D.%20Caspase-6%20is%20a%20key%20regulator%20of%20innate%20immunity,%20inflammasome%20activation,%20and%20host%20defense.%20Cell181,%20674%E2%80%93687.e613%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Fitzgerald, K. A. &amp; Kagan, J. C. Toll-like receptors and the control of immunity. <em>Cell</em><strong>180</strong>, 1044–1066 (2020).
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.02.041" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9358771/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32164908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Fitzgerald,%20K.%20A.%20&amp;%20Kagan,%20J.%20C.%20Toll-like%20receptors%20and%20the%20control%20of%20immunity.%20Cell180,%201044%E2%80%931066%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Cadena, C. &amp; Hur, S. Filament-like assemblies of intracellular nucleic acid sensors: commonalities and differences. <em>Mol. Cell</em><strong>76</strong>, 243–254 (2019).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2019.09.023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6880955/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31626748/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cadena,%20C.%20&amp;%20Hur,%20S.%20Filament-like%20assemblies%20of%20intracellular%20nucleic%20acid%20sensors:%20commonalities%20and%20differences.%20Mol.%20Cell76,%20243%E2%80%93254%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Roers, A., Hiller, B. &amp; Hornung, V. Recognition of endogenous nucleic acids by the innate immune system. <em>Immunity</em><strong>44</strong>, 739–754 (2016).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2016.04.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27096317/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roers,%20A.,%20Hiller,%20B.%20&amp;%20Hornung,%20V.%20Recognition%20of%20endogenous%20nucleic%20acids%20by%20the%20innate%20immune%20system.%20Immunity44,%20739%E2%80%93754%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Wu, J. &amp; Chen, Z. J. Innate immune sensing and signaling of cytosolic nucleic acids. <em>Annu. Rev. Immunol.</em><strong>32</strong>, 461–488 (2014).
</cite> [<a href="https://doi.org/10.1146/annurev-immunol-032713-120156" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24655297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu,%20J.%20&amp;%20Chen,%20Z.%20J.%20Innate%20immune%20sensing%20and%20signaling%20of%20cytosolic%20nucleic%20acids.%20Annu.%20Rev.%20Immunol.32,%20461%E2%80%93488%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Buck, M. D., Sowell, R. T., Kaech, S. M. &amp; Pearce, E. L. Metabolic instruction of immunity. <em>Cell</em><strong>169</strong>, 570–586 (2017).
</cite> [<a href="https://doi.org/10.1016/j.cell.2017.04.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5648021/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28475890/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Buck,%20M.%20D.,%20Sowell,%20R.%20T.,%20Kaech,%20S.%20M.%20&amp;%20Pearce,%20E.%20L.%20Metabolic%20instruction%20of%20immunity.%20Cell169,%20570%E2%80%93586%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>O’Neill, L. A., Kishton, R. J. &amp; Rathmell, J. A guide to immunometabolism for immunologists. <em>Nat. Rev. Immunol.</em><strong>16</strong>, 553–565 (2016).
</cite> [<a href="https://doi.org/10.1038/nri.2016.70" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5001910/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27396447/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?O%E2%80%99Neill,%20L.%20A.,%20Kishton,%20R.%20J.%20&amp;%20Rathmell,%20J.%20A%20guide%20to%20immunometabolism%20for%20immunologists.%20Nat.%20Rev.%20Immunol.16,%20553%E2%80%93565%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Everts, B. et al. TLR-driven early glycolytic reprogramming via the kinases TBK1-IKKvarepsilon supports the anabolic demands of dendritic cell activation. <em>Nat. Immunol.</em><strong>15</strong>, 323–332 (2014).
</cite> [<a href="https://doi.org/10.1038/ni.2833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4358322/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24562310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Everts,%20B.%20et%20al.%20TLR-driven%20early%20glycolytic%20reprogramming%20via%20the%20kinases%20TBK1-IKKvarepsilon%20supports%20the%20anabolic%20demands%20of%20dendritic%20cell%20activation.%20Nat.%20Immunol.15,%20323%E2%80%93332%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Lachmandas, E. et al. Microbial stimulation of different Toll-like receptor signalling pathways induces diverse metabolic programmes in human monocytes. <em>Nat. Microbiol.</em><strong>2</strong>, 16246 (2016).
</cite> [<a href="https://doi.org/10.1038/nmicrobiol.2016.246" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27991883/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lachmandas,%20E.%20et%20al.%20Microbial%20stimulation%20of%20different%20Toll-like%20receptor%20signalling%20pathways%20induces%20diverse%20metabolic%20programmes%20in%20human%20monocytes.%20Nat.%20Microbiol.2,%2016246%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>O’Carroll, S. M., Henkel, F. D. R. &amp; O’Neill, L. A. J. Metabolic regulation of type I interferon production. <em>Immunol. Rev.</em><strong>323</strong>, 276–287 (2024).
</cite> [<a href="https://doi.org/10.1111/imr.13318" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38465724/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?O%E2%80%99Carroll,%20S.%20M.,%20Henkel,%20F.%20D.%20R.%20&amp;%20O%E2%80%99Neill,%20L.%20A.%20J.%20Metabolic%20regulation%20of%20type%20I%20interferon%20production.%20Immunol.%20Rev.323,%20276%E2%80%93287%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Wells, L., Vosseller, K. &amp; Hart, G. W. Glycosylation of nucleocytoplasmic proteins: signal transduction and O-GlcNAc. <em>Science</em><strong>291</strong>, 2376–2378 (2001).
</cite> [<a href="https://doi.org/10.1126/science.1058714" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11269319/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wells,%20L.,%20Vosseller,%20K.%20&amp;%20Hart,%20G.%20W.%20Glycosylation%20of%20nucleocytoplasmic%20proteins:%20signal%20transduction%20and%20O-GlcNAc.%20Science291,%202376%E2%80%932378%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Love, D. C., Krause, M. W. &amp; Hanover, J. A. O-GlcNAc cycling: emerging roles in development and epigenetics. <em>Semin Cell Dev. Biol.</em><strong>21</strong>, 646–654 (2010).
</cite> [<a href="https://doi.org/10.1016/j.semcdb.2010.05.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2917487/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20488252/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Love,%20D.%20C.,%20Krause,%20M.%20W.%20&amp;%20Hanover,%20J.%20A.%20O-GlcNAc%20cycling:%20emerging%20roles%20in%20development%20and%20epigenetics.%20Semin%20Cell%20Dev.%20Biol.21,%20646%E2%80%93654%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Hart, G. W., Housley, M. P. &amp; Slawson, C. Cycling of O-linked beta-N-acetylglucosamine on nucleocytoplasmic proteins. <em>Nature</em><strong>446</strong>, 1017–1022 (2007).
</cite> [<a href="https://doi.org/10.1038/nature05815" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17460662/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hart,%20G.%20W.,%20Housley,%20M.%20P.%20&amp;%20Slawson,%20C.%20Cycling%20of%20O-linked%20beta-N-acetylglucosamine%20on%20nucleocytoplasmic%20proteins.%20Nature446,%201017%E2%80%931022%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Torres, C. R. &amp; Hart, G. W. Topography and polypeptide distribution of terminal N-acetylglucosamine residues on the surfaces of intact lymphocytes. Evidence for O-linked GlcNAc. <em>J. Biol. Chem.</em><strong>259</strong>, 3308–3317 (1984).
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/6421821/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Torres,%20C.%20R.%20&amp;%20Hart,%20G.%20W.%20Topography%20and%20polypeptide%20distribution%20of%20terminal%20N-acetylglucosamine%20residues%20on%20the%20surfaces%20of%20intact%20lymphocytes.%20Evidence%20for%20O-linked%20GlcNAc.%20J.%20Biol.%20Chem.259,%203308%E2%80%933317%20(1984)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Kreppel, L. K., Blomberg, M. A. &amp; Hart, G. W. Dynamic glycosylation of nuclear and cytosolic proteins. Cloning and characterization of a unique O-GlcNAc transferase with multiple tetratricopeptide repeats. <em>J. Biol. Chem.</em><strong>272</strong>, 9308–9315 (1997).
</cite> [<a href="https://doi.org/10.1074/jbc.272.14.9308" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9083067/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kreppel,%20L.%20K.,%20Blomberg,%20M.%20A.%20&amp;%20Hart,%20G.%20W.%20Dynamic%20glycosylation%20of%20nuclear%20and%20cytosolic%20proteins.%20Cloning%20and%20characterization%20of%20a%20unique%20O-GlcNAc%20transferase%20with%20multiple%20tetratricopeptide%20repeats.%20J.%20Biol.%20Chem.272,%209308%E2%80%939315%20(1997)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Yang, X. &amp; Qian, K. Protein O-GlcNAcylation: emerging mechanisms and functions. <em>Nat. Rev. Mol. Cell Biol.</em><strong>18</strong>, 452–465 (2017).
</cite> [<a href="https://doi.org/10.1038/nrm.2017.22" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5667541/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28488703/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20X.%20&amp;%20Qian,%20K.%20Protein%20O-GlcNAcylation:%20emerging%20mechanisms%20and%20functions.%20Nat.%20Rev.%20Mol.%20Cell%20Biol.18,%20452%E2%80%93465%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Hardiville, S. &amp; Hart, G. W. Nutrient regulation of signaling, transcription, and cell physiology by O-GlcNAcylation. <em>Cell Metab.</em><strong>20</strong>, 208–213 (2014).
</cite> [<a href="https://doi.org/10.1016/j.cmet.2014.07.014" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4159757/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25100062/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hardiville,%20S.%20&amp;%20Hart,%20G.%20W.%20Nutrient%20regulation%20of%20signaling,%20transcription,%20and%20cell%20physiology%20by%20O-GlcNAcylation.%20Cell%20Metab.20,%20208%E2%80%93213%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Hart, G. W., Slawson, C., Ramirez-Correa, G. &amp; Lagerlof, O. Cross talk between O-GlcNAcylation and phosphorylation: roles in signaling, transcription, and chronic disease. <em>Annu Rev. Biochem</em><strong>80</strong>, 825–858 (2011).
</cite> [<a href="https://doi.org/10.1146/annurev-biochem-060608-102511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3294376/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21391816/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hart,%20G.%20W.,%20Slawson,%20C.,%20Ramirez-Correa,%20G.%20&amp;%20Lagerlof,%20O.%20Cross%20talk%20between%20O-GlcNAcylation%20and%20phosphorylation:%20roles%20in%20signaling,%20transcription,%20and%20chronic%20disease.%20Annu%20Rev.%20Biochem80,%20825%E2%80%93858%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Levine, Z. G. &amp; Walker, S. The biochemistry of O-GlcNAc transferase: which functions make it essential in mammalian cells?. <em>Annu. Rev. Biochem.</em><strong>85</strong>, 631–657 (2016).
</cite> [<a href="https://doi.org/10.1146/annurev-biochem-060713-035344" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27294441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Levine,%20Z.%20G.%20&amp;%20Walker,%20S.%20The%20biochemistry%20of%20O-GlcNAc%20transferase:%20which%20functions%20make%20it%20essential%20in%20mammalian%20cells?.%20Annu.%20Rev.%20Biochem.85,%20631%E2%80%93657%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Bond, M. R. &amp; Hanover, J. A. O-GlcNAc cycling: a link between metabolism and chronic disease. <em>Annu Rev. Nutr.</em><strong>33</strong>, 205–229 (2013).
</cite> [<a href="https://doi.org/10.1146/annurev-nutr-071812-161240" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10483992/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23642195/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bond,%20M.%20R.%20&amp;%20Hanover,%20J.%20A.%20O-GlcNAc%20cycling:%20a%20link%20between%20metabolism%20and%20chronic%20disease.%20Annu%20Rev.%20Nutr.33,%20205%E2%80%93229%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Hart, G. W. Nutrient regulation of signaling and transcription. <em>J. Biol. Chem.</em><strong>294</strong>, 2211–2231 (2019).
</cite> [<a href="https://doi.org/10.1074/jbc.AW119.003226" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6378989/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30626734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hart,%20G.%20W.%20Nutrient%20regulation%20of%20signaling%20and%20transcription.%20J.%20Biol.%20Chem.294,%202211%E2%80%932231%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Li, T. et al. O-GlcNAc transferase links glucose metabolism to MAVS-mediated antiviral innate immunity. <em>Cell Host Microbe</em><strong>24</strong>, 791–803.e796 (2018).
</cite> [<a href="https://doi.org/10.1016/j.chom.2018.11.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6296827/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30543776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20T.%20et%20al.%20O-GlcNAc%20transferase%20links%20glucose%20metabolism%20to%20MAVS-mediated%20antiviral%20innate%20immunity.%20Cell%20Host%20Microbe24,%20791%E2%80%93803.e796%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Song, N. et al. MAVS O-GlcNAcylation is essential for host antiviral immunity against lethal RNA viruses. <em>Cell Rep.</em><strong>28</strong>, 2386–2396.e2385 (2019).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2019.07.085" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31461653/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song,%20N.%20et%20al.%20MAVS%20O-GlcNAcylation%20is%20essential%20for%20host%20antiviral%20immunity%20against%20lethal%20RNA%20viruses.%20Cell%20Rep.28,%202386%E2%80%932396.e2385%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Herzog, K. et al. Functional microRNA screen uncovers O-linked N-acetylglucosamine transferase as a host factor modulating hepatitis C virus morphogenesis and infectivity. <em>Gut</em><strong>69</strong>, 380–392 (2020).
</cite> [<a href="https://doi.org/10.1136/gutjnl-2018-317423" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7613422/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31076402/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Herzog,%20K.%20et%20al.%20Functional%20microRNA%20screen%20uncovers%20O-linked%20N-acetylglucosamine%20transferase%20as%20a%20host%20factor%20modulating%20hepatitis%20C%20virus%20morphogenesis%20and%20infectivity.%20Gut69,%20380%E2%80%93392%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Wang, Q. et al. O-GlcNAc transferase promotes influenza A virus-induced cytokine storm by targeting interferon regulatory factor-5. <em>Sci. Adv.</em><strong>6</strong>, eaaz7086 (2020).
</cite> [<a href="https://doi.org/10.1126/sciadv.aaz7086" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7159909/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32494619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Q.%20et%20al.%20O-GlcNAc%20transferase%20promotes%20influenza%20A%20virus-induced%20cytokine%20storm%20by%20targeting%20interferon%20regulatory%20factor-5.%20Sci.%20Adv.6,%20eaaz7086%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Wang, X. et al. O-GlcNAcylation modulates HBV replication through regulating cellular autophagy at multiple levels. <em>FASEB J.</em><strong>34</strong>, 14473–14489 (2020).
</cite> [<a href="https://doi.org/10.1096/fj.202001168RR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32892442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20X.%20et%20al.%20O-GlcNAcylation%20modulates%20HBV%20replication%20through%20regulating%20cellular%20autophagy%20at%20multiple%20levels.%20FASEB%20J.34,%2014473%E2%80%9314489%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Hu, J. et al. Hexosamine biosynthetic pathway promotes the antiviral activity of SAMHD1 by enhancing O-GlcNAc transferase-mediated protein O-GlcNAcylation. <em>Theranostics</em><strong>11</strong>, 805–823 (2021).
</cite> [<a href="https://doi.org/10.7150/thno.50230" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7738853/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33391506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hu,%20J.%20et%20al.%20Hexosamine%20biosynthetic%20pathway%20promotes%20the%20antiviral%20activity%20of%20SAMHD1%20by%20enhancing%20O-GlcNAc%20transferase-mediated%20protein%20O-GlcNAcylation.%20Theranostics11,%20805%E2%80%93823%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Mears, H. V. &amp; Sweeney, T. R. Better together: the role of IFIT protein-protein interactions in the antiviral response. <em>J. Gen. Virol.</em><strong>99</strong>, 1463–1477 (2018).
</cite> [<a href="https://doi.org/10.1099/jgv.0.001149" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30234477/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mears,%20H.%20V.%20&amp;%20Sweeney,%20T.%20R.%20Better%20together:%20the%20role%20of%20IFIT%20protein-protein%20interactions%20in%20the%20antiviral%20response.%20J.%20Gen.%20Virol.99,%201463%E2%80%931477%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Levine, Z. G. et al. Mammalian cell proliferation requires noncatalytic functions of O-GlcNAc transferase. <em>Proc. Natl. Acad. Sci. USA</em><strong>118</strong>. 10.1073/pnas.2016778118 (2021).</cite> [<a href="https://doi.org/10.1073/pnas.2016778118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7848692/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33419956/" class="usa-link">PubMed</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Authier, F., Ondruskova, N., Ferenbach, A. T., McNeilly, A. D., &amp; van Aalten, D. M. F. Neurodevelopmental defects in a mouse model of O-GlcNAc transferase intellectual disability. <em>Dis. Model. Mech</em>. <strong>17</strong>. 10.1242/dmm.050671 (2024).</cite> [<a href="https://doi.org/10.1242/dmm.050671" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11095632/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38566589/" class="usa-link">PubMed</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Clarke, A. J. et al. Structural insights into mechanism and specificity of O-GlcNAc transferase. <em>EMBO J.</em><strong>27</strong>, 2780–2788 (2008).
</cite> [<a href="https://doi.org/10.1038/emboj.2008.186" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2556091/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18818698/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Clarke,%20A.%20J.%20et%20al.%20Structural%20insights%20into%20mechanism%20and%20specificity%20of%20O-GlcNAc%20transferase.%20EMBO%20J.27,%202780%E2%80%932788%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Martinez-Fleites, C. et al. Structure of an O-GlcNAc transferase homolog provides insight into intracellular glycosylation. <em>Nat. Struct. Mol. Biol.</em><strong>15</strong>, 764–765 (2008).
</cite> [<a href="https://doi.org/10.1038/nsmb.1443" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18536723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martinez-Fleites,%20C.%20et%20al.%20Structure%20of%20an%20O-GlcNAc%20transferase%20homolog%20provides%20insight%20into%20intracellular%20glycosylation.%20Nat.%20Struct.%20Mol.%20Biol.15,%20764%E2%80%93765%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Lazarus, M. B., Nam, Y., Jiang, J., Sliz, P. &amp; Walker, S. Structure of human O-GlcNAc transferase and its complex with a peptide substrate. <em>Nature</em><strong>469</strong>, 564–567 (2011).
</cite> [<a href="https://doi.org/10.1038/nature09638" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3064491/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21240259/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lazarus,%20M.%20B.,%20Nam,%20Y.,%20Jiang,%20J.,%20Sliz,%20P.%20&amp;%20Walker,%20S.%20Structure%20of%20human%20O-GlcNAc%20transferase%20and%20its%20complex%20with%20a%20peptide%20substrate.%20Nature469,%20564%E2%80%93567%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Kenney, A. D. et al. IFITM3 protects the heart during influenza virus infection. <em>Proc. Natl. Acad. Sci. USA</em><strong>116</strong>, 18607–18612 (2019).
</cite> [<a href="https://doi.org/10.1073/pnas.1900784116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6744864/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31451661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kenney,%20A.%20D.%20et%20al.%20IFITM3%20protects%20the%20heart%20during%20influenza%20virus%20infection.%20Proc.%20Natl.%20Acad.%20Sci.%20USA116,%2018607%E2%80%9318612%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Zhang, Y. et al. Respiratory enterovirus (like parainfluenza virus) can cause chronic lung disease if protection by airway epithelial STAT1 is lost. <em>J. Immunol.</em><strong>202</strong>, 2332–2347 (2019).
</cite> [<a href="https://doi.org/10.4049/jimmunol.1801491" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6456410/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30804041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20Y.%20et%20al.%20Respiratory%20enterovirus%20(like%20parainfluenza%20virus)%20can%20cause%20chronic%20lung%20disease%20if%20protection%20by%20airway%20epithelial%20STAT1%20is%20lost.%20J.%20Immunol.202,%202332%E2%80%932347%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Iyer, S. P. &amp; Hart, G. W. Roles of the tetratricopeptide repeat domain in O-GlcNAc transferase targeting and protein substrate specificity. <em>J. Biol. Chem.</em><strong>278</strong>, 24608–24616 (2003).
</cite> [<a href="https://doi.org/10.1074/jbc.M300036200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12724313/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Iyer,%20S.%20P.%20&amp;%20Hart,%20G.%20W.%20Roles%20of%20the%20tetratricopeptide%20repeat%20domain%20in%20O-GlcNAc%20transferase%20targeting%20and%20protein%20substrate%20specificity.%20J.%20Biol.%20Chem.278,%2024608%E2%80%9324616%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Jinek, M. et al. The superhelical TPR-repeat domain of O-linked GlcNAc transferase exhibits structural similarities to importin alpha. <em>Nat. Struct. Mol. Biol.</em><strong>11</strong>, 1001–1007 (2004).
</cite> [<a href="https://doi.org/10.1038/nsmb833" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15361863/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jinek,%20M.%20et%20al.%20The%20superhelical%20TPR-repeat%20domain%20of%20O-linked%20GlcNAc%20transferase%20exhibits%20structural%20similarities%20to%20importin%20alpha.%20Nat.%20Struct.%20Mol.%20Biol.11,%201001%E2%80%931007%20(2004)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Abbas, Y. M., Pichlmair, A., Gorna, M. W., Superti-Furga, G. &amp; Nagar, B. Structural basis for viral 5’-PPP-RNA recognition by human IFIT proteins. <em>Nature</em><strong>494</strong>, 60–64 (2013).
</cite> [<a href="https://doi.org/10.1038/nature11783" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4931921/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23334420/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Abbas,%20Y.%20M.,%20Pichlmair,%20A.,%20Gorna,%20M.%20W.,%20Superti-Furga,%20G.%20&amp;%20Nagar,%20B.%20Structural%20basis%20for%20viral%205%E2%80%99-PPP-RNA%20recognition%20by%20human%20IFIT%20proteins.%20Nature494,%2060%E2%80%9364%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Hoffmann, E., Stech, J., Guan, Y., Webster, R. G. &amp; Perez, D. R. Universal primer set for the full-length amplification of all influenza A viruses. <em>Arch. Virol.</em><strong>146</strong>, 2275–2289 (2001).
</cite> [<a href="https://doi.org/10.1007/s007050170002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11811679/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hoffmann,%20E.,%20Stech,%20J.,%20Guan,%20Y.,%20Webster,%20R.%20G.%20&amp;%20Perez,%20D.%20R.%20Universal%20primer%20set%20for%20the%20full-length%20amplification%20of%20all%20influenza%20A%20viruses.%20Arch.%20Virol.146,%202275%E2%80%932289%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Jiang, Z. et al. IFI16 directly senses viral RNA and enhances RIG-I transcription and activation to restrict influenza virus infection. <em>Nat. Microbiol</em><strong>6</strong>, 932–945 (2021).
</cite> [<a href="https://doi.org/10.1038/s41564-021-00907-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33986530/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jiang,%20Z.%20et%20al.%20IFI16%20directly%20senses%20viral%20RNA%20and%20enhances%20RIG-I%20transcription%20and%20activation%20to%20restrict%20influenza%20virus%20infection.%20Nat.%20Microbiol6,%20932%E2%80%93945%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Pichlmair, A. et al. IFIT1 is an antiviral protein that recognizes 5’-triphosphate RNA. <em>Nat. Immunol.</em><strong>12</strong>, 624–630 (2011).
</cite> [<a href="https://doi.org/10.1038/ni.2048" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21642987/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pichlmair,%20A.%20et%20al.%20IFIT1%20is%20an%20antiviral%20protein%20that%20recognizes%205%E2%80%99-triphosphate%20RNA.%20Nat.%20Immunol.12,%20624%E2%80%93630%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Keane, S. C. et al. RNA structure. Structure of the HIV-1 RNA packaging signal. <em>Science</em><strong>348</strong>, 917–921 (2015).
</cite> [<a href="https://doi.org/10.1126/science.aaa9266" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4492308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25999508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Keane,%20S.%20C.%20et%20al.%20RNA%20structure.%20Structure%20of%20the%20HIV-1%20RNA%20packaging%20signal.%20Science348,%20917%E2%80%93921%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Chiang, J. J. et al. Viral unmasking of cellular 5S rRNA pseudogene transcripts induces RIG-I-mediated immunity. <em>Nat. Immunol.</em><strong>19</strong>, 53–62 (2018).
</cite> [<a href="https://doi.org/10.1038/s41590-017-0005-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5815369/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29180807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chiang,%20J.%20J.%20et%20al.%20Viral%20unmasking%20of%20cellular%205S%20rRNA%20pseudogene%20transcripts%20induces%20RIG-I-mediated%20immunity.%20Nat.%20Immunol.19,%2053%E2%80%9362%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Whelan, S. A., Lane, M. D. &amp; Hart, G. W. Regulation of the O-linked beta-N-acetylglucosamine transferase by insulin signaling. <em>J. Biol. Chem.</em><strong>283</strong>, 21411–21417 (2008).
</cite> [<a href="https://doi.org/10.1074/jbc.M800677200" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2490780/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18519567/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Whelan,%20S.%20A.,%20Lane,%20M.%20D.%20&amp;%20Hart,%20G.%20W.%20Regulation%20of%20the%20O-linked%20beta-N-acetylglucosamine%20transferase%20by%20insulin%20signaling.%20J.%20Biol.%20Chem.283,%2021411%E2%80%9321417%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Bullen, J. W. et al. Cross-talk between two essential nutrient-sensitive enzymes: O-GlcNAc transferase (OGT) and AMP-activated protein kinase (AMPK). <em>J. Biol. Chem.</em><strong>289</strong>, 10592–10606 (2014).
</cite> [<a href="https://doi.org/10.1074/jbc.M113.523068" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4036179/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24563466/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bullen,%20J.%20W.%20et%20al.%20Cross-talk%20between%20two%20essential%20nutrient-sensitive%20enzymes:%20O-GlcNAc%20transferase%20(OGT)%20and%20AMP-activated%20protein%20kinase%20(AMPK).%20J.%20Biol.%20Chem.289,%2010592%E2%80%9310606%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Seo, H. G. et al. Identification of the nuclear localisation signal of O-GlcNAc transferase and its nuclear import regulation. <em>Sci. Rep.</em><strong>6</strong>, 34614 (2016).
</cite> [<a href="https://doi.org/10.1038/srep34614" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5054401/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27713473/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Seo,%20H.%20G.%20et%20al.%20Identification%20of%20the%20nuclear%20localisation%20signal%20of%20O-GlcNAc%20transferase%20and%20its%20nuclear%20import%20regulation.%20Sci.%20Rep.6,%2034614%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Gao, J. et al. Proteomic analysis of the OGT interactome: novel links to epithelial-mesenchymal transition and metastasis of cervical cancer. <em>Carcinogenesis</em><strong>39</strong>, 1222–1234 (2018).
</cite> [<a href="https://doi.org/10.1093/carcin/bgy097" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6175026/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30052810/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gao,%20J.%20et%20al.%20Proteomic%20analysis%20of%20the%20OGT%20interactome:%20novel%20links%20to%20epithelial-mesenchymal%20transition%20and%20metastasis%20of%20cervical%20cancer.%20Carcinogenesis39,%201222%E2%80%931234%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Martinez, M. et al. Quantitative proteomics reveals that the OGT interactome is remodeled in response to oxidative stress. <em>Mol. Cell Proteom.</em><strong>20</strong>, 100069 (2021).</cite> [<a href="https://doi.org/10.1016/j.mcpro.2021.100069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8079276/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33716169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Martinez,%20M.%20et%20al.%20Quantitative%20proteomics%20reveals%20that%20the%20OGT%20interactome%20is%20remodeled%20in%20response%20to%20oxidative%20stress.%20Mol.%20Cell%20Proteom.20,%20100069%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Saka, H. A. &amp; Valdivia, R. Emerging roles for lipid droplets in immunity and host-pathogen interactions. <em>Annu. Rev. Cell Dev. Biol.</em><strong>28</strong>, 411–437 (2012).
</cite> [<a href="https://doi.org/10.1146/annurev-cellbio-092910-153958" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22578141/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saka,%20H.%20A.%20&amp;%20Valdivia,%20R.%20Emerging%20roles%20for%20lipid%20droplets%20in%20immunity%20and%20host-pathogen%20interactions.%20Annu.%20Rev.%20Cell%20Dev.%20Biol.28,%20411%E2%80%93437%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Laufman, O., Perrino, J. &amp; Andino, R. Viral generated inter-organelle contacts redirect lipid flux for genome replication. <em>Cell</em><strong>178</strong>, 275–289.e216 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.05.030" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7077330/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31204099/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Laufman,%20O.,%20Perrino,%20J.%20&amp;%20Andino,%20R.%20Viral%20generated%20inter-organelle%20contacts%20redirect%20lipid%20flux%20for%20genome%20replication.%20Cell178,%20275%E2%80%93289.e216%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>Herker, E. et al. Efficient hepatitis C virus particle formation requires diacylglycerol acyltransferase-1. <em>Nat. Med.</em><strong>16</strong>, 1295–1298 (2010).
</cite> [<a href="https://doi.org/10.1038/nm.2238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3431199/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20935628/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Herker,%20E.%20et%20al.%20Efficient%20hepatitis%20C%20virus%20particle%20formation%20requires%20diacylglycerol%20acyltransferase-1.%20Nat.%20Med.16,%201295%E2%80%931298%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR61">
<span class="label">61.</span><cite>Miyanari, Y. et al. The lipid droplet is an important organelle for hepatitis C virus production. <em>Nat. Cell Biol.</em><strong>9</strong>, 1089–1097 (2007).
</cite> [<a href="https://doi.org/10.1038/ncb1631" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17721513/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Miyanari,%20Y.%20et%20al.%20The%20lipid%20droplet%20is%20an%20important%20organelle%20for%20hepatitis%20C%20virus%20production.%20Nat.%20Cell%20Biol.9,%201089%E2%80%931097%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Munger, J. et al. Systems-level metabolic flux profiling identifies fatty acid synthesis as a target for antiviral therapy. <em>Nat. Biotechnol.</em><strong>26</strong>, 1179–1186 (2008).
</cite> [<a href="https://doi.org/10.1038/nbt.1500" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2825756/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18820684/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Munger,%20J.%20et%20al.%20Systems-level%20metabolic%20flux%20profiling%20identifies%20fatty%20acid%20synthesis%20as%20a%20target%20for%20antiviral%20therapy.%20Nat.%20Biotechnol.26,%201179%E2%80%931186%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Li, X. et al. O-GlcNAc transferase suppresses inflammation and necroptosis by targeting receptor-interacting serine/threonine-protein kinase 3. <em>Immunity</em><strong>50</strong>, 576–590.e576 (2019).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2019.01.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6426684/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30770249/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20X.%20et%20al.%20O-GlcNAc%20transferase%20suppresses%20inflammation%20and%20necroptosis%20by%20targeting%20receptor-interacting%20serine/threonine-protein%20kinase%203.%20Immunity50,%20576%E2%80%93590.e576%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Zhang, F. et al. O-GlcNAc modification is an endogenous inhibitor of the proteasome. <em>Cell</em><strong>115</strong>, 715–725 (2003).
</cite> [<a href="https://doi.org/10.1016/s0092-8674(03)00974-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14675536/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20F.%20et%20al.%20O-GlcNAc%20modification%20is%20an%20endogenous%20inhibitor%20of%20the%20proteasome.%20Cell115,%20715%E2%80%93725%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR65">
<span class="label">65.</span><cite>Yang, W. H. et al. Modification of p53 with O-linked N-acetylglucosamine regulates p53 activity and stability. <em>Nat. Cell Biol.</em><strong>8</strong>, 1074–1083 (2006).
</cite> [<a href="https://doi.org/10.1038/ncb1470" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16964247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yang,%20W.%20H.%20et%20al.%20Modification%20of%20p53%20with%20O-linked%20N-acetylglucosamine%20regulates%20p53%20activity%20and%20stability.%20Nat.%20Cell%20Biol.8,%201074%E2%80%931083%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR66">
<span class="label">66.</span><cite>Yan, W. et al. Cancer-cell-secreted miR-122 suppresses O-GlcNAcylation to promote skeletal muscle proteolysis. <em>Nat. Cell Biol.</em><strong>24</strong>, 793–804 (2022).
</cite> [<a href="https://doi.org/10.1038/s41556-022-00893-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9107513/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35469018/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yan,%20W.%20et%20al.%20Cancer-cell-secreted%20miR-122%20suppresses%20O-GlcNAcylation%20to%20promote%20skeletal%20muscle%20proteolysis.%20Nat.%20Cell%20Biol.24,%20793%E2%80%93804%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR67">
<span class="label">67.</span><cite>Song, T. et al. DOT1L O-GlcNAcylation promotes its protein stability and MLL-fusion leukemia cell proliferation. <em>Cell Rep.</em><strong>36</strong>, 109739 (2021).
</cite> [<a href="https://doi.org/10.1016/j.celrep.2021.109739" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34551297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Song,%20T.%20et%20al.%20DOT1L%20O-GlcNAcylation%20promotes%20its%20protein%20stability%20and%20MLL-fusion%20leukemia%20cell%20proliferation.%20Cell%20Rep.36,%20109739%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR68">
<span class="label">68.</span><cite>Dong, H., Liu, Z. &amp; Wen, H. Protein O-GlcNAcylation Regulates Innate Immune Cell Function. <em>Front. Immunol.</em><strong>13</strong>, 805018 (2022).
</cite> [<a href="https://doi.org/10.3389/fimmu.2022.805018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8850411/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35185892/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dong,%20H.,%20Liu,%20Z.%20&amp;%20Wen,%20H.%20Protein%20O-GlcNAcylation%20Regulates%20Innate%20Immune%20Cell%20Function.%20Front.%20Immunol.13,%20805018%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR69">
<span class="label">69.</span><cite>Ting, J. P. et al. The NLR gene family: a standard nomenclature. <em>Immunity</em><strong>28</strong>, 285–287 (2008).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2008.02.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2630772/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18341998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ting,%20J.%20P.%20et%20al.%20The%20NLR%20gene%20family:%20a%20standard%20nomenclature.%20Immunity28,%20285%E2%80%93287%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR70">
<span class="label">70.</span><cite>Bauernfried, S., Scherr, M. J., Pichlmair, A., Duderstadt, K. E. &amp; Hornung, V. Human NLRP1 is a sensor for double-stranded RNA. <em>Science</em><strong>371</strong>. 10.1126/science.abd0811 (2021).</cite> [<a href="https://doi.org/10.1126/science.abd0811" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33243852/" class="usa-link">PubMed</a>]</li>
<li id="CR71">
<span class="label">71.</span><cite>Allen, I. C. et al. The NLRP3 inflammasome mediates in vivo innate immunity to influenza A virus through recognition of viral RNA. <em>Immunity</em><strong>30</strong>, 556–565 (2009).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2009.02.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2803103/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19362020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Allen,%20I.%20C.%20et%20al.%20The%20NLRP3%20inflammasome%20mediates%20in%20vivo%20innate%20immunity%20to%20influenza%20A%20virus%20through%20recognition%20of%20viral%20RNA.%20Immunity30,%20556%E2%80%93565%20(2009)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR72">
<span class="label">72.</span><cite>Zhang, W. et al. Lactate is a natural suppressor of RLR signaling by targeting MAVS. <em>Cell</em><strong>178</strong>, 176–189.e115 (2019).
</cite> [<a href="https://doi.org/10.1016/j.cell.2019.05.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6625351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31155231/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20W.%20et%20al.%20Lactate%20is%20a%20natural%20suppressor%20of%20RLR%20signaling%20by%20targeting%20MAVS.%20Cell178,%20176%E2%80%93189.e115%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR73">
<span class="label">73.</span><cite>Wolf, A. J. et al. Hexokinase is an innate immune receptor for the detection of bacterial peptidoglycan. <em>Cell</em>. 10.1016/j.cell.2016.05.076 (2016).</cite> [<a href="https://doi.org/10.1016/j.cell.2016.05.076" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5534359/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27374331/" class="usa-link">PubMed</a>]</li>
<li id="CR74">
<span class="label">74.</span><cite>Angelova, M., Ortiz-Meoz, R. F., Walker, S. &amp; Knipe, D. M. Inhibition of O-linked N-acetylglucosamine transferase reduces replication of herpes simplex virus and human cytomegalovirus. <em>J. Virol.</em><strong>89</strong>, 8474–8483 (2015).
</cite> [<a href="https://doi.org/10.1128/JVI.01002-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4524265/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26041297/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Angelova,%20M.,%20Ortiz-Meoz,%20R.%20F.,%20Walker,%20S.%20&amp;%20Knipe,%20D.%20M.%20Inhibition%20of%20O-linked%20N-acetylglucosamine%20transferase%20reduces%20replication%20of%20herpes%20simplex%20virus%20and%20human%20cytomegalovirus.%20J.%20Virol.89,%208474%E2%80%938483%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR75">
<span class="label">75.</span><cite>Baum, A., Sachidanandam, R. &amp; Garcia-Sastre, A. Preference of RIG-I for short viral RNA molecules in infected cells revealed by next-generation sequencing. <em>Proc. Natl. Acad. Sci. USA</em><strong>107</strong>, 16303–16308 (2010).
</cite> [<a href="https://doi.org/10.1073/pnas.1005077107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2941304/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20805493/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baum,%20A.,%20Sachidanandam,%20R.%20&amp;%20Garcia-Sastre,%20A.%20Preference%20of%20RIG-I%20for%20short%20viral%20RNA%20molecules%20in%20infected%20cells%20revealed%20by%20next-generation%20sequencing.%20Proc.%20Natl.%20Acad.%20Sci.%20USA107,%2016303%E2%80%9316308%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR76">
<span class="label">76.</span><cite>Liu, G., Park, H. S., Pyo, H. M., Liu, Q. &amp; Zhou, Y. Influenza A virus panhandle structure is directly involved in RIG-I activation and interferon induction. <em>J. Virol.</em><strong>89</strong>, 6067–6079 (2015).
</cite> [<a href="https://doi.org/10.1128/JVI.00232-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4442436/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25810557/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liu,%20G.,%20Park,%20H.%20S.,%20Pyo,%20H.%20M.,%20Liu,%20Q.%20&amp;%20Zhou,%20Y.%20Influenza%20A%20virus%20panhandle%20structure%20is%20directly%20involved%20in%20RIG-I%20activation%20and%20interferon%20induction.%20J.%20Virol.89,%206067%E2%80%936079%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR77">
<span class="label">77.</span><cite>Li, Q., Pene, V., Krishnamurthy, S., Cha, H. &amp; Liang, T. J. Hepatitis C virus infection activates an innate pathway involving IKK-alpha in lipogenesis and viral assembly. <em>Nat. Med.</em><strong>19</strong>, 722–729 (2013).
</cite> [<a href="https://doi.org/10.1038/nm.3190" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3676727/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23708292/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20Q.,%20Pene,%20V.,%20Krishnamurthy,%20S.,%20Cha,%20H.%20&amp;%20Liang,%20T.%20J.%20Hepatitis%20C%20virus%20infection%20activates%20an%20innate%20pathway%20involving%20IKK-alpha%20in%20lipogenesis%20and%20viral%20assembly.%20Nat.%20Med.19,%20722%E2%80%93729%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR78">
<span class="label">78.</span><cite>Yan, B. et al. Characterization of the lipidomic profile of human coronavirus-infected cells: implications for lipid metabolism remodeling upon coronavirus replication. <em>Viruses</em><strong>11</strong>. 10.3390/v11010073 (2019).</cite> [<a href="https://doi.org/10.3390/v11010073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6357182/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30654597/" class="usa-link">PubMed</a>]</li>
<li id="CR79">
<span class="label">79.</span><cite>Dias, S. S. G. et al. Lipid droplets fuel SARS-CoV-2 replication and production of inflammatory mediators. <em>PLoS Pathog.</em><strong>16</strong>, e1009127 (2020).
</cite> [<a href="https://doi.org/10.1371/journal.ppat.1009127" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7773323/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33326472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dias,%20S.%20S.%20G.%20et%20al.%20Lipid%20droplets%20fuel%20SARS-CoV-2%20replication%20and%20production%20of%20inflammatory%20mediators.%20PLoS%20Pathog.16,%20e1009127%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR80">
<span class="label">80.</span><cite>Kimmel, A. R., Brasaemle, D. L., McAndrews-Hill, M., Sztalryd, C. &amp; Londos, C. Adoption of PERILIPIN as a unifying nomenclature for the mammalian PAT-family of intracellular lipid storage droplet proteins. <em>J. Lipid Res.</em><strong>51</strong>, 468–471 (2010).
</cite> [<a href="https://doi.org/10.1194/jlr.R000034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2817576/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19638644/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kimmel,%20A.%20R.,%20Brasaemle,%20D.%20L.,%20McAndrews-Hill,%20M.,%20Sztalryd,%20C.%20&amp;%20Londos,%20C.%20Adoption%20of%20PERILIPIN%20as%20a%20unifying%20nomenclature%20for%20the%20mammalian%20PAT-family%20of%20intracellular%20lipid%20storage%20droplet%20proteins.%20J.%20Lipid%20Res.51,%20468%E2%80%93471%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR81">
<span class="label">81.</span><cite>Kimmel, A. R. &amp; Sztalryd, C. The perilipins: major cytosolic lipid droplet-associated proteins and their roles in cellular lipid storage, mobilization, and systemic homeostasis. <em>Annu. Rev. Nutr.</em><strong>36</strong>, 471–509 (2016).
</cite> [<a href="https://doi.org/10.1146/annurev-nutr-071813-105410" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27431369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kimmel,%20A.%20R.%20&amp;%20Sztalryd,%20C.%20The%20perilipins:%20major%20cytosolic%20lipid%20droplet-associated%20proteins%20and%20their%20roles%20in%20cellular%20lipid%20storage,%20mobilization,%20and%20systemic%20homeostasis.%20Annu.%20Rev.%20Nutr.36,%20471%E2%80%93509%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR82">
<span class="label">82.</span><cite>Chu, J. et al. Pharmacological inhibition of fatty acid synthesis blocks SARS-CoV-2 replication. <em>Nat. Metab</em>. 10.1038/s42255-021-00479-4 (2021).</cite> [<a href="https://doi.org/10.1038/s42255-021-00479-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8475461/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34580494/" class="usa-link">PubMed</a>]</li>
<li id="CR83">
<span class="label">83.</span><cite>Yuan, S. et al. SREBP-dependent lipidomic reprogramming as a broad-spectrum antiviral target. <em>Nat. Commun.</em><strong>10</strong>, 120 (2019).
</cite> [<a href="https://doi.org/10.1038/s41467-018-08015-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6328544/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30631056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yuan,%20S.%20et%20al.%20SREBP-dependent%20lipidomic%20reprogramming%20as%20a%20broad-spectrum%20antiviral%20target.%20Nat.%20Commun.10,%20120%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR84">
<span class="label">84.</span><cite>Episcopio, D. et al. Atorvastatin restricts the ability of influenza virus to generate lipid droplets and severely suppresses the replication of the virus. <em>FASEB J.</em><strong>33</strong>, 9516–9525 (2019).
</cite> [<a href="https://doi.org/10.1096/fj.201900428RR" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6662987/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31125254/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Episcopio,%20D.%20et%20al.%20Atorvastatin%20restricts%20the%20ability%20of%20influenza%20virus%20to%20generate%20lipid%20droplets%20and%20severely%20suppresses%20the%20replication%20of%20the%20virus.%20FASEB%20J.33,%209516%E2%80%939525%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR85">
<span class="label">85.</span><cite>Sodi, V. L. et al. Nutrient sensor O-GlcNAc transferase controls cancer lipid metabolism via SREBP-1 regulation. <em>Oncogene</em><strong>37</strong>, 924–934 (2018).
</cite> [<a href="https://doi.org/10.1038/onc.2017.395" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5814337/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29059153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Sodi,%20V.%20L.%20et%20al.%20Nutrient%20sensor%20O-GlcNAc%20transferase%20controls%20cancer%20lipid%20metabolism%20via%20SREBP-1%20regulation.%20Oncogene37,%20924%E2%80%93934%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR86">
<span class="label">86.</span><cite>Baldini, S. F. &amp; Lefebvre, T. O-GlcNAcylation and the metabolic shift in high-proliferating cells: all the evidence suggests that sugars dictate the flux of lipid biogenesis in tumor processes. <em>Front Oncol.</em><strong>6</strong>, 6 (2016).
</cite> [<a href="https://doi.org/10.3389/fonc.2016.00006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4722119/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26835421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Baldini,%20S.%20F.%20&amp;%20Lefebvre,%20T.%20O-GlcNAcylation%20and%20the%20metabolic%20shift%20in%20high-proliferating%20cells:%20all%20the%20evidence%20suggests%20that%20sugars%20dictate%20the%20flux%20of%20lipid%20biogenesis%20in%20tumor%20processes.%20Front%20Oncol.6,%206%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR87">
<span class="label">87.</span><cite>Lei, Y. et al. The mitochondrial proteins NLRX1 and TUFM form a complex that regulates type I interferon and autophagy. <em>Immunity</em><strong>36</strong>, 933–946 (2012).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2012.03.025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3397828/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22749352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lei,%20Y.%20et%20al.%20The%20mitochondrial%20proteins%20NLRX1%20and%20TUFM%20form%20a%20complex%20that%20regulates%20type%20I%20interferon%20and%20autophagy.%20Immunity36,%20933%E2%80%93946%20(2012)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR88">
<span class="label">88.</span><cite>Lu, M. et al. N(6)-methyladenosine modification enables viral RNA to escape recognition by RNA sensor RIG-I. <em>Nat. Microbiol</em><strong>5</strong>, 584–598 (2020).
</cite> [<a href="https://doi.org/10.1038/s41564-019-0653-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7137398/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32015498/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lu,%20M.%20et%20al.%20N(6)-methyladenosine%20modification%20enables%20viral%20RNA%20to%20escape%20recognition%20by%20RNA%20sensor%20RIG-I.%20Nat.%20Microbiol5,%20584%E2%80%93598%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR89">
<span class="label">89.</span><cite>Guo, X. J. et al. Lung gammadelta T cells mediate protective responses during neonatal influenza infection that are associated with type 2 immunity. <em>Immunity</em><strong>49</strong>, 531–544.e536 (2018).
</cite> [<a href="https://doi.org/10.1016/j.immuni.2018.07.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6345262/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30170813/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Guo,%20X.%20J.%20et%20al.%20Lung%20gammadelta%20T%20cells%20mediate%20protective%20responses%20during%20neonatal%20influenza%20infection%20that%20are%20associated%20with%20type%202%20immunity.%20Immunity49,%20531%E2%80%93544.e536%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR90">
<span class="label">90.</span><cite>Kumar, N. et al. Dendritic molecular baskets for selective binding of toxic methotrexate. <em>Angew. Chem. Int Ed. Engl.</em><strong>64</strong>, e202420574 (2025).
</cite> [<a href="https://doi.org/10.1002/anie.202420574" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39529410/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kumar,%20N.%20et%20al.%20Dendritic%20molecular%20baskets%20for%20selective%20binding%20of%20toxic%20methotrexate.%20Angew.%20Chem.%20Int%20Ed.%20Engl.64,%20e202420574%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR91">
<span class="label">91.</span><cite>Chen, S., Zhou, Y., Chen, Y. &amp; Gu, J. fastp: an ultra-fast all-in-one FASTQ preprocessor. <em>Bioinformatics</em><strong>34</strong>, i884–i890 (2018).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/bty560" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6129281/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30423086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Chen,%20S.,%20Zhou,%20Y.,%20Chen,%20Y.%20&amp;%20Gu,%20J.%20fastp:%20an%20ultra-fast%20all-in-one%20FASTQ%20preprocessor.%20Bioinformatics34,%20i884%E2%80%93i890%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR92">
<span class="label">92.</span><cite>Kim, D., Paggi, J. M., Park, C., Bennett, C. &amp; Salzberg, S. L. Graph-based genome alignment and genotyping with HISAT2 and HISAT-genotype. <em>Nat. Biotechnol.</em><strong>37</strong>, 907–915 (2019).
</cite> [<a href="https://doi.org/10.1038/s41587-019-0201-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7605509/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31375807/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kim,%20D.,%20Paggi,%20J.%20M.,%20Park,%20C.,%20Bennett,%20C.%20&amp;%20Salzberg,%20S.%20L.%20Graph-based%20genome%20alignment%20and%20genotyping%20with%20HISAT2%20and%20HISAT-genotype.%20Nat.%20Biotechnol.37,%20907%E2%80%93915%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR93">
<span class="label">93.</span><cite>Zhao, B. et al. SUSD2 suppresses CD8(+) T cell antitumor immunity by targeting IL-2 receptor signaling. <em>Nat. Immunol.</em><strong>23</strong>, 1588–1599 (2022).
</cite> [<a href="https://doi.org/10.1038/s41590-022-01326-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9669207/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36266363/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhao,%20B.%20et%20al.%20SUSD2%20suppresses%20CD8(+)%20T%20cell%20antitumor%20immunity%20by%20targeting%20IL-2%20receptor%20signaling.%20Nat.%20Immunol.23,%201588%E2%80%931599%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR94">
<span class="label">94.</span><cite>Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <em>Genome Biol.</em><strong>15</strong>, 550 (2014).
</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Love,%20M.%20I.,%20Huber,%20W.%20&amp;%20Anders,%20S.%20Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2.%20Genome%20Biol.15,%20550%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR95">
<span class="label">95.</span><cite>Thul, P. J. et al. A subcellular map of the human proteome. <em>Science</em><strong>356</strong>. 10.1126/science.aal3321 (2017).</cite> [<a href="https://doi.org/10.1126/science.aal3321" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28495876/" class="usa-link">PubMed</a>]</li>
<li id="CR96">
<span class="label">96.</span><cite>Bardou, P., Mariette, J., Escudie, F., Djemiel, C. &amp; Klopp, C. Jvenn: an interactive Venn diagram viewer. <em>BMC Bioinform.</em><strong>15</strong>, 293 (2014).</cite> [<a href="https://doi.org/10.1186/1471-2105-15-293" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4261873/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25176396/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bardou,%20P.,%20Mariette,%20J.,%20Escudie,%20F.,%20Djemiel,%20C.%20&amp;%20Klopp,%20C.%20Jvenn:%20an%20interactive%20Venn%20diagram%20viewer.%20BMC%20Bioinform.15,%20293%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365020/bin/41467_2025_63085_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Information</a><sup> (4.9MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365020/bin/41467_2025_63085_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">41467_2025_63085_MOESM2_ESM.pdf</a><sup> (84.9KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365020/bin/41467_2025_63085_MOESM3_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Data 1</a><sup> (353.1KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365020/bin/41467_2025_63085_MOESM4_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Reporting Summary</a><sup> (148.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365020/bin/41467_2025_63085_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (421KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12365020/bin/41467_2025_63085_MOESM6_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Source Data</a><sup> (4MB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The RNA-seq data generated in this study have been deposited in the Gene Expression Omnibus (GEO) under accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE297354" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE297354</a>. The MS data generated in this study have been deposited in the MassIVE server under accession number MSV000097062 (<a href="https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=dffa2d6a49a04a69a79cf1df8218c2c4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=dffa2d6a49a04a69a79cf1df8218c2c4</a>). <a href="#Sec38" class="usa-link">Source data</a> are provided with this paper.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Nature Communications are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s41467-025-63085-y"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/41467_2025_Article_63085.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (3.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12365020/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12365020/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12365020%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12365020/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12365020/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12365020/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830102/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12365020/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830102/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12365020/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12365020/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="1S5o3wqt3qs3aXeCSNN1pSjUs9dropfwxihWufuUCJzeRfmjU1NrhqvbbI3q3VCq">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
