
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Mechanistic computational modeling of sFLT1 secretion dynamics - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE53464D8AF43D1305464D001BF98600.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ploscomp">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370208/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="PLOS Computational Biology">
<meta name="citation_title" content="Mechanistic computational modeling of sFLT1 secretion dynamics">
<meta name="citation_author" content="Amy Gill">
<meta name="citation_author_institution" content="Institute for Computational Medicine, Johns Hopkins University, Baltimore, Maryland, United States of America">
<meta name="citation_author_institution" content="Department of Biomedical Engineering, Johns Hopkins University, Baltimore, Maryland, United States of America">
<meta name="citation_author" content="Karina Kinghorn">
<meta name="citation_author_institution" content="Department of Biology, University of North Carolina, Chapel Hill, North Carolina, United States of America">
<meta name="citation_author_institution" content="Curriculum in Cell Biology and Physiology, University of North Carolina, Chapel Hill, North Carolina, United States of America">
<meta name="citation_author" content="Victoria L Bautch">
<meta name="citation_author_institution" content="Department of Biology, University of North Carolina, Chapel Hill, North Carolina, United States of America">
<meta name="citation_author_institution" content="Curriculum in Cell Biology and Physiology, University of North Carolina, Chapel Hill, North Carolina, United States of America">
<meta name="citation_author_institution" content="McAllister Heart Institute, University of North Carolina, Chapel Hill, North Carolina, United States of America">
<meta name="citation_author" content="Feilim Mac Gabhann">
<meta name="citation_author_institution" content="Institute for Computational Medicine, Johns Hopkins University, Baltimore, Maryland, United States of America">
<meta name="citation_author_institution" content="Department of Biomedical Engineering, Johns Hopkins University, Baltimore, Maryland, United States of America">
<meta name="citation_publication_date" content="2025 Aug 18">
<meta name="citation_volume" content="21">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e1013324">
<meta name="citation_doi" content="10.1371/journal.pcbi.1013324">
<meta name="citation_pmid" content="40825055">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370208/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370208/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370208/pdf/pcbi.1013324.pdf">
<meta name="description" content="Constitutively secreted by endothelial cells, soluble FLT1 (sFLT1 or sVEGFR1) binds and sequesters extracellular vascular endothelial growth factors (VEGF), thereby reducing VEGF binding to VEGF receptor tyrosine kinases and their downstream ...">
<meta name="og:title" content="Mechanistic computational modeling of sFLT1 secretion dynamics">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Constitutively secreted by endothelial cells, soluble FLT1 (sFLT1 or sVEGFR1) binds and sequesters extracellular vascular endothelial growth factors (VEGF), thereby reducing VEGF binding to VEGF receptor tyrosine kinases and their downstream ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370208/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12370208">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1371/journal.pcbi.1013324"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/pcbi.1013324.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370208%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12370208/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12370208/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370208/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ploscomp.png" alt="PLOS Computational Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to PLOS Computational Biology" title="Link to PLOS Computational Biology" shape="default" href="https://doi.org/10.1371/journal.pcbi.1013324" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">PLoS Comput Biol</button></div>. 2025 Aug 18;21(8):e1013324. doi: <a href="https://doi.org/10.1371/journal.pcbi.1013324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pcbi.1013324</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22PLoS%20Comput%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22PLoS%20Comput%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22PLoS%20Comput%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22PLoS%20Comput%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Mechanistic computational modeling of sFLT1 secretion dynamics</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gill%20A%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Amy Gill</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Amy Gill</span></h3>
<div class="p">
<sup>1</sup>Institute for Computational Medicine, Johns Hopkins University, Baltimore, Maryland, United States of America</div>
<div class="p">
<sup>2</sup>Department of Biomedical Engineering, Johns Hopkins University, Baltimore, Maryland, United States of America</div>
<div>Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Software, Visualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Gill%20A%22%5BAuthor%5D" class="usa-link"><span class="name western">Amy Gill</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>*</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kinghorn%20K%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Karina Kinghorn</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Karina Kinghorn</span></h3>
<div class="p">
<sup>3</sup>Department of Biology, University of North Carolina, Chapel Hill, North Carolina, United States of America</div>
<div class="p">
<sup>4</sup>Curriculum in Cell Biology and Physiology, University of North Carolina, Chapel Hill, North Carolina, United States of America</div>
<div>Conceptualization, Investigation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kinghorn%20K%22%5BAuthor%5D" class="usa-link"><span class="name western">Karina Kinghorn</span></a>
</div>
</div>
<sup>3,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bautch%20VL%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Victoria L Bautch</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Victoria L Bautch</span></h3>
<div class="p">
<sup>3</sup>Department of Biology, University of North Carolina, Chapel Hill, North Carolina, United States of America</div>
<div class="p">
<sup>4</sup>Curriculum in Cell Biology and Physiology, University of North Carolina, Chapel Hill, North Carolina, United States of America</div>
<div class="p">
<sup>5</sup>McAllister Heart Institute, University of North Carolina, Chapel Hill, North Carolina, United States of America</div>
<div>Conceptualization, Funding acquisition, Investigation, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bautch%20VL%22%5BAuthor%5D" class="usa-link"><span class="name western">Victoria L Bautch</span></a>
</div>
</div>
<sup>3,</sup><sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mac%20Gabhann%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Feilim Mac Gabhann</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Feilim Mac Gabhann</span></h3>
<div class="p">
<sup>1</sup>Institute for Computational Medicine, Johns Hopkins University, Baltimore, Maryland, United States of America</div>
<div class="p">
<sup>2</sup>Department of Biomedical Engineering, Johns Hopkins University, Baltimore, Maryland, United States of America</div>
<div>Conceptualization, Funding acquisition, Investigation, Methodology, Software, Supervision, Visualization, Writing – original draft, Writing – review &amp; editing</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mac%20Gabhann%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Feilim Mac Gabhann</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>
</div>
<div class="cg p">Editor: <span class="name western">Mariko Okada</span><sup>6</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="aff001">
<sup>1</sup>Institute for Computational Medicine, Johns Hopkins University, Baltimore, Maryland, United States of America</div>
<div id="aff002">
<sup>2</sup>Department of Biomedical Engineering, Johns Hopkins University, Baltimore, Maryland, United States of America</div>
<div id="aff003">
<sup>3</sup>Department of Biology, University of North Carolina, Chapel Hill, North Carolina, United States of America</div>
<div id="aff004">
<sup>4</sup>Curriculum in Cell Biology and Physiology, University of North Carolina, Chapel Hill, North Carolina, United States of America</div>
<div id="aff005">
<sup>5</sup>McAllister Heart Institute, University of North Carolina, Chapel Hill, North Carolina, United States of America</div>
<div id="edit1">
<sup>6</sup>Osaka University, JAPAN</div>
<div class="author-notes p">
<div class="fn" id="coi001"><p>The authors have declared that no competing interests exist.</p></div>
<div class="fn" id="cor001">
<sup>✉</sup><p class="display-inline">* E-mail: <span>agill18@jhmi.edu</span></p>
</div>
</div>
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Amy Gill</span></strong>: <span class="role">Conceptualization, Data curation, Formal analysis, Investigation, Methodology, Software, Visualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Karina Kinghorn</span></strong>: <span class="role">Conceptualization, Investigation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Victoria L Bautch</span></strong>: <span class="role">Conceptualization, Funding acquisition, Investigation, Writing – review &amp; editing</span>
</div>
<div>
<strong class="contrib"><span class="name western">Feilim Mac Gabhann</span></strong>: <span class="role">Conceptualization, Funding acquisition, Investigation, Methodology, Software, Supervision, Visualization, Writing – original draft, Writing – review &amp; editing</span>
</div>
<div class="p">
<strong class="contrib"><span class="name western">Mariko Okada</span></strong>: <span class="role">Editor</span>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 18; Accepted 2025 Jul 10; Collection date 2025 Aug.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 Gill et al</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a>, which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12370208  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40825055/" class="usa-link">40825055</a>
</div>
<div class="ra xbox p" role="complementary" aria-label="Related or updated information about this article"><div>
<strong>Previous version available:</strong> This article is based on a previously available preprint posted on bioRxiv on February 18, 2025: "<a href="/articles/PMC11870409/" class="usa-link">Mechanistic computational modeling of sFLT1 secretion dynamics</a>".</div></div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>Constitutively secreted by endothelial cells, soluble FLT1 (sFLT1 or sVEGFR1) binds and sequesters extracellular vascular endothelial growth factors (VEGF), thereby reducing VEGF binding to VEGF receptor tyrosine kinases and their downstream signaling. In doing so, sFLT1 plays an important role in vascular development and in the patterning of new blood vessels in angiogenesis. Here, we develop multiple mechanistic models of sFLT1 secretion and identify a minimal mechanistic model that recapitulates key qualitative and quantitative features of temporal experimental datasets of sFLT1 secretion from multiple studies. We show that the experimental data on sFLT1 secretion is best represented by a delay differential equation (DDE) system including a maturation term, reflecting the time required between synthesis and secretion. Using optimization to identify appropriate values for the key mechanistic parameters in the model, we show that two model parameters (extracellular degradation rate constant and maturation time) are very strongly constrained by the experimental data, and that the remaining parameters are related by two strongly constrained constants. Thus, only one degree of freedom remains, and measurements of the intracellular levels of sFLT1 would fix the remaining parameters. Comparison between simulation predictions and additional experimental data of the outcomes of chemical inhibitors and genetic perturbations suggest that intermediate values of the secretion rate constant best match the simulation with experiments, which would completely constrain the model. However, some of the inhibitors tested produce results that cannot be reproduced by the model simulations, suggesting that additional mechanisms not included here are required to explain those inhibitors. Overall, the model reproduces most available experimental data and suggests targets for further quantitative investigation of the sFLT1 system.</p></section><section class="abstract" id="abstract2"><h2>Author summary</h2>
<p>Proteins that are typically found outside cells are initially made inside cells, and later secreted into extracellular space. Many of these secreted proteins have important functions outside the cell that are well-studied; however, usually much less is known about the pre-secretion life of these molecules. Many computational models only represent the extracellular versions of secreted proteins, reducing all production and secretion steps into a single modeled process. Here, we develop a mechanistic model of the production and secretion of a specific secreted protein, sFLT1, which inhibits blood vessel growth by acting as an extracellular sponge for another set of secreted proteins, the vascular endothelial growth factors. We compare several models to existing experimentally-measured sFLT1 data, and we show that the data are most simply explained by including a delay between intracellular sFLT1 production and sFLT1 transport or degradation. This is consistent with the biology of the cell’s secretory pathway, where immature proteins are gradually processed into mature forms over minutes to hours. Our approach could be incorporated into improved models for any pathway involving secreted proteins, including sFLT1-regulated models of blood vessel biology.</p></section><section id="sec001"><h2 class="pmc_sec_title">Introduction</h2>
<p>Secreted proteins, which represent 15% of the human genome and 9% of the human proteome, are essential local and systemic signaling factors in multicellular systems [<a href="#pcbi.1013324.ref001" class="usa-link" aria-describedby="pcbi.1013324.ref001">1</a>]. Most proteins destined for the extracellular space, or for intracellular locations other than the cytoplasm and nucleus, mature via the secretory pathway, in which new proteins are cotranslationally translocated into the endoplasmic reticulum (ER), folded, transported to the Golgi apparatus, post-translationally modified, sorted, and trafficked by vesicles to target compartments [<a href="#pcbi.1013324.ref002" class="usa-link" aria-describedby="pcbi.1013324.ref002">2</a>]. Many proteins synthesized through the secretory pathway are released by exocytosis, while others reside and function mainly in intracellular compartments (such as the Golgi or endosomes) [<a href="#pcbi.1013324.ref003" class="usa-link" aria-describedby="pcbi.1013324.ref003">3</a>–<a href="#pcbi.1013324.ref005" class="usa-link" aria-describedby="pcbi.1013324.ref005">5</a>]. Many proteins known to function extracellularly or at the plasma membrane also have biologically meaningful interactions in intracellular compartments, including receptor tyrosine kinases [<a href="#pcbi.1013324.ref006" class="usa-link" aria-describedby="pcbi.1013324.ref006">6</a>–<a href="#pcbi.1013324.ref010" class="usa-link" aria-describedby="pcbi.1013324.ref010">10</a>]. Although all secreted proteins begin as intracellular proteins, intracellular roles and dynamics of extracellularly secreted proteins are generally understudied.</p>
<p>Here, we explore the secretion of soluble FLT1 (sFLT1, sVEGFR1), a key negative regulator of the vascular endothelial growth factors (VEGF), a family of cytokines that promote angiogenesis. sFLT1 is an alternatively spliced truncated isoform of VEGF Receptor 1 (VEGFR1 or FLT1) that retains the extracellular domains for binding ligands (including VEGFA, VEGFB, and placental growth factor (PlGF)) and heparan sulfate proteoglycans (HSPGs), but lacks transmembrane or intracellular tyrosine kinase domains, yielding a secreted, non-membrane anchored, and non-signaling receptor [<a href="#pcbi.1013324.ref011" class="usa-link" aria-describedby="pcbi.1013324.ref011">11</a>,<a href="#pcbi.1013324.ref012" class="usa-link" aria-describedby="pcbi.1013324.ref012">12</a>]. Constitutively secreted by endothelial cells [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>], sFLT1 binds and sequesters most extracellular VEGF isoforms, thereby reducing binding of these VEGF isoforms to VEGF receptor tyrosine kinases and the downstream signaling [<a href="#pcbi.1013324.ref014" class="usa-link" aria-describedby="pcbi.1013324.ref014">14</a>–<a href="#pcbi.1013324.ref016" class="usa-link" aria-describedby="pcbi.1013324.ref016">16</a>]. As a VEGF antagonist and antiangiogenic factor, sFLT1 prevents vascular overgrowth and is essential for corneal avascularity [<a href="#pcbi.1013324.ref017" class="usa-link" aria-describedby="pcbi.1013324.ref017">17</a>]. This inhibitory role is also important during angiogenesis for proper patterning of new blood vessel networks: sFLT1 from endothelial cells interacts extracellularly with stromal VEGF to reinforce VEGF gradients extending away from existing vasculature, guiding nascent vascular sprouts to reduce vascular overgrowth and disorganization [<a href="#pcbi.1013324.ref018" class="usa-link" aria-describedby="pcbi.1013324.ref018">18</a>–<a href="#pcbi.1013324.ref020" class="usa-link" aria-describedby="pcbi.1013324.ref020">20</a>].</p>
<p>Dysregulated sFLT1 production and secretion is a pathogenic factor in several disorders including preeclampsia, a syndrome of endothelial dysfunction during pregnancy characterized by hypertension, kidney damage, and significant maternal and fetal morbidity and mortality. In preeclampsia, excessive sFLT1 production and secretion from the placenta leads to elevated serum sFLT1, reducing free serum levels of VEGF and PlGF, inhibiting VEGF-mediated vasodilation and angiogenesis, causing hypertension and damaging vasculature [<a href="#pcbi.1013324.ref021" class="usa-link" aria-describedby="pcbi.1013324.ref021">21</a>]. While we do not focus on pathology here, a quantitative understanding of sFLT1 production and secretion from endothelial cells could be extended in the future to simulate pathologies.</p>
<p>While the extracellular role of sFLT1 is well characterized, less is known about its intracellular biology. Recent work suggests unexpectedly high intracellular sFLT1 levels in endothelial cells and long intracellular residence time [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>,<a href="#pcbi.1013324.ref022" class="usa-link" aria-describedby="pcbi.1013324.ref022">22</a>], motivating a quantitative characterization of sFLT1 trafficking and elucidation of the intracellular dynamics of sFLT1 and its possible intracellular roles. Past studies also suggest the full-length membrane-integral FLT1 (mFLT1) protein resides largely intracellularly [<a href="#pcbi.1013324.ref023" class="usa-link" aria-describedby="pcbi.1013324.ref023">23</a>], much of it in the Golgi, and it traffics to the plasma membrane in a calcium-dependent manner [<a href="#pcbi.1013324.ref008" class="usa-link" aria-describedby="pcbi.1013324.ref008">8</a>]. sFLT1 may be retained in secretory organelles by a similar mechanism to the larger mFLT1 isoform.</p>
<p>Mechanistic computational modeling represents biochemical molecular networks as systems of equations based on prior knowledge of regulatory interactions and parameterized with experimental data. These models allow virtual exploration of molecular networks while tracking components that may be difficult to measure experimentally, and are powerful tools for testing mechanistic hypotheses and screening new treatment approaches [<a href="#pcbi.1013324.ref024" class="usa-link" aria-describedby="pcbi.1013324.ref024">24</a>]. While many mechanistic models describe systems as sets of ordinary differential equations (ODEs), time delays between processes are common in biology, and recent work shows that delay differential equations (DDEs) can better represent the behavior of several common network motifs in biochemical reaction networks [<a href="#pcbi.1013324.ref025" class="usa-link" aria-describedby="pcbi.1013324.ref025">25</a>]. This study builds on our recent model of sFLT1 production and secretion in endothelial cells [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>], which we believe is the first mechanistic model to include intracellular sFLT1. Several previous models of the VEGF system have included the impact of sFLT1 secretion on extracellular sFLT1 and VEGF levels, including pharmacological-like distribution across tissues [<a href="#pcbi.1013324.ref026" class="usa-link" aria-describedby="pcbi.1013324.ref026">26</a>–<a href="#pcbi.1013324.ref028" class="usa-link" aria-describedby="pcbi.1013324.ref028">28</a>] and local gradients [<a href="#pcbi.1013324.ref019" class="usa-link" aria-describedby="pcbi.1013324.ref019">19</a>], but did not explore the intracellular dynamics of sFLT1 secretion. In our recent work characterizing intracellular sFLT1 trafficking from protein synthesis to secretion [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>], we briefly introduced a model linking intracellular and extracellular sFLT1 pools through protein synthesis, secretion, and intracellular and extracellular degradation. We used this model to estimate a basal secretion rate of ~30,000 molecules per cell per hour in human umbilical vein endothelial cells (HUVEC), comparable to other secreted proteins, and to infer that Golgi trafficking is absolutely required for sFLT1 secretion [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>].</p>
<p>Here, we extend our previous work to develop multiple mechanistic models of sFLT1 secretion and to identify a minimal mechanistic model that recapitulates key qualitative and quantitative features of experimental sFLT1 time course datasets. We compare several ODE-based and DDE-based models to capture the dynamics of sFLT1 secretion, using experimental data from multiple studies to parameterize the models and evaluate their relative performance. We then compare simulation predictions of perturbation by secretion pathway inhibitors to the outcomes of equivalent experiments.</p></section><section id="sec002"><h2 class="pmc_sec_title">Methods</h2>
<section id="sec003"><h3 class="pmc_sec_title">Models of sFLT1 production and secretion</h3>
<p>We constructed several mechanistic models to explore sFLT1 secretion dynamics, each of which consists of two coupled nonlinear differential equations describing the levels of sFLT1 in two locations: extracellular sFLT1 (<em>X</em>) that has been secreted out of the cell into conditioned media, and intracellular sFLT1 (<em>I</em>), a sum of immature and mature protein inside the cell, treating all intracellular locations (ER, Golgi, vesicles, etc.) as one well-mixed compartment. Molecular levels of <em>I</em> and <em>X</em> were tracked in units of number per cell (#/cell). We briefly introduced one example of these models in recent work [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>]; here we provide additional detail on model construction and analysis, compare it to additional model formulations, and re-parameterize based on additional experimental data.</p></section><section id="sec004"><h3 class="pmc_sec_title">ODE-based model</h3>
<p>We created an initial model using ordinary differential equations (ODEs) (<a href="#pcbi.1013324.g001" class="usa-link">Fig 1A</a>) to describe how the mechanistic processes impact the level of intracellular and extracellular sFLT1. The equations for intracellular and extracellular sFLT1 include rates for four mechanistic processes:</p>
<figure class="fig xbox font-sm" id="pcbi.1013324.g001"><h4 class="obj_head">Fig 1. Modeling sFLT1 secretion with ordinary differential equations (ODEs).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370208_pcbi.1013324.g001.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dda/12370208/a9b39d2038b4/pcbi.1013324.g001.jpg" loading="lazy" height="1392" width="699" alt="Fig 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013324.g001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Model of sFLT1 secretion dynamics describing the impact of production (rate <em>α</em>), secretion (rate constant <em>β</em>), intracellular degradation (<em>γ</em>), and extracellular degradation (<em>δ</em>) on the concentrations of intracellular (<em>I</em>) and extracellular (<em>X</em>) sFLT1 as a system of ordinary differential equations (ODEs). Created with BioRender.com. <strong>(B)</strong> Comparison of simulation (top row) and experimental (bottom row) approaches for studying sFLT1 secretion in the constitutive secretion scenario (left column) and pulse-chase secretion scenario (right column). <strong>(C)</strong> Simulated time courses (<em>n = 100</em>) of extracellular (<em>X</em>, top) and intracellular (<em>I</em>, bottom) sFLT1 secretion for pulse-chase (left) and constitutive (right) cases using the ODE model. <em>X</em> was normalized to its value at 8h (upper left) or 24h (upper right), while <em>I</em> was normalized to its value at 0h. The highlighted region (P) marks the 20-minute pulse. <strong>(D)</strong> Number of intracellular sFLT1 (<em>I</em>) molecules per cell over time and <strong>(E)</strong> distribution of steady state intracellular sFLT1 (<em>I</em><sub><em>SS</em></sub>) during simulations of constitutive secretion using the ODE model.</p></figcaption></figure><ul class="list" style="list-style-type:disc">
<li><p><em>production</em>, the synthesis of new intracellular protein (<em>∅</em> → <em>I</em>) characterized as a zeroth-order process with a rate constant α (when on) or 0 (when off), representing translation of intracellular protein and assuming no feedback dependence of sFLT1 production on current sFLT1 levels;</p></li>
<li><p><em>secretion</em>, the transport from the intracellular to extracellular space (<em>I → X</em>) characterized as a first order process with rate constant β, representing exocytosis of mature sFLT1 protein;</p></li>
<li><p><em>intracellular degradation</em>, the depletion of intracellular sFLT1 (<em>I</em> → <em>∅</em>) characterized as a first order process with rate constant γ, representing lysosomal degradation or other proteolytic degradation mediated by intracellular factors;</p></li>
<li><p><em>extracellular degradation</em>, the depletion of extracellular sFLT1 (<em>X →</em><em>∅</em>) characterized as a first order process with rate constant δ, representing proteolytic degradation mediated by soluble, membrane-bound, or matrix-bound extracellular factors.</p></li>
</ul>
<p>Thus, the following equations define the time evolution of the system:</p>
<table class="disp-formula p" id="pcbi.1013324.e001"><tr>
<td class="formula"><math id="M1" display="block" overflow="linebreak"><mrow><mfrac><mrow><mi>d</mi><mi>I</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mi>α</mi><mo>−</mo><mi>β</mi><mi>·</mi><mi>I</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>−</mo><mi>γ</mi><mi>·</mi><mi>I</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math></td>
<td class="label">(1)</td>
</tr></table>
<table class="disp-formula p" id="pcbi.1013324.e002"><tr>
<td class="formula"><math id="M2" display="block" overflow="linebreak"><mrow><mfrac><mrow><mi>d</mi><mi>X</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mi>β</mi><mi>·</mi><mi>I</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo><mo>−</mo><mi>δ</mi><mi>·</mi><mi>X</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math></td>
<td class="label">(2)</td>
</tr></table></section><section id="sec005"><h3 class="pmc_sec_title">DDE-based model</h3>
<p>We converted our initial ODE model to a system of delay differential equations (DDEs) (<a href="#pcbi.1013324.g002" class="usa-link">Fig 2A</a>) by introducing a fixed time delay τ to the secretion and intracellular degradation processes, representing a required protein maturation period before <em>I</em> can be transported or degraded:</p>
<figure class="fig xbox font-sm" id="pcbi.1013324.g002"><h4 class="obj_head">Fig 2. Modeling sFLT1 secretion with delay differential equations (DDEs).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370208_pcbi.1013324.g002.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dda/12370208/9caf87d4b7ef/pcbi.1013324.g002.jpg" loading="lazy" height="833" width="660" alt="Fig 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013324.g002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Model of sFLT1 secretion dynamics as a system of delay differential equations (DDEs), extending the ODE model (<a href="#pcbi.1013324.g001" class="usa-link">Fig 1A</a>) with a fixed maturation delay (τ) affecting secretion and intracellular degradation processes. Created with Biorender.com. <strong>(B)</strong> Simulated time courses <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e005"><math id="M5" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mi>n</mi><mo>=</mo><mn>594</mn><mo stretchy="false">)</mo></mrow></math></span> of extracellular (<em>X</em>, top) and intracellular (<em>I</em>, bottom) sFLT1 secretion for pulse-chase (left) and constitutive (right) cases using the base DDE model. <em>X</em> was normalized to its value at 8h (upper left) or 24h (upper right), while <em>I</em> was normalized to its value at 0h. The highlighted region (P) marks the 20-minute pulse.</p></figcaption></figure><table class="disp-formula p" id="pcbi.1013324.e003"><tr>
<td class="formula"><math id="M3" display="block" overflow="linebreak"><mrow><mfrac><mrow><mi>d</mi><mi>I</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mi>α</mi><mo>−</mo><mi>β</mi><mi>·</mi><mi>I</mi><mo stretchy="false">(</mo><mi>t</mi><mo>−</mo><mi>τ</mi><mo stretchy="false">)</mo><mo>−</mo><mi>γ</mi><mi>·</mi><mi>I</mi><mo stretchy="false">(</mo><mi>t</mi><mo>−</mo><mi>τ</mi><mo stretchy="false">)</mo></mrow></math></td>
<td class="label">(3)</td>
</tr></table>
<table class="disp-formula p" id="pcbi.1013324.e004"><tr>
<td class="formula"><math id="M4" display="block" overflow="linebreak"><mrow><mfrac><mrow><mi>d</mi><mi>X</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mi>β</mi><mi>·</mi><mi>I</mi><mo stretchy="false">(</mo><mi>t</mi><mo>−</mo><mi>τ</mi><mo stretchy="false">)</mo><mo>−</mo><mi>δ</mi><mi>·</mi><mi>X</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math></td>
<td class="label">(4)</td>
</tr></table></section><section id="sec006"><h3 class="pmc_sec_title">Candidate mechanistic models</h3>
<p>We formulated eight distinct candidate models containing different combinations of 3 processes (<a href="#pcbi.1013324.g003" class="usa-link">Fig 3A</a>):</p>
<figure class="fig xbox font-sm" id="pcbi.1013324.g003"><h4 class="obj_head">Fig 3. Comparing candidate models of sFLT1 secretion.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370208_pcbi.1013324.g003.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dda/12370208/588e6f6ca10c/pcbi.1013324.g003.jpg" loading="lazy" height="492" width="700" alt="Fig 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013324.g003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Structure of candidate model equations and equation term options for each process when present or absent in a given model. In the production decay term, the value of [<em>chase</em>] is 1 during the chase phase of pulse-chase simulations and 0 otherwise. <strong>(B)</strong> Top: Processes included (filled) or excluded (empty) in each model. ε: internalization, κ: production decay, τ: maturation delay. Middle: Cost (goodness of fit) of each model, defined as the sum of squared differences (SSD) between experimental data points and the lowest cost simulation of each model, with colors indicating the contribution of each experimental dataset. <em>X</em>: extracellular sFLT1, <em>I</em>: intracellular sFLT1. Bottom: Corrected Akaike Information Criterion (AIC<sub>C</sub>) scores for each model. All models are fit to the same number of observations (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e007"><math id="M7" display="inline" overflow="linebreak"><mrow><mi>n</mi><mo>=</mo><mn>26</mn></mrow></math></span>), while the number of parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e008"><math id="M8" display="inline" overflow="linebreak"><mrow><mi>k</mi></mrow></math></span> varies by model. <strong>(C)</strong> Impact of additional processes on the best-fit cost of the candidate models. Model nodes (points) are plotted as minimum model cost versus number of parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e009"><math id="M9" display="inline" overflow="linebreak"><mrow><mi>k</mi></mrow></math></span>. Edges (arrows) represent transitions between candidate model structures by adding the labeled process (τ (maturation delay): pink arrows, κ (production decay): blue arrows, ε (internalization): gray arrows. The vertical height of each arrow represents the decrease in cost from adding the labeled process to a given model.</p></figcaption></figure><ul class="list" style="list-style-type:disc">
<li><p><em>maturation delay</em>, the fixed time delay τ described above affecting secretion and intracellular degradation</p></li>
<li><p><em>internalization,</em> a first order transport reaction from the extracellular to intracellular space (<em>X → I</em>) with rate constant ε</p></li>
<li><p><em>production decay</em>, an exponential decay <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e006"><math id="M6" display="inline" overflow="linebreak"><mrow><msup><mrow><mi>e</mi></mrow><mrow><mo>−</mo><mi>κ</mi><mi>t</mi></mrow></msup></mrow></math></span> of production when transitioning from an expression pulse to chase at <em>t = 0</em>, with rate constant κ representing first order decay of labeled amino acids; note this does not affect the constitutive secretion scenario or the pulse segment of the pulse-chase scenario</p></li>
</ul>
<p>Model equations were constructed in piecewise fashion, with different rate terms depending on which processes were included in a given candidate model (<a href="#pcbi.1013324.s005" class="usa-link">S1 Table</a>). Note that the model with all processes absent (M1) is equivalent to our original ODE model (<a href="#pcbi.1013324.g001" class="usa-link">Fig 1A</a>), and the model with maturation delay only (M2) is equivalent to the base DDE model (<a href="#pcbi.1013324.g002" class="usa-link">Fig 2A</a>), and equivalent to the model formulation used in our previous publication [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>], but here with updated parameter values based on parameterization with additional experimental data.</p></section><section id="sec007"><h3 class="pmc_sec_title">Simulation &amp; analysis software</h3>
<p>We simulated differential equations in MATLAB version R2022a. We used R version 4.3.1 with RStudio release 2023.09.1 for all further data analysis and data visualization. See <a href="#pcbi.1013324.s004" class="usa-link">S3 Text</a> for the full list of packages and toolboxes used. All code is available at <a href="https://github.com/gillsignals/sFlt1-paper" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/gillsignals/sFlt1-paper</a>.</p></section><section id="sec008"><h3 class="pmc_sec_title">Simulating constitutive secretion experiments</h3>
<p>We simulate experiments in which cells secrete sFLT1 constitutively into the extracellular medium for a defined period of time (<a href="#pcbi.1013324.g001" class="usa-link">Fig 1B</a>, top). First, we run a pre-simulation, representing the time period of cell plating and culture before the experimental measurements, in which the cells come to a steady state. For this, we initialize intracellular and extracellular sFLT1 to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e010"><math id="M10" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><msub><mrow><mi>I</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo><msub><mrow><mi>X</mi></mrow><mrow><mn>0</mn></mrow></msub><mo stretchy="false">)</mo><mo>=</mo><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><mn>0</mn><mo stretchy="false">)</mo></mrow></math></span>, we set rate parameters to desired values, and we run the simulation to intracellular steady state <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e011"><math id="M11" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span> (defined as less than 0.5% change in <em>I</em> over 20h of simulation). These conditions are then used as the initial conditions for the simulation of the actual experiment, except that to represent media change at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e012"><math id="M12" display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math></span>, we reset the extracellular concentration <em>X</em> to 0 without changing the intracellular sFlt1 (<em>I = I</em><sub><em>SS</em></sub><em>)</em>. We then run the simulation for either 24h or 72h, matching the specific experiments being simulated, reporting <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e013"><math id="M13" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mi>I</mi><mo>,</mo><mi>X</mi><mo stretchy="false">)</mo></mrow></math></span> at 1-minute intervals.</p></section><section id="sec009"><h3 class="pmc_sec_title">Simulating pulse-chase experiments</h3>
<p>We also simulate experiments in which labeled media is introduced briefly and then removed, in order to track secretion of newly-synthesized (and thus labeled) sFLT1 <strong>(Fig 1B, bottom)</strong>. For these pulse-chase experiments, we initialize the intracellular and extracellular sFLT1 to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e014"><math id="M14" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><msub><mrow><mi>I</mi></mrow><mrow><mn>0</mn></mrow></msub><mo>,</mo><msub><mrow><mi>X</mi></mrow><mrow><mn>0</mn></mrow></msub><mo stretchy="false">)</mo><mo>=</mo><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><mn>0</mn><mo stretchy="false">)</mo></mrow></math></span>, set the parameters to desired values (including nonzero sFLT1 production rate constant α), and run the simulation for 20m from <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e015"><math id="M15" display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>=</mo><mo stretchy="false">(</mo><msub><mrow><mi>t</mi></mrow><mrow><mi>p</mi><mi>u</mi><mi>l</mi><mi>s</mi><mi>e</mi></mrow></msub><mo>,</mo><mn>0</mn><mo stretchy="false">)</mo></mrow></math></span> to mimic synthesis of labeled sFLT1 by cells exposed to pulse media. We used the resulting system state as the initial condition for the remaining simulation, with <em>X</em> reset to 0 at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e016"><math id="M16" display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math></span> without changing <em>I</em> to represent media change to unlabeled chase media. We then reduced ɑ to zero and continued the simulation for 10h, reporting <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e017"><math id="M17" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mi>I</mi><mo>,</mo><mi>X</mi><mo stretchy="false">)</mo></mrow></math></span> at 1-minute intervals. For candidate models with exponential decay of production, the production rate during the chase was instead <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e018"><math id="M18" display="inline" overflow="linebreak"><mrow><mi>α</mi><msup><mrow><mi>e</mi></mrow><mrow><mo>−</mo><mi>k</mi><mi>t</mi></mrow></msup></mrow></math></span>.</p></section><section id="sec010"><h3 class="pmc_sec_title">Simulation algorithms</h3>
<p>For ODE models, we simulated equations using ode23s in MATLAB with an absolute tolerance of 10<sup>-12</sup>, relative tolerance of 10<sup>-8</sup>, initial step size of 10<sup>-2</sup>, and all species constrained to non-negative values. For the DDE and other candidate models, we simulated equations using dde23 in MATLAB with an absolute tolerance of 10<sup>-4</sup>, relative tolerance of 10<sup>-3</sup>, initial step size of 10<sup>-2</sup>, and any negative concentrations that emerge due to numerical approximations (which are always very small) are forced to 0.</p></section><section id="sec011"><h3 class="pmc_sec_title">Experimental data for optimization</h3>
<p>We compiled three independently published datasets of the time course of sFLT1 secretion by cultured human umbilical vein endothelial cells (HUVECs) (<a href="#pcbi.1013324.s006" class="usa-link">S2 Table</a>). To distinguish the three datasets, we will refer to them by the name of the first author in each case. Two of these studies quantified constitutive sFLT1 secretion after media change in the presence of minimal serum and no added perturbation: one using ELISA measurements of absolute sFLT1 concentration in conditioned media (“Hornig” [<a href="#pcbi.1013324.ref029" class="usa-link" aria-describedby="pcbi.1013324.ref029">29</a>]) and one using quantitative Western blots of sFLT1 in both conditioned media and cell lysate (“Kinghorn” [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>]). The third study used a pulse-chase experiment to quantify newly-synthesized sFLT1 in both conditioned media and cell lysate with quantitative Western blots (“Jung” [<a href="#pcbi.1013324.ref022" class="usa-link" aria-describedby="pcbi.1013324.ref022">22</a>]). For the Jung and Hornig studies, summary data were extracted from published figures using the Figure Calibration function in ImageJ [<a href="#pcbi.1013324.ref030" class="usa-link" aria-describedby="pcbi.1013324.ref030">30</a>]. Two data points were omitted: Hornig 3h (assumed below the limit of quantitation for ELISA) and Jung 10h (trend inconsistent with all other experimental data and all simulations). The Kinghorn data were available in original tabulated form; to give all datasets comparable weight in optimization, the mean values at each time point were used in datasets where multiple measurements per time point were available (mean and median values were comparable). Extracted data are available in <a href="#pcbi.1013324.s001" class="usa-link">S1 Data</a>.</p></section><section id="sec012"><h3 class="pmc_sec_title">Initial guesses for parameter values</h3>
<p>We ran optimization on each model starting from <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e019"><math id="M19" display="inline" overflow="linebreak"><mrow><mi>n</mi><mo>=</mo><mn>100</mn></mrow></math></span> (ODE and candidate DDE models) or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e020"><math id="M20" display="inline" overflow="linebreak"><mrow><mi>n</mi><mo>=</mo><mn>1000</mn></mrow></math></span> (final DDE model) sets of initial guesses for parameter values. These initial values were randomly sampled from ranges (<a href="#pcbi.1013324.s007" class="usa-link">S3 Table</a>) based on previous rate estimates of protein synthesis, transport, and degradation processes from experimental proteomic approaches [<a href="#pcbi.1013324.ref031" class="usa-link" aria-describedby="pcbi.1013324.ref031">31</a>–<a href="#pcbi.1013324.ref033" class="usa-link" aria-describedby="pcbi.1013324.ref033">33</a>] and previous mechanistic models of VEGF receptor tyrosine kinases [<a href="#pcbi.1013324.ref026" class="usa-link" aria-describedby="pcbi.1013324.ref026">26</a>], including parameter values from our previous implementation of candidate model M2 [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>].</p></section><section id="sec013"><h3 class="pmc_sec_title">Optimization cost function</h3>
<p>For each dataset, we first converted the predicted <em>X</em> and <em>I</em> values (from the simulations) from units of #/cell to the units measured by the experimental assay. We then compared each observed data point <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e021"><math id="M21" display="inline" overflow="linebreak"><mrow><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mrow><mi>t</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><msub><mrow><mi>y</mi></mrow><mrow><mi>i</mi></mrow></msub><mo fence="true" form="postfix" stretchy="true">)</mo></mrow></mrow></math></span> to the corresponding simulated data point <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e022"><math id="M22" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><msub><mrow><mi>t</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><mi>f</mi><mo stretchy="false">(</mo><msub><mrow><mi>t</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><mtext> </mtext><mi>p</mi><mo stretchy="false">)</mo></mrow></math></span>) given the adjustable parameter values p. We calculated the cost <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e023"><math id="M23" display="inline" overflow="linebreak"><mrow><mi>C</mi></mrow></math></span> as the sum of squared relative errors between observed and simulated values:<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e024"><math id="M24" display="inline" overflow="linebreak"><mrow><mtext> </mtext><mi>C</mi><mo>=</mo><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>n</mi></mrow></msubsup><msup><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mrow><mi>v</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>−</mo><mi>f</mi><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><msub><mrow><mi>t</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>,</mo><mi>p</mi><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mrow><mn>2</mn></mrow></msup><mtext> </mtext></mrow></math></span>. All data points were equally weighted. To find the parameter set p which minimized the cost function, we performed nonlinear least squares optimization using <strong>lsqnonlin</strong> in MATLAB. We bounded the search space for each parameter as shown in <a href="#pcbi.1013324.s007" class="usa-link">S3 Table</a>.</p></section><section id="sec014"><h3 class="pmc_sec_title">Scoring of candidate models</h3>
<p>Each candidate model was assigned a quality score using the corrected Akaike Information Criterion (AIC<sub>C</sub>):</p>
<table class="disp-formula p" id="pcbi.1013324.e025"><tr>
<td class="formula"><math id="M25" display="block" overflow="linebreak"><mrow><mi>A</mi><mi>I</mi><msub><mrow><mi>C</mi></mrow><mrow><mi>C</mi></mrow></msub><mo>=</mo><mi>n</mi><mtext> </mtext><mi>l</mi><mi>o</mi><mi>g</mi><mtext> </mtext><mrow><mo fence="true" form="prefix" stretchy="true">(</mo><mfrac><mrow><mi>C</mi></mrow><mrow><mi>n</mi></mrow></mfrac><mo fence="true" form="postfix" stretchy="true">)</mo></mrow><mtext> </mtext><mo>+</mo><mn>2</mn><mi>k</mi><mo>+</mo><mfrac><mrow><mn>2</mn><mi>k</mi><mo stretchy="false">(</mo><mi>k</mi><mo>+</mo><mn>1</mn><mo stretchy="false">)</mo></mrow><mrow><mi>n</mi><mo>−</mo><mi>k</mi><mo>−</mo><mn>1</mn></mrow></mfrac></mrow></math></td>
<td class="label">(5)</td>
</tr></table>
<p>where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e026"><math id="M26" display="inline" overflow="linebreak"><mrow><mi>n</mi></mrow></math></span> is the number of experimental data points, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e027"><math id="M27" display="inline" overflow="linebreak"><mrow><mi>C</mi></mrow></math></span> is the cost of a given model calculated using the cost function described above, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e028"><math id="M28" display="inline" overflow="linebreak"><mrow><mi>k</mi></mrow></math></span> is the number of degrees of freedom (number of parameters + 1) for that model [<a href="#pcbi.1013324.ref034" class="usa-link" aria-describedby="pcbi.1013324.ref034">34</a>,<a href="#pcbi.1013324.ref035" class="usa-link" aria-describedby="pcbi.1013324.ref035">35</a>]. The first two terms represent the uncorrected AIC, and the last term is a correction factor for small sample size because <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e029"><math id="M29" display="inline" overflow="linebreak"><mrow><mfrac><mrow><mi>n</mi></mrow><mrow><mi>k</mi></mrow></mfrac><mo>&lt;</mo><mn>40</mn></mrow></math></span>. This correction factor represents the value of the variance, which is assumed constant over all experimental data points.</p></section><section id="sec015"><h3 class="pmc_sec_title">Sensitivity analysis</h3>
<p>Both local and global sensitivity analyses were performed on the DDE model starting from a baseline of the median optimized parameter set. Local univariate sensitivity of key output variables (such as sFlt1 concentrations) to key mechanistic parameters was calculated by running simulations with one parameter increased by 10%, and the relative change in the output variable compared to baseline. For global univariate sensitivity analysis, we ran simulations for a wide span of parameter values, each parameter in turn being scanned from 100x lower to 100x higher than the baseline value.</p></section><section id="sec016"><h3 class="pmc_sec_title">Simulation of chemical inhibition or genetic downregulation</h3>
<p>To represent the response to the acute addition of a chemical inhibitor of one of the model processes, we simulate constitutive secretion as defined above, with the pre-simulation running the system to steady state using the median set of optimized parameters, but when media change is applied at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e030"><math id="M30" display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math></span>, we also adjust a single process parameter (α, β, γ, δ, or τ) using <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e031"><math id="M31" display="inline" overflow="linebreak"><mrow><mtable displaystyle="true" rowspacing="3pt"><mtr><mtd columnalign="right"><msup><mi>p</mi><mrow><mi>′</mi></mrow></msup><mtext> </mtext><mo>=</mo><mtext> </mtext><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><mi>f</mi><mo stretchy="false">)</mo><mo>*</mo><mi>p</mi></mtd></mtr></mtable></mrow></math></span> where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e032"><math id="M32" display="inline" overflow="linebreak"><mrow><mi>p</mi></mrow></math></span> is the initial parameter value and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e033"><math id="M33" display="inline" overflow="linebreak"><mrow><mi>f</mi></mrow></math></span> is the inhibitor’s effect, i.e., the fractional inhibition of that process. To represent longer-term system perturbation, e.g., due to genetic mutation of a system component or by expression downregulation of a trafficking protein via RNA interference days before an experimental time course, we again simulate constitutive secretion, this time running both the pre-simulation to steady state and the experimental simulation with a single process parameter adjusted as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e034"><math id="M34" display="inline" overflow="linebreak"><mrow><mtable displaystyle="true" rowspacing="3pt"><mtr><mtd columnalign="right"><msup><mi>p</mi><mrow><mi>′</mi></mrow></msup><mtext> </mtext><mo>=</mo><mtext> </mtext><mo stretchy="false">(</mo><mn>1</mn><mo>−</mo><mi>f</mi><mo stretchy="false">)</mo><mo>*</mo><mi>p</mi></mtd></mtr></mtable></mrow></math></span>, where again <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e035"><math id="M35" display="inline" overflow="linebreak"><mrow><mi>p</mi></mrow></math></span> is the initial parameter value and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e036"><math id="M36" display="inline" overflow="linebreak"><mrow><mi>f</mi></mrow></math></span> is the magnitude of the downregulation, i.e., the fractional inhibition of that process. To compare the predicted effects of acute and chronic perturbations of trafficking mechanisms, we used experimental measurements of intracellular and extracellular sFLT1 following inhibitor treatment, expressed as a fraction of control (i.e., the absence of perturbation) [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>]. Using our model, evaluated at a range of inhibition strengths, we estimated the fractional inhibition for each inhibitor of that inhibitor’s target process, i.e., the predicted parameter reduction required for the model to produce the same relative change in sFLT1 as experimentally observed.</p></section></section><section id="sec017"><h2 class="pmc_sec_title">Results</h2>
<section id="sec018"><h3 class="pmc_sec_title">ODE model of sFLT1 secretion</h3>
<p>The ODE model (see <em>Methods</em>, <a href="#pcbi.1013324.e001" class="usa-link">Equations 1</a>–<a href="#pcbi.1013324.e002" class="usa-link">2</a>) includes rate terms for production, secretion, intracellular degradation, and extracellular degradation, described by four parameters (<a href="#pcbi.1013324.g001" class="usa-link">Fig 1A</a>). A general analysis of the ODE model solution dynamics (<a href="#pcbi.1013324.s008" class="usa-link">S1 Fig</a>) is described in more detail in <a href="#pcbi.1013324.s003" class="usa-link">S2 Text</a>. By setting the time derivatives to zero, we calculated theoretical steady state solutions:</p>
<table class="disp-formula p" id="pcbi.1013324.e037"><tr>
<td class="formula"><math id="M37" display="block" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi>α</mi></mrow><mrow><mi>β</mi><mo>+</mo><mi>γ</mi></mrow></mfrac></mrow></math></td>
<td class="label">(6)</td>
</tr></table>
<table class="disp-formula p" id="pcbi.1013324.e038"><tr>
<td class="formula"><math id="M38" display="block" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi>β</mi></mrow><mrow><mi>δ</mi></mrow></mfrac><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi>α</mi><mi>β</mi></mrow><mrow><mi>δ</mi><mo stretchy="false">(</mo><mi>β</mi><mo>+</mo><mi>γ</mi><mo stretchy="false">)</mo></mrow></mfrac></mrow></math></td>
<td class="label">(7)</td>
</tr></table>
<p>and characteristic times to half-maximal concentrations for intracellular and extracellular sFLT1 assuming first-order kinetics for secretion and degradation:</p>
<table class="disp-formula p" id="pcbi.1013324.e039"><tr>
<td class="formula"><math id="M39" display="block" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mn>50</mn><mi>_</mi><mi>I</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi>l</mi><mi>n</mi><mo stretchy="false">(</mo><mn>2</mn><mo stretchy="false">)</mo></mrow><mrow><mi>β</mi><mo>+</mo><mi>γ</mi></mrow></mfrac></mrow></math></td>
<td class="label">(8)</td>
</tr></table>
<table class="disp-formula p" id="pcbi.1013324.e040"><tr>
<td class="formula"><math id="M40" display="block" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mn>50</mn><mi>_</mi><mi>X</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi>l</mi><mi>n</mi><mo stretchy="false">(</mo><mn>2</mn><mo stretchy="false">)</mo></mrow><mrow><mi>δ</mi></mrow></mfrac></mrow></math></td>
<td class="label">(9)</td>
</tr></table>
<p>The derivations of <a href="#pcbi.1013324.e037" class="usa-link">Equations 6</a>–<a href="#pcbi.1013324.e040" class="usa-link">9</a> are given in <a href="#pcbi.1013324.s002" class="usa-link">S1 Text</a>.</p>
<p>To calibrate the model parameter values to sFLT1 biology, we used optimization algorithms to fit simulation predictions of sFLT1 concentrations simultaneously to intracellular and extracellular sFLT1 concentration datasets from three independent studies of sFLT1 secretion in human umbilical vein endothelial cells (HUVECs): one pulse-chase time course in relative units (“Jung” [<a href="#pcbi.1013324.ref022" class="usa-link" aria-describedby="pcbi.1013324.ref022">22</a>]) and two constitutive secretion time courses, one in relative units (“Kinghorn” [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>]) and one in absolute units (“Hornig” [<a href="#pcbi.1013324.ref029" class="usa-link" aria-describedby="pcbi.1013324.ref029">29</a>]) (<a href="#pcbi.1013324.g001" class="usa-link">Fig 1B</a> and <a href="#pcbi.1013324.s001" class="usa-link">S1 Data</a> and <a href="#pcbi.1013324.s006" class="usa-link">S2 Table</a>). The pulse-chase secretion experiment provided particularly useful data for characterizing early secretion behavior, while the constitutive secretion experiments allowed for estimation of biologically relevant production parameter values. The optimization appeared at first to be well-constrained, but resulted in a set of solutions with identical cost (goodness of fit) that produced identical extracellular and normalized intracellular time courses (<a href="#pcbi.1013324.g001" class="usa-link">Fig 1C</a>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e041"><math id="M41" display="inline" overflow="linebreak"><mrow><mi>n</mi><mo>=</mo><mn>100</mn></mrow></math></span> superimposed lines) and yet differed in absolute intracellular sFLT1 amount over 2 orders of magnitude (<a href="#pcbi.1013324.g001" class="usa-link">Fig 1D</a> and <a href="#pcbi.1013324.g001" class="usa-link">1E</a>). These optimized fits reproduced experimentally observed dynamics of constitutive sFLT1 secretion into culture media, save 2 outlier data points <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e042"><math id="M42" display="inline" overflow="linebreak"><mrow><msub><mrow><mo stretchy="false">(</mo><mi>X</mi></mrow><mrow><mn>72</mn><mi>h</mi></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e043"><math id="M43" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mn>24</mn><mi>h</mi></mrow></msub><mo stretchy="false">)</mo></mrow></math></span> that were inconsistent with other experimental observations. However, this initial model could not recapitulate two clearly observed features of early sFLT1 secretion dynamics in the chase phase of the pulse-chase scenario: (1) maintenance or increase of intracellular sFLT1 level for several hours after production ends and (2) a multi-hour delay in extracellular sFLT1 accumulation.</p></section><section id="sec019"><h3 class="pmc_sec_title">Delay differential equation (DDE) model of sFLT1 secretion</h3>
<p>To more accurately model the observed sFLT1 secretion dynamics, we converted our model to a system of two delay differential equations (DDEs) by introducing a fixed time delay between sFLT1 production and trafficking (<a href="#pcbi.1013324.g002" class="usa-link">Fig 2A</a>; see <em>Methods</em>, <a href="#pcbi.1013324.e003" class="usa-link">Equations 3</a>–<a href="#pcbi.1013324.e004" class="usa-link">4</a>). This “maturation delay” represents the time required for newly synthesized intracellular proteins to traverse the secretory pathway, reach a mature form, and be sorted into vesicles for trafficking. This delay is particularly important in the pulse-chase scenario because only newly synthesized (and therefore labeled) proteins are experimentally measured, and so even though unlabeled sFLT1 made before the media change might be secreted sooner, the labeled sFLT1 will only be extracellularly secreted and observed after the full delay. The model with the delay would therefore only represent the labeled sFLT1, for comparison to the experimental data. Under this scenario, the rate terms for secretion and intracellular degradation are functions of intracellular sFLT1 concentration at time <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e044"><math id="M44" display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>−</mo><mi>τ</mi></mrow></math></span>, for some time delay τ, rather than at time t as in the initial model. A general analysis of the DDE model solution dynamics (<a href="#pcbi.1013324.s009" class="usa-link">S2 Fig</a>) is described in more detail in <a href="#pcbi.1013324.s003" class="usa-link">S2 Text</a>. Because at steady state <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e045"><math id="M45" display="inline" overflow="linebreak"><mrow><mi>I</mi><mo stretchy="false">(</mo><mi>t</mi><mo>−</mo><mi>τ</mi><mo>=</mo><mtext> </mtext><mi>I</mi><mo stretchy="false">(</mo><mi>t</mi><mo stretchy="false">)</mo></mrow></math></span>, this model has the same steady state properties as the ODE model (<a href="#pcbi.1013324.e037" class="usa-link">Equations 6</a>, <a href="#pcbi.1013324.e038" class="usa-link">7</a>).</p>
<p>We optimized the DDE model parameters to the constitutive and pulse-chase secretion datasets similarly to the ODE model. The optimized DDE model generated pulse-chase dynamics that fit experimental data better than the ODE model, and fit the constitutive secretion data as effectively as the ODE model (<a href="#pcbi.1013324.g002" class="usa-link">Fig 2B</a>). However, note that in the pulse-chase experiment, intracellular sFLT1 increased above its <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e046"><math id="M46" display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math></span> baseline at 2h before decreasing, and in this DDE model formulation there is no mechanism by which intracellular sFLT1 could increase after production stops at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e047"><math id="M47" display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math></span>. This inconsistency could suggest that our model overlooks one or more processes essential to describing the biology of sFLT1 secretion, or it could be a consequence of noise in the data, in which case adding parameters to the model risks overfitting the data.</p></section><section id="sec020"><h3 class="pmc_sec_title">Testing alternate candidate models</h3>
<p>We examined whether we could further improve our model by including different combinations of three processes: the maturation delay described above; first-order internalization of extracellular sFLT1; and exponential decay of production at the end of an expression pulse (see <em>Methods</em>). We defined rate terms for each reaction for cases when each process was present or absent; for example, the internalization process had a rate of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e048"><math id="M48" display="inline" overflow="linebreak"><mrow><mi>ϵ</mi><mi>·</mi><mi>X</mi></mrow></math></span> when present but a rate of 0 when absent. Note that the absence of a process is equivalent to setting <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e049"><math id="M49" display="inline" overflow="linebreak"><mrow><mi>ε</mi><mo>=</mo><mn>0</mn></mrow></math></span> (internalization), <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e050"><math id="M50" display="inline" overflow="linebreak"><mrow><mi>κ</mi><mo>=</mo><mn>0</mn></mrow></math></span> (production decay), or <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e051"><math id="M51" display="inline" overflow="linebreak"><mrow><mi>τ</mi><mo>=</mo><mn>0</mn></mrow></math></span> (maturation). Using these terms, we built an ensemble of eight models by generating each model’s equations in a piecewise fashion (<a href="#pcbi.1013324.g003" class="usa-link">Fig 3A</a> and <a href="#pcbi.1013324.s005" class="usa-link">S1 Table</a>). Note that the model with all additional processes absent (M1) is equivalent to our original ODE model (<a href="#pcbi.1013324.g001" class="usa-link">Fig 1A</a>), and the model with maturation delay only (M2) is equivalent to the base DDE model (<a href="#pcbi.1013324.g002" class="usa-link">Fig 2A</a>). We applied our optimization algorithm to each model with 100 different sets of initial parameters to compare the ‘best fit’ version of each model.</p>
<p>When considering only the lowest cost fit for each model, we found that all models with additional processes had a slightly improved fit (lower cost, defined as the sum of squared differences between simulated and experimental data points) relative to the original ODE model M1 (<a href="#pcbi.1013324.g003" class="usa-link">Fig 3B</a>, middle). While models performed similarly in constitutive scenarios (<a href="#pcbi.1013324.s007" class="usa-link">S3B</a> and <a href="#pcbi.1013324.s007" class="usa-link">S3D Fig</a>), models that included maturation delay <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e052"><math id="M52" display="inline" overflow="linebreak"><mrow><mi>τ</mi></mrow></math></span> were better able to describe extracellular sFLT1 behavior in the pulse-chase scenario (<a href="#pcbi.1013324.s010" class="usa-link">S3A Fig</a>), while inclusion of either maturation delay <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e053"><math id="M53" display="inline" overflow="linebreak"><mrow><mi>τ</mi></mrow></math></span> or production decay <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e054"><math id="M54" display="inline" overflow="linebreak"><mrow><mi>κ</mi></mrow></math></span> improved model fit to intracellular sFLT1 during pulse-chase (<a href="#pcbi.1013324.s010" class="usa-link">S3C Fig</a>). Although every parameter addition improved goodness of fit, addition of maturation delay <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e055"><math id="M55" display="inline" overflow="linebreak"><mrow><mi>τ</mi></mrow></math></span> consistently caused the largest cost reduction (<a href="#pcbi.1013324.g003" class="usa-link">Fig 3C</a>).</p>
<p>Improvements in fit required introducing additional parameters, so to reduce the risk of overfitting, we scored the models by balancing cost and complexity (number of parameters) using the corrected Akaike Information Criterion (AIC<sub>C</sub>, <a href="#pcbi.1013324.e025" class="usa-link">Equation 5</a>) [<a href="#pcbi.1013324.ref034" class="usa-link" aria-describedby="pcbi.1013324.ref034">34</a>,<a href="#pcbi.1013324.ref035" class="usa-link" aria-describedby="pcbi.1013324.ref035">35</a>]. While adding maturation delay <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e056"><math id="M56" display="inline" overflow="linebreak"><mrow><mi>τ</mi></mrow></math></span> to any model reduced AIC<sub>C</sub>, adding production decay <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e057"><math id="M57" display="inline" overflow="linebreak"><mrow><mi>κ</mi></mrow></math></span> only reduced AIC<sub>C</sub> if maturation delay <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e058"><math id="M58" display="inline" overflow="linebreak"><mrow><mi>τ</mi></mrow></math></span> is not already included, and adding internalization <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e059"><math id="M59" display="inline" overflow="linebreak"><mrow><mi>ε</mi></mrow></math></span> never reduced AIC<sub>C</sub>. Model M2, which includes maturation delay <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e060"><math id="M60" display="inline" overflow="linebreak"><mrow><mi>τ</mi></mrow></math></span> but not internalization <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e061"><math id="M61" display="inline" overflow="linebreak"><mrow><mi>ε</mi></mrow></math></span> or production decay <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e062"><math id="M62" display="inline" overflow="linebreak"><mrow><mi>κ</mi></mrow></math></span> and was analyzed in <a href="#pcbi.1013324.g002" class="usa-link">Fig 2</a>, had the lowest AIC<sub>C</sub> and thus had the best fit to the data given its number of parameters (<a href="#pcbi.1013324.g003" class="usa-link">Fig 3B</a>: bottom). We interpret these results as showing that model M2 performs as well as the more complex models we tested based on the limited experimental data, so we therefore chose to move on to downstream analysis using this model (<a href="#pcbi.1013324.g002" class="usa-link">Fig 2A</a>).</p></section><section id="sec021"><h3 class="pmc_sec_title">Optimized parameter values</h3>
<p>To more thoroughly explore the properties of our selected DDE model, and to evaluate uncertainty in optimized parameter values for the model, we repeated the analysis with a larger set of 1000 optimization runs starting from randomly sampled initial values. While most resulting fits follow the experimental data well, numerous fits failed to converge to the experimental data (<a href="#pcbi.1013324.s011" class="usa-link">S4A Fig</a>). To enrich for high-quality fits, we filtered for optimization runs where the cost of the resulting fit was within 10% of the minimum cost (n = 594) (<a href="#pcbi.1013324.g002" class="usa-link">Figs 2B</a> and <a href="#pcbi.1013324.s011" class="usa-link">S4B</a>). Comparing the distribution of initial parameter values for all 1000 runs (<a href="#pcbi.1013324.s011" class="usa-link">S4C Fig</a>, yellow; <a href="#pcbi.1013324.s007" class="usa-link">S3 Table</a>) to the distribution of initial values for the 594 runs that passed the cost filter (<a href="#pcbi.1013324.s011" class="usa-link">S4C Fig</a>, black), we can see that the filtered-out runs generally had extreme initial parameter values, especially large initial values of β.</p>
<p>After filtering, optimized values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e063"><math id="M63" display="inline" overflow="linebreak"><mrow><mi>δ</mi></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e064"><math id="M64" display="inline" overflow="linebreak"><mrow><mi>τ</mi></mrow></math></span> were very well-constrained <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e065"><math id="M65" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mi>δ</mi><mo>=</mo><mn>0.057</mn><mi>±</mi><mn>0.0014</mn></mrow></math></span> 1/h, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e066"><math id="M66" display="inline" overflow="linebreak"><mrow><mi>τ</mi><mo>=</mo><mn>1.958</mn><mi>±</mi><mn>0.020</mn></mrow></math></span> h), while the other parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e067"><math id="M67" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mi>α</mi><mo>,</mo><mi>β</mi><mo>,</mo><mi>γ</mi><mo stretchy="false">)</mo></mrow></math></span> spanned 2–3 orders of magnitude (<a href="#pcbi.1013324.g004" class="usa-link">Figs 4A</a> and <a href="#pcbi.1013324.s012" class="usa-link">S5A</a> and <a href="#pcbi.1013324.t001" class="usa-link">Table 1</a>). Correlation plots show algebraic relationships between <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e068"><math id="M68" display="inline" overflow="linebreak"><mrow><mi>α</mi></mrow></math></span>, β, and γ, including an inverse nonlinear relationship between <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e069"><math id="M69" display="inline" overflow="linebreak"><mrow><mi>α</mi></mrow></math></span> and β, and an inverse linear relationship between β and γ (<a href="#pcbi.1013324.g004" class="usa-link">Figs 4B</a> and <a href="#pcbi.1013324.s012" class="usa-link">S5B</a>). Because of these apparent relationships, we looked for joint constraints among combinations of parameters and found that both the product <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e070"><math id="M70" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>=</mo><mi>α</mi><mi>β</mi></mrow></math></span> and the sum <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e071"><math id="M71" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>=</mo><mi>β</mi><mo>+</mo><mi>γ</mi></mrow></math></span> are well-constrained to single values (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e072"><math id="M72" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>=</mo><mn>7252</mn><mi>±</mi><mn>131</mn></mrow></math></span> #/cell/h<sup>2</sup>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e073"><math id="M73" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>=</mo><mn>0.173</mn><mi>±</mi><mn>0.00064</mn><mtext> </mtext></mrow></math></span>1/h, mean <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e074"><math id="M74" display="inline" overflow="linebreak"><mrow><mi>±</mi></mrow></math></span> SD) (<a href="#pcbi.1013324.g004" class="usa-link">Fig 4C</a> and <a href="#pcbi.1013324.t002" class="usa-link">Table 2</a>). This five-parameter model thus has four constraints <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e075"><math id="M75" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mi>δ</mi><mo>,</mo><mi>τ</mi><msub><mrow><mo>,</mo><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><msub><mrow><mo>,</mo><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub><mo stretchy="false">)</mo></mrow></math></span> and only one degree of freedom: if any one of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e076"><math id="M76" display="inline" overflow="linebreak"><mrow><mi>α</mi></mrow></math></span>, β, or γ is specified, then the values of the other two parameters are determined by the constants <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e077"><math id="M77" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e078"><math id="M78" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>. When these constraints are present, <a href="#pcbi.1013324.e037" class="usa-link">Equations 6</a> and <a href="#pcbi.1013324.e038" class="usa-link">7</a> can be extended as:</p>
<figure class="fig xbox font-sm" id="pcbi.1013324.g004"><h4 class="obj_head">Fig 4. Properties of optimized parameter sets from the DDE model of sFLT1 secretion.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370208_pcbi.1013324.g004.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dda/12370208/0893c2ec6075/pcbi.1013324.g004.jpg" loading="lazy" height="454" width="770" alt="Fig 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013324.g004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Distributions of initial (yellow) and optimized (black) model parameter values for the delay differential equation model after filtering for low-cost fits (<em>n</em> = 594). Y axes are normalized such that each distribution has a maximum density of 1. <strong>(B)</strong> Correlations between α, β, and γ values in optimized parameter sets. Each point represents the observed values of the listed parameters in a single run of the delay differential equation model after filtering for low-cost fits (<em>n</em>  = 594). <strong>(C)</strong> Distributions of initial (yellow) and optimized (black) values of calculated compound parameters c<sub><em>1</em></sub> = αβ and c<sub><em>2</em></sub> = β + γ for the delay differential equation model after filtering for low-cost fits (<em>n</em> = 594). <strong>(D-G)</strong> Correlation between intracellular steady state sFLT1 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e081"><math id="M81" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>) and <strong>(D)</strong> α/(β + γ)<strong>, (E) α, (F)</strong> β<strong>, (G)</strong> γ. Dashed lines indicate theoretical bounds (min(<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e082"><math id="M82" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>) <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e083"><math id="M83" display="inline" overflow="linebreak"><mrow><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><msubsup><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></math></span>, min(α) <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e084"><math id="M84" display="inline" overflow="linebreak"><mrow><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>, min(β) = min(γ) = 0, max(β) = max(γ) = 0; see <a href="#pcbi.1013324.s003" class="usa-link">S2 Text</a>. Bounds and highlighted trendlines are calculated using median values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e085"><math id="M85" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e086"><math id="M86" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span> (<a href="#pcbi.1013324.t001" class="usa-link">Table 1</a>).</p></figcaption></figure><section class="tw xbox font-sm" id="pcbi.1013324.t001"><h4 class="obj_head">Table 1. Statistics for the distribution of optimized values of core model parameters. n = 594 parameter sets. SD = standard deviation, CV = coefficient of variation, MAD = median absolute deviation, IQR = interquartile range.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead>
<tr>
<th align="left" rowspan="1" colspan="1">parameter</th>
<th align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e087">
<math id="M87" display="inline" overflow="linebreak"><mrow><mi mathvariant="bold-italic">α</mi></mrow></math>
</span>
</th>
<th align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e088">
<math id="M88" display="inline" overflow="linebreak"><mrow><mi mathvariant="bold-italic">β</mi></mrow></math>
</span>
</th>
<th align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e089">
<math id="M89" display="inline" overflow="linebreak"><mrow><mi mathvariant="bold-italic">γ</mi></mrow></math>
</span>
</th>
<th align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e090">
<math id="M90" display="inline" overflow="linebreak"><mrow><mi mathvariant="bold-italic">δ</mi></mrow></math>
</span>
</th>
<th align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e091">
<math id="M91" display="inline" overflow="linebreak"><mrow><mi mathvariant="bold-italic">τ</mi></mrow></math>
</span>
</th>
</tr>
<tr>
<th align="left" rowspan="1" colspan="1">unit</th>
<th align="left" rowspan="1" colspan="1">#/cell/h</th>
<th align="left" rowspan="1" colspan="1">1/h</th>
<th align="left" rowspan="1" colspan="1">1/h</th>
<th align="left" rowspan="1" colspan="1">1/h</th>
<th align="left" rowspan="1" colspan="1">h</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>mean</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e092">
<math id="M92" display="inline" overflow="linebreak"><mrow><mn>5.177</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mn>5</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e093">
<math id="M93" display="inline" overflow="linebreak"><mrow><mn>7.229</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e094">
<math id="M94" display="inline" overflow="linebreak"><mrow><mn>1.005</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e095">
<math id="M95" display="inline" overflow="linebreak"><mrow><mn>5.725</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e096">
<math id="M96" display="inline" overflow="linebreak"><mrow><mn>1.958</mn></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>SD</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e097">
<math id="M97" display="inline" overflow="linebreak"><mrow><mn>7.692</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mn>5</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e098">
<math id="M98" display="inline" overflow="linebreak"><mrow><mn>6.411</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e099">
<math id="M99" display="inline" overflow="linebreak"><mrow><mn>6.419</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e100">
<math id="M100" display="inline" overflow="linebreak"><mrow><mn>1.444</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>3</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e101">
<math id="M101" display="inline" overflow="linebreak"><mrow><mn>2.046</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>CV</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e102">
<math id="M102" display="inline" overflow="linebreak"><mrow><mn>1.487</mn></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e103">
<math id="M103" display="inline" overflow="linebreak"><mrow><mn>8.868</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e104">
<math id="M104" display="inline" overflow="linebreak"><mrow><mn>6.389</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e105">
<math id="M105" display="inline" overflow="linebreak"><mrow><mn>2.523</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e106">
<math id="M106" display="inline" overflow="linebreak"><mrow><mn>1.045</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>median</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e107">
<math id="M107" display="inline" overflow="linebreak"><mrow><mn>1.419</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mn>5</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e108">
<math id="M108" display="inline" overflow="linebreak"><mrow><mn>5.123</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e109">
<math id="M109" display="inline" overflow="linebreak"><mrow><mn>1.215</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e110">
<math id="M110" display="inline" overflow="linebreak"><mrow><mn>5.743</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e111">
<math id="M111" display="inline" overflow="linebreak"><mrow><mn>1.958</mn></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>MAD</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e112">
<math id="M112" display="inline" overflow="linebreak"><mrow><mn>1.470</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mn>5</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e113">
<math id="M113" display="inline" overflow="linebreak"><mrow><mn>6.639</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e114">
<math id="M114" display="inline" overflow="linebreak"><mrow><mn>6.649</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e115">
<math id="M115" display="inline" overflow="linebreak"><mrow><mn>2.361</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e116">
<math id="M116" display="inline" overflow="linebreak"><mrow><mn>1.570</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>IQR</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e117">
<math id="M117" display="inline" overflow="linebreak"><mrow><mn>6.310</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mn>5</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e118">
<math id="M118" display="inline" overflow="linebreak"><mrow><mn>1.337</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e119">
<math id="M119" display="inline" overflow="linebreak"><mrow><mn>1.337</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e120">
<math id="M120" display="inline" overflow="linebreak"><mrow><mn>3.149</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e121">
<math id="M121" display="inline" overflow="linebreak"><mrow><mn>2.097</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>median/IQR</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e122">
<math id="M122" display="inline" overflow="linebreak"><mrow><mn>4.447</mn></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e123">
<math id="M123" display="inline" overflow="linebreak"><mrow><mn>2.609</mn></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e124">
<math id="M124" display="inline" overflow="linebreak"><mrow><mn>1.1</mn></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e125">
<math id="M125" display="inline" overflow="linebreak"><mrow><mn>5.483</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e126">
<math id="M126" display="inline" overflow="linebreak"><mrow><mn>1.071</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>5</mn></mrow></msup></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>min</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e127">
<math id="M127" display="inline" overflow="linebreak"><mrow><mn>3.524</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mn>4</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e128">
<math id="M128" display="inline" overflow="linebreak"><mrow><mn>1.592</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>3</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e129">
<math id="M129" display="inline" overflow="linebreak"><mrow><mn>1.842</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e130">
<math id="M130" display="inline" overflow="linebreak"><mrow><mn>4.577</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e131">
<math id="M131" display="inline" overflow="linebreak"><mrow><mn>1.695</mn></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>max</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e132">
<math id="M132" display="inline" overflow="linebreak"><mrow><mn>4.563</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mn>6</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e133">
<math id="M133" display="inline" overflow="linebreak"><mrow><mn>1.745</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e134">
<math id="M134" display="inline" overflow="linebreak"><mrow><mn>1.767</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e135">
<math id="M135" display="inline" overflow="linebreak"><mrow><mn>7.296</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e136">
<math id="M136" display="inline" overflow="linebreak"><mrow><mn>2.197</mn></mrow></math>
</span>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pcbi.1013324.t001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><section class="tw xbox font-sm" id="pcbi.1013324.t002"><h4 class="obj_head">Table 2. Statistics for the distribution of compound parameters <em>c</em><sub><em>1</em></sub> and <em>c</em><sub><em>2</em></sub>. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e137"><math id="M137" display="inline" overflow="linebreak"><mrow><mrow><msub><mrow><mi mathvariant="bold-italic">c</mi></mrow><mrow><mn mathvariant="bold">1</mn></mrow></msub><mo mathvariant="bold">=</mo><mi mathvariant="bold-italic">α</mi><mi mathvariant="bold-italic">β</mi></mrow></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e138"><math id="M138" display="inline" overflow="linebreak"><mrow><mrow><msub><mrow><mi mathvariant="bold-italic">c</mi></mrow><mrow><mn mathvariant="bold">2</mn></mrow></msub><mo mathvariant="bold">=</mo><mi mathvariant="bold-italic">β</mi><mo mathvariant="bold">+</mo><mi mathvariant="bold-italic">γ</mi></mrow></mrow></math></span> result from the optimized parameter values (<a href="#pcbi.1013324.t001" class="usa-link">Table 1</a>). SD = standard deviation, CV = coefficient of variation, MAD = median absolute deviation, IQR = interquartile range.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead><tr>
<th align="left" rowspan="1" colspan="1">parameter</th>
<th align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e139">
<math id="M139" display="inline" overflow="linebreak"><mrow><mrow><msub><mrow><mi mathvariant="bold-italic">c</mi></mrow><mrow><mn mathvariant="bold">1</mn></mrow></msub><mo mathvariant="bold">=</mo><mi mathvariant="bold-italic">α</mi><mi mathvariant="bold-italic">β</mi></mrow></mrow></math>
</span>
</th>
<th align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e140">
<math id="M140" display="inline" overflow="linebreak"><mrow><mrow><msub><mrow><mi mathvariant="bold-italic">c</mi></mrow><mrow><mn mathvariant="bold">2</mn></mrow></msub><mo mathvariant="bold">=</mo><mi mathvariant="bold-italic">β</mi><mo mathvariant="bold">+</mo><mi mathvariant="bold-italic">γ</mi></mrow></mrow></math>
</span>
</th>
</tr></thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>unit</strong>
</td>
<td align="left" rowspan="1" colspan="1">#/cell/h<sup>2</sup>
</td>
<td align="left" rowspan="1" colspan="1">1/h</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>mean</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e141">
<math id="M141" display="inline" overflow="linebreak"><mrow><mn>7.252</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mn>3</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e142">
<math id="M142" display="inline" overflow="linebreak"><mrow><mn>1.728</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>SD</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e143">
<math id="M143" display="inline" overflow="linebreak"><mrow><mn>1.316</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e144">
<math id="M144" display="inline" overflow="linebreak"><mrow><mn>6.418</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>CV</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e145">
<math id="M145" display="inline" overflow="linebreak"><mrow><mn>1.814</mn><mtext> </mtext><mi>×</mi><mtext> </mtext><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e146">
<math id="M146" display="inline" overflow="linebreak"><mrow><mn>3.715</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>2</mn></mrow></msup></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>median</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e147">
<math id="M147" display="inline" overflow="linebreak"><mrow><mn>7.269</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mn>3</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e148">
<math id="M148" display="inline" overflow="linebreak"><mrow><mn>1.727</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>MAD</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e149">
<math id="M149" display="inline" overflow="linebreak"><mrow><mn>1.207</mn></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e150">
<math id="M150" display="inline" overflow="linebreak"><mrow><mn>2.302</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>7</mn></mrow></msup></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>IQR</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e151">
<math id="M151" display="inline" overflow="linebreak"><mrow><mn>1.606</mn></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e152">
<math id="M152" display="inline" overflow="linebreak"><mrow><mn>3.145</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>7</mn></mrow></msup></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>IQR/median</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e153">
<math id="M153" display="inline" overflow="linebreak"><mrow><mn>2.210</mn><msup><mrow><mtext> </mtext><mi>×</mi><mn>10</mn></mrow><mrow><mo>−</mo><mn>4</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e154">
<math id="M154" display="inline" overflow="linebreak"><mrow><mn>1.821</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>6</mn></mrow></msup></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>min</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e155">
<math id="M155" display="inline" overflow="linebreak"><mrow><mn>5.999</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mn>3</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e156">
<math id="M156" display="inline" overflow="linebreak"><mrow><mn>1.685</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math>
</span>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">
<strong>max</strong>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e157">
<math id="M157" display="inline" overflow="linebreak"><mrow><mn>8.460</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mn>3</mn></mrow></msup></mrow></math>
</span>
</td>
<td align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e158">
<math id="M158" display="inline" overflow="linebreak"><mrow><mn>1.835</mn><mi>×</mi><msup><mrow><mn>10</mn></mrow><mrow><mo>−</mo><mn>1</mn></mrow></msup></mrow></math>
</span>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pcbi.1013324.t002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><table class="disp-formula p" id="pcbi.1013324.e079"><tr>
<td class="formula"><math id="M79" display="block" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi>α</mi></mrow><mrow><mi>β</mi><mo>+</mo><mi>γ</mi></mrow></mfrac><mo>=</mo><mfrac><mrow><mi>α</mi></mrow><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></mfrac></mrow></math></td>
<td class="label">(6a)</td>
</tr></table>
<table class="disp-formula p" id="pcbi.1013324.e080"><tr>
<td class="formula"><math id="M80" display="block" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi>β</mi></mrow><mrow><mi>δ</mi></mrow></mfrac><mo>,</mo><mtext>  </mtext><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi>α</mi><mi>β</mi></mrow><mrow><mi>δ</mi><mo stretchy="false">(</mo><mi>β</mi><mo>+</mo><mi>γ</mi><mo stretchy="false">)</mo></mrow></mfrac><mo>=</mo><mfrac><mrow><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></mrow><mrow><mi>δ</mi><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></mfrac></mrow></math></td>
<td class="label">(7a)</td>
</tr></table></section><section id="sec022"><h3 class="pmc_sec_title">Steady state intracellular sFLT1</h3>
<p>During constitutive simulation, intracellular sFLT1 remains at a steady state (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e159"><math id="M159" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>) whose absolute amount varies across parameter sets within a range of 2.0 x 10<sup>5</sup> - 2.6 x 10<sup>7</sup> molecules per cell, with a median of 8.2 x 10<sup>5</sup> #/cell (<a href="#pcbi.1013324.s013" class="usa-link">S6 Fig</a>). The simulated steady state values match the theoretical definition of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e160"><math id="M160" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi>α</mi></mrow><mrow><mi>β</mi><mo>+</mo><mi>γ</mi></mrow></mfrac></mrow></math></span> across all fits (<a href="#pcbi.1013324.g004" class="usa-link">Fig 4D</a>, <a href="#pcbi.1013324.e037" class="usa-link">Equation 6</a>). Relationships between <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e161"><math id="M161" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e162"><math id="M162" display="inline" overflow="linebreak"><mrow><mi>α</mi></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e163"><math id="M163" display="inline" overflow="linebreak"><mrow><mi>β</mi></mrow></math></span>, and γ also match the theory as follows. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e164"><math id="M164" display="inline" overflow="linebreak"><mrow><mi>α</mi></mrow></math></span> is directly proportional to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e165"><math id="M165" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span> (<a href="#pcbi.1013324.g004" class="usa-link">Fig 4E</a>). <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e166"><math id="M166" display="inline" overflow="linebreak"><mrow><mi>β</mi></mrow></math></span> is inversely proportional to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e167"><math id="M167" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span> and approaches a maximum value of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e168"><math id="M168" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span> as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e169"><math id="M169" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span> approaches its minimum value (<a href="#pcbi.1013324.g004" class="usa-link">Fig 4F</a>); as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e170"><math id="M170" display="inline" overflow="linebreak"><mrow><mi>β</mi></mrow></math></span> increases, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e171"><math id="M171" display="inline" overflow="linebreak"><mrow><mi>α</mi></mrow></math></span> decreases (because <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e172"><math id="M172" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>=</mo><mi>α</mi><mi>β</mi></mrow></math></span>) and thus <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e173"><math id="M173" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span> declines. γ is inversely related to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e174"><math id="M174" display="inline" overflow="linebreak"><mrow><mi>β</mi></mrow></math></span> and approaches a maximum value of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e175"><math id="M175" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span> as <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e176"><math id="M176" display="inline" overflow="linebreak"><mrow><mi>β</mi></mrow></math></span> approaches 0; this causes <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e177"><math id="M177" display="inline" overflow="linebreak"><mrow><mi>α</mi></mrow></math></span> to increase, and so <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e178"><math id="M178" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span> approaches infinity (<a href="#pcbi.1013324.g004" class="usa-link">Fig 4G</a>). The presence of the <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e179"><math id="M179" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></math></span> constraint explains why<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e180"><math id="M180" display="inline" overflow="linebreak"><mrow><mtext> </mtext><mi>β</mi></mrow></math></span> and γ have opposite relationships to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e181"><math id="M181" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span> despite both being in the denominator of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e182"><math id="M182" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub><mo>=</mo><mfrac><mrow><mi>α</mi></mrow><mrow><mi>β</mi><mo>+</mo><mi>γ</mi></mrow></mfrac></mrow></math></span>. Of note, each intracellular steady state is associated with a unique set of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e183"><math id="M183" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mi>α</mi><mo>,</mo><mi>β</mi><mo>,</mo><mi>γ</mi><mo stretchy="false">)</mo></mrow></math></span> values, so high-confidence experimental measurements of intracellular sFLT1 would provide enough data to fully constrain our mechanistic model.</p></section><section id="sec023"><h3 class="pmc_sec_title">Process fluxes</h3>
<p>We explored the flux of each process, i.e., the rate of molecules traversing each edge of the reaction network graph (<a href="#pcbi.1013324.g005" class="usa-link">Fig 5A</a>–<a href="#pcbi.1013324.g005" class="usa-link">B</a>). During constitutive secretion, extracellular degradation flux increases hyperbolically over time, while the other three processes have constant fluxes over time (<a href="#pcbi.1013324.g005" class="usa-link">Fig 5C</a>-<a href="#pcbi.1013324.g005" class="usa-link">D</a>). The extracellular degradation flux converges towards a single time course directly proportional to the time course of extracellular sFLT1 (<a href="#pcbi.1013324.g002" class="usa-link">Fig 2B</a>). The secretion flux converges towards a single value of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e184"><math id="M184" display="inline" overflow="linebreak"><mrow><mfrac><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></mfrac><mo>=</mo><mfrac><mrow><mi>α</mi><mi>β</mi></mrow><mrow><mi>β</mi><mo>+</mo><mi>γ</mi></mrow></mfrac></mrow></math></span> across all model fits (<a href="#pcbi.1013324.g005" class="usa-link">Figs 5E</a> and <a href="#pcbi.1013324.s014" class="usa-link">S7A</a>). In contrast, production and intracellular degradation flux values differ across fits and their values are paired such that any increased production flux is offset by intracellular degradation (<a href="#pcbi.1013324.s014" class="usa-link">S7B Fig</a>), and both are algebraically related to the observed intracellular steady state concentration (<a href="#pcbi.1013324.s014" class="usa-link">S7C-D Fig</a>, <a href="#pcbi.1013324.s002" class="usa-link">S1 Text</a>). The fit-specific differences in production and intracellular degradation flux also occur during simulation of pulse-chase secretion (<a href="#pcbi.1013324.g005" class="usa-link">Fig 5F</a>-<a href="#pcbi.1013324.g005" class="usa-link">G</a>), though all models had similar relative fluxes (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e185"><math id="M185" display="inline" overflow="linebreak"><mrow><mi>Φ</mi><mo>/</mo><msub><mrow><mi>Φ</mi></mrow><mrow><mi>m</mi><mi>a</mi><mi>x</mi></mrow></msub></mrow></math></span>) over time (<a href="#pcbi.1013324.s015" class="usa-link">S8 Fig</a>).</p>
<figure class="fig xbox font-sm" id="pcbi.1013324.g005"><h4 class="obj_head">Fig 5. Analysis of simulated sFLT1 kinetic process fluxes.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370208_pcbi.1013324.g005.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dda/12370208/978dc14b4db8/pcbi.1013324.g005.jpg" loading="lazy" height="700" width="719" alt="Fig 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013324.g005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Reaction network graph showing process fluxes (Φ). <strong>(B)</strong> Equations for process fluxes. <strong>(C)</strong> Process fluxes over time during simulation of constitutive sFLT1 secretion. <strong>(D)</strong> Rates of change of extracellular sFLT1 (<em>dX</em>/<em>dt</em> = Φ(Secr) − Φ(XDeg)) and intracellular sFLT1 (<em>d</em>I/<em>dt</em>  = Φ(Prod) − Φ(Secr) − Φ(IDeg)) during constitutive secretion. <strong>(E)</strong> Constitutive flux distributions for time-constant processes. <strong>(F)</strong> Process fluxes over time during simulation of pulse-chase sFLT1 secretion. <strong>(G)</strong> Rates of change of extracellular and intracellular sFLT1 during pulse-chase secretion. In C, D, F, and G, each line represents a different optimized parameter set. Prod = production, Secr = secretion, IDeg = intracellular degradation, XDeg = extracellular degradation.</p></figcaption></figure></section><section id="sec024"><h3 class="pmc_sec_title">Local univariate sensitivity analysis</h3>
<p>We evaluated how uncertainty in each input parameter affects the uncertainty of selected output variables during constitutive secretion, including extracellular sFLT1 after 72h (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e186"><math id="M186" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mn>72</mn><mi>h</mi></mrow></msub></mrow></math></span>), intracellular sFLT1 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e187"><math id="M187" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mn>72</mn><mi>h</mi></mrow></msub></mrow></math></span>; note that for constitutive secretion, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e188"><math id="M188" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mn>72</mn><mi>h</mi></mrow></msub><mo>=</mo><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>), and time to half-steady-state for extracellular sFLT1 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e189"><math id="M189" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mn>50</mn><mi>_</mi><mi>X</mi></mrow></msub></mrow></math></span> = time to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e190"><math id="M190" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub><mo>/</mo><mn>2</mn></mrow></math></span>), where <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e191"><math id="M191" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span> is calculated theoretically. Starting from the median parameter set (<a href="#pcbi.1013324.t001" class="usa-link">Table 1</a>), we individually increased the value of each parameter by 10%, simulated constitutive secretion, and calculated the relative change in output variables (<a href="#pcbi.1013324.g006" class="usa-link">Fig 6A</a>). Intracellular and extracellular sFLT1 increase linearly with production rate constant <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e192"><math id="M192" display="inline" overflow="linebreak"><mrow><mi>α</mi></mrow></math></span> (sensitivity = 1). Increasing the secretion rate constant β causes a moderate rise in extracellular sFLT1 but a mild decrease in intracellular sFLT1, while increasing the intracellular degradation rate constant <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e193"><math id="M193" display="inline" overflow="linebreak"><mrow><mi>γ</mi></mrow></math></span> causes a decline in both intracellular and extracellular levels. The extracellular degradation rate constant<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e194"><math id="M194" display="inline" overflow="linebreak"><mrow><mtext> </mtext><mi>δ</mi></mrow></math></span> has a strong negative effect on extracellular sFLT1 but has no impact on intracellular protein. Notably, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e195"><math id="M195" display="inline" overflow="linebreak"><mrow><mi>δ</mi></mrow></math></span> is the only parameter which affects the time to half-steady-state (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e196"><math id="M196" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mn>50</mn><mi>_</mi><mi>X</mi></mrow></msub></mrow></math></span>) for extracellular sFLT1, with higher <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e197"><math id="M197" display="inline" overflow="linebreak"><mrow><mi>δ</mi></mrow></math></span> associated with a faster time to steady state.</p>
<figure class="fig xbox font-sm" id="pcbi.1013324.g006"><h4 class="obj_head">Fig 6. Local and global univariate sensitivity analysis of the sFLT1 secretion DDE model.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370208_pcbi.1013324.g006.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dda/12370208/7d96fea6c54d/pcbi.1013324.g006.jpg" loading="lazy" height="526" width="775" alt="Fig 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013324.g006/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Relative sensitivity of sFLT1 secretion output variables to local changes in input parameters during simulation of constitutive secretion. The base case is the median parameter set, and all other cells represent the observed fractional increase in the output variable per fractional increase in the parameter. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e198"><math id="M198" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mn>72</mn><mi>h</mi></mrow></msub></mrow></math></span>: extracellular sFLT1 after 72 hours; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e199"><math id="M199" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mn>72</mn><mi>h</mi></mrow></msub></mrow></math></span>: intracellular sFLT1 after 72 hours; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e200"><math id="M200" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mn>50</mn><mi>_</mi><mi>X</mi></mrow></msub></mrow></math></span>: time to half-steady-state for extracellular sFLT1. <strong>(B)</strong> Global univariate sensitivity analysis of extracellular and intracellular sFLT1 showing the effect of multiplying each input parameter by a scaling factor during constitutive secretion. The base case (scaling factor 1) is the median parameter set from the optimized parameter sets (<a href="#pcbi.1013324.t001" class="usa-link">Table 1</a>).</p></figcaption></figure></section><section id="sec025"><h3 class="pmc_sec_title">Global univariate sensitivity analysis</h3>
<p>To explore how individual parameters affect sFLT1 trafficking over a larger range of values, we varied each parameter individually over several orders of magnitude and simulated constitutive secretion (<a href="#pcbi.1013324.g006" class="usa-link">Figs 6B</a> and <a href="#pcbi.1013324.s016" class="usa-link">S9</a>). For all parameter sets tested, intracellular sFLT1 remains at its relevant steady state throughout a simulation, while extracellular sFLT1 increases hyperbolically towards a steady state. Consistent with the local sensitivity analysis, both extracellular and intracellular sFLT1 are most sensitive to changes in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e201"><math id="M201" display="inline" overflow="linebreak"><mrow><mi>α</mi></mrow></math></span>, with changes in <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e202"><math id="M202" display="inline" overflow="linebreak"><mrow><mi>α</mi></mrow></math></span> causing directly proportional changes in both steady state intracellular sFLT1 and extracellular sFLT1 72 hours after media change, across the range of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e203"><math id="M203" display="inline" overflow="linebreak"><mrow><mi>α</mi></mrow></math></span> values. β, γ, and δ have narrower ranges of effects than <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e204"><math id="M204" display="inline" overflow="linebreak"><mrow><mi>α</mi></mrow></math></span> (<a href="#pcbi.1013324.g006" class="usa-link">Figs 6B</a> and <a href="#pcbi.1013324.s016" class="usa-link">S9</a>), because below or above certain thresholds changing the parameter value has no effect (<a href="#pcbi.1013324.s016" class="usa-link">S9 Fig</a>). While β and γ affect extracellular sFLT1 in opposite directions, they influence intracellular sFLT1 in the same direction but with different magnitudes, with the response to γ being larger and over a wider range than β. δ does not affect intracellular sFLT1, but decreasing δ increases extracellular sFLT1 while slowing its rate of convergence to steady state. The constitutive secretion scenario masks any impact of changing the secretion delay τ, with altered simulation results at high τ only because long delay interferes with our method of initializing the system to steady state before media change. While the constraints <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e205"><math id="M205" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>,</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub><mo stretchy="false">)</mo></mrow></math></span> are fixed for our sFLT1 system, we also explored the sensitivity of the system to changes in these values (<a href="#pcbi.1013324.s003" class="usa-link">S2 Text</a> and <a href="#pcbi.1013324.s017" class="usa-link">S10 Fig</a>).</p></section><section id="sec026"><h3 class="pmc_sec_title">Chemical perturbation of sFLT1 secretion</h3>
<p>We explored how our model of sFLT1 secretion responds to either chemical inhibition or genetic downregulation of various processes during constitutive secretion. Chemical inhibition represents the application of a druglike inhibitor at a specific experimental time point, while genetic downregulation reflects a system where genes encoding cellular trafficking machinery are mutated or their expression is blocked by RNAi before the experimental time course begins (<a href="#pcbi.1013324.g007" class="usa-link">Fig 7A</a>). Chemical inhibition is acute, genetic downregulation more chronic; thus, to simulate chemical inhibition, we first ran the simulation to steady state with baseline parameter values, and then we applied media change <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e206"><math id="M206" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mi>X</mi><mo>→</mo><mn>0</mn><mo stretchy="false">)</mo></mrow></math></span> and inhibitor treatment (parameter value adjustment) at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e207"><math id="M207" display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math></span>, while to simulate genetic downregulation, we applied the parameter changes continually from the beginning of the simulation, before running the simulation to steady state.</p>
<figure class="fig xbox font-sm" id="pcbi.1013324.g007"><h4 class="obj_head">Fig 7. Simulated chemical inhibition and genetic downregulation of sFLT1 secretion.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370208_pcbi.1013324.g007.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dda/12370208/149ceb744310/pcbi.1013324.g007.jpg" loading="lazy" height="959" width="711" alt="Fig 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013324.g007/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Simulation timeline schematic for chemical inhibition (top) and genetic downregulation (bottom) of constitutive secretion. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e208"><math id="M208" display="inline" overflow="linebreak"><mrow><mi>p</mi><mo>=</mo><mi>p</mi><mrow><mi>′</mi></mrow></mrow></math></span> represents change of parameter values to apply inhibitor effect. <strong>(B)</strong> Effects of chemical inhibition of individual parameters by 90% on extracellular sFLT1 (<em>X</em>) and intracellular sFLT1 (<em>I</em>) at 18h and 72h, expressed as log2 fold changes relative to the control (uninhibited) case. <strong>(C)</strong> Time courses for absolute concentrations of extracellular (X) and intracellular (I) sFLT1 with 90% chemical inhibition of each parameter. The impact of inhibiting each process is how different each line is from the prediction of the median, unmodified parameter set (black line). The inset magnifies the trend for inhibition of α. Black points, experimental ELISA data for the uninhibited case [<a href="#pcbi.1013324.ref029" class="usa-link" aria-describedby="pcbi.1013324.ref029">29</a>]. <strong>(D)</strong> Annotated curves for chemical inhibition of β (red) and γ (orange) reproduced from <strong>(C)</strong>. Horizontal lines indicate experimentally observed relative changes in extracellular (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e209"><math id="M209" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mn>18</mn><mi>h</mi></mrow></msub></mrow></math></span>) and intracellular (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e210"><math id="M210" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mn>18</mn><mi>h</mi></mrow></msub></mrow></math></span>) sFLT1 18 hours after drug addition. Vertical lines indicate the fraction inhibition of the target parameter required to reproduce experimental results. <strong>(E)</strong> Effects of genetic downregulation of individual parameters by 90%, as described in <strong>(B)</strong>. <strong>(F)</strong> Time courses for absolute concentrations of extracellular (X) and intracellular (I) sFLT1 with 90% genetic downregulation of each parameter, as described in <strong>(C)</strong>. <strong>(G)</strong> Annotated curves for genetic downregulation of β (red) and γ (orange), as described in <strong>(D)</strong>.</p></figcaption></figure><p>Starting from the median parameter set (<a href="#pcbi.1013324.t001" class="usa-link">Table 1</a>), we simulated chemical inhibition of each individual parameter by 90% and investigated how inhibition affects intracellular and extracellular sFLT1 over time (<a href="#pcbi.1013324.g007" class="usa-link">Fig 7B</a>-<a href="#pcbi.1013324.g007" class="usa-link">C</a>). Inhibition of the production rate constant α causes a gradual but large decrease in intracellular sFLT1, which approaches a new lower steady state within 16h, while extracellular sFLT1 initially increases before declining to a new steady state within 48h (<a href="#pcbi.1013324.g007" class="usa-link">Fig 7C</a><strong>, inset</strong>). Inhibition of the secretion rate constant β causes a gradual small increase in intracellular sFLT1, which approaches a new higher steady state within 16h, while extracellular sFLT1 hyperbolically approaches steady state with lower concentration but similar dynamics as the uninhibited case. Inhibiting the intracellular degradation rate constant (γ) leads to a hyperbolic increase in both intracellular and extracellular sFLT1, both of which take over 48h to approach steady state. Inhibiting extracellular degradation (δ) does not alter intracellular sFLT1 but causes extracellular sFLT1 to increase continuously throughout the entire 72h simulation. Adjustment of the secretion delay time (τ) does not influence sFLT1 during constitutive secretion.</p>
<p>We then evaluated the impact of strength of inhibition on extracellular and intracellular sFLT1 at two time points: 18 hours, a time point at which inhibitors of sFLT1 secretion have been experimentally tested, 13 and 72 hours, the standard length of our constitutive simulation. Consistent with previous findings, inhibiting production (α) decreases both intracellular and extracellular sFLT1, inhibiting secretion (β) slightly increases intracellular sFLT1 but strongly decreases extracellular sFLT1, inhibiting intracellular degradation (γ) increases both intracellular and extracellular sFLT1, and inhibiting extracellular degradation (δ) does not affect intracellular sFLT1 but increases extracellular sFLT1 (<a href="#pcbi.1013324.s018" class="usa-link">S11A Fig</a>). All these effects increase in size as fraction inhibition increases, although the effect of inhibiting α increases linearly with fraction inhibition and the effects of inhibiting other parameters increase superlinearly. While the effects of inhibiting α and β are roughly equivalent at 18h and 72h, the impact of inhibiting γ and δ are larger at the later time point, consistent with only the α and β-perturbed simulations approaching their new steady state by 18h.</p>
<p>We compared our model predictions to previous experiments [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>] evaluating how chemical inhibitors of the secretory pathway affect intracellular and extracellular sFLT1 after 18h of treatment (<a href="#pcbi.1013324.g007" class="usa-link">Fig 7D</a> and <a href="#pcbi.1013324.t003" class="usa-link">Table 3</a>). Four chemical inhibitors were considered: brefeldin, a Golgi inhibitor that blocks protein maturation and secretion [<a href="#pcbi.1013324.ref036" class="usa-link" aria-describedby="pcbi.1013324.ref036">36</a>–<a href="#pcbi.1013324.ref038" class="usa-link" aria-describedby="pcbi.1013324.ref038">38</a>], described in our model as a component of the secretion rate constant β (a rate constant that combines all steps of secretion); chlorpromazine, an inhibitor of vesicular transport [<a href="#pcbi.1013324.ref039" class="usa-link" aria-describedby="pcbi.1013324.ref039">39</a>], described in our model as another component of β; TATNSF700, an inhibitor of exocytosis [<a href="#pcbi.1013324.ref040" class="usa-link" aria-describedby="pcbi.1013324.ref040">40</a>] described in our model as a component of β; and chloroquine, an inhibitor of lysosomal degradation [<a href="#pcbi.1013324.ref041" class="usa-link" aria-describedby="pcbi.1013324.ref041">41</a>,<a href="#pcbi.1013324.ref042" class="usa-link" aria-describedby="pcbi.1013324.ref042">42</a>], described in our model as the intracellular degradation rate constant γ. Based on observed experimental changes in extracellular sFLT1, our model suggests the lysosomal degradation inhibitor chloroquine inhibits the sFLT1 intracellular degradation rate constant γ by 98%, while the observed change in intracellular sFLT1 suggests a lower but still substantial 69% inhibition of γ. Similarly, based on extracellular sFLT1 experimental measurements, our model suggests the Golgi inhibitor brefeldin inhibits the sFLT1 secretion rate constant β by 89%, while the vesicular transport inhibitor chlorpromazine and the exocytosis inhibitor TATNSF700 only inhibit β by 58% and 54% respectively. However, the experimental intracellular sFLT1 concentrations observed for all 3 drugs exceeded the predicted concentration expected by our model even if secretion is fully blocked. This suggests our model may be missing an important component describing intracellular retention of sFLT1, as inhibiting secretion leads to much larger accumulation of intracellular sFLT1 than our model can explain. This could also suggest that the effect of secretion-inhibiting drugs is more complex than simply altering the secretion rate constant, possibly also reducing intracellular degradation, e.g., by relocating sFLT1 to subcellular compartments from which degradation does not occur.</p>
<section class="tw xbox font-sm" id="pcbi.1013324.t003"><h4 class="obj_head">Table 3. Predicted inhibition strengths of chemical inhibitors based on comparing experimental data [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>] to model simulations. *: Simulations match the direction of the experimental perturbation but cannot achieve the observed strength of impact.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Treatment</th>
<th align="left" rowspan="2" colspan="1">Inhibited process</th>
<th align="left" rowspan="2" colspan="1">Parameter affected</th>
<th align="left" colspan="2" rowspan="1">Percent inhibition in model to match</th>
</tr>
<tr>
<th align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e211">
<math id="M211" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mn>18</mn><mi>h</mi></mrow></msub></mrow></math>
</span>
</th>
<th align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e212">
<math id="M212" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mn>18</mn><mi>h</mi></mrow></msub></mrow></math>
</span>
</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">brefeldin</td>
<td align="left" rowspan="1" colspan="1">Golgi assembly and maturation [<a href="#pcbi.1013324.ref036" class="usa-link" aria-describedby="pcbi.1013324.ref036">36</a>–<a href="#pcbi.1013324.ref038" class="usa-link" aria-describedby="pcbi.1013324.ref038">38</a>]</td>
<td align="left" rowspan="1" colspan="1">β</td>
<td align="left" rowspan="1" colspan="1">89</td>
<td align="left" rowspan="1" colspan="1">&gt; 100*</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chlorpromazine</td>
<td align="left" rowspan="1" colspan="1">clathrin-mediated vesicle trafficking [<a href="#pcbi.1013324.ref039" class="usa-link" aria-describedby="pcbi.1013324.ref039">39</a>]</td>
<td align="left" rowspan="1" colspan="1">β</td>
<td align="left" rowspan="1" colspan="1">58</td>
<td align="left" rowspan="1" colspan="1">&gt; 100*</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">TATNSF700</td>
<td align="left" rowspan="1" colspan="1">exocytosis; vesicle fusion [<a href="#pcbi.1013324.ref040" class="usa-link" aria-describedby="pcbi.1013324.ref040">40</a>]</td>
<td align="left" rowspan="1" colspan="1">β</td>
<td align="left" rowspan="1" colspan="1">54</td>
<td align="left" rowspan="1" colspan="1">&gt; 100*</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">chloroquine</td>
<td align="left" rowspan="1" colspan="1">lysosomal degradation [<a href="#pcbi.1013324.ref041" class="usa-link" aria-describedby="pcbi.1013324.ref041">41</a>,<a href="#pcbi.1013324.ref042" class="usa-link" aria-describedby="pcbi.1013324.ref042">42</a>]</td>
<td align="left" rowspan="1" colspan="1">γ</td>
<td align="left" rowspan="1" colspan="1">98</td>
<td align="left" rowspan="1" colspan="1">69</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pcbi.1013324.t003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec027"><h3 class="pmc_sec_title">Genetic perturbation of sFLT1 secretion</h3>
<p>Genetic inhibition of individual parameters (<a href="#pcbi.1013324.t004" class="usa-link">Table 4</a>) by 90% produced similar results as chemical inhibition at later time points (<a href="#pcbi.1013324.g007" class="usa-link">Fig 7E</a>), but dynamics of early inhibition differed in many cases due to the presence of inhibition during the pre-simulation stage (<a href="#pcbi.1013324.g007" class="usa-link">Fig 7F</a>); for example, intracellular sFLT1 remains at steady state after media change at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e213"><math id="M213" display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math></span>, but this steady state level is reduced by inhibiting the production rate constant α or increased by inhibiting the secretion rate constant β or intracellular degradation rate constant γ. The steady state intracellular levels induced by 90% genetic inhibition are the same as the intracellular levels approached over time during 90% chemical inhibition, but because the genetic inhibitor is applied earlier, these steady state levels have already been reached by <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e214"><math id="M214" display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math></span>. Extracellular sFLT1 time courses for genetic and chemical inhibition are identical for inhibition of β, γ, and the extracellular degradation rate constant δ. Genetic inhibition of α causes extracellular sFLT1 to hyperbolically approach its decreased steady state level (<a href="#pcbi.1013324.g007" class="usa-link">Fig 7F</a><strong>, inset</strong>), unlike in chemical inhibition where extracellular sFLT1 initially overshoots its final level before a delayed decrease to the new steady state (<a href="#pcbi.1013324.g007" class="usa-link">Fig 7C</a><strong>, inset</strong>).</p>
<section class="tw xbox font-sm" id="pcbi.1013324.t004"><h4 class="obj_head">Table 4. Predicted inhibition strengths of genetic inhibitors based on comparing experimental data [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>] to model simulations. *: Simulations match the direction of the experimental perturbation but cannot achieve the observed strength of impact. <sup>†</sup>: Simulations cannot match experimental data as experimental perturbation has opposite outcome to simulated perturbation.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
<col align="left" valign="middle" span="1">
</colgroup>
<thead>
<tr>
<th align="left" rowspan="2" colspan="1">Treatment</th>
<th align="left" rowspan="2" colspan="1">Target role</th>
<th align="left" rowspan="2" colspan="1">Detailed role</th>
<th align="left" rowspan="2" colspan="1">Parameter affected</th>
<th align="left" colspan="2" rowspan="1">Percent inhibition in model to match</th>
</tr>
<tr>
<th align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e215">
<math id="M215" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mn>18</mn><mi>h</mi></mrow></msub></mrow></math>
</span>
</th>
<th align="left" rowspan="1" colspan="1">
<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e216">
<math id="M216" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mn>18</mn><mi>h</mi></mrow></msub></mrow></math>
</span>
</th>
</tr>
</thead>
<tbody>
<tr>
<td align="left" rowspan="1" colspan="1">siRAB27a</td>
<td align="left" rowspan="1" colspan="1">vesicle trafficking to plasma membrane</td>
<td align="left" rowspan="1" colspan="1">small GTPase coordinating vesicle transport along actin [<a href="#pcbi.1013324.ref043" class="usa-link" aria-describedby="pcbi.1013324.ref043">43</a>,<a href="#pcbi.1013324.ref044" class="usa-link" aria-describedby="pcbi.1013324.ref044">44</a>]</td>
<td align="left" rowspan="1" colspan="1">β</td>
<td align="left" rowspan="1" colspan="1">97</td>
<td align="left" rowspan="1" colspan="1">&lt; 0<sup>†</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">siSTX3</td>
<td align="left" rowspan="1" colspan="1">vesicle trafficking to plasma membrane</td>
<td align="left" rowspan="1" colspan="1">t-SNARE at plasma membrane [<a href="#pcbi.1013324.ref045" class="usa-link" aria-describedby="pcbi.1013324.ref045">45</a>]</td>
<td align="left" rowspan="1" colspan="1">β</td>
<td align="left" rowspan="1" colspan="1">82</td>
<td align="left" rowspan="1" colspan="1">&gt; 100*</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">siSTX6</td>
<td align="left" rowspan="1" colspan="1">vesicle trafficking from Golgi</td>
<td align="left" rowspan="1" colspan="1">t-SNARE in trans-Golgi network [<a href="#pcbi.1013324.ref046" class="usa-link" aria-describedby="pcbi.1013324.ref046">46</a>,<a href="#pcbi.1013324.ref047" class="usa-link" aria-describedby="pcbi.1013324.ref047">47</a>]</td>
<td align="left" rowspan="1" colspan="1">β</td>
<td align="left" rowspan="1" colspan="1">98</td>
<td align="left" rowspan="1" colspan="1">&lt; 0<sup>†</sup>
</td>
</tr>
<tr>
<td align="left" rowspan="1" colspan="1">siVAMP3</td>
<td align="left" rowspan="1" colspan="1">vesicle trafficking to plasma membrane</td>
<td align="left" rowspan="1" colspan="1">v-SNARE in endosome-derived vesicles [<a href="#pcbi.1013324.ref048" class="usa-link" aria-describedby="pcbi.1013324.ref048">48</a>]</td>
<td align="left" rowspan="1" colspan="1">β</td>
<td align="left" rowspan="1" colspan="1">78</td>
<td align="left" rowspan="1" colspan="1">&lt; 0<sup>†</sup>
</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/pcbi.1013324.t004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><p>Genetic parameter inhibition has the same directional effects on extracellular and intracellular sFLT1 at 18h and 72h as in chemical inhibition (<a href="#pcbi.1013324.s018" class="usa-link">S11B Fig</a>). The impact of inhibiting extracellular degradation (δ) is larger at the later time point, with the discrepancy increasing in size as fraction inhibition increases. For all other parameters, genetic inhibition led to the same changes in intracellular and extracellular sFLT1 at both 18h and 72h.</p>
<p>We compared our model predictions to previous experiments [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>] exploring how small interfering RNA (siRNA) inhibition of the secretory pathway affects intracellular and extracellular sFLT1 (<a href="#pcbi.1013324.g007" class="usa-link">Fig 7G</a> and <a href="#pcbi.1013324.t004" class="usa-link">Table 4</a>), assuming that the inhibition reached its maximum effectiveness by the media change at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e217"><math id="M217" display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math></span> and remained fully effective throughout the remaining experimental time. Based on observed experimental changes in extracellular sFLT1, our model suggests that inhibition of the tested Golgi and vesicular trafficking machinery (STX6, RAB27a, STX3, VAMP3) [<a href="#pcbi.1013324.ref043" class="usa-link" aria-describedby="pcbi.1013324.ref043">43</a>–<a href="#pcbi.1013324.ref048" class="usa-link" aria-describedby="pcbi.1013324.ref048">48</a>] decreases the effective secretion rate constant β by 78–98%, consistent with these factors playing a role in sFLT1 secretion. However, while our model predicts a minor increase in intracellular sFLT1 with inhibition of the secretion rate constant, genetic inhibition of STX3 leads to a much larger intracellular sFLT1 accumulation than expected by our model even if secretion is fully blocked, and genetic inhibition of the other tested factors caused a paradoxical decrease in intracellular sFLT1. These observations again suggest our model is an incomplete descriptor of intracellular sFLT1 biology, despite model agreement with other observed experimental data; for example, the targets of inhibition can affect the trafficking of many proteins, which may themselves have a role in sFLT1 trafficking. For example, STX6 also regulates trafficking of VEGFR2 to and from the cell surface [<a href="#pcbi.1013324.ref049" class="usa-link" aria-describedby="pcbi.1013324.ref049">49</a>], and VEGF-VEGFR2 signaling induces sFLT1 expression in endothelial cells [<a href="#pcbi.1013324.ref050" class="usa-link" aria-describedby="pcbi.1013324.ref050">50</a>]. Inactivating STX6 thus influences sFLT1 expression through both direct and indirect paths.</p></section><section id="sec028"><h3 class="pmc_sec_title">Evaluating uncertainty: simulating inhibition using different initial parameter sets</h3>
<p>Since there is some remaining uncertainty in parameter values, we explored how this uncertainty impacts the above insights into chemical and genetic inhibition. We generated five parameter sets consistent with the observed constraints, by setting the baseline secretion rate constant β to one of five different values (0.2*<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e218"><math id="M218" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>, 0.4*<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e219"><math id="M219" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>, 0.6*<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e220"><math id="M220" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>, 0.8*<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e221"><math id="M221" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e222"><math id="M222" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>), then calculating <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e223"><math id="M223" display="inline" overflow="linebreak"><mrow><mi>α</mi><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><mi>β</mi></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e224"><math id="M224" display="inline" overflow="linebreak"><mrow><mi>γ</mi><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>−</mo><mi>β</mi></mrow></math></span> for each case. To reiterate, these parameter sets all satisfy the underlying temporal data; so our question was: would comparing simulations for different parameter sets to the inhibitor data help us to distinguish which parameter set or sets are best? Thus, for each parameter set (<a href="#pcbi.1013324.s019" class="usa-link">S12 Fig</a>), we tested the effect of chemical or genetic inhibition by a range of inhibition fractions at both 18h and 72h (<a href="#pcbi.1013324.s020" class="usa-link">S13 Fig</a>). In chemical inhibition, the effects of inhibiting sFLT1 production (α) or extracellular degradation (δ) are independent of the initial parameter set, but the choice of parameter set affects the size (though not the direction) of the system’s response to inhibiting either sFLT1 secretion (β) or intracellular degradation (γ) (<a href="#pcbi.1013324.s020" class="usa-link">S13A Fig</a>). As β increases, intracellular sFLT1 becomes more sensitive to inhibiting β and less sensitive to inhibiting γ. Extracellular sFLT1 response to γ inhibition also depends on the initial parameter set, while the initial parameter set has a smaller impact on extracellular sFLT1 response to β inhibition and effects differ only at intermediate β inhibition fractions. These differences between parameter sets are more pronounced at 72h than at 18h. For genetic inhibition, simulated results resemble later time points for chemical inhibition simulations for the intracellular parameters α, β, and γ (<a href="#pcbi.1013324.s020" class="usa-link">S13B Fig</a>). Extracellular sFLT1 is more sensitive to increasing fraction inhibition of β or γ at 72h than at 18h, but this effect is independent of the initial parameter set, and for all other parameters the results of genetic inhibition match at these two time points.</p>
<p>We then compared our inhibition simulations to experimental results [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>] (<a href="#pcbi.1013324.g008" class="usa-link">Fig 8</a>). Based on extracellular sFLT1 data from chemical inhibitors, all parameter sets generated plausible estimates (&lt;100%) for inhibition fraction of secretion (β), with higher fraction inhibition estimated for higher initial β values (i.e., lower intracellular sFLT1 levels), and with more uncertainty in the effect size for the moderate inhibitors chlorpromazine and TATNSF700 (55–75%) than for the strong inhibitor brefeldin (88–95%) (<a href="#pcbi.1013324.g008" class="usa-link">Fig 8A</a>). In contrast, for intracellular sFLT1 following chemical inhibitors, the estimated β inhibition fraction increases as β decreases (intracellular sFLT1 increases), and a feasible fraction inhibition of β (&lt;100%) can only be estimated for parameter sets with initial β values above a certain threshold, which differs for each inhibitor (approximately, β &gt; 0.5 for TATNSF700, β &gt; 0.7 for brefeldin, and β &gt; 1.1 for chlorpromazine). When initial values of β are low, even full inhibition of β does not generate a large enough fold change in intracellular sFLT1 to match experimental results. This suggests an additional constraint on the value of the secretion rate constant β. The intracellular degradation (γ) inhibitor chloroquine shows the opposite pattern: for both intracellular and extracellular sFLT1, the estimated γ inhibition fraction increases as β increases (intracellular sFLT1 decreases), and a feasible fraction inhibition of γ (≤100%) can only be estimated at lower initial values of β (and thus high initial values of γ and intracellular sFLT1), approximately β &lt; 0.1. Collectively, these suggest that an intermediate value of β, similar to the median value of 0.072 (<a href="#pcbi.1013324.t001" class="usa-link">Table 1</a>), can provide a basis for most inhibitor observations; our earlier results showed that a constrained value of β also constrains α and I<sub>SS</sub>.</p>
<figure class="fig xbox font-sm" id="pcbi.1013324.g008"><h4 class="obj_head">Fig 8. Simulating chemical and genetic inhibition of sFLT1 secretion from different initial parameter sets.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12370208_pcbi.1013324.g008.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/6dda/12370208/c9227d99e657/pcbi.1013324.g008.jpg" loading="lazy" height="629" width="700" alt="Fig 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/pcbi.1013324.g008/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>(A)</strong> Annotated curves for chemical inhibition of β (secretion, first 3 columns) and γ (intracellular degradation, rightmost column) at 18 hours. Horizontal lines indicate experimentally observed relative changes in extracellular (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e225"><math id="M225" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mn>18</mn><mi>h</mi></mrow></msub></mrow></math></span>) and intracellular (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e226"><math id="M226" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mn>18</mn><mi>h</mi></mrow></msub></mrow></math></span>) sFLT1 18 hours after addition of brefeldin, chlorpromazine, TATNSF700, or chloroquine. Colored vertical lines mark the fraction inhibition required to match the observed change in sFLT1 when starting from the parameter set with the matching β value. <strong>(B)</strong> Annotated curves for genetic inhibition of β (secretion rate constant) at 18 hours post media change. Horizontal lines indicate experimentally observed relative changes in extracellular (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e227"><math id="M227" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mn>18</mn><mi>h</mi></mrow></msub></mrow></math></span>) and intracellular (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e228"><math id="M228" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mn>18</mn><mi>h</mi></mrow></msub></mrow></math></span>) sFLT1 after 18 hours after media change in cells transfected with siRNA targeting the listed secretion-linked protein 48 hours previously [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>]. Colored vertical lines mark the fraction inhibition required to match the observed change in sFLT1 when starting from the parameter set with the matching β value.</p></figcaption></figure><p>Comparing simulated and experimental genetic inhibitors of β (<a href="#pcbi.1013324.g008" class="usa-link">Fig 8B</a>), we find that all parameter sets except those with the highest initial β values (lowest intracellular sFLT1) led to reasonable (&lt;100%) estimates of inhibition fraction, again consistent with an intermediate β value. However, experimental downregulation of the secretion factors VAMP3, RAB27a, and STX6 decreased intracellular sFLT1, whereas in our simulations, inhibiting secretion increased intracellular sFLT1 for all inhibition fractions. Only experimental STX3 inhibition changed intracellular sFLT1 by a magnitude consistent with our simulations, and only for parameter sets with high initial values of β &gt; 0.11 (low intracellular sFLT1). Our interpretation is that these inhibitors may have mechanistic effects different to those included in our model – such as effects on the trafficking on other proteins that themselves impact sFLT1, as mentioned above – and bear further investigation.</p></section></section><section id="sec029"><h2 class="pmc_sec_title">Discussion</h2>
<p>Here, we have developed mechanistic computational models of sFLT1 secretion by endothelial cells. sFLT1 is a soluble form of the VEGFR1 protein, whose membrane-associated form is one of three receptor tyrosine kinases that bind VEGF family cytokines, a key family of ligands that induces blood vessel growth in health and disease [<a href="#pcbi.1013324.ref015" class="usa-link" aria-describedby="pcbi.1013324.ref015">15</a>]. Because sFLT1 can bind the ligand but has no tyrosine kinase domain, it does not initiate signaling and therefore acts as a sink for ligand, downregulating its signaling potential. Therefore, sFLT1 is an important negative regulator of blood vessel growth, and understanding its production, secretion, and levels outside the cell are important to understanding VEGF regulation.</p>
<p>There have been a limited number of experimental studies that quantified sFLT1 secretion over time by endothelial cells [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>,<a href="#pcbi.1013324.ref022" class="usa-link" aria-describedby="pcbi.1013324.ref022">22</a>,<a href="#pcbi.1013324.ref029" class="usa-link" aria-describedby="pcbi.1013324.ref029">29</a>,<a href="#pcbi.1013324.ref051" class="usa-link" aria-describedby="pcbi.1013324.ref051">51</a>], and here we have used the data on sFLT1 levels inside and outside the cell from several of those studies to parameterize the models. We also used data from a recent study [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>] that tested multiple chemical and genetic inhibitors of proteins known to be involved in protein trafficking and secretion, to determine which were relevant for sFLT1 secretion, and in so doing quantified the level of disruption to sFLT1 extracellular levels over time (and therefore sFLT1 secretion) due to these inhibitors. Since the inhibitors block or reduce the effectiveness of the mechanisms included in our models, we explored model perturbations equivalent to these inhibitors in the simulations also.</p>
<p>A key first step in our study was comparing different mechanistic models for sFLT1 secretion. These models differed in the incorporation or exclusion of mechanisms such as: a maturation time delay between production and availability for secretion; a gradual decay in production rate when production ceases; and the internalization of previously-secreted extracellular sFLT1. The addition of some of these time-based processes also requires shifting from a typical ODE-based model (<a href="#pcbi.1013324.g001" class="usa-link">Fig 1</a>) to a DDE (delay differential equation)-based model (<a href="#pcbi.1013324.g002" class="usa-link">Fig 2</a>). The resulting eight candidate models were tested, i.e., the parameters for each were optimized independently against the three experimental datasets described earlier and the predictions of the best-fit parameter sets for each model then compared across the eight models (<a href="#pcbi.1013324.g003" class="usa-link">Fig 3</a>). The criterion used for overall model choice was the Akaike Information Criterion (AIC), which balances goodness of fit (i.e., how closely the simulation predictions match the experimental data) with the complexity of the model (i.e., how many parameters are needed to define it, with the understanding that more parameters would require more data for determination) [<a href="#pcbi.1013324.ref035" class="usa-link" aria-describedby="pcbi.1013324.ref035">35</a>]. As a result, model M2 - a DDE-based model that included maturation delay but not internalization or production decay - was identified as the best model and was used for the remainder of our study. This does not mean that production decay and internalization do not happen in the actual cells, nor indeed that they don’t improve goodness of fit of the model when included in the simulations; rather, it means that their effect is relatively small and the improvement is not worth the additional model complexity.</p>
<p>Based on this chosen model, and comparing the results of hundreds of optimization runs attempting to fit to the experimental data, of the five mechanistic parameters (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e229"><math id="M229" display="inline" overflow="linebreak"><mrow><mi>α</mi></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e230"><math id="M230" display="inline" overflow="linebreak"><mrow><mi>β</mi></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e231"><math id="M231" display="inline" overflow="linebreak"><mrow><mi>γ</mi></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e232"><math id="M232" display="inline" overflow="linebreak"><mrow><mi>δ</mi></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e233"><math id="M233" display="inline" overflow="linebreak"><mrow><mi>τ</mi></mrow></math></span>), two of them (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e234"><math id="M234" display="inline" overflow="linebreak"><mrow><mi>δ</mi></mrow></math></span>, extracellular degradation; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e235"><math id="M235" display="inline" overflow="linebreak"><mrow><mi>τ</mi></mrow></math></span>, maturation delay) are very well constrained (<a href="#pcbi.1013324.g004" class="usa-link">Fig 4</a>). These parameters are likely well-constrained because they are integral to the shape of the concentration curves over time, something that is relatively invariant to the total amount of sFLT1 in the system; this is relevant because much of the experimental data is not in absolute concentration units, and thus the comparisons between simulation and experiment are done after both are normalized to be on the same scale. In the absence of an absolute intracellular sFLT1 measurement, the other parameters (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e236"><math id="M236" display="inline" overflow="linebreak"><mrow><mi>α</mi></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e237"><math id="M237" display="inline" overflow="linebreak"><mrow><mi>β</mi></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e238"><math id="M238" display="inline" overflow="linebreak"><mrow><mi>γ</mi></mrow></math></span>) are not fully constrained. Thus, we have a family of solutions - however, we observed that these <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e239"><math id="M239" display="inline" overflow="linebreak"><mrow><mi>α</mi></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e240"><math id="M240" display="inline" overflow="linebreak"><mrow><mi>β</mi></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e241"><math id="M241" display="inline" overflow="linebreak"><mrow><mi>γ</mi></mrow></math></span> values are not all independent, and two further constraints are apparent when comparing the optimized parameter sets: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e242"><math id="M242" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>=</mo><mi>α</mi><mi>β</mi></mrow></math></span> (i.e., production rate times secretion rate constant is a fixed value), and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e243"><math id="M243" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>=</mo><mi>β</mi><mo>+</mo><mi>γ</mi></mrow></math></span> (i.e., the sum of secretion and intracellular degradation rate constants is a fixed value). As a result, there is only one degree of freedom remaining, and setting any one of these three parameter values (or knowing the value of intracellular sFLT1 levels) determines the other parameters. Thus, even in the absence of a single completely unique parameter set, the resulting model of sFLT1 is still well defined because (a) the overall dynamics are fairly consistent due to the constraint of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e244"><math id="M244" display="inline" overflow="linebreak"><mrow><mi>δ</mi></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e245"><math id="M245" display="inline" overflow="linebreak"><mrow><mi>τ</mi></mrow></math></span>, and (b) the additional constraints on <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e246"><math id="M246" display="inline" overflow="linebreak"><mrow><mi>α</mi></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e247"><math id="M247" display="inline" overflow="linebreak"><mrow><mi>β</mi></mrow></math></span>, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e248"><math id="M248" display="inline" overflow="linebreak"><mrow><mi>γ</mi></mrow></math></span> mean that there is a unique solution for any given intracellular sFLT1 level, i.e., if it were possible to measure an absolute value for intracellular sFLT1 levels, then the system would be fully determined. Experiments that measure the dynamic responses to selective perturbations (e.g., of intracellular degradation) may help pin down the specific rate constants. We also explore the uncertainty in model predictions due to different parameter sets that are consistent with the experiments and span the range of possible parameter set values (<a href="#pcbi.1013324.g005" class="usa-link">Figs 5</a> and <a href="#pcbi.1013324.g006" class="usa-link">6</a>).</p>
<p>We use the parameterized model to simulate chemical inhibitors (e.g., small molecules that interfere with key trafficking processes) and genetic inhibitors (e.g., small interfering RNAs (siRNA) that downregulate the expression of key proteins involved in trafficking) (<a href="#pcbi.1013324.g007" class="usa-link">Fig 7</a>), in particular a subset of the inhibitors that were experimentally tested in a recent study [<a href="#pcbi.1013324.ref013" class="usa-link" aria-describedby="pcbi.1013324.ref013">13</a>]. The inhibitors are simulated in the model by the modification of the rate parameters for the process that the inhibitors are thought to interfere with; for example, if the inhibitor blocks intracellular degradation, the value of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e249"><math id="M249" display="inline" overflow="linebreak"><mrow><mi>γ</mi></mrow></math></span> is decreased. Note that there are known to be many sequential steps to secretion in cells, and the inhibitors tested included those targeting different molecules at different steps, but in our model we aggregate all secretion steps to one process. We distinguish between chemical and genetic inhibitors in our model based on when we apply the change in parameter values: to simulate chemical inhibition, we allow the system to reach steady state with baseline parameters before adjusting any parameter values, whereas we simulate genetic inhibition by adjusting the target parameter from the start of the simulation. This timing difference – longer-term genetic inhibition vs acute chemical inhibition – may give time for genetic inhibition to result in second-order effects where proteins and processes not included in the model are affected and can then themselves impact sFLT1 trafficking.</p>
<p>Overall, we see mixed results when comparing our simulations to the experimental observations. Our model is consistent with the experimental data for all tested chemical inhibitors of secretion and intracellular degradation for at least some of the possible parameter sets tested. For example, the secretion inhibitor brefeldin decreases extracellular sFLT1 and increases intracellular sFLT1, and our simulations predict the drug decreases the secretion rate constant by over 50% and likely closer to 90%, the precise value depending on which parameter set is used. However, the experimental data for several genetic inhibitors behave unexpectedly and change in the opposite direction compared to our model (<a href="#pcbi.1013324.g008" class="usa-link">Fig 8</a>). For example, we simulate inhibiting the vesicular trafficking factor VAMP3 as decreasing the secretion rate constant while leaving other parameters unchanged, which is predicted to decrease extracellular sFLT1 while increasing intracellular sFLT1; but experimental downregulation of VAMP3 instead decreases intracellular sFLT1. Furthermore, this mismatch occurs regardless of which initial conditions we choose from the family of optimized parameter sets.</p>
<p>It is not immediately clear how to interpret or resolve this discrepancy between our model and experiments for these genetic inhibitors. It is difficult to identify a single missing process that, if included in the model, would explain lower intracellular sFLT1 when secretion is blocked. Rather, as noted above it is more likely that there is some complex regulation at play for those specific inhibitors. Importantly, the chemical and genetic inhibitors explored are not specific to sFLT1, but regulate trafficking for many other proteins, some of which may themselves modulate sFLT1 synthesis and secretion [<a href="#pcbi.1013324.ref049" class="usa-link" aria-describedby="pcbi.1013324.ref049">49</a>,<a href="#pcbi.1013324.ref050" class="usa-link" aria-describedby="pcbi.1013324.ref050">50</a>]. Furthermore, nonspecific trafficking inhibitors like brefeldin can cause massive accumulation of immature protein in the ER and Golgi, and the resulting cell stress can lead to the unfolded protein response (UPR), a complex homeostatic response with multifactorial effects on the secretory pathway [<a href="#pcbi.1013324.ref052" class="usa-link" aria-describedby="pcbi.1013324.ref052">52</a>]. Among its many effects, the UPR increases protein turnover, and while global protein synthesis slows during the UPR, there is increased transcription and translation of secretory pathway components affecting protein maturation and trafficking [<a href="#pcbi.1013324.ref053" class="usa-link" aria-describedby="pcbi.1013324.ref053">53</a>]. As such, any inhibitor that triggers the UPR could have second-order effects altering the rate constants for sFLT1 production, trafficking, and degradation. Cases like this, where the experimental perturbations affect multiple model parameters and invoke dynamic feedback regulation, would be difficult to parse in this model. Focusing on acute inhibition or measuring changes as soon as feasible after inhibition may better serve to avoid second-order effects. What the model does do, however, is identify precisely those inhibitors whose observed behavior/outcome does not match what would be expected to happen, and therefore provides a useful shortlist of inhibitors whose more detailed mechanisms and effects would be interesting to follow up on.</p>
<p>Returning again to the remaining uncertainty in the model parameter values, we explored whether the inhibitor experiments could help to pin down the model parameters (<a href="#pcbi.1013324.g007" class="usa-link">Fig 7</a>). Each drug has a different set of parameter ranges for which modeling inhibition is well-behaved, however these ranges are not all consistent; for example, some drugs seem to support a high-β-value parameter set, others a low-β-value parameter set. An intermediate β-value is the most broadly consistent with experimental data, but further model development, perhaps including more detailed inhibitor mechanisms, may better constrain our parameter estimates.</p>
<p>Our results provide mechanistic insights about sFLT1 secretion in vitro and suggest intriguing questions about intracellular behavior of sFLT1. It remains to be seen whether these insights would also apply in vivo; if so, it suggests that physiological or pathological changes in sFLT1 concentration could have multiple causes. For example, elevated serum sFLT1 (analogous to extracellular sFLT1 in our model) could result from an increase in sFLT1 production, or increased secretion, or decreased intracellular or extracellular degradation, or any combination of these changes. Different disorders are likely driven by distinct changes in sFLT1 kinetics, with implications for treatment approaches. In vivo conditions are further complicated by multi-cellular expression and secretion of sFLT1; for example, in preeclampsia, non-endothelial cells in the placenta overexpress sFLT1 mRNA [<a href="#pcbi.1013324.ref021" class="usa-link" aria-describedby="pcbi.1013324.ref021">21</a>]. These models could be extended to understand the effects of such multicellular environments.</p>
<p>Interestingly, our model suggests that less than 20% of mature intracellular sFLT1 is removed from the intracellular compartment each hour under basal conditions, consistent with a previous estimate [<a href="#pcbi.1013324.ref022" class="usa-link" aria-describedby="pcbi.1013324.ref022">22</a>] and corresponding to a long intracellular residence time for sFLT1. This extended residence time raises the question of whether sFLT1 has functional VEGF-binding and VEGFR-binding roles within the secretory pathway. Interactions can happen because the full-length receptors accumulate in the same intracellular locations. The full-length mFLT1/VEGFR1 isoform localizes primarily to the Golgi and traffics to the plasma membrane in a calcium-dependent manner [<a href="#pcbi.1013324.ref008" class="usa-link" aria-describedby="pcbi.1013324.ref008">8</a>], while VEGFR2 is largely retained in the trans-Golgi and traffics to the cell surface in response to VEGF stimulation [<a href="#pcbi.1013324.ref049" class="usa-link" aria-describedby="pcbi.1013324.ref049">49</a>]. RTKs are also known to initiate signals from intracellular organelles [<a href="#pcbi.1013324.ref007" class="usa-link" aria-describedby="pcbi.1013324.ref007">7</a>–<a href="#pcbi.1013324.ref010" class="usa-link" aria-describedby="pcbi.1013324.ref010">10</a>] as well as from their more canonically considered cell surface locations, and this may also occur within the Golgi and be inhibited by Golgi-resident sFLT1.</p>
<p>Our approach identifies a minimal mechanistic model consistent with experimental data on sFLT1 secretion, but includes many assumptions that simplify the underlying biology, both for general features of the secretory pathway and for specific characteristics of sFLT1. The key assumptions involve homogeneity and the choice of model compartments, species, and reactions, each described in the following paragraphs. This mass action kinetics modeling approach uses several assumptions of homogeneity, including that all sFLT1 molecules in a compartment are identical and equally likely to react, all cells are identical with the same rate constants, and that system geometry is constant over space and time. However, biological systems are highly heterogeneous. Tissues consist of numerous cell types of different lineages that vary significantly in their gene expression and phenotypic profile, typically arranged into higher-order three-dimensional structures that are not spatially uniform. Individual cell types occupy various states and can change states over time. Even in mammalian cell culture of genetically identical cell lines, cell-intrinsic and cell-extrinsic stochastic processes lead to phenotypic differences across the cell population [<a href="#pcbi.1013324.ref054" class="usa-link" aria-describedby="pcbi.1013324.ref054">54</a>,<a href="#pcbi.1013324.ref055" class="usa-link" aria-describedby="pcbi.1013324.ref055">55</a>].</p>
<p>Furthermore, here we assume a constant number of cells over time when simulating or interpreting experimental results, while the cell culture environment allows cell growth and proliferation. Since HUVECs proliferate slowly in culture [<a href="#pcbi.1013324.ref056" class="usa-link" aria-describedby="pcbi.1013324.ref056">56</a>], this is likely not a major confounder at early time points, but cell number may have increased enough by our 72 hour time point to impact data interpretation. A hierarchical model that includes both sFLT1 trafficking and cell population dynamics may add insight and reveal how changing cell number affects interpretation of the experimental data.</p>
<p>An agent-based modeling (ABM) approach could incorporate cells of different types with distinct phenotypes and reaction propensities, simulate stochasticity, represent cell proliferation, and track spatial heterogeneity with partial differential equations. However, ABMs tend to be computationally expensive, we may not have sufficient data for parameterization, and their complexity may not be necessary to understand key features of sFLT1 secretion in cell culture [<a href="#pcbi.1013324.ref057" class="usa-link" aria-describedby="pcbi.1013324.ref057">57</a>]. An intermediate approach that allows modeling of heterogeneous cell communities without the full computational demand or spatial detail would be to use ODEs to model several separate intracellular compartments (corresponding to cell subtypes) linked to a common extracellular compartment. However, since sFLT1 expression varies not only across cell types but across adjacent endothelial cells [<a href="#pcbi.1013324.ref020" class="usa-link" aria-describedby="pcbi.1013324.ref020">20</a>], it may be more appropriate to use a spatially-aware modeling approach like an ABM to explore heterogeneous cell populations. While ABMs have been extensively applied to the biology of the VEGF family [<a href="#pcbi.1013324.ref058" class="usa-link" aria-describedby="pcbi.1013324.ref058">58</a>–<a href="#pcbi.1013324.ref062" class="usa-link" aria-describedby="pcbi.1013324.ref062">62</a>], some including extracellular sFLT1 [<a href="#pcbi.1013324.ref063" class="usa-link" aria-describedby="pcbi.1013324.ref063">63</a>,<a href="#pcbi.1013324.ref064" class="usa-link" aria-describedby="pcbi.1013324.ref064">64</a>], to our knowledge they have not been used to explore intracellular sFLT1 behavior and secretion.</p>
<p>In our model, all locations for sFLT1 are aggregated into only two compartments for sFLT1: intracellular and extracellular. In reality, both consist of several subcompartments where molecules can have different levels and be exposed to different conditions and therefore have different reaction propensities. For example, the intracellular space represents the entire endomembrane system of organelles (such as ER, Golgi, vesicles, and endosomes). While traversing these organelles, intracellular sFLT1 undergoes a series of maturation and trafficking steps that make a newly synthesized molecule more likely to be secreted over time, meaning reaction propensities are not constant. Although our final model addresses this maturation problem by including a maturation delay (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e250"><math id="M250" display="inline" overflow="linebreak"><mrow><mi>τ</mi></mrow></math></span>) in between synthesis and secretion or degradation, protein maturation is likely more variable and stochastic than can be represented with a fixed time delay. sFLT1 could also localize to noncanonical compartments; both VEGFR1 and VEGFR2 have been shown to localize to the nucleus under certain conditions in endothelial and microvascular cells [<a href="#pcbi.1013324.ref065" class="usa-link" aria-describedby="pcbi.1013324.ref065">65</a>]. Outside the cell, since sFLT1 binds HSPGs in the extracellular matrix (ECM) or on the cell surface, and also dimerizes with membrane-integral VEGFR isoforms, extracellular sFLT1 can also be subdivided into cell-surface-associated, ECM-associated, and freely soluble forms. Explicitly representing these subcompartments or maturation states would add new molecular species and reactions, increasing model complexity and increasing the range of possible behaviors in our model. However, they would also have significantly more parameters, and estimating those parameter values would require quantitative data about sFLT1 organelle distribution or post-translational modification. As a separate but related issue, our optimization to experimental data assumes that all extracellular sFLT1 is in conditioned media and that protein detected in cell lysates is intracellular. However, extracellular sFLT1 bound to ECM may not be aspirated with conditioned media, and extracellular sFLT1 associated with the cell surface (either dimerized to membrane VEGFR or nonspecifically bound to HSPGs) could be isolated with the lysate.</p>
<p>Our current model assumes that only sFLT1 affects the system, that we can omit other molecules and reactions from our model, and that the processes we do model are well-described by zeroth- and first-order kinetics with fixed rate constants. This framework does not address all the complexity of cell biology; for example, factors affecting global gene expression, such as ribosomal subunit expression and nutrient availability, could dynamically alter the sFLT1 production rate. Similarly, other secretory pathway cargo interacts with shared trafficking machinery, so high production of other secretory proteins could reduce the trafficking factors available to sFLT1 and reduce the effective secretion rate constant β.</p>
<p>sFLT1 binds other intracellular and extracellular components, which not only mediate its effects on VEGF signaling but also can affect the dynamics we depict on our model, as bound sFLT1 may not be available for trafficking reactions. sFLT1 acts as a ligand trap for extracellular VEGF and PlGF, and also potentially as a dimerization partner for membrane-integral VEGF receptor isoforms that creates dominant negative receptor complexes [<a href="#pcbi.1013324.ref012" class="usa-link" aria-describedby="pcbi.1013324.ref012">12</a>]. These networks of biochemical reactions may have emergent properties inconsistent with our assumption of invariant rate constants. For example, VEGF-A upregulates sFLT1 expression, which then sequesters VEGF and reduces VEGF signaling [<a href="#pcbi.1013324.ref050" class="usa-link" aria-describedby="pcbi.1013324.ref050">50</a>,<a href="#pcbi.1013324.ref051" class="usa-link" aria-describedby="pcbi.1013324.ref051">51</a>]. This negative feedback loop could generate pulses of sFLT1 expression or oscillations [<a href="#pcbi.1013324.ref066" class="usa-link" aria-describedby="pcbi.1013324.ref066">66</a>], either of which would change the sFLT1 production rate over time. Similar feedback effects could also affect proteins involved in secretion or degradation. Alternatively, some evidence suggests sFLT1 can be generated by proteolytic cleavage of membrane-integral VEGFR1 [<a href="#pcbi.1013324.ref067" class="usa-link" aria-describedby="pcbi.1013324.ref067">67</a>], a potential second production route for sFLT1, and a model incorporating that reaction would also need to track membrane isoforms and potentially be expanded to the level of gene expression and FLT1/VEGFR1 RNA alternative splicing. sFLT1 also contains heparin-binding domains that nonspecifically bind glycoproteins on the ECM and cell surface [<a href="#pcbi.1013324.ref068" class="usa-link" aria-describedby="pcbi.1013324.ref068">68</a>–<a href="#pcbi.1013324.ref070" class="usa-link" aria-describedby="pcbi.1013324.ref070">70</a>] and accumulates in the ECM [<a href="#pcbi.1013324.ref071" class="usa-link" aria-describedby="pcbi.1013324.ref071">71</a>]. sFLT1-ECM binding is important for establishing and maintaining extracellular VEGF gradients that are directional cues in angiogenesis. Importantly, protein glycosylation occurs in intracellular secretory pathway organelles, especially the Golgi [<a href="#pcbi.1013324.ref072" class="usa-link" aria-describedby="pcbi.1013324.ref072">72</a>], and interaction with intracellular glycoproteins is a potential mechanism for sFLT1 retention in the Golgi. Adding such extensive additional detail to the model would require substantial amounts of additional specific experimental data, and would likely not impact on the core observed secretion dynamics as reproduced by the current model.</p>
<p>These caveats and considerations noted above can provide further directions for development of the model of sFLT1 secretion developed here. Even without these additions, the model is already well-behaved, recreates most experimental observations of sFLT1 secretion, and identifies two classes of inhibitors or perturbations: those whose behavior is well represented by the model, and those whose behavior seems contrary to the model and worthy of further investigation.</p></section><section id="sec030"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="pcbi.1013324.s001"><div class="caption p">
<span>S1 Data. Experimental data extracted from past studies.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s001.xlsx</a><sup> (14.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s002"><div class="caption p">
<span>S1 Text. Supplemental methods.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s002.pdf</a><sup> (102.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s003"><div class="caption p">
<span>S2 Text. Supplemental results.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s003.pdf</a><sup> (76.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s004"><div class="caption p">
<span>S3 Text. Supplemental references.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s004.pdf</a><sup> (63.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s005"><div class="caption p">
<span>S1 Table. Complete set of equations for candidate models.</span><p>Note that model M1 is equivalent to the “ODE-based model” and model M2 is equivalent to the “DDE-based model” described above.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s005.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s005.pdf</a><sup> (60.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s006"><div class="caption p">
<span>S2 Table. Characteristics of sFLT1 datasets used for mechanistic model optimization.</span><p>Secretion scenario: C = constitutive, P = pulse-chase; Assay: E = ELISA, W = Western blot/ autoradiography; Time points: <sup>*</sup> = flattened by normalization, <sup>†</sup> = omitted (see Methods for rationale).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s006.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s006.pdf</a><sup> (99KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s007"><div class="caption p">
<span>S3 Table. Sampling distributions for initial parameter values and bounds for final parameter values during mechanistic model optimization.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s007.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s007.pdf</a><sup> (54.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s008"><div class="caption p">
<span>S1 Fig. Dynamics and phase plane behavior of an example ODE model solution.</span><p>(A) Solution of the ODE model as a function of time showing intracellular (<em>I</em>) and extracellular (<em>X</em>) sFLT1 protein dynamics in response to step changes in sFLT1 production (<em>α</em>). Time is normalized to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e251"><math id="M251" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mn>50</mn><mi>_</mi><mi>X</mi></mrow></msub></mrow></math></span>, the half-life of <em>X</em> (<a href="#pcbi.1013324.e004" class="usa-link">Eq. 4</a>). <em>I, X</em> are normalized to their steady state values <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e252"><math id="M252" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub><mo>,</mo><mtext> </mtext><msub><mrow><mi>X</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span> (<a href="#pcbi.1013324.e001" class="usa-link">Eq. 1</a>–<a href="#pcbi.1013324.e002" class="usa-link">2</a>). (B) Phase portrait of the shown ODE model solution. Points are evenly spaced every time unit and colored by time as in panel A. Arrowheads indicate the direction of increasing time starting with <em>t = 0</em> in the lower left. The dashed identity line represents potential steady states (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e253"><math id="M253" display="inline" overflow="linebreak"><mrow><mi>I</mi><mo>=</mo><mfrac><mrow><mi>β</mi></mrow><mrow><mi>δ</mi></mrow></mfrac><mi>·</mi><mi>X</mi></mrow></math></span>); total sFLT1 increases in regions above this line and decreases below. Parameter values: α = 10,000, β = 0.1, γ = 0.1, δ = 0.1. For discussion of this figure, see <a href="#pcbi.1013324.s003" class="usa-link">S2 Text</a>.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s008.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s008.pdf</a><sup> (307.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s009"><div class="caption p">
<span>S2 Fig. Dynamics and phase plane behavior of an example DDE model solution.</span><p>(A) Solution of the DDE model as a function of time showing intracellular (<em>I</em>) and extracellular (<em>X</em>) sFLT1 protein dynamics in response to step changes in sFLT1 production (α). Time is normalized to <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e254"><math id="M254" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mn>50</mn><mi>_</mi><mi>X</mi></mrow></msub></mrow></math></span>, the half-life of <em>X</em> (<a href="#pcbi.1013324.e004" class="usa-link">Eq. 4</a>). <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e255"><math id="M255" display="inline" overflow="linebreak"><mrow><mi>I</mi><mo>,</mo><mi>X</mi></mrow></math></span> are normalized to their steady state values <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e256"><math id="M256" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub><mo>,</mo><mtext> </mtext><msub><mrow><mi>X</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span> (<a href="#pcbi.1013324.e001" class="usa-link">Eq. 1</a>–<a href="#pcbi.1013324.e002" class="usa-link">2</a>). (B) Phase portrait of the shown DDE model solution. Points are evenly spaced every two time units and colored by time as in panel A. Arrowheads indicate the direction of increasing time starting with t = 0 in the lower left. The dashed identity line represents potential steady states <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e257"><math id="M257" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mi>I</mi><mo>=</mo><mfrac><mrow><mi>β</mi></mrow><mrow><mi>δ</mi></mrow></mfrac><mi>·</mi><mi>X</mi><mo stretchy="false">)</mo></mrow></math></span>; total sFLT1 increases in regions above this line and decreases below. Parameter values: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e258"><math id="M258" display="inline" overflow="linebreak"><mrow><mi>α</mi><mo>=</mo><mn>10</mn><mo>,</mo><mn>000</mn></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e259"><math id="M259" display="inline" overflow="linebreak"><mrow><mi>β</mi><mo>=</mo><mn>0.1</mn></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e260"><math id="M260" display="inline" overflow="linebreak"><mrow><mi>γ</mi><mo>=</mo><mn>0.1</mn></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e261"><math id="M261" display="inline" overflow="linebreak"><mrow><mi>δ</mi><mo>=</mo><mn>0.1</mn></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e262"><math id="M262" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>τ</mi><mo>=</mo><mi>T</mi></mrow><mrow><mn>50</mn><mi>_</mi><mi>X</mi></mrow></msub></mrow></math></span>. (C) Example DDE solutions as a function of time and (D) as a phase portrait for maturation delays <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e263"><math id="M263" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>τ</mi><mo>=</mo><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><mtext> </mtext><mn>0.5</mn><mo>,</mo><mtext> </mtext><mn>1</mn><mo>,</mo><mtext> </mtext><mn>1.5</mn><mo>,</mo><mtext> </mtext><mn>2</mn><mo stretchy="false">)</mo><mi>·</mi><mi>T</mi></mrow><mrow><mn>50</mn><mi>_</mi><mi>X</mi></mrow></msub></mrow></math></span>. For discussion of this figure, see <a href="#pcbi.1013324.s003" class="usa-link">S2 Text</a>.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s009.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s009.pdf</a><sup> (669.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s010"><div class="caption p">
<span>S3 Fig. Visual predictive checks of candidate models of sFLT1 secretion.</span><p>Comparison of experimental data (points) and simulated time courses (lines, <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e264"><math id="M264" display="inline" overflow="linebreak"><mrow><mi>n</mi><mo>=</mo><mn>100</mn></mrow></math></span> per plot) of (A, C) extracellular and (B, D) intracellular sFLT1 for (A, B) pulse-chase secretion and (C, D) constitutive secretion cases using eight distinct candidate models as described in Fig 3A and <a href="#pcbi.1013324.s005" class="usa-link">S1 Table</a>. In pulse-chase plots, the highlighted region (P) marks the 20-minute pulse. <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e265"><math id="M265" display="inline" overflow="linebreak"><mrow><mi>X</mi><mo>/</mo><msub><mrow><mi>X</mi></mrow><mrow><mn>8</mn><mi>h</mi></mrow></msub></mrow></math></span>: extracellular sFLT1 normalized to its value at 8h; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e266"><math id="M266" display="inline" overflow="linebreak"><mrow><mi>X</mi><mo>/</mo><msub><mrow><mi>X</mi></mrow><mrow><mn>24</mn><mi>h</mi></mrow></msub></mrow></math></span>: extracellular sFLT1 normalized to its value at 24h; <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e267"><math id="M267" display="inline" overflow="linebreak"><mrow><mi>I</mi><mo>/</mo><msub><mrow><mi>I</mi></mrow><mrow><mn>0</mn><mi>h</mi></mrow></msub></mrow></math></span>: intracellular sFLT1 normalized to its value at <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e268"><math id="M268" display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math></span>.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s010.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s010.pdf</a><sup> (536.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s011"><div class="caption p">
<span>S4 Fig. Filtering DDE optimization runs by cost.</span><p>(A) Visual predictive checks of unfiltered optimization runs (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e269"><math id="M269" display="inline" overflow="linebreak"><mrow><mi>n</mi><mo>=</mo><mn>1000</mn></mrow></math></span>) from the delay differential equation (DDE) model. Many runs with high cost have not converged to follow the experimental data. (B) Cost distribution of raw (left) and filtered (right) optimization runs of the DDE model. The red lines indicate the cost cutoff of 10% above the minimum cost. (C) Density plots of raw and filtered initial parameter values for the DDE model. Raw (yellow) distributions include all attempted optimizations <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e270"><math id="M270" display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mi>n</mi><mo>=</mo><mn>1000</mn><mo stretchy="false">)</mo></mrow></math></span>. Filtered (black) distributions include only fits with a cost within 10% of the best fit (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e271"><math id="M271" display="inline" overflow="linebreak"><mrow><mi>n</mi><mo>=</mo><mn>594</mn></mrow></math></span>). Y axes are normalized such that each distribution has a maximum density of 1.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s011.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s011.pdf</a><sup> (644.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s012"><div class="caption p">
<span>S5 Fig. Properties of optimized parameters from the DDE model of sFLT1 secretion.</span><p>(A) Violin plots of optimal parameter values (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e272"><math id="M272" display="inline" overflow="linebreak"><mrow><mi>n</mi><mo>=</mo><mn>594</mn></mrow></math></span>) for the delay differential equation (DDE) model. Units: α, #/cell/h; (β,γ,δ,ε), h<sup>-1</sup>. (B) Correlations between δ and τ and other optimized parameters. Each point represents the observed values of the listed parameters in a single run of the delay differential equation model after filtering for low-cost fits (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e273"><math id="M273" display="inline" overflow="linebreak"><mrow><mi>n</mi><mo>=</mo><mn>594</mn></mrow></math></span>).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s012.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s012.pdf</a><sup> (444.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s013"><div class="caption p">
<span>S6 Fig. Simulated absolute numbers of intracellular sFLT1 molecules.</span><p>(A) Time courses of absolute intracellular sFLT1 (<em>I</em>) during simulation of constitutive secretion. (B) Distribution of steady state values of intracellular sFLT1 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e274"><math id="M274" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>). The dashed red line indicates the theoretical lower bound min(<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e275"><math id="M275" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>) <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e276"><math id="M276" display="inline" overflow="linebreak"><mrow><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><msubsup><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></math></span> based on median values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e277"><math id="M277" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e278"><math id="M278" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span> (Table 2).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s013.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s013.pdf</a><sup> (123.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s014"><div class="caption p">
<span>S7 Fig. Steady state properties of process fluxes during simulation of constitutive secretion.</span><p>All annotation lines are derived in <a href="#pcbi.1013324.s002" class="usa-link">S1 Text</a> and calculated using median values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e279"><math id="M279" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e280"><math id="M280" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>. (A) Correlation between secretion flux (Φ(Secr)) and the compound parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e281"><math id="M281" display="inline" overflow="linebreak"><mrow><mi>α</mi><mi>β</mi><mo>/</mo><mo stretchy="false">(</mo><mi>β</mi><mo>+</mo><mi>γ</mi><mo stretchy="false">)</mo><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>. Both dashed lines indicate <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e282"><math id="M282" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>. (B) Correlation between intracellular degradation flux (Φ(IDeg)) and production flux (Φ(Prod)). Dashed lines indicate the theoretical lower bound min(Prod) <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e283"><math id="M283" display="inline" overflow="linebreak"><mrow><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span> and the asymptote Φ(IDeg) <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e284"><math id="M284" display="inline" overflow="linebreak"><mrow><mo>=</mo></mrow></math></span> Φ(Prod). (C) Correlation between steady state intracellular sFLT1 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e285"><math id="M285" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>) and intracellular degradation flux (Φ(IDeg)). Dashed lines indicate the theoretical lower bound min(<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e286"><math id="M286" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>) <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e287"><math id="M287" display="inline" overflow="linebreak"><mrow><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><msubsup><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></math></span> and the asymptote <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e288"><math id="M288" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub><mo>=</mo></mrow></math></span> Φ(IDeg)/<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e289"><math id="M289" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>. (D) Correlation between steady state intracellular sFLT1 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e290"><math id="M290" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>) and production flux (Φ(Prod)) during constitutive secretion. Dashed lines indicate theoretical lower bounds (min(<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e291"><math id="M291" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>) <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e292"><math id="M292" display="inline" overflow="linebreak"><mrow><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><msubsup><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></math></span>) and min(Φ(Prod)) <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e293"><math id="M293" display="inline" overflow="linebreak"><mrow><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>). Φ = Flux, Prod = production, Secr = secretion, IDeg = intracellular degradation, XDeg = extracellular degradation.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s014.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s014.pdf</a><sup> (426.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s015"><div class="caption p">
<span>S8 Fig. Relative process fluxes over time during simulations of pulse-chase secretion.</span><p>All species are normalized to their maximum observed values over 10 hours. Φ = flux, Prod = production, Secr = secretion, IDeg = intracellular degradation, XDeg = extracellular degradation.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s015.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s015.pdf</a><sup> (350.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s016"><div class="caption p">
<span>S9 Fig. Impact of individual parameter variations on sFLT1 for simulations of constitutive secretion.</span><p>Top: extracellular sFLT1 at 72 hours (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e294"><math id="M294" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mn>72</mn><mi>h</mi></mrow></msub></mrow></math></span>); bottom: steady state intracellular sFLT1 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e295"><math id="M295" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s016.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s016.pdf</a><sup> (154.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s017"><div class="caption p">
<span>S10 Fig. Sensitivity of extracellular and intracellular sFLT1 to the compound parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e296"><math id="M296" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>=</mo><mi>α</mi><mi>β</mi></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e297"><math id="M297" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>=</mo><mi>β</mi><mo>+</mo><mi>γ</mi></mrow></math></span> in the DDE model.</span><p>(A) Time courses of extracellular (<em>X</em>) and intracellular (<em>I</em>) sFLT1 during constitutive secretion with fixed <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e298"><math id="M298" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>=</mo><mn>7270</mn></mrow></math></span> #/cell/h [<a href="#pcbi.1013324.ref002" class="usa-link" aria-describedby="pcbi.1013324.ref002">2</a>] and varying <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e299"><math id="M299" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>. (B) Correlation between extracellular sFLT1 at 72 hours (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e300"><math id="M300" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mn>72</mn><mi>h</mi></mrow></msub></mrow></math></span>) or steady state intracellular sFLT1 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e301"><math id="M301" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>) with β, γ, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e302"><math id="M302" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span> when <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e303"><math id="M303" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></math></span> is constant during constitutive secretion. (C) Time courses of extracellular (X) and intracellular (I) sFLT1 during constitutive secretion with fixed <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e304"><math id="M304" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>=</mo><mn>0.173</mn></mrow></math></span> h<sup>-1</sup> and varying <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e305"><math id="M305" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></math></span>. (D) Correlation between extracellular sFLT1 at 72 hours (<span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e306"><math id="M306" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mn>72</mn><mi>h</mi></mrow></msub></mrow></math></span>) with <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e307"><math id="M307" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></math></span>, β, and γ when <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e308"><math id="M308" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span> is constant during constitutive secretion.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s017.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s017.pdf</a><sup> (367.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s018"><div class="caption p">
<span>S11 Fig. Simulating different strengths of chemical inhibition and genetic downregulation of sFLT1 secretion.</span><p>Change in extracellular (X) and intracellular (I) sFLT1 at 18 hours (solid) or 72 hours (dashed) with (A) chemical inhibition or (B) genetic downregulation of individual parameters at varying fraction inhibition. sFLT1 values are normalized to the 18-hour (solid) or 72-hour (dashed) values from simulation with the median parameter set.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s018.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s018.pdf</a><sup> (381.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s019"><div class="caption p">
<span>S12 Fig. Different sets of initial conditions for simulating chemical and genetic inhibition.</span><p>Time courses of extracellular (<em>X</em>) and intracellular (<em>I</em>) sFLT1 during constitutive simulation with different base parameter sets sharing constraints of <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e309"><math id="M309" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>=</mo><mi>α</mi><mi>β</mi><mo>=</mo><mn>7270</mn></mrow></math></span> #/cell/h [<a href="#pcbi.1013324.ref002" class="usa-link" aria-describedby="pcbi.1013324.ref002">2</a>] and <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="pcbi.1013324.e310"><math id="M310" display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>=</mo><mi>β</mi><mo>+</mo><mi>γ</mi><mo>=</mo><mn>0.173</mn></mrow></math></span> h<sup>-1</sup>. These time courses are used as the base cases for testing chemical and genetic inhibition from different initial conditions (Figs 9 and S13).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s019.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s019.pdf</a><sup> (124.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="pcbi.1013324.s020"><div class="caption p">
<span>S13 Fig. Variation in predicted effects of chemical and genetic perturbations across optimized parameter sets.</span><p>(A) Change in extracellular (<em>X</em>) and intracellular (<em>I</em>) sFLT1 at 18 hours (top) and 72 hours (bottom) with chemical inhibition of individual parameters at varying fraction inhibition for different base parameter sets (labeled by β values). (B) Change in extracellular (<em>X</em>) and intracellular (<em>I</em>) sFLT1 at 18 hours (top) and 72 hours (bottom) post media change with genetic inhibition of individual parameters at varying fraction inhibition for different base parameter sets (labeled by β values). <em>X</em> and <em>I</em> are normalized to the values at the end of each time course (18h or 72h, respectively) from the simulation with the corresponding base parameter set.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s020.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s020.pdf</a><sup> (533.8KB, pdf) </sup>
</div></div></section></section><section id="notes1"><h2 class="pmc_sec_title">Data Availability</h2>
<p>All code required to reproduce the model, output data, and analysis is available at <a href="https://github.com/gillsignals/sFlt1-paper" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/gillsignals/sFlt1-paper</a>.</p></section><section id="funding-statement1" lang="en"><h2 class="pmc_sec_title">Funding Statement</h2>
<p>This work was supported by National Institutes of Health grants R01-GM129074 (FMG, VLB), R01-HL101200 (FMG), and R35-HL139950 (VLB). [<a href="https://www.nih.gov/" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.nih.gov/</a>] The funders played no role in study design, data collection and analysis, decision to publish, or manuscript preparation.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="pcbi.1013324.ref001">
<span class="label">1.</span><cite>Uhlén M, Fagerberg L, Hallström BM, Lindskog C, Oksvold P, Mardinoglu A, et al. Proteomics. tissue-based map of the human proteome. Science. 2015;347(6220):1260419. doi: 10.1126/science.1260419

</cite> [<a href="https://doi.org/10.1126/science.1260419" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25613900/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Proteomics.%20tissue-based%20map%20of%20the%20human%20proteome&amp;author=M%20Uhl%C3%A9n&amp;author=L%20Fagerberg&amp;author=BM%20Hallstr%C3%B6m&amp;author=C%20Lindskog&amp;author=P%20Oksvold&amp;volume=347&amp;issue=6220&amp;publication_year=2015&amp;pages=1260419&amp;pmid=25613900&amp;doi=10.1126/science.1260419&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref002">
<span class="label">2.</span><cite>Palade G. Intracellular aspects of the process of protein synthesis. Science. 1975;189(4200):347–58. doi: 10.1126/science.1096303

</cite> [<a href="https://doi.org/10.1126/science.1096303" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/1096303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Intracellular%20aspects%20of%20the%20process%20of%20protein%20synthesis&amp;author=G%20Palade&amp;volume=189&amp;issue=4200&amp;publication_year=1975&amp;pages=347-58&amp;pmid=1096303&amp;doi=10.1126/science.1096303&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref003">
<span class="label">3.</span><cite>Uhlén M, et al. The human secretome. Sci Signal. 2019;12:eaaz0274.</cite> [<a href="https://doi.org/10.1126/scisignal.aaz0274" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31772123/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Signal&amp;title=The%20human%20secretome&amp;author=M%20Uhl%C3%A9n&amp;volume=12&amp;publication_year=2019&amp;pmid=31772123&amp;doi=10.1126/scisignal.aaz0274&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref004">
<span class="label">4.</span><cite>Munro S. Localization of proteins to the Golgi apparatus. Trends Cell Biol. 1998;8:11–5.
</cite> [<a href="https://doi.org/10.1016/S0962-8924(97)01197-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7172754/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9695801/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20Cell%20Biol&amp;title=Localization%20of%20proteins%20to%20the%20Golgi%20apparatus&amp;author=S%20Munro&amp;volume=8&amp;publication_year=1998&amp;pages=11-5&amp;pmid=9695801&amp;doi=10.1016/S0962-8924(97)01197-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref005">
<span class="label">5.</span><cite>Sun X, Mahajan D, Chen B, Song Z, Lu L. A quantitative study of the Golgi retention of glycosyltransferases. J Cell Sci. 2021;134(20):jcs258564. doi: 10.1242/jcs.258564

</cite> [<a href="https://doi.org/10.1242/jcs.258564" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34533190/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&amp;title=A%20quantitative%20study%20of%20the%20Golgi%20retention%20of%20glycosyltransferases&amp;author=X%20Sun&amp;author=D%20Mahajan&amp;author=B%20Chen&amp;author=Z%20Song&amp;author=L%20Lu&amp;volume=134&amp;issue=20&amp;publication_year=2021&amp;pmid=34533190&amp;doi=10.1242/jcs.258564&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref006">
<span class="label">6.</span><cite>Du Y, Shen J, Hsu JL, Han Z, Hsu M-C, Yang C-C, et al. Syntaxin 6-mediated Golgi translocation plays an important role in nuclear functions of EGFR through microtubule-dependent trafficking. Oncogene. 2014;33(6):756–70. doi: 10.1038/onc.2013.1

</cite> [<a href="https://doi.org/10.1038/onc.2013.1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3874427/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23376851/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Oncogene&amp;title=Syntaxin%206-mediated%20Golgi%20translocation%20plays%20an%20important%20role%20in%20nuclear%20functions%20of%20EGFR%20through%20microtubule-dependent%20trafficking&amp;author=Y%20Du&amp;author=J%20Shen&amp;author=JL%20Hsu&amp;author=Z%20Han&amp;author=M-C%20Hsu&amp;volume=33&amp;issue=6&amp;publication_year=2014&amp;pages=756-70&amp;pmid=23376851&amp;doi=10.1038/onc.2013.1&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref007">
<span class="label">7.</span><cite>Huang SS, Huang JS. Rapid turnover of the platelet-derived growth factor receptor in sis-transformed cells and reversal by suramin. Implications for the mechanism of autocrine transformation. J Biol Chem. 1988;263(25):12608–18. doi: 10.1016/s0021-9258(18)37798-6

</cite> [<a href="https://doi.org/10.1016/s0021-9258(18)37798-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2842336/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Rapid%20turnover%20of%20the%20platelet-derived%20growth%20factor%20receptor%20in%20sis-transformed%20cells%20and%20reversal%20by%20suramin.%20Implications%20for%20the%20mechanism%20of%20autocrine%20transformation&amp;author=SS%20Huang&amp;author=JS%20Huang&amp;volume=263&amp;issue=25&amp;publication_year=1988&amp;pages=12608-18&amp;pmid=2842336&amp;doi=10.1016/s0021-9258(18)37798-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref008">
<span class="label">8.</span><cite>Mittar S, Ulyatt C, Howell GJ, Bruns AF, Zachary I, Walker JH, et al. VEGFR1 receptor tyrosine kinase localization to the Golgi apparatus is calcium-dependent. Exp Cell Res. 2009;315(5):877–89. doi: 10.1016/j.yexcr.2008.12.020

</cite> [<a href="https://doi.org/10.1016/j.yexcr.2008.12.020" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19162007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Exp%20Cell%20Res&amp;title=VEGFR1%20receptor%20tyrosine%20kinase%20localization%20to%20the%20Golgi%20apparatus%20is%20calcium-dependent&amp;author=S%20Mittar&amp;author=C%20Ulyatt&amp;author=GJ%20Howell&amp;author=AF%20Bruns&amp;author=I%20Zachary&amp;volume=315&amp;issue=5&amp;publication_year=2009&amp;pages=877-89&amp;pmid=19162007&amp;doi=10.1016/j.yexcr.2008.12.020&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref009">
<span class="label">9.</span><cite>Sorkin A, von Zastrow M. Endocytosis and signalling: intertwining molecular networks. Nat Rev Mol Cell Biol. 2009;10(9):609–22. doi: 10.1038/nrm2748

</cite> [<a href="https://doi.org/10.1038/nrm2748" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2895425/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19696798/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&amp;title=Endocytosis%20and%20signalling:%20intertwining%20molecular%20networks&amp;author=A%20Sorkin&amp;author=M%20von%20Zastrow&amp;volume=10&amp;issue=9&amp;publication_year=2009&amp;pages=609-22&amp;pmid=19696798&amp;doi=10.1038/nrm2748&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref010">
<span class="label">10.</span><cite>Sorkin A, Goh LK. Endocytosis and intracellular trafficking of ErbBs. Exp Cell Res. 2008;314(17):3093–106. doi: 10.1016/j.yexcr.2008.08.013

</cite> [<a href="https://doi.org/10.1016/j.yexcr.2008.08.013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2605728/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18793634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Exp%20Cell%20Res&amp;title=Endocytosis%20and%20intracellular%20trafficking%20of%20ErbBs&amp;author=A%20Sorkin&amp;author=LK%20Goh&amp;volume=314&amp;issue=17&amp;publication_year=2008&amp;pages=3093-106&amp;pmid=18793634&amp;doi=10.1016/j.yexcr.2008.08.013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref011">
<span class="label">11.</span><cite>Kendall RL, Thomas KA. Inhibition of vascular endothelial cell growth factor activity by an endogenously encoded soluble receptor. Proc Natl Acad Sci U S A. 1993;90(22):10705–9. doi: 10.1073/pnas.90.22.10705

</cite> [<a href="https://doi.org/10.1073/pnas.90.22.10705" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC47846/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8248162/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Inhibition%20of%20vascular%20endothelial%20cell%20growth%20factor%20activity%20by%20an%20endogenously%20encoded%20soluble%20receptor&amp;author=RL%20Kendall&amp;author=KA%20Thomas&amp;volume=90&amp;issue=22&amp;publication_year=1993&amp;pages=10705-9&amp;pmid=8248162&amp;doi=10.1073/pnas.90.22.10705&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref012">
<span class="label">12.</span><cite>Kendall RL, Wang G, Thomas KA. Identification of a natural soluble form of the vascular endothelial growth factor receptor, FLT-1, and its heterodimerization with KDR. Biochem Biophys Res Commun. 1996;226:324–8.
</cite> [<a href="https://doi.org/10.1006/bbrc.1996.1355" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8806634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&amp;title=Identification%20of%20a%20natural%20soluble%20form%20of%20the%20vascular%20endothelial%20growth%20factor%20receptor,%20FLT-1,%20and%20its%20heterodimerization%20with%20KDR&amp;author=RL%20Kendall&amp;author=G%20Wang&amp;author=KA%20Thomas&amp;volume=226&amp;publication_year=1996&amp;pages=324-8&amp;pmid=8806634&amp;doi=10.1006/bbrc.1996.1355&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref013">
<span class="label">13.</span><cite>Kinghorn K, Gill A, Marvin A, Li R, Quigley K, Singh S, et al. A defined clathrin-mediated trafficking pathway regulates sFLT1/VEGFR1 secretion from endothelial cells. Angiogenesis. 2024;27(1):67–89. doi: 10.1007/s10456-023-09893-6

</cite> [<a href="https://doi.org/10.1007/s10456-023-09893-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10881643/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37695358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Angiogenesis&amp;title=A%20defined%20clathrin-mediated%20trafficking%20pathway%20regulates%20sFLT1/VEGFR1%20secretion%20from%20endothelial%20cells&amp;author=K%20Kinghorn&amp;author=A%20Gill&amp;author=A%20Marvin&amp;author=R%20Li&amp;author=K%20Quigley&amp;volume=27&amp;issue=1&amp;publication_year=2024&amp;pages=67-89&amp;pmid=37695358&amp;doi=10.1007/s10456-023-09893-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref014">
<span class="label">14.</span><cite>Hiratsuka S, Minowa O, Kuno J, Noda T, Shibuya M. Flt-1 lacking the tyrosine kinase domain is sufficient for normal development and angiogenesis in mice. Proc Natl Acad Sci U S A. 1998;95(16):9349–54. doi: 10.1073/pnas.95.16.9349

</cite> [<a href="https://doi.org/10.1073/pnas.95.16.9349" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC21341/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9689083/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proc%20Natl%20Acad%20Sci%20U%20S%20A&amp;title=Flt-1%20lacking%20the%20tyrosine%20kinase%20domain%20is%20sufficient%20for%20normal%20development%20and%20angiogenesis%20in%20mice&amp;author=S%20Hiratsuka&amp;author=O%20Minowa&amp;author=J%20Kuno&amp;author=T%20Noda&amp;author=M%20Shibuya&amp;volume=95&amp;issue=16&amp;publication_year=1998&amp;pages=9349-54&amp;pmid=9689083&amp;doi=10.1073/pnas.95.16.9349&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref015">
<span class="label">15.</span><cite>Bautch VL. VEGF-directed blood vessel patterning: from cells to organism. Cold Spring Harb Perspect Med. 2012;2(9):a006452. doi: 10.1101/cshperspect.a006452

</cite> [<a href="https://doi.org/10.1101/cshperspect.a006452" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3426816/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22951440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cold%20Spring%20Harb%20Perspect%20Med&amp;title=VEGF-directed%20blood%20vessel%20patterning:%20from%20cells%20to%20organism&amp;author=VL%20Bautch&amp;volume=2&amp;issue=9&amp;publication_year=2012&amp;pmid=22951440&amp;doi=10.1101/cshperspect.a006452&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref016">
<span class="label">16.</span><cite>Wu FTH, Stefanini MO, Mac Gabhann F, Kontos CD, Annex BH, Popel AS. A systems biology perspective on sVEGFR1: its biological function, pathogenic role and therapeutic use. J Cell Mol Med. 2010;14(3):528–52. doi: 10.1111/j.1582-4934.2009.00941.x

</cite> [<a href="https://doi.org/10.1111/j.1582-4934.2009.00941.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3039304/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19840194/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Mol%20Med&amp;title=A%20systems%20biology%20perspective%20on%20sVEGFR1:%20its%20biological%20function,%20pathogenic%20role%20and%20therapeutic%20use&amp;author=FTH%20Wu&amp;author=MO%20Stefanini&amp;author=F%20Mac%20Gabhann&amp;author=CD%20Kontos&amp;author=BH%20Annex&amp;volume=14&amp;issue=3&amp;publication_year=2010&amp;pages=528-52&amp;pmid=19840194&amp;doi=10.1111/j.1582-4934.2009.00941.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref017">
<span class="label">17.</span><cite>Ambati BK, Nozaki M, Singh N, Takeda A, Jani PD, Suthar T, et al. Corneal avascularity is due to soluble VEGF receptor-1. Nature. 2006;443(7114):993–7. doi: 10.1038/nature05249

</cite> [<a href="https://doi.org/10.1038/nature05249" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2656128/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17051153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Corneal%20avascularity%20is%20due%20to%20soluble%20VEGF%20receptor-1&amp;author=BK%20Ambati&amp;author=M%20Nozaki&amp;author=N%20Singh&amp;author=A%20Takeda&amp;author=PD%20Jani&amp;volume=443&amp;issue=7114&amp;publication_year=2006&amp;pages=993-7&amp;pmid=17051153&amp;doi=10.1038/nature05249&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref018">
<span class="label">18.</span><cite>Chappell JC, Taylor SM, Ferrara N., Bautch VL. Local Guidance of Emerging Vessel Sprouts Requires Soluble Flt-1. Dev. Cell
17, 377–86 (2009).
</cite> [<a href="https://doi.org/10.1016/j.devcel.2009.07.011" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2747120/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19758562/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Dev.%20Cell&amp;title=Local%20Guidance%20of%20Emerging%20Vessel%20Sprouts%20Requires%20Soluble%20Flt-1&amp;author=JC%20Chappell&amp;author=SM%20Taylor&amp;author=N.%20Ferrara&amp;author=VL%20Bautch&amp;volume=17&amp;publication_year=2009&amp;pages=377-86&amp;pmid=19758562&amp;doi=10.1016/j.devcel.2009.07.011&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref019">
<span class="label">19.</span><cite>Hashambhoy YL, Chappell JC, Peirce SM, Bautch VL, Mac Gabhann F. Computational modeling of interacting VEGF and soluble VEGF receptor concentration gradients. Front Physiol. 2011;2:62. doi: 10.3389/fphys.2011.00062

</cite> [<a href="https://doi.org/10.3389/fphys.2011.00062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3185289/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22007175/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Front%20Physiol&amp;title=Computational%20modeling%20of%20interacting%20VEGF%20and%20soluble%20VEGF%20receptor%20concentration%20gradients&amp;author=YL%20Hashambhoy&amp;author=JC%20Chappell&amp;author=SM%20Peirce&amp;author=VL%20Bautch&amp;author=F%20Mac%20Gabhann&amp;volume=2&amp;publication_year=2011&amp;pages=62&amp;pmid=22007175&amp;doi=10.3389/fphys.2011.00062&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref020">
<span class="label">20.</span><cite>Kappas NC, Zeng G, Chappell JC, Kearney JB, Hazarika S, Kallianos KG, et al. The VEGF receptor Flt-1 spatially modulates Flk-1 signaling and blood vessel branching. J Cell Biol. 2008;181(5):847–58. doi: 10.1083/jcb.200709114

</cite> [<a href="https://doi.org/10.1083/jcb.200709114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2396811/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18504303/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biol&amp;title=The%20VEGF%20receptor%20Flt-1%20spatially%20modulates%20Flk-1%20signaling%20and%20blood%20vessel%20branching&amp;author=NC%20Kappas&amp;author=G%20Zeng&amp;author=JC%20Chappell&amp;author=JB%20Kearney&amp;author=S%20Hazarika&amp;volume=181&amp;issue=5&amp;publication_year=2008&amp;pages=847-58&amp;pmid=18504303&amp;doi=10.1083/jcb.200709114&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref021">
<span class="label">21.</span><cite>Maynard SE, et al. Excess placental soluble fms-like tyrosine kinase 1 (sFlt1) may contribute to endothelial dysfunction, hypertension, and proteinuria in preeclampsia. J Clin Invest. 2003;111:649–58.
</cite> [<a href="https://doi.org/10.1172/JCI17189" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC151901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12618519/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&amp;title=Excess%20placental%20soluble%20fms-like%20tyrosine%20kinase%201%20(sFlt1)%20may%20contribute%20to%20endothelial%20dysfunction,%20hypertension,%20and%20proteinuria%20in%20preeclampsia&amp;author=SE%20Maynard&amp;volume=111&amp;publication_year=2003&amp;pages=649-58&amp;pmid=12618519&amp;doi=10.1172/JCI17189&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref022">
<span class="label">22.</span><cite>Jung J-J, Tiwari A, Inamdar SM, Thomas CP, Goel A, Choudhury A. Secretion of soluble vascular endothelial growth factor receptor 1 (sVEGFR1/sFlt1) requires Arf1, Arf6, and Rab11 GTPases. PLoS One. 2012;7(9):e44572. doi: 10.1371/journal.pone.0044572

</cite> [<a href="https://doi.org/10.1371/journal.pone.0044572" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3433446/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22962618/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Secretion%20of%20soluble%20vascular%20endothelial%20growth%20factor%20receptor%201%20(sVEGFR1/sFlt1)%20requires%20Arf1,%20Arf6,%20and%20Rab11%20GTPases&amp;author=J-J%20Jung&amp;author=A%20Tiwari&amp;author=SM%20Inamdar&amp;author=CP%20Thomas&amp;author=A%20Goel&amp;volume=7&amp;issue=9&amp;publication_year=2012&amp;pmid=22962618&amp;doi=10.1371/journal.pone.0044572&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref023">
<span class="label">23.</span><cite>Sarabipour S. et al. Trafficking dynamics of VEGFR1, VEGFR2, and NRP1 in human endothelial cells. PLOS Comput. Biol. 2024;20:e1011798.</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1011798" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10878527/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38324585/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLOS%20Comput.%20Biol&amp;title=Trafficking%20dynamics%20of%20VEGFR1,%20VEGFR2,%20and%20NRP1%20in%20human%20endothelial%20cells&amp;author=S%20Sarabipour&amp;volume=20&amp;publication_year=2024&amp;pmid=38324585&amp;doi=10.1371/journal.pcbi.1011798&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref024">
<span class="label">24.</span><cite>Clegg LE, Mac Gabhann F. Molecular mechanism matters: Benefits of mechanistic computational models for drug development. Pharmacol Res. 2015;99:149–54. doi: 10.1016/j.phrs.2015.06.002

</cite> [<a href="https://doi.org/10.1016/j.phrs.2015.06.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4567444/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26093283/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacol%20Res&amp;title=Molecular%20mechanism%20matters:%20Benefits%20of%20mechanistic%20computational%20models%20for%20drug%20development&amp;author=LE%20Clegg&amp;author=F%20Mac%20Gabhann&amp;volume=99&amp;publication_year=2015&amp;pages=149-54&amp;pmid=26093283&amp;doi=10.1016/j.phrs.2015.06.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref025">
<span class="label">25.</span><cite>Glass DS, Jin X, Riedel-Kruse IH. Nonlinear delay differential equations and their application to modeling biological network motifs. Nat Commun. 2021;12(1):1788. doi: 10.1038/s41467-021-21700-8

</cite> [<a href="https://doi.org/10.1038/s41467-021-21700-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7979834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33741909/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Nonlinear%20delay%20differential%20equations%20and%20their%20application%20to%20modeling%20biological%20network%20motifs&amp;author=DS%20Glass&amp;author=X%20Jin&amp;author=IH%20Riedel-Kruse&amp;volume=12&amp;issue=1&amp;publication_year=2021&amp;pages=1788&amp;pmid=33741909&amp;doi=10.1038/s41467-021-21700-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref026">
<span class="label">26.</span><cite>Wu FTH, Stefanini MO, Mac Gabhann F, Kontos CD, Annex BH, Popel AS. Computational kinetic model of VEGF trapping by soluble VEGF receptor-1: effects of transendothelial and lymphatic macromolecular transport. Physiol Genomics. 2009;38(1):29–41. doi: 10.1152/physiolgenomics.00031.2009

</cite> [<a href="https://doi.org/10.1152/physiolgenomics.00031.2009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2696151/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19351908/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Physiol%20Genomics&amp;title=Computational%20kinetic%20model%20of%20VEGF%20trapping%20by%20soluble%20VEGF%20receptor-1:%20effects%20of%20transendothelial%20and%20lymphatic%20macromolecular%20transport&amp;author=FTH%20Wu&amp;author=MO%20Stefanini&amp;author=F%20Mac%20Gabhann&amp;author=CD%20Kontos&amp;author=BH%20Annex&amp;volume=38&amp;issue=1&amp;publication_year=2009&amp;pages=29-41&amp;pmid=19351908&amp;doi=10.1152/physiolgenomics.00031.2009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref027">
<span class="label">27.</span><cite>Clegg LE, Mac Gabhann F. A computational analysis of in vivo VEGFR activation by multiple co-expressed ligands. PLoS Comput Biol. 2017;13(3):e1005445. doi: 10.1371/journal.pcbi.1005445

</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1005445" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5378411/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28319199/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=A%20computational%20analysis%20of%20in%20vivo%20VEGFR%20activation%20by%20multiple%20co-expressed%20ligands&amp;author=LE%20Clegg&amp;author=F%20Mac%20Gabhann&amp;volume=13&amp;issue=3&amp;publication_year=2017&amp;pmid=28319199&amp;doi=10.1371/journal.pcbi.1005445&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref028">
<span class="label">28.</span><cite>Clegg LE, Mac Gabhann F. A computational analysis of pro-angiogenic therapies for peripheral artery disease. Integr Biol (Camb). 2018;10(1):18–33. doi: 10.1039/c7ib00218a

</cite> [<a href="https://doi.org/10.1039/c7ib00218a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7017937/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29327758/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Integr%20Biol%20(Camb)&amp;title=A%20computational%20analysis%20of%20pro-angiogenic%20therapies%20for%20peripheral%20artery%20disease&amp;author=LE%20Clegg&amp;author=F%20Mac%20Gabhann&amp;volume=10&amp;issue=1&amp;publication_year=2018&amp;pages=18-33&amp;pmid=29327758&amp;doi=10.1039/c7ib00218a&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref029">
<span class="label">29.</span><cite>Hornig C, Barleon B, Ahmad S, Vuorela P, Ahmed A, Weich HA. Release and complex formation of soluble VEGFR-1 from endothelial cells and biological fluids. Lab Invest. 2000;80(4):443–54. doi: 10.1038/labinvest.3780050

</cite> [<a href="https://doi.org/10.1038/labinvest.3780050" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10780661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Lab%20Invest&amp;title=Release%20and%20complex%20formation%20of%20soluble%20VEGFR-1%20from%20endothelial%20cells%20and%20biological%20fluids&amp;author=C%20Hornig&amp;author=B%20Barleon&amp;author=S%20Ahmad&amp;author=P%20Vuorela&amp;author=A%20Ahmed&amp;volume=80&amp;issue=4&amp;publication_year=2000&amp;pages=443-54&amp;pmid=10780661&amp;doi=10.1038/labinvest.3780050&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref030">
<span class="label">30.</span><cite>Schindelin J, Arganda-Carreras I, Frise E, Kaynig V, Longair M, Pietzsch T, et al. Fiji: an open-source platform for biological-image analysis. Nat Methods. 2012;9(7):676–82. doi: 10.1038/nmeth.2019

</cite> [<a href="https://doi.org/10.1038/nmeth.2019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3855844/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22743772/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Methods&amp;title=Fiji:%20an%20open-source%20platform%20for%20biological-image%20analysis&amp;author=J%20Schindelin&amp;author=I%20Arganda-Carreras&amp;author=E%20Frise&amp;author=V%20Kaynig&amp;author=M%20Longair&amp;volume=9&amp;issue=7&amp;publication_year=2012&amp;pages=676-82&amp;pmid=22743772&amp;doi=10.1038/nmeth.2019&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref031">
<span class="label">31.</span><cite>Eden E, et al. Proteome half-life dynamics in living human cells. Science. 2011;331:764–8.
</cite> [<a href="https://doi.org/10.1126/science.1199784" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21233346/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Proteome%20half-life%20dynamics%20in%20living%20human%20cells&amp;author=E%20Eden&amp;volume=331&amp;publication_year=2011&amp;pages=764-8&amp;pmid=21233346&amp;doi=10.1126/science.1199784&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref032">
<span class="label">32.</span><cite>Schwanhäusser B, Busse D, Li N, Dittmar G, Schuchhardt J, Wolf J, et al. Global quantification of mammalian gene expression control. Nature. 2011;473(7347):337–42. doi: 10.1038/nature10098

</cite> [<a href="https://doi.org/10.1038/nature10098" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21593866/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Global%20quantification%20of%20mammalian%20gene%20expression%20control&amp;author=B%20Schwanh%C3%A4usser&amp;author=D%20Busse&amp;author=N%20Li&amp;author=G%20Dittmar&amp;author=J%20Schuchhardt&amp;volume=473&amp;issue=7347&amp;publication_year=2011&amp;pages=337-42&amp;pmid=21593866&amp;doi=10.1038/nature10098&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref033">
<span class="label">33.</span><cite>Hausser J, Mayo A, Keren L, Alon U. Central dogma rates and the trade-off between precision and economy in gene expression. Nat Commun. 2019;10(1):68. doi: 10.1038/s41467-018-07391-8

</cite> [<a href="https://doi.org/10.1038/s41467-018-07391-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6325141/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30622246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Central%20dogma%20rates%20and%20the%20trade-off%20between%20precision%20and%20economy%20in%20gene%20expression&amp;author=J%20Hausser&amp;author=A%20Mayo&amp;author=L%20Keren&amp;author=U%20Alon&amp;volume=10&amp;issue=1&amp;publication_year=2019&amp;pages=68&amp;pmid=30622246&amp;doi=10.1038/s41467-018-07391-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref034">
<span class="label">34.</span><cite>Akaike H. Information Theory and an Extension of the Maximum Likelihood Principle. Springer Series in Statistics. Springer New York. 1998. p. 199–213. doi: 10.1007/978-1-4612-1694-0_15</cite> [<a href="https://doi.org/10.1007/978-1-4612-1694-0_15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Springer%20Series%20in%20Statistics.%20Springer%20New%20York&amp;title=Information%20Theory%20and%20an%20Extension%20of%20the%20Maximum%20Likelihood%20Principle&amp;author=H%20Akaike&amp;publication_year=1998&amp;pages=199-213&amp;doi=10.1007/978-1-4612-1694-0_15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref035">
<span class="label">35.</span><cite>Bozdogan H. Model selection and Akaike’s Information Criterion (AIC): The general theory and its analytical extensions. Psychometrika. 1987;52:345–70.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Psychometrika&amp;title=Model%20selection%20and%20Akaike%E2%80%99s%20Information%20Criterion%20(AIC):%20The%20general%20theory%20and%20its%20analytical%20extensions&amp;author=H%20Bozdogan&amp;volume=52&amp;publication_year=1987&amp;pages=345-70&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref036">
<span class="label">36.</span><cite>Misumi Y, Misumi Y, Miki K, Takatsuki A, Tamura G, Ikehara Y. Novel blockade by brefeldin A of intracellular transport of secretory proteins in cultured rat hepatocytes. J Biol Chem. 1986;261(24):11398–403. 
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/2426273/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Novel%20blockade%20by%20brefeldin%20A%20of%20intracellular%20transport%20of%20secretory%20proteins%20in%20cultured%20rat%20hepatocytes&amp;author=Y%20Misumi&amp;author=Y%20Misumi&amp;author=K%20Miki&amp;author=A%20Takatsuki&amp;author=G%20Tamura&amp;volume=261&amp;issue=24&amp;publication_year=1986&amp;pages=11398-403&amp;pmid=2426273&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref037">
<span class="label">37.</span><cite>Fujiwara T, Oda K, Yokota S, Takatsuki A, Ikehara Y. Brefeldin A causes disassembly of the Golgi complex and accumulation of secretory proteins in the endoplasmic reticulum. J Biol Chem. 1988;263(34):18545–52. 
</cite> [<a href="https://pubmed.ncbi.nlm.nih.gov/3192548/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Biol%20Chem&amp;title=Brefeldin%20A%20causes%20disassembly%20of%20the%20Golgi%20complex%20and%20accumulation%20of%20secretory%20proteins%20in%20the%20endoplasmic%20reticulum&amp;author=T%20Fujiwara&amp;author=K%20Oda&amp;author=S%20Yokota&amp;author=A%20Takatsuki&amp;author=Y%20Ikehara&amp;volume=263&amp;issue=34&amp;publication_year=1988&amp;pages=18545-52&amp;pmid=3192548&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref038">
<span class="label">38.</span><cite>Lippincott-Schwartz J, Yuan LC, Bonifacino JS, Klausner RD. Rapid redistribution of Golgi proteins into the ER in cells treated with brefeldin A: evidence for membrane cycling from Golgi to ER. Cell. 1989;56(5):801–13. doi: 10.1016/0092-8674(89)90685-5

</cite> [<a href="https://doi.org/10.1016/0092-8674(89)90685-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7173269/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/2647301/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Rapid%20redistribution%20of%20Golgi%20proteins%20into%20the%20ER%20in%20cells%20treated%20with%20brefeldin%20A:%20evidence%20for%20membrane%20cycling%20from%20Golgi%20to%20ER&amp;author=J%20Lippincott-Schwartz&amp;author=LC%20Yuan&amp;author=JS%20Bonifacino&amp;author=RD%20Klausner&amp;volume=56&amp;issue=5&amp;publication_year=1989&amp;pages=801-13&amp;pmid=2647301&amp;doi=10.1016/0092-8674(89)90685-5&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref039">
<span class="label">39.</span><cite>Wang LH, Rothberg KG, Anderson RG. Mis-assembly of clathrin lattices on endosomes reveals a regulatory switch for coated pit formation. J Cell Biol. 1993;123(5):1107–17. doi: 10.1083/jcb.123.5.1107

</cite> [<a href="https://doi.org/10.1083/jcb.123.5.1107" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2119875/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/8245121/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biol&amp;title=Mis-assembly%20of%20clathrin%20lattices%20on%20endosomes%20reveals%20a%20regulatory%20switch%20for%20coated%20pit%20formation&amp;author=LH%20Wang&amp;author=KG%20Rothberg&amp;author=RG%20Anderson&amp;volume=123&amp;issue=5&amp;publication_year=1993&amp;pages=1107-17&amp;pmid=8245121&amp;doi=10.1083/jcb.123.5.1107&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref040">
<span class="label">40.</span><cite>Matsushita K, Morrell CN, Lowenstein CJ. A novel class of fusion polypeptides inhibits exocytosis. Mol Pharmacol. 2005;67(4):1137–44. doi: 10.1124/mol.104.004275

</cite> [<a href="https://doi.org/10.1124/mol.104.004275" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15608145/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Pharmacol&amp;title=A%20novel%20class%20of%20fusion%20polypeptides%20inhibits%20exocytosis&amp;author=K%20Matsushita&amp;author=CN%20Morrell&amp;author=CJ%20Lowenstein&amp;volume=67&amp;issue=4&amp;publication_year=2005&amp;pages=1137-44&amp;pmid=15608145&amp;doi=10.1124/mol.104.004275&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref041">
<span class="label">41.</span><cite>Mauthe M, Orhon I, Rocchi C, Zhou X, Luhr M, Hijlkema K-J, et al. Chloroquine inhibits autophagic flux by decreasing autophagosome-lysosome fusion. Autophagy. 2018;14(8):1435–55. doi: 10.1080/15548627.2018.1474314

</cite> [<a href="https://doi.org/10.1080/15548627.2018.1474314" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6103682/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29940786/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Autophagy&amp;title=Chloroquine%20inhibits%20autophagic%20flux%20by%20decreasing%20autophagosome-lysosome%20fusion&amp;author=M%20Mauthe&amp;author=I%20Orhon&amp;author=C%20Rocchi&amp;author=X%20Zhou&amp;author=M%20Luhr&amp;volume=14&amp;issue=8&amp;publication_year=2018&amp;pages=1435-55&amp;pmid=29940786&amp;doi=10.1080/15548627.2018.1474314&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref042">
<span class="label">42.</span><cite>Lie SO, Schofield B. Inactivation of lysosomal function in normal cultured human fibroblasts by chloroquine. Biochem Pharmacol. 1973;22(23):3109–14. doi: 10.1016/0006-2952(73)90197-4

</cite> [<a href="https://doi.org/10.1016/0006-2952(73)90197-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4271529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20Pharmacol&amp;title=Inactivation%20of%20lysosomal%20function%20in%20normal%20cultured%20human%20fibroblasts%20by%20chloroquine&amp;author=SO%20Lie&amp;author=B%20Schofield&amp;volume=22&amp;issue=23&amp;publication_year=1973&amp;pages=3109-14&amp;pmid=4271529&amp;doi=10.1016/0006-2952(73)90197-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref043">
<span class="label">43.</span><cite>Wu X, Rao K, Bowers MB, Copeland NG, Jenkins NA, Hammer JA 3rd. Rab27a enables myosin Va-dependent melanosome capture by recruiting the myosin to the organelle. J Cell Sci. 2001;114(Pt 6):1091–100. doi: 10.1242/jcs.114.6.1091

</cite> [<a href="https://doi.org/10.1242/jcs.114.6.1091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11228153/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&amp;title=Rab27a%20enables%20myosin%20Va-dependent%20melanosome%20capture%20by%20recruiting%20the%20myosin%20to%20the%20organelle&amp;author=X%20Wu&amp;author=K%20Rao&amp;author=MB%20Bowers&amp;author=NG%20Copeland&amp;author=NA%20Jenkins&amp;volume=114&amp;publication_year=2001&amp;pages=1091-100&amp;pmid=11228153&amp;doi=10.1242/jcs.114.6.1091&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref044">
<span class="label">44.</span><cite>Alzahofi N, Welz T, Robinson CL, Page EL, Briggs DA, Stainthorp AK, et al. Rab27a co-ordinates actin-dependent transport by controlling organelle-associated motors and track assembly proteins. Nat Commun. 2020;11(1):3495. doi: 10.1038/s41467-020-17212-6

</cite> [<a href="https://doi.org/10.1038/s41467-020-17212-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7359353/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32661310/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Commun&amp;title=Rab27a%20co-ordinates%20actin-dependent%20transport%20by%20controlling%20organelle-associated%20motors%20and%20track%20assembly%20proteins&amp;author=N%20Alzahofi&amp;author=T%20Welz&amp;author=CL%20Robinson&amp;author=EL%20Page&amp;author=DA%20Briggs&amp;volume=11&amp;issue=1&amp;publication_year=2020&amp;pages=3495&amp;pmid=32661310&amp;doi=10.1038/s41467-020-17212-6&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref045">
<span class="label">45.</span><cite>Sharma N, Low SH, Misra S, Pallavi B, Weimbs T. Apical targeting of syntaxin 3 is essential for epithelial cell polarity. J Cell Biol. 2006;173(6):937–48. doi: 10.1083/jcb.200603132

</cite> [<a href="https://doi.org/10.1083/jcb.200603132" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2063918/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16785322/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Biol&amp;title=Apical%20targeting%20of%20syntaxin%203%20is%20essential%20for%20epithelial%20cell%20polarity&amp;author=N%20Sharma&amp;author=SH%20Low&amp;author=S%20Misra&amp;author=B%20Pallavi&amp;author=T%20Weimbs&amp;volume=173&amp;issue=6&amp;publication_year=2006&amp;pages=937-48&amp;pmid=16785322&amp;doi=10.1083/jcb.200603132&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref046">
<span class="label">46.</span><cite>Bock JB, Klumperman J, Davanger S, Scheller RH. Syntaxin 6 functions in trans-Golgi network vesicle trafficking. Mol Biol Cell. 1997;8(7):1261–71. doi: 10.1091/mbc.8.7.1261

</cite> [<a href="https://doi.org/10.1091/mbc.8.7.1261" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC276151/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9243506/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol%20Biol%20Cell&amp;title=Syntaxin%206%20functions%20in%20trans-Golgi%20network%20vesicle%20trafficking&amp;author=JB%20Bock&amp;author=J%20Klumperman&amp;author=S%20Davanger&amp;author=RH%20Scheller&amp;volume=8&amp;issue=7&amp;publication_year=1997&amp;pages=1261-71&amp;pmid=9243506&amp;doi=10.1091/mbc.8.7.1261&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref047">
<span class="label">47.</span><cite>Wendler F, Tooze S. Syntaxin 6: the promiscuous behaviour of a SNARE protein. Traffic. 2001;2(9):606–11. doi: 10.1034/j.1600-0854.2001.20903.x

</cite> [<a href="https://doi.org/10.1034/j.1600-0854.2001.20903.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11555414/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Traffic&amp;title=Syntaxin%206:%20the%20promiscuous%20behaviour%20of%20a%20SNARE%20protein&amp;author=F%20Wendler&amp;author=S%20Tooze&amp;volume=2&amp;issue=9&amp;publication_year=2001&amp;pages=606-11&amp;pmid=11555414&amp;doi=10.1034/j.1600-0854.2001.20903.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref048">
<span class="label">48.</span><cite>Hu C, Hardee D, Minnear F. Membrane fusion by VAMP3 and plasma membrane t-SNAREs. Exp Cell Res. 2007;313(15):3198–209. doi: 10.1016/j.yexcr.2007.06.008

</cite> [<a href="https://doi.org/10.1016/j.yexcr.2007.06.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2696983/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17651732/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Exp%20Cell%20Res&amp;title=Membrane%20fusion%20by%20VAMP3%20and%20plasma%20membrane%20t-SNAREs&amp;author=C%20Hu&amp;author=D%20Hardee&amp;author=F%20Minnear&amp;volume=313&amp;issue=15&amp;publication_year=2007&amp;pages=3198-209&amp;pmid=17651732&amp;doi=10.1016/j.yexcr.2007.06.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref049">
<span class="label">49.</span><cite>Manickam V, Tiwari A, Jung J-J, Bhattacharya R, Goel A, Mukhopadhyay D, et al. Regulation of vascular endothelial growth factor receptor 2 trafficking and angiogenesis by Golgi localized t-SNARE syntaxin 6. Blood. 2011;117(4):1425–35. doi: 10.1182/blood-2010-06-291690

</cite> [<a href="https://doi.org/10.1182/blood-2010-06-291690" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3056478/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21063020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Blood&amp;title=Regulation%20of%20vascular%20endothelial%20growth%20factor%20receptor%202%20trafficking%20and%20angiogenesis%20by%20Golgi%20localized%20t-SNARE%20syntaxin%206&amp;author=V%20Manickam&amp;author=A%20Tiwari&amp;author=J-J%20Jung&amp;author=R%20Bhattacharya&amp;author=A%20Goel&amp;volume=117&amp;issue=4&amp;publication_year=2011&amp;pages=1425-35&amp;pmid=21063020&amp;doi=10.1182/blood-2010-06-291690&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref050">
<span class="label">50.</span><cite>Saito T, Takeda N, Amiya E, Nakao T, Abe H, Semba H, et al. VEGF-A induces its negative regulator, soluble form of VEGFR-1, by modulating its alternative splicing. FEBS Lett. 2013;587(14):2179–85. doi: 10.1016/j.febslet.2013.05.038

</cite> [<a href="https://doi.org/10.1016/j.febslet.2013.05.038" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23711375/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=FEBS%20Lett&amp;title=VEGF-A%20induces%20its%20negative%20regulator,%20soluble%20form%20of%20VEGFR-1,%20by%20modulating%20its%20alternative%20splicing&amp;author=T%20Saito&amp;author=N%20Takeda&amp;author=E%20Amiya&amp;author=T%20Nakao&amp;author=H%20Abe&amp;volume=587&amp;issue=14&amp;publication_year=2013&amp;pages=2179-85&amp;pmid=23711375&amp;doi=10.1016/j.febslet.2013.05.038&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref051">
<span class="label">51.</span><cite>Ahmad S, Hewett PW, Al-Ani B, Sissaoui S, Fujisawa T, Cudmore MJ, et al. Autocrine activity of soluble Flt-1 controls endothelial cell function and angiogenesis. Vasc Cell. 2011;3(1):15. doi: 10.1186/2045-824X-3-15

</cite> [<a href="https://doi.org/10.1186/2045-824X-3-15" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3173355/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21752276/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Vasc%20Cell&amp;title=Autocrine%20activity%20of%20soluble%20Flt-1%20controls%20endothelial%20cell%20function%20and%20angiogenesis&amp;author=S%20Ahmad&amp;author=PW%20Hewett&amp;author=B%20Al-Ani&amp;author=S%20Sissaoui&amp;author=T%20Fujisawa&amp;volume=3&amp;issue=1&amp;publication_year=2011&amp;pages=15&amp;pmid=21752276&amp;doi=10.1186/2045-824X-3-15&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref052">
<span class="label">52.</span><cite>de Galarreta MR, Navarro A, Ansorena E, Garzón AG, Mòdol T, López-Zabalza MJ, et al. Unfolded protein response induced by Brefeldin A increases collagen type I levels in hepatic stellate cells through an IRE1α, p38 MAPK and Smad-dependent pathway. Biochim Biophys Acta. 2016;1863(8):2115–23. doi: 10.1016/j.bbamcr.2016.05.002

</cite> [<a href="https://doi.org/10.1016/j.bbamcr.2016.05.002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27155082/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochim%20Biophys%20Acta&amp;title=Unfolded%20protein%20response%20induced%20by%20Brefeldin%20A%20increases%20collagen%20type%20I%20levels%20in%20hepatic%20stellate%20cells%20through%20an%20IRE1%CE%B1,%20p38%20MAPK%20and%20Smad-dependent%20pathway&amp;author=MR%20de%20Galarreta&amp;author=A%20Navarro&amp;author=E%20Ansorena&amp;author=AG%20Garz%C3%B3n&amp;author=T%20M%C3%B2dol&amp;volume=1863&amp;issue=8&amp;publication_year=2016&amp;pages=2115-23&amp;pmid=27155082&amp;doi=10.1016/j.bbamcr.2016.05.002&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref053">
<span class="label">53.</span><cite>Hetz C, Zhang K, Kaufman RJ. Mechanisms, regulation and functions of the unfolded protein response. Nat Rev Mol Cell Biol. 2020;21(8):421–38. doi: 10.1038/s41580-020-0250-z

</cite> [<a href="https://doi.org/10.1038/s41580-020-0250-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8867924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32457508/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Mol%20Cell%20Biol&amp;title=Mechanisms,%20regulation%20and%20functions%20of%20the%20unfolded%20protein%20response&amp;author=C%20Hetz&amp;author=K%20Zhang&amp;author=RJ%20Kaufman&amp;volume=21&amp;issue=8&amp;publication_year=2020&amp;pages=421-38&amp;pmid=32457508&amp;doi=10.1038/s41580-020-0250-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref054">
<span class="label">54.</span><cite>Stockholm D, Benchaouir R, Picot J, Rameau P, Neildez TMA, Landini G, et al. The origin of phenotypic heterogeneity in a clonal cell population in vitro. PLoS One. 2007;2(4):e394. doi: 10.1371/journal.pone.0000394

</cite> [<a href="https://doi.org/10.1371/journal.pone.0000394" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1851097/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17460761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=The%20origin%20of%20phenotypic%20heterogeneity%20in%20a%20clonal%20cell%20population%20in%20vitro&amp;author=D%20Stockholm&amp;author=R%20Benchaouir&amp;author=J%20Picot&amp;author=P%20Rameau&amp;author=TMA%20Neildez&amp;volume=2&amp;issue=4&amp;publication_year=2007&amp;pmid=17460761&amp;doi=10.1371/journal.pone.0000394&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref055">
<span class="label">55.</span><cite>Altschuler SJ, Wu LF. Cellular heterogeneity: do differences make a difference?. Cell. 2010;141(4):559–63. doi: 10.1016/j.cell.2010.04.033

</cite> [<a href="https://doi.org/10.1016/j.cell.2010.04.033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2918286/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20478246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Cellular%20heterogeneity:%20do%20differences%20make%20a%20difference?&amp;author=SJ%20Altschuler&amp;author=LF%20Wu&amp;volume=141&amp;issue=4&amp;publication_year=2010&amp;pages=559-63&amp;pmid=20478246&amp;doi=10.1016/j.cell.2010.04.033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref056">
<span class="label">56.</span><cite>Jaffe EA, Nachman RL, Becker CG, Minick CR. Culture of human endothelial cells derived from umbilical veins. Identification by morphologic and immunologic criteria. J Clin Invest. 1973;52(11):2745–56. doi: 10.1172/JCI107470

</cite> [<a href="https://doi.org/10.1172/JCI107470" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC302542/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/4355998/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Clin%20Invest&amp;title=Culture%20of%20human%20endothelial%20cells%20derived%20from%20umbilical%20veins.%20Identification%20by%20morphologic%20and%20immunologic%20criteria&amp;author=EA%20Jaffe&amp;author=RL%20Nachman&amp;author=CG%20Becker&amp;author=CR%20Minick&amp;volume=52&amp;issue=11&amp;publication_year=1973&amp;pages=2745-56&amp;pmid=4355998&amp;doi=10.1172/JCI107470&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref057">
<span class="label">57.</span><cite>Glen CM, Kemp ML, Voit EO. Agent-based modeling of morphogenetic systems: Advantages and challenges. PLoS Comput Biol. 2019;15(3):e1006577. doi: 10.1371/journal.pcbi.1006577

</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1006577" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6438454/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30921323/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Comput%20Biol&amp;title=Agent-based%20modeling%20of%20morphogenetic%20systems:%20Advantages%20and%20challenges&amp;author=CM%20Glen&amp;author=ML%20Kemp&amp;author=EO%20Voit&amp;volume=15&amp;issue=3&amp;publication_year=2019&amp;pmid=30921323&amp;doi=10.1371/journal.pcbi.1006577&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref058">
<span class="label">58.</span><cite>Bauer AL, Jackson TL, Jiang Y. A cell-based model exhibiting branching and anastomosis during tumor-induced angiogenesis. Biophys J. 2007;92:3105–21.
</cite> [<a href="https://doi.org/10.1529/biophysj.106.101501" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1852370/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17277180/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biophys%20J&amp;title=A%20cell-based%20model%20exhibiting%20branching%20and%20anastomosis%20during%20tumor-induced%20angiogenesis&amp;author=AL%20Bauer&amp;author=TL%20Jackson&amp;author=Y%20Jiang&amp;volume=92&amp;publication_year=2007&amp;pages=3105-21&amp;pmid=17277180&amp;doi=10.1529/biophysj.106.101501&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref059">
<span class="label">59.</span><cite>Qutub AA, Popel AS. Elongation, proliferation &amp; migration differentiate endothelial cell phenotypes and determine capillary sprouting. BMC Syst Biol. 2009;3:13. doi: 10.1186/1752-0509-3-13

</cite> [<a href="https://doi.org/10.1186/1752-0509-3-13" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2672076/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19171061/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Syst%20Biol&amp;title=Elongation,%20proliferation%20&amp;%20migration%20differentiate%20endothelial%20cell%20phenotypes%20and%20determine%20capillary%20sprouting&amp;author=AA%20Qutub&amp;author=AS%20Popel&amp;volume=3&amp;publication_year=2009&amp;pages=13&amp;pmid=19171061&amp;doi=10.1186/1752-0509-3-13&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref060">
<span class="label">60.</span><cite>Perfahl H, Byrne HM, Chen T, Estrella V, Alarcón T, Lapin A, et al. Multiscale modelling of vascular tumour growth in 3D: the roles of domain size and boundary conditions. PLoS One. 2011;6(4):e14790. doi: 10.1371/journal.pone.0014790

</cite> [<a href="https://doi.org/10.1371/journal.pone.0014790" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3076378/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21533234/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Multiscale%20modelling%20of%20vascular%20tumour%20growth%20in%203D:%20the%20roles%20of%20domain%20size%20and%20boundary%20conditions&amp;author=H%20Perfahl&amp;author=HM%20Byrne&amp;author=T%20Chen&amp;author=V%20Estrella&amp;author=T%20Alarc%C3%B3n&amp;volume=6&amp;issue=4&amp;publication_year=2011&amp;pmid=21533234&amp;doi=10.1371/journal.pone.0014790&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref061">
<span class="label">61.</span><cite>Walpole J, Mac Gabhann F, Peirce SM, Chappell JC. Agent-based computational model of retinal angiogenesis simulates microvascular network morphology as a function of pericyte coverage. Microcirc. 2017;24.</cite> [<a href="https://doi.org/10.1111/micc.12393" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5673505/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28791758/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microcirc&amp;title=Agent-based%20computational%20model%20of%20retinal%20angiogenesis%20simulates%20microvascular%20network%20morphology%20as%20a%20function%20of%20pericyte%20coverage&amp;author=J%20Walpole&amp;author=F%20Mac%20Gabhann&amp;author=SM%20Peirce&amp;author=JC%20Chappell&amp;volume=24&amp;publication_year=2017&amp;pmid=28791758&amp;doi=10.1111/micc.12393&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref062">
<span class="label">62.</span><cite>Song M, Finley SD. Mechanistic characterization of endothelial sprouting mediated by pro-angiogenic signaling. Microcirc. 2022;29(e12744).</cite> [<a href="https://doi.org/10.1111/micc.12744" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9285777/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34890488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Microcirc&amp;title=Mechanistic%20characterization%20of%20endothelial%20sprouting%20mediated%20by%20pro-angiogenic%20signaling&amp;author=M%20Song&amp;author=SD%20Finley&amp;volume=29&amp;publication_year=2022&amp;pmid=34890488&amp;doi=10.1111/micc.12744&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref063">
<span class="label">63.</span><cite>Walpole J, Chappell JC, Cluceru JG, Mac Gabhann F, Bautch VL, Peirce SM. Agent-based model of angiogenesis simulates capillary sprout initiation in multicellular networks. Integr Biol (Camb). 2015;7(9):987–97. doi: 10.1039/c5ib00024f

</cite> [<a href="https://doi.org/10.1039/c5ib00024f" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4558383/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26158406/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Integr%20Biol%20(Camb)&amp;title=Agent-based%20model%20of%20angiogenesis%20simulates%20capillary%20sprout%20initiation%20in%20multicellular%20networks&amp;author=J%20Walpole&amp;author=JC%20Chappell&amp;author=JG%20Cluceru&amp;author=F%20Mac%20Gabhann&amp;author=VL%20Bautch&amp;volume=7&amp;issue=9&amp;publication_year=2015&amp;pages=987-97&amp;pmid=26158406&amp;doi=10.1039/c5ib00024f&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref064">
<span class="label">64.</span><cite>Chappell JC, Cluceru JG, Nesmith JE, Mouillesseaux KP, Bradley VB, Hartland CM, et al. Flt-1 (VEGFR-1) coordinates discrete stages of blood vessel formation. Cardiovasc Res. 2016;111(1):84–93. doi: 10.1093/cvr/cvw091

</cite> [<a href="https://doi.org/10.1093/cvr/cvw091" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4909163/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27142980/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cardiovasc%20Res&amp;title=Flt-1%20(VEGFR-1)%20coordinates%20discrete%20stages%20of%20blood%20vessel%20formation&amp;author=JC%20Chappell&amp;author=JG%20Cluceru&amp;author=JE%20Nesmith&amp;author=KP%20Mouillesseaux&amp;author=VB%20Bradley&amp;volume=111&amp;issue=1&amp;publication_year=2016&amp;pages=84-93&amp;pmid=27142980&amp;doi=10.1093/cvr/cvw091&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref065">
<span class="label">65.</span><cite>Silva JAF, Qi X, Grant MB, Boulton ME. Spatial and temporal VEGF receptor intracellular trafficking in microvascular and macrovascular endothelial cells. Sci Rep. 2021;11(1):17400. doi: 10.1038/s41598-021-96964-7

</cite> [<a href="https://doi.org/10.1038/s41598-021-96964-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8405636/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34462507/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci%20Rep&amp;title=Spatial%20and%20temporal%20VEGF%20receptor%20intracellular%20trafficking%20in%20microvascular%20and%20macrovascular%20endothelial%20cells&amp;author=JAF%20Silva&amp;author=X%20Qi&amp;author=MB%20Grant&amp;author=ME%20Boulton&amp;volume=11&amp;issue=1&amp;publication_year=2021&amp;pages=17400&amp;pmid=34462507&amp;doi=10.1038/s41598-021-96964-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref066">
<span class="label">66.</span><cite>Alon U. An Introduction to Systems Biology. Chapman and Hall/CRC. 2006. doi: 10.1201/9781420011432</cite> [<a href="https://doi.org/10.1201/9781420011432" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=An%20Introduction%20to%20Systems%20Biology.%20Chapman%20and%20Hall/CRC&amp;author=U%20Alon&amp;publication_year=2006&amp;doi=10.1201/9781420011432&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref067">
<span class="label">67.</span><cite>Rahimi N, Golde TE, Meyer RD. Identification of ligand-induced proteolytic cleavage and ectodomain shedding of VEGFR-1/FLT1 in leukemic cancer cells. Cancer Res. 2009;69(6):2607–14. doi: 10.1158/0008-5472.CAN-08-2905

</cite> [<a href="https://doi.org/10.1158/0008-5472.CAN-08-2905" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2674781/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19276374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer%20Res&amp;title=Identification%20of%20ligand-induced%20proteolytic%20cleavage%20and%20ectodomain%20shedding%20of%20VEGFR-1/FLT1%20in%20leukemic%20cancer%20cells&amp;author=N%20Rahimi&amp;author=TE%20Golde&amp;author=RD%20Meyer&amp;volume=69&amp;issue=6&amp;publication_year=2009&amp;pages=2607-14&amp;pmid=19276374&amp;doi=10.1158/0008-5472.CAN-08-2905&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref068">
<span class="label">68.</span><cite>Roeckl W, Hecht D, Sztajer H, Waltenberger J, Yayon A, Weich HA. Differential binding characteristics and cellular inhibition by soluble VEGF receptors 1 and 2. Exp Cell Res. 1998;241(1):161–70. doi: 10.1006/excr.1998.4039

</cite> [<a href="https://doi.org/10.1006/excr.1998.4039" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9633524/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Exp%20Cell%20Res&amp;title=Differential%20binding%20characteristics%20and%20cellular%20inhibition%20by%20soluble%20VEGF%20receptors%201%20and%202&amp;author=W%20Roeckl&amp;author=D%20Hecht&amp;author=H%20Sztajer&amp;author=J%20Waltenberger&amp;author=A%20Yayon&amp;volume=241&amp;issue=1&amp;publication_year=1998&amp;pages=161-70&amp;pmid=9633524&amp;doi=10.1006/excr.1998.4039&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref069">
<span class="label">69.</span><cite>Park M, Lee ST. The fourth immunoglobulin-like loop in the extracellular domain of FLT-1, a VEGF receptor, includes a major heparin-binding site. Biochem Biophys Res Commun. 1999;264(3):730–4. doi: 10.1006/bbrc.1999.1580

</cite> [<a href="https://doi.org/10.1006/bbrc.1999.1580" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/10544000/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochem%20Biophys%20Res%20Commun&amp;title=The%20fourth%20immunoglobulin-like%20loop%20in%20the%20extracellular%20domain%20of%20FLT-1,%20a%20VEGF%20receptor,%20includes%20a%20major%20heparin-binding%20site&amp;author=M%20Park&amp;author=ST%20Lee&amp;volume=264&amp;issue=3&amp;publication_year=1999&amp;pages=730-4&amp;pmid=10544000&amp;doi=10.1006/bbrc.1999.1580&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref070">
<span class="label">70.</span><cite>Sela S, et al. Local retention versus systemic release of soluble VEGF receptor-1 are mediated by heparin-binding and regulated by heparanase. Circ Res. 2011;108:1063–70.
</cite> [<a href="https://doi.org/10.1161/CIRCRESAHA.110.239665" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21415391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Circ%20Res&amp;title=Local%20retention%20versus%20systemic%20release%20of%20soluble%20VEGF%20receptor-1%20are%20mediated%20by%20heparin-binding%20and%20regulated%20by%20heparanase&amp;author=S%20Sela&amp;volume=108&amp;publication_year=2011&amp;pages=1063-70&amp;pmid=21415391&amp;doi=10.1161/CIRCRESAHA.110.239665&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref071">
<span class="label">71.</span><cite>Orecchia A, Lacal PM, Schietroma C, Morea V, Zambruno G, Failla CM. Vascular endothelial growth factor receptor-1 is deposited in the extracellular matrix by endothelial cells and is a ligand for the alpha 5 beta 1 integrin. J Cell Sci. 2003;116(Pt 17):3479–89. doi: 10.1242/jcs.00673

</cite> [<a href="https://doi.org/10.1242/jcs.00673" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12865438/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J%20Cell%20Sci&amp;title=Vascular%20endothelial%20growth%20factor%20receptor-1%20is%20deposited%20in%20the%20extracellular%20matrix%20by%20endothelial%20cells%20and%20is%20a%20ligand%20for%20the%20alpha%205%20beta%201%20integrin&amp;author=A%20Orecchia&amp;author=PM%20Lacal&amp;author=C%20Schietroma&amp;author=V%20Morea&amp;author=G%20Zambruno&amp;volume=116&amp;publication_year=2003&amp;pages=3479-89&amp;pmid=12865438&amp;doi=10.1242/jcs.00673&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="pcbi.1013324.ref072">
<span class="label">72.</span><cite>Reily C, Stewart TJ, Renfrow MB, Novak J. Glycosylation in health and disease. Nat Rev Nephrol. 2019;15(6):346–66. doi: 10.1038/s41581-019-0129-4

</cite> [<a href="https://doi.org/10.1038/s41581-019-0129-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6590709/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30858582/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat%20Rev%20Nephrol&amp;title=Glycosylation%20in%20health%20and%20disease&amp;author=C%20Reily&amp;author=TJ%20Stewart&amp;author=MB%20Renfrow&amp;author=J%20Novak&amp;volume=15&amp;issue=6&amp;publication_year=2019&amp;pages=346-66&amp;pmid=30858582&amp;doi=10.1038/s41581-019-0129-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section></section><article class="sub-article" id="pcbi.1013324.r001"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS Comput Biol. doi: <a href="https://doi.org/10.1371/journal.pcbi.1013324.r001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pcbi.1013324.r001</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 0</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mendes%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Pedro Mendes</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Pedro Mendes</span></h3>
<div>Section Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mendes%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pedro Mendes</span></a>
</div>
</div>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Okada%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Mariko Okada</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Mariko Okada</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Okada%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mariko Okada</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.c" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.c" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.c" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Pedro Mendes</span></strong>: <span class="role">Section Editor</span>
</div>
<div>
<strong class="contrib"><span class="name western">Mariko Okada</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.c" class="d-panel p" style="display: none">
<div>© 2025 Mendes, Okada</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>17 Apr 2025</em>
</p>
<p>PCOMPBIOL-D-25-00322</p>
<p>Mechanistic computational modeling of sFLT1 secretion dynamics</p>
<p>PLOS Computational Biology</p>
<p>Dear Dr. Amy Gill,</p>
<p>Thank you for submitting your manuscript to PLOS Computational Biology. After careful consideration, we feel that it has merit but does not fully meet PLOS Computational Biology's publication criteria as it currently stands. Therefore, we invite you to submit a revised version of the manuscript that addresses the points raised during the review process.</p>
<p>Please submit your revised manuscript within 30 days. If you will need more time than this to complete your revisions, please reply to this message or contact the journal office at ploscompbiol@plos.org. When you're ready to submit your revision, log on to <a href="https://www.editorialmanager.com/pcompbiol/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.editorialmanager.com/pcompbiol/</a> and select the 'Submissions Needing Revision' folder to locate your manuscript file.</p>
<p>Please include the following items when submitting your revised manuscript:</p>
<p>* A rebuttal letter that responds to each point raised by the editor and reviewer(s). You should upload this letter as a separate file labeled 'Response to Reviewers'. This file does not need to include responses to formatting updates and technical items listed in the 'Journal Requirements' section below.</p>
<p>* A marked-up copy of your manuscript that highlights changes made to the original version. You should upload this as a separate file labeled 'Revised Manuscript with Track Changes'.</p>
<p>* An unmarked version of your revised paper without tracked changes. You should upload this as a separate file labeled 'Manuscript'.</p>
<p>If you would like to make changes to your financial disclosure, competing interests statement, or data availability statement, please make these updates within the submission form at the time of resubmission. Guidelines for resubmitting your figure files are available below the reviewer comments at the end of this letter.</p>
<p>We look forward to receiving your revised manuscript.</p>
<p>Kind regards,</p>
<p>Mariko Okada</p>
<p>Academic Editor</p>
<p>PLOS Computational Biology</p>
<p>Pedro Mendes</p>
<p>Section Editor</p>
<p>PLOS Computational Biology</p>
<p>
<strong>Journal Requirements:</strong>
</p>
<p>1) We ask that a manuscript source file is provided at Revision. Please upload your manuscript file as a .doc, .docx, .rtf or .tex. If you are providing a .tex file, please upload it under the item type u2018LaTeX Source Fileu2019 and leave your .pdf version as the item type u2018Manuscriptu2019.</p>
<p>2) Please upload all main figures as separate Figure files in .tif or .eps format. For more information about how to convert and format your figure files please see our guidelines: </p>
<p>
<a href="https://journals.plos.org/ploscompbiol/s/figures" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.plos.org/ploscompbiol/s/figures</a>
</p>
<p>3) We have noticed that you have uploaded Supporting Information files, but you have not included a list of legends. Please add a full list of legends for your Supporting Information files after the references list.</p>
<p>4) Some material included in your submission may be copyrighted. According to PLOSu2019s copyright policy, authors who use figures or other material (e.g., graphics, clipart, maps) from another author or copyright holder must demonstrate or obtain permission to publish this material under the Creative Commons Attribution 4.0 International (CC BY 4.0) License used by PLOS journals. Please closely review the details of PLOSu2019s copyright requirements here: PLOS Licenses and Copyright. If you need to request permissions from a copyright holder, you may use PLOS's Copyright Content Permission form.</p>
<p>Please respond directly to this email and provide any known details concerning your material's license terms and permissions required for reuse, even if you have not yet obtained copyright permissions or are unsure of your material's copyright compatibility. Once you have responded and addressed all other outstanding technical requirements, you may resubmit your manuscript within Editorial Manager. </p>
<p>Potential Copyright Issues:</p>
<p>i) Figures 1, and 2: We note that the figures are created through BioRender. Please confirm that you hold a Premium account and provide a pdf copy of the CC BY 4.0 License as provided by BioRender. For instructions on how to generate a CC BY 4.0 license for your figure, please see the guidelines here: <a href="https://help.biorender.com/hc/en-gb/articles/21282341238045-Publishing-in-open-access-resources" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://help.biorender.com/hc/en-gb/articles/21282341238045-Publishing-in-open-access-resources. </a></p>
<p>If you are using the free assets from BioRender, we are unable to publish these images as they are licenced under a stricter licence than CC BY 4.0. In this case we ask you to remove the BioRender images and replace them with open source alternatives.</p>
<p>See these open source resources you may use to replace images / clip-art:</p>
<p>- <a href="https://bioart.niaid.nih.gov/%C2%A0" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bioart.niaid.nih.gov/ </a></p>
<p>- <a href="https://bioicons.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bioicons.com/</a></p>
<p>- <a href="https://healthicons.org/%C2%A0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://healthicons.org/ </a></p>
<p>- <a href="https://scidraw.io/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://scidraw.io/</a></p>
<p>- <a href="https://reactome.org/icon-lib" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://reactome.org/icon-lib</a></p>
<p>- <a href="https://www.phylopic.org/images" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.phylopic.org/images</a></p>
<p>5) Please provide a completed 'Competing Interests' statement, including any COIs declared by your co-authors. If you have no competing interests to declare, please state "The authors have declared that no competing interests exist".</p>
<p>
<strong>Reviewers' comments:</strong>
</p>
<p>Reviewer's Responses to Questions</p>
<p>Reviewer #1: The authors describe 8 mechanistic models for the production and secretion of sFLT1 involved in VEGF signaling. Comparing these models to existing experimental sFLT1 data, the authors identify the appropriate models and parameters. The manuscript is clearly written and describes the modeling process, choice of models, parameter variations, the choice of the model best fitting the data, study conclusions and limitations. I especially praise the full documentation of the models available on GitHub.</p>
<p>There are a few minor comments:</p>
<p>- The use of Figures is non-sequential, with Figure 1A first, then Figure 2A, then Figure 1B, etc. Figure 3A should be also cited on page 10, when the authors start talking about candidate models.</p>
<p>- What are “model A” and “model B” in Supplemental Table S1? Should be M1 and M2.</p>
<p>- “summary data were extracted from published figures using the Figure Calibration function in ImageJ” – can you provide the data in the tabular form. Please be more specific about which figure and which panel where used for data extraction.</p>
<p>- Please refer to specific tables/figures in Jung, Horning and Kinghorn studies, and repeat this data in supplementary material, with captions stating what was omitted.</p>
<p>- What is “custom cost function” on page 13? Please remind the user of the values of n and k for each M1-M8 models.</p>
<p>- Figure 3B – would be useful to name columns M1-M8.</p>
<p>- Page 24: “we chemically inhibited each parameter individually by 90% and investigated how inhibition affects intracellular and extracellular sFLT1 over time” – perhaps the authors meant to say they simulated inhibition of each individual parameter by 90%.</p>
<p>- Page 25: “lysosomal degradation inhibitor chloroquine” and "the Golgi inhibitor brefeldin” – it’s the first time these inhibitors are introduced, authors need to put them into the context of models M1-M8.</p>
<p>Reviewer #2: This paper describes the development and analysis of a computational model of the secretion dynamics of a soluble form of the VEGF receptor called sFLT1. Data from 5 different experimental studies of the process covering two different experimental scenarios, constitutive and pulse-chase, are integrated into a single calibration set that is used to calibrate and evaluate a family of computational models that contain a set of six common processes and encompassing zero to three additional mechanisms, with the goal of identifying a minimal model that best captures the data. This goal is accomplished by comparing each best fit model using the corrected form of Akaike's Information Criterion (AIC_c), which chooses a model that includes only one of the three proposed additional mechanisms, maturation delay, which is modeled with the addition of a single time delay parameter using delayed differential equations. The paper presents a number of additional analyses of the selected model, including parameter identifiability and sensitivity analysis, which is extended to explore possible mechanisms for chemical and genetic perturbation of the system. A key result of this analysis is that while some of these effects are well-predicted by the model, others, notably those that are sensitive to intracellular trafficking dynamics, are not. These results provide a strong rationale for future development of models, presumably based on an expanded set of experimental measurements, that more accurately capture these dynamics. Overall, the paper is well-written and clear, and I have just minor comments that I think may improve the manuscript if they are addressed prior to publication. I congratulate the authors on their thorough and rigorous treatment of this system.</p>
<p>Minor comments</p>
<p>A general comment is that there are several instances of statements where factual statements are made without a corresponding reference. Examples include:</p>
<p>- p. 27: "...the targets of inhibition can affect the trafficking of many proteins, which may themselves have a role in sFLT1 trafficking."</p>
<p>- p. 34: " Importantly, the chemical and genetic inhibitors explored are not specific to sFLT1, but regulate trafficking for many other proteins, some of which may themselves modulate sFLT1 synthesis and secretion."</p>
<p>- p. 36: " Even in monolayer culture of single cell lines, cells are not identical, as the stochasticity of biochemical reactions leads to natural gradients and fluctuations."</p>
<p>p. 13. There is a typo in Eq. 5: log is repeated in the first term of the equation. Also, it is more standard to provide the AICc formula as an equation, in other words to write: "AIC_c = n \log (C/n) + 2k + ..." instead of just giving the formula. (Note also that C should be capitalized in the formula.)</p>
<p>p. 13. " is the cost of a given model calculated using the custom cost function." I think it would be helpful to have an explicit formula for C, referred to in Eq. 5., especially because of the use of the word "custom" here, the reader may think they have missed something.</p>
<p>p. 13. In describing use of AIC_c it should be mentioned that there is an additional assumption here that the variance is constant over all the experimental data points and that the additional model parameter is the value of the variance.</p>
<p>p. 19. In Figure 3C labels M4 and M7 are switched.</p>
<p>p. 24. "...while extracellular sFLT1 initially increases before declining to a new steady state within 48h." This trend cannot be seen on the graph - maybe show as an inset or in a separate graph.</p>
<p>p. 30ff. In the Discussion section it would be helpful to refer to specific figures when reviewing the key results of the paper that are being discussed to increase specificity of the points being made. For example, "A key first step in our study was comparing different mechanistic models for sFLT1</p>
<p>secretion [(Fig 2A and 3A)]." "As a result, model M2 - a DDE-based model that included maturation delay but not internalization or</p>
<p>production decay - was identified as the best model and was used for the remainder of our study [(Fig 3B,C)]." etc.</p>
<p>Fig. 3C - in the combined plot of Cost and AIC there is overlap of the y-axis tick labels for the top and bottom panels. It would help to remove one of the values</p>
<p>Fig. 5D, caption. dX/dt = f(prod)-f(secr) - f(ideg), should be dI/dt.</p>
<p>Fig. 5E, caption, duplicates 5D caption - shows component fluxes not total.</p>
<p>**********</p>
<p>
<strong>Have the authors made all data and (if applicable) computational code underlying the findings in their manuscript fully available?</strong>
</p>
<p>The <a href="https://journals.plos.org/ploscompbiol/s/materials-and-software-sharing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PLOS Data policy</a> requires authors to make all data and code underlying the findings described in their manuscript fully available without restriction, with rare exception (please refer to the Data Availability Statement in the manuscript PDF file). The data and code should be provided as part of the manuscript or its supporting information, or deposited to a public repository. For example, in addition to summary statistics, the data points behind means, medians and variance measures should be available. If there are restrictions on publicly sharing data or code —e.g. participant privacy or use of data from a third party—those must be specified.</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>PLOS authors have the option to publish the peer review history of their article (<a href="https://journals.plos.org/ploscompbiol/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a> ). If published, this will include your full peer review and any attached files.</p>
<p>If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p><strong>Do you want your identity to be public for this peer review?</strong> For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a> .</p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: No</p>
<p>[NOTE: If reviewer comments were submitted as an attachment file, they will be attached to this email and accessible via the submission site. Please log into your account, locate the manuscript record, and check for the action link "View Attachments". If this link does not appear, there are no attachment files.]</p>
<p>
<strong>Figure resubmission:</strong>
</p>
<p>While revising your submission, please upload your figure files to the Preflight Analysis and Conversion Engine (PACE) digital diagnostic tool, <a href="https://pacev2.apexcovantage.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://pacev2.apexcovantage.com/</a>. PACE helps ensure that figures meet PLOS requirements. To use PACE, you must first register as a user. Registration is free. Then, login and navigate to the UPLOAD tab, where you will find detailed instructions on how to use the tool. If you encounter any issues or have any questions when using PACE, please email PLOS at figures@plos.org. Please note that Supporting Information files do not need this step. If there are other versions of figure files still present in your submission file inventory at resubmission, please replace them with the PACE-processed versions.</p>
<p>
<strong>Reproducibility:</strong>
</p>
<p>To enhance the reproducibility of your results, we recommend that authors of applicable studies deposit laboratory protocols in protocols.io, where a protocol can be assigned its own identifier (DOI) such that it can be cited independently in the future. Additionally, PLOS ONE offers an option to publish peer-reviewed clinical study protocols. Read more information on sharing protocols at <a href="https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://plos.org/protocols?utm_medium=editorial-email&amp;utm_source=authorletters&amp;utm_campaign=protocols</a></p></section></article><article class="sub-article" id="pcbi.1013324.r002"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS Comput Biol. 2025 Aug 18;21(8):e1013324. doi: <a href="https://doi.org/10.1371/journal.pcbi.1013324.r002" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pcbi.1013324.r002</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Author response to Decision Letter 1</h1></hgroup><ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="anp_a.d" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a.d" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="anp_a.d" class="d-panel p" style="display: none"><div class="notes p"><section id="historyfront-stub2" class="history"><p>Collection date 2025 Aug.</p></section></div></div>
<div id="clp_a.d" class="d-panel p" style="display: none"><div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div></div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>18 Jun 2025</em>
</p>
<section class="sm xbox font-sm" id="pcbi.1013324.s022"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response to Reviewers.pdf</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s022.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s022.pdf</a><sup> (248.5KB, pdf) </sup>
</div></div></section></section></article><article class="sub-article" id="pcbi.1013324.r003"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS Comput Biol. doi: <a href="https://doi.org/10.1371/journal.pcbi.1013324.r003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pcbi.1013324.r003</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Decision Letter 1</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mendes%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Pedro Mendes</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Pedro Mendes</span></h3>
<div>Section Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mendes%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pedro Mendes</span></a>
</div>
</div>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Okada%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Mariko Okada</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Mariko Okada</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Okada%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mariko Okada</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.e" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.e" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.e" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Pedro Mendes</span></strong>: <span class="role">Section Editor</span>
</div>
<div>
<strong class="contrib"><span class="name western">Mariko Okada</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.e" class="d-panel p" style="display: none">
<div>© 2025 Mendes, Okada</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>
<em>10 Jul 2025</em>
</p>
<p>Dear Gill,</p>
<p>We are pleased to inform you that your manuscript 'Mechanistic computational modeling of sFLT1 secretion dynamics' has been provisionally accepted for publication in PLOS Computational Biology.</p>
<p>Before your manuscript can be formally accepted you will need to complete some formatting changes, which you will receive in a follow up email. A member of our team will be in touch with a set of requests.</p>
<p>Please note that your manuscript will not be scheduled for publication until you have made the required changes, so a swift response is appreciated.</p>
<p>IMPORTANT: The editorial review process is now complete. PLOS will only permit corrections to spelling, formatting or significant scientific errors from this point onwards. Requests for major changes, or any which affect the scientific understanding of your work, will cause delays to the publication date of your manuscript.</p>
<p>Should you, your institution's press office or the journal office choose to press release your paper, you will automatically be opted out of early publication. We ask that you notify us now if you or your institution is planning to press release the article. All press must be co-ordinated with PLOS.</p>
<p>Thank you again for supporting Open Access publishing; we are looking forward to publishing your work in PLOS Computational Biology. </p>
<p>Best regards,</p>
<p>Mariko Okada, PhD</p>
<p>Academic Editor</p>
<p>PLOS Computational Biology</p>
<p>Pedro Mendes</p>
<p>Section Editor</p>
<p>PLOS Computational Biology</p>
<p>***********************************************************</p>
<p>Reviewer's Responses to Questions</p>
<p>
<strong>Comments to the Authors:</strong>
</p>
<p>
<strong>Please note here if the review is uploaded as an attachment.</strong>
</p>
<p>Reviewer #1: The authors addressed all revisions properly.</p>
<p>Reviewer #2: The authors have thoroughly addressed the critiques raised by both reviewers.</p>
<p>**********</p>
<p>
<strong>Have the authors made all data and (if applicable) computational code underlying the findings in their manuscript fully available?</strong>
</p>
<p>The <a href="https://journals.plos.org/ploscompbiol/s/materials-and-software-sharing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">PLOS Data policy</a> requires authors to make all data and code underlying the findings described in their manuscript fully available without restriction, with rare exception (please refer to the Data Availability Statement in the manuscript PDF file). The data and code should be provided as part of the manuscript or its supporting information, or deposited to a public repository. For example, in addition to summary statistics, the data points behind means, medians and variance measures should be available. If there are restrictions on publicly sharing data or code —e.g. participant privacy or use of data from a third party—those must be specified.</p>
<p>Reviewer #1: Yes</p>
<p>Reviewer #2: Yes</p>
<p>**********</p>
<p>PLOS authors have the option to publish the peer review history of their article (<a href="https://journals.plos.org/ploscompbiol/s/editorial-and-peer-review-process#loc-peer-review-history" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">what does this mean?</a> ). If published, this will include your full peer review and any attached files.</p>
<p>If you choose “no”, your identity will remain anonymous but your review may still be made public.</p>
<p><strong>Do you want your identity to be public for this peer review?</strong> For information about this choice, including consent withdrawal, please see our <a href="https://www.plos.org/privacy-policy" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Privacy Policy</a> .</p>
<p>Reviewer #1: No</p>
<p>Reviewer #2: No</p></section></article><article class="sub-article" id="pcbi.1013324.r004"><section class="pmc-layout__citation font-secondary font-xs"><div>PLoS Comput Biol. doi: <a href="https://doi.org/10.1371/journal.pcbi.1013324.r004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1371/journal.pcbi.1013324.r004</a>
</div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Acceptance letter</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mendes%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Pedro Mendes</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Pedro Mendes</span></h3>
<div>Section Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mendes%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Pedro Mendes</span></a>
</div>
</div>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Okada%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Mariko Okada</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Mariko Okada</span></h3>
<div>Academic Editor</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Okada%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Mariko Okada</span></a>
</div>
</div>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a.f" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="clp_a.f" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a.f" class="d-panel p" style="display: none">
<h4 class="font-secondary">Roles</h4>
<div class="p">
<strong class="contrib"><span class="name western">Pedro Mendes</span></strong>: <span class="role">Section Editor</span>
</div>
<div>
<strong class="contrib"><span class="name western">Mariko Okada</span></strong>: <span class="role">Academic Editor</span>
</div>
</div>
<div id="clp_a.f" class="d-panel p" style="display: none">
<div>© 2025 Mendes, Okada</div>
<p>This is an open access article distributed under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution License</a> , which permits unrestricted use, distribution, and reproduction in any medium, provided the original author and source are credited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div></div>
</div></section><section class="body sub-article-body"><hr class="headless">
<p>PCOMPBIOL-D-25-00322R1</p>
<p>Mechanistic computational modeling of sFLT1 secretion dynamics</p>
<p>Dear Dr Gill,</p>
<p>I am pleased to inform you that your manuscript has been formally accepted for publication in PLOS Computational Biology. Your manuscript is now with our production department and you will be notified of the publication date in due course.</p>
<p>The corresponding author will soon be receiving a typeset proof for review, to ensure errors have not been introduced during production. Please review the PDF proof of your manuscript carefully, as this is the last chance to correct any errors. Please note that major changes, or those which affect the scientific understanding of the work, will likely cause delays to the publication date of your manuscript.</p>
<p>Soon after your final files are uploaded, unless you have opted out, the early version of your manuscript will be published online. The date of the early version will be your article's publication date. The final article will be published to the same URL, and all versions of the paper will be accessible to readers.</p>
<p>You will receive an invoice from PLOS for your publication fee after your manuscript has reached the completed accept phase. If you receive an email requesting payment before acceptance or for any other service, this may be a phishing scheme. Learn how to identify phishing emails and protect your accounts at <a href="https://explore.plos.org/phishing" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://explore.plos.org/phishing</a>.</p>
<p>Thank you again for supporting PLOS Computational Biology and open-access publishing. We are looking forward to publishing your work!</p>
<p>With kind regards,</p>
<p>Zsofia Freund</p>
<p>PLOS Computational Biology | Carlyle House, Carlyle Road, Cambridge CB4 3DN | United Kingdom ploscompbiol@plos.org | Phone +44 (0) 1223-442824 | ploscompbiol.org | @PLOSCompBiol</p></section></article><article class="sub-article" id="_ad93_"><section class="pmc-layout__citation font-secondary font-xs"><div></div></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Associated Data</h1></hgroup><ul class="d-buttons inline-list"></ul>
<div class="d-panels font-secondary-light"></div>
<div></div>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
</div></section><section class="body sub-article-body"><section id="_adsm93_" lang="en" class="supplementary-materials"><h2 class="pmc_sec_title">Supplementary Materials</h2>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p">
<span>S1 Data. Experimental data extracted from past studies.</span><p>(XLSX)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s001.xlsx" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s001.xlsx</a><sup> (14.5KB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p">
<span>S1 Text. Supplemental methods.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s002.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s002.pdf</a><sup> (102.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p">
<span>S2 Text. Supplemental results.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s003.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s003.pdf</a><sup> (76.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p">
<span>S3 Text. Supplemental references.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s004.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s004.pdf</a><sup> (63.5KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<span>S1 Table. Complete set of equations for candidate models.</span><p>Note that model M1 is equivalent to the “ODE-based model” and model M2 is equivalent to the “DDE-based model” described above.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s005.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s005.pdf</a><sup> (60.7KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="caption p">
<span>S2 Table. Characteristics of sFLT1 datasets used for mechanistic model optimization.</span><p>Secretion scenario: C = constitutive, P = pulse-chase; Assay: E = ELISA, W = Western blot/ autoradiography; Time points: <sup>*</sup> = flattened by normalization, <sup>†</sup> = omitted (see Methods for rationale).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s006.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s006.pdf</a><sup> (99KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="caption p">
<span>S3 Table. Sampling distributions for initial parameter values and bounds for final parameter values during mechanistic model optimization.</span><p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s007.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s007.pdf</a><sup> (54.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="caption p">
<span>S1 Fig. Dynamics and phase plane behavior of an example ODE model solution.</span><p>(A) Solution of the ODE model as a function of time showing intracellular (<em>I</em>) and extracellular (<em>X</em>) sFLT1 protein dynamics in response to step changes in sFLT1 production (<em>α</em>). Time is normalized to <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mn>50</mn><mi>_</mi><mi>X</mi></mrow></msub></mrow></math></span>, the half-life of <em>X</em> (<a href="#pcbi.1013324.e004" class="usa-link">Eq. 4</a>). <em>I, X</em> are normalized to their steady state values <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub><mo>,</mo><mtext> </mtext><msub><mrow><mi>X</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span> (<a href="#pcbi.1013324.e001" class="usa-link">Eq. 1</a>–<a href="#pcbi.1013324.e002" class="usa-link">2</a>). (B) Phase portrait of the shown ODE model solution. Points are evenly spaced every time unit and colored by time as in panel A. Arrowheads indicate the direction of increasing time starting with <em>t = 0</em> in the lower left. The dashed identity line represents potential steady states (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mi>I</mi><mo>=</mo><mfrac><mrow><mi>β</mi></mrow><mrow><mi>δ</mi></mrow></mfrac><mi>·</mi><mi>X</mi></mrow></math></span>); total sFLT1 increases in regions above this line and decreases below. Parameter values: α = 10,000, β = 0.1, γ = 0.1, δ = 0.1. For discussion of this figure, see <a href="#pcbi.1013324.s003" class="usa-link">S2 Text</a>.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s008.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s008.pdf</a><sup> (307.4KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material9_reqid_"><div class="caption p">
<span>S2 Fig. Dynamics and phase plane behavior of an example DDE model solution.</span><p>(A) Solution of the DDE model as a function of time showing intracellular (<em>I</em>) and extracellular (<em>X</em>) sFLT1 protein dynamics in response to step changes in sFLT1 production (α). Time is normalized to <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>T</mi></mrow><mrow><mn>50</mn><mi>_</mi><mi>X</mi></mrow></msub></mrow></math></span>, the half-life of <em>X</em> (<a href="#pcbi.1013324.e004" class="usa-link">Eq. 4</a>). <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mi>I</mi><mo>,</mo><mi>X</mi></mrow></math></span> are normalized to their steady state values <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub><mo>,</mo><mtext> </mtext><msub><mrow><mi>X</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span> (<a href="#pcbi.1013324.e001" class="usa-link">Eq. 1</a>–<a href="#pcbi.1013324.e002" class="usa-link">2</a>). (B) Phase portrait of the shown DDE model solution. Points are evenly spaced every two time units and colored by time as in panel A. Arrowheads indicate the direction of increasing time starting with t = 0 in the lower left. The dashed identity line represents potential steady states <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mi>I</mi><mo>=</mo><mfrac><mrow><mi>β</mi></mrow><mrow><mi>δ</mi></mrow></mfrac><mi>·</mi><mi>X</mi><mo stretchy="false">)</mo></mrow></math></span>; total sFLT1 increases in regions above this line and decreases below. Parameter values: <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mi>α</mi><mo>=</mo><mn>10</mn><mo>,</mo><mn>000</mn></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mi>β</mi><mo>=</mo><mn>0.1</mn></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mi>γ</mi><mo>=</mo><mn>0.1</mn></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mi>δ</mi><mo>=</mo><mn>0.1</mn></mrow></math></span>, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>τ</mi><mo>=</mo><mi>T</mi></mrow><mrow><mn>50</mn><mi>_</mi><mi>X</mi></mrow></msub></mrow></math></span>. (C) Example DDE solutions as a function of time and (D) as a phase portrait for maturation delays <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>τ</mi><mo>=</mo><mo stretchy="false">(</mo><mn>0</mn><mo>,</mo><mtext> </mtext><mn>0.5</mn><mo>,</mo><mtext> </mtext><mn>1</mn><mo>,</mo><mtext> </mtext><mn>1.5</mn><mo>,</mo><mtext> </mtext><mn>2</mn><mo stretchy="false">)</mo><mi>·</mi><mi>T</mi></mrow><mrow><mn>50</mn><mi>_</mi><mi>X</mi></mrow></msub></mrow></math></span>. For discussion of this figure, see <a href="#pcbi.1013324.s003" class="usa-link">S2 Text</a>.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s009.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s009.pdf</a><sup> (669.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material10_reqid_"><div class="caption p">
<span>S3 Fig. Visual predictive checks of candidate models of sFLT1 secretion.</span><p>Comparison of experimental data (points) and simulated time courses (lines, <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mi>n</mi><mo>=</mo><mn>100</mn></mrow></math></span> per plot) of (A, C) extracellular and (B, D) intracellular sFLT1 for (A, B) pulse-chase secretion and (C, D) constitutive secretion cases using eight distinct candidate models as described in Fig 3A and <a href="#pcbi.1013324.s005" class="usa-link">S1 Table</a>. In pulse-chase plots, the highlighted region (P) marks the 20-minute pulse. <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mi>X</mi><mo>/</mo><msub><mrow><mi>X</mi></mrow><mrow><mn>8</mn><mi>h</mi></mrow></msub></mrow></math></span>: extracellular sFLT1 normalized to its value at 8h; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mi>X</mi><mo>/</mo><msub><mrow><mi>X</mi></mrow><mrow><mn>24</mn><mi>h</mi></mrow></msub></mrow></math></span>: extracellular sFLT1 normalized to its value at 24h; <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mi>I</mi><mo>/</mo><msub><mrow><mi>I</mi></mrow><mrow><mn>0</mn><mi>h</mi></mrow></msub></mrow></math></span>: intracellular sFLT1 normalized to its value at <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mi>t</mi><mo>=</mo><mn>0</mn></mrow></math></span>.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s010.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s010.pdf</a><sup> (536.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material11_reqid_"><div class="caption p">
<span>S4 Fig. Filtering DDE optimization runs by cost.</span><p>(A) Visual predictive checks of unfiltered optimization runs (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mi>n</mi><mo>=</mo><mn>1000</mn></mrow></math></span>) from the delay differential equation (DDE) model. Many runs with high cost have not converged to follow the experimental data. (B) Cost distribution of raw (left) and filtered (right) optimization runs of the DDE model. The red lines indicate the cost cutoff of 10% above the minimum cost. (C) Density plots of raw and filtered initial parameter values for the DDE model. Raw (yellow) distributions include all attempted optimizations <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mo stretchy="false">(</mo><mi>n</mi><mo>=</mo><mn>1000</mn><mo stretchy="false">)</mo></mrow></math></span>. Filtered (black) distributions include only fits with a cost within 10% of the best fit (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mi>n</mi><mo>=</mo><mn>594</mn></mrow></math></span>). Y axes are normalized such that each distribution has a maximum density of 1.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s011.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s011.pdf</a><sup> (644.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material12_reqid_"><div class="caption p">
<span>S5 Fig. Properties of optimized parameters from the DDE model of sFLT1 secretion.</span><p>(A) Violin plots of optimal parameter values (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mi>n</mi><mo>=</mo><mn>594</mn></mrow></math></span>) for the delay differential equation (DDE) model. Units: α, #/cell/h; (β,γ,δ,ε), h<sup>-1</sup>. (B) Correlations between δ and τ and other optimized parameters. Each point represents the observed values of the listed parameters in a single run of the delay differential equation model after filtering for low-cost fits (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mi>n</mi><mo>=</mo><mn>594</mn></mrow></math></span>).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s012.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s012.pdf</a><sup> (444.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material13_reqid_"><div class="caption p">
<span>S6 Fig. Simulated absolute numbers of intracellular sFLT1 molecules.</span><p>(A) Time courses of absolute intracellular sFLT1 (<em>I</em>) during simulation of constitutive secretion. (B) Distribution of steady state values of intracellular sFLT1 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>). The dashed red line indicates the theoretical lower bound min(<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>) <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><msubsup><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></math></span> based on median values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span> (Table 2).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s013.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s013.pdf</a><sup> (123.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material14_reqid_"><div class="caption p">
<span>S7 Fig. Steady state properties of process fluxes during simulation of constitutive secretion.</span><p>All annotation lines are derived in <a href="#pcbi.1013324.s002" class="usa-link">S1 Text</a> and calculated using median values of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>. (A) Correlation between secretion flux (Φ(Secr)) and the compound parameter <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mi>α</mi><mi>β</mi><mo>/</mo><mo stretchy="false">(</mo><mi>β</mi><mo>+</mo><mi>γ</mi><mo stretchy="false">)</mo><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>. Both dashed lines indicate <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>. (B) Correlation between intracellular degradation flux (Φ(IDeg)) and production flux (Φ(Prod)). Dashed lines indicate the theoretical lower bound min(Prod) <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span> and the asymptote Φ(IDeg) <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mo>=</mo></mrow></math></span> Φ(Prod). (C) Correlation between steady state intracellular sFLT1 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>) and intracellular degradation flux (Φ(IDeg)). Dashed lines indicate the theoretical lower bound min(<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>) <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><msubsup><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></math></span> and the asymptote <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub><mo>=</mo></mrow></math></span> Φ(IDeg)/<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>. (D) Correlation between steady state intracellular sFLT1 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>) and production flux (Φ(Prod)) during constitutive secretion. Dashed lines indicate theoretical lower bounds (min(<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>) <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><msubsup><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow><mrow><mn>2</mn></mrow></msubsup></mrow></math></span>) and min(Φ(Prod)) <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><mo>=</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>/</mo><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>). Φ = Flux, Prod = production, Secr = secretion, IDeg = intracellular degradation, XDeg = extracellular degradation.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s014.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s014.pdf</a><sup> (426.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material15_reqid_"><div class="caption p">
<span>S8 Fig. Relative process fluxes over time during simulations of pulse-chase secretion.</span><p>All species are normalized to their maximum observed values over 10 hours. Φ = flux, Prod = production, Secr = secretion, IDeg = intracellular degradation, XDeg = extracellular degradation.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s015.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s015.pdf</a><sup> (350.9KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material16_reqid_"><div class="caption p">
<span>S9 Fig. Impact of individual parameter variations on sFLT1 for simulations of constitutive secretion.</span><p>Top: extracellular sFLT1 at 72 hours (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mn>72</mn><mi>h</mi></mrow></msub></mrow></math></span>); bottom: steady state intracellular sFLT1 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s016.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s016.pdf</a><sup> (154.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material17_reqid_"><div class="caption p">
<span>S10 Fig. Sensitivity of extracellular and intracellular sFLT1 to the compound parameters <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>=</mo><mi>α</mi><mi>β</mi></mrow></math></span> and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>=</mo><mi>β</mi><mo>+</mo><mi>γ</mi></mrow></math></span> in the DDE model.</span><p>(A) Time courses of extracellular (<em>X</em>) and intracellular (<em>I</em>) sFLT1 during constitutive secretion with fixed <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>=</mo><mn>7270</mn></mrow></math></span> #/cell/h [<a href="#pcbi.1013324.ref002" class="usa-link" aria-describedby="pcbi.1013324.ref002">2</a>] and varying <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span>. (B) Correlation between extracellular sFLT1 at 72 hours (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mn>72</mn><mi>h</mi></mrow></msub></mrow></math></span>) or steady state intracellular sFLT1 (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>S</mi><mi>S</mi></mrow></msub></mrow></math></span>) with β, γ, and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span> when <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></math></span> is constant during constitutive secretion. (C) Time courses of extracellular (X) and intracellular (I) sFLT1 during constitutive secretion with fixed <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>=</mo><mn>0.173</mn></mrow></math></span> h<sup>-1</sup> and varying <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></math></span>. (D) Correlation between extracellular sFLT1 at 72 hours (<span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>X</mi></mrow><mrow><mn>72</mn><mi>h</mi></mrow></msub></mrow></math></span>) with <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub></mrow></math></span>, β, and γ when <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub></mrow></math></span> is constant during constitutive secretion.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s017.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s017.pdf</a><sup> (367.3KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material18_reqid_"><div class="caption p">
<span>S11 Fig. Simulating different strengths of chemical inhibition and genetic downregulation of sFLT1 secretion.</span><p>Change in extracellular (X) and intracellular (I) sFLT1 at 18 hours (solid) or 72 hours (dashed) with (A) chemical inhibition or (B) genetic downregulation of individual parameters at varying fraction inhibition. sFLT1 values are normalized to the 18-hour (solid) or 72-hour (dashed) values from simulation with the median parameter set.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s018.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s018.pdf</a><sup> (381.1KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material19_reqid_"><div class="caption p">
<span>S12 Fig. Different sets of initial conditions for simulating chemical and genetic inhibition.</span><p>Time courses of extracellular (<em>X</em>) and intracellular (<em>I</em>) sFLT1 during constitutive simulation with different base parameter sets sharing constraints of <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>1</mn></mrow></msub><mo>=</mo><mi>α</mi><mi>β</mi><mo>=</mo><mn>7270</mn></mrow></math></span> #/cell/h [<a href="#pcbi.1013324.ref002" class="usa-link" aria-describedby="pcbi.1013324.ref002">2</a>] and <span xmlns:mml="http://www.w3.org/1998/Math/MathML"><math display="inline" overflow="linebreak"><mrow><msub><mrow><mi>c</mi></mrow><mrow><mn>2</mn></mrow></msub><mo>=</mo><mi>β</mi><mo>+</mo><mi>γ</mi><mo>=</mo><mn>0.173</mn></mrow></math></span> h<sup>-1</sup>. These time courses are used as the base cases for testing chemical and genetic inhibition from different initial conditions (Figs 9 and S13).</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s019.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s019.pdf</a><sup> (124.6KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material20_reqid_"><div class="caption p">
<span>S13 Fig. Variation in predicted effects of chemical and genetic perturbations across optimized parameter sets.</span><p>(A) Change in extracellular (<em>X</em>) and intracellular (<em>I</em>) sFLT1 at 18 hours (top) and 72 hours (bottom) with chemical inhibition of individual parameters at varying fraction inhibition for different base parameter sets (labeled by β values). (B) Change in extracellular (<em>X</em>) and intracellular (<em>I</em>) sFLT1 at 18 hours (top) and 72 hours (bottom) post media change with genetic inhibition of individual parameters at varying fraction inhibition for different base parameter sets (labeled by β values). <em>X</em> and <em>I</em> are normalized to the values at the end of each time course (18h or 72h, respectively) from the simulation with the corresponding base parameter set.</p>
<p>(PDF)</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s020.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s020.pdf</a><sup> (533.8KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material21_reqid_"><div class="caption p">
<span>Attachment</span><p>Submitted filename: <em>Response to Reviewers.pdf</em></p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12370208/bin/pcbi.1013324.s022.pdf" data-ga-action="click_feat_suppl" class="usa-link">pcbi.1013324.s022.pdf</a><sup> (248.5KB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>All code required to reproduce the model, output data, and analysis is available at <a href="https://github.com/gillsignals/sFlt1-paper" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/gillsignals/sFlt1-paper</a>.</p></section></section></article><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from PLOS Computational Biology are provided here courtesy of <strong>PLOS</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1371/journal.pcbi.1013324"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/pcbi.1013324.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.1 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12370208/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12370208/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12370208%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12370208/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12370208/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12370208/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40825055/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12370208/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40825055/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12370208/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12370208/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="cNl9nsRxfFAtpwxHZB4XbeNErT9Uv73XlRNyNYAw8lY4RcBJgs9BIAooVXtFVjVy">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
