
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Construction of a feature gene and machine prediction model for inflammatory bowel disease based on multichip joint analysis - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="AE5340788AF3C1630540780029164BC8.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="transmed">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366088/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Journal of Translational Medicine">
<meta name="citation_title" content="Construction of a feature gene and machine prediction model for inflammatory bowel disease based on multichip joint analysis">
<meta name="citation_author" content="Yan Chaosheng">
<meta name="citation_author_institution" content="Department of Gastroenterology, Affiliated Hospital of Jiangnan University, 214122 Wuxi city, Jiangsu Province China">
<meta name="citation_author_institution" content="Wuxi Medical College, Jiangnan University, 214062 Wuxi City, Jiangsu Province China">
<meta name="citation_author" content="Sun Haowen">
<meta name="citation_author_institution" content="Wuxi Medical College, Jiangnan University, 214062 Wuxi City, Jiangsu Province China">
<meta name="citation_author" content="Rao Jingjing">
<meta name="citation_author_institution" content="Department of Gastroenterology, Affiliated Hospital of Jiangnan University, 214122 Wuxi city, Jiangsu Province China">
<meta name="citation_author_institution" content="Wuxi Medical College, Jiangnan University, 214062 Wuxi City, Jiangsu Province China">
<meta name="citation_author" content="Dai Yuanyuan">
<meta name="citation_author_institution" content="Wuxi Medical College, Jiangnan University, 214062 Wuxi City, Jiangsu Province China">
<meta name="citation_author" content="Duan Wenhui">
<meta name="citation_author_institution" content="Wuxi Medical College, Jiangnan University, 214062 Wuxi City, Jiangsu Province China">
<meta name="citation_author" content="Sheng Yingyue">
<meta name="citation_author_institution" content="Department of Gastroenterology, Affiliated Hospital of Jiangnan University, 214122 Wuxi city, Jiangsu Province China">
<meta name="citation_author_institution" content="Wuxi Medical College, Jiangnan University, 214062 Wuxi City, Jiangsu Province China">
<meta name="citation_author" content="Xue Yuzheng">
<meta name="citation_author_institution" content="Department of Gastroenterology, Affiliated Hospital of Jiangnan University, 214122 Wuxi city, Jiangsu Province China">
<meta name="citation_author_institution" content="Wuxi Medical College, Jiangnan University, 214062 Wuxi City, Jiangsu Province China">
<meta name="citation_publication_date" content="2025 Aug 19">
<meta name="citation_volume" content="23">
<meta name="citation_firstpage" content="937">
<meta name="citation_doi" content="10.1186/s12967-025-06838-z">
<meta name="citation_pmid" content="40830794">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366088/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366088/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366088/pdf/12967_2025_Article_6838.pdf">
<meta name="description" content="Inflammatory bowel disease (IBD) is a chronic nonspecific inflammatory disorder triggered by immune responses and genetic factors. Currently, there is no cure for IBD, and its etiology remains unclear. As a result, early detection and diagnosis of ...">
<meta name="og:title" content="Construction of a feature gene and machine prediction model for inflammatory bowel disease based on multichip joint analysis">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Inflammatory bowel disease (IBD) is a chronic nonspecific inflammatory disorder triggered by immune responses and genetic factors. Currently, there is no cure for IBD, and its etiology remains unclear. As a result, early detection and diagnosis of ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366088/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12366088">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1186/s12967-025-06838-z"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/12967_2025_Article_6838.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366088%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12366088/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12366088/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366088/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-transmed.png" alt="Journal of Translational Medicine logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Journal of Translational Medicine" title="Link to Journal of Translational Medicine" shape="default" href="https://translational-medicine.biomedcentral.com/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">J Transl Med</button></div>. 2025 Aug 19;23:937. doi: <a href="https://doi.org/10.1186/s12967-025-06838-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1186/s12967-025-06838-z</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22J%20Transl%20Med%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22J%20Transl%20Med%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22J%20Transl%20Med%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22J%20Transl%20Med%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Construction of a feature gene and machine prediction model for inflammatory bowel disease based on multichip joint analysis</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chaosheng%20Y%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Yan Chaosheng</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Yan Chaosheng</span></h3>
<div class="p">
<sup>1</sup>Department of Gastroenterology, Affiliated Hospital of Jiangnan University, 214122 Wuxi city, Jiangsu Province China </div>
<div class="p">
<sup>2</sup>Wuxi Medical College, Jiangnan University, 214062 Wuxi City, Jiangsu Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Chaosheng%20Y%22%5BAuthor%5D" class="usa-link"><span class="name western">Yan Chaosheng</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Haowen%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Sun Haowen</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Sun Haowen</span></h3>
<div class="p">
<sup>2</sup>Wuxi Medical College, Jiangnan University, 214062 Wuxi City, Jiangsu Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Haowen%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sun Haowen</span></a>
</div>
</div>
<sup>2,</sup><sup>#</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jingjing%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Rao Jingjing</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Rao Jingjing</span></h3>
<div class="p">
<sup>1</sup>Department of Gastroenterology, Affiliated Hospital of Jiangnan University, 214122 Wuxi city, Jiangsu Province China </div>
<div class="p">
<sup>2</sup>Wuxi Medical College, Jiangnan University, 214062 Wuxi City, Jiangsu Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jingjing%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Rao Jingjing</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yuanyuan%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Dai Yuanyuan</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Dai Yuanyuan</span></h3>
<div class="p">
<sup>2</sup>Wuxi Medical College, Jiangnan University, 214062 Wuxi City, Jiangsu Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yuanyuan%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dai Yuanyuan</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wenhui%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Duan Wenhui</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Duan Wenhui</span></h3>
<div class="p">
<sup>2</sup>Wuxi Medical College, Jiangnan University, 214062 Wuxi City, Jiangsu Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wenhui%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Duan Wenhui</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yingyue%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Sheng Yingyue</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Sheng Yingyue</span></h3>
<div class="p">
<sup>1</sup>Department of Gastroenterology, Affiliated Hospital of Jiangnan University, 214122 Wuxi city, Jiangsu Province China </div>
<div class="p">
<sup>2</sup>Wuxi Medical College, Jiangnan University, 214062 Wuxi City, Jiangsu Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yingyue%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sheng Yingyue</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yuzheng%20X%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Xue Yuzheng</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Xue Yuzheng</span></h3>
<div class="p">
<sup>1</sup>Department of Gastroenterology, Affiliated Hospital of Jiangnan University, 214122 Wuxi city, Jiangsu Province China </div>
<div class="p">
<sup>2</sup>Wuxi Medical College, Jiangnan University, 214062 Wuxi City, Jiangsu Province China </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Yuzheng%20X%22%5BAuthor%5D" class="usa-link"><span class="name western">Xue Yuzheng</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>✉</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Gastroenterology, Affiliated Hospital of Jiangnan University, 214122 Wuxi city, Jiangsu Province China </div>
<div id="Aff2">
<sup>2</sup>Wuxi Medical College, Jiangnan University, 214062 Wuxi City, Jiangsu Province China </div>
<div class="author-notes p">
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
<div class="fn" id="_eqcntrb93pmc__">
<sup>#</sup><p class="display-inline">Contributed equally.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Mar 24; Accepted 2025 Jul 5; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12366088  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40830794/" class="usa-link">40830794</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<section id="sec1"><h3 class="pmc_sec_title">Background</h3>
<p id="Par1">Inflammatory bowel disease (IBD) is a chronic nonspecific inflammatory disorder triggered by immune responses and genetic factors. Currently, there is no cure for IBD, and its etiology remains unclear. As a result, early detection and diagnosis of IBD pose significant challenges. Therefore, investigating biomarkers in peripheral blood is highly important, as they can assist doctors in the early identification and management of IBD.</p></section><section id="sec2"><h3 class="pmc_sec_title">Methods</h3>
<p id="Par2">We used a multichip joint analysis approach to explore the database thoroughly. On the basis of methods such as artificial neural networks (ANNs), machine learning techniques, and the SHAP model, we developed a diagnostic model for IBD. To select genetic features, we utilized three machine learning algorithms, namely, least absolute shrinkage and selection operator (LASSO), support vector machine (SVM), and random forest (RF), to identify differentially expressed genes. Additionally, we conducted an in-depth analysis of the enriched molecular pathways of these differentially expressed genes through Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway enrichment analyses. Moreover, we used the SHAP model to interpret the results of the machine learning process. Finally, we examined the relationships between the differentially expressed genes and immune cells.</p></section><section id="sec3"><h3 class="pmc_sec_title">Results</h3>
<p id="Par3">Through machine learning, we identified four crucial biomarkers for IBD, namely, LOC389023, DUOX2, LCN2, and DEFA6. The SHAP model was used to elucidate the contribution of the differentially expressed genes to the diagnostic model. These genes were associated primarily with immune system modulation and microbial alterations. GO and KEGG pathway enrichment analyses indicated that the differentially expressed genes demonstrated associations with molecular pathways such as the antimicrobial and IL-17 signaling pathways. By performing correlation and differential analyses between differentially expressed genes and immune cells, we found that M1 macrophages exhibited stable differential changes in all four differentially expressed genes. M2 macrophages, resting mast cells, neutrophils, and activated memory CD4 T cells all showed significant differences in three of the differentially expressed genes.</p></section><section id="sec4"><h3 class="pmc_sec_title">Conclusion</h3>
<p id="Par4">We identified differentially expressed genes (LOC389023, DUOX2, LCN2, and DEFA6) with significant immune-related effects in IBD. Our findings suggest that machine learning algorithms outperform ANNs in the diagnosis of IBD. This research provides a theoretical foundation for the clinical diagnosis, targeted therapy, and prognostic evaluation of IBD.</p></section><section id="sec5"><h3 class="pmc_sec_title">Supplementary Information</h3>
<p>The online version contains supplementary material available at 10.1186/s12967-025-06838-z.</p></section><section id="kwd-group1" lang="en" class="kwd-group"><p><strong>Keywords:</strong> Inflammatory bowel disease, Machine learning, Artificial neural network, Diagnostic model, Immune differences</p></section></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par5">Inflammatory bowel disease (IBD) can be divided into ulcerative colitis (UC) and Crohn’s disease (CD). Since the emergence of IBD in the 20th century, the incidence rate of IBD has increased with industrialization and urbanization, similar to that of other immune-related diseases. Gabriel reported in the literature that the estimated prevalence of IBD affects approximately 5 million people and that there are approximately 400,000 new cases annually. In addition, age and sex are the key factors influencing the incidence rate and prevalence of IBD [<a href="#CR1" class="usa-link" aria-describedby="CR1">1</a>, <a href="#CR2" class="usa-link" aria-describedby="CR2">2</a>]. For example, 25% of cases occur in childhood, and the incidence rate is increasing. Research indicates that IBD typically affects young people with relatively low mortality rates, and there is currently no curative treatment available [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>, <a href="#CR4" class="usa-link" aria-describedby="CR4">4</a>].</p>
<p id="Par6">The increasing prevalence of IBD emphasizes the need for a deeper understanding of its molecular mechanisms to develop targeted therapies and diagnostic tools. Currently, in clinical practice, there are no biomarkers capable of accurately predicting the disease course or treatment response. This is attributed mainly to the complex molecular basis of IBD and variations in immune responses [<a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>].</p>
<p id="Par7">In reality, C-reactive protein (CRP), the erythrocyte sedimentation rate (ESR), fecal biomarkers, and calprotectin are often regarded as essential diagnostic tools for IBD. However, in practical applications, these so-called biomarkers have certain limitations. For example, fecal biomarkers have poor accuracy, the ESR is easily affected by multiple factors, and CRP production shows increased heterogeneity.</p>
<p id="Par8">In recent years, with the data collected from genomic research, the relationships between specific genes and the etiology of IBD have been widely explored. As mentioned earlier, the association between the NOD2 gene and CD has been well established. It has become a key predictive factor and is associated with an increased risk of disease complications [<a href="#CR6" class="usa-link" aria-describedby="CR6">6</a>]. Additionally, individuals with genetic variability in the PRDM1 and NDP52 genes are more susceptible to CD. Genes such as KIF9-AS1, LINC01272, and DIO3OS have proven useful for differentiating and detecting various types of IBD. Regrettably, no research has yet been able to clearly explain the activity pathways of pathogenic genes and the immune cells affected by them.</p>
<p id="Par9">Previous studies have indicated that IBD results from interactions among immune responses, genetic factors, and the microbiota [<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>]. Although the exact causes of IBD remain unclear, multiple interrelated factors, such as genetics, the immune system, the microbiota, and the environment, all play a role in the development of IBD [<a href="#CR3" class="usa-link" aria-describedby="CR3">3</a>]. Immune dysfunction can trigger persistent inflammation, a characteristic feature of IBD. This leads to a reduction in or destruction of intestinal crypts, along with a series of severe clinical manifestations and complications, significantly deteriorating the quality of life of patients [<a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>, <a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>].</p>
<p id="Par10">Our study revealed that, compared with normal controls, patients with IBD exhibited differential expression of several genes. These genes were enriched predominantly in pathways associated with inflammatory and immune responses, which is consistent with previous findings [<a href="#CR10" class="usa-link" aria-describedby="CR10">10</a>]. In addition to the differences in gene expression, we also discovered that the diagnosis of IBD could be reflected by changes in immune cells. We propose that differential gene expression serves as the initiating factor of IBD. Through several potential mechanisms, it acts on immune cells, leading to significant variations in their expression levels and quantities. First, differential gene expression may trigger excessive activation or inhibition of pathogenic molecular pathways in IBD patients, causing the release of an excessive amount of inflammatory factors. Second, the overexpression of inflammatory factors disrupts the immune response in patients. Eventually, the disordered immune response results in differential expression in immune cells in the body. In conclusion, differential gene expression plays a pivotal role in the initiation and progression of IBD and can be regarded as a key element in the development of novel diagnostic and therapeutic strategies for IBD. To explore this further, we used multichip joint analysis, artificial neural networks (ANNs), and machine learning algorithms to identify significantly differentially expressed genes. We then utilized the SHAP model to illustrate the contribution of these differentially expressed genes to the diagnosis. Finally, we performed correlation and differential analyses on immune cells to identify differences in gene expression among various immune cells.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec3"><h3 class="pmc_sec_title">Data source, preprocessing, and analysis</h3>
<p id="Par11">In this study, we conducted a comprehensive analysis of multiple databases, including GEO, IBDDMB, and UKB. Specifically, we retrieved disease datasets (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87466" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE87466</a> “<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87466" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.ncbi.nlm.nih.gov/geo/query/acc.cgi? acc=GSE87466</a>”, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE179285" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE179285</a> “<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE179285" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.ncbi.nlm.nih.gov/geo/query/acc.cgi? acc=GSE179285</a>”, and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87473" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE87473</a> “<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87473" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">www.ncbi.nlm.nih.gov/geo/query/acc.cgi? acc=GSE87473</a>”) from the GEO database. These datasets included data from 438 patients with inflammatory bowel disease and 51 healthy individuals. The patient biopsy data were sourced from the sigmoid colon, ascending/descending colon, and terminal ileum. The patient data included cases of moderate to severe active ulcerative colitis [<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a>]. We utilized R software (version 4.3.3) for data preparation. During the preprocessing stage, we removed probes corresponding to multiple genes and converted probe IDs into gene symbols with the annotation file of the platform. When dealing with multiple probes for the same gene, we retained only the probe with the highest signal value. To ensure data consistency and reliability, we took measures to reduce the potential impact of batch effects, which are commonly introduced during the data integration process. Batch effects can occur due to variations in experimental conditions, instruments, or sample processing over time or across different datasets and may severely confound the interpretation of results. Thus, we used the “limma”, “pheatmap”, and “ggplot2” packages to calibrate the data for different groups (diseased and nondiseased). Additionally, the “pheatmap” and “ggplot2” packages were used to generate heatmaps and volcano plots of the differentially expressed genes (DEGs), respectively. The data were analyzed by log2 transformation.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Transcriptome data refinement and analysis process</h3>
<p id="Par12">We utilized supplementary probe annotation files to convert the expression matrix from the probe level to the gene level. For genes associated with multiple probes, the arithmetic mean of the corresponding probe values was used to represent gene expression. Following this conversion, we standardized the dataset and then applied the SVA package for batch-effect correction. Principal component analysis (PCA) was utilized to assess the success of the standardization process. To identify the DEGs between the IBD and control samples, we used the limma package (linear model of microarray data). DEGs were defined as those with an absolute logarithmic fold change (|log FC|) greater than 2 and an adjusted <em>p</em> value less than 0.05. Particular emphasis was placed on genes that might be related to immune infiltration in IBD patients and normal individuals.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Enrichment analysis</h3>
<p id="Par13">To clarify the biological significance and pathway associations of DEGs, we carried out comprehensive Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) analyses. In the R programming environment, we systematically explored the effects of differentially expressed protein-modifying genes (PMGs) on key biological processes (BPs), molecular functions (MFs), and cellular components (CCs). We used R software for this analysis, utilizing tools such as the “clusterProfiler” and “org” packages, along with the “Hs.eg.db”, “enrichplot”, and “ggplot2” packages. We enriched our interpretation with a focus on KEGG pathway data. This integrated approach allows us to observe the complexity of the molecular landscape of the IBD and the control groups, offering a comprehensive framework for further exploration of the underlying mechanisms.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Artificial neural network (ANN)</h3>
<p id="Par14">ANN models play an increasingly crucial role in the realm of predictive modeling. This is because they are capable of capturing nonlinear relationships within high-dimensional datasets [<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a>]. ANN models can predict complex variable relationships that other models, such as logistic regression models, are unable to achieve. The working principle of an ANN model is inspired by biological neural networks. In an ANN, each neuron is interconnected with other neurons. Neurons have two main components: dendrites and axons. Dendrites function as receivers of information, whereas axons serve as transmitters. The nucleus of a neuron holds the information to be transmitted. An ANN typically consists of an input layer, one or more hidden layers, and an output layer. Information enters the model through the input layer, undergoes processing in the hidden layer(s), and is then output via the output layer [<a href="#CR13" class="usa-link" aria-describedby="CR13">13</a>]. In our study, we utilized R software for correlation analysis. To increase the intuitiveness of the data, we used packages such as “neuralnet” and “NeuralNetTools” to generate relevant graphics.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Machine learning algorithms</h3>
<p id="Par15">To identify candidate genes, we used the “VennDiagram” package to visualize the intersection of key genes among the DEGs. Three machine learning algorithms, namely, the least absolute shrinkage and selection operator (LASSO), support vector machine (SVM), and random forest (RF), were utilized to identify potential biomarkers. For the LASSO analysis, we used the “glmnet” package with a penalty parameter and 10-fold cross-validation. This approach was applied to select important variables from high-dimensional data. RF is an ensemble estimator that consists of multiple decision trees as its basic estimators. In the RF classification process, each tree determines a category, and the category receiving the highest number of votes is designated as the final output. The SVM considers each predictor as a dimension in a high-dimensional space. The SVM aims to find the optimal hyperplane for classifying samples and demonstrates excellent performance when dealing with highly complex data. Finally, the genes at the intersection of the results from the LASSO, RF, and SVM algorithms were identified as potential biomarkers for IBD.</p></section><section id="Sec8"><h3 class="pmc_sec_title">Explanation of machine learning models</h3>
<p id="Par16">Ten machine learning models were constructed with the selected predictive factors. These models included ridge least squares (RLS), RF, decision tree (DTS), SVM, logistic regression, k-nearest neighbors (KNN), extreme gradient boosting (XGBoost), gradient boosting machine (GBM), neural network, and generalized linear model boosting (GlmBoost). Some of these models possess high interpretability and are well suited for analyzing linear relationships. Other models are capable of capturing nonlinear relationships and interactions, making them suitable for handling high-dimensional data. For example, XGBoost is an optimized version of the gradient boosting framework. It supports parallel computing and can automatically handle missing values, showing excellent performance in clinical prediction tasks. We evaluated these models on the basis of several metrics, such as the receiver operating characteristic (ROC) curve, specificity, sensitivity, and accuracy, with the ROC curve serving as the primary evaluation indicator. The model that exhibited the best predictive performance was chosen as the main model for this study. To further understand the interpretability of the final prediction model, we used the Shapley sum interpretation (SHAP) method [<a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>]. This method helps explain how each feature contributes to the model’s prediction, providing valuable insights into the underlying mechanisms of the model.</p></section><section id="Sec9"><h3 class="pmc_sec_title">Gene set enrichment analysis and gene set variation analysis</h3>
<p id="Par17">Gene set enrichment analysis (GSEA) is a powerful and widely used approach in genomics research. It aims to uncover biological pathways, functions, or molecular features associated with distinct phenotypes or experimental conditions. In this study, we downloaded the immunological signature gene set (c7: immunological signature gene set) and conducted GSEA on all genes within the immune cell cluster (version 1.64.0). Conversely, gene set variation analysis (GSVA) offers a unique way to represent gene set variation within a sample. It achieves this by converting gene expression data into gene set activity scores without the need for data ranking. To perform GSVA, we calculated the average expression value of genes in each cell cluster in the immune-related gene set (h: hallmark gene set) (version 1.50.0). Finally, we visually presented the results of the GSVA analysis. This visualization helps to better understand the differences and trends in gene set activities, providing valuable insights into the underlying biological mechanisms related to the immune cell clusters.</p>
<p id="Par18">Immune Cell Infiltration and Correlation Analysis:</p>
<p id="Par19">The CIBERSORT algorithm was used to assess the relative abundance of immune cells in each normal and IBD sample. Spearman correlation analysis was subsequently performed to determine the relationships between potential biomarkers and immune cells. Additionally, a differential analysis of immune cells was conducted for each gene individually.</p></section><section id="Sec10"><h3 class="pmc_sec_title">Data analysis</h3>
<p id="Par20">Each experiment was performed at least three times. The ROC curve was established, and the area under the curve (AUC) and 95% confidence interval (CI) values were calculated and validated with SPSS software. The statistical significance between the two groups was determined via Student’s t test. The results were further analyzed by GraphPad Prism version 7 software. A <em>P</em> value less than 0.05 was considered to indicate statistical significance.</p>
<p id="Par21">Experimental verification: Real-time fluorescence quantitative PCR and Western blotting:</p>
<p id="Par22">According to local legislation and institutional requirements, research on human participants requires ethical review and approval. In compliance with national legislation and institutional requirements, an ethics approval number for this study (LS2024012) was obtained. The IBD samples in this study were obtained from the remaining specimens of IBD patients who underwent pathological tissue diagnosis during their initial and follow-up visits. Enteritis samples were obtained from patients diagnosed with common enteritis. Normal human samples were sourced from healthy individuals undergoing physical examinations.</p>
<p id="Par23">Western blotting (WB) was used to assess the protein expression levels of DUOX2, DEFA6, and LCN2 in 7 normal individuals, 4 patients with enteritis, and 9 patients with IBD, with GAPDH as the internal reference protein. Notably, LOC389023, which has been proven to be a pseudogene, is a long noncoding RNA located within chromosome 2q14.1 in the DPP10 gene; therefore, we did not perform Western blot detection on it. DUOX2 (Origene: 1000), LCN2 (Proteintech: 1000), DEFA6 (Proteintech: 1000), and GAPDH (Weiao: 1000) were used to quantify protein density with a Gel Pro analyzer.</p>
<p id="Par24">The sample source for the RT‒PCR experiment was the same batch as for the second Western blotting sample, and the mRNA expression levels of DUOX2, DEFA6, LCN2, and LOC389023 were assessed in 4 normal individuals and 4 IBD patients. The primers used for amplifying DUOX2 were as follows: the F1 timer was gaagttcgacctcaggacca, and the R1 timer was agggagtgtgaagaagggctc. The primers used for DEFA6 were as follows: the F1 timer was aaccctcaccatccactg, and the R1 timer was cggcaaagtcctggtcattt. The primers for LCN2 were as follows: the F1 timer was gagggaagtggtatgtggt, and the R1 timer was tgcgggtctttgtcctct. The primers used for LOC389023 were as follows: the F1 timer was agcttcaatcagactgccct, and the R1 timer was gttccttccccatgtaccca. We selected GAPDH as the internal reference gene, with its forward primer being gcgagatccctccaaatcaa and its reverse primer being gttcacacccatgacgaacat. The sequence refers to the sequences of the target genes in the GenBank database, and the primers used were designed by NCBI Primer BLAST. GAPDH was used as the internal control. The reliability of the PCR results was verified to be consistent with the dissolution curve. The cycle threshold (Ct, which is the inflection point on the amplification power curve) was calculated, and the 2-ΔΔCt method [ΔCt = Ct (target gene) - Ct (GAPDH), ΔΔCt = ΔCt (target gene) - ΔCt (GAPDH)] was used to calculate relative gene expression.</p></section></section><section id="Sec11"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec12"><h3 class="pmc_sec_title">PCA data correction</h3>
<p id="Par25">Figure <a href="#Fig1" class="usa-link">1</a> shows the flowchart of this study. Before batch calibration, samples from different experiments were separated, and a batch effect was observed between them. Through PCA, samples from different experiments were randomly shuffled to eliminate batch effects (Fig. <a href="#Fig2" class="usa-link">2</a>A and <a href="#Fig2" class="usa-link">B</a>).</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366088_12967_2025_6838_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9720/12366088/25547427e812/12967_2025_6838_Fig1_HTML.jpg" loading="lazy" id="d33e367" height="720" width="669" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Flowchart of this study</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366088_12967_2025_6838_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9720/12366088/8c20df3f82ee/12967_2025_6838_Fig2_HTML.jpg" loading="lazy" id="d33e390" height="653" width="669" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Analysis of the differences in gene expression between different datasets and the effectiveness of batch effect correction. (<strong>A</strong>) Distribution of samples from different experiments before batch calibration. (<strong>B</strong>) PCA involved randomly shuffling samples from different experiments. (<strong>C</strong>) Heatmap of DEGs in the IBD dataset. Red and blue indicate upregulated and downregulated DEGs, respectively. (<strong>D</strong>) Volcanic map of DEGs in the IBD dataset with |log2FC|&gt;2. Red indicates an increase, and blue indicates a decrease</p></figcaption></figure></section><section id="Sec13"><h3 class="pmc_sec_title">Limma DEGs</h3>
<p id="Par26">The Limma method was used to identify DEGs in the <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87466" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE87466</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE179285" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE179285</a>, and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87473" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE87473</a> datasets, with 12 upregulated genes and 5 downregulated genes identified (Fig. <a href="#Fig2" class="usa-link">2</a>C,</p></section><section id="Sec14"><h3 class="pmc_sec_title">Enrichment analysis results</h3>
<p id="Par29">The enrichment analysis of 17 candidate diagnostic genes revealed that they were predominantly associated with the immune response, microbial factors, and the inflammatory response, underscoring their crucial role in the pathogenesis and progression of IBD. KEGG analysis indicated that these genes were significantly enriched in categories such as adipocytokine secretion, bile acid-related pathways, the FOXO signaling pathway, the IL-17 signaling pathway, alcohol-related cancer-associated receptor-cytochrome interactions, and the AMPK-glucagon-leukocyte-<em>Staphylococcus aureus</em>-related pathways (Fig. <a href="#Fig3" class="usa-link">3</a>A-<a href="#Fig3" class="usa-link">D</a>). In terms of biological processes, GO analysis revealed the involvement of these candidate genes in areas such as antimicrobial activity and cellular responses to toxic substances and cAMP (Fig. <a href="#Fig3" class="usa-link">3</a>E-<a href="#Fig3" class="usa-link">H</a>). Moreover, molecular functional analysis revealed that the response to lipolysis was a significant category among these genes.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9720/12366088/294a1c22ffbf/12967_2025_6838_Fig3_HTML.jpg" loading="lazy" id="d33e435" height="2613" width="1323" alt="Fig. 3"></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>This set of images shows the analysis of DEGs, demonstrating different pathways and functional classifications. (<strong>A-B</strong>) KEGG pathway analysis, with the color becoming redder as the <em>P</em> value decreases. (<strong>E-F</strong>) GO analysis, including biological processes, cellular components, and molecular functions. The color becomes redder as the <em>P</em> value decreases. (<strong>C, D, G, H</strong>) Pathways enriched in the DEGs; the size of the dots indicates the number of DEGs contained in the corresponding pathways. The larger the number is, the larger the number of dots</p></figcaption></figure></section><section id="Sec15"><h3 class="pmc_sec_title">ANN prediction results</h3>
<p id="Par31">The ANN used in this study consisted of three layers: an input layer, a hidden layer, and an output layer. In the input layer, disease features and genes were assigned scores. Subsequently, on the basis of these scores and weights related to disease characteristics and genes, the hidden layer was generated. There were five nodes in the hidden layer, and the output layer was exported with these nodes and their respective weights. The output layer represents the properties of the sample. Result.matrix contained information such as training error (such as the mean square error (MSE)) and the number of iterations (epochs) that were used to evaluate training effectiveness. Weights stored the connection weights between all layers, which was used to analyze feature importance. Plotnet generated a network topology diagram that displayed the connection relationships among the input layer, hidden layer, and output layer. Prediction logic obtained the probability values of the model output through the compute() function and then converted them into category labels (con or treat) with which.max. The optimization algorithm was resilient back propagation (Rprop), which is an adaptive learning rate gradient descent algorithm. For loss function, by default, the least squares error (SSE) was used, which is suitable for regression problems but was used for classification (through probability output). The evaluation indicator used a confusion matrix to calculate the classification accuracy of the two groups. Unfortunately, the accuracy of our designed ANN in predicting the control group was only 21%, whereas the accuracy in predicting the experimental group reached 95.2%. To evaluate the overall diagnostic performance, we constructed ROC curves. The results revealed that the accuracy of the ANN for sample prediction was 93.7% (Fig. <a href="#Fig4" class="usa-link">4</a>A and <a href="#Fig4" class="usa-link">E</a>).</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366088_12967_2025_6838_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9720/12366088/39982b6d4314/12967_2025_6838_Fig4_HTML.jpg" loading="lazy" id="d33e520" height="495" width="669" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>DEGs were screened through various methods, and the differential results were visualized. (<strong>A, E</strong>) The experimental group was distinguished from the control group with an ANN. A ROC curve was constructed to evaluate the overall diagnostic performance. (<strong>B, F</strong>) The genes were screened with the LASSO algorithm. To obtain the optimal model, a 10-fold cross-validation method was used. The lowest gene number (<em>n</em> = 11) at the lowest point of the curve was most suitable for LASSO. (<strong>C, G</strong>) Through the SVM algorithm for screening genes, accurate graphs and cross-validation error graphs were obtained, and 8 disease-characteristic genes were identified. (<strong>D, H</strong>) Screening genes through the random forest algorithm. Ten important genes were identified through the random forest method. IncNodePurity was used to sort genes on the basis of their relative importance. (<strong>I</strong>) The intersection of the results of the three algorithms yielded four genes. (<strong>J</strong>) Volcanic diagram of DEGs, with red indicating upregulation and green indicating downregulation. (<strong>K</strong>) Box plot of DEGs, with the horizontal axis representing the names of the intersecting characteristic genes and the vertical axis representing the expression levels of the genes. Blue indicates the sample of the control group, and red indicates the sample of the experimental group. (<strong>L</strong>) The outermost circle of the chromosome diagram represents the chromosome number, and the second circle represents the shape of the chromosome. The names of the intersecting genes are labeled at the corresponding positions on the chromosome</p></figcaption></figure></section><section id="Sec16"><h3 class="pmc_sec_title">Selection results of the characteristic genes</h3>
<p id="Par32">We utilized LASSO regression to generate cross-validation graphs to identify the 11 disease-characteristic genes (Fig. <a href="#Fig4" class="usa-link">4</a>B and <a href="#Fig4" class="usa-link">F</a>). Through the SVM method, we obtained accurate graphs and cross-validation error graphs, through which 8 disease-characteristic genes were identified (Fig. <a href="#Fig4" class="usa-link">4</a>C and <a href="#Fig4" class="usa-link">G</a>). Through the RF method, a scoring map of forest trees and gene importance was obtained, and 10 disease-characteristic genes were selected (Fig. <a href="#Fig4" class="usa-link">4</a>D and <a href="#Fig4" class="usa-link">H</a>). A Venn diagram was used to display the intersection of genes selected by the three machine learning algorithms. As a result, a total of four intersecting genes were identified (Fig. <a href="#Fig4" class="usa-link">4</a>I).</p></section><section id="Sec17"><h3 class="pmc_sec_title">Visualization results</h3>
<p id="Par33">A box plot of the DEGs was constructed and combined with a volcano plot of the DEGs, which revealed that DUOX2, LCN2, and DEFA6 were upregulated in the experimental group and that LOC389023 was downregulated in the experimental group. The intersection of the characteristic genes and their distribution on the chromosome can be observed through the chromosome circle diagram (Figure <a href="#Fig4" class="usa-link">4</a>J-<a href="#Fig4" class="usa-link">L</a>).</p></section><section id="Sec18"><h3 class="pmc_sec_title">Machine learning model results</h3>
<p id="Par35">Through ten machine learning models, including RLS, RF, DTS, SVM, logistic, KNN, XGBoost, GBM, Neural Net, and GlmBoost, the diagnostic efficiency of the four IBD DEGs was evaluated through 10-fold cross-validation training control parameters. After the optimal diagnostic model was identified, the Type column of the training set was converted into binary labels, and the model was retrained with the optimal method. A ROC curve was constructed to evaluate the overall diagnostic performance (Fig. <a href="#Fig5" class="usa-link">5</a>A).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366088_12967_2025_6838_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9720/12366088/cf955859a3f2/12967_2025_6838_Fig5_HTML.jpg" loading="lazy" id="d33e567" height="668" width="669" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Prediction of the samples with machine learning models, and SHAP analysis of the machine learning models. (<strong>A</strong>) Ten machine learning models were used to construct ROC curves to evaluate the overall diagnostic performance. (<strong>B</strong>) Bar chart, with the vertical axis representing the gene name and the horizontal axis representing the mean absolute value of the SHAP value. The larger the value is, the more likely it is to indicate the gene and the greater the effect on the predicted results. (<strong>C</strong>) To present the results of the prediction of a single sample, the benchmark value was first determined, and then, for each gene, the result of the prediction was obtained. (<strong>D</strong>) The bee colony plot, with the vertical axis representing gene names and the horizontal axis representing SHAP values, allowed us to obtain the mean SHAP value for each gene. The larger the value is, the greater the contribution of that gene is. Each dot represents a sample, the color of the dot represents the gene expression level, with purple indicating low expression and orange indicating high expression. (<strong>E</strong>) Waterfall chart displaying the predicted results of a single sample. In this graph, the vertical axis represents the gene expression level, and the horizontal axis represents the predicted value. The larger the absolute value of the value is, the greater the effect of this gene on the predicted results. (<strong>F</strong>) Scatter plot in which the horizontal axis represents the expression level of one gene and the vertical axis represents the SHAP value. The dots represent the expression level of the gene, with purple indicating low expression and orange indicating high expression. The interaction relationship between these two genes and SHAP values can be observed</p></figcaption></figure></section><section id="Sec19"><h3 class="pmc_sec_title">SHAP analysis results</h3>
<p id="Par36">By analyzing the predictive performance of machine learning, bar charts, bee colony plots, scatter plots, waterfall plots, and force plots were obtained. In the bar chart, the larger the value is, the greater the effect of this gene on the prediction results. The bee colony plot shows the mean SHAP value of each gene, which represents the contribution to the machine learning model. A scatter plot is a three-dimensional graph that allows observation of the interaction between genes and SHAP values. A waterfall plot can display the predicted results of a single sample, and the larger the absolute value of the value is, the greater the effect of this gene on the predicted results. To display the predicted results of a single sample, the benchmark value was first determined, and then, for each gene, the predicted result was obtained (Fig. <a href="#Fig5" class="usa-link">5</a>B-<a href="#Fig5" class="usa-link">F</a>). In summary, through SHAP analysis, we explained the machine learning model and calculated the contribution size of each gene. In each sample, the patients could also be distinguished by gene expression. We interpreted the model by comprehensively analyzing the results of these genes.</p></section><section id="Sec20"><h3 class="pmc_sec_title">GSEA and GSVA results</h3>
<p id="Par38">Through GSEA/GSVA, the functions or pathways enriched in the high-expression group or low-expression group of the target genes were analyzed, and the top five pathways with the most significant enrichment were visualized Figure <a href="#Fig6" class="usa-link">6</a>A-<a href="#Fig6" class="usa-link">L</a>)</p>
<figure class="fig xbox font-sm" id="Fig6"><h4 class="obj_head">Fig. 6.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366088_12967_2025_6838_Fig6_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9720/12366088/47d8d4df4c1f/12967_2025_6838_Fig6_HTML.jpg" loading="lazy" id="d33e607" height="710" width="669" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The GSEA/GSVA results of the target gene analysis were visualized. (<strong>A, B, D, E, F, H, J, K</strong>). The bar chart of the GSEA data, with the horizontal axis representing the sorted genes and the vertical axis representing the enriched scores, visualizes the top five pathways with the most significant enrichment. (<strong>C, F, I, L</strong>). In the GSVA bar chart, the vertical axis represents pathways, the horizontal axis represents T test values, red indicates upregulation of the target gene, and green indicates downregulation of the target gene. Gray indicates no difference in the target gene. (<strong>A-C</strong>) LOC389023, (<strong>D-F</strong>) DUOX2, (<strong>G-I</strong>) LCN2, (<strong>J-L</strong>) DEFA6</p></figcaption></figure></section><section id="Sec21"><h3 class="pmc_sec_title">Immune cell infiltration analysis results</h3>
<p id="Par40">The content of immune cells in each sample was determined through immune cell infiltration analysis, and the results were visualized to obtain a bar chart. Notably, the content refers to the relative content, and the sum of all immune cells is one. In the box plot of differences, an asterisk above a certain immune cell indicates that there were differences in that immune cell type between the control group and the experimental group. The horizontal and vertical axes of the correlation graph represent the names of the immune cells, and the values inside represent the correlation coefficients (Fig. <a href="#Fig7" class="usa-link">7</a>A-<a href="#Fig7" class="usa-link">D</a>). To investigate the relationships between immune cells and LOC389023, DUOX2, LCN2, and DEFA6, scatter plots were generated for immune cells with significant correlations. Then, we visualized the correlation results and obtained correlation lollipop plots (Fig. <a href="#Fig8" class="usa-link">8</a>A-<a href="#Fig8" class="usa-link">D</a>).</p>
<figure class="fig xbox font-sm" id="Fig7"><h4 class="obj_head">Fig. 7.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366088_12967_2025_6838_Fig7_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9720/12366088/5a5910dd737e/12967_2025_6838_Fig7_HTML.jpg" loading="lazy" id="d33e665" height="650" width="669" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Immune-related analysis results. (<strong>A</strong>) Relationships between immune cells and target genes; *: <em>P</em> &lt; 0.05, **: <em>P</em> &lt; 0.01, ***: <em>P</em> &lt; 0.001. (<strong>B</strong>) The number of immune cells in each sample was determined through immune cell infiltration analysis. The results of immune cell recordings were visualized with a bar chart, where the horizontal axis represents the sample and the vertical axis represents the content of the immune cells. The sum of all immune cells is one. Different colors indicate different immune cells. (<strong>C</strong>) The horizontal and vertical axes of the graph represent the names of the immune cells. The values inside represent the correlation coefficient, with red indicating a positive correlation and green indicating a negative correlation. (<strong>D</strong>) Box plot of the differences, with the horizontal axis representing the names of immune cells and the vertical axis representing the content of immune cells. Green indicates the samples of the control group, and red indicates the samples of the experimental group. *: <em>P</em> &lt; 0.05, **: <em>P</em> &lt; 0.01, ***: <em>P</em> &lt; 0.001</p></figcaption></figure><figure class="fig xbox font-sm" id="Fig8"><h4 class="obj_head">Fig. 8.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366088_12967_2025_6838_Fig8_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9720/12366088/49dfe5af1877/12967_2025_6838_Fig8_HTML.jpg" loading="lazy" id="d33e695" height="1256" width="738" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Correlation analysis between differentially expressed genes and immune cells. (<strong>A</strong>) LOC389023, (<strong>B</strong>) DUOX2, (<strong>C</strong>) LCN2, and (<strong>D</strong>) DEFA6. Correlation lollipop charts in which the vertical axis represents the names of immune cells, the horizontal axis represents the correlation coefficient, the size of the circle represents the absolute value of the correlation coefficient, and the color of the circle represents the <em>P</em> value of the correlation test. Scatter plots in which the horizontal axis represents the expression level of the target gene, the vertical axis represents the content of immune cells, the R value represents the correlation coefficient, and the <em>P</em> value represents statistical validity</p></figcaption></figure></section><section id="Sec22"><h3 class="pmc_sec_title">Experimental results</h3>
<p id="Par43">We collected 3 samples of normal human intestinal tissue, 2 samples of intestinal tissue from patients with enteritis, and 5 samples from patients with IBD. The presence of target proteins in the sample was detected through WB, and quantitative analysis and measurement of grayscale values for the target proteins were performed.</p>
<p id="Par44">DUOX2, with a molecular weight of approximately 130 kDa, LCN2, with a molecular weight of 25 kDa, and DEFA6, with a molecular weight of 11 kDa, were detected in all three groups. Visual inspection of the bands revealed differences in expression levels between the groups, but there was a similar trend between the colitis group and the IBD group. Therefore, we conducted one-way ANOVA to test whether multiple population means were equal, analyzed intergroup and intragroup variations, and determined whether there were significant differences in the relative expression levels of target proteins between different groups. Although no obvious trend was observed in the visual bands, one-way ANOVA of the ratio of the target protein grayscale value to the internal reference protein GAPDH grayscale value revealed significant differences in the target protein among normal individuals, enteritis patients, and IBD patients.</p>
<p id="Par45">To eliminate the randomness of the experiment and reduce experimental errors, we reselected 4 samples of normal human intestinal tissue, 2 samples of intestinal tissue from patients with enteritis, and 4 samples of intestinal tissue from patients with IBD. We repeated the above experiment and found that the results were not significantly different from those of the first experiment. The experimental data revealed good discrimination for DUOX2, LCN2, and DEFA6 in normal individuals, enteritis patients, and IBD patients (Fig. <a href="#Fig9" class="usa-link">9</a>).</p>
<figure class="fig xbox font-sm" id="Fig9"><h4 class="obj_head">Fig. 9.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366088_12967_2025_6838_Fig9_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9720/12366088/6aa6576de123/12967_2025_6838_Fig9_HTML.jpg" loading="lazy" id="d33e724" height="328" width="669" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Detection of DUOX2, LCN2, and DEFA6 expression in 20 frozen human intestinal tissues with WB (<strong>A</strong>: first WB experiment, <strong>B</strong>: second WB experiment)</p></figcaption></figure><p id="Par46">We conducted real-time PCR experiments using samples from the second WB experiment, compared the expression levels of different genes in each sample group, and analyzed the expression characteristics and differences of each gene under different conditions. The experimental data revealed good discrimination for DUOX2, LCN2, and DEFA6 between normal individuals and IBD patients and poor discrimination for LOC389023 between normal individuals and IBD patients (<em>P</em> = 0.114) (Fig. <a href="#Fig10" class="usa-link">10</a>).</p>
<figure class="fig xbox font-sm" id="Fig10"><h4 class="obj_head">Fig. 10.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12366088_12967_2025_6838_Fig10_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/9720/12366088/6c5b38ae9610/12967_2025_6838_Fig10_HTML.jpg" loading="lazy" id="d33e754" height="658" width="669" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>The 2<sup>−ΔΔCt</sup> values of different genes (DUOX2, LCN2, DEFA6, and LOC389023) in multiple sample groups were determined by real-time PCR. The vertical axis represents relative gene expression levels, whereas the horizontal axis represents grouping information (*<em>p</em> &lt; 0.05, **<em>p</em> &lt; 0.01, ***<em>p</em> &lt; 0.001)</p></figcaption></figure></section></section><section id="Sec23"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par49">Our study represents a comprehensive genetic-based interpretation of the differences in genes, activity pathways, and immune responses in IBD. We integrated multichip analysis, ANN, and ten machine learning methods. Additionally, we performed interpretable analysis on the machine learning model to explore the contribution of DEGs to the diagnosis of IBD. By using DEGs as a benchmark, we investigated the changes in each gene in the immune response and examined the alterations in and correlations among immune cells. This allowed us to elaborate on the genetic factors of IBD patients from both genetic and immune response perspectives. On the basis of these findings, we developed a predictive model and validated it multiple times with data from the GEO database.</p>
<p id="Par50">IBD results from the interaction between immune responses and genetic factors, but the effects of factors such as microorganisms, the environment, and diet cannot be ignored [<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a>]. Currently, the clinical diagnosis of IBD is still restricted by various limitations, and a simple and convenient diagnostic method remains to be developed. The biomarkers identified through our screening can be detected in peripheral blood, enabling easy assessment of the likelihood of IBD in subjects [<a href="#CR16" class="usa-link" aria-describedby="CR16">16</a>, <a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>, <a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>].</p>
<p id="Par51">In our study, we successfully identified four biomarkers for the diagnosis of IBD, namely, LOC389023, DUOX2, LCN2, and DEFA6. These biomarkers play crucial roles in immune function. The ANN we designed exhibited an accuracy rate of merely 21% for predicting in the control group, yet it achieved a remarkable 95.2% accuracy rate for predicting in the experimental group. By constructing ROC curves to assess the overall diagnostic performance, the accuracy of sample prediction with the ANN reached 93.7%. We utilized these biomarkers to construct machine learning models. After selecting the optimal model through 10-fold cross-validation, we transformed the raw data for secondary validation. Among the machine learning algorithms, GBM and KNN demonstrated the highest diagnostic performance, reaching 95.2%. They were able to distinguish effectively between the experimental and control groups. However, there were numerous uncertainties regarding the diagnostic performance of the ANN. We made multiple attempts to modify the number of hidden layers and conducted small-scale validations on the data. Unfortunately, the results did not meet our expectations. After consulting the relevant literature and experts, our general hypothesis was that gene scoring might have led to data overfitting in the ANN. Nonetheless, we used various types of machine learning methods to analyze the data from multiple dimensions and perspectives, which effectively circumvented these issues. To this end, we compared the results of the ANN and machine learning models, which highlighted that the machine learning models were more reliable. Nevertheless, large-scale validation is still needed to confirm their generalizability.</p>
<p id="Par52">We assessed the potential of LOC389023, DUOX2, LCN2, and DEFA6 as diagnostic biomarkers through real-time fluorescence quantitative PCR and Western blot experiments. In our study of diagnostic biomarkers for IBD, we initially included patients with common enteritis as normal individuals in the sample, on the basis of a clear distinction between the two diseases in clinical practice. Common enteritis has different etiologies and relatively good prognoses and is fundamentally different from IBD. However, the Western blot results revealed that patients with common enteritis and patients with IBD exhibited the same trend, which was indeed unexpected. We conducted secondary experiments and reselected samples, and the results revealed that DUOX2, LCN2, and DEFA6 had good diagnostic efficacy. We comprehensively analyzed the data and found that the genes we screened are related to immunity, and thus, the increased expression in the enteritis group is also of practical significance. Moreover, the genes we screened exhibited good discrimination in normal individuals, enteritis patients, and IBD patients, which is a new discovery in our experiments. On the basis of the WB results, we conducted real-time fluorescence quantitative PCR experiments on samples from normal individuals and IBD patients for the second time and found that DUOX2 (<em>P</em> = 0.0253), LCN2 (<em>P</em> = 0.00873), and DEFA6 (<em>P</em> = 0.00348) still demonstrated outstanding discrimination, whereas LOC389023 performed poorly (<em>P</em> = 0.114), which may be related to its nature as a long noncoding RNA. It cannot be ruled out that our experiment may have errors.</p>
<p id="Par53">Research findings have indicated that LOC389023 plays a vital role in regulating gene expression and chromatin status. LOC389023 is a long noncoding RNA located within chromosome 2q14.1 and the DPP10 gene. LOC389023, in the nuclei of neurons, contains GC-rich stem-ring motifs. These motifs can bind to the SUZ12 protein in the chromatin and the polycomb repressive chromatin-modifying complex 2 (PRC2) complex. By doing so, it recruits chromatin-remodeling inhibitors, leading to a reduction in DPP10 gene expression. This regulatory effect exhibits cell type specificity. This study also revealed the relationships among LOC389023, gene expression, histone methylation, and voltage-gated K(+) channels during neural development [<a href="#CR19" class="usa-link" aria-describedby="CR19">19</a>]. DNA methylation and noncoding RNAs have been extensively investigated in patients with IBD. DNA methylation is dependent on dietary cofactors such as substrates and nutrients (folate, vitamin B12/D, etc.). It is associated with inflammation, microbiota composition, and microRNAs, which can influence IBD by interfering with T-cell differentiation. Marangoni et al. conducted a comprehensive analysis of the role of DNA methylation in IBD and its effect on the inflammatory process [<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a>]. Furthermore, studies have revealed that voltage-gated K(+) channels, such as KV1.3 and KCa3.1, are involved in K(+) conductance in T lymphocytes. They play crucial roles in cell proliferation, differentiation, apoptosis, and infiltration. In chemically induced IBD model mice, the activity and expression of KCa3.1 in CD4(+) T lymphocytes of mesenteric lymph nodes were increased. Moreover, its regulatory factor NDPK-B was positively expressed. When the KCa3.1 K(+) channel was blocked with TRAM-34 and/or ICA17043, the severity of IBD was significantly reduced. Symptoms such as diarrhea, fecal blood, inflammation, and colonic crypt injury were alleviated, and simultaneously, the expression levels of KCa3.1 and Th1 cytokines in CD4(+) T lymphocytes were restored. These findings indicate that abnormalities in the KCa3.1 channel are related to the development of IBD and that interventions targeting this channel can improve disease conditions [<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a>, <a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>]. In addition, the upregulation of K2P5.1 in T lymphocytes is associated with the pathogenesis of autoimmune diseases. Given that IBD also belongs to this category of diseases and that ion channel pre-mRNA splicing is related to disease, the mRNA splicing mechanism regulated by K2P5.1 K(+) channel transcription holds guiding significance for the treatment of IBD and other diseases [<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>].</p>
<p id="Par54">Lipocalin 2 (LCN2), a member of the adipokine protein family, is predominantly involved in processes such as cell growth, differentiation, metabolism, and the immune response. During inflammation, various cell types, including macrophages, epithelial cells, and neutrophils, secrete LCN2. It then exerts its effects on other cells via the bloodstream or the local tissue microenvironment, thereby influencing inflammatory responses, immune regulation, and other physiological processes. LCN2 is implicated in both acute and chronic inflammation and plays crucial pathogenic roles in diseases such as cancer, diabetes, obesity, and multiple sclerosis [<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a>, <a href="#CR24" class="usa-link" aria-describedby="CR24">24</a>]. Through a series of cell- and tissue-based studies, Xia et al. discovered that LCN2 is key in regulating iron homeostasis and the inflammatory response. LCN2 can interact with iron and iron carriers in diverse cell types, including immune cells and epithelial cells. By binding to bacterial iron carriers, it inhibits bacterial growth. Moreover, it can regulate cell survival, apoptosis, and other processes by modulating intracellular iron levels. Under inflammatory conditions, LCN2 stabilizes the iron pool and reduces iron-related toxicity. For example, in intestinal inflammation, LCN2 restricts the availability of iron in the intestine, safeguarding the mucosa from damage. In kidney diseases, it protects against acute kidney injury, yet in chronic kidney disease, it may exacerbate disease progression. Additionally, LCN2 has a complex role in tumor cells. It can both promote tumor cell growth and metastasis and potentially inhibit tumor development [<a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>–<a href="#CR27" class="usa-link" aria-describedby="CR27">27</a>]. Qun et al. conducted experiments in which mice were infected with harmful bacteria and reported that LCN2 is involved in microbial invasion, the inflammatory response, and tissue damage. It also participates in regulating the balance of the gut microbiota and its metabolites. When the LCN2 gene is absent, significant changes occur in the operational taxonomic units (OTUs) and alpha- and beta-diversity of the gut microbiota in mice infected with harmful bacteria. Simultaneously, intestinal metabolites are affected, with increased levels of metabolites such as taurodeoxycholic acid and undecylenic acid. These findings indicate that LCN2 may modulate the intestinal environment by regulating the composition and metabolite levels of the gut microbiota, thus influencing host health and disease resistance [<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a>].</p>
<p id="Par55">There are two types of dual oxidase (DUOX) enzymes, namely, DUOX1 and DUOX2. Their primary function is to generate reactive oxygen species (ROS) in tissues such as the thyroid, colon, respiratory tract, and lymphatic system. DUOX significantly contributes to the synthesis of hydrogen peroxide (H<sub>2</sub>O<sub>2</sub>), a substance that plays a pivotal role in the host defense system. H<sub>2</sub>O<sub>2</sub> is involved in processes such as signal transduction, cell differentiation, cell death programs, immune defense, microbial composition regulation, and hormone synthesis (specifically that of thyroid hormones) [<a href="#CR29" class="usa-link" aria-describedby="CR29">29</a>, <a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. Helmut carried out a multiomics whole-phenotype association study (PheWAS) on 2872 participants to analyze the relationship between DUOX2 gene variations and the IBD phenotype. This study revealed that rare variations in the DUOX2 gene were associated with an elevated risk of IBD. Through multiomics PheWAS and rare variation association analysis, the link between DUOX2 variations and the pathogenesis of IBD was elucidated [<a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>]. In the context of respiratory diseases, with respiratory viral infections as an example, Ducquin demonstrated that DUOX2 selectively regulates the cytokines and chemokines secreted by epithelial cells. This, in turn, affects the recruitment, adhesion, and degranulation of neutrophils, revealing that DUOX2 plays a crucial role in modulating the interaction between epithelial cells and immune cells [<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a>, <a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>]. In studies of congenital hypothyroidism (CH), DUOX2 mutations have been extensively explored. In a critical CH cohort, 50% of patients carried DUOX2 mutations (38%). These mutations are associated with patients’ biochemical characteristics, which influence the diagnosis and treatment of the disease. Moreover, current screening thresholds may result in missed diagnoses. At the cellular level, the expression of DUOX2 on the cell membrane and its mediation of H<sub>2</sub>O<sub>2</sub> production are highly important. When DUOX2-mediated H<sub>2</sub>O<sub>2</sub> production is completely lost, it further impairs the synthesis of thyroid hormones, ultimately leading to the development of CH. From the perspective of signaling pathways, Juan’s research indicated that DUOX2 plays a significant role in the development of IBD-related tumors. When the TLR4 signaling pathway is activated in epithelial cells, the expression of DUOX2 is upregulated. DUOX2 then catalyzes the production of H<sub>2</sub>O<sub>2</sub>, which is closely associated with the initiation and progression of tumors. Additionally, DUOX2 interacts with the microbiota. In this process, the generated H<sub>2</sub>O<sub>2</sub> promotes tumor development, affecting the transition from IBD to tumors [<a href="#CR33" class="usa-link" aria-describedby="CR33">33</a>, <a href="#CR34" class="usa-link" aria-describedby="CR34">34</a>].</p>
<p id="Par56">DEFA6 is secreted by Paneth cells, specialized epithelial cells located at the base of the small intestine crypts, and represents the most abundant antibacterial agent produced by Paneth cells in the small intestine. Its release into the crypt lumen is thought to safeguard against microbial invasion into the crypt microenvironment [<a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>]. As an antimicrobial peptide, DEFA6 plays a crucial role in the intestinal immune defense system by combating pathogens. A study was conducted a study on mucosal samples obtained from 88 CD patients who underwent ileocolonic resection. The results revealed that while the expression of DEFA6 in the healthy and diseased ileal mucosa of early- and late-stage CD patients did not significantly differ, there was an increasing trend in the expression of DEFA6 in the external validation cohort of late-stage CD patients. As the course of CD progresses, the expression of antimicrobial peptide-related genes, such as DEFB4A, increases in the affected mucosa. It is hypothesized that persistent mucosal damage may enable intestinal bacteria to interact with epithelial cells, thereby stimulating the expression of antimicrobial peptides. Abundant evidence indicates that alterations in DEFA6 expression are associated with the disease course of CD [<a href="#CR36" class="usa-link" aria-describedby="CR36">36</a>]. Stephen’s research revealed that DEFA6, which serves as a specific marker for Paneth cells in the small intestine, contributes to intestinal immune defense and the maintenance of the gut microbiota balance [<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a>, <a href="#CR38" class="usa-link" aria-describedby="CR38">38</a>]. Serena’s work confirmed that DEFA6 functions to maintain intestinal immune homeostasis in the Paneth cells of human small intestine organoids. Studies have shown that Paneth cells can produce antibacterial substances such as DEFA6 to preserve the gut microbiota balance. However, the expression level of DEFA6 in human small intestine organoids is extremely low and significantly differs from that in the source tissue. In contrast, mouse small intestine organoids can more effectively mimic the expression of α-defense factors in tissues. Moreover, scientists have reported that WNT signal stimulation fails to restore the expression of DEFA6 in human small intestine organoids. Nevertheless, after treatment with FOXO inhibitors, the mRNA expression of DEFA6 increased by more than 100,000-fold, nearly reaching the level in human tissue. These findings indicate that the FOXO signaling pathway is essential for regulating the expression of DEFA6 in human Paneth cells. Inhibiting the FOXO signaling pathway can effectively restore the expression of DEFA6, which is highly important for the study of intestinal diseases and the enhancement of intestinal immunity [<a href="#CR39" class="usa-link" aria-describedby="CR39">39</a>].</p>
<p id="Par57">GO and KEGG analyses revealed that the DEGs were predominantly enriched in the antibacterial aspects of the human immune system and the IL-17 signaling pathway. Nicholas conducted a study on 136 IBD patients and reported that patients with low IgG/G1 levels had poorer clinical survival data than did those with normal levels. These findings suggest that humoral immunity plays a pivotal role in the survival of IBD patients. When IBD patients experience compromised humoral immunity, their likelihood of requiring surgery increases. This implies that low IgG/G1 levels have differential effects on the surgical requirements of different subtypes of IBD patients. These findings also indicate that humoral immunity can serve as a predictor of IBD patient survival, which is highly important for their clinical management. IL-17 is closely associated with IBD and is essential in the pathological progression of IBD. Research has demonstrated that IL-17 is a key cytokine secreted by Th17 cells and plays a substantial role in the development of intestinal inflammation in IBD patients [<a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>]. In CD patients, IL-17-producing cells accumulate in large quantities in the submucosal and muscularis propria layers. Moreover, compared with that in healthy individuals, the number of IL-17-producing T cells in CD patients is significantly greater. Ample evidence points to a strong link between IL-17 and IBD [<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a>]. Furthermore, Kosaku’s research has also shown that IL-17 is closely related to IBD and plays a crucial role in its pathogenesis [<a href="#CR42" class="usa-link" aria-describedby="CR42">42</a>, <a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>]. Through whole-exome sequencing analysis of colon organoids from UC patients and healthy controls, it was discovered that the UC inflammatory epithelium accumulates somatic mutations in multiple genes associated with the IL-17 signaling pathway, such as NFKBIZ, ZC3H12A, and PIGR. These genes are rarely affected in colon cancer but are mutated within the inflammatory environment of UC. These findings indicate that the IL-17 signaling pathway is significantly perturbed in the pathological process of UC. Additionally, gene mutations related to the IL-17 signaling pathway may also be implicated in the occurrence and development of UC in humans. They may disrupt the intestinal immune balance by interfering with the IL-17 signaling pathway, thus promoting the progression of UC [<a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>].</p>
<p id="Par58">Our research revealed differences in plasma cells, follicular helper T cells, activated natural killer (NK) cells, M1 macrophages, resting mast cells, activated mast cells, M0 macrophages, and neutrophils between the control group and the experimental group. Through a subsequent investigation of the relationships between genes and immune cells, we revealed that there were significant differences in LOC389023 in M1 macrophages, M2 macrophages, activated mast cells, resting mast cells, and neutrophils. Significant differences in LCN2 were observed in eosinophils, M0 macrophages, M1 macrophages, M2 macrophages, resting mast cells, neutrophils, plasma cells, activated CD4 memory T cells, and CD8 T cells. Stronger associations of DUOX2 with M0 macrophages, M1 macrophages, M2 macrophages, activated mast cells, resting mast cells, neutrophils, plasma cells, activated memory CD4 T cells, and CD8 T cells were detected. For DEFA6, significant changes were noted in M1 macrophages, activated memory CD4 T cells, and regulatory T cells (Tregs). Moreover, M1 macrophages exhibited consistent differential changes when the four DEGs varied. M2 macrophages, resting mast cells, neutrophils, and activated memory CD4 T cells all presented obvious differences in three of the DEGs. Research by Chen revealed that macrophages can polarize into two phenotypes, M1 and M2. M1 macrophages are involved mainly in pro-inflammatory responses and can secrete pro-inflammatory factors such as IL-6, IL-12, and TNF. M2 macrophages are involved mainly in anti-inflammatory responses and contribute to tissue repair, with the characteristic expression of arginase-1 (Arg-1), mannose receptor (CD206), and the anti-inflammatory factor IL-10. IBD is an intestinal inflammatory disorder, and the inflammatory microenvironment in the intestine is closely related to macrophage polarization [<a href="#CR44" class="usa-link" aria-describedby="CR44">44</a>, <a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>]. On the basis of this author’s research findings, we speculate that the overactivation of M1 macrophages may exacerbate the intestinal inflammatory response in IBD patients, leading to tissue damage. In contrast, the anti-inflammatory and tissue-repair functions of M2 macrophages may help alleviate the inflammatory symptoms of IBD and promote the repair of intestinal tissue. Regrettably, none of these issues have been verified, and therefore, it cannot be simply stated that M1/M2 macrophages are closely associated with IBD. However, through bioinformatics analysis methods and the use of GEO data, the results of our comprehensive analysis indicate that macrophage polarization plays an important role in IBD [<a href="#CR45" class="usa-link" aria-describedby="CR45">45</a>].</p>
<p id="Par59">Zhen’s research revealed that, in colorectal cancer patients, the density of mast cells is lower than that in normal tissues, and their phenotype undergoes substantial changes, shifting from a quiescent state to an activated state. In the tumor microenvironment, activated mast cells release a variety of bioactive substances, including histamine, cytokines, proteases, and lipid mediators, which are capable of triggering inflammatory responses [<a href="#CR46" class="usa-link" aria-describedby="CR46">46</a>]. On the basis of these findings, we hypothesize that the abnormal expression of DEGs leads to the overactivation of immune cells. These activated immune cells then release active substances, thereby contributing to the onset and progression of IBD. Our findings indicate that mast cells exhibit significant differences in the expression of numerous DEGs [<a href="#CR47" class="usa-link" aria-describedby="CR47">47</a>].</p>
<p id="Par60">Neutrophils play a key role in the innate immune system of the intestine [<a href="#CR48" class="usa-link" aria-describedby="CR48">48</a>]. Camille’s study demonstrated that neutrophil infiltration serves as a marker of disease activity in IBD patients. Neutrophils can release various inflammatory mediators, such as ROS, cytotoxic particle contents, and neutrophil extracellular traps (NETs), thereby triggering inflammatory responses and causing tissue damage [<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a>, <a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>]. However, simultaneously, neutrophils are also essential for maintaining the intestinal barrier, host defense, and reducing inflammation. For example, CD177 + neutrophils have a protective effect. This study also revealed that neutrophil infiltration is associated with disease severity and that excessive neutrophil infiltration may lead to treatment failure. In CD, the dysfunction and reduced recruitment of neutrophils result in delayed bacterial clearance and the persistent presence of antigens, which trigger adaptive immune responses and the formation of granulomas. Additionally, the gut microbiota can regulate the production, function, and maturation of neutrophils, and neutrophils also influence the composition and function of the microbiota. These two factors interact and jointly affect the development of IBD [<a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>].</p>
<p id="Par61">CD4 + T cells display distinct characteristics in CD and UC. In CD patients, there is a significant expansion of CD4 + tissue-resident memory T (Trm) cells in the gut. Notably, CD4 + Trm subsets expressing CD161 and CCR5 in CD patients exhibit stronger cytotoxicity and are associated with disease activity. In contrast, CD45RA + CCR7 + naive CD4 + T cells and CXCR5 + T follicular helper (Tfh) cells are abundant in the intestines of UC patients. Moreover, CD4 + Trm (CDtrm) cells possess innate-like cell properties. These cells can rapidly secrete inflammatory cytokines such as IFN-γ upon cytokine stimulation without the need for T-cell receptor (TCR) activation. This ability can cause damage to intestinal epithelial cells. In UC, the increase in Tfh cells may be linked to the increase in pathological IgG + plasma cells [<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a>].</p>
<p id="Par62">Currently, in clinical practice, achieving an accurate diagnosis of IBD remains a challenge. The diagnosis relies mainly on medical history, clinical symptoms, laboratory tests, imaging examinations, endoscopy, and histological examinations. Colonoscopy is widely regarded as the “gold standard” for diagnosing IBD. However, colonoscopy is time-consuming and highly dependent on the diagnostic experience of relevant physicians. Additionally, the intestinal wall of IBD patients in severe stages is extremely fragile, and improper procedures during colonoscopy can lead to complications such as bleeding and perforation. Therefore, our objective is to identify more reliable and sensitive biomarkers that can directly reflect the different stages of IBD and its underlying disease mechanisms. To this end, we carried out rigorous screening in databases such as the GEO, IBDDMB, and the UKB. We imposed strict restrictions on the number of patients included in the study. Only patients from cohorts with a population size greater than 100 were considered, and the scope of inclusion covered all stages of IBD onset and the entire intestinal tissue. Unfortunately, despite obtaining a large amount of data from public databases for analysis and striving to include as much complete data as possible for real-world validation, our sample size is too small to rule out the possibility of data randomness. However, we have conducted multiple experiments to minimize errors as much as possible. Notably, we conducted a comprehensive analysis of existing IBD data by multiple methods and genetic-based research on specific immune cells exhibiting differences in the immune response, which is unprecedented. This method provides a new research perspective for the study of IBD.</p>
<p id="Par63">Currently, ANNs, machine learning, interpretable analysis, and multichip joint analysis are gaining popularity in medical research. Given the complex genetic basis of IBD and the limitations of existing diagnostic tools, the diagnosis of IBD often poses significant challenges. By analyzing extensive genomic data, ANNs and machine learning hold great promise for increasing diagnostic efficiency and accuracy. They can also identify complex multiallele patterns that may be associated with specific diseases. Through interpretable analysis, detailed insights can be provided into the contribution of DEGs within diagnostic models, thereby offering explanations for machine learning models [<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a>]. The application of artificial neural networks and computer technologies such as machine learning and deep learning to assist clinical doctors in disease diagnosis and treatment has become increasingly prevalent. Continuously evolving artificial intelligence enables more precise and scientific diagnosis and treatment, ultimately benefiting patients in need of medical care.</p></section><section id="Sec24"><h2 class="pmc_sec_title">Conclusion</h2>
<p id="Par64">In this study, we developed a diagnostic model for predicting IBD with a comprehensive ANN, machine learning, interpretable analysis, and multichip joint analysis methods. ANN, LASSO, SVM, and random forest algorithms were used for genetic feature selection. We identified four key biomarkers (LOC389023, DUOX2, LCN2, and DEFA6) and used ten machine learning methods and SHAP models to assist in IBD diagnosis, clarifying the genetic characteristics, molecular pathways, and differentially abundant immune cells of IBD. These genes were demonstrated to be related to immune system function. In addition, we conducted multiple validations with the dataset. Our findings indicate that machine learning algorithms can facilitate accurate diagnostic decisions for IBD, enabling clinicians to explore new treatment pathways and diagnostic methods.</p></section><section id="Sec25"><h2 class="pmc_sec_title">Electronic supplementary material</h2>
<p>Below is the link to the electronic supplementary material.</p>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12366088/bin/12967_2025_6838_MOESM1_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (556.7KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12366088/bin/12967_2025_6838_MOESM2_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (531KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12366088/bin/12967_2025_6838_MOESM3_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (511.4KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12366088/bin/12967_2025_6838_MOESM4_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4</a><sup> (517.9KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12366088/bin/12967_2025_6838_MOESM5_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5</a><sup> (520.5KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM6"><div class="media p"><div class="caption">
<a href="/articles/instance/12366088/bin/12967_2025_6838_MOESM6_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6</a><sup> (516.6KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM7"><div class="media p"><div class="caption">
<a href="/articles/instance/12366088/bin/12967_2025_6838_MOESM7_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 7</a><sup> (490.6KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM8"><div class="media p"><div class="caption">
<a href="/articles/instance/12366088/bin/12967_2025_6838_MOESM8_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 8</a><sup> (506.9KB, tif) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>Throughout the entire research process, all the authors not only provided professional academic guidance, helping to solve the challenges in experimental design and data analysis, but also patiently offered numerous revision suggestions during the paper-writing stage. This has significantly increased the quality of the paper. Additionally, they provided financial support.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>Yan Chaosheng: Data curation, Methodology, Writing - Original draft preparation, Formal analysis, and Participation in experimental design, sample collection, and conducting experiments. Sun Haowen: Resources and Writing - Original draft.Investigation and Participation in experimental design and conducting experiments. Rao Jingjing: Resources and Writing - Original draft. Dai Yuanyuan: Writing - Reviewing &amp; Editing. Duan Wenhui: Investigation. Sheng Yinyue: Methodology, Writing - Review &amp; Editing, Project administration, and Funding acquisition. Xue Yuzheng: Methodology, Writing - Review &amp; Editing, Project administration, and Funding acquisition.</p></section><section id="notes2"><h2 class="pmc_sec_title">Funding</h2>
<p>National Natural Science Foundation of China (32372302). National Natural Science Foundation of China (82405210).</p></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>The <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87466" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE87466</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE179285" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE179285</a>, and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87473" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE87473</a> microarray datasets used in this study were downloaded from the Gene Expression Omnibus database.</p></section><section id="notes4"><h2 class="pmc_sec_title">Declarations</h2>
<section id="FPar3"><h3 class="pmc_sec_title">Ethics approval and consent to participate</h3>
<p id="Par65">According to local legislation and institutional requirements, research on human participants requires ethical review and approval. In compliance with the ethical requirements of Jiangnan University Affiliated Hospital in Wuxi, Jiangsu Province, the ethical approval number for this study is LS2024012.</p></section><section id="FPar4"><h3 class="pmc_sec_title">Consent for publication</h3>
<p id="Par66">Not required.</p></section><section id="FPar2"><h3 class="pmc_sec_title">Competing interests</h3>
<p id="Par67">The author declares that there are no conflicts of interest.</p></section></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm">
<div class="fn p" id="fn1">
<p><strong>Publisher’s note</strong></p>
<p>Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p>
</div>
<div class="fn p" id="fn2"><p>Yan Chaosheng and Sun Haowen have contributed equally to this work.</p></div>
</div></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Saeid Seyedian S, Alimentary tract research center, ahvaz jundishapur university of medical science ahvaz. A review of the diagnosis, prevention, and treatment methods of inflammatory bowel disease[J/OL]. J Med Life. 2019;12(2):113–22. 10.25122/jml-2018-0075
</cite> [<a href="https://doi.org/10.25122/jml-2018-0075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6685307/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31406511/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saeid%20Seyedian%20S,%20Alimentary%20tract%20research%20center,%20ahvaz%20jundishapur%20university%20of%20medical%20science%20ahvaz.%20A%20review%20of%20the%20diagnosis,%20prevention,%20and%20treatment%20methods%20of%20inflammatory%20bowel%20disease%5BJ/OL%5D.%20J%20Med%20Life.%202019;12(2):113%E2%80%9322.%2010.25122/jml-2018-0075" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Hodson R. Inflammatory bowel disease[J/OL]. Nature. 2016;540(7634):S97–97. 10.1038/540S97a
</cite> [<a href="https://doi.org/10.1038/540S97a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28002398/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hodson%20R.%20Inflammatory%20bowel%20disease%5BJ/OL%5D.%20Nature.%202016;540(7634):S97%E2%80%9397.%2010.1038/540S97a" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>Diez-Martin E, Hernandez-Suarez L, MuÑoz-Villafranca C, et al. Inflammatory bowel disease: A comprehensive analysis of molecular bases, predictive biomarkers, diagnostic methods, and therapeutic Options[J/OL]. Int J Mol Sci. 2024;25(13):7062. 10.3390/ijms25137062
</cite> [<a href="https://doi.org/10.3390/ijms25137062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11241012/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39000169/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Diez-Martin%20E,%20Hernandez-Suarez%20L,%20Mu%C3%91oz-Villafranca%20C,%20et%20al.%20Inflammatory%20bowel%20disease:%20A%20comprehensive%20analysis%20of%20molecular%20bases,%20predictive%20biomarkers,%20diagnostic%20methods,%20and%20therapeutic%20Options%5BJ/OL%5D.%20Int%20J%20Mol%20Sci.%202024;25(13):7062.%2010.3390/ijms25137062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Bisgaard T H, Allin K H Keeferl, et al. Depression and anxiety in inflammatory bowel disease: epidemiology, mechanisms and treatment[J/OL]. Nat Reviews Gastroenterol Hepatol. 2022;19(11):717–26. 10.1038/s41575-022-00634-6</cite> [<a href="https://doi.org/10.1038/s41575-022-00634-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35732730/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bisgaard%20T%20H,%20Allin%20K%20H%20Keeferl,%20et%20al.%20Depression%20and%20anxiety%20in%20inflammatory%20bowel%20disease:%20epidemiology,%20mechanisms%20and%20treatment%5BJ/OL%5D.%20Nat%20Reviews%20Gastroenterol%20Hepatol.%202022;19(11):717%E2%80%9326.%2010.1038/s41575-022-00634-6" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Eftekhar Z, Aghaei M, Saki N. DNA damage repair in megakaryopoiesis: molecular and clinical aspects[J/OL]. Expert Rev Hematol. 2024;17(10):705–12. 10.1080/17474086.2024.2391102
</cite> [<a href="https://doi.org/10.1080/17474086.2024.2391102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39117495/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Eftekhar%20Z,%20Aghaei%20M,%20Saki%20N.%20DNA%20damage%20repair%20in%20megakaryopoiesis:%20molecular%20and%20clinical%20aspects%5BJ/OL%5D.%20Expert%20Rev%20Hematol.%202024;17(10):705%E2%80%9312.%2010.1080/17474086.2024.2391102" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>He X. Decoding the IBD paradox: A triadic interplay between REG3, enterococci, and NOD2[J/OL]. Cell Host Microbe. 2023;31(9):1425–7. 10.1016/j.chom.2023.08.008
</cite> [<a href="https://doi.org/10.1016/j.chom.2023.08.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37708849/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?He%20X.%20Decoding%20the%20IBD%20paradox:%20A%20triadic%20interplay%20between%20REG3,%20enterococci,%20and%20NOD2%5BJ/OL%5D.%20Cell%20Host%20Microbe.%202023;31(9):1425%E2%80%937.%2010.1016/j.chom.2023.08.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Gilliland A, Chan J J, De Wolfe T J, et al. Pathobionts in inflammatory bowel disease: origins, underlying mechanisms, and implications for clinical Care[J/OL]. Gastroenterology. 2024;166(1):44–58. 10.1053/j.gastro.2023.09.019
</cite> [<a href="https://doi.org/10.1053/j.gastro.2023.09.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37734419/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gilliland%20A,%20Chan%20J%20J,%20De%20Wolfe%20T%20J,%20et%20al.%20Pathobionts%20in%20inflammatory%20bowel%20disease:%20origins,%20underlying%20mechanisms,%20and%20implications%20for%20clinical%20Care%5BJ/OL%5D.%20Gastroenterology.%202024;166(1):44%E2%80%9358.%2010.1053/j.gastro.2023.09.019" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>CosÍn-Roger J. Inflammatory bowel disease: immune function, tissue fibrosis and current Therapies[J/OL]. Int J Mol Sci. 2024;25(12):6416. 10.3390/ijms25126416
</cite> [<a href="https://doi.org/10.3390/ijms25126416" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11203598/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38928122/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cos%C3%8Dn-Roger%20J.%20Inflammatory%20bowel%20disease:%20immune%20function,%20tissue%20fibrosis%20and%20current%20Therapies%5BJ/OL%5D.%20Int%20J%20Mol%20Sci.%202024;25(12):6416.%2010.3390/ijms25126416" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Agrawal M, Allin K H, Petralia F, et al. Multiomics to elucidate inflammatory bowel disease risk factors and pathways[J/OL]. Nat Reviews Gastroenterol Hepatol. 2022;19(6):399–409. 10.1038/s41575-022-00593-y</cite> [<a href="https://doi.org/10.1038/s41575-022-00593-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9214275/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35301463/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Agrawal%20M,%20Allin%20K%20H,%20Petralia%20F,%20et%20al.%20Multiomics%20to%20elucidate%20inflammatory%20bowel%20disease%20risk%20factors%20and%20pathways%5BJ/OL%5D.%20Nat%20Reviews%20Gastroenterol%20Hepatol.%202022;19(6):399%E2%80%93409.%2010.1038/s41575-022-00593-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Agrawal M, Spencer E A, Colombel J F, et al. Approach to the management of recently diagnosed inflammatory bowel disease patients: A user’s guide for adult and pediatric Gastroenterologists[J/OL]. Gastroenterology. 2021;161(1):47–65. 10.1053/j.gastro.2021.04.063
</cite> [<a href="https://doi.org/10.1053/j.gastro.2021.04.063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8640961/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33940007/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Agrawal%20M,%20Spencer%20E%20A,%20Colombel%20J%20F,%20et%20al.%20Approach%20to%20the%20management%20of%20recently%20diagnosed%20inflammatory%20bowel%20disease%20patients:%20A%20user%E2%80%99s%20guide%20for%20adult%20and%20pediatric%20Gastroenterologists%5BJ/OL%5D.%20Gastroenterology.%202021;161(1):47%E2%80%9365.%2010.1053/j.gastro.2021.04.063" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Torabizadeh M, Aghaei M, Saki N, et al. The association of nasal and blood eosinophils with serum IgE level in allergic rhinitis and asthma: A Case-Control Study[J/OL]. Health Sci Rep. 2024;7(11):e70191. 10.1002/hsr2.70191
</cite> [<a href="https://doi.org/10.1002/hsr2.70191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11541053/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39512245/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Torabizadeh%20M,%20Aghaei%20M,%20Saki%20N,%20et%20al.%20The%20association%20of%20nasal%20and%20blood%20eosinophils%20with%20serum%20IgE%20level%20in%20allergic%20rhinitis%20and%20asthma:%20A%20Case-Control%20Study%5BJ/OL%5D.%20Health%20Sci%20Rep.%202024;7(11):e70191.%2010.1002/hsr2.70191" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Stidham RW. Artificial intelligence for disease assessment in inflammatory bowel disease: how will it change our Practice?[J/OL]. Gastroenterology. 2022;162(5):1493–506. 10.1053/j.gastro.2021.12.238
</cite> [<a href="https://doi.org/10.1053/j.gastro.2021.12.238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8997186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34995537/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stidham%20RW.%20Artificial%20intelligence%20for%20disease%20assessment%20in%20inflammatory%20bowel%20disease:%20how%20will%20it%20change%20our%20Practice?%5BJ/OL%5D.%20Gastroenterology.%202022;162(5):1493%E2%80%93506.%2010.1053/j.gastro.2021.12.238" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Renganathan V. Overview of artificial neural network models in the biomedical domain[J/OL]. Bratislava Med J, 2019, 120(07): 536–40. 10.4149/BLL_2019_087</cite> [<a href="https://doi.org/10.4149/BLL_2019_087" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31602991/" class="usa-link">PubMed</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Shen M, Zhang Y, Zhan R, et al. Predicting the risk of cardiovascular disease in adults exposed to heavy metals: interpretable machine learning[J/OL]. Ecotoxicol Environ Saf. 2025;290:117570. 10.1016/j.ecoenv.2024.117570
</cite> [<a href="https://doi.org/10.1016/j.ecoenv.2024.117570" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39721423/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shen%20M,%20Zhang%20Y,%20Zhan%20R,%20et%20al.%20Predicting%20the%20risk%20of%20cardiovascular%20disease%20in%20adults%20exposed%20to%20heavy%20metals:%20interpretable%20machine%20learning%5BJ/OL%5D.%20Ecotoxicol%20Environ%20Saf.%202025;290:117570.%2010.1016/j.ecoenv.2024.117570" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Qiu P, Ishimoto T. The gut microbiota in inflammatory bowel Disease[J/OL]. Front Cell Infect Microbiol. 2022;12:733992. 10.3389/fcimb.2022.733992
</cite> [<a href="https://doi.org/10.3389/fcimb.2022.733992" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8902753/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35273921/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qiu%20P,%20Ishimoto%20T.%20The%20gut%20microbiota%20in%20inflammatory%20bowel%20Disease%5BJ/OL%5D.%20Front%20Cell%20Infect%20Microbiol.%202022;12:733992.%2010.3389/fcimb.2022.733992" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Rajabi L, Ebrahimdoost M, Mohammadi SA, et al. Aqueous and ethanolic extracts of Moringa oleifera leaves induce selective cytotoxicity in Raji and Jurkat cell lines by activating the P21 pathway independent of P53[J/OL]. Mol Biol Rep. 2025;52(1). 10.1007/s11033-024-10200-9</cite> [<a href="https://doi.org/10.1007/s11033-024-10200-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39760794/" class="usa-link">PubMed</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Torfi E, Bahreiny S S, Saki N, et al. Evaluation of Pro-BNP biomarker in heart failure patients and its relationship with complete blood count parameters: A case–control study[J/OL]. Health Sci Rep. 2024;7(9):e70083. 10.1002/hsr2.70083
</cite> [<a href="https://doi.org/10.1002/hsr2.70083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11424362/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39328979/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Torfi%20E,%20Bahreiny%20S%20S,%20Saki%20N,%20et%20al.%20Evaluation%20of%20Pro-BNP%20biomarker%20in%20heart%20failure%20patients%20and%20its%20relationship%20with%20complete%20blood%20count%20parameters:%20A%20case%E2%80%93control%20study%5BJ/OL%5D.%20Health%20Sci%20Rep.%202024;7(9):e70083.%2010.1002/hsr2.70083" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Aghaei M, Bahreiny S S, Zayeri Z D, et al. Evaluation of complete blood count parameters in patients with diabetes mellitus: A systematic Review[J/OL]. Health Sci Rep. 2025;8(2). 10.1002/hsr2.70488</cite> [<a href="https://doi.org/10.1002/hsr2.70488" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11847716/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39995796/" class="usa-link">PubMed</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Tushir JS. Chromatin-bound RNA and the neurobiology of psychiatric disease[J/OL]. Neuroscience. 2014;264:131–41. 10.1016/j.neuroscience.2013.06.051
</cite> [<a href="https://doi.org/10.1016/j.neuroscience.2013.06.051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3844067/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23831425/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tushir%20JS.%20Chromatin-bound%20RNA%20and%20the%20neurobiology%20of%20psychiatric%20disease%5BJ/OL%5D.%20Neuroscience.%202014;264:131%E2%80%9341.%2010.1016/j.neuroscience.2013.06.051" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Magro D O, Sassaki L Y, Chebli J M F. Interaction between diet and genetics in patients with inflammatory bowel disease[J/OL]. World J Gastroenterol. 2024;30(12):1644–50. 10.3748/wjg.v30.i12.1644
</cite> [<a href="https://doi.org/10.3748/wjg.v30.i12.1644" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11008370/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38617734/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Magro%20D%20O,%20Sassaki%20L%20Y,%20Chebli%20J%20M%20F.%20Interaction%20between%20diet%20and%20genetics%20in%20patients%20with%20inflammatory%20bowel%20disease%5BJ/OL%5D.%20World%20J%20Gastroenterol.%202024;30(12):1644%E2%80%9350.%2010.3748/wjg.v30.i12.1644" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Zeng B, Huang Y, Chen S, et al. Dextran sodium sulfate potentiates NLRP3 inflammasome activation by modulating the KCa3.1 potassium channel in a mouse model of colitis[J/OL]. Cell Mol Immunol. 2022;19(8):925–43. 10.1038/s41423-022-00891-0
</cite> [<a href="https://doi.org/10.1038/s41423-022-00891-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9338299/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35799057/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zeng%20B,%20Huang%20Y,%20Chen%20S,%20et%20al.%20Dextran%20sodium%20sulfate%20potentiates%20NLRP3%20inflammasome%20activation%20by%20modulating%20the%20KCa3.1%20potassium%20channel%20in%20a%20mouse%20model%20of%20colitis%5BJ/OL%5D.%20Cell%20Mol%20Immunol.%202022;19(8):925%E2%80%9343.%2010.1038/s41423-022-00891-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>Ohya S. Physiological role of K<sup>+</sup> channels in the regulation of T cell Function[J/OL]. Yakugaku Zasshi, 2016, 136(3): 479–83. 10.1248/yakushi.15-00246-4</cite> [<a href="https://doi.org/10.1248/yakushi.15-00246-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26935090/" class="usa-link">PubMed</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Gupta U, Wallace C Ghoshs. Increased LCN2 (lipocalin 2) in the RPE decreases autophagy and activates inflammasome-ferroptosis processes in a mouse model of dry. AMD[J/OL] Autophagy. 2023;19(1):92–111. 10.1080/15548627.2022.2062887
</cite> [<a href="https://doi.org/10.1080/15548627.2022.2062887" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9809950/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35473441/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gupta%20U,%20Wallace%20C%20Ghoshs.%20Increased%20LCN2%20(lipocalin%202)%20in%20the%20RPE%20decreases%20autophagy%20and%20activates%20inflammasome-ferroptosis%20processes%20in%20a%20mouse%20model%20of%20dry.%20AMD%5BJ/OL%5D%20Autophagy.%202023;19(1):92%E2%80%93111.%2010.1080/15548627.2022.2062887" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Li J, Simmons A J, Hawkins C V, et al. Identification and multimodal characterization of a specialized epithelial cell type associated with crohn’s disease[J/OL]. Nat Commun. 2024;15(1):7204. 10.1038/s41467-024-51580-7
</cite> [<a href="https://doi.org/10.1038/s41467-024-51580-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11339313/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39169060/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li%20J,%20Simmons%20A%20J,%20Hawkins%20C%20V,%20et%20al.%20Identification%20and%20multimodal%20characterization%20of%20a%20specialized%20epithelial%20cell%20type%20associated%20with%20crohn%E2%80%99s%20disease%5BJ/OL%5D.%20Nat%20Commun.%202024;15(1):7204.%2010.1038/s41467-024-51580-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Xiao X, Yeoh B S, Vijay-KUMAR M. Lipocalin 2: an emerging player in Iron homeostasis and Inflammation[J/OL]. Annu Rev Nutr. 2017;37(1):103–30. 10.1146/annurev-nutr-071816-064559
</cite> [<a href="https://doi.org/10.1146/annurev-nutr-071816-064559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28628361/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xiao%20X,%20Yeoh%20B%20S,%20Vijay-KUMAR%20M.%20Lipocalin%202:%20an%20emerging%20player%20in%20Iron%20homeostasis%20and%20Inflammation%5BJ/OL%5D.%20Annu%20Rev%20Nutr.%202017;37(1):103%E2%80%9330.%2010.1146/annurev-nutr-071816-064559" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>Wu D, Wang X, Han Y, et al. The effect of lipocalin-2 (LCN2) on apoptosis: a proteomics analysis study in an LCN2 deficient mouse model[J/OL]. BMC Genomics. 2021;22(1):892. 10.1186/s12864-021-08211-y
</cite> [<a href="https://doi.org/10.1186/s12864-021-08211-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8670060/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34903175/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wu%20D,%20Wang%20X,%20Han%20Y,%20et%20al.%20The%20effect%20of%20lipocalin-2%20(LCN2)%20on%20apoptosis:%20a%20proteomics%20analysis%20study%20in%20an%20LCN2%20deficient%20mouse%20model%5BJ/OL%5D.%20BMC%20Genomics.%202021;22(1):892.%2010.1186/s12864-021-08211-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Yang Y, Li S, Liu K et al. Lipocalin-2-mediated intestinal epithelial cells pyroptosis via NF-κB/NLRP3/GSDMD signaling axis adversely affects inflammation in colitis[J/OL]. Biochimica et biophysica acta (BBA) - Molecular basis of disease, 2024, 1870(7): 167279. 10.1016/j.bbadis.2024.167279</cite> [<a href="https://doi.org/10.1016/j.bbadis.2024.167279" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38844113/" class="usa-link">PubMed</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Huang Q, Xing J, Li G, et al. LCN2 regulates the gut microbiota and metabolic profile in mice infected with <em>Mycobacterium bovis</em>[J/OL]. mSystems. 2024;9(8):e00501–24. 10.1128/msystems.00501-24
</cite> [<a href="https://doi.org/10.1128/msystems.00501-24" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11334432/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39051782/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Huang%20Q,%20Xing%20J,%20Li%20G,%20et%20al.%20LCN2%20regulates%20the%20gut%20microbiota%20and%20metabolic%20profile%20in%20mice%20infected%20with%20Mycobacterium%20bovis%5BJ/OL%5D.%20mSystems.%202024;9(8):e00501%E2%80%9324.%2010.1128/msystems.00501-24" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Then A, Goenawan H, Lesmana R, et al. Exploring the potential regulation of DUOX in thyroid hormone–autophagy signaling via IGF–1 in the skeletal muscle (Review)[J/OL]. Biomedical Rep. 2024;22(3):39. 10.3892/br.2024.1917</cite> [<a href="https://doi.org/10.3892/br.2024.1917" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11704872/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39781041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Then%20A,%20Goenawan%20H,%20Lesmana%20R,%20et%20al.%20Exploring%20the%20potential%20regulation%20of%20DUOX%20in%20thyroid%20hormone%E2%80%93autophagy%20signaling%20via%20IGF%E2%80%931%20in%20the%20skeletal%20muscle%20(Review)%5BJ/OL%5D.%20Biomedical%20Rep.%202024;22(3):39.%2010.3892/br.2024.1917" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Grasberger H, Magis A T Shenge, et al. DUOX2 variants associate with preclinical disturbances in microbiota-immune homeostasis and increased inflammatory bowel disease risk[J/OL]. J Clin Invest. 2021;131(9):e141676. 10.1172/JCI141676
</cite> [<a href="https://doi.org/10.1172/JCI141676" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8087203/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33651715/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Grasberger%20H,%20Magis%20A%20T%20Shenge,%20et%20al.%20DUOX2%20variants%20associate%20with%20preclinical%20disturbances%20in%20microbiota-immune%20homeostasis%20and%20increased%20inflammatory%20bowel%20disease%20risk%5BJ/OL%5D.%20J%20Clin%20Invest.%202021;131(9):e141676.%2010.1172/JCI141676" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Kasumba D M, Huot S. Duox2 regulates secreted factors in virus-infected respiratory epithelial cells that contribute to neutrophil attraction and activation[J/OL]. FASEB J. 2023;37(2):e22765. 10.1096/fj.202201205R
</cite> [<a href="https://doi.org/10.1096/fj.202201205R" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10107641/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36607642/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kasumba%20D%20M,%20Huot%20S.%20Duox2%20regulates%20secreted%20factors%20in%20virus-infected%20respiratory%20epithelial%20cells%20that%20contribute%20to%20neutrophil%20attraction%20and%20activation%5BJ/OL%5D.%20FASEB%20J.%202023;37(2):e22765.%2010.1096/fj.202201205R" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Pillay Geerdinkrj. Neutrophils in respiratory syncytial virus infection: A target for asthma prevention[J/OL]. J Allergy Clin Immunol. 2015;136(4):838–47. 10.1016/j.jaci.2015.06.034
</cite> [<a href="https://doi.org/10.1016/j.jaci.2015.06.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7112351/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26277597/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Pillay%20Geerdinkrj.%20Neutrophils%20in%20respiratory%20syncytial%20virus%20infection:%20A%20target%20for%20asthma%20prevention%5BJ/OL%5D.%20J%20Allergy%20Clin%20Immunol.%202015;136(4):838%E2%80%9347.%2010.1016/j.jaci.2015.06.034" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Peters C, Nicholas A K, Schoenmakers E et al. <em>DUOX2</em> / <em>DUOXA2</em> Mutations Frequently Cause Congenital Hypothyroidism that Evades Detection on Newborn Screening in the United Kingdom[J/OL]. Thyroid, 2019;29(6):790–801. 10.1089/thy.2018.0587</cite> [<a href="https://doi.org/10.1089/thy.2018.0587" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6588112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31044655/" class="usa-link">PubMed</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>BurgueÑo JF, Fritsch J, GonzÁlez E E, et al. Epithelial TLR4 signaling activates DUOX2 to induce Microbiota-Driven Tumorigenesis[J/OL]. Gastroenterology. 2021;160(3):797–e8086. 10.1053/j.gastro.2020.10.031
</cite> [<a href="https://doi.org/10.1053/j.gastro.2020.10.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7879481/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33127391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Burgue%C3%91o%20JF,%20Fritsch%20J,%20Gonz%C3%81lez%20E%20E,%20et%20al.%20Epithelial%20TLR4%20signaling%20activates%20DUOX2%20to%20induce%20Microbiota-Driven%20Tumorigenesis%5BJ/OL%5D.%20Gastroenterology.%202021;160(3):797%E2%80%93e8086.%2010.1053/j.gastro.2020.10.031" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Simms L A, Doecke J D, Walsh M D, et al. Reduced -defensin expression is associated with inflammation and not NOD2 mutation status in ileal crohn’s disease[J/OL]. Gut. 2008;57(7):903–10. 10.1136/gut.2007.142588
</cite> [<a href="https://doi.org/10.1136/gut.2007.142588" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18305068/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Simms%20L%20A,%20Doecke%20J%20D,%20Walsh%20M%20D,%20et%20al.%20Reduced%20-defensin%20expression%20is%20associated%20with%20inflammation%20and%20not%20NOD2%20mutation%20status%20in%20ileal%20crohn%E2%80%99s%20disease%5BJ/OL%5D.%20Gut.%202008;57(7):903%E2%80%9310.%2010.1136/gut.2007.142588" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Angriman I, Bordignon G, Kotsafti A et al. Innate immunity activation in newly diagnosed Ileocolonic crohn’s disease: A cohort Study[J/OL]. Diseases of the Colon &amp; Rectum, 2024[2025-03-11]. <a href="https://journals.lww.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://journals.lww.com/</a>10.1097/DCR.0000000000003145. DOI:10.1097/DCR.0000000000003145.</cite> [<a href="https://doi.org/10.1097/DCR.0000000000003145" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38319717/" class="usa-link">PubMed</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Gaudino SJ, Beaupre M, Lin X, et al. IL-22 receptor signaling in Paneth cells is critical for their maturation, microbiota colonization, Th17-related immune responses, and anti-Salmonella immunity[J/OL]. Mucosal Immunol. 2021;14(2):389–401. 10.1038/s41385-020-00348-5
</cite> [<a href="https://doi.org/10.1038/s41385-020-00348-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7946635/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33060802/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Gaudino%20SJ,%20Beaupre%20M,%20Lin%20X,%20et%20al.%20IL-22%20receptor%20signaling%20in%20Paneth%20cells%20is%20critical%20for%20their%20maturation,%20microbiota%20colonization,%20Th17-related%20immune%20responses,%20and%20anti-Salmonella%20immunity%5BJ/OL%5D.%20Mucosal%20Immunol.%202021;14(2):389%E2%80%93401.%2010.1038/s41385-020-00348-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Degruttola A K, Low D, Mizoguchi A, et al. Current Understanding of dysbiosis in disease in human and animal Models:[J/OL]. Inflamm Bowel Dis. 2016;22(5):1137–50. 10.1097/MIB.0000000000000750
</cite> [<a href="https://doi.org/10.1097/MIB.0000000000000750" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4838534/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27070911/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Degruttola%20A%20K,%20Low%20D,%20Mizoguchi%20A,%20et%20al.%20Current%20Understanding%20of%20dysbiosis%20in%20disease%20in%20human%20and%20animal%20Models:%5BJ/OL%5D.%20Inflamm%20Bowel%20Dis.%202016;22(5):1137%E2%80%9350.%2010.1097/MIB.0000000000000750" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Eng S J, Nonnecke E B, De Lorimier A J et al. FOXO inhibition rescues α-defensin expression in human intestinal organoids[J/OL]. Proceedings of the National Academy of Sciences, 2023, 120(47): e2312453120. 10.1073/pnas.2312453120</cite> [<a href="https://doi.org/10.1073/pnas.2312453120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10666032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37956278/" class="usa-link">PubMed</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Moschen A R, Tilg H, IL-12 Rainet. IL-23 and IL-17 in IBD: immunobiology and therapeutic targeting[J/OL]. Nat Reviews Gastroenterol Hepatol. 2019;16(3):185–96. 10.1038/s41575-018-0084-8</cite> [<a href="https://doi.org/10.1038/s41575-018-0084-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30478416/" class="usa-link">PubMed</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Schmitt H, Neurath M F Atreyar. Role of the IL23/IL17 pathway in crohn’s Disease[J/OL]. Front Immunol. 2021;12:622934. 10.3389/fimmu.2021.622934
</cite> [<a href="https://doi.org/10.3389/fimmu.2021.622934" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8042267/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33859636/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Schmitt%20H,%20Neurath%20M%20F%20Atreyar.%20Role%20of%20the%20IL23/IL17%20pathway%20in%20crohn%E2%80%99s%20Disease%5BJ/OL%5D.%20Front%20Immunol.%202021;12:622934.%2010.3389/fimmu.2021.622934" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Deng Z, Wang S, Wu C, et al. IL-17 inhibitor-associated inflammatory bowel disease: A study based on literature and database analysis[J/OL]. Front Pharmacol. 2023;14:1124628. 10.3389/fphar.2023.1124628
</cite> [<a href="https://doi.org/10.3389/fphar.2023.1124628" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10076642/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37033665/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Deng%20Z,%20Wang%20S,%20Wu%20C,%20et%20al.%20IL-17%20inhibitor-associated%20inflammatory%20bowel%20disease:%20A%20study%20based%20on%20literature%20and%20database%20analysis%5BJ/OL%5D.%20Front%20Pharmacol.%202023;14:1124628.%2010.3389/fphar.2023.1124628" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Nanki K, Fujii M, Shimokawa M, et al. Somatic inflammatory gene mutations in human ulcerative colitis epithelium[J/OL]. Nature. 2020;577(7789):254–9. 10.1038/s41586-019-1844-5
</cite> [<a href="https://doi.org/10.1038/s41586-019-1844-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31853059/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nanki%20K,%20Fujii%20M,%20Shimokawa%20M,%20et%20al.%20Somatic%20inflammatory%20gene%20mutations%20in%20human%20ulcerative%20colitis%20epithelium%5BJ/OL%5D.%20Nature.%202020;577(7789):254%E2%80%939.%2010.1038/s41586-019-1844-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Shapouri-Moghaddam A, Mohammadian S, Vazini H, et al. Macrophage plasticity, polarization, and function in health and disease[J/OL]. J Cell Physiol. 2018;233(9):6425–40. 10.1002/jcp.26429
</cite> [<a href="https://doi.org/10.1002/jcp.26429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29319160/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Shapouri-Moghaddam%20A,%20Mohammadian%20S,%20Vazini%20H,%20et%20al.%20Macrophage%20plasticity,%20polarization,%20and%20function%20in%20health%20and%20disease%5BJ/OL%5D.%20J%20Cell%20Physiol.%202018;233(9):6425%E2%80%9340.%2010.1002/jcp.26429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>Yunna C, Mengru H, Lei W, et al. Macrophage M1/M2 polarization[J/OL]. Eur J Pharmacol. 2020;877:173090. 10.1016/j.ejphar.2020.173090
</cite> [<a href="https://doi.org/10.1016/j.ejphar.2020.173090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32234529/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Yunna%20C,%20Mengru%20H,%20Lei%20W,%20et%20al.%20Macrophage%20M1/M2%20polarization%5BJ/OL%5D.%20Eur%20J%20Pharmacol.%202020;877:173090.%2010.1016/j.ejphar.2020.173090" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Aponte-LÓpez A, MuÑoz-Cruz S. Mast Cells in the Tumor Microenvironment[M/OL]//BIRBRAIR A. Tumor Microenvironment: page 1273. Cham: Springer International Publishing, 2020: 159–173[2025-03-19]. <a href="http://link.springer.com/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://link.springer.com/</a>10.1007/978-3-030-49270-0_9. DOI:10.1007/978-3-030-49270-0_9.</cite> [<a href="https://doi.org/10.1007/978-3-030-49270-0_9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33119881/" class="usa-link">PubMed</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Xie Z, Niu L, Zheng G, et al. Single-cell analysis unveils activation of mast cells in colorectal cancer microenvironment[J/OL]. Cell Bioscience. 2023;13(1):217. 10.1186/s13578-023-01144-x
</cite> [<a href="https://doi.org/10.1186/s13578-023-01144-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10687892/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38031173/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Xie%20Z,%20Niu%20L,%20Zheng%20G,%20et%20al.%20Single-cell%20analysis%20unveils%20activation%20of%20mast%20cells%20in%20colorectal%20cancer%20microenvironment%5BJ/OL%5D.%20Cell%20Bioscience.%202023;13(1):217.%2010.1186/s13578-023-01144-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Liew P X Kubesp. The neutrophil’s role during health and Disease[J/OL]. Physiol Rev. 2019;99(2):1223–48. 10.1152/physrev.00012.2018
</cite> [<a href="https://doi.org/10.1152/physrev.00012.2018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30758246/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Liew%20P%20X%20Kubesp.%20The%20neutrophil%E2%80%99s%20role%20during%20health%20and%20Disease%5BJ/OL%5D.%20Physiol%20Rev.%202019;99(2):1223%E2%80%9348.%2010.1152/physrev.00012.2018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Papayannopoulos V. Neutrophils stepping through (to the other Side)[J/OL]. Immunity, 2018, 49(6): 992–4. 10.1016/j.immuni.2018.12.006</cite> [<a href="https://doi.org/10.1016/j.immuni.2018.12.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30566887/" class="usa-link">PubMed</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Danne C, Skerniskyte J, Marteyn B, et al. Neutrophils: from IBD to the gut microbiota[J/OL]. Nat Reviews Gastroenterol Hepatol. 2024;21(3):184–97. 10.1038/s41575-023-00871-3</cite> [<a href="https://doi.org/10.1038/s41575-023-00871-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38110547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Danne%20C,%20Skerniskyte%20J,%20Marteyn%20B,%20et%20al.%20Neutrophils:%20from%20IBD%20to%20the%20gut%20microbiota%5BJ/OL%5D.%20Nat%20Reviews%20Gastroenterol%20Hepatol.%202024;21(3):184%E2%80%9397.%2010.1038/s41575-023-00871-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Yokoi T, Murakami M, Kihara T et al. Identification of a unique subset of tissue-resident memory CD4 + T cells in Crohn’s disease[J/OL]. Proceedings of the National Academy of Sciences, 2023, 120(1): e2204269120. 10.1073/pnas.2204269120</cite> [<a href="https://doi.org/10.1073/pnas.2204269120" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9910620/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36574662/" class="usa-link">PubMed</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Roman-Naranjo P, Parra-Perez A M, Lopez-Escamez J A. A systematic review on machine learning approaches in the diagnosis and prognosis of rare genetic diseases[J/OL]. J Biomed Inform. 2023;143:104429. 10.1016/j.jbi.2023.104429
</cite> [<a href="https://doi.org/10.1016/j.jbi.2023.104429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37352901/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roman-Naranjo%20P,%20Parra-Perez%20A%20M,%20Lopez-Escamez%20J%20A.%20A%20systematic%20review%20on%20machine%20learning%20approaches%20in%20the%20diagnosis%20and%20prognosis%20of%20rare%20genetic%20diseases%5BJ/OL%5D.%20J%20Biomed%20Inform.%202023;143:104429.%2010.1016/j.jbi.2023.104429" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366088/bin/12967_2025_6838_MOESM1_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 1</a><sup> (556.7KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366088/bin/12967_2025_6838_MOESM2_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 2</a><sup> (531KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366088/bin/12967_2025_6838_MOESM3_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 3</a><sup> (511.4KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366088/bin/12967_2025_6838_MOESM4_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 4</a><sup> (517.9KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366088/bin/12967_2025_6838_MOESM5_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 5</a><sup> (520.5KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material6_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366088/bin/12967_2025_6838_MOESM6_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 6</a><sup> (516.6KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material7_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366088/bin/12967_2025_6838_MOESM7_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 7</a><sup> (490.6KB, tif) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material8_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12366088/bin/12967_2025_6838_MOESM8_ESM.tif" data-ga-action="click_feat_suppl" class="usa-link">Supplementary Material 8</a><sup> (506.9KB, tif) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87466" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE87466</a>, <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE179285" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE179285</a>, and <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE87473" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE87473</a> microarray datasets used in this study were downloaded from the Gene Expression Omnibus database.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Journal of Translational Medicine are provided here courtesy of <strong>BMC</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1186/s12967-025-06838-z"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/12967_2025_Article_6838.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (14.0 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12366088/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12366088/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12366088%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12366088/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12366088/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12366088/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40830794/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12366088/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40830794/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12366088/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12366088/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="jXVQSYpoa4KYkRsOg44tD6WvTWL3Yv3FqnZ6n2yVjM6ax48LjxcknkQOGix1JsAz">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
