
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            3D collagen high-throughput screen identifies drugs that induce epithelial polarity and enhance chemotherapy response in colorectal cancer - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="152536F38AF1C6D30536F3004F3D0906.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="commbiol">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373842/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Communications Biology">
<meta name="citation_title" content="3D collagen high-throughput screen identifies drugs that induce epithelial polarity and enhance chemotherapy response in colorectal cancer">
<meta name="citation_author" content="Sarah J Harmych">
<meta name="citation_author_institution" content="Department of Medicine, Vanderbilt University Medical Center, Nashville, TN USA">
<meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN USA">
<meta name="citation_author" content="Thomas P Hasaka">
<meta name="citation_author_institution" content="Vanderbilt Institute of Chemical Biology, High-Throughput Screening Facility, Vanderbilt University, Nashville, TN USA">
<meta name="citation_author" content="Chelsie K Sievers">
<meta name="citation_author_institution" content="Department of Medicine, Vanderbilt University Medical Center, Nashville, TN USA">
<meta name="citation_author" content="Seung Woo Kang">
<meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN USA">
<meta name="citation_author_institution" content="Epithelial Biology Center, Vanderbilt University Medical Center, Nashville, TN USA">
<meta name="citation_author_institution" content="Center for Computational Systems Biology, Vanderbilt University, Nashville, TN USA">
<meta name="citation_author" content="Marisol A Ramirez">
<meta name="citation_author_institution" content="Department of Biostatistics, Vanderbilt University Medical Center, Nashville, TN 37232 USA">
<meta name="citation_author_institution" content="Center for Quantitative Sciences, Vanderbilt University Medical Center, Nashville, TN USA">
<meta name="citation_author" content="Vivian Truong Jones">
<meta name="citation_author_institution" content="Department of Medicine, Vanderbilt University Medical Center, Nashville, TN USA">
<meta name="citation_author_institution" content="Department of Pharmacology, Vanderbilt University, Nashville, TN USA">
<meta name="citation_author" content="Zhiguo Zhao">
<meta name="citation_author_institution" content="Department of Biostatistics, Vanderbilt University Medical Center, Nashville, TN 37232 USA">
<meta name="citation_author" content="Oleg Kovtun">
<meta name="citation_author_institution" content="Department of Chemistry, Vanderbilt University, Nashville, TN USA">
<meta name="citation_author" content="Claudia C Wahoski">
<meta name="citation_author_institution" content="Department of Medicine, Vanderbilt University Medical Center, Nashville, TN USA">
<meta name="citation_author_institution" content="Program in Cancer Biology, Vanderbilt University, Nashville, TN USA">
<meta name="citation_author" content="Qi Liu">
<meta name="citation_author_institution" content="Department of Biostatistics, Vanderbilt University Medical Center, Nashville, TN 37232 USA">
<meta name="citation_author_institution" content="Center for Quantitative Sciences, Vanderbilt University Medical Center, Nashville, TN USA">
<meta name="citation_author" content="Ken S Lau">
<meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN USA">
<meta name="citation_author_institution" content="Epithelial Biology Center, Vanderbilt University Medical Center, Nashville, TN USA">
<meta name="citation_author_institution" content="Center for Computational Systems Biology, Vanderbilt University, Nashville, TN USA">
<meta name="citation_author_institution" content="Department of Surgery, Vanderbilt University Medical Center, Nashville, TN USA">
<meta name="citation_author_institution" content="Vanderbilt-Cancer Ingram Center, Vanderbilt University Medical Center, Nashville, TN USA">
<meta name="citation_author" content="Robert J Coffey">
<meta name="citation_author_institution" content="Department of Medicine, Vanderbilt University Medical Center, Nashville, TN USA">
<meta name="citation_author_institution" content="Epithelial Biology Center, Vanderbilt University Medical Center, Nashville, TN USA">
<meta name="citation_author" content="Joshua A Bauer">
<meta name="citation_author_institution" content="Vanderbilt Institute of Chemical Biology, High-Throughput Screening Facility, Vanderbilt University, Nashville, TN USA">
<meta name="citation_author_institution" content="Vanderbilt-Cancer Ingram Center, Vanderbilt University Medical Center, Nashville, TN USA">
<meta name="citation_author_institution" content="Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, TN USA">
<meta name="citation_author" content="Bhuminder Singh">
<meta name="citation_author_institution" content="Department of Medicine, Vanderbilt University Medical Center, Nashville, TN USA">
<meta name="citation_author_institution" content="Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN USA">
<meta name="citation_author_institution" content="Epithelial Biology Center, Vanderbilt University Medical Center, Nashville, TN USA">
<meta name="citation_author_institution" content="Program in Cancer Biology, Vanderbilt University, Nashville, TN USA">
<meta name="citation_author_institution" content="Vanderbilt-Cancer Ingram Center, Vanderbilt University Medical Center, Nashville, TN USA">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="8">
<meta name="citation_firstpage" content="1261">
<meta name="citation_doi" content="10.1038/s42003-025-08699-0">
<meta name="citation_pmid" content="40847193">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373842/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373842/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373842/pdf/42003_2025_Article_8699.pdf">
<meta name="description" content="Loss of polarity is a hallmark of cancer, and the related epithelial-to-mesenchymal transition (EMT) phenotype impacts prognosis and therapy outcomes, particularly in colorectal cancer (CRC). However, the mechanisms and drugs that impact EMT-related ...">
<meta name="og:title" content="3D collagen high-throughput screen identifies drugs that induce epithelial polarity and enhance chemotherapy response in colorectal cancer">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Loss of polarity is a hallmark of cancer, and the related epithelial-to-mesenchymal transition (EMT) phenotype impacts prognosis and therapy outcomes, particularly in colorectal cancer (CRC). However, the mechanisms and drugs that impact EMT-related ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373842/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Searchâ€¦
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12373842">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1038/s42003-025-08699-0"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/42003_2025_Article_8699.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373842%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12373842/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12373842/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373842/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-commbiol.jpg" alt="Communications Biology logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Communications Biology" title="Link to Communications Biology" shape="default" href="https://www.nature.com/commsbio/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Commun Biol</button></div>. 2025 Aug 22;8:1261. doi: <a href="https://doi.org/10.1038/s42003-025-08699-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1038/s42003-025-08699-0</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Commun%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Commun%20Biol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Commun%20Biol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Commun%20Biol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>3D collagen high-throughput screen identifies drugs that induce epithelial polarity and enhance chemotherapy response in colorectal cancer</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Harmych%20SJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Sarah J Harmych</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Sarah J Harmych</span></h3>
<div class="p">
<sup>1</sup>Department of Medicine, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div class="p">
<sup>2</sup>Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Harmych%20SJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Sarah J Harmych</span></a>
</div>
</div>
<sup>1,</sup><sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hasaka%20TP%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Thomas P Hasaka</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Thomas P Hasaka</span></h3>
<div class="p">
<sup>3</sup>Vanderbilt Institute of Chemical Biology, High-Throughput Screening Facility, Vanderbilt University, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hasaka%20TP%22%5BAuthor%5D" class="usa-link"><span class="name western">Thomas P Hasaka</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sievers%20CK%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Chelsie K Sievers</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Chelsie K Sievers</span></h3>
<div class="p">
<sup>1</sup>Department of Medicine, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Sievers%20CK%22%5BAuthor%5D" class="usa-link"><span class="name western">Chelsie K Sievers</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kang%20SW%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Seung Woo Kang</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Seung Woo Kang</span></h3>
<div class="p">
<sup>2</sup>Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN USA </div>
<div class="p">
<sup>4</sup>Epithelial Biology Center, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div class="p">
<sup>5</sup>Center for Computational Systems Biology, Vanderbilt University, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kang%20SW%22%5BAuthor%5D" class="usa-link"><span class="name western">Seung Woo Kang</span></a>
</div>
</div>
<sup>2,</sup><sup>4,</sup><sup>5</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ramirez%20MA%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Marisol A Ramirez</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Marisol A Ramirez</span></h3>
<div class="p">
<sup>6</sup>Department of Biostatistics, Vanderbilt University Medical Center, Nashville, TN 37232 USA </div>
<div class="p">
<sup>7</sup>Center for Quantitative Sciences, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ramirez%20MA%22%5BAuthor%5D" class="usa-link"><span class="name western">Marisol A Ramirez</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jones%20VT%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Vivian Truong Jones</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Vivian Truong Jones</span></h3>
<div class="p">
<sup>1</sup>Department of Medicine, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div class="p">
<sup>8</sup>Department of Pharmacology, Vanderbilt University, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Jones%20VT%22%5BAuthor%5D" class="usa-link"><span class="name western">Vivian Truong Jones</span></a>
</div>
</div>
<sup>1,</sup><sup>8</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20Z%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Zhiguo Zhao</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Zhiguo Zhao</span></h3>
<div class="p">
<sup>6</sup>Department of Biostatistics, Vanderbilt University Medical Center, Nashville, TN 37232 USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Zhao%20Z%22%5BAuthor%5D" class="usa-link"><span class="name western">Zhiguo Zhao</span></a>
</div>
</div>
<sup>6</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kovtun%20O%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Oleg Kovtun</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Oleg Kovtun</span></h3>
<div class="p">
<sup>9</sup>Department of Chemistry, Vanderbilt University, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kovtun%20O%22%5BAuthor%5D" class="usa-link"><span class="name western">Oleg Kovtun</span></a>
</div>
</div>
<sup>9</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wahoski%20CC%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Claudia C Wahoski</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Claudia C Wahoski</span></h3>
<div class="p">
<sup>1</sup>Department of Medicine, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div class="p">
<sup>10</sup>Program in Cancer Biology, Vanderbilt University, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wahoski%20CC%22%5BAuthor%5D" class="usa-link"><span class="name western">Claudia C Wahoski</span></a>
</div>
</div>
<sup>1,</sup><sup>10</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Q%22%5BAuthor%5D" class="usa-link" aria-describedby="id10"><span class="name western">Qi Liu</span></a><div hidden="hidden" id="id10">
<h3><span class="name western">Qi Liu</span></h3>
<div class="p">
<sup>6</sup>Department of Biostatistics, Vanderbilt University Medical Center, Nashville, TN 37232 USA </div>
<div class="p">
<sup>7</sup>Center for Quantitative Sciences, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Liu%20Q%22%5BAuthor%5D" class="usa-link"><span class="name western">Qi Liu</span></a>
</div>
</div>
<sup>6,</sup><sup>7</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lau%20KS%22%5BAuthor%5D" class="usa-link" aria-describedby="id11"><span class="name western">Ken S Lau</span></a><div hidden="hidden" id="id11">
<h3><span class="name western">Ken S Lau</span></h3>
<div class="p">
<sup>2</sup>Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN USA </div>
<div class="p">
<sup>4</sup>Epithelial Biology Center, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div class="p">
<sup>5</sup>Center for Computational Systems Biology, Vanderbilt University, Nashville, TN USA </div>
<div class="p">
<sup>11</sup>Department of Surgery, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div class="p">
<sup>12</sup>Vanderbilt-Cancer Ingram Center, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Lau%20KS%22%5BAuthor%5D" class="usa-link"><span class="name western">Ken S Lau</span></a>
</div>
</div>
<sup>2,</sup><sup>4,</sup><sup>5,</sup><sup>11,</sup><sup>12</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Coffey%20RJ%22%5BAuthor%5D" class="usa-link" aria-describedby="id12"><span class="name western">Robert J Coffey</span></a><div hidden="hidden" id="id12">
<h3><span class="name western">Robert J Coffey</span></h3>
<div class="p">
<sup>1</sup>Department of Medicine, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div class="p">
<sup>4</sup>Epithelial Biology Center, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Coffey%20RJ%22%5BAuthor%5D" class="usa-link"><span class="name western">Robert J Coffey</span></a>
</div>
</div>
<sup>1,</sup><sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bauer%20JA%22%5BAuthor%5D" class="usa-link" aria-describedby="id13"><span class="name western">Joshua A Bauer</span></a><div hidden="hidden" id="id13">
<h3><span class="name western">Joshua A Bauer</span></h3>
<div class="p">
<sup>3</sup>Vanderbilt Institute of Chemical Biology, High-Throughput Screening Facility, Vanderbilt University, Nashville, TN USA </div>
<div class="p">
<sup>12</sup>Vanderbilt-Cancer Ingram Center, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div class="p">
<sup>13</sup>Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Bauer%20JA%22%5BAuthor%5D" class="usa-link"><span class="name western">Joshua A Bauer</span></a>
</div>
</div>
<sup>3,</sup><sup>12,</sup><sup>13</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id14"><span class="name western">Bhuminder Singh</span></a><div hidden="hidden" id="id14">
<h3><span class="name western">Bhuminder Singh</span></h3>
<div class="p">
<sup>1</sup>Department of Medicine, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div class="p">
<sup>2</sup>Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN USA </div>
<div class="p">
<sup>4</sup>Epithelial Biology Center, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div class="p">
<sup>10</sup>Program in Cancer Biology, Vanderbilt University, Nashville, TN USA </div>
<div class="p">
<sup>12</sup>Vanderbilt-Cancer Ingram Center, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Singh%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Bhuminder Singh</span></a>
</div>
</div>
<sup>1,</sup><sup>2,</sup><sup>4,</sup><sup>10,</sup><sup>12,</sup><sup>âœ‰</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="Aff1">
<sup>1</sup>Department of Medicine, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div id="Aff2">
<sup>2</sup>Department of Cell and Developmental Biology, Vanderbilt University, Nashville, TN USA </div>
<div id="Aff3">
<sup>3</sup>Vanderbilt Institute of Chemical Biology, High-Throughput Screening Facility, Vanderbilt University, Nashville, TN USA </div>
<div id="Aff4">
<sup>4</sup>Epithelial Biology Center, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div id="Aff5">
<sup>5</sup>Center for Computational Systems Biology, Vanderbilt University, Nashville, TN USA </div>
<div id="Aff6">
<sup>6</sup>Department of Biostatistics, Vanderbilt University Medical Center, Nashville, TN 37232 USA </div>
<div id="Aff7">
<sup>7</sup>Center for Quantitative Sciences, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div id="Aff8">
<sup>8</sup>Department of Pharmacology, Vanderbilt University, Nashville, TN USA </div>
<div id="Aff9">
<sup>9</sup>Department of Chemistry, Vanderbilt University, Nashville, TN USA </div>
<div id="Aff10">
<sup>10</sup>Program in Cancer Biology, Vanderbilt University, Nashville, TN USA </div>
<div id="Aff11">
<sup>11</sup>Department of Surgery, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div id="Aff12">
<sup>12</sup>Vanderbilt-Cancer Ingram Center, Vanderbilt University Medical Center, Nashville, TN USA </div>
<div id="Aff13">
<sup>13</sup>Department of Biochemistry, Vanderbilt University School of Medicine, Nashville, TN USA </div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>âœ‰</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Feb 7; Accepted 2025 Aug 8; Collection date 2025.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>Â© The Author(s) 2025</div>
<p><strong>Open Access</strong> This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons licence, and indicate if changes were made. The images or other third party material in this article are included in the articleâ€™s Creative Commons licence, unless indicated otherwise in a credit line to the material. If material is not included in the articleâ€™s Creative Commons licence and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this licence, visit <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a>.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12373842Â Â PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40847193/" class="usa-link">40847193</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="Abs1"><h2>Abstract</h2>
<p id="Par1">Loss of polarity is a hallmark of cancer, and the related epithelial-to-mesenchymal transition (EMT) phenotype impacts prognosis and therapy outcomes, particularly in colorectal cancer (CRC). However, the mechanisms and drugs that impact EMT-related morphological changes are understudied, due to the complete failure of typical live/dead 2D high-throughput screens to capture morphology or the lack of robustness of 3D screens. We designed a high-throughput screen using 3D type I collagen cultures of CRC cells to assess morphological changes in colonies and identified several FDA-approved drugs that re-epithelialize CRC colonies. One of these drugs, azithromycin, increased colony circularity, enhanced E-cadherin membrane localization and ZO-1 localization to tight junctions, caused transcriptomic changes consistent with downregulation of EMT, and elevated sensitivity to the chemotherapeutic, irinotecan. A retrospective analysis of patient data demonstrated that the use of azithromycin in patients undergoing treatment for CRC with irinotecan had improved the 5â€‰year survival compared to the chemotherapy alone. These results highlight the importance of morphological screens to identify novel drug candidates and synergistic mechanisms.</p>
<section id="kwd-group1" class="kwd-group"><p><strong>Subject terms:</strong> Colorectal cancer, Cancer therapeutic resistance, High-throughput screening</p></section></section><section class="abstract" id="Abs2"><hr class="headless">
<p id="Par2">A 3D high-throughput screen in type I collagen is established to score drug efficacy by colony morphology, identifying three antibiotics that re-epithelialize colorectal cancer colonies and enhance chemotherapy response.</p></section><section id="Sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p id="Par3">Epithelial cells line a majority of body cavities where they organize into polarized monolayers. Most cancers arise from the epithelium, and loss of epithelial polarity is essential in the development of cancer, facilitating hyperproliferation, multi-layering, and metastasis<sup><a href="#CR1" class="usa-link" aria-describedby="CR1">1</a></sup>. Cell or tissue polarity is considered a non-canonical tumor suppressor, serving to regulate proliferation, survival, and apoptosis of epithelial cells<sup><a href="#CR2" class="usa-link" aria-describedby="CR2">2</a></sup>. The epithelial-to-mesenchymal transition (EMT) requires the loss of apicobasal polarity and the establishment of front-to-rear polarity, causing a significant change in cellular shape and morphology of colonies<sup><a href="#CR3" class="usa-link" aria-describedby="CR3">3</a></sup>. Given that EMT has also been implicated as a mechanism of therapy resistance<sup><a href="#CR4" class="usa-link" aria-describedby="CR4">4</a></sup>, assessing morphological changes associated with EMT could prove an effective method in identifying drugs that enhance current treatments.</p>
<p id="Par4">Features of epithelial polarity are better recapitulated in vitro by growing cells in 3D extracellular support matrices, like Matrigel, hydrogel, and collagen, than in 2D plastic cultures<sup><a href="#CR5" class="usa-link" aria-describedby="CR5">5</a>â€“<a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. For example, when grown in type I collagen, the human colorectal cancer (CRC) cell line, HCA-7 forms primarily unilamellar colonies with intact apicobasal polarity (cystic colonies) and less frequent colonies made of irregular masses of cells (spiky colonies)<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a></sup>. Cell lines generated from cystic and spiky colonies, termed CC and SC cells respectively, look indistinguishable when grown on plastic but have different morphological and functional properties in 3D<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. A notable difference between the cell lines is their sensitivity to the Epidermal Growth Factor Receptor (EGFR) neutralizing antibody cetuximab<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. CC cell colony formation in type I collagen was significantly decreased with cetuximab treatment. In contrast, cetuximab treatment had little impact on SC colony formation<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Resistance of SC cells to cetuximab was determined to be due to increased activation of the receptor tyrosine kinases MET and RON, which work in parallel to the EGFR pathway blocked by cetuximab<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Resistance to cetuximab was induced in CC cells through the overexpression of the MET ligand, HGF. CC cells overexpressing HGF (CC-HGF) formed spiky colonies like those of SC cells rather than cystic colonies typical of CC cells. Like SC cells, treatment of CC-HGF cells with a MET/RON inhibitor, restored sensitivity to cetuximab and their cystic morphology<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>â€“<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>.</p>
<p id="Par5">As demonstrated by the study of CC and SC cells and other past studies, the morphology of 3D cancer cellÂ colonies can reveal critical insights into functional characteristics, including response to drug treatment<sup><a href="#CR8" class="usa-link" aria-describedby="CR8">8</a>,<a href="#CR9" class="usa-link" aria-describedby="CR9">9</a>,<a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Numerous studies over the last five decades have shown the translational relevance of 3D cultures and the limitations of traditional 2D culturing methods<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a></sup>. Despite this, 2D culturing has remained the standard in the high-throughput screening field. Technical challenges, including the handling and plating of hydrogel-based artificial extracellular matrices and designing image analysis protocols to assess morphological changes in 3D cultures, have been hurdles in the widespread use of these cultures in drug screening<sup><a href="#CR7" class="usa-link" aria-describedby="CR7">7</a>,<a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>.</p>
<p id="Par6">In this study, we established a collagen-based high-throughput drug screen and identified several FDA-approved drugs which induce morphological changes in CRC colonies. The method developed for this study allowed for the identification of multiple distinct morphological phenotypes utilizing a variety of morphological characteristics. Furthermore, drugs identified in the screen for inducing an epithelial morphology were also able to enhance the response of colonies to chemotherapy treatment. The results of this study demonstrate the effectiveness of 3D morphological high-throughput screens in the identification of drugs which can be used to enhance current therapies for CRC.</p></section><section id="Sec2"><h2 class="pmc_sec_title">Results</h2>
<section id="Sec3"><h3 class="pmc_sec_title">3D high-throughput drug screen identifies distinct morphological classes of SC colonies</h3>
<p id="Par7">Our standard 12/24-well format 3D type I collagen culture (see methods) was adapted for use in a 384-well format with automated liquid handling equipment (Fig.Â <a href="#Fig1" class="usa-link">1A</a>). After growing cells in 3D with a library of 1059 FDA-approved compounds for 8â€‰days, colonies were stained with Calcein AM and confocal images were acquired and quantified for several parameters. The FDA-approved drug library contains drugs with a wide variety of functional targets, which are outlined in Fig.Â <a href="#Fig1" class="usa-link">1B</a>. Measurements of morphological characteristics for all drugs screened are available as supplemental data (Supplementary DataÂ <a href="#MOESM4" class="usa-link">1</a>) and raw images from the screen are available in a repository<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. All these characteristics were measured and quantified automatically by the InCarta software, which utilizes artificial intelligence to analyze images. Pair-wise Pearson correlation coefficients were calculated between each variable (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">1A</a>). Principal component analysis (PCA) was conducted subsequently, where 10 principal components (PCs) could describe over 95% of data variance (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">1B</a>). Using this PCA, we first sought to determine if there were any discernable changes in the morphology of SC colonies when treated with the drugs in the screen. A scatterplot mapping of PC1 and PC2 revealed five distinct clusters of wells separate from the main cluster of wells along both the PC1 and PC2 axes (Fig.Â <a href="#Fig1" class="usa-link">1C</a>). The PC loadings heatmap indicated energy and entropy were the main contributors of PC1, whereas kurtosis and skewness contributed the most to PC2 (Fig.Â <a href="#Fig1" class="usa-link">1D</a>). Clusters were identified by visual assessment of wells that had separated from the central cluster of wells. The central cluster contained untreated control wells and drugs in the screen which had little to no effect on colony morphology compared to control wells. Drugs which largely had a cell death effect, as determined by decreased colony count, were unsuitable for colony morphology assessment and were removed prior to conducting the PCA.</p>
<figure class="fig xbox font-sm" id="Fig1"><h4 class="obj_head">Fig. 1. High-throughput drug screen using 3D type I collagen cultures identifies morphological clusters.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373842_42003_2025_8699_Fig1_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97c6/12373842/f70b2ce62930/42003_2025_8699_Fig1_HTML.jpg" loading="lazy" id="d33e576" height="506" width="798" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Schematic of plating, treatment, and analysis of drug screen. Chilled 384-well plates were stamped with a 5â€‰ÂµL bottom layer of type I collagen and allowed to solidify. 1000 SC cells in 10â€‰ÂµL collagen were then added and allowed to solidify. Next, 1059 compounds from an FDA-approved drug library were added at three concentrations and incubated for 8â€‰days. Wells were then stained with Calcein AM, imaged, and analyzed to assess colony morphology. Created in BioRender. (Harmych, S. (2025) <a href="https://BioRender.com/4zfblch" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://BioRender.com/4zfblch</a>). <strong>B</strong> Pie graph depicting functional targets of drugs tested during screen. <strong>C</strong> Principal component analysis (PCA) plot of morphological characteristics of colonies from high-throughput drug screen. Morphological characteristics were obtained using the InCarta software. Each dot represents a compound-treated well from the screen. Distinct Clusters (<strong>A</strong>âˆ’<strong>E</strong>) separating from the central cluster were visually assessed and shown on the PCA plot. <strong>D</strong> Heatmap of PC loadings of each variable on PC1 and PC2. Brightfield and Calcein AM indicate which image was used by InCarta software to generate the value. <strong>E</strong> Representative whole well images of morphological clusters identified in the screen. Boxes correspond to images shared in Fig. 1F. Brightfield and fluorescent (Calcein AM) images taken with ImageXpress confocal HT.ai automated high-content imaging system at 4x magnification. (Scale bars: 500â€‰Âµm). <strong>F</strong> Insets of representative wells shown in Fig. 1E. (Scale bars: 100â€‰Âµm).</p></figcaption></figure><p id="Par8">The five clusters identified during PCA analysis each exhibited a distinct morphology (Fig.Â <a href="#Fig1" class="usa-link">1</a>E, F). Drugs used to treat wells from Clusters A and B appeared to induce some cell death, as these Clusters showed fewer and smaller colonies than other wells. Colonies in Cluster A became small and dense while the colonies in Cluster B became sparse. Both Cluster C and Cluster E generated large colonies without a centralized lumen. Colonies in Cluster C had a dense center with spiky protrusions while those in Cluster E were disorganized and spread out, lacking the density seen in Cluster C. Colonies in Cluster D were also large and some had formed a central lumen suggesting re-epithelialization may have taken place. Overall, these results indicate that several distinct colony morphologies induced by drug incubation can be monitored and quantified in this unsupervised high-throughput screening format.</p></section><section id="Sec4"><h3 class="pmc_sec_title">Median colony area and percent colonies with lumens used to identify drugs re-circularizing SC colonies</h3>
<p id="Par9">We next investigated which morphological characteristics could be used to identify our morphology of interest, a spiky to cystic conversion of colonies. The integrin Î²1 antibody (P4G11), which has previously been shown to re-epithelialize SC colonies in collagen<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup> was used as a positive or â€œcysticâ€ control for this screen. Including P4G11-treated wells, pair-wise Pearson correlation analysis (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">2A</a>) and PCA were conducted again, where 10 PCs could describe over 95% of data variance (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">2B</a>). A scatterplot mapping PC1 and PC2 revealed the positive control wells clearly separated from the other wells along the PC1 axis (Fig.Â <a href="#Fig2" class="usa-link">2A</a>). Interestingly, PC loadings heatmap revealed several variables â€“ such as minor axes, entropy, and related texture features â€“ as major contributors, underscoring the unbiased, unsupervised strength of our automated quantification approach (Fig.Â <a href="#Fig2" class="usa-link">2B</a>). Notably, some of these parameters also earlier showed relevance across the full screen (Fig.Â <a href="#Fig1" class="usa-link">1C</a>). While these parameters may prove valuable following further validation, particularly for assessing unknown drug effects, our primary focus in tracking reversal to epithelial phenotype was to ensure visual confirmation of morphological conversion, minimizing false positives.</p>
<figure class="fig xbox font-sm" id="Fig2"><h4 class="obj_head">Fig. 2. Median colony area and percent colonies with lumens identify re-epithelialized colonies.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373842_42003_2025_8699_Fig2_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97c6/12373842/21dc87a222a1/42003_2025_8699_Fig2_HTML.jpg" loading="lazy" id="d33e644" height="294" width="798" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Principal component analysis (PCA) plot of colony morphological characteristics from high-throughput drug screen. Each dot represents a well from the screen. Highlighted dots indicate wells treated with the positive control, P4G11. <strong>B</strong> Heatmap of PC loadings of each variable on PC1 and PC2. Brightfield and Calcein AM indicate which image was used by InCarta software to generate the value. <strong>C</strong> Comparison of DMSO and P4G11 control wells using median colony area and percent colonies with lumens determined using MetaXpress software. Each dot represents a well from the screen. <strong>D</strong>-<strong>E</strong> Comparison of (<strong>D</strong>) median colony area and (<strong>E</strong>) percent colonies with lumens for control wells. Box and whisker plots denote median (center line), interquartile range (box), and minimum and maximum (whiskers). Wilcoxon rank-sum test, <em>p</em>-values indicated. <strong>F</strong> Representative brightfield and fluorescent (Calcein AM) confocal images of DMSO and P4G11 control wells from screen. Left panel depicts whole well images (Scale bars: 500â€‰Âµm). Right panel depicts higher magnification of indicated section of the well (Scale bars: 100â€‰Âµm). Images taken with ImageXpress confocal HT.ai automated high-content imaging system at 4x magnification. Colony and lumen masks are overlayed on brightfield images and were generated using MetaXpress image analysis software.</p></figcaption></figure><p id="Par10">To that end, we prioritized features that were both visually intuitive and functionally meaningful. Metrics such as average and median colony area, perimeter, and lumen countâ€”derived from InCarta softwareâ€”aligned well with the expected formation of larger, smoother colonies with hollow lumens (Fig.Â <a href="#Fig2" class="usa-link">2B</a>). Among these, we then focused on â€œmedian colony areaâ€ and â€œpercent colonies with lumensâ€ that were quantified through MetaXpress image analysis software. Plotting of the untreated spiky control wells (DMSO) and the P4G11-treated cystic control wells based on both median colony area and percent colonies with lumens showed that the two controls formed distinct clusters (Fig.Â <a href="#Fig2" class="usa-link">2C</a>). Cystic control wells had significantly higher median colony area (Fig.Â <a href="#Fig2" class="usa-link">2D</a>) and percent colonies with lumens (Fig.Â <a href="#Fig2" class="usa-link">2E</a>) than the spiky control wells. Some lumens in the spiky controls likely reflect either true lumen-containing colonies, seen in about 22% of SC colonies<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>, or artifacts due to a small hole between colonies that was not filtered out during image analysis. While the InCarta software was used for unsupervised clustering (Figs.Â <a href="#Fig1" class="usa-link">1</a>C and <a href="#Fig2" class="usa-link">2B</a>), subsequent quantification (Fig.Â <a href="#Fig2" class="usa-link">2Câ€“E</a>) was performed using MetaXpress analysis, which more accurately identified colony areas. Representative masks of colonies and lumens (Fig.Â <a href="#Fig2" class="usa-link">2F</a>) were generated using Calcein AM staining and MetaXpress software for these analyses. Thus, the parameters colony size and percent colonies with lumens, identified from our positive control for cystic morphology were then used to identify hits from our drug screen that re-circularize SC cell colonies.</p></section><section id="Sec5"><h3 class="pmc_sec_title">Antibiotics azithromycin, clindamycin, and linezolid induced re-circularization of SC colonies in screen</h3>
<p id="Par11">Hits of interest from the screen were identified based on median colony area (Fig.Â <a href="#Fig3" class="usa-link">3A</a>) and percent colonies with lumens (Fig.Â <a href="#Fig3" class="usa-link">3B</a>). Both morphological characteristics were assessed using a control-based measurement and a control-independent measurement<sup><a href="#CR16" class="usa-link" aria-describedby="CR16">16</a></sup>. The control-based measurement, fold-change, was based on the DMSO control wells of the same plate. The control-independent measurement, B-score, accounts for both plate-to-plate variability and positional variability within a plate. Fold-change and B-score were used to plot each well and then the top 5% for each was calculated (Fig.Â <a href="#Fig3" class="usa-link">3A, B</a>). This generated an upper quadrant on the plot containing wells within the top 5% for both fold-change and B-score. Wells that fell within this upper quadrant for both median colony area and percent colonies with lumens were considered hits (Fig.Â <a href="#Fig3" class="usa-link">3C</a>). Ten wells in the screen met the criteria to be considered a hit. Based on visual assessment, two of these wells were determined to be false positives. The remaining eight wells corresponded to six different drugs, four of which were antibiotics.</p>
<figure class="fig xbox font-sm" id="Fig3"><h4 class="obj_head">Fig. 3. Antibiotics identified as top three hits in screen that re-epithelialize 3D CRC colonies.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373842_42003_2025_8699_Fig3_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97c6/12373842/2ee5f2344e15/42003_2025_8699_Fig3_HTML.jpg" loading="lazy" id="d33e776" height="818" width="797" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong>-<strong>B</strong> Plots of median colony area (<strong>A</strong>) and percent colonies with lumens (<strong>B</strong>) for hit identification. Fold change plotted on the <em>Y</em>-axis and corresponding B-score on the <em>X</em>-axis. Dotted lines indicate the top 5% of values along its axis. <strong>C</strong> Venn diagram depicting selection of hits wells based on plots in (<strong>A</strong>, <strong>B</strong>). <strong>D</strong>, <strong>E</strong> Comparison of (<strong>D</strong>) median colony area and (<strong>E</strong>) percent colonies with lumens for top three hits to the plate they were located on during the screen. Well number was assigned based on order of the wells. Solid line indicates the median for all wells of the plate and the dotted lines denote the interquartile range. <strong>F</strong> Insets of brightfield and fluorescent (Calcein AM) confocal images of wells treated with top three hits at three concentrations from the screen. (Scale bars: 250â€‰Âµm). Images taken with ImageXpress confocal HT.ai automated high-content imaging system at 4x magnification. Full well images shown in Fig.Â <a href="#MOESM2" class="usa-link">S4</a>. <strong>G</strong>âˆ’<strong>I</strong> Quantification of CRC (SC cells) colony circularity when treated with (<strong>G</strong>) azithromycin, (<strong>H</strong>) clindamycin, or (<strong>I</strong>) linezolid. SC cells were seeded in type I collagen and incubated with either 10â€‰ÂµM, 1â€‰ÂµM, or 0.1â€‰ÂµM of the indicated antibiotic for 14â€‰days. Results are quantified as circularity index. Higher circularity indicates rounder colonies and lower circularity index indicates spiky colonies. Left panel depicts density plot of the circularity for all colonies from each treatment for all three biological replicates. Vertical line indicates the median circularity for each treatment condition. Right panel depicts median circularity for each biological replicate. <em>n</em>â€‰&gt;â€‰150 for each biological replicate from 3 technical replicates (wells). Plotted as meanâ€‰Â±â€‰SEM. One-way ANOVA with Tukeyâ€™s post hoc test, <em>p</em>-values indicated. <strong>J</strong> Brightfield and fluorescent confocal images of SC cell colonies treated with indicated antibiotics. SC cells were seeded in type I collagen and incubated with 10â€‰ÂµM of the indicated antibiotic for 14â€‰days. Collagen layer containing colonies was isolated, fixed, and stained for DAPI (blue), Phalloidin (red), and E-cadherin (green). Colonies were imaged through the equatorial plane using a 10x objective. (Scale bars: 100â€‰Âµm).</p></figcaption></figure><p id="Par12">The four antibiotics identified as hits during the screen were azithromycin, clindamycin, linezolid, and methacycline hydrochloride. While these hits belong to different antibiotic classes, all four bind to the bacterial ribosome, inhibiting protein synthesis and ultimately bacterial replication. The other two drugs identified as hits were ruxolitinib, a JAK inhibitor, and diminazene aceturate, an antiparasitic. When the three concentrations of each drug were compared to their corresponding plate from the screen, at least one well fell above the upper interquartile range for both median colony area and percent colonies with lumens (Fig.Â <a href="#Fig3" class="usa-link">3D, E</a>, Supplementary Fig.Â <a href="#MOESM2" class="usa-link">4A</a>, B). Circular colonies with distinct lumens can be observed in at least one of the concentrations for each drug (Fig.Â <a href="#Fig3" class="usa-link">3</a>E, F, Supplementary Figs.Â <a href="#MOESM2" class="usa-link">3B</a>, <a href="#MOESM2" class="usa-link">4Bâ€“D</a>). Wells from each of these hits also clustered together when highlighted in the unbiased PCA (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">3A</a>).</p>
<p id="Par13">Azithromycin, clindamycin, and linezolid were chosen for further studies due to the presence of lumen-containing colonies in all three concentrations in the screen (Fig.Â <a href="#Fig3" class="usa-link">3F</a> and Supplementary Fig.Â <a href="#MOESM2" class="usa-link">3B</a>). The three concentrations tested for each of these antibiotics indicated a potential dose-dependent response. To formally test this, a larger concentration range with 9 concentrations for the three antibiotics was then conducted in the 384-well format. All three drugs showed a dose-dependent increase in median colony area (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">3C</a>) and percent colonies with lumens (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">3D</a>).</p>
<p id="Par14">We next sought to determine if the hits identified in the screen also induce re-circularization in our standard 24-well collagen cultures. SC cells were plated in type I collagen in a 24-well plate format and treated with 10, 1, or 0.1â€‰ÂµM of azithromycin, clindamycin, or linezolid for 14â€‰days. The cystic conversion by the antibiotics was quantified as a circularity index, which was described previously as a relationship between colony area and perimeter, where rounder colonies had a higher circularity index<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup>. Treatment with each of the antibiotics, azithromycin (Fig.Â <a href="#Fig3" class="usa-link">3G</a> and Supplementary Fig.Â <a href="#MOESM2" class="usa-link">5</a>), clindamycin (Fig.Â <a href="#Fig3" class="usa-link">3H</a> and Supplementary Fig.Â <a href="#MOESM2" class="usa-link">5</a>), and linezolid (Fig.Â <a href="#Fig3" class="usa-link">3I</a> and Supplementary Fig.Â <a href="#MOESM2" class="usa-link">5</a>), resulted in a dose-dependent increase in colony circularity. Treatment with a concentration of 10â€‰ÂµM of each antibiotic led to a significant increase in average colony circularity compared to the untreated control. Treatment with azithromycin at 10â€‰ÂµM also caused a significant increase in average colony circularity compared to treatment with 0.1â€‰ÂµM azithromycin (Fig.Â <a href="#Fig3" class="usa-link">3G</a>). We fixed and stained drug-treated colonies with the nuclear marker (DAPI), membrane/F-actin marker (Phalloidin), and an epithelial marker (E-cadherin). Compared to control colonies, colonies treated with all three antibiotics showed increased staining with E-cadherin, indicating potential emergence of an epithelial phenotype (Fig.Â <a href="#Fig3" class="usa-link">3J</a>). Fixed colonies treated with azithromycin were also stained for the tight junction protein ZO-1 (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">6A</a>). While ZO-1 was present in both untreated and treated colonies, only the antibiotic-treated colonies exhibited staining pattern consistent with its enrichment in tight junctions (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">6B</a>, <em>left panel</em>). These results indicate that these antibiotics reorganize the CRC 3D colonies to impart features of epithelial polarity.</p>
<p id="Par15">For broader applicability, we tested the effects of azithromycin treatment on the morphology of an additional CRC cell line, SW480. Compared to SC cells, SW480 cells have higher baseline colony circularity, but azithromycin treatment still led to a significant increase in colony circularity (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">7A, B</a>). A greater number of azithromycin-treated SW480 colonies had lumens and a defined edge, both indicative of restored epithelial polarity (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">7C, D</a>). Azithromycin-treated SW480 colonies also exhibited stronger E-cadherin membrane localization at the colony periphery compared to controls (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">7E, F</a>). Finally, unlike most data in Figs.Â <a href="#Fig1" class="usa-link">1</a>â€“<a href="#Fig3" class="usa-link">3</a> generated using the 384-well high-throughput format, Fig.Â <a href="#Fig3" class="usa-link">3Gâ€“I</a>, and Supplementary Figs.Â <a href="#MOESM2" class="usa-link">5</a>â€“<a href="#MOESM2" class="usa-link">7</a> were produced using our standard 24-well format with two CRC lines. The consistency of our resultsÂ across multipleÂ formats supports the broader validity of our high-throughput screening format.</p></section><section id="Sec6"><h3 class="pmc_sec_title">Transcriptional reprogramming of CRC colonies with azithromycin, clindamycin, and linezolid treatment</h3>
<p id="Par16">To assess the impact of our hits on CRC colonies, we compared gene expression between untreated control colonies and colonies treated for 12â€‰days withÂ the top three antibiotics. RNA-seq PCA showed clear separation along PC1Â and PC2, with azithromycin and clindamycin clustering together and linezolid positioned between them and controls (Fig.Â <a href="#Fig4" class="usa-link">4A</a>). A volcano plot analysis was then conducted to highlight genes that were differentially expressed upon antibiotic treatments (Fig.Â <a href="#Fig4" class="usa-link">4B</a>, and Supplementary Fig.Â <a href="#MOESM2" class="usa-link">8A</a>, <a href="#MOESM2" class="usa-link">B</a>). CLDN2 and KIF26B were among the most upregulated, and COL9A2 and NFKB2 were among the most downregulated in azithromycin-treated colonies (Fig.Â <a href="#Fig4" class="usa-link">4B</a>). Clindamycin showed a similar CLDN2/KIF26B up and COL9A2/MMP7 down pattern (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">8A</a>), while linezolid uniquely upregulated mitochondrial genes MT-ATP8, MT-CYB, and MT-ND4L (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">8B</a>). A heatmap of the top 50 differentially expressed transcripts for azithromycin-, clindamycin-, and linezolid-treated colonies are shared in Fig.Â <a href="#Fig3" class="usa-link">3D</a> and Supplementary Fig.Â <a href="#MOESM2" class="usa-link">9C</a>, <a href="#MOESM2" class="usa-link">D</a> respectively.</p>
<figure class="fig xbox font-sm" id="Fig4"><h4 class="obj_head">Fig. 4. Transcriptomic analysis of 3D CRC colonies treated with azithromycin.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373842_42003_2025_8699_Fig4_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97c6/12373842/0fd7664a738f/42003_2025_8699_Fig4_HTML.jpg" loading="lazy" id="d33e950" height="643" width="799" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> Principal component analysis (PCA) plot of RNA-seq data from SC cell 3D cultures treated with azithromycin, clindamycin, and linezolid for 12â€‰days. Each dot represents a biological replicate. <strong>B</strong> Volcano plot for all comparisons all correlation PCA for control and azithromycin treated 3D cultures. Differential expression analysis criteria: absolute fold changeâ€‰â‰¥â€‰2 and FDR adjusted <em>p-</em>valueâ€‰â‰¤â€‰0.05. <strong>C</strong> Heatmap of top 50 differentially expressed transcripts in control vs azithromycin-treated 3D cultures in triplicate; purpleâ€‰=â€‰up, blueâ€‰=â€‰down (expression scale in inset). <strong>D</strong> WebGestalt-based control vs azithromycin-treated pathway over-representation analysis. Top 10 biological processes significantly overrepresented (FDRâ€‰â‰¤â€‰0.5) are depicted with their respective enrichment ratios. <strong>E</strong> Hallmark gene set enrichment analysis comparison of RNA-seq data from control and azithromycin-treated 3D cultures. Hallmark pathways are indicated on the left and are represented as bubbles on the Normalized enrichment score on <em>x</em>-axis. Bubble size indicates core enrichment or number of leading-edge genes and bubble color represents FDR (false discovery rate); scales of both are indicated on the right. <strong>F</strong> Gene set enrichment analysis (GSEA) of RNA-seq data from control and azithromycin-treated cultures. Three select categories of interest are shown. Abbreviations: NESÂ Normalized enrichment score, FDR False discovery rate <em>q</em>-values.</p></figcaption></figure><p id="Par17">To explore pathways affected by antibiotic treatment, we further performed Gene Set Enrichment Analysis (GSEA) and WebGestalt analysis. WebGestalt analysis showed upregulation of cell-matrix adhesion, epithelial morphogenesis, and immune-related pathways in azithromycin-treated colonies (Fig.Â <a href="#Fig4" class="usa-link">4D</a>). Clindamycin and linezolid both upregulated extracellular matrix organization (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">9E</a>, <a href="#MOESM2" class="usa-link">F</a>). GSEA analysis showed all three antibiotics downregulated hallmark EMT and KRAS signaling pathways (Fig.Â <a href="#Fig4" class="usa-link">4E, F</a>, Supplementary Fig.Â <a href="#MOESM2" class="usa-link">8Gâ€“J</a>). Besides its antibiotic actions, azithromycin also displays anti-inflammatory properties, which was corroborated by our RNA-seq analysis showing downregulated hallmark inflammatory response in azithromycin-treated colonies. Azithromycin treatment also downregulated TNFÎ± signaling, STAT signaling, and angiogenesis (Fig.Â <a href="#Fig4" class="usa-link">4E</a>). Similarly, clindamycin downregulated inflammatory pathways and angiogenesis (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">8G</a> and <a href="#MOESM2" class="usa-link">I</a>), while linezolid upregulated apical junction proteins (Supplementary Figs.Â <a href="#MOESM2" class="usa-link">8H</a> and <a href="#MOESM2" class="usa-link">S8J</a>).</p>
<p id="Par18">Pathway analyses suggested that antibiotic treatment leads to a shift toward a more epithelial phenotype. To further characterize this, we examined expression of specific protein classes by RNA-seq. Claudins are a family of transmembrane proteins that are a critical component of tight junctions, and all three antibiotics significantly upregulated expression ofÂ claudinsâ€”CLDN2, CLDN4, and CLDN7, while azithromycin and clindamycin also upregulated CLDN3 (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">9A</a>)<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a></sup>. This aligns with an epithelial shift, as these are key tight junction proteins and epithelial markers<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>â€“<a href="#CR21" class="usa-link" aria-describedby="CR21">21</a></sup>. Notably, the observed CLDN7 upregulation is significant as its induced expression can restore epithelial features in CRC cells and its loss is an early event in CRC carcinogenesis<sup><a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR23" class="usa-link" aria-describedby="CR23">23</a></sup>. Conversely, CLDN1 and CLDN12, which are associated with cancer invasion and progression, were significantly reduced by all three antibiotics (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">9A</a>)<sup><a href="#CR17" class="usa-link" aria-describedby="CR17">17</a>,<a href="#CR22" class="usa-link" aria-describedby="CR22">22</a>,<a href="#CR24" class="usa-link" aria-describedby="CR24">24</a></sup>. Additionally, all three antibiotics significantly upregulated intermediate filaments (keratins)â€”KRT8 and KRT19, while azithromycin and clindamycin also increased KRT18 expression (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">9C</a>). These keratins (KRT8, KRT18, and KRT19) are established epithelial markers<sup><a href="#CR18" class="usa-link" aria-describedby="CR18">18</a>â€“<a href="#CR20" class="usa-link" aria-describedby="CR20">20</a></sup>, and their upregulation is consistent with re-epithelialization, as their loss is associated with EMT and CRC progression<sup><a href="#CR25" class="usa-link" aria-describedby="CR25">25</a>â€“<a href="#CR28" class="usa-link" aria-describedby="CR28">28</a></sup>. Surprisingly, only linezolid increased RNA expression of all 13 mitochondrial-encoded electron transport chain (ETC) genes, whereas azithromycin and clindamycin induced fewer ETC genes and to a lesser extent (Supplementary Fig.Â <a href="#MOESM2" class="usa-link">9B</a>). Taken together, these findings indicate that antibiotic treatment promotes re-epithelialization in CRC colonies, as shown by both pathway analysis and specific gene expression changes.</p></section><section id="Sec7"><h3 class="pmc_sec_title">Azithromycin increases the efficacy of treatment with Irinotecan</h3>
<p id="Par19">Given the role of EMT in the development of treatment resistance by CRC, we sought to determine the effect of azithromycin on treatment response of SC cells grown in collagen. For this, we elected to focus on the interaction of azithromycin with irinotecan, which is a key component of several chemotherapeutic regimens for the treatment of advanced CRC<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. By itself, azithromycin had a mild effect on the number of colonies formed by SC cells in type I collagen, causing a 5% decrease in colony numbers compared to untreated cells. When cells were treated with the active metabolite of the chemotherapy irinotecan, SN-38, there was a significant drop in colony number, with an 80% decrease in number of colonies formed compared to the control (Fig.Â <a href="#Fig5" class="usa-link">5A, C</a>). There was an even greater decrease in colony formation when cells were treated with both SN-38 and azithromycin compared to either drug alone. The combination therapy demonstrated almost a 95% reduction in colony formation relative to control, which was 15% greater than the observed effect when cells were only treated with SN-38 (Fig.Â <a href="#Fig5" class="usa-link">5A, C</a>).</p>
<figure class="fig xbox font-sm" id="Fig5"><h4 class="obj_head">Fig. 5. Azithromycin enhances the efficacy of chemotherapy in CRC.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12373842_42003_2025_8699_Fig5_HTML.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/97c6/12373842/7a520c32a613/42003_2025_8699_Fig5_HTML.jpg" loading="lazy" id="d33e1087" height="487" width="799" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/Fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p><strong>A</strong> 1250 SC cells were seeded in 3D in type I collagen and incubated for 12â€‰days with azithromycin (5â€‰ÂµM), SN-38 (5â€‰nM), or both. Points represent three technical replicates (wells) for each biological replicate. Normalized colony counts are plotted as meanâ€‰Â±â€‰SEM. Student unpaired <em>t</em>-test conducted for selected groups, <em>p</em>-values indicated. Berferroniâ€™s correction was used to account for family-wise error rate, with an adjusted <em>p</em>-value threshold of 0.025. <strong>B</strong> Kaplan-Meier plot of overall survival for patients treated with irinotecan-based chemotherapy with or without concurrent azithromycin use from a retrospective cohort of de-identified electronic health records from a single academic medical center. Manual annotation of patientâ€™s date of death was used to determine survival at 5â€‰years (1826â€‰days) after diagnosis. Log-rank test was used to compare treatment groups; <em>p</em>-value indicated. Risk table (bottom panel) shows patients still alive at given time for each treatment group. <strong>C</strong> Representative whole well images and insets of SC cells grown in 24-well type I collagen cultures for 12â€‰days with azithromycin (5â€‰ÂµM), SN-38 (5â€‰nM), or both. Images taken using MuviCyte Live-Cell Imaging System. Boxes indicate inset location (Scale bars: 500â€‰Âµm, Insets: 500â€‰Âµm). <strong>D</strong> Forest plot showing results of multivariable cox proportional hazard model analysis. Age at diagnosis indicates age of the patient at initial diagnosis of cancer, which was broken down into early and late onset with early onset being defined as &lt;50â€‰years of age. Tumor stage indicates the stage of the tumor at initial diagnosis, not necessarily the stage of the cancer when irinotecan treatment was given. Patients with early-stage tumors at diagnosis were likely to have received irinotecan following disease progression or recurrence based on when irinotecan is commonly given. Manual annotation of patientâ€™s date of death was used to determine survival at 5â€‰years (1826â€‰days) after diagnosis.</p></figcaption></figure><p id="Par20">Irinotecan is a topoisomerase inhibitor that has been used to treat CRC and other cancers for more than two decades and remains a critical component of standard-of-care chemotherapy for patients with metastatic CRC<sup><a href="#CR29" class="usa-link" aria-describedby="CR29">29</a></sup>. Azithromycin is a broad-spectrum macrolide antibiotic, that is among the most prescribed antimicrobial drugs in the United States<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR31" class="usa-link" aria-describedby="CR31">31</a></sup>. Given the regular use of both these drugs and the likelihood that at least some patients receiving irinotecan had also received concurrent azithromycin during their treatment course, we utilized de-identified electronic health records from an academic medical center to retrospectively interrogate cancer patient outcomes. This retrospective cohort was established with the following parameters (1) had a diagnosis of CRC, (2) was diagnosed between 1990 and 2024, (3) received irinotecan as a part of their CRC treatment and (4) stratified by whether they received a course of azithromycin while they were receiving irinotecan. Survival outcomes were determined from date of CRC diagnosis and date of death. 157 patients were included in the final cohort, 59 of whom received azithromycin at least once during their irinotecan course, and 98 who did not. A flow chart depicting the inclusion and exclusion of patients for this study is shared in Supplementary Fig.Â <a href="#MOESM2" class="usa-link">10</a>. At baseline there were no differences in patient age, stage at diagnosis, or primary tumor location (Supplementary TableÂ <a href="#MOESM2" class="usa-link">1</a>). Based on univariate analysis, patients who received azithromycin had a median overall survival of 40.3â€‰months (95% CI: 32-not reached) versus 22.4â€‰months (95% CI: 18.9â€“34.2) for those who had not received azithromycin, resulting in a 5â€‰year overall survival rate of 44% (95% CI: 32â€“59%) versus 24% (95% CI: 16â€“35%) respectively (Fig.Â <a href="#Fig5" class="usa-link">5B</a>). When adjusting for covariates, compared with those who did not use azithromycin, patients who used azithromycin had a 42% lower risk of mortality (HRâ€‰=â€‰0.58, 95% CI: 0.37â€“0.90, <em>p</em>â€‰=â€‰0.02) (Fig.Â <a href="#Fig5" class="usa-link">5D</a>). Taken together, these data suggest a synergistic interaction between azithromycin-induced re-epithelialization of CRC cells in our 3D collagen cultures and cell death induced by chemotherapy.</p></section></section><section id="Sec8"><h2 class="pmc_sec_title">Discussion</h2>
<p id="Par21">Treatment resistance is a major clinical concern in cancer, including CRC<sup><a href="#CR32" class="usa-link" aria-describedby="CR32">32</a>â€“<a href="#CR34" class="usa-link" aria-describedby="CR34">34</a></sup>. EMT has been implicated as a resistance mechanism to chemo- and targeted therapies in both CRC and other cancers<sup><a href="#CR35" class="usa-link" aria-describedby="CR35">35</a>â€“<a href="#CR37" class="usa-link" aria-describedby="CR37">37</a></sup>. The changes in a cell as it goes through EMT can be observed as morphological changes in individual cells and overall colony morphology in 3D<sup><a href="#CR38" class="usa-link" aria-describedby="CR38">38</a></sup>. We capitalized on observable changes in colony morphology to conduct a high-throughput morphological screen to identify drugs which reverse EMT and potentially enhance chemotherapy response in CRC.</p>
<p id="Par22">Traditionally, most high-throughput screens are conducted using cells grown in 2D, adhered to a plastic dish<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. While a majority of 2D screens are simple cell death or viability assays, newer screens have performed microscopic analysis to extract morphological features<sup><a href="#CR39" class="usa-link" aria-describedby="CR39">39</a></sup>. Despite these advances, 2D cultures remain inherently ineffective in capturing cell-cell and cell-matrix interactions that are well-represented in 3D cultures. More than 90% of drugs tested in 2D ultimately fail in clinical testing, indicating a need for better in vitro 3D testing systems<sup><a href="#CR40" class="usa-link" aria-describedby="CR40">40</a>,<a href="#CR41" class="usa-link" aria-describedby="CR41">41</a></sup>. However, adoption of 3D cultures (e.g. Matrigel, collagen, hydrogel) has been slow, in part, due to technical challenges like automated handling of viscous fluids and paucity of methods to analyze the effects of drugs on 3D colonies<sup><a href="#CR12" class="usa-link" aria-describedby="CR12">12</a></sup>. Our high-throughput 3D drug screen identified several distinct morphological classes of CRC colonies in collagen, revealing that specific antibioticsâ€”azithromycin, clindamycin, and linezolidâ€”consistently induced dramatic re-circularization and re-epithelialization of CRC colonies. This effect was robust across multiple concentrations and was validated in both high-throughput and standard 24-well formats, further validated by orthogonal RNA-seq and immunofluorescence assays, as well as in another CRC cell line, highlighting the reproducibility of these findings.</p>
<p id="Par23">Our method is the first high-throughput drug screen in 3D type I collagen to be conducted in a 384-well format which utilized morphological markers to identify hits to the best of our knowledge. Not only were we able to identify drugs which induce an epithelial phenotype, which was of interest to us in this study, but an unbiased analysis of a range of morphological markers demonstrated other morphologies can be identified by this method. Limitations of the protocol were necessary to make it feasible, including a shorter incubation period compared to larger format collagen cultures and no replacement of media, which lead to a lower-than-expected percent colonies with lumens for P4G11 treated wells<sup><a href="#CR14" class="usa-link" aria-describedby="CR14">14</a>,<a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>. Despite these limitations, the hits identified in the screen were confirmed in the standard 24-well format collagen cultures and demonstrate a correlative relationship to improved patient response, indicating that this method has potential translational relevance. Other iterations of our 3D screen may include the addition of other ECM components, growth factors, metabolites, other cell types, and staining of colonies with other dyes as desired. This method presents a critical development in high-throughput screening providing a more physiologically relevant culturing and analysis method for future studies. Finally, given the recent US-FDA initiative to phase out animal testing in favor of human-relevant models, such as 3D organoid systems, our new 3D high-throughput morphological assay aligns directly with this shift.</p>
<p id="Par24">The top three hits identified in our screen, azithromycin, clindamycin, and linezolid, are all antibiotics which target the bacterial ribosome to inhibit bacterial growth<sup><a href="#CR42" class="usa-link" aria-describedby="CR42">42</a></sup>. Previous studies of these drugs suggest off target binding to and inhibition of the mitochondrial ribosome (mitoribosome), impacting mitochondrial biogenesis, function, and mitophagy<sup><a href="#CR43" class="usa-link" aria-describedby="CR43">43</a>â€“<a href="#CR49" class="usa-link" aria-describedby="CR49">49</a></sup>. While normal cells may be relatively unperturbed by a modest reduction in mitochondrial function, these antibiotics would have a disproportionately larger effect on cancer cells, which have a higher metabolic burden. Mitochondrial dysfunction could also impact EMT, since it requires major metabolic changes in the cell, specifically a decrease in oxidative phosphorylation and downregulation of mitochondrial proteins combined with increased glycolysis<sup><a href="#CR50" class="usa-link" aria-describedby="CR50">50</a>,<a href="#CR51" class="usa-link" aria-describedby="CR51">51</a></sup>. Our findings in 3D cultures suggest an impact of these antibiotics on EMT, epithelial polarity, and mitochondrial function that remained unappreciated in 2D cultures.</p>
<p id="Par25">Our detailed RNA-seq analysis showed that all three antibiotics downregulated hallmark EMT and KRAS signaling pathways, both of which are central to cancer cell plasticity and invasiveness. Notably, azithromycin and clindamycin also suppressed inflammatory and angiogenic pathways, consistent with their known anti-inflammatory properties<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a>,<a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Linezolid, in contrast, uniquely upregulated mitochondrially-encoded ETC genes. At gene expression level, these antibiotics restored expression of proteins associated with tight junctions, intermediate filaments, and other epithelial markers. Together, these findings highlight common and unique antibiotic actions. The ability of antibiotics to induce re-epithelialization and suppress pro-invasive gene expression in CRC colonies suggests a previously unappreciated potential for repurposing these agents to modulate tumor cell plasticity and differentiation.</p>
<p id="Par26">As shown in the results and discussed below, azithromycin-induced EMT reversal enhanced chemotherapy efficacy in vitro and correlated with improved chemotherapy response in CRC patients. Similarly, other combinations could be explored based on antibiotic-driven transcriptional reprogramming, such as pairing with KRAS inhibitors, given KRAS signaling suppression with all three antibiotics. A potential role of antibiotics in enhancing immunotherapy may also be considered, as azithromycin and clindamycin downregulated inflammatory and immune-evasive pathways (e.g., TNFÎ±, STAT). These findings suggest that antibiotics may not only limit tumor cell plasticity but also modulate the tumor microenvironment in ways that could enhance immune surveillance. Our 3D culture system is well-suited for testing such combinations, with or without added cell types, or changes in composition of ECM proteins and ligands.</p>
<p id="Par27">Azithromycin is routinely used for its antibiotic properties in the treatment of respiratory, urogenital, dermal and other bacterial infections, and for its anti-inflammatory properties in the treatment of cystic fibrosis, chronic obstructive pulmonary disease, and asthma<sup><a href="#CR30" class="usa-link" aria-describedby="CR30">30</a></sup>. In epithelial cells, azithromycin has been shown to maintain transepithelial resistance in the presence of virulence factors that induce permeability<sup><a href="#CR53" class="usa-link" aria-describedby="CR53">53</a></sup>. Azithromycin also altered processing and arrangement of tight junction adhesion molecules in airway epithelial cells<sup><a href="#CR54" class="usa-link" aria-describedby="CR54">54</a></sup>. These known azithromycin effects may contribute to the re-epithelialization of CRC 3D colonies. Interestingly, azithromycin is also prescribed for a non-cancerous EMT disorder of the conjunctival epithelial tissue, Pterygium<sup><a href="#CR52" class="usa-link" aria-describedby="CR52">52</a></sup>. Here, azithromycin is prescribed for its anti-inflammatory actions, however, our findings indicate that its anti-EMT actions might also contribute to azithromycin efficacy in this context. Cooperation of azithromycin with aÂ chemotherapeutic is surprising but not without precedence. Azithromycin has been shown to be a potent autophagy inhibitor in cancer<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a>â€“<a href="#CR58" class="usa-link" aria-describedby="CR58">58</a></sup>. In pancreatic cancer cells, azithromycin was shown to enhance the cytotoxicity of the EGFR inhibitor gefitinib through autophagy inhibition<sup><a href="#CR55" class="usa-link" aria-describedby="CR55">55</a></sup>. Thus, studying the effect of azithromycin on anticancer response warrants further investigation, and our tractable 3D system presents as a robust alternative to 2D cultures.</p>
<p id="Par28">While the widespread use of azithromycin allowed us to conduct a retrospective cohort study on its potential interaction with irinotecan, there were limitations to this study. To ensure confounding variables, such as tumor stage at diagnosis and tumor location did not significantly impact the results, we conducted a multivariate analysis. While indication for and duration of azithromycin are potentially confounding variables, these were not included in this analysis. Patients were excluded for the following factors (1) those who received azithromycin either before or after their treatment course with irinotecan-based chemotherapy (2) cancers of non-CRC origin, (3) unclear date of death. Other factors such as comorbid medical conditions and other antibiotics taken during treatment with CRC could not be taken into consideration due to small sample size and limited data availability in the electronic health record. Despite the limitations of our retrospective study, the results, and the results of our in vitro experiments, demonstrate a convincing correlation between the addition of azithromycin to irinotecan-based chemotherapy and improved 5â€‰year survival for patients with CRC.</p>
<p id="Par29">This 3D high-throughput drug screen in type I collagen identified compounds that enhance chemotherapy efficacy. Further validation in prospective trials and/or animal models, as well as determining the mechanism of action of these drugs are important future directions. While it is clear these drugs are inducing re-epithelialization of CRC colonies in collagen, which would account for the increased efficacy of chemotherapy treatment, the binding targets of these antibiotics which mediate this process are unknown. In addition to the drugs shared in this study, data shared for the effect of &gt;1000 FDA-approved compounds on 3D colony morphology (see Supplementary DataÂ <a href="#MOESM4" class="usa-link">1</a> and associated repository<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>) may be of broader interest as an additional parameter to measure the effect of anti-cancer drugs. The results of this study demonstrate the effectiveness of 3D morphological high-throughput screens in the development of novel therapies or combination therapies for the treatment of CRC and other cancers.</p></section><section id="Sec9"><h2 class="pmc_sec_title">Materials and methods</h2>
<section id="Sec10"><h3 class="pmc_sec_title">Reagents</h3>
<p id="Par30">PureCol<sup>Â®</sup> bovine type I collagen was purchased from Advanced Biomatrix, Inc. (San Diego, CA, USA, #5005). Unless specified otherwise, all cell culture components were purchased from Hyclone laboratories, Inc. (Omaha, NE, USA). DMEM was purchased from Corning (Corning, NY, USA, #10-027-CV). 10X MEM and GlutaMAX<sup>TM</sup> were purchased from Gibco (Waltham, MA, USA, #11430030, #35050061). Sodium hydroxide 10 Normal was purchased from VWR Chemicals BDH (Radnor, PA, USA, #BDH3247-1). Anti-Integrin Î²1 Antibody, clone P4G11 was purchased from Sigma-Aldrich (Damstadt, Germany, #MAB1951). Azithromycin, clindamycin, and linezolid were purchased from Selleck Chemicals LLC (Houston, TX, USA, #S1835, #S2830, #S1408). SN-38 was purchased from MedChemExpress LLC (Princeton, NJ, USA, #HY-13704). Anti-E-cadherin was purchased from BD Transduction Laboratories (Franklin Lakes, NJ, USA, #610181). Anti-ZO-1 antibody was purchased from Invitrogen (Waltham, MA, USA, #61-7300).</p>
<p id="Par31">384-well plates (CELLSTAR Cell Culture Microplate, Cell-Culture Treated, Flat-Bottom, with Lid) were purchased from Greiner Bio-One (KremsmÃ¼nster, Austria, #781090). Reagent reservoirs were purchased from ThermoFisher Scientific (Waltham, MA, USA, #1064-05-7). P30 Aligent Compatible tips were purchased from Fluotics (New York, NY, USA #AGI-30.ST). Calcein Green AM was purchased from Invitrogen (Waltham, MA, USA, #C1430). FDA approved drug library was purchased from Selleck Chemicals LLC (Houston, TX, USA, #L1300).</p></section><section id="Sec11"><h3 class="pmc_sec_title">Cell lines and cell culture (maintenance of cells on plastic)</h3>
<p id="Par32">The HCA-7 cell line was obtained from Susan Kirkland (Imperial Cancer Research Fund); generation of its derivative (SC) has been described earlier<sup><a href="#CR9" class="usa-link" aria-describedby="CR9">9</a></sup>. Unless otherwise indicated, cells were maintained in DMEM supplemented with 10% bovine growth serum, 1% non-essential amino acids, 1% L-glutamine, and 1% penicillin/streptomycin.</p></section><section id="Sec12"><h3 class="pmc_sec_title">3D type I collagen culture in 24-well format</h3>
<p id="Par33">The concentration of collagen was 2â€‰mg/mL (PureCol Type I Collagen, #5005) in 1X MEM (diluted from 10X MEM with sterile water) and supplemented with 10% fetal bovine serum and pH adjusted to neutral with 10â€‰N NaOH. For a 24-well dish, three distinct collagen layers at 250â€‰ÂµL each were layered on top of each other (collagen layer, single-cell collagen suspension layer, and collagen layer). Cells were seeded at 5,000 cells/mL (1,250 cells/well) for colony countingÂ and circularity experiments. After the last colony layer was polymerized, 250â€‰ÂµL of media with indicated treatment was added to each well. Media changes occurred every 2â€“3â€‰days. Colonies were counted orÂ imaged after 10-14â€‰days. Colonies were counted using GelCount (Oxford Optronix) with identical acquisition and analysis settings.Â Alternatively colonies were assessed for circularity as described below.Â </p></section><section id="Sec13"><h3 class="pmc_sec_title">3D type I collagen culture in 384-wells</h3>
<p id="Par34">High throughput drug screen was performed in collaboration with the Vanderbilt High Throughput Screening (VHTS) facility.</p>
<section id="Sec14"><h4 class="pmc_sec_title">Preparing reagents for high throughput drug screen</h4>
<p id="Par35">The day prior to plating, 384-well plates, tips, and reagent reservoirs were placed at âˆ’20Â°C until used for plating. Type I collagen was prepared at a concentration of 2â€‰mg/mL in 1X MEM (diluted from 10X MEM with sterile water), supplemented with 10% fetal bovine serum and pH adjusted to neutral with 10â€‰N NaOH. Two collagen aliquots per round of plating were stored on ice until use. A single-cell suspension of SC cells was added to one aliquot of type I collagen at a concentration of 1.0â€‰Ã—â€‰10<sup>5</sup> cells/mL (1000 cells/well) and the type I collagen was then inverted to evenly distribute cells. This aliquot was used to plate the layer of cells in collagen.</p></section><section id="Sec15"><h4 class="pmc_sec_title">High-throughput drug screen</h4>
<p id="Par36">Using an Agilent BRAVO automated pipette liquid transfer system (Agilent Technologies, Santa Clara, CA, USA), chilled plates were stamped with a 5â€‰ÂµL bottom layer of collagen. Plates were incubated at -20Â°C for 15â€‰min, with sides of the plates being tapped every 5â€‰min to ensure the bottom of each well was entirely coated with collagen. Plates were incubated for another 15â€‰min at 37Â°C. Plates were stamped with a 10â€‰ÂµL layer of cells in collagen using the BRAVO, then incubated at 37Â°C with 5% CO<sub>2</sub> for 30â€‰min. Drugs and small molecules were transferred from the library stock plates at 10â€‰mM DMSO solutions using an ECHO acoustic liquid transfer system (Beckman Coulter Life Science, Indianapolis, IN, USA) to 384-well drug plate. Drug plates were diluted with complete DMEM supplemented with 10% fetal bovine serum, non-essential amino acids, L-glutamine, penicillin/streptomycin, and GlutaMAX<sup>TM</sup>. 40â€‰ÂµL of media containing drugs and small molecules was added to the cell plates at three final concentrations (0.1â€‰ÂµM, 1â€‰ÂµM, and 10â€‰ÂµM). Final concentrations of drugs and small molecules account for volume of collagen in the wells. Cells were incubated for 8â€‰days and then stained with Calcein AM at a final concentration of 3.75â€‰ÂµM for 3â€‰h. Plates were imaged on an ImageXpress confocal HT.ai automated high-content imaging system (Molecular Devices, LLC, San Jose, CA, USA) using the transmitted light and FITC channels and 4x objective. Z-series were collected containing 17 planes with a step size of 50â€‰ÂµM and maximum intensity projections were saved.</p></section></section><section id="Sec16"><h3 class="pmc_sec_title">Analysis of 3D collagen culture in 384 wells</h3>
<section id="Sec17"><h4 class="pmc_sec_title">PCA analysis</h4>
<p id="Par37">Images were analyzed using the InCarta image analysis software (Molecular Devices, LLC, San Jose, CA, USA). A deep learning-based model (SINAP: Segmentation Is Not a Problem) was specifically trained to identify and segment both the cystic and spiky colonies in the transmitted light images. Details on the acquisition and definitions for each of the morphological measurements taken can be found on the Molecular Devices website (<a href="https://www.moleculardevices.com/products/cellular-imaging-systems/high-content-analysis/in-carta-image-analysis-software" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.moleculardevices.com/products/cellular-imaging-systems/high-content-analysis/in-carta-image-analysis-software</a>). Images of colonies from control treated wells were used as the training set data. Data frames were filtered to remove empty wells and incomplete data points, Scikit-learn<sup><a href="#CR59" class="usa-link" aria-describedby="CR59">59</a></sup> was used to generate pair-wise intervariable Pearson correlation coefficients. Using the StandardScaler class of the scikit-learn Python library, raw reads were z-score standardized per variables and principal component analysis was conducted. PC loadings were calculated: <span xmlns:mml="http://www.w3.org/1998/Math/MathML" id="IEq1"><math id="d33e1302"><mstyle><mtext>eigenvector</mtext></mstyle><mo>*</mo><mspace width="0.25em"></mspace><msqrt><mrow><mstyle><mtext>eigenvalues</mtext></mstyle></mrow></msqrt></math></span>.</p></section><section id="Sec18"><h4 class="pmc_sec_title">Median area, percent colonies with lumens, and total colony counts</h4>
<p id="Par38">Fluorescent images were analyzed using the MetaXpress high content image analysis and acquisition software (Molecular Devices, LLC., San Jose, CA, USA). Intensity thresholds were used to remove out of focus colonies, monolayers, or lumens. Size filtering was used to remove colonies and lumens above or below a given size, often representing false positives. The masks generated from this pipeline was used to determine the area of individual colonies which were exported and R software (<a href="https://www.r-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.r-project.org/</a>)<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup> used to calculate the median area, number of colonies, and percent colonies with lumens for each well. Fold change median colony area and percent colonies with lumens were calculated by dividing the value for each well by the average for the DMSO control wells of the corresponding plate. B Scores were calculated, as previously described<sup><a href="#CR15" class="usa-link" aria-describedby="CR15">15</a></sup>, for each drug treated well.</p></section></section><section id="Sec19"><h3 class="pmc_sec_title">Circularity analysis</h3>
<p id="Par39">Circularity analysis was conducted as previously described<sup><a href="#CR11" class="usa-link" aria-describedby="CR11">11</a></sup> but in brief; cells were seeded at 5000 cells/mL in type I collagen. Colonies were grown with indicated treatment for 10-14â€‰days. Sixteen images per well were captured on the final day using the MuviCyte Live-Cell Imaging System (PerkinElmer). Images were analyzed in ImageJ (NIH) with the MorphoLibJ plugin<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Briefly, background of images was removed and a mask of colonies generated. Connected components labeling in the MorphoLibJ plugin was used to identify colonies and the size filtering used to remove colonies which were too large or small<sup><a href="#CR61" class="usa-link" aria-describedby="CR61">61</a></sup>. Characteristics of colonies were recorded, and colony circularity calculated using the equation: (4*Area*Ï€)/(Perimeter<sup>2</sup>).</p></section><section id="Sec20"><h3 class="pmc_sec_title">Immunofluorescence</h3>
<p id="Par40">After 10-14â€‰days in culture, middle layers of mature collagen cultures were separated using tweezers and washed with PBS three times, 5â€‰min each. The collagen layer was then fixed with 4% PFA at 4Â°C for 1â€‰h with gentle rocking. Collagen layer was washed again with PBS 3â€‰times, 5â€‰min each and then placed in IF buffer overnight at 4Â°C with gentle rocking (IF buffer: 1% BSA, 0.1% Triton X-100, 0.01% sodium azide in PBS). All remaining steps are performed in IF buffer at room temperature with gentle rocking. Collagen layers were then blocked with 3% normal donkey serum for 2â€“4â€‰h. Primary antibodies were added for 1â€‰h followed by 3â€‰Ã—â€‰15â€‰min washes. Secondary antibodies were then added for 1â€‰h followed by 3â€‰Ã—â€‰15â€‰min washes. DAPI and Phalloidin dyes were added during the secondary antibody incubation. Stained collagen layers were then placed in chamber slides with coverslip bottoms for imaging.</p></section><section id="Sec21"><h3 class="pmc_sec_title">Construction of patient cohort and patient data extraction</h3>
<p id="Par41">Patient data was obtained from the Synthetic Derivative, a deidentified version of Vanderbiltâ€™s electronic medical record<sup><a href="#CR62" class="usa-link" aria-describedby="CR62">62</a>â€“<a href="#CR64" class="usa-link" aria-describedby="CR64">64</a></sup>. PatientsÂ were initially identified based on whether they met specific screening criteria including (1) colorectal cancer diagnosis documented by ICD9 codes 153.0â€“153.9 or ICD10 codes C18.0-C18.9, and (2) treatment with irinotecan. This identified a total of 1381 patients including 236 patients who had received irinotecan and azithromycin, and 1145 patients who received irinotecan only. Of the patients who had received irinotecan only, 160 were randomly selected for further review and all of the 236 patients who received irinotecan and azithromycin were selected for further review. Each patientâ€™s Synthetic Derivative chart was then manually reviewed, and the following information were extracted: (1) age at diagnosis, (2) location of tumor, (3) date of diagnosis, (4) stage at diagnosis (5) date of death (if applicable), (6) date of first documented irinotecan dose, and (7) date of first documented azithromycin dose (if applicable). Patients were excluded if cancer was not colorectal in origin, irinotecan was never received, azithromycin was not given during irinotecan treatment, or date of death could not be determined. Thus, the irinotecan and azithromycin group included patients that received azithromycin during irinotecan-based chemotherapy treatment course. After manual review, 98 patients met the criteria in the irinotecan only group and 59 met the criteria in the irinotecan and azithromycin group. A schematic showing patient inclusion/exclusion decisions is additionally included in Fig.Â <a href="#MOESM2" class="usa-link">S10</a> flow chart.</p></section><section id="Sec22"><h3 class="pmc_sec_title">Statistics and reproducibility</h3>
<p id="Par42">384-well drug screen was performed at three concentrations, with one technical replicate each; 14 DMSO and P4G11 control replicates were included in each 384-well dish. 384-well dose curves were performed with three technical replicates at each concentration. All remaining experiments were performed in at least three technical and three biological replicates. All statistical analyses and graphing were performed using R software version 4.4<sup><a href="#CR60" class="usa-link" aria-describedby="CR60">60</a></sup>. Detailed descriptions of the statistical tools and methods for each analysis are provided in table and figure legends. For analysis of patient data, patientsâ€™ characteristics were presented as medians with interquartile ranges or as frequencies with percentages. Comparisons of patient characteristics between treatment groups were made using the Wilcoxon rank-sum test or the chi-squared test. To evaluate the association between treatment and overall survival, the Kaplan-Meier method and Cox proportional hazards model were employed. For all analyses, hazard ratios (HRs), 95% confidence intervals, and <em>p</em>-values were reported.</p></section><section id="Sec23"><h3 class="pmc_sec_title">Reporting summary</h3>
<p id="Par43">Further information on research design is available in theÂ <a href="#MOESM5" class="usa-link">Nature Portfolio Reporting Summary</a> linked to this article.</p></section></section><section id="Sec24"><h2 class="pmc_sec_title">Supplementary information</h2>
<section class="sm xbox font-sm" id="MOESM1"><div class="media p"><div class="caption">
<a href="/articles/instance/12373842/bin/42003_2025_8699_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (218.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM2"><div class="media p"><div class="caption">
<a href="/articles/instance/12373842/bin/42003_2025_8699_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplemental Material</a><sup> (82.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM3"><div class="media p"><div class="caption">
<a href="/articles/instance/12373842/bin/42003_2025_8699_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">42003_2025_8699_MOESM3_ESM.pdf</a><sup> (45.2KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="MOESM4"><div class="media p"><div class="caption">
<a href="/articles/instance/12373842/bin/42003_2025_8699_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplemental Data 1</a><sup> (2.1MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="MOESM5"><div class="media p"><div class="caption">
<a href="/articles/instance/12373842/bin/42003_2025_8699_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">NR Reporting Summary</a><sup> (2MB, pdf) </sup>
</div></div></section></section><section id="ack1" class="ack"><h2 class="pmc_sec_title">Acknowledgements</h2>
<p>We thank Sarah E. Glass for editing this manuscript. We thank Martha Shrubsole and Alex Hawkins for epidemiology or treatment-related clarifications. C.K.S. was supported by the Vanderbiltâ€“Ingram Cancer Center Brock Family Fellowship. V.T.J. was supported in part by T32GM07628. K.S.L. is supported in part by funds from U01CA294527 (HTAN), R01DK103831. R.J.C. acknowledges support from NCI R35 CA197570, P50 236733, and the Nicholas Tierney GI Cancer Memorial Fund. J.A.B. is supported by an NCI R50 Research Specialist Award R50CA211206. The Vanderbilt High-Throughput Screening (VHTS) Core receives support from the Vanderbilt Institute of Chemical Biology and the Vanderbilt Ingram Cancer Center (P30CA68485). The ImageXpress Confocal HT.ai instrument and Hamilton Verso compound storage system are housed and managed by the VHTS Core Facility, and were funded by NIH Shared Instrumentation Grant S10OD028719 and S10OD028715. B.S. is supported in part by funds from NIH/NCI R01CA248505, American Cancer Societyâ€”Research Scholar Grant RSG-20â€“130-01-DDC, and P50 CA236733. We acknowledge VUMC core scholarships from VUMC Cancer Center Support Grant (CCSG) P30CA068485 and VUMC Digestive Diseases Research Center grant (DDRC) P30DK058404.</p></section><section id="notes1"><h2 class="pmc_sec_title">Author contributions</h2>
<p>S.J.H., T.P.H., C.K.S., V.T.J., J.A.B., and B.S. conceived research. S.J.H., T.P.H., C.K.S., V.T.J, and C.C.W. performed experiments. S.J.H., T.P.H., S.W.K., M.A.R., Z.Z., O.K., Q.L., K.S.L., R.J.C., J.A.B., and B.S. conducted data analysis and interpretation. S.J.H. assembled figures and table. S.J.H and B.S. wrote initial draft of manuscript. All authors reviewed and approved the manuscript.</p></section><section id="notes2"><h2 class="pmc_sec_title">Peer review</h2>
<section id="FPar1"><h3 class="pmc_sec_title">Peer review information</h3>
<p id="Par44"><em>Communications Biology</em> thanks the anonymous reviewers for their contribution to the peer review of this work. Primary Handling Editors: Georgios Giamas and Christina Karlsson Rosenthal. A peer review file is available.</p></section></section><section id="notes3"><h2 class="pmc_sec_title">Data availability</h2>
<p>Data to generate graphs and additional data from the screen available in Supplementary DataÂ <a href="#MOESM4" class="usa-link">1</a>. Images generated in high-throughput screen are available in associated repository<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. RNA-seq data has been deposited into the NCBI GEO under the accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE304081" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE304081</a>. Any additional information related to this paper is available from the corresponding author upon reasonable request.</p></section><section id="notes4"><h2 class="pmc_sec_title">Materials availability</h2>
<p>This study did not generate unique reagents.</p></section><section id="FPar2"><h2 class="pmc_sec_title">Competing interests</h2>
<p id="Par45">The authors declare no competing interests.</p></section><section id="fn-group1" class="fn-group"><h2 class="pmc_sec_title">Footnotes</h2>
<div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="fn1"><p><strong>Publisherâ€™s note</strong> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section id="sec25"><h2 class="pmc_sec_title">Supplementary information</h2>
<p>The online version contains supplementary material available at 10.1038/s42003-025-08699-0.</p></section><section id="Bib1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="Bib1_sec2"><ul class="ref-list font-sm" style="list-style-type:none">
<li id="CR1">
<span class="label">1.</span><cite>Peglion, F. &amp; Etienne-Manneville, S. Cell polarity changes in cancer initiation and progression. <em>J. Cell Biol</em>. <strong>223</strong>, e202308069 (2024).</cite> [<a href="https://doi.org/10.1083/jcb.202308069" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10720656/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38091012/" class="usa-link">PubMed</a>]</li>
<li id="CR2">
<span class="label">2.</span><cite>Lee, M. &amp; Vasioukhin, V. Cell polarity and cancer - cell and tissue polarity as a non-canonical tumor suppressor. <em>J. Cell Sci.</em><strong>121</strong>, 1141â€“1150 (2008).
</cite> [<a href="https://doi.org/10.1242/jcs.016634" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18388309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lee,%20M.%20&amp;%20Vasioukhin,%20V.%20Cell%20polarity%20and%20cancer%20-%20cell%20and%20tissue%20polarity%20as%20a%20non-canonical%20tumor%20suppressor.%20J.%20Cell%20Sci.121,%201141%E2%80%931150%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR3">
<span class="label">3.</span><cite>GandaloviÄovÃ¡, A., Vomastek, T., Rosel, D. &amp; BrÃ¡bek, J. Cell polarity signaling in the plasticity of cancer cell invasiveness. <em>Oncotarget</em><strong>7</strong>, 25022â€“25049 (2016).</cite> [<a href="https://doi.org/10.18632/oncotarget.7214" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5041887/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26872368/" class="usa-link">PubMed</a>]</li>
<li id="CR4">
<span class="label">4.</span><cite>Sabouni, E. et al. Unraveling the function of epithelial-mesenchymal transition (EMT) in colorectal cancer: Metastasis, therapy response, and revisiting molecular pathways. <em>Biomed. Pharmacother</em>. <strong>160</strong>, 114395 (2023).</cite> [<a href="https://doi.org/10.1016/j.biopha.2023.114395" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36804124/" class="usa-link">PubMed</a>]</li>
<li id="CR5">
<span class="label">5.</span><cite>Kozlowski, M. T., Crook, C. J. &amp; Ku, H. T. Towards organoid culture without Matrigel. <em>Commun. Biol.</em><strong>4</strong>, 1387 (2021).
</cite> [<a href="https://doi.org/10.1038/s42003-021-02910-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8664924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34893703/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kozlowski,%20M.%20T.,%20Crook,%20C.%20J.%20&amp;%20Ku,%20H.%20T.%20Towards%20organoid%20culture%20without%20Matrigel.%20Commun.%20Biol.4,%201387%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR6">
<span class="label">6.</span><cite>UrzÃ¬, O. et al. Three-dimensional cell cultures: the bridge between in vitro and in vivo models. <em>Int. J. Mol. Sci.</em><strong>24</strong>, 12046 (2023).
</cite> [<a href="https://doi.org/10.3390/ijms241512046" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10419178/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37569426/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Urz%C3%AC,%20O.%20et%20al.%20Three-dimensional%20cell%20cultures:%20the%20bridge%20between%20in%20vitro%20and%20in%20vivo%20models.%20Int.%20J.%20Mol.%20Sci.24,%2012046%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR7">
<span class="label">7.</span><cite>Simian, M. &amp; Bissell, M. J. Organoids: A historical perspective of thinking in three dimensions. <em>J. Cell Biol.</em><strong>216</strong>, 31â€“40 (2017).
</cite> [<a href="https://doi.org/10.1083/jcb.201610056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5223613/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28031422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Simian,%20M.%20&amp;%20Bissell,%20M.%20J.%20Organoids:%20A%20historical%20perspective%20of%20thinking%20in%20three%20dimensions.%20J.%20Cell%20Biol.216,%2031%E2%80%9340%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR8">
<span class="label">8.</span><cite>Li, C. et al. Excess PLAC8 promotes an unconventional ERK2-dependent EMT in colon cancer. <em>J. Clin. Invest.</em><strong>124</strong>, 2172â€“2187 (2014).
</cite> [<a href="https://doi.org/10.1172/JCI71103" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4001536/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24691442/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20C.%20et%20al.%20Excess%20PLAC8%20promotes%20an%20unconventional%20ERK2-dependent%20EMT%20in%20colon%20cancer.%20J.%20Clin.%20Invest.124,%202172%E2%80%932187%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR9">
<span class="label">9.</span><cite>Li, C. et al. Three-dimensional culture system identifies a new mode of cetuximab resistance and disease-relevant genes in colorectal cancer. <em>Proc. Natl. Acad. Sci. USA.</em><strong>114</strong>, E2852â€“E2861 (2017).
</cite> [<a href="https://doi.org/10.1073/pnas.1618297114" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5389279/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28320945/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Li,%20C.%20et%20al.%20Three-dimensional%20culture%20system%20identifies%20a%20new%20mode%20of%20cetuximab%20resistance%20and%20disease-relevant%20genes%20in%20colorectal%20cancer.%20Proc.%20Natl.%20Acad.%20Sci.%20USA.114,%20E2852%E2%80%93E2861%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR10">
<span class="label">10.</span><cite>Graves-Deal, R. et al. Broad-spectrum receptor tyrosine kinase inhibitors overcome de novo and acquired modes of resistance to EGFR-targeted therapies in colorectal cancer. <em>Oncotarget</em><strong>10</strong>, 1320â€“1333 (2019).
</cite> [<a href="https://doi.org/10.18632/oncotarget.26663" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6407678/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30863492/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Graves-Deal,%20R.%20et%20al.%20Broad-spectrum%20receptor%20tyrosine%20kinase%20inhibitors%20overcome%20de%20novo%20and%20acquired%20modes%20of%20resistance%20to%20EGFR-targeted%20therapies%20in%20colorectal%20cancer.%20Oncotarget10,%201320%E2%80%931333%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR11">
<span class="label">11.</span><cite>Jones, V. T. et al. Inhibition of autocrine HGF maturation overcomes cetuximab resistance in colorectal cancer. <em>Cell. Mol. Life Sci.</em><strong>81</strong>, 28 (2024).
</cite> [<a href="https://doi.org/10.1007/s00018-023-05071-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10784391/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38212428/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Jones,%20V.%20T.%20et%20al.%20Inhibition%20of%20autocrine%20HGF%20maturation%20overcomes%20cetuximab%20resistance%20in%20colorectal%20cancer.%20Cell.%20Mol.%20Life%20Sci.81,%2028%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR12">
<span class="label">12.</span><cite>Booij, T. H. et al. Development of a 3D tissue culture-based high-content screening platform that uses phenotypic profiling to discriminate selective inhibitors of receptor tyrosine kinases. <em>J. Biomol. Screen.</em><strong>21</strong>, 912â€“922 (2016).
</cite> [<a href="https://doi.org/10.1177/1087057116657269" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5030728/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27412535/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Booij,%20T.%20H.%20et%20al.%20Development%20of%20a%203D%20tissue%20culture-based%20high-content%20screening%20platform%20that%20uses%20phenotypic%20profiling%20to%20discriminate%20selective%20inhibitors%20of%20receptor%20tyrosine%20kinases.%20J.%20Biomol.%20Screen.21,%20912%E2%80%93922%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR13">
<span class="label">13.</span><cite>Harmych, S. J. et al. 3D collagen high-throughput screen identifies drugs that induce epithelial polarity and enhance chemotherapy response in colorectal cancer. <em>BioStudies</em>10.6019/S-BIAD2215 (2025).</cite> [<a href="https://doi.org/10.1038/s42003-025-08699-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/40847193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harmych,%20S.%20J.%20et%20al.%203D%20collagen%20high-throughput%20screen%20identifies%20drugs%20that%20induce%20epithelial%20polarity%20and%20enhance%20chemotherapy%20response%20in%20colorectal%20cancer.%20BioStudies10.6019/S-BIAD2215%20(2025)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR14">
<span class="label">14.</span><cite>Starchenko, A. et al. Cell surface integrin Î±5ÃŸ1 clustering negatively regulates receptor tyrosine kinase signaling in colorectal cancer cells via glycogen synthase kinase 3. <em>Integr. Biol.</em><strong>13</strong>, 153â€“166 (2021).</cite> [<a href="https://doi.org/10.1093/intbio/zyab009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8204629/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34037774/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Starchenko,%20A.%20et%20al.%20Cell%20surface%20integrin%20%CE%B15%C3%9F1%20clustering%20negatively%20regulates%20receptor%20tyrosine%20kinase%20signaling%20in%20colorectal%20cancer%20cells%20via%20glycogen%20synthase%20kinase%203.%20Integr.%20Biol.13,%20153%E2%80%93166%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR15">
<span class="label">15.</span><cite>Starchenko, A. et al. Clustering of integrin Î±5 at the lateral membrane restores epithelial polarity in invasive colorectal cancer cells. <em>Mol. Biol. Cell</em><strong>28</strong>, 1288â€“1300 (2017).
</cite> [<a href="https://doi.org/10.1091/mbc.E16-12-0852" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5426844/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28356422/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Starchenko,%20A.%20et%20al.%20Clustering%20of%20integrin%20%CE%B15%20at%20the%20lateral%20membrane%20restores%20epithelial%20polarity%20in%20invasive%20colorectal%20cancer%20cells.%20Mol.%20Biol.%20Cell28,%201288%E2%80%931300%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR16">
<span class="label">16.</span><cite>Brideau, C., Gunter, B., Pikounis, B. &amp; Liaw, A. Improved statistical methods for hit selection in high-throughput screening. <em>J. Biomol. Screen.</em><strong>8</strong>, 634â€“647 (2003).
</cite> [<a href="https://doi.org/10.1177/1087057103258285" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14711389/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Brideau,%20C.,%20Gunter,%20B.,%20Pikounis,%20B.%20&amp;%20Liaw,%20A.%20Improved%20statistical%20methods%20for%20hit%20selection%20in%20high-throughput%20screening.%20J.%20Biomol.%20Screen.8,%20634%E2%80%93647%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR17">
<span class="label">17.</span><cite>Kyuno, D. et al. Role of tight junctions in the epithelial-to-mesenchymal transition of cancer cells. <em>Biochim. Biophys. Acta BBA - Biomembr.</em><strong>1863</strong>, 183503 (2021).</cite> [<a href="https://doi.org/10.1016/j.bbamem.2020.183503" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33189716/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kyuno,%20D.%20et%20al.%20Role%20of%20tight%20junctions%20in%20the%20epithelial-to-mesenchymal%20transition%20of%20cancer%20cells.%20Biochim.%20Biophys.%20Acta%20BBA%20-%20Biomembr.1863,%20183503%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR18">
<span class="label">18.</span><cite>Tan, T. Z. et al. Epithelialâ€mesenchymal transition spectrum quantification and its efficacy in deciphering survival and drug responses of cancer patients. <em>EMBO Mol. Med.</em><strong>6</strong>, 1279â€“1293 (2014).
</cite> [<a href="https://doi.org/10.15252/emmm.201404208" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4287932/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25214461/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tan,%20T.%20Z.%20et%20al.%20Epithelial%E2%80%90mesenchymal%20transition%20spectrum%20quantification%20and%20its%20efficacy%20in%20deciphering%20survival%20and%20drug%20responses%20of%20cancer%20patients.%20EMBO%20Mol.%20Med.6,%201279%E2%80%931293%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR19">
<span class="label">19.</span><cite>Youssef, K. K. et al. Two distinct epithelial-to-mesenchymal transition programs control invasion and inflammation in segregated tumor cell populations. <em>Nat. Cancer</em><strong>5</strong>, 1660â€“1680 (2024).
</cite> [<a href="https://doi.org/10.1038/s43018-024-00839-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11584407/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39414946/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Youssef,%20K.%20K.%20et%20al.%20Two%20distinct%20epithelial-to-mesenchymal%20transition%20programs%20control%20invasion%20and%20inflammation%20in%20segregated%20tumor%20cell%20populations.%20Nat.%20Cancer5,%201660%E2%80%931680%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR20">
<span class="label">20.</span><cite>Haerinck, J., Goossens, S. &amp; Berx, G. The epithelialâ€“mesenchymal plasticity landscape: principles of design and mechanisms of regulation. <em>Nat. Rev. Genet.</em><strong>24</strong>, 590â€“609 (2023).
</cite> [<a href="https://doi.org/10.1038/s41576-023-00601-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37169858/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Haerinck,%20J.,%20Goossens,%20S.%20&amp;%20Berx,%20G.%20The%20epithelial%E2%80%93mesenchymal%20plasticity%20landscape:%20principles%20of%20design%20and%20mechanisms%20of%20regulation.%20Nat.%20Rev.%20Genet.24,%20590%E2%80%93609%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR21">
<span class="label">21.</span><cite>Ikenouchi, J., Matsuda, M., Furuse, M. &amp; Tsukita, S. Regulation of tight junctions during the epithelium-mesenchyme transition:direct repression of the gene expression of claudins/occludin by Snail. <em>J. Cell Sci.</em><strong>116</strong>, 1959â€“1967 (2003).
</cite> [<a href="https://doi.org/10.1242/jcs.00389" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12668723/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Ikenouchi,%20J.,%20Matsuda,%20M.,%20Furuse,%20M.%20&amp;%20Tsukita,%20S.%20Regulation%20of%20tight%20junctions%20during%20the%20epithelium-mesenchyme%20transition:direct%20repression%20of%20the%20gene%20expression%20of%20claudins/occludin%20by%20Snail.%20J.%20Cell%20Sci.116,%201959%E2%80%931967%20(2003)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR22">
<span class="label">22.</span><cite>TabariÃ¨s, S. &amp; Siegel, P. M. The role of claudins in cancer metastasis. <em>Oncogene</em><strong>36</strong>, 1176â€“1190 (2017).
</cite> [<a href="https://doi.org/10.1038/onc.2016.289" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27524421/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Tabari%C3%A8s,%20S.%20&amp;%20Siegel,%20P.%20M.%20The%20role%20of%20claudins%20in%20cancer%20metastasis.%20Oncogene36,%201176%E2%80%931190%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR23">
<span class="label">23.</span><cite>Bhat, A. A. et al. Claudin-7 expression induces mesenchymal to epithelial transformation (MET) to inhibit colon tumorigenesis. <em>Oncogene</em><strong>34</strong>, 4570â€“4580 (2015).
</cite> [<a href="https://doi.org/10.1038/onc.2014.385" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4804637/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25500541/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bhat,%20A.%20A.%20et%20al.%20Claudin-7%20expression%20induces%20mesenchymal%20to%20epithelial%20transformation%20(MET)%20to%20inhibit%20colon%20tumorigenesis.%20Oncogene34,%204570%E2%80%934580%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR24">
<span class="label">24.</span><cite>Cox, K. E. et al. The expression of the claudin family of proteins in colorectal cancer. <em>Biomolecules</em><strong>14</strong>, 272 (2024).
</cite> [<a href="https://doi.org/10.3390/biom14030272" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10967842/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38540693/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cox,%20K.%20E.%20et%20al.%20The%20expression%20of%20the%20claudin%20family%20of%20proteins%20in%20colorectal%20cancer.%20Biomolecules14,%20272%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR25">
<span class="label">25.</span><cite>Stenvall, C.-G. A. et al. Targeted deletion of keratin 8 in intestinal epithelial cells disrupts tissue integrity and predisposes to tumorigenesis in the colon. <em>Cell. Mol. Life Sci.</em><strong>79</strong>, 10 (2022).</cite> [<a href="https://doi.org/10.1007/s00018-021-04081-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8709826/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34951664/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Stenvall,%20C.-G.%20A.%20et%20al.%20Targeted%20deletion%20of%20keratin%208%20in%20intestinal%20epithelial%20cells%20disrupts%20tissue%20integrity%20and%20predisposes%20to%20tumorigenesis%20in%20the%20colon.%20Cell.%20Mol.%20Life%20Sci.79,%2010%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR26">
<span class="label">26.</span><cite>KnÃ¶sel, T. et al. Cytokeratin profiles identify diagnostic signatures in colorectal cancer using multiplex analysis of tissue microarrays. <em>Anal. Cell. Pathol.</em><strong>28</strong>, 167â€“175 (2006).</cite> [<a href="https://doi.org/10.1155/2006/354295" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4615968/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16988472/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kn%C3%B6sel,%20T.%20et%20al.%20Cytokeratin%20profiles%20identify%20diagnostic%20signatures%20in%20colorectal%20cancer%20using%20multiplex%20analysis%20of%20tissue%20microarrays.%20Anal.%20Cell.%20Pathol.28,%20167%E2%80%93175%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR27">
<span class="label">27.</span><cite>Zhang, J., Hu, S. &amp; Li, Y. KRT18 is correlated with the malignant status and acts as an oncogene in colorectal cancer. <em>Biosci. Rep.</em><strong>39</strong>, BSR20190884 (2019).
</cite> [<a href="https://doi.org/10.1042/BSR20190884" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6692566/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31345960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Zhang,%20J.,%20Hu,%20S.%20&amp;%20Li,%20Y.%20KRT18%20is%20correlated%20with%20the%20malignant%20status%20and%20acts%20as%20an%20oncogene%20in%20colorectal%20cancer.%20Biosci.%20Rep.39,%20BSR20190884%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR28">
<span class="label">28.</span><cite>Saha, S. K., Yin, Y., Chae, H. S. &amp; Cho, S.-G. Opposing regulation of cancer properties via KRT19-mediated differential modulation of Wnt/Î²-catenin/notch signaling in breast and colon cancers. <em>Cancers</em><strong>11</strong>, 99 (2019).
</cite> [<a href="https://doi.org/10.3390/cancers11010099" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6357186/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30650643/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Saha,%20S.%20K.,%20Yin,%20Y.,%20Chae,%20H.%20S.%20&amp;%20Cho,%20S.-G.%20Opposing%20regulation%20of%20cancer%20properties%20via%20KRT19-mediated%20differential%20modulation%20of%20Wnt/%CE%B2-catenin/notch%20signaling%20in%20breast%20and%20colon%20cancers.%20Cancers11,%2099%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR29">
<span class="label">29.</span><cite>Kciuk, M., Marciniak, B. &amp; Kontek, R. Irinotecanâ€”still an important player in cancer chemotherapy: a comprehensive overview. <em>Int. J. Mol. Sci.</em><strong>21</strong>, 4919 (2020).
</cite> [<a href="https://doi.org/10.3390/ijms21144919" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7404108/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32664667/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kciuk,%20M.,%20Marciniak,%20B.%20&amp;%20Kontek,%20R.%20Irinotecan%E2%80%94still%20an%20important%20player%20in%20cancer%20chemotherapy:%20a%20comprehensive%20overview.%20Int.%20J.%20Mol.%20Sci.21,%204919%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR30">
<span class="label">30.</span><cite>Parnham, M. J. et al. Azithromycin: Mechanisms of action and their relevance for clinical applications. <em>Pharmacol. Ther.</em><strong>143</strong>, 225â€“245 (2014).
</cite> [<a href="https://doi.org/10.1016/j.pharmthera.2014.03.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24631273/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Parnham,%20M.%20J.%20et%20al.%20Azithromycin:%20Mechanisms%20of%20action%20and%20their%20relevance%20for%20clinical%20applications.%20Pharmacol.%20Ther.143,%20225%E2%80%93245%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR31">
<span class="label">31.</span><cite>Dinos, G. P. The macrolide antibiotic renaissance. <em>Br. J. Pharmacol.</em><strong>174</strong>, 2967â€“2983 (2017).
</cite> [<a href="https://doi.org/10.1111/bph.13936" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5573421/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28664582/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Dinos,%20G.%20P.%20The%20macrolide%20antibiotic%20renaissance.%20Br.%20J.%20Pharmacol.174,%202967%E2%80%932983%20(2017)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR32">
<span class="label">32.</span><cite>Housman, G. et al. Drug resistance in cancer: an overview. <em>Cancers</em><strong>6</strong>, 1769â€“1792 (2014).
</cite> [<a href="https://doi.org/10.3390/cancers6031769" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4190567/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25198391/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Housman,%20G.%20et%20al.%20Drug%20resistance%20in%20cancer:%20an%20overview.%20Cancers6,%201769%E2%80%931792%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR33">
<span class="label">33.</span><cite>Khan, S. U., Fatima, K., Aisha, S. &amp; Malik, F. Unveiling the mechanisms and challenges of cancer drug resistance. <em>Cell Commun. Signal.</em><strong>22</strong>, 109 (2024).
</cite> [<a href="https://doi.org/10.1186/s12964-023-01302-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10860306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38347575/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Khan,%20S.%20U.,%20Fatima,%20K.,%20Aisha,%20S.%20&amp;%20Malik,%20F.%20Unveiling%20the%20mechanisms%20and%20challenges%20of%20cancer%20drug%20resistance.%20Cell%20Commun.%20Signal.22,%20109%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR34">
<span class="label">34.</span><cite>Wang, Q., Shen, X., Chen, G. &amp; Du, J. Drug resistance in colorectal cancer: from mechanism to clinic. <em>Cancers</em><strong>14</strong>, 2928 (2022).
</cite> [<a href="https://doi.org/10.3390/cancers14122928" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9221177/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35740594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Wang,%20Q.,%20Shen,%20X.,%20Chen,%20G.%20&amp;%20Du,%20J.%20Drug%20resistance%20in%20colorectal%20cancer:%20from%20mechanism%20to%20clinic.%20Cancers14,%202928%20(2022)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR35">
<span class="label">35.</span><cite>Smith, B. &amp; Bhowmick, N. Role of EMT in metastasis and therapy resistance. <em>J. Clin. Med.</em><strong>5</strong>, 17 (2016).
</cite> [<a href="https://doi.org/10.3390/jcm5020017" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4773773/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26828526/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Smith,%20B.%20&amp;%20Bhowmick,%20N.%20Role%20of%20EMT%20in%20metastasis%20and%20therapy%20resistance.%20J.%20Clin.%20Med.5,%2017%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR36">
<span class="label">36.</span><cite>Du, Y. et al. Development of a miniaturized 3D organoid culture platform for ultra-high-throughput screening. <em>J. Mol. Cell Biol.</em><strong>12</strong>, 630â€“643 (2020).
</cite> [<a href="https://doi.org/10.1093/jmcb/mjaa036" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7751183/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32678871/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Du,%20Y.%20et%20al.%20Development%20of%20a%20miniaturized%203D%20organoid%20culture%20platform%20for%20ultra-high-throughput%20screening.%20J.%20Mol.%20Cell%20Biol.12,%20630%E2%80%93643%20(2020)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR37">
<span class="label">37.</span><cite>Cao, H., Xu, E., Liu, H., Wan, L. &amp; Lai, M. Epithelialâ€“mesenchymal transition in colorectal cancer metastasis: A system review. <em>Pathol. Res. Pract.</em><strong>211</strong>, 557â€“569 (2015).
</cite> [<a href="https://doi.org/10.1016/j.prp.2015.05.010" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26092594/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Cao,%20H.,%20Xu,%20E.,%20Liu,%20H.,%20Wan,%20L.%20&amp;%20Lai,%20M.%20Epithelial%E2%80%93mesenchymal%20transition%20in%20colorectal%20cancer%20metastasis:%20A%20system%20review.%20Pathol.%20Res.%20Pract.211,%20557%E2%80%93569%20(2015)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR38">
<span class="label">38.</span><cite>Nurmagambetova, A., Mustyatsa, V., Saidova, A. &amp; Vorobjev, I. Morphological and cytoskeleton changes in cells after EMT. <em>Sci. Rep.</em><strong>13</strong>, 22164 (2023).
</cite> [<a href="https://doi.org/10.1038/s41598-023-48279-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10719275/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38092761/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Nurmagambetova,%20A.,%20Mustyatsa,%20V.,%20Saidova,%20A.%20&amp;%20Vorobjev,%20I.%20Morphological%20and%20cytoskeleton%20changes%20in%20cells%20after%20EMT.%20Sci.%20Rep.13,%2022164%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR39">
<span class="label">39.</span><cite>Kelley, M. E. et al. High-content microscopy reveals a morphological signature of bortezomib resistance. <em>eLife</em><strong>12</strong>, e91362 (2023).
</cite> [<a href="https://doi.org/10.7554/eLife.91362" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10584373/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37753907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kelley,%20M.%20E.%20et%20al.%20High-content%20microscopy%20reveals%20a%20morphological%20signature%20of%20bortezomib%20resistance.%20eLife12,%20e91362%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR40">
<span class="label">40.</span><cite>Breslin, S. &amp; Oâ€™Driscoll, L. Three-dimensional cell culture: the missing link in drug discovery. <em>Drug Discov. Today</em><strong>18</strong>, 240â€“249 (2013).
</cite> [<a href="https://doi.org/10.1016/j.drudis.2012.10.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23073387/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Breslin,%20S.%20&amp;%20O%E2%80%99Driscoll,%20L.%20Three-dimensional%20cell%20culture:%20the%20missing%20link%20in%20drug%20discovery.%20Drug%20Discov.%20Today18,%20240%E2%80%93249%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR41">
<span class="label">41.</span><cite>Hait, W. N. Anticancer drug development: the grand challenges. <em>Nat. Rev. Drug Discov.</em><strong>9</strong>, 253â€“254 (2010).
</cite> [<a href="https://doi.org/10.1038/nrd3144" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20369394/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Hait,%20W.%20N.%20Anticancer%20drug%20development:%20the%20grand%20challenges.%20Nat.%20Rev.%20Drug%20Discov.9,%20253%E2%80%93254%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR42">
<span class="label">42.</span><cite>Lin, J., Zhou, D., Steitz, T. A., Polikanov, Y. S. &amp; Gagnon, M. G. Ribosome-targeting antibiotics: modes of action, mechanisms of resistance, and implications for drug design. <em>Annu. Rev. Biochem.</em><strong>87</strong>, 451â€“478 (2018).
</cite> [<a href="https://doi.org/10.1146/annurev-biochem-062917-011942" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9176271/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29570352/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Lin,%20J.,%20Zhou,%20D.,%20Steitz,%20T.%20A.,%20Polikanov,%20Y.%20S.%20&amp;%20Gagnon,%20M.%20G.%20Ribosome-targeting%20antibiotics:%20modes%20of%20action,%20mechanisms%20of%20resistance,%20and%20implications%20for%20drug%20design.%20Annu.%20Rev.%20Biochem.87,%20451%E2%80%93478%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR43">
<span class="label">43.</span><cite>Svetlov, M. S. et al. Context-specific action of macrolide antibiotics on the eukaryotic ribosome. <em>Nat. Commun.</em><strong>12</strong>, 2803 (2021).
</cite> [<a href="https://doi.org/10.1038/s41467-021-23068-1" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8121947/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33990576/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Svetlov,%20M.%20S.%20et%20al.%20Context-specific%20action%20of%20macrolide%20antibiotics%20on%20the%20eukaryotic%20ribosome.%20Nat.%20Commun.12,%202803%20(2021)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR44">
<span class="label">44.</span><cite>Leach, K. L. et al. The site of action of oxazolidinone antibiotics in living bacteria and in human mitochondria. <em>Mol. Cell</em><strong>26</strong>, 393â€“402 (2007).
</cite> [<a href="https://doi.org/10.1016/j.molcel.2007.04.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17499045/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Leach,%20K.%20L.%20et%20al.%20The%20site%20of%20action%20of%20oxazolidinone%20antibiotics%20in%20living%20bacteria%20and%20in%20human%20mitochondria.%20Mol.%20Cell26,%20393%E2%80%93402%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR45">
<span class="label">45.</span><cite>BÃ¶ttger, E. C., Springer, B., Prammananan, T., Kidan, Y. &amp; Sander, P. Structural basis for selectivity and toxicity of ribosomal antibiotics. <em>EMBO Rep.</em><strong>2</strong>, 318â€“323 (2001).
</cite> [<a href="https://doi.org/10.1093/embo-reports/kve062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1083859/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11306553/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?B%C3%B6ttger,%20E.%20C.,%20Springer,%20B.,%20Prammananan,%20T.,%20Kidan,%20Y.%20&amp;%20Sander,%20P.%20Structural%20basis%20for%20selectivity%20and%20toxicity%20of%20ribosomal%20antibiotics.%20EMBO%20Rep.2,%20318%E2%80%93323%20(2001)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR46">
<span class="label">46.</span><cite>Lamb, R. et al. Antibiotics that target mitochondria effectively eradicate cancer stem cells, across multiple tumor types. treating cancer like an infectious disease. <em>Oncotarget</em><strong>6</strong>, 4569âˆ’84 (2015).</cite> [<a href="https://doi.org/10.18632/oncotarget.3174" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4467100/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25625193/" class="usa-link">PubMed</a>]</li>
<li id="CR47">
<span class="label">47.</span><cite>Kalghatgi, S. et al. Bactericidal antibiotics induce mitochondrial dysfunction and oxidative damage in mammalian cells. <em>Sci. Transl. Med</em>. <strong>5</strong>, 192ra85 (2013).</cite> [<a href="https://doi.org/10.1126/scitranslmed.3006055" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3760005/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23825301/" class="usa-link">PubMed</a>]</li>
<li id="CR48">
<span class="label">48.</span><cite>Mathur, S. et al. Regulation of metastatic potential by drug repurposing and mitochondrial targeting in colorectal cancer cells. <em>BMC Cancer</em><strong>24</strong>, 323 (2024).
</cite> [<a href="https://doi.org/10.1186/s12885-024-12064-5" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10921801/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38459456/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mathur,%20S.%20et%20al.%20Regulation%20of%20metastatic%20potential%20by%20drug%20repurposing%20and%20mitochondrial%20targeting%20in%20colorectal%20cancer%20cells.%20BMC%20Cancer24,%20323%20(2024)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR49">
<span class="label">49.</span><cite>Duewelhenke, N., Krut, O. &amp; Eysel, P. Influence on mitochondria and cytotoxicity of different antibiotics administered in high concentrations on primary human osteoblasts and cell lines. <em>Antimicrob. Agents Chemother.</em><strong>51</strong>, 54â€“63 (2007).
</cite> [<a href="https://doi.org/10.1128/AAC.00729-05" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1797653/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/17088489/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Duewelhenke,%20N.,%20Krut,%20O.%20&amp;%20Eysel,%20P.%20Influence%20on%20mitochondria%20and%20cytotoxicity%20of%20different%20antibiotics%20administered%20in%20high%20concentrations%20on%20primary%20human%20osteoblasts%20and%20cell%20lines.%20Antimicrob.%20Agents%20Chemother.51,%2054%E2%80%9363%20(2007)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR50">
<span class="label">50.</span><cite>Kang, H., Kim, H., Lee, S., Youn, H. &amp; Youn, B. Role of metabolic reprogramming in epithelialâ€“mesenchymal transition (EMT). <em>Int. J. Mol. Sci.</em><strong>20</strong>, 2042 (2019).
</cite> [<a href="https://doi.org/10.3390/ijms20082042" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6514888/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31027222/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kang,%20H.,%20Kim,%20H.,%20Lee,%20S.,%20Youn,%20H.%20&amp;%20Youn,%20B.%20Role%20of%20metabolic%20reprogramming%20in%20epithelial%E2%80%93mesenchymal%20transition%20(EMT).%20Int.%20J.%20Mol.%20Sci.20,%202042%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR51">
<span class="label">51.</span><cite>Bhattacharya, D. &amp; ScimÃ¨, A. Metabolic regulation of epithelial to mesenchymal transition: implications for endocrine cancer. <em>Front. Endocrinol.</em><strong>10</strong>, 773 (2019).</cite> [<a href="https://doi.org/10.3389/fendo.2019.00773" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6901924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31849832/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Bhattacharya,%20D.%20&amp;%20Scim%C3%A8,%20A.%20Metabolic%20regulation%20of%20epithelial%20to%20mesenchymal%20transition:%20implications%20for%20endocrine%20cancer.%20Front.%20Endocrinol.10,%20773%20(2019)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR52">
<span class="label">52.</span><cite>Kagkelaris, K. A., Makri, O. E., Georgakopoulos, C. D. &amp; Panayiotakopoulos, G. D. An eye for azithromycin: review of the literature. <em>Ther. Adv. Ophthalmol.</em><strong>10</strong>, 2515841418783622 (2018).
</cite> [<a href="https://doi.org/10.1177/2515841418783622" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6066808/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30083656/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Kagkelaris,%20K.%20A.,%20Makri,%20O.%20E.,%20Georgakopoulos,%20C.%20D.%20&amp;%20Panayiotakopoulos,%20G.%20D.%20An%20eye%20for%20azithromycin:%20review%20of%20the%20literature.%20Ther.%20Adv.%20Ophthalmol.10,%202515841418783622%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR53">
<span class="label">53.</span><cite>Halldorsson, S. et al. Azithromycin maintains airway epithelial integrity during pseudomonas aeruginosa infection. <em>Am. J. Respir. Cell Mol. Biol.</em><strong>42</strong>, 62â€“68 (2010).
</cite> [<a href="https://doi.org/10.1165/rcmb.2008-0357OC" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19372247/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Halldorsson,%20S.%20et%20al.%20Azithromycin%20maintains%20airway%20epithelial%20integrity%20during%20pseudomonas%20aeruginosa%20infection.%20Am.%20J.%20Respir.%20Cell%20Mol.%20Biol.42,%2062%E2%80%9368%20(2010)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR54">
<span class="label">54.</span><cite>Asgrimsson, V., Gudjonsson, T., Gudmundsson, G. H. &amp; Baldursson, O. Novel effects of azithromycin on tight junction proteins in human airway epithelia. <em>Antimicrob. Agents Chemother.</em><strong>50</strong>, 1805â€“1812 (2006).
</cite> [<a href="https://doi.org/10.1128/AAC.50.5.1805-1812.2006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1472195/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16641453/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Asgrimsson,%20V.,%20Gudjonsson,%20T.,%20Gudmundsson,%20G.%20H.%20&amp;%20Baldursson,%20O.%20Novel%20effects%20of%20azithromycin%20on%20tight%20junction%20proteins%20in%20human%20airway%20epithelia.%20Antimicrob.%20Agents%20Chemother.50,%201805%E2%80%931812%20(2006)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR55">
<span class="label">55.</span><cite>Mukai, S. et al. Macrolides sensitize EGFR-TKI-induced non-apoptotic cell death via blocking autophagy flux in pancreatic cancer cell lines. <em>Int. J. Oncol.</em><strong>48</strong>, 45â€“54 (2016).
</cite> [<a href="https://doi.org/10.3892/ijo.2015.3237" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4734605/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26718641/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Mukai,%20S.%20et%20al.%20Macrolides%20sensitize%20EGFR-TKI-induced%20non-apoptotic%20cell%20death%20via%20blocking%20autophagy%20flux%20in%20pancreatic%20cancer%20cell%20lines.%20Int.%20J.%20Oncol.48,%2045%E2%80%9354%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR56">
<span class="label">56.</span><cite>Takano, N. et al. Azithromycin, a potent autophagy inhibitor for cancer therapy, perturbs cytoskeletal protein dynamics. <em>Br. J. Cancer</em><strong>128</strong>, 1838â€“1849 (2023).
</cite> [<a href="https://doi.org/10.1038/s41416-023-02210-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10147625/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36871041/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Takano,%20N.%20et%20al.%20Azithromycin,%20a%20potent%20autophagy%20inhibitor%20for%20cancer%20therapy,%20perturbs%20cytoskeletal%20protein%20dynamics.%20Br.%20J.%20Cancer128,%201838%E2%80%931849%20(2023)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR57">
<span class="label">57.</span><cite>Moriya, S. et al. Macrolide antibiotics block autophagy flux and sensitize to bortezomib via endoplasmic reticulum stress-mediated CHOP induction in myeloma cells. <em>Int. J. Oncol.</em><strong>42</strong>, 1541â€“1550 (2013).
</cite> [<a href="https://doi.org/10.3892/ijo.2013.1870" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3661227/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23546223/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Moriya,%20S.%20et%20al.%20Macrolide%20antibiotics%20block%20autophagy%20flux%20and%20sensitize%20to%20bortezomib%20via%20endoplasmic%20reticulum%20stress-mediated%20CHOP%20induction%20in%20myeloma%20cells.%20Int.%20J.%20Oncol.42,%201541%E2%80%931550%20(2013)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR58">
<span class="label">58.</span><cite>Qiao, X., Wang, X., Shang, Y., Li, Y. &amp; Chen, S. Azithromycin enhances anticancer activity of TRAIL by inhibiting autophagy and upâ€regulating the protein levels of DR4/5 in colon cancer cells in vitro and in vivo. <em>Cancer Commun.</em><strong>38</strong>, 1â€“13 (2018).</cite> [<a href="https://doi.org/10.1186/s40880-018-0309-9" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6029027/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29970185/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Qiao,%20X.,%20Wang,%20X.,%20Shang,%20Y.,%20Li,%20Y.%20&amp;%20Chen,%20S.%20Azithromycin%20enhances%20anticancer%20activity%20of%20TRAIL%20by%20inhibiting%20autophagy%20and%20up%E2%80%90regulating%20the%20protein%20levels%20of%20DR4/5%20in%20colon%20cancer%20cells%20in%20vitro%20and%20in%20vivo.%20Cancer%20Commun.38,%201%E2%80%9313%20(2018)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR59">
<span class="label">59.</span><cite>Pedregosa, F. et al. Scikit-learn: machine learning in python. <em>J. Mach. Learn. Res.</em><strong>12</strong>, 2825â€“2830 (2011).</cite> [<a href="https://scholar.google.com/scholar_lookup?Pedregosa,%20F.%20et%20al.%20Scikit-learn:%20machine%20learning%20in%20python.%20J.%20Mach.%20Learn.%20Res.12,%202825%E2%80%932830%20(2011)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR60">
<span class="label">60.</span><cite>R Core Team. <em>R: A Language and Environment for Statistical Computing</em>. <a href="https://www.r-project.org/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.r-project.org/</a> (2021).</cite>
</li>
<li id="CR61">
<span class="label">61.</span><cite>Legland, D., Arganda-Carreras, I. &amp; Andrey, P. MorphoLibJ: Integrated library and plugins for mathematical morphology with ImageJ. <em>Bioinformatics</em><strong>32</strong>, 3532â€“3534 (2016).
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btw413" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27412086/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Legland,%20D.,%20Arganda-Carreras,%20I.%20&amp;%20Andrey,%20P.%20MorphoLibJ:%20Integrated%20library%20and%20plugins%20for%20mathematical%20morphology%20with%20ImageJ.%20Bioinformatics32,%203532%E2%80%933534%20(2016)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR62">
<span class="label">62.</span><cite>Roden, D. et al. Development of a large-scale de-identified DNA biobank to enable personalized medicine. <em>Clin. Pharmacol. Ther.</em><strong>84</strong>, 362â€“369 (2008).
</cite> [<a href="https://doi.org/10.1038/clpt.2008.89" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3763939/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/18500243/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Roden,%20D.%20et%20al.%20Development%20of%20a%20large-scale%20de-identified%20DNA%20biobank%20to%20enable%20personalized%20medicine.%20Clin.%20Pharmacol.%20Ther.84,%20362%E2%80%93369%20(2008)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR63">
<span class="label">63.</span><cite>Harris, A. L. Hypoxiaâ€”a key regulatory factor in tumour growth. <em>Nat. Rev. Cancer</em><strong>2</strong>, 38â€“47 (2002).
</cite> [<a href="https://doi.org/10.1038/nrc704" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11902584/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Harris,%20A.%20L.%20Hypoxia%E2%80%94a%20key%20regulatory%20factor%20in%20tumour%20growth.%20Nat.%20Rev.%20Cancer2,%2038%E2%80%9347%20(2002)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="CR64">
<span class="label">64.</span><cite>Danciu, I. et al. Secondary use of clinical data: the vanderbilt approach. <em>J. Biomed. Inform.</em><strong>52</strong>, 28â€“35 (2014).
</cite> [<a href="https://doi.org/10.1016/j.jbi.2014.02.003" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4133331/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24534443/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?Danciu,%20I.%20et%20al.%20Secondary%20use%20of%20clinical%20data:%20the%20vanderbilt%20approach.%20J.%20Biomed.%20Inform.52,%2028%E2%80%9335%20(2014)." class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ul></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373842/bin/42003_2025_8699_MOESM1_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Transparent Peer Review file</a><sup> (218.2KB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373842/bin/42003_2025_8699_MOESM2_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">Supplemental Material</a><sup> (82.2MB, pdf) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373842/bin/42003_2025_8699_MOESM3_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">42003_2025_8699_MOESM3_ESM.pdf</a><sup> (45.2KB, pdf) </sup><p>Description of Additional Supplementary Files</p>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373842/bin/42003_2025_8699_MOESM4_ESM.xlsx" data-ga-action="click_feat_suppl" class="usa-link">Supplemental Data 1</a><sup> (2.1MB, xlsx) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="media p"><div class="caption">
<a href="/articles/instance/12373842/bin/42003_2025_8699_MOESM5_ESM.pdf" data-ga-action="click_feat_suppl" class="usa-link">NR Reporting Summary</a><sup> (2MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Data to generate graphs and additional data from the screen available in Supplementary DataÂ <a href="#MOESM4" class="usa-link">1</a>. Images generated in high-throughput screen are available in associated repository<sup><a href="#CR13" class="usa-link" aria-describedby="CR13">13</a></sup>. RNA-seq data has been deposited into the NCBI GEO under the accession number <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE304081" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">GSE304081</a>. Any additional information related to this paper is available from the corresponding author upon reasonable request.</p>
<p>This study did not generate unique reagents.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Communications Biology are provided here courtesy of <strong>Nature Publishing Group</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1038/s42003-025-08699-0"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/42003_2025_Article_8699.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.3Â MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12373842/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12373842/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12373842%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12373842/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12373842/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12373842/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40847193/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12373842/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40847193/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12373842/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12373842/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="lG1FnFisIpQWtN1a72ulkn91eZzYKCaUzboMziYAzlX68XB4tNW4k63YRaMbnq03">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
        <script type="module" crossorigin="" src="/static/assets/math-574fdcc6.js"></script>
    
    

    </body>
</html>
