
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Genomic Landscape of High‐Altitude Adaptation in East African Mountain Honey Bees ( Apis mellifera ) - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="D2E4AFC18AF3244305AFC100324D5D53.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="ece">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367272/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="Ecology and Evolution">
<meta name="citation_title" content="Genomic Landscape of High‐Altitude Adaptation in East African Mountain Honey Bees ( Apis mellifera )">
<meta name="citation_author" content="Marco Mazzoni">
<meta name="citation_author_institution" content="Department of Livestock Population Genomics, Institute of Animal Science, University of Hohenheim, Stuttgart, Germany">
<meta name="citation_author" content="Florian Loidolt">
<meta name="citation_author_institution" content="Behavioural Physiology and Sociobiology, Biocenter, Universität Würzburg, Würzburg, Germany">
<meta name="citation_author" content="Sonja Kersten">
<meta name="citation_author_institution" content="Department of Livestock Population Genomics, Institute of Animal Science, University of Hohenheim, Stuttgart, Germany">
<meta name="citation_author" content="Deborah Ruth Amulen">
<meta name="citation_author_institution" content="The College of Veterinary Medicine Animal Resources and Bio‐security (COVAB), Makerere University, Kampala, Uganda">
<meta name="citation_author" content="Patrick Vudriko">
<meta name="citation_author_institution" content="The College of Veterinary Medicine Animal Resources and Bio‐security (COVAB), Makerere University, Kampala, Uganda">
<meta name="citation_author" content="Philipp Meyer">
<meta name="citation_author_institution" content="Institute for Integrative Nature Conservation Research, University of Natural Resources and Life Sciences Vienna, Vienna, Austria">
<meta name="citation_author" content="Victor Sebastian Scharnhorst">
<meta name="citation_author_institution" content="Institute for Integrative Nature Conservation Research, University of Natural Resources and Life Sciences Vienna, Vienna, Austria">
<meta name="citation_author" content="Ricarda Scheiner">
<meta name="citation_author_institution" content="Behavioural Physiology and Sociobiology, Biocenter, Universität Würzburg, Würzburg, Germany">
<meta name="citation_author" content="Martin Hasselmann">
<meta name="citation_author_institution" content="Department of Livestock Population Genomics, Institute of Animal Science, University of Hohenheim, Stuttgart, Germany">
<meta name="citation_author_institution" content="KomBioTa‐Center for Biodiversity and Integrative Taxonomy, University of Hohenheim and State Museum of Natural History, Stuttgart, Germany">
<meta name="citation_publication_date" content="2025 Aug 20">
<meta name="citation_volume" content="15">
<meta name="citation_issue" content="8">
<meta name="citation_firstpage" content="e71846">
<meta name="citation_doi" content="10.1002/ece3.71846">
<meta name="citation_pmid" content="40843029">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367272/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367272/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367272/pdf/ECE3-15-e71846.pdf">
<meta name="description" content="Understanding the evolutionary processes leading to differentiation within species is a central goal in population biology. A key process is local adaptation, for which organisms evolve traits enhancing the survival and reproduction in specific ...">
<meta name="og:title" content="Genomic Landscape of High‐Altitude Adaptation in East African Mountain Honey Bees ( Apis mellifera )">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="Understanding the evolutionary processes leading to differentiation within species is a central goal in population biology. A key process is local adaptation, for which organisms evolve traits enhancing the survival and reproduction in specific ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367272/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12367272">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1002/ece3.71846"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/ECE3-15-e71846.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367272%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12367272/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12367272/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367272/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-ece.gif" alt="Ecology and Evolution logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to Ecology and Evolution" title="Link to Ecology and Evolution" shape="default" href="http://onlinelibrary.wiley.com/journal/10.1002/(ISSN)2045-7758" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">Ecol Evol</button></div>. 2025 Aug 20;15(8):e71846. doi: <a href="https://doi.org/10.1002/ece3.71846" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1002/ece3.71846</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22Ecol%20Evol%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Ecol%20Evol%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22Ecol%20Evol%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22Ecol%20Evol%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Genomic Landscape of High‐Altitude Adaptation in East African Mountain Honey Bees (
<em>Apis mellifera</em>
)</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mazzoni%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Marco Mazzoni</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Marco Mazzoni</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Livestock Population Genomics, Institute of Animal Science, University of Hohenheim, Stuttgart, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Mazzoni%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Marco Mazzoni</span></a>
</div>
</div>
<sup>1,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Loidolt%20F%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Florian Loidolt</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Florian Loidolt</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Behavioural Physiology and Sociobiology, Biocenter, Universität Würzburg, Würzburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Loidolt%20F%22%5BAuthor%5D" class="usa-link"><span class="name western">Florian Loidolt</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kersten%20S%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Sonja Kersten</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Sonja Kersten</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Livestock Population Genomics, Institute of Animal Science, University of Hohenheim, Stuttgart, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Kersten%20S%22%5BAuthor%5D" class="usa-link"><span class="name western">Sonja Kersten</span></a>
</div>
</div>
<sup>1</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Amulen%20DR%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Deborah Ruth Amulen</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Deborah Ruth Amulen</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>The College of Veterinary Medicine Animal Resources and Bio‐security (COVAB), Makerere University, Kampala, Uganda</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Amulen%20DR%22%5BAuthor%5D" class="usa-link"><span class="name western">Deborah Ruth Amulen</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vudriko%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Patrick Vudriko</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Patrick Vudriko</span></h3>
<div class="p">
<sup>
<sup>3</sup>
</sup>The College of Veterinary Medicine Animal Resources and Bio‐security (COVAB), Makerere University, Kampala, Uganda</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Vudriko%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Patrick Vudriko</span></a>
</div>
</div>
<sup>3</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meyer%20P%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Philipp Meyer</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Philipp Meyer</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Institute for Integrative Nature Conservation Research, University of Natural Resources and Life Sciences Vienna, Vienna, Austria</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Meyer%20P%22%5BAuthor%5D" class="usa-link"><span class="name western">Philipp Meyer</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Scharnhorst%20VS%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Victor Sebastian Scharnhorst</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Victor Sebastian Scharnhorst</span></h3>
<div class="p">
<sup>
<sup>4</sup>
</sup>Institute for Integrative Nature Conservation Research, University of Natural Resources and Life Sciences Vienna, Vienna, Austria</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Scharnhorst%20VS%22%5BAuthor%5D" class="usa-link"><span class="name western">Victor Sebastian Scharnhorst</span></a>
</div>
</div>
<sup>4</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Scheiner%20R%22%5BAuthor%5D" class="usa-link" aria-describedby="id8"><span class="name western">Ricarda Scheiner</span></a><div hidden="hidden" id="id8">
<h3><span class="name western">Ricarda Scheiner</span></h3>
<div class="p">
<sup>
<sup>2</sup>
</sup>Behavioural Physiology and Sociobiology, Biocenter, Universität Würzburg, Würzburg, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Scheiner%20R%22%5BAuthor%5D" class="usa-link"><span class="name western">Ricarda Scheiner</span></a>
</div>
</div>
<sup>2</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hasselmann%20M%22%5BAuthor%5D" class="usa-link" aria-describedby="id9"><span class="name western">Martin Hasselmann</span></a><div hidden="hidden" id="id9">
<h3><span class="name western">Martin Hasselmann</span></h3>
<div class="p">
<sup>
<sup>1</sup>
</sup>Department of Livestock Population Genomics, Institute of Animal Science, University of Hohenheim, Stuttgart, Germany</div>
<div class="p">
<sup>
<sup>5</sup>
</sup>KomBioTa‐Center for Biodiversity and Integrative Taxonomy, University of Hohenheim and State Museum of Natural History, Stuttgart, Germany</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Hasselmann%20M%22%5BAuthor%5D" class="usa-link"><span class="name western">Martin Hasselmann</span></a>
</div>
</div>
<sup>1,</sup><sup>5</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="ece371846-aff-0001">
<sup>
<sup>1</sup>
</sup>Department of Livestock Population Genomics, Institute of Animal Science, University of Hohenheim, Stuttgart, Germany</div>
<div id="ece371846-aff-0002">
<sup>
<sup>2</sup>
</sup>Behavioural Physiology and Sociobiology, Biocenter, Universität Würzburg, Würzburg, Germany</div>
<div id="ece371846-aff-0003">
<sup>
<sup>3</sup>
</sup>The College of Veterinary Medicine Animal Resources and Bio‐security (COVAB), Makerere University, Kampala, Uganda</div>
<div id="ece371846-aff-0004">
<sup>
<sup>4</sup>
</sup>Institute for Integrative Nature Conservation Research, University of Natural Resources and Life Sciences Vienna, Vienna, Austria</div>
<div id="ece371846-aff-0005">
<sup>
<sup>5</sup>
</sup>KomBioTa‐Center for Biodiversity and Integrative Taxonomy, University of Hohenheim and State Museum of Natural History, Stuttgart, Germany</div>
<div class="author-notes p">
<div class="fn" id="correspondenceTo">
<sup>*</sup><p class="display-inline">

<strong>Correspondence:</strong>
 
Marco Mazzoni (<span>marco.mazzoni@uni-hohenheim.de</span>)
</p>
</div>
<div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div>
</div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jun 30; Accepted 2025 Jul 14; Collection date 2025 Aug.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>© 2025 The Author(s). <em>Ecology and Evolution</em> published by British Ecological Society and John Wiley &amp; Sons Ltd.</div>
<p>This is an open access article under the terms of the <a href="https://creativecommons.org/licenses/by/4.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">http://creativecommons.org/licenses/by/4.0/</a> License, which permits use, distribution and reproduction in any medium, provided the original work is properly cited.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12367272  PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/40843029/" class="usa-link">40843029</a>
</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>ABSTRACT</h2>
<p>Understanding the evolutionary processes leading to differentiation within species is a central goal in population biology. A key process is local adaptation, for which organisms evolve traits enhancing the survival and reproduction in specific environments. Honey bees (
<em>Apis mellifera</em>
) in East Africa are well adapted to highland environments, showing different phenotypes, including behavior, compared to lowland bees. Despite these differences, highland and lowland honey bees show very low genetic differentiation, with the exception of two segments on chromosome 7 (r7) and chromosome 9 (r9), which were previously identified as chromosomal inversions. These inversions are rare in lowland populations, suggesting a key role in adaptation to high‐elevation habitats. In this study, we obtained 24 whole genomes from honey bees of Western Uganda and compared these with existing data from Kenya. We show that the chromosomal inversions play a pivotal role in local adaptation in both regions but with substantial differentiation. Genome‐wide analysis of polymorphism revealed additional genomic regions potentially involved in high‐altitude adaptation. The acquisition of transcriptome data from highland and lowland honey bees in Uganda has enabled the first insights into the differential expression of genes between these bees. Our findings elucidate the involvement of genes in behavioral and oxygen consumption processes. This paves the way to clarify the interplay of r7 and r9 with gene expression and to unravel the regulatory network underlying 
<em>A. mellifera</em>
 adaptation to high‐elevation habitats. Our study will contribute to a better understanding of the evolutionary processes in honey bee populations driven by environmental conditions.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p>Highland and lowland honey bees in East Africa exhibit distinct phenotypes despite low overall genetic differentiation. To investigate this further, we sampled honey bees from a mountain system in Uganda (red circle) and compared them with samples from Kenya. Our analyses confirmed that two chromosomal inversions (r7 and r9) were strongly associated with high‐altitude adaptation. Additionally, genome and transcriptome data revealed genes involved in behaviour and oxygen consumption, highlighting key genetic and regulatory mechanisms underlying local adaptation in <em>Apis mellifera</em>.</p>
<section class="bt xbox font-sm" id="ece371846-blkfxd-0001"><p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dbfb/12367272/70091f8425f6/ECE3-15-e71846-g005.jpg" loading="lazy" id="jats-graphic-1" height="188" width="331" alt="graphic file with name ECE3-15-e71846-g005.jpg"></p></section></section><section id="ece371846-sec-0001"><h2 class="pmc_sec_title">1. Introduction</h2>
<p>Understanding the molecular mechanism that leads to the adaptation of organisms to different environments is a longstanding goal in evolutionary genetics (Goncearenco et al. <a href="#ece371846-bib-0040" class="usa-link" aria-describedby="ece371846-bib-0040">2013</a>; Bomblies and Peichel <a href="#ece371846-bib-0015" class="usa-link" aria-describedby="ece371846-bib-0015">2022</a>; Martínez Sosa and Pilot <a href="#ece371846-bib-0064" class="usa-link" aria-describedby="ece371846-bib-0064">2023</a>; Byers et al. <a href="#ece371846-bib-0019" class="usa-link" aria-describedby="ece371846-bib-0019">2016</a>). In light of ongoing climate change and decline of biodiversity (Franks and Hoffmann <a href="#ece371846-bib-0033" class="usa-link" aria-describedby="ece371846-bib-0033">2012</a>; Mahecha et al. <a href="#ece371846-bib-0062" class="usa-link" aria-describedby="ece371846-bib-0062">2022</a>), this knowledge is further valuable for the conservation of wild populations (Waldvogel et al. <a href="#ece371846-bib-0095" class="usa-link" aria-describedby="ece371846-bib-0095">2020</a>; Webster et al. <a href="#ece371846-bib-0099" class="usa-link" aria-describedby="ece371846-bib-0099">2023</a>). Adapting to different environments present numerous challenges for the organism (Matur et al. <a href="#ece371846-bib-0065" class="usa-link" aria-describedby="ece371846-bib-0065">2023</a>). High mountain environments impose various stressors that can interfere with insect physiology, such as temperature, air density, and oxygen partial pressure (Dillon <a href="#ece371846-bib-0027" class="usa-link" aria-describedby="ece371846-bib-0027">2006</a>). For example, it has been demonstrated that these stresses have the capacity to reduce metabolic rates and increase hatching times in 
<em>Manduca sexta</em>
 while also increasing the developmental time and reducing body size in 
<em>Tenebrio molitor</em>
 and 
<em>Drosophila melanogaster</em>
 (Greenberg and Ar <a href="#ece371846-bib-0041" class="usa-link" aria-describedby="ece371846-bib-0041">1996</a>; Woods and Hill <a href="#ece371846-bib-0101" class="usa-link" aria-describedby="ece371846-bib-0101">2004</a>; Dillon <a href="#ece371846-bib-0027" class="usa-link" aria-describedby="ece371846-bib-0027">2006</a>). Although low temperatures relative to low altitudes may pose risks, evidence suggests that insects exposed to low oxygen levels prefer colder rather than warmer environments (Frazier et al. <a href="#ece371846-bib-0034" class="usa-link" aria-describedby="ece371846-bib-0034">2001</a>).</p>
<p>Interestingly, studies in human populations residing in distinct high‐altitude regions have revealed a disparate pattern of selected genes, indicating that the process of adaptation can occur independently in each population (Scheinfeldt and Tishkoff <a href="#ece371846-bib-0084" class="usa-link" aria-describedby="ece371846-bib-0084">2010</a>; Bigham et al. <a href="#ece371846-bib-0011" class="usa-link" aria-describedby="ece371846-bib-0011">2010</a>; Bigham <a href="#ece371846-bib-0012" class="usa-link" aria-describedby="ece371846-bib-0012">2016</a>).</p>
<p>Major implications of structural variants (SVs) have been identified in various adaptation processes. SVs can have an effect outside the boundaries of the SV, and this effect can influence gene expression, metabolic pathways, and finally speciation. Studies in <em>Drosophila</em> showed that the recombination in inversion polymorphisms can be extended even outside the inverted region (Stevison et al. <a href="#ece371846-bib-0089" class="usa-link" aria-describedby="ece371846-bib-0089">2011</a>). Normally, linkage disequilibrium between two SNPs decays proportionally to the distance between the two; however, SVs, in particular inversions, disrupt this pattern, leading to extensive LD (Fuller et al. <a href="#ece371846-bib-0035" class="usa-link" aria-describedby="ece371846-bib-0035">2017</a>). Inversions are a portion of DNA that is reversed end‐to‐end; they do not cause a loss of genetic material; however, they disrupt homologous pairing during meiosis. Crossing‐over events in heterozygous inversions can lead to the formation of abnormal chromatids. These are generally removed during gametogenesis, thus leading to the independent accumulation of genetic differences due to a lack of recombination. However, in some cases, double crossing‐over or gene conversion events could homogenize, at least in part, the genetic differences. This effect is observed only in the case of SNPs not strongly associated with divergent selection between populations (Feder and Nosil <a href="#ece371846-bib-0032" class="usa-link" aria-describedby="ece371846-bib-0032">2009</a>; Korunes and Noor <a href="#ece371846-bib-0054" class="usa-link" aria-describedby="ece371846-bib-0054">2018</a>). Finally, SVs can alter gene expression mainly through the disruption of genes at their breakpoints (Puig et al. <a href="#ece371846-bib-0071" class="usa-link" aria-describedby="ece371846-bib-0071">2004</a>). There is substantial evidence that both small SVs, such as insertion and deletion, and large SVs, inversion and Copy Number Variants (CNVs), play a role in evolutionary and speciation processes (Schaeffer et al. <a href="#ece371846-bib-0081" class="usa-link" aria-describedby="ece371846-bib-0081">2003</a>; Lin et al. <a href="#ece371846-bib-0060" class="usa-link" aria-describedby="ece371846-bib-0060">2017</a>; Stalder et al. <a href="#ece371846-bib-0088" class="usa-link" aria-describedby="ece371846-bib-0088">2022</a>). Indels in coding regions could cause a frameshift leading to changes in protein sequence and possibly lossoffunction of the protein (Lin et al. <a href="#ece371846-bib-0060" class="usa-link" aria-describedby="ece371846-bib-0060">2017</a>). However, if the gene is not essential for the organism's fitness, it will just contribute to the phenotypic variability of the population. Inversion polymorphisms, particularly large ones, can capture different adaptive alleles and limit recombination between orientations, which can facilitate the fixation of these alleles in one or the other orientation. In some instances, such inversions may function as supergenes or superalleles and have been implicated in shaping the genetic basis of certain complex traits (Berdan et al. <a href="#ece371846-bib-0009" class="usa-link" aria-describedby="ece371846-bib-0009">2022</a>; Gadau and Fewell <a href="#ece371846-bib-0037" class="usa-link" aria-describedby="ece371846-bib-0037">2022</a>). Studies on inversion polymorphisms, which rely mainly on <em>Drosophila</em> sp., observed that inversion polymorphisms are associated with heat resistance, cold tolerance, and body size (Weeks et al. <a href="#ece371846-bib-0100" class="usa-link" aria-describedby="ece371846-bib-0100">2002</a>; Bochdanovits and Jong <a href="#ece371846-bib-0013" class="usa-link" aria-describedby="ece371846-bib-0013">2003</a>; Anderson et al. <a href="#ece371846-bib-0002" class="usa-link" aria-describedby="ece371846-bib-0002">2005</a>). However, these studies also highlight that not every inversion should be considered as a superallele (Schaeffer et al. <a href="#ece371846-bib-0081" class="usa-link" aria-describedby="ece371846-bib-0081">2003</a>).</p>
<p>In light of the continuous advancement and refinement of genome sequencing techniques, a growing body of evidence has emerged demonstrating the role of SVs in a variety of adaptive processes across a multitude of species. For example, in three‐spined sticklebacks (
<em>Gasterosteus aculeatus</em>
), inversion polymorphisms have facilitated colonization of both freshwater and marine ecosystems (Jones et al. <a href="#ece371846-bib-0049" class="usa-link" aria-describedby="ece371846-bib-0049">2012</a>; Reid et al. <a href="#ece371846-bib-0073" class="usa-link" aria-describedby="ece371846-bib-0073">2021</a>). In the yellow monkeyflower (
<em>Mimulus guttatus</em>
) an inversion polymorphism affected adaptive flowering time (Lowry and Willis <a href="#ece371846-bib-0501" class="usa-link" aria-describedby="ece371846-bib-0501">2010</a>). Additionally, in deer mice (
<em>Peromyscus maniculatus</em>
), 13 inversions have been identified that contribute to differentiation between forest and prairie ecotypes, influencing traits related to local adaptation (Harringmeyer and Hoekstra <a href="#ece371846-bib-0044" class="usa-link" aria-describedby="ece371846-bib-0044">2022</a>).</p>
<p>

<em>Apis mellifera</em>
, the western honey bee, is well adapted to a wide range of habitats and environments. It originated in Asia, where it diverged from its sister taxa, 
<em>Apis cerana</em>
. From there, it colonized the Middle East, Africa, and Europe through various expansion events (Dogantzis et al. <a href="#ece371846-bib-0028" class="usa-link" aria-describedby="ece371846-bib-0028">2021</a>). Each newly colonized environment poses threats to colony survival, making worker traits crucial for adaptation. Therefore, these traits often undergo positive selection in new areas (Harpur et al. <a href="#ece371846-bib-0043" class="usa-link" aria-describedby="ece371846-bib-0043">2014</a>; Dogantzis et al. <a href="#ece371846-bib-0028" class="usa-link" aria-describedby="ece371846-bib-0028">2021</a>). Honey bees maintain high genetic diversity, with more than 30 subspecies described, currently grouped into seven distinct lineages (Dogantzis et al. <a href="#ece371846-bib-0028" class="usa-link" aria-describedby="ece371846-bib-0028">2021</a>). Among these, predominantly African honey bees in the A‐lineage represent the highest genetic diversity. In East Africa, two subspecies are identified around mountain peaks, 
<em>Apis mellifera scutellata</em>
 (<em>A. m. scutellata</em>) and 
<em>Apis mellifera monticola</em>
 (<em>A. m. monticola</em>; Ruttner <a href="#ece371846-bib-0079" class="usa-link" aria-describedby="ece371846-bib-0079">1998</a>). <em>A. m. scutellata</em> inhabits the lowland savannah, while <em>A. m. monticola</em> inhabits the highland mountain forest environment (Smith <a href="#ece371846-bib-0087" class="usa-link" aria-describedby="ece371846-bib-0087">1961</a>). Both environments pose severe threats to honey bees: the savannah is hot with scarce food sources, while the mountain forest is rainy, cold, and high‐altitude, causing potential hypoxia due to lower barometric pressure. The two subspecies exhibit differences in behavior and phenotype; <em>A. m. scutellata</em> is more aggressive, smaller, and brighter in color, whereas <em>A. m. monticola</em> is less aggressive, larger, and darker (Smith <a href="#ece371846-bib-0087" class="usa-link" aria-describedby="ece371846-bib-0087">1961</a>; Ruttner <a href="#ece371846-bib-0079" class="usa-link" aria-describedby="ece371846-bib-0079">1998</a>; Gruber et al. <a href="#ece371846-bib-0042" class="usa-link" aria-describedby="ece371846-bib-0042">2013</a>).</p>
<p>Honey bee populations inhabiting East African regions are well adapted to both highland montane areas and lowland savannahs, with two inversion polymorphisms on chromosome 7 (named <a href="https://www.ncbi.nlm.nih.gov/nuccore/CM009937.2" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CM009937.2</a> in the latest 
<em>Apis mellifera</em>
 reference genome, Accession: GCA_003254395) and 9 (named <a href="https://www.ncbi.nlm.nih.gov/nuccore/CM009939.2" class="usa-link" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">CM009939.2</a> in the latest 
<em>Apis mellifera</em>
 reference genome, Accession: GCA_003254395), named r7 and r9 respectively, which are found at high frequency in the highland population (Wallberg et al. <a href="#ece371846-bib-0097" class="usa-link" aria-describedby="ece371846-bib-0097">2017</a>; Christmas et al. <a href="#ece371846-bib-0023" class="usa-link" aria-describedby="ece371846-bib-0023">2018</a>). The r7 (~591 Kbp) region harbors 27 genes and two lncRNAs while r9 (~1736 Kbp) contains nine genes, four microRNAs, one tRNA, and six lncRNAs. Both regions harbor potential candidate genes for the adaptation to high elevation habitats as previously described (Wallberg et al. <a href="#ece371846-bib-0097" class="usa-link" aria-describedby="ece371846-bib-0097">2017</a>): r7 includes four out of five octopamine receptors (LOC413698, LOC412994 and LOC412896), which regulate complex behaviors such as division of labor, foraging, memory formation, and thermogenesis (Behrends and Scheiner <a href="#ece371846-bib-0007" class="usa-link" aria-describedby="ece371846-bib-0007">2012</a>; Scheiner et al. <a href="#ece371846-bib-0083" class="usa-link" aria-describedby="ece371846-bib-0083">2014</a>; Reim and Scheiner <a href="#ece371846-bib-0074" class="usa-link" aria-describedby="ece371846-bib-0074">2014</a>; Kaya‐Zeeb et al. <a href="#ece371846-bib-0051" class="usa-link" aria-describedby="ece371846-bib-0051">2022</a>). On the other hand, r9 includes a peripheral plasma membrane protein CASK (LOC411347), which is reported to interact with CaMKII and together promotes memory formation (Wallberg et al. <a href="#ece371846-bib-0097" class="usa-link" aria-describedby="ece371846-bib-0097">2017</a>; Scholl et al. <a href="#ece371846-bib-1002" class="usa-link" aria-describedby="ece371846-bib-1002">2015</a>) So far, these two inversions have been identified only in populations from three East African mountain systems (Mau region, Mt. Kenya, and Mt. Elgon). However, their presence in other East African mountains has yet to be confirmed. Everitt et al. (<a href="#ece371846-bib-0030" class="usa-link" aria-describedby="ece371846-bib-0030">2023</a>) examined high‐altitude populations in South America and found no evidence of these inversions there, suggesting that their role in altitude adaptation may be geographically restricted.</p>
<p>However, it is conceivable that not only genes within the two inverted regions are not the sole determinants of adaptation to high‐elevation habitats in East African regions, but that genome‐wide alleles may also play a role in this process, following a polygenic hypothesis (Höllinger et al. <a href="#ece371846-bib-0046" class="usa-link" aria-describedby="ece371846-bib-0046">2019</a>). We investigated genome‐wide selective sweeps in three different populations: two previously studied (Wallberg et al. <a href="#ece371846-bib-0097" class="usa-link" aria-describedby="ece371846-bib-0097">2017</a>) from the Mau region and Mt. Kenya, and a newly sequenced population collected recently from the Rwenzori Mountains on the Ugandan slope. We obtained transcriptomes from highland and lowland Ugandan honey bees and identified Differentially Expressed Genes (DEGs). Our research aims to address the following questions: (1) What are the specific genes within the r7 and r9 inversions that contribute to high‐altitude adaptation? (2) Are there other genomic regions that play a significant role in adaptation to high‐elevation habitats? (3) Could the two inversion polymorphisms influence gene expression within the inverted regions by maintaining divergent regulatory variants between haplotypes? (4) Do we find evidence for genetic divergence and differential gene expression associated with highland and lowland habitats? By answering these questions, we aim to better understand the genetic mechanisms underlying the adaptation of honey bees to diverse environmental conditions in East Africa.</p></section><section id="ece371846-sec-0002"><h2 class="pmc_sec_title">2. Materials and Methods</h2>
<section id="ece371846-sec-0003"><h3 class="pmc_sec_title">2.1. Samples Collection and Molecular Methods</h3>
<p>Honey bees were collected on Rwenzori mountains, Western Uganda. A total of six <em>A. m. monticola</em> colonies were sampled in the mountain forest area at an altitude ranging from 2100–3000 m a.s.l. inside the Rwenzori national park. Corresponding <em>scutellata</em> bees were collected in the nearby lowland savannah at an altitude of 1123 m a.s.l. from two beehives (Table <a href="#ece371846-supitem-0005" class="usa-link">S1</a>). Returning forager honey bees were collected individually in a small glass vial and immediately killed by decapitation. The head was stored in RNA‐Later (Thermo Fisher) for RNA preservation, and the remaining body parts were stored in ultra‐pure ethanol (concentration &gt; 99%) for DNA conservation. DNA was extracted from the full thorax from three bees per colony (highland: <em>n</em> = 18; lowland: <em>n</em> = 6) using the Puregene Tissue Kit (QIAGEN) following the manufacturer's procedure. Total RNA was extracted from 12 honey bee heads collected at different elevations: six were collected at high elevation and were sampled from two different colonies, while the remaining six were sampled in the lowland following the same sampling method. The manufacturer's protocol of the TRIzol reagent (Thermo Fisher) was used to perform the RNA extraction. Briefly, the honey bee head was homogenized in 500 μL of TRIzol, further processed using 100 μL chloroform, and precipitated with 300 μL isopropanol. After solvent evaporation, the RNA pellet was dissolved in 30 μL of RNase‐free water and stored at −80°C until further processing. The 24 samples were barcoded, and 2× 150 bp paired‐end reads were sequenced on an Illumina Novaseq 6000 platform targeting 25× coverage, an output of 10 Gb per sample for whole genome sequencing data, and 40 million reads with a total of 12 Gb output per sample for transcriptome sequencing (Biomarker Technologies (BMK) GmbH).</p>
<p>To get a more comprehensive picture of differentiation between the African honey bee populations, we integrated <em>monticola</em> and <em>scutellata</em> in a larger dataset including additional African honey bee subspecies from publicly available whole genome data (Short Read Archive (SRA; Table <a href="#ece371846-supitem-0005" class="usa-link">S2</a>)). We included 11 <em>A. m. scutellata</em> (<em>n</em> = 11, 38× mean coverage) from Harpur et al. (<a href="#ece371846-bib-0043" class="usa-link" aria-describedby="ece371846-bib-0043">2014</a>; Bioproject: PRJNA216922); <em>A. m. adansonii</em> (<em>n</em> = 7), <em>A. m. capensis</em> (<em>n</em> = 12), <em>A. m. intermissa</em> (<em>n</em> = 15), <em>A. m. lamarckii</em> (<em>n</em> = 6), <em>A. m. monticola</em> (<em>n</em> = 6), and <em>A. m. scutellata</em> (<em>n</em> = 28), all with 68× mean coverage from Dogantzis et al. (<a href="#ece371846-bib-0028" class="usa-link" aria-describedby="ece371846-bib-0028">2021</a>; Bioproject: PRJNA729035); <em>A. m. monticola</em> (<em>n</em> = 1), <em>A. m. scutellata</em> (<em>n</em> = 5), and <em>A. m. jemenitica</em> (<em>n</em> = 3) all with 26× mean coverage from (Fuller et al. <a href="#ece371846-bib-0036" class="usa-link" aria-describedby="ece371846-bib-0036">2015</a>; Bioproject: PRJNA237819). We included the dataset from Wallberg et al. (<a href="#ece371846-bib-0097" class="usa-link" aria-describedby="ece371846-bib-0097">2017</a>; Bioproject: PRJNA357367) comprising <em>A. m. scutellata</em> (<em>n</em> = 19) and <em>A. m. monticola</em> (<em>n</em> = 20) genomes, all with 12× mean coverage. Finally, we obtained new genome sequences of <em>A. m. monticola</em> (<em>n</em> = 18) and <em>A. m. scutellata</em> (<em>n</em> = 6) collected in the Rwenzori mountains and the surrounding savannah in Uganda, with a mean coverage of 25×.</p></section><section id="ece371846-sec-0004"><h3 class="pmc_sec_title">2.2. 
DNA‐Seq Read Mapping and Variant Calling</h3>
<p>Two datasets were generated. The first one included all honey bees described before. The second dataset included only the honey bee sequence data from (Wallberg et al. <a href="#ece371846-bib-0097" class="usa-link" aria-describedby="ece371846-bib-0097">2017</a>) of Kenya and the newly sequenced honey bees from Uganda. Adapter sequences from raw Illumina sequencing reads were trimmed using <span class="font-variant-small-caps">cutadapt</span> v2.6 (Martin <a href="#ece371846-bib-0063" class="usa-link" aria-describedby="ece371846-bib-0063">2011</a>). For paired‐end reads, we additionally set the minimum read length ‘‐m’ to 50 and enabled the ‘paired‐output’ option for read trimming. The quality of the reads was assessed with <span class="font-variant-small-caps">fastqc</span> v0.12.1 (Andrews et al. <a href="#ece371846-bib-0003" class="usa-link" aria-describedby="ece371846-bib-0003">2012</a>). The reads were then aligned to the honey bee reference genome <em>Amel_HAv3.1</em> (Wallberg et al. <a href="#ece371846-bib-0096" class="usa-link" aria-describedby="ece371846-bib-0096">2019</a>) using the <span class="font-variant-small-caps">burrow wheeler aligner</span> (BWA) v0.7.17.1188 with the BWA‐MEM algorithm (Li and Durbin <a href="#ece371846-bib-0058" class="usa-link" aria-describedby="ece371846-bib-0058">2009</a>).</p>
<p>SNP calling was carried out with ‘HaplotypeCaller’ from <span class="font-variant-small-caps">gatk4 v4.3.0.0</span>, followed by joint genotyping using the ‘GenotypeGVCFs’ function (Poplin et al. <a href="#ece371846-bib-0070" class="usa-link" aria-describedby="ece371846-bib-0070">2017</a>; Van der Auwera et al. <a href="#ece371846-bib-0093" class="usa-link" aria-describedby="ece371846-bib-0093">2013</a>). Data were imported by chromosome with ‘GenomicsDBImport’ and a batch size of 20. The resulting VCFs were merged with <span class="font-variant-small-caps">bcftools</span> v1.11 (Danecek et al. <a href="#ece371846-bib-0026" class="usa-link" aria-describedby="ece371846-bib-0026">2021</a>). Allele balance for each variant was annotated with <span class="font-variant-small-caps">gatk</span>3 v3.8.1.0 and filtered using <span class="font-variant-small-caps">vcffilter</span> with the condition‘ABHet &gt; 0.25 &amp; ABHet &lt; 0.75|ABHet &lt; 0.01’, to allow for homozygous variants, too (Garrison et al. <a href="#ece371846-bib-0038" class="usa-link" aria-describedby="ece371846-bib-0038">2022</a>).</p>
<p>After evaluating the distribution of the parameters from the INFO field in the final VCF, additional filters were applied with <span class="font-variant-small-caps">bcftools view</span> v1.11 (Danecek et al. <a href="#ece371846-bib-0026" class="usa-link" aria-describedby="ece371846-bib-0026">2021</a>). The filtering criteria for variant sites included the following thresholds: MQ &lt; 40, MQRankSum &lt; −5, MQRankSum &gt; 5, ExcessHet &gt; 20, AC = 1, INFO/DP &lt; 11,660, INFO/DP &gt; 21,654, AN &lt; 826, FS &gt; 20, SOR &gt; 5, and QD &lt; 5. Only biallelic SNPs were considered, and indels were excluded. The <span class="font-variant-small-caps">gatk4</span> function ‘SelectVariants’ was used to retain only variant sites in chromosomes for downstream analysis (Van der Auwera et al. <a href="#ece371846-bib-0093" class="usa-link" aria-describedby="ece371846-bib-0093">2013</a>). Additionally, samples with a coverage of less than 6× were removed from the dataset. Finally, the remaining variants were phased using <span class="font-variant-small-caps">beagle</span> v5.4 (22Jul22.46e.jar; Browning et al. <a href="#ece371846-bib-0018" class="usa-link" aria-describedby="ece371846-bib-0018">2018</a>, <a href="#ece371846-bib-0017" class="usa-link" aria-describedby="ece371846-bib-0017">2021</a>).</p></section><section id="ece371846-sec-0005"><h3 class="pmc_sec_title">2.3. 
RNA‐Seq Data and Gene Expression Analysis</h3>
<p>For transcriptome sequencing the reads quality was checked using <span class="font-variant-small-caps">fastqc</span> (Andrews et al. <a href="#ece371846-bib-0003" class="usa-link" aria-describedby="ece371846-bib-0003">2012</a>). Adapters were removed and reads were trimmed using <span class="font-variant-small-caps">trimmomatic</span> v0.36 (Bolger et al. <a href="#ece371846-bib-0014" class="usa-link" aria-describedby="ece371846-bib-0014">2014</a>). After trimming, <span class="font-variant-small-caps">fastqc</span> was used again to check the quality of the trimming process. Trimmed reads were used by <span class="font-variant-small-caps">kallisto</span> (Bray et al. <a href="#ece371846-bib-0016" class="usa-link" aria-describedby="ece371846-bib-0016">2016</a>) to quantify the abundances of transcripts. Then, <span class="font-variant-small-caps">deseq2</span> (Love et al. <a href="#ece371846-bib-0061" class="usa-link" aria-describedby="ece371846-bib-0061">2014</a>) was used to analyse differentially expressed genes. Finally, a GO enrichment analysis was done with <span class="font-variant-small-caps">gprofiler2</span> (Kolberg et al. <a href="#ece371846-bib-0053" class="usa-link" aria-describedby="ece371846-bib-0053">2023</a>).</p></section><section id="ece371846-sec-0006"><h3 class="pmc_sec_title">2.4. Principal Component Analysis Population Genetics Statistics and Inversion Classification</h3>
<p>
<span class="font-variant-small-caps">plink</span> v1.90 (Chang et al. <a href="#ece371846-bib-0021" class="usa-link" aria-describedby="ece371846-bib-0021">2015</a>) was used to convert the final .<em>vcf</em> file to .<em>bed</em>, .<em>bim</em>, and .<em>fam</em> plink format files. During the process, the dataset was filtered for Hardy–Weinberg equilibrium (‐‐hwe 0.0001), missing data (‐‐geno 0.01) and minor allele frequency (‐‐maf 0.01) and linkage disequilibrium, analyzing 50 k single nucleotide polymorphisms (SNP) at a time with a sliding window of 5 k SNPs and using a threshold of r2 of 0.5 (‐‐indep‐pairwise 50 5 0.5). Filtering pass SNPs were used to compute a principal component analysis (PCA) using the ‐‐pca plink function.</p>
<p>We calculated the population genetics parameters <em>F</em>
<sub>ST</sub>, <em>D</em>xy, and <em>π</em> to further gain insights into the populations we are examining. The fixation index <em>F</em>
<sub>ST</sub> was used to quantify relative genetic differentiation between populations. <em>D</em>xy was calculated as the average number of nucleotide differences per site between populations, providing an estimate of absolute divergence between populations, whereas <em>π</em>, a measure of nucleotide diversity, is computed as differences per site within each population. All three of these indexes were computed using <span class="font-variant-small-caps">pixy v2.0.0</span> (Korunes and Samuk <a href="#ece371846-bib-0055" class="usa-link" aria-describedby="ece371846-bib-0055">2021</a>). Also, where needed to test balancing selection, <span class="font-variant-small-caps">betascan2</span> was used as the main tool (Siewert and Voight <a href="#ece371846-bib-1003" class="usa-link" aria-describedby="ece371846-bib-1003">2020</a>); this software allows us to compute beta values that are indicative of balancing selection. A value of 2 assigned by the tool could suggest balancing selection; higher and continuous values across a genomic region strengthen this assumption.</p>
<p>Previously identified r7 and r9 regions were then extracted from the final .vcf file using <span class="font-variant-small-caps">vcftools</span> v0.1.16 (Danecek et al. <a href="#ece371846-bib-0025" class="usa-link" aria-describedby="ece371846-bib-0025">2011</a>) with the information relative to breakpoints described in Christmas et al. (<a href="#ece371846-bib-0023" class="usa-link" aria-describedby="ece371846-bib-0023">2018</a>). As previously described, <span class="font-variant-small-caps">plink</span> v1.90 was used to convert the dataset into plink file format applying the same filtering. Then, inversion polymorphisms in r7 and r9 were classified using the <span class="font-variant-small-caps">invclust</span> R package. The <em>invClust</em> R package is designed to infer individual genotypes for chromosomal inversion using SNP. The method is based on a clustering‐based approach: (1) a dimensionality reduction is applied, then (2) INVCLUST applies a Gaussian mixture model to classify individuals into three distinct groups, and finally, (3) the clustering model is validated through Bayesian Information Criterion (BIC) (Cáceres and González <a href="#ece371846-bib-0020" class="usa-link" aria-describedby="ece371846-bib-0020">2015</a>). This method was applied to Mau, Mt. Kenya, and Rwenzori populations.</p></section><section id="ece371846-sec-0007"><h3 class="pmc_sec_title">2.5. 
McDonald‐Kreitman Test and Selection Scan (Selscan)</h3>
<p>To compute the McDonald‐Kreitman test for genes within the r7 and r9 regions, the software <span class="font-variant-small-caps">degenotate</span> (Mirchandani et al. <a href="#ece371846-bib-1004" class="usa-link" aria-describedby="ece371846-bib-1004">2024</a>) was utilized. This test assesses the ratio of synonymous to non‐synonymous changes to identify genes potentially undergoing positive selection. However, the McDonald‐Kreitman test is limited by its exclusive focus on coding sequences and its tendency to be overly conservative. To address these limitations and to broaden the analysis, <span class="font-variant-small-caps">selscan v2.0</span> (Szpiech and Hernandez <a href="#ece371846-bib-0090" class="usa-link" aria-describedby="ece371846-bib-0090">2014</a>) was additionally applied to identify genomic regions exhibiting significant deviations from neutrality. Within the framework of Selscan, the tool XP‐nSL (Szpiech et al. <a href="#ece371846-bib-0091" class="usa-link" aria-describedby="ece371846-bib-0091">2021</a>) was used to detect signatures of selection in a test population given another population as reference. Highland versus lowland populations were compared using default parameters for the Mau region (highland: <em>n</em> = 9; lowland: <em>n</em> = 10), Mt. Kenya (highland: <em>n</em> = 10; lowland: <em>n</em> = 10), and the Rwenzori mountain system (highland: <em>n</em> = 18; lowland: <em>n</em> = 6). Raw XP‐nSL values were obtained (selscan flags ‐‐xpnsl ‐‐vcf mountain‐population.vcf ‐‐vcf‐ref savannah‐population.vcf) and scores were normalized with the companion program <span class="font-variant-small-caps">norm</span> v1.3.0 (norm flags –xpnsl), considering everything above an XP‐nSL value of 2 or below −2 as potentially significant, as reported in the Selscan manual. During normalization, we also scanned by 10 kb non‐overlapping windows to detect regions of high differentiation. We then selected the top 1% positive scoring windows, which are those related to adaptation to high elevation habitat. The maximum value present in the window was assigned then to all genes within, as in (Szpiech et al. <a href="#ece371846-bib-0091" class="usa-link" aria-describedby="ece371846-bib-0091">2021</a>). Positive values of the XP‐nSL statistic indicate a potential beneficial mutation in the test population, whereas negative values potentially indicate that the SNP is favorable in the reference population. By using this tool, we can get a more complete picture of the alleles involved in adaptation to high altitude in 
<em>Apis mellifera</em>
.</p>
<p>Genomic positions above the threshold for each population were mapped to the latest version of the 
<em>Apis mellifera</em>
 .<em>gff</em> file using <span class="font-variant-small-caps">bedtools</span> v2.26.0 (Quinlan and Hall <a href="#ece371846-bib-0072" class="usa-link" aria-describedby="ece371846-bib-0072">2010</a>). Only SNPs mapping to a gene were kept, including the gene features promoter, untranslated regions, and coding sequences that were then used for a Gene Ontology (GO) enrichment analysis with <span class="font-variant-small-caps">gprofiler2</span> (Kolberg et al. <a href="#ece371846-bib-0053" class="usa-link" aria-describedby="ece371846-bib-0053">2023</a>).</p></section></section><section id="ece371846-sec-0008"><h2 class="pmc_sec_title">3. Results</h2>
<section id="ece371846-sec-0009"><h3 class="pmc_sec_title">3.1. Ugandan Honey Bees Are Highly Similar to Kenyan Honey Bees</h3>
<p>For the first time, we sampled honey bees inhabiting a distinct mountain system in western Uganda, known as the Rwenzori mountain system. This area, situated approximately 800 km from the previous sampling points, presented an intriguing research opportunity. A total of six honey bees were sequenced from the lowland savannah environment, and 18 were sequenced from the high mountain area. These honey bees were used along with those from Wallberg et al. (<a href="#ece371846-bib-0097" class="usa-link" aria-describedby="ece371846-bib-0097">2017</a>) to infer population genetics questions (Figure <a href="#ece371846-fig-0001" class="usa-link">1A</a>).</p>
<figure class="fig xbox font-sm" id="ece371846-fig-0001"><h4 class="obj_head">FIGURE 1.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dbfb/12367272/e7719e6666f2/ECE3-15-e71846-g002.jpg" loading="lazy" id="jats-graphic-3" height="796" width="709" alt="FIGURE 1"></p>
<div class="p text-right font-secondary"><a href="figure/ece371846-fig-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A) Sampling location of the honey bees analyzed in this study, brown triangles represent honey bees collected at high elevation, while yellow triangles show honey bees collected in the nearby savannah. Ugandan honey bees were collected at an altitude ranging from 1123 m a.s.l. to 2600 m. Kenyan honey bees analyzed in this study were collected and described by Wallberg et al. (<a href="#ece371846-bib-0097" class="usa-link" aria-describedby="ece371846-bib-0097">2017</a>) (B) Principal component analysis of A‐Lineage honey bees from Africa. (C) Pairwise <em>F</em>
<sub>ST</sub> of highland (<em>n</em> = 18) and lowland honey bees (<em>n</em> = 6) population collected on Rwenzori mountain system, Uganda.</p></figcaption></figure><p>After quality check and trimming, the dataset was left with 6,312,219 single‐nucleotide polymorphisms (SNPs) which were used for the following analysis. Whole genome SNPs were used to perform an explorative principal component analysis in order to observe if the newly sequenced Ugandan honey bees form a distinct cluster with respect to other honey bees belonging to the A‐Lineage (Figure <a href="#ece371846-fig-0001" class="usa-link">1B</a>; Table <a href="#ece371846-supitem-0005" class="usa-link">S1</a>). The PCA computed showed a clear distinction of some honey bee subspecies (i.e., <em>capensis, adansonii</em> and <em>intermissa</em>), while <em>monticola</em> and <em>scutellata</em> honey bees tend to cluster according to sample location (i.e., Kenya, South Africa) rather than subspecies (Figure <a href="#ece371846-fig-0001" class="usa-link">1B</a>). This pattern holds also for newly sequenced samples for which it is not possible to observe a distinct cluster with respect to East African <em>monticola</em> and <em>scutellata</em> bees.</p></section><section id="ece371846-sec-0010"><h3 class="pmc_sec_title">3.2. Inversion Polymorphisms in Uganda Mountain Honey Bees Are Substantially Different From Their Kenyan Counterparts</h3>
<p>The following analysis was conducted using only the newly generated data and the Kenyan honey bees collected by Wallberg et al. (<a href="#ece371846-bib-0097" class="usa-link" aria-describedby="ece371846-bib-0097">2017</a>). This dataset is the only suitable one for our purposes since it includes honey bees collected in highland and lowland from geographically close populations (Figure <a href="#ece371846-fig-0001" class="usa-link">1A</a>). Closer analysis of the populations considered revealed little to no differentiation between the three populations (Table <a href="#ece371846-tbl-0001" class="usa-link">1</a>) with the lowest <em>F</em>
<sub>ST</sub> value of ~0.002 found when considering Mau highland versus Mt. Kenya lowland and the highest value of ~0.028 found when considering Rwenzori lowland versus Mt. Kenya highland This pattern is consistent with both previous findings and the PCA which includes all samples (Figure <a href="#ece371846-fig-0001" class="usa-link">1B</a>), where it is not possible to identify two clear clusters when considering the Rwenzori and Kenyan populations. Along with <em>F</em>
<sub>ST</sub>, we also calculated Dxy and π. All together, these metrics can be used to assess diversity and differentiation since each one gives insight into something important that is not covered by the others (Figure <a href="#ece371846-supitem-0001" class="usa-link">S1</a>; Table <a href="#ece371846-tbl-0001" class="usa-link">1</a>). Additionally, we ran Admixture with five different seeds with <em>K</em> = 1–6. The lowest cross‐validation error was detected at <em>K</em> = 1 for all five seeds considered (Table <a href="#ece371846-supitem-0005" class="usa-link">S3</a>, Figure <a href="#ece371846-supitem-0002" class="usa-link">S2</a>), supporting the view that all honey bees used in this study belong to the same ancestry.</p>
<section class="tw xbox font-sm" id="ece371846-tbl-0001"><h4 class="obj_head">TABLE 1.</h4>
<div class="caption p"><p>

<em>Apis mellifera scutellata</em>
 and 
<em>Apis mellifera monticola</em>
 dataset and genetic differentiation (<em>F</em>
<sub>ST</sub>).</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Region</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Population</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Label</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>N</em>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>π</em>
</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">MF</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">MS</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">MKF</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">MKS</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">RWHL</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">RWLL</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" rowspan="2" valign="top" colspan="1">Mau</td>
<td align="center" valign="top" rowspan="1" colspan="1">Highland</td>
<td align="center" valign="top" rowspan="1" colspan="1">MF</td>
<td align="center" valign="top" rowspan="1" colspan="1">9</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.1223</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.009</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.004</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.004</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.012</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.016</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">Lowland</td>
<td align="center" valign="top" rowspan="1" colspan="1">MS</td>
<td align="center" valign="top" rowspan="1" colspan="1">10</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.1224</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.009</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.011</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.002</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.016</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.022</td>
</tr>
<tr>
<td align="left" rowspan="2" valign="top" colspan="1">Mt. Kenya</td>
<td align="center" valign="top" rowspan="1" colspan="1">Highland</td>
<td align="center" valign="top" rowspan="1" colspan="1">MKF</td>
<td align="center" valign="top" rowspan="1" colspan="1">10</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.1212</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.004</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.011</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.009</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.015</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.028</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">Lowland</td>
<td align="center" valign="top" rowspan="1" colspan="1">MKS</td>
<td align="center" valign="top" rowspan="1" colspan="1">9</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.1216</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.004</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.002</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.009</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.017</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.023</td>
</tr>
<tr>
<td align="left" rowspan="2" valign="top" colspan="1">Rwenzori</td>
<td align="center" valign="top" rowspan="1" colspan="1">Highland</td>
<td align="center" valign="top" rowspan="1" colspan="1">RWHL</td>
<td align="center" valign="top" rowspan="1" colspan="1">18</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.1278</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.012</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.016</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.015</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.017</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.024</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">Lowland</td>
<td align="center" valign="top" rowspan="1" colspan="1">RWLL</td>
<td align="center" valign="top" rowspan="1" colspan="1">6</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.1241</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.024</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.022</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.028</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.023</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.024</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ece371846-tbl-0001/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="ece371846-note-0002"><p>
<em>Note:</em> The symmetric matrix represent the genome wide comparison between all population involved in this study. The low <em>F</em>
<sub>ST</sub> values highlight the very small genetic difference between highland and lowland honey bees in East Africa.</p></div>
<div class="fn" id="ece371846-note-0003"><p>Abbreviations: MF, Mau forest; MKF, Mt. Kenya forest; MKS, Mt. Kenya savannah; MS, Mau savannah; RWHL, Rwenzori forest; RWLL, Rwenzori savannah.</p></div>
</div></section><p>As detected in previous studies, an <em>F</em>
<sub>ST</sub> significant increment was found in correspondence with previously described r7 and r9 regions, also when comparing Uganda highland honey bees to lowland Ugandan honey bees. This increase in <em>F</em>
<sub>ST</sub> values is evident despite the sample size of just six honey bees in the lowlands (Figure <a href="#ece371846-fig-0001" class="usa-link">1C</a>).</p>
<p>Given the fact that population genetics statistics detected an increase in the genomic regions in correspondence of r7 and r9, an accurate classification of the r7 and r9 chromosomal inversion status was carried out using the <em>invClust</em> R package (Cáceres and González <a href="#ece371846-bib-0020" class="usa-link" aria-describedby="ece371846-bib-0020">2015</a>). In the Rwenzori mountain population, the frequency of inverted homozygous individuals (INV) in r7 in the highland was 0.83, the heterozygous (HET) were 0.11, while homozygous inverted individuals were 0.06 (STD). In the lowland, no homozygous individuals were detected, while 0.5 were classified as heterozygous and 0.5 were classified as STD. In the r9 region in the highland, 0.83 honey bees were classified as INV and 0.17 were classified as HET. In the lowland, 0.2 individuals were assigned as HET, while the remaining 0.8 were classified as STD. In both situations, the increase of the inverted allele in the highland populations shows a clear association between highland environment and the INV genotype (Table <a href="#ece371846-tbl-0002" class="usa-link">2</a>).</p>
<section class="tw xbox font-sm" id="ece371846-tbl-0002"><h4 class="obj_head">TABLE 2.</h4>
<div class="caption p"><p>Frequency of r7 and r9 inversion polymorphisms in the three considered populations, and relative <em>F</em>
<sub>ST</sub> values when compared to other populations.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom">
<tr style="border-bottom:solid 1px #000000">
<th align="left" rowspan="3" valign="bottom" colspan="1"></th>
<th align="center" rowspan="2" valign="bottom" colspan="1"></th>
<th align="center" rowspan="2" colspan="2" valign="bottom">Frequency</th>
<th align="center" colspan="6" valign="bottom" rowspan="1">Genetic differentiation (<em>F</em>
<sub>ST</sub>)</th>
</tr>
<tr style="border-bottom:solid 1px #000000"><th align="center" colspan="6" valign="bottom" rowspan="1">r9</th></tr>
<tr style="border-bottom:solid 1px #000000">
<th align="center" valign="bottom" rowspan="1" colspan="1">Label</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">r7</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">r9</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">MF</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">MS</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">MKF</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">MKS</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">RWHL</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">RWLL</th>
</tr>
</thead>
<tbody valign="top">
<tr>
<td align="left" rowspan="6" valign="top" colspan="1">r7</td>
<td align="center" valign="top" rowspan="1" colspan="1">MF</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.9</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.9</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.108</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.005</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.176</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.067</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.279</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">MS</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.1</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.1</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.348</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.131</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.007</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.167</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.067</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">MKF</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.0</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.89</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.045</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.480</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.199</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.073</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.242</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">MKS</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.2</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.2</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.442</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.035</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.563</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.226</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.011</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">RWHL</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.89</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.92</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.099</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.410</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.150</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.493</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.279</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">RWLL</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.25</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.08</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.309</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.024</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.493</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.139</td>
<td align="center" valign="top" rowspan="1" colspan="1">0.395</td>
<td align="center" valign="top" rowspan="1" colspan="1">—</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ece371846-tbl-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="ece371846-note-0004"><p>
<em>Note:</em> The frequency reported is the frequency of the INV allele over the overall number of alleles; the lower triangle shows F<sub>ST</sub> values relative to the r7 inversion; the upper triangle shows F<sub>ST</sub> values relative to the r9 inversions.</p></div>
<div class="fn" id="ece371846-note-0005"><p>Abbreviations: MF, Mau Forest; MKF, Mount Kenya Forest; MKS, Mount Kenya Savannah; MS, Mau Savannah; RWHL, Rwenzori Highland; RWLL, Rwenzori Lowland.</p></div>
</div></section><p>A similar inversion‐genotype classification was performed for populations from the Mau region and Mt. Kenya, using the same methodological framework as in Wallberg et al. (<a href="#ece371846-bib-0097" class="usa-link" aria-describedby="ece371846-bib-0097">2017</a>). In the Mau savannah, the INV, HET, and STD of the r7 inversion sample frequencies were 0.0, 0.3, and 0.7, respectively, while in the Mau forest environment, INV, HET, and STD were 0.89, 0.11, and 0.0. The r9 region follows a similar pattern, with two out of nine samples classified as HET in the highland and seven out of nine samples classified as INV. Lowland samples showed a decrease in the INV individuals, with two out of ten INV, two out of 10 HET, and the remaining ones classified as STD (Table <a href="#ece371846-tbl-0002" class="usa-link">2</a>).</p>
<p>The Mt. Kenya population followed a similar pattern. In the savannah r7, the STD allele was completely fixed, while in the highland, the INV allele reached fixation. The r9 region showed similar frequencies, with 0.9 INV and 0.1 HET individuals in the highland. In the Mt. Kenya savannah population, we could not observe INV samples, and the majority of collected specimens were classified as STD (0.78) with the remaining 0.12 being observed as HET (Table <a href="#ece371846-tbl-0002" class="usa-link">2</a>).</p>
<p>A PCA was computed for r7 and r9 regions only. The resulting plot clearly highlights different clusters relative to the genotype of the individuals (INV = inverted homozygous, HET = heterozygous, NI = non inverted homozygous). We observe that both the INV and HET samples are further subdivided between Kenyan and Ugandan samples (Figure <a href="#ece371846-fig-0002" class="usa-link">2A,B</a>). This differentiation can be attributed to the presence of the inverted allele in both the INV and HET individuals. It is evident that the diversity is concentrated in this allele, as its absence is concomitant with the absence of the allele itself (i.e., NI honey bees in Figure <a href="#ece371846-fig-0002" class="usa-link">2A,B</a>). The increased diversity derives probably from the lack of gene flow between the Ugandan inverted haplotype and Kenyan inverted haplotypes. We computed a sliding window Dxy on all chromosomes and found evidence of this differentiation in both chromosome 7 and chromosome 9. It is evident from Figure <a href="#ece371846-fig-0002" class="usa-link">2C,D</a> that the average Dxy in r7 and r9 is larger for the comparison between the Rwenzori mountain highland population and highland Kenyan populations than for the comparison between highland Kenyan populations. Possibly the shift in frequency of many SNPs inside these regions is due to genetic drift and lack of gene flow between high mountain peaks in Uganda and Kenya; however, in all populations analyzed (Mau, Mt. Kenya and Uganda) the average Dxy observed is similar when comparing highland and lowland from the same mountain region. We also can observe that the average Dxy computed in r9 is relatively lower than the one observed in r7. This behavior could be attributed to the sizes of the two inversion polymorphisms, which may lead to differences in levels of recombination. The r7 region spans approximately 0.6 Mb, while the r9 region has an approximate dimension of 1.7 Mb. It is conceivable that genetic exchange may occur in inverted regions in the event of a double crossing‐over or gene conversion events (Rozas and Aguadé <a href="#ece371846-bib-0078" class="usa-link" aria-describedby="ece371846-bib-0078">1994</a>; Schaeffer and Anderson <a href="#ece371846-bib-0080" class="usa-link" aria-describedby="ece371846-bib-0080">2005</a>; Schaeffer et al. <a href="#ece371846-bib-0082" class="usa-link" aria-describedby="ece371846-bib-0082">2024</a>). The probability of the first occurring, however, is thought to increase with the distance from the breakpoints (Navarro et al. <a href="#ece371846-bib-0068" class="usa-link" aria-describedby="ece371846-bib-0068">1997</a>; Faria and Navarro <a href="#ece371846-bib-0031" class="usa-link" aria-describedby="ece371846-bib-0031">2010</a>; Stevison et al. <a href="#ece371846-bib-0089" class="usa-link" aria-describedby="ece371846-bib-0089">2011</a>). While we detected a considerable number of gene conversion tracts in both r7 and r9, which may represent the primary recombination force at work in these two inversion polymorphisms (data not shown), it is still probable that, given the size of the r9 region, double crossing‐over events occurred in the past, reducing haplotype difference even if the haplotype in Uganda significantly differs from the one observed in Kenya.</p>
<figure class="fig xbox font-sm" id="ece371846-fig-0002"><h4 class="obj_head">FIGURE 2.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dbfb/12367272/a7f7505d74c5/ECE3-15-e71846-g001.jpg" loading="lazy" id="jats-graphic-5" height="702" width="709" alt="FIGURE 2"></p>
<div class="p text-right font-secondary"><a href="figure/ece371846-fig-0002/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A, B) Principal component analysis of r7 and r9 inversion polymorphisms, shape of the dots represents inversion status, while the color represents the population. (C, D) Sliding window Dxy, of highland population comparison. Populations show the same overall values except in correspondence of the inversion polymorphisms where comparison between Rwenzori mountain population (RWM) and Kenyan mountain population (MF and MKF) show a significant increase with respect to comparison between MF and MKF.</p></figcaption></figure></section><section id="ece371846-sec-0011"><h3 class="pmc_sec_title">3.3. Chromosome 5 Shows r7‐ and r9‐Like Patterns</h3>
<p>We further explored the increased divergency detected at the beginning of chromosome five. The first SNP of increased Dxy value was detected at position 3811, with the last one at position 599,431 (Figure <a href="#ece371846-fig-0003" class="usa-link">3A,B</a>). This makes this region ~0.6 Mb large. The region encompasses 64 genes; no specific enrichment was observed when a GO term analysis was performed with the 64 genes; however, we noted the gene PHRF1 (GeneID_727274) which is a PHD and ring finger domain 1 protein, and it is highly involved in the regulation of the epigenome by maintaining the correct DNA methylation patterns. We computed the genotype of single individuals, evaluating the quantity of 1|1 (homozygous alternative), 0|1 and 1|0 (heterozygous), and 0|0 (homozygous reference). We noted that the heterozygous individuals are mostly distributed across the different populations. In particular, the heterozygous genotype is present at a higher frequency in different populations (i.e., Uganda; Figure <a href="#ece371846-fig-0003" class="usa-link">3C</a>). Interestingly, there are strong signs of LD (Figure <a href="#ece371846-fig-0003" class="usa-link">3D</a>) supporting the view of potential recent selection signals given the high Dxy and nucleotide diversity, particularly in Rwenzori mountain samples. By using BetaScan2, we tested highland samples for signals of balancing selection given the very high amount of heterozygous sites in the newly discovered region. We observed long runs of heterozygosity with beta1 index values exceeding 2 in highland populations, indicating potential regions under balancing selection (Siewert and Voight <a href="#ece371846-bib-1003" class="usa-link" aria-describedby="ece371846-bib-1003">2020</a>). In contrast, this pattern was not evident in lowland honey bee populations (Figure <a href="#ece371846-supitem-0003" class="usa-link">S3</a>).However, it was difficult to confirm if this region actively involves structural variants.</p>
<figure class="fig xbox font-sm" id="ece371846-fig-0003"><h4 class="obj_head">FIGURE 3.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dbfb/12367272/ac6bf973d206/ECE3-15-e71846-g006.jpg" loading="lazy" id="jats-graphic-7" height="919" width="709" alt="FIGURE 3"></p>
<div class="p text-right font-secondary"><a href="figure/ece371846-fig-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>(A, B) Dxy and Pi sliding window of 10 kb non overlapping SNPs of chromosome 5. (C) Genotype distribution of homozygous alternative (1!1) heterozygous (0!1) and homozygous reference (0|0) variants inside the region. (D) Linkage disequilibrium pattern of the first half of chromosome 5, the LDheatmap R package was used to generate the heatmap (Shin et al. <a href="#ece371846-bib-0086" class="usa-link" aria-describedby="ece371846-bib-0086">2006</a>).</p></figcaption></figure></section><section id="ece371846-sec-0012"><h3 class="pmc_sec_title">3.4. Selection Scan Patterns Confirm r7 and r9 as Crucial to Highland Adaptation</h3>
<p>A comparative analysis was conducted between highland and lowland populations utilizing XP‐nSL as the analytical tool. Along with this, we tested genes under positive/negative selection using the McDonald‐Kreitman test on genes in the r7 and r9 regions; however, no results were gathered with the exception of a few uncharacterized genes. This could be due to the nature of the test that considers only coding sequences and due to the sample size (Tables <a href="#ece371846-supitem-0005" class="usa-link">S4</a> and <a href="#ece371846-supitem-0005" class="usa-link">S5</a>). The lowland population was employed as the reference group to identify evidence of adaptation to high elevation habitats. In all three cases, lowland savannah populations were kept as reference.</p>
<p>The genes falling within the top 1% differentiated windows were extracted and subjected to a GO enrichment analysis; the XP‐nSL value assigned to the window was then assigned to all genes that were falling inside the region as proposed in (Szpiech et al. <a href="#ece371846-bib-0091" class="usa-link" aria-describedby="ece371846-bib-0091">2021</a>). The analyses were conducted against a database of annotated genes, with a threshold of significant GO terms offset at 0.05 and applying the g:SCS correction for multiple testing (Kolberg et al. <a href="#ece371846-bib-0053" class="usa-link" aria-describedby="ece371846-bib-0053">2023</a>). Our analysis revealed a variable number of genes involved in each population. In Rwenzori mountains population XP‐nSl a total of 302 genes were detected, in the Mau region from the selscan analysis, and after a GO term analysis, no results could be gathered. The Mt. Kenya yielded the most abundant result (494 genes), however, as for the Mau region, no significant GO terms were observed. Finally, the analysis performed on the Rwenzori mountain population exhibited a total of 262 genes; the following analysis revealed GO terms involved in cognition, memory formation, and G‐protein coupled receptor activity (Tables <a href="#ece371846-supitem-0005" class="usa-link">S6–S9</a>). A truly remarkable note is that the highest XP‐nSL values were observed in the r7 region in all three populations (Figure <a href="#ece371846-supitem-0004" class="usa-link">S4</a>), with octopamine receptors consistently ranking among the top values. A comparable pattern can be appreciated in the r9 region; although the values reported here are not as high as those located in r7. This outcome was anticipated in light of the genetic differentiation between the two regions and the previously calculated <em>F</em>
<sub>ST</sub> and Dxy values. Finally, we could not observe the same pattern for the region of increased diversity observed at the beginning of chromosome five.</p></section><section id="ece371846-sec-0013"><h3 class="pmc_sec_title">3.5. Differentially Expressed Genes Are Enriched for GO Terms Involved in Stress Adaptation</h3>
<p>We enriched our dataset by 12 transcriptomes obtained for the first time from honey bees of the Rwenzori mountains. Following the removal of low‐quality reads, the lowland samples which served as control cases had an average of approximately 40 million reads and a GC content of 45%, whereas the highland honey bees had an average of approximately 41 million reads and a GC content of 46%. A total of 181 DEGs were identified with an adjusted <em>p</em>‐value lower than 0.05. However, to reduce the discovery of false positives, we selected a threshold of log<sub>2</sub> fold change (log<sub>2</sub>FC) of ±0.7. In total, 29 genes were cut out from the analysis, leaving the dataset with 152 effective DEGs, divided into 39 downregulated genes and 113 exhibiting increased expression (Figure <a href="#ece371846-fig-0004" class="usa-link">4</a>). A GO enrichment analysis was conducted on up‐and downregulated genes independently against the annotated gene set of <em>Apis mellifera</em>, with a <em>p</em>‐value threshold of 0.05 and the g:SCS correction for multiple testing (Kolberg et al. <a href="#ece371846-bib-0053" class="usa-link" aria-describedby="ece371846-bib-0053">2023</a>). When analyzing only the downregulated genes, no significant GO terms were observed, while when we considered only upregulated genes, we found evidence of 33 different GO terms involved in total. Seven of these were identified in the context of molecular function (MF), 21 in the context of biological pathways (BP) and five enriched KEGG pathways (Table <a href="#ece371846-tbl-0003" class="usa-link">3</a>).</p>
<figure class="fig xbox font-sm" id="ece371846-fig-0004"><h4 class="obj_head">FIGURE 4.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dbfb/12367272/3a05f8a9af42/ECE3-15-e71846-g003.jpg" loading="lazy" id="jats-graphic-9" height="657" width="661" alt="FIGURE 4"></p>
<div class="p text-right font-secondary"><a href="figure/ece371846-fig-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Volcano plot of Differentially Expressed Genes (DEGs), blue and orange dots represent up‐ and downregulated genes in the highland respectively. Significance level was capped at 10e‐7 for a better representation of data. A total of 39 genes were detected as downregulated (orange) and a total 113 were detected as upregulated (blue), given a threshold of log<sub>2</sub>FC = ±0.7.</p></figcaption></figure><section class="tw xbox font-sm" id="ece371846-tbl-0003"><h4 class="obj_head">TABLE 3.</h4>
<div class="caption p"><p>GO enrichment analysis using DEGs.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">GO ID</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Term name</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Source</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>p</em>
</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0016491</td>
<td align="center" valign="top" rowspan="1" colspan="1">Oxidoreductase activity</td>
<td align="center" valign="top" rowspan="1" colspan="1">MF</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.83E‐04</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0102965</td>
<td align="center" valign="top" rowspan="1" colspan="1">Alcohol‐forming long‐chain fatty acyl‐CoA reductase activity</td>
<td align="center" valign="top" rowspan="1" colspan="1">MF</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.92E‐03</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0016620</td>
<td align="center" valign="top" rowspan="1" colspan="1">Oxidoreductase activity, acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor</td>
<td align="center" valign="top" rowspan="1" colspan="1">MF</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.10E‐02</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0016885</td>
<td align="center" valign="top" rowspan="1" colspan="1">Ligase activity, forming carbon–carbon bonds</td>
<td align="center" valign="top" rowspan="1" colspan="1">MF</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.87E‐02</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0080019</td>
<td align="center" valign="top" rowspan="1" colspan="1">Alcohol‐forming very long‐chain fatty acyl‐CoA reductase activity</td>
<td align="center" valign="top" rowspan="1" colspan="1">MF</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.28E‐02</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0003824</td>
<td align="center" valign="top" rowspan="1" colspan="1">Catalytic activity</td>
<td align="center" valign="top" rowspan="1" colspan="1">MF</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.53E‐02</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0016903</td>
<td align="center" valign="top" rowspan="1" colspan="1">Oxidoreductase activity, acting on the aldehyde or oxo group of donors</td>
<td align="center" valign="top" rowspan="1" colspan="1">MF</td>
<td align="center" valign="top" rowspan="1" colspan="1">4.09E‐02</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0044281</td>
<td align="center" valign="top" rowspan="1" colspan="1">Small‐molecule metabolic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.23E‐05</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0043436</td>
<td align="center" valign="top" rowspan="1" colspan="1">Oxoacid metabolic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">7.70E‐05</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0019752</td>
<td align="center" valign="top" rowspan="1" colspan="1">Carboxylic acid metabolic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">7.70E‐05</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0006082</td>
<td align="center" valign="top" rowspan="1" colspan="1">Organic acid metabolic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">8.29E‐05</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0032787</td>
<td align="center" valign="top" rowspan="1" colspan="1">Monocarboxylic acid metabolic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.20E‐04</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0044283</td>
<td align="center" valign="top" rowspan="1" colspan="1">Small‐molecule biosynthetic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.49E‐04</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0006739</td>
<td align="center" valign="top" rowspan="1" colspan="1">NADP metabolic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.50E‐04</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0006633</td>
<td align="center" valign="top" rowspan="1" colspan="1">Fatty acid biosynthetic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.05E‐04</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0072330</td>
<td align="center" valign="top" rowspan="1" colspan="1">Monocarboxylic acid biosynthetic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.05E‐04</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0046394</td>
<td align="center" valign="top" rowspan="1" colspan="1">Carboxylic acid biosynthetic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">7.87E‐04</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0016053</td>
<td align="center" valign="top" rowspan="1" colspan="1">Organic acid biosynthetic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">9.04E‐04</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0006631</td>
<td align="center" valign="top" rowspan="1" colspan="1">Fatty acid metabolic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.10E‐03</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0006629</td>
<td align="center" valign="top" rowspan="1" colspan="1">Lipid metabolic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.14E‐03</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:1901568</td>
<td align="center" valign="top" rowspan="1" colspan="1">Fatty acid derivative metabolic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.38E‐03</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0072524</td>
<td align="center" valign="top" rowspan="1" colspan="1">Pyridine‐containing compound metabolic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.75E‐03</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0006098</td>
<td align="center" valign="top" rowspan="1" colspan="1">Pentose‐phosphate shunt</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">4.74E‐03</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0051156</td>
<td align="center" valign="top" rowspan="1" colspan="1">Glucose 6‐phosphate metabolic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">4.74E‐03</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0006740</td>
<td align="center" valign="top" rowspan="1" colspan="1">NADPH regeneration</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">4.74E‐03</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0005975</td>
<td align="center" valign="top" rowspan="1" colspan="1">Carbohydrate metabolic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.15E‐02</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0046496</td>
<td align="center" valign="top" rowspan="1" colspan="1">Nicotinamide nucleotide metabolic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.80E‐02</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">GO:0019362</td>
<td align="center" valign="top" rowspan="1" colspan="1">Pyridine nucleotide metabolic process</td>
<td align="center" valign="top" rowspan="1" colspan="1">BP</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.80E‐02</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">KEGG:01100</td>
<td align="center" valign="top" rowspan="1" colspan="1">Metabolic pathways</td>
<td align="center" valign="top" rowspan="1" colspan="1">KEGG</td>
<td align="center" valign="top" rowspan="1" colspan="1">5.81E‐04</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">KEGG:01212</td>
<td align="center" valign="top" rowspan="1" colspan="1">Fatty acid metabolism</td>
<td align="center" valign="top" rowspan="1" colspan="1">KEGG</td>
<td align="center" valign="top" rowspan="1" colspan="1">4.89E‐03</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">KEGG:01200</td>
<td align="center" valign="top" rowspan="1" colspan="1">Carbon metabolism</td>
<td align="center" valign="top" rowspan="1" colspan="1">KEGG</td>
<td align="center" valign="top" rowspan="1" colspan="1">9.49E‐03</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">KEGG:00030</td>
<td align="center" valign="top" rowspan="1" colspan="1">Pentose phosphate pathway</td>
<td align="center" valign="top" rowspan="1" colspan="1">KEGG</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.93E‐02</td>
</tr>
<tr>
<td align="left" valign="top" rowspan="1" colspan="1">KEGG:04146</td>
<td align="center" valign="top" rowspan="1" colspan="1">Peroxisome</td>
<td align="center" valign="top" rowspan="1" colspan="1">KEGG</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.95E‐02</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ece371846-tbl-0003/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="ece371846-note-0006"><p>
<em>Note:</em> The threshold selected for significant genes to be submitted to the GO term analysis was ±0.7 log<sub>2</sub>FC and <em>p</em>‐value ≤ 0.05.</p></div>
<div class="fn" id="ece371846-note-0007"><p>Abbreviations: BP, biological process; CC, cellular component; KEGG, Kyoto Encyclopedia of Genes and Genomes; MF, molecular function.</p></div>
</div></section></section><section id="ece371846-sec-0014"><h3 class="pmc_sec_title">3.6. r7 and r9 Inversion Polymorphisms Likely Alter Gene Expression</h3>
<p>To further test the effect of the r7 and r9 inversion polymorphisms on the gene expression, we divided samples based on inversion status rather than elevation. This left the dataset with four INV, four HET and four STD samples. We then analysed the expression levels of genes in the three different status the r7 inversion status as variable in the DESeq2 model. This was done to better understand the expression patterns of STD, INV and HET all together. DESeq2 yielded a total of four DEGs when STD was placed against INV with three of them (LOC413698, LOC552586, LOC552005) residing inside the r7 region (Figure <a href="#ece371846-fig-0005" class="usa-link">5</a>). While a comparison between STD and HET, and HET and INV did not yield any significant results regarding genes inside the r7 inversion. The expression levels of the three detected genes were plotted to see how the expression of RNA changes when inverted alleles are present (INV and HET). In case of LOC413698, HET and STD samples are not statistically different between each other, but they are significantly different from INV samples. In the case of LOC552586, HET samples have higher expression rates than STD individuals and are significantly different from them. INV samples show a similar pattern. Finally, in LOC552005 STD and HET samples are not statistically different, but both are statistically different from INV samples. It is truly remarkable that data from all three cases demonstrate a correlation between the presence or absence of the inverted allele and the expression rates of these three genes (Figure <a href="#ece371846-fig-0005" class="usa-link">5</a>; Table <a href="#ece371846-tbl-0004" class="usa-link">4</a>).</p>
<figure class="fig xbox font-sm" id="ece371846-fig-0005"><h4 class="obj_head">FIGURE 5.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><img class="graphic" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/dbfb/12367272/80a2d1a92903/ECE3-15-e71846-g004.jpg" loading="lazy" id="jats-graphic-11" height="889" width="709" alt="FIGURE 5"></p>
<div class="p text-right font-secondary"><a href="figure/ece371846-fig-0005/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<figcaption><p>Differential expression in different r7 and r9 inversion genotypes (STD= non‐inverted homozygous, HET = heterozygous, INV = inverted homozygous). Expression patterns of LOC413698 (A), LOC552586 (B), LOC552005 (C), located in the r7 inversion, and LOC409025 (D), located in the r9 inversion. This gene has only two inversion statuses due to the lack of HET individuals in the population. Colors define the inversion status, while shape defines the sampling elevation. Groups of samples flanked by the same number of stars are not statistically different given a Tukey's HSD test.</p></figcaption></figure><section class="tw xbox font-sm" id="ece371846-tbl-0004"><h4 class="obj_head">TABLE 4.</h4>
<div class="caption p"><p>Significant differentially expressed genes (DEGs) with a model accounting for inversion status only.</p></div>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<col align="left" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<col align="center" span="1">
<thead valign="bottom"><tr style="border-bottom:solid 1px #000000">
<th align="left" valign="bottom" rowspan="1" colspan="1">Gene</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Log<sub>2</sub>fc</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>p</em> (DESeq2)</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">Comparison</th>
<th align="center" valign="bottom" rowspan="1" colspan="1">
<em>p</em> (Tukey HSD)</th>
</tr></thead>
<tbody valign="top">
<tr>
<td align="left" rowspan="3" valign="top" colspan="1">LOC413698</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1">STD – INV</td>
<td align="center" valign="top" rowspan="1" colspan="1">6.58e‐04**</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">−1.582</td>
<td align="center" valign="top" rowspan="1" colspan="1">7.50e‐05**</td>
<td align="center" valign="top" rowspan="1" colspan="1">STD – HET</td>
<td align="center" valign="top" rowspan="1" colspan="1">3.19e‐02*</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1">HET – INV</td>
<td align="center" valign="top" rowspan="1" colspan="1">5.38e‐02</td>
</tr>
<tr>
<td align="left" rowspan="3" valign="top" colspan="1">LOC552586</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1">STD – INV</td>
<td align="center" valign="top" rowspan="1" colspan="1">4.75e‐05**</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">1.074</td>
<td align="center" valign="top" rowspan="1" colspan="1">9.35e‐03*</td>
<td align="center" valign="top" rowspan="1" colspan="1">STD – HET</td>
<td align="center" valign="top" rowspan="1" colspan="1">6.57e‐04**</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1">HET – INV</td>
<td align="center" valign="top" rowspan="1" colspan="1">9.31e‐02</td>
</tr>
<tr>
<td align="left" rowspan="3" valign="top" colspan="1">LOC552005</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1">STD – INV</td>
<td align="center" valign="top" rowspan="1" colspan="1">6.89e‐04**</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">2.263</td>
<td align="center" valign="top" rowspan="1" colspan="1">1.19e‐02*</td>
<td align="center" valign="top" rowspan="1" colspan="1">STD – HET</td>
<td align="center" valign="top" rowspan="1" colspan="1">6.79e‐02</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1">HET – INV</td>
<td align="center" valign="top" rowspan="1" colspan="1">2.68e‐02*</td>
</tr>
<tr>
<td align="left" rowspan="3" valign="top" colspan="1">LOC409025</td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1">N/A</td>
<td align="center" valign="top" rowspan="1" colspan="1">N/A</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1">1.051</td>
<td align="center" valign="top" rowspan="1" colspan="1">4.67e‐02*</td>
<td align="center" valign="top" rowspan="1" colspan="1">N/A</td>
<td align="center" valign="top" rowspan="1" colspan="1">N/A</td>
</tr>
<tr>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1"></td>
<td align="center" valign="top" rowspan="1" colspan="1">N/A</td>
<td align="center" valign="top" rowspan="1" colspan="1">N/A</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/ece371846-tbl-0004/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p"><div class="fn" id="ece371846-note-0008"><p>
<em>Note:</em> Tukey's Honestly Significance Difference (HSD) test was applied to understand weather all three groups, inverted homozygous (INV), non‐inverted homozygous (STD), and heterozygous (HET), where equal or different. A star (*) near the <em>p</em>‐value represent a significance level of <em>α</em> ≤ 0.05, two stars near the <em>p</em>‐value represent a significance level of <em>α</em> ≤ 0.001. N/A indicates that the comparison could not be performed due to the absence of HET individuals; therefore, only the DESeq2 analysis is reported for these cases.</p></div></div></section><p>The same comparison involving STD, HET, and INV haplotypes could not be performed on r9 due to a lack of heterozygous specimens, and only a comparison of STD against INV could be made. In this case, only LOC409025 (apolipoprotein D, log<sub>2</sub>FC = −1.051, <em>p</em> = 4.61e‐02) was found to be significant; however, this is a very important result since it shows that r9 could potentially alter gene expression (Figure <a href="#ece371846-fig-0005" class="usa-link">5</a>; Table <a href="#ece371846-tbl-0004" class="usa-link">4</a>).</p>
<p>A comparison was conducted between the results of the selection scan and the DEGs to identify commonalities. This revealed that five genes were shared between the two datasets. LOC551123 is an RNA‐binding protein Musashi which in <em>Drosophila</em> positively regulates neural differentiation and negatively regulates the hypoxia‐inducible factor (HIF) pathway. Furthermore, it is involved in the determination of cell fate and the cellular response during normoxic and hypoxic conditions (Bertolin et al. <a href="#ece371846-bib-0010" class="usa-link" aria-describedby="ece371846-bib-0010">2016</a>; Landínez‐Macías and Urwyler <a href="#ece371846-bib-0056" class="usa-link" aria-describedby="ece371846-bib-0056">2021</a>). LOC413698 is the octopamine receptor beta‐1R which is involved in behavior and social division of labor in honey bees (Balfanz et al. <a href="#ece371846-bib-0006" class="usa-link" aria-describedby="ece371846-bib-0006">2013</a>; Rein et al. <a href="#ece371846-bib-0075" class="usa-link" aria-describedby="ece371846-bib-0075">2013</a>; Reim and Scheiner <a href="#ece371846-bib-0074" class="usa-link" aria-describedby="ece371846-bib-0074">2014</a>; Schilcher et al. <a href="#ece371846-bib-0085" class="usa-link" aria-describedby="ece371846-bib-0085">2021</a>). Furthermore, octopamine in honey bees is responsible for elevating appetite levels when the organism is hungry (Akülkü et al. <a href="#ece371846-bib-0001" class="usa-link" aria-describedby="ece371846-bib-0001">2021</a>). LOC408343 encodes a potassium voltage‐gated channel (Shaker protein) which in PC12 cells is involved in hypoxia sensitivity. Finally, LOC552844 encodes a complexin which is a SNARE protein responsible for the regulation of neurotransmission (Cho et al. <a href="#ece371846-bib-0022" class="usa-link" aria-describedby="ece371846-bib-0022">2014</a>). Among the previously described genes, LOC413698 is the only one located in the r7 region.</p></section></section><section id="ece371846-sec-0015"><h2 class="pmc_sec_title">4. Discussion</h2>
<p>The adaptation of 
<em>Apis mellifera</em>
 to high altitude encompasses a multitude of parameters. The high mountain environment presents a dual challenge. On one hand, certain behavioral adaptations are required, while on the other hand, different selection regimes are needed for specific genes in highland and lowland environments to enable the organisms to cope with it, such as tolerance to hypoxia or pathogens. The sequencing of 24 new honey bees collected on the Ugandan slope of Rwenzori Mountains highlights the presence of r7 and r9, two previously described inversion polymorphisms (Wallberg et al. <a href="#ece371846-bib-0097" class="usa-link" aria-describedby="ece371846-bib-0097">2017</a>; Christmas et al. <a href="#ece371846-bib-0023" class="usa-link" aria-describedby="ece371846-bib-0023">2018</a>), which display high frequency in highland populations. This finding suggests that these polymorphisms could be present on other East African mountain systems that have not yet been studied. Moreover, evidence indicates that <em>A.m. monticola</em> honey bees collected on Rwenzori Mountains r7 and r9 regions genetically drifted from those collected in Kenya. The frequency of the inversions in the highland suggests, however, that both r7 and r9 capture adaptive alleles for highland adaptation. The low gene flow and genetic drift contributed to shaping the genetic diversity between the two ~800 km away sampling areas. An <em>F</em>
<sub>ST</sub> scan conducted using only INV individuals from the Rwenzori Mountains population and Kenyan populations revealed a high number of highly differentiated SNPs; this was confirmed also through a sliding window Dxy (Figure <a href="#ece371846-fig-0002" class="usa-link">2A,B</a>). In many instances, these mutations are fixed in one of the populations and are absent or present at very low frequency in the other.</p>
<p>We conducted an in‐depth analysis of the newly described region on chromosome 5, and found a high level of differentiation (Dxy) between highland and lowland populations. Nucleotide diversity (<em>π</em>) was generally low in lowland populations, with the exception of RWLL. However, this exception may be attributable to sample size limitations (Figure <a href="#ece371846-fig-0003" class="usa-link">3A,B</a>). We could also observe high levels of heterozygosity and LD in the region, suggesting the potential involvement of a structural variant. However, we have not been able to define it. Finally, given the high level of heterozygosity in the highland population, we tested for balancing selection, and we found evidence for it in the highland population (Figure <a href="#ece371846-supitem-0003" class="usa-link">S3</a>), which is not evident in the lowland, suggesting that the heterozygous individuals can benefit from it in highland environments. This result could benefit from additional data from more populations, including an increased sample size.</p>
<p>The selection scan analysis proved inadequate in detecting significant and crucial GO terms in the Mau and Mt. Kenya regions, with only a few terms observed in the Rwenzori mountain population, predominantly involved in G‐protein coupled receptor activity, memory, learning, and cognition (Table <a href="#ece371846-supitem-0005" class="usa-link">S7</a>). An exceptional finding is that r7 and r9 were consistently detected in the analysis of the three populations. An eye‐catching result is that Octopamine receptors were ranked by the algorithm among the genes with the highest XP‐nSL values in all three populations, supporting the view of one central likely candidate of selection in highland environments. Octopamine receptors are G‐protein coupled receptors and were already pointed out as a potential candidate for adaptation to high elevation (Wallberg et al. <a href="#ece371846-bib-0097" class="usa-link" aria-describedby="ece371846-bib-0097">2017</a>). Their role has been extensively studied in honey bees in relation to a range of behavioral traits, including division of labor, appetite regulation, memory formation, learning, and thermogenesis (Kaya‐Zeeb et al. <a href="#ece371846-bib-0051" class="usa-link" aria-describedby="ece371846-bib-0051">2022</a>). Although octopamine, the neurotransmitter associated with these receptors, plays a crucial role in these behaviors, a clear association with adaptation to high elevation in honey bees has yet to be established. Studies on <em>Locusta migratoria</em> suggested that octopamine is important for the correct transmission of signals during hypoxic conditions (Money et al. <a href="#ece371846-bib-0066" class="usa-link" aria-describedby="ece371846-bib-0066">2016</a>).</p>
<p>Despite the detection of genes that could play a fundamental role in adaptation to high elevation such as G protein‐coupled receptors, which are also involved in hypoxia tolerance in certain human tissues, no direct connection was found between our results and the three main genes involved in the hypoxia initiation pathway (Azevedo et al. <a href="#ece371846-bib-0004" class="usa-link" aria-describedby="ece371846-bib-0004">2011</a>). AmSima (LOC408852), AmTango (LOC411234) and AmFatiga (LOC413929).</p>
<p>RNA‐seq was employed to ascertain the preferred pathways of mountain bees in comparison to lowland savannah honey bees. In total, 12 individual transcriptomes were compared; 29 downregulated and 113 upregulated genes were detected with an adjusted <em>p</em>‐value of 0.05 and a log<sub>2</sub>FC of ±0.7. The GO terms analysis of downregulated genes did not reveal any significant GO term, despite the presence of one of the previously suspected candidate genes (LOC413698, octopamine receptor beta 1). On the other hand, the GO term analysis on upregulated genes revealed an enrichment of different terms that were already associated with hypoxic environments. Notably, the enrichment for oxidoreductase activity (which includes oxidoreductase activity acting on the aldehyde or oxo group of donors, NAD or NADP as acceptor) and the carbohydrate metabolic process were detected in 
<em>Tribolium castaneum</em>
 when exposed to very low oxygen concentration (Wang et al. <a href="#ece371846-bib-0098" class="usa-link" aria-describedby="ece371846-bib-0098">2018</a>). In particular, numerous genes involved in the carbohydrate pathway were identified, the majority of which were associated with glycolysis. All these genes exhibited increased expression in the highland population, suggesting that these proteins may play a pivotal role in optimizing energy consumption in non‐normoxic conditions. We found log<sub>2</sub>FC differences in proteins involved in the carbohydrate pathway ranging from 0.43–0.55 (LOC551103: probable pyruvate dehydrogenase, Pyruvate Kinase: LOC552007 and Enolase: LOC552678) to 1.58 (LOC411202: 1,5‐anhydro‐D‐fructose reductase). We hypothesize that honey bees analyzed in this dataset are not experiencing extreme hypoxia; however, they could be adapted to lower levels of oxygen with respect to honey bees observed in the lowland savannah. Many GO terms we observed are associated in some way with response to a lower level of oxygen availability. We suggest that the adaptation observed among the highland population involves an increased expression of such pathways, albeit not to the same extent as in other populations or laboratory experiments (Wang et al. <a href="#ece371846-bib-0098" class="usa-link" aria-describedby="ece371846-bib-0098">2018</a>). Other notable genes detected include Hsp70‐4, which is upregulated in highland honey bees. This gene encodes for a heat shock protein cognate 4, which is involved in low oxygen tolerance in numerous species (Wen et al. <a href="#ece371846-bib-1005" class="usa-link" aria-describedby="ece371846-bib-1005">2021</a>) and longevity (KEGG: ame04213); and LOC411569, which encodes for a TANC2‐like protein. A distinguished role of this protein is that it functions as a signal for the inhibition of the mTOR pathway in mammals (Kim et al. <a href="#ece371846-bib-0052" class="usa-link" aria-describedby="ece371846-bib-0052">2021</a>). The mTOR pathway inhibition is caused by hypoxia, and the gene encoding for mTOR in honey bees has been found to be under selection in highland Africanized bee populations (Everitt et al. <a href="#ece371846-bib-0030" class="usa-link" aria-describedby="ece371846-bib-0030">2023</a>). In the present study, it was not possible to ascertain a definitive involvement of hypoxia‐related factors in this context. This is likely due to the absence of extreme hypoxia conditions; however, an adaptation to high elevation habitats was clear through the upregulation of genes involved in pathways aimed at mitigating the effects of low oxygen levels. A significant proportion of studies where very high expression levels of hypoxia factors are observed involve organisms exposed to very low oxygen concentrations (Azevedo et al. <a href="#ece371846-bib-0004" class="usa-link" aria-describedby="ece371846-bib-0004">2011</a>; Wang et al. <a href="#ece371846-bib-0098" class="usa-link" aria-describedby="ece371846-bib-0098">2018</a>).</p>
<p>Interestingly, we observed an enrichment of DEGs in KEGG:04146, representing the peroxisomes pathway. Peroxisomes are intracellular organelles involved in a multitude of metabolic processes such as fatty acid oxidation, lipid synthesis, and reactive oxygen species (ROS) metabolism. Additionally, they serve as a barrier against hypoxic stress (Jain et al. <a href="#ece371846-bib-0048" class="usa-link" aria-describedby="ece371846-bib-0048">2020</a>; He et al. <a href="#ece371846-bib-0045" class="usa-link" aria-describedby="ece371846-bib-0045">2021</a>).</p>
<p>Finally, we identified in DEGs some genes associated with the cuticular hydrocarbons (CHC) pathway. CHCs are vital molecules for honey bees, facilitating communication and preventing desiccation. They help honey bees recognize one another, and to a certain extent, they are specific to each hive. Distinct patterns of CHC exist in multiple species. High altitude natal fruit flies (
<em>Ceratitis rosa</em>
) exhibit markedly disparate CHC profiles between highland and lowland populations; the same is valid for Himalayan bumble bees, which have different CHC compositions based on elevational gradient (Vaníčková et al. <a href="#ece371846-bib-0094" class="usa-link" aria-describedby="ece371846-bib-0094">2015</a>; Narah et al. <a href="#ece371846-bib-0067" class="usa-link" aria-describedby="ece371846-bib-0067">2024</a>). Our analysis identified a notable number of genes with differentially expressed patterns implicated in the CHC pathway. LOC412166 is a fatty acyl‐CoA reductase involved in the CHC pathway. We also observed LOC412166 and LOC552417, which both encode for an acyl‐CoA Delta(11) desaturase. LOC725031 encodes for an elongation protein; these classes of proteins are crucial to determine the length of the fatty acid that will compose the final CHC profile. LOC412815 and FAR1 encode for a fatty acid synthase and a fatty acyl‐CoA reductase 1 and are central genes in the formation of CHCs.</p>
<p>Following a thorough analysis, no evidence was found to indicate different gene expression patterns within the r7 and r9 inversion polymorphisms, except for a few genes. Two genes were significantly downregulated: LOC413698 (Octopamine receptor beta 1 log<sub>2</sub>FC = −1.03, <em>p</em> = 0.03) and LOC726656 (Ca2+ independent phospholipase A2 gamma log<sub>2</sub>FC = −0.63, <em>p</em> = 0.04). Regarding r9, no gene was identified as significantly differentially expressed. Subsequently, an in‐depth investigation was conducted to ascertain the impact of r7 and r9 inversion on gene expression. The data were analyzed using the r7 inversion status as variable in the DESeq2 model which lead to the detection of expression differences in three genes when STD and INV were compared LOC413698 (octopamine receptor beta 1 log<sub>2</sub>FC = −1.582, <em>p</em> = 7.50e‐05), LOC552586 (cytochrome oxidase assembly factor 7 homolog log<sub>2</sub>FC = 1.074, <em>p</em> = 9.0e‐03) and LOC552005 (uncharacterized log<sub>2</sub>FC = 1.017, <em>p</em> = 1.19e‐02). For r9, we were only able to compare INV and STD individuals due to a lack of sufficient HET samples. Given the evident limitation we could however extract a very interesting result, in which we could observe LOC409025 as differentially expressed (log<sub>2</sub>FC = −1.051, <em>p</em> = 4.61e‐02). This gene encodes for a Apolipoprotein D, which was already described as protein involved in stress response in insects (Zhou et al. <a href="#ece371846-bib-0103" class="usa-link" aria-describedby="ece371846-bib-0103">2020</a>). Together with the other genes could be a target for adaptation mechanism to high elevational habitats. However, this result may be subject to bias due to the potential influence of environmental factors acting directly on gene expression. The extent of this possible bias needs further exploration to understand its impact on our findings. To gain a more comprehensive understanding of the genes whose expression levels are altered by these factors, it is essential to extend the sampling and sequencing of INV, HET and STD individuals from a single elevation, thereby reducing the impact of environmental variables. Together with the detection of the r7 and r9 inversions and the RNA‐seq study performed here in this study we could clearly observe divergent selection acting in highland environments in the two inversion polymorphism. Nevertheless, it remains unclear whether <em>A. m. monticola</em> represents genuine adaptations to high‐elevation habitats or if their ability to survive in such environments is due to the high degree of plasticity exhibited by 
<em>Apis mellifera</em>
. Addressing this question is challenging but crucial. Future studies should consider translocation and gene expression experiments, relocating lowland <em>A. m. scutellata</em> bees to the highland mountain forest environment and highland <em>A. m. monticola</em> bees to the lowland savannah environment. Additionally, evaluating the distribution and frequency of r7 and r9 in other East African mountain systems (e.g., Mt. Kilimanjaro) will help determine the significance of these chromosomal rearrangements and the genes they encompass. Such studies will not only enhance our understanding of bee adaptation but also inform conservation strategies in the face of environmental changes.</p></section><section id="ece371846-sec-0020"><h2 class="pmc_sec_title">Author Contributions</h2>
<p>
<strong>Victor Sebastian Scharnhorst:</strong> resources (supporting), writing – review and editing (supporting). <strong>Ricarda Scheiner:</strong> conceptualization (lead), funding acquisition (lead), project administration (lead), supervision (lead), writing – review and editing (equal). <strong>Martin Hasselmann:</strong> conceptualization (lead), funding acquisition (lead), project administration (lead), supervision (lead), writing – original draft (equal), writing – review and editing (equal). <strong>Florian Loidolt:</strong> data curation (supporting), writing – review and editing (equal). <strong>Philipp Meyer:</strong> resources (supporting), writing – review and editing (supporting). <strong>Sonja Kersten:</strong> data curation (equal), resources (equal), software (equal), writing – review and editing (supporting). <strong>Patrick Vudriko:</strong> writing – review and editing (supporting). <strong>Deborah Ruth Amulen:</strong> writing – review and editing (supporting). <strong>Marco Mazzoni:</strong> data curation (equal), formal analysis (lead), visualization (lead), writing – original draft (lead), writing – review and editing (lead).</p></section><section id="ece371846-sec-0019"><h2 class="pmc_sec_title">Conflicts of Interest</h2>
<p>The authors declare no conflicts of interest.</p></section><section id="sec18"><h2 class="pmc_sec_title">Supporting information</h2>
<section class="sm xbox font-sm" id="ece371846-supitem-0001"><div class="caption p"><p>
<strong>Figure S1:</strong> Genome wide and pi scan of analyzed population using a sliding window of 10 kb.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367272/bin/ECE3-15-e71846-s003.png" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71846-s003.png</a><sup> (2.8MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="ece371846-supitem-0002"><div class="caption p"><p>
<strong>Figure S2:</strong> Admixture cross‐validation error from <em>K</em> = 1 to <em>K</em> = 6 for five different random seeds.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367272/bin/ECE3-15-e71846-s004.png" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71846-s004.png</a><sup> (40.2KB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="ece371846-supitem-0003"><div class="caption p"><p>
<strong>Figure S3:</strong> Beta‐values computed for highland and lowland populations with a set threshold of 2 using a 150 SNP window.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367272/bin/ECE3-15-e71846-s001.png" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71846-s001.png</a><sup> (190.6KB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="ece371846-supitem-0004"><div class="caption p"><p>
<strong>Figure S4:</strong> Selection scan XP‐nSL values for the three populations analyzed: Mau (Top), Mt.Kenya (Center) and Rwenzori Mountains (Bottom). The black line shows sliding window values of 10 kb. Red dots represent potential SNPs involved in high elevation adaptation, while blue dots represent potential SNPs involved in lowland adaptation.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367272/bin/ECE3-15-e71846-s002.png" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71846-s002.png</a><sup> (1.2MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="ece371846-supitem-0005"><div class="caption p">
<p>
<strong>Table S1:</strong> Uganda elevation collection and coordinates.</p>
<p>
<strong>Table S2:</strong> ece371846‐sup‐0005‐Tables.xlsx. Samples included in the study and relatives coordinates of collection (where it was described in the original paper).</p>
<p>
<strong>Table S3:</strong> ece371846‐sup‐0005‐Tables.xlsx. Admixture cross‐validation results for 5 different random seeds for K going from 1 to 6.</p>
<p>
<strong>Table S4:</strong> ece371846‐sup‐0005‐Tables.xlsx. MKT results for genes inside the r7 inverted region.</p>
<p>
<strong>Table S5:</strong> ece371846‐sup‐0005‐Tables.xlsx. MKT results for genes inside the r9 inverted region.</p>
<p>
<strong>Table S6:</strong> ece371846‐sup‐0005‐Tables.xlsx. Genes falling inside 1% windows in Mau region.</p>
<p>
<strong>Table S7:</strong> ece371846‐sup‐0005‐Tables.xlsx. Genes falling inside 1% windows in Mt. Kenya region.</p>
<p>
<strong>Table S8:</strong> ece371846‐sup‐0005‐Tables.xlsx. Genes falling inside 1% windows in Rwenzori mountain region.</p>
<p>
<strong>Table S9:</strong> ece371846‐sup‐0005‐Tables.xlsx. GO terms for selection scan analysis of Rwenzori region population.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367272/bin/ECE3-15-e71846-s005.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71846-s005.xlsx</a><sup> (75.3KB, xlsx) </sup>
</div></div></section></section><section id="ece371846-sec-0016" class="ack"><h2 class="pmc_sec_title">Acknowledgments</h2>
<p>We would like to thank Mark Ojungu for the incredible help he gave us in scouting and collecting honey bee samples, the UWA rangers who accompanied us on the field trip as well as the whole UWA organization for supporting the research project and helping us plan the field trip. Finally, we would like to thank the local community of porters in Kasese and the surrounding villages for the huge help they provided us during the whole excursion by carrying equipment and sampling material inside the Rwenzori mountains national park. We thank Stephen Wade Schaeffer for valuable comments on the manuscript. This research was supported by grants from the Deutsche Forschungsgemeinschaft to M.H. and R.S. (HA5499/11‐1; SCHE 1573/14‐1). Open Access funding enabled and organized by Projekt DEAL.</p></section><section id="notes1"><p>


Mazzoni, M.
, Loidolt F., Kersten S., et al. 2025. “Genomic Landscape of High‐Altitude Adaptation in East African Mountain Honey Bees (
<em>Apis mellifera</em>
).” Ecology and Evolution
15, no. 8: e71846. 10.1002/ece3.71846.
</p>
<section id="ece371846-ntgp-0001" class="fn-group"><div class="fn-group p font-secondary-light font-sm"><div class="fn p" id="ece371846-note-0001"><p>
<strong>Funding:</strong> This work was supported by Deutsche Forschungsgemeinschaft (HA‐5499/11‐1, SCHE 1573/14‐1).</p></div></div></section></section><section id="ece371846-sec-0018"><h2 class="pmc_sec_title">Data Availability Statement</h2>
<p>Illumina WGS reads of the newly sequenced 24 honey bees as well as RNASeq reads of the 12 honey bees used in this study are available at the NCBI SRA under the accession no. PRJNA1179698. The Figures <a href="#ece371846-supitem-0001" class="usa-link">[Link]</a>, <a href="#ece371846-supitem-0004" class="usa-link">[Link]</a> and Tables <a href="#ece371846-supitem-0005" class="usa-link">S1–S9</a> as well as the scripts used to perform the analysis and plot the graphs are available at <a href="https://github.com/MarcoMazzoni93/Uganda" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/MarcoMazzoni93/Uganda</a>.</p></section><section id="ece371846-bibl-0001" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ece371846-bibl-0001_sec2"><ol class="ref-list font-sm">
<li id="ece371846-bib-0001">
<cite>

Akülkü, İ.
, Ghanem S., Filiztekin E., Suwannapong G., and Mayack C.. 2021. “Age‐Dependent Honey Bee Appetite Regulation Is Mediated by Trehalose and Octopamine Baseline Levels.” Insects
12: 863.
</cite> [<a href="https://doi.org/10.3390/insects12100863" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8539172/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34680632/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Insects&amp;title=Age%E2%80%90Dependent%20Honey%20Bee%20Appetite%20Regulation%20Is%20Mediated%20by%20Trehalose%20and%20Octopamine%20Baseline%20Levels&amp;author=%C4%B0.%20Ak%C3%BClk%C3%BC&amp;author=S.%20Ghanem&amp;author=E.%20Filiztekin&amp;author=G.%20Suwannapong&amp;author=C.%20Mayack&amp;volume=12&amp;publication_year=2021&amp;pages=863&amp;pmid=34680632&amp;doi=10.3390/insects12100863&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0002">
<cite>

Anderson, A. R.
, Hoffmann A. A., and McKechnie S. W.. 2005. “Response to Selection for Rapid Chill‐Coma Recovery in <em>Drosophila melanogaster</em>: Physiology and Life‐History Traits.” Genetical Research
85: 15–22.
</cite> [<a href="https://doi.org/10.1017/s0016672304007281" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16089033/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetical%20Research&amp;title=Response%20to%20Selection%20for%20Rapid%20Chill%E2%80%90Coma%20Recovery%20in%20Drosophila%20melanogaster:%20Physiology%20and%20Life%E2%80%90History%20Traits&amp;author=A.%20R.%20Anderson&amp;author=A.%20A.%20Hoffmann&amp;author=S.%20W.%20McKechnie&amp;volume=85&amp;publication_year=2005&amp;pages=15-22&amp;pmid=16089033&amp;doi=10.1017/s0016672304007281&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0003">
<cite>

Andrews, S.
, Krueger F., Seconds‐Pichon A., et al. 2012. FastQC. Babraham Bioinformatics.</cite> [<a href="https://scholar.google.com/scholar_lookup?title=FastQC&amp;author=S.%20Andrews&amp;author=F.%20Krueger&amp;author=A.%20Seconds%E2%80%90Pichon&amp;publication_year=2012&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0004">
<cite>

Azevedo, S. V.
, Caranton O. A. M., de Oliveira T. L., and Hartfelder K.. 2011. “Differential Expression of Hypoxia Pathway Genes in Honey Bee (<em>Apis mellifera</em> L.) Caste Development.” Journal of Insect Physiology
57: 38–45.
</cite> [<a href="https://doi.org/10.1016/j.jinsphys.2010.09.004" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20887729/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Insect%20Physiology&amp;title=Differential%20Expression%20of%20Hypoxia%20Pathway%20Genes%20in%20Honey%20Bee%20(Apis%20mellifera%20L.)%20Caste%20Development&amp;author=S.%20V.%20Azevedo&amp;author=O.%20A.%20M.%20Caranton&amp;author=T.%20L.%20de%20Oliveira&amp;author=K.%20Hartfelder&amp;volume=57&amp;publication_year=2011&amp;pages=38-45&amp;pmid=20887729&amp;doi=10.1016/j.jinsphys.2010.09.004&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0006">
<cite>

Balfanz, S.
, Jordan N., Langenstück T., Breuer J., Bergmeier V., and Baumann A.. 2013. “Molecular, Pharmacological, and Signaling Properties of Octopamine Receptors From Honeybee (<em>Apis mellifera</em>) Brain.” Journal of Neurochemistry
129: 284–296.
</cite> [<a href="https://doi.org/10.1111/jnc.12619" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24266860/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Neurochemistry&amp;title=Molecular,%20Pharmacological,%20and%20Signaling%20Properties%20of%20Octopamine%20Receptors%20From%20Honeybee%20(Apis%20mellifera)%20Brain&amp;author=S.%20Balfanz&amp;author=N.%20Jordan&amp;author=T.%20Langenst%C3%BCck&amp;author=J.%20Breuer&amp;author=V.%20Bergmeier&amp;volume=129&amp;publication_year=2013&amp;pages=284-296&amp;pmid=24266860&amp;doi=10.1111/jnc.12619&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0007">
<cite>

Behrends, A.
, and Scheiner R.. 2012. “Octopamine Improves Learning in Newly Emerged Bees But Not in Old Foragers.” Journal of Experimental Biology
215, no. Pt 7: 1076–1083.
</cite> [<a href="https://doi.org/10.1242/jeb.063297" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22399652/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Biology&amp;title=Octopamine%20Improves%20Learning%20in%20Newly%20Emerged%20Bees%20But%20Not%20in%20Old%20Foragers&amp;author=A.%20Behrends&amp;author=R.%20Scheiner&amp;volume=215&amp;issue=Pt%207&amp;publication_year=2012&amp;pages=1076-1083&amp;pmid=22399652&amp;doi=10.1242/jeb.063297&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0009">
<cite>

Berdan, E. L.
, Flatt T., Kozak G. M., Lotterhos K. E., and Wielstra B.. 2022. “Genomic Architecture of Supergenes: Connecting Form and Function.” Philosophical Transactions of the Royal Society B: Biological Sciences
377: 20210192.</cite> [<a href="https://doi.org/10.1098/rstb.2021.0192" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9189501/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35694757/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Philosophical%20Transactions%20of%20the%20Royal%20Society%20B:%20Biological%20Sciences&amp;title=Genomic%20Architecture%20of%20Supergenes:%20Connecting%20Form%20and%20Function&amp;author=E.%20L.%20Berdan&amp;author=T.%20Flatt&amp;author=G.%20M.%20Kozak&amp;author=K.%20E.%20Lotterhos&amp;author=B.%20Wielstra&amp;volume=377&amp;publication_year=2022&amp;pages=20210192&amp;pmid=35694757&amp;doi=10.1098/rstb.2021.0192&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0010">
<cite>

Bertolin, A. P.
, Katz M. J., Yano M., et al. 2016. “Musashi Mediates Translational Repression of the Drosophila Hypoxia Inducible Factor.” Nucleic Acids Research
44: 7555–7567.
</cite> [<a href="https://doi.org/10.1093/nar/gkw372" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5027473/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27141964/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Musashi%20Mediates%20Translational%20Repression%20of%20the%20Drosophila%20Hypoxia%20Inducible%20Factor&amp;author=A.%20P.%20Bertolin&amp;author=M.%20J.%20Katz&amp;author=M.%20Yano&amp;volume=44&amp;publication_year=2016&amp;pages=7555-7567&amp;pmid=27141964&amp;doi=10.1093/nar/gkw372&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0011">
<cite>

Bigham, A.
, Bauchet M., Pinto D., et al. 2010. “Identifying Signatures of Natural Selection in Tibetan and Andean Populations Using Dense Genome Scan Data.” PLoS Genetics
6: e1001116.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1001116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2936536/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20838600/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&amp;title=Identifying%20Signatures%20of%20Natural%20Selection%20in%20Tibetan%20and%20Andean%20Populations%20Using%20Dense%20Genome%20Scan%20Data&amp;author=A.%20Bigham&amp;author=M.%20Bauchet&amp;author=D.%20Pinto&amp;volume=6&amp;publication_year=2010&amp;pages=e1001116&amp;pmid=20838600&amp;doi=10.1371/journal.pgen.1001116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0012">
<cite>

Bigham, A. W.

2016. “Genetics of Human Origin and Evolution: High‐Altitude Adaptations.” Current Opinion in Genetics and Development
41: 8–13.
</cite> [<a href="https://doi.org/10.1016/j.gde.2016.06.018" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5161537/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27501156/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Opinion%20in%20Genetics%20and%20Development&amp;title=Genetics%20of%20Human%20Origin%20and%20Evolution:%20High%E2%80%90Altitude%20Adaptations&amp;author=A.%20W.%20Bigham&amp;volume=41&amp;publication_year=2016&amp;pages=8-13&amp;pmid=27501156&amp;doi=10.1016/j.gde.2016.06.018&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0013">
<cite>

Bochdanovits, Z.
, and Jong G. d.. 2003. “Experimental Evolution in 
<em>Drosophila melanogaster</em>
: Interaction of Temperature and Food Quality Selection Regimes.” Evolution
57: 1829–1836.
</cite> [<a href="https://doi.org/10.1111/j.0014-3820.2003.tb00590.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/14503624/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=Experimental%20Evolution%20in%20Drosophila%20melanogaster%20:%20Interaction%20of%20Temperature%20and%20Food%20Quality%20Selection%20Regimes&amp;author=Z.%20Bochdanovits&amp;author=G.%20d.%20Jong&amp;volume=57&amp;publication_year=2003&amp;pages=1829-1836&amp;pmid=14503624&amp;doi=10.1111/j.0014-3820.2003.tb00590.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0014">
<cite>

Bolger, A. M.
, Lohse M., and Usadel B.. 2014. “Trimmomatic: A Flexible Trimmer for Illumina Sequence Data.” Bioinformatics
30: 2114–2120.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btu170" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4103590/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24695404/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Trimmomatic:%20A%20Flexible%20Trimmer%20for%20Illumina%20Sequence%20Data&amp;author=A.%20M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;volume=30&amp;publication_year=2014&amp;pages=2114-2120&amp;pmid=24695404&amp;doi=10.1093/bioinformatics/btu170&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0015">
<cite>

Bomblies, K.
, and Peichel C. L.. 2022. “Genetics of adaptation.” Proceedings of the National Academy of Sciences of the United States of America
119: e2122152119.
</cite> [<a href="https://doi.org/10.1073/pnas.2122152119" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9335183/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35858399/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Genetics%20of%20adaptation&amp;author=K.%20Bomblies&amp;author=C.%20L.%20Peichel&amp;volume=119&amp;publication_year=2022&amp;pages=e2122152119&amp;pmid=35858399&amp;doi=10.1073/pnas.2122152119&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0016">
<cite>

Bray, N. L.
, Pimentel H., Melsted P., and Pachter L.. 2016. “Near‐Optimal Probabilistic RNA‐Seq Quantification.” Nature Biotechnology
34: 525–527.</cite> [<a href="https://doi.org/10.1038/nbt.3519" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27043002/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Biotechnology&amp;title=Near%E2%80%90Optimal%20Probabilistic%20RNA%E2%80%90Seq%20Quantification&amp;author=N.%20L.%20Bray&amp;author=H.%20Pimentel&amp;author=P.%20Melsted&amp;author=L.%20Pachter&amp;volume=34&amp;publication_year=2016&amp;pages=525-527&amp;pmid=27043002&amp;doi=10.1038/nbt.3519&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0017">
<cite>

Browning, B. L.
, Tian X., Zhou Y., and Browning S. R.. 2021. “Fast Two‐Stage Phasing of Large‐Scale Sequence Data.” American Journal of Human Genetics
108: 1880–1890.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2021.08.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8551421/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34478634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=Fast%20Two%E2%80%90Stage%20Phasing%20of%20Large%E2%80%90Scale%20Sequence%20Data&amp;author=B.%20L.%20Browning&amp;author=X.%20Tian&amp;author=Y.%20Zhou&amp;author=S.%20R.%20Browning&amp;volume=108&amp;publication_year=2021&amp;pages=1880-1890&amp;pmid=34478634&amp;doi=10.1016/j.ajhg.2021.08.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0018">
<cite>

Browning, B. L.
, Zhou Y., and Browning S. R.. 2018. “A One‐Penny Imputed Genome From Next‐Generation Reference Panels.” American Journal of Human Genetics
103: 338–348.
</cite> [<a href="https://doi.org/10.1016/j.ajhg.2018.07.015" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6128308/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30100085/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=American%20Journal%20of%20Human%20Genetics&amp;title=A%20One%E2%80%90Penny%20Imputed%20Genome%20From%20Next%E2%80%90Generation%20Reference%20Panels&amp;author=B.%20L.%20Browning&amp;author=Y.%20Zhou&amp;author=S.%20R.%20Browning&amp;volume=103&amp;publication_year=2018&amp;pages=338-348&amp;pmid=30100085&amp;doi=10.1016/j.ajhg.2018.07.015&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0019">
<cite>

Byers, K. J. R. P.
, Xu S., and Schlüter P. M.. 2016. “Molecular Mechanisms of Adaptation and Speciation: Why Do We Need An Integrative Approach?” Molecular Ecology
26: 277–290.
</cite> [<a href="https://doi.org/10.1111/mec.13678" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27230590/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Molecular%20Mechanisms%20of%20Adaptation%20and%20Speciation:%20Why%20Do%20We%20Need%20An%20Integrative%20Approach?&amp;author=K.%20J.%20R.%20P.%20Byers&amp;author=S.%20Xu&amp;author=P.%20M.%20Schl%C3%BCter&amp;volume=26&amp;publication_year=2016&amp;pages=277-290&amp;pmid=27230590&amp;doi=10.1111/mec.13678&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0020">
<cite>

Cáceres, A.
, and González J. R.. 2015. “Following the Footprints of Polymorphic Inversions on SNP Data: From Detection to Association Tests.” Nucleic Acids Research
43: e53.
</cite> [<a href="https://doi.org/10.1093/nar/gkv073" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4417146/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25672393/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Following%20the%20Footprints%20of%20Polymorphic%20Inversions%20on%20SNP%20Data:%20From%20Detection%20to%20Association%20Tests&amp;author=A.%20C%C3%A1ceres&amp;author=J.%20R.%20Gonz%C3%A1lez&amp;volume=43&amp;publication_year=2015&amp;pages=e53&amp;pmid=25672393&amp;doi=10.1093/nar/gkv073&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0021">
<cite>

Chang, C. C.
, Chow C. C., Tellier L. C., Vattikuti S., Purcell S. M., and Lee J. J.. 2015. “Second‐Generation PLINK: Rising to the Challenge of Larger and Richer Datasets.” GigaScience
4: 7.
</cite> [<a href="https://doi.org/10.1186/s13742-015-0047-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4342193/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25722852/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GigaScience&amp;title=Second%E2%80%90Generation%20PLINK:%20Rising%20to%20the%20Challenge%20of%20Larger%20and%20Richer%20Datasets&amp;author=C.%20C.%20Chang&amp;author=C.%20C.%20Chow&amp;author=L.%20C.%20Tellier&amp;author=S.%20Vattikuti&amp;author=S.%20M.%20Purcell&amp;volume=4&amp;publication_year=2015&amp;pages=7&amp;pmid=25722852&amp;doi=10.1186/s13742-015-0047-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0022">
<cite>

Cho, R. W.
, Kümmel D., Li F., et al. 2014. “Genetic Analysis of the Complexin Trans‐Clamping Model For Cross‐Linking SNARE Complexes In Vivo.” Proceedings of the National Academy of Sciences of the United States of America
111: 10317–10322.
</cite> [<a href="https://doi.org/10.1073/pnas.1409311111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4104896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24982161/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Genetic%20Analysis%20of%20the%20Complexin%20Trans%E2%80%90Clamping%20Model%20For%20Cross%E2%80%90Linking%20SNARE%20Complexes%20In%C2%A0Vivo&amp;author=R.%20W.%20Cho&amp;author=D.%20K%C3%BCmmel&amp;author=F.%20Li&amp;volume=111&amp;publication_year=2014&amp;pages=10317-10322&amp;pmid=24982161&amp;doi=10.1073/pnas.1409311111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0023">
<cite>

Christmas, M. J.
, Wallberg A., Bunikis I., Olsson A., Wallerman O., and Webster M. T.. 2018. “Chromosomal Inversions Associated With Environmental Adaptation in Honeybees.” Molecular Ecology
28: 1358–1374.
</cite> [<a href="https://doi.org/10.1111/mec.14944" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30431193/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Chromosomal%20Inversions%20Associated%20With%20Environmental%20Adaptation%20in%20Honeybees&amp;author=M.%20J.%20Christmas&amp;author=A.%20Wallberg&amp;author=I.%20Bunikis&amp;author=A.%20Olsson&amp;author=O.%20Wallerman&amp;volume=28&amp;publication_year=2018&amp;pages=1358-1374&amp;pmid=30431193&amp;doi=10.1111/mec.14944&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0025">
<cite>

Danecek, P.
, Auton A., Abecasis G., et al. 2011. “The Variant Call Format and VCFtools.” Bioinformatics
27: 2156–2158.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btr330" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3137218/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21653522/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=The%20Variant%20Call%20Format%20and%20VCFtools&amp;author=P.%20Danecek&amp;author=A.%20Auton&amp;author=G.%20Abecasis&amp;volume=27&amp;publication_year=2011&amp;pages=2156-2158&amp;pmid=21653522&amp;doi=10.1093/bioinformatics/btr330&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0026">
<cite>

Danecek, P.
, Bonfield J. K., Liddle J., et al. 2021. “Twelve Years of SAMtools and BCFtools.” GigaScience
10: giab008.
</cite> [<a href="https://doi.org/10.1093/gigascience/giab008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7931819/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33590861/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=GigaScience&amp;title=Twelve%20Years%20of%20SAMtools%20and%20BCFtools&amp;author=P.%20Danecek&amp;author=J.%20K.%20Bonfield&amp;author=J.%20Liddle&amp;volume=10&amp;publication_year=2021&amp;pages=giab008&amp;pmid=33590861&amp;doi=10.1093/gigascience/giab008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0027">
<cite>

Dillon, M. E.

2006. “Into Thin Air: Physiology and Evolution of Alpine Insects.” Integrative and Comparative Biology
46: 49–61.
</cite> [<a href="https://doi.org/10.1093/icb/icj007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21672722/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Integrative%20and%20Comparative%20Biology&amp;title=Into%20Thin%20Air:%20Physiology%20and%20Evolution%20of%20Alpine%20Insects&amp;author=M.%20E.%20Dillon&amp;volume=46&amp;publication_year=2006&amp;pages=49-61&amp;pmid=21672722&amp;doi=10.1093/icb/icj007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0028">
<cite>

Dogantzis, K. A.
, Tiwari T., Conflitti I. M., et al. 2021. “Thrice out of Asia and the Adaptive Radiation of the Western Honey Bee.” Science Advances
7: eabj2151.
</cite> [<a href="https://doi.org/10.1126/sciadv.abj2151" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8641936/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34860547/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Science%20Advances&amp;title=Thrice%20out%20of%20Asia%20and%20the%20Adaptive%20Radiation%20of%20the%20Western%20Honey%20Bee&amp;author=K.%20A.%20Dogantzis&amp;author=T.%20Tiwari&amp;author=I.%20M.%20Conflitti&amp;volume=7&amp;publication_year=2021&amp;pages=eabj2151&amp;pmid=34860547&amp;doi=10.1126/sciadv.abj2151&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0030">
<cite>

Everitt, T.
, Wallberg A., Christmas M. J., et al. 2023. “The Genomic Basis of Adaptation to High Elevations in Africanized Honey Bees.” Genome Biology and Evolution
15: evad157.
</cite> [<a href="https://doi.org/10.1093/gbe/evad157" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10484329/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37625795/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology%20and%20Evolution&amp;title=The%20Genomic%20Basis%20of%20Adaptation%20to%20High%20Elevations%20in%20Africanized%20Honey%20Bees&amp;author=T.%20Everitt&amp;author=A.%20Wallberg&amp;author=M.%20J.%20Christmas&amp;volume=15&amp;publication_year=2023&amp;pages=evad157&amp;pmid=37625795&amp;doi=10.1093/gbe/evad157&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0031">
<cite>

Faria, R.
, and Navarro A.. 2010. “Chromosomal Speciation Revisited: Rearranging Theory With Pieces of Evidence.” Trends in Ecology and Evolution
25: 660–669.
</cite> [<a href="https://doi.org/10.1016/j.tree.2010.07.008" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20817305/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Ecology%20and%20Evolution&amp;title=Chromosomal%20Speciation%20Revisited:%20Rearranging%20Theory%20With%20Pieces%20of%20Evidence&amp;author=R.%20Faria&amp;author=A.%20Navarro&amp;volume=25&amp;publication_year=2010&amp;pages=660-669&amp;pmid=20817305&amp;doi=10.1016/j.tree.2010.07.008&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0032">
<cite>

Feder, J. L.
, and Nosil P.. 2009. “Chromosomal Inversion and Species Differences: When Are Genes Affecting Adaptive Divergence and Reproductive Isolation Expected to Reside Within Inversions?” Evolution
63: 3061–3075.
</cite> [<a href="https://doi.org/10.1111/j.1558-5646.2009.00786.x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19656182/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolution&amp;title=Chromosomal%20Inversion%20and%20Species%20Differences:%20When%20Are%20Genes%20Affecting%20Adaptive%20Divergence%20and%20Reproductive%20Isolation%20Expected%20to%20Reside%20Within%20Inversions?&amp;author=J.%20L.%20Feder&amp;author=P.%20Nosil&amp;volume=63&amp;publication_year=2009&amp;pages=3061-3075&amp;pmid=19656182&amp;doi=10.1111/j.1558-5646.2009.00786.x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0033">
<cite>

Franks, S. J.
, and Hoffmann A. A.. 2012. “Genetics of Climate Change Adaptation.” Annual Review of Genetics
46: 185–208.</cite> [<a href="https://doi.org/10.1146/annurev-genet-110711-155511" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22934640/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Genetics&amp;title=Genetics%20of%20Climate%20Change%20Adaptation&amp;author=S.%20J.%20Franks&amp;author=A.%20A.%20Hoffmann&amp;volume=46&amp;publication_year=2012&amp;pages=185-208&amp;pmid=22934640&amp;doi=10.1146/annurev-genet-110711-155511&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0034">
<cite>

Frazier, M.
, Woods H., and Harrison J.. 2001. “Interactive Effects of Rearing Temperature and Oxygen on the Development of <em>drosophila melanogaster</em>
.” Physiological and Biochemical Zoology
74: 641–650.
</cite> [<a href="https://doi.org/10.1086/322172" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/11517449/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Physiological%20and%20Biochemical%20Zoology&amp;title=Interactive%20Effects%20of%20Rearing%20Temperature%20and%20Oxygen%20on%20the%20Development%20of%20drosophila%20melanogaster&amp;author=M.%20Frazier&amp;author=H.%20Woods&amp;author=J.%20Harrison&amp;volume=74&amp;publication_year=2001&amp;pages=641-650&amp;pmid=11517449&amp;doi=10.1086/322172&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0035">
<cite>

Fuller, Z. L.
, Haynes G. D., Richards S., and Schaeffer S. W.. 2017. “Genomics of Natural Populations: Evolutionary Forces That Establish and Maintain Gene Arrangements in <em>Drosophila pseudoobscura</em>
.” Molecular Ecology
26: 6539–6562.
</cite> [<a href="https://doi.org/10.1111/mec.14381" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29055159/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Genomics%20of%20Natural%20Populations:%20Evolutionary%20Forces%20That%20Establish%20and%20Maintain%20Gene%20Arrangements%20in%20Drosophila%20pseudoobscura&amp;author=Z.%20L.%20Fuller&amp;author=G.%20D.%20Haynes&amp;author=S.%20Richards&amp;author=S.%20W.%20Schaeffer&amp;volume=26&amp;publication_year=2017&amp;pages=6539-6562&amp;pmid=29055159&amp;doi=10.1111/mec.14381&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0036">
<cite>

Fuller, Z. L.
, Niño E. L., Patch H. M., et al. 2015. “Genome‐Wide Analysis of Signatures of Selection in Populations of African Honey Bees (<em>Apis mellifera</em>) Using New Web‐Based Tools.” BMC Genomics
16: 518.
</cite> [<a href="https://doi.org/10.1186/s12864-015-1712-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4496815/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26159619/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=Genome%E2%80%90Wide%20Analysis%20of%20Signatures%20of%20Selection%20in%20Populations%20of%20African%20Honey%20Bees%20(Apis%20mellifera)%20Using%20New%20Web%E2%80%90Based%20Tools&amp;author=Z.%20L.%20Fuller&amp;author=E.%20L.%20Ni%C3%B1o&amp;author=H.%20M.%20Patch&amp;volume=16&amp;publication_year=2015&amp;pages=518&amp;pmid=26159619&amp;doi=10.1186/s12864-015-1712-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0037">
<cite>

Gadau, J.
, and Fewell J. H.. 2022. “Supergenes, Supergenomes, and Complex Social Traits.” Proceedings of the National Academy of Sciences of the United States of America
119: e2118971118.
</cite> [<a href="https://doi.org/10.1073/pnas.2118971118" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8764664/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34987104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Supergenes,%20Supergenomes,%20and%20Complex%20Social%20Traits&amp;author=J.%20Gadau&amp;author=J.%20H.%20Fewell&amp;volume=119&amp;publication_year=2022&amp;pages=e2118971118&amp;pmid=34987104&amp;doi=10.1073/pnas.2118971118&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0038">
<cite>

Garrison, E.
, Kronenberg Z. N., Dawson E. T., Pedersen B. S., and Prins P.. 2022. “A Spectrum of Free Software Tools for Processing the VCF Variant Call Format: vcflib, bio‐vcf, cyvcf2, hts‐nim and slivar.” PLoS Computational Biology
18: e1009123.
</cite> [<a href="https://doi.org/10.1371/journal.pcbi.1009123" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9286226/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35639788/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Computational%20Biology&amp;title=A%20Spectrum%20of%20Free%20Software%20Tools%20for%20Processing%20the%20VCF%20Variant%20Call%20Format:%20vcflib,%20bio%E2%80%90vcf,%20cyvcf2,%20hts%E2%80%90nim%20and%20slivar&amp;author=E.%20Garrison&amp;author=Z.%20N.%20Kronenberg&amp;author=E.%20T.%20Dawson&amp;author=B.%20S.%20Pedersen&amp;author=P.%20Prins&amp;volume=18&amp;publication_year=2022&amp;pages=e1009123&amp;pmid=35639788&amp;doi=10.1371/journal.pcbi.1009123&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0040">
<cite>

Goncearenco, A.
, Ma B.‐G., and Berezovsky I. N.. 2013. “Molecular Mechanisms of Adaptation Emerging From the Physics and Evolution of Nucleic Acids and Proteins.” Nucleic Acids Research
42: 2879–2892.
</cite> [<a href="https://doi.org/10.1093/nar/gkt1336" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3950714/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24371267/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=Molecular%20Mechanisms%20of%20Adaptation%20Emerging%20From%20the%20Physics%20and%20Evolution%20of%20Nucleic%20Acids%20and%20Proteins&amp;author=A.%20Goncearenco&amp;author=B.%E2%80%90G.%20Ma&amp;author=I.%20N.%20Berezovsky&amp;volume=42&amp;publication_year=2013&amp;pages=2879-2892&amp;pmid=24371267&amp;doi=10.1093/nar/gkt1336&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0041">
<cite>

Greenberg, S.
, and Ar A.. 1996. “Effects of Chronic Hypoxia, Normoxia and Hyperoxia on Larval Development in the Beetle 
<em>Tenebrio molitor</em>

.” Journal of Insect Physiology
42: 991–996.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Insect%20Physiology&amp;title=Effects%20of%20Chronic%20Hypoxia,%20Normoxia%20and%20Hyperoxia%20on%20Larval%20Development%20in%20the%20Beetle%20Tenebrio%20molitor&amp;author=S.%20Greenberg&amp;author=A.%20Ar&amp;volume=42&amp;publication_year=1996&amp;pages=991-996&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0042">
<cite>

Gruber, K.
, Schöning C., Otte M., Kinuthia W., and Hasselmann M.. 2013. “Distinct Subspecies or Phenotypic Plasticity? Genetic and Morphological Differentiation of Mountain Honey Bees in East Africa.” Ecology and Evolution
3: 3204–3218.
</cite> [<a href="https://doi.org/10.1002/ece3.711" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3797471/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24223262/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology%20and%20Evolution&amp;title=Distinct%20Subspecies%20or%20Phenotypic%20Plasticity?%20Genetic%20and%20Morphological%20Differentiation%20of%20Mountain%20Honey%20Bees%20in%20East%20Africa&amp;author=K.%20Gruber&amp;author=C.%20Sch%C3%B6ning&amp;author=M.%20Otte&amp;author=W.%20Kinuthia&amp;author=M.%20Hasselmann&amp;volume=3&amp;publication_year=2013&amp;pages=3204-3218&amp;pmid=24223262&amp;doi=10.1002/ece3.711&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0043">
<cite>

Harpur, B. A.
, Kent C. F., Molodtsova D., et al. 2014. “Population Genomics of the Honey Bee Reveals Strong Signatures of Positive Selection on Worker Traits.” Proceedings of the National Academy of Sciences of the United States of America
111: 2614–2619.
</cite> [<a href="https://doi.org/10.1073/pnas.1315506111" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3932857/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24488971/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Population%20Genomics%20of%20the%20Honey%20Bee%20Reveals%20Strong%20Signatures%20of%20Positive%20Selection%20on%20Worker%20Traits&amp;author=B.%20A.%20Harpur&amp;author=C.%20F.%20Kent&amp;author=D.%20Molodtsova&amp;volume=111&amp;publication_year=2014&amp;pages=2614-2619&amp;pmid=24488971&amp;doi=10.1073/pnas.1315506111&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0044">
<cite>

Harringmeyer, O. S.
, and Hoekstra H. E.. 2022. “Chromosomal Inversion Polymorphisms Shape the Genomic Landscape of Deer Mice.” Nature Ecology and Evolution
6: 1965–1979.
</cite> [<a href="https://doi.org/10.1038/s41559-022-01890-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9715431/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36253543/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Ecology%20and%20Evolution&amp;title=Chromosomal%20Inversion%20Polymorphisms%20Shape%20the%20Genomic%20Landscape%20of%20Deer%20Mice&amp;author=O.%20S.%20Harringmeyer&amp;author=H.%20E.%20Hoekstra&amp;volume=6&amp;publication_year=2022&amp;pages=1965-1979&amp;pmid=36253543&amp;doi=10.1038/s41559-022-01890-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0045">
<cite>

He, A.
, Dean J. M., and Lodhi I. J.. 2021. “Peroxisomes as Cellular Adaptors to Metabolic and Environmental Stress.” Trends in Cell Biology
31: 656–670.
</cite> [<a href="https://doi.org/10.1016/j.tcb.2021.02.005" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8566112/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33674166/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Trends%20in%20Cell%20Biology&amp;title=Peroxisomes%20as%20Cellular%20Adaptors%20to%20Metabolic%20and%20Environmental%20Stress&amp;author=A.%20He&amp;author=J.%20M.%20Dean&amp;author=I.%20J.%20Lodhi&amp;volume=31&amp;publication_year=2021&amp;pages=656-670&amp;pmid=33674166&amp;doi=10.1016/j.tcb.2021.02.005&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0046">
<cite>

Höllinger, I.
, Pennings P. S., and Hermisson J.. 2019. “Polygenic Adaptation: From Sweeps to Subtle Frequency Shifts.” PLoS Genetics
15: e1008035.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1008035" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6443195/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30893299/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&amp;title=Polygenic%20Adaptation:%20From%20Sweeps%20to%20Subtle%20Frequency%20Shifts&amp;author=I.%20H%C3%B6llinger&amp;author=P.%20S.%20Pennings&amp;author=J.%20Hermisson&amp;volume=15&amp;publication_year=2019&amp;pages=e1008035&amp;pmid=30893299&amp;doi=10.1371/journal.pgen.1008035&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0048">
<cite>

Jain, I. H.
, Calvo S. E., Markhard A. L., et al. 2020. “Genetic Screen for Cell Fitness in High or Low Oxygen Highlights Mitochondrial and Lipid Metabolism.” Cell
181: 716–727.e11.
</cite> [<a href="https://doi.org/10.1016/j.cell.2020.03.029" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7293541/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32259488/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Genetic%20Screen%20for%20Cell%20Fitness%20in%20High%20or%20Low%20Oxygen%20Highlights%20Mitochondrial%20and%20Lipid%20Metabolism&amp;author=I.%20H.%20Jain&amp;author=S.%20E.%20Calvo&amp;author=A.%20L.%20Markhard&amp;volume=181&amp;publication_year=2020&amp;pages=716-727&amp;pmid=32259488&amp;doi=10.1016/j.cell.2020.03.029&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0049">
<cite>

Jones, F. C.
, Grabherr M. G., Chan Y. F., et al. 2012. “The Genomic Basis of Adaptive Evolution in Threespine Sticklebacks.” Nature
484: 55–61.
</cite> [<a href="https://doi.org/10.1038/nature10944" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3322419/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/22481358/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The%20Genomic%20Basis%20of%20Adaptive%20Evolution%20in%20Threespine%20Sticklebacks&amp;author=F.%20C.%20Jones&amp;author=M.%20G.%20Grabherr&amp;author=Y.%20F.%20Chan&amp;volume=484&amp;publication_year=2012&amp;pages=55-61&amp;pmid=22481358&amp;doi=10.1038/nature10944&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0051">
<cite>

Kaya‐Zeeb, S.
, Delac S., Wolf L., Marante A. L., Scherf‐Clavel O., and Thamm M.. 2022. “Robustness of the Honeybee Neuro‐Muscular Octopaminergic System in the Face of Cold Stress.” Frontiers in Physiology
13: 1002740.
</cite> [<a href="https://doi.org/10.3389/fphys.2022.1002740" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9551396/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36237520/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Physiology&amp;title=Robustness%20of%20the%20Honeybee%20Neuro%E2%80%90Muscular%20Octopaminergic%20System%20in%20the%20Face%20of%20Cold%20Stress&amp;author=S.%20Kaya%E2%80%90Zeeb&amp;author=S.%20Delac&amp;author=L.%20Wolf&amp;author=A.%20L.%20Marante&amp;author=O.%20Scherf%E2%80%90Clavel&amp;volume=13&amp;publication_year=2022&amp;pages=1002740&amp;pmid=36237520&amp;doi=10.3389/fphys.2022.1002740&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0052">
<cite>

Kim, S.‐G.
, Lee S., Kim Y., et al. 2021. “Tanc2‐Mediated mTOR Inhibition Balances mTORC1/2 Signaling in the Developing Mouse Brain and Human Neurons.” Nature Communications
12: 2695.</cite> [<a href="https://doi.org/10.1038/s41467-021-22908-4" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8113471/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33976205/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature%20Communications&amp;title=Tanc2%E2%80%90Mediated%20mTOR%20Inhibition%20Balances%20mTORC1/2%20Signaling%20in%20the%20Developing%20Mouse%20Brain%20and%20Human%20Neurons&amp;author=S.%E2%80%90G.%20Kim&amp;author=S.%20Lee&amp;author=Y.%20Kim&amp;volume=12&amp;publication_year=2021&amp;pages=2695&amp;pmid=33976205&amp;doi=10.1038/s41467-021-22908-4&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0053">
<cite>

Kolberg, L.
, Raudvere U., Kuzmin I., Adler P., Vilo J., and Peterson H.. 2023. “g:Profiler—Interoperable Web Service for Functional Enrichment Analysis and Gene Identifier Mapping (2023 Update).” Nucleic Acids Research
51: W207–W212.
</cite> [<a href="https://doi.org/10.1093/nar/gkad347" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10320099/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37144459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic%20Acids%20Research&amp;title=g:Profiler%E2%80%94Interoperable%20Web%20Service%20for%20Functional%20Enrichment%20Analysis%20and%20Gene%20Identifier%20Mapping%20(2023%20Update)&amp;author=L.%20Kolberg&amp;author=U.%20Raudvere&amp;author=I.%20Kuzmin&amp;author=P.%20Adler&amp;author=J.%20Vilo&amp;volume=51&amp;publication_year=2023&amp;pages=W207-W212&amp;pmid=37144459&amp;doi=10.1093/nar/gkad347&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0054">
<cite>

Korunes, K. L.
, and Noor M. A. F.. 2018. “Pervasive Gene Conversion in Chromosomal Inversion Heterozygotes.” Molecular Ecology
28: 1302–1315.
</cite> [<a href="https://doi.org/10.1111/mec.14921" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6475484/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30387889/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=Pervasive%20Gene%20Conversion%20in%20Chromosomal%20Inversion%20Heterozygotes&amp;author=K.%20L.%20Korunes&amp;author=M.%20A.%20F.%20Noor&amp;volume=28&amp;publication_year=2018&amp;pages=1302-1315&amp;pmid=30387889&amp;doi=10.1111/mec.14921&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0055">
<cite>

Korunes, K. L.
, and Samuk K.. 2021. “pixy: Unbiased Estimation of Nucleotide Diversity and Divergence in the Presence of Missing Data.” Molecular Ecology Resources
21: 1359–1368.
</cite> [<a href="https://doi.org/10.1111/1755-0998.13326" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8044049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33453139/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology%20Resources&amp;title=pixy:%20Unbiased%20Estimation%20of%20Nucleotide%20Diversity%20and%20Divergence%20in%20the%20Presence%20of%20Missing%20Data&amp;author=K.%20L.%20Korunes&amp;author=K.%20Samuk&amp;volume=21&amp;publication_year=2021&amp;pages=1359-1368&amp;pmid=33453139&amp;doi=10.1111/1755-0998.13326&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0056">
<cite>

Landínez‐Macías, M.
, and Urwyler O.. 2021. “The Fine Art of Writing a Message: RNA Metabolism in the Shaping and Remodeling of the Nervous System.” Frontiers in Molecular Neuroscience
14: 755686.
</cite> [<a href="https://doi.org/10.3389/fnmol.2021.755686" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8670310/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34916907/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Molecular%20Neuroscience&amp;title=The%20Fine%20Art%20of%20Writing%20a%20Message:%20RNA%20Metabolism%20in%20the%20Shaping%20and%20Remodeling%20of%20the%20Nervous%20System&amp;author=M.%20Land%C3%ADnez%E2%80%90Mac%C3%ADas&amp;author=O.%20Urwyler&amp;volume=14&amp;publication_year=2021&amp;pages=755686&amp;pmid=34916907&amp;doi=10.3389/fnmol.2021.755686&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0058">
<cite>

Li, H.
, and Durbin R.. 2009. “Fast and Accurate Short Read Alignment With Burrows–Wheeler Transform.” Bioinformatics
25: 1754–1760.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btp324" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2705234/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/19451168/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=Fast%20and%20Accurate%20Short%20Read%20Alignment%20With%20Burrows%E2%80%93Wheeler%20Transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;volume=25&amp;publication_year=2009&amp;pages=1754-1760&amp;pmid=19451168&amp;doi=10.1093/bioinformatics/btp324&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0060">
<cite>

Lin, M.
, Whitmire S., Chen J., Farrel A., Shi X., and Guo J. T.. 2017. “Effects of Short Indels on Protein Structure and Function in Human Genomes.” Scientific Reports
7: 9313.
</cite> [<a href="https://doi.org/10.1038/s41598-017-09287-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5570956/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28839204/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Scientific%20Reports&amp;title=Effects%20of%20Short%20Indels%20on%20Protein%20Structure%20and%20Function%20in%20Human%20Genomes&amp;author=M.%20Lin&amp;author=S.%20Whitmire&amp;author=J.%20Chen&amp;author=A.%20Farrel&amp;author=X.%20Shi&amp;volume=7&amp;publication_year=2017&amp;pages=9313&amp;pmid=28839204&amp;doi=10.1038/s41598-017-09287-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0061">
<cite>

Love, M. I.
, Huber W., and Anders S.. 2014. “Moderated Estimation of Fold Change and Dispersion for RNA‐Seq Data With DESeq2.” Genome Biology
15: 550.
</cite> [<a href="https://doi.org/10.1186/s13059-014-0550-8" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4302049/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Moderated%20Estimation%20of%20Fold%20Change%20and%20Dispersion%20for%20RNA%E2%80%90Seq%20Data%20With%20DESeq2&amp;author=M.%20I.%20Love&amp;author=W.%20Huber&amp;author=S.%20Anders&amp;volume=15&amp;publication_year=2014&amp;pages=550&amp;pmid=25516281&amp;doi=10.1186/s13059-014-0550-8&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0501">
<cite>

Lowry, D. B.
, and Willis J. H.. 2010. “A Widespread Chromosomal Inversion Polymorphism Contributes to a Major Life‐History Transition, Local Adaptation, and Reproductive Isolation.” PLoS Biology
8, no. 9: e1000500.
</cite> [<a href="https://doi.org/10.1371/journal.pbio.1000500" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2946948/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20927411/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Biology&amp;title=A%20Widespread%20Chromosomal%20Inversion%20Polymorphism%20Contributes%20to%20a%20Major%20Life%E2%80%90History%20Transition,%20Local%20Adaptation,%20and%20Reproductive%20Isolation&amp;author=D.%20B.%20Lowry&amp;author=J.%20H.%20Willis&amp;volume=8&amp;issue=9&amp;publication_year=2010&amp;pages=e1000500&amp;pmid=20927411&amp;doi=10.1371/journal.pbio.1000500&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0062">
<cite>

Mahecha, M. D.
, Bastos A., Bohn F. J., et al. 2022. “Biodiversity Loss and Climate Extremes — Study the Feedbacks.” Nature
612: 30–32.
</cite> [<a href="https://doi.org/10.1038/d41586-022-04152-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36446988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Biodiversity%20Loss%20and%20Climate%20Extremes%20%E2%80%94%20Study%20the%20Feedbacks&amp;author=M.%20D.%20Mahecha&amp;author=A.%20Bastos&amp;author=F.%20J.%20Bohn&amp;volume=612&amp;publication_year=2022&amp;pages=30-32&amp;pmid=36446988&amp;doi=10.1038/d41586-022-04152-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0063">
<cite>

Martin, M.

2011. “Cutadapt Removes Adapter Sequences From High‐Throughput Sequencing Reads.” EMBnet.Journal
17: 10.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=EMBnet.Journal&amp;title=Cutadapt%20Removes%20Adapter%20Sequences%20From%20High%E2%80%90Throughput%20Sequencing%20Reads&amp;author=M.%20Martin&amp;volume=17&amp;publication_year=2011&amp;pages=10&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0064">
<cite>

Martínez Sosa, F.
, and Pilot M.. 2023. “Molecular Mechanisms Underlying Vertebrate Adaptive Evolution: A Systematic Review.” Genes
14: 416.
</cite> [<a href="https://doi.org/10.3390/genes14020416" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9957108/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36833343/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genes&amp;title=Molecular%20Mechanisms%20Underlying%20Vertebrate%20Adaptive%20Evolution:%20A%20Systematic%20Review&amp;author=F.%20Mart%C3%ADnez%20Sosa&amp;author=M.%20Pilot&amp;volume=14&amp;publication_year=2023&amp;pages=416&amp;pmid=36833343&amp;doi=10.3390/genes14020416&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0065">
<cite>

Matur, F.
, Keskin E., and Sezgin E.. 2023. “Editorial: Population Genomics and Adaptation to Novel Environments: Challenges and Opportunities.” Frontiers in Ecology and Evolution
11: 1190218.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Ecology%20and%20Evolution&amp;title=Editorial:%20Population%20Genomics%20and%20Adaptation%20to%20Novel%20Environments:%20Challenges%20and%20Opportunities&amp;author=F.%20Matur&amp;author=E.%20Keskin&amp;author=E.%20Sezgin&amp;volume=11&amp;publication_year=2023&amp;pages=1190218&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-1004">
<cite>

Mirchandani, C. D.
, Shultz A. J., Thomas G. W. C., et al. 2024. “A Fast, Reproducible, High‐Throughput Variant Calling Workflow for Population Genomics.” Molecular Biology and Evolution
41, no. 1: msad270. 10.1093/molbev/msad270.
</cite> [<a href="https://doi.org/10.1093/molbev/msad270" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10764099/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38069903/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=A%20Fast,%20Reproducible,%20High%E2%80%90Throughput%20Variant%20Calling%20Workflow%20for%20Population%20Genomics&amp;author=C.%20D.%20Mirchandani&amp;author=A.%20J.%20Shultz&amp;author=G.%20W.%20C.%20Thomas&amp;volume=41&amp;issue=1&amp;publication_year=2024&amp;pages=msad270&amp;pmid=38069903&amp;doi=10.1093/molbev/msad270&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0066">
<cite>

Money, T. G. A.
, Sproule M. K. J., Cross K. P., and Robertson R. M.. 2016. “Octopamine Stabilizes Conduction Reliability of an Unmyelinated Axon During Hypoxic Stress.” Journal of Neurophysiology
116: 949–959.
</cite> [<a href="https://doi.org/10.1152/jn.00354.2016" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5009204/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27281750/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Neurophysiology&amp;title=Octopamine%20Stabilizes%20Conduction%20Reliability%20of%20an%20Unmyelinated%20Axon%20During%20Hypoxic%20Stress&amp;author=T.%20G.%20A.%20Money&amp;author=M.%20K.%20J.%20Sproule&amp;author=K.%20P.%20Cross&amp;author=R.%20M.%20Robertson&amp;volume=116&amp;publication_year=2016&amp;pages=949-959&amp;pmid=27281750&amp;doi=10.1152/jn.00354.2016&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0067">
<cite>

Narah, J.
, Streinzer M., Chakravorty J., et al. 2024. “Cuticular Hydrocarbon Profiles of Himalayan Bumble Bees (Hymenoptera: Bombus Latreille) Are Species‐Specific and Show Elevational Variation.” Journal of Chemical Ecology
50: 969–977.
</cite> [<a href="https://doi.org/10.1007/s10886-024-01486-x" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11717848/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38470528/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Chemical%20Ecology&amp;title=Cuticular%20Hydrocarbon%20Profiles%20of%20Himalayan%20Bumble%20Bees%20(Hymenoptera:%20Bombus%20Latreille)%20Are%20Species%E2%80%90Specific%20and%20Show%20Elevational%20Variation&amp;author=J.%20Narah&amp;author=M.%20Streinzer&amp;author=J.%20Chakravorty&amp;volume=50&amp;publication_year=2024&amp;pages=969-977&amp;pmid=38470528&amp;doi=10.1007/s10886-024-01486-x&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0068">
<cite>

Navarro, A.
, Betrán E., Barbadilla A., and Ruiz A.. 1997. “Recombination and Gene Flux Caused by Gene Conversion and Crossing Over in Inversion Heterokaryotypes.” Genetics
146: 695–709.
</cite> [<a href="https://doi.org/10.1093/genetics/146.2.695" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1208008/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/9178017/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Recombination%20and%20Gene%20Flux%20Caused%20by%20Gene%20Conversion%20and%20Crossing%20Over%20in%20Inversion%20Heterokaryotypes&amp;author=A.%20Navarro&amp;author=E.%20Betr%C3%A1n&amp;author=A.%20Barbadilla&amp;author=A.%20Ruiz&amp;volume=146&amp;publication_year=1997&amp;pages=695-709&amp;pmid=9178017&amp;doi=10.1093/genetics/146.2.695&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0070"><cite>

Poplin, R.
, Ruano‐Rubio V., DePristo M. A., et al. 2017. “Scaling Accurate Genetic Variant Discovery to Tens of Thousands of Samples.”
</cite></li>
<li id="ece371846-bib-0071">
<cite>

Puig, M.
, Cáceres M., and Ruiz A.. 2004. “Silencing of a Gene Adjacent to the Breakpoint of a Widespread Drosophila Inversion by a Transposon‐Induced Antisense RNA.” Proceedings of the National Academy of Sciences of the United States of America
101: 9013–9018.
</cite> [<a href="https://doi.org/10.1073/pnas.0403090101" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC428464/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15184654/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Silencing%20of%20a%20Gene%20Adjacent%20to%20the%20Breakpoint%20of%20a%20Widespread%20Drosophila%20Inversion%20by%20a%20Transposon%E2%80%90Induced%20Antisense%20RNA&amp;author=M.%20Puig&amp;author=M.%20C%C3%A1ceres&amp;author=A.%20Ruiz&amp;volume=101&amp;publication_year=2004&amp;pages=9013-9018&amp;pmid=15184654&amp;doi=10.1073/pnas.0403090101&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0072">
<cite>

Quinlan, A. R.
, and Hall I. M.. 2010. “BEDTools: A Flexible Suite of Utilities for Comparing Genomic Features.” Bioinformatics
26: 841–842.
</cite> [<a href="https://doi.org/10.1093/bioinformatics/btq033" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2832824/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20110278/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=BEDTools:%20A%20Flexible%20Suite%20of%20Utilities%20for%20Comparing%20Genomic%20Features&amp;author=A.%20R.%20Quinlan&amp;author=I.%20M.%20Hall&amp;volume=26&amp;publication_year=2010&amp;pages=841-842&amp;pmid=20110278&amp;doi=10.1093/bioinformatics/btq033&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0073">
<cite>

Reid, K.
, Bell M. A., and Veeramah K. R.. 2021. “Threespine Stickleback: A Model System For Evolutionary Genomics.” Annual Review of Genomics and Human Genetics
22: 357–383.</cite> [<a href="https://doi.org/10.1146/annurev-genom-111720-081402" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8415275/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33909459/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Genomics%20and%20Human%20Genetics&amp;title=Threespine%20Stickleback:%20A%20Model%20System%20For%20Evolutionary%20Genomics&amp;author=K.%20Reid&amp;author=M.%20A.%20Bell&amp;author=K.%20R.%20Veeramah&amp;volume=22&amp;publication_year=2021&amp;pages=357-383&amp;pmid=33909459&amp;doi=10.1146/annurev-genom-111720-081402&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0074">
<cite>

Reim, T.
, and Scheiner R.. 2014. “Division of Labour in Honey Bees: Age‐ and Task‐Related Changes in the Expression of Octopamine Receptor Genes.” Insect Molecular Biology
23: 833–841.
</cite> [<a href="https://doi.org/10.1111/imb.12130" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25187440/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Insect%20Molecular%20Biology&amp;title=Division%20of%20Labour%20in%20Honey%20Bees:%20Age%E2%80%90%20and%20Task%E2%80%90Related%20Changes%20in%20the%20Expression%20of%20Octopamine%20Receptor%20Genes&amp;author=T.%20Reim&amp;author=R.%20Scheiner&amp;volume=23&amp;publication_year=2014&amp;pages=833-841&amp;pmid=25187440&amp;doi=10.1111/imb.12130&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0075">
<cite>

Rein, J.
, Mustard J. A., Strauch M., Smith B. H., and Galizia C. G.. 2013. “Octopamine Modulates Activity of Neural Networks in the Honey Bee Antennal Lobe.” Journal of Comparative Physiology A
199: 947–962.</cite> [<a href="https://doi.org/10.1007/s00359-013-0805-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3825135/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/23681219/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Comparative%20Physiology%20A&amp;title=Octopamine%20Modulates%20Activity%20of%20Neural%20Networks%20in%20the%20Honey%20Bee%20Antennal%20Lobe&amp;author=J.%20Rein&amp;author=J.%20A.%20Mustard&amp;author=M.%20Strauch&amp;author=B.%20H.%20Smith&amp;author=C.%20G.%20Galizia&amp;volume=199&amp;publication_year=2013&amp;pages=947-962&amp;pmid=23681219&amp;doi=10.1007/s00359-013-0805-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0078">
<cite>

Rozas, J.
, and Aguadé M.. 1994. “Gene Conversion is Involved in the Transfer of Genetic Information Between Naturally Occurring Inversions of Drosophila.” Proceedings of the National Academy of Sciences of the United States of America
91: 11517–11521.
</cite> [<a href="https://doi.org/10.1073/pnas.91.24.11517" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC45262/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/7972094/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Gene%20Conversion%20is%20Involved%20in%20the%20Transfer%20of%20Genetic%20Information%20Between%20Naturally%20Occurring%20Inversions%20of%20Drosophila&amp;author=J.%20Rozas&amp;author=M.%20Aguad%C3%A9&amp;volume=91&amp;publication_year=1994&amp;pages=11517-11521&amp;pmid=7972094&amp;doi=10.1073/pnas.91.24.11517&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0079"><cite>

Ruttner, F.

1998. “Biogeography and Taxonomy of Honeybees.”
</cite></li>
<li id="ece371846-bib-0080">
<cite>

Schaeffer, S. W.
, and Anderson W. W.. 2005. “Mechanisms of Genetic Exchange Within the Chromosomal Inversions of <em>Drosophila pseudoobscura</em>
.” Genetics
171: 1729–1739.
</cite> [<a href="https://doi.org/10.1534/genetics.105.041947" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC1456099/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/16143625/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genetics&amp;title=Mechanisms%20of%20Genetic%20Exchange%20Within%20the%20Chromosomal%20Inversions%20of%20Drosophila%20pseudoobscura&amp;author=S.%20W.%20Schaeffer&amp;author=W.%20W.%20Anderson&amp;volume=171&amp;publication_year=2005&amp;pages=1729-1739&amp;pmid=16143625&amp;doi=10.1534/genetics.105.041947&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0081">
<cite>

Schaeffer, S. W.
, Goetting‐Minesky M. P., Kovacevic M., et al. 2003. “Evolutionary Genomics of Inversions in 
<em>Drosophila pseudoobscura</em>
: Evidence for Epistasis.” Proceedings of the National Academy of Sciences of the United States of America
100: 8319–8324.
</cite> [<a href="https://doi.org/10.1073/pnas.1432900100" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC166227/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12824467/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Proceedings%20of%20the%20National%20Academy%20of%20Sciences%20of%20the%20United%20States%20of%20America&amp;title=Evolutionary%20Genomics%20of%20Inversions%20in%20Drosophila%20pseudoobscura%20:%20Evidence%20for%20Epistasis&amp;author=S.%20W.%20Schaeffer&amp;author=M.%20P.%20Goetting%E2%80%90Minesky&amp;author=M.%20Kovacevic&amp;volume=100&amp;publication_year=2003&amp;pages=8319-8324&amp;pmid=12824467&amp;doi=10.1073/pnas.1432900100&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0082">
<cite>

Schaeffer, S. W.
, Richards S., and Fuller Z. L.. 2024. “Genomics of Natural Populations: Gene Conversion Events Reveal Selected Genes Within the Inversions of <em>Drosophila pseudoobscura</em>
.” G3: Genes, Genomes, Genetics
14: jkae176.
</cite> [<a href="https://doi.org/10.1093/g3journal/jkae176" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC11457094/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/39073776/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=G3:%20Genes,%20Genomes,%20Genetics&amp;title=Genomics%20of%20Natural%20Populations:%20Gene%20Conversion%20Events%20Reveal%20Selected%20Genes%20Within%20the%20Inversions%20of%20Drosophila%20pseudoobscura&amp;author=S.%20W.%20Schaeffer&amp;author=S.%20Richards&amp;author=Z.%20L.%20Fuller&amp;volume=14&amp;publication_year=2024&amp;pages=jkae176&amp;pmid=39073776&amp;doi=10.1093/g3journal/jkae176&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0083">
<cite>

Scheiner, R.
, Toteva A., Reim T., Søvik E., and Barron A. B.. 2014. “Differences in the Phototaxis of Pollen and Nectar Foraging Honey Bees Are Related to Their Octopamine Brain Titers.” Frontiers in Physiology
5: 116.
</cite> [<a href="https://doi.org/10.3389/fphys.2014.00116" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3975121/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/24734024/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Frontiers%20in%20Physiology&amp;title=Differences%20in%20the%20Phototaxis%20of%20Pollen%20and%20Nectar%20Foraging%20Honey%20Bees%20Are%20Related%20to%20Their%20Octopamine%20Brain%20Titers&amp;author=R.%20Scheiner&amp;author=A.%20Toteva&amp;author=T.%20Reim&amp;author=E.%20S%C3%B8vik&amp;author=A.%20B.%20Barron&amp;volume=5&amp;publication_year=2014&amp;pages=116&amp;pmid=24734024&amp;doi=10.3389/fphys.2014.00116&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0084">
<cite>

Scheinfeldt, L. B.
, and Tishkoff S. A.. 2010. “Living the High Life: High‐Altitude Adaptation.” Genome Biology
11: 133.
</cite> [<a href="https://doi.org/10.1186/gb-2010-11-9-133" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC2965377/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/20979669/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology&amp;title=Living%20the%20High%20Life:%20High%E2%80%90Altitude%20Adaptation&amp;author=L.%20B.%20Scheinfeldt&amp;author=S.%20A.%20Tishkoff&amp;volume=11&amp;publication_year=2010&amp;pages=133&amp;pmid=20979669&amp;doi=10.1186/gb-2010-11-9-133&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0085">
<cite>

Schilcher, F.
, Thamm M., Strube‐Bloss M., and Scheiner R.. 2021. “Opposing Actions of Octopamine and Tyramine on Honeybee Vision.” Biomolecules
11: 1374.
</cite> [<a href="https://doi.org/10.3390/biom11091374" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8468255/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34572588/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biomolecules&amp;title=Opposing%20Actions%20of%20Octopamine%20and%20Tyramine%20on%20Honeybee%20Vision&amp;author=F.%20Schilcher&amp;author=M.%20Thamm&amp;author=M.%20Strube%E2%80%90Bloss&amp;author=R.%20Scheiner&amp;volume=11&amp;publication_year=2021&amp;pages=1374&amp;pmid=34572588&amp;doi=10.3390/biom11091374&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-1002">
<cite>

Scholl, C.
, Kübert N., Muenz T. S., and Rössler W.. 2015. “CaMKII Knockdown Affects Both Early and Late Phases of Olfactory Long‐Term Memory in the Honeybee.” Journal of Experimental Biology
218, no. 23: 3788–3796.
</cite> [<a href="https://doi.org/10.1242/jeb.124859" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26486369/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Biology&amp;title=CaMKII%20Knockdown%20Affects%20Both%20Early%20and%20Late%20Phases%20of%20Olfactory%20Long%E2%80%90Term%20Memory%20in%20the%20Honeybee&amp;author=C.%20Scholl&amp;author=N.%20K%C3%BCbert&amp;author=T.%20S.%20Muenz&amp;author=W.%20R%C3%B6ssler&amp;volume=218&amp;issue=23&amp;publication_year=2015&amp;pages=3788-3796&amp;pmid=26486369&amp;doi=10.1242/jeb.124859&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0086">
<cite>

Shin, J.‐H.
, Blay S., Graham J., and McNeney B.. 2006. “LDheatmap: An R Function for Graphical Display of Pairwise Linkage Disequilibria Between Single Nucleotide Polymorphisms.” Journal of Statistical Software
16: 1–9.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Statistical%20Software&amp;title=LDheatmap:%20An%20R%20Function%20for%20Graphical%20Display%20of%20Pairwise%20Linkage%20Disequilibria%20Between%20Single%20Nucleotide%20Polymorphisms&amp;author=J.%E2%80%90H.%20Shin&amp;author=S.%20Blay&amp;author=J.%20Graham&amp;author=B.%20McNeney&amp;volume=16&amp;publication_year=2006&amp;pages=1-9&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-1003">
<cite>

Siewert, K. M.
, and Voight B. F.. 2020. “BetaScan2: Standardized Statistics to Detect Balancing Selection Utilizing Substitution Data.” Genome Biology and Evolution
12, no. 2: 3873–3877.
</cite> [<a href="https://doi.org/10.1093/gbe/evaa013" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7058154/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32011695/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology%20and%20Evolution&amp;title=BetaScan2:%20Standardized%20Statistics%20to%20Detect%20Balancing%20Selection%20Utilizing%20Substitution%20Data&amp;author=K.%20M.%20Siewert&amp;author=B.%20F.%20Voight&amp;volume=12&amp;issue=2&amp;publication_year=2020&amp;pages=3873-3877&amp;pmid=32011695&amp;doi=10.1093/gbe/evaa013&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0087">
<cite>

Smith, F. G.

1961. “The Races of Honeybees in Africa.” Bee World
42: 255–260.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Bee%20World&amp;title=The%20Races%20of%20Honeybees%20in%20Africa&amp;author=F.%20G.%20Smith&amp;volume=42&amp;publication_year=1961&amp;pages=255-260&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0088">
<cite>

Stalder, L.
, Oggenfuss U., Mohd‐Assaad N., and Croll D.. 2022. “The Population Genetics of Adaptation Through Copy Number Variation in a Fungal Plant Pathogen.” Molecular Ecology
32, no. April: 2443–2460.
</cite> [<a href="https://doi.org/10.1111/mec.16435" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35313056/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Ecology&amp;title=The%20Population%20Genetics%20of%20Adaptation%20Through%20Copy%20Number%20Variation%20in%20a%20Fungal%20Plant%20Pathogen&amp;author=L.%20Stalder&amp;author=U.%20Oggenfuss&amp;author=N.%20Mohd%E2%80%90Assaad&amp;author=D.%20Croll&amp;volume=32&amp;issue=April&amp;publication_year=2022&amp;pages=2443-2460&amp;pmid=35313056&amp;doi=10.1111/mec.16435&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0089">
<cite>

Stevison, L. S.
, Hoehn K. B., and Noor M. A. F.. 2011. “Effects of Inversions on Within‐ and Between‐Species Recombination and Divergence.” Genome Biology and Evolution
3: 830–841.
</cite> [<a href="https://doi.org/10.1093/gbe/evr081" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC3171675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/21828374/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Genome%20Biology%20and%20Evolution&amp;title=Effects%20of%20Inversions%20on%20Within%E2%80%90%20and%20Between%E2%80%90Species%20Recombination%20and%20Divergence&amp;author=L.%20S.%20Stevison&amp;author=K.%20B.%20Hoehn&amp;author=M.%20A.%20F.%20Noor&amp;volume=3&amp;publication_year=2011&amp;pages=830-841&amp;pmid=21828374&amp;doi=10.1093/gbe/evr081&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0090">
<cite>

Szpiech, Z. A.
, and Hernandez R. D.. 2014. “selscan: An Efficient Multithreaded Program to Perform EHH‐Based Scans for Positive Selection.” Molecular Biology and Evolution
31: 2824–2827.
</cite> [<a href="https://doi.org/10.1093/molbev/msu211" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4166924/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25015648/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecular%20Biology%20and%20Evolution&amp;title=selscan:%20An%20Efficient%20Multithreaded%20Program%20to%20Perform%20EHH%E2%80%90Based%20Scans%20for%20Positive%20Selection&amp;author=Z.%20A.%20Szpiech&amp;author=R.%20D.%20Hernandez&amp;volume=31&amp;publication_year=2014&amp;pages=2824-2827&amp;pmid=25015648&amp;doi=10.1093/molbev/msu211&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0091">
<cite>

Szpiech, Z. A.
, Novak T. E., Bailey N. P., and Stevison L. S.. 2021. “Application of a Novel Haplotype‐Based Scan for Local Adaptation to Study High‐Altitude Adaptation in Rhesus Macaques.” Evolution Letters
5: 408–421.
</cite> [<a href="https://doi.org/10.1002/evl3.232" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8327953/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34367665/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolution%20Letters&amp;title=Application%20of%20a%20Novel%20Haplotype%E2%80%90Based%20Scan%20for%20Local%20Adaptation%20to%20Study%20High%E2%80%90Altitude%20Adaptation%20in%20Rhesus%20Macaques&amp;author=Z.%20A.%20Szpiech&amp;author=T.%20E.%20Novak&amp;author=N.%20P.%20Bailey&amp;author=L.%20S.%20Stevison&amp;volume=5&amp;publication_year=2021&amp;pages=408-421&amp;pmid=34367665&amp;doi=10.1002/evl3.232&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0093">
<cite>

Van der Auwera, G. A.
, Carneiro M. O., Hartl C., et al. 2013. “From FastQ Data to High‐Confidence Variant Calls: The Genome Analysis Toolkit Best Practices Pipeline.” Current Protocols in Bioinformatics
43: 11.10.1–11.10.33.</cite> [<a href="https://doi.org/10.1002/0471250953.bi1110s43" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4243306/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25431634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Current%20Protocols%20in%20Bioinformatics&amp;title=From%20FastQ%20Data%20to%20High%E2%80%90Confidence%20Variant%20Calls:%20The%20Genome%20Analysis%20Toolkit%20Best%20Practices%20Pipeline&amp;author=G.%20A.%20Van%20der%20Auwera&amp;author=M.%20O.%20Carneiro&amp;author=C.%20Hartl&amp;volume=43&amp;publication_year=2013&amp;pages=11.10.1-11.10.33&amp;pmid=25431634&amp;doi=10.1002/0471250953.bi1110s43&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0094">
<cite>

Vaníčková, L.
, Břízová R., Pompeiano A., Ekesi S., and Meyer M. D.. 2015. “Cuticular Hydrocarbons Corroborate the Distinction Between Lowland and Highland Natal Fruit Fly (Tephritidae, <em>Ceratitis rosa</em>) Populations.” ZooKeys
540: 507–524.</cite> [<a href="https://doi.org/10.3897/zookeys.540.9619" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4714085/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26798275/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ZooKeys&amp;title=Cuticular%20Hydrocarbons%20Corroborate%20the%20Distinction%20Between%20Lowland%20and%20Highland%20Natal%20Fruit%20Fly%20(Tephritidae,%20Ceratitis%20rosa)%20Populations&amp;author=L.%20Van%C3%AD%C4%8Dkov%C3%A1&amp;author=R.%20B%C5%99%C3%ADzov%C3%A1&amp;author=A.%20Pompeiano&amp;author=S.%20Ekesi&amp;author=M.%20D.%20Meyer&amp;volume=540&amp;publication_year=2015&amp;pages=507-524&amp;pmid=26798275&amp;doi=10.3897/zookeys.540.9619&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0095">
<cite>

Waldvogel, A.‐M.
, Feldmeyer B., Rolshausen G., et al. 2020. “Evolutionary Genomics Can Improve Prediction of Species' Responses to Climate Change.” Evolution Letters
4: 4–18.
</cite> [<a href="https://doi.org/10.1002/evl3.154" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7006467/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32055407/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Evolution%20Letters&amp;title=Evolutionary%20Genomics%20Can%20Improve%20Prediction%20of%20Species'%20Responses%20to%20Climate%20Change&amp;author=A.%E2%80%90M.%20Waldvogel&amp;author=B.%20Feldmeyer&amp;author=G.%20Rolshausen&amp;volume=4&amp;publication_year=2020&amp;pages=4-18&amp;pmid=32055407&amp;doi=10.1002/evl3.154&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0096">
<cite>

Wallberg, A.
, Bunikis I., Pettersson O. V., et al. 2019. “A Hybrid De Novo Genome Assembly of the Honeybee, <em>Apis mellifera</em>, With Chromosome‐Length Scaffolds.” BMC Genomics
20: 275.
</cite> [<a href="https://doi.org/10.1186/s12864-019-5642-0" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6454739/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30961563/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=BMC%20Genomics&amp;title=A%20Hybrid%20De%20Novo%20Genome%20Assembly%20of%20the%20Honeybee,%20Apis%20mellifera,%20With%20Chromosome%E2%80%90Length%20Scaffolds&amp;author=A.%20Wallberg&amp;author=I.%20Bunikis&amp;author=O.%20V.%20Pettersson&amp;volume=20&amp;publication_year=2019&amp;pages=275&amp;pmid=30961563&amp;doi=10.1186/s12864-019-5642-0&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0097">
<cite>

Wallberg, A.
, Schöning C., Webster M. T., and Hasselmann M.. 2017. “Two Extended Haplotype Blocks Are Associated With Adaptation to High Altitude Habitats in East African Honey Bees.” PLoS Genetics
13: e1006792.
</cite> [<a href="https://doi.org/10.1371/journal.pgen.1006792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5444601/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28542163/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20Genetics&amp;title=Two%20Extended%20Haplotype%20Blocks%20Are%20Associated%20With%20Adaptation%20to%20High%20Altitude%20Habitats%20in%20East%20African%20Honey%20Bees&amp;author=A.%20Wallberg&amp;author=C.%20Sch%C3%B6ning&amp;author=M.%20T.%20Webster&amp;author=M.%20Hasselmann&amp;volume=13&amp;publication_year=2017&amp;pages=e1006792&amp;pmid=28542163&amp;doi=10.1371/journal.pgen.1006792&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0098">
<cite>

Wang, L.
, Cui S., Liu Z., Ping Y., Qiu J., and Geng X.. 2018. “Inhibition of Mitochondrial Respiration Under Hypoxia and Increased Antioxidant Activity After Reoxygenation of <em>Tribolium castaneum</em>
.” PLoS One
13: e0199056.
</cite> [<a href="https://doi.org/10.1371/journal.pone.0199056" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6002095/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/29902250/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=PLoS%20One&amp;title=Inhibition%20of%20Mitochondrial%20Respiration%20Under%20Hypoxia%20and%20Increased%20Antioxidant%20Activity%20After%20Reoxygenation%20of%20Tribolium%20castaneum&amp;author=L.%20Wang&amp;author=S.%20Cui&amp;author=Z.%20Liu&amp;author=Y.%20Ping&amp;author=J.%20Qiu&amp;volume=13&amp;publication_year=2018&amp;pages=e0199056&amp;pmid=29902250&amp;doi=10.1371/journal.pone.0199056&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0099">
<cite>

Webster, M. T.
, Beaurepaire A., Neumann P., and Stolle E.. 2023. “Population Genomics for Insect Conservation.” Annual Review of Animal Biosciences
11: 115–140.
</cite> [<a href="https://doi.org/10.1146/annurev-animal-122221-075025" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36375448/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Annual%20Review%20of%20Animal%20Biosciences&amp;title=Population%20Genomics%20for%20Insect%20Conservation&amp;author=M.%20T.%20Webster&amp;author=A.%20Beaurepaire&amp;author=P.%20Neumann&amp;author=E.%20Stolle&amp;volume=11&amp;publication_year=2023&amp;pages=115-140&amp;pmid=36375448&amp;doi=10.1146/annurev-animal-122221-075025&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0100">
<cite>

Weeks, A. R.
, McKechnie S. W., and Hoffmann A. A.. 2002. “Dissecting Adaptive Clinal Variation: Markers, Inversions and Size/Stress Associations in 
<em>Drosophila melanogaster</em>
 From a Central Field Population.” Ecology Letters
5: 756–763.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Ecology%20Letters&amp;title=Dissecting%20Adaptive%20Clinal%20Variation:%20Markers,%20Inversions%20and%20Size/Stress%20Associations%20in%20Drosophila%20melanogaster%20From%20a%20Central%20Field%20Population&amp;author=A.%20R.%20Weeks&amp;author=S.%20W.%20McKechnie&amp;author=A.%20A.%20Hoffmann&amp;volume=5&amp;publication_year=2002&amp;pages=756-763&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-1005">
<cite>

Wen, Y.
, Wang J., Liu X., Li S., Hu J., and Luo Y.. 2021. “Regulating Glycolysis and Heat Shock Proteins in Gannan Yaks (<em>Bos grunniens</em>) in Response to Hypoxia of the Qinghai–Tibet Plateau.” Archives Animal Breeding
64, no. 2: 345–353.
</cite> [<a href="https://doi.org/10.5194/aab-64-345-2021" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8386194/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34458561/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Archives%20Animal%20Breeding&amp;title=Regulating%20Glycolysis%20and%20Heat%20Shock%20Proteins%20in%20Gannan%20Yaks%20(Bos%20grunniens)%20in%20Response%20to%20Hypoxia%20of%20the%20Qinghai%E2%80%93Tibet%20Plateau&amp;author=Y.%20Wen&amp;author=J.%20Wang&amp;author=X.%20Liu&amp;author=S.%20Li&amp;author=J.%20Hu&amp;volume=64&amp;issue=2&amp;publication_year=2021&amp;pages=345-353&amp;pmid=34458561&amp;doi=10.5194/aab-64-345-2021&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0101">
<cite>

Woods, H. A.
, and Hill R. I.. 2004. “Temperature‐Dependent Oxygen Limitation in Insect Eggs.” Journal of Experimental Biology
207: 2267–2276.
</cite> [<a href="https://doi.org/10.1242/jeb.00991" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/15159431/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Journal%20of%20Experimental%20Biology&amp;title=Temperature%E2%80%90Dependent%20Oxygen%20Limitation%20in%20Insect%20Eggs&amp;author=H.%20A.%20Woods&amp;author=R.%20I.%20Hill&amp;volume=207&amp;publication_year=2004&amp;pages=2267-2276&amp;pmid=15159431&amp;doi=10.1242/jeb.00991&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="ece371846-bib-0103">
<cite>

Zhou, Y.
, Li Y., Li X., et al. 2020. “Apolipoprotein D in Lepidoptera: Evolution and Functional Divergence.” Biochemical and Biophysical Research Communications
526: 472–478.
</cite> [<a href="https://doi.org/10.1016/j.bbrc.2020.03.112" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32234238/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biochemical%20and%20Biophysical%20Research%20Communications&amp;title=Apolipoprotein%20D%20in%20Lepidoptera:%20Evolution%20and%20Functional%20Divergence&amp;author=Y.%20Zhou&amp;author=Y.%20Li&amp;author=X.%20Li&amp;volume=526&amp;publication_year=2020&amp;pages=472-478&amp;pmid=32234238&amp;doi=10.1016/j.bbrc.2020.03.112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><p>
<strong>Figure S1:</strong> Genome wide and pi scan of analyzed population using a sliding window of 10 kb.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367272/bin/ECE3-15-e71846-s003.png" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71846-s003.png</a><sup> (2.8MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material2_reqid_"><div class="caption p"><p>
<strong>Figure S2:</strong> Admixture cross‐validation error from <em>K</em> = 1 to <em>K</em> = 6 for five different random seeds.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367272/bin/ECE3-15-e71846-s004.png" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71846-s004.png</a><sup> (40.2KB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material3_reqid_"><div class="caption p"><p>
<strong>Figure S3:</strong> Beta‐values computed for highland and lowland populations with a set threshold of 2 using a 150 SNP window.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367272/bin/ECE3-15-e71846-s001.png" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71846-s001.png</a><sup> (190.6KB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material4_reqid_"><div class="caption p"><p>
<strong>Figure S4:</strong> Selection scan XP‐nSL values for the three populations analyzed: Mau (Top), Mt.Kenya (Center) and Rwenzori Mountains (Bottom). The black line shows sliding window values of 10 kb. Red dots represent potential SNPs involved in high elevation adaptation, while blue dots represent potential SNPs involved in lowland adaptation.</p></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367272/bin/ECE3-15-e71846-s002.png" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71846-s002.png</a><sup> (1.2MB, png) </sup>
</div></div></section><section class="sm xbox font-sm" id="db_ds_supplementary-material5_reqid_"><div class="caption p">
<p>
<strong>Table S1:</strong> Uganda elevation collection and coordinates.</p>
<p>
<strong>Table S2:</strong> ece371846‐sup‐0005‐Tables.xlsx. Samples included in the study and relatives coordinates of collection (where it was described in the original paper).</p>
<p>
<strong>Table S3:</strong> ece371846‐sup‐0005‐Tables.xlsx. Admixture cross‐validation results for 5 different random seeds for K going from 1 to 6.</p>
<p>
<strong>Table S4:</strong> ece371846‐sup‐0005‐Tables.xlsx. MKT results for genes inside the r7 inverted region.</p>
<p>
<strong>Table S5:</strong> ece371846‐sup‐0005‐Tables.xlsx. MKT results for genes inside the r9 inverted region.</p>
<p>
<strong>Table S6:</strong> ece371846‐sup‐0005‐Tables.xlsx. Genes falling inside 1% windows in Mau region.</p>
<p>
<strong>Table S7:</strong> ece371846‐sup‐0005‐Tables.xlsx. Genes falling inside 1% windows in Mt. Kenya region.</p>
<p>
<strong>Table S8:</strong> ece371846‐sup‐0005‐Tables.xlsx. Genes falling inside 1% windows in Rwenzori mountain region.</p>
<p>
<strong>Table S9:</strong> ece371846‐sup‐0005‐Tables.xlsx. GO terms for selection scan analysis of Rwenzori region population.</p>
</div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12367272/bin/ECE3-15-e71846-s005.xlsx" data-ga-action="click_feat_suppl" class="usa-link">ECE3-15-e71846-s005.xlsx</a><sup> (75.3KB, xlsx) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>Illumina WGS reads of the newly sequenced 24 honey bees as well as RNASeq reads of the 12 honey bees used in this study are available at the NCBI SRA under the accession no. PRJNA1179698. The Figures <a href="#ece371846-supitem-0001" class="usa-link">[Link]</a>, <a href="#ece371846-supitem-0004" class="usa-link">[Link]</a> and Tables <a href="#ece371846-supitem-0005" class="usa-link">S1–S9</a> as well as the scripts used to perform the analysis and plot the graphs are available at <a href="https://github.com/MarcoMazzoni93/Uganda" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://github.com/MarcoMazzoni93/Uganda</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from Ecology and Evolution are provided here courtesy of <strong>Wiley</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1002/ece3.71846"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/ECE3-15-e71846.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (4.2 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12367272/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12367272/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12367272%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12367272/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12367272/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12367272/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/similar-article-links/40843029/"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12367272/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                                data-source-url="/resources/cited-by-links/40843029/"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12367272/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12367272/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="pAMIIe70mDTbYr3gGXsh1vJDyOS7zA6RnVxyAK2rhWgrAMYxxkMi6O2iddhGSFAM">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
