
<!DOCTYPE html>
<html lang="en" >
    <head >

        <meta charset="UTF-8" />
        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
        <meta name="HandheldFriendly" content="True" />
        <meta name="MobileOptimized" content="320" />
        <meta name="viewport" content="width=device-width, initial-scale=1.0" />

        
        

        
        
  <link rel="stylesheet" href="/static/assets/style-8258669d.css" />
<script type="module" crossorigin="" src="/static/assets/base_style-f4a99799.js"></script>

  <link rel="stylesheet" href="/static/assets/style-ef962842.css" />
<link rel="stylesheet" href="/static/assets/style-3ade8b5c.css" />
<script type="module" crossorigin="" src="/static/assets/article_style-d757a0dd.js"></script>

  
  
    <style>
  
  
  @media screen and (min-width: 64em) {
    div.pmc-wm {
      background: repeat-y;
      background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' width='20' height='350' xmlns:xlink='http://www.w3.org/1999/xlink'%3E%3Cdefs%3E%3Cfilter x='-.02' y='0' width='1.05' height='1' id='c'%3E%3CfeFlood flood-color='%23FFF'/%3E%3CfeComposite in='SourceGraphic'/%3E%3C/filter%3E%3Ctext id='b' font-family='Helvetica' font-size='11pt' style='opacity:1;fill:%23005ea2;stroke:none;text-anchor:middle' x='175' y='14'%3E%3C/text%3E%3Cpath id='a' style='fill:%23005ea2' d='M0 8h350v3H0z'/%3E%3C/defs%3E%3Cuse xlink:href='%23a' transform='rotate(90 10 10)'/%3E%3Cuse xlink:href='%23b' transform='rotate(90 10 10)' filter='url(%23c)'/%3E%3C/svg%3E");
      padding-left: 3rem;
    }
  }
</style>

  



        
            <link rel="apple-touch-icon"
                  sizes="180x180"
                  href="/static/img/favicons/apple-touch-icon.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="48x48"
                  href="/static/img/favicons/favicon-48x48.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="32x32"
                  href="/static/img/favicons/favicon-32x32.png" />
            <link rel="icon"
                  type="image/png"
                  sizes="16x16"
                  href="/static/img/favicons/favicon-16x16.png" />
            <link rel="manifest" href="/static/img/favicons/site.webmanifest" />
            <link rel="mask-icon"
                  href="/static/img/favicons/safari-pinned-tab.svg"
                  color="#0071bc" />
            <meta name="msapplication-config"
                  content="/static/img/favicons/browserconfig.xml" />
            <meta name="theme-color" content="#ffffff" />
        

        <title>
            Hybrid nucleobase–heterocycle–2-oxindole scaffolds as innovative cell cycle modulators with potential anticancer activity - PMC
        </title>

        
        
  
  <!-- Logging params: Pinger defaults -->
<meta name="ncbi_app" content="cloudpmc-viewer" />
<meta name="ncbi_db" content="pmc" />
<meta name="ncbi_phid" content="15251FAB8AEF62A3051FAB0014998E08.m_1" />
<meta name="ncbi_pinger_stat_url" content="https://pmc.ncbi.nlm.nih.gov/stat" />
<!-- Logging params: Pinger custom -->
<meta name="ncbi_pdid" content="article" />
  
    <link rel="preconnect" href="https://www.google-analytics.com" />

    
        <link rel="preconnect" href="https://cdn.ncbi.nlm.nih.gov" />
    

    <!-- Include USWDS Init Script -->
    <script src="/static/assets/uswds-init.js"></script>


    <meta name="ncbi_domain" content="rscadv">
<meta name="ncbi_type" content="fulltext">
<meta name="ncbi_pcid" content="journal">
<meta name="ncbi_feature" content="associated_data">
<link rel="canonical" href="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377214/">
<meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE">
<meta name="citation_journal_title" content="RSC Advances">
<meta name="citation_title" content="Hybrid nucleobase–heterocycle–2-oxindole scaffolds as innovative cell cycle modulators with potential anticancer activity">
<meta name="citation_author" content="Gehan Ahmed Abdel-Hafez">
<meta name="citation_author_institution" content="Medicinal Chemistry Department, Faculty of Pharmacy, Merit University, Sohag, Egypt">
<meta name="citation_author" content="Dagmara Kłopotowska">
<meta name="citation_author_institution" content="Department of Experimental Oncology, Hirszfeld Institute of Immunology and Experimental Therapy, Polish Academy of Sciences, Wrocław, 53-114, Poland">
<meta name="citation_author" content="Beata Filip-Psurska">
<meta name="citation_author_institution" content="Department of Experimental Oncology, Hirszfeld Institute of Immunology and Experimental Therapy, Polish Academy of Sciences, Wrocław, 53-114, Poland">
<meta name="citation_author" content="Ahmed S Aboraia">
<meta name="citation_author_institution" content="Medicinal Chemistry Department, Faculty of Pharmacy, Assiut University, Assiut, 71526, Egypt, Email: gehanahmed@aun.edu.eg, Email: Fadl@aun.edu.eg">
<meta name="citation_author" content="Joanna Wietrzyk">
<meta name="citation_author_institution" content="Department of Experimental Oncology, Hirszfeld Institute of Immunology and Experimental Therapy, Polish Academy of Sciences, Wrocław, 53-114, Poland">
<meta name="citation_author" content="Tarek Aboul-Fadl">
<meta name="citation_author_institution" content="Medicinal Chemistry Department, Faculty of Pharmacy, Assiut University, Assiut, 71526, Egypt, Email: gehanahmed@aun.edu.eg, Email: Fadl@aun.edu.eg">
<meta name="citation_author" content="Adel F Youssef">
<meta name="citation_author_institution" content="Medicinal Chemistry Department, Faculty of Pharmacy, Assiut University, Assiut, 71526, Egypt, Email: gehanahmed@aun.edu.eg, Email: Fadl@aun.edu.eg">
<meta name="citation_publication_date" content="2025 Aug 22">
<meta name="citation_volume" content="15">
<meta name="citation_issue" content="36">
<meta name="citation_firstpage" content="29753">
<meta name="citation_doi" content="10.1039/d5ra04997k">
<meta name="citation_abstract_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377214/">
<meta name="citation_fulltext_html_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377214/">
<meta name="citation_pdf_url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377214/pdf/RA-015-D5RA04997K.pdf">
<meta name="description" content="A series of hybrid molecules 6a–6d–13a–13d combining pyrazolo[3,4-d]pyrimidine or aminopurine frameworks with an oxindole moiety were designed as multitarget anticancer agents. Several compounds, especially 8b and 12a–12d, showed potent ...">
<meta name="og:title" content="Hybrid nucleobase–heterocycle–2-oxindole scaffolds as innovative cell cycle modulators with potential anticancer activity">
<meta name="og:type" content="article">
<meta name="og:site_name" content="PubMed Central (PMC)">
<meta name="og:description" content="A series of hybrid molecules 6a–6d–13a–13d combining pyrazolo[3,4-d]pyrimidine or aminopurine frameworks with an oxindole moiety were designed as multitarget anticancer agents. Several compounds, especially 8b and 12a–12d, showed potent ...">
<meta name="og:url" content="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377214/">
<meta name="og:image" content="https://cdn.ncbi.nlm.nih.gov/pmc/cms/images/pmc-card-share.jpg?_=0">
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@ncbi">
    
    

    </head>
    <body >
        
    <a class="usa-skipnav " href="#main-content">
      Skip to main content
    </a>


        
            

<section class="usa-banner " aria-label="Official website of the United States government" >
    <div class="usa-accordion">
        <header class="usa-banner__header">
            <div class="usa-banner__inner">
                <div class="grid-col-auto">
                    <img aria-hidden="true"
                         class="usa-banner__header-flag"
                         src="/static/img/us_flag.svg"
                         alt="" />
                </div>

                <div class="grid-col-fill tablet:grid-col-auto" aria-hidden="true">
                    <p class="usa-banner__header-text">
                        An official website of the United States government
                    </p>
                    <span class="usa-banner__header-action">Here's how you know</span>
                </div>

                



















    
        <button
            type="button"
        
    
    class="usa-accordion__button usa-banner__button
           

           
               
               
               
               
            

           
           
           
           "
    aria-expanded="false"
    aria-controls="gov-banner-default"
    
    data-testid="storybook-django-banner"
    
    >
    
        

        
                    <span class="usa-banner__button-text">Here's how you know</span>
                

        
    
        
            </button>
        


            </div>
        </header>

        <div class="usa-banner__content usa-accordion__content"
             id="gov-banner-default"
             hidden>
            <div class="grid-row grid-gap-lg">
                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-dot-gov.svg"
                         alt=""
                         aria-hidden="true" />
                    <div class="usa-media-block__body">
                        <p>
                            <strong>Official websites use .gov</strong>
                            <br />
                            A
                            <strong>.gov</strong> website belongs to an official
                            government organization in the United States.
                        </p>
                    </div>
                </div>

                <div class="usa-banner__guidance tablet:grid-col-6">
                    <img class="usa-banner__icon usa-media-block__img"
                         src="/static/img/icon-https.svg"
                         alt=""
                         aria-hidden="true" />

                    <div class="usa-media-block__body">
                        <p>
                            <strong>Secure .gov websites use HTTPS</strong>
                            <br />
                            A <strong>lock</strong> (
                            <span class="icon-lock">
                                <svg xmlns="http://www.w3.org/2000/svg"
                                     width="52"
                                     height="64"
                                     viewBox="0 0 52 64"
                                     class="usa-banner__lock-image"
                                     role="graphics-symbol"
                                     aria-labelledby="banner-lock-description"
                                     focusable="false">
                                    <title id="banner-lock-title">Lock</title>
                                    <desc id="banner-lock-description">
                                    Locked padlock icon
                                    </desc>
                                    <path fill="#000000"
                                          fill-rule="evenodd"
                                          d="M26 0c10.493 0 19 8.507 19 19v9h3a4 4 0 0 1 4 4v28a4 4 0 0 1-4 4H4a4 4 0 0 1-4-4V32a4 4 0 0 1 4-4h3v-9C7 8.507 15.507 0 26 0zm0 8c-5.979 0-10.843 4.77-10.996 10.712L15 19v9h22v-9c0-6.075-4.925-11-11-11z" />
                                </svg>
</span>) or <strong>https://</strong> means you've safely
                                connected to the .gov website. Share sensitive
                                information only on official, secure websites.
                            </p>
                        </div>
                    </div>
                </div>
            </div>
        </div>
    </section>

        

        
    
    
    

<div class="usa-overlay">
</div>



<header class="usa-header usa-header--extended usa-header--wide" data-header data-testid="header"    >
    <div class="ncbi-header">
        <div class="ncbi-header__container">
            <a class="ncbi-header__logo-container"
               href="https://www.ncbi.nlm.nih.gov/">
                <img alt="NCBI home page"
                     class="ncbi-header__logo-image"
                     src="/static/img/ncbi-logos/nih-nlm-ncbi--white.svg"
                     width="410"
                     height="100" />
            </a>

            <!-- Mobile menu hamburger button -->
            



















    
        <button
            type="button"
        
    
    class="usa-menu-btn ncbi-header__hamburger-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Show menu"
    data-testid="navMenuButton"
    
    >
    
        

        
                <svg aria-hidden="true"
                     class="ncbi-hamburger-icon"
                     fill="none"
                     focusable="false"
                     height="21"
                     viewBox="0 0 31 21"
                     width="31"
                     xmlns="http://www.w3.org/2000/svg">
                    <path clip-rule="evenodd"
                          d="M0.125 20.75H30.875V17.3333H0.125V20.75ZM0.125 12.2083H30.875V8.79167H0.125V12.2083ZM0.125 0.25V3.66667H30.875V0.25H0.125Z"
                          fill="#F1F1F1"
                          fill-rule="evenodd" />
                </svg>
            

        
    
        
            </button>
        



            
                <!-- Desktop buttons-->
                <div class="ncbi-header__desktop-buttons">
                    
                        <!-- Desktop search button -->
                        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button
           "
    aria-expanded="false"
    aria-controls="search-field-desktop-navigation"
    aria-label="Show search overlay"
    data-testid="toggleSearchPanelButton"
    data-toggle-search-panel-button
    >
    
        

        
                            



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#search" />
    </svg>


                            Search
                        

        
    
        
            </button>
        


                    

                    <!-- Desktop login dropdown -->
                    
                        <div class="ncbi-header__login-dropdown">
                            



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--unstyled ncbi-header__desktop-button ncbi-header__login-dropdown-button
           "
    aria-expanded="false"
    aria-controls="login-dropdown-menu"
    aria-label="Show login menu"
    data-testid="toggleLoginMenuDropdown"
    data-desktop-login-button
    >
    
        

        
                                



    <svg class="usa-icon " role="graphics-symbol" aria-hidden="true" >
        
        <use xlink:href="/static/img/sprite.svg#person" />
    </svg>



                                <span data-login-dropdown-text>Log in</span>

                                <!-- Dropdown icon pointing up -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-less ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-up-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_less" />
    </svg>



                                <!-- Dropdown icon pointing down -->
                                



    <svg class="usa-icon ncbi-header__login-dropdown-icon ncbi-header__login-dropdown-icon--expand-more ncbi-header__login-dropdown-icon--hidden" role="graphics-symbol" aria-hidden="true" data-login-dropdown-down-arrow>
        
        <use xlink:href="/static/img/sprite.svg#expand_more" />
    </svg>


                            

        
    
        
            </button>
        



                            <!-- Login dropdown menu -->
                            <ul class="usa-nav__submenu ncbi-header__login-dropdown-menu"
                                id="login-dropdown-menu"
                                data-desktop-login-menu-dropdown
                                hidden>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                                    </li>
                                
                                    <li class="usa-nav__submenu-item">
                                        <!-- Uses custom style overrides to render external and document links. -->
                                        









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                                    </li>
                                
                                <li class="usa-nav__submenu-item">
                                    



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           usa-button--outline ncbi-header__login-dropdown-logout-button
           "
    
    
    
    data-testid="desktopLogoutButton"
    data-desktop-logout-button
    >
    
        

        Log out

        
    
        
            </button>
        


                                </li>
                            </ul>
                        </div>
                    
                </div>
            
        </div>
    </div>

    <!-- Search panel -->
    
        <div class="ncbi-search-panel ncbi--show-only-at-desktop"
             data-header-search-panel
             hidden>
            <div class="ncbi-search-panel__container">
                <form action="https://www.ncbi.nlm.nih.gov/search/all/"
                      
                      autocomplete="off"
                      class="usa-search usa-search--big ncbi-search-panel__form"
                      data-testid="desktop-navigation-search-form"
                      method="GET"
                      role="search">
                    <label class="usa-sr-only" for="search-field-desktop-navigation">
                        Search…
                    </label>
                    <input class="usa-input"
                           id="search-field-desktop-navigation"
                           name="term"
                           
                               placeholder="Search NCBI"
                           
                           type="search"
                           value="" />
                    



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                        <span class="usa-search__submit-text">
                            Search NCBI
                        </span>
                    

        
    
        
            </button>
        


                </form>

                
            </div>
        </div>
    

    <nav aria-label="Primary navigation" class="usa-nav">
        <p class="usa-sr-only" id="primary-navigation-sr-only-title">
            Primary site navigation
        </p>

        <!-- Mobile menu close button -->
        



















    
        <button
            type="button"
        
    
    class="usa-nav__close ncbi-nav__close-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    aria-label="Close navigation menu"
    data-testid="navCloseButton"
    
    >
    
        

        
            <img src="/static/img/usa-icons/close.svg" alt="Close" />
        

        
    
        
            </button>
        



        
            <!-- Mobile search component -->
            <form class="usa-search usa-search--small ncbi--hide-at-desktop margin-top-6"
                  action="https://www.ncbi.nlm.nih.gov/search/all/"
                  
                  autocomplete="off"
                  data-testid="mobile-navigation-search-form"
                  method="GET"
                  role="search">
                <label class="usa-sr-only" for="search-field-mobile-navigation">
                    Search
                </label>

                <input class="usa-input"
                       id="search-field-mobile-navigation"
                       type="search"
                       
                           placeholder="Search NCBI"
                       
                       name="term" />

                



















    
        <button
            type="submit"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           
           "
    
    
    
    
    
    >
    
        

        
                    <!-- This SVG should be kept inline and not replaced with a link to the icon as otherwise it will render in the wrong color -->
                    <img src="data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIGhlaWdodD0iMjQiIHZpZXdCb3g9IjAgMCAyNCAyNCIgd2lkdGg9IjI0Ij48cGF0aCBkPSJNMCAwaDI0djI0SDB6IiBmaWxsPSJub25lIi8+PHBhdGggZmlsbD0iI2ZmZiIgZD0iTTE1LjUgMTRoLS43OWwtLjI4LS4yN0E2LjQ3MSA2LjQ3MSAwIDAgMCAxNiA5LjUgNi41IDYuNSAwIDEgMCA5LjUgMTZjMS42MSAwIDMuMDktLjU5IDQuMjMtMS41N2wuMjcuMjh2Ljc5bDUgNC45OUwyMC40OSAxOWwtNC45OS01em0tNiAwQzcuMDEgMTQgNSAxMS45OSA1IDkuNVM3LjAxIDUgOS41IDUgMTQgNy4wMSAxNCA5LjUgMTEuOTkgMTQgOS41IDE0eiIvPjwvc3ZnPg=="
                         class="usa-search__submit-icon"
                         alt="Search" />
                

        
    
        
            </button>
        


            </form>

            
        

        <!-- Primary navigation menu items -->
        <!-- This usa-nav__inner wrapper is required to correctly style the navigation items on Desktop -->
        

        
            <div class="ncbi-nav__mobile-login-menu ncbi--hide-at-desktop"
                 data-mobile-login-menu
                 hidden>
                <p class="ncbi-nav__mobile-login-menu-status">
                    Logged in as:
                    <strong class="ncbi-nav__mobile-login-menu-email"
                            data-mobile-login-email-text></strong>
                </p>
                <ul class="usa-nav__primary usa-accordion">
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/" class="usa-link  "  >
        

        
            Dashboard
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/myncbi/collections/bibliography/" class="usa-link  "  >
        

        
            Publications
        

        
    </a>


                        </li>
                    
                        <li class="usa-nav__primary-item">
                            









    <a href="https://www.ncbi.nlm.nih.gov/account/settings/" class="usa-link  "  >
        

        
            Account settings
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        

        



















    
        <button
            type="button"
        
    
    class="usa-button
           

           
               
               
               
               
            

           
           
           ncbi-nav__mobile-login-button ncbi--hide-at-desktop
           "
    
    
    
    data-testid="mobileLoginButton"
    data-mobile-login-button
    >
    
        

        Log in

        
    
        
            </button>
        


    </nav>
</header>

    
    
        

<section class="pmc-header pmc-header--basic" aria-label="PMC Header with search box">
    <div class="pmc-nav-container">
        <div class="pmc-header__bar">
           <div class="pmc-header__logo">
               <a href="/" title="Home" aria-label="PMC Home"></a>
           </div>
            <button
                    type="button"
                    class="usa-button usa-button--unstyled pmc-header__search__button"
                    aria-label="Open search"
                    data-ga-category="search"
                    data-ga-action="PMC"
                    data-ga-label="pmc_search_panel_mobile"
            >
                <svg class="usa-icon width-4 height-4 pmc-icon__open" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#search"></use>
                </svg>
                <svg class="usa-icon width-4 height-4 pmc-icon__close" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
            </button>
        </div>
        <div class="pmc-header__search">
            


<form class="usa-search usa-search--extra usa-search--article-right-column  pmc-header__search__form" id="pmc-search-form" autocomplete="off" role="search">
<label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
<span class="autoComplete_wrapper flex-1">
<input class="usa-input width-full maxw-none" required="required" placeholder="Search PMC Full-Text Archive" id="pmc-search"  type="search" name="term" data-autocomplete-url="https://pmc.ncbi.nlm.nih.gov/autocomp/search/autocomp/"/>
</span>
<button
class="usa-button"
type="submit"
formaction="https://www.ncbi.nlm.nih.gov/pmc/"
data-ga-category="search"
data-ga-action="PMC"
data-ga-label="PMC_search_button"
>
<span class="usa-search__submit-text">Search in PMC</span>
<img
src="/static/img/usa-icons-bg/search--white.svg"
class="usa-search__submit-icon"
alt="Search"
/>
</button>
</form>
            <div class="display-flex flex-column tablet:flex-row tablet:flex-justify flex-justify-center flex-align-center width-full desktop:maxw-44">
                <ul class="pmc-header__search__menu">
                    <li>
                        
                            <a class="usa-link" href="/journals/" data-ga-action="featured_link" data-ga-label="journal list">
                                Journal List
                            </a>
                        
                    </li>
                    <li>
                        
                            <a class="usa-link" href="/about/userguide/" data-ga-action="featured_link"
                            data-ga-label="user guide">
                                User Guide
                            </a>
                        
                    </li>
                </ul>
                
                    <button form="pmc-search-form" formaction="https://pmc.ncbi.nlm.nih.gov/search/" type="submit" class="usa-button usa-button--unstyled hover:text-no-underline text-no-underline width-auto margin-top-1 tablet:margin-top-0">
     <span class="bg-green-label padding-05 text-white">New</span><span class="text-underline text-primary">Try this search in PMC Beta Search</span>
</button>
                
            </div>
        </div>
    </div>
</section>

    


        
        

       
  <div class="usa-section padding-top-0 desktop:padding-top-6 pmc-article-section" data-article-db="pmc" data-article-id="12377214">

    

   



<div class="grid-container pmc-actions-bar" aria-label="Actions bar" role="complementary">
    <div class="grid-row">
        <div class="grid-col-fill display-flex">
             <div class="display-flex">
                <ul class="usa-list usa-list--unstyled usa-list--horizontal">
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <button
                                type="button"
                                class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                                aria-label="Open resources"
                                data-extra-class="is-visible-resources"
                                data-ga-category="resources_accordion"
                                data-ga-action="click"
                                data-ga-label="mobile_icon"
                        >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#more_vert"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex mob">
                        <a
                                href="https://doi.org/10.1039/d5ra04997k"
                                class="usa-link display-flex usa-tooltip"
                                role="button"
                                target="_blank"
                                rel="noreferrer noopener"
                                title="View on publisher site"
                                data-position="bottom"
                                aria-label="View on publisher site"
                                data-ga-category="actions"
                                data-ga-action="click"
                                data-ga-label="publisher_link_mobile"
                        >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#launch"></use>
                                </svg>
                        </a>
                    </li>
                    
                    
                        <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                             <a
                                     href="pdf/RA-015-D5RA04997K.pdf"
                                     class="usa-link display-flex usa-tooltip"
                                     role="button"
                                     title="Download PDF"
                                     data-position="bottom"
                                     aria-label="Download PDF"
                                     data-ga-category="actions"
                                     data-ga-action="click"
                                     data-ga-label="pdf_download_mobile"
                             >
                                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                                </svg>
                            </a>
                        </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button
                            class="usa-button usa-button--unstyled  usa-tooltip collections-dialog-trigger collections-button display-flex collections-button-empty"
                            title="Add to Collections"
                            data-position="bottom"
                            aria-label="Save article in MyNCBI collections."
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="collections_button_mobile"
                            data-collections-open-dialog-enabled="false"
                            data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12377214%2F%3Freport%3Dclassic%23open-collections-dialog"
                            data-in-collections="false"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="margin-right-2 mobile-lg:margin-right-4 display-flex">
                        <button role="button" class="usa-button usa-button--unstyled usa-tooltip citation-dialog-trigger display-flex"
                            aria-label="Open dialog with citation text in different styles"
                            title="Cite"
                            data-position="bottom"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="cite_mobile"
                            data-all-citations-url="/resources/citations/12377214/"
                            data-citation-style="nlm"
                            data-download-format-link="/resources/citations/12377214/export/"
                        >
                            <svg class="usa-icon width-4 height-4 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                            </svg>
                        </button>
                    </li>
                    
                    <li class="pmc-permalink display-flex" >
                         <button
                                 type="button"
                                 title="Permalink"
                                 data-position="bottom"
                                 class="usa-button usa-button--unstyled display-flex usa-tooltip"
                                 aria-label="Show article permalink"
                                 aria-expanded="false"
                                 aria-haspopup="true"
                                 data-ga-category="actions"
                                 data-ga-action="open"
                                 data-ga-label="permalink_mobile"
                         >
                            <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                                <use xlink:href="/static/img/sprite.svg#share"></use>
                            </svg>
                        </button>
                        

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377214/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
                    </li>
                </ul>
            </div>
            <button
                    type="button"
                    class="usa-button pmc-sidenav__container__open usa-button--unstyled width-auto display-flex"
                    aria-label="Open article navigation"
                    data-extra-class="is-visible-in-page"
                    data-ga-category="actions"
                    data-ga-action="open"
                    data-ga-label="article_nav_mobile"
            >
                <svg class="usa-icon width-4 height-4" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#list"></use>
                </svg>
            </button>
        </div>
    </div>
</div>
    <div class="grid-container desktop:padding-left-6">
      <div id="article-container" class="grid-row grid-gap">
        <div class="grid-col-12 desktop:grid-col-8 order-2 pmc-layout__content">
            <div class="grid-container padding-left-0 padding-right-0">
                <div class="grid-row desktop:margin-left-neg-6">
                    <div class="grid-col-12">
                        <div class="pmc-layout__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a class="usa-link" data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/about/disclaimer/">PMC Disclaimer</a>
    |
    <a class="usa-link" data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/about/copyright/">
        PMC Copyright Notice
    </a>
</div>
                    </div>
                </div>
                <div class="grid-row pmc-wm desktop:margin-left-neg-6">
                    <!-- Main content -->
                    <main
                      id="main-content"
                      class="usa-layout-docs__main usa-layout-docs grid-col-12 pmc-layout pmc-prose padding-0"
                    >

                      
                        <section class="pmc-journal-banner text-center line-height-none" aria-label="Journal banner"><img src="https://cdn.ncbi.nlm.nih.gov/pmc/banners/logo-rscadv.png" alt="RSC Advances logo" usemap="#pmc-banner-imagemap" width="500" height="75"><map name="pmc-banner-imagemap"><area alt="Link to RSC Advances" title="Link to RSC Advances" shape="default" href="https://www.rsc.org/journals-books-databases/about-journals/rsc-advances/" target="_blank" rel="noopener noreferrer"></map></section><article lang="en"><section aria-label="Article citation and metadata"><section class="pmc-layout__citation font-secondary font-xs"><div>
<div class="display-inline-block"><button type="button" class="cursor-pointer text-no-underline bg-transparent border-0 padding-0 text-left margin-0 text-normal text-primary" aria-controls="journal_context_menu">RSC Adv</button></div>. 2025 Aug 22;15(36):29753–29776. doi: <a href="https://doi.org/10.1039/d5ra04997k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">10.1039/d5ra04997k</a>
</div>
<nav id="journal_context_menu" hidden="hidden"><ul class="menu-list font-family-ui" role="menu">
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/pmc/?term=%22RSC%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem">Search in PMC</a></li>
<li role="presentation"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22RSC%20Adv%22%5Bjour%5D" lang="en" class="usa-link" role="menuitem">Search in PubMed</a></li>
<li role="presentation"><a href="https://www.ncbi.nlm.nih.gov/nlmcatalog?term=%22RSC%20Adv%22%5BTitle%20Abbreviation%5D" class="usa-link" role="menuitem">View in NLM Catalog</a></li>
<li role="presentation"><a href="?term=%22RSC%20Adv%22%5Bjour%5D" class="usa-link" role="menuitem" data-add-to-search="true">Add to search</a></li>
</ul></nav></section><section class="front-matter"><div class="ameta p font-secondary font-xs">
<hgroup><h1>Hybrid nucleobase–heterocycle–2-oxindole scaffolds as innovative cell cycle modulators with potential anticancer activity</h1></hgroup><div class="cg p">
<a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abdel-Hafez%20GA%22%5BAuthor%5D" class="usa-link" aria-describedby="id1"><span class="name western">Gehan Ahmed Abdel-Hafez</span></a><div hidden="hidden" id="id1">
<h3><span class="name western">Gehan Ahmed Abdel-Hafez</span></h3>
<div class="p">
<sup>a</sup>Medicinal Chemistry Department, Faculty of Pharmacy, Merit University, Sohag, Egypt</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Abdel-Hafez%20GA%22%5BAuthor%5D" class="usa-link"><span class="name western">Gehan Ahmed Abdel-Hafez</span></a>
</div>
</div>
<sup>a</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22K%C5%82opotowska%20D%22%5BAuthor%5D" class="usa-link" aria-describedby="id2"><span class="name western">Dagmara Kłopotowska</span></a><div hidden="hidden" id="id2">
<h3><span class="name western">Dagmara Kłopotowska</span></h3>
<div class="p">
<sup>b</sup>Department of Experimental Oncology, Hirszfeld Institute of Immunology and Experimental Therapy, Polish Academy of Sciences, Wrocław, 53-114, Poland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22K%C5%82opotowska%20D%22%5BAuthor%5D" class="usa-link"><span class="name western">Dagmara Kłopotowska</span></a>
</div>
</div>
<sup>b</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Filip-Psurska%20B%22%5BAuthor%5D" class="usa-link" aria-describedby="id3"><span class="name western">Beata Filip-Psurska</span></a><div hidden="hidden" id="id3">
<h3><span class="name western">Beata Filip-Psurska</span></h3>
<div class="p">
<sup>b</sup>Department of Experimental Oncology, Hirszfeld Institute of Immunology and Experimental Therapy, Polish Academy of Sciences, Wrocław, 53-114, Poland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Filip-Psurska%20B%22%5BAuthor%5D" class="usa-link"><span class="name western">Beata Filip-Psurska</span></a>
</div>
</div>
<sup>b</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aboraia%20AS%22%5BAuthor%5D" class="usa-link" aria-describedby="id4"><span class="name western">Ahmed S Aboraia</span></a><div hidden="hidden" id="id4">
<h3><span class="name western">Ahmed S Aboraia</span></h3>
<div class="p">
<sup>c</sup>Medicinal Chemistry Department, Faculty of Pharmacy, Assiut University, Assiut, 71526, Egypt, Email: gehanahmed@aun.edu.eg, Email: Fadl@aun.edu.eg</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aboraia%20AS%22%5BAuthor%5D" class="usa-link"><span class="name western">Ahmed S Aboraia</span></a>
</div>
</div>
<sup>c</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wietrzyk%20J%22%5BAuthor%5D" class="usa-link" aria-describedby="id5"><span class="name western">Joanna Wietrzyk</span></a><div hidden="hidden" id="id5">
<h3><span class="name western">Joanna Wietrzyk</span></h3>
<div class="p">
<sup>b</sup>Department of Experimental Oncology, Hirszfeld Institute of Immunology and Experimental Therapy, Polish Academy of Sciences, Wrocław, 53-114, Poland</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Wietrzyk%20J%22%5BAuthor%5D" class="usa-link"><span class="name western">Joanna Wietrzyk</span></a>
</div>
</div>
<sup>b</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aboul-Fadl%20T%22%5BAuthor%5D" class="usa-link" aria-describedby="id6"><span class="name western">Tarek Aboul-Fadl</span></a><div hidden="hidden" id="id6">
<h3><span class="name western">Tarek Aboul-Fadl</span></h3>
<div class="p">
<sup>c</sup>Medicinal Chemistry Department, Faculty of Pharmacy, Assiut University, Assiut, 71526, Egypt, Email: gehanahmed@aun.edu.eg, Email: Fadl@aun.edu.eg</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Aboul-Fadl%20T%22%5BAuthor%5D" class="usa-link"><span class="name western">Tarek Aboul-Fadl</span></a>
</div>
</div>
<sup>c,</sup><sup>✉</sup>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Youssef%20AF%22%5BAuthor%5D" class="usa-link" aria-describedby="id7"><span class="name western">Adel F Youssef</span></a><div hidden="hidden" id="id7">
<h3><span class="name western">Adel F Youssef</span></h3>
<div class="p">
<sup>c</sup>Medicinal Chemistry Department, Faculty of Pharmacy, Assiut University, Assiut, 71526, Egypt, Email: gehanahmed@aun.edu.eg, Email: Fadl@aun.edu.eg</div>
<div class="p">Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=%22Youssef%20AF%22%5BAuthor%5D" class="usa-link"><span class="name western">Adel F Youssef</span></a>
</div>
</div>
<sup>c</sup>
</div>
<ul class="d-buttons inline-list">
<li><button class="d-button" aria-controls="aip_a" aria-expanded="false">Author information</button></li>
<li><button class="d-button" aria-controls="anp_a" aria-expanded="false">Article notes</button></li>
<li><button class="d-button" aria-controls="clp_a" aria-expanded="false">Copyright and License information</button></li>
</ul>
<div class="d-panels font-secondary-light">
<div id="aip_a" class="d-panel p" style="display: none">
<div class="p" id="affa">
<sup>a</sup>Medicinal Chemistry Department, Faculty of Pharmacy, Merit University, Sohag, Egypt</div>
<div id="affb">
<sup>b</sup>Department of Experimental Oncology, Hirszfeld Institute of Immunology and Experimental Therapy, Polish Academy of Sciences, Wrocław, 53-114, Poland</div>
<div id="affc">
<sup>c</sup>Medicinal Chemistry Department, Faculty of Pharmacy, Assiut University, Assiut, 71526, Egypt, Email: gehanahmed@aun.edu.eg, Email: Fadl@aun.edu.eg</div>
<div class="author-notes p"><div class="fn" id="_fncrsp93pmc__">
<sup>✉</sup><p class="display-inline">Corresponding author.</p>
</div></div>
</div>
<div id="anp_a" class="d-panel p" style="display: none"><div class="notes p"><section id="historyarticle-meta1" class="history"><p>Received 2025 Jul 12; Accepted 2025 Aug 6; Collection date 2025 Aug 18.</p></section></div></div>
<div id="clp_a" class="d-panel p" style="display: none">
<div>This journal is © The Royal Society of Chemistry</div>
<p>This article is licensed under a <a href="https://creativecommons.org/licenses/by-nc/3.0/" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Creative Commons Attribution-Non Commercial 3.0 Unported Licence</a>. You can use material from this article in other publications without requesting further permissions from the RSC, provided that the correct acknowledgement is given and it is not used for commercial purposes.</p>
<div class="p"><a href="/about/copyright/" class="usa-link">PMC Copyright notice</a></div>
</div>
</div>
<div>PMCID: PMC12377214</div>
</div></section></section><section aria-label="Article content"><section class="body main-article-body"><section class="abstract" id="abstract1"><h2>Abstract</h2>
<p>A series of hybrid molecules 6a–6d–13a–13d combining pyrazolo[3,4-<em>d</em>]pyrimidine or aminopurine frameworks with an oxindole moiety were designed as multitarget anticancer agents. Several compounds, especially 8b and 12a–12d, showed potent antiproliferative effects against three human cancer cell lines: A498 (kidney carcinoma), HepG-2 (hepatocellular carcinoma), and MDA-MB-231 (breast adenocarcinoma). Compounds 6b, 7b, 8b, and 12a–12c exhibited remarkable CDK6 inhibition (pIC<sub>50</sub> of up to 7.17), outperforming palbociclib, and VEGFR-2 inhibition comparable to sorafenib. These compounds also inhibited xanthine oxidase. Notably, 12a and 12c induced sub-G1 cell cycle arrest in HepG2 cells. Molecular modeling confirmed stable binding to CDK6 and VEGFR-2, while <em>in silico</em> ADMET profiling suggested favorable pharmacokinetics. These results support 8b and 12a–12c as strong leads for further multitarget cancer therapy development.</p></section><section class="abstract" id="abstract2"><hr class="headless">
<p>A series of hybrid molecules 6a–6d–13a–13d combining pyrazolo[3,4-<em>d</em>]pyrimidine or aminopurine frameworks with an oxindole moiety were designed as multitarget anticancer agents.<a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377214_d5ra04997k-ga.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/387c/12377214/3c067e9db2f5/d5ra04997k-ga.jpg" loading="lazy" id="ga" height="472" width="716" alt="graphic file with name d5ra04997k-ga.jpg"></a></p></section><section id="sec1"><h2 class="pmc_sec_title">Introduction</h2>
<p>Cancer is a major global health and economic burden, causing nearly one in six deaths worldwide.<sup><a href="#cit1" class="usa-link" aria-describedby="cit1">1,2</a></sup> By 2050, more than 35 million new cases are projected due to population growth and aging.<sup><a href="#cit3" class="usa-link" aria-describedby="cit3">3</a></sup> Current treatment generally involves surgery, radiation, and chemotherapy, which face challenges like drug resistance and severe side effects. There is a pressing need for new anticancer agents with greater efficacy and lower toxicity. A deeper understanding of cell cycle control, particularly key checkpoints (late G1, G2/M, and metaphase-to-anaphase transition), offers a promising strategy for developing more effective cancer therapies.<sup><a href="#cit4" class="usa-link" aria-describedby="cit4">4</a></sup></p>
<p>Cyclin-dependent kinases (CDKs) are serine/threonine protein kinases that regulate the cell cycle by forming heterodimeric complexes with cyclin subunits. Dysregulation of CDK or cyclin levels is frequently observed in cancers.<sup><a href="#cit5" class="usa-link" aria-describedby="cit5">5,6</a></sup> Among these, CDK4 and CDK6 are critical drivers of oncogenesis in specific tumors, making them promising targets for therapeutic development. Cyclin D-CDK4/6 complexes promote the transition from the G1 phase to the S phase by phosphorylating the retinoblastoma (RB) tumor suppressor protein, thereby facilitating cell cycle progression.<sup><a href="#cit7" class="usa-link" aria-describedby="cit7">7–10</a></sup> Inhibiting CDK4 and CDK6 is essential for targeting this critical cell cycle phase. Selective CDK4/6 inhibitors have shown effectiveness in treating various cancers, including melanoma, breast cancer, osteosarcoma, bladder cancer, lung cancer, and stomach cancer while minimizing toxicity in non-cancer cells. This therapeutic strategy holds promise for improving cancer treatment outcomes.<sup><a href="#cit11" class="usa-link" aria-describedby="cit11">11–14</a></sup> Various reported CDK4/6 inhibitors are based on various molecular scaffolds that enable specific binding to these kinases, thereby blocking their activity. Several “ciclib” scaffolds have been identified as core structural frameworks in the development of CDK4/6 inhibitors<sup><a href="#cit15" class="usa-link" aria-describedby="cit15">15,16</a></sup> are shown in <a href="#fig1" class="usa-link">Fig. 1</a>. The pyrido[3,4-<em>d</em>]pyrimidin-4-one scaffold has been extensively explored for the development of experimental CDK4/6 inhibitors. Its core structure offers flexibility for optimizing its affinity towards CDK4/6, enabling the creation of selective inhibitors with varied pharmacokinetic properties. One of the first notable examples is palbociclib I,<sup><a href="#cit17" class="usa-link" aria-describedby="cit17">17</a></sup> which became the first FDA-approved CDK4/6 inhibitor. Dalpiciclib V is another potent and orally bioavailable CDK4/6 inhibitor, which differs from palbociclib by featuring a piperidine ring in place of the piperazine ring.</p>
<figure class="fig xbox font-sm" id="fig1"><h3 class="obj_head">Fig. 1. Representative scaffolds of some reported CDK4/6 inhibitors.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377214_d5ra04997k-f1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/387c/12377214/9f20186af2fa/d5ra04997k-f1.jpg" loading="lazy" height="685" width="786" alt="Fig. 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The pyrrolo[2,3-<em>d</em>] pyrimidine scaffold is represented by ribociclib II, the second FDA-approved oral CDK4/6 inhibitor. Birociclib VI is another selective CDK4/6 inhibitor containing this scaffold that has also gained FDA approval.<sup><a href="#cit18" class="usa-link" aria-describedby="cit18">18,19</a></sup></p>
<p>The 6-(pyrimidin-4-yl)-1<em>H</em>-benzo[<em>d</em>]imidazole scaffold is exemplified by abemaciclib III, an oral CDK4/6 inhibitor, approved by the FDA. BPI-16350 VII is a highly potent CDK4/6 inhibitor with structural similarity to abemaciclib.<sup><a href="#cit20" class="usa-link" aria-describedby="cit20">20</a></sup> Finally, the tricyclic lactam scaffold underlies trilaciclib IV, the first FDA-approved CDK4/6 inhibitor for chemotherapy-induced myelosuppression. Lerociclib VIII, a structural analog of trilaciclib, is equally potent and selective for CDK4/6.</p>
<p>Vascular endothelial growth factor receptors (VEGFRs) are crucial receptor protein tyrosine kinases involved in tumor-induced angiogenesis, which are essential for tumor growth, progression, and metastasis.<sup><a href="#cit21" class="usa-link" aria-describedby="cit21">21</a></sup> Targeting the VEGFR-2 signaling pathway is a promising cancer therapy strategy, with FDA-approved VEGFR-2 inhibitors, like the “nibs” sorafenib, foretinib, regorafenib, and tivozanib.<sup><a href="#cit22" class="usa-link" aria-describedby="cit22">22–24</a></sup></p>
<p>Xanthine oxidase (XO) overexpression is linked to cancer due to its role in generating reactive oxygen species (ROS) and activating carcinogens. XO oxidizes hypoxanthine to xanthine and uric acid, producing ROS that promote tumor progression, inflammation, angiogenesis, and cell migration. XO inhibitors can suppress proinflammatory and proapoptotic proteins, potentially aiding cancer treatment. Elevated uric acid levels correlate with poorer survival rates in cancer patients, highlighting XO as a diagnostic and prognostic marker. Although XO inhibitors reduce oxidative stress and inflammation, their impact on cancer progression and metastasis remains largely unexplored.<sup><a href="#cit25" class="usa-link" aria-describedby="cit25">25,26</a></sup></p>
<p>Current studies focus on developing next-generation inhibitors targeting CDK4/6, VEGFR-2, and XO. These targets are critical in cancer progression, with CDK4/6 regulating the cell cycle, VEGFR-2 facilitating angiogenesis, and XO generating ROS.<sup><a href="#cit27" class="usa-link" aria-describedby="cit27">27</a></sup> By targeting all three, it may be possible to enhance anticancer efficacy by inhibiting cell proliferation and tumor blood supply. Specific chemical scaffolds, such as oxindoles, purines, pyrimidines, and pyrazolopyrimidines, are explored for their biological activities. Oxindoles exhibit anticancer properties by inhibiting key enzymes, with C3-substituted derivatives enhancing interactions for novel drug design. FDA-approved oxindole-based drugs, like sunitinib and nintedanib, treat renal cell carcinoma and adenocarcinoma, while indirubin shows CDK inhibition, and toceranib is used in veterinary oncology. Other oxindole inhibitors are orantinib and target receptor tyrosine kinases.<sup><a href="#cit28" class="usa-link" aria-describedby="cit28">28,29</a></sup> Additionally, thiazolidinone and thiazolylidene derivatives, featuring an –N<span aria-roledescription="private character" aria-label="DOUBLE BOND, LENGTH AS M-DASH" aria-description="double bond, length as m-dash">

<svg xmlns="http://www.w3.org/2000/svg" version="1.0" width="13.200000pt" height="16.000000pt" viewBox="0 0 13.200000 16.000000" preserveAspectRatio="xMidYMid meet"><metadata>
Created by potrace 1.16, written by Peter Selinger 2001-2019
</metadata><g transform="translate(1.000000,15.000000) scale(0.017500,-0.017500)" fill="currentColor" stroke="none"><path d="M0 440 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z M0 280 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z"/></g></svg>

</span>C–S– moiety, display diverse biological activities, with ponesimod gaining FDA approval for multiple sclerosis.<sup><a href="#cit30" class="usa-link" aria-describedby="cit30">30,31</a></sup> Purines and pyrimidines, which are fundamental to DNA and RNA, are crucial in drug discovery. Purine derivatives serve as anticancer antimetabolites, while pyrimidines exhibit diverse biological activities. Targeting nucleotide synthesis with analogs enhances chemotherapy.<sup><a href="#cit32" class="usa-link" aria-describedby="cit32">32</a></sup> Pyrazolopyrimidines serve as bioisosteres of adenine, mimic ATP interactions, and function as CDK inhibitors. Allopurinol, a pyrazolopyrimidine, inhibits xanthine oxidase and exhibits cytotoxic effects. Its structural versatility and tautomeric forms enhance its therapeutic potential.<sup><a href="#cit33" class="usa-link" aria-describedby="cit33">33,34</a></sup> In this framework, structure-based drug design (SBDD) and molecular docking were performed to evaluate the binding affinities of the designed compounds within the active site of CDK6, a key enzyme in cancer progression. Their affinities were compared with palbociclib to validate the design strategy.</p></section><section id="sec2"><h2 class="pmc_sec_title">Design rational</h2>
<p>A novel series of compounds incorporating pyrazolo[3,4-<em>d</em>] pyrimidine and aminopurine scaffolds, fused with an oxindole ring, was strategically designed to inhibit CDK6, VEGFR-2, and xanthine oxidase, which are critical enzymes involved in cancer progression. The structural design of these compounds focuses on enhancing both potency and multi-targeted therapeutic effectiveness and is summarized as follows (<a href="#fig2" class="usa-link">Fig. 2</a>).</p>
<figure class="fig xbox font-sm" id="fig2"><h3 class="obj_head">Fig. 2. Feature similarities of the palbociclib ligand and the newly designed compounds 10a–d–13a–d as potent CDK4/6 inhibitors.</h3>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377214_d5ra04997k-f2.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/387c/12377214/52b5e1e48eff/d5ra04997k-f2.jpg" loading="lazy" height="654" width="773" alt="Fig. 2"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><section id="sec3"><h3 class="pmc_sec_title">(1) Scaffold modification</h3>
<p>The pyrido[3,4-<em>d</em>] pyrimidine scaffold of palbociclib was replaced with pyrazolo[3,4-<em>d</em>] pyrimidine and aminopurine, combined with an oxindole ring. This adjustment enables noncovalent bonding with Val101 and Asp163 in the CDK6 active site.</p></section><section id="sec4"><h3 class="pmc_sec_title">(2) Oxindole substitutions</h3>
<p>Variably substituted oxindole rings incorporating electron-donating (EDGs) and electron-withdrawing groups (EWGs) were designed to evaluate the effects of electronic modifications on biological activity.</p></section><section id="sec5"><h3 class="pmc_sec_title">(3) Thiosemicarbazide addition</h3>
<p>A thiosemicarbazide motif was attached at the oxindole's C-3 position to boost the potency of the target compounds.</p></section><section id="sec6"><h3 class="pmc_sec_title">(4) Thiosemicarbazone cyclization</h3>
<p>Cyclized forms, such as 2-hydrazinylidine thiazolidin-4-one and 2-hydrazinylidine-2,3-dihydrothiazole, were introduced at oxindole's C-3 position to enhance binding affinity and potency <em>via</em> alternative interactions.</p>
<p>These design elements maximize structural diversity and binding efficiency, paving the way for novel multi-targeted cancer therapies.</p></section></section><section id="sec7"><h2 class="pmc_sec_title">Results and discussion</h2>
<section id="sec8"><h3 class="pmc_sec_title">Computational studies</h3>
<section id="sec9"><h4 class="pmc_sec_title">Molecular docking of the design compounds onto the CDK4/6 inhibitor</h4>
<p>In our study, we focused on developing new compounds that target the ATP binding site of the CDK6 enzyme. Our approach was inspired by the structural analysis of palbociclib, a known CDK6 inhibitor, which was studied in complex with the enzyme (PDB ID: 5L2I).<sup><a href="#cit35" class="usa-link" aria-describedby="cit35">35</a></sup><a href="#fig3" class="usa-link">Fig. 3</a> shows how palbociclib binds to the ATP active site, serving as a foundation for designing improved CDK6 inhibitors.</p>
<figure class="fig xbox font-sm" id="fig3"><h5 class="obj_head">Fig. 3. Schematic of the binding regions within the ATP active site of CDK6 in complex with palbociclib. The key interaction regions are (1) hinge (adenine) region—Val101, Leu152, Glu99, and Phe98 residues; (2) ribose pocket—Asp163 and Lys43 residues; (3) hydrophobic region—Phe98 and Leu152 residues; (4) solvent region—Asp163 and Lys43 residues; and (5) phosphate pocket—Lys43, Asp145, and Glu61 residues.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377214_d5ra04997k-f3.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/387c/12377214/09be8a77fa94/d5ra04997k-f3.jpg" loading="lazy" height="680" width="779" alt="Fig. 3"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Docking studies for the newly designed compounds 6a–6d–13a–13d were conducted using MOE 2020.01 software, utilizing the three-dimensional structure of the CDK6 enzyme in complex with palbociclib, obtained from the Protein Data Bank (PDB).</p>
<p>To ensure the accuracy of the docking protocol, palbociclib was redocked into the CDK6 active site, yielding a root-mean-square deviation (RMSD) of 1.03 Å and a binding score of −8.23 kcal mol<sup>−1</sup>. The docking procedure successfully reproduced all key interactions observed in the co-crystallized structure, confirming the reliability of the methodology. This validated approach was then employed to investigate the binding interactions of the newly designed compounds, allowing for a comparative analysis of palbociclib within the active site. The co-crystal structure of CDK6 with palbociclib revealed strong interactions within the ATP binding site, contributing to the compound's stability and selectivity. Palbociclib is anchored by three key hydrogen bonds: two with Val101 in the hinge region and one with Asp163 in the DFG (Asp–Phe–Gly) motif. Additionally, it establishes a favorable electrostatic interaction with a solvent-exposed ridge formed by Asp104 and Thr107. Further stabilization is provided by multiple polar interactions between the ribose and phosphate groups and residues Val27 and Ala162. A defining feature of palbociclib is its high specificity for CDK4/6, with minimal activity against other CDKs or non-CDKs. This selectivity is largely attributed to interactions with CDK4/6-specific residues, particularly His100 and Thr107 near the hinge region. Notably, His100 is exclusive to CDK4 and CDK6, further reinforcing its specificity. Additionally, kinase selectivity may be influenced by palbociclib's lipophilicity, as its cLog <em>P</em> value facilitates deeper accommodation within the ATP-binding cleft (Fig. S1, SI).</p>
<p>The designed compounds 6a–6d–13a–13d exhibited a common binding pattern, forming halogen bonds (XB) and/or hydrogen bonds (HB) with Val101, along with hydrogen bonding interactions with Asp163 residues in the hinge (adenine) region, which are critical for CDK6 inhibition. Additionally, these compounds engage in hydrophobic interactions with key gatekeeper residues, including Phe98, Val27, and Ile19. Beyond these core interactions, the new compounds also formed additional contacts with amino acids, such as Asp104, Asn150, Gln149, Gly20, Gly165, Leu96, Leu152, Leu166, Lys43, Lys147, Phe164, and Val77. These extended interactions, facilitated by the molecular design, contributed to the overall stability of the ligand–protein complex. Several compounds, specifically 8b, 9b, 10a–10c, 11b, 12a–12d and 13c–13d, demonstrated binding scores ranging from Δ<em>G</em> −8.29 to −9.00 kcal mol<sup>−1</sup>, surpassing the binding score of palbociclib (−8.23 kcal mol<sup>−1</sup>). In contrast, compounds 6a–6d and 7a-7d exhibited lower binding scores, ranging from −6.95 to −7.45 kcal mol<sup>−1</sup>. However, these compounds still showed an affinity for the ATP binding site of CDK6 by interacting with Val101 and Asp163 and penetrating the ribose pocket through interactions with Leu152, Leu166, and Phe164, as exemplified by compounds 6b and 7b (<a href="#fig4" class="usa-link">Fig. 4</a>). These findings support the synthesis of docked compounds and encourage further evaluation of their potential as CDK6 inhibitors.</p>
<figure class="fig xbox font-sm" id="fig4"><h5 class="obj_head">Fig. 4. 2D interaction of the designed compounds 6b and 7b.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377214_d5ra04997k-f4.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/387c/12377214/6e595b5aa4ad/d5ra04997k-f4.jpg" loading="lazy" height="283" width="736" alt="Fig. 4"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Thiosemicarbazone groups 8a–8d and 9a–9d showed slightly better binding energy, ranging from −7.71 to −8.35 kcal mol<sup>−1</sup>, due to their interaction with essential amino acids Val101 and His100, which matched the binding interaction with palbociclib. Additionally, they bind to extra amino acids, including Leu96, Gln149, Leu152, and Gly165, as depicted in <a href="#fig5" class="usa-link">Fig. 5</a>. The higher Δ<em>G</em> of 8b was attributed to the strong electrostatic Cl⋯O attraction (<em>d</em> = 3.14 Å) contributed to CDK6 inhibition activity.</p>
<figure class="fig xbox font-sm" id="fig5"><h5 class="obj_head">Fig. 5. 2D interaction of the designed compounds 8b and 9b.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377214_d5ra04997k-f5.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/387c/12377214/27cbda55fd81/d5ra04997k-f5.jpg" loading="lazy" height="316" width="718" alt="Fig. 5"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The thiazolidinone moiety of compounds 10a–10d and 11a–11d binds to the crucial amino acids Val101 and Glu99, plus the added interactions with Asn150, Asp 104, Leu96, Leu152, Gly20, Gly 165, and Tyr24. These compounds exhibit a higher binding energy than palbociclib. The sulphur atom within the thiazolylidene moiety in 12a–12d and 13a–13d binds in hydrogen bonding with the solvent region of the CDK6 enzyme's active site, particularly with Asp104. Meanwhile, the thiazolylidene moiety is positioned toward the ribose pocket, facilitating hydrophobic interactions with Gly20. These compounds not only bind to crucial amino acids but also interact with additional residues, like Leu96, Leu166, Phe164, Val77, Gln149, and Gly165. They exhibit the highest binding scores of Δ<em>G</em>s of 12a–12d and 13a–13d among the compounds studied, as shown in <a href="#fig6" class="usa-link">Fig. 6</a>.</p>
<figure class="fig xbox font-sm" id="fig6"><h5 class="obj_head">Fig. 6. 2D interaction of the designed compounds 12b and 13b.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377214_d5ra04997k-f6.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/387c/12377214/1c37db4f9834/d5ra04997k-f6.jpg" loading="lazy" height="368" width="767" alt="Fig. 6"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The 5-chloro-substituent plays a significant role in binding to the CDK6 enzyme's active site. Among the designed compounds, variations at the 5-position include H, Cl, OCH<sub>3</sub>, and F. Notably, compounds featuring 5-chloro substituents exhibit the most favorable binding interactions with the CDK6 enzyme's active site, particularly through interactions with essential residues Val101 and His100. Representative examples of compounds 6b, 6c and 6d contain similar structural substituents but differ solely in the 5-position, Fig. S2, SI.</p>
<p>To assess the selectivity of the newly designed compounds for CDK6 over CDK2, docking studies were conducted using the CDK2-sunitinib complex (PDB ID: 3TI1). The results showed that the tested compounds exhibited different binding modes compared to sunitinib, which interacts with CDK2 through hydrogen bonding with Glu81 and Leu83 in the ATP-binding site, along with hydrophobic interactions involving Phe80, Ile10, and Asp86.</p>
<p>The tested compounds displayed interaction energies ranging from Δ<em>G</em> = −7.07 to −8.23 kcal mol<sup>−1</sup>, while the lead compounds had a slightly stronger interaction energy of Δ<em>G</em> = −8.65 kcal mol<sup>−1</sup>. Notably, compounds 8b, 12a–12c, and 13b failed to establish crucial hydrogen bonds with Leu83 and lacked hydrophobic interactions with Phe80. Similarly, compounds 6b and 7b did not form key hydrogen bonds with Leu83 or interact hydrophobically with Phe80. These findings suggest that the designed compounds exhibit distinct binding preferences, reinforcing their selectivity for CDK6 over CDK2 (Fig. S3, SI).</p></section><section id="sec10"><h4 class="pmc_sec_title">Docking studies of the VEGFR-2 inhibitor</h4>
<p>Docking studies were conducted on the most active compounds 6b, 7b, 8b, 12a–d, and 13b using the VEGFR-2 co-crystal structure with sorafenib (PDB ID: 3WZE)<sup><a href="#cit36" class="usa-link" aria-describedby="cit36">36–39</a></sup> and MOE 2020.01 software. The docking protocol was validated by redocking sorafenib into the VEGFR-2 active site, which yielded an RMSD of 1.02 Å and a binding score of −10.16 kcal mol<sup>−1</sup>, successfully reproducing key interactions with critical binding site residues (<a href="#tab1" class="usa-link">Table 1</a>). Sorafenib interacts with VEGFR-2 through five hydrogen bonds: two with Cys919, two with Glu885, and one with Asp1046. Its <em>N</em>-methyl picolinamide group fits into a pocket formed by Cys919, Phe918, Leu1035, Lys920, Glu917, Val848, and Leu889, while the terminal 3-trifluoromethyl-4-chlorophenyl group occupies a hydrophobic channel involving residues such as Phe1047, Cys1045, and Ile1044. The urea spacer plays a key role in binding, enhancing sorafenib's affinity for VEGFR-2 (<a href="#fig7" class="usa-link">Fig. 7</a>). These findings highlight the potential of modifying linker structures to develop more potent VEGFR-2 inhibitors.</p>
<section class="tw xbox font-sm" id="tab1"><h5 class="obj_head">Table 1. Binding energy score of the docked most active cytotoxic compounds on VEGFR-2.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1">
<col span="1">
<col span="1">
</colgroup>
<thead><tr>
<th rowspan="1" colspan="1">Compound</th>
<th rowspan="1" colspan="1">Binding score Δ<em>G</em> (kcal mol<sup>−1</sup>)</th>
<th rowspan="1" colspan="1">Amino acids involved in interaction</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="1" colspan="1">Sorafenib</td>
<td align="center" rowspan="1" colspan="1">−10.16</td>
<td rowspan="1" colspan="1">Asp1046, Glu885, Cys919, Phe1047, Leu840 and Leu1035</td>
</tr>
<tr>
<td rowspan="1" colspan="1">6b</td>
<td align="center" rowspan="1" colspan="1">−7.68</td>
<td rowspan="1" colspan="1">Asp1046, Glu885, Leu889, Cys1045 and Phe1047</td>
</tr>
<tr>
<td rowspan="1" colspan="1">7b</td>
<td align="center" rowspan="1" colspan="1">−7.33</td>
<td rowspan="1" colspan="1">Glu885, Cys1024, Leu889, Ile888, and His1026</td>
</tr>
<tr>
<td rowspan="1" colspan="1">8b</td>
<td align="center" rowspan="1" colspan="1">−9.13</td>
<td rowspan="1" colspan="1">Asp1046, Glu885, Cys919, Cys1045, Leu840 and Lys868</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12a</td>
<td align="center" rowspan="1" colspan="1">−8.99</td>
<td rowspan="1" colspan="1">Asp1046, Glu885, Cys1045, Leu889, Ile888, Cys1024 and Arg1027</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12b</td>
<td align="center" rowspan="1" colspan="1">−8.84</td>
<td rowspan="1" colspan="1">Asp1046, Glu885, Cys1045, Leu889 and Ile888</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12c</td>
<td align="center" rowspan="1" colspan="1">−8.13</td>
<td rowspan="1" colspan="1">Asp1046, Glu885, Ile888, Cys1024, Ile1025 and Arg1027</td>
</tr>
<tr>
<td rowspan="1" colspan="1">13b</td>
<td align="center" rowspan="1" colspan="1">−7.89</td>
<td rowspan="1" colspan="1">Asp1046, Leu889, Ile888, Glu815, His816 and Lys868</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section><figure class="fig xbox font-sm" id="fig7"><h5 class="obj_head">Fig. 7. 2D and 3D interactions of co-crystallized sorafenib shown in green and red, adopting the Phe1047 “DFG-out” conformation.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377214_d5ra04997k-f7.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/387c/12377214/79de4e7bb2dd/d5ra04997k-f7.jpg" loading="lazy" height="250" width="715" alt="Fig. 7"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Compound 8b exhibited a binding mode closely resembling that of sorafenib, with a binding score of −9.00 kcal mol<sup>−1</sup> and the formation of five hydrogen bonds. The pyrazolopyrimidine ring forms one hydrogen bond with Cys919, while the hydrazino groups establish three hydrogen bonds with Asp1046, Cys1045, and Glu885. Additionally, the sulfur atom of the thiosemicarbazide moiety forms a hydrogen bond with Lys868. The 1-ethyl pyrazolo pyrimidine scaffold fits into a pocket formed by Leu1035, Lys920, Cys919, Phe918, Glu917, Ala866, and Val848, while the terminal thiosemicarbazone tail occupies a hydrophobic channel created by Phe1046, Ile1044, Val916, Val914, and Leu889 (Fig. S4, SI). These key interactions likely contributed to the strong anticancer activity of compound 8b.</p>
<p>The predicted binding mode of compound 12a closely resembles that of sorafenib and 8b, with a binding score of −8.95 kcal mol<sup>−1</sup> and four hydrogen bonds. The pyrazolopyrimidine ring forms hydrogen bonds with Cys1024 and Ile888, while the ethylene linker interacts with Asp1046. Additionally, the NH group of the thiazolylidene forms a hydrogen bond with Glu885, and the sulfur atom establishes another with Cys1045. The 1-ethyl pyrazolopyrimidine and oxindole scaffolds fit within a hydrophobic pocket composed of Arg1027, Ile1025, Leu1019, Ile892, Leu889, and Glu815, while the terminal thiazolylidene tail is accommodated in a hydrophobic channel formed by His1026, Val916, Val899, Val898, and Lys898 (Fig. S5, SI).</p>
<p>Compound 12b displayed a binding score of −8.84 kcal mol<sup>−1</sup> and formed three hydrogen bonds. The ethylene linker interacts with Asp1046, while the NH group and sulfur atom of the thiazolylidene form hydrogen bonds with Glu885 and Cys1045, respectively. Its 1-ethyl pyrazolopyrimidine and oxindole scaffold occupy a hydrophobic pocket consisting of Arg1027, Cys1024, Ile1025, Leu1019, Ile892, Leu889, Lys868, and Glu815. Terminal thiazolylidene tail is positioned within a hydrophobic channel formed by Ile1044, His1026, Val916, Val899, and Val898 (Fig. S6, SI).</p>
<p>These docking results suggest that the linkers used in these compounds align with the urea linker in sorafenib, serving a similar function and enhancing affinity for the VEGFR-2 enzyme. The formation of hydrogen bonds with Cys919, Asp1046, and Glu889, along with key hydrophobic interactions, likely contributes to the strong binding of the heterocyclic pyrazolopyrimidine scaffold to VEGFR-2.</p></section></section><section id="sec11"><h3 class="pmc_sec_title">Molecular dynamics simulation</h3>
<p>Compound 12b exhibited good stability within the CDK6 active site, maintaining an average RMSD of 2.5489 Å, which is comparable to palbociclib (2.4922 Å) (Fig. S7, SI). The interaction energies further supported this stability, with total electrostatic and van der Waals energies of −40.17 kcal mol<sup>−1</sup> for 12b and −36.94 kcal mol<sup>−1</sup> for palbociclib. Both compounds formed hydrogen bonds with key residues, contributing to their stable binding (Fig. S8 and S9, SI). MM-PBSA calculations revealed that 12b had a slightly better binding free energy (−9.3551 kcal mol<sup>−1</sup>) than palbociclib (−9.0804 kcal mol<sup>−1</sup>), aligning with its <em>in vitro</em> inhibitory activity (IC<sub>50</sub> = 0.123 μM). Protein stability assessments showed similar RMSD values (12b: 2.2934 Å, palbociclib: 2.2148 Å) (Fig. S10) and RMSF values (12b, 1.5675 Å; palbociclib, 1.6492 Å) profiles (Fig. S11), indicating the structural stability of the protein. Additional parameters, including the radius of gyration (12b: 19.42 Å, palbociclib: 19.47 Å) (Fig. S12 and S13) and solvent-accessible surface area (SASA ∼ 14 360 Å<sup>2</sup>) (Fig. S14), further confirmed the compactness and stability of the CDK6 complexes.</p>
<p>Overall, the 50 ns molecular dynamics simulation demonstrated that compound 12b exhibits strong stability and binding affinity, highlighting its potential as a potent CDK6 inhibitor (<a href="#fig8" class="usa-link">Fig. 8</a> and <a href="#fig9" class="usa-link">9</a>).</p>
<figure class="fig xbox font-sm" id="fig8"><h4 class="obj_head">Fig. 8. Snapshots of palbociclib inside the active site of CDK6 over 50 ns long MDS (5, 10, 20, 30, 40 and 50 ns).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377214_d5ra04997k-f8.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/387c/12377214/27c0a34b89db/d5ra04997k-f8.jpg" loading="lazy" height="516" width="779" alt="Fig. 8"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig8/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><figure class="fig xbox font-sm" id="fig9"><h4 class="obj_head">Fig. 9. Snapshots of compound 12b inside the active site of CDK6 over 50 ns long MDS (5, 10, 20, 30, 40 and 50 ns).</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377214_d5ra04997k-f9.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/387c/12377214/eb99bc6c2765/d5ra04997k-f9.jpg" loading="lazy" height="392" width="754" alt="Fig. 9"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig9/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure></section><section id="sec12"><h3 class="pmc_sec_title">Chemistry</h3>
<p>Target molecules 6a–d–13a–d were synthesized as outlined in <a href="#sch1" class="usa-link">Scheme 1</a>, and their structures were confirmed through spectral analyses (Fig. S15–S30).</p>
<figure class="fig xbox font-sm" id="sch1"><h4 class="obj_head">Scheme 1. Synthesis of the target compounds 6a–d–13a–d. (i) K<sub>2</sub>CO<sub>3</sub>, 1,2-dibromoethane, DMF, r.t., 24 h; (ii) K<sub>2</sub>CO<sub>3</sub>, DMF, 60 °C, 6–8 h; (iii) thiosemicarbazide, EtOH, reflux, 2–4 h; (iv) ethyl bromoacetate or monochloroacetic acid, sod. acetate, EtOH, reflux, 24 h; and (v) chloroacetone, sod. acetate, EtOH, reflux, 24 h.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377214_d5ra04997k-s1.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/387c/12377214/13358a5fb7b4/d5ra04997k-s1.jpg" loading="lazy" height="566" width="749" alt="Scheme 1"></a></p>
<div class="p text-right font-secondary"><a href="figure/sch1/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>It has been reported that the reaction of isatin with thiosemicarbazide is stereoselective, producing only the <em>Z</em>-diastereomer. The chemical shifts of the synthesized compounds 8a–8d–9a–9d are comparable with the reported <sup>1</sup>H NMR of <em>Z</em>-thiosemicarbazone<sup><a href="#cit40" class="usa-link" aria-describedby="cit40">40,41</a></sup> (Fig. S31–S46). These compounds exhibit highly deshielded hydrazine protons resonated at a far downfield shift around <em>δ</em> 12.22–12.09 ppm stemming from the strong intramolecular hydrogen bond formed with the lactam oxygen atom of oxindole (N⋯H⋯O <em>d</em> = 2.09 Å, and <em>Φ</em> = 130°) to stabilize the <em>Z</em> configuration,<sup><a href="#cit42" class="usa-link" aria-describedby="cit42">42</a></sup> as depicted in <a href="#fig10" class="usa-link">Fig. 10</a>.</p>
<figure class="fig xbox font-sm" id="fig10"><h4 class="obj_head">Fig. 10. Representative example for the <em>Z</em> configuration of 9b.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377214_d5ra04997k-f10.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/387c/12377214/2dce9e7bad22/d5ra04997k-f10.jpg" loading="lazy" height="487" width="685" alt="Fig. 10"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig10/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The cyclization of the SCNH<sub>2</sub> moiety was performed using ethyl bromoacetate and a catalytic amount of anhydrous sodium acetate, yielding oxindole-allopurinol-2-hydrazonothiazolidin-4-one 10a–10d. Alternatively, cyclization in the presence of monochloroacetic acid resulted in oxindole-adenine-2-hydrazonothiazolidin-4-one 11a-11d.</p>
<p>Further modification of the SCNH<sub>2</sub> cyclization polarity was achieved using chloroacetone, leading to the formation of oxindole-allopurinol/adenine thiazol-2(3<em>H</em>)-ylidenes 12a–12d and 13a–13d. The structures and purity of the synthesized compounds were confirmed through spectral and elemental analyses (Fig. S47–S77).</p>
<p>As expected, the restricted rotation of the thiazolylidene ring and the imine group in 12a–12d and 13a–13d revealed <em>E</em> and <em>Z</em> diasteroisomers. The proton peaks of the <em>E</em>-diastereomer are significantly downfield shifted relative to those of the <em>Z</em>-diastereomer. For example, in the <sup>1</sup>H-NMR spectrum of compound 13a<em>E</em>/<em>Z</em>%, 38 : 62, the formation of the thiazolyidene ring is confirmed by the presence of two characteristic singlet peaks for SCH<sub>(</sub><em><sub>E</sub></em><sub>)</sub> and SCH<sub>(</sub><em><sub>Z</sub></em><sub>)</sub> at 6.51 and 6.36 ppm, respectively. Additionally, two characteristic doublet peaks were assigned to NCH<sub>(</sub><em><sub>E</sub></em><sub>)</sub> and NCH<sub>(</sub><em><sub>Z</sub></em><sub>)</sub> at 7.15 and 6.89 ppm, respectively. The chemical shift of the indole proton at C4H is particularly useful for differentiating between <em>E</em>- and <em>Z</em>-diastereomers. The C4H proton of the <em>E</em>-diastereomer resonates more downfield, appearing as a doublet around <em>δ</em> 8.26 ppm, while the C4H proton of the <em>Z</em>-diastereomer resonates relatively upfield, appearing as a doublet around <em>δ</em> 7.48 ppm. The same pattern was observed in <sup>13</sup>C-NMR spectra, NCH<sub>(</sub><em><sub>E</sub></em><sub>)</sub> and NCH<sub>(</sub><em><sub>Z</sub></em><sub>)</sub>, around <em>δ</em> 122.8 and 121.8 ppm, respectively, and was observed in C4<sub>(</sub><em><sub>E</sub></em><sub>)</sub> and C4<sub>(</sub><em><sub>Z</sub></em><sub>)</sub> around <em>δ</em> 130.0 and 129.5 ppm, respectively. The differentiation of <em>E</em>- and <em>Z</em>-diastereomers can be effectively achieved using 2D-NMR techniques. The <sup>1</sup>H–<sup>1</sup>H NOESY experiment confirmed the presence of both diastereomers (<em>E</em>/<em>Z</em>) in isatin-thiazol-2(3<em>H</em>)ylidene, enabling the assignment of <em>E</em> and <em>Z</em> protons. As a representative example, compound 10b exhibited no NOE cross-peak between the NH and C4–H protons, indicating that these groups are spatially separated and oriented differently. Additionally, the presence of cross-peaks between the C7–H and N1–CH<sub>2</sub> protons of oxindole further supports the <em>Z</em>-configuration of compound 10b, which can be extrapolated to other derivatives, as illustrated in <a href="#fig11" class="usa-link">Fig. 11</a>.</p>
<figure class="fig xbox font-sm" id="fig11"><h4 class="obj_head">Fig. 11. 
<sup>1</sup>H–<sup>1</sup>H-NOESY spectrum of compound 10b.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377214_d5ra04997k-f11.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/387c/12377214/a9794cca0dd2/d5ra04997k-f11.jpg" loading="lazy" height="526" width="759" alt="Fig. 11"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig11/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>The purity of the synthesized compounds was confirmed by elemental analysis, with deviations within acceptable limits (±0.4%). Furthermore, HPLC chromatograms of representative compounds 6b, 8b, 12a, 12b, and 12c, using allopurinol as an internal standard, indicated high chemical purity, with all tested leads showing purity ≥95% and several exceeding 97% at their respective retention times. These findings confirm the successful synthesis and isolation of highly pure compounds suitable for subsequent biological studies and formulation efforts. Detailed chromatographic data are provided in the SI (Fig. S78–S82).</p></section></section><section id="sec13"><h2 class="pmc_sec_title">Biological evaluation</h2>
<section id="sec14"><h3 class="pmc_sec_title">Antiproliferative activity assay</h3>
<p>The antiproliferative activities of allopurinol-isatin and adenine-isatin derivatives 6a–6d and 7a–7d, along with thiosemicarbazone derivatives 8a–8d and 9a–9d, and cyclized products 2-hydrazinylidine thiazolidin-4-one 10a–10d and 11a–11d, as well as 2-hydrazinylidine-2,3-dihydrothiazole 12a–12d and 13a–13d, were evaluated against three cancer cell lines: A498 (kidney carcinoma), HepG-2 (hepatocellular carcinoma), and MDA-MB-231 (breast adenocarcinoma). Additionally, their effects on the normal murine embryonic fibroblast cell line BALB/3T3 (clone A31) were assessed using the sulforhodamine B (SRB) assay to determine antiproliferative activity compared to the reference drug sorafenib. The results shown in <a href="#tab2" class="usa-link">Tables 2</a>, S1 and Fig. S83 clearly reveal that most of the synthesized compounds have pIC<sub>50</sub> in the range of 4.03–5.82, matching or higher than the reference sorafenib. Of these active compounds, 30 structures carry the allopurinol feature and nine carry the adenine. Among series 6a–6d and 7a–7d, the most active ligands are those with a higher lipophilic nature, carrying 5-chloroindoline-2-one fragment 6b (clog <em>P</em> = 0.77) and 7b (clog <em>P</em> = 1.54). The allopurinol compound 6b R<sub>1</sub> = Cl showed the most distinct effect against the three cancer cell lines pIC<sub>50</sub> (4.79, 4.82, and 4.88), and in the adenine, series 7b showed antiproliferative effects against HepG2 and MDA-MB-231 pIC<sub>50</sub> (5.22 and 4.71, respectively). Both most probably privileged the contribution of halogen bond formation that improved the binding affinity of ligands to the residues in the active site compared to 6d pIC<sub>50</sub> (4.21, 4.32, and 4.41) and 7d with promising activity against the three cancer cell lines pIC<sub>50</sub> (3.78, 4.76, and 4.08).</p>
<section class="tw xbox font-sm" id="tab2"><h4 class="obj_head">Table 2. Antiproliferative activity of the synthesized compounds against A-498, HepG2, and MDA-MB-231 tumor cell lines using SRB assay with sorafenib as a positive control.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
</colgroup>
<thead>
<tr>
<th rowspan="2" colspan="1">Entry</th>
<th rowspan="2" colspan="1">R<sub>1</sub>
</th>
<th colspan="3" rowspan="1">pIC<sub>50</sub><a href="#tab2fnb" class="usa-link"><sup>b</sup></a>
</th>
<th rowspan="2" colspan="1">Entry</th>
<th colspan="3" rowspan="1">pIC<sub>50</sub><a href="#tab2fnb" class="usa-link"><sup>b</sup></a>
</th>
</tr>
<tr>
<th rowspan="1" colspan="1">A-498</th>
<th rowspan="1" colspan="1">HepG2</th>
<th rowspan="1" colspan="1">MDA-MB-231</th>
<th rowspan="1" colspan="1">A-498</th>
<th rowspan="1" colspan="1">HepG2</th>
<th rowspan="1" colspan="1">MDA-MB-231</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="1" colspan="1">6a</td>
<td rowspan="1" colspan="1">H</td>
<td rowspan="1" colspan="1">4.23</td>
<td rowspan="1" colspan="1">3.82</td>
<td rowspan="1" colspan="1">4.04</td>
<td rowspan="1" colspan="1">7a</td>
<td rowspan="1" colspan="1">n.a.<a href="#tab2fna" class="usa-link"><sup>a</sup></a>
</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">3.68</td>
</tr>
<tr>
<td rowspan="1" colspan="1">6b</td>
<td rowspan="1" colspan="1">Cl</td>
<td rowspan="1" colspan="1">4.79</td>
<td rowspan="1" colspan="1">4.82</td>
<td rowspan="1" colspan="1">4.88</td>
<td rowspan="1" colspan="1">7b</td>
<td rowspan="1" colspan="1">4.28</td>
<td rowspan="1" colspan="1">5.22</td>
<td rowspan="1" colspan="1">4.71</td>
</tr>
<tr>
<td rowspan="1" colspan="1">6c</td>
<td rowspan="1" colspan="1">OCH<sub>3</sub>
</td>
<td rowspan="1" colspan="1">4.52</td>
<td rowspan="1" colspan="1">4.03</td>
<td rowspan="1" colspan="1">4.32</td>
<td rowspan="1" colspan="1">7c</td>
<td rowspan="1" colspan="1">3.74</td>
<td rowspan="1" colspan="1">4.11</td>
<td rowspan="1" colspan="1">3.87</td>
</tr>
<tr>
<td rowspan="1" colspan="1">6d</td>
<td rowspan="1" colspan="1">F</td>
<td rowspan="1" colspan="1">4.41</td>
<td rowspan="1" colspan="1">4.21</td>
<td rowspan="1" colspan="1">4.32</td>
<td rowspan="1" colspan="1">7d</td>
<td rowspan="1" colspan="1">3.78</td>
<td rowspan="1" colspan="1">4.76</td>
<td rowspan="1" colspan="1">4.08</td>
</tr>
<tr>
<td rowspan="1" colspan="1">8a</td>
<td rowspan="1" colspan="1">H</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">9a</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
</tr>
<tr>
<td rowspan="1" colspan="1">8b</td>
<td rowspan="1" colspan="1">Cl</td>
<td rowspan="1" colspan="1">4.85</td>
<td rowspan="1" colspan="1">4.88</td>
<td rowspan="1" colspan="1">4.27</td>
<td rowspan="1" colspan="1">9b</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
</tr>
<tr>
<td rowspan="1" colspan="1">8c</td>
<td rowspan="1" colspan="1">OCH<sub>3</sub>
</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">9c</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">4.53</td>
<td rowspan="1" colspan="1">4.14</td>
</tr>
<tr>
<td rowspan="1" colspan="1">8d</td>
<td rowspan="1" colspan="1">F</td>
<td rowspan="1" colspan="1">3.98</td>
<td rowspan="1" colspan="1">4.60</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">9d</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">4.25</td>
<td rowspan="1" colspan="1">n.a.</td>
</tr>
<tr>
<td rowspan="1" colspan="1">10a</td>
<td rowspan="1" colspan="1">H</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">11a</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
</tr>
<tr>
<td rowspan="1" colspan="1">10b</td>
<td rowspan="1" colspan="1">Cl</td>
<td rowspan="1" colspan="1">4.03</td>
<td rowspan="1" colspan="1">5.27</td>
<td rowspan="1" colspan="1">4.29</td>
<td rowspan="1" colspan="1">11b</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
</tr>
<tr>
<td rowspan="1" colspan="1">10c</td>
<td rowspan="1" colspan="1">OCH<sub>3</sub>
</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">11c</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
</tr>
<tr>
<td rowspan="1" colspan="1">10d</td>
<td rowspan="1" colspan="1">F</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">11d</td>
<td rowspan="1" colspan="1">3.72</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">3.72</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12a</td>
<td rowspan="1" colspan="1">H</td>
<td rowspan="1" colspan="1">5.12</td>
<td rowspan="1" colspan="1">5.82</td>
<td rowspan="1" colspan="1">5.35</td>
<td rowspan="1" colspan="1">13a</td>
<td rowspan="1" colspan="1">3.72</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12b</td>
<td rowspan="1" colspan="1">Cl</td>
<td rowspan="1" colspan="1">5.29</td>
<td rowspan="1" colspan="1">5.76</td>
<td rowspan="1" colspan="1">5.41</td>
<td rowspan="1" colspan="1">13b</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
<td rowspan="1" colspan="1">n.a.</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12c</td>
<td rowspan="1" colspan="1">OCH<sub>3</sub>
</td>
<td rowspan="1" colspan="1">4.26</td>
<td rowspan="1" colspan="1">5.25</td>
<td rowspan="1" colspan="1">4.81</td>
<td rowspan="1" colspan="1">13c</td>
<td rowspan="1" colspan="1">3.98</td>
<td rowspan="1" colspan="1">3.84</td>
<td rowspan="1" colspan="1">3.75</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12d</td>
<td rowspan="1" colspan="1">F</td>
<td rowspan="1" colspan="1">4.51</td>
<td rowspan="1" colspan="1">5.21</td>
<td rowspan="1" colspan="1">4.93</td>
<td rowspan="1" colspan="1">13d</td>
<td rowspan="1" colspan="1">3.85</td>
<td rowspan="1" colspan="1">3.72</td>
<td rowspan="1" colspan="1">n.a.</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Sorafenib</td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1">4.20</td>
<td rowspan="1" colspan="1">4.35</td>
<td rowspan="1" colspan="1">4.09</td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
<td rowspan="1" colspan="1"></td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab2/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div>
<div class="tw-foot p">
<div class="fn" id="tab2fna">
<sup>a</sup><p class="display-inline">n.a. – not active.</p>
</div>
<div class="fn" id="tab2fnb">
<sup>b</sup><p class="display-inline">pIC<sub>50</sub> = −log IC<sub>50</sub> (molar concentration).</p>
</div>
</div></section><p>However, the other substituents H and OMe in 6a, 6c, 7a and 7c displayed relatively lower potency against the three cell lines.</p>
<p>The attachment of the thiosemicarbazide fragment to the ligands 6a–6d and 7a–7d demoted the scope and potency of the produced thiosemicarbazones against the three cancer cell lines. Nonetheless, the thiosemicarbazones 8b, 8d, and 9c showed <em>potent antiproliferative activity</em> pIC<sub>50</sub> (4.53–4.88) against the A-498 and HepG2 cell lines. Further modifications of 8a–8d and 9a–9d were conducted <em>via</em> two cyclization reactions of the SCNH<sub>2</sub> moiety. The first cyclized moiety was 2-hydrazinylidenethiazolidin-4-ones 10a–10d and 11a–11d. Incorporation of the lactam scaffold resulted in a reduction of one hydrogen bond donor (HBD) and an increase of two hydrogen bond acceptors (HBAs) per molecule relative to precursors 8a–8d and 9a–9d. The conversion of the –SCNH<sub>2</sub> functionality into a lactam framework led to a restricted antiproliferative profile and diminished cytotoxic potency when compared with the corresponding thiosemicarbazone analogues. An exception was 10b, which displayed high potency (pIC<sub>50</sub> = 5.27) against the HepG-2 cell line, and moderate potency against A-498 and MDA-MB-231 (pIC<sub>50</sub> = 4.03 and 4.29, respectively). The other compounds in this series exhibited lower activity. This observed inconsistency is most probably attributed to the spatial orientation of the majority of the 2-iminothiazolidin-4-one moiety within the active site of the targeted cell lines, which favors the two effectors, allopurinol and 5-chloroxindole nuclei. To improve the activities of 10a–10d and 11a–11d, we intended to attenuate the polarity and to avoid the HBA motif provided by the O in the lactam fragment. This goal was fulfilled by the cyclization of the terminal SCNH<sub>2</sub> fragment following the procedure outlined in <a href="#sch1" class="usa-link">Scheme 1</a> to yield a series of allopurinol/adenine-oxindole 2-hydrazinylideno thiazole 12a–12d and 13a–13d. Fortunately, the antiproliferative potency and scope of action of the allopurinol series shown by 12a–12d have surpassed the pIC<sub>50</sub> of the reference drug sorafenib. The activities of 12a and 12b against A-498 are 2-fold, against HepG2 are 8-fold, and against MDA-MB-231 are 5-fold those of sorafenib, which exhibits low cytotoxicity towards BALB/3T3 cells, indicating safety for normal cells. The activities of 12c and 12d against A-498 are 8-fold, against HepG2 are 30-fold, and against MDA-MB-231 are 20-fold those of sorafenib. However, adenine series 13a–13d showed marginal activity against any one of the cell lines. The observed strong anticancer activity of compounds 12a–12d can be attributed to the robust interactions of the motifs attached to the terminals of the dimethylene linker, namely the pyrazolo pyrimidinone, on one hand, and the fragments attached to the oxindole nucleus, on the other hand, showing multiple binding forces in the active site of the CDK6 enzyme. This interaction is further supported by their high binding scores, confirming their strong affinity for the target.</p>
<p>The selectivity indices (SI) were calculated for the most active compounds using the ratio SI = (IC<sub>50</sub> μM against normal cell line BALB/3T3)/(IC<sub>50</sub> μM against respective cancerous cell line) to evaluate the toxicity of tested compounds against normal cells and to predict their therapeutic potential. A favorable SI &gt; 1.0 indicates a drug with efficacy against tumor cells greater than the toxicity against normal cells. The most active compounds 6b, 7b, 8b and 12a–12d displayed an SI median of 2.8 against A498; against HepG2, 6a, 7b, and 8b showed an SI median of 2.03; and against MDA-MB-231 compounds, 6b, 7b and 12b showed an SI median of 2.49 (<a href="#tab3" class="usa-link">Table 3</a>). The most active compounds with pIC<sub>50</sub> ≥ 4.5 showed SI values higher than the reference, which entitled them to further development as therapeutic candidates, especially for targeting the HepG2 cell line.</p>
<section class="tw xbox font-sm" id="tab3"><h4 class="obj_head">Table 3. SI of the most active compounds 6b, 7b, 8b, 12a–d and sorafenib.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
</colgroup>
<thead>
<tr>
<th rowspan="1" colspan="1">Entry</th>
<th colspan="3" rowspan="1">Selectivity index (SI)</th>
</tr>
<tr>
<th rowspan="1" colspan="1">Cell lines</th>
<th rowspan="1" colspan="1">A498</th>
<th rowspan="1" colspan="1">HepG2</th>
<th rowspan="1" colspan="1">MDA-MB-231</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="1" colspan="1">6b</td>
<td align="center" rowspan="1" colspan="1">2.03</td>
<td align="center" rowspan="1" colspan="1">2.15</td>
<td align="center" rowspan="1" colspan="1">2.49</td>
</tr>
<tr>
<td rowspan="1" colspan="1">7b</td>
<td align="center" rowspan="1" colspan="1">1.37</td>
<td align="center" rowspan="1" colspan="1">11.89</td>
<td align="center" rowspan="1" colspan="1">3.64</td>
</tr>
<tr>
<td rowspan="1" colspan="1">8b</td>
<td align="center" rowspan="1" colspan="1">2.45</td>
<td align="center" rowspan="1" colspan="1">2.60</td>
<td align="center" rowspan="1" colspan="1">0.63</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12a</td>
<td align="center" rowspan="1" colspan="1">0.59</td>
<td align="center" rowspan="1" colspan="1">2.95</td>
<td align="center" rowspan="1" colspan="1">1.00</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12b</td>
<td align="center" rowspan="1" colspan="1">0.79</td>
<td align="center" rowspan="1" colspan="1">2.80</td>
<td align="center" rowspan="1" colspan="1">1.25</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12c</td>
<td align="center" rowspan="1" colspan="1">0.29</td>
<td align="center" rowspan="1" colspan="1">9.92</td>
<td align="center" rowspan="1" colspan="1">0.99</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12d</td>
<td align="center" rowspan="1" colspan="1">0.35</td>
<td align="center" rowspan="1" colspan="1">1.78</td>
<td align="center" rowspan="1" colspan="1">0.91</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Sorafenib</td>
<td align="center" rowspan="1" colspan="1">0.014</td>
<td align="center" rowspan="1" colspan="1">0.02</td>
<td align="center" rowspan="1" colspan="1">0.011</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab3/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec15"><h3 class="pmc_sec_title">Cell cycle analysis</h3>
<p>Cell cycle distribution for compounds 6b, 7b, 12a, and 12c was assessed at two concentrations using the MV4-11 cell line, corresponding to their anti-proliferative IC<sub>50</sub> values (5.85, 11.77, 0.29, and 0.34 μM), with comparisons to a control group. These compounds, which exhibited potent cytotoxicity, were chosen for further investigation of their effects on cell cycle progression. The results, presented in <a href="#tab4" class="usa-link">Table 4</a> and Fig. S84, reveal that compounds 12a and 12c notably reduced the percentage of cells in the G2/M phase (6.64% and 10.59%, respectively) compared to the control group (20.98%). Additionally, the percentage of cells in the subG1 phase increased by approximately 4- and 3-fold, relative to the control, indicating that these compounds induced cell arrest at the subG1 phase. At higher concentrations (1 μg mL<sup>−1</sup>), compounds 12a and 12c caused an increase in cell death, as evidenced by a higher proportion of cells in the subG1 phase, reflecting an enhanced cytotoxic effect. In contrast, compound 7b at 10 μg mL<sup>−1</sup> caused a statistically significant (<em>p</em> &lt; 0.05) increase in the S phase by 1.49-fold more than the control and a decrease in the G2/M phase but did not induce cell death, as indicated by the subG1 percentage (6.15 ± 2.74%). The subG1 checkpoint prevents the replication of damaged DNA, allowing repair and inhibiting the proliferation of damaged cells. Arrest at this phase indicates severe DNA damage, potentially triggering apoptosis. The sub G1 phase arrest induced by compounds 12a and 12c highlights their selectivity for CDK6, a key regulator of the G1 phase transition. Palbociclib was reported to induce a cell cycle arrest at the G1 phase in the MV4-11 cancer cell line.<sup><a href="#cit43" class="usa-link" aria-describedby="cit43">43</a></sup></p>
<section class="tw xbox font-sm" id="tab4"><h4 class="obj_head">Table 4. Cell cycle analysis results presented as the means of cell percentage [%] ± SD.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
</colgroup>
<thead>
<tr>
<th rowspan="2" colspan="1">Entry</th>
<th colspan="4" rowspan="1">Cell percentage [%] ± SD</th>
</tr>
<tr>
<th rowspan="1" colspan="1">SubG1</th>
<th rowspan="1" colspan="1">G0/G1</th>
<th rowspan="1" colspan="1">S</th>
<th rowspan="1" colspan="1">G2/M</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="1" colspan="1">Ctrl</td>
<td align="center" rowspan="1" colspan="1">7.12 ± 0.569</td>
<td align="center" rowspan="1" colspan="1">45.40 ± 3.051</td>
<td align="center" rowspan="1" colspan="1">24.54 ± 2.475</td>
<td align="center" rowspan="1" colspan="1">20.98 ± 0.614</td>
</tr>
<tr>
<td rowspan="1" colspan="1">6b, 1 μg mL<sup>−1</sup>
</td>
<td align="center" rowspan="1" colspan="1">3.48 ± 1.178</td>
<td align="center" rowspan="1" colspan="1">50.37 ± 4.619</td>
<td align="center" rowspan="1" colspan="1">27.15 ± 3.651</td>
<td align="center" rowspan="1" colspan="1">17.67 ± 1.285</td>
</tr>
<tr>
<td rowspan="1" colspan="1">7b, 10 μg mL<sup>−1</sup>
</td>
<td align="center" rowspan="1" colspan="1">6.15 ± 2.739</td>
<td align="center" rowspan="1" colspan="1">44.36 ± 4.193</td>
<td align="center" rowspan="1" colspan="1">36.66 ± 4.326</td>
<td align="center" rowspan="1" colspan="1">13.32 ± 2.763</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12a, 1 μg mL<sup>−1</sup>
</td>
<td align="center" rowspan="1" colspan="1">31.28 ± 10.88</td>
<td align="center" rowspan="1" colspan="1">35.43 ± 5.182</td>
<td align="center" rowspan="1" colspan="1">24.34 ± 8.875</td>
<td align="center" rowspan="1" colspan="1">6.64 ± 2.959</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12c, 1 μg mL<sup>−1</sup>
</td>
<td align="center" rowspan="1" colspan="1">21.16 ± 6.283</td>
<td align="center" rowspan="1" colspan="1">38.41 ± 4.391</td>
<td align="center" rowspan="1" colspan="1">29.22 ± 4.063</td>
<td align="center" rowspan="1" colspan="1">10.59 ± 1.830</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab4/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec16"><h3 class="pmc_sec_title">Enzymatic caspase-3/7 activity assay</h3>
<p>Apoptosis is essential for controlling tumor growth and treatment efficacy. Many cancer therapies target this process by activating apoptotic pathways. A central mediator of apoptosis is <em>caspase-3</em>, a protein produced by the <em>CASP3</em> gene, which plays a pivotal role in executing programmed cell death.<sup><a href="#cit44" class="usa-link" aria-describedby="cit44">44</a></sup> In this study, compounds 6b, 7b, 12a, and 12c were tested. Based on the test results, hepatic cancer lines (HepG2) were selected as the targeted model due to their high sensitivity to the synthesized derivatives, as well as the presence of relevant caspases, such as caspase-3/7, and their expression of multiple metabolizing enzymes.<sup><a href="#cit45" class="usa-link" aria-describedby="cit45">45</a></sup> Evaluating the activity of the released caspase-3 is considered the most accurate method to examine the sound effects of the most active compounds 6b, 7b, 12a and 12c on the apoptotic role of caspase 3. The results presented in <a href="#tab5" class="usa-link">Table 5</a> and Fig. S85 reveal the capability of compound 7b at a 5 μg mL<sup>−1</sup> dose to elevate the activity of caspase-3/7 more than that of the control. Additionally, 6b, 12a, and 12c did not increase caspase 3/7 activity, compared to the control. Among these compounds, 7b, containing an adenine nucleus, induced apoptosis through the contribution of caspase-3/7 activity slightly more than those containing allopurinol. The results suggest that caspase activation is not a primary mechanism of action for most of the tested compounds.</p>
<section class="tw xbox font-sm" id="tab5"><h4 class="obj_head">Table 5. Relative caspase-3/7 activity compared with the control against HepG2 cells.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1">
<col span="1">
</colgroup>
<thead><tr>
<th rowspan="1" colspan="1">Entry</th>
<th rowspan="1" colspan="1">Relative caspase-3/7 activity ± SD</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="1" colspan="1">Ctrl</td>
<td rowspan="1" colspan="1">1</td>
</tr>
<tr>
<td rowspan="1" colspan="1">6b, 5 μg mL<sup>−1</sup>
</td>
<td rowspan="1" colspan="1">0.982 ± 0.128</td>
</tr>
<tr>
<td rowspan="1" colspan="1">7b, 5 μg mL<sup>−1</sup>
</td>
<td rowspan="1" colspan="1">1.296 ± 0.190</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12a, 5 μg mL<sup>−1</sup>
</td>
<td rowspan="1" colspan="1">0.840 ± 0.200</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12c, 5 μg mL<sup>−1</sup>
</td>
<td rowspan="1" colspan="1">0.927 ± 0.185</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab5/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec17"><h3 class="pmc_sec_title">
<em>In vitro</em> CDK6 inhibitory assay</h3>
<p>The most active towards cell proliferation compounds 6b, 7b, 8b and 12a–12c were screened owing to their potential inhibition of CDK6 enzyme using palbociclib as the reference drug, as shown in <a href="#tab6" class="usa-link">Table 6</a> and Fig. S86. Fortunately, all compounds proved to be potent inhibitors with pIC<sub>50</sub> ranging from 5.82 ± 0.059 to 7.17 ± 0.003. Notably, compound 12a exhibited pIC<sub>50</sub>, matching the reference palbociclib (pIC<sub>50</sub> 6.79 ± 0.006), while compounds 8b, 12b and 12c showed a higher pIC<sub>50</sub> range of 6.91 ± 0.048 to 7.17 ± 0.003. These results position these compounds as strong candidates for further exploration and development as potential therapeutic CDKI leads. The lowest CDK6 inhibition effect of compound 7b has a pIC<sub>50</sub> value of 5.82. In the studied series, the most active compounds 12a–12d bind with the assigned fingerprint amino acids of the targeted active site. Furthermore, they bind with His100, which denotes selectivity for the CDK6 inhibitor. Overall, the ligand–protein binding pattern of 12a–12d with some extra amino acids out of the fingerprint residues kept within the ATP pocket enhances the binding affinity to the target site, possibly enhancing their inhibitory efficacy.</p>
<section class="tw xbox font-sm" id="tab6"><h4 class="obj_head">Table 6. 
<em>In vitro</em> inhibitory activities of 6b, 7b, 8b, 12a–c, and 13b against CDK6, VEGFR-2 and xanthine oxidase enzymes compared with palbociclib, sorafenib and allopurinol reference compounds, respectively.</h4>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
</colgroup>
<thead><tr>
<th rowspan="1" colspan="1">Entry</th>
<th rowspan="1" colspan="1">pIC<sub>50</sub> CDK6 ± SD</th>
<th rowspan="1" colspan="1">pIC<sub>50</sub> VEGFR-2 ± SD</th>
<th rowspan="1" colspan="1">pIC<sub>50</sub> XO ± SD</th>
</tr></thead>
<tbody>
<tr>
<td rowspan="1" colspan="1">6a</td>
<td rowspan="1" colspan="1">6.22 ± 0.023</td>
<td rowspan="1" colspan="1">6.51 ± 0.012</td>
<td rowspan="1" colspan="1">4.36 ± 0.55</td>
</tr>
<tr>
<td rowspan="1" colspan="1">7b</td>
<td rowspan="1" colspan="1">5.82 ± 0.059</td>
<td rowspan="1" colspan="1">6.29 ± 0.02</td>
<td rowspan="1" colspan="1">4.18 ± 0.82</td>
</tr>
<tr>
<td rowspan="1" colspan="1">8b</td>
<td rowspan="1" colspan="1">7.17 ± 0.003</td>
<td rowspan="1" colspan="1">7.15 ± 0.003</td>
<td rowspan="1" colspan="1">4.66 ± 0.33</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12a</td>
<td rowspan="1" colspan="1">6.81 ± 0.006</td>
<td rowspan="1" colspan="1">7.27 ± 0.002</td>
<td rowspan="1" colspan="1">4.94 ± 0.16</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12b</td>
<td rowspan="1" colspan="1">6.91 ± 0.048</td>
<td rowspan="1" colspan="1">6.92 ± 0.006</td>
<td rowspan="1" colspan="1">4.51 ± 0.62</td>
</tr>
<tr>
<td rowspan="1" colspan="1">12c</td>
<td rowspan="1" colspan="1">6.93 ± 0.005</td>
<td rowspan="1" colspan="1">6.84 ± 0.009</td>
<td rowspan="1" colspan="1">4.83 ± 0.23</td>
</tr>
<tr>
<td rowspan="1" colspan="1">13b</td>
<td rowspan="1" colspan="1">6.09 ± 0.032</td>
<td rowspan="1" colspan="1">6.49 ± 0.012</td>
<td rowspan="1" colspan="1">4.41 ± 0.49</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Palbociclib</td>
<td rowspan="1" colspan="1">6.79 ± 0.006</td>
<td rowspan="1" colspan="1">—</td>
<td rowspan="1" colspan="1">—</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Sorafenib</td>
<td rowspan="1" colspan="1">—</td>
<td rowspan="1" colspan="1">7.25 ± 0.002</td>
<td rowspan="1" colspan="1">—</td>
</tr>
<tr>
<td rowspan="1" colspan="1">Allopurinol</td>
<td rowspan="1" colspan="1">—</td>
<td rowspan="1" colspan="1">—</td>
<td rowspan="1" colspan="1">4.57 ± 0.13</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab6/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section><section id="sec18"><h3 class="pmc_sec_title">
<em>In vitro</em> VEGFR-2 inhibitory assay</h3>
<p>The most potent antitumor derivatives 6b, 7b, 8b and 12a–12c were tested for VEGFR-2 kinase inhibition using sorafenib as the reference compound. The results of pIC<sub>50</sub> values are presented in <a href="#tab6" class="usa-link">Table 6</a> and Fig. S86. Compounds 8b and 12a showed the most potent inhibition of VEGFR-2, achieving pIC<sub>50</sub> values of 7.15 and 7.27, respectively, comparable to the standard inhibitor, sorafenib (pIC<sub>50</sub> = 7.25).</p>
<p>The docking results revealed that the ethylene linker between the two heteroaromatic rings shared the same groove as the urea linker in sorafenib, contributing to a greater affinity for the VEGFR-2 enzyme. The formation of hydrogen bonds with Cys919, Asp1046, and Glu889, along with hydrophobic interactions, likely enhances the affinity of the heterocyclic pyrazolopyrimidine scaffold for the VEGFR-2 enzyme. In addition to the strong correlation between binding energies and pIC<sub>50</sub> values, as described in <a href="#eqn1" class="usa-link">eqn (1)</a>, <a href="#fig12" class="usa-link">Fig. 12</a> indicates that ligand binding is primarily driven by interactions with the protein. This relationship is expressed as follows:</p>
<table class="disp-formula p" id="eqn1"><tr>
<td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML">Binding energy = −1.8467 × pIC<sub>50</sub> + 4.2391.</span></td>
<td class="label">1</td>
</tr></table>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><em>N</em> = 7, <em>r</em><sup>2</sup> = 0.91, <em>p</em> &lt; 0.05</span></p>
<figure class="fig xbox font-sm" id="fig12"><h4 class="obj_head">Fig. 12. Correlation between the calculated binding energy score Δ<em>G</em> and pIC<sub>50</sub> for compounds 6b, 7b, 8b, 12a–c and 13b.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377214_d5ra04997k-f12.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/387c/12377214/1e4476e91dbf/d5ra04997k-f12.jpg" loading="lazy" height="480" width="779" alt="Fig. 12"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig12/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><section id="sec19"><h4 class="pmc_sec_title">Structure–activity relationship (SAR)</h4>
<p>Although a full SAR study is premature at this stage, initial observations reveal that the substituent pattern defined in <a href="#sch1" class="usa-link">Scheme 1</a> plays a pivotal role in modulating inhibitory activity. Compounds 6b, 7b, 8b, 10b, and 12b, all of which carry a 5-chloro substituent, demonstrate significantly enhanced potency. This enhancement may be explained by chlorine's capacity to participate in specific non-covalent interactions, including hydrogen and halogen bonding, which contribute to tighter receptor binding. Structural modeling indicates that the chlorine atom in these compounds forms halogen bonds with the carbonyl oxygen of Val101 (3.01–3.26 Å), supported by bond angles (C–Cl⋯O) in the range of 170–174.9° and favorable Cl⋯O–C angles (111–121°). These features enable stable and directed interactions within the CDK6 active site. In compound 12b, for example, the 5-chloro group facilitates binding to both His100 and Val101 (<a href="#fig13" class="usa-link">Fig. 13</a>), further enhancing receptor affinity. Moreover, compounds 12a–12d, bearing a fused pyrazolopyrimidine and thiazole framework, exhibit an extended binding mode within the CDK6 pocket, which may account for their potent inhibitory effects.</p>
<figure class="fig xbox font-sm" id="fig13"><h5 class="obj_head">Fig. 13. Representative example of a halogen bond in compound 12b with His100 and vicinal Val101 in the CDK6 active site.</h5>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377214_d5ra04997k-f13.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/387c/12377214/d56803aedac7/d5ra04997k-f13.jpg" loading="lazy" height="449" width="762" alt="Fig. 13"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig13/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Additionally, the predicted binding modes of compounds 8b and 12b within the CDK6 template structures were well-aligned, as shown in SI Fig. S87. These compounds demonstrated strong affinity for the ATP-binding site of CDK6, particularly interacting with Val101 and Asp163, with alignment scores of −126.70 and −116.67 kcal mol<sup>−1</sup>, respectively.</p></section></section><section id="sec20"><h3 class="pmc_sec_title">Evaluation of xanthine oxidase inhibitory activity</h3>
<p>The results of the <em>in vitro</em> XO inhibitory activity assay, as presented in <a href="#tab6" class="usa-link">Table 6</a> and Fig. S86, demonstrated that screened compounds 6b, 7b, 8b and 12a–c exhibited significant xanthine oxidase inhibitory activity with pIC<sub>50</sub> values ranging from 4.18 ± 0.82 to 4.94 ± 0.16. Among this series, the most active compounds 12a and 12c against XO have pIC<sub>50</sub> 4.94 and 4.83 distinct from allopurinol pIC<sub>50</sub> 4.57, respectively.</p>
<section id="sec21"><h4 class="pmc_sec_title">Kinetics of xanthine oxidase (XO) inhibition</h4>
<p>Enzyme inhibition can generally be categorized as competitive, uncompetitive, noncompetitive, or mixed.<sup><a href="#cit46" class="usa-link" aria-describedby="cit46">46,47</a></sup> The Lineweaver–Burk plot (Fig. S88) for compound 12a displays intersecting lines to the left of the <em>y</em>-axis and above the <em>X</em>-axis, which is characteristic of mixed-type inhibition. This indicates that the inhibitor can bind both to the free enzyme and to the enzyme–substrate complex although with varying affinities.<sup><a href="#cit48" class="usa-link" aria-describedby="cit48">48,49</a></sup> Simultaneous increase in <em>K</em><sub>m</sub> and decrease in Vmax shown by 12a and the reference allopurinol support the suggested mixed XOI mechanism. These conclusions are supported by strong correlation coefficients of 0.996 for 12a and 0.958 for allopurinol (<a href="#tab7" class="usa-link">Table 7</a>).</p>
<section class="tw xbox font-sm" id="tab7"><h5 class="obj_head">Table 7. Kinetic parameters of allopurinol and 12a against xanthine oxidase in the enzyme kinetic study.</h5>
<div class="tbl-box p" tabindex="0"><table class="content" frame="hsides" rules="groups">
<colgroup span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
<col span="1">
</colgroup>
<thead>
<tr>
<th rowspan="2" colspan="1">Conc. (μM)</th>
<th colspan="2" rowspan="1">12a</th>
<th colspan="2" rowspan="1">Allopurinol</th>
</tr>
<tr>
<th rowspan="1" colspan="1">
<em>K</em>
<sub>m</sub>
</th>
<th rowspan="1" colspan="1">
<em>V</em>
<sub>max</sub>
</th>
<th rowspan="1" colspan="1">
<em>K</em>
<sub>m</sub>
</th>
<th rowspan="1" colspan="1">
<em>V</em>
<sub>max</sub>
</th>
</tr>
</thead>
<tbody>
<tr>
<td rowspan="1" colspan="1">XO</td>
<td align="center" rowspan="1" colspan="1">40</td>
<td align="center" rowspan="1" colspan="1">2.25</td>
<td align="center" rowspan="1" colspan="1">40</td>
<td align="center" rowspan="1" colspan="1">2.21</td>
</tr>
<tr>
<td rowspan="1" colspan="1">0.1</td>
<td align="center" rowspan="1" colspan="1">75</td>
<td align="center" rowspan="1" colspan="1">1.66</td>
<td align="center" rowspan="1" colspan="1">50</td>
<td align="center" rowspan="1" colspan="1">1.51</td>
</tr>
<tr>
<td rowspan="1" colspan="1">1</td>
<td align="center" rowspan="1" colspan="1">112</td>
<td align="center" rowspan="1" colspan="1">1.38</td>
<td align="center" rowspan="1" colspan="1">64</td>
<td align="center" rowspan="1" colspan="1">1.33</td>
</tr>
<tr>
<td rowspan="1" colspan="1">10</td>
<td align="center" rowspan="1" colspan="1">123</td>
<td align="center" rowspan="1" colspan="1">0.98</td>
<td align="center" rowspan="1" colspan="1">51</td>
<td align="center" rowspan="1" colspan="1">0.55</td>
</tr>
</tbody>
</table></div>
<div class="p text-right font-secondary"><a href="table/tab7/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></section></section></section><section id="sec22"><h3 class="pmc_sec_title">2D-QSAR study</h3>
<p>A 2D-Quantitative Structure–Activity Relationship (2D-QSAR) model was developed to predict the root for the perspective design of new ligands against HepG2 hepatocellular carcinoma. The synthesized compounds were divided into a training set of 16 compounds for model development and a test set of 3 derivatives reserved for external validation. Molecular descriptors were calculated using the MOE (Molecular Operating Environment) software and categorized into seven clusters: physical properties, subdivided surface areas, atom and bond counts, connectivity-based descriptors, partial charge descriptors, pharmacophore feature descriptors, and distance matrix descriptors.<sup><a href="#cit50" class="usa-link" aria-describedby="cit50">50–52</a></sup></p>
<p>To identify the most significant structural predictors of activity, Stepwise Linear Regression Analysis (SLRA) was employed.<sup><a href="#cit53" class="usa-link" aria-describedby="cit53">53</a></sup> The QSAR model was developed using the Partial Least Squares (PLS) method with a dataset of 16 compounds. In accordance with QSAR modeling guidelines, the number of descriptors was limited to a maximum of three, ensuring a descriptor-to-compound ratio of approximately 1 : 5 to maintain model robustness and prevent overfitting.</p>
<p>The best QSAR model was selected based on its performance, which was evaluated using the highest correlation coefficient (<em>R</em><sup>2</sup> = 0.90) and the lowest root mean square error (RMSE = 0.203), as shown in Fig. S89 and Table S2. The final model was a linear equation derived from using the Partial Least Squares (PLS) method and was based on three key molecular descriptors, as demonstrated in <a href="#eqn2" class="usa-link">eqn (2)</a>:</p>
<table class="disp-formula p" id="eqn2"><tr>
<td class="formula"><span xmlns:mml="http://www.w3.org/1998/Math/MathML">pIC<sub>50</sub> = 6.42560 + 0.10829 × <em>E</em><sub>vdW</sub> − 0.00705 × TPSA + 0.29934 × log <em>P</em>(o/w).</span></td>
<td class="label">2</td>
</tr></table>
<p><span xmlns:mml="http://www.w3.org/1998/Math/MathML"><em>n</em> = 16, RMSE = 0.20301, <em>r</em><sup>2</sup> = 0.90112</span></p>
<p>The best-fit QSAR model revealed that the antiproliferative activity of the synthesized derivatives was influenced by three key molecular descriptors. van der Waals energy (<em>E</em><sub>vdW</sub>) exhibited a positive correlation with anticancer activity, as it plays a vital role in the formation of protein–ligand complexes. Similarly, lipophilicity (log <em>P</em>(o/w)) is crucial for determining a compound's ability to penetrate hydrophobic barriers, such as cell membranes, to reach its site of action. However, the polar surface area (TPSA) showed an inverse correlation with the antiproliferative activity of the synthesized derivatives. TPSA reflects the ability of a ligand to form polar interactions, particularly hydrogen bonds, with specific biological targets. The Z-score method was used to identify outliers in the QSAR model, as presented in Table S2. In our study, none of the compounds had a <em>Z</em>-score above 2.5, indicating that all the compounds were consistent with the model predictions. This finding underscores the robustness and accuracy of the QSAR model. To validate the model, an independent test set comprising five compounds (7b, 9c, and 12c) was used for external validation. Additionally, two approved drugs, sunitinib and cisplatin, were included to strengthen the correlation and assess the model's predictive accuracy. The model's predictive performance was assessed by plotting the experimental pIC<sub>50</sub> values against the predicted pIC<sub>50</sub> values for these compounds. The model demonstrated a high predictive accuracy, with a correlation coefficient of <em>R</em><sup>2</sup> = 0.93, as shown in Fig. S90. This strong correlation between the predicted and experimental values confirmed the model's robustness and reliability.</p>
<p>For the future development of novel compounds in this series, it is recommended to prioritize structural modifications that enhance <em>E</em><sub>vdW</sub> and log <em>P</em>(o/w). This strategy is expected to improve the binding affinity of the compounds to biological targets, increase membrane permeability, and strengthen interactions.</p></section><section id="sec23"><h3 class="pmc_sec_title">
<em>In vitro</em> chemical stability of compounds 6b, 8b, and 12a–12c</h3>
<p>To assess the chemical stability of the most active compounds 6b, 8b, and 12a–12c under simulated physiological conditions, each compound was incubated in phosphate buffer (pH 7.4) at 37 °C, and UV-Vis spectra were recorded over the 190–800 nm range at regular intervals up to 24 hours. The spectra remained consistent throughout the study, with no significant changes in absorbance intensity or the appearance of new peaks, indicating no degradation. These results support the chemical integrity of the compounds in buffered aqueous solutions, confirming their suitability for further biological studies. The full data are given in the SI, Fig. S91.</p></section><section id="sec24"><h3 class="pmc_sec_title">Determination of global reactivity descriptors</h3>
<p>Density Functional Theory (DFT) calculations were conducted at the B3LYP level using the 6-31G basis set to investigate the structural characteristics of compounds 6b, 7b, 8b and 12a–12c, which exhibit the highest activity on the target enzyme CDK6. These calculations provided key insights into the molecular electronic structure through global reactivity descriptors. The descriptors include the energies of the highest occupied molecular orbital (HOMO) and lowest unoccupied molecular orbital (LUMO), as well as derived reactivity parameters, such as ionization potential (<em>I</em>), electron affinity (<em>A</em>), electronegativity (<em>χ</em>), chemical hardness (<em>η</em>), softness (<em>s</em>), chemical potential (<em>μ</em>), and electrophilicity (<em>ω</em>) (Table S3). These descriptors, particularly HOMO and LUMO energies, are critical for predicting the charge transfer, chemical reactivity, bioactivity, and stability of the compounds.<sup><a href="#cit54" class="usa-link" aria-describedby="cit54">54–57</a></sup> A higher HOMO energy suggests a stronger electron-donating ability and, therefore, greater chemical reactivity. In contrast, a lower LUMO energy indicates a higher electron-accepting capacity, also pointing to enhanced reactivity. In ligands 6b, 8b and 12a–12c, HOMO electron density is primarily localized over the ethylene link and extended in the direction of the 5-Cl-oxindole and the thiazolylidene ring, indicating a propensity for electron donation during interactions with the CDK6 enzyme. The LUMO electron density, however, is distributed over the oxindole ring, as well as the sulfur and nitrogen atoms in the thiazolylidene ring, facilitating electron delocalization and influencing interactions with target molecules. <a href="#fig14" class="usa-link">Fig. 14</a> illustrates the orbital isosurface of the frontier molecular orbitals (HOMO and LUMO) of compounds 6b, 8b and 12b, with positive phases depicted in green and negative phases in red. The energy gap (Δ<em>E</em> = LUMO − HOMO) serves as a key indicator of both chemical and biological activity; smaller energy gaps indicate higher reactivity and lower excitation energy requirements. In the studied series, the energy gap follows the trend: thiazolylidene derivatives 12a–12c &lt; thiosemicarbazone 8b &lt; unsubstituted compounds 6b and 7b. Among the 5-substituted oxindole derivatives, Δ<em>E</em>: 5-MeO &lt; 5-Cl &lt; 5-F &lt; 5-H, signifying the increased reactivity of 5-substituted oxindole derivatives relative to their unsubstituted counterparts. Electrophilic and nucleophilic indices determine a molecule's ability to exchange electron density during reactions. The order of electron affinity, electronegativity, and chemical potential follows unsubstituted 6b, 7b &gt; thiosemicarbazone 8b &gt; thiazolylidene derivatives 12a–12c, indicating that thiazolylidene derivatives have the highest electron transfer capacity.</p>
<figure class="fig xbox font-sm" id="fig14"><h4 class="obj_head">Fig. 14. Representative frontier molecular orbitals of compounds 6b, 8b and 12b.</h4>
<p class="img-box line-height-none margin-x-neg-2 tablet:margin-x-0 text-center"><a class="tileshop" target="_blank" href="https://www.ncbi.nlm.nih.gov/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=12377214_d5ra04997k-f14.jpg"><img class="graphic zoom-in" src="https://cdn.ncbi.nlm.nih.gov/pmc/blobs/387c/12377214/60e573d9c018/d5ra04997k-f14.jpg" loading="lazy" height="245" width="773" alt="Fig. 14"></a></p>
<div class="p text-right font-secondary"><a href="figure/fig14/" class="usa-link" target="_blank" rel="noopener noreferrer">Open in a new tab</a></div></figure><p>Molecular polarizability is linked to softness and hardness: “soft” molecules with lower hardness and higher softness are more polarized. Among the synthesized compounds, thiazolylidene derivatives 12a–12c, with small energy gaps (Δ<em>E</em> = 2.558–2.800 eV), are classified as soft molecules, making them highly polarizable and biologically active. The electrophilicity index is essential for assessing a drug candidate's binding affinity to biological targets. Higher electrophilicity strengthens covalent interactions with nucleophilic residues in enzymes or receptor proteins, enhancing binding strength and specificity. Additionally, electron-withdrawing groups boost electrophilicity by stabilizing the LUMO, reducing the HOMO–LUMO energy gap, and increasing reactivity toward nucleophilic attack.</p>
<p>In conclusion, compounds 12b and 12c, which demonstrated the highest CDK6 inhibitory activity, exhibited a small energy gap and lower electrophilicity. These calculated descriptors provide valuable insights into drug design strategies, particularly in the optimization studies of these compounds.</p></section><section id="sec25"><h3 class="pmc_sec_title">Prediction of physicochemical properties of the most active compounds</h3>
<p>The drug-likeness and molecular properties of the most active compounds, along with the reference drugs palbociclib and sorafenib, were evaluated using Molinspiration cheminformatics software<sup><a href="#cit58" class="usa-link" aria-describedby="cit58">58</a></sup> (Table S4). These parameters are essential for predicting the pharmacokinetics of the compounds, including their transport properties, reactivity, and metabolic stability within biological systems. All the targeted compounds, except for 10b and 12c, fully complied with Lipinski's Rule of Five. The number of rotatable bonds in the synthesized compounds was fewer than 10, indicating compliance with Veber's criteria<sup><a href="#cit59" class="usa-link" aria-describedby="cit59">59</a></sup> for molecular flexibility, suggesting that these compounds are likely to exhibit good oral bioavailability. Additionally, the TPSA values of the synthesized compounds ranged from 92.35 to 139.41 Å<sup>2</sup>, further supporting their potential for oral bioavailability. Substituents at the C-5 position of the oxindole ring (–F, –Cl, and –OCH<sub>3</sub>) increased the log <em>P</em> values and enhanced hydrophobicity and, consequently, the inhibitory potency against CDK6 through hydrophobic and van der Waals interactions. The most active compounds (6b, 8b, and 12a–12d) adhered to both Lipinski's and Veber's rules, highlighting their strong potential to inhibit the activities of CDK6, VEGFR-2, and xanthine oxidase.</p></section></section><section id="sec26"><h2 class="pmc_sec_title">Conclusion</h2>
<p>This study presents a new class of oxindole derivatives derived from allopurinol and adenine, with promising multi-target anticancer potential. Compounds 6b, 7b, 8b, and 12a–d displayed potent cytotoxic effects, particularly toward HepG2 cells. Notably, allopurinol-derived hybrids (12a–d) showed enhanced activity over adenine-based analogs, attributable to chlorine-induced electronic effects. Mechanistic assays demonstrated pro-apoptotic activity (12a and 12c), S-phase arrest (7b), and caspase activation. Enzymatic profiling and molecular modeling confirmed potent inhibition of CDK6, VEGFR-2, and xanthine oxidase, with key halogen/hydrogen bonding interactions at the CDK6 hinge region. The superior bioactivity of 12a–c correlated with its soft electronic character. These findings position compounds 8b, 12a, and 12b as compelling candidates for further development in anticancer therapeutics.</p></section><section id="sec27"><h2 class="pmc_sec_title">Experimental</h2>
<section id="sec28"><h3 class="pmc_sec_title">Molecular docking study</h3>
<p>The molecular docking simulation studies were performed on a Dell Precision T3600 workstation with Intel Xeon® CPU-1650.0 @ 3.20 GHz with a Windows 10 operating system using Molecular Operating Environment software (MOE 2020.01). The X-ray crystal structure of CDK6 co-crystalized with palbociclib (PDB ID: 5L2I) and Sorafenib with VEGFR-2 (PDB code 3WZE) was obtained from the protein data bank. Since the crystal structure contains a ligand molecule, the program automatically identifies the binding site, and we dock the tested ligands on it. Docking of the conformation database of the target ligands was done using the MOE-DOCK software wizard. To compare the conformers, London dG was used as a scoring function; lower values indicate more favorable poses. All the key interactions accomplished by the co-crystalized ligand with the key amino acids in the binding site are reproducible using the following docking setup.</p></section><section id="sec29"><h3 class="pmc_sec_title">Molecular dynamics simulation</h3>
<p>The initial CDK6 complexes with palbociclib and compound 12b were obtained <em>via</em> molecular docking. Molecular dynamics simulations (MDS) were conducted using NAMD 3.0 with the CHARMM27 force field.<sup><a href="#cit60" class="usa-link" aria-describedby="cit60">60</a></sup> Hydrogen atoms were added <em>via</em> the psfgen plugin in VMD 1.9. The systems were solvated in TIP3P water with 0.15 M Na<sup>+</sup> and Cl<sup>−</sup> ions, energy minimized, heated to 300 K, and equilibrated for 1 ns. A 50 ns MDS was performed,<sup><a href="#cit61" class="usa-link" aria-describedby="cit61">61</a></sup> recording trajectory data every 0.1 ns to analyze RMSD, RMSF, hydrogen bonds, radius of gyration, and SASA. Ligand parameters were determined using CHARMM-GUI. Binding free energy was estimated <em>via</em> MM-PBSA using the fastDRH webserver with AmberTools 21, calculated as follows: Δ<em>G</em><sub>binding</sub> = <em>G</em><sub>complex</sub> − (<em>G</em><sub>receptor</sub> + <em>G</em><sub>inhibitor</sub>). Each term involves calculating energy components, such as van der Waals, electrostatic, molecular mechanics, internal, and polar solvation energy.</p></section><section id="sec30"><h3 class="pmc_sec_title">Chemistry</h3>
<section id="sec31"><h4 class="pmc_sec_title">Materials and methods</h4>
<p>All the chemicals used for the synthesis of the target compounds were commercially available reagent grade and used without further purification, except the solvents, which were purified before use. <em>N</em>,<em>N</em>-Dimethylformamide (DMF) was purchased from Biochem and dried over molecular sieves (4 Å) prior to use. Isatin 98%, 5-chloroisatin 98%, and adenine 98% were obtained from Alfa Aesar Co., Germany. 5-Methoxyisatin 97% and 5-fluoroisatin 98% were obtained from Acros Organic Co., United Kingdom. Chemical reactions were monitored on thin layer chromatography (TLC) using precoated silica gel plates, 60 G F254, obtained from Merck, Darmstadt (Germany).</p>
<p>Silica gel (60–120 mesh) was used for column chromatography, and the developing systems used were: system (A): dichloromethane/methanol (9.5 : 0.5 v/v), system (B): chloroform/ethanol (9.7 : 0.3 v/v) and system (C): chloroform/ethyl acetate (2 : 1 v/v).</p>
<p>Melting points were measured using a Stuart® SMP10 melting point apparatus (Stuart Scientific, England) and were uncorrected. Infrared spectra (IR) were recorded using a Nicolet® iS10 FT-IR Spectrometer as KBr pellets at the Faculty of Science-Assiut University, Assiut, Egypt. The data are reported as the frequency of absorption (cm<sup>−1</sup>). The NMR spectra were recorded using an AVANCE-III High Performance FT-NMR spectrometer, Brucker-Biospin International AG, Switzerland (400 MHz for <sup>1</sup>H and 100 MHz for <sup>13</sup>C) at the Faculty of Science-Zagazig University, Zagazig/Egypt. Chemical shifts are reported in parts per million (ppm, <em>δ</em> scale) using TMS as referenced relative to residual solvent DMSO-d6 (<em>δ</em><sup>1</sup>H = 2.50, <sup>13</sup>C = 39.5 ppm). Coupling constants (<em>J</em>) for <sup>1</sup>H were reported in Hz, and spin multiplicities are represented by the following signals: singlet (s), doublet (d), triplet (t), doublet of doublet (dd) and multiplet (m). All raw fid files were processed, and the resulting spectra were analyzed using the program MestReNOVA 14.2 from Mestrelab Research S.L. Mass spectra (MS) were obtained using the inlet part to mass analyser in ThermoFisher scientific DPC-Direct Probe Controller model ISQ7000 single quadruple mass spectrometer at the Faculty of Science, Assiut University, Assiut, Egypt. Values are reported as the ratio of mass to charge in Daltons. The purity of the most active compounds 6b, 8b, 12a, 12b and 12c is assessed by HPLC analysis at the Institute for Drug Development and Innovation Research, Assiut University, Assiut, Egypt.</p></section></section><section id="sec32"><h3 class="pmc_sec_title">General method for synthesis of 2-bromoethyl-substituted pyrazolopyrimidine and purine derivatives 3, 4</h3>
<p>To a stirred suspension of allopurinol 1 or adenine 2 (1 mmol) and potassium carbonate (0.32 g, 2.3 mmol) in DMF (20 mL), 1,2-dibromoethane (0.75 g, 4 mmol) was added in a single portion. The mixture was stirred at room temperature for 24 h, after which the starting material was no longer detectable by TLC MeOH : DCM 1 : 9 and a new spot was detected. Then, the mixture was poured into crushed ice and left for 1 h. The precipitate was filtered, washed with cold water and dried at 80 °C.</p>
<section id="sec33"><h4 class="pmc_sec_title">1-(2-Bromoethyl)-1,5-dihydro-4<em>H</em>-pyrazolo[3,4-<em>d</em>]pyrimidin-4-one 3</h4>
<p>Yield 2.0 g (82%), a pale-orange solid, mp 261–264 °C, recrystallized from ethanol. FT IR: 3435, 3083, 2969, 1694, 1582, 782, 695. <sup>1</sup>H NMR <em>δ</em>: 8.44–8.12 (m, 3H), 4.70–4.66 (t, <em>J</em> = 4 Hz, 2H), 4.37–4.28 (t, <em>J</em> = 4 Hz, 1H), 3.93–3.91 (t, <em>J</em> = 4 Hz, 1H). Mass spectrum = 242.02 (M. wt: 243.06).</p></section><section id="sec34"><h4 class="pmc_sec_title">9-(2-Bromoethyl)-9<em>H</em>-purin-6-amine 4</h4>
<p>Yield 1.75 g (72%), an orange solid, mp: 295–298 °C (297–298 lit.<sup><a href="#cit62" class="usa-link" aria-describedby="cit62">62</a></sup>), washed with water. IR: 3362, 3323, 3150, 1652, 1599, 1385, 1325, 1309, 1246, 1074, 796, 708, 647. <sup>1</sup>H NMR <em>δ</em>: 8.19 (s, 1H), 8.15 (s, 1H), 7.27 (s, 2H), 4.57 (t, <em>J</em> = 5.97, 2H), 3.95 (t, <em>J</em> = 5.97, 2H).</p></section></section><section id="sec35"><h3 class="pmc_sec_title">General method for the synthesis of 1,5-disubstitutedindole-2,3-dione 6a–d and 7a–d</h3>
<p>To a stirred solution of 5-(un) substituted-1<em>H</em>-indole-2,3-dione 5a–d (4 mmol) and anhydrous potassium carbonate (0.83 g, 6 mmol) in dry DMF (5 mL), compounds 3 and 4 (4 mmol) were added. The reaction mixture was stirred at 60 °C for 6–8 h. The reaction mixture was then cooled to room temperature, poured into crushed ice and left for 1 h. The products were separated by filtration, dried and purified by column chromatography on a silica gel column using developing systems (A) as the mobile phase.</p>
<section id="sec36"><h4 class="pmc_sec_title">1-[2-(1,5-Dihydro-4-oxopyrazolo[3,4-<em>d</em>]pyrimidin-1-yl) ethyl] indole-2, 3-dione 6a</h4>
<p>Yield 1.35 (88%), orange powder, reaction time 6 h, mp: 223–225 °C. IR: 3435, 3084, 2952, 1745, 1694, 1608, 1582, 1473. <sup>1</sup>H NMR <em>δ</em>: 8.33–8.00 (m, 3H), 7.55 (dd, <em>J</em> = 13.1, 6.9 Hz, 2H), 7.09 (dd, <em>J</em> = 14.9, 7.0 Hz, 1H), 6.87 (d, <em>J</em> = 7.9 Hz, 1H), 4.55 (s, 2H), 4.19–4.08 (m, 2H). <sup>13</sup>C NMR <em>δ</em>: 183.0, 158.3, 156.5, 151.8, 150.4, 150.1, 138.1, 135.0, 128.1, 124.5, 123.2, 110.3, 105.0, 44.3, 43.3. Elemental analyses found: C%, 58.43; H%, 3.77; N%, 22.91. Calc. for C<sub>15</sub>H<sub>11</sub>N<sub>5</sub>O<sub>3</sub>: C%, 58.25; H%, 3.58; O%, 22.64.</p></section><section id="sec37"><h4 class="pmc_sec_title">5-Chloro-1-[2-(1,5-dihydro-4-oxopyrazolo[3,4-<em>d</em>]pyrimidin-1-yl)ethyl]indole-2, 3-dione 6b</h4>
<p>Yield 1.2 (70%), pale orange powder, reaction time 6.5 h, mp: 214–215 °C. IR: 3467, 3084, 2953, 1745, 1694, 1610, 1584, 1473. <sup>1</sup>H NMR <em>δ</em>: 8.11–7.97 (m, 3H), 7.60 (d, <em>J</em> = 7.4 Hz, 1H), 7.56 (d, <em>J</em> = 2.3 Hz, 1H), 6.92 (d, <em>J</em> = 8.0 Hz, 1H), 4.60–4.53 (m, 2H), 4.34–4.23 (m, 2H). <sup>13</sup>C NMR <em>δ</em>: 181.9, 157.8, 156.5, 151.6, 150.7, 148.6, 137.1, 136.0, 127.4, 124.0, 118.4, 111.9, 105.0, 44.3, 42.8. Elemental analyses found: C%, 52.67; H%, 3.05; N%, 20.63. Calc. for C<sub>15</sub>H<sub>10</sub>ClN<sub>5</sub>O<sub>3</sub>: C%, 52.41; H%, 2.93; N%, 20.37.</p></section><section id="sec38"><h4 class="pmc_sec_title">1-[2-(1,5-Dihydro-4-oxopyrazolo[3,4-<em>d</em>]pyrimidin-1-yl)ethyl]5-methoxyindole-2,3-dione 6c</h4>
<p>Yield 1.2 g (71%), reddish orange powder, reaction time 7 h, mp: 226–228 °C. IR: 3458, 3084, 2945, 1736, 1697, 1621, 1598, 1490. <sup>1</sup>H NMR <em>δ</em>: 8.10–7.96 (m, 3H), 7.11 (d, 1H), 7.09 (d, <em>J</em> = 18.6 Hz, 2H), 6.79 (s, 1H), 4.64–4.27 (m, 2H), 4.27–4.03 (s, 1H), 4.03–3.90 (m, 1H), 3.73 (s, 3H). <sup>13</sup>C NMR <em>δ</em>: 183.2, 158.1, 156.5, 155.6, 151.6, 150.8, 144.2, 135.0, 124.0, 117.7, 111.1, 109.2, 105.0, 55.8, 44.3, 43.2. Elemental analyses found: C%, 56.82; H%, 4.03; N%, 20.89. Calc. for C<sub>16</sub>H<sub>13</sub>N<sub>5</sub>O<sub>4</sub>: C, % 56.64; H%, 3.86; N%, 20.64.</p></section><section id="sec39"><h4 class="pmc_sec_title">5-Fluoro-1-[(2-(1,5-dihydro-4-oxopyrazolo[3,4-<em>d</em>]pyrimidin-1-yl)ethyl)]indole-2,3-dione 6d</h4>
<p>Yield 1 g (63%), pale orange powder, reaction time 6.5 h, mp 219–220 °C, IR: 3450, 3056, 2952, 1741, 1697, 1624, 1583,1486. <sup>1</sup>H NMR <em>δ</em>: 8.35–8.01 (m, 2H),7.46 (s, 2H), 6.93 (s, 1H), 4.54 (s, 2H), 4.22–4.09 (m, 2H). <sup>13</sup>C NMR <em>δ</em>: 182.8, 160.1, 158.5, 157.6, 156.9, 152, 147.1, 136.9, 135.4, 128.5, 124.5, 112.0, 105.4, 44.7, 43.9. Elemental analyses found: C%, 55.31; H%, 3.20; N%, 21.67. Calc. for C<sub>15</sub>H<sub>10</sub>FN<sub>5</sub>O<sub>3</sub>: C%, 55.05; H%, 3.08; N%, 21.40.</p></section><section id="sec40"><h4 class="pmc_sec_title">1-[2-(6-Amino-9<em>H</em>-purin-9-yl)ethyl]-2,3-dihydro-1<em>H</em>-indole-2,3-dione 7a</h4>
<p>Yield 1.4 g (91%), orange powder, reaction time 8 h, mp: 244–246 °C (decomp.), (lit. 242 °C).<sup><a href="#cit62" class="usa-link" aria-describedby="cit62">62</a></sup> IR: 3441, 3317, 3157, 3103, 2940, 1740, 1651, 1610, 1596, 1473. <sup>1</sup>H NMR <em>δ</em>: 8.16 (s, 1H), 7.95 (s, 1H), 7.65–7.38 (m, 2H), 7.16 (s, 2H), 7.04 (t, <em>J</em> = 7.5 Hz, 1H), 6.89 (d, <em>J</em> = 7.9 Hz, 1H), 4.44 (t, <em>J</em> = 5.3 Hz, 2H), 4.11 (t, <em>J</em> = 5.3 Hz, 2H). <sup>13</sup>C NMR <em>δ</em>: 183.3, 158.7, 156.2, 152.7, 150.7, 150.1, 141.3, 138.2, 124.7, 123.5,119, 117.8, 110.3, 41.1, 40.3. Mass spectrum: <em>m</em>/<em>z</em> 308.26 [M]<sup>+</sup>; M. wt: 308.29. Elemental analyses found: C%, 58.67; H%, 4.08; N%, 27.49. Calc. for C<sub>15</sub>H<sub>12</sub>N<sub>6</sub>O<sub>2</sub>: C%, 58.44; H%, 3.92; N%, 27.26.</p></section><section id="sec41"><h4 class="pmc_sec_title">1-[2-(6-Amino-9<em>H</em>-purin-9-yl)ethyl]-5-chloro-2,3-dihydro-1<em>H</em>-indole-2,3-dione 7b</h4>
<p>Yield 1.5 g (87.5%), pale orange powder, reaction time 7.5 h, mp: 259–261 °C. IR: 3441, 3320, 3174, 3031, 1740, 1654, 1605, 1577, 1477. <sup>1</sup>H NMR <em>δ</em>: 8.15 (s, 1H), 7.93 (s, 1H), 7.62–7.48 (m, 2H), 7.16 (s, 2H), 6.93 (d, <em>J</em> = 8.3 Hz, 1H), 4.41 (s, 2H), 4.11 (d, <em>J</em> = 4.7 Hz, 1H). <sup>13</sup>C NMR <em>δ</em>: 181.9, 159.2, 158, 155.8, 152.4, 148.8, 140.9, 136.7, 127.2, 123.7, 118.6, 113.8, 111.6, 42.7, 40.6. Elemental analyses found: C%, 52.73; H%, 3.42; N%, 24.68. Calc. for C<sub>15</sub>H<sub>11</sub>ClN<sub>6</sub>O<sub>2</sub>: C, 52.56; H, 3.23; N, 24.52.</p></section><section id="sec42"><h4 class="pmc_sec_title">1-[2-(6-Amino-9<em>H</em>-purin-9-yl)ethyl]-2,3-dihydro-5-methoxy-1<em>H</em>-indole-2,3-dione 7c</h4>
<p>Yield 1.2 g (71%), reddish orange powder, reaction time 6 h, mp: 260–264 °C. IR: 3365, 3308, 3143, 2963, 1740, 1667, 1600, 1576, 1492. <sup>1</sup>H NMR <em>δ</em>: 8.14 (s, 1H), 7.96 (s, 1H), 7.15 (s, 2H), 7.05 (d, <em>J</em> = 8.1 Hz, 2H), 6.84 (d, <em>J</em> = 7.3 Hz, 1H) 4.41 (s, 2H), 4.07 (s, 2H), 3.72 (s, 3H). <sup>13</sup>C NMR <em>δ</em>: 179.1, 161.3, 156.2, 152.7, 150.1, 141.3, 136.7, 131.5, 120.7, 119, 117, 110.4, 106.6, 56.0, 41.3, 40.5. Elemental analyses found: C%, 56.88; H%, 4.35; N%, 24.99. Calc. for C<sub>16</sub>H<sub>14</sub>N<sub>6</sub>O<sub>3</sub>: C%, 56.80; H%, 4.17; N%, 24.84.</p></section><section id="sec43"><h4 class="pmc_sec_title">1-[2-(6-Amino-9<em>H</em>-purin-9-yl)ethyl]-2,3-dihydro-5-fluoro-1<em>H</em>-indole-2,3-dione 7d</h4>
<p>Yield 1.45 g (92%), pale orange powder, reaction time 6.5 h, mp: 230–233 °C. IR: 3368, 3325, 3159, 3103, 1739, 1652, 1603, 1577, 1485. <sup>1</sup>H NMR <em>δ</em>: 8.15 (s, 1H), 7.93 (s, 1H), 7.36 (d, <em>J</em> = 7.0 Hz, 2H), 7.16 (s, 2H), 6.93 (d, <em>J</em> = 5.5 Hz, 1H), 4.44 (t, <em>J</em> = 5.3, 2H), 4.10 (t, <em>J</em> = 5.3, 2H). <sup>13</sup>C NMR <em>δ</em>: 182.9, 159.9, 158.7, 157.5, 156.2, 152.8, 150.1, 146.9, 141.4, 124.3, 124.1, 118.9, 111.6, 43.0, 41.1. Elemental analyses found: C%, 55.41; H%, 3.57; N%, 26.04. Calc. for C<sub>15</sub>H<sub>11</sub>FN<sub>6</sub>O<sub>2</sub>: C%, 55.22; H%, 3.40; N%, 25.76.</p></section></section><section id="sec44"><h3 class="pmc_sec_title">General method of synthesis of oxindole-3-thio-semicarbazone derivatives 8a–d–9a–d</h3>
<p>The appropriate isatin derivative 6a–d, 7a–d (1 mmol) and thiosemicarbazide (0.09 g, 1 mmol) were dissolved in warm ethanol (20 mL). The reaction mixture was acidified by the addition of few drops of glacial acetic acid and refluxed for 2–4 h. The mixture was cooled to room temperature, and the separated precipitate was filtrated, washed with ethanol, and dried. The product was purified by recrystallization from ethanol.</p>
<section id="sec45"><h4 class="pmc_sec_title">(2<em>Z</em>)-2-{2-Oxo-1-[2-(4-oxo-1,5-dihydrpyrazolo[3,4-<em>d</em>]pyrimidin-1-yl)ethyl]-1,2-dihydro-3<em>H</em>-indol-3-ylidene}hydrazine-1-carbothioamide 8a</h4>
<p>Yield 0.28 g (75%), yellow powder, reaction time 2 h, mp 245–248 °C. IR: 3420, 3260, 3163, 3083, 2951, 1694, 1612, 1358, 782. <sup>1</sup>H NMR <em>δ</em>: 12.22 (s, 1H), 9.09 (s, 1H), 8.72 (s, 1H), 8.26–7.96 (m, 2H), 7.67–7.62 (t, <em>J</em> = 8 Hz, 1H), 7.30–7.26 (m, <em>J</em> = 8 Hz, 1H), 7.08 (d, <em>J</em> = 6.8 Hz, 1H), 6.79 (dd, <em>J</em> = 8, 7.5 Hz, 1H), 4.65 (s, 2H), 4.30–4.07 (m, 2H). <sup>13</sup>C NMR <em>δ</em>: 179.4, 162.8, 161.2, 156.7, 152.1, 151.1, 142.7, 135.4, 131.4, 123.1, 121.1, 119.7, 109.8, 105.7, 44.7, 43. Elemental analyses, found: C%, 50.49; H%, 3.80; N%, 29.47; S%, 8.46. Calc. for C<sub>16</sub>H<sub>14</sub>N<sub>8</sub>O<sub>2</sub>S: C%, 50.25; H%, 3.69; N%, 29.30; S%, 8.39.</p></section><section id="sec46"><h4 class="pmc_sec_title">(2<em>Z</em>)-2-{5-Chloro-2-oxo-1-[2-(4-oxo-1,5-dihydropyrazolo[3,4-<em>d</em>]pyrimidin-1-yl)ethyl]-1,2-dihydro-3<em>H</em>-indol-3-ylidene}hydrazine-1-carbothioamide 8b</h4>
<p>Yield 0.3 g (70%), yellow powder, reaction time 2.5 h, mp: 250–252 °C. IR: 3431, 3265, 3172, 3056, 2928, 1694, 1597, 1348, 782. <sup>1</sup>H NMR <em>δ</em>: 12.07 (s, 1H), 9.18 (s, 1H), 8.85 (s, 1H), 8.27–7.98 (m, 3H), 7.77 (d, <em>J</em> = 16.5 Hz, 1H), 7.36 (s, 1H), 6.84 (d, <em>J</em> = 14.8 Hz, 1H), 4.58 (s, 2H), 4.29–4.14 (m, 2H). <sup>13</sup>C NMR <em>δ</em>: 178.7, 160.5, 156.4, 151.5, 150.6, 140.9, 135.0, 130.1, 129.3, 127.1, 121.0, 120.3, 110.7, 105.0, 45.3, 44.5. Elemental analyses found: C%, 46.32; H%, 3.40; N%, 27.15; S%, 7.78. Calc. for C<sub>16</sub>H<sub>13</sub>ClN<sub>8</sub>O<sub>2</sub>S: C%, 46.10; H%, 3.14; N%, 26.88; S%, 7.69.</p></section><section id="sec47"><h4 class="pmc_sec_title">(2<em>Z</em>)-2-{5-Methoxy-2-oxo-1-[2-(4-oxo-1,5-dihydropyrazolo[3,4-<em>d</em>]pyrimidin-1-yl)ethyl]-1,2-dihydro-3<em>H</em>-indol-3-ylidene}hydrazine-1-carbothioamide 8c</h4>
<p>Yield 0.25 g 68.5%, pale orange powder, reaction time 4 h, mp: 252–253 °C. IR: 3429, 3265–3168, 3056, 2943, 1693, 1596, 1357, 782. <sup>1</sup>H NMR <em>δ</em>: 12.16 (s, 1H), 9.09 (s, 1H), 8.75 (s, 1H), 8.21–7.99 (m, 3H), 7.37 (d, <em>J</em> = 14.4 Hz, 1H), 7.04–6.73 (m, 2H), 4.53 (s, 2H), 4.29–4.14 (m, 2H), 3.73 (s, 3H). <sup>13</sup>C NMR <em>δ</em>: 179.2, 160.8, 156.4, 155.6, 151.6, 150.6, 136.1, 134.9, 130.9, 120.1, 116.8, 110.1, 106.3, 105.0, 55.6, 44, 43.7. Elemental analyses found: C%, 49.78; H%, 4.05; N%, 27.30; S%, 7.91. Calc. for C<sub>17</sub>H<sub>16</sub>N<sub>8</sub>O<sub>3</sub>S: C%, 49.51; H%, 3.91; N%, 27.17; S%, 7.77.</p></section><section id="sec48"><h4 class="pmc_sec_title">(2<em>Z</em>)-2-{5-Fluoro-2-oxo-1-[2-(4-oxo-1,5-dihydropyrazolo[3,4-<em>d</em>]pyrimidin-1-yl)ethyl]-1,2-dihydro-3<em>H</em>-indol-3-ylidene}hydrazine-1-carbothioamide 8d</h4>
<p>Yield 0.3 g (82%), yellow powder, reaction time 2.5 h, mp 230–233 °C. IR: 3430, 3268, 3168, 3056, 2952, 1694, 1583, 1352, 782. <sup>1</sup>H NMR <em>δ</em>: 12.09 (s, 1H), 9.18 (s, 1H), 8.83 (s, 1H), 8.28–8 (m, 3H), 7.53 (s, 1H), 7.16 (s, 1H), 6.84 (d, <em>J</em> = 17.7 Hz, 1H), 4.55 (s, 2H), 4.35–4.13 (m, 2H). <sup>13</sup>C NMR <em>δ</em>: 180.1, 161.2, 157.7, 156.9, 152.1, 151.1, 139.0, 135.6, 130.4, 121.3, 117.6, 110.9, 108.2, 105.0, 44.9, 43.2. Elemental analyses found: C%, 48.23; H%, 3.45; N%, 28.17; S%, 8.14. Calc. for C<sub>16</sub>H<sub>13</sub>FN<sub>8</sub>O<sub>2</sub>S: C%, 48.00; H%, 3.27; N%, 27.99; S%, 8.01.</p></section><section id="sec49"><h4 class="pmc_sec_title">(2<em>Z</em>)-2-{2-Oxo-1-[2-(6-aminopurine-9-yl)ethyl]-1,2-dihydro-3<em>H</em>-indol-3-ylidene}hydrazine-1-carbothioamide 9a</h4>
<p>Yield 0.35 g (94%), bright yellow powder, reaction time 2.5 h, mp: 276–277 °C, IR: 3373, 3312, 3265, 3170, 2931, 1677, 1600, 1304, 1172. <sup>1</sup>H NMR <em>δ</em>: 12.26 (s, 1H), 9.10 (s, 1H), 8.74 (s, 1H), 8.10 (s, 1H), 8.00 (s, 1H), 7.63 (d, <em>J</em> = 7.3 Hz, 1H), 7.24 (t, <em>J</em> = 7.4 Hz, 1H), 7.16 (s, 2H), 7.06 (t, <em>J</em> = 7.4 Hz, 1H), 6.82 (d, <em>J</em> = 7.8 Hz, 1H), 4.46 (t, <em>J</em> = 5.5 Hz, 2H), 4.18 (t, <em>J</em> = 5.5 Hz, 2H). <sup>13</sup>C NMR <em>δ</em>: 179.1, 161.4, 156.2, 152.8, 150.1, 143.0, 141.2, 131.4, 131.3, 123.2, 121.1, 119.8, 119.1, 109.5, 41.3, 40.3. Elemental analyses found: C%, 50.52; H%, 4.17; N%, 33.19; S%, 8.53. Calc. for C<sub>16</sub>H<sub>15</sub>N<sub>9</sub>OS: C%, 50.38; H%, 3.96; N%, 33.05; S%, 8.41.</p></section><section id="sec50"><h4 class="pmc_sec_title">(2<em>Z</em>)-2-{5-Chloro-2-oxo-1-[2-(6-aminopurine-9-yl)ethyl]-1,2-dihydro-3<em>H</em>-indol-3-ylidene}hydrazine-1-carbothioamide 9b</h4>
<p>Yield 0.4 g (94%), yellow powder, reaction time 3.5 h, mp 281–283 °C. IR: 3450, 3248, 3289, 3248, 3110, 2951, 1697, 1610, 1303, 1170. <sup>1</sup>H NMR <em>δ</em>: 12.07 (s, 1H), 9.15 (s, 1H), 8.84 (s, 1H), 8.09 (s, 1H), 7.98 (s, 1H), 7.72 (d, <em>J</em> = 2.2 Hz, 1H), 7.29 (dd, <em>J</em> = 8.5, 2.2 Hz, 1H), 7.14 (s, 2H), 6.84 (d, <em>J</em> = 8.5 Hz, 1H), 4.44 (t, <em>J</em> = 5.5 Hz, 2H), 4.17 (t, <em>J</em> = 5.5 Hz). <sup>13</sup>C NMR <em>δ</em>: 178.7, 160.6, 155.8, 152.3, 149.6, 141.2, 140.8, 140.4, 129.9, 126.9, 121.1, 120.3, 118.5, 110.6, 42.6, 40.8. Elemental analyses found: C%, 46.34; H%, 3.48; N%, 30.17; S%, 7.84. Calc. for C<sub>16</sub>H<sub>14</sub>ClN<sub>9</sub>OS: C%, 46.21; H%, 3.39; N%, 30.31; S%, 7.71.</p></section><section id="sec51"><h4 class="pmc_sec_title">(2<em>Z</em>)-2-{5-Methoxy-2-oxo-1-[2-(6-aminopurine-9-yl)ethyl]-1,2-dihydro-3<em>H</em>-indol-3-ylidene} hydrazine-1-carbothioamide 9c</h4>
<p>Yield 0.3 g (82%), yellowish orange powder, and reaction time 4 h, mp 244–245 °C. IR: 3446, 3294, 3244, 3142, 2927, 1694, 1603, 1304, 1164. <sup>1</sup>H NMR <em>δ</em>: 12.19 (s, 1H), 9.08 (s, 1H), 8.74 (s, 1H), 8.06 (s, 1H), 8.01 (s, 1H), 7.29 (s, 1H), 7.13 (s, 2H), 6.80–6.74 (m, 2H), 4.43 (s, 2H), 4.13 (s, 2H), 3.72 (s, 3H). <sup>13</sup>C NMR <em>δ</em>: 178.7, 160.9, 155.8, 155.6, 152.3, 149.7, 140.8, 136.3, 131.1, 120.16, 118.6, 116.6, 109.9, 106.2, 55.6, 42.6, 40.8. Elemental analyses found: C%, 49.89; H%, 4.30; N%, 30.82; S%, 7.90. Calc. for C<sub>17</sub>H<sub>17</sub>N<sub>9</sub>O<sub>2</sub>S: C%, 49.63; H%, 4.16; N%, 30.64; S%, 7.79.</p></section><section id="sec52"><h4 class="pmc_sec_title">(2<em>Z</em>)-2-{5-Fluoro-2-oxo-1-[2-(6-aminopurine-9-yl)ethyl]-1,2-dihydro-3<em>H</em>-indol-3-ylidene}hydrazine-1-carbothioamide 9d</h4>
<p>Yield 0.35 g (95%), yellow powder, reaction time 3 h, mp 279–281 °C. IR: 3420, 3362, 3321, 3150, 2951, 1652, 1599, 1309, 1174; <sup>1</sup>H NMR <em>δ</em>: 12.12 (s, 1H), 9.15 (s, 1H), 8.78 (s, 1H,), 8.08 (s, 1H), 7.97 (s, 1H), 7.45 (d, <em>J</em> = 5.3 Hz, 1H), 7.14 (s, 2H), 7.08 (d, <em>J</em> = 8.1 Hz, 1H), 6.82 (s, 1H), 4.44 (s, 2H), 4.16 (s, 2H). <sup>13</sup>C NMR <em>δ</em>: 178.8, 161.0, 157.3, 155.8, 152.4, 149.7, 140.9, 138.9, 130.4, 120.8, 118.5, 116.9, 110.3, 107.9, 40.9, 40.1. Elemental analyses found: C%, 48.27; H%, 3.49; N%, 31.75; S%, 8.11. Calc. for C<sub>16</sub>H<sub>14</sub>FN<sub>9</sub>OS: C%, 48.11; H%, 3.53; N%, 31.56; S%, 8.03.</p></section></section><section id="sec53"><h3 class="pmc_sec_title">Synthesis of 1,5-disubstituted-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(4-oxo-1,3-thiazolidin-2-ylidene)hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 10a–d and 11a–d</h3>
<p>A mixture of appropriate 2-substituted hydrazine-1-carbothioamides series 8a–d, 9a–d (1 mmol), ethyl bromoacetate for 8a–d or monochloroacetic acid for 9a–d (0.17 g, 1 mmol) and anhydrous sodium acetate (0.16 g, 2 mmol) in ethanol (50 ml) were refluxed for 24 h, the reaction mixture was cooled, poured into crushed ice. The formed precipitate was filtered, dried, and purified by silica gel column using the developing system (B) for compounds 10a–d and (C) for 11a–d.</p>
<section id="sec54"><h4 class="pmc_sec_title">1-[2-(4-Oxo-1,5-dihydro-4<em>H</em>-pyrazolo[3,4-<em>d</em>]pyrimidine-1-yl) ethyl]-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(4-oxo-1,3-thiazolidin-2-ylidene)hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 10a</h4>
<p>Yield 0.3 g (68%), orange powder, mp: 265–267 °C, IR: 3435, 3050, 2954, 1744, 1697, 1597, 1351, 783. <sup>1</sup>H NMR <em>δ</em>: 12.23 (s, 1H), 8.24–7.97 (m, 3H), 7.49 (dd, <em>J</em> = 13.6, 7.3 Hz, 1H), 7.30 (d, <em>J</em> = 7.1 Hz, 1H), 7.00 (d, <em>J</em> = 6.6 Hz, 2H), 6.87–6.61 (m, 1H), 4.55 (s, 2H), 4.19 (d, <em>J</em> = 7.8 Hz, 3H) 3.97 (s, 2H). <sup>13</sup>C NMR <em>δ</em>: 171.8, 166.4, 156.9, 156.3,151.5, 150.4, 145.9, 134.8, 132.3, 130, 122.2, 121.2, 119.5, 108.7, 104.9, 48, 44.6, 32.2. Mass spectrum: <em>m</em>/<em>z</em> [M]<sup>+</sup> = 422.42; found 422.21. Elemental analyses found: C%, 51.42; H%, 3.52; N%, 26.80; S%, 7.71. Calc. for C<sub>16</sub>H<sub>13</sub>FN<sub>8</sub>O<sub>2</sub>S: C%, 51.18; H%, 3.34; N%, 26.53; S%, 7.59.</p></section><section id="sec55"><h4 class="pmc_sec_title">5-Chloro-1-[2-(4-Oxo-1,5-dihydro-4<em>H</em> pyrazolo[3,4-<em>d</em>]pyrimidine-1-yl) ethyl]-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(4-oxo-1,3-thiazolidin-2-ylidene) hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 10b</h4>
<p>Yield 0.27 g (61.5%), reddish orange powder, mp: 288–290 °C dec., IR: 3435, 3084, 2952, 1694 (2C<span aria-roledescription="private character" aria-label="DOUBLE BOND, LENGTH AS M-DASH" aria-description="double bond, length as m-dash">

<svg xmlns="http://www.w3.org/2000/svg" version="1.0" width="13.200000pt" height="16.000000pt" viewBox="0 0 13.200000 16.000000" preserveAspectRatio="xMidYMid meet"><metadata>
Created by potrace 1.16, written by Peter Selinger 2001-2019
</metadata><g transform="translate(1.000000,15.000000) scale(0.017500,-0.017500)" fill="currentColor" stroke="none"><path d="M0 440 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z M0 280 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z"/></g></svg>

</span>O), 1582, 1374, 783. <sup>1</sup>H NMR <em>δ</em>: 12.06 (s, 1H), 8.26–7.97 (m, 3H), 7.74–7.34 (m, 2H), 6.80–6.70 (m, 1H), 4.53 (s, 2H), 4.21–4.07 (m, 2H), 3.94 (s, 2H). <sup>13</sup>C NMR <em>δ</em>: 171.9, 166.7, 163.0, 158.4, 157.8, 156.6, 151.8, 150.75, 149.6, 145.0, 135.1, 130.2, 126.4, 106.0, 104.9, 45.4, 44.6, 31.2. Elemental analyses found: C%, 47.59; H%, 3.05; N%, 24.71; S%, 7.14. Calc. for C<sub>18</sub>H<sub>13</sub>ClN<sub>8</sub>O<sub>3</sub>S: C%, 47.32; H%, 2.87; N%, 24.53; S%, 7.02.</p></section><section id="sec56"><h4 class="pmc_sec_title">5-Methoxy-1-[2-(4-oxo-1,5-dihydro-4<em>H</em>-pyrazolo[3,4-<em>d</em>]pyrimidine-1-yl)ethyl]-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(4-oxo-1,3-thiazolidin-2-ylidene) hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 10c</h4>
<p>Yield 0.25 g (57%), orange powder, mp 290–291 °C dec., IR: 3446, br., 3084, 2952, 1696, 1596, 1352, 783. <sup>1</sup>H NMR <em>δ</em>: 12.58 (s, 1H), 8.50–7.97 (m, 3H), 7.00–6.63 (m, 3H), 466–4.50 (m, 2H), 4.19 (s, 2H), 3.94 (s, 2H), 3.74 (s, 3H). <sup>13</sup>C NMR <em>δ</em>: 171.8, 166.7, 158.4, 157.8, 157.1, 156.4, 155.1, 151.5, 150.5, 135.0, 130.1, 118.0, 109.3, 106.0, 105.0, 55.6, 44.5, 43.7, 32.3. Elemental analyses found: C%, 50.67; H%, 3.62; N%, 24.85; S%, 7.18. Calc. for C<sub>19</sub>H<sub>16</sub>N<sub>8</sub>O<sub>4</sub>S: C%, 50.44; H%, 3.56; N%, 24.77; S%, 7.09.</p></section><section id="sec57"><h4 class="pmc_sec_title">5-Fluoro-1-[2-(4-oxo-1,5-dihydro-4<em>H</em>-pyrazolo[3,4-<em>d</em>]pyrimidine-1-yl)ethyl]-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(4-oxo-1,3-thiazolidin-2-ylidene)hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 10d</h4>
<p>Yield 0.3 g (68%), orange powder, mp: 256–257 °C dec., IR: 3467, 3084, 2952, 1698. <sup>1</sup>H NMR <em>δ</em>: 12.75 (s, 1H), 8.50–7.98 (m, 3H), 7.74–6.62 (m, 3H), 4.51 (s, 2H), 4.21–3.96 (m, 4H). <sup>13</sup>C NMR <em>δ</em>: 171.8, 166.7, 157.0, 156.4, 151.5, 150.5, 147.4, 145.5, 134.9, 128.0, 116.6, 114.9, 110.2, 105.0, 44.5, 43.6, 32.6. Elemental analyses found: C%, 49.31; H%, 3.17; N%, 25.30; S%, 7.42. Calc. for C<sub>18</sub>H<sub>13</sub>FN<sub>8</sub>O<sub>3</sub>S: C%, 49.09; H%, 2.98; N%, 25.44; S%, 7.28.</p></section><section id="sec58"><h4 class="pmc_sec_title">1-[2-(6-Amino purine-9-yl)ethyl]-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(4-oxo-1,3-thiazolidin-2-ylidene) hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 11a</h4>
<p>Yield 0.25 g (65%), dark yellow powder, mp: 302 °C dec., IR: 3366, 3188, 3067, 2983, 1698, 1613, 1541, 1337. <sup>1</sup>H NMR <em>δ</em>: 12.53 (s, 1H), 8.07 (s, 1H), 8.01 (s, 1H), 7.47 (d, <em>J</em> = 7.4 Hz, 1H), 7.28 (t, <em>J</em> = 8 Hz, 1H), 7.20 (s, 2H), 7.01 (t, <em>J</em> = 7.5 Hz, 1H), 6.81 (d, <em>J</em> = 7.6 Hz, 1H), 4.42 (t, <em>J</em> = 4 Hz, 2H), 4.12 (t, <em>J</em> = 4 Hz, 2H), 3.95 (s, 2H). <sup>13</sup>C NMR <em>δ</em>: 172.1, 163.5, 155.5, 155.2, 152.3, 151.9, 140.9, 134.9, 132.3, 131.8, 129.4, 128.2, 122.3, 118.5, 108.5, 42.2, 41.8, 33.3. Elemental analyses found: C%, 51.47; H%, 3.65; N%, 30.14; S%, 7.69. Calc. for C<sub>18</sub>H<sub>15</sub>N<sub>9</sub>O<sub>2</sub>S: C%, 51.30; H%, 29.91; S%, 7.61.</p></section><section id="sec59"><h4 class="pmc_sec_title">1-[2-(6-Amino purine-9-yl) ethyl]-5-chloro-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(4-oxo-1,3-thiazolidin-2-ylidene)hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 11b</h4>
<p>Yield 0.28 g (64%), yellowish green powder, mp: 285 °C dec (decomp.), IR: 3429, 3232, 3079, 2927, 1705, 1622, 1540, 1334. <sup>1</sup>H NMR <em>δ</em>: 12.64 (br. s, 1H), 8.08 (s, 1H), 7.97 (s, 1H), 7.40 (d, <em>J</em> = 1.3 Hz, 1H), 7.33 (dd, <em>J</em> = 4.6, 2.4 Hz, 1H), 7.15 (s, 2H), 6.85 (t, <em>J</em> = 8 Hz, 1H), 4.42 (d, <em>J</em> = 4.7 Hz, 2H), 4.12 (t, <em>J</em> = 5.2 Hz, 2H), 4.03 (s, 2H). <sup>13</sup>C NMR <em>δ</em>: 174.5, 163.7, 156.1, 153, 152.6, 150.1, 146.1, 143.2, 141.3, 131.8, 128, 126.7, 119.0, 118.1, 110.4, 41.9, 41.2, 34.2. Elemental analyses found: C%, 47.60; H%, 3.24; N%, 27.88; S%, 7.12. Calc. for C<sub>18</sub>H<sub>14</sub>ClN<sub>9</sub>O<sub>2</sub>S: C%, 47.42; H%, 3.10; N%, 27.65; S%, 7.03.</p></section><section id="sec60"><h4 class="pmc_sec_title">1-[2-(6-Amino purine-9-yl) ethyl]-5-methoxy-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(4-oxo-1,3-thiazolidin-2-ylidene) hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 11c</h4>
<p>Yield 0.25 g (57%), dark orange powder, mp: 290 °C, IR: 3362, 3145, 3070, 2928, 1701, 1600, 1541, 1337. <sup>1</sup>H NMR <em>δ</em>. 12.58 (s, 1H), 8.50–7.97 (m, 3H), 7.00–6.63 (m, 3H), 4.43 (s, 2H), 4.26 (s, 2H), 4.13 (s, 2H), 3.74 (s, 3H). <sup>13</sup>C NMR <em>δ</em>: 71.8, 166.7, 156.4, 155.1, 151.5, 150.5, 146.3, 144.1, 135.0, 130.1, 125.3, 120.6, 119.2, 118.0, 109.3, 55.6, 44.5, 43.7, 33.1. Elemental analyses found: C%, 50.74; H%, 3.97; N%, 27.78; S%, 7.21. Calc. for C<sub>19</sub>H<sub>17</sub>N<sub>9</sub>O<sub>3</sub>S: C%, 50.55; H%, 3.80; N%, 27.92; S%, 7.10.</p></section><section id="sec61"><h4 class="pmc_sec_title">1-[2-(6-Amino purine-9-yl) ethyl]-5-fluoro-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(4-oxo-1,3-thiazolidin-2-ylidene) hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 11d</h4>
<p>Yield 0.3 g (68%), dark yellow powder, mp: 298–299 °C, IR: 3378, 3336, 3072, 2926, 1705, 1631, 1543, 1336. <sup>1</sup>H NMR <em>δ</em>: 12.66 (s, 1H), 8.06 (s, 1H), 7.97 (s, 1H), 7.22 (d, <em>J</em> = 6.2 Hz, 1H,), 7.13 (s, 3H), 6.84 (d, <em>J</em> = 3.3 Hz, 1H), 4.40 (t, <em>J</em> = 4 Hz, 2H), 4.11 (t, <em>J</em> = 4 Hz, 2H), 3.96 (s, 2H). <sup>13</sup>C NMR <em>δ</em>: 174.6, 165.8, 157.1, 156.6, 155.5, 152.0, 149.7, 140.8, 140.3, 139.4, 121.3, 118.5, 118.1, 117.7, 109.8, 41.7, 40.8, 33. 2. Elemental analyses found: C%, 49.08; H%, 3.40; N%, 28.92; S%, 7.45. Calc. for C<sub>18</sub>H<sub>14</sub>FN<sub>9</sub>O<sub>2</sub>S: C%, 3.21; N%, 28.69; S%, 7.30.</p></section></section><section id="sec62"><h3 class="pmc_sec_title">General method for the synthesis of 1,5-disubstituted-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(1,3-thiazol-2(3<em>H</em>)-ylidene) hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 12a–d and 13a–d</h3>
<p>A mixture of appropriate isatin-3-thiosemicarbazones 8a–d, 9a–d (1 mmol), chloroacetone (0.09 g, 1 mmol) and anhydrous sodium acetate (0.16 g, 2 mmol) in ethanol (50 mL) was refluxed for 24 h. The reaction mixture was cooled and poured into crushed ice. The precipitate was filtered and dried to give yellow to orange products. The crude powder was purified by column chromatography on a silica gel column using developing systems (A) as the mobile phase.</p>
<section id="sec63"><h4 class="pmc_sec_title">1-[2-(4-Oxo-1,5-dihydro-4<em>H</em>-pyrazolo[3,4-<em>d</em>]pyrimidine-1-yl)ethyl]-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(1,3-thiazol-2(3<em>H</em>)-ylidene)hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 12a</h4>
<p>Yield 0.24 g (59%), reddish brown powder, mp: 223–225 °C, IR: 3435, 3084, 2961, 1694, 1583, 1351, 783. <sup>1</sup>H NMR <em>δ</em>: (<em>E</em>/<em>Z</em>%, 50 : 50): 12.92 (s, 1H, NH<sub>(<em>E</em>)</sub>), 12.44 (s, 1H, NH<sub>(<em>Z</em>)</sub>), 8.70–8.45 (m, 1H<sub>(<em>E</em>)</sub>), 8.33–8.15 (m, 1H<sub>(<em>Z</em>)</sub>), 8.02–7.96 (m, 3H), 7.75 (m, 1H<sub>(<em>E</em>)</sub>), 7.55–7.36 (m, 1H<sub>(<em>Z</em>)</sub>), 7.26 (dd, <em>J</em> = 12.8, 7.5 Hz, 1H), 7.17 (dd, <em>J</em> = 13.4, 6.3 Hz, 1H<sub>(<em>Z</em>)</sub>), 7.06 (dd, <em>J</em> = 7.1, 3.6 Hz, 1H<sub>(<em>E</em>)</sub>), 7.04–6.91 (m, 1H<sub>(<em>Z</em>)</sub>), 6.84–6.75 (m, 1H<sub>(<em>E</em>)</sub>), 6.68–6.57 (m, 1H<sub>(<em>Z</em>)</sub>), 6.49 (s, 1H<sub>(<em>E</em>)</sub>), 6.33 (s, 1H<sub>(<em>Z</em>)</sub>), 4.53 (d, <em>J</em> = 4.7 Hz, 2H), 4.36–3.97 (m, 2H). <sup>13</sup>C NMR <em>δ</em>: 178.5<sub>(<em>E</em>)</sub>, 178.1<sub>(<em>Z</em>)</sub>, 164.7<sub>(<em>E</em>)</sub>, 164.6<sub>(<em>Z</em>)</sub>, 156.8, 152.1<sub>(<em>E</em>)</sub>, 151.9<sub>(<em>Z</em>)</sub>, 150.9<sub>(<em>E</em>)</sub>, 150.5<sub>(<em>Z</em>)</sub>, 141.9, 136.1, 135.3<sub>(<em>E</em>)</sub>, 135.1<sub>(<em>Z</em>)</sub>,130.3<sub>(<em>E</em>)</sub>, 130.1<sub>(<em>Z</em>)</sub>, 129.8, 126.5<sub>(<em>E</em>)</sub>, 126.3<sub>(<em>Z</em>)</sub>, 122.2<sub>(<em>E</em>)</sub>, 121.9<sub>(<em>Z</em>)</sub>, 118.5<sub>(<em>E</em>)</sub>, 117.8<sub>(<em>Z</em>)</sub>, 108.3, 105.9<sub>(<em>E</em>)</sub>, 105.3<sub>(<em>Z</em>)</sub>, 102.1<sub>(<em>E</em>)</sub>, 101.5<sub>(<em>Z</em>)</sub>, 48.3, 44.9. Elemental analyses found: C%, 53.40; H%, 3.59; N%, 27.41; S%, 8.01. Calc. for C<sub>18</sub>H<sub>14</sub>N<sub>8</sub>O<sub>2</sub>S: C%, 53.19; H%, 3.47; N%, 27.57; S%, 7.89.</p></section><section id="sec64"><h4 class="pmc_sec_title">5-Chloro-1-[2-(4-oxo-1,5-dihydro-4<em>H</em>-pyrazolo[3,4-<em>d</em>] pyrimidine-1-yl)ethyl]-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(1,3-thiazol-2(3<em>H</em>)-ylidene)hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 12b</h4>
<p>Yield 0.29 g (68%), yellowish brown powder, mp 234–237 °C dec., IR: 3444, 3084, 2925, 1694, 1583, 1351, 783. <sup>1</sup>H NMR <em>δ E</em>/<em>Z</em> %, (40 : 60): 12.69 (s, 1H<sub>(<em>E</em>)</sub>), 12.06 (s, 1H<sub>(<em>Z</em>)</sub>), 8.58–8.38 (m, 1H<sub>(<em>E</em>)</sub>), 8.39–7.73 (m, 3H), 7.84–7.50 (m, 1H<sub>(<em>Z</em>)</sub>), 7.26 (d, <em>J</em> = 6.3 Hz, 1H), 7.18 (d, <em>J</em> = 6.4 Hz, 1H<sub>(<em>Z</em>)</sub>), 7.05 (s, 1H), 6.59 (d, <em>J</em> = 8.8 Hz, 1H), 6.40 (s, 1H) 4.50 (s, 2H), 4.39–3.96 (m, 2H). <sup>13</sup>C NMR <em>δ</em>: 178.7<sub>(<em>E</em>)</sub>, 178.4<sub>(<em>Z</em>)</sub>, 164.1<sub>(<em>E</em>)</sub>, 163.9<sub>(<em>Z</em>)</sub>, 156.3, 151.6, 150.5<sub>(<em>E</em>)</sub>,150.3<sub>(<em>Z</em>)</sub>, 140.1, 135.7, 134.8, 128.4<sub>(<em>E</em>)</sub>, 128.2<sub>(<em>Z</em>)</sub>, 125.8, 125.5<sub>(<em>E</em>)</sub>, 125.4<sub>(<em>Z</em>)</sub>, 125.1<sub>(<em>E</em>)</sub>, 124.9<sub>(<em>Z</em>)</sub>, 118.7<sub>(<em>E</em>)</sub>, 118.4<sub>(<em>Z</em>)</sub>, 108.8, 105.0, 101.9<sub>(<em>E</em>)</sub>, 101.4<sub>(<em>Z</em>)</sub>, 45.0, 43.2. Elemental analyses found: C%, 48.79; H%, 3.13; N%, 25.68; S%, 7.41. Calc. for C<sub>18</sub>H<sub>13</sub>ClN<sub>8</sub>O<sub>2</sub>S: C%, 49.04; H%, 2.97; N%, 25.42; S%, 7.27.</p></section><section id="sec65"><h4 class="pmc_sec_title">5-Methoxy-1-[2-(4-oxo-1,5-dihydro-4<em>H</em>-pyrazolo[3,4-<em>d</em>]pyrimidine-1-yl) ethyl]-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(1,3-thiazol-2(3<em>H</em>)-ylidene)hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 12c</h4>
<p>Yield 0.26 g (61%), reddish brown powder, mp: 226–228 °C dec., IR: 3436, br (NH), 3084 (CH aromatic), 2945, 1694, 1542, 1356, 782. <sup>1</sup>H NMR <em>δ</em>: (<em>E</em>/<em>Z</em>%, 30 : 70): 12.97 (s, 1H<sub>(<em>E</em>)</sub>), 12.41 (s, 1H, NH<sub>(<em>Z</em>)</sub>), 8.50–8.30 (m, 1H<sub>(<em>E</em>)</sub>), 8.26–7.91 (m, 3H), 7.83–7.55 (m, 1H<sub>(<em>Z</em>)</sub>), 7.43 (d, <em>J</em> = 12.6 Hz, 1H <sub>(<em>E</em>)</sub>), 7.02 (d, <em>J</em> = 13.1 Hz, 1H<sub>(<em>Z</em>)</sub>), 6.83 (s, 1H<sub>(<em>E</em>+<em>Z</em>)</sub>), 6.74 (s, 1H,), 6.56 (s, 1H<sub>(<em>E</em>)</sub>), 6.34 (s, 1H<sub>(<em>Z</em>)</sub>), 4.52 (s, 2H),4.05–4.18 (m, 2H), 3.75 (s, 3H). <sup>13</sup>C NMR <em>δ</em>: 178.4<sub>(<em>E</em>)</sub>, 177.6<sub>(<em>Z</em>)</sub>, 164.9<sub>(<em>E</em>)</sub>, 164.7<sub>(<em>Z</em>)</sub>, 157.5<sub>(<em>E</em>)</sub>, 156.9<sub>(<em>Z</em>)</sub>, 155.7, 155.4, 155.2, 152.4, 152.1, 151.1<sub>(<em>E</em>)</sub>, 151.0<sub>(<em>Z</em>)</sub>, 136.0, 135.4, 118.8, 114.4, 113.5, 105.6, 101.9<sub>(<em>E</em>)</sub>, 101.6<sub>(<em>Z</em>)</sub>, 56.1, 45.3, 43.7. Elemental analyses found: C%, 52.43; H%, 3.81; N%, 25.89; S%, 7.52. Calc. for C<sub>19</sub>H<sub>16</sub>N<sub>8</sub>O<sub>3</sub>S: C%, 52.29; H%, 3.70; N%, 25.67; S%, 7.35.</p></section><section id="sec66"><h4 class="pmc_sec_title">5-Fluoro-1-[2-(4-oxo-1,5-dihydro-4<em>H</em>-pyrazolo[3,4-<em>d</em>]pyrimidine-1-yl) ethyl]-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(1,3-thiazol-2(3<em>H</em>)-ylidene)hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 12d</h4>
<p>Yield 0.22 g (52%), yellowish brown powder, mp 242–246 °C dec., IR: 3435, 3107, 2927, 1697, 1541, 1355, 782. <sup>1</sup>H NMR <em>δ</em> (<em>E</em>/<em>Z</em> %, 35 : 65): 12.59 (s, 1H), 8.37–8.16 (m, 1H<sub>(<em>E</em>)</sub>), 8.14–7.89 (m, 4H<sub>(<em>Z</em>)</sub>), 7.77 (dd, <em>J</em> = 17.5, 8.4 Hz, 1H), 7.14–6.90 (m, 1H), 6.62 (dd, <em>J</em> = 8.3, 4.0 Hz, 1H), 6.55 (s, 1H<sub>(<em>E</em>)</sub>), 6.40 (d, <em>J</em> = 3.1 Hz, 1H<sub>(<em>Z</em>)</sub>), 4.51 (d, <em>J</em> = 4.6 Hz, 2H), 4.39–3.89 (m, 2H). <sup>13</sup>C NMR <em>δ</em>: 178.7 (C<span aria-roledescription="private character" aria-label="DOUBLE BOND, LENGTH AS M-DASH" aria-description="double bond, length as m-dash">

<svg xmlns="http://www.w3.org/2000/svg" version="1.0" width="13.200000pt" height="16.000000pt" viewBox="0 0 13.200000 16.000000" preserveAspectRatio="xMidYMid meet"><metadata>
Created by potrace 1.16, written by Peter Selinger 2001-2019
</metadata><g transform="translate(1.000000,15.000000) scale(0.017500,-0.017500)" fill="currentColor" stroke="none"><path d="M0 440 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z M0 280 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z"/></g></svg>

</span>O<sub>(<em>E</em>)</sub>), 177.6 (C<span aria-roledescription="private character" aria-label="DOUBLE BOND, LENGTH AS M-DASH" aria-description="double bond, length as m-dash">

<svg xmlns="http://www.w3.org/2000/svg" version="1.0" width="13.200000pt" height="16.000000pt" viewBox="0 0 13.200000 16.000000" preserveAspectRatio="xMidYMid meet"><metadata>
Created by potrace 1.16, written by Peter Selinger 2001-2019
</metadata><g transform="translate(1.000000,15.000000) scale(0.017500,-0.017500)" fill="currentColor" stroke="none"><path d="M0 440 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z M0 280 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z"/></g></svg>

</span>O<sub>(<em>Z</em>)</sub>), 164.7 (thiazol C<span aria-roledescription="private character" aria-label="DOUBLE BOND, LENGTH AS M-DASH" aria-description="double bond, length as m-dash">

<svg xmlns="http://www.w3.org/2000/svg" version="1.0" width="13.200000pt" height="16.000000pt" viewBox="0 0 13.200000 16.000000" preserveAspectRatio="xMidYMid meet"><metadata>
Created by potrace 1.16, written by Peter Selinger 2001-2019
</metadata><g transform="translate(1.000000,15.000000) scale(0.017500,-0.017500)" fill="currentColor" stroke="none"><path d="M0 440 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z M0 280 l0 -40 320 0 320 0 0 40 0 40 -320 0 -320 0 0 -40z"/></g></svg>

</span>N<sub>(<em>E</em>)</sub>), 164.5<sub>(<em>Z</em>)</sub>, 157.2<sub>(<em>E</em>)</sub>, 156.9<sub>(<em>Z</em>)</sub>, 155.9, 154.8, 152, 151<sub>(<em>E</em>)</sub>, 150.8<sub>(<em>Z</em>)</sub>, 140.4, 138.7, 138.2, 136.9, 136.2, 135.2, 108.6, 105.4, 102.2<sub>(<em>E</em>)</sub>, 101.9<sub>(<em>Z</em>)</sub>, 45.4, 45.0. Elemental analyses found: C%, 51.20; H%, 3.23; N%, 26.61; S%, 7.68. Calc. for C<sub>18</sub>H<sub>13</sub>FN<sub>8</sub>O<sub>2</sub>S: C%, 50.94; H%, 3.09; N%, 26.40; S%, 7.56.</p></section><section id="sec67"><h4 class="pmc_sec_title">1-[2-(6-Amionpurine-9-yl)ethyl]-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(1,3-thiazol-2(3<em>H</em>)-ylidene)hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 13a</h4>
<p>Yield 0.23 g (62%), brown powder, mp 249–250 °C dec., IR: 3338, 3181, 3104, 2924, 1682, 1603, 1544, 1329. <sup>1</sup>H NMR <em>δ</em>: (<em>E</em>/<em>Z</em> %, 38 : 62): 12.93 (s, 1H<sub>(<em>E</em>)</sub>), 12.44 (s, 1H<sub>(<em>Z</em>)</sub>), 8.26 (d, <em>J</em> = 7.4 Hz, 2H<sub>(<em>E</em>)</sub>), 8.09 (s, 1H), 8.02 (s, 1H), 7.48 (d, <em>J</em> = 7.3 Hz, 1H<sub>(<em>Z</em>)</sub>), 7.24 (t, <em>J</em> = 7.2 Hz, 1H<sub>(<em>E</em>)</sub>), 7.15–7.07 (m, 3H), 7.05 (t, <em>J</em> = 7.2 Hz, 1H<sub>(<em>Z</em>)</sub>), 6.96 (t, <em>J</em> = 7.5 Hz, 1H<sub>(<em>E</em>)</sub>), 6.90 (d, <em>J</em> = 7.9 Hz, 1H), 6.77 (d, <em>J</em> = 8.4 Hz, 1H), 6.51 (s, 1H<sub>(<em>E</em>)</sub>), 6.36 (s, 1H<sub>(<em>Z</em>)</sub>), 4.44 (dd, <em>J</em> = 13.3, 8.0 Hz, 2H), 4.19 (dd, <em>J</em> = 14.8, 9.5 Hz, 2H). <sup>13</sup>C NMR <em>δ</em>: 177.9, 164.3, 155.7, 152.3, 149.8, 141.8, 141.4<sub>(<em>E</em>)</sub>, 141.0<sub>(<em>Z</em>)</sub>, 137.2, 135.8, 130.0<sub>(<em>E</em>)</sub>, 129.5<sub>(<em>Z</em>)</sub>, 126.1, 122.9<sub>(<em>E</em>)</sub>, 121.9<sub>(<em>Z</em>)</sub>, 119.2<sub>(<em>E</em>)</sub>, 118.7<sub>(<em>Z</em>)</sub>, 117.6, 109.2<sub>(<em>E</em>)</sub> 107.7<sub>(<em>Z</em>)</sub>, 101,4, 41.3, 41.1. Elemental analyses found: C%, 53.57; H%, 3.65; N%, 31.30; S%, 7.85. Calc. for C<sub>18</sub>H<sub>15</sub>N<sub>9</sub>OS: C%, 53.32; H%, 3.73; N%, 31.09; S%, 7.91.</p></section><section id="sec68"><h4 class="pmc_sec_title">1-[2-(6-Amionpurine-9-yl) ethyl]-5-chloro-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(1,3-thiazol-2(3<em>H</em>)-ylidene)hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 13b</h4>
<p>Yield 0.25 g (59%), brownish orange powder, mp 214–215 °C dec., IR: 3365, 3152, 3076, 2926, 1672, 1601, 1545, 1325. <sup>1</sup>H NMR <em>δ</em>: 12.70 (s, 1H), 8.28 (s, 1H), 8.05 (s, 1H), 8.01 (s, 1H), 7.79 (s, 1H), 7.15 (d, <em>J</em> = 9.3 Hz, 3H), 6.77 (d, <em>J</em> = 8.3 Hz, 1H), 6.44 (s, 1H), 4.41 (s, 2H), 4.15 (s, 2H). <sup>13</sup>C NMR <em>δ</em>: 178.4, 163.9, 155.7, 152.6, 149.7, 140.8, 140.5, 140.2, 135.5, 128.2, 125.6, 125.0, 118.7, 118.6, 108.6, 102.0, 42.6, 41. Elemental analyses, found: C%, 49.42; H%, 3.47; N%, 28.90; S%, 7.41. Calc. for C<sub>18</sub>H<sub>14</sub>ClN<sub>9</sub>OS: C%, 49.15; H%, 3.21; N%, 28.66; S%, 7.29.</p></section><section id="sec69"><h4 class="pmc_sec_title">1-[2-(6-Amionpurine-9-yl) ethyl]-5-methoxy-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(1,3-thiazol-2(3<em>H</em>)-ylidene)hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 13c</h4>
<p>Yield 0.27 g (64%), dark orange powder, mp: 265–267 °C dec., IR: 3338, 3176, 3108, 2925, 1675, 1598, 1544, 1325. <sup>1</sup>H NMR <em>δ</em>: 12.98 (s, 1H<sub>(<em>E</em>)</sub>), 12.41 (s, 1H<sub>(<em>Z</em>)</sub>), 8.08 (s, 1H), 8.05 (s, 1H<sub>(<em>E</em>)</sub>), 8.01 (s, 1H), 7.94 (s, 1H<sub>(<em>Z</em>)</sub>), 7.15 (s, 2H), 7.00 (s, 1H), 6.83 (s, 1H), 6.73–6.70 (m, 1H), 6.68 (s, 1H), 4.42 (d, <em>J</em> = 16 Hz, 2H), 4.14 (d, <em>J</em> = 20 Hz, 2H), 3.74 (s, 3H). <sup>13</sup>C NMR <em>δ</em>: 177.6, 164.2, 155.7, 154.8, 152.2, 149.6, 140.8, 137.4, 135.5, 135.1, 118.6, 118.2, 113.8, 113.0, 107.9, 101.2, 55.6, 41.1, 40.9. Elemental analyses found: C%, 52.62; H%, 4.18; N%, 29.07; S%, 7.43. Calc. for C<sub>19</sub>H<sub>17</sub>N<sub>9</sub>O<sub>2</sub>S: C%, 52.40; H%, 3.93; N%, 28.95; S%, 7.36.</p></section><section id="sec70"><h4 class="pmc_sec_title">1-[2-(6-Amionpurine-9-yl) ethyl]-5-fluoro-(3<em>Z</em>)-3-[(2<em>E</em>/<em>Z</em>)-(1,3-thiazol-2(3<em>H</em>)-ylidene)hydrazinylidene]-1,3-dihydro-2<em>H</em>-indol-2-one 13d</h4>
<p>Yield 0.26 g (61%), brownish orange powder, mp: 232 °C dec., IR: 3362, 3152, 3067, 2926, 1680, 1599, 1543, 1324. <sup>1</sup>H NMR <em>δ</em>: 12.59 (s, 1H), 8.05 (s, 2H), 8.01 (s, 1H), 7.14 (s, 2H), 6.98 (s, 1H), 6.77 (s, 1H), 6.42 (s, 1H,), 4.41 (s, 2H), 4.15 (s, 2H). <sup>13</sup>C NMR <em>δ</em>: 178.2, 164.2, 155.7, 152.2, 149.7,140.8, 137.8, 136.3, 135.8, 118.6, 115, 114.8, 112.8, 112.5, 108.3, 101.8, 41.0, 40.8. Elemental analyses found: C%, 51.27; H%, 3.45; N%, 29.64; S%, 7.49. Calc. for C<sub>18</sub>H<sub>14</sub>FN<sub>9</sub>OS: C%, 51.06; H%, 3.33; N%, 29.77; S%, 7.57.</p></section></section><section id="sec71"><h3 class="pmc_sec_title">HPLC purity assessment of compounds 6b, 8b, 12a, 12b, and 12c</h3>
<p>The purity of the most active compounds (6b, 8b, 12a, 12b, and 12c) was evaluated using high-performance liquid chromatography (HPLC). Analyses were performed using a reverse-phase C18 column (250 mm × 4.6 mm, 5 μm) at ambient temperature. The mobile phase consisted of phosphate buffer (pH 6), adjusted with 0.1 N NaOH and acetonitrile in a 40 : 60 (v/v) ratio, filtered and degassed before use. The flow rate was set to 1.0 mL min<sup>−1</sup>, with UV detection at 254 nm. Allopurinol was used as an internal standard in all samples. Purity was quantified by calculating the ratio of the analyte peak area to that of the internal standard. Full chromatographic data are provided in the SI.</p></section><section id="sec72"><h3 class="pmc_sec_title">Biological investigation</h3>
<section id="sec73"><h4 class="pmc_sec_title">Cell lines</h4>
<p>All cell lines were obtained from the American Type Culture Collection (ATCC, Rockville, Maryland, USA) and are maintained at the Hirszfeld Institute of Immunology and Experimental Therapy, Polish Academy of Sciences (Wroclaw, Poland). Enzyme assay studies were conducted at the confirmatory diagnostic unit in VACSERA, Egypt.</p></section><section id="sec74"><h4 class="pmc_sec_title">Culture media</h4>
<p>The HepG2 cell line was cultured in Eagle's medium (IIET PAS, Wroclaw, Poland) containing 10% fetal bovine serum (HyClone, Life Technologies Poland) and 2 mM <span class="font-variant-small-caps">l</span>-glutamine (Sigma-Aldrich Chemie GmbH, Steinheim, Germany). The MV-4-11 cell line was cultured in RPMI 1640 with a stable glutamine medium (Biowest, Nuaillé, France) containing 10% fetal bovine serum and 1.0 mM sodium pyruvate (both Sigma-Aldrich Chemie GmbH, Steinheim, Germany). All culture media were supplemented with 100 U mL<sup>−1</sup> penicillin (Polfa Tarchomin S.A., Warsaw, Poland) and 100 μg mL<sup>−1</sup> streptomycin (Sigma-Aldrich Chemie GmbH, Steinheim, Germany). The cells were grown at 37 °C in a humid atmosphere saturated with 5% CO<sub>2</sub>.</p></section><section id="sec75"><h4 class="pmc_sec_title">Compounds</h4>
<p>7 compounds were tested. Cisplatin (CDDP, Cisplatin 1 mg mL<sup>−1</sup> Concentrate for Solution for Infusion, Accord Healthcare Poland), the active cytostatic drug, was used as a control. Sunitinib and sorafenib (both from Sigma-Aldrich, Merck, Darmstadt, Germany) were used as positive controls for the compounds. The samples of the compounds were prepared freshly before being added to the cells. Prior to usage, the compounds were weighed and dissolved in DMSO to a concentration of 10 mg mL<sup>−1</sup> and subsequently diluted in culture medium to reach the required concentration depending on the type of test. Camptothecin (Pol-Aura, Poland) and doxorubicin (Doxorubicin Accord 2 mg mL<sup>−1</sup>, Accord Healthcare Poland) were used as controls for the caspase-3/7 activity assay. Sunitinib, sorafenib and camptothecin were dissolved in DMSO. All cytostatics were diluted in culture medium to reach the required concentrations depending on the type of test.</p></section></section><section id="sec76"><h3 class="pmc_sec_title">Antiproliferative assay</h3>
<p>The antiproliferative activity of the tested analogs was evaluated using the SRB assay for adherent cells and the MTT assay for suspension (leukemic) cells. The A-498 cells were seeded at 1500 cells per well, HepG2, BALB/3T3 clone A31 and MV4-11 at 2000 cells per well, MDA-MB-231 at 2500 cells per well in 384-well plates (Greiner Bio-One, Kremsmünster, Austria). After 24 hours, cells were treated with tested compounds (0.1–100 μg mL<sup>−1</sup>) and incubated for 72 hours at 37 °C with 5% CO<sub>2</sub>. For the SRB assay, the cells were fixed with cold 50% (w/v) trichloroacetic acid (Avantor Performance Materials, Gliwice, Poland) and stained with a 0.4% (w/v) solution of sulforhodamine B (Sigma-Aldrich, Germany). Protein-bound dye was extracted from the stained cells with a 10 mM TRIS base (Avantor Performance Materials, Gliwice, Poland) solution, and absorbance was measured at 540 nm using a Biotek Hybrid H4 reader (BioTek Instruments, Inc., Winooski, Vermont, USA). For the MTT assay, MTT solution was added. After 4 h of incubation, formazan crystals were dissolved in DMF solution, and absorbance was recorded at a wavelength of 570 nm. Sorafenib served as a reference, and IC<sub>50</sub> values were determined using Cheburator 0.4 software. Each compound was tested in triplicate, and the experiments were repeated at least three times.</p></section><section id="sec77"><h3 class="pmc_sec_title">Cell cycle analysis</h3>
<p>Cells (1 × 10<sup>6</sup>) were washed twice with cold PBS and then fixed in 70% ethanol at −20 °C for 24 hours. Following PBS washing, the cells were treated with RNAse (8 mg mL<sup>−1</sup>, Thermo Scientific, Waltham, MA, USA) at 37 °C for 1 hour. Next, the cells were stained with propidium iodide (0.5 mg mL<sup>−1</sup>, Sigma-Aldrich Chemie GmbH, Germany) at 4 °C for 30 minutes, and the cellular DNA content was analyzed using BD LSR Fortessa II (Becton Dickinson, San Jose, CA, USA). The percentage of stained cells was calculated using ModFit LT version 3.2 (Verity Software House, USA) and Flowing Software version 2.5.1 (freeware developed by Perttu Terho, Turku Centre for Biotechnology, Finland).</p></section><section id="sec78"><h3 class="pmc_sec_title">Enzymatic caspase-3/7 activity assay</h3>
<p>An enzymatic caspase-3/7 activity assay was conducted using the HepG2 (hepatocellular carcinoma) cell line to evaluate the four test compounds. Cisplatin served as a cytostatic control. Camptothecin and doxorubicin were used as positive controls for the caspase-3/7 activity assay. All test compounds were dissolved in DMSO, diluted in culture medium, and applied to cells seeded in 24-well plates at a density of 1.0 × 10<sup>5</sup> cells per mL. After a 72 hour treatment, the cells were lysed, and caspase-3/7 activity was measured using Ac-DEVD-ACC as a fluorogenic substrate. Fluorescence was recorded for 2 hours at 37 °C. In parallel, an SRB antiproliferative assay was performed to assess cell viability. After 72 hours of compound exposure, the cells were fixed with trichloroacetic acid and stained with sulforhodamine B, and absorbance was measured at 540 nm. Both assays were conducted in duplicate, and the experiments were repeated three times.</p></section><section id="sec79"><h3 class="pmc_sec_title">
<em>In vitro</em> CDK6 inhibitory assay</h3>
<p>CDK6 inhibitory activity was evaluated using the ADP-Glo™ Kinase Assay CDK6 kit (BPS Bioscience, San Diego, CA, USA) following the provided protocol. This assay utilizes the Kinase-Glo® MAX reagent for detection and is formatted in a 96-well plate with sufficient quantities of purified recombinant CDK6/Cyclin D3 enzyme and CDK substrate peptide. The reaction was initiated by adding 20 μL of diluted CDK6/Cyclin D3 enzyme to each well, which contained varying concentrations of test compounds and a test inhibitor control. The plate was then incubated at 30 °C for 45 minutes. After incubation, 50 μL of Kinase-Glo Max reagent was added to each well, and the plate was covered with aluminum foil and incubated at room temperature for an additional 15 minutes. Luminescence was measured using a Tecan Spark reader with an excitation wavelength of 360 nm and an emission wavelength of 450 nm. Blank values were subtracted from all measurements to ensure accuracy.</p></section><section id="sec80"><h3 class="pmc_sec_title">
<em>In vitro</em> VEGFR-2 inhibitory activity assay</h3>
<p>VEGFR-2 kinase inhibition was evaluated using the VEGFR-2 Kinase Assay Kit (San Diego, CA, USA) following the manufacturer's protocol (<a href="https://bpsbioscience.com/vegfr2-kdrkinase-assay-kit-40325" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://bpsbioscience.com/vegfr2-kdrkinase-assay-kit-40325</a>). This 96-well format assay utilized Kinase-Glo Max for detection. A master mixture containing kinase buffer, ATP, PTK substrate, and water was prepared and added to each well. Test inhibitor wells received 5 μL of inhibitor solution, while control and blank wells received a solution without the inhibitor. VEGFR-2 enzyme was diluted to 1 ng μL<sup>−1</sup> and added to test and positive control wells, followed by incubation at 30 °C for 45 minutes. Afterward, Kinase-Glo Max reagent was added, and luminescence was measured using a microplate reader. IC<sub>50</sub> values were determined using non-linear regression analysis, with results presented as mean IC<sub>50</sub> (nM) ± SD from at least two independent experiments performed in triplicate.</p></section><section id="sec81"><h3 class="pmc_sec_title">Xanthine oxidase (XO) inhibition assay</h3>
<p>The xanthine oxidase (XO) inhibitory activity of compounds 6b, 7b, 8b, 12a–12c, and 13b was evaluated by measuring uric acid production at 292 nm using a spectrophotometric assay. Allopurinol served as a positive control. The reaction mixture comprised 1.5 mL of 50 mM potassium phosphate buffer (pH 7.4), 1 mL of the test compound at varying concentrations, and 0.5 mL of freshly prepared XO enzyme solution. After a 15 minute pre-incubation to allow enzyme–compound complex formation, 1 mL of 0.10 mM xanthine solution was added, followed by 30 minute incubation. The reaction was terminated with 1 mL of 1 M HCl. Control experiments were performed using phosphate buffer instead of the xanthine substrate. All measurements were conducted in triplicate, and the results were reported as IC<sub>50</sub> values ± standard deviation (SD). Kinetic parameters, including the Michaelis–Menten constant (<em>K</em><sub>m</sub>) and maximum velocity (<em>V</em><sub>max</sub>), were determined using either non-linear interpolation of the Michaelis–Menten equation or Lineweaver–Burk plot analysis.</p></section><section id="sec82"><h3 class="pmc_sec_title">
<em>In vitro</em> chemical stability assessment</h3>
<p>To investigate chemical stability under simulated physiological conditions, compounds 6b, 8b, and 12a–12c were prepared at 30 μM in phosphate-buffered saline (PBS, pH 7.4) and incubated at 37 ± 0.5 °C. Samples were collected at predetermined intervals (0, 0.5, 1, 2, 4, 6, 8, 10, and 24 h) and analyzed by UV-Vis spectrophotometry across the 190–800 nm range using a 1 cm quartz cuvette. The instrument was blanked with PBS prior to each measurement. Particular attention was given to identifying shifts in the absorption maxima (<em>λ</em><sub>max</sub>), variations in absorbance, or new spectral features indicating chemical degradation.</p></section></section><section id="sec83"><h2 class="pmc_sec_title">Author contributions</h2>
<p>All authors contributed significantly to the work. Gehan Ahmed Abdel Hafez: conducted the synthesis, biological evaluation, data curation, and prepared the original draft of the manuscript. Dagmara Kłopotowska, Beata Filip Psurska, Joanna Wietrzyk: performed biological evaluation and contributed to the experimental work. Ahmed S. Aboraia: carried out data analysis, molecular docking studies, figure preparation, and manuscript editing. Tarek Aboul Fadl: conceptualization, methodology design, supervision, project administration, and critical revision of the manuscript. Adel F. Youssef: conceptualization, supervision, project administration, data analysis, and critical manuscript revision. All authors reviewed and approved the final version of the manuscript.</p></section><section id="sec84"><h2 class="pmc_sec_title">Conflicts of interest</h2>
<p>The authors declare that they have no conflicts of interest regarding the publication of this article.</p></section><section id="sec85"><h2 class="pmc_sec_title">Supplementary Material</h2>
<section class="sm xbox font-sm" id="RA-015-D5RA04997K-s001"><div class="caption p"><span>RA-015-D5RA04997K-s001</span></div>
<div class="media p" id="d67e4469"><div class="caption">
<a href="/articles/instance/12377214/bin/RA-015-D5RA04997K-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">RA-015-D5RA04997K-s001.pdf</a><sup> (5.8MB, pdf) </sup>
</div></div></section></section><section id="sec86"><h2 class="pmc_sec_title">Data availability</h2>
<p>The authors declare that all the data generated or analyzed during this study are included in this manuscript and its SI. See DOI: <a href="https://doi.org/10.1039/d5ra04997k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1039/d5ra04997k</a>.</p></section><section id="ref-list1" class="ref-list"><h2 class="pmc_sec_title">References</h2>
<section id="ref-list1_sec2"><ol class="ref-list font-sm">
<li id="cit1">
<cite>Bray F. Laversanne M. Weiderpass E. Soerjomataram I. The ever-increasing importance of cancer as a leading cause of premature death worldwide. Cancer. 2021;127(16):3029–3030. doi: 10.1002/cncr.33587.</cite> [<a href="https://doi.org/10.1002/cncr.33587" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34086348/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Cancer&amp;title=The%20ever-increasing%20importance%20of%20cancer%20as%20a%20leading%20cause%20of%20premature%20death%20worldwide&amp;author=F.%20Bray&amp;author=M.%20Laversanne&amp;author=E.%20Weiderpass&amp;author=I.%20Soerjomataram&amp;volume=127&amp;issue=16&amp;publication_year=2021&amp;pages=3029-3030&amp;pmid=34086348&amp;doi=10.1002/cncr.33587&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit2">
<cite>Chen S. Cao Z. Prettner K. Kuhn M. Yang J. Jiao L. 
et al., Estimates and projections of the global economic cost of 29 cancers in 204 countries and territories from 2020 to 2050. JAMA Oncol. 2023;9(4):465–472. doi: 10.1001/jamaoncol.2022.7826.</cite> [<a href="https://doi.org/10.1001/jamaoncol.2022.7826" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9951101/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36821107/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=JAMA%20Oncol.&amp;title=et%20al.,%20Estimates%20and%20projections%20of%20the%20global%20economic%20cost%20of%2029%20cancers%20in%20204%20countries%20and%20territories%20from%202020%20to%202050&amp;author=S.%20Chen&amp;author=Z.%20Cao&amp;author=K.%20Prettner&amp;author=M.%20Kuhn&amp;author=J.%20Yang&amp;volume=9&amp;issue=4&amp;publication_year=2023&amp;pages=465-472&amp;pmid=36821107&amp;doi=10.1001/jamaoncol.2022.7826&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit3">
<cite>Long Z. Zuo Y. Li R. Le Y. Dong Y. Yan L. Design, synthesis and biological evaluation of 4-arylamino-pyrimidine derivatives as focal adhesion kinase inhibitors. J. Bioorg. Chem. 2023;140:106792. doi: 10.1016/j.bioorg.2023.106792.</cite> [<a href="https://doi.org/10.1016/j.bioorg.2023.106792" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37633129/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Bioorg.%20Chem.&amp;title=Design,%20synthesis%20and%20biological%20evaluation%20of%204-arylamino-pyrimidine%20derivatives%20as%20focal%20adhesion%20kinase%20inhibitors&amp;author=Z.%20Long&amp;author=Y.%20Zuo&amp;author=R.%20Li&amp;author=Y.%20Le&amp;author=Y.%20Dong&amp;volume=140&amp;publication_year=2023&amp;pages=106792&amp;pmid=37633129&amp;doi=10.1016/j.bioorg.2023.106792&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit4">
<cite>Ding L. Cao J. Lin W. Chen H. Xiong X. Ao H. 
et al., The roles of cyclin-dependent kinases in cell-cycle progression and therapeutic strategies in human breast cancer. Int. J. Mol. Sci. 2020;21(6):1960. doi: 10.3390/ijms21061960.</cite> [<a href="https://doi.org/10.3390/ijms21061960" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7139603/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32183020/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=et%20al.,%20The%20roles%20of%20cyclin-dependent%20kinases%20in%20cell-cycle%20progression%20and%20therapeutic%20strategies%20in%20human%20breast%20cancer&amp;author=L.%20Ding&amp;author=J.%20Cao&amp;author=W.%20Lin&amp;author=H.%20Chen&amp;author=X.%20Xiong&amp;volume=21&amp;issue=6&amp;publication_year=2020&amp;pages=1960&amp;pmid=32183020&amp;doi=10.3390/ijms21061960&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit5">
<cite>
CDK inhibitors from past to present: A new wave of cancer therapy, Seminars in Cancer Biology, ed. M. J. Mughal, K. Bhadresha and H. F. Kwok, Elsevier, 2023</cite> [<a href="https://doi.org/10.1016/j.semcancer.2022.12.006" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36565895/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?title=Seminars%20in%20Cancer%20Biology&amp;publication_year=2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit6">
<cite>Yousuf M. Khan P. Shamsi A. Shahbaaz M. Hasan G. M. Haque Q. M. R. 
et al., Inhibiting CDK6 activity by quercetin is an attractive strategy for cancer therapy. ACS Omega. 2020;5(42):27480–27491. doi: 10.1021/acsomega.0c03975.</cite> [<a href="https://doi.org/10.1021/acsomega.0c03975" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7594119/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33134711/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Omega&amp;title=et%20al.,%20Inhibiting%20CDK6%20activity%20by%20quercetin%20is%20an%20attractive%20strategy%20for%20cancer%20therapy&amp;author=M.%20Yousuf&amp;author=P.%20Khan&amp;author=A.%20Shamsi&amp;author=M.%20Shahbaaz&amp;author=G.%20M.%20Hasan&amp;volume=5&amp;issue=42&amp;publication_year=2020&amp;pages=27480-27491&amp;pmid=33134711&amp;doi=10.1021/acsomega.0c03975&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit7">
<cite>Chong Q.-Y. Kok Z.-H. Xiang X. Wong A. L.-A. Yong W.-P. Sethi G. 
et al., A unique CDK4/6 inhibitor: current and future therapeutic strategies of abemaciclib. Pharmacol. Res. 2020;156:104686. doi: 10.1016/j.phrs.2020.104686.</cite> [<a href="https://doi.org/10.1016/j.phrs.2020.104686" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32068118/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmacol.%20Res.&amp;title=et%20al.,%20A%20unique%20CDK4/6%20inhibitor:%20current%20and%20future%20therapeutic%20strategies%20of%20abemaciclib&amp;author=Q.-Y.%20Chong&amp;author=Z.-H.%20Kok&amp;author=X.%20Xiang&amp;author=A.%20L.-A.%20Wong&amp;author=W.-P.%20Yong&amp;volume=156&amp;publication_year=2020&amp;pages=104686&amp;pmid=32068118&amp;doi=10.1016/j.phrs.2020.104686&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit8">
<cite>Goel S. Bergholz J. S. Zhao J. J. Targeting CDK4 and CDK6 in cancer. Nat. Rev. Cancer. 2022;22(6):356–372. doi: 10.1038/s41568-022-00456-3.</cite> [<a href="https://doi.org/10.1038/s41568-022-00456-3" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9149100/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35304604/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Nat.%20Rev.%20Cancer&amp;title=Targeting%20CDK4%20and%20CDK6%20in%20cancer&amp;author=S.%20Goel&amp;author=J.%20S.%20Bergholz&amp;author=J.%20J.%20Zhao&amp;volume=22&amp;issue=6&amp;publication_year=2022&amp;pages=356-372&amp;pmid=35304604&amp;doi=10.1038/s41568-022-00456-3&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit9">
<cite>Hu Q. Huang T. Regulation of the cell cycle by ncRNAs affects the efficiency of CDK4/6 inhibition. Int. J. Mol. Sci. 2023;24(10):8939. doi: 10.3390/ijms24108939.</cite> [<a href="https://doi.org/10.3390/ijms24108939" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10219488/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37240281/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Regulation%20of%20the%20cell%20cycle%20by%20ncRNAs%20affects%20the%20efficiency%20of%20CDK4/6%20inhibition&amp;author=Q.%20Hu&amp;author=T.%20Huang&amp;volume=24&amp;issue=10&amp;publication_year=2023&amp;pages=8939&amp;pmid=37240281&amp;doi=10.3390/ijms24108939&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit10">
<cite>Yuan K. Wang X. Dong H. Min W. Hao H. Yang P. Selective inhibition of CDK4/6: A safe and effective strategy for developing anticancer drugs. Acta Pharm. Sin. B. 2021;11(1):30–54. doi: 10.1016/j.apsb.2020.05.001.</cite> [<a href="https://doi.org/10.1016/j.apsb.2020.05.001" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7838032/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33532179/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Acta%20Pharm.%20Sin.%20B&amp;title=Selective%20inhibition%20of%20CDK4/6:%20A%20safe%20and%20effective%20strategy%20for%20developing%20anticancer%20drugs&amp;author=K.%20Yuan&amp;author=X.%20Wang&amp;author=H.%20Dong&amp;author=W.%20Min&amp;author=H.%20Hao&amp;volume=11&amp;issue=1&amp;publication_year=2021&amp;pages=30-54&amp;pmid=33532179&amp;doi=10.1016/j.apsb.2020.05.001&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit11">
<cite>Łukasik P. Baranowska-Bosiacka I. Kulczycka K. Gutowska I. Inhibitors of cyclin-dependent kinases: types and their mechanism of action. Int. J. Mol. Sci. 2021;22(6):2806. doi: 10.3390/ijms22062806.</cite> [<a href="https://doi.org/10.3390/ijms22062806" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8001317/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33802080/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Mol.%20Sci.&amp;title=Inhibitors%20of%20cyclin-dependent%20kinases:%20types%20and%20their%20mechanism%20of%20action&amp;author=P.%20%C5%81ukasik&amp;author=I.%20Baranowska-Bosiacka&amp;author=K.%20Kulczycka&amp;author=I.%20Gutowska&amp;volume=22&amp;issue=6&amp;publication_year=2021&amp;pages=2806&amp;pmid=33802080&amp;doi=10.3390/ijms22062806&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit12">
<cite>Zhang M. Zhang L. Hei R. Li X. Cai H. Wu X. 
et al., CDK inhibitors in cancer therapy, an overview of recent development. Am. J. Cancer Res. 2021;11(5):1913.</cite> [<a href="/articles/PMC8167670/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34094661/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Am.%20J.%20Cancer%20Res.&amp;title=et%20al.,%20CDK%20inhibitors%20in%20cancer%20therapy,%20an%20overview%20of%20recent%20development&amp;author=M.%20Zhang&amp;author=L.%20Zhang&amp;author=R.%20Hei&amp;author=X.%20Li&amp;author=H.%20Cai&amp;volume=11&amp;issue=5&amp;publication_year=2021&amp;pages=1913&amp;pmid=34094661&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit13">
<cite>Izadi S. Hojjat-Farsangi M. Karpisheh V. Jadidi-Niaragh F. Pan Cyclin-Dependent Kinase Inhibitors for the Treatment of Breast Cancer. Int. J. Drug Res. Clin. 2023;1(1):e2–e.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Int.%20J.%20Drug%20Res.%20Clin.&amp;title=Pan%20Cyclin-Dependent%20Kinase%20Inhibitors%20for%20the%20Treatment%20of%20Breast%20Cancer&amp;author=S.%20Izadi&amp;author=M.%20Hojjat-Farsangi&amp;author=V.%20Karpisheh&amp;author=F.%20Jadidi-Niaragh&amp;volume=1&amp;issue=1&amp;publication_year=2023&amp;pages=e2-e&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit14">
<cite>Sun J. Liang S. Liu X. Zhang S. Li M. Zhang Q. 
et al., Insights into the selectivity of a brain-penetrant CDK4/6 vs CDK1/2 inhibitor for glioblastoma used in multiple replica molecular dynamics simulations. J. Biomol. Struct. Dyn. 2023:1–20. doi: 10.1080/07391102.2023.2294175.</cite> [<a href="https://doi.org/10.1080/07391102.2023.2294175" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38112295/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Biomol.%20Struct.%20Dyn.&amp;title=et%20al.,%20Insights%20into%20the%20selectivity%20of%20a%20brain-penetrant%20CDK4/6%20vs%20CDK1/2%20inhibitor%20for%20glioblastoma%20used%20in%20multiple%20replica%20molecular%20dynamics%20simulations&amp;author=J.%20Sun&amp;author=S.%20Liang&amp;author=X.%20Liu&amp;author=S.%20Zhang&amp;author=M.%20Li&amp;publication_year=2023&amp;pages=1-20&amp;pmid=38112295&amp;doi=10.1080/07391102.2023.2294175&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit15">
<cite>Abd El-wahab H. A. Mansour H. S. Ali A. M. El-Awady R. Aboul-Fadl T. New cell cycle checkpoint pathways regulators with 2-Oxo-indoline scaffold as potential anticancer agents: Design, synthesis, biological activities and in silico studies. Bioorg. Chem. 2022;120:105622. doi: 10.1016/j.bioorg.2022.105622.</cite> [<a href="https://doi.org/10.1016/j.bioorg.2022.105622" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35091290/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Chem.&amp;title=New%20cell%20cycle%20checkpoint%20pathways%20regulators%20with%202-Oxo-indoline%20scaffold%20as%20potential%20anticancer%20agents:%20Design,%20synthesis,%20biological%20activities%20and%20in%20silico%20studies&amp;author=H.%20A.%20Abd%20El-wahab&amp;author=H.%20S.%20Mansour&amp;author=A.%20M.%20Ali&amp;author=R.%20El-Awady&amp;author=T.%20Aboul-Fadl&amp;volume=120&amp;publication_year=2022&amp;pages=105622&amp;pmid=35091290&amp;doi=10.1016/j.bioorg.2022.105622&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit16">
<cite>Campos J. F. Besson T. Berteina-Raboin S. Review on the Synthesis and Therapeutic Potential of Pyrido [2, 3-d],[3, 2-d],[3, 4-d] and [4, 3-d] pyrimidine Derivatives. Pharmaceuticals. 2022;15(3):352. doi: 10.3390/ph15030352.</cite> [<a href="https://doi.org/10.3390/ph15030352" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8949896/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35337149/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmaceuticals&amp;title=Review%20on%20the%20Synthesis%20and%20Therapeutic%20Potential%20of%20Pyrido%20%5B2,%203-d%5D,%5B3,%202-d%5D,%5B3,%204-d%5D%20and%20%5B4,%203-d%5D%20pyrimidine%20Derivatives&amp;author=J.%20F.%20Campos&amp;author=T.%20Besson&amp;author=S.%20Berteina-Raboin&amp;volume=15&amp;issue=3&amp;publication_year=2022&amp;pages=352&amp;pmid=35337149&amp;doi=10.3390/ph15030352&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit17">
<cite>Nikhar S. Siokas I. Schlicher L. Lee S. Gyrd-Hansen M. Degterev A. 
et al., Design of pyrido [2, 3-d] pyrimidin-7-one inhibitors of receptor interacting protein kinase-2 (RIPK2) and nucleotide-binding
oligomerization domain (NOD) cell signaling. Eur. J. Med. Chem. 2021;215:113252. doi: 10.1016/j.ejmech.2021.113252.</cite> [<a href="https://doi.org/10.1016/j.ejmech.2021.113252" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8009825/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33601309/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Med.%20Chem.&amp;title=et%20al.,%20Design%20of%20pyrido%20%5B2,%203-d%5D%20pyrimidin-7-one%20inhibitors%20of%20receptor%20interacting%20protein%20kinase-2%20(RIPK2)%20and%20nucleotide-binding%20oligomerization%20domain%20(NOD)%20cell%20signaling&amp;author=S.%20Nikhar&amp;author=I.%20Siokas&amp;author=L.%20Schlicher&amp;author=S.%20Lee&amp;author=M.%20Gyrd-Hansen&amp;volume=215&amp;publication_year=2021&amp;pages=113252&amp;pmid=33601309&amp;doi=10.1016/j.ejmech.2021.113252&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit18">
<cite>Bhurta D. Bharate S. B. Analyzing the scaffold diversity of cyclin-dependent kinase inhibitors and revisiting the clinical and preclinical pipeline. Med. Res. Rev. 2022;42(2):654–709. doi: 10.1002/med.21856.</cite> [<a href="https://doi.org/10.1002/med.21856" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34605036/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Med.%20Res.%20Rev.&amp;title=Analyzing%20the%20scaffold%20diversity%20of%20cyclin-dependent%20kinase%20inhibitors%20and%20revisiting%20the%20clinical%20and%20preclinical%20pipeline&amp;author=D.%20Bhurta&amp;author=S.%20B.%20Bharate&amp;volume=42&amp;issue=2&amp;publication_year=2022&amp;pages=654-709&amp;pmid=34605036&amp;doi=10.1002/med.21856&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit19">
<cite>Liang T. Yang Y. Wang J. Xie Z. Chen X. The application of pyrrolo [2, 3-d] pyrimidine scaffold in medicinal chemistry from 2017 to 2021. Mini-Rev. Med. Chem. 2023;23(10):1118–1136. doi: 10.2174/1389557523666230111161810.</cite> [<a href="https://doi.org/10.2174/1389557523666230111161810" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36635929/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mini-Rev.%20Med.%20Chem.&amp;title=The%20application%20of%20pyrrolo%20%5B2,%203-d%5D%20pyrimidine%20scaffold%20in%20medicinal%20chemistry%20from%202017%20to%202021&amp;author=T.%20Liang&amp;author=Y.%20Yang&amp;author=J.%20Wang&amp;author=Z.%20Xie&amp;author=X.%20Chen&amp;volume=23&amp;issue=10&amp;publication_year=2023&amp;pages=1118-1136&amp;pmid=36635929&amp;doi=10.2174/1389557523666230111161810&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit20">
<cite>Lv S. Yang J. Lin J. Huang X. Zhao H. Zhao C. 
et al., CDK4/6 inhibitors in lung cancer: current practice and future directions. Eur. Respir. Rev. 2024;33(171):230145. doi: 10.1183/16000617.0145-2023.</cite> [<a href="https://doi.org/10.1183/16000617.0145-2023" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10865100/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38355149/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20Respir.%20Rev.&amp;title=et%20al.,%20CDK4/6%20inhibitors%20in%20lung%20cancer:%20current%20practice%20and%20future%20directions&amp;author=S.%20Lv&amp;author=J.%20Yang&amp;author=J.%20Lin&amp;author=X.%20Huang&amp;author=H.%20Zhao&amp;volume=33&amp;issue=171&amp;publication_year=2024&amp;pages=230145&amp;pmid=38355149&amp;doi=10.1183/16000617.0145-2023&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit21">
<cite>Abuzenadah A. M. Al-Sayes F. Mahafujul Alam S. S. Hoque M. Karim S. Hussain I. M. 
et al., Elucidating antiangiogenic potential of Rauwolfia serpentina: VEGFR-2 targeting-based molecular docking study. J. Evidence-Based Complementary Altern. Med. 2022;2022(1):6224666. doi: 10.1155/2022/6224666.</cite> [<a href="https://doi.org/10.1155/2022/6224666" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8860507/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35198035/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Evidence-Based%20Complementary%20Altern.%20Med.&amp;title=et%20al.,%20Elucidating%20antiangiogenic%20potential%20of%20Rauwolfia%20serpentina:%20VEGFR-2%20targeting-based%20molecular%20docking%20study&amp;author=A.%20M.%20Abuzenadah&amp;author=F.%20Al-Sayes&amp;author=S.%20S.%20Mahafujul%20Alam&amp;author=M.%20Hoque&amp;author=S.%20Karim&amp;volume=2022&amp;issue=1&amp;publication_year=2022&amp;pages=6224666&amp;pmid=35198035&amp;doi=10.1155/2022/6224666&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit22">
<cite>Liu Y. Li Y. Wang Y. Lin C. Zhang D. Chen J. 
et al., Recent progress on vascular endothelial growth factor receptor inhibitors with dual targeting capabilities for tumor therapy. J. Hematol. Oncol. 2022;15(1):89. doi: 10.1186/s13045-022-01310-7.</cite> [<a href="https://doi.org/10.1186/s13045-022-01310-7" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9263050/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35799213/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Hematol.%20Oncol.&amp;title=et%20al.,%20Recent%20progress%20on%20vascular%20endothelial%20growth%20factor%20receptor%20inhibitors%20with%20dual%20targeting%20capabilities%20for%20tumor%20therapy&amp;author=Y.%20Liu&amp;author=Y.%20Li&amp;author=Y.%20Wang&amp;author=C.%20Lin&amp;author=D.%20Zhang&amp;volume=15&amp;issue=1&amp;publication_year=2022&amp;pages=89&amp;pmid=35799213&amp;doi=10.1186/s13045-022-01310-7&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit23">
<cite>Moradi M. Mousavi A. Emamgholipour Z. Giovannini J. Moghimi S. Peytam F. 
et al., Quinazoline-based VEGFR-2 inhibitors as potential anti-angiogenic agents: A contemporary perspective of SAR and molecular docking studies. Eur. J. Med. Chem. 2023:115626. doi: 10.1016/j.ejmech.2023.115626.</cite> [<a href="https://doi.org/10.1016/j.ejmech.2023.115626" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37453330/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Med.%20Chem.&amp;title=et%20al.,%20Quinazoline-based%20VEGFR-2%20inhibitors%20as%20potential%20anti-angiogenic%20agents:%20A%20contemporary%20perspective%20of%20SAR%20and%20molecular%20docking%20studies&amp;author=M.%20Moradi&amp;author=A.%20Mousavi&amp;author=Z.%20Emamgholipour&amp;author=J.%20Giovannini&amp;author=S.%20Moghimi&amp;publication_year=2023&amp;pages=115626&amp;pmid=37453330&amp;doi=10.1016/j.ejmech.2023.115626&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit24">
<cite>Zuo Y. Li R. Zhang Y. Bao G. Le Y. Yan L. 
et al., Design, synthesis and antitumor activity of 5-trifluoromethylpyrimidine derivatives as EGFR inhibitors. J. Enzyme Inhib. Med. Chem. 2022;37(1):2742–2754. doi: 10.1080/14756366.2022.2128797.</cite> [<a href="https://doi.org/10.1080/14756366.2022.2128797" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9542405/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36176072/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Enzyme%20Inhib.%20Med.%20Chem.&amp;title=et%20al.,%20Design,%20synthesis%20and%20antitumor%20activity%20of%205-trifluoromethylpyrimidine%20derivatives%20as%20EGFR%20inhibitors&amp;author=Y.%20Zuo&amp;author=R.%20Li&amp;author=Y.%20Zhang&amp;author=G.%20Bao&amp;author=Y.%20Le&amp;volume=37&amp;issue=1&amp;publication_year=2022&amp;pages=2742-2754&amp;pmid=36176072&amp;doi=10.1080/14756366.2022.2128797&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit25">
<cite>Huang C.-Y. Chang Y.-Y. Chang S.-T. Chang H.-T. Xanthine oxidase inhibitory activity and chemical composition of Pistacia chinensis leaf essential oil. Pharmaceutics. 2022;14(10):1982. doi: 10.3390/pharmaceutics14101982.</cite> [<a href="https://doi.org/10.3390/pharmaceutics14101982" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC9609098/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36297418/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Pharmaceutics&amp;title=Xanthine%20oxidase%20inhibitory%20activity%20and%20chemical%20composition%20of%20Pistacia%20chinensis%20leaf%20essential%20oil&amp;author=C.-Y.%20Huang&amp;author=Y.-Y.%20Chang&amp;author=S.-T.%20Chang&amp;author=H.-T.%20Chang&amp;volume=14&amp;issue=10&amp;publication_year=2022&amp;pages=1982&amp;pmid=36297418&amp;doi=10.3390/pharmaceutics14101982&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit26">
<cite>Joshi G. Sharma M. Kalra S. Gavande N. S. Singh S. Kumar R. J. B. C. Design, synthesis, biological evaluation of 3, 5-diaryl-4, 5-dihydro-1H-pyrazole carbaldehydes as non-purine xanthine oxidase inhibitors: Tracing the anticancer mechanism via xanthine oxidase inhibition. Bioorg. Chem. 2021;107:104620. doi: 10.1016/j.bioorg.2020.104620.</cite> [<a href="https://doi.org/10.1016/j.bioorg.2020.104620" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/33454509/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Chem.&amp;title=Design,%20synthesis,%20biological%20evaluation%20of%203,%205-diaryl-4,%205-dihydro-1H-pyrazole%20carbaldehydes%20as%20non-purine%20xanthine%20oxidase%20inhibitors:%20Tracing%20the%20anticancer%20mechanism%20via%20xanthine%20oxidase%20inhibition&amp;author=G.%20Joshi&amp;author=M.%20Sharma&amp;author=S.%20Kalra&amp;author=N.%20S.%20Gavande&amp;author=S.%20Singh&amp;volume=107&amp;publication_year=2021&amp;pages=104620&amp;pmid=33454509&amp;doi=10.1016/j.bioorg.2020.104620&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit27">
<cite>Li Y. Cao T.-T. Guo S. Zhong Q. Li C.-H. Li Y. 
et al., Discovery of novel allopurinol derivatives with anticancer activity and attenuated xanthine oxidase inhibition. Molecules. 2016;21(6):771. doi: 10.3390/molecules21060771.</cite> [<a href="https://doi.org/10.3390/molecules21060771" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC5538589/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27331805/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=et%20al.,%20Discovery%20of%20novel%20allopurinol%20derivatives%20with%20anticancer%20activity%20and%20attenuated%20xanthine%20oxidase%20inhibition&amp;author=Y.%20Li&amp;author=T.-T.%20Cao&amp;author=S.%20Guo&amp;author=Q.%20Zhong&amp;author=C.-H.%20Li&amp;volume=21&amp;issue=6&amp;publication_year=2016&amp;pages=771&amp;pmid=27331805&amp;doi=10.3390/molecules21060771&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit28">
<cite>Hassan H. Abbas S. H. Beshr E. A. Ezelarab H. A. Ali T. F. The main biotargets of indole or 2-oxoindole-based hybrids acting as promising antiproliferative agents. J. Adv. Biomedical Pharm. Sci. 2023;6(4):174–183.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Adv.%20Biomedical%20Pharm.%20Sci.&amp;title=The%20main%20biotargets%20of%20indole%20or%202-oxoindole-based%20hybrids%20acting%20as%20promising%20antiproliferative%20agents&amp;author=H.%20Hassan&amp;author=S.%20H.%20Abbas&amp;author=E.%20A.%20Beshr&amp;author=H.%20A.%20Ezelarab&amp;author=T.%20F.%20Ali&amp;volume=6&amp;issue=4&amp;publication_year=2023&amp;pages=174-183&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit29">
<cite>Turhal G. Demirkan B. Baslilar I. N. Yuncu N. S. Baytas S. N. Demiroglu-Zergeroglu A. Preliminary evaluation of antiproliferative and apoptotic activities of novel indolin-2-one derivatives. Drug Dev. Res. 2024;85(5):e22229. doi: 10.1002/ddr.22229.</cite> [<a href="https://doi.org/10.1002/ddr.22229" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38958104/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Drug%20Dev.%20Res.&amp;title=Preliminary%20evaluation%20of%20antiproliferative%20and%20apoptotic%20activities%20of%20novel%20indolin-2-one%20derivatives&amp;author=G.%20Turhal&amp;author=B.%20Demirkan&amp;author=I.%20N.%20Baslilar&amp;author=N.%20S.%20Yuncu&amp;author=S.%20N.%20Baytas&amp;volume=85&amp;issue=5&amp;publication_year=2024&amp;pages=e22229&amp;pmid=38958104&amp;doi=10.1002/ddr.22229&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit30">
<cite>Younis M. H. Mohammed E. R. Georgey H. H. 4-Thiazolidinones: A structural motif of great synthetic and biological activities. ERU. Res. J. 2023;2(3):525–540.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=ERU.%20Res.%20J.&amp;title=4-Thiazolidinones:%20A%20structural%20motif%20of%20great%20synthetic%20and%20biological%20activities&amp;author=M.%20H.%20Younis&amp;author=E.%20R.%20Mohammed&amp;author=H.%20H.%20Georgey&amp;volume=2&amp;issue=3&amp;publication_year=2023&amp;pages=525-540&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit31">
<cite>Desai N. C. Jadeja D. J. Jethawa A. M. Ahmad I. Patel H. Dave B. Design and synthesis of some novel hybrid molecules based on 4-thiazolidinone bearing pyridine-pyrazole scaffolds: molecular docking and molecular dynamics simulations of its major constituent onto DNA gyrase inhibition. Mol. Diversity. 2024;28(2):693–709. doi: 10.1007/s11030-023-10612-y.</cite> [<a href="https://doi.org/10.1007/s11030-023-10612-y" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36750538/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Diversity&amp;title=Design%20and%20synthesis%20of%20some%20novel%20hybrid%20molecules%20based%20on%204-thiazolidinone%20bearing%20pyridine-pyrazole%20scaffolds:%20molecular%20docking%20and%20molecular%20dynamics%20simulations%20of%20its%20major%20constituent%20onto%20DNA%20gyrase%20inhibition&amp;author=N.%20C.%20Desai&amp;author=D.%20J.%20Jadeja&amp;author=A.%20M.%20Jethawa&amp;author=I.%20Ahmad&amp;author=H.%20Patel&amp;volume=28&amp;issue=2&amp;publication_year=2024&amp;pages=693-709&amp;pmid=36750538&amp;doi=10.1007/s11030-023-10612-y&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit32">
<cite>Zokaasadi M. Altunel E. Dayanidhi D. L. Mantyh J. B. Watlington W. K. McCall S. J. 
et al., Efficacy of fadraciclib (CYC065), a novel dual CDK2/9 inhibitor, on patient-derived models of colorectal cancer. J. Clin.
Oncol. 2024;42(16):3596.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Clin.%20Oncol.&amp;title=et%20al.,%20Efficacy%20of%20fadraciclib%20(CYC065),%20a%20novel%20dual%20CDK2/9%20inhibitor,%20on%20patient-derived%20models%20of%20colorectal%20cancer&amp;author=M.%20Zokaasadi&amp;author=E.%20Altunel&amp;author=D.%20L.%20Dayanidhi&amp;author=J.%20B.%20Mantyh&amp;author=W.%20K.%20Watlington&amp;volume=42&amp;issue=16&amp;publication_year=2024&amp;pages=3596&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit33">
<cite>Zaghary W. Nasr T. El-Hameed A. Sakr T. M. Osman D. A. El-Adl K. Pyrazolopyrimidines as anticancer agents; syntheses and mode of action. Al-Azhar J. Pharm. Sci. 2024;69(1):62–107.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Al-Azhar%20J.%20Pharm.%20Sci.&amp;title=Pyrazolopyrimidines%20as%20anticancer%20agents;%20syntheses%20and%20mode%20of%20action&amp;author=W.%20Zaghary&amp;author=T.%20Nasr&amp;author=A.%20El-Hameed&amp;author=T.%20M.%20Sakr&amp;author=D.%20A.%20Osman&amp;volume=69&amp;issue=1&amp;publication_year=2024&amp;pages=62-107&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit34">
<cite>Adel D. El-Adl K. Nasr T. Sakr T. M. Zaghary W. Pyrazolo [3, 4-d] pyrimidine derivatives as EGFRT790M and VEGFR-2 dual TK inhibitors: Design, synthesis, molecular docking, ADMET profile and anticancer evaluations. J. Mol. Struct. 2023;1291:136047.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Struct.&amp;title=Pyrazolo%20%5B3,%204-d%5D%20pyrimidine%20derivatives%20as%20EGFRT790M%20and%20VEGFR-2%20dual%20TK%20inhibitors:%20Design,%20synthesis,%20molecular%20docking,%20ADMET%20profile%20and%20anticancer%20evaluations&amp;author=D.%20Adel&amp;author=K.%20El-Adl&amp;author=T.%20Nasr&amp;author=T.%20M.%20Sakr&amp;author=W.%20Zaghary&amp;volume=1291&amp;publication_year=2023&amp;pages=136047&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit35">
<cite>Chen P. Lee N. V. Hu W. Xu M. Ferre R. A. Lam H. 
et al., Spectrum and degree of CDK drug interactions predicts clinical performance. Mol. Cancer Ther. 2016;15(10):2273–2281. doi: 10.1158/1535-7163.MCT-16-0300.</cite> [<a href="https://doi.org/10.1158/1535-7163.MCT-16-0300" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/27496135/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Mol.%20Cancer%20Ther.&amp;title=et%20al.,%20Spectrum%20and%20degree%20of%20CDK%20drug%20interactions%20predicts%20clinical%20performance&amp;author=P.%20Chen&amp;author=N.%20V.%20Lee&amp;author=W.%20Hu&amp;author=M.%20Xu&amp;author=R.%20A.%20Ferre&amp;volume=15&amp;issue=10&amp;publication_year=2016&amp;pages=2273-2281&amp;pmid=27496135&amp;doi=10.1158/1535-7163.MCT-16-0300&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit36">
<cite>Adel D. El-Adl K. Nasr T. Sakr T. M. Zaghary W. Pyrazolo [3, 4-d] pyrimidine derivatives as EGFRT790M and VEGFR-2 dual TK inhibitors: Design, synthesis, molecular docking, ADMET profile and anticancer evaluations. J. Mol. Struct. 2023;1291:136047.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Struct.&amp;title=Pyrazolo%20%5B3,%204-d%5D%20pyrimidine%20derivatives%20as%20EGFRT790M%20and%20VEGFR-2%20dual%20TK%20inhibitors:%20Design,%20synthesis,%20molecular%20docking,%20ADMET%20profile%20and%20anticancer%20evaluations&amp;author=D.%20Adel&amp;author=K.%20El-Adl&amp;author=T.%20Nasr&amp;author=T.%20M.%20Sakr&amp;author=W.%20Zaghary&amp;volume=1291&amp;publication_year=2023&amp;pages=136047&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit37">
<cite>Moradi M. Mousavi A. Emamgholipour Z. Giovannini J. Moghimi S. Peytam F. 
et al., Quinazoline-based VEGFR-2 inhibitors as potential anti-angiogenic agents: A contemporary perspective of SAR and molecular docking studies. Eur. J. Med. Chem. 2023:115626. doi: 10.1016/j.ejmech.2023.115626.</cite> [<a href="https://doi.org/10.1016/j.ejmech.2023.115626" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37453330/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Med.%20Chem.&amp;title=et%20al.,%20Quinazoline-based%20VEGFR-2%20inhibitors%20as%20potential%20anti-angiogenic%20agents:%20A%20contemporary%20perspective%20of%20SAR%20and%20molecular%20docking%20studies&amp;author=M.%20Moradi&amp;author=A.%20Mousavi&amp;author=Z.%20Emamgholipour&amp;author=J.%20Giovannini&amp;author=S.%20Moghimi&amp;publication_year=2023&amp;pages=115626&amp;pmid=37453330&amp;doi=10.1016/j.ejmech.2023.115626&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit38">
<cite>Okamoto K. Ikemori-Kawada M. Jestel A. von König K. Funahashi Y. Matsushima T. 
et al., Distinct binding mode of multikinase inhibitor lenvatinib revealed by biochemical characterization. ACS Med. Chem. Lett. 2015;6(1):89–94. doi: 10.1021/ml500394m.</cite> [<a href="https://doi.org/10.1021/ml500394m" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4291723/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25589937/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Med.%20Chem.%20Lett.&amp;title=et%20al.,%20Distinct%20binding%20mode%20of%20multikinase%20inhibitor%20lenvatinib%20revealed%20by%20biochemical%20characterization&amp;author=K.%20Okamoto&amp;author=M.%20Ikemori-Kawada&amp;author=A.%20Jestel&amp;author=K.%20von%20K%C3%B6nig&amp;author=Y.%20Funahashi&amp;volume=6&amp;issue=1&amp;publication_year=2015&amp;pages=89-94&amp;pmid=25589937&amp;doi=10.1021/ml500394m&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit39">
<cite>Hassan A. Badr M. Abdelhamid D. Hassan H. A. Abourehab M. A. Abuo-Rahma G. E. D. A. Design, synthesis, in vitro antiproliferative evaluation and in silico studies of new VEGFR-2 inhibitors based on 4-piperazinylquinolin-2 (1H)-one scaffold. Bioorg. Chem. 2022;120:105631. doi: 10.1016/j.bioorg.2022.105631.</cite> [<a href="https://doi.org/10.1016/j.bioorg.2022.105631" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/35091289/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Chem.&amp;title=Design,%20synthesis,%20in%20vitro%20antiproliferative%20evaluation%20and%20in%20silico%20studies%20of%20new%20VEGFR-2%20inhibitors%20based%20on%204-piperazinylquinolin-2%20(1H)-one%20scaffold&amp;author=A.%20Hassan&amp;author=M.%20Badr&amp;author=D.%20Abdelhamid&amp;author=H.%20A.%20Hassan&amp;author=M.%20A.%20Abourehab&amp;volume=120&amp;publication_year=2022&amp;pages=105631&amp;pmid=35091289&amp;doi=10.1016/j.bioorg.2022.105631&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit40">
<cite>Kohli E. Arora R. Kakkar R. Theoretical study of the stability of tautomers and conformers of isatin-3-thiosemicarbazone (IBT) Can. Chem. Trans. 2014;2(3):327–342.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Can.%20Chem.%20Trans.&amp;title=Theoretical%20study%20of%20the%20stability%20of%20tautomers%20and%20conformers%20of%20isatin-3-thiosemicarbazone%20(IBT)&amp;author=E.%20Kohli&amp;author=R.%20Arora&amp;author=R.%20Kakkar&amp;volume=2&amp;issue=3&amp;publication_year=2014&amp;pages=327-342&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit41">
<cite>Shahi N. Yadav P. N. Chaudhary U. Saad M. Mahiya K. Khan A. 
et al., 5-Methoxyisatin N (4)-Pyrrolidinyl Thiosemicarbazone (MeOIstPyrd) Restores Mutant p53 and Inhibits the Growth of Skin Cancer Cells, In Vitro. ACS Omega. 2023;8(35):31998–32016. doi: 10.1021/acsomega.3c03824.</cite> [<a href="https://doi.org/10.1021/acsomega.3c03824" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10483675/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37692215/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=ACS%20Omega&amp;title=et%20al.,%205-Methoxyisatin%20N%20(4)-Pyrrolidinyl%20Thiosemicarbazone%20(MeOIstPyrd)%20Restores%20Mutant%20p53%20and%20Inhibits%20the%20Growth%20of%20Skin%20Cancer%20Cells,%20In%20Vitro&amp;author=N.%20Shahi&amp;author=P.%20N.%20Yadav&amp;author=U.%20Chaudhary&amp;author=M.%20Saad&amp;author=K.%20Mahiya&amp;volume=8&amp;issue=35&amp;publication_year=2023&amp;pages=31998-32016&amp;pmid=37692215&amp;doi=10.1021/acsomega.3c03824&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit42">
<cite>Caron G. Kihlberg J. Ermondi G. Intramolecular hydrogen bonding: An opportunity for improved design in medicinal chemistry. Med. Res. Rev. 2019;39(5):1707–1729. doi: 10.1002/med.21562.</cite> [<a href="https://doi.org/10.1002/med.21562" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30659634/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Med.%20Res.%20Rev.&amp;title=Intramolecular%20hydrogen%20bonding:%20An%20opportunity%20for%20improved%20design%20in%20medicinal%20chemistry&amp;author=G.%20Caron&amp;author=J.%20Kihlberg&amp;author=G.%20Ermondi&amp;volume=39&amp;issue=5&amp;publication_year=2019&amp;pages=1707-1729&amp;pmid=30659634&amp;doi=10.1002/med.21562&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit43">
<cite>Tadesse S. Yu M. Mekonnen L. B. Lam F. Islam S. Tomusange K. 
et al., Highly potent, selective, and orally bioavailable 4-thiazol-N-(pyridin-2-yl) pyrimidin-2-amine cyclin-dependent kinases 4 and 6 inhibitors as anticancer drug candidates: design, synthesis, and evaluation. J. Med. Chem. 2017;60(5):1892–1915. doi: 10.1021/acs.jmedchem.6b01670.</cite> [<a href="https://doi.org/10.1021/acs.jmedchem.6b01670" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28156111/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&amp;title=et%20al.,%20Highly%20potent,%20selective,%20and%20orally%20bioavailable%204-thiazol-N-(pyridin-2-yl)%20pyrimidin-2-amine%20cyclin-dependent%20kinases%204%20and%206%20inhibitors%20as%20anticancer%20drug%20candidates:%20design,%20synthesis,%20and%20evaluation&amp;author=S.%20Tadesse&amp;author=M.%20Yu&amp;author=L.%20B.%20Mekonnen&amp;author=F.%20Lam&amp;author=S.%20Islam&amp;volume=60&amp;issue=5&amp;publication_year=2017&amp;pages=1892-1915&amp;pmid=28156111&amp;doi=10.1021/acs.jmedchem.6b01670&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit44">
<cite>Asadi M. Taghizadeh S. Kaviani E. Vakili O. Taheri-Anganeh M. Tahamtan M. 
et al., Caspase-3: structure, function, and biotechnological aspects. Biotechnol. Appl. Biochem. 2022;69(4):1633–1645. doi: 10.1002/bab.2233.</cite> [<a href="https://doi.org/10.1002/bab.2233" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34342377/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Biotechnol.%20Appl.%20Biochem.&amp;title=et%20al.,%20Caspase-3:%20structure,%20function,%20and%20biotechnological%20aspects&amp;author=M.%20Asadi&amp;author=S.%20Taghizadeh&amp;author=E.%20Kaviani&amp;author=O.%20Vakili&amp;author=M.%20Taheri-Anganeh&amp;volume=69&amp;issue=4&amp;publication_year=2022&amp;pages=1633-1645&amp;pmid=34342377&amp;doi=10.1002/bab.2233&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit45">
<cite>Jin S. Park H.-J. Oh Y. N. Kwon H. J. Kim J.-H. Choi Y. H. 
et al., Anti-cancer activity of Osmanthus matsumuranus extract by inducing G2/M arrest and apoptosis in human hepatocellular carcinoma Hep G2 cells. J. Cancer Prev. 2015;20(4):241. doi: 10.15430/JCP.2015.20.4.241.</cite> [<a href="https://doi.org/10.15430/JCP.2015.20.4.241" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC4699751/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/26734586/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Cancer%20Prev.&amp;title=et%20al.,%20Anti-cancer%20activity%20of%20Osmanthus%20matsumuranus%20extract%20by%20inducing%20G2/M%20arrest%20and%20apoptosis%20in%20human%20hepatocellular%20carcinoma%20Hep%20G2%20cells&amp;author=S.%20Jin&amp;author=H.-J.%20Park&amp;author=Y.%20N.%20Oh&amp;author=H.%20J.%20Kwon&amp;author=J.-H.%20Kim&amp;volume=20&amp;issue=4&amp;publication_year=2015&amp;pages=241&amp;pmid=26734586&amp;doi=10.15430/JCP.2015.20.4.241&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit46">
<cite>Takahama U. Koga Y. Hirota S. Yamauchi R. Inhibition of xanthine oxidase activity by an oxathiolanone derivative of quercetin. Food Chem. 2011;126(4):1808–1811. doi: 10.1016/j.foodchem.2010.12.009.</cite> [<a href="https://doi.org/10.1016/j.foodchem.2010.12.009" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25213960/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Food%20Chem.&amp;title=Inhibition%20of%20xanthine%20oxidase%20activity%20by%20an%20oxathiolanone%20derivative%20of%20quercetin&amp;author=U.%20Takahama&amp;author=Y.%20Koga&amp;author=S.%20Hirota&amp;author=R.%20Yamauchi&amp;volume=126&amp;issue=4&amp;publication_year=2011&amp;pages=1808-1811&amp;pmid=25213960&amp;doi=10.1016/j.foodchem.2010.12.009&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit47">
<cite>de Araújo M. E. M. B. Franco Y. E. M. Alberto T. G. Messias M. C. F. Leme C. W. Sawaya A. C. H. F. 
et al., Kinetic study on the inhibition of xanthine oxidase by acylated derivatives
of flavonoids synthesised enzymatically. J. Enzyme Inhib. Med. Chem. 2017;32(1):978–985. doi: 10.1080/14756366.2017.1347165.</cite> [<a href="https://doi.org/10.1080/14756366.2017.1347165" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6445226/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/28718686/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Enzyme%20Inhib.%20Med.%20Chem.&amp;title=et%20al.,%20Kinetic%20study%20on%20the%20inhibition%20of%20xanthine%20oxidase%20by%20acylated%20derivatives%20of%20flavonoids%20synthesised%20enzymatically&amp;author=M.%20E.%20M.%20B.%20de%20Ara%C3%BAjo&amp;author=Y.%20E.%20M.%20Franco&amp;author=T.%20G.%20Alberto&amp;author=M.%20C.%20F.%20Messias&amp;author=C.%20W.%20Leme&amp;volume=32&amp;issue=1&amp;publication_year=2017&amp;pages=978-985&amp;pmid=28718686&amp;doi=10.1080/14756366.2017.1347165&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit48">
<cite>Kumar D. Kaur G. Negi A. Kumar S. Singh S. Kumar R. Synthesis and xanthine oxidase inhibitory activity of 5, 6-dihydropyrazolo/pyrazolo [1, 5-c] quinazoline derivatives. Bioorg. Chem. 2014;57:57–64. doi: 10.1016/j.bioorg.2014.08.007.</cite> [<a href="https://doi.org/10.1016/j.bioorg.2014.08.007" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/25222504/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Bioorg.%20Chem.&amp;title=Synthesis%20and%20xanthine%20oxidase%20inhibitory%20activity%20of%205,%206-dihydropyrazolo/pyrazolo%20%5B1,%205-c%5D%20quinazoline%20derivatives&amp;author=D.%20Kumar&amp;author=G.%20Kaur&amp;author=A.%20Negi&amp;author=S.%20Kumar&amp;author=S.%20Singh&amp;volume=57&amp;publication_year=2014&amp;pages=57-64&amp;pmid=25222504&amp;doi=10.1016/j.bioorg.2014.08.007&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit49">
<cite>Rullo R. Cerchia C. Nasso R. Romanelli V. Vendittis E. D. Masullo M. 
et al., Novel reversible inhibitors of xanthine oxidase targeting the active site of the enzyme. Antioxidants. 2023;12(4):825. doi: 10.3390/antiox12040825.</cite> [<a href="https://doi.org/10.3390/antiox12040825" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10135315/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/37107199/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Antioxidants&amp;title=et%20al.,%20Novel%20reversible%20inhibitors%20of%20xanthine%20oxidase%20targeting%20the%20active%20site%20of%20the%20enzyme&amp;author=R.%20Rullo&amp;author=C.%20Cerchia&amp;author=R.%20Nasso&amp;author=V.%20Romanelli&amp;author=E.%20D.%20Vendittis&amp;volume=12&amp;issue=4&amp;publication_year=2023&amp;pages=825&amp;pmid=37107199&amp;doi=10.3390/antiox12040825&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit50">
<cite>Nguyen P. T. V. Van Dat T. Mizukami S. Nguyen D. L. H. Mosaddeque F. Kim S. N. 
et al., 2D-quantitative structure–activity relationships model using PLS method for anti-malarial activities of anti-haemozoin compounds. Malar. J. 2021;20:1–15. doi: 10.1186/s12936-021-03775-2.</cite> [<a href="https://doi.org/10.1186/s12936-021-03775-2" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC8196453/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/34116665/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Malar.%20J.&amp;title=et%20al.,%202D-quantitative%20structure%E2%80%93activity%20relationships%20model%20using%20PLS%20method%20for%20anti-malarial%20activities%20of%20anti-haemozoin%20compounds&amp;author=P.%20T.%20V.%20Nguyen&amp;author=T.%20Van%20Dat&amp;author=S.%20Mizukami&amp;author=D.%20L.%20H.%20Nguyen&amp;author=F.%20Mosaddeque&amp;volume=20&amp;publication_year=2021&amp;pages=1-15&amp;pmid=34116665&amp;doi=10.1186/s12936-021-03775-2&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit51">
<cite>Yellasubbaiah N. Velmurugan V. QSAR Modeling, Molecular Docking, and ADME Studies of Novel 5-Oxo- Imidazoline Derivatives as Anti-Breast Cancer Drug Compounds against MCF-7 Cell Line. J. Med. Chem. Sci. 2023;6:3087–3112.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.%20Sci.&amp;title=QSAR%20Modeling,%20Molecular%20Docking,%20and%20ADME%20Studies%20of%20Novel%205-Oxo-%20Imidazoline%20Derivatives%20as%20Anti-Breast%20Cancer%20Drug%20Compounds%20against%20MCF-7%20Cell%20Line&amp;author=N.%20Yellasubbaiah&amp;author=V.%20Velmurugan&amp;volume=6&amp;publication_year=2023&amp;pages=3087-3112&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit52">
<cite>Lahyaoui M. El-Idrissi H. Saffaj T. Ihssane B. Saffaj N. Mamouni R. 
et al., QSAR modeling, molecular docking and molecular dynamic simulation of phosphorus-substituted quinoline derivatives as topoisomerase I inhibitors. Arabian J. Chem. 2023;16(6):104783.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Arabian%20J.%20Chem.&amp;title=et%20al.,%20QSAR%20modeling,%20molecular%20docking%20and%20molecular%20dynamic%20simulation%20of%20phosphorus-substituted%20quinoline%20derivatives%20as%20topoisomerase%20I%20inhibitors&amp;author=M.%20Lahyaoui&amp;author=H.%20El-Idrissi&amp;author=T.%20Saffaj&amp;author=B.%20Ihssane&amp;author=N.%20Saffaj&amp;volume=16&amp;issue=6&amp;publication_year=2023&amp;pages=104783&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit53">
<cite>Ismail M. A. Negm A. Arafa R. K. Abdel-Latif E. El-Sayed W. M. Anticancer activity, dual prooxidant/antioxidant effect and apoptosis induction profile of new bichalcophene-5-carboxamidines. Eur. J. Med. Chem. 2019;169:76–88. doi: 10.1016/j.ejmech.2019.02.062.</cite> [<a href="https://doi.org/10.1016/j.ejmech.2019.02.062" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/30856408/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Eur.%20J.%20Med.%20Chem.&amp;title=Anticancer%20activity,%20dual%20prooxidant/antioxidant%20effect%20and%20apoptosis%20induction%20profile%20of%20new%20bichalcophene-5-carboxamidines&amp;author=M.%20A.%20Ismail&amp;author=A.%20Negm&amp;author=R.%20K.%20Arafa&amp;author=E.%20Abdel-Latif&amp;author=W.%20M.%20El-Sayed&amp;volume=169&amp;publication_year=2019&amp;pages=76-88&amp;pmid=30856408&amp;doi=10.1016/j.ejmech.2019.02.062&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit54">
<cite>Suhasini M. Sailatha E. Gunasekaran S. Ramkumaar G. Vibrational and electronic investigations, thermodynamic parameters, HOMO and LUMO analysis on Lornoxicam by density functional theory. J. Mol. Struct. 2015;1100:116–128.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Mol.%20Struct.&amp;title=Vibrational%20and%20electronic%20investigations,%20thermodynamic%20parameters,%20HOMO%20and%20LUMO%20analysis%20on%20Lornoxicam%20by%20density%20functional%20theory&amp;author=M.%20Suhasini&amp;author=E.%20Sailatha&amp;author=S.%20Gunasekaran&amp;author=G.%20Ramkumaar&amp;volume=1100&amp;publication_year=2015&amp;pages=116-128&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit55">
<cite>Rajpurohit A. S. Rajesh R. Muhamed R. R. Jaccob M. Baskar A. J. A. Kannappan V. J. H. DFT investigation of role of N–H⋯O and N–H⋯π interactions in the stabilization of the hydrogen bonded complexes of anisole with aromatic amines. Heliyon. 2019;5(7):e02155. doi: 10.1016/j.heliyon.2019.e02155.</cite> [<a href="https://doi.org/10.1016/j.heliyon.2019.e02155" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC6667824/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/31388586/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Heliyon&amp;title=DFT%20investigation%20of%20role%20of%20N%E2%80%93H%E2%8B%AFO%20and%20N%E2%80%93H%E2%8B%AF%CF%80%20interactions%20in%20the%20stabilization%20of%20the%20hydrogen%20bonded%20complexes%20of%20anisole%20with%20aromatic%20amines&amp;author=A.%20S.%20Rajpurohit&amp;author=R.%20Rajesh&amp;author=R.%20R.%20Muhamed&amp;author=M.%20Jaccob&amp;author=A.%20J.%20A.%20Baskar&amp;volume=5&amp;issue=7&amp;publication_year=2019&amp;pages=e02155&amp;pmid=31388586&amp;doi=10.1016/j.heliyon.2019.e02155&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit56">
<cite>Aceves-Hernández J. M. Inés Nicolás Vázquez M. Luis Garza Rivera J. Espinoza Godínez A. Mateo Flores J. de Jesús Cruz Guzmán J. 
et al., Palbociclib (PD 0332991) Interaction with Kinases. Theoretical and Comparative Molecular Docking Study. Chem. Biodiversity. 2023;20(3):e202200554. doi: 10.1002/cbdv.202200554.</cite> [<a href="https://doi.org/10.1002/cbdv.202200554" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/36799136/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Chem.%20Biodiversity&amp;title=et%20al.,%20Palbociclib%20(PD%200332991)%20Interaction%20with%20Kinases.%20Theoretical%20and%20Comparative%20Molecular%20Docking%20Study&amp;author=J.%20M.%20Aceves-Hern%C3%A1ndez&amp;author=M.%20In%C3%A9s%20Nicol%C3%A1s%20V%C3%A1zquez&amp;author=J.%20Luis%20Garza%20Rivera&amp;author=A.%20Espinoza%20God%C3%ADnez&amp;author=J.%20Mateo%20Flores&amp;volume=20&amp;issue=3&amp;publication_year=2023&amp;pages=e202200554&amp;pmid=36799136&amp;doi=10.1002/cbdv.202200554&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit57">
<cite>Gul S. Jan F. Alam A. Shakoor A. Khan A. AlAsmari A. F. 
et al., Synthesis, molecular docking and DFT analysis of novel bis-Schiff base derivatives with thiobarbituric acid for α-glucosidase inhibition assessment. Sci. Rep. 2024;14(1):3419. doi: 10.1038/s41598-024-54021-z.</cite> [<a href="https://doi.org/10.1038/s41598-024-54021-z" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10858901/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38341468/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=Sci.%20Rep.&amp;title=et%20al.,%20Synthesis,%20molecular%20docking%20and%20DFT%20analysis%20of%20novel%20bis-Schiff%20base%20derivatives%20with%20thiobarbituric%20acid%20for%20%CE%B1-glucosidase%20inhibition%20assessment&amp;author=S.%20Gul&amp;author=F.%20Jan&amp;author=A.%20Alam&amp;author=A.%20Shakoor&amp;author=A.%20Khan&amp;volume=14&amp;issue=1&amp;publication_year=2024&amp;pages=3419&amp;pmid=38341468&amp;doi=10.1038/s41598-024-54021-z&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit58"><cite>
<a href="https://www.molinpiration.com/cgi-bin/properties" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://www.molinpiration.com/cgi-bin/properties</a>, June, 2024</cite></li>
<li id="cit59">
<cite>Veber D. F. Johnson S. R. Cheng H.-Y. Smith B. R. Ward K. W. Kopple K. D. Molecular properties that influence the oral bioavailability of drug candidates. J. Med. Chem. 2002;45(12):2615–2623. doi: 10.1021/jm020017n.</cite> [<a href="https://doi.org/10.1021/jm020017n" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/12036371/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Med.%20Chem.&amp;title=Molecular%20properties%20that%20influence%20the%20oral%20bioavailability%20of%20drug%20candidates&amp;author=D.%20F.%20Veber&amp;author=S.%20R.%20Johnson&amp;author=H.-Y.%20Cheng&amp;author=B.%20R.%20Smith&amp;author=K.%20W.%20Ward&amp;volume=45&amp;issue=12&amp;publication_year=2002&amp;pages=2615-2623&amp;pmid=12036371&amp;doi=10.1021/jm020017n&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit60">
<cite>Phillips J. C. Hardy D. J. Maia J. D. Stone J. E. Ribeiro J. V. Bernardi R. C. 
et al., Scalable molecular dynamics on CPU and GPU architectures with NAMD. J. Chem. Phys. 2020;153(4):044130. doi: 10.1063/5.0014475.</cite> [<a href="https://doi.org/10.1063/5.0014475" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC7395834/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/32752662/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=J.%20Chem.%20Phys.&amp;title=et%20al.,%20Scalable%20molecular%20dynamics%20on%20CPU%20and%20GPU%20architectures%20with%20NAMD&amp;author=J.%20C.%20Phillips&amp;author=D.%20J.%20Hardy&amp;author=J.%20D.%20Maia&amp;author=J.%20E.%20Stone&amp;author=J.%20V.%20Ribeiro&amp;volume=153&amp;issue=4&amp;publication_year=2020&amp;pages=044130&amp;pmid=32752662&amp;doi=10.1063/5.0014475&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit61">
<cite>Chagaleti B. K. Saravanan V. Vellapandian C. Kathiravan M. K. Exploring cyclin-dependent kinase inhibitors: a comprehensive study in search of CDK-6 inhibitors using a pharmacophore modelling and dynamics approach. RSC Adv. 2023;13(48):33770–33785. doi: 10.1039/d3ra05672d.</cite> [<a href="https://doi.org/10.1039/d3ra05672d" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">DOI</a>] [<a href="/articles/PMC10655667/" class="usa-link">PMC free article</a>] [<a href="https://pubmed.ncbi.nlm.nih.gov/38019988/" class="usa-link">PubMed</a>] [<a href="https://scholar.google.com/scholar_lookup?journal=RSC%20Adv.&amp;title=Exploring%20cyclin-dependent%20kinase%20inhibitors:%20a%20comprehensive%20study%20in%20search%20of%20CDK-6%20inhibitors%20using%20a%20pharmacophore%20modelling%20and%20dynamics%20approach&amp;author=B.%20K.%20Chagaleti&amp;author=V.%20Saravanan&amp;author=C.%20Vellapandian&amp;author=M.%20K.%20Kathiravan&amp;volume=13&amp;issue=48&amp;publication_year=2023&amp;pages=33770-33785&amp;pmid=38019988&amp;doi=10.1039/d3ra05672d&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
<li id="cit62">
<cite>Bogdanov A. Sirazieva A. Voloshina A. Abzalilov T. Samorodov A. Mironov V. Synthesis and antimicrobial, antiplatelet, and anticoagulant activities of new isatin derivatives containing a hetero-fused imidazole fragment. Russ. J. Org. Chem. 2022;58(3):327–334.</cite> [<a href="https://scholar.google.com/scholar_lookup?journal=Russ.%20J.%20Org.%20Chem.&amp;title=Synthesis%20and%20antimicrobial,%20antiplatelet,%20and%20anticoagulant%20activities%20of%20new%20isatin%20derivatives%20containing%20a%20hetero-fused%20imidazole%20fragment&amp;author=A.%20Bogdanov&amp;author=A.%20Sirazieva&amp;author=A.%20Voloshina&amp;author=T.%20Abzalilov&amp;author=A.%20Samorodov&amp;volume=58&amp;issue=3&amp;publication_year=2022&amp;pages=327-334&amp;" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">Google Scholar</a>]</li>
</ol></section></section><section id="_ad93_" lang="en" class="associated-data"><h2 class="pmc_sec_title">Associated Data</h2>
<p class="font-secondary"><em>This section collects any data citations, data availability statements, or supplementary materials included in this article.</em></p>
<section id="_adsm93_" lang="en" class="supplementary-materials"><h3 class="pmc_sec_title">Supplementary Materials</h3>
<section class="sm xbox font-sm" id="db_ds_supplementary-material1_reqid_"><div class="caption p"><span>RA-015-D5RA04997K-s001</span></div>
<div class="media p"><div class="caption">
<a href="/articles/instance/12377214/bin/RA-015-D5RA04997K-s001.pdf" data-ga-action="click_feat_suppl" class="usa-link">RA-015-D5RA04997K-s001.pdf</a><sup> (5.8MB, pdf) </sup>
</div></div></section></section><section id="_adda93_" lang="en" class="data-availability-statement"><h3 class="pmc_sec_title">Data Availability Statement</h3>
<p>The authors declare that all the data generated or analyzed during this study are included in this manuscript and its SI. See DOI: <a href="https://doi.org/10.1039/d5ra04997k" class="usa-link usa-link--external" data-ga-action="click_feat_suppl" target="_blank" rel="noopener noreferrer">https://doi.org/10.1039/d5ra04997k</a>.</p></section></section></section><footer class="p courtesy-note font-secondary font-sm text-center"><hr class="headless">
<p>Articles from RSC Advances are provided here courtesy of <strong>Royal Society of Chemistry</strong></p></footer></section></article>

                      

                    </main>
                </div>
            </div>
        </div>

        



<!-- Secondary navigation placeholder -->
<div class="pmc-sidenav desktop:grid-col-4 display-flex">
    <section class="pmc-sidenav__container" aria-label="Article resources and navigation">
        <button type="button" class="usa-button pmc-sidenav__container__close usa-button--unstyled">
            <img src="/static/img/usa-icons/close.svg" role="img" alt="Close" />
        </button>
    <div class="display-none desktop:display-block">
       <section class="margin-top-4 desktop:margin-top-0">
              <h2 class="margin-top-0">ACTIONS</h2>
           <ul class="usa-list usa-list--unstyled usa-list--actions">
               
               <li>
                     <a
                             href="https://doi.org/10.1039/d5ra04997k"
                             class="usa-button usa-button--outline width-24 font-xs display-inline-flex flex-align-center flex-justify-start padding-left-1"
                             target="_blank"
                             rel="noreferrer noopener"
                             data-ga-category="actions"
                             data-ga-action="click"
                             data-ga-label="publisher_link_desktop"
                     >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#launch"></use>
                         </svg>
                         <span class="display-inline-flex flex-justify-center flex-1 padding-right-2">View on publisher site</span>
                     </a>
               </li>
               
               
               <li>
                    <a
                            href="pdf/RA-015-D5RA04997K.pdf"
                            class="usa-button usa-button--outline width-24 display-inline-flex flex-align-center flex-justify-start padding-left-1"
                            data-ga-category="actions"
                            data-ga-action="click"
                            data-ga-label="pdf_download_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#file_download"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1">PDF (2.7 MB)</span>
                    </a>
               </li>
               
                
               <li>
                   <button role="button" class="usa-button width-24 citation-dialog-trigger display-inline-flex flex-align-center flex-justify-start padding-left-1"
                        aria-label="Open dialog with citation text in different styles"
                        data-ga-category="actions"
                        data-ga-action="open"
                        data-ga-label="cite_desktop"
                        data-all-citations-url="/resources/citations/12377214/"
                        data-citation-style="nlm"
                        data-download-format-link="/resources/citations/12377214/export/"
                    >
                        <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#format_quote"></use>
                        </svg>
                       <span class="display-inline-flex flex-justify-center flex-1 button-label">Cite</span>
                    </button>
               </li>
                
               <li>

                        <button class="usa-button width-24 collections-dialog-trigger collections-button display-inline-flex flex-align-center flex-justify-start padding-left-1 collections-button-empty"
                              aria-label="Save article in MyNCBI collections."
                              data-ga-category="actions"
                              data-ga-action="click"
                              data-ga-label="collections_button_desktop"
                              data-collections-open-dialog-enabled="false"
                              data-collections-open-dialog-url="https://account.ncbi.nlm.nih.gov/?back_url=https%3A%2F%2Fpmc.ncbi.nlm.nih.gov%2Farticles%2FPMC12377214%2F%3Freport%3Dclassic%23open-collections-dialog"
                              data-in-collections="false">
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-full" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-full.svg#icon"></use>
                            </svg>
                            <svg class="usa-icon width-3 height-3 usa-icon--bookmark-empty" aria-hidden="true" focusable="false" role="img" hidden>
                                <use xlink:href="/static/img/action-bookmark-empty.svg#icon"></use>
                            </svg>
                            <span class="display-inline-flex flex-justify-center flex-1">Collections</span>
                       </button>
               </li>
               <li class="pmc-permalink">
                    <button
                            type="button"
                            class="usa-button width-24 display-inline-flex flex-align-center flex-justify padding-left-1 shadow-none"
                            aria-label="Show article permalink"
                            aria-expanded="false"
                            aria-haspopup="true"
                            data-ga-category="actions"
                            data-ga-action="open"
                            data-ga-label="permalink_desktop"
                    >
                         <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img" hidden>
                            <use xlink:href="/static/img/sprite.svg#share"></use>
                        </svg>
                        <span class="display-inline-flex flex-justify-center flex-1 button-label">Permalink</span>
                    </button>
                   

<div class="pmc-permalink__dropdown" hidden>
    <div class="pmc-permalink__dropdown__container">
          <h2 class="usa-modal__heading margin-top-0 margin-bottom-2 text-uppercase font-sans-xs">PERMALINK</h2>
          <div class="pmc-permalink__dropdown__content">
              <input type="text" class="usa-input" value="https://pmc.ncbi.nlm.nih.gov/articles/PMC12377214/" aria-label="Article permalink">
              <button class="usa-button display-inline-flex pmc-permalink__dropdown__copy__btn margin-right-0" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                  <svg class="usa-icon" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#content_copy"></use>
                  </svg>
                  <span class="margin-left-1">Copy</span>
              </button>
          </div>
    </div>
</div>
               </li>
           </ul>
       </section>
     </div>

        <section class="pmc-resources margin-top-6 desktop:margin-top-4" data-page-path="/articles/PMC12377214/">
            <h2 class="margin-top-0">RESOURCES</h2>
            
                <div class="usa-accordion usa-accordion--multiselectable" data-allow-multiple>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-similar-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_similar_articles"
                        data-ga-label="/articles/PMC12377214/"
                        data-action-open="open_similar_articles"
                        data-action-close="close_similar_articles"
                        >
                            Similar articles
                        </button>
                    </h3>
                    <div
                            id="resources-similar-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                        
                    </div>
                    <h3 class="usa-accordion__heading">
                        <button
                        type="button"
                        class="usa-accordion__button"
                        aria-expanded="false"
                        aria-controls="resources-cited-by-other-articles"
                        data-ga-category="resources_accordion"
                        data-ga-action="open_cited_by"
                        data-ga-label="/articles/PMC12377214/"
                        data-action-open="open_cited_by"
                        data-action-close="close_cited_by"
                        >
                             Cited by other articles
                        </button>
                    </h3>
                    <div
                            id="resources-cited-by-other-articles"
                            class="usa-accordion__content usa-prose"
                            
                    >
                          
                    </div>
                    
                        <h3 class="usa-accordion__heading">
                            <button
                            type="button"
                            class="usa-accordion__button"
                            aria-expanded="false"
                            aria-controls="resources-links-to-ncbi-databases"
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/articles/PMC12377214/"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                            >
                                 Links to NCBI Databases
                            </button>
                        </h3>
                        <div
                                id="resources-links-to-ncbi-databases"
                                class="usa-accordion__content usa-prose"
                                data-source-url="/resources/db-links/12377214/"
                        >
                        </div>
                    
                    
                </div>
            
        </section>


        <section
        class="usa-in-page-nav usa-in-page-nav--wide margin-top-6 desktop:margin-top-4"
        data-title-text="On this page"
        data-title-heading-level="h2"
        data-scroll-offset="0"
        data-root-margin="-10% 0px -80% 0px"
        data-main-content-selector="main"
        data-threshold="1"
        hidden
        ></section>
    </section>
</div>


        

<div class="overlay" role="dialog" aria-label="Citation Dialog" hidden>
    <div class="dialog citation-dialog" aria-hidden="true">
        <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
            <h2 class="usa-modal__heading margin-0">Cite</h2>
             <button type="button" class="usa-button usa-button--unstyled close-overlay text-black width-auto"  tabindex="1">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#close"></use>
                </svg>
             </button>
        </div>

        

<div class="citation-text-block">
  <div class="citation-text margin-bottom-2"></div>
  <ul class="usa-list usa-list--unstyled display-inline-flex flex-justify width-full flex-align-center">
      <li>
        <button
          class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center copy-button dialog-focus"
          data-ga-category="save_share"
          data-ga-action="cite"
          data-ga-label="copy"
          tabindex="2">
            <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                <use xlink:href="/static/img/sprite.svg#content_copy"></use>
            </svg>
            <span>Copy</span>
        </button>
      </li>
      <li>
          <a
              href="#"
              role="button"
              class="usa-button usa-button--unstyled text-no-underline display-flex flex-align-center export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
                <svg class="usa-icon width-3 height-3" aria-hidden="true" focusable="false" role="img">
                    <use xlink:href="/static/img/sprite.svg#file_download"></use>
                </svg>
                <span class="display-none mobile-lg:display-inline">Download .nbib</span>
                <span class="display-inline mobile-lg:display-none">.nbib</span>
            </a>
      </li>
      <li>
          

<div class="display-inline-flex flex-align-center">
  <label class="usa-label margin-top-0">Format:</label>
  <select aria-label="Format" class="usa-select citation-style-selector padding-1 margin-top-0 border-0 padding-right-4" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
      </li>
  </ul>
</div>
    </div>
</div>

        <div class="overlay" role="dialog" hidden>
  <div id="collections-action-dialog" class="dialog collections-dialog" aria-hidden="true">
   <div class="display-inline-flex flex-align-center flex-justify width-full margin-bottom-2">
        <h2 class="usa-modal__heading margin-0">Add to Collections</h2>
    </div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="usa-form maxw-full collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/list-existing-collections/"
      data-add-to-existing-collection-url="/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

    <input type="hidden" name="csrfmiddlewaretoken" value="7iXj2GTYp3uHCbq5zIlEszogl8bXeMyg5BXjkFIGRqniXIVlMl58WIgD91LIV4zb">

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-new"
            class="usa-radio__input usa-radio__input--tile collections-new  margin-top-0"
            name="collections"
            value="new"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_new" />
            <label class="usa-radio__label" for="collections-action-dialog-new">Create a new collection</label>
        </div>
        <div class="usa-radio">
            <input type="radio"
            id="collections-action-dialog-existing"
            class="usa-radio__input usa-radio__input--tile collections-existing"
            name="collections"
            value="existing"
            checked="true"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="collections_radio_existing" />
            <label class="usa-radio__label" for="collections-action-dialog-existing">Add to an existing collection</label>
        </div>
    </fieldset>

    <fieldset class="usa-fieldset margin-bottom-2">
        <div class="action-panel-control-wrap new-collections-controls">
           <label for="collections-action-dialog-add-to-new" class="usa-label margin-top-0">
                Name your collection
               <abbr title="required" class="usa-hint usa-hint--required text-no-underline">*</abbr>
          </label>
          <input
            type="text"
            name="add-to-new-collection"
            id="collections-action-dialog-add-to-new"
            class="usa-input collections-action-add-to-new"
            pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
            maxlength=""
            data-ga-category="collections_button"
            data-ga-action="create_collection"
            data-ga-label="non_favorties_collection"
            required
          />
        </div>
        <div class="action-panel-control-wrap existing-collections-controls">
             <label for="collections-action-dialog-add-to-existing" class="usa-label margin-top-0">
                Choose a collection
              </label>
              <select id="collections-action-dialog-add-to-existing"
                      class="usa-select collections-action-add-to-existing"
                      data-ga-category="collections_button"
                      data-ga-action="select_collection"
                      data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
              </select>
              <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
                Unable to load your collection due to an error<br>
                <a href="#">Please try again</a>
              </div>
        </div>
    </fieldset>

    <div class="display-inline-flex">
        <button class="usa-button margin-top-0 action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
          Add
        </button>
        <button class="usa-button usa-button--outline margin-top-0 action-panel-cancel"
                aria-label="Close 'Add to Collections' panel"
                ref="linksrc=close_collections_panel"
                data-ga-category="collections_button"
                data-ga-action="click"
                data-ga-label="cancel">
          Cancel
        </button>
    </div>
</form>
    </div>
  </div>
</div>

        

      </div>
    </div>
  </div>



        
    
    

<footer class="ncbi-footer ncbi-dark-background " >
    
        <div class="ncbi-footer__icon-section">
            <div class="ncbi-footer__social-header">
                Follow NCBI
            </div>

            <div class="grid-container ncbi-footer__ncbi-social-icons-container">
                
                    <a href="https://twitter.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="40"
                             height="40"
                             viewBox="0 0 40 40"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on X (formerly known as Twitter)</span>
                    </a>
                

                
                    <a href="https://www.facebook.com/ncbi.nlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="16"
                             height="29"
                             focusable="false"
                             aria-hidden="true"
                             viewBox="0 0 16 29"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg">
                            <path d="M3.8809 21.4002C3.8809 19.0932 3.8809 16.7876 3.8809 14.478C3.8809 14.2117 3.80103 14.1452 3.54278 14.1492C2.53372 14.1638 1.52334 14.1492 0.514288 14.1598C0.302626 14.1598 0.248047 14.0972 0.248047 13.8936C0.256034 12.4585 0.256034 11.0239 0.248047 9.58978C0.248047 9.37013 0.302626 9.30224 0.528931 9.3049C1.53798 9.31688 2.54837 9.3049 3.55742 9.31555C3.80103 9.31555 3.8809 9.26097 3.87957 9.00272C3.87158 8.00565 3.85428 7.00592 3.90753 6.00884C3.97142 4.83339 4.31487 3.73115 5.04437 2.78467C5.93095 1.63318 7.15699 1.09005 8.56141 0.967577C10.5582 0.79319 12.555 0.982221 14.5518 0.927641C14.7102 0.927641 14.7462 0.99287 14.7449 1.13664C14.7449 2.581 14.7449 4.02668 14.7449 5.47104C14.7449 5.67604 14.6517 5.68669 14.4946 5.68669C13.4523 5.68669 12.4113 5.68669 11.3703 5.68669C10.3506 5.68669 9.92057 6.10868 9.90593 7.13904C9.89661 7.7647 9.91525 8.39303 9.89794 9.01869C9.88995 9.26364 9.96583 9.31822 10.2015 9.31688C11.7204 9.30623 13.2393 9.31688 14.7595 9.3049C15.0257 9.3049 15.0723 9.3728 15.0444 9.62439C14.89 10.9849 14.7515 12.3467 14.6144 13.7085C14.5691 14.1571 14.5785 14.1585 14.1458 14.1585C12.8386 14.1585 11.5313 14.1665 10.2254 14.1518C9.95119 14.1518 9.89794 14.2317 9.89794 14.4899C9.90593 19.0799 9.89794 23.6752 9.91125 28.2612C9.91125 28.5674 9.8407 28.646 9.53186 28.6433C7.77866 28.6273 6.02414 28.6366 4.27094 28.634C3.82499 28.634 3.87158 28.6992 3.87158 28.22C3.87602 25.9472 3.87913 23.6739 3.8809 21.4002Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on Facebook</span>
                    </a>
                

                
                    <a href="https://www.linkedin.com/company/ncbinlm"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="25"
                             height="23"
                             viewBox="0 0 26 24"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M14.6983 9.98423C15.6302 9.24808 16.5926 8.74754 17.6762 8.51991C19.673 8.09126 21.554 8.30824 23.1262 9.7526C24.2351 10.7723 24.7529 12.1115 25.0165 13.5612C25.1486 14.3363 25.2105 15.1218 25.2015 15.9081C25.2015 18.3043 25.2015 20.6898 25.2082 23.0806C25.2082 23.3468 25.1549 23.444 24.8621 23.4414C23.1297 23.4272 21.3992 23.4272 19.6704 23.4414C19.4041 23.4414 19.3429 23.3588 19.3442 23.1019C19.3535 20.5194 19.3442 17.9368 19.3442 15.3543C19.3442 14.0005 18.3258 12.9448 17.0266 12.9488C15.7273 12.9528 14.6983 14.0071 14.6983 15.361C14.6983 17.9328 14.6917 20.5047 14.6983 23.0753C14.6983 23.3708 14.6198 23.444 14.3296 23.4427C12.6185 23.4294 10.9079 23.4294 9.19779 23.4427C8.93155 23.4427 8.86099 23.3735 8.86232 23.1086C8.8783 19.7619 8.88628 16.4144 8.88628 13.066C8.88628 11.5688 8.87874 10.0708 8.86365 8.57182C8.86365 8.3575 8.90758 8.27896 9.14054 8.28029C10.9048 8.29094 12.6687 8.29094 14.4321 8.28029C14.6464 8.28029 14.6983 8.34818 14.6983 8.54653C14.6903 9.00047 14.6983 9.45441 14.6983 9.98423Z">
                            </path>
                            <path d="M6.55316 15.8443C6.55316 18.2564 6.55316 20.6699 6.55316 23.082C6.55316 23.3629 6.48127 23.4388 6.19906 23.4374C4.47737 23.4241 2.75568 23.4241 1.03399 23.4374C0.767751 23.4374 0.69986 23.3629 0.701191 23.1006C0.709178 18.2648 0.709178 13.4281 0.701191 8.59053C0.701191 8.34026 0.765089 8.27237 1.01669 8.2737C2.74991 8.28435 4.48048 8.28435 6.20838 8.2737C6.47462 8.2737 6.5465 8.33627 6.54517 8.6065C6.54783 11.0186 6.55316 13.4308 6.55316 15.8443Z">
                            </path>
                            <path d="M3.65878 0.243898C5.36804 0.243898 6.58743 1.45529 6.58743 3.1406C6.58743 4.75801 5.32145 5.95742 3.60819 5.96807C3.22177 5.97614 2.83768 5.90639 2.47877 5.76299C2.11985 5.61959 1.79344 5.40546 1.51897 5.13334C1.24449 4.86123 1.02755 4.53668 0.881058 4.17902C0.734563 3.82136 0.661505 3.43788 0.666231 3.05141C0.67555 1.42601 1.9362 0.242566 3.65878 0.243898Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on LinkedIn</span>
                    </a>
                

                
                    <a href="https://github.com/ncbi"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="28"
                             height="27"
                             viewBox="0 0 28 28"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M16.7228 20.6334C17.5057 20.5527 18.2786 20.3944 19.0301 20.1608C21.3108 19.4193 22.5822 17.8259 22.963 15.4909C23.1228 14.5112 23.1814 13.5287 22.9883 12.5437C22.8106 11.6423 22.4013 10.8028 21.8007 10.1076C21.7526 10.0605 21.7197 10 21.7064 9.934C21.6931 9.86799 21.7 9.79952 21.7262 9.73748C22.0856 8.6206 21.9711 7.51969 21.601 6.42677C21.582 6.3497 21.5345 6.2827 21.468 6.23923C21.4016 6.19577 21.3211 6.17906 21.2429 6.19248C20.7329 6.21649 20.2313 6.33051 19.7611 6.52928C19.1103 6.7908 18.4899 7.12198 17.9104 7.51703C17.84 7.56996 17.7581 7.60551 17.6713 7.62078C17.5846 7.63605 17.4954 7.6306 17.4112 7.60489C15.2596 7.05882 13.0054 7.06203 10.8554 7.61421C10.7806 7.63586 10.7018 7.63967 10.6253 7.62534C10.5487 7.611 10.4766 7.57892 10.4148 7.53167C9.64788 7.03247 8.85171 6.58918 7.96368 6.33359C7.65781 6.24338 7.34123 6.19458 7.02239 6.18849C6.94879 6.17986 6.87462 6.19893 6.81432 6.242C6.75402 6.28507 6.71191 6.34904 6.69621 6.42145C6.32342 7.51437 6.2209 8.61527 6.56307 9.73348C6.59635 9.84264 6.64694 9.93316 6.54177 10.0516C5.47666 11.2604 5.09988 12.6834 5.19574 14.2676C5.2663 15.4244 5.46201 16.5466 6.01454 17.5769C6.84399 19.1171 8.21664 19.9119 9.85158 20.3352C10.3938 20.4706 10.9444 20.5698 11.4998 20.632C11.5384 20.7492 11.4506 20.7798 11.408 20.8291C11.1734 21.1179 10.9894 21.4441 10.8634 21.7942C10.7622 22.0458 10.8315 22.4039 10.6065 22.5516C10.263 22.7766 9.83827 22.8485 9.42421 22.8871C8.17936 23.0056 7.26471 22.4877 6.6283 21.4348C6.25552 20.8184 5.76956 20.3325 5.08523 20.0663C4.76981 19.9325 4.42139 19.8967 4.08537 19.9638C3.7898 20.029 3.73788 20.1901 3.93891 20.4111C4.03639 20.5234 4.14989 20.6207 4.27575 20.6999C4.9796 21.1318 5.51717 21.7884 5.80152 22.5636C6.37002 23.9973 7.48039 24.5697 8.93825 24.6323C9.43741 24.6575 9.93768 24.615 10.4254 24.5058C10.5892 24.4672 10.6531 24.4872 10.6517 24.6762C10.6451 25.4936 10.6637 26.3123 10.6517 27.131C10.6517 27.6635 10.1684 27.9297 9.58663 27.7393C8.17396 27.2671 6.84977 26.5631 5.66838 25.656C2.59555 23.2891 0.720966 20.1861 0.217704 16.3376C-0.357453 11.9127 0.911353 8.00824 3.98551 4.73881C6.11909 2.42656 8.99932 0.939975 12.1203 0.540191C16.5351 -0.0601815 20.4347 1.14323 23.7232 4.16373C26.2449 6.47869 27.724 9.37672 28.1048 12.7726C28.5828 17.0325 27.3686 20.7945 24.4768 23.9827C22.9762 25.6323 21.0956 26.8908 18.9982 27.6488C18.8783 27.6927 18.7585 27.738 18.636 27.7726C18.0356 27.9404 17.6189 27.6395 17.6189 27.0098C17.6189 25.7452 17.6308 24.4806 17.6295 23.2159C17.6329 22.9506 17.6128 22.6856 17.5696 22.4238C17.4325 21.6664 17.3419 21.484 16.7228 20.6334Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI on GitHub</span>
                    </a>
                

                
                    <a href="https://ncbiinsights.ncbi.nlm.nih.gov/"
                       class="ncbi-footer__social-icon ncbi-footer__social-icon--gray"
                       target="_blank"
                       rel="noreferrer noopener">
                        <svg width="26"
                             height="26"
                             viewBox="0 0 27 27"
                             fill="none"
                             xmlns="http://www.w3.org/2000/svg"
                             focusable="false"
                             aria-hidden="true">
                            <path d="M23.7778 26.4574C23.1354 26.3913 22.0856 26.8024 21.636 26.3087C21.212 25.8444 21.4359 24.8111 21.324 24.0347C19.9933 14.8323 14.8727 8.80132 6.09057 5.85008C4.37689 5.28406 2.58381 4.99533 0.779072 4.99481C0.202773 4.99481 -0.0229751 4.83146 0.00455514 4.21479C0.0660406 3.08627 0.0660406 1.95525 0.00455514 0.826734C-0.0413285 0.0815827 0.259669 -0.0193618 0.896534 0.00266238C6.96236 0.222904 12.3693 2.24179 16.9889 6.16209C22.9794 11.2478 26.1271 17.7688 26.4372 25.648C26.4629 26.294 26.3179 26.5271 25.6609 26.4684C25.0827 26.417 24.4991 26.4574 23.7778 26.4574Z">
                            </path>
                            <path d="M14.8265 26.441C14.0924 26.441 13.2371 26.6795 12.6626 26.3786C12.0092 26.0372 12.3781 25.0644 12.246 24.378C11.1154 18.5324 6.6849 14.5497 0.74755 14.1001C0.217135 14.0615 -0.0104482 13.9422 0.0134113 13.3659C0.0519536 12.1454 0.0482829 10.9213 0.0134113 9.69524C-0.00127145 9.14464 0.196946 9.03268 0.703502 9.04736C9.21217 9.27128 16.5994 16.2511 17.2804 24.7231C17.418 26.4446 17.418 26.4446 15.6579 26.4446H14.832L14.8265 26.441Z">
                            </path>
                            <path d="M3.58928 26.4555C2.64447 26.4618 1.73584 26.0925 1.06329 25.4289C0.39073 24.7653 0.00933763 23.8617 0.0030097 22.9169C-0.00331824 21.9721 0.365937 21.0635 1.02954 20.3909C1.69315 19.7184 2.59675 19.337 3.54156 19.3306C4.48637 19.3243 5.39499 19.6936 6.06755 20.3572C6.7401 21.0208 7.1215 21.9244 7.12782 22.8692C7.13415 23.814 6.7649 24.7226 6.10129 25.3952C5.43768 26.0677 4.53409 26.4491 3.58928 26.4555Z">
                            </path>
                        </svg>
                        <span class="usa-sr-only">NCBI RSS feed</span>
                    </a>
                
            </div>
        </div>
    

    <div data-testid="gridContainer"
         class="grid-container ncbi-footer__container">
        <div class="grid-row ncbi-footer__main-content-container"
             data-testid="grid">
            
                <div class="ncbi-footer__column">
                    
                        <p class="ncbi-footer__circled-icons-heading">
                            Connect with NLM
                        </p>
                    

                    <div class="ncbi-footer__circled-icons-list">
                        
                            <a href=https://twitter.com/nlm_nih class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="32"
                                     height="32"
                                     viewBox="0 0 40 40"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="m6.067 8 10.81 13.9L6 33.2h4.2l8.4-9.1 7.068 9.1H34L22.8 18.5 31.9 8h-3.5l-7.7 8.4L14.401 8H6.067Zm3.6 1.734h3.266l16.8 21.732H26.57L9.668 9.734Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on X (formerly known as Twitter)</span>
                            </a>
                        

                        
                            <a href=https://www.facebook.com/nationallibraryofmedicine class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="13"
                                     height="24"
                                     viewBox="0 0 13 24"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M4.11371 23.1369C4.11371 23.082 4.11371 23.0294 4.11371 22.9745V12.9411H0.817305C0.6709 12.9411 0.670898 12.9411 0.670898 12.8016C0.670898 11.564 0.670898 10.3287 0.670898 9.09341C0.670898 8.97903 0.705213 8.95158 0.815017 8.95158C1.8673 8.95158 2.91959 8.95158 3.97417 8.95158H4.12057V8.83263C4.12057 7.8055 4.12057 6.7738 4.12057 5.74897C4.1264 4.92595 4.31387 4.11437 4.66959 3.37217C5.12916 2.38246 5.94651 1.60353 6.95717 1.1921C7.64827 0.905008 8.3913 0.764035 9.13953 0.778051C10.0019 0.791777 10.8644 0.830666 11.7268 0.860404C11.8869 0.860404 12.047 0.894717 12.2072 0.90158C12.2964 0.90158 12.3261 0.940469 12.3261 1.02968C12.3261 1.5421 12.3261 2.05452 12.3261 2.56465C12.3261 3.16857 12.3261 3.7725 12.3261 4.37642C12.3261 4.48165 12.2964 4.51367 12.1912 4.51138C11.5369 4.51138 10.8804 4.51138 10.2261 4.51138C9.92772 4.51814 9.63058 4.5526 9.33855 4.61433C9.08125 4.6617 8.84537 4.78881 8.66431 4.97766C8.48326 5.16652 8.3662 5.40755 8.32972 5.66661C8.28476 5.89271 8.26027 6.1224 8.25652 6.35289C8.25652 7.19014 8.25652 8.02969 8.25652 8.86923C8.25652 8.89439 8.25652 8.91955 8.25652 8.95615H12.0219C12.1797 8.95615 12.182 8.95616 12.1614 9.10714C12.0768 9.76596 11.9876 10.4248 11.9029 11.0813C11.8312 11.6319 11.7626 12.1824 11.697 12.733C11.6719 12.9434 11.6787 12.9434 11.4683 12.9434H8.26338V22.899C8.26338 22.979 8.26338 23.0591 8.26338 23.1392L4.11371 23.1369Z">
                                    </path>
                                </svg>
                                <span class="usa-sr-only">NLM on Facebook</span>
                            </a>
                        

                        
                            <a href=https://www.youtube.com/user/NLMNIH class="ncbi-footer__social-icon ncbi-footer__social-icon--circled" target="_blank" rel="noreferrer noopener">
                                <svg width="21"
                                     height="15"
                                     viewBox="0 0 21 15"
                                     fill="none"
                                     xmlns="http://www.w3.org/2000/svg"
                                     focusable="false"
                                     aria-hidden="true">
                                    <path d="M19.2561 1.47914C18.9016 1.15888 18.5699 0.957569 17.2271 0.834039C15.5503 0.678484 13.2787 0.655608 11.563 0.65332H9.43556C7.71987 0.65332 5.4483 0.678484 3.77151 0.834039C2.43098 0.957569 2.097 1.15888 1.74242 1.47914C0.813665 2.32097 0.619221 4.62685 0.598633 6.89384C0.598633 7.31781 0.598633 7.74101 0.598633 8.16345C0.626084 10.4121 0.827391 12.686 1.74242 13.521C2.097 13.8412 2.4287 14.0425 3.77151 14.1661C5.4483 14.3216 7.71987 14.3445 9.43556 14.3468H11.563C13.2787 14.3468 15.5503 14.3216 17.2271 14.1661C18.5676 14.0425 18.9016 13.8412 19.2561 13.521C20.1712 12.6929 20.3725 10.451 20.3999 8.22064C20.3999 7.74025 20.3999 7.25986 20.3999 6.77946C20.3725 4.54907 20.1689 2.30724 19.2561 1.47914ZM8.55942 10.5311V4.65201L13.5601 7.50005L8.55942 10.5311Z"
                                          fill="white" />
                                </svg>
                                <span class="usa-sr-only">NLM on YouTube</span>
                            </a>
                        
                    </div>
                </div>
            

            
                <address class="ncbi-footer__address ncbi-footer__column">
                    
        <p>
            <a class="usa-link usa-link--external"
            href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/%4038.9959508,
            -77.101021,17z/data%3D!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb%3A0x19156f88b27635b8!8m2!3d38.9959508!
            4d-77.0988323"
            rel="noopener noreferrer" target="_blank">National Library of Medicine
            <br/> 8600 Rockville Pike<br/> Bethesda, MD 20894</a>
        </p>
    
                </address>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/web_policies.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Web Policies
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            FOIA
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS Vulnerability Disclosure
        

        
    </a>


                        </li>
                    
                </ul>
            

            
                <ul class="usa-list usa-list--unstyled ncbi-footer__vertical-list ncbi-footer__column">
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://support.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Help
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/accessibility.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Accessibility
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__vertical-list-item">
                            









    <a href="https://www.nlm.nih.gov/careers/careers.html" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            Careers
        

        
    </a>


                        </li>
                    
                </ul>
            
        </div>

        
            <div class="grid-row grid-col-12" data-testid="grid">
                <ul class="ncbi-footer__bottom-links-list">
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nlm.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NLM
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.nih.gov/" class="usa-link  usa-link--alt ncbi-footer__link"  >
        

        
            NIH
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.hhs.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            HHS
        

        
    </a>


                        </li>
                    
                        <li class="ncbi-footer__bottom-list-item">
                            









    <a href="https://www.usa.gov/" class="usa-link usa-link--external usa-link--alt ncbi-footer__link" rel="noreferrer noopener" target='_blank' >
        

        
            USA.gov
        

        
    </a>


                        </li>
                    
                </ul>
            </div>
        
    </div>
</footer>

    


        
        
    
  <script  type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


    
        

<button class="back-to-top" data-ga-category="pagination" data-ga-action="back_to_top">
    <label>Back to Top</label>
    <svg class="usa-icon order-0" aria-hidden="true" focusable="false" role="img">
        <use xlink:href="/static/img/sprite.svg#arrow_upward"></use>
    </svg>
</button>
    


        
    
    
    
        
    <script type="application/javascript">
    window.ncbi = window.ncbi || {};
    window.ncbi.pmc = window.ncbi.pmc || {};
    window.ncbi.pmc.options = {
        logLevel: 'INFO',
        
        staticEndpoint: '/static/',
        
        citeCookieName: 'pmc-cf',
    };
</script>
    <script type="module" crossorigin="" src="/static/assets/base-133c6271.js"></script>
    
    <script type="text/javascript" src="https://cdn.ncbi.nlm.nih.gov/core/jquery/jquery-3.6.0.min.js">&#xA0;</script>
    <script type="text/javascript">
        jQuery.getScript("https://cdn.ncbi.nlm.nih.gov/core/alerts/alerts.js", function () {
            galert(['div.nav_and_browser', 'div.header', '#universal_header', '.usa-banner', 'body > *:nth-child(1)'])
        });
    </script>


    <script type="text/javascript">var exports = {};</script>
     <script src="/static/CACHE/js/output.13b077bc3ffd.js"></script>

    <script type="module" crossorigin="" src="/static/assets/article-44d04358.js"></script>
    
    

    </body>
</html>
